<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/3/933" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/3/933.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/3/933.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/3/933.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/3/933/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/3/933.atom" />
<meta name="HW.pisa" content="ggg;8/3/933" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition" />
<meta name="DC.Identifier" content="10.1534/g3.117.300354" />
<meta name="DC.Date" content="2018-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2018 Elias et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Plants competing for available resources is an unavoidable phenomenon in a field. We conducted studies in cassava ( Manihot esculenta Crantz) in order to understand the pattern of this competition. Taking into account the competitive ability of genotypes while selecting parents for breeding advancement or commercialization can be very useful. We assumed that competition could occur at two levels: (i) the genotypic level, which we call interclonal, and (ii) the plot level irrespective of the type of genotype, which we call interplot competition or competition error. Modification in incidence matrices was applied in order to relate neighboring genotype/plot to the performance of a target genotype/plot with respect to its competitive ability. This was added into a genomic selection (GS) model to simultaneously predict the direct and competitive ability of a genotype. Predictability of the models was tested through a 10-fold cross-validation method repeated five times. The best model was chosen as the one with the lowest prediction root mean squared error (pRMSE) compared to that of the base model having no competitive component. Results from our real data studies indicated that &lt;10% increase in accuracy was achieved with GS-interclonal competition model, but this value reached up to 25% with a GS-competition error model. We also found that the competitive influence of a cassava clone is not just limited to the adjacent neighbors but spreads beyond them. Through simulations, we found that a 26% increase of accuracy in estimating trait genotypic effect can be achieved even in the presence of high competitive variance." />
<meta name="DC.Contributor" content="Ani A. Elias" />
<meta name="DC.Contributor" content="Ismail Rabbi" />
<meta name="DC.Contributor" content="Peter Kulakow" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="article:published_time" content="2018-03-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Plants competing for available resources is an unavoidable phenomenon in a field. We conducted studies in cassava (&lt;i&gt;Manihot esculenta&lt;/i&gt; Crantz) in order to understand the pattern of this competition. Taking into account the competitive ability of genotypes while selecting parents for breeding advancement or commercialization can be very useful. We assumed that competition could occur at two levels: (i) the genotypic level, which we call interclonal, and (ii) the plot level irrespective of the type of genotype, which we call interplot competition or competition error. Modification in incidence matrices was applied in order to relate neighboring genotype/plot to the performance of a target genotype/plot with respect to its competitive ability. This was added into a genomic selection (GS) model to simultaneously predict the direct and competitive ability of a genotype. Predictability of the models was tested through a 10-fold cross-validation method repeated five times. The best model was chosen as the one with the lowest prediction root mean squared error (pRMSE) compared to that of the base model having no competitive component. Results from our real data studies indicated that &amp;lt;10% increase in accuracy was achieved with GS-interclonal competition model, but this value reached up to 25% with a GS-competition error model. We also found that the competitive influence of a cassava clone is not just limited to the adjacent neighbors but spreads beyond them. Through simulations, we found that a 26% increase of accuracy in estimating trait genotypic effect can be achieved even in the presence of high competitive variance.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/03/01" />
<meta name="citation_mjid" content="ggg;8/3/933" />
<meta name="citation_id" content="8/3/933" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/3/933" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/3/933.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/3/933.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/3/933.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300354" />
<meta name="citation_pmid" content="29358232" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="933" />
<meta name="citation_lastpage" content="944" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ani A. Elias" />
<meta name="citation_author_institution" content="Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_email" content="anianna01@gmail.com" />
<meta name="citation_author_email" content="jeanluc.work@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3360-6979" />
<meta name="citation_author" content="Ismail Rabbi" />
<meta name="citation_author_institution" content="International Institute of Tropical Agriculture, Ibadan 200001, Nigeria and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9966-2941" />
<meta name="citation_author" content="Peter Kulakow" />
<meta name="citation_author_institution" content="International Institute of Tropical Agriculture, Ibadan 200001, Nigeria and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7574-2645" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Station (USDA-ARS), Robert W. Holley Center for Agriculture and Health, Ithaca, New York 14853-2901" />
<meta name="citation_author_email" content="anianna01@gmail.com" />
<meta name="citation_author_email" content="jeanluc.work@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4849-628X" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=J. Besag;citation_author=R. Kempton;citation_title=Statistical analysis of field experiments using neighbouring plots.;citation_pages=231;citation_volume=42;citation_year=1986;citation_doi=10.2307/2531047" />
<meta name="citation_reference" content="citation_author=M. Cannell;citation_pages=120-148;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=E. P. Cappa;citation_author=R. J. Cantet;citation_title=Direct and competition additive effects in tree breeding: Bayesian estimation from an individual tree mixed model.;citation_pages=45-55;citation_volume=57;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_title=A novel individual-tree mixed model to account for competition and environmental heterogeneity: a Bayesian approach.;citation_author=E. P. Cappa;citation_author=F. MuÃ±oz;citation_author=L. Sanchez;citation_author=R. J. C. Cantet;citation_volume=11;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_title=Identification and joint modeling of competition effects and environmental heterogeneity in three Douglas-fir (Pseudotsuga menziesii var. menziesii) trials.;citation_author=E. P. Cappa;citation_author=M. U. Stoehr;citation_author=C.-Y. Xie;citation_author=A. D. Yanchuk;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Cassava: New Potential for a Neglected Crop;citation_author=J. H. Cock;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Biol.;citation_author=T. Connolly;citation_author=I. D. Currie;citation_author=J. E. Bradshaw;citation_author=J. W. McNicol;citation_title=Inter-plot competition in yield trials of potatoes (Solanum tuberosum L.) with single-drill plots.;citation_pages=367-377;citation_volume=123;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=B. R. Cullis;citation_author=A. B. Smith;citation_author=N. E. Coombes;citation_title=On the design of early generation variety trials with correlated data.;citation_pages=381-393;citation_volume=11;citation_year=2006;citation_doi=10.1198/108571106X154443" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=O. David;citation_author=R. A. Kempton;citation_author=I. M. Nevison;citation_title=Designs for controlling interplot competition in variety trials.;citation_pages=285;citation_volume=127;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Rev. Mat. Estat.;citation_author=M. D. V. de Resende;citation_author=J. Stringer;citation_author=B. Cullis;citation_author=R. Thompson;citation_title=Joint modelling of competition and spatial variability in forest field trials.;citation_pages=7-22;citation_volume=23;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=C. M. Donald;citation_title=The breeding of crop ideotypes.;citation_pages=385-403;citation_volume=17;citation_year=1968;citation_doi=10.1007/BF00056241" />
<meta name="citation_reference" content="citation_journal_title=Appl. Stat.;citation_author=N. R. Draper;citation_author=I. Guttman;citation_title=Incorporating overlap effects from neighbouring units into response surface models.;citation_pages=128;citation_volume=29;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=M. Durban;citation_author=I. D. Currie;citation_author=R. A. Kempton;citation_title=Adjusting for fertility and competition in variety trials.;citation_pages=129-140;citation_volume=136;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=A. A. Elias;citation_author=I. Rabbi;citation_author=P. Kulakow;citation_author=J.-L. Jannink;citation_title=Improving genomic prediction in cassava field experiments using spatial analysis.;citation_volume=8;citation_year=2018;citation_pmid=29109156;citation_doi=10.1534/g3.117.300323" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=A. R. Gilmour;citation_author=B. R. Cullis;citation_author=A. P. Verbyla;citation_author=A. P. Verbyla;citation_title=Accounting for natural and extraneous variation in the analysis of field experiments.;citation_pages=269;citation_volume=2;citation_year=1997;citation_doi=10.2307/1400446" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=K. Hinson;citation_author=W. Hanson;citation_title=Competition Studies in Soybeans;citation_pages=117-123;citation_volume=2;citation_year=1962;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=C. H. Hunt;citation_author=A. B. Smith;citation_author=D. R. Jordan;citation_author=B. R. Cullis;citation_title=Predicting additive and non-additive genetic effects from trials where traits are affected by interplot competition.;citation_pages=53-63;citation_volume=18;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Plant Prod. Sci.;citation_author=Y. Izumi;citation_author=E. Yuliadi;citation_author=S. Sunyoto;citation_author=M. Iijima;citation_title=Root system development including root branching in cuttings of cassava with reference to shoot growth and tuber bulking.;citation_pages=267-272;citation_volume=2;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=R. A. Kempton;citation_title=Adjustment for competition between varieties in plant breeding trials.;citation_pages=599;citation_volume=98;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=R. A. Kempton;citation_author=G. Lockwood;citation_title=Inter-plot competition in variety trials of field beans (Vicia faba L.).;citation_pages=293;citation_volume=103;citation_year=1984;citation_doi=10.1017/S0021859600047249" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=D. Ly;citation_author=M. Hamblin;citation_author=I. Rabbi;citation_author=G. Melaku;citation_author=M. Bakare;citation_title=Relatedness and genotype Ã environment interaction affect prediction accuracies in genomic selection: a study in cassava.;citation_pages=1312;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.11.0653" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=S. Magnussen;citation_title=A method to adjust simultaneously for spatial microsite and competition effects.;citation_pages=985-995;citation_volume=24;citation_year=1994;citation_doi=10.1139/x94-129" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. M. Muir;citation_title=Incorporation of competitive effects in forest tree or animal breeding programs.;citation_volume=170;citation_year=2005;citation_pmid=15911590;citation_doi=10.1534/genetics.104.035956" />
<meta name="citation_reference" content="citation_journal_title=Plant Soil;citation_author=J. R. Pardales;citation_author=A. Yamauchi;citation_title=REGULATION OF ROOT DEVELOPMENT IN SWEETPOTATO AND CASSAVA BY SOIL MOISTURE DURING THEIR ESTABLISHMENT PERIOD;citation_pages=201-208;citation_volume=255;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=S. C. Pearce;citation_title=EXPERIMENTING WITH ORGANISMS AS BLOCKS;citation_pages=141-149;citation_volume=44;citation_year=1957;citation_issue=1-2;citation_doi=10.1093/biomet/44.1-2.141" />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=G. J. Rebetzke;citation_author=R. T. A. Fischer;citation_author=A. F. van Herwaarden;citation_author=D. G. Bonnett;citation_author=K. Chenu;citation_title=Plot size matters: interference from intergenotypic competition in plant phenotyping studies.;citation_pages=107;citation_volume=41;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=D. O. Stram;citation_author=J. W. Lee;citation_title=Variance components testing in the longitudinal mixed effects model.;citation_pages=1171-1177;citation_volume=50;citation_year=1994;citation_issue=4;citation_pmid=7786999;citation_doi=10.2307/2533455" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=J. K. Stringer;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_title=Joint modeling of spatial variability and within-row interplot competition to increase the efficiency of plant improvement.;citation_pages=269-281;citation_volume=16;citation_year=2011" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
<meta name="twitter:description" content="Plants competing for available resources is an unavoidable phenomenon in a field. We conducted studies in cassava ( Manihot esculenta Crantz) in order to understand the pattern of this competition. Taking into account the competitive ability of genotypes while selecting parents for breeding advancement or commercialization can be very useful. We assumed that competition could occur at two levels: (i) the genotypic level, which we call interclonal, and (ii) the plot level irrespective of the type of genotype, which we call interplot competition or competition error. Modification in incidence matrices was applied in order to relate neighboring genotype/plot to the performance of a target genotype/plot with respect to its competitive ability. This was added into a genomic selection (GS) model to simultaneously predict the direct and competitive ability of a genotype. Predictability of the models was tested through a 10-fold cross-validation method repeated five times. The best model was chosen as the one with the lowest prediction root mean squared error (pRMSE) compared to that of the base model having no competitive component. Results from our real data studies indicated that &lt;10% increase in accuracy was achieved with GS-interclonal competition model, but this value reached up to 25% with a GS-competition error model. We also found that the competitive influence of a cassava clone is not just limited to the adjacent neighbors but spreads beyond them. Through simulations, we found that a 26% increase of accuracy in estimating trait genotypic effect can be achieved even in the presence of high competitive variance." />
<meta name="og-title" property="og:title" content="Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/3/933" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Plants competing for available resources is an unavoidable phenomenon in a field. We conducted studies in cassava ( Manihot esculenta Crantz) in order to understand the pattern of this competition. Taking into account the competitive ability of genotypes while selecting parents for breeding advancement or commercialization can be very useful. We assumed that competition could occur at two levels: (i) the genotypic level, which we call interclonal, and (ii) the plot level irrespective of the type of genotype, which we call interplot competition or competition error. Modification in incidence matrices was applied in order to relate neighboring genotype/plot to the performance of a target genotype/plot with respect to its competitive ability. This was added into a genomic selection (GS) model to simultaneously predict the direct and competitive ability of a genotype. Predictability of the models was tested through a 10-fold cross-validation method repeated five times. The best model was chosen as the one with the lowest prediction root mean squared error (pRMSE) compared to that of the base model having no competitive component. Results from our real data studies indicated that &lt;10% increase in accuracy was achieved with GS-interclonal competition model, but this value reached up to 25% with a GS-competition error model. We also found that the competitive influence of a cassava clone is not just limited to the adjacent neighbors but spreads beyond them. Through simulations, we found that a 26% increase of accuracy in estimating trait genotypic effect can be achieved even in the presence of high competitive variance." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
    <title>Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Rnyr_lgDZQZzPUQs_DGyKMH3KXThSLGghza9i4T0FIM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488767","apath":"\/ggg\/8\/3\/933.atom","pisa":"ggg;8\/3\/933","ac":{"\/ggg\/8\/3\/933.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/3\/933.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/3\/933"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/3\/933"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/3\/933":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022syjqZJ4YiCYeAaSHuAUXneQLw7HZYmFzeYvcjn\\\\\\\/Ynmvd8LME0O+s3yuviszOxN\\\\\\\/7gaNqjHkYnmbfLriX+1WHlRHq0cDSTL+\\\\\\\/fwmrU\\\\\\\/jqFhY4FWv+mP5BVRJlSp6gRlA5lmULwGRpPCr91VisecToCaSzUMzCNpyw6UqS4S2jq4lXI1C5DGH2elWaZydDZSP5qNo0xhTt+berpq1NDmb7QqYObBcUBpkO5t49f4Q2774mvFMM+qjBPy6EupyIGpSW4ZaPHIoFmnWuTjnA+bDShcqCj+ZiG7c5D05OZcNk1x8JGzv6CoqGeQ9mtXqQ05zK9RuhoeOnMx404YehDqSPjX9NltpwYjrNoMWJsEchGUbcHrsZ5IitCXSm0CAqSUj1pZ\\\\\\\/XctbGdYpia80dPZerFJgkbYxkwzbK3fGh7D2JSRtC1DorCZIIakRgvdUY9hUWOPvxzHPPpbrFkydmGwVb7YbeCeO8cjRW3jjG0Q4eYr5bWrICF0aNwQQKGmso1Odo90+ckEMLrxBTG8D3P4ZhAEVTHKBAesGE9GJOZFPWryLNPPm9Bq+Wbz9+PIyGI15PHm88SSHevz3yDohtX4dOzzd+ws9UQMjv5mkWsESt+J1pYUSaooSHxgWmFDdLcfQ+odrdJNds4BH98mH\\\\\\\/A0ur7crUVn5yPcJUVtbqTwIrt8WuNqbI7r91Zf1pxx+7I7dPyq4sa+IIPGXhq0iT9ouZ4fLV4LjnPwCQ6KH84iTPsb+L5ySa7IP+1jfMECKpp4c7xw0VXfITOwdKBawOoEADVUpmEesIUhIphXm3JfDxicQJ7O689re8qdxPOL\\\\\\\/byN3oY\\\\\\\/0WY\\\\\\\/\\\\\\\/\\\\\\\/b+MEs1n4gnB\\\\\\\/7fJ5noFn49paBazfGrrKkUmkiOmAacfT0e0IA0MctPFIytZ9A2VH8Q7JKpmgghS61IGzSz1QlROr23ySbuZAg2vLJQ7E3RV+H0mg4Tt5EutrAAm4I5S+Q14J5uGIdnXE66SRyn2qT2NOyNn0u3MhIJGqrGaLSmsfSTturjPnXFI23EYG\\\\\\\/INVufb9P6jNlJPcqQJbOevdvh8D0d8LN0YnQN02he1BWeUJ9GHzN5pS30L+CbZf7jyq7MKiSRFv01XrZHPRdk6PDIrDWP0arFGg6EQjDK2jltIatI0LG7h4Emlwgx6kpbdkvjDxxn54Lr6gZMDRV0V26zBUlet4BYmet3SU4zcKRoXcPbtE\\\\\\\/yJTSVN6gEpAb3HIe+HWgJktg\\\\\\\/kAMPEG2mRi9DJMQoiobYW6l7V4aZVufbFvWflv7BsL\\\\\\\/CF+\\\u0022,\\\u0022iv\\\u0022:\\\u00228ii\\\\\\\/i8aB3m7KKEpkLp7Yeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c4a48ef8789a4088652ba1839693aa8f99b9c4a739f54ab15559a37b25b52f19\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Sug7QzGqdwvVcVc3vpou\\\\\\\/1H4Yl\\\\\\\/uyuW3+MXXeL7SpVFQAPS6I4\\\\\\\/k63iCQFCWJifd853JlDuKQGX1nV9\\\\\\\/xuuwWAO\\\\\\\/e+GgrL2EA5MPcfwW\\\\\\\/UgNm2j+WzLiOMOr9j7t2N7LL2HnEfWfmvpB3mHjDT6IqOruvHeVvVlllNQnIl11qAE2+7ZTYnn44hr0WKk6WAKe3OHv+5ammSDcoMyD6R33OjrmASJFi3ejCpDvXZPg6Mvt\\\\\\\/W9D8ryC9anicbUiUAspCTxq0VKIgxWhAPAlwYBYq7HHYsEUrMC3xhigoVXaHH5coL7FrHcIKbbwFRQsmxCiY6XedS8BoZN42XFzpVlg4gmvJiTt0J+SP2kVxgzHYd0bLwmFUKagInTPczTXR6R\\\\\\\/v7dDen1G1OZn5mmjQf4wOrg3vacWWTtz8+lg1hE7bq4aqC1YXrRtA2w\\\\\\\/\\\\\\\/CkM0fhfVgZhPagX4xl\\\\\\\/uMmtJbvy75E36P6jl39ZalzoNuEC\\\\\\\/zUS8GQ0MgcMaAKXIFN9+qrY7fieg3145hUxoyGjjJvGQBFqZ0+US3FGHUhbmO64ZViYOdY\\\\\\\/LUQWqhbXmc1eDEWBvlXeQj2USMF3ecb18hG4OW\\\\\\\/L508LyHUxG72L4VPJtd8EZMvGlfmJEsUXZui3Dp0QajYdYoKh1JpXz+6PNaQ1oJoGZmANgk9w4DyEnvRRDNVWyxRMf49mudQUxyQzpjssSD7i48xGk87CTmW\\\\\\\/HSOSdphmBg09y+ct+DHsZ+V81VaYsCiCtNaNpiyxEdgcMFysv8kwYLN7jiZ3yK61eD3GJthLuTYR+Vzm6LGRft1oa2zTklEP9Ez1lVOve097AiBeUbm4OW5v6JIkeYh24geNz97Hl+i5CzchG+PY\\\\\\\/03hUZW51paOYuBhvYNvwDTpPeHzj+qvQZyiDy50k6G7obF\\\\\\\/eI5XPCpsa9vAFPmi7Ibod3ws1nD42cb2z1U\\\\\\\/cemRK6AjRyp9sKjTNmi6j3FGsP872L+uGDAwsdMh\\\\\\\/kp88VdHq1CvlMDZmwM+XNtOh\\\\\\\/\\\\\\\/qXlc6VSxV0ArWt8xYnbFlS9uTkuPgZHd1TkekjAMrcKSdQurgxJZF29TpLFaSW9zRrrxSZolDeNCDhTonIVJV1QxR+zEQX4yO8YSRKkdRDcztx1hZv0ZW1rBKETGL1Hv5oTkADSxY0+wQXaPRKiqe4s91BsmdFR6b0p4F+Ed1yK4ZSOw6SommS5RDG6EVXN\\\\\\\/SeDtAY\\\\\\\/h\\\\\\\/OcQDo5U3XKg\\\\\\\/BbsVNrBLIQFZY8Qfa1w5OQzWLKUa1eFC3rds3OJUmENpZdHeprR+ndYdECWmaynmhEUj0jt8iX6Ci2G8w5Svg+9umSHka+cLWFW+pKPSNQlOe2CW2KCDkMpkZAf\\\\\\\/3a\\\\\\\/\\\\\\\/+4NG0b+34cIFZ3SZQqvOVgV5zI2TlWfHkCNWNdyWE9C8IwPnSe8M8iUPj7Yr3WnDlJ8eWfr2z2r2+mp3LG4PCzyulgN6aOaBkNYC\\\\\\\/Bxu3acbcIXybVcTomppfSzWyRYZG\\\\\\\/bNAW9aS4DP11zqNy2WczkBIxo7ZcMdotYgpkXyoEyG0anSKHTtNvALh59K2EB5t4GZTo4M8qnKer89VTqMt+vBU2m2aJd2MUyXfzcU1gpebzrsrDPIZyB11C6D4OzX6Fb5yzPckdXjZYvIyh2fgAPDkawE2oZyjA4o4T8jaWRRf4KotA\\\\\\\/EekwhynIEO2TZveTAowy2emHYCVYzAIgv5BcQiDD0Lh3SxKMwVdJT42GJV3SdFHyIB23AkfFJHUHtorZfIMFDWO2YKZBoNFcqHgJl3hheemvOrZQLhYuemwNfgWb6PKcIbTqa9dC72DHbFbaqBTtpkG7KIEedX9VZuLD1+u4q6dwW\\\\\\\/QG9Ho2jRVIKvLSStHXAJM1Tbu6i0blET1sE5mF+IJcgVxgqT4WXzsaDtweA6YRiuTIsXCfNZzGsf1A42AL\\\\\\\/I8kI4RkX+JCwEAQ87zSV6lMuTlZ4\\\\\\\/8lm3q7FBYcf46UkvblbWgQoZjMelBrJIDRdE4r4Opq+OZISjFcUN+cznf5pPAJPZ5XefnXUgr0QPpSjv18yk2XFafs7cl5vAjgv3rUbiNWLKUvAYsmp5x8nckFC\\\\\\\/B7Zr74VgQDa918Xh5TcjeNfVhbacT+IAXkgbHAxzj1kuZOmt9e3Lzl4lGvTILpSYGDnjMqZgvZVVwLlIjAFzOJbvGecAnE5GooZb2QRFiHiiINBPPkl3I2u3vxe0H2PWypdSTlUix6XGAIcVNuZ9C7T41uL4pF8R9tghfrMg2BT9jqIl1u8z0SSAfvwLUBmsbiU\\\\\\\/5ENJv7OcSTzJSxPb0Iu3bqADF57swdTe8iG7mBUIqaGHzzLPOvXqIIrIOwlXdlWL8eBcOJ5rcJZQdQ6\\\\\\\/LxUz2KPt+CV85va49PK8K8v\\\\\\\/u8o4NnKtSWJh5rMIfFur7pCyBgXGqbseiGeWbiyTE+2jx5HCuxaTJPCJ6tto2Sl2pjT4E5pz4OZy5i8C9g6SOqZBIXAI0sNnfNg8DQwZRlxjzQbdmBgkwBGe0TtX3YU2WayatdoBw54AP4YXJxMDvVBLoQ\\\\\\\/af31Nw+72sqBdEuU9BzHmBNKOl9tWqh7MoAvsrkMvqzUAXZCtiBUjwR+hwAuUUl+ZoXMdooPPg2csT9Wzvc3EdszMTQkfKIbbV\\\\\\\/DoHWCg8sJCRMzLpeGhAToDxb2b5OpmST6g4JQERDiVlbremdCehB8mS6kNySObyHGJblMiJ0kw3016vMRqA+omXEf5gxXA26nO4v1EA2+lptBCms9w\\\\\\\/RfBPOA7pnLEdyXYeIscAKU0XZcsbPRLAPFCNYV89uBCvUB5qXYX4kC8M0CBl4hGeNJg4WlWPd2tfUnpyUhSueA9MRk6PmpZ1m7BVFeF26m+gKBYdARrzTffG\\\\\\\/m8D0ZTAu7QyUGbckPQ3Yc\\\\\\\/eRp14lOZdcT7E\\\\\\\/8arF9XMCkt4RVz\\\\\\\/JcTzHo4Jhm1oOWmED6beEw5gPQFE1pd\\\\\\\/ZP\\\\\\\/kJYGQgNPUvEQs0Hud9E9tFSuz09CDDeGUyAMrVh3XMWVptjh1gw+QeiTzOb6RhIr1PV9KcBbhGJBOcg6rBUL3Q2yyCx40u3cmRF1hTZnwCFToTidD0fzbgAks3AFq5EqFoqWd\\\\\\\/58nnuRw4n032meRAMrSo912mWkWiGwSPW+GQyOXVMQAyv6N60J3zLMYlG5Bl9P9B2crYDGeS34SCR26Xw4qicfBZxMdYlcjD0B231QXbRC\\\\\\\/b6Ja1Tz1msuj58qYHmD3soGDE16njjPe1O+uNFIn+j0AQsa28PLhNm5dQvn21gH7uSnjNEu7EeyoHpBzRHjlx1A6Bt8o+8IWF7gzpRPcgG6Nwy0Mo9bC0q+X0ooh0hg6rv9zRZtx8ntQvN696QyXNG\\\\\\\/Xj2vvW\\\\\\\/wo+hKlkCO45eSvzy+mH5Ou9E2ZqnYlFNBWYUlEPTCbw0XSJiCbNhFbK2XeCuEnlvFBEaLdwIVz4BZtlFaqeL+3bViPN6XUiFcdWAAbQPy5v7QPnxds8flRgjZKS44sMnuWS8eBu1QP\\\\\\\/3KwcvRRIoi5O\\\\\\\/LESmhGUnjp6+XWZUVgoTibrb7j5Wyea7yOjjl6MY4wyYlldMv0GZlU0M+fdTqgw9qk4tORba1xjxRNqlMAQvbSopr+3PH0zTi9RKoA2H\\\\\\\/wgckFf+pcy9FjXQCdA5EY2hHXapKQbC\\\\\\\/FcFArrWqDZHgXDalzJjAnT\\\\\\\/di\\\\\\\/V8D\\\\\\\/4BofXL1IsVcbn6dZ2KVPRCAg5EZi7b+gqB7+czYOjDeMSM3vu4A\\\\\\\/U\\\\\\\/9Ry+VmbBrsIs65mPeRzBzMGmBOBCjJFKog93P9rh20KRgPz0HWS\\\\\\\/S1y8Zgs3lRu67VMLPug5FUeBZCurok8BbUMhEDhALs9SRuJ2DJQTpJ7BS1oYXGhiSPFy6V\\\\\\\/\\\\\\\/qScWNEfhBcKBTxbe9zno4dvr4JkVxR9KmvkSfio6gTKKN7c0KdLXSOjBf1bC9wyAVZfRtAUXlybMvKHg0E2QEBEyg\\\\\\\/lj0cwjHp+YXEW5orNIrvDMiipF9nt5qmaXb1P9SfBu6xvlPZNzxModR\\\\\\\/3Jj6YB+rdLcbVRPijtMMeXi0A368dgAdfWx0QbXT+UI7K\\\\\\\/u51LYvKXWffIW\\\\\\\/\\\\\\\/Mj3HYj6\\\\\\\/et\\\\\\\/oQSROK0IbUUA\\\\\\\/ZOzuSFHMDSXv8fhji4\\\\\\\/T0njHNY+CwS00S1KkhGsSGMwWmJlPl37UwuLVi3wu4qmcMV6+OYeytyHlilMjEx9nhfHlL12oPc3DkgTwdCx58Vs0Vdent56h+AITiv2NxpZ1LJTDTRR3YNxvZ9L5Frffi\\\\\\\/6G3Czd1iK22bctv7+HhDmwh7WpqRLjdgu0muTe96h51K+CZD\\\\\\\/BmPI8YqJP3PWRV37PqrjbzJ\\\\\\\/SFSkVVThna9iUEnp34Ptns3VskGGbYu5Eay2rHvyxBnZVry7+TElg9deAcCIHlaPVB7PMascsdHvFG4GxBTSRTgqPvPMW8QlasYMtFGUvUJ85rIZCjudGbDS3S1rEYYgoTd5qKDGujMW7YGSy2ZrrrXP7b2dhO+WxG4S3vlt0Wj1ehK+9SHeuhF3HjKIIthkL\\\\\\\/eTo1bCVqIaBk82HkiXAfdtnYjLVBI2KbUqmf1wnUWhL\\\\\\\/I5UPlouK4eYaK0xOC5QPXpSxyx0j0BmGPOS0cOjLEvp2Ht8UhLti+nDOugRwmWjj3UVqm8dQSfT6KMdT3UnljnhZRyirOGVS66zgefGFkEu887fdhHTvCuJ36+bffJsvRGq\\\\\\\/RvFfCRT7KMAUuJo9EHjUw4Jj6+hcxX6ktNwaPYbkE5SEJoO3PhqInNwrQY6rEGSF7Y1QFFiCDhc1urg2nM\\\\\\\/9\\\\\\\/7SsJVSUPr0XXEzZLcpqeLwVm3w8sSl1JyyKvkJcLMLOAHvFNRdd\\\\\\\/3a9qPBNBpFcubozIjJ3bRA9CeY4vKbOtWheJAVvUWit7Hbu5gaLX8UqoG+MY5dL9IM3vKJqX0vcDQTllagwpvqs8w6NWk1wbXbEYjLg36ptG8XKih36SBzuv4MstO43L0FhGxs8GT+Itm3ll1Bq7830VENg2wZGBWpUD6PVmpsaL6Sjbi0myXnsskPPZETw6oM9ma4Hz3XTCCCTLAId1Yeta1qo4CYUGyiZDX1sfYFNMrnzwm2B3U5KqDEKrglNWFbRripbny\\\\\\\/7c99eUZ+oE19yWI6Mw05rwbuaSwHb2NhJZA0uAP8akom\\\\\\\/IeucRHf9+vwJCQhDE2EQby9S7o0qPpi4k6anWgKqjye6peT5+5GvWjVuap6MuNJDqpBDRMaKBnaYoazRaudyNg2niKhPrkxCCtlkuh9DkK1ja6YZZC0R3CjUw\\\\\\\/VKksIxg03PKLyla7Eq7GJghxwJgZYH5ET2BqrziPq\\\\\\\/KjC\\\\\\\/iomjg74egTICvtetBjk1EA4cRRdQ2DUUKAr9Fsp+uS\\\\\\\/HwbRISWEh8jDMPiSRmHD\\\\\\\/Y76ZdDDRn6wnG7ZZotWUltYdSjySRfjuzMbXBvt92g0id5RzldJUSOcUW5Ycdg\\\\\\\/SVNpy3mWDLN3EBMTYX778qeYGlHw93nLh0GafaGipC2OLglL4VpOeT1ec84RuPEf12e41fp5xcydrulxEK6Qqf6YaDgBbacpZCc12yGbzCjHzWRd77IEcNwv4A036wzzWfeTBFv3dgbstqUkKhPOz3BCEBv3Cia7agRTRhp4cpvCypXYWmVlW9mpa4oGKC6k\\\\\\\/cCxtTueknHuE557gsBRipQWCU16vk+a3RmcZy9NexAlClpTkRbpizT1jBFgQvN5J8oUcwhBdwwRoJFJkOnnUrIK9hEECohxeBbjPfg+mGQf6mrCWQr6uKIRH5p76ooFHX4xshy3iBopmk4JOtEYvHbiz3pc2e0TGrCshal3jOuImN5lGu0FZ6HwB85aKk9gJ+5d\\\\\\\/8k+oMWVXE8k+JqevFYqT30xXXlMWmIQE0UfxFD\\\\\\\/KdHQmNuSMTCsNFAfrICZNhukeU9u1bEzYW\\\\\\\/m1Nbr6nf49o4oiRZNQgrFFjQse+EiyfMXPr1Ctk86K3TjJN7KRz7KN60Lod\\\\\\\/ebsp6ypKElMO0zcy8CdE\\\\\\\/MWj3FfBKUoRl4lh+3X1poeFKxo1BoLyWjGrEUCniz5P\\\\\\\/CEbGwUwaK8378APcAYHwEcLKYM99xhrSEJtY+OG4MHijwkk0BVJjWUIIhVH5Ss9GqgeNcgvrT91jgV5B+kFfdN+axN35MpCwNyB5jSWW47BKn6uOxsPH6VK5GMAVS9JgwN68rrzZUX6JjxIFxZ3U7j2yDhW9WELInX2vBouB5Dw1jia+j0aEqIGzKZCfoyLxxNJy5ev6nLEdJmtP4Mh7yPXOmO4uZplbSfrgv81OLzSQKaz0JminUClu155H0Pjh9iUdm0RAylFxRUns8JmSpfdgJk45VWWg\\\\\\\/cBIrWbEy\\\\\\\/k9QA5mc0pUYZScUAJCCBynKSwH\\\\\\\/5A94hnyF5OHsEKDeMwMEzmO332iygRulK4UasAtoCDtheF6wmBcHNXDfrVY0f+o4yqXGycdSNvs+QGUox8EtWxsWjJ0NGPSy2VaSYBphC46p4UO35z4hmdJcN3n\\\\\\\/oU3So8+\\\\\\\/GxSDZE6wRAgJzj0m626RmTJOIyerunqEz544eUXkD+KunGD8nqHn90wqUpxJyd+cwGP3+xPiFeBR\\\\\\\/a6aVyMICIyH9g6xwTuvBcqxI6bKijmQQvaS4lwdwwtcgGEUfBrIO9iefR0aOawLYMflDuKDrbjGe+SFMPAqS0oMtUoQVHKNU2MaFAgDSZSUDl1YBQy8RkqiOHKA9U71r+WZN4+sZ6O1BkaRAUTUuKUTE\\\\\\\/XU1sjm\\\\\\\/PsGNjiiFZaESUO9urOLhDVaIUPmPlgQbdDVCi1r0QGbvW4MN3cRQOm3IuhDHSvAa9aAmRAkAg2k22jHe53ZCrUX+w+DGUvHp6KCZkHwUbg4XIdXjEz0Dpb+6QoRaIABZtNm3PnWPvgXUTSqYMSUGgGvHHHLKC8htWW+FcFojgkgpTxyTWnuk76vkKt8ium2IGui86vSmPSUhw8NebQ4a\\\\\\\/5aeQif7+iOTRGhfzHsdZLra58gQD081Zes2STu\\\\\\\/gn8qsxkrfheWEYe3EpVIyNZUXdAcZlp92m5W7rs\\\\\\\/3zJkxrVDeMCl4bK3Z783fiY10gJvonZ5Fe56xnEccQ3P4RgZGK0weHIAzRWQ3SbrAC3miZUVfjnN8u5y4TBKSFVm4p4MIGXWE9evAOqR1ulSycVlq52MtxXgTtgaLHQuLhwyrYDlXzqyyLslic\\\\\\\/17kLay5a3i5Vr6kxb8cYfpSj6v44y4ULafjD8qk5G7oBB1gkktCr446Ad+H00tH+2rNcqsnBcicq4t\\\\\\\/a6nX1kp\\\\\\\/UHOscRZuVzfxvuFqQ7PIjTqwUJrbTl0QaAN1hMXszqsgLdmx0o+XRQ4+D6VBKemTJw+qohLrGZiIszRkIi84cpuZI0wGi\\\\\\\/v9r8LNU4uc2mObHd5XAG2WhI7fzFVQvyiZh\\\\\\\/MZOW7nQ0vo+eU4IdnscKqLiQJZPl8MkRk4tIQu90xijIULDF0mxk5L0GRZBArgGmWifzTITO8dInMVm+NeH6rbDZrnUnD+zP0I9Qr61kL4KdtZwVbBvM24fu6TF9zFM\\\\\\\/PKALmEadwrvswuZXJWz8I4o01AIfTUt2OU2doOraBA\\\\\\\/m6NidGGPA36IQ0L\\\\\\\/cmRBeuktX4ZXcx7pHnejPL3eLf0QsjqpJAflC2R7+Abgv5mNvsFK0xbVqXxL50yaQhng4pYUt02KjrBcxAOVsrDVGDUiqoBVT8LO41SS7pI0SKju4GpLjppSdLcUo7uWCmeLr2\\\\\\\/SEPeKMtJKp0e5fuYblxYcNwCVuu4bExlpvy5NETLntMAYxEyYw9eV0GoU\\\\\\\/J6goZwGcmWAL2MLlY9ALQktbWZuKl6zLBqMS6WxyfgK8WFWqXnkhKVYmzp+ejjP87gjX2byU6lQ239NDXM14J6VGmkjnbrtllATvuqY1MOq5LXWjnn9S9uiSJ5\\\\\\\/x8Sq5jm7zd0akk5z4hruHfuR+gr4qc6rnQcKbVqT47PRcMUDVX\\\\\\\/6Iu9mKrLLTB6hNkl2NJ9uYLOjA5J\\\\\\\/2ja1xMnQyaG13vEblr+Y3v8kCStS\\\\\\\/x5A1wAXbLyydg9ADtvbFjVtiaJep8znXjQM8jWnAIpjbl1d1O3ReqqOaMp9+Enm96Vt\\\\\\\/E9QIO8w4Eg1oxenbQHl9LQVC\\\\\\\/7Rac4tnoRVU9Ae1Xq3wDyQCJ0a6+L8pqMr39eqwU+FNBDaS5ZcF3jEjx+2fSVmj3hwk+5AMazMm4YWLpFqhBIxschAcNfVnWeobBIB5oFl2Rij2u4NXzBFZgci02tItwTavIeFGQMolckf7We0ZfQdao+wwYj15bX3e4H+P\\\\\\\/eEXWAUtfbq4rvK\\\\\\\/IvLn7yAjw\\\\\\\/wzeOs6Bq\\\\\\\/KB25kd4coGF7KtP8lNmir5xFArq3hG3ukGpRs63IYwxx2eUX+J5lgL0C5vLhjWfFRebpFzEKZnKKnW+K0q0nVmNc34PxeqNowGszS82XZniYQBAJj+PqiD9oh1oYyr46jQeP09q\\\\\\\/IOWfqPNTnW5yIpZskbVXYygH4ziVEhbSMc\\\\\\\/4ysVgoKHCobOO7mQEZbTZ9JN90srVHrw90W3bBEabUotNqsniQQxFVQi+labYPlb9v3mupmIP5To6q0gzhzJmlOOAs0Tug1SnSMgRkzXd5YFv3DTdmTODBy1ryY1qYvzDMAXDhXaNtVTa87db4YOh9jHU4VZM4GbQm4h9N1eFZ0FIktj\\\\\\\/SZ3cP0Q0nyh0YumMxbK1oJs\\\\\\\/sjgLE9gAc2yVwvOI1kIUyptPWwtmLMXQlNjdC+lzJ1phhA7B\\\\\\\/JuLu3\\\\\\\/X5I7zw9BVTa0rDw8UKybQFGhQaR23tlBG9nGT0kfsow4+g6F8RCaEMtaua0pVkomtnZFtkEqyvpn8OUvt3NEWYaOw5Zwn2w3cyzAqClb42oEebuk6\\\\\\\/zAnPSNQTwyBcnqnkUQDyqQLDWqJlJqQQbccJTqUj6Q39EqKDe1lmCv6WwvOkcRcorlk6CLtTZZXignioYNYa2xbHA93jBdpsRNV87k9OBIYFwD3FfiVSYwJKEI82F\\\\\\\/Y7sgdf7A+cL7dWPKRWVN7HvwcBvZvSPmo5fUnxLYcdfEMqE0t1eQBNEBEq4VVEyAv+MpFUE5xJz\\\\\\\/KW0i\\\\\\\/ZVvB01KeivoV2YjxNrBSkN0BWABw4hI\\\\\\\/szF1rijRhcWsVdMUH3FDCoVrELXj+CtAcwePSQABfNtLWevDhtJcFcLxk2\\\\\\\/\\\\\\\/9h3DZhWSnwrbbWPJm3i6AIammP\\\\\\\/bCrJB\\\\\\\/tDeYAM45PYmWv45OAIdpCnLOnnvFNeZXIptF5ksGVopBVtpkLcq3vyb7cxrHSsmIUEh3FWEqiezO1f2on0ukMmchs9KiiLkQc1K7Klu8tanK81WfELtAKBPkAgJuyaQ\\\\\\\/t8F9Va+cY5ybdhCx95Sv9RdGK0h343EaZpEp82RFVF2VWUu8IVGlUDDZpSXImQbekBy90t5+5\\\\\\\/Fwgb9AqpqsjEy9x+g4AF1ID6q7FtEPRVY0DafeFJziL3bwWyDIxo9r5wjqjUraAv170eqIaKGo7qkxBeTk7T3S17HPqXqW6dshcgHIdgyCoHfPaTTkUkAdBGSsYExVY2t5EUKkRatZcpBkoczFr34YIr3Oi3r1Ka2vcjJBodc5tZqfii2FlIPnAZVRTsNU+CHDCzHMDomfv+kVNkr3tv+lWmP8t74vs6kVMl2+8m6nRUgVJbqv9\\\\\\\/vN0JQ0xCpmbxQ6CMbJzxdP\\\\\\\/6a\\\\\\\/69RDA8PC2KLejRcvWCViOWGrUNz4xsJ7Jncd4i+sTwA\\\\\\\/r9o7vJ3FAg1WG1j1JIsfzHp99TKzvtjPB5Lw4VoJNDP\\\\\\\/8IMbbuNSnk0CNrG49eOGhECrBoiHg7b\\\\\\\/t12HwF3h0jXdkc+2yBzsRfr203p6bfjCxfuKFZ9Q3NM9Usvbs4cajMsCmux6dW+\\\\\\\/t295r1Jlb9\\\\\\\/As8cLbJw88XVt7c\\\\\\\/w\\\\\\\/spJrX\\\\\\\/dTkn38QlyTMyCOm8i++72JqWKgYMc3Xzzo19T+cBZKt0SJbfYZcItl\\\\\\\/snQxBnMEtFXSKctwyt9Q9nV2SqKZR50sFPtvy7EORi0FTyvqTpDliAces1y46CuFtf3ZVMCZFZaA6a7+OpfRrep7e2EysgDko4XVTrPMyrG1u8emeFjGBGtOPrra9AIk8N13VZrclU2UmgNpp41wfoyv1y1\\\\\\\/Fc8OgnJNGztw8Q8mazV+284FWPbb3zHgtAR7XyICX+ZgcOA5nSyBRLGz3qSa38mKZdm15morvHryDtyFpigP9F\\\\\\\/N373fqyZOa00wNAbhOKSGfe\\\\\\\/AbizqO54RlxXJozIB3eiT+Z5u+D3drAZ8Lpa+etBHU8fM\\\\\\\/tV03nTrQFitRCrK7JVTkxR7Hl0hl2q2Uf1MwGRlMC8QKjXx6OYwAjjV6nPws\\\\\\\/PyRPYZJEz3JZ\\\\\\\/a9OAJiJJw25x67AYMiJfDF9SJwROrMr7LvzFEvJBMZiiZoafRnqnFcRKf6EgKyn36BUzpXDFHlpWGQlt\\\\\\\/\\\\\\\/I6i0ftjS6e9qcGVOuDsp2NDcrAiGJ1lZdM26o5pkttpqVFXV5\\\\\\\/GxCln4bhdr1ZiRVdihtkG1J5QBAj5h0R+yvoPHBDPqUEpLINPvnqZkbAvcyleTrPYncaB9oZf+ALLQ9fi5gVyZqEKe3mHV22IltfbFt5dzmAcM\\\\\\\/cqcwbtLvkvjIhXmuQNsz80sWc8cFDKEHVeC\\\\\\\/TdImA\\\\\\\/JORCQxGBtmfLU7C8dBWG19TyffFasSwZtkPWUPMUX+BwCn6oxFIOlGgdXfYvQrl\\\\\\\/Lp8m9hXZIcvqbaV1ostdpdX\\\\\\\/Dm6j0OtCdwRrihWh2w+YWtgZ7GDpjdGDoSM7dP2efDcgdAPrqiHPMSZjW1UemK4Aiubtsyv7plP6u7fTizVGiy99pU0Olo+fGZ8660dh\\\\\\\/WakYrq8WXCwaGt4Kjdgnzyonu8aK+3u77lMIZ7v7TRnqr8IZeFam12JqS7e2Z93WujNjqdQcB0tkdQtNZxck48pGchhI5z7R35+aaQFJ\\\\\\\/EMXfSxiFK6CXya\\\\\\\/cWY4WvJDbQqv+aoY4HftxlUS5vMrPf0FNf2Tf5ZkKi77wYjgLAEfxwZveT493vBd9catJ6eaBblFgK8n4XGCZiYGmzJV3wufZr4JSzxKEun7jf5U1gBMdp0GbcdYA62A08bJN8BEojfKfpXIQ2tasgE60dfBdV6nRzU9bQ3VW5adgx\\\\\\\/FUuIBRt401LKdBZVjOPlouM6RdbXYvNOVFT\\\\\\\/cDAZEADGhyGyMR6RP749V4WIE0GlQxc0+WugnjevzQ0YwDYZPCOH55JR7F6jWCuBZBV7x64NRAj7rZjE+wBYfey2YceXrGUarZg5s3O+OvmfEtXGjIJiF\\\\\\\/\\\\\\\/WaszjRF6LJsjiLFdX39tyOzFMa+QWq+20cofmu21mlxbvpEG8qz3hx7e7OQDf5Gprz0HPBSS5I7Z3MU04snM0Gi\\\\\\\/lDnPh2\\\\\\\/H17uqEJjx+V+UgtxC5kzc4RaQhqg5hddeBpWtDOFe80c4EZI+TveveO7vKko0U20RWCjV6y4z8O7ZsLbcSJaMWBaWtZVIe+oh0BcgLjMhUMQDylK\\\\\\\/rkI4eG8MOPtOug1LeW0QOuUD2PUOsp3PUavecag3vcptlam+kejVUvw5M4jbUzHACWzZY8ZS05DUUmcXG1eVwpsHLIkk3tmr9kn0Ea3KJnqjJeUIgpMrThk7u7mvsYaTJTm8\\\\\\\/Art0gOh6cYwIovndQR8IDPDV64DSaWNetKCMpj4pY8F3g7yOb1zmSt97QiFZA\\\\\\\/InAfG6nO5nnJM8oAbd008lo2Jtmqpn+v60ogANE4N8il9Pmp5lvcrGUCPC8t5z3pUuM9dHQTnLcHTyxF55k\\\\\\\/iJ5gCpdSWorsT6Zihk\\\\\\\/KoMSkypIGZk3h9h7VWDZfSKtAy8SQuUYoL4uYkNkyaKDNDqlZHRpSe9dDgpi7CKmxZrdiEuT8vgMBEEnQZZod0ZvI+gy8DezpdD4jFOVcHaznl2gd9mHNMDoF9SShd3IpeTELPYfus8HqSOiUSQVV012PSbkPNr\\\\\\\/cg49gTem0PWNxXnn6AF1Rsup38zxCOnURusvXujk6egD+62NHN0e+2FY7RXGj6TjyGuGlrWEaoCzXpeZDVwRovFHENu8ZI9jkIXN9liMCc2CZGnVGwXCRAl2OFI9Ge273xY3jmuWHPQLs\\\\\\\/SQ4fQLx8x70Ja9MqZHlK4q82CZyy\\\\\\\/TWshIwd3zahr7u4OG0934zz0ScXY7Vih35S\\\\\\\/xHHxK1w8ucmKBS21oZgW0tvH1GJGAn4rlm4HTgTNYk6jaCRRS0qKL0tx9JUvRowUggk0QOTL\\\\\\\/UPASwjy939LL7LbYusVrpK2ENXhRC5qzftcXgo5b3+TpdvxQR6TYcxzacMyb9GL9l9r4NJuVGnu7N+TSZ3aWMu2ND4qz+QMJTsq94RwEHNptQKCErq01HMdotnfZ7Sax7Rkft2JTA7eLZf3Ol4L5sDPsigngChIrz\\\\\\\/BEqFlbNzM88Aodqd3V4pJ7BZN6kdKvyYrn9IfocWetfg8z7sSf3NHB3\\\\\\\/Dotn+XwYesAlnoIKVCqNygyhYc7I2tirDa4Ns+3N6FlC49effLlDjhu9EAH1ZP1gVBWbcMJbtFWF1cChlfuUEgAoRPzhG8FZIszNsQD82azJ5Hn5Bfb81ia24xyJCFe8s84LrM0+cnprVvXWUjjy0AikOC88hm1lOSnPujWLkfzPDKFO2\\\\\\\/ax\\\\\\\/aOUClUMQZIUNj4Iov8PELD+e42+VBB9qMJwvwxZB4gG3V5WENboRPSdfonCb8cGGGCXVMwzKDbd5DBalQ6D9JlNEGsThHC1tZHgo3xhU2Rv5attsTGtuQ4uEQS+qNWiFhLZ+V+H+2Rj8ag\\\\\\\/X31mz62FLO09XaXlTtIqXF0t2ydy19UziCq\\\\\\\/Wn7cMBJxvtucT0+V2TnjF3KUbVOfcy0UusAANX8NpUA94e1ASPW8\\\\\\\/tOwbkLojTBmTFK0FpZrdUAEoixIUapsp7Lj9fVXk\\\\\\\/cG2sZjJlEEFw9elIKtsaCdtNZVd\\\\\\\/fiFb8oTB57ORKi++cw6zq1Ua0+gUnLxizEVfuHXEWNamDJ7mQLeUgmr0nn2TcwbUULZVyQlBMiYDmeFcqusCuG\\\\\\\/QYUpGXzNOOIoaG1tpgA8pJNa76g8FT7au0DHDm0F\\\\\\\/4UAOMudzyA39ifI65QcNraquFuuOmVa9zpKAa7PtPyanT0tr6LeU0ogy9C1UtOc9a6d2sCn8p+QCjpk9UJLEDqhfw2b2cWOhsys8BWuQExAHIJoyjbjJ87WuvWNmGT9wa5f1WWeLNhGpSkkVAz1a2OGuEtv8bv+KHAnIV3Wslj7Gr1jdqp7OGyyGnZiu97VEcoMockW0xmAd6TrBpmgaGWOIfnOlKH9ze9ucHf2X+TOipkivuF+C9t4TIcjrNqKvx7NxqHWwUNId2Hdm08D2HtgyV3uheeKTy0hU+Kzj1lsr+Ia4eKxMXZH0iAHEgMG8VjuxliXbgBIbnVwmxtpjfWx0xcudV64NDp7hTFdi+ykpuak617b3490rMdTVB0RSd4adEHikkjlP2vOvJKo\\\\\\\/l8ZIW8yRQ21wO1bwovw\\\\\\\/ycqnrpSTHEPF09ONdPEXRHLjb8gXMNU3MLMmCL5XIzf0N6Yb7HzZalrdnsnCo82xFA+vEQ7\\\\\\\/onrPDN0OawV5UDbJMrtl\\\\\\\/7gMAq8rc2CWOjFYJEpxPPdSoHMqiuK7cXdcGPA+O9aN1QqKOcgyTxh8bwjLmeYSODLW+3L\\\\\\\/7XNtfDvvATTebux2x7d3OAFUrS3mIolrn0Uxe9d96sRyRAxTWcvNeEjQ8UW7OYZGwvxIY1ou8aPRqjtClk+LLyXXZ5\\\\\\\/dFxg+oyeKBI5vRjBQvKNrNv7ysyr5jk7RMDWeEgV3pGa50Obqr8MmnOx506KLpsWVEG73ueIFaau\\\\\\\/efGgO0FcjPwJN8fdoizgQUe\\\\\\\/xQm5YNDtSvvFFbjhTZvE4N+FruZPJod44aOtL4duoFsOmmUAoTA1a0F2UgFwWSdxvPohHdJ+ENrlQxrIGXyLEUHgDD0P1RV2iSHu7w+59msgrXsQ3WEerp9CgE3GZi8+EOZ0rSIIKCRGSrdSkN2jW6ilH0HBUR9jNRQaK5ZnkVzH6IuQ1VCKyiEZ\\\\\\\/xE2dG\\\\\\\/9nZMXz+\\\\\\\/WwQRV7wKuEhoFqgq3tertt7XlURh+rm9MUQ+pfFoAzbslp6bwAaTfVdloMBddkvZ+e77aVqq5J9vI+2QwpWNGZiREAoc+R2EC+xl9G7PoWfP0tmK4JZeylM7iJjysIPEzmXED9sfzyw30uM2ho+wbgHBSFK+CTzC91X8F6rXOWP0pfRghDDYM3DGYhAC3P7areoWnIGYGYTbykHe\\\\\\\/biQa2gAB2BfeLSanLIBKYnAaaxxyNXocvMF2k3iA+XpZNB5cvxCCgD66uXOX8pNaf8wVgY45GW2urjLwq4+vUW1avydaIHpkWHXCYYmTj12s8nUxz3EBcwjR2wLNqkNOoihU8WsbJiLC\\\\\\\/N5XKsOiv1kr5tDa6oi+sg1HQtalHfyXPcbTNVmeGgTTKJCnjHoOJc62DXflDjYvKiLeK\\\\\\\/U0LuyX1aaVlbm6lpVhgWYhbEDHR\\\\\\\/eLESn5Uw4GGtbIXiHjZwTLSVbUAF5fJOeiayFQN3Tw0MIgd8QKXeJg+RdUk6MO4Jh0HnU8awnUkpZUHA2pyvBMsEWwrBeuU5yA2oL2ScS0B9LGF5F3x8NIDGQmouNZRmoJKNXPb700u8Geg+NkOXAMUD0v7AD6UyURD+U04w7MYKaQeSR3PmHOOYRgqDRCc2C1o\\\\\\\/N+sJ3aT4q9U6Xos6S7Jj0W7+F5C9FRWEddwpDrip3fgFj+MmQ3gYChBrfLYni+WtmTdFjFvO\\\\\\\/BfjULPpZ0Iz67XJw4QstEBG7TLT99LwqCyY20mEKAOH365\\\\\\\/pvZk\\\\\\\/n7sxPFVjq+\\\\\\\/cp99HCnHazRqrLhD6Ec+LVe7JJLIDyqb19cdR9KoD7fV81WALsLKde4C+SelL3I0bEyX\\\\\\\/zv8IIOhPp+phYjN7pvPrWLMkLSaCrDHf2jO1co3Q1R3gK8gKdfiUVKOIPCpaJZ\\\\\\\/Qqeg\\\\\\\/p+KWOnuS9C5hF9JIA0gmoNPBnNMTOiRbY7rWfZCgTzGCISU8XvKhaoOkfW8mXMEBfsHw3rLiNM7H4RA\\\\\\\/Mbl9QfVNGscitXEJsPYChUqtIPQFVICiJXPTefXcGtqVyjDXcLttYAoKOLgsvlOjAsxKWKb4pjGP8D5fa4BKxExOoT+CADw\\\\\\\/9ElT2yPmJ8SDRASJhwZGMHZI70aUHFv8VSfG0Sih9vHYgMsj2WA6WE3HSbne2G\\\\\\\/YzsUb9NI6GEPX+2EI+CdtyO+RsBznu2LbbxC9vktD8tpvB5PwTJqxzYfnhWzxJgm8xAZr8XOvohHlGHFOryCuI9jCN\\\\\\\/QBSuUgR\\\\\\\/9LsDxBV7XBUx9BNscSIWqxVXOn1G09Y80yzSQyPpqQBQwNUjzpQ58Gd+SkVCMrjwpU0zNtnd7jDE4hxW59gMOX6zv8evBhGM7HrqHYej4+aWXqFFTFxZXocW78qZSmlbIGWthcaggYY0mpgQQVagiS1RTVVHYQBRteiYo7avi1y+5R0x\\\\\\\/uG8v8j5iEGu91MA5I814s5OUcRfIbhjW\\\\\\\/GutLj2uHIsR2POPxZOAiIm0ZO1wyaXNt0uHGKDCmrqi74jDBa3bBAV\\\\\\\/FXH37ecP1aVpZwm6dT3WDvNsLfd99rSYWdiLX+6q1Z6vdWIlyM\\\\\\\/ADoCNAz\\\\\\\/kQVB8gQO7yxotYUfZJ7CMpJP308HYUO\\\\\\\/tglUundrirJdWPTr7lJquMU+yyoFSX6P+8BXr419apk7pSVXC1AUPur+AildVsTub5o1jyWLtjP22adB6jsn\\\\\\\/gRctLLDMp1djuqlM5QFQV7EpXzsjGvU1iZ01E+Yp38F1LEOClvAGv5mExkWSmCC54WRZ5WZLRRX1IwzRN+kn59uMbrixs9pflC3yTn8VyiOTq\\\\\\\/EF8Oohp6sCnrFKpBPGg3\\\\\\\/I+SG9eMfEQRMtmP768QGb3PJgy28WZan0SlqFjqIZZ6NbMk8z3bZWsNPs557\\\\\\\/I3ZTvy+4ovP0r+2neaNnDcQ4YdMJh9J80VzMCOhxr1RINO92k0gSvlg7xHNGpNIjU51laFTUouv0aucle7+i8OQWWZaIiiNrCDlv+9QrB0IFvQkuD41laZJqIk46mDJXam9YD6IBHl83Zp5SjBMKsq14n1eXhXfxJUivUeYgQVRbCOVOZ\\\\\\\/JPHxQJRnyROjJ+yVxvNf6+m5+wLvYQlrQ6aqjWQsUaVT7uFduszzXIZlOI9H+ZxLOjjZLaMoqKzvuIT9K3iPzPKwbZHhFhglDn6B+yrl6tyAn8INGn9Kv4Y31ATTORy+\\\\\\\/Xu+GKEZYefY0qYdCxVIeJYdsIt+17Sttp\\\\\\\/yNoRC6Vpkoi6D2ZKLodzxaziWRenjxA4QrJyyzcNYAoZ8kp71YzHvKPurAF9hjDn8Mrhwnv5FR4BTJSbnCgD1l9zUp6D3hDp85298C8fRBJPH0fYArk8Xn4R1OzXwvhE1xkDGl1RkDjzLP1AYWS0pxBNmU\\\\\\\/umbFeYMtWZJMqOYkXXvquZer46\\\\\\\/olimdQGT\\\\\\\/pJkpNyFIR\\\\\\\/+rQGYi7TCd5+\\\\\\\/H\\\\\\\/5uPmNHDJH\\\\\\\/IeAr8oYcIUXaYIHIRQbmlqG873f7NL\\\\\\\/Eg+kWAKE1ubB2d4GiSrPBBu0c9dI3nEm5+04dy4hOvZbGknKpO\\\\\\\/p9aIZTo5W2CzfR2SSG47jeWBKA8HnZsE+uqgZ6NqyybfjzOiTPYFyd8LEozlEVLaUALG92cJtLvpAjQcckFHsjgY7WWt4reMqq\\\\\\\/iuFmFplTGNyVPNyjj+duZ68+j4n8gJE4o4AfQcHf1Xi1YZhVfhH\\\\\\\/7bPQnmHqcLmI4\\\\\\\/KD2vBVSEmMv6b\\\\\\\/ByWwB36zanBcr1TfKZ1qIaltesdwyPKK\\\\\\\/3zvj3cqhlPHKyJkG3XYPwypWY8JXmqIT7E2hKIfNX7i1zZ4exXu1chL9pTHdJuZyRrQC1E0JdanGSgO1kEQoPT2yRzrzO9nvprzN6C9HzjkzK6XlQUVbnUs8SC29EuBriLQy2O9ML3jee0tkyZ7tvKH2eR5z70plsphRCF2awG45yhqlIG9ZC7CBzD+Ep52sZEWOxMwddH4y58B9f9af2bu8Qs6j7lHSK9grLKftFERahTHLPPLUDGLOVu\\\\\\\/TTSVF8+zLd3ZvOM1U7SKxP1jGRDAuaLL5QEQEPByS8W525HKSe\\\\\\\/bnmZiw+\\\\\\\/R7Fdk3PDcXZBeIpseQUWnHiGMjKs9mQ1xSAhfNOkvpEfvyvfJZ2fFOO55NCgWgy75Fq8SHzN3U3Mg8+B\\\\\\\/1+CEIjTzpjFfjcHDtAKI1WEovosp2urh0R2gxvETn\\\\\\\/mWjL85JrnFhO6vhd4lGwrFsM7nFm0Fw9f9mHo0ZnUZ\\\\\\\/RePN3rT8\\\\\\\/No76ptbIGKt71EaZoVyF4OI8YJSw\\\\\\\/lR96yDzffptG2wJbxbjK\\\\\\\/2U86RTS1lnf\\\\\\\/RdZVmYEzK82VjpWz7YtVJwGjbOuhMYspsI7YBM9vN4sMjG6vjEGbKU2qrn7qRbDIptlY5UeoXLaANourOqHteNecq272vKxU3FgGQH5h\\\\\\\/kf3haiHSPQpAIBBsdTSVSLgMNeW3Q0AtlpevO2uU2\\\\\\\/Wep4CPGP6ZPEK2VFp\\\\\\\/ceGDKaWSnHw5wFfAsafLIQ3N0XWul7VUkk89LDLI9TcjrSJgIY9MUlY3y0U614FzZN261j7tlEBJFcFFtDlYLVN\\\\\\\/vCFEtEtFbP903L9Z\\\\\\\/zI4kJAXHaZLCDGLfNREyMs8jHUWEiEScWkdofrFu\\\\\\\/rtLxqb5gUdQDNFj9PdMXJGc0cUhG3kFM6XwZewOsYxqIXCg+3NufoYVoIVflCOADtCdcwRw8\\\\\\\/M470qmCjQU+Y3OVzySBDGg1ylZh8DR8PFYih9xTCz3vib3qZRMeDoXFWIvVGcR47sUG4hVjG\\\\\\\/bXmFdMtoYg4LoeyWfSYO66plvwCiyBrkfCJ1viMWVRMp\\\\\\\/wFXESqI0OQAyAkD2MQa5r8aeYe7pAfKe4gg4mGS7Es5PLaLXtg7Jo4V2IPJIWOqjGLexmXMLF9VLCWRCJKcrcnOTz6dO5AdSSZzaSI+sWpelI1SjFJPgDbAXvFcP8oMjLPlfoht+nceMWIwXhwvO72u2kWgwbC0dY2COhmj1\\\\\\\/aDRB2AN65aqWH9X1CtTX5D\\\\\\\/L01bxdFbpKytObD1v\\\\\\\/0n3BCpVW0lGHMx9RE6g\\\\\\\/uZ2cTNQ8v2gNECOVvlucZkoS82AIjkZlhBgOP1QSojH31Jsv5pfYaVOyqjtc1ZCSdVVEb1mfg1Ex9jzC13TViDKjyRFuNgKJAgCDSss2rDdwi01nAOI4dr5yQMOLEdkvN9b5WxAXchUXoRiERRJxAuhbo0LOEQi7IDv8m3Ix7LkQvs+Dqsu3RdTYrXo5MStMYeCV14pB74tiw5c\\\\\\\/itr1OmCwN0pBD2sujG5OLhkvf3FgMXYOQD4GhXmFfe97hkghplg15DJTwV\\\\\\\/+WBzw4pKJ1p1GNOQ7Cx5BN+b2XD89kUcfB\\\\\\\/XC\\\\\\\/02p3cCw2Y4j7PUohMsShqgVzQsFByW+lD7HsSMdI37\\\\\\\/T6Ow5g\\\\\\\/NxEWJAmB7lP8UaoLLbIob3xWBPkFfGD24Di6Eo+MnFa2r9cqRwvD\\\\\\\/8l2cQERDv5yH+HGzQkWWoLDtENn0r6TVnPNf2QhAKO83iy3Ae\\\\\\\/RJbRDsBaoE1Wyj0DJ3N\\\\\\\/3CgS4xTLvsShZV0gdkI5bdrjkdXMV2ygoEhC6LUXtAFhzOVZmbbTHTTAhHe+oM1UjSUI3cXOs1PXvKRfIiVzXs\\\\\\\/ZNC3bewOePa4i1K+\\\\\\\/dEE0fYQD9Wz7aS7MJMm0uJn6SaxxA8ts0GytT8eVdPt31xrkPNXCPnj\\\\\\\/dJxGg7oZ2ATsBZrcHTn49QguBnc3Y0Xd0qkBg6Eowl98sveSGUMz9YXFEzvUaPQw4M9EU\\\\\\\/+GusU3203dLk11N9maHvg7vOF8hZBmN3JygomGnCH7jPQNuJ7ICMFNh1BAs8iGYya4tZfMYiclj3SjGRmeQ6ugZGja1kYAu44BLxKTw4GvMbHxgWnjg6eY2eY1xlDsd6b2WfyMAeHcPAYrg3YplWryxRZz+bTdV2tfMeRFVf4O4mnY4ZnoIabG4UzsBUVmQTroCo1wnJN9OZo1tavSvTdv3UHWVqXFnuXMl3vbqiiVVKzRZyT4tUPKrQfN7CGbq7l7sc4ra8Tl9+WV1iTkpz+LgWtmw3wM+JoXd2aWMmCerY4HYrB2S5bE2CGI8Xo0Gcs8g9Kms2opdUvQseCqhO\\\\\\\/oL3od5qjeYvGgWtdn7ezQUCnx2FKzEVtCKQyGVkULJTBThU3xAXy5ul8LL\\\\\\\/v5mCLYWChj1YDhISlf+rIdEfQ8kQA86z7BdYjPoqDyAA9srFYrI9xinTU6H7fltuDgrrBTi\\\\\\\/0FEi0EQj3swTexeAXV3Gd3iWi7VZrNf\\\\\\\/tPTcLBBs4Ih0EjXdaLVRnvec6HZVC58Rqgs0c3D6Zb+bmXORalz4PaAqzhefVP6cKj4\\\\\\\/0x3nB+VCoDbU8UQbr2M82z1bEKvbnz6rGjesJrEx2rCti4ePdA+jEykpBe+xChLgHgzhb8ypZkSrKcGFLx8ijQ8Xf3VhSgNRXrnxha0\\\\\\\/ashCPoamQNeCZDL6jJxZJACJZUAB7xWZK0hs49zMGsyKt2gLMjoVy+vetTq9E0ePtUP5SmYBIrFB8vPHa58xVnk7baUKBOo6nYiQd2\\\\\\\/W2Fvw6CMyKozAkpWtMm7uGMzR6KXGP2boP0OdeVj8f9CySXv1jMlzGUvqMDNTez0C9oah4pqbU+B1\\\\\\\/p\\\\\\\/M8+R+pG8MhZd4Z+EKhIlRO00s4kJKuzoSmL3w+FXB+TAi8KNpuY0l\\\\\\\/NS0UwGnwg4fwKeFG9XXRepPVIUfJ\\\\\\\/dhPAzgiwdAwiy0w1C37q3oBVV8UfGGwRFMfuoHqQIHhg5Cx0A0XHnovJkFCTopD6kg6jqLrK\\\\\\\/hbvzu7kDW6aUSK6zL+ifqbFxgB7OJqLb9J5LZqTXvonGMce2HKPQ7Iczyjw8s2hSe8J8rSvOdye4dDYiibNnEiAURcex62DSsuwHBgVPUi\\\\\\\/S7W+ZqlibONkPu7vlngF2J3HaSWtlrUPerVe4O8V9DwYDZeMwR7IPoQMGaJ2h7ft2I+uQwfQXlxaerFNX9BPlC885XD1+UiQx+EPJ9BRmBmm+hFCLpX1cuRNWMAMnNUoXp062+d7hWeGlrkeWQ1fo71I9iYUGJRJTZEZ26qkWKnhdosek323BD48noQdp+5SBJhA94aeimApclJgu+kTQFRtmjrQkQdYHlzfTEFtbvZD14tmmvPP+iUDwd70iMRw+BDggd6PU9Qvr1VZYIn47x5REvgOSaNgR7qfONIvDOYCt+4pyWhQLHomgoWn7Lbz2nSZhs5pEFIqC0++NTnRpXPj8AeGBjnqTzMnLOnXbSsdsFy1cvbRAYkz0tHGSxAYHDkIqeuMKqWIfVu0gs6AUEgvDnOUgeelxp+AO\\\\\\\/qreAfbG\\\\\\\/U4GVSPKhVfk+4J3P2umENClHpBAzgrnzygoFz697dqVuwQkKqVtaK2cdGuXqbb4Lme9yd51eiYRcDWrd2u1nMS1veQwf7lmHJzB8e71o+gH0qkO304H\\\\\\\/u71soBC9KuRyADe4U785rXamBOheU\\\\\\\/TScEJTNwkrO3uB7FyYqRRscUcN8U5v1xZZTubpBzhsTYBMzPnGqYSID3Uzwy7MwjeK7JF+5f7rFS2uDltwwktogrLbbbHwANznMSiAmJW84Li5mtjcJ\\\\\\\/wVmklOQHRkfnMY15q15h9RqV5BAmpVCYDCWBtZlJBiErtaHnsflAj\\\\\\\/bWEWK8Uo9t1RA5\\\\\\\/eB9hxWGJMAkiSoe6CGEf+hobWQWiMO\\\\\\\/Hftzp\\\\\\\/baip\\\\\\\/rG\\\\\\\/AtwUUU0h6PTb9mk75nGfD3aOdZvHlLj3Een34BSdHsOGHzevLNaFfbgLNsE3fU+PAx0l3oakaBrN3saFEY4KHN8ExjtSSyeWUwcE29frF\\\\\\\/8YdDdd1vvQ6+vSzd5NacVlmdJQ4oz2htBqqrj9FX26\\\\\\\/yddoTDwmC5l6gHprCK\\\\\\\/VJ1rLcpNzzb3ms6u7gbf\\\\\\\/jt4u2p642CiaW1QC0FmWqj+G5Ik3wOxcKp11Yao9xJe\\\\\\\/JkjJoY5J3YWTMl1G5JZChIdbLKgcyGRDlbYr\\\\\\\/xgoxS7aBbGqttmjY8IZmLoVj5OAkOLVk\\\\\\\/lOQDEt8EdJmSxa91uU\\\\\\\/JkjAkPnPok1APBNl54f7S\\\\\\\/eWdvGsCgUVzcjH643+NK57NLZ1Ve8XtLfFMAwwUhF1cVS\\\\\\\/ancibXAE27dQmO8\\\\\\\/o3nd8CzqZhJ0S2qbziOuwaNFqYHEMqKtPjWeB568zvQAx5BAh9AZ0WnAkyxm1169btGAQ2LqToBH\\\\\\\/bh3Ry1oxX\\\\\\\/kNYfRds6RpGySdNrLtU5gMS\\\\\\\/obzpnL9Xh574YLQ4h96sL6VLvsCJ5ojzZ87VzZfg4rzMYdSLXMePo4HLyajmIY+9IjpXVVZUB4KeavpNSnrkVJKV2Yu8NYm1rVjtFExX3ZZLRFUd+cUJPUmDiT+th9l2gvg\\\\\\\/dWESFEFT7hu6e+7wsFFOclcITZ4e8Cm0cqwh9AixrWvx8a9B\\\\\\\/S4NPfzQwKXVkN4Uu2jYipE9ELUG379UhEZGXaTSOOSDsyUWNffe3VCysN47tMkIqdDl1EPUjt6plHgJALhjy12cUs0CSpjrGjNn0iDK\\\\\\\/L\\\\\\\/tIDhDqIwiVGnvtGNd4X19WNZ0Tvcwv9TCUOTgWm1O1NdcySbzBmIZwk8N\\\\\\\/61shedovgDZSXV5XkgVqmDq+WcnNl1j3hAlNXqavuiFk+eVvF3iIL+\\\\\\\/fcb7X61gWl6SVgPBwNTXATrbjKIo+rEAAXZeAkn2DO2x9+IfDeA3gFEFpQRXcumQS70WbpfNJguZGQk3p5hE+rxAOGk3xbxOrU9C9kpMY2W6i4QRIp4fFFDzBnkoIyo2bnwyn0Xs3vea5YAg6yYsc2tHqhKX\\\\\\\/awuiwZRk42fwncp3n9oQ8Hk\\\\\\\/Px7HTO9fqY0mvTXpi9YBhEXm68Ue5RSuxVM3ML9GVggaFPnlacFo393wbOb1lmSkGW0X+PTgl68R3\\\\\\\/+HvFyg22aQD7W+HICDubBCTeUphQDjqz7+iaFzb\\\\\\\/9CYxOrp1m8mymES8aoOtFm5cCXQQcNrqULr1jzHiihWLbtzMvhswn2EiYmVVIzx\\\\\\\/riRHe4aK62h6ZJwmaVkxhOYqMDIzbEeSbL1cQhfNja1IpafH6nzQSZpF8uaSqpjVWcMqJZ\\\\\\\/tHFuUzBVGdkFWfgD6VuZ\\\\\\\/JlNg50z5H5o0KETh8QTvv\\\\\\\/Vlee97DKJd\\\\\\\/R8cO+09mzNqs6BzsoiHq2uTo+Uzsg\\\\\\\/E1VuHvTy0CxfyqtY8xqsImA2GV8xphcfOJ0YsyKJHJI0rBWEOWVLgA+xUbD5Y6S6BynajQiXBwgSXFfRhUqBXNU3rCExvw6HwNvUnL\\\\\\\/gBTcLgKQ7KHf8ebv4cTuhrzriORsJwcTqSVxwKgRoRMVGX5FfVy0455lA\\\\\\\/INjshMBcT+zn3fLyyZAt+yzAwUhb2acMcw21gDAPaZnJ\\\\\\\/li7+xfcWpum8Q3UZrelLmWcmJPHnDuZNgXEOVkRtg9VjhgS4tDjQRfZ+wVrlUFwxrRgR7KyJV\\\\\\\/TMTg6VJQ6tSYugH6FxzfvGXaPW6I+PFnr+Pbo1VLWQgztx5A\\\\\\\/dPYCfTnyROiF8xyeSXxL8E0e8K5VhCAKHVC4nhLQNSRP2xOmZ0N20D4MDi\\\\\\\/GEu8m6Zh\\\\\\\/N2kpKKy8Zbae+tXR97Zx6OPvZUAsR7eSL6YPz2rawk9HLmDewDjyWI26t\\\\\\\/r4QkoSFI+BsxCHV9lEBOF0io6j8LGESyd\\\\\\\/d2y75\\\\\\\/XuXwPtSux5OQW7nDxq1gtnP06xwqOxmw893lFZ58BNhtac2eUi6TUKCU8iqBM04yRs1+CZFXkk29YHR\\\\\\\/IdyhFvTInFzf\\\\\\\/OOxOkqdGbRhZio5apXu3W0oXEnUFYdwgarkxBwahA0ezZNa\\\\\\\/b+Olk2UMfW0LF\\\\\\\/8GwGI\\\\\\\/ZzsbdMKlTU5vnWpDLxHCyOJE7sOlGCzQFHIsuRfgkhrJlXg2ge5R8dtvbYhloi+Oqv\\\\\\\/Tx+EiH2hWNa8pO\\\\\\\/lmEYE8Ss0oA6CikpYZVrJ22wcw8mfObfXQ28waTDLO\\\\\\\/Ge0xxa9\\\\\\\/mKjDxVkJrMyHKgzURzVYldnFQM8q\\\\\\\/WCOlioQbcu3CyVY0ZOLvHeB5R3N41heMvPT1OwfoBPkGsZKcdgPV55sdgBSI3dkkhLybfBA9+uCfUckECogZ7\\\\\\\/I0w36uyp6DnOD2+s3irBlpqcFIbjDoEjMofnggT1I\\\\\\\/pB4xcDI2+8TN1dAJj8X8dzH+IHWR8BEeMtmSxb7wYxxoflpYijzG62\\\\\\\/VEXCpYUyiTeVUuOYeWUiu1cEXfoa3ufWQ8nXnvXNJjBn+zSqn7Xli\\\\\\\/aoeaNvwaf+JFMT\\\\\\\/YpKCfNblbrVlq+Wz67JWx4P133Hb4mm2wYLrVZW0IyJfVY95+P0xTyUFtNuuFGTkztmffPVm3TTLoutYFVZBZxYS7oC3Yx7QGFTdH4X+SgGPtAC2ONvmjsjTR\\\\\\\/wLIiCLTfVNpLCqcxVwCA1qoW1OEBKRVlsh5bKXH9g0zJyhoB6QmkGVN\\\\\\\/68ml71jIl+TDzhIUAEU\\\\\\\/03uGKVq+dVogOLvy6LTvv2wQAkB9uXe+2\\\\\\\/6k+ljGEcErjI2pu1sRQvtlf+NOd1n\\\\\\\/B1N7Py\\\\\\\/GSL5SvTQ3ZWu1P0ypk1GfGO67w7SFqoZb3WNwcJcj4zhjDbh33zb6lefmd8HnTSPY6IYMHgLJ4s6e7Res1VTU3YAuh9g2gAM6pO6gWTlRS3fnur3ie+z5aUR0wsFlvoDMBeHNMefF6fk1cO+AwoMGMUnpdXNcBUvXJ36dYFsvEgbGPdIHJiV6IwcvzPYieUMABS5brUj5KG3\\\\\\\/Ig6rryn\\\\\\\/fsu0K4tjv+dYuvczMjF61PFDIMnE3Xbu9JORqrDRxZ3Co4OtLbQtdbrH\\\\\\\/HGlcYG+gPU0xwD55eGplT8l4PvpKwFPrpOH17aDKiuF\\\\\\\/FksGN6kwvFb53v8Dx7TDq36axmlJp26pCYekjo9VDVfFxGewvaGZIQUMv5Hil27XV00fKqZz2\\\\\\\/h0KWpI3dR3xbi4FaC\\\\\\\/yByrUeHGRNnwo0ZLRaMuJW07et5NEZswq4ifZpjmHcDHad4NVYqHuH3bnssKJH5NI1oJ3RLLpJJd5qAy8sRfNLff4daISjWjfpGyWPPi0WX7DLdwWGZW2korjla3AZi\\\\\\\/PhN+uP18dsymJqrvoGhPGPD1bVUtxnbAZXYoF2COqiRYHb3fwzqJM8i3\\\\\\\/7KdgF5GssMSRZG34lREdFXv5O5F2h5Y0qaywnsIrDB\\\\\\\/KpYxMv3f33djgF11fHWq12tRijt\\\\\\\/TCgpIW6cCJ+XqnaIMmRWXhCUZhIRl02HvtHVjDWnZYnlEoVKw3tKYvrj+1LyqvO3cIFkrfUOkVdTFsV6ZkUZaiMgGTeooH48KrSWp4BPdGt5CxB7FUe4fp6kVZ7pup04I8v\\\\\\\/z\\\\\\\/s53+8sZZJw\\\\\\\/Uo+OqHukxXAiVPiK3yC23K3GT3jl6TNJa0BCXXKcwHaFoUGYlJvSVzmpK44L+\\\\\\\/joJgtAmxDpI3gtn33c1OL+sw+0Rc05naIieiex+7mDJ8ntmhSDeA4zOgnskZY2gARXbSD1YP8wcVjRYObSz+Vb48Q9o025b2TvtKNbLQ8\\\\\\\/bxw\\\\\\\/PqDT7LVzbCW1Qp6Q2\\\\\\\/W9gXohzXUEU+vX2BKkBhzgkJ20r870bmVZZeA9Lm8U52mfPweMTg\\\\\\\/OMfu2PFg5THrYOty\\\\\\\/w1m4aHAbHDIPo8KD08OfvjJlFl1355WlgRaXyT562i90TC5sacX7E5hLp54Xzt25KhrLJV5SdUznnTkMyAl\\\\\\\/CDQSEzX89zaHDgy\\\\\\\/AGAuHrZsXMHSRvpPFykZ0Wyp17m24Gs9d49oK71yXXPupp\\\\\\\/qaoihgLttdPAfY9lof9pCK8D76sS9LQ6eMe2mQb1wILfn5bQZUCsvI2RUuNldY2kGF939Hds1kC+EYOH0B8fO0xSgOfUuyT2p0qXc1UT8NFdYGWAWgMNlCNZLBVQfinliYfNfMXfRVnKXPQc\\\\\\\/wt1wI6dy75keg4ZOyQGefBEZTtDe92DQQwLEZS7E5\\\\\\\/TIAIeMiz97zIgCx9VA6DZPKSF7V+M7PD8DJEKWJ7zrglpCfJkURr2lG0z0jGUOlG7hC5yXjuai70v6pVSaGExxtd27I6giUA8xW712xwd56ru9otipp70Ne7GjogZd3lxs0FBS4Eu09WDtEokDqtIG4Cp8x\\\\\\\/i7qW\\\\\\\/9QhZrWeIBVmcEj7J\\\\\\\/0TfjWnEX6gZilj44Bc4ijw95Ti46LeUGjDL\\\\\\\/2Nbu5nrmLRwrJyn7MDvJK0WhDs8YKsTgK8tvEFnW5qBCqhhSwz+vZGD3SqU8gxG3Vuz2DYJbomLQxNRhnWBhhkhuRIBKi9t+gfA\\\\\\\/e+NLB\\\\\\\/9kPgPFTr20vdKp+6sqbYg7m9IVzDdcp3K\\\\\\\/1tdZ16+Tm2x6GtAsTFPjCfcSg1Fp7zJztJ4wEmsN9dosi\\\\\\\/twwOZqlTYrZlrBf5T1neiEjefElp59dk4kxfjfzrxZHA6dgE5Ym7iIwyDIacUc\\\\\\\/eLkdnENc8I8gUTeUy6W8LHXW8B3iXpqUsBvi53fsi6+gA\\\\\\\/QexB1fW8g9LxsfMmyZAyTenIVgJuMLqHpE8iUNH6KkJWpJBCA8IgGTuvfWPwweMaMZu25aKCrEeudlBe7nS5nrpNxt5O9eNLiopex128A20BLpKnk6G7mdkDR3zF7TqW4B9KEtFKPG5byk9\\\\\\\/VEpQlRW0NIfUxxHwTf3ajveTYqoxtQ47PGKTYlBJdjWKBqCpQ2Q1gsaUMpQem1LL3IaXDLhcpX6NqTutzeKBzBiidwL+HCjWnEm\\\\\\\/hK85laR1USYA2wsQV6QkxEcCcFkYgUV8cQ4bTr4MXagP7yeo3FKO6bkQmjgVEWN3+11HrZg1EGlus3DRrcBnYxOEIf6XpT19o7aZde8mf3OmA0Q\\\\\\\/A98ysqxAUg9DX76mM3f61X3fl\\\\\\\/UL8FAzIX8NlKxf2f3Opq9AzslIwPRI\\\\\\\/xVK10TsxptFRmrLzGbyqtni\\\\\\\/lcHDAr9a00PXUbCvd1aIWPnsr5SR+ctHIXObuUa\\\\\\\/zZGy3hR0CLTHPWTglQ\\\\\\\/YNI0YiqwsI454v0HzPD+wlDImzMIRHzNlS9S523qOzzymmnQ3vohjGlTWui0BAvWc8gGY+kU6wYPimOghUzZR9cZP\\\\\\\/G6cAM75wWzAHLFhFRE9F1yV1DBBhWPzndvnZu8TUqsIGbQbX9GZwTIp9+3AAFBFqRryWODc0\\\\\\\/p0oj8JXg3MPaLviZuWxNtTKN5P4qzY\\\\\\\/l8xw6Icf50mGBJ2TcLSlP\\\\\\\/PHi0hdEd7WrCCadOySf+tQJXG3X9IajQRPJjyjzhPTLuNgrHajr99g4cPqyr23YVilW+DuOgPnInYvv+tcOWGy1iebv8X+1bwMItszf0Cir9wUBozvtgq1miF\\\\\\\/\\\\\\\/njvqnNTEcQx6CN47iHSWpqCgN1ky51PMEbBhSMKyKH8Sa1l1lHi8Nal5yQTKPOPy3iJl+\\\\\\\/CGpL0Tedjsz5qMUEZGa3fioKIISuCsEo3fM3sUaDNShpiy9MKIc7NMIlLM24jQIwDItERegi8KF9kAP1xtz0a3tTYT\\\\\\\/mRhmjmqkwcZY3WeOHDlHnQrQozHg3L1Na2OyTFF3J1tHQAyAEucRJheXYBnwOtDRxogaVpFdLpyL5hCgmbVAF4StNP0\\\\\\\/msEIg\\\\\\\/z9X8PSMGguPZ5uXsJFUkG\\\\\\\/kVAmnp9AzX5bsFnQOF4hIy1PB2IXxYTKMsNGEWHB3\\\\\\\/7M\\\\\\\/52c+6WxZ7IoiKmytt4D36l93DUOMb\\\\\\\/xV365OBGAyccXrOYB4TS8yStUVUou\\\\\\\/nLlcOTr1qV0ypLo4ObPvopExRfJB3v37aec7u2MuiV20qmENHXV0b8G\\\\\\\/rZeYAWLF+77VfHG27IvNtheMizCQr6lg5y4+W2Q0zCm9kSRZptrye+yk1Z+wIcdV2r0uFPGJdGyonHPzt3MqsFngFr\\\\\\\/\\\\\\\/\\\\\\\/wdCxx81bt\\\\\\\/oCf3jbHAqabPVWADnV\\\\\\\/lgi7ABVbsSAREvP5QYZFqjTdnVFKBB8RfHbdcCPsu3BzG0eNDLyliuKNumCHQenLMzlv1af4L9hPPO2F62UI76\\\\\\\/EhbhlS\\\\\\\/4rN7i5n2gnzYaBadpAF+bnrIn6osDRUi0WKp5j8WWtMtUVkHMuVl0X7qFwIuwkLxU\\\\\\\/doWrU1mTOpxVV6ZouY\\\\\\\/RVfzKM\\\\\\\/wTvqMpuT2duTMpDRWPEjeRfbxAmru0WIxNuZymIZDzQBHCKz5ifpX7DCqVN6XGSwKrYu1P7E7PNB6G+C1GSETU5W+NPM1\\\\\\\/5bT4cFBlX6wjKM3Cd4iONs9TcEUTDzzOKAx6DlP024DgS8961uHnhiiBHYsFg+X5fSPMo6K7C4jvvzI1RZXz4hs+maLP1af\\\\\\\/RafhR5RzlHVYoF0kpicL4yN4N3xWhU6BoLbmnWpor1ymGX965fC\\\\\\\/d69kXP7Bt6c7AUIdz09BBCouKFQTRGiA+T\\\\\\\/OZMTRkOtGCSjhtzyK3eXGvyYoDdugzthlGlm\\\\\\\/9AQ2aeaaUdacwKy9qRrDebdYmFIMytU8ScQCyIGJ6VJpRmUKteuU\\\\\\\/EPLkChRQ+aXs\\\\\\\/lICWRPdXiwQqtSnCPdInsHbROF6pKvJVFojhVbanz7nJjirs2ttOXCrkAEUOenCFj5IymxIhdqdeudb4+IOotIjYrN3W0yK0whY8h5WwBVrkRP5VQSBkgtN2\\\\\\\/l6eruDCDxr9RqTJZVMtJLHw4JfFvRk+jMkjvYp2BVvvgIRbqGFqquzoJbNaJgu9GxbuHn57SyUEAi5JdIiVXInsjQrrHS0nVFL\\\\\\\/1En6CV6A\\\\\\\/2jxsKWiriSc6hG5btyqw6FC3isY5\\\\\\\/92iCMpPIHT9\\\\\\\/ObduiS51tbi20TAdgNMMIiGStHocGsVROCX\\\\\\\/QxcKZKJ0Hz+\\\\\\\/CJejQk9nLc9+BFAY6PLYEA99dkTT0SWZ9nr6Kn9J+nYEJoT57+um3ZXmrB8icYWQPEON1lLDJdP\\\\\\\/EFDH5xB+dwlV3SZtKXq\\\\\\\/a4oh+K23Z83X3av4QOb032eTfA2baBuL57xiZZmQPMlbyFP9mfYYvGB13LahFKjgJlczdPl019qLy4hm0gZ1FCeNlId4aPbDE7xgaQKgRgyfIP8z8IIowwQYzTAH11AcFrAD50PVtK9peZ5+XdUzlnE6Mw1SaKdEOX5oq\\\\\\\/TN9JZAFzZ1YuJnOjf3PcTg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xde30ok3dcD8YYrqH3FgkQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222c10f175b56564083011d1dedae00efc9a92b9cd34015644e9eaeb65f9b24020\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WgOK7NcE4OSJuFeDrU\\\\\\\/6kpssX67trHp2xysdLKnz8v5\\\\\\\/ZS3PJLApPdDy3swvsXoDY6Ytf0XHgErzJU4LbMKizja2P3ZKgbphbE6ccRr0VbXxFbHda+eBPQDFWlFZwiyTgzt5zjLKLPH8BE2h\\\\\\\/Lli\\\\\\\/fErtTlX0Q63bLIfk8vM5o6KLPw9FZwLIwK66mstzQ936p8NkENtAt+AyDmkJHwoSU6Elt09hbUQK9WAJzH130G6dee0cCNrmR8OXo+I9fRYzELiDQicIavJsTm5L0Glt4xFmaCucm0pcDK91h0Fg6O\\\\\\\/StYcTKnjedPyUU0o3gSC2nWztqsbiDBM0W5huqYtbpuUJKFjGi9dQJ7xZbBloXfUVWtaYKQfsYm7So8RTNcTt2eY83MWBibi0EN1j2BO43egoN5FOMNmRBFU7Oaq96XzURFEbLn7Ni\\\\\\\/nObx6nrISAXXkl\\\\\\\/rLd1HDBrsPzB0Qjyp9Szqkff4U6kg5WMUIif4lTcR0DPpUrWU80yakTvgDDtPmpLoL4mzEVGqO4L7MhKy\\\\\\\/\\\\\\\/e9JekAaCFU11UZ3ZyqPhsMS2G6oV+mGmlhgDS47HKo6MnbT4XdkAbFoutJUr8rrb5dvOYxcuZ8W+T\\\\\\\/MqFB+K01mIldcuVqwY8HPtWUiYjza5\\\\\\\/tjUGWgYFDpnKUywmPzDghG1f99gouMXi1ROuAqYrgzIewkWx537DR5KHIf7u2cwbNGBGSEA3X7Vzj5lWU486l2YYxTfdbyAEiXvxihtMX4UBZ8UyIoAn3vKWTXwFZEtRlIo2gzmsw2UV2VDPIjS4mXE4tOBbnZUOlTB\\\\\\\/oUlOY8P0W\\\\\\\/3ot4w5fl72AGpPsnIv\\\\\\\/in\\\\\\\/AkvCIjpvoghUBFOSxWkLrYSwqC5ZLNYfS\\\\\\\/QIqwyhhV+5WK91vtSI1V642yT2tLslf4X8qr5P+xjzj7DX37pWHq2RQkD8pBmlScIGwQMbYa6D1p34C7SQyidN1uxwwEZfa+NfhGCBfstIOsy3\\\\\\\/wJHw0TojkhSGUucr8vd1lfz8H\\\\\\\/2rkM5TpzEuFXo\\\\\\\/UvgfzQjmhthxGS038zc+oh+JBNr0Fzw+VPKYqzPYD3JyLJPY6ySsFQ2pNW7pi3llpq94UIOykFEqEO4Zw74AGC5\\\\\\\/JHLQ8wUQ7rule0KGdkU4C\\\\\\\/xeCu88LGfpWwmbI+9NuH10Zh7Pwo6suet8\\\\\\\/lJy+o+QueLzSK+1MR8h0aokDGNCJcHaEvTSJnJ8fHOR\\\\\\\/29nSPQVv19Wv5HiZ78M5ZNpJ9uE30bZRS5hXi006DT1IGmOgtUw9dtjCOn0paUdQDow\\\\\\\/10vzWiH+QJZ1i54NlXeAlL1KU7xozSxemiV6M0q22PnMDSl20EqWSBtfWg83bmje5JLfl4+48hVi3AvBkE+NaQGZK6EQ\\\\\\\/3NT7x+ovRRbcnqiGKArll4Fk7VAz+tZwzOjAjhn1yHuTkuGZmedEAZBYHvaUHacs9ZCQkynMAxyC3UVKK1SqIUwpBAkRTvk7Ja1y54fTrymzzPtsqYtTmCEXRjIopQU9FKyfONvycUQUKQonUY6syO+7un0kWxpRpeMwdPFan3YEeVYd\\\\\\\/xyxJxhbOif4d9VFEd9YLeb5TOTJcGwXDQvmylrWFnH9NllAjVlTlJy9WyANeURRIIe6gF+4PPUPx2YbwP8nj0PGgHGJzxoJ0pUhs09O7wuUyX900CrHNXGzAfmB+MlQnpsmxN+hPRhRlmgOn9UASFC4Ilw1F4A5DEPPNGHKLmmLE\\\\\\\/oF3LOi6hvZ+sC2tN\\\\\\\/QBtbFCl8tS56JINcQrZrppHRFKmLpKamrtsERjBwMx4i8Rryj4YME\\\\\\\/SpB9Xc\\\\\\\/un8HxP\\\\\\\/tvJYYKae2bkFUaQlf5EWybqNaWaRsTILSESVnqRMVINPR2YBdBRHoKg62BWJMx6g8VBQYE9HQXRCq1S61vT6hNfW\\\\\\\/Lm6eJe\\\\\\\/scu03m7bgZb7NRmhwUOS9DBbt9x5oqhRdkCaDy7cPR+EAWm9PMhA99FrVbRa\\\\\\\/\\\\\\\/DAkwK1aVrSIkqs9Sg4rTUKUyy9DpZ2kg\\\\\\\/3MQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LME9p8BHaEzEZsqAMoOn4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002251847efbf52038bafa9412212553ef1fd5de4125abf29e2d4381b2ba8bf43e5a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488767 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/933" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dudXn2GNzItz367LdlJWIFqrWskkRpy1Ls_-JmMg1n8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/933" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jcxAKmIOTLFJrnZOWM-5fo0bYS8wzrE9R08CA5QXMN8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/3/923" title="Identification and Validation of a New Source of Low Grain Cadmium Accumulation in Durum Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/3/945" title="Genetic Diversity, Molecular Phylogeny, and Selection Evidence of Jinchuan Yak Revealed by Whole-Genome Resequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488767" id="node488767" data-pisa="ggg;8/3/933" data-pisa-master="ggg;g3.117.300354" data-apath="/ggg/8/3/933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3360-6979" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ani A. Elias</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ismail Rabbi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">933-944;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300354 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488767" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ani A. Elias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ani%2BA.%2BElias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Elias%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAni%2BA.%2BElias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3360-6979" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ani A. Elias</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anianna01@gmail.com" class="" data-icon-position="" data-hide-link-title="0">anianna01@gmail.com</a>
<a href="mailto:jeanluc.work@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jeanluc.work@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ismail Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Institute of Tropical Agriculture, Ibadan 200001, Nigeria and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsmail%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ismail Rabbi</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Institute of Tropical Agriculture, Ibadan 200001, Nigeria and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agriculture-Agricultural Research Station (USDA-ARS), Robert W. Holley Center for Agriculture and Health, Ithaca, New York 14853-2901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anianna01@gmail.com" class="" data-icon-position="" data-hide-link-title="0">anianna01@gmail.com</a>
<a href="mailto:jeanluc.work@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jeanluc.work@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/3/933" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488767" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/933.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488767" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/933.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488767" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/933.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488767" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/3/933.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488767" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Plants competing for available resources is an unavoidable phenomenon in a field. We conducted studies in cassava (<em>Manihot esculenta</em> Crantz) in order to understand the pattern of this competition. Taking into account the competitive ability of genotypes while selecting parents for breeding advancement or commercialization can be very useful. We assumed that competition could occur at two levels: (i) the genotypic level, which we call interclonal, and (ii) the plot level irrespective of the type of genotype, which we call interplot competition or competition error. Modification in incidence matrices was applied in order to relate neighboring genotype/plot to the performance of a target genotype/plot with respect to its competitive ability. This was added into a genomic selection (GS) model to simultaneously predict the direct and competitive ability of a genotype. Predictability of the models was tested through a 10-fold cross-validation method repeated five times. The best model was chosen as the one with the lowest prediction root mean squared error (pRMSE) compared to that of the base model having no competitive component. Results from our real data studies indicated that &lt;10% increase in accuracy was achieved with GS-interclonal competition model, but this value reached up to 25% with a GS-competition error model. We also found that the competitive influence of a cassava clone is not just limited to the adjacent neighbors but spreads beyond them. Through simulations, we found that a 26% increase of accuracy in estimating trait genotypic effect can be achieved even in the presence of high competitive variance.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cassava" class="hw-term hw-article-keyword hw-article-keyword-cassava" rel="nofollow">cassava</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/interplot-competition" class="hw-term hw-article-keyword hw-article-keyword-interplot-competition" rel="nofollow">interplot competition</a></li><li class="kwd"><a href="/keyword/predictability" class="hw-term hw-article-keyword hw-article-keyword-predictability" rel="nofollow">predictability</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">The main goals of field evaluation are prediction of genetic values for targeted genotypes and selection of elite genotypes for advancement and commercialization. In field trials, above and below ground interplot competition can be expected for resources such as water, light, nutrients, and space. It is important to account for competition when present, as ignoring it can result in increased error in genetic value estimates that can subsequently limit the advancement in the breeding program (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Besag and Kempton 1986</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Stringer <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rebetzke <em>et al.</em> 2014</a>). This interplot competition can be confirmed by negative correlation between harvest yield of neighboring plots. The intensity of this correlation depends on the genetic relatedness and spatial arrangement of genotypes in the field (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hinson and Hanson 1962</a>). Bias due to competition is expected to decrease when related genotypes are placed in proximity. However, field trials are often limited to a single plot of a genotype when resources are limited and a large number of genotypes are required to be evaluated. In this scenario, the genetic relatedness can be accounted by incorporating a relationship matrix in the mixed effect statistical model used for prediction (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Muir 2005</a>). Genomic selection (GS) models that use the whole genome marker data information in calculating this additive matrix are useful in this scenario. This approach can potentially capture all the quantitative trait loci (QTL) that contribute to the variation in a trait (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hayes <em>et al.</em> 2009</a>), and, consequently, can deliver more accurate prediction (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jannink <em>et al.</em> 2010</a>). In this study, GS models are extended to quantify potential genotypic and error variation in competitive ability, and, thereby, to improve prediction of genetic values.</p><p id="p-4">Competition is more important in scenarios where genotypes are placed in small plots without bordering. In such plots, performance is more strongly influenced by the competitive ability of neighboring plots. For example, when there is a significant height difference, the yields of shorter genotypes are often depressed by shading from the taller ones (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kempton and Lockwood 1984</a>), which are aggressive in using resources and thus grow fast. Related genotypes when placed together can also exhibit significant competition in a high density field (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kempton 1982</a>).</p><p id="p-5">From an agronomic perspective, there are two popular analytical approaches in estimating competition: the phenotypic interference model (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kempton 1982</a>), where competition is assumed to be directly related to yield from neighboring plots; and the genotypic inference model (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pearce 1957</a>), where competition is assumed to be related to other agronomical characteristics and is therefore a trait of the genotype as a whole. Assuming that competition is directly related to the yield of neighboring individuals is well applicable to root/tuber crops (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Besag and Kempton 1986</a>). Studies done by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Connolly <em>et al.</em> (1993)</a> in potato revealed that accounting for competition in estimation increased the shrinkage of extreme values of yield. This shrinkage indicated that accounting for the competition effect reduced the error in genetic values due to the fact that high yielding genotypesâ performance was at the cost of low yielding/weak genotypes.</p><p id="p-6">In field experiments, error in prediction of genotypic values can also occur due to the presence of spatial heterogeneity. There are ways to incorporate both spatial and competition effects in the same mixed effect model. Spatial correlation can be modeled as a random effect and competition can be modeled by an effect given to the neighboring plots (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Durban <em>et al.</em> 2001</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Matassa 2003</a>). To accommodate both effects <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Magnussen (1994)</a> suggested modeling them by means of nearest neighbor adjustment and a standard two-way ANOVA in forest trees. <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Durban <em>et al.</em> (2001)</a> used the inference modeling (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Besag and Kempton 1986</a>) in order to adjust for competition, and a one-dimensional smoothing spline to account for spatial heterogeneity in sugar beet experiments, but considered direct genetic effects as fixed. <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Stringer <em>et al.</em> (2011)</a> proposed using a first-order autoregressive (AR1) covariance structure (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gilmour <em>et al.</em> 1997</a>) for spatial heterogeneity, and second or third order autoregressive residual structure for competition in sugarcane. They also considered genetic effects as random facilitating genotype selection. Pedigree information was used for assessing direct and competition effects by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hunt <em>et al.</em> (2013)</a>. Their competition modeling was based on the inference model suggested by <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Besag and Kempton (1986)</a>, and its modification suggested by <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Draper and Guttman (1980)</a>. The spatial heterogeneity was controlled using first order AR1 covariance structure (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gilmour <em>et al.</em> 1997</a>). Earlier, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de Resende <em>et al.</em> (2005)</a> used a very similar approach but without considering the genetic relationship. <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cappa <em>et al.</em> (2015)</a> also suggested (in order to take advantage of the genetic relatedness among individuals) using the additive relationship matrix to estimate direct and competition effects, and a two-dimensional smoothing spline to assess the spatial heterogeneity. In their model, the variance components were estimated using a Bayesian approach. The competition modeling in above-mentioned models was done by modifying the incidence matrix to give weight to the nearest neighbors around a targeted plot <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cappa <em>et al.</em> (2015)</a>. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cappa and Cantet (2008)</a> considered a function of the inverse of distance between the targeted plot and neighbors, as well as the number of neighbors, while designing the competition incidence matrix. The distance function to modify competition incidence matrix was initially suggested by <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Muir (2005)</a> as a function of the square of the inverse of distance. The models proposing simultaneous fitting of competition and spatial heterogeneity were useful when strong effects of these factors were present in order to accurately estimate the genetic variance (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cappa <em>et al.</em> 2016</a>).</p><p id="p-7">In collaboration with the International Institute of Tropical Agriculture (IITA), we analyzed the competitive ability in cassava (<em>Manihot esculenta</em> Crantz), a root crop, in order to enhance the selection process in the breeding program. Cassava is a subsistence crop in sub-Saharan Africa, and is the main source of calories for half a billion people (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">FAO 2004</a>). Breeding in cassava is initiated with the production of seeds. The plant produced from a seed is later used for clonal propagation through stem cuttings. For clonal evaluation, these cuttings are planted in single row plots without border rows. In such a scenario, a differential effect of competition among neighbors that affects the phenotype is a concern among breeders. We propose three different functions of distance to account for interplot competition and reduce the error in predicting genetic value. Unlike in previous studies, we assumed that the competition could go beyond the nearest neighbor but that it declines with distance. We also account for the dimension of plots as competition between neighbors along the longer plot edge would be expected to be higher than that along the shorter edge of a nonsquare plot. Previously, we conducted an exploratory analysis to estimate spatial correlation patterns in the performance of clones in the field (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elias <em>et al.</em> 2018</a>). The GS-competition model proposed in this manuscript was also extended to account for the presence of significant spatial correlation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Materials and design of experiment</h3><p id="p-8">We used information from cassava breeding field trials conducted in 2013 and 2014 by IITA in Ibadan, Ikenne, and Mokwa in Nigeria. The clones used for the trials consisted of IITA Genetic Gain (GG) population in a preliminary yield trial (PYT), population from cycleÂ 1 (C1; progeny of GG), and cycleÂ 2 (C2; progeny primarily of C1). In summary, 83 parents from GG population gave rise to 2187 progenies for C1. Later, 84 C1 and 13 GG clones (total 97) were selected as parents giving rise to 2466 progenies for C2. The clones were assigned to a field in a randomized design using replicated check clones (1â10 check clones). The fields were partitioned into Ranges and Columns, with unreplicated test clones belonging to the same family assigned to adjacent rectangular plots in a Range in C1 and C2 trials. The relatedness of genotypes was calculated based on the additive relationship matrix using all the SNP markers with &gt;1% minor allele frequency. A detailed description of the population used for this study, experimental design, field and plot orientation, plot dimension, and method for calculating the relatedness of genotypes studied can be found in a companion paper (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Elias <em>et al.</em> 2018</a>). Total number of observations and unique genotypes for each trial and trait can be found in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Supplemental Material, Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>. An observation was calculated as the average of all observations in a plot. Four agronomic traits were evaluated to estimate genetic values of clones: fresh weight yield of storage roots (abbreviated here as FYLD), root dry matter content (DM), fresh weight of shoots (SHTWT), and harvest index (HI). The DM is the percentage dry root for 100Â g of fresh root. The FYLD is the fresh root weight measured in kilograms. The SHTWT is the total fresh weight of harvested foliage and stems measured in kilograms. The HI is the proportion FYLD to the total harvested weight (FYLDÂ +Â SHTWT) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ly <em>et al.</em> 2013</a>). The HI indicates the ability of plants to partition biomass to below ground roots that are harvested. The traits FYLD and SHTWT can be considered direct measures of production.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488842/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488842/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Root DM, root FYLD and SHTWT at harvest, and HI from various experiments that showed decrease in pRMSE value in Model 1 or 2* compared to Base, and where chi-square values are nonzero</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Competition model framework</h3><p id="p-11">In a GS model framework, the phenotype of a plot is affected by the genotypic effect for the trait of the clone in the plot. This effect can be estimated using the genomic relationship matrix as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-1.gif"/></noscript></span></span>where <strong>Y</strong> is the response variable (<em>e.g.</em>, DM); <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-2.gif"/></noscript></span></span> is the general mean; <strong>Z1</strong> is the design matrix for genotypic effect, <em>n</em> is the number of observations, and <em>g</em> is the number of unique genotypes in the data; <strong><em>g</em></strong> is the vector of genotypic effect; <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-3.gif"/></noscript></span></span> is the vector of residual error; <strong>I</strong> is the incidence matrix for residual error; and <strong>K</strong> is the genomic relationship matrix (here, the additive relationship matrix).</p><p id="p-12">However, in breeding evaluation trials, the phenotype of a plot is affected both by the genotypic effect for the trait of the clone in the plot and by the genotypic effect for competitive ability of clones in neighboring plots, as well as by the impact of that competition on the trait (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Muir 2005</a>). Thus, a statistical model can be used to estimate two genotypic effects: one for the trait and one for competitive ability. The incidence matrices for these genotypic effects differ: the incidence for the trait is to the clone in the plot while the incidence for competitive ability is to the clones in neighboring plots. If we assume that both the trait and competitive ability are primarily determined by additive gene action, then both genotypic effects have a covariance matrix proportional to the additive relationship matrix, though the additive genetic variance for the two traits will differ. In addition, there will be a genetic covariance between the two traits. Based on this hypothesis, a Model 1 can be built as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-4.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-5.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-6.gif"/></noscript></span></span>where <strong>Z2</strong> is the incidence matrix for the genotypic effect for competitive ability whose construction will be discussed below; and <strong><em>c</em></strong> is the vector of competitive ability genotypic effects.</p><p id="p-13">As for the measurement of the trait in the plot, there can be error in the estimate of the competitive ability coming from a neighboring plot. For example, a clone planted in an adjacent plot will exert less influence if it was established from weak propagules and therefore does not grow vigorously. Thus, as for the genetic effect, there can be error in the measurement of the trait and of the competitive ability, as well as a covariance between those errors. While modeling this scenario, the dimension of the incidence matrix is the as same as that of the residual error, but, unlike in residual error, the diagonal is zero because the incidence of this competition error is to the neighboring plots. A covariance between residual and competition errors can also be expected. In summary, the residual error (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-7.gif"/></noscript></span></span>) from the Base model can be partitioned into a competition error and a residual error. Applying this idea to the Base model gives:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-8.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-9.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-10.gif"/></noscript></span></span>where <strong>II</strong> is the incidence matrix for competition error whose construction is similar to that of <strong>Z2</strong>; <strong><em>p</em></strong> is the vector of competition errors; <strong><em>r</em></strong> is the vector of residual errors; and <strong>K</strong> is the genomic relationship matrix.</p><p id="p-14">A full model accounting for the direct and competitive genotypic effect, and the competition error results from combining models 1 and 2:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-11.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-12.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-13.gif"/></noscript></span></span>The construction of <strong>Z2</strong> and <strong>II</strong> follows from these considerations. The ability of a cassava clone to influence the growth of neighbors need not necessarily be limited to the nearest neighbor plot as roots can forage for water and nutrients, and shade can be cast further than that (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Izumi <em>et al.</em> 1999</a>). We are not aware, however, of studies that evaluate the rate of decay of competitive effects with distance, and we wanted to allow for fast or slow decay. To account for all these possibilities, we used three different functions relating distance to the extent of competition: nearest neighbor (NN), fast decay (FD), and slow decay (SD). The NN was the simplest, and we tested two NN functions: (i) only the neighbors along the longest edge of the rectangular plot were considered, and they were given an incidence of 1; and (ii) all the neighbors were considered with neighbors on longer edge given an incidence of 1, neighbors on the shorter edge an incidence of 0.5, and diagonal neighbors an incidence of 0.2. The FD and SD incidences were smooth functions of the distance between plot centers:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-14.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-15.gif"/></noscript></span></span>where D is the distance between plots; <em>k</em> is the FD parameter whose values ranged from 0.1 to 2 to accommodate different competition intensities as the distance between neighbors increases (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). When <em>k</em> = 0.4, the incidence matrix is similar to the square of inverse of distance (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Muir 2005</a>). The SD function is a modified logistic function where parameters <em>c</em> and <em>b</em>, respectively, takes a value from a pair set containing pairs of values as {(0.9,0.1), (0.99,0.01),â¦} (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The FD function assumes that the competition fades considerably beyond the nearest neighbors, but can continue feebly as the distance increases. The SD function assumes that competition fades slowly and its intensity can remain high and stable. The SD function is also modified such that competition stops beyond a specified distance. Use of distance-based competition functions and exploratory model fitting both contribute to account for missing/dead plants. Specifically, the deviation in the competition effect from a plot relative to what would be expected had there been no missing/dead plants is captured in the competition error term, irrespective of the genotype.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/933/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Competition index depicted for FD (A) and SD (B) functions with respect to distance in meters. For FD, the competition reduces to 5&#xA0;m, which usually is the distance between adjacent columns in the real data study. For SD, the intensity of competition is assumed to decay slowly and is conditionally terminated at a certain distance. In the depiction here, it is terminated at 5&#xA0;m and still have &gt;60% intensity. Parameters k and b, which were used to generate FD and SD functions, respectively, are represented with different colors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310653034" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Competition index depicted for FD (A) and SD (B) functions with respect to distance in meters. For FD, the competition reduces to &lt;25% when the distance between plots is &gt;5&#xA0;m, which usually is the distance between adjacent columns in the real data study. For SD, the intensity of competition is assumed to decay slowly and is conditionally terminated at a certain distance. In the depiction here, it is terminated at 5&#xA0;m and still have &gt;60% intensity. Parameters &lt;em&gt;k&lt;/em&gt; and &lt;em&gt;b&lt;/em&gt;, which were used to generate FD and SD functions, respectively, are represented with different colors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/933/F1.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/3/933/F1.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/933/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/933/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488866" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Competition index depicted for FD (A) and SD (B) functions with respect to distance in meters. For FD, the competition reduces to &lt;25% when the distance between plots is &gt;5Â m, which usually is the distance between adjacent columns in the real data study. For SD, the intensity of competition is assumed to decay slowly and is conditionally terminated at a certain distance. In the depiction here, it is terminated at 5Â m and still have &gt;60% intensity. Parameters <em>k</em> and <em>b</em>, which were used to generate FD and SD functions, respectively, are represented with different colors.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Simulation studies</h3><p id="p-16">Because the competition model is complicated, we discuss the approach we used to simulate competitive effects before we discuss analysis of real data. This discussion will serve to highlight intricacies of the statistical model that needs to be fit. We conducted a simulation study to test the ability of the GS-competition Model 3 to (a) reduce error, (b) correctly partition phenotypic variance, and (c) improve accuracy of the estimation of genetic effects of measured traits. A dataset containing 829 genotypes, including 11 check genotypes and their field coordinate information, was used to start the simulations. A genomic relationship matrix (<strong>K</strong>) was also provided for the dataset in use.</p><p id="p-17">Trait genotypic effects were simulated with zero mean and unit variance. Values for competition genotypic variance (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-16.gif"/></noscript></span></span>), correlation between trait and competitive genotypic effect (Gcor), trait genotypic ratio (<em>gr</em>, see below), correlation between competition error and residual error (Ecor), and fraction of competition error (fraE) were also provided as simulation parameters. A genetic variance-covariance matrix (<strong>G</strong>) was produced using Gcor, and the two genotypic variance values:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-17.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-18.gif"/></noscript></span></span>Calculation of genotypic effects was broken down in the following fashion. First, the Cholesky decomposition of <strong>G</strong> (chol.G) and <strong>K</strong> (chol.K) matrices was performed. Second, two separate random effect values (geno) following normal distribution of mean 0 and SD of 1 were generated for each genotype. Third, genotypic effects (geno.G) were calculated using chol.G matrix as following:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-19.gif"/></noscript></span></span>Fourth, the geno.G values for direct and competitive genotypic effect were multiplied by the chol.K matrix in order to calculate the genotypic effects in the presence of <strong>K</strong> matrix, as follows:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-20.gif"/></noscript></span></span>The genotypic effects matrix can be separated into direct and competitive genotypic effects. The genotypic effects were calculated for low and high direct genotypic ratio scenarios. The direct genotypic ratio (<em>gr</em>) can be formulated as<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-21.gif"/></noscript></span></span>With trait genotypic effect fixed at unit variance, the error variance can be calculated as<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-22.gif"/></noscript></span></span>The fraE value was used to partition the total error variance into that for competition error and residual error:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-23.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-24.gif"/></noscript></span></span>Later, the error effects were calculated in a same way as the geno.G effects, producing separate vector of effects for competition and residual errors.</p><p id="p-18">Afterward, the <strong>Z2</strong> matrix was created using the distance-based functions. Two different competition functions were used to simulate the <strong>Z2</strong> matrixâNN for neighbors along the longer edge of a rectangular plot and SD. NN was chosen as the simplest form of competition and SD was chosen as it turned out to represent most instances in the real data study. The distance between plots was calculated on the assumption that plot dimension was 2Â ÃÂ 1. The <strong>II</strong> matrix was created using NN function for neighbors along the longer edge of the rectangular plot. In order to calculate the phenotypic expression, the <strong>Z1</strong>, <strong>Z2</strong>, and <strong>II</strong> matrices were multiplied by the direct effect, competitive effect, and competition error effect, respectively. The resulting values were added to the residual error values to calculate the phenotype.</p><p id="p-19">The variance of the competitive ability effect was considered to vary from 0.4 to 0.1 when direct genotypic ratio values were high (0.7), and to vary from 1 to 0.1 when the ratio values were low (0.3). Two scenarios of correlation between direct and competitive ability genotypic effect were considered: (i) no correlation and (ii) a correlation of 0.4. In scenarios where competitive genotypic effect is very high, the variance due to it contributes mostly to the total genotypic variance. To illustrate this, consider the scenario of unit variance for both direct and competitive genotypic ability. When it comes to phenotypic expression, the competitive influence on the neighboring plots is calculated as indicated by the <strong>Z2</strong> matrix. Because a plot has multiple neighbors, multiple competitive effects may affect it, leading to a high impact of the competitive ability variance compared to the trait genotypic variance, even in the relatively simple NN scenario.</p><p id="p-20">Two scenarios of correlation between competition and residual error effects were accounted for: (i) no relation and (ii) a high positive correlation of 0.8. A combination of values ranging from 0.3 to 0.9 corresponding to the fraction of competition to total error variance was used simulate the competition and residual error effects. The combination was used to determine the variance of competition and residual error effects.</p><p id="p-21">Three scenarios for clone replication were considered in the simulation. First, the scenario with minimum replication contained only the checks replicated at least twice with all the test clones represented once. Second, 50% of the test clones were replicated twice in addition to the presence of replicated check clones. A third scenario was considered where all clones were replicated at least twice.</p><p id="p-22">The GS-competition Model 3 with <strong>Z2</strong> matrix modified using NN, FD, and SD competition functions was tested along with the Base model on simulated data. The simulation and analysis was repeated 20 times. The best model was selected as the one with the lowest root mean squared error (RMSE), <em>i.e.</em>, the mean squared deviation between the estimated and the simulated genotypic effect. Accuracy (the correlation between true and estimated genotypic effects) was also used as a criterion for selecting these models.</p></div><div id="sec-5" class="subsection"><h3>Real data studies</h3><p id="p-23">ModelÂ 3 and its two variants (ModelÂ 1 and 2) were used to evaluate potential competition effects in real data. A 10-fold cross validation (CV) repeated five times was used. The best model was chosen as the one with the lowest prediction root mean squared error (pRMSE) between observed (Y) and predicted (Å¶) response values for the test dataset. Models using various incidence matrices to account for competition were compared to the Base model having no competition component, and a model was selected based on its predictability and significance. Details on the CV method are given in the companion paper (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Elias <em>et al.</em> 2018</a>). Relative reduction in pRMSE was calculated as the ratio of difference in model pRMSE to Base pRMSE. Values for prediction correlation (pCOR or accuracy) were also recorded for the best model as the correlation between observed and calculated response values for the test dataset. Relative increase in pCOR was calculated as the ratio of difference in pCOR to 1âBase pCOR. The relative difference values were multiplied by 100 to convert them to percentages.</p><p id="p-24">In our companion paper (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Elias <em>et al.</em> 2018</a>) we used a GS-spatial model (hereafter called ModelÂ 4) to evaluate potential spatial correlation in these field trials:<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-25.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-26.gif"/></noscript></span></span>where <strong>s</strong> is the vector of spatial effect and <strong>S</strong> is the selected spatial correlation structure for the trait. In this paper, the selected GS-competition model from among ModelÂ 1, 2, or 3 for a trait was updated as ModelÂ 5 if a significant spatial component from ModelÂ 4 was found. An illustration of the impact of direct genotypic effect, competitive effect on target plot and residuals can be found in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/933/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of direct, competitive, and spatial effect on shoot weight of Ibadan_2014_PYT as illustrated by (A) original observation, (B) direct effect from Base, (C) direct effect from Model 2, (D) competitive effect from Model 2, (E) spatial effect from Model 5, (F) residual from base, (G) residual from Model 2, and (H) residual from Model 5. The genotype in the target plot present in Range 5 and column 11 (inside the red oval) having weak direct genotypic effect (C) has high shoot weight value (A) due to relatively low NN competitive effect (D). The target plot, however, exhibited a relatively strong competitive influence on its NN neighbors resulting in their lower shoot weight. Residual values are modified in the presence of significant competitive and spatial effects (F&#x2013;H)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310653034" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of direct, competitive, and spatial effect on shoot weight of Ibadan_2014_PYT as illustrated by (A) original observation, (B) direct effect from Base, (C) direct effect from Model 2, (D) competitive effect from Model 2, (E) spatial effect from Model 5, (F) residual from base, (G) residual from Model 2, and (H) residual from Model 5. The genotype in the target plot present in Range 5 and column 11 (inside the red oval) having weak direct genotypic effect (C) has high shoot weight value (A) due to relatively low NN competitive effect (D). The target plot, however, exhibited a relatively strong competitive influence on its NN neighbors resulting in their lower shoot weight. Residual values are modified in the presence of significant competitive and spatial effects (F&#x2013;H).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/933/F2.medium.gif" width="411" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/3/933/F2.medium.gif" width="411" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/933/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/933/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488791" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Comparison of direct, competitive, and spatial effect on shoot weight of Ibadan_2014_PYT as illustrated by (A) original observation, (B) direct effect from Base, (C) direct effect from Model 2, (D) competitive effect from Model 2, (E) spatial effect from Model 5, (F) residual from base, (G) residual from Model 2, and (H) residual from Model 5. The genotype in the target plot present in Range 5 and column 11 (inside the red oval) having weak direct genotypic effect (C) has high shoot weight value (A) due to relatively low NN competitive effect (D). The target plot, however, exhibited a relatively strong competitive influence on its NN neighbors resulting in their lower shoot weight. Residual values are modified in the presence of significant competitive and spatial effects (FâH).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The data were fitted using the âregressâ package in R v. 3.2.5 using restricted maximum likelihood (REML). A Chi-square test with a significance threshold of 2.706 at <em>Î±</em> value of 0.1 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Stram and Lee 1994</a>) was used to test the significance of the additional variance component in the selected model compared to the base. The threshold value was changed with degrees of freedom. For example, on comparing Base with ModelÂ 5 having two extra components, the threshold was taken as 4.605 at <em>Î±</em> value of 0.1. Trait heritability was calculated for the selected model, and compared with Base based on the approach introduced by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cullis <em>et.al</em>. (2006)</a> as follows:<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-27.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-28.gif"/></noscript></span></span> is the mean variance of the difference across all pairs of genotypic BLUPs; <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3/933/embed/mml-math-29.gif"/></noscript></span></span> is the estimated variance of genotypes.</p><p id="p-27">Total phenotypic variance was calculated as the sum of variance values from different components of the model after analysis. The datasets and SNP files used for performing this study can be found in the following link: <a href="ftp://ftp.cassavabase.org/manuscripts/Elias_et_al_2017_datasets/">ftp://ftp.cassavabase.org/manuscripts/Elias_et_al_2017_datasets/</a>.</p></div><div id="sec-6" class="subsection"><h3>Automation of real data analysis, simulation of data, and its analysis</h3><p id="p-28">Functions were written to automate the real and simulated data analysis. For the real data analysis, the minimum requirements for the function are a .csv file having field coordinates (Ranges and Columns), trait(s), and clone names, and a genotypic relationship matrix based on marker or pedigree data. Providing a plot dimension (widthÂ ÃÂ length to calculate distance between ranges and columns) can help to identify the best model. The function can take care of the initial processing of the dataset including removal of missing values for a particular trait, matching the genotypes with those in the relationship matrix, and removal of outlier points whose residuals are &gt;2.5 times the error SD (after testing using the Base model). The output of the algorithm will be saved in the working directory of R. The output will contain the predictions for direct and competition genotypic effect, pRMSE, and predicted correlation (pCOR) values, number of failed models as trapped by the âtryâ function in R during the evaluation of a model, and summaries of the Base and the selected model including the parameter values (if the selected model is different from the Base). For the simulations, a dataset with genotypes and field coordinates must be given along with different parameter values, such as correlation of direct to competition effect, correlation of competition to residual error, genotypic ratio, competition variance, fraction of competition to total error, and plot dimension. The output contains a .csv file of RMSE and correlation between true and estimated genotypic direct and competition effects, error effects, fraction of competition error, genotypic ratio, and number of models failed. The functions are available in the following web link: <a href="ftp://ftp.cassavabase.org/manuscripts/Elias_et_al_2017_competition.zip">ftp://ftp.cassavabase.org/manuscripts/Elias_et_al_2017_competition.zip</a>.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-29">Tables S1âS4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a> contains ANOVA tables from analyzing the simulated data. The legend of Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a> compiles the parameter values used for simulation studies. Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a> provides information on the best model selected after CV in all the trial-trait scenarios. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1">Table S6</a> has information on scenarios where Model 5 fit best. Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a> shows the original observation, direct genotypic BLUP, and competitive effect from ModelÂ 2 or 3 for all the trial-traits mentioned in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> except for that in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Simulation studies</h3><p id="p-30">We showed from our real data studies that estimates of the trait genotypic effect are affected even when the competition variance was small (&lt;10% of the total variance in most of the scenarios). Therefore, we conducted simulation studies to see the impact of instances where competitive variance would be larger (10â100 of the trait genotypic effect.</p><p id="p-31">The model fitting the function with which the data were simulated outperformed other models in all the scenarios (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The accuracy in estimating the trait genotypic effect was increased by a median value of 9% compared to Base while evaluating a NN simulated dataset (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A.1</a>). This accuracy was increased by 26% if the underlying SD competitive effect was correctly identified (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A.2</a>). However, with an increase in competition variance, the accuracy for predicting the trait genotypic effect for all the models diminished with relatively less reduction for the best model (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). The contribution of competitive variance component to the total genotypic variance can be higher than that of the trait genotypic effect with increase in competitive variance. This higher contribution is because of the cumulative competitive influence from neighbors, as accounted for by the <strong>Z2</strong> matrix. In estimating the trait genotypic effect, these neighbor effects are essentially a source of noise that becomes stronger as the competitive variance increases. Conversely, an increase in competitive variance in the simulated data resulted in a corresponding increase in accuracy for predicting the competitive effect (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations (accuracy) between true and estimated genotypic values from simulations. ANOVA tables are given in supplemental section. The Base model only estimates the direct genotypic effect, while _FD, _NN, and _SD models also estimate genotypic and error competition effects analyzed using FD, NN, and SD competition decay functions. The prefixes Dir or Comp indicate that the correlations are for the trait or the competition effect, respectively. Rows A and B show the impact of the genotypic competition variance on accuracy of direct genotypic effect (A) and competition genotypic effect (B) averaged over trait genotypic ratio. Rows C and D show the impact of trait genotypic ratio on trait genotypic accuracy, as affected by analysis model (C) or genotypic competition variance (D). Plots in the first (left hand) and second (right hand) columns show analyses of data simulated with the NN and SD competition decay functions, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310653034" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlations (accuracy) between true and estimated genotypic values from simulations. ANOVA tables are given in supplemental section. The Base model only estimates the direct genotypic effect, while _FD, _NN, and _SD models also estimate genotypic and error competition effects analyzed using FD, NN, and SD competition decay functions. The prefixes Dir or Comp indicate that the correlations are for the trait or the competition effect, respectively. Rows A and B show the impact of the genotypic competition variance on accuracy of direct genotypic effect (A) and competition genotypic effect (B) averaged over trait genotypic ratio. Rows C and D show the impact of trait genotypic ratio on trait genotypic accuracy, as affected by analysis model (C) or genotypic competition variance (D). Plots in the first (left hand) and second (right hand) columns show analyses of data simulated with the NN and SD competition decay functions, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-3.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-3.medium.gif" width="423" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488946" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations (accuracy) between true and estimated genotypic values from simulations. ANOVA tables are given in supplemental section. The Base model only estimates the direct genotypic effect, while _FD, _NN, and _SD models also estimate genotypic and error competition effects analyzed using FD, NN, and SD competition decay functions. The prefixes Dir or Comp indicate that the correlations are for the trait or the competition effect, respectively. Rows A and B show the impact of the genotypic competition variance on accuracy of direct genotypic effect (A) and competition genotypic effect (B) averaged over trait genotypic ratio. Rows C and D show the impact of trait genotypic ratio on trait genotypic accuracy, as affected by analysis model (C) or genotypic competition variance (D). Plots in the first (left hand) and second (right hand) columns show analyses of data simulated with the NN and SD competition decay functions, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310653034" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlations (accuracy) between true and estimated genotypic values from simulations. ANOVA tables are given in supplemental section. The Base model only estimates the direct genotypic effect, while _FD, _NN, and _SD models also estimate genotypic and error competition effects analyzed using FD, NN, and SD competition decay functions. The prefixes Dir or Comp indicate that the correlations are for the trait or the competition effect, respectively. Rows A and B show the impact of the genotypic competition variance on accuracy of direct genotypic effect (A) and competition genotypic effect (B) averaged over trait genotypic ratio. Rows C and D show the impact of trait genotypic ratio on trait genotypic accuracy, as affected by analysis model (C) or genotypic competition variance (D). Plots in the first (left hand) and second (right hand) columns show analyses of data simulated with the NN and SD competition decay functions, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-4.medium.gif" width="425" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-4.medium.gif" width="425" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/933/F3/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488946" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Correlations (accuracy) between true and estimated genotypic values from simulations. ANOVA tables are given in supplemental section. The Base model only estimates the direct genotypic effect, while _FD, _NN, and _SD models also estimate genotypic and error competition effects analyzed using FD, NN, and SD competition decay functions. The prefixes Dir or Comp indicate that the correlations are for the trait or the competition effect, respectively. Rows A and B show the impact of the genotypic competition variance on accuracy of direct genotypic effect (A) and competition genotypic effect (B) averaged over trait genotypic ratio. Rows C and D show the impact of trait genotypic ratio on trait genotypic accuracy, as affected by analysis model (C) or genotypic competition variance (D). Plots in the first (left hand) and second (right hand) columns show analyses of data simulated with the NN and SD competition decay functions, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">When data were simulated with NN, modeling with the SD function exhibited the lowest accuracy (excluding Base) and vice versa (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3, A and B</a>). The FD function performed similarly to the NN function, while the SD function remained distinct from the rest of the models (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3, A and B</a>). The function NN assumes that competitive influence can only occur between adjacent neighbors, and, thus, that it decays rapidly, like the FD function (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). This property made the NN and FD functions behave similarly. The robustness of FD can be justified as both FD and SD functions accommodated competition beyond the nearest neighbor. The competitive influence between adjacent plots was same for SD and FD functions. However, in SD, the influence beyond the adjacent neighbors decreased slowly, and never diminished to &lt;60%. This property made the SD function distinct (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a> column 2), and poorly approximated by other functions. From our real data studies, we found that non-NN functions explained the competition variance in most of instances, and that the SD function was slightly more frequent.</p><p id="p-34">The accuracy of direct and competition genotypic effect for ModelÂ 3 can be increased with increase in number of replications for genotypes (Tables S1âS4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>). Increase in genotypic ratio was advantageous to ModelÂ 3, and clear distinction was observed while analyzing data where true competition followed a SD pattern (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3C</a>). Influence of increase in competition variance while simulating the data with increasing genotypic ratio can be visualized in <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3D</a>. A higher value for accuracy was observed in all the instances where competition variance is low. Accuracy increased with increase in genotypic ratio where the increase was more profound for low variance in data simulated with NN. In the case of data simulated with SD, accuracy decreased with increase in genotypic ratio for high competition variance, and increased for low competition variance.</p></div><div id="sec-10" class="subsection"><h3>Real data studies</h3><p id="p-35">We tested for genotypic competition in nine trials and four traits. In 12 of the 36 combinations a significant competition effect was observed (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). A significant genotypic competition effect was observed for one or more of the traits tested in all the trials except for C2 trials in Ikenne and Mokwa (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The competition variance in two out of 12 cases was better explained as competition error. On trials with large numbers of genotypes (C1 trials from all locations with &gt;600 genotypes), more than one trait showed a significant competition effect.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488922/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488922/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of results from prediction analysis using real datasets comparing base to GS-competition model for all the traits under study where significant competition effect at <em>Î±</em> = 0.1 was observed, out of 36 trial/trait combinations</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">We observed a reduction in pRMSE as high as 4.5%, and an increase in pCOR as high as 25% when significant competition was explained by ModelÂ 1 or 2 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The results also showed that the interplot competitive effect was not limited to the NN. The NN function best explained competition in two cases while a non-NN function did better in the remaining eight cases (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>). Usage of distance based non-NN function also helped to identify the distance at which the competition stopped or diminished markedly (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>, and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>).</p><p id="p-38">Results from ModelÂ 2 (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a> and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>) also showed that accounting for interplot competition modified the trait genotypic variance and reduced the error variance. An exceptional scenario was found in Ikenne_2014_C1 for DM, which can be an artifact of the competition function. In this case, the residual variance was found to be negligible, and all the variance in the phenotype was explained by direct or competition genotypic effect, even though the competition explained only 2.6% of total variance. The variance explained by interplot competition component was &lt;10% in all of the scenarios.</p><p id="p-39">Heritability of the traits ranged from 0.27 to 0.84 (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). Slight or no modifications in heritability was observed when competitive ability was accounted for in the model. An exception was the 13% decrease in heritability observed in the PYT trial for SHTWT.</p><p id="p-40">The trait genotypic effect was moderate to strongly positively correlated to observed phenotypic value, while the competition effect was weakly correlated to it. The competition effect was also weakly correlated to the trait and residual effects. The exception to this behavior was HI and DM from 2014_C1 trials conducted in Ibadan and Ikenne, exhibiting a moderately negative correlation between competition and residual effects (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>). Weak correlation between genotypic trait and competition variance indicated that not all the highly productive genotypes were aggressive, and not all the high aggressive genotypes were highly productive.</p><p id="p-41">Strong positive correlation (0.7) between above- and below-ground competition effect in Ibadan_2013_C1 indicated that the competitiveness was similar for the whole plant in these genotypes. In Ikenne_2013_C1 moderate positive correlation (0.45) between above- and below-ground impact of competition for the genotypes was observed. The HI showed most instances of a significant competition effect. Significant competition in HI indicated that cassava could change its biomass allocation strategy when influenced by a neighboring competitive genotype. The competition effect, which explained only a very small percentage of variance in phenotype, was, however, weakly correlated with the trait genotypic effect in all the traits. However, all the trials with significant competitive effects exhibited it for below ground biomass where the harvest product of interest was located.</p><p id="p-42">From the visual assessment (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>), no pattern of border effect with similar colored plots around the edge of the field was evident. This observation was consistent for observed phenotypic values, and for both of the genotypic effects. The genotypic effects were distributed randomly and among the family of genotypes.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/933/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Original observation (column 1), direct genotypic BLUP (column 2), and competition BLUP OR competition error BLUP* (column 3) from model for SHTWT illustrated from (1) IBA_2013_C1 and (2) *IBA_2014_PYT. Visualization of values from all other trial-traits that exhibited significant competitive effect can be found in Figure S1 in File S1. Missing values were linearly interpolated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310653034" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Original observation (column 1), direct genotypic BLUP (column 2), and competition BLUP OR competition error BLUP* (column 3) from model for SHTWT illustrated from (1) IBA_2013_C1 and (2) *IBA_2014_PYT. Visualization of values from all other trial-traits that exhibited significant competitive effect can be found in Figure S1 in File S1. Missing values were linearly interpolated.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/933/F4.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/3/933/F4.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/933/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/933/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488814" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Original observation (column 1), direct genotypic BLUP (column 2), and competition BLUP OR competition error BLUP* (column 3) from model for SHTWT illustrated from (1) IBA_2013_C1 and (2) *IBA_2014_PYT. Visualization of values from all other trial-traits that exhibited significant competitive effect can be found in Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>. Missing values were linearly interpolated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">In Mokwa_2014_C1 FYLD (Figure S1C.3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1/FileS1">File S1</a>) and Ibadan_2014_PYT SHTWT (Figure 4.2), the variance due to competition was explained by the competition error. In Mokwa plots with low competitive ability placed together across the Ranges and Columns could be contributing to the low FYLD values. Some evidence of spatial correlation can also be found for this particular trait (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Elias <em>et al.</em> 2018</a>). In Ibadan, low trait genotypic effect paired with low competitive ability might have contributed to low SHTWT. Strong evidence for the coexistence of interplot competition and spatial correlation was observed for this particular trait (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1">Table S6</a>).</p><p id="p-45">We analyzed these data for the same traits to evaluate the presence of spatial correlation (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Elias <em>et al.</em> 2018</a>), and we used that information to build ModelÂ 5. There were four trials that exhibited significant spatial and competition effect with a significant improvement of the spatial model over the Base or competition model (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1">Table S6</a>). All the trial/traits except in Ibadan_2014_C1 HI, interclonal competition and spatial correlation along with trait genotypic variance explained significant variance in the phenotype. In Ibadan_2014_C1 HI, accounting for competition variance along with trait genotypic variance was found to be better in explaining the variation present in the phenotype.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-46">This study applies a GS model that accounts for variance among genotypes for competitive ability for the first time. These analyses use non-iid relationship matricesâan additive genomic relationship matrix based on SNP markers for trait and competition genotypic effects. Using this approach, we found that the predictive ability of the GS model increased significantly even in the absence of replication for test genotypes. Our simulation studies indicated that the predictive ability of GS models for competitive as well as trait genotypic ability can increase with an increase in number of replications.</p><p id="p-47">Another innovative element in this study is considering the competition of a genotype beyond its nearest neighbor. Competitive ability is the genetically determined ability of an individual to influence its neighbors. This may be unrelated to productivity or to genotypic effect for other traits. Cassavaâs rapidly growing and highly proliferating adventitious and lateral roots can penetrate deep and extensively in search of nutrients and water (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cock 1985</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Izumi <em>et al.</em> 1999</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pardales and Yamauchi 2003</a>). This growth can influence the neighbors beyond the nearest ones, as in the case of a highly competitive clone whose roots can encroach into the root zone of a noncompeting neighbor. The extensive nature of cassava roots could also be explain why no distinct pattern of border effect was evident from our field studies. Competition of a clone beyond the nearest neighbor could also result from a diminishing domino effect, where neighbors beyond the nearest ones are indirectly influenced: a competitive clone might cause its neighbors to extract resources away from it, thereby leading them to encroach more on their neighbors. The proposed GS competition models can identify significant competition, and estimate its variance even when randomization is restricted. Restricted randomization was applied in the field study to simplify evaluation of clones within families by keeping clones from the same family together. This type of design is also expected to decrease the bias due to competition (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">David <em>et al.</em> 1996</a>). With the information from our analysis this design can be modified by not placing genotypes with marked differences in competitive ability near each other.</p><p id="p-48">A third innovative application of our model is the consideration of plot orientation and dimension. In most of our field studies, plants were arranged in plots with single rows where the longer edge of the plot was shared among ranges. The effect of competition is expected to be higher on the neighbors along the longer edge (along the Range) compared to that on the shorter edge (along the Column). This difference was accounted for in the distance-dependent competition functions. However, we have no information on the intraplot competition in multiple row plots of the same clone.</p><p id="p-49">All plants in a single row plot might be able to use the water and nutrients provided in the interplot space fairly equally. Moreover, cassava being a clonally propagated plant, the portion of stem used as planting material could also determine the amount of interplot competition. A plot planted with vigorous stem cuttings might capture more resources than the same clone planted with less vigorous cuttings. To account for this source of error, we included a competition error termâa fourth innovation in the GS models. Understanding the competition error could improve the choice of planting material to mitigate errors in performance due to propagule vigor.</p><p id="p-50">The genotypes selected based on trait and competitive genetic effects can be used for the next cycle of breeding or for commercialization. When spatial correlation is present, it should be adjusted using the proposed model before selecting the elite genotypes. The trait genotypic effect should be the first component to consider while selecting, as performance was moderate to strongly positively correlated to it. Competition variance for genotypes were &lt;10% of their trait genotypic variance in most the scenarios in real data studies. During such instances, genotypes with high trait genotypic effect (after adjusting for the competition effect) can be selected as elite. Therefore, a wise selection for yield increase would be that with high direct and moderate competitive ability. If highly competing plants are selected, they may decrease whole plot performance (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Donald 1968</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Kempton 1982</a>). Selecting less competitive genotypes in order to increase the yield per unit area is effective if the trait and competitive effects of genotypes are negatively related (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Cannell 1979</a>). In our real data studies, we observed that the competition effect is weakly or negatively correlated to the trait genotypic effect. <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Muir (2005)</a> suggested a linear index for selecting individuals when trait and competition effects are present as a weighted sum of these two effects. The index showed that, even in scenarios where the competition effect was small, its contribution could be higher than the trait effect in large groups of individuals.</p><div id="sec-12" class="subsection"><h3>Conclusion</h3><p id="p-51">Through real data studies, we showed that a small but significant increase in predictive ability was attained when interclonal competition was taken into account in a GS model. This predictive ability was markedly increased when accounting for the competition error if present. Results from simulation studies indicated that noticeable increases of accuracy in predicting the breeding value could be attained. These increases occurred especially when competitive influence reached beyond the adjacent neighbors and was accounted for in the GS model. Knowing the competitive ability of a genotype can help in selecting genotypes for breeding advancement. This knowledge can also be useful in modifying the spatial arrangement when genotypes with marked differences in competitive ability need to evaluate together in clonal evaluation trials. This knowledge can also facilitate adjusting the timing and quantity of irrigation and nutrient application, and application of management practices such as pruning and trimming.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">Thanks to A.I. Smith and technical teams at the International Institute of Tropical Agriculture (IITA), and to A. Agbona for data curation. We acknowledge the Bill and Melinda Gates Foundation and UKaid (Grant 1048542; <a href="http://www.gatesfoundation.org">http://www.gatesfoundation.org</a>) and support from the Consultative Group on International Agricultural Research (CGIAR) Research Program on Roots, Tubers and Bananas (<a href="http://www.rtb.cgiar.org">http://www.rtb.cgiar.org</a>).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-53">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300354/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-54"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-10-12"><span class="received-label">Received </span>October 12, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-01-09"><span class="accepted-label">Accepted </span>January 9, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2018 Elias <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.1" data-doi="10.2307/2531047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besag</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kempton</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span>â<span class="cit-article-title">Statistical analysis of field experiments using neighbouring plots.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D42%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2531047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2531047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-confproc" id="cit-8.3.933.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cannell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span>âImproving per hectare forest productivity, pp. <span class="cit-fpage">120</span>â<span class="cit-lpage">148</span> in <em>Proceedings of the 5th North American Forest Biology Workshop</em>. <span class="cit-publ-name">University of Florida</span>, <span class="cit-publ-loc">Gainesville, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cappa</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantet</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Direct and competition additive effects in tree breeding: Bayesian estimation from an individual tree mixed model.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">45</span>â<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D57%26rft.spage%253D45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cappa</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MuÃ±oz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantet</span>  <span class="cit-name-given-names">R. J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">A novel individual-tree mixed model to account for competition and environmental heterogeneity: a Bayesian approach.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">11</span>: 120.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cappa</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoehr</span>  <span class="cit-name-given-names">M. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanchuk</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Identification and joint modeling of competition effects and environmental heterogeneity in three Douglas-fir (<em>Pseudotsuga menziesii</em> var. <em>menziesii</em>) trials.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">12</span>: 102. </cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-book" id="cit-8.3.933.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cock</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-source">Cassava: New Potential for a Neglected Crop</span>. <span class="cit-publ-name">Westview Press</span>, <span class="cit-publ-loc">Boulder, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Connolly</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currie</span>  <span class="cit-name-given-names">I. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNicol</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Inter-plot competition in yield trials of potatoes (<em>Solanum tuberosum</em> L.) with single-drill plots.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Biol.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">367</span>â<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BBiol.%26rft.volume%253D123%26rft.spage%253D367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.8" data-doi="10.1198/108571106X154443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombes</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">On the design of early generation variety trials with correlated data.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">381</span>â<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D11%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1198%252F108571106X154443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/108571106X154443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000242477500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kempton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevison</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Designs for controlling interplot competition in variety trials.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D127%26rft.spage%253D285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Joint modelling of competition and spatial variability in forest field trials.</span> <abbr class="cit-jnl-abbrev">Rev. Mat. Estat.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">7</span>â<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BMat.%2BEstat.%26rft.volume%253D23%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.11" data-doi="10.1007/BF00056241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donald</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span>â<span class="cit-article-title">The breeding of crop ideotypes.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">385</span>â<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D17%26rft.spage%253D385%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00056241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00056241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1968C714500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Draper</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span>â<span class="cit-article-title">Incorporating overlap effects from neighbouring units into response surface models.</span> <abbr class="cit-jnl-abbrev">Appl. Stat.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BStat.%26rft.volume%253D29%26rft.spage%253D128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durban</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currie</span>  <span class="cit-name-given-names">I. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kempton</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Adjusting for fertility and competition in variety trials.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">129</span>â<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D136%26rft.spage%253D129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.14" data-doi="10.1534/g3.117.300323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elias</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulakow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Improving genomic prediction in cassava field experiments using spatial analysis.</span> <abbr class="cit-jnl-abbrev">G3</abbr> (Bethesda) <span class="cit-vol">8</span>: <span class="cit-fpage">53</span>â<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300323%26rft_id%253Dinfo%253Apmid%252F29109156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjgvMS81MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-confproc" id="cit-8.3.933.15"><div class="cit-metadata unstructured">FAO, 2004âThe global cassava development strategy and implementation plan, volume 1, p. 70 in <em>Proceedings of the Validation Forum on the Global Cassava Development Strategy</em>. Reprint from 2001, Food and Agriculture Organization, Rome.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.16" data-doi="10.2307/1400446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Accounting for natural and extraneous variation in the analysis of field experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D2%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1400446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1400446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.17" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Invited review: genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>â<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanson</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span>â<span class="cit-article-title">Competition studies in soybeans.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">117</span>â<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHinson%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D117%26rft.epage%253D123%26rft.atitle%253DCompetition%2BStudies%2Bin%2BSoybeans%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Predicting additive and non-additive genetic effects from trials where traits are affected by interplot competition.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">53</span>â<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D18%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Izumi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuliadi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunyoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iijima</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Root system development including root branching in cuttings of cassava with reference to shoot growth and tuber bulking.</span> <abbr class="cit-jnl-abbrev">Plant Prod. Sci.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">267</span>â<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BProd.%2BSci.%26rft.volume%253D2%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.21" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>â<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempton</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span>â<span class="cit-article-title">Adjustment for competition between varieties in plant breeding trials.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D98%26rft.spage%253D599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.23" data-doi="10.1017/S0021859600047249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lockwood</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">Inter-plot competition in variety trials of field beans (<em>Vicia faba</em> L.).</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D103%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859600047249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859600047249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.24" data-doi="10.2135/cropsci2012.11.0653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melaku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakare</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Relatedness and genotype Ã environment interaction affect prediction accuracies in genomic selection: a study in cassava.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D1312%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.11.0653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.11.0653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.25" data-doi="10.1139/x94-129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magnussen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">A method to adjust simultaneously for spatial microsite and competition effects.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">985</span>â<span class="cit-lpage">995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D24%26rft.spage%253D985%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx94-129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x94-129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-book" id="cit-8.3.933.26"><div class="cit-metadata unstructured">Matassa, V., 2003âOptimisation of experimental design and analysis for sugarcane variety trials. Ph.D. Thesis, School of Land and Food Sciences, The University of Queensland, St. Lucia, QLD, Australia.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.27" data-doi="10.1534/genetics.104.035956"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Incorporation of competitive effects in forest tree or animal breeding programs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1247</span>â<span class="cit-lpage">1259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035956%26rft_id%253Dinfo%253Apmid%252F15911590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzEyNDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pardales</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamauchi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Regulation of root development in sweetpotato and cassava by soil moisture during their establishment period.</span> <abbr class="cit-jnl-abbrev">Plant Soil</abbr> <span class="cit-vol">255</span>: <span class="cit-fpage">201</span>â<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSoil%26rft.volume%253D255%26rft.spage%253D201%26rft.atitle%253DREGULATION%2BOF%2BROOT%2BDEVELOPMENT%2BIN%2BSWEETPOTATO%2BAND%2BCASSAVA%2BBY%2BSOIL%2BMOISTURE%2BDURING%2BTHEIR%2BESTABLISHMENT%2BPERIOD%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.29" data-doi="10.1093/biomet/44.1-2.141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearce</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span>â<span class="cit-article-title">Experimenting with organisms as blocks.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">141</span>â<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPEARCE%26rft.auinit1%253DS.%2BC.%26rft.volume%253D44%26rft.issue%253D1-2%26rft.spage%253D141%26rft.epage%253D149%26rft.atitle%253DEXPERIMENTING%2BWITH%2BORGANISMS%2BAS%2BBLOCKS%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F44.1-2.141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/44.1-2.141&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1957XF49000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebetzke</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">R. T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Herwaarden</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Plot size matters: interference from intergenotypic competition in plant phenotyping studies.</span> <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D41%26rft.spage%253D107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.31" data-doi="10.2307/2533455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stram</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Variance components testing in the longitudinal mixed effects model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1171</span>â<span class="cit-lpage">1177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DStram%26rft.auinit1%253DD.%2BO.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D1171%26rft.epage%253D1177%26rft.atitle%253DVariance%2Bcomponents%2Btesting%2Bin%2Bthe%2Blongitudinal%2Bmixed%2Beffects%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533455%26rft_id%253Dinfo%253Apmid%252F7786999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7786999&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994QP05000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.3.933.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stringer</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Joint modeling of spatial variability and within-row interplot competition to increase the efficiency of plant improvement.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">269</span>â<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D16%26rft.spage%253D269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/3/933.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/3/923" title="Identification and Validation of a New Source of Low Grain Cadmium Accumulation in Durum Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/3/945" title="Genetic Diversity, Molecular Phylogeny, and Selection Evidence of Jinchuan Yak Revealed by Whole-Genome Resequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/3"><span>Volume 8 Issue 3, March 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/3.cover-source.jpg?itok=o8h2cMy2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/933" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AWi3mTPxNX9IQGhX8IJGaT-qy4JI4hL6VX7BqY1ALt0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/3/933" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/3/933" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/3/933" class="active" data-icon-position="" data-hide-link-title="0">Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488767" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-aW7Q-cDCK4KIS6zI4HZ0B-p1tLqJxFMmzN4teVi2Cs8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/3/933.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/3/933" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-mV198Q0wN8rkeeQHXSBRXHxOqQpbTr2xjnl5S6wEJHQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/3/933" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/3/933.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488767" id="node488767--2" data-pisa="ggg;8/3/933" data-pisa-master="ggg;g3.117.300354" data-apath="/ggg/8/3/933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3360-6979" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ani A. Elias</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ismail Rabbi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">933-944;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300354 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488767--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ani A. Elias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ani%2BA.%2BElias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Elias%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAni%2BA.%2BElias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3360-6979" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ani A. Elias</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anianna01@gmail.com" class="" data-icon-position="" data-hide-link-title="0">anianna01@gmail.com</a>
<a href="mailto:jeanluc.work@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jeanluc.work@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ismail Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Institute of Tropical Agriculture, Ibadan 200001, Nigeria and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsmail%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ismail Rabbi</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Institute of Tropical Agriculture, Ibadan 200001, Nigeria and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agriculture-Agricultural Research Station (USDA-ARS), Robert W. Holley Center for Agriculture and Health, Ithaca, New York 14853-2901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anianna01@gmail.com" class="" data-icon-position="" data-hide-link-title="0">anianna01@gmail.com</a>
<a href="mailto:jeanluc.work@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jeanluc.work@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/3/933" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/3/933&amp;title=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/3/933&amp;title=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/3/933&amp;title=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/3/933&amp;text=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/3/933&amp;title=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/3/933&amp;t=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/3/933&amp;title=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/3/933&amp;title=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488767/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488767" id="node488767--3" data-pisa="ggg;8/3/933" data-pisa-master="ggg;g3.117.300354" data-apath="/ggg/8/3/933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improving Genomic Prediction in Cassava Field Experiments by Accounting for Interplot Competition</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3360-6979" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ani A. Elias</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ismail Rabbi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Kulakow</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">933-944;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300354 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488767--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ani A. Elias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ani%2BA.%2BElias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Elias%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAni%2BA.%2BElias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3360-6979" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ani A. Elias</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anianna01@gmail.com" class="" data-icon-position="" data-hide-link-title="0">anianna01@gmail.com</a>
<a href="mailto:jeanluc.work@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jeanluc.work@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ismail Rabbi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Institute of Tropical Agriculture, Ibadan 200001, Nigeria and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ismail%2BRabbi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rabbi%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsmail%2BRabbi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9966-2941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ismail Rabbi</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Kulakow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Institute of Tropical Agriculture, Ibadan 200001, Nigeria and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BKulakow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulakow%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BKulakow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7574-2645" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Kulakow</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agriculture-Agricultural Research Station (USDA-ARS), Robert W. Holley Center for Agriculture and Health, Ithaca, New York 14853-2901</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anianna01@gmail.com" class="" data-icon-position="" data-hide-link-title="0">anianna01@gmail.com</a>
<a href="mailto:jeanluc.work@gmail.com" class="" data-icon-position="" data-hide-link-title="0">jeanluc.work@gmail.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=933&amp;rft.epage=944&amp;rft.atitle=Improving%20Genomic%20Prediction%20in%20Cassava%20Field%20Experiments%20by%20Accounting%20for%20Interplot%20Competition&amp;rft.volume=8&amp;rft.issue=3&amp;rft.date=2018-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Elias%2C+Ani+A.&amp;rft.au=Rabbi%2C+Ismail&amp;rft.au=Kulakow%2C+Peter&amp;rft.au=Jannink%2C+Jean-Luc"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488767/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488767/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488767/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488767/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488767/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488767/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488767/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488767/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488767/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488767/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488767/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488767/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype Ã Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/3/933" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/3/933.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/3/933.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/3/933.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/3/933.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
