<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/3/999" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/3/999.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/3/999.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/3/999.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/3/999.atom" />
<meta name="HW.pisa" content="ggg;8/3/999" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Tuning CRISPR-Cas9 Gene Drives in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.117.300557" />
<meta name="DC.Date" content="2018-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Roggenkamp et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.118.200668" />
<meta name="DC.Description" content="Control of biological populations is an ongoing challenge in many fields, including agriculture, biodiversity, ecological preservation, pest control, and the spread of disease. In some cases, such as insects that harbor human pathogens ( e.g. , malaria), elimination or reduction of a small number of species would have a dramatic impact across the globe. Given the recent discovery and development of the CRISPR-Cas9 gene editing technology, a unique arrangement of this system, a nuclease-based “gene drive,” allows for the super-Mendelian spread and forced propagation of a genetic element through a population. Recent studies have demonstrated the ability of a gene drive to rapidly spread within and nearly eliminate insect populations in a laboratory setting. While there are still ongoing technical challenges to design of a more optimal gene drive to be used in wild populations, there are still serious ecological and ethical concerns surrounding the nature of this powerful biological agent. Here, we use budding yeast as a safe and fully contained model system to explore mechanisms that might allow for programmed regulation of gene drive activity. We describe four conserved features of all CRISPR-based drives and demonstrate the ability of each drive component—Cas9 protein level, sgRNA identity, Cas9 nucleocytoplasmic shuttling, and novel Cas9-Cas9 tandem fusions—to modulate drive activity within a population." />
<meta name="DC.Contributor" content="Emily Roggenkamp" />
<meta name="DC.Contributor" content="Rachael M. Giersch" />
<meta name="DC.Contributor" content="Madison N. Schrock" />
<meta name="DC.Contributor" content="Emily Turnquist" />
<meta name="DC.Contributor" content="Megan Halloran" />
<meta name="DC.Contributor" content="Gregory C. Finnigan" />
<meta name="article:published_time" content="2018-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Tuning CRISPR-Cas9 Gene Drives in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Control of biological populations is an ongoing challenge in many fields, including agriculture, biodiversity, ecological preservation, pest control, and the spread of disease. In some cases, such as insects that harbor human pathogens (&lt;i&gt;e.g.&lt;/i&gt;, malaria), elimination or reduction of a small number of species would have a dramatic impact across the globe. Given the recent discovery and development of the CRISPR-Cas9 gene editing technology, a unique arrangement of this system, a nuclease-based “gene drive,” allows for the super-Mendelian spread and forced propagation of a genetic element through a population. Recent studies have demonstrated the ability of a gene drive to rapidly spread within and nearly eliminate insect populations in a laboratory setting. While there are still ongoing technical challenges to design of a more optimal gene drive to be used in wild populations, there are still serious ecological and ethical concerns surrounding the nature of this powerful biological agent. Here, we use budding yeast as a safe and fully contained model system to explore mechanisms that might allow for programmed regulation of gene drive activity. We describe four conserved features of all CRISPR-based drives and demonstrate the ability of each drive component—Cas9 protein level, sgRNA identity, Cas9 nucleocytoplasmic shuttling, and novel Cas9-Cas9 tandem fusions—to modulate drive activity within a population.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/03/01" />
<meta name="citation_mjid" content="ggg;8/3/999" />
<meta name="citation_id" content="8/3/999" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/3/999" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/3/999.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/3/999.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/3/999.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300557" />
<meta name="citation_pmid" content="29348295" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="999" />
<meta name="citation_lastpage" content="1018" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Emily Roggenkamp" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Rachael M. Giersch" />
<meta name="citation_author_institution" content="Department of Biology, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Madison N. Schrock" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author_institution" content="Department of Biology, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Emily Turnquist" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Megan Halloran" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author" content="Gregory C. Finnigan" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author_email" content="gfinnigan@ksu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5710-173X" />
<meta name="citation_reference" content="citation_journal_title=JAMA;citation_author=J. Abbasi;citation_title=National academies hit the brakes on gene drive-modified organisms.;citation_pages=482-483;citation_volume=316;citation_year=2016;citation_pmid=27410484;citation_doi=10.1001/jama.2016.8830" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Z. Adelman;citation_author=O. Akbari;citation_author=J. Bauer;citation_author=E. Bier;citation_author=C. Bloss;citation_title=Rules of the road for insect gene drive research and testing.;citation_pages=716-718;citation_volume=35;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=O. S. Akbari;citation_author=H. J. Bellen;citation_author=E. Bier;citation_author=S. L. Bullock;citation_author=A. Burt;citation_title=BIOSAFETY. Safeguarding gene drive experiments in the laboratory.;citation_volume=349;citation_year=2015;citation_pmid=26229113;citation_doi=10.1126/science.aac7932" />
<meta name="citation_reference" content="citation_journal_title=J. Biotechnol.;citation_author=E. M. Anderson;citation_author=A. Haupt;citation_author=J. A. Schiel;citation_author=E. Chou;citation_author=H. B. Machado;citation_title=Systematic analysis of CRISPR-Cas9 mismatch tolerance reveals low levels of off-target activity.;citation_pages=56-65;citation_volume=211;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Prog. Mol. Biol. Transl. Sci.;citation_author=N. J. Baltes;citation_author=J. Gil-Humanes;citation_author=D. F. Voytas;citation_title=Genome engineering and agriculture: opportunities and challenges.;citation_pages=1-26;citation_volume=149;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=R. Barrangou;citation_author=J. A. Doudna;citation_title=Applications of CRISPR technologies in research and beyond.;citation_pages=933-941;citation_volume=34;citation_year=2016;citation_doi=10.1038/nbt.3659" />
<meta name="citation_reference" content="citation_journal_title=Traffic;citation_author=N. C. Bauer;citation_author=P. W. Doetsch;citation_author=A. H. Corbett;citation_title=Mechanisms regulating protein localization.;citation_pages=1039-1061;citation_volume=16;citation_year=2015;citation_pmid=26172624;citation_doi=10.1111/tra.12310" />
<meta name="citation_reference" content="citation_journal_title=Malar. J.;citation_author=A. Beaghton;citation_author=P. J. Beaghton;citation_author=A. Burt;citation_title=Vector control with driving Y chromosomes: modelling the evolution of resistance.;citation_pages=286;citation_volume=16;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=J. L. Bennetzen;citation_author=B. D. Hall;citation_title=The primary structure of the Saccharomyces cerevisiae gene for alcohol dehydrogenase;citation_pages=3018-3025;citation_volume=257;citation_year=1982;citation_issue=6;citation_pmid=6277922" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=M. F. Bolukbasi;citation_author=A. Gupta;citation_author=S. Oikemus;citation_author=A. G. Derr;citation_author=M. Garber;citation_title=DNA-binding-domain fusions enhance the targeting range and precision of Cas9.;citation_pages=1150-1156;citation_volume=12;citation_year=2015;citation_pmid=26480473;citation_doi=10.1038/nmeth.3624" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. B. Brachmann;citation_author=A. Davies;citation_author=G. J. Cost;citation_author=E. Caputo;citation_author=J. Li;citation_title=Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.;citation_pages=115-132;citation_volume=14;citation_year=1998;citation_issue=2;citation_pmid=9483801;citation_doi=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Protocols;citation_journal_abbrev=Cold Spring Harb Protoc;citation_author=A. E. Briner;citation_author=R. Barrangou;citation_title=Guide RNAs: A Glimpse at the Sequences that Drive CRISPR-Cas Systems;citation_pages=pdb.top090902-pdb.top090902;citation_volume=2016;citation_year=2016;citation_issue=7;citation_pmid=27371605;citation_doi=10.1101/pdb.top090902" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=A. E. Briner;citation_author=P. D. Donohoue;citation_author=A. A. Gomaa;citation_author=K. Selle;citation_author=E. M. Slorach;citation_title=Guide RNA functional modules direct Cas9 activity and orthogonality.;citation_pages=333-339;citation_volume=56;citation_year=2014;citation_pmid=25373540;citation_doi=10.1016/j.molcel.2014.09.019" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=A. Burt;citation_title=Site-specific selfish genes as tools for the control and genetic engineering of natural populations;citation_pages=921-928;citation_volume=270;citation_year=2003;citation_issue=1518;citation_pmid=12803906;citation_doi=10.1098/rspb.2002.2319" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Cao;citation_author=L. Wu;citation_author=S. M. Zhang;citation_author=M. Lu;citation_author=W. K. Cheung;citation_title=An easy and efficient inducible CRISPR/Cas9 platform with improved specificity for multiple gene targeting.;citation_volume=44;citation_year=2016;citation_pmid=27458201;citation_doi=10.1093/nar/gkw660" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. Chaikind;citation_author=J. L. Bessen;citation_author=D. B. Thompson;citation_author=J. H. Hu;citation_author=D. R. Liu;citation_title=A programmable Cas9-serine recombinase fusion protein that operates on DNA sequences in mammalian cells.;citation_volume=44;citation_year=2016;citation_pmid=27515511;citation_doi=10.1093/nar/gkw707" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Champer;citation_author=R. Reeves;citation_author=S. Y. Oh;citation_author=C. Liu;citation_author=J. Liu;citation_title=Novel CRISPR/Cas9 gene drive constructs reveal insights into mechanisms of resistance allele formation and drive efficiency in genetically diverse populations.;citation_pages=e1006796;citation_volume=13;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=T. W. Christianson;citation_author=R. S. Sikorski;citation_author=M. Dante;citation_author=J. H. Shero;citation_author=P. Hieter;citation_title=Multifunctional yeast high-copy-number shuttle vectors.;citation_pages=119-122;citation_volume=110;citation_year=1992;citation_issue=1;citation_pmid=1544568;citation_doi=10.1016/0378-1119(92)90454-W" />
<meta name="citation_reference" content="citation_journal_title=Genesis;citation_author=T. P. Clements;citation_author=B. Tandon;citation_author=H. A. Lintel;citation_author=J. H. McCarty;citation_author=D. S. Wagner;citation_title=RICE CRISPR: rapidly increased cut ends by an exonuclease Cas9 fusion in zebrafish.;citation_pages=e23044;citation_volume=55;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=L. Cong;citation_author=F. A. Ran;citation_author=D. Cox;citation_author=S. Lin;citation_author=R. Barretto;citation_title=Multiplex genome engineering using CRISPR/Cas systems.;citation_volume=339;citation_year=2013;citation_pmid=23287718;citation_doi=10.1126/science.1231143" />
<meta name="citation_reference" content="citation_journal_title=EMBO Rep.;citation_author=V. Courtier-Orgogozo;citation_author=B. Morizot;citation_author=C. Boete;citation_title=Agricultural pest control with CRISPR-based gene drive: time for public debate: should we use gene drive for pest control?;citation_volume=18;citation_year=2017;citation_pmid=28512121;citation_doi=10.15252/embr.201744205" />
<meta name="citation_reference" content="citation_journal_title=EMBO Rep.;citation_author=V. Courtier-Orgogozo;citation_author=B. Morizot;citation_author=C. Boete;citation_title=Using CRISPR-based gene drive for agriculture pest control.;citation_volume=18;citation_year=2017;citation_pmid=28784604;citation_doi=10.15252/embr.201744822" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=Y. Dang;citation_author=G. Jia;citation_author=J. Choi;citation_author=H. Ma;citation_author=E. Anaya;citation_title=Optimizing sgRNA structure to improve CRISPR-Cas9 knockout efficiency.;citation_pages=280;citation_volume=16;citation_year=2015;citation_pmid=26671237;citation_doi=10.1186/s13059-015-0846-3" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. E. DiCarlo;citation_author=J. E. Norville;citation_author=P. Mali;citation_author=X. Rios;citation_author=J. Aach;citation_title=Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems.;citation_volume=41;citation_year=2013;citation_pmid=23460208;citation_doi=10.1093/nar/gkt135" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. E. DiCarlo;citation_author=A. Chavez;citation_author=S. L. Dietz;citation_author=K. M. Esvelt;citation_author=G. M. Church;citation_title=Safeguarding CRISPR-Cas9 gene drives in yeast.;citation_pages=1250-1255;citation_volume=33;citation_year=2015;citation_pmid=26571100;citation_doi=10.1038/nbt.3412" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. G. Doench;citation_author=N. Fusi;citation_author=M. Sullender;citation_author=M. Hegde;citation_author=E. W. Vaimberg;citation_title=Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9.;citation_pages=184-191;citation_volume=34;citation_year=2016;citation_pmid=26780180;citation_doi=10.1038/nbt.3437" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Dong;citation_author=S. Guo;citation_author=Y. Wang;citation_author=S. Zhu;citation_author=. Wang;citation_title=Structural basis of CRISPR-SpyCas9 inhibition by an anti-CRISPR protein.;citation_pages=436-439;citation_volume=546;citation_year=2017;citation_pmid=28448066;citation_doi=10.1038/nature22377" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. A. Doudna;citation_author=E. Charpentier;citation_title=The new frontier of genome engineering with CRISPR-Cas9;citation_pages=1258096-1258096;citation_volume=346;citation_year=2014;citation_issue=6213;citation_pmid=25430774;citation_doi=10.1126/science.1258096" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=L. E. Dow;citation_author=J. Fisher;citation_author=K. P. O’Rourke;citation_author=A. Muley;citation_author=E. R. Kastenhuber;citation_title=Inducible in vivo genome editing with CRISPR-Cas9.;citation_pages=390-394;citation_volume=33;citation_year=2015;citation_pmid=25690852;citation_doi=10.1038/nbt.3155" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=D. W. Drury;citation_author=A. L. Dapper;citation_author=D. J. Siniard;citation_author=G. E. Zentner;citation_author=M. J. Wade;citation_title=CRISPR/Cas9 gene drives in genetically variable and nonrandomly mating wild populations;citation_pages=e1601910-e1601910;citation_volume=3;citation_year=2017;citation_issue=5;citation_doi=10.1126/sciadv.1601910" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. A. Duina;citation_author=M. E. Miller;citation_author=J. B. Keeney;citation_title=Budding Yeast for Budding Geneticists: A Primer on the Saccharomyces cerevisiae Model System;citation_pages=33-48;citation_volume=197;citation_year=2014;citation_issue=1;citation_pmid=24807111;citation_doi=10.1534/genetics.114.163188" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=N. Eckert-Boulet;citation_author=M. L. Pedersen;citation_author=B. O. Krogh;citation_author=M. Lisby;citation_title=Optimization of ordered plasmid assembly by gap repair in Saccharomyces cerevisiae.;citation_pages=323-334;citation_volume=29;citation_year=2012;citation_issue=8;citation_pmid=22806834;citation_doi=10.1002/yea.2912" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Biotechnol.;citation_author=R. Estrela;citation_author=J. H. Cate;citation_title=Energy biotechnology in the CRISPR-Cas9 era.;citation_pages=79-84;citation_volume=38;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=K. M. Esvelt;citation_author=N. J. Gemmell;citation_title=Conservation demands safe gene drive.;citation_pages=e2003850;citation_volume=15;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=K. M. Esvelt;citation_author=P. Mali;citation_author=J. L. Braff;citation_author=M. Moosburner;citation_author=S. J. Yaung;citation_title=Orthogonal Cas9 proteins for RNA-guided gene regulation and editing.;citation_pages=1116-1121;citation_volume=10;citation_year=2013;citation_pmid=24076762;citation_doi=10.1038/nmeth.2681" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Farboud;citation_author=B. J. Meyer;citation_title=Dramatic enhancement of genome editing by CRISPR/Cas9 through improved guide RNA design.;citation_volume=199;citation_year=2015;citation_pmid=25695951;citation_doi=10.1534/genetics.115.175166" />
<meta name="citation_reference" content="citation_journal_title=Bio Protoc.;citation_author=G. C. Finnigan;citation_author=J. Thorner;citation_title=Complex in vivo ligation using homologous recombination and high-efficiency plasmid rescue from Saccharomyces cerevisiae.;citation_pages=1521e;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. C. Finnigan;citation_author=J. Thorner;citation_title=mCAL: a new approach for versatile multiplex action of Cas9 using one sgRNA and loci flanked by a programmed target sequence.;citation_pages=2147-2156;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=G. C. Finnigan;citation_author=S. M. Sterling;citation_author=A. Duvalyan;citation_author=E. N. Liao;citation_author=A. Sargsyan;citation_title=Coordinate action of distinct sequence elements localizes checkpoint kinase Hsl1 to the septin collar at the bud neck in Saccharomyces cerevisiae.;citation_volume=27;citation_year=2016;citation_pmid=27193302;citation_doi=10.1091/mbc.E16-03-0177" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. S. Flick;citation_author=M. Johnston;citation_title=Two systems of glucose repression of the GAL1 promoter in Saccharomyces cerevisiae.;citation_pages=4757-4769;citation_volume=10;citation_year=1990;citation_issue=9;citation_pmid=2201902;citation_doi=10.1128/MCB.10.9.4757" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=N. Freitas;citation_author=C. Cunha;citation_title=Mechanisms and signals for the nuclear import of proteins.;citation_pages=550-557;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=20514217;citation_doi=10.2174/138920209789503941" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. X. Fu;citation_author=R. P. St Onge;citation_author=A. Z. Fire;citation_author=J. D. Smith;citation_title=Distinct patterns of Cas9 mismatch tolerance in vitro and in vivo.;citation_volume=44;citation_year=2016;citation_pmid=27198218;citation_doi=10.1093/nar/gkw417" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Y. Fu;citation_author=J. A. Foden;citation_author=C. Khayter;citation_author=M. L. Maeder;citation_author=D. Reyon;citation_title=High-frequency off-target mutagenesis induced by CRISPR-Cas nucleases in human cells.;citation_pages=822-826;citation_volume=31;citation_year=2013;citation_pmid=23792628;citation_doi=10.1038/nbt.2623" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. M. Gantz;citation_author=N. Jasinskiene;citation_author=O. Tatarenkova;citation_author=A. Fazekas;citation_author=V. M. Macias;citation_title=Highly efficient Cas9-mediated gene drive for population modification of the malaria vector mosquito Anopheles stephensi.;citation_volume=112;citation_year=2015;citation_pmid=26598698;citation_doi=10.1073/pnas.1521077112" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=Y. Gao;citation_author=X. Xiong;citation_author=S. Wong;citation_author=E. J. Charles;citation_author=W. A. Lim;citation_title=Complex transcriptional modulation with orthogonal and inducible dCas9 regulators.;citation_pages=1043-1049;citation_volume=13;citation_year=2016;citation_pmid=27776111" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=L. A. Gilbert;citation_author=M. H. Larson;citation_author=L. Morsut;citation_author=Z. Liu;citation_author=G. A. Brar;citation_title=CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes.;citation_pages=442-451;citation_volume=154;citation_year=2013;citation_pmid=23849981;citation_doi=10.1016/j.cell.2013.06.044" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=H. C. J. Godfray;citation_author=A. North;citation_author=A. Burt;citation_title=How driving endonuclease genes can be used to combat pests and disease vectors.;citation_pages=81;citation_volume=15;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. P. Guilinger;citation_author=D. B. Thompson;citation_author=D. R. Liu;citation_title=Fusion of catalytically inactive Cas9 to FokI nuclease improves the specificity of genome modification.;citation_pages=577-582;citation_volume=32;citation_year=2014;citation_pmid=24770324;citation_doi=10.1038/nbt.2909" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Guo;citation_author=T. Gong;citation_author=X. D. Gao;citation_title=Identification of an amphipathic helix important for the formation of ectopic septin spirals and axial budding in yeast axial landmark protein Bud3p.;citation_pages=e16744-e16744;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21408200;citation_doi=10.1371/journal.pone.0016744" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=A. Hammond;citation_author=R. Galizi;citation_author=K. Kyrou;citation_author=A. Simoni;citation_author=C. Siniscalchi;citation_title=A CRISPR-Cas9 gene drive system targeting female reproduction in the malaria mosquito vector Anopheles gambiae.;citation_pages=78-83;citation_volume=34;citation_year=2016;citation_pmid=26641531;citation_doi=10.1038/nbt.3439" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. M. Hammond;citation_author=K. Kyrou;citation_author=M. Bruttini;citation_author=A. North;citation_author=R. Galizi;citation_title=The creation and selection of mutations resistant to a gene drive over multiple generations in the malaria mosquito.;citation_pages=e1007039;citation_volume=13;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=L. R. Heasley;citation_author=M. A. McMurray;citation_title=Roles of septins in prospore membrane morphogenesis and spore wall assembly in Saccharomyces cerevisiae.;citation_volume=27;citation_year=2016;citation_pmid=26680739;citation_doi=10.1091/mbc.E15-10-0721" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=P. D. Hsu;citation_author=D. A. Scott;citation_author=J. A. Weinstein;citation_author=F. A. Ran;citation_author=S. Konermann;citation_title=DNA targeting specificity of RNA-guided Cas9 nucleases.;citation_pages=827-832;citation_volume=31;citation_year=2013;citation_pmid=23873081;citation_doi=10.1038/nbt.2647" />
<meta name="citation_reference" content="citation_journal_title=Microb. Cell Fact.;citation_author=E. D. Jensen;citation_author=R. Ferreira;citation_author=T. Jakociunas;citation_author=D. Arsovska;citation_author=J. Zhang;citation_title=Transcriptional reprogramming in yeast using dCas9 and combinatorial gRNA strategies.;citation_pages=46;citation_volume=16;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=F. Jiang;citation_author=D. W. Taylor;citation_author=J. S. Chen;citation_author=J. E. Kornfeld;citation_author=K. Zhou;citation_title=Structures of a CRISPR-Cas9 R-loop complex primed for DNA cleavage.;citation_volume=351;citation_year=2016;citation_pmid=26841432;citation_doi=10.1126/science.aad8282" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Jinek;citation_author=K. Chylinski;citation_author=I. Fonfara;citation_author=M. Hauer;citation_author=J. A. Doudna;citation_title=A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.;citation_volume=337;citation_year=2012;citation_pmid=22745249;citation_doi=10.1126/science.1225829" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_journal_abbrev=elife;citation_author=M. Jinek;citation_author=A. East;citation_author=A. Cheng;citation_author=S. Lin;citation_author=E. Ma;citation_title=RNA-programmed genome editing in human cells;citation_pages=e00471-e00471;citation_volume=2;citation_year=2013;citation_issue=0;citation_pmid=23386978;citation_doi=10.7554/eLife.00471" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Jinek;citation_author=F. Jiang;citation_author=D. W. Taylor;citation_author=S. H. Sternberg;citation_author=E. Kaya;citation_title=Structures of Cas9 endonucleases reveal RNA-mediated conformational activation.;citation_volume=343;citation_year=2014;citation_pmid=24505130;citation_doi=10.1126/science.1247997" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Insect Sci.;citation_author=M. Jonsson;citation_author=R. Kaartinen;citation_author=C. S. Straub;citation_title=Relationships between natural enemy diversity and biological control.;citation_pages=1-6;citation_volume=20;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Kalderon;citation_author=B. L. Roberts;citation_author=W. D. Richardson;citation_author=A. E. Smith;citation_title=A short amino acid sequence able to specify nuclear location.;citation_pages=499-509;citation_volume=39;citation_year=1984;citation_issue=3 Pt 2;citation_pmid=6096007;citation_doi=10.1016/0092-8674(84)90457-4" />
<meta name="citation_reference" content="citation_journal_title=ACS Synth. Biol.;citation_author=H. S. Kang;citation_author=Z. Charlop-Powers;citation_author=S. F. Brady;citation_title=Multiplexed CRISPR/Cas9- and TAR-mediated promoter engineering of natural product biosynthetic gene clusters in yeast.;citation_pages=1002-1010;citation_volume=5;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.;citation_author=T. Karvelis;citation_author=G. Gasiunas;citation_author=V. Siksnys;citation_title=Harnessing the natural diversity and in vitro evolution of Cas9 to expand the genome editing toolbox.;citation_pages=88-94;citation_volume=37;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Insects;citation_author=A. Kergunteuil;citation_author=M. Bakhtiari;citation_author=L. Formenti;citation_author=Z. Xiao;citation_author=E. Defossez;citation_title=Biological control beneath the feet: a review of crop protection against insect root herbivores.;citation_pages=70;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Chem. Ecol.;citation_author=Z. Khan;citation_author=C. A. Midega;citation_author=A. Hooper;citation_author=J. Pickett;citation_title=Push-pull: chemical ecology-based integrated pest management technology.;citation_pages=689-697;citation_volume=42;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Y. B. Kim;citation_author=A. C. Komor;citation_author=J. M. Levy;citation_author=M. S. Packer;citation_author=K. T. Zhao;citation_title=Increasing the genome-targeting scope and precision of base editing with engineered Cas9-cytidine deaminase fusions.;citation_pages=371-376;citation_volume=35;citation_year=2017;citation_pmid=28191901;citation_doi=10.1038/nbt.3803" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=H. Koike-Yusa;citation_author=Y. Li;citation_author=E. P. Tan;citation_author=C. Velasco-Herrera Mdel;citation_author=K. Yusa;citation_title=Genome-wide recessive genetic screening in mammalian cells with a lentiviral CRISPR-guide RNA library.;citation_pages=267-273;citation_volume=32;citation_year=2014;citation_pmid=24535568;citation_doi=10.1038/nbt.2800" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.;citation_author=E. V. Koonin;citation_author=K. S. Makarova;citation_author=F. Zhang;citation_title=Diversity, classification and evolution of CRISPR-Cas systems.;citation_pages=67-78;citation_volume=37;citation_year=2017;citation_pmid=28605718;citation_doi=10.1016/j.mib.2017.05.008" />
<meta name="citation_reference" content="citation_journal_title=Traffic (Copenhagen, Denmark);citation_journal_abbrev=Traffic;citation_author=S. Kosugi;citation_author=M. Hasebe;citation_author=M. Tomita;citation_author=H. Yanagawa;citation_title=Nuclear export signal consensus sequences defined using a localization-based yeast selection system.;citation_pages=2053-2062;citation_volume=9;citation_year=2008;citation_issue=12;citation_pmid=18817528;citation_doi=10.1111/j.1600-0854.2008.00825.x" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. Lange;citation_author=R. E. Mills;citation_author=C. J. Lange;citation_author=M. Stewart;citation_author=S. E. Devine;citation_title=Classical nuclear localization signals: definition, function, and interaction with importin alpha.;citation_volume=282;citation_year=2007;citation_pmid=17170104;citation_doi=10.1074/jbc.R600026200" />
<meta name="citation_reference" content="citation_journal_title=J. Biotechnol.;citation_author=L. Lin;citation_author=T. S. Petersen;citation_author=K. T. Jensen;citation_author=L. Bolund;citation_author=R. Kuhn;citation_title=Fusion of SpCas9 to E. coli Rec A protein enhances CRISPR-Cas9 mediated gene knockout in mammalian cells.;citation_pages=42-49;citation_volume=247;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Lin;citation_author=T. J. Cradick;citation_author=M. T. Brown;citation_author=H. Deshmukh;citation_author=P. Ranjan;citation_title=CRISPR/Cas9 systems have off-target activity with insertions or deletions between target DNA and guide RNA sequences.;citation_volume=42;citation_year=2014;citation_pmid=24838573;citation_doi=10.1093/nar/gku402" />
<meta name="citation_reference" content="citation_journal_title=Nat. Chem. Biol.;citation_author=K. I. Liu;citation_author=M. N. Ramli;citation_author=C. W. Woo;citation_author=Y. Wang;citation_author=T. Zhao;citation_title=A chemical-inducible CRISPR-Cas9 system for rapid control of genome editing.;citation_pages=980-987;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. S. Longtine;citation_author=A. McKenzie;citation_author=D. J. Demarini;citation_author=N. G. Shah;citation_author=A. Wach;citation_title=Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae.;citation_pages=953-961;citation_volume=14;citation_year=1998;citation_issue=10;citation_pmid=9717241;citation_doi=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Ma;citation_author=A. Naseri;citation_author=P. Reyes-Gutierrez;citation_author=S. A. Wolfe;citation_author=S. Zhang;citation_title=Multicolor CRISPR labeling of chromosomal loci in human cells.;citation_volume=112;citation_year=2015;citation_pmid=25713381;citation_doi=10.1073/pnas.1420024112" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=H. Ma;citation_author=L. C. Tu;citation_author=A. Naseri;citation_author=M. Huisman;citation_author=S. Zhang;citation_title=CRISPR-Cas9 nuclear dynamics and target recognition in living cells.;citation_volume=214;citation_year=2016;citation_pmid=27551060;citation_doi=10.1083/jcb.201604115" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=H. Ma;citation_author=L. C. Tu;citation_author=A. Naseri;citation_author=M. Huisman;citation_author=S. Zhang;citation_title=Multiplexed labeling of genomic loci with dCas9 and engineered sgRNAs using CRISPRainbow.;citation_pages=528-530;citation_volume=34;citation_year=2016;citation_pmid=27088723;citation_doi=10.1038/nbt.3526" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. M. Marshall;citation_author=A. Buchman;citation_author=C. H. Sanchez;citation_author=O. S. Akbari;citation_title=Overcoming evolved resistance to population-suppressing homing-based gene drives.;citation_pages=3776;citation_volume=7;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=S. Ménoret;citation_author=A. De Cian;citation_author=L. Tesson;citation_author=S. Remy;citation_author=C. Usal;citation_title=Homology-directed repair in rodent zygotes using Cas9 and TALEN engineered proteins.;citation_pages=14410;citation_volume=5;citation_year=2015;citation_pmid=26442875;citation_doi=10.1038/srep14410" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Mohanraju;citation_author=K. S. Makarova;citation_author=B. Zetsche;citation_author=F. Zhang;citation_author=E. V. Koonin;citation_title=Diverse evolutionary roots and mechanistic variations of the CRISPR-Cas systems;citation_pages=aad5147-aad5147;citation_volume=353;citation_year=2016;citation_issue=6299;citation_pmid=27493190;citation_doi=10.1126/science.aad5147" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=M. A. Moreno-Mateos;citation_author=C. E. Vejnar;citation_author=J. D. Beaudoin;citation_author=J. P. Fernandez;citation_author=E. K. Mis;citation_title=CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo.;citation_pages=982-988;citation_volume=12;citation_year=2015;citation_pmid=26322839;citation_doi=10.1038/nmeth.3543" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. L. Morgan;citation_author=N. C. Mariano;citation_author=A. Bermudez;citation_author=N. L. Arruda;citation_author=F. Wu;citation_title=Manipulation of nuclear architecture through CRISPR-mediated chromosomal looping.;citation_pages=15993;citation_volume=8;citation_year=2017;citation_pmid=28703221" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Nelson;citation_author=P. Silver;citation_title=Context affects nuclear protein localization in Saccharomyces cerevisiae.;citation_pages=384-389;citation_volume=9;citation_year=1989;citation_issue=2;citation_pmid=2496300;citation_doi=10.1128/MCB.9.2.384" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=H. Nishimasu;citation_author=F. A. Ran;citation_author=P. D. Hsu;citation_author=S. Konermann;citation_author=S. I. Shehata;citation_title=Crystal structure of Cas9 in complex with guide RNA and target DNA.;citation_pages=935-949;citation_volume=156;citation_year=2014;citation_pmid=24529477;citation_doi=10.1016/j.cell.2014.02.001" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=C. Noble;citation_author=J. Olejarz;citation_author=K. M. Esvelt;citation_author=G. M. Church;citation_author=M. A. Nowak;citation_title=Evolutionary dynamics of CRISPR gene drives;citation_pages=e1601964-e1601964;citation_volume=3;citation_year=2017;citation_issue=4;citation_doi=10.1126/sciadv.1601964" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. M. Nowak;citation_author=S. Lawson;citation_author=M. Zerez;citation_author=L. Bleris;citation_title=Guide RNA engineering for versatile Cas9 functionality.;citation_volume=44;citation_year=2016;citation_pmid=27733506;citation_doi=10.1093/nar/gkw908" />
<meta name="citation_reference" content="citation_journal_title=J. Biosci. Bioeng.;citation_author=M. Numamoto;citation_author=H. Maekawa;citation_author=Y. Kaneko;citation_title=Efficient genome editing by CRISPR/Cas9 with a tRNA-sgRNA fusion in the methylotrophic yeast Ogataea polymorpha.;citation_pages=487-492;citation_volume=124;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=ACS Chem. Biol.;citation_author=J. K. Nuñez;citation_author=L. B. Harrington;citation_author=J. A. Doudna;citation_title=Chemical and biophysical modulation of Cas9 for tunable genome engineering.;citation_pages=681-688;citation_volume=11;citation_year=2016;citation_pmid=26857072;citation_doi=10.1021/acschembio.5b01019" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=N. Pante;citation_author=M. Kann;citation_title=Nuclear pore complex is able to transport macromolecules with diameters of about 39 nm.;citation_volume=13;citation_year=2002;citation_pmid=11854401;citation_doi=10.1091/mbc.01-06-0308" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=A. Pawluk;citation_author=N. Amrani;citation_author=Y. Zhang;citation_author=B. Garcia;citation_author=Y. Hidalgo-Reyes;citation_title=Naturally occurring off-switches for CRISPR-Cas9.;citation_pages=1829-1838.e9;citation_volume=167;citation_year=2016;citation_pmid=27984730;citation_doi=10.1016/j.cell.2016.11.017" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. A. Peterson;citation_author=P. J. Ode;citation_author=C. Oliveira-Hofman;citation_author=J. D. Harwood;citation_title=Integration of plant defense traits with biological control of arthropod pests: challenges and opportunities.;citation_pages=1794;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Chem. Biol.;citation_author=L. R. Polstein;citation_author=C. A. Gersbach;citation_title=A light-inducible CRISPR-Cas9 system for control of endogenous gene activation.;citation_pages=198-200;citation_volume=11;citation_year=2015;citation_pmid=25664691;citation_doi=10.1038/nchembio.1753" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_title=Dodging silver bullets: good CRISPR gene-drive design is critical for eradicating exotic vertebrates.;citation_author=T. A. A. Prowse;citation_author=P. Cassey;citation_author=J. V. Ross;citation_author=C. Pfitzner;citation_author=T. A. Wittmann;citation_volume=284;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=B. J. Rauch;citation_author=M. R. Silvis;citation_author=J. F. Hultquist;citation_author=C. S. Waters;citation_author=M. J. McGregor;citation_title=Inhibition of CRISPR-Cas9 with bacteriophage proteins.;citation_pages=150-158.e10;citation_volume=168;citation_year=2017;citation_pmid=28041849;citation_doi=10.1016/j.cell.2016.12.009" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Insect Sci.;citation_author=W. Reid;citation_author=D. A. O’Brochta;citation_title=Applications of genome editing in insects.;citation_pages=43-54;citation_volume=13;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Front. Immunol.;citation_author=L. Rénia;citation_author=Y. S. Goh;citation_title=Malaria parasites: the great escape.;citation_pages=463;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. Richter;citation_author=I. Fonfara;citation_author=B. Bouazza;citation_author=C. H. Schumacher;citation_author=M. Bratovic;citation_title=Engineering of temperature- and light-switchable Cas9 variants.;citation_volume=44;citation_year=2016;citation_pmid=27744350;citation_doi=10.1093/nar/gkw930" />
<meta name="citation_reference" content="citation_journal_title=J. Infect. Dis.;citation_author=S. A. Ritchie;citation_author=B. J. Johnson;citation_title=Advances in vector control science: rear-and-release strategies show promise. but don’t forget the basics.;citation_pages=S103-S108;citation_volume=215;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=E. Roggenkamp;citation_author=R. M. Giersch;citation_author=E. Wedeman;citation_author=M. Eaton;citation_author=E. Turnquist;citation_title=CRISPR-UnLOCK: multipurpose Cas9-based strategies for conversion of yeast libraries and strains.;citation_pages=1773;citation_volume=8;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=O. W. Ryan;citation_author=J. H. Cate;citation_title=Multiplex engineering of industrial yeast genomes using CRISPRm.;citation_pages=473-489;citation_volume=546;citation_year=2014;citation_pmid=25398354;citation_doi=10.1016/B978-0-12-801185-0.00023-4" />
<meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual;citation_author=J. Sambrook;citation_author=D. W. Russell;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Y. Sasano;citation_author=K. Nagasawa;citation_author=S. Kaboli;citation_author=M. Sugiyama;citation_author=S. Harashima;citation_title=CRISPR-PCS: a powerful new approach to inducing multiple chromosome splitting in Saccharomyces cerevisiae.;citation_pages=30278;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.;citation_author=R. S. Shapiro;citation_author=A. Chavez;citation_author=C. B. M. Porter;citation_author=M. Hamblin;citation_author=C. S. Kaas;citation_title=A CRISPR-Cas9-based gene drive platform for genetic interaction analysis in Candida albicans.;citation_pages=73-82;citation_volume=3;citation_year=2018;citation_doi=10.1038/s41564-017-0043-0" />
<meta name="citation_reference" content="citation_journal_title=Cell Res.;citation_author=B. Shen;citation_author=J. Zhang;citation_author=H. Wu;citation_author=J. Wang;citation_author=K. Ma;citation_title=Generation of gene-modified mice via Cas9/RNA-mediated gene targeting.;citation_pages=720-723;citation_volume=23;citation_year=2013;citation_pmid=23545779;citation_doi=10.1038/cr.2013.46" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=J. Shin;citation_author=F. Jiang;citation_author=J. J. Liu;citation_author=N. L. Bray;citation_author=B. J. Rauch;citation_title=Disabling Cas9 by an anti-CRISPR DNA mimic;citation_pages=e1701620-e1701620;citation_volume=3;citation_year=2017;citation_issue=7;citation_pmid=28706995;citation_doi=10.1126/sciadv.1701620" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Sikorski;citation_author=P. Hieter;citation_title=A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae.;citation_pages=19-27;citation_volume=122;citation_year=1989;citation_issue=1;citation_pmid=2659436" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. D. Smith;citation_author=S. Suresh;citation_author=U. Schlecht;citation_author=M. Wu;citation_author=O. Wagih;citation_title=Quantitative CRISPR interference screens in yeast identify chemical-genetic interactions and new rules for guide RNA design.;citation_pages=45;citation_volume=17;citation_year=2016;citation_pmid=26956608;citation_doi=10.1186/s13059-016-0900-9" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Biochem.;citation_author=R. Sorek;citation_author=C. M. Lawrence;citation_author=B. Wiedenheft;citation_title=CRISPR-mediated adaptive immune systems in bacteria and archaea.;citation_pages=237-266;citation_volume=82;citation_year=2013;citation_pmid=23495939;citation_doi=10.1146/annurev-biochem-072911-172315" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. A. Stenberg;citation_author=M. Heil;citation_author=I. Ahman;citation_author=C. Bjorkman;citation_title=Optimizing crops for biocontrol of pests and disease.;citation_pages=698-712;citation_volume=20;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=S. H. Sternberg;citation_author=J. A. Doudna;citation_title=Expanding the Biologist’s toolkit with CRISPR-Cas9.;citation_pages=568-574;citation_volume=58;citation_year=2015;citation_pmid=26000842;citation_doi=10.1016/j.molcel.2015.02.032" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=P. I. Thakore;citation_author=A. M. D’Ippolito;citation_author=L. Song;citation_author=A. Safi;citation_author=N. K. Shivakumar;citation_title=Highly specific epigenome editing by CRISPR-Cas9 repressors for silencing of distal regulatory elements.;citation_pages=1143-1149;citation_volume=12;citation_year=2015;citation_pmid=26501517;citation_doi=10.1038/nmeth.3630" />
<meta name="citation_reference" content="citation_journal_title=Stem Cell Reports;citation_author=R. Torres-Ruiz;citation_author=M. Martinez-Lage;citation_author=M. C. Martin;citation_author=A. Garcia;citation_author=C. Bueno;citation_title=Efficient recreation of t(11;22) EWSR1–FLI1+ in human stem cells using CRISPR/Cas9.;citation_pages=1408-1420;citation_volume=8;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=ACS Synth. Biol.;citation_author=I. Tsarmpopoulos;citation_author=G. Gourgues;citation_author=A. Blanchard;citation_author=S. Vashee;citation_author=J. Jores;citation_title=In-yeast engineering of a bacterial genome using CRISPR/Cas9.;citation_pages=104-109;citation_volume=5;citation_year=2016;citation_pmid=26592087;citation_doi=10.1021/acssynbio.5b00196" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=S. Ugolini;citation_author=C. V. Bruschi;citation_title=The red/white colony color assay in the yeast Saccharomyces cerevisiae: epistatic growth advantage of white ade8-18, ade2 cells over red ade2 cells.;citation_pages=485-492;citation_volume=30;citation_year=1996;citation_issue=6;citation_pmid=8939809;citation_doi=10.1007/s002940050160" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. L. Unckless;citation_author=A. G. Clark;citation_author=P. W. Messer;citation_title=Evolution of resistance against CRISPR/Cas9 gene drive.;citation_volume=205;citation_year=2017;citation_pmid=27941126;citation_doi=10.1534/genetics.116.197285" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=R. Wang;citation_author=M. G. Brattain;citation_title=The maximal size of protein to diffuse through the nuclear pore is larger than 60kDa.;citation_pages=3164-3170;citation_volume=581;citation_year=2007;citation_issue=17;citation_pmid=17588566;citation_doi=10.1016/j.febslet.2007.05.082" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=T. Wang;citation_author=J. J. Wei;citation_author=D. M. Sabatini;citation_author=E. S. Lander;citation_title=Genetic screens in human cells using the CRISPR-Cas9 system.;citation_volume=343;citation_year=2014;citation_pmid=24336569;citation_doi=10.1126/science.1246981" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=W. Wen;citation_author=J. L. Meinkoth;citation_author=R. Y. Tsien;citation_author=S. S. Taylor;citation_title=Identification of a signal for rapid export of proteins from the nucleus.;citation_pages=463-473;citation_volume=82;citation_year=1995;citation_issue=3;citation_pmid=7634336;citation_doi=10.1016/0092-8674(95)90435-2" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=A. Weninger;citation_author=A. Glieder;citation_author=T. Vogl;citation_title=A toolbox of endogenous and heterologous nuclear localization sequences for the methylotrophic yeast Pichia pastoris.;citation_volume=15;citation_year=2015;citation_pmid=26347503;citation_doi=10.1093/femsyr/fov082" />
<meta name="citation_reference" content="citation_journal_title=Prog. Mol. Biol. Transl. Sci.;citation_author=J. D. Wolt;citation_title=Safety, security, and policy considerations for plant genome editing.;citation_pages=215-241;citation_volume=149;citation_year=2017;citation_doi=10.1016/bs.pmbts.2017.03.005" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=A. V. Wright;citation_author=J. K. Nunez;citation_author=J. A. Doudna;citation_title=Biology and applications of CRISPR systems: harnessing nature’s toolbox for genome engineering.;citation_pages=29-44;citation_volume=164;citation_year=2016;citation_pmid=26771484;citation_doi=10.1016/j.cell.2015.12.035" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=X. Wu;citation_author=D. A. Scott;citation_author=A. J. Kriz;citation_author=A. C. Chiu;citation_author=P. D. Hsu;citation_title=Genome-wide binding of the CRISPR endonuclease Cas9 in mammalian cells.;citation_pages=670-676;citation_volume=32;citation_year=2014;citation_pmid=24752079;citation_doi=10.1038/nbt.2889" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Xu;citation_author=T. Xiao;citation_author=C. H. Chen;citation_author=W. Li;citation_author=C. A. Meyer;citation_title=Sequence determinants of improved CRISPR sgRNA design.;citation_volume=25;citation_year=2015;citation_pmid=26063738;citation_doi=10.1101/gr.191452.115" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell.;citation_author=H. Yang;citation_author=D. J. Patel;citation_title=Inhibition mechanism of an Anti-CRISPR suppressor AcrIIA4 targeting SpyCas9.;citation_pages=117-127.e5;citation_volume=67;citation_year=2017;citation_pmid=28602637;citation_doi=10.1016/j.molcel.2017.05.024" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=B. Zetsche;citation_author=S. E. Volz;citation_author=F. Zhang;citation_title=A split-Cas9 architecture for inducible genome editing and transcription modulation.;citation_pages=139-142;citation_volume=33;citation_year=2015;citation_pmid=25643054;citation_doi=10.1038/nbt.3149" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. P. Zhang;citation_author=X. L. Li;citation_author=A. Neises;citation_author=W. Chen;citation_author=L. P. Hu;citation_title=Different effects of sgRNA length on CRISPR-mediated gene knockout efficiency.;citation_pages=28566;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=L. Zheng;citation_author=U. Baumann;citation_author=J. L. Reymond;citation_title=An efficient one-step site-directed and site-saturation mutagenesis protocol;citation_pages=e115-e115;citation_volume=32;citation_year=2004;citation_issue=14;citation_pmid=15304544;citation_doi=10.1093/nar/gnh110" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=T. Zheng;citation_author=Y. Hou;citation_author=P. Zhang;citation_author=Z. Zhang;citation_author=Y. Xu;citation_title=Profiling single-guide RNA specificity reveals a mismatch sensitive core sequence.;citation_pages=40638;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep40638" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Y. Zong;citation_author=Y. Wang;citation_author=C. Li;citation_author=R. Zhang;citation_author=K. Chen;citation_title=Precise base editing in rice, wheat and maize with a Cas9-cytidine deaminase fusion.;citation_pages=438-440;citation_volume=35;citation_year=2017;citation_pmid=28244994;citation_doi=10.1038/nbt.3811" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Tuning CRISPR-Cas9 Gene Drives in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
<meta name="twitter:description" content="Control of biological populations is an ongoing challenge in many fields, including agriculture, biodiversity, ecological preservation, pest control, and the spread of disease. In some cases, such as insects that harbor human pathogens ( e.g. , malaria), elimination or reduction of a small number of species would have a dramatic impact across the globe. Given the recent discovery and development of the CRISPR-Cas9 gene editing technology, a unique arrangement of this system, a nuclease-based “gene drive,” allows for the super-Mendelian spread and forced propagation of a genetic element through a population. Recent studies have demonstrated the ability of a gene drive to rapidly spread within and nearly eliminate insect populations in a laboratory setting. While there are still ongoing technical challenges to design of a more optimal gene drive to be used in wild populations, there are still serious ecological and ethical concerns surrounding the nature of this powerful biological agent. Here, we use budding yeast as a safe and fully contained model system to explore mechanisms that might allow for programmed regulation of gene drive activity. We describe four conserved features of all CRISPR-based drives and demonstrate the ability of each drive component—Cas9 protein level, sgRNA identity, Cas9 nucleocytoplasmic shuttling, and novel Cas9-Cas9 tandem fusions—to modulate drive activity within a population." />
<meta name="og-title" property="og:title" content="Tuning CRISPR-Cas9 Gene Drives in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/3/999" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Control of biological populations is an ongoing challenge in many fields, including agriculture, biodiversity, ecological preservation, pest control, and the spread of disease. In some cases, such as insects that harbor human pathogens ( e.g. , malaria), elimination or reduction of a small number of species would have a dramatic impact across the globe. Given the recent discovery and development of the CRISPR-Cas9 gene editing technology, a unique arrangement of this system, a nuclease-based “gene drive,” allows for the super-Mendelian spread and forced propagation of a genetic element through a population. Recent studies have demonstrated the ability of a gene drive to rapidly spread within and nearly eliminate insect populations in a laboratory setting. While there are still ongoing technical challenges to design of a more optimal gene drive to be used in wild populations, there are still serious ecological and ethical concerns surrounding the nature of this powerful biological agent. Here, we use budding yeast as a safe and fully contained model system to explore mechanisms that might allow for programmed regulation of gene drive activity. We describe four conserved features of all CRISPR-based drives and demonstrate the ability of each drive component—Cas9 protein level, sgRNA identity, Cas9 nucleocytoplasmic shuttling, and novel Cas9-Cas9 tandem fusions—to modulate drive activity within a population." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
    <title>Tuning CRISPR-Cas9 Gene Drives in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jvzaEn_RpBbKtrTQqCp-r6skHhh1wDPpEfjNITyTXjk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488753","apath":"\/ggg\/8\/3\/999.atom","pisa":"ggg;8\/3\/999","ac":{"\/ggg\/8\/3\/999.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/3\/999.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/3\/999"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/3\/999"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/3\/999":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HheNj77ZZf\\\\\\\/28hbSJrDRlyax6b9GeGNLEVDItWDAN3YNBlYQr+V+zIKZtF6KDdI\\\\\\\/LdWLFxrmAyzC8xeU4x+pXuJLFJpg2uxbRImkpwSnBImalhRnRprGfmOnqaM5L7OmO5e5O6cZAKCOsNC1YDDyR9c7Rr7qBiTS4ulNc53\\\\\\\/jWxe503qd2ARmUIq1EugWiGyB6zDJUwcDwKFwsMHj6CFEnDz1yUVxA8SCRlthu9NnHQOJcds5ohmtvphqcDkfS7TJ6hu2MrrhFK\\\\\\\/7+o5jHAfbe6cJQkf7qYV+vM7+fd6qBTZlmoJutjdjqlcwoHUlWxDllJsWsUqygoxfVvYcZu0leP2i+jAAya8JUnRFsLi3eFz3d5jxbV1ccVx0xgWx4d6PVQk9ld2iEr4NzgRvgGQkQpLeXpBZgwRRRuWCM0EIMuuQdHHWCUikMjtijJVDUQmycVlYBNEohUDfaR6YeumYA9zUNmRvxiEAvakMf1nBGIbzHVO3jOS00QkZPrGX8tPgE6VTjljXJLq+5nVsKulPmeZ\\\\\\\/EsAvPliO88UG48JVwuee0CYcd4YOaZHxg3ePs061C6kV6T3+OfnsW5vVIsdtRYFr9Rs6WiRrnHkyiWav9Fm\\\\\\\/CP90nkaAtvJ4FWf5ArzLoqHGQhvrQr8ktcQzM37bbmhaVcH+4OVe4LyG8jVk2Hogi6awK\\\\\\\/2ix2pwU6uNG8p\\\\\\\/y6P6j0+3PQFO3+6ebUsxMcpo250DHYGFTmgsho4PVR\\\\\\\/4fN6N9K1prP3hV8ZgZ28RFzCU8Q\\\\\\\/YST6TBIiNnbCFfkPyJ\\\\\\\/rhZIqjzlB8iPa5J87j2LQzoAakFqjYpqrCQglZCmMRR1GTx5EQqEeV929yQUlfj6CKhMJZRFnY31RTXJ0qzzUVf2v4U7b++buRozgJj77gaCzbdXzMDwMbWmZaNIC4TvhFkyDzhx1\\\\\\\/AtjCkcEeeUTb3JP7InB6mvXkhoHBWF\\\\\\\/gYjifgmbWv02q0uF2XNvWvihZWy4uqf20wEWNI3vDBdam95Bkc8dKxdGQxqVBHz6xLy0Ves5erLl+MBW6hOaj30kEV73JY3SXODTCMw2tApnvkz7ep3ZefdZH8MrLPQ2+npoewIFLBUVuipaFAf4SdU7ketjlRDNqw7lKU8BQp2Jgz+387RnbU5joH\\\\\\\/6YwMfJHFnLjdgtpitHZ2zrkMThyszmL\\\\\\\/Cuz\\\\\\\/3mbwJPFBg7G6pG4M7GmJ2gEsttMmu28drs74DCQg8xrLub43IldgemmIbHCaXCs1HP676JzP4IxIl94h4a+X+DNJCFF9K\\\u0022,\\\u0022iv\\\u0022:\\\u0022bcuAiopeHA\\\\\\\/swikQGB+qEQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d37c49204a1fe064ee45078c856ba0a29195efe04edb6b7d21dde606d210718d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002271ZtqV8Sp6rOBNZjgLshLQ5yJ4pRzxgkZ8TBE9hZM5jFoBGRKYtl\\\\\\\/UHleJpsHr\\\\\\\/l5wwjauGIS5m14QMPXkNT1VAZXa8MulXtEWUO7\\\\\\\/djsVQ0PyG7LOH3INzzknF+k2m1\\\\\\\/g5mS6ZK8LQMwCoH8MkIFKKDqlqmupDqNVmW4zHCTQZ552\\\\\\\/wtOUZTTHwGNPeHCBvbklCwsCsGg5jh8H+X96cIx8Bgw5+cYHz\\\\\\\/yGlP5\\\\\\\/PP0Zz2TBKZ0X6GsRJ4Jmjz8Jm15gitcrs\\\\\\\/zPgkIwPkcXnOsCd9iwqttxB+zcUYlD3OPwk3\\\\\\\/4FtaEBALrtc5r1LrLjq5iwbghQj1KUGhtJjmcGsPPFL0PrDgA73o4\\\\\\\/+3bv18kar0cmAhh97u+jaym7\\\\\\\/1P01aQOnMxJrZ49IUT3ujsk5G5nSx5Opy9wMv4XHPqk1bOxoFDq1M1ZIOUt\\\\\\\/UOTKq9km8NJZQRJMsSHJ8aFkhNUHn+yHHfd0w03kmKM994B5gxSigy7at7g0oy2cri4vIYWL5zzLqVosZ0ewXsKq\\\\\\\/lD01qrdGIzsvS5r5ln8O12dfHxUDEhyBxlYL8khm+XfS9oG+70r5LDPa8kALUpdRwt6\\\\\\\/5d5wMZfeQPzZ5i8JGYbLUl7epHw7NhM6Tw6hV+yG7AM3Y5AaralhLk7SWmav0p+FGRmBF5XO+UULrhtFPpyVTfD2+HTtXu2JieOj3y\\\\\\\/SAT2Pu4qXsX4DaLggNSQbpoT8NCmVoN8PBJib7rRiaJJ4NPG828uMloRXMPRJZ8ruC8a7gx\\\\\\\/ed+3cqdnCYHkl8beZvWTbmaEsm41LDudCF2ldOAgfsCj\\\\\\\/jvI2vdpWDmaCl0gei6BMaNOxqBEnlumNJ6SIrUv7pFOiqNQG9QYgPHSO7WEdFTlKVr9RMbZWSvkqynNPIdpATuPkP8qLefrVy8JBviC60mZG9c7r7ACMTjoWFqI23jMKFoEQHnw2o9fuF8lwB6W7QL4Drnj+6ydVUk4gyrYH14CTMDUejgIlBoo8Vz8ghwViqObjrPNZXHWd+iHLT\\\\\\\/faiggHmQ7yvNtuIu525o0sHzI+DMmqtO2GTN1jAg2iatqJPXtY70hDZuIBoq4wIGZ93ptciCva0prP3pFUvD8BKIQ+lLnhLkxqt39Wnc48NGdxafS1z9a4pbxVgy\\\\\\\/4kNhlTSjMmUtJ7FtYZbtRzSAXhcEwfYl0uBSvlmjQrnwpbf0YZ98T2QQYefdGDqjBTGDg8iekuH+k\\\\\\\/i9AyvTqXl7JVvigOo7QOTcpxoudIIuHEBEJaMQifEGNi7Pc1S4j+\\\\\\\/RI9NyuZeTgV0sauatK1UQITWZtWBHcQbD4vod8ZZW2bv0iirHyTfGEVBXXM5OayJ9AjhyQZY+YJyoT4iDQOExga2iQE5u+5nuR6\\\\\\\/QFoRd8slaIuLsGT3iRA85kHHYokVpBOKI8C7a2Lm2okkA2iLnNSRLuLyjRaZzzczNrIyWC2BcEnT1VhYrFIxEglDddi0XnqlrXKsNQ1bVc6VONavcT5zkDcZ7peN5TxdHIsEAoE+vrxqYKlLw6lJrygPVFqFLJFw3eawOkmrdJl1cpgW\\\\\\\/N0iMKxo5DqP1V9vv8IKGWU+B7VHAq8gbHWNu7HIFCJvRlFqjEspyeDK\\\\\\\/vGEz7eI0GlrE1s5v9iKzjl65HL0Y4tN1qvEa86ua1ezUc2HlI8vxZ55TjTpUdevcxBUcN52OPgBVkEQ2i+m\\\\\\\/XTpZI1DuaPxp6fAgw4e3ESsLctXl8By7EezLVjEhSDLuWb3+FNULNXsDJ6\\\\\\\/iHJWh5RqPnnvfgsxvQKzA\\\\\\\/huTETGj9PLdgi76gQSGGSzLxlkOaZ9T2QNmH80mrJy1wPWjDJeVCmehBzEmBCvwcrv6wdTVjg345zDDcfcKB2TiB+Wy870Q6yy9L+pX+FrdkLBR55\\\\\\\/0CTAIGuVedbCMFE0tsLNtYAhmHebo\\\\\\\/0Z1vscgAUF8r7BubGLSKxn7dfONjka2AXPjf6gXTluaTu26DjvPNeKtzEjV5dJ1p0V\\\\\\\/PXnmku1m3QlGjtcP8ilkyef1Fp0net29K1k2qwLxsbWLXYdYSxnfkNJSE+FJRq7cRpYRrjkAIjSysx+l5Jv1vtI\\\\\\\/Ofsn\\\\\\\/AMMaP2HIuwElAm1wiI+rFQXIgntdpJHv8HOmHcs11psy2E25ISSuJRlHmif1LwGKt\\\\\\\/Go+7d5dN0E28aqfe3ZEDqYBMPG+3wpZutm9+olkVf6Ht1i5f6sklYzh\\\\\\\/9iSOvnPgytUC\\\\\\\/ZVFk+JMmPq6mXXKBHPYUb+01N2giY2ttJXUrpoHnOU+rV772pfuw3p\\\\\\\/psB8dGRaipGm+Wq24zFoJhlMcsXd41iRX4nn6tWouB2kFFm41DVqsIfFcwFx7QQ4Gq8UYg1J5n1AwC3PnnnFCiSFybbi02bhkMB3qHY9q6THehO4Z1lvewic\\\\\\\/G4ER7b2I35wHvheFRsUwG9lMIsWYOfx661KQoi5T4+6I8\\\\\\\/meOTZrKQC2\\\\\\\/zkZR09WtWmBTC73qP4mg9T2IToDEY9CHGOMBylrLDAXed3\\\\\\\/nb605cYJyF1Coh88uDvpjOCE+FCk74uUUD\\\\\\\/5gM8WhZm8qCBhLO5CbFPUEhrseKFNVFCXTXQ0QMTz5JnHZJHP2FN\\\\\\\/RFtVfs2uWcjS+Tr2HjpG8jhAlBkpFodTL1IOPtdvBlFC6E6M26vElvgL6BGG8Or7M\\\\\\\/\\\\\\\/0q7CYtXmNFioK4M9RmQ+2HCd3vTlDfIpqnEGNMCTWJwiKm77RRPCr3\\\\\\\/Wdho0PPAQ1LeRTyMZ4tk\\\\\\\/mNd\\\\\\\/Li\\\\\\\/1X1fW9eL4CD+9BL2Ft1QTFH3B1Ge2QNjclFrC3tnBWN\\\\\\\/EibiXgIg22RlHjUWFfdbtKhmF7uCJbw9RkIsGsuyalKHCPrkrDzyNa6wySXmQe\\\\\\\/UJO7gSiW90mT5ZESRRm\\\\\\\/Oh2NdzCYZg6yN1t8V+MBTvqY8zGa+NENAGOMeKDpZYmeXlmNZmqDz+a+7XSTrccSYiPZcW+g9hIWc34NAeoMaEySzTYJU9svnUFO3D2mC6pWt2SQq+OSyFM2mvdJIXFbp\\\\\\\/vW3txw9Y355GT7v9\\\\\\\/gIaWrepRpheswgiwlinyICIrYK6o9Gk2pECFeFO+Sk+m6BaleC02DJqgODtEpnAfqpTVHrvZPCzt3wPAS4L\\\\\\\/VoPxliypU0EjZIJW0y\\\\\\\/Iv82mgG2+0KsnHxsHi0tWd3m4+Xh86dFGRFV3ST\\\\\\\/Yem79EFShFk4SUKpWlnQ69\\\\\\\/mGvyNqWlOJfmFPJpsKTJYuR5C3Il97wKpkeUoq4uiOemWDJh1PoAnOQEkuMj6PltHvO1bYrKBhdJiXHRKSn7UhoX7yHOhQUbtbdu++9Cc9Cat0VN31mCzncjugN+8lgYfec1Hno3vNlIcBz0BBqOBL+0R9haxlBiHr9uIcG77n77CbSxjRX5WKiDE3Jbi\\\\\\\/MpOxjkVBI1VSVr+s2Bo01deen+T+vaqJvU7IN6StXy8NoN3etz04TAvVb2EWNb0T1vhjMe1iGkjNAd843mxWTxK9uqXNZsxpMyB75SSK+Wes2CAOTskUiqpBs6qpaUWm7QGR6YGyroFBkLYwS3R+2qq748eei17N9Vmk9wvoeph8Lxfcx+1XXmDg+lYs7Y1LSPzpIak\\\\\\\/xHQ8KXecBUHKwjonkRrM3rbb8qO9EKRvmMLjFWMtNXHcmNSaI6y9WZ+WH2kvJO0\\\\\\\/kbPAgjLp48XV9xFVguHBePugtkK6NzVrXDtRlUxy8rRCuO\\\\\\\/1weTJbApjuUfnWHYjMtseTO+8ZdCvIzlYQub1RMTYeAXn5gPdbsCY\\\\\\\/sGB+Fqu5UXlXNB1g6s4JtSfmKIDo\\\\\\\/vl06DNR3EFw4wl\\\\\\\/kiJiAwSWOHwkp9cgRF35ZWrwWB+vrg1y+mtFzTo+ArITJg1GgqyvgN18alvIADP\\\\\\\/m7g4GLthX+AwnV1bFZDPqn7CBx\\\\\\\/QH4RBrPKBifmBJabP76tQpnuwkk9x07MU\\\\\\\/VgIR4zCzzc6TcTpMusJLk6ds+55l1Dz34C88+qs0kXIP2QoyH\\\\\\\/G0XsD2ELfIQRL5L\\\\\\\/ZFNHFMe93UG4jGYtsJZL6JpJhO\\\\\\\/bMqNvyUX+Sz1hrs3EYZ0Akn+sFmj5x5Uik+2XXdQcYMbytZyiuR+ljqe+9laclNbJly2fzK6TFFVQzro8YHtTimBs6Kc60bkIOKREY8cFE7xQ66D\\\\\\\/ZqWwvD2PPSf0NSYPQnWtjyY72otXJACHPnzGzDYLDqnqrhDznbzg79G1dK9eBhM2ncQaEJhMTOyzktrBz7X4+NIirGVrSLu+kJKQOEx5UPMf8itHxzlFi48Lk2dhg1Rn9KGYi\\\\\\\/oTfyHmhDJThoMTWxMrhuS5A\\\\\\\/lP0nO6tkVWbuBmLBjfUddU3Ey4dRLTd\\\\\\\/lU7vYfw4MohSuYpJM3caYh5tXCfIYqM0LFefHNbnoopOrTuvnjuSObc1iSVqFU+1usmQkGK60xw76MMxEQfyhf+F+Y+aTHjz9hXAPVmrAVYaLyOcZhOtmwzSmi4y44PqHYV161nB3aDEBfpfSh4pfGTGUCmtG9BB6C64N20Mu3injJgdIsDM9ENkVZnWjo3hSU9R\\\\\\\/y2LSAgQXZY8aZKlqCkO61bnMq6WrSuV+ap11uc6+tutZqbOQTuw2D5yoRgSc8DHg+cP4YBN7HoC51nIrR6dvF9uz4dPvybYGq6w8SHxxg92y1uXfLGUbq+Z48mX7Xn6JtkqKL\\\\\\\/uoQEtwAd1U12MavN0Js2SGvBaZhjyyjne++6CCcy89UyXEAgiaLJG1c9cskObtF7VUXBm6tBd5Z6AghC9vmgGtXRMbWVD6vETaY+s4AmfeYe\\\\\\\/J18LtPcJOs3yfGrtWeUoTVG54XqQqQTC\\\\\\\/A\\\\\\\/06L4T2cCpcJnwxA6gWV0k3rzV6UEbExDDcfwtgeM9KdkE\\\\\\\/DWseFa+rlK4UQpBe2c0UWk3RQNibG7ppreUTfGsekk2bwfo2b2sxq19j+IP+ezsxspxY+C9bNQK9hHytBmkcxjm+hbx9QFJ+hD8MYV3h9VGs4DWLc5vtScaNA8nGKUzRBWnmqb+OcKxbN4SeizTKviVC8w4kqLNuB41ENc6whVdZiCI+9OJg4T9gmg7A+eYQgb8vykfxOHdIigL7LWYVeo+tJaZ6mEFq7NufjeCjeVDlK72zWx\\\\\\\/UDQ5nPgoTL3LhremKIz3FMzjlCkj2FXx5OSUDQ8kgJ9R5QVzxnKoNSRIjhSV8UctIXZqzqn4xZjtULKcQ+4SgVVX2mhPZC5qJHrnhsHQVsbkktOoZKVVunwhMCSENIuqrF0AfvucnNmQAmCXQ6mhOzsSXtOGRQRdgZXLcjzE7LkmjVHlcNeW\\\\\\\/7easz0gEicODPgY6N5u3IvsR8LUhTD1b5fx9u6m3HbJb1Zynfr3hk+vOTvbix\\\\\\\/zqCqVpmaCo2PacVit4ZkQBshVkRnYpr4au8UE5nCuiJU1uYuaFxLA+r0hcvN+xNyxj9dZDgeiUqThEL\\\\\\\/KZ2UIHuTtXCFfwCNld0dKHUz\\\\\\\/oo2MqT9\\\\\\\/iyoyqY8SGwNpcbRhUFkpB57T0lCE7zo03wyNwAeqHtURE4NxmfronQfwE6DhmVVeL\\\\\\\/6zsrBRUOsy9lSuZwrU4F8QnGy0lQxVqlFK+8iRQ9re\\\\\\\/reArgkAEBqfmP4tOGYZwVed+MfilCS8OLd8p37BMbp71PIslqVOjRNfkqc6Yxyp+X9P3bQk3WsZ6TXjAP2NYOW+6ullD1IzrJgfUKDzzr5gg+A+X2gw5TzsccAdK\\\\\\\/\\\\\\\/gjlGeZ1VyyJl4\\\\\\\/2iIyyNp5YMCPPCB5IrPO9uFMYOSFoGduvK35SNz1AJS8mbaTNKKz5i8+V5VxxTn+pYLTmArZnT5lDYSRsiCnf+1jf0KkrFdIvfsAAsz3IbNg25boft4NooNf1zVFNYORrluKDIi5i++RAD9QvEXWGCI\\\\\\\/JaQwLYRjW4pcyQO\\\\\\\/9BniZbwVHfwa4c0YNUB3I6Vp7cWOwJvffjGVU6H4IKPfgiN6+sa3l19v9lOqeE1RKNDxG721QLKRujgpy3sHPPAect5VNzmUYkD5\\\\\\\/MCP7bKEXpvDls2JHLjmxGdEuJ0m2s+SPO1b8aWzekoXkjRxOFjRr\\\\\\\/vyJ\\\\\\\/3mro0g4L6DoGFxUdtD8nvMs5A2qcYg3IAUVTksSx6oRVnKBZnSxg7j2CwwtOOYYWR3r7\\\\\\\/txjyqRBE78DvLX\\\\\\\/\\\\\\\/SdwpieCoaJOQ+j7X8037P15KxqLr7uogllWg44URvs8m4IkpnhM5E67g4kdRzEv3uNO\\\\\\\/H5ofXvt25bjULdkx75pg73sey6gt59aU\\\\\\\/AQPdrKhQmnNEy8XGHI\\\\\\\/1BaQaSRXQTlWpHbLQ9OfiskwO\\\\\\\/ncQEH8P0CvNct+cfmVffDRggTvynVHAbDbxNU3Bh\\\\\\\/8+sf\\\\\\\/L8FjqwXuYnDmVX8bbxZN3Od8gxuaWOuWNv\\\\\\\/R35xwDI2gVgyhvejm\\\\\\\/bDfYLp2zEwlOxDpFZ5+ZcLEhWWGJktLh63J7uqjkiMcFTeVUrBSqbO17lsz6nqwn0mkE6fczZPU2rjJfRu3DJ1Yjwi+SqOzwtjU74j\\\\\\\/6zLhIUuI3h\\\\\\\/qskIPdSGEgIffuz2mdwkx5h+yC0ycQMDc2wI\\\\\\\/EjGYJcqHPZI7nTM\\\\\\\/a2u2U0WFojB+a\\\\\\\/2oa2tsmwF953sW7yn1KpMBXQGMkCD4JTebnWuBVEt9nNPLyOdyCyvu2W+EqCqY9ocopJ2xAiShL2tx0RaJ4kNAnDA6uKwdASZ3WKeOzSO7sIYh5HBn4eU1Xz28ITuIlnMKUzFUSEpwLLwEFryKJvsVaSoH7SrxmlT+WOU8N\\\\\\\/i+dC\\\\\\\/25WR3dt8eBOY5rkkazx8hwovtPuuso6gjkYwTbURuWTD1WDxuJeGY4p11TIxRhDPMwufnM6utnWDTsXDlm1nQUBFt2jgPE006Y8syTE6d5oqmrHOSTc\\\\\\\/WXgAnfI4+XWC8\\\\\\\/w1ep48kQ5jORvXI0SULEeTTyTyXXiC+VWuxTaS49CiYqXO4an8s94abP0Y8X\\\\\\\/AN5s0dmedpY3PJe2ZZI5Qg0jxmRb4HS\\\\\\\/gVM5MoyYpycdRzbxJKTnthUD5wELGLnDKQHAfvkm4oxZYKS2Njvr8dHHT7umwk+\\\\\\\/41CTP8+Hk+F3Gto3vSCeHPCcqCniu4jr3sx8HPdJjHNqRoAFlCSSmmIex7zHiIJaRFqjgAGAf1C5CyFw+fBj0+Drip718MikdchG4acot7ae\\\\\\\/6rLJKJJmaDHZ3Yu+8+cpngyqYjgEQIfPAIpTXZzw2JAKaiZFJ6lEH9awOdeAi\\\\\\\/E\\\\\\\/W9mUPSn6ukYl5P6Vpde6xRhwtbQ10mo1IzOGjryIYS1uYbz6Fb5MtrA0SLcuLfSn5ifA+x82NkHJdAAeKMkGUKpQS6JSTZ2Wo2njFh9MdiNdAIPIAO9L7O\\\\\\\/dH3KaCgRa0NqOcNLN8\\\\\\\/CongLNW2R7UzYHx4g5LnNFc\\\\\\\/2GAiOFtTQKqWIOgjGd3+kelDh+IGIIJrXTpc8nVYQa4uhlpgTllsYRwOlBNqt6rBeaK+mvtB\\\\\\\/bOiMzAZmrUkrUfRSZ0MXVx9JTz7fO2dqXslx\\\\\\\/\\\\\\\/xC1W6tRX\\\\\\\/+oS2\\\\\\\/vkjTNPref+TxdtYX++\\\\\\\/CpB9auC3\\\\\\\/\\\\\\\/CVGPnTkACG4WoESn\\\\\\\/NSWTKaGXsAUjUYoUThRSkoejVN9NPYQ0VRreE0zjigkoWkbL0XGbL8RNbd+iAPCA2FyU1Yg04qhHMuyFxHDr7IEkCLx1b7P5BYJiiKHpwR2uua6k\\\\\\\/XifD4Z320ntgA9yCNU3LWrD5ubROn3393zzbM5nuJqK8scTHmIf5K3fruDOePC8CaQY+KcCb4Ar9xNIgUQ4+vZwoYbicNR\\\\\\\/iN1cevbIoHvKYoiyb01R3EEw8oRRtqf7H4QxuWlfZPIMN6j169Rhx76hLBr5\\\\\\\/uJifEX9uiEiWCWiEecJnjuJz1HhgHToUD+v069hmtlDtlMYh09asjaytADA1pQG7ovtT7Q2RdcMl7JlvaZO2\\\\\\\/NBL4du5wD+Kfftx64o3twAhPKhJl8Oo6DRdPRI9L935HOd6FBq18FR7oW5b3qhhgei4M+EbOPngVYr6y8GD8Yy8uWUSJiLUkDpDT5hEUP559LsKPFLdxXwc\\\\\\\/gxdFp+9P7Iv0eAxeOriJRWrLEbldczSEzZGMvqLVZdqAMIHhnz5sgRodzdiQ\\\\\\\/LSU2B8xbCgBVwkWabKkXVzhBFq1evkWHP9frOYq2P+mN8tPdHp48OhWmlIe0pd7vwTWIoJ02pXouuFROy57HzQg+oiDCZqX+TPIaT3GKtPymBgfGazuj1Yt7xs9WuND3g2cs7bT32qVQSJCM\\\\\\\/sx5YmoUrcPbiIYU+GZEXJ\\\\\\\/Oj6kq+4Z+dz7LvIggUYFZgZiDX2MgClQtog4wdRNQas6zehkrPGSMfamB9f5wnY+mOttg+tmFi4pLCUTD5vapBkQdVByzp8XqRdB13hMmSUA4RLzXFjacEia9bvbUGasZZmwBeFhZnkBYK0fRw4gv2xD0hkofocKeuXa4mKAgJ+KEkSL5upOWiivYFtBinDBc8Q78K2BFI+Ocs6UKkkjLh8X6Mzl5xwyFGgeYqnwfwh8P+UAPRYnaU+kg6sZbmP8675T4E1xAhJJ\\\\\\\/nmsrMkGkM9GtWnbKGcE38efVYDN9XYt7jDSB++Hm18vy9z1gm4vkAbFh7okOLype1EKKw6lx+dfYrQS1ToOw8krbRG8Dg2siTUD2PU9UY2R2JmLS6TES0aK+AUiT9r9TthAahi\\\\\\\/be6QHsnSfExGO5U\\\\\\\/tpKhLnCOp2E4vxhf4WVM+rN6W1H\\\\\\\/tHU3oPTSh4vpK89roG66a1Rh0c1yYvnUmBiUu2\\\\\\\/7pCe7BItLOAZ25jjN9OQDhtcAYdRMq7IQdFo4FFlaSSJNQrr5z6qcDICSmFHjqYJjBGnnI10GIqqAgIeb1vmhWk6MlrUqL9HIxMV2Zct+MJWPLs3BETrqIHgP9eaYqpSeDKHMAej2ti7rmc44XoPpdKK++HW\\\\\\\/UarxeQ4tPTPQ0RjTeYBN6Y6qPj5T3GpSRBor2Pf8f\\\\\\\/aBziSb+bx\\\\\\\/YcRz89UlAyc6l899NggjYyOwoUlNGjCDD56RdCZvxIFJJBUzrFr1Lx+5iYpGt5QOFgmtZ7nMfObypxws1YAcTSZfRJbaAFcJMkJXwcYuAr4q2fhRhcj\\\\\\\/\\\\\\\/WtWn0aY2H5zITGC8YW1BoVETMDPK8pDSM2QfVdKvYvHzAD0e\\\\\\\/dmMW+MsG+udpZdzZh9GvwcI34PfRwM7TW6XHXPylV2rBIa6Me2qBT1BSihDe033GhDGy2011qqSEpmtI0uvUkw5sZ4x9rVZMxHPddJ9EjvwQuSGcwoEojNxwnnWRS9T3E9jodHecwFK2IWkV+5jFv0dKA\\\\\\\/CGieOpE2ZTAybxzLScxTXvTfZVjOGBOXpNMUk1a+qix6k7N4+\\\\\\\/H1DpZT2lRAPyc9iSqdE\\\\\\\/x0QUXo4QufYiCtKsJt\\\\\\\/3VswK75gXZyT5EKe3nYhvaGwSJz3Byokwgih+Gba5DSltfHpRA8+4U3yJRKtgc1Xyl+Uh4atVuXNpXl8NUsrB4oYn2G9STMfdEc0badPA4WOeGLDNL1tyU7SwqZ6rwpivpelufl8PQyXc0worbMC2if4T\\\\\\\/PX9qAFHvFnRhPIpR7\\\\\\\/0N5S2mCFD1MGmlLYNOhjBGaeX9Wle0BXSbJUuGqVOIiWOVIKBjX4zI3mdjPvg7tKv3el9TTCChCFhMgr4AAu2P0wOooXmN8dQdZaHZbGLfnELDyYtFEfgmAsIjF3\\\\\\\/A4DzBcqgV1Pk+Jf+kmwgk56n8f\\\\\\\/ws0ICwCxrhoiHVKXcekaEMQIi4nB0Ptfow3pWqe9rgD1CICc6Lja4WSVTWOFvuVTf8OE00kOmKQPzmTmRwqt5EJTpKn2MtWfCYDPQvlfTIsGPk\\\\\\\/YWuk7l4wY7n7pb8Tynkduk0A6wpk7cu1WYWMfetMZc21istKniZvCaibgtowIf2gFJceCtuyLC96YGtbNgWAosOn3reQIrZ1NZfzWmsBGff0TS\\\\\\\/I9BEJvSGnkuH29vlwZe8YFaeLIN7Sd8s6SAbehHQNfZGdCvpEXQfMYDLJ4ayp1UdhSVVKT4ZpO8DaZQu3j\\\\\\\/9wEueyk3TjCv7IY6muKJskoPpdrRR6bB7+tg77wv05oQXL4D3cewz0hd6UIRvo1KW94yUYGO3155s5nUJkXbkewbzW4lSPttzHnKng8QUGP6h+7xWC44HtI2LqyGwvalrFaVnm3UmJhBOFtka2uqDZvw8rA5GVlr1UHgJtVw+vk3Kd7pgH\\\\\\\/RKeL1LZcBXcb35HySnKvLmwzN5m9B+OPEIWbyyG0IeJYh4dY7UZIDJ5HRvT2DgnUv4MYKMD3Ppzlei2PGUf0HX3TyF8ubedUUN6uGweQ0GZf89XmyGk7BowZvTGeG8so+9aR1A7wHg0y+mWeAjbkT6QS8mwVBl81b+vxBWq11OHZFxp54Pny30Jq03VnIY\\\\\\\/efxNqRse\\\\\\\/4iq2seNjSwG2J\\\\\\\/YZxr\\\\\\\/lBTkvqgOhGoPGdeglpt69tZ7yyUNbXLKJh0YQKrc28EuwpGNt\\\\\\\/XpxQ\\\\\\\/PMp31NN1y+ilkZpyA6SAnzXFyrHl4R97POjoVRVLemzT62hRgdL3dTHNKlQj4+U+AJhkVb5EpofayIasku4IJztJaT5KJOhaU+xHAOHn9KhGNRY4TpeVGyQAjgjEk\\\\\\\/Tux9P01d+TiO\\\\\\\/pUMxDA0nYSTvfhEzb2Sf8NAScyiXOnpuk\\\\\\\/sBESh9K26xNJG\\\\\\\/8lXeZHloHgnCwTRp4CW7wB6UZ5VaciLO57n1qBcD8Dd+nxvFLA6gqUwXLSzU3spp80ktQnEwUr7kGsugw5EHlZkhWP9qjZ02NwYj6d+pAoew6CHXmI83ZcQmlHTuSdExH2y6NQcomf5E+AZ9zi5xczhfb40q26471fAFYlhZO7Q9L0zYzyXQv\\\\\\\/maznE42RRCYJ0KW1\\\\\\\/pfy8pxJxVgvEOfFe9dd5g0qCHFacwmXLr44LtQRniponPwTEqI\\\\\\\/kn\\\\\\\/RiZ\\\\\\\/ven3dOTQMka2ODAf9Z1vG3LHcZzEecwTVnjpCeJ0Bq+6huNrsRnhYJyUzsBI8jVhQKmBWwQoDpvViNvysW0\\\\\\\/zUGi1nPvaKVdM4w7b7BdeZN2VtUFb6z+ZPGAZc0\\\\\\\/Wzqp\\\\\\\/J7J4KRoggJsgWcujhHF3E1LozSp9aTyOXd6T0Qr+0Onv5T1A8Kcu9zvtRfCMA8gureWA6B5+V71Ys8SYwNgXaUu5o7ugrdd4duVQwbq8N1dzrANh\\\\\\\/O5+OICzg3kPAU+7skWU0b7dYxtYoaH8cptYZNy5laq5AVnZD14bvy05QQT2cxzYEeiWp7ufn+EiEfQPE1Ud4CfIB4XtdkMk2Sy80gIsM9kxphsfx7SPThZXWEXT10CgE2QXRVxPh1ypsPkgSr7dPGruuieaAoKQGQTZlKuuJuMVG\\\\\\\/m85Ky2nZ3XYmwZ8FyMoJtkXvDix1cTbuIEA+g7XWLEWy8LnCgNklMrYVYqvVIlBnwBKYMEHLq1KMUfQMJRekqAcmxhwgZivJ+5tGenZ\\\\\\\/IwH0c\\\\\\\/9lPzNFl\\\\\\\/NATqdOf\\\\\\\/R4DkKeXV5JTf7zFYHR8G+\\\\\\\/7AdaFVd+JGJCiyNCoO3giuWlPY5iSRr1or1Jd0QMjW78zQZbotNCdzmvYE4Iv3ZBIBCZiQDNZIFMW7KuVOon3Jsmi3PKL7HKQ8xx9NRzCR5gyHnFDaFUlMJF981RwRbc3bCjssGt8ffb5Dcw7aEkQ7CFiIPeDW4tEHbVcfq68HR2TID4D7BU1X5GlB7H4RTXmQzzWRtTmrCJBxiuPOTLX0Mv\\\\\\\/3BNdBqnwckzEyi0Q4iesWJ2vVLh+vXYiyDJLPV4a9Y1uA25FVe2Ghvu8zYFPObrUlJLKlfBGDTHYxLFpeBdob65LvUv6BmJvfdjh0Cw5zdrfC5NyteitDTqNYlE9CR7ZydpmgonthOsLFMIoU434RhFW7pp5sB3oHpuzcs6aQh28fqVK1irQDr\\\\\\\/0oc5yJb+Nq7eFUCVRSiDzFndV\\\\\\\/gOPHY7SiZjZ1SDun91FgFA\\\\\\\/NixYJLLsE\\\\\\\/dV6cVyz1FFpm0TudD2Pza4A9hFlzgvd+eN77TM1tP+j5m6hDKAiuLKeok\\\\\\\/twHONOePEWQ1FIqNe3dJYmDiekLvZ\\\\\\\/81xKloO4P5eu3xMxN0Zs\\\\\\\/8L68eE5Dctb1igCtC7g03QZJe+n69L5tcw0tHTl8toMmkrKc518h0gIpoRMddJySV+fMu+mZw3RaRpIsGsVIvTCFOyMsuf3gl1+tN7G38w6JBefeH2kLnaHjuqrFNPPXUU0Zr5JVZzZZfbR4hvvfRc+VpeHYgfKUxe4EspW9BInS69A7fOyKME0tpDLKgN4MEvn9FOWDq0to5T8qBKG9tzjJnvCmihsa0oHoxY2IgQ4p\\\\\\\/nW32JmmGuubIYTdSk\\\\\\\/0STmkcCCjwQ0tLGHj0Mo\\\\\\\/5Yutqaoq9A00ldTMXpkK170M1AjzkEEaR47ocbrf3MGhWEBQqRJlOLnKblXW0fP8OjjAkIp4T8cRfGu16hFf4Qyaf2vo5YQ7jdxUm8JxPg9J18\\\\\\\/ps3ViI38u5mnK9VQWp200FicJoOzT9xJjnIu7ZkgZJSvxumCqTAHrcTLr4Hef5O0C7Q+n\\\\\\\/VzkkJ1b7HXultHBjbHIndl35E3sG3eTqKIwbCvfD1Q3uNsGScIaaBcWCzGY84jvhRfVhid8NeE9p0oBMLEOV9flwsiildCHXQFj+oeqAmeQ\\\\\\\/gkcb6\\\\\\\/10eoY5He\\\\\\\/eBjqwLq0tGz3Hcn2Orzw9AF01Ypu9e3iZOjYNVptpP+xgJLGKq\\\\\\\/AXM32\\\\\\\/jltR1kqpiPwRN2e71S1MrZjd7hkivopyT1W2oSe8Ov0ZJhsik2GO9q6ql2dXInZtizFh51UlLdpp2ZgYUx8JBtFLKBvpMOfTJat0mwTz\\\\\\\/uKodmgUJWUQrwwJRZW1cuYAX\\\\\\\/hTUTADB00icDT2VxhTMnhcCoc9yEvUnW46MPgaxBblcRlfrSU+mrH2iisRdld78Y6GZRZvIhDPwGxKSE4FrAnmhvMP5vxWkBTpVembalfuDoZwQc7M72RCcL9uGvXYa+SEQpSx8IEm3hr33i7UbQTabchdFjtqEYhROGPCUP5mD99nmKVxFdxADg69s87\\\\\\\/Xz0MMbcFBWtJyaloU2\\\\\\\/xIJS7Rx7sQsxl584vbZ77GWuY0O4oLb6aI8orIoIp34yUClroUqmBRf1U65rW4oGw5v\\\\\\\/zbMQUGPUZJR+LGtJH6lO4rw8D2fMS9moKnxMw2HvO46uoZaZflPvmZZDYnTRkQ3\\\\\\\/XLlzW8\\\\\\\/iExcRTn1tzEq722UIJ3ATAJ\\\\\\\/YwgFcYbjJddMCq6UKqhR9NOo530\\\\\\\/MBkMvSBzpji2MJYcq6x277VqjMyonc2TvJCKLYXCxjkgtpCljCtKuuqItDUDZ1KFygjr69Rl7uZfL97FRLy5wip3BvAdiMbAuZjlcMuWjfUc7m9+oJKxOYMwh5zA7+Xg+nTSRDdKGjV163iNqpgkEPXm0HyJ6sjWVJ1ILUdHkh4QZAp8yCU4hr8h3clvqfhHc5+pS9YHRZQXWYYRljKRE6bA1Y4pbP3dJxlzVKonqnkwm+vqGb7E0dxTbuQddS3YoAyGBLwVopCHEH+qjGKXLCBy1EPfoN+djGM1vxOMgg\\\\\\\/p1q3R5CzRZIICj+\\\\\\\/tPn1YtUJWOcmi3QgXtv4lCAHft33SvN\\\\\\\/OQFxj\\\\\\\/J+6FWd1jETinhQR\\\\\\\/Gu+bBgZllkGL+FpGq6LH5mzfAx2c\\\\\\\/U6hP2Wt5ekcj6NeRJdAdpHWX+JdqynKXfwIu7JD4X2Y9vcR24UZysz69ssK4aAZCotRhPu2AcBK+7zozzeQmFnTo9FpAcq9\\\\\\\/FLcnxoHRtee8lbmjJjDcrJMd3u6j8h0Vq5gKKak3qhp7dr0bWsdLikQbMVz2EqtQjJ6RfYuRSiYzED6Whm0YaRHVKjHQjok\\\\\\\/T+tmRRCf2VpOGoqAXHgh6CZJdE2fZt1xlMP8LdVANgymDkOTLRXV1Og2kN4J\\\\\\\/JSwm806nh8SQhzcMig7kp3UmEiJun5uOCHhd5gzE\\\\\\\/KXcWCTqpqTVwP3t8Dt9UdqBlnxZtLq2jlYx1igMxmjdIKgSqmSZWBsGcj83pw8tiyt\\\\\\\/+IDZNh\\\\\\\/\\\\\\\/rFj0SGVFhOshSJJgpG3YNQH+9kKcv+Mtj5043qQTSQghBdgnfVV64EbFRnSmuMjpxFuw0O2ichLIbNOp6vjFVVhngbTYEbrH1qoDr3DPrPn0Ng83ExokUFL4Q4RGVVyGf12wBsfT8YEXaMDxd9M59kjSe\\\\\\\/nU2nnQ\\\\\\\/uuSceoh1cvKq4NtSaiZGyVJPSPFDpUy+V52NhKkT10edkhp+gPyWGYW3la0w3MIXAGR42eBUSBHtMXQtf3Mxw06jkwxRFIAB1llawX0uOmK8ivpToENqM2fb6ICqnh5g1OGLCNyR6z3QCWNdqYBPU0w0Es9ej45jOMAonAT6fgyleWFesVFSI7egWpLR356cYN+OvINV8k9clWgmIaRWEp2eFJByTJqmUbaKmq7PJJXGC08K9bgVB20FBodgu8qN6J6tjfhDVtaSDz7Q860k6Ml3zH4VozdPAK+K3aNvw5Eb8m7qOldFgMGQiYI6LRtrbHcertEzC8zCCXLug5\\\\\\\/JSgJIOQOpxQKigQB2Iqf29tzrhaxh88rStAsRKSrsEccvFyWq524h0xWK4\\\\\\\/adlwAMqsUSrwD67gJfd\\\\\\\/VehEXVLL3CmEQI0fgI51O7cGsiMWjDAmmpMNfzpaw70X392qD2I8STh4xnYWtbOEJFZQTKvo\\\\\\\/n4WZR4PHzPxJGSzBlSqKAV8enFtmtbPPykHBOMQhlISYT5aXuKpGTa7ioFVNBINsTPpmV4eRD7sIgLGNtZr35rRhm+zl+HSuIYFYZiaEbSbX3hGn4vf4Qnj7yq6pOmlMV5oem+IJKNGPTvnhXhfaLh5RogFmHXfoSOSsZ7sqM5+rZ+AeCZEJQePtVW974YhRtfiTDXBfJZbW9kKZKHYgZhb9ffP0tJo3X1bmEygUOMFMnE4xtD1MqmaZm\\\\\\\/2WiMgPLkK0vYFgRJohMWm3lgr\\\\\\\/wMK5QUdl0i7Bzj6YqYwBH7nQpqMWig0mwWRqA1jJjeGVuH\\\\\\\/86ySOFW669B0P\\\\\\\/bqHU8cPAGL8zC4gNVj7w4B01+KoeN2wSjvmJUKLhgofxgovktcIhlksXPabUiwGBki7mW4kpEF3XxqB\\\\\\\/oswh3rB1CzqzyQ0XBYZCiabmWLcdWPbEjrQhwsxWm31sVCMf37qp+kV32JGQpzO41SYYlu1HNxvCsfaZykZBYRp14XKg1gsBRuJEDZ+TZiWb84oqhUuC7pBT9eJlPjz1TiF++xMEd2URzbxWGuQu2SZXtc5t92e1HfnzpSHLRz1qaq9qOnjglmooZkh1Sq8LA6UPeXGzvnPMgAO7aKLUFGmkaiLQXH6FrZIGxQ2UIwCoCFoIYFg4Cr9gPuPz0lfaCmoKd8JGp6MMK1E4JqsS2hyLMNrOij5Wi0GUxbWdYo\\\\\\\/JLGai7d8befAkXY8ReErbgPKJbFD8n4UZkEdtAq1lvTjxbD94HOadRqOjthS8KRNcr8uDhlPKQbSZnJ+x1PtWzEER8vLtu0sMYTuNR1cJKXjoX1l5zDTQuj9M2xPEnx0vSaLxvUfuOUfKLUPM9LjKxFg26ZHT+yhP54ZaKLMFdsjRWfXBtP3csZdzZSYlgsTIbJ8Q39zqcTssu4zlPY5y2mEN+k+Qqn8Pirng95tfLCeAJnk3hZQDSzoiSN\\\\\\\/\\\\\\\/tXSwGtRVWPTHIwr80ZC0ZfX6Bq8bpek1OkY0Acd58u8wW7dOkpewbtirYLwaJ93NKZDtaww5ZXvO3zzZjlg9EAr\\\\\\\/ddWhVmIxq0IKxfyFSuhVF8afRHUw5gOfffc8MrYiNTZD0c5+Cgn8EE0OhCBZ8IIEi7+x75gCO5XRU339lnWGLLalpsPovYjgk4sR\\\\\\\/SenM04\\\\\\\/8Phy\\\\\\\/ZTHfnIX1jBHfWHi6c7LkA8coDeRsg+q97y5rNQA\\\\\\\/tAz5SE2y28QDU3PzzrjTilORMdiZmQ9Eo768rS+vFXhOpVyTEgvgGtbfRXlL9OxBj36XM2Lv7WtAZvx5DyXXzHcujf9r\\\\\\\/rQPlnxFsGIZMYQY27nKSpAp5wd1+88hNG5zmg3Z\\\\\\\/Jsg7sacTiYLjdbrh+YDP0XLFYIealMeA7ookf\\\\\\\/AhmJyW\\\\\\\/skqZtuY4Te\\\\\\\/TjTpj\\\\\\\/nCiNb2Xw\\\\\\\/LUkRA534laxpNHWTDJPssYUEBkIIA033cOBotPTnoIEVjg3Em8CiGO8MeRlWSfkbo46BaTvRePKk3qe5nNNb7z47GCIt5wD6O2nSJXUoPIkiyYvn+FGuBqZrqKc8+SYBPXYqNyeEVvO4SNp1O8xEtyi9nQ0s4iAsVqRUB0yRan1oon2C5PSN7kEko9F0kZz+6znf2VFzFDL0oNrbaxGE\\\\\\\/8O7zYgkLnBoo9wLE8NtC5LImKQWDQ+DptjD89dODpQujCL9c65Rj7LNA8gQ09QB3p38BxZr6cjNfZnfOJVkB\\\\\\\/wWT+0\\\\\\\/OHfm4SuB91ybw6imd0s7RiW37uS30LUlUyUUkBznvzqmr\\\\\\\/1rHGad2Nlk8n9Z3Rt8Ay8\\\\\\\/bTAONlBuLnqC6yizCdQo8GSNRgoOVrwhAGcHva736FNZzw7rrPI9+C50awr8E0dwH5pczw3gNAGvErKzBryenUlwktvcDiUvmPTaS8z10Ic3PzqsfzMq2QnE\\\\\\\/RmQvoi\\\\\\\/PiUo9d7AfshsQXCLpplhwZf782ZaQaKjFlXH4ARdt6FVAn01QxwD\\\\\\\/XDAYqhWv34GuWt2YSIocscU70idcSCBCy0AiI\\\\\\\/JrObxZhcwBIVhToYtJY9664bN0o6\\\\\\\/SL+sjsl3GS9G6la44rM2sqRoSyKUNjMmnvE\\\\\\\/G4ThGE7q5QFrOqEGZs6H8q1GlYW45fNyR5loasZqfN+4XA\\\\\\\/83IrDyzREuVOmz9kZ3lIfy2mA6ytOY1kxjttIZHXwAt8UuE22+uymXSOmOSdii9wunTVzHS3a3NzNwj8TtM34aUA41w5M0ivL+TF8bToDThZhFYzG+0OtX07osUB2042ZVbwfB9BC+oNGNt1mZwOSQTxop\\\\\\\/iZDBas2ka71M6biuvIkOfYmMd931sS91SXhfqM3aOW\\\\\\\/1UJNIiSom0lprZHrdXgFVLrHBBAu\\\\\\\/eeDQtyYREQ3XmzjGtDcr4fOYCeAwcuneLObNdJy5Q09HtThkJjK5yURsDepDW91\\\\\\\/bCkO6NBku5Yk4CmS0Q3GdVKHCUu+dFXnxD2\\\\\\\/5JaZioBTbC0BxJDbijijv2p8NKCKQLPBhZFWrjt1V0MyLb\\\\\\\/4oYiZOx\\\\\\\/PXhsk3H1GvyN7KM6D1hfVB2w04U9aLxavf0RJwdisG8ATUeZ\\\\\\\/WUfHC\\\\\\\/Vq16JEE0e7K6l13S\\\\\\\/kVBovzkrcZ7bnKRvPqWmWs4UrS+moKd5uKMy9ra3h4VsHb1a6lCRRaZyfHBcCNfJpRfMZY8jYNbBJtncOf6Skp6j0jdbl8B8jU9SA26vdqHToRysE+gKJQiFaw+22GN6PB+SWHgdtrGxOznKPkXwNjbCt5LG1o4nvvgtDWTIcgU7l3Vh47hItPBRgDft0\\\\\\\/eGwF2LQcDAZ6AuXmEC8BFkkcP81Ptb6ClGoEMRgS16ejpy2Xuo2Zscv1lDEEcl4yoPU3w3hmHBFTy0AI0Q4U7SWAedwO7vSSqxhMLu2ECPY5P4aEF46ch6gBzXYcHfumsgGsm7Sahh9\\\\\\\/c3PSHcgCxSS0Tt6LmkIwqOQZPFxLjSnx16JeAm1pOmbow0Dvcs4zwFK8dsGG8IR3gwyTKM1\\\\\\\/2uhZFWURjFnvewHajDzbD8JcN3IQjqttsM++OCzzsRIjgZpk3EtkAhSPRMOO98yrEFZYR09UOBkhphZFh2WU4KbE4KtWcH3tn3pBnthnurqw9UbZTr6Yhmwq2tp1alexQVWYpRxkpb7VHSwnbMXPCNMfipMGSjLOArO45Vel76TJcJKYKtO\\\\\\\/8gDp2GiZ9KwCPNvlYhdlAvaDj0VQlh0nA+qPaD\\\\\\\/qyY7R29c\\\\\\\/ch+WZsqhpK6lH8v3\\\\\\\/kUqGGgEglXWYM6SsGE5+Kw24XF7yGM\\\\\\\/unTvKrIbtxizblmIpwib52AizoRWAyqbsexhGQAROSgJUK0w+f5RkX8RzgdMjy+afl40v9gZ5LO6dHcN6VNv5vnE9ZRhtb4FBq9uv+EnhWZBquIEWNwo3OMOzmnejXCYCinhoEdYMBr5Febzkz5UH4jJ1MqcgRitN1RMzzVdZYiYtnMh03170aFhfgBF7tpz2fScY08gjpr3bdzVY7\\\\\\\/1dadLwCVCNDaS4iCc92QW3MeyO1rHzUuU2bHsBB9kAwUlRe9WtySnoq+TnVU2f9c293SYA3lXwj1ze3sIBbmTPSDexKo4+pivSSP60AlA4\\\\\\\/ego0lnrIl3EGX\\\\\\\/62zbUSksaWxRUG2J82l6bEJ+VZrgp2sKq10Fs+1a1PDo3MM+UQDPzOmVosGuW2rQPLyOc9JvosBskOoHfs5PW9ab1V4c7wsgs+COPB7FcjlV7mPfs67Hi2OIw7yfifaZcaRt9VArrZTgsj1V2cYIgZxi9tDcFQwHIMUBkDFbMAyJ3E7qnnRTf1KkjmBokDtOZAPTyElgOOfsOyo2CTbcIeKFZNPuarSlc1E+acHS6OmmoQikSodGr7VBvjd+acygwTZjQABQXShLL3OMI3LIU+28HoENwKVuO2azja5R81Jx2Dil0iaSC+InyOtEk+nGcRlRprawbiPqZCJWCUjIkhcAc1QVmAPP005gjZ0lmo\\\\\\\/2nnElnsL9RhfVEUBDsn\\\\\\\/UGufPHK\\\\\\\/Wwmn+EOPKAt4UKln\\\\\\\/EgreNHhlS3xPqfAxS43QqeWSgtrRzAK6xC0RGMwuK9whdcAKsMkQvmOsNwGBpSFLMZwfGCN+T28z72RH5tinufP5g4Bm1VGsOnhahF4Qq3uVvfX0TurGhveLoN5OcBNgOLHxuqD+qdoBTyINWRIeBDJRvk1dZ8pC\\\\\\\/e42gIkfCDweQKn4YhGac+Up6S6444+MMdJCR2SPLzRThrgA4CN\\\\\\\/cFfuGj8IFugUmx0t0nhAyKBOIuGHW0jGsyTlto2OQ5IkBd7yHlN3wRbPgBN5AAqQRLHUR23DncBRYcdNm4wa3uHkoDmqRIKAwQBg1xDkIkpVkSsuUeSfh3i7PLC4XGW8fGkTg3ct7++\\\\\\\/5PpGxlhoMemHFs6eKzNBJPrRxr3hCoNiGRr9oya986A9Ggfk\\\\\\\/Kg2TeLyBax0G1kWb\\\\\\\/XpAQdm9Z1f8dr20146jb56yVK7BNFDqNBXApTS+\\\\\\\/IL7j8gjUGxetnbw4zct763wr+uTF5uNA4Mu5TjZcNBHlcu0DLLwysUNrnG3DAjElzfN4enE1UxqxyNFi2rZb90fMRfRCp2wbCK50SMXFonSKmMyDouu9KjdQsxYCqCUUXyBWDBfdayp23mPL3K2jLihlyD3OFMjwJC+du\\\\\\\/bUlAP+f4eHnDJEBD2rhU\\\\\\\/Ugq3dHA5lnFT7e0j7TGQ7m36KpKuUmJUzV7Vyd8BYKaB\\\\\\\/mvVipk8xZozrFX6xapnKot95+HitGTD5GJLFi2g4tsnTmXX4d8VLrHzmrMhE3FfJ0hleZB72cAXL3yZjZpVhtwTNB8rUtdTC8D6zQ7vy74AOYfHahRw3ULm4ZFO5AZXVZPXke\\\\\\\/zib9DPLY9eSCf7mk3SgHbpZc44fXz\\\\\\\/5dB1nIrLgXlGOm4Pjz+N8ajE7U5cOGBGPhR8GolQiHXNNEwm2g6C\\\\\\\/+pvc7sZ1Hrs\\\\\\\/q9fFxEHFYtc7522SZFXHmltFBzp9+os70kleV14dY5QTDl1rzI7C\\\\\\\/73Bji1KqMbNGe5AIgSZgsvt+K+o1hijt4qrXHIF\\\\\\\/nL6xd0EkAPNdluQrtF0GDYxAMeypdNy2eUHYaQ2GlWNLfm1k8LReYyd+iqqtoYe0XDtdJe9mEAds3M305+Y2oyNmm5Zc6rLhAxFrkqeFRvYJaA9qh+CaASuLjo0fsRFMYKvCRVKaRAS3o9QvLSKLfa\\\\\\\/3ZMEeeTH1d7s2K+kuHzVeHca9rELOm\\\\\\\/nlCuphwFtWP6pPPADBrmQZH49SansiW7r15Qe1pRt2yOT+unONQlLKHuB2PjPCyVVzEiWrfJXuw1hUNUMcHqtrcbn\\\\\\\/n4cJRq4VaAL5dEv1v5G+CO289DK88e3EBP0mQCDncRtsK16sODaM2HcujmtFPy6b07lFpTv2sEgi\\\\\\\/LuI\\\\\\\/LYVi3vuv7z5ZVkznOnP2g7apBVozVoSUrMF3CDpU9m1HGkOwB1ya1gOlcOnsOtoDH6Mh+1csiv0Bqr6TWkwznRMVteIL6DjSD0GZsnFALoNN1S+LVufAeW2drK7lEfdYXkOXspeNl2NNUK79RE2jxuO8gzS4km1wbWqw0EEiL6cfnYRP+X9yUeW58AlBo7bWe+uoT4yCTN+f\\\\\\\/jwzNLNYCRg7AOVMOJrjD5sW9IqzXNAeLQh8flKTmpSaDEo1g62nLkKRW2TNDtcIsLH8uPnTam6lCmjOGZmoDf1f7MQe62D1EZXi1WGg7LWC6zasekOAFMfVVAt9\\\\\\\/Svaq3xyqrYB4n2BwV64bfgEKtrdvMeW0dmoUlB\\\\\\\/jQFA\\\\\\\/GBgvg6Tke5J8j3IzUdDTgTIjlWm6ALloam8gs0ivhLyrsXTQmCqK4jYPFTpwHMJInKGCOmoeNB1LijzRlN4FNsxB16fl23PZMyKKpw4FcAUoKgbJxnaYDSTS3H8iaujIHBznL4MiADTD5gD+XOVgHuMWPrBds2hnyJ383lWJYu3zy+uZQb+s7dNRk2zThMZvEeWzWqPVzSzrrU+yg85e1X7L4EakG52AeTkSaRZ\\\\\\\/9hV4HfIjCs8q6\\\\\\\/BeITKIUg\\\\\\\/7MKmG\\\\\\\/TQF5MROIUGQiygIkHLULU3wpw+4mgkWL8v8Pf8hjlhBsNsF0E+A6VLe3FSBe590axNqq1UMBB64TPfkRi813sLsYBOKzeEvESBPxboeOiKFVldIAtAOKHqgvNkdSrIRdComCbFOmkjVl\\\\\\\/AvhM7OdvLfSAF4T9s1hYp\\\\\\\/Q834BEFhf6hgvkxL60du0CVj7MDyX18pZgZesRmhM8BhHRdxC49yurP4v3rH6EneAfd80vlfONio\\\\\\\/WhxA8Wa3kpuTW6mhiTxXtGbrIY98HbVKsMXcu8tRyZDvMXBPG34M4hRawgR1N\\\\\\\/sXhlWov5JmecmJ+HEWyRTi\\\\\\\/\\\\\\\/YjmHSJboA\\\\\\\/Wlt5LR0wup3Ah9traRFfXNYzFuHnrRkN4qb5c1VS0xbFbOicRZvteJeuXIKZEDKdE7zH7tdAr97zUrz1+we5GDFLUYzPpN\\\\\\\/H1j3nuuXx3aZaaXWus6yZbXtHOSs69LV1Rlo\\\\\\\/vDWR5sQFLT+5wc2ZfTAV2tNiMOQOiesnamnRdVuxsCBkGAvpEM5cMFxnqWs\\\\\\\/grI0c\\\\\\\/TTMKfSZv\\\\\\\/cVYrU8275Zb6iRnhcvoEDPCzF1Lshi15mYl\\\\\\\/Kxfl7Uv1R1R4DAwE69asUKkM3OT0GLoJLkYmK3qjDxfNkbs1ZsrVoHfCsKHxAIsy02Uk6GdJR60EAYWZrENNtpMm+OW5JabxF4U76GRaBP5090RkOt6F26V0FbU1a5HrgYz1TRpLDfqjdvTNv6IOZRb58wlv191WOx9SYmw6AnL1GVRUks4FAWHyTDYvcWkEKifNffCV8LfgxYgdPwi\\\\\\\/m\\\\\\\/yge\\\\\\\/d8hJLZ7ISXbMFCEUth0GGv0gMuI01n+Ymoa2B8wwhSic5\\\\\\\/UB\\\\\\\/bB9Nc\\\\\\\/rmcxXH1dKynUwmvOKGjUPXGETGXf2DgYtFbPuaVkrah9F3CpaEcWciz5j0MlX0TvlO5o+573ERcEaVgesNF13dPjf27+sdca3Crt8nfb1TF8tKVdDFUuGC1LmMBBwiTa2kpUX00QgHadAqnvUpuUXQd2GXNtTdeOIg\\\\\\\/xeiC3mKM6l1Uwazp3rw4JmqTXSPLcPhN+OP+XOvjiq9TakVbf02Pu0sILXbstErUj+FA2Y4Sx1mCYCiCcW5oPHY84deNzHLRUlNungryXaLMCFzRL+3Ji6fL82fZRty8BKDGBnB9u434vexngLKwt9iokj+MXSigNKfM8NnteP+im8hqQ2usxUbxsIsUCXAmNGKPsOwrNzJZpc75R983IRHb8WH5gCspMmbW7WB4UfiLrBDmAlJtfrIw81\\\\\\\/curPdj0YlOp+FE3U3oHKIcr++BcEz8XV1+H4q\\\\\\\/hfBZfvusfZUpOPndzNX5aSyS6yyJFnqOUI4gykThnAoVQDGgFuh6BEsLiajTHMjnqUvOEUz5LUcokApnY92kPZiC6QdqlDI9+sOsw1TPDeDUMCHcmoP1F+V2jWaZSFqmInzsiP9iWhdIA7jDi2D\\\\\\\/TnOUpCYlFTGispcP88MyYTqMBsPMeAP4K2u2QOtVJntx7E5yA3Z1wCIvpmLnCNjcF\\\\\\\/EsEgBfLUIzjZqkofJn4w+9KLLC6vi2\\\\\\\/if1TVXyuDtZoPSs4mH6iAPqLM5cb\\\\\\\/6qDr3l\\\\\\\/bJ4Rk2nqF4AkG7CzBlinw4wn6SIGcuENCor1GMcxjAAgTKKADQMTw\\\\\\\/Fx4hEUwr3rZBSSSBS5wK4YB7ewX\\\\\\\/q4xAJffwG3g75N7+rs0Gqf8l4nrTcB4diFn5Zauc6jAuKVsIilmiHn\\\\\\\/HvugzkSUx1JquNTS1EUBs5kzHzfLhmqRsbHLx6GMFWfV50JKmeyhGuupCCRGstfUdYrKmd7G+W07TFLP5ULQNaGKZP8VrhO11SuUm9BT6EDgPo2qyoPYPqZ4gIx0BBkcCnQazLgL\\\\\\\/FnCsO4AZbQdHyQHlMy3rY1377oYZxxBzOcDVKb3qwy+14+ewGXndbqkC2FCJkrL+FPGScXtNTqJDdeGDf5Sg\\\\\\\/Zxz4oI7wmel3dCO1DeFfZaxZvjXIILJ61XnHpjVN27KU\\\\\\\/WiDVdX9ieaueQlEZuCb1LUMDdtTGY7Ab05HHQYR6yEcUrL6fRE8wYR0fT1vC44oJVLXityVOrFG8OpaovpNMIgX0bWXiED7loDoBPYo5oRPeXiDt4GaQnOQXyk5CUuZagQ7AXOYPGCH4dkppqbrzjl4xU0W54n\\\\\\\/Z8xRiviLnsbv6FWqySk6DZ0xyvngy3\\\\\\\/dLB\\\\\\\/+aQAxiOXqn4iNnbrLT7+49JHDPsjmCf+kir3ZBwugrSr+9U31LiDUbZ0fFQEEMdfStNbP93TQbKrPmfDyW7ZZ9y5FC785b2vde1flgphOREp0iwQD\\\\\\\/M6DyhyJwtc2JJL778dtbIMqodVE6QtRxJENUOGT6FMzaV8anaUNJid1xfRAipG5wZmOal8sdc1rBPUxOfMOw7wQMrQT2S7\\\\\\\/ng76Mwbl+z3GJ6SvXtB3FC9ZUERn7kikpjZee99kwhfjADab\\\\\\\/YnP8gnAo6l\\\\\\\/NETSc2Tao2i6t6\\\\\\\/WUwZmntxymoXHG4pAUCQYQe2rebBlRF+7ByApkN4\\\\\\\/\\\\\\\/BqCFlnLXAhZW1fi2MWo8kqXi7eBoiMHOpz3ZrEEEvn6PfZlR4sULs\\\\\\\/h1hBwsbqGfcyLHWxEzUzUNHnY3ybx\\\\\\\/C8y0IAVRTg1LWsIDJglqzyC8Zj9UjNybMXxAWbazCHBSWFbqxCHUq6RZZsHUELzm2Kb7xxOE5HL\\\\\\\/bbpNtNNML4nykF4MUM8V02oV1EJtoVTOHuPHCHae6eXbFeLism6t5uT8VzOmshzE4Uzn7WsZkuWcDmt5D+a52CdeRu4sdjJrs6MIcN\\\\\\\/AfjmYJixEgWljpbJqlmqOLfg2IAFE9vsJmE1cPyOwJoQ25Kv+uGlyVGj7OVo1KVm\\\\\\\/VAT\\\\\\\/omxbX9753kG8n2ll6FyUMApp9E4UweA1iAAt\\\\\\\/XtD5Ql4lsLngYo0JyMappXXwmqV54shkosj1t+7e1HpAmCLURTnbhpl\\\\\\\/2sMi5qnmPcswLuh3ueN7EzDuUta5vDxJ4jmOhV+yw+CpkuYBzZB143brgLMViqGSIFbPG4xmUhTOVlACFeL4jpIRPRvqpCCcgjq3d4NGjZyXhDxC8VRSP0t5VUwSJPDYUwf\\\\\\\/Cs+ZXMCuvrAnXkvgTftv6zqBpB8pCEkTiSstI8Z1YpTA9HVMMNw62XHYEUtsmpskFiC1cR9e5TaLyL0uQO9\\\\\\\/\\\\\\\/Mmz8CkWlQ4RFzeWi086MGB\\\\\\\/\\\\\\\/sVpflfuHkVR8g6ig\\\\\\\/cic8XQsIhpXU5\\\\\\\/4t9Ubke8sLJlCn6LaM2xkhOug9mNCjbYcLaQex1lVNVqKF1EqD0lk3lABekStOBgMEBbw7uX5kGGSZ3wWemsmgfB3Oy7oaFwxfx+Sip0TpnY5cpdDXYqa8VahSx3VgHW6ZBvUxWNMoeXxNHpt8T\\\\\\\/GsIvf7vcHxaTJEQjlSBAG0d85kOQykEu5qs5O5w5pBISaOhlotOp9UJ7h\\\\\\\/fXXSXbk1tJqaeXr6FVibQ8gDcggwlxdlYUO3gVEdWkGRjfs2FVuawwSbiAAd1dJgu1Z0FouCL5txHY7Qm\\\\\\\/9YnJjqYg9j8E5Eus4MK9mSG6mO6TViZq89da9Z2yFUaCXQR2sTopfNDgztGhuWgIzRH+oRtb\\\\\\\/C6En0nzFq5gJWJ60TWbMgSoCMDsStpI3T+3ZoceV3nDlLsuENNfu4\\\\\\\/w5B4qphRtfzbdpTBqnNh+zfHOy\\\\\\\/XpzAXBrXP65HsiAwInQEsc5ZYXJPvKP6qA3h\\\\\\\/LNHjraql8j\\\\\\\/9ZyLWUIUwALc10e3ieEwoN84wBaeYHgqQDG0VplElWkUFoH\\\\\\\/7Jz7SC0triXyAH7NuMDjR4kAd5mbRaRTsE+KSVY2h4jRBc5EEbRSs\\\\\\\/FLH0iKSgwQaZg2a6IIBXzA4gm\\\\\\\/xA49VJTOOvcZ0F5zSi5j2+cBjmoyidM4\\\\\\\/zuxUuktn51xDJxfa27wxp+1uJ64zkJDpHTZQ1xeiyexHl8NvKM10orNXOqcrabUZw4ezHYMieG3x5Z71A0PzVSntVrr1xkWsC6PbESf49LMgkY\\\\\\\/GjBtjSNJrysZr4xNHUzFDCjOodSaSL46myx85x\\\\\\\/SXya1zHgH82Kl94fwary3xz41c\\\\\\\/5q9jEyXUf\\\\\\\/gyQPtvwWF\\\\\\\/TDJry8tOOx9HdAMLid52ngmeewpWm7BNitstEEJk13+sFo39avzgQ3\\\\\\\/HZtko4+LI2iLK8eHnCJfvB8xoh1Cl+i3qCFS\\\\\\\/iYclpW+fZMXdMToX+oLkg2qZiyzfoCiuvm8VrXBPJKtelRgN\\\\\\\/hXkNR0bfor\\\\\\\/CAh9i3rG2AcxRn0DMQvjmWfi4Ym5xlwh3YI\\\\\\\/uqGPXHjMLqpyO7pwXTdIAlGRnlo2LGetbrmpzJGeoUyP1xCLffvQlm5\\\\\\\/tnjQIFuJy6mHfCNKOhHn7VKADr3wjJXwkRR2xprhPyscUWC+3NVcVf4xaZjSnWTBOGGFdAFEtjZWjFub+T4ynPHMCtDnCjZSC+5NanoeCr9NCSaWAFBjXZibrPnnzDhoYGM80Xs4vgV8hI6W7CFm\\\\\\\/jDpiedPY\\\\\\\/ifQDKuL+H\\\\\\\/zDAzYaw+eQK7kGoZd8LWbZ3WWmRRMy\\\\\\\/x39RhdM98DdQ2fYlPb6CA0tqXo6p\\\\\\\/c8g2cERxfpIObZtQkI\\\\\\\/EnmbmKTy6nSyJjH83M5p+pni3yMZovuLK03UKMSvbWijHqupUtK8dcagj1WZ\\\\\\\/B6Ofsz5zxYo7xZjsucQIsuyZd2yhbAHWA+vIZ8E+dWciZF2934i8EVSk9OoDbaTTKfmIn9N9jjmmOxKHRuseQL8RgHxhtTdCpnwBZCZ+yBXDKrUxclzrYeGdHnharbL2mI9B0cpq5\\\\\\\/ZxQBbyaozEPbVz9NYnYQiIJ\\\\\\\/K1G\\\\\\\/xm5DYIBjwrX67OO9KCTjxC88h1rB8p1UWz0+e1jvlINrf609IZ5+ZVmg2yLpBiNJ\\\\\\\/Rqj0hPl9CeTuwGZAvlEMhvUY2LJIfDGXbjtYXc0lp14hY9gQo\\\\\\\/Ui0FkXnv\\\\\\\/eQDAGMUl7oJQn0\\\\\\\/h2y\\\\\\\/HskJ1+R6LmafCZN5Omr7eMsqjSw6flixcOZGvAgL6qrZdNp8IfWB5jGmf1R2NvlvU\\\\\\\/4Pd7tRa1PXVeeuRgJJ7E5I0T5XJenJ03wyE3xhFIR9RnOWEyeaO7jFPtmoPVj4nPXImuFe5sZeclPXOvkKVZWPoQki9icDiOBT0PMxQh6Tx5CkBdoIoyuuLV\\\\\\\/AM1A3mrSlMqVA41xtkk9PNvZFc8uSShM\\\\\\\/EY05sZj3KAzFdVro5aZv5FUh1XJbR4n04tClGEz4BBsDF8ikVagSkXisORF7sivo70NvDSezDzPZe0FJr2ZwrfNrayrzVbBwq0ZK3b5oTzazsH1BR6gSpmjiLEPUpIwpGvGmE0cshn3z4V0DLVuTTndJnVWuXNdn91p8nFNyfn3Xeg+2Fo8EwhjLqCDxgm2dGy0IVEKMDjaG1QdG3OYwYgti+ZMn8r7qBMjj0LpCg2acGHQYbbhiF\\\\\\\/XwnvTM\\\\\\\/tyoUXgUPewpGxwSF0DGRp2l57h5pV8IBKptKQ0TbCTPgQSGSFJMgzF6KadTxffnNsvZieTa8XN7JNNx3qP+dxQvwLupcXW\\\\\\\/VZGmOxipP7CdrO0PbOQvqW2bmEqp7BTrsIsYhXhEz663lUmFbC6TpQdSfMcvFV6rWnzvmrEQvlvH+IlWn70b7GilLUpsA0Iy\\\\\\\/iFOk2aAccZZuaFEb0UugyCD0J\\\\\\\/aZRWu79\\\\\\\/yDrenq6nGLSR+HDURhSp\\\\\\\/eLoKsQSDCEhx2qKblJdCWvuGUqWDlQdoRoItIADtXoBmWjR7LhRoF8R23tHL1Em5WLfnZt+vta2F\\\\\\\/SkaedTeYF3cFBQnDdhv+1fOxuiBFD07nZLPegYBC8J6IDIiWyI2Gs8gT2LFZKaQmwwChPltIw73a85sxt6Q4VdUUe5QonipOJOAVVzwOhMiDHxfIVrOEyP70Qth4pO+S4s2WFhsPzvctnwVCmr6BxIeedAMncsoyhkz8q\\\\\\\/52Jp5Rwll8n\\\\\\\/GkCVVetc9\\\\\\\/tiA7V8L7\\\\\\\/WJ9mB22k40g7vGXgiaYizvH1QdWUCsQJdemvvifUVMZI97uCSgR7paeaBxqhvl6MSmIUiDB7BLfav8m9eRV7Yo4VF6yUr2RlsbYQkfZiETR5Z86MwA7\\\\\\\/hXK1\\\\\\\/9nMI\\\\\\\/9f5ml+e6L7LyixqbfXgJLkhCKCsAER6TX7O8DsUHB11hs20IG9nPE5POjOJbP8\\\\\\\/seBS87BTKKrAwRO4q\\\\\\\/EcRJSLHKFZyoIanO8E1Pdyl2Cr8XBpGj3qT\\\\\\\/JgXUJd+S0APy\\\\\\\/3oXrl\\\\\\\/Fe1xGsBcaV95D1MTT9sxHrctNBow3APjFVxK6g+HJPfju0XHoUF9G3AAMv7PUWZUT3vq3wpn6+QKHlZNC\\\\\\\/GCVeN0LBq9K81gHjxHZ1DpORjxC7kyo\\\\\\\/Wzuk3PxeAn1kQBT3ObFDU24HRGDZmF+B\\\\\\\/j2HUe9RKcxrlsltSmqpTp81xuOjr\\\\\\\/9HRVrF3b3z6al54vYVxfsem8rjdhQNhreGMUXCErtij+iQSWCaE06Ezr2vDjCCf2QA2gmqWCQ9jsek7ytHCov745Ajow7BDIwaNZNOlCo3BWoSaDG5XP\\\\\\\/yQTCzlxIxZOjLRIdq4UfOmSG7He84lkYWSd\\\\\\\/Bg2+1lCvzAHfW9jYiDkoHtrQrwwiOnPJK0Ubpb1OfQiVrNLOFuca0N2XZJaieegM8CVpOOJHCHRQcbVZ01ZcwgyBf0GQnaidfj8AcglDursGG3pFtsdZj14wNXJmFmKLF30r7D9iRrh71BZWPS1bR4Joc\\\\\\\/E\\\\\\\/feeW15gMChHuwAn1EML9lyqBne7q5FhF1qn9lvW0R4Y74wxBnu2ct39sZ6XasdARPIWpr6uCRdMTxBRwNlR+6gt5UeVPINFcsBneyUyotvDw5bZCERMMNo7ihYaN5jzutRzUq0UstVUcMX0RaqBrYX7xHOqm6lzf3JgByQ8pcrNmUQhyByM3vhQB8VGC32s\\\\\\\/oLZulexclwLQDdJac0BD\\\\\\\/M1WWJ+Ml6Ppal3vcGTIX21H9aZeZhyWctxziiSyoMwk\\\\\\\/JT1oAVm326oMiyLTeY1rSYFByfFdWZ6RUOfjLAZzVRwt17ApUWW3ZGnyeQjaGs75MFuYZ0pKfacpqui7+0hrMSlUL++2JXQOnkawfa7A2kgoHZqw+YLU\\\\\\\/Van\\\\\\\/aHz2brN9yMP8EBRzLi\\\\\\\/AtYYunTjb1QPi20niRxmvIkQrxJl3RnDaiglDEPe6AZkLVRyD\\\\\\\/jhehJStWnZNMcD6TCRd2LmzgR5ZaxiDznY5GRSWTAo4EKWSqLl3gySk7a\\\\\\\/gVqH\\\\\\\/ig+3O0IUkHc01R2d+EwttMgGEWq+EVjJukz4V7n2V0NhokVaXF9fIbzayE+WQbJyf5Xl1nUuBo\\\\\\\/e9VQAQxEZ4BpED4WAYmQjF+OOJfC8phV7QCgeQ39cJoD6kjEM1o9BarP0fo9qs4vSfBTpdSOAW8ZraintnO4MdkcxPwi8qOW0s9Ou2gMo2ExqO4g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022KvcVzu+pKUP8hOjx0KMGNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ce665610395d1fe5ebcb948c6053eb2c219b0209c9bb943fbac62b1ef0fbbb1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aK7VA6KqTbRJIk1QpVt9zbjWtqPoSxkj\\\\\\\/LbaZJC\\\\\\\/NivG+Zf2HmBvFfi2iyr+K40PtdV+\\\\\\\/GCoAdswS6YrqXM0JpOtFa0jgieDLKjnrc4+0mxSpiiINfJ5o80bVB7fcviybz1JRiyxI\\\\\\\/f++vlwrS+zRS0QAGWbAUf0RdfvNPwmB0eo8cJu2kK9js60Qq\\\\\\\/+vdmfzkwPRWHAi0LlRl1KpoT1v3w1k1PuerARm3Ym2lwcgTh1wKz14nAT7HnibBJI5IzDMR52ZEgLrzYbyj1NTA4lz4+1Z7wtMW4w0RGsvjf5xQLWoPTaWSbgwhUUrSDGoA1AXHkg\\\\\\\/GEqLqL05pswkkukfwI5l6tICD9gja\\\\\\\/k1NgXE90XgfAh\\\\\\\/Yen0041PwWxXElaMX\\\\\\\/uqRxeiKBC0su18NKfuMYgKFVSjeMPhhZrG6\\\\\\\/hdcFZyVpyJg52xGbhavXl9RVwDkFTbYsONCzFT+IGyhZl2KGIJ4\\\\\\\/zvf9buqYrkilT4bACtK\\\\\\\/LNI+7Ojr+dhEI1Mdq4rhs0H+0TTLL++nJ\\\\\\\/vYPXrX\\\\\\\/YV7OVA0xWDlCWv5Rd5O\\\\\\\/V7O6so2anNa3c9SQPg95SegRdLfkiroGOjKQEGwDfnFCYck5xt28IHwYQHfulVIodKiZMP4T5n5I6xNvJBDn89dkfBBnZm70jPcd8c1s4TDD3XIWpG5EAw9DkAWTsim14vJBMk94elUvXBXMyQbN5AmKjm69TNEXJOKkVA8keFNNmNRJkph9R0Pnf0LqAfaIeYW+uenMgsjNVH3difpv01CpETnj9TabWmVwI5UBmh+EFmX516s6SJ6EUZyKIKMpVZMq6mY0a84XPU5d6T2pWj\\\\\\\/1jjhb9gPK7RoRbdgLlGY7lM5v3k6YgTxp2hRBdHcjXI5PpXz914QvCj\\\\\\\/w1A5T87V6pM7ZV76Vs11MjHGFxkgdEJLFXc36iB6YkPEhWwZukS8k8hB0iPDSlQOCcs7tFiXDwqHDuwYbcrfFdJBkWpXNDuvOB+QaT0GHx1ipB+fd0WBOzojzA+xsI134en8LnAP5a1ZEUDoYmRiO\\\\\\\/UGEOukG12QPLPSiksGoI0Bh2xLMyAStdAOfsfkQ+C9EMzqnvDkyH4VMEK\\\\\\\/TbxwjOmFhbEfDDTzOBZUSacMaQwSNZLcD3c39tXhugf3g5090Hv5CzfvHM7dd1NjhFb2jatsCtxBU9vQgbvI\\\\\\\/jWwHM1x6M84iPf835rzql99p5QANwoijcicJ90K+2dPJQgOpQtT5Y1wJdhwngd+nZnzyoDtk6f59Pt9owoO9GNZRw6oHYbL0jBl3Bj9S\\\\\\\/5QecPd19fWzTscbjdRdSuA6Z1PUwcxhGpL0worAxNTrgwIN\\\\\\\/hUCkp1NIKGqk1y3yUxZJ6GDLPNVv6nO462t25JVqDH4TNV2uCHc8vsDsrD\\\\\\\/VFh8A06oHcJx5v6RYsTFPgBlhvfOorN4esHRBkTcXPSrlVrPSo481V0hfvpwg3FGxyRR4XGxiaaXC8xx7cA94M7XGVEhCD3Rp+qUUa9fOVQX23YSMht2HX0l1PPJg9NXB9wAMO15SRojki\\\\\\\/EevUiKFVSJ55zzFjXyxuKOrE4L4Dv2Hy2xx1XyvehU0htT+uXBrZB41sTffbKXm2\\\\\\\/MDd50MliLrubtIIsClC887jbH3gANbtFBq2iO1zQTebYhAQZPXTGiVWMq2sWeX3V6ey83oF+zqTipmR69U\\\\\\\/UC4DiDo8YIkm\\\\\\\/G4eCqhPVV1IEXKXf8KYK8ablJhJv2MZvTXDi0k\\\\\\\/keIvUCjhycldOdC07WEkbKlvl0TDYJi6YbECZ02+A7SFHeIs8WU6MRJ4x+whZCsgapl6z3QAthpUqAmEvvdtlAw7NX2FgwUa310uZnAY32Tl2fauLCWIwXAzYCzzdrIX3x67flocLUf3DWhFACmADNnEuZPhuBlOWaC46rs++5O6hI0nRLwwkNf7YGAYLYL4UBvEFZZC+Ee7g92getguLa0o2M7MPFWVQRNKTjBV1daiAnZjpshsdfgyPC8Lfe6zBgVYMN4488A==\\\u0022,\\\u0022iv\\\u0022:\\\u00222LRPkLyHrKD+e7eK52UBVQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022914e53f358df9751675498e48cb765b81aadcdcf053a188501c2693117f5ac22\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488753 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/999" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-y2ubg_WJ7n5K--EkpKM3QdyjX1I0G35mkgmOAmX1vI4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/999" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AZ-pix1-h7uV92zQeooMs88GecGEaKlMXS6Yy7-Ss2Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/3/981" title="Genomic Identification and Functional Characterization of Essential Genes in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/3/1019" title="Functional Analysis of Cancer-Associated DNA Polymerase ε Variants in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488753" id="node488753" data-pisa="ggg;8/3/999" data-pisa-master="ggg;g3.117.300557" data-apath="/ggg/8/3/999.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tuning CRISPR-Cas9 Gene Drives in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Emily Roggenkamp</span>, <span class="highwire-citation-author" data-delta="1">Rachael M. Giersch</span>, <span class="highwire-citation-author" data-delta="2">Madison N. Schrock</span>, <span class="highwire-citation-author" data-delta="3">Emily Turnquist</span>, <span class="highwire-citation-author" data-delta="4">Megan Halloran</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-5710-173X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory C. Finnigan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">999-1018;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300557 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488753" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emily Roggenkamp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BRoggenkamp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roggenkamp%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BRoggenkamp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rachael M. Giersch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachael%2BM.%2BGiersch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giersch%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachael%2BM.%2BGiersch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Madison N. Schrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madison%2BN.%2BSchrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schrock%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadison%2BN.%2BSchrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emily Turnquist </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BTurnquist%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turnquist%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BTurnquist%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Megan Halloran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BHalloran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halloran%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMegan%2BHalloran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gregory C. Finnigan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BC.%2BFinnigan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Finnigan%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BC.%2BFinnigan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5710-173X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory C. Finnigan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gfinnigan@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gfinnigan@ksu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/3/999" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488753" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/999.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488753" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/999.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488753" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/999.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488753" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/3/999.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488753" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-74336 first last"><a href="/content/8/10/3383" class="" data-icon-position="" data-hide-link-title="0">CORRIGENDUM - October 01, 2018</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Control of biological populations is an ongoing challenge in many fields, including agriculture, biodiversity, ecological preservation, pest control, and the spread of disease. In some cases, such as insects that harbor human pathogens (<em>e.g.</em>, malaria), elimination or reduction of a small number of species would have a dramatic impact across the globe. Given the recent discovery and development of the CRISPR-Cas9 gene editing technology, a unique arrangement of this system, a nuclease-based “gene drive,” allows for the super-Mendelian spread and forced propagation of a genetic element through a population. Recent studies have demonstrated the ability of a gene drive to rapidly spread within and nearly eliminate insect populations in a laboratory setting. While there are still ongoing technical challenges to design of a more optimal gene drive to be used in wild populations, there are still serious ecological and ethical concerns surrounding the nature of this powerful biological agent. Here, we use budding yeast as a safe and fully contained model system to explore mechanisms that might allow for programmed regulation of gene drive activity. We describe four conserved features of all CRISPR-based drives and demonstrate the ability of each drive component—Cas9 protein level, sgRNA identity, Cas9 nucleocytoplasmic shuttling, and novel Cas9-Cas9 tandem fusions—to modulate drive activity within a population.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/crispr" class="hw-term hw-article-keyword hw-article-keyword-crispr" rel="nofollow">CRISPR</a></li><li class="kwd"><a href="/keyword/cas9" class="hw-term hw-article-keyword hw-article-keyword-cas9" rel="nofollow">Cas9</a></li><li class="kwd"><a href="/keyword/budding-yeast" class="hw-term hw-article-keyword hw-article-keyword-budding-yeast" rel="nofollow">budding yeast</a></li><li class="kwd"><a href="/keyword/gene-drive" class="hw-term hw-article-keyword hw-article-keyword-gene-drive" rel="nofollow">gene drive</a></li><li class="kwd"><a href="/keyword/sgrna" class="hw-term hw-article-keyword hw-article-keyword-sgrna" rel="nofollow">sgRNA</a></li><li class="kwd"><a href="/keyword/regulating-gene-drives" class="hw-term hw-article-keyword hw-article-keyword-regulating-gene-drives" rel="nofollow">regulating gene drives</a></li><li class="kwd"><a href="/keyword/biotechnology" class="hw-term hw-article-keyword hw-article-keyword-biotechnology" rel="nofollow">biotechnology</a></li></ul><p id="p-4">The issue of biological control over the population, spread, and nature of a given species has been a monumental challenge that has major implications for agriculture, human health, and the environment. Many metazoans, including insects, are carriers for human pathogens such as malaria. Plant, fungal, and insect species are also pests that can affect planting, pollination, harvesting, transportation, or storage of agricultural products. Finally, the introduction of invasive species (accidental or malicious) into a nonnative environment can disrupt ecological food chains, impact biodiversity, or inflict damage on human infrastructure. Current measures to combat these widespread issues include a full range of solutions, such as chemical treatment (and genetically modified crops), physical barriers, and natural predators, and each mechanism has its benefits and limitations (<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Stenberg <em>et al.</em> 2015</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Kergunteuil <em>et al.</em> 2016</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Khan <em>et al.</em> 2016</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Peterson <em>et al.</em> 2016</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Jonsson <em>et al.</em> 2017</a>). However, in situations that seem to call for the near eradication of an entire single species type (<em>e.g.</em>, insect genus <em>Anopheles</em> as a vector for the malaria-causing alveolate parasites from genus <em>Plasmodium</em>, or the tsetse fly from genus <em>Glossina</em> for sleeping sickness caused by protozoa genus <em>Trypanosoma</em>), no solution has been developed with the ability to selectively control populations.</p><p id="p-5">The discovery of the CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) gene editing system has revolutionized the entire fields of molecular and cellular biology and genetics, providing a new level of control over the genomes of all living species. Briefly, the Cas9 nuclease from <em>Streptococcus pyogenes</em> was identified as a critical part of a bacterial immune system used to identify, bind, and destroy invading bacteriophage DNA sequences (<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Sorek <em>et al.</em> 2013</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Mohanraju <em>et al.</em> 2016</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Koonin <em>et al.</em> 2017</a>). The CRISPR system was recognized for its biotechnological application as a programmable genome editor: given a particular DNA sequence, a double-stranded break (DSB) could be introduced at a precise chromosomal location. The Cas9 protein is first primed by a single guide RNA (sgRNA) fragment containing a CRISPR RNA (crRNA) sequence (typically 20 bp) that contains the corresponding bases to the intended genomic target fused to the tracer RNA (tracrRNA) sequence, a species-specific structural nucleic fragment that complexes with the Cas9 nuclease (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Jinek <em>et al.</em> 2012</a>). The bound Cas9/sgRNA complex then scans DNA for a feature only present within the target nucleotide sequence, called the protospacer adjacent motif (PAM). For <em>S. pyogenes</em> Cas9, the sequence 5′-NGG-3′ found at the 3′ end of the intended target sequence allows the nuclease to create a DNA DSB either <em>in vitro</em> or <em>in vivo</em> (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Jinek <em>et al.</em> 2012</a>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">2014</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Nishimasu <em>et al.</em> 2014</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Jiang <em>et al.</em> 2016</a>). Following cleavage, eukaryotic cells have evolved several DNA repair pathways that activate to fix the Cas9-induced break including nonhomologous end joining (NHEJ) or, in the presence of exogenous donor DNA, homology directed repair (HDR). The combination of Cas9-targeted genomic breaks coupled with native repair systems allows for precise editing (deletion, disruption, replacement) of nearly any genomic sequence. The power and potential of the CRISPR-Cas9 editing system cannot be overstated: a myriad of research, biotechnology, agricultural, and medical industries in both the public and private sectors have begun applying this genomic editing system to many current problems in biology and human health (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Doudna and Charpentier 2014</a>; <a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Sternberg and Doudna 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barrangou and Doudna 2016</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Estrela and Cate 2016</a>; <a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">Wright <em>et al.</em> 2016</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baltes <em>et al.</em> 2017</a>).</p><p id="p-6">One unique arrangement of Cas9 has garnered a lot of interest in recent years, the nuclease based “gene drive.” Briefly, both the Cas9 gene and the expression cassette for the sgRNA are integrated at a particular genomic locus (usually disrupting and replacing a given gene). The nuclease function of Cas9 is unique within the context of a gene drive because the intended target for the guide RNA is the WT copy of the gene that has been disrupted by placement of the drive-expression cassette (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">DiCarlo <em>et al.</em> 2015</a>). Upon induction of a DSB within a diploid cell, the homologous chromosome pair serves as the DNA “donor” and, via HDR, will copy the entire gene drive (Cas9 and its sgRNA) to the second chromosome, replacing and deleting the WT copy of the target gene in the process. This results in the transformation of every heterozygous pairing to be converted to a homozygous diploid state; “forced” propagation of a desired loss of function allele through a population is super-Mendelian in nature as it defies the basic mechanisms of heredity. In this way, the introduction of a single affected individual harboring one copy of the gene drive can rapidly sweep through a population in only a few generations. The identity of the targeted gene(s) may differ (early renditions have used a biased male/female sex ratio to reduce population sizes), but the goal is the same: the preprogrammed, automated destruction of the entire population of (only) the given species. This potent biological agent would, theoretically, have the potential to rapidly introduce a given allele of choice to modify millions (or billions) of individuals for any intended purpose with no external input. To date, only a few laboratories (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Gantz <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Hammond <em>et al.</em> 2016</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Champer <em>et al.</em> 2017</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Drury <em>et al.</em> 2017</a>) have demonstrated use of a CRISPR gene drive in insects and a single study piloted the technology in yeast (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">DiCarlo <em>et al.</em> 2015</a>). There are still many technical hurdles to overcome in the design and application of gene drives into wild populations, such as evolved resistance to the action of the drive, genetic diversity within populations, and containment/reversal of drive systems. Furthermore, there are serious ecological and ethical considerations regarding the issue of actual use of gene drives to modify (or eradicate) native populations, regardless of the intended purpose (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akbari <em>et al.</em> 2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Esvelt and Gemmell 2017</a>). Therefore, we have focused our efforts on an important, and currently overlooked aspect of gene drives for future implementation and application in the wild: a programmable and tunable drive system that would result in a full spectrum of drive efficiencies (from 0 up to 99%) within a population.</p><p id="p-7">In this study, we have developed an artificial genetic system in budding yeast to (i) provide maximum containment and biosafety, (ii) rapidly explore and test many aspects of the CRISPR editing system using the available molecular tools within <em>Saccharomyces cerevisiae</em>, and (iii) identify and characterize sources within the gene drive system for future control and regulation. We describe the study of conserved gene drive components regardless of the organism of choice: the Cas9 nuclease, its subcellular localization, and the sgRNA. We demonstrate that the following independent mechanisms can all modulate drive activity to varying degrees: (i) the level of Cas9 (using an inducible promoter), (ii) the sgRNA length and mismatch to the target sequence, (iii) nucleocytoplasmic shuttling of Cas9 using localization signals, and (iv) tandem fusions between Cas9(s) and/or a nuclease dead (dCas9) variant. Moreover, these alterations can be done in combination resulting in a wide spectrum of gene drive penetrance within a population from zero activity to maximal activity. These findings represent an important step in the future design, regulation, and possible application of gene drives in the wild.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and plasmids</h3><p id="p-8"><em>S. cerevisiae</em> strains used in this study are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Standard molecular and cellular biology procedures were used to manipulate all plasmids and yeast strains (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Sambrook and Russell 2001</a>). The Cas9 gene was synthesized <em>de novo</em> with a yeast codon bias (Genscript, Piscataway, NJ). The second Cas9* gene used for the tandem fusions was also synthesized after manual manipulation of each codon to an alternate codon (primarily within the Wobble position). Enzymatically dead Cas9 (D10A H840A) was generated by a modified PCR mutagenesis protocol (<a id="xref-ref-126-1" class="xref-bibr" href="#ref-126">Zheng <em>et al.</em> 2004</a>) on the pUC57-based plasmid(s) harboring the Cas9 gene using a high-fidelity DNA polymerase (KOD Hot Start; EMD Millipore). The general strategy for integration of Cas9 (or the target gene cassette) into the yeast genome was as follows. First, a CEN-based (pRS316) plasmid was generated by <em>in vivo</em> plasmid assembly (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Finnigan and Thorner 2015</a>) including the <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a>/10</em> promoter (814 bp), Cas9 open reading frame (ORF), a C-terminal NLS signal (SRADPKKKRKV), the <em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em> terminator (238 bp) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bennetzen and Hall 1982</a>), and the MX-based kanamycin resistance cassette (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Longtine <em>et al.</em> 1998</a>). Second, the assembled Cas9 gene cassette was PCR amplified and a second round of <em>in vivo</em> ligation was performed using a second vector (pGF-IVL974) to insert 992 bp of <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> 5′ UTR, 993 bp 3′ UTR, and two (u2) sites (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>) upstream of the <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a>/10</em> promoter and downstream of the MX terminator. Third, the entire ensemble was amplified in two fragments (generating 120 bp of overlapping sequence within the Cas9 ORF), treated with <em>Dpn</em>I enzyme, and transformed into BY4741 yeast using a modified lithium acetate protocol (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Eckert-Boulet <em>et al.</em> 2012</a>) for integration at the native <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus (<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a></em>∆<em>1</em>). Colonies resistant to G418 sulfate (and lacking the selectable marker for the yeast vector used as a PCR template) were tested by diagnostic PCR and Sanger DNA sequencing (Genscript) to generate GFY-2383.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488831/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488831/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Yeast strains used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">DNA plasmids used in this study are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. <em>In vivo</em> plasmid assembly was used to construct all Cas9 and gene target vectors including those used for integration into the genome. Plasmids expressing the sgRNA cassette were created from previous vectors (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">DiCarlo <em>et al.</em> 2013</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>). Briefly, the <em><a href="http://www.yeastgenome.org/locus/S000006443/overview">SNR52</a></em> promoter (269 bp), crRNA sequence (16–22 bp), tracrRNA sequence (79 bp), and <em><a href="http://www.yeastgenome.org/locus/S000006782/overview">SUP4</a></em> terminator (20 bp) were synthesized <em>de novo</em> (Genscript), subcloned into the pRS423, pRS425, or pRS426 (high-copy) plasmids (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Christianson <em>et al.</em> 1992</a>), and mutagenized by PCR to generate all sgRNA variants. All vectors were confirmed by Sanger DNA sequencing (Genscript).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488901/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488901/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Plasmids used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Culture conditions</h3><p id="p-27">Yeast were propagated in solid or liquid medium including YPD (2% peptone, 1% yeast extract, 2% dextrose) or synthetic media containing a nitrogen base, ammonium sulfate, and all necessary amino acid supplements. Preinduction medium included a 2% raffinose and 0.2% sucrose mixture. For experiments requiring induction of the <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a>/10</em> promoter, liquid media containing 2% galactose was used. All sugars were filtered sterilized rather than autoclaved. The nomenclature for synthetic media is as follows: “S” refers to synthetic complete (<em>e.g.</em>, SD-URA, synthetic complete plus dextrose minus uracil).</p></div><div id="sec-4" class="subsection"><h3>CRISPR-Cas9-based editing</h3><p id="p-28">The mCAL system (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>) was used for all Cas9 editing in haploid yeast and in diploid gene drive–containing strains. Briefly, this system harnesses two artificial, programmed Cas9 target sequences (u1 and u2) that contain a maximum mismatch to the <em>S. cerevisiae</em> genome. Two identical (u1) sites flank the gene “target” locus whereas two identical (u2) sites flank the Cas9 gene cassette itself. Both the (u1) and (u2) sites within the genome also contain the 5′-NGG-3′ PAM required for <em>S. pyogenes</em> Cas9. All sgRNAs were designed to target either the (u1) or (u2) artificial sites. For plasmid-driven Cas9, the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em>-based vector (<em>e.g.</em>, pGF-IVL1116) was transformed into the appropriate yeast strain prior to editing for several reasons: (i) growth on dextrose repressed Cas9 expression, (ii) rapid counterselection of the vector could be achieved on media containing 5-FOA, and (iii) consistent propagation of the plasmid could be maintained prior to introduction of the sgRNA-expressing plasmid.</p><p id="p-29">Activation of Cas9 and gene editing in haploid yeast was performed as previously described (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>). Strains harboring the Cas9 vector were cultured overnight in a raffinose/sucrose mixture at 30° to saturation. Next, yeast were back-diluted into a YP + galactose mixture to an OD<sub>600</sub> of ∼0.35 OD/ml and grown for an additional 4.5 hr. Cells were harvested and transformed with 1000 ng of sgRNA-containing plasmid, heat-shocked for 0.75 hr at 42°, incubated in YP + galactose overnight (∼16 hr) without shaking, spread onto SD-URA-LEU plates, and incubated for 3–4 d prior to imaging. The number of colonies was quantified using a single-blind protocol (researchers who were counting were unaware of the genotype of each plate) and a sectoring method (when appropriate). Several random fractions (one-quarter or one-eighth, <em>etc</em>.) were sampled and averaged to estimate the total number of colonies. For plates containing &lt;500 colonies, the entire plate was quantified. To assess the drug resistance of individual colonies, 50–200 colonies were randomly selected and transferred to an identical plate type as a small patch (to increase the surface area) and incubated for 1–2 additional days. Next, plates (each containing between 50 and 100 colony patches) were replica-plated using sterile velvet cloths to rich medium containing dextrose and hygromycin (300 μg/ml) or G418 (200 μg/ml). Assessment of the <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus was performed by reselecting clonal isolates, preparing purified chromosomal DNA, and PCR amplification and DNA sequencing (when appropriate).</p></div><div id="sec-5" class="subsection"><h3>Gene drives and containment</h3><p id="p-30">Cas9 gene drives were constructed and manipulated using the following protocol. First, the Cas9-expression cassette was integrated at the <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus and maintained on dextrose (to repress the <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a>/10</em> promoter). Second, yeast were transformed with the sgRNA(u1) plasmid; since the target (u1) sequence does not exist within <em>S. cerevisiae</em>, editing is halted even if Cas9 was present and primed with the guide RNA. Third, haploid yeast expressing the sgRNA were mated to the gene “target” containing strain (harboring (u1) sites) of the opposite mating type on rich media containing dextrose for 24 hr at 30°. Fourth, yeast were transferred to SD-LEU-HIS plates using sterile velvet cloths to select for diploid yeast and incubated for 24 hr at 30°. The diploid selection step was repeated a second (or sometimes third) time on the same media type. The choice of the <em>Schizosaccharomyces pombe HIS5</em> gene within the target genome to select for diploids ensures any (rare) promiscuous Cas9 that may have prematurely been activated will be destroyed on SD-LEU-HIS medium. Fifth, diploid yeast were cultured overnight in synthetic medium containing raffinose, sucrose, and lacking leucine to saturation. Sixth, cells were back-diluted to an OD<sub>600</sub> of ∼0.35 OD/ml in YP medium containing galactose for 0–24 hr at 30°, depending on the time course. Following induction of Cas9, yeast were harvested, washed with water once, and diluted to a density of ∼1000 cells/ml. Between 250 and 1000 cells were spread onto SD-LEU plates and incubated for 2–3 d. Yeast were transferred to a fresh SD-LEU plate and SD-HIS plate using sterile velvet cloths and incubated for 24 hr prior to imaging. The number of colonies sensitive on SD-HIS medium was quantified (over total number of colonies on SD-LEU) to obtain the percentage of active drives within a given genotype. Analysis of individual colonies was done by reselecting yeast on SD-LEU medium and testing individual clonal isolates for growth on various media types, ploidy status (by mating to <em>MAT</em><strong>a</strong> and <em>MAT</em><strong>⍺</strong> control strains), and chromosomal DNA isolation for diagnostic PCRs and DNA sequencing. For gene drives harboring two plasmids (<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em>-marked), the procedure included diploid selection on SD-URA-LEU-HIS plates, and final testing on SD-URA-LEU to maintain the presence of both plasmids.</p><p id="p-31">A number of safeguards were included to ensure the safe, ethical, and contained use of all yeast strains harboring the (potentially) active CRISPR gene drive arrangement. First, the most powerful safeguard includes the use of programmed target DNA site (u1) at the <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus that does not exist within the native budding yeast genome (<a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>). This sequence has a maximum mismatch to any other sequence within <em>S. cerevisiae</em> reducing the possibility of inappropriate editing (off-target effects) and virtually eliminating the possibility of the drive to propagate within a wild yeast population of any strain type or related species. Second, all haploid gene drive strains [containing Cas9 and the (u1)-targeting sgRNA plasmid] were grown on dextrose to repress transcription of Cas9. Within the diploid strain, Cas9 was only activated for a limited amount of time (0–24 hr). Third, the sgRNA to target the (u1) sequence was exclusively maintained on a high-copy yeast plasmid (pRS425). Previous work (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">DiCarlo <em>et al.</em> 2015</a>) has suggested that separation of the guide sequence from the Cas9 gene can provide an additional safeguard. Here, we have also documented the rapid loss of sgRNA(u1) plasmid from diploid yeast in the absence of any selective pressure (Supplemental Material, Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). Fourth, all the Cas9-expression cassettes are also flanked by the artificial (u2) site. This serves as a specific safeguard to exactly excise the entire Cas9 gene and associated sequence from any haploid or diploid genome in a single step. We demonstrate that introduction (by direct transformation or mating via a strain of the opposite mating type) of the (u2) sgRNA plasmid causes removal and destruction of the existing drive system (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). All gene drive strains were constructed with this (u2) system in place. Fifth, the laboratory diploid strain (BY4741/BY4742) has been previously documented to be very inefficient at sporulation, even under optimal conditions that induce meiosis and spore formation (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Heasley and McMurray 2016</a>). Sixth, careful destruction of all haploid and diploid yeast immediately following experimentation was performed (including capture of all washes of glassware for autoclaving). All materials used (tubes, velvet cloths, pipette tips, plates, wooden sticks, liquid cultures, <em>etc</em>.) were autoclaved at &gt;121° for at least 0.75 hr before disposal or reuse. Diploid yeast strains containing gene drive systems were all destroyed (not frozen) after experimentation.</p></div><div id="sec-6" class="subsection"><h3>Fluorescence microscopy</h3><p id="p-32">Yeast were grown overnight in a preinduction culture containing raffinose and sucrose to saturation and back-diluted into YP + galactose for 4.5 hr. Cells were harvested, washed with water, prepared on a microscope slide with a coverslip, and imaged within 15 min. Yeast were imaged on an inverted Leica DMI6500 fluorescence microscope (Leica Microsystems Inc., Buffalo Grove, IL) with a 100× objective lens, and fluorescence filters (Semrock, GFP-4050B-LDKM-ZERO and mCherry-C-LDMK-ZERO). A Leica DFC340 FX camera, Leica Microsystems Application Suite AF software, and ImageJ (National Institutes of Health) were used to process all images. All images were obtained using identical exposure times and were processed and rescaled together. The “merged” images were generated in ImageJ and do not contain any additional processing from the GFP/mCherry images. The yeast cell periphery (and the yeast vacuole) was determined using the DIC image. Representative cells were chosen for each image.</p></div><div id="sec-7" class="subsection"><h3>Data and reagent availability</h3><p id="p-33">Upon request, we will freely send renewable research materials (yeast strains and plasmids) and reagents that were created during the course of this study for research purposes (not-for-profit institutions). The exception includes diploid yeast strains harboring active (guide RNA plasmid) gene drive configurations (all were destroyed)—haploid parental strains will be provided instead.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>An artificial system for Cas9-based gene editing in budding yeast</h3><p id="p-34">The goal of this study was to identify aspects of CRISPR gene editing that have the potential to modify, in a predictable manner, the activity and effectiveness of gene drives. We envision that the molecular mechanisms we identify and study would have direct relevance and application to CRISPR drive systems in other organisms for various reasons. Given the recent interest and potential applications of nuclease-based gene drives across numerous industries, we sought to develop a safe, programmable system in <em>S. cerevisiae</em> to investigate control of Cas9 editing <em>in vivo</em>. Our methodology includes several unique aspects that (i) remove concerns of off-target effects, (ii) minimize the number of sgRNA constructs required, (iii) allow for an unbiased assessment of editing (no selective pressure for or against an editing event), (iv) allow for full excision of a target marker (rather than rely on disruption of a coding sequence by a single targeted event), and (v) provide a potent genetic safeguard for use within a gene drive system (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). We developed a “target” strain of yeast flanked by two unique artificial sequences, termed (u1). We have previously described use and application of placement of these unique sites throughout the genome to allow for multiplexing with only one sgRNA construct and Cas9 (<a id="xref-ref-38-8" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>). Since these programmed (u1) sequences do not exist within the yeast genome (and provide a maximum mismatch to the closest native sequences), this reduces any possibility of off-target effects, or biased results based on similar gene target(s) or repetitive sequences. Moreover, our design includes two (u1) sites flanking a selectable marker (conferring hygromycin or G418 resistance). This is in stark contrast to numerous other Cas9 editing assays (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">DiCarlo <em>et al.</em> 2013</a>, <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">2015</a>), which target the coding sequence of a given gene (or marker) and depend upon disruption of the final protein product to provide a detectible growth phenotype. This approach includes several issues that might arise from “traditional” targeting with a single sgRNA construct, opposed than our system of a “clean” excision of the entire physical gene. First, since disruption of the gene (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/locus/S000000789/overview">CAN1</a></em>) product allows for growth (on medium containing canavanine), there would be selective pressure to perform either NHEJ or HDR and allow for cell survival. Second, while traditional red/white colony color screening has been a very useful genetic tool for screening (adenine biosynthesis), the presence of the red pigment is slightly toxic to cells (<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Ugolini and Bruschi 1996</a>). Third, editing events that faithfully repair the DNA break without including an insertion or deletion would be virtually undetectable as there would be no change in sequence following repair via NHEJ. Fourth, use of two distinct sgRNAs to target two positions at a genomic locus of interest would require different target sequences, and this may allow for preference or bias to one cut site. Our novel system controls for any difference in target sequence since all targets are identical (and allow for up to 22 bp targeting with a 3 bp PAM); this is the first setup to provide two identical Cas9 cut sites flanking a selectable marker that imposes no native selective pressure for or against said marker, since selection for the target is not assayed until after the editing event. This is an important technical difference between our system and traditional Cas9 editing approaches. Successful targeting of Cas9 to both sites, introduction of two DSBs, and subsequent repair of the broken DNA by NHEJ would recreate a single (u1) site from fusion of the two flanking “half sites,” along with full excision of the marker gene (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). In this way, we would be able to detect Cas9 editing of a single sequence in the absence of any generated indel by NHEJ.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A safe, programmable system to test CRISPR-based gene editing in haploid yeast. (A) Our design for a yeast system for analysis of CRISPR editing includes (i) an inducible S. pyogenes Cas9 expressed from a URA3-based plasmid, (ii) a sgRNA expression cassette on a high-copy LEU2-based plasmid, and (iii) a programmable gene &#x201C;target&#x201D; (consisting of a drug resistance marker cassette) at a safe-harbor locus (HIS3) flanked by two &#x201C;unique&#x201D; DNA sequences (u1) that do not exist within the S. cerevisiae genome (Finnigan and Thorner 2016). Induction of Cas9 allows targeting and double-stranded break formation at the identical (u1) sequences. In the absence of exogenous DNA (e.g., amplified PCR product) to be used for HDR, NHEJ of the exposed chromosomal ends causes full excision of the selectable marker. However, given the unique arrangement of the identical (u1) sites, NHEJ in the absence of any insertion/deletion mutation at the Cas9 cut site (left) recreates another WT (u1) site and subsequent re-editing of the same target sequence until Cas9 expression is shutoff or a mutation is positioned within the (u1) site (right). (B) Cas9-dependent editing results in cell inviability. GFY-2353 yeast already harboring Cas9-NLS on a vector (pGF-IVL1116) or an empty vector control (pRS316) were induced in medium containing galactose, transformed with the sgRNA(u1)-expression cassette on either a CEN-based (pGF-V1215) or 2&#x3BC;-based (pGF-V1220) plasmid, and plated onto SD-URA-LEU media. (C) GFY-2588 yeast containing pGF-IVL1342 were transformed with sgRNA(u1) plasmid (pGF-V1216) and selected on SD-URA-LEU medium. The isolated chromosomal DNA of individual clonal (surviving) isolates was assayed by PCR using DNA oligonucleotides (F1/R1, Table S1 in File S1) to the flanking HIS3 UTR. The expected product sizes of the amplified PCR fragments are &#x223C;379 bp (depending on the type of insertion/deletion(s) at the cut site, if any), or 1839 bp in the absence of any editing. Colonies were tested for resistance on medium containing G418 (below). (D) Clonal isolates from Cas9 editing (a dozen independent experiments) using the high copy sgRNA(u1) plasmid from (B) and that had also excised the selection cassette were analyzed by DNA sequencing at the HIS3 locus. The number of each genotype obtained is listed (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A safe, programmable system to test CRISPR-based gene editing in haploid yeast. (A) Our design for a yeast system for analysis of CRISPR editing includes (i) an inducible &lt;em&gt;S. pyogenes&lt;/em&gt; Cas9 expressed from a &lt;em&gt;URA3&lt;/em&gt;-based plasmid, (ii) a sgRNA expression cassette on a high-copy &lt;em&gt;LEU2&lt;/em&gt;-based plasmid, and (iii) a programmable gene &#x201C;target&#x201D; (consisting of a drug resistance marker cassette) at a safe-harbor locus (&lt;em&gt;HIS3&lt;/em&gt;) flanked by two &#x201C;unique&#x201D; DNA sequences (u1) that do not exist within the &lt;em&gt;S. cerevisiae&lt;/em&gt; genome (Finnigan and Thorner 2016). Induction of Cas9 allows targeting and double-stranded break formation at the identical (u1) sequences. In the absence of exogenous DNA (&lt;em&gt;e.g.&lt;/em&gt;, amplified PCR product) to be used for HDR, NHEJ of the exposed chromosomal ends causes full excision of the selectable marker. However, given the unique arrangement of the identical (u1) sites, NHEJ in the absence of any insertion/deletion mutation at the Cas9 cut site (left) recreates another WT (u1) site and subsequent re-editing of the same target sequence until Cas9 expression is shutoff or a mutation is positioned within the (u1) site (right). (B) Cas9-dependent editing results in cell inviability. GFY-2353 yeast already harboring Cas9-NLS on a vector (pGF-IVL1116) or an empty vector control (pRS316) were induced in medium containing galactose, transformed with the sgRNA(u1)-expression cassette on either a CEN-based (pGF-V1215) or 2&#x3BC;-based (pGF-V1220) plasmid, and plated onto SD-URA-LEU media. (C) GFY-2588 yeast containing pGF-IVL1342 were transformed with sgRNA(u1) plasmid (pGF-V1216) and selected on SD-URA-LEU medium. The isolated chromosomal DNA of individual clonal (surviving) isolates was assayed by PCR using DNA oligonucleotides (F1/R1, Table S1 in File S1) to the flanking &lt;em&gt;HIS3&lt;/em&gt; UTR. The expected product sizes of the amplified PCR fragments are &#x223C;379 bp (depending on the type of insertion/deletion(s) at the cut site, if any), or 1839 bp in the absence of any editing. Colonies were tested for resistance on medium containing G418 (below). (D) Clonal isolates from Cas9 editing (a dozen independent experiments) using the high copy sgRNA(u1) plasmid from (B) and that had also excised the selection cassette were analyzed by DNA sequencing at the &lt;em&gt;HIS3&lt;/em&gt; locus. The number of each genotype obtained is listed (right).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F1.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/3/999/F1.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488812" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-35" class="first-child">A safe, programmable system to test CRISPR-based gene editing in haploid yeast. (A) Our design for a yeast system for analysis of CRISPR editing includes (i) an inducible <em>S. pyogenes</em> Cas9 expressed from a <em>URA3</em>-based plasmid, (ii) a sgRNA expression cassette on a high-copy <em>LEU2</em>-based plasmid, and (iii) a programmable gene “target” (consisting of a drug resistance marker cassette) at a safe-harbor locus (<em>HIS3</em>) flanked by two “unique” DNA sequences (u1) that do not exist within the <em>S. cerevisiae</em> genome (<a id="xref-ref-38-9" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>). Induction of Cas9 allows targeting and double-stranded break formation at the identical (u1) sequences. In the absence of exogenous DNA (<em>e.g.</em>, amplified PCR product) to be used for HDR, NHEJ of the exposed chromosomal ends causes full excision of the selectable marker. However, given the unique arrangement of the identical (u1) sites, NHEJ in the absence of any insertion/deletion mutation at the Cas9 cut site (left) recreates another WT (u1) site and subsequent re-editing of the same target sequence until Cas9 expression is shutoff or a mutation is positioned within the (u1) site (right). (B) Cas9-dependent editing results in cell inviability. GFY-2353 yeast already harboring Cas9-NLS on a vector (pGF-IVL1116) or an empty vector control (pRS316) were induced in medium containing galactose, transformed with the sgRNA(u1)-expression cassette on either a CEN-based (pGF-V1215) or 2μ-based (pGF-V1220) plasmid, and plated onto SD-URA-LEU media. (C) GFY-2588 yeast containing pGF-IVL1342 were transformed with sgRNA(u1) plasmid (pGF-V1216) and selected on SD-URA-LEU medium. The isolated chromosomal DNA of individual clonal (surviving) isolates was assayed by PCR using DNA oligonucleotides (F1/R1, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>) to the flanking <em>HIS3</em> UTR. The expected product sizes of the amplified PCR fragments are ∼379 bp (depending on the type of insertion/deletion(s) at the cut site, if any), or 1839 bp in the absence of any editing. Colonies were tested for resistance on medium containing G418 (below). (D) Clonal isolates from Cas9 editing (a dozen independent experiments) using the high copy sgRNA(u1) plasmid from (B) and that had also excised the selection cassette were analyzed by DNA sequencing at the <em>HIS3</em> locus. The number of each genotype obtained is listed (right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Yeast containing our (u1) flanked target gene and an inducible <em>S. pyogenes</em> Cas9 (<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a>/10</em> promoter) were transformed with either control (empty) plasmids or sgRNA(u1)-expressing plasmids (low or high copy) in the presence of galactose (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Importantly, the subsequent selection step following introduction of the guide RNA plasmid and activation of Cas9 (located on a separate plasmid) was for the presence of both vectors. In the absence of any donor DNA fragment to repair the edited <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus, yeast activate the NHEJ pathway and repair the cleaved DNA together (without our selectable marker). In the presence of both the sgRNA(u1) guide and active Cas9, only a small number of surviving colonies were obtained; this is consistent with previous studies in budding yeast using <em>S. pyogenes</em> Cas9 (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">DiCarlo <em>et al.</em> 2013</a>; <a id="xref-ref-38-10" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>). Following introduction of a DSB, the NHEJ system allows for precise repair of the severed (u1) sites. However, NHEJ with no alteration of sequence would result in a functional target (u1) DNA site and a second round of Cas9-dependent cleavage. Consecutive rounds of editing followed by repair occur during treatment with galactose and, given the excess of Cas9 available per cell, results in few cells that are viable once plated onto selective media. (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). In this way, general cell viability serves as a potent selection mechanism for assaying Cas9 editing in yeast. When individual surviving clones were tested for drug resistance following editing, the majority had fully excised the gene cassette as assayed by diagnostic PCRs of the <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). Finally, DNA sequencing of a large pool of isolated clones yielded a diverse assortment of insertions and deletions at the “repaired” (u1) site following editing by Cas9 (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>). Many clones (27 separate isolates) did not contain any alteration at the expected Cas9 cut site (+3 position upstream of the 5′ end of the PAM). However, this was not due to a lack of editing because the entire drug selection marker was not present and had been fully excised. Interestingly, plating of yeast onto galactose containing medium (rather than dextrose) following transformation of the sgRNA plasmid resulted in a 10-fold decrease in the number of surviving colonies (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). Moreover, none of the surviving clones had excised the selectable marker or included any indel at the (u1) site(s), suggesting that prolonged Cas9 activation increases the stringency of selection. Thus, we have provided the first evidence to suggest that, in fact, estimates of editing are limited to detection of a generated indel at the proposed editing site. Our arrangement clearly demonstrates the ability of NHEJ to repair cleaved DNA with no alteration of sequence, at least in budding yeast. Additionally, our assay for Cas9 editing does not impose selection for any altered genetic marker until after the editing event, prevents off-target effects, and can also achieve multiplexing using identical sequences preprogrammed within a genome and an sgRNA fragment.</p></div><div id="sec-10" class="subsection"><h3>Exploring control of CRISPR editing in haploid cells</h3><p id="p-37">Since our long-term goal is the exploration of gene drive control and its possible application in plants, fungi, microbes, and metazoans, we limited our focus to the conserved components of the CRISPR system, namely, the Cas9 nuclease and the sgRNA. While additional yeast-based (or fungal-based) assays might be explored in the future, our objective was to investigate common components of all gene drive systems. Given that previous studies have analyzed the contribution of sgRNA identity, GC content, length, mismatch, stem-loop structure, and other modifications (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dang <em>et al.</em> 2015</a>; <a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">Xu <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Doench <em>et al.</em> 2016</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Nowak <em>et al.</em> 2016</a>; <a id="xref-ref-125-1" class="xref-bibr" href="#ref-125">Zhang <em>et al.</em> 2016</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Numamoto <em>et al.</em> 2017</a>), we tested a set of sgRNAs variants within our yeast editing system (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Of note, we have maintained an identical target DNA sequence [namely, the (u1) artificial target sites] in order to directly compare modifications to each guide sequence. However, we recognize that others (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Hsu <em>et al.</em> 2013</a>; <a id="xref-ref-127-1" class="xref-bibr" href="#ref-127">Zheng <em>et al.</em> 2017</a>) have demonstrated that the effect of sgRNA mismatches on targeting varies between sgRNAs. We first altered the guide length from 16 to 22 bp and found that editing in our system required a crRNA length between 19 and 22 bp (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Next, for guide lengths of 17–22 bp, we varied the number of 5′ mismatches (one, two, or three consecutive mutations: A/T to G/C and vice versa, chosen at random) as well as deletion of the penultimate base at the 5′ end (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). Testing of isolated clones for excision of the selectable marker largely mirrored the cell viability results—guide RNAs allowing efficient Cas9 editing resulted in loss of the marker whereas a lack of or reduction in editing was paired with drug resistance. Our results indicate that sgRNA length and identity had either no effect on editing or resulted in a complete loss of editing with only a single exception, a guide length of 19 bp with a single mismatch at the 5′ end. Numerous independent trials found an intermediate level of editing for this sgRNA to the dual (u1) sites. A total of 85% of yeast clones had excised the drug resistance cassette, yet editing with this sgRNA always resulted in more surviving colonies. We suspect that, given the unique arrangement of our (u1) flanked system, repair of the broken chromosome ends results in presentation of a (third) wild-type (WT) (u1) target site that can also be edited by Cas9 (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a>). A slight reduction in Cas9 targeting may impede this editing and “re-editing” cycle to allow an increase in surviving yeast clones. However, altering the 5′ mismatch from G→A to either G→C or G→T resulted in a near loss of editing and a phenocopy of the 18 bp guide length sgRNA (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). These results suggest editing using the 19 bp guide length with a 5′ mismatch is likely sequence and/or context dependent. We also observed a similar trend for other sgRNA variants that included mismatches. For instance, a 20 bp guide with two mismatches at the 5′ end provided a slight reduction in the total number of colonies (compared with the 18 bp guide length with no mismatch) and a small fraction of colonies that had properly excised the drug marker. While our analyses of sgRNAs is by no means comprehensive, our results demonstrate that all but one guide sequence (32/33 tested) either allow for optimal editing, or do not allow for editing at all, with potentially only a few rare exceptions (such as the 19 bp guide with one 5′ mismatch) that may result in an intermediate level of editing.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of sgRNA length and 5&#x2032; target mismatch on Cas9 editing efficiency. (A) GFY-2353 yeast containing the Cas9-NLS vector (pGF-IVL1116) were transformed with sgRNA(u1) cassettes (plasmids pGF-V1216&#x2013;pGF-V1222) with guide sequences of varying length along with an empty pRS425 vector control. The number of colonies was quantified (left) for three independent trials. Error, SD. Representative plates are shown (right). A random sampling of colonies was chosen across all three trials following editing on SD-URA-LEU plates and tested for growth on rich medium containing hygromycin. The percentage of isolates displaying sensitivity to the drug were quantified. For conditions (e.g., sgRNA(u1) 20 bp length) where a small number of colonies were viable, all surviving isolates (typically 5&#x2013;20 total) were tested on hygromycin; for other combinations, between 150 and 200 colonies were sampled. (B) Cas9 editing was repeated as in (A) using sgRNA(u1) cassettes containing varying mismatches at the 5&#x2032; end of the guide sequence. A single mismatch at the 5&#x2032; end (pGF-V1223 &#x2013; pGF-V1228), two mismatches (pGF-V1229 &#x2013; pGF-V1234), three mismatches (pGF-V1235 &#x2013; pGF-V1240), or a deletion of one base at the penultimate &#x2212;2 position from the 5&#x2032; end (pGF-V1241&#x2013;pGF-V1246) were assayed for both total surviving colonies and the percentage of isolates with an excised marker cassette at the target locus (top). Select comparisons with the sgRNA(u1) 19 bp guide with one mismatch data were performed using an unpaired t-test (bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of sgRNA length and 5&#x2032; target mismatch on Cas9 editing efficiency. (A) GFY-2353 yeast containing the Cas9-NLS vector (pGF-IVL1116) were transformed with sgRNA(u1) cassettes (plasmids pGF-V1216&#x2013;pGF-V1222) with guide sequences of varying length along with an empty pRS425 vector control. The number of colonies was quantified (left) for three independent trials. Error, SD. Representative plates are shown (right). A random sampling of colonies was chosen across all three trials following editing on SD-URA-LEU plates and tested for growth on rich medium containing hygromycin. The percentage of isolates displaying sensitivity to the drug were quantified. For conditions (&lt;em&gt;e.g.&lt;/em&gt;, sgRNA(u1) 20 bp length) where a small number of colonies were viable, all surviving isolates (typically 5&#x2013;20 total) were tested on hygromycin; for other combinations, between 150 and 200 colonies were sampled. (B) Cas9 editing was repeated as in (A) using sgRNA(u1) cassettes containing varying mismatches at the 5&#x2032; end of the guide sequence. A single mismatch at the 5&#x2032; end (pGF-V1223 &#x2013; pGF-V1228), two mismatches (pGF-V1229 &#x2013; pGF-V1234), three mismatches (pGF-V1235 &#x2013; pGF-V1240), or a deletion of one base at the penultimate &#x2212;2 position from the 5&#x2032; end (pGF-V1241&#x2013;pGF-V1246) were assayed for both total surviving colonies and the percentage of isolates with an excised marker cassette at the target locus (top). Select comparisons with the sgRNA(u1) 19 bp guide with one mismatch data were performed using an unpaired &lt;em&gt;t&lt;/em&gt;-test (bottom).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F2.medium.gif" width="366" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/3/999/F2.medium.gif" width="366" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488925" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">Effect of sgRNA length and 5′ target mismatch on Cas9 editing efficiency. (A) GFY-2353 yeast containing the Cas9-NLS vector (pGF-IVL1116) were transformed with sgRNA(u1) cassettes (plasmids pGF-V1216–pGF-V1222) with guide sequences of varying length along with an empty pRS425 vector control. The number of colonies was quantified (left) for three independent trials. Error, SD. Representative plates are shown (right). A random sampling of colonies was chosen across all three trials following editing on SD-URA-LEU plates and tested for growth on rich medium containing hygromycin. The percentage of isolates displaying sensitivity to the drug were quantified. For conditions (<em>e.g.</em>, sgRNA(u1) 20 bp length) where a small number of colonies were viable, all surviving isolates (typically 5–20 total) were tested on hygromycin; for other combinations, between 150 and 200 colonies were sampled. (B) Cas9 editing was repeated as in (A) using sgRNA(u1) cassettes containing varying mismatches at the 5′ end of the guide sequence. A single mismatch at the 5′ end (pGF-V1223 – pGF-V1228), two mismatches (pGF-V1229 – pGF-V1234), three mismatches (pGF-V1235 – pGF-V1240), or a deletion of one base at the penultimate −2 position from the 5′ end (pGF-V1241–pGF-V1246) were assayed for both total surviving colonies and the percentage of isolates with an excised marker cassette at the target locus (top). Select comparisons with the sgRNA(u1) 19 bp guide with one mismatch data were performed using an unpaired <em>t</em>-test (bottom).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Next, we investigated whether nucleocytoplasmic shuffling might be used to control Cas9 editing. Recently, a Cas9 variant (iCas) used occlusion of an NLS signal to prevent entry into the nucleus until treatment with an added external chemical (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Liu <em>et al.</em> 2016</a>). We hypothesized that nuclear localization and residence time might provide a conserved means to control and titrate editing when all other factors (Cas9 protein, sgRNA, target DNA) are held constant. We constructed a C-terminal Cas9-eGFP fusion and confirmed that it was fully functional and competent for editing compared with Cas9 alone (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). This provided three locations (N-terminus, C-terminus, and between the Cas9 and eGFP fusion) onto which to include an NLS sequence. We tested eight variants of Cas9 containing zero, one, two, or three NLS signals in all possible combinations and their ability to edit <em>in vivo</em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>). A Cas9-eGFP (lacking any added NLS) variant was still able to edit, albeit at a lower level compared with all other Cas9-eGFP variants harboring at least one NLS sequence. We suspect that Cas9 itself may (i) harbor a cryptic NLS signal(s) consisting of a cluster of positively charged residues and/or (ii) achieve a low level of diffusion into the nucleus, despite its large molecular weight (<a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">Wang and Brattain 2007</a>). There is precedence for unintended (artificial) nuclear localization of exposed peptides or nonnative protein sequences (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Guo <em>et al.</em> 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Finnigan <em>et al.</em> 2016</a>). We did not observe any significant differences in this assay for all Cas9 fusions containing only one to three NLS signals (one NLS was sufficient to promote maximum editing), although others (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Ménoret <em>et al.</em> 2015</a>; <a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Torres-Ruiz <em>et al.</em> 2017</a>) have reported increases in editing following the addition of numerous NLS sequences in other cell systems.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nucleocytoplasmic shuttling of Cas9 to control gene editing. (A) 16 variations of Cas9-eGFP were constructed that included combinations of NLS and/or NES signals at various protein positions. Either 0, 1, 2, or 3 (identical) NLS signals were included along with either 0 or 1 NES signals; the positions chosen included the N-terminus, between Cas9 and eGFP, or at the C-terminus (left). GFY-2353 yeast were transformed with each Cas9 fusion (pGF-IVL1162 &#x2013; pGFIVL1177) along with Cas9-NLS (pGF-IVL1116) as a positive control. Editing was performed by induction of Cas9 expression followed by transformation of equimolar amounts of sgRNA(u1) (20 bp WT guide) plasmid in triplicate. The strain expressing Cas9-NLS served as a control (transformed with either sgRNA(u1) or an empty pRS425 vector). The total number of surviving colonies (SD-URA-LEU medium) was quantified. Error, SD. Following editing, randomly selected isolates from all trials (n = 100&#x2013;200) were tested for growth on rich media containing hygromycin. For combinations where only a few surviving colonies existed, all possible isolates were tested for hygromycin sensitivity. (B) Comparisons of the colony counts between two strains from (A) were analyzed using an unpaired t-test. Red text indicates p-values &gt; 0.05. (C) Six Cas9-eGFP fusions were integrated into the yeast genome at the HIS3 locus in a strain expressing an endogenously tagged Nup188-mCherry to mark the nuclear periphery (strains GFY-3264&#x2013;GFY-3267, GFY-3273, and GFY-3275). Cultures were induced in galactose for 4.5 hr prior to imaging by fluorescence microscopy. Scale bar, 3 &#x3BC;m. White dotted lines, cell periphery. White triangles, yeast vacuole. Strain numbers (right) refer to the Cas9 fusions in (A) for clarity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Nucleocytoplasmic shuttling of Cas9 to control gene editing. (A) 16 variations of Cas9-eGFP were constructed that included combinations of NLS and/or NES signals at various protein positions. Either 0, 1, 2, or 3 (identical) NLS signals were included along with either 0 or 1 NES signals; the positions chosen included the N-terminus, between Cas9 and eGFP, or at the C-terminus (left). GFY-2353 yeast were transformed with each Cas9 fusion (pGF-IVL1162 &#x2013; pGFIVL1177) along with Cas9-NLS (pGF-IVL1116) as a positive control. Editing was performed by induction of Cas9 expression followed by transformation of equimolar amounts of sgRNA(u1) (20 bp WT guide) plasmid in triplicate. The strain expressing Cas9-NLS served as a control (transformed with either sgRNA(u1) or an empty pRS425 vector). The total number of surviving colonies (SD-URA-LEU medium) was quantified. Error, SD. Following editing, randomly selected isolates from all trials (&lt;em&gt;n&lt;/em&gt; = 100&#x2013;200) were tested for growth on rich media containing hygromycin. For combinations where only a few surviving colonies existed, all possible isolates were tested for hygromycin sensitivity. (B) Comparisons of the colony counts between two strains from (A) were analyzed using an unpaired &lt;em&gt;t&lt;/em&gt;-test. Red text indicates p-values &gt; 0.05. (C) Six Cas9-eGFP fusions were integrated into the yeast genome at the &lt;em&gt;HIS3&lt;/em&gt; locus in a strain expressing an endogenously tagged Nup188-mCherry to mark the nuclear periphery (strains GFY-3264&#x2013;GFY-3267, GFY-3273, and GFY-3275). Cultures were induced in galactose for 4.5 hr prior to imaging by fluorescence microscopy. Scale bar, 3 &#x3BC;m. White dotted lines, cell periphery. White triangles, yeast vacuole. Strain numbers (right) refer to the Cas9 fusions in (A) for clarity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F3.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/3/999/F3.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488848" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Nucleocytoplasmic shuttling of Cas9 to control gene editing. (A) 16 variations of Cas9-eGFP were constructed that included combinations of NLS and/or NES signals at various protein positions. Either 0, 1, 2, or 3 (identical) NLS signals were included along with either 0 or 1 NES signals; the positions chosen included the N-terminus, between Cas9 and eGFP, or at the C-terminus (left). GFY-2353 yeast were transformed with each Cas9 fusion (pGF-IVL1162 – pGFIVL1177) along with Cas9-NLS (pGF-IVL1116) as a positive control. Editing was performed by induction of Cas9 expression followed by transformation of equimolar amounts of sgRNA(u1) (20 bp WT guide) plasmid in triplicate. The strain expressing Cas9-NLS served as a control (transformed with either sgRNA(u1) or an empty pRS425 vector). The total number of surviving colonies (SD-URA-LEU medium) was quantified. Error, SD. Following editing, randomly selected isolates from all trials (<em>n</em> = 100–200) were tested for growth on rich media containing hygromycin. For combinations where only a few surviving colonies existed, all possible isolates were tested for hygromycin sensitivity. (B) Comparisons of the colony counts between two strains from (A) were analyzed using an unpaired <em>t</em>-test. Red text indicates p-values &gt; 0.05. (C) Six Cas9-eGFP fusions were integrated into the yeast genome at the <em>HIS3</em> locus in a strain expressing an endogenously tagged Nup188-mCherry to mark the nuclear periphery (strains GFY-3264–GFY-3267, GFY-3273, and GFY-3275). Cultures were induced in galactose for 4.5 hr prior to imaging by fluorescence microscopy. Scale bar, 3 μm. White dotted lines, cell periphery. White triangles, yeast vacuole. Strain numbers (right) refer to the Cas9 fusions in (A) for clarity.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Additionally, we directly fused a C-terminal NES signal sequence to each of the eight Cas9-NLS variants to determine if the interplay between import and export shuttling would affect editing (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, A and B</a>). Our results demonstrate that the direct competition between nuclear export and import can titrate the level of gene editing in otherwise isogenic strains. Direct fusion of an NES signal to the Cas9-eGFP variant greatly reduced editing, yet did not fully eliminate nuclear access (compare fusions 1 with 9) because nearly 50% of surviving clones had properly excised the target drug resistance marker (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Furthermore, this supported the observation that our Cas9-eGFP could access the nucleus in the absence of any added NLS signal. Direct competition of one NLS <em>vs.</em> one NES signal had the strongest reduction in gene editing (compare fusions 2/3/4 with 10/11/12). Altering the competition to two NLSs <em>vs.</em> one NES shifted the level of editing (compare fusions 5/6/7 with 13/14/15). Finally, a Cas9 variant harboring three NLS signals and one C-terminal NES displayed WT levels of editing (compare fusions 2/3/4 with 16). Live cell imaging of strains harboring one, two, or three NLS were compared with the same constructs also containing an NES (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). The nuclear periphery was marked with Nup188-mCherry; steady-state levels of Cas9-eGFP were found within the nucleus for fusions containing only NLS signals (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>, left) whereas the presence of the NES signal caused spatial exclusion from the nucleus (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>, right).</p><p id="p-42">These data demonstrate that, given sub-optimal nuclear localization (via the presence of an appended NES signal) there is a definite contribution of having more than one NLS sequence present. In our yeast system, given (i) sufficient <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a>/10</em>-driven expression of Cas9 and (ii) an extended recovery phase overnight in medium containing galactose, the SV40 NLS sequence was sufficient to promote maximal editing (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3A</a>). Therefore, in other cell systems where the SV40 may represent a nonoptimal signal sequence, or a lower level/amount of Cas9 protein present, it appears the addition of extra nuclear localization signals can improve import and subsequent editing.</p><p id="p-43">Finally, we examined whether there would be an additive effect between sgRNA identity (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) and nuclear shuttling (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>) by testing each of the Cas9-eGFP fusions with three different sgRNAs (20 bp WT, 19 bp WT, and 19 with one 5′ G→A mismatch) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). By comparing the number of yeast colonies for each of the guide RNAs (editing) to the number of colonies for an empty sgRNA vector (control) transformation, we determined a standardized “editing percentage.” For combinations of Cas9 and sgRNA that resulted in zero colonies, the editing percentage would be 100%. Conversely, a combination with 1000 colonies where the empty vector control also yielded 1000 colonies would calculate as 0% editing. As expected, the eight Cas9 fusions lacking any NES signal provided nearly 100% editing. Use of the sgRNA with a 19 bp guide and single 5′ mismatch reduced the editing by ∼15–25% (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Testing of the Cas9 fusions in the presence of a C-terminal NES signal displayed the same pattern (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>) for both 20 bp (WT) and 19 bp (WT) sgRNAs—a reduction in editing based on the number of competing NLS signals present. Use of the 19 bp guide RNA with a single mismatch caused a significant reduction (15–75%) in editing that was also correlated to the number of NLS signals, as more NLS sequences buffered against the effects of the less-effective guide RNA. Additionally, we observed the Cas9-eGFP-NLS-NES construct (fusion 12) deviated from the other two single NLS <em>vs.</em> single NES variants (fusions 10 and 11). In our experiments, the NLS-NES tag (fusion 12) nearly phenocopied the NES tag alone (fusion 9). Given the immediate proximity of the two signals (separated by only two residues), we suspect that physical access to the penultimate NLS might be restricted by binding and competition by nuclear export machinery (and possibly vice versa as well). We demonstrate using these 48 combinations of Cas9 fusions and sgRNAs a wide range of editing efficiencies can be achieved with the majority between 75 and 100% effectiveness (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). Moreover, our data illustrate there is a statistically significant added alteration in editing when using the 19 bp guide RNA with a single 5′ mismatch across 15 out of 16 of our Cas9 fusions with the only exception being Cas9-eGFP with a triple NLS signal (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Editing of haploid yeast using a combination of sgRNA mismatch and Cas9 nuclear localization. (A) GFY-2353 yeast containing 16 Cas9-eGFP fusions with NLS/NES combinations (pGF-IVL1162&#x2013;pGF-IVL1177) from Figure 3 were transformed with sgRNA(u1)-expressing plasmids (pGF-V1219, pGF-V1220, and pGF-V1225) or empty pRS425 and the total number of colonies quantified. For each Cas9-eGFP fusion, all four plasmids were transformed in triplicate. The editing efficiency is displayed as a percentage by the following calculation: (i) the total number of colonies (per sample) was first divided by the total number of colonies obtained for the empty vector control followed by (ii) 100% minus the calculated percentage from (i). Error, SD from (i). (B) The data from (A) is displayed from lowest to highest editing percentage (left) or as a histogram with 10% binning categories (right). (C) Select comparisons (sgRNA(u1) 19 WT vs. 19 with one mismatch) between editing percentages from (A) were analyzed using an unpaired t-test. Red text, p-values &gt; 0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Editing of haploid yeast using a combination of sgRNA mismatch and Cas9 nuclear localization. (A) GFY-2353 yeast containing 16 Cas9-eGFP fusions with NLS/NES combinations (pGF-IVL1162&#x2013;pGF-IVL1177) from Figure 3 were transformed with sgRNA(u1)-expressing plasmids (pGF-V1219, pGF-V1220, and pGF-V1225) or empty pRS425 and the total number of colonies quantified. For each Cas9-eGFP fusion, all four plasmids were transformed in triplicate. The editing efficiency is displayed as a percentage by the following calculation: (i) the total number of colonies (per sample) was first divided by the total number of colonies obtained for the empty vector control followed by (ii) 100% minus the calculated percentage from (i). Error, SD from (i). (B) The data from (A) is displayed from lowest to highest editing percentage (left) or as a histogram with 10% binning categories (right). (C) Select comparisons (sgRNA(u1) 19 WT &lt;em&gt;vs.&lt;/em&gt; 19 with one mismatch) between editing percentages from (A) were analyzed using an unpaired &lt;em&gt;t&lt;/em&gt;-test. Red text, p-values &gt; 0.05.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F4.medium.gif" width="411" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/3/999/F4.medium.gif" width="411" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488787" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Editing of haploid yeast using a combination of sgRNA mismatch and Cas9 nuclear localization. (A) GFY-2353 yeast containing 16 Cas9-eGFP fusions with NLS/NES combinations (pGF-IVL1162–pGF-IVL1177) from <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a> were transformed with sgRNA(u1)-expressing plasmids (pGF-V1219, pGF-V1220, and pGF-V1225) or empty pRS425 and the total number of colonies quantified. For each Cas9-eGFP fusion, all four plasmids were transformed in triplicate. The editing efficiency is displayed as a percentage by the following calculation: (i) the total number of colonies (per sample) was first divided by the total number of colonies obtained for the empty vector control followed by (ii) 100% minus the calculated percentage from (i). Error, SD from (i). (B) The data from (A) is displayed from lowest to highest editing percentage (left) or as a histogram with 10% binning categories (right). (C) Select comparisons (sgRNA(u1) 19 WT <em>vs.</em> 19 with one mismatch) between editing percentages from (A) were analyzed using an unpaired <em>t</em>-test. Red text, p-values &gt; 0.05.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Four independent mechanisms to titrate gene drive activity</h3><p id="p-45">Based upon our design for haploid gene editing, we construed a safe and programmable gene drive system in budding yeast (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Our drive strain included Cas9 under the inducible <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a>/10</em> promoter, a drug resistance cassette (Kan<sup>R</sup>), and flanking artificial (u2) sites (as a genetic safeguard, see <em>Materials and Methods</em> and Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). For biosecurity reasons, we have kept the sgRNA-expression cassette separate from the physical chromosomal gene drive and, instead, maintained it on a high-copy plasmid (see <em>Materials and Methods</em>, Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). We generated an artificial gene “target” strain in yeast of the opposite mating type containing flanking (u1) sites at the <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus including (ii) a target gene to be excised and (ii) a distinct selection cassette (<em>S. pombe HIS5</em>) under control of the constitutive <em><a href="http://www.yeastgenome.org/locus/S000004100/overview">CCW12</a></em> promoter. Importantly, we included unique terminator sequences (<em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002384/overview">SHS1</a></em> 3′ UTR) for both the drive and target systems; the presence of any identical sequences between the two homologous chromosomes can allow for inappropriate crossover within the drive itself (E. Roggenkamp, R. M. Giersch, M. N. Schrock, E. Turnquist, M. Halloran, and G. C. Finnigan, unpublished results).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Altering levels of Cas9 to activate an artificial gene drive in diploid yeast cells. (A) Our design of a programmable gene drive included (i) an integrated copy of S. pyogenes Cas9 (asterisk denotes use of various Cas9 fusions in an otherwise identical construct) under the inducible GAL1/10 promoter at the HIS3 locus in MATa cells, (ii) a Kanamycin-resistance gene cassette, (iii) flanking unique sites (u2) (Finnigan and Thorner 2016) surrounding the entire gene drive system to be used as a genetic failsafe (see Figure S6 in File S1), and (iv) an artificial gene &#x201C;target&#x201D; containing a different selectable marker (S. pombe HIS5) and flanked by (u1) artificial Cas9 sites at the HIS3 locus in a strain of the opposite mating type (MAT&#x3B1;). (B) Activation and testing of all gene drives was performed as follows. First, the Cas9-containing strain (shown, GFY-2383) was transformed with the sgRNA(u1) plasmid (pGF-IVL1220) or an empty vector (pRS425) control and maintained on dextrose. Second, the gene drive strain (MATa) harboring the sgRNA(u1) plasmid was mated to the target strain (MAT&#x3B1;; GFY-3206 or GFY-3207) on rich medium for 24 hr at 30&#xB0;. Third, diploid yeast were selected twice on SD-LEU-HIS medium (24 hr incubation at 30&#xB0;). Fourth, diploids were cultured overnight in S-LEU + Raffinose/Sucrose liquid medium. Fifth, strains were back-diluted to an OD600 of &#x223C;0.35 OD/ml in YP + Galactose and grown at 30&#xB0; for various amounts of time. Sixth, yeast were harvested by a brief centrifugation, washed with water, diluted to &#x223C;1000 cells/ml, and 0.5 ml was plated onto SD-LEU medium and incubated at 30&#xB0; for 2 d. Finally, yeast were transferred by replica-plating to SD-LEU and SD-HIS plates and incubated for 24 additional hours before imaging. Representative plates are shown for the GFY-3206 cross. (C) Quantification of the percentage of colonies displaying an active gene drive (assayed by sensitivity on SD-HIS medium). Error, SD. Statistically significant comparisons are denoted using an unpaired t-test. N.S., not significant. The value for 0 hr is 0% drive activity, not 50%. Experimental runs with an empty plasmid (pRS425) were also performed and displayed a value of zero drive activity for all time points. (D) Clonal isolates were randomly selected from SD-LEU plates from (B) and retested on G418 and SD-HIS media. Multiple crosses were used to determine ploidy status. Diagnostic PCRs (A&#x2013;D) were performed on isolated diploid chromosomal DNA to assess the HIS3 locus at 0 and 12 hr post galactose shift (also see Figure S4 in File S1). Oligonucleotides used can be found in Table S1 in File S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Altering levels of Cas9 to activate an artificial gene drive in diploid yeast cells. (A) Our design of a programmable gene drive included (i) an integrated copy of &lt;em&gt;S. pyogenes&lt;/em&gt; Cas9 (asterisk denotes use of various Cas9 fusions in an otherwise identical construct) under the inducible &lt;em&gt;GAL1/10&lt;/em&gt; promoter at the &lt;em&gt;HIS3&lt;/em&gt; locus in &lt;em&gt;MAT&lt;/em&gt;a cells, (ii) a Kanamycin-resistance gene cassette, (iii) flanking unique sites (u2) (Finnigan and Thorner 2016) surrounding the entire gene drive system to be used as a genetic failsafe (see Figure S6 in File S1), and (iv) an artificial gene &#x201C;target&#x201D; containing a different selectable marker (&lt;em&gt;S. pombe HIS5&lt;/em&gt;) and flanked by (u1) artificial Cas9 sites at the &lt;em&gt;HIS3&lt;/em&gt; locus in a strain of the opposite mating type (&lt;em&gt;MAT&lt;/em&gt;&#x3B1;). (B) Activation and testing of all gene drives was performed as follows. First, the Cas9-containing strain (shown, GFY-2383) was transformed with the sgRNA(u1) plasmid (pGF-IVL1220) or an empty vector (pRS425) control and maintained on dextrose. Second, the gene drive strain (&lt;em&gt;MAT&lt;/em&gt;a) harboring the sgRNA(u1) plasmid was mated to the target strain (&lt;em&gt;MAT&lt;/em&gt;&#x3B1;; GFY-3206 or GFY-3207) on rich medium for 24 hr at 30&#xB0;. Third, diploid yeast were selected twice on SD-LEU-HIS medium (24 hr incubation at 30&#xB0;). Fourth, diploids were cultured overnight in S-LEU + Raffinose/Sucrose liquid medium. Fifth, strains were back-diluted to an OD&lt;sub&gt;600&lt;/sub&gt; of &#x223C;0.35 OD/ml in YP + Galactose and grown at 30&#xB0; for various amounts of time. Sixth, yeast were harvested by a brief centrifugation, washed with water, diluted to &#x223C;1000 cells/ml, and 0.5 ml was plated onto SD-LEU medium and incubated at 30&#xB0; for 2 d. Finally, yeast were transferred by replica-plating to SD-LEU and SD-HIS plates and incubated for 24 additional hours before imaging. Representative plates are shown for the GFY-3206 cross. (C) Quantification of the percentage of colonies displaying an active gene drive (assayed by sensitivity on SD-HIS medium). Error, SD. Statistically significant comparisons are denoted using an unpaired &lt;em&gt;t&lt;/em&gt;-test. N.S., not significant. The value for 0 hr is 0% drive activity, not 50%. Experimental runs with an empty plasmid (pRS425) were also performed and displayed a value of zero drive activity for all time points. (D) Clonal isolates were randomly selected from SD-LEU plates from (B) and retested on G418 and SD-HIS media. Multiple crosses were used to determine ploidy status. Diagnostic PCRs (A&#x2013;D) were performed on isolated diploid chromosomal DNA to assess the &lt;em&gt;HIS3&lt;/em&gt; locus at 0 and 12 hr post galactose shift (also see Figure S4 in File S1). Oligonucleotides used can be found in Table S1 in File S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F5.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/3/999/F5.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488952" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Altering levels of Cas9 to activate an artificial gene drive in diploid yeast cells. (A) Our design of a programmable gene drive included (i) an integrated copy of <em>S. pyogenes</em> Cas9 (asterisk denotes use of various Cas9 fusions in an otherwise identical construct) under the inducible <em>GAL1/10</em> promoter at the <em>HIS3</em> locus in <em>MAT</em>a cells, (ii) a Kanamycin-resistance gene cassette, (iii) flanking unique sites (u2) (<a id="xref-ref-38-11" class="xref-bibr" href="#ref-38">Finnigan and Thorner 2016</a>) surrounding the entire gene drive system to be used as a genetic failsafe (see Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>), and (iv) an artificial gene “target” containing a different selectable marker (<em>S. pombe HIS5</em>) and flanked by (u1) artificial Cas9 sites at the <em>HIS3</em> locus in a strain of the opposite mating type (<em>MAT</em>α). (B) Activation and testing of all gene drives was performed as follows. First, the Cas9-containing strain (shown, GFY-2383) was transformed with the sgRNA(u1) plasmid (pGF-IVL1220) or an empty vector (pRS425) control and maintained on dextrose. Second, the gene drive strain (<em>MAT</em>a) harboring the sgRNA(u1) plasmid was mated to the target strain (<em>MAT</em>α; GFY-3206 or GFY-3207) on rich medium for 24 hr at 30°. Third, diploid yeast were selected twice on SD-LEU-HIS medium (24 hr incubation at 30°). Fourth, diploids were cultured overnight in S-LEU + Raffinose/Sucrose liquid medium. Fifth, strains were back-diluted to an OD<sub>600</sub> of ∼0.35 OD/ml in YP + Galactose and grown at 30° for various amounts of time. Sixth, yeast were harvested by a brief centrifugation, washed with water, diluted to ∼1000 cells/ml, and 0.5 ml was plated onto SD-LEU medium and incubated at 30° for 2 d. Finally, yeast were transferred by replica-plating to SD-LEU and SD-HIS plates and incubated for 24 additional hours before imaging. Representative plates are shown for the GFY-3206 cross. (C) Quantification of the percentage of colonies displaying an active gene drive (assayed by sensitivity on SD-HIS medium). Error, SD. Statistically significant comparisons are denoted using an unpaired <em>t</em>-test. N.S., not significant. The value for 0 hr is 0% drive activity, not 50%. Experimental runs with an empty plasmid (pRS425) were also performed and displayed a value of zero drive activity for all time points. (D) Clonal isolates were randomly selected from SD-LEU plates from (B) and retested on G418 and SD-HIS media. Multiple crosses were used to determine ploidy status. Diagnostic PCRs (A–D) were performed on isolated diploid chromosomal DNA to assess the <em>HIS3</em> locus at 0 and 12 hr post galactose shift (also see Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). Oligonucleotides used can be found in Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">The gene drive strain was transformed with the sgRNA(u1) plasmid (20 bp guide WT), mated to the target strain, and diploids were selected while maintaining growth on dextrose to repress Cas9 expression. Cultures were shifted to galactose for a time course between 0 and 12 hr and plated onto SD-LEU medium. Additionally, activity (or lack thereof) of the drive was not necessary for cell survival. In this way, we provided an unbiased assessment of the proportion of cells within a given sampling that were able to (i) express Cas9, (ii) bind sgRNA, (iii) target the dual (u1) artificial sequence(s), and (iv) undergo homologous recombination to repair the DSB and copying the entirety of the drive to the opposite chromosome to replace the target gene. Like our experimental design for assaying Cas9 activity in haploids (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), there is no selective pressure of any kind during the editing and repair events. Once single colonies were sufficiently grown, the entire sampling was transferred to both SD-LEU and SD-HIS plates where we assessed the status of the <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus gene drive and/or target. 100% of colonies maintained G418 resistance regardless of whether an active drive was induced (E. Roggenkamp, R. M. Giersch, M. N. Schrock, E. Turnquist, M. Halloran, and G. C. Finnigan, unpublished results). However, as cells were grown in galactose for increasing amounts of time, causing higher expression of Cas9, a larger proportion of the population was sensitive on the SD-HIS condition (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, B and C</a>). After 4 hr postgalactose shift, 99% of colonies had lost the <em>S. pombe HIS5</em> marker within the target locus. A random sampling of clonal isolates (all tested and confirmed as diploids) from each time point were chosen from SD-LEU plates and the <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus was interrogated by multiple diagnostic PCRs (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a> and Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). There was a 100% correlation between colonies sensitive on SD-HIS medium and lack of the entire target locus as assayed by PCR. For those (few) surviving colonies on SD-HIS (even after the 12 hr galactose shift), the diploid genome still contained both the Cas9 gene drive and the target cassette (E. Roggenkamp, R. M. Giersch, M. N. Schrock, E. Turnquist, M. Halloran, and G. C. Finnigan, unpublished results). These data suggest that titration of the Cas9 nuclease itself correlates strongly with gene drive activity within a population.</p><p id="p-48">We also examined the effects of altering the sgRNA length and mismatch (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>) in the context of our gene drive system (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). We obtained nearly identical results to our haploid editing experiments; only guide lengths of 19–22 allowed for an active gene drive even when the galactose induction time was increased to 24 hr. However, the 19 bp guide length with one 5′ mismatch (G→A) did provide the only partially functioning (∼45% active) drive system. We recognize that this is reduced compared with the efficiency of editing in haploid cells (80–85% effective) yet there are major differences in the mode of repair between the two assays (NHEJ with the option to re-edit multiple rounds <em>vs.</em> homologous recombination in a diploid genome for action of a gene drive). Altering the 5′ base to either C or T resulted in a near total loss of drive activity (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Varying sgRNA identity to control gene drives. GFY-2383 yeast was transformed with the collection of sgRNA(u1) cassettes from Figure 2. Yeast were mated to the target strains (GFY-3206 and GFY-3207), diploids selected, and drives were activated as described in Figure 5. Diploids were induced in YP + Galactose for 24 hr prior to plating in triplicate. For sgRNA(u1) 20 bps (WT) and 19 bps (one mismatch), six independent trials were performed (also see Figure S3 in File S1). The percentage of yeast colonies with an active gene drive was quantified. The total number of dead colonies on SD-HIS plates compared with the corresponding colonies on SD-LEU plates represented the active gene drive percentage. Error, SD. The two comparisons highlighted were analyzed using an unpaired t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Varying sgRNA identity to control gene drives. GFY-2383 yeast was transformed with the collection of sgRNA(u1) cassettes from Figure 2. Yeast were mated to the target strains (GFY-3206 and GFY-3207), diploids selected, and drives were activated as described in Figure 5. Diploids were induced in YP + Galactose for 24 hr prior to plating in triplicate. For sgRNA(u1) 20 bps (WT) and 19 bps (one mismatch), six independent trials were performed (also see Figure S3 in File S1). The percentage of yeast colonies with an active gene drive was quantified. The total number of dead colonies on SD-HIS plates compared with the corresponding colonies on SD-LEU plates represented the active gene drive percentage. Error, SD. The two comparisons highlighted were analyzed using an unpaired &lt;em&gt;t&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F6.medium.gif" width="268" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/3/999/F6.medium.gif" width="268" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488876" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-49" class="first-child">Varying sgRNA identity to control gene drives. GFY-2383 yeast was transformed with the collection of sgRNA(u1) cassettes from <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>. Yeast were mated to the target strains (GFY-3206 and GFY-3207), diploids selected, and drives were activated as described in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>. Diploids were induced in YP + Galactose for 24 hr prior to plating in triplicate. For sgRNA(u1) 20 bps (WT) and 19 bps (one mismatch), six independent trials were performed (also see Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). The percentage of yeast colonies with an active gene drive was quantified. The total number of dead colonies on SD-HIS plates compared with the corresponding colonies on SD-LEU plates represented the active gene drive percentage. Error, SD. The two comparisons highlighted were analyzed using an unpaired <em>t</em>-test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">We observed the same trend (lower gene drive activity when compared with haploid NHEJ-based editing) when we examined the Cas9-eGFP fusions containing various NLS/NES combinations (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Given that our WT Cas9-based drive (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>) was nearly 99% active after 4 hr postgalactose shift, we tested our 16 Cas9-eGFP variants at 1.25, 2.5, and 5.0 hr following induction (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). Indeed, all constructs containing only added NLS signals (fusions 2–8) displayed nearly 99% active drive efficiencies after 5 hr of Cas9 expression; the Cas9-eGFP fusion with no added signal sequence was markedly lower at 75% (fusion 1). Addition of the C-terminal NES signal resulted in a dramatic reduction in nearly all drive activities (fusions 9–16). In fact, constructs containing only an NES or NLS-NES motif showed no activity at the 5 hr mark; however, this was not due to a lack of Cas9 expression (or nuclease activity) as these constructs were readily expressed, excluded from the nucleus, and were able to initiate editing in both haploid cells and gene drive diploids at a later time point (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>). The same general trend was observed for the competition between NLS and NES signals with additional NLS motifs providing higher drive activity (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>). These results highlight that titration of Cas9 protein expression with nucleocytoplasmic shuttling can provide a wide range of gene drive efficiencies from very low activity (0%) to WT levels (99%) (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>). Moreover, our findings demonstrate that nuclear exclusion (or limited residence time) may serve as a reasonable mechanism to fully inhibit or titrate drive activity (depending on the presence of other signals). Finally, given less time for Cas9 induction (&lt;5 hr), there was still a modest increase in drive activity for constructs with two or three NLS sequences compared with only one nuclear signal, even in the absence of any NES (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Modulation of gene drive activity by titration of Cas9 and nuclear shuttling. (A) Gene drives were generated based on the 16 plasmid-borne Cas9 constructs from Figure 3 and integrated at the HIS3 locus. All gene drive strains (GFY-2751&#x2013;GFY-2766) were transformed with the sgRNA(u1) 20 bp WT guide plasmid and mated to the two target strains (GFY-3206 and GFY-3207). Following diploid selection and preinduction in a raffinose/sucrose mixture, diploid yeast were cultured in YP + galactose for 1.25, 2.5, or 5.0 hr prior to plating. Representative plates (the Cas9-eGFP fusion number illustrated for clarity) for two groupings are illustrated at the 5 hr time point on SD-LEU and SD-HIS medium (left). The percentage of yeast with active gene drives (percentage of colonies dead on SD-HIS) was quantified in triplicate (right). Error, SD. (B) Two-way comparisons between strains from (A) were performed using an unpaired t-test. Red text, p-values &gt; 0.05. Asterisk, the collective average of all three strains was used for comparisons. (C) The data from (A) was reordered from least to greatest percentage of active gene drive (top). The data from (A) is presented in a histogram with 10% binning categories (bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Modulation of gene drive activity by titration of Cas9 and nuclear shuttling. (A) Gene drives were generated based on the 16 plasmid-borne Cas9 constructs from Figure 3 and integrated at the &lt;em&gt;HIS3&lt;/em&gt; locus. All gene drive strains (GFY-2751&#x2013;GFY-2766) were transformed with the sgRNA(u1) 20 bp WT guide plasmid and mated to the two target strains (GFY-3206 and GFY-3207). Following diploid selection and preinduction in a raffinose/sucrose mixture, diploid yeast were cultured in YP + galactose for 1.25, 2.5, or 5.0 hr prior to plating. Representative plates (the Cas9-eGFP fusion number illustrated for clarity) for two groupings are illustrated at the 5 hr time point on SD-LEU and SD-HIS medium (left). The percentage of yeast with active gene drives (percentage of colonies dead on SD-HIS) was quantified in triplicate (right). Error, SD. (B) Two-way comparisons between strains from (A) were performed using an unpaired &lt;em&gt;t&lt;/em&gt;-test. Red text, p-values &gt; 0.05. Asterisk, the collective average of all three strains was used for comparisons. (C) The data from (A) was reordered from least to greatest percentage of active gene drive (top). The data from (A) is presented in a histogram with 10% binning categories (bottom).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F7.medium.gif" width="376" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/3/999/F7.medium.gif" width="376" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488780" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-51" class="first-child">Modulation of gene drive activity by titration of Cas9 and nuclear shuttling. (A) Gene drives were generated based on the 16 plasmid-borne Cas9 constructs from <a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a> and integrated at the <em>HIS3</em> locus. All gene drive strains (GFY-2751–GFY-2766) were transformed with the sgRNA(u1) 20 bp WT guide plasmid and mated to the two target strains (GFY-3206 and GFY-3207). Following diploid selection and preinduction in a raffinose/sucrose mixture, diploid yeast were cultured in YP + galactose for 1.25, 2.5, or 5.0 hr prior to plating. Representative plates (the Cas9-eGFP fusion number illustrated for clarity) for two groupings are illustrated at the 5 hr time point on SD-LEU and SD-HIS medium (left). The percentage of yeast with active gene drives (percentage of colonies dead on SD-HIS) was quantified in triplicate (right). Error, SD. (B) Two-way comparisons between strains from (A) were performed using an unpaired <em>t</em>-test. Red text, p-values &gt; 0.05. Asterisk, the collective average of all three strains was used for comparisons. (C) The data from (A) was reordered from least to greatest percentage of active gene drive (top). The data from (A) is presented in a histogram with 10% binning categories (bottom).</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Finally, we tested whether enzymatically dead Cas9 (dCas9), which is still able to associate with the guide RNA and to target DNA, could serve as a direct competitor of active Cas9 (of the same species) when provided with an identical sgRNA (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). Rather than construct two separate expression cassettes for active Cas9 and dCas9, we created three separate tandem fusions between <em>S. pyogenes</em> Cas9 and either a second active nuclease, or an enzymatically dead version for several reasons (while the protein products were identical, the coding sequences were altered to prevent inappropriate homologous recombination and potential loss of one copy) (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>). First, translational fusions to Cas9 (or dCas9) have been extensively used by many groups to add fluorescent proteins (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Ma <em>et al.</em> 2015</a>), chromatin-modifying enzymes (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Morgan <em>et al.</em> 2017</a>), transcriptional activators/repressors (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Jensen <em>et al.</em> 2017</a>), or other DNA-modifying enzymes (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Kim <em>et al.</em> 2017</a>). Second, a gene fusion between otherwise identical Cas9 enzymes would circumvent the need for exacting titration of transcript/translation for a direct one-to-one comparison. Third, incorporation of both Cas9 genes at the site of the gene drive (<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> locus) would require the addition of unique promoter and terminator sequences flanking the secondary gene copy in a “separate” arrangement. Fourth, use of an additional yeast-specific (inducible) promoter sequence to drive expression of a second Cas9 variant would have limited utility to gene drive systems in other organisms. Thus, we have focused our initial efforts on tandem <em>S. pyogenes</em> Cas9 gene fusions, although we recognize that future iterations might consist of “competing” freely expressed Cas9 nucleases to titrate editing.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Novel fusions of enzymatically active and inactive Cas9 reduce gene drive activity. (A) Model of tandem Cas9 fusion design. A second Cas9 gene (asterisk) was synthesized de novo by altering &gt;90% of the codons (primarily within the Wobble position). A 15-residue flexible linker was inserted between the two Cas9 copies. Dead Cas9 contains the mutations D10A and H840A. (B) GFY-2383, GFY-3250, GFY-3099, GFY-3100, and GFY-3336 yeast were transformed with equimolar amounts of either an empty vector (pRS425, duplicate), or a plasmid expressing the sgRNA(u2) 20 bp WT cassette (pGF-V809, triplicate), plated onto SD-LEU medium and incubated for 3 d. The total number of viable colonies were quantified (left). Error, SD. Two-strain comparisons were performed using an unpaired t-test (right). Red text, p-values &gt; 0.05. (C) Yeast strains from (B) were each transformed with two plasmids (URA3 and LEU2 markers) resulting in four conditions: (i) sgRNA(u1)/sgRNA(u1), (ii) sgRNA(u1)/empty, (iii) empty/sgRNA(u1), and (iv) empty/empty. These included pRS425, pRS426, pGF-V1220, and pGF-V1625. Only the data for one of the sgRNA(u1)/empty combinations (pGF-V1625/pRS425) is presented. Strains were mated to the gene drive target strains (GFY-3206 and GFY-3207) and diploids were selected on SD-URA-LEU-HIS three consecutive rounds. Strains harboring either (i) two empty vectors or (ii) expressing a single copy of dCas9, were only mated to GFY-3206. Diploid yeast were preinduced overnight as previously described, and Cas9 expression was induced for 5, 12, or 24 hr in YPGal medium prior to dilution onto SD-URA-LEU plates. Finally, yeast were transferred to SD-URA-LEU and SD-HIS plates before imaging (top). The percentage of active gene drives (percentage of colonies dead on SD-HIS plates) was quantified (bottom). Error, SD. Comparisons between strains (all time points included) were performed using an unpaired t-test. p-values &gt; 0.10 (red text), between 0.05 and 0.10 (green text), and " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Novel fusions of enzymatically active and inactive Cas9 reduce gene drive activity. (A) Model of tandem Cas9 fusion design. A second Cas9 gene (asterisk) was synthesized &lt;em&gt;de novo&lt;/em&gt; by altering &gt;90% of the codons (primarily within the Wobble position). A 15-residue flexible linker was inserted between the two Cas9 copies. Dead Cas9 contains the mutations D10A and H840A. (B) GFY-2383, GFY-3250, GFY-3099, GFY-3100, and GFY-3336 yeast were transformed with equimolar amounts of either an empty vector (pRS425, duplicate), or a plasmid expressing the sgRNA(u2) 20 bp WT cassette (pGF-V809, triplicate), plated onto SD-LEU medium and incubated for 3 d. The total number of viable colonies were quantified (left). Error, SD. Two-strain comparisons were performed using an unpaired &lt;em&gt;t&lt;/em&gt;-test (right). Red text, p-values &gt; 0.05. (C) Yeast strains from (B) were each transformed with two plasmids (&lt;em&gt;URA3&lt;/em&gt; and &lt;em&gt;LEU2&lt;/em&gt; markers) resulting in four conditions: (i) sgRNA(u1)/sgRNA(u1), (ii) sgRNA(u1)/empty, (iii) empty/sgRNA(u1), and (iv) empty/empty. These included pRS425, pRS426, pGF-V1220, and pGF-V1625. Only the data for one of the sgRNA(u1)/empty combinations (pGF-V1625/pRS425) is presented. Strains were mated to the gene drive target strains (GFY-3206 and GFY-3207) and diploids were selected on SD-URA-LEU-HIS three consecutive rounds. Strains harboring either (i) two empty vectors or (ii) expressing a single copy of dCas9, were only mated to GFY-3206. Diploid yeast were preinduced overnight as previously described, and Cas9 expression was induced for 5, 12, or 24 hr in YPGal medium prior to dilution onto SD-URA-LEU plates. Finally, yeast were transferred to SD-URA-LEU and SD-HIS plates before imaging (top). The percentage of active gene drives (percentage of colonies dead on SD-HIS plates) was quantified (&lt;em&gt;bottom&lt;/em&gt;). Error, SD. Comparisons between strains (all time points included) were performed using an unpaired &lt;em&gt;t&lt;/em&gt;-test. p-values &gt; 0.10 (red text), between 0.05 and 0.10 (green text), and &lt;0.05 (black text). For individual time point comparisons, see Table S2 in File S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F8.medium.gif" width="336" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/3/999/F8.medium.gif" width="336" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-53" class="first-child">Novel fusions of enzymatically active and inactive Cas9 reduce gene drive activity. (A) Model of tandem Cas9 fusion design. A second Cas9 gene (asterisk) was synthesized <em>de novo</em> by altering &gt;90% of the codons (primarily within the Wobble position). A 15-residue flexible linker was inserted between the two Cas9 copies. Dead Cas9 contains the mutations D10A and H840A. (B) GFY-2383, GFY-3250, GFY-3099, GFY-3100, and GFY-3336 yeast were transformed with equimolar amounts of either an empty vector (pRS425, duplicate), or a plasmid expressing the sgRNA(u2) 20 bp WT cassette (pGF-V809, triplicate), plated onto SD-LEU medium and incubated for 3 d. The total number of viable colonies were quantified (left). Error, SD. Two-strain comparisons were performed using an unpaired <em>t</em>-test (right). Red text, p-values &gt; 0.05. (C) Yeast strains from (B) were each transformed with two plasmids (<em>URA3</em> and <em>LEU2</em> markers) resulting in four conditions: (i) sgRNA(u1)/sgRNA(u1), (ii) sgRNA(u1)/empty, (iii) empty/sgRNA(u1), and (iv) empty/empty. These included pRS425, pRS426, pGF-V1220, and pGF-V1625. Only the data for one of the sgRNA(u1)/empty combinations (pGF-V1625/pRS425) is presented. Strains were mated to the gene drive target strains (GFY-3206 and GFY-3207) and diploids were selected on SD-URA-LEU-HIS three consecutive rounds. Strains harboring either (i) two empty vectors or (ii) expressing a single copy of dCas9, were only mated to GFY-3206. Diploid yeast were preinduced overnight as previously described, and Cas9 expression was induced for 5, 12, or 24 hr in YPGal medium prior to dilution onto SD-URA-LEU plates. Finally, yeast were transferred to SD-URA-LEU and SD-HIS plates before imaging (top). The percentage of active gene drives (percentage of colonies dead on SD-HIS plates) was quantified (<em>bottom</em>). Error, SD. Comparisons between strains (all time points included) were performed using an unpaired <em>t</em>-test. p-values &gt; 0.10 (red text), between 0.05 and 0.10 (green text), and &lt;0.05 (black text). For individual time point comparisons, see Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">Transformation of these tandem Cas9-Cas9 fusions along with controls with the self-excising guide RNA (u2) demonstrated that both orientations (Cas9-dCas9 and dCas9-Cas9) resulted in a reduced level of editing in haploids compared with WT Cas9 expressed alone (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8B</a>). Interestingly, the tandem fusion of two nuclease-active Cas9 proteins (strain I) displayed a marked increase in editing compared with the dCas9-containing fusions yet was still significantly impeded when compared with free WT Cas9. We next examined these Cas9 variants in the context of our gene drive system. We hypothesized that one possible contributing factor to the overall reduction in editing with all Cas9 fusions could be the requirement of additional sgRNA fragments per Cas9 polypeptide: the fusion might serve as an sgRNA “sink” requiring double the level of RNA compared with free Cas9. Therefore, we tested three conditions at various time points using two identical sgRNA-expressing plasmids (marked with <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em>): (i) two empty vectors, (ii) two sgRNA(u1) plasmids, and (iii) one sgRNA/one empty plasmid (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8C</a>). Both combinations of the sgRNA/empty plasmid condition were tested and displayed nearly identical results (E. Roggenkamp, R. M. Giersch, M. N. Schrock, E. Turnquist, M. Halloran, and G. C. Finnigan, unpublished data). We found that the Cas9-dCas9 and dCas9-Cas9 fusions resulted in ∼45–70% drive activities (24 hr) (conditions A and B). Testing of the dual active Cas9-Cas9 fusion revealed a marked increase (24 hr, 80–90%) in overall drive activity, especially at earlier time points, yet still fell short of the optimal freely expressed WT Cas9 (24 hr, &gt;95%) (condition C).</p><p id="p-55">Interestingly, we observed only a modest contribution of having multiple guide RNA-expressing plasmids under all conditions tested, and the majority were not statistically significant (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1/FileS1.pdf">File S1</a>) suggesting that, in our experimental setup (high-copy plasmid and <em><a href="http://www.yeastgenome.org/locus/S000006443/overview">SNR52</a></em> promoter), there is likely near-sufficient guide RNA present. Next, there was no difference between the placement of the dCas9 variant on the N- or C-terminus of active Cas9 (both are separated by a 15-residue flexible linker). Also, use of the Cas9-Cas9 variant, identical save for two mutational substitutions, illustrated that there is a direct contribution of the second active nuclease within this arrangement and there is likely direct competition of the dCas9 variant within the mixed fusions.</p><p id="p-56">Finally, the decrease in drive activity (and haploid editing) of the Cas9-Cas9 fusion compared with free WT Cas9 may result from slowed nuclear import, as the expected molecular weight of the tandem fusions is expected to be nearly 320 kDa. However, passive diffusion of chimeric model proteins was found to exceed the 60 kDa limit in a previous study, suggesting that, perhaps, the nuclear pore complex can accommodate much larger proteins, especially those constructed from protein fusions, rather than natively assembled masses (<a id="xref-ref-115-2" class="xref-bibr" href="#ref-115">Wang and Brattain 2007</a>). Moreover, upper estimates of a eukaryotic nuclear pore complex cargo size included a diameter of up to 39 nm (using cargo-receptor-gold particle complexes) (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Pante and Kann 2002</a>), whereas the diameter of a single Cas9 protein is well below 15 nm (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Jinek <em>et al.</em> 2014</a>). In support of this model of slowed nuclear import, the earliest time point (5 hr) for this fusion displayed a dramatic difference with WT Cas9, yet given additional induction time (12 or 24 hr), the Cas9-Cas9 variant was able to achieve a significant level of editing and drive activity. Collectively, our results present four molecular aspects of CRISPR-based gene drives that have the potential to modulate—in a programmable, predictable fashion—the activity and effectiveness of a given gene drive system (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/999/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model of four independent mechanisms for titration of Cas9-based gene drive activity. (1) Expression level of Cas9 protein from an inducible promoter can alter gene drive effectiveness. (2) The nucleocytoplasmic shuttling of Cas9, with varying NLS and NES signal combinations, provides a mechanism for achieving a wide range of gene drive activities ranging from 0 to 99%. (3) Altering the sgRNA length (19 bp) and level of mismatch (one mutation at the 5&#x2032; end), can reduce drive activity by &#x223C;50%. This may be specific to the different substitutions. (4) A dual Cas9 fusion between active and dead Cas9 (in either orientation) or a tandem fusion of two active Cas9 proteins can also partially reduce drive activity. All four mechanisms can be combined and the effects on drive effectiveness compounded (left). Across a population, each of these mechanisms may result in a proportion of individuals that achieve proper activity and copying of the gene drive system; other individuals will be unable to propagate the drive in a super-Mendelian fashion (right). Together, these systems may be used to titrate a specific success (propagation) rate for a CRISPR gene drive within a population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2129399897" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Model of four independent mechanisms for titration of Cas9-based gene drive activity. (1) Expression level of Cas9 protein from an inducible promoter can alter gene drive effectiveness. (2) The nucleocytoplasmic shuttling of Cas9, with varying NLS and NES signal combinations, provides a mechanism for achieving a wide range of gene drive activities ranging from 0 to 99%. (3) Altering the sgRNA length (19 bp) and level of mismatch (one mutation at the 5&#x2032; end), can reduce drive activity by &#x223C;50%. This may be specific to the different substitutions. (4) A dual Cas9 fusion between active and dead Cas9 (in either orientation) or a tandem fusion of two active Cas9 proteins can also partially reduce drive activity. All four mechanisms can be combined and the effects on drive effectiveness compounded (left). Across a population, each of these mechanisms may result in a proportion of individuals that achieve proper activity and copying of the gene drive system; other individuals will be unable to propagate the drive in a super-Mendelian fashion (right). Together, these systems may be used to titrate a specific success (propagation) rate for a CRISPR gene drive within a population.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/999/F9.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/3/999/F9.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/999/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/999/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488863" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-57" class="first-child">Model of four independent mechanisms for titration of Cas9-based gene drive activity. (1) Expression level of Cas9 protein from an inducible promoter can alter gene drive effectiveness. (2) The nucleocytoplasmic shuttling of Cas9, with varying NLS and NES signal combinations, provides a mechanism for achieving a wide range of gene drive activities ranging from 0 to 99%. (3) Altering the sgRNA length (19 bp) and level of mismatch (one mutation at the 5′ end), can reduce drive activity by ∼50%. This may be specific to the different substitutions. (4) A dual Cas9 fusion between active and dead Cas9 (in either orientation) or a tandem fusion of two active Cas9 proteins can also partially reduce drive activity. All four mechanisms can be combined and the effects on drive effectiveness compounded (left). Across a population, each of these mechanisms may result in a proportion of individuals that achieve proper activity and copying of the gene drive system; other individuals will be unable to propagate the drive in a super-Mendelian fashion (right). Together, these systems may be used to titrate a specific success (propagation) rate for a CRISPR gene drive within a population.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Rationale for a “programmable” gene drive</h3><p id="p-58">The discovery and rapid expansion of CRISPR-based gene editing across all fields of molecular biology has led to many unique applications of this biotechnology. These range from “traditional” editing (deletion, modification, or replacement of genomic loci) (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">DiCarlo <em>et al.</em> 2013</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Jinek <em>et al.</em> 2013</a>), to genome-wide screens (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Koike-Yusa <em>et al.</em> 2014</a>; <a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">Wang <em>et al.</em> 2014</a>), to repurposing of dCas9 to modulate gene transcription (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Gilbert <em>et al.</em> 2013</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Gao <em>et al.</em> 2016</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Jensen <em>et al.</em> 2017</a>), to imaging of chromosome dynamics in real-time (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Ma <em>et al.</em> 2015</a>, <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">2016b</a>). Arguably one of the most powerful (and ethically concerning) uses of CRISPR-Cas9 is within the nuclease-based gene drive. This system has been recognized as means of biological control that could combat insect pests, invasive species and, most notably, insect-borne pathogens such as malaria. Indeed, early studies in the laboratory have demonstrated the unnatural power of a gene drive system to impose strong selection and over a 95% reduction in a population in only a few generations (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Gantz <em>et al.</em> 2015</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Hammond <em>et al.</em> 2016</a>). Clearly, use of Cas9 in this genetic arrangement has the potential to impose a new level of control over native populations never seen before. However, early efforts have recently recognized several new challenges facing the development and application of gene drive–containing systems in the wild. First, given the incredibly strong selective pressure on a population, insect species have been shown to evolve resistance to the gene drive (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beaghton <em>et al.</em> 2017</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Champer <em>et al.</em> 2017</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Hammond <em>et al.</em> 2017</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Noble <em>et al.</em> 2017</a>). Second, wild populations may naturally have a diverse set of polymorphisms within a given gene target (and thus provide a native source to evade a gene drive) (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Drury <em>et al.</em> 2017</a>). Third, there is ongoing competition between HDR-based propagation of the drive and repair of the DSB via NHEJ (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Prowse <em>et al.</em> 2017</a>). Fourth, there are major ethical and scientific concerns regarding the implementation and use of a biological agent capable of (potentially) unrestricted population control on a large scale (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abbasi 2016</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Reid and O’Brochta 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Adelman <em>et al.</em> 2017</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Courtier-Orgogozo <em>et al.</em> 2017a</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">b</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Ritchie and Johnson 2017</a>; <a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">Wolt 2017</a>).</p><p id="p-59">We remain optimistic that the technical challenges facing gene drive success will be resolved given additional and ongoing laboratory study; indeed, the field is incredibly new and only a few labs have begun examining various drive designs. Moreover, there are already proposed solutions to increase the effectiveness of drives within wild populations including (i) multiple gene drives (and multiple targets), (ii) numerous sgRNAs (four or more to circumvent evolved resistance) (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Marshall <em>et al.</em> 2017</a>; <a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Prowse <em>et al.</em> 2017</a>), (iii) targeting of highly conserved/invariant (within a given species/subspecies) DNA sequences within a gene target to avoid polymorphisms (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Burt 2003</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Hammond <em>et al.</em> 2016</a>), (iv) efforts to study and (eventually) disrupt the host-pathogen pairing rather than killing of the entire (host) insect (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Gantz <em>et al.</em> 2015</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Rénia and Goh 2016</a>), and (v) other proposed drive arrangements including “anti-drive” drives (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">DiCarlo <em>et al.</em> 2015</a>; <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Noble <em>et al.</em> 2017</a>). Therefore, our goal was to investigate one major aspect of gene drives that has never been studied or considered to date: a methodology to program a desired drive effectiveness that is fully self-contained and requires no external regulatory mechanism.</p><p id="p-60">We envision numerous reasons why a suboptimal drive arrangement may be desirable in practice. First, lowering the rate by which a gene drive can propagate through a given population would require more time to achieve full penetrance (<em>e.g.</em>, 10% effective drive <em>vs.</em> 99%). Future work will be required to carefully map the relationship between drive activity, organism generation time, and penetrance within a population; indeed, recent studies have begun using <em>in silico</em> modeling to examine the dynamics of gene drives within populations (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Godfray <em>et al.</em> 2017</a>; <a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">Noble <em>et al.</em> 2017</a>; <a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Unckless <em>et al.</em> 2017</a>). A titratable drive system might be useful in the initial stages of field testing or studying optimization of a drive system in a native or controlled population: increasing the number of generations required for full conversion would allow for (i) control over the length of time required for full penetrance which may be of use for organisms with a very rapid (or too rapid) generation time and (ii) the option to counter, halt, or release a fail-safe drive to reverse, inhibit, or destroy the primary drive should the need arise. Population suppression (rather than elimination) is still a useful goal that might benefit from a tunable drive system (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Godfray <em>et al.</em> 2017</a>).</p><p id="p-61">Second, the use of gene drives does not necessarily have to be restricted to application within wild populations. Indeed, several studies have demonstrated that gene drives represent powerful genetic screening tools that can be used within basic laboratory research (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Roggenkamp <em>et al.</em> 2017</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Shapiro <em>et al.</em> 2018</a>). A tunable gene drive could be useful in studying population dynamics, evolved resistance, or in the generation of a heterogeneous population of edited cells/cell types for use in high throughput screening. Third, nearly every form of biological control (<em>e.g.</em>, especially that used in agriculture such as natural predators, chemical agents, or physical barriers/traps) of pests or pathogens includes the ability to titrate the proposed solution to a level that is safe (to surrounding plant and animal life and to humans), cost-effective, and appropriate given the nature of the problem at hand. In its current state, deployment of an active gene drive would have two levels: 0% (not active) or nearly 99% (fully active) rates. Fourth, components that can partially reduce/slow drive activity may be able to selectively, or in combination, impose maximum reduction in drive activity should the need arise and serve as a controllable and inducible (temporary or permeant) off switch. Therefore, we have performed an investigation into invariant components of all gene drive systems regardless of the proposed organism of use: Cas9 and the sgRNA.</p></div><div id="sec-14" class="subsection"><h3>Molecular mechanisms to tune drive activity in yeast</h3><p id="p-62">Use of <em>S. cerevisiae</em> as a model system to study gene drives comes with the numerous benefits of this popular and genetically tractable model organism. As a model eukaryote, many have begun to use yeast to demonstrate novel applications and uses of Cas9, including synthetic genome construction (<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Tsarmpopoulos <em>et al.</em> 2016</a>), chromosome splitting (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Sasano <em>et al.</em> 2016</a>), dCas9 transcriptional modulation (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Jensen <em>et al.</em> 2017</a>), reengineering of biosynthetic pathways (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Ryan and Cate 2014</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Kang <em>et al.</em> 2016</a>) and, recently, gene drives (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">DiCarlo <em>et al.</em> 2015</a>; <a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Roggenkamp <em>et al.</em> 2017</a>). We have developed a system for examination of Cas9 editing both in a haploid genome and in the diploid state (for gene drive activity). This model system provides several important benefits to the study of gene drives: the risk of unintended (or malicious) drive release is minimized because laboratory yeast do not sporulate well, are not airborne and, as we have documented here, can be programmed with multiple genetic safeguards that render any escaped drive inviable. These simple, yet powerful genetic additions (our artificial “unique” target sequences and self-excising sequences flanking Cas9 itself) ensure no native yeast strain or population would ever be inappropriately targeted by the gene drive. Moreover, as the Church laboratory demonstrated (<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">DiCarlo <em>et al.</em> 2015</a>), and we document as well, separation of the sgRNA (on a plasmid) serves as a potent natural failsafe for studying of gene drives in yeast. Finally, the rapid lifecycle, molecular tools available to the yeast community, and high-throughput infrastructure (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Duina <em>et al.</em> 2014</a>) allow for a true exploration and investigation of hundreds, if not thousands, of gene drive arrangements to be tested. Given the technical challenges to constructing only one or two viable drive systems in insects (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Champer <em>et al.</em> 2017</a>), it would be extremely difficult to explore more than a single variable given the challenging and time-consuming <em>in vivo</em> systems in arthropods or higher eukaryotes.</p><p id="p-63">Here, we explored four conserved mechanisms of the CRISPR system and demonstrate that all four are independent means to titrate gene drive activity within our yeast system. We envision further development of each of these modes of control to varying degrees. While an inducible promoter system driving Cas9 transcription may not be a practical solution to study/test in an insect model, this could still be useful in promoter choice, and possibly complex modulation of the Cas9 promoter itself. For instance, we envision that multiple layers of transcriptional regulation could be assembled onto the “primary” Cas9 nuclease used to initiate the gene drive by a secondary copy of dCas9 fused to either an activator or repressor to modulate expression. Moreover, numerous evolved variants of Cas9 are now paired with either an inducible promoter or external stimuli including small molecules, temperature, and even light (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Dow <em>et al.</em> 2015</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Polstein and Gersbach 2015</a>; <a id="xref-ref-124-1" class="xref-bibr" href="#ref-124">Zetsche <em>et al.</em> 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cao <em>et al.</em> 2016</a>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Liu <em>et al.</em> 2016</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Nuñez <em>et al.</em> 2016</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Richter <em>et al.</em> 2016</a>).</p><p id="p-64">Our sampling of sgRNA lengths and identities was by no means comprehensive. A plethora of studies have extensively tested many variables surrounding sgRNA design: single or two-part guides, length, mismatch, deletions, stem-loop identities, and chemical modifications or fusions (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Briner <em>et al.</em> 2014</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Lin <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Anderson <em>et al.</em> 2015</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Dang <em>et al.</em> 2015</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Farboud and Meyer 2015</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Moreno-Mateos <em>et al.</em> 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Briner and Barrangou 2016</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Doench <em>et al.</em> 2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Fu <em>et al.</em> 2016</a>; <a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Smith <em>et al.</em> 2016</a>). Based on our limited set, and given further study, we are hopeful that there is a viable population of guide RNA options that might produce a titratable level of editing (like our 19 bp guide with a single 5′ G→A mismatch) in the context of a drive system. Importantly, other groups have also reported varying levels of editing given distinct mismatches within the two most 5′ bases of the crRNA sequence (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Anderson <em>et al.</em> 2015</a>; <a id="xref-ref-127-2" class="xref-bibr" href="#ref-127">Zheng <em>et al.</em> 2017</a>) as we have demonstrated in this study. This might be explained by varied expression, stability, and/or Cas9 loading of the RNA. The commonly used U6 promoter requires a 5′ G base pair for expression (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cong <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Fu <em>et al.</em> 2013</a>), although crRNA transcript levels might also be affected by other positions within the guide, including the seed region (<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">Wu <em>et al.</em> 2014</a>).</p><p id="p-65">The primary finding of our study involves titration of Cas9 nuclear occupancy through the active nucleocytoplasmic shuttling achieved by presentation of multiple NLS or NES signal(s). Nuclear Cas9/sgRNA complex residence time has been shown to limit editing efficiency (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Ma <em>et al.</em> 2016a</a>). Given that the mechanisms for nuclear transport of proteins are conserved (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Lange <em>et al.</em> 2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Freitas and Cunha 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bauer <em>et al.</em> 2015</a>) and that canonical signal sequences including, but not restricted to, the SV40 NLS (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Kalderon <em>et al.</em> 1984</a>) are used across model systems, we envision this as a viable and rich option for modulation of editing in gene drives. The increased contribution to promoting Cas9 editing has already been demonstrated in other cell types and current Cas9 systems use two or three NLS sequences to maximize editing (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Ménoret <em>et al.</em> 2015</a>; <a id="xref-ref-111-2" class="xref-bibr" href="#ref-111">Torres-Ruiz <em>et al.</em> 2017</a>). In our designed system, using the potent <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a>/10</em> promoter, a single SV40 NLS appeared sufficient to direct trafficking of Cas9 in either haploid cells or diploid gene drive setups. However, our work demonstrates that, like other cell systems that often require more than one nuclear signal, multiple NLS sequences provide a more robust import signal when challenged with either a suboptimal degree of Cas9 expression and/or opposing nuclease export signal. Of note, our genetically encoded Cas9 system uses one of the highest expressed promoters in yeast (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Flick and Johnston 1990</a>), and other CRISPR editing systems utilize a variety of means of Cas9 delivery, including chromosomally encoded Cas9, plasmid-expressed, or microinjected purified Cas9/sgRNA ribonucleoprotein. Furthermore, different cell types have been shown to display highly variable Cas9 localizations regardless of the presence of one or two NLS sequences (<a id="xref-ref-111-3" class="xref-bibr" href="#ref-111">Torres-Ruiz <em>et al.</em> 2017</a>), making a direct comparison across systems challenging. Finally, the placement of NLS signal sequences (distance from the nuclease coding sequence) may also be a factor in accessibility of nuclear import machinery (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Shen <em>et al.</em> 2013</a>) as well as context to the fused protein of interest (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Nelson and Silver 1989</a>). However, as a general strategy, our study and others (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Cong <em>et al.</em> 2013</a>) have concluded that the addition of more than one NLS sequence can serve to increase nuclear localization and editing, and may buffer against other factors that could interfere with optimal import. The identification and characterization of endogenous NLS signal sequences (<a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">Weninger <em>et al.</em> 2015</a>) specific to the organism of interest would also provide an additional suite of options for either optimized or titratable nuclear import and subsequent Cas9 editing. Using the dynamic nuclear import/export of Cas9, we have demonstrated that both the level of Cas9 and its nuclear occupancy can modulate drive activity over a wide range of efficiencies. In fact, one possible mechanism for shutoff (or reduction) of Cas9 editing could be induced attachment or recruitment of a NES-containing peptide or protein and subsequent nuclear exclusion. This could even be coupled with the newly discovered “anti-CRISPR” family of short peptides that serve to directly bind to and inhibit Cas9 nuclease activity (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Pawluk <em>et al.</em> 2016</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Dong <em>et al.</em> 2017</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Rauch <em>et al.</em> 2017</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Shin <em>et al.</em> 2017</a>; <a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">Yang and Patel 2017</a>).</p><p id="p-66">Finally, we have piloted the use of a set of Cas9-Cas9 fusions to illustrate that (i) dCas9 can compete with native nuclease-active Cas9 and (ii) the large size of a tandem active fusion can partially impede editing. We recognize that alternative Cas9 orthologs (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Esvelt <em>et al.</em> 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Karvelis <em>et al.</em> 2017</a>) would also provide a unique mode of titration between one active and one dead nuclease (or other combinations therein) competing for identical or nearby target sequences. Our system was developed because (i) only a single sgRNA cassette was required, (ii) only a single gene module was needed to express the dual Cas9 fusion (rather than two separate or identical promoters), (iii) a flexible linker provided both the N- and C-terminal nucleases ample spacing, and (iv) addition of coding sequence as part of a gene fusion could be directly applicable to gene drive systems in other organisms. We are excited about the potential that unique Cas9 fusions may serve within the context of gene drives given the rapid explosion of new variants that currently exist. Indeed, dCas9 has provided the expansion of an entirely new field of fusing other enzymes of interest from DNA modifying enzymes, to transcriptional regulators, to “base editors,” to fluorescent proteins, to other nuclease enzymes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Guilinger <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bolukbasi <em>et al.</em> 2015</a>; <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Ma <em>et al.</em> 2015</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Thakore <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chaikind <em>et al.</em> 2016</a>; <a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Nuñez <em>et al.</em> 2016</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Clements <em>et al.</em> 2017</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Kim <em>et al.</em> 2017</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Lin <em>et al.</em> 2017</a>; <a id="xref-ref-128-1" class="xref-bibr" href="#ref-128">Zong <em>et al.</em> 2017</a>). Similarly, placement of different arrangements of Cas9 fusions (or expressed as separate proteins, or a cleavable fusion), different linker lengths or restrictions between fused proteins, or the presence of other nonnuclease modifying enzymes or tags within the drive system could aid to optimize, inhibit, or, as we have demonstrated, titrate the level of overall drive activity.</p><p id="p-67">Given the technical, societal, and ethical challenges facing application of gene drives in the wild, additional study in controlled laboratory settings is critical. Our yeast drive system represents a safe, contained, and rapid testing platform to explore the numerous new Cas9 variants, sgRNA arrangements, and the subcellular trafficking of the Cas9/sgRNA complex to identify new means for future control, regulation, or inhibition in fungi, plant, or metazoan hosts and possible application in wild populations.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-68">We thank David Brown and Salvador Valdez (Kansas State University) for their laboratory assistance on this study and Jeremy Thorner, Angela Duvalyan, and Elizabeth Liao (University of California, Berkeley) for useful discussions. This project was supported by an Institutional Development Award (IDeA) from the National Institute of General Medical Sciences of the National Institutes of Health under grant number P20 GM103418 and USDA National Institute of Food and Agriculture, Hatch Project 1013520 (both to G.C.F.). This work was also supported by an Innovative Research Award (to G.C.F.) and an Undergraduate Cancer Research Award (to E.T.) from the Johnson Cancer Research Center at Kansas State University. Additional support included Undergraduate Research Awards from the College of Arts and Sciences at Kansas State University (to E.R., R.M.G., and M.N.S.). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of General Medical Sciences or the National Institutes of Health.</p><p id="p-69">Author contributions: R.M.G., E.R., M.N.S., E.T., M.H., and G.C.F. built all reagents (plasmids and yeast) and performed all experiments. E.R., R.M.G., and G.C.F. performed all data analyses and figure preparation. G.C.F. wrote the article. Regarding the technology presented in this article, G.C.F. (Kansas State Univ.) filed for a provisional patent on September 29, 2017 (U.S. Provisional Patent Application Serial No. 62/565,651, “Programmed modulation of CRISPR/Cas9 activity”), followed by a full patent filing on January 31, 2018.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-19"><p id="p-70">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300557/-/DC1</a>.</p></li><li class="fn" id="fn-20"><p id="p-71"><em>Communicating editor: A. Dudley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-23"><span class="received-label">Received </span>August 23, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-01-16"><span class="accepted-label">Accepted </span>January 16, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Roggenkamp <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.1" data-doi="10.1001/jama.2016.8830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abbasi</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">National academies hit the brakes on gene drive-modified organisms.</span> <abbr class="cit-jnl-abbrev">JAMA</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">482</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft.volume%253D316%26rft.spage%253D482%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2016.8830%26rft_id%253Dinfo%253Apmid%252F27410484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.2016.8830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27410484&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adelman</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akbari</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloss</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Rules of the road for insect gene drive research and testing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D35%26rft.spage%253D716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.3" data-doi="10.1126/science.aac7932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akbari</span>  <span class="cit-name-given-names">O. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellen</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bullock</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BIOSAFETY. Safeguarding gene drive experiments in the laboratory.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">349</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac7932%26rft_id%253Dinfo%253Apmid%252F26229113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDkvNjI1MS85MjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haupt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiel</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Systematic analysis of CRISPR-Cas9 mismatch tolerance reveals low levels of off-target activity.</span> <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiotechnol.%26rft.volume%253D211%26rft.spage%253D56%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baltes</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gil-Humanes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome engineering and agriculture: opportunities and challenges.</span> <abbr class="cit-jnl-abbrev">Prog. Mol. Biol. Transl. Sci.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProg.%2BMol.%2BBiol.%2BTransl.%2BSci.%26rft.volume%253D149%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.6" data-doi="10.1038/nbt.3659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrangou</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Applications of CRISPR technologies in research and beyond.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">933</span>–<span class="cit-lpage">941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D933%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.7" data-doi="10.1111/tra.12310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doetsch</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mechanisms regulating protein localization.</span> <abbr class="cit-jnl-abbrev">Traffic</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1039</span>–<span class="cit-lpage">1061</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTraffic%26rft.volume%253D16%26rft.spage%253D1039%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftra.12310%26rft_id%253Dinfo%253Apmid%252F26172624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tra.12310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26172624&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaghton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaghton</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Vector control with driving Y chromosomes: modelling the evolution of resistance.</span> <abbr class="cit-jnl-abbrev">Malar. J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalar.%2BJ.%26rft.volume%253D16%26rft.spage%253D286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">The primary structure of the Saccharomyces cerevisiae gene for alcohol dehydrogenase.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">257</span>: <span class="cit-fpage">3018</span>–<span class="cit-lpage">3025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBennetzen%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D257%26rft.issue%253D6%26rft.spage%253D3018%26rft.epage%253D3025%26rft.atitle%253DThe%2Bprimary%2Bstructure%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgene%2Bfor%2Balcohol%2Bdehydrogenase%26rft_id%253Dinfo%253Apmid%252F6277922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNTcvNi8zMDE4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.10" data-doi="10.1038/nmeth.3624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolukbasi</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oikemus</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derr</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA-binding-domain fusions enhance the targeting range and precision of Cas9.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1150</span>–<span class="cit-lpage">1156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D1150%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3624%26rft_id%253Dinfo%253Apmid%252F26480473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3624&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26480473&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.3.999.11" data-doi="10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachmann</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cost</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Designer deletion strains derived from <em>Saccharomyces cerevisiae</em> S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBrachmann%26rft.auinit1%253DC.%2BB.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D132%26rft.atitle%253DDesigner%2Bdeletion%2Bstrains%2Bderived%2Bfrom%2BSaccharomyces%2Bcerevisiae%2BS288C%253A%2Ba%2Buseful%2Bset%2Bof%2Bstrains%2Band%2Bplasmids%2Bfor%2BPCR-mediated%2Bgene%2Bdisruption%2Band%2Bother%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%252819980130%252914%253A2%253C115%253A%253AAID-YEA204%253E3.0.CO%253B2-2%26rft_id%253Dinfo%253Apmid%252F9483801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9483801&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071817800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.12" data-doi="10.1101/pdb.top090902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Briner</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrangou</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Guide RNAs: a glimpse at the sequences that drive CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Protoc.</abbr> <span class="cit-vol">2016</span>: <span class="cit-fpage">pdb.top090902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BProtocols%26rft.stitle%253DCold%2BSpring%2BHarb%2BProtoc%26rft.aulast%253DBriner%26rft.auinit1%253DA.%2BE.%26rft.volume%253D2016%26rft.issue%253D7%26rft.spage%253Dpdb.top090902%26rft.epage%253Dpdb.top090902%26rft.atitle%253DGuide%2BRNAs%253A%2BA%2BGlimpse%2Bat%2Bthe%2BSequences%2Bthat%2BDrive%2BCRISPR-Cas%2BSystems%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fpdb.top090902%26rft_id%253Dinfo%253Apmid%252F27371605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicHJvdG9jb2xzIjtzOjU6InJlc2lkIjtzOjIwOiIyMDE2LzcvcGRiLnRvcDA5MDkwMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.13" data-doi="10.1016/j.molcel.2014.09.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Briner</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donohoue</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomaa</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selle</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slorach</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Guide RNA functional modules direct Cas9 activity and orthogonality.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D56%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.09.019%26rft_id%253Dinfo%253Apmid%252F25373540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.09.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25373540&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000344484600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.14" data-doi="10.1098/rspb.2002.2319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Site-specific selfish genes as tools for the control and genetic engineering of natural populations.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">270</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DBurt%26rft.auinit1%253DA.%26rft.volume%253D270%26rft.issue%253D1518%26rft.spage%253D921%26rft.epage%253D928%26rft.atitle%253DSite-specific%2Bselfish%2Bgenes%2Bas%2Btools%2Bfor%2Bthe%2Bcontrol%2Band%2Bgenetic%2Bengineering%2Bof%2Bnatural%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2002.2319%26rft_id%253Dinfo%253Apmid%252F12803906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjcwLzE1MTgvOTIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.15" data-doi="10.1093/nar/gkw660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An easy and efficient inducible CRISPR/Cas9 platform with improved specificity for multiple gene targeting.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw660%26rft_id%253Dinfo%253Apmid%252F27458201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw660&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27458201&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.16" data-doi="10.1093/nar/gkw707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaikind</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A programmable Cas9-serine recombinase fusion protein that operates on DNA sequences in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">9758</span>–<span class="cit-lpage">9770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw707%26rft_id%253Dinfo%253Apmid%252F27515511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw707&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27515511&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Champer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeves</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Novel CRISPR/Cas9 gene drive constructs reveal insights into mechanisms of resistance allele formation and drive efficiency in genetically diverse populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1006796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.18" data-doi="10.1016/0378-1119(92)90454-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christianson</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shero</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Multifunctional yeast high-copy-number shuttle vectors.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DChristianson%26rft.auinit1%253DT.%2BW.%26rft.volume%253D110%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D122%26rft.atitle%253DMultifunctional%2Byeast%2Bhigh-copy-number%2Bshuttle%2Bvectors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252892%252990454-W%26rft_id%253Dinfo%253Apmid%252F1544568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(92)90454-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1544568&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HE64700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lintel</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">RICE CRISPR: rapidly increased cut ends by an exonuclease Cas9 fusion in zebrafish.</span> <abbr class="cit-jnl-abbrev">Genesis</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">e23044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenesis%26rft.volume%253D55%26rft.spage%253De23044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.20" data-doi="10.1126/science.1231143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barretto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiplex genome engineering using CRISPR/Cas systems.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">819</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1231143%26rft_id%253Dinfo%253Apmid%252F23287718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjEyMS84MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.21" data-doi="10.15252/embr.201744205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Courtier-Orgogozo</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morizot</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boete</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Agricultural pest control with CRISPR-based gene drive: time for public debate: should we use gene drive for pest control?</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">878</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembr.201744205%26rft_id%253Dinfo%253Apmid%252F28512121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6ODoiMTgvNi84NzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.22" data-doi="10.15252/embr.201744822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Courtier-Orgogozo</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morizot</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boete</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Using CRISPR-based gene drive for agriculture pest control.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembr.201744822%26rft_id%253Dinfo%253Apmid%252F28784604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6OToiMTgvOS8xNDgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.23" data-doi="10.1186/s13059-015-0846-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anaya</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimizing sgRNA structure to improve CRISPR-Cas9 knockout efficiency.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D280%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0846-3%26rft_id%253Dinfo%253Apmid%252F26671237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0846-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26671237&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.24" data-doi="10.1093/nar/gkt135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DiCarlo</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norville</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome engineering in <em>Saccharomyces cerevisiae</em> using CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">4336</span>–<span class="cit-lpage">4343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt135%26rft_id%253Dinfo%253Apmid%252F23460208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23460208&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318167900046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.25" data-doi="10.1038/nbt.3412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DiCarlo</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietz</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Safeguarding CRISPR-Cas9 gene drives in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1250</span>–<span class="cit-lpage">1255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D1250%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3412%26rft_id%253Dinfo%253Apmid%252F26571100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26571100&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.26" data-doi="10.1038/nbt.3437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doench</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fusi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullender</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegde</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaimberg</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3437%26rft_id%253Dinfo%253Apmid%252F26780180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26780180&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.27" data-doi="10.1038/nature22377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Structural basis of CRISPR-SpyCas9 inhibition by an anti-CRISPR protein.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D436%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22377%26rft_id%253Dinfo%253Apmid%252F28448066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28448066&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.28" data-doi="10.1126/science.1258096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome editing. The new frontier of genome engineering with CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">1258096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDoudna%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D346%26rft.issue%253D6213%26rft.spage%253D1258096%26rft.epage%253D1258096%26rft.atitle%253DThe%2Bnew%2Bfrontier%2Bof%2Bgenome%2Bengineering%2Bwith%2BCRISPR-Cas9%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1258096%26rft_id%253Dinfo%253Apmid%252F25430774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDYvNjIxMy8xMjU4MDk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.29" data-doi="10.1038/nbt.3155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dow</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Rourke</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kastenhuber</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inducible <em>in vivo</em> genome editing with CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D390%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3155%26rft_id%253Dinfo%253Apmid%252F25690852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25690852&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.30" data-doi="10.1126/sciadv.1601910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drury</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dapper</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siniard</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zentner</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CRISPR/Cas9 gene drives in genetically variable and nonrandomly mating wild populations.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e1601910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DDrury%26rft.auinit1%253DD.%2BW.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253De1601910%26rft.epage%253De1601910%26rft.atitle%253DCRISPR%252FCas9%2Bgene%2Bdrives%2Bin%2Bgenetically%2Bvariable%2Band%2Bnonrandomly%2Bmating%2Bwild%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1601910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMy81L2UxNjAxOTEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.31" data-doi="10.1534/genetics.114.163188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Budding yeast for budding geneticists: a primer on the <em>Saccharomyces cerevisiae</em> model system.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDuina%26rft.auinit1%253DA.%2BA.%26rft.volume%253D197%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D48%26rft.atitle%253DBudding%2BYeast%2Bfor%2BBudding%2BGeneticists%253A%2BA%2BPrimer%2Bon%2Bthe%2BSaccharomyces%2Bcerevisiae%2BModel%2BSystem%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.163188%26rft_id%253Dinfo%253Apmid%252F24807111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk3LzEvMzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.32" data-doi="10.1002/yea.2912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert-Boulet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogh</span>  <span class="cit-name-given-names">B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisby</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Optimization of ordered plasmid assembly by gap repair in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DEckert-Boulet%26rft.auinit1%253DN.%26rft.volume%253D29%26rft.issue%253D8%26rft.spage%253D323%26rft.epage%253D334%26rft.atitle%253DOptimization%2Bof%2Bordered%2Bplasmid%2Bassembly%2Bby%2Bgap%2Brepair%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.2912%26rft_id%253Dinfo%253Apmid%252F22806834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.2912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22806834&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Estrela</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cate</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Energy biotechnology in the CRISPR-Cas9 era.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BBiotechnol.%26rft.volume%253D38%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemmell</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Conservation demands safe gene drive.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">e2003850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D15%26rft.spage%253De2003850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.35" data-doi="10.1038/nmeth.2681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braff</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moosburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yaung</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Orthogonal Cas9 proteins for RNA-guided gene regulation and editing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1116</span>–<span class="cit-lpage">1121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D1116%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2681%26rft_id%253Dinfo%253Apmid%252F24076762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24076762&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326507600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.36" data-doi="10.1534/genetics.115.175166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farboud</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dramatic enhancement of genome editing by CRISPR/Cas9 through improved guide RNA design.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175166%26rft_id%253Dinfo%253Apmid%252F25695951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzQvOTU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finnigan</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Complex <em>in vivo</em> ligation using homologous recombination and high-efficiency plasmid rescue from <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Bio Protoc.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBio%2BProtoc.%26rft.volume%253D5%26rft.spage%253D1521e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finnigan</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">mCAL: a new approach for versatile multiplex action of Cas9 using one sgRNA and loci flanked by a programmed target sequence.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2147</span>–<span class="cit-lpage">2156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D2147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.39" data-doi="10.1091/mbc.E16-03-0177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finnigan</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterling</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duvalyan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargsyan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Coordinate action of distinct sequence elements localizes checkpoint kinase Hsl1 to the septin collar at the bud neck in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2213</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E16-03-0177%26rft_id%253Dinfo%253Apmid%252F27193302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xNC8yMjEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.40" data-doi="10.1128/MCB.10.9.4757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flick</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Two systems of glucose repression of the GAL1 promoter in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">4757</span>–<span class="cit-lpage">4769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFlick%26rft.auinit1%253DJ%2BS%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253D4757%26rft.epage%253D4769%26rft.atitle%253DTwo%2Bsystems%2Bof%2Bglucose%2Brepression%2Bof%2Bthe%2BGAL1%2Bpromoter%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.9.4757%26rft_id%253Dinfo%253Apmid%252F2201902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzkvNDc1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.41" data-doi="10.2174/138920209789503941"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freitas</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunha</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mechanisms and signals for the nuclear import of proteins.</span> <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DFreitas%26rft.auinit1%253DN.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D550%26rft.epage%253D557%26rft.atitle%253DMechanisms%2Band%2Bsignals%2Bfor%2Bthe%2Bnuclear%2Bimport%2Bof%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920209789503941%26rft_id%253Dinfo%253Apmid%252F20514217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920209789503941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20514217&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271381500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.42" data-doi="10.1093/nar/gkw417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Distinct patterns of Cas9 mismatch tolerance <em>in vitro</em> and <em>in vivo</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">5365</span>–<span class="cit-lpage">5377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw417%26rft_id%253Dinfo%253Apmid%252F27198218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27198218&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.43" data-doi="10.1038/nbt.2623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foden</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khayter</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeder</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-frequency off-target mutagenesis induced by CRISPR-Cas nucleases in human cells.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">822</span>–<span class="cit-lpage">826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D822%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2623%26rft_id%253Dinfo%253Apmid%252F23792628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23792628&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.44" data-doi="10.1073/pnas.1521077112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gantz</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasinskiene</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatarenkova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fazekas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macias</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Highly efficient Cas9-mediated gene drive for population modification of the malaria vector mosquito <em>Anopheles stephensi</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">E6736</span>–<span class="cit-lpage">E6743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1521077112%26rft_id%253Dinfo%253Apmid%252F26598698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzQ5L0U2NzM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charles</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Complex transcriptional modulation with orthogonal and inducible dCas9 regulators.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1043</span>–<span class="cit-lpage">1049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D1043%26rft_id%253Dinfo%253Apmid%252F27776111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=27776111&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.46" data-doi="10.1016/j.cell.2013.06.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morsut</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">442</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D442%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.06.044%26rft_id%253Dinfo%253Apmid%252F23849981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.06.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23849981&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321950700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Godfray</span>  <span class="cit-name-given-names">H. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">North</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">How driving endonuclease genes can be used to combat pests and disease vectors.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D15%26rft.spage%253D81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.48" data-doi="10.1038/nbt.2909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guilinger</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fusion of catalytically inactive Cas9 to FokI nuclease improves the specificity of genome modification.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D577%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2909%26rft_id%253Dinfo%253Apmid%252F24770324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2909&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24770324&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.49" data-doi="10.1371/journal.pone.0016744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">X. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of an amphipathic helix important for the formation of ectopic septin spirals and axial budding in yeast axial landmark protein Bud3p.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e16744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGuo%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De16744%26rft.epage%253De16744%26rft.atitle%253DIdentification%2Bof%2Ban%2Bamphipathic%2Bhelix%2Bimportant%2Bfor%2Bthe%2Bformation%2Bof%2Bectopic%2Bseptin%2Bspirals%2Band%2Baxial%2Bbudding%2Bin%2Byeast%2Baxial%2Blandmark%2Bprotein%2BBud3p.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016744%26rft_id%253Dinfo%253Apmid%252F21408200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21408200&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.50" data-doi="10.1038/nbt.3439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galizi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kyrou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simoni</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siniscalchi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A CRISPR-Cas9 gene drive system targeting female reproduction in the malaria mosquito vector <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3439%26rft_id%253Dinfo%253Apmid%252F26641531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3439&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26641531&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kyrou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruttini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">North</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galizi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The creation and selection of mutations resistant to a gene drive over multiple generations in the malaria mosquito.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1007039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1007039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.52" data-doi="10.1091/mbc.E15-10-0721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heasley</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMurray</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Roles of septins in prospore membrane morphogenesis and spore wall assembly in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">442</span>–<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E15-10-0721%26rft_id%253Dinfo%253Apmid%252F26680739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI3LzMvNDQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.53" data-doi="10.1038/nbt.2647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstein</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konermann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DNA targeting specificity of RNA-guided Cas9 nucleases.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D827%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2647%26rft_id%253Dinfo%253Apmid%252F23873081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23873081&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakociunas</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arsovska</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Transcriptional reprogramming in yeast using dCas9 and combinatorial gRNA strategies.</span> <abbr class="cit-jnl-abbrev">Microb. Cell Fact.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob.%2BCell%2BFact.%26rft.volume%253D16%26rft.spage%253D46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.55" data-doi="10.1126/science.aad8282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornfeld</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Structures of a CRISPR-Cas9 R-loop complex primed for DNA cleavage.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">867</span>–<span class="cit-lpage">871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aad8282%26rft_id%253Dinfo%253Apmid%252F26841432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTEvNjI3NS84NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.56" data-doi="10.1126/science.1225829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chylinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonfara</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1225829%26rft_id%253Dinfo%253Apmid%252F22745249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5Ni84MTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.57" data-doi="10.7554/eLife.00471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">East</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA-programmed genome editing in human cells.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D2%26rft.issue%253D0%26rft.spage%253De00471%26rft.epage%253De00471%26rft.atitle%253DRNA-programmed%2Bgenome%2Bediting%2Bin%2Bhuman%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.00471%26rft_id%253Dinfo%253Apmid%252F23386978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjIvMC9lMDA0NzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.58" data-doi="10.1126/science.1247997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaya</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Structures of Cas9 endonucleases reveal RNA-mediated conformational activation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">1247997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1247997%26rft_id%253Dinfo%253Apmid%252F24505130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDMvNjE3Ni8xMjQ3OTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jonsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaartinen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Straub</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Relationships between natural enemy diversity and biological control.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Insect Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BInsect%2BSci.%26rft.volume%253D20%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.60" data-doi="10.1016/0092-8674(84)90457-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalderon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">A short amino acid sequence able to specify nuclear location.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKalderon%26rft.auinit1%253DD.%26rft.volume%253D39%26rft.issue%253D3%2BPt%2B2%26rft.spage%253D499%26rft.epage%253D509%26rft.atitle%253DA%2Bshort%2Bamino%2Bacid%2Bsequence%2Bable%2Bto%2Bspecify%2Bnuclear%2Blocation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252884%252990457-4%26rft_id%253Dinfo%253Apmid%252F6096007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(84)90457-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6096007&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984ABW9700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlop-Powers</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multiplexed CRISPR/Cas9- and TAR-mediated promoter engineering of natural product biosynthetic gene clusters in yeast.</span> <abbr class="cit-jnl-abbrev">ACS Synth. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1002</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol.%26rft.volume%253D5%26rft.spage%253D1002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karvelis</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasiunas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siksnys</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Harnessing the natural diversity and <em>in vitro</em> evolution of Cas9 to expand the genome editing toolbox.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BMicrobiol.%26rft.volume%253D37%26rft.spage%253D88%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kergunteuil</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakhtiari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Formenti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Defossez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Biological control beneath the feet: a review of crop protection against insect root herbivores.</span> <abbr class="cit-jnl-abbrev">Insects</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsects%26rft.volume%253D7%26rft.spage%253D70%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Midega</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooper</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickett</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Push-pull: chemical ecology-based integrated pest management technology.</span> <abbr class="cit-jnl-abbrev">J. Chem. Ecol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BEcol.%26rft.volume%253D42%26rft.spage%253D689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.65" data-doi="10.1038/nbt.3803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Komor</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Packer</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increasing the genome-targeting scope and precision of base editing with engineered Cas9-cytidine deaminase fusions.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D35%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3803%26rft_id%253Dinfo%253Apmid%252F28191901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28191901&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.66" data-doi="10.1038/nbt.2800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koike-Yusa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velasco-Herrera Mdel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yusa</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide recessive genetic screening in mammalian cells with a lentiviral CRISPR-guide RNA library.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D267%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2800%26rft_id%253Dinfo%253Apmid%252F24535568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24535568&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.67" data-doi="10.1016/j.mib.2017.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarova</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Diversity, classification and evolution of CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BMicrobiol.%26rft.volume%253D37%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2017.05.008%26rft_id%253Dinfo%253Apmid%252F28605718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2017.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28605718&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.3.999.68" data-doi="10.1111/j.1600-0854.2008.00825.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosugi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasebe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanagawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Nuclear export signal consensus sequences defined using a localization-based yeast selection system.</span> <abbr class="cit-jnl-abbrev">Traffic</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2053</span>–<span class="cit-lpage">2062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTraffic%2B%2528Copenhagen%252C%2BDenmark%2529%26rft.stitle%253DTraffic%26rft.aulast%253DKosugi%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D2053%26rft.epage%253D2062%26rft.atitle%253DNuclear%2Bexport%2Bsignal%2Bconsensus%2Bsequences%2Bdefined%2Busing%2Ba%2Blocalization-based%2Byeast%2Bselection%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1600-0854.2008.00825.x%26rft_id%253Dinfo%253Apmid%252F18817528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1600-0854.2008.00825.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18817528&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261086000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.69" data-doi="10.1074/jbc.R600026200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devine</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Classical nuclear localization signals: definition, function, and interaction with importin alpha.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">5101</span>–<span class="cit-lpage">5105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R600026200%26rft_id%253Dinfo%253Apmid%252F17170104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODIvOC81MTAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolund</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Fusion of SpCas9 to <em>E. coli</em> Rec A protein enhances CRISPR-Cas9 mediated gene knockout in mammalian cells.</span> <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">247</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiotechnol.%26rft.volume%253D247%26rft.spage%253D42%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.71" data-doi="10.1093/nar/gku402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cradick</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshmukh</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CRISPR/Cas9 systems have off-target activity with insertions or deletions between target DNA and guide RNA sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">7473</span>–<span class="cit-lpage">7485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku402%26rft_id%253Dinfo%253Apmid%252F24838573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24838573&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338769400063&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramli</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A chemical-inducible CRISPR-Cas9 system for rapid control of genome editing.</span> <abbr class="cit-jnl-abbrev">Nat. Chem. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">980</span>–<span class="cit-lpage">987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BChem.%2BBiol.%26rft.volume%253D12%26rft.spage%253D980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.73" data-doi="10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">A.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demarini</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Additional modules for versatile and economical PCR-based gene deletion and modification in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DLongtine%26rft.auinit1%253DM.%2BS.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D953%26rft.epage%253D961%26rft.atitle%253DAdditional%2Bmodules%2Bfor%2Bversatile%2Band%2Beconomical%2BPCR-based%2Bgene%2Bdeletion%2Band%2Bmodification%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199807%252914%253A10%253C953%253A%253AAID-YEA293%253E3.0.CO%253B2-U%26rft_id%253Dinfo%253Apmid%252F9717241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9717241&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075005100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.74" data-doi="10.1073/pnas.1420024112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naseri</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Gutierrez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multicolor CRISPR labeling of chromosomal loci in human cells.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">3002</span>–<span class="cit-lpage">3007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1420024112%26rft_id%253Dinfo%253Apmid%252F25713381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzEwLzMwMDIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.75" data-doi="10.1083/jcb.201604115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naseri</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">CRISPR-Cas9 nuclear dynamics and target recognition in living cells.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201604115%26rft_id%253Dinfo%253Apmid%252F27551060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIxNC81LzUyOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.76" data-doi="10.1038/nbt.3526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naseri</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huisman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Multiplexed labeling of genomic loci with dCas9 and engineered sgRNAs using CRISPRainbow.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D528%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3526%26rft_id%253Dinfo%253Apmid%252F27088723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27088723&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akbari</span>  <span class="cit-name-given-names">O. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Overcoming evolved resistance to population-suppressing homing-based gene drives.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D3776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.78" data-doi="10.1038/srep14410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ménoret</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Cian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Homology-directed repair in rodent zygotes using Cas9 and TALEN engineered proteins.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">14410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D14410%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep14410%26rft_id%253Dinfo%253Apmid%252F26442875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep14410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26442875&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.79" data-doi="10.1126/science.aad5147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohanraju</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarova</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zetsche</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Diverse evolutionary roots and mechanistic variations of the CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>: <span class="cit-fpage">aad5147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMohanraju%26rft.auinit1%253DP.%26rft.volume%253D353%26rft.issue%253D6299%26rft.spage%253Daad5147%26rft.epage%253Daad5147%26rft.atitle%253DDiverse%2Bevolutionary%2Broots%2Band%2Bmechanistic%2Bvariations%2Bof%2Bthe%2BCRISPR-Cas%2Bsystems%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aad5147%26rft_id%253Dinfo%253Apmid%252F27493190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTMvNjI5OS9hYWQ1MTQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.80" data-doi="10.1038/nmeth.3543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Mateos</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vejnar</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaudoin</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mis</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting <em>in vivo</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">982</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D982%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3543%26rft_id%253Dinfo%253Apmid%252F26322839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26322839&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariano</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bermudez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arruda</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Manipulation of nuclear architecture through CRISPR-mediated chromosomal looping.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">15993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D15993%26rft_id%253Dinfo%253Apmid%252F28703221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=28703221&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.82" data-doi="10.1128/MCB.9.2.384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Context affects nuclear protein localization in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">384</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNelson%26rft.auinit1%253DM%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D384%26rft.epage%253D389%26rft.atitle%253DContext%2Baffects%2Bnuclear%2Bprotein%2Blocalization%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.9.2.384%26rft_id%253Dinfo%253Apmid%252F2496300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjc6IjkvMi8zODQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.83" data-doi="10.1016/j.cell.2014.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishimasu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konermann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shehata</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Crystal structure of Cas9 in complex with guide RNA and target DNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D156%26rft.spage%253D935%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.02.001%26rft_id%253Dinfo%253Apmid%252F24529477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24529477&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332083900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.84" data-doi="10.1126/sciadv.1601964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olejarz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evolutionary dynamics of CRISPR gene drives.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e1601964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DNoble%26rft.auinit1%253DC.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253De1601964%26rft.epage%253De1601964%26rft.atitle%253DEvolutionary%2Bdynamics%2Bof%2BCRISPR%2Bgene%2Bdrives%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1601964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMy80L2UxNjAxOTY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.85" data-doi="10.1093/nar/gkw908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zerez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleris</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Guide RNA engineering for versatile Cas9 functionality.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">9555</span>–<span class="cit-lpage">9564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw908%26rft_id%253Dinfo%253Apmid%252F27733506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27733506&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Numamoto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maekawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaneko</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Efficient genome editing by CRISPR/Cas9 with a tRNA-sgRNA fusion in the methylotrophic yeast <em>Ogataea polymorpha</em>.</span> <abbr class="cit-jnl-abbrev">J. Biosci. Bioeng.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiosci.%2BBioeng.%26rft.volume%253D124%26rft.spage%253D487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.87" data-doi="10.1021/acschembio.5b01019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nuñez</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrington</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Chemical and biophysical modulation of Cas9 for tunable genome engineering.</span> <abbr class="cit-jnl-abbrev">ACS Chem. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BChem.%2BBiol.%26rft.volume%253D11%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facschembio.5b01019%26rft_id%253Dinfo%253Apmid%252F26857072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acschembio.5b01019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26857072&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.88" data-doi="10.1091/mbc.01-06-0308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pante</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Nuclear pore complex is able to transport macromolecules with diameters of about 39 nm.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.01-06-0308%26rft_id%253Dinfo%253Apmid%252F11854401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjEzLzIvNDI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.89" data-doi="10.1016/j.cell.2016.11.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pawluk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amrani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hidalgo-Reyes</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Naturally occurring off-switches for CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1829</span>–<span class="cit-lpage">1838.e9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D167%26rft.spage%253D1829%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.11.017%26rft_id%253Dinfo%253Apmid%252F27984730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.11.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27984730&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ode</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira-Hofman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harwood</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Integration of plant defense traits with biological control of arthropod pests: challenges and opportunities.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D1794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.91" data-doi="10.1038/nchembio.1753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polstein</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gersbach</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A light-inducible CRISPR-Cas9 system for control of endogenous gene activation.</span> <abbr class="cit-jnl-abbrev">Nat. Chem. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BChem.%2BBiol.%26rft.volume%253D11%26rft.spage%253D198%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnchembio.1753%26rft_id%253Dinfo%253Apmid%252F25664691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nchembio.1753&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25664691&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.92"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prowse</span>  <span class="cit-name-given-names">T. A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cassey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfitzner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittmann</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dodging silver bullets: good CRISPR gene-drive design is critical for eradicating exotic vertebrates.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">284</span>: 20170799.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.93" data-doi="10.1016/j.cell.2016.12.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rauch</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silvis</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hultquist</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGregor</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Inhibition of CRISPR-Cas9 with bacteriophage proteins.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">158.e10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D168%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.12.009%26rft_id%253Dinfo%253Apmid%252F28041849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28041849&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Brochta</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Applications of genome editing in insects.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Insect Sci.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BInsect%2BSci.%26rft.volume%253D13%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rénia</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goh</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Malaria parasites: the great escape.</span> <abbr class="cit-jnl-abbrev">Front. Immunol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BImmunol.%26rft.volume%253D7%26rft.spage%253D463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.96" data-doi="10.1093/nar/gkw930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonfara</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouazza</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schumacher</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bratovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Engineering of temperature- and light-switchable Cas9 variants.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">10003</span>–<span class="cit-lpage">10014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw930%26rft_id%253Dinfo%253Apmid%252F27744350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27744350&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Advances in vector control science: rear-and-release strategies show promise... but don’t forget the basics.</span> <abbr class="cit-jnl-abbrev">J. Infect. Dis.</abbr> <span class="cit-vol">215</span>(<span class="cit-supplement">suppl_2</span>): <span class="cit-fpage">S103</span>–<span class="cit-lpage">S108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInfect.%2BDis.%26rft.volume%253D215%26rft.spage%253DS103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.98"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roggenkamp</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giersch</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wedeman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eaton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turnquist</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CRISPR-UnLOCK: multipurpose Cas9-based strategies for conversion of yeast libraries and strains.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D8%26rft.spage%253D1773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.99" data-doi="10.1016/B978-0-12-801185-0.00023-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">O. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cate</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiplex engineering of industrial yeast genomes using CRISPRm.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D546%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-801185-0.00023-4%26rft_id%253Dinfo%253Apmid%252F25398354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-801185-0.00023-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398354&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-book" id="cit-8.3.999.100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source"><em>Molecular Cloning: A Laboratory Manual</em></span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaboli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugiyama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harashima</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CRISPR-PCS: a powerful new approach to inducing multiple chromosome splitting in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">30278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D30278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.102" data-doi="10.1038/s41564-017-0043-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">C. B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaas</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A CRISPR-Cas9-based gene drive platform for genetic interaction analysis in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Microbiol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMicrobiol.%26rft.volume%253D3%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41564-017-0043-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41564-017-0043-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.103" data-doi="10.1038/cr.2013.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Generation of gene-modified mice via Cas9/RNA-mediated gene targeting.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes.%26rft.volume%253D23%26rft.spage%253D720%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2013.46%26rft_id%253Dinfo%253Apmid%252F23545779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2013.46&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23545779&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318483200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.104" data-doi="10.1126/sciadv.1701620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bray</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rauch</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Disabling Cas9 by an anti-CRISPR DNA mimic.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e1701620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DShin%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De1701620%26rft.epage%253De1701620%26rft.atitle%253DDisabling%2BCas9%2Bby%2Ban%2Banti-CRISPR%2BDNA%2Bmimic%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1701620%26rft_id%253Dinfo%253Apmid%252F28706995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMy83L2UxNzAxNjIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.3.999.105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSikorski%26rft.auinit1%253DR.%2BS.%26rft.volume%253D122%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D27%26rft.atitle%253DA%2Bsystem%2Bof%2Bshuttle%2Bvectors%2Band%2Byeast%2Bhost%2Bstrains%2Bdesigned%2Bfor%2Befficient%2Bmanipulation%2Bof%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2659436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTIyLzEvMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.106" data-doi="10.1186/s13059-016-0900-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suresh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlecht</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagih</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Quantitative CRISPR interference screens in yeast identify chemical-genetic interactions and new rules for guide RNA design.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0900-9%26rft_id%253Dinfo%253Apmid%252F26956608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-0900-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26956608&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.107" data-doi="10.1146/annurev-biochem-072911-172315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiedenheft</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CRISPR-mediated adaptive immune systems in bacteria and archaea.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiochem.%26rft.volume%253D82%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-072911-172315%26rft_id%253Dinfo%253Apmid%252F23495939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-072911-172315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23495939&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321698100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stenberg</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heil</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahman</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjorkman</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimizing crops for biocontrol of pests and disease.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">698</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D20%26rft.spage%253D698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.109" data-doi="10.1016/j.molcel.2015.02.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Expanding the Biologist’s toolkit with CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D58%26rft.spage%253D568%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.02.032%26rft_id%253Dinfo%253Apmid%252F26000842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.02.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26000842&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.110" data-doi="10.1038/nmeth.3630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thakore</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Ippolito</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Safi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shivakumar</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Highly specific epigenome editing by CRISPR-Cas9 repressors for silencing of distal regulatory elements.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D1143%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3630%26rft_id%253Dinfo%253Apmid%252F26501517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26501517&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres-Ruiz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Lage</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bueno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Efficient recreation of t(11;22) EWSR1–FLI1+ in human stem cells using CRISPR/Cas9.</span> <abbr class="cit-jnl-abbrev">Stem Cell Reports</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1408</span>–<span class="cit-lpage">1420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStem%2BCell%2BReports%26rft.volume%253D8%26rft.spage%253D1408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.112" data-doi="10.1021/acssynbio.5b00196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsarmpopoulos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gourgues</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vashee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jores</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">In-yeast engineering of a bacterial genome using CRISPR/Cas9.</span> <abbr class="cit-jnl-abbrev">ACS Synth. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol.%26rft.volume%253D5%26rft.spage%253D104%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facssynbio.5b00196%26rft_id%253Dinfo%253Apmid%252F26592087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acssynbio.5b00196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26592087&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.113" data-doi="10.1007/s002940050160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ugolini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruschi</span>  <span class="cit-name-given-names">C. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The red/white colony color assay in the yeast Saccharomyces cerevisiae: epistatic growth advantage of white ade8–18, ade2 cells over red ade2 cells.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DUgolini%26rft.auinit1%253DS.%26rft.volume%253D30%26rft.issue%253D6%26rft.spage%253D485%26rft.epage%253D492%26rft.atitle%253DThe%2Bred%252Fwhite%2Bcolony%2Bcolor%2Bassay%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae%253A%2Bepistatic%2Bgrowth%2Badvantage%2Bof%2Bwhite%2Bade8-18%252C%2Bade2%2Bcells%2Bover%2Bred%2Bade2%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002940050160%26rft_id%253Dinfo%253Apmid%252F8939809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002940050160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8939809&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996WH82700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.114" data-doi="10.1534/genetics.116.197285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unckless</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evolution of resistance against CRISPR/Cas9 gene drive.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197285%26rft_id%253Dinfo%253Apmid%252F27941126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA1LzIvODI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference  in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.115" data-doi="10.1016/j.febslet.2007.05.082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brattain</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The maximal size of protein to diffuse through the nuclear pore is larger than 60kDa.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">581</span>: <span class="cit-fpage">3164</span>–<span class="cit-lpage">3170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DWang%26rft.auinit1%253DR.%26rft.volume%253D581%26rft.issue%253D17%26rft.spage%253D3164%26rft.epage%253D3170%26rft.atitle%253DThe%2Bmaximal%2Bsize%2Bof%2Bprotein%2Bto%2Bdiffuse%2Bthrough%2Bthe%2Bnuclear%2Bpore%2Bis%2Blarger%2Bthan%2B60kDa.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2007.05.082%26rft_id%253Dinfo%253Apmid%252F17588566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2007.05.082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17588566&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248145800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference  in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.116" data-doi="10.1126/science.1246981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatini</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic screens in human cells using the CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1246981%26rft_id%253Dinfo%253Apmid%252F24336569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNDMvNjE2Ni84MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.3.999.117" data-doi="10.1016/0092-8674(95)90435-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meinkoth</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsien</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Identification of a signal for rapid export of proteins from the nucleus.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWen%26rft.auinit1%253DW.%26rft.volume%253D82%26rft.issue%253D3%26rft.spage%253D463%26rft.epage%253D473%26rft.atitle%253DIdentification%2Bof%2Ba%2Bsignal%2Bfor%2Brapid%2Bexport%2Bof%2Bproteins%2Bfrom%2Bthe%2Bnucleus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252895%252990435-2%26rft_id%253Dinfo%253Apmid%252F7634336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(95)90435-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7634336&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RP24200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference  in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.118" data-doi="10.1093/femsyr/fov082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weninger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glieder</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogl</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A toolbox of endogenous and heterologous nuclear localization sequences for the methylotrophic yeast <em>Pichia pastoris</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">fov082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffov082%26rft_id%253Dinfo%253Apmid%252F26347503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsyr/fov082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26347503&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference  in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.119" data-doi="10.1016/bs.pmbts.2017.03.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolt</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Safety, security, and policy considerations for plant genome editing.</span> <abbr class="cit-jnl-abbrev">Prog. Mol. Biol. Transl. Sci.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProg.%2BMol.%2BBiol.%2BTransl.%2BSci.%26rft.volume%253D149%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.pmbts.2017.03.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.pmbts.2017.03.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference  in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.120" data-doi="10.1016/j.cell.2015.12.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunez</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Biology and applications of CRISPR systems: harnessing nature’s toolbox for genome engineering.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D164%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.12.035%26rft_id%253Dinfo%253Apmid%252F26771484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.12.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26771484&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference  in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.121" data-doi="10.1038/nbt.2889"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide binding of the CRISPR endonuclease Cas9 in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">670</span>–<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D670%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2889%26rft_id%253Dinfo%253Apmid%252F24752079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2889&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24752079&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference  in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.122" data-doi="10.1101/gr.191452.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequence determinants of improved CRISPR sgRNA design.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1147</span>–<span class="cit-lpage">1157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.191452.115%26rft_id%253Dinfo%253Apmid%252F26063738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMTE0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference  in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.123" data-doi="10.1016/j.molcel.2017.05.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Inhibition mechanism of an Anti-CRISPR suppressor AcrIIA4 targeting SpyCas9.</span> <abbr class="cit-jnl-abbrev">Mol. Cell.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">127.e5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%26rft.volume%253D67%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2017.05.024%26rft_id%253Dinfo%253Apmid%252F28602637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2017.05.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28602637&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-124-1" title="View reference  in text" id="ref-124">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.124" data-doi="10.1038/nbt.3149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zetsche</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A split-Cas9 architecture for inducible genome editing and transcription modulation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3149%26rft_id%253Dinfo%253Apmid%252F25643054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25643054&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-125-1" title="View reference  in text" id="ref-125">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neises</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Different effects of sgRNA length on CRISPR-mediated gene knockout efficiency.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D28566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-126-1" title="View reference  in text" id="ref-126">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.126" data-doi="10.1093/nar/gnh110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumann</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reymond</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An efficient one-step site-directed and site-saturation mutagenesis protocol.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZheng%26rft.auinit1%253DL.%26rft.volume%253D32%26rft.issue%253D14%26rft.spage%253De115%26rft.epage%253De115%26rft.atitle%253DAn%2Befficient%2Bone-step%2Bsite-directed%2Band%2Bsite-saturation%2Bmutagenesis%2Bprotocol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgnh110%26rft_id%253Dinfo%253Apmid%252F15304544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gnh110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15304544&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-127-1" title="View reference  in text" id="ref-127">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.127" data-doi="10.1038/srep40638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Profiling single-guide RNA specificity reveals a mismatch sensitive core sequence.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">40638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D40638%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep40638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep40638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-128-1" title="View reference  in text" id="ref-128">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.999.128" data-doi="10.1038/nbt.3811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Precise base editing in rice, wheat and maize with a Cas9-cytidine deaminase fusion.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D35%26rft.spage%253D438%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3811%26rft_id%253Dinfo%253Apmid%252F28244994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28244994&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/3/999.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/3/981" title="Genomic Identification and Functional Characterization of Essential Genes in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/3/1019" title="Functional Analysis of Cancer-Associated DNA Polymerase ε Variants in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/3"><span>Volume 8 Issue 3, March 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/3.cover-source.jpg?itok=o8h2cMy2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/999" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Er5ZcLdA9QBJCj-7fK77dmi27w4XDDnYtQnvgf69vzI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/3/999" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/3/999" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/3/999" class="active" data-icon-position="" data-hide-link-title="0">Tuning CRISPR-Cas9 Gene Drives in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488753" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-e8jPtdLBQ0qUxcnwxHjweqW_1Ua2kQpyEbjl_-sXeE0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/3/999.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/3/999" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-GPsMW80WtaYwcHshCisBcWTj97dmI53xo8KeN6i8-0s" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/3/999" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/3/999.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488753" id="node488753--2" data-pisa="ggg;8/3/999" data-pisa-master="ggg;g3.117.300557" data-apath="/ggg/8/3/999.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tuning CRISPR-Cas9 Gene Drives in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Emily Roggenkamp</span>, <span class="highwire-citation-author" data-delta="1">Rachael M. Giersch</span>, <span class="highwire-citation-author" data-delta="2">Madison N. Schrock</span>, <span class="highwire-citation-author" data-delta="3">Emily Turnquist</span>, <span class="highwire-citation-author" data-delta="4">Megan Halloran</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-5710-173X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory C. Finnigan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">999-1018;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300557 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488753--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emily Roggenkamp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BRoggenkamp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roggenkamp%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BRoggenkamp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rachael M. Giersch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachael%2BM.%2BGiersch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giersch%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachael%2BM.%2BGiersch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Madison N. Schrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madison%2BN.%2BSchrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schrock%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadison%2BN.%2BSchrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emily Turnquist </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BTurnquist%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turnquist%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BTurnquist%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Megan Halloran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BHalloran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halloran%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMegan%2BHalloran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gregory C. Finnigan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BC.%2BFinnigan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Finnigan%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BC.%2BFinnigan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5710-173X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory C. Finnigan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gfinnigan@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gfinnigan@ksu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/3/999" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/3/999&amp;title=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/3/999&amp;title=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/3/999&amp;title=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/3/999&amp;text=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/3/999&amp;title=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/3/999&amp;t=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/3/999&amp;title=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/3/999&amp;title=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488753/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488753" id="node488753--3" data-pisa="ggg;8/3/999" data-pisa-master="ggg;g3.117.300557" data-apath="/ggg/8/3/999.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tuning CRISPR-Cas9 Gene Drives in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Emily Roggenkamp</span>, <span class="highwire-citation-author" data-delta="1">Rachael M. Giersch</span>, <span class="highwire-citation-author" data-delta="2">Madison N. Schrock</span>, <span class="highwire-citation-author" data-delta="3">Emily Turnquist</span>, <span class="highwire-citation-author" data-delta="4">Megan Halloran</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-5710-173X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory C. Finnigan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">999-1018;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300557 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488753--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emily Roggenkamp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BRoggenkamp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roggenkamp%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BRoggenkamp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rachael M. Giersch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rachael%2BM.%2BGiersch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giersch%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARachael%2BM.%2BGiersch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Madison N. Schrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madison%2BN.%2BSchrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schrock%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadison%2BN.%2BSchrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emily Turnquist </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BTurnquist%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turnquist%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BTurnquist%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Megan Halloran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BHalloran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halloran%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMegan%2BHalloran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gregory C. Finnigan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biophysics, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BC.%2BFinnigan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Finnigan%20GC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BC.%2BFinnigan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5710-173X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory C. Finnigan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gfinnigan@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gfinnigan@ksu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=999&amp;rft.epage=1018&amp;rft.atitle=Tuning%20CRISPR-Cas9%20Gene%20Drives%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=8&amp;rft.issue=3&amp;rft.date=2018-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Roggenkamp%2C+Emily&amp;rft.au=Giersch%2C+Rachael+M.&amp;rft.au=Schrock%2C+Madison+N.&amp;rft.au=Turnquist%2C+Emily&amp;rft.au=Halloran%2C+Megan&amp;rft.au=Finnigan%2C+Gregory+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488753/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488753/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488753/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488753/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488753/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488753/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488753/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488753/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488753/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488753/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488753/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488753/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/3/999" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/3/999.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/3/999.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/3/999.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/3/999.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
