<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/3/1079" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/3/1079.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/3/1079.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/3/1079.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/3/1079.atom" />
<meta name="HW.pisa" content="ggg;8/3/1079" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum" />
<meta name="DC.Identifier" content="10.1534/g3.117.300301" />
<meta name="DC.Date" content="2018-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Addo-Quaye et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The accurate detection of induced mutations is critical for both forward and reverse genetics studies. Experimental chemical mutagenesis induces relatively few single base changes per individual. In a complex eukaryotic genome, false positive detection of mutations can occur at or above this mutagenesis rate. We demonstrate here, using a population of ethyl methanesulfonate (EMS)-treated Sorghum bicolor BTx623 individuals, that using replication to detect false positive-induced variants in next-generation sequencing (NGS) data permits higher throughput variant detection with greater accuracy. We used a lower sequence coverage depth (average of 7×) from 586 independently mutagenized individuals and detected 5,399,493 homozygous single nucleotide polymorphisms (SNPs). Of these, 76% originated from only 57,872 genomic positions prone to false positive variant calling. These positions are characterized by high copy number paralogs where the error-prone SNP positions are at copies containing a variant at the SNP position. The ability of short stretches of homology to generate these error-prone positions suggests that incompletely assembled or poorly mapped repeated sequences are one driver of these error-prone positions. Removal of these false positives left 1,275,872 homozygous and 477,531 heterozygous EMS-induced SNPs, which, congruent with the mutagenic mechanism of EMS, were &gt;98% G:C to A:T transitions. Through this analysis, we generated a collection of sequence indexed mutants of sorghum. This collection contains 4035 high-impact homozygous mutations in 3637 genes and 56,514 homozygous missense mutations in 23,227 genes. Each line contains, on average, 2177 annotated homozygous SNPs per genome, including seven likely gene knockouts and 96 missense mutations. The number of mutations in a transcript was linearly correlated with the transcript length and also the G+C count, but not with the GC/AT ratio. Analysis of the detected mutagenized positions identified CG-rich patches, and flanking sequences strongly influenced EMS-induced mutation rates. This method for detecting false positive-induced mutations is generally applicable to any organism, is independent of the choice of in silico variant-calling algorithm, and is most valuable when the true mutation rate is likely to be low, such as in laboratory-induced mutations or somatic mutation detection in medicine." />
<meta name="DC.Contributor" content="Charles Addo-Quaye" />
<meta name="DC.Contributor" content="Mitch Tuinstra" />
<meta name="DC.Contributor" content="Nicola Carraro" />
<meta name="DC.Contributor" content="Clifford Weil" />
<meta name="DC.Contributor" content="Brian P. Dilkes" />
<meta name="article:published_time" content="2018-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The accurate detection of induced mutations is critical for both forward and reverse genetics studies. Experimental chemical mutagenesis induces relatively few single base changes per individual. In a complex eukaryotic genome, false positive detection of mutations can occur at or above this mutagenesis rate. We demonstrate here, using a population of ethyl methanesulfonate (EMS)-treated &lt;i&gt;Sorghum bicolor&lt;/i&gt; BTx623 individuals, that using replication to detect false positive-induced variants in next-generation sequencing (NGS) data permits higher throughput variant detection with greater accuracy. We used a lower sequence coverage depth (average of 7×) from 586 independently mutagenized individuals and detected 5,399,493 homozygous single nucleotide polymorphisms (SNPs). Of these, 76% originated from only 57,872 genomic positions prone to false positive variant calling. These positions are characterized by high copy number paralogs where the error-prone SNP positions are at copies containing a variant at the SNP position. The ability of short stretches of homology to generate these error-prone positions suggests that incompletely assembled or poorly mapped repeated sequences are one driver of these error-prone positions. Removal of these false positives left 1,275,872 homozygous and 477,531 heterozygous EMS-induced SNPs, which, congruent with the mutagenic mechanism of EMS, were &amp;gt;98% G:C to A:T transitions. Through this analysis, we generated a collection of sequence indexed mutants of sorghum. This collection contains 4035 high-impact homozygous mutations in 3637 genes and 56,514 homozygous missense mutations in 23,227 genes. Each line contains, on average, 2177 annotated homozygous SNPs per genome, including seven likely gene knockouts and 96 missense mutations. The number of mutations in a transcript was linearly correlated with the transcript length and also the G+C count, but not with the GC/AT ratio. Analysis of the detected mutagenized positions identified CG-rich patches, and flanking sequences strongly influenced EMS-induced mutation rates. This method for detecting false positive-induced mutations is generally applicable to any organism, is independent of the choice of &lt;i&gt;in silico&lt;/i&gt; variant-calling algorithm, and is most valuable when the true mutation rate is likely to be low, such as in laboratory-induced mutations or somatic mutation detection in medicine.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/03/01" />
<meta name="citation_mjid" content="ggg;8/3/1079" />
<meta name="citation_id" content="8/3/1079" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/3/1079" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/3/1079.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/3/1079.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/3/1079.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300301" />
<meta name="citation_pmid" content="29378822" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1079" />
<meta name="citation_lastpage" content="1094" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Charles Addo-Quaye" />
<meta name="citation_author_institution" content="Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0598-5980" />
<meta name="citation_author" content="Mitch Tuinstra" />
<meta name="citation_author_institution" content="Department of Agronomy, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author" content="Nicola Carraro" />
<meta name="citation_author_institution" content="Department of Agronomy, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author" content="Clifford Weil" />
<meta name="citation_author_institution" content="Department of Agronomy, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author" content="Brian P. Dilkes" />
<meta name="citation_author_institution" content="Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_email" content="bdilkes@purdue.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2799-954X" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. Addo-Quaye;citation_author=E. Buescher;citation_author=N. Best;citation_author=V. Chaikam;citation_author=I. Baxter;citation_title=Forward genetics by sequencing EMS variation-induced inbred lines.;citation_volume=7;citation_year=2017;citation_pmid=28040779;citation_doi=10.1534/g3.116.029660" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. M. Alonso;citation_author=J. R. Ecker;citation_title=Moving forward in reverse: genetic technologies to enable genome-wide phenomic screens in Arabidopsis.;citation_pages=524-536;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16755288;citation_doi=10.1038/nrg1893" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=K. Arumuganathan;citation_author=E. D. Earle;citation_title=Nuclear DNA content of some important plant species.;citation_pages=208-218;citation_volume=9;citation_year=1991;citation_doi=10.1007/BF02672069" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. L. Bailey;citation_author=M. Boden;citation_author=F. Buske;citation_author=M. Frith;citation_author=C. E. Grant;citation_title=MEME SUITE: tools for motif discovery and searching.;citation_volume=37;citation_year=2009;citation_pmid=19458158;citation_doi=10.1093/nar/gkp335" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. W. Bevan;citation_author=C. Uauy;citation_author=B. B. H. Wulff;citation_author=J. Zhou;citation_author=K. Krasileva;citation_title=Genomic innovation for crop improvement.;citation_pages=346-354;citation_volume=543;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=C. K. Blomstedt;citation_author=R. M. Gleadow;citation_author=N. O’Donnell;citation_author=P. Naur;citation_author=K. Jensen;citation_title=A combined biochemical screen and TILLING approach identifies mutations in Sorghum bicolor L. Moench resulting in acyanogenic forage production.;citation_pages=54-66;citation_volume=10;citation_year=2012;citation_issue=1;citation_pmid=21880107;citation_doi=10.1111/j.1467-7652.2011.00646.x" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Y. Cheng;citation_author=Y.-Y. Teo;citation_author=R. T.-H. Ong;citation_title=Assessing single nucleotide variant detection and genotype calling on whole-genome sequenced individuals.;citation_volume=30;citation_year=2014;citation_pmid=24558117;citation_doi=10.1093/bioinformatics/btu067" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Cheung;citation_author=X. Estivill;citation_author=R. Khaja;citation_author=J. R. MacDonald;citation_author=K. Lau;citation_title=Genome-wide detection of segmental duplications and potential assembly errors in the human genome sequence.;citation_pages=R25-R25;citation_volume=4;citation_year=2003;citation_issue=4;citation_pmid=12702206;citation_doi=10.1186/gb-2003-4-4-r25" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=P. Cingolani;citation_author=V. M. Patel;citation_author=M. Coon;citation_author=T. Nguyen;citation_author=S. J. Land;citation_title=Using Drosophila melanogaster as a Model for Genotoxic Chemical Mutational Studies with a New Program, SnpSift.;citation_pages=35-35;citation_volume=3;citation_year=2012;citation_pmid=22435069;citation_doi=10.3389/fgene.2012.00035" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C. Coulondre;citation_author=J. H. Miller;citation_title=Genetic studies of the lac repressor. IV. Mutagenic specificity in the lacI gene of Escherichia coli.;citation_pages=577-606;citation_volume=117;citation_year=1977;citation_issue=3;citation_pmid=416218;citation_doi=10.1016/0022-2836(77)90059-6" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=X. Estivill;citation_author=J. Cheung;citation_author=M. A. Pujana;citation_author=K. Nakabayashi;citation_author=S. W. Scherer;citation_title=Chromosomal regions containing high-density and ambiguously mapped putative single nucleotide polymorphisms (SNPs) correlate with segmental duplications in the human genome;citation_pages=1987-1995;citation_volume=11;citation_year=2002;citation_issue=17;citation_pmid=12165560;citation_doi=10.1093/hmg/11.17.1987" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Firtina;citation_author=C. Alkan;citation_title=On genomic repeats and reproducibility.;citation_volume=32;citation_year=2016;citation_pmid=27153582;citation_doi=10.1093/bioinformatics/btw139" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Flibotte;citation_author=M. L. Edgley;citation_author=I. Chaudhry;citation_author=J. Taylor;citation_author=S. E. Neil;citation_title=Whole-genome profiling of mutagenesis in Caenorhabditis elegans.;citation_volume=185;citation_year=2010;citation_pmid=20439774;citation_doi=10.1534/genetics.110.116616" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Fredman;citation_author=S. J. White;citation_author=S. Potter;citation_author=E. E. Eichler;citation_author=J. T. Den Dunnen;citation_title=Complex SNP-related sequence variation in segmental genome duplications.;citation_pages=861-866;citation_volume=36;citation_year=2004;citation_issue=8;citation_pmid=15247918;citation_doi=10.1038/ng1401" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. M. Goodstein;citation_author=S. Shu;citation_author=R. Howson;citation_author=R. Neupane;citation_author=R. D. Hayes;citation_title=Phytozome: a comparative platform for green plant genomics.;citation_volume=40;citation_year=2012;citation_pmid=22110026;citation_doi=10.1093/nar/gkr944" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. A. Greene;citation_author=C. A. Codomo;citation_author=N. E. Taylor;citation_author=J. G. Henikoff;citation_author=B. J. Till;citation_title=Spectrum of Chemically Induced Mutations From a Large-Scale Reverse-Genetic Screen in Arabidopsis;citation_pages=731-740;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807792" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=C. S. Grierson;citation_author=S. R. Barnes;citation_author=M. W. Chase;citation_author=M. Clarke;citation_author=D. Grierson;citation_title=One hundred important questions facing plant science research.;citation_pages=6-12;citation_volume=192;citation_year=2011;citation_issue=1;citation_pmid=21883238;citation_doi=10.1111/j.1469-8137.2011.03859.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=I. M. Henry;citation_author=U. Nagalakshmi;citation_author=M. C. Lieberman;citation_author=K. J. Ngo;citation_author=K. V. Krasileva;citation_title=Efficient genome-wide detection and cataloging of EMS-induced mutations using exome capture and next-generation sequencing.;citation_volume=26;citation_year=2014;citation_pmid=24728647;citation_doi=10.1105/tpc.113.121590" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=G. V. James;citation_author=V. Patel;citation_author=K. J. Nordström;citation_author=J. R. Klasen;citation_author=P. A. Salomé;citation_title=User guide for mapping-by-sequencing in Arabidopsis.;citation_pages=R61;citation_volume=14;citation_year=2013;citation_pmid=23773572;citation_doi=10.1186/gb-2013-14-6-r61" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=Y. Jiao;citation_author=J. Burke;citation_author=R. Chopra;citation_author=G. Burow;citation_author=J. Chen;citation_title=A Sorghum mutant resource as an efficient platform for gene discovery in grasses.;citation_volume=28;citation_year=2016;citation_pmid=27354556;citation_doi=10.1105/tpc.16.00373" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. R. Johnson;citation_author=J. M. Yeoh;citation_author=C. Coruh;citation_author=M. J. Axtell;citation_title=Improved placement of multi-mapping small RNAs.;citation_pages=2103-2111;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=S. Koegel;citation_author=N. Ait Lahmidi;citation_author=C. Arnould;citation_author=O. Chatagnier;citation_author=F. Walder;citation_title=The family of ammonium transporters (AMT) in Sorghum bicolor: two AMT members are induced locally, but not systemically in roots colonized by arbuscular mycorrhizal fungi.;citation_pages=853-865;citation_volume=198;citation_year=2013;citation_pmid=23461653;citation_doi=10.1111/nph.12199" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. E. Kohalmi;citation_author=B. Kunz;citation_title=Role of neighbouring bases and assessment of strand specificity in ethylmethanesulphonate and N-methyl-N'-nitro-N-nitrosoguanidine mutagenesis in the SUP4-o gene of Saccharomyces cerevisiae.;citation_pages=561-568;citation_volume=204;citation_year=1988;citation_issue=3;citation_pmid=3066906;citation_doi=10.1016/0022-2836(88)90355-5" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. V. Krasileva;citation_author=H. A. Vasquez-Gross;citation_author=T. Howell;citation_author=P. Bailey;citation_author=F. Paraiso;citation_title=Uncovering hidden variation in polyploid wheat.;citation_volume=114;citation_year=2017;citation_pmid=28096351;citation_doi=10.1073/pnas.1619268114" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. R. Krieg;citation_title=ETHYL METHANESULFONATE-INDUCED REVERSION OF BACTERIOPHAGE T4rII MUTANTS;citation_pages=561-580;citation_volume=48;citation_year=1963;citation_issue=4;citation_pmid=14035786" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Krothapalli;citation_author=E. M. Buescher;citation_author=X. Li;citation_author=E. Brown;citation_author=C. Chapple;citation_title=Forward genetics by genome sequencing reveals that rapid cyanide release deters insect herbivory of Sorghum bicolor.;citation_volume=195;citation_year=2013;citation_pmid=23893483;citation_doi=10.1534/genetics.113.149567" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=P. Kumar;citation_author=S. Henikoff;citation_author=P. C. Ng;citation_title=Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.;citation_pages=1073-1081;citation_volume=4;citation_year=2009;citation_issue=7;citation_pmid=19561590;citation_doi=10.1038/nprot.2009.86" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C.-L. F. Li;citation_author=B. Santhanam;citation_author=A. N. Webb;citation_author=B. Zupan;citation_author=G. Shaulsky;citation_title=Gene discovery by chemical mutagenesis and whole-genome sequencing in Dictyostelium.;citation_volume=26;citation_year=2016;citation_pmid=27307293;citation_doi=10.1101/gr.205682.116" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=G. Li;citation_author=M. Chern;citation_author=R. Jain;citation_author=J. A. Martin;citation_author=W. S. Schackwitz;citation_title=Genome-wide sequencing of 41 rice (Oryza sativa L.) mutated lines reveals diverse mutations induced by fast-neutron irradiation.;citation_pages=1078-1081;citation_volume=9;citation_year=2016;citation_pmid=27018389;citation_doi=10.1016/j.molp.2016.03.009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Methods;citation_author=K. Liu;citation_author=M. McCormack;citation_author=J. Sheen;citation_title=Targeted parallel sequencing of large genetically-defined genomic regions for identifying mutations in Arabidopsis.;citation_pages=12-12;citation_volume=8;citation_year=2012;citation_pmid=22462410;citation_doi=10.1186/1746-4811-8-12" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Loveless;citation_title=Increased rate of plaque-type and host-range mutation following treatment of bacteriophage in vitro with ethyl methane sulphonate.;citation_pages=1212-1213;citation_volume=181;citation_year=1958;citation_issue=4617;citation_pmid=13541432;citation_doi=10.1038/1811212a0" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Loveless;citation_title=Possible relevance of O-6 alkylation of deoxyguanosine to the mutagenicity and carcinogenicity of nitrosamines and nitrosamides.;citation_pages=206-207;citation_volume=223;citation_year=1969;citation_issue=5202;citation_pmid=5791738;citation_doi=10.1038/223206a0" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. L. Metzker;citation_title=Sequencing technologies - the next generation.;citation_pages=31-46;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=19997069;citation_doi=10.1038/nrg2626" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Nakamura;citation_author=T. Oshima;citation_author=T. Morimoto;citation_author=S. Ikeda;citation_author=H. Yoshikawa;citation_title=Sequence-specific error profile of Illumina sequencers.;citation_volume=39;citation_year=2011;citation_pmid=21576222;citation_doi=10.1093/nar/gkr344" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Nordborg;citation_author=D. Weigel;citation_title=Next-generation genetics in plants.;citation_pages=720-723;citation_volume=456;citation_year=2008;citation_issue=7223;citation_pmid=19079047;citation_doi=10.1038/nature07629" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=K. J. V. Nordström;citation_author=M. C. Albani;citation_author=G. V. James;citation_author=C. Gutjahr;citation_author=B. Hartwig;citation_title=Mutation identification by direct comparison of whole-genome sequencing data from mutant and wild-type individuals using k-mers.;citation_pages=325-330;citation_volume=31;citation_year=2013;citation_pmid=23475072;citation_doi=10.1038/nbt.2515" />
<meta name="citation_reference" content="citation_journal_title=Genome Med.;citation_author=J. O’Rawe;citation_author=T. Jiang;citation_author=G. Sun;citation_author=Y. Wu;citation_author=W. Wang;citation_title=Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing.;citation_pages=28;citation_volume=5;citation_year=2013;citation_pmid=23537139;citation_doi=10.1186/gm432" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. R. Page;citation_author=U. Grossniklaus;citation_title=The art and design of genetic screens: Arabidopsis thaliana.;citation_pages=124-136;citation_volume=3;citation_year=2002;citation_issue=2;citation_pmid=11836506;citation_doi=10.1038/nrg730" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Plant Genomics;citation_author=A. H. Paterson;citation_title=Genomics of sorghum.;citation_pages=362451-362451;citation_volume=2008;citation_year=2008;citation_pmid=18483564" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. F. Pedersen;citation_author=S. R. Bean;citation_author=R. Graybosch;citation_author=S. H. Park;citation_author=M. Tilley;citation_title=Characterization of waxy grain sorghum lines in relation to granule-bound starch synthase.;citation_pages=151-156;citation_volume=144;citation_year=2005;citation_doi=10.1007/s10681-005-5298-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. J. Peters;citation_author=M. Jenks;citation_author=P. J. Rich;citation_author=J. D. Axtell;citation_author=G. Ejeta;citation_title=Mutagenesis, Selection, and Allelic Analysis of Epicuticular Wax Mutants in Sorghum;citation_pages=1250-1258;citation_volume=49;citation_year=2009;citation_issue=4;citation_doi=10.2135/cropsci2008.08.0461" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. G. Peterson;citation_author=S. R. Schulze;citation_author=E. B. Sciara;citation_author=S. A. Lee;citation_author=J. E. Bowers;citation_title=Integration of Cot analysis, DNA cloning, and high-throughput sequencing facilitates genome characterization and gene discovery.;citation_volume=12;citation_year=2002;citation_pmid=11997346;citation_doi=10.1101/gr.226102" />
<meta name="citation_reference" content="citation_journal_title=Biotechnol. Biofuels;citation_author=C. Petti;citation_author=A. E. Harman-Ware;citation_author=M. Tateno;citation_author=R. Kushwaha;citation_author=A. Shearer;citation_title=Sorghum mutant RG displays antithetic leaf shoot lignin accumulation resulting in improved stem saccharification properties.;citation_pages=146;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=C. Petti;citation_author=K. Hirano;citation_author=J. Stork;citation_author=S. DeBolt;citation_title=Mapping of a cellulose-deficient mutant named dwarf1–1 in Sorghum bicolor to the green revolution gene gibberellin20-oxidase reveals a positive regulatory association between gibberellin and cellulose biosynthesis.;citation_volume=169;citation_year=2015;citation_pmid=26198258;citation_doi=10.1104/pp.15.00928" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=L. Prakash;citation_author=F. Sherman;citation_title=Mutagenic specificity: reversion of iso-1-cytochrome c mutants of yeast.;citation_pages=65-82;citation_volume=79;citation_year=1973;citation_issue=1;citation_pmid=4355598;citation_doi=10.1016/0022-2836(73)90270-2" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=H. J. Price;citation_author=S. L. Dillon;citation_author=G. Hodnett;citation_author=W. L. Rooney;citation_author=L. Ross;citation_title=Genome Evolution in the Genus Sorghum (Poaceae);citation_pages=219-227;citation_volume=95;citation_year=2005;citation_issue=1;citation_pmid=15596469;citation_doi=10.1093/aob/mci015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=G. Rizal;citation_author=V. Thakur;citation_author=J. Dionora;citation_author=S. Karki;citation_author=S. Wanchana;citation_title=Two forward genetic screens for vein density mutants in sorghum converge on a cytochrome P450 gene in the brassinosteroid pathway.;citation_pages=257-266;citation_volume=84;citation_year=2015;citation_pmid=26333774;citation_doi=10.1111/tpj.13007" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. Robasky;citation_author=N. E. Lewis;citation_author=G. M. Church;citation_title=The role of replicates for error mitigation in next-generation sequencing.;citation_pages=56-62;citation_volume=15;citation_year=2014;citation_pmid=24322726;citation_doi=10.1038/nrg3655" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=B. A. Rowan;citation_author=D. Weigel;citation_author=D. Koenig;citation_title=Developmental genetics and new sequencing technologies: the rise of nonmodel organisms.;citation_pages=65-76;citation_volume=21;citation_year=2011;citation_issue=1;citation_pmid=21763609;citation_doi=10.1016/j.devcel.2011.05.021" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Sarin;citation_author=V. Bertrand;citation_author=H. Bigelow;citation_author=A. Boyanov;citation_author=M. Doitsidou;citation_title=Analysis of multiple ethyl methanesulfonate-mutagenized Caenorhabditis elegans strains by whole-genome sequencing.;citation_volume=185;citation_year=2010;citation_pmid=20439776;citation_doi=10.1534/genetics.110.116319" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. E. Sattler;citation_author=A. Saballos;citation_author=Z. Xin;citation_author=D. L. Funnell-Harris;citation_author=W. Vermerris;citation_title=Characterization of novel Sorghum brown midrib mutants from an EMS-mutagenized population.;citation_pages=2115-2124;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Publ. Gr.;citation_author=C. Schlötterer;citation_author=R. Tobler;citation_author=R. Kofler;citation_author=V. Nolte;citation_title=Sequencing pools of individuals—mining genome-wide polymorphism data without big funding.;citation_pages=749-763;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. Schneeberger;citation_title=Using next-generation sequencing to isolate mutant genes from forward genetic screens.;citation_pages=662-676;citation_volume=15;citation_year=2014;citation_pmid=25139187;citation_doi=10.1038/nrg3745" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=K. Schneeberger;citation_author=D. Weigel;citation_title=Fast-forward genetics enabled by new sequencing technologies.;citation_pages=282-288;citation_volume=16;citation_year=2011;citation_issue=5;citation_pmid=21439889;citation_doi=10.1016/j.tplants.2011.02.006" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_author=E. D. Scully;citation_author=T. Gries;citation_author=D. L. Funnell-Harris;citation_author=Z. Xin;citation_author=F. A. Kovacs;citation_title=Characterization of novel Brown midrib 6 mutations affecting lignin biosynthesis in sorghum.;citation_pages=136-149;citation_volume=58;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=G. A. Sega;citation_title=A review of the genetic effects of ethyl methanesulfonate.;citation_pages=113-142;citation_volume=134;citation_year=1984;citation_issue=2-3;citation_pmid=6390190;citation_doi=10.1016/0165-1218(84)90033-8" />
<meta name="citation_reference" content="citation_journal_title=Carcinogenesis;citation_author=N. Shrivastav;citation_author=D. Li;citation_author=J. M. Essigmann;citation_title=Chemical biology of mutagenesis and DNA repair: cellular responses to DNA alkylation.;citation_volume=31;citation_year=2010;citation_pmid=19875697;citation_doi=10.1093/carcin/bgp262" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. Singer;citation_author=H. Fraenkel-Conrat;citation_author=J. T. Kuśmierek;citation_title=Preparation and template activities of polynucleotides containing O2- and O4-alkyluridine.;citation_pages=1722-1726;citation_volume=75;citation_year=1978;citation_issue=4;citation_pmid=273902;citation_doi=10.1073/pnas.75.4.1722" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=B. Singer;citation_author=J. T. Kuśmierek;citation_author=J. T. Ku;citation_title=Chemical mutagenesis.;citation_pages=655-693;citation_volume=51;citation_year=1982;citation_pmid=7051963;citation_doi=10.1146/annurev.bi.51.070182.003255" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. E. Suzek;citation_author=H. Huang;citation_author=P. McGarvey;citation_author=R. Mazumder;citation_author=C. H. Wu;citation_title=UniRef: comprehensive and non-redundant UniProt reference clusters.;citation_volume=23;citation_year=2007;citation_pmid=17379688;citation_doi=10.1093/bioinformatics/btm098" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=Z. Swigoňová;citation_author=J. Lai;citation_author=J. Ma;citation_author=W. Ramakrishna;citation_author=V. Llaca;citation_title=Close Split of Sorghum and Maize Genome Progenitors;citation_pages=1916-1923;citation_volume=14;citation_year=2004;citation_issue=10a;citation_pmid=15466289;citation_doi=10.1101/gr.2332504" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=N. Uchida;citation_author=T. Sakamoto;citation_author=T. Kurata;citation_author=M. Tasaka;citation_title=Identification of EMS-induced causal mutations in a non-reference Arabidopsis thaliana accession by whole genome sequencing.;citation_volume=52;citation_year=2011;citation_pmid=21398646;citation_doi=10.1093/pcp/pcr029" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=Activities at the Universal Protein Resource (UniProt).;citation_volume=42;citation_year=2014;citation_pmid=24253303;citation_doi=10.1093/nar/gkt1140" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=R. Waugh;citation_author=D. J. Leader;citation_author=N. McCallum;citation_author=D. Caldwell;citation_title=Harvesting the potential of induced biological diversity.;citation_pages=71-79;citation_volume=11;citation_year=2006;citation_issue=2;citation_pmid=16406304;citation_doi=10.1016/j.tplants.2005.12.007" />
<meta name="citation_reference" content="citation_journal_title=Experientia;citation_journal_abbrev=Experientia;citation_author=M. Westergaard;citation_title=Chemical mutagenesis in relation to the concept of the gene.;citation_pages=224-234;citation_volume=13;citation_year=1957;citation_issue=6;citation_pmid=13447934;citation_doi=10.1007/BF02157427" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=Y. Wu;citation_author=L. Yuan;citation_author=X. Guo;citation_author=D. R. Holding;citation_author=J. Messing;citation_title=Mutation in the seed storage protein kafirin creates a high-value food trait in sorghum.;citation_pages=2217;citation_volume=4;citation_year=2013;citation_pmid=23948869" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Methods;citation_author=Z. Xin;citation_author=J. Chen;citation_title=A high throughput DNA extraction method with high yield and quality.;citation_pages=26-26;citation_volume=8;citation_year=2012;citation_issue=1;citation_pmid=22839646;citation_doi=10.1186/1746-4811-8-26" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Res.;citation_author=Z. Xin;citation_author=M. L. Wang;citation_author=G. Burow;citation_author=J. Burke;citation_title=An induced Sorghum mutant population suitable for bioenergy research.;citation_pages=10-16;citation_volume=2;citation_year=2009;citation_doi=10.1007/s12155-008-9029-3" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
<meta name="twitter:description" content="The accurate detection of induced mutations is critical for both forward and reverse genetics studies. Experimental chemical mutagenesis induces relatively few single base changes per individual. In a complex eukaryotic genome, false positive detection of mutations can occur at or above this mutagenesis rate. We demonstrate here, using a population of ethyl methanesulfonate (EMS)-treated Sorghum bicolor BTx623 individuals, that using replication to detect false positive-induced variants in next-generation sequencing (NGS) data permits higher throughput variant detection with greater accuracy. We used a lower sequence coverage depth (average of 7×) from 586 independently mutagenized individuals and detected 5,399,493 homozygous single nucleotide polymorphisms (SNPs). Of these, 76% originated from only 57,872 genomic positions prone to false positive variant calling. These positions are characterized by high copy number paralogs where the error-prone SNP positions are at copies containing a variant at the SNP position. The ability of short stretches of homology to generate these error-prone positions suggests that incompletely assembled or poorly mapped repeated sequences are one driver of these error-prone positions. Removal of these false positives left 1,275,872 homozygous and 477,531 heterozygous EMS-induced SNPs, which, congruent with the mutagenic mechanism of EMS, were &gt;98% G:C to A:T transitions. Through this analysis, we generated a collection of sequence indexed mutants of sorghum. This collection contains 4035 high-impact homozygous mutations in 3637 genes and 56,514 homozygous missense mutations in 23,227 genes. Each line contains, on average, 2177 annotated homozygous SNPs per genome, including seven likely gene knockouts and 96 missense mutations. The number of mutations in a transcript was linearly correlated with the transcript length and also the G+C count, but not with the GC/AT ratio. Analysis of the detected mutagenized positions identified CG-rich patches, and flanking sequences strongly influenced EMS-induced mutation rates. This method for detecting false positive-induced mutations is generally applicable to any organism, is independent of the choice of in silico variant-calling algorithm, and is most valuable when the true mutation rate is likely to be low, such as in laboratory-induced mutations or somatic mutation detection in medicine." />
<meta name="og-title" property="og:title" content="Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/3/1079" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The accurate detection of induced mutations is critical for both forward and reverse genetics studies. Experimental chemical mutagenesis induces relatively few single base changes per individual. In a complex eukaryotic genome, false positive detection of mutations can occur at or above this mutagenesis rate. We demonstrate here, using a population of ethyl methanesulfonate (EMS)-treated Sorghum bicolor BTx623 individuals, that using replication to detect false positive-induced variants in next-generation sequencing (NGS) data permits higher throughput variant detection with greater accuracy. We used a lower sequence coverage depth (average of 7×) from 586 independently mutagenized individuals and detected 5,399,493 homozygous single nucleotide polymorphisms (SNPs). Of these, 76% originated from only 57,872 genomic positions prone to false positive variant calling. These positions are characterized by high copy number paralogs where the error-prone SNP positions are at copies containing a variant at the SNP position. The ability of short stretches of homology to generate these error-prone positions suggests that incompletely assembled or poorly mapped repeated sequences are one driver of these error-prone positions. Removal of these false positives left 1,275,872 homozygous and 477,531 heterozygous EMS-induced SNPs, which, congruent with the mutagenic mechanism of EMS, were &gt;98% G:C to A:T transitions. Through this analysis, we generated a collection of sequence indexed mutants of sorghum. This collection contains 4035 high-impact homozygous mutations in 3637 genes and 56,514 homozygous missense mutations in 23,227 genes. Each line contains, on average, 2177 annotated homozygous SNPs per genome, including seven likely gene knockouts and 96 missense mutations. The number of mutations in a transcript was linearly correlated with the transcript length and also the G+C count, but not with the GC/AT ratio. Analysis of the detected mutagenized positions identified CG-rich patches, and flanking sequences strongly influenced EMS-induced mutation rates. This method for detecting false positive-induced mutations is generally applicable to any organism, is independent of the choice of in silico variant-calling algorithm, and is most valuable when the true mutation rate is likely to be low, such as in laboratory-induced mutations or somatic mutation detection in medicine." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
    <title>Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yoEPOJ6ehVbSz2rq4QdkvmVWnTvEunoueox4ML1PpBY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488752","apath":"\/ggg\/8\/3\/1079.atom","pisa":"ggg;8\/3\/1079","ac":{"\/ggg\/8\/3\/1079.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/3\/1079.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/3\/1079"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/3\/1079"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/3\/1079":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ijjSTRtxr2yQhJnfH+nKX0R5ZujAb2+ksqH30SiTYg8x+n+6wGPxrWYNw8jW6Y5TU3IKTvAGQj5nnz0UoyA02m2VT2SjuWrGauDDTQeQaWRmBAzgTkaYanqUKTxCJbCWapv9FZ7+9WeSw5ydLc7X\\\\\\\/LsX64SbTpoCn64vVnbl1sCvmTTE4BEJNvR62sHgjnvo6PpvH7C5J3D0EZCbDjUt4PUdMDUdBevfohYGAzo+RBdyxmzzIUhUVaKXfAu5e0H43eTSENp4LdncVLFhW\\\\\\\/eqG5EZfFRckFEQhcXBMattYojY8YKtNQTppZHToj40E6tVbRafyjpAya\\\\\\\/P0fYAWA2D5fPmOab2kKThhWNIghmC7hv+m0DOfdQ6eFJMbyUtfFQ6zYwJZtEOtBjuau\\\\\\\/bM3ENDVbH38rXqHwKhmSX\\\\\\\/oSdBsmSVzMKW3XeUZXLTCG\\\\\\\/GCVfVA84\\\\\\\/k6MRV+nu+Cct5d5BA1vPvSimWcdVJ5HCfo6JnWySjRpxzmz5dibB4feAYpZ94PtgxSG2ROyb0gH6FMY612A1GAgYKgQ15zIGmBpTdc40f0x1o9GgkgEleFFBqmC9WQ0SV8v5bQoiO6vdVBzIEuRVh9YXXlTOXy6Ef1HqnE7F6JvxeY6ElS4WNajfNbV00yZBhC8FUB42PXcfi6WGSCUffsGZgm3dqUew0HmQJThua3URaKSMhF7tRI\\\\\\\/DotWgFi96LMIi3Jg5gwPFaWFDuo34LbEItpwzvVXRLJPRj2V6zTUMbzeilsJ\\\\\\\/9foTmoaYO9ir\\\\\\\/D3ugkMycPNV4Gn3EH6Vyau26ctSQVZLsBIT9btOp8r\\\\\\\/goaX7+v6JrNbgN261W+01Tax1Lh\\\\\\\/Rop0nlFqcnt6iGk1gU9AI2AH57tQVltr36kXB31qkJbPSwNxhmApoPnbQ6Z7DxaY7LlOMboQFb3\\\\\\\/l3ghFV0ktjsWgIkaVYZUKdgUsirT6v7sDN8OQ9ZclO9VDtKOimUoOqkFIzntrBMNGcsoTKmv83FA8sAq\\\\\\\/PvcQFlyxWEpsz90f4xCBT9Xq3Z8NFk+mIF5Jatn5VjwYS1Na9ODErIGJBKkeW1KIWqS+\\\\\\\/kJ\\\\\\\/PZVZij7rkFhIu4f\\\\\\\/1mtQo5tAL+Bm1TdqApNiCUYX4c5FX+CeR07\\\\\\\/+vzB02lMoKE3CUuHrQwc0mawXy1V5nC7l\\\\\\\/GVW9Chwy9tZNTj4hfucQjNVL1PjMuwWpGmK\\\\\\\/zpux2sLd49QhHrDuyVLji7rDjm\\\\\\\/tZETg4CgsIm77cN7zOEWUeW\\\\\\\/fiJ7eE+sT6vqU6FwAAQBXwrkKpJ5XEPgy\\\u0022,\\\u0022iv\\\u0022:\\\u00220HxjpO0pfEBKeAM+B0\\\\\\\/8jA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022732728af650e2dc0238d176942aca21630565f75cc0d7e269ac7897fbd9ff225\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228Nl70O\\\\\\\/LJXuW8u4qgs83EOhngONzHQre\\\\\\\/j5FBUZLm9h1vDAw0WrvRiHazushC1xBr6KpDtSQMweEOHOWnpKsFfa9XGle5ZV\\\\\\\/94NUEqd8G6nfVFs79yxSCj9lNvTjRtbZk6M\\\\\\\/KOHznHTSmYg8Ioi1CZ+\\\\\\\/ghJN\\\\\\\/QrnH97s2ofUictGPYLKEI3RG+M+LWrWH+sUQU74gwGPjUITCeDC+ycGh1pFP1dfZ8A8otM\\\\\\\/lLgQOYbMoc0S1MhHVIHthxKncAY2i+i3BinO38iHoOvQLFN5KRN3pkPhFlug04mP9KoYaQDopGlLMmY\\\\\\\/hegPywU4hwPCxITPAtv9QmqrpNrFF5tY2hiiYDkczSHe8LHGetUZRxIO1teTunn1t4Jd\\\\\\\/UN+TpSODJGE2Tmfe0n9j8V\\\\\\\/tNEIV1NJqftEIabCovWx7c567eI7lqbtmbkcoCCjRXb8l+oyTdOZN7CZd+EnmMGqSlo+8\\\\\\\/RZ1Q5ORpjsuQnA8oRsKJltl3MTpLRtT0CXUYjwwihcyeogDcatinTpaVzNPKjZgudKvgnbfm1YWRQxDggoN8jKp9kdR8HweQsDHUclkIFvQuWrAUC1TeM+pyrSCcNobWY3BBl1SN6dF5m6xvpE1X8S+\\\\\\\/\\\\\\\/jB0lyVvLbNg7qfIWtDBHjRR7QqQTya4OC6EZtwLG\\\\\\\/0nybIIM5i7Z0cTQU+uh5Ay2iugaghBRFM8LTiRHUSvpxmU5\\\\\\\/sm6xyOxmyJUTNJz58LS4HYu9MwB6uM6Gic02Ek8SoFJlJX3MZ79mZWvK0smkNYa7BHahRNDO4Hp\\\\\\\/uw04GROTvGwmlR6WlkQiXpgw3VDvDTwEcTcNy5Yf4s3GXlCbQxO2lIiH5s7PV33PCs+3nZ8oxrtvYtxela4AqrK9NlbjOoHuXqx2ojkHoBGrGo7L1+c6Io4s2jMkNiNj+SQuV06+QcSEW1c64azABuZ5MeBUL3\\\\\\\/SZ7iBFpzlZfaZZYMaKOT07Me2UBiTYw8mQh8ZE0fJ8YcQIidiczhcIfpRBrXT8f\\\\\\\/H\\\\\\\/\\\\\\\/w6aaLpQCoSapk0JkkJMidauu1hOnDMBkE7dj825l7w5J4IAldQ5FY6AVHPfYtwKgaSoYhtWj48ME0nH6hQTcZ5R7vX1hqmnbVYkvt\\\\\\\/do1Ej\\\\\\\/\\\\\\\/au\\\\\\\/QhRzY3a16h5uqF8biwdSLpuV9xIcP8c0K\\\\\\\/R66d0fvPpIp2e6oM22TDPQSiNKMPmlipCoiq1R8+nFseHoj\\\\\\\/cii5eowKaSLFO3P1QNGVNsjL6i5j2Qs9Q+0bkzomlaaa+CRdynCBZBcmNj0ynkvW+f6CR8BXYYY0N76XbtX5m5mXxW6iiuPoN79PJG3Pr4PHHOnFWG0FnBdhT12RIuUUMpg0Ct9leZEnk\\\\\\\/fLJF9vaaIN5zV9v\\\\\\\/25Z0XiNPJADNWU+LNeffIEFjWU0\\\\\\\/GBmIhhDqc1ujaBKNEON8yz\\\\\\\/qVGSkNZnn8xIOA44GyxitD9CSsgLLMEidyp3\\\\\\\/boqHrxoud0wJbu1+Yw0zzsdaC0UlWdAlf8wqe6GLwwjnCjapXGpuAVKqa6NFJULF0In3IpYyZYXih7QNYmGnXv5n\\\\\\\/fSxjxJ9hApg8enlH4bs4enL47dIWmXxcPI4gJ9Z\\\\\\\/UI+QsyRyToenXpIx4JdxU1O2ifQCQs+etrH6U+S2rRpK5pcSrgzxVv2P9HlmaPTBsuKOWfx5vBbBBbyAQspvSCMruRSL9CFMV\\\\\\\/cYbzHwA4j3UjtyxZH9E65KoTLQtygy+l\\\\\\\/3BkBXK5Ss\\\\\\\/ljV0ca+7Z1lYDT4W1um\\\\\\\/mzzd9T35pHnor948girp+K\\\\\\\/7VDQWpGrlYck3YmcXofKCLkzjFB4f0GRTbyTru21EWHQ+i1KRbb1QiEdNYz7j\\\\\\\/icFg8acZEAKBHJUL2yaN8E9Q7ZGYYekfvTCjIXwWA9Tv4RAIQTB9Q9O6PcQEja\\\\\\\/8i4N085jXRrXX2V09PmxWK9HtnLfpjbGNOl92upQU7HN3LZJ1bZqy8z9+kICiZ+pIC0kypv\\\\\\\/Rr2JnsVpsawBPOYW5cb6NACQyWX3tsUfh84y6qV44DmQdsdF1VE9jBKodgD\\\\\\\/LY4eLkHmN5nYlLMBjob2Gj\\\\\\\/OJG40pBpzVrTZY9pRGKqxUBluBExQzJ84wzfxHXjplJk0oOomZKkdRu\\\\\\\/QWzGpvNfQ5zmCMvhA+bdxMHU2h67s2JyvzB5+qi0fJme9hQpEi0Y0v5VlwJPGZz6iEWOkv4EBjMkgQMGut3Ip1yI2q0YqAwFO9IBkR0Ie729NWUN+NQAw3fzBMAK0f1JkNjtkOhgWnlQQt4PMnXvjhckMOdvllvk8MwfxHkfhix1ulnWoFFyfc6uYAMiGFUuYlqmtH52IINwcB8tUCEViUk46cQ96okK0xzl4KUxRW7qK7u+D7lcNxbQbJo5Cbs8mhhDrnHwQFhhz56QBIDkJ8g9zogWE5A7Fr5\\\\\\\/6xMF0JcKqqeMeiuWoilKBLmeF4bg6BSrU5T7pIzlR7T3E06R42xpT3HeYX3xSO8pnw09mpouuSuZ1vMV8UxqNHp+oi\\\\\\\/u80suv+CTDoqsk\\\\\\\/uHpzHyfzdYeychHOoBRMuX1L9SMtkAoQNNsBiiAQW9Kz6\\\\\\\/pKrYZRT1200Z\\\\\\\/zxDhRw5pzhnNBqJTb9H9Wsq\\\\\\\/HUMUCDHpeGD+0dCxyYHsF0BdwxNx9zBDJ3x80P3teidMz6ifiLgv0YrlgXBSDWNHqMdzki+MURBAhdenCtNgwOToNUI4UbUeK2B4vbOF90wnHotons4PmiQyIX2gmLPiMkNxtQb3lFFxqeZElQWwJHYOggQTgaJt0Rj54ro38P4Thb\\\\\\\/lccGPCWbzOGJ+heOJdnhol7e95mTrMFuy9NhCmHkUWCoh6NQKG59ds+mxjYUxaX3d8IpMLWlTzrrRlMxFgzYfDC2Sl5l7fBvP037DXvChUZ4HachOrpVlEctugNe5QDbuhkoJbyXySSzI+qP\\\\\\\/i68VwQqx5i7Q0mmcXICgF23N0BjFIp0d2L\\\\\\\/ZWAmq1h2rXcnVFFG1dKwjqilvrjSoDXxvo9fnZZrJ+QNg3LdNWsTw4+CohKSaZb3OIITAh7O26OcyrdJfaYjipq8sN7+539VDivT4MSsrPY5YuM\\\\\\\/Eu4N9siNqplxBSTp51aMg8\\\\\\\/llJwvLMM2HL\\\\\\\/rZZFTpI1A3BbWrPOp5UrInMiRj6CgoszxrfsUAwgAhL87HLH2jh51clFQLEW9oxIpkKSKZ31aPQjIHyvqQ4LIaBOXaaWe9KhIw1YVU2JMnlF5uHSwEhS9g0cy\\\\\\\/e8sgfic\\\\\\\/lM+60g2SqqjEFUEvuQKzvn\\\\\\\/NBvZasVtS6os4fy6+nw4B5rKRZNRDn0aHTjSACzF52P9U3H+Z5IloHwF\\\\\\\/MHq3+QHOGkkpekpHvI1b4Ca2APiqCD+csafpHrDCEiZVWX2JDNffUNd3YUDbTeVc810OmpE\\\\\\\/UDGdXrdyBa5I4y8PBJCmQPqIUszx6XZ+U4GBd9yqka6S9aukFacFoNAn0VOoc9ivYD3Y1QALXV2a2xzCVvintVr7R9sbjPCoAG1RO5JAlYkEdXkecRH2MnQKaPZGjvBvy46rKaB3V3653CCt5xHT1aP5vILwdiUhgzutASL1hffo1VDjmLo1sAQ4hUITOpeMljoqS7rvQ\\\\\\\/wl9drAEP+M28PulyIGjB2kFLzYuj7+13gJT\\\\\\\/GchSFelFdjTBBC2tiCAua6cCP+1MDO0ifzKAVNojPDWcMBnFNVsCG6YjCDQp4IqKq4+ChZs1FgzSyKXYOGE1l5gvGY4IKAp5eYkC5b+wI2NIPHZNtwafvZ77gcTyNWFx1xfDOBRw1gVdi9NkG3ZTPU6\\\\\\\/W07gm6JmCIkOVT27ggBwPINMpm4RhKueU1fV0KAWhmX1NSJ+JpqWFhSzUw\\\\\\\/cRUiqcBVVsN022okNO2vahRXKux\\\\\\\/6KfVJtkPiLxYnliSDFl1tBFNda\\\\\\\/p+YsYEuGTIK5mvhKow9wVibjEx006KEEu9IaFgAPJqxgGMM4DFz+4qP\\\\\\\/WZf5Ac7PgZULDq0ldM\\\\\\\/UtzjVtxfqGQ2dhyACGa7f4cIXZgG9j8rHYfOAinezKaeytZaaGSKZlMK4R\\\\\\\/6S59E1+sHgnMLUi4xbX15TUZM6feDT1n2xjXXSSCFTqdplI4G1vLoKszxL3Ghf+XCWrvfiSVMrYgXNFC6nmAQMPNqT97W9UjLHFMJA2v4C\\\\\\\/Yt+tDkSqzrt3FFfw0bomEKo+3Tdgq4J2bLrTeNdrjAtVDAQY5HImCVrgHTiX8QkcXScJrvCd9Ffbf06I5fABbF\\\\\\\/irXeVrEb6sMiwdQiAesj7ehL6TBtszFpnFhnktFlInemOOsMYv45SDUKTVz+nZc+U0WzCl9fJ0nzC1CEMtdxjIGZxYCnOplbNIeDPJcHd8mXSIniLxw8jF0jnemJKeuiKT+FGaC+Lr4GrTKhz7PBTwrbLQQrFsL0FjZ7yjEdsNqrBNeRlJxZ20fa2X2tzRd776KIWJ49FpASbGTfqPKwGqQ8t9h8XPvOryNixpISKwvhaB5X6NtmZjjQBiBHlT95ePZcNYJcsuTVlvM4xnOBh6ysB9ccdzwEPEI1jPU1doppXIgIU3UEwn7bDbj2Ml03pnJ2Our3GmNUwXjDcM947ZnyeYL0Kn46cQEv14XtXzbTSWDscTlO4XCWBCFUDwWBj4RZlkyA+2VaimPLLL7cWnD0W0PzBi93aLGVk+znLcgoYA5IlZOtFAX2sLqTOMR9yTr8\\\\\\\/6Qu6RXpXFHXHbS1gZrgunMpjio8u8o17CQkG+RMcc85bsApcUoWTc9Ax6zf4SG51i3HdbmPK0T20vgL1r1eEIx\\\\\\\/apYX9hHKKPidAi9mh\\\\\\\/YpllLuKN5gU6sI+aMKRyw3dps72NUvwVgCXs\\\\\\\/kI8fsZcAtZ6VjbI3Hg+t14M41EYyGCFxbeQXamMhC480TS2rPAdDm08SjJGXg+DjRt\\\\\\\/Tl0w5QdPUy9DojzaGmDkyTTGCndP6cVukpoKr4bqeZNN9RPLIDo64KLwTE6O6LJvuAiTKrjfR0Tt4sr6BE1Do0CAMnBU3p2IYKsJu5h3iXH9Wy2ch9XcPpOy2PLp\\\\\\\/mrah2BOacAoDuCGmvHxJeWi92gEUdvKwXKNY63P0r7KjvFhqyRacPqxtKUkv9HLvUC+IFSKZkL74gQ1Jw7micXOHNOSLUo5ICIdf090\\\\\\\/aK\\\\\\\/CKMaiMps9bji3JffGltYbJOOGLqwSCa+5gTF5ZCNNnS2tQgQ3Dg34DaOE3evIE+R0Zq5n+osZlQrVDtwcYTAW0SNygmps8Z5foniVHVXHRyZT+8c2y1hATPH0zoo9JtXH3mQMMwDCzL8chNlc7GZZztsb2ZbQj8TU8Sbv59ZkdvXAKjILDqYub+vONcwIQeCgMNrc679PW9Bm0MimEydAkLxRWaUzulb9EO1arDFyfyzVt7eb0s\\\\\\\/Yk8+CPpqscMfSQH9G\\\\\\\/EmgdVoENArmELLH\\\\\\\/wWyYY7mJ48szWLlJVSa1UacN80a8oFDCFKvducYjlYez9PbLp3l2V4uclFT\\\\\\\/rsG3Zf2K96ZoESGXjC6V1Xb\\\\\\\/aiRQfgOqhUaTt1j35dKsBr5aPpMojWsnGj6uE4d0g50xK8Bc7Zj+CgOhzAtcqGMM4Sm03lCM3fqRoJ7m8jq6CNDBjlq2fAUm7XRVyLel3GYdKqAyufYHDaSvirwpXqHIVXV5v8toydlyE+fOt74xbby56mj6m6nsTL1D0\\\\\\\/h1FjhgIuCUC9b891wOixLFQ5lXoE9mbeXbd3UyV9ux4hmiWIsPF1dRcJtQsPgg4fNwa2MYyJEXbdt\\\\\\\/BA0N8UnzO5tHxKlCK9MwNpshyD\\\\\\\/EFNpJLbenFUF84uUV0Lsl458c072ocseC9lry85GKLEYMYuCo6QSUEGl+c4yAfC7Woq+CQF9XVJpVl4+jqN+w5mQYpIpRLbrL+UFzLi5+3jHEl\\\\\\\/caOeh\\\\\\\/v3gbcXxeLlUTtUreh4cyGxJwgaL9OC\\\\\\\/VQOArVIi6YtHa8YwdU9ZbysLqyf7pqG6M980kEKOybhot5XJoZrUhl5l75V2clJEnIHDPAg+0HRcvIDBvlMYH9UEAGpOeRl0zIukUBYgvhbLhbOt9sG04Lksk95jZO4OcuAQK7AVZb1vSxrHgbzZx2X+NncZ\\\\\\\/712ajRKVbBICyo5yGmoEEcdFwzvYUFWGvSxVDvuaiI+gygHm8ICJ5jXY7r\\\\\\\/TGH1uAk0Xv\\\\\\\/RLhQyYIfce39LmAbtXnJyjt2zyH5Hij6Nd0k8htY2E9EJHchWULpGA9M6WJKWa4ST4bk32LFvdcf85uustezMVQP7+X+E\\\\\\\/9MhFJRhTlQGUzEh2ih3Tg9JkEgNNiu1XYdNPCaeoWmw0yZikcldkRFi23uEGS9UFOSn43+98FqhCT2ACq1C0LsJ56W8iJTI4Nt8KL13sVmnQNMoKpXD7pSBMjGvQUjjHRJElkOolVzbpJQNY3XVn5SpqdsBp3VSV3WKhXVgyLeP1GFqyP4IsxUPHRU\\\\\\\/Ge9te4Vq9FQbo+5AUYRsqWdgs\\\\\\\/5pvrpf5DI\\\\\\\/EOZ457plPciOJaY1cgfpE6pSu8Uq2ZQwFRe\\\\\\\/LlSYNcGOJInOJzk96xc1PwuGdQVYOKq0Nz8f4uuIojhUzMAbmva25FkZSOW5hQN0MaqbN4wWCaMxe7UT4TXpHDRsOpSZtEgRTD7A162\\\\\\\/xCxlU+DJpDJZqo9YA4aDtgexQ80dFJcB7vvMk4gg7wsPt6s3Iv0XswN6qbPfoQ60CLNfCXW7h\\\\\\\/Tv00U\\\\\\\/MQG662iwcwCUFKBvNd7oFA3Ntqut5mVQ3RiQuCFmIu4zcdAcNHa7AbRISVo52A505p2RkrwtxD4huqL9gRN6506dP7Y7xv4CDRrtsYCYkm6mcHCx44RiaYqpA7BTx8TURIWOZEd7ZUa\\\\\\\/6EGhsi89o08bOjOHalWa5PW1mSdA4tqlKNz68Xqqt1kBJMDE+VOmHJMTmOgN2ablBX5IOidLmaJ\\\\\\\/OHhSQnlQbY+bC\\\\\\\/zqbn2Fl8fH\\\\\\\/UXMW9OFy\\\\\\\/ljN\\\\\\\/bkxZ\\\\\\\/nf1nqJihdDZUk0FicDyoahRRGZSW30fuTEGTyU4tPJpg7VLO66iXs437OCPKZOksZpuZTvhYLVR+THBMCZcuEb3BNadXG5GjYoW5ExhIQ6TfkTegbLm4qa2k3KrflCQ102g\\\\\\\/nspPpj3jo8LJO3lq5oOYNIHcBCka8MTAQ2zzeWEbsahj+HWrUrFauoc7BUwi0mrgm1udmU6NBu5Q9eEvw03WAD3\\\\\\\/C3ko3zc2tkPynYvEIWLfCe0c9kb5FOrvsV94dURUHXuWOjZ4Xbm2Cn0SgaIOo6owYn4bYNcalphq7L0rexrRQy1iSKxdQok1lzhc8TTx6cuzHYwGyS5i8L0mXCitzizftSjbVBrvfd4Geq0Zgo7MI1GIMG+XKkcudYXDmqFiwMUU7O\\\\\\\/zvcDeaqj\\\\\\\/1ZcxZ+qIQufqwKpN66t98TvAYIBSb6q6pz\\\\\\\/qjfjlok7w39C5N6eVhZpavcUe6ABtJ7XDHa6qG4X9+Mel8P4G5kG\\\\\\\/ttNrfFajAcsQL\\\\\\\/7yHsiSoHr3BhlEb3JT8zg7VK33NuxG4+hjkn9OdnhFk7LoWqnuG968K49xVjQiCEm9gYFgydZeGnZ0\\\\\\\/EIk+NezUdEksnKOnBIK9HqNHUJggFApI2PD5PfJwWXNYvbmz5vs2HTEnIKX7keLVbb42gWauTFf1mA8sGpiYGjLYAuj1OgB3HfKrAYrCjMV5tANmxATNtThslWMHSnK+HF9J9hzDj8e1HLheOxOGAJe9\\\\\\\/agnDmLPejpjL52uR95h9Byj2ps0G4MP\\\\\\\/dZNg9W3\\\\\\\/qyenGaD4wTJcUJLtCa3O95azX6lh1AtbVxeldLhtz2jc7qbyOZlWin2qHUeSntLqqnJnhgYhH1hZ2g4qOqH8Qe55FUUMi2qz0OSDVwn8pfrfxETRvsq8m+FxDcOSFdM8N4sC1Wq\\\\\\\/z50m30HaCpp6xqNx9AVtLL9y85QZqW87mDcJXFHjCw8ljYf01KjG2lB8VRPe0vINjtrLcA1BcxzkBA3kxHfH0FwO7fnyZQLjlrfZBloY+tqiFatzkwavq3dhMOxgSFhgnzBLWgDWJ3vkV4iy2jXJomcdKcsSd7is4aIQk0KtDvmXzd3+hE6Sr8NWBA58ngJPpOD4kCWR3DD9vz+OlqwtIYqcCBIeraiftUD9VHo1msIOczRAZiEjuFZEt6keJ4vnTZd+VcPhV8mJfgvLtn3d1wuEamN+KK3T7KjmLThHcXiohER5MPh9QPVvRM5SgSM+NLSsAuEhYfdy5xcmlYDzV+B\\\\\\\/+ssALl8mJPi30AIJrTMboSvx0gR44msi6yXCElkzvVb\\\\\\\/VyRh05ImrH5s2UcEnmXDf9Z3iyh7zL0UyILZD+oKy16TrkNevjfrZ5+GrHBI7N75oluIh4QP1gkxQcbJQeO0nw9F7rTfQVp\\\\\\\/9xRQ6pHFZaAPEMKSThlcE33oUpPScGnzVMSMRQAHM\\\\\\\/otJuccO4sWIQLwjaw3kvnmzGixGXb43bR0I\\\\\\\/WsEgYgfN2jRokSgbIxeeIB4ioHQpBEjtgWFPmNcIFhRC4U60kTMLB7m5cb7EZAKmngSru\\\\\\\/w245ICcZkK1RgP83irTjwvNXxpb\\\\\\\/G8lD64oS\\\\\\\/3MEgfl\\\\\\\/\\\\\\\/PYJVhTrGO6iDvXJD3UIWBi75P168R\\\\\\\/OlGMrIThg+W2FI18IXJwhmmlZXoFQBJi+nxhG1+d8+uNAI0Sa03aZDxJTphCg4BpE42Q\\\\\\\/U0K4VsVNL2yfVO17A2id6\\\\\\\/LVrq8HCsJQnXTf75UJ0MYdZa3a9KE76j96UWiAKGOsYRNDGwe7UnbpXAGM\\\\\\\/0FUecKv9+VokBvitFiAyGqlLbGs+e7Q37iBaWiF\\\\\\\/ioxET7\\\\\\\/InZMYFWsnt5rxGIS9W6Q248e+U9iF45NM+EQh2OZUoQza6ktrkZxhsibeiHj7u2MtqGu+4IN7NswfNtBJPjCYyfBqA8k7Gb6g7+gMJgwL6O9g5nW5KfyYxx6+YctyBPvzeITjut02xADyQgBO3Ws2wV3IOPXe6JGHjydNaeOEIBnVcZ1JeE47AoQJux8EmootpmX8jP5z\\\\\\\/ZSTgozJxCTf7D1jhX4WDOrzpCFFU\\\\\\\/+AJqVMQTjUxKE\\\\\\\/8pKQ6gRUajjPiQQcPlf7++koDplNBccgrYgFeW0o1IiM2tT1b\\\\\\\/7Gp72YVcaZDIupmVdzrKeHlG50eUdCEr56wK1i46pgWfzD6iAlC64u\\\\\\\/0zEzd\\\\\\\/LgxFj1Bsnc4YEbgKMKu5SxMOIlxsuqbrB8C+GeVFwl2xUAaTqIAQPCVP33A+auW5qQXDwQMP+VAWTgB4DflYuPlJKGYyKBkYi8dXlNJDlxhfMmznzqGf0cxb4C2aVIzReAv4tFZJVStxANRYknibsXgNLbdzb2PIl\\\\\\\/SHxRCH+NPImpqWW4ZXeZcumwsCW8+AVKcs5tCGeuUnghHl80R7ct5jJnB0\\\\\\\/EbmN7UcjdkvD\\\\\\\/0\\\\\\\/RUpP1qB4ORjb3ITutIOuGAELu4dSKt3DmV2vcYitfuA8M8SleRUGSBLTcekAjgKwyjrtfRrpHpWk4lLeCV+B7ysjwHwWHaW+BrDr3CGbCFtL4TEjF9eNkSCpwWD2Ul0HvP6vNu3q1THhq+YdV215Csd7k2p+XU2ybRyh6nJPZIERu3tQcCcIswqImeni8It1rrRV26MbNU5XsjCe8ZNH\\\\\\\/f54yEGy7AObXesNvx6MO92K+VMQuVO2zLxdc3zOm8asTwlgmrXJ+f8Z41eEjNObu6gWKEc5vnRtcIPSEbK2iYHA1HflNIPAkJCfDVZcYF8PA\\\\\\\/IfH9YGXDZw405f6jbilQU6KLlT8fuNwwDB2e5Grvaafny1yc+rNX1DRH8DUqawiaqQdAsoXTkAQURQ2ugq7hjhCeUuDO2vO1DtqaswLN6YY9nJ5BYGVy1avKD7GtEcHvXOZfAGDVoTNDpVJczQnFZ25gLKvySkERbyxman9qHpehMJn6I10atpHDjXFovinCLRz3DOmT3KUF5b2vavs7Hu\\\\\\\/\\\\\\\/f05XFyfPkAJNZ2ghXMka82kf0YBszkCTP9c8vY7g7\\\\\\\/HYLAU7hVj71a+DpgpveUSj3CKWYqIUmpgORcpb8Eu5heA\\\\\\\/lV8yNJ5p2GY2LcgefMad+lq3Yh3cNFt4bOx2tRy8T1YuFgLrLeIvKOE87UhclF6ZkQc6BIJ1I1rJm2qLyt2p4qG6Ob1nO6pRs9uAIh8MrQSohQDLy+sCuh+qZN0283QUA5vP0BuX4FUKzrA2yv0FVRf4K0RI3L7S6UgCO4y5kPlQntRba7ISDWVN+pdJsxrTOtMXxGOKO2WaE69hKOm0J2mqf+ID56iPJPns8gHIdxqoHK2uhF5DpvHBUr\\\\\\\/E4UkMmwBKO1fSiXGmhKYPklI740\\\\\\\/igNihpSptvP4L5FavE9KnUpzLQxd6OTlH1G3Z8VgSKCNb9cEB8DTKVNcjwtoN4AQ\\\\\\\/XIU6lxUNxH6zmZ9lN6gj1l\\\\\\\/i3nBwbnpwPmDsGFC2Pnk8AeIcbIhHmRFjQI1Wj4LEZdriCrfI29oH20gCDQblQPS5ZtxsFzLTr3v61iey3X4USd69xjh7cCgSBZZ925MccNIJlFj9NzfhwF1OLrCO0oVKShMAbYM2AIeRwMbWe1UQR6kJ\\\\\\\/jl+167ybNdJYXC9gXwYoNdgBUWHF3MHepuBWN7Svsb4I\\\\\\\/nGk5eTU5LYSzxToqya0sNzsnJGpw2OVFPr7ZpboWc\\\\\\\/Guaff\\\\\\\/N4KxoLdsW\\\\\\\/OpM3FwZ1h1D373NpO2NqCDbCuzlOTb6o7Jr\\\\\\\/9CGnub30zxHrEEqe77phKjNR6Mw3UabJ\\\\\\\/\\\\\\\/SaissZijSBmaMlkFFx1eRLChOfcBZN6xJKqiSLr87Jkh\\\\\\\/+KyKgfs9IUtDY\\\\\\\/ZY7tYQ8d4DDESRA5zAXWGcrbB9WEUG00ZE6s8WtdaSasDm7ZH8iBfyorqqAIzRQVG1Ka\\\\\\\/Qd6vra\\\\\\\/ZWQLAqpk2Q82xWI0iH8uCFcpg6\\\\\\\/bbteuh4OHVupemEVa\\\\\\\/SMnNuwiWWQkaMQiMCBblI\\\\\\\/edm4eWaVco2MZlI0fA7dJf5Py1FD83vgwuTnTTw9LJAXAH6kLIPC9xU4OU9R41AmSt1Ha4LzNfQGe7LR8QyHev6NnXMswLwS7qt2gbuWcp2V6Af7\\\\\\\/+Awa0tA6Q2yz6qN2u1i1ZgzJizuYE5v8FfGLAL+PZ4At6R7VowlknfCX9CgGAk0umUYG2Ja6ml4bxHgExuKShNFddPhRZkRm1H1vDNezgjAXUv4JxllFOATBlE2dU\\\\\\\/mqHky2n8205wcrcP2A5gpUr\\\\\\\/gYYEAE\\\\\\\/j1lWXvBPMsDTJdLx497iQvGV\\\\\\\/DOWamf5QzMC910Odep8J48UvGEVyeZBRrf+lbnWuwDsqNEF+VxaBSXB1aTZBH0cCNODpXO7WRVhZO816rC15P13X3sEn9gfQi5H5EIQfvjog2JG3h8bkzCLZEox\\\\\\\/04I9Lb2ASt\\\\\\\/NUWgzASrWXgNJ3RuhwMXDuxoRDIyOsNWVWIB4wL3hmnUVNUFiJDymRvFDTl51GCTRnEiwm5XId9uTVXKf0NAytd4WD1P4oc5vjYgPety\\\\\\\/HUW7ns7Mvs7m4NEwCXB+C2EXXzapo7IiwMLcW+aiFeSkevQ5VyIUeFjw4B0vNf4AJovd1fFR933+EntxO1b8UGZEFI+Gc\\\\\\\/TsGH4GlBdWq7vvQi9RJueLfi\\\\\\\/9vhPV2dduH5PwS\\\\\\\/l7xZWka4gDlGctfCgVp70fPUtd6ZDq5d5DWLqVxsBNij9S61Scee2usASi\\\\\\\/B\\\\\\\/LO45oyrnck+zeCojXqQB3KMEzyPTEEcSloL9K1fEMdFjB+V3vI8BUscPIvG2VCWOhdGpv267kT2jCcS42NK5axRz4bngPc0v5+r9LBxKxmC2\\\\\\\/eUocUQQgubsmqG2JpqyoPrFqXaiaDC\\\\\\\/PYlRsQqTPAG\\\\\\\/ooL5bxQs8cVI2rFUClxOIeBO1ZggKjrz02wCqCWuriUT81Rt7jApIW3X\\\\\\\/pvqwmDmwRx9Zh+2jJZ+KQyTVY43oC7mmoCqtQjNzB\\\\\\\/4PUFMncUn94NGGM6LSfg9mQmMJ5FxzdsXmuUYoFiqH\\\\\\\/tJAUWPGrQmtWn8L+NkzSPaR8k6gCtbNq1PF\\\\\\\/o6dePuSUBYWZixJKfyJyJfV28ongkOMwqdfNkI65T4vGh0G0TdzVsP6tVY\\\\\\\/W22sqhLmvjkTnAPx18KC3cAMaMvcwo7yuburrN8ojakMQWZCu9WNaL08\\\\\\\/WSRq\\\\\\\/pAIc\\\\\\\/SMl79vw7DhPlvxhVJkwnvwG3NyXUoE3g480NQzqsXHAtGFiuu\\\\\\\/rcjgusk6fy20zP3ngmZUM5VVvj0Vla7Gmw5gw+h8x2VJv71wzkkLJ0pgUzgjW0WFhHm21GjHraukVZrkyxSU\\\\\\\/nah+8bPQZ+Ctem9+cUCuBB8husSu7RPWeo2ZqU74oA7+KRUwycv6dzq6p01ssbwT59MRAl85mJDZAwItSH6wJjjExGSt1WOMr1fEvAxUOftCs0TO9ygEZmk2gOzAKdE\\\\\\\/MtepfM\\\\\\\/9qfAbHnTB6ldZQihhfWausBWb3h4nruEw9\\\\\\\/gfsE7cmZxKV1o5gW5r9K8bwlRLr9j2F\\\\\\\/YUA\\\\\\\/Gsnfqgfc3WyMCDTumwZGGtWBq2dF8dPkjM8QnnAzqatT5HrgaUYkgMvt4VHS9r\\\\\\\/mw7Fv\\\\\\\/SnB0h44pTkrR8O\\\\\\\/Vw44eOISUe4hTYpHE5TkPMU+OfljpD51PdXSTLutsNtizEDxLM4lbphqEq3\\\\\\\/8n\\\\\\\/sVOWHuwfcY2U8G3Yd4cQTIiI7uEn0t9VF\\\\\\\/A9XgWIArKFT2UfEqTdP49WVeL+2M7doMA1O1L9nVdEKAt2P6ET6zVH+C5ONToGmCXYxQzwJ5Q1AFWOKh7xUGAJud\\\\\\\/jCYEjkGF0FQRU220OvN5GpLXoaTKjnE82bhRDPJzZv7TtByW5UF93xHyVbPM4vdJvhUMDFQVxa\\\\\\\/aHEKg6OKphY2JOGIZZhKkYwaopTZqLjqvODPpNi1oFYSaesHCpbYDefWrCh+Q9Pb2WjceoDF6e2DwrWrB499jQgzprgHT4H+VHgY6GBKSCZrRMjOuVeLJLij\\\\\\\/ke2ccrsYS2Hjn800XmuWMzmns6L7haCP03fi7mHezBa58aQX8M0bh2R9EQ50S3OpeKZpQmTfzr+jjqO79D26Se4EfL1UGr5JXhZUuLGbYMIJtHLrvfBHQUuHcmZOZtLzn7x7ePOwEuPhpt7qJodaD4v4wYpuNyp3\\\\\\\/cuY0gI4EsiVccbbNk7kINebrvCsYV0ojyuaKEDmIltvBqKwmbvbfEXgNKHXvE5YSybefmCtAz46l6uG1Rk1ZFh4KE7sQT8xLnghXdmJrVd4EI6iVfAj2vZX2wQQoOkJ0k1y3pbkZMcVSpS5NF0gujyswvKSVvY+Q66latJBMdHwO80EYVVa1ztyygwar73+\\\\\\\/MB3VG5YTg2ekBtxGcQ4s1b3YX5TJE8AYfklojV60sDPHAA7wrEpr1t6f2e+7f2gRbrjaA5ptYdmsDaeUnsrmLtY37AyxlzZBD9RG9tSOAurjhYy17nPrbP08IkKj6JpzuQyBt6uPbT\\\\\\\/EqNzsPbhXKmcBESkGue6vQlizTHuO\\\\\\\/4lBy8cpaxaxvjx4cXOqabxst1b3Yz\\\\\\\/+y9TdqgigUd\\\\\\\/8hGYnyYUzpdAs0hSPjThFdPrFKTYR4Za5tE0lg3D09mrA0ovF\\\\\\\/bhHOG8jDvl1TmX40hR+t+j95UM5eH0IecvC+ovKPjMBiFpjMjAbmlh0Exz2Ysz8h\\\\\\\/9brlVp3DlVIZVWfk0sbh6\\\\\\\/\\\\\\\/7pkn2ZKjEeq5uUZHl7xw3VMYJGupnpAsdl42\\\\\\\/XfOB15oiBc6i\\\\\\\/dzd\\\\\\\/886MiOkmq2\\\\\\\/cf5FvltUl3+nkTfmZ4JIuXbrA5UbtKyzAOjth\\\\\\\/R2eDoCyUeUzAt2\\\\\\\/6Jc9O3R4TGFVZIIBAkPHiGvpGH525EhcniEr53iXEYOqgUsj5Ra5GmcBNoOa57ql9wkyhz\\\\\\\/TYZuv0PTpRH\\\\\\\/DTsXv1Z9geBWHHHhqY2d59jO10+57TsABg8J0jfhx+R8aGXJX4Y82LIXm4BXlvTolr8BHtNpXKFDVGctwDLOH5hoZLUjY0IobQZpOQ4ymMulmK+I82EPJuMpx6DJ0uPlCzSxmci3b1T\\\\\\\/0LMI+7HlCcsGNR09hBYKj6fVzJ7vxGPbw83gwvXqYTH9o3F6zTLnp1COO1tCOx23yiE5ETVwBVUi61tCJMcU9JT9A7UZw6xr6gzei1kKCOZX+yfTsQ1shu+Iioyw4o8B0uiaxUcFNowXsMNqQ77Sk\\\\\\\/NbFYDhK+ELk4bOXP2S3PWWtjjV\\\\\\\/GlF1XGttteDglb3Xzxo3iDyQ+L\\\\\\\/DklAUm0f4+L9sDDJ\\\\\\\/huoxnm1E7cremD\\\\\\\/K5gDaMPASeT6\\\\\\\/4HgOfMivOAOJNoKG9dg1MqjEAlaf0Rbh6pcJczMUKFW5ZdVaZuWMe11HDLBND39NdDQp1w3e0M8BHPTk5fTPAdX3k9+f2aassTMODq7D4GK1wMJhfVMUuOlVAaKTqOpM7RCOP3HF8i7210gNul3D8VOndjuWdrX9lvw2iOdEPdIUIF5loFYKCuW30vaFEtPfwD9WnKGFpAs2x+RYAjDR6o9jznRXBBCdDfjE1\\\\\\\/fyW\\\\\\\/OBxOBJL4U\\\\\\\/HVgX+nwbWDGkQPGPlvJ+CPqJBE01eJwZgKItmkURS6e9goRR0TvHmlO97QaStWsHYMZJofOlY8LZbk3DJRKfGeyLb\\\\\\\/FTR1y+lv5JE7p1YlC2W\\\\\\\/Ay1jpgNutqXYkeyOw4SpymQJXVkFY0+NOnGFEjekYzeV3WoRKol2pXhzQULVpZ0nECfk4D8aCz0LLV6+GeaxsSS4J1Za1j6f9nhAJf6ujoIMpsvwIlgl++JwbzHioZIQpBjBNlQs9dUo8OlezUx7XOkaQ2XCJOImkrCcEgjWrdm3F0G6GAb6mJdANV3Ajy\\\\\\\/BoFzj6iJBfln4H+oP+9ySOYPkYaogzemvBivpS\\\\\\\/6XXjnpNdMykf1SJ2OA+AtgvkAAM4Aq1kIeEi4GdfCr4GOYRpONl8ZV4o1lXJL3MZDhIeg6dUv\\\\\\\/J2rU+RtYsF8cNkNVQ9COb+P1SaTRF2sCw8nfQY1hp\\\\\\\/9UgS3wAen5V39grqy0qhiACY97yatomHosr4uvDDESvMjYHjzL0o7Oo+5yq+S2Fn81bHb542geIAkIIXzHLDWOHmtjBX1GpdVSrZ8LBlag5R0NwndCKhlyxbsZNdy4yjMXH0aFgVXpcAufBbyq+JbqRzzQOE70ABgzhWyxMdF\\\\\\\/18IzpIdI5HitMb\\\\\\\/5xKTnHpVzVNZvzt6cXSArsKZBr5QQ3wMBjOKe\\\\\\\/gYE62VEtS62+R7fLgNfJiZfUls3KhRc8p4bvqRDIetjn0jqg0mFwohTbu9G66HRJzCKLLnJ\\\\\\\/qc3ryqkFtXbqL0BhZ5vl17PgE5tsJSMvgH0xzweFZFyuDtDIcOet\\\\\\\/53SfIJ0LH\\\\\\\/458t29qr5QQBt28x80OhNKbikc\\\\\\\/7vpz8Rj8O5OqFvqknPdhgk72CGQnPWdiPuF9UpF14cEgE5t7JtIQ3vW1llLpbSFgMN5Zl+DwLyn\\\\\\\/wW0KSdPTs3fK67EahH1gN6v2SEZLwSBPUrD7qNU0fvMiBFsFl5r6xmu1a8Cki2K1Zj3OKIXO95+WsrE+GgY6eTISTeZ3Mecu6wpKjcPw98gcMyYYZ1Ou0kv1CxsHbqXvwUIbLwb6BN20BaoN7+VXUCJ3CzKjnCt3K1rbvvUt0E0cy+DpnUE794cuIdxr1jVe3W2L55D3Ihi2Y4V1JMNrEASBLMqmKYDyZXn2MrJNBuGfGIpGBe7Q1yh79F9okn0BeKvwfbdHT3bdDZBDS\\\\\\\/CZbTUY2Ho00erIQrsW2kK8Wp8Q2Ifforkky6L2v53ereNaZqzGE9JiFFWNFrhTNayV5dL\\\\\\\/8XBG54Bhyp3rHalKRpk3H52mldIGPSlrPlG3QKdgAejSkykuG6GgEdY\\\\\\\/hhdjhEGfcxmhra4f9DqKi2P\\\\\\\/XUIaIxoNu5SjPq1EIJwEehXeGWGNtcm99Y5YPMFesKfpp2CAFdaYR1DWIfzDSGyIZMIyxPb1Ikxk2ZuvMbF19MYOGbTNbaHSVE\\\\\\\/Bh0WRjeeKmnSAlT3hdhbTlvR+xmmpytWeuomqOL2TlhstUCx1btxr6OD53dQJOPG4NKM1IAx2hdldVRmI9tRqtsF+DLxSIX6mA4xMMifbve9jEb8PmNSpmY\\\\\\\/614R6KOPk9LALFhImHG\\\\\\\/GCi\\\\\\\/al9FredEh1t8qPKJzaq16yrgv6hv3AZr3B0bYlfaeXKAR0D9gA5xWz3\\\\\\\/2L2T1+Wp7IdMgN8rbOnj0aHnMltGWtvceZxMn+wSwm+LLZjuLJoslA+YsaRxG2mFd5KWe2fwUSBWhuSPK0pXg9ldaGgdhog8DHNwVVSD4QjkuK840I2vnrp\\\\\\\/rjl18LipYHOTFO8VOe8qbeP0oFWLBxPtDHfcwOSTDif3VWictjgaOjq+Hi\\\\\\\/lh+Micc+b7AZs\\\\\\\/AByxkgjhQw07H3nTIg3KcvfXwJTWi+Or6vCr9c1EkNQx5cZvrguDwrTgTC+vFDLmX2ri+B+hr7DRbzjSjL7dj03uB+9mwSqdrsBpkuXTad3IQ6PRl85YjDxPnPgq2izGoJGl\\\\\\\/AIN1aJVi6I7xj87vq7TFnRBu+CoalvSFAIyQpMq+GgPEe56T5Bt5l0YLHuOB4kisVPtZKpvj0S2EscbS7E3t4UzEN1lQ1FIW2TrYb0AF659DGtr0QsmFk+68is4SzYmyzYQW5swwmM3kjEPs3MOHCx\\\\\\\/UzXERPrKDI+aq\\\\\\\/ocRj3AYxKNbqyVtXxWcmQUPrvjOBaXX5zMK0dr\\\\\\\/Tu3Crm+GF3FpBoluuugenUDRSFEslW1t9mEwtUVCqwTIShDE54I3yPXwzOaOTsGXBOXUbqxNztB8qpej28ExRTLz\\\\\\\/QUtCuFoVX1llSvZY8lp\\\\\\\/SpP1jrPb6XCV4W9qp1d1RQmgcMd6N0TPjGRLcJX5CluohW+dh9twoyJCN6\\\\\\\/rC3sVLpi+ZN5zf8akrAg\\\\\\\/jv+v1MlctB8X\\\\\\\/16MwTmC\\\\\\\/cuPOWmMi2Up+KexkmaUiY+zE8sEJWP9pFmW6hD06A5jalZN9S8IO1MkaL\\\\\\\/6KbfQYLtTLIvllRpeOWTVsrVc9aVly1ZUw8EdKG7mJpC13XCYOx+IA8ZS3SGNGe2PH\\\\\\\/J0LFWDKLZLV4ETkHHrfArvvadpGQfq2eoxj6owlkZ1UHiPZVzExrraFvCuBnNeQzXQPKHpSLqlylY9PTET6Vn3TftJrR\\\\\\\/e6jTeaMRdsxu0sIn2es8uU\\\\\\\/Y5sFi79v19ETLcjED0VEkmmBrWKv70KhYg9TVb8wYFS8qFKwCIfPpZ9qye\\\\\\\/W78J8EHZrs8uSd3PLykaVDlAoQuvFwb1ceLCSrNkGanl26D5X\\\\\\\/74HLtT0Nxic3SfsSig6ZgPDVoycds6ig+KqMD+PTQQo9Avbflzl4wr8qNwhT\\\\\\\/LDN0p5Oi2gdr6CkLie6t7K3Vun1hDE426+RAlxhOtZiHvp1z8nvpbvsrTju8K1n1Luf\\\\\\\/RfKFHtarMoxDn1ips1OC+UI1tINofeRpZNbiCVhXf2b4py8t2AimewmeLZw\\\\\\\/fT\\\\\\\/s3JP744XwRzhK7HiPQkDPd4jDDx2NK77IJkgod3cxjMkvSRo1Yu0i1egYSbc+KNEDG0zdKe1g0f+4o2lMkHqwjn9ugZYgDZGLJxPzT6\\\\\\\/CqJMsevLwtx\\\\\\\/MeWb8jIXItlsyE46D1lzY9k9NaFc3ZewCTS7N4A\\\\\\\/8WgpPBSidvelTE\\\\\\\/865cWzI2Hip6snVbWbqVFMJjTk0n5XIPdtbmswSEFrQ1QwR72t12VcI68qGF4zzKZjybjrbxmYZHSFqOEZifeC6tmmCDsk1rWPso0rQ2Ot2hcVyfi\\\\\\\/8UonnFUJWciMewpBfcRGgejzGjTd4L4n2N\\\\\\\/l7qL7AsdAmpFHFwBwCg6q1LSKefDPm61\\\\\\\/gqJs+l2hjMGocVmIzbuhk\\\\\\\/W2aapH0QBtb1zSSsXXushXQWzXYEAzhfIqQ3eyyFZK4+MXI61gJp2uYSzXB4W5tng5BI4YdrhwUITWO\\\\\\\/Z2c6Ehbsh4f\\\\\\\/hSndUY5CXWbM+O\\\\\\\/mj3992KvTH6+4ySNoeOAl2Zy\\\\\\\/zG\\\\\\\/Pbf\\\\\\\/V7uWEa9UMnTdJmXPvqbTZaAqzh1IzNFvSfChfsopZx0xMfhscJ+KR\\\\\\\/W6WgpVk2z5yYJdslizqjaRbEi8x0+Ld2rRCSzdLk70K1jN1QgM3A0Gpue0VGlJSEkYSJsiB6b7onrYo2TjmayDYOivzSdftUm5rwVA6szxC\\\\\\\/HW0\\\\\\\/yS4pzFqnXdkt1zRF9vaBg\\\\\\\/r4u49RGCxkNKTpQVLTzFAKrC7NezrZV8rHRMqmJ+zZaeh9Q+FM++r6Wl88hF4B8UNzdi8cSod0K+A96XsfVbg7jzbDv5BFyAVFpRo0hh6v61Z2FqqpG\\\\\\\/rzHwqMPOP6VR9jFoRT2QnNfbK50pFzBNy4pM3rswoWSLzSF8mV12NxeD9vLv4L0PtqxDimCnrI11JgZ7HaF9jjS0acFy2L+I\\\\\\\/66dIFxvRxqHkq\\\\\\\/YkxDjVXElvoKLsaYkBCh96vJrVWBzMNl\\\\\\\/UG8aM+afMDDchbY+0btUtKZHLjVYr+JJci08LCJsf05JheiXCU6\\\\\\\/rjz7WeKHzz9vx\\\\\\\/XRKgNtGfmp1XfZVBvTOUN6aMvU+wfRiil6UmJdwwJHef\\\\\\\/73QuZuTLC9QwS7FcBEDTkU9\\\\\\\/6aBeM0e4cRxSTsEHo9S4BflvUDx2n+VZeTvmr55nQtkc2cWsrdWOCA89DmWnUCclpV+XzfKU0EKCJZlNTNuU2UDtRkHmJ9MQgnrk7iVNtqiEPMaxAGG2db8ZO2Th7My5Y0jJ42ndxVRDfXbQBwjDhh5qrpIGVdoR77ovs90j87SllJbDdYkqIXXs9dLWxpyPq2oAg9wCRqxAaozaXGcwnpffkEsd8ek8kCkb0q+jH9mIBOK91m1qbwoGvD6i7H2WLNo\\\\\\\/CiGG9Vr+DqlhZBw45oUxPQU9ByTCgLilIHzvrPmrAp1wERAS4wtlmlyD8wsZ+lXaJWjxlcgPRn0ePFND5NAeiDrsBKw8S4KiQFc6iA6oucRarTdZWK+7rGg7sppMphnjWMpgRkOjsM29cfTxm\\\\\\\/b2+U6ZWFWHWjxMKi\\\\\\\/REb4L9sg21X6diky3v\\\\\\\/Z2mrQvoW+f2IzyXTYx+IDySMAkGCCGG63jCgB2B\\\\\\\/+2Khv3IvBKrcVs2fR2jBk+K3+eLb9VXU4cUzM5hKOZM6SEqoXBsgMXtNgXgOaMpC6cVh\\\\\\\/ZQf9hqtHU\\\\\\\/NaRe0ZXMX1rGdBF2K3HeMmwX6JyHWedA4FYL2cj55sE4+dZfwq6rxKncuceQffIXXAlHVLbG4GvYD81DY5MHPuymbZGrUMWQyq8Nwv8ZxLbn+tUPOpaLRX8GjtHMnEQ4Mp+uOldY+3mjaaprp2\\\\\\\/HWGmMVod9ceHCdWbETBf0BDGke4wKO3okjgW9uetCzT40h6VKlYvrv3fepFgM9CpRqc1HkUHbvEn7rqXGs0qXpICan1ML\\\\\\\/b0Qqf2L0VDs5NL48QfU5e26ZM2yahOBySJWn5WXPnB4\\\\\\\/IvWMK25E1gLhttANXYjxGNma6\\\\\\\/enXfyHOXKgt8dgJVfNyk3APYkhko1NcmYNLaKBE2wJFtoTPqGw7HtxwjIKP2zb+7gkqvQplxZvivAPxpVfiTS+lHhz6O05TgwC3EXTX4Ue4PVytqCcWH0RYHETjdNMp5bNWqhn\\\\\\\/4mLAf5R07BsmxPPBovx6YX5VUDcVEbGPBW7+M\\\\\\\/967EGEI7GZ8M\\\\\\\/Xc34x+i5LeLWXKBiY5GqsKkgcNtg5xTFiWxjsS3oW43J8BBd9uts\\\\\\\/2ESsYL1jBGjrhdHWu9wpEK70Q6yMaDu6bVHBKK\\\\\\\/EJUeX0IL0nh4\\\\\\\/wUx4qXD\\\\\\\/hoRwGbRfYDcUojNvDUrcZyRodiCzsQeMVVtJvd6PlziOSWRPJcaeO114S+Ip0YvabXzRKHMWA4YGqw83U7qsdoJaoujjMNVrq866puY1kRWdWjIG4P6WlQhA0Z5dPz00PFh9dPyRaVINb8rBYJW7D13jXY8bbCkd8hXj19jvxnpTkS1i6z+3K0bxPnxOWGLlqOweBNv+Hqnaoh19KWOGiXgyZCzWql6dMuetM2llLk+TdLHGDT1wyAkonOpev5Ns3mMU2gecApRFYnStcO4F8jQdMjTpn5RJaLjMCSlVK8D0nIN4KOu3d8vBl1Wd9+72LgN\\\\\\\/HvEAzY7lrppcXnV7g3UIchoVQFeKUhHAYgDEKSfq1G3hXVbOr3v\\\\\\\/USZh\\\\\\\/3l\\\\\\\/T1mieOcJD3MnWxqbkNqqF4BOkUpEHLZnus3pE2gfLuAOrJdLiUQit3dChLtiZy\\\\\\\/gmDVIUFwxTlTIcrYulAd2ZxRD51e0PwyjzhzeOmxpjsGBrk074ZR0tf9QRKsFaYD8ZBLaOTloRa6Bjrx0YWIpTyriTM6e5TYEkEmEEyI0SIyJQzn3Tb+WW4yk0vADbuVfjA\\\\\\\/6lylgj8msD1Sq133B0\\\\\\\/pVNtV2l7yvQpn1i323kcvuWl3qePCeaSHeNF6Yk8r5r4bXVQqG9YW0oNQD6y4RFJ05T0\\\\\\\/BkJ8wRIgcmxGW7z\\\\\\\/9WTRLaWcDBsctVN3BHTzJ5kGWMN69v7dNKCmJre9s1PvqtjKteAN+vynuMiHIK1l47vVcQdYLxgzMTdxcEzYuxJM\\\\\\\/iIkqzXps92fbB50dfObcBiNPBVvrKTFD5PZfayaCSy8\\\\\\\/DIM+4y6hfOFlZXcQZ5X0OCVEkkvDVvGmrqWAh75b1ayI7t0kZr+4TgcGsyEFMfumibQtkh45NiPP6FaeUCm56Zv3IHISiWltYP+zdLwMykOR3BreFKr5yQj4FdyowE8ft2rnXKV1X1\\\\\\\/ac4io5fzwer4LfU4N\\\\\\\/zEXLWs6d6oKeEemGKid35JQMXz1cPl57\\\\\\\/+Wld1NfIKGGOH4WU9Xuyie0D2xNM+ZYFLLpcTDUl+EpUFD\\\\\\\/zFj1eQL3JGqCSPnAzjuj\\\\\\\/vQd\\\\\\\/0Uy390QnxoLwjzACYZQVDbYRTOk0nsWe9GObkshHT8w7XHV4X8nm8oDUAeudWFRl8mb9ivK50diSJCuTYYr5GthIE1vpqYm\\\\\\\/rVFigpyIDEGJIjCqTi6\\\\\\\/4n8bceZsUnaC7W3WSbJhNNmudvZc\\\\\\\/mF2aWukJgtHnB4THEtZE+WojWkhNEk\\\\\\\/hizZRJ79A7V3TY6wONK7Jhb+k6J0FVWP28DbLUTorP\\\\\\\/BhxE99X8n7cQOphnGpzneyidtMkuXTGj7niMKjyMIRv9QNHWd2v8JOskZpsi0dHIoGo\\\\\\\/I0pPDTozzQe+M5SVScLL4JclzRr6VaDGTzjmmtqznFKBp9rX\\\\\\\/d+iPSGTw6YI+drnzS76co4Ea+hXiY9ei590o99H9v\\\\\\\/yrRS1EfGZodX29QdQD7QO00T1RI+4EmtObCoAdpoxEaAjjcPNtiKxvFO4r0a86qdXCD5ytfGkBidjLFW7VolOiJ8foo+k9JEBWpfi0r1Xc\\\\\\\/+3yhvqG8788edp2+UG+peHuJritt7Dkvp5G4oSDtjqaWVtVehfvOjDc6rfgA7KFgODe7ZvSfdzrP1yK3LxRZE6yg2NK4lO6wTvgHWbLeRlqGV1BGZ63FqJASy7PdylkWUXa1LhEZ1BNoOMU0zG4A3+VYXUE77ikKBbtZ1Eb7LNQBb5BBTL3KkIxgl5nU7zOsbjGAw3S3PPckkbXtjtHbZlaRH0nfnGAHEOnsL2z+40xtEVDLZ9e\\\\\\\/8rrvvJl\\\\\\\/oX+dB\\\\\\\/bZt4omXeBsKso\\\\\\\/RqAIP79urEiTYbqAAXc7h+ecBlpevVonE+9+pCXYcKR\\\\\\\/g2rv5n7RG8\\\\\\\/mTTG\\\\\\\/0T5DiKMzHHJyq5oik6vanPzFTQ3ggC1lJerYhBME3VlrHkU06KyoDBRI7ENJeNPKHRA94vhvzizT8fXwBlFNkhgSCogTc0zdM7k\\\\\\\/2wUPH+5b3rP3GbGwFUR4SB3oRB4YrZg+CJA\\\\\\\/L6QeZYshV9SlmQ93wFeePH6s8BcBbCHfpCm9T961WZDZgcZtZ87OFHRAzG58gIj60y9LCZcRrYYJG2bJqINkkC+gF8rfZSuSucOALe8QAtMlhtm87YO3xmz1xqGcK3oR5B2nw0jT3WxXaqvPd\\\\\\\/7OBci\\\\\\\/uX6Oy5RKYBGjdwDW57KBPCXW03TBnjd9GPTwgUtw+NzY9s3ytzKrQWy1vwv+I3r7CriK6W0Mw97vYUgcXy5pW+iKn5aJRyqi00gLgEfWtLW9GYyma4uDJCzk9Tg7JkzrYPOcJicAEgmgAukj+dm50PQLJ6B9e92h+CH3mImPhy6gNetBly+VZWU0+q4LWOIp5iKuSS9Mpz3V+81TO8AcsWQurxFgLof\\\\\\\/09a1MJ3rXsH+irGt0neStRVdPpKew+naZMvQM+9Y0TwYCa2JV4+ZLXg2loSRXCRyVr6sWqyW4kX57PnOLuZm4VUSAIsj0Gl3yhxNHOiFk89JZG3MDxP3N99hTUUY0dY2Owp\\\\\\\/e+NBhvtm\\\\\\\/uIf0IwfiCv5euOZTfycOwSds3V2RW2DLkoV3tASA1PCeDI6qmO\\\\\\\/tzmuCy58PZHSh4Tu6txCF\\\\\\\/FvtmG7beTTPkaokfv9A7AX0ncdDWNpfY8HUQBGRjS1H7KunoFF\\\\\\\/7EDdwa9H0FV5K7JtWEWOGEvTUwOMSLHX0dj2DmVr5FcGinuarsMf+H0m7n96eYTWNd5sBezOwmHmd1oYPhBRiv77vAR6WjhjQUPDDT4uWLU33Fq9bTfh18vHVhybTOlDE4Kq1WrFXIklmbzfd2k8+K4yCdx7SpbYuLbirYX5mwK9EbU8\\\\\\\/5DfyCLZ134ubrwDaoAjYlp4wUh\\\\\\\/3SmHMYJgTPy1G40caClfhQME2yTBzX1RhE9hv\\\\\\\/VHCFeAbeOXS+FKOjzsFdG+5ChSar9tOAAzaQuN1P2BNNTumC+FmUZvlnENqgZk535Vs\\\\\\\/5FTTHAEzhdAwA2\\\\\\\/glJrLx62\\\\\\\/qHr1iE4fATcMg0mp0kkFPs0TbLzBqt\\\\\\\/Tcdh9rgqgotGve3muVz3aBxRI+N0uQYGSlH03RGsMZTIFBJwAxDWIqJKg0dTkYnMMpJN6ApyNqLP06gyzlvQmbuLdo0r9zIuvHF8xgHAJfaF2Ep8fJWDFiBCOfZ6hbJ5j4SsIsto5r10nEO8ST2gSC0fkwQa+ltlNvkQ779B+t9UmPpClj2zjaPOjzglPM09J4Lt2LLhDvkfSkH+YNxNZ+Xf\\\\\\\/DKIF2ujgX5\\\\\\\/VQWw17UftmBu6u4BHTky46W3C2202f\\\\\\\/qeYlM+UmpDhUlqQLR87Lazxj1ahX8lmZ29vamMYjp\\\\\\\/Rp7T\\\\\\\/gzYtlwppNf+pVYKEllW\\\\\\\/n\\\\\\\/OK7Z74mTK63nk5UvYxzbMaUhOnSCs1UTG9iScKF3MXYax0i9cHB3fjuDwBLg2CwdL2D9wqpOmASyJ0b1C4RT9TaygzFQ4jzm4lVja8UM7+nX1ItZF2fP08LBL4hj\\\\\\\/MehGfWjwKIHVOZB2Xq3aYQJC+zVEbRFkV71BLRdR1QmCn0cRw+O4e8qQEXim8+qCYMi9Cgbj4GfehYEHdQqMs758X368sfddCm0RX8\\\\\\\/\\\\\\\/e6jB1yhouW0YGqhjj0TSe8hlbMZ6B587JiSBkrS4pSn5t0rpvguf1YLCgVCGqSZgO+rKYBtr0saYWJygShKO8GeTY7did9Dv8\\\\\\\/oTgsiej2mZuWI4W1oM00faAjOAKBNuYxnwbD5KP1TDw5AzL4xg1XYm7SDCG37PAndl9uP+EiwMkmLF3Dy2TAi9HcQapKCF5YR5wnqMLiVcC4B+EGtIntettRb1lm4SZeSogBWLVnMgt\\\\\\\/X9GQ90gfglRme0RVy+gCsVDAE\\\\\\\/700ZKDO5Df01+UE22qjUDasA8dEk96xD+bsgQOy3ztXyxInWrv9YIt2zqjzwWPGyEYoPI1HLDXc2lXZlRuJskCw2WmKumcWPzwi9JOkbxWSF95IA\\\\\\\/35mYqqzta1bG3eAYcawSOtzk6z5pc5lopL9yIOFYOEnhKFJCDrhSu+cyLqQZTZsiZcW3RQmcCVZDQP7HjcjXVEcmWQvOns0okPAcJCm9kBYCFSDk0GzmwaiViD\\\\\\\/9A+ngPTkblAtIIt8SzNm7SA5zQw53Ya+dGjRUD5JXAT1lZtksuReEcZjMaf3pMcwTb9VrhjJpODfTAeLFcX7IENpWTVzzSf9q83ypOZycJJ6O\\\\\\\/cgwrPdJseNZDrUIhJDrIvpEeeR6G2QdfSyTkQY0Fl93X2g7psl2MEWP3Q4irZa7nPn++9XAVh9sPMXoaY\\\\\\\/LPf6ti5\\\\\\\/fPepSIyt2rHWaY8gTON2W4dloCZhpxa++t35IquuI9PlQDPd1M2hJ5wOiQ6ziofLMt+mfYNg7kJr0RIIvgJBjnibqBoOZ9Qb1eWuO2TOm\\\\\\\/PtsAF0MGaPgNrNy8JIBwl4p5INMvVDFH7TFqRZiKigX9QoWXDKYpEUW2EDnA6DpHSC6zpyOhYFEhMYOkMpZ0rAzYXeW+1m3NUCA1rBva06fsqd0zIktt9buAU+Oynp4dKBcjKFULni+BckcvYp4eBcGnltt6VO5LtPCHN6yVT1C1Twt16H\\\\\\\/BB7tUZxw6+uwQepVIu2qC1ANzci63mRhoFjRwMcRojyLoJ+aS3DiLMUSZ9xC6uqPaOJpP52IxuA\\\\\\\/yQT8tdxeC5GJyjgSbR5xtaij2qkE36im8I0W1\\\\\\\/s\\\\\\\/fpJz9\\\\\\\/4yHZP4KkCQA1A+fVxI6T0hMjoZoAfe0jCqIq8MuwTDXIKa6JJOL35Bl+F3MR2QjVvoWRvOotyOCLBY5roGUbJG8tEHEyuTmk4Yh4NRUqaX\\\\\\\/40+4oMxfIjpv+CaKtASZICxIK1iendIb3B22FWmkvMfNJeDtiyIrJRXB3LhA\\\\\\\/ao3vKibssgERJuk5Gg6oIP4R\\\\\\\/I7Ke9wXVuyqodS7zTxXAZ82UDPoJRAa8H4QGvNzXdaEH16p9YYiaJhnIy\\\\\\\/hUVC701WH1VTLCwluBEwRG2tC874+8f1XH15NFQWoioquW6LYTTcLb4SDW8AvqsubV6gDxgsyoy6b2UoJ\\\\\\\/j+GaSMwoB\\\\\\\/\\\\\\\/uCfy3evtyYUry7+PfAmYg9JzwqcyRNlYFnUkhPYzN98wkHA9vFKcgOReG1tBvqvPvPtm9Kmm6d4212cgR9UNAVD3DIX3MiJEoxlgS3yv3YyW4KWLFXul4dQyniY7l13U58Zndx3OZq72db8UZFkBsRKhY1VQP29oOW29QgcBH4hQChSMhrC9CZgX23ts84Xb0kfaDk4r0kvrUKvfv4NtOpVMPyTrGT3TQChDlQ7ZfTbLBPwZtbsLwYpDt8KzQiv9PJYpOeM9+fThOthlf0NbJuCTQNZqoLeWcvJVXGA18R5rgvqkcQ6uiovsLTHbjNme6HpDp0pYSsAHITiUHrHpCHBNUTUPYMfuUo9tBI3zyqmbs1G6w06unnAFs13WAsmeJa3c+FBRwO2qnZfjCUmxXP6SAyj1tAwbTay0TbZb\\\\\\\/CG3RS1yCJB8bCtRgVwGzJ\\\\\\\/I0\\\\\\\/SCOL1P+c+vHKHEsTAholymsnxQ0VkMfx5LZaHV65bBH5Bc3zcqM9Z2LbDEdOmbPIeiPR76Eb9xltN6d4JhMxL7603EOImbhssTTYki1CGnOlSMK10zstbiGj8Pkv84wagWNo40sEHAkspujFyTJQ5FvV9v1PEp1bCvxQNWt+IRh6ej12KRder4LGObA4PyWDGXwe5jjBV9\\\\\\\/TatnbiOQwuAQRhFyd8EBX5BeO4DKDYE1841eWzyzjXcLyxESoUxadXal8OKi3TZ\\\\\\\/wAcSZDYnvNM5wLVPvqmVnEXbUgRfRyuViHAdXAnql9Bfw0gjKAHWZVNn6\\\\\\\/pzahDglhsD\\\\\\\/tykWu6fhhAOnAMRll0U1MQUK7LYEyQQhNG0AphpquFql3342B8lavvTsTpwGi\\\\\\\/Qv5Zi2qJ0x16Ki3pfuYeA5uRoUcuyw0odP2EK4nv2WExeQqSvCsJSnKH7LiIJHp6liKaTLMVyxQIOP4sQAjbF9l1VMOItgAngvYGjGIzCj7XKY4jTZz499RK06EVFCLa\\\\\\\/qDQ62pvF11Vq6S3Y6ieRa5YQLKS5SAoL\\\\\\\/61qmgus5yRybQjuUrGV6MSFuhPp4ej1bLjiRb1Suc7rhbhSFDwObSBRP1lU6YkDoFMDE0n\\\\\\\/AkvjCO3EzZRMjC755i\\\\\\\/rLVa24\\\\\\\/whPJRqWHSe+Sk3wIIgIKJyr6BPiwfmJUIpdynVSi92xO4T\\\\\\\/zncvW0QauONZdQSwNwUbxEx91gfDAzJLrZDX4SDj+KRxZBJ8XScnWIUMrr1RZ+cBWzJ0dKJUZY\\\\\\\/p\\\\\\\/KJQqf7JoDOT8eAbn6M9Vv+UIPArpTapaoi5Apfd2X3KiIehJIMSVzxsaqjGU+mKEywdHC2UrseD2hl66NFt5bJyPc0doawYCiyKSs6UUNjgbpGRV\\\\\\\/fVw5P2h13D0h1aERkotBirW50Fv30e2bWb2udxXkutadwZepttxF1TlYXUn4uShydcVHkv7A4tDmXBOwmnemvMLiUCP1t0h4IntnIOiLVtCzepHefxZYhT10yQgIj6W402b7nYW+6RM6ad9oAsRpOpt7uikSnx7nTVrc+aLN3v2MixL7QGLKT3TG2ku5DwTHtnWeXfJY8VJrVjg1S3pn79oXOhAQP4yEt3ESxJ9Q85oLPgoEU06HFPMAltgKFq8NMHZHy3ko41Oewzg7k1V4zVAgSPOvh1++SmWn1Ex1GUH121lLNRmZAufcW6Ajm6\\\\\\\/JNXotDyzWAwXqVY0lMBTBcQAH3vdz5V+sbxzIEn0rq1KqUlJl\\\\\\\/a6IhNZvSjuiyzlOoEYIPpiAD2L\\\\\\\/pf88WIdGhay7eg3WAimfJukptd5Fz18PMnVIbgzmMaSECEQFSQisfv2ygk0mhYSJhCOOS6k\\\\\\\/36pUNl4BayGKX9hRltV47yT+Tb9zFsdyzsi3xW5hqVrSDySYouUuRLdkkhgXuE+EeraKgEaanXslV2LMQQSLDG2C1rT9B5SWfWy081frDtuMjRKmd\\\\\\\/CTB3DlXQz1A8gv1JCpOJMF7v2ZlHwKq2yT7i0+H2dbauOP+8WSyHt13DmqO4IDGVxlY9rPRQLXLK29Vgr4SaKnNYKytS5aBHTsVQ18sMItCUaqiL0GlUDOso5jZE2smfk1WdfsDmZFL+DKWw7xx4\\\\\\\/qNNQdr0vTn7jqP1QcDw02uuHh6BGlezGMajeHWhGBB3ynoGo6rFUb1invygk+LWxuKoFJI9\\\\\\\/ADTjBxEomXkaYLk4cotz8wVXiskRCmJFitBfhCf5bN3Ie3kq4\\\\\\\/Yy2fD70hHugsWzJstSrbYh7Z237fHaH5vsCHMegsS74ThNjNjr3486VIqXF8kol30M8X0PlUx3xBlLztE\\\\\\\/DHRJ5P0vE2Cwtea4Qpxaz9k3XgIKWkX+4RF52OUCI\\\\\\\/YzTgjUk095bbZTqPorV+hFbWBrKi2iQ6fy2QP9AWY+g7EUgPUwUR9CzTGI57bNZ4oWM5mpIS+uCc4blYQCC7YSUE5TYP\\\\\\\/9Iox\\\\\\\/ii37KKjmdBuibolETyc0rbmd6B8lahVfs9mjIgUGutc6xLGXweQz+mEvr\\\\\\\/a\\\\\\\/aaqWVXNG\\\\\\\/C0j8mX4VAY6A7XI57tT\\\\\\\/gXGMlRlQhU3JMYqx2Xd43GfBPj4IRHHZFcir\\\\\\\/xw8exryiMxElYIk3tjPwDOQMXSzl898Ts0usI64IplHD6XjkMytFuLEKc9sc\\\\\\\/63ePUs6lP7lre8WepucbPgDQHVbwYc\\\\\\\/r\\\\\\\/J5JQ3ZlPGakmwKVo6+EiRSZ1q6lqLNdBQyvpNFqj8n8khkZdd3D4m71XeFQ1QaD5leA\\\\\\\/TZhEPVr4ciJSAb0NIKt6lMDA7rgBU4O46zTIUcYQ448Qj6LgQr8xMVYt+q3WFeOEMBi27CSFpKStxVGwwF6dim15VGhOZ6ClzMNBFq1eGRvAmwGY\\\\\\\/Q7lQdPixQp2yFhfL6I4Dw3qWGXu\\\\\\\/AA5lYUksjzR+C9G+Cb\\\\\\\/hYWKQnYsrU6iRt6PpgyXhdgfOxJWflGgUWEzUiOAjhq3xpm70Ow9APz2QTXd9KJ+Md6MzDFdYkGmtCEnjZL3Sa9EL\\\\\\\/uuUBCkQfxWBJ8znI9b4IfasYeKBLdcD12WbEfTtFGyyYS+X1pgxMv3wZ+w2T4hmIR+t7oiH958zSXyMBuy9oK8buOzvGqj4yZD7ODJUe7vgm4X1HTAfo4HyjnVt24KfbGHcP8JVPnolf8z4edgSTtrYduy06B\\\\\\\/Dk0OqCswHXP0Ai1KD1JuBfDorV7BMMZJagNmZrTcR2MHLKNShZ1NF6J7eTELM\\\\\\\/AgBgaxUQdCl3\\\\\\\/+hicFngv9zqs82e2UtMtKlOAy5jPBKHnljGuhytT30qZAIBOLpNOS7Y6V3gESaopi7sZdsO2IUAUZw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xE+WYd1R5oZX597FdRQIzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224598b2177638cf81bc63b55eadb72722494e60b6582dbdd19ce7fc84c6681fc3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VFtKBinCdFAGdpQ2iF+Us0gjyLVhQPRvqtx2Ggv3+wVaKtp96AJ1Ky0Ru9fWGZcPQGIGEtiTHj6h48tLe2EmWlSoL+x17SMsL+HIoiWAwV0w+uUAxGAKxU51pqG0TjVraltyI\\\\\\\/4EAuTDUoJ\\\\\\\/gU4x09upEaUqU9IYonLtiXIabvh\\\\\\\/azKUFiXauH7E4l94xtZYPQjp8m9N2Hmot68W8FFvMJ04CWiuJBdKIdCq544LCOBmX3wH1QRsSOZfXc0mwV6vEpSXet7vcnCleARFXQ1nMc\\\\\\\/yH9GYYVpuWoVraJB2dOdELW0C4iHi4fseQQpL4EsvdFMuizcCHavyoFFGOZ2cixgyWJ44rN4+6Qnv5xDaZP\\\\\\\/2uEkwx3hMBhUpCwfO872Ex376OWz1RzcfdVEAhane7oyH0f510bDi61ZpqaxDu8ASFDnmb3vzRJzXVichQpiR0GMT36asrmZ9MzV9U0iLxE\\\\\\\/rTZyqUGaRqBdrXDW9c2kzZPXiUR6fzwDux6sK56u20w5mNG7qqqFmFnuWbws49ZswTHfChwtPXtt4wOS36HWMgTCMy+iawbNROnlwr0cbfNZWgh6L3oaWJ8d+EtCR3pPs\\\\\\\/ytaUBJt3weGFe1i5ALnkoM6Z0RCcvqIOTqaAhbhyix3hdaG0pLk3sT7nRBhRnwYGMz5tPuWHfT3hjL5wZjVCmQGjFLdKlwjdjq0HIUV6O+nAdaVdr5MRo14d1y5VX5tAafLt274ZC46qzLuJ9iQqbJ7\\\\\\\/jOvHun9PZZFSx3ooMSLe\\\\\\\/WkpYcqwsNMGN3nc7kq51IagSVHsMFsWXLBSnmaz92NtiygGDdGL9oksS7UrxGWLjemd\\\\\\\/me9SQTf0+UqKZDtjb+TK9klbcJpfeVlGgPQ4tkH\\\\\\\/KLiAZD8BtCVJuwoH7wCqt0TETSbcj9tziuzz4j0cbHCxXajMjFKDtx+e87h0UGa9FAevdm7M9Shxp0u\\\\\\\/QQH0dXZ8+pydpxIrORnjsCttXPog1i+3OkujZ3ObjuesbC1zNZl0\\\\\\\/NCQEaDluOy5edexJNI65e44PImAeSqHvEQK0kakeQcfFt1u\\\\\\\/j72dH2I5FgKS3t95SvMEvrFBTBTLVjychRvZDDOPqvBhRbxb69pFtzczgDMhsMLUQ4N1JSmvR+esY\\\\\\\/KK4DofdNKWUrRMlr3kNi8\\\\\\\/erCGetASu9wJyXDHFcJyM0q+W8YzVPFV2szmkiTm5y5bFkg5mg7rtv121x1GySOJKF6il6O+Jc3v7Rzp4sO+6gYHVOicty04z1KBvMjEs4DwuNPBGNPKwqP9WhGkuXjUnezKGWr2RsDjZrnepdPzRL3eoI22vu7jPy91QagrhvTnN+sWb3hbOLtm1d0hcdYW8gs6C\\\\\\\/lVfbxv\\\\\\\/ljXr3JZLhIYCkZOKi2EoS8T2xbL2OluGkO4Y8u3QUL1cFnOMrZFZyW+QMNCGkjUyR7DC7hc3K9ldVyrc+acA2mhrJDIuQ6UOzkS+7AJQ2fsONaEMu7dWmIiUQnNozoNOlAytWS9knqGORjb+CJZHxvzvn4T32Zj7hDljIirG4vqVuilJWU99QhqsZeTc0Bx7eklhXsJS0BrptPpEv4Vf4PSc3LSZ9eoscj0E7ar8B7kjtAVuIi1tQoZpYuTcdfbfBa1kDXN0tVkx4clH25oIWq\\\\\\\/wcLxm9mDXdLjunh2kIkxG1VUQ4mR7kbNp+GGr3H\\\\\\\/gTlhHquyU5qKFWKvh2YpzCHgFi+793HHV1I+wYQ6dEs9n8Li2UlGvC7uzHhzeFaTWRItdBPw6iBB1yWwHwau8d1FrfTFvfDmg6CwhPMsr8LnRXGKVMV8nbe6Q5v0pZxcPNf8ZYJxlyGNZKXHmdE1kn0ieNS+Lap\\\\\\\/YwHtbalPx32\\\\\\\/yrNn70DqlF47aH+tjQO\\\\\\\/gJ3eD4yWI1F6ImQmqkJ1w7XbtFA1DUIrq+0yBetWClOOGMCV\\\\\\\/3KCmTg8KlttJrjAGnsQKR\\\\\\\/bkcFbhLDDPEZrlKjKHisuJF6oBEXITQJx4TfJFUCZTHsoCIET8QeGd2w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cfplF47DjmeI7v9WuYq7ww==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cfc025da1f32f926fa544b6ebc8adcc819a273780f1c3856159668dcb55aa4b4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488752 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/1079" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CEMpEerLwBgkacpOuk1Y1zevghq8exczs7yPVQCzPJY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/1079" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FIRT_lLKALwz7nDRrN2twf4gxv1nEX4Z5Je01R8dkM0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/3/1067" title="Pleiotropic Functions of the Chromodomain-Containing Protein Hat-trick During Oogenesis in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1095" title="Identification of Two Distinct Classes of the Human INO80 Complex Genome-Wide" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488752" id="node488752" data-pisa="ggg;8/3/1079" data-pisa-master="ggg;g3.117.300301" data-apath="/ggg/8/3/1079.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Addo-Quaye</span>, <span class="highwire-citation-author" data-delta="1">Mitch Tuinstra</span>, <span class="highwire-citation-author" data-delta="2">Nicola Carraro</span>, <span class="highwire-citation-author" data-delta="3">Clifford Weil</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1079-1094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300301 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488752" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles Addo-Quaye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BAddo-Quaye%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addo-Quaye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BAddo-Quaye%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Addo-Quaye</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mitch Tuinstra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitch%2BTuinstra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tuinstra%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMitch%2BTuinstra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicola Carraro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicola%2BCarraro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carraro%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicola%2BCarraro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Clifford Weil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clifford%2BWeil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weil%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClifford%2BWeil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/3/1079" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488752" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488752/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/1079.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488752" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488752/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/1079.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488752" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488752/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/1079.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488752" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488752/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/3/1079.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488752" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488752/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The accurate detection of induced mutations is critical for both forward and reverse genetics studies. Experimental chemical mutagenesis induces relatively few single base changes per individual. In a complex eukaryotic genome, false positive detection of mutations can occur at or above this mutagenesis rate. We demonstrate here, using a population of ethyl methanesulfonate (EMS)-treated <em>Sorghum bicolor</em> BTx623 individuals, that using replication to detect false positive-induced variants in next-generation sequencing (NGS) data permits higher throughput variant detection with greater accuracy. We used a lower sequence coverage depth (average of 7×) from 586 independently mutagenized individuals and detected 5,399,493 homozygous single nucleotide polymorphisms (SNPs). Of these, 76% originated from only 57,872 genomic positions prone to false positive variant calling. These positions are characterized by high copy number paralogs where the error-prone SNP positions are at copies containing a variant at the SNP position. The ability of short stretches of homology to generate these error-prone positions suggests that incompletely assembled or poorly mapped repeated sequences are one driver of these error-prone positions. Removal of these false positives left 1,275,872 homozygous and 477,531 heterozygous EMS-induced SNPs, which, congruent with the mutagenic mechanism of EMS, were &gt;98% G:C to A:T transitions. Through this analysis, we generated a collection of sequence indexed mutants of sorghum. This collection contains 4035 high-impact homozygous mutations in 3637 genes and 56,514 homozygous missense mutations in 23,227 genes. Each line contains, on average, 2177 annotated homozygous SNPs per genome, including seven likely gene knockouts and 96 missense mutations. The number of mutations in a transcript was linearly correlated with the transcript length and also the G+C count, but not with the GC/AT ratio. Analysis of the detected mutagenized positions identified CG-rich patches, and flanking sequences strongly influenced EMS-induced mutation rates. This method for detecting false positive-induced mutations is generally applicable to any organism, is independent of the choice of <em>in silico</em> variant-calling algorithm, and is most valuable when the true mutation rate is likely to be low, such as in laboratory-induced mutations or somatic mutation detection in medicine.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/accuracy" class="hw-term hw-article-keyword hw-article-keyword-accuracy" rel="nofollow">accuracy</a></li><li class="kwd"><a href="/keyword/ems" class="hw-term hw-article-keyword hw-article-keyword-ems" rel="nofollow">EMS</a></li><li class="kwd"><a href="/keyword/mutagenesis" class="hw-term hw-article-keyword hw-article-keyword-mutagenesis" rel="nofollow">mutagenesis</a></li><li class="kwd"><a href="/keyword/mutants" class="hw-term hw-article-keyword hw-article-keyword-mutants" rel="nofollow">mutants</a></li><li class="kwd"><a href="/keyword/mutations" class="hw-term hw-article-keyword hw-article-keyword-mutations" rel="nofollow">mutations</a></li><li class="kwd"><a href="/keyword/polymorphisms" class="hw-term hw-article-keyword hw-article-keyword-polymorphisms" rel="nofollow">polymorphisms</a></li><li class="kwd"><a href="/keyword/rare-variants" class="hw-term hw-article-keyword hw-article-keyword-rare-variants" rel="nofollow">rare variants</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/sorghum" class="hw-term hw-article-keyword hw-article-keyword-sorghum" rel="nofollow">sorghum</a></li></ul><p id="p-4">Forward and reverse genetics are ubiquitous methods for gene function discovery (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Page and Grossniklaus 2002</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alonso and Ecker 2006</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nordborg and Weigel 2008</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Schneeberger and Weigel 2011</a>). The principal step in these methods is the induction of mutations in the germline of an organism using various categories of high energy (such as UV light, X-rays, γ rays, and fast neutrons) or chemical mutagens (such as EMS, MMS, and ENU). The objectives are the production of genetic diversity in a population and the identification of phenotypic variants (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Waugh <em>et al.</em> 2006</a>). The chemical EMS is extensively used for mutagenesis (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Sega 1984</a>), and predominantly induces G:C to A:T transitions in the genome of a treated organism (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Westergaard 1957</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Loveless 1958</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">1969</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Krieg 1963</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Prakash and Sherman 1973</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Coulondre and Miller 1977</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Singer <em>et al.</em> 1978</a>). The accurate detection of these induced mutations is critical to establish evidence of gene function.</p><p id="p-5">The ascendancy of cost-effective DNA sequencing methods has enhanced our capacity to detect induced polymorphisms (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Metzker 2010</a>), and several sequencing-based methods have been proposed for the detection of induced polymorphisms in plant genetic studies (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Nordborg and Weigel 2008</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rowan <em>et al.</em> 2011</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Schneeberger and Weigel 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nordström <em>et al.</em> 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Schlötterer <em>et al.</em> 2014</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Schneeberger 2014</a>). Despite successes and widespread use of these methods, their accuracy is impacted negatively by the inherently higher error rates of NGS technologies (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nakamura <em>et al.</em> 2011</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nielsen <em>et al.</em> 2011</a>), disparities in variant-calling methods (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">O’Rawe <em>et al.</em> 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheng <em>et al.</em> 2014</a>), qualities of <em>in silico</em> reconstructed reference genome sequences (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cheung <em>et al.</em> 2003</a>), and the genome complexity of target organisms (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Estivill <em>et al.</em> 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fredman <em>et al.</em> 2004</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Robasky <em>et al.</em> 2014</a>). Improving the accuracy of methods for detecting induced polymorphisms in a cost-effective manner would be valuable for gene function discovery studies outside of genetic model species, which are often characterized by small genome sizes.</p><p id="p-6">Sorghum is a fully-sequenced panicoid grass, an important source of food and feed, and a potential biomass for biofuel production (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Paterson 2008</a>). It is well adapted to hot, semiarid conditions, nutrient-poor soils, and has particularly high drought tolerance relative to other cultivated cereals (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Paterson <em>et al.</em> 2009</a>). At 0.8 Gb, the sorghum genome is more compact than maize (2.3 Gb) and sugarcane (10 Gb), and has not undergone a recent polyploidy event (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Arumuganathan and Earle 1991</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Peterson <em>et al.</em> 2002</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Swigoňová <em>et al.</em> 2004</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Price <em>et al.</em> 2005</a>). Thus, it offers a potentially simpler system in which to discover mechanisms of drought tolerance, plant nutrition, weed biology, the evolution of C4 photosynthesis, and undertake biofuel research. Recent forward genetics studies using EMS mutagenesis have provided key insights into the function of genes involved in insect–herbivory defense, cell wall and lignin biosynthesis, protein digestibility, leaf vein density, and plant height (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pedersen <em>et al.</em> 2005</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Peters <em>et al.</em> 2009</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Xin <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blomstedt <em>et al.</em> 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Koegel <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Krothapalli <em>et al.</em> 2013</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Petti <em>et al.</em> 2013</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">2015</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wu <em>et al.</em> 2013</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sattler <em>et al.</em> 2014</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rizal <em>et al.</em> 2015</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Scully <em>et al.</em> 2015</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Addo-Quaye <em>et al.</em> 2017</a>). The recent characterization of mutations in pooled samples derived from a population of 256 EMS-mutagenized sorghum BTx623 lines is a major step toward improving resources for sorghum genetics (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jiao <em>et al.</em> 2016</a>). The availability of such large-scale collections of high-quality and well-annotated mutations will play a pivotal role in addressing key questions in plant science research (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Grierson <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bevan <em>et al.</em> 2017</a>).</p><p id="p-7">We have created an additional EMS-mutagenized sorghum population of ∼12,000 lines, and present here the large-scale detection and analysis of SNPs and insertions/deletions (indels) derived from low-coverage NGS of 586 of these individuals. We treated these individuals as a population of replicates of sequencing error to remove false positive SNP calls. This cost-effective and accurate method effectively detects <em>bona fide</em> EMS-induced mutations and eliminates false positive calls. We characterized the predicted impact of these mutations on sorghum gene function and created a database of mutation descriptions, the corresponding altered genes, and the gene function descriptions of their homologs in <em>Arabidopsis</em> and maize. This method can be applied generally to genetic studies in most organisms, and is independent of the <em>in silico</em> variant-calling method utilized. We anticipate that this valuable genetic resource and analytical approach will be useful to geneticists, plant breeders, and the general plant science community.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mutagenesis and DNA preparation</h3><p id="p-8">Seeds of the reference sorghum BTx623 genotype grown at Purdue University were treated with EMS (Sigma-Aldrich), by imbibing the seeds overnight in distilled water at 20° with shaking (150 rpm) and then soaking in 1 L of 45 mM EMS in 10 mM KH<sub>2</sub>PO<sub>4</sub> (pH 7.0) at 20° for 6 hr shaking at 150 rpm. Seeds were then washed 10 times with 1 L of distilled water and allowed to air dry at room temperature prior to planting. M<sub>0</sub> seeds were planted in an isolation block at ∼20 seeds/m. Unmutagenized BTx623 rows were interspersed with mutagenized seeds to serve as pollen parents to improve seed set. Plants were allowed to open pollinate and one head was harvested from ∼12,000 mutagenized plants, dried, and threshed individually to produce M<sub>1</sub> seed. A total of 20 M<sub>1</sub> seeds were then planted for 7304 lines. The 10th plant in each plot was self-fertilized to produce M<sub>2</sub> seed. A total of 40 M<sub>2</sub> seeds from 4800 lines were planted to score for segregating phenotypes. M<sub>2</sub> plants in each plot were self-fertilized to produce M<sub>3</sub> seed. M<sub>3</sub> seeds for each line were planted in the greenhouse. For each line, adult leaf tissue was harvested from a single plant for DNA preparation and stored at −20°; the same plant was then self-fertilized by controlled pollination to produce M<sub>4</sub> seed. All DNA was prepared using a CTAB extraction as described by Xin and Chen (2012), except that incubations were done at 55° rather than 60°. The M<sub>4</sub> seeds were also reserved for stock center deposit, phenotypic assessment, and future research.</p></div><div id="sec-3" class="subsection"><h3>Sequencing protocols</h3><p id="p-9">DNA was processed using the Illumina TruSeq DNA PCR-Free LT Library Preparation protocol, to make single indexed, 550 bp insert libraries. These were titred individually using an Applied Biosystems Step One kit with Kapa Biosystems Illumina Library Quantification Kit reagents. After flowcell clustering using a TruSeq PE Cluster Kitv3-cBot-HS kit, sequence data were generated as paired, 101 base reads by an Illumina HiSequation 2500 using SBS v3-HS reagents. “Stage 1” pools of 24 samples with compatible indices were created and a single lane of each pool was sequenced. Based on demultiplexing results from the initial lane of sequence, the library titres were refined and a new “stage 2” pool created to correct for excesses/deficiencies in numbers of reads per library from the first lane. Two lanes of the stage 2 pool sequence were then run. Reads from all three runs were merged for each library.</p></div><div id="sec-4" class="subsection"><h3>NGS read mapping</h3><p id="p-10">Illumina HiSeq paired-end sequencing reads were mapped to version 2.1 of the sorghum genome reference sequences (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Paterson <em>et al.</em> 2009</a>) downloaded from the Phytozome (version 9.1) web portal (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Goodstein <em>et al.</em> 2012</a>). Read alignment was performed using the BWA program for short reads alignment (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li and Durbin 2009</a>; version 0.6.2) and the variant calls were computed using the SAMtools software suite (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2009</a>; version 0.1.18). Genome coverage was estimated from the alignments in the BAM files using the <em>genomeCoverageBed</em> command in the BEDTools genome arithmetic software suite (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Quinlan and Hall 2010</a>; version 2.17.0). Insert sizes for the paired-end reads were estimated using the <em>CollectInsertSizeMetrics</em> command in the Picard software package (<a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>; version 1.80), with parameter option “VALIDATION_STRINGENCY=LENIENT.”</p></div><div id="sec-5" class="subsection"><h3>Detection of genomic positions with nonreference alignments</h3><p id="p-11">Whole-genome variant calling started with the detection of positions with statistical support from nonreference sequence in each sequenced sorghum individual. We used the SAMtools <em>mpileup</em> command to compute mapping information for all genomic positions in every BAM file. We selected the following command line parameters to ensure that only DNA bases in sequenced reads with base call qualities of ≥20 were used in variant detection: “-Q 20 -P Illumina -uf.” The <em>mpileup</em> output resulted in a Binary Call Format (BCF) file, which was redirected to the BCFtools <em>view</em> program. To produce a subset containing only the possible variant positions, we used the BCFtools <em>view</em> command with parameter options: “-vcg” to output a variant call (VCF) file limited to likely variants only. This file contains <em>bona fide</em> SNP and indel positions, as well as false positive variant calls that satisfied these low-stringency criteria, which accepted any position at which any read returned a nonreference alignment to be a variant position.</p></div><div id="sec-6" class="subsection"><h3>SNP filtering and refinement</h3><p id="p-12">The output of the variant-calling stage was refined in multiple steps. We initially used the <em>varFilter</em> command (with nondefault parameter option: “-D 100”) in the SAMtools <em>vcfutils.pl</em> Perl utility script to remove variants with a low-quality score (&lt;10), low read coverage (read depth &lt; 2), and that were possibly derived from repetitive origins (read depth &gt; 100). We refer to this initially filtered SNPs as the “Repeat-Filtered” SNPs. We then used the SnpSift component (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cingolani <em>et al.</em> 2012a</a>) in the SnpEff java package (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cingolani <em>et al.</em> 2012b</a>; version 3.1) for further SNP variant filtering. Using the SnpSift <em>filter</em> command with query string “(QUAL&gt;=20) &amp; (isHom(GEN[0]) &amp; (isVariant(GEN[0]),” we retained homozygous SNPs with variant call quality of ≥ 20. This set of filtered SNPs was designate as the homozygous “Q20” SNPs. Previous studies have shown that strand-biased SNPs tend to be a source of false positive SNP detections (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">DePristo <em>et al.</em> 2011</a>). The remaining SNPs were further filtered using the SnpSift <em>filter</em> command with query string “(((DP4[0]=0) &amp; (DP4[1]=0)) &amp; ((DP4[2]&gt;0)&amp; (DP4[3]&gt; 0)))” to retain SNPs with at least one nonreference read mapping to both the forward and reverse strands of the reference genome, which had no reference reads at the SNP position. This set is referred to as the “Standard Filtered” SNPs. We then used the SnpSift <em>filter</em> command with the query string “((REF=“G” &amp; ALT=“A”) | (REF=“C” &amp; ALT=“T”))” to catalog the subset of the standard filtered SNPs that were either G to A, or C to T, substitutions.</p><p id="p-13">To detect heterozygous SNPs, we modified the above procedure for detecting homozygous SNPs. We replaced the SnpSift <em>filter</em> query string “(QUAL&gt;=20) &amp; (isHom(GEN[0])) &amp; (isVariant(GEN[0]))” with “(QUAL&gt;=20) &amp; (isHet(GEN[0])) &amp; (isVariant(GEN[0]))” to select the heterozygous Q20 SNPs. The low read depth in our genomic sequencing required a relaxed stringency for the presence of mapped reference reads at the heterozygous SNP genomic position to retain likely heterozygous positions. We accomplished this by replacing the SnpSift <em>filter</em> query string “(((DP4[0]=0) &amp; (DP4[1]=0)) &amp; ((DP4[2]&gt;0) &amp; (DP4[3]&gt; 0)))” with “((DP4[2]&gt;0) &amp; (DP4[3]&gt; 0)).” This checks only for the mutant allele by determining the presence of at least a single nonreference read on both the forward and reverse strands.</p></div><div id="sec-7" class="subsection"><h3>Indel detection and refinement</h3><p id="p-14">Indels in the VCFs were cataloged using the SnpSift <em>filter</em> command with query string “(exists INDEL).” Homozygous and heterozygous indels with variant quality of ≥20 were retained using the <em>filter</em> command with query strings “(QUAL&gt;=20) &amp; (isHom(GEN[0])) &amp; (isVariant(GEN[0]))” and “(QUAL&gt;=20) &amp; (isHet(GEN[0])) &amp; (isVariant(GEN[0])),” respectively. To eliminate homozygous and heterozygous indels from positions with strand-biased alignment, we required variant calls on both strands using the <em>filter</em> query strings “(((DP4[0]=0) &amp; (DP4[1]=0)) &amp; ((DP4[2]&gt;0) &amp; (DP4[3]&gt; 0)))” and “((DP4[2]&gt;0) &amp; (DP4[3]&gt; 0)),” respectively.</p></div><div id="sec-8" class="subsection"><h3>Detection of probable error-prone genomic positions</h3><p id="p-15">Following the set of variant filters described above, the standard filtered SNPs detected in the preliminary set of 570 sequenced individuals were concatenated into a single list. These alleles were sorted by chromosome and genomic position. The number of appearances of each genomic position among the called SNPs was tallied. To the extent that EMS mutagenesis is stochastic and even in its ability to affect mutation, the same SNPs should rarely occur in more than one individual in an organism with a sufficiently large genome size. Thus, positions with a tally of two or more SNP call instances were recorded as probable error-prone positions. We refer to this set of probable error-producing positions in the sorghum reference genome (version 2.1) as the Error-Prone SNP Positions. This list of identified positions was then used to subtract the error-prone portion of the genome from the standard filtered SNP calls for each of the final 586 sequenced individuals, and included the preliminary 570 sequenced individuals. The remaining, unrepeated or sample-specific SNPs were unique to each line. The subtracted repeat SNPs and the unrepeated SNPs we refer to as the Replicate and Nonreplicate SNPs, respectively. We applied the same filtering technique to the indel variant calls using the indel event starting nucleotide position of each indel as the sort item. Indels starting at positions present in more than one line were removed from all indel lists. To detect possible false negative replicate SNPs with origins in mutational hotspots (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Krasileva <em>et al.</em> 2017</a>), we further grouped the replicate SNP positions by the count of the number of individuals containing the SNP position. For each group, we calculated the percentages for each substitution type and performed signal-to-noise analysis by using the G:C to A:T substitution percentage as the signal. Based on the signal-to-noise analysis, we then created a separate catalog of tentative false negative G:C to A:T SNPs. Lists of these positions, and the counts of observed SNPs and indels in the 586 lines, are provided as supplemental information.</p></div><div id="sec-9" class="subsection"><h3>Classification of SNP and indel effects on gene function</h3><p id="p-16">We used the SnpEff program (version 3.1) to predict and classify the effects of the detected SNPs and indels on sorghum gene function. To use SnpEff, we had to create a custom SnpEff database and entry for the sorghum genome reference assembly (version 2.1) by using the SnpEff <em>build</em> command with parameters “-gff3 -v.” Predicted effects of the detected SNPs and indels were computed by the SnpEff <em>eff</em> command with parameter option “-c.” We retained only the subset of variants predicted to have either medium or high impact on an encoded protein sequence. Finally, the genome-wide distribution of indel lengths was estimated using the vcftools program (version 0.1.14) with parameter option “–hist-indel-len” (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Danecek <em>et al.</em> 2011</a>).</p></div><div id="sec-10" class="subsection"><h3>Gene function annotation</h3><p id="p-17">We used the predicted protein sequences of <em>Sorghum bicolor</em> BTx623 (version 2.1), maize B73 (annotation version 5b.60), and <em>Arabidopsis thaliana</em> (TAIR release version 10) from Phytozome (version 9.1) to annotate the SNP-encoding genes. We created protein BLAST search databases for the maize and <em>Arabidopsis</em> sequences using the <em>makeblastdb</em> command in the BLAST suite of programs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Camacho <em>et al.</em> 2009</a>; version 2.2.30+), with the nondefault parameters: “-input_type fasta –dbtype prot.” We aligned each sorghum protein sequence to the maize and <em>Arabidopsis</em> protein sequences using the <em>blastp</em> (protein BLAST) command with nondefault parameters: “-evalue 1E-05 -num_threads 16 -max_target_seqs 5 -outfmt 6 -seg yes.” The gene function description files for sorghum (version 2.1), maize B73 (version 5b.60), and <em>Arabidopsis</em> (TAIR version 10) were obtained from Phytozome, and annotations were appended as additional columns to the VCFs for the affected sorghum locus, along with the gene identifiers and annotations for the best maize hit and best <em>Arabidopsis</em> BLAST hit. The top hits for maize and <em>Arabidopsis</em> were appended only when BLAST <em>e</em>-values were less than a threshold value of E−30.</p></div><div id="sec-11" class="subsection"><h3>Search for sequence motifs in flanking sequences of SNPs</h3><p id="p-18">We converted the formatted VCFs containing the nonreplicate and likely EMS-induced homozygous SNPs into the BED format by replacing the VCF genomic position with the respective BED start and end positions of a 21 bp window. The set of flanking sequence contexts of all the SNP positions was saved as both FASTA and TAB delimited formats using the BEDTools <em>getfasta</em> program with parameters “-bed -fo” and “-bed -tab,” respectively. A subset of 10,000 sequences was randomly selected with uniform distribution from the FASTA-formatted file containing the 21 nucleotide (nt) sequences (SNP and each flank). Using the sorghum reference genome sequences (version 2.1), we generated a third-order background Markov model using the <em>fasta-get-markov</em> utility program in the MEME ungapped motif-finding software suite (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bailey <em>et al.</em> 2009</a>; version 4.9.1) with nondefault parameter: “-m 3.” To detect an enrichment of ungapped motifs in the sequence context of the 10,000 selected SNPs, we used the parallel version of the <em>meme</em> program for ungapped motif discovery with parameters “-dna -mod zoops -revcomp -bfile BKGRND -oc directory -maxsize 300000 -p 16 -nmotifs 50 -minw 5 -evt 0.01.” Using these parameters, we limited the search results to the top 50 ungapped motifs with minimum length of 5, an <em>e</em>-value threshold of 0.01, and the profiling evaluated both the original sequences and their reverse complements.</p></div><div id="sec-12" class="subsection"><h3>Analysis of the DNA sequence context of the entire set of likely EMS-induced mutations</h3><p id="p-19">The FASTA-formatted file containing the extracted DNA sequence contexts of all the likely EMS-induced SNPs was partitioned into two files, based on whether the SNP reference base was a G or C nucleotide. For each of the 20 positions surrounding the SNP, we tabulated the number of occurrences for all four DNA nucleotide types in each sequence. The tabulation was performed separately for each of the above two files. Next, we repeated the process using the same sample sizes but with two files containing 21 nt sequence contexts of randomly selected G and C reference base positions in the sorghum genome. For each DNA nucleotide type, and at each of the 20 positions, we computed the percentage changes (deviations) in the number of occurrences from the random data set to the EMS-induced SNP data set. We analyzed the tabulations of the pairs of data sets for the G and C reference base cases separately.</p></div><div id="sec-13" class="subsection"><h3>Detection of paralogs of the sequence contexts of error-prone SNP positions</h3><p id="p-20">Due to the complexity of the sorghum genome, we determined that ∼50 flanking bases are sufficient to distinguish the sequence context of a specified genomic position from another. We hence extracted the 51 nt sequence context for the detected error-prone SNP positions. These were aligned to the sorghum reference genome sequences in a BLAST-indexed database using the <em>blastn</em> program in the BLAST software suite (version 2.2.30+) with parameter options: “-evalue 1.0E-10 -num_threads 16 -outfmt 0 -ungapped -dust no -word_size 11.” The BLAST <em>e</em>-value threshold was selected based on initial preliminary runs in order to exclude a high percentage of spurious matches in the alignment results. The BLAST ungapped alignment output results were initially parsed to retain only a subset of alignment results, which consists of global alignments. The stringency of the filtering was subsequently increased to only retain global alignments containing a single substitution, coinciding with the 26th position (error-prone SNP position). The alignments and the single nucleotide substitution base were then cataloged. For each of the error-prone SNP positions, we compared the DNA base in the single nucleotide substitution in the BLAST alignments with the false positive SNPs calls at the error-prone reference position, and kept track of matching and mismatching bases. We repeated this procedure for two other data sets, with each consisting separately of 58,000 randomly selected genomic positions, and likely EMS-induced homozygous SNPs. We tabulated and compared the BLAST results for the three data sets.</p></div><div id="sec-14" class="subsection"><h3>Annotation of likely adverse effect of missense mutations using SIFT</h3><p id="p-21">We followed the recommended steps for the setup and application of the SIFT program to predict the effect of missense mutations on protein function (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kumar <em>et al.</em> 2009</a>; version 5.2.2). We downloaded the UniRef90 protein database (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">UniProt Consortium 2014</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Suzek <em>et al.</em> 2007</a>; release 2015_03) from The European Bioinformatics Institute website (<a href="http://www.ebi.ac.uk/uniprot">http://www.ebi.ac.uk/uniprot</a>; retrieved in March, 2015). The FASTA-formatted sequences were then formatted into a protein BLAST database using the <em>formatdb</em> command in the BLAST package with parameters: “-i uniref90.fasta -p T –n uniref90.” The SnpEff annotated VCF-formatted files were parsed to extract the subset of likely EMS-induced SNPs, which were predicted to cause missense mutation. The output file consisted of the amino acid changes, their positions in the protein sequences, and the genes that encode them. Using this table, we generated an input file for each sorghum gene transcript with at least one missense EMS-induced SNP. The file for each gene transcript contained all the amino acid changes detected for that gene in all 586 sequenced individuals. The amino acid sequence of each affected gene transcript was downloaded from Phytozome and stored in separate FASTA-formatted files. The individual sorghum protein sequences, and the UniRef90 BLAST database and amino acid changes files for each sorghum gene transcript, were used as input to the <em>SIFT_for_submitting_fasta_seq.csh</em> C-shell utility script, which was provided in the SIFT program to predict the effect of missense amino acid changes on protein function. For SIFT prediction, we specified a median conservation value of 2.75 and a SIFT score threshold of 0.05. The <em>seqs_chosen_via_median_info.csh</em> script in the SIFT package was modified to increase the CPU time limit from 30 to 60 min (“limit cputime 60m”) and the <em>psiblast</em> alignment program was executed in the multithreaded mode (“-num_threads 16”). A mutation was predicted to be deleterious if the computed SIFT score was lower than the threshold. We parsed the output results of the SIFT program to retain amino acid changes with predicted “DELETERIOUS” and “TOLERATED” designations. These designations were appended to the corresponding amino change columns in the EMS-induced SNPs annotation file. The main scripts for the computational pipeline implemented for SNP detection and annotation are included as supplemental information (see Supplemental Material, File S25).</p></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-22">File S1, File S2, File S3, File S4, File S5, File S6, File S7, File S8, File S9, File S10, File S11, File S12, File S13, File S14, File S15, File S16, File S17, File S18, File S19, File S20, File S21, File S22, File S23, File S24, and File S25 are available from the Dryad Digital Repository: <a href="https://doi.org/10.5061/dryad.t80gj">https://doi.org/10.5061/dryad.t80gj</a>. The Illumina NGS data sets generated for this project have been deposited in the NCBI SRA (Short Reads Archive), under the Project Title: “Purdue University: Functional Gene Function Discovery Platform for Sorghum,” with the SRA Study Accession Number SRP065118. The BioProject ID# is PRJNA297450 and the BioSample ID#s range from SAMN04128840 to SAMN04129425. Seed samples for the EMS-mutagenized sorghum population used in this study are available on the Germplasm Resources Information Network database.</p></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><div id="sec-17" class="subsection"><h3>NGS sequencing and read mapping</h3><p id="p-23">A total of 29,972,216,082 Illumina NGS paired-end reads were generated from 586 EMS-mutagenized sorghum BTx623 individuals (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Of these reads, 29,651,940,529 (99%) aligned to the sorghum BTx623 reference genome (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Paterson <em>et al.</em> 2009</a>). From the alignment, the estimated median paired-end insert size was 527 bp. The average genome coverage of the alignments was 7×. These data were used to identify SNPs that were likely to be the result of the EMS treatment.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488910/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488910/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary sequencing and alignment statistics for the 586 EMS-mutagenized sorghum BTx623 individuals</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Standard SNP detection and filtering</h3><p id="p-24">To detect the high-quality EMS-induced SNPs from the nearly 30 billion sequencing reads, we implemented a highly decoupled SNP prediction and annotation pipeline (See <em>Materials and Methods</em>). Initially, we detected 31,178,985 putative SNPs in the 586 EMS-mutagenized individuals. The default criteria for SNP calling resulted in an unreasonably high number of positives: a polymorphism for every ∼14 kb of genome with 53,206 SNP sites per individual (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The removal of SNPs characterized by low quality, repetitive origin, low read coverage, or strand bias, retained a total of 5,399,493 high-quality standard filtered homozygous SNPs and an average of 9214 per individual (File S1 and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Similarly, we retained 7,444,547 standard filtered heterozygous SNPs and an average of 12,704 per individual (File S1 and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488852/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488852/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary statistics for the detection of single nucleotide polymorphisms in the 586 EMS-mutagenized sorghum BTx623 individuals</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Prediction of error-prone SNP positions</h3><p id="p-26">Because pollen and seed contamination might result in the mistaken resequencing of the same line twice, we compared the SNP calls between lines. This analysis identified 16 lines with &gt;68% overlap between the called SNPs and patterns of SNP coincidence that were consistent with pollen or seed contamination. For additional analysis, we therefore focused on the 570 lines without siblings. For these 570 lines, the SNP detection and standard filtering retained a total of 5,244,176 standard filtered homozygous SNPs produced at 1,290,635 distinct genomic positions. Surprisingly, 4,011,413 of the 5,244,176 SNPs called were predicted repeatedly from only 57,872 distinct genomic positions. This means that roughly 4% of the distinct locations produced &gt;76% of the SNP calls in the sequence analysis of these 570 individuals (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). For example, there were a total of 1085 positions in the sorghum genome, where the same SNP was detected in between 451 and 500 EMS-treated sorghum individuals. This group of SNPs accounted for 515,902 redundant SNP detections (see Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). The vast majority of these positions are not random poor base calls. For example, for nearly all (57,802 positions) of these replicate SNPs, the predicted change was the same for all lines (File S2). For example, in 567 lines, a C to G substitution was predicted at genomic position 37,580,764 on chromosome 5. This pointed to some structural basis for the repeated observations. Similarly, of the 7,238,377 standard filtered heterozygous SNPs, 6,775,202 SNPs were derived from replicate SNPs at only 71,294 positions. Thus, roughly 13% of the 534,469 distinct genomic positions produced replicate heterozygous SNPs accounting for 94% of the heterozygous SNP predictions in the 570 sorghum individuals (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of likely bona fide EMS-induced variants by genome origin characterization of the standard filtered DNA variants. In contrast to the nonreplicate or sample-specific variants, the replicate variants were predicted recurrently from relatively few, distinct genomic positions. Over 76% of the standard filtered homozygous SNPs were predicted repeatedly from only 4% of the total distinct homozygous SNP positions in 570 sorghum individuals. For the standard filtered heterozygous SNPs, 94% were predicted repeatedly from only 13% of the distinct heterozygous SNP positions. Similarly, &gt;99% of the homozygous and 98% of the heterozygous standard filtered indels were predicted repeatedly from 56 to 50% of their respective sets of indel positions. EMS, ethyl methanesulfonate; indel, insertion/deletion; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45765907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Detection of likely &lt;em&gt;bona fide&lt;/em&gt; EMS-induced variants by genome origin characterization of the standard filtered DNA variants. In contrast to the nonreplicate or sample-specific variants, the replicate variants were predicted recurrently from relatively few, distinct genomic positions. Over 76% of the standard filtered homozygous SNPs were predicted repeatedly from only 4% of the total distinct homozygous SNP positions in 570 sorghum individuals. For the standard filtered heterozygous SNPs, 94% were predicted repeatedly from only 13% of the distinct heterozygous SNP positions. Similarly, &gt;99% of the homozygous and 98% of the heterozygous standard filtered indels were predicted repeatedly from 56 to 50% of their respective sets of indel positions. EMS, ethyl methanesulfonate; indel, insertion/deletion; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/1079/F1.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/3/1079/F1.medium.gif" width="440" height="325"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488807" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Detection of likely <em>bona fide</em> EMS-induced variants by genome origin characterization of the standard filtered DNA variants. In contrast to the nonreplicate or sample-specific variants, the replicate variants were predicted recurrently from relatively few, distinct genomic positions. Over 76% of the standard filtered homozygous SNPs were predicted repeatedly from only 4% of the total distinct homozygous SNP positions in 570 sorghum individuals. For the standard filtered heterozygous SNPs, 94% were predicted repeatedly from only 13% of the distinct heterozygous SNP positions. Similarly, &gt;99% of the homozygous and 98% of the heterozygous standard filtered indels were predicted repeatedly from 56 to 50% of their respective sets of indel positions. EMS, ethyl methanesulfonate; indel, insertion/deletion; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We observed that only 39% (2,050,166 SNPs) of the standard filtered homozygous SNP predictions were G:C to A:T substitutions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS1.jpg">Figure S1</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). Similarly, only 36% (2,588,504 SNPs) of the heterozygous SNPs were G:C to A:T substitutions. With &lt;40% of the retained, high-quality standard filtered SNPs showing signatures of EMS-derived mutagenesis, it is more likely that a significant proportion of the 5,244,176 homozygous and 7,238,377 heterozygous SNPs are false positives. Remarkably, removal of the replicate SNPs showed that, consistent with the experimentally determined effect of EMS mutagenesis, 98% (1,205,140) of the retained nonreplicate or sample-specific homozygous SNPs were G:C to A:T transitions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS2.jpg">Figure S2</a>). The second and third largest substitution classes were A:T to G:C transitions and A:T to T:A substitutions, accounting for 0.7% (9066) and 0.5% (6194) of the SNPs, respectively. Similarly, in the case of the heterozygous standard filtered SNPs, after removal of the replicate SNPs, the remaining nonreplicate SNPs were 94% (435,414) G:C to A:T transitions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS2.jpg">Figure S2</a>).</p><p id="p-29">The detected probable error-prone SNP positions in the 570 unique lines were used to mask likely false positives from the standard filtered SNPs called for each of the 586 lines. The additional 16 sorghum individuals with overlapping genomic positions were excluded from the masking set, so as not to remove true EMS-induced SNPs that were due to contaminated pedigrees. Cumulatively, 1,275,872 (24%) of the 5,399,493 standard filtered homozygous SNPs were nonreplicate SNPs in the 586 sorghum individuals (File S3 and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), and 98% (1,247,187) were G:C to A:T transitions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). Protein-coding gene loci comprised 246,062 (19%) of the likely homozygous EMS-induced SNPs and &gt;99% (243,775) were G:C to A:T mutations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Similarly, 477,531 nonreplicate heterozygous SNPs (6%) were retained in the 586 lines after removal of replicates in the 7,444,547 heterozygous standard filtered SNPs (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, File S4, and <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Hence for our M<sub>3</sub> population, we observed 27% of the final 1,753,403 nonreplicate SNPs to be heterozygous, and the ratio of homozygous to heterozygous SNPs for the population was 2.7:1. A total of 515 individuals (88%) were homozygous for ≥60% of their SNPs, and only 21 samples were ≤20% homozygous (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and File S5). Further analysis revealed a positive correlation (<em>R</em> = 0.51) between the final number of predicted heterozygous SNPs and the read mapping coverage for individuals in the mutagenized population. On the other hand, we found a negligible correlation (<em>R</em> = 0.07) between the number of homozygous SNPs and mapping coverage. Hence, the final number of heterozygous mutations we predicted may be overestimated, especially for the mutant individuals with lower mapping coverage and lower homozygous percentage. Overall, we predicted an average of 2177 ± 906 homozygous and 815 ± 622 heterozygous nonreplicate SNPs per individual, and observed ∼1 SNP per 243 kb (with one homozygous SNP per 334 kb).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The mutation spectrum and frequency distribution of the likely EMS-induced homozygous and heterozygous mutations in the whole genomes and protein-coding regions of the 586 resequenced sorghum BTx623 individuals. CDS, coding sequence; EMS, ethyl methanesulfonate; Hets, heterozygotes; Homs, homozygotes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45765907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The mutation spectrum and frequency distribution of the likely EMS-induced homozygous and heterozygous mutations in the whole genomes and protein-coding regions of the 586 resequenced sorghum BTx623 individuals. CDS, coding sequence; EMS, ethyl methanesulfonate; Hets, heterozygotes; Homs, homozygotes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/1079/F2.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/3/1079/F2.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488937" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">The mutation spectrum and frequency distribution of the likely EMS-induced homozygous and heterozygous mutations in the whole genomes and protein-coding regions of the 586 resequenced sorghum BTx623 individuals. CDS, coding sequence; EMS, ethyl methanesulfonate; Hets, heterozygotes; Homs, homozygotes.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The correlation between sequence mapping coverage and the level of SNPs homozygosity in the genomes of the 586 EMS-treated sorghum BTx623 individuals. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45765907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The correlation between sequence mapping coverage and the level of SNPs homozygosity in the genomes of the 586 EMS-treated sorghum BTx623 individuals. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/1079/F3.medium.gif" width="440" height="299"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/3/1079/F3.medium.gif" width="440" height="299"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488932" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">The correlation between sequence mapping coverage and the level of SNPs homozygosity in the genomes of the 586 EMS-treated sorghum BTx623 individuals. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Detection of likely false negative replicate SNP positions</h3><p id="p-32">The signal-to-noise analysis showed a precipitous drop in the G:C to A:T substitution percentages for replicate SNP positions detected in more than two individuals (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>), and this is further depicted by the point of inflection at <em>N</em> = 2 on the <em>x</em>-axis of the plot in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS3.jpg">Figure S3</a>. Over 61% (9291) of the homozygous and 53% (5709) of the heterozygous replicate SNP positions detected in exactly two individuals were G:C to A:T substitutions. Hence, we cataloged an extra 18,582 homozygous and 11,418 heterozygous G:C to A:T substitutions as tentative false negative EMS-induced mutations (File S6 and File S7).</p></div><div id="sec-21" class="subsection"><h3>Functional classification of SNPs and their impacts</h3><p id="p-33">The majority of the SNP calls were outside predicted protein-coding regions. Intergenic regions and introns contained 1,032,427 (81%) and 109,144 (9%) of the nonreplicate homozygous SNPs, respectively (File S8, File S9, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). The remaining 134,301 SNP calls were in protein-coding sequences, with 60,442 effecting changes in protein-coding capacity in 23,866 distinct sorghum genes. Homozygous mutations of high impact (loss of reading frame) were predicted for 4035 SNPs disrupting 3637 distinct genes. These mutations included 2880 nonsense alleles and 1043 mutations removing splice sites. A total of 56,407 nonreplicate homozygous SNPs were predicted to cause nonsynonymous amino acid substitutions in 23,198 distinct sorghum genes. In addition, we detected five lost stop codons associated with nonreplicate SNP changes. These five mutations were not G:C to A:T mutations, as no such mutations can induce a stop codon loss, which may explain the relatively small number relative to the 145 start codon losses. The stop codon losses included a single homozygous and four heterozygous mutations. Synonymous changes from one stop codon to another were associated with G:C to A:T mutations, and we detected 148 synonymous (silent) stop codon substitutions consisting of 114 homozygous and 34 heterozygous mutations. The silent stop-lost mutations included 92 TGA and 56 TAG codon changes to TAA codons (File S10 and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). Mutations causing loss of start codons included 111 homozygous and 34 heterozygous changes (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). Intergenic regions and introns contained 387,570 (81%) and 42,357 (9%) of the 477,531 nonreplicate heterozygous SNPs, respectively. The heterozygous SNPs also included 1813 high-impact and 19,800 nonsynonymous mutations (File S11, File S12, and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Taken together, counting mutations in exons, splice sites, and untranslated regions, we detected 188,371 alleles in 31,163 (94%) sorghum genes, including 139,600 homozygous alleles in 30,167 genes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Within protein-coding sequences, we also predicted 79,765 nonsynonymous and 46,234 synonymous mutations, as expected based on codon degeneracy. On average, we predicted seven high-impact (<em>e.g.</em>, nonsense mutation or loss of splice site) and 96 amino acid substitution alleles per individual. The average individual also encoded three high-impact and 34 moderate-impact nonreplicate heterozygous SNPs per individual. We estimated an average of 5.7 alleles per sorghum gene. In addition, the tentative false negative SNPs included 40 high-impact and 716 moderate-impact homozygous SNPs, and likewise, 14 high-impact and 326 moderate-impact heterozygous SNPs (File S13 and File S14).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488891/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488891/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Classification of the predicted EMS-induced SNPs and their effects in the 586 EMS-mutagenized sorghum BTx623 individuals</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of allele counts for all genes impacted by likely EMS-induced mutations in the resequenced 586 sorghum BTx623 individuals. Only EMS-induced mutations overlapping protein-coding exons, splice sites, and untranslated regions were considered in the allele counts. EMS, ethyl methanesulfonate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45765907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The distribution of allele counts for all genes impacted by likely EMS-induced mutations in the resequenced 586 sorghum BTx623 individuals. Only EMS-induced mutations overlapping protein-coding exons, splice sites, and untranslated regions were considered in the allele counts. EMS, ethyl methanesulfonate.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/1079/F4.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/3/1079/F4.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488859" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">The distribution of allele counts for all genes impacted by likely EMS-induced mutations in the resequenced 586 sorghum BTx623 individuals. Only EMS-induced mutations overlapping protein-coding exons, splice sites, and untranslated regions were considered in the allele counts. EMS, ethyl methanesulfonate.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Prediction of indel polymorphisms</h3><p id="p-36">We initially detected 3,312,791 indel polymorphisms in the 586 sorghum lines and the standard filtering retained 1,989,078 indels (File S15, File S16, and Tables S7 and S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). Less than 1% (17,760) of the remaining indels were retained after replicate indel filtering, indicating an unacceptably high rate of false positives (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The average length of the retained indels was 2.6 bp with a remarkably skewed distribution (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS4.jpg">Figure S4</a>). Among the nonreplicate indels, 77% were ≤ 2 bp in length and 97% of the indels were 10 bases or shorter. A total of 277 and 180 indels were predicted to have high and moderate impact on gene function, respectively. One deletion, in line SbEMS0456-2, disrupts the sorghum ortholog, Sobic.003G386900, of the <em>Monoculm1</em> gene from rice, LOC_Os05g42130. The phenotype of this line is reduced height and a complete absence of tillers. Twenty-two indel variations affected splice sites and 1094 occurred in untranslated regions of sorghum protein-coding genes. Another 14,341 and 1947 indels occurred in intergenic and intronic regions, respectively.</p></div><div id="sec-23" class="subsection"><h3>Predicting the impact of missense SNPs on protein function</h3><p id="p-37">Over 93% (68,379) of the 73,346 homozygous nonsynonymous EMS-induced SNPs were missense mutations. These affected the coding sequences of 27,943 transcripts encoded by 23,227 distinct sorghum genes. Using the SIFT program in conjunction with the 32,168,296 sequences in the high-quality UniRef90 protein database, we were able to predict the impact of 56,289 missense SNPs. Of these, 25,013 (44%), affecting 12,840 genes, returned a SIFT score &lt;0.05 and likely encode changes deleterious to protein function. The remaining 31,276 SNPs were more likely to be tolerated mutations in 14,559 genes (File S17). Similarly, 24,191 (92%) heterozygous missense mutations were found in 16,118 transcripts encoded by 13,340 (40%) distinct sorghum genes. SIFT analysis predicted the impact of 21,183 of these heterozygous mutations and 9659 (46%) of them were predicted to be deleterious changes. The remaining 11,524 heterozygous SNPs affecting 7417 genes were classified as likely tolerated mutations (File S18). The detailed summary statistics on the sequencing, mapping, SNPs prediction, filtering, annotation, and classification of the homozygous and heterozygous SNPs, as well as the indel polymorphisms, are included as supplemental information (see File S19). The detailed classification of the medium- or high-impact SNPs and indels, and annotations for the affected genes in all 586 sequenced lines, are included as supplemental information (see File S20, File S21, and File S22).</p></div><div id="sec-24" class="subsection"><h3>Analysis of EMS-induced mutations in gene transcripts</h3><p id="p-38">The sorghum genome annotation includes 39,441 transcripts derived from 33,032 distinct protein-coding genes. We detected SNPs in 89% of these genes (35,221 transcripts and 29,296 genes). Our analysis found no strong transcriptome-wide linear correlation between GC content and the number of SNPs (<em>r</em><sup>2</sup> = 0.00279; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Unsurprisingly, substituting GC content (dinucleotide percentage calculation) with G+C count, we found that the number of SNPs was linear correlated with G+C count (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>; <em>r</em><sup>2</sup> = 0.61069; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS5.jpg">Figure S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS6.jpg">Figure S6</a>). Transcripts containing SNPs had mean G and C counts of 361.1 and 340.4, respectively, while the mean count of Gs and Cs for transcripts with no SNPs were 133 and 128.2, respectively, indicating that, in the absence of saturation, genes without SNPs were enriched for genes that represented a smaller mutational target (File S23). Consistent with this, we found a transcriptome-wide linear correlation between the number of SNPs and the length of a transcript (<em>r</em><sup>2</sup> = 0.57184; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). While the mean length of SNP-containing transcripts was 1295, the mean length of SNP-lacking transcripts was only 480. Hence, in general, transcript length and/or the number of G (or C) nucleotides in a transcript impact the likelihood of an EMS-induced mutation. Further mutagenesis to saturate the sorghum genome may benefit from both a greater population size and a diversity of mutagens.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The examination of attributes that impact the number of predicted EMS-induced mutations in a gene transcript. (A) Scatterplot shows no direct linear correlation between mutation count and GC content. (B) Scatterplot depicting a linear correlation between mutation count and G+C count. (C) Linear correlation between mutation count and transcript sequence length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45765907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The examination of attributes that impact the number of predicted EMS-induced mutations in a gene transcript. (A) Scatterplot shows no direct linear correlation between mutation count and GC content. (B) Scatterplot depicting a linear correlation between mutation count and G+C count. (C) Linear correlation between mutation count and transcript sequence length.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-5.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-5.medium.gif" width="330" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488938" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The examination of attributes that impact the number of predicted EMS-induced mutations in a gene transcript. (A) Scatterplot shows no direct linear correlation between mutation count and GC content. (B) Scatterplot depicting a linear correlation between mutation count and G+C count. (C) Linear correlation between mutation count and transcript sequence length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45765907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The examination of attributes that impact the number of predicted EMS-induced mutations in a gene transcript. (A) Scatterplot shows no direct linear correlation between mutation count and GC content. (B) Scatterplot depicting a linear correlation between mutation count and G+C count. (C) Linear correlation between mutation count and transcript sequence length.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-6.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-6.medium.gif" width="440" height="319"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F5/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488938" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">The examination of attributes that impact the number of predicted EMS-induced mutations in a gene transcript. (A) Scatterplot shows no direct linear correlation between mutation count and GC content. (B) Scatterplot depicting a linear correlation between mutation count and G+C count. (C) Linear correlation between mutation count and transcript sequence length.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>Sequence context of EMS-induced SNPs</h3><p id="p-40">The MEME ungapped motif search program found no single fixed length uniform motif associated with the 10,000 randomly selected SNPs. However, we observed enrichment for motifs that were present in clusters of SNPs. The most significant motif (p-value of 5.7e−304) was a 16 nt long GC-rich sequence, which is present in a cluster of only 834 SNPs out of the 10,000 homozygous SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS7.jpg">Figure S7</a>). In the absence of uniform flanking motifs, we observed significant sequence changes in the single base immediately upstream (position −1), and two bases immediately downstream of the 608,485 mutated G-nucleotide positions, when compared to the context of random G-nucleotide positions (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). In the first base position immediately upstream of the mutated central G position, we observed a 28% increase (99,332 <em>vs.</em> 127,287) in the count of Cs, and a 17% decrease (177,307 <em>vs.</em> 146,321) in the count of A bases. At the first downstream position (+1) we observed a 32% increase (141,485 <em>vs.</em> 186,396) in the count of Cs and a 26% decrease (144,587 <em>vs.</em> 106,292) in the count of Gs. At the second downstream position (+2) we observed a 38% increase (139,477 <em>vs.</em> 192,222) in the G count and an 18% decrease (133,840 <em>vs.</em> 109,737) in the count of Cs. The reverse complement of the results of the analysis for the 606,560 mutated C reference bases yields the same pattern (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS8.jpg">Figure S8</a>). Based on the nucleotide counts and the percentage changes at each flanking position, we observe an overrepresentation of a 5′-C<strong>G</strong>CG-3′ motif (with the mutagenized G position showed in bold) and an underrepresentation of 5′-AHGM-3′ [where the ambiguous DNA codes H=(A,C,T) and M=(A,C), respectively]. In comparison to the genome, the CG dinucleotide content was 57% higher in the 21 nt sequence context of the mutated positions (3.4% <em>vs.</em> 5.3%) and the 5′-C<strong>G</strong>CG-3′ motif is also clearly delineated in the positions 6–9 in the MEME motif results.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The 21 nt sequence context comparison for the mutated G-residue DNA sites in the EMS-induced SNPs and randomly selected positions in the sorghum reference genome. The percentage change in the nucleotide frequency for all DNA base types in all DNA positions shows an overrepresentation of a 5&#x2032;-CGCG-3&#x2032; motif. The mutated G-residue is shown in bold. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45765907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The 21 nt sequence context comparison for the mutated G-residue DNA sites in the EMS-induced SNPs and randomly selected positions in the sorghum reference genome. The percentage change in the nucleotide frequency for all DNA base types in all DNA positions shows an overrepresentation of a 5&#x2032;-CGCG-3&#x2032; motif. The mutated G-residue is shown in bold. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/1079/F6.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/3/1079/F6.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488959" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">The 21 nt sequence context comparison for the mutated G-residue DNA sites in the EMS-induced SNPs and randomly selected positions in the sorghum reference genome. The percentage change in the nucleotide frequency for all DNA base types in all DNA positions shows an overrepresentation of a 5′-CGCG-3′ motif. The mutated G-residue is shown in bold. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Sequence context of the error-prone SNP positions</h3><p id="p-42">We obtained BLAST global alignment results for the 51 nt sequence context of 57,404, 55,085, and 57,994 of the error-prone SNP positions, random genomic locations, and nonreplicate EMS-induced SNP positions, respectively. Of these, 38% (21,993 positions), 37% (20,156 positions), and 24% (14,099 positions) of the error-prone, random genomic, and random EMS-induced SNPs, respectively, had perfect matching paralogous sequences in the reference genome. Considering only end-to-end ungapped alignments of these sequences, we detected a total of 68,568,482 near-perfect matches for 41,733 (73%) of the error-prone SNP positions. For the randomly selected genomic positions, we detected ungapped near-perfect matches for nearly as many loci (33,980 corresponding to 62%) but in only 29,849,107 global alignments, indicating lower copy numbers for the paralogous sequences. This pattern of lower paralog coverage for the 51 nt SNP context was also true for the nonreplicate SNPs, where we recovered threefold fewer paralogous locations on average (21,870,737 global alignments for 33,605 sites corresponding to 58%). Interestingly, 51% (34,966,393 alignments) of the ungapped alignments to error-prone SNP sequence contexts contained a mismatch at the central SNP position. In contrast, only 7% (1,996,337 alignments) and 11% (2,369,537 alignments) of the ungapped alignments for the random genomic positions and the nonreplicate EMS-induced SNPs, respectively, contained a mismatch at the central (SNP) position (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The total length of the super scaffolds that have not been assigned to any chromosome in sorghum genome reference assembly version 2.1 is 68 Mb, and accounts for 9.4% of the genome assembly. Yet, &gt;51% (17,735,536 alignments) of the global alignments with a central mismatch in the error-prone SNP alignment data had super scaffold origins. For the nonreplicate EMS-induced SNPs and random genomic position data sets, only 32% (763,384 alignments) and 28% (563,042 alignments), respectively, of the global alignments with a central mismatch occurred in super scaffolds. The substitutions at the central mismatch positions in the global alignments matched the SNP call for 58% of the error-prone SNP positions (33,083 SNPs) but only 34.5% of the nonreplicate EMS-induced SNP positions (20,011 SNPs). The end-to-end ungapped (global) alignments may contain more than one mismatched position. Further filtering of the global alignment results to retain only alignments containing a single mismatch at the central (SNP) position identified 2,032,958 alignments for the error-prone SNP sequence contexts. Of these, 47% (951,217 alignments) were to super scaffold assemblies (File S24). Similarly, we also obtained 113,515 and 120,547 single (central) mismatched global alignments for the random genome and EMS-induced SNP data sets, respectively, but only 19 (22,011 alignments) and 28% (33,904 alignments) matched super scaffolds assemblies, respectively. This method was more stringent, and only recapitulated 26 (14,913 SNP positions) and 7% (4191 SNP positions) of the error-prone and EMS-induced SNP positions, respectively (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). These results demonstrate that paralogous sequences from poorly assembled regions of the genome match the error-prone sites detected as replicate SNPs in whole-genome sequence data from the reference genotype. The DNA represented by these assemblies disproportionally contributed to the detection of false positives in shotgun sequencing experiments.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488939/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488939/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">BLAST results for the global alignment of the 51 nt sequence context of the error-prone SNPs, random genomic, and random EMS-induced SNP positions</span><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of BLAST global alignments of the 51 nt sequence context of the probable error-prone SNPs to the sorghum reference genome. The sample alignments depict paralogous alignments that result in false positive G:C to A:T EMS-induced mutations. The reference and SNP positions are highlighted in bold font, with the mutated position underlined. BLAST, Basic Local Alignment Search Tool; EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism. (A) Alignment showing a G to A substitution. (B) Alignment showing a C to T substitution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45765907" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Examples of BLAST global alignments of the 51 nt sequence context of the probable error-prone SNPs to the sorghum reference genome. The sample alignments depict paralogous alignments that result in false positive G:C to A:T EMS-induced mutations. The reference and SNP positions are highlighted in bold font, with the mutated position underlined. BLAST, Basic Local Alignment Search Tool; EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism. (A) Alignment showing a G to A substitution. (B) Alignment showing a C to T substitution.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/1079/F7.medium.gif" width="440" height="144"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/3/1079/F7.medium.gif" width="440" height="144"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/1079/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488786" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-44" class="first-child">Examples of BLAST global alignments of the 51 nt sequence context of the probable error-prone SNPs to the sorghum reference genome. The sample alignments depict paralogous alignments that result in false positive G:C to A:T EMS-induced mutations. The reference and SNP positions are highlighted in bold font, with the mutated position underlined. BLAST, Basic Local Alignment Search Tool; EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism. (A) Alignment showing a G to A substitution. (B) Alignment showing a C to T substitution.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-27" class="subsection"><h3>EMS-induced variant confirmation</h3><p id="p-45">A total of 29 randomly selected individuals from the collection of 586 sequenced individuals were resequenced using leaf tissues from the M<sub>5</sub> generation and the Illumina WideSeq paired-end sequencing protocol. The sequencing yielded mean genome coverage of 0.06×. These reads were aligned to the sorghum genome and the positions of previously identified SNPs in these 29 lines were assessed. Given that a mean of 2177 homozygous nonreplicate SNPs was found in each line, this should overlap with 130 SNPs per line and provide ample depth to test line identity. The low-depth sequence overlapped with 3843 nonreplicate SNP positions and 96% of these were unambiguously confirmed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS9.jpg">Figure S9</a>). This validated our SNP calling procedures, and also verified the identity of our lines and the validity of our pedigrees. There were 1745 reads that contained positions called as heterozygous nonreplicate SNPs in the M<sub>3</sub> sequence data. For these heterozygous SNPs, our low-pass sequence confirmed 41% (710 out of 1745) of the mutations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS10.jpg">Figure S10</a>). This rate of validation again confirmed our pedigrees and line identities. The lower rate of confirmation is consistent with the expectations of segregation, where 37.5% of the M<sub>5</sub> should now be homozygous references for the heterozygous position and of the 25% still heterozygous, we would recover the nonreference allele in half of the reads. This should have resulted in 50% nonreference alleles, or 48% if the validation rate was the same for heterozygous and homozygous SNPs. Consistent with the likely higher false positive rate of the heterozygous SNPs, suggested by the lower G:C to A:T SNP percentage in the nonreplicate SNPs, we validated fewer SNPs.</p></div></div><div class="section discussion" id="sec-28"><h2 class="">Discussion</h2><p id="p-46">The accuracy of the <em>in silico</em> detection of sequence variants can be impacted by the inherent sequencing error rates of NGS technologies, nature of the variant-calling programs, genome complexities, sequence alignment programs, and the quality of reference genome assemblies (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Estivill <em>et al.</em> 2002</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cheung <em>et al.</em> 2003</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Fredman <em>et al.</em> 2004</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Nielsen <em>et al.</em> 2011</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">O’Rawe <em>et al.</em> 2013</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cheng <em>et al.</em> 2014</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Robasky <em>et al.</em> 2014</a>). We utilized an additional step to improve current standard practices for filtering false positive variant calls detected in genetic studies using mutagenizing agents such as EMS, which induce random mutations. EMS mutagenesis should result in DNA mutations at nearly random subsets of the mutable positions in the genome of a treated organism. For an organism with a sufficiently large genome, the likelihood of a randomly induced mutation occurring at the same genomic position in two individuals that were treated independently is extremely low. Hence, a nonreplicated high-quality SNP detected at a genomic position that is unique to a single individual in a population of independently EMS-mutagenized individuals is most likely a true positive and represents a real signal in the SNP detection. If a sequencing error, on the other hand, is nonrandomly distributed, a SNP detected in multiple lines at the same genomic position is mostly likely a replicated false positive SNP. Applying this cost-effective and highly accurate filtering step to the large-scale NGS data set produced from the M<sub>3</sub> generation of an EMS-mutagenized population of the agronomically important crop species <em>S. bicolor</em>, we showed that 76% of the homozygous SNPs and 94% of the heterozygous SNPs detected using standard SNP detection and filtering were false positives (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS1.jpg">Figure S1</a>). The NGS sequencing coverage depth for the 586 sorghum lines in this study was 7×, and this is lower than the 20× coverage depth often described as the minimum for accurate SNP calling (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Flibotte <em>et al.</em> 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liu <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">James <em>et al.</em> 2013</a>). Two previous efforts had independently used a similar method to refine their variant calling (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sarin <em>et al.</em> 2010</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Uchida <em>et al.</em> 2011</a>). However, our characterization and results clearly reveals empirical evidence of the accuracy and effectiveness of the approach. We identified and cataloged the relatively few genomic positions of the sorghum reference genome, consisting of 57,872 positions, that were most probably error-prone, and which produced 4,123,621 SNP calls of high statistical quality (File S2). Similar approaches for elucidating this experimentally identifiable set of replicate SNP positions in any species, which lack the expected EMS mutation spectrum, correspond to repeated sequences, and contain paralogs with the predicted SNP, should improve the accuracy of SNP calling (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Henry <em>et al.</em> 2014</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Addo-Quaye <em>et al.</em> 2017</a>). Similarly, filtering of the indel alleles retained &lt;1% (17,760) as nonreplicate indels (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS4.jpg">Figure S4</a> and Tables S7 and S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). Consistent with the analysis of indels induced by fast neutron irradiation of rice (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li G. <em>et al.</em> 2016</a>), the vast majority of our indel alleles were short in length (2.6 bases) with sizes ranging from 1 to 50 bases. In agreement with the high false positive detection rate for indels, the recent genome-wide analysis of chemically induced-mutations obtained by N-methyl-N’-nitro-N-nitrosoguanidine mutagenesis in the social soil amoeba <em>Dictyostelium discoideum</em> also uncovered very few likely true positive alleles, while discarding 99% of initially predicted variants (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li C.-L. F. <em>et al.</em> 2016</a>).</p><p id="p-47">The mutagenic effect of EMS is well-characterized in a wide variety of viruses, prokaryotes, fungal, plant, and animal genomes (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Loveless 1958</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">1969</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Singer <em>et al.</em> 1982</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Sega 1984</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Shrivastav <em>et al.</em> 2010</a>). The predominant DNA reactive site that produces DNA lesions after EMS alkylation is O<sup>6</sup> guanine. The ethylation of O<sup>6</sup> guanine results in low mutagenicity in the EMS-treated organism (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Loveless 1969</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Singer <em>et al.</em> 1982</a>). We found an average of only 2177 homozygous and 815 heterozygous nonreplicate SNP calls per sorghum individual (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). Overall, 27% of the nonreplicate SNPs were heterozygous, and this was slightly higher than the expected ∼22% heterozygosity for an M<sub>3</sub> population. Two possible dynamics contributed to the minor discrepancy between the observed and expected levels of SNP heterozygosity: false positives due to <em>in silico</em> overestimation and <em>bona fide</em> heterozygous calls due to pollen contamination. In contrast to the homozygous SNP predictions, we observed a positive correlation (<em>R</em> = 0.51) between the number of <em>in silico</em> heterozygous SNPs detected and the read mapping coverage for the sequenced individuals. In attempting to compensate for a lack of sufficient sequencing coverage for predicting the heterozygous SNPs, our relaxed computational stringency for predicting heterozygous SNPs may have caused an overestimation in the number of heterozygous calls. This explanation applies more specifically to samples with lower sequencing coverage. For some individuals, the consequence of pollen contamination from outcrossing to previous generations, and not sequencing coverage, may have caused an increase in the level of heterozygosity. For example, 21 of the sequenced individuals with at least 7× sequencing coverage were at most only 20% homozygous (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). The mispairing of O<sup>6</sup> alkyl guanine with thymine results in G:C to A:T mutations (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Loveless 1969</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Prakash and Sherman 1973</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kohalmi and Kunz 1988</a>). The mutation spectra of the detected homozygous and heterozygous nonreplicate SNPs showed that 98 and 94% were G:C to A:T transitions, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS2.jpg">Figure S2</a>). The slightly lower percentage of G:C to A:T transitions for the heterozygous SNPs may be attributed to higher false positive rates due to the inclusion of a greater number of sites with paralogous positions within the heterozygous SNP class, as well as the lower overall stringency when non-SNP containing reads are allowed. The notion that we still have some false positives within our SNP calls is supported by the fact that within the protein-coding regions &gt;99% of the homozygous SNPs that we detected were G:C to A:T substitutions. These positions are less likely to possess unassembled paralogous sequences than the intergenic positions. This frequency matches a previous study in <em>Arabidopsis</em> that found that &gt;99% of EMS-induced mutations in a set of 192 genes were G:C to A:T substitutions (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Greene <em>et al.</em> 2003</a>). Although DNA lesions from EMS mutagenesis are predominantly associated with O<sup>6</sup> guanine ethylation, other minor sites such as O<sup>2</sup> and O<sup>4</sup> thymine had also been suggested. Studies show that the alkylation of O<sup>4</sup> thymine results in mispairing with guanine and that this leads to A:T to G:C transitions (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Prakash and Sherman 1973</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Coulondre and Miller 1977</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Singer <em>et al.</em> 1978</a>). Our analysis revealed that A:T to G:C transitions were the second most abundant class of likely EMS-induced mutations and accounted for 0.7% of the homozygous SNPs. This observation corroborates the finding in viruses that suggests that EMS produces A:T to G:C transitions at almost two orders of magnitude less frequently than G:C to A:T transitions (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Krieg 1963</a>).</p><p id="p-48">Previous large-scale analysis of flanking positions of predicted EMS-induced mutations in sorghum reported an overrepresentation of cytosine residues at positions −2 and +1 (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jiao <em>et al.</em> 2016</a>). In addition to these sites, we detected a more extensive signal characterized by a genome-wide overrepresentation of a 5′-CGCG-3′ motif and an underrepresentation of a 5′-AHGM-3′ motif (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS7.jpg">Figure S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS8.jpg">Figure S8</a>). In particular, the intriguing upsurge in the C count occurring at the position immediately preceding the mutated G implied a greater proportion of CG dinucleotides among the mutated bases. These dinucleotides are underrepresented (3.6%) in the sorghum genome (version 2.1), and are the sites of DNA methylation. Methylated cytosines are known to spontaneously deaminate to thymidine. For this reason, the G to A mutational impact of EMS, which would pair perfectly with thymidine, may be more readily affected at these locations and go undetected by repair processes. We observed that the CG dinucleotide content in the 21 nt context of the mutation was 57% higher than that of the sorghum reference genome. The observation that a similar sequence occurs as an overrepresented motif was described previously in the large-scale exome sequencing of rice EMS mutants (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Henry <em>et al.</em> 2014</a>). This observation suggests that something in common between the two systems, either inherent to the structure of the DNA sequence, the repair process, or the methylation of cytosines, is responsible for the higher rate of mutation. Whereas the rice EMS-induced mutations were more likely at unmethylated cytosines (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Henry <em>et al.</em> 2014</a>), sorghum EMS-induced mutations were overrepresented at methylated cytosines (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Jiao <em>et al.</em> 2016</a>). It is important to note that, while the relationships between cytosine methylation and EMS-induced mutations were in opposite directions in rice and sorghum, methylation did not affect the majority of positions.</p><p id="p-49">Since EMS mutagenesis targets predominantly random G positions in the genome, we found that the number of mutations in a gene is correlated with the gene length and the G+C count, but not with GC content. Hence, we observed that, in the absence of saturation, short gene transcripts with fewer G:C sites were likely to have few or no mutations in our population. Due to the nucleotide composition of the three canonical translational stop codons (TAA, TGA, and TAG), and the fact that the majority of the likely EMS-induced mutations are G:C to A:T substitutions, stop codons are primarily buffered from the effects of EMS mutagenesis and we saw very few losses of stop codons. TAA codons do not contain any G and/or C nucleotide targets of EMS, while the G to A substitution of TGA or TAG converts them silently to TAA stop codons. We did observe 148 such silent stop codon substitutions in our SNP annotation (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>), which may be of use to others interested in the regulation of stop codons, the contribution of stop codon usage to mRNA turnover, or nonsense suppression. We observed a single homozygous and four heterozygous stop codon loss SNPs that were due to non-G:C to A:T substitutions that should result in translation read-through (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-50">Previous studies have revealed the potential for genome assembly errors in the presence of repetitive sequences and segmental duplications that result in false positive or ambiguous variants (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Estivill <em>et al.</em> 2002</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cheung <em>et al.</em> 2003</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Fredman <em>et al.</em> 2004</a>). Our examination of the sequence context of the identified error-prone positions showed evidence that paralogous sequence variants, with origins in repetitive and duplicated genomic regions, contributed to false positive detections. We observed that the replicate SNPs were present in poorly assembled regions of the genome and present in a greater number of paralogs. The nonreplicate SNP positions were drawn from a dramatically lower proportion of positions with perfect match paralogs in the genome than expected by random chance (24% in the nonreplicate SNPs but 37% of randomly selected genomic locations). This demonstrates that SNP calling in polyploids, even paleopolyploids, will be suppressed at positions of conservation by ambiguous alignment and the erroneous assessment of no variation. Special procedures for calling SNPs at paralogs should improve SNP detection in subsets of genomes characterized by identity. Another source of the replicated SNP calls could be spontaneous mutations that arose in the parental line used in the mutagenesis, but that were absent in the reference genome genotype. For example, a recent resequencing of the sorghum BTx623 parental line utilized in a mutagenesis experiment revealed 13,243 high-confidence variations when compared to the reference sorghum genotype (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Jiao <em>et al.</em> 2016</a>). We found that the average sequence coverage depths at the error-prone SNP positions and the <em>bona fide</em> SNP position profiles were not different in our data. This may be due to the randomized assignment of nonunique reads from repeated sequences to each repeat location by the BWA aligner (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Firtina and Alkan 2016</a>). Novel methods for improving the assignment of multi-mapping reads are emerging and may improve SNP calling when widely implemented (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Johnson <em>et al.</em> 2016</a>).</p><p id="p-51">Although the removal of the shared variants is a very effective technique for identifying likely false positive EMS-induced mutations, a few caveats need to be highlighted. First, the method should be applied meticulously to prevent the introduction of false negatives. For example, sample mislabeling and unexpected hybridization between individuals within the population can result in the sequencing of multiple DNA samples containing shared (identity by descent) mutagenized chromosomes. Sequence contamination originating from sample or library preparation stages can also negatively impact the detection of false positives. In all the above-mentioned scenarios, technical errors or contaminants would result in the erroneous purging of true DNA variants that were shared due to identity by descent from a common ancestor. In contrast to noncontaminated samples, prior to population-wide subtraction of shared variations, sample pairs connected by contamination are characterized by an abnormally high percentage of shared variation between the two individuals. Prior to the removal of common variants, pairwise comparison of detected variants in all the sequenced individuals in the population is mandated to expose contaminated samples with shared pedigrees, regardless of how certain biologists are that there is a lack of contamination in their work. Second, although the likelihood is extremely low, it is not statistically impossible for a mutation to be independently induced at the same genomic position in two or more individuals (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Krasileva <em>et al.</em> 2017</a>). Hence, it is noteworthy that there is a possibility that a few replicate variants are real. This frequency increases as the sample size and mutation density go up, and as mutations are nonrandomly distributed in the genome. A careful examination of the subtracted replicate SNPs could recover false negative SNPs with origins in potential mutation hotspots. Our subsequent signal-to-noise analysis of the replicate SNPs predicted at the error-prone genomic positions found a nontrivial (G:C to A:T substitution percentage) signal for the cluster of genomic positions where the same SNP was predicted in at most two individuals (see <em>Materials and Methods</em> section; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FigureS3.jpg">Figure S3</a> and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1/FileS26.docx">File S26</a>). Hence, we selected an additional 18,582 homozygous and 11,418 heterozygous replicates or nonunique SNPs as likely false negative coincidental SNPs.</p><p id="p-52">In biological experiments, replication plays a critical role by adding statistical power to both detect differences and reject errant stochastic results, as discussed previously for NGS experiments (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Robasky <em>et al.</em> 2014</a>). Based on our work in a homozygous model system, we reckon that anyone applying NGS sequencing to identify a causal variant in the sequencing data from a single mutant line would likely have to sort through a vast majority of false positives. In the absence of intersample variant comparisons, the majority of the standard filtered SNPs in each line will appear to be possible causative mutations. If the same line were to be resequenced, it would return a subset of the false positive calls a second time, meeting the requirement of reproducibility, and these will be falsely confirmed as true DNA variants. As such we are skeptical of rare variant detection claims that have not sought rigorous explanations for the repeated detection of SNPs in experimental samples. For instance, the relevance of the error-prone positions to the cataloging of rare variants induced during cancer progression is needed. Hence, the quality of reference genome assemblies, the complexity of genomes, and a careful, critical view of NGS data demonstrate the need for a more sophisticated approach to false positive detection when rare variant identification is the experimental goal.</p><div id="sec-29" class="subsection"><h3>Conclusions</h3><p id="p-53">The prevalence of false positive DNA variants negatively impacts the efficiency of forward and reverse genetics studies. We utilized and described a cost-effective, high-throughput, and highly accurate NGS method for improving the <em>in silico</em> detection of induced mutations via the identification of false positive variants. This method is generally applicable to any genetics study and is also independent of the choice of the variant-calling program. Using the method, we have detected and described the set of EMS-induced mutations in a population of <em>S. bicolor</em>, which is a next-generation crop species with tremendous agronomical potential. We hope that the genetics community and plant breeders will find the described genetic resource to be very useful.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">We thank Damon Lisch for helpful comments on this article; Phillip SanMiguel and Rick Westerman of the Purdue University Genomics Core Facility for the next-generation sequencing and data warehousing support; and the Information Technology staff of the Purdue University Rosen Center for Advanced Computing for the research computing and cyber infrastructure support. This project was funded by grants from the Bill and Melinda Gates Foundation (grant number OPP 1052924) to M.T., C.W., and B.P.D.; the National Science Foundation (grant number PGRP 1444503) to B.P.D.; and Institutional Development Award funding to C.A.-Q. (grant #P20GM103408).</p><p id="p-55">Author contributions: M.T., C.W., C.A.-Q., and B.P.D. designed the experiments. C.A.-Q. and B.P.D. performed the bioinformatics analysis. C.A.-Q. and N.C. confirmed the detected SNPs. C.A.-Q. and B.P.D. drafted the article. B.P.D., M.T., C.W., C.A.-Q., and N.C. edited the article.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300301/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-57"><em>Communicating editor: R. Dawe</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-22"><span class="received-label">Received </span>September 22, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-01-21"><span class="accepted-label">Accepted </span>January 21, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Addo-Quaye <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.1" data-doi="10.1534/g3.116.029660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addo-Quaye</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buescher</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaikam</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Forward genetics by sequencing EMS variation-induced inbred lines.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029660%26rft_id%253Dinfo%253Apmid%252F28040779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi80MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzMvMTA3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.2" data-doi="10.1038/nrg1893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Moving forward in reverse: genetic technologies to enable genome-wide phenomic screens in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAlonso%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D524%26rft.epage%253D536%26rft.atitle%253DMoving%2Bforward%2Bin%2Breverse%253A%2Bgenetic%2Btechnologies%2Bto%2Benable%2Bgenome-wide%2Bphenomic%2Bscreens%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1893%26rft_id%253Dinfo%253Apmid%252F16755288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16755288&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238377200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.3" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.4" data-doi="10.1007/BF02672069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arumuganathan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Earle</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Nuclear DNA content of some important plant species.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D9%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02672069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02672069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.5" data-doi="10.1093/nar/gkp335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buske</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">MEME SUITE: tools for motif discovery and searching.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">W202</span>–<span class="cit-lpage">W208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp335%26rft_id%253Dinfo%253Apmid%252F19458158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19458158&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267889100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bevan</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uauy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wulff</span>  <span class="cit-name-given-names">B. B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasileva</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic innovation for crop improvement.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">543</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D543%26rft.spage%253D346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.7" data-doi="10.1111/j.1467-7652.2011.00646.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blomstedt</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleadow</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naur</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A combined biochemical screen and TILLING approach identifies mutations in Sorghum bicolor L. Moench resulting in acyanogenic forage production.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DBlomstedt%26rft.auinit1%253DC.%2BK.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D66%26rft.atitle%253DA%2Bcombined%2Bbiochemical%2Bscreen%2Band%2BTILLING%2Bapproach%2Bidentifies%2Bmutations%2Bin%2BSorghum%2Bbicolor%2BL.%2BMoench%2Bresulting%2Bin%2Bacyanogenic%2Bforage%2Bproduction.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2011.00646.x%26rft_id%253Dinfo%253Apmid%252F21880107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2011.00646.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21880107&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.8" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.9" data-doi="10.1093/bioinformatics/btu067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teo</span>  <span class="cit-name-given-names">Y.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">R. T.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Assessing single nucleotide variant detection and genotype calling on whole-genome sequenced individuals.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1707</span>–<span class="cit-lpage">1713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu067%26rft_id%253Dinfo%253Apmid%252F24558117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24558117&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338109200047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.10" data-doi="10.1186/gb-2003-4-4-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estivill</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khaja</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacDonald</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide detection of segmental duplications and potential assembly errors in the human genome sequence.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCheung%26rft.auinit1%253DJ.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DGenome-wide%2Bdetection%2Bof%2Bsegmental%2Bduplications%2Band%2Bpotential%2Bassembly%2Berrors%2Bin%2Bthe%2Bhuman%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2003-4-4-r25%26rft_id%253Dinfo%253Apmid%252F12702206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2003-4-4-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12702206&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.11" data-doi="10.3389/fgene.2012.00035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Using <em>Drosophila melanogaster</em> as a model for genotoxic chemical mutational studies with a new program, SnpSift.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DUsing%2BDrosophila%2Bmelanogaster%2Bas%2Ba%2BModel%2Bfor%2BGenotoxic%2BChemical%2BMutational%2BStudies%2Bwith%2Ba%2BNew%2BProgram%252C%2BSnpSift.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2012.00035%26rft_id%253Dinfo%253Apmid%252F22435069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2012.00035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22435069&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.12" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.13" data-doi="10.1016/0022-2836(77)90059-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coulondre</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Genetic studies of the lac repressor. IV. Mutagenic specificity in the lacI gene of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCoulondre%26rft.auinit1%253DC.%26rft.volume%253D117%26rft.issue%253D3%26rft.spage%253D577%26rft.epage%253D606%26rft.atitle%253DGenetic%2Bstudies%2Bof%2Bthe%2Blac%2Brepressor.%2BIV.%2BMutagenic%2Bspecificity%2Bin%2Bthe%2BlacI%2Bgene%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252877%252990059-6%26rft_id%253Dinfo%253Apmid%252F416218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(77)90059-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=416218&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977EJ07400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.14" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.15" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.16" data-doi="10.1093/hmg/11.17.1987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Estivill</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujana</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakabayashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Chromosomal regions containing high-density and ambiguously mapped putative single nucleotide polymorphisms (SNPs) correlate with segmental duplications in the human genome.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1987</span>–<span class="cit-lpage">1995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DEstivill%26rft.auinit1%253DX.%26rft.volume%253D11%26rft.issue%253D17%26rft.spage%253D1987%26rft.epage%253D1995%26rft.atitle%253DChromosomal%2Bregions%2Bcontaining%2Bhigh-density%2Band%2Bambiguously%2Bmapped%2Bputative%2Bsingle%2Bnucleotide%2Bpolymorphisms%2B%2528SNPs%2529%2Bcorrelate%2Bwith%2Bsegmental%2Bduplications%2Bin%2Bthe%2Bhuman%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252F11.17.1987%26rft_id%253Dinfo%253Apmid%252F12165560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/11.17.1987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12165560&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177356600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.17" data-doi="10.1093/bioinformatics/btw139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Firtina</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">On genomic repeats and reproducibility.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2243</span>–<span class="cit-lpage">2247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw139%26rft_id%253Dinfo%253Apmid%252F27153582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153582&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.18" data-doi="10.1534/genetics.110.116616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neil</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome profiling of mutagenesis in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116616%26rft_id%253Dinfo%253Apmid%252F20439774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNDMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC8zLzEwNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.19" data-doi="10.1038/ng1401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fredman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Den Dunnen</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Complex SNP-related sequence variation in segmental genome duplications.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFredman%26rft.auinit1%253DD.%26rft.volume%253D36%26rft.issue%253D8%26rft.spage%253D861%26rft.epage%253D866%26rft.atitle%253DComplex%2BSNP-related%2Bsequence%2Bvariation%2Bin%2Bsegmental%2Bgenome%2Bduplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1401%26rft_id%253Dinfo%253Apmid%252F15247918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15247918&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222974000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.20" data-doi="10.1093/nar/gkr944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodstein</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neupane</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Phytozome: a comparative platform for green plant genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1178</span>–<span class="cit-lpage">D1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr944%26rft_id%253Dinfo%253Apmid%252F22110026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22110026&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300176&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Codomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Spectrum of chemically induced mutations from a large-scale reverse-genetic screen in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGreene%26rft.auinit1%253DE.%2BA.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D731%26rft.epage%253D740%26rft.atitle%253DSpectrum%2Bof%2BChemically%2BInduced%2BMutations%2BFrom%2Ba%2BLarge-Scale%2BReverse-Genetic%2BScreen%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Apmid%252F12807792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNzMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC8zLzEwNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.22" data-doi="10.1111/j.1469-8137.2011.03859.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grierson</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chase</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grierson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">One hundred important questions facing plant science research.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DGrierson%26rft.auinit1%253DC.%2BS.%26rft.volume%253D192%26rft.issue%253D1%26rft.spage%253D6%26rft.epage%253D12%26rft.atitle%253DOne%2Bhundred%2Bimportant%2Bquestions%2Bfacing%2Bplant%2Bscience%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03859.x%26rft_id%253Dinfo%253Apmid%252F21883238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03859.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21883238&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294559400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.23" data-doi="10.1105/tpc.113.121590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieberman</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngo</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasileva</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient genome-wide detection and cataloging of EMS-induced mutations using exome capture and next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1382</span>–<span class="cit-lpage">1397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.121590%26rft_id%253Dinfo%253Apmid%252F24728647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzQvMTM4MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvMy8xMDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.24" data-doi="10.1186/gb-2013-14-6-r61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordström</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klasen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salomé</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">User guide for mapping-by-sequencing in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR61%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r61%26rft_id%253Dinfo%253Apmid%252F23773572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23773572&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.25" data-doi="10.1105/tpc.16.00373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chopra</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Sorghum mutant resource as an efficient platform for gene discovery in grasses.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1551</span>–<span class="cit-lpage">1562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.16.00373%26rft_id%253Dinfo%253Apmid%252F27354556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI4LzcvMTU1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvMy8xMDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeoh</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coruh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improved placement of multi-mapping small RNAs.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2103</span>–<span class="cit-lpage">2111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D2103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.27" data-doi="10.1111/nph.12199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koegel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ait Lahmidi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnould</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatagnier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walder</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The family of ammonium transporters (AMT) in <em>Sorghum bicolor</em>: two AMT members are induced locally, but not systemically in roots colonized by arbuscular mycorrhizal fungi.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D198%26rft.spage%253D853%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12199%26rft_id%253Dinfo%253Apmid%252F23461653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23461653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.28" data-doi="10.1016/0022-2836(88)90355-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohalmi</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Role of neighbouring bases and assessment of strand specificity in ethylmethanesulphonate and <em>N</em>-methyl-<em>N</em>′-nitro-<em>N</em>-nitrosoguanidine mutagenesis in the <em>SUP</em>4-o gene of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKohalmi%26rft.auinit1%253DS.%2BE.%26rft.volume%253D204%26rft.issue%253D3%26rft.spage%253D561%26rft.epage%253D568%26rft.atitle%253DRole%2Bof%2Bneighbouring%2Bbases%2Band%2Bassessment%2Bof%2Bstrand%2Bspecificity%2Bin%2Bethylmethanesulphonate%2Band%2BN-methyl-N%2527-nitro-N-nitrosoguanidine%2Bmutagenesis%2Bin%2Bthe%2BSUP4-o%2Bgene%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252888%252990355-5%26rft_id%253Dinfo%253Apmid%252F3066906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(88)90355-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3066906&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988R362100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.29" data-doi="10.1073/pnas.1619268114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krasileva</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasquez-Gross</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paraiso</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Uncovering hidden variation in polyploid wheat.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E913</span>–<span class="cit-lpage">E921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1619268114%26rft_id%253Dinfo%253Apmid%252F28096351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE0LzYvRTkxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvMy8xMDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krieg</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Ethyl methanesulfonate-induced reversion of bacteriophage T4rII mutants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKrieg%26rft.auinit1%253DD.%2BR.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D561%26rft.epage%253D580%26rft.atitle%253DETHYL%2BMETHANESULFONATE-INDUCED%2BREVERSION%2BOF%2BBACTERIOPHAGE%2BT4rII%2BMUTANTS%26rft_id%253Dinfo%253Apmid%252F14035786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0OC80LzU2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvMy8xMDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.31" data-doi="10.1534/genetics.113.149567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krothapalli</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buescher</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapple</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Forward genetics by genome sequencing reveals that rapid cyanide release deters insect herbivory of <em>Sorghum bicolor</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.149567%26rft_id%253Dinfo%253Apmid%252F23893483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvMzA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC8zLzEwNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.32" data-doi="10.1038/nprot.2009.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKumar%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253D1073%26rft.epage%253D1081%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bcoding%2Bnon-synonymous%2Bvariants%2Bon%2Bprotein%2Bfunction%2Busing%2Bthe%2BSIFT%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.86%26rft_id%253Dinfo%253Apmid%252F19561590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561590&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268858700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.33" data-doi="10.1101/gr.205682.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.-L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santhanam</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webb</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zupan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaulsky</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Gene discovery by chemical mutagenesis and whole-genome sequencing in <em>Dictyostelium</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1268</span>–<span class="cit-lpage">1276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.205682.116%26rft_id%253Dinfo%253Apmid%252F27307293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI2LzkvMTI2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvMy8xMDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.34" data-doi="10.1016/j.molp.2016.03.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chern</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schackwitz</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide sequencing of 41 rice (<em>Oryza sativa</em> L.) mutated lines reveals diverse mutations induced by fast-neutron irradiation.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1078</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D9%26rft.spage%253D1078%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2016.03.009%26rft_id%253Dinfo%253Apmid%252F27018389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2016.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27018389&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.35" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.36" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.37" data-doi="10.1186/1746-4811-8-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormack</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Targeted parallel sequencing of large genetically-defined genomic regions for identifying mutations in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BMethods%26rft.aulast%253DLiu%26rft.auinit1%253DK.%2BH.%26rft.volume%253D8%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DTargeted%2Bparallel%2Bsequencing%2Bof%2Blarge%2Bgenetically-defined%2Bgenomic%2Bregions%2Bfor%2Bidentifying%2Bmutations%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-8-12%26rft_id%253Dinfo%253Apmid%252F22462410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-8-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22462410&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.38" data-doi="10.1038/1811212a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loveless</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">Increased rate of plaque-type and host-range mutation following treatment of bacteriophage <em>in vitro</em> with ethyl methane sulphonate.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1212</span>–<span class="cit-lpage">1213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLoveless%26rft.auinit1%253DA.%26rft.volume%253D181%26rft.issue%253D4617%26rft.spage%253D1212%26rft.epage%253D1213%26rft.atitle%253DIncreased%2Brate%2Bof%2Bplaque-type%2Band%2Bhost-range%2Bmutation%2Bfollowing%2Btreatment%2Bof%2Bbacteriophage%2Bin%2Bvitro%2Bwith%2Bethyl%2Bmethane%2Bsulphonate.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F1811212a0%26rft_id%253Dinfo%253Apmid%252F13541432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/1811212a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13541432&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.39" data-doi="10.1038/223206a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loveless</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Possible relevance of O-6 alkylation of deoxyguanosine to the mutagenicity and carcinogenicity of nitrosamines and nitrosamides.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">223</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLoveless%26rft.auinit1%253DA.%26rft.volume%253D223%26rft.issue%253D5202%26rft.spage%253D206%26rft.epage%253D207%26rft.atitle%253DPossible%2Brelevance%2Bof%2BO-6%2Balkylation%2Bof%2Bdeoxyguanosine%2Bto%2Bthe%2Bmutagenicity%2Band%2Bcarcinogenicity%2Bof%2Bnitrosamines%2Band%2Bnitrosamides.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F223206a0%26rft_id%253Dinfo%253Apmid%252F5791738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/223206a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5791738&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1969D617900056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.40" data-doi="10.1038/nrg2626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing technologies - the next generation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMetzker%26rft.auinit1%253DM.%2BL.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D46%26rft.atitle%253DSequencing%2Btechnologies%2B-%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2626%26rft_id%253Dinfo%253Apmid%252F19997069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997069&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.41" data-doi="10.1093/nar/gkr344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morimoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshikawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Sequence-specific error profile of Illumina sequencers.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr344%26rft_id%253Dinfo%253Apmid%252F21576222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21576222&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.42" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.43" data-doi="10.1038/nature07629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation genetics in plants.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D456%26rft.issue%253D7223%26rft.spage%253D720%26rft.epage%253D723%26rft.atitle%253DNext-generation%2Bgenetics%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07629%26rft_id%253Dinfo%253Apmid%252F19079047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079047&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261559900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.44" data-doi="10.1038/nbt.2515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordström</span>  <span class="cit-name-given-names">K. J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albani</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutjahr</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartwig</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutation identification by direct comparison of whole-genome sequencing data from mutant and wild-type individuals using k-mers.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DNordstrom%26rft.volume%253D31%26rft.spage%253D325%26rft.atitle%253DMutation%2Bidentification%2Bby%2Bdirect%2Bcomparison%2Bof%2Bwhole-genome%2Bsequencing%2Bdata%2Bfrom%2Bmutant%2Band%2Bwild-type%2Bindividuals%2Busing%2Bk-mers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2515%26rft_id%253Dinfo%253Apmid%252F23475072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23475072&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.45" data-doi="10.1186/gm432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Rawe</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D5%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm432%26rft_id%253Dinfo%253Apmid%252F23537139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23537139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.46" data-doi="10.1038/nrg730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossniklaus</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The art and design of genetic screens: Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPage%26rft.auinit1%253DD.%2BR.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D124%26rft.epage%253D136%26rft.atitle%253DThe%2Bart%2Band%2Bdesign%2Bof%2Bgenetic%2Bscreens%253A%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg730%26rft_id%253Dinfo%253Apmid%252F11836506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11836506&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173652700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomics of sorghum.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2008</span>: <span class="cit-fpage">362451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BPlant%2BGenomics%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D2008%26rft.spage%253D362451%26rft.epage%253D362451%26rft.atitle%253DGenomics%2Bof%2Bsorghum.%26rft_id%253Dinfo%253Apmid%252F18483564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18483564&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.48" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.49" data-doi="10.1007/s10681-005-5298-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bean</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graybosch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tilley</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Characterization of waxy grain sorghum lines in relation to granule-bound starch synthase.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D144%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-005-5298-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-005-5298-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000232056200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.50" data-doi="10.2135/cropsci2008.08.0461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenks</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rich</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ejeta</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mutagenesis, selection, and allelic analysis of epicuticular wax mutants in Sorghum.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPeters%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D1250%26rft.epage%253D1258%26rft.atitle%253DMutagenesis%252C%2BSelection%252C%2Band%2BAllelic%2BAnalysis%2Bof%2BEpicuticular%2BWax%2BMutants%2Bin%2BSorghum%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000268109800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.51" data-doi="10.1101/gr.226102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sciara</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Integration of Cot analysis, DNA cloning, and high-throughput sequencing facilitates genome characterization and gene discovery.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.226102%26rft_id%253Dinfo%253Apmid%252F11997346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzUvNzk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC8zLzEwNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petti</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harman-Ware</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tateno</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kushwaha</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shearer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sorghum mutant RG displays antithetic leaf shoot lignin accumulation resulting in improved stem saccharification properties.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Biofuels</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnol.%2BBiofuels%26rft.volume%253D6%26rft.spage%253D146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.53" data-doi="10.1104/pp.15.00928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petti</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirano</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stork</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeBolt</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mapping of a cellulose-deficient mutant named dwarf1–1 in Sorghum bicolor to the green revolution gene gibberellin20-oxidase reveals a positive regulatory association between gibberellin and cellulose biosynthesis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00928%26rft_id%253Dinfo%253Apmid%252F26198258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjkvMS83MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzMvMTA3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.54" data-doi="10.1016/0022-2836(73)90270-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Mutagenic specificity: reversion of iso-1-cytochrome c mutants of yeast.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPrakash%26rft.auinit1%253DL.%26rft.volume%253D79%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D82%26rft.atitle%253DMutagenic%2Bspecificity%253A%2Breversion%2Bof%2Biso-1-cytochrome%2Bc%2Bmutants%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252873%252990270-2%26rft_id%253Dinfo%253Apmid%252F4355598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(73)90270-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4355598&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.55" data-doi="10.1093/aob/mci015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodnett</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome evolution in the genus Sorghum (Poaceae).</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DPRICE%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D219%26rft.epage%253D227%26rft.atitle%253DGenome%2BEvolution%2Bin%2Bthe%2BGenus%2BSorghum%2B%2528Poaceae%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmci015%26rft_id%253Dinfo%253Apmid%252F15596469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mci015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15596469&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.56" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.57" data-doi="10.1111/tpj.13007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rizal</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dionora</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanchana</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Two forward genetic screens for vein density mutants in sorghum converge on a cytochrome P450 gene in the brassinosteroid pathway.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13007%26rft_id%253Dinfo%253Apmid%252F26333774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26333774&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.58" data-doi="10.1038/nrg3655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robasky</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The role of replicates for error mitigation in next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3655%26rft_id%253Dinfo%253Apmid%252F24322726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3655&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24322726&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.59" data-doi="10.1016/j.devcel.2011.05.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowan</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koenig</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Developmental genetics and new sequencing technologies: the rise of nonmodel organisms.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DRowan%26rft.auinit1%253DB.%2BA.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D76%26rft.atitle%253DDevelopmental%2Bgenetics%2Band%2Bnew%2Bsequencing%2Btechnologies%253A%2Bthe%2Brise%2Bof%2Bnonmodel%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2011.05.021%26rft_id%253Dinfo%253Apmid%252F21763609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2011.05.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21763609&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293310200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.60" data-doi="10.1534/genetics.110.116319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigelow</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyanov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doitsidou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of multiple ethyl methanesulfonate-mutagenized <em>Caenorhabditis elegans</em> strains by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116319%26rft_id%253Dinfo%253Apmid%252F20439776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNDE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC8zLzEwNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sattler</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saballos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funnell-Harris</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vermerris</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of novel Sorghum <em>brown midrib</em> mutants from an EMS-mutagenized population.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2115</span>–<span class="cit-lpage">2124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D2115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing pools of individuals—mining genome-wide polymorphism data without big funding.</span> <abbr class="cit-jnl-abbrev">Nat. Publ. Gr.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPubl.%2BGr.%26rft.volume%253D15%26rft.spage%253D749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.63" data-doi="10.1038/nrg3745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using next-generation sequencing to isolate mutant genes from forward genetic screens.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3745%26rft_id%253Dinfo%253Apmid%252F25139187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25139187&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.64" data-doi="10.1016/j.tplants.2011.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast-forward genetics enabled by new sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DSchneeberger%26rft.auinit1%253DK.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D282%26rft.epage%253D288%26rft.atitle%253DFast-forward%2Bgenetics%2Benabled%2Bby%2Bnew%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2011.02.006%26rft_id%253Dinfo%253Apmid%252F21439889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2011.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21439889&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291191000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scully</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gries</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funnell-Harris</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovacs</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Characterization of novel <em>Brown midrib 6</em> mutations affecting lignin biosynthesis in sorghum.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPlant%2BBiol.%26rft.volume%253D58%26rft.spage%253D136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.66" data-doi="10.1016/0165-1218(84)90033-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sega</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">A review of the genetic effects of ethyl methanesulfonate.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DSega%26rft.auinit1%253DG.%2BA.%26rft.volume%253D134%26rft.issue%253D2-3%26rft.spage%253D113%26rft.epage%253D142%26rft.atitle%253DA%2Breview%2Bof%2Bthe%2Bgenetic%2Beffects%2Bof%2Bethyl%2Bmethanesulfonate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0165-1218%252884%252990033-8%26rft_id%253Dinfo%253Apmid%252F6390190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0165-1218(84)90033-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6390190&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TX05100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.67" data-doi="10.1093/carcin/bgp262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shrivastav</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Essigmann</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Chemical biology of mutagenesis and DNA repair: cellular responses to DNA alkylation.</span> <abbr class="cit-jnl-abbrev">Carcinogenesis</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCarcinogenesis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcarcin%252Fbgp262%26rft_id%253Dinfo%253Apmid%252F19875697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/carcin/bgp262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19875697&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273493100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.68" data-doi="10.1073/pnas.75.4.1722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraenkel-Conrat</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuśmierek</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Preparation and template activities of polynucleotides containing O2- and O4-alkyluridine.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1722</span>–<span class="cit-lpage">1726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSinger%26rft.auinit1%253DB.%26rft.volume%253D75%26rft.issue%253D4%26rft.spage%253D1722%26rft.epage%253D1726%26rft.atitle%253DPreparation%2Band%2Btemplate%2Bactivities%2Bof%2Bpolynucleotides%2Bcontaining%2BO2-%2Band%2BO4-alkyluridine.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.75.4.1722%26rft_id%253Dinfo%253Apmid%252F273902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3NS80LzE3MjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzMvMTA3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.69" data-doi="10.1146/annurev.bi.51.070182.003255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuśmierek</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Chemical mutagenesis.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DSinger%26rft.auinit1%253DB.%26rft.volume%253D51%26rft.spage%253D655%26rft.epage%253D693%26rft.atitle%253DChemical%2Bmutagenesis.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.bi.51.070182.003255%26rft_id%253Dinfo%253Apmid%252F7051963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.bi.51.070182.003255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7051963&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982NW04100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.70" data-doi="10.1093/bioinformatics/btm098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzek</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGarvey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazumder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">UniRef: comprehensive and non-redundant UniProt reference clusters.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1282</span>–<span class="cit-lpage">1288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm098%26rft_id%253Dinfo%253Apmid%252F17379688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17379688&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247348300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.71" data-doi="10.1101/gr.2332504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swigoňová</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishna</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llaca</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Close split of sorghum and maize genome progenitors.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1916</span>–<span class="cit-lpage">1923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSwigonova%26rft.auinit1%253DZ.%26rft.volume%253D14%26rft.issue%253D10a%26rft.spage%253D1916%26rft.epage%253D1923%26rft.atitle%253DClose%2BSplit%2Bof%2BSorghum%2Band%2BMaize%2BGenome%2BProgenitors%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2332504%26rft_id%253Dinfo%253Apmid%252F15466289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjExOiIxNC8xMGEvMTkxNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvMy8xMDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.72" data-doi="10.1093/pcp/pcr029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uchida</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tasaka</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of EMS-induced causal mutations in a non-reference <em>Arabidopsis thaliana</em> accession by whole genome sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcr029%26rft_id%253Dinfo%253Apmid%252F21398646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/pcp/pcr029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21398646&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289310900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.73" data-doi="10.1093/nar/gkt1140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">UniProt Consortium</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Activities at the Universal Protein Resource (UniProt).</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D191</span>–<span class="cit-lpage">D198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1140%26rft_id%253Dinfo%253Apmid%252F24253303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24253303&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.74" data-doi="10.1016/j.tplants.2005.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waugh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leader</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Harvesting the potential of induced biological diversity.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DWaugh%26rft.auinit1%253DR.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D71%26rft.epage%253D79%26rft.atitle%253DHarvesting%2Bthe%2Bpotential%2Bof%2Binduced%2Bbiological%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2005.12.007%26rft_id%253Dinfo%253Apmid%252F16406304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2005.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16406304&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236080700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.75" data-doi="10.1007/BF02157427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westergaard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-article-title">Chemical mutagenesis in relation to the concept of the gene.</span> <abbr class="cit-jnl-abbrev">Experientia</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">224</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperientia%26rft.stitle%253DExperientia%26rft.aulast%253DWestergaard%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D224%26rft.epage%253D234%26rft.atitle%253DChemical%2Bmutagenesis%2Bin%2Brelation%2Bto%2Bthe%2Bconcept%2Bof%2Bthe%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02157427%26rft_id%253Dinfo%253Apmid%252F13447934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02157427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13447934&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holding</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messing</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutation in the seed storage protein kafirin creates a high-value food trait in sorghum.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2217%26rft_id%253Dinfo%253Apmid%252F23948869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23948869&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.3.1079.77" data-doi="10.1186/1746-4811-8-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high throughput DNA extraction method with high yield and quality</span>. <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BMethods%26rft.aulast%253DXin%26rft.auinit1%253DZ.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DA%2Bhigh%2Bthroughput%2BDNA%2Bextraction%2Bmethod%2Bwith%2Bhigh%2Byield%2Band%2Bquality.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-8-26%26rft_id%253Dinfo%253Apmid%252F22839646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-8-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22839646&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F1079.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.1079.78" data-doi="10.1007/s12155-008-9029-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An induced Sorghum mutant population suitable for bioenergy research.</span> <abbr class="cit-jnl-abbrev">BioEnergy Res.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BRes.%26rft.volume%253D2%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-008-9029-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-008-9029-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/3/1079.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/3/1067" title="Pleiotropic Functions of the Chromodomain-Containing Protein Hat-trick During Oogenesis in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/4/1095" title="Identification of Two Distinct Classes of the Human INO80 Complex Genome-Wide" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/3"><span>Volume 8 Issue 3, March 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/3.cover-source.jpg?itok=o8h2cMy2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/1079" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1Bis8aHevhoZZIVm9s6tLJO1i5Dxg1WEeQmBRkWHs9o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/3/1079" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/3/1079" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/3/1079" class="active" data-icon-position="" data-hide-link-title="0">Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488752" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ysEXNkLJnFENmzF_Rr7tJ2gmaJY7iNKwe7FphqgBBJg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/3/1079.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/3/1079" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-LqZfBdri29PoSrJsfUAau0utMUWiai1EaTrZ15itJco" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/3/1079" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/3/1079.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488752" id="node488752--2" data-pisa="ggg;8/3/1079" data-pisa-master="ggg;g3.117.300301" data-apath="/ggg/8/3/1079.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Addo-Quaye</span>, <span class="highwire-citation-author" data-delta="1">Mitch Tuinstra</span>, <span class="highwire-citation-author" data-delta="2">Nicola Carraro</span>, <span class="highwire-citation-author" data-delta="3">Clifford Weil</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1079-1094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300301 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488752--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles Addo-Quaye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BAddo-Quaye%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addo-Quaye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BAddo-Quaye%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Addo-Quaye</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mitch Tuinstra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitch%2BTuinstra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tuinstra%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMitch%2BTuinstra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicola Carraro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicola%2BCarraro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carraro%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicola%2BCarraro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Clifford Weil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clifford%2BWeil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weil%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClifford%2BWeil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/3/1079" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/3/1079&amp;title=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/3/1079&amp;title=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/3/1079&amp;title=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/3/1079&amp;text=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/3/1079&amp;title=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/3/1079&amp;t=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/3/1079&amp;title=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/3/1079&amp;title=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488752/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488752" id="node488752--3" data-pisa="ggg;8/3/1079" data-pisa-master="ggg;g3.117.300301" data-apath="/ggg/8/3/1079.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Addo-Quaye</span>, <span class="highwire-citation-author" data-delta="1">Mitch Tuinstra</span>, <span class="highwire-citation-author" data-delta="2">Nicola Carraro</span>, <span class="highwire-citation-author" data-delta="3">Clifford Weil</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1079-1094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300301 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488752--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles Addo-Quaye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BAddo-Quaye%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addo-Quaye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BAddo-Quaye%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Addo-Quaye</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mitch Tuinstra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitch%2BTuinstra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tuinstra%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMitch%2BTuinstra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicola Carraro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicola%2BCarraro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carraro%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicola%2BCarraro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Clifford Weil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clifford%2BWeil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weil%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClifford%2BWeil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1079&amp;rft.epage=1094&amp;rft.atitle=Whole-Genome%20Sequence%20Accuracy%20Is%20Improved%20by%20Replication%20in%20a%20Population%20of%20Mutagenized%20Sorghum&amp;rft.volume=8&amp;rft.issue=3&amp;rft.date=2018-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Addo-Quaye%2C+Charles&amp;rft.au=Tuinstra%2C+Mitch&amp;rft.au=Carraro%2C+Nicola&amp;rft.au=Weil%2C+Clifford&amp;rft.au=Dilkes%2C+Brian+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488752/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488752/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488752/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488752/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488752/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488752/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488752/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488752/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488752/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488752/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488752/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488752/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/3/1079" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-28" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/3/1079.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/3/1079.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/3/1079.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/3/1079.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
