<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/3/981" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/3/981.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/3/981.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/3/981.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/3/981.atom" />
<meta name="HW.pisa" content="ggg;8/3/981" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Identification and Functional Characterization of Essential Genes in Caenorhabditis elegans" />
<meta name="DC.Identifier" content="10.1534/g3.117.300338" />
<meta name="DC.Date" content="2018-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Qin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Using combined genetic mapping, Illumina sequencing, bioinformatics analyses, and experimental validation, we identified 60 essential genes from 104 lethal mutations in two genomic regions of Caenorhabditis elegans totaling ∼14 Mb on chromosome III(mid) and chromosome V(left). Five of the 60 genes had not previously been shown to have lethal phenotypes by RNA interference depletion. By analyzing the regions around the lethal missense mutations, we identified four putative new protein functional domains. Furthermore, functional characterization of the identified essential genes shows that most are enzymes, including helicases, tRNA synthetases, and kinases in addition to ribosomal proteins. Gene Ontology analysis indicated that essential genes often encode for enzymes that conduct nucleic acid binding activities during fundamental processes, such as intracellular DNA replication, transcription, and translation. Analysis of essential gene shows that they have fewer paralogs, encode proteins that are in protein interaction hubs, and are highly expressed relative to nonessential genes. All these essential gene traits in C. elegans are consistent with those of human disease genes. Most human orthologs (90%) of the essential genes in this study are related to human diseases. Therefore, functional characterization of essential genes underlines their importance as proxies for understanding the biological functions of human disease genes." />
<meta name="DC.Contributor" content="Zhaozhao Qin" />
<meta name="DC.Contributor" content="Robert Johnsen" />
<meta name="DC.Contributor" content="Shicheng Yu" />
<meta name="DC.Contributor" content="Jeffrey Shih-Chieh Chu" />
<meta name="DC.Contributor" content="David L. Baillie" />
<meta name="DC.Contributor" content="Nansheng Chen" />
<meta name="article:published_time" content="2018-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic Identification and Functional Characterization of Essential Genes in Caenorhabditis elegans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Using combined genetic mapping, Illumina sequencing, bioinformatics analyses, and experimental validation, we identified 60 essential genes from 104 lethal mutations in two genomic regions of &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; totaling ∼14 Mb on chromosome III(mid) and chromosome V(left). Five of the 60 genes had not previously been shown to have lethal phenotypes by RNA interference depletion. By analyzing the regions around the lethal missense mutations, we identified four putative new protein functional domains. Furthermore, functional characterization of the identified essential genes shows that most are enzymes, including helicases, tRNA synthetases, and kinases in addition to ribosomal proteins. Gene Ontology analysis indicated that essential genes often encode for enzymes that conduct nucleic acid binding activities during fundamental processes, such as intracellular DNA replication, transcription, and translation. Analysis of essential gene shows that they have fewer paralogs, encode proteins that are in protein interaction hubs, and are highly expressed relative to nonessential genes. All these essential gene traits in &lt;i&gt;C. elegans&lt;/i&gt; are consistent with those of human disease genes. Most human orthologs (90%) of the essential genes in this study are related to human diseases. Therefore, functional characterization of essential genes underlines their importance as proxies for understanding the biological functions of human disease genes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/03/01" />
<meta name="citation_mjid" content="ggg;8/3/981" />
<meta name="citation_id" content="8/3/981" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/3/981" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/3/981.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/3/981.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/3/981.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300338" />
<meta name="citation_pmid" content="29339407" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="981" />
<meta name="citation_lastpage" content="997" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhaozhao Qin" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada" />
<meta name="citation_author" content="Robert Johnsen" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada" />
<meta name="citation_author" content="Shicheng Yu" />
<meta name="citation_author_institution" content="Key Laboratory of Combinatorial Biosynthesis and Drug Discovery, School of Pharmaceutical Sciences, Wuhan University, 430071, China" />
<meta name="citation_author_institution" content="Wuhan Frasergen Bioinformatics, Wuhan East Lake High-Tech Zone, 430075, China" />
<meta name="citation_author" content="Jeffrey Shih-Chieh Chu" />
<meta name="citation_author_institution" content="Wuhan Frasergen Bioinformatics, Wuhan East Lake High-Tech Zone, 430075, China" />
<meta name="citation_author" content="David L. Baillie" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada" />
<meta name="citation_author" content="Nansheng Chen" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada" />
<meta name="citation_author_email" content="chenn@sfu.ca" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Amsterdam;citation_author=R. M. Nissen;citation_author=Z. Sun;citation_author=E. C. Swindell;citation_author=S. Farrington;citation_title=Identification of 315 genes essential for early zebrafish development.;citation_volume=101;citation_year=2004;citation_pmid=15256591;citation_doi=10.1073/pnas.0403929101" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=T. Andrews;citation_author=F. Honti;citation_author=R. Pfundt;citation_author=N. de Leeuw;citation_author=J. Hehir-Kwa;citation_title=The clustering of functionally related genes contributes to CNV-mediated disease.;citation_volume=25;citation_year=2015;citation_pmid=25887030;citation_doi=10.1101/gr.184325.114" />
<meta name="citation_reference" content="citation_journal_title=Leukemia;citation_author=J. Broséus;citation_author=T. Alpermann;citation_author=M. Wulfert;citation_author=L. Florensa Brichs;citation_author=S. Jeromin;citation_title=Age, JAK2(V617F) and SF3B1 mutations are the main predicting factors for survival in refractory anaemia with ring sideroblasts and marked thrombocytosis.;citation_pages=1826-1831;citation_volume=27;citation_year=2013;citation_pmid=23594705;citation_doi=10.1038/leu.2013.120" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. S. Chu;citation_author=R. C. Johnsen;citation_author=S. Y. Chua;citation_author=D. Tu;citation_author=M. Dennison;citation_title=Allelic ratios and the mutational landscape reveal biologically significant heterozygous SNVs.;citation_volume=190;citation_year=2012;citation_pmid=22267497;citation_doi=10.1534/genetics.111.137208" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. S. Chu;citation_author=S. Y. Chua;citation_author=K. Wong;citation_author=A. M. Davison;citation_author=R. Johnsen;citation_title=High-throughput capturing and characterization of mutations in essential genes of Caenorhabditis elegans.;citation_pages=361;citation_volume=15;citation_year=2014;citation_pmid=24884423;citation_doi=10.1186/1471-2164-15-361" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=D. V. Clark;citation_author=R. C. Johnsen;citation_author=K. S. McKim;citation_author=D. L. Baillie;citation_title=Analysis of lethal mutations induced in a mutator strain that activates transposable elements in Caenorhabditis elegans.;citation_pages=109-114;citation_volume=33;citation_year=1990;citation_issue=1;citation_pmid=2158925" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=G. C. Conant;citation_author=A. Wagner;citation_title=Duplicate genes and robustness to transient gene knock-downs in Caenorhabditis elegans;citation_pages=89-96;citation_volume=271;citation_year=2004;citation_issue=1534;citation_pmid=15002776;citation_doi=10.1098/rspb.2003.2560" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. W. Davis;citation_author=M. Hammarlund;citation_author=T. Harrach;citation_author=P. Hullett;citation_author=S. Olsen;citation_title=Rapid single nucleotide polymorphism mapping in C. elegans.;citation_pages=118-118;citation_volume=6;citation_year=2005;citation_pmid=16156901;citation_doi=10.1186/1471-2164-6-118" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=N. J. Dibb;citation_author=I. N. Maruyama;citation_author=M. Krause;citation_author=J. Karn;citation_title=Sequence analysis of the complete Caenorhabditis elegans myosin heavy chain gene family.;citation_pages=603-613;citation_volume=205;citation_year=1989;citation_issue=3;citation_pmid=2926820;citation_doi=10.1016/0022-2836(89)90229-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. E. Dickerson;citation_author=A. Zhu;citation_author=D. L. Robertson;citation_author=K. E. Hentges;citation_title=Defining the role of essential genes in human disease.;citation_pages=e27368-e27368;citation_volume=6;citation_year=2011;citation_issue=11;citation_pmid=22096564;citation_doi=10.1371/journal.pone.0027368" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=K. Dolinski;citation_author=D. Botstein;citation_title=Orthology and functional conservation in eukaryotes.;citation_pages=465-507;citation_volume=41;citation_year=2007;citation_pmid=17678444;citation_doi=10.1146/annurev.genet.40.110405.090439" />
<meta name="citation_reference" content="citation_author=L. G. Edgar;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genetic Maps;citation_author=M. L. Edgley;citation_author=D. L. Riddle;citation_title=The nematode Caenorhabditis elegans.;citation_pages=111-133;citation_volume=5;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=A. Bateman;citation_author=J. Clements;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_title=Pfam: the protein families database.;citation_volume=42;citation_year=2014;citation_pmid=24288371;citation_doi=10.1093/nar/gkt1223" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Flibotte;citation_author=M. L. Edgley;citation_author=I. Chaudhry;citation_author=J. Taylor;citation_author=S. E. Neil;citation_title=Whole-genome profiling of mutagenesis in Caenorhabditis elegans.;citation_volume=185;citation_year=2010;citation_pmid=20439774;citation_doi=10.1534/genetics.110.116616" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Frech;citation_author=C. Choo;citation_author=N. Chen;citation_title=FeatureStack: Perl module for comparative visualization of gene features.;citation_volume=28;citation_year=2012;citation_pmid=23023981;citation_doi=10.1093/bioinformatics/bts572" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=S. Gerdes;citation_author=R. Edwards;citation_author=M. Kubal;citation_author=M. Fonstein;citation_author=R. Stevens;citation_title=Essential genes on metabolic maps.;citation_pages=448-456;citation_volume=17;citation_year=2006;citation_issue=5;citation_pmid=16978855;citation_doi=10.1016/j.copbio.2006.08.006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. B. Gerstein;citation_author=Z. J. Lu;citation_author=E. L. Van Nostrand;citation_author=C. Cheng;citation_author=B. I. Arshinoff;citation_title=Integrative analysis of the Caenorhabditis elegans genome by the modENCODE project.;citation_volume=330;citation_year=2010;citation_pmid=21177976;citation_doi=10.1126/science.1196914" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. I. Goh;citation_author=M. E. Cusick;citation_author=D. Valle;citation_author=B. Childs;citation_author=M. Vidal;citation_title=The human disease network.;citation_volume=104;citation_year=2007;citation_pmid=17502601;citation_doi=10.1073/pnas.0701361104" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Z. Gu;citation_author=L. M. Steinmetz;citation_author=X. Gu;citation_author=C. Scharfe;citation_author=R. W. Davis;citation_title=Role of duplicate genes in genetic robustness against null mutations.;citation_pages=63-66;citation_volume=421;citation_year=2003;citation_issue=6918;citation_pmid=12511954;citation_doi=10.1038/nature01198" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=T. Hart;citation_author=K. R. Brown;citation_author=F. Sircoulomb;citation_author=R. Rottapel;citation_author=J. Moffat;citation_title=Measuring error rates in genomic perturbation screens: gold standards for human functional genomics;citation_pages=733-733;citation_volume=10;citation_year=2014;citation_issue=7;citation_pmid=24987113;citation_doi=10.15252/msb.20145216" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. W. Hillier;citation_author=V. Reinke;citation_author=P. Green;citation_author=M. Hirst;citation_author=M. A. Marra;citation_title=Massively parallel sequencing of the polyadenylated transcriptome of C. elegans.;citation_volume=19;citation_year=2009;citation_pmid=19181841;citation_doi=10.1101/gr.088112.108" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. Hutter;citation_author=M. P. Ng;citation_author=N. Chen;citation_title=GExplore: a web server for integrated queries of protein domains, gene expression and mutant phenotypes.;citation_pages=529-529;citation_volume=10;citation_year=2009;citation_pmid=19917126;citation_doi=10.1186/1471-2164-10-529" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. B. Lane;citation_author=R. D. Machado;citation_author=M. W. Pauciulo;citation_author=J. R. Thomson;citation_author=J. A. Phillips;citation_title=Heterozygous germline mutations in BMPR2, encoding a TGF-beta receptor, cause familial primary pulmonary hypertension.;citation_pages=81-84;citation_volume=26;citation_year=2000;citation_issue=1;citation_pmid=10973254;citation_doi=10.1038/79226" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=D. L. Janke;citation_author=J. E. Schein;citation_author=T. Ha;citation_author=N. W. Franz;citation_author=N. J. O’Neil;citation_title=Interpreting a Sequenced Genome: Toward a Cosmid Transgenic Library of Caenorhabditis elegans;citation_pages=974-985;citation_volume=7;citation_year=1997;citation_issue=10;citation_pmid=9331368;citation_doi=10.1101/gr.7.10.974" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Jeong;citation_author=S. P. Mason;citation_author=A. L. Barabasi;citation_author=Z. N. Oltvai;citation_title=Lethality and centrality in protein networks.;citation_pages=41-42;citation_volume=411;citation_year=2001;citation_issue=6833;citation_pmid=11333967;citation_doi=10.1038/35075138" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=R. C. Johnsen;citation_author=D. L. Baillie;citation_title=Formaldehyde mutagenesis of the eT1 balanced region in Caenorhabditis elegans: dose-response curve and the analysis of mutational events.;citation_pages=137-147;citation_volume=201;citation_year=1988;citation_issue=1;citation_pmid=3419443;citation_doi=10.1016/0027-5107(88)90120-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Johnsen;citation_author=D. L. Baillie;citation_title=Genetic analysis of a major segment [LGV(left)] of the genome of Caenorhabditis elegans.;citation_pages=735-752;citation_volume=129;citation_year=1991;citation_issue=3;citation_pmid=1752418" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. R. Jones;citation_author=J. S. Maydan;citation_author=S. Flibotte;citation_author=D. G. Moerman;citation_author=D. L. Baillie;citation_title=Oligonucleotide Array Comparative Genomic Hybridization (oaCGH) based characterization of genetic deficiencies as an aid to gene mapping in Caenorhabditis elegans.;citation_pages=402-402;citation_volume=8;citation_year=2007;citation_pmid=17986355;citation_doi=10.1186/1471-2164-8-402" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=M. R. Jones;citation_author=S. Y. Chua;citation_author=N. J. O’Neil;citation_author=R. C. Johnsen;citation_author=A. M. Rose;citation_title=High-resolution array comparative genomic hybridization analysis reveals unanticipated complexity of genetic deficiencies on chromosome V in Caenorhabditis elegans.;citation_pages=37-46;citation_volume=282;citation_year=2009;citation_issue=1;citation_pmid=19330515" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. S. Kamath;citation_author=A. G. Fraser;citation_author=Y. Dong;citation_author=G. Poulin;citation_author=R. Durbin;citation_title=Systematic functional analysis of the Caenorhabditis elegans genome using RNAi.;citation_pages=231-237;citation_volume=421;citation_year=2003;citation_issue=6920;citation_pmid=12529635;citation_doi=10.1038/nature01278" />
<meta name="citation_reference" content="citation_journal_title=Clin. Genet.;citation_author=T. Kanbayashi;citation_author=F. Saito;citation_author=T. Matsukawa;citation_author=H. Oba;citation_author=K. Hokkoku;citation_title=Adult-onset vanishing white matter disease with novel missense mutations in a subunit of translational regulator, EIF2B4.;citation_pages=401-403;citation_volume=88;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=E. V. Koonin;citation_title=HOW MANY GENES CAN MAKE A CELL: The Minimal-Gene-Set Concept;citation_pages=99-116;citation_volume=1;citation_year=2000;citation_issue=1;citation_pmid=11701626;citation_doi=10.1146/annurev.genom.1.1.99" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Koressaar;citation_author=M. Remm;citation_title=Enhancements and modifications of primer design program Primer3.;citation_volume=23;citation_year=2007;citation_pmid=17379693;citation_doi=10.1093/bioinformatics/btm091" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. M. Kramer;citation_author=R. P. French;citation_author=E. C. Park;citation_author=J. J. Johnson;citation_title=The Caenorhabditis elegans rol-6 gene, which interacts with the sqt-1 collagen gene to determine organismal morphology, encodes a collagen.;citation_pages=2081-2089;citation_volume=10;citation_year=1990;citation_issue=5;citation_pmid=1970117;citation_doi=10.1128/MCB.10.5.2081" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Larkin;citation_author=G. Blackshields;citation_author=N. P. Brown;citation_author=R. Chenna;citation_author=P. A. McGettigan;citation_title=Clustal W and Clustal X version 2.0.;citation_volume=23;citation_year=2007;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=H. Liang;citation_author=W. H. Li;citation_title=Gene essentiality, gene duplicability and protein connectivity in human and mouse.;citation_pages=375-378;citation_volume=23;citation_year=2007;citation_issue=8;citation_pmid=17512629;citation_doi=10.1016/j.tig.2007.04.005" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=B. Y. Liao;citation_author=J. Zhang;citation_title=Mouse duplicate genes are as essential as singletons.;citation_pages=378-381;citation_volume=23;citation_year=2007;citation_issue=8;citation_pmid=17559966;citation_doi=10.1016/j.tig.2007.05.006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. Y. Liao;citation_author=N. M. Scott;citation_author=J. Zhang;citation_title=Impacts of gene essentiality, expression pattern, and gene compactness on the evolutionary rate of mammalian proteins.;citation_volume=23;citation_year=2006;citation_pmid=16887903;citation_doi=10.1093/molbev/msl076" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Maciejowski;citation_author=J. H. Ahn;citation_author=P. G. Cipriani;citation_author=D. J. Killian;citation_author=A. L. Chaudhary;citation_title=Autosomal genes of autosomal/X-linked duplicated gene pairs and germ-line proliferation in Caenorhabditis elegans.;citation_volume=169;citation_year=2005;citation_pmid=15687263;citation_doi=10.1534/genetics.104.040121" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. S. Maser;citation_author=R. Zinkel;citation_author=J. H. Petrini;citation_title=An alternative mode of translation permits production of a variant NBS1 protein from the common Nijmegen breakage syndrome allele.;citation_pages=417-421;citation_volume=27;citation_year=2001;citation_issue=4;citation_pmid=11279524;citation_doi=10.1038/86920" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=C. C. Mello;citation_author=J. M. Kramer;citation_author=D. Stinchcomb;citation_author=V. Ambros;citation_title=Efficient gene transfer in C.elegans: extrachromosomal maintenance and integration of transforming sequences.;citation_pages=3959-3970;citation_volume=10;citation_year=1991;citation_issue=12;citation_pmid=1935914" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=A. G. Mendonça;citation_author=R. J. Alves;citation_author=J. B. Pereira-Leal;citation_title=Loss of genetic redundancy in reductive genome evolution.;citation_pages=e1001082-e1001082;citation_volume=7;citation_year=2011;citation_issue=2;citation_pmid=21379323;citation_doi=10.1371/journal.pcbi.1001082" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=H. Mi;citation_author=A. Muruganujan;citation_author=J. T. Casagrande;citation_author=P. D. Thomas;citation_title=Large-scale gene function analysis with the PANTHER classification system.;citation_pages=1551-1566;citation_volume=8;citation_year=2013;citation_pmid=23868073;citation_doi=10.1038/nprot.2013.092" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. A. Mok;citation_author=V. Au;citation_author=O. A. Thompson;citation_author=M. L. Edgley;citation_author=L. Gevirtzman;citation_title=MIP-MAP: high-throughput mapping of Caenorhabditis elegans temperature-sensitive mutants via molecular inversion probes.;citation_volume=207;citation_year=2017;citation_pmid=28827289;citation_doi=10.1534/genetics.117.300179" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=N. Mulder;citation_author=R. Apweiler;citation_title=InterPro and InterProScan: tools for protein sequence classification and comparison.;citation_pages=59-70;citation_volume=396;citation_year=2007;citation_pmid=18025686;citation_doi=10.1007/978-1-59745-515-2_5" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. A. Nowak;citation_author=M. C. Boerlijst;citation_author=J. Cooke;citation_author=J. M. Smith;citation_title=Evolution of genetic redundancy.;citation_pages=167-171;citation_volume=388;citation_year=1997;citation_issue=6638;citation_pmid=9217155;citation_doi=10.1038/40618" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=E. Papaemmanuil;citation_author=M. Cazzola;citation_author=J. Boultwood;citation_author=L. Malcovati;citation_author=P. Vyas;citation_title=Somatic SF3B1 mutation in myelodysplasia with ring sideroblasts.;citation_pages=1384-1395;citation_volume=365;citation_year=2011;citation_issue=15;citation_pmid=21995386;citation_doi=10.1056/NEJMoa1103283" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Parkinson;citation_author=M. Mitreva;citation_author=C. Whitton;citation_author=M. Thomson;citation_author=J. Daub;citation_title=A transcriptomic analysis of the phylum Nematoda.;citation_pages=1259-1267;citation_volume=36;citation_year=2004;citation_issue=12;citation_pmid=15543149;citation_doi=10.1038/ng1472" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. P. Phillips;citation_author=J. A. Tainer;citation_author=E. D. Getzoff;citation_author=G. L. Boulianne;citation_author=K. Kirby;citation_title=Subunit-destabilizing mutations in Drosophila copper/zinc superoxide dismutase: neuropathology and a model of dimer dysequilibrium.;citation_pages=8574-8578;citation_volume=92;citation_year=1995;citation_issue=19;citation_pmid=7567977;citation_doi=10.1073/pnas.92.19.8574" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. K. Ragge;citation_author=A. G. Brown;citation_author=C. M. Poloschek;citation_author=B. Lorenz;citation_author=R. A. Henderson;citation_title=Heterozygous mutations of OTX2 cause severe ocular malformations.;citation_pages=1008-1022;citation_volume=76;citation_year=2005;citation_issue=6;citation_pmid=15846561;citation_doi=10.1086/430721" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Remm;citation_author=C. E. Storm;citation_author=E. L. Sonnhammer;citation_title=Automatic clustering of orthologs and in-paralogs from pairwise species comparisons.;citation_pages=1041-1052;citation_volume=314;citation_year=2001;citation_issue=5;citation_pmid=11743721;citation_doi=10.1006/jmbi.2000.5197" />
<meta name="citation_reference" content="citation_journal_title=Biochem. J.;citation_author=T. M. Ritty;citation_author=T. J. Broekelmann;citation_author=C. C. Werneck;citation_author=R. P. Mecham;citation_title=Fibrillin-1 and -2 contain heparin-binding sites important for matrix deposition and that support cell attachment.;citation_volume=375;citation_year=2003;citation_pmid=12837131;citation_doi=10.1042/bj20030649" />
<meta name="citation_reference" content="citation_journal_title=WormBook;citation_author=H. M. Robertson;citation_author=J. H. Thomas;citation_title=The putative chemoreceptor families of C. elegans.;citation_pages=1-12;citation_volume=6;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. M. Rose;citation_author=N. J. O’Neil;citation_author=M. Bilenky;citation_author=Y. S. Butterfield;citation_author=N. Malhis;citation_title=Genomic sequence of a mutant strain of Caenorhabditis elegans with an altered recombination pattern.;citation_pages=131-131;citation_volume=11;citation_year=2010;citation_pmid=20178641;citation_doi=10.1186/1471-2164-11-131" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=R. E. Rosenbluth;citation_author=D. L. Baillie;citation_title=The genetic analysis of a reciprocal translocation, eT1(III; V), in Caenorhabditis elegans.;citation_pages=415-428;citation_volume=99;citation_year=1981" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Fundam. Mol. Mech. Mutagen.;citation_author=R. E. Rosenbluth;citation_author=C. Cuddeford;citation_author=D. L. Baillie;citation_title=Mutagenesis in Caenorhabditis elegans. I. A rapid eukaryotic mutagen test system using the reciprocal translocation eTI(III;V).;citation_pages=39-48;citation_volume=110;citation_year=1983;citation_doi=10.1016/0027-5107(83)90016-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. E. Rosenbluth;citation_author=C. Cuddeford;citation_author=D. L. Baillie;citation_title=MUTAGENESIS IN CAENORHABDITIS ELEGANS. II. A SPECTRUM OF MUTATIONAL EVENTS INDUCED WITH 1500 R OF gamma-RADIATION;citation_pages=493-511;citation_volume=109;citation_year=1985;citation_issue=3;citation_pmid=3979812" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=R. E. Rosenbluth;citation_author=T. M. Rogalski;citation_author=R. C. Johnsen;citation_author=L. M. Addison;citation_author=D. L. Baillie;citation_title=Genomic organization in Caenorhabditis elegans: deficiency mapping on linkage group V(left).;citation_pages=105-118;citation_volume=52;citation_year=1988;citation_doi=10.1017/S0016672300027476" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Sarin;citation_author=S. Prabhu;citation_author=M. M. O’Meara;citation_author=I. Pe’er;citation_author=O. Hobert;citation_title=Caenorhabditis elegans mutant allele identification by whole-genome sequencing.;citation_pages=865-867;citation_volume=5;citation_year=2008;citation_issue=10;citation_pmid=18677319;citation_doi=10.1038/nmeth.1249" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=D. Schnabel;citation_author=M. Schroder;citation_author=W. Furst;citation_author=A. Klein;citation_author=R. Hurwitz;citation_title=Simultaneous deficiency of sphingolipid activator proteins 1 and 2 is caused by a mutation in the initiation codon of their common gene;citation_pages=3312-3315;citation_volume=267;citation_year=1992;citation_issue=5;citation_pmid=1371116" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=C. Stark;citation_author=B. J. Breitkreutz;citation_author=T. Reguly;citation_author=L. Boucher;citation_author=A. Breitkreutz;citation_title=BioGRID: a general repository for interaction datasets;citation_pages=D535-D539;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381927;citation_doi=10.1093/nar/gkj109" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=H. I. Stewart;citation_author=R. E. Rosenbluth;citation_author=D. L. Baillie;citation_title=Most ultraviolet irradiation induced mutations in the nematode Caenorhabditis elegans are chromosomal rearrangements.;citation_pages=37-54;citation_volume=249;citation_year=1991;citation_issue=1;citation_pmid=2067542;citation_doi=10.1016/0165-1218(91)90107-W" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=H. I. Stewart;citation_author=N. J. O’Neil;citation_author=D. L. Janke;citation_author=N. W. Franz;citation_author=H. M. Chamberlin;citation_title=Lethal mutations defining 112 complementation groups in a 4.5 Mb sequenced region of Caenorhabditis elegans chromosome III.;citation_pages=280-288;citation_volume=260;citation_year=1998;citation_issue=2-3;citation_pmid=9862482" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. T. Thomas;citation_author=M. W. Kilpatrick;citation_author=K. Lin;citation_author=L. Erlacher;citation_author=P. Lembessis;citation_title=Disruption of human limb morphogenesis by a dominant negative mutation in CDMP1.;citation_pages=58-64;citation_volume=17;citation_year=1997;citation_issue=1;citation_pmid=9288098;citation_doi=10.1038/ng0997-58" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. D. Thompson;citation_author=T. J. Gibson;citation_author=F. Plewniak;citation_author=F. Jeanmougin;citation_author=D. G. Higgins;citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.;citation_volume=25;citation_year=1997;citation_pmid=9396791;citation_doi=10.1093/nar/25.24.4876" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=O. Thompson;citation_author=M. Edgley;citation_author=P. Strasbourger;citation_author=S. Flibotte;citation_author=B. Ewing;citation_title=The million mutation project: a new approach to genetics in Caenorhabditis elegans.;citation_volume=23;citation_year=2013;citation_pmid=23800452;citation_doi=10.1101/gr.157651.113" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdottir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergasser;citation_author=I. Cutcutache;citation_author=T. Koressaar;citation_author=J. Ye;citation_author=B. C. Faircloth;citation_title=Primer3–new capabilities and interfaces.;citation_volume=40;citation_year=2012;citation_pmid=22730293;citation_doi=10.1093/nar/gks596" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=I. A. Vergara;citation_author=C. Frech;citation_author=N. Chen;citation_title=CooVar: Co-occurring variant analyzer.;citation_pages=615-615;citation_volume=5;citation_year=2012;citation_issue=1;citation_pmid=23116482;citation_doi=10.1186/1756-0500-5-615" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. J. Verster;citation_author=E. B. Styles;citation_author=A. Mateo;citation_author=W. B. Derry;citation_author=B. J. Andrews;citation_title=Taxonomically restricted genes with essential functions frequently play roles in chromosome segregation in Caenorhabditis elegans and Saccharomyces cerevisiae.;citation_volume=7;citation_year=2017;citation_pmid=28839119;citation_doi=10.1534/g3.117.300193" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Wagner;citation_title=Robustness against mutations in genetic networks of yeast.;citation_pages=355-361;citation_volume=24;citation_year=2000;citation_issue=4;citation_pmid=10742097;citation_doi=10.1038/74174" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=A. H. Yona;citation_author=I. Frumkin;citation_author=Y. Pilpel;citation_title=A relay race on the evolutionary adaptation spectrum.;citation_pages=549-559;citation_volume=163;citation_year=2015;citation_pmid=26496602;citation_doi=10.1016/j.cell.2015.10.005" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Identification and Functional Characterization of Essential Genes in Caenorhabditis elegans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
<meta name="twitter:description" content="Using combined genetic mapping, Illumina sequencing, bioinformatics analyses, and experimental validation, we identified 60 essential genes from 104 lethal mutations in two genomic regions of Caenorhabditis elegans totaling ∼14 Mb on chromosome III(mid) and chromosome V(left). Five of the 60 genes had not previously been shown to have lethal phenotypes by RNA interference depletion. By analyzing the regions around the lethal missense mutations, we identified four putative new protein functional domains. Furthermore, functional characterization of the identified essential genes shows that most are enzymes, including helicases, tRNA synthetases, and kinases in addition to ribosomal proteins. Gene Ontology analysis indicated that essential genes often encode for enzymes that conduct nucleic acid binding activities during fundamental processes, such as intracellular DNA replication, transcription, and translation. Analysis of essential gene shows that they have fewer paralogs, encode proteins that are in protein interaction hubs, and are highly expressed relative to nonessential genes. All these essential gene traits in C. elegans are consistent with those of human disease genes. Most human orthologs (90%) of the essential genes in this study are related to human diseases. Therefore, functional characterization of essential genes underlines their importance as proxies for understanding the biological functions of human disease genes." />
<meta name="og-title" property="og:title" content="Genomic Identification and Functional Characterization of Essential Genes in Caenorhabditis elegans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/3/981" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Using combined genetic mapping, Illumina sequencing, bioinformatics analyses, and experimental validation, we identified 60 essential genes from 104 lethal mutations in two genomic regions of Caenorhabditis elegans totaling ∼14 Mb on chromosome III(mid) and chromosome V(left). Five of the 60 genes had not previously been shown to have lethal phenotypes by RNA interference depletion. By analyzing the regions around the lethal missense mutations, we identified four putative new protein functional domains. Furthermore, functional characterization of the identified essential genes shows that most are enzymes, including helicases, tRNA synthetases, and kinases in addition to ribosomal proteins. Gene Ontology analysis indicated that essential genes often encode for enzymes that conduct nucleic acid binding activities during fundamental processes, such as intracellular DNA replication, transcription, and translation. Analysis of essential gene shows that they have fewer paralogs, encode proteins that are in protein interaction hubs, and are highly expressed relative to nonessential genes. All these essential gene traits in C. elegans are consistent with those of human disease genes. Most human orthologs (90%) of the essential genes in this study are related to human diseases. Therefore, functional characterization of essential genes underlines their importance as proxies for understanding the biological functions of human disease genes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/3.cover-source.jpg" />
    <title>Genomic Identification and Functional Characterization of Essential Genes in Caenorhabditis elegans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8VtuqIzUJ2DucbMnUCmvMyxvXZGMx1nLrfCl4EPBfJs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488772","apath":"\/ggg\/8\/3\/981.atom","pisa":"ggg;8\/3\/981","ac":{"\/ggg\/8\/3\/981.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/3\/981.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/3\/981"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/3\/981"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/3\/981.full":true,"\/content\/8\/3\/981":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HpHt4GpDB4Pa64oMo51sYoQO1RIu89Xjb3RkYNxR+EWqPMLteUAoIJiSFrNq4JN\\\\\\\/HwKp7osYKi5C+mab20kQ5BK4bZjYPtw5Hya+VmDB\\\\\\\/bymvEUAUZczxdPRqdNj3Tm1Q8nE6c5NF7b4itmwfCOU71Pb5\\\\\\\/yiYNerNnMFqkI6ICZVHEgvIh2hM+c5FSox0F735FPZ49YWR9CGaf4WDJ2JkeZfTDPblAOzev0CwCAkoAjuMfVMlHHnuYVRGglg7fElsgZ3FSTEmx3WfJ3TD+gPFgRzLDvS6Q6Yneu3zZN6N+DwKpeYwvFODcdI8eL4rCRbRWNYUimNWKsFnb0m78sbyM9b9ebt\\\\\\\/klm99wTpOcGTzRytTGoF2Tks27+tAVafDHGq1ymyE73G3njj0Bzzd45M3wkQy09JNDeBVh9\\\\\\\/\\\\\\\/iYG68lKA4\\\\\\\/ZRAm9XBLYzMeJ1j6Zn\\\\\\\/MwLyyvfR4oesBQMNwzZXWtEu+8NiAdBDqRNM\\\\\\\/RFSIFlr8Syh2\\\\\\\/Crp2d1f3VnPIPXI+reNYWJAd7N3QNMYbV3xh3BFqDrIleklHsdxBs0rXtLCR1IsS\\\\\\\/Ho3HwjZH88l8eNa1FD4fc6LP+a9H3ZWNRazaq8UvhQnBZ8Sg3U5XxRWVO3ZOjMMjDMzWry3mMwpkwDG71GSQxcgYDsa2K2Irgj3HvNOffU4LUUggrXonmlPB4HEJLjnv1yojIKrOK2DMVdacQEzBFcMdq6OmKVnt3qYgyHKQnQBh0GxuKEx9eZ4tsUQHSC4IzhHcdIqCrmwOxE1YkDDRAgGSUYzHvD+WN2QZbwaLvRWpcYyaT\\\\\\\/8El5oOkKfjEdOqezFDmyH1KlkCgZTCRHjEJuzey9wTiltItY6E7COp261LuYA6Zndz5\\\\\\\/oQS84dYmG3xae70MbQXW4RH0O3ywAM71FkKZ1fntf8pRbIhkvJtj+7DvTIOsZ1bKGThTSviJrqTnqYf+SHD5an3kiIIijOy0UUeKdqjEO8sac\\\\\\\/+vshXoN7Ex+ghXNJ+9Ulb9\\\\\\\/vd8nBxJc9PS5lj35NbViZJPB\\\\\\\/lAUOWYlwbrGDt3s3lmFFeyEMFkTihooTE\\\\\\\/nZmBStHiYSP3TRBEMiw5h0Ir\\\\\\\/E4JQG6aOmEMrV70Z\\\\\\\/B6o2op4ttpNFkEcttbP1lwARLXRDDlDS2tFOIWHKWtHzsvvZzWNdj+ztlrV0dbbOtlvvAraMPXGY0gDteoS7xLt7ckDUWCXq5ypxr8SZPAjZM\\\\\\\/o2cBjaXWsjLswAjioXmgT2hUR1\\\\\\\/fvXeahvYhrk6zmAYFd+x8E9JJ\\\\\\\/Kz2zaDg\\\u0022,\\\u0022iv\\\u0022:\\\u0022VvDqxULFfyU+CBd8FpvMaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002294d99154b646824723b039a478975f06eb4048ed8ff00bb438cfbfd0e98589dd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jFgn78Cm0OP\\\\\\\/l98AF0xV58mlDvJ7Y850Qbt8cid4twENgM1Rk5YG3hDae6u\\\\\\\/ll+N\\\\\\\/yUSEyy3MrfM3hbbau8to8RFjLTt2mWuQ\\\\\\\/642O\\\\\\\/R+3AbBqiYW0BiX0hwSF3q9qL4bBZ+bgyu\\\\\\\/HvxghZuwcU0ZbnfPJcigtQITkJtwzaZFVf9uoCbOgqSDNxyLSQXfQJmWCdlimJE+WfkUWftxmjyZP86fZVcbkmeumsMn9bSJmCcyxYkRjxRe+gieGwUVi2+bBuGpJi2QBqLUPqozPXz4veT89V3zG9HAidXyLhHJOd\\\\\\\/1nrhFqTSp5POixaUpX5IhrdSAm+LG0jVzCziR4ELnqjV2Ru\\\\\\\/gISkgzbdKA\\\\\\\/MIJ6N0gub2LHDgF5igXgw4uCc+ol4Awepdv4tdZND7L2obSiiwRB5avduP26ZSPWtxoT+dJ1F02Q1rBvSQPLopSqGnvO4SDFbopYKKFJ+tMQt9CShJ9U7nhNt9+jo3y+ZMCWMJHTOX8O3Gt0Sv7ziN9VEnyj3oUiNNbqpQqewi6UZKq\\\\\\\/ogyPEzbdBNAAu7c5n2RIE9JEWKi9V1jPw734hsR9Lw7SvI6lX9ZY2KzUyxLE+x2TyJoydijD8sIX3AisXX9DlBT7dKEsgCcSHJimjJafwNITv18OnQHub9GVXBnsnToUALwNSHbPbgOhw2itWTYniLV2kfan4tV\\\\\\\/cp9DdvQkZlkymzxBrqbyl0FGnQJ3fcLV3lM7vDj\\\\\\\/ave7SKygPTE+5dANuIzIKk\\\\\\\/ReYW5FPb5zw\\\\\\\/zmEH5rYL+gY0JpvfkO1IV+PzrwJhKd4ElEVhraBA+j9tyBqnnADc6RYfVVyhdB2plCAxI0B7PbrPvMaW+ZB8TRSeIo0GEoydiOwOkFX38aD0aZHGxjld1Ml\\\\\\\/kdGdTSTFRFomcje+Uv9oc2mHiEJwChgrOZkld22wk3vMSCEv3SfKC37FElTavHQTFbGf4hB+woOBxB9WiTF5JlkKyxJKQ9D0CJQ1+QdIruhU2tOebMzyGHGmsxAD0EtbEFilKRX1AchLytXGyZp0n5Nu7kpJMcC1GAqlhh8xz6+Z6MMLWlNbq+7zheggAaEw6YQUKNVMpHlH3aCGQr3nenR\\\\\\\/N45MXBE5GkdLAoRX+rdibbcPyiWGpsfdeZ\\\\\\\/ej7VpGJaAZCBR4y7FtywoRz9Rez7dtn5Fwt\\\\\\\/xkaCZTgw0cR9t\\\\\\\/TSYOdjiHabXgm4dks+S1mHsUlMI+TxBXhA9LDwkWayp4JG8dgyUnyL1rqwbhNCyd43\\\\\\\/yrCjtbrTmcvqY4R3s\\\\\\\/V5Hh32IwaR\\\\\\\/a7fsqVppaaPTg3PLWOJM+ylLVk\\\\\\\/IMlkeJfcsAWxv0IvKMyVIzKKalB244K6nucHQZ4w19Aaal1+uMsSxLsI88tpUy\\\\\\\/6EHYtauiHNEkjsAK8FagpzMyv7ZpKoeNr7qxLc7fjbwFFEJji+Mvm\\\\\\\/Ew2uEuL4v9b5DcwLCyjNurQXlx1sI99HDh6TKcRFTp5ee7CJW1U7iHRnwoGr75CBNr1WsmB3DTWRiuO5\\\\\\\/W5CVtraJZelnsHXxoUE85TkhEpN7\\\\\\\/52EbHTVeHWMW3E1Nakdk7k6kT8GkkpR6BRZr7nQvz51ApMod25xt\\\\\\\/\\\\\\\/Q5BVQFbWoxPMPWlr7ofH3Bs3BzUWvChCmVn97NZCopvIknditZPbjtGKvor+8VSl1IJKNGKIfVedK6Xy2OUgcOw\\\\\\\/A8MZHHzO9twC\\\\\\\/cDoWpmD+horKJUhbiHooyjZKrEH9Q0u9XtKbBQWl2oEp3WFjO6Plhe5EpxKiA6gfTUpc7KfFUjkoBio0tBJOMQB92haEJC0afJjVi8Ba7DhOm68HBhUPWSaSPgUBxxPkUpKtJdktD6oYiB9RXEF8iS7AHVh9ScaYI+QNac\\\\\\\/QzxPYMS3bpTauL1Ck7anwNA5eqG0g1\\\\\\\/CcdkJgJfQjIG+ow0UPNkjam5NTQW1Nwdl38sVtvLbL1ejlUEIWobvArXKPwPxNTOXusBmQ7qaHtv8vvKUEcn2clUDDbP0YDgbk64oNI+o5WOF1cNp\\\\\\\/HybsNnsYcnPERg1c3IqNgNy8UQEMqQH9TL7OHanCrVioik0iFku61hE30p6u4ObV09etlGFVgodK45Tzx9maHG05KaPiltvmLo4TfaXZezzpjdqWnbyaiyVa+uemjaU9kWSmfXqKrBk1+vgpvgEhlesmOJjh5J5OAu9bByRgfnfRN3xx0qv\\\\\\\/gJ3lMqxLhCu79ymq+JOth\\\\\\\/xSv9Mam04d6iQs7LAGvXx0EBh40MzDBeie3IzIiNMWapQZkzqCDhYpji91pkKEPt4+GlULJljGIind3MUe+0WvhSF6SGOo3bOKVowJgumpNAof7MUVn2yCUlhmf3TV9Ukry5JswTgGXKeOqJu8YP4B3GCs35srqTtkknJYCWM3AqgMwG6NjsMXFeICQaG43aULFMl4CV2oT9xDu8E7Neoc8GEucV30sXtwnFXQQKruL1+1R0H3c5nVNcWYBf\\\\\\\/gHaAUhSciHEapwt62qnwn0AjnGP5KS1ptnQIamxa5IKri58b4aVMuI3nMQU2zOSatYhrufTlbi1voJKraHMocs6F7jKXepwOV3dhwUMTTIZhOYofJJSdGHx++uwOLcM7QZ53YWk\\\\\\\/to+enDJDNSHXAizch7JAVntJVx9o+gQ9Y92cLAkjnTp+kmA97sbmlAcDrTF\\\\\\\/vZxlTXk2+RdkikotSpD+a8YTeeCD8E\\\\\\\/AZXl47XN5kI0R2tKA\\\\\\\/Nd3YoBoIk8tbQUXVPSV+ufAbyaxDBPLGvGt0r3u5VOeN4yaaBXFy8PZ35Y00dBpl49AmyrFdxk+dFQoFDVxsosnZpWdA0QfsPtXFdSf+QTJPrn7\\\\\\\/QrwbxNj\\\\\\\/aoaRVwlSngFoddCinsGfbVOgTQlR8b7H2KMFW5XfcZftZgLJyv5X9C5NeYoep9ugVO3nnFOCh6WTJ9U9RrUOBAGnzUwwuqaVOCGKOKtoskzbjCI+fahlNdLwl550jSMyYywRteCEf4fm9xgaib2UmZc2RmB2UQI8JQ72feGP9QW\\\\\\\/VH47UB+VAWtbZ3PTdWbTSoZyYtQL4QUF2OnlO5buNfFnYaNqQL7lHKUVi1ue4NhZdYK46uUD3nRMuuoEGrEMmDxqSUMHjTrq4EKyhEQdDfVxrx+lXFFU84vAQkqLgelVb6MQc7cK23m7EF2nJX59VEkjpi2zh+KAaBAwwZeqo\\\\\\\/iwSNCJ20WSUMWjhXYepIMeBswc1qT\\\\\\\/krGSLti4tn38aywD+n0M3Hf3kGIA9Gx\\\\\\\/5+cX4Pow7oIkL\\\\\\\/7gKGzZmqMXHJ\\\\\\\/bmn1LaRqabGBMaWaLQiXfJCl4ZNFsRjkfEAGnrwGTyuXJ2uus1qVe64W2kVCyedoCP3KmDnPvOH5elsKclkPUguprKGDltJ7sC6SpzncNkTnUvNMc3gFIUdukBiktIZeuMeNEShE3W6UmHBewZwCByKMdfqx1hUiwy6Y6vyuBlfNIsV32u1f+cu0Z+d+0Z1\\\\\\\/92rQ\\\\\\\/uAePooaOeZg5F1b4ajhKV4qqjoPuDsGxKmIi2Y+7iynPeUKFJ+VR4pWciKWh2hP9ZmwS7N3TR29kS6mHmmXBbav7g8Evkq+HublnJVpKLgwhPWN\\\\\\\/oZ05CXTAhJp97lgW4Ws0E0MUl++DDZoSqpGA76aq0s56bzdRAotCNKLr38E3c+Asdj23d2stELKvoQX3nHSMtDRgsBfFcWyjxEQDPfjPBtR6infPu\\\\\\\/GeW+qaQJeXKVvLNt4EE+QJgcemHY\\\\\\\/EyIiiaVHP3fwPncIaTiK5DLOLD4dxTK42nCceKf4MQ34WBETZ9kYiAqEZyyWXvmzvJO+GovhP7xVY2jDGv336bt5VnoROSnE7g91qGEv1rDz5trpEXBI7viigR1w\\\\\\\/D0LgHuo8LHkYzltffLSwQJi66ErzOiDyVDzwy9Ta3xz5EfD6ZI4VhaktKHxV9MExXgMgpsj8P64WbDAvG6jIjNOgsTOvuFC9s6NSijrt5crGEy+ZKPRR5puYrS38npXnHWEfcLEIWBhexQzbIcHlm86ywoTMYV9uO1qGQOoOb6FABdpRgxKtgD9bkVCgnYhvUHEuDmAQPw29V8COcSoM\\\\\\\/XQAlTMbFy4HdXf9mMxGQOX7L8EFzkmtsfG5GU1B4De9sJnuocCC+bk5G+6G4vu1MGLLUeKd89AmVj+8Cdcig8\\\\\\\/BdS2KoQv5q5rmnCTwYYHMbzfaoRo8WOgqeksVVD34y+O3QVmtI6M7sxNekv2lKXMXOyp2mkfQU4HaEb+h64inqy2zrzU0ATRLWkdEVkI\\\\\\\/djxDpxVv6yI6GZ5m3wwAXHNtvZre\\\\\\\/kIQgiya1vOhDbpabsccTkjv7D0I931jglzTNt08HJW2RqVQxjk+ISZ1YI28ybn6JxEA\\\\\\\/eQ7GWhXJs8hwqg8c9o0Lg\\\\\\\/2C198akBA9TZaMhb5wDAxcWsuj7cZcVXEsRlcUug2YIxD\\\\\\\/2JyiTw4HeIS9OMPOrm7q9dSPd9EYQNVUAA6ATJeZC9JGCrnnOuJ7gbeE1b6iD\\\\\\\/zfU8YdmJa03O0eTgsw+ih9mHBXH5FwZvfFt\\\\\\\/40CRSdDN5Suo4Q9goLMT\\\\\\\/ns8B\\\\\\\/VFt4C8ZJdPnAXYVZZ0r2HVRETm2CkGktMj4F4ov0lEeXuS4\\\\\\\/QPu4LgS1MNMvxzMUl1z6gbXfSUanv57+BbI5rjyTXgPwonTe\\\\\\\/rPhJj62ck4eZZpVvDWOEUg1z6yy9Unnq8b5dBZz5yvRDpg+9YgftLdMRcxPW+QJIxGK6ByEriumWkZuIjJN1a81kkwOdz5bVc8jzDW2siTfwRNdSVWQlTqVFeyuptGtdAQl1uLO+TwR75u7QlWzLLCor1AVp9UwDoFSgzlFnTtNgCZsYIqgaj9mE06PZHiFETiaJgDYBDb+QVseWuYC9gt7GRoJU\\\\\\\/sRxr5gZH6jh1zJAlBVUW1AeGcn1\\\\\\\/RyGuDl6E6jSpE5f7nUE+T1IW+lnUCo3e2WmYX23Fj\\\\\\\/hi1T8ycY3IPJbghanCCWkf6VTM2MMOGTC+jGUaGBitRYBGWoQ2GOQ4o1fXPxfSpBWEJ9FkjsWPMjdY5qifmAVi6JZ7K5mNsd9EM8sidxiFZT7VQvqkZIAlPVaqKCQhs+yLU2ZAPRpFqdkvP\\\\\\\/5KiU60MZxlED+fcxOabHNRerMsR721nk\\\\\\\/y66nqC30BNVx+OVOwYKmn2+OxO294BDR78HVM5epW\\\\\\\/OTrv3xi2h80Z0xDaJcv2BKJo+SDnP8UmE4oZCCE0\\\\\\\/Us4c0fDhtd329TgWvLivIYpJYUWjHWIKTzDo4nVcixw46pY8ZZPqu8lBBuG6Ah4\\\\\\\/ikWFZXxYo3B5oE8QMQaPLz7LJzhJjgKYK5T77rvFEkNkosO0jia48+HV94PSsOf2\\\\\\\/zRr233ARkbMnZWM5EhI4jdSuGbwVMFtijqi\\\\\\\/ftl5fqeFM7uKWZ8aNzVldChf7zB4tEgIg60mMUt7GJsTYRmBeu3Kv6YZWPBGZNqyYXWdGi0mwcHztfBJr3IgOF3R7oWzvwHoVeTyYW6rXOR7CZTHvuUzjkSJW4vk9qV16pxD5ZAsDSqeCahUoK7x50\\\\\\\/MX+GhYVgMUcowhqQZWzxUOJjyF51hUfVztoaqhsB9KEORY1xHNTfCkF1+RIpWVlbMtAI6z0mWnauw7RigkLZ\\\\\\\/swNwgIksIxUmGo0987uXccd\\\\\\\/qd6u\\\\\\\/WJN3F37EHPDHjWV4annYpDVr73gof4ujyrMONMH4tUwN6qAhOn1Rgs04kXIOCbvnLmrsmlDSrnyALY8K\\\\\\\/xMqRVvIocQbpOqJvaD83moq7tRWYExPf3nxd5w8tYYFkthlUAH1hkndCnQ1GYORuXtKiOjMu7CJcW1Kau6LJddM+3bKHBRVDqmEv0z4tftOZY3zX22DQ6Xj4e0uRfGYu9XErQsribaJygNcHfo3TumonG6hBN8nVpgP0qcZOH4ggufa+dRxjLmqATbnyLB23qpODSL97ZxJ7HufJepLExrkcIO2B0VfIvo7tqb7fOtkxKVJh5D6E5hdVMdAuEg\\\\\\\/jhqUMSse62zGrGlP175i\\\\\\\/ZKygw8+M7di7a9\\\\\\\/y4wQrUO25DAQSdDkT97FcW0A9BETM8UnRtAOCCHtIyE\\\\\\\/Z7mmQF4C0zn7rjSuLBM3zxcamoXesGghwh+KN\\\\\\\/5oa+SVLb4ZJ5tV7gSljiGx8BugVUtdK\\\\\\\/qwICqgsU6uwZE8OebemEGZu9+15y1hcVTn\\\\\\\/ubFuT3Jjy2d7I3GXm53nNU2xBAkoVW9nqDWRN+8eu+3iAIlUPkhUpkHX7dqg9J8vR1TF8E5WO5hGvxNf9EvodmNdeG7+x42DMxF5bqU09iYNCB2l19SrhNnU37mHVRVbJHBRZ2nnCIlHHlM57YpnDHEB23OdqDLtni9uhGQvxD5YfXfWsSqr181hhhbRwPWY8MpFx4tSKaxMuTglrg0y1f0ZNlc5G28d9bJULBO0Y\\\\\\\/7ub6+WevbWca7PjhQcsyfTZCuVRKkFlMn6vFRGH2I59fvrO8vVjJFoBZqFfnvXz0FTlAQ3focy8TBUb9yTiA\\\\\\\/AE76HLrbablUWTWGr\\\\\\\/9QLM\\\\\\\/gyMXwi3ejRKi9JQvJvh+K1MZ6+POdbvUnZrvmcJckW7nCFmohU0+TikbTKpNIYRM2zeaIAx7Y84MvRAqI0WKBzA28Hn34RlOvbDMZOMw7euPYrptOABXxulQ4\\\\\\\/fUws3bLJBci6c5LbDBT\\\\\\\/BdXglgnlvxa3ilwfN5szjEAq8F1ZHUPMCcpIKZuM+8RWVtjXXKSuhOvwH0kdg0\\\\\\\/749ZxLb6p+JP8PyMjBRWu8h93\\\\\\\/FdcDVS29hbDJ+ETqkU6kHE\\\\\\\/k9e63UhEegyaMQiOESQF\\\\\\\/tT1M+G18nv7SmtZrP9abcDg+TaA8vDZ\\\\\\\/iIx2WoEmRuAiLsFJSLrpNMotneuksVGHDoYzKDtoV1+GNYxlYtICibZjFXEKfErE5MECMgpiyFpLn9DxSFX6DQUcRsBwhijq1gGilDZ7YuhQtSH3FqeWJg3ZAfeN1Av9Pjvru1xZnNLzU\\\\\\\/90AzQCElbkI7nVsA3sCjDCvIkBQQ9NsKHvWJqT+5rgSAOUwYo7R3MrXceiK6p2ZY4UsfCzOlJ7Tl\\\\\\\/DOOlph1fhzaIx82tVpJIEuN2j+fT3XCgMjYl+XeAE9XcmwkOLct6hjJ26JyTV3uBQlsXkMwn7DchIMRtRCkx5dd8+x6sJ9XT1QLUdoHUkvm2Q0KYl9mQyceulao9aDJfxMs7nsxDofkhUnB7ChpCCM1xU\\\\\\\/R7oLy540443RlLS2y7yVL9J+avHcUi3zMTDPhXJyNjuOPj5bh9i8QCL4DPMp7cF66f+WO8yzOyGJvVdn08lbndstiPH4XdKfDI1b9FO3mSRKUj\\\\\\\/v1WM4rfRuZ8+FeTPC7GohHtyp0kTm4ujsMvjwzdNFmumHZE4JVOLgBBYJdHHCowMQGZZVFE\\\\\\\/eOPlB18G3MZSMk75E0R1qVYywn3zPyGaL0uV4fe8UET2Wg9YVDfGcR635ajCCtjW1t06py\\\\\\\/qOOoWQOF2zhwPEQ1bjoOOBJ4petOexfbY+Xs\\\\\\\/f9bz4m9fmDVa728\\\\\\\/CVMaofsKno0Gek1Ite4Wt8wyFk8oDtil7FGe4alyzNSyDOVnFxYidu4TLv5fWPLUScqW3sMeTFal8ZILwAgjCJaJJoo0i5Poru8wTl5HpW3CVI9qO8t6glqnSjfD0NNjCNQFUPy3t3WcmJObIFw5YRkUiGt0UDfFTP00ZIZnrqpzPn8pGWAgeNEuOQ94Flyl088cfQ+KSfpleqqji2zr10Oxncg13LiDZ4yxAY6zT81EmfScDtYE+KCicmx\\\\\\\/uNtdMgX8M1thSux8VUkpaKXVLQbFkrDftr2cZFDeYcOW4stoV57L6DWd+dCcJer\\\\\\\/0OtKOBYDLc1QLyCEhHQBD0jhSBcz7d9msQbmz8xMmtcjEERqrb8Y8BJFAtGNiL6At3IGjofa5WcYm8Kko\\\\\\\/hitiyigPHVcVV\\\\\\\/XHHX0jki3L8Ri5a4ln+JRzR7Zd\\\\\\\/7Ia27dlhhUCJa\\\\\\\/gjQck3ZMdxWm6LZjIAptpOWJZTdBs5QUk9Zg+szrtvyA08w4oqwySgRS90xFNAGkEJmdHvq\\\\\\\/5BW05wmFmLN3jXseDtPdUEsKI8GqBgajSMIovoBZLfJPd1Jiisd5YcmFVG6WosDEUDRYqaM5IS2rPgXVf6tZSJ+axZG+m4CnUCOEiKMGdtdI+pLZTaMEcYkJtDrwi+mh1kE7rK\\\\\\\/CUJbH2hejay3V9qR83Hu30eKNS5v13UeRA8m5T2WQh5ob5eC7QwbE39fWQpcX7newXSEtHJDg4XFWsLOqNses5HU2Wky+ngjEIBFzoR9vJGsbdLNUoxnL2FfXGeDLvsEDRhVf6RzwmcOVdLp0ne2CmOHQbOMTQ2XXanEgFKpLxvJIq7bK\\\\\\\/qLf4Xg6iFEn4lHPcwHwoaTPDupB29ZUQ\\\\\\\/ehZ+CC\\\\\\\/71gpnLFwR50qNV4Mgx4VzEiqktZgoLRFxAIX8c+7836zr09h+bTt1uMAmQs0Wh5vZXHjOQox6pgvUzUbx292bNmNKeeCik1Uq2TKQAWxgZcd0yKeow\\\\\\\/Jvcj3X6RRGfmDUdCQwZoh3V0JqnZnk4Qyb8kmmeXmL64qC6vj9wlDeQhoDGiDn9MmjPQ6Pg5ugk+1K+UGn9Faec67EjoRQAFs96hlk7KdR+IcJac6bwp+i7zvMGfEDm8Eb\\\\\\\/ER51AV1kUTBvjxzcCoNnhWH3y\\\\\\\/3gQFAWUAqALFtXPEcrP3HiJdz0eZKHG\\\\\\\/mar1NdSm1zEXihI\\\\\\\/5yspN8RxH5kHRFbm4oyNWCZrx1VEOTFRBBjx3SD1GXa79Cz9\\\\\\\/FHsLy3lhjtbAEgE5niXfTVyEu+yfL4A2rPS5Du3m7+8YhIyR0jUL6H3UCXkt9CY2\\\\\\\/ZbybxwjM8IWm\\\\\\\/ueOaK9kWvxdGZqJmRwK2YCBttueoKMkBM0Yg9iOD5+nqZ\\\\\\\/Rk93f659V0zF2ixykY6xQvWlkJRGgFMztCjXIjX+WxMBg4U83mPfYffP9xCV6SsNQCWxMZgBgYcflq01u1dbJwvCO1qaJ0fAJpDX1z0SUmCunM1hxynjgcxzYgDcNST5b1cq2xtMc4EmMbqXtq1pxc1Q6lhOXwsF00B2\\\\\\\/iGId+EYld17KHHHGuO+GeVvA2BBpyvFpX\\\\\\\/jrvMR1HTEmwjSp7rSPjaz1Q9T3xgGmF56dKRNFVtQ1vlMs75yXPoAJcVIbOukkS\\\\\\\/geTO0+hkQeuvnlg3gpPrVARA5gUETk3jSR8jkX\\\\\\\/tLkb4h3v1Vb4fGPAWebgMvVXCun5hRsgwpoGBSmXghxDI\\\\\\\/+\\\\\\\/HlU\\\\\\\/3esoNaHq0uOwmY7EzjrudDX\\\\\\\/9JxVNoWdlA4QoZVDiucVazaTD78E6Sk1CAoGZTM\\\\\\\/NDyib6jxVkQj+kAW+k5SRvvTYIG4SlEAwEEA+kM3FRELeWkXa9NcAHZNJ5\\\\\\\/DLjv3\\\\\\\/RUpn2GEnGLEWtmqkgPJbvQbInWqbxaZIbeMOly51BN8WvJMTf0LmD9ed5ECUhsR9IS\\\\\\\/tO3PLNe3d4sWIG\\\\\\\/UApQv67BkDumUDTXE2qgQMYd5bh1fgrGrKX1bzmpZ\\\\\\\/6Kl4t4fnv6wvuKV3Tr9ABV1Qk+W0MZUTBLUsLZLncwPWvApUSgwD1UIGyw5B09LxqVn+hAD+eLJLWnoN9Tf0kU5j1WMcQm1bdzD9G3ZpceVJS7jo6F9id\\\\\\\/2mGhmCQ\\\\\\\/IRq4jMa82PYkt38x+auZhccJP8TMyHvRHen\\\\\\\/HuM8C97y47TfmfluemDYGDtPN4mq2TrkTBrBfKaQShQF2+TOpwNRJ8JfqGZxSwBv5RQG+fotSIcZLrXAwamjTrV2lCGtPKnfr6o2Pk8yD+mYba42hBJjxlj1gmMYgUfx7dB7NIpF\\\\\\\/ftMCJy8kBX6+b9O8w\\\\\\\/HBKAC1vF2WFWWyNnqG7pPIk9VOV9FHREPbIvGFzGxId+X7d29ZVMq+dUdYjG4DAGbOMO8aiYXcqVoha32uXXjm1WXS4ROlyKnIxrpdubLC0TYaVbvTnRdVs0hA2Nw\\\\\\\/KMKIk8cGX7j1pJWeMjLF+ZiuGOsiM+R9bgebrwOws3MAuMSrEBMPrLQFIHiIhv\\\\\\\/cWfCZy8ewqIjTBNdlnLlIzz22zVgp08QnLySFixOMlNscxaA4KLU4dkwoeqj4B7cUa1\\\\\\\/fy85Vx7QpmEvIL3wDLIgpgojouigCCWAis1EUY0G8eNCgd5rl4ShwY8v5zaTdZrbze+34GjAHeBC+A1gAa7rrlWXBTLwRCXNHxFl92Wdsm07rCWPxhG5Djln\\\\\\\/TiDfPZXD9zQS46aLU3OMRANYzd3Ob8zzNMFxglzeJ1h3iBg9\\\\\\\/t6atmVy9HGlePkfIqrktCOJiBGAcYs86LHjDF8YEt\\\\\\\/+qNbzs0aWT6oPlQ0W5YaXWHWAgRfxBYXJhmxJCXSIehtSr3evpjFJzN+c1Pc3KgRamQat4aObfKtzbacwKOKqacZSftTDyaiF4YWrSW4o8iteoeKzTPNivCEfKuatmleyKfvaG0M4xLapDNTNVidkW9Bue5Xe7TB\\\\\\\/l4XbZniNsAZC3foi2o121+7gh6rHU21La8uaKwNB\\\\\\\/iE63Bj6LtXbs\\\\\\\/lr8Mmchfn3meYm8GUBdacacn7qcxKT6eJlmm72t9v2atdslq8Ed\\\\\\\/och87j8TghJT4kQqvS9oRUXd4gyyzw\\\\\\\/2oPiC3iHe533MZn0HM217+zopflHTcUSFS3dYlhhVoifwsQziuiuUswpuVvjDZhty1+jbqkP2wTPpYYae\\\\\\\/dJt2MNl0K6VsV3owrN4B5765zPRf+xaVdFwkRWBhYlPxh0AmdNBA2bQ31BnNmciXOoyfKd3YAP3kXhtTQhI7zrtn58FIVCuq\\\\\\\/9NWi\\\\\\\/qZIjVih3BViWkNI\\\\\\\/GO307t3GAWfoa1GsyeRXd95iYaTQ2AZJKXBe1r00LuMAESj8sgkn1hv1yF7cXD6CQQWi83uzrfh2gjna6\\\\\\\/UsF3xZ+iu1WsCbz6tm48DOg3FY\\\\\\\/0nCZxUbEShx70DDNHuFrLinyNPbrk5BOCVuIUy6q1oSfrVxYVNGcWFWNmwS09eHaPblnBHfhg2TwgXsENW3P6\\\\\\\/vDCRb0vMwsLwjEWEgDMiH+9tjqsREM8AFibahlEtsT9z\\\\\\\/vvApe+WwsKtcFhFDAJLOyY53GnOhXqialCfcbJ7QR5uo7js1PZDvRHDHNzsRQHyMCSPOsAZJTouXsV6dDMG8PG7YXMFwXNFm+pm+nuQ2ihAR8QPZjt0Nu3ZOgFUlfN1mHeRC42KiVhYxvz+EaaOO6NAboRp5ag9pP0nXHYn8tiLhrBfAgjd4ZB2DtGwTYf3mlqxz+CnAvt+LP5BEliWLg+LzuAsJmBNEdhKnuKKqj4VxJMvaTzudxDizphDfGuMGhGLOWxmy9LNfeJnT42fdoYDOBwcbSs3hJbh\\\\\\\/L7X\\\\\\\/XskDvOPgz2HH7Fnz3gvRWDY75qEwqMNLhnMzl14YXQsQmvZlut80i3DKLJMnwHs6SK1DtcHELvtq0Nf7xeToeguTK49CgngXI0uph7scnG6TbzbNGml2IY0IYLwg6cBsDoTRHZL+ZbHGCAgbj3wlAxQWW6XI0iO1sc6jCElqU+kb\\\\\\\/iW+3IJUvL3MBl0kyL1ErZRFt431KmHsMl8lVu\\\\\\\/+EdhBmVCU0dq579Ym0vedsS9X4suHGYn5mF7TeEEbkSVnnIHDoMxcoOV0hTe4XV8eA6Gj29vbvNts3q5xvVp95d4+9KOY475I9BsGdq24PiyrcId+khkrUocRmsmXfaZglf3V406xL5cDhIlyQyTSEihyO0jqPaB8fw6HB8SsmUSOA+zpxK8J+W1GTg2\\\\\\\/+XVYhFmhvg5mtAO7yFKvksOvo4UJ6zOrFuSWz6hMo4jloQHnoi0g+iWw8c3Dz0TXb3EZHpOviEDiQ\\\\\\\/c8x\\\\\\\/cf4yibLu79Bm919wKV+FQiNtWafwXH2zYPF7QEzWyNlVQYc8xkorQ6ZpxPGrPFvQz34leGvmslgE113JZ4jWcI9Tw1uSPbbF7a4WRP09C2ayJ4dH3+CAYhBLSk1ZKXq+\\\\\\\/w+NmcDUw2uQcQzIhM\\\\\\\/rApMuBgzdxj\\\\\\\/QeayWj5CicPmvG5mVrumldI6sa8zQVNBQMg8GP1euvzkL6Rw+qsauW1p0iWcy199JdVhisStCxkckAY5i27ht1laYpJ5UGnbUo+65Lbhfig7xsZejAKYnzMmAkC9xmyYgwe7TK25lj3b1owpgRByJLHbUadTd7fxS\\\\\\\/ooUKaICYKijyl5u7hd8ChAWrctgA7TBrpLWzT4Bp70qK+pDF+5sQb1BsSBSSdh1c362zPuEsyzZQCMEAUsBu\\\\\\\/LNeEVumLpyb7gml+toOfB8RfunCHxn8awbGHTNfNdlZ3TzEZWvj6aIn52AZ7D1jHK7J6k8xdPSYLhHipRIVgOtArCyD9Tbr8VQCn+9UlL2YonFnE6pgS4SXflJLe+pYfaJENgemAx4rR4ggLhmqV8n5eUE0M2ojudJQ2+L6m2xoZZ6XbSnxm7cUT7LYxmgibfuJngCDBLwvgBlTfWjfmHaV3Xr4svsSkRqhYW5mH5qg1uNBssAoRGUylBrRJIHBYvxBtKRYMjQgNSKKg2Pxwo7uB59IzZ3g+OgehB0+ZY8b1RZCbogk1XjikZ9LH5vzEzle1JXHstMWZk6mIgSh3o8EIKAlmWnSjR+kNTKjyHN\\\\\\\/ZAH4Vh1cDJQw4URqi8niBa3rY49kFAUNqbhFaRvGCoicy5RtsjpiHO3xGq4WHbJoGsz33yaaqNDEEYzDAkZsFvLQKQznX1cfgRFMVtQgqSPbfSBVcC5XvG++xDyVyG8Xem\\\\\\\/zEh2\\\\\\\/\\\\\\\/9xErKNEGTWQMgciZ0zbOQfRdlEIDjwQwVC+cyeFAL6AgVekDwXmLm6jyQfMwJwkYLlP\\\\\\\/qCvukQnruJoX8hORzmYGs4i7jxdUuixwKZIyU8SgUHLmdrn1GnFt7Xk7CHObSUiMv3tcCH3g6JOgpqxngqUAFrrjqQS9f0aNZlnINgjQDLJH0Sj9f91t9+X5LXQuF5k5VViHbhVc5LJYK+FELiTYMLIP7lJo0QxU3hx1mRZ9jiw\\\\\\\/+6lv3hOLN5Oe2xXY2ouu+Bpd\\\\\\\/8p\\\\\\\/0\\\\\\\/kkowdYO3MGLv+0HG0cgr8PiILemyygV6sWhm9yQss4iadY+tUw8Mrhsrm1i6l4ahWq\\\\\\\/cTGYhZsf\\\\\\\/EewfTBYbGLevaEHzgyeTXeFRkUhc0ZqyBY2j8F7676C5n\\\\\\\/L3KhsFYh9oH3b4CIRtpBzzrmwtVT+tbGQ2ykk0S8DEHw+asL1h4NizTd0REsbyo3No2ILGbpppt7Ir9iSoS5y0o+kZABSHTUvUWQKFltDO4\\\\\\\/Ohc+BwnQtO45I\\\\\\\/Szho4Yg1Xmcdit3ewnwr1WjLiClTEN1Vybjqq0k2Re0Fkd9\\\\\\\/H0dQ3I7a\\\\\\\/p2PYQrVDqfeRX1yskBcwtQ9+ohs4G8R2LyLo4iMa+l1kJm6aEgWk\\\\\\\/2vzTCmjCEh6dhLJssJ+E7SEsdABkz0QTYRK1oDthNFTh+HTtmHF3W+3K\\\\\\\/HvK3egoyTiMQ6l2XFh314Ph3hTddZgUXCqcNDvX9bVbDyIxQzAKk7ea9yN79SJuhIQ0ChQyXQ77fdmV9rvw9wMU\\\\\\\/8DhUe8xz2F+DXlfERbJQeBBMvLy5DCTky7LPb9wKxEuA4fF3mEajoqICDI8pLVlvALbkdNtv40XJU1T51\\\\\\\/uOYDmc0cLavbVYPqCk01ZRYRkn5disgff5RjNZFUa7r6xPdDUdvjRbJwsLQIDEhC7qsxUEQe+HWB2JUgV1BJrEHZQ+m49cwsDl0NJIcudpSoydKz6N+t7\\\\\\\/fF8MpvjPzJKQePn9I\\\\\\\/5xN6He4Go4hXRFuuDGRMykCZg3JiwEC++kTjnqcW0veDvHL3abuRkV96lQuMQtyz7tZlstEuX6kDdSbyaujd0pEtsMO4eobfyYatj0+NDiEQt1\\\\\\\/\\\\\\\/TOuVdU46xY9jWPIVvJGL3hdJLjweIkQucRoKCeJOrbdUoO0+dxQ4iEiqPmXA8U\\\\\\\/3o1j6XV0CtCXulgJ\\\\\\\/rRaFcZdbCw7+EfEybXBRD4tKZ4usKJ0LsDDA5hZ1bG1X1VwhGhsEm7zplGvJVywECzwMmY0tLuHK8Pxw52h0x+Wrd8jmmZQbeTTheEa49KCCeIoXhXZJWyK71dA6X4bxVsq3N09I1MeiWBGIgjjVoc8\\\\\\\/GDckKzOpVaM1i1LsWauaQ\\\\\\\/bj5JaI98E7zlzWa4G+HQlTSJBeBH0Hs0LTSgwVXoLPrSHw5d12\\\\\\\/v06XPULQx+baXgL\\\\\\\/bpx9wtAcFU8qD8jtTPapRdWOJMHBsBsOvHeJFwvjflv6k7dR8MXHxBn5F00UYh2XUqywdpgxo1OU3bherJX2mKRx5jnKDYOBaw\\\\\\\/j7QMptROOyWmmbBzNVeJeAis2ToAPwwnp0V9pKW8eiy2Kw3etm+LsIo3CCTYo8XAoWHyUEoGWAGIa+06Le5dXyRAHOQwH\\\\\\\/8QPqf\\\\\\\/6U8jImDk9m6ieGjqDp2NQVceXy9NdwUwsYCyRrEqyW4sRHDPGTsp\\\\\\\/TtVwp\\\\\\\/u3FSKBKUe6MFQQbfsl7cOBt0MZhWi2epZRW2cTgzoeEP1hLmgOGrzK71iB292EutKskHGXDuS922\\\\\\\/lt59A6H5kA9cGHk8jdz\\\\\\\/sjORwpDYkKFUXuaJHaxxGNToJ0eS06M+P2cfLngtYeMfs33GFvDiy+Oaj4cBxZVCTf5e4BZCg5MqiunSmmYgGifFtj8HqkN+GpLi7N8UyL\\\\\\\/Jj5+T9P8VXwjrq9RNN\\\\\\\/ns5gf2oSMBrEO2AEr\\\\\\\/pswoRwmGw7+Pn\\\\\\\/pnTGKKqM8FQqdQvJSeuAo2MRgFR0XIuD5Wsx\\\\\\\/A3LRfQgOjCPJ1YGrtxodbJCmP2T\\\\\\\/QcZd0o\\\\\\\/IA+2REJ83atBPf19kc02iTWF2eEfV+CHOi8horWn0ETB2dBeImAQ22bhscI0t1QHAWD2+IK\\\\\\\/+EmuFftAoNoin2mBlRnrF2n4jdJQvVm0wTsHwMmboHFBdrNhvJdV1nRxnxfVI8QPpZPz5ZGTqOdmhySFapPC+Ny3pIf7O9Q\\\\\\\/cd4Pzo6kshzyQzEFRelhCwrTAAot\\\\\\\/haJm7YiIoGEwOb7IWxo\\\\\\\/oXUe\\\\\\\/WIeU8k2a66875tH1icZNgvoQL3CPZBSRDWbMFcvumboQIIDha8c1AZI6IOqKaBuLgVNFhh6lrvVYI1bCTLuukwbE+6r0NeE6A8yxg1F6GfBljTKooPZFKYgupP7my79IpUFtpMkvLo2r3SWZBQzpLwqGEmV6H7nV+0VRViJ+V8mEqqMmkFn\\\\\\\/TVZIYebe3lYWa6NzG84z3\\\\\\\/mwlS6JrxyWjRiuPrD69D9mLU1gqdXwMCHDG0oZPkD8SkQGHMHpbHvejKHCRs1kNnwADa+0bBtDGWzjBrvQ7AnA7z\\\\\\\/RuP+H7kN86Wd2Sd8tmQONiKRUlqory0Rb1FtHEZ6VlZ9r7vNxc90NT03o5Ai1Qv+YhqdRhxO05tUSiLsP\\\\\\\/WysxJI\\\\\\\/jg2H7VKMA4TLZg57IagnTnxb4mwtrJWMq8hsUQatK9R2+9+vMpwkSqMIEwqa+e1d+a2uzitSe01AVX55gqJrInjqIP\\\\\\\/9rhwg8e4XOWA7Sqs6kPeKeboVDEUKUu82fu\\\\\\\/mZf\\\\\\\/AsS94R69i29hOt+0eFbu4oj2ggdX9yGDLCZF5MYW3+YAIswkXS9JsHRu047ocyMp\\\\\\\/WjWpb3ZAWrZqpl4zqVfFbWX7ic1Pdioh45tutSGnZ7J1xgkGtDLiy6vVRYKQp54sbwR33\\\\\\\/h78nFV3a2T+PesTnx3m+9TT0mUqHmOe5l54GNnORZtNqd135on2KXbO+GYYO8OU05W1UHarZN6jy4jIgaL\\\\\\\/RGm8NE8Aw4c05218t2H7GwAOdVg2VtfinjjHg1MpdS9Zl6l6p8kfNubsktJO7zPe3MKLRfpPM220laydTGBEDKU2KoQxKNyVB+aNMTA8QMKEyecinlNMOtv7tyggGKA+c40LuaEmz4owQt8Tkmlrx+eQqJDGZOD1zx\\\\\\\/rvsJhY5IQxfzqIPOs4b5bdFxVO4vjPKXrdXoQCuJOof\\\\\\\/YLyZUWB4Z11UeSDh8z1dP6p3bqCK0aKP1fNrqQe8pQQyB9qv8toksSpoXZF59iJe45fwrN6inaO+JMYy\\\\\\\/9ikxqhQztHkBLkVwABE9DE3hln1jbhC1eMewMTBXMuHUiTuHaUoE1oLeOt\\\\\\\/No2tzfzy5TF7OgQxIgxpKJpRND1yHKV1cUV9e8mWgOctJbKubdJTqYMlJCHyKjED0I5gr\\\\\\\/E88mWcInwCfhpsDiP5ShUpAt0PizSEwVIhL1bje1+hOrjun6EeAOAd4QVEee0gFtpFVfAJKcC40710r98EWZqIiwpfD\\\\\\\/V7k5kzthEfrbYkyRswSvsTLUtQ5drso\\\\\\\/ISaX4zOXXTlsIE\\\\\\\/LePZmxRHb\\\\\\\/EeqJL8BS2b9EpSmPlnMGteCSBnw8efQh+9MeMUyFJRUJE1iqVlwCKont4ZZCFafhDPOZrzmpgA3DknXlbdNoCHjRc7\\\\\\\/y5JyeHL6rLgz6k6QziW3\\\\\\\/AUvXq9Fa2KG+g500uc6fZBWKwBR6exnpeWEq\\\\\\\/CZC3yq38rOO3S+1MTvlxyyURHLoyk97uYZ5ldyGVWOq1YHfqXri9V+f7d9D3WqO4v+PMfg3963JZmtgMiqMVNHrGC0ATgQJImldFKMvEe4jcucyILKlr2f2jqPmZKqttpK0o8ZPNVIUBEv3GAve2\\\\\\\/ZoUlkShHV0yzpKafI6sFl3HZI3Ig9JvK+ow75MNvAi1IKUg9xMXtCJzu1gc304wL7IQTI3cVUqQoKO5B7nHH6n8KHiNoPjtpsexaR8uIGW2Kuz7EXbrcCDlc\\\\\\\/qrtNXJaMzZpJTm\\\\\\\/WnLRq4fN9luQDSeerHZ1zu0oW1W8DrsW8RY8N7xTXnPJlVa0CXF5Hs5Norrb1L6dWIBxFsDGk2OgFs7+eo\\\\\\\/pT8eEFZYFzBkU35fYo8YPt8i8EC6n5M6O8oLtPoMdtOGLLqhmDTtaKq2qgLLiOhsURMiKu4XjbNHIAduOFQ9xYP4qoZPNVuMbf9vpC6ptB4t2XbLF6sRKs76DmENFa9BTgKSaDvAu+atmsmlZHXpHNpLhTGhYgVDaqRv8YQwfRnCe7QiTGELwsdsQbpmKl3GzrYIqhnEQT91Vybqm1b+t+Mjkx6VCJMUiXScyj9WqAS7cnt7zgkBeA84tToruRu3EjucwYCkQZbB2lEZdYQ1sS\\\\\\\/9tBhMg6gJEqDR4ahEQKPpaGMCaG\\\\\\\/Z8OFOcjelQi8ssJEHREDP46NnEiHbfO27FM5xD1GBn3hJltRfytxOU7+BH\\\\\\\/01sg2FdsHI\\\\\\\/0Pzx9sTcobNFR\\\\\\\/XmUHEHDHcmiymj3WXessukPjCtvnmJ8y4Q75HWDsWD2YqARqUzba99kSUXJZWQa7a8BE6eCnZo5pifgQywI+xvQLAIK7gTmBI+kM6X4WX1MkiqxS4dpCnBBLyV8uK2UfvwoDLTqsWGNB9qHjfdJxhDsfLylNWAoofTHqLZ7C8iTn6Wa4TTj0GKTVMrvcZrzgKsI11jjPNPyshL59Z96rIBXSjMrvM8jKC4Ruzatzp86T+g+2u6oMZf3qvNIYEJtX95SgsC4fmDH8Cv90h3jPt2cLOY43AMqEmRHw84hCte0hOeolqa8T0ihl3zcJfoIQ3MOfmF7yXxSXSCPM3O0v34bdeYszQhLQash+T8M14zTQLSXXAGkCWz6zHYebU\\\\\\\/XOgG4d01coIoTDjCtolZ5skpiGn3peZo4HBKrOMhLcynmztgQzgnqIyEULFho4ciHiB34Bap2ZSd05cCCDfOC05d1pkKxgSE\\\\\\\/mH12\\\\\\\/utFFP8N559XDBfMkPwo3maNhpeDIrVXfDjz0pAaZgr+P2HGRk2odHx8pEQyVD4+OikCCk\\\\\\\/dNQyxvm9Na0iqClj8jLV6r0o49pZPT9D9jj4L0FcNXBNJkzN68MbVmdq3pty1b1\\\\\\\/SCZwM+t5sRGQ3LAzn3DHXLW+OWpvQWLCxAfTCkmJGnqyzxL3OasAzqczlBkkjYvsbngJwOuXwFlqlyRSXoAI2snL7txUH28o\\\\\\\/EMOye9VQue3dkdm4WXTNLeb2\\\\\\\/mk3CQV4TdxZSkf6gMEdg\\\\\\\/mSsy2osWVEPHYfMn4rrPjOumvVRYf\\\\\\\/ze12N2f3qIfVDl5694hbm23i9q+TB3sXGvVUYGfPFM5BdPslzeeLrDD6+jU3DmZPSJoRmOPA4GGlDQlyCP\\\\\\\/JXZHPQHuOAO5UmGLTtvGU5kJfZw9mYFdlKOFN4N1XoyvNx\\\\\\\/NI6bLowclq5iqC4qU0+3NV3IDJe+r8dYUY9WnQ\\\\\\\/9MAT+hoVnMGzd+w6GArmasAhGVhPR0pIeov5Wfo9SAo75jd2K3Ijm4fCPuF4lYH31vQQ\\\\\\\/ySiHRxjRmgyJUYAgQSzbFXdMp1akQQydXZwJW1ILs4R2s4vNi1gdLHY+8gYYJCNYSz5\\\\\\\/2GjYhplW07L8j7u+0r1UcJcdbjKvPrWWz9pswvV0Ii8NIRy+UQmQf4bbmm+s3vrjsqOGwgtL3MGKFhRXyU5dGq77nzti77mF2FnoBsbYeQdREPpaTweiT1htD2svWjSxQ20ZcmBHH95JCbnk1islyYYI9Tg+Gv1vZyiY+kV5fxb6\\\\\\\/+0azTqQy8ley3BVy0+uQuHctS1\\\\\\\/aYwhCZCrC6yHPOh9yba07Mo4wUj+BcUhSEpsDWx5ucPmweZgJvPjT2uP5PU2uujngwn+DD13NvlETwGwy8ajclKvbB\\\\\\\/DI511ekrZwCrinDi8lzv+ArxgFQwk14qLY6Mmb3NyUUJrN4o4BqcJ+pF1CJHan0bNnurQTPHz7tvQ0xRoD9Q5dE3ybWMdMe3dfjDEYnBt62Fss4J5Rc+xnCSEX21Xb5jzU\\\\\\\/2XOBsSf9rCVKjLD0Md8lSsvXQxROSanyX83vaHVuFYqsAfgA66SExqo8S\\\\\\\/V23REwmYxhvm8sEAXD1Z1ZHQho4qChwm5EUCPLNlrKZFpHrzelbssu6IWVOa5kT+AxVMJ8zPkm6pvEHaQT31RTXAnn+2aYy3jyY6hySwKH4mtV0AfzIrvoKtBzXrcIQgFxEb9wDGVqwG9bNyls64MQQ2Ku8QtPWrahK2+Zpa1AtHZAFmgFRyc+bBOzynjXGTZnJlGox2coSVfG3Xy+V8PgjDfBKXqYUoXKsmt3wRdi5m9zwtk1KBwnkIMHx172Yaz5OHrGzJRZLQRN4\\\\\\\/WVD6NARN9w5IBTzofgIu75JRTv5SWMStLf34wBBsdYo8QXzkiu+cMVoys1hRuofkSHwTkEad61qS5qFN2mINxXW11aLs40nBDoBWV+473PJ1oJRHb1P+9oeNA5UgLyuvuzp8Txiqnsd3XhU03ym2IgCG+m7uqxuxfbK+Wdz2NCFjIkWJyORTvKB4uT39QktDwkrTuAJEjEIg4QKm61yH1RhvcHSKMfVMQar6ksA6GqzI+UM38bKcRmd\\\\\\\/AYZIczLX9QymawQJlPaeeAx7gOOAwZjpW4SjAQ5KkW5t5Xwf6RzGqEla8BnfIzl5\\\\\\\/oQmB1NCn0AWZXl7GXLyWdEUvhyNCRJDhT5\\\\\\\/H2VGRQyH\\\\\\\/mD6fX8Py6qLzld7lkrct2GVM3ycOiSaSXmRhyx\\\\\\\/3+C9nsyt2VTN1\\\\\\\/vrMhDFsR2k0LiUpyJJ\\\\\\\/5Fmp18KBSO4Yb4PyTWFTsWMA4+3kJBgZRZyaJcBHTorAWguYaHH2WHREATGoZFmmJuva00qb5A3JzvGqTxJqoUDa6PPeA5lV2C4zvj2NWPqj0xlFmU8l6PzAazOJQL5yZWoEhlhsaFSTuCOoTmKEGY76lB\\\\\\\/HQLOrL0jXzi23AZ4PaT1bQ+VfTBiaj4E7rSyLroCSWmMiHvFNPP1r14sneDZmNdZz9uEnOHY6tRFnrtE90EV31CmTxy+WkB4iB+u7t7GZ\\\\\\\/r\\\\\\\/saSslI03Ci7+P\\\\\\\/+Pr1HcKGdO\\\\\\\/cA2Ni5HleTFmrjgSl8\\\\\\\/NxvjIK2NV63ON9aodAsnTk7GrYfiJ50v1xd\\\\\\\/+nQV5jBew4D5bkofMWu3qMCf84o+YvoXYwCmNBbXCQEy4iZUJ+eIBXoIlme40q9HiDoWCQe6F2cJww6ClfBBFAYJw7relYCOSDdzNPtXNn2CpgA0Z2lTf3QyMPIgtPdIs9YBq2zJqBF1PUpPfQfuCFluQKcvcGeVSfPC4gZQYGkEU0bKjpVKoQfgdmcfQqvUOFNQV8mNmv7f7h7agK86CpooVQkPOLjwppa\\\\\\\/0diQ32XpUpueOTxcuMwkwFfiUkfZJnHCb7gD+goE\\\\\\\/7QDISaiA6p2Q4Q9wTDjdHQjqIwHKx6u4k1orUixYNuJ21CvCcRnkQmr3SAXaEOAgN2zgnVzUPSpdb9QRBHuEDZp1OD0LkSNX4+VBayeSgQonJBDH\\\\\\\/p6FP1tVuHbBPyfeUWtZbjDI5eEbxbIu3ZY1RHmtrJT+PO9dddr7\\\\\\\/4AyJ6XdoNBTYl9XLYdipP9feqcrAFpUk6Xj5syUNdQ33KiEcPZtCbmjxD1itgc4ML\\\\\\\/wB7qzMPuTepmsZi9ZwfIOQCPE+Xf4Lcaags0SvzUP3I6JDqbbrhSBffJ3Jj0LEnP1FV\\\\\\\/gh2Nvb0T7At6kKA01LNsV+gXC0EI2ksGX2o\\\\\\\/XvqRQ6m\\\\\\\/17H3PrIw86c5mAxvxkuYecuXdxWS4FNdgS\\\\\\\/2dmgGc1tScHIkjqM57Uoowju4k90Upb1MBle6M07A4ZO5nPwhw3ZYQQDdydZicgsqQOTuJ\\\\\\\/piUsd1zqArEW5yufK+0vS\\\\\\\/dP1k3Y\\\\\\\/WpIPRHyE3q6dIBg8CltJHKl7QYyOjojEf9BCAnGDzewCFDOLslWz7Pxk+rqrwN26dusdt5z\\\\\\\/dMrQhbrcXiI7u+WFCzY1JgEjkvXJR0itftDienDQWs64RHO76Yg4Oh5iiZdonOsM9AvUfAx+M\\\\\\\/xR9PC1av100OhcyPI1fqRygaBJp+CCj17u5EyB\\\\\\\/qAsMeiw4bKqEl9BGLAG4yuO7gkbXh5e9aL2Q6l70qz4v452Q1qSVGm+ul1mpWHBLgSMt\\\\\\\/3ODyYu1fe8vmUMCeRenDZOfjt8+0giNT+rAtrRq4sBkhTi520X0CEtwvY\\\\\\\/dHpbtfXinNz38uC3\\\\\\\/Ipn8JcpKG4tzbi8ftbGBD0hv9DW\\\\\\\/R7gxNcNCPUSir6X\\\\\\\/s3MN9z0JV42cGeiFIRYQfe1skUApwTW3mBe9Rnev0M+LlD8wldzvxWhrmkL\\\\\\\/isp5bfRHpPPniM0lCg4CMe64GEG9mcd8h+uVjodP689st0jKigmkrDcgE0oV2nG3BY5IlLG71CJ5rR9ZwlpemC4XhlwnOfKGnILpgwGr5JYLXqWJGZ6czjIS07krjUGjaCfP3XNncNxBK4CnQXhkZGe8b2Ogo0\\\\\\\/wkMwzgtHWWWuhThqR\\\\\\\/nlpuC22ZoZ831fidc+D2yN8Fz\\\\\\\/WvS3uK3su7sEOP2R49AcXoEx9V9\\\\\\\/GX4Y\\\\\\\/Uc3X3ZQ36KJd3drbhBqBfW9jOS5If8o8xh2Msi3hSRaXAjZhwRw5LKCk5yR1czD211ICN\\\\\\\/DQY0HIj+MzeY7TcJXzehaPAEVAMevMUWGPkDWrn8e70vb8S\\\\\\\/e\\\\\\\/hIWTBvhnT5Y19SA1ieH6Y+DuQ\\\\\\\/D0e2CbyI3\\\\\\\/Of1YCs6oNrlF44ZqSOqS0foaqhoWpx7gHp+J27VHoNg3PrxwqlG0YH+rNFGxXWckONzfpNUn1OwdWHeJciJMz6fqkyYtK6RzMt3a50HgmiMevn3In\\\\\\\/1IKy1bXxO7tAo\\\\\\\/dbaVjAZHI\\\\\\\/6sbJG355Wa8I75FOeExJktuU\\\\\\\/J00ctH0gxumLLI3SyKfh28d67bgIA4H9PBwGBZNx3XLABMrpf0ZNJA4K9PNKzIQgdJK6jZZh+n6oR4At+kxMQfEbi4IjAXhibeVz1540ByLE4EgEIr1Bozm0Es1jlEHS2o\\\\\\\/0m0ZSpe0lefnsEGAYcM4aaXW\\\\\\\/8yon6qGSiU7kqOHXICWnGvXxmCW+bc\\\\\\\/V6IvYFh8dy8QV44I5fCgiyIP5xJwmb1b5kY0UGDp67vrbJA8JMUPjbFQdW4I\\\\\\\/1bq8EeA7YlFZmoV7Z+dh3PV8OLbQrXC5wiv9yN+CKyRL1dakTFrV\\\\\\\/ZEKZOhEmMuAHbyp7gIMsQnQWri4ced2HSxRuwn3Vzp0yZNp8ydO9EW5IZN62iUID8lZbKYjX3753sAXyog+4HKBZGkk1GJHBBckF838A95nwJCjacS4WUxtWC0efj8wc8J3AP4+6EfUBXNYs67QW1kCW1xemjjBRTpxjFufdbWvwpPD\\\\\\\/eOmCHEuimDQYsJTSpussgG5vo3gFp1ldCv++4DqOVNh1sFi+P25D64VgkWaCOa5Gs6eajSsvWzS7NbHd2Pc8Tt4NFuorTlJykARSWM9IRNGozoctJbiuJkmEH0+mdYnrLyOeKxTcZmPJ+ZtqeA5qSlTE69sWmioftPPyHA\\\\\\\/qYqQzAzMDYf2u9Cq7BnkTL\\\\\\\/+EP5U6opCNVGYthO5J\\\\\\\/CWNss9GF3shgqo8dPEm5iSokwnu9arBHpkBqzrIizVGZch+zRMLcVtaj\\\\\\\/yDNl5GOIbcPq1v+FqcHek3f36EgwFJhCqrHXaDrvD7+PEAz7TZ2mUBcaDP+wQ5R19y5zzpmqU4fAxne7x3ilYjTLaT6F2t1AyTV72HUQV0TU4JTIZfkTHIzyc\\\\\\\/YD3iEycA\\\\\\\/44HrWc\\\\\\\/m9iOkohptnqpYNXs7+3PtTW\\\\\\\/7B5HmalEEMpRCHxNox1CJdwKuLIqZ5l+TLXhHilHk\\\\\\\/VcocYBOKn+AXpf0YNobMbkUlNzh9+RDerGi2sklOO6OcAOxJEeKfzP+XDkMBeRiZFcONJDbuEemVaN3TYAO0ouKgfplOEyUSAZulRCYpc+qRiGNOcJGrH9LpcPP+tA7C8uc7ZkY\\\\\\\/trLPnnrK5YCmLxbcmEMzWddeJ4TQFkT02SUDFdHMY3TJN6BiTRiSkKqc\\\\\\\/zbVUC+iavuaBCZc3OYMZerlpXPrROjmfzuAVtxGFX82sDY\\\\\\\/2SqWSm0f21uFbs3\\\\\\\/X8smFBMd0AqbqLjLzJeOuUhoNcPN\\\\\\\/m8V90JBA6dW9Ayeg2RekDYTUFGnZ66k6GQkF0c4gOAQ7ZyVDgd9\\\\\\\/r5FZp+RtoIRu9RHkTGCeW2IcvpIjGWtwr6jE41fDYaV5g9esBms0WF9V1nN9VowoQ2H22v4nNcWKiezuHNjDC2WNKhKNYog1MB8bPVEO5WCAkUWbqzBk2MvYCuJsEbMgQj7hbSIAgtCqEc\\\\\\\/ONVhnZk8WKNUWNXnPchcc2Xt8ZY2roJlyONCw+pTVNJ4ef4YT0vceJ3Z2Rtkigcqr11rQ6u9ecccugjk+cDH\\\\\\\/8HtUhS3yVsrKW7mb5zDD6JsEPdzPL8F8TPzgqUfn7lbkg8+f8Mj8WqQQVxJVsPeWZpnBlhv2zHLfEvTM3rKljwN+K\\\\\\\/2N1gSidHNBDIHNEuZ8WDsZ8gU\\\\\\\/J29Bv6XXOwIizgj+5hG6PBSxrbHHK1l55wbkI7g1Bcys4Rzz2rkgW1ZW+uta733Y\\\\\\\/bfL53x3IaFQcH+XcSdySuxBvnmJPMTd0fA\\\\\\\/s32y2v\\\\\\\/i\\\\\\\/evgKBOlH90VPcRsdupxNnZJUFslt6pnNjEX1pjJ6sEpuYNxrRaM+hMRHoalVHfZlym0LIXhhM251d5fYSxqFuoPfcMPLUQ188bcACYH3fUzKg9UpNAvcGymJB8FSYZaz\\\\\\\/1JEHqJ0389AvWn8+eXU5HCStmRJGRfO+Mwy+8+HlJ+XYynEIgBibYjkGRn7fdRGAwAio4y2+AhgK+cSwHNQnDN\\\\\\\/WlmWMkKjNXPf7x+VQTEdfyKpR9x78Y7WPO6cdoU+Yra3ZbCUQLumEmDxxaBsEIt1N\\\\\\\/GwQKRgX2Sk2ktDdIrhsOoTUjk2woY9kt1ptrOvde3fDGj6mPtMF2H1OhrqUygQQ+QN9+CC5odBKrYPT6LJB3FlfmaRt00sEIiFYVuKLspvVHmop8kZVrhxhb6wUtpHSqvD+SdBSmiEcPNcn0fxxZUB9+CGqP4JRpdTMVORlQ\\\\\\\/5yjClOsAXQJ\\\\\\\/nGmMipqRB1fIvIwqyJd5EHsbbdgymSKsl+ST0NprFjV+PHfdGGdTLUjLURJwl0hiCQ7WLqsgLZf\\\\\\\/2PlKUDm2Rf6MWcmP1GSrxPejulXdKEVLvu19\\\\\\\/\\\\\\\/4plolDmgDsbi9v1QevbLEjwFoON63WnbxkvPkTbGTNAFkiZbFrY1wuOHSHbq\\\\\\\/ve518G0M2j9JeI\\\\\\\/Xon42wZEBukMVgXLagSQrBWf+iM8Kz7GNaT4vhpC136RBZlL9BW\\\\\\\/tMmrK\\\\\\\/RHlB9E4sh44EJAxYIqEQCRWamk6owCcXjR2+O3yqVSqspxlxlvDJ3DsXh8Mt2PDzc9FQvRFMB7WZvnUaRKAO387p3V\\\\\\\/\\\\\\\/wiaeqspw9VGcMJtyA54vLHlO0ySRGkXCBDOXl67hdQv9f3AeK8fnHI+heojPpWUqtvxiwHc2QUbEt+jGIsdVTTO6pwFiwOKrK4+uSn2WDAC85N\\\\\\\/Q19KNhgPvJ7OjS3ZwmJyUlWhorcQ0GSpB4ApdpANUzo2AEo4\\\\\\\/I0lCr3vY+a0B1YhQhgtQYxhlg49rZXEw+FQ9r\\\\\\\/QHTbB72g82c0viaGFjkDULJKF701pPVzq3ylztJuGJ0oS7lr5iwy6V9\\\\\\\/yvSUivEUXkeyWULkePZIK0R8nhiYaMoUZD3hU2Zi1eirVUmPSux1iiMeA78S6dp0jpmw\\\\\\\/5bQDExJU7k++fe7jfpTSB7Rq+prGwrd7Y1w+3aO23AeskIIJVW\\\\\\\/uM8Td8h6sAVObQXoo0zU896Ftudeg4J4M4898LkmAyEgDMrKPmcM6QQ5dlpuwgBoXRioQUshu3ZZlmq9QOfOUod4o0Gpdf5WU3G7F2bmkY5htXOk3F+zn2e2UUJ88jztrjUhEew9Hs8QAs9a\\\\\\\/BZkRbEG8PyfKBfrbeVhBomh79SYgOuPG1G8e515v+IqzovhezYvUm9LOjwb94sRoz7wXpk3GVYVj3oMOmLujXsFcW5LDHw19FBmZWswF\\\\\\\/pQ0cSrDt\\\\\\\/y+HESQIhbeCVPerZcoZpPaYZJ232h9g6XuNMQEC8Ahr6UDCCmQhVNCLmZ78ZAaFlRaPLcpfDTpOyHDJl\\\\\\\/P7YMmevt5xu0LTBcv58IGyd8JTsEk+YsHWLuw9E+HUdWVhMoOowPYgmvcaY0qiM1iwYy2VflPTpFl75JCAX24fCpHq+gQbZV17svCYcUbS2bvJFI7Sgxf58M6iQhcyK7b8PddYTLszJJJ4p\\\\\\\/rq2OSfyN75H0Fnk7f7Hkt2nO1brwFUAv1+08oOKJXtdDUtM05fIW0G0fb7DkGLGUP7L9os1IRgslQBtI00rm9lwiQYiOQBZsMJSEHcCgdk+ajn0wA5rHqQshvkB+jRoQlNV+bZvZaIQ3oNIpKNgEkY7XvfbRFeRRlQC\\\\\\\/NL98Q9Qc5DGO5lYgo8+wx2k2EoakF\\\\\\\/yYA6y5FuC7bWe4ZmViyxnbw1Cox+0RRy9myquvKUJwxZ7hiK9\\\\\\\/xzzvwG64EdwCJTxmnYgeuWLwiJNruDbmptUH+SVOlfxaj6XqhBRgQMw3QUCfkx+xwg7iMcQXAAOkVj2U5kWuwt\\\\\\\/jq44+xvDd3SQ9cUfUV07b\\\\\\\/GRsi64WzkrMtImWtM9DN54ZmXyKIZcrKloMht38IhfDYvwoHplmyOjMTVCPWH+xZNKY3AFab+IOV6BawMR9eqQRSDrgLwaHuD6LRbIYDkoTDquilQxVYewSTR\\\\\\\/4ZVgyx+7bLvqH08DteBN60eI7ObNoW+A6MIAYuTcf3423P6lxNWlrCAjmsQ5BlBHQllh0d1p5LtHzJ+7\\\\\\\/6c0Q9VseHfh4SUoYEryIWo7xl2GafkBot06oMMG\\\\\\\/LPi1nDdA+D5lLCzfosU3xEQ\\\\\\\/E4RzUb2U87E0G9TK+VrfSoLlVkG6pE1wcEUOAg6IxY30u9nbjQcq7h3waIbGAerIgmHwQMhCIDfS+fWpaMFnCRFGLRfc68PwyF1m0hypKnPXpZyX+553OeQi81tTbpktmVR04bPGgTVhuLokjdTEBz5H90iaaWvj8eDTvhGYkGEphtbjQxXYJ906RlF5G7ruDf+cxDFaMgv2EvEFfpN4rFeAR+Jwh7zq66CY8kpHS3ogRilCUGNO4epVFdMOT30+dhPCuwxHAlEt3ON08tUxL5+GKlCWRoMXytxYF3VeLDnG+5qCsIqAZ7YSeKbgdKTa4bCnHabsx6WxSjFYc7M75eLEAUboIVGqvGsjnBdUK+mfBD9fNN3+WI2b3XLUQlGJ9vTEbzhpcsU8IoXZO0o4IYaZWd8v\\\\\\\/gpBZZtfX9HO\\\\\\\/H1dQbT1rWXUbZNhhKRDGLujCjwd3PPKljfVtxVrzPaSYT9ApBAzIt984ZiZcagOVu2rONTExsyPgEtfHkSTDddXowvKMq7g9H76FvJs+iL1dTx4RgU3DCqdgsOPWdYZknNvGXQ0NHf36+yhAtsgALuPQQAPpNM8KX1ORsLyfnyAhjwHJg2YYDVhcOvjRZ\\\\\\\/dBlJwNUnCkEl9tfeAT8LJinxy6zTh4GK7k9Spuj+6KzoSZcaBj7ADDQYv49UAoYfycuTMzd45sRxnXYHrgEm1Yskj4Izb\\\\\\\/eUD8dSvtkbDJx2aySR3zHo3kceock+TKj1m\\\\\\\/3RuZO4vp2bbbciUAK4aYhizHQ+wLmqMQrn+RZyjLR+FNjgIzW6Odv6si5RzIuiqUUJlEgwAjUmHDWNR43aar0aLF5tfolQgDVObcaMUoK1gOfY5HNbdzIKrQsu9BTt0e5xAveKgotaznxcsXVzSaW1wUAvlLR41EDvur\\\\\\\/HFnFr3W5k1aJsinSvbwlLAVhR7Az4tIWQ6EGoOJeE6S0Vv7SlYb4fuibALEYBaC3o141nwedIB1+uH2rmTXixp\\\\\\\/KwnE+SJ8p9jGHqRWAssBZg4oEXnYCHj1X7uL2gvqwNJTdGT+\\\\\\\/pb4UnxxHenwCtNkt9oehVpxbbAk5y2yDzJ0RVB7V2rNtiI7KGnXuIMWJ8YTvB1MQXIuKMMKpHxUwnqj\\\\\\\/u18cmcFPBfiV984tTBzo65QFDxt9GOBKnkOPsEFFqarDcGXw5mTibyr4PBAKxSrmDacEf1SpCS826R9Z+AIQDbPv7ZeVLBnwICAi+ALQkPszepHSiPSF6bcoStEi0IlcAfbjgxn6YJNQx93lzP0TxdKBqXMEm1fZnm2k6CnNgwICB2P1ibUtvGMimpwokaDf6eMuJL661Lqlemr3FnUxHHzbf2Ds58OVXBsBSnBV\\\\\\\/52Npdy96xkOQxCpaZtc2aI482Q8dM4gYhzvdBtzttX7GEh1z9yg7P0ayVlyKZQQ6if4kFf2EZtUsppCWHz8eTnNPibk9GV1t15ltlHmyvwWSbYSYuZrFGICGok7rq2irEnlx\\\\\\\/bWzadK3LnTWVITdGJiENvI6B962ZEsvfhXiXyABgnLZZTd39OIl+y0jNWCW1doNvHhqAWM0dOcnj30+Gciv+lxdvPF\\\\\\\/zPMiEfXTOXpnqPSnZYTO33Bd1sJLq83Cgf\\\\\\\/DkA\\\\\\\/DyVbNyqAL72OHiEb9jtwBloA2WBElw+kWf0HwR0RX2prs1qEAOhyRq08yDCjd5FcShL\\\\\\\/XFaEzVryDFiRFhurNTl96EiaAzRl8KEy4SSFBNXHsLx0hQ0jvb06AZ4RuL8HFjP3iDlXVOc+Lb86DKuI5AqEmrNfpH8FT0neZDb17lQOdVVws6lTjOykkgdhKWspNA2DH7szd5ypUf\\\\\\\/ZUAWrC5\\\\\\\/bKfPEDzp7e4XT2bUWH3\\\\\\\/b60LKjMt7JqXVRKH7C\\\\\\\/6dIjm8aLDLWF05mQgvTo5gryQDYrxMo+VItBF1bNJPQfd2ARe+neUnYCa6mCr4DCKFFuzdBk9viyE9PwPVB5aoNTETpcfjIM+gy2ulEEpm\\\\\\\/oveMI\\\\\\\/NY\\\\\\\/Dq6zbyQId5gNAqR7N5GbO21tys\\\\\\\/YddiGH1w3ndeIJnVc2T70VpXBjXmRwGQwCYF1RJ19qeKBhGOPILECakmamL1efP4hF2aSmMtV0i4FsRoMEoFe9eNfsHeFul+jkwmGYgTI1vNdf6TeLDChyMpVTTaFQ+2sBbIolLw8j3r2KV\\\\\\\/RzL+xWa3Mz60qiCv\\\\\\\/SNABIIP9Pp66uMIxlR7Vqsf3PTk9hStdxkUy1JOtIMYFAYQyrjZ4xO+dySuzB8hcUJ8eEJzfcqXmXENM7BoKNkKBy\\\\\\\/hOlDUX6E\\\\\\\/xgNPTlIihrkdV0voFuoopNuBlcRQ6tbkOcJnfCO971Fc7BPm9OyYGX6Ljhb3KXsMuRl6GSAQRNKJzUibs6MKxA6SXO5Gulh4lehXbK\\\\\\\/QH5dSfrweOAo2t00EApJivFRMpSfUk06HeJLivu+bAYid9zxHdwmnawgozGm0fu\\\\\\\/mbdg43Jy4kv\\\u0022,\\\u0022iv\\\u0022:\\\u0022Xsj5wW6jP4MtxgQbScSvXg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224ce06007d84f497118398c7d64761bff4774a07071731951e46d41db9ab2932b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GTlaUNWbmKcrrwHj3T4BH5AFe\\\\\\\/71ZfsOfaDtB4QfsBU09P3hCfB7M1rCNMRhB3rnv9XT+1\\\\\\\/n2bkUNDKDUSAob16fCpWThwLZJXhHqjMsw3gkOD1aY5bb5RBnySkZgGP+YRYPTKfQSnzfjiOBS5ZSUg\\\\\\\/YALRlB+DzsxUpWsz1cbxFG16urq\\\\\\\/UsBj9OEdJCnnXD7TC4yIyi5OTvLl0ze6sXYaMkLT9Td0LQFGSecf\\\\\\\/uMCri\\\\\\\/dX2gpJiaYO2R0GSPNTQvDGhWVR4Q9716MKr\\\\\\\/I3Z\\\\\\\/dKT53WtteDqKnKASiHt0hX+h1FqKsYOFFuih9k8qScfzpYUX9dsjHyqwyVkY1PhYT73vE4PIU5A4WdB3Jmp\\\\\\\/C\\\\\\\/iEH22Rz24bOHr8arCDgzcmyP62+sCBgC+oLbiuwL0gotIJt1nV2gzZSwHj8h5F0fshuiMMIUriSf3QkZxqWz5hhXahr6VA+oSebLpeG4dSmjbtxWm9zx3u1RBZ9Iy2p2h4+0L1qso7jGj+GsrclCSaGNabT32Wbmxnc96TY7Ppo8SJYLTsL91w7XqILGNOvgxKeRT9ajhDTxtaRQ4Nl9qQDLBNHyy0cpTFLu+\\\\\\\/op0z4OmsW80S2Tzs5YLLnxoDsDswGkuGDkrRjilKKY39Urms+Cg56VZBEPKJLsD9+FhiW02WdZY5AA+Ks03rSa7jvx\\\\\\\/hurHQgFoE8resjdm5dpSBj91v+uQp5DRiwiH7rRdlots\\\\\\\/d7pFrtY6rTEQFxxKQ5MR5CkWsekpRCKqAiW7CQgM4Tu5X2uV\\\\\\\/y\\\\\\\/gWQJELjWwBIqsGTIMGF5SEJWXtL2ECTF8QQxK9eg8op1fJDQcRKJMwF9\\\\\\\/Dem2ly+b2d5JqFh4+Rx+xvOm2EmbhozZP693qjuN0DSAqHAfJCcdi9DpuGIeQTCo0VQj+xfukysRTjKToGBR4fR4dRC+5YebKggopckwEgHC+Tb\\\\\\\/iv610rd4CBP0CImKrgkEo3h709qo1xq6A\\\\\\\/loljki1zMIc4YwayC6IiOBQ7qu2jUrzrBrlpjX8qevqFOtxxlz8VUu631sckiEM69yVJUKerUZ\\\\\\\/sMH64VYHvHUCwz6HiYQH0v6efWpODeyC6WzmLhsRGpR5zXBmAEC6XwmAhyCKa7gM+J2DRsYO6\\\\\\\/Kz1qbEnYEIukVpS7yhpmzChjFEr+oGFzw8ZBcnZZw2xUGZPhhkOF0F8Dho\\\\\\\/elWgHGyzsOjP9VdmAdCGLv2R2E\\\\\\\/Mt1509gf1kyiulkpEGsVxNC4mrSz4Iq4icWdUZW\\\\\\\/OhsultRDc9NGUc4+ggIYG7f2WImEEU1IzN94Z8040aQ\\\\\\\/W+cAFFzhCWuL5SH9abjq1FIwwOWWDciXP9Kd0WpTG7M2oVYjj+\\\\\\\/NyBGfssJda5poiwyFXApZJi5Tt+9tskIgKnX0nh39JVdpcFNXx1bnYKfRNd7tMP2GLsvBx5rLPUOtyGOWFiqdjKHsGKrVsP1CoWAjS2AvbBATduYRzSCZQfB1UbGZBmCWNTULYgP2hXaQwW9wRTRTeRYwXIYq4bdNAvgZnehSOGisKdSdWACzzrH4xgt8wroj9B2IjuUerAbON9nVdn3Lc7DAVMVC0KHl7mW+oy9kfK67Fvsek5US76nNolSjVjvxTqSuhbjvj6M+rkcd0MMZmULv4MlNNUbXl9N+wFqUR48UWjLSHzK3F9MxtwIbPQdPPotk+LrGC6DZjv5JBEyfkk4aAytyLqGiTXvylYl8fNus4LMhDC7YspqGueEqe15LX306HBslGFQdxfm+Hw+0RuToq8yk\\\\\\\/sw1eVqONpxfz5gNMYCBpo0\\\\\\\/n0AxpICxbDAYpKK2Nhm1N\\\\\\\/KVu\\\\\\\/GmTuZ5OQ+yagKhF45bso3X6H3nkAtEoIPN9njVrkcEzjdOyhpUJqvkJQ+9Hw1Pumd2owQsRx7UbvcRSZoGPe01HzhK7Fqn4cij7\\\\\\\/gnx21h6T4fT40i69DFnt4iPvJa\\\\\\\/IRnOE+j0zQkeW\\\\\\\/5BWv65AB0FrkRE4y8UzG0VNlK5cWuEh31EkrkEs2n0GFt0XfHkJp8PeauzhHioYkXNiFI=\\\u0022,\\\u0022iv\\\u0022:\\\u00223hCAXjb3ALoW7I5fQH4aPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002224de8968b2798c057438b27eb0eb42999befd5c2a72fff679a9c5ce04aa69123\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488772 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/981" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IkIvZQZ1yYvWm_sQQ1al1w7rMM82JeMuqasIDU8qts0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/981" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LoRCj27wCacXRdQCZNALv0M0guAM6oHmZ9PDxHhGE5k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/3/967" title="Systematic Functional Characterization of Human 21st Chromosome Orthologs in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/3/999" title="Tuning CRISPR-Cas9 Gene Drives in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488772" id="node488772" data-pisa="ggg;8/3/981" data-pisa-master="ggg;g3.117.300338" data-apath="/ggg/8/3/981.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Identification and Functional Characterization of Essential Genes in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhaozhao Qin</span>, <span class="highwire-citation-author" data-delta="1">Robert Johnsen</span>, <span class="highwire-citation-author" data-delta="2">Shicheng Yu</span>, <span class="highwire-citation-author" data-delta="3">Jeffrey Shih-Chieh Chu</span>, <span class="highwire-citation-author" data-delta="4">David L. Baillie</span> and <span class="highwire-citation-author" data-delta="5">Nansheng Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">981-997;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300338 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488772" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhaozhao Qin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaozhao%2BQin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qin%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaozhao%2BQin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert Johnsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJohnsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnsen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJohnsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shicheng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Combinatorial Biosynthesis and Drug Discovery, School of Pharmaceutical Sciences, Wuhan University, 430071, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wuhan Frasergen Bioinformatics, Wuhan East Lake High-Tech Zone, 430075, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shicheng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShicheng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey Shih-Chieh Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wuhan Frasergen Bioinformatics, Wuhan East Lake High-Tech Zone, 430075, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BShih-Chieh%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BShih-Chieh%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David L. Baillie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BBaillie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baillie%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BL.%2BBaillie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nansheng Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nansheng%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANansheng%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chenn@sfu.ca" class="" data-icon-position="" data-hide-link-title="0">chenn@sfu.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/3/981" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488772" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488772/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/981.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488772" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488772/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/981.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488772" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488772/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/3/981.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488772" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488772/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/3/981.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488772" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488772/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Using combined genetic mapping, Illumina sequencing, bioinformatics analyses, and experimental validation, we identified 60 essential genes from 104 lethal mutations in two genomic regions of <em>Caenorhabditis elegans</em> totaling ∼14 Mb on chromosome III(mid) and chromosome V(left). Five of the 60 genes had not previously been shown to have lethal phenotypes by RNA interference depletion. By analyzing the regions around the lethal missense mutations, we identified four putative new protein functional domains. Furthermore, functional characterization of the identified essential genes shows that most are enzymes, including helicases, tRNA synthetases, and kinases in addition to ribosomal proteins. Gene Ontology analysis indicated that essential genes often encode for enzymes that conduct nucleic acid binding activities during fundamental processes, such as intracellular DNA replication, transcription, and translation. Analysis of essential gene shows that they have fewer paralogs, encode proteins that are in protein interaction hubs, and are highly expressed relative to nonessential genes. All these essential gene traits in <em>C. elegans</em> are consistent with those of human disease genes. Most human orthologs (90%) of the essential genes in this study are related to human diseases. Therefore, functional characterization of essential genes underlines their importance as proxies for understanding the biological functions of human disease genes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/essential-gene" class="hw-term hw-article-keyword hw-article-keyword-essential-gene" rel="nofollow">essential gene</a></li><li class="kwd"><a href="/keyword/lethal" class="hw-term hw-article-keyword hw-article-keyword-lethal" rel="nofollow">lethal</a></li><li class="kwd"><a href="/keyword/genetic-balancer" class="hw-term hw-article-keyword hw-article-keyword-genetic-balancer" rel="nofollow">genetic balancer</a></li><li class="kwd"><a href="/keyword/whole-genome-sequencing-wgs" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing-wgs" rel="nofollow">whole genome sequencing (WGS)</a></li><li class="kwd"><a href="/keyword/functional-characterization" class="hw-term hw-article-keyword hw-article-keyword-functional-characterization" rel="nofollow">functional characterization</a></li></ul><p id="p-4">Genes that are required for survival, or genes that strongly contribute to fitness and robust competitive growth are essential genes (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gerdes <em>et al.</em> 2006</a>). In humans, it has been demonstrated that mutations in essential genes contribute to a spectrum of human diseases, from developmental diseases (often resulting in spontaneous abortions) to a range of cancers, including brain and breast cancers (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dickerson <em>et al.</em> 2011</a>). Knowing what essential genes are and how they work will allow researchers to develop a deeper understanding of the molecular nature of such diseases. Identification of essential genes should also help to determine the minimal gene set, which is the smallest group of genes that are necessary and sufficient to sustain a functioning cell or organism under the most favorable conditions [reviewed in <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Koonin (2000)</a>]. Thus, identifying essential genes will further our understanding of the basics of cell functioning.</p><p id="p-5">Studies of essential genes are ubiquitous in model organisms. In mouse models using individual knockout strains, ∼2000 genes have been classified as essential. The deletion of these genes leads to either lethality before reproduction, or to sterility (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Liao and Zhang 2007</a>). In zebrafish, ∼390 genes have been identified through mutagenesis screens as essential for embryonic and early larval development (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amsterdam <em>et al.</em> 2004</a>). In <em>C. elegans</em>, genome-wide functional analysis, using RNAi, indicates that at least 1170 genes are essential (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kamath <em>et al.</em> 2003</a>). Although RNAi is an effective high-throughput technology for identifying essential genes, this technology has limitations that prevent it from being used to identify all essential genes in <em>C. elegans</em> (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kamath <em>et al.</em> 2003</a>).</p><p id="p-6">Genome-wide forward genetic screens are capable of isolating genes that play important roles in given phenotypes, including lethality. For over 35 yr, researchers have taken advantage of genetic balancers and forward genetic screens to isolate lethal mutations and identify essential genes in <em>C. elegans</em>. Using traditional genetic methods, including genetic mapping and transgenic rescue assays, many essential genes have been identified. However, due to the limitations of traditional genetic methods, the process of identifying essential genes is slow.</p><p id="p-7">Here we used whole genome sequencing (WGS) technology to speed up the identification of essential genes in <em>C. elegans</em> mutants isolated from genetic screens (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Rosenbluth and Baillie 1981</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Rosenbluth <em>et al.</em> 1983</a>, <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">1985</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Johnsen and Baillie 1991</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> 1998</a>). <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Janke <em>et al.</em> (1997)</a> was a harbinger of our work when they used sequenced cosmids to rescue lethal phenotypes on chromosome III and thus molecularly identify essential genes. In <em>C. elegans</em>, WGS is capable of capturing molecular lesions that were induced by ethyl methane sulfonate (EMS) mutagenesis in both homozygous (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sarin <em>et al.</em> 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rose <em>et al.</em> 2010</a>) and heterozygous backgrounds (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chu <em>et al.</em> 2012</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2014</a>). Along with bioinformatics analysis, different types of mutations can be detected, such as base pair alterations, direct repeat sequences, and deletions, as well as insertions of transposable elements (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Rose <em>et al.</em> 2010</a>). However, to decide which mutation is responsible for a lethal phenotype, genetic mapping evidence, used for assigning sequenced lethal mutations into individual genetic mapping zones, proved to be indispensable. The mapping zones were defined via complementation tests between sets of overlapping deficiencies and duplications. Once zones were established lethals were complementation tested against those rearrangements to determine which zone each lethal fell into. After that, the lethals were complementation tested against lethals in the same zone to determine if they were allelic or newly identified genes (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Rosenbluth and Baillie 1981</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rosenbluth <em>et al.</em> 1988</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Johnsen and Baillie 1991</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> 1998</a>). Since the breakpoints of several deficiencies on both chromosome III(mid) and chromosome V(left) have been precisely located in a physical map, the boundaries of the corresponding genetic mapping zones are also well defined (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2007</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2009</a>) thus narrowing considerably the molecular regions where the genes must reside. Methods such as single nucleotide polymorphism mapping (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Davis <em>et al.</em> 2005</a>) or a recently devised molecular inversion probes–based method (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mok <em>et al.</em> 2017</a>) are necessary when the approximate position of the mutations are not known. In our case, with well-mapped mutations, the bioinformatics analysis needed to discover the exact positions of the recovered lethal lesions is simpler.</p><p id="p-8">Homozygous mutations have a 100% allelic ratio, but lethal mutations need to be maintained as heterozygotes, which increases the difficulty of identifying lethal mutations. Furthermore, different balancer systems give rise to different heterozygous allelic ratios. For example, there are two copies of a recessive lethal mutation in a duplication-balanced region and one wild-type allele on the balancer. As a result, the allele frequency is two mutations to one wild type, which should result in a 66.7% variation frequency in the sequencing data. For translocations, only one copy of a lethal allele and one copy of a wild-type allele are present, and the variation frequency should be 50%. It has been demonstrated that duplication balanced essential genes can be identified when there is a 2:1 mutation-to-wild type allele ratio (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chu <em>et al.</em> 2012</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">2014</a>); however, when the ratio is 1:1 (using a translocation balancer), the identification becomes more difficult. In this study, we used genetic mapping data, WGS techniques, bioinformatics analyses, and experimental validation, to identify 60 essential genes from 104 lethal mutations in <em>C. elegans</em>. This was followed by functional characterization of those essential genes. We also studied the relationship between gene essentiality and gene duplicability, protein connectivity, gene expression, and showed that compared with nonessential genes, essential genes have fewer paralogs, encode proteins that are in interaction hubs, and are more highly expressed. The essential genes we identified provide a rich resource for future studies. The conserved essential genes should also prove useful for understanding the functions of homologous genes in human, especially disease-related genes. In addition, species-specific essential genes could be good candidates for targeting pathogenic nematodes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>C. elegans strains</h3><p id="p-9">All the strains that were submitted for sequencing and for complementation tests are included in the Supplemental Material, Table S2, Table S3, and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>WGS and computational analysis</h3><p id="p-10">Worms were grown on large (100 by 15 mm) Petri plates until the worms starved or the plates were covered in healthy gravid adults. We rinsed the worms off the plates at room temperature with M9 buffer (6 g Na<sub>2</sub>HPO<sub>4</sub>, 3 g KH<sub>2</sub>PO<sub>4</sub>, 5 g NaCl, 0.2 g MgSO<sub>4</sub> in 1 liter of H<sub>2</sub>0) into 15 ml polypropylene tubes, and pelleted them by centrifuging at 1500 rpm for 2 min at 4°. We then washed them in 12 ml iced M9 buffer, for three times. The worms were placed on a rocker for 2–3 hr at room temperature to digest bacteria. Worms were pelleted at 1500 rpm for 2 min at 4°. Then, the supernatant was removed and the worms were washed and pelleted in 4 ml room temperature M9 buffer. The pellets were frozen at −80°.</p><p id="p-11">The genomic DNA collection for Illumina sequencing was constructed using the QIAGEN DNeasy Blood &amp; Tissue Kit (cat. no. 69504), following the protocol “Purification of Total DNA from Animal Tissues (Spin-Column Protocol).” DNA concentrations were determined using the Qubit dsDNA High Sensitivity Assay Kit performed with the Qubit 2.0 Fluorometer (Life Technologies). The purity of the DNA was assessed using the Nanodrop ND-1000 Spectrophotometer-T* (Thermo Scientific). For each sample, 0.6–2 μg of purified genomic DNA was submitted for sequencing at the British Columbia Cancer Agency Canada’s Michael Smith Genome Sciences Centre using the Illumina PET HiSeq technology.</p></div><div id="sec-4" class="subsection"><h3>WGS data analysis</h3><p id="p-12">For chromosome III, genomic DNA libraries of 49 strains were prepared and sequenced using Illumina PET HiSeq to generate 100-bp-long paired end reads. BWA (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li and Durbin 2009</a>), GATK (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>), and SAMtools (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>) were used to align the reads to the <em>C. elegans</em> reference genome (WS246) and called for variants. For chromosome V, the sequencing reads were first aligned with the <em>C. elegans</em> reference genome using BWA with default settings. WormBase WS249 was used as reference genome for the alignment. This step was followed by filtering out PCR-caused duplicates by using SAMtools (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>), which was also used for analyzing the sequencing depth of each strain. The breakpoints of large deletions, medium insertions, and translocations were detected and viewed using the Integrative Genomics Viewer (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Robinson <em>et al.</em> 2011</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Thorvaldsdottir <em>et al.</em> 2013</a>). The effect of the variations on each CDS in the genome was analyzed using CooVar (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Vergara <em>et al.</em> 2012</a>).</p></div><div id="sec-5" class="subsection"><h3>PCR sequencing and the rescue assay</h3><p id="p-13">Primers were designed using Primer3 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Koressaar and Remm 2007</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Untergasser <em>et al.</em> 2012</a>) to amplify the coding sequences of each candidate essential gene. PCR products were subjected to PCR purification (QIAGEN MinElute PCR Purification Kit, cat. no. 28004) or gel purification (QIAGEN QIAquick Gel Extraction Kit, cat. no. 28704) before they were sequenced using the Sanger technique. The same primers that were used in PCR amplification were also used for Sanger sequencing. The sequencing results were analyzed using the software SeqMan (DNASTAR).</p><p id="p-14">Fosmid DNA was prepared using the Pharmacia Mini-Prep Kit Plus (minus the final column purification step) and was diluted in ddH<sub>2</sub>0. Primers were designed and PCR amplifications were performed to validate the presence of the candidate genes in the tested fosmids and their absence in the negative control fosmids.</p><p id="p-15">The transgenic strains constructed for this research used the coinjection semidominant marker <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00248869;class=Variation">su1006</a></em>) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kramer <em>et al.</em> 1990</a>) in plasmid pCes1943 and another co-injected GFP marker <em>Pmyo-2</em>::<em>GFP</em>, which is expressed in the pharyngeal muscle (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dibb <em>et al.</em> 1989</a>). DNA mixtures that contain 2 ng fosmid DNA, 80 ng <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00248869;class=Variation">su1006</a></em>), and 2.5 ng <em>Pmyo-2</em>::<em>GFP</em> were directly injected into the syncytial gonad of young adults of the mutant hermaphrodites (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Mello <em>et al.</em> 1991</a>). The presence of DpyUnc F<sub>2</sub> progeny with coinjected markers indicated successful rescue.</p></div><div id="sec-6" class="subsection"><h3>Interallelic complementation tests</h3><p id="p-16">Complementation tests were carried out for genetic validation of the candidate essential genes. For lethals balanced by <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em>, Unc-32 hermaphrodites were crossed to <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> males. Phenotypically wild-type F<sub>1</sub> males were then crossed to another lethal with the same genetic background. The presence of fertile DpyUnc F<sub>2</sub> hermaphrodites indicates complementation, while the absence of fertile DpyUnc F<sub>2</sub> hermaphrodites indicates failure to complement (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> 1998</a>). For mutants balanced with <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>, phenotypically wild-type males containing a lethal mutation on chromosome V were crossed to another <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em> balanced lethal. The presence of fertile DpyUnc F<sub>1</sub> progeny indicates complementation, while the absence of fertile DpyUnc progeny indicates failure to complement. In some cases, late blocking, sterile, or maternal effect individual F<sub>1</sub>s were set up in order to observe the terminal phenotypes (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Johnsen and Baillie 1991</a>).</p></div><div id="sec-7" class="subsection"><h3>Essential genes functional analyses</h3><div id="sec-8" class="subsection"><h4>Protein domain analysis:</h4><p id="p-17">A known annotated domain for each protein was searched with InterProScan (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mulder and Apweiler 2007</a>) using the Pfam database (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Finn <em>et al.</em> 2014</a>) and visualized with the Perl module FeatureStack (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Frech <em>et al.</em> 2012</a>). Identification of orthologs of <em>C. elegans</em> proteins was conducted using InParanoid (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Remm <em>et al.</em> 2001</a>) in 25 other nematode species and four model organisms: <em>Homo sapiens</em>, <em>Drosophila melanogaster</em>, <em>Mus musculus</em>, and <em>Danio rerio</em>. This was followed by multiple sequence alignments with ClustalX2 (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Thompson <em>et al.</em> 1997</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Larkin <em>et al.</em> 2007</a>) between <em>C. elegans</em> proteins and their orthologs. The alignment was examined using Jalview version 2 with the “ClustalX” color scheme.</p></div><div id="sec-9" class="subsection"><h4>Gene Ontology analysis:</h4><p id="p-18">Gene Ontology (GO) was performed using the PANTHER classification system (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mi <em>et al.</em> 2013</a>) from the website <a href="http://pantherdb.org/">http://pantherdb.org/</a>. Three GO term categories (cellular component, biological process, and molecular function) were examined individually.</p></div><div id="sec-10" class="subsection"><h4>Gene duplicability:</h4><p id="p-19">An all-against-all BLASTP search was conducted for the whole set of <em>C. elegans</em> proteins (WS250). Only the longest isoform was used if there are multiples transcripts of a gene. This was followed by the computation of the global PID. Only protein pairs that had a PID equal to or higher than 50% were kept and sorted by <em>e</em>-value.</p></div><div id="sec-11" class="subsection"><h4>Protein connectivity:</h4><p id="p-20">The whole genome protein interactions in <em>C. elegans</em> were downloaded from BioGRID (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Stark <em>et al.</em> 2006</a>), which is an interaction repository with data compiled through comprehensive curation efforts. BioGRID currently holds &gt;8037 nonredundant interactions in <em>C. elegans</em> with 3949 unique genes that were derived from 205 publications, including both physical and genetic evidence. This was followed by the filtration of proteins that are from one of four groups.</p></div><div id="sec-12" class="subsection"><h4>Gene expression:</h4><p id="p-21">Gene expression data were downloaded through the <em>GExplore</em> (version 1.4) expression search interface (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hutter <em>et al.</em> 2009</a>), which includes a series of <em>C. elegans</em> developmental stages from parts of the NHGRI modENCODE project (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hillier <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gerstein <em>et al.</em> 2010</a>). The data were derived from synchronized whole animals from embryonic and postembryonic stages, followed by RNA-seq.</p></div></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-22">Strains are available upon request. Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a> contains detailed information of all identified essential genes and lethal mutations. Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a> includes a total of 86 strains with <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>;<em>V</em>) that were sequenced. Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a> contains a total of 49 strains with <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> sequenced. Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a> includes all the strains used in the complementation tests. The WGS data of all 135 sequenced strains are publicly available on the NCBI with the BioProject accession number PRJNA416306.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Balancer systems</h3><p id="p-23">Lethal mutations isolated from two balancers systems were used in our study: <em><a href="http://www.wormbase.org/db/get?name=WBGene00001077;class=Gene">dpy-18</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143150;class=Variation">e364</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>); <em><a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>V</em>) and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation">e189</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene">lin-12</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00089843;class=Variation">n941</a></em>) <em>III</em>;<em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> (<em>III</em>;<em>f</em>). The first, the eT1-system, is a reciprocal translocation between the right half of chromosome III [<em>LGIII</em>(right)] and the left half of chromosome V [<em>LGV</em>(left)], recombinationally balancing those two regions (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Rosenbluth and Baillie 1981</a>), which is ∼20% of <em>C. elegans</em>’ genome. <em>LGV</em>(left) contains ∼7% (23 map units) of the recombination distance in the genome and ∼10% of its DNA. <em>LGV</em>(left) has been subdivided into 22 main zones and several additional subzones by sets of overlapping rearrangements (deficiencies and one duplication) (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> courtesy of M. Jones) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Johnsen and Baillie 1988</a>, <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">1991</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Rosenbluth <em>et al.</em> 1988</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Clark <em>et al.</em> 1990</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Stewart <em>et al.</em> 1991</a>). Over 120 nonessential (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Edgley and Riddle 1990</a>) and essential genes have been mapped to the zones in <em>LGV</em>(left), making it a genetically well-defined region. A number of different types of mutagens have been characterized in the <a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a>-system including formaldehyde (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Johnsen and Baillie 1988</a>), ultraviolet irradiation (UV) (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Stewart et al. 1991</a>), the transposable element Tc1 (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Clark et al. 1990</a>), and also EMS, which induces primarily point mutations (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Rosenbluth <em>et al.</em> 1983</a>; <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Johnsen and Baillie 1991</a>), and other mutations that generally induce rearrangements γ-irradiation (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Rosenbluth <em>et al.</em> 1985</a>). <em><a href="http://www.wormbase.org/db/get?name=WBGene00001077;class=Gene">dpy-18</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143150;class=Variation">e364</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>); <em><a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>V</em>) is phenotypically wild type; pseudolinked <em><a href="http://www.wormbase.org/db/get?name=WBGene00001077;class=Gene">dpy-18</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em> have a dumpy uncoordinated (Dpy-18 Unc-46) phenotype while <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>) breaks in <em><a href="http://www.wormbase.org/db/get?name=WBGene00006772;class=Gene">unc-36</a></em>, thus giving homozygous <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em> a visible Unc-36 (uncoordinated) phenotype. Screening for the absence of fertile Dpy-18 Unc-46 indicates a lethal mutation in the balanced regions, after which the lethal mutations can be mapped to either <em>LGIII</em> or <em>LGV</em>. Those mutations on <em>LGV</em> can be complementation tested against the set of overlapping rearrangement to determine the zone in which the lethal mutation lies. Further complementation tests against genes in that zone determine which gene the mutation is allelic to.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/981/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Physical deficiency map of LGIII(mid) and LGV(left). The top figure shows the middle region (4.5 Mb) of chromosome III that is balanced by a free duplication (sDp3), which has been subdivided into 14 main zones. The bottom figure shows the left half of chromosome V that is balanced by a reciprocal translocation (eT1), which has been subdivided into 22 main zones and several additional subzones by sets of overlapping rearrangements (deficiencies and one duplication)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1298489251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Physical deficiency map of &lt;em&gt;LGIII&lt;/em&gt;(mid) and &lt;em&gt;LGV&lt;/em&gt;(left). The top figure shows the middle region (4.5 Mb) of chromosome III that is balanced by a free duplication (&lt;em&gt;sDp3&lt;/em&gt;), which has been subdivided into 14 main zones. The bottom figure shows the left half of chromosome V that is balanced by a reciprocal translocation (&lt;em&gt;eT1&lt;/em&gt;), which has been subdivided into 22 main zones and several additional subzones by sets of overlapping rearrangements (deficiencies and one duplication).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/981/F1.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/3/981/F1.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/981/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/981/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488935" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Physical deficiency map of <em>LGIII</em>(mid) and <em>LGV</em>(left). The top figure shows the middle region (4.5 Mb) of chromosome III that is balanced by a free duplication (<em>sDp3</em>), which has been subdivided into 14 main zones. The bottom figure shows the left half of chromosome V that is balanced by a reciprocal translocation (<em>eT1</em>), which has been subdivided into 22 main zones and several additional subzones by sets of overlapping rearrangements (deficiencies and one duplication).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The second system (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> 1998</a>) uses a free duplication (<em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em>) (which does not recombine with the normal <em>LGIII</em> chromosome) to balance a 4.5 Mb region of <em>LGIII</em> (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> system consists of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation">e189</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene">lin-12</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00089843;class=Variation">n941</a></em>) <em>III</em>;<em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> (<em>III</em>;<em>f</em>) in which <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> covers <em><a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em> (which has a dumpy phenotype) (<em><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene">lin-12</a></em> is also covered by <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> and has a visible phenotype) but not <em><a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em> (which has an uncoordinated phenotype), so when the duplication is present over <em><a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation">e189</a></em>), the phenotype is Unc-32 but when absent it is Dpy-17 Unc-32. <em><a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation">e189</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00003001;class=Gene">lin-12</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00089843;class=Variation">n941</a></em>) <em>III</em>;<em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> (<em>III</em>;<em>f</em>) (which has a wild-type phenotype) was mutagenized and the F<sub>2</sub> were screened for the absence of Dpy-17 Unc-32 progeny, indicating a putative lethal mutation in the <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> balanced region on the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation">e189</a></em>) marked chromosome, <em>i.e.</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em>let-x <a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation">e189</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em>let-x <a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation">e189</a></em>) <em>III</em>;<em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> (<em>III</em>;<em>f</em>). Strains were maintained as homozygotes with <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> by picking Unc-32. The <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> balanced region has also been divided into zones by a set of overlapping rearrangements. <a id="xref-ref-69-5" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> (1998</a>) reported 112 essential genes in the <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> balanced region. A difference between the two systems is that in the eT1-system, the lethals were isolated as heterozygotes whereas in the sDp3-system the lethals were picked up as homozygotes covered by duplication. We did not note any obvious differences in the types of genes isolated by the two systems.</p></div><div id="sec-16" class="subsection"><h3>Identification of genomic variations in chromosome III mutations balanced by sDp3</h3><p id="p-26">Genomic DNA libraries for 49 lethals (<em><a href="http://www.wormbase.org/db/get?name=WBGene00001076;class=Gene">dpy-17</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143004;class=Variation">e164</a></em>) <em>let-x <a href="http://www.wormbase.org/db/get?name=WBGene00006768;class=Gene">unc-32</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143023;class=Variation">e189</a></em>);<em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> (<em>III</em>;<em>f</em>)) were prepared and sequenced using Illumina PET HiSeq to generate 100 bp paired end reads. We used BWA (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Li and Durbin 2009</a>), GATK (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>), and SAMtools (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>) to align the reads to a <em>C. elegans</em> reference genome (WS246) and called for variants. The average coverage for the 49 strains was 25×, with the lowest being 12× and the highest being 38×. The lethal mutations on chromosome III were balanced by <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> and thus we expected the variant frequencies to be ∼67%. Single nucleotide variations (SNVs) that occurred with variant frequencies between 40 and 90% were selected from the 49 strains and subjected to three filtration steps. First, mutations found as part of the Million Mutation Project (MMP) are presumptively false positives, since the way MMP was designed should drastically reduce but not eliminate the chances of capturing lethal mutations (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Flibotte <em>et al.</em> 2010</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Thompson <em>et al.</em> 2013</a>). One possible way the MMP could capture a lethal mutations is to capture a corresponding suppressor, but our initial assumption is “presumptive false positives.” Second, some variations (background variations) could stem from the starting strain that was used for mutagenesis. To be cost-effective, we did not sequence the starting strain because the majority of the <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> balanced strains stemmed from it, thus they shared the same genetic background; instead, we assumed all variations that occurred multiple times in the 49 sequenced strained originated in the starting strain and excluded them. Third, we required the variations to be supported by at least eight reads in both forward and reverse directions. After the three steps of filtration the remaining SNVs were subjected to further lethal mutation identification <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488858/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488858/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average number of SNVs per strain with 20–100% variation frequency in the <em>eT1(V)</em> balanced region, before and after the removal of background variations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Identification of genomic variations in eT1 balanced chromosome V mutations</h3><p id="p-27">Before identifying the genomic variations in the <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em> balanced mutations, there was a technical challenge that we needed to address. The progeny of these mutants include both phenotypically wild-type animals, which are <em><a href="http://www.wormbase.org/db/get?name=WBGene00001077;class=Gene">dpy-18</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143150;class=Variation">e364</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>); <em>let-x <a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>V</em>), and Unc-36 <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>;<em>V</em>) animals (because <a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a> breaks in <em><a href="http://www.wormbase.org/db/get?name=WBGene00006772;class=Gene">unc-36</a></em>). These homozygous Unc-36 worms are fertile and contribute to the total sequenced genomic DNA, which makes the allelic ratios of lethal mutations very difficult to predict and effectively impossible to analyze (D. L. Baillie, unpublished data). An important step is to drastically minimize the contribution of <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em> DNA so that the majority of sequenced DNA comes from <em><a href="http://www.wormbase.org/db/get?name=WBGene00001077;class=Gene">dpy-18</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143150;class=Variation">e364</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>); <em>let-x <a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>V</em>), allowing the allelic ratio of lethal mutations to approach 50% and thus facilitate the identification of lethal mutations. To do this, we constructed new strains for each mutant using <em><a href="http://www.wormbase.org/db/get?name=WBGene00002692;class=Gene">let-500</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242249;class=Variation">s2165</a></em>), which is located on the <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em> balancer resulting in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001077;class=Gene">dpy-18</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143150;class=Variation">e364</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>); <em><a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em>) <em>let-x/<a href="http://www.wormbase.org/db/get?name=WBGene00002692;class=Gene">let-500</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242249;class=Variation">s2165</a></em>) <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>V</em>). <em><a href="http://www.wormbase.org/db/get?name=WBGene00002692;class=Gene">let-500</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242249;class=Variation">s2165</a></em>) blocks development early and thus drastically reduces the contribution of homozygous <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em> DNA to our sequence analyzes. This step proved extremely effective and useful for the identification of heterozygous mutations.</p><p id="p-28">Genomic DNA libraries for 86 mutants were prepared and sequenced using Illumina PET HiSeq to generate 150 bp paired end reads. BWA (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Li and Durbin 2009</a>) and SAMtools (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>) were used to align the paired end reads to the WS249 <em>C. elegans</em> reference genome and postalignment processing analysis, respectively. This yielded an average of 31-fold genome coverage per sample (lowest coverage 23-fold, highest coverage 44-fold). We then used VarScan2 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Koboldt <em>et al.</em> 2012</a>) to identify SNVs that occurred with variant frequencies between 20 and 100%, which was followed by three filtration steps as used in the identification of variations in the <em><a href="http://www.wormbase.org/db/get?name=sDp3;class=Rearrangement">sDp3</a></em> balanced region. After filtration, the numbers of remaining SNVs are summarized in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><p id="p-29">The 86 sequenced strains have four different visible markers. Two (<em><a href="http://www.wormbase.org/db/get?name=WBGene00001077;class=Gene">dpy-18</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143150;class=Variation">e364</a></em>)<em>III</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em>)<em>V</em>) were molecularly identified before and successfully validated in this study. Both those visible markers are heterozygous and balanced by <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>;<em>V</em>), and the sequencing data showed average variation frequencies of <em><a href="http://www.wormbase.org/db/get?name=WBVar00143150;class=Variation">e364</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em> are 50 and 51%, respectively. Our analysis also identified the previously unknown mutations in the other two <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em>(<em>III</em>;<em>V</em>) balanced visible markers (<em><a href="http://www.wormbase.org/db/get?name=WBGene00006778;class=Gene">unc-42</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143077;class=Variation">e270</a></em>)<em>V</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006794;class=Gene">unc-60</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143397;class=Variation">e677</a></em>)<em>V</em>). We curated their mutation types and genomic positions: <em><a href="http://www.wormbase.org/db/get?name=WBVar00143077;class=Variation">e270</a></em> (A-&gt;T at position 976,514) causes a valine to glutamic acid change in four isoforms of <em><a href="http://www.wormbase.org/db/get?name=WBGene00006778;class=Gene">unc-42</a></em>; and <em><a href="http://www.wormbase.org/db/get?name=WBVar00143397;class=Variation">e677</a></em> (G-&gt;A at position 1,476,888) causes an amino acid change from glycine to arginine, which inhibits the normal function of the muscle-specific isoform UNC-60B.</p></div><div id="sec-18" class="subsection"><h3>Identification of essential genes and experimental validation</h3><p id="p-30">Using the identified genomic variations, we performed essential gene identification and experimental validation for both chromosome III(mid) and chromosome V(left).</p><p id="p-31">For the chromosome III(mid) dataset, 49 mutants with 49 lethal mutations representing 43 genes were sequenced. Six genes had two sequenced alleles while 37 genes had single alleles sequenced. The 49 mutations had previously been assigned to 14 genetic mapping zones (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a>). The 86 mutants from chromosome V(left) had previously been mapped to 23 zones (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a>), representing 46 genes. Two alleles were sequenced for 40 of the 46 genes while only six genes had a single allele sequenced. In total, there are 89 essential genes from chromosome III(mid) and chromosome V(left) that need to be identified molecularly.</p><p id="p-32">The molecular identification of essential genes on chromosome III(mid) and chromosome V(left) was determined by using five lines of evidence. First, previous genetic mapping data narrowed down the location of the lethal mutation for each strain (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Rosenbluth and Baillie 1981</a>; <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Rosenbluth <em>et al.</em> 1988</a>; <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Johnsen and Baillie 1991</a>; <a id="xref-ref-69-6" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> 1998</a>). Second, bioinformatics analyses identified a limited number of variations in the genetically determined regions (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Third, there are two independent sequenced alleles for most of the genes analyzed, which not only makes genomic identification easier but the result for each allele provides validation for the other allele. Fourth, we used WormBase information about lethal phenotypes, including those supported by RNAi (<a href="http://www.wormbase.org">www.wormbase.org</a>), to further narrow our search for variations in our sequences. Last, the MMP dataset was used to help verify possible essential genes. In MMP, 2007 strains were subjected to mutagenesis and &gt;800,000 different mutations were recovered in 20,115 genes. The large number of variations provides an unprecedented genetic resource for <em>C. elegans</em> research. However, during the MMP genetic screens only mutants with nonlethal phenotypes were isolated, and so most chain-terminating mutations such as the majority of nonsense mutations or splicing mutations in essential genes would be selected against (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Thompson <em>et al.</em> 2013</a>). In general, only missense mutations that do not cause malfunctioning proteins would be recovered. An essential gene example is the ribosomal large subunit <em><a href="http://www.wormbase.org/db/get?name=WBGene00004434;class=Gene">rpl-22</a></em>, for which MMP recovers only missense mutations. Using this hypothesis, we checked candidate genes by searching the MMP database for lack of chain-terminating mutations. With the above methodology we identified, with high confidence, 62 out of 89 sequenced essential genes in the chromosome III and chromosome V balanced regions. The majority of genes with only one allele identified on chromosome III had secondary support by lethal alleles from other laboratories, published RNAi analysis, or PCR results (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a>). In the case of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002904;class=Gene">let-786</a></em>, sequencing led to two candidate genes: <em><a href="http://www.wormbase.org/db/get?name=WBGene00004187;class=Gene">prp-8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006840;class=Gene">unc-116</a></em>. Two alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002846;class=Gene">let-712</a></em> (<em><a href="http://www.wormbase.org/db/get?name=WBVar00242285;class=Variation">s2439</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBVar00242360;class=Variation">s2598</a></em>) are in <em><a href="http://www.wormbase.org/db/get?name=WBGene00004187;class=Gene">prp-8</a></em>. Both <em><a href="http://www.wormbase.org/db/get?name=WBGene00002846;class=Gene">let-712</a></em> alleles complement <em><a href="http://www.wormbase.org/db/get?name=WBGene00002904;class=Gene">let-786</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242384;class=Variation">s2631</a></em>), therefore <em><a href="http://www.wormbase.org/db/get?name=WBGene00002904;class=Gene">let-786</a></em> is not <em><a href="http://www.wormbase.org/db/get?name=WBGene00004187;class=Gene">prp-8</a></em>. <em><a href="http://www.wormbase.org/db/get?name=WBGene00006840;class=Gene">unc-116</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00241507;class=Variation">rh24</a></em>) is lethal, therefore <em><a href="http://www.wormbase.org/db/get?name=WBGene00006840;class=Gene">unc-116</a></em> is a lethal gene and we concluded that <em><a href="http://www.wormbase.org/db/get?name=WBGene00006840;class=Gene">unc-116</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002904;class=Gene">let-786</a></em> are likely the same gene.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488841/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488841/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The number of essential genes identified in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">Fourteen genes were confirmed by experiments using PCR sequencing, rescue assay, and complementation tests. In the chromosome III and chromosome V balanced regions, 37 and 4, respectively, have only one Illumina sequenced allele. For five genes on III and four on V, there were multiple variations in different candidate genes in the zones they were mapped to. To test the candidate genes, we took advantage of the second alleles of those genes (<a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Johnsen and Baillie 1991</a>; <a id="xref-ref-69-7" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> 1998</a>). We PCR-amplified the candidate genes using genomic DNA of a strain with the second allele, which was followed by Sanger sequencing both ends of the PCR fragments. Using this method, we were able to identify five genes on chromosome III and four on chromosome V.</p><p id="p-34"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> includes two examples of validated genes, one from chromosome III and the other from chromosome V, which were successfully validated. Due to the heterozygosity of the lethal mutation balanced by duplications or translocations, one would expect to see overlapping sequencing peaks for both the wild-type allele and mutation allele at the same position. Because duplication balancers cover two lethal alleles over one wild-type allele, we see the height of the sequence peak of the mutant allele is slightly higher than the wild-type allele. However, the displayed nucleotide can be the wild-type allele or the mutant allele when Sanger sequencing was performed from both ends of the PCR products (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). For translocations, the ratio of mutant allele to wild-type allele is 1:1, therefore the sequencing peaks from the two alleles have identical heights. However, the displayed nucleotide for <em><a href="http://www.wormbase.org/db/get?name=WBGene00002552;class=Gene">let-327</a></em> is the wild-type allele when sequenced from both strands (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). In either case, considering that the relative abundances of the wild-type allele and mutated allele are not easily distinguishable in heterozygotes, the lethal mutations could be easily missed.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Three examples of PCR sequencing validation and rescue assay. (A) Gene validation of algn-1/T26AA5.4 for let-747 on chromosome III. (B) Gene validation of T08B1.1 for let-327 on chromosome V. The gene model in each figure shows that the designed primers cover all coding exons of each gene. The red arrows are forward primers while the arrows in dark blue are reverse primers. Under each gene model, there are partials of Sanger sequencing results using forward and reverse PCR primers as sequencing primers. For let-747/algn-1/T26AA5.4, a point mutation (G-&gt;A) was found in the last exon causing a premature stop codon. A C-&gt;T point mutation was found in the second exon of T08B1.1a and the first exon of T08B1.1b. A premature stop codon occurred in both T08B1.1 transcripts because of this mutation. (C) Fosmids used for the rescue assay. The fosmid (WRM0619aE07) containing the candidate gene, F11A3.2, is in the red box while WRM066cA08 without F11A3.2 is in the green box. F11A3.2 is in the black box with the designed forward primer and reverse primer underneath. The gel image shows the presence of F11A3.2 in WRM0619aE07 with a target band of 794 bp and the absence of F11A3.2 in WRM066cA08 without the 794 bp PCR product. The weak band of &#x223C;200 bp might be a nonspecific band because it can be found in both products using two fosmids as template." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1298489251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Three examples of PCR sequencing validation and rescue assay. (A) Gene validation of &lt;em&gt;algn-1/T26AA5.4&lt;/em&gt; for &lt;em&gt;let-747&lt;/em&gt; on chromosome III. (B) Gene validation of &lt;em&gt;T08B1.1&lt;/em&gt; for &lt;em&gt;let-327&lt;/em&gt; on chromosome V. The gene model in each figure shows that the designed primers cover all coding exons of each gene. The red arrows are forward primers while the arrows in dark blue are reverse primers. Under each gene model, there are partials of Sanger sequencing results using forward and reverse PCR primers as sequencing primers. For &lt;em&gt;let-747/algn-1/T26AA5.4&lt;/em&gt;, a point mutation (G-&gt;A) was found in the last exon causing a premature stop codon. A C-&gt;T point mutation was found in the second exon of &lt;em&gt;T08B1.1a&lt;/em&gt; and the first exon of &lt;em&gt;T08B1.1b&lt;/em&gt;. A premature stop codon occurred in both &lt;em&gt;T08B1.1&lt;/em&gt; transcripts because of this mutation. (C) Fosmids used for the rescue assay. The fosmid (&lt;em&gt;WRM0619aE07&lt;/em&gt;) containing the candidate gene, &lt;em&gt;F11A3.2&lt;/em&gt;, is in the red box while &lt;em&gt;WRM066cA08&lt;/em&gt; without &lt;em&gt;F11A3.2&lt;/em&gt; is in the green box. &lt;em&gt;F11A3.2&lt;/em&gt; is in the black box with the designed forward primer and reverse primer underneath. The gel image shows the presence of &lt;em&gt;F11A3.2&lt;/em&gt; in &lt;em&gt;WRM0619aE07&lt;/em&gt; with a target band of 794 bp and the absence of &lt;em&gt;F11A3.2&lt;/em&gt; in &lt;em&gt;WRM066cA08&lt;/em&gt; without the 794 bp PCR product. The weak band of &#x223C;200 bp might be a nonspecific band because it can be found in both products using two fosmids as template.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-2.medium.gif" width="417" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-2.medium.gif" width="417" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488930" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Three examples of PCR sequencing validation and rescue assay. (A) Gene validation of algn-1/T26AA5.4 for let-747 on chromosome III. (B) Gene validation of T08B1.1 for let-327 on chromosome V. The gene model in each figure shows that the designed primers cover all coding exons of each gene. The red arrows are forward primers while the arrows in dark blue are reverse primers. Under each gene model, there are partials of Sanger sequencing results using forward and reverse PCR primers as sequencing primers. For let-747/algn-1/T26AA5.4, a point mutation (G-&gt;A) was found in the last exon causing a premature stop codon. A C-&gt;T point mutation was found in the second exon of T08B1.1a and the first exon of T08B1.1b. A premature stop codon occurred in both T08B1.1 transcripts because of this mutation. (C) Fosmids used for the rescue assay. The fosmid (WRM0619aE07) containing the candidate gene, F11A3.2, is in the red box while WRM066cA08 without F11A3.2 is in the green box. F11A3.2 is in the black box with the designed forward primer and reverse primer underneath. The gel image shows the presence of F11A3.2 in WRM0619aE07 with a target band of 794 bp and the absence of F11A3.2 in WRM066cA08 without the 794 bp PCR product. The weak band of &#x223C;200 bp might be a nonspecific band because it can be found in both products using two fosmids as template." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1298489251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Three examples of PCR sequencing validation and rescue assay. (A) Gene validation of &lt;em&gt;algn-1/T26AA5.4&lt;/em&gt; for &lt;em&gt;let-747&lt;/em&gt; on chromosome III. (B) Gene validation of &lt;em&gt;T08B1.1&lt;/em&gt; for &lt;em&gt;let-327&lt;/em&gt; on chromosome V. The gene model in each figure shows that the designed primers cover all coding exons of each gene. The red arrows are forward primers while the arrows in dark blue are reverse primers. Under each gene model, there are partials of Sanger sequencing results using forward and reverse PCR primers as sequencing primers. For &lt;em&gt;let-747/algn-1/T26AA5.4&lt;/em&gt;, a point mutation (G-&gt;A) was found in the last exon causing a premature stop codon. A C-&gt;T point mutation was found in the second exon of &lt;em&gt;T08B1.1a&lt;/em&gt; and the first exon of &lt;em&gt;T08B1.1b&lt;/em&gt;. A premature stop codon occurred in both &lt;em&gt;T08B1.1&lt;/em&gt; transcripts because of this mutation. (C) Fosmids used for the rescue assay. The fosmid (&lt;em&gt;WRM0619aE07&lt;/em&gt;) containing the candidate gene, &lt;em&gt;F11A3.2&lt;/em&gt;, is in the red box while &lt;em&gt;WRM066cA08&lt;/em&gt; without &lt;em&gt;F11A3.2&lt;/em&gt; is in the green box. &lt;em&gt;F11A3.2&lt;/em&gt; is in the black box with the designed forward primer and reverse primer underneath. The gel image shows the presence of &lt;em&gt;F11A3.2&lt;/em&gt; in &lt;em&gt;WRM0619aE07&lt;/em&gt; with a target band of 794 bp and the absence of &lt;em&gt;F11A3.2&lt;/em&gt; in &lt;em&gt;WRM066cA08&lt;/em&gt; without the 794 bp PCR product. The weak band of &#x223C;200 bp might be a nonspecific band because it can be found in both products using two fosmids as template.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-3.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-3.medium.gif" width="342" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/981/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488930" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Three examples of PCR sequencing validation and rescue assay. (A) Gene validation of <em>algn-1/T26AA5.4</em> for <em>let-747</em> on chromosome III. (B) Gene validation of <em>T08B1.1</em> for <em>let-327</em> on chromosome V. The gene model in each figure shows that the designed primers cover all coding exons of each gene. The red arrows are forward primers while the arrows in dark blue are reverse primers. Under each gene model, there are partials of Sanger sequencing results using forward and reverse PCR primers as sequencing primers. For <em>let-747/algn-1/T26AA5.4</em>, a point mutation (G-&gt;A) was found in the last exon causing a premature stop codon. A C-&gt;T point mutation was found in the second exon of <em>T08B1.1a</em> and the first exon of <em>T08B1.1b</em>. A premature stop codon occurred in both <em>T08B1.1</em> transcripts because of this mutation. (C) Fosmids used for the rescue assay. The fosmid (<em>WRM0619aE07</em>) containing the candidate gene, <em>F11A3.2</em>, is in the red box while <em>WRM066cA08</em> without <em>F11A3.2</em> is in the green box. <em>F11A3.2</em> is in the black box with the designed forward primer and reverse primer underneath. The gel image shows the presence of <em>F11A3.2</em> in <em>WRM0619aE07</em> with a target band of 794 bp and the absence of <em>F11A3.2</em> in <em>WRM066cA08</em> without the 794 bp PCR product. The weak band of ∼200 bp might be a nonspecific band because it can be found in both products using two fosmids as template.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">No mutation was identified in the zones containing <em><a href="http://www.wormbase.org/db/get?name=WBGene00002681;class=Gene">let-470</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242144;class=Variation">s1581</a></em>) and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002681;class=Gene">let-470</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242172;class=Variation">s1629</a></em>). However, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002681;class=Gene">let-470</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242144;class=Variation">s1581</a></em>) has only <em><a href="http://www.wormbase.org/db/get?name=WBGene00008670;class=Gene">F11A3.2</a></em> as a candidate essential gene, with a splicing acceptor mutation besides the zone boundary. To test if <em><a href="http://www.wormbase.org/db/get?name=WBGene00008670;class=Gene">F11A3.2</a></em> is <em><a href="http://www.wormbase.org/db/get?name=WBGene00002681;class=Gene">let-470</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242172;class=Variation">s1629</a></em>), we performed rescue assay using two different fosmids. The result showed that the fosmids containing the wild-type copy of <em><a href="http://www.wormbase.org/db/get?name=WBGene00008670;class=Gene">F11A3.2</a></em> rescues the lethal phenotype of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002681;class=Gene">let-470</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242172;class=Variation">s1629</a></em>) while the fosmids without the candidate gene did not rescue the lethal phenotype. <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a> shows the genomic locations of two fosmids that were used for the rescue assay. PCR was also conducted to confirm the presence or absence of the candidate gene in the two fosmids.</p><p id="p-37">Complementation tests were carried out on four genes. <em><a href="http://www.wormbase.org/db/get?name=WBGene00002672;class=Gene">let-459</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242165;class=Variation">s1615</a></em>) contains a mutation that changes the splicing signal, causing a frame shift and premature stop codon in <em><a href="http://www.wormbase.org/db/get?name=WBGene00017982;class=Gene">hpo-18</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00017982;class=Gene">F32D1.2</a></em>. WGS analysis did not find a corroborating mutation in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002672;class=Gene">let-459</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242043;class=Variation">s1432</a></em>) sequence. A knockout allele <em><a href="http://www.wormbase.org/db/get?name=WBGene00017982;class=Gene">hpo-18</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00094440;class=Variation">ok3436</a></em>) is lethal. We did complementation tests between <em><a href="http://www.wormbase.org/db/get?name=WBGene00017982;class=Gene">hpo-18</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00094440;class=Variation">ok3436</a></em>) and both <em><a href="http://www.wormbase.org/db/get?name=WBGene00002672;class=Gene">let-459</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242043;class=Variation">s1432</a></em>) and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002672;class=Gene">let-459</a></em>(<em>1615</em>). Both tests showed failure to complement, which confirmed that <em><a href="http://www.wormbase.org/db/get?name=WBGene00017982;class=Gene">hpo-18</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002672;class=Gene">let-459</a></em> are the same gene. Complementation tests also showed that <em><a href="http://www.wormbase.org/db/get?name=WBGene00002662;class=Gene">let-447</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001177;class=Gene">egl-8</a></em> are the same gene. The phenotype of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001177;class=Gene">egl-8</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00088858;class=Variation">md1971</a></em>) includes reduced locomotion, reduced body flexion at rest, and loopy backing. Both <em><a href="http://www.wormbase.org/db/get?name=WBGene00002662;class=Gene">let-447</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242181;class=Variation">s1664</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=WBGene00001177;class=Gene">egl-8</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00088858;class=Variation">md1971</a></em>) and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002662;class=Gene">let-447</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242062;class=Variation">s1457</a></em>)<em>/<a href="http://www.wormbase.org/db/get?name=WBGene00001177;class=Gene">egl-8</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00088858;class=Variation">md1971</a></em>) have the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001177;class=Gene">egl-8</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00088858;class=Variation">md1971</a></em>) phenotype, and therefore failure to complement. Strains used for complementation tests are summarized in Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a>.</p><p id="p-38">Complementation tests were used to recheck genes whose mutations were found in genes with different “let” names. <em><a href="http://www.wormbase.org/db/get?name=WBGene00002655;class=Gene">let-439</a></em> has two sequenced alleles and both of them have mutations in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019762;class=Gene">M03F8.3</a></em>, showing that it is <em><a href="http://www.wormbase.org/db/get?name=WBGene00002655;class=Gene">let-439</a></em>. However, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002659;class=Gene">let-443</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242030;class=Variation">s1417</a></em>) had previously been mapped to a different zone than <em><a href="http://www.wormbase.org/db/get?name=WBGene00002655;class=Gene">let-439</a></em>, but contained a nonsense mutation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00019762;class=Gene">M03F8.3</a></em>. We complementation-tested <em><a href="http://www.wormbase.org/db/get?name=WBGene00002659;class=Gene">let-443</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242030;class=Variation">s1417</a></em>) against both alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002655;class=Gene">let-439</a></em> and they failed to complement, which supported the sequencing result; therefore, we renamed <em><a href="http://www.wormbase.org/db/get?name=WBGene00002659;class=Gene">let-443</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242030;class=Variation">s1417</a></em>) <em><a href="http://www.wormbase.org/db/get?name=WBGene00002655;class=Gene">let-439</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242030;class=Variation">s1417</a></em>). Complementation tests also showed that both <em>let-757</em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar02148776;class=Gene">s2460</a></em>) <em>and let-757</em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar02148777;class=Gene">s2867</a></em>) fail to complement <em>let-821</em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242401;class=Variation">s2804</a></em>) and sequence analysis showed them to be <em><a href="http://www.wormbase.org/db/get?name=WBGene00022816;class=Gene">fbn-1</a></em> (<em><a href="http://www.wormbase.org/db/get?name=WBGene00022816;class=Gene">ZK783.1</a></em>). These two results reduce the number of genes from 62 to 60, so 60 identified genes have been mapped to their molecular counterpart (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). In addition, a total of 104 lethal mutations (from 135 sequenced strains) were molecularly discovered in 60 essential genes with 41 missense mutations and 39 nonsense mutations (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a>). We did not identify any lethal mutations in the other 31 sequenced strains.</p><p id="p-39">All these cases show the value of combined genetic and sequencing analyses because they not only help with narrowing down the search region for lethal mutations, but also aid in correcting and improving the quality of genetic mapping.</p><p id="p-40">We checked the 60 genes on WormBase for lethal alleles (including gk, tm, and ok alleles) that were presented in previous publications. We summarized the genes into three categories and details of each gene are listed in the last column of Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a>. The first category is genes published with lethal alleles (<em>n</em> = 12; two with only tm alleles, these are highlighted in green). Five of those genes, <em><a href="http://www.wormbase.org/db/get?name=WBGene00006840;class=Gene">unc-116</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00003002;class=Gene">lin-13</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00001177;class=Gene">egl-8</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00004947;class=Gene">sos-1</a></em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003242;class=Gene">mig-6</a></em>, were published prior to <a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Johnsen and Baillie (1991)</a> and <a id="xref-ref-69-8" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> (1998)</a>. See <a id="xref-ref-69-9" class="xref-bibr" href="#ref-69">Stewart <em>et al</em>. (1998)</a> for <em><a href="http://www.wormbase.org/db/get?name=WBGene00002904;class=Gene">let-786</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002877;class=Gene">let-752</a></em> and <a id="xref-ref-28-9" class="xref-bibr" href="#ref-28">Johnsen and Baillie (1991)</a> for <em><a href="http://www.wormbase.org/db/get?name=WBGene00002662;class=Gene">let-447</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002566;class=Gene">let-344</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002642;class=Gene">let-423</a></em>. Therefore, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002904;class=Gene">let-786</a></em> is <em><a href="http://www.wormbase.org/db/get?name=WBGene00006840;class=Gene">unc-116</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002877;class=Gene">let-752</a></em> is <em><a href="http://www.wormbase.org/db/get?name=WBGene00003002;class=Gene">lin-13</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002662;class=Gene">let-447</a></em> is <em><a href="http://www.wormbase.org/db/get?name=WBGene00001177;class=Gene">egl-8</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002566;class=Gene">let-344</a> is <a href="http://www.wormbase.org/db/get?name=WBGene00004947;class=Gene">sos-1</a></em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002642;class=Gene">let-423</a></em> is <em><a href="http://www.wormbase.org/db/get?name=WBGene00003242;class=Gene">mig-6</a></em>. The other seven were first published in <a id="xref-ref-69-10" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> (1998)</a> and <a id="xref-ref-28-10" class="xref-bibr" href="#ref-28">Johnsen and Baillie (1991)</a> and therefore the “let” names in those publications take precedent. The second category is genes that have known lethal alleles that have not been curated and characterized (<em>n</em> = 18; all of the alleles listed are gk, tm, or ok; these are black in the table. Their correct names were first published in <a id="xref-ref-69-11" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> (1998)</a> or <a id="xref-ref-28-11" class="xref-bibr" href="#ref-28">Johnsen and Baillie (1991)</a>. The third category is genes that have no alleles in WormBase (<em>n</em> = 30; highlighted in red in the table). Their gene names are published in <a id="xref-ref-69-12" class="xref-bibr" href="#ref-69">Stewart <em>et al.</em> (1998)</a> or <a id="xref-ref-28-12" class="xref-bibr" href="#ref-28">Johnsen and Baillie (1991)</a>.</p></div><div id="sec-19" class="subsection"><h3>Functions of the identified 60 essential genes</h3><p id="p-41">We looked at the functions of the 60 essential genes that we identified. Most of these genes, based on orthologous relationships, have biological functions that have been predicted in other organisms. Among the 60 essential genes, 22 encode enzymes, including helicases, kinases, and tRNA synthetases. There are also structure proteins including ribosomal proteins and transcription factors, as well as mRNA splicing factors. Note that seven of our essential genes have no known functions. Five of these seven genes have orthologs in nematodes but no other organisms.</p><p id="p-42">We took advantage of WormBase to investigate the evolutionary conservation of each gene by looking for orthologs in Nematodes (N), Invertebrate (<em>D. melanogaster</em>) (I), Mammals (mouse as well as human) (M), and single-cell Fungi (<em>Saccharomycetaceae</em>) (F), as shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488816/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488816/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Biological functions of the 60 essential genes</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">Because of the functional importance of essential genes, one would expect that they are conserved and found in most organisms. Consistent with this, we found that 44 out of 60 (73.3%) of our essential genes have orthologs in all the examined organisms but eight of the 60 (13.3%) are nematode-specific. This is consistent with the results found in a previous study based on a transcriptome analysis of the phylum Nematoda (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Parkinson <em>et al.</em> 2004</a>). Eight genes were found in nematodes, invertebrates, and mammals, but not in single-cell fungi, suggesting that there are essential genes that are specific to multicellular organisms. For instance, <em>let-757/<a href="http://www.wormbase.org/db/get?name=WBGene00022816;class=Gene">fbn-1</a></em> encodes the extracellular matrix protein fibrillin, which is highly conserved in multicellular organisms and is a key component of elastic fiber in connective tissues (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ritty <em>et al.</em> 2003</a>).</p></div><div id="sec-20" class="subsection"><h3>Four new protein domains revealed by examining missense lethal mutations</h3><p id="p-45">Protein functions can be affected by missense mutations that reside in conserved regions, such as functional domains. Thus, we want to explore whether missense lethal mutations are more likely to be located within annotated Pfam domains.</p><p id="p-46">To gain a better understanding of lethal mutations in <em>C. elegans</em>, we wanted a large dataset and so incorporated 62 essential genes identified in previous studies that examined lethal mutations balanced by <em><a href="http://www.wormbase.org/db/get?name=sDp2;class=Rearrangement">sDp2</a></em> on chromosome I (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Chu <em>et al.</em> 2014</a>) and <em><a href="http://www.wormbase.org/db/get?name=eT1;class=Rearrangement">eT1</a></em> on chromosome V (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2007</a>), and the essential genes identified in this study. We found 70 missense mutations in 59 essential genes of which 64 are within Pfam domains of 53 genes, suggesting the functions of the essential proteins are disrupted. We looked at the remaining six missense mutations in six essential genes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/981/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Six missense mutations in six essential genes that are not in annotated functional domains. (B) One gene model (let-522/M05B5.2) that has a missense mutation h240 (C68Y) that does not reside in an annotated domain (Chu et al. 2014). The gene model was created using the Perl module FeatureStack. Exons are drawn to scale relative to each other in gray boxes; black lines with triangle shapes represent introns, which are not drawn to scale; white boxes represent UTRs. Blue bars indicate the missense mutation. Multiple alignments were performed between the essential genes and their orthologs in 25 other nematodes and four model organisms (Homo sapiens, Drosophila melanogaster, Mus musculus, and Danio rerio). However, no orthologs were found in the four model organisms. Left columns show the names of nematode species. The alignment figure shows a fragment of 100 amino acids. The mutated amino acid (C68Y) as indicated by the black arrow is in the center with 50 amino acids upstream and 49 amino acids downstream. The alignment color scheme is based on &#x201C;ClustalX,&#x201D; in which the color of a symbol depends on the residue type and the occurrence frequency in one column. Black boxes highlight the input genes with both wild type and mutant copy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1298489251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Six missense mutations in six essential genes that are not in annotated functional domains. (B) One gene model (&lt;em&gt;let-522/M05B5.2&lt;/em&gt;) that has a missense mutation &lt;em&gt;h240&lt;/em&gt; (C68Y) that does not reside in an annotated domain (Chu &lt;em&gt;et al.&lt;/em&gt; 2014). The gene model was created using the Perl module FeatureStack. Exons are drawn to scale relative to each other in gray boxes; black lines with triangle shapes represent introns, which are not drawn to scale; white boxes represent UTRs. Blue bars indicate the missense mutation. Multiple alignments were performed between the essential genes and their orthologs in 25 other nematodes and four model organisms (&lt;em&gt;Homo sapiens&lt;/em&gt;, &lt;em&gt;Drosophila melanogaster&lt;/em&gt;, &lt;em&gt;Mus musculus&lt;/em&gt;, and &lt;em&gt;Danio rerio&lt;/em&gt;). However, no orthologs were found in the four model organisms. Left columns show the names of nematode species. The alignment figure shows a fragment of 100 amino acids. The mutated amino acid (C68Y) as indicated by the black arrow is in the center with 50 amino acids upstream and 49 amino acids downstream. The alignment color scheme is based on &#x201C;ClustalX,&#x201D; in which the color of a symbol depends on the residue type and the occurrence frequency in one column. Black boxes highlight the input genes with both wild type and mutant copy.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/981/F3.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/3/981/F3.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/981/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/981/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488828" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-47" class="first-child">(A) Six missense mutations in six essential genes that are not in annotated functional domains. (B) One gene model (<em>let-522/M05B5.2</em>) that has a missense mutation <em>h240</em> (C68Y) that does not reside in an annotated domain (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Chu <em>et al.</em> 2014</a>). The gene model was created using the Perl module FeatureStack. Exons are drawn to scale relative to each other in gray boxes; black lines with triangle shapes represent introns, which are not drawn to scale; white boxes represent UTRs. Blue bars indicate the missense mutation. Multiple alignments were performed between the essential genes and their orthologs in 25 other nematodes and four model organisms (<em>Homo sapiens</em>, <em>Drosophila melanogaster</em>, <em>Mus musculus</em>, and <em>Danio rerio</em>). However, no orthologs were found in the four model organisms. Left columns show the names of nematode species. The alignment figure shows a fragment of 100 amino acids. The mutated amino acid (C68Y) as indicated by the black arrow is in the center with 50 amino acids upstream and 49 amino acids downstream. The alignment color scheme is based on “ClustalX,” in which the color of a symbol depends on the residue type and the occurrence frequency in one column. Black boxes highlight the input genes with both wild type and mutant copy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">Two missense mutations were nucleotide transitions (G-&gt;A) that changed the ATG start codon to ATA. It is possible that the next in-frame ATG would be recognized as an initiation codon, but the truncated product could have a hypomorphic defect (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Maser <em>et al.</em> 2001</a>). In both cases in our study, the next ATG was not in-frame, which probably resulted in a lack of wild-type products causing loss of function (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schnabel <em>et al.</em> 1992</a>). The observed lethal phenotypes also show that start codon mutations can be deleterious.</p><p id="p-49">For the other four missense mutations, even though they are not located within annotated domains (by searching InterPro database), it turns out that the altered amino acids are still highly conserved in orthologs from different species. In a window size of 100 amino acids, which is the average size of protein domains based on the statistical analysis on the size of Pfam domains from all protein isoforms (data not shown), the sequence regions containing the target amino acids are also highly conserved (one example is shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). It suggests new protein domains that had not been identified as functional domains. It is also possible that the effected amino acids are important for maintaining protein structures and the mutations disrupt these structures, resulting in malfunctioning proteins.</p></div><div id="sec-21" class="subsection"><h3>GO analysis of essential genes</h3><p id="p-50">To conduct GO analysis, four groups of genes were used for comparison. Group one (G1): essential genes that were isolated through genetic screens in the Baillie laboratory or collaborations with the Baillie laboratory and have been fully sequenced or rescued by fosmids (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Maciejowski <em>et al.</em> 2005</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2007</a>; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Chu <em>et al.</em> 2014</a>; S. Ono, personal communication) (143 in total, including 60 genes from the current study). Group two (G2): essential genes that have published lethal alleles (1208 in total). Group three (G3): genetically identified nonessential genes (796 in total). Group four (G4): genes with no observed phenotype by either RNAi or alleles (12,811 in total). This putatively nonessential group contains the majority of protein-coding genes in <em>C. elegans</em>.</p><p id="p-51">We compared the G1 essential genes to genes in the other three groups based on three GO categories: cellular component, molecular function, and biological process (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/981/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genes from each group annotated with three ontology terms: (A) molecular function, (B) cellular component, and (C) biological process. The x-axis lists several GO terms in each GO category and the y-axis is the proportion of genes for each GO term over the total number of annotated genes in each group. Four groups are shown in separate colors: G1 in blue, G2 in orange, G3 in gray, and G4 in gold. Statistical difference was calculated for G1 vs. G2, G3, and G4 individually by using Fisher&#x2019;s exact test (* P-value &lt; 0.05, ** P-value &lt; 0.01, *** P-value &lt; 0.001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1298489251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genes from each group annotated with three ontology terms: (A) molecular function, (B) cellular component, and (C) biological process. The &lt;em&gt;x&lt;/em&gt;-axis lists several GO terms in each GO category and the &lt;em&gt;y&lt;/em&gt;-axis is the proportion of genes for each GO term over the total number of annotated genes in each group. Four groups are shown in separate colors: G1 in blue, G2 in orange, G3 in gray, and G4 in gold. Statistical difference was calculated for G1 &lt;em&gt;vs.&lt;/em&gt; G2, G3, and G4 individually by using Fisher&#x2019;s exact test (* &lt;em&gt;P&lt;/em&gt;-value &lt; 0.05, ** &lt;em&gt;P&lt;/em&gt;-value &lt; 0.01, *** &lt;em&gt;P&lt;/em&gt;-value &lt; 0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/981/F4.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/3/981/F4.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/981/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/981/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488820" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-52" class="first-child">Genes from each group annotated with three ontology terms: (A) molecular function, (B) cellular component, and (C) biological process. The <em>x</em>-axis lists several GO terms in each GO category and the <em>y</em>-axis is the proportion of genes for each GO term over the total number of annotated genes in each group. Four groups are shown in separate colors: G1 in blue, G2 in orange, G3 in gray, and G4 in gold. Statistical difference was calculated for G1 <em>vs.</em> G2, G3, and G4 individually by using Fisher’s exact test (* <em>P</em>-value &lt; 0.05, ** <em>P</em>-value &lt; 0.01, *** <em>P</em>-value &lt; 0.001).</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">Essential proteins from G1 and G2 have no significant difference in any molecular function related annotation, including nucleic acid binding and especially RNA binding (GO:0003723). These annotations are significantly lower for the proteins of nonessential genes for both G3 and G4 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). This is consistent with our observation in the cellular component analysis, in which annotations of the ribonucleoprotein complex (GO:0030529) are high in essential proteins (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). In contrast, receptor activity (GO:0004872) is lower in essential proteins (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>). This might also explain the observation that the proportion of nonessential proteins located in plasma membranes (GO:0005886) is significantly higher than that of essential proteins (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>).</p><p id="p-54">With regard to biological processes, essential proteins in G1 and G2 are significantly enriched for primary metabolic processes, as well as translation and mRNA processing, suggesting that essential genes tend to be involved in protein synthesis. In contrast, nonessential proteins are significantly enriched for regulation of cellular functions, such as cell communication and sensory perception (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4C</a>). If there is a disruption in these processes, the worms might show visible phenotypes; however, these are generally not lethal.</p><p id="p-55">GO term analysis indicated that essential genes tend to execute enzyme and nucleic acid binding activities during fundamental processes, such as DNA replication, translation, and transcription intracellularly.</p></div><div id="sec-22" class="subsection"><h3>Gene essentiality analysis</h3><p id="p-56">Gene essentiality, gene duplicability, protein interaction networks, and gene expression are biological factors that can influence the evolutionary rate of proteins (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Liao <em>et al.</em> 2006</a>). Thus, to develop an understanding of how essential proteins function in cells, we assessed the properties of essential genes from three perspectives: gene duplicability, protein interactions, and gene expression, applying the same dataset used in GO term analysis.</p><div id="sec-23" class="subsection"><h4>Gene essentiality <em>vs.</em> gene duplicability:</h4><p id="p-57">In <em>C. elegans</em>, researchers showed that closely related gene duplicates are responsible for mitigating the effects of mutations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Conant and Wagner 2004</a>), which is also known as mutational robustness (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mendonca <em>et al.</em> 2011</a>). Studies performed in yeast showed that duplicate genes have a higher probability of functional compensation than singletons (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gu <em>et al.</em> 2003</a>). However, there are analyses in both yeast and mouse suggesting that duplicate genes are as essential as singletons (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wagner 2000</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Liang and Li 2007</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Liao and Zhang 2007</a>). Thus, we want to revisit this question using our collected datasets. Using the most stringent criteria of <em>e</em>-value with 1E−20 for quantitative comparison, we see that most collected genes are singletons, no matter which group they belong to (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). This is consistent with the theory that redundant duplicates are not evolutionarily stable and tend to be removed over long periods of time (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nowak <em>et al.</em> 1997</a>), although this theory is still debated (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Yona <em>et al.</em> 2015</a>). Despite that, the proportions of singletons to duplicates in each group show differences. The proportions of both singletons and duplicates in G1 are similar to those in G2, which is as we expected, considering both groups contain only essential genes. On the other hand, the proportions of singletons are significantly less in non-essential genes (both G3 and G4). Based on this analysis, essential genes are more likely to be unique, with over 90% of essential genes present as single copies (in both G1 and G2) compared with nonessential genes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/3/981/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene essentiality analysis. Group one (G1): essential genes that were isolated through genetic screens in the Baillie laboratory or collaborations and are fully sequenced or rescued by fosmids (Maciejowski et al. 2005; Jones et al. 2007; Chu et al. 2014; S. Ono, personal communication) (143 in total, including 60 genes from the current study). Group two (G2): essential genes that have published alleles supporting lethal phenotypes (1208 in total). Group three (G3): nonessential genes that have known alleles supporting nonlethal phenotypes (796 in total). Group four (G4): no-phenotype supporting genes that have no observable phenotypes caused by either RNAi or known alleles (12,811 in total). Given that this group contains the majority of protein-coding genes in C. elegans, most genes in this group are expected to be nonessential. The four groups are labeled in blue, orange, gray, and gold, respectively. (A) The proportion of singletons and duplicates in the four groups. The x-axis shows singletons and duplicates from the groups. Significant differences (*** P-value &lt; 0.001) were observed between G1 and G3, and between G1 and G4. (B) The distribution of number of interactions per protein (Stark et al. 2006). The x-axis shows the number of interactions per protein and the y-axis is the proportion of proteins. (C) The developmental stage-specific expression pattern and expression level of genes in four groups. The x-axis shows 23 developmental stages, embryos to young adults, from four cells. m in the x-axis is short for minutes of embryonic development. Each bar represents the distribution of the normalized expression level (RPKM in log2 scale) of each gene in each of four groups for each developmental stage. The linear regression trend line is drawn for each group in their respective color. The stages of embryonic elongation are underlined in red. The boxplot was computed using R and the function ggplot2, which uses a 95% C.I." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1298489251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene essentiality analysis. Group one (G1): essential genes that were isolated through genetic screens in the Baillie laboratory or collaborations and are fully sequenced or rescued by fosmids (Maciejowski &lt;em&gt;et al.&lt;/em&gt; 2005; Jones &lt;em&gt;et al.&lt;/em&gt; 2007; Chu &lt;em&gt;et al.&lt;/em&gt; 2014; S. Ono, personal communication) (143 in total, including 60 genes from the current study). Group two (G2): essential genes that have published alleles supporting lethal phenotypes (1208 in total). Group three (G3): nonessential genes that have known alleles supporting nonlethal phenotypes (796 in total). Group four (G4): no-phenotype supporting genes that have no observable phenotypes caused by either RNAi or known alleles (12,811 in total). Given that this group contains the majority of protein-coding genes in &lt;em&gt;C. elegans&lt;/em&gt;, most genes in this group are expected to be nonessential. The four groups are labeled in blue, orange, gray, and gold, respectively. (A) The proportion of singletons and duplicates in the four groups. The &lt;em&gt;x&lt;/em&gt;-axis shows singletons and duplicates from the groups. Significant differences (&lt;em&gt;*** P&lt;/em&gt;-value &lt; 0.001) were observed between G1 and G3, and between G1 and G4. (B) The distribution of number of interactions per protein (Stark &lt;em&gt;et al.&lt;/em&gt; 2006). The &lt;em&gt;x&lt;/em&gt;-axis shows the number of interactions per protein and the &lt;em&gt;y&lt;/em&gt;-axis is the proportion of proteins. (C) The developmental stage-specific expression pattern and expression level of genes in four groups. The &lt;em&gt;x&lt;/em&gt;-axis shows 23 developmental stages, embryos to young adults, from four cells. m in the &lt;em&gt;x&lt;/em&gt;-axis is short for minutes of embryonic development. Each bar represents the distribution of the normalized expression level (RPKM in log2 scale) of each gene in each of four groups for each developmental stage. The linear regression trend line is drawn for each group in their respective color. The stages of embryonic elongation are underlined in red. The boxplot was computed using R and the function ggplot2, which uses a 95% C.I.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/3/981/F5.medium.gif" width="233" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/3/981/F5.medium.gif" width="233" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/3/981/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/3/981/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488892" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-58" class="first-child">Gene essentiality analysis. Group one (G1): essential genes that were isolated through genetic screens in the Baillie laboratory or collaborations and are fully sequenced or rescued by fosmids (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Maciejowski <em>et al.</em> 2005</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2007</a>; <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Chu <em>et al.</em> 2014</a>; S. Ono, personal communication) (143 in total, including 60 genes from the current study). Group two (G2): essential genes that have published alleles supporting lethal phenotypes (1208 in total). Group three (G3): nonessential genes that have known alleles supporting nonlethal phenotypes (796 in total). Group four (G4): no-phenotype supporting genes that have no observable phenotypes caused by either RNAi or known alleles (12,811 in total). Given that this group contains the majority of protein-coding genes in <em>C. elegans</em>, most genes in this group are expected to be nonessential. The four groups are labeled in blue, orange, gray, and gold, respectively. (A) The proportion of singletons and duplicates in the four groups. The <em>x</em>-axis shows singletons and duplicates from the groups. Significant differences (<em>*** P</em>-value &lt; 0.001) were observed between G1 and G3, and between G1 and G4. (B) The distribution of number of interactions per protein (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Stark <em>et al.</em> 2006</a>). The <em>x</em>-axis shows the number of interactions per protein and the <em>y</em>-axis is the proportion of proteins. (C) The developmental stage-specific expression pattern and expression level of genes in four groups. The <em>x</em>-axis shows 23 developmental stages, embryos to young adults, from four cells. m in the <em>x</em>-axis is short for minutes of embryonic development. Each bar represents the distribution of the normalized expression level (RPKM in log2 scale) of each gene in each of four groups for each developmental stage. The linear regression trend line is drawn for each group in their respective color. The stages of embryonic elongation are underlined in red. The boxplot was computed using R and the function ggplot2, which uses a 95% C.I.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h4>Gene essentiality <em>vs.</em> gene connectivity:</h4><p id="p-59">Proteins do not function alone; they function as parts of pathways, macromolecular structures, and regulatory networks (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dolinski and Botstein 2007</a>). By characterizing the yeast proteome, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jeong <em>et al.</em> (2001)</a> found that the most highly connected proteins in a cell are the most important ones for survival (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jeong <em>et al.</em> 2001</a>). Based on this theory, we hypothesize that essential proteins tend to engage in protein interaction hubs, with more interactors connecting to them than to nonessential proteins.</p><p id="p-60">Using the whole genome protein interactions in <em>C. elegans</em> from BioGRID (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Stark <em>et al.</em> 2006</a>), <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a> shows the distribution of the number of interactions per protein, essential proteins from G1 and G2 tend to have more interaction partners, <em>i.e.</em>, &gt;10 proteins compared with G3 and G4 (<em>P</em>-value &lt; 0.001). This suggests that essential proteins tend to be in the interaction hubs. It is interesting to note that proteins that have no phenotypic consequence are much more likely to have only one interactor.</p></div><div id="sec-25" class="subsection"><h4>Gene essentiality <em>vs.</em> gene expression:</h4><p id="p-61">Expression levels are frequently used to infer the activities and functions of gene products. Expression levels have also been used as a parameter to analyze the nonessential gene datasets in the human genome (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hart <em>et al.</em> 2014</a>). Therefore, we want to examine the expression level of all <em>C. elegans</em> genes in our collected data by using the gene expression dataset from the <em>GExplore</em> (version 1.4) expression search interface (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hutter <em>et al.</em> 2009</a>) and the NHGRI modENCODE project (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hillier <em>et al.</em> 2009</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gerstein <em>et al.</em> 2010</a>).</p><p id="p-62"><a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a> shows that essential genes from G1 and G2 have significantly higher expression level in all developmental stages than nonessential genes from G3 and G4. With respect to the expression pattern over the developmental stages, essential genes are more consistently expressed across all developmental stages than nonessential genes. However, the expression levels of nonessential genes show a twofold increase in the embryonic elongation stage (470–640 min). During this period, morphogenesis is happening, which in more complex multicellular organisms could be considered as the assembly of cells into functioning tissues and organs. Morphogenesis likely proceeds via conserved signaling pathways, regulatory mechanisms, and effector genes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edgar 1999</a>) (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>).</p></div></div></div><div class="section discussion" id="sec-26"><h2 class="">Discussion</h2><p id="p-63">A total of 62 out of 89 essential genes were successfully identified and 60 out of 62 were matched to their molecular counterpart. This large number demonstrates the value of our method. However, we were not able to identify any appropriate mutations in the sequences of the other 27 genes. Complementation tests and PCR sequencing were conducted to validate possible candidates for seven of those 27; however, we found that they were not essential genes. We do not know the reason(s) for this, but we hypothesize several possible reasons. First is strain mix-ups; however, this is unlikely because during the construction of the strains containing <em><a href="http://www.wormbase.org/db/get?name=WBGene00002692;class=Gene">let-500</a></em> the terminal phenotypes of all the lethals were the same as noted when the lethals were first analyzed. Second is sequencing errors, as it is possible that there were not enough sequencing reads to support some of the lethal mutations. For example, only one out of 32 sequencing reads (3%) supports C to T change in <em><a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em>) in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002636;class=Gene">let-417</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00241760;class=Variation">s204</a></em>) strain, but in the remaining 78 sequenced mutants with <em><a href="http://www.wormbase.org/db/get?name=WBGene00006782;class=Gene">unc-46</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em>), the variation frequencies of <em><a href="http://www.wormbase.org/db/get?name=WBVar00143011;class=Variation">e177</a></em> range from 23 to 78%. This reason is not adequate because most of the time there was sufficient read support to get good results. Third, mutations may have been mismapped. An example is <em><a href="http://www.wormbase.org/db/get?name=WBGene00002659;class=Gene">let-443</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002655;class=Gene">let-439</a></em>, which were mapped genetically into two different zones. Sequence analysis showed they were actually the same gene. This reason is weak because this result shows that we can find mutations in different zones but careful scanning did not find mutations in any zone for the unsolved genes. Fourth, mutations may reside in noncoding regions. An example is <em><a href="http://www.wormbase.org/db/get?name=WBGene00002681;class=Gene">let-470</a></em> where no mutation was found in its genetic zone for <em><a href="http://www.wormbase.org/db/get?name=WBVar00242172;class=Variation">s1629</a></em> but a splicing acceptor mutation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00008670;class=Gene">F11A3.2</a></em> was found for <em><a href="http://www.wormbase.org/db/get?name=WBVar00242144;class=Variation">s1581</a></em> by a zone boundary, and fosmid rescue assays showed that <em><a href="http://www.wormbase.org/db/get?name=WBGene00008670;class=Gene">F11A3.2</a></em> rescues <em><a href="http://www.wormbase.org/db/get?name=WBGene00002681;class=Gene">let-470</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar00242172;class=Variation">s1629</a></em>). This reason does not explain why we found a mutation in noncoding DNA for one allele but not for the other. As stated above, we do not have a good reason to explain why 27 genes were not identified, but we are pleased that 62 were identified, with 60 of the 62 mapped to their molecular counterparts.</p><p id="p-64">Of the 60 identified essential genes, five are without RNAi evidence supporting their lethal phenotype (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1/FileS1">File S1</a>). This finding is meaningful because it indicates that although RNAi is an efficient method for essential gene identification, it does miss roughly 8.3% (five out of 60) of the genes that can be isolated in mutagenesis screens.</p><p id="p-65">Among the 143 essential genes collected in this study, we found 15 genes related to cell division and chromosome partitioning; surprisingly, 11 of those are on chromosome I. However, there was no other functional clustering of genes observed on chromosome III or V. Nevertheless, chemoreceptor gene families are unusually enriched on chromosome V, especially on the chromosomal arms, which might be responsible for signal transduction in <em>C. elegans</em> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Robertson and Thomas 2006</a>). Members of this large gene family are not essential genes but inhabit a lot of chromosome V(left), which is consistent with the fact that chromosome V(left) has few essential genes relative to the other autosomes and has a high degree of gene duplications (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Kamath <em>et al.</em> 2003</a>). The concentration of functionally related genes on chromosomes in a nonrandom pattern is important for understanding how genomes function and evolved. Besides, it may have clinical significance in understanding or predicting disease-causing. clustered genes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andrews <em>et al.</em> 2015</a>).</p><p id="p-66">We noticed that nonsense mutations are significantly enriched relative to missense mutations over duplication and translocation balancers (data not shown). Considering the heterozygosity background for each mutation, we propose that it might be due to the different functional effects of missense mutations and nonsense mutations in heterozygotes. Missense alleles might confer negative interaction with the wild-type allele in heterozygotes by being located in protein functional domains or the subunit interfaces, causing selection against these heterozygotes during genetic screens (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Phillips <em>et al.</em> 1995</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Thomas <em>et al.</em> 1997</a>). On the other hand, the nonsense-mediated mRNA decay (NMD) surveillance system degrades nonsense allele products, which suggests that almost no interfering nonsense mutant products are produced, therefore there is no such negative interaction using either duplication or translocation balancers. Consistent with this theory, we found that nonsense mutations are located toward N-terminal protein sequences. To test our hypothesis, one could utilize NMD defective mutants that harbor a genetically balanced strain to isolate lethal mutations. One would expect that the selective enrichment of nonsense alleles would be reduced in a NMD mutant background, because the amount of truncated protein would be greater in the defective NMD background.</p><p id="p-67">This allele-dependent property of essential genes is related to functional effects causing human diseases. Human orthologs of mouse essential genes are linked to numerous human diseases that affect a variety of physiological systems. It has been shown that mutations in human orthologs of embryonic lethal phenotypes, caused by loss-of-function essential mouse genes, can cause lethality in human pregnancies and may result in spontaneous abortions (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Goh <em>et al.</em> 2007</a>). However, phenotypic symptoms induced by defective essential genes are not restricted to developmental abnormalities, but can also contribute to adult diseases. Unlike null alleles that cause loss of function, some point mutations in these genes do not necessarily lead to a complete loss of protein function (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dickerson <em>et al.</em> 2011</a>). There are orthologs in human of mouse essential genes with missense mutations that may lead to abnormal phenotypes in the heterozygous state (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">International PPH Consortium <em>et al.</em> 2000</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Ragge <em>et al.</em> 2005</a>). Therefore, not only miscarriage and birth defects, but also other human diseases can be induced by defective essential genes, including different types of cancer (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Dickerson <em>et al.</em> 2011</a>).</p><p id="p-68">How many human orthologs of the essential genes in <em>C. elegans</em> that we identified are related to human disorders? Of the 143 essential genes that were gathered for the functional analysis, 108 were identified as having a putative ortholog in human. Among those 108 genes, 97 (90%) are associated with 1218 different diseases. For instance, <em>SF3B1</em>, which encodes an RNA-splicing factor and is orthologous to <em><a href="http://www.wormbase.org/db/get?name=WBGene00002887;class=Gene">let-763</a>/<a href="http://www.wormbase.org/db/get?name=WBGene00011605;class=Gene">sftb-1</a></em>, has mutations related to myeloid cancers and refractory anemias (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Papaemmanuil <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broseus <em>et al.</em> 2013</a>). In another case, the ortholog of <a href="http://www.wormbase.org/db/get?name=WBGene00008670;class=Gene">F11A3.2</a> in human is a translation initiation factor EIF2B4, which is associated with the inheritable disease Vanishing White Matter (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kanbayashi <em>et al.</em> 2015</a>). Moreover, 60 of the 97 genes have existing variations that were found to be responsible for 163 different diseases. Consistent with previous findings (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Dickerson <em>et al.</em> 2011</a>), mutations in these genes can lead to a broad spectrum of human disorders. The most abundant diseases tend to be related to cancer or carcinogenesis, including breast carcinoma and colorectal cancer, suggesting links between adult cellular abnormalities and developmental functions.</p><p id="p-69">A large number of essential genes were molecularly identified in this study, which significantly enlarges the database of essential genes in <em>C. elegans</em>. The identified lethal alleles provide a rich resource for future studies on essential genes, considering the difficulty of isolating and maintaining lethal mutations. Moreover, conserved essential genes could be good gene models for developing our understanding of the functions of homologous genes in humans, especially disease-related genes. On the other hand, according to a recent study, the essential genes that are only conserved in nematodes can play critical roles in core eukaryotic processes, especially in chromosome segregation (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Verster <em>et al.</em> 2017</a>). Meanwhile, species-specific essential genes could be good candidates for targeting pathogenic nematodes. Considering the functional importance of essential genes, studies can be conducted on the ones that currently have no annotated functions.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">We thank the National Bioresource Project for the Experimental Animal “Nematode <em>C. elegans</em>,” the <em>Caenorhabditis</em> Genetics Center for providing strains, and the Michael Smith Genome Sciences Centre for Illumina sequencing. Special thanks go to Ann Rose for helping with the sequencing. We also thank Jiarui Li for advice and comments. This research was sponsored by grants from the Natural Sciences and Engineering Research Council of Canada Discovery Grants Program and the Canada Research Chairs Program to D.B. The authors declare that they have no competing interests.</p><p id="p-71">Author contributions: J.S.-C.C. and Z.Q. prepared genomic DNA for WGS. Z.Q. analyzed the WGS data of chromosome V mutants and S.Y. analyzed the WGS data of chromosome III mutants. S.Y. and Z.Q. conducted essential gene identification analyses on both chromosome III and chromosome V mutants. R.J. performed genetic crosses. Z.Q. performed all the functional studies. D.L.B. and N.C. led the study. Z.Q., S.Y., J.S.-C.C., and N.C. wrote the article. All authors read and approved the final article.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-72">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300338/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-73"><em>Communicating editor: S. Lee</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-10-06"><span class="received-label">Received </span>October 6, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-01-12"><span class="accepted-label">Accepted </span>January 12, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Qin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.1" data-doi="10.1073/pnas.0403929101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amsterdam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nissen</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swindell</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrington</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identification of 315 genes essential for early zebrafish development.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">12792</span>–<span class="cit-lpage">12797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403929101%26rft_id%253Dinfo%253Apmid%252F15256591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM1LzEyNzkyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.2" data-doi="10.1101/gr.184325.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honti</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfundt</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leeuw</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hehir-Kwa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The clustering of functionally related genes contributes to CNV-mediated disease.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">802</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.184325.114%26rft_id%253Dinfo%253Apmid%252F25887030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzYvODAyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.3" data-doi="10.1038/leu.2013.120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broséus</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alpermann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wulfert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florensa Brichs</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeromin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Age, JAK2(V617F) and SF3B1 mutations are the main predicting factors for survival in refractory anaemia with ring sideroblasts and marked thrombocytosis.</span> <abbr class="cit-jnl-abbrev">Leukemia</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1826</span>–<span class="cit-lpage">1831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLeukemia%26rft.volume%253D27%26rft.spage%253D1826%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fleu.2013.120%26rft_id%253Dinfo%253Apmid%252F23594705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/leu.2013.120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23594705&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324162700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.4" data-doi="10.1534/genetics.111.137208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennison</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Allelic ratios and the mutational landscape reveal biologically significant heterozygous SNVs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1225</span>–<span class="cit-lpage">1233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137208%26rft_id%253Dinfo%253Apmid%252F22267497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzEyMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.5" data-doi="10.1186/1471-2164-15-361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-throughput capturing and characterization of mutations in essential genes of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D361%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-361%26rft_id%253Dinfo%253Apmid%252F24884423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24884423&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKim</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Analysis of lethal mutations induced in a mutator strain that activates transposable elements in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DClark%26rft.auinit1%253DD.%2BV.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D114%26rft.atitle%253DAnalysis%2Bof%2Blethal%2Bmutations%2Binduced%2Bin%2Ba%2Bmutator%2Bstrain%2Bthat%2Bactivates%2Btransposable%2Belements%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F2158925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2158925&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.7" data-doi="10.1098/rspb.2003.2560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Duplicate genes and robustness to transient gene knock-downs in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DConant%26rft.auinit1%253DG.%2BC.%26rft.volume%253D271%26rft.issue%253D1534%26rft.spage%253D89%26rft.epage%253D96%26rft.atitle%253DDuplicate%2Bgenes%2Band%2Brobustness%2Bto%2Btransient%2Bgene%2Bknock-downs%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2003.2560%26rft_id%253Dinfo%253Apmid%252F15002776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMToiMjcxLzE1MzQvODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.8" data-doi="10.1186/1471-2164-6-118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammarlund</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrach</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hullett</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rapid single nucleotide polymorphism mapping in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDavis%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.spage%253D118%26rft.epage%253D118%26rft.atitle%253DRapid%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmapping%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-6-118%26rft_id%253Dinfo%253Apmid%252F16156901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-6-118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16156901&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.9" data-doi="10.1016/0022-2836(89)90229-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dibb</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruyama</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karn</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Sequence analysis of the complete <em>Caenorhabditis elegans</em> myosin heavy chain gene family.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDibb%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D205%26rft.issue%253D3%26rft.spage%253D603%26rft.epage%253D613%26rft.atitle%253DSequence%2Banalysis%2Bof%2Bthe%2Bcomplete%2BCaenorhabditis%2Belegans%2Bmyosin%2Bheavy%2Bchain%2Bgene%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252889%252990229-5%26rft_id%253Dinfo%253Apmid%252F2926820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(89)90229-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2926820&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T211600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.10" data-doi="10.1371/journal.pone.0027368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickerson</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hentges</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Defining the role of essential genes in human disease.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e27368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDickerson%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De27368%26rft.epage%253De27368%26rft.atitle%253DDefining%2Bthe%2Brole%2Bof%2Bessential%2Bgenes%2Bin%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0027368%26rft_id%253Dinfo%253Apmid%252F22096564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0027368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22096564&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.11" data-doi="10.1146/annurev.genet.40.110405.090439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Orthology and functional conservation in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">465</span>–<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DDolinski%26rft.auinit1%253DK.%26rft.volume%253D41%26rft.spage%253D465%26rft.epage%253D507%26rft.atitle%253DOrthology%2Band%2Bfunctional%2Bconservation%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.40.110405.090439%26rft_id%253Dinfo%253Apmid%252F17678444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.40.110405.090439&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17678444&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252359500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-8.3.981.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">L. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> Morphogenesis and organogenesis in Caenorhabditis elegans, pp. 269–297 in <em>Development</em>, edited by V. E. A. Russo, D. J. Cove, L. G. Edgar, R. Jaenisch, F. Salamini. Springer, Berlin, Heidelberg.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riddle</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetic Maps</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2BMaps%26rft.volume%253D5%26rft.spage%253D111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.14" data-doi="10.1093/nar/gkt1223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pfam: the protein families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D222</span>–<span class="cit-lpage">D230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1223%26rft_id%253Dinfo%253Apmid%252F24288371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24288371&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.15" data-doi="10.1534/genetics.110.116616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neil</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome profiling of mutagenesis in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116616%26rft_id%253Dinfo%253Apmid%252F20439774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNDMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.16" data-doi="10.1093/bioinformatics/bts572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frech</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">FeatureStack: Perl module for comparative visualization of gene features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3137</span>–<span class="cit-lpage">3138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts572%26rft_id%253Dinfo%253Apmid%252F23023981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts572&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23023981&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311902700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.17" data-doi="10.1016/j.copbio.2006.08.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerdes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Essential genes on metabolic maps.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">448</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DGerdes%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D448%26rft.epage%253D456%26rft.atitle%253DEssential%2Bgenes%2Bon%2Bmetabolic%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2006.08.006%26rft_id%253Dinfo%253Apmid%252F16978855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2006.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16978855&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241580500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.18" data-doi="10.1126/science.1196914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Nostrand</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arshinoff</span>  <span class="cit-name-given-names">B. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrative analysis of the <em>Caenorhabditis elegans</em> genome by the modENCODE project.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1775</span>–<span class="cit-lpage">1787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1196914%26rft_id%253Dinfo%253Apmid%252F21177976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMi8xNzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.19" data-doi="10.1073/pnas.0701361104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goh</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cusick</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The human disease network.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8685</span>–<span class="cit-lpage">8690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701361104%26rft_id%253Dinfo%253Apmid%252F17502601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzIxLzg2ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.20" data-doi="10.1038/nature01198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scharfe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Role of duplicate genes in genetic robustness against null mutations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGu%26rft.auinit1%253DZ.%26rft.volume%253D421%26rft.issue%253D6918%26rft.spage%253D63%26rft.epage%253D66%26rft.atitle%253DRole%2Bof%2Bduplicate%2Bgenes%2Bin%2Bgenetic%2Brobustness%2Bagainst%2Bnull%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01198%26rft_id%253Dinfo%253Apmid%252F12511954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12511954&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180165500037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.21" data-doi="10.15252/msb.20145216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sircoulomb</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rottapel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moffat</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Measuring error rates in genomic perturbation screens: gold standards for human functional genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DHart%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D733%26rft.epage%253D733%26rft.atitle%253DMeasuring%2Berror%2Brates%2Bin%2Bgenomic%2Bperturbation%2Bscreens%253A%2Bgold%2Bstandards%2Bfor%2Bhuman%2Bfunctional%2Bgenomics%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20145216%26rft_id%253Dinfo%253Apmid%252F24987113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjEwLzcvNzMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.22" data-doi="10.1101/gr.088112.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinke</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marra</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Massively parallel sequencing of the polyadenylated transcriptome of <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.088112.108%26rft_id%253Dinfo%253Apmid%252F19181841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzQvNjU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.23" data-doi="10.1186/1471-2164-10-529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">GExplore: a web server for integrated queries of protein domains, gene expression and mutant phenotypes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHutter%26rft.auinit1%253DH.%26rft.volume%253D10%26rft.spage%253D529%26rft.epage%253D529%26rft.atitle%253DGExplore%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bintegrated%2Bqueries%2Bof%2Bprotein%2Bdomains%252C%2Bgene%2Bexpression%2Band%2Bmutant%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-529%26rft_id%253Dinfo%253Apmid%252F19917126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19917126&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.24" data-doi="10.1038/79226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International PPH Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauciulo</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">J. A.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Heterozygous germline mutations in BMPR2, encoding a TGF-beta receptor, cause familial primary pulmonary hypertension.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLane%26rft.auinit1%253DK.%2BB.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D84%26rft.atitle%253DHeterozygous%2Bgermline%2Bmutations%2Bin%2BBMPR2%252C%2Bencoding%2Ba%2BTGF-beta%2Breceptor%252C%2Bcause%2Bfamilial%2Bprimary%2Bpulmonary%2Bhypertension.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F79226%26rft_id%253Dinfo%253Apmid%252F10973254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/79226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10973254&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089078000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.25" data-doi="10.1101/gr.7.10.974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janke</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franz</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Interpreting a sequenced genome: toward a cosmid transgenic library of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">974</span>–<span class="cit-lpage">985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DJanke%26rft.auinit1%253DD.%2BL.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253D974%26rft.epage%253D985%26rft.atitle%253DInterpreting%2Ba%2BSequenced%2BGenome%253A%2BToward%2Ba%2BCosmid%2BTransgenic%2BLibrary%2Bof%2BCaenorhabditis%25A0elegans%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7.10.974%26rft_id%253Dinfo%253Apmid%252F9331368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjcvMTAvOTc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.26" data-doi="10.1038/35075138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barabasi</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oltvai</span>  <span class="cit-name-given-names">Z. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Lethality and centrality in protein networks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJeong%26rft.auinit1%253DH.%26rft.volume%253D411%26rft.issue%253D6833%26rft.spage%253D41%26rft.epage%253D42%26rft.atitle%253DLethality%2Band%2Bcentrality%2Bin%2Bprotein%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35075138%26rft_id%253Dinfo%253Apmid%252F11333967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35075138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11333967&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168432800033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.27" data-doi="10.1016/0027-5107(88)90120-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Formaldehyde mutagenesis of the eT1 balanced region in <em>Caenorhabditis elegans</em>: dose-response curve and the analysis of mutational events.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DJohnsen%26rft.auinit1%253DR.%2BC.%26rft.volume%253D201%26rft.issue%253D1%26rft.spage%253D137%26rft.epage%253D147%26rft.atitle%253DFormaldehyde%2Bmutagenesis%2Bof%2Bthe%2BeT1%2Bbalanced%2Bregion%2Bin%2BCaenorhabditis%2Belegans%253A%2Bdose-response%2Bcurve%2Band%2Bthe%2Banalysis%2Bof%2Bmutational%2Bevents.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252888%252990120-0%26rft_id%253Dinfo%253Apmid%252F3419443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(88)90120-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3419443&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic analysis of a major segment [LGV(left)] of the genome of Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJohnsen%26rft.auinit1%253DR.%2BC.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D735%26rft.epage%253D752%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Ba%2Bmajor%2Bsegment%2B%255BLGV%2528left%2529%255D%2Bof%2Bthe%2Bgenome%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F1752418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzMvNzM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.29" data-doi="10.1186/1471-2164-8-402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maydan</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moerman</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Oligonucleotide array comparative genomic hybridization (oaCGH) based characterization of genetic deficiencies as an aid to gene mapping in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DJones%26rft.auinit1%253DM.%2BR.%26rft.volume%253D8%26rft.spage%253D402%26rft.epage%253D402%26rft.atitle%253DOligonucleotide%2BArray%2BComparative%2BGenomic%2BHybridization%2B%2528oaCGH%2529%2Bbased%2Bcharacterization%2Bof%2Bgenetic%2Bdeficiencies%2Bas%2Ban%2Baid%2Bto%2Bgene%2Bmapping%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-402%26rft_id%253Dinfo%253Apmid%252F17986355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17986355&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-resolution array comparative genomic hybridization analysis reveals unanticipated complexity of genetic deficiencies on chromosome V in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DJones%26rft.auinit1%253DM.%2BR.%26rft.volume%253D282%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D46%26rft.atitle%253DHigh-resolution%2Barray%2Bcomparative%2Bgenomic%2Bhybridization%2Banalysis%2Breveals%2Bunanticipated%2Bcomplexity%2Bof%2Bgenetic%2Bdeficiencies%2Bon%2Bchromosome%2BV%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F19330515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19330515&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.31" data-doi="10.1038/nature01278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamath</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Systematic functional analysis of the <em>Caenorhabditis elegans</em> genome using RNAi.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKamath%26rft.auinit1%253DR.%2BS.%26rft.volume%253D421%26rft.issue%253D6920%26rft.spage%253D231%26rft.epage%253D237%26rft.atitle%253DSystematic%2Bfunctional%2Banalysis%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bgenome%2Busing%2BRNAi.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01278%26rft_id%253Dinfo%253Apmid%252F12529635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12529635&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180397600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanbayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsukawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hokkoku</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Adult-onset vanishing white matter disease with novel missense mutations in a subunit of translational regulator, EIF2B4.</span> <abbr class="cit-jnl-abbrev">Clin. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BGenet.%26rft.volume%253D88%26rft.spage%253D401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.33" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.34" data-doi="10.1146/annurev.genom.1.1.99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">How many genes can make a cell: the minimal-gene-set concept.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D99%26rft.epage%253D116%26rft.atitle%253DHOW%2BMANY%2BGENES%2BCAN%2BMAKE%2BA%2BCELL%253A%2BThe%2BMinimal-Gene-Set%2BConcept%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.1.1.99%26rft_id%253Dinfo%253Apmid%252F11701626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.1.1.99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11701626&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165768900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.35" data-doi="10.1093/bioinformatics/btm091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Enhancements and modifications of primer design program Primer3.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm091%26rft_id%253Dinfo%253Apmid%252F17379693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17379693&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247348300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.36" data-doi="10.1128/MCB.10.5.2081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The Caenorhabditis elegans rol-6 gene, which interacts with the sqt-1 collagen gene to determine organismal morphology, encodes a collagen.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2081</span>–<span class="cit-lpage">2089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKramer%26rft.auinit1%253DJ%2BM%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D2081%26rft.epage%253D2089%26rft.atitle%253DThe%2BCaenorhabditis%2Belegans%2Brol-6%2Bgene%252C%2Bwhich%2Binteracts%2Bwith%2Bthe%2Bsqt-1%2Bcollagen%2Bgene%2Bto%2Bdetermine%2Borganismal%2Bmorphology%252C%2Bencodes%2Ba%2Bcollagen.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.5.2081%26rft_id%253Dinfo%253Apmid%252F1970117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzUvMjA4MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.37" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenna</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGettigan</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal W and Clustal X version 2.0.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846036&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251197700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.38" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.39" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.40" data-doi="10.1016/j.tig.2007.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene essentiality, gene duplicability and protein connectivity in human and mouse.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DToth%26rft.auinit1%253DA.%2BL.%26rft.volume%253D23%26rft.issue%253D8%26rft.spage%253D375%26rft.epage%253D378%26rft.atitle%253DGene%2Bessentiality%252C%2Bgene%2Bduplicability%2Band%2Bprotein%2Bconnectivity%2Bin%2Bhuman%2Band%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.04.005%26rft_id%253Dinfo%253Apmid%252F17512629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17512629&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248919300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.41" data-doi="10.1016/j.tig.2007.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">B. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mouse duplicate genes are as essential as singletons.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">378</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLiao%26rft.auinit1%253DB.%2BY.%26rft.volume%253D23%26rft.issue%253D8%26rft.spage%253D378%26rft.epage%253D381%26rft.atitle%253DMouse%2Bduplicate%2Bgenes%2Bare%2Bas%2Bessential%2Bas%2Bsingletons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.05.006%26rft_id%253Dinfo%253Apmid%252F17559966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17559966&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248919300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.42" data-doi="10.1093/molbev/msl076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">B. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Impacts of gene essentiality, expression pattern, and gene compactness on the evolutionary rate of mammalian proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2072</span>–<span class="cit-lpage">2080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl076%26rft_id%253Dinfo%253Apmid%252F16887903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16887903&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241094700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.43" data-doi="10.1534/genetics.104.040121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maciejowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cipriani</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Killian</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Autosomal genes of autosomal/X-linked duplicated gene pairs and germ-line proliferation in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1997</span>–<span class="cit-lpage">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.040121%26rft_id%253Dinfo%253Apmid%252F15687263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzE5OTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzMvOTgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.44" data-doi="10.1038/86920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maser</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinkel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrini</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">An alternative mode of translation permits production of a variant NBS1 protein from the common Nijmegen breakage syndrome allele.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMaser%26rft.auinit1%253DR.%2BS.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D417%26rft.epage%253D421%26rft.atitle%253DAn%2Balternative%2Bmode%2Bof%2Btranslation%2Bpermits%2Bproduction%2Bof%2Ba%2Bvariant%2BNBS1%2Bprotein%2Bfrom%2Bthe%2Bcommon%2BNijmegen%2Bbreakage%2Bsyndrome%2Ballele.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F86920%26rft_id%253Dinfo%253Apmid%252F11279524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/86920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11279524&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167811200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.45" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mello</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stinchcomb</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambros</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Efficient gene transfer in C.elegans: extrachromosomal maintenance and integration of transforming sequences.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">3959</span>–<span class="cit-lpage">3970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DMello%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D3959%26rft.epage%253D3970%26rft.atitle%253DEfficient%2Bgene%2Btransfer%2Bin%2BC.elegans%253A%2Bextrachromosomal%2Bmaintenance%2Band%2Bintegration%2Bof%2Btransforming%2Bsequences.%26rft_id%253Dinfo%253Apmid%252F1935914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1935914&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GQ58800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.47" data-doi="10.1371/journal.pcbi.1001082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mendonça</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira-Leal</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Loss of genetic redundancy in reductive genome evolution.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DMendonca%26rft.auinit1%253DA.%2BG.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001082%26rft.epage%253De1001082%26rft.atitle%253DLoss%2Bof%2Bgenetic%2Bredundancy%2Bin%2Breductive%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1001082%26rft_id%253Dinfo%253Apmid%252F21379323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1001082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21379323&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.48" data-doi="10.1038/nprot.2013.092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muruganujan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casagrande</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Large-scale gene function analysis with the PANTHER classification system.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1551</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1551%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.092%26rft_id%253Dinfo%253Apmid%252F23868073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23868073&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.49" data-doi="10.1534/genetics.117.300179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mok</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gevirtzman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">MIP-MAP: high-throughput mapping of <em>Caenorhabditis elegans</em> temperature-sensitive mutants via molecular inversion probes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300179%26rft_id%253Dinfo%253Apmid%252F28827289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA3LzIvNDQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.50" data-doi="10.1007/978-1-59745-515-2_5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">InterPro and InterProScan: tools for protein sequence classification and comparison.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">396</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DMulder%26rft.auinit1%253DN.%26rft.volume%253D396%26rft.spage%253D59%26rft.epage%253D70%26rft.atitle%253DInterPro%2Band%2BInterProScan%253A%2Btools%2Bfor%2Bprotein%2Bsequence%2Bclassification%2Band%2Bcomparison.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-515-2_5%26rft_id%253Dinfo%253Apmid%252F18025686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-515-2_5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18025686&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.51" data-doi="10.1038/40618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerlijst</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Evolution of genetic redundancy.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">388</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNowak%26rft.auinit1%253DM.%2BA.%26rft.volume%253D388%26rft.issue%253D6638%26rft.spage%253D167%26rft.epage%253D171%26rft.atitle%253DEvolution%2Bof%2Bgenetic%2Bredundancy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F40618%26rft_id%253Dinfo%253Apmid%252F9217155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/40618&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9217155&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XK10900048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.52" data-doi="10.1056/NEJMoa1103283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Papaemmanuil</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cazzola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boultwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malcovati</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Somatic SF3B1 mutation in myelodysplasia with ring sideroblasts.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DPapaemmanuil%26rft.auinit1%253DE.%26rft.volume%253D365%26rft.issue%253D15%26rft.spage%253D1384%26rft.epage%253D1395%26rft.atitle%253DSomatic%2BSF3B1%2Bmutation%2Bin%2Bmyelodysplasia%2Bwith%2Bring%2Bsideroblasts.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1103283%26rft_id%253Dinfo%253Apmid%252F21995386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1103283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21995386&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295782500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.53" data-doi="10.1038/ng1472"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parkinson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitreva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitton</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A transcriptomic analysis of the phylum Nematoda.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1259</span>–<span class="cit-lpage">1267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DParkinson%26rft.auinit1%253DJ.%26rft.volume%253D36%26rft.issue%253D12%26rft.spage%253D1259%26rft.epage%253D1267%26rft.atitle%253DA%2Btranscriptomic%2Banalysis%2Bof%2Bthe%2Bphylum%2BNematoda.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1472%26rft_id%253Dinfo%253Apmid%252F15543149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1472&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15543149&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225354100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.54" data-doi="10.1073/pnas.92.19.8574"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tainer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Getzoff</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulianne</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Subunit-destabilizing mutations in Drosophila copper/zinc superoxide dismutase: neuropathology and a model of dimer dysequilibrium.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">8574</span>–<span class="cit-lpage">8578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPhillips%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D92%26rft.issue%253D19%26rft.spage%253D8574%26rft.epage%253D8578%26rft.atitle%253DSubunit-destabilizing%2Bmutations%2Bin%2BDrosophila%2Bcopper%252Fzinc%2Bsuperoxide%2Bdismutase%253A%2Bneuropathology%2Band%2Ba%2Bmodel%2Bof%2Bdimer%2Bdysequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.19.8574%26rft_id%253Dinfo%253Apmid%252F7567977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTIvMTkvODU3NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.55" data-doi="10.1086/430721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ragge</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poloschek</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Heterozygous mutations of OTX2 cause severe ocular malformations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1008</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DRagge%26rft.auinit1%253DN.%2BK.%26rft.volume%253D76%26rft.issue%253D6%26rft.spage%253D1008%26rft.epage%253D1022%26rft.atitle%253DHeterozygous%2Bmutations%2Bof%2BOTX2%2Bcause%2Bsevere%2Bocular%2Bmalformations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F430721%26rft_id%253Dinfo%253Apmid%252F15846561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/430721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15846561&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229047900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.56" data-doi="10.1006/jmbi.2000.5197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storm</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Automatic clustering of orthologs and in-paralogs from pairwise species comparisons.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">1041</span>–<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DRemm%26rft.auinit1%253DM.%26rft.volume%253D314%26rft.issue%253D5%26rft.spage%253D1041%26rft.epage%253D1052%26rft.atitle%253DAutomatic%2Bclustering%2Bof%2Borthologs%2Band%2Bin-paralogs%2Bfrom%2Bpairwise%2Bspecies%2Bcomparisons.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.5197%26rft_id%253Dinfo%253Apmid%252F11743721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.5197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11743721&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173365300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.57" data-doi="10.1042/bj20030649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritty</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broekelmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werneck</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mecham</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Fibrillin-1 and -2 contain heparin-binding sites important for matrix deposition and that support cell attachment.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">375</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj20030649%26rft_id%253Dinfo%253Apmid%252F12837131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiMzc1LzIvNDI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The putative chemoreceptor families of <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">WormBook</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWormBook%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.59" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.60" data-doi="10.1186/1471-2164-11-131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilenky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butterfield</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malhis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic sequence of a mutant strain of <em>Caenorhabditis elegans</em> with an altered recombination pattern.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRose%26rft.auinit1%253DA.%2BM.%26rft.volume%253D11%26rft.spage%253D131%26rft.epage%253D131%26rft.atitle%253DGenomic%2Bsequence%2Bof%2Ba%2Bmutant%2Bstrain%2Bof%2BCaenorhabditis%2Belegans%2Bwith%2Ban%2Baltered%2Brecombination%2Bpattern.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-131%26rft_id%253Dinfo%253Apmid%252F20178641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178641&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenbluth</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">The genetic analysis of a reciprocal translocation, <em>eT1(III</em>; <em>V)</em>, in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D99%26rft.spage%253D415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.62" data-doi="10.1016/0027-5107(83)90016-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenbluth</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuddeford</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Mutagenesis in <em>Caenorhabditis elegans</em>. I. A rapid eukaryotic mutagen test system using the reciprocal translocation <em>eTI(III;V)</em>.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Fundam. Mol. Mech. Mutagen.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BFundam.%2BMol.%2BMech.%2BMutagen.%26rft.volume%253D110%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252883%252990016-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(83)90016-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenbluth</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuddeford</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Mutagenesis in <em>Caenorhabditis elegans</em>. II. A spectrum of mutational events induced with 1500 r of γ-radiation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRosenbluth%26rft.auinit1%253DR.%2BE.%26rft.volume%253D109%26rft.issue%253D3%26rft.spage%253D493%26rft.epage%253D511%26rft.atitle%253DMUTAGENESIS%2BIN%2BCAENORHABDITIS%2BELEGANS.%2BII.%2BA%2BSPECTRUM%2BOF%2BMUTATIONAL%2BEVENTS%2BINDUCED%2BWITH%2B1500%2BR%2BOF%2Bgamma-RADIATION%26rft_id%253Dinfo%253Apmid%252F3979812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA5LzMvNDkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.64" data-doi="10.1017/S0016672300027476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenbluth</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogalski</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Addison</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Genomic organization in <em>Caenorhabditis elegans</em>: deficiency mapping on linkage group V(left).</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D52%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300027476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300027476&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988R286300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.65" data-doi="10.1038/nmeth.1249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Meara</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pe’er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> mutant allele identification by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSarin%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D865%26rft.epage%253D867%26rft.atitle%253DCaenorhabditis%2Belegans%2Bmutant%2Ballele%2Bidentification%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1249%26rft_id%253Dinfo%253Apmid%252F18677319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677319&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259650100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furst</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurwitz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Simultaneous deficiency of sphingolipid activator proteins 1 and 2 is caused by a mutation in the initiation codon of their common gene.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">267</span>: <span class="cit-fpage">3312</span>–<span class="cit-lpage">3315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DSchnabel%26rft.auinit1%253DD.%26rft.volume%253D267%26rft.issue%253D5%26rft.spage%253D3312%26rft.epage%253D3315%26rft.atitle%253DSimultaneous%2Bdeficiency%2Bof%2Bsphingolipid%2Bactivator%2Bproteins%2B1%2Band%2B2%2Bis%2Bcaused%2Bby%2Ba%2Bmutation%2Bin%2Bthe%2Binitiation%2Bcodon%2Bof%2Btheir%2Bcommon%2Bgene%26rft_id%253Dinfo%253Apmid%252F1371116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNjcvNS8zMzEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.67" data-doi="10.1093/nar/gkj109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reguly</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">BioGRID: a general repository for interaction datasets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D535</span>–<span class="cit-lpage">D539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStark%26rft.auinit1%253DC.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD535%26rft.epage%253DD539%26rft.atitle%253DBioGRID%253A%2Ba%2Bgeneral%2Brepository%2Bfor%2Binteraction%2Bdatasets%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj109%26rft_id%253Dinfo%253Apmid%252F16381927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16381927&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239307700116&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.68" data-doi="10.1016/0165-1218(91)90107-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenbluth</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Most ultraviolet irradiation induced mutations in the nematode <em>Caenorhabditis elegans</em> are chromosomal rearrangements.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">249</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DStewart%26rft.auinit1%253DH.%2BI.%26rft.volume%253D249%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D54%26rft.atitle%253DMost%2Bultraviolet%2Birradiation%2Binduced%2Bmutations%2Bin%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans%2Bare%2Bchromosomal%2Brearrangements.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0165-1218%252891%252990107-W%26rft_id%253Dinfo%253Apmid%252F2067542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0165-1218(91)90107-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2067542&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FV53400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janke</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franz</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlin</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Lethal mutations defining 112 complementation groups in a 4.5 Mb sequenced region of <em>Caenorhabditis elegans</em> chromosome III.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">260</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DStewart%26rft.auinit1%253DH.%2BI.%26rft.volume%253D260%26rft.issue%253D2-3%26rft.spage%253D280%26rft.epage%253D288%26rft.atitle%253DLethal%2Bmutations%2Bdefining%2B112%2Bcomplementation%2Bgroups%2Bin%2Ba%2B4.5%2BMb%2Bsequenced%2Bregion%2Bof%2BCaenorhabditis%2Belegans%2Bchromosome%2BIII.%26rft_id%253Dinfo%253Apmid%252F9862482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9862482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077435400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.70" data-doi="10.1038/ng0997-58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilpatrick</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erlacher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lembessis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Disruption of human limb morphogenesis by a dominant negative mutation in CDMP1.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DThomas%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D58%26rft.epage%253D64%26rft.atitle%253DDisruption%2Bof%2Bhuman%2Blimb%2Bmorphogenesis%2Bby%2Ba%2Bdominant%2Bnegative%2Bmutation%2Bin%2BCDMP1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0997-58%26rft_id%253Dinfo%253Apmid%252F9288098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0997-58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9288098&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU72400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.71" data-doi="10.1093/nar/25.24.4876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plewniak</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeanmougin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">4876</span>–<span class="cit-lpage">4882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.24.4876%26rft_id%253Dinfo%253Apmid%252F9396791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.24.4876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9396791&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071498400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.72" data-doi="10.1101/gr.157651.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strasbourger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The million mutation project: a new approach to genetics in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.157651.113%26rft_id%253Dinfo%253Apmid%252F23800452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMC8xNzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8zLzk4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.73" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.74" data-doi="10.1093/nar/gks596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Primer3–new capabilities and interfaces.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks596%26rft_id%253Dinfo%253Apmid%252F22730293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22730293&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.75" data-doi="10.1186/1756-0500-5-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vergara</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frech</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CooVar: co-occurring variant analyzer.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DVergara%26rft.auinit1%253DI.%2BA.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DCooVar%253A%2BCo-occurring%2Bvariant%2Banalyzer.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-5-615%26rft_id%253Dinfo%253Apmid%252F23116482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-5-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23116482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.76" data-doi="10.1534/g3.117.300193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verster</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mateo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derry</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Taxonomically restricted genes with essential functions frequently play roles in chromosome segregation in <em>Caenorhabditis elegans</em> and <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3337</span>–<span class="cit-lpage">3347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300193%26rft_id%253Dinfo%253Apmid%252F28839119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjcvMTAvMzMzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMy85ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.77" data-doi="10.1038/74174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Robustness against mutations in genetic networks of yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWagner%26rft.auinit1%253DA.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D361%26rft.atitle%253DRobustness%2Bagainst%2Bmutations%2Bin%2Bgenetic%2Bnetworks%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F74174%26rft_id%253Dinfo%253Apmid%252F10742097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/74174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10742097&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086192800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.3.981.78" data-doi="10.1016/j.cell.2015.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yona</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frumkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A relay race on the evolutionary adaptation spectrum.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D549%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.10.005%26rft_id%253Dinfo%253Apmid%252F26496602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26496602&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F3%2F981.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/3/981.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/3/967" title="Systematic Functional Characterization of Human 21st Chromosome Orthologs in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/3/999" title="Tuning CRISPR-Cas9 Gene Drives in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/3"><span>Volume 8 Issue 3, March 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/3.cover-source.jpg?itok=o8h2cMy2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/3/981.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TMS0X1AePRBxuT_cKqr1tAqPmN39Dl5mnqbuh4NxCVE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/3/981" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/3/981" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/3/981" class="active" data-icon-position="" data-hide-link-title="0">Genomic Identification and Functional Characterization of Essential Genes in Caenorhabditis elegans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488772" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ug9QRM-UdPyYNS8HCRdnT1kxNYcAhe5t43rtLgscwmc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/3/981.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/3/981" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FCfU4oVkbjHCBZ5Yp06IWx5bzqWcXVHg2NjfoVvNZwc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/3/981" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/3/981.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488772" id="node488772--2" data-pisa="ggg;8/3/981" data-pisa-master="ggg;g3.117.300338" data-apath="/ggg/8/3/981.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Identification and Functional Characterization of Essential Genes in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhaozhao Qin</span>, <span class="highwire-citation-author" data-delta="1">Robert Johnsen</span>, <span class="highwire-citation-author" data-delta="2">Shicheng Yu</span>, <span class="highwire-citation-author" data-delta="3">Jeffrey Shih-Chieh Chu</span>, <span class="highwire-citation-author" data-delta="4">David L. Baillie</span> and <span class="highwire-citation-author" data-delta="5">Nansheng Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">981-997;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300338 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488772--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhaozhao Qin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaozhao%2BQin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qin%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaozhao%2BQin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert Johnsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJohnsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnsen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJohnsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shicheng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Combinatorial Biosynthesis and Drug Discovery, School of Pharmaceutical Sciences, Wuhan University, 430071, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wuhan Frasergen Bioinformatics, Wuhan East Lake High-Tech Zone, 430075, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shicheng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShicheng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey Shih-Chieh Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wuhan Frasergen Bioinformatics, Wuhan East Lake High-Tech Zone, 430075, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BShih-Chieh%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BShih-Chieh%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David L. Baillie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BBaillie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baillie%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BL.%2BBaillie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nansheng Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nansheng%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANansheng%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chenn@sfu.ca" class="" data-icon-position="" data-hide-link-title="0">chenn@sfu.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/3/981" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/3/981&amp;title=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/3/981&amp;title=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/3/981&amp;title=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/3/981&amp;text=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/3/981&amp;title=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/3/981&amp;t=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/3/981&amp;title=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/3/981&amp;title=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488772/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488772" id="node488772--3" data-pisa="ggg;8/3/981" data-pisa-master="ggg;g3.117.300338" data-apath="/ggg/8/3/981.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Identification and Functional Characterization of Essential Genes in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhaozhao Qin</span>, <span class="highwire-citation-author" data-delta="1">Robert Johnsen</span>, <span class="highwire-citation-author" data-delta="2">Shicheng Yu</span>, <span class="highwire-citation-author" data-delta="3">Jeffrey Shih-Chieh Chu</span>, <span class="highwire-citation-author" data-delta="4">David L. Baillie</span> and <span class="highwire-citation-author" data-delta="5">Nansheng Chen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">981-997;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300338 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488772--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhaozhao Qin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaozhao%2BQin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qin%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaozhao%2BQin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert Johnsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJohnsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnsen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJohnsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shicheng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Combinatorial Biosynthesis and Drug Discovery, School of Pharmaceutical Sciences, Wuhan University, 430071, China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wuhan Frasergen Bioinformatics, Wuhan East Lake High-Tech Zone, 430075, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shicheng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShicheng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey Shih-Chieh Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Wuhan Frasergen Bioinformatics, Wuhan East Lake High-Tech Zone, 430075, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BShih-Chieh%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BShih-Chieh%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David L. Baillie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL.%2BBaillie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baillie%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BL.%2BBaillie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nansheng Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, British Columbia V5A 1S6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nansheng%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANansheng%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chenn@sfu.ca" class="" data-icon-position="" data-hide-link-title="0">chenn@sfu.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=981&amp;rft.epage=997&amp;rft.atitle=Genomic%20Identification%20and%20Functional%20Characterization%20of%20Essential%20Genes%20in%20Caenorhabditis%20elegans&amp;rft.volume=8&amp;rft.issue=3&amp;rft.date=2018-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Qin%2C+Zhaozhao&amp;rft.au=Johnsen%2C+Robert&amp;rft.au=Yu%2C+Shicheng&amp;rft.au=Chu%2C+Jeffrey+Shih-Chieh&amp;rft.au=Baillie%2C+David+L.&amp;rft.au=Chen%2C+Nansheng"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488772/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488772/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488772/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488772/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488772/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488772/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488772/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488772/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488772/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488772/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488772/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488772/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/3/981" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/3/981.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/3/981.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/3/981.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/3/981.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
