<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/5/537" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/5/537.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/5/537.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/5/537.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/5/537/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/5/537.atom" />
<meta name="HW.pisa" content="ggg;2/5/537" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Two-Phase Analysis in Consensus Genetic Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.112.002428" />
<meta name="DC.Date" content="2012-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Ronin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Numerous mapping projects conducted on different species have generated an abundance of mapping data. Consequently, many multilocus maps have been constructed using diverse mapping populations and marker sets for the same organism. The quality of maps varies broadly among populations, marker sets, and software used, necessitating efforts to integrate the mapping information and generate consensus maps. The problem of consensus genetic mapping (MCGM) is by far more challenging compared with genetic mapping based on a single dataset, which by itself is also cumbersome. The additional complications introduced by consensus analysis include inter-population differences in recombination rate and exchange distribution along chromosomes; variations in dominance of the employed markers; and use of different subsets of markers in different labs. Hence, it is necessary to handle arbitrary patterns of shared sets of markers and different level of mapping data quality. In this article, we introduce a two-phase approach for solving MCGM. In phase 1, for each dataset, multilocus ordering is performed combined with iterative jackknife resampling to evaluate the stability of marker orders. In this phase, the ordering problem is reduced to the well-known traveling salesperson problem (TSP). Namely, for each dataset, we look for order that gives minimum sum of recombination distances between adjacent markers. In phase 2, the optimal consensus order of shared markers is selected from the set of allowed orders and gives the minimal sum of total lengths of nonconflicting maps of the chromosome. This criterion may be used in different modifications to take into account the variation in quality of the original data (population size, marker quality, etc .). In the foregoing formulation, consensus mapping is considered as a specific version of TSP that can be referred to as “synchronized TSP.” The conflicts detected after phase 1 are resolved using either a heuristic algorithm over the entire chromosome or an exact/heuristic algorithm applied subsequently to the revealed small non-overlapping regions with conflicts separated by non-conflicting regions. The proposed approach was tested on a wide range of simulated data and real datasets from maize." />
<meta name="DC.Contributor" content="Y. Ronin" />
<meta name="DC.Contributor" content="D. Mester" />
<meta name="DC.Contributor" content="D. Minkov" />
<meta name="DC.Contributor" content="R. Belotserkovski" />
<meta name="DC.Contributor" content="B. N. Jackson" />
<meta name="DC.Contributor" content="P. S. Schnable" />
<meta name="DC.Contributor" content="S. Aluru" />
<meta name="DC.Contributor" content="A. Korol" />
<meta name="article:published_time" content="2012-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Two-Phase Analysis in Consensus Genetic Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Numerous mapping projects conducted on different species have generated an abundance of mapping data. Consequently, many multilocus maps have been constructed using diverse mapping populations and marker sets for the same organism. The quality of maps varies broadly among populations, marker sets, and software used, necessitating efforts to integrate the mapping information and generate consensus maps. The problem of consensus genetic mapping (MCGM) is by far more challenging compared with genetic mapping based on a single dataset, which by itself is also cumbersome. The additional complications introduced by consensus analysis include inter-population differences in recombination rate and exchange distribution along chromosomes; variations in dominance of the employed markers; and use of different subsets of markers in different labs. Hence, it is necessary to handle arbitrary patterns of shared sets of markers and different level of mapping data quality. In this article, we introduce a two-phase approach for solving MCGM. In phase 1, for each dataset, multilocus ordering is performed combined with iterative jackknife resampling to evaluate the stability of marker orders. In this phase, the ordering problem is reduced to the well-known traveling salesperson problem (TSP). Namely, for each dataset, we look for order that gives minimum sum of recombination distances between adjacent markers. In phase 2, the optimal consensus order of shared markers is selected from the set of allowed orders and gives the minimal sum of total lengths of nonconflicting maps of the chromosome. This criterion may be used in different modifications to take into account the variation in quality of the original data (population size, marker quality, &lt;i&gt;etc&lt;/i&gt;.). In the foregoing formulation, consensus mapping is considered as a specific version of TSP that can be referred to as “synchronized TSP.” The conflicts detected after phase 1 are resolved using either a heuristic algorithm over the entire chromosome or an exact/heuristic algorithm applied subsequently to the revealed small non-overlapping regions with conflicts separated by non-conflicting regions. The proposed approach was tested on a wide range of simulated data and real datasets from maize.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/05/01" />
<meta name="citation_mjid" content="ggg;2/5/537" />
<meta name="citation_id" content="2/5/537" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/5/537" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/5/537.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/5/537.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/5/537.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002428" />
<meta name="citation_pmid" content="22670224" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="5" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="537" />
<meta name="citation_lastpage" content="549" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Y. Ronin" />
<meta name="citation_author_institution" content="Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel" />
<meta name="citation_author" content="D. Mester" />
<meta name="citation_author_institution" content="Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel" />
<meta name="citation_author" content="D. Minkov" />
<meta name="citation_author_institution" content="Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel" />
<meta name="citation_author" content="R. Belotserkovski" />
<meta name="citation_author_institution" content="Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel" />
<meta name="citation_author" content="B. N. Jackson" />
<meta name="citation_author_institution" content="Department of Electrical and Computer Engineering, Iowa State University, Iowa 50014" />
<meta name="citation_author" content="P. S. Schnable" />
<meta name="citation_author_institution" content="Center for Plant Genomics, Iowa State University, Iowa 50011" />
<meta name="citation_author" content="S. Aluru" />
<meta name="citation_author_institution" content="Department of Electrical and Computer Engineering, Iowa State University, Iowa 50014" />
<meta name="citation_author" content="A. Korol" />
<meta name="citation_author_institution" content="Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel" />
<meta name="citation_author_email" content="korol@research.haifa.ac.il" />
<meta name="citation_reference" content="citation_author=O. Bräysy;citation_author=M. Gendreau;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. De Givry;citation_author=M. Bouchez;citation_author=P. Chabrier;citation_author=D. Milan;citation_author=T. Schiex;citation_title=CarthaGene: multipopulation integrated genetic and radiation hybrid mapping;citation_volume=21;citation_year=2005;citation_pmid=15598829;citation_doi=10.1093/bioinformatics/bti222" />
<meta name="citation_reference" content="citation_journal_title=BMC molecular biology [electronic resource];citation_journal_abbrev=BMC Mol Biol;citation_author=E. De Keyser;citation_author=O. Y. Shu;citation_author=E. Van Bockstaele;citation_author=J. De Riek;citation_title=Multipoint-likelihood maximization mapping on 4 segregating populations to achieve an integrated framework map for QTL analysis in pot azalea (Rhododendron simsii hybrids).;citation_pages=1-1;citation_volume=11;citation_year=2010;citation_pmid=20070894;citation_doi=10.1186/1471-2199-11-1" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=C. T. Falk;citation_title=Preliminary ordering of multiple linked loci using pairwise linkage data.;citation_pages=367-375;citation_volume=9;citation_year=1992;citation_issue=5;citation_pmid=1427024;citation_doi=10.1002/gepi.1370090507" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=T. Faraut;citation_author=S. de Givry;citation_author=P. Chabrier;citation_author=T. Derrien;citation_author=F. Galibert;citation_title=A comparative genome approach to marker ordering;citation_pages=e50-e56;citation_volume=23;citation_year=2007;citation_issue=2;citation_pmid=17237105;citation_doi=10.1093/bioinformatics/btl321" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Hall;citation_author=M. Bhandarkar;citation_author=J. Arnold;citation_author=T. Jiang;citation_title=Physical mapping with automatic capture of hybridization data;citation_pages=205-213;citation_volume=3;citation_year=2001" />
<meta name="citation_reference" content="citation_author=B. N. Jackson;citation_author=P. S. Schnable;citation_author=S. Aluru;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. E. Klein;citation_author=R. R. Klein;citation_author=S. W. Cartinhour;citation_author=P. E. Ulanch;citation_author=J. Dong;citation_title=A High-throughput AFLP-based Method for Constructing Integrated Genetic and Physical Maps: Progress Toward a Sorghum Genome Map;citation_pages=789-807;citation_volume=10;citation_year=2000;citation_issue=6;citation_pmid=10854411;citation_doi=10.1101/gr.10.6.789" />
<meta name="citation_reference" content="citation_author=A. B. Korol;citation_year=2001" />
<meta name="citation_reference" content="citation_author=A. B. Korol;citation_author=D. Mester;citation_author=Z. Frenkel;citation_author=Y. Ronin;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. E. Kwitek;citation_author=P. J. Tonellato;citation_author=D. Chen;citation_author=J. Gullings-Handley;citation_author=Y. S. Cheng;citation_title=Automated Construction of High-Density Comparative Maps Between Rat, Human, and Mouse;citation_pages=1935-1943;citation_volume=11;citation_year=2001;citation_issue=11;citation_pmid=11691858" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. Menotti-Raymond;citation_author=V. A. David;citation_author=Z. Q. Chen;citation_author=K. A. Menotti;citation_author=S. Sun;citation_title=Second-Generation Integrated Genetic Linkage/Radiation Hybrid Maps of the Domestic Cat (Felis catus);citation_pages=95-106;citation_volume=94;citation_year=2003;citation_issue=1;citation_pmid=12692169;citation_doi=10.1093/jhered/esg008" />
<meta name="citation_reference" content="citation_journal_title=Comput. Oper. Res.;citation_author=D. Mester;citation_author=O. Bräysy;citation_title=Active guided evolution strategies for large-scale vehicle routing problems with time windows;citation_pages=1593-1614;citation_volume=32;citation_year=2005;citation_doi=10.1016/j.cor.2003.11.017" />
<meta name="citation_reference" content="citation_journal_title=Comput. Oper. Res.;citation_author=D. Mester;citation_author=O. Bräysy;citation_title=Active guided evolution strategies for large-scale capacitated vehicle routing problems;citation_pages=2964-2975;citation_volume=34;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Mester;citation_author=Y. Ronin;citation_author=D. Minkov;citation_author=E. Nevo;citation_author=A. Korol;citation_title=Constructing large-scale genetic maps using an evolutionary strategy algorithm;citation_pages=2269-2282;citation_volume=165;citation_year=2003a;citation_pmid=14704202" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Mester;citation_author=Y. Ronin;citation_author=J. Hu;citation_author=J. Peng;citation_author=E. Nevo;citation_title=Efficient multipoint mapping: making use of dominant repulsion-phase markers.;citation_pages=1102-1112;citation_volume=107;citation_year=2003b;citation_issue=6;citation_pmid=12928774;citation_doi=10.1007/s00122-003-1305-1" />
<meta name="citation_reference" content="citation_journal_title=Computational biology and chemistry;citation_journal_abbrev=Comput Biol Chem;citation_author=D. Mester;citation_author=Y. Ronin;citation_author=E. Nevo;citation_author=A. Korol;citation_title=Fast and high precision algorithms for optimization in large-scale genomic problems.;citation_pages=281-290;citation_volume=28;citation_year=2004;citation_issue=4;citation_pmid=15548455;citation_doi=10.1016/j.compbiolchem.2004.08.003" />
<meta name="citation_reference" content="citation_journal_title=Computational biology and chemistry;citation_journal_abbrev=Comput Biol Chem;citation_author=D. Mester;citation_author=Y. Ronin;citation_author=M. Korostishevsky;citation_author=V. Pikus;citation_author=A. Glazman;citation_title=Multilocus consensus genetic maps (MCGM): formulation, algorithms, and results.;citation_pages=12-20;citation_volume=30;citation_year=2006;citation_issue=1;citation_pmid=16301000;citation_doi=10.1016/j.compbiolchem.2005.09.007" />
<meta name="citation_reference" content="citation_journal_title=Expert Syst. Appl.;citation_author=D. Mester;citation_author=O. Bräysy;citation_author=W. Dullaert;citation_title=A multi-parametric evolution strategies algorithm for vehicle routing problems;citation_pages=508-517;citation_volume=32;citation_year=2007;citation_doi=10.1016/j.eswa.2005.12.014" />
<meta name="citation_reference" content="citation_title=Handbook of Optimization Theory: Decision Analysis and Application;citation_title=Discrete optimization for some TSP-like genome mapping problems;citation_author=D. Mester;citation_author=Y. Ronin;citation_author=M. Korostishevsky;citation_author=Z. Frenkel;citation_author=O. Bräysy;citation_pages=1-40;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. Molinari;citation_author=G. R. A. Margarido;citation_author=R. Vencovsky;citation_author=A. A. F. Garcia;citation_title=Evaluation of algorithms used to order markers on genetic maps.;citation_pages=494-502;citation_volume=103;citation_year=2009;citation_issue=6;citation_pmid=19639011;citation_doi=10.1038/hdy.2009.96" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. F. Mott;citation_author=A. V. Grigoriev;citation_author=E. Maier;citation_author=J. D. Hoheisel;citation_author=H. Lehrach;citation_title=Algorithms and software tools for ordering clone libraries: application to the mapping of the genome of Schizosaccharomyces pombe;citation_pages=1965-1974;citation_volume=21;citation_year=1993;citation_issue=8;citation_pmid=8493107;citation_doi=10.1093/nar/21.8.1965" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. C. Nelson;citation_author=M. E. Sorrells;citation_author=A. E. Van Deynze;citation_author=Y. H. Lu;citation_author=M. Atkinson;citation_title=Molecular mapping of wheat: major genes and rearrangements in because groups 4, 5, and 7;citation_pages=721-731;citation_volume=141;citation_year=1995;citation_pmid=8647405" />
<meta name="citation_reference" content="citation_title=Evolutionstrategie;citation_author=I. Rechenberg;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=Nat. Science;citation_author=Y. Ronin;citation_author=D. Mester;citation_author=D. Minkov;citation_author=A. B. Korol;citation_title=Building reliable genetic maps: different mapping strategies may result in different maps;citation_pages=576-589;citation_volume=2;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=ISMB;citation_author=T. Schiex;citation_author=C. Gaspin;citation_title=Carthagene: constructing and joining maximum likelihood genetic maps;citation_pages=258-267;citation_volume=5;citation_year=1997;citation_pmid=9322047" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Stam;citation_title=Construction of integrated genetic linkage maps by means of a new computer package: JoinMap;citation_pages=739-744;citation_volume=3;citation_year=1993;citation_doi=10.1111/j.1365-313X.1993.00739.x" />
<meta name="citation_reference" content="citation_title=Numeriche optimierung von vomputer-modellen mittels der evolutions strategie;citation_author=H.-P. Schwefel;citation_year=1977" />
<meta name="citation_reference" content="citation_author=C. Voudouris;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=D. Weeks;citation_author=K. Lange;citation_title=Preliminary ranking procedures for multilocus ordering.;citation_pages=236-242;citation_volume=1;citation_year=1987;citation_issue=3;citation_pmid=3446584;citation_doi=10.1016/0888-7543(87)90050-4" />
<meta name="citation_reference" content="citation_author=R. W. Williams;citation_author=J. Gu;citation_author=S. Qi;citation_author=L. Lu;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=J. Wu;citation_author=T. Maehara;citation_author=T. Shimokawa;citation_author=S. Yamamoto;citation_author=C. Harada;citation_title=A Comprehensive Rice Transcript Map Containing 6591 Expressed Sequence Tag Sites;citation_pages=525-535;citation_volume=14;citation_year=2002;citation_issue=3;citation_pmid=11910001;citation_doi=10.1105/tpc.010274" />
<meta name="citation_reference" content="citation_author=Y. Wu;citation_author=T. J. Close;citation_author=S. Lonardi;citation_year=2008" />
<meta name="citation_reference" content="citation_author=Y. Wu;citation_author=T. J. Close;citation_author=S. Lonardi;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. V. Yap;citation_author=D. Schneidler;citation_author=J. Kleinberg;citation_author=D. Mattews;citation_author=S. Cartinhour;citation_title=A graph-theoretic approach to comparing and integrating genetic, physical and sequence-based maps;citation_pages=2235-2247;citation_volume=165;citation_year=2003;citation_pmid=14704199" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Two-Phase Analysis in Consensus Genetic Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
<meta name="twitter:description" content="Numerous mapping projects conducted on different species have generated an abundance of mapping data. Consequently, many multilocus maps have been constructed using diverse mapping populations and marker sets for the same organism. The quality of maps varies broadly among populations, marker sets, and software used, necessitating efforts to integrate the mapping information and generate consensus maps. The problem of consensus genetic mapping (MCGM) is by far more challenging compared with genetic mapping based on a single dataset, which by itself is also cumbersome. The additional complications introduced by consensus analysis include inter-population differences in recombination rate and exchange distribution along chromosomes; variations in dominance of the employed markers; and use of different subsets of markers in different labs. Hence, it is necessary to handle arbitrary patterns of shared sets of markers and different level of mapping data quality. In this article, we introduce a two-phase approach for solving MCGM. In phase 1, for each dataset, multilocus ordering is performed combined with iterative jackknife resampling to evaluate the stability of marker orders. In this phase, the ordering problem is reduced to the well-known traveling salesperson problem (TSP). Namely, for each dataset, we look for order that gives minimum sum of recombination distances between adjacent markers. In phase 2, the optimal consensus order of shared markers is selected from the set of allowed orders and gives the minimal sum of total lengths of nonconflicting maps of the chromosome. This criterion may be used in different modifications to take into account the variation in quality of the original data (population size, marker quality, etc .). In the foregoing formulation, consensus mapping is considered as a specific version of TSP that can be referred to as “synchronized TSP.” The conflicts detected after phase 1 are resolved using either a heuristic algorithm over the entire chromosome or an exact/heuristic algorithm applied subsequently to the revealed small non-overlapping regions with conflicts separated by non-conflicting regions. The proposed approach was tested on a wide range of simulated data and real datasets from maize." />
<meta name="og-title" property="og:title" content="Two-Phase Analysis in Consensus Genetic Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/5/537" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Numerous mapping projects conducted on different species have generated an abundance of mapping data. Consequently, many multilocus maps have been constructed using diverse mapping populations and marker sets for the same organism. The quality of maps varies broadly among populations, marker sets, and software used, necessitating efforts to integrate the mapping information and generate consensus maps. The problem of consensus genetic mapping (MCGM) is by far more challenging compared with genetic mapping based on a single dataset, which by itself is also cumbersome. The additional complications introduced by consensus analysis include inter-population differences in recombination rate and exchange distribution along chromosomes; variations in dominance of the employed markers; and use of different subsets of markers in different labs. Hence, it is necessary to handle arbitrary patterns of shared sets of markers and different level of mapping data quality. In this article, we introduce a two-phase approach for solving MCGM. In phase 1, for each dataset, multilocus ordering is performed combined with iterative jackknife resampling to evaluate the stability of marker orders. In this phase, the ordering problem is reduced to the well-known traveling salesperson problem (TSP). Namely, for each dataset, we look for order that gives minimum sum of recombination distances between adjacent markers. In phase 2, the optimal consensus order of shared markers is selected from the set of allowed orders and gives the minimal sum of total lengths of nonconflicting maps of the chromosome. This criterion may be used in different modifications to take into account the variation in quality of the original data (population size, marker quality, etc .). In the foregoing formulation, consensus mapping is considered as a specific version of TSP that can be referred to as “synchronized TSP.” The conflicts detected after phase 1 are resolved using either a heuristic algorithm over the entire chromosome or an exact/heuristic algorithm applied subsequently to the revealed small non-overlapping regions with conflicts separated by non-conflicting regions. The proposed approach was tested on a wide range of simulated data and real datasets from maize." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5.cover-source.jpg" />
    <title>Two-Phase Analysis in Consensus Genetic Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8_Hexgx2sVgvZjAHP5JHGKfEVhE2zRkXQ_RWwnRVkd0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459280","apath":"\/ggg\/2\/5\/537.atom","pisa":"ggg;2\/5\/537","ac":{"\/ggg\/2\/5\/537.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/5\/537.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/5\/537"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/5\/537"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/5\/537":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223WEzOQTnnwsNmK0BtbjHBPw4tUdKezRFH75PDD+nUgOrfZO+0C05qompbRai+HdJu0xJ9sxoq6AFSj7nba0GyaA9mv6pK6IzBDkCJxO8B6tVsM\\\\\\\/+zn+MpClxZHFaTl8+vkpPwkxCuR4jOqG+d6XxEEGtVQyJQzybch1p\\\\\\\/rFoMFEdfFfh\\\\\\\/obBAXFB2D8o9t4xCBW2eeeREMkSD0Mna7HnLDJvl8AcA4tKBrR563cSirmWOKGGpvU4f6jSmnMqw0vevhXzfo2R8fHBofa6b1uE2TMxj\\\\\\\/VubL4d0htYqaJ+0ADoDI2qLEFb5N7lzx4dI4vr0O3vOJJUp5NJxD\\\\\\\/rbsXbBIRONvJ09l9AUV\\\\\\\/XlZctK+ZR0TXxNDtziSS3p7K6CgqzIZJU3wb5KTe+PeI9HeSmkwEPZC+7FpKWdqkmX9B3txWBiwt+YEQS9+Iujx6Ff23M8CR1EIvW6fERbORdL55mBNJIkoWf5Wy\\\\\\\/Qz5iBWqirKH4u7eWSR8bWEZuK2kwj\\\\\\\/Z7J4W3JI4OwlG4mYnbslTDStHTofSlHBuSbHTNZAyX0oWjfVT0gov0wHJ1h\\\\\\\/+bH7fPoHUupP6kgIBzHFoA+SIdZhqQhS06TdfUD2h\\\\\\\/e8ZvrQVf1QXIIYAJr9RI4UeOdgUMEqLYF1qZuN7o7sm346ksAzC9DPJA4Gy\\\\\\\/5BSwErvca57eXaK8EGMioY3\\\\\\\/ipn2S3ZojIx7nXcyAbcZqdcwFzS3uNAtwngs4es6MjKUMMnWK2RUhpBdM95+PW9GsFgyqRwvcGM7GUbbkqEyMOkqk87jRZyl3zdbNOaNZk5kOfcpSniJ5dUvvFtvwYsSLwlOIavgFUCgqJlZh8mM2djPgREszCCQxpY552i9BKPWKEWhk5xZYjFm4Ag0r7GG5R9bl0NIrZ\\\\\\\/oTWOB90Vvb7JzcM\\\\\\\/Dyh3l7fbbzMKCXjO3eeOXmRCqXBVm1abeg0QpkG63V\\\\\\\/yLG\\\\\\\/dsOq9aAYRh0++QU+9d9ayTs\\\\\\\/qGlEumOs+OmjpKpGZt0Sfjj1Vrzw91LePYlhft8zwqb8byca0SeW25XIG3Pizyi7lLJKTTKUNHzcRFnoaTux8ktr2FEy7fhlFFo21dE+um6FKHBWSz++rmZTdmp6MMay9n23pCbLnWwXboxXRILxlWnMLRLs8FtyEqmPeCgkXM2myEYQvfItJmn1MPgMLjpF6KA7rmq0DyaiZShLzKwuahZ5EVtsmWhWRVfCOGP5w2v7EqSdW76UuMKkPu7TkLoNrczlLjuHm9jcacS9BOcdF3wIA+L7Cm2TxtjQiF\\\u0022,\\\u0022iv\\\u0022:\\\u0022gTYukalMwvDTVOpaX1xWAg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002291e8b1fb14603dae7a1a9760509004bf9c9b30c31ada6ec9d49321d40c830213\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022p\\\\\\\/r1DCoHOpvo3B+\\\\\\\/7jbqVNIULZAlExSnWzQ4Rp7hEEIY\\\\\\\/cdRrC+cIL9fbL+vyMwr85ONZU9OGS0mL8za+hpw2sDb3P3I58jw5pRz\\\\\\\/NTVXLtAQCr70LXmZd\\\\\\\/l3DpXXDK0FiiIZWCgMqyOaAWgjoBVk\\\\\\\/tbsgzxU2of7yyX5frbmBhCHIBCQmCk3yWwONCQwgdF9D80i6xUPo+ZUaXNHhlDz\\\\\\\/t2r9taaDnrUWus7qnW1WFMzPN0qgAsPfSWb5y2GZAad5zXawNEg5H9gfFwCjA2kZxXd9BIfFDiM53JuwPCLNdjSYyJYDYXQMSqSLKXNDyvRwbO6XU7DRIfTO1ipzQK0enKFnlcUAz7Rol0NguyU1VU2NsZIPm7Lsyq8aAUcYxu07TodLL\\\\\\\/PglMGxoqr9vX+6LRerJBLCg4IK9ilWuzN9wDfWlsDKELEkMk7O4LW2Rez1oBsA9qojbD5jj9yBWjwwvayI38vmaPaoI7uU87NC6DpAJBLfGqVl6Jau092k1uPVRXqJ4ic\\\\\\\/HCiPPV+jbYVN8FMXgJAZu0Lv6gNnY4KgPs0O6lTYEy1fDWftJiIJpGufXhyU8JwscvBBvdPhG7xL3MbBR5FD2zSnU\\\\\\\/5XB+7g5Tk0r2JXWQj\\\\\\\/PKRn4P1qtPoYDVcvvQ\\\\\\\/Pnk6wCczz8+o9xGR4+X1cgFvvxpPNwKthHmzF2sTZc2nkxiWeenWWH5AGP14XgvfLLyc9hY4dx\\\\\\\/AaGwismk9CrlnrzPDDnpE6orMMrH6TmFwFTXJOC68jlMB0eaeP+6I0Q\\\\\\\/64m8YP60wGVBfYKUm932lL\\\\\\\/bZ7bjqLMUSvd0LAPdX7ZSouAeW5eTt6RijuIpLbVfk\\\\\\\/F7ZaS5Ih1gt9dLiDcVGghnBM9hza6o\\\\\\\/iQd4r3tiSLp5Fpo9Teg7GPn08ZFXp5GKOTAGN015svvJ1BTFBjlsBqg9OVTU9ZKgYNFYDNe99S59GixR5n8Q5AZnO+nkKt+TgLWiXQgSzdwh54VGA+\\\\\\\/RbxWGP9JNlE2l\\\\\\\/Al3aHkDxfgZdkxCQJ0hXeNPVWTeJBRJwm05RoFUwuQX84hd4jdAysK+6hfxnsTBPyTX2Dmdvp+2qkBdU3cfIKcacXopu4Ev2oV4HigMLmveSGrk4kz9V+uf9a+Kc3\\\\\\\/jpZZqr1ll\\\\\\\/FH5FjEVpQC72jej3lER7oWO17s2YkIecwzNZeYcLU1mDiXkq8m+f4L+Uy\\\\\\\/JN7BZWBzvchtoAmHyQMA01CCawNoZo1lWYSC8PgufYS+Iw7tBZxfQKflFc\\\\\\\/LioOUSofStzCH4DJEoKfMeVx34CI+LwXQF3D65tGDSuL5VnlE0EHLJxBU+OKL9HYB7RDHqzsS3DdHrwmypAHJSkYcY8tMGkzVBkgIQllgi5jpyjBMqDvo65lvknKcoCQWmXLJraRJeFPoR4NZZ7\\\\\\\/TgRjLhwUKFZjrGid9JgqJNvdLYgadKom2VL3d\\\\\\\/MZ8r5WE14gcg0ZocFjMNZPv4UdBswWGKHpc29mYvRq+7o38I3qkntoD9nStoumaiH6IwkwJsXzHC28FcaEcX6OWz\\\\\\\/\\\\\\\/Pzu4pVzeW9othk8RHGLJpB2IUtVn8HZYOwGBoKz6ITM8YlHfoFjbrXEgGCKNhs9tVKME+nSob\\\\\\\/Pml36yNcSOh2HlbR8GpinJXy7e6+MWleXFTQF5QbGAKcb2W8KGXuP7F8ULE0ToHQotFn8V3FHD2qagjFn2vtGyaxvhxesRdfw9xGPXZ4SvIvJRw\\\\\\\/tF5\\\\\\\/3SzibPPPqCUTuHxWc2EGinaUpvAwV8LCq1dm\\\\\\\/Fp37VSDXOkRLuRszuYyn5ReMTGusRkXnLwmOwAAcsVWsBz01KpVhXyp5Zj8WoWkNLtXnCrAZ1fD5KwloC\\\\\\\/AlXKLCq0ErnNK6qGXTria\\\\\\\/HLOUDt5grE4+PYkAQjzHy\\\\\\\/poIx5bQf2Ekz4lobpna5XT3DqBEe05LSdZlNT1RgYXQ9dJ1XiVggcqP1g\\\\\\\/ZxrTsetbts8aFBuY1yfCeViSN29YNZ15seTkWooedQlVat27f7O1mejQoIXcMpSUNPFxhspIr1mVBmq7yrwE1pC0SIi4OVfY88wd0jCyIyV\\\\\\\/NAYo5vxcjkLndwWuaegEpxozvdV0J4X1L\\\\\\\/NY3x5z1nx55Gkbn1Lm5GxvtVTkMEA6mp7On9+QEWZypkqRYBDi9g1Vq4jv228UcBNozHgD2XAuENqmcy\\\\\\\/DP9C2IREejprPKMYq96V8kiIR\\\\\\\/ExsGsM675zLBTxIwbXcd0x0mKj9siv5\\\\\\\/ssGH5xE3yQKvYMg7jLexXBBwOSZ2MA5UrhQVwyYH9QOlnclbHEhLFVRWckT\\\\\\\/mfp5UXjSbYbz0C8\\\\\\\/HiQnjhT3KOqy8dNUXkfbucN\\\\\\\/Cvs6A3e\\\\\\\/TSNvooDFCz+TQyYdabqKFYGu92Zve2kVTpmlH\\\\\\\/m49UTI8WkEER3hAUWDD7gaTJTI785nbk7ppt\\\\\\\/Cz2RzqlafzNb16vetp5+ISAoCGaF1JhXwpu909ZsPle8XuCsvyLGSJ0aKN2Zd8uSblYGcYo6rL0Vto+Kpj6xF2BZzoXpB\\\\\\\/2rN8e+hIDncf8Xt0Y9vuCIjO2e1PYe5c\\\\\\\/YGqsQKYlRA4fIXjYNN7H69NjGFKJZdt3b6AsIr4RhBkzL038satHkIuVtezpel7Z\\\\\\\/AUfzyUKZzytbmCFKnnWmKklGOYx6Y8QOKd31NUt0QZJUmuwomRsKEPhVaLnFYXsFgaODIvHrzcsGhkvtLZJH30nfkbq9aHXKbYMAL6Kdv5d2l\\\\\\\/2eV1gf87gMKxEfy8xw1pDtPHl\\\\\\\/FU0jVcUa5Z3mTAbPbL9bqylnvxzMlhnqwhNYnFVhefpx7+XblhH3zXGfk7dd5qsTfi3ItFnItu4bUOwVCzg04Sd\\\\\\\/7a3ZVAvx0gnw8sbTaP+d\\\\\\\/JamM4sz4zqXEg4dSHD3bHccbsgRBcvTQEiTGDotbg0qu7S1V6B2A7yOMrzBFJL0Fr42LnPw3pI6GVbMRzSt1+lJZaaUrfp9k2MLE7zgVB3CTPfFC05zRBRk3pdMM3VBTj4M7J41Zz5C+3nv85yZGYHS6CLjwpWsSVsMcFkZFa2Y3E4Zl0oqH0C85GKsP2lcBvLYMmweR75fszyaKhfFToj\\\\\\\/RJyuTi3YejsAKsJAkqnb3BIfZNNDGVB6ZY\\\\\\\/fpuhyVOwD+DESqVEEXmioKhAQtFs+j8bI5BuJUYkXMNRtaaK5ROkS6WPF7iKP6tJpp4Rg8VSbUMNyyqVV\\\\\\\/+E1RNEjQiFnQEDu2S8uOOR9kzjK2m+RPZgH108D3Rbds\\\\\\\/aAvfTSLxbNR\\\\\\\/dVafdHw88a8e7oShI\\\\\\\/B05r+o80D57napJHBmBpaiWnFU2x1s2\\\\\\\/gs0LxhVKshs6Q27J8nTiDNou702aXPvpEiwXqB3hVqDHKGsSCCcZF\\\\\\\/YV0qoE1uTw66Dvaat+cobpsittnLOSwKtjW9OQJ0Q30dPlMo6rZLQSL6C0bMV7bTQzYNFMBtpo7+jmGreXW6PzAVL27jZyHcdb65AXSO5QEmfjwCTLOTocF4ZOBJenUTczS\\\\\\\/gn1hOHmBNxgZu8kC7q8BGmsdZpssoN\\\\\\\/dOv\\\\\\\/Zo5DbFsxftZNWhVH\\\\\\\/VpbrfmnmJfK5DAxSvw2kM6m5LuRswN2TaTNykO2jBFSyTg\\\\\\\/trbiuB204LUJFvwyq\\\\\\\/OcwGouv7tqhTYG0blOQgZ0nPO0O\\\\\\\/BLg3M6TB7\\\\\\\/M95X7XzcsrGatTM6iioEm4bdA7EF7E7e9X5GMqeZ1pzfzKYNm4\\\\\\\/5drzO9vnUGz4o4bcX+KckP+ltMEwtA43MRjp2WtESABkA0hzPNeAKe0iaAFlC+q8uWLx0JC90llZ+fO1k0PQ\\\\\\\/nw4OxZtjMcHdW8db8c\\\\\\\/T1pFRIr9T4fUww+dumoPZ5fp4mclIMNPTjiRU0l5nvyd7El2AZzDBrVfeFmd4KJYHwj1JNwQ1CqhZZ3riuNkQqO7l\\\\\\\/ZL0QxJjSe6Ipwouu6i4O9E8SA5n8gcXvG4jWmXQSFD+W6cJ1YObCS1JGXNVpdVCET2bNoeXoWnZq5TaQus8W86vBoKCqlhrmUdZnoN8IbNBLujK9MKTP1IC8dLww6lHSU0XL9qElZnz1hoI33uECGJ3oXbwuNmdthjpL3j7Zb5R1S8dgy1slvvRCn8PSnZUVzpoUa0+IAmqCoKWlfQTlT7VCD7B68w13PaKz0U3jNzkuR0z4fhkuSrc414C9E+7Bzc6m3adYfzEXl6dsgS84zTpmZLhikyaUyjOIqp3clbYF+bFsoat9KdTuoXfv3bsj58NKnDJnXiwWi6\\\\\\\/mg9pGXyfbgU2OUjg62RYSaHJZH13E1jB1SwFquRBKc3JZ1N4lTO+GDm+Wbj9y80eiC4l0391Wa1fbPmYkobCiz0f6YGyHtsbRzXqXzV5nshbzc3q9h3VDdZ1nLOLkpIMS4HfqyzzWPIvRJixXUAjus8pZEdg39GO7pyHPHzj7AuQ+k5VMf6lU96KLFhYGz+f3T1P+I0Kw01GPkRmngRJxZilYCwcgGejAw3MEaSu1zFchsS0Pt0ts77tNUsLqKdH4dwcihZxb50dI+fDREdwpboZH5EfG2pZnYCTvVNmAwLWqRztV8y\\\\\\\/J55EeVATqjXg1f\\\\\\\/aOzadBurvrHwBHMeMwJqCmGtUvzsuJJ+y\\\\\\\/g+EznsWTmuxGrn1fO+pKXacnYjVTA04w80ZvH1NnKa4nz0vQXLstWjXgsFs0\\\\\\\/+a\\\\\\\/oiuTpMe0I3FCjRSP+NeyI376ZJ3oWihAgckn+5CvH0Y8DvQoU5yb36VbPcz3pCXcqcoz23O0Pj3XoHhHN90tZawFkOXmsSBv8x0CUi80+iOjKrraMB5AgyAAUXjkGWZ\\\\\\\/jb3xNUhoVgLQ9RA5BMOjzO8ElmrgF66U1JBZ4V3\\\\\\\/s9xbFsjaxd3+yoY1FpGtU9eqf+EzIaTNgrvvuTCNGSRdnY67mXAmIdYd0ACEeVvilhOQt48fZ3hte4tzSMo5Nc9NzMjBJ\\\\\\\/Mp+5anfjx69r\\\\\\\/p\\\\\\\/tR9xXEFOE7YeB9MNjto5uGWRRm6OgnHRvtK496fbTItNrziAyhivpl179zpBBupvoqOOlX4TZzjmHnoo9en6JCYqtoRwdiVi4HzY6CFJ33R2HokUhhqUiz9z9BIlDtVcOncSd9sURa8ydgSeGl3eXO+Ih3YCBV1JwLzJizOcgOCarY\\\\\\\/tYm9Nypz6yvKJyiTnhp13ULNZzb6MAd\\\\\\\/TO7VS3VmBHOdhJcclhN7li+9Sg3teRjjOMP6ifH0DypcAQHquHUydsbitx0S2Ae6PgvSDSR+b55JUBHpK0UvUUH8JLVpwqWqseKSytaMWeIhzDbUvkNEP95goMXwNSLvMOL8P\\\\\\\/D4RlpQ20KPJbqMC2i6K4SIyAFrnLleg1FLAYXmsp1yQGx40stF2yULRBBwxxmRDkYzhg3dz14mpkiiLOSSfg+IgiWO6hfp2nzYcQspXj\\\\\\\/UHko65w0X0NooI\\\\\\\/uiYK9sDEbncQmCmhYvKScm+iF\\\\\\\/5Ta7HgiGq+VF151ngqyeIK7BE6PHt0XM0qkJLjRpT4b1piTjNDnsHMfHH4KsHqdp2iMj\\\\\\\/\\\\\\\/VblQeE+0+w817OtIyHBDtmlL1LSGy67+cROmHdVIYKMs24MT\\\\\\\/9l9e+MhQ4peSjS7dxlLWirQGyAslTJyja\\\\\\\/iS\\\\\\\/UxaIMvBNLcOEbj00c7rnPjQCSlqmPqvWb92DAsFaKINT7qgLaDU2vFdchfZ+Zg7VTp9Wh5CgGkrU6gy+xjP5jahHMdzC+ChbKOvRFxroOcrwG4ZRdOi1\\\\\\\/8ciF50VMqNdk5u1PJHli6ETarg6WaraNkPmIxrgaFxW\\\\\\\/7p08GSWe5Xh4h8V4HAMXEgnbgcAODcAidr9MdlBQ9wfQo6iESgOA1AjMjLjY9hgOVP3owehkeHVqhPpVls6eUWF1h6LztBjNBW+vNQCwfRA5Hb5G5IrCOF3x1VxB8MxisE2F1WxiaVdO1yLj+3r3yphWdWXH9am50VdfgagfYwb9bLiRmfHibP2WkMya1PT+NnOntVjEF68DYT\\\\\\\/SseQ5ospP+DSNjNvBTiulxGWUTP5hoS5JkjHgC2HBMs58\\\\\\\/ieZsmLoV+1aDrtNRcYKP9rOZCf2uToF5d0dlfbrRJieReNOVb0JzkugiPj6EWXyHxhP0Bwrki1dwhVBK\\\\\\\/gM4yipntAte5YfoRkwapahEHGzD1Q\\\\\\\/t+sL7TMbXSdSvFEW70LsSRsa5MbXxP2H2q4BWKirdvfXwESLBbF+C7W\\\\\\\/wlFoC54u2dvLZ0QRFW0IO9ttuc3xchK+lquZmvHud9DoTH7rrdPbXIKNrqgYXfiz8r4P\\\\\\\/2hYILnoQEIMm4DKz+igPb6gwTySxx0mZfF8sdVANTvVJUIULSJr61oMQyUMPcPgJAAAKurmc1VW6zRYx7iGsut68l7NOh42xoTiR2Kzjs4rIGWSVN20ay9Ezdks153MCjLG4iYnLZvXKBTp0NAaUFz2SdTnTDFpy9oePIJZs+htYG1oqawDWEPrsrq8BWqj1CjXl3WSxBG+dJywkyQYg9V+661EWb2KdoiSUEQ4Z0ukyZCkV5DGv2W2RDaRl1LFjZ\\\\\\\/5jVpy14GdTbAfqsXStbJ7K\\\\\\\/5B+ylpHtFlqw1KFx73rrJ3oJPl5r+XEg5X8+h9E0f43Zopmtf\\\\\\\/8kg1vUO395MHQVnd2KnehGk86FYy8UT5JdjNHrYc5vTRfvGvMmULjBlBhhA19t283\\\\\\\/j8nj0pOkHZ2gCKQepvU46ZpCpdgbchicX4p3ohm9rMW13gDb8F2R8KD3zAsMg5YLGULuvi4PicT1qji9vJbXdlKD5hMYQCnBoryz3XOswjXjpbn\\\\\\\/vuvSqmsOy2kC9HyzxibFHLv5223otEjLCSeu53tlgX\\\\\\\/lgRedkj0fjhZyWpMPgTQH16hjDbTAEs0J7P9OaEy056epYDBr91G4oUPtO+jInW+eGRANtUtls1ZEtsDo4N26Okudw1Hpa1\\\\\\\/rUAZzJiZShzJDneka1BQ59rlPKxE0MOiuv8qJtkw2xNnopB83tnugjK8\\\\\\\/iLVY9SoPM\\\\\\\/Z+DmzDSxF+bY+fQmxxRZW+PltvSJ724TehTQ4iMlnd4fkTDJanmOHmiRlm7Oq0x1mVAPFs9S9KyN+NHZxUmAkRrizdJ3FSFzpX0POHLdDtowIy3FWy8CQZENLNb+Sbf6XfHubKQtkmLfREt1xx7OFpJ8YqQlrlsIHEuEr2D+bYBraGUtIGFd6Xecp3s2BhgunGBTb4QKvP4YQ9bqdUAdBIse\\\\\\\/6KykN6DombT15dlRlnmqkZMKHbHFFtoswmApkj+4gUuNbgbyPRS9o1B5WnZ3fh0e1rrOMoemIx78NJZQwS4R1FML40YdzJn8UQ\\\\\\\/MtCl03psMmNxtcydAf0go2SIQWn2PYW0sTnxuU8G7txDlGdD+qFG39NL8m6h\\\\\\\/CXeyashoZXoCyrb0E8q7iuiW0jfmNsIeeIEIuq\\\\\\\/Px7zYgU7DIfUm7C1LZMD9qD8hW7yMrXJ8UjF1DQrl4qYoQKw1C0Vy1SFcuedYzySmnMXTeYRkbNR8spxQtALawVHuytsD1kehONDZ9cVXArgxlFz\\\\\\\/7OmOV4l8qHZNyTTPgnH8zBHT3Jcu8h5HbNfetf0t6eUKNRxrIeinbNTWi+B1LLiqip+zPOBczsmupEIJm45CNEzqOHlkhjEQtXXh8u8kH4CXv2kxFRGSgosweSMr2bzAi5RCY5FXdAjm+D3+AW3Z+U97Yj1MVCGUTobWaWBrMbKOipXR\\\\\\\/hKjrYlLzN7rMcEqrwZs7ADxUVWFyyMh+Few+DVAKdd33amBcDRNWttf9so3UrqOkXQIbShjZziqrbBmrVHJWKFSK4KHMvwr2NCNEYjQXXgiANC\\\\\\\/00Q8qf2E0FTN2UHsJVv9F6Prqk7ExCobBg1CKjX4TsEDcdo4jcaGOC6xR3arJP59x2LRzfwcuMvTxdRJPyFpXyb4tALBVCM2rOHAwQbN\\\\\\\/qfRqeV+ysmdGnf2Vnz8WOiN0z8uwBcegaTDmw\\\\\\\/e4AeoNcLliScoBQ2KqAtLBfmEKtptKzTNFPxjGn2nLNsltTUU9jZRtyWN+zXgcYlljhMEw13yCunIZWKeABkq80cwDnU4sigTscy3vtLBX2tg4aeGnScyVQk4NQW5Uai5w8YEO4fYBa9WqN\\\\\\\/jxYRPtGrVoq6At64SRm2SyeDers\\\\\\\/87wby734rvmpWeWHhgHYQQIouZl3pkWCaHUYCc8hVDpNp+0y0+3ZMFoS+41JoryIUwtduwy5EQYU2HSomdtmTiuOzPQYVNk0IXe+4P0HbhPbnBveTNzGJWn2BqfS90EjQFiVN8h3ouH0Aj90KRVssI9WSoHN3dAVuzrUZTY1IPKY7X5a9vLCoUhBVMGvchyA8XyCOcyTvdcHdn2kq1sPok9oTeovSTfh8WJikx43js1tXFUv2v5R+lT7jyzsw\\\\\\\/7COZz9H8t7xOlsCUJApV3ZTMC+Sw+nX0YDCBIHIZw0+5JDMKovgTpnMKxTLSwYcUGQzXoU363PU2Lzk\\\\\\\/mKIWOjH+YFyPp60RCnTQlfmBXcALOChjT3ZubtZHlVdthh+yyGzGKv1frWWmT4Y0pzH+lKX0vJNXBHUBR\\\\\\\/sI0C6RtQ7DzGEJyPPHtEYYpmFWmCO3Wd8wsmhUU5tmzVcg98o9jlZyQ6629vhw1ngANg0xgMiZnzc2AEMjfXx6GgOka8\\\\\\\/ww2p7MsOODak4Xj3QIdDBEviy08tddvDPnOFw\\\\\\\/qHkaTON+oOwKvc3VYTdf0M63m+Ntl5vt8qhbqDSjnlIka\\\\\\\/jVXLn67EP6j8WmLHE390WAw96SSqWU2jxZ2SwKHw9AGweCOuvTE8wrLJYNU9MAFdFvcelZPDYDVNgxpFDq6ciNoVl2V9THaQhbNUr4nSpAZ+aEftU8yWtlaCbQ4GgT+dTk\\\\\\\/LDy0U4j40OIjfzMdEYglYvYf5ENmpnibkuqRY16E+sQj\\\\\\\/DsR5E3iHb4MQYlT\\\\\\\/Dc1VQfMbwKX5D7Bt8qOgIda4UjZGc8qfgLZ7bUrT0n98ZZcwao1WN+FVfN9HX5fTqEFAGEgt59gJR+3cbevx69zaUlRx+2vhODDx0JI4O1X4TBuqO1pVfSoJKjAFAi6EifzPnTGxJAUNKWCEZy4vUGJUPGPq0vzntIcZ0G7d0MyaUcCVZHsPerdsw1YRxZH5wIbXgNcefQNLOaJNOgPuAtUiCoqeR000+eNMpRCna+e\\\\\\\/sG2fdcI2kUTia6omPaEcDdkrNwcARFL545QUL2ZP8RK0xYuakShtHmUwb9JR+141SncTYDFrW14Qndl+GL236VZgkXM7iRg2ZCQjmvdoiD5xxaYAIuTDnTV13a9yWKhXKCdnAcwZxSweB6h+ibQNanVe\\\\\\\/hjUiQ0iGbsjosbZiPYBvu79opHzb47slve6LHupQCmYCodn63kCM7\\\\\\\/lTgI0L3iBJU+O5q9L0Fj0Oosf6JpQ5kQmppiDF9h6MXfXnhO+flTrp0YUHTzOB9ArYQxuijiH\\\\\\\/zm+ET80fVfxW5ayFZIJelKHy2Otv\\\\\\\/2+C2fBadJZU4wDvvskGWmGia2KtuEPBWm2DV8Mnmzh4GtK4pd7Me8aMl\\\\\\\/pjGMyLWEsvGkr1vAUGURiE+2udKCONa9LgR1ilw1OdLrL9WsGb31uLSW8TUfnIeBqr8c6DOfhRd5FjgSNKqqK6yCh\\\\\\\/Qar6bBs04yihiUN1XL1e8kmJjsyA6\\\\\\\/WL36hvwg2aaOvnbYY1mWmrsE9UZYuMLiYOaOpLON48Eg0Vxzi3rWK9Qz79FFkQO3h1s5O9M3qJUD8dsT3VatuJZrOTHz4GbRtgI2CwRl60cre6YBiJjOwPVx1Jq8RwvoXH0sbQGzADDCBSWSLgc7DH8SFMjF0BorwREGpA20JeASdATn1AwV9MuIMnz3CZGBl9myfiv7C1cUnp0WVX9C\\\\\\\/YH6prq+FjUmguzp2Qu1s16+GkyFPIlzku81Xzz2Gf6V48cQJl3Miqg063JiDeARolB594lLq+PlQDMwkpQwr1Ht8eLw\\\\\\\/w3noWxHEfV6f1YL1\\\\\\\/52Jw2Xa9W+iUtNkdEeWrTUu6b27uq250\\\\\\\/VbiYILNl9ZS1jYwEJNOjRk\\\\\\\/Jbxqsn3jVVyjU+qHlQHhqPZp4rH2\\\\\\\/aQokO\\\\\\\/V1Q3fh\\\\\\\/tJ36TZr3rjrWbcJoH+KnMrOmf15tbjmpZAR5r9zW0qrwiVLBwZcw6EyAeKvaCJ76bRdnWY75eZPxwH8fLj94BRZbWE79tkIv106CNtoQBhtQtxTsvsP2ADjWJ+8m\\\\\\\/0z+bg1vo+ZxoXmEgkf6385mCQZGP6ax5piR3VEYaU7pbA+\\\\\\\/k1dkNcPQJZhmRFeMP7Dw7CC+VExNpaLxuWMoABCZU5N5kb+1xnBPEa9IceJNBXrziqfQBH5HjzEG8h5Rm6ilmIMXilkyQEUVuAUtlU2Q1ckHI8vg+G5GZFUkrR9FdrFIdLQZSM57NHugUamV4+pzOqoaW1nvGRxdG6vITONHOACpR84VHD6aykGFNZiy6QU52UdVUC5fGYtZSOunUbaUyi94lt+JJrb2\\\\\\\/pgLpsonicQ3sOrinFjnjLczZzmb2KNlaV5O5ztlTogazeUkaSHtp9dabqhLgwlyccQoYw3pGT7srIavMjADqDKkaENnjfSzw9c3cebnfuWxQARhQybC1qvi0oWxHHANz7fSaCzvFK86n+6qq0UUvgseIk0k06Zosjf1VNEmFzKAXvSua3Onv2gW2jPJUp8t3A45eDyQz6gkcdctMF1XxTpySSfK8MHK6o39IPnRElGWRP\\\\\\\/jnaNEfPSrZ2O4VkstXDZnY9drVbT6jOBDnvrc0ADcicQt6niQsn+NvMh+8JkruMaD+HQ+2685lbMQaQEdnFu4Xvl8AnpZm3bBkW4cS7Smu8nv+VnnH1rv9tJXFXcxMghzNohj2TaUcDkPtCvWoQ2LxCShqQHx5tqbWbiXEQfYTWNfnFjtsn7h4D2vYxSbMcmwd2X9SPfa5ALzdobiGbP4zcY7hREwR77WaIu9ZWtotAXo\\\\\\\/9UlRoxi\\\\\\\/II5u3tZoryp9i1K8FMZlrZGp+epWTiLe8Azy5lw1RJf3kqVDkrEku7Ss9tUgPj+OlymgoxlBEHCMYPArKSdJHc8lU7gQxuWG1fpjd6lfl9ET5cb+Y36jWNqEig+jxT2tSZCk2Hlql0F2qktGGDH5K8yt7RpOIRqnmRSgrT4NPLRooPW2KXdEqZrzPuL+Kg+ngRXC7d5wQ28BOrg45hsE2lK1oLl36neJ1Dx9G2QLX0AmA7s\\\\\\\/kV7hj78c\\\\\\\/ASZy7b5B3W+h0LBmEVy2pB1GVL2kf9qh1kfYxJQc3fGOnBwk0ywF9hdUgpTDIkNfSWishZ3CFUFK5lVaT5RTFyVZdzfNOcnjyWJ\\\\\\\/BEyrBffefX7aGApRqBCGkYlEaRd1VoNDdWkiQSls2gvRyh5QwIxE34DAizlc0oqA1C1yNaY9wbPPvada3oh\\\\\\\/+9F2tsp0s+qJZ7Fs0Me24b\\\\\\\/TnTrejzbUxWXapSavPduVjWHDbK8qxz\\\\\\\/vhTMwZscshEo\\\\\\\/1V1Wd95dDGCTlbENye9I+q9KakpieqdTKt+ci23jZLRWLzVjJ1kBd6p\\\\\\\/1RYRN7a0zKTLVth\\\\\\\/l85bqZa+mY8TtbwxniEmZT6WFGzTt4gY4pqA\\\\\\\/L8Py5dogVXEcyESXbGX0KB4XTbwtCy\\\\\\\/nWvzbpbbYmHnqTsotGrqKoaR+6TSIIGdvvQcniZ3QsiH9J8i2WfOudk4keOUTkYErYuga1HsbfAZmJgEUkKWxgMHWjxo3OInUUvkgejy8xHtSVpGU+LcPmLGL\\\\\\\/8H5YoW21\\\\\\\/XeduyIFYDqG3WrB5CNXmH14la2uhUygFNMsed1e8CuCr0tnerrh6PkQhWE1+RdGSAgDtFy+sOq7uDjexMtE83JQaq\\\\\\\/KLIus8GAEguNPqv6JJJL9sBECU97BYHv8JUN83Uy9\\\\\\\/N6VoZW2jkS64OCTHxITrC8ptQXB3nBws88lMIRI4nedgwkXbvWxMilk+lrMe+H+drQAlSYsoZ0ff8okjp6WYCboXM4GzPhU3vPT4CqGiP868a5zFEyEQGq7P5AFhQ4vIf+S7JvhjpGZsSB0C1OyE4EjgQR6IJ0Bma2CAQHQqUKIqlvH2nkeueRy4WCGjw7Mxxjs+JTneFyw5\\\\\\\/T1bb6m\\\\\\\/aacTq9SQPtGScDP6zxhocE2pdG\\\\\\\/uBaatUgFmxUbrN+uJw+7m6oO67EXiV1rea3HEtcSraYdjegF8T\\\\\\\/TGN83\\\\\\\/cLIoqGOl8j9rvSyOK3QmED4MgM9rJsEu1Qt7cbyF5oP2Ss9Ev6rfHeCCXx8wPMSk1VTQPtNScb45aL+m5Dy7CzwymwQ4WxM8ZL8BcawJeW+K+\\\\\\\/JgbmoPno5djbHNVrOTxsIvyI34SH4iBZQcc6B277ratVdLNHHtdU52Oq+eRKkrFAFOyjmmbWmWJd0\\\\\\\/VZAdsyS03nSMiOB2vD+RAiimrHJebBjpdG4HThmVGAPxmT+465v5RaZG1U9YJcv+vMoY9ct+K7wwHcpfwHaMHBGGbUUWCHB+4bYdVoA5OAF6fl4YNVY1PmOkbcGyI3jabFVh1fWGVN2hLvEh9XP6Cr1\\\\\\\/yb+NFvmF0MuCWhZL4eS\\\\\\\/gVDa7FFwDX8Mq2+guQ1h\\\\\\\/MN7A\\\\\\\/UvgL2bQZnxGi8xw6XtSTqXXajLOhPfIRdLczAaf5YF9Lx4BKUs8mDxGa4y0250DhVZZ3h+YKGSpQxb5PTi1r6n8M1TP6mTnICKYBxAZmjIFqpSPSCkKcFwicGBIJuQR5r5zxLdzF6yO23GVA8ZaG5uP3pvgnFW47RFUmwei1Wc3srcH4odvBxZlPt44fKNxDflRJeiqzSUk3vHEHohU2wS\\\\\\\/wBr0M4gKPaQxb6mvXujziB9ZyAd0VO7l4smDah6oUVMNPa5flTBQkvzaaLazcoAEn1\\\\\\\/2pPa4uztC3t2EywvC\\\\\\\/uwWgNBsxLsmALtIYJ6neD2cthkuSyZ2t9tV5yzIyzTf6xoQ7SS5iN2Oa8am2Dq4V98xZIJ2XNJkHFiN7DRfYJy0Wtds5hDjNhyNJfcmhd8sG+aGNbRqIXxmdur\\\\\\\/iSRaIgos74wjMdlM5Kvy5sFN0O17+fVp7ytfCY1GlMxyzVcccM6o0NFHsvd7MgGarwpDgG2H4QlTPPGWyO05mcPFF1mP5PUmquAiW\\\\\\\/6\\\\\\\/viu+0LsBoKh17s\\\\\\\/UDOyCzMU5\\\\\\\/FKzdv\\\\\\\/kdeJIKlh\\\\\\\/JlL0JjmBdv9AY2lEYxTrPIv+\\\\\\\/n4N3W1iSy6z4M\\\\\\\/KjECU\\\\\\\/GWHK5gzdObWQMfSA2ZoTi4FRUb1C6tOp9ExDl4yD8ynbWKwH8qfThIMmje0X1PPdGAdR6OjyAik1F6G+i2UYTiwFuBzpqWewN3h97GHUv04+Wy23VxKNmcgyfRV137nIM72U22DRSh401V+B0LeWYzkTHHDZ8JxqWaUPOAQB8JTaHLWD4Ybs6yHpa61I\\\\\\\/LunqgLmeKdGt+rQO73IaifkY0XJ+WwBsB0rX6BcKJP9S5ymnJHCYk7ex\\\\\\\/SRUoTkmxiUvgq9WRMl1wk2PraNdRfGzeKlE51ZGSmpYFAIV9B4elPdBfalQyuZn2maq\\\\\\\/kUZsr7SKlhdr2bfS3y5LvcP2fU150onLZpoJnOmZAiXa99XHedAKhhZMchdn5DV5ha86pM1qzMfLrlL\\\\\\\/1aHsTsg6aWtzls8qUKOYCVNR7E5SBbtWOcV+f8dvqkRhlYO2wsWCpI+kaZJ1vhXz+L2ulp\\\\\\\/R6JGsasjHMACVuVcini8xsT7AjlPSKBdwb+b7RkLs2ubK+uZgnxQYn2jbtf88GyKPcwNeSXAsbHUsR2EsA5L8zsQgdgemOr+z7OgJJD0XAejznyptj\\\\\\\/Qqz9W4+ZZ1tbkZmYqJ0b7b0xuwos6MHna58L9nYsoS0wnvb8JYdqTipxDP9D4\\\\\\\/ajoERUFu05MAsnqfIk2ZbRYmBaaHNmysYjYPSZXiFtmPBAIgm6aBUxOqhc62MD0z7nDsaBDrw39KQ3lyBJX1KZFC8y6pjx3WJrfJIdDKz1iahXuBneZdIpu5NFGXWtj\\\\\\\/T3+CAi70mZDdOwLqiZ6N+odht64lZ63f6n6pccEj0S3BFfkssZdJ53ni989MhsUOVxjRD2o0i7jD6oyRuwgolDXP4oaonYxMQ5vya3Fj3+EEsfLexnZSCTeVmi3vNxAqdaIchUZcITHiRqXgWS7xNvNp8GDXsxaKgW1xh6ehVrpLDls5HHERHIW+o10OJ6nrlrd5zWiZ6DkRiXzhauxnl1w1Kb3nWim3TyC38qlPV+c0KzFrE3HxrFz3NbyU4\\\\\\\/Z78oa+2K8\\\\\\\/LhvFUyXozQ0yx+Et1\\\\\\\/muRkn0wDonU9xo3+6S5i8WXv4UTT+LU0G50wD4uOqA4kg2lrbAlY\\\\\\\/a5rl16zMzWuHCCFTCi5NnWaSxHJqZIV\\\\\\\/QWiLdG09pY6vXJt+3JVvQsYc5ox\\\\\\\/O5gUVcjAcgPWvbwyu1S\\\\\\\/kAbDHZ0LpR0d2XD7\\\\\\\/jIJsCjichVvaQxr6vdaLHKLKYhlHa+XK4jf+jV7EChcIaOMPKsLnE5G7Ks3K03p8koI\\\\\\\/AlJG+il8dowWkoB16C7ioxL7cfZ6jOXkt\\\\\\\/uY2fgSxVY0Nd4+VW\\\\\\\/RNqcDcj3IO7JAr+Ga9fIy6EIAZOYsR1ckulpJ6YldfAAm1HD1kpJwCY5rAQQdelKJki7\\\\\\\/rseuZP8OPTLUd4ORUHuaFQ3y0LPAuS1dUGMNnqUc5ad0bEO6buchzSiAu+FObNmbizjHPz6EBX7DfDwUdk3gpIs2WqOBMQLDGjoQIpabQDE9UDLjWdaC1zHAtY66LN\\\\\\\/AL\\\\\\\/GzWI4Q1\\\\\\\/iBMePPZoi94FDh4kUMfyHY6NDRLtbCvjHM5JDDGabl9UjaV0cyJGUCR26vOxlpgAe6Cq72t7s4R6nJ3Tk9Q10JWuuA6y4X5q2pTI7ZYmGAivr3pbFADPgIxp\\\\\\\/0gDMKQQ5IdIbdqhRXHAa7HvpCjo+dY4I8T810vOmFBqhG1VfQw5gejZoEiNCr9RaxFM1S4wcaD9fyZd4yGwsKmrh67d5A9+pc3NX5\\\\\\\/Jg\\\\\\\/r6lIoIYt0mRffS1A7\\\\\\\/g6lnpEXJVmMKSGu9VTliasw5AhZEZtNtbYyeRAMbbTLbQ\\\\\\\/pE2ToObJ7M3yWFXDpmcSA3jqQSJE6EVm6u5oppEk0mjuKuWtRovBJh9ge+qNDA8nX1gGVDAiip\\\\\\\/Osf\\\\\\\/dJ0LmkdKGKGyXjYtvq1hbBKGzHCFOu3K4Y6NcCWbHRiPIx9NhrTOmfUk2VD+SoLijrr+0UGMuV\\\\\\\/KNN5dfQSX2S+mOulyFyO5umpFj0hwbeT+PODoxDj0r8JT6dDBU7v+0PQ3tL9S5ycFkgJSSl2zG1WaBlUr7gz0A7MwjTNNYwmIrrGTvwcX1wCko4638Mnhh+EcvKMe0zgNE+\\\\\\\/1SkS\\\\\\\/pe+evmvyjvuYstwQENcSmCF4sfy9BUHNms7Pa\\\\\\\/DMNqvGxktgKYUajk2HswLxlYzhu1WakSlp0DcD85oXt6jZjpN2iTOGGqA9E9IYQ4cZSEGwkfHyjEJa9NOPOnwSptMo5dpcfTG\\\\\\\/Sh4rQbST0ZamHE8V3NBWKoT3vNPHiKwe7AZMouoS6eatHghhS381qp54Zs8x8rcbSqkHJo3meWAQX5vv\\\\\\\/d9W9CYkpCWM0GKZA2w3IIOwR+oqnOjblgtmmpAvnW6WrgnfaR8G5tF8t8+z4Tr0WWoUBm2lWgVE+IqhjQrF19Q1L557i9mqDHn65KJAp4BU+LMblR4JU1aSyO0rMhAMLoIDkROxRkJ0YkOMDyAnoVYdHgLsnTm1o1Lh3yJIpxpUkVfhFbKNklPL4iwFapaUTrUwVgAfLZ\\\\\\\/tC9Pf3pDlk7r8HiJfBEpSSFcU5rrVIYnZJvq7+O9bK773m6UeZszoELiJ7OHWHZudk4RDMSYoSpeC9kJEz6G2uYZ4ClTB4uslLui2CYDPB9+Xe68NC+xHqHpCSgFeLzDNcbyM5lg0BNDEh\\\\\\\/o0BvQCVd6xFSbjyqCQ7vsJYQ7vJc3k0FwevjVI0RFXcQyQ8cVh3OSFqQny0wxX9PX8sI78\\\\\\\/w+5aGfy30MdOrwBbWpCHoG7Xb44igWkT5UPWcFqPauTRLxAbnzRp6UVaTShcCjRgEOYaNYe8brwHJvGkoJScvvKCt7Ck+t8gX\\\\\\\/O3LIa4Ktc9nLZlkSy2ilngnOZTU1PrFUIN4+R8suKmC5Nie7XLTZJP79hvRTty8G3VM4RzyArsxbspZmIIJLwi\\\\\\\/1yR7nadaKXJnTqAa7jP86guT14lS59AC6sFYKeAia6dr0bA3k82wwkc\\\\\\\/oIfqXnLdZgVGAen1zq4hqpU\\\\\\\/gCVj9VIQ1lzN43WmAEqgoF5DgJwlJKpLe9OfRh7ZoYv+7TG1DQDidBFmpCfd3lKufCtFDEdCvjtoudvaf5jY0ujNVcYb+I3UcvyeA\\\\\\\/lKzEW66kmRZQIpVArDKS8m5xR90ARGskI3cFE0THps3yYncJ5S\\\\\\\/X1vBfJNJDBjlkXdvm66I88+xVOa2giHqpvmMBeOJ03UjK7qLE2ru4+nMVKvCQCH+nTnekyYJwv38wtKmL5tm5vYb1uBOrm1yIapVF+EtML54FphFWxSZXKAczfcibayswKQAFJvuudNVa7yhhbCwXfn2l+PeJgwggC7gXCb0Hdo1LCnb1oE40SgpFGO\\\\\\\/pbLG1MmZGhy3e\\\\\\\/Mg0hgNiKNy4GeQNiE6e2Dtn5oP9cL4LrrrQ1DJUry5AbtEBbIuH4JF6ndccWaZsJuhAUPgpAfkur073z7TD+Hekrzg5wlUsMCgMXU7AksXkm+w5H0Qra\\\\\\\/RkXOzZaaX5UiYTtlxBYqzzDwSiqeWeN77QmN6buBFWgpheM1SnHKNAH8eTCPfbz5PvW6dQAnthV6J4wT3JhOiaaMbX13YSc6Nv3o8BROOqS5lY\\\\\\\/J8AT2AuJVo67ZCnY901FF8BPyA2z9rtaCbitpkXLlRW7rOjoYjhf7m+\\\\\\\/FoKL6+jVo4eSd++7yDsZXKnruV+p2Us35FmawAstl2u+LykIwovRYeTkb9dWYI8jm6CZ8wtK0iyOwwKsKZF9qQ0QHrbkPCKLKWbPEAoP8jpkEGXY23KObnioh9tFCmK8K7VQynha9Aomk83RrJKI+iXO8QIm1dn1n8sU9BLFGvefLhOKXHSScCdaWiPdQzgzeFJM2nyLlI1NHTjYWP\\\\\\\/erllbsK7ZgWJxgZjGGuC1UwJ1SFL7yzxa6as3IrchR+vBGZDd4ys8srsV44ZXtYXZ35SBlkjCP7cKEYbBLhTw9ZX6Za4hbAh0w29XYrYo4x2Q3sBT3LW2WYMW5KuJxPCvvDMhd5r1cuJQoqiN\\\\\\\/BBwccmMv4UU4HtyEo9ir3YVnGdxT25nwEEEQuY9y9JX9D21uBzCBlnTjN6cfRiMs9h30ogpEuhpZHaaJZVQvfjcaV6iVAigD7hPOnM0swbI9ti5Pvn3251YGZmWM6bJBYcU\\\\\\\/Fjj6Fjqbss\\\\\\\/NIAUE5O8A2HSGotAg8LYWGNkvJAcqLDvEfMqRvGg4C64viYwFpdT2ubk6EIlcY7N32o44sMb31bjHiwxaRWzjuiVg9WaW2oUfw8wWbdfmJVnipfAoZ3A9lYg3ACo7FCahdCUCbGcBFnJjHDPwIY3LeprTBXAsA662T\\\\\\\/C8W0p14Q8dVvT7cHapHlCfnP4vulCGA9lVPJEMVCHmmFrclOzkKKj+aYfQPUBOfbk65P8alVjqvruhfTIGhkOeSixyFDsGTNMJ+q+gJmRTmLbEV20paaLn2bX3UAhDyCFJS6RhRnGRO2Bv7gzYG51H+SAKn9GjMiWB6UqCtlt4jXa9xpNX+OCGZMmyizXswkZj1h3l9jkUmAANhATahcgLPRJh70Ik6T5IZvRL5ZsswzF2NM93+Adn+urYhayK+DEbe1NpFB7J3qP9ttzkkhl4C2Z2LBibiptjVcmowNA7x4wMqXBc8ennb+0a\\\\\\\/wFs9H4kAdECZWvBz6KLD7qXaS5FWRh7x7pOcDaWQemslYY7OAtI69AhHG31ybXiAImJmZLCNMSg3jyT9r+6bTbWFOpMhxqNoSJjGpa8TuNUQvnmDQAZvRWbjaUCgox0mDuKA9rIUvvl+IwtHoomFcGninQfvUr1AnTJ18RaE+ywJRA2PKtL2nnLTR5CnZPAhV0PLSwxN6S8heuLrt8IVWd3dOspR4KYJG94TJ1pi4Io+lbtxvPqeRG8xX9bWkc1OBEx4NIyvLVmx8M2ZX25VDjVcxdNuCYYfICQaWpHcl09ewW1cTHJ32MIL29whsRWBT11GMTP3C\\\\\\\/ftlnjsOacRVs3PXkYnXVwfgdk5jvBiZDoFZRUP+gYbdcWu0yqq7CmyYI3yJoNy+Uz6KMyEUaxAJ8y8KjM1UQ18sscjPD1Ypm0u6jjeru4SnjbbUJ40az7MllU7nCCixpbFMvFP44iTg8BMuan3ES7\\\\\\\/FEqkUhCgFgbKl83FmkBRTWLJgcwVVPZztK8\\\\\\\/PBT6uRHav98c4xmHHHhgn7y1yYOFZCyR7G68AfcOVeXijP78vKIzfDkeJRFWnG7Ok9GhVH2VeK89oEVJsfhzxKFZHQpmCJLiRqusNhKcKPq0JUOKdPI+2Ic2151ZHCVl2LrAor29jyy2v3elTHfq\\\\\\\/+3O6GGOT+spi\\\\\\\/5WVccN7OqSsls3sD4IPA3dkGYiJWONZX9V2b\\\\\\\/+zyaCS5xw9Aw5Ft4tOQN\\\\\\\/Yc2pRtevZXIhEfY2HJ\\\\\\\/sBdMjVy9vOYNM4BVwdV7gv855Pc9MIAvfHzHUjKXB\\\\\\\/7l3YIx3NQtrPsYG67ZPrlK\\\\\\\/4TpAdr9fHV5v33uxk6lbO\\\\\\\/HTgNNgfdoIR93pNssw4gJDe1wOFvUeM+fQ4bkvgI65mc++AddIVUMgBVdfDVG2Q7htbp26zq+ketqzGUaCXhrhoaK42FGo67o9TAGGdw+NGuj3YWARXU61A0VvcZWCxSO96z6pLfTH2EK24R4l0fVGGG46stfJ+U32qtLtLn6ZpXR4gP5qfLNf2LzIEm6SMyqu+Doffut63dRnL7hEDk9xWXAeCZc2pNuQqbHjo8L9xrMxARgBC9F1phoNJsKBUrsgG4To4LAU93np5ufzuratdCVbOFx58CDUl2Nb7YObORTXUPzn5fIG4nIIEmdzigNTOo5PEeQp\\\\\\\/L5KoLRmsQp5W\\\\\\\/Jxz+gunpdhXaE1MXWnoYF2vZO9PXcIIt+hZgn1QvBVQPowPncCmz\\\\\\\/0xKRbW5hqhzQCARVazdSoypXT0exbX79azh5+NqqTUB6xD5hB5F1Lel27YgOKNOrP2csP0ItTCIqta9NWExxKaRgWsBaSZPYdvFpLyAt3A+tMzV8FfcOkjLGmPG0a1SI4xlaFIJjRexHIjW2WF9xrHNtBcvANongYHvkstYmhGDJwnXNtNMogQWR46uYGAnZCjj5bLgYLUSJOUcvtg3Awk043+VAkg869I5g8TTh1u4vuvvE\\\\\\\/BQPiecWvt4jduHvyDcfE7+V2qJ3o6EbGB6DuWjVBk9obFuAr7siORPLU5t49K97rKx7bCiJH\\\\\\\/pN7wYIoszwPi1QbUPVMsMKMuqZJGLiA3850W90ea8+zk3Xj6lYXBPTQt6+mW02T8uav5743ievlxAVp062xtKdqXLj3knjUTPnZ8\\\\\\\/LY1x4CfxEfVMgu85hzgqajndoQx7h1\\\\\\\/wOXtMcZKx4AuzOFm3eNGu7R+Ei89GZ4WVboiAC3OLoRZutnTyHHkOSOeEERtsDADqCw2BRqtuJMjrbnzqZF7JZDoAcHeD0DJl\\\\\\\/eQlk1WZB49odjdPBaTZOfftQJ+OyGHpkDUXlEt1Px7bwWuxoISHQNARDn7XSFqvgjrZfZUzpjjz5CsYQ6MCwQzWVWFyDE1qaCDSeaKE7854E0NhPsMIcNWN5kgLyuOubs9nYl8lxqXP8M46IRwfGDs+h9S0HG1fqydf\\\\\\\/c0gSHtYamdp+\\\\\\\/SaRkV6sQLHAPgcAmNVafADWz0FgRZ+bSGVTgegMbxbzzjwPucYD7Fk7v49RazNZRd6XXXCdub063grBtQBIS5vaangaIVfsT5WJqKrtrWwel8Ty6HBD8zJjvquxdlUPVfGwWR+2DiN5YGl48njIL33VvGIhgZU4kL7x9TjGCTd5Miv1nugqyY56Q2SMIh9n+1QFjAGY1\\\\\\\/Uer3bqW6P6FpveQ\\\\\\\/6fxc5bzJM9pjS7vBEK5VdSC+LsdvvKSOgRutb1kjr3Jl+ff+OwC9s5fk9zDhUESYUSXocd9adlD4pRLPVy9thvkXEWSvjlAomjpR6y1RAeIDoxYVKvwhsKrSSdc3GBI22Jl32SgYUeDxx6Wfr0cQrx8thdre35yh8W7ZcpYQJw3agBYr8QOfufv8diLDpl\\\\\\\/6ut92IcmGz8tnkqtOVa5jsWifUxxUi5qRQawNdhrRbZkK+fuZvs5ZvOx8OLieMt77tWXr3Koxippey85wt4riW++Ww7XXQOkOrsmm+2zlI8rh7jNJjehvAkoghkqZ+XK8yc\\\\\\\/P+Qnkx5JKhhN+rgORryrPbeLRSS31hi1xEvSh\\\\\\\/\\\\\\\/XbgZ1BXeTbhHg41Nn\\\\\\\/M9KkXFCcy4i41KSpJEhH5wBNpgqVX460cUFIldC1+7c3ehRP7B0a+fkGvGfCImk3WbR9OzNr8sTCiht2B1dI0alNCgK0\\\\\\\/akd8EjbJzq\\\\\\\/5dI9ZihmtR6pXDUGhNuhn+SsZt92YtCI9rYA+lzHZ6Vsv0yxmiX+87ZciycFrACqYj9OAiyu0wwYni8U4z2izsFN+6KZ0hm0RHiPi7bc7otjl58yGUWjC2bD\\\\\\\/5sNognqG3GWdoV4G4yL2KQrsUU7fJw1LY2poUpaHekv+H3MRP12Qa0E37Z1EwVVaHVQWK320W+uzdY3QKevbP1aaxFGEVvd5ZPvBgBXTQqDqfXuy36l0O0eJU+wV3PKhrdCZ2sx\\\\\\\/7qDShFIFgWu4pNkIP1031ef5zxwPnB2kGDT2UPWT6SWN8q9lQyFWcafezyvcEU8uWUXgtiOVq6Y4tc0KhdFQRCrb8bCMZ+TLkkXEEcHKUmO4FnBVmzBD\\\\\\\/ygOeUuoDGL5fnwgZZFpqXx9uOCAGBrub1hJRdmbqCGG9DC62sWqsQmkQ72rIHMN5bfQ05kZTc7oatRjE9\\\\\\\/m1mjhQRj39ju1JVktZ51a8GpBJRNYgW2yQL41AyUMHtHshHKnMTedcxWpAsPDHcA7\\\\\\\/6bc2n8CQegS+PKmPLHAC7wEUxikyem3O+ICBppNwuRVIECpOeKn7HNgOLe8\\\\\\\/7DVicEFCzW1tJuszu7Xh\\\\\\\/BNnHxGPv1\\\\\\\/aH0MJx5PXsworJ3GeXRVuDd6potBALAytFAfWE8snEG93m0QQqHZf3RXH\\\\\\\/RnmRZkDp4juRAhMNiUBUzE9gJr\\\\\\\/dvMgkpn3vTBsbvC+p9BihfVzJ06yM9dzuqiFTAGAndGXI6W2j8VckuGmamD+PdGz+1scdrj61\\\\\\\/+603KkpRlDr4JAGTQNooGml31g2Yl5syvZSDwGhTNqaGVglGinVNd\\\\\\\/5GiSnyX46fYXgzr07DvUjD7iSl7ckj40ZfMo\\\\\\\/F3Edk4qInoDGH78NP0+LHtD3+4AtDAUPeHa224kL4mPAzegpDn8GriAh4GiOP4cJbMF57UG34o5b\\\\\\\/AU99ZQs73J4Blz1wBjoryo2ZpMOdLcrRYKsHWosyMhYxjRUgOqnZgtX3BNd6dVcNR2typA25SYVKeY3BEhWxJaGZEcoONnBxMOA6l+GKOfF452++t4z9Yxo1g9CD3TNiWfG\\\\\\\/4ZiPTdkKMvCTNP4Plx36GnIjq0VEsIbZPZsKhFwVDVQ8dMig8Gl8dSos+V0JyKpOGNcYvmZyLEmMnfql0UYU44Tv04uLIz+yGZhVhq96qlNXyogQfdiYEWbhl3tNMwG3DI6DKuMH4LRyM5SXDgdEoJOcKhjzAOpoJvaKrZYxpNbk\\\\\\\/9QMzn80YOa5vjDQgbF70y4G02fL4yinWSD1PoxQRtfd\\\\\\\/w1kbGkHdqPXSoh2QACCPNQsXinm2SJiy+WhqcR88D6lxq7aVleTXNQIApnKzX9qDk0SHDO4ntMjznFK574AMjgAjT8vDP67pJD3jz2nMGRPS8sgLbAcgOb8FLo6ViKQFzPqHhma2qANYTptTyWImsN4PLSTDk\\\\\\\/V1JOKNJm2LyC6meuBoSVbX5PlxPHNZQJ9Ktjuid5\\\\\\\/2NM\\\\\\\/ke\\\\\\\/+\\\\\\\/LNiZZYToTIWmT9f7byzsxcw5YIFi2sg6SQiZ38m708VFxSjOBzoXeiocf4LW2vcDJwcmXb0AVqy9z21o8iWtVHmdySkpWiNgm+nnDRau\\\\\\\/WaDi\\\\\\\/3z427+jj+Yx+yWYKhp1unzh7P\\\\\\\/Su3aUXb6SrTkiaPrCKlZG\\\\\\\/\\\\\\\/GpNtMTWRNcVqGtl\\\\\\\/uzLf2hUXcLKUitidlzTrgefvxZJqzLDo7eA6C3QNxrRTKnuw4cMGwpc+HDrbYWSLeYzbpqnCjhbJ1led+DBu7MWgcGz5Zzm\\\\\\\/fAkIw1c9TPBBqkwkE1S5zxi3+6kqgsBHj44FP6vI1A0eRVrSWpDYY5q9OBBPjFzJSSBMDGyUmTVuny3355tPM4lXRRGXO861Vrh5WeNNvMR08t1wwBAjA5XsmGDZj8VZNa7E5cd8sHl6qNi71uXfSOFJxwO1hWU0IatJ5ihzBBehMI5arOSttzK8u7daeudUIccyGeMacQCLbbVKBjUcv0LRlVIEtDNe8NMlwxjboJHLF0oELELmxQlNbj3fAqVgPULsF9lub3Mv+16TITRWC29BSUg3o9u+dungn0OGxpwgSKXElWNWY7bnBMXEk5436BOWF2CWsjbIGy3fECctP5semFSKfeGivGLOOVgbIA3tXgJty0NNjtp3HPokx8u86b\\\\\\\/AnjoHf93HEgdYnnQEb7YcdZdUtocmGqiOii6nVp01gFgDTDedpG7vmnuZ0rcwJZoKcpHAbumJt3BVkI+LOmEeuBqFOutiK+MMdxqK37mTR09g0miq1Uz\\\\\\\/QYguHGg7bI6876OW+6GeXLbGhDsik+wn49ggbXZwiAhPx0\\\\\\\/tXBc8u0UYcHncu01Z5Vm5KniR3oBALHxyyO\\\\\\\/HDCWY06dhPihQIXZjSfrZrU8RfyiOj3n9erSNoW6m48qWXFMXLOte+AAZcYcGVGs8zMeJ2rZQCyGQs+trbDNOFo\\\\\\\/3L+ECcdxVxSXKRubG\\\\\\\/exiszvtJlGMGxE+KV9qtzgWwbur2OU7E7c8uP\\\\\\\/Aw4l5hZi7wSQ\\\\\\\/ZnzFecTH8wDLigk8wG+TuCw07FvY8xqVT0a3M7uT3cV7y7kA0juFBHYOaAMOblGZQSgHo7JZ4\\\\\\\/kBLzWW26L0BitzrlJk6RRGBPqPpXxxXENKfxkvPzdWCaDwDxxJuvbShDGRApJtES1NWkMy7IXv8ZYu3ZGBMtDxgYkKyqcdUBdHduyIqeCE4LuDWBr4s3bkNQ6oVJMk9zSlf4mLoD4FhMm8gwZHp\\\\\\\/m+MeqZXjNigHf3379UBTTtC9kGB1XIpFEnVIoUFLXA51iWD3ErEW5M\\\\\\\/kgDLN+pf\\\\\\\/ajtCdjLJ5d8RXgmEfTOYR3X\\\\\\\/4OcoxcO+P2pNR12qIzAngRaQBgH98D\\\\\\\/r0tFlLeE0\\\\\\\/ZI8bbmGldWsFZVWMQWLHENzR+kKsywrCXgfN6i0Unqhap9mjaXsTUv6Mf8X9\\\\\\\/eP2HYmK+HqXL5MIgu5WJCLmWMUmT2xohPCvqtCRWSzeygRq7Dn+23j8kDRXqCSWmyel1TDNOw0kBTM9hlw7M8RtYSUi+Hm1z3o03uByx\\\\\\\/gdR2E0kvsYqyBUxT2wSisN6pcChPkqtGHBGlYbcfUzG3+laQDFRR2YlOXc+yDFmATzVF9hD89hdhVhorsHUkzUBa+y9W6kSOmqVv3D8SFhC7ZoZiM16bIV++XUHCeGeW7hMncJgySjSpyRQbeM1LyGJlprlyxoKmC9H3gD\\\\\\\/OMdb3L\\\\\\\/tiGWkZqrmj4lxiyJYAv9HamKzbtXWqzgsOYdG3roCgRY5xLX+S2rzSlAGKDH5R4yM\\\\\\\/7CKEdn3xDErqTWlxQj\\\\\\\/zbVipR+rMfK2B5Pvk5Y9AOoep3UkFJBc33eD7TqfhrKM4V+2OJunU113sYV6bZ9RxT3rHw56W+nwzKWYs3a3wOfXpFxE2i\\\\\\\/mg16vEFbd+\\\\\\\/18\\\\\\\/hxDE+XuPkwhiXbimEKZfACuTUYNPNWTXhV\\\\\\\/uXrnZjNKUDwG1iTDwntXDxsudb9CFnY5ciVGkYctm8g3RnXjtNANiXXzlFUcjVkRSdr2MOO5g8TSJA9wALDMCjin2UsZJHhsgw8HMC0lVXtsh+IxZ5RVoAIX9tlt944MQrQEMbHk2ubfE7zsKyoveGveXxSUUMFhc\\\\\\\/t9x1KvK25ojph6ZmpxLIqTpBdJtlmDwOLCIgjaCNOBIznx6m7Bvlbfi75vTXmYC+ynEcHnLw1E4x\\\\\\\/yf6FkCixUiBZt2+HHYd2mxMMv8bs04de5bTgyqeUb8tWy6B7H8IkxufQ0JXE89KDJDj\\\\\\\/YMVD6TGdwMVbo8YdyShfd5xUDjz0wGTNJrsGluq4EYNYycZcGC7360fzbZ4DETkQWPiwmwj+Mp8R89sFPCXllEJsafhYETBTirQDuhoCl018jubZXGJoM6+ZbmqWew4+5HAd760ovp6W1H1CcP78vW3JwF+uqxojJdXm7hJo9zOKye3cP53J3m2Elmn7DtlqxplT\\\\\\\/f1VHOcAwFkWPgKKynvTHx6PU8WvE3WL44DpnSo1IWI33xZ1BwiBfR47zIBNjukkW346RO8RR8BE3t6heg5EUBI1yG++Nx97Ofb4fFYO37rsN+cNPy8XISKvsSzQfOfk6G8Xi3dfi8lFJ3b\\\\\\\/fmm5wsO5eykLuMIVKvouc5jtsjrqv8K3Km4kks4gYmLurht6PPdB0czfY98TDUhX77VsYgAoD\\\\\\\/VoKhdN+6eOyWNEXkZU7cIh1nWyPavg0wiyM8IU5Bc8hvPTWerymrpORA\\\\\\\/QVhEyMenxng+NJSYU\\\\\\\/8lUblRjIseFDL+kO05LD4RiyKLiPbPnxffc3Ap7LQLBwxAXDPuYcYMr5iOZ0RBMjlISMKYiQqmeV2LTEXkmQAyG7IZA5kjFDFutD8v8L5FJ47JhzwKtB8skQ8JYKrntqUlKAin3zM4J0GYFdSlhAwDsoxVw5MbV5p1OX2LQeBp1ykDFMumzDxvMzIfW4FO9OeW0OxdZNB+aJCYifaJba5Lj7vYYlN5eOBR+LohBS0GiZ+xg8PvuJnIon\\\\\\\/x5SZT5zYqqTQK8Cowr1sgG9ycxy5YGs6keRZvISFyFy+7bjsr+a3PnhGjxaEd3MaaljCElPFcWLwsEf74+OJwMDyfLQ65L\\\\\\\/UOnxULmly0T4bPKUNG70wLFWXsvjAyL5W20HkT87JN5uk3ASrR0zEEmQvf2F+zS0bnTk8a5pkqwh5CWpodlEiSpkyp2alFHqeofzadH\\\\\\\/j7kPUsOPc+j2hxeaAN1pLyDJAdz9y7NpCsZOG26e2NJ+K4EjnFse9W5WNJxZQQIuOBJGJQvHXyK5gw6LQsXjfVAaH611GOaLiKVx15Wc\\\\\\\/qEZjA2+6aiwykCEzADaIzCrY0zH2y3He620juz5HdWsafgo3as4MNyHEgwJgNgvcooTbY5rVZrhhSHObsvHML\\\\\\\/0ZBv9rLcWOJcISBh\\\\\\\/lHJQ99CrlsiCPR0sSHGPsSw171bCXb5blOzV8ydft+mU1JlDnV+eGF+PkmoLSCHnJ1r0jrXsEUl+XMHLrw\\\\\\\/HCISs3gCQgcd9Sahdk2EofIygSkq+7n1t9+JbbFSZElvl\\\\\\\/iXnaMREVCceyfjTFlMfp89ApwBfMR3MgFi8r4QO\\\\\\\/k2c035qKWmALbHPizZ4FUQYMEzeVu2mJF3zmOu1r2wvXPv6AhSv412Ed4jLLZ9J1yxxxyQPB6baummu\\\\\\\/ifpJh7V5ObcRyor\\\\\\\/6uVZsjmLxNobVFD4eDadzyhqR2mt2B2sSGSjl2WNfjKAVNe8iMLiKIO5f1Utw7bxoaqy1M907wfvwxmavYkzhKydvNO9ztBXyV1OlH7JjsL0dtGpXYNWaE90gYepkx0rhnqIWNAoCQlFxMiGjGAdmZpeKxJqLzQLsz1F+P60MpBcww16TZ8vK0p+9hix84YJWoQL4umrkU3QB\\\\\\\/LLjyt3gnA9ApH8aUtxPluMk\\\\\\\/kFIOB\\\\\\\/dD7uZ3ccJ05hijjFgXkWA6HFGKTqT9swPtwbWSClPHn12zC4aK+V79Nrn78esj\\\\\\\/nYBoiSOfawezgNELIVU5QsaOCeXP5FB4auGTLwiGU2Rtr6FidLH9rcDjwR\\\\\\\/aHYNghsalOSLecXsv4aTYFVDVEQ+HDcOEVEpREGXPM1+BcJRkVaR\\\\\\\/\\\\\\\/pxZB81eJ+nnRwSmh15SLQ+xqMknXKxBzFX5c9XXFid2F8tMjSMtEgePK5L3qAZ9qo1FVOLMYYia2HLYHMB7Q8AanCz2Y5Al53zSuA1CnPZ\\\\\\\/DGRTR\\\\\\\/KgRWpY8bK1y+8Tr4gVswnr2BHBTaPrrwG4RH4zYXt03qkmnwobjZiAbA3yMxoIufZHfOw5eQ3ppUXtwokrYasY2gFz15RNrOIjr3cDqpXX7AtVvP2jTzT1iM6T7caXiWRZZCukICqvIFJOgfxLgxSRYVx3qwxeSiHnegx30RCXSxh0HMKX8qsBpP3M5XDZwyn4P13FR5DgliF0tjweAH51EJn7h4vM0K5GopeicOqaof3B8HHW+qlje22kGOy4DHYP8KhCpAWQlb+wk6MK+f40MNw2R4FTgtQVWJ3MDAvHi+er\\\\\\\/alyq0oq9o6BCUEsCviJsFuWc+1DaHNyhm9Ic2iD\\\\\\\/ytgvDAyvk82PnbeI3wEIcCFn2QbiYSBDZrBKA9IikQQ7l+Bw2JJjp2gFxAdpLQ+ayaiYBVkBmFqJA6lcCfhe\\\\\\\/DDuXyebNtFSqM8iLVg23k6R1njuIxZCQU\\\\\\\/uIg+j1Mglj8L6c97DyrzP8zOChCh8i6XUUAaZGbc\\\\\\\/Bpl\\\\\\\/sNTPHISAC7dx+RYXrG1S13plRHqqwtiDa6QMV99OSgMoaFraWsRtAkG2SfiWwmDu11p3x\\\\\\\/sRDFEdmY4PPNJS6bCrboP3ZLG99iSY+RRH3zioJrg9zjDj19a9WvEHCE\\\\\\\/EKl0NMVAgii0uDXeZ8KDDWseDvIoY7StUQXZHkZQXEAlBkWdvrKSxKDsyeFztlmPtLgV4B\\\\\\\/9joj8Z2\\\\\\\/ZalNceCFivY0Mcm7uto8\\\\\\\/sLMclCnnRGT9lSXCh\\\\\\\/MhrQSNy8iEvFgo6PH2bUjYKZqUrKfbdAFeCy2QMJlZlznUyBm0AFt6S0R1bizg8vhAWSf+PJk6KtnrglHdQb84SLc6SpTYUsOl\\\\\\\/0u8EmMzq7FGUnePFLDnnlEy6Pp8EhC+7OQ8FZfFgqgeZsVWuo+Zpl7lz3UPYlxSDNtaiZmWQoVbMu4APZ7uHa5RL1KBCVPegr0jH3U4tEcf5QugcgNJ4JLn1opkv\\\\\\\/dtFshnVzb8iLVk\\\\\\\/qnp5bSU86j1eLElhL5hZBuAfTgLJSNn3uiXiVH25QHgdMd8LJp5vXlnPnvCJSghq6ixg49bK6Yd\\\\\\\/PBXwc20fv8pUHdx\\\\\\\/15gCw+DFz+3gP2lkPY\\\\\\\/SbVZEBF9Ua9C\\\\\\\/tW++tESPdxQdxodn31Upg1QJ+PH7ajgXm5fB6ZaCdoPKxcEmq5GDQUVxCnEd3pl9OZtmNOBe6XcPh349PJEwWGW9BBw9S984R\\\\\\\/smVxnQXKcVi8FB174xVLvneq9rqJUnJYJAWaG8L3K2FX1Cwk7reKt1bZEgif7cUfgF\\\\\\\/\\\\\\\/ZeEwtfUzcNUSvwu0L77vsR668ZsrKojTM\\\\\\\/i1tV6XvKYghpFlYIk7VDekB9tLiC6C8ol\\\\\\\/ROtjbXM\\\\\\\/ApyDsFjdmPkIDUFKrbQJgd75YnEli9tOCkkP0GKDCjl0x971pA1Zqlgpo0yiP07pjWg0Lr99Pdsy8NEgc6Wt1z1\\\\\\\/8sxBHlwR1VmA3Cs9c8pTq5N63Ce54fTVQafAGsS0eo4kaXTv3Sex0kh7YKaSYCX+nr1rIL53F6oHhvOUALXWBgirojj+qJ+pO4mqoDNQ9fgSUBW6RdDfqBxl29bDfs0AxmbUmFB9HMHWtHJULWuortqqtJD\\\\\\\/hbJGJWs0jesOgJDN0osQ40r56YHYIj30W+nngv8vjSQI0nAUWEtmU+3TjOc2p6INKzvzepiHr743KIiCU1oILrntjU4CvoqH43m2EuxWb8IJiUF5agCwq0wbmLPNdq3I+BASu0bzKGXlCmjur4o6DXkSdMTrUsBHGWJXbw0ayJhgg8mrzYT+GIokCgjCDGS0WHF2Wjcnt5J0KqD2iD7qUbnw8RGW\\\\\\\/CDDow4+\\\\\\\/mCKWYxhU8B4zuskr2NVHA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EC+JhrnEi6dP\\\\\\\/lytBi5hTA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002211a76fe178efeb36443aa897c9a04f76630f0f72101d85efcfc48295e8f4e761\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kDtpbrzXF2wQB20V9IV25UKfsQfKBLq1k4+Z7FYcLdd79iVuyNO2p81mxVULYFLlSuw4v\\\\\\\/9c3MWp3ZLpjeuozIcBzWKtyr1EuSPsQ0QlqccmOB3uuyq9Hk9bxj1\\\\\\\/Q\\\\\\\/\\\\\\\/sbc7Q6JvaOadY4l9Hk7kHIG0TkKIJ\\\\\\\/jxgOX13DPGz1RCD+tAmiCVcywrgfmSEJtUoQwKsf3TrpiGqDxZdql3c+X\\\\\\\/N2G7wmN7RUmEjBHpFo2b10fPNMSYqDgRj+3+xAEFN\\\\\\\/BKLlxJ9PElvkoqbGvekdSFKxzA+qPE16Cg9jWu\\\\\\\/Y7p\\\\\\\/4v9\\\\\\\/Z+KfZjvBDyWTiNjMDbwdy2AwHN7AUWyfiANdOJ6GA9wns9NRbL\\\\\\\/0G9DnLTmxkjHKGZt65GsbT4lMkvAQL4vhfYBoD3DkqSfNvA00E8E8MMT8Opswpq43CIT0VhRJflrVK6c\\\\\\\/dPKo\\\\\\\/mLep7c85Hm0cV36vtayVQEqI5mLUtz4Pq+J9tTwe68yp9EsxYeQp9m187L2d\\\\\\\/pt9v6KcZ5sN\\\\\\\/PmkfNZ7lRKTb3vhyyMgkk7wrLJ10U6xdRHgVizDJtw60nD+N5iCXDwcoJXI9BxT53NW8\\\\\\\/\\\\\\\/b+hmgV8B\\\\\\\/r5aKa1Ql0gUxFQLicOkq3nXJFdJcYUk4WCizoSui0E94dnRSfddQ6W7\\\\\\\/G7KA0LoL3hzPyDTkE\\\\\\\/1VzHQm\\\\\\\/fSG28rwKyizcoJbRr1eFECupV1t+vTHxzk0ILM+EGjyq7\\\\\\\/qDA4BvDOg6ABuQsDtoswgwQutWNvhfScmeOKCTf6MpFlWzptoHyMo06GhShaqDVat96zMQZPfsCfF7978o+CBl46gtQvTWxTGxv7L5w332zyQlhNtpRJJEIZWoYV65s+fMP4JcWeUVJM7eCQ4y7k4ScWnSVPJsDLiu\\\\\\\/QM+p7sO+avsFIZERpOOG15U3M9NNDX8HtDez7e1Vf5vjdXWRbHuQZr7lgZCDWj8H2NEDizE9jnMdBRxmnpoDnGB6TJTtsvitBU7KPCu63QE3Sin3l7WPz85b935x3I+5tMIkeJ66X+Gcmzr94B2n9s7butyiq71r\\\\\\\/a3wMAfukVxuvtN3hbNULJGR7vqQ4B2ksT4Hsvt+KXYJlE7NsK46LLaOCxL0Mbj7hpbGVihaiNAxREVpswyWUSg4lr0WyJQ1yyJzIW89bNUZDgxc8i\\\\\\\/t6PsAIS4S9AFuTPWAf2ULfriKr0Hqg6gtFZHyjzzkzODDTnxRZuiYHGecS2o8DPbUoQtBmsi+rwrW2jI4chv1AEQgt2jBziLfEAmr48171MauwOGdPGiN0x98Z0UojnrCG\\\\\\\/7biNXVLu4kaOrV5jywwDvk6ffZjNb\\\\\\\/2KNOX+zPBDqbit6MxUYfkqCm0DCa1NbloMPQ4cM\\\\\\\/JAcNpTEdCcrGELnUJmBnw0ynHP22VlBBsaEl2ubPkKtpjJGtV0YaEiCfKYkouul6QW0ZvqbpGFJtjmochh+riLtCSsaZ8KSYBScgLEic\\\\\\\/KFUIJSVT+pYAJqmP3rT7Xi4acJ1s8evNC7jh+UwqSOItl6AvsJsqwv6Moj4zQ1xzjA6hfmvFcO9fLxuysq4fKqPB+27bQno1XVgwrMPoK\\\\\\\/ojSA3nY4lkXq\\\\\\\/QwFCU4MsNe1AGttwae9M91lA6O0jPemgGzlx+bEP7e3GOmUvZAGoLXwtksKfWoUbEwumE3rU5oII2qW\\\\\\\/m\\\\\\\/KmJolGNwTGUP94ca2egKTh+yaUH5wpaN+oec6e7BtQAnTA5ZkhPAWQMDeKFfoNuK45KKTV10f4EEMwM3E0g7gElDN9ig6ZBKkEUKHayORVPMOGeykzYRgfdxCso1nwbmft4yvEJ4zFNDdNca+TO3BO+nwXTpdMo6XoRDlig6hQhDuqXCQ3UBaoyxEvGVxNm6p0iCes32c7SfY6hV9Z8x\\\\\\\/+8suaSqVtlIPVSNQ7LjMXGGD5g3etaF4ti8lqctqg5yRFSYojSbsJA9gVJMwPxm1pFjuF9wp8JU5EWHG\\\\\\\/fkFJWdNj9QizExydz8eWu0tMZMd7O19EUFbEn\\\\\\\/AAG6Q6aFw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/etibRS2PM9neTmyn1PnkA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f2df36ce5d2594da4769b785398b59f09ac1f63fce48f581f5816b3b39bf85a7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459280 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/537" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Sb8_nh3fzZY54C718Sr4x9T4drgZ_aqJytirryNiA9s" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/537" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9lzDHZxBcSsC64JeBQk0B8fo5igcVF-3dfYWYK5yaLo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/5/527" title="Genetics of Ascites Resistance and Tolerance in Chicken: A Random Regression Approach" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/5/551" title="Simplified Insertion of Transgenes Onto Balancer Chromosomes via Recombinase-Mediated Cassette Exchange" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459280" id="node459280" data-pisa="ggg;2/5/537" data-pisa-master="ggg;2/5/537" data-apath="/ggg/2/5/537.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Two-Phase Analysis in Consensus Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Y. Ronin</span>, <span class="highwire-citation-author" data-delta="1">D. Mester</span>, <span class="highwire-citation-author" data-delta="2">D. Minkov</span>, <span class="highwire-citation-author" data-delta="3">R. Belotserkovski</span>, <span class="highwire-citation-author" data-delta="4">B. N. Jackson</span>, <span class="highwire-citation-author" data-delta="5">P. S. Schnable</span>, <span class="highwire-citation-author" data-delta="6">S. Aluru</span> and <span class="highwire-citation-author" data-delta="7">A. Korol</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">537-549;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002428 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459280" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Y. Ronin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Y.%2BRonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ronin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AY.%2BRonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. Mester </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BMester%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mester%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BMester%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">D. Minkov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BMinkov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minkov%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BMinkov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">R. Belotserkovski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BBelotserkovski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belotserkovski%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BBelotserkovski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">B. N. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Electrical and Computer Engineering, Iowa State University, Iowa 50014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BN.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20BN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BN.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">P. S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Genomics, Iowa State University, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P.%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AP.%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">S. Aluru </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Electrical and Computer Engineering, Iowa State University, Iowa 50014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BAluru%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aluru%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BAluru%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">A. Korol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BKorol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Korol%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AA.%2BKorol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:korol@research.haifa.ac.il" class="" data-icon-position="" data-hide-link-title="0">korol@research.haifa.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/5/537" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459280" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/537.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459280" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/537.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459280" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/5/537.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459280" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/5/537.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459280" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459280/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Numerous mapping projects conducted on different species have generated an abundance of mapping data. Consequently, many multilocus maps have been constructed using diverse mapping populations and marker sets for the same organism. The quality of maps varies broadly among populations, marker sets, and software used, necessitating efforts to integrate the mapping information and generate consensus maps. The problem of consensus genetic mapping (MCGM) is by far more challenging compared with genetic mapping based on a single dataset, which by itself is also cumbersome. The additional complications introduced by consensus analysis include inter-population differences in recombination rate and exchange distribution along chromosomes; variations in dominance of the employed markers; and use of different subsets of markers in different labs. Hence, it is necessary to handle arbitrary patterns of shared sets of markers and different level of mapping data quality. In this article, we introduce a two-phase approach for solving MCGM. In phase 1, for each dataset, multilocus ordering is performed combined with iterative jackknife resampling to evaluate the stability of marker orders. In this phase, the ordering problem is reduced to the well-known traveling salesperson problem (TSP). Namely, for each dataset, we look for order that gives minimum sum of recombination distances between adjacent markers. In phase 2, the optimal consensus order of shared markers is selected from the set of allowed orders and gives the minimal sum of total lengths of nonconflicting maps of the chromosome. This criterion may be used in different modifications to take into account the variation in quality of the original data (population size, marker quality, <em>etc</em>.). In the foregoing formulation, consensus mapping is considered as a specific version of TSP that can be referred to as “synchronized TSP.” The conflicts detected after phase 1 are resolved using either a heuristic algorithm over the entire chromosome or an exact/heuristic algorithm applied subsequently to the revealed small non-overlapping regions with conflicts separated by non-conflicting regions. The proposed approach was tested on a wide range of simulated data and real datasets from maize.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/consensus-genetic-mapping" class="hw-term hw-article-keyword hw-article-keyword-consensus-genetic-mapping" rel="nofollow">consensus genetic mapping</a></li><li class="kwd"><a href="/keyword/two-phase-approach" class="hw-term hw-article-keyword hw-article-keyword-two-phase-approach" rel="nofollow">two-phase approach</a></li><li class="kwd"><a href="/keyword/synchronized-marker-ordering-analysis" class="hw-term hw-article-keyword hw-article-keyword-synchronized-marker-ordering-analysis" rel="nofollow">synchronized marker ordering analysis</a></li><li class="kwd"><a href="/keyword/guided-evolution-strategy-optimization" class="hw-term hw-article-keyword hw-article-keyword-guided-evolution-strategy-optimization" rel="nofollow">guided evolution strategy optimization</a></li></ul><p id="p-4">Numerous mapping projects have generated an abundance of mapping data. Consequently, many multilocus maps have been constructed using diverse mapping populations and marker sets for the same species. The quality of maps varies broadly between populations, marker sets, and applied software. As one would expect, there might be some inconsistencies among different versions of maps for the same organism. This calls for new efforts to integrate the mapping information and generate consensus genetic maps (MCGM) and integrate genetic and physical maps (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Stam 1993</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Klein <em>et al.</em> 2000</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kwitek <em>et al.</em> 2001</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Williams <em>et al.</em> 2001</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wu <em>et al.</em> 2002</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Menotti-Raymond <em>et al.</em> 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Yap <em>et al.</em> 2003</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">De Givry <em>et al.</em> 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Mester <em>et al.</em> 2006</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Korol <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">De Keyser <em>et al.</em> 2010</a>; and others). MCGM problem is even more challenging compared with mapping based on one dataset due to additional complications, namely, genetic and ecological differences in recombination rates and exchange distribution along chromosomes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Korol 2001</a>), variations in dominance of the employed markers (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mester <em>et al.</em> 2003b</a>), and different subsets of markers used by different labs. The maps created by different labs for an organism could contain different numbers of markers, and not all markers will be presented in each map. Markers presented in two or more maps can be referred to as <em>shared</em>, in contrast to <em>unique</em> markers presented in one map only. Four major approaches for consensus mapping can be considered:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-5"><span class="list-label">(i) </span>Reducing the consensus-mapping problem to single dataset ordering via constructing the synthetic distance matrix from all datasets (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Wu <em>et al.</em> 2002</a>). This approach does not use the notion of conflicting markers; the approximate EM algorithm is applied for optimization.</p></li><li id="list-item-2"><p id="p-6"><span class="list-label">(ii) </span>Graph-theoretic approach by looking for non-conflicting subset of markers via joint analysis of the previously constructed individual maps. This analysis is based on searching shared orders among the already constructed genetic maps from different mapping projects. For a conflicting pair of markers, the solution is obtained using heuristics that are based on some “voting” criterion (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Yap <em>et al.</em> 2003</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Jackson <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wu <em>et al.</em> 2008</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2011</a>). For example, Jackson <em>et al.</em> consider a genetic map as a partial order, and the problem is solved by combining the graphs corresponding to individual partial orders and removing cyclic conflicts through the identification of minimum-feedback arc set. A combination of a heuristic algorithm and an exact solver are employed in the approach. This approach does not use the whole matrix of marker distances.</p></li><li id="list-item-3"><p id="p-7"><span class="list-label">(iii) </span>Using maximum-likelihood approach in two versions (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">De Givry <em>et al.</em> 2005</a>): (a) if one assumes that the data from different sets represent a single map, <em>i.e.</em> same order and distances (this situation is referred to as <em>genetic merging</em>); or (b) if the same order is assumed, but recombination rates differ between the mapping populations (this situation is referred to as <em>order merging</em>).</p></li><li id="list-item-4"><p id="p-8"><span class="list-label">(iv) </span>Reducing consensus mapping to a specific (constrained) version of the traveling salesperson problem (TSP) that can be referred to as “synchronized TSP.” To solve the problem, we search for the best multilocus order corresponding to minimum of weighted sum of map lengths among non-conflicting orders, testing only those orders that fit the condition “shared markers must be in shared order” for any subgroup of mapping populations (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Mester <em>et al.</em> 2003b</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">2006</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">2010</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Korol <em>et al.</em> 2009</a>). This approach uses the whole matrix of marker distances for each mapping population included in the analysis. A combination of heuristic and exact algorithms is used in the approach.</p></li></ul><p id="p-9">These approaches to solve MCGM were implemented as software packages: (i) JoinMap (<a href="http://www.kyazma.nl/index.php/mc.JoinMap/">http://www.kyazma.nl/index.php/mc.JoinMap/</a>); (ii) MergeMap (<a href="http://138.23.178.42/mgmap">http://138.23.178.42/mgmap</a>); (iii) CarthaGene (<a href="http://www.inra.fr/internet/Departements/MIA/T//CarthaGene/">http://www.inra.fr/internet/Departements/MIA/T//CarthaGene/</a>); and (iv) MultiPoint (<a href="http://www.multiqtl.com/">http://www.multiqtl.com/</a>). Here, we introduce new algorithms for the forth approach, which are more effective than our previously published variant (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Mester <em>et al.</em> 2006</a>). The algorithms were implemented in MultiPoint-consensus software (<a href="http://evolution.haifa.ac.il/images/stories/korol_lab/Programs/MultiPointConsensus_DemoG.zip">http://evolution.haifa.ac.il/images/stories/korol_lab/Programs/MultiPointConsensus_DemoG.zip</a>), validated on simulated datasets and illustrated on real data.</p><p id="p-10">The major points of our approach concern the chosen optimization criteria for searching for the best multilocus order and representation of the integral map resulting from consensus analysis of multiple datasets. For building consensus (non-conflicting) maps, we employ methods of conditional minimization (the condition is “shared order for shared markers”) of the weighted sum of individual chromosome lengths. In using this approach, we take advantage of high efficiency of our hybrid heuristics for discrete optimization. In building the integral map, we take into account the possibility of local map uncertainty (branching) caused by the presence of unique (set-specific) markers. This uncertainty, inescapable in many cases, is ignored by some mapping packages in which the integral map is based on averaged recombination distances. An interesting perspective to address this problem is to take advantage of the comparative genomic approach (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Faraut <em>et al.</em> 2007</a>) based on the evolutionary conservation of chromosome segments between related species.</p><div class="section" id="sec-1"><h2 class="">Models, Methods, and Algorithms</h2><div id="sec-2" class="subsection"><h3>Models and methods</h3><p id="p-11">In our approach, the optimal consensus order of shared markers is defined by searching the minimal total length of non-conflicting maps of the chromosome. The employed criterion is based on the sum of recombination rates across adjacent segments and is widely used in map construction for single-population data (see <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Molinari <em>et al.</em> 2009</a> for review). Unlike the analysis of a single dataset, in consensus mapping the optimization is conducted simultaneously on all sets or some subset of all mapping populations. The additional request is non-conflicting order of shared markers. Let <em>W<sub>i</sub></em> be the set of all markers of the analyzed chromosome of the <em>i</em><sup>th</sup> population (dataset). Then, <sub><span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5/537/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5/537/embed/mml-math-1.gif"/></noscript></span></span></sub> is the set of all markers of the consensus mapping problem. Consider markers that appear in at least one pair of mapping populations. The set <em>R</em> of such markers is referred to as <em>shared markers</em>. Correspondingly, <em>V = W-R</em> is the set of markers that appear in only one of the mapping populations and can be called unique markers.</p><p id="p-12">Denote <strong><em>G</em></strong> as the set of all possible partial orders <em>g</em> of shared markers and by <em>U</em>(<em>i</em>, <em>g</em>(<em>i</em>)) the set of all permutations of unique markers of the <em>i</em><sup>th</sup> dataset relative to order <em>g</em>(<em>i</em>)⊂<em>g</em> of shared markers of this set. An optimal partial order <em>g</em>* of shared markers can be defined as one from the set <strong><em>G</em></strong> minimizing the sum<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5/537/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5/537/embed/mml-math-2.gif"/></noscript></span></span></p><p id="p-13">Here <em>S<sub>i</sub></em> is the sum of all recombination rates along the map for the <em>i</em><sup>th</sup> set (population) of the analyzed chromosome; order <em>u<sub>i</sub></em>(<em>g</em>(<em>i</em>)) from set <em>U</em>(<em>i</em>, <em>g</em>(<em>i</em>)) is the one that provides the minimum of <em>S<sub>i</sub></em>. Coefficient <em>w<sub>i</sub></em> is the weight of <em>i</em><sup>th</sup> set in the optimization criterion that can characterize the quality or reliability of the data. This criterion can be modified to take into account the variation in the quality of original datasets (<em>e.g.</em> population size and/or marker quality).</p><p id="p-14">Generally, our approach is based on joint analysis of raw mapping data using a two-phase analytical scheme. In phase 1, multilocus ordering combined with iterative resampling (to evaluate the stability of marker orders) is performed separately for each dataset (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mester <em>et al.</em> 2003a</a>). In phase 2 we employ two methods, <em>global</em> and <em>local</em>, for solving the problem. In the global method, consensus mapping is conducted by reducing the problem to synchronized TSP (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Mester <em>et al.</em> 2006</a>). In the local method, the regions of conflicting order are defined and for each such region the analysis is conducted using criterion (1). If a conflicting region is small, an effective scheme of full enumeration is proposed; otherwise, we employ the heuristics of synchronized TSP. The results of local analysis can also be used as starting point for the global analysis. The main reason for using local analysis is the complexity of optimization for TSP with constrains. Even a single constrain may request more sophisticated algorithms and a considerable increase in computation time (sometimes by orders of magnitude; see TSP-like benchmarks and results in <a href="http://www.sintef.no/Projectweb/TOP/Problems/VRPTW/Homberger-benchmark/">http://www.sintef.no/Projectweb/TOP/Problems/VRPTW/Homberger-benchmark/</a>).</p><p id="p-15">In case of MCGM, ordering of shared markers across all analyzed datasets tremendously increases the computational complexity compared with the standard single-set TSP-like formulation, which itself is computationally challenging (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Weeks and Lange 1987</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Falk 1992</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mott <em>et al.</em> 1993</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Schiex and Gaspin 1997</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Hall <em>et al.</em> 2001</a>). As mentioned earlier, we employ in our approach the criterion of total sum length of the chromosome maps or the criterion of the total length of each region of conflict. These criteria can be modified to take into account variation in the quality of original datasets (<em>e.g.</em> population size and/or marker quality). Moreover, various other criteria employed for single-map construction can be adapted to the proposed here synchronous ordering approach.</p></div><div id="sec-3" class="subsection"><h3>Algorithms for consensus mapping</h3><p id="p-16">For consensus mapping (phase 2), we developed algorithms for the aforementioned local and global analysis. The local method includes: (a) defining the regions of local conflicts based on pairwise conflicts and (b) the algorithm of full enumeration of all non-conflicting orders of shared and unique markers in the defined conflict region (see below) to find the solution (local order with the best value of the criterion) in reasonable time. The algorithm for the global method uses special heuristics for global discrete optimization of synchronized TSP for all markers (unique, shared conflicting, and shared non-conflicting) (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Korol <em>et al.</em> 2009</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Mester <em>et al.</em> 2010</a>). Here the process of optimization is conducted along the entire chromosome using the solution obtained with the local method as an initial point. It is noteworthy that the global algorithm may be very useful in the local method when the size of the conflicting region is too large to employ the optimization based on full enumeration.</p><p id="p-17">Let us consider in more detail the developed algorithms. After getting the non-synchronized solutions based on independent mapping for each dataset (single-population mapping), conflict regions are formed by the analysis of all pairs of the resulting individual maps. Each conflict region contains shared conflicting and non-conflicting markers, and some unique markers. The remaining non-conflicting shared markers <em>between</em> the conflict regions are considered as “frozen” anchors during the solution process for each conflict region (hence, only conflict region markers participate in the optimization process). This version of the algorithm significantly reduces the computation time. Moreover, for certain sizes of conflict region, an exact solution can be obtained. The sequential steps of the analysis include:</p><ol class="list-ord " id="list-2"><li id="list-item-5"><p id="p-18">Building skeleton maps for each of the initial datasets using the methodology described in <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Mester <em>et al.</em> (2003a)</a>, <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Korol <em>et al.</em> (2009)</a>, and <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ronin <em>et al.</em> (2010)</a>.</p></li><li id="list-item-6"><p id="p-19">Revealing conflicting shared markers for each pair of chromosomes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p></li><li id="list-item-7"><p id="p-20">Integration of the results on detected pairwise conflicts (across all involved datasets) allows defining the conflicting regions separated by non-conflicting regions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p></li><li id="list-item-8"><p id="p-21">In case of a large interval between neighboring markers, the corresponding map may display an “inversion” or “transposition” of a large segment resulting in multiple conflicts with other maps. In such a case, before starting the main consensus procedure, we employ heuristic rules of transposition and inversion to get a better initial point for consensus ordering (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p></li><li id="list-item-9"><p id="p-22">Evaluating the computation complexity of the conflict regions. On this stage, a crude estimate of the computation time needed to resolve the conflict by testing all possible permutations is provided. If the needed time is not affordable, the alternative is to use heuristic optimization.</p></li><li id="list-item-10"><p id="p-23">Resolving the conflicts in each conflict region by synchronous optimization. This means that the allowed trial solutions must fit the following condition: for the considered chromosome, the map order in none of the mapping populations is in conflict with the order in any other mapping population. Obviously, for such a formulation, one may take advantage of parallel computations.</p></li><li id="list-item-11"><p id="p-24">Detecting and resolving higher-order conflicts and loops, which are much less frequent than pair conflicts (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p></li></ol><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/537/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of local conflicts: (A) pairwise conflict and (B) higher-order conflict." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-892793578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Examples of local conflicts: (A) pairwise conflict and (B) higher-order conflict.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/537/F1.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/5/537/F1.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/537/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/537/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461479" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Examples of local conflicts: (A) pairwise conflict and (B) higher-order conflict.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/537/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stepwise determination of conflict regions. White circles denote shared markers, and gray circles denote unique markers. The figure illustrates the process of delineating conflict regions based on overlapping regions of pairwise conflicts. Conflict between markers a and b includes the first five sets; the conflict region is bordered by a single line. Conflict between b and d starts in the first region (at set 4) and extends until the end (set 7) (its part outside the first region is marked by double lines). The whole conflict region is highlighted in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-892793578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Stepwise determination of conflict regions. White circles denote shared markers, and gray circles denote unique markers. The figure illustrates the process of delineating conflict regions based on overlapping regions of pairwise conflicts. Conflict between markers a and b includes the first five sets; the conflict region is bordered by a single line. Conflict between b and d starts in the first region (at set 4) and extends until the end (set 7) (its part outside the first region is marked by double lines). The whole conflict region is highlighted in gray.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/537/F2.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/5/537/F2.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/537/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/537/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461472" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Stepwise determination of conflict regions. White circles denote shared markers, and gray circles denote unique markers. The figure illustrates the process of delineating conflict regions based on overlapping regions of pairwise conflicts. Conflict between markers a and b includes the first five sets; the conflict region is bordered by a single line. Conflict between b and d starts in the first region (at set 4) and extends until the end (set 7) (its part outside the first region is marked by double lines). The whole conflict region is highlighted in gray.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/537/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inversion and transposition transformations in consensus analysis. Black and gray circles represent shared and unique markers, respectively. Shared markers with the same name are connected with solid lines. The left part of the figure illustrates the operation &#x201C;inversion&#x201D;: due to a large interval in set 2 (recombination rate &#x223C;0.31), the distal part of the map in set 2 is in conflict with the one in set 1. Operation &#x201C;inversion&#x201D; returns a nonconflicting order. Similarly, in the right part of the figure, a source of the conflict is the interval with loose linkage in set 1. Operation &#x201C;transposition&#x201D; returns a nonconflicting order." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-892793578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Inversion and transposition transformations in consensus analysis. Black and gray circles represent shared and unique markers, respectively. Shared markers with the same name are connected with solid lines. The left part of the figure illustrates the operation &#x201C;inversion&#x201D;: due to a large interval in set 2 (recombination rate &#x223C;0.31), the distal part of the map in set 2 is in conflict with the one in set 1. Operation &#x201C;inversion&#x201D; returns a nonconflicting order. Similarly, in the right part of the figure, a source of the conflict is the interval with loose linkage in set 1. Operation &#x201C;transposition&#x201D; returns a nonconflicting order.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/537/F3.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/5/537/F3.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/537/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/537/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461437" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Inversion and transposition transformations in consensus analysis. Black and gray circles represent shared and unique markers, respectively. Shared markers with the same name are connected with solid lines. The left part of the figure illustrates the operation “inversion”: due to a large interval in set 2 (recombination rate ∼0.31), the distal part of the map in set 2 is in conflict with the one in set 1. Operation “inversion” returns a nonconflicting order. Similarly, in the right part of the figure, a source of the conflict is the interval with loose linkage in set 1. Operation “transposition” returns a nonconflicting order.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">As a conflict region, we imply a set of segments of the maps for the mapping populations that include shared and unique markers and fit the conditions: (a) unique markers are flanked by or are adjacent to the shared markers and (b) at least for one pair of populations, two or more shared markers from the region appear in conflict order. The procedure of defining conflict region is illustrated in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. The analysis of several datasets may reveal a few conflict regions separated by regions of no conflict. In <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, the process of delineating a conflict region is based on overlaps of pairwise conflicts. The conflict between markers <em>a</em> and <em>b</em> includes the first five sets; the conflict region is bordered by a single line. A second conflict, between <em>b</em> and <em>d</em> markers, starts at set 4 and extends until set 7. In defining the boundaries of the conflict region, we employ the following rules: (i) the size of the region depends on the conflict markers, where a conflict marker can coincide with the boundary only if it is simultaneously the end marker of the linkage group (in the example in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, conflict markers <em>а</em> and <em>b</em> coincide with the end in sets 1 and 2, respectively); (ii) in all other cases, boundary marker for the conflict region may be either a non-conflict shared marker or a unique marker; and (iii) depending on the width of the conflict region, we employ the exact or heuristic optimization algorithm.</p><p id="p-29">Local algorithm of consensus analysis, in contrast to global analysis, applies only to the conflict region. With several regions, the conflicts are resolved successively in each such frame. A conflict region should include at least a pair of shared markers, the order of which in at least one set is opposite to the order in other sets. The conflict region of the entire set of mapping populations should include the considered pair of conflicting markers as well as all markers in between this pair, either shared or unique. In turn, the shared markers in the conflict region can also be in conflict with other markers. This conglomerate of pairwise conflicts makes up a conflict region that can also be extended by including neighbor markers from both sides that do not create any conflict (see <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). This extension is needed because during the process of resolving the conflict(s) within the conflict region, the flanking markers are frozen. Note that if one of the ends of the conflict region coincides with the end of the linkage group, then only one of the ends (the internal one) remains frozen during the optimization.</p><p id="p-30">The computation complexity and the efficiency of the proposed scheme of exact solution can be illustrated by the following calculations. Consider a conflicted interval of <em>k</em> shared conflicted markers bounded by a pair of non-conflicted markers. Let <em>n</em> be the number of unique markers in one dataset. In this situation, the total number of different orders is (<em>k</em>+<em>n</em>)!. For each order, a sum of <em>k</em>+<em>n</em>+1 elements should be calculated. According to the proposed scheme, in the first stage we prepare a table of 2<em><sup>n</sup></em>((<em>k</em>+2)(<em>k</em>+1)/2−1) optimal orders for each of the possible subsets of <em>n</em> unique markers in the intervals bounded by the pair of shared markers (for each possible pairs of <em>k</em>+2 shared markers, excluding the pair of bounded markers). The worst case corresponds to situations where all of <em>n</em> unique markers are situated in one interval of shared markers (the number of possible orders in this situation is <em>n</em>!). In the second stage, we test all <em>k</em>!(<em>k</em>+1)<em><sup>n</sup></em> combinations of the orders of <em>k</em> shared markers and arrangements of <em>n</em> unique markers among <em>k</em>+1 resulting intervals of <em>k</em>+2 shared markers. For each such combination, we calculate the criterion value as a sum of <em>k</em>+1 elements of the table prepared in the first stage. This tabulation-based method significantly reduces the computation time for finding the exact solutions. For example, in a situation with <em>k</em> = 4 and <em>n</em> = 10, using this scheme reduced the calculation time about 325-fold compared with the time one would need for consequent testing of all possible permutations. In fact, we employ a scheme of exhaustive permutations, where with increasing number of markers, the majority of marker orders can be skipped due to the preliminary computations saved in the memory. With <em>k</em>+<em>n</em> &lt; 15 (of which the number of either shared <em>k</em> or unique <em>n</em> is &lt;11), the exact solution can be obtained in less than an hour.</p><p id="p-31">The main idea of solving MCGM using the global method is to generate possible partial orders <em>g</em> of shared markers <strong><em>R</em></strong> and evaluate them by means of criterion of minimal total length <em>S</em>(<em>g</em>) (equation 1) taken across all datasets of the problem. The optimal consensus order of shared markers is defined by searching minimal total length of non-conflicting maps of the chromosome. Unlike the analysis of a single dataset, where the order is optimized based on the sum of recombination distances for one map only, here the optimization is conducted simultaneously on all sets or on some subset of all sets. Each dataset <em>i</em> contains some shared markers <strong><em>R</em></strong>(<em>i</em>)ε<strong><em>R</em></strong> named as <em>anchors</em>, and some unique (set-specific) markers <em>V</em>(<em>i</em>). Here computing the criterion (1) includes determination for each dataset <em>i</em> the optimal order of unique markers <em>V</em>(<em>i</em>) using the order of shared markers as anchors. The mapping problem is reduced to a TSP with special restriction on order of anchor markers. We do not know about any existing specific method for the TSP that can be directly applied to this specific sophisticated situation. Therefore, for searching the solution to the problem, we decided to use a meta-heuristic, guided evolution strategy (GES). It is a hybrid heuristic algorithm that combines the power of guided local search (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Voudouris 1997</a>) and evolution strategy (ES) algorithms (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Mester <em>et al.</em> 2003a</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2007</a>, <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">2010</a>). Our GES algorithm uses the basic ideas of ES (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rechenberg 1973</a>) and (<em>μ</em>, <em>λ</em>)-strategy (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Schwefel 1977</a>), which were supplemented with new elements. Namely, we extended the ES algorithm by including variable size mutations, random mutation schemes, and a heuristic algorithm that defines the “reasonable” places of mutation (for large-scale problems) (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Mester <em>et al.</em> 2010</a>). GES successfully solves some constrained discrete optimization TSP-like problems (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Mester and Braysy 2005</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">2006</a>; <a href="http://www.sintef.no/Projectweb/TOP/Problems/VRPTW/Homberger-benchmark/">http://www.sintef.no/Projectweb/TOP/Problems/VRPTW/Homberger-benchmark/</a>), classic TSP (<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Mester <em>et al.</em> 2010</a>), and genetic mapping problems formulated in TSP-like terms with the anchor constraints (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Mester <em>et al.</em> 2004</a>, <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">2006</a>, <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">2010</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Korol <em>et al.</em> 2009</a>). The GES algorithm was adapted to solve synchronized optimization problems when the exact algorithm of the local method is not effective due to the large size of the conflict region. Namely, GES was strengthened by three additional local search procedures for working with anchor-marker constrains: “Reinsert,” “Reverse-Reinsert,” and “Exchange 1*1” (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Braysy and Gendreau 2001</a>). A new <em>multi-parametric generator</em> was developed to create random orders <em>g</em> of shared markers. In our scheme, six parameters define mutation of current best solution <em>g</em><sup>best</sup> (<a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">Mester <em>et al.</em> 2010</a>).</p><p id="p-32">For getting the exact solution of MCGM via the global method we must generate all (<em>n</em>!/2) possible orders of shared markers in <em>g</em>. Clearly, it does not make sense trying all generated orders <em>g</em> of shared markers with respect to criterion (1), which includes both shared and unique markers, because the majority of orders will be very far from the optimal order. For estimating the quality of generated order <em>g<sup>t</sup></em> at step <em>t</em>, the second (<em>skeleton</em>) criterion, is used. It is computed as total map length <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup><sub>shared</sub> = <em>∑<sub>i</sub> w<sub>i</sub> S<sub>i</sub></em>(<em>g<sup>t</sup></em>), taking much less time. Note that the total map length <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup>, referred to as <em>main criterion</em>, is calculated according to eq.1, <em>i.e.</em> includes both the shared and unique markers. During the optimization process, all current <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup><sub>shared</sub> and corresponding best <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup> values are stored in a special list. The values of <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup><sub>shared</sub> and <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup> are highly correlated, although the extremes do not necessarily coincide (<em>i.e.</em> minimum of <em>S</em> may not correspond to the minimum of <em>S</em><sub>shared</sub>). Therefore, checking the main criterion for each generated order <em>g</em> of the shared markers does not seem to be a good idea. We compute the criterion <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup> only for those <em>g<sup>t</sup></em> that satisfy the condition <em>S</em><sub>shared</sub>(<em>g<sup>t</sup></em>) ≤ <em>q</em>, where parameter <em>q</em> is the threshold value of <em>S</em><sub>shared</sub> (<a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">Mester <em>et al.</em> 2010</a>). After a new successful step of optimization (that provides an improvement of <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup>), the new (<em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup><sub>shared</sub>, <em>S</em><sup>(</sup><em><sup>t</sup></em><sup>)</sup>) pair is included to the list and <em>q</em> is recalculated. Changing <em>q</em> along (and dependent on) the optimization trajectory can be considered as a learning process that allowed reducing 20- to 100-fold the computation time.</p><p>Two starting points are used in the global method: <em>g</em>* (optimal solution obtained by the local method) and quick skeleton solution <em>g</em>** (minimum of <em>S</em><sub>shared</sub>) defined by the GES algorithm (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Mester <em>et al.</em> 2006</a>). Main steps of the global method can be presented as:</p><ol class="list-ord " id="list-3"><li id="list-item-12"><p id="p-34">Define initial solution<span class="disp-formula" id="disp-formula-2"><em><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5/537/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/5/537/embed/mml-math-3.gif"/></noscript></span></em></span></p></li><li id="list-item-13"><p id="p-35"><em>t</em> = <em>t</em>+1</p></li><li id="list-item-14"><p id="p-36">Generate new <em>g<sup>t</sup></em> on <em>g</em><sup>best</sup> via muti-parametric generator</p></li><li id="list-item-15"><p id="p-37">If <em>S</em><sub>shared</sub>(<em>g<sup>t</sup></em>) ≤ <em>q</em> then</p><p id="p-38">{</p></li><li id="list-item-16"><p id="p-39">Define <em>S</em>(<em>g</em><sup><em>t</em></sup>) via GES algorithm</p></li><li id="list-item-17"><p id="p-40">If <em>S</em>(<em>g<sup>t</sup></em>) &lt; <em>S</em>(<em>g</em><sup>best</sup>) then <em>g</em><sup>best</sup> = <em>g<sup>t</sup></em></p><p id="p-41">}</p></li><li id="list-item-18"><p id="p-42">If not finished go to step 2</p></li></ol><p id="p-43">To test the efficiency of GES, we simulated single mapping populations with up to 1000 markers per chromosome, whereas for testing constrained optimization problems in consensus mapping, our simulations included multiple populations (up to 16) with up to 100 markers per population with different proportions of shared markers.</p><p id="p-44">The general scheme of the proposed two-phase approach of consensus mapping analysis is presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/537/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The general scheme of the proposed two-phase approach of consensus mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-892793578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The general scheme of the proposed two-phase approach of consensus mapping.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/537/F4.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/5/537/F4.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/537/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/537/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461514" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">The general scheme of the proposed two-phase approach of consensus mapping.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><div id="sec-5" class="subsection"><h3>Analysis of simulated data</h3><p id="p-46">Simulated data are the main instrument allowing the objective comparison of different analytical schemes of consensus analysis. It is highly desirable to imitate as many as possible features of real data that may be important for the solution process and affect the solution quality. Many of these features are the same as in multilocus mapping for single-population situations (<em>e.g.</em> <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Mester <em>et al.</em> 2003a</a>, <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">b</a>), but there are also some differences:</p><ul class="list-unord " id="list-4"><li id="list-item-19"><p id="p-47">Differences between the populations in the rate and chromosomal distribution of recombination events</p></li><li id="list-item-20"><p id="p-48">Variation in interference (positive and negative)</p></li><li id="list-item-21"><p id="p-49">Various levels of missing data</p></li><li id="list-item-22"><p id="p-50">Scoring errors varying among the loci and populations</p></li><li id="list-item-23"><p id="p-51">Variation in the proportions of shared markers</p></li></ul><p id="p-52">The main quality characteristic in our previous single-population analysis was the recovery coefficient (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Mester <em>et al.</em> 2003a</a>). This characteristic can also be applied in simulation analysis of consensus mapping analysis, with a small modification: the recovery coefficient should be considered for shared markers.</p><p id="p-53">Before moving to the results obtained for simulated data, a short explanation of employed marker names should be provided. In each set, markers located sequentially along the chromosome were enumerated sequentially. The names of shared markers look like <em>mar</em>1, <em>mar</em>2, <em>mar</em>3, etc., and the unique marker names also include the set number. Thus, markers located consequentially along chromosome for <em>i</em><sup>th</sup> set can be represented as <em>mar</em>1, <em>mar</em>2<em>_i</em>, <em>mar</em>3<em>_i</em>, <em>mar</em>4, <em>mar</em>5, <em>mar</em>6<em>_i,…</em>, where the 2<sup>nd</sup>, 3<sup>rd</sup>, and 6<sup>th</sup> markers are unique to set <em>i</em>.</p><p id="p-54">In the first two examples, we compare the effects of missing data and small sample size with the number of conflicts and quality of consensus mapping. Namely, the population size in the first example was 150, but due to missing data in the first eight sets, the average size per marker pair was 56; in the second example, the sample size for the first eight sets was 50 and no missing data were simulated. The simulated datasets used in this article can be downloaded (<a href="http://evolution.haifa.ac.il/images/stories/korol_lab/Programs/SE.rar">http://evolution.haifa.ac.il/images/stories/korol_lab/Programs/SE.rar</a>).</p><div id="sec-6" class="subsection"><h4>Example 1.1:</h4><p id="p-55">Example 1.1 includes single-chromosome data on 16 F<sub>2</sub> populations each with sample size 150 genotypes scored for 50 codominant markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/002428SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/FileS1.zip">File S1</a>, example 1.1). In 75% of the cases, the interval length was generated as an evenly distributed variable in range [1, 4] cM, and in 25%, in range [4, 20] cM. For the first eight variants, 40% of scores were simulated as missing data, whereas in the remainder 8 variants the missing scores comprised 20%. In addition, in 10% of individuals, 20% of loci of the first group and 80% of the second group were simulated as misclassified. Namely, for loci with misclassification, 5% of heterozygous genotypes H = AB are classified as AA and 5% as BB; and 10% of homozygotes AA and 10% of BB are classified as AB. The results of consensus analysis of the data are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Here, as well as in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS1.pdf">Table S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS2.pdf">Table S2</a>, we show the map length for each population obtained based on individual analysis of the population data (Ind) and based on joint (consensus) analysis (Cons); the order of shared markers is also shown. Marker names are represented by consecutive numbers in the chromosomes, so that the true order corresponds to monotonically increasing numbers. Places with wrong order are highlighted in gray; for such places, left parts of the columns correspond to individual ordering, whereas right parts correspond to consensus mapping. We can see that in the vast majority of cases consensus analysis corrects the local deviations from the true order.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461495/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461495/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Effect of high missing data on map quality in individual and consensus mapping (simulated data of example 1.1)</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461496/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461496/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Effect of reduced sample size (comparable with missing data in example 1.1) on map quality in individual and consensus mapping (simulated data of example 1.2)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h4>Example 1.2:</h4><p id="p-57">In contrast to the previous example, no missing data were in the first eight populations, but their sample size was only 50 per population, <em>i.e.</em> 1/3 of the sample size of the second group of eight populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/FileS2.zip">File S2</a>, example 1.2). All other parameters were as in example 1.1. The results of consensus analysis are shown in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>.</p><p id="p-58">In both examples, the increased level of scoring errors resulted in inflation of map lengths in the second group of populations. The differences between the results obtained for examples 1.1 and 1.2 can easily be explained. In example 1.1, due to 40% of missing data, only 56 genotypes remain, on average, for estimation of pairwise distances between markers, <em>i.e.</em> nearly the same as in the first eight populations of example 1.2. In example 1.2, individual analysis of each dataset resulted in maps with 28 cases of deviation from the simulated order of shared markers (in total) across the first eight datasets; in example 1.1 the number of such cases was nearly 2-fold (52). Thus, missing has a more influential effect on the quality of multilocus ordering than an equivalent decreasing of the sample size. Nevertheless, in both examples, consensus analysis managed to recover the true order, except for one case in example 1.1.</p></div><div id="sec-8" class="subsection"><h4>Example 2.1:</h4><p id="p-59">The sample size for each of the 16 populations was 100. The first 8 populations were simulated with no missing data, whereas 20% of missing data were simulated for the remaining 8 populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/FileS3.zip">File S3</a>, example 2.1). Moreover, zero level of scoring errors was simulated in the first group and 10% of error alleles at 75% of loci were in the second group. The absence of missing scores and errors in the first group of populations resulted in exact individual ordering of each chromosome, unlike the second group with more than three wrong orders per set (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS1.pdf">Table S1</a>). Two rounds of consensus analysis were conducted. In the first, equal weights were applied to all populations in the optimization criterion. In all sets, except one, erroneous local neighborhoods caused by missing data and marker scoring errors were corrected via consensus analysis. Only one segment from variant 13 remained unfixed and, during consensus analysis, caused an ordering error in set 1. Thus, only one out of 28 segments remained uncorrected. This exemplifies a situation in which consensus analysis results in an increased order quality compared with the wrong solutions obtained for individual sets due to the noise effects. It is noteworthy that this result was obtained without using an important resource: the information about the quality of scoring data.</p><p id="p-60">Obviously, one would expect the results obtained with noisy data to be less accurate; hence, it makes sense to decrease the contribution of more noisy data to the optimization criterion compared with less noisy data. The simplest indicator of data quality may be just the map length <em>L<sub>i</sub></em> for the corresponding <em>i</em><sup>th</sup> dataset analyzed individually (before consensus step). Indeed, with other things being equal, the map length will inflate monotonically with the level of scoring errors. This effect can be easily seen in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS1.pdf">Table S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS2.pdf">Table S2</a>, in which noisy variants have, on average, 2-fold longer maps than the “pure” variants. To reduce the impact of noisy variants to the consensus solution, one can conduct a second round of calculations using 1/<em>L<sub>i</sub></em> as weights. In our example, when 1/<em>L<sub>i</sub></em> weights were employed, the less noisy (more informative) dataset 1 overweighed the effect of the more noisy dataset 13, resulting in a correct order in the region of markers 18–19.</p></div><div id="sec-9" class="subsection"><h4>Example 2.2:</h4><p id="p-61">Example 2.2 is based on eight simulated F<sub>2</sub> mapping populations each with <em>N</em> = 100 genotypes and <em>m</em> = 50 markers. In the first four sets, there were neither missing nor scoring errors, whereas in the remaining four sets 20% of missing scores across all marker loci and 10% erroneous scores across 75% of loci were simulated. In the example, after phase 1 analysis, we observed altogether 17 conflicts; all of these were resolved in phase 2. For illustration, let us consider one of these conflicts in more detail, for <em>m25</em> and <em>m26</em> loci, involving four out of the eight sets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS2.pdf">Table S2</a>). The conflict is caused by inconsistency of the optimal orders of these markers in sets 5–7 with the optimal order (<em>i.e.</em> the correct one, coinciding with the simulated order) of these markers in set 1. For these four sets, <em>S<sub>i</sub></em> values at the solution orders resulted from separate analysis of individual sets were 244, 552, 578, and 542 cM, respectively. The two conflicting orders are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/FigureS1.pdf">Figure S1</a>. With the erroneous solution, the contributions of the defined conflict region to <em>S<sub>i</sub></em> are equal to 0.25, 0.32, 0.81, and 0.47, respectively (the non-weighted sum = 1.84, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/FigureS1.pdf">Figure S1</a>A). With the correct solution, the contributions were 0.18, 0.32, 0.81, and 0.51, respectively (sum = 1.82, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/FigureS1.pdf">Figure S1</a>B). Clearly, the second solution is preferable: its criterion value is smaller by 0.02. Using non-equal weights <em>w<sub>i</sub></em> decreases the contribution of the erroneous solution to optimization criterion <em>S</em> because missing data and scoring errors resulted in an elongation of maps in datasets from the second group. This increases the difference between the correct and erroneous solutions about 4-fold. Clearly, the simple “voting” approach would not work here because only one out of the four sets displayed the correct order between <em>m25</em> and <em>m26</em>.</p></div></div><div id="sec-10" class="subsection"><h3>Analysis of real data</h3><p id="p-62">We illustrate here the developed methodology and algorithms in consensus mapping example on maize. Two groups of data were analyzed, one with six BC populations and the other with 24 RIL populations (<a href="http://www.panzea.org/db/gateway?file_id=NAM_map_and_genos">http://www.panzea.org/db/gateway?file_id=NAM_map_and_genos</a>). The results for both groups of data include correction of the individual maps based on consensus analysis and building of integral maps.</p><div id="sec-11" class="subsection"><h4>Maize six BC populations:</h4><p id="p-63">The major step in our multilocus mapping is building a framework (skeleton) map. With this approach, the skeleton map is cleaned from markers that fit one of these conditions: (i) are absolutely linked; (ii) violate monotonic growth of recombination with its subsequent neighbors; (iii) have unstable location relative to other markers (as detected by jackknife resampling). During consensus mapping, the first step is also building skeleton maps for each dataset, but now we should retain as many shared markers as possible. Thus, on the first step, we delete only unique markers that fit one of the three conditions and each of the shared markers that is absolutely linked to the same shared marker in other sets. This allows obtaining a richer integral map at the end of the analysis. Below we present the results on consensus analysis of six BC populations (Set1–Set6). Comparing the lengths of maps constructed based on individual and consensus analysis allows judging the quality of the data.</p><p id="p-64">Three situations can be considered: (a) no difference between the lengths, implying no conflicts between the individual maps (like in chromosome 9, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS3.pdf">Table S3</a>); (b) small differences, which point to local conflicts resolved by consensus analysis with a minor elongation of the map (this situation was found in most chromosomes for most of the sets; see, for example, chromosome 1 in sets 1 and 5, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS3.pdf">Table S3</a>, and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>); and (c) larger difference, indicating a considerable conflict, as found for chromosome 6. The situation of chromosome 6 in set 4 before and after consensus analysis is presented in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>. Considerable inflation of the map length after consensus analysis may reflect a low quality of some of the markers or be a result of a real biological reason (<em>e.g.</em> inversion or transposition). In any case, the <em>cost of consensus</em> in a situation like the one found in chromosome 6 (set 4) may be too high, calling for detection and removing of the “responsible” marker from the set in which its position in the initial map seriously differs from that in other datasets. In the considered example, the map expansion was caused by the presence of marker <em>idp</em>1973. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS3.pdf">Table S3</a> shows the results for map lengths for all 10 chromosomes, including the corrected situation for chromosome 6 after removing marker <em>idp</em>1973 from set 4. We can see a good correspondence between the results of individual and consensus mapping. In addition to the previously mentioned case of “high cost” of consensus, a few additional cases may deserve similar consideration: Chr2-Set2 (144.8–167.7); Chr5-Set3(158.3–166.4); and Chr7-Set3(191.7–200.6) (see <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, C–E, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS3.pdf">Table S3</a>). For Chr2-Set2 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>), removing marker <em>idp</em>618 solves the problem of the high cost of consensus caused, actually, by two markers, <em>idp</em>618 and <em>idp</em>488 involved in non-local conflict in sets 1 and 2, and separated by 21 cM in set 1 and 11.2 cM in set 2. For Chr5-Set3 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>), markers <em>idp</em>1625 and <em>idp</em>2587, which caused an expansion of <em>idp</em>722-<em>сidp</em>1251 from 2.1 cM to 25.3 cM, were removed. Similarly, for Chr7-Set3 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5E</a>), markers <em>idp</em>1670 and <em>idp</em>1977, which caused an expansion of the interval from 39.6 cM to 61.9 cM, were also removed. On the basis of consensus analysis of the six BC mapping populations, we obtained six corrected conflict-free variants of each chromosome, as well as an integrated map (based on all six datasets). Here, as an example, we provide a fragment of the integral map for chromosome 1 (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, dark and light ellipses represent shared and unique markers, respectively). This example demonstrates that consensus analysis resolves the conflict between the individual maps, <em>i.e.</em> brings the maps to one order of shared markers. Nevertheless, the obtained solution is not free of uncertainties, especially for unique markers (see <em>Discussion</em>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/537/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples illustrating the &#x201C;cost of consensus&#x201D; (based on joint analysis of six BC maize populations): (A) low cost when there is no need in removing markers to reduce the cost; (B, C) high cost justifying removal of one marker in each case; and (D, E) moderate cost caused by marker pairs extending the map (deleting these markers reduced the segment length 2-fold)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-892793578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Examples illustrating the &#x201C;cost of consensus&#x201D; (based on joint analysis of six BC maize populations): (A) low cost when there is no need in removing markers to reduce the cost; (B, C) high cost justifying removal of one marker in each case; and (D, E) moderate cost caused by marker pairs extending the map (deleting these markers reduced the segment length 2-fold).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/537/F5.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/5/537/F5.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/537/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/537/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461451" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-65" class="first-child">Examples illustrating the “cost of consensus” (based on joint analysis of six BC maize populations): (A) low cost when there is no need in removing markers to reduce the cost; (B, C) high cost justifying removal of one marker in each case; and (D, E) moderate cost caused by marker pairs extending the map (deleting these markers reduced the segment length 2-fold).</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/537/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A fragment of the integral map of chromosome 1 (based on data of six BC mapping populations of maize). Dark nodes denote shared markers, and light nodes denote unique markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-892793578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A fragment of the integral map of chromosome 1 (based on data of six BC mapping populations of maize). Dark nodes denote shared markers, and light nodes denote unique markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/537/F6.medium.gif" width="221" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/5/537/F6.medium.gif" width="221" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/537/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/537/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461459" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-66" class="first-child">A fragment of the integral map of chromosome 1 (based on data of six BC mapping populations of maize). Dark nodes denote shared markers, and light nodes denote unique markers.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h4>Maize 24 RIL populations:</h4><p id="p-67">For each population, the data include marker scores for each of the 10 chromosomes. Unlike the previously mentioned BC data, these sets have a very high level of shared markers and a high quality of marker scoring. For each chromosome, we show the length of the map (in cM) based on ordering for each dataset separately (first row) and based on consensus analysis (second row). Except for a few cases (marked by frames in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/TableS4.pdf">Table S4</a>), the map lengths for individual-set ordering and consensus ordering were identical or very similar. This means that the unique maps built independently for each dataset displayed no conflicts of multilocus orders, pointing to a high-quality marker scoring. Still, for some set-chromosome combinations, the differences were moderate (3–10 cM) or rather high (&gt;40 cM!). The last situation means that the cost of consensus is “not affordable” and that corresponding markers in the specific dataset should be removed, no matter which mechanism generated such a discrepancy of the specific population from all others. However, such a situation may also result from low marker density in some sets and real variation of marker positions among mapping populations, <em>e.g.</em> due to chromosomal rearrangements (see <em>Discussion</em>).</p><p id="p-68">Here we provide more details on the above-noted case with considerable difference between the map length before and after consensus analysis (89.2 cM <em>vs.</em> 132.9 cM) for chromosome 9 in set 21. As in the BC data, one marker caused the effect. Here it was marker I00098. In this case, the effect was due to a very sparse map, which is actually represented by only four loci separated by 34, 28, and 36 cM. With such distances, unequivocal ordering is problematic (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/FigureS2.pdf">Figure S2</a>). We can see from the illustration that unique order of marker I00098 in set 21 differs from the consensus order: the attempt for consensus resulted in moving this marker to the opposite end of the map, increasing the distance between I00098 and its closest neighbor from 36 cM to nearly 80 cM. It is noteworthy that, in all other sets, the size of the same interval varied from 7 cM in set 11 to 20 cM in set 22. This may mean either very high noise in scoring I00098 in set 21 or hotspot of recombination in this interval in the heterozygote from which set 21 was established.</p><p id="p-69">For illustration, we show here the integral map of chromosome 1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1/FigureS3.pdf">Figure S3</a>). Obviously, this integral map differs significantly from the one shown for BC populations. This is due to the simple fact that BC data included a considerable proportion of non-shared markers, whereas in the RIL data, the proportion of such markers is close to zero. As a result, the integral map for the RIL problem is mainly represented as one-dimensional chain interrupted by some simple loops caused by unique patterns of chromosomal distribution of partially shared markers.</p></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-70">The application of the proposed approach based on synchronized-TSP formulation (hence, constrained discrete optimization) is illustrated on simulated data and real data on maize. In a comparison with other approaches, several aspects of the proposed methodology and employed criteria are worth discussing. Building multilocus maps can be based on various criteria, and choosing “the best” is not a simple task (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Molinari <em>et al.</em> 2009</a>).</p><p id="p-71">Reduction of the consensus mapping problem to single-dataset mutilocus ordering using the synthetic distance matrix constructed from all datasets seems to be the simplest possible approach to the problem. It “automatically” resolves all conflicts because ordering is based on only one (synthetic) matrix. With this approach, irresolvable problems on mutual positioning of shared markers belonging to non-overlapping subsets are overlooked because of the assumption of proportional parallel changes of recombination rates along the chromosome in different sets that may contradict the reality (see example in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). In the considered example, <em>m<sub>ij</sub></em> are population-specific (nonshared) markers. Marker <em>m</em><sub>22</sub> in population 2 is closer to the shared marker <em>Gm</em><sub>2</sub> than marker <em>m</em><sub>12</sub> in population 1, due to uneven and population-specific distribution of recombination along the segment. Without additional shared markers, this difference cannot be taken into account. Assuming “proportional” changes in recombination rates, one will get a wrong order, whereas the correct order will be obtained if additional shared markers <em>Gm</em><sub>3</sub> and <em>Gm</em><sub>4</sub> are available. Thus, with only two shared markers <em>Gm</em><sub>1</sub> and <em>Gm</em><sub>2</sub> in the region, the relative positions of markers <em>m<sub>ij</sub></em> cannot be fully resolved, and employing surrogate information based on linear approximation of recombination rates will result in a wrong local order in the integral map. Similarly, a wrong order will be obtained for shared markers that belong to poorly overlapping subsets of mapping populations, if the missing experimental information on recombination rates between markers from these subsets is “derived” analytically based on linear approximation.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/5/537/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correct positioning of unique (nonshared) markers on the integral map is not always possible if there are not enough shared markers. (Left) Wrong order of nonshared markers in the integral map built using the synthetic distance matrix constructed from two datasets, with unmeasured recombination rates calculated assuming proportional parallel changes of recombination along the chromosome (in our version of building the integral map, these markers form a loop). (Right) Correct order due to information provided by two additional shared markers in the considered chromosomal region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-892793578" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correct positioning of unique (nonshared) markers on the integral map is not always possible if there are not enough shared markers. (Left) Wrong order of nonshared markers in the integral map built using the synthetic distance matrix constructed from two datasets, with unmeasured recombination rates calculated assuming proportional parallel changes of recombination along the chromosome (in our version of building the integral map, these markers form a loop). (Right) Correct order due to information provided by two additional shared markers in the considered chromosomal region.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/5/537/F7.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/2/5/537/F7.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/5/537/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/5/537/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461455" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-72" class="first-child">Correct positioning of unique (nonshared) markers on the integral map is not always possible if there are not enough shared markers. (Left) Wrong order of nonshared markers in the integral map built using the synthetic distance matrix constructed from two datasets, with unmeasured recombination rates calculated assuming proportional parallel changes of recombination along the chromosome (in our version of building the integral map, these markers form a loop). (Right) Correct order due to information provided by two additional shared markers in the considered chromosomal region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-73">Much more reasonable is the graph-theoretic approach (GTA) based on the detection of non-conflicting subset of markers in the previously constructed individual maps (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Yap <em>et al.</em> 2003</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Jackson <em>et al.</em> 2008</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Wu <em>et al.</em> 2008</a>). There are some important advantages of the method presented in the current paper over GTA. In GTA, pairwise distances are employed between adjacent markers only, whereas our approach utilizes information from the whole distance matrix. Once conflicts among individual maps are detected, the correction in GTA is based on deleting some of the markers, whereas our approach is based on reordering the markers and utilizing the option of marker deletion only if the cost of consensus is high. If the initial maps were erroneous (due to small sample sizes, scoring errors, weak mapping algorithm, <em>etc</em>.), the GTA consensus map will absorb these errors (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Jackson <em>et al.</em> 2008</a>), whereas the approach proposed in this paper allows checking the data quality for each set based on resampling analysis, cleaning up the most problematic markers, and giving lower weight to datasets with inflated map lengths.</p><p id="p-74">Choice of optimization criterion is the basis of any mapping approach. Here we extend the criterion of minimum total map length, widely employed by many authors dealing with multilocus map construction in the case of a single mapping population (<em>e.g.</em> <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Weeks and Lange 1987</a>), to consensus mapping problem. For consensus mapping, we employ the sum of the map lengths taken across all datasets included in analysis. With this criterion, we are testing only those orders that fit the condition: shared order for shared markers. Obviously, this criterion can easily be extended to take into account the variation in information content in different datasets caused by variation in sample sizes and data quality. It is interesting that this approach may return correct multilocus order even in situations where intuitively clear and attractive voting criteria may result in wrong orders, <em>e.g.</em> with data different in signal/noise level (see example 2.2 above).</p><p id="p-75">A specific aspect of the consensus mapping approach proposed here is the cost of consensus. This notion is closely related to the initial formulation of our approach, with synchronous re-analysis of the initial mapping data based on testing only non-conflicting multilocus orders. Consensus ordering will be accompanied with high cost when some of the markers in the initial maps (constructed during separate analysis of each population) appear in conflict. In fact, consensus analysis moves such markers to new positions, resulting in map expansion compared with the initial map. Local corrections of marker positions during consensus analysis will cause insignificant increase in map length, but considerable changes cannot be excluded, as could be seen from our examples on real data. There might be several reasons for such considerable changes, calling for careful analysis followed by corresponding decision making (<em>e.g.</em> deleting the problematic marker from one or several datasets). Indeed, one of the reasons of high cost may be high noise in marker scoring in one of the populations. But biological reasons can also play a role, including chromosomal rearrangements (<em>e.g.</em> <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nelson <em>et al.</em> 1995</a>). In such cases, a marker may have a different map position in a certain population compared with other populations (in principle, chromosomal rearrangement may also result in more serious changes of map order, involving considerable segments). With the graph-theoretic approach, the previously mentioned problem is absent because the trouble-making markers are always deleted. With our approach, the need to remove a marker due to high cost of consensus is quite rare. Usually, local repositioning of the markers resolves the noise-generated conflicts without substantial increase in map length.</p><p id="p-76">Although the general scheme of our approach includes synchronous optimization over the entire chromosomal length (global analysis), a higher computational efficiency was achieved by local analysis based on segmentation of the total map into regions of local conflicts separated by regions of no conflicts and employing the consensus optimization criterion for each conflict region separately. What are the reasons justifying the use of local analysis? The main cause of conflicts is the noise in recombination data, originating from three factors: errors in marker allele identification; missing data for some or many markers; and statistical noise decreasing with sample size. Low noise does not affect the quality of multilocus ordering (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Mester <em>et al.</em> 2003a</a>). With increasing noise, local wrong orders will be manifested as local conflicts between the individual maps. Further increase in the noise level will cause a widening of the conflict regions that will still be local, and only rare moderate noise will throw a marker far enough. Thus, local analysis is justified by the assumption of low-to-moderate noise in recombination scores. The other assumption is that resolving local conflicts should provide the same solution as global analysis based on optimization of the main criterion (total map length, equation 2). Obviously, larger size of the entire chromosome compared with the sizes of regions of local conflicts considerably complicates the global solution process using heuristic algorithms.</p><p id="p-77">Unlike many applications of TSP, where differences in portions of percentage in the achieved optimal value of the criterion have no practical value, in genetic mapping, the value of the criterion is not important by itself. However, even small differences in the criterion value can be associated with considerable deviation of the calculated order from the true order. Therefore, as the first step in the consensus analysis, we conduct ordering separately for each dataset taking advantage of jackknife resampling verification of the obtained orders (<a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Mester <em>et al.</em> 2003a</a>). The next step is detection of conflicts and local analysis to resolve the conflicts keeping the non-conflict regions “protected” (frozen). With small sizes of the conflict region, an exact analysis is affordable; otherwise, we employ heuristic algorithms for synchronous-TSP optimization (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). At this step, the cost of consensus should be taken into account to prevent considerable map length inflation due to noisy markers. After all local conflicts are resolved, the obtained solution can be employed as a starting point to run the global optimization over the entire chromosome to ensure that the obtained solution cannot be further improved.</p><p id="p-78">An interesting perspective for improving the quality of consensus analysis may be mutual controlling of the results obtained with synchronous-TSP heuristics and GTA. Namely, the multilocus orders constructed by reanalysis or raw data and coordinated with resampling quality control may be employed as a starting point for both approaches. Although GTA is less sparing (less economical) with respect to conflicting markers (hence, fewer markers will remain in the GTA consensus maps than in the synchronous-TSP maps), if the two approaches perform well, the subset of markers remaining in the final consensus maps from the two methods are expected to be in the same order. Such correspondence will point to the quality of the solution. Other possibilities of building hybrid algorithms taking advantage of the complementary properties of the two approaches remain to be investigated.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-79">This research was partially supported by the Binational Agricultural Research and Development Fund (BARD research project US-3873-06), by the Israeli Ministry of Absorption, and by FP7-212019 research grant. The constructive comments of the reviewers are acknowledged with appreciation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002428/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-80"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-02-19"><span class="received-label">Received </span>February 19, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-03-01"><span class="accepted-label">Accepted </span>March 1, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Ronin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-2.5.537.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bräysy</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gendreau</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-comment">Metaheuristics for the vehicle routing problem with time windows (Internal Report STF42 A01025). SINTEF Applied Mathematics, Department of Optimization, Oslo, Norway</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.2" data-doi="10.1093/bioinformatics/bti222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Givry</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabrier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiex</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">CarthaGene: multipopulation integrated genetic and radiation hybrid mapping</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1703</span>–<span class="cit-lpage">1704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti222%26rft_id%253Dinfo%253Apmid%252F15598829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS84LzE3MDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.3" data-doi="10.1186/1471-2199-11-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Keyser</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">O. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Bockstaele</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Riek</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multipoint-likelihood maximization mapping on 4 segregating populations to achieve an integrated framework map for QTL analysis in pot azalea (<em>Rhododendron simsii</em> hybrids)</span>. <abbr class="cit-jnl-abbrev">BMC Mol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bmolecular%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BMol%2BBiol%26rft.aulast%253DDe%2BKeyser%26rft.auinit1%253DE.%26rft.volume%253D11%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DMultipoint-likelihood%2Bmaximization%2Bmapping%2Bon%2B4%2Bsegregating%2Bpopulations%2Bto%2Bachieve%2Ban%2Bintegrated%2Bframework%2Bmap%2Bfor%2BQTL%2Banalysis%2Bin%2Bpot%2Bazalea%2B%2528Rhododendron%2Bsimsii%2Bhybrids%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2199-11-1%26rft_id%253Dinfo%253Apmid%252F20070894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2199-11-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20070894&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.4" data-doi="10.1002/gepi.1370090507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falk</span>  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Preliminary ordering of multiple linked loci using pairwise linkage data</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DFalk%26rft.auinit1%253DC.%2BT.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D367%26rft.epage%253D375%26rft.atitle%253DPreliminary%2Bordering%2Bof%2Bmultiple%2Blinked%2Bloci%2Busing%2Bpairwise%2Blinkage%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.1370090507%26rft_id%253Dinfo%253Apmid%252F1427024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.1370090507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1427024&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.5" data-doi="10.1093/bioinformatics/btl321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faraut</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Givry</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabrier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derrien</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galibert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A comparative genome approach to marker ordering</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">e50</span>–<span class="cit-lpage">e56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DFaraut%26rft.auinit1%253DT.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253De50%26rft.epage%253De56%26rft.atitle%253DA%2Bcomparative%2Bgenome%2Bapproach%2Bto%2Bmarker%2Bordering%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl321%26rft_id%253Dinfo%253Apmid%252F17237105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMy8yL2U1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhandarkar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Physical mapping with automatic capture of hybridization data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D3%26rft.spage%253D205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aluru</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-comment">Consensus genetic maps as median orders from inconsistent sources. IEEE/ACM Trans Comput Biol Bioinform. 5: 161–171</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.8" data-doi="10.1101/gr.10.6.789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartinhour</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulanch</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A high-throughput AFLP-based method for constructing integrated genetic and physical maps: progress toward a Sorghum genome map</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DKlein%26rft.auinit1%253DP.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D789%26rft.epage%253D807%26rft.atitle%253DA%2BHigh-throughput%2BAFLP-based%2BMethod%2Bfor%2BConstructing%2BIntegrated%2BGenetic%2Band%2BPhysical%2BMaps%253A%2BProgress%2BToward%2Ba%2BSorghum%2BGenome%2BMap%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.6.789%26rft_id%253Dinfo%253Apmid%252F10854411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzYvNzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzUzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-comment">Recombination, pp. 53–71 in <em>Encyclopedia of Biodiversity</em>. V. 5. Academic Press, San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frenkel</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-comment">Methods for genetic analysis in the <em>Triticeae</em>, pp. 163–200 in <em>Genetics and Genomics of the Triticeae</em>, edited by C. Feuillet and G. J. Muehlbauer. Springer Science+Business Media, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kwitek</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonellato</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gullings-Handley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Automated construction of high-density comparative maps between rat, human, and mouse</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1935</span>–<span class="cit-lpage">1943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DKwitek%26rft.auinit1%253DA.%2BE.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1935%26rft.epage%253D1943%26rft.atitle%253DAutomated%2BConstruction%2Bof%2BHigh-Density%2BComparative%2BMaps%2BBetween%2BRat%252C%2BHuman%252C%2Band%2BMouse%26rft_id%253Dinfo%253Apmid%252F11691858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMS8xOTM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzUzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.12" data-doi="10.1093/jhered/esg008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menotti-Raymond</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menotti</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Second-generation integrated genetic linkage/radiation hybrid maps of the domestic cat (<em>Felis catus</em>)</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DMenotti-Raymond%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D1%26rft.spage%253D95%26rft.epage%253D106%26rft.atitle%253DSecond-Generation%2BIntegrated%2BGenetic%2BLinkage%252FRadiation%2BHybrid%2BMaps%2Bof%2Bthe%2BDomestic%2BCat%2B%2528Felis%2Bcatus%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesg008%26rft_id%253Dinfo%253Apmid%252F12692169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6Ijk0LzEvOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzUvNTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.13" data-doi="10.1016/j.cor.2003.11.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bräysy</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Active guided evolution strategies for large-scale vehicle routing problems with time windows</span>. <abbr class="cit-jnl-abbrev">Comput. Oper. Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1593</span>–<span class="cit-lpage">1614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BOper.%2BRes.%26rft.volume%253D32%26rft.spage%253D1593%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cor.2003.11.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cor.2003.11.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bräysy</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Active guided evolution strategies for large-scale capacitated vehicle routing problems</span>. <abbr class="cit-jnl-abbrev">Comput. Oper. Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2964</span>–<span class="cit-lpage">2975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BOper.%2BRes.%26rft.volume%253D34%26rft.spage%253D2964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minkov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003a</span> <span class="cit-article-title">Constructing large-scale genetic maps using an evolutionary strategy algorithm</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2269</span>–<span class="cit-lpage">2282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D165%26rft.spage%253D2269%26rft_id%253Dinfo%253Apmid%252F14704202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14704202&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187805300049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.16" data-doi="10.1007/s00122-003-1305-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003b</span> <span class="cit-article-title">Efficient multipoint mapping: making use of dominant markers repulsion-phase</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1102</span>–<span class="cit-lpage">1112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMester%26rft.auinit1%253DD.%2BI.%26rft.volume%253D107%26rft.issue%253D6%26rft.spage%253D1102%26rft.epage%253D1112%26rft.atitle%253DEfficient%2Bmultipoint%2Bmapping%253A%2Bmaking%2Buse%2Bof%2Bdominant%2Brepulsion-phase%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1305-1%26rft_id%253Dinfo%253Apmid%252F12928774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1305-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12928774&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185811300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.17" data-doi="10.1016/j.compbiolchem.2004.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Fast and high precision algorithms for optimization in large scale genomic problems</span>. <abbr class="cit-jnl-abbrev">Comput. Biol. Chem.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2Bbiology%2Band%2Bchemistry%26rft.stitle%253DComput%2BBiol%2BChem%26rft.aulast%253DMester%26rft.auinit1%253DD.%2BI.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D281%26rft.epage%253D290%26rft.atitle%253DFast%2Band%2Bhigh%2Bprecision%2Balgorithms%2Bfor%2Boptimization%2Bin%2Blarge-scale%2Bgenomic%2Bproblems.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compbiolchem.2004.08.003%26rft_id%253Dinfo%253Apmid%252F15548455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compbiolchem.2004.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15548455&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.18" data-doi="10.1016/j.compbiolchem.2005.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korostishevsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pikus</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glazman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Multilocus consensus genetic maps (MCGM): formulation, algorithms, and results</span>. <abbr class="cit-jnl-abbrev">Comput. Biol. Chem.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2Bbiology%2Band%2Bchemistry%26rft.stitle%253DComput%2BBiol%2BChem%26rft.aulast%253DMester%26rft.auinit1%253DD.%2BI.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D20%26rft.atitle%253DMultilocus%2Bconsensus%2Bgenetic%2Bmaps%2B%2528MCGM%2529%253A%2Bformulation%252C%2Balgorithms%252C%2Band%2Bresults.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compbiolchem.2005.09.007%26rft_id%253Dinfo%253Apmid%252F16301000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compbiolchem.2005.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16301000&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.19" data-doi="10.1016/j.eswa.2005.12.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bräysy</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dullaert</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A multi-parametric evolution strategies algorithm for vehicle routing problems</span>. <abbr class="cit-jnl-abbrev">Expert Syst. Appl.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2BSyst.%2BAppl.%26rft.volume%253D32%26rft.spage%253D508%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.eswa.2005.12.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.eswa.2005.12.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korostishevsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frenkel</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bräysy</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Discrete optimization for some TSP-like genome mapping problems</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">40</span> <span class="cit-comment">in</span> <span class="cit-source">Handbook of Optimization Theory: Decision Analysis and Application</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Varela</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Acuna</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-publ-name">Nova Science Publishers, Inc.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.21" data-doi="10.1038/hdy.2009.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molinari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vencovsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evaluation of algorithms used to order markers on genetic maps</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMollinari%26rft.auinit1%253DM.%26rft.volume%253D103%26rft.issue%253D6%26rft.spage%253D494%26rft.epage%253D502%26rft.atitle%253DEvaluation%2Bof%2Balgorithms%2Bused%2Bto%2Border%2Bmarkers%2Bon%2Bgenetic%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.96%26rft_id%253Dinfo%253Apmid%252F19639011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.96&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19639011&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.22" data-doi="10.1093/nar/21.8.1965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoheisel</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehrach</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Algorithms and software tools for ordering clone libraries: application to the mapping of the genome of Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1965</span>–<span class="cit-lpage">1974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMott%26rft.auinit1%253DR.%26rft.volume%253D21%26rft.issue%253D8%26rft.spage%253D1965%26rft.epage%253D1974%26rft.atitle%253DAlgorithms%2Band%2Bsoftware%2Btools%2Bfor%2Bordering%2Bclone%2Blibraries%253A%2Bapplication%2Bto%2Bthe%2Bmapping%2Bof%2Bthe%2Bgenome%2Bof%2BSchizosaccharomyces%2Bpombe%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F21.8.1965%26rft_id%253Dinfo%253Apmid%252F8493107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTk2NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvNS81MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Molecular mapping of wheat: major genes and rearrangements in because groups 4, 5, and 7</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D141%26rft.spage%253D721%26rft_id%253Dinfo%253Apmid%252F8647405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8647405&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RX01900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rechenberg</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-source">Evolutionstrategie</span>, <span class="cit-publ-name">Fromman-Holzboog</span>, <span class="cit-publ-loc">Stuttgart</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mester</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minkov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Building reliable genetic maps: different mapping strategies may result in different maps</span>. <abbr class="cit-jnl-abbrev">Nat. Science</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BScience%26rft.volume%253D2%26rft.spage%253D576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiex</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaspin</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Carthagene: constructing and joining maximum likelihood genetic maps</span>. <abbr class="cit-jnl-abbrev">ISMB</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISMB%26rft.volume%253D5%26rft.spage%253D258%26rft_id%253Dinfo%253Apmid%252F9322047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9322047&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.27" data-doi="10.1111/j.1365-313X.1993.00739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of integrated genetic linkage maps by means of a new computer package: JoinMap</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D3%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.1993.00739.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.1993.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LC75800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwefel</span>  <span class="cit-name-given-names">H.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-source">Numeriche optimierung von vomputer-modellen mittels der evolutions strategie</span>, <span class="cit-publ-name">Birkauser</span>, <span class="cit-publ-loc">Basel</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voudouris</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-comment">Guided local search for combinatorial problems. Ph.D. Thesis, University of Essex, Colchester, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.30" data-doi="10.1016/0888-7543(87)90050-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Preliminary ranking procedures for multilocus ordering</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">236</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DWeeks%26rft.auinit1%253DD.%2BE.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D236%26rft.epage%253D242%26rft.atitle%253DPreliminary%2Branking%2Bprocedures%2Bfor%2Bmultilocus%2Bordering.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252887%252990050-4%26rft_id%253Dinfo%253Apmid%252F3446584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(87)90050-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3446584&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-comment">The genetic structure of recombinant inbred mice: high-resolution consensus maps for complex trait analysis. Genome Biol. 2: RESEARCH0046</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.32" data-doi="10.1105/tpc.010274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maehara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimokawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harada</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A comprehensive rice transcript map containing 6591 expressed sequence tag sites</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DWu%26rft.auinit1%253DJ.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D525%26rft.epage%253D535%26rft.atitle%253DA%2BComprehensive%2BRice%2BTranscript%2BMap%2BContaining%2B6591%2BExpressed%2BSequence%2BTag%2BSites%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.010274%26rft_id%253Dinfo%253Apmid%252F11910001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE0LzMvNTI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi81LzUzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-comment">On the accurate construction of consensus genetic maps. Comput. Syst. Bioinformatics Conf. <strong>7:</strong> 285–296</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-2.5.537.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-comment">Accurate construction of consensus genetic maps via integer linear programming. IEEE/ACM Trans. Comput. Biology Bioinform. 8: 381–394</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.5.537.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneidler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mattews</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartinhour</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A graph-theoretic approach to comparing and integrating genetic, physical and sequence-based maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2235</span>–<span class="cit-lpage">2247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D165%26rft.spage%253D2235%26rft_id%253Dinfo%253Apmid%252F14704199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14704199&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F5%2F537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187805300046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/5/537.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/5/527" title="Genetics of Ascites Resistance and Tolerance in Chicken: A Random Regression Approach" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/5/551" title="Simplified Insertion of Transgenes Onto Balancer Chromosomes via Recombinase-Mediated Cassette Exchange" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/5"><span>Volume 2 Issue 5, May 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/5.cover-source.jpg?itok=3sppLJvG" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/5/537" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DB1rJpZEFLK0xEEixme0lNin4LZ2GJB6lVNqNth6-f0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/5/537" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/5/537" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/5/537" class="active" data-icon-position="" data-hide-link-title="0">Two-Phase Analysis in Consensus Genetic Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459280" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9cUyWk7xAP3Gohwx1TT1ot1RHnzFZ7JsYnuAMnI3-Ts" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/5/537.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/5/537" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-LEwNogp5K3_gQT8GFkEaE8hofE4VnASEez9K8UbSbfI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/5/537" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/5/537.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459280" id="node459280--2" data-pisa="ggg;2/5/537" data-pisa-master="ggg;2/5/537" data-apath="/ggg/2/5/537.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Two-Phase Analysis in Consensus Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Y. Ronin</span>, <span class="highwire-citation-author" data-delta="1">D. Mester</span>, <span class="highwire-citation-author" data-delta="2">D. Minkov</span>, <span class="highwire-citation-author" data-delta="3">R. Belotserkovski</span>, <span class="highwire-citation-author" data-delta="4">B. N. Jackson</span>, <span class="highwire-citation-author" data-delta="5">P. S. Schnable</span>, <span class="highwire-citation-author" data-delta="6">S. Aluru</span> and <span class="highwire-citation-author" data-delta="7">A. Korol</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">537-549;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002428 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459280--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Y. Ronin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Y.%2BRonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ronin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AY.%2BRonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. Mester </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BMester%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mester%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BMester%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">D. Minkov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BMinkov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minkov%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BMinkov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">R. Belotserkovski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BBelotserkovski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belotserkovski%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BBelotserkovski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">B. N. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Electrical and Computer Engineering, Iowa State University, Iowa 50014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BN.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20BN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BN.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">P. S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Genomics, Iowa State University, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P.%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AP.%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">S. Aluru </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Electrical and Computer Engineering, Iowa State University, Iowa 50014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BAluru%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aluru%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BAluru%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">A. Korol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BKorol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Korol%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AA.%2BKorol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:korol@research.haifa.ac.il" class="" data-icon-position="" data-hide-link-title="0">korol@research.haifa.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/5/537" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/5/537&amp;title=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/5/537&amp;title=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/5/537&amp;title=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/5/537&amp;text=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/5/537&amp;title=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/5/537&amp;t=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/5/537&amp;title=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/5/537&amp;title=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459280/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459280" id="node459280--3" data-pisa="ggg;2/5/537" data-pisa-master="ggg;2/5/537" data-apath="/ggg/2/5/537.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Two-Phase Analysis in Consensus Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Y. Ronin</span>, <span class="highwire-citation-author" data-delta="1">D. Mester</span>, <span class="highwire-citation-author" data-delta="2">D. Minkov</span>, <span class="highwire-citation-author" data-delta="3">R. Belotserkovski</span>, <span class="highwire-citation-author" data-delta="4">B. N. Jackson</span>, <span class="highwire-citation-author" data-delta="5">P. S. Schnable</span>, <span class="highwire-citation-author" data-delta="6">S. Aluru</span> and <span class="highwire-citation-author" data-delta="7">A. Korol</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">537-549;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002428 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459280--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Y. Ronin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Y.%2BRonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ronin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AY.%2BRonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">D. Mester </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BMester%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mester%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BMester%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">D. Minkov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BMinkov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minkov%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BMinkov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">R. Belotserkovski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BBelotserkovski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belotserkovski%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BBelotserkovski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">B. N. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Electrical and Computer Engineering, Iowa State University, Iowa 50014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BN.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20BN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BN.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">P. S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Genomics, Iowa State University, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P.%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AP.%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">S. Aluru </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Electrical and Computer Engineering, Iowa State University, Iowa 50014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BAluru%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aluru%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BAluru%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">A. Korol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolution, University of Haifa, Mount Carmel, Haifa 31905, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BKorol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Korol%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AA.%2BKorol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:korol@research.haifa.ac.il" class="" data-icon-position="" data-hide-link-title="0">korol@research.haifa.ac.il</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=537&amp;rft.epage=549&amp;rft.atitle=Two-Phase%20Analysis%20in%20Consensus%20Genetic%20Mapping&amp;rft.volume=2&amp;rft.issue=5&amp;rft.date=2012-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ronin%2C+Y.&amp;rft.au=Mester%2C+D.&amp;rft.au=Minkov%2C+D.&amp;rft.au=Belotserkovski%2C+R.&amp;rft.au=Jackson%2C+B.+N.&amp;rft.au=Schnable%2C+P.+S.&amp;rft.au=Aluru%2C+S.&amp;rft.au=Korol%2C+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459280/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459280/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459280/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459280/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459280/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459280/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459280/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459280/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459280/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459280/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459280/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459280/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/5/537" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Models, Methods, and Algorithms</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/5/537.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/5/537.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/5/537.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/5/537.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
