<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/4/579" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/4/579.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/4/579.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/4/579.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/4/579/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/4/579.atom" />
<meta name="HW.pisa" content="ggg;4/4/579" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Analysis of Functional and Evolutionary Features of Tele-Enhancers" />
<meta name="DC.Identifier" content="10.1534/g3.114.010447" />
<meta name="DC.Date" content="2014-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Huang and Ovcharenko. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We investigated sequence features of enhancers separated from their target gene by at least one intermediate gene/exon (named tele -enhancers in this study) and enhancers residing inside their target gene locus. In this study, we used whole genome enhancer maps and gene expression profiles to establish a large panel of tele -enhancers. By contrasting tele -enhancers to proximal enhancers targeting heart genes, we observed that heart tele -enhancers use unique regulatory mechanisms based on the cardiac transcription factors SRF, TEAD, and NKX-2.5, whereas proximal heart enhancers rely on GATA4 instead. A functional analysis showed that tele -enhancers preferentially regulate house-keeping genes and genes with a metabolic role during heart development. In addition, tele -enhancers are significantly more conserved than their proximal counterparts. Similar trends have been observed for non-heart tissues and cell types, suggesting that our findings represent general characteristics of tele -enhancers." />
<meta name="DC.Contributor" content="Di Huang" />
<meta name="DC.Contributor" content="Ivan Ovcharenko" />
<meta name="article:published_time" content="2014-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Analysis of Functional and Evolutionary Features of Tele-Enhancers" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We investigated sequence features of enhancers separated from their target gene by at least one intermediate gene/exon (named &lt;i&gt;tele&lt;/i&gt;-enhancers in this study) and enhancers residing inside their target gene locus. In this study, we used whole genome enhancer maps and gene expression profiles to establish a large panel of &lt;i&gt;tele&lt;/i&gt;-enhancers. By contrasting &lt;i&gt;tele&lt;/i&gt;-enhancers to proximal enhancers targeting heart genes, we observed that heart &lt;i&gt;tele&lt;/i&gt;-enhancers use unique regulatory mechanisms based on the cardiac transcription factors SRF, TEAD, and NKX-2.5, whereas proximal heart enhancers rely on GATA4 instead. A functional analysis showed that &lt;i&gt;tele&lt;/i&gt;-enhancers preferentially regulate house-keeping genes and genes with a metabolic role during heart development. In addition, &lt;i&gt;tele&lt;/i&gt;-enhancers are significantly more conserved than their proximal counterparts. Similar trends have been observed for non-heart tissues and cell types, suggesting that our findings represent general characteristics of &lt;i&gt;tele&lt;/i&gt;-enhancers.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/04/01" />
<meta name="citation_mjid" content="ggg;4/4/579" />
<meta name="citation_id" content="4/4/579" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/4/579" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/4/579.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/4/579.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/4/579.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010447" />
<meta name="citation_pmid" content="24496725" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="579" />
<meta name="citation_lastpage" content="593" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Di Huang" />
<meta name="citation_author_institution" content="Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Ivan Ovcharenko" />
<meta name="citation_author_institution" content="Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author_email" content="ovcharen@nih.gov" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=N. Ahituv;citation_author=S. Prabhakar;citation_author=F. Poulin;citation_author=E. M. Rubin;citation_author=O. Couronne;citation_title=Mapping cis-regulatory domains in the human genome using multi-species conservation of synteny.;citation_volume=14;citation_year=2005;citation_pmid=16155111;citation_doi=10.1093/hmg/ddi338" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Akalin;citation_author=D. Fredman;citation_author=E. Arner;citation_author=X. Dong;citation_author=J. Bryne;citation_title=Transcriptional features of genomic regulatory blocks.;citation_pages=R38-R38;citation_volume=10;citation_year=2009;citation_issue=4;citation_pmid=19374772;citation_doi=10.1186/gb-2009-10-4-r38" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Balasubramanian;citation_author=D. Zheng;citation_author=Y.-J. Liu;citation_author=G. Fang;citation_author=A. Frankish;citation_title=Comparative analysis of processed ribosomal protein pseudogenes in four mammalian genomes.;citation_pages=R2-R2;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19123937;citation_doi=10.1186/gb-2009-10-1-r2" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=J.-M. Belton;citation_author=R. P. McCord;citation_author=J. H. Gibcus;citation_author=N. Naumova;citation_author=Y. Zhan;citation_title=Hi-C: A comprehensive technique to capture the conformation of genomes.;citation_pages=268-276;citation_volume=58;citation_year=2012;citation_issue=3;citation_pmid=22652625;citation_doi=10.1016/j.ymeth.2012.05.001" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. J. Blow;citation_author=D. J. McCulley;citation_author=Z. Li;citation_author=T. Zhang;citation_author=J. A. Akiyama;citation_title=ChIP-Seq identification of weakly conserved heart enhancers.;citation_pages=806-810;citation_volume=42;citation_year=2010;citation_issue=9;citation_pmid=20729851;citation_doi=10.1038/ng.650" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. W. Busser;citation_author=L. Taher;citation_author=Y. Kim;citation_author=T. Tansey;citation_author=M. J. Bloom;citation_title=A machine learning approach for identifying novel cell type-specific transcriptional regulators of myogenesis.;citation_pages=e1002531-e1002531;citation_volume=8;citation_year=2012;citation_issue=3;citation_pmid=22412381;citation_doi=10.1371/journal.pgen.1002531" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Carter;citation_author=L. Chakalova;citation_author=C. S. Osborne;citation_author=Y.-f. Dai;citation_author=P. Fraser;citation_title=Long-range chromatin regulatory interactions in vivo.;citation_pages=623-626;citation_volume=32;citation_year=2002;citation_issue=4;citation_pmid=12426570;citation_doi=10.1038/ng1051" />
<meta name="citation_reference" content="citation_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=E. A. Carver;citation_author=L. Stubbs;citation_title=Zooming in on the Human-Mouse Comparative Map: Genome Conservation Re-examined on a High-Resolution Scale;citation_pages=1123-1137;citation_volume=7;citation_year=1997;citation_issue=12;citation_pmid=9414318;citation_doi=10.1101/gr.7.12.1123" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. L. Clarke;citation_author=J. E. VanderMeer;citation_author=A. M. Wenger;citation_author=B. T. Schaar;citation_author=N. Ahituv;citation_title=Human developmental enhancers conserved between deuterostomes and protostomes.;citation_pages=e1002852-e1002852;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22876195;citation_doi=10.1371/journal.pgen.1002852" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=C. Cortes;citation_author=V. Vapnik;citation_title=Support-vector networks.;citation_pages=273-297;citation_volume=20;citation_year=1995;citation_doi=10.1007/BF00994018" />
<meta name="citation_reference" content="citation_journal_title=Mol. Psychiatry;citation_author=S. Davidson;citation_author=K. A. Miller;citation_author=A. Dowell;citation_author=A. Gildea;citation_author=A. MacKenzie;citation_title=A remote and highly conserved enhancer supports amygdala specific expression of the gene encoding the anxiogenic neuropeptide substance-P.;citation_pages=410-421;citation_volume=11;citation_year=2006;citation_doi=10.1038/sj.mp.4001787" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Davidson;citation_author=A. Starkey;citation_author=A. MacKenzie;citation_title=A first-generation microsatellite-based genetic linkage map of the Siberian jay (Perisoreus infaustus): insights into avian genome evolution.;citation_pages=1-1;citation_volume=10;citation_year=2009;citation_pmid=19121221;citation_doi=10.1186/1471-2164-10-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. R. Dixon;citation_author=S. Selvaraj;citation_author=F. Yue;citation_author=A. Kim;citation_author=Y. Li;citation_title=Topological domains in mammalian genomes identified by analysis of chromatin interactions.;citation_pages=376-380;citation_volume=485;citation_year=2012;citation_issue=7398;citation_pmid=22495300;citation_doi=10.1038/nature11082" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Djebali;citation_author=C. A. Davis;citation_author=A. Merkel;citation_author=A. Dobin;citation_author=T. Lassmann;citation_title=Landscape of transcription in human cells.;citation_pages=101-108;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955620;citation_doi=10.1038/nature11233" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=X. Dong;citation_author=D. Fredman;citation_author=B. Lenhard;citation_title=Synorth: exploring the evolution of synteny and long-range regulatory interactions in vertebrate genomes.;citation_pages=1-14;citation_volume=10;citation_year=2009;citation_doi=10.1186/gb-2009-10-1-r1" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=J.-D. Drenckhahn;citation_title=Heart development: mitochondria in command of cardiomyocyte differentiation.;citation_pages=392-393;citation_volume=21;citation_year=2011;citation_issue=3;citation_pmid=21920309;citation_doi=10.1016/j.devcel.2011.08.021" />
<meta name="citation_reference" content="citation_journal_title=Journal of the American Statistical Association.;citation_author=O. J. Dunn;citation_title= Multiple Comparisons among Means.;citation_pages=52-64;citation_volume=56;citation_year=1961;citation_issue=293;citation_doi=10.2307/2282330" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=E. Eisenberg;citation_author=E. Y. Levanon;citation_title=Human housekeeping genes are compact.;citation_pages=362-365;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12850439;citation_doi=10.1016/S0168-9525(03)00140-9" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. G. Engström;citation_author=S. J. Ho Sui;citation_author=Ø. Drivenes;citation_author=T. S. Becker;citation_author=B. Lenhard;citation_title=Genomic regulatory blocks underlie extensive microsynteny conservation in insects.;citation_volume=17;citation_year=2007;citation_pmid=17989259;citation_doi=10.1101/gr.6669607" />
<meta name="citation_reference" content="citation_journal_title=Cell research;citation_journal_abbrev=Cell Res;citation_author=J. Ernst;citation_title=Mapping enhancer and promoter interactions.;citation_pages=789-790;citation_volume=22;citation_year=2012;citation_issue=5;citation_pmid=22453239;citation_doi=10.1038/cr.2012.49" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Ernst;citation_author=M. Kellis;citation_title=ChromHMM: automating chromatin-state discovery and characterization.;citation_pages=215-216;citation_volume=9;citation_year=2012;citation_issue=3;citation_pmid=22373907;citation_doi=10.1038/nmeth.1906" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Gaszner;citation_author=G. Felsenfeld;citation_title=Insulators: exploiting transcriptional and epigenetic mechanisms.;citation_pages=703-713;citation_volume=7;citation_year=2006;citation_issue=9;citation_pmid=16909129;citation_doi=10.1038/nrg1925" />
<meta name="citation_reference" content="citation_journal_title=The American Journal of Clinical Nutrition;citation_journal_abbrev=Am J Clin Nutr;citation_author=M. K. Georgieff;citation_title=Nutrition and the developing brain: nutrient priorities and measurement;citation_pages=614S-620S;citation_volume=85;citation_year=2007;citation_issue=2;citation_pmid=17284765" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=A. B. Glinskii;citation_author=S. Ma;citation_author=J. Ma;citation_author=D. Grant;citation_author=C.-U. Lim;citation_title=Networks of intergenic long-range enhancers and snpRNAs drive castration-resistant phenotype of prostate cancer and contribute to pathogenesis of multiple common human disorders.;citation_pages=3571-3597;citation_volume=10;citation_year=2011;citation_issue=20;citation_pmid=22067658;citation_doi=10.4161/cc.10.20.17842" />
<meta name="citation_reference" content="citation_journal_title=Cardiovascular Research;citation_journal_abbrev=Cardiovasc Res;citation_author=S. Goffart;citation_author=J.-C. von Kleist-Retzow;citation_author=R. J. Wiesner;citation_title=Regulation of mitochondrial proliferation in the heart: power-plant failure contributes to cardiac failure in hypertrophy;citation_pages=198-207;citation_volume=64;citation_year=2004;citation_issue=2;citation_pmid=15485678;citation_doi=10.1016/j.cardiores.2004.06.030" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Goode;citation_author=G. M. Cooper;citation_author=J. Schmutz;citation_author=M. Dickson;citation_author=E. Gonzales;citation_title=Evolutionary constraint facilitates interpretation of genetic variation in resequenced human genomes.;citation_volume=20;citation_year=2010;citation_pmid=20067941;citation_doi=10.1101/gr.102210.109" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=D. L. Halligan;citation_author=A. Eyre-Walker;citation_author=P. Andolfatto;citation_author=P. D. Keightley;citation_title=Patterns of Evolutionary Constraints in Intronic and Intergenic DNA of Drosophila;citation_pages=273-279;citation_volume=14;citation_year=2004;citation_issue=2;citation_pmid=14762063;citation_doi=10.1101/gr.1329204" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Investigation;citation_journal_abbrev=JCI;citation_author=J. M. Huss;citation_author=D. P. Kelly;citation_title=Mitochondrial energy metabolism in heart failure: a question of balance;citation_pages=547-555;citation_volume=115;citation_year=2005;citation_issue=3;citation_pmid=15765136;citation_doi=10.1172/JCI200524405" />
<meta name="citation_reference" content="citation_title=Sci. Rep.;citation_title=Comparative genomics of the Hedgehog loci in chordates and the origins of Shh regulatory novelties.;citation_author=M. Irimia;citation_author=J. L. Royo;citation_author=D. Burguera;citation_author=I. Maeso;citation_author=J. L. Gómez-Skarmeta;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Jiao;citation_author=B. T. Sherman;citation_author=D. W. Huang;citation_author=R. Stephens;citation_author=M. W. Baseler;citation_title=DAVID-WS: a stateful web service to facilitate gene/protein list analysis.;citation_volume=28;citation_year=2012;citation_pmid=22543366;citation_doi=10.1093/bioinformatics/bts251" />
<meta name="citation_reference" content="citation_title=Mammalian Protein Metabolism;citation_title=Evolution of protein molecules;citation_author=T. H. Jukes;citation_author=C. R. Cantor;citation_pages=21-132;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Kikuta;citation_author=M. Laplante;citation_author=P. Navratilova;citation_author=A. Z. Komisarczuk;citation_author=P. G. Engström;citation_title=Genomic regulatory blocks encompass multiple neighboring genes and maintain conserved synteny in vertebrates.;citation_volume=17;citation_year=2007;citation_pmid=17387144;citation_doi=10.1101/gr.6086307" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. A. Kleinjan;citation_author=V. van Heyningen;citation_title=Long-range control of gene expression: emerging mechanisms and disruption in disease.;citation_pages=8-32;citation_volume=76;citation_year=2005;citation_issue=1;citation_pmid=15549674;citation_doi=10.1086/426833" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=L. A. Lettice;citation_author=S. J. H. Heaney;citation_author=L. A. Purdie;citation_author=L. Li;citation_author=P. de Beer;citation_title=A long-range Shh enhancer regulates expression in the developing limb and fin and is associated with preaxial polydactyly;citation_pages=1725-1735;citation_volume=12;citation_year=2003;citation_issue=14;citation_pmid=12837695;citation_doi=10.1093/hmg/ddg180" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=G. Li;citation_author=M. Fullwood;citation_author=H. Xu;citation_author=F. H. Mulawadi;citation_author=S. Velkov;citation_title=ChIA-PET tool for comprehensive chromatin interaction analysis with paired-end tag sequencing.;citation_pages=R22-R22;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20181287;citation_doi=10.1186/gb-2010-11-2-r22" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=Q. Li;citation_author=K. R. Peterson;citation_author=X. Fang;citation_author=G. Stamatoyannopoulos;citation_title=Locus control regions.;citation_volume=100;citation_year=2002;citation_pmid=12384402;citation_doi=10.1182/blood-2002-04-1104" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=A. MacKenzie;citation_author=K. A. Miller;citation_author=J. M. Collinson;citation_title=Is there a functional link between gene interdigitation and multi-species conservation of synteny blocks?;citation_pages=1217-1224;citation_volume=26;citation_year=2004;citation_issue=11;citation_pmid=15499588;citation_doi=10.1002/bies.20117" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=G. A. Maston;citation_author=S. K. Evans;citation_author=M. R. Green;citation_title=Transcriptional regulatory elements in the human genome.;citation_pages=29-59;citation_volume=7;citation_year=2006;citation_pmid=16719718;citation_doi=10.1146/annurev.genom.7.080505.115623" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=V. Matys;citation_author=O. V. Kel-Margoulis;citation_author=E. Fricke;citation_author=I. Liebich;citation_author=S. Land;citation_title=TRANSFAC(R) and its module TRANSCompel(R): transcriptional gene regulation in eukaryotes;citation_pages=D108-D110;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381825;citation_doi=10.1093/nar/gkj143" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. May;citation_author=M. J. Blow;citation_author=T. Kaplan;citation_author=D. J. McCulley;citation_author=B. C. Jensen;citation_title=Large-scale discovery of enhancers from human heart tissue.;citation_pages=89-93;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=22138689;citation_doi=10.1038/ng.1006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. H. McDonald;citation_author=M. Kreitman;citation_title=Adaptive protein evolution at the Adh locus in Drosophila.;citation_pages=652-654;citation_volume=351;citation_year=1991;citation_issue=6328;citation_pmid=1904993;citation_doi=10.1038/351652a0" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Narlikar;citation_author=N. J. Sakabe;citation_author=A. A. Blanski;citation_author=F. E. Arimura;citation_author=J. M. Westlund;citation_title=Genome-wide discovery of human heart enhancers.;citation_volume=20;citation_year=2010;citation_pmid=20075146;citation_doi=10.1101/gr.098657.109" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics Proteomics;citation_author=P. Navratilova;citation_author=T. S. Becker;citation_title=Genomic regulatory blocks in vertebrates and implications in human disease.;citation_volume=8;citation_year=2009;citation_pmid=19561171;citation_doi=10.1093/bfgp/elp019" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Ovcharenko;citation_author=G. G. Loots;citation_author=B. M. Giardine;citation_author=M. Hou;citation_author=J. Ma;citation_title=Mulan: multiple-sequence local alignment and visualization for studying function and evolution.;citation_volume=15;citation_year=2005;citation_pmid=15590941;citation_doi=10.1101/gr.3007205" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Ovcharenko;citation_author=G. G. Loots;citation_author=M. A. Nobrega;citation_author=R. C. Hardison;citation_author=W. Miller;citation_title=Evolution and functional classification of vertebrate gene deserts.;citation_volume=15;citation_year=2005;citation_pmid=15590943;citation_doi=10.1101/gr.3015505" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. W. Perry;citation_author=A. N. Boettiger;citation_author=J. P. Bothma;citation_author=M. Levine;citation_title=Shadow enhancers foster robustness of Drosophila gastrulation.;citation_pages=1562-1567;citation_volume=20;citation_year=2010;citation_issue=17;citation_pmid=20797865;citation_doi=10.1016/j.cub.2010.07.043" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Pevzner;citation_author=G. Tesler;citation_title=Human and mouse genomic sequences reveal extensive breakpoint reuse in mammalian evolution.;citation_volume=100;citation_year=2003;citation_pmid=12810957;citation_doi=10.1073/pnas.1330369100" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. E. Phillips;citation_author=V. G. Corces;citation_title=CTCF: master weaver of the genome.;citation_pages=1194-1211;citation_volume=137;citation_year=2009;citation_issue=7;citation_pmid=19563753;citation_doi=10.1016/j.cell.2009.06.001" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. M. Pomerantz;citation_author=N. Ahmadiyeh;citation_author=L. Jia;citation_author=P. Herman;citation_author=M. P. Verzi;citation_title=The 8q24 cancer risk variant rs6983267 shows long-range interaction with MYC in colorectal cancer.;citation_pages=882-884;citation_volume=41;citation_year=2009;citation_issue=8;citation_pmid=19561607;citation_doi=10.1038/ng.403" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=U. Pozzoli;citation_author=G. Menozzi;citation_author=G. P. Comi;citation_author=R. Cagliani;citation_author=N. Bresolin;citation_title=Intron size in mammals: complexity comes to terms with economy.;citation_pages=20-24;citation_volume=23;citation_year=2007;citation_issue=1;citation_pmid=17070957;citation_doi=10.1016/j.tig.2006.10.003" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Prabhakar;citation_author=J. P. Noonan;citation_author=S. Pääbo;citation_author=E. M. Rubin;citation_title=Accelerated Evolution of Conserved Noncoding Sequences in Humans;citation_pages=786-786;citation_volume=314;citation_year=2006;citation_issue=5800;citation_pmid=17082449;citation_doi=10.1126/science.1130738" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Sandelin;citation_author=W. Alkema;citation_author=P. Engström;citation_author=W. W. Wasserman;citation_author=B. Lenhard;citation_title=JASPAR: an open-access database for eukaryotic transcription factor binding profiles;citation_pages=D91-94;citation_volume=32;citation_year=2004;citation_issue=90001;citation_pmid=14681366;citation_doi=10.1093/nar/gkh012" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Sanyal;citation_author=B. R. Lajoie;citation_author=G. Jain;citation_author=J. Dekker;citation_title=The long-range interaction landscape of gene promoters.;citation_pages=109-113;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955621;citation_doi=10.1038/nature11279" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=T. Sexton;citation_author=F. Bantignies;citation_author=G. Cavalli;citation_title=Genomic interactions: chromatin loops and gene meeting points in transcriptional regulation.;citation_pages=849-855;citation_volume=20;citation_year=2009;citation_issue=7;citation_pmid=19559093;citation_doi=10.1016/j.semcdb.2009.06.004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. J. Shapiro;citation_author=E. J. Alm;citation_title=Comparing patterns of natural selection across species using selective signatures.;citation_pages=e23-e23;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18266472;citation_doi=10.1371/journal.pgen.0040023" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Siepel;citation_author=G. Bejerano;citation_author=J. S. Pedersen;citation_author=A. S. Hinrichs;citation_author=M. Hou;citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.;citation_volume=15;citation_year=2005;citation_pmid=16024819;citation_doi=10.1101/gr.3715005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=G. D. Stormo;citation_title=DNA binding sites: representation and discovery;citation_pages=16-23;citation_volume=16;citation_year=2000;citation_issue=1;citation_pmid=10812473;citation_doi=10.1093/bioinformatics/16.1.16" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. I. Su;citation_author=T. Wiltshire;citation_author=S. Batalov;citation_author=H. Lapp;citation_author=K. A. Ching;citation_title=A gene atlas of the mouse and human protein-encoding transcriptomes.;citation_volume=101;citation_year=2004;citation_pmid=15075390;citation_doi=10.1073/pnas.0400782101" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. R. Abecasis;citation_author=D. Altshuler;citation_author=A. Auton;citation_author=L. D. Brooks;citation_author=R. M. Durbin;citation_title=A map of human genome variation from population-scale sequencing.;citation_pages=1061-1073;citation_volume=467;citation_year=2010;citation_issue=7319;citation_pmid=20981092;citation_doi=10.1038/nature09534" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. E. Thurman;citation_author=E. Rynes;citation_author=R. Humbert;citation_author=J. Vierstra;citation_author=M. T. Maurano;citation_title=The accessible chromatin landscape of the human genome.;citation_pages=75-82;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955617;citation_doi=10.1038/nature11232" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. F. Vanin;citation_title=Processed Pseudogenes: Characteristics and Evolution;citation_pages=253-272;citation_volume=19;citation_year=1985;citation_issue=1;citation_pmid=3909943;citation_doi=10.1146/annurev.ge.19.120185.001345" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. E. Vinogradov;citation_title=“Genome design” model: evidence from conserved intronic sequence in human–mouse comparison.;citation_volume=16;citation_year=2006;citation_pmid=16461636;citation_doi=10.1101/gr.4318206" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Visser;citation_author=M. Kayser;citation_author=R.-J. Palstra;citation_title=HERC2 rs12913832 modulates human pigmentation by attenuating chromatin loop formation between a long-range enhancer and the OCA2 promoter.;citation_volume=22;citation_year=2012;citation_pmid=22234890;citation_doi=10.1101/gr.128652.111" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=H. Wan;citation_author=C. Liu;citation_author=S. E. Wert;citation_author=W. Xu;citation_author=Y. Liao;citation_title=CDC42 is required for structural patterning of the lung during development.;citation_pages=46-57;citation_volume=374;citation_year=2013;citation_pmid=23219958;citation_doi=10.1016/j.ydbio.2012.11.030" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=A. G. West;citation_author=P. Fraser;citation_title=Remote control of gene transcription;citation_pages=R101-R111;citation_volume=14;citation_year=2005;citation_issue=suppl_1;citation_pmid=15809261;citation_doi=10.1093/hmg/ddi104" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. Wingender;citation_author=X. Chen;citation_author=E. Fricke;citation_author=R. Geffers;citation_author=R. Hehl;citation_title=The TRANSFAC system on gene expression regulation;citation_pages=281-283;citation_volume=29;citation_year=2001;citation_issue=1;citation_pmid=11125113;citation_doi=10.1093/nar/29.1.281" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=J. Zhang;citation_author=H. M. Poh;citation_author=S. Q. Peh;citation_author=Y. Y. Sia;citation_author=G. Li;citation_title=ChIA-PET analysis of transcriptional chromatin interactions.;citation_pages=289-299;citation_volume=58;citation_year=2012;citation_issue=3;citation_pmid=22926262;citation_doi=10.1016/j.ymeth.2012.08.009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. Zhang;citation_author=C.-H. Wong;citation_author=R. Y. Birnbaum;citation_author=G. Li;citation_author=R. Favaro;citation_title=Chromatin connectivity maps reveal dynamic promoter-enhancer long-range associations.;citation_pages=306-310;citation_volume=504;citation_year=2013" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Analysis of Functional and Evolutionary Features of Tele-Enhancers" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="twitter:description" content="We investigated sequence features of enhancers separated from their target gene by at least one intermediate gene/exon (named tele -enhancers in this study) and enhancers residing inside their target gene locus. In this study, we used whole genome enhancer maps and gene expression profiles to establish a large panel of tele -enhancers. By contrasting tele -enhancers to proximal enhancers targeting heart genes, we observed that heart tele -enhancers use unique regulatory mechanisms based on the cardiac transcription factors SRF, TEAD, and NKX-2.5, whereas proximal heart enhancers rely on GATA4 instead. A functional analysis showed that tele -enhancers preferentially regulate house-keeping genes and genes with a metabolic role during heart development. In addition, tele -enhancers are significantly more conserved than their proximal counterparts. Similar trends have been observed for non-heart tissues and cell types, suggesting that our findings represent general characteristics of tele -enhancers." />
<meta name="og-title" property="og:title" content="Genome-Wide Analysis of Functional and Evolutionary Features of Tele-Enhancers" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/4/579" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We investigated sequence features of enhancers separated from their target gene by at least one intermediate gene/exon (named tele -enhancers in this study) and enhancers residing inside their target gene locus. In this study, we used whole genome enhancer maps and gene expression profiles to establish a large panel of tele -enhancers. By contrasting tele -enhancers to proximal enhancers targeting heart genes, we observed that heart tele -enhancers use unique regulatory mechanisms based on the cardiac transcription factors SRF, TEAD, and NKX-2.5, whereas proximal heart enhancers rely on GATA4 instead. A functional analysis showed that tele -enhancers preferentially regulate house-keeping genes and genes with a metabolic role during heart development. In addition, tele -enhancers are significantly more conserved than their proximal counterparts. Similar trends have been observed for non-heart tissues and cell types, suggesting that our findings represent general characteristics of tele -enhancers." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
    <title>Genome-Wide Analysis of Functional and Evolutionary Features of Tele-Enhancers | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ebyqk_R85E-QY7-NSxNcTpX9OnaCUF5AuMwgdgoNIdA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459838","apath":"\/ggg\/4\/4\/579.atom","pisa":"ggg;4\/4\/579","ac":{"\/ggg\/4\/4\/579.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/4\/579.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/4\/579"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/4\/579"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/4\/579":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZgmKtmzZvAt\\\\\\\/NQYfzMBE2osEnNYaNZinjHIRgn6yW2nhkdBkvChgEUMiwTOEUvPl+XRDqc79MyIOj6unw5iM1sdtinLDFTcMpsk9kPSSjv8cIuQ40gegzp039fU3XUeQmvJ4kBfxY7MG6tHjiSl3C4sUQiNST5S7DmHl3hyXV\\\\\\\/7sqVrAjnSTQucJlvuY6ip2Kso29ENFPNQ8W31EeiwvRJyh8HnP1qmlPB4oZvmfnAKn40TOe7LHDxd5CUPQPDjq6\\\\\\\/vFkq6rvH7asDlMbVuUMdbnmOwV6RQvABgCGtcnVHORXNFEKyqbdswiRGZuQR1iPF7aImKvdkWTLaRhAaj+YWCHY8kmvEL3tRd0Gdy+9ybnUUTotViiIyLkySNvIKsNeERSbc7GpKFZljE10+23zcMDNfsxYR3FKZCVCV+Oe7aMJjNhojVWBMOhvRJ2Bj1WDZx8sN13lr4gTbRlIg5DAE03aUoGJHW5DWy5soPYSeoV53zZ5XtUnHg81YkdrtXFZABKZvRm0fmyz8DCrrrrHQdeKPcCWFBLqjuotYCRMgnvRaG46YZxv+CZB6w9Njz27ok1vrTwoLLWRlxHQvcCt61jtzfHHXoSP64yMhGSx7k3GT3DW5GEfg4p4KmArCYd7S\\\\\\\/lHLiTcJIRL1wj5yeORG+CRV2bIPsVahB9\\\\\\\/ppHeiqTCTG5Zdk7RcmaqBfDced0ZShy2PzdFpx5ZeGQtdeUIndvPqW9lYqk\\\\\\\/TMug6NSYpiiQs0CY\\\\\\\/2\\\\\\\/k0\\\\\\\/wUGEq2av5eiKWVB+0Fvt8SRe9oZkunw77g+W3SGjmG7bXd8hHSeYODETzT629n1\\\\\\\/5rW4nQUBHvsIilSnQpb7Egbl5\\\\\\\/VrorxbC\\\\\\\/dUafhJfqOUXP+064a7fgn1qRyaSGjbpPzBA4oagZQoKZ1z7GycJDRPd0azScZVb97OpOYAOoRVOZVEyxvRKpZaXeS3JZljLtd8UJ9VOq2Wd5Cl5LHoehVhS4I8p\\\\\\\/mXGumw9\\\\\\\/gSFPA\\\\\\\/y40pD7uw\\\\\\\/B\\\\\\\/7pJy22M1w1ypY6sGjnL+R2LPjEgNCEkJbew7afQXH1o+JJhNbWWEeBPmFjC9GlEvvBGdoXD5LMX4CjzAB+qNFUWxGm\\\\\\\/lhJXdGKBt6wvexS0T5G9j+C2\\\\\\\/8EtuGlRiGYajQtkn25\\\\\\\/Xh9McqD0W2Z951+uwEu8rgE7M8iNxFAwkxrOVNySaif860b6b3n1qipM2kjylQV+5ZOrnzt4eTTwgtVgC\\\\\\\/\\\\\\\/Ct8nsb\\\\\\\/+pP1twNjqWg\\\\\\\/oGdlSwBRg02zANg1MiiyAk\\\\\\\/Gmo8xTemS7wE9y\\\u0022,\\\u0022iv\\\u0022:\\\u00223afeaJo8QVbdo84XZNcN\\\\\\\/Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002215d0e150057ae0dfb681dfbd813b172721e0f385b02997bc3343d8e16093e488\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CbkU4Fkrn8mQBtfsD5AlozjM0vcTLeXFiQX8wG+wgJtzoDV1WloDI7cS9eYzq6DHwp3WuWr5LcoATySdPn5+9OJU3EftFwoBNE3mbHSBgSb1uf\\\\\\\/EVD1EYrSxNcPuaYJEqrvsnBkb2aD49vUAoMPjlGGsXt7u6ox49IN6E8al4NTNInAx2uxESemqoEeoYwG4G\\\\\\\/3p\\\\\\\/mRi0LqMrj5FQ53jRGyDk4G6qxiQCbf5UKK5VM0gXxukZcoxzBZoKqhgwkbscN8nK58lMifxShmQZWM3uQMfymGhwV7OzldNxnWo5goU+SC3K14skcEp\\\\\\\/2mjYCvErBvaawoYBYQxiJOGNpJfMkrQVko7vLnJy0VumcKh2y3M30r7VlvMsNJK8dTiiem5M5vpw91woYV+SotJouy8s1+qr6DKArLjH0oHCLQUNxa5kmTihrX7AxWEgz\\\\\\\/zUZ5jxqjHp8da3xV7hED6uTt28RaO2qkxkiFv6zxpwJI0nOvLk6XMEGXygbciYrvLKKxX3+QoVL\\\\\\\/ClVllJxO+g1NjjRE04aJbmMje5ImP2C4tss9q8YlBHLp7vWp4KSc40JYQH56Kv\\\\\\\/2Z2TJelvACyPHWJtKNP+lqR8wB9\\\\\\\/prN\\\\\\\/7rCyKVUR82gMis+y6oD+e35xQliX2OYyCP5nhx9pT\\\\\\\/g3cya092E6a\\\\\\\/U+Op1NaiqsZnbsmmGZ0wuRcxgM3DCsNYg7d7W8WCNtaHJCY1RTGbZ9xJ2MTnE0gSVaLFZysp2bP6tt5VE0RZgN5zvzJUkLcwJU8x2OVegiZ2zGqCQiW3W\\\\\\\/nMXq34VeYJTbGO004kR9alInEkgZgN9TEC9sPBzOrw2TNRsHLPv8csuFw9GDmU1eZqVCV\\\\\\\/Evyx5S9MlymIsdVtDX1BzgBYQFsFzM2rwglD8uOc2nc032r2w8JnJPfRozcSnrvETJ3VGv30RKWglUGNvK4UVmaUugJxgNuHeKieZ\\\\\\\/Y0h0Hf77wNyyRTwEdy6\\\\\\\/AQnYfAOyfFcEyv2YzxR00EwDwDNp7GuvbsiW4J8+1cUfR5jPKgqVImu\\\\\\\/kMmv1hUZDO50HFlvJORVzZv4kCS8k5OXtcXuOcOngaKgIPrKWJDRSurtjzZbjiBfhcYt7TlpmAukJUzUk7c0dhYJIssZkF0Mss861Xfx9JQROezGu09Ks14ys7XVAfNZvANlord92lqF2G1BUJ+X0ar9tG5PoKIz9+lY4HgOz8qOxGqWTu1WHmcN5hqyf5k5ItUjxpUYr9yeDspvwiFr9Gkzs5Di7gZcyXz90egkPK0T2c2sEPzvx9d9jp5K3tkpSiGAWt+rOg0XTUqXLgPvTSv4hMjz+ryI4q11EL1ReMtMOjYNgWE\\\\\\\/RrYjZ3sloBU71z\\\\\\\/TmP0xmesODfVZqEt\\\\\\\/TCA8G6O1qs4P+m1+71pz3WBYh8YKfrR2AuTMiyVbn7OtYYizrvAKhS+1z7spImHq2cq4eroetGAXLBHlOiT0kxQiurFLpyaJhD6uaKByw77tlppie9aCMh05208+Ovsp5XfbYb7wNPBTo2\\\\\\\/AIjSOfuZ9JliIP7IcgbJWxYI71X2naKWJFfmeP2HMneA1ZVR+5jq7LpV0ol2SLqmGZChJ+T7f4BK71DiCL5Gi7H237va24uWNV0650900dki\\\\\\\/\\\\\\\/YvcQZggGdO\\\\\\\/Gl5r8LKYFjov9maWZAjydoubk00y8P9xYxFh1kkTPmkYkBIE+pN5gZ139Lnad2dNxUSVjRNdRHQSAZG8oq6tnj3HV3LUohjY8WmwAGrpGyEYG\\\\\\\/itIwHmVjYswTTaunS02WAeEc1c4lpRhpQzXxiWoaMZTXCf+m44Knp4BtdhwIytaedNbwS9lqER3Gq8R5SQYsKLeYHPMvyfDQc+\\\\\\\/bu9IXs27CXssbBdfkhchnr2XrZNiC0y2gmsA156sJnnqcDOxioyEzL+nWHXW\\\\\\\/jdd5D7AhAoXTuQ4NjSOPFFIUKFGWs9EfH1xddZTWrYQQgESB4efnhfhySFjRM1Nd3F3VmH3DKL\\\\\\\/KyAF1ZLu0rvIOqPEQiZ0vOSIecgrzWdFC6kY4Ku8\\\\\\\/r4O9uk0BGAMhXeumnliJf+iggKu0\\\\\\\/U2TrEOzfklz1y53fC7DOBJnG6T+z6YSofbKsNu+XCjds4KfGt\\\\\\\/wBNQP5DHG3C7VBxafwKQA2pQH5IydyHSWY5cWhySyOjl+CqJAKRzU3p2QBSizAC0xAMj\\\\\\\/Cww4di87vA1oUxuGs\\\\\\\/xRrUz\\\\\\\/yLnoQuyydvSXzhHdRGx1zMR9tGV3rwWFdRynlEiYacWuLegkRMVj92szqJqrbQsbDnQnjW6qHM1Y\\\\\\\/Z4JGQgDt1N14IXqM99UnBZgQNiWqTnIOpyUkzBPaGkeyhPFlvFVlue7Nlc6acR4efzThpdpNzui9kqBqRaGIdb4vjnHZ3MPWhYNvxaUWewXMSBnFfc+E3VVR7kCXuRoqqo+gXbwtx0Hd6WILuWTBBex3qUGxXTNEQQErSfXDIv6gOvvo1kAWT9TjSbsKd9OnTNpIasG580kqbY\\\\\\\/oRzc7\\\\\\\/QU6fKbqR4+GAxbFkfjmRweX644OppgExyuYF2EYKaWPunYqbr5H1oGjPhiX9gtnvAngczScoBBSH9ci0hQBjX8bjcWgt6sf1dIH\\\\\\\/utC6M\\\\\\\/UOlrHLSXINGJ9tqtjhApkYJooaPcsyhAhSFCR5d5TYYVb74SbewwJpNSBNKAhCFz4LwgZ9NMiABnt\\\\\\\/pmp25evh8dePeL4TIOKjvJdexSWmd2KYyCV84pjbDIvtoKbe3jvyzQ4jgcbgswn5H7g8ePV5TXa2AiKc3n28oKMdsu1LLthtphoHfa8+j3aeBVcAo3O\\\\\\\/dh385tXcydZvnE\\\\\\\/QSTw0SI+eLVZ8PS2C8aWdpZ8BcjGs8OzjzuImjs+\\\\\\\/AgYGNU5sntVF8t8n8h312asHgpvyz9Br6LRnMAflQCviVQBZqu83S8mLUm6eDMfTDxx59M5SXT1VAnhV0rfMr8eYIrg\\\\\\\/TTjguHFEiWeRHfO19MA4tA1vG5ui13MdyFz6vP9ND3gL0udhWKPIMZCJ2f1\\\\\\\/74MEfygBL1egH4HES5GQnUI9VV8xOXk55AzI0CCmX3LfWdvbaaKylZvNJpUaSEPI6DhDlZuwWwzAZvO\\\\\\\/BT0M\\\\\\\/yG0W0nYwggN8hckraJTLZf09QvRmdVzvkijk8Y\\\\\\\/9WKDwd24DzC+hJlDfSfilal\\\\\\\/MEbKKaKiZZM1HJQgxTcgO5ntsPb0D+zx9y1uWNJgKH5XTBdAFVUkaIVbE8sxsF8j9a2YRAAyhvMh+hyFYPUfbV7AS01wDwXUnF9c26VaVm8spxHBZqxmJDdGDYophasE9i9w3Kx9ejgCl0VweqJdcMuZR7OplmxJbsy8w1rpK8rOnOVfasyfMSfAk9drbFhZucw0nv\\\\\\\/C\\\\\\\/aoHAZzpJCjkybGh5XlC23wVfgDycGJLdV\\\\\\\/KeHUR9DWllO9HIXaLR0\\\\\\\/bwZcT4EzxVFeIQMkHASpWhxCjY8GfvZRyPJOxHZi1P6ejU5D4e11ndwWM0\\\\\\\/2Y2WeuFRW5ttUPbwQqjwvSixTlvKlFyg4lp92fgUOdowBsSJyogLJwwU1uaNcaz6Zv0lP3Ugusg7EaDPc13X7un721dsBIy5BMf7g4hgrXxHCuHvEFhW3Bgb55WtNI0dCel27DznIw45u5OKPoLpJa1X6uo4zQls4Rsq3dQiMUvpBWEu+C3YDo9UZamBqd+UwYyXhLeRo\\\\\\\/bRgO0Q6mZxmxKhVjNtwwDyF\\\\\\\/V+bPjlv+d4Fph5FaO+ZUHBfTAJ6eyJ5QK7D7kEThraPLn91dHVirJJ300NGITRPnUi3AIFno1d4dY0B6Ib4luru18z5ehasy5ROBH5pz2AHCyM3eZrRPoNpNHYdAEQw0CNgJpTEUDyXThNQAfYeNdfyxX\\\\\\\/e\\\\\\\/y3V4EeOZ3w0wywjkVRkxbjoPvbyFObdMaMHP7MLbPicG4a8dsCpue6lloGHnXIPnSb3P4uHsSH\\\\\\\/dT4BWQ+eo1CdHH+Ks4BXTFCs3ORJkN1Sl9aKaKZ1OXeu\\\\\\\/xdTo0QhMnrS5kiyL4vn2vZIKzEeHYD3QTSHUi++cPxLIiAFs\\\\\\\/2oKIaZHQHoNGvBmv1onjA50vECLe1HwM1pRpnfh7H3Z6xVAvfNUaIdGlUdvO9fvX2Hk9MSwZgeUjlwtG6\\\\\\\/IVS8JMdEpfyyzXr4rNfQbj7HnUmiz0lWnLax5WhqplG5rzFEe55KmiYd3mjy6NwWhwc2H8kukpjFA4tEQ9A37G+x9Hs\\\\\\\/cnpz5swzjbAkJlKB5R5HiXkdtDbTf3TG9HGHbWR7e\\\\\\\/BjNeNe\\\\\\\/JuXugT+4TCPWB8Ylv3HYRuggIgwFAljjyG5aBQ92vqOFeu4oXG3ZHPttVdIZdsj\\\\\\\/qOdhuJWZlqVyDVx11I0PwCMWHJWW1yuZQ\\\\\\\/o2SpnVEKFyL4nmTq5wJEgKuaj1fE6zrkoJo3WUGMOc7fRRmelhZFKnzWWeTS7srK+\\\\\\\/lenJgQBn0SOtnrrsV3Xt2NvQcR64Vg12BIf7T6zas9OWvv3BWCTdewAADL2d5wHNLd3FAlAGjQxo+6Z5iLybk9detc2v1hwDVXy36p2Zlh96L7L2X9nSWfeDr4yKlDIsUTTVgWAlvOKk5yrD9Uq4DlKIRprqo1\\\\\\\/WjhjeyPndap5+1\\\\\\\/Dvh9mljTV4T\\\\\\\/8bE7YqmlKX\\\\\\\/VlucBbaqzGkDXalOqDjt93nWGtRVH3cRZMD+7OQo4g1OhxPHxGgH9t8j59y\\\\\\\/Q0juYCP31N5BglWrzFXBN4RSoCWamTb9gzZJERAYA8Ncne\\\\\\\/NXbIOHlWeiFwPlkuTI3nrqKfedtI6UOlE8cjwDeS4a23fSG9DrbRjTfVaq2bwY7TSyGQXNJSgCaxVUqQbQFrta9B9UzoequmkUAB5klfCJWfs7rOfysrxs9ImZ0xCjHU9TQuFHGOJ737gL3AKnPTR+Smx\\\\\\\/xD7j2DHigviG1uAT15+jaiN1CQLKPxoWYIozFhCtOTsLtvfSYyPfnj0xWTQcC86WEltLW5tSJSdZDRU6MbSzQ7sYSwVrhbIFC60nAo2cVmeOvuRBzSo7WavdbGnDWYAKsur4DNDiTGZq36PhKYBT+rWQup6u1IN61SnE8A1mNdeOn1KMPI+54Uw4mUfUkYnbuRghpgnfVN\\\\\\\/oeH0rFagHMuOJVhhpak3bblANjnS+ASekj5J5P9HTCTWy5JbgBbe1D7IbY66fkzndN328E4pEzoHZbz5GxoKuxtJEVqp0HNoW\\\\\\\/kRMHrEaSZO083iZdepBtCzdRlRHUfgxnmzH6xwZlswgK4DSrgGOsYqsvJdgB0kkLW+uVDCSW3bq6TmOAN4Hj9VajxdyRtU5bGJ5sFjSYYCTHysiV0K7d4XiZO2SHbUMAojHx1twP7K1o80OVoRHzx4RqnPj74UavwfuF\\\\\\\/OBCRoK2qmJmOyE17d39m+o7wdMlfEK9J8v7m6on+HxgOdD5U85DhARrHftIhrszmMlmKHey4QCbHRce\\\\\\\/vcq0fSU9m3p5vnK4gdSc2zWF+qjuxJhYMLySa7ZX1po5iV+8d2xPyCKEaUMSOC74KoeDCAidYJEvPQvhZmdc5ysbamIF9qxSRXtvLevgyrPGszAGIrXb\\\\\\\/55d+v\\\\\\\/lTdbpeDbvyvILuLYVkcg9al9EruHr9nQLcYb0KBfYN49I9T8D50\\\\\\\/XLvJ5hrTc4favKqE4ReqhYf4F8Pk+Up1AbcPnJ\\\\\\\/pmgkjkVpkZrfNrfnNOLL5lf4wt4gY8Lkmz9JFM7p9OtgV5uME1BTjzSkhWIF6KZKVVCTMZfGkkZx5mB8k0lG5QKYQVn9Ep\\\\\\\/Ix0lKcPAa6Oif9bUC\\\\\\\/tLifU1myDmw+iks+2RPGRdWeX40dxgB3AYQteStaOI7S1ZkA4YKpCTkRqSRzNcqHvOdjYxsLqJkxg\\\\\\\/l5vU1g73W2loo41UyhBEfbuinGe3UNA17x+P6q7MSS+13M8VVW9mGPFh9HeACYsmG2DgfEvgO6bGoSNyQY6z8A7TOk4g9Hee50P5unX9mF\\\\\\\/M\\\\\\\/PXvpwE7KIRVm3QuAg+bgaA8sIIFtbavl5A5CDXxBk\\\\\\\/dwb7AvsvlYOef\\\\\\\/8OklzNVNOoOlukTrmLxcDAJTItIthuOeY6adJG3a1DLwTN+xgKJspnFLxn3c8n2yTictZkUbbLKROnajNVCOs9bYgR5f9uouCLcZwD1EmJBXPkXimyaVL+Sxo+aydH7BCAakz5EAdrkF+27c9KKEUhjZAYe0B\\\\\\\/vx+BSdbF7Av8nqWIC3ZzxEimAz+PmX981D+AwBscB3HABHV27MNHJvQ0pClqvU1qrs78wKIyRrFaRvQ8DslWHLkcjEHNiTBAvkUOaB2LBo4TlqdcB04HLYFojKXGdSlHAQzaZGlzo6K4hxRfH+bcJ2IsDMExGDLFLO7gc8Lb+UWv5vFMLlEaC9dwJlybDH\\\\\\\/YteweK7NJ4xizwnfWxbfMgpqEG9hGdXCVlqd1Oi8eOuGgQk0YDdqO5+FpqV4UNhMUUdz89kiCBTK2oKkDg0mxGngUIHR08AS2fJjKc8+4\\\\\\\/HJcSX1apSik3OX18GRuXT36cPcMrDX1qM2nvpDkB0Zm78z8ONQWQF7DAavXN67iMaWtwQtDB\\\\\\\/y9dNmsU8j0hZyRx\\\\\\\/MMCyieugASWTqKbxWGb8I6wsF0ertEEgOJdocLUGvNfosu3Gsbb1XoV5lbECWmbCJtPdrQad26Fkuxhfg2v+KcsS\\\\\\\/2yrhOOfO9CKzW3R92g3GcbYxLYaDzw4Xia2LXCNFeBgHrfqec0ugVuJ7l\\\\\\\/\\\\\\\/\\\\\\\/m8reh3JWrNVpaTbX0Tk+GQgvsjMjeLZPw9NaVu+qb+Nbpcs1oIqsUl2wlIsSqLKCoo1K1X64psGeP79Xaf01LKNuL1LyWInTH1EZYuOScGSwMxOKaLC0tU9uV58cPNjK2VGPInE1EiEhqTEASiIQvcmihqc1RTZ3pYLZg7PsRkg5QtBe+ydVNAhlE+HdjF5UNKHZLRab5pKa8IiLw\\\\\\\/vJz1A3SQ4d8oSNGYVSaBNrG5xLo3jFz3lWjIio8GQDMRvLW5fFuJS16J8WO\\\\\\\/ZZaKS21Qp2Uq9kZ2cHMftLPATc1mi9vnT9SM7E7fwx6ZaRPV9nO4qC5YS0G9wouDXAWMLSeOtMhBQYpTY4St70c6yyoMtmqfeVhxXYErzUoQ3AKIYSxTJhE\\\\\\\/7AFrKmU29YDGThWz7Mnod72xraYbDV1ptcmMkh1kiCi7w2gemBHOFW4D02m1fn8ArGNd9KlcNzqfCltjjvX0NWAvHg0865e\\\\\\\/\\\\\\\/BkqFfwi9MkJPAMkXf23fyAC8XTIPTu8y7HWIAeG72WR1ZURTk8ZbCxG3QzJOzoklmIModpLAKv9jyvkKAzu8gsVjGFfCBjq0Tv2xY6yJh3SfrXvxwnQGiNiEwvVhwoHJ+AtV7JSl8bHbkdF4HXKtMdOHj22HsmYa1OuoUHau65nldsEA\\\\\\\/Gms5uAUTC\\\\\\\/frrSUeBNh7aH1XOIu4ZdFhpRCdUhAhT+vWMD4Piwq5n2UeVGXM1YITlhQYWv0Q1I+U3FbmMf6DZVZ9rhCRKuja6Ydssai2oun3U06pfA2hMjTBKF0IXL+D07whjQVhXcsh3XMGXh44xaStUpxFHSBi5+pIcwLG6faZcFPmbuk+A6KwCWs9Vgbqsd6mVBqcxN+aZwuMF\\\\\\\/P1pgEivNTYwwd4wIJvLlYP7lQdMKevie09k6455F8P+uzzUP2LkkdvRGjZQanMPEWx55PVRpkIkld869ejfsl8L+zRJS\\\\\\\/ZyoZlcRkm\\\\\\\/ayLmMRxRD0+LHBtfwLC1ffnoV+PYqGiTnfaauS+fn9zZvd+oe0rRErOvBFvMyLxRpVHCN9op\\\\\\\/UQNb\\\\\\\/EZhrUnaSM9l5BcF\\\\\\\/8FxL4Js5k8mRQvlh8N2OAJK0EgADSMdYrl\\\\\\\/Nn3N0QoXsgmZ+gGJBgWtSzyFa5y9W8doDGclrt7OwavrwomTEw8puZBuUrjah3X5OUb62\\\\\\\/jMn9uGzJ50zpkn9+eC3iZEx2C6vZi336mUdsXgPbyyhq3Ba1zZ7ZbEhc9FVvLMciDZU3LX60wYd2j4IKIZQPepDmri1xpMi9jFBuYxYQEPLMZzbPOzn6V9FlaXpaLquFsN1E4FatOEKqYUjRhPM6SoUffUYyPG68jhtHqoNzPmQNu87KgrafVntzs4VY1C3RpLSpBlVtrBD1Oazo3S9M+7oV2iG9LYkDJe+N4TRQNLbUNo\\\\\\\/o\\\\\\\/15+Vkke6wmqARHlWQrgFLgLngSQUglFbK7D1xsPz5QwBXC8VLxCqWY\\\\\\\/mT4Nj5TcC2RtEw07U0FBPJh3jz3OLAn2zRxnRqVntr2WJr2IVCh4up9Xx9cNRhGXP7iNnHZfC1kaqg1kW27\\\\\\\/j9Nq4NczOjhIuAnTuKIZjhOes5ROujNnNTb5Iw+PFkV2Dlu7uiwvNWgbTURs4nSsfH2QGkjEwFVylvEWB3C1qBoLIhV6QU8P4yx9rOYLv1emaRK3SC\\\\\\\/wRINON7A5W1UzQubupL5zqlLgljyvvmLKGtUdFxsH56Syei7Van1oc8lMdP7b34bRLdsBk1lDH5IvRcaC\\\\\\\/ECF\\\\\\\/SMcYSu4co4BcfDBc71xbz74auvWjCyJY6b4nl15DgmnzaSTzWnjNh7zkSwAeIUGJNx9dxFbIy2XvspAT5gRTIc\\\\\\\/JIFivYuW2bl8ppzPwUsC5DnWCDbsW1Q47U+worNtHQFBt3w6o7m5ST1M0D9Dj2zxwfXxXBniN2lvYI1X8n8jK6+ATcd3yYEbkgQOuFhmu4cxov5RU6cFLPp1\\\\\\\/hKQPMHlQc3gNw+D8IDcKVWo6tbeqJzqrG8u71hdZ4IhBHB2UIMU9ZrcJdQBWWnNiMIQbUv\\\\\\\/tRszkQWL09ZuAmSEynedHHRILxPsDZ8VLu4mxGdAEegAXYrCQx1cUgMcmNR9Hw5VNFfRHz3oYew\\\\\\\/7DhL8qkCALBD7sh3a\\\\\\\/ISl4nJ9lq+dP46ZnvNcVU020Lqmww9UXdtHP56+SeYbgDourwUU8GC0xPwzb1EV4AvcK1AOJZWK904kQT2wLJaGX+2h\\\\\\\/YqpG+2mZTcnGEly5ued0cEtgwTaLNXQ2KJVCGdjHXDQk2hsAt+9S08SG9BdGzOeptJFTR\\\\\\\/9iJXwtjBFoUYnBpE\\\\\\\/lGp8O8lVnj8mkE+nuohrghZFOjzfOW1Y+66k2n7pLs2A2s441AyQrlzwesZ3otvukSuiX6nm6DczLv2j4+e1HerRb4g7vNABUnO9p8WaMzeSX3BqTBxriQAyl5G4D6szyOZnXnQnoOjH2ScbNZEYvG3+rpvu4\\\\\\\/UblubQv8cslNrHKkTr8UWcZiNB6C+jkdwHPrXwPdtW9u6fLHKGFWZvsTOaM7teznvMKboN3rhyUJ2xwuQPGh32hZK5acd5SAV0HMN3UXo8UzEcTSpD7s0eujSR3srrVPsEwZllACCQZCwgRKMGPYUPX5ekGJtHXZqhdrRBt2dYt1TSo7ENRlqEGc20coBBLrsVoRiMxi9mfHO\\\\\\\/FixG1ONsV7jFGDbG+LFE\\\\\\\/0va8A\\\\\\\/KLzEymeAe7EhMq9YIq3vCU8QNIp3nOc0UpSJBjhdpg7kpStLe4hIV\\\\\\\/D1yGsjVtDuii\\\\\\\/1nzSvBdTslnuf+NpVCr\\\\\\\/Y688yYzz5QPtd0\\\\\\\/\\\\\\\/avcguz+4Op6uIayjxXqLOtxhYWIcr1YfoSjKguuVhXno2uqS3\\\\\\\/E1+V8eCFVzN4GKgudYFrWU5mn2cCG5yxk5ib7W8jqh6YfSfw8dWZnK9qkG7b2c2fGmYCWCa5hpgnY72t111LYSIGBMvvNu59eXv2+pn0XQmMC96nkayXfoBBXmE5Id7ZGtGLHKPdfjOUKE7RUUY6Gh2DCsNzqg8TWzHq\\\\\\\/JOMTsKhBCWM3066iFeM0kzYDtATv8O4ZUGlB6vhry8W0jatT\\\\\\\/6nDE\\\\\\\/UxgtT2WkakmuEJIx1kvuAa6BZtJYFLHFFAqQ4kO4xNyq4Z8lYVzf5lzGQZSCpdOMV9bTpdDYgp+gvx0\\\\\\\/zSaC\\\\\\\/gBJKwD5Qd7802K+A5AYJF52YovorEdDYNNVqpZf6qzHpBq7UJdUQJWyPDQ5VULb\\\\\\\/ganudN82CzhTVAB1IMr8MF8a6xgR4LvyYbsekgV8Oxf7xmjspeQ0OqqNFuM1RdikWJWXpLpVmtEZtk8oVVjNXEalB4lrqzwmINedk7LCJJyo8yaQewzGkDscqniIe8SsTttsHNLK10mbnUJC4SA3ahSa9iJNJfCmbN16GzvSc8roV0hiGMomCvczVnTZJCNA3O4vq0q6\\\\\\\/3\\\\\\\/ISifPwyHSEkUXnRo+Jhhe7X82wB1kgM7jf9M1qD+eo\\\\\\\/4RW1I\\\\\\\/OA6T8BQZ5Qdn44VSS8gFV4VyMjfHFg0i7IKX0fHx7RooS52cAEQUzuLjXHmI\\\\\\\/RzmUzPYoWKtCdeOg93fBOze5QmYJtdWUk5gCe3w3KsZf+Uc\\\\\\\/sI4ENBPrH0oJHbnsmEb2wPHZO+8hIG82gJ4KdCuTQjB+M9RaxftwGTiPXKr2EFXuFqwV8r9EX7CEwi0q9PZIwk+G15JTA02nos5VHyqViPmuZaC5S1KiRzLBhqAe1YN2W7t4PoQCicWEJVoHL0ldQ9zj\\\\\\\/Jmewza3w0qAd+ZWNw2of7asG+iHyCKXEN5rhdyxllB39teKHut1DmxpAGSG584cxjm59RiGKLonjU67uAE\\\\\\\/BHXzoKeQham7FwXltaSkk5tCatBd5OHKg44kLlCjHTv1DZggFMFMZP0USyLbhWLFfFtCpsv8OUHaLlW\\\\\\\/VR+dOk02lLMF8aKq+dtXWRYX10QU4oGmigdboxa6qqHwgMPBI\\\\\\\/gJqzHsw34rPj9iV5TPJBkHCPoill5PqmuGo6SbRvUR7jn3k18FohQ8+unc65Qiuc1pco5fgG1LcrMpS+kbSwEH+uz8N18oFgwCyZHyZWvOpj0303XrdREjubFvSabzD+ueFpN9Zu29u4rWUATlXQfX+YHSgH\\\\\\\/716VPBChFASH\\\\\\\/\\\\\\\/wYPgVkgwC7kAFFRz\\\\\\\/yoAHtMUofXxjPqn\\\\\\\/BFfELOv9kYAm+niyg7k8ZeWtetwQNnDpt9vhQ11BpwvDLBQctFDtmFpm\\\\\\\/KHqbOogd93MKd6n5NDGgs1LY\\\\\\\/rhrhA6Aku6V2e4GHUVszc1ifH9juKTouMr9Fbl+wdNMU3\\\\\\\/1E6MN7SoB7h87AuZGgkdBI9DdjLpW4oraIm+DpcttNSXpkAs16RX\\\\\\\/SkTcMKpLo3lWiV9B\\\\\\\/S1A1NqOuXV8krlLUR2SrkXlKGWSkqijKsXmx2Vy3vQuiNKzuA9t1tmpC+mlot1b8bh3aJuyb46qN5kTJUwYp29ECXM+C2piY9ONAz1uT0bpYa8abtoLHbVhcL2g1EkpK42rTwchwbVNvK1kCFD706G93d4Sb2OFGkKp8SRozT+3\\\\\\\/akyqxbVexLCJWL2Ed1K+390igcYj6VuUYScbDdKZ+fjSLAN4j4Rk30bcSmq2BFRbl+r2s9IaRPq1AivLK+POzKxXtKBUkV1Y4IzvtsGGN0qBfBbKOrF7RX7gLbmeBbxYJFc3pgd8F4g0gRddBR3fietQZNucB8U2Gt8MryXh0NPqd6bzv8P\\\\\\\/uu1J6c8e5yvShN\\\\\\\/NjBa9iVdRfTxLVg46f5soyfjcH1hz+Efj2L+0lavZVS9mXf+uuqraGEXLC3GlW3kN4XH0PwXSLIWOGV24hKtZMHKNBvNxs8d3pOhXs24AuQNdZJBjTY6VHctCUZc5O7GPLUmw1rrNeoMCa0qUiqBrb9RRYkCnzev\\\\\\\/C2b6lLoMV0A7OSh0YXctd6qJcBIPEBNogIQgX9vv7EDuJiU\\\\\\\/hFYbmDd2cLQfUiBrL1FP3KnPeIwOgz2L+BQd8o\\\\\\\/B+58ptaoliwP0v9pgg76Jk+OpDzs7cdtYn\\\\\\\/4DsMArrV2n2re7vNmmZu7xy0nAkMB05ZlRMqHiIZqwG0Kb4tmas+6DdoEAaIv1WE3n2zm+Wp1RsNhk6OsQbeBlP1rMOb\\\\\\\/KUgQUNi2SQyZsYJW9J2c2RFkRSRaLDH4C1OmL5rjAMq92+Fkl5ptOxKS9vyx5xH1amvBRv4pkN9XbknQuSJc4j5C6hvUbXOX3s0fQZbp78fv0wRU6DlJcQndc8brQZrBbhgmeRE8X\\\\\\\/3\\\\\\\/EIKh3NYlQWnJluwuwCS5ja0CRh23FqfYyicqRdbaMC5Z82NOORDusFsumJnWON+nwwtgXy9qEG4OrHHKUPVPiZwndGQE5rX7+rwVI2\\\\\\\/X\\\\\\\/+BbE3+KieXNMhcHVcLAK1au4+Hw1+n3sVuFXuBt7b\\\\\\\/JK8r1ZCod9ma3Z\\\\\\\/vS3fJhBTK4j4aeJTwOqJnV\\\\\\\/ARX+w02HlwX8wz\\\\\\\/N5ecgGzwGBCll5yy5dV8at9TR8m2BCTwJ83WDX2u8w7eoMrdW52VBqugPdBt\\\\\\\/t95XwFJIMNBUNmgI5d5eD6MFP2h+2ZqRg\\\\\\\/FkDJSgkJu+RHczavvPLqpjKM6CFA0WePdHTJz9\\\\\\\/nNCBnt1Q884n68w\\\\\\\/4TUhVXQjXvN3obZ0\\\\\\\/abPtFNyr7Gt3WI2QE6XeJDyvFHF\\\\\\\/khRJ6fXspBI538bHyvA7jXnrbB+Pp73vFt\\\\\\\/tyclzMGKE74ovgrd5FKYGRbco1H7rp6EX5W5VaKrIMH2M9ekl6hQQjQ5OSyekcAac\\\\\\\/vE+1dLPTGrIHASjlqyJTe0y6sISBypBLAdrQQc7EqYzbvp2mg2JilFd98zKMVkAR40GpTlcvv5ndlLqvKs6sZOyIKLEUuRLoIMamQPlaCSmTykKVrn8ULSOuCco81+xbdfR1mPTFJ\\\\\\\/J9cNw2JbyqGlyjAXwK3jBsK07LFQ++exah\\\\\\\/dkeWcJwvZG5HpATKwkNXVtJbLyiBx\\\\\\\/OyagXEV6V+kvmxvrycok5oW8ECoDdqsPx+xpFj\\\\\\\/OPwUiLigT9GXc8ExZDzuMqnJfX+l+47IFODmmHNK0MVlhz8AbYnWn9YeOaI8KgmmJlMf8sLasQ6TmE2ZRbpLCSHOznzKYHqtWSCXwQ7QpzSzL7bdypWozpy7uNuZIbaPd13ijZ7+blfCzy\\\\\\\/I6Wn\\\\\\\/mYn62zU0iXQAMSIx2mFlp+xY77oeQP0OEsWBrUug1te0SZfGq4AhKkZkFSvoc5f25A9GajVsOuj6PDiGIXsAWeVmKLM6QeOHc\\\\\\\/ifdP0ohL4Z0Vn8\\\\\\\/q4lqKLGnzrEi0oeqz1Tg\\\\\\\/zCS01TuU14HLE74jUDHuy5iurwAUh4aUdUh4XsXrno\\\\\\\/jL7XeksQGUO\\\\\\\/kjP60IU\\\\\\\/TddcY8l2urHv\\\\\\\/ot0aGCI2nCLVIyRzb0EGXS5w\\\\\\\/hsjRly83IkBOgk3uQt5NYiLzQyGaE36m9dkms1qIaz2SMpmk3JHdkacl6fEr2FUPWV70CTbkzOAk\\\\\\\/7r0XM9gyG8zv8tAlf+W7M1p9JQTlgXYM86A\\\\\\\/8eB2a3RNnPj3KHzgv9iIRLnn1+U0\\\\\\\/uhduTVUv3ubYFlwA3Z798r6jil\\\\\\\/KR4vzIrLyClgfdXJD5+J0rrSmDqS2Y+R3MfS337AlLfJejmE1W9a+zDoZJBjwmu+ycgDucxBK+Eh0VGwuC7qA4hDl8xX8iCpDGmrZ9fVhSQcse5quRieKCXK7xC9t42k0o3FrAWuXafqFX2P\\\\\\\/p5YBipL3pvNWe9OAc5Ux\\\\\\\/jGNTZfXGMJ51iRkR6wUiw+IqIlr1bKaY3wj9t8oCREcKUG9Ece4p98NuTFAtTkUTl4JEnLo91yjJpQEqhHASgncFdY7wKsWZEHbtHeZ0splBm4dAiAP\\\\\\\/KwpyDRrgNQcJaHoACurkX8+Za7OT9lsBKunnRJHciOvas\\\\\\\/kH5NNTeUHayo+EgMNF8BRbY7T\\\\\\\/8HchSy62Kdz4C26jW6QPl0UBYz\\\\\\\/9Y0S6GW7SBS+mnVlksTzU8iyjCJ\\\\\\\/ujEl2K103RxsrfVbmuDzvCnb3bK2x89cmAYNjquj744pgZgYq2LXawxgpWhtph\\\\\\\/PkDdURf0kKuzSgGVAiQLTKsc+pECoLb+UyA2n3d7WiSN3qO2jY5st2F2Mt31CmPsSoepZjdj7Ds4Zy\\\\\\\/nLiCNRqhpvMJPMqAke3btISNXHKuXC6xvYgn5uDGUVIAi1iClOZmlVXvTU50dJ6jRj9sYs8BqN0CKAsiokd76X3rbKBAR9UURjCd+sWWA+1FbXchZM6s0jx2DyIgzwBGqDSA4QIbWR3hj5uQGBChqLl6AzKozHgh1ovEfjsES6cB44uakJ1QCrW3bj0T0kA\\\\\\\/Gc1TcwJuKBu4GLQHlHfRF7\\\\\\\/E5drnBqUwrF2MzraPH4vN1LgjT9NFMkpykV3wgWfE7Z7O2rBX1MY\\\\\\\/ynqE\\\\\\\/pHCCM8bZ23U+Z2Bw4M5YwUCpP8FNU4ZTM9vAKJG+UxKR2\\\\\\\/HPvFY55Yyfk4RIphjfZ+fRtN0UQUDuuc9LTg\\\\\\\/CxQchEavjVEoL78mm81S1G2tNHLqMpTPRFRerPrt3IwDYxn8VZNNG7eTf3pE3Ur4qoQqnR9NBhbSF4i7AGWG\\\\\\\/mEqpdKxZpk2csLZwDX1pLwXtCC13Ic2JhmwAy\\\\\\\/tL\\\\\\\/zGRoGfkgggcmvAJ9nJpXhRD5gNJb2rR6fqvEvZDoNhI+XACL7CV2K6TLsaZoam0wQSUv4Bx+ngcBR2+ns9iN171Btmjj3MVyCOgND0T1LUYKy336ojxYpBN+JzRFqmsombZXZEdgOi\\\\\\\/Jn7God2vuftScc1LsdyTaVdyjtqJrSVYgiNU3RYs0hX9XnA5eZGK7Ue8EA7FkXqRWEduOmFvlJvj3sZABMr+UKBkThBFHXFBh6xhMY+B8T7k+ZG23jlgJ0GRSgcTBwWPoSZLAJwVckyTnKVrc83tx3Q8gPKJOTGaeQCCtS4x3r2J+cDZmn7CCVl1Vlpwi6UEgNCdjlDHmLk9bz+FcOSdGbK9O6KO3rq4sL9BIHyda887N2H7B0vVavHPXnFv4Yg0dWfB72fP\\\\\\\/yodwlTd9Z4v1ZxzCxrpS5wNHH9U5bSRPxPzBqTN79DKIrbFIp6oVomEdxKCE7UqUjuAOzMhv\\\\\\\/bdvchmjGdZ0pzK5rkKOFjRwFe3P0w3Re0tfNPPnBF5l+7aKmtJHrwUF3LA+kpXPSDPe0cz\\\\\\\/jNmJI2K2ytAHK1ZQDFWdZGzrAfOizZONa7dw16d0eEZUhI7w3rkOBsviwwNwjU2RpsGDGBN1163FZkK4mDL1o0cZWETwVSxCPChlzGuhashPhvaz4c+Pvj6RMx5oKRMUpvc27u\\\\\\\/SI8lQ39i+aJ0\\\\\\\/kJi54+XDjvfnv4Q5aN\\\\\\\/fxmc4iISpbPpzH7a4aA5BLhtohkd0vk5JDtpQKuajg+GUYfxLT6OABROEx0LpwbzYSGdDH3D0gU21g9PQABLNNoXZtdMtqT4Gi4Vz1X1iA33hXr1rKhCXN5or9CUr0RMh8t+t0V1wLuOG05MaOqZ2D\\\\\\\/WJ1xxrl545t6RfUUadPX75F30EVxFNE31jUbrK0kNWI\\\\\\\/3gTI2kFxM91u2O7eE9DqRcFmaDDpU8BfeeCd98JiBltfbCVho9Dx3A\\\\\\\/do+WjRVVot5grAFSbgygHVeE8X\\\\\\\/+WW6fcE38GpbIqsQajmNvEu4bJwBDX83uYj23vSSXLXK6\\\\\\\/ruJL2PsPAPy8mmZteR4WWZ5FVdxqdPszi2DbzeiRr7rV7t8qtINthuSkbFt2HOk123R9EkS6SIuH7YDDl30wfDD8+qSiXZhygBGATAaQ2pQ2maq9IH+p2dW3rlgc1XLA3WaIJM+hPd7nPMuR2k2XCevVkTnMDqVWgnw\\\\\\\/ZnMdX4qiYhehmoQpEKPrFmMcQaHhlaGakv\\\\\\\/TFJFJY9ifUWxrUc3I4KgG1N6XInlkdi2iVY0roVSQB9BS\\\\\\\/B4XHr3XRbxbZJGL3quqHxp5sVX7CvXDilm\\\\\\\/lVqym+\\\\\\\/lNUJkB9uhOZ9e\\\\\\\/seRQT90q+97kpy6\\\\\\\/1NhXmSZWiaHpFAMBY+2ipsPN7ADY7BZT42GlOHVaAxYfrcWB0GUOfxSGj5L3w8dBTJvbZjG2TmnPzDR9ZPXk19YGYZCv4r9cj9UMMLEMVSqym1Dh7pxlxbPcBSHfeB2LTu+qBSNzP8ed0oIXwkY2MHrJaqYDZY2P7cToCn6M6CGhTWXjxLR0vFIE8OY7L1UFssJet1mYMh4C69De5ox5j2ujaKQpAOSxbhNhQm+8rvqfTJ6U4zcLh56X5ZXwQAlKtS0cil066UX7gSXNm3sH6KpVah7CtLLX4o3kPyyIH0Af6la2gLK+6hgxBTiS1mCp6vys730P4S1Iz5Svdoqe5cAxWFfLOtRsRb70k7Cl9Hp+A8U7KlLKc6EKHryvdvb72Pnjc3z0OZWEREBTRJNlkkj2ve3SoH4+zXUfumrrjWGa1fTo7KS5x21tK\\\\\\\/JIaD2ornZ\\\\\\\/fRJXdxW8yJnC3Ckl48xCCFhOC7zR28Cf+FdiwjCoCazD7FJ47lNw8WRuYt3FyXvg5R5OWM4nIW6ZPgNwJoFpQx94mAmukihkFB5z9osVpirySGDCbdtP71S1BM4r27eHjRT34bNHrqUCH3kcz2aLVgO\\\\\\\/lt0flh9\\\\\\\/jtFcNdb0GADt4++to3Q7gFGQfIg+AgUnEKFnh2jOlTdayeVOTRkb4gRqZvLgfhz2QltKXu5+4YNYec81q2+XoCPuvatvfY63Zy7rDSltBEMM5KgPGuU6wkl2yZtUKqLIZHTU+f\\\\\\\/A5is7z4Y+8sugsOp9biJQrufFwKXsHs8V4JIQp4DQC6snHqBHi3kKyInIeN\\\\\\\/WkJd8uqRFoRjU9j3bLXy2HAWLDk\\\\\\\/0g3uJ\\\\\\\/o9viFcZLaqPpk3RVZOsuhBuzNw3lejVBtxh\\\\\\\/94DV9rtTUAam8vk30fokTYOTVKKDZVAvgtp1YaUqayzzhPNsMCS3MeuHFtVwn4VYgtfLyMGT1IcymPiZ8dIJIQYQMSyP+pWFRXW70vG0FNnOiydWdcpykfAa2MrFtPzzGr82XDaV6AVhNZvYf7MCr3KkG7CNq1BjyZs\\\\\\\/qMo3xUbQ5AEcCJ3CgMK\\\\\\\/1ai8BWmQKKGE36VKW4VNmpmY2ukmKs5HFVUjpIeKDLmkpDSlSDyUfcI7nyVcwo\\\\\\\/4fK\\\\\\\/ze3Aoco1Nhg9PTTl\\\\\\\/ZFue+OQzfMDrovXGhLpfH2ThrFdkkvVeSe1QZbYG8nHqzUxfGrX1PRKc0rxR2nD0IPwLMHvOSqQ+FMZ8SGmXEmf7vKuNSqpJXininZAG4x2AYIMGcSFr9TpafRNQWAQQ09lYR29hDEQ4v5o3DFwe5P67Vry9E1qXV38GQVdmQmKkN\\\\\\\/+EILx144K37846aVH5iF6zdu\\\\\\\/a5eG1m2jr1aZHYHqfz9iI6f0hwlG2jakrNtjJD4qSf4qj+Z7AmOUxkQm43WY8wuWGMQkQ0fVNZUghxfcTzhpbvQCmpVJ\\\\\\\/7DJqCyaF2WK4CuUJN6Z9smR29szYNdLQ0jADK2Hhbi7T5442sDY8GX\\\\\\\/hD5RsGzueZCrXcG8a1oexuHAd8r6def7ZkY1fwCk50pQ4l5dZ\\\\\\\/FQMObuIOLuE+vhA3qtYcbRjWXFC0bZIvnUj0ZxtOVJkS1cGYDkCKzIhYHT4o9Hj5584qPyzsS8OLKjWRn7eSIzjE0KbeTSK8P8hwSnitMHTplqPbFjS8zh7AKy4ZTEc\\\\\\\/AyK5KoS3QG\\\\\\\/fOFOts4MAzfV7aX78q\\\\\\\/q0bs7JOitZbBtQ9twr+UA71nU7ZJqeiqI4HQxGLvYl3B9jhjm2QMigvm5C9wijBrgXZU1HtudTG2dwPt0L3oTBxhqwJhDDERjB8lw37nSaQsP+64xJ28NcrKyfqFq9zWsmiEA9EU+Srqw2t\\\\\\\/0v4l7rgC6gk3EX3Oh9AWKtaRlwoMBKNXiWZpPRxJ5\\\\\\\/OaNhd\\\\\\\/e3QWGq5faaKhvIg6dBUbCE9fJh8OkwwxzI3A3e5heiNdl1et+LSeHt0at8K9WONPuOPzagkpy9S9EfCv8IqYV3Ifrrbr\\\\\\\/BleHTUmSiezZMjkiL\\\\\\\/hVmM6OHUIaB\\\\\\\/4Dfu3FAZXCZ\\\\\\\/tMgLhUZLspEX649Z+cbwCWv0QwsQpA17XaJ0L+vCiCj14ffKdUgA1hUIW\\\\\\\/MMKAIJmen78cY5RCk1A0tm9npxga1MgttarLh302bsO\\\\\\\/pudPqPAfdxqU6URGMBLPkYyKWHjAXekYiyKfb4QIMY3C+t++SGUbtmhv4fyBYpKZCZc195B5wXRR3aGzrAndkMHXLLxLw8bJGkXX5x+FGYWwkubulWGZZACCEXKCoGd6wHvwN10KmJtyEvPbUx8Gl9iCGlfMCFMq0ee1ePVGqvUUvKUxVVwonYEtgGPD\\\\\\\/xliK2LGd7wr92vr\\\\\\\/iJoHTxFWmDjaflOxqeYQyXDYDgKtpTVQ7KwFDukhm3+5dO2gdG5clc3cj+zpVtFktuS3MkGj4Aaxe2I66locFjfqa6nePJUefekF3ckqC825p447Kdedi78T5fHi2y0KBMhiMiAcfqDPnNiDi9E4yfRrqQVuyvH\\\\\\\/oddTWJqBCtFo1cqRzrsTI66isu+w5mf9eLClWnkTJH2gs\\\\\\\/LofvLIGUkUDUNnze+dtUChIuz8pTrk2tfClkeIqg3JdAVVdySNRpBwKN0DbHsJ9+QeFPvX9s71fsJIsWwNMiwiU0qgyynCovJCLciqXEc0Kk8bLEvffBSKb7w9Y719Q5fnXqFsIcxix7itZe4PKdhrm2fVcFjV1BPvHSNRLMghw124Yrg0EvbC1u2EXbE+Qf2Y1DHf0zlxqJCWg4Vhu0CY15xAM3MMSea6KSwWKWGVb4EbjSnjcK2H09zlWuz\\\\\\\/SxWBheyfoWC+HMwUzg4Tkju1B0jnkNZVAF4+JmbojxMNkNaLHDwM7NWQz\\\\\\\/H7YFzHrTaV70DsPtPB7ABzhFo3rvGeVOkCtJIQSwRkRt3gu\\\\\\\/6n9KD9YfPxQWWgYt\\\\\\\/Xtd80e48y+3pWSRBchYuF+hyfCpa\\\\\\\/VzVPQUkM2hYjh9iZpYm5yCsWwIUUdeQBoXb+b04jP1\\\\\\\/7XGhfwhjVGoiQYPABoPGdbuN2\\\\\\\/JGe9a7LMPf\\\\\\\/DijB1CEPVoOIBYyS6bVfRSTzaaR7f2OD3610p3NPe1w2wp\\\\\\\/IO2T9FNEwjdLzyb12MzdWWjnvgK9Hp0G2+Ii5DiT+4FPyZwj5jljYqDCA20Y7kAHsxC64yoSzjblDqOMfN22CjOnLmNHHGGtIa7\\\\\\\/auLvveFUK9HSxvC3qoAMe1iUGDPkJ2yRKTWk8Iju3km5GiXtjMQ1yFlS44UmSuPgkfaZ35i5xxC5ewQ2XWENB2Em9DIjYW59BiGRPGrlIpPsT+Jta4Dsp7HVcf0tJRSPx2zhsDZ8AQ\\\\\\\/m5+1hbbYHqZmEv7ff5UfPhxDsD9id8wQqMzCdqID+0MHaQqXAba3rPLkxg+6J3fCt2rmJ0tKkSGHzwW8VFpvJdfaarkyDbhcpSoBy65zPxzlf9IbV7hUYDW2cU8Q+1Lwnu7G\\\\\\\/I2re8ZZlh\\\\\\\/nimfZ\\\\\\\/wbsV9VYVq48yiz8035W2H+Jox0rJH93OOMon1WLV4Yeiht9aHhHvi1BOsoef9exQGNajl84ex2LoG8gi0vncsZUK5ammL7Zz4xBwNDN6mU5lGcf+vZ2X85d4CNesr9G2PqTcj3cc8otyc2geKatp2PnyPzuDwRd828KbEvIDaPaLfT6DHubufwTMthwlQetzf0saCFcjYzKB73Vf0t3g9xWlwxxJX2QO1AtWAamQiujcqPrLcuiLkak43um4oL+Mk9qqJJkWYhjhbOH0ybNjnlh6J\\\\\\\/Bd9Fqb00ndr81CT9aeaQZLj\\\\\\\/ETRl9yXzuSlMKzUj\\\\\\\/BatES+7jvv+VofSdwfznYbSP67VwwjlrQvY5ffHyHeIC5+kPcPVnqez3Rl5MYHSvRrFGnEAT3JKsdKrlq\\\\\\\/3KBWZ3wjejZ0fWYMkyzWqOVaTE0n0t8I1XM6BTqjS5gn8imgmggJ\\\\\\\/HV4I4m9\\\\\\\/JZmFteQMnjsvco\\\\\\\/1ErWGrFbfE5GI5zxyAaz6OyvJ8N5t+8yUpHW1ZUEJkG7kJBd7TVa1JS+B1jy41VWM+wFgQ+wcqtV1+RDgkik+BKDXejWkbzsOr5O82Xn3jYWXuoG7m8DqFOU1+szDnTokIqHzv1qIRcdly2vSeQpjHqkao3+jaOpqNSruD2iL1as2wKqwxVavPM0OBQDEVLoXATj95\\\\\\\/0P29xhju4FtYPM+OjvuZlD6O8RAJDzIOVhvrOzgMuc53zi2wEEUnZirIbX3NRcfMjQKOIBfMFbskyHKxK\\\\\\\/v5tbPNw0H2qzjOJruZxPIsWkiK5ixjS+PT2\\\\\\\/xpKB5L\\\\\\\/hr8qVrK4Ge9r5brsj9BXz6lZMGOMIyKjo\\\\\\\/1UStMg5u6toV136Xbeg\\\\\\\/jLWvIzD03O0ppRWCHZ9YJfb8Ph7XzuazVhxhAtZ36XK\\\\\\\/oNBmDDN2wZJYSmb9eWrQY6HQHjN\\\\\\\/UqS8tZIKTW7KGBSyq6SggcNtzd2D5ZD7GJ+rCBibbpUCIHy4fkbxTtunwx3N76crJ\\\\\\\/otP9NnoaMP5XrphbiIaHlElcsfmoCXg50ncp\\\\\\\/iLTc757ODo6C3nXHygvUKNanwo21CMcfYZHzmTrmhnce1fQPCUKPk9N\\\\\\\/O\\\\\\\/1cNWyO5rhslPRW5FXQq8JMnENcALwxCAT42CaY1oVMjGei83s2qcH+MSnD0X4oK0NHQtlZfva5NLgIVv4ywdLXMSHSpghULmuTj79FgTa0hmaGIqIdFHxNovUX5ffgaKyhtC8+8wZ64f8FhoifvyuBGG97+BWSvIJOgjhy61E8BXWB1sBeBLcTp6uFxctiZRGkRCeiA1dUMGgEQXBvXrctd62+fpav44ATOc04p8NLLaXQLTmwohugxqh+FQiiuu227CO07h2u78tin1N6wwO4FISTci5C\\\\\\\/OvAqqioPn\\\\\\\/51n7QpUEL43jf93xk3Gpd722XRQNs+TFY1\\\\\\\/T5kXOxY9FqiTqVzAlUvchVOJujgkFJ5wwnRnip66Q87tTczYJPUOOKz2czGkUK7i8n5LGb2c4rDUIar3krWsnfArr4b\\\\\\\/HPQM6WvMwIRZbtB0f3PBkwJL7vKFDf+J3513nz7Bb\\\\\\\/TdMcgB632laIHjiO6Ls9Jjr3nhJtqHZTFEcWiEVSKTXlJHI\\\\\\\/\\\\\\\/gxSHWuLUPz37yptARXD6FkFJEsT+ADzSSNEzIjrzTrz5cdZslsKJmdi\\\\\\\/Y2AuPApd0vkhgJjbdY3vRNdmK2YTz+ye+q1DHcIStuIa4pi5HBwOQnbYUZzJceAnemkurzxP+v3oQkD5dVwtHnCtcTeDgFvP6WbxS5qLoLAAGLR76fPruTisZr0DBtGoq1ElT7P7+DSW0phsnCBEWpik5fA9BLNs4YeWHrD9IWQzqbMQ4na0lviSeMxRMZGYgQY6GcY0iX8hGOvCJFVNoKT77sbxD0zUU1pVYU4bnVYqcB16KsX30ns3soipwSY5u1owQTFjzeSkhuZX+o7\\\\\\\/BbzjfelSSpdqjpWK+P7KZKiqMhJv7vm3svStqDd6PG6Slf2XavDV7Td2clq9Kj87\\\\\\\/IZFaj+5s7XMVF3\\\\\\\/1w0Y7RMbxqQ0iau0Q2sNKgXh+qTjS2\\\\\\\/aTLT0vv41MPTzj+z+LozHd8rDOjqUeCWyKQYvICGDMrwOuiMxh42riOimxHxASBEb6SKorjQIaIjJZQ0hBmLjJovAlh0179NjVDtNc\\\\\\\/wX\\\\\\\/4jZzDxmaqAFyMP3aN9CAWFC150bpRAHCuCjIe+HFlPEzI5kiDQh4jP3\\\\\\\/455OzV6N+94VEYcbW8HR5TBfpwF9Tedh1pQLOFXAiNhmOjNxWw+k\\\\\\\/CX22gj+028dZAFSCHyaPqh6uKSDstF7i0GZnQur2S3r3BV4\\\\\\\/Qv5nfqI0kXTYueURhWb9Ow1BcFEDy6a897YfoD4bM5bmzBqUePazYhmb17XAwL\\\\\\\/Opa1xERXrEVJyic5Rk3akllCjp6ZYDewq65GJjp9Lp\\\\\\\/MzdHlOPjOVRNDIHmx0myw+1yVO2pmwoP7P0AAv\\\\\\\/42p1llzVuUvYy+UUq3S\\\\\\\/kCTHQ\\\\\\\/G7PpL9HiRPyhYWbKlEfeGiNBWSAHBjxr7Lr6IboZNomNqq2\\\\\\\/rk0uFkauQgigVYEVPs0JNBIfiBASNdk9JPU0p3i0wh9xhlhrGNWx0SQajO1lqOArgyEZHu58Wpm5ycRZCNbXQAR+qQikQAwfo0eusk8f0SUHxjPG9x9l7Ffl+gCawxvbQC+71decGNPwf3nwEIScdxL8kbw37hRGSET0AJeBpM\\\\\\\/Q6+OwySeoXw8jUGiOd1At6DkvqwvkbhCVCdw\\\\\\\/cpfnQywJcHBFMqXTcEFWSH+4hojGVifhe4hPLZhdmUM6SpF3UVgWPfrUQiocOuJ3nxHYpkzFnEFIJm3iUcclwwMqaa+R16j1sz1hPMVN7HDv9dM1hfN8Bszvho8oVD\\\\\\\/yxE3ftTtUv56WiEIBtkapc\\\\\\\/IbpejJ2\\\\\\\/kiW5CUe0g6ZleLcFM6mhukmegYGvAo6sAlVG8NBbgSCaqdhK5dla6Lz\\\\\\\/O5IrrOV5\\\\\\\/dw1a0G0+lPUmaKJYQcng7Jzo1Mlt+HtltdUp0a2H7wfVJpAltM7G87mt77Y4jIzJjYxz0cKdDFui2hhBx\\\\\\\/PUIB3+IxEvcdOVFt26Lyp8kc1NOxR6TlYXanKuH0351eueCllHZ\\\\\\\/JOGvv4tfJxpHCzebgyWGXLNtKuheOWidIGiCx0iNIBFJ9MS6a8Hereps6\\\\\\\/d8r3AgB+jqoEekcMIYOAcvjGWkR6LgwiU0Ft2rWqbF9+6Fcn9pBXBFreD3SU+XeEYIx8+8Rj\\\\\\\/PPPxWkp02xshPtq5BjNB++me8x2sdl+Sbl691yKuseB3xi1EBTEHGbEQ11j6DOflcdOaCA118\\\\\\\/AHFc9FpcWxa3ZZhiwLUcWGD4buqFmbgWo9idMBrkZqzoZ4UiGuxmg9pS7EEyq8h07XWcXj9g97kRyFve\\\\\\\/GJlwv4e0MO5I5Xuu2VHXHQcvec3oXDv6iXiPUjHkGrcOdYugrxWhZ\\\\\\\/1G1mH41wrVUGzTXAMEiBkUYTChQBo8FYJRj+bOupSnss+bRp1cblsnIejctE\\\\\\\/sal2Bjaytc1KPoNr0e\\\\\\\/xBdOkeCN6lCDl6mQrZxRMxOC76\\\\\\\/XT9rPkbpKqSSWuHZbxSJ0XMWDB7gjMhhLdNHI0WJP3eonhGbj29XUP5mTGconeSmInrs0sCoWcc5DXxtD2TbXtANG3gZM5DUI6p20uycXrApYD40A3YJ84+A+ik\\\\\\\/u3zVJUwNYOBC8+ms56xIWbeO1sgisKk\\\\\\\/dlcGNp1gPaDZUD1HapQ09XZUyBMLzvsuRcCMxTQ0YLGBhQfxjIc9dl6Sz46whYR0gV2W8SM6nuyvjbV9iZTxrCQArZk06XaenN90FOeAtHeFZtihtFwsHeDdAPw94TU4hrSGGXyA\\\\\\\/6\\\\\\\/pfY3g9J\\\\\\\/rUXpHcN7dxKjjweEPaYgsm0Yg6o88jv1NIYoEojhGkUF2gn6jJWGC5JAKf33rSPvc5bKdH13fE8Be2bkfv4tfRo+laZt2LoVcKxtBfamKzH6\\\\\\\/iOxFlnbves9c\\\\\\\/s1lCpvGVD7Pbiwe9IHbl8XsRm0yqcddg1HDnVyLgAXCQtOoQL9GrTwd1ZnhLkI3Uy5gLKIQuLhDRgExuYsAlBagrPbINb8JMycEY9dqC3nVt5vicVbINtuaC8au\\\\\\\/naqjkEnx2eVDfr3QjPKAbVvTv6114MR0zQmxsuYwlgrX1Tumw2L5g+FVpFq9verJAZPXLjtm6+eXDdxxU2f0tjrqvBkYb+UkmV1L6\\\\\\\/nSrEtExawrp6SnnRGFVXIIu0olUT0XcpNgUX7rm49T4ZZ63YLqqbjA7tJLpOa2hgfm0YYUmdJniM3WCU\\\\\\\/56q1H5IRVj1OpYv2ndF7sn+4Y78wTR2InYyjHn85EMEYYuMc\\\\\\\/wKeSWBzaDauWCh2KkXjKHU2IkmRrZIkGuSTIed2JFtCBmvsWdu4CvsU2H0wnUtDE\\\\\\\/sn6c8emkeg\\\\\\\/rdVEWYTUtoclI0h7pO9+awjdICBThYS7mFOPB+eEZ\\\\\\\/tD1r2QEUI0u2qvitRdcVOffMNfwG0xsejCpQ50eX\\\\\\\/47dTfAZu18ZWewsklXHbIcKOolQoRbsVV2+gh1GQ75t8Vxtg8KlSUg6CoLYNqLIgarLRCQc9wucqZdSOqldNODffiVGNULwu3mkXFccT+A8A0Jk280b7hGarTL1bNcPsQs4ryQfcqnDD6KZ4P8aXTtpCRKORSlZxegE2AqlBDdouD02rr0iESGn8q1+Qgnki44jFHJpZ5d85ZwcP3DNjq1jTKmI1xdBmRPuTZpK+o6hlXAXyo6O9uWHniheQ+1nIe8l6lWWfPYdN9wt1qxFNL\\\\\\\/zeImKzlZShzQjuCi4krHWNlNa3W4tWbmCDVc06zAY3lsCOFl9IbDkbO8RyVToLC9IRMaiPpBmSVKLX5G69jvumnRwvqv2uUNDebTVymG3TfFjPc1+vRIXF\\\\\\\/KyizV01bmQOPKvllAtVxlkqQaV72+nQ5Ej5tVn4d9Q8BjtGiU3mT22JR1C8GVrzyZp4yOYDtVy6h4jnanL4xhwxM49NtqmCtTrH5NDA2E6Pgs7SHsose7I4gkzp4qoAOtpdY9fWEEFb56kov+DXX1Y6EltbVvnx9\\\\\\\/CSOR2d2gBcv3IY4ThEoq\\\\\\\/VmjDBKI6oA1ytpQ9VkGJedE7Yu9eePIgb84XJrBo1qPw1PLI7aTqq8i3GdLLQ5NChACIVj\\\\\\\/n9gS88NN0Q6RYqpMa8vgoV31Qkry7r5Djeh4WWPKVyem8JYTS8tArEa9pvr4FxqqMKjKpreT2RvGV8FcNGeLXugR8i1qzqUxN2YtL8qOAMqGnYLn5ZRb5xLx7gcPI2olp7iHhOv9S5FmL3cRY7n8WsQKuIBcWgqNtjoyr25FAjnGI\\\\\\\/533xokawMWVGESgFt9Bzq5TQpE5nEgFUTOZQkOWKyMN0AU+dGIt9XSbp97D1TMlpvPnYo98r7oUMi4L94K\\\\\\\/izddk6b1vanJjxFFDq9CeIhnWEBvezYwxau7vAa4P390UAZ0RVpd8rY5B6x9LANCO2lhcYOPWSp9klcCt39UKhTSY94WsWkkxFA\\\\\\\/PUoPRCESZJZl3wZrQhC0CGvtZ\\\\\\\/YbASkb6WjLZKXlvkgG3ZK09lc1fbF8C5uLTMLLspbeCQAjOXloHXMBSralKv7jTsDBDS\\\\\\\/k0tCRTh4qHRZE1jZepecvhjTdNxuk8OUru2UfTst39hoKO+IHbHF8ArAq0CEg4GHJiuvyg6qQkAy\\\\\\\/ud2eYXGLjzqtbQVm54YKTxfWdjnALRH9wyOjyhNAe92tbGzhxYUg\\\\\\\/eSAuikWDfrd1OLgk7puemD\\\\\\\/vR5mFRpdfeMUyIHKToWlTdhURZDA4ZlyD82uhcVQkZdCBYeVPbbLEpeO0+IxbMQ7x15iCC+3x90gHzu1GnG26exQmIawLPzTbHQHcqq7kAqxYiq7EEq92CzVvbhRtNKpK2Ep\\\\\\\/F6xTt9ynhfp5Rakd+ZtL5+gM9LuYhpj5XyrAb3QeRQf+YFesLKXC6OEehdQtZAPgmrRZJe+TCqmE3XL8jNvel668ZpaCYE7T4tFMze+p6s4Ry\\\\\\\/0CnisxgnkOu+fh1nNHgE5CCcThaSVkqYUJNsZm\\\\\\\/mmzDt9mXD+Psj2zCAszoYUXUrFzjzzTbu9XtA3UD95091BL8QR437KZ25xfY0n2jYeTmAo69VkVQ6UTCI0Q\\\\\\\/\\\\\\\/DDN1\\\\\\\/b4RKdDGmze9m5NrzWSCyUkVgqSUvPTx0goB3e5Nk4rIsbYTsM9rzk6kPTkvuGRSZBPf9bRgo+lguW7uZOxeQnct\\\\\\\/EK6bb6bXNJmR9iYghIHaM1s3JXW6BQUQOfSHMTbjyrw34YsHSwIoO6uSMaxq8hb9Idjx6vflRXhxoIVjhaocSDAekA6sQe+vwFGhcyBdmhoGfSqwfWYMLz25G+AqaxI81GJdOEo8lrqXf3J74r\\\\\\\/3EK5M0C5pxR38cdBZ3Fe7RHMjG4rV9jDcIf3QZpg614ZbdVJVsV\\\\\\\/91rE0RdpwKtl3S1u9P+dhqrmh5QUw8+iLwun\\\\\\\/BTeQ+WBKN3XDeoK5eHk3XMnNMyrjVeU7mdRC2HP+x1\\\\\\\/29dZByGLqiU5tOm6Dw46dJNAyG\\\\\\\/YInlwoBvH\\\\\\\/DAL73vU3EM46hhpgUAeXWxMX7Efi\\\\\\\/np0QjIzEWbFS2w\\\\\\\/bTwqU6yzbQFVao6XLOfZ9Cr8xaAfh\\\\\\\/1iyXWAieloWAjA2uy\\\\\\\/uo5iv0Wx1iNuM6QmkjKv4r2G3T206\\\\\\\/jKGtwYeBqS6U0GDX5t9JMQw10eQuj791HJGjcC3ujus7fl0bhwjKfadZvlXiYYjfxblUgsXE74PMyeggA0ttTJEKrjaOyfGr0juGq\\\\\\\/C0SqT5xgaMudqJaNwkDu1+UD89mS8VY1n\\\\\\\/xOPHh553gsGVS53ywS3WmaQI16D5pUyV2b2rPbAc7nvJMjfXq+gyBLET5aKXysrzhJVCq7GV5qJnR\\\\\\\/q9wZCfdZqCNhu9WOU+xu7nFQ6qt\\\\\\\/BxQVpZa5Cg794nNr8mbFnOjB3thmkjk1EL+r6nMfShiYZbQesG4pIXPB5s8NRudIXkjXxCl16sDV6wJvmNBaSY0ckEXP3jO7qnJ8gEzA290AxHIstayyW4j1yqkufRrB6jdVYdKRNUse3o+TY4Xu9ztd5loM+mDwMZcu\\\\\\\/r44QpYZlNc2Ok7X\\\\\\\/U0vSw0xd19kfQAkT5TTPZ524\\\\\\\/vd4M7Ls6UuJIJGXCEcdI1JaK1UeOQ\\\\\\\/JY2y0ZqyOeNkHEzjDvDEViEJ2HP5sCEjNE\\\\\\\/1c2D52tBRBVchMMdTHnfkjRdZ0B3b9jANExLBRWgWPpT1zhwP1KJS7XO56kqZ6+NEHwjCsQ8rVq\\\\\\\/8AqHp+n5a9JRx72JroMMd7mfjEm6UiWzLP+tcd6voPATWL75GFUAisQhDlBI5xoiWaoyVGwJo+0zbuD4iztfjtoQBzGxLRQQtQK7QRj\\\\\\\/cUAVcs\\\\\\\/VE1HJSHUgUC+fWfzaZMLwjt3oaT21klg5Rh5pW06a\\\\\\\/gNx8If9ZpktxFZ9jNu\\\\\\\/3q0BYcruXjfnO2vN7NFKSVe3\\\\\\\/BHGJgulQnRGgw6AoLZNbiIpA8al399w1WemYUUzYHHazNvxXOGuXXK6KMGJcw6Xe6\\\\\\\/4Q79CB8cNU0tPyo9vWrIqbxP0+8ecEemA5ML79N3wYhCCPpm8gRjTR6hRAktW5zekJwIwDPzMmHI661Ea5yHkenrklQI7Xb2ajZ5KMVCfSwwYMNllc0ILhyxey58+M97fJBtJ70M0XeTaPp60JwuDoPPCR1kbVPwsXUZyfBSGgtnjE0ME6BrdRlZFr9k1rwPN1PLw7125mzD83sV3O5A0FOgwPWbZNpIj6CBFeIHXLInbXcXjiKZHK7+QtDov9qJoEPcH69gwoTCfZjkOu41czoK1i0u9Ko+oE4RSbH1QDXcGFL9uHqFP1YnhhTwWlappOu6DjTYuK7DTjzk\\\\\\\/AeSZQ4JmFuHLIerlIQz5B4COPB1VWjDrXsZS5tdyqI0sdDXYHUv9tJixxl3i6\\\\\\\/NcF3fwEArvvQioROvDvJ9lCMvkzL0Q8R4DvWGSAawnm9yDfby5aFusS6RPq7qVNUVMr6m8gjrKqQsWuFyRLDgT4VMKyem8Ddz9f7l3QIS6g2W8IRVENSNjFH1fURzFusm0GdRH8BlYO1qbNEnU0ixmLBvX2SGhHgzbOswIUwyRDR02qRhMaNOPMeRxdcTYmxbPPRM2vMhzaF9bv+ihj\\\\\\\/qcMrrDGJELJkrv+EXn8LBrxmmYeEYbT6o3GbH9D8NXF9LXcVrrRlgQ7CQK05m1JItodjINHF1mt6anp6119mzxzR4TBW6OUuqLelFUvl8bcGxpzsg9z+BOgTOFk3Ka6srBv3+6NLSiwSaggJ\\\\\\\/ZKHT9YGe8z1j7AIbZHDBK6t8OCu\\\\\\\/zun597Gl0T2btpZBk\\\\\\\/1BI4xt7zNe06c2mneWxe+izRZH\\\\\\\/eXiloMb+gCEKoeJLxU9VCap1pWQexpoLIBQFnEGgozKpRt42UyjTMOUVu00GMcDqDQV2DM8PhhhN+usl0n6lUFH+EYQ8EStbITmzWt6ZF6glATEqWhbSvM8nU33zxRTZOLtIDbRUoWiituYn9LwO+bKNK3WbrVRcs\\\\\\\/qNNRUmBximQ\\\\\\\/EnZ6SSY+jAYR\\\\\\\/j9LcCdnynHc90Jl7xehKxsUDNLmm4vJ8Ia4TH\\\\\\\/AAxLbaHDmVNAwdP8PDa7031cdMOjZ2ZebQBznF0HeCxdFxMbmbinVpT2Y6uVhW5HiJXzDyFubAAzIkbrulWw0rmjMC2HoJOF+1vjL9SoU06GPKOJGxdkqq3D8Zi8TlRuIKp4I2UK3Lz+yx0cscbbhNtv\\\\\\\/ItlXuZdN6XSXlRcniethzAPQ63KDtdM61gX3WuhnTZ\\\\\\\/et9SWgmtUj2p3haMDUFKcQWtsgDQsaCESOtMGi2aCDZf0JrqpQdFcR62vRWWq0NUDHDKnbXY\\\\\\\/WEod2y7tz2kMylI\\\\\\\/g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XqFf3e4hwpf\\\\\\\/A0hN3FGwPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e6a02863407da1b90d7859c61e098f180fcd0c0f6d0205fc3d4a317e07a90bc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AS5eFEXB\\\\\\\/OVvZWiRPO0vh64f1Go2r1CDImeyNv+FMBWA+iO10d9e2TnByT4ZsDBKaP24Uhqsfze90FYTOSLKjbup5Zjo\\\\\\\/mevHQbwc4vBKfc8nVDoN\\\\\\\/rVVkQSLzvngCPgjD8rA1vWyHeDV9uZwrqLOI3oAKcHtfZBiZhn3zIIBRwZBP5041cXEfC3PNPIm4SdkOO1C0+qSVls3Mbf\\\\\\\/QiP+424qImkpGT47IAZYP\\\\\\\/M\\\\\\\/7f6SKco0VSD3INUI+g+tper7BT7Vmqr15V67pCpBByx38ekaOmYyvkIDf5ZvQ1NP75rPBcNgdfqmf4Yygru8wdKG7KtLz+gDHGsa8ewxgoZWj+OjPWxBLQ2yDpkZCQkThPOOJbi3dKwwe8cdjqCgl1xL01aPgENF0uao5KwV88+QUcdKmO+ABC6RsgkEsaVcMXgbgObT9GV6dysJIFKIIriRMuzrDYTKI0zyUAISqAWkoeJR1UFz84f1nem\\\\\\\/sK3Vogd9MiXoTKjnHVOWWn2rA+XdBy2t\\\\\\\/SrSesBW8ljcNGqTa5sNc4ADzmr6GRHhR0HPO4gm1\\\\\\\/G+5maTs3G+KQHVbiW18R\\\\\\\/y8k6U8NdDbKAbrRo8RygJ5QXIGYefJneUKCbVDLRohd7Vy3yiAbfO7HgimMrT7CnGP777G2s5XFidh96ViP4KpXj+7K+z8ib7wM9ha\\\\\\\/wMERf1\\\\\\\/XTHvlb7BKgxolCrwPt4ewES93Iw4G0Mf5vKHcbILDrsbyo09KAuj+Cp1\\\\\\\/2OCN5K4+dQW4iBQKDv7tWI+yOcy6py4aGEhXUVaN34jNVdkxjttaCo+gs6jZNGeLFGxSoPRWDKni3Y\\\\\\\/nap65A+qtdIrd4\\\\\\\/k5\\\\\\\/cXgbBytrddk2Wj0HKUd0DJVZtRmYvsmt5yQq7DCjJfQHcRVbDEQYL4YsKyD+CjEwpn0HqUB7nxTaF4R6SaoszpujwykTa+b+ksPVVJME\\\\\\\/MG+xEyEVranttwpDblEY98IM+ifiEorqvSzbbhZL4dr6qx4WyqwTP6iK3xm0CYnc8fuQsf3qrEdnXC01kRMZvZESjGZV3SCnsM3KCHFK09XntBaNzwOC+uZQnYzs1\\\\\\\/U1tWtLNQBl1rqEY0vqhY+afLrFPnZDmwIHBVyZjGVs7PMV+yOIofOoGu84JmOnc\\\\\\\/\\\\\\\/1C49tS2blId7+vcDCddtQzflWxINctUE1G\\\\\\\/BlvRQT+NaIELo1lCVdRp\\\\\\\/m9fB0HN6wC0mNgNVBkRkRcuEm0t+zdhRhuPVJZU99UiIuUjwvpANuKX+42TUFhlLcnZuVEE\\\\\\\/uGzQ\\\\\\\/nVKZ9+8JUavnmsHQgigxtTqUicdM54bGiGzC9JYdIZsz5ADYXkIZrpuOjnuSHziHDC7Tc4cuxjdXmUuWTdlt4O5t2yIDF3AAG2RwkRr3Y2r0AEy2AkqKDvFhAWWfi0qofCjgAXjp\\\\\\\/81TWyZMQJ50tNnzHrVTWdXTsPPu2Ax2oHyw+Lh2tjM5vLoLFuL\\\\\\\/1AttHcguoTugexhqCegKM6UTLZPgkDkuklufU+ZBZM0w\\\\\\\/kFfPbIBUd8rC0QaLe1F4eqZxKiMZLBN5OHI\\\\\\\/m7xJhAIK4XV8HE3aFlKTc484OrNTiXmdFpAR1zBFIEbEYNuYh0dM7FBGEHVtihUPt377pYt2DKHDKqn+ImDgDg\\\\\\\/GQuteAnDNcy33QyRzLpGyEhL4VdST2aMKPVCEZppWgu6Gtzd775aIWDQe+onSxo78ddC8YvvQ\\\\\\\/9iDVhC6IxSObN\\\\\\\/SBpxKibgm4Fra0dhX\\\\\\\/9VQDIJRnjrE1tIyVhWEJspjzEsM\\\\\\\/rMddsoKwQMOJSrHl\\\\\\\/hRa5aEr6ySFkI6tsAyxQqMDnYk+Imn9AQDIEgQNRpOoCfWjltj2VGSN8WFvPjWcrsw8TwzgsaFQr0NXvfpboBRbgmxu47NVuMJ911Nn8ym6YPw2W+mIQ2X+Ainl5zty+M1KbIvAZEIU3cpVVIUoI6MsPejQc09yJCI57ZBQ6tAtiBzv6odgUSmKoiSV4zqyUwAeXsuWJlw9m5l2KmovwLcSeHd56JLOtdA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mo+mLVRdnQtQJKbQfJoGcw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b53303343bde98f8203753ac89be6bd74a5f8104fa0cf72306d5fea90f31f796\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459838 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/579" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TM1DAW9syYPUO6Gx0S7hMbt0rBAmT6Tr8u8bnokMHmE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/579" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XLpovysb9yguWSpif_yk9C7QfinolavaV8vOWc9T8UQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/561" title="Comparative Genomic and Transcriptomic Analysis of Wangiella dermatitidis, A Major Cause of Phaeohyphomycosis and a Model Black Yeast Human Pathogen" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/595" title="Identification of Major and Minor QTL for Ecologically Important Morphological Traits in Three-Spined Sticklebacks (Gasterosteus aculeatus)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459838" id="node459838" data-pisa="ggg;4/4/579" data-pisa-master="ggg;g3.114.010447" data-apath="/ggg/4/4/579.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Functional and Evolutionary Features of <em>Tele</em>-Enhancers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Di Huang</span> and <span class="highwire-citation-author" data-delta="1">Ivan Ovcharenko</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">579-593;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010447 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459838" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Di Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Di%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADi%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Ovcharenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BOvcharenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ovcharenko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvan%2BOvcharenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ovcharen@nih.gov" class="" data-icon-position="" data-hide-link-title="0">ovcharen@nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/4/579" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459838" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/579.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459838" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/579.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459838" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/579.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459838" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/4/579.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459838" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459838/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">We investigated sequence features of enhancers separated from their target gene by at least one intermediate gene/exon (named <em>tele</em>-enhancers in this study) and enhancers residing inside their target gene locus. In this study, we used whole genome enhancer maps and gene expression profiles to establish a large panel of <em>tele</em>-enhancers. By contrasting <em>tele</em>-enhancers to proximal enhancers targeting heart genes, we observed that heart <em>tele</em>-enhancers use unique regulatory mechanisms based on the cardiac transcription factors SRF, TEAD, and NKX-2.5, whereas proximal heart enhancers rely on GATA4 instead. A functional analysis showed that <em>tele</em>-enhancers preferentially regulate house-keeping genes and genes with a metabolic role during heart development. In addition, <em>tele</em>-enhancers are significantly more conserved than their proximal counterparts. Similar trends have been observed for non-heart tissues and cell types, suggesting that our findings represent general characteristics of <em>tele</em>-enhancers.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/enhancer" class="hw-term hw-article-keyword hw-article-keyword-enhancer" rel="nofollow">enhancer</a></li><li class="kwd"><a href="/keyword/nucleotide-divergence" class="hw-term hw-article-keyword hw-article-keyword-nucleotide-divergence" rel="nofollow">nucleotide divergence</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism-0" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism" rel="nofollow">single-nucleotide polymorphism</a></li><li class="kwd"><a href="/keyword/tissue-specificity" class="hw-term hw-article-keyword hw-article-keyword-tissue-specificity" rel="nofollow">tissue specificity</a></li><li class="kwd"><a href="/keyword/transcription-factor-binding-motif" class="hw-term hw-article-keyword hw-article-keyword-transcription-factor-binding-motif" rel="nofollow">transcription factor binding motif</a></li></ul><p id="p-4">Enhancers, which are key to the precise regulation of spatiotemporal gene expression, often reside at a distance from their target genes (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Maston <em>et al.</em> 2006</a>) and function through long-range regulatory mechanisms (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Glinskii <em>et al.</em> 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lettice <em>et al.</em> 2003</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pomerantz <em>et al.</em> 2009</a>). Although some enhancers are found in the proximity of the transcription start sites of their target gene, there is mounting evidence of distant enhancers incorporated into the structure of neighboring genes or looping over intermediate unaffected genes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Irimia <em>et al.</em> 2012</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lettice <em>et al.</em> 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Perry <em>et al.</em> 2010</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Visser <em>et al.</em> 2012</a>). As such, identifying target genes of enhancers, especially enhancers that regulate genes outside of the locus they reside in (which we dubbed <em>tele</em>-enhancers), poses a great challenge in the research area of gene regulation. To address this challenge, many have used evolutionary conservation of DNA sequences and focused on highly conserved enhancers (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clarke <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davidson <em>et al.</em> 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Engström <em>et al.</em> 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kikuta <em>et al.</em> 2007</a>). Assuming that enhancers and their target genes are located within a genomic regulatory block encompassing a set of neighboring genes and regulatory regions, regulatory relations between highly conserved enhancers and their target genes have been established, and transgenic models have been used to validate these regulatory relations (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dong <em>et al.</em> 2009</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kikuta <em>et al.</em> 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">MacKenzie <em>et al.</em> 2004</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Navratilova and Becker 2009</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sanyal <em>et al.</em> 2012</a>). These studies have successfully identified enhancers and their distant target genes, which are instrumental for understanding the mechanisms and evolution of gene regulation. However, these investigations focused on individual <em>tele</em>-enhancers. Recently, with the advancement of sequencing techniques, such as chromatin conformation capture (3C, Hi-C) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Belton <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dixon <em>et al.</em> 2012</a>) and chromatin interaction paired-end tagging (<em>i.e.</em>, ChIA-PET) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2010</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhang <em>et al.</em> 2012</a>), it became possible to experimentally characterize interactions between enhancers and their target genes. However, these experiments are notoriously difficult to run, and genome-wide data are available only in a few cell types. As a result, without the knowledge of a substantial number of <em>tele</em>-enhancers, we still have a scant genome-wide understanding of interactions between long-range enhancers and their target genes.</p><p id="p-5">In this study, we binned heart enhancers into two groups—proximal and <em>tele</em>-enhancers—based on their genomic location relative to the nearest heart gene, and then systematically investigated the differences between proximal and distant regulatory mechanisms of transcriptional activation in the human heart. We observed that <em>tele</em>-enhancers have unique biological functions—regulating housekeeping genes and playing a role in basic biological processes. Next, we demonstrated that <em>tele</em>-enhancer sequences feature a distinguishable binding motif signature. Although both <em>tele</em>- and proximal heart enhancers are enriched for the binding motifs of several cardiac transcription factors (TFs) (such as MEF2A), <em>tele</em>-enhancer sequences are enriched for TEAD1 and NKX-2.5 binding motifs and are depleted of the GATA4 binding motif compared with proximal enhancers. We also demonstrated that <em>tele</em>-enhancers (1) are significantly more conserved than proximal enhancers, (2) display lower single-nucleotide polymorphism (SNP) density, and (3) are enriched for low derived allele frequency (DAF) SNPs—all suggesting that <em>tele</em>-enhancers have been and are currently evolving under a stronger negative selective pressure than their proximal counterparts. We then extended our analysis to other tissues and cell types (including fetal brain, fetal lung, HepG2, K526, HSMM, and H1-hESC cell lines), for which genome-wide enhancer maps were identified using different experimental methods, and observed similar evolutionary trends as well as sequence and functional features of <em>tele</em>-enhancers, suggestive of the general trends of this novel regulatory mechanism described by our study.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Heart enhancers and genes</h3><p id="p-6">We used the set of 5047 human heart enhancers identified by a p300 chromatin immunoprecipitation sequencing (ChIP-seq) study of fetal heart tissue (gestational week 16; Gene Expression Omnibus [GEO] data set series 32587) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">May <em>et al.</em> 2012</a>). We assigned each enhancer to the two nearest heart genes located within 500 kbs from the enhancer. As a result, an enhancer was assigned to at most two heart genes. After discarding enhancers not associated with any heart gene and heart genes not associated with any enhancer, we obtained 3391 enhancers linked to 1832 heart genes. We dubbed enhancers separated from their target genes by at least one nonheart gene or exon as <em>tele</em>-enhancers, and the enhancers located within the loci of heart genes as proximal enhancers (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We also generated a set of control sequences. For each enhancer, two random noncoding human DNA sequences with matching length, guanine-cytosine (GC) content, and repeat density were retrieved.</p></div><div id="sec-3" class="subsection"><h3>Enhancers and genes in other tissues and cell lines</h3><p id="p-7">Fetal brain enhancers were obtained from the H3K4me1 map established by ChIP-seq experiments carried out on the human fetal brain tissue (GEO dataset GSM706850, human fetal brain at day 122). To ensure that the nucleotide sequences used in this study are enhancers and not proximal promoters, we discarded sequences located within 3 kb of the transcription start site of any human gene. To obtain brain genes, we collected the genes annotated to the “brain development” or its child categories in the Gene Ontology (GO) database. We also used gene expression profiles (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2004</a>) and identified the top 20% of genes highly expressed in fetal brain with respect to other tissues and cell lines. After applying our method to these data, we assigned 7649 brain enhancers to 1957 brain genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/010447SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-8">Fetal lung enhancers also were obtained from the H3K4me1 map established by ChIP-seq experiments carried out on the human fetal lung tissue (GEO dataset GSM706853, human fetal lung at day 101). We similarly discarded sequences located within 3 kb of any transcription start site. To the end, we linked 5996 lung enhancers to 1716 lung genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-9">In cell lines including GM12878, 1H-hESC, HepG2, HSMM, HUVEC, K562, and NHEK, we used ChromHMM enhancer maps (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Djebali <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ernst and Kellis 2012</a>). Also, based on RNA-seq data reported in the ENCODE project (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Djebali <em>et al.</em> 2012</a>), we extracted the top 20% of genes highly expressed in a cell with respect to other cells as cell-specific genes. After applying our pipeline to these data (<em>i.e.</em>, cell-specific enhancers and genes), we identified ∼6000 <em>tele</em>-enhancers in each line (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS2.pdf">Table S2</a>).</p></div><div id="sec-4" class="subsection"><h3>Evaluation of enhancer-gene association</h3><p id="p-10">Enhancer-promoter maps have been previously constructed using the distribution of DNase I hypersensitive sites (DHS) across 79 distinct cell types, and the reported regulatory connections have been confirmed using 5C and chromatin immunoprecipitation paired-end sequencing data (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Thurman <em>et al.</em> 2012</a>).</p><p id="p-11">Although the distal-DHS-promoter connection map was established on the basis of an extensive panel of cells, this map does not cover the entire regulatory-element-promoter landscape for all tissues and cell lines. For example, 1538 (of 3391) heart enhancers assigned to at least one heart gene could not be retrieved from this distal-DHS-promoter connection map. As such, we built the regulatory block for each gene based on distal-DHS-promoter connection map (denoted as DHS-based regulatory block in this study). Given a gene, the genomic boundaries of its DHS-based regulatory block were defined using the most distal DHS connected to that gene. Because the enhancers located within this block more likely regulate that gene than those located outside of this block, we evaluated whether enhancers reside within the DHS-based regulatory blocks of their associated genes. We estimated the fraction of the enhancers that fell into the regulatory blocks of their associated genes, and compared this fraction with a null distribution. The null distribution was established by randomly selecting genes. In detail, given an enhancer and its associated gene (the distance between them is D), we randomly selected a gene with the regulatory block of a similar length to the tested gene (<em>i.e.</em>, the length of the regulatory block of randomly selected genes was required to reside in the range of [0.9z, 1.1z] where z is the length of the regulatory block of the tested gene). Then, we checked whether the sequence located away from the selected gene at a distance of D fell into the regulatory block of the selected genes. For each enhancer, we repeated this process 20 times, and used the fraction of the sequences residing in the respective regulatory blocks as expected.</p></div><div id="sec-5" class="subsection"><h3>Evolutionary synteny blocks (ESBs) and density of evolutionary breakpoints</h3><p id="p-12">ESBs are commonly used to identify boundaries of regulatory interactions, as regulatory elements and their target genes tend to reside within the same synteny block, if the regulatory mechanisms in question play an important functional role. Accordingly, the density of synteny breakpoints should be reduced between regulatory elements and their target genes.</p><p id="p-13">We downloaded ESBs from the ECRBase database (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ovcharenko <em>et al.</em> 2005b</a>) and evaluated the breakpoint density in the regions separating heart enhancers and their target genes. After that, the density of synteny breakpoints was compared with an expectation. Because the evolutionary breakpoints are not evenly distributed along genome (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carver and Stubbs 1997</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pevzner and Tesler 2003</a>), we estimated the expected density of breakpoints “locally” instead of “globally.” That is, for each region spanning an enhancer and one of its target genes, we extended this region by 500 kb along both the upstream and downstream direction, and then excluded gene coding regions from the extended sequence. We used these extended regions as controls to estimate the expectation of breakpoint density.</p></div><div id="sec-6" class="subsection"><h3>Control genes</h3><p id="p-14">To eliminate the potential impact of locus length on the functional analysis of genes, we generated control genes with a similar locus length to a given gene set. For a gene, we randomly selected five genes having a similar locus length. After repeating this process for all genes, we generated a control set for a tested gene set.</p></div><div id="sec-7" class="subsection"><h3>Support vector machine (SVM) classification models and binding motifs enriched in enhancer sequences</h3><p id="p-15">To discriminate enhancer sequences from controls, we designed a computation system that included two main steps. In the first step, putative TF binding sites were mapped in the DNA sequence of enhancers and controls. For this, sequences were scanned using tfSearch (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ovcharenko <em>et al.</em> 2005a</a>) with the position weight matrices from the TRANSFAC and JASPAR databases (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sandelin <em>et al.</em> 2004</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wingender <em>et al.</em> 2001</a>). In the second step, TF binding site vectors of enhancers and controls were used to build a linear SVM (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cortes and Vapnik 1995</a>) to discriminate between enhancers and controls. Given a training set of instances <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-1.gif"/></noscript></span></span> with associated labels <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-2.gif"/></noscript></span></span>, a linear SVM <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-3.gif"/></noscript></span></span> was built by solving the optimization problem <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-4.gif"/></noscript></span></span> subject to <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-5.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-6.gif"/></noscript></span></span>(<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cortes and Vapnik 1995</a>). In such linear SVM, a linear weight w<sub>i</sub> is assigned to each TF binding site. A large positive w<sub>i</sub> indicates a binding site that is strongly associated with the enhancers, whereas negative weights correspond to the binding sites associated with the control set. We ranked sites according to w<sub>i</sub>, <em>i.e.</em>, the positive association with the tested enhancer set.</p></div><div id="sec-8" class="subsection"><h3>Functional analysis based on GO annotation</h3><p id="p-16">The enrichment of a GO functional category (C<sub>i</sub>) for a group of genes (G) was measured according to<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-7.gif"/></noscript></span></span>where m is the size of the overlap between C<sub>i</sub> and G. M and N are the size of C<sub>i</sub> and all genes, respectively. n is the size of G. To control the false-positive rate, we adopted the conservative Bonferroni multiple-testing correction strategy (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dunn 1961</a>) to adjust Pr<sub>i</sub> as aPr<sub>i</sub> = KPr<sub>i</sub>, where K is the number of GO functional categories for the test.</p></div><div id="sec-9" class="subsection"><h3>Nucleotide divergence and diversity</h3><p id="p-17">To evaluate nucleotide divergence, we aligned each enhancer sequence to chimpanzee and rhesus by using axt alignment files available from the UCSC Genome Browser (<a href="http://genome.ucsc.edu/index.html">http://genome.ucsc.edu/index.html</a>).</p><p id="p-18">Nucleotide difference (d) of a DNA region was defined as the number of nucleotides different across species per site. According to the three-way alignment among human, chimpanzee, and macaque, we evaluated d specific to each species. Given a DNA region, d specific to human (dh) is measured as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-8.gif"/></noscript></span></span>where La is the length of three-way-alignable fragment along the given region, and L<sub>h≠(c=r)</sub> is the number of aligned nucleotides where chimpanzee is equal to macaque, and human is the outlier with respect to chimpanzee and macaque. Nucleotide divergence specific to human (Dh) is then estimated by adjusting dh with Juke-Cantor correction for multiple hits (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jukes and Cantor 1969</a>). In a similar way, the nucleotide divergence D specific to chimp (Dc) and specific to macaque (Dm) were measured.</p><p id="p-19">To evaluate selection neutrality within the human lineage, we defined the neutrality index (NI), in which human-specific nucleotide divergence (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-9.gif"/></noscript></span></span> of enhancers is compared with non-human divergence (<em>i.e.</em>, Dr + Dc) with respect to the neutral reference.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/579/embed/mml-math-10.gif"/></noscript></span></span>where Nh, Nr, and Nc are the divergences specific to human, rhesus, and chimp. A large NI means a large nucleotide difference during the evaluation of human (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Also, based on Dh, Dr + Dc, Nh and Nr + Nc, we carried out the McDonald-Kreitman test (MK test) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McDonald and Kreitman 1991</a>) to estimate the significance of nucleotide divergence.</p><p id="p-20">Nucleotide diversity (π) of a region is estimated as the number of SNPs per site in the alignable fraction of a region.</p></div><div id="sec-10" class="subsection"><h3>Genes regulating mitochondrial biological processes</h3><p id="p-21">After going through the GO database (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ashburner <em>et al.</em> 2000</a>), we collected 282 genes annotated to a mitochondrial biological process.</p></div><div id="sec-11" class="subsection"><h3>Pseudogenes</h3><p id="p-22">Pseudogenes, dysfunctional gene homologs (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vanin 1985</a>), were used as a neutral reference in this study. We downloaded pseudogenes from the Peseudogene.org database (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Balasubramanian <em>et al.</em> 2009</a>).</p></div><div id="sec-12" class="subsection"><h3>Human-accelerated conserved noncoding sequences (HACNSs)</h3><p id="p-23">HACNSs used in this study were from <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Prabhakar <em>et al.</em> (2006)</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Identification of <em>Tele</em>-heart enhancers and their target genes</h3><p id="p-24">In this study, we used a conservative definition of <em>tele</em>-enhancers, requiring a presence of an intermediate gene (or an exon of an intermediate gene) between an enhancer and its target gene. To identify heart <em>tele</em>-enhancers, we first compiled a list of developmental heart genes according to gene expression profiles and gene annotation information. After ranking all human genes according to their expression level in the human heart relative to the other 78 tissues/cell types (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2004</a>) (<a href="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/">http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/</a>), we selected the top 20% (2342) of genes and observed that these genes are significantly enriched (1) in the heart development GO category (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Ashburner, Ball <em>et al.</em> 2000</a>) [<em>P</em> = 2 × 10<sup>−11</sup>, using DAVID (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jiao <em>et al.</em> 2012</a>)], (2) in the neighborhood of p300 heart enhancers (hypergeometric distribution <em>P</em> = 4 × 10<sup>−34</sup>), and (3) in the heart disease genes reported by the GeneTests database (<a href="http://www.ncbi.nlm.nih.gov/gtr/">http://www.ncbi.nlm.nih.gov/gtr/</a>; <em>P</em> = 4 × 10<sup>−13</sup>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/FileS1.pdf">File S1</a>), suggesting a significant association between these highly expressed genes and human heart development (see <em>Materials and Methods</em>). In addition, we combined these highly expressed genes with heart development genes annotated in GO (total = 348 genes), and then established a collection of 2430 distinct heart genes. The genomic landscape of these heart genes was superimposed onto 3391 heart development enhancers that were previously identified in a p300 ChIP-seq fetal human heart tissue experiment (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blow <em>et al.</em> 2010</a>), and were located within no more than 500 kb from the transcription start site of a heart gene. Following a general rule postulated by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Busser <em>et al.</em> (2012)</a>, we associated each heart enhancer with the two closest heart genes—one upstream and the other downstream of the enhancer. Enhancers separated from their nearest heart gene by at least one “nonheart” gene/exon were considered <em>tele</em>-enhancers (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). In total, 3319 heart enhancers were associated with 1832 heart genes, out of which a large proportion of genes—1171 (64%)—had no proximal enhancers located on either the inside of the gene body, or in between the gene and its two flanking genes. These findings are in line with the report that proximity is not the governing rule for enhancers regulating genes, and less than 50% of expressed genes have proximal enhancers in several cell types (including embryonic stem cells) (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhang <em>et al.</em> 2013</a>). We hypothesized that the heart genes lacking proximal enhancers are likely regulated by <em>tele</em>-enhancers (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). For simplicity, we dubbed a heart gene with at least one proximal heart enhancer GeneP, and a heart gene linked to <em>tele</em>-enhancer(s) only—GeneT. Although genes lacking proximal enhancers may be controlled through activation by regulatory elements other than enhancers, such as locus control regions (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2002</a>), most genome-wide gene regulation studies are confined to promoters and enhancers for simplicity and generalization (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gaszner and Felsenfeld 2006</a>). Therefore, we assumed that GeneTs are primarily regulated by <em>tele</em>-enhancers. Also, before systemically characterizing <em>tele</em>- and proximal enhancers, we evaluated the reliability of the established assignments between genes and enhancers.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/579/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proximal and tele-enhancers. (A) Schematic definition of two classes of enhancers. (B) Distribution of heart GeneTs and GenePs (top) and proximal and tele-enhancers (bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-77238227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proximal and &lt;em&gt;tele&lt;/em&gt;-enhancers. (A) Schematic definition of two classes of enhancers. (B) Distribution of heart GeneTs and GenePs (top) and proximal and &lt;em&gt;tele&lt;/em&gt;-enhancers (bottom).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/579/F1.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/4/579/F1.medium.gif" width="440" height="206"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/579/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/579/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464116" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Proximal and <em>tele</em>-enhancers. (A) Schematic definition of two classes of enhancers. (B) Distribution of heart GeneTs and GenePs (top) and proximal and <em>tele</em>-enhancers (bottom).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Regulatory landscape of heart genes</h3><p id="p-26">We used ESBs to confirm regulatory relationships between the <em>tele</em>-enhancers and their assigned target genes (see <em>Materials and Methods</em> for details). ESBs are a known characteristic of long-range regulatory interactions as the separation of critical regulatory elements from their target genes through chromosomal rearrangements is selected against (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Ovcharenko <em>et al.</em> 2005b</a>); ESBs have been successfully used to detect target genes of long-range enhancers (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Dong <em>et al.</em> 2009</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Kikuta <em>et al.</em> 2007</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Navratilova and Becker 2009</a>). Accordingly, we expected <em>tele</em>-enhancers and their target genes to reside within the same ESB more often than expected by chance. We used the defined ESBs based on the sequence alignments between human and chimpanzee, macaque, mouse, cow, and chicken (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Ovcharenko <em>et al.</em> 2005b</a>) and compared the density of the evolutionary breakpoints between enhancers and their target genes with the density expected in the neighborhood of enhancers (see <em>Materials and Methods</em>). Our comparative results showed that the genomic space separating enhancers from their target genes exhibited a significantly lower density of evolutionary breakpoints than would be expected. As such, the enhancers, either proximal or <em>tele</em>-enhancers, and their associated genes were predominantly located in the same ESB (<em>P</em> &lt; 10<sup>−5</sup> in all cases, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/579/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heart genes and enhancers linked to them. (A) The average density of evolutionary breakpoints between enhancers and their target genes. A low density of breakpoint indicates a high probability of the enhancers and their target genes being located inside the same evolutionary synteny block. **Indicates that the corresponding density is significantly lower than expected, i.e., P-value &lt; 1 &#xD7; 10&#x2212;5. (B) Length distribution of DNase I hypersensitive sites (DHS)-based regulatory blocks of GeneTs and GenePs. (C) Fraction of enhancers residing within the DHS-based regulatory blocks of their associated genes. (D) Length distribution of noncoding regions (intergenic and intronic) of GeneTs and GenePs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-77238227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heart genes and enhancers linked to them. (A) The average density of evolutionary breakpoints between enhancers and their target genes. A low density of breakpoint indicates a high probability of the enhancers and their target genes being located inside the same evolutionary synteny block. **Indicates that the corresponding density is significantly lower than expected, &lt;em&gt;i.e.&lt;/em&gt;, &lt;em&gt;P&lt;/em&gt;-value &lt; 1 &#xD7; 10&lt;sup&gt;&#x2212;5&lt;/sup&gt;. (B) Length distribution of DNase I hypersensitive sites (DHS)-based regulatory blocks of GeneTs and GenePs. (C) Fraction of enhancers residing within the DHS-based regulatory blocks of their associated genes. (D) Length distribution of noncoding regions (intergenic and intronic) of GeneTs and GenePs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/579/F2.medium.gif" width="362" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/4/579/F2.medium.gif" width="362" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/579/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/579/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464102" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Heart genes and enhancers linked to them. (A) The average density of evolutionary breakpoints between enhancers and their target genes. A low density of breakpoint indicates a high probability of the enhancers and their target genes being located inside the same evolutionary synteny block. **Indicates that the corresponding density is significantly lower than expected, <em>i.e.</em>, <em>P</em>-value &lt; 1 × 10<sup>−5</sup>. (B) Length distribution of DNase I hypersensitive sites (DHS)-based regulatory blocks of GeneTs and GenePs. (C) Fraction of enhancers residing within the DHS-based regulatory blocks of their associated genes. (D) Length distribution of noncoding regions (intergenic and intronic) of GeneTs and GenePs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Additional validation of the predicted enhancer-gene relationships was obtained using a genome-wide map of enhancer-promoter associations constructed based on a DNase I comparative profiling of the human genome (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Thurman <em>et al.</em> 2012</a>). Although the reported DNase I map of enhancer-promoter relationships is an approximation across a large panel of cell types, which does not necessarily represent a comprehensive collection of long-range regulatory activities in the heart, it estimates the extent of regulatory interactions across different gene loci. After defining a DHS-based regulatory block for each gene based on its most distal enhancers, we first noticed that GeneTs have longer DHS-based regulatory blocks as compared to GenePs (Wilcoxon rank sum <em>P</em> = 0.06, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Also, we computed the fraction of the enhancers located within the regulatory block of their associated genes, and demonstrated that 82% of <em>tele</em>-enhancers reside within the regulatory blocks of their associated genes, which was significantly higher than expected (binomial test <em>P</em> = 3 × 10<sup>−15</sup>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). Similarly, 83% of proximal enhancers were located within the regulatory blocks of their associated genes, which also was significantly higher than expected (binomial test <em>P</em> = 3 × 10<sup>−11</sup>, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). These results further support the established regulatory relationship between the enhancers, either proximal or <em>tele</em>-enhancers, and their associated genes.</p><p id="p-29">Next, we examined the size of intronic and intergenic regions of heart gene loci and noticed that, on average, GeneTs featured 3.0-fold shorter intronic regions than GenePs (Wilcoxon rank sum <em>P</em> = 2 × 10<sup>−26</sup>, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>). Similarly, the intergenic intervals flanking GeneTs were 2.78-fold shorter compared to GenePs (Wilcoxon rank sum <em>P</em> = 6 × 10<sup>−64</sup>, <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>). It is likely that the small locus size of GeneTs predisposes these genes to the acquisition of <em>tele</em>-enhancers. Given the recent evolutionary nature of many heart enhancers (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Blow <em>et al.</em> 2010</a>), an alternative hypothesis of an intermediate gene insertion in between a heart enhancer and a GeneT is unlikely. From the evolutionary viewpoint, it has been found that the maintenance of gene function over a long evolutionary time leaves a selection signature of gene structure (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Vinogradov 2006</a>), and that tissue-specific genes might harbor long noncoding regions containing multiple regulatory regions, whereas widely expressed genes, lacking strong intron constraints, might have been subjected to selective pressure to reduce the length of noncoding regions (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eisenberg and Levanon 2003</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pozzoli <em>et al.</em> 2007</a>). With this knowledge, and based on our finding that GeneTs showed significantly shorter intronic and intergenic spread than GenePs, we hypothesized that <em>tele</em>-enhancers and proximal enhancers, which respectively regulate GeneTs and GenePs, have different biological functions and undergo different evolutionary processes. To explore this hypothesis, we next examined functional and evolutionary features of <em>tele</em>- and proximal heart enhancers.</p></div><div id="sec-16" class="subsection"><h3><em>Tele</em>-enhancers regulate mitochondrial biological processes</h3><p id="p-30">We analyzed the function of the two heart enhancer groups—proximal and <em>tele</em>-enhancers—according to the GO function of their target genes (GenePs and GeneTs, respectively) (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Ashburner <em>et al.</em> 2000</a>). In this study, to account for the different locus lengths of GenePs and GeneTs (as discussed previously), which may cause a bias in a gene function analysis, we generated controls for GenePs and GeneTs separately by randomly selecting genes with similar length intergenic and intronic regions (see <em>Materials and Methods</em>), and compared GenePs/GeneTs with the respective control genes. As expected, both proximal and <em>tele</em>-enhancers play an important role in biological processes related to heart development, such as heart morphogenesis, cardiac muscle differentiation, etc. (multiple testing corrected binomial test <em>P</em> &lt; 1 × 10<sup>−3</sup>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS3.pdf">Table S3</a>). On the other hand, each heart enhancer group featured distinct functions—proximal enhancers were strongly associated with various processes related to heart development, whereas <em>tele</em>-enhancers were involved in basic metabolic functions. For example, of 18 genes associated with ventricular cardiac muscle cell differentiation (GO:0055012), 11 (<em>P</em> = 4 × 10<sup>−4</sup>) genes were GeneP, whereas only 3 were GeneT. Similarly, among 27 genes that fell into the category of artery morphogenesis (GO:0048844), 13 genes (<em>P</em> = 1 × 10<sup>−7</sup>) had proximal enhancers, while 7 genes were GeneT (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS3.pdf">Table S3</a>). By contrast, 73 precursor metabolites and energy (GO:0006091) genes were categorized as GeneT, whereas only 28 as GenePs. We also identified genes taking part in the regulation of mitochondrial biological processes and observed that GeneTs, but not GenePs, were significantly enriched for those genes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, 2.9% of GenePs <em>vs.</em> 5.1% of GeneTs, binomial test <em>P</em> = 2 × 10<sup>−5</sup>). Our observation is supported by reports that regulation of mitochondrial processes is essential to heart development (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Drenckhahn 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Goffart <em>et al.</em> 2004</a>) and has been used as a therapeutic target in heart failure (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Huss and Kelly 2005</a>). Moreover, whereas both GeneTs and GenePs showed significant heart specificity compared with the expected (<em>i.e.</em>, respective control genes, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/FigureS2.pdf">Figure S2</a>), GeneTs had significantly weaker heart specificity than GenePs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>, the average of GeneTs and GenePs are 1.7 and 1.9, respectively, Wilcoxon rank-sum test <em>P</em> = 7 × 10<sup>−3</sup>). This further indicates that, compared with GenePs, GeneTs more likely partake in basic biological processes and display lower heart specificity.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/579/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Function of tele- and proximal enhancers. (A) Functional analysis based on GO annotation. The enrichment is measured as the ratio of the fraction of the target genes having a tested function to the expectation with the matching locus length (i.e., gene as well as its intergenic and intronic regions). The P-value is estimated using the binomial test. (B) Enrichment of mitochondrial genes among GeneTs and GenePs (corresponding to tele- and proximal enhancers, respectively). (C) The relative expression of GeneTs and GenePs&#x2014;given a gene, a low relative expression indicates a weak specificity to heart." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-77238227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Function of &lt;em&gt;tele&lt;/em&gt;- and proximal enhancers. (A) Functional analysis based on GO annotation. The enrichment is measured as the ratio of the fraction of the target genes having a tested function to the expectation with the matching locus length (&lt;em&gt;i.e.&lt;/em&gt;, gene as well as its intergenic and intronic regions). The &lt;em&gt;P&lt;/em&gt;-value is estimated using the binomial test. (B) Enrichment of mitochondrial genes among GeneTs and GenePs (corresponding to &lt;em&gt;tele&lt;/em&gt;- and proximal enhancers, respectively). (C) The relative expression of GeneTs and GenePs&#x2014;given a gene, a low relative expression indicates a weak specificity to heart.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/579/F3.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/4/579/F3.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/579/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/579/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464229" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Function of <em>tele</em>- and proximal enhancers. (A) Functional analysis based on GO annotation. The enrichment is measured as the ratio of the fraction of the target genes having a tested function to the expectation with the matching locus length (<em>i.e.</em>, gene as well as its intergenic and intronic regions). The <em>P</em>-value is estimated using the binomial test. (B) Enrichment of mitochondrial genes among GeneTs and GenePs (corresponding to <em>tele</em>- and proximal enhancers, respectively). (C) The relative expression of GeneTs and GenePs—given a gene, a low relative expression indicates a weak specificity to heart.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We also looked into the function of bystander genes of <em>tele</em>-enhancers, <em>i.e.</em>, the genes which are located closer to <em>tele</em>-enhancers than the target genes of these enhancers, but are not regulated by these enhancers during heart development. The functional GO analysis indicated that these bystander genes were not significantly enriched for any biological function (data not shown), which further supports the established connections between <em>tele</em>-enhancers and their target genes.</p><p id="p-33">Collectively, <em>tele</em>-enhancers and proximal enhancers have different biological functions. The former partake in basic biological processes, regulating mitochondrial biological processes, while the latter are more specific to heart development. Because <em>tele</em>-enhancers showed functions distinct from proximal enhancers, we hypothesized that <em>tele</em>-enhancers and proximal enhancers are involved in different transcriptional mechanisms and could be activated by different sets of TFs.</p></div><div id="sec-17" class="subsection"><h3><em>Tele</em>-enhancers feature distinguishable binding motif compositions</h3><p id="p-34">We adapted a machine learning approach that has been previously used to identify motifs specific to heart enhancers (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Narlikar <em>et al.</em> 2010</a>). This method identifies specific sequence patterns for a set of non-coding sequences relying primarily on known TF binding motifs from the TRANSFAC and JASPAR databases (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Matys <em>et al.</em> 2006</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Stormo 2000</a>). After mapping 981 vertebrate TF binding motifs onto enhancers and controls with similar GC content, repeat density and sequence length, we built two SVM classifiers with linear kernels based on the occurrence of TF binding motifs (see <em>Materials and Methods</em>)—one for <em>tele</em>-enhancers and another for proximal enhancers. TF motifs strongly associated with training enhancers received large positive weights. We selected the motifs with positive weights in either classifier and clustered these motifs based on the SVM weights, <em>i.e.</em>, the association with enhancer classes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Only 26% (30 of 117) of TF binding motifs, in which cardiac TFs such as MEF2A were included, were shared between <em>tele</em>- and proximal enhancers. <em>Tele</em>-enhancers featured positive association with the binding motifs of well-known cardiac TFs SMAD1, SRF, NKX-2.5, and TEAD and no association with the binding motifs of the cardiac TF GATA4, which were specific to proximal enhancers (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS4.pdf">Table S4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/579/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Association of TF binding motifs with different enhancer classes. (A) The weight of support vector machines built to discriminate enhancer sequences from controls. (B) Enrichment fold of transcription factor binding motifs in tele- and proximal enhancers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-77238227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Association of TF binding motifs with different enhancer classes. (A) The weight of support vector machines built to discriminate enhancer sequences from controls. (B) Enrichment fold of transcription factor binding motifs in &lt;em&gt;tele&lt;/em&gt;- and proximal enhancers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/579/F4.medium.gif" width="424" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/4/579/F4.medium.gif" width="424" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/579/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/579/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464242" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Association of TF binding motifs with different enhancer classes. (A) The weight of support vector machines built to discriminate enhancer sequences from controls. (B) Enrichment fold of transcription factor binding motifs in <em>tele</em>- and proximal enhancers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">We also investigated the enrichment of TF binding motifs in <em>tele</em>- and proximal enhancers and observed a striking difference in their motif composition (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS5.pdf">Table S5</a>). For example, the binding motifs of NKX-2.5, TBX5, and TEAD were strongly enriched in <em>tele</em>-enhancers but not in proximal enhancers, while the binding motifs of NFAT4 and GATA4 were specific to proximal enhancers. These results indicate existence of a set of cardiac TFs needed for the activation of both proximal enhancers and <em>tele</em>-enhancers, and a specific transcriptional modulation by different cardiac TFs within these two groups of enhancers. The differences in transcriptional mechanisms employed by proximal enhancers and <em>tele</em>-enhancers can potentially explain the difference in the biological function and expression patterns driven by these two groups of enhancers. The slight discrepancy between TF binding motif enrichment and the weights in linear SVMs (for example, the binding motif of TBX5 was exclusively enriched in <em>tele</em>-enhancer sequences while this motif has not received a large weight in either <em>tele</em>- or proximal enhancer SVMs) may be explained by the fact that enrichment evaluates TF binding motifs individually, while a linear SVM model estimates binding motifs collectively. Also, a high enrichment fold does not necessarily indicate that the abundance of a binding motif in enhancers is sufficient to distinguish enhancers from the rest of the non-coding genome. For example, the binding motif of TBX5, although exhibiting relatively high enrichment fold (1.3) in <em>tele</em>-enhancers, had a relatively low occurrence (0.03 per 1000 bps), which led to a small linear weight assigned to this motif in our SVM models.</p></div><div id="sec-18" class="subsection"><h3>Different selective signatures of <em>Tele</em>- and proximal enhancers</h3><p id="p-37">It is known that the regulatory elements sharing the same cellular function and being activated in the same biological pathway evolve in concert and tend to have correlated selective signatures (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Shapiro and Alm 2008</a>). We next analyzed the selective constraints imposed on heart enhancer groups. Although developmental enhancers, as a whole, are under strong evolutionary constraint, the selective pressure imposed on enhancers varies greatly—phastCons, a conservation score, ranges from 0 (indicating no conservation) to 1 (perfect conservation) (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Blow <em>et al.</em> 2010</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Clarke <em>et al.</em> 2012</a>). To analyze the selective constraint of enhancers, we used phastCons derived from 46 placental mammal sequence alignments (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Siepel <em>et al.</em> 2005</a>) and assigned the average phastCons along the tested sequence to that enhancer. Compared with proximal enhancers, <em>tele</em>-enhancers were more conserved (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/FigureS3.pdf">Figure S3</a>): 15% of <em>tele</em>-enhancers had a phastCons greater than 0.2, whereas 13% of proximal enhancers reached this conservation level (binomial test, <em>P</em> = 7 × 10<sup>−3</sup>, <em>tele</em>- <em>vs.</em> proximal enhancers).</p><p id="p-38">We next evaluated the selective pressure acting on enhancer sequences within the human lineage. After generating human-chimpanzee-macaque three-way alignments, we estimated the nucleotide divergence of enhancer sequence between any two species and evaluated the human-specific and non-human-specific divergence (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The divergence rate of enhancer sequences was compared with the neutral divergence rate computed using pseudogenes. The comparative results showed that enhancers, either proximal or remote, had remarkably lower divergence levels than the control sequences (which were randomly generated along non-coding DNA with matched repeat density and GC content and the same length as enhancers) and HACNS (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Next, we evaluated the selection constraints acting on sequences along the human lineage using NI. NI is defined in such a way that low NI (&lt;1) and high NI (&gt;1) indicate negative and positive selection during the human lineage evolution, respectively (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). According to the NI estimates, although both proximal enhancers and <em>tele</em>-enhancers featured negative selection, <em>tele</em>-enhancers with the average NI of 0.75 evolved under stronger negative selection constraints than proximal enhancers with the average NI of 0.77 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Also, we evaluated the patterns of selective constraints in proximal enhancers and tele-enhancers using the MK test (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">McDonald and Kreitman 1991</a>). Compared with the neutral reference, both proximal and <em>tele</em>-enhancers evolved under a significant negative selection pressure (<em>P</em> &lt; 6 × 10<sup>−23</sup>, Fisher’s exact test, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Moreover, the negative selection pressure acting on <em>tele</em>-enhancers was significantly stronger compared with proximal enhancers (<em>P</em> = 2 × 10<sup>−2</sup>, Fisher’s exact test).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/579/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic depiction of the human divergence, chimpanzee divergence, and macaque divergence based on three-way genome sequence alignments (see Materials and Methods). Through a comparison with neutral reference (pseudogenes in this study), the selective pressure acting on a tested region is measured using the neutrality index (NI). NI &gt; 1 indicates positive selection, whereas NI &lt; 1 corresponds to negative selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-77238227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic depiction of the human divergence, chimpanzee divergence, and macaque divergence based on three-way genome sequence alignments (see &lt;em&gt;Materials and Methods&lt;/em&gt;). Through a comparison with neutral reference (pseudogenes in this study), the selective pressure acting on a tested region is measured using the neutrality index (NI). NI &gt; 1 indicates positive selection, whereas NI &lt; 1 corresponds to negative selection.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/579/F5.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/4/579/F5.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/579/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/579/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464182" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Schematic depiction of the human divergence, chimpanzee divergence, and macaque divergence based on three-way genome sequence alignments (see <em>Materials and Methods</em>). Through a comparison with neutral reference (pseudogenes in this study), the selective pressure acting on a tested region is measured using the neutrality index (NI). NI &gt; 1 indicates positive selection, whereas NI &lt; 1 corresponds to negative selection.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/579/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Divergence and diversity of enhancers across species. (A) Human divergence (y-axis) is plotted against nonhuman divergence (x-axis). HACNSs are human accelerated conserved noncoding sequences. (B) Fraction of enhancers under positive and negative selective pressure. (C) The derived allele frequency spectrum of SNPs in enhancers, control sequences, neutral reference, and HACNSs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-77238227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Divergence and diversity of enhancers across species. (A) Human divergence (y-axis) is plotted against nonhuman divergence (x-axis). HACNSs are human accelerated conserved noncoding sequences. (B) Fraction of enhancers under positive and negative selective pressure. (C) The derived allele frequency spectrum of SNPs in enhancers, control sequences, neutral reference, and HACNSs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/579/F6.medium.gif" width="419" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/4/579/F6.medium.gif" width="419" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/579/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/579/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464197" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Divergence and diversity of enhancers across species. (A) Human divergence (y-axis) is plotted against nonhuman divergence (x-axis). HACNSs are human accelerated conserved noncoding sequences. (B) Fraction of enhancers under positive and negative selective pressure. (C) The derived allele frequency spectrum of SNPs in enhancers, control sequences, neutral reference, and HACNSs.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464113/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464113/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Nucleotide divergence of heart enhancers, pseudogenes (neutral reference), control sequences, and HACNSs</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">Next, we applied the MK test to evaluate the selective constraints of individual enhancers, and found that, compared to control sequences, both proximal and <em>tele</em>-enhancers tended to be under strong negative constraints, showing less cases of positive selection and more cases of purifying selection than controls (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). On the other hand, compared with proximal enhancers, fewer <em>tele</em>-enhancers were under positive selection. For example, with the <em>P</em>-value cutoff of 1 × 10<sup>−4</sup>, proximal enhancers were more likely to be under positive selective pressure than <em>tele</em>-enhancers—2.5% proximal and 1.5% <em>tele</em>-enhancers evolved under positive selection (binomial test <em>P</em>-value = 4 × 10<sup>−3</sup>, proximal <em>vs.</em> <em>tele</em>-enhancers). In summary, the nucleotide divergence across different species indicates that <em>tele</em>-enhancers are under stronger negative selection than proximal enhancers.</p><p id="p-43">Because of the difficulties in aligning sequences, nucleotide divergence is not as straightforward as SNPs in assessing the signature of selection of DNA regions. We therefore used SNP and allele frequency to evaluate the selective pressure acting on heart enhancers during modern human history. Using the genome variation data from the 1000 Genomes Project (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">1000 Genomes Project Consortium <em>et al.</em> 2010</a>), we first observed that <em>tele</em>-enhancers harbor significantly less SNPs than proximal enhancers (binomial test <em>P</em> = 2 × 10<sup>−3</sup>, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Since a shift toward lower DAF indicates negative selection in the modern human history (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Goode <em>et al.</em> 2010</a>), we also analyzed DAF distribution of SNPs in heart enhancers. The results demonstrated that (1) <em>tele</em> and proximal enhancers represent a 2.4% and 1.7% excess of low-DAF SNPs (<em>i.e.</em>, SNPs having DAF &lt; 0.05) compared with neutral reference, respectively (binomial test, <em>P</em> = 4 × 10<sup>−13</sup> for <em>tele</em>-enhancers <em>vs.</em> neutral reference, and <em>P</em> = 3 × 10<sup>−6</sup> for proximal enhancer <em>vs.</em> neutral reference, <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>); (2) <em>tele</em>-enhancers are enriched in low-DAF SNPs compared with proximal counterpart (binomial test, <em>P</em> = 1 × 10<sup>−2</sup>, <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>).</p><p id="p-44">Also, we further partitioned <em>tele</em>-enhancers into subgroups—intronic and intergenic <em>tele</em>-enhancers with a premise that genomic locations of sequences may partially determine their evolutionary patterns (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Halligan <em>et al.</em> 2004</a>). We observed intronic <em>tele</em>-enhancers being under stronger selection constraint than intergenic tele-enhancers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS6.pdf">Table S6</a>), which is in accordance with the finding that functional intronic regions are under stronger selection pressure as compared to intergenic counterpart in mammals (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Davidson <em>et al.</em> 2009</a>). Furthermore, we compared intronic and intergenic <em>tele</em>-enhancers with their proximal counterparts, and observed that <em>tele</em>-enhancers are consistently more conserved than their proximal counterparts. In summary, <em>tele</em>-enhancers (1) are enriched for conserved sequences (those with &gt;0.20 phastCons, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/FigureS3.pdf">Figure S3</a>, where the weak significant <em>P</em>-value is partially due to the small sample pool used for statistical analysis); (2) show lower NIs; (3) have low SNP density; and (4) exhibit preference toward low-DAF SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS6.pdf">Table S6</a>).</p><p id="p-45">Finally, both the nucleotide divergence and SNP-based analyses lead to the same conclusion that although heart enhancers are under strong negative selection, the selection pressure acting on <em>tele</em>-enhancers (either intergenic or intronic) is even stronger than that on proximal counterparts. This is in accordance with the finding that many highly conserved enhancers are separated from their target genes by “bystander” genes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akalin <em>et al.</em> 2009</a>). Also, since <em>tele</em> and proximal enhancers share similar functions (for example, both these enhancer types play a role in heart development and heart morphogenesis), display common cardiac TF binding motifs (such as those of MEF2A) and exhibit common selective features (both of them are highly conserved as compared to controls), it could be expected that the evolutionary constraint difference between <em>tele</em> and proximal enhancers is only weakly significant (<em>P</em>-values are between 0.001 and 0.05) in almost all cases.</p></div><div id="sec-19" class="subsection"><h3><em>Tele</em>-enhancers from different tissues show consistent evolutionary and functional features</h3><p id="p-46">We extended the study to other tissues, including fetal brain and lung, in an effort to analyze the generalizability of our results. In each tissue, we collected potential enhancers based on ChIP-seq experiments targeting H3K4me1, an enhancer-associated histone mark, along with their target genes retrieved using gene expression profiles and Gene Ontology gene annotations. We observed that a large fraction of highly-expressed genes were GeneTs, <em>i.e.</em>, the genes with no proximal enhancers. For example, 47% of brain genes and 50% of lung genes were GeneTs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-47">In the case of lung and brain enhancers, similarly to heart enhancers, <em>tele</em>-enhancers were strongly associated with the development of the corresponding tissue (multitest-corrected hypergeometric distribution <em>P</em>-values = 0, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> as well as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS7.pdf">Table S7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS8.pdf">Table S8</a>). In addition, <em>tele</em>-enhancers were more strongly associated with basic processes than proximal enhancers (multi-test corrected hypergeometric distribution P-values &lt; 1 × 10<sup>−3</sup>). For example, among 23 genes regulating the response of nutrient level, which have influence on brain development (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Georgieff 2007</a>), six genes (26%) were brain GeneTs, whereas none were GenePs. Also lung GeneTs, not GenePs, were significantly enriched for the genes taking part in GTP metabolic and catabolic process [those genes play an essential role in structural patterning during lung development (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wan <em>et al.</em> 2013</a>)]. Furthermore, GeneTs in brain displayed significantly lower relative expression than GenePs (the average of relative expressions of GeneTs and GenePs was 1.4 and 1.9; Wilcoxon rank sum test <em>P</em>-value = 3 × 10<sup>−22</sup>). Similarly, in lung, the relative expression of GeneTs was significantly lower than that of GenePs (the average of relative expression levels of GeneTs and GenePs was 1.9 and 2.1, respectively; <em>P</em>-value = 2 × 10<sup>−4</sup>). These suggest that GeneTs in brain and lung show relatively low tissue specificity in brain and lung, similar to GeneTs in heart.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464234/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464234/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Functional analysis of proximal and <em>tele</em>- lung enhancers</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464172/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464172/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Functional analysis of proximal and <em>tele</em>- brain enhancers</span><div class="sb-div caption-clear"></div></div></div><p id="p-50">The analysis of sequence divergence of these enhancers indicated that the enhancers activated in different tissues evolve under negative selection but at a different degree of evolutionary constraint. With the lowest nucleotide divergence, brain enhancers were much more conserved than heart enhancers, which is consistent with previous reports (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Blow <em>et al.</em> 2010</a>) (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). <em>Tele</em>-enhancers exhibited significant lower human-specific divergence than proximal enhancers (Fisher’s exact test <em>P</em>-value = 5 × 10<sup>−3</sup> for brain, and <em>P</em>-value = 1 × 10<sup>−2</sup> for lung, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), suggesting stronger negative constraints imposed on <em>tele</em>-enhancers than on their proximal counterparts across all tested tissues. Additionally, SNP-based results consistently showed that <em>tele</em>-enhancers harbor less SNPs than proximal enhancers across all tested tissues. In heart and lung, the SNP density difference between <em>tele</em>- and proximal enhancers was significant (binomial test <em>P</em>-values &lt; 4 × 10<sup>−4</sup>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>), whereas this difference in brain was not significant. Also, <em>tele</em>-enhancers contained more low-DAF SNPs than proximal enhancers in all tested tissues (binomial test <em>P</em>-value = 3 × 10<sup>−2</sup> for heart and lung, and <em>P</em>-value = 7 × 10<sup>−2</sup> for brain, <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464199/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464199/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Nucleotide divergence of lung and brain enhancers according to the human-chimpanzee-macaque sequence alignment</span><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/579/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Single-nucleotide polymorphism (SNP)-based analysis of enhancers in three tissues (heart, brain, and lung). (A) Number of SNPs per kilobase of tele- and proximal enhancers. (B) Fraction of SNPs with low derived allele frequency (DAF) in tele- and proximal enhancers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-77238227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Single-nucleotide polymorphism (SNP)-based analysis of enhancers in three tissues (heart, brain, and lung). (A) Number of SNPs per kilobase of &lt;em&gt;tele&lt;/em&gt;- and proximal enhancers. (B) Fraction of SNPs with low derived allele frequency (DAF) in &lt;em&gt;tele&lt;/em&gt;- and proximal enhancers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/579/F7.medium.gif" width="440" height="144"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/4/579/F7.medium.gif" width="440" height="144"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/579/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/579/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464221" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-52" class="first-child">Single-nucleotide polymorphism (SNP)-based analysis of enhancers in three tissues (heart, brain, and lung). (A) Number of SNPs per kilobase of <em>tele</em>- and proximal enhancers. (B) Fraction of SNPs with low derived allele frequency (DAF) in <em>tele</em>- and proximal enhancers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">Taken together, <em>tele</em>-enhancers, although having different selective signatures across different tissues, have been consistently evolving under stronger negative constraints than proximal enhancers both during modern human history and during the separation of vertebrates, indicating that the observations we obtained from the study of heart enhancers are applicable to other tissues and represent a general trend in the evolution of proximal enhancers and <em>tele</em>-enhancers.</p></div><div id="sec-20" class="subsection"><h3><em>Tele</em>-enhancers from diverse cell types show consistent evolutionary and functional features</h3><p id="p-54">We also extended our study to a large panel of diverse cell types, including GM12878, 1H-hESC, HepG2, HSMM, HUVEC, K562, and NHEK, for which gene expression and ChromHMM enhancer maps have been previously reported (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Djebali <em>et al.</em> 2012</a>) (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Ernst and Kellis 2012</a>). We applied our pipeline to each of these cells and identified ∼6000 <em>tele</em>-enhancers in different cell lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS2.pdf">Table S2</a>).</p><p id="p-55">In agreement with our heart <em>tele</em>-enhancer observations, GeneTs were found to be enriched in house-keeping genes as compared to the respective GenePs in all cell types. In all cells, GeneTs showed lower relative expression than GenePs. In all cases except K256, the relative-expression difference between GeneTs and GenePs was significant (Wilcoxon rank sum test <em>P</em>-values &lt; 2 × 10<sup>−3</sup>, <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). These observations indicate that <em>tele</em>-enhancers likely play basic and fundamental biological roles independent of their cell-type specificity. Across all cell types, <em>tele</em>-enhancers featured remarkably lower human-specific divergence than their proximal counterparts (Fisher’s exact test <em>P</em>-values &lt; 2 × 10<sup>−3</sup>) and neutrally evolving DNA (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/TableS9.pdf">Table S9</a>), confirming the uniform nature of strong purifying selection acting on them. Moreover, <em>tele</em>-enhancers displayed a significantly lower SNP density across all cell types as compared to proximal enhancers (binomial test <em>P</em>-values &lt; 7 × 10<sup>−6</sup>, <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8C</a>). They also displayed a strong preference toward low-DAF SNPs—the fraction of low-DAF SNPs in <em>tele</em>-enhancers was lower than that in respective proximal enhancers in all cell types (binomial test <em>P</em>-values &lt; 4 × 10<sup>−3</sup>, <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8D</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/579/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Features of tele-enhancers as compared to the respective proximal enhancers in seven cell types. (A) Relative expression of GeneTs and GenePs. (B) Nucleotide divergence of tele- and proximal enhancers. (C) Number of single-nucleotide polymorphisms (SNPs) per kilobase. All the differences between tele- and proximal enhancers are significant, i.e., binomial test P-values &lt; 7 &#xD7; 10&#x2212;6. (D) Fraction of SNPs having a low derived allele frequency (DAF &lt; 0.05). All differences between tele- and proximal enhancers are significant, i.e., P-values &lt; 4 &#xD7; 10&#x2212;3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-77238227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Features of &lt;em&gt;tele&lt;/em&gt;-enhancers as compared to the respective proximal enhancers in seven cell types. (A) Relative expression of GeneTs and GenePs. (B) Nucleotide divergence of &lt;em&gt;tele&lt;/em&gt;- and proximal enhancers. (C) Number of single-nucleotide polymorphisms (SNPs) per kilobase. All the differences between &lt;em&gt;tele&lt;/em&gt;- and proximal enhancers are significant, &lt;em&gt;i.e.&lt;/em&gt;, binomial test &lt;em&gt;P&lt;/em&gt;-values &lt; 7 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;. (D) Fraction of SNPs having a low derived allele frequency (DAF &lt; 0.05). All differences between &lt;em&gt;tele&lt;/em&gt;- and proximal enhancers are significant, &lt;em&gt;i.e.&lt;/em&gt;, &lt;em&gt;P&lt;/em&gt;-values &lt; 4 &#xD7; 10&lt;sup&gt;&#x2212;3&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/579/F8.medium.gif" width="218" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/4/579/F8.medium.gif" width="218" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/579/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/579/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464118" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-56" class="first-child">Features of <em>tele</em>-enhancers as compared to the respective proximal enhancers in seven cell types. (A) Relative expression of GeneTs and GenePs. (B) Nucleotide divergence of <em>tele</em>- and proximal enhancers. (C) Number of single-nucleotide polymorphisms (SNPs) per kilobase. All the differences between <em>tele</em>- and proximal enhancers are significant, <em>i.e.</em>, binomial test <em>P</em>-values &lt; 7 × 10<sup>−6</sup>. (D) Fraction of SNPs having a low derived allele frequency (DAF &lt; 0.05). All differences between <em>tele</em>- and proximal enhancers are significant, <em>i.e.</em>, <em>P</em>-values &lt; 4 × 10<sup>−3</sup>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-57">Understanding chromatin looping and its role in accurately positioning long-range enhancers into the immediate proximity of their target promoters has remained one of the most challenging problems of the postgenome era (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carter <em>et al.</em> 2002</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ernst 2012</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Sanyal <em>et al.</em> 2012</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sexton <em>et al.</em> 2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">West and Fraser 2005</a>). We observed that an underappreciated, large fraction of tissue-specific genes—64% of heart genes, 47% of brain genes, and 50% of lung genes—lack proximal enhancers and are being regulated by so-called <em>tele</em>-enhancers (enhancers that are separated from their target genes by at least one bystander gene or exon). This explicitly demonstrated that spatial proximity is not the mechanism by which enhancers activate their target genes and a large portion of enhancers might recognize their distant partners while “skipping” bystander genes (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Zhang <em>et al.</em> 2013</a>). However, our knowledge of the genome-wide distribution of <em>tele</em>-enhancers is very limited because previous studies of these requlatory elements focused on individual sequences instead of a genome-wide distribution (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Dong <em>et al.</em> 2009</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Navratilova and Becker 2009</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Sanyal <em>et al.</em> 2012</a>). To improve the understanding of <em>tele</em>-enhancers, we analyzed genome-wide enhancer maps established in a panel of tissues/cell types based on the activity of different biomarkers, such as the transcriptional coactivator p300, enhancer-associated histone mark H3K4me1, and a combination of regulatory-related chromatin signals. After assigning enhancers to their target genes on a genome-wide scale, we established the maps of <em>tele</em>-enhancers in three tissues (fetal heart, brain, and lung) and seven cell types (such as GM12878, H1-hESC, K562, etc.). We then compared <em>tele</em>-enhancers to proximal enhancers systematically and investigated functional, regulatory, and evolutionary mechanisms specific to <em>tele</em>-enhancers.</p><p id="p-58">We demonstrated that the genes associated with heart <em>tele</em>-enhancers (GeneTs) partake in basic biological processes, showing lower heart specificity than genes associated with proximal enhancers (GeneP). Also, heart GeneTs have significantly shorter noncoding space in their neighborhood than GenePs. These findings are in line with the “selection for economy” model stating that widely-expressed gene loci are compact due to strong pressure for shortening non-coding regions and this might explain why they rely on <em>tele</em>-enhancers for transcription activation (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Eisenberg and Levanon 2003</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Pozzoli <em>et al.</em> 2007</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Vinogradov 2006</a>). Also, heart <em>tele</em>-enhancers have a TF binding motif signature distinct from proximal heart enhancers. For example, the binding motifs of TEAD and NKX-2.5 were overrepresented in <em>tele</em>-enhancers comparing to proximal enhancers whereas GATA4 displayed an opposite trend. These finding suggests that heart <em>tele</em>-enhancers, as compared to proximal counterparts, regulate distinct biological processes, and recruit different transcriptional activators.</p><p id="p-59">We also measured the nucleotide divergence of heart enhancers between human, chimpanzee, and macaque and observed that heart <em>tele</em>-enhancers displayed low human-specific divergence. Also, heart <em>tele</em>-enhancers harbored less SNPs and more likely contained low-DAF SNPs compared with a neutral reference and their proximal counterparts. All these findings consistently suggested that heart <em>tele</em>-enhancers are under stronger negative selective pressure than proximal counterparts. Also <em>tele</em>- and proximal enhancers showed almost identical GC content, CpG site density, and ChIP-seq signal magnitude (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1/FigureS4.pdf">Figure S4</a>). After eliminating these possible confounding factors, we further ascertained that the genomic location of enhancers (<em>i.e.</em>, the position relative to potential target genes) was one of the determinant factors of functional and evolutionary signatures of heart enhancers.</p><p id="p-60">We extended our study to fetal brain and lung, where enhancers were identified in ChIP-seq experiments targeting H3K4me1, and a panel of seven cell lines where enhancers were predicted according to chromatin signatures. The obtained results suggest that our results represent a common trend across different tissues and cell types.</p><p id="p-61">How enhancers “travel” over intermediate regions and interact with remote core promoters to initiate transcription is one of the most enigmatic aspects of gene regulation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kleinjan and van Heyningen 2005</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Phillips and Corces 2009</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Sexton <em>et al.</em> 2009</a>). Our findings shed light on the interactions between remote enhancers and their targets, which are directly relevant to the development of future strategies for analyzing <em>tele</em>-enhancers and understanding their role in establishing complex gene regulatory landscapes of vertebrate genomes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We are grateful to Mohamed Aniba and Cindy Clark for critical reading of the manuscript. This study was supported by the Intramural Research Program of the National Institutes of Health, National Library of Medicine.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010447/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-63"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-01-24"><span class="received-label">Received </span>January 24, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-01-26"><span class="accepted-label">Accepted </span>January 26, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Huang and Ovcharenko</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.4.579.1" data-doi="10.1093/hmg/ddi338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahituv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhakar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couronne</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mapping cis-regulatory domains in the human genome using multi-species conservation of synteny.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">3057</span>–<span class="cit-lpage">3063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddi338%26rft_id%253Dinfo%253Apmid%252F16155111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8yMC8zMDU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.2" data-doi="10.1186/gb-2009-10-4-r38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryne</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transcriptional features of genomic regulatory blocks.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAkalin%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253DR38%26rft.epage%253DR38%26rft.atitle%253DTranscriptional%2Bfeatures%2Bof%2Bgenomic%2Bregulatory%2Bblocks.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-4-r38%26rft_id%253Dinfo%253Apmid%252F19374772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-4-r38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19374772&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.3" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene Ontology: tool for the unification of biology.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.4" data-doi="10.1186/gb-2009-10-1-r2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balasubramanian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankish</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparative analysis of processed ribosomal protein pseudogenes in four mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBalasubramanian%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253DR2%26rft.epage%253DR2%26rft.atitle%253DComparative%2Banalysis%2Bof%2Bprocessed%2Bribosomal%2Bprotein%2Bpseudogenes%2Bin%2Bfour%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-1-r2%26rft_id%253Dinfo%253Apmid%252F19123937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-1-r2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19123937&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.5" data-doi="10.1016/j.ymeth.2012.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belton</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCord</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibcus</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naumova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hi–C: a comprehensive technique to capture the conformation of genomes.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DBelton%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D268%26rft.epage%253D276%26rft.atitle%253DHi-C%253A%2BA%2Bcomprehensive%2Btechnique%2Bto%2Bcapture%2Bthe%2Bconformation%2Bof%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2012.05.001%26rft_id%253Dinfo%253Apmid%252F22652625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2012.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22652625&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312360000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.6" data-doi="10.1038/ng.650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blow</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulley</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akiyama</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ChIP-Seq identification of weakly conserved heart enhancers.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">806</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBlow%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D42%26rft.issue%253D9%26rft.spage%253D806%26rft.epage%253D810%26rft.atitle%253DChIP-Seq%2Bidentification%2Bof%2Bweakly%2Bconserved%2Bheart%2Benhancers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.650%26rft_id%253Dinfo%253Apmid%252F20729851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20729851&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281388400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.7" data-doi="10.1371/journal.pgen.1002531"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Busser</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tansey</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A machine learning approach for identifying novel cell type–specific transcriptional regulators of myogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBusser%26rft.auinit1%253DB.%2BW.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002531%26rft.epage%253De1002531%26rft.atitle%253DA%2Bmachine%2Blearning%2Bapproach%2Bfor%2Bidentifying%2Bnovel%2Bcell%2Btype-specific%2Btranscriptional%2Bregulators%2Bof%2Bmyogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002531%26rft_id%253Dinfo%253Apmid%252F22412381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002531&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22412381&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.8" data-doi="10.1038/ng1051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakalova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborne</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">Y.-f.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Long-range chromatin regulatory interactions in vivo.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCarter%26rft.auinit1%253DD.%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D623%26rft.epage%253D626%26rft.atitle%253DLong-range%2Bchromatin%2Bregulatory%2Binteractions%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1051%26rft_id%253Dinfo%253Apmid%252F12426570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12426570&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179593000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-other" id="cit-4.4.579.9" data-doi="10.1101/gr.7.12.1123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carver</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stubbs</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>  <span class="cit-article-title">Zooming in on the human–mouse comparative map: genome conservation re-examined on a high-resolution scale.</span> <span class="cit-source">Genome Res.</span> <span class="cit-vol">7</span>: <span class="cit-fpage">1123</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCarver%26rft.auinit1%253DE.%2BA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1123%26rft.epage%253D1137%26rft.atitle%253DZooming%2Bin%2Bon%2Bthe%2BHuman-Mouse%2BComparative%2BMap%253A%2BGenome%2BConservation%2BRe-examined%2Bon%2Ba%2BHigh-Resolution%25A0Scale%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7.12.1123%26rft_id%253Dinfo%253Apmid%252F9414318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjcvMTIvMTEyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC81NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.10" data-doi="10.1371/journal.pgen.1002852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanderMeer</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaar</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahituv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Human developmental enhancers conserved between deuterostomes and protostomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DClarke%26rft.auinit1%253DS.%2BL.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002852%26rft.epage%253De1002852%26rft.atitle%253DHuman%2Bdevelopmental%2Benhancers%2Bconserved%2Bbetween%2Bdeuterostomes%2Band%2Bprotostomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002852%26rft_id%253Dinfo%253Apmid%252F22876195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002852&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22876195&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.11" data-doi="10.1007/BF00994018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cortes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vapnik</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Support-vector networks.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D20%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00994018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00994018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RX35400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.12" data-doi="10.1038/sj.mp.4001787"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dowell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gildea</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A remote and highly conserved enhancer supports amygdala specific expression of the gene encoding the anxiogenic neuropeptide substance-P.</span> <abbr class="cit-jnl-abbrev">Mol. Psychiatry</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">410</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPsychiatry%26rft.volume%253D11%26rft.spage%253D410%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.mp.4001787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.mp.4001787&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000236339700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.13" data-doi="10.1186/1471-2164-10-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starkey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evidence of uneven selective pressure on different subsets of the conserved human genome; implications for the significance of intronic and intergenic DNA.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DJaari%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DA%2Bfirst-generation%2Bmicrosatellite-based%2Bgenetic%2Blinkage%2Bmap%2Bof%2Bthe%2BSiberian%2Bjay%2B%2528Perisoreus%2Binfaustus%2529%253A%2Binsights%2Binto%2Bavian%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-1%26rft_id%253Dinfo%253Apmid%252F19121221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19121221&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.14" data-doi="10.1038/nature11082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selvaraj</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Topological domains in mammalian genomes identified by analysis of chromatin interactions.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDixon%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D485%26rft.issue%253D7398%26rft.spage%253D376%26rft.epage%253D380%26rft.atitle%253DTopological%2Bdomains%2Bin%2Bmammalian%2Bgenomes%2Bidentified%2Bby%2Banalysis%2Bof%2Bchromatin%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11082%26rft_id%253Dinfo%253Apmid%252F22495300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22495300&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304099100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.15" data-doi="10.1038/nature11233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lassmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Landscape of transcription in human cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDjebali%26rft.auinit1%253DS.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D101%26rft.epage%253D108%26rft.atitle%253DLandscape%2Bof%2Btranscription%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11233%26rft_id%253Dinfo%253Apmid%252F22955620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955620&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.16" data-doi="10.1186/gb-2009-10-1-r1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenhard</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Synorth: exploring the evolution of synteny and long-range regulatory interactions in vertebrate genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-1-r1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-1-r1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.17" data-doi="10.1016/j.devcel.2011.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drenckhahn</span>  <span class="cit-name-given-names">J.-D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Heart development: mitochondria in command of cardiomyocyte differentiation.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DDrenckhahn%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D392%26rft.epage%253D393%26rft.atitle%253DHeart%2Bdevelopment%253A%2Bmitochondria%2Bin%2Bcommand%2Bof%2Bcardiomyocyte%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2011.08.021%26rft_id%253Dinfo%253Apmid%252F21920309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2011.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21920309&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.18" data-doi="10.2307/2282330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">O. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title"> Multiple Comparisons among Means.</span> <abbr class="cit-jnl-abbrev">Journal of the American Statistical Association.</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">293</span>): <span class="cit-fpage">52</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BStatistical%2BAssociation.%26rft.volume%253D56%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2282330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2282330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19611734300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.19" data-doi="10.1016/S0168-9525(03)00140-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisenberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levanon</span>  <span class="cit-name-given-names">E. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Human housekeeping genes are compact.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DEisenberg%26rft.auinit1%253DE.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DHuman%2Bhousekeeping%2Bgenes%2Bare%2Bcompact.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252803%252900140-9%26rft_id%253Dinfo%253Apmid%252F12850439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(03)00140-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12850439&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184538400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.20" data-doi="10.1101/gr.6669607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engström</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho Sui</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drivenes</span>  <span class="cit-name-given-names">Ø.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenhard</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic regulatory blocks underlie extensive microsynteny conservation in insects.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1898</span>–<span class="cit-lpage">1908</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6669607%26rft_id%253Dinfo%253Apmid%252F17989259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8xODk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.21" data-doi="10.1038/cr.2012.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mapping enhancer and promoter interactions.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.stitle%253DCell%2BRes%26rft.aulast%253DErnst%26rft.auinit1%253DJ.%26rft.volume%253D22%26rft.issue%253D5%26rft.spage%253D789%26rft.epage%253D790%26rft.atitle%253DMapping%2Benhancer%2Band%2Bpromoter%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2012.49%26rft_id%253Dinfo%253Apmid%252F22453239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2012.49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22453239&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303611500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.22" data-doi="10.1038/nmeth.1906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ChromHMM: automating chromatin-state discovery and characterization.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DErnst%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D215%26rft.epage%253D216%26rft.atitle%253DChromHMM%253A%2Bautomating%2Bchromatin-state%2Bdiscovery%2Band%2Bcharacterization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1906%26rft_id%253Dinfo%253Apmid%252F22373907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22373907&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300890400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.23" data-doi="10.1038/nrg1925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaszner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felsenfeld</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Insulators: exploiting transcriptional and epigenetic mechanisms.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBanzhaf%26rft.auinit1%253DW.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253D703%26rft.epage%253D713%26rft.atitle%253DInsulators%253A%2Bexploiting%2Btranscriptional%2Band%2Bepigenetic%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1925%26rft_id%253Dinfo%253Apmid%252F16909129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16909129&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239941900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Georgieff</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Nutrition and the developing brain: nutrient priorities and measurement.</span> <abbr class="cit-jnl-abbrev">Am. J. Clin. Nutr.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">614S</span>–<span class="cit-lpage">620S</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BJournal%2Bof%2BClinical%2BNutrition%26rft.stitle%253DAm%2BJ%2BClin%2BNutr%26rft.aulast%253DGeorgieff%26rft.auinit1%253DM.%2BK%26rft.volume%253D85%26rft.issue%253D2%26rft.spage%253D614S%26rft.epage%253D620S%26rft.atitle%253DNutrition%2Band%2Bthe%2Bdeveloping%2Bbrain%253A%2Bnutrient%2Bpriorities%2Band%2Bmeasurement%26rft_id%253Dinfo%253Apmid%252F17284765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpjbiI7czo1OiJyZXNpZCI7czo5OiI4NS8yLzYxNFMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.25" data-doi="10.4161/cc.10.20.17842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glinskii</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">C.-U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Networks of intergenic long-range enhancers and snpRNAs drive castration-resistant phenotype of prostate cancer and contribute to pathogenesis of multiple common human disorders.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">3571</span>–<span class="cit-lpage">3597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DGlinskii%26rft.auinit1%253DA.%2BB.%26rft.volume%253D10%26rft.issue%253D20%26rft.spage%253D3571%26rft.epage%253D3597%26rft.atitle%253DNetworks%2Bof%2Bintergenic%2Blong-range%2Benhancers%2Band%2BsnpRNAs%2Bdrive%2Bcastration-resistant%2Bphenotype%2Bof%2Bprostate%2Bcancer%2Band%2Bcontribute%2Bto%2Bpathogenesis%2Bof%2Bmultiple%2Bcommon%2Bhuman%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.10.20.17842%26rft_id%253Dinfo%253Apmid%252F22067658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.10.20.17842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22067658&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296570700036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.26" data-doi="10.1016/j.cardiores.2004.06.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffart</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Kleist-Retzow</span>  <span class="cit-name-given-names">J.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiesner</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Regulation of mitochondrial proliferation in the heart: power-plant failure contributes to cardiac failure in hypertrophy.</span> <abbr class="cit-jnl-abbrev">Cardiovasc. Res.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCardiovascular%2BResearch%26rft.stitle%253DCardiovasc%2BRes%26rft.aulast%253DGoffart%26rft.auinit1%253DS.%26rft.volume%253D64%26rft.issue%253D2%26rft.spage%253D198%26rft.epage%253D207%26rft.atitle%253DRegulation%2Bof%2Bmitochondrial%2Bproliferation%2Bin%2Bthe%2Bheart%253A%2Bpower-plant%2Bfailure%2Bcontributes%2Bto%2Bcardiac%2Bfailure%2Bin%2Bhypertrophy%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cardiores.2004.06.030%26rft_id%253Dinfo%253Apmid%252F15485678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTM6ImNhcmRpb3Zhc2NyZXMiO3M6NToicmVzaWQiO3M6ODoiNjQvMi8xOTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.27" data-doi="10.1101/gr.102210.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goode</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzales</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolutionary constraint facilitates interpretation of genetic variation in resequenced human genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102210.109%26rft_id%253Dinfo%253Apmid%252F20067941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMzAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.28" data-doi="10.1101/gr.1329204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Patterns of evolutionary constraints in intronic and intergenic DNA of Drosophila.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHalligan%26rft.auinit1%253DD.%2BL.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D273%26rft.epage%253D279%26rft.atitle%253DPatterns%2Bof%2BEvolutionary%2BConstraints%2Bin%2BIntronic%2Band%2BIntergenic%2BDNA%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1329204%26rft_id%253Dinfo%253Apmid%252F14762063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMjczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.29" data-doi="10.1172/JCI200524405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huss</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mitochondrial energy metabolism in heart failure: a question of balance.</span> <abbr class="cit-jnl-abbrev">J. Clin. Invest.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.stitle%253DJCI%26rft.aulast%253DHuss%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D115%26rft.issue%253D3%26rft.spage%253D547%26rft.epage%253D555%26rft.atitle%253DMitochondrial%2Benergy%2Bmetabolism%2Bin%2Bheart%2Bfailure%253A%2Ba%2Bquestion%2Bof%2Bbalance%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI200524405%26rft_id%253Dinfo%253Apmid%252F15765136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI200524405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15765136&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227392700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-other" id="cit-4.4.579.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irimia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royo</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burguera</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeso</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gómez-Skarmeta</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>  <span class="cit-article-title">Comparative genomics of the Hedgehog loci in chordates and the origins of Shh regulatory novelties.</span> <span class="cit-source">Sci. Rep.</span> <span class="cit-vol">2</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.31" data-doi="10.1093/bioinformatics/bts251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baseler</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DAVID-WS: a stateful web service to facilitate gene/protein list analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1805</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts251%26rft_id%253Dinfo%253Apmid%252F22543366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTMvMTgwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC81NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-4.4.579.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Evolution of protein molecules</span>, pp. <span class="cit-fpage">21</span>–<span class="cit-lpage">132</span> in <span class="cit-source">Mammalian Protein Metabolism</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Munro</span>  <span class="cit-name-given-names">H. N.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.33" data-doi="10.1101/gr.6086307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kikuta</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laplante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navratilova</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Komisarczuk</span>  <span class="cit-name-given-names">A. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engström</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic regulatory blocks encompass multiple neighboring genes and maintain conserved synteny in vertebrates.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6086307%26rft_id%253Dinfo%253Apmid%252F17387144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzUvNTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.34" data-doi="10.1086/426833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kleinjan</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heyningen</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Long-range control of gene expression: emerging mechanisms and disruption in disease.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DKleinjan%26rft.auinit1%253DD.%2BA.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D8%26rft.epage%253D32%26rft.atitle%253DLong-range%2Bcontrol%2Bof%2Bgene%2Bexpression%253A%2Bemerging%2Bmechanisms%2Band%2Bdisruption%2Bin%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F426833%26rft_id%253Dinfo%253Apmid%252F15549674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/426833&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15549674&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225643600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.35" data-doi="10.1093/hmg/ddg180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lettice</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heaney</span>  <span class="cit-name-given-names">S. J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purdie</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Beer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A long-range Shh enhancer regulates expression in the developing limb and fin and is associated with preaxial polydactyly.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1725</span>–<span class="cit-lpage">1735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DLettice%26rft.auinit1%253DL.%2BA.%26rft.volume%253D12%26rft.issue%253D14%26rft.spage%253D1725%26rft.epage%253D1735%26rft.atitle%253DA%2Blong-range%2BShh%2Benhancer%2Bregulates%2Bexpression%2Bin%2Bthe%2Bdeveloping%2Blimb%2Band%2Bfin%2Band%2Bis%2Bassociated%2Bwith%2Bpreaxial%2Bpolydactyly%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddg180%26rft_id%253Dinfo%253Apmid%252F12837695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xNC8xNzI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.36" data-doi="10.1186/gb-2010-11-2-r22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fullwood</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulawadi</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velkov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ChIA-PET tool for comprehensive chromatin interaction analysis with paired-end tag sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLi%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253DR22%26rft.epage%253DR22%26rft.atitle%253DChIA-PET%2Btool%2Bfor%2Bcomprehensive%2Bchromatin%2Binteraction%2Banalysis%2Bwith%2Bpaired-end%2Btag%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-2-r22%26rft_id%253Dinfo%253Apmid%252F20181287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-2-r22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20181287&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.37" data-doi="10.1182/blood-2002-04-1104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatoyannopoulos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Locus control regions.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">3077</span>–<span class="cit-lpage">3086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2002-04-1104%26rft_id%253Dinfo%253Apmid%252F12384402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzkvMzA3NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC81NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.38" data-doi="10.1002/bies.20117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collinson</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Is there a functional link between gene interdigitation and multi-species conservation of synteny blocks?</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DMackenzie%26rft.auinit1%253DA.%26rft.volume%253D26%26rft.issue%253D11%26rft.spage%253D1217%26rft.epage%253D1224%26rft.atitle%253DIs%2Bthere%2Ba%2Bfunctional%2Blink%2Bbetween%2Bgene%2Binterdigitation%2Band%2Bmulti-species%2Bconservation%2Bof%2Bsynteny%2Bblocks%253F%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20117%26rft_id%253Dinfo%253Apmid%252F15499588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15499588&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224909200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.39" data-doi="10.1146/annurev.genom.7.080505.115623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maston</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Transcriptional regulatory elements in the human genome.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.volume%253D7%26rft.spage%253D29%26rft.epage%253D59%26rft.atitle%253DTranscriptional%2Bregulatory%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.7.080505.115623%26rft_id%253Dinfo%253Apmid%252F16719718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.7.080505.115623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16719718&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242032200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.40" data-doi="10.1093/nar/gkj143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matys</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kel-Margoulis</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fricke</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebich</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D108</span>–<span class="cit-lpage">D110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMatys%26rft.auinit1%253DV.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD108%26rft.epage%253DD110%26rft.atitle%253DTRANSFAC%2528R%2529%2Band%2Bits%2Bmodule%2BTRANSCompel%2528R%2529%253A%2Btranscriptional%2Bgene%2Bregulation%2Bin%2Beukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj143%26rft_id%253Dinfo%253Apmid%252F16381825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8xL0QxMDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.41" data-doi="10.1038/ng.1006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blow</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulley</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Large-scale discovery of enhancers from human heart tissue.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMay%26rft.auinit1%253DD.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D93%26rft.atitle%253DLarge-scale%2Bdiscovery%2Bof%2Benhancers%2Bfrom%2Bhuman%2Bheart%2Btissue.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1006%26rft_id%253Dinfo%253Apmid%252F22138689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22138689&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.42" data-doi="10.1038/351652a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Adaptive protein evolution at the Adh locus in Drosophila.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcDonald%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D351%26rft.issue%253D6328%26rft.spage%253D652%26rft.epage%253D654%26rft.atitle%253DAdaptive%2Bprotein%2Bevolution%2Bat%2Bthe%2BAdh%2Blocus%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F351652a0%26rft_id%253Dinfo%253Apmid%252F1904993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/351652a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1904993&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FT11200066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.43" data-doi="10.1101/gr.098657.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narlikar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakabe</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanski</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arimura</span>  <span class="cit-name-given-names">F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westlund</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide discovery of human heart enhancers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.098657.109%26rft_id%253Dinfo%253Apmid%252F20075146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvMzgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.44" data-doi="10.1093/bfgp/elp019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navratilova</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">T. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic regulatory blocks in vertebrates and implications in human disease.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics Proteomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felp019%26rft_id%253Dinfo%253Apmid%252F19561171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI4LzQvMzMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.45" data-doi="10.1101/gr.3007205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ovcharenko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loots</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giardine</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span> <span class="cit-article-title">Mulan: multiple-sequence local alignment and visualization for studying function and evolution.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3007205%26rft_id%253Dinfo%253Apmid%252F15590941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzEvMTg0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.46" data-doi="10.1101/gr.3015505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ovcharenko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loots</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nobrega</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardison</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span> <span class="cit-article-title">Evolution and functional classification of vertebrate gene deserts.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3015505%26rft_id%253Dinfo%253Apmid%252F15590943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzEvMTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.47" data-doi="10.1016/j.cub.2010.07.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boettiger</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bothma</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Shadow enhancers foster robustness of Drosophila gastrulation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1562</span>–<span class="cit-lpage">1567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPerry%26rft.auinit1%253DM.%2BW.%26rft.volume%253D20%26rft.issue%253D17%26rft.spage%253D1562%26rft.epage%253D1567%26rft.atitle%253DShadow%2Benhancers%2Bfoster%2Brobustness%2Bof%2BDrosophila%2Bgastrulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.07.043%26rft_id%253Dinfo%253Apmid%252F20797865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.07.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20797865&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281941100034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.48" data-doi="10.1073/pnas.1330369100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pevzner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Human and mouse genomic sequences reveal extensive breakpoint reuse in mammalian evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">7672</span>–<span class="cit-lpage">7677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1330369100%26rft_id%253Dinfo%253Apmid%252F12810957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzEzLzc2NzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.49" data-doi="10.1016/j.cell.2009.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corces</span>  <span class="cit-name-given-names">V. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">CTCF: master weaver of the genome.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1194</span>–<span class="cit-lpage">1211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPhillips%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D137%26rft.issue%253D7%26rft.spage%253D1194%26rft.epage%253D1211%26rft.atitle%253DCTCF%253A%2Bmaster%2Bweaver%2Bof%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.06.001%26rft_id%253Dinfo%253Apmid%252F19563753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19563753&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267373400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.50" data-doi="10.1038/ng.403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pomerantz</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmadiyeh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verzi</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The 8q24 cancer risk variant rs6983267 shows long-range interaction with MYC in colorectal cancer.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">882</span>–<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPomerantz%26rft.auinit1%253DM.%2BM.%26rft.volume%253D41%26rft.issue%253D8%26rft.spage%253D882%26rft.epage%253D884%26rft.atitle%253DThe%2B8q24%2Bcancer%2Brisk%2Bvariant%2Brs6983267%2Bshows%2Blong-range%2Binteraction%2Bwith%2BMYC%2Bin%2Bcolorectal%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.403%26rft_id%253Dinfo%253Apmid%252F19561607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561607&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268432900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.51" data-doi="10.1016/j.tig.2006.10.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pozzoli</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menozzi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comi</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cagliani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bresolin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Intron size in mammals: complexity comes to terms with economy.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DPozzoli%26rft.auinit1%253DU.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D24%26rft.atitle%253DIntron%2Bsize%2Bin%2Bmammals%253A%2Bcomplexity%2Bcomes%2Bto%2Bterms%2Bwith%2Beconomy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2006.10.003%26rft_id%253Dinfo%253Apmid%252F17070957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2006.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17070957&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243709500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.52" data-doi="10.1126/science.1130738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prabhakar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noonan</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pääbo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Accelerated evolution of conserved noncoding sequences in humans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPrabhakar%26rft.auinit1%253DS.%26rft.volume%253D314%26rft.issue%253D5800%26rft.spage%253D786%26rft.epage%253D786%26rft.atitle%253DAccelerated%2BEvolution%2Bof%2BConserved%2BNoncoding%2BSequences%2Bin%2BHumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1130738%26rft_id%253Dinfo%253Apmid%252F17082449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTQvNTgwMC83ODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.53" data-doi="10.1093/nar/gkh012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandelin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkema</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engström</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenhard</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">JASPAR: an open-access database for eukaryotic transcription factor binding profiles.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">D91</span>–<span class="cit-lpage">D94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSandelin%26rft.auinit1%253DA.%26rft.volume%253D32%26rft.issue%253D90001%26rft.spage%253DD91%26rft.epage%253D94%26rft.atitle%253DJASPAR%253A%2Ban%2Bopen-access%2Bdatabase%2Bfor%2Beukaryotic%2Btranscription%2Bfactor%2Bbinding%2Bprofiles%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh012%26rft_id%253Dinfo%253Apmid%252F14681366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzMi9zdXBwbF8xL0Q5MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC81NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.54" data-doi="10.1038/nature11279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanyal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lajoie</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The long-range interaction landscape of gene promoters.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSanyal%26rft.auinit1%253DA.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D109%26rft.epage%253D113%26rft.atitle%253DThe%2Blong-range%2Binteraction%2Blandscape%2Bof%2Bgene%2Bpromoters.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11279%26rft_id%253Dinfo%253Apmid%252F22955621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955621&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.55" data-doi="10.1016/j.semcdb.2009.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sexton</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bantignies</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalli</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic interactions: chromatin loops and gene meeting points in transcriptional regulation.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DSexton%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D849%26rft.epage%253D855%26rft.atitle%253DGenomic%2Binteractions%253A%2Bchromatin%2Bloops%2Band%2Bgene%2Bmeeting%2Bpoints%2Bin%2Btranscriptional%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2009.06.004%26rft_id%253Dinfo%253Apmid%252F19559093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2009.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19559093&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270578600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.56" data-doi="10.1371/journal.pgen.0040023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alm</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparing patterns of natural selection across species using selective signatures.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DShapiro%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De23%26rft.epage%253De23%26rft.atitle%253DComparing%2Bpatterns%2Bof%2Bnatural%2Bselection%2Bacross%2Bspecies%2Busing%2Bselective%2Bsignatures.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0040023%26rft_id%253Dinfo%253Apmid%252F18266472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0040023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18266472&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.57" data-doi="10.1101/gr.3715005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bejerano</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3715005%26rft_id%253Dinfo%253Apmid%252F16024819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTAzNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC81NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.58" data-doi="10.1093/bioinformatics/16.1.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stormo</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">DNA binding sites: representation and discovery.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DStormo%26rft.auinit1%253DG.%2BD.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D23%26rft.atitle%253DDNA%2Bbinding%2Bsites%253A%2Brepresentation%2Band%2Bdiscovery%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F16.1.16%26rft_id%253Dinfo%253Apmid%252F10812473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo3OiIxNi8xLzE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.59" data-doi="10.1073/pnas.0400782101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batalov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapp</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ching</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A gene atlas of the mouse and human protein-encoding transcriptomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">6062</span>–<span class="cit-lpage">6067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0400782101%26rft_id%253Dinfo%253Apmid%252F15075390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE2LzYwNjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.60" data-doi="10.1038/nature09534"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed cit-collab">1000 Genomes Project Consortium</span></li></ol><cite>, <span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span> <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A map of human genome variation from population-scale sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.61" data-doi="10.1038/nature11232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rynes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humbert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vierstra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurano</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The accessible chromatin landscape of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DThurman%26rft.auinit1%253DR.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D75%26rft.epage%253D82%26rft.atitle%253DThe%2Baccessible%2Bchromatin%2Blandscape%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11232%26rft_id%253Dinfo%253Apmid%252F22955617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955617&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.62" data-doi="10.1146/annurev.ge.19.120185.001345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vanin</span>  <span class="cit-name-given-names">E. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Processed pseudogenes: characteristics and evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DVanin%26rft.auinit1%253DE.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D253%26rft.epage%253D272%26rft.atitle%253DProcessed%2BPseudogenes%253A%2BCharacteristics%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.19.120185.001345%26rft_id%253Dinfo%253Apmid%252F3909943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.19.120185.001345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3909943&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AWM4700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.63" data-doi="10.1101/gr.4318206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinogradov</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">“Genome design” model: evidence from conserved intronic sequence in human–mouse comparison.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4318206%26rft_id%253Dinfo%253Apmid%252F16461636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvMzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.64" data-doi="10.1101/gr.128652.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kayser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palstra</span>  <span class="cit-name-given-names">R.-J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">HERC2 rs12913832 modulates human pigmentation by attenuating chromatin loop formation between a long-range enhancer and the OCA2 promoter.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">446</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.128652.111%26rft_id%253Dinfo%253Apmid%252F22234890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNDQ2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.65" data-doi="10.1016/j.ydbio.2012.11.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wert</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CDC42 is required for structural patterning of the lung during development.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">374</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D374%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2012.11.030%26rft_id%253Dinfo%253Apmid%252F23219958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2012.11.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23219958&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.66" data-doi="10.1093/hmg/ddi104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Remote control of gene transcription.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R101</span>–<span class="cit-lpage">R111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DWest%26rft.auinit1%253DA.%2BG.%26rft.volume%253D14%26rft.issue%253Dsuppl_1%26rft.spage%253DR101%26rft.epage%253DR111%26rft.atitle%253DRemote%2Bcontrol%2Bof%2Bgene%2Btranscription%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddi104%26rft_id%253Dinfo%253Apmid%252F15809261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjE1OiIxNC9zdXBwbF8xL1IxMDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.67" data-doi="10.1093/nar/29.1.281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wingender</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fricke</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geffers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hehl</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The TRANSFAC system on gene expression regulation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWingender%26rft.auinit1%253DE.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D281%26rft.epage%253D283%26rft.atitle%253DThe%2BTRANSFAC%2Bsystem%2Bon%2Bgene%2Bexpression%2Bregulation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.1.281%26rft_id%253Dinfo%253Apmid%252F11125113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI5LzEvMjgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.68" data-doi="10.1016/j.ymeth.2012.08.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peh</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sia</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ChIA-PET analysis of transcriptional chromatin interactions.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D289%26rft.epage%253D299%26rft.atitle%253DChIA-PET%2Banalysis%2Bof%2Btranscriptional%2Bchromatin%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2012.08.009%26rft_id%253Dinfo%253Apmid%252F22926262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2012.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22926262&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F579.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312360000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.579.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birnbaum</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Favaro</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromatin connectivity maps reveal dynamic promoter-enhancer long-range associations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">504</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D504%26rft.spage%253D306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/4/579.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/561" title="Comparative Genomic and Transcriptomic Analysis of Wangiella dermatitidis, A Major Cause of Phaeohyphomycosis and a Model Black Yeast Human Pathogen" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/595" title="Identification of Major and Minor QTL for Ecologically Important Morphological Traits in Three-Spined Sticklebacks (Gasterosteus aculeatus)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/4"><span>Volume 4 Issue 4, April 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/4.cover-source.jpg?itok=Rf0YhETZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/579" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--m_MZM0ObBgLQBzhAACaSD2m9CCoZQL83buD5vCZwyM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/4/579" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/4/579" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/4/579" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Analysis of Functional and Evolutionary Features of Tele-Enhancers</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459838" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-DAMCC42BnQt6BdgSNqyRKZZSYmy-PEOOldIpIyOVvdA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/579.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/4/579" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rQRr79yu34IWcNaNTpTFr-vN185Y2TnzA5VFAx7FUD8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/4/579" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/579.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459838" id="node459838--2" data-pisa="ggg;4/4/579" data-pisa-master="ggg;g3.114.010447" data-apath="/ggg/4/4/579.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Functional and Evolutionary Features of <em>Tele</em>-Enhancers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Di Huang</span> and <span class="highwire-citation-author" data-delta="1">Ivan Ovcharenko</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">579-593;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010447 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459838--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Di Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Di%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADi%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Ovcharenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BOvcharenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ovcharenko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvan%2BOvcharenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ovcharen@nih.gov" class="" data-icon-position="" data-hide-link-title="0">ovcharen@nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/4/579" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/4/579&amp;title=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/4/579&amp;title=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/4/579&amp;title=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/4/579&amp;text=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/4/579&amp;title=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/4/579&amp;t=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/4/579&amp;title=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/4/579&amp;title=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459838/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459838" id="node459838--3" data-pisa="ggg;4/4/579" data-pisa-master="ggg;g3.114.010447" data-apath="/ggg/4/4/579.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Functional and Evolutionary Features of <em>Tele</em>-Enhancers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Di Huang</span> and <span class="highwire-citation-author" data-delta="1">Ivan Ovcharenko</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">579-593;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010447 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459838--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Di Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Di%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADi%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ivan Ovcharenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BOvcharenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ovcharenko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvan%2BOvcharenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ovcharen@nih.gov" class="" data-icon-position="" data-hide-link-title="0">ovcharen@nih.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=579&amp;rft.epage=593&amp;rft.atitle=Genome-Wide%20Analysis%20of%20Functional%20and%20Evolutionary%20Features%20of%20Tele-Enhancers&amp;rft.volume=4&amp;rft.issue=4&amp;rft.date=2014-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Huang%2C+Di&amp;rft.au=Ovcharenko%2C+Ivan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459838/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459838/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459838/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459838/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459838/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459838/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459838/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459838/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459838/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459838/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459838/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459838/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/4/579" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/4/579.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/4/579.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/4/579.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/4/579.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
