<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/4/707" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/4/707.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/4/707.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/4/707.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/4/707.atom" />
<meta name="HW.pisa" content="ggg;4/4/707" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Sequence Profiling of the Saccharomyces cerevisiae Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection" />
<meta name="DC.Identifier" content="10.1534/g3.113.009464" />
<meta name="DC.Date" content="2014-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Jubin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Advances in high-throughput sequencing (HTS) technologies have accelerated our knowledge of genomes in hundreds of organisms, but the presence of repetitions found in every genome raises challenges to unambiguously map short reads. In particular, short polymorphic reads that are multialigned hinder our capacity to detect mutations. Here, we present two complementary bioinformatics strategies to perform more robust analyses of genome content and sequencing data, validated by use of the Saccharomyces cerevisiae fully sequenced genome. First, we created an annotated HTS profile for the reference genome, based on the production of virtual HTS reads. Using variable read lengths and different numbers of mismatches, we found that 35 nt-reads, with a maximum of 6 mismatches, targets 89.5% of the genome to unique (U) regions. Longer reads consisting of 50−100 nt provided little additional benefits on the U regions extent. Second, to analyze the remaining multialigned (M) regions, we identified the intragenomic single-nucleotide variants and thus defined the unique (MU) and multialigned (MM) subregions, as exemplified for the polymorphic copies of the six flocculation genes and the 50 Ty retrotransposons. As a resource, the coordinates of the U and M regions of the yeast genome have been added to the Saccharomyces Genome Database ([www.yeastgenome.org][1]). The benefit of this advanced method of genome annotation was confirmed by our ability to identify acquired single nucleotide polymorphisms in the U and M regions of an experimentally sequenced variant wild-type yeast strain.

 [1]: http://www.yeastgenome.org" />
<meta name="DC.Contributor" content="Claire Jubin" />
<meta name="DC.Contributor" content="Alexandre Serero" />
<meta name="DC.Contributor" content="Sophie Loeillet" />
<meta name="DC.Contributor" content="Emmanuel Barillot" />
<meta name="DC.Contributor" content="Alain Nicolas" />
<meta name="article:published_time" content="2014-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Sequence Profiling of the Saccharomyces cerevisiae Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Advances in high-throughput sequencing (HTS) technologies have accelerated our knowledge of genomes in hundreds of organisms, but the presence of repetitions found in every genome raises challenges to unambiguously map short reads. In particular, short polymorphic reads that are multialigned hinder our capacity to detect mutations. Here, we present two complementary bioinformatics strategies to perform more robust analyses of genome content and sequencing data, validated by use of the &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; fully sequenced genome. First, we created an annotated HTS profile for the reference genome, based on the production of virtual HTS reads. Using variable read lengths and different numbers of mismatches, we found that 35 nt-reads, with a maximum of 6 mismatches, targets 89.5% of the genome to unique (U) regions. Longer reads consisting of 50−100 nt provided little additional benefits on the U regions extent. Second, to analyze the remaining multialigned (M) regions, we identified the intragenomic single-nucleotide variants and thus defined the unique (M&lt;sub&gt;U&lt;/sub&gt;) and multialigned (M&lt;sub&gt;M&lt;/sub&gt;) subregions, as exemplified for the polymorphic copies of the six flocculation genes and the 50 Ty retrotransposons. As a resource, the coordinates of the U and M regions of the yeast genome have been added to the &lt;i&gt;Saccharomyces&lt;/i&gt; Genome Database (www.yeastgenome.org). The benefit of this advanced method of genome annotation was confirmed by our ability to identify acquired single nucleotide polymorphisms in the U and M regions of an experimentally sequenced variant wild-type yeast strain.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/04/01" />
<meta name="citation_mjid" content="ggg;4/4/707" />
<meta name="citation_id" content="4/4/707" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/4/707" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/4/707.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/4/707.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/4/707.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.009464" />
<meta name="citation_pmid" content="24558267" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="707" />
<meta name="citation_lastpage" content="715" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Claire Jubin" />
<meta name="citation_author_institution" content="Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05" />
<meta name="citation_author" content="Alexandre Serero" />
<meta name="citation_author_institution" content="Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05" />
<meta name="citation_author" content="Sophie Loeillet" />
<meta name="citation_author_institution" content="Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05" />
<meta name="citation_author" content="Emmanuel Barillot" />
<meta name="citation_author_institution" content="Institut Curie Centre de Recherche, INSERM U900, 75248 Paris, France. Ecole des Mines Paris Tech, 77300 Fontainebleau, France" />
<meta name="citation_author" content="Alain Nicolas" />
<meta name="citation_author_institution" content="Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05" />
<meta name="citation_author_email" content="alain.nicolas@curie.fr" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. B. Brachmann;citation_author=A. Davies;citation_author=G. J. Cost;citation_author=E. Caputo;citation_author=J. Li;citation_title=Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.;citation_pages=115-132;citation_volume=14;citation_year=1998;citation_issue=2;citation_pmid=9483801;citation_doi=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Campagna;citation_author=A. Albiero;citation_author=A. Bilardi;citation_author=E. Caniato;citation_author=C. Forcato;citation_title=PASS: a program to align short sequences.;citation_volume=25;citation_year=2009;citation_pmid=19218350;citation_doi=10.1093/bioinformatics/btp087" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=L. Clarke;citation_author=X. Zheng-Bradley;citation_author=R. Smith;citation_author=E. Kulesha;citation_author=C. Xiao;citation_title=The 1000 Genomes Project: data management and community access.;citation_pages=459-462;citation_volume=9;citation_year=2012;citation_issue=5;citation_pmid=22543379;citation_doi=10.1038/nmeth.1974" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. Homer;citation_author=B. Merriman;citation_author=S. F. Nelson;citation_title=BFAST: an alignment tool for large scale genome resequencing.;citation_pages=e7767-e7767;citation_volume=4;citation_year=2009;citation_issue=11;citation_pmid=19907642;citation_doi=10.1371/journal.pone.0007767" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=B. W. Birren;citation_author=E. S. Lander;citation_title=Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.;citation_pages=617-624;citation_volume=428;citation_year=2004;citation_issue=6983;citation_pmid=15004568;citation_doi=10.1038/nature02424" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Koehler;citation_author=H. Issac;citation_author=N. Cloonan;citation_author=S. M. Grimmond;citation_title=The uniqueome: a mappability resource for short-tag sequencing.;citation_volume=27;citation_year=2011;citation_pmid=21075741;citation_doi=10.1093/bioinformatics/btq640" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Lee;citation_author=M. C. Schatz;citation_title=Genomic dark matter: the reliability of short read mapping illustrated by the genome mappability score.;citation_volume=28;citation_year=2012;citation_pmid=22668792;citation_doi=10.1093/bioinformatics/bts330" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. J. Lemoine;citation_author=N. P. Degtyareva;citation_author=K. Lobachev;citation_author=T. D. Petes;citation_title=Chromosomal translocations in yeast induced by low levels of DNA polymerase a model for chromosome fragile sites.;citation_pages=587-598;citation_volume=120;citation_year=2005;citation_issue=5;citation_pmid=15766523;citation_doi=10.1016/j.cell.2004.12.039" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Li;citation_author=J. Ruan;citation_author=R. Durbin;citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores.;citation_volume=18;citation_year=2008;citation_pmid=18714091;citation_doi=10.1101/gr.078212.108" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Li;citation_author=Y. Li;citation_author=K. Kristiansen;citation_author=J. Wang;citation_title=SOAP: short oligonucleotide alignment program.;citation_volume=24;citation_year=2008;citation_pmid=18227114;citation_doi=10.1093/bioinformatics/btn025" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. W. Mewes;citation_author=K. Albermann;citation_author=M. Bahr;citation_author=D. Frishman;citation_author=A. Gleissner;citation_title=Overview of the yeast genome.;citation_pages=7-65;citation_volume=387;citation_year=1997;citation_issue=6632 Suppl;citation_pmid=9169865" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Serero;citation_author=C. Jubin;citation_author=S. Loeillet;citation_author=P. Legoix-Ne;citation_author=A. G. Nicolas;citation_title=Mutational landscape of yeast mutator strains.;citation_volume=111;citation_year=2014;citation_pmid=24449905;citation_doi=10.1073/pnas.1314423111" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=D. Sharon;citation_author=H. Tilgner;citation_author=F. Grubert;citation_author=M. Snyder;citation_title=A single-molecule long-read survey of the human transcriptome.;citation_pages=1009-1014;citation_volume=31;citation_year=2013;citation_pmid=24108091;citation_doi=10.1038/nbt.2705" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. J. Treangen;citation_author=S. L. Salzberg;citation_title=Repetitive DNA and next-generation sequencing: computational challenges and solutions.;citation_pages=36-46;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22124482;citation_doi=10.1038/nrg3117" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. H. Wolfe;citation_author=D. C. Shields;citation_title=Molecular evidence for an ancient duplication of the entire yeast genome.;citation_pages=708-713;citation_volume=387;citation_year=1997;citation_issue=6634;citation_pmid=9192896;citation_doi=10.1038/42711" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Sequence Profiling of the Saccharomyces cerevisiae Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="twitter:description" content="Advances in high-throughput sequencing (HTS) technologies have accelerated our knowledge of genomes in hundreds of organisms, but the presence of repetitions found in every genome raises challenges to unambiguously map short reads. In particular, short polymorphic reads that are multialigned hinder our capacity to detect mutations. Here, we present two complementary bioinformatics strategies to perform more robust analyses of genome content and sequencing data, validated by use of the Saccharomyces cerevisiae fully sequenced genome. First, we created an annotated HTS profile for the reference genome, based on the production of virtual HTS reads. Using variable read lengths and different numbers of mismatches, we found that 35 nt-reads, with a maximum of 6 mismatches, targets 89.5% of the genome to unique (U) regions. Longer reads consisting of 50−100 nt provided little additional benefits on the U regions extent. Second, to analyze the remaining multialigned (M) regions, we identified the intragenomic single-nucleotide variants and thus defined the unique (MU) and multialigned (MM) subregions, as exemplified for the polymorphic copies of the six flocculation genes and the 50 Ty retrotransposons. As a resource, the coordinates of the U and M regions of the yeast genome have been added to the Saccharomyces Genome Database ([www.yeastgenome.org][1]). The benefit of this advanced method of genome annotation was confirmed by our ability to identify acquired single nucleotide polymorphisms in the U and M regions of an experimentally sequenced variant wild-type yeast strain.

 [1]: http://www.yeastgenome.org" />
<meta name="og-title" property="og:title" content="Sequence Profiling of the Saccharomyces cerevisiae Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/4/707" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Advances in high-throughput sequencing (HTS) technologies have accelerated our knowledge of genomes in hundreds of organisms, but the presence of repetitions found in every genome raises challenges to unambiguously map short reads. In particular, short polymorphic reads that are multialigned hinder our capacity to detect mutations. Here, we present two complementary bioinformatics strategies to perform more robust analyses of genome content and sequencing data, validated by use of the Saccharomyces cerevisiae fully sequenced genome. First, we created an annotated HTS profile for the reference genome, based on the production of virtual HTS reads. Using variable read lengths and different numbers of mismatches, we found that 35 nt-reads, with a maximum of 6 mismatches, targets 89.5% of the genome to unique (U) regions. Longer reads consisting of 50−100 nt provided little additional benefits on the U regions extent. Second, to analyze the remaining multialigned (M) regions, we identified the intragenomic single-nucleotide variants and thus defined the unique (MU) and multialigned (MM) subregions, as exemplified for the polymorphic copies of the six flocculation genes and the 50 Ty retrotransposons. As a resource, the coordinates of the U and M regions of the yeast genome have been added to the Saccharomyces Genome Database ([www.yeastgenome.org][1]). The benefit of this advanced method of genome annotation was confirmed by our ability to identify acquired single nucleotide polymorphisms in the U and M regions of an experimentally sequenced variant wild-type yeast strain.

 [1]: http://www.yeastgenome.org" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
    <title>Sequence Profiling of the Saccharomyces cerevisiae Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Ov3qKqURmyFufnnky5FRhyy4-vtB4E7xiaJL5LfK-40","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459821","apath":"\/ggg\/4\/4\/707.atom","pisa":"ggg;4\/4\/707","ac":{"\/ggg\/4\/4\/707.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/4\/707.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/4\/707"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/4\/707"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/4\/707":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VVeiE5B+otP1q7j0wzTF5Mv\\\\\\\/qsC3eVmlUWx2gabFZvoOvALbnpJG7eImImCwO2dbe7Agn7CL7BNW4JTboQbJQahnOFcZp+XnP13g07GpTwHhSUhexoMY\\\\\\\/uGdHXHZhxATnwd7SxlckUxGkN9w+Ghy4ab6eKel7vfbdiwJCljvbNuvAYWtGSBw53QNPQ7c3iW\\\\\\\/woNoaTeGI7czF3fLtst+6cWH4nwZU1BmxELBfDJJjVMuCZbA302ik0h\\\\\\\/37ZRBZtEt6ZtlldH+uBPHRiuE+fbIIDje\\\\\\\/c0xjP2FezyqcXgNOGq7YyiU9iUolkfos+vre5VCLgXvGCJZM1BcePZrvOk5zUkwBMrhaqRECPk2Mvsg9+dlpIREH4IngezXxK7JDWSa52JaPfbll9DkNo6wRkAYazs\\\\\\\/WmdcZ1SEjRUZv53R4u6MfguMbKD0YUZUIBSKMBVCX+ATqD99aVoO4o2V+gepCbUDv5zQqoWDi\\\\\\\/MGbCSZMFbP5Ya69pt2bUsZbKI5iTLHXS6hX0VJLduuZe6eSpN3DmASVFWcQfIGSvmX461p3XiRBnCww89GXvj8RCPvOwWWu5H95csPvNLL9VaNg3lbq1LWCvUjMrrlogRjbS49uml9Q+riajuS9oAUVDuqXyZMSw+OlWNPYmnzLblag+Unnvb2\\\\\\\/qQc\\\\\\\/xWx2qDiDEWp7Y2zKFZXcz1d9ymXk2Jv+6kgTj+lvMcQ9nf0IuAdP7iARhDpvgZWnCZ7I6tHWclo3LjAq7IxX4r\\\\\\\/PSVHdVf5Vl7snT1zrDQ1tIczbPdn5s\\\\\\\/w3u5XdX5u02M9HGiwMomeH6\\\\\\\/5kSxk7bgAK1yWyuOuw\\\\\\\/e5\\\\\\\/ZRNPmJejYo21dB06iKHwugFPfsPqmvJrpPF7V\\\\\\\/AqwcuRTCiwT3G5DpbVwlfbFZC+zgZxU4A\\\\\\\/zkdfXXZXJayZSex0DYXn40UzsBQOHFRKIgogsLT60EUiUYhrcssgW69t6w0L8x1UeDQv1ixT7ynTIIX6a2q+VBpUDLumP0I6Ge43rJTEP+bMkHLo9z\\\\\\\/1ecdD7Rlc5Hv7oGX++p1PjXoUfPw+8WnIYb1Py\\\\\\\/Bnmo5MV8r2AUGrEFPzAfxGU3Md3xZAWXsGixwhtuDsvAEKTaA577duz4OtLm1i\\\\\\\/U1d7o74YhFhpqw4U6DhQxDTCNw1V37zNKeMUzxih+2WL0xb64Z57Kt5xmeN6dZvLDcW8l4bQjfo6KMEo7BKR0jW1UCb0fmtIlkfv30En+c3mAUbaCAnBOpZAiUx3wwCYIo2qSmhmu7etlZ8raXB+fAHVGyJnk\\\u0022,\\\u0022iv\\\u0022:\\\u00226J9KzZdMJphBGmmNVkvywQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223c2389f9262bfa387ef16eae540b743dac5316e0af2219f81a9be7b1950018d2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZuVuur3fv26FGsvoCbQq\\\\\\\/sB9YmwSaB1eznFIiGeDZUn4nw0qh8vZWfLRNeDD59S\\\\\\\/dicMH2qt22N+ph56xUYOVFv9EhFvyskL9YBmlhj6IWZrn9eS4\\\\\\\/7zIHk2fgM27JK5mSlDe5yx30sslWhy3M045w1rJF2Om0y3CfeQcPnlfQPLgp9fWPFKxBloiIlq6611QZaTc6+q6cr\\\\\\\/Op8ssdvfd6l62TD0fM+pByG2lRcAuWB2UhRRWbP7KdygDLaLCBkDQIpXGlysmyFfV7EJpFC9C9v96KciFdxEvQs2UhFzDoPAcBU8VtXVDmq1T282nrUqWhzjS5fdPp0uKglPD2v+PF8ty3\\\\\\\/Dq\\\\\\\/2LPL5u2+Mk9T9ICFo4XbMpC3ZcOd2+EKhua5qgS\\\\\\\/ID7ENkzEZYpIHbsLbg6kRVc4l8\\\\\\\/xaPSmkK8ouc2ADpgBnQrZJWcEMyS0SCBb3e93yucWycE4FVF0Mqh0kcidy8+aN3EGei2LalIq\\\\\\\/r5q6eZS0KregUJF2GIuRcjqg\\\\\\\/hbG+GlIHx2EpHbNkNt3uu5hBlJZ66bZVHwV\\\\\\\/8GiDJ56nOhAsQ+RK1IHtkCr4L8D2lkAR67vITtPXxQT\\\\\\\/fFnf4TxbbVvTFFzUzEYWa6b+dugjsFRQIa\\\\\\\/ZrIjWUmzlUCWPgk0LFUWSq06t546k6LjeIRblLtWysfS6mq7P53HrVQxN7tjNUZc7nWuuAQgVMhD44h6n40pGIZ237j+MO7sKsqLKMgibrEm+4Wgev1awhnPjdpYkZPPZxjjoPqm3r8gtoWDWoU61qGvDR35VSTh079fy4WIumrLdL9HFmikOH5mAqQTgx6WSPLM0E2qHwo5OKYXoeiyTVQqP1Xf1VST58TkCEQkU2a65iuZ9SVKLpDGOBcniRGT+PiaXJAmXrSf5twH5drPsEhQ4OiEG\\\\\\\/Nr+An7w5mbgo7bExqGQNRCQlvfkWTZYdMlI5eERqI\\\\\\\/\\\\\\\/lSbBA5psj+amOD42qp+IHJRUJaLWASf18bzvkawU2c6+edBF3VRh4INnxdI22i5dxrVIafQZrNFW1u6a4zR5grsRVb0OTHApbTnHjxS8wUG+uucCyNjOpZavs1B5hVg4Zsg1yTfceUXzNTRJaWtkmIfVEyK0jkm5fiKP7ixQy0wY3pE+En6n\\\\\\\/dlnGySXbDm2Xky4GyG\\\\\\\/5U14SoCi1\\\\\\\/2aBO47OmPmwqrci4sWCrjBFjNyk350shA3lMjFOjFQiiIjsqvYi7NBBXkhdbYlCCk1Zy4WuKlSz7h217Obw0VVaj1Aaj4OZUfifh\\\\\\\/3xxDRp4LDP3jAx2Z9R\\\\\\\/TIz86e9+pSxajqqE4qkT3MyIT7P0BRj82UpZgVh51QGXx+uDEnC0\\\\\\\/54X\\\\\\\/YdudrM9pTibN5U+qlE1W0iX077qVRj0od0cDkDKJK1Lk58Akbn4StHztSpf56Or2Rqcdz\\\\\\\/n6KxOOSFp1lwJuL\\\\\\\/\\\\\\\/CpuUBUYf89B4TZ8bTloxl7u7VQm7dm6yaOsMuZR21RZykANcdCSGMR6u7dk2zoCtKW5ppXm21ZSgMbgMdKCdzhpb2cZx6CI3BN9RU4l2fzpIEjyZTa8CJ06UAtKuKM8msInOxBEQFaj25D+CW8eTwt7fy\\\\\\\/lZCzRHNW5sxtMrplMCbMGENHjkicILTbU\\\\\\\/jlQYeqlcaqX4vhrY63LUj4vWrE2NfGmEnxriNupY9wP7j5o\\\\\\\/EsNeEuT8D3s9bNqGseMLTLcfcnT9n4+SNbI6y+DsyVfrtO\\\\\\\/QbS8oEe\\\\\\\/6m\\\\\\\/5KM\\\\\\\/Gq2gA0Z0M+PYF99p\\\\\\\/H8lKnLENGsUGWnOZ2NFUxeMAKI5PS2JTVwNhRLx8i3TDGwTWsbW3xf6I4cracnvptNY+1M3cYYYyOP9BpGh5kBCHRYEIa9bUzB00toVlcD35gxvJMH44AtlWw0rCCJUxuP0wTyR9SjcoyfeSZ7YWwIZ8TWHPIDH7j4LewQ\\\\\\\/S3sEyT1SAnFeDnw4TD4LSA3cMOmTDGU3GbtEarPI1b78wUzQ1YnngvRIrKsmjaiBP2S9lb4NgN+0JZEfy5TqbAkHeY8Ox1hN6yRCIdJlFqo9VioBPQXko7Fyght8\\\\\\\/8aRevAqQ8LIJJBL379xILaxdI6DrHo+YYdhhI9nSMffRFjTJ1bl7NyYybNEBBB4hAUcggBYovKeS\\\\\\\/T0zipocVawkiJv\\\\\\\/UwIdq\\\\\\\/bOKc0apvls7bjUgVI4SsJix1vWY31TiDRFDTkVDgl3m6CXngWrF\\\\\\\/we5HFi+NzF9y4\\\\\\\/9R1sCd6mxoOdEPD05gQ2X84O26bry1dLXpCvG+0cM+1BC49DoXNTi4vKAFAYaVe4SFqeP4k\\\\\\\/lvfMNtTg7zt3zblooueKevaV60waDlGnn20f9\\\\\\\/shg0seA68aXebE1TVvGY39Cucp+MqQ9Ep7XaxBj78xpNrmd8KebF39tqB0ndG3tDXWEefkBfYjexE5rmXQ\\\\\\\/7XvVi9moexLwVRquzexp7BrLbLgnuG97GTcbVCHsKAoYGQmx\\\\\\\/Lj9UpMyln3QuSK0DTck9rY6VoHM\\\\\\\/J59QYlpBKDyngG7dJbBhgJJGh5NCGSZ2VyL43cu4c\\\\\\\/K3huDdb7P7EF+\\\\\\\/getglKHA+xjF6+MvPtFTERWlURxGOUzSJfR4eIPELds1cI+gHoiwM4KtRPAX7PtjDYPON9+Letn1SJV9ourx0fKLVoICKVVIJ0NPvajIMyq8M8ItsNw0+fMLbGbXtV5ITk732sF5IKa6dTeAgX1comEGLhsQkFISeRgECzxW1dGXC+i0CLKqtuvKCzsejL6fmSGbVvDgU183K46i0EqOHp3wjgYS5g4PFB0l\\\\\\\/x3M8ctxhFcvJcak\\\\\\\/Jnt2KdLilJItSQ32oJxjPy++ixFbZCQZwULCDM7IGfNaDNsakksNUD32BNFiA07SvDcf48N5Z\\\\\\\/D\\\\\\\/cDZZUNDe7qMbPejRPn+VSjhCeUnaWuz53OhP\\\\\\\/jX90yQRa6SCQIC1IPLooAAq4Oi0tJvyBUexKgR0F6eqMhPDAL2ZOY3d33dE+HoOuhL+ycInGEJ+Irr66YAB6HDcGl8Uqm3IH0wen1b3msCAqsDyFZtxU7LxRu39y5N\\\\\\\/nfC0sSRoIV\\\\\\\/uv\\\\\\\/VSA79GjJUQU5ahayERCJIevAJldnGExTKP0kcWPT0uVHlF1qfhoLYrw2rzZftLRK4jyFPWZUuzezDwmfljHq26OJMchzHlmxON+JhPumhVKM6UbMEWGwMNT8qydnDg3BxF+IIEeqIJm433cDWJja\\\\\\\/PR5NEJ+qfkOjhxoPMnXOd4l2eevsE6Q6ghWStz7zNMNKbUJLzcVrsRUqx+21OsBSnPSnBM+2acqfUuCG+Ckn2ZR0JkrU\\\\\\\/xFpF9aNq0s\\\\\\\/juzraN8FQpDBObj2J7ZXwqIB61zeh9cu3H0QqF5IXfhx4FFMg90wBR8x5MF3anCj2Ajo8BP52aRltk8S2eeFXaGRnOFi0WxNlsrJnkHLibSwqWvJkgIBiEicoppnp3Z06ECdLns7AhoCzcahHW4dRVpVZAQGbX3gkNb8SYModuXu7aOdHKdzouDKAdeotiyXa5LGS9KCAhjK2RPBbXozeaLo0UahAk55nkGJMC0HuDe6DWrwsE+ZK9F2Q1W2psKKTHqlGJQhP7\\\\\\\/rwb4s5M8H\\\\\\\/f2KDA6UUz8lFzCfXWg996wKcT4WzytnSU\\\\\\\/oseeu\\\\\\\/CuiJ1Foa9K0ro9gKpRkys\\\\\\\/EVvpWrhDqSucDBjDOjgK+Csoq0LLDKNklUfqc+MR2MF2VrToXDG1Zh06t4P6rRE0KMscN8YQVWa3H41\\\\\\\/lGsXVaOwfwjTrOQwHwCkk6yQmtzQTqYSRjj7aLyxIe6S+SByQaN1+mZrzPLHqs9b+Gn0lYzwCOERJQATQof\\\\\\\/PPMzptC30ACXIIdcNAS6yBZTRrwVhHEZ3flTlSMyDSq1bHN6fiIN7haOyiONf6yrxmN0xks1cl+K4iaVQ\\\\\\\/Hqxl\\\\\\\/yrBv6jAk0zRl5fqkUIPt2HkuFPyXl6Udj+LYjRznAPGlY48G0bUEfMiS04Oz13zEGVG9xKjz6Ke3l\\\\\\\/pke6VpUxyaeTnpcWnTgoKX05AumGI9RPL4f3WyFMm8i7Drx5kmzYjKpShm\\\\\\\/r0RraQLUr5\\\\\\\/SplKmZEHGVoB49PZu0jiKXvw1rG+kN+eHZLjJtUvJa6Fe0zBBh+vR9uxMDHxUEfl2bI3uMy7DO6Zf46VLBxHJ+EB2Bwtl3RJz1WDesW8AjyG5JHuTOp4oRw3AUCcFQronnWdrnq2PFeCNLUsqMKD+E2bH30GQj6p5z\\\\\\\/u328bh9zEeadO2cs6OJm1B0lKE2XfNzkyn3AI46YD6NTlN+OQRzTVASQFB2sj4J+andADUilGSeTEmbyu8qcnsFlfVqyIXDr9x1fuRZIuKLVKlpN3RnYEaHmDLSJCSM5erUFfj1qsXy0qkU7tSua3nz\\\\\\\/lGutQyuVNOPUzfpYw1GNEzzz2KXahrU0GahwzL+m83WEZZxylUiZ1O8wFq3McrnLgymHNkSxmTn6KhC44a8gm9T6CTjDtPTxp5eCLpmz\\\\\\\/\\\\\\\/rEwv\\\\\\\/qLkXRa23EgIhJIvhf9pECUQ8TiOLC1ew\\\\\\\/nfpVYXLxnSn4EY4YhS0+j\\\\\\\/JG6h51HyMg0LuULtTyGsLBCxlTJDbsZhAD2Gy9cmQ1a2BPcKLhID9eIP0HPxc7qCE9wzQ4QXUSd9nwWu5Tjss+0L0dDV5jmryVHhiW8AmmAbkzrsIR\\\\\\\/BrljIuTNTYSKQH0DsAP3ZFseCr1FdhJHW6RPw3KhjvuBpAlB9erffztAfC1tvn6V6PYaQ43ju07rTNbmyAPY51VI8pnqmXXOyP0OIrFftFVckq1y7IThNezIYwZV9Ru\\\\\\\/c5yo\\\\\\\/dO7c7XOgLrFVRryt0PIewSGMozQWmaONJ7\\\\\\\/\\\\\\\/TyALObOtpDYJrO+\\\\\\\/ZBOnIGuF+EjwsCRxF\\\\\\\/QuxgCViXMDoHwAKQtbLGqmaJcXx1I55Y+BUwBdPwv7b0orb64NGF9H42138MQ93vUzc5iETSwDrmMJ3VGjmMCVshbrW5PKkDhpFguPhRcKbnLx7sco1JzpFjHIIrRqAO\\\\\\\/i\\\\\\\/7Za9XHAzu8EPGNa90TsyUYawnqhiSqmRNGzL2fZ4WRyZK3N7b7AXr2AO9p2+Bs\\\\\\\/R2Q+avJ\\\\\\\/Rbq\\\\\\\/vhEAJ0fY13Xb5aOrxlS6B9fS1l8HX\\\\\\\/oFmy8G9Xy0l\\\\\\\/jzW3Xk5IWARwKp2DsTQ8HiVrbt05ZBMfkQV\\\\\\\/FcRSzOd0urqAFb2QYC9TvhbnhqYOc3rlCOUTT\\\\\\\/TR6yoWk477h5L7sojHpJtJp3fM4klaBDxW72LtnnesrI5jbNIT1neum2pXjrpTViSO90plssVUkpdg3oYvQbpHWtTLMe5zBi06jJZgaoxoNzh3typC4L4aQHfTuZ8pR5yzrP6aX+nErhhyQljnXWlglM4ZIsXh6RICPhiTYIE7wH3szExjYiUNQ7hXZKDn1TNdnD3gEnnxOWMUVkCI1JAFXrM8duAuGxlaxeIXAYDwvWm45wwF8n423kA31FSRAWXfwahC8ausa2Cv6LGxA00orRtZ\\\\\\\/k5Fi5rvXhJOsp75pUzpKLv4DPFuoR5\\\\\\\/fmbKT+VGmJ3A4VCcuD+mBnwMaAsjjDlDdFZ3m1061laI+sipLshcPAlqm2AnlluiJhwkpLK1Ht50b6Q+Sr3vwRKr5Z1Cc3\\\\\\\/P\\\\\\\/+ka6yIO2ccJ3d1I0IIYHdeKS2F5\\\\\\\/K32EcngA\\\\\\\/Z\\\\\\\/dED2bEVHg7o81HT4WOVNR3pUYaT9uRL0\\\\\\\/XPCFEvgwERpItt6FXUxoN0oR501oHV3ADD2YRbu\\\\\\\/in3z1z1raWvmFJl+1BXYGohEtwb3kbj2MxZpbYgt4jbjuZc11Avq1nmV3B+DortJAOcGr3POvmJ5QMjP2CAL081m3YQ+gKZMgUYWimuK5kl933yAROGbD2UrKrV5AFEK0zS3eJs9GgFbCXp3hKW41mMfgrPKqKrfDHzdAh7dfpffR1LSQy5MHbmkx3pGYBmmdFSEh27zo9rivle\\\\\\\/KRsPTaoC5SPVa6QgPLPO+DzHeNGyN52\\\\\\\/Bldk8h1QH+9RIkVPG9a0SuVnQFidf+Wh1XUIuY2wmDXnYAMtkW+cNvvKFnHNKsSFCT4iro0nUxPrBDEgL0WHiRprm8SuOeWfXBwnKFzY+NIhHaRn1J2TbxX5ZSJZXO8XficCIVw0xeC2+rGgovcEii0pRvlwBwrMbt409MlcVftcpUWKB0b6kzUe8R58BKbw6sKDu1WPVgJveGaj2e6Qvdw0YBN\\\\\\\/JMWm+q1VY37ZE4c1QtB7csWExcTX3y68I5J3Sb15LQ8Fhb7i\\\\\\\/zqKr\\\\\\\/UMT+BY77wWKV\\\\\\\/X4hHFDs74G55i\\\\\\\/HnfEokMYHAQfu8qPv+HmXcs1aeylsWnEWVIyj65bwCSvyMQFylIRthelRgXXth30PeAKm30yUFpv4o5uDLkwKDohaGDX6vGUoP1jMtSlRH\\\\\\\/FXKCKYx1\\\\\\\/jJnkJBHSTNWD16Z7Js9wJOOTcuVyRj0e1\\\\\\\/DTHpdlgeCR9o1b4mIkzsne+qyURRIxtU8LvZ+Me9V1NhYxNdN1n4ZioqZEbdZGYRN57vWUJXDsAMY+lxw86Jm2VLB4802o4TchNlRV1LUE4\\\\\\\/sGMnr3HhiqCZX1pbg10hrBKh\\\\\\\/JPdtzuI25JJ5UZj7U2jGruw3Sd2+9dMQtnfi6d5O9F4iRkdwIOT\\\\\\\/3OqRQHBJIIvGAa8b4aGpB6TA1x1LcMz1+VWC4wRWdTmpXq7WnEynhn2Qi2qeWvXN\\\\\\\/wtlzoTPsFJmXj95MyAuPbNyL9WT67xfi3OZzkxWWf0oS+2Cp4uzyblHlL2mQUrL+xQM2VaLP0GXXGvQWRiApyzJsZqVc80sMObejSqsHuWGWlyKjR8v+CLJ5Owte3gGdwxDSxQtWKz8\\\\\\\/DpxX01VWjT6xaL+\\\\\\\/TJYhVKyvoaitJAy98X2EpQf+YE+ucQjn\\\\\\\/yomZFvv0PusViTh5JBuye4v+xkfbSQkW2kvk4qsR5tM6yLO9IWTVe6AtcQeEuz0kX08FBDTN1uLEAH+0zMjnsJPJ\\\\\\\/8T+Zko411nC+0HtaChIeGuomfTvXETjrv0csg5T9TctsmfTPrL8piOQ4B6RktUlrR2XoajAWo4t4rPusEBOlvPbf0DXPkCbWeQVfCu5aLdCCmuq2Dt0CieaWirjCHzXMcp9pyrnRETxmJqzr5nHRmFDQ\\\\\\\/lQQxN5\\\\\\\/X0d85bnsVuA3MoAop2KW\\\\\\\/plVkFf9lU+wTZDYpk3BrBmqbnB2WuMyXJJy1ZyNozhNiSCWpQC9SzdGgy00oOl8xt1\\\\\\\/8xv8vWTa5xGsmlKJX0bFfYlxggD0nJFiDjdBPdFhN3cVGV1880Wse4aWeX8bc9AbAxeuM575+ZPdwp7ChHx7EAD9WL1D7a9fqG6Cz2Z\\\\\\\/pdAnWnTZ8LM2uh3LPqMwHAgGv2kMN2dTuA6YcHp7RiSxw2SmuDQTASEKip2CTbo6GWI8rTbws0iCQ5CUmb1v\\\\\\\/4auklUKo4A+FtvM+MF0kbADTfunqboof5wto0WG0qDniKGCTtRE8Cr90ZPhxJoY9YZyts\\\\\\\/L7NXOztuD7ZU2jUM0aN16bZ0SCTHcYgaa++qU6K6bBMdsYe9WQLx0eyIuKKdDanvvVwAnUjQk2GJCClKwZmVfImmE+qVCbljiTuSd8VflyqHKgnL+fvsiv9slAVeb0aFPfxSX8DTc+3TD0TZnJbH4Dfd5aHPwrYSs\\\\\\\/T\\\\\\\/8CqKL6Upy72Uqt\\\\\\\/gQQAnvRMnC8K8r\\\\\\\/ifnpcXzNHyZeL8PZ0MwHZaIvXba8EKwDacXSecACY7KSd2E+BKpvoCN4\\\\\\\/PV9iKSciVmXIGUFABjUO9GWNZsELUe2YWmSg4z9F3mLX1g6zpuZGtA4gFhxadTTQ7aeG1WjUJH\\\\\\\/OlQMGXgGTSTBJISEdVszZasTGA7ARgHwieJCvj7Lv3atH0Pij5QOiQ4sVwbTeobrPkh+N5053dVDI6aLx7SzhleY6Eo\\\\\\\/om3MLRrfQbwuNh1YK3kIuLIHG1GMYwZDKZrc0t9O8qAbJPfhTPW\\\\\\\/SYTfEa+gCuUvMvvRR9dMrsjKF8f9Mh6pVOau2VdURP+RVprhHqr4LHGbpVXoNZ9Fio\\\\\\\/n6XrjU0sRIAy3e3\\\\\\\/I5JR5FpFa2SDFCfOMReFZz2\\\\\\\/YTEuq7qYdyUHAU7I4uyr3gyoMkxeD2fZmhh5yIDzkCOOfaf3l\\\\\\\/tC50zYcyoadizczCKUT8IKrFx7DKjaY1GgDSzqds1X7te0bPUUT0MtJdIc7TsMcT5e9SNiRemMXg6jGE0PV9Bqkhjs8Jg3VCaMuzFGCOYuCj8eMOU1ZpK\\\\\\\/cZc90a9KZ\\\\\\\/kJOFC610ZVBcXAzwC6OxopZUAdsN5sfovwlG2GTnwiKc9FDG67usWgB\\\\\\\/cKaRnnNWaQYjQ1v587rIGlFc\\\\\\\/fnze\\\\\\\/rKG+lt4RlFpv1qMVEwxZW6kov28gejDwf+iVrcpzMDzePt\\\\\\\/GemkJz3NADOt3HmO5IpIm90Ggezs1nFjTOtkvX95u0+GbiasCPThWPzG9PRfS+6o0ubWT92wLR4Kz7t6\\\\\\\/nzt4wNy3gkidAZDGJFZra\\\\\\\/9YJWCSiJh4waX55waINntnfj+6VBJh9unH6PRdQhxwH8cyYoJjK2DNqI3Uv9FAcXfR\\\\\\\/k7sQ71NkH2FKS+Sy\\\\\\\/JK7FvFTx2aTiHVeVX8M2CSin8+2gto+uhkbcnvnesPO95ZVEXFd5epxXZ7fpWZLDHe\\\\\\\/pbiFVnPdHcAuK8zANZlBY0oo74JvSzgBCPQEvsLij5Zr53xx72mSj0B8vvJapKQTJbb8CRxTSzuqN6Wml9ZszgGMWjpb28Q6n4PDa2EgvRObqqFfNe2K802Gg4uKhyJeWvk6rVnv+QOZvO0RIQiG0YyCza3gg9\\\\\\\/vgf9ihJOHFFrKdimXi61FhlyeHanehVWf2RX8wx6zkhv4jR+znHoAzWKXRBxrwiKYRp3AEE5pv0Mwrlot\\\\\\\/i9NgTq+jlHagZpL0\\\\\\\/LW3oOyUyj\\\\\\\/gHqziRaEx4VJQLcqNktX\\\\\\\/axlnXOP1BoGYvjuoaMMRIB9iZGTgX8moet24TnYuKY+DiDieKrtCYenTaR4pooGbpetZaIW4GNCWQg6vbpDurUDRtrE8hT0O0pEEVGn9L1f3uh\\\\\\\/dn4raiPiK88xDqHArF6WhE0Pw87brlrCz6mbsAyBbXM7PSPA4OuQcrsAG5BZtueKhhj4yoaS\\\\\\\/KVRbY7XHBtQexv\\\\\\\/83nnue226KpkgzFSfc\\\\\\\/vk9JObJxFdRvEH0sM+e2ojMUWpcCQiQrNLk\\\\\\\/c8U7XmKjZkSNHedl+XXbd\\\\\\\/aE0v\\\\\\\/m\\\\\\\/7GIqjKZUkobGdTt5vja2Hz1l1S02XvFn9QEd5Duip4ko\\\\\\\/iNdtoKyw1csdyvTshUv8PrQYComQz2Zd0AdOfZ0FMU020VNXoRcVBUG+D\\\\\\\/kV1VN3Kx+DoVPwOk7SlLrHqHGYAppNFbeDW1QWFctBhbvhBkJ2mUxzyGcltAArZGYDqJKAgl+gwXOOK7sE99\\\\\\\/Zx8hB+6sQzVZAku7\\\\\\\/2HldmtWGUvkb8VP7yrMUZe6Vu7Wz3vONJwBninbNk2jrdXXhHxnl77iJzY9uCC0ccqqXy4DH3sHVswDYs0c8ygkzwraqHGfY928BLbuNYoUNwXnn1JxfVipcmUiSnAVO8SAAk1El5k2KKEvns4e4Sj8XEDqFS7NihxSTeqXRPb2rAie49khABhAE6jGhbnyeCU\\\\\\\/VZWWCAqSXDhUchYi\\\\\\\/H5vADpyLtMfWgDaOxvqZNXzHCFq3nLZ4sufG2GJ3Jg7Os7CKR09IVij4ILHWM\\\\\\\/Im5svEBc7pp59IFkpBgScAT7pEKYMEQ+CnK0VKfS0pNbO2qXQbq9AesBtnazq4rkHtQ\\\\\\\/yPTQf2MJQhr80ypxxll3ETwhbhQ4AD9S4idKPhYssZKRRxN5cQi+XNJettRR5AbHaBFDB3s0BNZl+jOofhnS7eoMRwqODl40uEIh7A5XgG4Vb0Jk6Mv1XvxIN2mozXMZnp6MUGU5nkTTbLyzUpTUqy937CSOE9llbvhGi2Uj9viY4xqOJUw0YRfJqc3ys6BulS0AGtsOWEGMQogbK9fRmnl\\\\\\\/RDnjtkOFhG0HEpXNIa6DTaa6H+LNqpH\\\\\\\/V18GcfBJBVNg3gY50b5p43l3Jig1q9vqhEeP\\\\\\\/ATMZwrJ\\\\\\\/7U29PydX4G74bsSdkbFn8tf0XE1n7SUE998cJhs0Bird+MRmSm7vxOQ9gEc0N8bAM5MvcgZBI4rDHOTAloZJ17M+AJiIADx8FeWslk2FfAeqgPwjZcpRgCaPr4GnsrcOfP55bX4BpTGrI0iwHOY9Fen9m8Gs8EBkTb\\\\\\\/0vU3yG+JulD0qthJLGYFWSdLXLSLLsI7RVTOSDRcC6+H0XWx1wIdjEyEgZgi8cHFbdsBUBDjbaTlMEBgUA4vaqJ2cUsCMBInf9kEjyWJRILOVbd0+li6iUOMBejYCkEzS5JqUT1avLQjXVaRQarlmqcPJQXIB9wbkAEOKIZsUF6KmKMXVm3bvf5NpPuXGSfFUOSiyrCQxQo6yzCb0zzIAUKjgcvQjwW5FNhjTFYenDJjb8Y\\\\\\\/8FgrOu+cymfXN9Fk4gESjl2bEwygpEjB5SWxigHdO0WveMfUpLPb8jfZp5ycik\\\\\\\/RWvAZnq+RT9BgZIkZNm3yKENhF\\\\\\\/XBxFrqr\\\\\\\/epakTQ5b0mvaQ68\\\\\\\/x1JPTY5XxZ3tXMD8fvAVj6Fv47jeq\\\\\\\/9UMxz+lfh7q9UuVCClW2Cot3o\\\\\\\/B\\\\\\\/NUPSCVLoph7t+t\\\\\\\/fAfsKwu2KmM7onA+LwT7fuQCQJrF3LffwmmubSZzqIFNCT+je0POTVXUfpm5rjRk1+ugcP1bWgUX4QsLCvyGHsqtTAVbaV+eD8G7VksYth1fhEaCpLk8k+BDYwUYmTumyuyFATL5zC3H\\\\\\\/8bYemgg74dR0Z+g6DlGg7u9l9oAo1yB3JuO6eYD3btMIjEjxRHs+19XVIdEnFZp1wxrzuu2Lh\\\\\\\/XxwxNh5w\\\\\\\/WSmjFfEDl7tvCywIanDyTZSVgepRMRlHpl+cMptCqpmGojdwTgDAEoIQbxfwU5mINi3Oz7r+HRWTmkfkkZ56\\\\\\\/AR4khwdw0KpBBKEjA1hPixHOb67xuOijaSqK+sGja37Ho+AvgexLpHQ18VsPbIVKbviWOI4w3d5FYsrxNPCDAY1\\\\\\\/T4rXKsy\\\\\\\/2pBCioLcGvX\\\\\\\/nSuAEXbk07yI9NgvPb1ELQXlO2tBbJP\\\\\\\/\\\\\\\/te1rtL\\\\\\\/WttfpKui3ZFyjQYm1FNnU9xIEbuCvwOVGzcykj2WlfNd09muh4eTQ\\\\\\\/rTpqlkEbSIxEn6tHJLBvaGBZqVveE3YM4u+vH3DCNBV4D7dfVIaSlUHDQQy08Efib63uyo5VZyD0TymM5Aldc9XAVzFZeqBB6ILUnSgBhyxeIGtPk2B0brCIFeydEqZibCEl3BB6bE9zQ4kxJpuFxXpajiIj0pkoM8H1pm0365t1LZ0405mbcM6Y13FkC\\\\\\\/rNDy\\\\\\\/5Ip9lC\\\\\\\/PqyypeuPHzSolyIyI5rPx0xZPETevoy79expRmakNztVE9nNwVasRt0eqozQR8rlpAJTFB7XMMkLVn7taOvHMhelC07qv+OpDOIfO6DDHXEIP6s3XrOmo+0LpdGdFQE08GH\\\\\\\/Eb1kukk4OqOKam7Qbx86aV6kSxbQGQszY4r61fXsexY0njAoYlkr6mugqMGLaenkyirQMnKmerwBgUUHDG78H7VmKJpSX8LBIgKT2u5\\\\\\\/pJ4o+l3LHTyurw\\\\\\\/98l7RK5Zy8nrKgqDASoZJSDMR8Gu2uhZVXP5DIwi62oQiQmYgTAOhQqQEegBYKy8ibpqI6w6uNEdYLNj1KjuHQNf8eHeSRb\\\\\\\/g1Meo0Gdm4qK7I4QuYQItlvDiGI7mDOjIkTlBpZJmukO8w0e+d1r3hQsflUsbvhGOv89YJLRBwGrrOAzqOzOSGXM4pbvXgFJFKKdL6jZ+sz7CuVXA7CGEN9GrLr61q09Cb9jHTCFbSZL5+7xNVTSdmsjcu81z3TuTt7AXf+ix+Nl6D\\\\\\\/WcCRypiUBvPvxzZLzHDRn2kGScd3JwPvJFF2U\\\\\\\/Fri0T\\\\\\\/G43bali4VQrF6w6bvjXzPoPaU7kUvnQteky69ll70wOJkka6HkRDE2MX8Z6Z\\\\\\\/LrLZQAYfqimE7f4qw\\\\\\\/AEhqMqxDs60sGm0NMTfL1U+EYhccLRTtsJuhG10OlKl5Eew\\\\\\\/1THIfsqAo4sw3ThJGBWeIVwkAGMp8c8VGqmCeCapnBTyHQ1TxtM4FUPM3c73IogMuY\\\\\\\/DMkwQeZjAWutRffc1Bh1+wPlQLPOjod1fzkkUDzSF49+suNuFxVd8A1ccW9xJX8IUjBkiox3fxYVaThV3lx+gJ90seGLqOjxuUU+JAp87h8V0YCvqD0E8d1pgDiB\\\\\\\/cwp0kwF6ZAXW+dPw9TqbXfMo2EfP9QoQhDrEd8P64jU5wgujkbnuP6+j2MW9yJjG2VApS6ClRCMLu40BeOcTHvq4QJhT9iwavnNhT6PGwNknAax2p1cTSg9mXO1eMM15KWnJag5stukzzKQ\\\\\\\/mBcWi5RKqS434STG9notG8svLi9P9rPeCfiAxLo68HT2peT8F4ATWJXiIFoMhLsC70jsFwxJusCgtDIF0C2DD+UdGr+HFslB\\\\\\\/7hRHAi4\\\\\\\/NLEvcpbM4Gi8opUaEb1pwxKf1oqwUx5DZSVHQqFP7yKMTz6lGAaXjz489372AVgawzddclTAiNlszrIZvm9UYTDblcn3HlKSiSwgGRDLPvM5b2DuIIgoDioYm1bcC3a8o0TJgOOcyf1SeLbCMYDTvm63df6Ovc4KSwkvjPDOT54WGW2nrfanGrzok+SjKXV0cHPWrKN0oMA\\\\\\\/Ftk7I8KtrWUTSSivGggWErZpWcvfbX+9HsSsiepSruJbEN7kiPCZ6EXTWBJAva718riyKCHs6izzqs85rB55NuDh5QrRyjHJYK8TnoR0O0sEWN6bvNpokX3a+j+yUlscBhW9OrOiea2SrP2ZkjHL5UO3IMUUZOzmKO79R2\\\\\\\/Aoci9GbAIIsctAQW\\\\\\\/2fVdzOT0nx5fgsPfJ1QEJCfbfiko3Gy3dO8eR8Py0rTaHAka8C\\\\\\\/pVPXGu0rkwQ\\\\\\\/sutMOaxyk3+ctOhFwBEBKWUDyBDh+4ZRgsseCAv4hw3uHChN4kFmU1We9rC031ZRzWtOgLjmCA+PfrPyMdqNlu99Nm9bntkGFaUqRZHFHXcGQ7kXHoaygGA8CFUJDREKin0wWHxGLGRg0EF1PhPpiP3sg7pq66N+osQr4bbN1Tq2qkG42bh6QiHidLU07PJYWS+ENkeB21GH5RWFyCrVTrZcBsiigEOIhWubGw1rmw2nWYOBghRpegOZQboJQgXTxsisRgoObUyhq6rrWYzT\\\\\\\/zHGwra6Bse1eV8FsLG0pTA5FVnWB2TVfmXuAWTHDFXCjV+m4bUttubA9BBlQJOOCWvyVnEFAfDq5tYWVbUE22L2JW7jwtDFlSJ1UDKRIqVy45nMVdF6EimcouHhF25ahQLG7L+uPMOZfuKyvxnsZf4bxnwpu2mCZ6nYPnSpk9FyyC5FRwpT6uyGgZ+6HrjBGEH+fZYZSqVSbpoGlLp2GrT35sQa2ejikztTnTXe87LJukagDMTbmLrn\\\\\\\/ySxT3a\\\\\\\/Vb3nD7lJ\\\\\\\/k\\\\\\\/nTRkXqWPC4G7yzW4Tmdp+GkfL53c7ZqGkukN4RbDZemrzI2BQGR5ILiohAFprConsEe+vgizzaPORd60QhAiL7Az89UxBlidV1ki+u7Ot60s7aBKQCrhWEKqf\\\\\\\/AzbrkQiZERrS9ye08SG8F2n5cVfzODUywHCZ5b8BbVzw6Dobaj5mq9omgA5W7wZYx0sw5YfSBmGJJ9T8\\\\\\\/b04yn3Y8KpKWtcudgSo6h3Nm0FzdTuHjwgYep+q527oqAV6W+2d4MMkPzQ3IXt0tMiyzgR9x0xuWnD66Lww+fyJVMcCz\\\\\\\/L\\\\\\\/TFiCpUvmHt5mNm1iKbww8OXu2qt96rXBgsW43F05MY66O8+60b1KlRcllK7URKZVpn7AkhLQB\\\\\\\/h0hDYGw1sTMkFCSHxg8Q6gJ9KIazxTgK22sgVS+oGGezxB7QRu9nq+87rwVWnGRFFnXnrCUK441\\\\\\\/UA3bWLfislBEBwQaY+H+kB7PN8Roj1IwsQOPbAm5y2aHvmhvAZS6qFYmFhqJZdgbFSztkK4dO2ix82W8\\\\\\\/qJEc0bmVQctHehTUxyuGGmfbv5BeYQCkzYWjxLWBGQxtnQSkIOut8ye+MVuLcyzDC5MmsNk9qSiL7dqJtnFcyGdwhRqJrsVM8L2++v6dJcEstA2XUoP68YQFZosUla+f\\\\\\\/JapLUyIR6gFUOAVlNCfJuCk2ijkLLg0Ckx\\\\\\\/Bawvb5qElzp1CsFHM09k5nt1uPZLObEf+7Coj7eEeC9eWU+w+IrRWBT+Owypp1B9Inv6xrs6oo0TO6cYEgb9rI+7ozS9kf9o4n8E6pEWpNw0n4F6W9WTWPvyu7Ud8DKNfjj1IhUKtLTLLlkqCeS1ifa2rFHfhzZSgPamEz8GwHJlMM0AT4KnTmKDmcaNfUlyocryo9bWkejchJERS9lIg+dSv\\\\\\\/KNOxaQ\\\\\\\/n0CAVgoJKB+xUVO9pIrHRtuwOftYz5BIVuzoQRIRZ\\\\\\\/gi9Qe+6opF0t\\\\\\\/9Aeg3+uqxKSyl4gwwYf0u63UGaGEZunvAKBCsUsCSgOKhjfEn9nCHj+T9wi+r1BvqRDXHUnuvTnxDmainTsP9TTaQ7vy+lleMBBgFdMCI5N6+PtSOYUoxEr133yzCbswpfMUmgqFdRzCy6of8PYgAC994w7VHDLBiU4lBrnBSHWgjt6fLeWfjT4LWt8MRRl\\\\\\\/JtuOWHtu5z6uYZXHvxYcrrwfv6ipl\\\\\\\/MKQ5A+vg6iAVOGxkvOW3uHUUeBw0noIgUrQqqGFrz5GgbIDeSXNN+jJoMlk1bZZvShHQ1QGGGm2BgFd9pS5bwHkMSJg87UM6IvITAuMJSW5E8N7a5+WN2CkWO3ugHsuKEIU\\\\\\\/cISsWbP9vymQjo7BTULL564hW4hJsbFC63HOZwaq+VU9vodsZtfePQNcvkF0ebbujg4oykJL7axTQH2Mp9vsrPwV\\\\\\\/\\\\\\\/54VHU6nez2rDN\\\\\\\/13VahebyBdeFbjexjJK6lJg7CCCMZ7D1vblt7VS1winieR8PHC1uJIAhVPCYa2NLOdatVAGnNrJEmCyRz6ft70AYJE0+6LIg2vuGJeO1oOUhOSDTek\\\\\\\/iUW3BGlNp2fMegEvgEXJOlsWumI40YHx5nHWsX1vBMZ9An6VbxbxKFo74iWrF8UAtrPLCFHWbLTQANfQ2oj2FD3+7ymZoukTPAumRq4EnhdySaHtApu9LQHDeRKZU6swPQBrlNdpCJvdwAZHh8pTK0EOOiOoIyCroPwvg1pp7f72oEZ\\\\\\\/QPc2htxMVdtybzpOW\\\\\\\/6FwVaKzVCGJmTEWV7xmF0iXcyV72B94r7\\\\\\\/SETxwPrKyGS+D525nYIH+o1M3du4npDjlit2LUyRL3820mCVziymO9CiWhqgN3wsy50jSe2XlsOH8FVoAwYtcHgkO6VsfUi2\\\\\\\/tHYquRZbFEZMcH4UWBgK\\\\\\\/uLq15k5Gb41id\\\\\\\/KFHRVZiWGzszrLIIVqH6zoEaRf5MucioQp4mPrEQ1B24MyHVv0vr3+O4FIqWuBMhs72rVOq\\\\\\\/ylFjRnSmndMAq8cyNOqY0KAdehYGFNCC62LxsMEyDDGz5NxIQ6N\\\\\\\/G\\\\\\\/Nkurj+99uWbQFxvO5Ylgkzt5dV24mj3yaxTIqyqR0dPV5PP9HRiPtkVh3iQ4UyIl6IpiUYG\\\\\\\/SmlXqstMYodEw8E7jkLcylaQZ+jghb9OnldhOnSN1f+KxnXK6S8dLbQVMOCS\\\\\\\/ioJm3niS5yotBrJL9ookVNriI6awFaiuSsf7IWjLEZPnZKfH+0wtkV+bj+N3kJDgNwFJY\\\\\\\/iUsfDmQ\\\\\\\/nB88qC+3eWmtbWB3XjU6Gs3q81O\\\\\\\/aQm4KgR+ZugK+cBqSO9dFvPgeUvBW4KziDBEZa44pX+3N3hhpz0H3SUhNcGQ\\\\\\\/hdkgwWMTahA3PshrpabQ9c\\\\\\\/cl8lPd101niwera8Bg2ugjcrFmK6fGsMaKdmOMR0J7KS+UWlV4DHzPja\\\\\\\/lGBa8w7m2vlbsynBDGbWw7ObFbFQMTU8SvxiUiOT9z0DolgVP4g8+yxTvOmj84ItrMdinIxnrRNsx\\\\\\\/CnHRab6CqXk\\\\\\\/pwvUGjAN\\\\\\\/zNTELDk9iWbzJSHj\\\\\\\/JTgtk4U4kW5bL3AUxVisXJbIp4h\\\\\\\/9gp54yXcM1S6MQL\\\\\\\/pBoN5rfK68+1zN0ax13XvynJIE\\\\\\\/Ieiot86r80nT9Agv7dx4Q9KncE\\\\\\\/h1R44EGYBMj8NNu\\\\\\\/jVQOvpFkmd6gPtCJD9tw1qqmqh3hGNF73fXt478E85mBHsqiMethqcZCK9aM0vMeEQ\\\\\\\/qz2ELpEv5lQjxUDVNSW7GJHA98gzeLKQSCmWSrtRH4WkIzty58fmBzkYDMbWOX1IKKR3jsgtcPZL67vKnZu0BnGG6oDGN\\\\\\\/4\\\\\\\/U3T9CAyYeCzOCi3ebvfQXxf3M89I3hzFxkLhEa10ZWSXVdgamVbnAUCylG3v3vjkgREInuHrsZHo\\\\\\\/0rmf52yOW+uYJmz6vUrgBTIyyGiNA3GZRhlzo0eZsIfFNgIqDmpdeGlYFpqal4ZNCWSZNMGHvdOy1Y6YqpnpGp8wntTe+55ZnaxhZe5m6F9hUJuYTC0FRe+qsVcjXYK3vj1ncHVubljMY+FjzHVuz1K7UIJGOfuojAtR+fsgDecy\\\\\\\/i8JhS0fdqVamd4H\\\\\\\/MT8zo8oOcrC08RszPjLbmeBLOgVtuDnn3YEPLF0wR4WBjwJKtb6RGyMwGs4Kg\\\\\\\/i0NC+lko74ojrikD2NdjHQ7AqiSWc+jCGZNcugW4Yso3n6aNPDMWytxi7tNWeaSU4GVjGVxfgZizV\\\\\\\/rtpsci9quhkktx1B6XNTK2Zz+TNoDKuE6AkSb8lcOzN3JXyHQxzPSaE7apiPcykQSnmkJphTJ8ff3LJ40Yt3\\\\\\\/0X23S9M1k2PKOK24TMDodls0PbjSiv6keVkwrNRfqFCivdQ84NMPeX8bB\\\\\\\/0CMjUGXk4zQ7+NTgmtCWQcjwek6Xqvl8hvid\\\\\\\/kO\\\\\\\/LQ3+OGp+2\\\\\\\/4KEj7HCVHiXHxlnuGJQShr4\\\\\\\/xsd9N6ArTbOXSk+VxMbx4bvUCDXGvCgmC5SyP9\\\\\\\/zq0vIvH\\\\\\\/82DSFBWoJHj1axcy77DNGfNEdhQRtm+O8C36P10QVFWXVytx5Q4DdfuvezKz11pT7jmQSzXSbluyyzekHA6H0GJamclyQsvSx5Ssa3bKP5UOYUkEyHb75VoKfPytUtKMjs\\\\\\\/AZ3o9kACgtBoMunR+yatjEpfyVbli1Do4Hs17FphIX01dxvXc\\\\\\\/bmpMWAtsd+HaEI\\\\\\\/btNwL5cgI5EynL+tfGwlc2sbVzvy+3oDwK1JaN8yrQS96VOIw4nAdM+LPoHwzhJcnwtdv8VLauJHeHOUYfyVV5qoXBIP5z+gSgXfPutfKK\\\\\\\/rP16IYMjOzJ\\\\\\\/GCGncOX1uGL4ijJlQvPTwkXf8+8p4toEHFhwfvgKaV3jpTrzeQ2i+gWjVsgAynS5MihQvBXhCHiP8wUi\\\\\\\/OljFcP77YV96MzycbYiyB+5Vcqlycy7NWM6zuRuJQfMQLp4TPBZVBSmEBNBs4Yyud2yEaNmMfu8rGQJajweFkC6zyxL\\\\\\\/lQpD+OFiAFSA2g61A1oxgOKJO8aZ6AQzhQ4ngcJLQs9VnMePmc8Ql5xIYmcn5iqBOTvog8dohEtek76LZND7XfE\\\\\\\/IPmtwfnQ8bmXEHMXtm4qQOPQqTBLITgonBv5mm+TE2s8nGVPCmgtpbTIibNMagb6u+tqF4VAw0yiqbYUmthRiOIxuDoZblJDRpqQndoF7pHtCTlpJZokKw\\\\\\\/Vr0+eRWrSdD27sqL7UG302jAm9gmvHzT\\\\\\\/SxSPGc3JUYzGVuVz1Ep5CgG83JZB22A\\\\\\\/p8FkU4Uuy7\\\\\\\/lCuZASPWVPPVAdKfKHV6vbCjsBjsoc0pBFFM4wTNlbUebhVHp\\\\\\\/5T4XBikBLBOJ6GXMkM0\\\\\\\/uyQgJBCJppR26+yx4M51xnYwzCCciWRlKV9iKAfp+BuP5XZtu+yQT4IIvYJM7Yae8cb2fPAaIEaMN2eimdf6zZHtCz\\\\\\\/8yitlPjVrOsACWIIj8QTtVKboyyy1nTAefvbAFVl3a\\\\\\\/oDSRZw6rdPxr6Jr4M7oNwipPMZjXYR3Y0sPll4c072PyKnesZxwpH0Bhn9yTWYjCDZ1kEvl8UGD\\\\\\\/IjOYZ5EcO8bN3jRB\\\\\\\/v+qigFxKR1EMyCnUfqenTmh\\\\\\\/+W7+0Xs4fIkxJsZpk6KsRKsBsSTDRElNn6haLcxAxf56FzDzVArGqtRST9y6DISFmnVVTEwrY5OcLPtGKfarXy+7rXORE8H2dZTQjVo6S\\\\\\\/x5r+JJh58SO6Z\\\\\\\/zUrShJlEi6LtK+8IkPPgsOe1Mi01miHFY1Us9SGMBu8y4JoldfGtgUvfTctFoVMiSas03OK2qtEwfvWa1X4u2Vbz55A\\\\\\\/yNp6+qR3oOiGYDQpSbRcyoHUaDXJRQa0QWQlk4+3K7xArEBXxuGyNKQfKC8xR31dB3+OROdq8gHkQAKN5J0mX7EVwb5iwH1sspgcXR\\\\\\\/zAIveRAHcmpYPITlm\\\\\\\/LOKERjmUqxF+TLkJNlCrKvboMf4+7Mnq87Ht4TfE54rNnvVBDjeiLMtowTECjzq5N4jeG8vC1MfKBGuv9TdVzEi8pG5A5GOhs1FPdd\\\\\\\/IQs7LtBN6ErNn6hl3JCFjb32Lw42bl3h5aD3RcpjFxU7OF92Or5nYZSM4SbsN0UMnmYtkqngmear3avjdmPhMwI1itFkv2o5BVyJLBnNsv3vjhQPmtQbN6WzuMPAExSgZ7QZ3KFPa4N9Bf1H3dJoEzBRNkK\\\\\\\/BdTrgmgSxpYc+DFzdfdYLuzDsNIag4r4RJxsG1rmtxuv\\\\\\\/5YSZSIE6kdC\\\\\\\/rM+0fSBzhiZMbO9AxyuGbNoLJuWkAHMBcUWcve42mwZ1JVS8yFV8NF\\\\\\\/GEXx7B\\\\\\\/XfuGsz4OgcdlpYm2yOHxSi98nD5quZQzKjoCPq2qGkKee4P5Ko\\\\\\\/tqpSn+7YttFcPnL2nP0jG9bwOBJnXbELV\\\\\\\/QbP9IkL88oecmyvcHNTU2Yo0M+blkEjhmCOE0Qn5Cyhp4QtUdiHKj29oRlnizxCn2I5SWQuLbGJ0x3HjIeKnM5\\\\\\\/aHQW2D7w5xPqrj6pPdj8X9qoNHKrH0MuNAojfCovVPG6zMmmOHjwmPrHDuxvxbFDBqja7aEmnWFAENOzWic0pH\\\\\\\/AMRvXFLBLEJRnB95DD+soem8MMTb1jShKOFzL6Y1r57fL1bL5x5v\\\\\\\/I4fIoKupOAOBZdKOoKdTP0W+dSDoZrJZ2YZRHvqoZWMmIpuRBmUvR0RR2zCcz7VJHlIYMrD04uf7zcCJf7gOr2On8ejN6hQKuFUE6dFzzhQnNiMey2RRV9gyrqvwMTXqxmnbYJYltf96FVgopRMkXGrFQhNO\\\\\\\/DZ4o56gaMClILmBobBG2vUwAOzGvIfuBUeLYVqrAbLydgAUO1qREBZRxQMiJcr3tL5Z09j0k2NHoyfYg2O5zBJUaTpayNAwoRfI2\\\\\\\/UT0R3AXRLj9eD26wGDQGCL5P4HxlKK+bmlULLl1faL2rW6faACmlXbLASGs9jIHPWdjDRH\\\\\\\/ISTxvQgTj3gz1rBqYa1LIgQ6m9cKARcPEakVfV5ZfZFCpaWPgFuu0R6UsfMqm9woi5SUu0++2xDR4urnbF9KXbS8mNBMTPE+hcPPwKui2RwwntKFHDArtci7KU375feMi+Imn0e097hAigg7jyP11zA88eigNFyDvzZdhzb9\\\\\\\/jiOaaBiNakYyGfRPPMT0U9M90XmyiwybP7I5C0rGslxP6wAnh7Vm1zHHBaSgLAdd8Z2eyd7QSM+cWyjGQ+GQvIpVzHnsILHGqdHy86YUKPQ0DmvCOrx+nmYDBxKpcZ3RzME3hr\\\\\\\/wKL6h5xYUwksKGbllYvURLqnCHq5jaOHazUnfA+ypb6Kwr1batXK9lxqkGU46xwaXgcry3nz0a8hE1yTK4lpu29+mzpo3gVSFsboAdrDHP3c790DH1ZVJUrHiXm2sWVur5Ad6JQcSY3hykL\\\\\\\/1mqV5jPnwbAnnWPkTK8EK+CQjVmf0Dcd6Hla082+AO1Fen7caK1WG1kn2DnBvreGhwi7V3ySnP7jOLKfb5ig3ah4lhGa2w+uGt6XRu1l5N+8SleKkyXa0YxUEuGhYNK39S\\\\\\\/o4m8VPCrax8ntFYUQ+4Dr1kdRIBt5+ekZxjTCwCgO6p\\\\\\\/Zl6LP6EMW7l\\\\\\\/qYyw144VMWiVUehjmuuDA6v9QAM4KCd+gtivGuh3reupzQ1uHEd7ZQHkdOyoib63\\\\\\\/sbHBQNXSrHwy7t95XgS08VC1NgwrFhCmxYLoO9BGxV7J3\\\\\\\/uV67gtvJbB2jFcc5ujxtp5lWzIsgA3H62Xe1SQQyhQWiB44+wTAt4Fzp0PERCENC14hwDedtTmL1yVMbixdDTKQiSsjipd3KAjCShAImoW6ePRzeb7f48LIFD3TpWwvl6TESibu8ZsvMhagbfNmi\\\\\\\/MhJ+cI6om15l9lNDtLyOgG3VOXXc42Q6IrZ9KIE2HH3WaCyVPX2OjPBxt8U1fh1eN6b9FGftia5ZyDIa\\\\\\\/IPXV5QmMf\\\\\\\/zHprgKYMF5qBqc+13YlXb\\\\\\\/bnhlda+CsNo55eZ4webG3tijyJm6rAsdHEfarh3QzVKPuUO3ULcX6qltYZj07q0UF1JJIwbZC7nHLipkF+q7FlmK7FaOU\\\\\\\/gW813Ldurf81BRjHTCBJm54Clgd7bNUQmmOz5FB9BD3nYEbB5tU1BIvdcd8ka2vp3IWuE6UpcRmSo\\\\\\\/YEhKCPrlNiWyXnLW4pxR93cq2v143F7AkPbuVn+GKDp7MCe0Fp5pJRiu3SHm4Zd2wwypf3uZWszGL29Wr6I\\\\\\\/vpkh+8LJzomOiCGGE3UN7\\\\\\\/wLejpK8X6c2r9VSAtiVgeKo5VyutBELP8R1ylmTNjhr4oqeRHXglgEkcx4hNvkbIQSFD5J\\\\\\\/5UjVnNGrCeDSrWqxAGqGaz9g\\\\\\\/oKfzNa9Qw5i\\\\\\\/fBOevMkV5qwpRSAGE3Ib6Vj67eqr4uHcwUA0\\\\\\\/CwFMmVDSCKAUp0anaNYnRbrYp8imHyQaG895boRWB5HqS3ovQCMmTY770o3oT5Rp1gNLRoe3musyTb2xkPoKfnzzT8d1mG4wTc8+7z44VoAgt39C1Fr0BXQZ+oUQhT+8g8hrmj5rtcXfmGb\\\\\\\/TuAOkubcxO3BqTXQBOLkdNK+HvyZ8LWN0IJS3A95F212uXuEXPq2EFKcZvJN2+Xs6Z9\\\\\\\/mJPsNfT0R8aGHbssax2AEWghuiuYZV9MtlZlMz6LTGsDqxBnbjLzY9yHbEVl9oq3V2HKPxj1eTWmioz8\\\\\\\/\\\\\\\/lugtXKt8ebmYScgfMjyL+\\\\\\\/XCiUFOQM711X84G\\\\\\\/r\\\\\\\/GRGum7Cjr6fL5uIBw3DxpFtO9s5w4G2CF3wcJPzhSZcT3gOz1jyMhPNmCJ2ofYIpxTj75BA30VazjZrzeiKTWXul5tcxcKW81Ije8M+v3G0L4oxZINZr5wb4BlqiqfpXsP7jXNQ1Z\\\\\\\/Yg3yExZPhSEOn3Qsn0EEA7loExnBmKsK2q8l8GcmykhM8dRHh9IOTWCqqXrIByCizaqsYhfSzvM9QPq9NVn8rBnp1CPT3hWi9Rc8H5w2SRyV6OGyDd8QsRYWQHCkb+V9s\\\\\\\/SUoy\\\\\\\/aB8\\\\\\\/fZR+Ky+cyAAd5y79muhQxQkMDozU2TIb9kssRFM4pDakxrojR7YDlmjikOXtoulpsJYQUd+QvrbGMJZyxuVEq7u6\\\\\\\/CiM0C0mudyEt3rk0dn8H8oz7Ko0CNpG3LGPAB8qeG4eesqotjSOpHVzMPDxPIHr5zymHGQ5pYy8ibHtj798wb7oxOKsK0dkJQKLE7P3coj+ZcSwqPbj7PG9NRWkdWGYXaXzU3\\\\\\\/6SVOWGuPmHShN81RwDjW0a5C0brUUtNWmlsEgztcO9lNx82N1X9dZcCmBcfbLLibullWQWaDD9F5ZBzy7hrHjxmdKViv3VEcz50o87h2U3TyXxxfAPruZDFQSJhs1wf79sCL35t5OZ+uKrUT\\\\\\\/kno6ExlbfUoBxFVXlsQk4kNBVhyUrXrZczidxzWflez58TsbYtXLaqqxslNwkxOQexE0wMdeHbnKzmXUC6sgafZMc+zZOjuizkrbB93DHCx7oFzTENceZDvN\\\\\\\/VhS6p+\\\\\\\/nhPTXPRN0GQmi5dEZ3Vb1Hab9B9597IvN1NIrl7YD4PZm8ieoVn3+bujyyxm6S\\\\\\\/IoCGf8ylBDsTLP7X210QC4PPwXDU79FhvJL7aD8Kf+bJ47L5yuNl4OQA7I8+9avMdl0F1jakHrjqna3E6KuYmRy5w3aN\\\\\\\/Fcq6VhV7hmljITDGPNn294GLMYWCSOFTT5pGdBO0VeTfm6c2OYqH0aQ458aFGABSlqkmTGjOrf5hS6444fMmapQnJrDn\\\\\\\/cIHKaaQ41hS5dIwaMyqGe8wgCMHCqdXmVRyE9arravfCh2P6FffIk6r\\\\\\\/xjw\\\\\\\/Qfupkjxn8NUgLMSufbFyAtF+bMWZH4O0xpXIeZpVTawF5bLbRqLNux86\\\\\\\/jyofCjHwALP+izB5mvn0fvclqKK56VLAej0Kg6HELZ3JEVntQboa0q0FYMob6SNR61Md7b2lJVcoZywZLI9PFamIWhkz+Oq2ke7oHg3BqsfTz+L498OU47w4Fwq75WdO0aWIHF8mYV+Ne3LQao3pCt7864MEem1RM+XtVAEbP1uQCxjONLthSge+U\\\\\\\/OF6FnKa9PnrHKUoEemhdmSPJ7OfxBEw\\\\\\\/eTNKmufzNvcfZWA78ZAiOew2Q6nzQMfoqSCawZqs+C\\\\\\\/+lVHJuWKTUexoY2+JPQrBwfjvUDUZ9C5\\\\\\\/j6gZIzWawX+5fguyPRHGKlPHTHrplS7FdSvOxGZ1tw4KzXXXF8l763p7FDTeNECPIVA846HBI3pH3pD07SkwSSNyWeRtpAAfJRZFMJsi84njumgwgD4Tt8ypbV4motvKJiVmSRFq8MwLvXuuGLTPfdi4rae4Z\\\\\\\/dEvZD+GwifbudxmXE5v9DDM5suCpiv2bzCt2rRJHHWXmFRBV6Q9WjQX1iqFSIrRxfQxQ6ijtYCIEEHEqXbscnKGkowrJDjaVno2\\\\\\\/OSLt5w3YWmDteELrllJnm5xOcYhY+Haostu4Izgyjrkai2\\\\\\\/vdkNGwMVWORv6IsWcF6Pmwz6Ry3ZDs7Y9ab5vFtVBCMhj2UhecZnHNwsP8\\\\\\\/8dMfMte468Y3FD62SnctDLS3+oXatG5d\\\\\\\/kmuU6Q53RJ2fW2PUhduQAM77zJaDadSYEGNgn1WPysL5qmMy60Q6wLBONsDnPhMeT90+CRzi1qbwinya5encqG6ov1aYC4SUC6PCnfFltZ6einZJ70DqbRMPsMCCfj2eGZnlsTn05PrqWNDLm+I5RuGWJM\\\\\\\/CNxJt1d9xjYhJSFLSot1cNjqc\\\\\\\/kk+xNWMDgi\\\\\\\/Q6pUG8\\\\\\\/s8bi6PXFyq63jXbggIynkF+895cNHJNie7nTjDG\\\\\\\/7hUWMPbJ2LuAJ7Mb9EFSgEVrFgrTd1pJuBOt+mHDgme\\\\\\\/I9V\\\\\\\/ieSXwnOD8g59\\\\\\\/d1PljKzzByuSmc0A5Ip92jPX1Kd7SxHifsT95IYIw0WVyGZpC+aqYl\\\\\\\/q39YnGNEwv5+CUkT8qBAlmgplNjrGDYeue1Bbxz4EhEmndAQSWd1xaT9ooJWdLT9KnYrUx\\\\\\\/olX7HowNGOMN0q15tIalMLLYadDUqU6mKhlnls+aRG2gXKGkjW5fFpJIi9PS7yFzfC+DnLDRoBrwt1ORj5ufrUZ2hFmPvl+DW0yf+nR4dmWJPMVndFq65ySBNLzVad5Hgn\\\\\\\/8nwMC74sN8kyjTxZWxZfEf\\\\\\\/7PxfBXIsei38o9BmILcjHe+UApRynLSR\\\\\\\/FcScY9EY+aXNgAZ9ep\\\\\\\/uoMXiw\\\\\\\/lpEf0egci6uDd0v81HHWvfYrJGqCvl7nnDujmuL8iLpnU2hcUknxD681g0Ac+mm5pKc1mAaJMRYTg9Z9yR0l5JbsAYIsfql5WHNoaZAkSszokShJ6n3ZlE1dmZuAaIhqW3g9lspY9b9vetXG0HLdGPuOBxeLHqU8+610RupQQ1k15\\\\\\\/pODqLxlWxtGrBva8QXg31pt\\\\\\\/2sX4YQtIZ2rFOO\\\\\\\/TEpDRLi\\\\\\\/C3nEFiW3bQtjF8PMypt1QzmojxI9O+brTm6QQD7nDxN9HNQsPfLI9KzkOiKGjfUSXbFRj8ycAW\\\\\\\/KftVPNVRz3O6XjnPYqfAHb39WD7ydrvXxdzKfrNN5ULZ3uPc3TSFiJbyaIeTZPTTekata9imD2Mz8W6Jm4xipBzFHGyelFi31AE69gqkHRwT0AE0uTPb6+jyo0uE38\\\\\\\/xKP6aGiNcn5VdtOCTBxLIhtOMI3oUPrQEUHCJNYrsu7sO5DChDmjMR4vMOE7DB\\\\\\\/0tck\\\\\\\/wbGB+P2FNtrjxFQICo5RCYnu0hAjBXKlzmxzl5CgSCL6qHwXdhNC4DBns\\\\\\\/eN3OQQUpXr0fLlhXk399lvN8L7kK4KKQw7CTKZTNoPCKVxE0DxcAs4MATiDE8KVsNAP1jLKiCuNGhPZxtQic2kRKT05DVfy3B65KVyyN0fyZKQwE3qIzOP0SAaVlgeDv0zDAvNREEl6eE6AD0QeSHTpAyq1HkXXgMRAg6hFyz6Bqe8IOf6IcdgeTA3ARAN7QSHPXnnzTYWM\\\\\\\/h9CopKdzk4rJhWWpj8xg\\\\\\\/lUnbif0+JZhGzjiV7Izj2ge8NQm4n5G+gmHxGIOpZE1ML+UbNYnbE41YYupC1v0aZdgwBM1zUoa6P2bmapw0uu78I2p5\\\\\\\/H82VKT\\\\\\\/8CABL+RpzXAt4eGaRLIz8EN25jf5maZzNPBFAPNvTXnB5+JYcegwCifDChCRhYqyuHKcRFyCQRXObJNKdENAy0\\\\\\\/gifjI4htZiDG6RumA18cQHehL7JVtP70eOQVirLFKn7XjYo686OTLFwaoYbY553pU4dgdng3JNMHU2aWZ6wX9s6WxCMWw2LeLHDvaBu8XqpzXX6EvV9U4Caqlr5DTh0qMpZE5djEzvu4NWMXJVZWoOGmT5nLfZ9twBXJKahrkhWRJrGxKlHH\\\\\\\/FEwFN5TxesRch9NSGA0TUKhUvye3YRCtSUsxEg+\\\\\\\/+UdFwfOmEY5qb7ZQwg1kUI8RIjFOPAg\\\\\\\/8OVW9Vb1AJM+ibjn4FJD8+dxNqsx+KtnlH0BnjwFmyZy4XPCFM6Tu5p+DxS1ScEVSOhNd06iJvONm56gC0Dz4i4SXZrEmnRaisb479\\\\\\\/1yjipaL+wGnWeuingzET+7crPQgvm0UfU3VbviwOdEfYRv732HjXlOzmo6jPqslHvow+lQb1oF\\\\\\\/KvWV2ADZ09AJ+NPQa9K5JExLPdsJLjKSi21R1f9Fpfd4L8YovjUClUA3MXlpAeNdlq+yVeEWlQbeD6Ay3iu7NSNcDO10fBZMEzdKBayRZxxag4d4xDX2NmVKFRrsH8BVZ9E2LmXqwozIExn7pC\\\\\\\/2Oaag\\\\\\\/OiS5KcLUp9vDWduJt+Msz9JrbvbTPcMimQBa2m1wvi2r3iebgjfTdoxkcP\\\\\\\/Lu+zFQ60\\\\\\\/S1XT1MdgqLBWgQIMU3SdBvmlq\\\\\\\/zG2\\\\\\\/Csc+OngoMoUaw+Rj4H6jqpZH45hlemxH8ouj1kPY2jQ\\\\\\\/KMRqqmQWxXNjbJXTgQrJM648GWo4LmqfTPVA8h+psOiDchbrK4DuugHkc+oAgtWNSApyFr+ONournBUuJ3jbYIpvwheTtRVsdMnjI+\\\\\\\/5dmGoBsaNnMvPYoWjlAJKl1Swov7PLqMjx\\\\\\\/inL022K9LbfdOwiLznHKryjEcbakJS\\\\\\\/fuCi0iD9P+mo6i7B7cVICl1um4Ih7Ntl5Dns1Gx0H8f5Gy0mai0Y7SsVO+1s+aLNVEfE6sFNC\\\\\\\/ee1tS4frNJ4Gg\\\\\\\/mY99WrxGl2zsKAD3\\\\\\\/YzsnsQu2RMhLbP4cx335VRffag6XHV2HtSvXJePZ3i4la09H9Qgd0IdxIjDJGaqQSqnb132QfE7LFLqyH439usWJRjyD8AW0qru3nD1RQvQ7E7wufy+IDHSd80FuXTGnpnFfGUk3b\\\\\\\/UaWzxciBVPlTrLJCqQ11R7BAqZxyHjLxOc15yymAonpTh9L3Ezcc\\\\\\\/U7RuHpCS72icgN5Fsyh8Cw3bntGxYBLjRdPIQOvTf9I9Jc+v5WdfH5njgCkVYfAKyl1h5TSqgAjEkQMumqeiyOsWHI5FPigkFarCh5nTd2ZjTTJf7EZpJlKIXE6Ix3JfBaM4XfYmV3D73lR5BuKyHV7EveNOhIBmxxLuFofyGbosXJOnsPj\\\\\\\/7cJbIlO7S5nBmhM7Cg4iudrsetWO0nvi3Ye8yME6geFl39cqIF\\\\\\\/2c2IxtZQk2LPERf0iKkCI1ONCcNe6n6GJon9IZIx\\\\\\\/ktNNllL6ZRaZixWv0gLGey6s\\\\\\\/7su\\\\\\\/eahy3y1hst1aSeWP4CVH3sCAPDwrMX2+8QmoU9s7c3PIMQmot9RAFNBiAgg4xzovHflfk\\\\\\\/HOqYtknU4M5XHReGgEeplmGxlorn9+hreBTRJw9dRXX+XTN9usp5w+iCMBbyUW8gXWl3yNzAwwy5OevLrY25rX4BafRI4BgCPbRD82PFeHpjaE\\\\\\\/T78PUl8gHlhsuX2fugtYOmVfNbuLXzywlFmfNVSwyPrAz1WKUV5rKRM\\\\\\\/IFhqY0PgGKzEfBBn+\\\\\\\/XehdnoF1o5Wmw48PPVcgYTXf9gIqHamPBhXzwlMgE4ttsmcM5dCous0TK1Xaq6izyd\\\\\\\/oFxlABHGk+QFoya+csUannw77lXrdgbTiJ92wfkEGWk06Sf7weVt\\\\\\\/0zm9JsYVuLW8tU4scB2r3JjYYVCXPJ7JvtKXIR+GcYsl78\\\\\\\/xBV+l+6tKtX2fB93MufEUwNgqH27a3uoXzy6eTveeB\\\\\\\/zbdZn9Y+J0ViNtiYGp1RIgXfQqbeVilYcGzhVuPfJDnLz8QgHdzxYwePTpVbiSen+Tn3xDUSGcY7KBz5EFjEWjrALhZ\\\\\\\/kllAAiWN9\\\\\\\/tuTPDIrzHd+FnIAcB0Q8nIQxb0Iwv6lR5Uyn2gpW8rtq8SrHXgSPSERLnaiS7revvKNeZoqI4z90ULyHVBaUvsJYZKVggUD+\\\\\\\/MJSXVH52PM9OJCs4USWtkf+XU34XU3XoG8vbUd014C\\\\\\\/g+hN0m+EgpzgwXU7kna5NzPOx0k8aI4lLMT53MW3jIls+GzQwVL4bttK+2EzNRpycVeDM5uSDlgPHcGu8K4M5gr8+SMc0VIMSVEQrzMaoEv\\\\\\\/Elcorpj5nwNCyMfKjuHuI30T13nBYm7E7wrfEWB+1PD4YyOFYykgCBDsq\\\\\\\/U\\\\\\\/qVvXX1S6Mg1T5s4Kc3jdt5UXOQKuswOg\\\\\\\/DK9AulW7FKnCkI8mykQR\\\\\\\/4ruuCgP+ZIX5gPlI5tgGg9McrCBZ3qJbdYhlP+6pF6Kd5l29rdjiCpMnCUY\\\\\\\/KhtbbyVU3qi1aueHQ2JnjRpvpnlfCTY\\\\\\\/4\\\\\\\/HSjHvjB3EIl0yKLoljfAci9yJfqLwFmLw2FKiXEToMrfA4Ke+8bJko1a+uFQRtvKEluu64uB0328xlAix8b6to\\\\\\\/JcKkVFtd8uiJGLGKirr62J0qd2dncdu+hsCV9QLphKPRhXCs+9vpUh2gl\\\\\\\/He13JAjk+lgAcj6dDLvzkisdvM1c6RhHjbAwlPo6UMe2B0yGuP+1536FX4r7zURizKN2UGql3bAKMe1nyPCP\\\\\\\/ufEfNgqsr4M4s3DcD6TXpB6oo1zpYxgPMJdo9R7CQw2KYPPR4opsxC6103YPhnNH1wrEAH66hK0kQDRp2v7FdHnI\\\\\\\/LT8Zgu1PQfAE48Cbapa+yGqzn+8FRfUpQo30uevtSSywihrjov7jyIu8+5s2kZdl7iQA7hSTEPcqxDcthdjYJZmb5glWbnOXZBul7FMgACl070sMzg392lhgOJyitgW718mYG2EpI+mnwuCcfK6I2jsRcNEYcpA1d4YQl1lEJbUrLNBMmy8Q+BmArWISZDGmRcAzfTTSATuecuXNBqPYvnpG1XfU6XFZa0WAxEFW2I8cQC6HRqf2G4av14JNIRu5IOgufuOBOSTSPaCn3Sz5bpH5hvTpTXUKYRYcKj5GzbKl17+NzEf\\\\\\\/njGEaIAsvrA1b4Ke8QkWxEoHJjfpvGCei2tVF+e46fai2i8O7W0UmBKglLqXUp2x8js8OoA5pg3H2MTHg5rvhbKz6T0Wqbf37ZQ+7ULQHq9IdLSZzMQfnSUpAMgMgm4swGDxXvWsAY1sHEl8bQy98Q66lydIVWICAdA3q8qrH269vrpFSRqZndAJAwNEarM4DiWvBKAAshUayK56aij3nImFh\\\\\\\/V\\\\\\\/uADf3j2h1Q==\\\u0022,\\\u0022iv\\\u0022:\\\u00227H0c71vU5KaF8mvwTrLihA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e45480e2bca15e19920068edc5889ad226beecfe8c7e4b5102e281ff985373b7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fAnTDa\\\\\\\/dUBhHgYpR9CpIfzmhIvqqjI1+3JIF1EQ2GoJ7nWEh7eoxKvzZ8Pgx8VU2hRse7U1GNWsvodMEO0C50kE1hrVQJ0eilyhUBW+MW8wl2KdZPYQ4heYeHDp1RfyDUyi\\\\\\\/kGpxH8Oabo6Yzl7imSYvF5DQ9VALeHbnW0O7yhQf6a7djJL6YEVYKnjmGP8FKVIK2a+9PNwiJUF\\\\\\\/ITnf34GMS9xvlbJeDmdScooKdS7vNJhyVrvL5SznDaRwXniWX6m25Go3lzKZHEwPmQMHKYaQBkdvyAYeB5SmB9cvdNRHUcs3MQm2NsJ3HsdiucMaXB+BpXlBq9WzB00cPKtyjFuZaSo5V9NJN53rUaYkgjkCuno8K77P8r8C9H4vI5dA9kK\\\\\\\/ppJV5NJl3rYGydJNtkAyjXMXe3y7YzUD5o4J5awhJ8ZWib72Hw74M9IUo+lX9ROo2XLbDYwSjC8Vycxc3q7MGg7gcVknhjcOyl2NHc7qBhi2e\\\\\\\/naFcfEDZB\\\\\\\/l2XcUDVlwkJHiIOkwCkzsJ+c2n3Bvl2GM1sOcR7wu1gvdA5LdS0Qe0j3b5DZ2cwWiPRsv6wVQXGq3b\\\\\\\/TZRtLHl2u8DcXxVZanG+CvAZUsgKyCgMVpZqR+0dTgaDddQG1NZQMzKtHIleNjPFCX2XIwQQS1jIRjunJNdgI78kckXbfmvOEvvZOS4CvwFsnSt2wE4QZ7rfd1V4on0HYs8zMRa2JUm\\\\\\\/s0QJ5KcjfZQYIpiwDN+yCQ9cm5fAUcyHk+i2pOwRDbAJ4hLEKq0\\\\\\\/X7P+uxFPTGbhlFfZkteNPvzG+5v8RJ0c39upD2EsG0BhrmnpWqjHLlkg\\\\\\\/RWWIzMBgxORNLKDUcO0Go14MCdD4Iy2DeHVtWwG2fu80XpRyw\\\\\\\/G55dRJDgGLYkEco+pkyG\\\\\\\/rR\\\\\\\/6v4DP9\\\\\\\/YlUEzNcYuOY2UrtX6Ga44qNl5nACWed0xGo4qRhILJXoUC4Om5Zr2wyGf8wGJTuj3n25NIPNH1YAHlcFXx0CqigLUa8dsJozZAV8GTurlEK0tsSiVtvB6WDzIaliO4icqsBwxHacUfF8ZD6JD1CcSBPM2474Gd8rrTZFzY9Af2ou3ipcgMI\\\\\\\/CGvjwFFKMg7Jpgyfqsc0X1lqRziZJpVA77Gho\\\\\\\/zVpKPLYGOMF3o6ArW4+Au4QMa0aQ\\\\\\\/gbrDErhB51\\\\\\\/52agHMusZEJg6ACLjYZNIhTCPyQ9gwJmPi84kHiNhuc4G\\\\\\\/dP4xGOdS2CM90dI0B6CH7YaDPXct0ZFLTNSrrp3JriEhcv2TKUQaZNhHb3rly6q257MhqYYqQGiztTJTcBWQXmE\\\\\\\/V2LajWcGRUKeLkLcp5Gb6nyymhQzhhWjbZIrrIqneIo4v\\\\\\\/17pHj71z77lY9m0G+HvXs+ntiRyQj8B0qBvuBGJu7eeHON9V6J0WVlrYm4NYT28hBIH8kvLAD+uAySiqaeu\\\\\\\/0VLzikx\\\\\\\/NvuSVkHuNf+BlfcTIzyavhvejUGXce7fUIurc4b+Uga+4eq9MKPlslN3FZjDv+qZcD00mAVDE\\\\\\\/hI8+TQlLJj3HIx\\\\\\\/auXU7DfpDdKlNoxGzKjUbPdXKIUyw56BU2jwhYcaxN+Qyw8fDxIFIDnqcY3a4rG2NRJG+m55oTpn3FEApcm317RgqrlgIWF6Dq2e+Qt2OSI\\\\\\\/eCVLeEkuu0qDeIEiPH0zz3tfHC7oVbuFV\\\\\\\/9Hv4j93roQv6xa8ZL3xq2\\\\\\\/TNGLiVDuA2\\\\\\\/XtSRhmUhAlSbH2E72J4oAggHGNX\\\\\\\/TMoSefLzKc7sWy24xDllF+ESU9FY54I7IxnRp5UW659pbYO+GEzK49DDRx9BrXvcln9SV2ZISWyUSLtTwLrSbsFpB181Y8q2RDRxtFj546QsjIwT3EGkYgQp4pWl60xHHUhfMpTE+TkL2EnzqFEvIGsWAaA8qryXPwrccD+O5Zhp2DXkZFm6I2ptSJGguCdA7z4eLlMs84L7Dzuf0g28waHiXwkExgZkxeykczVwR7tuVCnSSijiJAO1\\\\\\\/u\\\\\\\/V+ZNI5KJxhZC483A==\\\u0022,\\\u0022iv\\\u0022:\\\u002289I\\\\\\\/Te8NLblqxftY81Cgnw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ffb32fba20dd5ad89cd323d3721817efae6d0f6909f5a7bf56fd03d6e545dc47\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459821 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/707" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bTZEwPAp0ft0vWhdRaZnsYAZMoSEoL7VjsAiRtyboio" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/707" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dCTw0Gmk78HguQ2m70EunK6VhPJNAtqwTYoWEeKxd0k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/693" title="unfulfilled Interacting Genes Display Branch-Specific Roles in the Development of Mushroom Body Axons in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/717" title="Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent Haemophilus influenzae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459821" id="node459821" data-pisa="ggg;4/4/707" data-pisa-master="ggg;g3.113.009464" data-apath="/ggg/4/4/707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequence Profiling of the <em>Saccharomyces cerevisiae</em> Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Claire Jubin</span>, <span class="highwire-citation-author" data-delta="1">Alexandre Serero</span>, <span class="highwire-citation-author" data-delta="2">Sophie Loeillet</span>, <span class="highwire-citation-author" data-delta="3">Emmanuel Barillot</span> and <span class="highwire-citation-author" data-delta="4">Alain Nicolas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">707-715;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009464 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459821" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Claire Jubin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BJubin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jubin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BJubin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandre Serero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BSerero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Serero%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BSerero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sophie Loeillet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BLoeillet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loeillet%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BLoeillet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emmanuel Barillot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut Curie Centre de Recherche, INSERM U900, 75248 Paris, France. Ecole des Mines Paris Tech, 77300 Fontainebleau, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmanuel%2BBarillot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barillot%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmmanuel%2BBarillot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alain Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alain.nicolas@curie.fr" class="" data-icon-position="" data-hide-link-title="0">alain.nicolas@curie.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/4/707" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459821" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459821/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/707.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459821" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459821/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/707.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459821" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459821/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/707.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459821" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459821/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/4/707.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459821" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459821/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Advances in high-throughput sequencing (HTS) technologies have accelerated our knowledge of genomes in hundreds of organisms, but the presence of repetitions found in every genome raises challenges to unambiguously map short reads. In particular, short polymorphic reads that are multialigned hinder our capacity to detect mutations. Here, we present two complementary bioinformatics strategies to perform more robust analyses of genome content and sequencing data, validated by use of the <em>Saccharomyces cerevisiae</em> fully sequenced genome. First, we created an annotated HTS profile for the reference genome, based on the production of virtual HTS reads. Using variable read lengths and different numbers of mismatches, we found that 35 nt-reads, with a maximum of 6 mismatches, targets 89.5% of the genome to unique (U) regions. Longer reads consisting of 50−100 nt provided little additional benefits on the U regions extent. Second, to analyze the remaining multialigned (M) regions, we identified the intragenomic single-nucleotide variants and thus defined the unique (M<sub>U</sub>) and multialigned (M<sub>M</sub>) subregions, as exemplified for the polymorphic copies of the six flocculation genes and the 50 Ty retrotransposons. As a resource, the coordinates of the U and M regions of the yeast genome have been added to the <em>Saccharomyces</em> Genome Database (<a href="http://www.yeastgenome.org">www.yeastgenome.org</a>). The benefit of this advanced method of genome annotation was confirmed by our ability to identify acquired single nucleotide polymorphisms in the U and M regions of an experimentally sequenced variant wild-type yeast strain.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/analysis-repetitive-regions" class="hw-term hw-article-keyword hw-article-keyword-analysis-of-repetitive-regions" rel="nofollow">analysis of repetitive regions</a></li><li class="kwd"><a href="/keyword/reference-sequence-bias" class="hw-term hw-article-keyword hw-article-keyword-reference-sequence-bias" rel="nofollow">reference sequence bias</a></li><li class="kwd"><a href="/keyword/genome-hts-annotation" class="hw-term hw-article-keyword hw-article-keyword-genome-hts-annotation" rel="nofollow">genome HTS annotation</a></li></ul><p id="p-5">The efficiency of DNA sequencing has tremendously increased in recent years, due to the advances of high-throughput sequencing (HTS) technologies and new instruments that allow rapid retrieval of sequence information from total genomes, including the human genome, in an appropriately equipped research laboratory setting. Massive parallel short-read sequencing technologies produce a large collection of individual reads. Afterward, the extraction of information is usually performed by analyzing the results of reads that are aligned against a reference sequence. The success of read mapping greatly depends on the proximity, accuracy, and degree of assembly of the reference genome to the experimental sample and is inevitably filled with noise due to the presence, in every genome, of repetitive regions. Repeated sequences lead to the production of identical or slightly polymorphic reads that originate from different chromosomal locations. Robust analyses therefore depend on how efficiently one can minimize the confusing effects that multialignments pose.</p><p id="p-6">The most intuitive strategy is based on the selection of the “best” alignment among multialignments according to the number of mismatches. Short Oligonucleotide Alignment Program (SOAP) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Li <em>et al.</em> 2008b</a>) and Program to Align Short Sequences (PASS) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Campagna <em>et al.</em> 2009</a>) programs implement such strategies in which the “best” alignment is defined as the one with the smallest number of mismatches. A second approach is implemented in a program MApping short reads and calling variants using mapping Quality score MAQ (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Li <em>et al.</em> 2008a</a>) uses the notion of “mappability.” Each alignment is associated with a score, reflecting the probability of correctly placed reads. This information is then used to filter alignments before the search of variants. The third approach, developed here, is a reverse strategy that consists of an advanced preanalysis of the reference sequence. It aims to identify, at a nucleotide-level resolution, the unique and multialigned subgenomic regions. This is achieved by conducting a simulated HTS experiment. Virtual reads are generated along the whole genome and then are back aligned onto this sequence. This is the strategy initiated by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Koehler <em>et al.</em> (2011)</a> to analyze the unique subpart of a genome where reads are uniquely aligned, while the rest of the genome remains unexplored.</p><p id="p-7">The difficulty in exploring regions containing multi-aligned reads depends on several factors. Biologically, it depends on the global complexity of the genome sequence and the density of repeated motifs, their size, and degree of polymorphisms. Technically, mapping also depends on the accuracy of sequencing, the length of the reads, the number of mismatches allowed during the read mapping step, and in the case of repeated regions, on their polymorphisms. Short reads also decrease the power of assembly for building accurate contigs because repetitive regions create ambiguities (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Treangen and Salzberg 2012</a>). Altogether, these factors affect the robustness of the alignment information and lead to potential false-positive (FP) calls in resequencing experiments that aim at identifying genome variants. Thus, in many HTS studies, multialigned reads are set aside, if not completely discarded. For example, in the whole-genome resequencing study of the 1000 Genomes project (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Clarke <em>et al.</em> 2012</a>), the variant calling was restricted to 80–85% of the human sequence. Referred to as the “accessible genome,” this is defined as the portion of the reference sequence that remains after excluding regions containing ambiguously placed reads or unexpectedly low/high read coverage. More recently, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Lee and Schatz (2012)</a> used the expression “dark matter” when referring to multialignment regions that are difficult to analyze. The efforts to identify the “dark matter” consisted of setting-up a new parameter referred to as, the Genome Mappability Score (GMS). GMS is a statistical measure for the probability that a position is at risk of being multialigned, based on read mapping. This parameter gives a score at each position of the genome that can be used to filter the experimental results, according to a threshold that defines the single nucleotide variants (SNVs) termed robustness. Operationally, it reduces FP calls but finally restricts the analysis to highly “mappable” regions.</p><p id="p-8">Here, we report two generic and complementary bioinformatics strategies based on advanced annotations of the reference genome. As a proof of concept, it allowed us to (i) deconvolute the <em>Saccharomyces cerevisiae</em> genome into unique (U) and multialigned (M) regions using virtual read analyses, (ii) establish a list of the intragenomic SNVs that characterize whole-genome repetitiveness (provided as a resource in the Saccharomyces Genome Database [SGD] database; <a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>), and (iii) explore the M regions to define subsets of unique regions (M<sub>U</sub>) and identify M region distinguishing variants. The alignment filter we developed, “g-deNoise,” anticipates the impact of intragenomic SNVs in multialignments. These advances allowed us to accurately identify acquired SNVs in both U and M regions of an experimentally sequenced variant yeast strain.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Production of virtual reads</h3><p id="p-9">To generate virtual HTS short reads, the complete genome of <em>S. cerevisiae</em> was extracted from the SGD, version February 2011. The reference sequence comes from the S288c strain background, which is a standard genetic background for functional studies (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Mewes <em>et al.</em> 1997</a>). To match our experimental HTS reads, the reference sequence was first converted into the dinucleotide color space, specific to SOLiD sequencing (Life Technologies), with help of a color to base-space conversion matrix (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Breu 2008</a>). Virtually perfect reads of different lengths were generated on the forward strand at each position of all chromosomes analyzed. Virtually perfect 50-nt base-space reads on the forward strand were generated from the same reference.</p></div><div id="sec-3" class="subsection"><h3>Reads mapping</h3><p id="p-10">The simulated dinucleotide color space reads were back aligned against the <em>S. cerevisiae</em> reference genome using the “mapreads” program from Applied Biosystems SOLiD “corona lite 4.2.2” software package. Our standard mapping parameters were set to allow six mismatches per 50 nt-read alignment. To generate an exhaustive mapping result, no threshold was fixed to limit the number of reported alignments per read. It is noteworthy that six mismatches in color-space are not equivalent to six mismatches in base-space (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Breu 2008</a>). In the base-space simulation experiments, the virtual reads were aligned on the reference sequence with the software BFAST (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Homer <em>et al.</em> 2009</a>). Only alignments without gaps and containing at most three base mismatches were considered. No threshold was fixed to limit the number of alignments reported per read. The draft sequence of the strains BY4741 (BY4741_Toronto_2012) and BY4742 (BY4742_Toronto_2012) (<a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>) also were used to identify constitutive Single Nucleotide Polymorphisms (SNPs) of our laboratory BY4741 and BY4742 strains and were compared with our mutation accumulation lines in the reference S288c sequence.</p><p id="p-11">The experimental 50 nt-reads were aligned on the <em>S. cerevisiae</em> reference sequence using the same program, mapreads, and the same parameters were used as for simulated 50 nt-reads: six mismatches were allowed and no threshold was fixed on the number of reported alignments.</p></div><div id="sec-4" class="subsection"><h3>Annotation of U <em>vs.</em> M regions</h3><p id="p-12">After the mapping step, the alignment coverage plot was computed on each chromosome of the reference genome. These graphs help to visually assess the coverage pattern of the virtual HTS profile. A read is considered to be multialigned if it has more than one alignment. Identification of M regions is performed after discarding the unique alignments from total alignments. Thus, the M regions correspond to genomic regions still covered by these “ambiguous” alignments, whereas the U regions are defined as the complementary regions located between two consecutive M regions. The coordinates of the U and M regions were defined at the level of single-nucleotide resolution, using the S288C reference genome. M region identification was performed for 25-, 35-, 50-, 75-, and 100-bp reads, and for a maximal number of allowed mismatches equaling zero to six. The coordinates of the M regions are available on SGD for 50 and 100 nt-reads and a maximal number of mismatches equal to six (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FileS1.zip">File S1</a>).</p></div><div id="sec-5" class="subsection"><h3>Identification of intragenomic SNVs</h3><p id="p-13">Because the virtual HTS reads were generated without introducing nucleotide variations, the presence of mismatched alignments within the M regions only reflects SNVs that distinguish intragenomic repeated sequences. Their coordinates were determined using AnnotateChanges (v0.2, corona-1.0.1r0-4) provided by ABI (Life Technologies). Intragenomic SNVs identified with 50 and 100-nt virtual reads are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-6" class="subsection"><h3>Yeast strains sequencing</h3><p id="p-14">Starting from our frozen <em>S. cerevisiae</em> BY4741 strain (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Brachmann <em>et al.</em> 1998</a>) cell stock (−80° in glycerol), we constructed mutation accumulation lines. Four independent yeast colonies were picked and grown in parallel on YPD (rich medium) plates at 30° upon 100 single cell to colony bottlenecks, generating at the end, four independent clonal lines, wt_A100, wt_B100, wt_C100, and wt_E100. Barcoded and paired-end libraries (50+35 nt), containing fragments of approximately 200 bp, were prepared according to Applied Biosystems (Life Technologies) standard protocols and sequenced on the SOLiD v4 instrument at the HTS platform at the Institut Curie. The sequenced reads we used contain more than 80% of bases with a percentage of sequencing errors less than 0.1%. After mapping with mapreads, from the Applied Biosystems SOLiD corona_lite 4.2.2 software package, the total number of paired-end reads uniquely aligned in U regions was: wt_A100 (15,780,119), wt_B100 (15,438,542), wt_C100 (16,614,434), and wt_E100 (19,658,287), corresponding to a 130- to 200-fold coverage.</p></div><div id="sec-7" class="subsection"><h3>Detection of acquired mutation and biological validation</h3><p id="p-15">To identify acquired mutations in the four wild-type experimental samples, SNP identification was first performed with Bioscope (V1.3) in the herein defined U regions. A SNP call was retained if the base change was seen in at least 70% of unique alignments and supported on both strands. Bioscope filters are not adapted to SNP analysis in repetitive regions. Thus, in M regions, we first performed base change calls starting from color mismatches with AnnotateChanges, then we filtered multialignments with the new “g-deNoise” filtering method reported here. At last we extracted acquired SNPs supported by alignments that became uniquely aligned after filtering. A call was validated if it was detected by 10 unique alignments on each strand corresponding at least to five different starting points. “Noisy” homopolymers and microsatellite tracts were eliminated. The parameters were set such that any FP was obtained after Sanger sequencing confirmation. To align the reads, we used the S288C version June 2008 (S288C_reference_genome_R61-1-1_20080605), and the results were converted into coordinates consistent with the version from February 2011.</p><p id="p-16">To validate calls for SNP variation between the reference genome and the sequenced strains, the corresponding regions were PCR amplified and Sanger sequenced (Big Dye Terminator V1.1 Cycle sequencing kit from Life Technologies) using standard conditions. Results from these four wild-type strains were also used to compare the mutational landscape of wild-type and mutator strains (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Serero <em>et al.</em> 2014</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-17">To generate a virtual HTS profile from a reference genome, our strategy as illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> was the following. Stepwise, we: (i) broke down the reference genome sequence into virtual short reads; (ii) aligned these reads back onto the reference genome, using the same parameters as used for the experimental HTS data; (iii) identified the chromosomal coordinates of the unique (U) and multialigned (M) regions; and (iv) annotated the SNV variants that internally distinguish the M regions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/707/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principle of virtual HTS profiling of a reference genome. (A) Virtual read production. A read is generated at each genomic position on the forward strand so that the coverage reaches the read length (50&#xD7; in this example). The next steps are illustrated with the chromosome II. (B) Read mapping. Although reads were only generated on the forward strand, some of them, coming from repetitive regions within the same chromosome (black reads) or different chromosomes (colored reads) exhibit multialignments and/or are aligned on the reverse strand. (C) Annotation of the M and U regions. The start and end coordinates of continuous M regions are reported after unique alignments were discarded. Gaps between M regions are converted to U regions. (D) Intragenomic SNV identification. The mismatches analysis in M regions allowed identification of the indicated SNVs found in repetitive regions of the genome. HTS, high-throughput sequencing; SNV, single-nucleotide variant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1622443995" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principle of virtual HTS profiling of a reference genome. (A) Virtual read production. A read is generated at each genomic position on the forward strand so that the coverage reaches the read length (50&#xD7; in this example). The next steps are illustrated with the chromosome II. (B) Read mapping. Although reads were only generated on the forward strand, some of them, coming from repetitive regions within the same chromosome (black reads) or different chromosomes (colored reads) exhibit multialignments and/or are aligned on the reverse strand. (C) Annotation of the M and U regions. The start and end coordinates of continuous M regions are reported after unique alignments were discarded. Gaps between M regions are converted to U regions. (D) Intragenomic SNV identification. The mismatches analysis in M regions allowed identification of the indicated SNVs found in repetitive regions of the genome. HTS, high-throughput sequencing; SNV, single-nucleotide variant.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/707/F1.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/4/707/F1.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/707/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/707/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464190" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Principle of virtual HTS profiling of a reference genome. (A) Virtual read production. A read is generated at each genomic position on the forward strand so that the coverage reaches the read length (50× in this example). The next steps are illustrated with the chromosome II. (B) Read mapping. Although reads were only generated on the forward strand, some of them, coming from repetitive regions within the same chromosome (black reads) or different chromosomes (colored reads) exhibit multialignments and/or are aligned on the reverse strand. (C) Annotation of the M and U regions. The start and end coordinates of continuous M regions are reported after unique alignments were discarded. Gaps between M regions are converted to U regions. (D) Intragenomic SNV identification. The mismatches analysis in M regions allowed identification of the indicated SNVs found in repetitive regions of the genome. HTS, high-throughput sequencing; SNV, single-nucleotide variant.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-9" class="subsection"><h3>Mapping of the U and M regions</h3><p id="p-19">Virtual reads of various lengths (25, 35, 50, 75, and 100 nt) were generated at each nucleotide position of the yeast reference genome on the forward strand (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Afterward, as illustrated in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>, the virtual reads were back aligned against the reference genome with a given number of allowed mismatches. Thus, the final coverage, or full coverage, depends on the read length. For example, virtual read generation for a 50 nt-read length produced a 50-fold coverage. After mapping, the uniquely aligned U regions exhibited the expected 50-fold coverage, whereas some regions exhibited a variable excess of coverage, reflecting multi-aligned M regions. Although the virtual reads were generated on the forward strand, reads were occasionally aligned on the reverse strand, indicating multialigned sequences on opposite strands (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). The start and end coordinates of the continuous genomic regions, covered by multi-alignments, define the M regions (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). The sum of the U regions defines the <em>S. cerevisiae</em> U genome, or “uniqueome” as termed by <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Koehler <em>et al.</em> (2011)</a>, and complementarily, the sum of M regions defines the <em>S. cerevisiae</em> M genome, or “dark matter” as termed by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Lee and Schatz (2012)</a>, thus providing an operational annotation for HTS experiments and a readout of the genome sequence architecture and its complexity. The M regions are visualized as peaks with different heights, meaning that reads, generated in other regions of the genome, are cross-aligned. The annotation of these peaks is described below. The annotated details of the M regions within the <em>S. cerevisiae</em> genome, based on the virtual HTS profile, are available at SGD. The visualization of M regions in the SGD browser is illustrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/009464SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS1.pdf">Figure S1</a>.</p></div><div id="sec-10" class="subsection"><h3>Annotation of the M regions according to the read length and mismatch tolerance</h3><p id="p-20">Varying the read lengths and the tolerance for mismatches is expected to influence the extent of multialignments during the mapping step. Therefore, we examined the impact of these parameters for virtual read lengths of 25, 35, 50, 75, and 100 nucleotides, while varying the maximal number of mismatches permitted. The results are reported in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>. As we see for 25 nt-reads, the cumulative sum of M regions rapidly increases from three to six mismatches so that the whole-genome sequence becomes fully covered by M regions. In contrast, for read lengths ≥35 nt, the cumulative length of M regions is much smaller (approximately 10% of the genome) and is weakly sensitive to the number of mismatches permitted. As an example, the impact of mismatch tolerance is illustrated in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> for chromosome II. This result shows that increasing the number of mismatches leads to the appearance of additional and specific peaks of increased coverage with variable heights. For low coverage peaks, we observe that there are a few isolated multialigned positions, corresponding to low noise motif redundancies, a feature that is expected to increase with enhanced mismatch tolerances per read. Indeed, increasing the number of mismatches from zero to six leads to few additional peaks of low coverage, but roughly similar M profiles are observed. Quantitatively, if one now varies the length of the reads with a maximum tolerance of six mismatches, the percentage of the yeast genome covered by M regions only slightly varies from 10.5% (35 nt-reads) to 8.4% (50 nt-reads), down to 7.7% with 100 nt-reads. When read length increases, the total number of M regions in the genome regularly decreases from 5765 (35 nt), 1824 (50 nt), 1148 (75 nt), and 873 (100 nt) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figures 2, B and C</a>). Altogether, these analyses indicate that in terms of repetitiveness within the <em>S. cerevisiae</em> genome, there is a marked loss of genome complexity when one considers polymorphic sequence motifs shorter than 35 nt-reads. However, it is expected that this threshold of genome complexity will vary from one organism to another, depending on the overall base composition (GC percent), strand biases and the genome content in terms of repetitiveness and evolved history, for example if derived from an ancestral genomic duplication.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/707/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of M regions. (A) Cumulative length of M regions according to the read length and the number of tolerated mismatches. Histogram of M region lengths annotated with (B) 50nt color space reads with six mismatches allowed, (C) 100nt color space reads with six mismatches allowed, and (D) 50nt base space reads with three mismatches allowed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1622443995" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Characterization of M regions. (A) Cumulative length of M regions according to the read length and the number of tolerated mismatches. Histogram of M region lengths annotated with (B) 50nt color space reads with six mismatches allowed, (C) 100nt color space reads with six mismatches allowed, and (D) 50nt base space reads with three mismatches allowed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/707/F2.medium.gif" width="440" height="298"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/4/707/F2.medium.gif" width="440" height="298"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/707/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/707/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464139" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Characterization of M regions. (A) Cumulative length of M regions according to the read length and the number of tolerated mismatches. Histogram of M region lengths annotated with (B) 50nt color space reads with six mismatches allowed, (C) 100nt color space reads with six mismatches allowed, and (D) 50nt base space reads with three mismatches allowed.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/707/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="50 nt-read multialignment coverage on the S. cerevisiae chromosome II for 0, 2, 4, and 6 allowed mismatches. Peaks greater than 50 are cut off." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1622443995" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;50 nt-read multialignment coverage on the &lt;em&gt;S. cerevisiae&lt;/em&gt; chromosome II for 0, 2, 4, and 6 allowed mismatches. Peaks greater than 50 are cut off.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/707/F3.medium.gif" width="333" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/4/707/F3.medium.gif" width="333" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/707/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/707/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464166" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">50 nt-read multialignment coverage on the <em>S. cerevisiae</em> chromosome II for 0, 2, 4, and 6 allowed mismatches. Peaks greater than 50 are cut off.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Robustness of the M regions according to the method of virtual read production</h3><p id="p-23">To compare reads generated with different sequencing technologies, we also performed the HTS simulation by generating virtual reads in base space (for example as in Illumina sequencing) rather than color space (like in SOLiD sequencing). Using fixed parameters, 50 nt-reads and a maximal number of three mismatches in base space (equivalent to six mismatches in color space), the percentage of genome covered by M regions, after mapping with the BFAST software, was slightly lower than in color space (6.55% <em>vs.</em> 8.4%). The M regions identified in base space simulation are included in the M regions identified in color space. The additional M-specific regions identified with color space simulation are due to the enhanced calls in low complexity sequences and exquisitely discriminate the Ty regions (see peaks at 5−6 kb in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> compared with <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). Overall, the identification of the M regions is robust and is only marginally sensitive to the method of virtual read-encoding and alignment strategies.</p></div><div id="sec-12" class="subsection"><h3>Comparison between simulated and experimental U and M regions</h3><p id="p-24">To validate the accuracy of our virtual HTS results, we performed HTS of the wild-type BY4741 strain (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Brachmann <em>et al.</em> 1998</a>), a strain closely related to the S288c <em>S. cerevisiae</em> reference sequence (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Mewes <em>et al.</em> 1997</a>). As detailed in the section <em>Materials and Methods</em>, four independent yeast lines, subjected to 100 single-cell colony bottlenecks (wt_A100, wt_B100, wt_C100, and wt_E100) were paired-ended sequenced (50+35 nt). The alignment of the experimental 50 nt-reads was set to tolerate a maximum of six color-space mismatches. Then, the read coverage was calculated using the set of U regions defined by our virtual HTS profile and corresponding to 91.6% of the genome. The comparison of the U and M regions of the <em>S. cerevisiae</em> genome, defined by the virtual HTS procedure, and the experimental data, is illustrated in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The correlation between these two sets of data are very strong (r<sup>2</sup> = 0.99), indicating that the preannotation of the reference genome is extremely accurate to help the analysis of experimental samples.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/707/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation of coverage between the virtual HTS profile and experimental HTS profile from the S. cerevisiae chromosome II. Each point corresponds to the mean coverage of chromosome positions (2-kb window), in Log10 scale, computed before any alignment filtering (U + M regions), on virtual HTS profile (x-axis) and on experimental HTS (y-axis). Coverage is computed for 50 nt-reads and six mismatches. HTS, high-throughput sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1622443995" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlation of coverage between the virtual HTS profile and experimental HTS profile from the &lt;em&gt;S. cerevisiae&lt;/em&gt; chromosome II. Each point corresponds to the mean coverage of chromosome positions (2-kb window), in Log10 scale, computed before any alignment filtering (U + M regions), on virtual HTS profile (x-axis) and on experimental HTS (y-axis). Coverage is computed for 50 nt-reads and six mismatches. HTS, high-throughput sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/707/F4.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/4/707/F4.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/707/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/707/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464147" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Correlation of coverage between the virtual HTS profile and experimental HTS profile from the <em>S. cerevisiae</em> chromosome II. Each point corresponds to the mean coverage of chromosome positions (2-kb window), in Log10 scale, computed before any alignment filtering (U + M regions), on virtual HTS profile (x-axis) and on experimental HTS (y-axis). Coverage is computed for 50 nt-reads and six mismatches. HTS, high-throughput sequencing.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Deeper annotation of the M regions</h3><p id="p-26">To develop a bioinformatics strategy, to analyze repetitive DNA, we first examined the number and length distribution of the M regions, as defined in the virtual sequencing using 50 nt-reads and a maximum of six mismatches per read. The M regions exhibit large length variations (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). The longest M region (28,934 nt) corresponds to a region containing a Ty (yeast retrotransposon) and the rDNA clusters (only one cluster is annotated in the reference genome) on chromosome XII, whereas many M regions exhibit much shorter lengths with a minimal length of 50 nt, limited by the read length resolution. In comparison, the longest U region measures 107,899 nt (144,151−252,049 on chromosome XIV) and can be as short as one nucleotide when it separates two flanking M regions, for example at position 699,128 on chromosome XV.</p><p id="p-27">The M regions with larger lengths and higher coverage correspond to known repetitive elements within the yeast genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/TableS1.pdf">Table S1</a>). This includes the Ty families, the associated and solo long terminal repeats, transfer RNA genes, and telomeric regions. Included are the duplicated genes encoding the H3 and H4 histones on chromosomes II and XIV, and other dispersed or clustered multigene families such as the <em>FLO</em> genes, involved in cell flocculation. In other organisms, as in <em>S. cerevisiae</em>, multialigned reads from various sources are expected to occur, but with higher frequencies and in larger sized genomes and/or genomes with marked differences in sequence complexity (GC content or presence of pseudogenes and low complexity motifs).</p><p id="p-28">Despite the fact that virtual reads were generated on the forward strand, certain reads aligned on the reverse strand. As an example, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS3.pdf">Figure S3</a> reports the forward and reverse coverage on chromosome II. Alignments on the reverse strand illustrate that some reads are generated within inverted repeat motifs elsewhere, in the genome. Effects of multialignments, due to duplication, also are observed within the histone <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000213">HHF1</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">HHF2</a></em> cluster multialignments in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS4.pdf">Figure S4</a>. Because these sequences are duplicated in opposite chromosomal orientations, multiple forward and reverse reads are observed and aligned on each of the homologous regions on chromosomes II and XIV, as expected. Formally, the presence of both forward and reverse reads at a given position places this region within the M category. Forward/reverse strand multialignments come from various sources. They can reflect stochastic sequence redundancies between chromosomes or local head to tail repeats. In yeast, this includes the highly repetitive Ty elements that integrate within a chromosome, irrespective of strand orientation. The paucity of inverted repeats likely reflects their spontaneous instability that generates translocation events (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Lemoine <em>et al.</em> 2005</a>) when DNA replication is perturbed. Also, a few duplicated regions, reminiscent of the ancestral duplication of the <em>S. cerevisiae</em> whole-genome (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Wolfe and Shields 1997</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Kellis <em>et al.</em> 2004</a>) and its evolutionarily selected and drifted remnants contain constitutive M regions. Altogether, using 50 nt-reads and allowing up to six mismatches, we could estimate that the cumulative sum of the M regions corresponds to 8.4% of the genome (approximately 10<sup>6</sup> nt), comprising 1823 M regions of various lengths (50−28,934 nt), dispersed within all of the chromosomes. Consistently, the sum of the M regions, defined from our experimental HTS data was 8.7%. This slight difference is likely explained by the production of experimental reads containing sequencing errors that are sufficient to generate higher noise and further multialigned reads.</p></div><div id="sec-14" class="subsection"><h3>SNVs annotation</h3><p id="p-29">Multialigned reads are either fully identical to the reference sequence, or slightly polymorphic because of nearly exact repetitive sequences. We then asked whether we could use these variations, since they provide specific sequence signatures that can be used to mark some, if not all, members of a given repeat family. To do so, we identified and annotated all of the base changes from multialigned simulated reads obtained from virtual HTS mapping (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>) and then used these base change annotations as predictive molecular markers to deconvolute the M reads. This annotation, available at SGD, provides a new resource to analyze the diversity of the yeast M subgenome and precisely pinpoints variable positions within repetitive sequences.</p></div><div id="sec-15" class="subsection"><h3>G-deNoise filtering</h3><p id="p-30">The aforementioned SNVs annotation led us to set up a new bioinformatics method, that we termed g-deNoise, to efficiently filter polymorphic multialigned reads generated from virtual and experimental sequencing data. As illustrated in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, reads generated from a duplicated region containing a single base change (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>) align to both regions (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). The principle of g-deNoise, is to use the preannotated SNV as a specific signature to distinguish the source of the individual reads (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). G-deNoise differs from a “Best match” strategy because its filtering principle is only based on the presence of a predicted SNV within the alignments, rather than a best match that is sensitive to the potential presence of an unpredicted sequence variation. First, performing g-deNoise on the virtual HTS mapping allowed us to identify certain regions (M<sub>U</sub>) containing specific polymorphisms that became uniquely aligned, after filtering. These could then be removed and the regions that remained multi-aligned were named M<sub>M</sub>. For example, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS4.pdf">Figure S4</a> illustrates our annotation of the duplicated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000213">HHF1</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">HHF2</a></em> genes. As expected, in the virtual HTS mapping, the reads from the duplicated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">HHF2</a></em> gene align with the reverse strand and exhibit the expected eight SNVs that uniquely distinguish these duplicated genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS4.pdf">Figure S4</a>B). Furthermore, our experimental reads from the line wt_A100, containing these eight SNVs, were also clearly detected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS4.pdf">Figure S4</a>C). In summary, the effect of the g-deNoise filter on the initial <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000213">HHF1</a></em> multialignments identifies the internal M<sub>U</sub> regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS4.pdf">Figure S4</a>E), thus becoming prone to a robust detection of potential variants.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/707/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principle of g-deNoise filtering. Illustration with two near-exact copies of the same repetitive sequence. (A) Read1 and Read2 are sequenced on two copies of the same repetitive sequence. Copy1 and Copy2 are intragenomic repeats that differ by a single nucleotide (A&#x2194;G). (B) After mapping, and before any filtering, Read1 and Read2 are multialigned. Each of these reads has one perfect alignment at the genomic position from which it originates, and one mismatched alignment at the position of the polymorphic copy. The mismatched position corresponds to an intragenomic SNV, preannotated in multialignments of virtual reads. (C) Filtering by g-deNoise consists of discarding alignments allowing such intra SNVs and therefore avoids the miscalling of an acquired mutation. SNV, single-nucleotide variant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1622443995" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principle of g-deNoise filtering. Illustration with two near-exact copies of the same repetitive sequence. (A) Read1 and Read2 are sequenced on two copies of the same repetitive sequence. Copy1 and Copy2 are intragenomic repeats that differ by a single nucleotide (A&#x2194;G). (B) After mapping, and before any filtering, Read1 and Read2 are multialigned. Each of these reads has one perfect alignment at the genomic position from which it originates, and one mismatched alignment at the position of the polymorphic copy. The mismatched position corresponds to an intragenomic SNV, preannotated in multialignments of virtual reads. (C) Filtering by g-deNoise consists of discarding alignments allowing such intra SNVs and therefore avoids the miscalling of an acquired mutation. SNV, single-nucleotide variant.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/707/F5.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/4/707/F5.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/707/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/707/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464257" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Principle of g-deNoise filtering. Illustration with two near-exact copies of the same repetitive sequence. (A) Read1 and Read2 are sequenced on two copies of the same repetitive sequence. Copy1 and Copy2 are intragenomic repeats that differ by a single nucleotide (A↔G). (B) After mapping, and before any filtering, Read1 and Read2 are multialigned. Each of these reads has one perfect alignment at the genomic position from which it originates, and one mismatched alignment at the position of the polymorphic copy. The mismatched position corresponds to an intragenomic SNV, preannotated in multialignments of virtual reads. (C) Filtering by g-deNoise consists of discarding alignments allowing such intra SNVs and therefore avoids the miscalling of an acquired mutation. SNV, single-nucleotide variant.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">As a second example, we chose to identify sequence variations in the highly polymorphic retrotransposon regions. The reference genome contains 50 copies of these Ty1, Ty2, Ty3, Ty4, or Ty5 6-kb elements. The results illustrated in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/FigureS5.pdf">Figure S5</a> show that although several M<sub>M</sub> regions of variable length remain indistinguishable between Ty copies, numerous M<sub>U</sub> regions exist.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/707/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of M regions from two classes of repetitive elements. (A) MU regions overlapping the Ty1-Ty5 elements. (B) MU regions overlapping the six FLO gene copies. The MU regions are represented by horizontal bold bars. (C) Detection of acquired mutations within the polymorphic FLO1, FLO5, and FLO9 multi-copy genes. (1) Alignment of the FLO5, FLO1, FLO9, and FLO1 variant (identified here) gene sequences. (2) Visualization of the forward and reverse strand read mapping of the FLO1 gene on the chromosome I in the experimental HTS from the wt_A100 wild type line with the SOLiD Alignment Browser. The coordinates of the regions are indicated (arrows). The fluorescent green bars outline positions of mismatches consistent with a base change. (3) Same as (2) after using g-deNoise. The reads containing the intragenomic SNVs were removed and the unpredicted (acquired) SNP at position 207,394 persisted. The nature and position of the acquired mutation is indicated in green in the FLO1 variant sequence in (1). This identification of this acquired SNP was based on the examination of the 50 nt-reads, containing the adjacent FLO1 specific polymorphisms. It was also confirmed by Sanger sequencing of a polymerase chain reaction product amplified using primers designed in the U regions flanking the FLO1 gene. SNP, single-nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1622443995" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of M regions from two classes of repetitive elements. (A) M&lt;sub&gt;U&lt;/sub&gt; regions overlapping the Ty1-Ty5 elements. (B) M&lt;sub&gt;U&lt;/sub&gt; regions overlapping the six &lt;em&gt;FLO&lt;/em&gt; gene copies. The M&lt;sub&gt;U&lt;/sub&gt; regions are represented by horizontal bold bars. (C) Detection of acquired mutations within the polymorphic &lt;em&gt;FLO1&lt;/em&gt;, &lt;em&gt;FLO5&lt;/em&gt;, and &lt;em&gt;FLO9&lt;/em&gt; multi-copy genes. (1) Alignment of the &lt;em&gt;FLO5&lt;/em&gt;, &lt;em&gt;FLO1&lt;/em&gt;, &lt;em&gt;FLO9&lt;/em&gt;, and &lt;em&gt;FLO1&lt;/em&gt; variant (identified here) gene sequences. (2) Visualization of the forward and reverse strand read mapping of the &lt;em&gt;FLO1&lt;/em&gt; gene on the chromosome I in the experimental HTS from the wt_A100 wild type line with the SOLiD Alignment Browser. The coordinates of the regions are indicated (arrows). The fluorescent green bars outline positions of mismatches consistent with a base change. (3) Same as (2) after using g-deNoise. The reads containing the intragenomic SNVs were removed and the unpredicted (acquired) SNP at position 207,394 persisted. The nature and position of the acquired mutation is indicated in green in the &lt;em&gt;FLO1&lt;/em&gt; variant sequence in (1). This identification of this acquired SNP was based on the examination of the 50 nt-reads, containing the adjacent &lt;em&gt;FLO1&lt;/em&gt; specific polymorphisms. It was also confirmed by Sanger sequencing of a polymerase chain reaction product amplified using primers designed in the U regions flanking the &lt;em&gt;FLO1&lt;/em&gt; gene. SNP, single-nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/707/F6.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/4/707/F6.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/707/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/707/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464244" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-33" class="first-child">Analysis of M regions from two classes of repetitive elements. (A) M<sub>U</sub> regions overlapping the Ty1-Ty5 elements. (B) M<sub>U</sub> regions overlapping the six <em>FLO</em> gene copies. The M<sub>U</sub> regions are represented by horizontal bold bars. (C) Detection of acquired mutations within the polymorphic <em>FLO1</em>, <em>FLO5</em>, and <em>FLO9</em> multi-copy genes. (1) Alignment of the <em>FLO5</em>, <em>FLO1</em>, <em>FLO9</em>, and <em>FLO1</em> variant (identified here) gene sequences. (2) Visualization of the forward and reverse strand read mapping of the <em>FLO1</em> gene on the chromosome I in the experimental HTS from the wt_A100 wild type line with the SOLiD Alignment Browser. The coordinates of the regions are indicated (arrows). The fluorescent green bars outline positions of mismatches consistent with a base change. (3) Same as (2) after using g-deNoise. The reads containing the intragenomic SNVs were removed and the unpredicted (acquired) SNP at position 207,394 persisted. The nature and position of the acquired mutation is indicated in green in the <em>FLO1</em> variant sequence in (1). This identification of this acquired SNP was based on the examination of the 50 nt-reads, containing the adjacent <em>FLO1</em> specific polymorphisms. It was also confirmed by Sanger sequencing of a polymerase chain reaction product amplified using primers designed in the U regions flanking the <em>FLO1</em> gene. SNP, single-nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Analyzing 50 nt and six mismatches per read genome-wide, we estimated that 614,530 bp (40% of the total length of the M regions) of the <em>S. cerevisiae</em> genome contain M<sub>U</sub> regions. Overall, we now estimate that the robust analysis of the U+M<sub>U</sub> regions reach 94.96% of the <em>S. cerevisiae</em> genome.</p></div><div id="sec-16" class="subsection"><h3>Application to the detection of SNP variants in the U and M regions</h3><p id="p-35">Sequencing of four wild-type <em>S. cerevisiae</em> mutation accumulation cell lines of the BY4741 strain background, led us to examine the presence of SNPs both within the U and M regions. Using the Bioscope pipeline and the 2008 S288c reference sequence, we first examined the U regions. We identified 408 SNPs that were present in all four lines (constitutive). Consistently, 357 SNPs (87.5%) were corrected in the 2011 S288c reference genome (current release). Soon after, partial sequences from the BY4741 and BY4742 yeast strains, that comprise 14 of the 16 total chromosomes, have been provided on SGD (BY4741_Toronto_2012.fsa and BY4742_Toronto_2012.fsa). Among the 51 SNPs that we identified to differ between BY4741 and S288c, 38 were also present in the BY genome releases. We confirmed that they represent true polymorphisms that distinguish the BY and S288c strain backgrounds. To further assess the remaining 13 SNPs, we performed Sanger sequencing and in each case confirmed our NGS call.</p><p id="p-36">Concerning the acquired variants (present in only one out of the four lines) we identified 39 SNPs in the U regions. In the M regions, we used g-deNoise to filter multi-alignments. We identified two constitutive SNPs in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006809">YBLWTy2-1</a></em> element on chromosome II (C &gt; T, position 30,012 and A &gt; G, position 30,050) and two acquired SNPs, present in only one of the four wild-type lines. Precisely, we identified a C &gt; T at position 207,394 on chromosome I in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000084">FLO1</a></em> gene (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6C</a>) and a G &gt; C at position 156,571 on chromosome X in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003672">RPS21B</a></em> gene encoding the small (40S) ribosomal subunit, in the lines wt_A100 and the wt_B100, respectively. These four SNPs were confirmed by Sanger sequencing of a PCR-product amplified using primers of the U regions flanking the SNPs. Altogether, these results demonstrate the power of using the preannotated HTS profile of the U and M regions to robustly identify sequence variations. The lists of the constitutive and acquired SNPs identified in the present studies of the commonly used BY4741 strain are reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1/TableS3.pdf">Table S3</a>.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-37">Herein, we report novel methods for SNPs calling within unique and repeated genomic regions. These methods rely on two complementary bioinformatics strategies: the production of simulated HTS experiments to generate virtual HTS profiles, allowing us to define U and M regions of the genome, whereas in a second step, the “g-deNoise” filtering is used to discriminate the M<sub>M</sub> and M<sub>U</sub> regions. Concerning the analyses of the experimental data, our alignment filtering consists of (i) discarding alignments consistent with a SNV from intragenomic origins, (ii) retaining the unique alignments, and (iii) safely analyzing the U and M<sub>U</sub> regions. Compared with other filters using “best alignment” strategies as SOAP (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Li <em>et al.</em> 2008b</a>) and PASS (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Campagna <em>et al.</em> 2009</a>) mapping programs that filter experimental data, the advantage of our method is to use noise-less simulated reads for the preannotation of the intragenomic SNVs. Compared with GMS (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Lee and Schatz 2012</a>), that associates a probability to each position of the genome that pinpoints noisy repetitive regions but does not propose an ultimate FP binary status such as “is” or “is not” to each call, our approach is decisive and thereof facilitates the detection of alternative variations at the same position (for example, if there is an A in the reference sequence but a T in a repeated region, we are able to identify C or G changes if an acquired SNP is present in an experimental sample). Other applications could be derived from the establishment of virtual HTS profiles; for example, the filtering of small insertions/deletions and rearrangements, as well as for the identification of copy number changes when combined with the analysis of experimental HTS coverage. It can be implemented for whole genome, exome, and draft sequences. The knowledge of the intragenomic SNVs can also optimize the filtering of low-coverage cases. To follow the specification of the experimental reads in terms of length and quality that are in constant evolution, the parameters of virtual HTS profile determinations can be modulated.</p><p id="p-38">Concerning the analyses of the repetitive regions, the ability to identify the M<sub>U</sub> regions and use their internal polymorphisms, provides a significant progress, but due to the lack of polymorphisms, the M<sub>M</sub> regions remain difficult to analyze. In these regions, the occurrence of acquired polymorphisms can be identified, but their ability to be mapped to a given unique repeat remains a challenge. Our pair-ended libraries that generated two reads separated by ~200 bp, were insufficient to resolve certain multialignments because many repetitive sequences are greater than 200 bp. The usage of mate-paired libraries with larger inserts between the reads or extra-long single molecule read technologies (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Sharon <em>et al.</em> 2013</a>) associated with powerful <em>de novo</em> assembly software will be necessary to be exhaustive.</p><p id="p-39">Our analysis of the <em>S. cerevisiae</em> genome is a resource of general interest available at SGD. This resource will have multiple benefits: (i) to define the U and M regions, (ii) to provide a simple and global U/M readout of the genome complexity that can be used to compare variant strains and species, (iii) to allow precise measurements of HTS coverage by using the set of U regions as the reference genome ploidy index to count experimental reads, and (iv) to facilitate the identification of constitutive and acquired mutations in both unique and highly repetitive regions of variant strains.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We thank Allyson Holmes for her critical reading of the manuscript. This work was funded by the “Agence Nationale pour la Recherche,“ grant “MUTome” #ANR-09-BLAN-0278-01 to A.N. The IC-NGS platform received financial support from the ANR (investissements d’avenir ANR-10-EQPX-03 and ANR-INBS-09-08), the Cancéropôle Ile de France, the Région Ile de France, and France Génomique. C.J. received a doctoral grant from the “Fondation ARC Pour La Recherche Sur Le Cancer“ and from the “Cellule d’enseignement de l’Institut Curie.“</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Sequence data from this article have been deposited with the NCBI Sequence Read Archive (<a href="http://www.ncbi.nlm.nih.gov/Traces/sra">http://www.ncbi.nlm.nih.gov/Traces/sra</a>) under accession nos. SRR1014750, SRR1014753, SRR1014752, and SRR1014754.</p></li><li class="fn" id="fn-2"><p id="p-2">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009464/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-41"><em>Communicating editor: C. Boone</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-11-18"><span class="received-label">Received </span>November 18, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-02-15"><span class="accepted-label">Accepted </span>February 15, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Jubin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.1" data-doi="10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachmann</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cost</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Designer deletion strains derived from <em>Saccharomyces cerevisiae</em> S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBrachmann%26rft.auinit1%253DC.%2BB.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D132%26rft.atitle%253DDesigner%2Bdeletion%2Bstrains%2Bderived%2Bfrom%2BSaccharomyces%2Bcerevisiae%2BS288C%253A%2Ba%2Buseful%2Bset%2Bof%2Bstrains%2Band%2Bplasmids%2Bfor%2BPCR-mediated%2Bgene%2Bdisruption%2Band%2Bother%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%252819980130%252914%253A2%253C115%253A%253AAID-YEA204%253E3.0.CO%253B2-2%26rft_id%253Dinfo%253Apmid%252F9483801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9483801&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071817800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-4.4.707.2"><div class="cit-metadata unstructured">Breu, H., 2008 A theoretical understanding of 2 base color codes and its application to annotation, error detection, and error correction: methods for annotating 2 base color encoded reads in the SOLiD™ system. Applied Biosystems White paper. Available at: <a href="https://www3.appliedbiosystems.com/cms/groups/mcb_marketing/documents/generaldocuments/cms_058265.pdf">https://www3.appliedbiosystems.com/cms/groups/mcb_marketing/documents/generaldocuments/cms_058265.pdf</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.3" data-doi="10.1093/bioinformatics/btp087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campagna</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albiero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilardi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caniato</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcato</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">PASS: a program to align short sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp087%26rft_id%253Dinfo%253Apmid%252F19218350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNS83Lzk2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.4" data-doi="10.1038/nmeth.1974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng-Bradley</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulesha</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The 1000 Genomes Project: data management and community access.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DClarke%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D459%26rft.epage%253D462%26rft.atitle%253DThe%2B1000%2BGenomes%2BProject%253A%2Bdata%2Bmanagement%2Band%2Bcommunity%2Baccess.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1974%26rft_id%253Dinfo%253Apmid%252F22543379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543379&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.5" data-doi="10.1371/journal.pone.0007767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Homer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merriman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BFAST: an alignment tool for large scale genome resequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e7767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHomer%26rft.auinit1%253DN.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De7767%26rft.epage%253De7767%26rft.atitle%253DBFAST%253A%2Ban%2Balignment%2Btool%2Bfor%2Blarge%2Bscale%2Bgenome%2Bresequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007767%26rft_id%253Dinfo%253Apmid%252F19907642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19907642&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.6" data-doi="10.1038/nature02424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Proof and evolutionary analysis of ancient genome duplication in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D428%26rft.issue%253D6983%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DProof%2Band%2Bevolutionary%2Banalysis%2Bof%2Bancient%2Bgenome%2Bduplication%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02424%26rft_id%253Dinfo%253Apmid%252F15004568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15004568&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220697200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.7" data-doi="10.1093/bioinformatics/btq640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koehler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Issac</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cloonan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimmond</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The uniqueome: a mappability resource for short-tag sequencing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq640%26rft_id%253Dinfo%253Apmid%252F21075741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy8yLzI3MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.8" data-doi="10.1093/bioinformatics/bts330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic dark matter: the reliability of short read mapping illustrated by the genome mappability score.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2097</span>–<span class="cit-lpage">2105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts330%26rft_id%253Dinfo%253Apmid%252F22668792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTYvMjA5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.9" data-doi="10.1016/j.cell.2004.12.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemoine</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degtyareva</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobachev</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Chromosomal translocations in yeast induced by low levels of DNA polymerase a model for chromosome fragile sites.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLemoine%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D120%26rft.issue%253D5%26rft.spage%253D587%26rft.epage%253D598%26rft.atitle%253DChromosomal%2Btranslocations%2Bin%2Byeast%2Binduced%2Bby%2Blow%2Blevels%2Bof%2BDNA%2Bpolymerase%2Ba%2Bmodel%2Bfor%2Bchromosome%2Bfragile%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2004.12.039%26rft_id%253Dinfo%253Apmid%252F15766523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2004.12.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15766523&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227771400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.10" data-doi="10.1101/gr.078212.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span> <span class="cit-article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1851</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078212.108%26rft_id%253Dinfo%253Apmid%252F18714091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xODUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.11" data-doi="10.1093/bioinformatics/btn025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristiansen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span> <span class="cit-article-title">SOAP: short oligonucleotide alignment program.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn025%26rft_id%253Dinfo%253Apmid%252F18227114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNC81LzcxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mewes</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albermann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frishman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleissner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Overview of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMewes%26rft.auinit1%253DH.%2BW.%26rft.volume%253D387%26rft.issue%253D6632%2BSuppl%26rft.spage%253D7%26rft.epage%253D65%26rft.atitle%253DOverview%2Bof%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F9169865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9169865&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.13" data-doi="10.1073/pnas.1314423111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jubin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeillet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legoix-Ne</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mutational landscape of yeast mutator strains.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1897</span>–<span class="cit-lpage">1902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1314423111%26rft_id%253Dinfo%253Apmid%252F24449905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTExLzUvMTg5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.14" data-doi="10.1038/nbt.2705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tilgner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A single-molecule long-read survey of the human transcriptome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D1009%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2705%26rft_id%253Dinfo%253Apmid%252F24108091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24108091&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.15" data-doi="10.1038/nrg3117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repetitive DNA and next-generation sequencing: computational challenges and solutions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DTreangen%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D46%26rft.atitle%253DRepetitive%2BDNA%2Band%2Bnext-generation%2Bsequencing%253A%2Bcomputational%2Bchallenges%2Band%2Bsolutions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3117%26rft_id%253Dinfo%253Apmid%252F22124482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22124482&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.707.16" data-doi="10.1038/42711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Molecular evidence for an ancient duplication of the entire yeast genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D387%26rft.issue%253D6634%26rft.spage%253D708%26rft.epage%253D713%26rft.atitle%253DMolecular%2Bevidence%2Bfor%2Ban%2Bancient%2Bduplication%2Bof%2Bthe%2Bentire%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F42711%26rft_id%253Dinfo%253Apmid%252F9192896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/42711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9192896&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XD86900055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/4/707.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/693" title="unfulfilled Interacting Genes Display Branch-Specific Roles in the Development of Mushroom Body Axons in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/717" title="Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent Haemophilus influenzae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/4"><span>Volume 4 Issue 4, April 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/4.cover-source.jpg?itok=Rf0YhETZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/707" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-P7cTmmmbZpAXeSptmwznWcL51eqDXYzC3BFFBn_aKK0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/4/707" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/4/707" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/4/707" class="active" data-icon-position="" data-hide-link-title="0">Sequence Profiling of the Saccharomyces cerevisiae Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459821" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-YmWtubgl3-TNf1A4fSZJ7Z18YntBbz_pSrLhQzF-rWs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/707.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/4/707" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OiXdZsQrkQTFWjLYaYq9AXjg52UMGJRCua7AykCX6c0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/4/707" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/707.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459821" id="node459821--2" data-pisa="ggg;4/4/707" data-pisa-master="ggg;g3.113.009464" data-apath="/ggg/4/4/707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequence Profiling of the <em>Saccharomyces cerevisiae</em> Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Claire Jubin</span>, <span class="highwire-citation-author" data-delta="1">Alexandre Serero</span>, <span class="highwire-citation-author" data-delta="2">Sophie Loeillet</span>, <span class="highwire-citation-author" data-delta="3">Emmanuel Barillot</span> and <span class="highwire-citation-author" data-delta="4">Alain Nicolas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">707-715;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009464 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459821--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Claire Jubin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BJubin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jubin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BJubin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandre Serero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BSerero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Serero%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BSerero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sophie Loeillet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BLoeillet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loeillet%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BLoeillet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emmanuel Barillot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut Curie Centre de Recherche, INSERM U900, 75248 Paris, France. Ecole des Mines Paris Tech, 77300 Fontainebleau, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmanuel%2BBarillot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barillot%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmmanuel%2BBarillot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alain Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alain.nicolas@curie.fr" class="" data-icon-position="" data-hide-link-title="0">alain.nicolas@curie.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/4/707" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/4/707&amp;title=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/4/707&amp;title=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/4/707&amp;title=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/4/707&amp;text=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/4/707&amp;title=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/4/707&amp;t=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/4/707&amp;title=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/4/707&amp;title=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459821/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459821" id="node459821--3" data-pisa="ggg;4/4/707" data-pisa-master="ggg;g3.113.009464" data-apath="/ggg/4/4/707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequence Profiling of the <em>Saccharomyces cerevisiae</em> Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Claire Jubin</span>, <span class="highwire-citation-author" data-delta="1">Alexandre Serero</span>, <span class="highwire-citation-author" data-delta="2">Sophie Loeillet</span>, <span class="highwire-citation-author" data-delta="3">Emmanuel Barillot</span> and <span class="highwire-citation-author" data-delta="4">Alain Nicolas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">707-715;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009464 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459821--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Claire Jubin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BJubin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jubin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BJubin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandre Serero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BSerero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Serero%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BSerero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sophie Loeillet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BLoeillet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loeillet%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BLoeillet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emmanuel Barillot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut Curie Centre de Recherche, INSERM U900, 75248 Paris, France. Ecole des Mines Paris Tech, 77300 Fontainebleau, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmanuel%2BBarillot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barillot%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmmanuel%2BBarillot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alain Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut Curie Centre de Recherche, UMR3244 CNRS, Université Pierre et Marie Curie, 75248 Paris Cedex 05</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alain%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlain%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alain.nicolas@curie.fr" class="" data-icon-position="" data-hide-link-title="0">alain.nicolas@curie.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=707&amp;rft.epage=715&amp;rft.atitle=Sequence%20Profiling%20of%20the%20Saccharomyces%20cerevisiae%20Genome%20Permits%20Deconvolution%20of%20Unique%20and%20Multialigned%20Reads%20for%20Variant%20Detection&amp;rft.volume=4&amp;rft.issue=4&amp;rft.date=2014-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jubin%2C+Claire&amp;rft.au=Serero%2C+Alexandre&amp;rft.au=Loeillet%2C+Sophie&amp;rft.au=Barillot%2C+Emmanuel&amp;rft.au=Nicolas%2C+Alain"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459821/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459821/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459821/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459821/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459821/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459821/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459821/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459821/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459821/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459821/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459821/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459821/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/4/707" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/4/707.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/4/707.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/4/707.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/4/707.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
