<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/4/717" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/4/717.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/4/717.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/4/717.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/4/717.atom" />
<meta name="HW.pisa" content="ggg;4/4/717" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent Haemophilus influenzae" />
<meta name="DC.Identifier" content="10.1534/g3.113.009597" />
<meta name="DC.Date" content="2014-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Mell et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Naturally competent bacterial species actively take up environmental DNA and can incorporate it into their chromosomes by homologous recombination. This can bring genetic variation from environmental DNA to recipient chromosomes, often in multiple long “donor” segments. Here, we report the results of genome sequencing 96 colonies of a laboratory Haemophilus influenzae strain, which had been experimentally transformed by DNA from a diverged clinical isolate. Donor segments averaged 6.9 kb (spanning several genes) and were clustered into recombination tracts of ~19.5 kb. Individual colonies had replaced from 0.1 to 3.2% of their chromosomes, and ~1/3 of all donor-specific single-nucleotide variants were present in at least one recombinant. We found that nucleotide divergence did not obviously limit the locations of recombination tracts, although there were small but significant reductions in divergence at recombination breakpoints. Although indels occasionally transformed as parts of longer recombination tracts, they were common at breakpoints, suggesting that indels typically block progression of strand exchange. Some colonies had recombination tracts in which variant positions contained mixtures of both donor and recipient alleles. These tracts were clustered around the origin of replication and were interpreted as the result of heteroduplex segregation in the original transformed cell. Finally, a pilot experiment demonstrated the utility of natural transformation for genetically dissecting natural phenotypic variation. We discuss our results in the context of the potential to merge experimental and population genetic approaches, giving a more holistic understanding of bacterial gene transfer." />
<meta name="DC.Contributor" content="Joshua Chang Mell" />
<meta name="DC.Contributor" content="Jae Yun Lee" />
<meta name="DC.Contributor" content="Marlo Firme" />
<meta name="DC.Contributor" content="Sunita Sinha" />
<meta name="DC.Contributor" content="Rosemary J. Redfield" />
<meta name="article:published_time" content="2014-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent Haemophilus influenzae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Naturally competent bacterial species actively take up environmental DNA and can incorporate it into their chromosomes by homologous recombination. This can bring genetic variation from environmental DNA to recipient chromosomes, often in multiple long “donor” segments. Here, we report the results of genome sequencing 96 colonies of a laboratory &lt;i&gt;Haemophilus influenzae&lt;/i&gt; strain, which had been experimentally transformed by DNA from a diverged clinical isolate. Donor segments averaged 6.9 kb (spanning several genes) and were clustered into recombination tracts of ~19.5 kb. Individual colonies had replaced from 0.1 to 3.2% of their chromosomes, and ~1/3 of all donor-specific single-nucleotide variants were present in at least one recombinant. We found that nucleotide divergence did not obviously limit the locations of recombination tracts, although there were small but significant reductions in divergence at recombination breakpoints. Although indels occasionally transformed as parts of longer recombination tracts, they were common at breakpoints, suggesting that indels typically block progression of strand exchange. Some colonies had recombination tracts in which variant positions contained mixtures of both donor and recipient alleles. These tracts were clustered around the origin of replication and were interpreted as the result of heteroduplex segregation in the original transformed cell. Finally, a pilot experiment demonstrated the utility of natural transformation for genetically dissecting natural phenotypic variation. We discuss our results in the context of the potential to merge experimental and population genetic approaches, giving a more holistic understanding of bacterial gene transfer.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/04/01" />
<meta name="citation_mjid" content="ggg;4/4/717" />
<meta name="citation_id" content="4/4/717" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/4/717" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/4/717.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/4/717.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/4/717.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.009597" />
<meta name="citation_pmid" content="24569039" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="717" />
<meta name="citation_lastpage" content="731" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Joshua Chang Mell" />
<meta name="citation_author_institution" content="Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z3, Canada" />
<meta name="citation_author_email" content="joshua.mell@DrexelMed.edu" />
<meta name="citation_author" content="Jae Yun Lee" />
<meta name="citation_author_institution" content="Genome Sciences and Technology Graduate Program, University of British Columbia, Vancouver, BC V6T 1Z3, Canada" />
<meta name="citation_author" content="Marlo Firme" />
<meta name="citation_author_institution" content="Genome Sciences and Technology Graduate Program, University of British Columbia, Vancouver, BC V6T 1Z3, Canada" />
<meta name="citation_author" content="Sunita Sinha" />
<meta name="citation_author_institution" content="Department of Pharmacy Sciences, University of British Columbia, Vancouver, BC V6T 1Z3, Canada" />
<meta name="citation_author" content="Rosemary J. Redfield" />
<meta name="citation_author_institution" content="Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z3, Canada" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.;citation_author=A. Agrawal;citation_author=T. F. Murphy;citation_title=Haemophilus influenzae infections in the H. influenzae type b conjugate vaccine era.;citation_volume=49;citation_year=2011;citation_pmid=21900515;citation_doi=10.1128/JCM.05476-11" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=B. Bartolome;citation_author=Y. Jubete;citation_author=E. Martinez;citation_author=F. de la Cruz;citation_title=Construction and properties of a family of pACYC184-derived cloning vectors compatible with pBR322 and its derivatives.;citation_pages=75-78;citation_volume=102;citation_year=1991;citation_issue=1;citation_pmid=1840539;citation_doi=10.1016/0378-1119(91)90541-I" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=B. W. Catlin;citation_author=J. W. Bendler;citation_author=S. H. Goodgal;citation_title=The Type b Capsulation Locus of Haemophilus influenzae: Map Location and Size;citation_pages=411-422;citation_volume=70;citation_year=1972;citation_issue=3;citation_pmid=4402481;citation_doi=10.1099/00221287-70-3-411" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=I. Chen;citation_author=D. Dubnau;citation_title=DNA uptake during bacterial transformation.;citation_pages=241-249;citation_volume=2;citation_year=2004;citation_issue=3;citation_pmid=15083159;citation_doi=10.1038/nrmicro844" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=J. P. Claverys;citation_author=B. Martin;citation_author=P. Polard;citation_title=The genetic transformation machinery: composition, localization, and mechanism.;citation_volume=33;citation_year=2009;citation_pmid=19228200;citation_doi=10.1111/j.1574-6976.2009.00164.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. J. Croucher;citation_author=S. R. Harris;citation_author=C. Fraser;citation_author=M. A. Quail;citation_author=J. Burton;citation_title=Rapid Pneumococcal Evolution in Response to Clinical Interventions;citation_pages=430-434;citation_volume=331;citation_year=2011;citation_issue=6016;citation_pmid=21273480;citation_doi=10.1126/science.1198545" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=N. J. Croucher;citation_author=S. R. Harris;citation_author=L. Barquist;citation_author=J. Parkhill;citation_author=S. D. Bentley;citation_title=A high-resolution view of genome-wide pneumococcal transformation.;citation_pages=e1002745-e1002745;citation_volume=8;citation_year=2012;citation_issue=6;citation_pmid=22719250;citation_doi=10.1371/journal.ppat.1002745" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=X. Didelot;citation_author=M. C. Maiden;citation_title=Impact of recombination on bacterial evolution.;citation_pages=315-322;citation_volume=18;citation_year=2010;citation_issue=7;citation_pmid=20452218;citation_doi=10.1016/j.tim.2010.04.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Didelot;citation_author=D. Lawson;citation_author=A. Darling;citation_author=D. Falush;citation_title=Inference of homologous recombination in bacteria using whole-genome sequences.;citation_volume=186;citation_year=2010;citation_pmid=20923983;citation_doi=10.1534/genetics.110.120121" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=M. S. Dworkin;citation_author=L. Park;citation_author=S. M. Borchardt;citation_title=The changing epidemiology of invasive Haemophilus influenzae disease, especially in persons &gt; or = 65 years old.;citation_volume=44;citation_year=2007;citation_pmid=17304452;citation_doi=10.1086/511861" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. J. Erickson;citation_author=J. C. Copeland;citation_title=CONGRESSION OF UNLINKED MARKERS AND GENETIC MAPPING IN THE TRANSFORMATION OF BACILLUS SUBTILIS 168;citation_pages=13-21;citation_volume=73;citation_year=1973;citation_issue=1;citation_pmid=4631599" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=E. J. Feil;citation_author=B. G. Spratt;citation_title=RECOMBINATION AND THE POPULATION STRUCTURES OF BACTERIAL PATHOGENS;citation_pages=561-590;citation_volume=55;citation_year=2001;citation_issue=1;citation_pmid=11544367;citation_doi=10.1146/annurev.micro.55.1.561" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. J. Feil;citation_author=E. C. Holmes;citation_author=D. E. Bessen;citation_author=M. S. Chan;citation_author=N. P. Day;citation_title=Recombination within natural populations of pathogenic bacteria: Short-term empirical estimates and long-term phylogenetic consequences;citation_pages=182-187;citation_volume=98;citation_year=2001;citation_issue=1;citation_pmid=11136255;citation_doi=10.1073/pnas.98.1.182" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. D. Fleischmann;citation_author=M. D. Adams;citation_author=O. White;citation_author=R. A. Clayton;citation_author=E. F. Kirkness;citation_title=Whole-genome random sequencing and assembly of Haemophilus influenzae Rd;citation_pages=496-512;citation_volume=269;citation_year=1995;citation_issue=5223;citation_pmid=7542800;citation_doi=10.1126/science.7542800" />
<meta name="citation_reference" content="citation_journal_title=The Journal of General Physiology;citation_journal_abbrev=JGP;citation_author=S. H. Goodgal;citation_author=R. M. Herriott;citation_title=Studies on Transformations of Hemophilus influenzae: I. Competence;citation_pages=1201-1227;citation_volume=44;citation_year=1961;citation_issue=6;citation_pmid=13707010;citation_doi=10.1085/jgp.44.6.1201" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=T. A. Gray;citation_author=J. A. Krywy;citation_author=J. Harold;citation_author=M. J. Palumbo;citation_author=K. M. Derbyshire;citation_title=Distributive conjugal transfer in mycobacteria generates progeny with meiotic-like genome-wide mosaicism, allowing mapping of a mating identity locus.;citation_pages=e1001602;citation_volume=11;citation_year=2013;citation_pmid=23874149;citation_doi=10.1371/journal.pbio.1001602" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. C. Gupta;citation_author=E. I. Golub;citation_author=M. S. Wold;citation_author=C. M. Radding;citation_title=Polarity of DNA strand exchange promoted by recombination proteins of the RecA family;citation_pages=9843-9848;citation_volume=95;citation_year=1998;citation_issue=17;citation_pmid=9707563;citation_doi=10.1073/pnas.95.17.9843" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Guy;citation_author=J. R. Kultima;citation_author=S. G. Andersson;citation_title=genoPlotR: comparative gene and genome visualization in R.;citation_volume=26;citation_year=2010;citation_pmid=20624783;citation_doi=10.1093/bioinformatics/btq413" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. L. Gwinn;citation_author=R. Ramanathan;citation_author=H. O. Smith;citation_author=J. F. Tomb;citation_title=A New Transformation-Deficient Mutant of Haemophilus influenzae Rd with Normal DNA Uptake;citation_pages=746-748;citation_volume=180;citation_year=1998;citation_issue=3;citation_pmid=9457884" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. P. Hanage;citation_author=C. Fraser;citation_author=J. Tang;citation_author=T. R. Connor;citation_author=J. Corander;citation_title=Hyper-Recombination, Diversity, and Antibiotic Resistance in Pneumococcus;citation_pages=1454-1457;citation_volume=324;citation_year=2009;citation_issue=5933;citation_pmid=19520963;citation_doi=10.1126/science.1171908" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=A. Harrison;citation_author=D. W. Dyer;citation_author=A. Gillaspy;citation_author=W. C. Ray;citation_author=R. Mungur;citation_title=Genomic Sequence of an Otitis Media Isolate of Nontypeable Haemophilus influenzae: Comparative Study with H. influenzae Serotype d, Strain KW20;citation_pages=4627-4636;citation_volume=187;citation_year=2005;citation_issue=13;citation_pmid=15968074;citation_doi=10.1128/JB.187.13.4627-4636.2005" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. S. Hogg;citation_author=F. Z. Hu;citation_author=B. Janto;citation_author=R. Boissy;citation_author=J. Hayes;citation_title=Characterization and modeling of the Haemophilus influenzae core and supragenomes based on the complete genomic sequences of Rd and 12 clinical nontypeable strains.;citation_pages=R103-R103;citation_volume=8;citation_year=2007;citation_issue=6;citation_pmid=17550610;citation_doi=10.1186/gb-2007-8-6-r103" />
<meta name="citation_reference" content="citation_journal_title=Research in Microbiology (Paris);citation_journal_abbrev=Research in Microbiology (Paris);citation_author=O. Johnsborg;citation_author=V. Eldholm;citation_author=L. S. Havarstein;citation_title=Natural genetic transformation: prevalence, mechanisms and function.;citation_pages=767-778;citation_volume=158;citation_year=2007;citation_issue=10;citation_pmid=17997281;citation_doi=10.1016/j.resmic.2007.09.004" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=S. C. Kowalczykowski;citation_author=A. K. Eggleston;citation_title=Homologous Pairing and DNA Strand Exchange Proteins;citation_pages=991-1043;citation_volume=63;citation_year=1994;citation_issue=1;citation_pmid=7979259;citation_doi=10.1146/annurev.bi.63.070194.005015" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=N. J. Kruger;citation_author=K. Stingl;citation_title=Two steps away from novelty--principles of bacterial DNA uptake.;citation_pages=860-867;citation_volume=80;citation_year=2011;citation_issue=4;citation_pmid=21435041;citation_doi=10.1111/j.1365-2958.2011.07647.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=S. Lacks;citation_author=M. Neuberger;citation_title=Membrane location of a deoxyribonuclease implicated in the genetic transformation of Diplococcus pneumoniae.;citation_pages=1321-1329;citation_volume=124;citation_year=1975;citation_issue=3;citation_pmid=366" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. G. Lawrence;citation_title=Gene transfer in bacteria: speciation without species?;citation_pages=449-460;citation_volume=61;citation_year=2002;citation_issue=4;citation_pmid=12167364;citation_doi=10.1006/tpbi.2002.1587" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long read alignment with Burrows-Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=E. A. Lin;citation_author=X. S. Zhang;citation_author=S. M. Levine;citation_author=S. R. Gill;citation_author=D. Falush;citation_title=Natural transformation of helicobacter pylori involves the integration of short DNA fragments interrupted by gaps of variable size.;citation_pages=e1000337-e1000337;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19282979;citation_doi=10.1371/journal.ppat.1000337" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. Maamar;citation_author=A. Raj;citation_author=D. Dubnau;citation_title=Noise in gene expression determines cell fate in Bacillus subtilis.;citation_volume=317;citation_year=2007;citation_pmid=17569828;citation_doi=10.1126/science.1140818" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=J. R. MacNeil;citation_author=A. C. Cohn;citation_author=M. Farley;citation_author=R. Mair;citation_author=J. Baumbach;citation_title=Current epidemiology and trends in invasive Haemophilus influenzae disease–United States, 1989–2008.;citation_volume=53;citation_year=2011;citation_pmid=22080119;citation_doi=10.1093/cid/cir735" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_title=Extensive variation in natural competence in Haemophilus Influenzae.;citation_author=H. Maughan;citation_author=R. J. Redfield;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Maughan;citation_author=R. J. Redfield;citation_title=Tracing the evolution of competence in Haemophilus influenzae.;citation_pages=e5854-e5854;citation_volume=4;citation_year=2009;citation_issue=6;citation_pmid=19516897;citation_doi=10.1371/journal.pone.0005854" />
<meta name="citation_reference" content="citation_pages=79-98" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=J. C. Mell;citation_author=S. Shumilina;citation_author=I. M. Hall;citation_author=R. J. Redfield;citation_title=Transformation of natural genetic variation into Haemophilus influenzae genomes.;citation_pages=e1002151-e1002151;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21829353;citation_doi=10.1371/journal.ppat.1002151" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=D. Moradigaravand;citation_author=J. Engelstadter;citation_title=The evolution of natural competence: disentangling costs and benefits of sex in bacteria.;citation_pages=E112-E126;citation_volume=182;citation_year=2013;citation_pmid=24021408;citation_doi=10.1086/671909" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=P. Morey;citation_author=V. Cano;citation_author=P. Marti-Lliteras;citation_author=A. Lopez-Gomez;citation_author=V. Regueiro;citation_title=Evidence for a non-replicative intracellular stage of nontypable Haemophilus influenzae in epithelial cells.;citation_volume=157;citation_year=2011;citation_pmid=20929955;citation_doi=10.1099/mic.0.040451-0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=I. Mortier-Barriere;citation_author=M. Velten;citation_author=P. Dupaigne;citation_author=N. Mirouze;citation_author=O. Pietrement;citation_title=A key presynaptic role in transformation for a widespread bacterial protein: DprA conveys incoming ssDNA to RecA.;citation_pages=824-836;citation_volume=130;citation_year=2007;citation_issue=5;citation_pmid=17803906;citation_doi=10.1016/j.cell.2007.07.038" />
<meta name="citation_reference" content="citation_journal_title=Infection, genetics and evolution :  journal of molecular epidemiology and evolutionary genetics in infectious diseases;citation_journal_abbrev=Infect Genet Evol;citation_author=M. Perez-Losada;citation_author=E. B. Browne;citation_author=A. Madsen;citation_author=T. Wirth;citation_author=R. P. Viscidi;citation_title=Population genetics of microbial pathogens estimated from multilocus sequence typing (MLST) data.;citation_pages=97-112;citation_volume=6;citation_year=2006;citation_issue=2;citation_pmid=16503511;citation_doi=10.1016/j.meegid.2005.02.003" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. L. Pifer;citation_author=H. O. Smith;citation_title=Processing of donor DNA during Haemophilus influenzae transformation: analysis using a model plasmid system.;citation_pages=3731-3735;citation_volume=82;citation_year=1985;citation_issue=11;citation_pmid=2987941;citation_doi=10.1073/pnas.82.11.3731" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular medicine;citation_journal_abbrev=Methods Mol Med;citation_author=G. Poje;citation_author=R. J. Redfield;citation_title=General methods for culturing Haemophilus influenzae.;citation_pages=51-56;citation_volume=71;citation_year=2003;citation_pmid=12374030" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular medicine;citation_journal_abbrev=Methods Mol Med;citation_author=G. Poje;citation_author=R. J. Redfield;citation_title=Transformation of Haemophilus influenzae.;citation_pages=57-70;citation_volume=71;citation_year=2003;citation_pmid=12374031;citation_doi=10.1385/1-59259-321-6:57" />
<meta name="citation_reference" content="citation_journal_title=BMC Microbiology;citation_journal_abbrev=BMC Microbiology;citation_author=P. M. Power;citation_author=S. D. Bentley;citation_author=J. Parkhill;citation_author=E. R. Moxon;citation_author=D. W. Hood;citation_title=Investigations into genome diversity of Haemophilus influenzae using whole genome sequencing of clinical isolates and laboratory transformants.;citation_pages=273-273;citation_volume=12;citation_year=2012;citation_issue=1;citation_pmid=23176117;citation_doi=10.1186/1471-2180-12-273" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. L. Ray;citation_author=K. Harms;citation_author=O. G. Wikmark;citation_author=I. Starikova;citation_author=P. J. Johnsen;citation_title=Sexual isolation in Acinetobacter baylyi is locus-specific and varies 10,000-fold over the genome.;citation_volume=182;citation_year=2009;citation_pmid=19474200;citation_doi=10.1534/genetics.109.103127" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. J. Redfield;citation_title=Do bacteria have sex?;citation_pages=634-639;citation_volume=2;citation_year=2001;citation_issue=8;citation_pmid=11483988;citation_doi=10.1038/35084593" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=R. J. Redfield;citation_author=A. D. Cameron;citation_author=Q. Qian;citation_author=J. Hinds;citation_author=T. R. Ali;citation_title=A novel CRP-dependent regulon controls expression of competence genes in Haemophilus influenzae.;citation_pages=735-747;citation_volume=347;citation_year=2005;citation_issue=4;citation_pmid=15769466;citation_doi=10.1016/j.jmb.2005.01.012" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=S. L. Salzberg;citation_author=A. J. Salzberg;citation_author=A. R. Kerlavage;citation_author=J. F. Tomb;citation_title=Skewed oligomers and origins of replication.;citation_pages=57-67;citation_volume=217;citation_year=1998;citation_issue=1-2;citation_pmid=9795135;citation_doi=10.1016/S0378-1119(98)00374-6" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=S. Sinha;citation_author=J. C. Mell;citation_author=R. J. Redfield;citation_title=Seventeen Sxy-dependent cyclic AMP receptor protein site-regulated genes are needed for natural transformation in Haemophilus influenzae.;citation_volume=194;citation_year=2012;citation_pmid=22821979;citation_doi=10.1128/JB.00671-12" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=W. L. Steinhart;citation_author=R. M. Herriott;citation_title=Genetic Integration in the Heterospecific Transformation of Haemophilus influenzae Cells by Haemophilus parainfluenzae Deoxyribonucleic Acid;citation_pages=1725-1731;citation_volume=96;citation_year=1968;citation_issue=5;citation_pmid=4882021" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. H. Stuy;citation_author=R. B. Walter;citation_title=Addition, deletion, and substitution of long nonhomologous deoxyribonucleic acid segments by genetic transformation of Haemophilus influenzae.;citation_pages=565-571;citation_volume=148;citation_year=1981;citation_issue=2;citation_pmid=6975273" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=C. M. Thomas;citation_author=K. M. Nielsen;citation_title=Mechanisms of, and barriers to, horizontal gene transfer between bacteria.;citation_pages=711-721;citation_volume=3;citation_year=2005;citation_issue=9;citation_pmid=16138099;citation_doi=10.1038/nrmicro1234" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdottir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Clinical Microbiology Reviews;citation_journal_abbrev=Clin. Microbiol. Rev.;citation_author=S. Tristram;citation_author=M. R. Jacobs;citation_author=P. C. Appelbaum;citation_title=Antimicrobial Resistance in Haemophilus influenzae;citation_pages=368-389;citation_volume=20;citation_year=2007;citation_issue=2;citation_pmid=17428889;citation_doi=10.1128/CMR.00040-06" />
<meta name="citation_reference" content="citation_journal_abbrev=ISME J;citation_author=M. Vos;citation_author=X. Didelot;citation_title=A comparison of homologous recombination rates in bacteria and archaea.;citation_pages=199-208;citation_volume=3;citation_year=2009;citation_issue=2;citation_pmid=18830278;citation_doi=10.1038/ismej.2008.93" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=K. W. Wilcox;citation_author=H. O. Smith;citation_title=Isolation and characterization of mutants of Haemophilus influenzae deficient in an adenosine 5'-triphosphate-dependent deoxyribonuclease activity.;citation_pages=443-453;citation_volume=122;citation_year=1975;citation_issue=2;citation_pmid=165169" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=B. J. Williams;citation_author=G. Morlin;citation_author=N. Valentine;citation_author=A. L. Smith;citation_title=Serum Resistance in an Invasive, Nontypeable Haemophilus influenzae Strain;citation_pages=695-705;citation_volume=69;citation_year=2001;citation_issue=2;citation_pmid=11159957;citation_doi=10.1128/IAI.69.2.695-705.2001" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=D. J. Wilson;citation_title=Insights from genomics into bacterial pathogen populations.;citation_pages=e1002874-e1002874;citation_volume=8;citation_year=2012;citation_issue=9;citation_pmid=22969423;citation_doi=10.1371/journal.ppat.1002874" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent Haemophilus influenzae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="twitter:description" content="Naturally competent bacterial species actively take up environmental DNA and can incorporate it into their chromosomes by homologous recombination. This can bring genetic variation from environmental DNA to recipient chromosomes, often in multiple long “donor” segments. Here, we report the results of genome sequencing 96 colonies of a laboratory Haemophilus influenzae strain, which had been experimentally transformed by DNA from a diverged clinical isolate. Donor segments averaged 6.9 kb (spanning several genes) and were clustered into recombination tracts of ~19.5 kb. Individual colonies had replaced from 0.1 to 3.2% of their chromosomes, and ~1/3 of all donor-specific single-nucleotide variants were present in at least one recombinant. We found that nucleotide divergence did not obviously limit the locations of recombination tracts, although there were small but significant reductions in divergence at recombination breakpoints. Although indels occasionally transformed as parts of longer recombination tracts, they were common at breakpoints, suggesting that indels typically block progression of strand exchange. Some colonies had recombination tracts in which variant positions contained mixtures of both donor and recipient alleles. These tracts were clustered around the origin of replication and were interpreted as the result of heteroduplex segregation in the original transformed cell. Finally, a pilot experiment demonstrated the utility of natural transformation for genetically dissecting natural phenotypic variation. We discuss our results in the context of the potential to merge experimental and population genetic approaches, giving a more holistic understanding of bacterial gene transfer." />
<meta name="og-title" property="og:title" content="Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent Haemophilus influenzae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/4/717" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Naturally competent bacterial species actively take up environmental DNA and can incorporate it into their chromosomes by homologous recombination. This can bring genetic variation from environmental DNA to recipient chromosomes, often in multiple long “donor” segments. Here, we report the results of genome sequencing 96 colonies of a laboratory Haemophilus influenzae strain, which had been experimentally transformed by DNA from a diverged clinical isolate. Donor segments averaged 6.9 kb (spanning several genes) and were clustered into recombination tracts of ~19.5 kb. Individual colonies had replaced from 0.1 to 3.2% of their chromosomes, and ~1/3 of all donor-specific single-nucleotide variants were present in at least one recombinant. We found that nucleotide divergence did not obviously limit the locations of recombination tracts, although there were small but significant reductions in divergence at recombination breakpoints. Although indels occasionally transformed as parts of longer recombination tracts, they were common at breakpoints, suggesting that indels typically block progression of strand exchange. Some colonies had recombination tracts in which variant positions contained mixtures of both donor and recipient alleles. These tracts were clustered around the origin of replication and were interpreted as the result of heteroduplex segregation in the original transformed cell. Finally, a pilot experiment demonstrated the utility of natural transformation for genetically dissecting natural phenotypic variation. We discuss our results in the context of the potential to merge experimental and population genetic approaches, giving a more holistic understanding of bacterial gene transfer." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
    <title>Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent Haemophilus influenzae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Gk0Ot8QFvCCcHwadO6hWsxjtIYyADflWdrPAkRMfS94","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459828","apath":"\/ggg\/4\/4\/717.atom","pisa":"ggg;4\/4\/717","ac":{"\/ggg\/4\/4\/717.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/4\/717.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/4\/717"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/4\/717"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/4\/717":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pekZgZEpQS0l4SWsHlGYBlvpMgSY32QSSE1fZZgCRifnDazc4fJ7CHuDpiTjfzQ6qYJzrX0zqPzYONR187yaor2lBtvIl04lMDwjOIKHiBl5Pkktp77JEYxq6zbYVSWLbx0XDHB2\\\\\\\/YCsjgtTTVJDf02RJpzbI8F5USG3V\\\\\\\/o8HZqfkHs0FnLZl42RLeiZJjIP6fY+c4nX7VaTJ7djEOM0ZlIC3cyKHpi1\\\\\\\/WnTYR4S19Hjz\\\\\\\/RbjlefspyxS0mqvGozEbtFp34dFrn4S+jnG9Mv3y46SEMTjuFMqRs1BolPO6E4K1bSHNo3R8IMD0GRaqUlSAZx2GPEGwYTS1LMw4cl4+L\\\\\\\/mLJVk5QN8NSdYpOPkmvd8j5RTdokI48QzPfeUwDpf62ztrOnRFrrYv3coWH\\\\\\\/fQLaxRy\\\\\\\/8tmq8NkgI4LgIxTxisy5gLqWTbvGEP6iZjxopfvYZcwc7lBxK6M04ZFi4FBu56OM7+cPHOfuh15prdhRRRZ9MTl6K2WBBrzzQil2hJoFqvhCEZJCqLWzvUh53TVQoaywojm9FLAlcOMpsse5eBZFnbNQxpXfB2igWaqWZJ\\\\\\\/ZngHuSiVZPsvQQ9Ccg4AXI00vMULL69wMn6Rnf\\\\\\\/WsvyKYhEWfEoIrjQrtkt+0P\\\\\\\/IudvLTuF0KObi1\\\\\\\/N\\\\\\\/RvZKZybjl6sEf6PxSalVL3I27VACwFJ5t2rowc5I4+gu3IUFA1X8PdkJn\\\\\\\/tvpcXe43cljdyO4wzJOnQbwECt57qcj4oDQliKfrm4JUzcUA1AA6OO9TqzY7JyXHpL05S\\\\\\\/7wNq\\\\\\\/f95Ct+xJ3w73fMFdA32eiCJeDNXFdlbDtyjiF+zZpu0HjbDv1kwcpH2DSgNkoQoTMutl2hCk\\\\\\\/09zsQ3Lsxk7N0pNDFz3Kd2nNLM\\\\\\\/JjH5s2GKEUm0quwSwqB6eq+gOFojd4FCvOy2CyPXJspa7QefFH6\\\\\\\/i4V2s4xwi6PxtRibIdSAcKiqmalVtPc52MH6kkUh2B+b1MfR64+VAGuxD8U94cITXvSjxOj3DBg+KO0HOhFEqIzuGC+x3yU6\\\\\\\/g9kw6AoRfy3fO6eY5bGivAKMa4HKX+m8ejp+vbvoyrk3vKOdaL28KScQgPhXUkHxPohcSYHozj+KujMTXVtaht8Eo3ZL5VCol\\\\\\\/BjuSRCJgJoEjYsIRTg+YT7pPXEvXOKX8rAX1R5qhzK2Fnnqk8WKpeLdJWpMwGV\\\\\\\/BePQHGe7bOGrG2sbHjuxeUoDnips9SsM4MOie5YPZYk\\\\\\\/qO2qx7Fk2CvWkE0HklHITcXM3JciK59KS8\\\u0022,\\\u0022iv\\\u0022:\\\u00222P3+Qrh+LnD5GA2VOHIDdw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002218a019a409214b31efd4288ed18754d2d57242c00bcf9d2a111540efdb260a09\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002282CKHepZdw9LdqkvM93sT6j5Qa26FAM4yWNRGIokTHI+VmZlH9clTEt7oT9KWQbMK\\\\\\\/fi2LFa6p89KXYfJRuUvPiwWcw4HVia\\\\\\\/OIeqnIveiifkckTfZBwRelOeetN4UWxvsWl1kRxDmqDJQt3XSIhtm8G5vwzqVm2CNh+r09M84p8WPW\\\\\\\/HfQczrTt2H3w1YegqDVBNEeiDIvLVHSAr5zlyB2Lk9V1tAMObOqdV0R79QmEPwMdnVjLWJgVOxsrjGRTHWsdATTgU8Ce9lifLXAoGx0V+Zz7vF1VQn9bMnlONZUugQ67yBTvBED0VQs7Jt8xi9bQbN6SaRmtJCJwatg624fbQelTJm0rYff9oDwVlZzesLfwV5sfP4WrwTXgCT9OWkf51W0eOqEwBVLi0rtZwh2IyXGMWuAETyX4gXnWgNNU65MXizkI0AfY6gBlMXqG+hKSExsrD1PYc67oVIX7TSbbOhswjNk\\\\\\\/eHMA7t2Gg3ZK1+pLpTUVqyEINqcpsbgBzccIeSl5zEOByAr0xWOxEBlixW0JJOnDwrZJ\\\\\\\/gKWe0xISCxPuBUbrlQkxWixJL5OuS0Jr8eqRjvikVsTQ1i+GSFKlQ0r2pYrxffXcioHLFHWtfIolhcplE3XVw3goWD2o+dtb90ifNMMY\\\\\\\/eu8Za330XPTzMhKzmNb05xJkgqeQvb3t9JH1oIz\\\\\\\/yPxmHI\\\\\\\/QS8358JH5sItG+m55sn\\\\\\\/KFpTT8VTcVhhwo6SZwgcjqb+NtSN\\\\\\\/\\\\\\\/xZUBuQK\\\\\\\/+HPm4V4T2Han4RPGx5Tug3rx\\\\\\\/HOot5rEqyxV3FZk88UgDC3s3Qg5l2g+lMZ94VF2HHjzoReksqMJB4rqQa31bY+UQu4BQpdm8UYdI2n4L3ge1lOz2rRVTKtp25f6JJLPWIMDNyzVepyPqemR\\\\\\\/OTMUyuvMVZsNpkR2OM6T+xuDXlKh4fPFPEbLAiKe82pCFWZ1OxHDIyXoQtOx38quNKeXPJ8tyQH3oD+d\\\\\\\/6ijpLF2mQs53YHupSxf\\\\\\\/yPcT2N\\\\\\\/6vIMxnfxZK8777jsaYN6JNIyV0CNB5hF\\\\\\\/3+EE2yLAOlhX3qHGIpTx0lJMPSufrMwbSBdZ8XKZy5zfQ3QNiDa8IviP2\\\\\\\/1WqZedySb6NhQUpjwTaObYzR7HZhM6TolvZQDLX4gC3Q8IB3\\\\\\\/JsUtqE131Ffkhr\\\\\\\/cn2ZAlzHWF3MXnt3XejLLbw9pWOpe4Dqeaq\\\\\\\/+msDYI3bsK1sLfY89RlopwnUighTqk97gXc\\\\\\\/QMAWKDtPHk7uuEt1e6jmblmgAtyPGWS2cF3hoxcTYYT7WJEWrwJkZlY7v8c5a4qZeXg82bnshM+q4M1c+OchNzyCtdzJEsU8hAcyYdh4HI48uYDYII3vnNk\\\\\\\/VHX\\\\\\\/o5L5pV0rFthde7DlEfWYOCZ7qwpZQrlo2sG2aBaqhIuHdY4J6HViQ8BHMcvmTjLIcfrxpZk5nJvfU\\\\\\\/0fpECzRf\\\\\\\/+ndoOfGQyLoi8SeH+CrYRa5k0uwtob5HdWpgD0pJKxmn5uMRDka\\\\\\\/LyigpxtuCsuM3tVGaApzlJjBErlVReWBSZ3Mr5lufyOG\\\\\\\/BK\\\\\\\/\\\\\\\/0YTgqu0rl8MJHPfSOoslJrjO9h+C7fuAuLKfCXiIdBwN62gXr2zbli2DGXq8hfLKfYcEd9wJqnzOHTy4Ckdx9o4U9LoBu22cYpxzOBhHs8wWjPJwacT6X6Przr90N2o744G8Jt5ahK0huTUf2UeqYXxx6FuFH7W7W2vffqJMMxd2btjVtVhrZ8PSRuYs2BGdN8D72EK9ODztrxZOT4+HfYH\\\\\\\/R\\\\\\\/14A9jrkDu2pBLxnFcRe4tBRzq+8Z2n2LuvNi5Zkr4XqVEiJQJjZlmL9BuW\\\\\\\/XhejwVflJSwaAi8OfkiLlox1ID6udBQyPFjnhYgza4Ctah++e8vYpUmdRunJeto7Ly7WPIWuOVMId64uyeM6oT3iPOKhdIESyeZAj02b2cyb7\\\\\\\/cGqwkzC\\\\\\\/le5cUcRNPFvRvXPf1Wwj0cDAtiVHuro25y883B00L0ldqplM4wN07gm3090TFvtemt0sYPVtzQs9xyIOfHFd4\\\\\\\/1Bbtyg3Rk6jX6dz2v069\\\\\\\/ATfChA01+k\\\\\\\/cELgewf9XFgpYFW7ge8iS6AT+KQmK9tv4yPfaQhL1LVila0B8orpEMGQ2MTAb8l9klvuDTeS5ZKQj2zbXHXtNmGPxJEGYMKiTqwTr8lgCZ0LWT9uguN605WNpOrAIevxnPAMZXzTpww897LTRrcMVIFCTc6hfDXN3cQzaKHCdvzKV4WrH26tOzwYMaHj0fyHpcvbpGRLmMJmwjpR9v\\\\\\\/1EY6qNxkekK+ChP9di7yWj9yKED8Lwr9LnaLAZL8sahcFuUE16Wdzk91CKJ\\\\\\\/y2aGAbxQkadjnKJR5NcezvUA9PXdRxsTesxsnRzDyptMXRbUcCkFP8GaKlnyHAcje6pJrQ3lZjv7FsLEx6koxh1vP16lqRrnCFWnYbR99pU\\\\\\\/rB5t2jhD5NWbVGjJiTyVNjLhbJZD0K68fdghYrv\\\\\\\/DpmkflRHnCQhe+GfWCpFfmMNiX34mo9eSgXM+NgVV2Pv4OMcB43GqM2Q5Mm7C7Cr\\\\\\\/DVDwMRGsti\\\\\\\/uIgBto00NcZHf0pqZ9vgrnD691vqJrQfVhkVDHWJ8paS8NBj01hM94IQpLKPsCy2YyHhp6nHApIJi9OrRfJ+5AbzX\\\\\\\/SngcBOKoH18h6unHERug2FdmYnHN4e+gZmoy6HXWGmWqBZvHCnv95SfLD\\\\\\\/ViGYig93NROvoLbfY4kwevBP\\\\\\\/pSI\\\\\\\/EZn6EEliwAjLRYiZDcc3dsrAGalx+4jS\\\\\\\/2NyNOrkDhbGB1jRrvQfTXvfHdV10kb34XtlgzDhQh1wdj\\\\\\\/SqH5KmwCTsFo+iGtxt0S5XSfdQN9XYIXwj8Vp\\\\\\\/\\\\\\\/xr6ae06q588JdoEl9nNt\\\\\\\/zoPIkdl+9bN+Ec28IB2pI2O1HEmGEBU846nIHcscvRLiVfTvDCFA3+jAG7\\\\\\\/DyagNCuMc2cbAj1Mm5Jn92+ZFH7C4r0TbGwmVz\\\\\\\/ffRrfvEmeTpaD7J23nasHwaOPwWGBtCAcSgAUiC6jHltXx+yF4uigM\\\\\\\/+gv+WUQ+NPgn29WlRfF\\\\\\\/YaZ\\\\\\\/UZA\\\\\\\/aIvBTLF0ExNtLjaYK6MKmjHZGEtyX86jPsjQU8HQipdyNGQ0ju4H2dK2ouDbH3wC2xxFhIHOadC8NOJTx9pSq4E9q4lNx7JqVSkYK7p+lH7Ezw0bABq1\\\\\\\/GYY9A8X\\\\\\\/kzn8Jm4HmZmvfNZ83u84WAA8l9eCjIKMMwFDZBNrCEdAir\\\\\\\/XN\\\\\\\/uvJ2GWgPCt\\\\\\\/X6zpyK9fRMDwxoKxuOTX3MVIbhEVDVJL2BOcohVyDFDCgy7AeoCtp5OmPCV6ViuMpkLMBrIHmA2SiW1pev9lICJ4B0ZmNmG\\\\\\\/mQ9pdVyOTJkQ5KTnsUYjyUmC0jouBqm6COLKNRj4rKG1wZ7HR1OEeNIWnlB3uiwhW6PquGLgMKwB+zIee+oLSKAF4VVA0NDzWalvA10lKjbdAs5LbaBs8T32bzfkIYwemHBfMRiy7ZtZJLVLwOWX\\\\\\\/SkFRfh\\\\\\\/9jUxGVBco8beJub\\\\\\\/CNMDTtL\\\\\\\/o7qLeBq9PVQKwxoRrZEUr2zBkrtEpsCTizMLk701UZ7+nypqVxGPEcbq4o5imzGreMpvx778hMi69uBwmqCFFtcGTDN9UWXawY06n2yKTmXeEPKkd5ZvhpwwVZZ3pdInqDQDzstZ+sidd6lXdlEt\\\\\\\/3vPdf3d5EGsrsQHhVVbTH13iA6rMK0gxzZ54W+Os+DXTTSK1+IeLBC7D1KWIMesW961OwjvOrscDTh\\\\\\\/glQb89h4DzvOKV3Zw2ZP9LpvrhDjS+Mf5VgV19s+mX0rqOLLvi6UlDX\\\\\\\/C4DFk5rV3qLx6s4BlA9sj9NLqWPUEFO1uiVZFO5Bv6iOQElUBtT7oLxJJLRsP8g0BvPh7UeD6WS8RM3bOWQQ0TtF7m5qsoYFO+JLzHiHGUH+iMYTqwCxoc2\\\\\\\/Y3EGzbPEwzXREqyATJ\\\\\\\/mQdg7lnCHfG+ckgH1ht7BjUKPFdKjF1G6MJ472\\\\\\\/b\\\\\\\/Jq5GHaJnKjC2og4bJ7K0\\\\\\\/QmXYGy\\\\\\\/4YQ6ABZvtRrbZA9dlfRZVxFpUQwDG6+KMCJfnwXv7rkjckG3hdTXJHLj63Y8ttGH2K43btW0KDLlUZfy41WDx8IbFCQl378czUyARb0fQiKQq+uT6\\\\\\\/5IekqAsn0xKnqP2YtF7OhzNA85I7U58F1+M9EJSdQldodf0Ngh\\\\\\\/Z7h+Gfp0SOmkS68I3d4WiueoE4FXzll3c4fdDl0w97UO4FXT4fEs3y5WUCV\\\\\\\/+KK1OIXKpSJuhwWxKIRME\\\\\\\/4gGGPvsM\\\\\\\/CxXhYrzmgW73HTjREQqpp16Rpa\\\\\\\/LxQPZkPXdMvaKtXebO6jRCM2myEsNFfeomc\\\\\\\/pYJMCYJP4hHW1nyOqFdPMqB8OwaeqWB9Y++8luCdjQGK7aPQDQeyntmYvQwbIoTsMbFK8X0W5t9RcofJJK9AG5GTNPmF1TErGGDEIqxyYqbGlhfGjwfZ\\\\\\\/\\\\\\\/tTEIycDEMreClyLQAcHjmvgRz4muqUYDKLXOTdY3Zq\\\\\\\/SOppxw6oWQwwczfERox0\\\\\\\/0lEwUvFJdSaI1a8\\\\\\\/r8zoTy\\\\\\\/ICkKSJyPLAtSGyFGJr1kE5kxcVYpY2g7NgAQBzCO\\\\\\\/8QUx7B6+wGwdPtHFuHp2GHYrO+VYwrH\\\\\\\/eKe6p3euYmaYY2X+cFeWd8dnIo079sTSScnZQlr\\\\\\\/d49GlA4l5PTyNcsYR8IyBPQCM6k2J6FcxAwndICMfZc0xeg4C1bmwD7CN6LSHkt8fPMFFRGRQSYj53Voz7vubgQPIaSYLgreRNhN5AG8Ok60SF4e1IwOuJyIrAnOIyPhuLQV89QHEzjBN2VRdEYXnREF2vS8arvQNwsf1N\\\\\\\/ritqSOEAMSKfkzcbxCvSbbDuCpqVASjoEl8WjgD6TpzWSoUOxDykVpTqGGZqOdHa9S07LJqAM02pTXyOiQiSk8SLAWXTm2jtwjxdhgF7FtPjMMw\\\\\\\/fdbSCKXxdhM\\\\\\\/BBA+\\\\\\\/NdrN6cXj4dkL\\\\\\\/kzaRNHWmL6\\\\\\\/V1M7t\\\\\\\/RM\\\\\\\/OjPjIL+2i\\\\\\\/7RFpnpfOt3PfB6\\\\\\\/0AOuzTUeX3qH\\\\\\\/U3aY27F8LfO+BSCxb1cBdITACkFMFXv8X5H9EWJMaJonLHmZC\\\\\\\/\\\\\\\/OVYYXeEjDUtKsiULL\\\\\\\/gRmCF+fCJAqSjpvCAqhkMYD2shmnauH\\\\\\\/M6B94xAhRWSDh\\\\\\\/u9jNb1vJUNzCjRxgvpgHTU6Isig+cAcaIZJNK3gmfQVotUC6g4i9T6PrI\\\\\\\/bTFpLHqtwwWAdHs6wo6+7mSHEPcD32MDgd5q71oNqn+fZQPEsyHlVIc9eFCWMTVxP+L9f0I56o6gApEeyB3BK7cwY+4wQVGoOzXItZ\\\\\\\/QybBhkpxREkk3rs+gXg2P140lQKrY90AoX7sIiZRyrofx5U0DH851zNGUU96Jsrutzm5TS5QfljuNmGj0wTePeydfzsWxSXG\\\\\\\/GaiJFMUSlWgYpOYG9ApXibMnM1HkfNUnsbY\\\\\\\/0Z5hVCbIpaD1T2OWaGmelBvY8qXn89NWaw3PtDrKFQajwkUpSnJvm\\\\\\\/DxWgWuNcftA6gCMZ\\\\\\\/moJ1ddbk+dzZKGweetV9EyObczgNp1jRVD\\\\\\\/wxs4QgaaTtVxXkGgFWmYqU6AT\\\\\\\/qEXVWgdr33a3s2Gt8qRePkADSQhEvm\\\\\\\/2RsWcHl62kBkF01ToDHjrRjUUVnIqqP+gA8Bm4ji\\\\\\\/kloVAdkidYb1Hb7l4Bwqy\\\\\\\/oCQzqEqAcyfsCOBE1snu\\\\\\\/oVWae\\\\\\\/TgpvzbLBDgNjExL+bC\\\\\\\/c5p6XBn6+2N1nLmtECUB8lFp6MA2\\\\\\\/m0t0moNyxBfYNkdUcpVGcxhMPcF\\\\\\\/xZtSkii64JDqMCExGyE9Yg2A5GWPmXc7oh5OULAG4Rx0rxgMuNkYYQzkrXjOolTjmDjS7w1IFbiSNJVgoBvCATFEPH9jmj+BdM\\\\\\\/q31TA0hF0JZ2S8Kk\\\\\\\/en\\\\\\\/vqBLEIbazk\\\\\\\/7yXwTcIfUKCNAZTlUYjAroaSHU6vP5RfzoueprGxjZFkMyEBLf5p76pp4+olq457+ZPo5GarLPE64M8iM0iaNNq7G0JwN5u3inxno8AKPkNNkykq1jL5eauTfopTomGV0NlK82aq5f1Ch\\\\\\\/JFD0uKdf+JJ0a5KRwrQNqqM3DhlTMxCDdMGl0XTk955HlWR6Fbv1\\\\\\\/HUeu8pbyWP36IOViSLWsD8tzhat5RBIYTeSPH0Uw0OuS5SpaxqeyJ3sBhCbB36wkPeX0\\\\\\\/crL1c4NMSU9lnN+DCZh50EebG802jAV9WN5SitTXMrUVEtKggK1DvmI5E6Mx7gG4W7JQVM74Y7LtDjKGa0lhAjha2nsd7Z+Apdf+\\\\\\\/whsi6BpumVvgFcwsRezXj0wJNpnJ5fg27h8dkxcqfEd6R4UR0zr6hUPjG+t2CVxzz7lOLt0rXBtYhk3wnMtNzcgT6DWjiBhIuSPg0KuTaVXa6RyXIM8d2raAiaXqI9j2i43pL7OExqD2ZVJLlTcsLPzXeZbCbo+B\\\\\\\/mgoa25MsDBdZwVb5CMJO1WidikI6GMvjWmGCLf2HxKYoef8YYI4HT5D9XJ3hQHzKfeLWn5ajrjtuOyYivy30kyaDvE4k11mbmohSGjK6pWzWPQyCZFQBwXCelVF1edlKFQ66Dx3\\\\\\\/7nfoniE2xhlpOa\\\\\\\/KDEPX\\\\\\\/tP7aWnrdoJhWoLvyIxMWoFg3Lzilhit2O+DQTAkvdulg2e9j+Zfr6Hu0h\\\\\\\/jfC8N0Gpuw\\\\\\\/yarW+4+eTHG\\\\\\\/r6q\\\\\\\/vq2qpJvpHAG1xpirXEsXshhFlFF8tKJzQZw3zD351+r+WtMdMmmyzKyIikGlj887FJfrlwWrcHtSWbviPeKEtLSdkDS6s2IaEENARFeczc2xEgfptH+Rrxwppvfl3k\\\\\\\/tfFDnuMp6aBHizJpFXAJ5W7wRVY9AS\\\\\\\/okLchQUdtU7EcMrCPW5l1+yriw0epsSJvYxuHwk4OcNMwhEaMvsi1eP8DoRRYdfprwIkApkJTiXF9rImenp\\\\\\\/3xalr88IrHH1cALLXTam2\\\\\\\/RDO88R5xrbFtiNq957a3aClzOyf2OBfQ4+ahK2rKKP+JEhSS4dJ81mX9YfRDZe2Q2eeGMb4OFoWGyK7GbwPUTEhD6t5dHdlQBN+\\\\\\\/LqfdgQSc4otXDNJi2\\\\\\\/591LqRHxDjOhRjYtgjrvb0aP3xMACj59sXodPN+xy1mTPnjuGehW9HauYttWg832hwBSOVZt0kMnwR4EoFukgrrAI11fmR1GN6vA1BGwAVd5ju6ZdMCYnqIKG+1OAVgowDjwDqFx9C2DpsXNWeKX\\\\\\\/WGqpY2ztGWmmKzmDB6pRxgB7wW0Dr9xi14kO\\\\\\\/Gax+ZC35Oz+kmOiBNHpqqrwesp+RONXl1sO\\\\\\\/\\\\\\\/iAHdb5mCty+swDG\\\\\\\/nojouBxsyaF5qh71NzAxlBXy\\\\\\\/5Slk\\\\\\\/JdRZH8FrAkH8eHtiALj5\\\\\\\/cQeK8mBuycEm8ZnnD1CPfKbiYCVW9zVVV75IWFyZQxRMoMP7o2+0fXjdDDqIHbDxVtZvXM2IPxHBn7k7nCxbzUTCkoozoJCcAXYB8SB318sNccW8sxR5MkWJx1jNpntuCyANYG7CknXocTjDacQki9LJlcZwmgnt73laadon9uCpl23Za4yGfIAGwspLSWDBCBotOocPZ457CZF+DSAV+njO0A8OVIeHB6g5oAuUOeGAgK5Y3ShemPN109TX7jXcOdEUET97AS\\\\\\\/X\\\\\\\/AJvMswKhtHyhlesVxgjDbpeuT50\\\\\\\/g6vYkQ9fGdC2I+MIE3sOC\\\\\\\/eY1jHBDQj6Nyrsz10mR1MUKsyUNyfXjCZ6BoPUxjBxxEfzXmZRBOmEQLcK69kjkQP9D8KMCKD\\\\\\\/cmeKzgJMzVcPHOyhBR3Hs9OAzb22zZ5nDmGDwTXyD+y5TVyqLUEIilKMxu+IH1x47uMTe6Xu1kaXK04WXX9C5Kqnre+TyL3SZiCUZAmYvb+qoMaumwjyNzS13H0GSP8zS\\\\\\\/8DpYkFbaYlyC3UhmspPIoNCtAqyojyPfEDLRQ1tjjc\\\\\\\/RHfb+Je7qYrWaJcNYV7tMFWAdjdSIbYm81biIiRp+4xU5Iosx7tPwYt16yjzpf4qcVBpQoBc97xqn8Hd4+biASXPILJ6s2379AQ9+u0Xb9FV6SzUt4N7D2NN5md15uBzLGLN\\\\\\\/5yHW6G4bh46bZCIXpYl5O\\\\\\\/ouNa5AHrfeZkikE49YDioK7XiqSDiy9YIj1880986e0m8jB+VIItPzCJZBjBLTTzKgs5s+ZcUBttgz\\\\\\\/w+0r7FzxHJuWk95LRNhtr3JOqPjF28BYDI3gLwKw0Ppf0e4E34x5+AksY0cM99eD4fORN9ax\\\\\\\/OMJYCmW\\\\\\\/uQLvYl98ZFhQttCb5cwoQGkf5XIJ9CuvN9+FNuEGogmPpYSBxl5YtQ0SYXKRdSRN\\\\\\\/l\\\\\\\/hPl1SXl0Vyhos1JfjnLGD6eeWT7wShr3Ipj1lYVSHlVtQefTAOmk1v\\\\\\\/eZa3MEC+ZK7hQPpVbei4NIVzS488ktXln4U37l5JHCBwPcaD9GZDX2ENEJryokvKGOEdZNvBKBWi0pBd7UFa6UGwH+b2jUOFeVsc1tu2ucI2WlckX2rjP5uckCg6+S1EQ56ji2DpszeRJJg5+pGK3fcK1HRN+P9syz\\\\\\\/V8LsZW1ZmftkZdy\\\\\\\/tWiD5h+zF9DfJQtclRaDFkB7KwAjzAnBn7km3IQj1MOukh+Oh4lOmrkgp0S8KUSxccMGzqIwD4LNCrs4tkcPSRJAYynil3uWFl7eJSBWgmx8pCdwbYccduMmIElH5\\\\\\\/b0gWRlAB7MW\\\\\\\/6JyBEbQtZKN8ePpIZsTAXeNOIZwSUBRReslHaC7XG7+DpDZqwUDN3E7e\\\\\\\/kmnWlta7pGhx8U\\\\\\\/C9m+tKdbl8zs5NnlMZEWgDEwKSk+g3UBxGrm7r4IdLwj\\\\\\\/T12ZygI5352TZJMKvRRi3GK8yyHt1Hx17D9WntLggiEjm8jYUAZfkzgUzcTPDg0kD4xBlpokp943mC4iwq9wRCx8FNWKIq1\\\\\\\/yp7JQEGEV9a+Aril82MulANKZ6jbE45h2cYuwJ48n\\\\\\\/+N+FW55SKWFv9eKADRLE\\\\\\\/7Idn828qLtwXVta8oUwwa4QdcXeNopRIUiQXquhw+lXpMppk0wXEl9nsCFzcvn0Wl+CJRe4Mfa20rNBRtu9AO4XvxVaNNeKR0nHtW84P2Ea5Pjvtix8j+OR6eCef9ypmusOjkUNZes+j2G+PhuOvEZg5NxQ7J80DcNsSyxXXdnj47YOHUiE5qZY+01vubVe+sujrwMTDkXvRXSlLEpiCIido8aMJZRv3CfDRYm65kSYfuEioeTGtQSewi32bQ6Wuk59TZGkYnzpjG7dAKkKJZF+BcDyZbX+odU0h3VPkcGMkSYJGOjhJJIaSNXPv5hY3ydvy8BLeGmDkQl4VCPyLDWuWtvZEiwXnUaCifTGPh0Go+Gyi\\\\\\\/BeSP5kRKkJpjB+uP+Cwb67mNztunH\\\\\\\/sTB4jg9e9vjvSPFXSNua5tBPgEyO+JufRhZ9Sl9+UDq6Hj9ZOTt9KVwph\\\\\\\/Ys3Pk2Vhep8FpMYnGEkveSW\\\\\\\/SCB6tfoo6FO4PnCpFvf\\\\\\\/rFSL5\\\\\\\/NLwcJFzMhOEM5vVoFX0z6zRYFExTJmS\\\\\\\/Twikc2zS2fBZD8ZOWk6Fwl5jdhCEyOjbZbvXJ\\\\\\\/PXcJCezo6ZzbrNhMij42UbMpw6we2hCnIvBUBXQ0fbMs\\\\\\\/8TlMYGY6Z5ogEJinflE6ggDlG70vSkFfYYdZrnaACp2NZYQRb6F8xQoVzlicK3DzOB9noI9RL3Y\\\\\\\/lGH9gV0FdH0tF2D\\\\\\\/j3dDhtIlxTyoBL2MS29rc6n0PMsKGS35HJ5ihzNlUF1ZvYfLJHGtvFuRNIszzvals+mC6rDCaJPH+54sXo4H4tugN7nV21F7Z9GL24rTa2meHJzmxevw0F7n44yprr1jNfOOqnbaQ+itYh05l5DLN1hLtLAf0wI+PGcev2WMjft7WtUD7nLcy+6vm7EWSEcUhphrhQiQ7Me+MV9Z\\\\\\\/mvTFDkK+n6mPxz1iloNM7Vr6FxZxdz8tGKCGZZQyQeOJaGiVco352\\\\\\\/D68aRvJ1A5hQuI\\\\\\\/Hu1P7dDtnEK3h\\\\\\\/1ZwIoIAonwm+7VWdsG7oWzB0lswgU0CewynYzDPS5ni0UC5HNDo+bbQJvpOij4vu\\\\\\\/SShPV6VQNUDRpyrtl5\\\\\\\/KzdVJG\\\\\\\/GktION1yPmRkZFpwjHJmPtcVjvdgiLlIXAnxx0M0gs49OwRwi4MSUlGaFei6uDKNMaxgIGTW\\\\\\\/U6J4d3dPxkEdD8vgIDnMEVJeoVZqu47ecizOKChHyHEZOL4LaA7+ZqLo3uYfoG7wROhqBc49SKCHKKEgYsb1lSKHj0VwbQxTK\\\\\\\/JpS3pdqYxwEiJtdRLZ2Suoy5XsjDUJECrkuaxzCJu34p6DZxTdK730tUz7mbM2kat+4FnuS1QoIqYtzFDO0n5hVZy2Tm0fAhbaKxTrlCHKHJ4dny7CXkEq1UtRYli7GRdA6WUAhnfptTgdd3mSv6wJS0cSAm8udWUEp\\\\\\\/zin0+hrVMqrytFldyYdnNI1sCml17pOd8d\\\\\\\/TaWvS\\\\\\\/RDAQIUvEjfOHXkwEVigjdRdiBsa63vnSdmuA1JO0H6\\\\\\\/cBHk\\\\\\\/RgZP\\\\\\\/AT\\\\\\\/8XgS+PesRAw4DRZcakpyefsLTVm\\\\\\\/igHCbHFxxtHpYClqEuvhVy5600C1cvnFCSwIBLliWYYlbx6vppnrjFs7ndnwPvdBLNrk\\\\\\\/\\\\\\\/zAOM9jnfOseF4Tbek3Gv5vuDP4geLJAkWq9iVa5MhFKUO38QbxyrdESTuR+er22hlY5aZ+cTC+mXUJHbVWFwDlD2jmp3Kr88qhW4rw4msV9pLl3ujieu4B34oOxyum7EQO5V0YPT1525KCKTTv6smJTxkEY6xWvf8o2OPL59dHtITSy6j8hQMS5I8\\\\\\\/8eWHosXWX9Jf3FkS1zapGah6nvGmnWY5QDoXhhR1OPx2yW\\\\\\\/dhrRzKLpQpUDkuSGK80hLz2R03XirZdEycHCoDVrTc+J6LtpklGOthxG6QXf4dhtAZmA2uqIaU85MKupbY5Ig1qU4I\\\\\\\/TEg5PPHPKHCz3ILvQhhXuCybuz+ab39ZXkLR34JcrcHfwX+eqRQcShEZq9Gj3ugp+JmwDqaAGcTKJD0Sj4DWHMZDqdjHOL\\\\\\\/AfKCEUl+OpbOX3ZY2k\\\\\\\/sX5QvmQG8uCwN60y0jL+6BRIAn03ED\\\\\\\/1SguaP6YILNj2zYR\\\\\\\/MWkBiqxxzqPU8jdcqFUcgGGLILupCQVpe722SOerdU+5at3B2Ew4RShz2HLTUrGULKw\\\\\\\/aTMIx5q\\\\\\\/rqp6VhJaXn6piATmAgpLrd\\\\\\\/GEZkMz2MfdU4ENqXCHs4g8YlKF9bBVV7gcmJqtorhZfDFfRSKlp4XT+wnlm3zRRYsOHG593BzpG39q4LWsVSWLY3gAhwdslNZek7n6DSlROXNRvgOXDEekFz7+wVNZvFxY78OvbM7eabAPuGqQz2MrnXZAY4XVmxT+mXbSSXT6VZp74Zr90dCKFrkTMB4RYz6Xhc7fSCyOjhpNk8S4H+svxo7MEdku7c\\\\\\\/nrSepXOIfJfCZWs\\\\\\\/9gaOL\\\\\\\/mxpFwLIgpLIiraI3Mm3Xi4ODrptvy4yjaG3gKGKuo\\\\\\\/iNoWVLlERHcwtlRwDUNF07UDYHsMaFnG3J960W0XPFmprDGDUdxv9ksH32OycPhqueisOuJN53wnz5IaIAG4za5MRa6lbJVSIcURabq8+kIrLhpiy+jaOB7NRULzIoQGISEoJOsGYYnX2RVGQSSGzSydJxAXqAyBwfjdg9o51uo0A2kvti871XLAwqsMm\\\\\\\/62TXN3IZy17BxJ3BdcFKNHGUYpSmnxKXS5r3ztICPapBkuDXngvx3UieXfygX9cLFRZ5Jx5RAWl+jYNOfxauoV5q0cQx0le1cQ495YfVo0+SBavUP4w5ZL83tt8zdwAHrKCDRcvNCnQIZZwwKXsMC\\\\\\\/9f9FFobbnzH4RfpdDJmAENF48UKK9z9L7TASsh3GPhr+4PCXAUo0ln\\\\\\\/5zCjJVWTf8g9sLhEMsmmbtptVBm4WteqrFWQ6O9lM5ldIRovWscB06Qq34oViAeA2thF1VkjHcVBo56jWgal1iPgVU5McSON7+X5MDf4Mu8nCfR+xRnp161mW4it61wYZFg90qLSP5GP8ve2Y2ObCiZe8O8x80fmbCXPYkvkXR0uTkF2lMCDf7B4csVXQS\\\\\\\/BJIGl8\\\\\\\/ZaCpVRLjnu44J1sev2jwS7vQsrVAplqZiGOkOeiCw0D\\\\\\\/gyJz0GHCzFlYditDiHN8xWkbmlsQX7lahGMi9fd8vpc0U2vAjuUEYBXUJXXNxIoGR1+xL5zLFriHMtAy0I9AiqnATHogkTMZ0W6l6gy3iOybY5dq5zgWDkgX7yTnZ\\\\\\\/wUEKCDuLVbY5TktVmbueYgtozQZn9Balh\\\\\\\/j22FvhOAXEFgU36QllkQ2e6T6M2XdInPoC0EsK1eyOxPKzGWj0Xep1QKpLDDRoPxylBOk+UWA5lTDUGXyM722sJEJS58PWeE+Fi05DOSmaZ8fjoO3V2cctxRIgIU7pQtrP4t9XNRA0RN08KzTpRkBHeQz60D6+JjPIXuhjezMJiaCXp5Gxv71JjhELfrtqKn8tm8yZy3yK6jBY+qt7ANKJ15OV5Zo37leA\\\\\\\/j7tTMeu0Y048ItLvFr8DBbk\\\\\\\/9hbRB+q7Ew+W9VxP6bbnBenf3TOFKYJ413D2YIKFvTWCMUsvSGo3zif\\\\\\\/PE04Npq+V9PNYutxXW63fTbvHDDlzYPeU72F9oyem0tQgqo0hZz+bikrriljT1ykect1dWOMN+ensdL91TIGwXcAE2CX0HcnUtK3v3nFSfX1L9f1zL4Tn22f1Fg5AmmjsFKrpoErhQ+jBcKZYabIs\\\\\\\/7EP56Y13uhijPDMtymRtKLKsfTlkBXodAw+n25TXm1jvU02jfDlYbf0VQEGOTEpIXwhzdTuWp6zI6q9cI\\\\\\\/HBebmCd8Tbq5dGpnt3zPsytQFbP3CTlsrRg\\\\\\\/+eJQscVH4F9lNhpAUju4VwyDnbPrKBUTZlf87rMex1psLFFc20c1WivIwgWcLWsnHHQfCB9ihFtjAqCxDbBpaoi+io8J4HErFb5pC4mh8TBqhvX8lefxPlq2SQgc0bH3dDMNL26DzxqL48dgodFOkK\\\\\\\/JhYUKS7NUsy\\\\\\\/fCvLqV6wW6Xr2\\\\\\\/lbmb4IGmAYP0KOQqT5qtWm3ESxIl49Krqy0yd1DIAtA0\\\\\\\/gdQaGwEdLV\\\\\\\/\\\\\\\/mqsnBvUDmDyI6XR2NatnIL5XdRbpM2xJzego9QkhebxqJgypn+4w+mH8bD1d8yPWUaIHueaQu0dT4SFst18\\\\\\\/rb\\\\\\\/xYDQzG66pJQog+n7yoFZxttxVdrpZGdImD+dIMI\\\\\\\/qq07tU6LRiU0TEl5\\\\\\\/u2sb+vKEmN8bHUhlbolvJT2ynKHJch7eS93dYOMo1Ol7cFGpXNQn4GYoCwZ+pWiJAu36nHKTfGUzr0aS6bfsc7NbPmB4jk5axeEqRfzU1gxBmc9AFX+1eU3MvC9dPzQba\\\\\\\/K9AjmSldSEBGEw4WYCY+Ux+HbU\\\\\\\/k0zlEK\\\\\\\/JTIKwKJ61lAImtQXWqIqSdTxCt293cgLeU9hd3ZDUMeAcKtDsMeaj9Wxwrc4SUGtVH5IlH63WNJ70WuURg00Y49mCgbGdbwlYlxiizi6cbhpg6Ra1B9axjLecu5IOELwPabhxpLmQovVxC6T1yEgnaV8vLqUkFYArT4V8YnP10+SHM18XwtMcKBwecSvhWqbxBdqGrrfiUBHxsKUfTxo9L35RL6emzDjGOZNlwNGDWK\\\\\\\/DaZKuDrCdskipRpsfCtYO1iKSVjC3SRHkyKCEXcWHFC4dcW5Ac5lj0thnRc3ujbLdcUKsgnc9EbMH8QaL5GEf9vXkbc5ctZoNcAza1C+9mUCnnuDuaT5dAOgSWJ8emeJgJUNry668W4bxY1ZynSs86JZo6hMMNmJ6IatFjCXe5EhttEyqWsBm7s8fDKsZAhP2xHeFvQ7kvF4HvRcuDpDc5xJvT1s+FnEjRt68m\\\\\\\/\\\\\\\/s8\\\\\\\/Wgd0OLboaMOr+90qpTNGz\\\\\\\/BOZHr\\\\\\\/cvDVwZIc9vzF\\\\\\\/KBx7qg7wiVXKuoqLd58pTj5ChCpLZgxwDxfSLOx9qMiae1NvACKZHTxNDN0QZ3cgpzhf4Uq\\\\\\\/j3s15Q+1apfBxpyRn0l2J1oXtWqYpEhxAwhXmZ2b8oWwzELcff3niDKFkQla1nMsg8LGhQzADzSogaLtoTcQ0fRJTNltEnp8HgZY8N+fDT5Hn+0zKrQ9EhScnHDJTnxdfenN4jGgrGWdqAh35Rj6U8zS+TiVSucmq5W\\\\\\\/6Fl\\\\\\\/TqUmPT9F+m2cxyAu1xrVm3x0eppnUZbcqu+vw4ZBP1l8Kbx8kCR7ZJJvGFS5jKYfxeyO95p8WpkpVY14g6yuRMQTZikYRvj8VVcu4NHDgwklGwmVoptRYa2N32RCGEr84gYmv1NokdHLZfrBumpxm1o9m7UCnbFIth\\\\\\\/Ne1I8n7HfwMaairu8MHvXPQaN26MSfH\\\\\\\/1tmZIiFytInyL5UJrxVwfgSCT5Zcd2beUVFxe2lVtb3kJF\\\\\\\/RbIV1sNFp1i3XZHaKXk\\\\\\\/4jLbY2hfq4r3r4haO9J6MqFWm6UF9b\\\\\\\/7O8mrhQQb+Ubsi2VivgHiBnXxLerJfKc5Kr2k9iJ9mBxzJZCpqZ6uzLx\\\\\\\/bdppexNdj\\\\\\\/ST\\\\\\\/XbqhFRgDAvAnZW0M5Wc\\\\\\\/pF4udHWO8AaJNXdZiBIMZp7gGE88l8QxgoqjzeP+Xwj5B3OysBRHpM22wRtFf8iwrTgp31cfybg8AKp4v4YXaOOpYXF8Q51NWSk\\\\\\\/zO2Ky4FRuqgRt+ItCKk2bWSWOX\\\\\\\/J2ZRSxUAZSWwT5eFRGIAPQdar4A7+KU6JR4cmsEo1Ic\\\\\\\/YtGp+3gkI7ueSZ\\\\\\\/qxvyTSm1CXflG8Z2r\\\\\\\/w\\\\\\\/ZlBCn0t5CMjvPliPeEnycMb5wG5N23q1DT4wMIiDnli0eQYfQSfAJ3cf5Nii93mHSDdF8Q7gHF4iJf7YY5JxQxKWSWYR9TZDNgcdmZjAPsAszxlanUWlfJq\\\\\\\/VsdqtDzthFXbOlRAiApgqFUZES+woD3FMrq8SryXe5Te7RUmAsNZuW1NwYPHVsECEFHB1HnyJTLARdIiY6fTwl51RDAFK26BQGzWgarMHpRn58hRRQel9kBTryOkBHMkyyEPW+W8O7hY4JPZiBPNsVkc70cEEhtd36UnYafq1h0lV9pcumGCnHOahuklco3FxHweGkW004gejZyjdd7Go2Vqrw3t1qqXrG0y\\\\\\\/l\\\\\\\/nBLh\\\\\\\/pEggsh8U02lTU1udceVFj1cYfbnigzqjRHRchbobtKv7IMFYp9pWhJnACGbZv6dto02j4MNepuryo31LvZr5cBLq\\\\\\\/+Lx0WtZhcO\\\\\\\/SK0YIewMS+uhN\\\\\\\/q1jlC+WmyTOeUbShNwEOemiUsVt2KWukG+aQwZKXVkcONX23rSQsAd6gCyK9D4qF7QdzXY4XxGRufiJveLcAFr08DkEgxcFLc8v3yutYmGHwwNqq7IaB7AdkampOS377E8Z9FFbs0KIPsueIl5JDHc1C8gX1RZA0V2PjmXaF97wnldsarL7d7D33TcrS3POHUrq+a7mlTT4m+Tvcrpw2RcgR6fnTfAXpxz2s2uELsUPbmonTfL7R6RJ6tU27rM4Ix5XKo4UFAwcskHKrqHE7cyao8ZSMhw8na5Gj4Kv8cznf\\\\\\\/TQJeI4Z\\\\\\\/aDrFV9eWHNHUrB9hhnOBwo0CI3T\\\\\\\/5qWM163EgFwBOl5N+To11E7dqu9WNib5CGKvs0AlU0Jw3fWO1vBl06kLJzPbin0CeU04UMvriOcsai37NT+BtuR\\\\\\\/oqOwB1pLIDFg7FKS7CIo9\\\\\\\/jf8tJKKTCUjmRPWN0ReYookP22Q9m+9aL9obdMER7hUebmMnhGeApZyRzyBcroM0q96dI1Se2sFWpr1vi446LBjmnj9+gMWAeZmGFujTZxD8vBiKHLOt3jL7zilmuZpMlJApHLg2ALSHbpCltF2P6mCDgW0mMIaF89jxOy1Ej4qoQGf+M7MjIDRMeNxJhCSIVmr0EpGslSw7Ya2Rx+qHXt3w8lBLHyp0hAvZ\\\\\\\/3Ym+I1tQ6PtcWnMKOso5xidxbxHh4r2G6cVblbBTp+t14hZz73GlAqIeoZFm\\\\\\\/io9IKW2Gk0SPu5k6x9cc2TqqoSvWzqsOX3HHEgeQRTggEiV\\\\\\\/6CKQmmYQ37iCZh+hsrbDrEcfBpektX3Sn8YrONcI0W9jkyeGqLJmlEuZF8b0AGdOYIMnb+5cUWrB8hWgpL\\\\\\\/Bqvm19a2b0R+9y6bdzRGSLSJjFerLPPxI84tz3cshvaDNvvFAEAm7QZIO0eOPrtKC0Rc04RfM4piVKDWAnAjsR7LATDcnlvZ9i5v5xGRO0PZsWTqnsAfeLIixr2fqQsL83KsG2g2+1UIa+RFR8HYIrQ935JwV9w74sqGyJ0liXn1AHCZjnEa14SsvnWAZ8KZLusESH2vSRseBJ6JDRZOnrjjPvAI\\\\\\\/I8wSUQuZEAs9jMW6PH\\\\\\\/Dp\\\\\\\/8lvVzvQZJqTLQteBKUvrwgPTIIMH8oP1fjmptMnNIrWlyMreO3LPnhYOI2p+i7jiurOQabZyKQ9RkvcpIxQo0s4ciq9XqtsPCVMU5GVl6NEbeCpbPdVY+aNendM39LeqIIXQdq1vNYTB6BDKet4afkvljQWP0wG7eT3yj4SS1h7v70zMxe1rAMXrTfspTa\\\\\\\/wLUFkhMkfQfOH73DSaJFWyn4qlsd07zNBclYYu6mWQdfbEQ0F93dw1HrbRLO4LSiood\\\\\\\/nWyjEpQeypYkP2AgvP4sKf0J57cMGwLcsLA66Ps7iR6bRRT9FWC8MdezysXNmhZjpkoHUCNHYbJ6pH9X\\\\\\\/3BYwj05ybLp89kbXFgwXQRBeOjMAtLL6rmzZo2sFzNlDrvNcmxhFwy8jmM6LGar1o0PASwWxjsu+7Qu74P0AynEf4MJ1M7LeJGcambRDIqYuBqVFTc4LKM3a3Y36P4rHQyWdJea4jW9AJnLLGIzZ8fy7JaYqbmALxBRnigCEuXVL+Ljt3IRgbiCK4plpIyR3x+YIE7KaUzcox7aw\\\\\\\/kYSqy+la8GnpPpcb+scHBgnJ03SmHKP8N3Ji8WdGvWfuF+cV0cvPaesNU1\\\\\\\/cS0eB6mN4V3LgFsTW67LmjJ4xLUvsoqtcojiFe6OP+m8kbU5WVXwXHaJv9awYjdjv0FqeyrZT1QmqEVgc72G3Y5cIaQ4til179Ro5r9dphskP7zAy5TSUlVyWR7Np99ZS36yN1GghFRFpgQBwSEXfU+hBAdCv2\\\\\\\/a6smDgpznrXPdY9meJ6TAnw6\\\\\\\/hD+5oLgVDvz3o08dfRhim+f9jdA+lVMkNuh5osamp+LewclYXHiHIv6Lru2e9V\\\\\\\/5QTYoU\\\\\\\/qjFZu7ytblIt84CGQw1CQY3\\\\\\\/KFdKU7oG0CwgJawLHcn1o9jUym6JPr004W8k7n7tgKvftZrYom6i+XIVE\\\\\\\/yyuN1XEBrXqWJs473xvJ1+O4CLdaIZqlcKBWbIatU4jr4JBOZ2bPLVPEkOiOBQQ15FA4DkpNOsD2DuCEPb+jvL1rGTtaBC1ALvXLX8dtgZORSqfSPnmNUUix20Vn8UvjdPWcbvq7xz3LwXtGXmKl568zEah\\\\\\\/3jBzdmTn9wlSA++IjgSiu+aHqDezWGrTc21tvrJsMuCC\\\\\\\/SEAmVmwhFbK83o2uEBMnSPhQRHGPoP8mV5jqLdYcXOKOp\\\\\\\/Mt69+KlRj8fVyhQc3TnFcyoCiskTLtNW6UpUcHGe8sX2efmaRhz\\\\\\\/2NXCGIYPTE69I9zRHisP12xmEJHLcGRIXjtz57rT\\\\\\\/dB1R0P9wC6H2EJ600UnBS7sYtbnlFi4n9gs8sfbljHNECV+VpQtyV4PZztOn5zYThz8oRnAPNY6E4gOf07nWpAFaUiFjgEwF0FgRZTIHvOQNkhQIa\\\\\\\/hQFQ8EdvszJFoY9Wj6ZPIg7O4T8bA+O+n9kdnl1j+afX8f03IQ2KMamsVUYClpNZ2mY8OqkdEInR1iu0b2u5B0pjC8HJLX+6ddUKhZ2DWMwiiH1XxCVbzUrfz\\\\\\\/h0rwyZz5BwqYtfiKaFLe3eWIs8qoULy7vU4z\\\\\\\/WXa3dUjd1fEUl7FF7eR2ECzAj8u7E321lqeN8kwukgPxtaoRPvt5o213L+K7wSKKS0aLhPQqO7XgifyTKdUbP7ravbDsRQ51pUOgR59MfKAcQx4ozPdVDOn4dVx7C49dXibKBm9hxB9rb9Iretrex53isXI\\\\\\\/+hjPEVV5pCMgX8eJyMjFR0xexki4LtpZszhZU+xX0VB37Xv2j5T7OwYkVJf9BaDiokDjlsM\\\\\\\/kEKolGbuoC98BV+ZbsBhOfrgh\\\\\\\/CAnBvektsZK72VzmwjkEEFdWEus7D1w+IkxxfrWGZEzx4FBMVoFp\\\\\\\/yEigvIO6NnQE4Kut1zu+R\\\\\\\/6mx7\\\\\\\/zF64aobuVNr1pilbSZfIeQ3at5+I6UG8V0KS1bx0I433TRyFZj4ly3C9SxW0AVNo7ut6lBu6f20AnaDy\\\\\\\/upIwls0rbzfFZk8U12uap+3q1Mmf8J8PwtOUHbltFw4hU\\\\\\\/z2XiMCwQj4Iz+Ws80UINbFdkFK3jQdyq+xulEj8FUAIIX0\\\\\\\/r5OoCL363TdTCBCHfEnYJUF+B\\\\\\\/rJMEFGLP4QSxVwUIqX\\\\\\\/2W7n55qAcovJlIEFCA9pN4vOSXhcbxSUrFfEIjpN2YCHyPV88oIdPg+olyk0m6Y4qnTZQ4T9Gcwtxxo9ezN6bOHeoidoVwuxoLttACkAmcQ511\\\\\\\/z8snUrHkgTprJIbEwpK7ulC7sMndjiXKwtwGlWRPpbWeIxScpR2BfNKAgbotOoADatt7ttPvC2wciHSyco\\\\\\\/GyFHVRoj2QjzhVCPg+Q03gVx2UfI9UHCx9DbxoVL9tiHhFpJbMDF7RGrudkt99k+JgEA0M\\\\\\\/UtLXFvRi2WovixgaRE84H69vXBfH8P1a9fGgabhzUebeCjt4aWOS7O9NPHGPYSC8QBu9FDbrkkqMWyyqRrcO2HUlqcnD0lXyp2TSpG3ZuzX\\\\\\\/5EFdRYtI8vs2RaZlfRvBlKZc7e5Za+jjMKRC\\\\\\\/TZwC1BjR6QWbnoRrqHW5Ms3+hj4nNcKi3ZYBOfWHdTDSIpcUr2qm0j9juhz2BDTWMku2zFIh8ArNczL4\\\\\\\/JQ2slLxKFRSB68NTdFeLGGf+ZtzhlNPOYDTB42fkKD\\\\\\\/YNI+824iLlq+sBcaB\\\\\\\/vY49uL0vSyF\\\\\\\/kGPheJYGbcyHfy\\\\\\\/5Egw0AJdakVaDc7vdDWoApwM0nXqpRf76jzNA63q7V8BuDOXPtNVHDpAOoFU+b6qHO6i5RB8uRS37lGWSbhsGkBPFdfl5RuIyyiAjvjf\\\\\\\/H\\\\\\\/ZUAaV6hTzUptwvDgbYm2apF2J9gMQjcovW8CCTDKmj9KZKyUjiRZeaNUJ7V6FCEhPqesvD+C44MWD\\\\\\\/is+dyaj4PaGllBCUJBjQ2jISdZQztLkJhMNhQmnh+TcUgd44Ge9Zqxdhx60Vkx63dJIqexttdjanIFSYEnkZPvTa+hmscAa18N6kfj8Qwk0sKelTwjrGutHiUqpVh8Rul3Zwkk4lzjefaVF9tX6MWtXtiy2N5HUaTeopqmHogEkkaPGUGVh0s5zGvQHMHrbp9gH3T2BAHRe\\\\\\\/QlbBvx0BnoQT6nu2a\\\\\\\/KqtvEkJXYaEwSSZu2AKU7MuSgod2Fc+AjBTMMIrrDlRyGej9HeFt6n27Y\\\\\\\/sxos+0QJCqWtxdgJDBSDnj0murfXsIsum8AG+NE7mMIiAHuhdzwNMsE\\\\\\\/n41GGblnxAb4zkrB2BHl9f8JecAEg9K0W1LZ2vUF1nwemROZTMuwlWRBz+tOx9cRi+pzxD2eZyzd\\\\\\\/fK6jJI\\\\\\\/zF5WmWqW6E24q0g4BVDu8wxaqhV1LoZChkst52IDLDpuu8FS9J71XT04T+MFqXMbdvC04D8LToXwkCQ7Af\\\\\\\/oeI7QrLPhbt0Nffhg9zgwKA6HwPYfR9hGlfOeHqhAiFHXwdYcOpoP1r5XQiBLaxLYj8F9S9oECGVsJMWov6k5xcttqA6KLVbNeFWVSmNYUNqW9zy8MEBozOd5BveMkUGiAeKxBVlAVx2\\\\\\\/G5TSDaqRdYayU6NHSrhfpoEkFTnUJJEHdYNvaoeS07JoPbhFpg1qhYl+0md7HXPEsJgpn3pp3qhI1HGVcit4rb1xcDSTEd\\\\\\\/TK8qtGrmknrGiNqroFk\\\\\\\/fkaIYURFYwYCQsqyRR0Prr2mQhwCf+vzn\\\\\\\/ESsDbZKtB0+iUrJB9h5w8zym1N4SO+lLtRKbsNW1d6fa9Vm92fg1v\\\\\\\/BPVYMcJ7nNmgJ3uiXphuN5yl8BFbPm7Hrd0ynIbKlFBaHrGG53LbE9tiA3YIcCoemtMNlQ1iiUlbq6W0cqm4yHNUbusujnJt7aSdRT8DZrwRNkMsJWU2cT0xSRbD65JpPED2Yi4kGpDBhnpu0K+kF+ScN5caOVBJBriqCCU4Jyt2fGZ6RlSbCuozEvOv5Sd0WDYIHIAXV8anO+OP7IZ0caOcdwtq8p8U0yKDgq0bRgez5geYeu7sff7YqB\\\\\\\/iJ8tjqjCAiN8x9uzlCQrklOrHnc4JEZoeeVZzpxKfCmdXQxSRR176mfbYfcO5wM078R2uWERQUu5gLQ49qUvo8acKbj8H\\\\\\\/6CCFTFHQO8sjy\\\\\\\/Yy3eA+7oBTZBwTskDiEUCwWiUNWwmwBNxRiEgq3AlP9HphIXZl+l3dROkNRFRKoJFhfHDWHJKMF5sHaSLPEkN1nmAY3lPmumi9ix9ipXn2sF+j0Tea2vQBpuIMXwTFUyPFG5TbPa6uOvSRCLXUt8+RSlx7Q6q5zZNDHfGPloN5aZq\\\\\\\/AlmNlWyu05rQa0JZ0d2cd+5\\\\\\\/TAbMJNwyTFypmqr0rY8P+xY2UFrkpRWwseJDTXeupdFYM+twmyNRzUI5OFYhDX4p9IgUd8U53nMXVnyMFlYmcX6tPLLsKb4lR0X\\\\\\\/sQ3zgNU+jOj+yfDpcq\\\\\\\/T4npLRQDiC3BUpTRZgQDFJLsqFDIQNmVjtW9PI2YykbBWHrgVgUv1mWsw9m5hJCqomSHXuStGWNP5YsQptp0MEiu+kQQ7DDPyUtSRhkmGjEMmjlUSTBWJQ3kFEpNp6qU0YeAcRH2wEEeM5mjlRHXgMty2qWjTuV8mh3UdVCG5U9qEWtzTA3Jb1YA26Ektb2JkYZzdtjeK9z7fPqPzrxPatRMH0aQj5eBjNAWn6QkrqFJEYSsjQO5NbD2b8aCl5TPHL+CCPoYdKzd7MDzNu4pE+02aiN\\\\\\\/DN\\\\\\\/qxejnMdIEgd5IshcOyT1Cv3Iwd266I3e+gitBNnE+xzuhAqqRndiVzYZrgfSiVVVSszj6dPA77CVvmc9Ky4gm\\\\\\\/TvQcDiDhf13yEoiEKVwy6Vz4uFhYwWY5n3rM81Lkg\\\\\\\/0fgdcysNepsxewHy+FLd2CILllEXMmlWMYCJPVBXyuSxD3dIhTZXLmVg33ma31m8YpPyGcmf4oj4oxquXb3gZz1h8HiChb9Dk4DW3EPCe5CiFW3gTej6\\\\\\\/Fa+tSga\\\\\\\/FwmTIBMvDuoiI8WO7bakm4hQTZ\\\\\\\/bkAkmFao0JOC9HaFnB4AoFCjYuc3JzhUsdHhfs2FmyiQsd4Qbu\\\\\\\/MFCmNifnHhoD\\\\\\\/xC9pKpoAjKhn8gaT2gP241lMZAhta4uQMCzzKiPUVX36AuYwjVlfTqQWOcIGrKEzlZXolSqCe7OE7T1GpHPNQr7J+ScwlrEhxqms12SLHFnSfc+zqov5OCKlbEaEmJZjaavus\\\\\\\/W0KVR2AiFxBzSCESh3gvm47sL6QUIAbA48C5y9ObqD33cnJABqfjd3OWrYpI+wx1e3da6gk\\\\\\\/EGGpylQlCRnuq3Sfr7x8chLFzRUocpM3x\\\\\\\/YLXQMXsTR9JUOnIm7OJetpfu1Q5ZAa8lXIVxiS483I09FHP\\\\\\\/OOhR1iWfAoC5bD6F4f1T8WQYagx3pRFhZD78EoCJb\\\\\\\/htmmDyWvCQBEYHsiJS2aVsxVN47oQQBb3mrm55d+cKa\\\\\\\/O7cSz7dRZ9QBdvJaJHJbm6qUWks2kpCiGjUpqLoOOM8+8+UW\\\\\\\/6K\\\\\\\/GNJh7joW7TRPMZXZyc25LtICXqkgBH5rnqY2H+MOrtO5BsRyO2D2UcxaQjUItTWoVTJGzPgiVt8xZuIgR6vxHCJlt6aKo1vo3oqYHjClhd6mKJ2h8QX2dLqEa2YUIF231uYjYQ2eZT9xTc7MbzyaYo6bLez9peax6yhd+N4zGlwgnMbPWMD4oxc1cv8FrhtnvmHohp\\\\\\\/k8NPVBWpt7IJDivCNHE3CAMTkACpd78HDmIJpaJ\\\\\\\/EOo\\\\\\\/RgHuVT8orxT+WyDA2iQurzplT308HGJ2vsZ2yNSGUeynPt+GGXLwXTF3eVEMJoUupqrn3jVwSlaVI\\\\\\\/UZIpkEX89d3fiUZfuIdDNJNb6Z3DodClW5gNKWU9RnIO8PUP9Px+dwe6PtkyFjaR2ohy\\\\\\\/rTJiQQW+OMHjuLUtLwB7\\\\\\\/CfGHnQ+o2NEFNBDBKpFzTuip7iaNS9OOhHX0lTSRema8iWiQT\\\\\\\/IX3Gxhh7GoEY9NV20sKUPjw1Y1+ZhRLg5DMXODC4gd9nU3k5djGGZkKXjdX4qr2+Zta1o7HzYxsOU1AYdLlSpvUiOgIm33O0OfTA4X84uVrNNd7NmrrWOFG7JDATF0Cq9ol2bxO1+YUp3WwG5Axc\\\\\\\/qK93M\\\\\\\/TWDPpiT+BI83aCGzpcX\\\\\\\/1LMaA4PMo9FOKFi3uXBFBRKYaFqgYLruAy0FALbLBusD9p3uvNdMmZ7vikegQbMigy6jMmQ+Kmc\\\\\\\/hykmJlbB9xk7GitBlLUQ8kkEbq5ThGVMNkC9iJ1Ui9j5x10Rbl2jWzfsqQiOrrYbZyQMz2EK+EXXNjxt979JLRMG7ZpBR51vTIDA7VT\\\\\\\/i++ywyc+fLJl\\\\\\\/2E5VU+ahUG0Cc5i8Breq4mv5UV6sa1v4XERLA2mwZNLaiz+VYL8bLmd+luxrn2\\\\\\\/wQ4b8OfIKLjyZAjdgaWevD\\\\\\\/6CbSZ9TP8UCMPeMdRPOBTgPvEp5bfcJGb1s\\\\\\\/O1otysXX9+oocvtrZF4PShi2+G7xrcHnebtC\\\\\\\/T3OmduCuI5mH0QI5afufgpeg1c7R23HRxlcAe3lghKXMVRuRA\\\\\\\/xzr8sR+057\\\\\\\/BndwbpZGmSL+Ylz\\\\\\\/KbpkTy3XrXaeo0D1KktC2aUOnkqpjfXttZBH8qcBwFri9k2dzXGviXzzEZ5Dq13k9NAT99ujpA\\\\\\\/8yxhGknHf2j7Ja2KIDJL7F\\\\\\\/\\\\\\\/upXiLpxx8h7Z1RQhs4Nd8JNzBAEzVRw5GAvlYUXrykSUl+q28vW2ZTfRuAcX0wzant9eIgufAV3Fhvl9py0RQNKiWX\\\\\\\/\\\\\\\/wnvJN32PilFiXnd36dJH+GtvrVU5mNmayDr8hAiFzfafw8G\\\\\\\/vK03rT2fWXBbn94u8+FLhz9VDGZ1\\\\\\\/jPk1b9GcvrPWca6jpecyTEYFRURJajT3tpOLUf47rqhk6TC9UkknPWBLXCgZjOlyCGD6+hCCJdw2A2LaaDCsIDi\\\\\\\/cnRPhvxSYUdsEwtn91Og0TuUZ4f7G9EpxcpJkoMknK2as2OLTT3AfB9J5baxy9c7Dcp6F9RNPROWfkcx41jAsv6s2xZpUo8v9NjAD71vO\\\\\\\/5sm89Ine+DrOGTdgUCEby5TCFGfpUlNccm9aszQODlD8SCt7u4EsneuSNlCBzB6ry+SBHM63emkSESS7d3wGNAnGEBYGLYN1mMBHN0fZmDqei350XjCuowZCnoNo+T6eCw8Ue5pmkUQC8tmzJZh+q74TL3sqgFhikWXB39+ueqwD\\\\\\\/EfEz554TM+xpmeOP4gj+S8cxkhwWlSYoDVMs3gAb7qCOKHP7IPdA1JIdwwz6n8uD08FRF\\\\\\\/N3K3rhaXkL2hQu88xX8zeBGL83CMrx2zZjhWLFlIRgP4f7mnwWy3l7JO0uRPoeBcLVUSap0aLHMPNhuRKnL9usGAf0+WBxn\\\\\\\/YJ+DUyPbHGxlnUi2RHp10t6W1gjfe4V\\\\\\\/ATMALu7Hnl9Z0TDCb+ThuxtjyziODFHgakPQaK7lRQf2Sw81sic9GeOBHOy\\\\\\\/mm1PPMpGAowaicnqTxL+6TqrSNHeGiFE9Y8fFqWVAZNmUWwmjiAY0H2iHyXysuGTGR4h75VtGcIg8uqUpCspG8UwAujnqGcVZYtPWp\\\\\\\/6sWnHuREG3lo5GKUNCY9aNPX8ygFsKGrRRJs4VmtG0RcAkJllfwqM\\\\\\\/+PtEvw3GRD4rsdkN14EIERunutLsLA1nowhI1smjDQiL+A+udUtvbQNZ58RtTcMP25sQZ4ZP1cvlconUN8zkbqcJniuQocKaI6rRYDJp3os9RzltVqvVRMWWNPBtBa7FNvctIQ4StRWH4esnbxdmFNSN0tj6uxKWBQWqc+RUAJQpL3tlX+9+6PZ7HaqoiMQV4WGGhoRHU4J0iFymsXgzTvQxS4XCb3HWGUyL3FZBjOzmdK9zUgBON3yKObY2hYM3AMq2jsffh+YdRRhkX7+J\\\\\\\/f\\\\\\\/rFUYZ57LiHUj4KFwHYenXfbOdIsRK5zctcMP\\\\\\\/mWj81kZpbq+uSIsGytdKcuRYUtlrhZ1IiOE7dJSVblbxv5GMMDoMcUnRLakxUjOxUWmDoMJpwwPSF1CPAvDLABvVTh\\\\\\\/PyQGDgCM8KKbRvEFx4WR+blfnNnugeSRoPtU8lVhlPNpdd6Sw9I+WUH2v9\\\\\\\/RazAfKYdBWaQaeT2u0SKQAgzYoIrDIlmN6hF8Q49txekoOF4nY3tIC1XdMkd05ovNctobLA9GWD+NwMKQiTIpCebVu\\\\\\\/MNDQxpztRZGYaf9S+ggyjbtx3A9Dgw8O147lkI9HriojaZIJ+roPsnrBeb8hZXfvPmw8qnB7pq1Vamm7XivO1HRoEWuuzi98ww09BB8Xq5dpl1qdcEYrw3wDi6JIFSuOC9y9+MW5XYaBV26PHM1s81ks+AKHpCLMx9wuiympnnb7nXDaq2XqgUwTD1rn41vUWczvE1j40bOugAaHYcHNoS7IiYsSkoqJrWsuzR08PHvvi4abaoMplJtsFXq5Sp9IMDxedL8TVQROxUFlen4IW44SYL5LiRYRrjW9LlILAl45VsSz3v+8z6hEJsVYuzBpO1hUZ2LizdTE6mMGxP6HcuZ1wzEzVLY60IJYrGNWc3uf3Us0xqz0hxAhTaFuP+7Oj2fD61OKYihRdYvYYW5PMzlh1waHZpmFTbbRW3vdA8Jx8OGoarEFW\\\\\\\/hVNdFQns1hI217RTsv5w9lzAt2eA49vu9gUY5vTvX4omU0xZRq44U7dL+Bj7aYjeQM5Y07eZ5nY4xtRJ6owyE\\\\\\\/MVOf\\\\\\\/FQwYN1TNAQGRODGhIHbGdBTibHdomaFROG9rLjvBeViWwlOd2+AW5106MRLm47yXxP9HzLIDzVvQjlSVekQ8Yn3Yr8lpMYb2kotpdakFCEBW0fi3ZT0eVD8w1JsIoW\\\\\\\/yx9xbgjEpyXx0R3+7X92f1QzgELStO8P7p4eJgVSGTGSl1UsqoqfgRN5N2Z5t3YoPNrbnHqEekHy7dl3NovkkNec46V96CIeew+nfxVMS+sTCRHAhHeSISfAWrOXLJwmPUyA4A3wZQkYffppwvpbEdJtcaiwJU9PtXOGN8f+IEQ5LnSgokwTMqey1+6XRlW7Pj0QFExYyDNItBdr1\\\\\\\/fXLejWBHY5abjGlRm55duzlrwoldeJVSj2KtGtV+VFOqzb+EKlT7uRrbnNn2X8WbOgU4O0F0RmGslY7Su2Lg6Ja2QxlK5u1h7lBzaQSLTMxZZ4Lf3zbBEfN6hhTYQ+ZfFs+FwdfAOlfHNhIbimQX3z5HQ7nEojr\\\\\\\/6ATPIQzXGrP47Q9huVbFTIvde9Wsp6W6mayKrIEHGcrOhClSwoqPgPk00m7LbJkIRFwFLtdeyijH3fxU6Q73CtsOyKXQxMOoJg+5XbmDufsi33TujLTI+3GRLtpkM8DFfk8EDT26kznUDdzWOTCIo2YXnK\\\\\\\/\\\\\\\/u+TnytS8+h61gt3RNbusTg+M8g2GHB5MKQiCx87z4h5Cp8v7eFZ3+VPXqfd3vLWm04czscyNIZS9qtddCbBlfBEdS\\\\\\\/ylH5HvesDtGK4DAQ70jWTtysOPDsiA4uIk64HK+bfxgu81vYRLam02TCuoeR9kygCJ3IwKx32VOztOGEyTitAQHU5WC9y41CycKYZYSx7wo7dp8Pkufw+fGXUo8WVQISG+enOTRj\\\\\\\/qvta69uv3Iaup6L2iWygvU0NukSn0ZJj0gVN4nnld3Vcczse93YXctpuKp7g7gve4H1o9EkW19U9Ev15vLnvsQy\\\\\\\/NBAmGz0QV8cH7R3BWGHzS2TRp2MS4QArSuZrD\\\\\\\/tdC0qpR+SGh1AOlNCJ8PDsAzQ13s8sq+StxAz1GjQdCn\\\\\\\/IE51GbqKURNDPUMj00cWR\\\\\\\/LQgR5gEs+7TsVdIjVJkZGwbHx7Syx\\\\\\\/9t5n9t37J+Warsz0BIkpjspX1tUQYvX2w3brusqUvd8\\\\\\\/tRLOCWrp4t6yuJfT5gwZQs0fBy95hZagz95CLeDne\\\\\\\/wDkwODw2OIiaamlM+yqehxpqhiskjP+vAYqhEENJ2E57XkAsKd0hcKpNZnlbCHYsqj23VBuR89nwynVYHAAgE8fX1nrXuhsj1S\\\\\\\/tRgWVUBoV4E3bH8xaqr7ZDdjfpe0i\\\\\\\/RAeDxFI2yt8qz1M2H\\\\\\\/VjNG0RCtXkeCcrXgWlC32N\\\\\\\/JadvvzA1raHEY\\\\\\\/FLNkeNewD+fLgW7ZnlQow2vGv68LTb0TwstXpussx8E\\\\\\\/W5a9lvuNJH06SrYfEvtrlB74\\\\\\\/qWRzr3VMmiYzB82Qh\\\\\\\/KhQ9Mr0h7nmjsOIFygy1xFoztndzC+ggxfW4qxk6rRjloKSPGgUi9uUKFDh\\\\\\\/MQ96Ks4K3dAzvyK6FfsSdo2b2J4rTk1aHo7NyfrnVIM8yml9\\\\\\\/jAC+FXElgJDIagBHtmcNFtjRCbeJgIpUTRY9vDitO9aLbE10N1HzL\\\\\\\/ANJ\\\\\\\/v0Qj1RYvhONEAaNSF+smMyDMYhALj\\\\\\\/vpKpsdBtrLiSuKzLdR3y3xPOvUaCO06lUReOQHBDvxQohl\\\\\\\/TtE9vpNEJGY+Zl0oSEdjaSGIgLescYpInla+E+X476Xm\\\\\\\/3Dipil5kerpsV3dnkQteQElxrw93525\\\\\\\/ODksjwnLC5iQZECHhl2QiZaw4VvKdRxjHBWjBu9IaTiUAUPpRzGCeRVOrHNCRnilQNoSkzIP85pdStGvtBkLhpHAf8EISja5kAnVjtNZyes6m0d6EBZy2xdxnS7VqVPm+AMxVMnuIPpQ6F9vMjGeFhbTUVHvY7azDHbuixX+X275nxce0InHm76pBFPts2Vz3bK2ZXaNOq9nwdVWDBRAhaijho2hx\\\\\\\/nUntxtAFDtebtcPFUHbB\\\\\\\/uSx83JCUBInQhC8Iz4zKdi1DwS+HZeBEqPir14YfrtakyKXk\\\\\\\/B94TV4b+Tz1BgUQm8H6\\\\\\\/RcZH4ALOWZk\\\\\\\/wLzQusJM9QZX2jM6002HzitH2Y\\\\\\\/7HkdI9uPRsJenzNAcq+K4EIT1O4df\\\\\\\/19GzdLsCeCnjKS40ovDv1HCv5VWkWCa\\\\\\\/BsX7mGg4WP66USkqUOVNvioTyhpGr1MUJzu5r9QMCUZJfDkIidKg1HWkM9ZwLr2tM8xQ8DjE3rPyyPRkJJIXfgoyBth0yYkHHOFT2CxPzKuMi5YaaqH6sKVw6iHmqQbeN1i5tNdRac96gCYSI7tt7m8c\\\\\\\/\\\\\\\/Ki8I3QlH9ldKwW1jDBILBP\\\\\\\/3r+Rmus+kUwdUVCR+nZfr914aVqKwA1ziBOdTjkn55mzWhqZ5GIL8MybyNuhSLlfIjhjEhPf6MLjfnc8HS2MEN3jh3UHe7IJKrSfLZ4w0hoH\\\\\\\/i4Db\\\\\\\/DC6p0gHSdD6SNgBa6Mk03\\\\\\\/ziM15EAldINGBjXeJiiKtjMs2TZUlgEHB\\\\\\\/alvtGKTWRuS9CIWEPfmJfPyS9rDXcwy9Es5s8OOzVSCLLhlFPTyqPcWJnBxfYKYaLJ8qlONPZxv5FqxfrBmH3\\\\\\\/0XPw05fSexC2TWZhmICNCUjTYA1OCiCjGmHEF52UasyUj63LxhQGgZmIHSt+RUP6qLaju9vKxMdH+30da9SRNbY43hUrMdES38446XMsP5bDVW8DVAVzG1ggOfiobbmvWrI4FbZq69zdFwgQo8+1ChegC6Cy0\\\\\\\/v178cWhDFJ8RqTxDOwJ5M7LJ79tDVArt3sPbhL7QRIP57DTb0mLMjLu6Vij9ks18ADf\\\\\\\/MVAnSzU3CyVx2lIIdfPeUsAntFJzt58XPiFtLaDVGOOzQ\\\\\\\/PFeDtSlH42G8RimoROkwt+UozOXFV72\\\\\\\/g0\\\\\\\/o5p\\\\\\\/61vnnChRM7bcqw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DNcnwvTNbJhmISlBuqO\\\\\\\/aQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f3eb7e35122720a01da0674039bde9be7167c2ea4a8648b1ee49ba03f11d2101\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TS1T+EcjYlzrE9Q7ps5CIFGkk3WeYf+BuJuhg2LUz5tfMRL0hWxGZdBHEbHfsBugrQe0Le+HE94cBvVxWZ5XAsjSabI40BO2ye5WATcg8p3mqhVKOhKC9xM8KxOFEwIT5Jly\\\\\\\/ZneRnU15RjyjcTg+I7C6LjpDfeS7JWXLyhko5mootz0LT6UEEyD1\\\\\\\/bl6X0usJ9jzRzEQ5zcYRwXo1+EW4sfNfE26CsBUv2Q\\\\\\\/VB8t69AzPgJM\\\\\\\/XhpuA0UWOktpMxYQiMWnnc+M+5SAXuw3ltX7uJ6QyLnauL2zpgbbdQ0\\\\\\\/02kHlNzah7cSdD4tlU0Py+K9hEmPjHx5pOE1wYUBkFI\\\\\\\/63GpMa7+QLaTQ3Stem6lGmLAG\\\\\\\/ZnwpJ9fOrn4PjCXmLFs2XhaF742aHkRKe7SFfy\\\\\\\/UyUXlfAWVDPtHdfCg4BJpiUYjWfh9Fp8GSumUnltNkFNkLrnW5ksJ66Cm6kMMiJ7S1snMR+R6rhwMtALNFge+7pd+IPaC785LxqNF66nZa+aU6FYAS5uzwg9svOTBsKyI3QrxyisJd8BC6j3AGAE805BBGMsLGQeCqXdGr\\\\\\\/e2M4t9tcmPmtDm6K5RePnsrWPfX\\\\\\\/BAYO+Y+I3Ga7SLUrQa8qpdWHAhQ6oZlbmo5mLobgaz5qIZcrsw6f68eWuzaYz44TEbMI8jaPjMlhikL9dY+WNghYGZqaZyE2ssGRot6GV6uLa3IJIip852j2E4lt7mOaC0m+fq9+FRoyzGEyTM80rbISB\\\\\\\/Hi\\\\\\\/zgBYaNr0hiWY6\\\\\\\/GgKlgJBjYDt1Qu1cl83sSkIlmGKej\\\\\\\/vZvNIlX7k9W8\\\\\\\/Jb1E\\\\\\\/KpZrDwkkW3eD1Y5\\\\\\\/ucEfuSQJgPD3lss8D1+1YVJUVP5hwF+vGJ\\\\\\\/2EojcmiZdtXA\\\\\\\/owpLUpRuqDnUcsAdhOw7kKdi8qwmSs36NYfs9\\\\\\\/KYZKmhxWn7kBmkcuX4Z6NJ8QuvJG+zY6YKTDb6P20\\\\\\\/uad2q6lSblF2CR6rm75tHxCObDes6FPrCJKw3T8o9iSG5fGy9iIxaa4v\\\\\\\/z3SSNM920YvOZ\\\\\\\/TMYxf1LwSQdODnY\\\\\\\/EY7ynCN5YXTymVU+oEpueUieinozekFR50WRALtSeiJ0EoniB5j6bxx3zpbl0+fNHWOOva+6noT3JyVRmNmsDI42oGJCzB1FYC6qTuXyM2BgACU5MXcZywy5zHqGb\\\\\\\/+vB35y77\\\\\\\/6MEnKzwPFHcd4H8b\\\\\\\/KUKyvu567o7dsHL9COyJAyO3i+Z0Yp6nDEoui5c9FmOsOx9yOVI2KM\\\\\\\/eFeUyIYz4sNb+ntlxYW16bQEhQzFUo5VeKU\\\\\\\/H8Bpqbg2BG7nM3JGfKQpeKq1LzYrQO6EONxslWC+OLdA\\\\\\\/WRtUZks7JfTmLZomTXtkmUxmS2kkqy3S1TKX\\\\\\\/nB2WPUPYrbY8Hwyfhi8E2gyIoWXsEhjDUT8RHLOspl6L8iiOqt9peKQuThMjfS8OF9Zrgo3nYv1np6fny1GRA4+LbqQ3RJKnHMNPIdAUawwNltiDazHj0Ab0COEUfxuJkq0OyjIi2JDEuAtFmD5WOqN0anE9ef+O6Zdr1+fkbtWuC2fNrXswRCCI5u1WujRMcSq41TbodZCPFJnWCF0Y+7NxqEW5SX7DKvyV1EpgT851aN200v8WVQJrkfh\\\\\\\/jlBtr5WKES\\\\\\\/pn7oSDs1hw3itClhB423pFfdV07oJOD9yRxdtgKwvsfEBZgH+YeTvd70qG1MbBcXqLRXAKzbqbq+s+7tcDzW9AAxDUfhUlIInPYGZ75RWYMBpZWRBqOhMokfWgaE7vEcBZ8G8YZER+giOjvepVQYSRTzz1tEqzQAY9kFGfL7BuF4lYoxM2jEEh482shmlrXpnxlIbMw520+F0gjRAee3f+6Avnm5vQc89MqIIQEJm9O7F8VMxmjUU\\\\\\\/FJe64lQ\\\\\\\/N34TRK1Vb8NMbfb7FONni7l86GpIELAHDrbgiQKWEsHzwe6N+M2E2mpFhpBDCJDhDe3Gu6HasROAhwizIPdOynVjtCMQkib3YmUtopj4biuh1ptA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZxSqjg9vuEVpaGLMHir5oA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002213e98e6a66dc857c5441af18e5beaf5fc9fcc74feb8b0041a9536d7e6c1e1fbf\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459828 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/717" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mcAVZH6sIl9P_zvNAm70x_-J-pR9zgh8foAkK8nC5S4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/717" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vQm86pj80mYu70wveP8k27-3BOpOKNKlRc0cuo2B0lI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/707" title="Sequence Profiling of the Saccharomyces cerevisiae Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/733" title="Use of an Activated Beta-Catenin to Identify Wnt Pathway Target Genes in Caenorhabditis elegans, Including a Subset of Collagen Genes Expressed in Late Larval Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459828" id="node459828" data-pisa="ggg;4/4/717" data-pisa-master="ggg;g3.113.009597" data-apath="/ggg/4/4/717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent <em>Haemophilus influenzae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joshua Chang Mell</span>, <span class="highwire-citation-author" data-delta="1">Jae Yun Lee</span>, <span class="highwire-citation-author" data-delta="2">Marlo Firme</span>, <span class="highwire-citation-author" data-delta="3">Sunita Sinha</span> and <span class="highwire-citation-author" data-delta="4">Rosemary J. Redfield</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">717-731;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009597 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459828" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua Chang Mell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BChang%2BMell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mell%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BChang%2BMell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:joshua.mell@DrexelMed.edu" class="" data-icon-position="" data-hide-link-title="0">joshua.mell@DrexelMed.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jae Yun Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Sciences and Technology Graduate Program, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jae%2BYun%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20JY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJae%2BYun%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marlo Firme </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Sciences and Technology Graduate Program, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marlo%2BFirme%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Firme%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarlo%2BFirme%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sunita Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pharmacy Sciences, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunita%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASunita%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rosemary J. Redfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosemary%2BJ.%2BRedfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Redfield%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARosemary%2BJ.%2BRedfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/4/717" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459828" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/717.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459828" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/717.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459828" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/717.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459828" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/4/717.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459828" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Naturally competent bacterial species actively take up environmental DNA and can incorporate it into their chromosomes by homologous recombination. This can bring genetic variation from environmental DNA to recipient chromosomes, often in multiple long “donor” segments. Here, we report the results of genome sequencing 96 colonies of a laboratory <em>Haemophilus influenzae</em> strain, which had been experimentally transformed by DNA from a diverged clinical isolate. Donor segments averaged 6.9 kb (spanning several genes) and were clustered into recombination tracts of ~19.5 kb. Individual colonies had replaced from 0.1 to 3.2% of their chromosomes, and ~1/3 of all donor-specific single-nucleotide variants were present in at least one recombinant. We found that nucleotide divergence did not obviously limit the locations of recombination tracts, although there were small but significant reductions in divergence at recombination breakpoints. Although indels occasionally transformed as parts of longer recombination tracts, they were common at breakpoints, suggesting that indels typically block progression of strand exchange. Some colonies had recombination tracts in which variant positions contained mixtures of both donor and recipient alleles. These tracts were clustered around the origin of replication and were interpreted as the result of heteroduplex segregation in the original transformed cell. Finally, a pilot experiment demonstrated the utility of natural transformation for genetically dissecting natural phenotypic variation. We discuss our results in the context of the potential to merge experimental and population genetic approaches, giving a more holistic understanding of bacterial gene transfer.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/bacteria" class="hw-term hw-article-keyword hw-article-keyword-bacteria" rel="nofollow">bacteria</a></li><li class="kwd"><a href="/keyword/horizontal-gene-transfer" class="hw-term hw-article-keyword hw-article-keyword-horizontal-gene-transfer" rel="nofollow">horizontal gene transfer</a></li><li class="kwd"><a href="/keyword/heteroduplex-segregation" class="hw-term hw-article-keyword hw-article-keyword-heteroduplex-segregation" rel="nofollow">heteroduplex segregation</a></li><li class="kwd"><a href="/keyword/nearly-isogenic-lines" class="hw-term hw-article-keyword hw-article-keyword-nearly-isogenic-lines" rel="nofollow">nearly isogenic lines</a></li></ul><p id="p-5">Many bacterial species are naturally competent, able to take up DNA from their environment and incorporate it into their chromosome by homologous recombination (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen and Dubnau 2004</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Claverys <em>et al.</em> 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Johnsborg <em>et al.</em> 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kruger and Stingl 2011</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Thomas and Nielsen 2005</a>). This natural transformation is the major pathway of genetic transfer between related competent lineages, and it has had a profound influence on bacterial evolution, transferring both allelic variation and whole loci between lineages, similar to the genetic shuffling generated during eukaryotic sexual reproduction (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Moradigaravand and Engelstadter 2013</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Vos and Didelot 2009</a>).</p><p id="p-6">Comparing DNA sequences from natural bacterial isolates has found that intraspecific recombination is often common, especially in species known to be naturally competent (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Didelot and Maiden 2010</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Feil <em>et al.</em> 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Feil and Spratt 2001</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Perez-Losada <em>et al.</em> 2006</a>). In pathogens these recombination events appear to have promoted adaptation to host defenses and antibiotic resistance (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Croucher <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hanage <em>et al.</em> 2009</a>). Interpreting this historical evidence, however, first requires disentangling the immediate consequences of recombination from the other evolutionary forces that act on recombinant genomes, such as mutation, selection, and population substructure (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Feil and Spratt 2001</a>). To do this, we need to first understand how the mechanism of natural transformation determines the distribution of donor genetic variation brought into recipient chromosomes.</p><p id="p-7">In contrast to the recombination that repairs DNA double-stranded breaks (for example, that arising during DNA replication and meiotic recombination), the donor DNAs in natural transformation are transported into the cytoplasm as single strands with their 3′-end leading through a conserved membrane pore (Rec2/ComEC), whereas the complementary strands are simultaneously degraded. If translocated single-stranded DNA (ssDNA) has high sequence similarity to a segment of the competent cell’s chromosome, it can undergo recombination once coated with the strand exchange ATPase RecA; otherwise, translocated DNAs also are degraded. Thus transformational recombination occurs without the formation or resolution of Holliday junctions, since the substrates are ssDNA donors and dsDNA recipients (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Claverys <em>et al.</em> 2009</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kruger and Stingl 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Maughan <em>et al.</em> 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mortier-Barriere <em>et al.</em> 2007</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pifer and Smith 1985</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model of transformation and possible outcomes of heteroduplex segregation for two independent donor DNAs (blue) in the absence of heteroduplex correction. One fragment carries a selectable antibiotic resistance allele (R); the other is unselected (U). Strands that go on to produce a resistant colony are indicated with an asterisk (*). (A) The U fragment replaces resident sequences on the same strand as the R fragment, leading to a resistant colony homogeneous for the R and U recombination tracts. (B) The U fragment replaces the opposite strand as the R fragments, leading to a resistant colony with only the R tract. (C) and (D) are as in (A), except that the two donor molecules transform asynchronously with respect to segregation. This could be either temporal (one fragment transformed before replication and the other after) or spatial (the replication fork had passed through one locus but not the other when the fragments transformed). (C) The R fragment transforms first, leading to a colony either heterogeneous for the U fragment (shown by the blue circle) or missing the U fragment, depending on the strands transformed. (D) The U fragment transforms first, leading to a resistant colony homogeneous for either both fragments or only the R fragment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Model of transformation and possible outcomes of heteroduplex segregation for two independent donor DNAs (blue) in the absence of heteroduplex correction. One fragment carries a selectable antibiotic resistance allele (R); the other is unselected (U). Strands that go on to produce a resistant colony are indicated with an asterisk (*). (A) The U fragment replaces resident sequences on the same strand as the R fragment, leading to a resistant colony homogeneous for the R and U recombination tracts. (B) The U fragment replaces the opposite strand as the R fragments, leading to a resistant colony with only the R tract. (C) and (D) are as in (A), except that the two donor molecules transform asynchronously with respect to segregation. This could be either temporal (one fragment transformed before replication and the other after) or spatial (the replication fork had passed through one locus but not the other when the fragments transformed). (C) The R fragment transforms first, leading to a colony either heterogeneous for the U fragment (shown by the blue circle) or missing the U fragment, depending on the strands transformed. (D) The U fragment transforms first, leading to a resistant colony homogeneous for either both fragments or only the R fragment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F1.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/4/717/F1.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464130" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Model of transformation and possible outcomes of heteroduplex segregation for two independent donor DNAs (blue) in the absence of heteroduplex correction. One fragment carries a selectable antibiotic resistance allele (R); the other is unselected (U). Strands that go on to produce a resistant colony are indicated with an asterisk (*). (A) The U fragment replaces resident sequences on the same strand as the R fragment, leading to a resistant colony homogeneous for the R and U recombination tracts. (B) The U fragment replaces the opposite strand as the R fragments, leading to a resistant colony with only the R tract. (C) and (D) are as in (A), except that the two donor molecules transform asynchronously with respect to segregation. This could be either temporal (one fragment transformed before replication and the other after) or spatial (the replication fork had passed through one locus but not the other when the fragments transformed). (C) The R fragment transforms first, leading to a colony either heterogeneous for the U fragment (shown by the blue circle) or missing the U fragment, depending on the strands transformed. (D) The U fragment transforms first, leading to a resistant colony homogeneous for either both fragments or only the R fragment.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Unless mismatch repair intervenes, the resulting heteroduplex recombination products (genetically equivalent to gene conversion intermediates) will simply segregate upon DNA replication to give rise to two homoduplex daughter molecules, one carrying donor alleles and the other unchanged from the recipient. This heteroduplex segregation can cause single cells transformed by more than one donor DNA molecule to give rise to genetically heterogeneous daughter cells (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, A and B</a>). Such heterogeneity would be even more frequent when independent DNAs transform recipient chromosomes asynchronously with respect to DNA replication, with doubly transformed chromosomes giving rise to potentially &gt;2 distinct genotypes, depending on which strand of each donor molecule had been translocated into the cytoplasm (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, C and D</a>).</p><p id="p-10">Further complicating the picture, previous studies of natural transformation have shown that contiguous runs of donor-specific variation (“donor segments”) often are clustered into complex mosaic patterns, suggesting that long DNA molecules frequently are disrupted after uptake. This could be due to either endonucleases acting on DNA prior to strand exchange or heteroduplex correction mechanisms acting after recombination products are formed (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lin <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>).</p><p id="p-11">Although recombination tracts are limited by the size distribution of donor DNA fragments, they can span many genes and typically include regions of both high and low nucleotide divergence between donor and recipient (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lin <em>et al.</em> 2009</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>). Further, structural variants (SVs; insertions, deletions, and other rearrangements) can cotransform with flanking markers as parts of longer recombination tracts, though they appear to transform less readily than single-nucleotide variants (SNVs) (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Stuy and Walter 1981</a>).</p><p id="p-12">Previous experimental work has shown that naturally competent cells commonly take up and recombine more than one independent DNA molecule. This permits use of cotransformation frequencies to investigate properties of the competent cell population. Such analysis frequently finds unexpectedly high cotransformation frequencies of markers on independent molecules (“congression”), indicating that only a fraction of cells in a competent culture take up and recombine DNA (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Goodgal and Herriott 1961</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>; but see <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Erickson and Copeland 1973</a>). Although congression suggests that competence may be an on/off cellular state, it could also reflect more continuous variation in the transformability of competent cells (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Maamar <em>et al.</em> 2007</a>).</p><p id="p-13">Experimental studies of natural transformation have typically focused on the extent of recombination around selectable antibiotic resistance markers, but the advent of inexpensive DNA sequencing allows investigation of its genome-wide effects (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Power <em>et al.</em> 2012</a>). Wild strains typically differ at thousands of positions, and recombinants can be comprehensively genotyped by genome sequencing, thereby identifying recombinant segments with high precision. Such recombinants also more closely mirror the most interesting natural conditions, where diverged strains in polyclonal populations can generate complex recombinants. Eventually, such work will contribute to a synthesis between experimental studies of recombination and evolutionary studies of its consequences.</p><p id="p-14">Here we report work using <em>Haemophilus influenzae</em>, a model system for natural competence and an important member of the human microbiome. <em>H. influenzae</em> is primarily found as a commensal of the upper respiratory tract in children but is also often an agent of serious disease, especially in patients with respiratory conditions such as cystic fibrosis and chronic obstructive pulmonary disease (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agrawal and Murphy 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dworkin <em>et al.</em> 2007</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">MacNeil <em>et al.</em> 2011</a>). We previously reported the use of genome sequencing to identify all the genetic changes in four <em>Haemophilus influenzae</em> clones transformed by DNA from a divergent strain (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>). The clones had replaced 1–3% of their chromosomes with donor DNA at independent locations, underlining the genetic consequences of transformation for single competent cells. However, the number of clones investigated was too small for statistical analysis of either transformation’s overall extent in individual cells or its distribution around the chromosome. The work here updates our comprehensive genotyping approach and extends genome-wide analysis to a larger set of 96 transformed chromosomes, thereby providing parameter estimates that will be useful for inferring historical recombination in natural populations. We also show that natural transformation offers a potentially powerful approach to mapping the underlying genetic causes of natural phenotypic variation in competent bacterial species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Collection of transformed clones</h3><p id="p-15"><em>Haemophilus influenzae</em> strains are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and detailed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/009597SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS2.zip">File S2</a>. Culturing and manipulation of <em>H. influenzae</em> were performed via standard methods (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Poje and Redfield 2003a</a>). Competent cells of the Rd strain (KW20, RR722) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fleischmann <em>et al.</em> 1995</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wilcox and Smith 1975</a>) were prepared by transfer of exponentially growing cells to MIV starvation medium, as described (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Poje and Redfield 2003b</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Steinhart and Herriott 1968</a>) and incubated with genomic DNA extracted from donor strains [~1 genome equivalent per cell, ~2 µg/10<sup>9</sup> colony-forming units (CFUs) in 1 mL] at 37° for 30 min. Cultures were then treated with DNase I (1 µg/mL) for 10 min to degrade extracellular donor fragments, followed by 1:5 dilutions into the rich medium sBHI (Difco brain-heart infusion supplemented with 10μg/ml hemin and 2μg/ml NAD) and incubation at 37° for 80 min to allow expression of the Nal<sup>R</sup> resistance allele. A late-log transformation was conducted in the same manner, except using a culture grown in sBHI that had reached stationary phase (OD<sub>600</sub> = 1.2).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464230/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464230/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Transformation experiments and clone isolation</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">Donor DNA was either from strain RR3131, a Nov<sup>R</sup>Nal<sup>R</sup> derivative of the nontypeable clinical isolate 86−028NP (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Harrison <em>et al.</em> 2005</a>; <a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>) or the control MAP7 strain (RR666), a multiantibiotic resistant derivative of the recipient Rd strain (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Catlin <em>et al.</em> 1972</a>). In both cases, the Nov<sup>R</sup> and Nal<sup>R</sup> single-nucleotide markers (in the <em>gyrB</em> and <em>gyrA</em> genes) were &gt;750 kb apart and hence always carried on different molecules in the donor DNA preparations.</p><p id="p-22">Transformed cultures were diluted and plated onto sBHI agar ± antibiotics (2.5 µg/mL for novobiocin and 3 µg/mL for nalidixic acid) to measure transformation frequencies, calculated as Nov<sup>R</sup> or Nal<sup>R</sup> antibiotic-resistant colonies per CFU. For MIV cultures, observed Nov<sup>R</sup>Nal<sup>R</sup> cotransformation frequencies were compared with those expected if all cells were equally competent, (Nov<sup>R</sup>/CFU) * (Nal<sup>R</sup>/CFU), providing estimates of the fraction of competent cells in the culture if competence is a binary on/off cellular state (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Goodgal and Herriott 1961</a>; <a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>).</p><p id="p-23">To permit detection of heteroduplex segregation that had occurred within the original transformed cells, antibiotic-resistant colonies were (a) not restreaked prior to inoculation for sequence analysis and (b) picked from plates with &lt;30 colonies to reduce the chance that individual colonies arose from more than one cell. Picked colonies were inoculated into 5-mL sBHI cultures overnight; 1 mL of each resulting culture was stored in 15% glyercol at −80°, and the remaining 4 mL was centrifuged at 4000<em>g</em> for 5 min to collect cell pellets for DNA extracts.</p></div><div id="sec-3" class="subsection"><h3>DNA extractions</h3><p id="p-24">Cell pellets were resuspended in 500 µL of resuspension buffer (50 mM Tris-HCl, 50mM EDTA, pH 8), lysed by adding lysis buffer (final concentrations of 1% sodium dodecyl sulfate and 200 µg/mL proteinase K), then incubated at 65° for 60 min. Potassium acetate (pH 8) was added to a final concentration of 1.25M, and the mixtures were centrifuged at 10,000<em>g</em> at 4° for 10 min to remove cell debris. Supernatants were precipitated in 1 vol of isopropanol, and pellets were then washed in 70% ethanol, resuspended in 500 mL of TE (10mM Tris-HCl, 1 mM EDTA, pH 8) + 20 µg/mL RNase A, and incubated for 1 hr at 37°. Resuspensions were extracted 2× in 1 vol phenol/chloroform/isoamyl alcohol (24:23:1). DNA was precipitated from the aqueous fractions by addition of 0.1 vol of 3M sodium acetate and 2 vols of 100% ethanol, followed by centrifugation, washing in 70% ethanol for 10 min, and resuspension in 50 µL TE. This typically yielded ~15−20 µg of pure DNA fragments. DNAs were checked for purity and yield by spectroscopy (NanoDrop 1000) and agarose gel electrophoresis, diluted to 100 ng/µL, and arrayed into a 96-well plate for sequencing. DNAs extracted from parental strains RR722, RR3131, and RR666 were included as controls.</p><p id="p-25">A total of 99 genomic DNAs were extracted (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>): three control strains, 36 Nov<sup>R</sup> and 36 Nal<sup>R</sup> colonies grown from three independent MIV transformations by RR3131 donor DNA, eight Nov<sup>R</sup> colonies grown from a late-log transformation, and 16 unselected Nov<sup>S</sup> Nal<sup>S</sup> colonies grown from MIV transformations. Because these last 16 genomic DNAs were expected to be mostly untransformed, they were pooled 1:1 in pairs prior to library construction, for a total of 91 DNA samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS2.zip">File S2</a>).</p></div><div id="sec-4" class="subsection"><h3>DNA sequencing and data processing</h3><p id="p-26">Methods for sequencing, alignments, and variant calling were a modification of our previous approach (<a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>), and are detailed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS1.pdf">File S1</a>. In brief, sequence reads from each DNA sample were aligned to both donor and recipient genome references, followed by re-alignment of reads with short indels (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li and Durbin 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Quinlan and Hall 2010</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Thorvaldsdottir <em>et al.</em> 2013</a>). Multisample variant calls were generated from the two sets of read alignments, followed by application of stringent filters to define a set of “gold-standard” variants that reliably distinguished donor from recipient—using either reference genome—for both SNVs and SVs (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Filtering comprised cross-validation between the two control datasets (donor and recipient) and both corresponding reference sequences, as well as exclusion of error-prone positions arising due to systematic sequencing and alignment errors. The filtering also excluded novel alleles absent from either control dataset; such putative new mutations are considered further below. For each sequenced DNA, the genotype at each variant position in the gold standard set was scored as recipient, donor, or mixed (when both alleles were observed). “Donor segments” were defined as contiguous runs of positions with donor-specific variants. Donor segments containing only a single mixed variant (n = 18) were excluded to eliminate probable sequencing artifacts. “Recombination tracts” were defined as clusters of donor segments whose outermost end points fell within 100 kb. A complete listing of all donor segments and their putative clustering is in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS3.zip">File S3</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464226/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464226/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Sequencing statistics</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464123/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464123/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">SNVs distinguishing donor from recipient genomes—Filtering and cross-validation</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464180/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464180/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">SVs distinguishing donor from recipient—Filtering and cross-validation</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Data analysis and statistics</h3><p id="p-54">Plotting, curve fitting, and linear modeling used the R statistical programming language, including the fitdistrplus package. The Kolmogorov-Smirnov statistic (<em>D</em>) was used to evaluate goodness-of-fit (K-S test). Permutation tests were used to generate random distributions of donor segment locations in order to calculate the probability of overlapping donor segments between different clones. Differences between the size of breakpoint intervals and genome-wide SNP spacing were evaluated by K-S test. Fisher’s exact tests were used to compare transforming insertions to transforming deletions and to compare breakpoint classes with their expected distributions. Student’s <em>t</em>-tests (two-tailed, paired) were used to evaluate differences in transformability between strains.</p></div><div id="sec-6" class="subsection"><h3>Polymerase chain reaction (PCR) validation of donor segments in DNAs with mixed genotypes</h3><p id="p-55">For one of the samples with two mixed segments (RR4074), allele-specific PCR was used to confirm that the original sequenced DNA sample contained distinct cell populations with either the donor or recipient versions of both segments. The stored strain was streaked and individual colonies containing each of the two genotypes were identified by colony PCR. Because RR4107 and RR4108 were sequenced as a pool, PCR was also used to confirm which clone contained the observed recombination tract. Primers used are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-7" class="subsection"><h3>Mapping a transformation frequency modifier</h3><p id="p-56">Each of the 96 sequenced strains, along with donor and recipient controls, was initially assayed for transformability using a “transformation-during-growth assay,” in which exponentially growing cultures at OD<sub>600</sub> = 0.2 were diluted 1/100 in sBHI containing 10 µg/mL MAP7 DNA, followed by incubation at 37° for 8 hr, dilution, and plating onto sBHI agar ± kanamycin (7 µg/mL). Candidate recombinants, including RR4108, were reassayed with the MIV starvation method. Radiolabeled DNA uptake assays were performed as described (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Maughan and Redfield 2009a</a>).</p><p id="p-57">Deletions of the <em>comM</em> gene (HI1117) were generated in three distinct strain backgrounds (RR722, RR3131, and RR4108). Construction of the RR3131 <em>comM</em>::spec and RR4108 <em>comM</em>::spec strains used transformation by a 4.25-kb amplicon across the <em>comM</em>::spec allele generated from a previously characterized <em>comM</em>::spec deletion made in the RR722 background (RR3121)(<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sinha <em>et al.</em> 2012</a>) (primers <em>comM</em>-F and <em>comM</em>-R in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/TableS1.pdf">Table S1</a>). Mutants were selected on 200 µg/mL spectinomycin and confirmed by PCR and Sanger sequencing.</p><p id="p-58">The Rd and 86-028NP alleles of <em>comM</em> also were cloned into the <em>Eco</em>RI site of the low-copy vector pSU20 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bartolome <em>et al.</em> 1991</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Poje and Redfield 2003a</a>) with the native promoter to generate pRd<em>comM</em> and pNP<em>comM</em>; the gene was amplified with ~700-bp flanks from the appropriate genomic DNA using primers <em>comM</em>_<em>Eco</em>RI_F and <em>comM</em>_<em>Eco</em>RI_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/TableS1.pdf">Table S1</a>), and electroporated into the RR722 <em>comM</em>::spec and RR4108 <em>comM</em>::spec strains. Complementation assays of the resulting strains were carried out with wild-type controls, measuring the Kan<sup>R</sup> transformation frequencies from MIV-induced cultures. The gene map was made with the assistance of GenoPlotR (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guy <em>et al.</em> 2010</a>).</p></div><div id="sec-8" class="subsection"><h3>Data deposition</h3><p id="p-59">Sequence data for each sample were deposited—to the NCBI short-read archive under project accession SRP036875—as BAM files reporting alignments to the Rd (KW20) genome reference, with filename prefixes according to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS2.zip">File S2</a>. Deposited read alignments (Illumina 8.1+ quality encoding) were purged of paired reads failing Illumina chastity filters or containing adaptor sequences, but unaligned reads were retained to allow realignment to the 86-028NP donor genome reference.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Summary of recombinant clone isolation and genome sequencing</h3><p id="p-60"><em>Haemophilus influenzae</em> recipient cultures of the lab strain Rd were transformed with donor genomic DNA from a derivative of the clinical isolate 86-028NP carrying two antibiotic resistance markers (Nov<sup>R</sup> and Nal<sup>R</sup>). The divergence between the strains provides a high density of genetic markers; the donor and recipient genomes differ at 2.34% of alignable nucleotide positions (at SNVs), and ~10% of each genome is absent from the other (at indels and other rearrangements, <em>i.e.</em>, SVs), typical for a pair of <em>H. influenzae</em> isolates (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Harrison <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hogg <em>et al.</em> 2007</a>; <a id="xref-ref-37-10" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>).</p><p id="p-61">Colonies were propagated from four transformation experiments (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), and their purified DNA was short-read sequenced on an Illumina HiSeq to give nearly complete identification of every nucleotide in each genomic DNA sample (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FigureS1.pdf">Figure S1</a>). Median read depth per position was ~50-fold, and although variation in read depth along the genome was high, this variation was consistent between libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FigureS2.pdf">Figure S2</a>). Short-read sequence datasets were processed using an improved and expanded version of our previous approach (<a id="xref-ref-37-11" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>), which now incorporates genotype calling at SVs, as detailed in the section <em>Materials and Methods</em>. This provided a comprehensive genotype table for all the sequenced chromosomes at 35,063 SNVs and 868 SVs, 121 of which affected ≥100 bp (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p></div><div id="sec-11" class="subsection"><h3>Global pattern of transformation in selected recombinants</h3><p id="p-62">Three of the four transformation experiments used the standard competence-induction protocol, in which exponentially growing cells are transferred to the starvation medium MIV to induce maximal competence, followed by incubation with donor genomic DNA (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). From each transformation, 12 Nov<sup>R</sup> and 12 Nal<sup>R</sup> recombinant colonies were propagated for sequencing (72 in all). The antibiotic resistance selection ensured that each sequenced clone had acquired at least one donor segment, precluding inadvertent resequencing of untransformed recipient chromosomes.</p><p id="p-63">Modeling the process of natural transformation for population genetic studies requires that the statistical distribution of recombination events be determined. The 72 selected recombinants carried extensive donor genetic variation in their chromosomes in 216 long contiguous segments, with each clone carrying at least one donor segment spanning the expected antibiotic resistance allele (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS3.zip">File S3</a>). The total extent of recombination per clone was consistent with a lognormal distribution (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>; K-S test for total donor SNVs: <em>D</em> = 0.073, <em>p</em>-value = 0.84; and for total kb replaced: <em>D = 0.049</em>, <em>p</em>-value = 0.99). Individual recombinant chromosomes contained on average 394 donor SNVs in 1−10 donor segments, collectively replacing between 2.7 and 58.5 kb of the recipient chromosome, or on average 1.13% of genome length. This overall extent of recombination was approximately twofold higher (mean 1.13% genome length) than that predicted from the transformation frequencies of the Nov<sup>R</sup> and Nal<sup>R</sup> markers alone (mean 0.58% per CFU, <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), suggesting that these particular markers might transform at lower rates than the genome average.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Extent of recombination in 96 transformed colonies. Each row indicates a genome, with coordinates are according to the recipient Rd genome sequence. Blue hatches indicate donor-specific SNVs. Turquoise hatches indicate a mixture of donor- and recipient-specific SNVs. (A) 36 NovR-selected recombinants from MIV transformations. (B) 36 NalR-selected recombinants from MIV transformations. (C) Eight pools of two unselected clones from MIV transformations (such that turquoise hatches indicate donor variation in one of the two pooled clones). *Indicates the recombination tract in RR4108 that confers altered transformability (see text). (D) Eight NovR-selected recombinants from a late-log transformation. SNV, single-nucleotide variant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Extent of recombination in 96 transformed colonies. Each row indicates a genome, with coordinates are according to the recipient Rd genome sequence. Blue hatches indicate donor-specific SNVs. Turquoise hatches indicate a mixture of donor- and recipient-specific SNVs. (A) 36 Nov&lt;sup&gt;R&lt;/sup&gt;-selected recombinants from MIV transformations. (B) 36 Nal&lt;sup&gt;R&lt;/sup&gt;-selected recombinants from MIV transformations. (C) Eight pools of two unselected clones from MIV transformations (such that turquoise hatches indicate donor variation in one of the two pooled clones). *Indicates the recombination tract in RR4108 that confers altered transformability (see text). (D) Eight Nov&lt;sup&gt;R&lt;/sup&gt;-selected recombinants from a late-log transformation. SNV, single-nucleotide variant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F2.medium.gif" width="292" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/4/717/F2.medium.gif" width="292" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464178" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-64" class="first-child">Extent of recombination in 96 transformed colonies. Each row indicates a genome, with coordinates are according to the recipient Rd genome sequence. Blue hatches indicate donor-specific SNVs. Turquoise hatches indicate a mixture of donor- and recipient-specific SNVs. (A) 36 Nov<sup>R</sup>-selected recombinants from MIV transformations. (B) 36 Nal<sup>R</sup>-selected recombinants from MIV transformations. (C) Eight pools of two unselected clones from MIV transformations (such that turquoise hatches indicate donor variation in one of the two pooled clones). *Indicates the recombination tract in RR4108 that confers altered transformability (see text). (D) Eight Nov<sup>R</sup>-selected recombinants from a late-log transformation. SNV, single-nucleotide variant.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary histograms of transformation in the 72 selected colonies. (A) Total donor SNVs; (B) combined length of donor segments in each clone (measured from outermost donor-specific SNVs aligned to the recipient genome); (C) length of individual donor segments; (D) same as in (C), but with segment length on a log-scale and the 11 donor segments containing only one SNV are indicated with an arrow. Best fits are shown as red lines in (A&#x2212;C) used distributions that best fit the data, as determined using the R package fitdistrplus [log-normal for (A&#x2212;B) and exponential for (C)]. &#x201C;Mixed&#x201D; segments were included in these measurements. SNV, single-nucleotide variant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary histograms of transformation in the 72 selected colonies. (A) Total donor SNVs; (B) combined length of donor segments in each clone (measured from outermost donor-specific SNVs aligned to the recipient genome); (C) length of individual donor segments; (D) same as in (C), but with segment length on a log-scale and the 11 donor segments containing only one SNV are indicated with an arrow. Best fits are shown as red lines in (A&#x2212;C) used distributions that best fit the data, as determined using the R package fitdistrplus [log-normal for (A&#x2212;B) and exponential for (C)]. &#x201C;Mixed&#x201D; segments were included in these measurements. SNV, single-nucleotide variant.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F3.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/4/717/F3.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464214" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-65" class="first-child">Summary histograms of transformation in the 72 selected colonies. (A) Total donor SNVs; (B) combined length of donor segments in each clone (measured from outermost donor-specific SNVs aligned to the recipient genome); (C) length of individual donor segments; (D) same as in (C), but with segment length on a log-scale and the 11 donor segments containing only one SNV are indicated with an arrow. Best fits are shown as red lines in (A−C) used distributions that best fit the data, as determined using the R package fitdistrplus [log-normal for (A−B) and exponential for (C)]. “Mixed” segments were included in these measurements. SNV, single-nucleotide variant.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Donor segment properties</h3><p id="p-66">The average length of the 216 donor segments was 6.9 kb (median 4.8 kb), when defined by their outermost donor-specific variants (breakpoint intervals themselves are analyzed below). Segment lengths followed an exponential distribution (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>; K-S test for all donor segments: <em>D</em> = 0.076, <em>p</em>-value = 0.16), with the largest segment 43.1 kb long, or 2.4% of the genome length,. The exponential fit to the distribution of segment lengths was improved by excluding the eleven outliers that contained only a single SNV (indicated by the arrow in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3D</a>; K-S test for donor segments containing &gt;1 SNV: <em>D</em> = 0.053, <em>p</em>-value = 0.62), suggesting that these arise by a distinct process. These and other very short donor segments are discussed below as potential remnants of donor-directed mismatch repair. These results with <em>H. influenzae</em> contrast with the shorter recombinant segments detected in <em>Streptococcus pneumoniae</em> and <em>Helicobacter pylori</em> transformants (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Lin <em>et al.</em> 2009</a>). This suggests that mechanistic differences in uptake (or subsequent steps) might favor recombination of longer segments in <em>H. influenzae</em>, although importantly, variation in average segment length within species has not been evaluated.</p><p id="p-67">In 8 colonies, 13 “mixed” donor segments were observed where the base at each SNV was supported by sequence reads of both donor and recipient alleles (to exclude sequencing artifacts, “mixed” segments had to consist of &gt;2 SNVs) (turquoise segments in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, A and B</a>, and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). These mixed segments were contiguous runs of 9−157 variant positions with mixed donor/recipient base calls (0.9−13.7 kb), which could not have arisen by sequencing error or cross-contamination of DNA samples. Instead they most likely arose by heteroduplex segregation, as discussed below.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative SNV transformation. Count of donor-specific (dark blue) and mixed (turquoise) SNVs across the 72 selected transformants illustrated in Figure 2, A and B (y-axis) using the recipient genome coordinate (x-axis). Bottom pink track shows all 35,063 SNV markers used to distinguish donor from recipient. Regions with low marker density correspond to recipient-specific insertions, e.g., a recipient-specific prophage insertion at ~1600 kb. SNV, single-nucleotide variant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cumulative SNV transformation. Count of donor-specific (dark blue) and mixed (turquoise) SNVs across the 72 selected transformants illustrated in Figure 2, A and B (y-axis) using the recipient genome coordinate (x-axis). Bottom pink track shows all 35,063 SNV markers used to distinguish donor from recipient. Regions with low marker density correspond to recipient-specific insertions, &lt;em&gt;e.g.&lt;/em&gt;, a recipient-specific prophage insertion at ~1600 kb. SNV, single-nucleotide variant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F4.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/4/717/F4.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464185" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-68" class="first-child">Cumulative SNV transformation. Count of donor-specific (dark blue) and mixed (turquoise) SNVs across the 72 selected transformants illustrated in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, A</a> and B (y-axis) using the recipient genome coordinate (x-axis). Bottom pink track shows all 35,063 SNV markers used to distinguish donor from recipient. Regions with low marker density correspond to recipient-specific insertions, <em>e.g.</em>, a recipient-specific prophage insertion at ~1600 kb. SNV, single-nucleotide variant.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Total recombination across the selected recombinants</h3><p id="p-69">Transformation at unselected sites was sufficiently high that—considering all 72 recombinants—29.8% of all “gold-standard” donor SNVs were present in at least one recombinant (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), predicting that a set of several hundred independent recombinants could potentially contain most of the donor-specific variation in short discrete intervals.</p><p id="p-70">No obvious transformation hotspots were observed; two overlapping segments were seen at only six intervals distant from the selected sites (see below), and three overlaps at only one interval (at ~1150 kb in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). As a control, random shuffling of donor segment locations found 3 overlapping segments in &lt;5% of 1000 permutations. However, a long interval devoid of donor segments was observed at ~1500−1700 kb, possibly indicating a transformation coldspot (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Notably, this interval is roughly opposite the predicted origin of replication at ~603 kb (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Salzberg <em>et al.</em> 1998</a>). These results do not rule out variation along the genome in transformation frequency (as observed in <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Ray <em>et al.</em> 2009</a>), since the power to detect hotspots and coldspots in this study was limited by the number of clones sequenced.</p></div><div id="sec-14" class="subsection"><h3>Local pattern of recombination around the selected sites</h3><p id="p-71">Selection for a single donor SNV (Nov<sup>R</sup> or Nal<sup>R</sup>) resulted in clones with extensive cotransformation of adjacent donor-specific variation (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). If transformation frequency were uniform across the genome, the donor allele frequency would be expected to decrease exponentially with increasing distance from the selected site (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>), but this was not seen. Instead we observed two distinct phenomena flanking the selected sites: (a) intervals of reduced cotransformation at or near positions of structural variation (indels and other rearrangements), and (b) a markedly skewed distribution of cotransforming donor alleles to one side of the Nal<sup>R</sup>-selected site within the <em>gyrA</em> locus.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cotransformation around selected sites. (A) and (B) show zooms around the NovR and NalR selected sites for NovR and NalR clones, respectively, whereas (C) and (D) draw the genotype for each specific clone. Donor SNVs are shown in light blue, whereas donor SVs are shown in dark blue (following the width of the recipient allele). The bottoms of the green trapezoids along the x-axis show the width of the SVs according to the donor genome. For example, the SV at ~584 kb is a 2.7-kb transposon specific to the donor strain, and the SV at ~1347 kb has 2 kb in the recipient that do not align with 300 bp in the donor. SNV, single-nucleotide variant, SV, structural variant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cotransformation around selected sites. (A) and (B) show zooms around the Nov&lt;sup&gt;R&lt;/sup&gt; and Nal&lt;sup&gt;R&lt;/sup&gt; selected sites for Nov&lt;sup&gt;R&lt;/sup&gt; and Nal&lt;sup&gt;R&lt;/sup&gt; clones, respectively, whereas (C) and (D) draw the genotype for each specific clone. Donor SNVs are shown in light blue, whereas donor SVs are shown in dark blue (following the width of the recipient allele). The bottoms of the green trapezoids along the x-axis show the width of the SVs according to the donor genome. For example, the SV at ~584 kb is a 2.7-kb transposon specific to the donor strain, and the SV at ~1347 kb has 2 kb in the recipient that do not align with 300 bp in the donor. SNV, single-nucleotide variant, SV, structural variant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F5.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/4/717/F5.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464181" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-72" class="first-child">Cotransformation around selected sites. (A) and (B) show zooms around the Nov<sup>R</sup> and Nal<sup>R</sup> selected sites for Nov<sup>R</sup> and Nal<sup>R</sup> clones, respectively, whereas (C) and (D) draw the genotype for each specific clone. Donor SNVs are shown in light blue, whereas donor SVs are shown in dark blue (following the width of the recipient allele). The bottoms of the green trapezoids along the x-axis show the width of the SVs according to the donor genome. For example, the SV at ~584 kb is a 2.7-kb transposon specific to the donor strain, and the SV at ~1347 kb has 2 kb in the recipient that do not align with 300 bp in the donor. SNV, single-nucleotide variant, SV, structural variant.</p><div class="sb-div caption-clear"></div></div></div><p id="p-73">Local reductions in transformation around the selected sites often corresponded to positions of indels (trapezoids in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, A and B</a>), suggesting that the unrecombined heterologous DNA embedded within (or at the edge of) longer heteroduplex molecules is trimmed out by nucleases after strand exchange (<a id="xref-ref-37-12" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>) (discussed in the next section). In <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>, the strong skew of donor allele frequency to one side of the Nal<sup>R</sup> selected clones suggests that transformation is promoted by some sequence factor to the left of the Nal<sup>R</sup> marker, while transformation at the other end is limited by the presence of a 23-kb prophage specific to the donor strain (right-hand triangle).</p></div><div id="sec-15" class="subsection"><h3>Clustering of donor segments into “recombination tracts”</h3><p id="p-74">As observed in previous work in several organisms, donor segments in individual recombinants were often clustered near other segments, suggesting that endonucleases disrupt long DNAs after uptake (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5, C and D</a>) (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Lin <em>et al.</em> 2009</a>; <a id="xref-ref-37-13" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>). To estimate the number of independent transformation events in each genome, we treated clustered segments as “recombination tracts” derived from the same donor molecule, using as an upper limit a total span of &lt;100 kb (consistent with the length of DNA in high molecular weight preps and the high frequency of cotransformation around the Nov<sup>R</sup> and Nal<sup>R</sup> alleles). Although this 100 kb cut-off is somewhat arbitrary, it is likely generous, since defining recombination tracts in this way gives a mean tract size of 19.5 kb (median 12.9 kb), and only 16 of the 130 tracts (collapsed from 216 segments) exceeded the size of the largest single uninterrupted donor segment (43 kb; segment #71 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS3.zip">File S3</a>). However, because the original taken up DNA fragments are unknown, it is important to note that some apparent clustering may instead result from coincidental transformation by two independent fragments. Clustering was determined using donor rather than recipient genome coordinates, since rearrangements between the two genomes otherwise caused the spurious appearance of clustering or independence of segments in several clones (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-75">Consolidation of segments into tracts allowed us to estimate the minimum number of independent transformation events in each recombinant and to distinguish properties of selected tracts from those that were acquired independently of the selected loci. Twenty-five of 72 colonies (35%) had a single recombination tract spanning the selected Nov<sup>R</sup> or Nal<sup>R</sup> locus, whereas the remainder had 1-3 additional tracts (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Notably, if segregation of heteroduplex recombination products is the norm, then the extent of transformation in these genomes underestimates by twofold the extent of donor strands recombined at unselected sites in the genomes of individual competent cells. These would have a ~50% chance of segregating to an antibiotic-sensitive cell and thus be missed by this study, depending on which donor strands were originally translocated and recombined (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of maximal donor segment clustering. Histograms are shown of (A) donor segments detected per clone, (B) minimum &#x201C;recombination tracts&#x201D; per clone, defined as sets of adjacent segments falling within 100-kb windows. The black line estimates the number of independent recombination events in the original competent cells, assuming heteroduplex segregation and a maximum of four events." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates of maximal donor segment clustering. Histograms are shown of (A) donor segments detected per clone, (B) minimum &#x201C;recombination tracts&#x201D; per clone, defined as sets of adjacent segments falling within 100-kb windows. The black line estimates the number of independent recombination events in the original competent cells, assuming heteroduplex segregation and a maximum of four events.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F6.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/4/717/F6.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464203" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-76" class="first-child">Estimates of maximal donor segment clustering. Histograms are shown of (A) donor segments detected per clone, (B) minimum “recombination tracts” per clone, defined as sets of adjacent segments falling within 100-kb windows. The black line estimates the number of independent recombination events in the original competent cells, assuming heteroduplex segregation and a maximum of four events.</p><div class="sb-div caption-clear"></div></div></div><p id="p-77">Based on the observed number of tracts per recombinant, the line in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a> estimates how many independent molecules the original competent cells had taken up and recombined into their chromosomes (recombination events), given the assumptions that: (a) the probability of each event is independent of the others, (b) each event produces heteroduplex that segregates to two cells, and (c) cells have a maximum of four events. This last assumption simplifies the calculation; since it is conservative the actual distribution may instead be shifted further to the right. This analysis suggests that ≥89% of cells selected for transformation at a single locus had sustained at least one additional recombination event.</p></div><div id="sec-16" class="subsection"><h3>No apparent effect of nucleotide divergence on limiting donor segment location</h3><p id="p-78">As we previously observed, the Nov<sup>R</sup> and Nal<sup>R</sup> alleles transformed strain Rd about threefold more efficiently when the donor background was Rd than when the donor background was the diverged 86-028NP (<a id="xref-ref-37-14" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>) (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>, <em>p</em>-value = 0.024, 1-tailed paired <em>t</em>-test). The lower transformation frequency with DNA from a diverged background is presumably due to sequence heterology around the selected alleles, which could either reduce the probability of strand exchange or increase the probability of recipient-directed heteroduplex correction.</p><p id="p-79">Although this result predicts that donor segments would be preferentially found in parts of the genome with lower sequence divergence, their locations showed no obvious correlation with local nucleotide identity between the donor and recipient genomes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). Instead, the number of donor SNVs per segment was linearly correlated with donor segment length (R<sup>2</sup> = 0.92), closely matching the genome-wide mean SNV density (superimposed solid lines in <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>; 19.3 SNVs/kb genome average <em>vs.</em> a linear estimate of 18.1 ± 0.3 SNVs/kb in donor segments). The lack of effect of heterology was not an artifact of selection for the Nov<sup>R</sup> and Nal<sup>R</sup> alleles, since it held even when only donor segments &gt;100 kb from the selected sites were considered (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a>; linear estimate of 17.3 ± 0.8 SNVs/kb; R<sup>2</sup> = 0.85). This result is consistent with recent observations in <em>Streptococcus pneumoniae</em>, as well as with the finding in <em>H. influenzae</em> that individual donor segments span regions of both high and low nucleotide divergence (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>; <a id="xref-ref-37-15" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of sequence divergence. (A) Nucleotide divergence has no observable effect. Plot shows the count of donor SNVs in each segment as a function of segment size. Colored symbols indicates whether the segment spanned either the NovR or NalR selected site (red triangles), was in the same tracts as a selected segment (green triangles), or was considered independent of the selected segment (blue circles). Black line indicates genome-wide average. Solid and dotted lines indicate the linear fit and 95% prediction intervals for all segments (dark gray) and only segments more than 100 kb from the selected site (light gray). (B) Indels/SVs changed length of the recombined segments. The x-axis shows length of recipient sequences replaced by each donor segment, while the y-axis shows the change in length between the recipient and donor segments. Color-coding as in (A), with blue lines to highlight unselected segments. The recurrent 2.4-kb insertion and 1.7-kb deletion correspond to the SVs adjacent to the NovR and NalR sites, as described in the example in Figure 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effects of sequence divergence. (A) Nucleotide divergence has no observable effect. Plot shows the count of donor SNVs in each segment as a function of segment size. Colored symbols indicates whether the segment spanned either the Nov&lt;sup&gt;R&lt;/sup&gt; or Nal&lt;sup&gt;R&lt;/sup&gt; selected site (red triangles), was in the same tracts as a selected segment (green triangles), or was considered independent of the selected segment (blue circles). Black line indicates genome-wide average. Solid and dotted lines indicate the linear fit and 95% prediction intervals for all segments (dark gray) and only segments more than 100 kb from the selected site (light gray). (B) Indels/SVs changed length of the recombined segments. The x-axis shows length of recipient sequences replaced by each donor segment, while the y-axis shows the change in length between the recipient and donor segments. Color-coding as in (A), with blue lines to highlight unselected segments. The recurrent 2.4-kb insertion and 1.7-kb deletion correspond to the SVs adjacent to the Nov&lt;sup&gt;R&lt;/sup&gt; and Nal&lt;sup&gt;R&lt;/sup&gt; sites, as described in the example in Figure 5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F7.medium.gif" width="294" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/4/717/F7.medium.gif" width="294" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464241" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-80" class="first-child">Effects of sequence divergence. (A) Nucleotide divergence has no observable effect. Plot shows the count of donor SNVs in each segment as a function of segment size. Colored symbols indicates whether the segment spanned either the Nov<sup>R</sup> or Nal<sup>R</sup> selected site (red triangles), was in the same tracts as a selected segment (green triangles), or was considered independent of the selected segment (blue circles). Black line indicates genome-wide average. Solid and dotted lines indicate the linear fit and 95% prediction intervals for all segments (dark gray) and only segments more than 100 kb from the selected site (light gray). (B) Indels/SVs changed length of the recombined segments. The x-axis shows length of recipient sequences replaced by each donor segment, while the y-axis shows the change in length between the recipient and donor segments. Color-coding as in (A), with blue lines to highlight unselected segments. The recurrent 2.4-kb insertion and 1.7-kb deletion correspond to the SVs adjacent to the Nov<sup>R</sup> and Nal<sup>R</sup> sites, as described in the example in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Transforming indels</h3><p id="p-81">Although SVs (insertions, deletions, and other rearrangements) often acted as blocks to recombination progression, they also sometimes transformed as parts of longer recombination tracts (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>). Of the 868 gold-standard SVs, 186 were found in ≥1 recombinant (21.4% of the total), including 22 that affected &gt;100 bp (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). All transforming SVs were accompanied by co-transforming SNVs, and were thus parts of longer recombination tracts. Complex SVs—which included 10 inversions and relocations—did not transform any of the 72 recombinants.</p><p id="p-82">One effect of transforming SVs is to change the length of the replaced recipient segment (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7B</a>). Many length changes were caused by indels flanking the selected Nov<sup>R</sup> and Nal<sup>R</sup> variants (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>), so transforming SVs within selection-independent donor segments were examined separately to test whether there were biases toward deletion or insertions. For unselected segments, seven segments had a net deletion of &gt;100 bp, but only 1 had a net insertion of &gt;100 bp (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>, blue lollipops), although the donor and recipient genomes are distinguished by roughly equal numbers of insertions and deletions (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). This difference was not significant by Fisher’s exact test (<em>p</em>-value = 0.14), but a similar pattern was reported for <em>S. pneumoniae</em> transformants (also not significant) (<a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>). Such a bias need not be intrinsic to the mechanism of recombination, but could occur because, for a given length of recombining DNA, donor-specific insertions have less flanking homology than donor-specific deletions. This predicts that in the absence of selective forces, ‘accessory loci’ of the pan-genome might be more likely be lost than gained by transformation, depending on the size distribution of donor DNA (<a id="xref-ref-7-9" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>).</p></div><div id="sec-18" class="subsection"><h3>Donor segment breakpoint intervals are slightly longer than expected but flank more indels</h3><p id="p-83">We defined donor segment intervals by the outermost variants of contiguous runs of donor-specific alleles, but the true recombination breakpoints must lie between these positions and their nearest flanking recipient-specific alleles. Because the lengths of breakpoint intervals are also measures of local sequence identity, we tested whether they were distinct from the genome-average variant spacing. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> shows that breakpoint intervals were significantly longer than the average spacing between variants (median 106 bp, compared with 15 bp for genome-wide variant spacing; K-S test: <em>D =</em> 0.50, <em>p</em>-value &lt;&lt; 0.001). This difference was seen, even for subsets of the data that included only breakpoint intervals for unselected donor segments (&gt;100 kb away) and when only unique breakpoint intervals were used (median 92.5 bp and 87 bp, respectively).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Breakpoint interval lengths are longer than genome average. (A) Beanplot of sizes of breakpoint intervals (distance between outermost donor-specific SNVs and innermost recipient-specific SNVs): &#x201C;All&#x201D; includes all breakpoint intervals; &#x201C;Clean&#x201D; includes only unique breakpoints of unselected donor segments unaffected by SVs; and &#x201C;Genome&#x201D; includes all intervariant spacings (35,931 total). Black lines show the medians; width of bean is proportional to the density of values with that spacing. (B) Plot of the same data, where the x-axis is the cumulative proportion of the data with interval size less than the value indicated on the y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Breakpoint interval lengths are longer than genome average. (A) Beanplot of sizes of breakpoint intervals (distance between outermost donor-specific SNVs and innermost recipient-specific SNVs): &#x201C;All&#x201D; includes all breakpoint intervals; &#x201C;Clean&#x201D; includes only unique breakpoints of unselected donor segments unaffected by SVs; and &#x201C;Genome&#x201D; includes all intervariant spacings (35,931 total). Black lines show the medians; width of bean is proportional to the density of values with that spacing. (B) Plot of the same data, where the x-axis is the cumulative proportion of the data with interval size less than the value indicated on the y-axis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F8.medium.gif" width="440" height="419"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/4/717/F8.medium.gif" width="440" height="419"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464136" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-84" class="first-child">Breakpoint interval lengths are longer than genome average. (A) Beanplot of sizes of breakpoint intervals (distance between outermost donor-specific SNVs and innermost recipient-specific SNVs): “All” includes all breakpoint intervals; “Clean” includes only unique breakpoints of unselected donor segments unaffected by SVs; and “Genome” includes all intervariant spacings (35,931 total). Black lines show the medians; width of bean is proportional to the density of values with that spacing. (B) Plot of the same data, where the x-axis is the cumulative proportion of the data with interval size less than the value indicated on the y-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-85">The high sequence identity at breakpoints likely underestimates the effect of sequence divergence on recombination breakpoints, since many were likely to have been pre-determined by the donor fragment ends or by degradation of ssDNAs after uptake. The connection between high local identity at recombination breakpoints—yet no connection with limiting the locations of donor segments themselves—implies that initiation of strand exchange from the ends of RecA-coated donor DNA may require higher levels of sequence identity than the subsequent progression of strand exchange, consistent with biochemical analyses of RecA-mediated strand exchange (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gupta <em>et al.</em> 1998</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kowalczykowski and Eggleston 1994</a>).</p><p id="p-86">In contrast, breakpoint intervals were ∼fourfold more likely to have a SV as the nearest recipient-specific variant (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>): 10.9% of breakpoints had a flanking recipient SV (47 of 432 total), whereas only 2.4% of genome-wide intervals are flanked by SVs (Fisher’s exact test <em>p</em>-value &lt;&lt; 0.001). Only part of this effect is due to recurrent breakpoints at the same SV (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>), since enrichment of SVs at breakpoints was still seen even when only unique breakpoint locations and/or only unselected segments were considered (<em>p</em>-values &lt;&lt; 0.001). This enrichment could arise because strand exchange frequently terminates at SVs, or because heteroduplex molecules with mismatched SVs are more efficient targets for heteroduplex correction.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464211/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464211/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Classification of recombination breakpoint intervals</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Evidence of heteroduplex segregation</h3><p id="p-88">We observed 13 mixed donor segments in eight recombinants, with SNVs within each segment consisting of ~25–75% reads supporting the donor allele and the remainder supporting the recipient allele (turquoise bars in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>). The experimental design facilitated recovery of colonies consisting of two genotypes that arose by heteroduplex segregation at unselected sites (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>); cell were plated shortly after transformation and at low density to limit the risk of independent transformants forming mixed colonies. Consistent with their origin as segregants from a single transformant, each of the colonies with a mixed-genotype segment also had a normal unmixed segment containing the selected Nov<sup>R</sup> or Nal<sup>R</sup> allele (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>); if two transformants had contributed to the same colony, their independent recombination breakpoints would be unlikely to be the same at both ends (see <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5</a>), which would leave mixed base calls flanking the two independently selected segments. The origin of one of the selected colonies from two distinct genotypes was confirmed by allele-specific PCR from individual colonies streaked from the stored culture. Strikingly, all but one of the mixed segments are within 200 kb of the predicted origin of replication at ~603 kb (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Salzberg <em>et al.</em> 1998</a>), suggesting that most mixed tracts arose by segregation of recombination products at unselected loci after DNA replication but before DNA replication through the selected loci, as diagrammed in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>.</p></div><div id="sec-20" class="subsection"><h3>Translocation and recombination of individual fragments</h3><p id="p-89">More details were revealed by analysis of allele frequencies in the two colonies that contained clustered segments with a mixed genotype. In one of these colonies (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>), the two clustered segments both had their donor alleles present in ~80% of reads, consistent with both occurring in the same one of the two subclones, and thus with both originating from the same translocated DNA strand (recombination <em>in cis</em>). This was confirmed by allele-specific PCR from colonies streaked from the stored culture. In the other mixed colony (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B</a>), the two clustered segments had complementary donor allele frequencies (~30% or ~70% of reads). Although this recombination <em>in trans</em> could be coincidental translocation of independent fragments complementary to opposite strands, the close spacing of the two segments suggests the more intriguing possibility that both strands of a single long dsDNA fragment in the periplasm were translocated, one from each end. The unmixed donor segments of an additional colony (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FigureS4.pdf">Figure S4</a>) are also consistent with double translocation of a single donor DNA, in this case spanning an inversion breakpoint.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transformants with evidence of clustered heteroduplex donor segments. Top panels show illustrations of each recombinant. Middle panels shows zoom on the clustered &#x201C;mixed&#x201D; donor segments, with the y-axis showing the donor-specific allele frequency. Bottom panel illustrates the inferred recombination intermediate. (A) Heteroduplex segregation of two molecules that transformed the same recipient strand. Clustering suggests a single long strand was translocated into the cytoplasm, but endonuclease activities disrupted the ssDNA before or during recombination. (B) Heteroduplex segregation of two molecules that transformed complementary recipient strands. If derived from a single molecule, each end of the periplasmic dsDNA would have been independently translocated. Since translocation occurs by transport of ssDNA by its 3&#x2032;-end, translocation from both ends through two different pores would yield two cytoplasmic ssDNAs adjacent to each other, but complementary to opposite strands. Figure S4 shows a clone without mixed tracts that would also have required double-translocation across an inversion, if the clustered segments were derived from the same molecule. ssDNA, single-stranded DNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Transformants with evidence of clustered heteroduplex donor segments. Top panels show illustrations of each recombinant. Middle panels shows zoom on the clustered &#x201C;mixed&#x201D; donor segments, with the y-axis showing the donor-specific allele frequency. Bottom panel illustrates the inferred recombination intermediate. (A) Heteroduplex segregation of two molecules that transformed the same recipient strand. Clustering suggests a single long strand was translocated into the cytoplasm, but endonuclease activities disrupted the ssDNA before or during recombination. (B) Heteroduplex segregation of two molecules that transformed complementary recipient strands. If derived from a single molecule, each end of the periplasmic dsDNA would have been independently translocated. Since translocation occurs by transport of ssDNA by its 3&#x2032;-end, translocation from both ends through two different pores would yield two cytoplasmic ssDNAs adjacent to each other, but complementary to opposite strands. Figure S4 shows a clone without mixed tracts that would also have required double-translocation across an inversion, if the clustered segments were derived from the same molecule. ssDNA, single-stranded DNA.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F9.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/4/4/717/F9.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464189" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-90" class="first-child">Transformants with evidence of clustered heteroduplex donor segments. Top panels show illustrations of each recombinant. Middle panels shows zoom on the clustered “mixed” donor segments, with the y-axis showing the donor-specific allele frequency. Bottom panel illustrates the inferred recombination intermediate. (A) Heteroduplex segregation of two molecules that transformed the same recipient strand. Clustering suggests a single long strand was translocated into the cytoplasm, but endonuclease activities disrupted the ssDNA before or during recombination. (B) Heteroduplex segregation of two molecules that transformed complementary recipient strands. If derived from a single molecule, each end of the periplasmic dsDNA would have been independently translocated. Since translocation occurs by transport of ssDNA by its 3′-end, translocation from both ends through two different pores would yield two cytoplasmic ssDNAs adjacent to each other, but complementary to opposite strands. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FigureS4.pdf">Figure S4</a> shows a clone without mixed tracts that would also have required double-translocation across an inversion, if the clustered segments were derived from the same molecule. ssDNA, single-stranded DNA.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Evidence of heteroduplex correction</h3><p id="p-91">For 11 donor segments only a single unambiguous donor-specific SNV is present; 7 of these are not clustered into longer recombination tracts. These likely represent genuine outcomes of transformation, rather than being artifacts (either new mutations or sequencing errors that coincidentally produced a donor-specific SNV). Two analyses suggest that the frequency such artifacts is expected to be very low. First, no new high quality variants (alleles not present in either parent) were found in the 72 recombinants, suggesting that new mutations are uncommon. Second, only 10 high-quality variants were identified by sequencing another lab strain, MAP7, an Rd derivative resistant to seven antibiotics (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Catlin <em>et al.</em> 1972</a>), and only 3 of these are not accounted for by its antibiotic resistances (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464236/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464236/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">MAP7 SNVs</span><div class="sb-div caption-clear"></div></div></div><p id="p-94">Because these singlet segments are clearly more abundant than predicted by the overall distribution of donor segments (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>), they likely arise by a distinct process. We posit that singlet donor segments are best explained by donor-directed mismatch repair from a recombination tract that was subsequently lost by heteroduplex segregation.</p></div><div id="sec-22" class="subsection"><h3>Only a subset of cells in competent cultures is transformed</h3><p id="p-95">Observed cotransformation frequencies of Nov<sup>R</sup> and Nal<sup>R</sup> were ~10-fold higher than predicted by the product of their single marker transformation frequencies (6.2- to 14.5-fold, <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>), and these values were consistent for two donor DNAs, both the RR3131 donor DNA and DNA from an Rd-derived strain MAP7 (RR666) (<em>t</em>-test <em>p</em>-value = 0.26). This excess of double transformants (termed congression) is usually assumed to reflect the presence of noncompetent cells in the culture, and suggests that only ~10% of cells actively took up and recombined donor DNA into their chromosomes (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Goodgal and Herriott 1961</a>).</p><p id="p-96">To directly test whether congression measures % competence, we sequenced 16 unselected clones from the MIV transformations. Only 2 of 16 had been transformed, each carrying a single recombination tract, consistent with congression predictions of ~10% competent cells (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS3.zip">File S3</a>). The sizes of the two tracts fell within the range seen in selected clones (2.7 kb and 40.2 kb). The estimates in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a> also reject the null hypothesis that all cells are equally competent, since they predict that 65% of unselected colonies would have recombination tracts (Fisher’s exact test <em>p</em>-value = 0.01).</p><p id="p-97">Cultures grown to high density in rich medium (“late-log”) are ~100-fold less transformable than MIV-induced cultures (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Redfield <em>et al.</em> 2005</a>). To clarify whether competent cells in such cultures are rare or just weakly transformable, a late-log culture was transformed with donor DNA as described previously, giving Nov<sup>R</sup>/CFU of 1.15 × 10<sup>−5</sup> (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>), and 8 Nov<sup>R</sup> transformants were sequenced. The donor segments of these clones were comparable in size, number, and distribution to those of the MIV-selected transformants and included one ‘mixed’ segment (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2D</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FileS3.zip">File S3</a>, segments averaged 8.6 ± 5.8), suggesting that transformation in late-log cultures results from a very small fraction of fully competent cells. This conclusion is also supported by co-transformation data from late-log cultures, which indicates ~0.1% competent cells (data not shown).</p><p id="p-98">Although we could not rule out subtle differences or stochastic variation in level of transformability these data are consistent with competence induction being an on/off response, in which the starvation conditions of MIV induce a much higher proportion of cells to become competent than late-log conditions.</p></div><div id="sec-23" class="subsection"><h3>Natural variation in transformability is polygenic</h3><p id="p-99">Competence varies widely within species, but the underlying causes are unknown; understanding the genetic architecture of competence in natural populations could lead to a better understanding of the evolutionary forces that maintain it. For example, the 86-028NP donor strain has ~500-fold lower transformability than the Rd recipient and DNA uptake is ~20-fold lower (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Maughan and Redfield 2009a</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">b</a>). To assess the practicality of using natural transformation to map this variation, all 96 sequenced clones were tested for changes in transformability in a “transformation-during-growth” screen (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10A</a>), and the seven clones with the lowest transformation frequencies were re-assayed in MIV starvation medium (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FigureS5.pdf">Figure S5</a>A). This identified a single Nov<sup>S</sup>Nal<sup>S</sup> recombinant clone (RR4108, from the unselected set, * in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2C</a>) with a reproducible 10-fold decrease in transformation (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10B</a>) (<em>t</em>-test <em>p</em>-value = 0.025). This recombinant has normal DNA uptake compared to Rd, so it must not have acquired alleles from the donor that affect either competence regulation or the outer membrane DNA uptake machinery: Following the method of (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Sinha <em>et al.</em> 2012</a>), assays using 50 ng of <sup>33</sup>P-labeled 222 bp DNA fragment per milliliter of competent cells gave 26.0 ± 2.1% uptake by the recombinant RR4108, compared to 25.3 ± 3.4% uptake by Rd and 1.3 ± 0.4% uptake by 86-028NP.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/717/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transformation mapping of a transformability factor. (A) For the primary screen, &#x201C;transformation-during growth&#x201D; assays measure transformation frequencies (KanR/CFU) of the set of 96 recombinants. Data are normalized to the frequencies observed for RR722 recipient controls in each experiment to account for experimental variation over the 11 independent assays performed. Thick lines indicate median value and thin lines indicate 5% and 95% quartiles. The RR3131 donor strain was included as a control. (B) Map of the donor segment interval in RR4108 is shown in blue replacing the syntenic genes in the recipient chromosome (pink). Black lines show donor segment endpoints. The recipient-specific xylose ABC transporter (shown with the gray triangle) was deleted in the recombinant. The comM gene is highlighted with a black box. (C) Transformation frequencies (KanR/CFU) using the standard MIV method for the recipient and recombinant RR4108 strains with and without the comM gene and complementation by plasmids carrying either the donor or recipient alleles of the comM gene. The donor RR3131 (NP) strain is shown as a control. The comM deletion in RR3131 had a transformation frequency below the limit of detection. Bars show mean transformation frequencies and error bars show standard deviations from triplicate experiments. Letters above the bars group strains with no significant difference by paired t-test. Note that the difference between donor and recipient in the standard MIV assay is larger than in the primary screen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648020722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transformation mapping of a transformability factor. (A) For the primary screen, &#x201C;transformation-during growth&#x201D; assays measure transformation frequencies (Kan&lt;sup&gt;R&lt;/sup&gt;/CFU) of the set of 96 recombinants. Data are normalized to the frequencies observed for RR722 recipient controls in each experiment to account for experimental variation over the 11 independent assays performed. Thick lines indicate median value and thin lines indicate 5% and 95% quartiles. The RR3131 donor strain was included as a control. (B) Map of the donor segment interval in RR4108 is shown in blue replacing the syntenic genes in the recipient chromosome (pink). Black lines show donor segment endpoints. The recipient-specific xylose ABC transporter (shown with the gray triangle) was deleted in the recombinant. The &lt;em&gt;comM&lt;/em&gt; gene is highlighted with a black box. (C) Transformation frequencies (Kan&lt;sup&gt;R&lt;/sup&gt;/CFU) using the standard MIV method for the recipient and recombinant RR4108 strains with and without the &lt;em&gt;comM&lt;/em&gt; gene and complementation by plasmids carrying either the donor or recipient alleles of the &lt;em&gt;comM&lt;/em&gt; gene. The donor RR3131 (NP) strain is shown as a control. The &lt;em&gt;comM&lt;/em&gt; deletion in RR3131 had a transformation frequency below the limit of detection. Bars show mean transformation frequencies and error bars show standard deviations from triplicate experiments. Letters above the bars group strains with no significant difference by paired &lt;em&gt;t&lt;/em&gt;-test. Note that the difference between donor and recipient in the standard MIV assay is larger than in the primary screen.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/717/F10.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/4/4/717/F10.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/717/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/717/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464227" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-100" class="first-child">Transformation mapping of a transformability factor. (A) For the primary screen, “transformation-during growth” assays measure transformation frequencies (Kan<sup>R</sup>/CFU) of the set of 96 recombinants. Data are normalized to the frequencies observed for RR722 recipient controls in each experiment to account for experimental variation over the 11 independent assays performed. Thick lines indicate median value and thin lines indicate 5% and 95% quartiles. The RR3131 donor strain was included as a control. (B) Map of the donor segment interval in RR4108 is shown in blue replacing the syntenic genes in the recipient chromosome (pink). Black lines show donor segment endpoints. The recipient-specific xylose ABC transporter (shown with the gray triangle) was deleted in the recombinant. The <em>comM</em> gene is highlighted with a black box. (C) Transformation frequencies (Kan<sup>R</sup>/CFU) using the standard MIV method for the recipient and recombinant RR4108 strains with and without the <em>comM</em> gene and complementation by plasmids carrying either the donor or recipient alleles of the <em>comM</em> gene. The donor RR3131 (NP) strain is shown as a control. The <em>comM</em> deletion in RR3131 had a transformation frequency below the limit of detection. Bars show mean transformation frequencies and error bars show standard deviations from triplicate experiments. Letters above the bars group strains with no significant difference by paired <em>t</em>-test. Note that the difference between donor and recipient in the standard MIV assay is larger than in the primary screen.</p><div class="sb-div caption-clear"></div></div></div><p id="p-101">The recombinant contains a single 40.3 kb recombination tract carrying 483 donor-specific SNVs, spanning ~30 genes and deleting a 3.7 kb xylose ABC transporter operon specific to the recipient strain (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10B</a>). At least one locus outside the RR4108 donor segment interval must be responsible for the remaining 50-fold difference in transformability between 86-028NP and Rd. Because 86-028NP has substantially reduced DNA uptake as well as transformability, some of this variation must affect either competence regulation or the uptake of DNA through the outer membrane. The normal transformation frequencies of the other 81 recombinants rule out major effects of ~1/3 of the donor-specific variation, including variation at 12 known competence genes. Subtle differences due to variation in these genes are possible, although normal MIV-induced transformation frequencies were seen for the six recombinants whose segments include any of the twelve (<em>t</em>-test <em>p</em>-value &gt; 0.2 for all six; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1/FigureS5.pdf">Figure S5</a>B).</p><p id="p-102">One known competence-regulated gene, <em>comM</em>, is found within the RR4108 donor segment. Null mutants of <em>comM</em> have a 42-fold defect in the Rd strain background (<a id="xref-fig-10-4" class="xref-fig" href="#F10">Figure 10C</a>; <em>t</em>-test <em>p</em>-value = 0.028); its protein product acts after DNA uptake to promote recombination, consistent with the normal uptake observed in the RR4108 recombinant (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gwinn <em>et al.</em> 1998</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Sinha <em>et al.</em> 2012</a>). To test whether the two nonsynonymous differences distinguishing the <em>comM</em>-Rd and <em>comM</em>-NP alleles affect the gene’s activity, plasmids carrying each were used to test complementation of the deletion mutant’s phenotype. Both plasmids complemented the Rd <em>comMΔ</em>::<em>spc</em> mutant equally well (<a id="xref-fig-10-5" class="xref-fig" href="#F10">Figure 10C</a>; <em>t</em>-test <em>p</em>-value = 0.118), indicating nonsynonymous <em>comM</em> differences are not responsible for the phenotypic change.</p><p id="p-103">Additional experiments complicated matters; these indicate that although genetic variation within <em>comM</em> is not responsible for the altered phenotype, the chromosomal <em>comM</em> locus is required for the difference between the strains to be observed: First, deleting <em>comM</em> from RR4108 reduced transformation to the same levels as in Rd (<a id="xref-fig-10-6" class="xref-fig" href="#F10">Figure 10C</a>; <em>t</em>-test <em>p</em>-value = 0.32). This might suggest that ComM activity contributes more to transformability in the recipient background, but the requirement for ComM was substantially stronger in the 86-028NP donor background (wild type: 3.9 ± 0.6 × 10<sup>−6</sup> Kan<sup>R</sup>/CFU; <em>comM</em> deletion: &lt;7.9 ± 0.2 × 10<sup>−9</sup>). Second, the plasmids carrying the Rd and NP alleles of <em>comM</em> failed to complement a RR4108 <em>comMΔ</em>::<em>spc</em> mutant (<a id="xref-fig-10-7" class="xref-fig" href="#F10">Figure 10C</a>; <em>t</em>-test <em>p</em>-values &gt; 0.05). This might indicate that variation in the RR4108 donor segment modulates <em>comM</em> activity <em>in cis</em>, but both promoter and terminator are invariant between the strains and the plasmid insert carrying the <em>comM</em> alleles was flanked by ±700 bp.</p><p id="p-104">In summary, this pilot mapping experiment has used natural transformation to identify a genetic interval carrying natural variation that affects transformability itself. We found that the variation in transformability is a polygenic trait with differences affecting both pre- and postuptake steps, and we have begun unraveling complex interactions between the <em>comM</em> locus and unidentified but linked natural variation.</p></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><div id="sec-25" class="subsection"><h3>Experimental genomics and bacterial genetic recombination</h3><p id="p-105">Although the evolutionarily selected function of natural competence remains controversial—since DNA can provide nutrients as well as genetic information—the consequences of transformational recombination are not in doubt (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Didelot and Maiden 2010</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Feil and Spratt 2001</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Moradigaravand and Engelstadter 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Redfield 2001</a>). In this study, we examined the genome-wide recombination that arose when competent <em>H. influenzae</em> cells were incubated with DNA from a diverged clinical isolate, finding that the average cell’s daughters replaced ~1% of their chromosomes in multiple independent recombination events that spanned many genes. These data will help constrain parameters used for genome-wide inferences of historical recombination in natural populations, for example by providing a rate parameter to describe the exponential distribution of donor segment lengths (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Didelot <em>et al.</em> 2010</a>). Comparisons of such parameter estimates with those from genome-wide studies of other species or other horizontal gene transfer processes will help disentangle how modulations in the mechanism of genetic recombination affect its long-term genomic consequences (<a id="xref-ref-7-10" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Feil and Spratt 2001</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gray <em>et al.</em> 2013</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wilson 2012</a>).</p></div><div id="sec-26" class="subsection"><h3>Comparison of natural transformation in <em>H. influenzae</em> and <em>S. pneumoniae</em></h3><p id="p-106">Although the species are very distantly related, our genome-wide analyses of transformation in <em>H. influenzae</em> reached conclusions similar to those for the Gram-positive bacterium <em>S. pneumoniae</em> (<a id="xref-ref-7-11" class="xref-bibr" href="#ref-7">Croucher <em>et al.</em> 2012</a>; <a id="xref-ref-37-16" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>): (a) Competent cells typically take up and recombine multiple donor molecules; (b) recombination tracts can span dozens of genes and hundreds of genetic variants; (c) tracts are often mosaic clusters of donor segments; (d) nucleotide divergence has surprisingly little effect on the locations of recombinant segments; (e) SVs transform at lower rates than SNVs; (f) deletions may transform more easily than insertions; and (g) no strong transformation hotspots were identified. Herein, we contrast the genome-wide analyses of transformation in <em>H. influenzae</em> and <em>S. pneumoniae</em>, but it is worth noting that results from the two studies might not reflect species-wide recombination parameters, as we still know very little about how such parameters vary within species or for different combinations of donor and recipient genomes.</p><p id="p-107">Donor segments in <em>S. pneumoniae</em> are shorter than in <em>H. influenzae</em> (mean size 2.3 <em>vs.</em> 6.9 kb), which could reflect their processing by distinct DNA-uptake mechanisms. Although the Gram-negative <em>H. influenzae</em> translocates large donor molecules from the periplasm through its inner membrane, <em>S. pneumoniae</em> and other Gram-positive bacteria process dsDNAs with membrane-bound nucleases before translocation through their single cell membrane (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lacks and Neuberger 1975</a>). The mosaic recombination tracts seen in both species likely only sometimes result from heteroduplex correction mechanisms and instead could often be due to endonuclease activities acting prior to strand exchange, or prior to translocation into the cytoplasm.</p><p id="p-108">Although transformation hotspots were not identified in either species, the power to detect variation in transformation frequency across the genome was limited in both cases, and genetic experiments have found that transformation rates are affected by nucleotide divergence and can independently vary between loci (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lawrence 2002</a>; <a id="xref-ref-37-17" class="xref-bibr" href="#ref-37">Mell <em>et al.</em> 2011</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Ray <em>et al.</em> 2009</a>). Our observation in <em>H. influenzae</em> that transformation decreases irregularly around the selected antibiotic resistance loci (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a>) contrasts with <em>S. pneumoniae</em> and may indicate higher heterogeneity in transformation frequencies.</p><p id="p-109">Accurate quantification of genome-wide variation in transformation frequency will require either sequencing much larger sets of recombinants or deeply sequencing large pools of recombinant genomes. These experiments could more precisely tease apart the effects of divergence and such locus-specific factors as proximity to the origin of DNA replication or the effects of chromosome structural proteins (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Ray <em>et al.</em> 2009</a>). Estimating these effects will be particularly crucial for population genetic inferences, for example distinguishing between natural selection at a locus favoring recombination and simply higher recombination rates at that locus.</p></div><div id="sec-27" class="subsection"><h3>Heteroduplex segregation and correction</h3><p id="p-110">Our observation of mixed donor segments clustered around the predicted origin of replication is consistent with heteroduplex segregation, where recombination events occurred after DNA replication of the unselected locus but before replication of the selected locus (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). Consideration of heteroduplex segregation also led to our interpretation that the excess of very short donor segments (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a>) could be the result of donor-directed heteroduplex correction from recombined donor DNA that was subsequently lost by segregation. Confirming that mismatch repair mutants lack these very short segments would have implications for population-level analyses of recombination, since such methods typically identify putative recombination events by their inclusion of many nearby polymorphic sites.</p></div><div id="sec-28" class="subsection"><h3>Proof-of-principle genetic mapping of natural variation in transformability</h3><p id="p-111">Our pilot mapping experiment shows that plummeting sequencing costs make natural transformation combined with genomics a powerful means to map natural phenotypic variation. Recombinants generated by natural transformation are equivalent to the “nearly isogenic lines” used for genetic mapping in eukaryotes, though in bacteria they are considerably easier to generate and cheaper to comprehensively genotype. This allowed us to use our small mapping population to rapidly identify an interval (comprising ~2% of the genome) that contained natural variation with a 10-fold effect on transformability. To our knowledge, this is the first time transformation has been used to identify a bacterial quantitative trait locus, and importantly the causal variation is not accounted for by any of the known competence genes (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Sinha <em>et al.</em> 2012</a>). Extending this transformation genomics approach to other phenotypes will be particularly powerful when the trait conferred by donor DNA is selectable, as this will obviate the need to screen for phenotypic changes in large recombinant mapping populations. Many such traits are important to pathogenesis and vary widely between clinical isolates of <em>H. influenzae</em>; these include antibiotic resistances, resistance to human complement-mediated killing, and intracellular invasion of airway epithelial cells (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Morey <em>et al.</em> 2011</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Tristram <em>et al.</em> 2007</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Williams <em>et al.</em> 2001</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-112">We thank Anastasia Timofiv, Lexi Mithel, and Kartik Suri for assistance with experiments; Yongjun Zhao at the Michael Smith Genome Sciences Centre for assistance with sequencing; and Ira Hall for help with data analysis. We also thank the anonymous reviewers for helpful comments. Funding for this work was provided by grants from Genome British Columbia to RJR and JCM, the Canadian Institutes of Health Research to R.J.R., and the National Institutes of Health to J.C.M.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009597/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data from this article have been deposited with the National Center for Biotechnology Information short-read archive project under accession number SRP036875.</p></li><li class="fn" id="fn-38"><p id="p-113"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-11-25"><span class="received-label">Received </span>November 25, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-02-19"><span class="accepted-label">Accepted </span>February 19, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Mell <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.1" data-doi="10.1128/JCM.05476-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agrawal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Haemophilus influenzae</em> infections in the <em>H. influenzae</em> type b conjugate vaccine era.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">3728</span>–<span class="cit-lpage">3732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.05476-11%26rft_id%253Dinfo%253Apmid%252F21900515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI0OS8xMS8zNzI4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.2" data-doi="10.1016/0378-1119(91)90541-I"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartolome</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jubete</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Cruz</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Construction and properties of a family of pACYC184-derived cloning vectors compatible with pBR322 and its derivatives.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DBartolom%25C3%25A9%26rft.auinit1%253DB.%26rft.volume%253D102%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D78%26rft.atitle%253DConstruction%2Band%2Bproperties%2Bof%2Ba%2Bfamily%2Bof%2BpACYC184-derived%2Bcloning%2Bvectors%2Bcompatible%2Bwith%2BpBR322%2Band%2Bits%2Bderivatives.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252891%252990541-I%26rft_id%253Dinfo%253Apmid%252F1840539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(91)90541-I&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1840539&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GA43800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.3" data-doi="10.1099/00221287-70-3-411"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catlin</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bendler</span>  <span class="cit-name-given-names">J. W.</span>  <span class="cit-name-suffix">3rd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodgal</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">The type b capsulation locus of <em>Haemophilus influenzae</em>: map location and size.</span> <abbr class="cit-jnl-abbrev">J. Gen. Microbiol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DCATLIN%26rft.auinit1%253DB.%2BW.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D411%26rft.epage%253D422%26rft.atitle%253DThe%2BType%2Bb%2BCapsulation%2BLocus%2Bof%2BHaemophilus%2Binfluenzae%253A%2BMap%2BLocation%2Band%2BSize%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-70-3-411%26rft_id%253Dinfo%253Apmid%252F4402481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-70-3-411&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4402481&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1972M555600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.4" data-doi="10.1038/nrmicro844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubnau</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">DNA uptake during bacterial transformation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DChen%26rft.auinit1%253DI.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253D241%26rft.epage%253D249%26rft.atitle%253DDNA%2Buptake%2Bduring%2Bbacterial%2Btransformation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro844%26rft_id%253Dinfo%253Apmid%252F15083159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15083159&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220431800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.5" data-doi="10.1111/j.1574-6976.2009.00164.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Claverys</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polard</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic transformation machinery: composition, localization, and mechanism.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2009.00164.x%26rft_id%253Dinfo%253Apmid%252F19228200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3JlIjtzOjU6InJlc2lkIjtzOjg6IjMzLzMvNjQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.6" data-doi="10.1126/science.1198545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croucher</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rapid pneumococcal evolution in response to clinical interventions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCroucher%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D331%26rft.issue%253D6016%26rft.spage%253D430%26rft.epage%253D434%26rft.atitle%253DRapid%2BPneumococcal%2BEvolution%2Bin%2BResponse%2Bto%2BClinical%2BInterventions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1198545%26rft_id%253Dinfo%253Apmid%252F21273480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzEvNjAxNi80MzAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.7" data-doi="10.1371/journal.ppat.1002745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croucher</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barquist</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhill</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-resolution view of genome-wide pneumococcal transformation.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DCroucher%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002745%26rft.epage%253De1002745%26rft.atitle%253DA%2Bhigh-resolution%2Bview%2Bof%2Bgenome-wide%2Bpneumococcal%2Btransformation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002745%26rft_id%253Dinfo%253Apmid%252F22719250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22719250&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.8" data-doi="10.1016/j.tim.2010.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maiden</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Impact of recombination on bacterial evolution.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DDidelot%26rft.auinit1%253DX.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D315%26rft.epage%253D322%26rft.atitle%253DImpact%2Bof%2Brecombination%2Bon%2Bbacterial%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2010.04.002%26rft_id%253Dinfo%253Apmid%252F20452218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2010.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20452218&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280054200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.9" data-doi="10.1534/genetics.110.120121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Inference of homologous recombination in bacteria using whole-genome sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1435</span>–<span class="cit-lpage">1449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120121%26rft_id%253Dinfo%253Apmid%252F20923983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni80LzE0MzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.10" data-doi="10.1086/511861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dworkin</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borchardt</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The changing epidemiology of invasive <em>Haemophilus influenzae</em> disease, especially in persons &gt; or = 65 years old.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">810</span>–<span class="cit-lpage">816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F511861%26rft_id%253Dinfo%253Apmid%252F17304452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY2lkIjtzOjU6InJlc2lkIjtzOjg6IjQ0LzYvODEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erickson</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copeland</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Congression of unlinked markers and genetic mapping in the transformation of <em>Bacillus subtilis</em> 168.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DErickson%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D21%26rft.atitle%253DCONGRESSION%2BOF%2BUNLINKED%2BMARKERS%2BAND%2BGENETIC%2BMAPPING%2BIN%2BTHE%2BTRANSFORMATION%2BOF%2BBACILLUS%2BSUBTILIS%2B168%26rft_id%253Dinfo%253Apmid%252F4631599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiNzMvMS8xMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.12" data-doi="10.1146/annurev.micro.55.1.561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spratt</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination and the population structures of bacterial pathogens.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DFeil%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D561%26rft.epage%253D590%26rft.atitle%253DRECOMBINATION%2BAND%2BTHE%2BPOPULATION%2BSTRUCTURES%2BOF%2BBACTERIAL%2BPATHOGENS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.55.1.561%26rft_id%253Dinfo%253Apmid%252F11544367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.55.1.561&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11544367&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171732600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.13" data-doi="10.1073/pnas.98.1.182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination within natural populations of pathogenic bacteria: short-term empirical estimates and long-term phylogenetic consequences.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFeil%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D182%26rft.epage%253D187%26rft.atitle%253DRecombination%2Bwithin%2Bnatural%2Bpopulations%2Bof%2Bpathogenic%2Bbacteria%253A%2BShort-term%2Bempirical%2Bestimates%2Band%2Blong-term%2Bphylogenetic%2Bconsequences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.98.1.182%26rft_id%253Dinfo%253Apmid%252F11136255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OC8xLzE4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.14" data-doi="10.1126/science.7542800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fleischmann</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clayton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirkness</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Whole-genome random sequencing and assembly of <em>Haemophilus influenzae</em> Rd.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFleischmann%26rft.auinit1%253DR.%26rft.volume%253D269%26rft.issue%253D5223%26rft.spage%253D496%26rft.epage%253D512%26rft.atitle%253DWhole-genome%2Brandom%2Bsequencing%2Band%2Bassembly%2Bof%2BHaemophilus%2Binfluenzae%2BRd%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7542800%26rft_id%253Dinfo%253Apmid%252F7542800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjkvNTIyMy80OTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.15" data-doi="10.1085/jgp.44.6.1201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodgal</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herriott</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Studies on transformations of <em>Hemophilus influenzae</em>. I. Competence.</span> <abbr class="cit-jnl-abbrev">J. Gen. Physiol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BGeneral%2BPhysiology%26rft.stitle%253DJGP%26rft.aulast%253DGoodgal%26rft.auinit1%253DS.%2BH.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1201%26rft.epage%253D1227%26rft.atitle%253DStudies%2Bon%2BTransformations%2Bof%2BHemophilus%2Binfluenzae%253A%2BI.%2BCompetence%26rft_id%253Dinfo%253Adoi%252F10.1085%252Fjgp.44.6.1201%26rft_id%253Dinfo%253Apmid%252F13707010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamdwIjtzOjU6InJlc2lkIjtzOjk6IjQ0LzYvMTIwMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.16" data-doi="10.1371/journal.pbio.1001602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krywy</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harold</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palumbo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derbyshire</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Distributive conjugal transfer in mycobacteria generates progeny with meiotic-like genome-wide mosaicism, allowing mapping of a mating identity locus.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1001602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001602%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001602%26rft_id%253Dinfo%253Apmid%252F23874149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874149&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.17" data-doi="10.1073/pnas.95.17.9843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golub</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radding</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Polarity of DNA strand exchange promoted by recombination proteins of the RecA family.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">9843</span>–<span class="cit-lpage">9848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGupta%26rft.auinit1%253DR.%2BC.%26rft.volume%253D95%26rft.issue%253D17%26rft.spage%253D9843%26rft.epage%253D9848%26rft.atitle%253DPolarity%2Bof%2BDNA%2Bstrand%2Bexchange%2Bpromoted%2Bby%2Brecombination%2Bproteins%2Bof%2Bthe%2BRecA%2Bfamily%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.17.9843%26rft_id%253Dinfo%253Apmid%252F9707563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTUvMTcvOTg0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.18" data-doi="10.1093/bioinformatics/btq413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kultima</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">genoPlotR: comparative gene and genome visualization in R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2334</span>–<span class="cit-lpage">2335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq413%26rft_id%253Dinfo%253Apmid%252F20624783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTgvMjMzNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gwinn</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramanathan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomb</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A new transformation-deficient mutant of <em>Haemophilus influenzae</em> Rd with normal DNA uptake.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">746</span>–<span class="cit-lpage">748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGwinn%26rft.auinit1%253DM.%2BL.%26rft.volume%253D180%26rft.issue%253D3%26rft.spage%253D746%26rft.epage%253D748%26rft.atitle%253DA%2BNew%2BTransformation-Deficient%2BMutant%2Bof%2BHaemophilus%2Binfluenzae%2BRd%2Bwith%2BNormal%2BDNA%2BUptake%26rft_id%253Dinfo%253Apmid%252F9457884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTgwLzMvNzQ2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.20" data-doi="10.1126/science.1171908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanage</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connor</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corander</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Hyper-recombination, diversity, and antibiotic resistance in pneumococcus.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">1454</span>–<span class="cit-lpage">1457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHanage%26rft.auinit1%253DW.%2BP.%26rft.volume%253D324%26rft.issue%253D5933%26rft.spage%253D1454%26rft.epage%253D1457%26rft.atitle%253DHyper-Recombination%252C%2BDiversity%252C%2Band%2BAntibiotic%2BResistance%2Bin%2BPneumococcus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1171908%26rft_id%253Dinfo%253Apmid%252F19520963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjQvNTkzMy8xNDU0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.21" data-doi="10.1128/JB.187.13.4627-4636.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillaspy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mungur</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic sequence of an otitis media isolate of nontypeable <em>Haemophilus influenzae</em>: comparative study with <em>H. influenzae</em> serotype d, strain KW20.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">4627</span>–<span class="cit-lpage">4636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DHarrison%26rft.auinit1%253DA.%26rft.volume%253D187%26rft.issue%253D13%26rft.spage%253D4627%26rft.epage%253D4636%26rft.atitle%253DGenomic%2BSequence%2Bof%2Ban%2BOtitis%2BMedia%2BIsolate%2Bof%2BNontypeable%2BHaemophilus%2Binfluenzae%253A%2BComparative%2BStudy%2Bwith%2BH.%2Binfluenzae%2BSerotype%2Bd%252C%2BStrain%2BKW20%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.187.13.4627-4636.2005%26rft_id%253Dinfo%253Apmid%252F15968074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4Ny8xMy80NjI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.22" data-doi="10.1186/gb-2007-8-6-r103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hogg</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">F. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janto</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boissy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Characterization and modeling of the <em>Haemophilus influenzae</em> core and supragenomes based on the complete genomic sequences of Rd and 12 clinical nontypeable strains.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHogg%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253DR103%26rft.epage%253DR103%26rft.atitle%253DCharacterization%2Band%2Bmodeling%2Bof%2Bthe%2BHaemophilus%2Binfluenzae%2Bcore%2Band%2Bsupragenomes%2Bbased%2Bon%2Bthe%2Bcomplete%2Bgenomic%2Bsequences%2Bof%2BRd%2Band%2B12%2Bclinical%2Bnontypeable%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-6-r103%26rft_id%253Dinfo%253Apmid%252F17550610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-6-r103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17550610&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.23" data-doi="10.1016/j.resmic.2007.09.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnsborg</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eldholm</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Havarstein</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Natural genetic transformation: prevalence, mechanisms and function.</span> <abbr class="cit-jnl-abbrev">Res. Microbiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.stitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.aulast%253DJohnsborg%26rft.auinit1%253DO.%26rft.volume%253D158%26rft.issue%253D10%26rft.spage%253D767%26rft.epage%253D778%26rft.atitle%253DNatural%2Bgenetic%2Btransformation%253A%2Bprevalence%252C%2Bmechanisms%2Band%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.resmic.2007.09.004%26rft_id%253Dinfo%253Apmid%252F17997281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.resmic.2007.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17997281&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252370300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.24" data-doi="10.1146/annurev.bi.63.070194.005015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eggleston</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Homologous pairing and DNA strand-exchange proteins.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">1043</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DKowalczykowski%26rft.auinit1%253DS.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D991%26rft.epage%253D1043%26rft.atitle%253DHomologous%2BPairing%2Band%2BDNA%2BStrand%2BExchange%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.bi.63.070194.005015%26rft_id%253Dinfo%253Apmid%252F7979259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.bi.63.070194.005015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7979259&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NV05900028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.25" data-doi="10.1111/j.1365-2958.2011.07647.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruger</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stingl</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Two steps away from novelty–principles of bacterial DNA uptake.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">860</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DKruger%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D860%26rft.epage%253D867%26rft.atitle%253DTwo%2Bsteps%2Baway%2Bfrom%2Bnovelty--principles%2Bof%2Bbacterial%2BDNA%2Buptake.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2011.07647.x%26rft_id%253Dinfo%253Apmid%252F21435041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2011.07647.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21435041&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacks</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuberger</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Membrane location of a deoxyribonuclease implicated in the genetic transformation of <em>Diplococcus pneumoniae</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1321</span>–<span class="cit-lpage">1329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLacks%26rft.auinit1%253DS%26rft.volume%253D124%26rft.issue%253D3%26rft.spage%253D1321%26rft.epage%253D1329%26rft.atitle%253DMembrane%2Blocation%2Bof%2Ba%2Bdeoxyribonuclease%2Bimplicated%2Bin%2Bthe%2Bgenetic%2Btransformation%2Bof%2BDiplococcus%2Bpneumoniae.%26rft_id%253Dinfo%253Apmid%252F366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjEyNC8zLzEzMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.27" data-doi="10.1006/tpbi.2002.1587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Gene transfer in bacteria: speciation without species?</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DLawrence%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D61%26rft.issue%253D4%26rft.spage%253D449%26rft.epage%253D460%26rft.atitle%253DGene%2Btransfer%2Bin%2Bbacteria%253A%2Bspeciation%2Bwithout%2Bspecies%253F%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1587%26rft_id%253Dinfo%253Apmid%252F12167364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12167364&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177739500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.28" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.29" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.30" data-doi="10.1371/journal.ppat.1000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Natural transformation of <em>Helicobacter pylori</em> involves the integration of short DNA fragments interrupted by gaps of variable size.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DLin%26rft.auinit1%253DE.%2BA.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000337%26rft.epage%253De1000337%26rft.atitle%253DNatural%2Btransformation%2Bof%2Bhelicobacter%2Bpylori%2Binvolves%2Bthe%2Bintegration%2Bof%2Bshort%2BDNA%2Bfragments%2Binterrupted%2Bby%2Bgaps%2Bof%2Bvariable%2Bsize.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000337%26rft_id%253Dinfo%253Apmid%252F19282979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19282979&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.31" data-doi="10.1126/science.1140818"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maamar</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubnau</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Noise in gene expression determines cell fate in <em>Bacillus subtilis</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1140818%26rft_id%253Dinfo%253Apmid%252F17569828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTgzNy81MjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.32" data-doi="10.1093/cid/cir735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacNeil</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohn</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mair</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumbach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Current epidemiology and trends in invasive <em>Haemophilus influenzae</em> disease–United States, 1989–2008.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1230</span>–<span class="cit-lpage">1236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fcir735%26rft_id%253Dinfo%253Apmid%252F22080119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY2lkIjtzOjU6InJlc2lkIjtzOjEwOiI1My8xMi8xMjMwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Extensive variation in natural competence in <em>Haemophilus Influenzae</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.34" data-doi="10.1371/journal.pone.0005854"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maughan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Tracing the evolution of competence in <em>Haemophilus influenzae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMaughan%26rft.auinit1%253DH.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De5854%26rft.epage%253De5854%26rft.atitle%253DTracing%2Bthe%2Bevolution%2Bof%2Bcompetence%2Bin%2BHaemophilus%2Binfluenzae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005854%26rft_id%253Dinfo%253Apmid%252F19516897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005854&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19516897&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-4.4.717.35"><div class="cit-metadata"><cite>Maughan, H., S. Sinha, L. Wilson, and R. J. Redfield, 2008 Competence, DNA Uptake and Transformation in the Pasteurellaceae, pp. <span class="cit-fpage">79</span>–<span class="cit-lpage">98</span>, in <em>Pasteurellaceae: Biology</em>, <em>Genomics and Molecular Aspects</em>, edited by P. Kuhnert and H. Christensen. Caister Academic Press, Norfolk, UK.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.36" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.37" data-doi="10.1371/journal.ppat.1002151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mell</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumilina</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transformation of natural genetic variation into <em>Haemophilus influenzae</em> genomes.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DMell%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002151%26rft.epage%253De1002151%26rft.atitle%253DTransformation%2Bof%2Bnatural%2Bgenetic%2Bvariation%2Binto%2BHaemophilus%2Binfluenzae%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002151%26rft_id%253Dinfo%253Apmid%252F21829353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002151&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829353&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.38" data-doi="10.1086/671909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moradigaravand</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engelstadter</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The evolution of natural competence: disentangling costs and benefits of sex in bacteria.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">E112</span>–<span class="cit-lpage">E126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D182%26rft.spage%253DE112%26rft_id%253Dinfo%253Adoi%252F10.1086%252F671909%26rft_id%253Dinfo%253Apmid%252F24021408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/671909&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24021408&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327901800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.39" data-doi="10.1099/mic.0.040451-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marti-Lliteras</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Gomez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regueiro</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence for a non-replicative intracellular stage of nontypable <em>Haemophilus influenzae</em> in epithelial cells.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.040451-0%26rft_id%253Dinfo%253Apmid%252F20929955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.040451-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20929955&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286649800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.40" data-doi="10.1016/j.cell.2007.07.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortier-Barriere</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupaigne</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirouze</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pietrement</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A key presynaptic role in transformation for a widespread bacterial protein: DprA conveys incoming ssDNA to RecA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">824</span>–<span class="cit-lpage">836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMortier-Barriere%26rft.auinit1%253DI.%26rft.volume%253D130%26rft.issue%253D5%26rft.spage%253D824%26rft.epage%253D836%26rft.atitle%253DA%2Bkey%2Bpresynaptic%2Brole%2Bin%2Btransformation%2Bfor%2Ba%2Bwidespread%2Bbacterial%2Bprotein%253A%2BDprA%2Bconveys%2Bincoming%2BssDNA%2Bto%2BRecA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.07.038%26rft_id%253Dinfo%253Apmid%252F17803906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.07.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17803906&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249581500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.41" data-doi="10.1016/j.meegid.2005.02.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez-Losada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browne</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wirth</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viscidi</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population genetics of microbial pathogens estimated from multilocus sequence typing (MLST) data.</span> <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2Bgenetics%2Band%2Bevolution%2B%253A%2B%2Bjournal%2Bof%2Bmolecular%2Bepidemiology%2Band%2Bevolutionary%2Bgenetics%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DInfect%2BGenet%2BEvol%26rft.aulast%253DPerez-Losada%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D112%26rft.atitle%253DPopulation%2Bgenetics%2Bof%2Bmicrobial%2Bpathogens%2Bestimated%2Bfrom%2Bmultilocus%2Bsequence%2Btyping%2B%2528MLST%2529%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2005.02.003%26rft_id%253Dinfo%253Apmid%252F16503511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2005.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16503511&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236245400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.42" data-doi="10.1073/pnas.82.11.3731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pifer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Processing of donor DNA during <em>Haemophilus influenzae</em> transformation: analysis using a model plasmid system.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">3731</span>–<span class="cit-lpage">3735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPifer%26rft.auinit1%253DM.%2BL.%26rft.volume%253D82%26rft.issue%253D11%26rft.spage%253D3731%26rft.epage%253D3735%26rft.atitle%253DProcessing%2Bof%2Bdonor%2BDNA%2Bduring%2BHaemophilus%2Binfluenzae%2Btransformation%253A%2Banalysis%2Busing%2Ba%2Bmodel%2Bplasmid%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.82.11.3731%26rft_id%253Dinfo%253Apmid%252F2987941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODIvMTEvMzczMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poje</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">a</span> <span class="cit-article-title">General methods for culturing <em>Haemophilus influenzae</em>.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Med.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bmedicine%26rft.stitle%253DMethods%2BMol%2BMed%26rft.aulast%253DPoje%26rft.auinit1%253DG.%26rft.volume%253D71%26rft.spage%253D51%26rft.epage%253D56%26rft.atitle%253DGeneral%2Bmethods%2Bfor%2Bculturing%2BHaemophilus%2Binfluenzae.%26rft_id%253Dinfo%253Apmid%252F12374030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12374030&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.44" data-doi="10.1385/1-59259-321-6:57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poje</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">b</span> <span class="cit-article-title">Transformation of <em>Haemophilus influenzae</em>.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Med.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bmedicine%26rft.stitle%253DMethods%2BMol%2BMed%26rft.aulast%253DPoje%26rft.auinit1%253DG.%26rft.volume%253D71%26rft.spage%253D57%26rft.epage%253D70%26rft.atitle%253DTransformation%2Bof%2BHaemophilus%2Binfluenzae.%26rft_id%253Dinfo%253Adoi%252F10.1385%252F1-59259-321-6%253A57%26rft_id%253Dinfo%253Apmid%252F12374031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1385/1-59259-321-6:57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12374031&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.45" data-doi="10.1186/1471-2180-12-273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Power</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhill</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moxon</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hood</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Investigations into genome diversity of <em>Haemophilus influenzae</em> using whole genome sequencing of clinical isolates and laboratory transformants.</span> <abbr class="cit-jnl-abbrev">BMC Microbiol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMicrobiology%26rft.stitle%253DBMC%2BMicrobiology%26rft.aulast%253DPower%26rft.auinit1%253DP.%2BM.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D273%26rft.epage%253D273%26rft.atitle%253DInvestigations%2Binto%2Bgenome%2Bdiversity%2Bof%2BHaemophilus%2Binfluenzae%2Busing%2Bwhole%2Bgenome%2Bsequencing%2Bof%2Bclinical%2Bisolates%2Band%2Blaboratory%2Btransformants.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2180-12-273%26rft_id%253Dinfo%253Apmid%252F23176117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2180-12-273&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23176117&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.46" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC83MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.47" data-doi="10.1534/genetics.109.103127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harms</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wikmark</span>  <span class="cit-name-given-names">O. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starikova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sexual isolation in <em>Acinetobacter baylyi</em> is locus-specific and varies 10,000-fold over the genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103127%26rft_id%253Dinfo%253Apmid%252F19474200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzExNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNzE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.48" data-doi="10.1038/35084593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Do bacteria have sex?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">634</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRedfield%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253D634%26rft.epage%253D639%26rft.atitle%253DDo%2Bbacteria%2Bhave%2Bsex%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35084593%26rft_id%253Dinfo%253Apmid%252F11483988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35084593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11483988&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170332000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.49" data-doi="10.1016/j.jmb.2005.01.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A novel CRP-dependent regulon controls expression of competence genes in <em>Haemophilus influenzae</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">347</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DRedfield%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D347%26rft.issue%253D4%26rft.spage%253D735%26rft.epage%253D747%26rft.atitle%253DA%2Bnovel%2BCRP-dependent%2Bregulon%2Bcontrols%2Bexpression%2Bof%2Bcompetence%2Bgenes%2Bin%2BHaemophilus%2Binfluenzae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2005.01.012%26rft_id%253Dinfo%253Apmid%252F15769466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2005.01.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15769466&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227959200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.50" data-doi="10.1016/S0378-1119(98)00374-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerlavage</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomb</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Skewed oligomers and origins of replication.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">217</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DSalzberg%26rft.auinit1%253DS.%2BL.%26rft.volume%253D217%26rft.issue%253D1-2%26rft.spage%253D57%26rft.epage%253D67%26rft.atitle%253DSkewed%2Boligomers%2Band%2Borigins%2Bof%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252898%252900374-6%26rft_id%253Dinfo%253Apmid%252F9795135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(98)00374-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9795135&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076093100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.51" data-doi="10.1128/JB.00671-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mell</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redfield</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Seventeen Sxy-dependent cyclic AMP receptor protein site-regulated genes are needed for natural transformation in <em>Haemophilus influenzae</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">5245</span>–<span class="cit-lpage">5254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00671-12%26rft_id%253Dinfo%253Apmid%252F22821979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5NC8xOS81MjQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinhart</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herriott</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Genetic integration in the heterospecific transformation of <em>Haemophilus influenzae</em> cells by <em>Haemophilus parainfluenzae</em> deoxyribonucleic acid.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1725</span>–<span class="cit-lpage">1731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSteinhart%26rft.auinit1%253DW.%2BL.%26rft.volume%253D96%26rft.issue%253D5%26rft.spage%253D1725%26rft.epage%253D1731%26rft.atitle%253DGenetic%2BIntegration%2Bin%2Bthe%2BHeterospecific%2BTransformation%2Bof%2BHaemophilus%2Binfluenzae%2BCells%2Bby%2BHaemophilus%2Bparainfluenzae%2BDeoxyribonucleic%2BAcid%26rft_id%253Dinfo%253Apmid%252F4882021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiOTYvNS8xNzI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuy</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Addition, deletion, and substitution of long nonhomologous deoxyribonucleic acid segments by genetic transformation of <em>Haemophilus influenzae</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DStuy%26rft.auinit1%253DJ%2BH%26rft.volume%253D148%26rft.issue%253D2%26rft.spage%253D565%26rft.epage%253D571%26rft.atitle%253DAddition%252C%2Bdeletion%252C%2Band%2Bsubstitution%2Bof%2Blong%2Bnonhomologous%2Bdeoxyribonucleic%2Bacid%2Bsegments%2Bby%2Bgenetic%2Btransformation%2Bof%2BHaemophilus%2Binfluenzae.%26rft_id%253Dinfo%253Apmid%252F6975273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTQ4LzIvNTY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.54" data-doi="10.1038/nrmicro1234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of, and barriers to, horizontal gene transfer between bacteria.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DThomas%26rft.auinit1%253DC.%2BM.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D711%26rft.epage%253D721%26rft.atitle%253DMechanisms%2Bof%252C%2Band%2Bbarriers%2Bto%252C%2Bhorizontal%2Bgene%2Btransfer%2Bbetween%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro1234%26rft_id%253Dinfo%253Apmid%252F16138099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro1234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16138099&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231591100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.55" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMTc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.56" data-doi="10.1128/CMR.00040-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tristram</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Appelbaum</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Antimicrobial resistance in <em>Haemophilus influenzae</em>.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Rev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">368</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DTristram%26rft.auinit1%253DS.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D368%26rft.epage%253D389%26rft.atitle%253DAntimicrobial%2BResistance%2Bin%2BHaemophilus%2Binfluenzae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00040-06%26rft_id%253Dinfo%253Apmid%252F17428889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMzY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.57" data-doi="10.1038/ismej.2008.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comparison of homologous recombination rates in bacteria and archaea.</span> <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DISME%2BJ%26rft.aulast%253DVos%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D199%26rft.epage%253D208%26rft.atitle%253DA%2Bcomparison%2Bof%2Bhomologous%2Brecombination%2Brates%2Bin%2Bbacteria%2Band%2Barchaea.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2008.93%26rft_id%253Dinfo%253Apmid%252F18830278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2008.93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18830278&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263261800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Isolation and characterization of mutants of <em>Haemophilus influenzae</em> deficient in an adenosine 5′-triphosphate-dependent deoxyribonuclease activity.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DWilcox%26rft.auinit1%253DK%2BW%26rft.volume%253D122%26rft.issue%253D2%26rft.spage%253D443%26rft.epage%253D453%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Bmutants%2Bof%2BHaemophilus%2Binfluenzae%2Bdeficient%2Bin%2Ban%2Badenosine%2B5%2527-triphosphate-dependent%2Bdeoxyribonuclease%2Bactivity.%26rft_id%253Dinfo%253Apmid%252F165169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTIyLzIvNDQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.59" data-doi="10.1128/IAI.69.2.695-705.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morlin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valentine</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Serum resistance in an invasive, nontypeable <em>Haemophilus influenzae</em> strain.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DWilliams%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D69%26rft.issue%253D2%26rft.spage%253D695%26rft.epage%253D705%26rft.atitle%253DSerum%2BResistance%2Bin%2Ban%2BInvasive%252C%2BNontypeable%2BHaemophilus%2Binfluenzae%2BStrain%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.69.2.695-705.2001%26rft_id%253Dinfo%253Apmid%252F11159957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjg6IjY5LzIvNjk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzcxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.717.60" data-doi="10.1371/journal.ppat.1002874"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Insights from genomics into bacterial pathogen populations.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DWilson%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253De1002874%26rft.epage%253De1002874%26rft.atitle%253DInsights%2Bfrom%2Bgenomics%2Binto%2Bbacterial%2Bpathogen%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002874%26rft_id%253Dinfo%253Apmid%252F22969423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002874&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22969423&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F717.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/4/717.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/707" title="Sequence Profiling of the Saccharomyces cerevisiae Genome Permits Deconvolution of Unique and Multialigned Reads for Variant Detection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/733" title="Use of an Activated Beta-Catenin to Identify Wnt Pathway Target Genes in Caenorhabditis elegans, Including a Subset of Collagen Genes Expressed in Late Larval Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/4"><span>Volume 4 Issue 4, April 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/4.cover-source.jpg?itok=Rf0YhETZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/717" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ASWvxzCvJzc5UdwdH0hCq2GoJxV20hSRlmCGcZk888E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/4/717" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/4/717" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/4/717" class="active" data-icon-position="" data-hide-link-title="0">Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent Haemophilus influenzae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459828" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LRABSqKqRLjjyfptOc5NSsXxC3vGuHrnDvzWDswZufs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/717.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/4/717" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-9UWHJBbUqhbUTVcx9JeoyvZyaC_Diw242UYDAOjJnnE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/4/717" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/717.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459828" id="node459828--2" data-pisa="ggg;4/4/717" data-pisa-master="ggg;g3.113.009597" data-apath="/ggg/4/4/717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent <em>Haemophilus influenzae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joshua Chang Mell</span>, <span class="highwire-citation-author" data-delta="1">Jae Yun Lee</span>, <span class="highwire-citation-author" data-delta="2">Marlo Firme</span>, <span class="highwire-citation-author" data-delta="3">Sunita Sinha</span> and <span class="highwire-citation-author" data-delta="4">Rosemary J. Redfield</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">717-731;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009597 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459828--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua Chang Mell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BChang%2BMell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mell%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BChang%2BMell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:joshua.mell@DrexelMed.edu" class="" data-icon-position="" data-hide-link-title="0">joshua.mell@DrexelMed.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jae Yun Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Sciences and Technology Graduate Program, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jae%2BYun%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20JY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJae%2BYun%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marlo Firme </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Sciences and Technology Graduate Program, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marlo%2BFirme%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Firme%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarlo%2BFirme%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sunita Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pharmacy Sciences, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunita%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASunita%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rosemary J. Redfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosemary%2BJ.%2BRedfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Redfield%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARosemary%2BJ.%2BRedfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/4/717" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/4/717&amp;title=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/4/717&amp;title=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/4/717&amp;title=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/4/717&amp;text=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/4/717&amp;title=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/4/717&amp;t=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/4/717&amp;title=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/4/717&amp;title=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459828/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459828" id="node459828--3" data-pisa="ggg;4/4/717" data-pisa-master="ggg;g3.113.009597" data-apath="/ggg/4/4/717.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensive Cotransformation of Natural Variation into Chromosomes of Naturally Competent <em>Haemophilus influenzae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Joshua Chang Mell</span>, <span class="highwire-citation-author" data-delta="1">Jae Yun Lee</span>, <span class="highwire-citation-author" data-delta="2">Marlo Firme</span>, <span class="highwire-citation-author" data-delta="3">Sunita Sinha</span> and <span class="highwire-citation-author" data-delta="4">Rosemary J. Redfield</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">717-731;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009597 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459828--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua Chang Mell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BChang%2BMell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mell%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BChang%2BMell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:joshua.mell@DrexelMed.edu" class="" data-icon-position="" data-hide-link-title="0">joshua.mell@DrexelMed.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jae Yun Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Sciences and Technology Graduate Program, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jae%2BYun%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20JY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJae%2BYun%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marlo Firme </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genome Sciences and Technology Graduate Program, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marlo%2BFirme%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Firme%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarlo%2BFirme%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sunita Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Pharmacy Sciences, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sunita%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASunita%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rosemary J. Redfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z3, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosemary%2BJ.%2BRedfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Redfield%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARosemary%2BJ.%2BRedfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=717&amp;rft.epage=731&amp;rft.atitle=Extensive%20Cotransformation%20of%20Natural%20Variation%20into%20Chromosomes%20of%20Naturally%20Competent%20Haemophilus%20influenzae&amp;rft.volume=4&amp;rft.issue=4&amp;rft.date=2014-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mell%2C+Joshua+Chang&amp;rft.au=Lee%2C+Jae+Yun&amp;rft.au=Firme%2C+Marlo&amp;rft.au=Sinha%2C+Sunita&amp;rft.au=Redfield%2C+Rosemary+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459828/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459828/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459828/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459828/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459828/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459828/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459828/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459828/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459828/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459828/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459828/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459828/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/4/717" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/4/717.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/4/717.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/4/717.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/4/717.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
