<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/4/623" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/4/623.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/4/623.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/4/623.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/4/623/embed/mml-math-3.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/4/623.atom" />
<meta name="HW.pisa" content="ggg;4/4/623" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation" />
<meta name="DC.Identifier" content="10.1534/g3.114.010504" />
<meta name="DC.Date" content="2014-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Badke et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection has the potential to increase genetic progress. Genotype imputation of high-density single-nucleotide polymorphism (SNP) genotypes can improve the cost efficiency of genomic breeding value (GEBV) prediction for pig breeding. Consequently, the objectives of this work were to: (1) estimate accuracy of genomic evaluation and GEBV for three traits in a Yorkshire population and (2) quantify the loss of accuracy of genomic evaluation and GEBV when genotypes were imputed under two scenarios: a high-cost, high-accuracy scenario in which only selection candidates were imputed from a low-density platform and a low-cost, low-accuracy scenario in which all animals were imputed using a small reference panel of haplotypes. Phenotypes and genotypes obtained with the PorcineSNP60 BeadChip were available for 983 Yorkshire boars. Genotypes of selection candidates were masked and imputed using tagSNP in the GeneSeek Genomic Profiler (10K). Imputation was performed with BEAGLE using 128 or 1800 haplotypes as reference panels. GEBV were obtained through an animal-centric ridge regression model using de-regressed breeding values as response variables. Accuracy of genomic evaluation was estimated as the correlation between estimated breeding values and GEBV in a 10-fold cross validation design. Accuracy of genomic evaluation using observed genotypes was high for all traits (0.65−0.68). Using genotypes imputed from a large reference panel (accuracy: R 2 = 0.95) for genomic evaluation did not significantly decrease accuracy, whereas a scenario with genotypes imputed from a small reference panel ( R 2 = 0.88) did show a significant decrease in accuracy. Genomic evaluation based on imputed genotypes in selection candidates can be implemented at a fraction of the cost of a genomic evaluation using observed genotypes and still yield virtually the same accuracy. On the other side, using a very small reference panel of haplotypes to impute training animals and candidates for selection results in lower accuracy of genomic evaluation." />
<meta name="DC.Contributor" content="Yvonne M. Badke" />
<meta name="DC.Contributor" content="Ronald O. Bates" />
<meta name="DC.Contributor" content="Catherine W. Ernst" />
<meta name="DC.Contributor" content="Justin Fix" />
<meta name="DC.Contributor" content="Juan P. Steibel" />
<meta name="article:published_time" content="2014-04-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection has the potential to increase genetic progress. Genotype imputation of high-density single-nucleotide polymorphism (SNP) genotypes can improve the cost efficiency of genomic breeding value (GEBV) prediction for pig breeding. Consequently, the objectives of this work were to: (1) estimate accuracy of genomic evaluation and GEBV for three traits in a Yorkshire population and (2) quantify the loss of accuracy of genomic evaluation and GEBV when genotypes were imputed under two scenarios: a high-cost, high-accuracy scenario in which only selection candidates were imputed from a low-density platform and a low-cost, low-accuracy scenario in which all animals were imputed using a small reference panel of haplotypes. Phenotypes and genotypes obtained with the PorcineSNP60 BeadChip were available for 983 Yorkshire boars. Genotypes of selection candidates were masked and imputed using tagSNP in the GeneSeek Genomic Profiler (10K). Imputation was performed with BEAGLE using 128 or 1800 haplotypes as reference panels. GEBV were obtained through an animal-centric ridge regression model using de-regressed breeding values as response variables. Accuracy of genomic evaluation was estimated as the correlation between estimated breeding values and GEBV in a 10-fold cross validation design. Accuracy of genomic evaluation using observed genotypes was high for all traits (0.65−0.68). Using genotypes imputed from a large reference panel (accuracy: &lt;i&gt;R&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; = 0.95) for genomic evaluation did not significantly decrease accuracy, whereas a scenario with genotypes imputed from a small reference panel (&lt;i&gt;R&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt; = 0.88) did show a significant decrease in accuracy. Genomic evaluation based on imputed genotypes in selection candidates can be implemented at a fraction of the cost of a genomic evaluation using observed genotypes and still yield virtually the same accuracy. On the other side, using a very small reference panel of haplotypes to impute training animals and candidates for selection results in lower accuracy of genomic evaluation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/04/01" />
<meta name="citation_mjid" content="ggg;4/4/623" />
<meta name="citation_id" content="4/4/623" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/4/623" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/4/623.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/4/623.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/4/623.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010504" />
<meta name="citation_pmid" content="24531728" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="623" />
<meta name="citation_lastpage" content="631" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yvonne M. Badke" />
<meta name="citation_author_institution" content="Department of Animal Science, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Ronald O. Bates" />
<meta name="citation_author_institution" content="Department of Animal Science, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Catherine W. Ernst" />
<meta name="citation_author_institution" content="Department of Animal Science, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Justin Fix" />
<meta name="citation_author_institution" content="Smithfield Premium Genetics Group, Rose Hill, North Carolina 28458" />
<meta name="citation_author" content="Juan P. Steibel" />
<meta name="citation_author_institution" content="Department of Animal Science, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_institution" content="Department of Fisheries &amp; Wildlife, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_email" content="steibelj@msu.edu" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. M. Badke;citation_author=R. O. Bates;citation_author=C. W. Ernst;citation_author=C. Schwab;citation_author=J. P. Steibel;citation_title=Estimation of linkage disequilibrium in four US pig breeds.;citation_pages=24-24;citation_volume=13;citation_year=2012;citation_pmid=22252454;citation_doi=10.1186/1471-2164-13-24" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=Y. M. Badke;citation_author=R. O. Bates;citation_author=C. W. Ernst;citation_author=C. Schwab;citation_author=J. Fix;citation_title=Methods of tagSNP selection and other variables affecting imputation accuracy in swine.;citation_pages=8-8;citation_volume=14;citation_year=2013;citation_issue=1;citation_pmid=23433396" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. A. Clark;citation_author=J. M. Hickey;citation_author=H. D. Daetwyler;citation_author=J. H. J. van der Werf;citation_title=The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes.;citation_pages=4-4;citation_volume=44;citation_year=2012;citation_pmid=22321529;citation_doi=10.1186/1297-9686-44-4" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=M. A. Cleveland;citation_author=J. M. Hickey;citation_title=Practical implementation of cost-effective genomic selection in commercial pig breeding using imputation.;citation_volume=91;citation_year=2013;citation_pmid=23736050;citation_doi=10.2527/jas.2013-6270" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=M. A. Cleveland;citation_author=J. M. Hickey;citation_author=S. Forni;citation_title=A common dataset for genomic analysis of livestock populations.;citation_pages=429-435;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_author=R. Pong-Wong;citation_author=G. de Los Campos;citation_author=J. M. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=R. Dassonneville;citation_author=R. F. Brondum;citation_author=T. Druet;citation_author=S. Fritz;citation_author=F. Guillaume;citation_title=Effect of imputing markers from a low-density chip on the reliability of genomic breeding values in Holstein populations.;citation_pages=3679-3686;citation_volume=94;citation_year=2011;citation_issue=7;citation_pmid=21700057;citation_doi=10.3168/jds.2011-4299" />
<meta name="citation_reference" content="citation_title=The Genetics of the Pig;citation_title=Genetic Improvement of the Pig;citation_author=J. C. M. Dekkers;citation_author=P. K. Mathur;citation_author=E. F. Knol;citation_pages=390-425;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_author=T. H. E. Meuwissen;citation_title=Using the genomic relationship matrix to predict the accuracy of genomic selection.;citation_pages=409-421;citation_volume=128;citation_year=2011;citation_issue=6;citation_pmid=22059574;citation_doi=10.1111/j.1439-0388.2011.00964.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. L. Gualdrón Duarte;citation_author=R. O. Bates;citation_author=C. W. Ernst;citation_author=N. E. Raney;citation_author=R. J. Cantet;citation_title=Genotype imputation accuracy in a F2 pig population using high density and low density SNP panels.;citation_pages=38;citation_volume=14;citation_year=2013;citation_pmid=23651538" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic selection using low-density marker panels.;citation_volume=182;citation_year=2009;citation_pmid=19299339;citation_doi=10.1534/genetics.108.100289" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. Habier;citation_author=J. Tetens;citation_author=F.-R. Seefried;citation_author=P. Lichtner;citation_author=G. Thaller;citation_title=The impact of genetic relationship information on genomic breeding values in German Holstein cattle.;citation_pages=5-5;citation_volume=42;citation_year=2010;citation_pmid=20170500;citation_doi=10.1186/1297-9686-42-5" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=H. D. Daetwyler;citation_author=J. W. Kijas;citation_author=J. H. J. van der Werf;citation_title=Accuracy of genotype imputation in sheep breeds.;citation_pages=72-80;citation_volume=43;citation_year=2012;citation_issue=1;citation_pmid=22221027" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=Y. Huang;citation_author=J. M. Hickey;citation_author=M. A. Cleveland;citation_author=C. Maltecca;citation_title=Assessment of alternative genotyping strategies to maximize imputation accuracy at minimal cost.;citation_pages=25-25;citation_volume=44;citation_year=2012;citation_pmid=22849718;citation_doi=10.1186/1297-9686-44-25" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Luan;citation_author=J. A. Woolliams;citation_author=S. Lien;citation_author=M. Kent;citation_author=M. Svendsen;citation_title=The accuracy of genomic selection in Norwegian red cattle assessed by cross-validation.;citation_volume=183;citation_year=2009;citation_pmid=19704013;citation_doi=10.1534/genetics.109.107391" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_title=Linear Models for the Prediction of Animal Breeding Values,;citation_author=R. Mrode;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian linear regression package in R.;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=M. A. Pérez-Cabal;citation_author=A. I. Vazquez;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_title=Accuracy of Genome-Enabled Prediction in a Dairy Cattle Population using Different Cross-Validation Layouts.;citation_pages=27-27;citation_volume=3;citation_year=2012;citation_pmid=22403583" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. M. Ramos;citation_author=R. P. M. A. Crooijmans;citation_author=N. A. Affara;citation_author=A. J. Amaral;citation_author=A. L. Archibald;citation_title=Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.;citation_pages=e6524-e6524;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19654876;citation_doi=10.1371/journal.pone.0006524" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Strandén;citation_author=D. J. Garrick;citation_title=Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit;citation_pages=2971-2975;citation_volume=92;citation_year=2009;citation_issue=6;citation_pmid=19448030;citation_doi=10.3168/jds.2008-1929" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=T. Tribout;citation_author=C. Larzul;citation_author=F. Phocas;citation_title=Efficiency of genomic selection in a purebred pig male line.;citation_volume=90;citation_year=2012;citation_pmid=22859761;citation_doi=10.2527/jas.2012-5107" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_author=C. P. V. Tassell;citation_author=G. R. Wiggans;citation_author=T. S. Sonstegard;citation_author=R. D. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=P. M. VanRaden;citation_author=J. R. O’Connell;citation_author=G. R. Wiggans;citation_author=K. A. Weigel;citation_title=Genomic evaluations with many more genotypes.;citation_pages=10-10;citation_volume=43;citation_year=2011;citation_pmid=21366914;citation_doi=10.1186/1297-9686-43-10" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=A. I. Vazquez;citation_author=D. M. Bates;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_author=K. A. Weigel;citation_title=Technical note: an R package for fitting generalized linear mixed models in animal breeding.;citation_volume=88;citation_year=2010;citation_pmid=19820058;citation_doi=10.2527/jas.2009-1952" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=H. Wang;citation_author=I. Misztal;citation_author=I. Aguilar;citation_author=A. Legarra;citation_author=W. M. Muir;citation_title=Genome-wide association mapping including phenotypes from relatives without genotypes.;citation_pages=73-83;citation_volume=94;citation_year=2012;citation_issue=2;citation_pmid=22624567;citation_doi=10.1017/S0016672312000274" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=D. Gianola;citation_title=Accuracy of direct genomic values derived from imputed single nucleotide polymorphism genotypes in Jersey cattle.;citation_pages=5423-5435;citation_volume=93;citation_year=2010;citation_issue=11;citation_pmid=20965358;citation_doi=10.3168/jds.2010-3149" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. C. J. Wientjes;citation_author=R. F. Veerkamp;citation_author=M. P. L. Calus;citation_title=The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction.;citation_volume=193;citation_year=2013;citation_pmid=23267052;citation_doi=10.1534/genetics.112.146290" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=G. R. Wiggans;citation_author=P. M. Vanraden;citation_author=T. A. Cooper;citation_title=The genomic evaluation system in the United States: past, present, future.;citation_pages=3202-3211;citation_volume=94;citation_year=2011;citation_issue=6;citation_pmid=21605789;citation_doi=10.3168/jds.2010-3866" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=G. R. Wiggans;citation_author=T. A. Cooper;citation_author=P. M. VanRaden;citation_author=K. M. Olson;citation_author=M. E. Tooker;citation_title=Use of the Illumina Bovine3K BeadChip in dairy genomic evaluation.;citation_pages=1552-1558;citation_volume=95;citation_year=2012;citation_issue=3;citation_pmid=22365235;citation_doi=10.3168/jds.2011-4985" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection has the potential to increase genetic progress. Genotype imputation of high-density single-nucleotide polymorphism (SNP) genotypes can improve the cost efficiency of genomic breeding value (GEBV) prediction for pig breeding. Consequently, the objectives of this work were to: (1) estimate accuracy of genomic evaluation and GEBV for three traits in a Yorkshire population and (2) quantify the loss of accuracy of genomic evaluation and GEBV when genotypes were imputed under two scenarios: a high-cost, high-accuracy scenario in which only selection candidates were imputed from a low-density platform and a low-cost, low-accuracy scenario in which all animals were imputed using a small reference panel of haplotypes. Phenotypes and genotypes obtained with the PorcineSNP60 BeadChip were available for 983 Yorkshire boars. Genotypes of selection candidates were masked and imputed using tagSNP in the GeneSeek Genomic Profiler (10K). Imputation was performed with BEAGLE using 128 or 1800 haplotypes as reference panels. GEBV were obtained through an animal-centric ridge regression model using de-regressed breeding values as response variables. Accuracy of genomic evaluation was estimated as the correlation between estimated breeding values and GEBV in a 10-fold cross validation design. Accuracy of genomic evaluation using observed genotypes was high for all traits (0.65−0.68). Using genotypes imputed from a large reference panel (accuracy: R 2 = 0.95) for genomic evaluation did not significantly decrease accuracy, whereas a scenario with genotypes imputed from a small reference panel ( R 2 = 0.88) did show a significant decrease in accuracy. Genomic evaluation based on imputed genotypes in selection candidates can be implemented at a fraction of the cost of a genomic evaluation using observed genotypes and still yield virtually the same accuracy. On the other side, using a very small reference panel of haplotypes to impute training animals and candidates for selection results in lower accuracy of genomic evaluation." />
<meta name="og-title" property="og:title" content="Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/4/623" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection has the potential to increase genetic progress. Genotype imputation of high-density single-nucleotide polymorphism (SNP) genotypes can improve the cost efficiency of genomic breeding value (GEBV) prediction for pig breeding. Consequently, the objectives of this work were to: (1) estimate accuracy of genomic evaluation and GEBV for three traits in a Yorkshire population and (2) quantify the loss of accuracy of genomic evaluation and GEBV when genotypes were imputed under two scenarios: a high-cost, high-accuracy scenario in which only selection candidates were imputed from a low-density platform and a low-cost, low-accuracy scenario in which all animals were imputed using a small reference panel of haplotypes. Phenotypes and genotypes obtained with the PorcineSNP60 BeadChip were available for 983 Yorkshire boars. Genotypes of selection candidates were masked and imputed using tagSNP in the GeneSeek Genomic Profiler (10K). Imputation was performed with BEAGLE using 128 or 1800 haplotypes as reference panels. GEBV were obtained through an animal-centric ridge regression model using de-regressed breeding values as response variables. Accuracy of genomic evaluation was estimated as the correlation between estimated breeding values and GEBV in a 10-fold cross validation design. Accuracy of genomic evaluation using observed genotypes was high for all traits (0.65−0.68). Using genotypes imputed from a large reference panel (accuracy: R 2 = 0.95) for genomic evaluation did not significantly decrease accuracy, whereas a scenario with genotypes imputed from a small reference panel ( R 2 = 0.88) did show a significant decrease in accuracy. Genomic evaluation based on imputed genotypes in selection candidates can be implemented at a fraction of the cost of a genomic evaluation using observed genotypes and still yield virtually the same accuracy. On the other side, using a very small reference panel of haplotypes to impute training animals and candidates for selection results in lower accuracy of genomic evaluation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
    <title>Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"dV5Ou4ohESXzRd11WHhc9miyk8K-Qfw2_trSJEwQ9Xk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459840","apath":"\/ggg\/4\/4\/623.atom","pisa":"ggg;4\/4\/623","ac":{"\/ggg\/4\/4\/623.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/4\/623.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/4\/623"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/4\/623"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/4\/623":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lQoDGfgAO\\\\\\\/Wp3OaY\\\\\\\/areb6dQ+hdNQgtIFqeUgB6VCvBSSrgLMrN1X4uKQX3NkXJRWZiiDxNCWeW1Yov9pIoldF4dpMBuqVoceQS17hQmP\\\\\\\/+BEoJAWrdqo0onVHUdrl1O2Hr+PjNHp7NXBgm6vE\\\\\\\/KWF5f+l8TCVwY2jcxhLNPeAUTAlmBc7J5vymIDw10sP4e4zyMhafBZTzXMVwqBg9FhDKrwlXrT2Yj8qKdGLQPu+kBnFdXmVFfjqYwi4UOqNg74\\\\\\\/qae\\\\\\\/SjeT85BL8KMS42agORO0TOCRH2CVzCKLzjSbUXuZp7J2pz6sZ8lT6Pg8wIy9anOxAlbFfk4mGWynBSpxhU22cNZEo4wJPYeSZHFiwch2V\\\\\\\/\\\\\\\/6EmroujRaHVYOYyP8+6koivNghdJOJ2Cq66xD0GhIxVRj3nEcjtZYHo7CmIZpUNZJq9m\\\\\\\/+MVjm\\\\\\\/dJoGBkwzSvT87CUqKLaIbbUUBMR7dAb5tix3CmVhBfFfzl85llefIFTOWZK5\\\\\\\/pMBAZJqDtTaqRblnsoRGnLVxBw7WZ8ZdV7+CI0ZPjr5HaUKQnDUnqf3hvHS7me8e5BcSHLYJ4Yq6ZYtBT0wGMeE1S+peZAXh7vY9T\\\\\\\/w+f0OlrmCJ9JGcSY3Vq8ZEeH04PDE\\\\\\\/Vef9C6e3dEjzHRY6XjNCFcmB3A+RjNEihkLLoD3ZUaNApVYPztJqu05DQ0f060mT+Wwbqz2EE44QtV6XaA7zzoLiMbmNGGOs+weoDJmuPvPjOgC8qP81ZDelxmrXK2zic1TTE2LkwbZrIdCfAnoSU+8Quq2IIjX6jX63zoOYut9dRHwm\\\\\\\/9ylNmvugE3Bi6hMwsf8MI7Yfw0osYjsn1jxF\\\\\\\/1tBrBh1691vp1qL4FiAuE9wjXZxqKxaWPDQ\\\\\\\/uGPT7GB+\\\\\\\/a2sOH+w1voAnhGGvmdYW3XUJoK+ZAEA9EnDv8zeD0F6F0G+nKd0EJE\\\\\\\/8wG0iA1Yur2BPiyCbO+JiEUIcaLhkek3YAxkApTTi2MrEcxjOqKs1L6fr37NGP9IFOXCYzoxbOfDuXwZA0XKaGTyfPc3VzA6\\\\\\\/lGa29xngVmRtFLtTyhFl2tCxaE9oczd\\\\\\\/riqPSbtKny+cuHIEgzwsb7eHWQvHfi2JGF8bQSvzGbQeVirnx4ZYFT+pbiUe3ivNlupY2aqWJEywByomqZYZirS7SShHQZmU5cHYk5oR7R143KWJf84aTjxpC4ow7W6p6521nrA5oFQFTykxWtZOC3\\\\\\\/OfK0fiTl47aJ\\\\\\\/JX6hvT3wyp+Fiu1u+xJ7FzXCaW99evX1CDYw\\\u0022,\\\u0022iv\\\u0022:\\\u0022SZE5wSPJcoxMgB\\\\\\\/0eTroFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d0902975d0cbe70dc6a9f4e7ff738b1feef53236ca0fb496dcdd89236813c261\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gH+gWsIz3zAR5160j0RWC4gkAy9kiHlGm9pxuymjhsUgGscKxW8meUbKyEE0r6aIantESeFmkkrNfvtAh1+Xr9FW2qzbfGJPMPYCbKWTiZcwVKdx3QLXGWBnOuG+YHErp4LKrFE6t1zp7qHPIy5CcvesGhIvEekGtonErvFG9LYpRjPmt4V3f8F32ljUsy0J0Buxpai8GxGFJWHZZpBq++y1DFBOsFG3WwQtBg015s83QZV16\\\\\\\/+vP4lx9BLyp4E3xy8QKEVZV1N\\\\\\\/xmkAMZw88pncWUWYeoEtjzff3pdMZYutSB6UMS62X33otuZe+j29qXF9mOA+rcvUnb9iV4YftYxjWLhGElNWpu+CwxSV7EKfKoR\\\\\\\/2sPxWcq9ny1AwfstK75cTUXaH9\\\\\\\/MUdKOTY8Xk3B3b\\\\\\\/Keq4LstqdskDDOmeOTfh9+Pl7CjRcqLUSwlEUar3N5x2\\\\\\\/FctczB9UECvnRfJPfG4cHFejcjpCR8\\\\\\\/WqX9W2Y5cHmL4nK\\\\\\\/8xddRPRpve0qztlexycHaxia5eezUaQEjPk+HbFeoSZqS0j63i85imRd5bL1atHm9IiZzjtV94fxF2Ikd47TX68ZoZldBHPchJ0oGclPdVkW91XZFMtzGta+tjE1MasB6zLQRpSS9wMW7jJSiRAuZtPT3SdXE9Vf56IRj1Bp9jtANkBCqrmkci\\\\\\\/qWRnwFrn\\\\\\\/Cr+SpGZiCTrPG5CO1nGS31k2iAls\\\\\\\/yuQS2z4DzMouUsQOx9jKt5RuzeVYL8uPHdOhh1TahA\\\\\\\/xPsFU5v2A06LbOsf+V2E+vReom\\\\\\\/VpFMCFp3vFn44xG7m8R\\\\\\\/vdetCHrZJHSrHkjE5AlvaFXVAl5clKBWBmS2lK6tlju+3zYX+3F2WswHk234RSRW7s5h2IF+tIMtAkbgkHmcNQQuPDY6YySEgEiwm1P6Lw+yUykIrofPkJ4vKIZDdIoaw41QAqOCL7H463JGa2RLafA5C3USodFGgkP0OWFjYRNIXgRGfA6ez6l+hbu1CacixluzFiAk9GW9zK04oBX9myN4H0Cx79hSa1IhZibTck+Z1LKNHF\\\\\\\/3tgN9SkeqdcPN+8Ak9TccSotwHTBQKevBP3h27RqqRQuPFlPGlMhiW3giMM4DNat8gCLuvtv1LaQUegfkDOlQ512KdI3SVTg5hQjWous95lL8rnoT4M4Irgj6VFAh+2zMFOM2iaFDhsRqjM4M3E82\\\\\\\/vkKNjV+sK5ucVzb2W4QGdpcHeqNkCrOhPXyHuVy+j\\\\\\\/sI7TjD2x8vjYAHuwASJFesoVHvD8KJ9uCpsscTTYwa4JLB4fSlJWOan52fnVRImgM2pGzlNbSdDkILqbo3pl0AwOZCp06lj0VfTMeKUrGOX43S1r\\\\\\\/LebQf17pTyn3IV6UJfxB890CJqTNwHJ3Ot7+Z7u7yGD1xAc5zC8acUFPkeGehIc5VLyGbzJpAg1G\\\\\\\/HgKpvcgT33kU6eHJw7k\\\\\\\/R90xyM8tqSgZVQ1hXa8Vm1PIW0yPf+E4xGvMSRnecIO+727mfDyOGE6VsF8t1Bpxpz0EZS5vd2eTCxo4YQoP21bx3BwoOysc2b83Q3JjjkWwcWM\\\\\\\/i6hULh8a7KmNxMdX7cL0uIGH\\\\\\\/s0ZvUXO+eO3hmIC0rAMSIM72dBCuVEqQ3z78Z9UVEFobMeTz7tL++NsqoDAwdFr6AdP7kmYG7UXFgH1kSiYeob6q4qebHj2qNfUoXdRDghKLwO6X966ko8BcZu8SORR4X+2xa1O1\\\\\\\/JR\\\\\\\/sFkLjK1WzA3muIwTd0CxDsebbiKtgD6Vyxx+L\\\\\\\/8UO8iGZzzPx00HBmSgSsH1JTyzhnTbIAM+oI8I+zo0LnfkYK3rSqjsM7w+gZ7hzbfPwimcnDCswcwsDBdTB4uib8QsJ5u\\\\\\\/npC4PdByNDnCMu58hq\\\\\\\/14hYfXjQF0AmgA6wN2gfbndAx45QEfX8JeaBVrmbTONaG6JSDF70b0MMrtxE0WuJ569qFTYGsQSjL2UP9WDILiUemJ\\\\\\\/\\\\\\\/G4B0N5vNu5UswQPh6LMAEep+EcmM\\\\\\\/xacIbGOcOSPKCcgi1RD9JKwL+JEr7hoLdpk1gG3UtGc+mI0N4EVZbH3Jd5bvH4F234kx5l1\\\\\\\/xbDJTlZc6QLtsUC+nmqa8TzbLXt331MkaU5LhUi8lur5bCUorNGie9ItTHwaPKOU2GM4VnTRQvRLqIRU+wT3d59doX+nwCdxSryph3fJzHK3dDN5DOz1q4ljzsxMo2P4x1cC+4\\\\\\\/M\\\\\\\/0meE5jYJLfUpyNJ3CXnMIUES1n0Mm10tBn5mqbzZL93S27V7QROJAp22VxlnAviOH3S8sqKKJHzFrwkXber5J3fQ2mMyZEmGPFEGH+9VYLct19oFZW+qoUHJICEGc209cpqGRbkgbvLzPGuUtBAoUZAc5YazoSn30VgY5KZNSvBGFxXMZSBABXnWmyDUx903e+GKsGNtkBrGyMBs65uCPTFsBKh8ge\\\\\\\/ZG4LKqdasesMvxpsapGy0dorZBDWrEdfhbq10ze0N9h+t64NPAeYA+YghFh8DseNSanJ8RsY1NYSF5YZDfsSGXEIii4budnFGhQpe+1oZbIcB99qdYOU2jCFznklsz8Pn+sVyXn0SP\\\\\\\/cfhbiQcKwPoNav2OFlDPXxssalxfmOY51zRYjT1OZuiRbhSlmMYlyeU7imRYMAcmLTxyAB90G5l\\\\\\\/0igeIgOah3VVmR+f7VMYNzU268zqSee+c2QJar5k3CnWfOCG+CiaIo5oVE7sDlhhk69KRY3fvvWLNk1Ey2Fn3Sg4siNSzpQdeD0boYYJIz\\\\\\\/Gk6uliCF7\\\\\\\/QNEzESvrnuVcG+lUbo7yXaqj7mkRwQbFYOQkIOrLxjTSEtCkMhLuVMOKwTrx9xyRFQEBYrNGtLshuYlDPqvWQ7Bj7Mjd9DON8x8gjkGBOXnlwJl+OQTewsqMSXjTlXwPUiHH\\\\\\\/jQQ0oDgap+CcfnrHT75NFHJtQ6N5lLMIlDcwIFdg4c9aWKU2UenoQfVzBEPAbRgmUxJhuRYNsAjiAEmyXxGK99xnfYSZ45jVRuXXd3ZaR\\\\\\\/tgwWXlNu4\\\\\\\/usHltfJJgxmtZATXmFVgY78APsAuQVkktVHk5k9fyZfM82TR8HkJ+M64mG+bOG7Ayv0e4b3HOiIoAkwLKC3qa1gqX\\\\\\\/oYTzRjKgsf4zb7plSHMDFRTewS2ebIu4XqKmc2FvPv1ZXu3jEAclH8bE3fxKvgogDxZTlWiU1pMXcslYdggJRYb3kN5nebt+4W6VdrDde\\\\\\\/nWgiAIJWjxqgYrpUadTrbW0FaN+oijg+ouxn+Bl+vgWXg9YrryqkX8j8pK4C9OU0N80uwFvc8C\\\\\\\/0CJbY\\\\\\\/YaoHdS2ksoILyFRZHYe1KE1a37koVx7V4HUl+\\\\\\\/1PlgT7LJ+m1U3112HfsgJlQUc8I1vmTKsfJdzeXLqhP0Ab9wQG\\\\\\\/PixAdJluD6NNyk+oBFetTRxNwifiWNg8z0o3gB+ovK2r+bRcYX5FDVSGWsKx8vKl1Jn+26b6Z4CytXPXpVnthL0VNm4A5s9gD9VTsmRvdCWd89\\\\\\\/s9Lgn18ZEDrpNDQFPQcd+QffgwDyBYMV\\\\\\\/fEvRX\\\\\\\/\\\\\\\/zKoork5sfsctVawcNJi1inMiuYuXWLZuCuayvskgYEmQAFmBQfITNUDrZvYD4GNX52OqBv9mJYPVRTQmwhPHzwxpnM9P1YfXSlc9i7EMSK+IujUL270E4CKHbm3Pm+ZGGOJ6pq+jxWH8qy321OP3HSL+JJhs8iuv6ZBQ1Jb8oNPwzc5vca7ziGYCtHcWY187HYL3nfkeb1QxPuWPbt6nL6riKEWl9ZKbzz8pUdhxzIXRn\\\\\\\/zxrmoqOekPVoFq\\\\\\\/0gtSIhAZPv3hPCkhOrKIDLPq9iTD1yBWGNXDizxZgM5cDn41AuPWAHuuBkmMkgOlbS+cFXEYVoKr5G\\\\\\\/H53eR6\\\\\\\/j3IgR0b6+P9flDvqzUz\\\\\\\/muUYL84lGHI2R2ByWnnMmNyYZDBElnqYcEgFHZkQgtbQwQYPZK0R5IT5H7X7f\\\\\\\/yQcWXqe8xjAzbzmHvsELQKYj+slaW7Yg29sMI7pyUijGu3WjJG+MUZqbHkWbSklgIOECbIOy6uCnvx3vj0OETL76siy39n8jY3eTg1i\\\\\\\/5k\\\\\\\/GNrhb0V9K87t2yp90tLgRWx23Q\\\\\\\/W9bbTyteMk2OXW+aGISKx1MrK3Vp6zOnF1GvbCSy2hAP9D+o7nR0eYPxYctavrBJHVO5eL25dTUa7\\\\\\\/KOtxS5Zm71la\\\\\\\/dLc0VmmhKxSwr+woWh0HajR4Nm6fEzlmxQDkaORmp14szp1apca+cn35ov7hZbvQ0jz2996AuoPAcRZXMoJE6LcEmRV5bTsbG93Rzz2BwvJghy4wInmfwyOLB0JfcUJmJzos+YWZxcnrhR5qMi\\\\\\\/wtoQcxIEJix4VKsdNcsIJbvvp4So6NyHVmKEpnYZNzXgt5Z8UzojF7ECIehXJgFLI+eYRxHSqv+F+uaRAl7UUpdLrmYRWKqtQjlX2okvtGf6M2LBJh7j9ATgbhWfL2\\\\\\\/uPOhkXSPLz8nmnLFDwca8Hppu+dpr24wXZhowSH+S2QounxgH5xGpuGithuFBDubwBoqeTHOESp8V\\\\\\\/FEHzUtTVCdmjAeLVyLXgTLf63ZvMAkhZtpFovtAH0njWONOCr3s2q3XKXiMEcdvtjAndZRy+EcfpwYo38Zrjighgf5PJA0ETYdGU2ADzx39n7AXjHjyh3F+QDR43ZRqheVleRqgjjsil1EuQN5gyLtNEJFaV+4DhFjmGIPx6jPdsClC0RNvqYBxFR2nkl832nMIFlzUSk+gIykMpZtoxvbDW5ORJlL5kIlwXPTqndJUCv4Fn5lmldvivt\\\\\\\/asRD5bCgOUn0AKIp3+l+sNQPUWTQenifyzJ3qE\\\\\\\/VskZK3vZdc1GQdr6hfYJqUc+6nLMjdDd97ahVtO1AG0SdLbtALoPUBkhm+u2q7HzhcmvfsXitmi2MCZo0IgoFG449ua94q81FwpyXaIGzEriUYRVJo4eQIa34uIz7OJzYwdravWTr7NlhDkG1zE\\\\\\\/fhNv9sQev9FJ+j1+1PHeKeT3GrPjsgDxlNeWl2dWNJSd5A3RTihHhyXoI4AQYULRanE8GVSFqcWOdR3jCfvbNceiE\\\\\\\/Xn+8HaG4DjkkyWTMgIuh9IjfZWZ20HkFXCDfGThZqeeYW9NX3Su47hHykcijqyRYb00hxQIoZl95ap2I0P\\\\\\\/ckWW2JTktCaaY1hAxsYP4ASFfaY54YbgJVHQY+dU+wKTbLXnGpKgHHBX9Lw6KtJ1UsAtF3bh\\\\\\\/cNumGqq8B\\\\\\\/elkGTSEHKuESl2ENRzp1gHLahmrhnX9LO0\\\\\\\/ZEKqK9bOZyc6tj7Zjwp79F0I4orBq4naOQn9pGDTgzyqzw7wyXBzDfpIpIUX53Ev+RjWmMs5wVANJp+LhGY106gOLkGYNEIKoUjMSFkhEMRzvEnbN4VRi52+BiJbtqXQoE79+pgPlT9oWSyZOlWnffp7FGKDRurJcn4acaBb33r2bcvT3NBCllnHfvzsd8jHD+klAZs1BAwxJh7bFAz4K9PvH\\\\\\\/vmFNzvq3IOUSkL4Hb8Ii85FQGK67R3oo7rC4rpLz9ilXwv\\\\\\\/MAWa4JRxl8BspNH4xsbnLxvSsubCGAl9ram05YrMPbDoeOehfopw0wE7lhgGzmw7eztLwgL3g+fsji1JZFL7\\\\\\\/poTpfJjp8CLnpImRQLElpZhCde3ANNFTIu5jsWzV84TJnTLxsB4cOpOPyqRnVY18k5aSE1VWW50hOkPCXhe70vPu2WGbpv8G5+FDIFZV1tP6LUBOEQHJp1UgtqHARRqxW6uAzTuE+sbaObk0WsXMPYz0\\\\\\\/9Hn9xFNQkU8dTkefpPcl+wD97FHnuQIjm2T9X3OM9LohFM3j0uqflEOBRtqe3STxvRTwsEy1eKnwB54G26HQmKMNSEQOIlaZkPVW5ChYlngbw\\\\\\\/Fh1HA0hVeC9IlBKpL34CgSWOzEJ\\\\\\\/DaakFB3iXygXUmO93IDAtR0iPW75fg1RUfotsUSV5G9TqMI0bmpnOxGmGcl6mshcbiRwPgIEPDRWDcC5p7IV\\\\\\\/sxM6hfFesAwrPVB8J\\\\\\\/nIr7C2AM36s7Ov8WxGqJ1ToNFAr17v8KkXYvhXbnZNJbbW7w7sjq+bN7V0oDZCoJsMotcu7f8+uHhH6TE9xtwY\\\\\\\/2y1llB2OO2bTmzpfsEPFbvKd8XMUYyqOyzpLDWDXm8Rpz7PjKM2PPmWC8LyH4OhejudWZMEJQkb18V0yMBKbfUzuL1+AoIubR5XDL844ytcbBSwGHoewoUDWluI3i1Jd2\\\\\\\/oTI+2mTeMmO\\\\\\\/hGFCDY0F+BNcm0qYvKQDb4vph3sD2sVYxQOAb93UhEKm++UqqWiSl\\\\\\\/qIVd1hIXH\\\\\\\/nFM4Fl5NhUup1y\\\\\\\/gkDfkEMVPkpFVDJS0AfXCJmlGkLiRFoh3dhYU2DIP9kuMdL9\\\\\\\/H0tcz+orjJsDkhpliMKWMFBsSfSZjmY9kP700YXvtNbGg29\\\\\\\/\\\\\\\/3y9Q\\\\\\\/NV8ftwUnZVX8hSCgmcVBrst45bxrgoOepZ51rX7NR1uhE+EQ46rQjESvzRssVZwglnck7lA1gdAwECR4IxwCmTn3H8VRiJwtBPg8GcfUNRKdYeUl+CFhPBX3RxDdquD6c+MB1aZPTQIKL65xqYWGuvyCGk7erJ490hLwTANF3xAzrLxIPiQ\\\\\\\/VsckAHfj7HnRMI7X8G5JKZ1GMyGmEz+6Fg7wcpk6MjqUvF9B5qwQpJJIbBLKOG5lnIgTB\\\\\\\/ZdvC9jfE8XNlr+Vel5cB2Otnb5jqfWfZFXkdRJQg5A7ISPs644IumWUzKvhMq0chH0uX\\\\\\\/L+QA9i39lnkUiRd1Mz6lVRGqRadvAr4aDt1ZBDdDDR+Qr15UL8d\\\\\\\/2+pZHVLm6w6D0eNxpqwtq0VWaJCKFtCNWS4sLXrF5NTIXCv\\\\\\\/ios+COx1XACNRRoRiUn9GRfE3p7qG8IJStizfvBZ866UAKrJ2KR35csXOqZ8lJCNjNF24VKQFVkZCx8WeLRVwntlYosNQ60jJPS6hfXSk622rmDoUm5+9ApilGtcughGdWckwyxQ2FIawEHLs\\\\\\\/LUK09EeBlBuoUs7EmYVMmmSm+hCeSiv1GS2BULtiYjSQZFGgeMXyYmuqAL5ae6abCspBxD1mBKjpPZd\\\\\\\/MDq4rrAVNaKlr0+rZNguMt0+JCgSMjLEOtF7lQSGf4pHEeTUl0WhEW5LyxZOI5LJ2LAhRRXopWS81YaEXye2FIDoCy\\\\\\\/1+1yrnocMMl\\\\\\\/XZw+sEDE+ZUtfyrgvh0\\\\\\\/Y7asLl7kqoF57TPa8Bd3qeCn8cokbRpseY8QQ515AL78dXvf+B7X2NYtUgE+61CHkyloHNLtqdghjuHkYpClRW51Ex49ZHdV3u4ODnreUpTvjWbovaw8dg\\\\\\\/TSPYvRzBqtbHjujcL6D\\\\\\\/ftH1R\\\\\\\/fpSM2MUDnLoirN0\\\\\\\/6uftTArtDgAz92VGmuPpVqre4FiFAwZN\\\\\\\/40+xOxlfYZQDNKpKBad2Xy94t6ZMQwQjX6fqDsPQXSa9VYSCepX8MG66iXo08PzZnaQKDTpPiZOhDc\\\\\\\/iY8oTB6HXZ2UuI79SniVT2AAcmeMyOaI\\\\\\\/hlNyyYoqkIVNW\\\\\\\/cPjqyQvkfvzwcZv\\\\\\\/u\\\\\\\/mPv\\\\\\\/DX3yrIQ4+X6hxMNHKnu5RLo\\\\\\\/rE42f6dvhBZCJ8SStlwn6QYzrD+RPCbg1ZA6zn9o8Gjzd2Lz9riSpqvkpsOsf2IiCVEdPTvQVlxiLlGTvY++kAeErM+iGiUyCDSRZY\\\\\\\/sy98k+OXryyo68clFJUXEXfCszCDqCROsgm3Xt1mx\\\\\\\/DxPocsfkPmNe3guYgT6Igpa2uHOIGTkLke9bwCvBTx8iIrFo91mkWxSTNFGaTHWbXwujBW1cX\\\\\\\/6T\\\\\\\/UcwLWekBHkWGP2Jg9pXk2w+pPOfpggMWMUkvtdh\\\\\\\/DyOVv6n44uMsOhnzh5D6fE9WvfXvg+RuuCLjDmrLo9uutOQ3LfiP1bXAwbzG4yhIpmzsnHHdqQ4s6lHVTiE0BQYxEBDa+RIPE1bYxOv9u2w35DQOarUdz87uWE1Rs6DVzyOxlVXzpiEfAl8yXWCf6WjUAdDTVSezFbx48YxWvIhDSuqj4oI8qLZRN9tS6O53e2doTZ++Gj6BJzDIKpUZfLICyEYXUVei\\\\\\\/Q4beu1DXkCmunGg\\\\\\\/PCq84pfNrfL8Ep4h\\\\\\\/JqcWEK9eU4DoXCuVRvB3kfkqEYWNklpJv\\\\\\\/A+7gMrW\\\\\\\/cHPP+lJLmbAVDijcEjzAg1UMQokKBO0edk2pW9CoG3gNzOWcWoGrS7DXXtZgvYVy\\\\\\\/MBzpBhFHf65EzmEEkLgXt04Z6LCs+agfZcIidjGWausubiy3quWmGpjxiXdcRAiWJNxips6IW6b5wpvWjTyhFWtkXbblcBd1us+ODoYz8hNnv7abG2JJQ6EUTFl8XcdLmhWcELEZQDZ159P0VMsqXF\\\\\\\/urDN4JmgksL0PkxqmBKf9Zhlz8o8qLLgVMEA4iqKNlmsz7rXptBDe7inQZix6DBmkSFrYeyZmIqhJ08anWgcGlTT\\\\\\\/chgZwhCy01E2Ot7vkijOn7JGUv4Qts\\\\\\\/cXFZg3A2zm3hJLarWFUvYusRkLKKQF8TjXhqRfDXUV\\\\\\\/9sWke4Yt80o6pJGeD+Wq06f3ANl9XTOzyb490vd51j0Z8qSYrqkesPtK2uqplaLFZ4HtATj8j7gJSNFMcCwmPPBVTZ5mszrCXD\\\\\\\/kC3aEMlVtAYNhMCwYyKWfXXDD0aWwycgoAuZ03+PqPNG5Zird1FCMMU2ccDCQNlcGbluplAMa0waDdws93XTjrL8Qv1bpVaHZQkfduWuIvlCseGFaeiwRAsOHESvYrtfUdkwSrroPzBYbu29254rP1xgLtDgkB5DEI2UkIiGhls\\\\\\\/XW72fPon2kjU9r3\\\\\\\/uJuoDBINDt\\\\\\\/nHGzHJzBlPhGPLWsynKLNer8k4vvTL6MWq5P8TZGUArwAijt70QmqLwub1Aw4ZrPyvweLQanyIBqx61QLg2sMG8sLGMgt\\\\\\\/LytDDv339o2ZCXWPbYfnCGssgvE492eBM7nBhLrl2nQv7V4fV+W3xbTdvJ\\\\\\\/UR3E1vrg6VPhi\\\\\\\/0zYZs5iDcx60Z5gTY33LaQiXVHIsyoVyDvLI3cJFhoF7wfx0HRCa82vPez2DC4NLaGLPGWybrxZNQEJKr\\\\\\\/d8naqkyL37bpJ5O0naPEiunaf3xiXUaL14Er4Q4bVgcUDix26a\\\\\\\/5Lt\\\\\\\/cOC259NQ7\\\\\\\/Zb+UJOsfZsEtqqoX5PPW0aBxzkjDVCpgy3PxJLOvZI9Lvnw\\\\\\\/yikC1z3zHxzuXp5ROBOpKMiYWHfxLK1JvYfYeLNmvodsh4CYjXOXngBLXz9WYTzrh0SNBgSeNKIKEB9myXyhRmZTHzOG1AkAoIxM32olVSpu3Njo3afKLGcWscySosCMGFj8ltlIhaC+CgXrFtlO3YcZAVg9rh9L59KSVu5vbC24UVPsSZzSN3GEjMPDld8sezyQqMBsBLDqQIV6zbTrQdDsF7UfAXfWk6NboTGiIfpyZLJ59iVMh+DbvQgWnxoePx9MkUfjyAW5725ldUFHkYtsi6qTRjv8lX49VDCKloDSKRtoutDwXBoKtCWkoy4SW90jSPJdnVVSOPmy\\\\\\\/XjeqVUXM54tDryxHF8CLDmrXyOchtPhK7ErwDqeERozSIjC\\\\\\\/0Dzfg6EyTh3\\\\\\\/p2PbNNJPTpDkE3Awy5v2NUiG+4VdwCZU+R22EzqDmaoGP1VZIjq91vTu6x4drIl34yX2gPp474VDAmDllldqya+e0lCqWNzHkdt\\\\\\\/vc3gJYSSCtjN0kPNh7r05gDG1kfaiqdbzBgOqFzpl9D8V5y7fkEl\\\\\\\/cclYDMVwh+7zrE\\\\\\\/BgtfZFjHOWj+CvNtyMR9Xgj3P04BqtzTyglxatCqxKI1HqHhAmMRrhbSV4JW660ufCt9h0J5vbTo8MavfAlaa7ix7k\\\\\\\/xLBD4WbQCprsLSqEmXDXkiVhdyqESEo8pdJt5+SBJVKek+raHsXpJ0SMR\\\\\\\/5XVNHgSIES+IdS2WfDkB5EVZYLOLxX7DI204hsFk\\\\\\\/hOTU1RwGwj5\\\\\\\/LXLMfsHVCn5ZQCcN0Ml6MnOsQ\\\\\\\/YwDtpjY26fvgcXm0ciQpdHDnI1zGSP89jMzEP1EyjJ6jdeLLfB42YqlDdsMNhn7OdAoa7KHOVsGhTqytmd07Kh5jwPO8j8VIi6BoLAqFzx2xesHAs7xPRory0x2SpsBlHAIQc+r3Fm16Gzy8qvx38OxH4FHuqZ5+O1SkHnleyE9hNN0hJ4S4jkAKpNR9muHtwysd8bmTmZxz6c0sFNprb82TZcqP+tTVD9h6mN5SLv642GV79fEXK0xVVTBO6aZKZQCg06qgQ95\\\\\\\/3XM8vHSArGuuGysW0PNuYeDihvXTEwoknA0a0g5M1NFxAQb8HT6qyb8LoppQoay\\\\\\\/IrNxvvnlzItI8aUd+DTf7KM9MplGF0Pt4qahhK4TYjXIY4lU0Tju5S9D3Tcv6\\\\\\\/pBxAwnLBvz1ya7zOjHyfiUk+RKxUJetsa5JIG3obveotA4h1ySRNvm+v3nDiAIKWx0CqtgDz4Y16LjQ8jqpuq9acnZD6VwoHehhYUeWzEwbzanGRjuAegwwO+no7qRsw3opaRyjaeMw6iOeSOnClW3+WHmMxk6PWOAE8lT7CqWnUY4PVsUBs7rZ\\\\\\\/m3eC7wCA0bwJJkKEb8CkvFUrVKMuTHKu8\\\\\\\/8QKAfZwaMzPkCIMJj\\\\\\\/HGA3uUfhP5i02stcA3mBQBWGpRBhTFup+P64ykkZvKjSERQG32qkrehyRmcFkDWkfIaS5NWfuVPq83S2UNcqEFF0uBia4Ft2QOmK0DpBaNgf2cUi0qAKYdGemHQvGx9Ky\\\\\\\/ggVfgcbvjjtkIjoxRT2YF4OeCxaZN16mhwqf78Kviv+1yc9NsBZ\\\\\\\/pSpdONCBfocTD67bFtYyafUaaY+9ggTWmKuj8t1bpXrZ1wSCPZFzC0HSlvegaS0cpbToGfJ5c3ir+EpNDAMzVqBfWrjJvyXH7jYLZrqgRIPK4hVBKyhinjZb0hi6iLS4WE95bJBz5Zf5PtqYKPx6ewIMZ2yD9JOAM96BduFiZxUh3ar2h2HwSrfa1+f9h9PrNm1CsSvAVe8ebZp3yGT4XI6O8Yytot4nVG1NgJAn\\\\\\\/mGFCXCf3fFQS3WQsg2Aw7d8INAHnb31cAG+OwPpCw+LI9eh1UL\\\\\\\/hltDnkF7fPmVoOvdmiMxfR3KEQ32H9CQ\\\\\\\/PGnzG9JwySFWLNOJbY3eKeEEfxa6fOpAl9Rofou6IeRzu4nZJzeSuqBIYXNz5BKJL0LbR9qg8yzeD12FCoGAzZ1k1aua4bqwgsz4oVyrlmwXjrt9rKBFpzMYJzYiL6o6H7xxwNG8mz48JkjgplA4FsCjTUaXWzfCvIrzWbqRFOS2XEs5BZkXo1ZYSBdo4pobCdQmwx7GKoy0dJRK8dRGS7iwmA\\\\\\\/VNUipWvAagxHT359ncSpjTI3dRfSoCmlc0hMCNPt1RX6JYHSU2ykQH6mXtILcWLjWyOTYZiUg\\\\\\\/knIpvM9dAf02q4TAvmNwOaCjsbdc55O6i0fque\\\\\\\/yboWTlSBk4+B9G\\\\\\\/QNPx4xGYlyhM1rNTCnQVsAfGNLcXZ9zYDtEZFcRSS1K6eJ0JLAqMnrb6dB2DknQj+0qM1kQBGge9jt0mlIFT2jxRkPFEu7yiDUx5cQ6zfAALZRWHspvw3uOaBG3RIUEzHT1gZn1gPov3sxTbnfR0gcLUOcKlE23n3oGhlkkS8oGqeoAO91COcTIHL49Feq9sEBjzAcKKnS0LxzORgIAdA5aLfVKaYCGN329Lm7frqejfJJ11Moyw2jOeyLkYROM73FWsHvOjTwsDtqqXuh6mI0hmzVfGdGgh\\\\\\\/DtT44ztZMTiJrXm\\\\\\\/qx8QWGzleZowg\\\\\\\/+F5dvLEG+kpsin9jZpz7IUyQBRJX0EMzR0S++prNvrZ5V7liWwkX5Y7X22W55s+C6VDY5OGX8m8ukF8Y5EqnSb7Yb7xu7dMnXiEhH2ZAtyiv2r\\\\\\\/Gi8fjI7sKVCfbKSFFELtdwdDutbzjJUwJbyAdXv+ky4ql89ZNoH3VjZw5Tj8oCdXMn8786kgl2kvP3p6RubEGgdfOSVWPhGk1P0bT0+jv+wS8jBFkhxXDcHjY0MFY2G4L8QeA1UZZELNSU8ixqK8PSob1tUPQ2RiwEcDQN4vMTFate5Vm2dI+XTXZIIuFhqGqxObZp7Srcax4Agee2G3CplCvjNrPans1Bjs23nj7vhiIJThC83kvYDF9RE0VjGhSZcZyHv4PuFUr8Nn4AgAmNxh6YE9Zv94vlfU\\\\\\\/NISGreD6nrQGOGU4YrXitPf6TaZSDzZevwtCPz4F181jahsBrNZm84339nILhxMoQTSIH5tFRcaWyrDyzb9WZefUeGZuDOnxKMI6Cv6GQI1nqg3DQSbF\\\\\\\/Q2bHnwlQtEDHcSPJmxC2c6jPCJd7dJSxKbNI4Uuq5T\\\\\\\/yd6DjExyzuhJDitsaDlG0v8N71Bkk2cmOGaFct5C+4V9WsXyTjUFYg53bk7KRf1s6UyBjUPmw1cjYNdVhlrXyfup3mRjEbLzkpW37zoUKvmOQ0ner\\\\\\\/RPn3BebrsIp6+eLPsbTIHALDKbPKSzOHZnr6yu2lnC1Ua2Gc03H2m6rsZbK\\\\\\\/z8mI\\\\\\\/ZjQZDemd4s6qS6DUQU32gcFrh\\\\\\\/spR3k7s3vOoQIEZJf\\\\\\\/BK5N7L0CjZtoa5vQN+MP1lwQQ9w8EE1WR9WWBJvSFeMpn84BYN1tPuM5jyLWokznKRChxxmB7DdIAXRvOw1wBRIBmkD\\\\\\\/r1anRzeWNM3LdQJo\\\\\\\/ywGzAkEytAV1lVdO\\\\\\\/+FWFLfGokqhhg3T0i119bIgKeAyO8zMu+CnRufIWfuXcYGz9\\\\\\\/LxOZlWNUiTzoMMtBm8\\\\\\\/G+kIloP82Ky\\\\\\\/TQT0MN08gOuWtDwcOv88IX7bmPb4lidkVaN3UhLS\\\\\\\/AXFGAdcVXUXfpu4lTBZ8vounGSnaJpJNb8tX+xT+9IKgRfEXCEA\\\\\\\/X34g6UCPg70lula\\\\\\\/q5y7VWjPw78U\\\\\\\/g4kJik7cyDwh8zGCcKhHRGupiJKh7y7M0nfBnN80WeuD0tBQZD63QmAb3erSelX2+C9y7ea1ToHfu5rDk6yNyMftywRHWdeColNEkeU9YCd7kzCOxHKP7sG12eJCuRxxpVmh+0mbMNFxoPJAqSnBOXafBnyYwhCFiZ5xXiV50yI9W+rp7l9rnbe3R4WdW8zzCAbshlV+7qDsWkcRTLdH9qiZwRA0soOqSFigrsh7Q5Jp0dW7NbOb49IcXCrGlGH62o6Zwc\\\\\\\/YIl\\\\\\\/awh9t+Qhf3tLCPmQKXwOYajtW5MT8vT7hnOiWvSbgSl1aVa9Ef2L9CcxSSrejyITnChc18oJZ+1kHLKOmL0UKNhHQ5ly42DoYeIYE90B+z4mkeGeN7jaRevQMhpwoNFVJcx6KHaxrSEJaF1GRCHR7QEtfeUadGu+\\\\\\\/9ftUHqz3zqAyckqppEZfBZfTTQB+wI8tpGrnyH8aECwXd1eBomGhjzXKSbCLUKkoOoOC8L9ChBe0rCs6FJqsMsEuZHQFV0gGVedQBtJXnKfFiCblZ4Gl4e\\\\\\\/6UHkz7vtJvHNUPa5pSPyfqBTnTypxFs3utn72epryJLejWGWdgjhzNd+uEJ9l4wTI6EoMV2yc8EgZ\\\\\\\/UnvXKop8zTD1gxebVrYSz2vHZQC2FKnN29LOYa8uEzcRgXkQzSX\\\\\\\/4rTd\\\\\\\/xirQT\\\\\\\/TXgiZkySLgQsqE0EKCWts9JzYxofb1HkBLJHbHocEU6nARDYeux7gyE1oxofQCdoltm\\\\\\\/yKI5byfNmjFTPbwVok4CtKmsEuCyeSTNFzMaHsjX2dK8i0UFtafifMzbv\\\\\\\/DM2OBWZW+ecWCctY9roDt0Zsg46UWHl6IZRPvn7VO9T\\\\\\\/E23qbXGd3nLQ\\\\\\\/8QKXtMEfxsvEO6aLG331Exa8rN7\\\\\\\/Gdu0Gf7VhJVcAHelLY5jbht2D5YwbUtFixpRdH+WvUh4dZT6VGokSRu9\\\\\\\/EfjSjDvS4YZAqZ0\\\\\\\/bSXBJkdiJ9ely7t\\\\\\\/sauAtgU4L7IEQvLp4pxrykIvR2IgftHb9e2uQBGnauSVJXewUJFNR7o8BF6D4e32XWB3JxbsGGn4MqUk\\\\\\\/79jKWHnrA\\\\\\\/U0J7X6hmXW\\\\\\\/+lZhpyPzXhrXXNRzL+f21qxDkSKmtcm9IqRnBQkD6f54tTcvYr08prQhCXrCG7R6oXH8un0ji\\\\\\\/rUaRWDnQFouMUGteoyWMpvt76TWoZ2bBozyaGdS8IrBOv8R+5S0yqPoZGCrD6Tv90t5dc3\\\\\\\/OLN\\\\\\\/VvGfgaoiv4xnDF0Ke+NbyIf9XawOS6x1myZaUfoOGZJAX+1VfPX5O+pdIq4Psa7om9+rFcOcXAwH7\\\\\\\/hOtLpQR8sEOKETN9je2sbzDh7yMKECIE3GRRKh4ceqWgxWY7TuWiXyQDheYZsacV36ym0wwVKOwogY3fds33GaUP8BEbSxRcmWZ4lwDYxn7u4HCM0cgirEF\\\\\\\/lPuk\\\\\\\/HXdk0ZxktoUenGEW8qKg8zdysB6\\\\\\\/6TH8kXTwcuDucX9mUffTmylSesDwRWEqLKZUyv3gkUMMKIQGTyOq02475aMVqNYrEFh8ROEx9\\\\\\\/LdOLt1hXs6UScrlgwFB5dHY2ywgwrunq2jzKp701vZplsK+p9tnpSCWAGUg9n14Igqmn7alqLp3YexOXjWLqPcTK5EyM4rpusQX+7aHH2eCjg8w5N4TIYGh8bWABFl\\\\\\\/nmpX740zF1uMBUVaqhQlYzuZ6ecOUZdx8gJ2n1LhIUG3jJoXKOt2M56wsNpN+m1MGGvGtO8E5VmqkjT7GbkZ1e+9h4d4dv6GkVzBuJSUgDEr3u0NbZTjojn3FqZiW7L+Vyl2ygfXZrgTfsd3qzM5jrK4OROrhajlGcCNSyIJhs0+g4KnzPkjNwYRo6IlFFVXmmAAf8IiD6XLkwVHMMDRzakpuiIURwixXjbvK3I4SMYOq6ptGHNVQ9pEosgF0hu4vtS5Z8wOalkC0PyZ1kxgN2BC6Wt8C4jqKf23tghpwrfjZgedJYzLQXZBduVfCK6sMjQxTQxEXYma0f766y0OTw7wCEIRU+n7JFFo+HtX7QFXzFFVGFWJkO6xcHE95Fq\\\\\\\/6QWXePSdXqNfQI6PRmB9HRa+pmF28ZtQHqWn9gkcDJUjhXy4WL37SG+PtGHidzK3LoPHM0fInxSofIdUJ9r8xf5XVO9nNtKVTZrdUiFfl8EVGIQDncEk78pIY4llFyBSOZ1urolWLNRnnxuRayJTg98UhBar9smsbanx\\\\\\\/TB7rJQVBF2QCAm7gTaF+16Yn59Svrv5qh\\\\\\\/Y2gt745FHOGHl0KFFGN\\\\\\\/ljn4JpJ1L5yatXgRbBSl13tEm6DB8gbhLj7U+jPzfbhosmqw2FuHP\\\\\\\/lJ\\\\\\\/PP6WC0GHpKhW\\\\\\\/urfx5rJgfKhzkyzOSdHw8FJc8yiFIBAlJDcx7Gac68x1hapevO6gcyWHZdpF4+V+Melz74RV6Fh6RdSa20tvYuajX9rC\\\\\\\/vGI4auoCqtNqZRVrD\\\\\\\/pUFkvqzJv6+lGZAXh40vU12Js8D927ruo9+SziNp4Ad8H7VCp248crZ65H+cyLoTkdL77XnJaioED3r9FAlWsB6VmYekD3qM+2AQBH20TzbMpAZK94qlj9NMqdKMkPjXc7SrUcHEY6r8sXrK0u+ELj\\\\\\\/CDKSgESfg3x0F\\\\\\\/yItOkYn6Uppia3e6UFTiqFaNTLVGmUf79wlIHoKMW\\\\\\\/fWH9gEYzOFW1ZTy9UekR9VMm4lHV9RMQGL4oqsHPkdBjgXZByPoya8GoQboztZ2VAPuHroJ2SsqDrMyeNVpBzXDUKyDTQj2A9gAwM7+9Dml6f3RYbr853guzhq1+SCacUtppx+Zp3GIdodtdx\\\\\\\/+527Aep3J8hjpSrHEmhqBMk0YNnNISOZ9ia3B2AdeCA0SC\\\\\\\/7IUR+YQh2bh74mlV4ohU6JMyZZ4nJVBNBBXDrbh1nVtAmcxjMgtYlsAxaKQO32tK63tGUHMAYz9TEBemqwYS04hh5Mc0se0zEXSmxBHkVGC1WzomnJaKUJCY79NrSUwA\\\\\\\/NS6+HWCJt\\\\\\\/m7DGbsQAMaKPxu4m\\\\\\\/aTi4lbrczaVOePVUKSkinWB3mRwx2ro5pFQgs61OClKL+H4k7MTE8zuk5J9V5aQ9jSBEpoC\\\\\\\/Upr1li3wtHFWiHU40UIOShb8g7eovMb3iQfj2JNVgTPprCDc\\\\\\\/\\\\\\\/bTHScd\\\\\\\/ci5j+zu9ZpWRqLmp\\\\\\\/gor\\\\\\\/ys+GOFNSUMXmF29\\\\\\\/b1DUtP6dmiUItXzF7NY4whjW7mLPjIBZorlqJOj3WbUumMv9Q4Ja+E97FGeY6xaCnhaTrAWWB0xg\\\\\\\/pyBKd00Fy+egxQjDCz3m3A5aMCqKaY+eb\\\\\\\/oQvqJkUSIRyvwH7yUqT3gMZB7+aa0Vt8IUy0uLHyZxT34\\\\\\\/eVdbSbjdhlFVRT4znsdjdW6hRVdDc\\\\\\\/4+N9LZrM7yWA7Z2UU0IzOjTp2npFMJZPEo5fhXX4U5vNt8m8SYCrl1g7jOC82rC3Mwti7KS0E3g70hVP\\\\\\\/zR+3Z04SQ0M\\\\\\\/zP5YwRI1NCRW41vMAXDGXfbM7xbH2RP+QTxqrrL2NPnNh2c2U8zYOwKh34YDkgnm6wRVew9wwSFbpQbZJVaHHX4iWhu0jRBIPBJe1cxuzXct+epG4+Qg0UjEQnzOsmXfqRQhpehqwDNf6Ejx9Mn294ValleaQRSIjV9ND6sRG1opMNGtzg+Ba4opVsl1jrBIgBJn3TybRseWKY55PHVA9Iw\\\\\\\/RTN8nn2HIGjyTdPwMPUvyt4NxyyTJq\\\\\\\/2vOE1ol9NOOUAEWY+s++AXs6u+SKtMgLw0O2E3FgjrVho8SggCIKGPUcvDicD8Yzy5Q4E05Ko36Na1f\\\\\\\/z853Ec9ZJkHvOKnYeWHBTfj58VhqHi6bu6pt9gB880B+7itJLQT6RurCzRiP4fX2F0Q506hUWsU\\\\\\\/enVuTG\\\\\\\/4pu7qAjLfirrEr3Gvx9r1fYl2FK2j7GokjwiW4IJsBmn6fM81etFyepqYY2l4pks4b1GLemmxuH91Qe3Oc3iRe2V5nVhIIbZqvmPo1a8uNZZIk5UGn6tkLRLoPhsfNd81T3yMUb1Q7VVfZkPKMZ9LQfYpz5S\\\\\\\/mET4luPWPSE0Zn4oZN5wxvbkvqE3gcOdArdtbK0DUAg2SfOUA\\\\\\\/+AsdifqU8QEwBevkYXQKsHK8Om1bMOhCz91ZkuI0aaKkK66AuNDcVpu0O6jeXd88d1bKThpggzwxUCeBGjPAOIBLDOnqbBRu19hCUkoaM1DGi2UDS\\\\\\\/736f9DjU2BEzOPMnQY6TA8dkQJG+epEao52ltDJVeMYqZIi13pOeqDXLQkaAoMkmdUF1KHFt65ydh3wZQnm+YuZ07kgel+LsEyiM3MZ0\\\\\\\/p2S9xlYVzzATlUqUR0JPWKisGZ2aMtSvTm51pnw1gcn8EVEn+XjZe68O7r+A+ZGbJzasO4RDapAINLTqsoMcyBwOBUkvyiVEid5lmbd5dXatvY3igd34yIw3DQ5bntJuD9lWuWM3g+qi7AjrhhUncBbaZLDTowtka80Tup3ATvy0hGyJCawSnDxfxYznO9grBzBtshumcFGF\\\\\\\/mUdKjVCO1XrwzNo3USrjyUKAp8FKlRIwTNYN2iSX+KsCTA+7CvOI41HT8quYBUbUlp2DymopPiYsrBDnFFF\\\\\\\/Trqhk1pkVgbAph3W9HcFm9qo4lGt7XJG5LZas98M8oS77Xb0VtM6tcaj+kEn3sxm3RYnJFkWbLYTq0QnqpsA\\\\\\\/Kud8M3yw4uf0Eqq\\\\\\\/7ICBeWHHy9C4qgQ+tzfHB\\\\\\\/4NCbO8f354sX935yODN69BOhiuM4W2cU08FfNwi5ja38psE0\\\\\\\/WVTv7BZn4ZiHxH3RuXMZfZxLbfqn9Lhs4Spq8fOMkgGL2NZOrcA9xzSIgR2QgCoqI8KmrhiF++KQoFUNkfvlAV9L0hssw2R2TWoxecdPo72ez7P6cVONX+8ohE7BV5Ew4lz+sYeCvi6xs1ixD5LGq+RKvRDf+koB4\\\\\\\/bwOiN5y6c+oOzzqIDqM5\\\\\\\/ubDzOhb4kvQ6zFL521zE1oZCA2atakrsCG6eISLcik4yRQu0CrG+jQYjLiyW3OsL+J+JbgCb1wsp99FbHhWY3uCgfukA+eoWXZq0qkyVSZdpznAtRq5Hdmp5NrAJvy7E0X+ngG7AYM30NWf114ewVPhFLaBEkbvquSaicDcQLjYpBddiD47f8+2L1syTzV2B5zeMH5oS3Vo\\\\\\\/s7xZ9Unv7OKg7spmh2PhE\\\\\\\/oPfxFHJNp+ibcV3Do5BNOtEa+H05fQJOAEDlLOr22WwKNfSP0sTvBjNp\\\\\\\/5GEOOKBbbqHv\\\\\\\/d4eMfUXQGTfzZKf3ihCYVXRN1vEPKFQjDLvGxJ49p9c1tIj8BxcD\\\\\\\/VIBNDapothJTK9lMPFqoS9JCPNLcKAvLUPyGU63UIlYbHmh9urruYGuHdKLnYzH\\\\\\\/a9DHhNVnl6HjF2E7kHqf\\\\\\\/3erd64\\\\\\\/yHcgkR6FrrdXtw5b46k4zbIWFXmvuxFNnzbu9Vx26Yu7+4RsEFBSimo6okDrAH82LlOchPoiMsxl5PQpR\\\\\\\/HPowf3HRZgc\\\\\\\/VkTC0HiSFmCo1B1R8gESBMFeyqAqvp\\\\\\\/PIx05O\\\\\\\/CF1hT3jWRaGXXV18LZx97iClR97M\\\\\\\/AajkREe1PdZ6TSIP9JNgDJvpv\\\\\\\/LERviN8Kv8P8WwUpCuVPFCEel8\\\\\\\/GYtJmDvOoX4qvBJQSYxSxzX3jnkMWj1DgjMXzXos+s+WCBPLDliOn8beMhg+WzGiJPjl+nxnu1xyX7PP+9iWDjJJJtCO3rMv5tpY7herThmJol\\\\\\\/I9wKUsOjJmd5xTwSONTISYTfu6xadnjBRg6ZOTlCs\\\\\\\/Ep0wp5Zrhb9yfNIrJCP4zFbNYJSUyuimKeq+3CovBllNaddECcQJDCxi9CYFBsscnCsCNWi8JdIC\\\\\\\/d5rN0Ay+uAUJfj6cJZYpDTcFgqB+hYgJqYDpyVkXInvJWE0hUmRvkcl2ulx17OKg3aUU+so7v8MA7H+C+kQj\\\\\\\/jsmHHUcDTspdI3HVBOzJTEzwuh8+CSPpIeCoVWiLvveFXjxlfODHmttgZ1C9XKMw7uY27Qj5FCt49P4SPCn9pWtaPhOOpd7l90Z5XQUpJksk0MVnDY8k2gwL2\\\\\\\/PQtLGPPYe5DfFuuFWUyq+MBGtdssmrcqpgziQo+fPMj\\\\\\\/KBFPH6FeVgmmdmWcf1uoVqXjB6p4Ne0UsgCybBrrau+Z3FRKhnSJBIu4rKhIujAd0BAlA2IM86cg3Z8j2lEpOV0CFREw4mmhtFS234+iUsFdzpSl1HMwuYs3aN5AOHhTjHje01CtNaaSVG\\\\\\\/2KjzcNQUW4m89gzE2rGJwcP2WiM8YdljRJ3eoUhupnTjJsIaAH3Bfnf+mSS\\\\\\\/buGap7rMCDq\\\\\\\/FypyEIhhROwVIBsBUo1\\\\\\\/qnda8mluyGQdhqUBJy9FU\\\\\\\/UMNhGi2XSQMh+5+6tPz8a7Sv2K0YAuzcSrkdNuVPP92pZGB5CMnvU+cj7kutPlMLH0Hna39GSnyDoXVw3sKNYZ2GSkpmZuw2qfv27chPr1CdjafsPAKQiFGc\\\\\\\/sScj5WeobX4KZhSd32c+wnaRfoE1Qha1qskNaAXkf5rdS3Mskg0Rz\\\\\\\/1OTJVGrGEziRqKMG\\\\\\\/kdsdbLklhfbXEnlxw6mWp+COmB9eNPjU05jfRO2KVT5mmnBxCvvocT5M+bSCqpfF3LLEokQwmwTtfCGpJZWf9nxp+h1qxm2rrFoQelpyFn9JUJ68vW+T4Rceg7dQMRvPMAi7x9zUgO9f+36VL6T\\\\\\\/SVD3obueV0AeXCuqy8IZ9bYWwLRRaA3yPHOuoOlx7\\\\\\\/24D9waRQt7ZiaUjdWi6BpJQiJrerep2M5aj5MTF7CRfnsH+mEYuIV+fcjx71vCQeB\\\\\\\/d3opxe6B7ItOpkpmLJ48\\\\\\\/g8TNEGvC0ckG1h8TJbbW60LaHWMMPd7zNbXB9RFAOtmPI89rVjxiISrhCF6LIZO3xrTY50ZM2SAV82yyCJr9LpN6alBXjkpqeeRmDeRP9Ml8\\\\\\\/8F7FEBj8sCnsZu2kJxqQtZvUnPxdFtX\\\\\\\/on3ZdhZQZcjxph39nMJatMzmA2dJxpWAB81p6VS6ew0Vesmrj3EW1Ll+ivwqZIqIL364uun6jG2S4Hg6uWTKQsUtHDw6HgvvUvXBOvuheOBemDhP9fFJyX8Hj+snDjyDcfYOFws7of54Ccvhyi063mLSthkYdjDyuTNHOsKwlDdrsznKpnbhNhNDlFnNsJClc5li1HGYbKZOsbkCyfGJnsFlfaudrtHmD1mGSjPlxoEBV6xgRdpkA97Vh12Yr4fhShLLSs\\\\\\\/N71PtorfFj19YUxOECsTBrDKFi4XSEVA4uZxhKsMNvQQQxZRoFUSF\\\\\\\/\\\\\\\/95aG7NUB2nuzu0BTH+hQ3b4cM7TQ7SEbkv4CYGMUN\\\\\\\/kGY9leps0843lsRFz0W+T0PYXl\\\\\\\/9GuLqjxW1ME6BDMlM0SvVuzBwptR+mbvS4cS4xUiefDHpg8owdnhLkS6ZCNWmbq\\\\\\\/SIXq577rGBJtr8MHKV0Wd8QTmonXNHg9lfsirCx49m2M8qgEQ83rd4YCEe6kirU8eOjdQxLSzvRSHMX1noP8UYZ5pA7qCeBtdvkPKzlHktZDlypSqnOR94q13QC8in1UJR9mOqN5\\\\\\\/aLWc5TZbevQaFSJV6+Kn5jH7ZfKpPsIH7RYNnZRO3XDTqWqsAzIldSPEBaRyVHZiBzXF6o3jVL9pX1OWIWzSHldJJ0SkWlt3ySi\\\\\\\/L0lqIWdQt8Qu7JyuL9+YvDeffdkVHBHseMORhdSFvcFMHeyoW1GRl60oUoahZVE7QB\\\\\\\/P+RWRy5UYlmJhmws1InxBUVBCCbWFBYPxnXUe1D77n+CJoNcoXaAdDkTT6AsUXKO\\\\\\\/K9Q9VFymBpRhoG2YEgUywElEgfa2RT\\\\\\\/BS5SL0qK4gkKkM6uy4K1dV0d8ulCswYnUkAB4ItNVvtigcTaVFn4J7X0A0wIP8xRfmpINo7SDbvYb+w7hP87vcbgCgSjN\\\\\\\/m5v\\\\\\\/+4APQnAjogAzAjdd4+cqgwLWGefYk3dqAoczVbwicf0WPl+EpAdZWq4WNIjpbXMH0VwLi5zbGtTYHgRsb0TSeIFGjiIbcLDiu3tiI9oksfs5NUriSXaLdk0EVaJcRsAJp2uGh9tacL9w6B3ZbIUggtjIM\\\\\\\/6\\\\\\\/TsIGVMTVLIUV8GZ245ChXOHzOM4WBbbVB9aZVr9k5esA2LUWTq3YzKSLrn3NuUM+NiSV00jvfeUGpu09D\\\\\\\/KlNMtejB5hD+9Ly1222T7TWzc1sM7RRy5KjpbSqUUSFG+WOISU4RGSmVvk\\\\\\\/sO8+THHjyUwwGNLR1bw1ozE+fW6pNq7Ps\\\\\\\/s8UjMdBKD1YZ56LmLasYLf2ZrCCRvWj2lQdlEL0Z8CEMc1RCxAB3eSHCfgvWSTgWS4mwSohu2p23bSFFMU4ZB3YgmJCR3qkhD9Q0hQbkEeubKbm9PCKGBcUVZ0UO2oS1S8toogX+gVMs4p4cqXsFKp9ua44ljVGSpy27vb1ZummOhLUfnkTi0cJPpq67jBS7KcZ6\\\\\\\/kh8OYaIXWCS9qVJ8XlziV5qKg1ODqMxhtwNMFy7dpIInmq9KePxBZ3nmji1qaQJTABcNr42kv2hoiv9CAL3Ah6\\\\\\\/zvKqbM5V1RpL3zHZHZmyfDOn8uGRsZW2HIgwq6nk1CkWysiTIp2JecHlbwnqTMci+9ernFg0vyuc1FPuIj67N4RHVOX8nRkKoeo\\\\\\\/lU4xdbI2j3GHVoREhvmfOmkOkAwxoHnuRRVudukwa6Fj4TN\\\\\\\/RNIqA4s+aMqw9oZFX5exnYlbak0ZmPcLkXpUN+muvjsoTfnQP\\\\\\\/4jZjfqz68aH6B8C6GzE+Ob1WaVg48WuohBo84xEXZIq\\\\\\\/RX+BrdZk+622cdZkHxWKHkklmPdHNdBohNJGjaT7fDeD3h1iPyODeDOP1BDwpdjvNJnujPyeopjlbU6narriLyq1j64iHeK9nTJViX9dZTrZ7WJtMjcTPv\\\\\\\/lH77s7qZR2XT0i9pyekWVHtcxOTMMOw8up3eGEungsKqTIJKS+HYbZQcPcT9OUuFFJJ84A2hGSEHrtuW7i8PCxa39Xxak78bNxRJjkJSG\\\\\\\/5qMG1h1aJwtdlPKyfmjKyNs7Kle6CeMhf1EYGzI4GAYnfgOCotTnAp0xnkukk4ChQfF4tzxFMDvJgoYSDMm8jaglqK7caYPN763Gyn5tR38Wvg3QpqHsI+EYYpjdYA5Tmv6H56P580FDOxzNBhtSBMWo6a+MwROm7i4TrYR7untz0AWy0n9OHQrDPhHT1XqjtO7Aq5WDSUEurDlyGoOz79AFeEAvS1JOh3WhTvaBeL4qiVD1G5hmFvxrsVhNMC0ONE4BgbaHzZb8O1i2VNzp\\\\\\\/8awZSum5BUJCApz9QcOaAmlMk3zxn5Smpcx1mM0dmbdHZ8JmrZHEWLMDnIBSnBc4Jp8p7jcdINm9mgqTLHnYiO++2UqV8bfbHGn88jyfYWNgSeNv3T\\\\\\\/DYAGnes8oU5e2DKVcfsYXEgpsws\\\\\\\/3EwpFrGUFKBqDUqsfuBhnC8dsylm8RvVjpVjYNLqo6GsiT1dgQO5YlAxUQM6rpnKTjRzGGLGY1\\\\\\\/xkw\\\\\\\/FZiR15wQn\\\\\\\/kknukfrpY1KNi8UNZLQeUTE2W2gY0YvAdxZZaLZm7Xo3E3ZqPjacda1QD7v2wbvNtD\\\\\\\/O86EoGGjknNCtwU1yvBTLjD4xLAM7+2F12fFQd4ZplyTqNKs6RkDCpJe42n2CcQyqoIQ90EgCmfT3kmALhQ891Ot4AUdPDTjTYic1PjbxWZt8+ZFJlzmTYN6i55fcd24zv0K+qqXEJcHupNnwJQIzcFBoIq97OSRS4rUYfd5v6hdS908Yfj9kIfzjC7XACSK9itl85CVGGeYYghnDxiDx+gczyxD0yJOI4tK6hisK\\\\\\\/X\\\\\\\/rCzAmxvMU8IBHSb5k8tLPEXdEW3MSW63+JOWBJ07A0vgjZuW22gnfpi2WoOFaUvsDt3+GB5R2aXaeuEgfVQ1R83jhoXxy6q38e6\\\\\\\/cXe+2z3hmeCwhsP9zwean1qI3dfmKr0CxpCMl7Gp1UZMPU6AAHK6EptF64Ep65OmK5FhfdHmPOg5fvswke912X3XvVx7ecnLihI3Wvrg1FUsaGLDZYE2onrgsYpMNBsaJ66+YWD7rB3E8mm+5ey+yQSiK9pXf6sVMsqsGerWD+O7LQPLsINAOZH07llLQjVuXMzzKhWCpZbddlfIBT8dWFSTyWrP6UKrNGIIQTlpN\\\\\\\/rFNF9q4MjR4uzWqloFFsFdQivYpzIdjfxV88u6M\\\\\\\/3HHJbg7E35H3OKnJ9qGyhetaAaiDozhlc89KgMIn1YJoL64Z33GwW+yB+XinBmGB4OPKa1HhcecUen56N5\\\\\\\/Y\\\\\\\/pGYkHWQHfUb8zEhd6O0jnHPZRQ3ThKCyB12l8dART5rebFiw0KPZXwrDLapJk+FhyRPGyBfDE5LQbKtYDdQJyRzRd9ImJU343RR6v852txAURumz6tmr9+ok88LVRS\\\\\\\/64A\\\\\\\/afx4jLNZfAMCeqV4QYJRvanyrw28k7opbbum7Nb7jFIm+ncO335ClYHHR8m0tAfQog811rXJtR8n\\\\\\\/pti4+gpy4LYEwftstaqTV9Fhex4vFF3Ye3SKGTfJOgbGv1zBTmlKx9FrCiZiTyTrXL5DD2qwzKOZtjrzHiPNnZoWdJ09htA4UFZkZD8O1u0nlNZY4DTXVhmSE4fbdHYMfeQgfWpYT3DgFrgur0gmPg0DpBVvcJq62ZM5j2x47FLftWvKWtIirb2hsjVJhRbjQ+835HmeLENETHia3KlPXKeXP73+Z0WDsoPcsv\\\\\\\/ytnD25IF++ki0RqURIHpEyPPC1F1qg6c0Vtx50xWasBffID4IgU8NQcuqvTmL0BJc6TVaSM71F64TFjeSVUnWYVw3RfGMgYv2mhjj2i67aBoiJN4PHXxsHmlx8HLBoweB8aW44orw4xl3v4fjy1trpDPcD21zRDD85iOb5A42AwsiLU+lhwgvoIad7CV70NshfELBVHF53wC5JZIIRC3UM+Rf2dT4UdPUQh8AUH6CWYQ0G+n9juLjbxdn3kqKfUCp51h4jlnsNt5wMURUPd8H6ndaEltk8D1bUJhfPSOsekhgpv0bEmuilumWT3lFZhkZ6HO1ibmKY3oojVYY3z95AhC+5l4jfLJSmxyfhXGgyHfvoyymZYF4GkNGC81a47IwG2BTbny\\\\\\\/Qz1nY0cSKgSmnu2XbAv3pE+Yo3tJLdrvjjtAkzLbXJnoeETXMa9SDBwLKJz2QMrCATCV4ASzmROItCmVu55ydGoLpm82f5LVUmv81JEjOf30X7GujhFs75DYJYuXhJpXqh\\\\\\\/8iVqHQUPSg+vk5NB4n8qIh3sV\\\\\\\/RzXuUA6SrhPu00czW+OQ+1zPVroa95qPDZw2XWHIcdEsmqb1sSuLp+tJd1l3wrCjW6ZH+e73JaLUzMy16Xh8bbU9xRcVKqODAsHeatxk\\\\\\\/nSvqNlhB4kCul4NK+qB3xKLZXDS+XeSH1Xp2uC9HWfBbFqxOXaGJRKwnKtWQBm3HxgmsS2Kz+hsQcyRllbduwA5nD1iL7NkOZy2h50sqDZ95xbWyvGR2por\\\\\\\/Q5e2fCLZx\\\\\\\/+0dPcEIP4LxGcjLhy+LqTbZk\\\\\\\/T8obR0kxaKEx4Iyc7C2+ulVs1JLONn605Obwxb00PaabnTu0W0iIe8DrHwCXMbTq5i8BHln4zO7R4eOovk7BtWIAwSdfZcCZ3hQzT8eEmntYgECIB08h4EsQZ1YfsisDZpY5kITmYB4bC7+pLeaEET2cF0jIkOzu5lSGUDd6maZeE9xOcCM7HKjIkRrEORwEfqMdaL7zFVyG5hwaLLlqk3tQQ3yrGNUx9sdTq22iHGpMJ9edQnFVXkT714E3wK8ihUnI4zLCyCT5Y747lzl2EElc0Dz6gfhZjhEuk8wShAbxGNRslblqDwvTmLABP4YdJYE1HDle2yE5ufcS\\\\\\\/nKvX4sEGfHgFnpfYrB98Puw3f8OWnlAdMAR4t2LoX8Kmcx8AzGhKX5RZUf1Q3vHGlQSvy\\\\\\\/kjo32s8jI93WOcBjzaAU1nesbmqkqW+qV2dqh+\\\\\\\/eqwp2tYfQFen9WFhPYLiSNsb6hlinV3FoUJVO1MuSmSjdiTD+60VhS80AJnqSpoJs0O1ShYuv2gZ22WkutDJ\\\\\\\/GkFyV7WY4wZbFeuAxNRYotXDwWua5OV\\\\\\\/k25w27t0xlkj6v\\\\\\\/xjMihYFYCpAxNbjMDTuTUddfSplrF4DP2eKuj1OFJs3wW52ijM0bfIxPI51KfQPk3soty7yermY4VpFOy0jRx9B6BywbQAw7kKZmqlMLHOG1tbRUzFtEyZ0ZsiSuC404im9W0m5sq1wrjYHPL\\\\\\\/kQU3mbdDnIPjHClnNt5MVSe1NOefNnWOLiBFXLAEcFyJ2qjFiJ0OKRy+wb6M+8xrtM8NqbelD0fs14Rz1cdoAnLB3ATqC2IJBayXs5u2El2tmG93DnbVaNEIo6KUY0xOLn1i0DeNiAXvFlQZrbxtJ+CWXV\\\\\\\/B\\\\\\\/WKVbQXGyljMWHYcJvqpARJKvDTnQqFOguEUP08P6IZVzwN1YZo3+sIHPCrEvcvmphqX\\\\\\\/xgR57+y80vWb7DErhyjAR9EQ2B1RlMQUZu+aJEY\\\\\\\/9giWdAksqo\\\\\\\/G+9TLjAM44dsjFdbOvY9fW3wg4fJDsrNdWipGyzyZ0CTl5Br5VQ+yHe\\\\\\\/N4Bdk7cJNOmxsit0SCeCbUZWrUD2J7rOJP844BVTU4oDch20uYwq5K2rKtA9xytIb9rDVEuRCgvUMN9dClsh7ytRWuPD283hOfYvWdoA8rfvNlY3kydyOD\\\\\\\/B1gGL4iu1+hqZc1\\\\\\\/SCUXWPFE\\\\\\\/SIw4f+0c4FjG3fGC\\\\\\\/25Mpi4kQ5CiWoyVdv7GrBSpO0Dtw1\\\\\\\/LsTa3AauR4XrQyWjRytXpuvWjzqPc1s66KtQcU6EtEVmlSItZR\\\\\\\/zuR9hqwrMxEy22zPx4HXe84XeL8YPZSyOP6lu8jdVyuG0I1SEvxHnB4TrYtW6o1hIL6\\\\\\\/Jq2JU7HgMGb\\\\\\\/ILmLSVtT17JcZhN26CRn7EIaHD1mOQJqdM6ZU6r\\\\\\\/QjYdhg7c3ccNo9XW6wjScqcUf1JDQDC3EW7oY1QueReP\\\\\\\/e1cYq7mxkMWgO5igc0fayMwi0wYgVE7L07WyFSAnId6Uze5OSvJUfUQ9j\\\\\\\/R+xHDYi9+9Z+PaZlc8pBwUfmaEcXxkdgX99UdZmDnsIofXokAr107oRMvVZUhx\\\\\\\/HNe4lUepkr0Q\\\\\\\/xdpuy3uXsmoRad0Bj3zCfHxUid5vXVTTCPlxOabPskoiL2EowZ3pM1RhFCMaB5l9AemhCUW5YRj5tS0kagCRcXjsVfJOpNrsx9+e058TYbD4eB7q9YZjKZKP4sQBW+R5iYbHW8LkZrVRqv6HRlyyqn9MD1+ouMEKPBYIdh5aIR+qwbn0wPSQY0kbpGTPgxbkGTugScJnUGATdrs\\\\\\\/JAt\\\\\\\/lNqxZDF+31ofeG118ZgDKJvn+8rkmYrWu8xhP1k6yMfhVZm4NsQ4p\\\\\\\/pXOFHYkO0pG1+1T5aC5qZbbUBCh2NKBeQsZQSDGl\\\\\\\/7yRs1x9GmpCMxek1GLb3Lav4gINpy\\\\\\\/XYN3CxUBiXWUAgsnbQMW3P+iz4sznfTZRiuyybcXbMY5IfQRmII+Yu2gy5Jjp3olBqXQGJQ9EKtWUfsnJS8xS\\\\\\\/Hh21b+BnEkK5rNxeXjroWMz5QZrHgaCsrWf8RFJCtNTO05zyzIO2B49hFDVl8q6eKieY5tAUUOvqimSI\\\\\\\/wQCO7iIflqzQ3ydZwY6vxKD7isOEfejsZF2SPMHgU6XOT9gdPUjK0dsLjC18NJz7dFwwOGXLqQKFM5CYS1bTehZVi5Ad\\\\\\\/khH2mXjIchiTLsGftHjQ7yeyN\\\\\\\/8FMVyCTIIhBHv\\\\\\\/Y\\\\\\\/jSn9HuOfzuVvg0ICl5gyJm1jZhBrqmoetfv0DkiY\\\\\\\/\\\\\\\/014ab+MEs\\\\\\\/Cfxl6mWz2kr1kZG5qHroSueX0PNkpic8I\\\\\\\/Li404ejIw41E0UK1iwIGDI9ko01DB+V6QmeFihWzS+DUq7zge5W71mWx3+T+pWkDJP4yoPc2KWqlHLKpL\\\\\\\/rzXOfVzHDrW6zHAZ1OvTp2NaBncp9hzpEvHVyGYmx24ZPih+77ZbZ6tAUOxN8Y0Y90JNtGJTrfta94LG7vyjeShDpunw5TSJUguMfC8noPlJq\\\\\\\/K4tOcrgl9nxFlzfFbJy6cZ9vpgMWHD5zKz3tA6\\\\\\\/NOX2GNBgLKn0dmvoLTHbOMMvBQKycoXKtuOIMN1ppGyDZi+7qfE0lKzo4wc1fZNna3EUZAL5Y3jKkKmEpfXeZaY6FuMLkmALaD3X9L61PP98qrHK4odZlD+y7nMkIpzgRVYhqH+5VZ5pCwucKO+Kfz4LALqQLyQ9F+Gh9xM7V4XRxtmOlwFfIw5J2SjTPHHR9cwPtXdrLJCqARUQPvGRh5KFwkDiOOdF9DNfO53M89fryG2RVRatwZWMzMBr7ucaagucqq7R\\\\\\\/dD7PL5Cvs9NAP2T\\\\\\\/58oO5BAB9yrbUyCYHRFUM4Di8Vuv179RWQtvoRozRKIPACSMTTwRI\\\\\\\/jdpzEqBcjs0sHZTT9qX5YODhS1qNy1lUyt21KrRKYpgHdiTl8MTQpAW8lmsIC3+NFRqFrAjeslOlNGl9ITWyN33oM6\\\\\\\/7fO\\\\\\\/q5Idt5\\\\\\\/LmQGdQSY0rccsKRvFRlooK4qcIS8ByYnIAbEdvJDsuQkVHYq211B9CqhWnwOS\\\\\\\/RbLMJqmNlHobeTRXz0ZRL17geBEUN8Phefo4eT8SaEP5zvcLVdYYuST+4HGnVmm1wyrNSUSjFXvpCxx191r81nQSx+TDAtMiNYS98ctxgOg3rv+9UV5kLdeMJLzfHtJwcZDAW0\\\\\\\/avbpnhzXRFRWUF0R7PnSyDiyojiRooEmkm4eqFRgmKz+nObwKNOfrjqLtwOS24MAC8bhDEZEYhiNKXd17if9M3JDda20Tb2TNBHw1mPmqnVYOOSGpu\\\\\\\/upWRFnkfhn9rzjM0U49xRNmLf2Fl3+00atI\\\\\\\/StwZE7tP2TgNQlTTyFC+AKZe\\\\\\\/v\\\\\\\/DDwLmMJRao9EUH0PfD1nMgH5eU1ASDEWpiC6mh5W2h8fgBCP8DVEfIGyGdqleiIgN54RO0M74kzB04nmfYFVxeql\\\\\\\/9VvTn2cxw4yHM4Vu7z1p1UWjYesyD+Hw4YMm7zEVrjg+Lkhe0Yp11Afc6WXAA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022N6MCObSmpKMnK7fgYeN38w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227f2ea6842409a27536e59ac1fab0a8a5a5597f4b69b1a6ade1c33bf7a592e160\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ufoA3iFlM1JhV8h8PbQr55+VE1reESPkqHv4PlUptnNjSmFwgjvpfXgJYn0vexVkd9KMhz6+eb54BODseEvEafV7vM9NYdjFXDT4U4gbA6vs4XmaZvUhAu7FkHT5oMm2r3mSzE2LBUsy3DszAhMV6HBAnoM2HXKFeUVw7g3Wt0wqmzZfyn6ciYNw0gyJtNi1bPC7qFrGvxSNTbxmrIzid9JVN96zWm\\\\\\\/4D3+AIHvzh+CNzRm1pOMEM4ms5nowMrwLk7uwVxNnG7oVHLwyXqiTbfaSUabOHv7JXJ+L1NG1VIvCAVJbvP3pSsxLS57M4SwyyhgNkaiRcbZTiWbzJ71LAChRZrmrTcrYI2A1zbw\\\\\\\/XVM7UInSFz3qGDYl13YTbhBkGszuNrTXrUjhsUFGcaDSfhFIZeJ5y0PoITlroYWL5zHV\\\\\\\/XuvDSQMfINIYYbSKep5rlapG0snCZ7AKS1qt6h7dsWD3A1Jn8667SEE\\\\\\\/65BDhMIYau98l8FLc9ZBoc7guKQacfdnuLJR9sJH4FShnJznJwT+m2jCRa66Ppq\\\\\\\/kCBNDGkkMhfT9mp0SEMGX7MA9+FlxWtfZZWLtmfSKZGOKwZuQC3lmHxI4WhOZl+aPc8q5NicdUNq5ii51wkoRkGEzZ2I+QS4vImJe3uVehdi3+Srik8ln1b3iRnH\\\\\\\/ArTR+nm2FMJUArXd6Z4wGuHt3QhQMFeI6\\\\\\\/NE++1qB\\\\\\\/wKY0PBv6jcnne7nErCqAA8ANoQAkfkrnHEEbpuMiLWjLkiWfOxnECcx6YicYtl+M0o\\\\\\\/HKxrCzFdo\\\\\\\/\\\\\\\/2AkmWdlMLi+2Mi3nDEAshpo1c90z512AER4bdyER8iD6x5U4EUlq\\\\\\\/2yycjph2AhOUwtjUUoAiFkYdeUZhmxQJtP6d2Fe4C3J0zSQp8eYM74pDzQE7POcaOoG6cWqelPo\\\\\\\/F+vRlYfQetU0M\\\\\\\/feUg+SCT5qp\\\\\\\/OMzX1odEp+j\\\\\\\/ol2zU1OJseZPm5zWD45C2JedFE7TsfwN+8kVU4g5Bd3ZRc5LPml3E7BOGXpqNXzWBJD+bsQUa6E1jpQhzgL600TzkSVLPnOZGwdzeuhr5emyc+lVmbhj+5WBkFB8fPFlUCTng3gq7A1JsXVZ3GtzfFB9HbQ4OR01j+fh0vczZSSJOgF5h+4YssJIa9ufI2Ex+dQFXdrE\\\\\\\/4fz26Ydxph4B0dC7VJ\\\\\\\/VtL3GgDfgy6\\\\\\\/ICwmQoDZlJDAKu55TJ28WpQW8yVnBb7LOLVepUpggedrIm5aybLdsTSDee2ahn\\\\\\\/kIcgNbyF2lY\\\\\\\/YDSbWMq2+uZESJyMXGpk3zWdqCjTdpWxjGueDghJhZreF8DLAFmjMDdJImtT9G+oi\\\\\\\/Pjf4eaqHaJMeD8TweMkjVn+P+850yX0nLuu6S6tmx6tO1P6cU+MCyALmy+rFyDvQvl1d7zjRF54eWG87bEdCeSYnJrvW3uqyk8KQevYuqwajF1mwfMyjcqPxbmNWo1PgDs5mPOh9a0tOJPZvF44q\\\\\\\/Leer8FM29iPHS2iS8iEf8CwkygvWLt6jszUgvR3np8AN2J+7NrfeAh5gBEiOmXrlDXPDjiB0cCltWk4cVMB3tNglTw8ejx8M6SEGQworgyV8btGU8T0Xxxmyu+YEjJ84KhXfR8B4UQ+rCqpF3NxFtbzPyJKBMKJA8vHILO3djpJFVB8ysWq+XbI30eWyxJkukf7SQ5dYWBIMl+2tWqkO3kW6XL+7Vlc9GQLr6AG6EdeVz4So4ExU+9IWaxdyMv68mbtZuvxlxnAPpiFTNcD0X0wdwGrx\\\\\\\/bzvqQZFOgDf5keh48qTIDkI9aF6DwZN9HO7HupieRi9stlvOBGyp+Za4fqTrCRxKFpM9Oe+89FbNZuqalJt5TSPj\\\\\\\/xZtXpJa1Np02l+XVad\\\\\\\/rjr1\\\\\\\/ZeeA5uZb8JPt4yswMwbGPGJ7UkxsdHg6M1eUTPDHztxyJDiDPtw3HmKyfdjpw4E5lV570V+8s6CBs5TqgwM\\\\\\\/4S5QSpWQrj1N1CxOsBGn4cRPzyQM0a1cN9u76TBrw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nH369PzV+hAA3lRVdPTM\\\\\\\/g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002294513bd8f08321f2d8158f145b47a935897f55604e519b25f15089fd14d8a07f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459840 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/623" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-L6SitoCSCgvyG9m2Gn7LscGVWDSo36sllrkon_LCkQc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/623" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-q7rzQrxT1vsm8gytAza4sVrO9Gwdvi0wenvqc071YNE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/613" title="Edc3 Function in Yeast and Mammals Is Modulated by Interaction with NAD-Related Compounds" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/633" title="Centromeric Barrier Disruption Leads to Mitotic Defects in Schizosaccharomyces pombe" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459840" id="node459840" data-pisa="ggg;4/4/623" data-pisa-master="ggg;g3.114.010504" data-apath="/ggg/4/4/623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yvonne M. Badke</span>, <span class="highwire-citation-author" data-delta="1">Ronald O. Bates</span>, <span class="highwire-citation-author" data-delta="2">Catherine W. Ernst</span>, <span class="highwire-citation-author" data-delta="3">Justin Fix</span> and <span class="highwire-citation-author" data-delta="4">Juan P. Steibel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">623-631;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010504 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459840" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yvonne M. Badke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BM.%2BBadke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Badke%20YM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYvonne%2BM.%2BBadke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ronald O. Bates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BO.%2BBates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bates%20RO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BO.%2BBates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Catherine W. Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BW.%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BW.%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Justin Fix </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Smithfield Premium Genetics Group, Rose Hill, North Carolina 28458</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BFix%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fix%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BFix%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan P. Steibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Fisheries & Wildlife, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BP.%2BSteibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steibel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BP.%2BSteibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:steibelj@msu.edu" class="" data-icon-position="" data-hide-link-title="0">steibelj@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/4/623" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459840" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/623.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459840" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/623.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459840" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/623.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459840" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/4/623.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459840" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459840/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genomic selection has the potential to increase genetic progress. Genotype imputation of high-density single-nucleotide polymorphism (SNP) genotypes can improve the cost efficiency of genomic breeding value (GEBV) prediction for pig breeding. Consequently, the objectives of this work were to: (1) estimate accuracy of genomic evaluation and GEBV for three traits in a Yorkshire population and (2) quantify the loss of accuracy of genomic evaluation and GEBV when genotypes were imputed under two scenarios: a high-cost, high-accuracy scenario in which only selection candidates were imputed from a low-density platform and a low-cost, low-accuracy scenario in which all animals were imputed using a small reference panel of haplotypes. Phenotypes and genotypes obtained with the PorcineSNP60 BeadChip were available for 983 Yorkshire boars. Genotypes of selection candidates were masked and imputed using tagSNP in the GeneSeek Genomic Profiler (10K). Imputation was performed with BEAGLE using 128 or 1800 haplotypes as reference panels. GEBV were obtained through an animal-centric ridge regression model using de-regressed breeding values as response variables. Accuracy of genomic evaluation was estimated as the correlation between estimated breeding values and GEBV in a 10-fold cross validation design. Accuracy of genomic evaluation using observed genotypes was high for all traits (0.65−0.68). Using genotypes imputed from a large reference panel (accuracy: <em>R</em><sup>2</sup> = 0.95) for genomic evaluation did not significantly decrease accuracy, whereas a scenario with genotypes imputed from a small reference panel (<em>R</em><sup>2</sup> = 0.88) did show a significant decrease in accuracy. Genomic evaluation based on imputed genotypes in selection candidates can be implemented at a fraction of the cost of a genomic evaluation using observed genotypes and still yield virtually the same accuracy. On the other side, using a very small reference panel of haplotypes to impute training animals and candidates for selection results in lower accuracy of genomic evaluation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/genotype-imputation" class="hw-term hw-article-keyword hw-article-keyword-genotype-imputation" rel="nofollow">genotype imputation</a></li><li class="kwd"><a href="/keyword/swine" class="hw-term hw-article-keyword hw-article-keyword-swine" rel="nofollow">swine</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li></ul><p id="p-4">Genetic improvement through breeding for lean growth, reproductive performance, meat quality, and health traits is an important tool in the pig-breeding industry to assure its continued competitiveness and success. Traditional estimated breeding values (EBVs) derived from pedigree information have resulted in continuous genetic improvement but have several limitations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dekkers <em>et al.</em> 2010</a>). Notably, some important phenotypes are difficult and expensive to observe, impairing estimation of accurate EBV.</p><p id="p-5">The use of genomic breeding values (GEBVs), estimated using a large number of genetic markers across the genome, is expected to overcome a number of those limitations (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dekkers <em>et al.</em> 2010</a>) and allow for the selection of animals at a young age, thereby shortening generation intervals (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hayes <em>et al.</em> 2009a</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Vanraden <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Wiggans <em>et al.</em> 2011</a>). Several papers have reported the progress and success of genomic selection in dairy cattle (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hayes <em>et al.</em> 2009a</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">VanRaden <em>et al.</em> 2009</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Wiggans <em>et al.</em> 2011</a>), and it is expected to be equally useful in pigs (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Tribout <em>et al.</em> 2012</a>). High-density genotypes in pigs can be obtained from the PorcineSNP60 BeadChip (Illumina, San Diego, CA) containing roughly 62K single-nucleotide polymorphisms (SNPs) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ramos <em>et al.</em> 2009</a>).</p><p id="p-6">First implementations of genomic prediction in pigs included evaluations for total number of pigs born in a litter and percent stillborn (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cleveland <em>et al.</em> 2010</a>). The results of this study indicated that GEBV in pigs can reach accuracies comparable with those observed in dairy cattle if the training population is large enough (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cleveland <em>et al.</em> 2010</a>). In addition, several strategies to increase cost efficiency through the use of low-density genotypes have been explored, but the accuracy of GEBV was reasonable only for certain traits, likely due to differences in the genetic architecture of the traits (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Cleveland <em>et al.</em> 2010</a>). However, when genotypes were imputed with high accuracy, results for genomic evaluation were promising for several traits in a commercial pig population (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cleveland and Hickey 2013</a>).</p><p id="p-7">A question that was not investigated in those papers and that we want to answer in this study is how different imputation scenarios (of varying cost and accuracy) translate into accuracy of genomic predictions. The posed question is important because the relatively high genotyping cost per animal currently limits the widespread commercial use of high-density genotypes for genomic selection purposes in pigs. One strategy to improve the cost efficiency of genotyping schemes is the use of genotype imputation for a portion of the population. In the interest of cost efficiency, it is likely that selection candidates will not be genotyped using a high-density array such as the PorcineSNP60 but rather will be genotyped on a low-density array like the recently released GeneSeek Genomic Profiler for Porcine LD (GGP-Porcine: GeneSeek Inc., a Neogen Co., Lincoln, NE), a subset of the PorcineSNP60 containing roughly 10K SNP. We showed (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Badke <em>et al.</em> 2013</a>) that genotypes in pigs can be imputed from the GGP-Porcine to the PorcineSNP60 with accuracy of <em>R</em><sup>2</sup> = 0.88 using linkage disequilibrium (LD)-based imputation algorithms with a small reference panel of haplotypes (<em>N</em> = 128 haplotypes). We also showed that imputation accuracy can be further improved by adding animals to the reference panel (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Badke <em>et al.</em> 2013</a>), or in case of a pedigreed population, by exploiting Mendelian segregation and population-wide LD (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gualdrón Duarte <em>et al.</em> 2013</a>). In this paper, we use genotypes imputed based on population wide LD, offering a strategy that can be applied universally in any population, for which a suitable reference panel can be assembled.</p><p id="p-8">Our objective was to estimate the accuracy of genomic evaluation using observed or imputed genotypes. Moreover, we consider two contrasting imputation scenarios: (a) a higher-cost and high-accuracy scenario in which high-density genotypes from training animals and from a reference panel are used to impute genotypes in candidates for selection and (b) a low-cost and lower-accuracy scenario in which a small reference panel of high-density haplotypes is used to impute genotypes in training animals and candidates for selection.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Materials</h3><div id="sec-3" class="subsection"><h4>Animals and genotypes:</h4><p id="p-9">Data used in this study were collected from 983 Yorkshire sires. A pedigree of 4092 individuals spanning 22 generations and including all 983 sires and their registered ancestors was available from the National Swine Registry (NSR). Of 983 genotyped sires, 575 had their sire genotyped as well, 341 had a grand sire, and 597 animals had at least one half sib among the 983 animals. The number of full sibs was much lower, and only 110 sires had a full sib genotyped. Details on these quantities can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1/010504SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1/FigureS1.pdf">Figure S1</a>. High-density genotypes for these animals were obtained from samples provided by the NSR. Genotyping was performed at a commercial laboratory (GeneSeek) using the Illumina PorcineSNP60 BeadChip. The same dataset was previously used to assess the effect of genotype imputation (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Badke <em>et al.</em> 2013</a>) and is publicly available at: <a href="https://www.msu.edu/~steibelj/JP_files/imputation.html">https://www.msu.edu/~steibelj/JP_files/imputation.html</a>. Animal protocols were approved by the Michigan State University All University Committee on Animal Use and Care (AUF# 03/09-046-00). Genotyping rate of at least 90% of both animals and SNP and a minor allele frequency (MAF) of at least 5% were required for genotypes to be included in the analysis, leaving a total of 41,248 markers in 983 animals. SNPs that were not assigned to an autosomal position in map build 10.2 were excluded from the analysis. It was our goal to estimate the GEBV of male offspring of a sire and since sires will not pass an X chromosome to their male offspring, these SNP do not contribute to the sons’ GEBV (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">VanRaden <em>et al.</em> 2009</a>). In addition to genotypes for 983 Yorkshire sires, a set of 128 Yorkshire haplotypes was available as a reference panel for genotype imputation from a previous study (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Badke <em>et al.</em> 2012</a>). These haplotypes are also freely available at <a href="https://www.msu.edu/~steibelj/JP_files/LD_estimate.html">https://www.msu.edu/~steibelj/JP_files/LD_estimate.html</a>, and details on the design and phasing can be found in <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Badke <em>et al.</em> (2012)</a>.</p></div><div id="sec-4" class="subsection"><h4>Phenotypes:</h4><p id="p-10">For every animal and their parents, EBVs and accuracies were obtained for three traits from NSR through their traditional genetic evaluation. These traits were: backfat thickness (BF), number of days to 250 lb (D250), and loin muscle area (LEA). Descriptive statistics of EBV and accuracies are presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. All code and data used in this paper have been assembled into an R package, accessible at: <a href="http://tinyurl.com/MSURGEBV">http://tinyurl.com/MSURGEBV</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464224/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464224/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Descriptive statistics of EBVs</span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-5" class="subsection"><h3>Methods</h3><div id="sec-6" class="subsection"><h4>De-regression of breeding values:</h4><p id="p-14">De-regressed breeding values (dEBVs) were used as response variables throughout the analysis. We computed individual animal dEBVs and their weights (<em>w<sub>i</sub></em>) with the parent average removed by following the procedure outlined by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Garrick <em>et al.</em> (2009)</a>. We discarded records with a negative weight. The weight of an animal will only be below 0 if the unknown information content on this particular animal and its offspring is below 0, such that there is no individual information observed. This would be the case in a young animal, where all observed information came from ancestors and parents of this animal. To avoid double counting, these animals were eliminated from the analysis because they did not contribute individual information. After de-regression and filtering a total of 965, 936, and 938 animals remained for the traits BF, D250, and LEA, respectively.</p></div><div id="sec-7" class="subsection"><h4>Estimation of genomic relationship matrix:</h4><p id="p-15">The genomic relationship matrix was estimated from observed or imputed high density (~41 <em>K</em>) SNP genotypes. Genotypes were expressed as allelic dosage, which is the number of copies of the minor allele, such that genotypes were entered into a marker matrix <strong>W</strong> as a decimal number in the interval [0, 2]. We obtained matrix <strong>Z</strong> by subtracting twice the allelic frequency of the minor allele (<em>p<sub>i</sub></em>), from columns of <strong>W</strong> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">VanRaden 2008</a>). The genomic relationship matrix was then calculated as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-4.gif"/></noscript></span></span> is a normalizing constant (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wang <em>et al.</em> 2012</a>) summing expected variances across markers scaling <strong>G</strong> toward the numerator relationship matrix (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">VanRaden 2008</a>). The allele frequency <em>p<sub>i</sub></em> was obtained using all available animals (N = 983). Average relatedness between animals was obtained from the row/column vectors of <strong>G</strong>. We quantified relatedness in this study as the average of the top 10 relationships observed within the <strong>G</strong> matrix (<em>re</em>/10). The choice of top 10 as opposed of another number is arbitrary but driven by the fact that each animal had a very limited number of close and distant relatives in the training set (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1/FigureS1.pdf">Figure S1</a>). Moreover, other studies have used this measure and proposed its inclusion in future work on genomic selection to promote comparability (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Daetwyler <em>et al.</em> 2013</a>).</p></div><div id="sec-8" class="subsection"><h4>Implementation of prediction model:</h4><p id="p-16">Using the genomic relationship matrix from equation (1), an animal-centric model for genomic evaluations can be written as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <strong>y</strong> is the vector of dEBV, <em>μ</em> is the overall mean, <strong>a</strong> is the vector of <em>n</em> animal effects <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-6.gif"/></noscript></span></span>, and <strong>e</strong> is a vector of random residuals <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-7.gif"/></noscript></span></span>. The variance of the dEBV is <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-8.gif"/></noscript></span></span>, where <strong>R</strong> is a diagonal matrix with diagonal elements <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-9.gif"/></noscript></span></span>, the inverse of the weights of the dEBV (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">VanRaden <em>et al.</em> 2011</a>). Equivalently, the information in <strong>G</strong> can also be included in the incidence matrix of the animal effects <strong>a</strong> as follows (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Vazquez <em>et al.</em> 2010</a>):<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-10.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <strong>C</strong> is the Cholesky decomposition of <strong>G</strong>, such that <strong>G</strong> = <strong>CC</strong>′, <em>μ</em> is the overall mean, <strong>a</strong>* is the vector of animal effects with <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-11.gif"/></noscript></span></span> noticing that <strong>a</strong> = C<strong>a</strong>*, and <strong>e</strong> is a vector of residual effects <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-12.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-13.gif"/></noscript></span></span>. The variance terms for models (2) and (3) are equal, such that the two models are in fact equivalent if variance components are assumed known. Likewise, when estimating the parameters under these two models, we found virtually identical results, but model (3) was computationally more efficient resulting in a twofold reduction in compute time (results not shown). The BLR package (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pérez <em>et al.</em> 2010</a>) in R (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">R Development Core Team 2011</a>) was used to fit the mixed model equations. Model parameters <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-14.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-15.gif"/></noscript></span></span> were sampled from their corresponding full conditional distribution using a Gibbs sampler. Prior distributions were elicited based on equations presented by <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Pérez <em>et al.</em> (2010)</a>. The prior distribution of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-17.gif"/></noscript></span></span> were an inverse <em>χ</em><sup>2</sup> distribution with degrees of freedom <em>df</em> and scale <em>S</em>. To ensure proper priors with finite expectations, we set <em>df</em> = 3. The scale parameters were obtained as a function of the <em>df</em> and assuming values of the genetic variance (<em>V<sub>a</sub></em>) and error variance (<em>V<sub>e</sub></em>) (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Pérez <em>et al.</em> 2010</a>):<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-18.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-19.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-20.gif"/></noscript></span></span>, is the average inbreeding coefficient, set equal to 1 in this case, assuming no inbreeding. Heritability was assumed to be <em>h</em><sup>2</sup> = 0.5, such that after the value for <em>V<sub>e</sub></em> was arbitrarily set to 0.4, <em>V<sub>a</sub></em> was estimated <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-21.gif"/></noscript></span></span>. The Gibbs sampler implemented in BLR (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Pérez <em>et al.</em> 2010</a>) was used to obtain a total of 100,000 samples, 10,000 of which were discarded as burn-in. The reported estimates of <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-22.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-23.gif"/></noscript></span></span>, animal effects (<strong>a*</strong>), and GEBV <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-24.gif"/></noscript></span></span> were based on the posterior means of the remaining 90,000 iterations. We assessed convergence of the Markov chain Monte Carlo method as well as sensitivity to priors to ensure robustness of estimates to priors (results not shown).</p></div></div><div id="sec-9" class="subsection"><h3>Genomic prediction under cross-validation</h3><p id="p-17">Accuracy of genomic evaluation was estimated in a 10-fold cross-validation design. Approximately 10% of the animals were randomly assigned to a validation panel (<em>V</em>) in which predictions would be made, whereas the remaining 90% were used as the training panel (<em>T</em>) to estimate the parameters necessary for prediction. A total of 10 separate datasets were created such that each animal would be used for validation once. Across cross-validation datasets we fit model (3) to the training animals; we refer to that subset by adding a subindex <em>T</em>:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-25.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>to estimate the BLUP of <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-26.gif"/></noscript></span></span> (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">VanRaden <em>et al.</em> 2011</a>):<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-27.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>where the matrices <strong>G</strong> and <strong>C</strong> are partitioned into block structure such that<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-28.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>The relation between the BLUP for <strong>a</strong> based on model (2) and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-29.gif"/></noscript></span></span> based on model (3) can be expressed as:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-30.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>The GEBVs of training animals in model (2) were computed as:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-31.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>Subsequently, the GEBVs of the validation animals <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-32.gif"/></noscript></span></span> were estimated from <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-33.gif"/></noscript></span></span> using the following equation:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-34.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-35.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-36.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-37.gif"/></noscript></span></span> are estimated using model (4), which is equivalent to applying model (3) to the training animals.</p><div id="sec-10" class="subsection"><h4>Estimation of accuracy:</h4><p id="p-18">Accuracy of genomic evaluation is the correlation between the estimated GEBV and the unknown true breeding values (TBVs) (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hayes <em>et al.</em> 2009a</a>). However, the TBVs are unknown. Consequently, the accuracy of genomic evaluation has to be approximated using the available information. <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Hayes <em>et al.</em> (2009a)</a> proposed to express the correlation between GEBV and TBV as a function of the correlation between GEBV and EBV:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-38.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-39.gif"/></noscript></span></span> is the estimated reliability of the EBV. <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">VanRaden <em>et al.</em> (2009)</a> replaced <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-40.gif"/></noscript></span></span> with the arithmetic mean of the reliability of the EBV. <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Daetwyler <em>et al.</em> (2013)</a> proposed to report a simple Pearson correlation coefficient between GEBV and EBV to allow for comparability of results across studies. We estimate accuracy of genomic evaluation as the Pearson correlation coefficient between GEBV and EBV (<em>r</em><sub>(</sub><em><sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em><sub>)</sub>) and the Pearson correlation coefficient adjusted for the average accuracy of the EBV to facilitate such comparison <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-41.gif"/></noscript></span></span>.</p><p id="p-19">Accuracies of individual GEBV were obtained analogous to the accuracy of EBV in an animal model (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Goddard <em>et al.</em> 2011</a>) through inversion of the mixed model equations (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mrode 2005</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">VanRaden 2008</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">VanRaden <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Strandén and Garrick 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Clark <em>et al.</em> 2012</a>). The accuracy of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-42.gif"/></noscript></span></span> of the model (2) can be expressed as (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Mrode 2005</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Strandén and Garrick 2009</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Clark <em>et al.</em> 2012</a>):<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-43.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>This equation and its derivation can be found in <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Strandén and Garrick (2009)</a> and <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">VanRaden (2008)</a> and was used to estimate the accuracy of individual GEBV for validation animals.</p></div><div id="sec-11" class="subsection"><h4>Genotype imputation:</h4><p id="p-20">LD-based genotype imputation was performed with BEAGLE version 3.3.1 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Browning and Browning 2009</a>). We used the standard settings for BEAGLE: 10 iterations of the phasing algorithm, drawing four samples per iteration. Previous results from our group (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Badke <em>et al.</em> 2013</a>) and other studies (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> 2012</a>) showed negligible improvement in imputation accuracy as a result of an increase in iterations or samples per iteration. Imputation of 10K SNP chip [6890 SNP after filtering for minor allele frequency (MAF) and missing rate] were used as tagSNP to impute 60K SNP (41,248 after filtering).</p><p id="p-21">We implemented two separate imputation experiments that differ in the size of the high-density reference panel used for imputation: (1) a reference panel of 128 Yorkshire haplotypes or (2) a reference panel combining the 128 Yorkshire haplotypes with the haplotypes of all animals that are part of the training panel (~1700 additional haplotypes) in the respective cross-validation dataset. To assess the effect of genotype imputation on genomic prediction we considered the following four scenarios: (1) the reference scenario in which genomic evaluation was based on observed genotypes in training and validation animals, (2) genomic evaluation based on observed genotypes in the training animals and genotypes imputed from a large reference panel (~1800 haplotypes) in the validation animals, (3) genomic evaluation based on observed genotypes in the training animals and genotypes imputed from a small reference panel (128 haplotypes) in the validation animals, and (4) genomic evaluation based on imputed genotypes in training and validation animals using a small (128 haplotypes) but representative reference panel for imputation. All genotype imputation and subsequent estimation of imputation accuracy was implemented using the R package impute.R (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Badke <em>et al.</em> 2013</a>). To compare average accuracy of genomic evaluation across these four scenarios, we fitted a linear model with the average accuracy of genomic evaluation as response variable and the genotype imputation scenario as independent variable, adding the effect of the random cross-validation dataset in which accuracy of genomic evaluation was estimated as a random blocking factor.</p></div></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Accuracy of genomic evaluation and GEBV using observed genotypes</h3><p id="p-22">When genotypes were observed in both training and prediction animals, the accuracy of genomic evaluation, measured as the weighted mean of the Pearson correlation coefficient between EBV and predicted GEBV across 10 cross-validation datasets, was 0.68, 0.66, and 0.65 for BF, D250, and LEA, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). When the measure of accuracy was adjusted for the average reliability of the EBV of the training animals, the observed accuracy of genomic evaluation was 0.80, 0.82, and 0.76 for BF, D250, and LEA, respectively (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464120/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464120/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimates of accuracy for genomic evaluation and individual GEBV across imputation scenarios</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">We observed a significant difference between the estimates of accuracy of genomic evaluation across 10 randomly assigned cross-validation datasets for three traits (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). That variation across cross-validation datasets was partially explained by a significant effect of the average EBV accuracy of validation animals on accuracy of genomic evaluation (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) in three traits and a significant effect of top 10 relatedness on accuracy of genomic evaluation in D250. In general, D250 had slightly lower average EBV accuracy due to an increased frequency of EBV with intermediate accuracy (<em>r<sub>EBV</sub></em> close to 0.6, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1/FigureS2.pdf">Figure S2</a>). As expected, this resulted in slightly lower correlation of EBV and GEBV because the ‘true value’ (EBV) is subject to more uncertainty. Another source of difference of accuracy of genomic evaluation across cross-validation datasets could be the population structure. This would be revealed through differences in estimated variance components. We did not expected differences in variance components estimated from randomly assigned validation datasets. We confirmed this assumption by studying the distribution of estimated heritability <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-47.gif"/></noscript></span></span> and included the obtained results in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1/FigureS3.pdf">Figure S3</a>. We observed that the posterior distributions of heritabilities did not change across folds. Conversely, in the presence of population structure, the relationships of animals of different cross validation datasets will change (depending on who else is in the training set), and we expect that to affect the estimate of heritability.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464160/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464160/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Significance of variables affecting accuracy of genomic evaluation</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">The average accuracy of the genomic evaluation and the assessment of the accuracy of individual GEBV using equation 10 is equally important in a practical implementation of genomic selection. Average accuracy of individual GEBV was 0.69, 0.66, and 0.69 for BF, D250, and LEA, respectively with a 95% highest posterior density interval ranging from roughly 0.51 to 0.80 across all traits (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-40">As can be seen in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, the accuracy of GEBV (<em>r<sub>GEBV</sub></em>) and accuracy of EBV (<em>r<sub>EBV</sub></em>) are not linearly related. The accuracy of EBV was higher than the estimated accuracy of GEBV for most animals in three traits, especially when <em>r<sub>EBV</sub></em> &gt; 0.8. For a few animals with <em>r<sub>EBV</sub></em> between 0.4 and 0.8, the accuracy of GEBV was higher than their respective EBV accuracy. Hypothetically, individual differences in <em>r<sub>GEBV</sub></em> can be explained by the presence or absence of relatives of the predicted animal in the training set (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Clark <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pérez-Cabal <em>et al.</em> 2012</a>). We investigated this assertion in two ways: (1) by computing average <em>r<sub>GEBV</sub></em> for animals with different number of relatives in training panel and (2) by regressing <em>r<sub>GEBV</sub></em> on the average top 10 relatedness in the genomic relationship matrix. Following <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Pérez-Cabal <em>et al.</em> (2012)</a>, we defined close relatives as sires and full sibs and distant relatives as maternal grand sires and half sibs. We found that increasing the number of close relatives from one to four in the training panel increased average <em>r<sub>GEBV</sub></em> by about 0.1 decimal points (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) across the three traits in this study (from an average of = 0.63 to = 0.73 regardless of the trait considered). The presence of distant relatives in the training set also resulted in an increase of <em>r<sub>GEBV</sub></em> of similar magnitude when comparing individuals without any distant relative to individuals with at least five distant relatives in the training set (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). A similar relationship was observed when comparing <em>r<sub>GEBV</sub></em> with the average relationship to the 10 most-related individuals in the training set. We observed an almost linear increase in <em>r<sub>GEBV</sub></em> as top 10 relatedness increased (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), which was statistically significant (<em>P</em> &lt; 0.01). To further investigate the effect of relatedness between training and validation animals, we selected the youngest 87 animals (approximately 10% of the population) that included 82 animals with at least a sire or a grand-sire in the training panel. We repeated genomic evaluation with this validation panel and estimated the accuracy of GEBV. As expected, average accuracy of GEBV for this validation panel was higher than the average observed across the cross-validation datasets with 0.72 for BF and LEA, and 0.54 for D250. However, when looking at the range of accuracies observed for all 10 cross-validation datasets these values do not exceed the maximum accuracy observed. One interesting finding was that estimates of individual accuracy, or accuracy of GEBV predicted through the genomic relationship matrix, were much larger than the observed accuracy of genomic evaluation in all three traits (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Goddard <em>et al.</em> (2011)</a> proposed to use this measure of accuracy of individual GEBV when using them for selection but also to screen for animals whose GEBV could be expected to be highly accurate. Our results show that while it is true that individuals with close relatives in the training panel will have on average more accurate GEBV, the individual accuracies obtained from the <strong>G</strong> matrix would be overestimated.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/623/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of GEBV by observed accuracy of EBV for (A) BF, (B) D250, and (C) LEA rGEBV in relation to the animals rEBV, with the 1-1 line of the regression (green line) and a loess smoother (red line), which is a local weighted mean of the rGEBV. GEBV, genomic breeding value; EBV, estimated breeding value; BF, backfat thickness; D250, number of days to 250 lb; LEA, loin muscle area." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1941327178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accuracy of GEBV by observed accuracy of EBV for (A) BF, (B) D250, and (C) LEA &lt;em&gt;r&lt;sub&gt;GEBV&lt;/sub&gt;&lt;/em&gt; in relation to the animals &lt;em&gt;r&lt;sub&gt;EBV&lt;/sub&gt;&lt;/em&gt;, with the 1-1 line of the regression (green line) and a loess smoother (red line), which is a local weighted mean of the &lt;em&gt;r&lt;sub&gt;GEBV&lt;/sub&gt;&lt;/em&gt;. GEBV, genomic breeding value; EBV, estimated breeding value; BF, backfat thickness; D250, number of days to 250 lb; LEA, loin muscle area.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/623/F1.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/4/623/F1.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/623/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/623/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464193" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-41" class="first-child">Accuracy of GEBV by observed accuracy of EBV for (A) BF, (B) D250, and (C) LEA <em>r<sub>GEBV</sub></em> in relation to the animals <em>r<sub>EBV</sub></em>, with the 1-1 line of the regression (green line) and a loess smoother (red line), which is a local weighted mean of the <em>r<sub>GEBV</sub></em>. GEBV, genomic breeding value; EBV, estimated breeding value; BF, backfat thickness; D250, number of days to 250 lb; LEA, loin muscle area.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/623/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of GEBV by average top 10 relatedness between the individual and training panel for (A) BF, (B) D250, and (C) LEA rGEBV in relation to the animals rel10, a loess smoother (red line), which is a local weighted mean of the rGEBV. GEBV, genomic breeding value; BF, backfat thickness; D250, number of days to 250 lb; LEA, loin muscle area." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1941327178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accuracy of GEBV by average top 10 relatedness between the individual and training panel for (A) BF, (B) D250, and (C) LEA &lt;em&gt;r&lt;sub&gt;GEBV&lt;/sub&gt;&lt;/em&gt; in relation to the animals &lt;em&gt;rel&lt;/em&gt;10, a loess smoother (red line), which is a local weighted mean of the &lt;em&gt;r&lt;sub&gt;GEBV&lt;/sub&gt;&lt;/em&gt;. GEBV, genomic breeding value; BF, backfat thickness; D250, number of days to 250 lb; LEA, loin muscle area.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/623/F2.medium.gif" width="440" height="146"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/4/623/F2.medium.gif" width="440" height="146"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/623/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/623/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464268" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-42" class="first-child">Accuracy of GEBV by average top 10 relatedness between the individual and training panel for (A) BF, (B) D250, and (C) LEA <em>r<sub>GEBV</sub></em> in relation to the animals <em>rel</em>10, a loess smoother (red line), which is a local weighted mean of the <em>r<sub>GEBV</sub></em>. GEBV, genomic breeding value; BF, backfat thickness; D250, number of days to 250 lb; LEA, loin muscle area.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Effect of genotype imputation on accuracy of genomic evaluation and GEBV</h3><p id="p-43">Accuracy of imputation (<em>R</em><sup>2</sup>) for each animal was measured as the squared correlation between the observed and imputed allelic dosage across all SNP (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Badke <em>et al.</em> 2013</a>). Average accuracy of imputation was <em>R</em><sup>2</sup> = 0.88 for the scenario using a small (128) haplotype reference panel, and it increased to <em>R</em><sup>2</sup> = 0.95, when a larger reference panel (~ 1800 haplotypes) was used. In our previous study (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Badke <em>et al.</em> 2013</a>), we found that increasing the size of the reference panel led to an improved imputation, especially of SNP that appear difficult to impute, such as SNP with low (&lt;0.1) MAF and those located in the chromosomal extremes. These results were repeated in this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1/FigureS4.pdf">Figure S4</a>). For BF we found that the average accuracy of genomic evaluation under scenario 2 (<em>r<sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em> = 0.68), where genotypes in the validation animals were imputed with high accuracy (<em>R</em><sup>2</sup> = 0.95), was not significantly different from the accuracy (<em>r<sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em> = 0.68) estimated in the reference scenario, where all genotypes were observed. However, average accuracy of genomic evaluation was significantly lower (<em>r<sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em> = 0.66), when genotypes were imputed in both training and validation with lower accuracy (<em>R</em><sup>2</sup> = 0.88 using a small reference panel of haplotypes (scenario 3). For D250, there was no significant difference in accuracy of genomic evaluation between the reference scenario (<em>r<sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em> = 0.66) and the scenario where genotypes were imputed in the validation animals (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). However, when genotypes were imputed in both training and validation (scenario 3), the accuracy of genomic selection was significantly lower (<em>r<sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em> = 0.65). For LEA there was also no difference in accuracy of genomic evaluation between the reference scenario (<em>r<sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em> = 0.65) and scenario 2 (<em>r<sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em> = 0.65). There was a significant decrease in accuracy of genomic evaluation when genotypes were imputed with lower accuracy (<em>R</em><sup>2</sup> = 0.88) in scenario 3 (<em>r<sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em> = 0.63). To assess the effect of genotype imputation on the results of a genomic evaluation, we compared the top 5% sires (<em>n</em> = 46), ranked by their estimated GEBV across imputation scenarios. Again, scenario 1 was used as a reference scenario to compare how many of the top 5% ranked animals were also top ranked under the imputation scenarios. The proportion of top 5% ranked sires that were conserved when genotypes were imputed in validation animals with high accuracy (scenario 2) was 0.96 for BF and 0.98 for D250 and LEA. When genotypes were imputed with low accuracy in training and validation, the proportion of top 5% sires conserved in comparison with the reference design showed a small decrease compared with the design with only validation animals imputed for BF (0.88) and for D250 (0.89), and a more substantial decrease for LEA (0.81). Accuracy of individual GEBV is estimated using the genomic relatedness between training and validation animals. Using genotypes imputed with high accuracy (<em>R</em><sup>2</sup> = 0.95) the estimated <em>r<sub>GEBV</sub></em> remained constant in all traits, compared with estimates obtained from observed genotypes. Accuracy of imputation was correlated with <em>r<sub>GEBV</sub></em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1/FigureS5.pdf">Figure S5</a>). However, this does not imply that high imputation accuracy caused an increase in <em>r<sub>GEBV</sub></em>. Another possibility is that genotypes from animals with relatives in the reference panel will be imputed with high accuracy and their GEBV will also be predicted more accurately. We believe that this was the case for our population because the correlation between GEBV and EBV did not differ significantly when imputation was used (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>, compare scenario 1 and 2). Moreover, when genotypes were imputed with less accuracy (<em>R</em><sup>2</sup> = 0.88), the observed accuracy of GEBV was increased even with respect to the reference scenario (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>, compare scenario 3 to 1 and 2). This result is counterintuitive, and we investigated the reason for this increase. Examining the estimation procedure for <em>r<sub>GEBV</sub></em> we found that the increase was due to smaller estimates of the diagonal elements of the genomic relationship matrix between the validation elements (<strong>G</strong><em><sub>V</sub></em>) in the scenario with all imputed genotypes. This is the result of all imputed animals conditional on a small reference panel looking genetically more similar than they really are (because they are all imputed toward the haplotype frequencies in the small panel). Those diagonal elements of <strong>G</strong> were used to scale values of <em>r<sub>GEBV</sub></em> (equation 10), and smaller values in the denominator resulted in the larger estimates of <em>r<sub>GEBV</sub></em> we saw for animals in scenario 3. Comparing unscaled values of <em>r<sub>GEBV</sub></em> individual accuracy was higher in the reference scenario for all animals.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><div id="sec-16" class="subsection"><h3>Accuracy of genomic evaluation and GEBV using observed genotypes</h3><p id="p-44">The size of the training population used to train the prediction equation in this study was small compared with previous genomic evaluations published in swine (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Cleveland <em>et al.</em> 2010</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2012</a>), and especially compared with studies applying genomic evaluation in European (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dassonneville <em>et al.</em> 2011</a>) or US dairy cattle (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Weigel <em>et al.</em> 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wiggans <em>et al.</em> 2012</a>). Observed accuracy of genomic evaluation in this study was in good agreement with previously published results for genomic evaluation in pigs, assessing five unspecified commercial traits with comparable heritability (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cleveland <em>et al.</em> 2012</a>) and earlier results for two reproductive traits (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Cleveland <em>et al.</em> 2010</a>). Accuracy of genomic evaluation was high across three traits (BF: <em>r<sub>GEBV</sub></em> = 0.6810; D250: <em>r<sub>GEBV</sub></em> = 0.6603; LEA: <em>r<sub>GEBV</sub></em> = 0.6516). In addition, we report accuracy adjusted for the fact that the Pearson correlation between EBV and GEBV will underestimate the true quantity of interest (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Luan <em>et al.</em> 2009</a>). Assessing the variation in accuracy of genomic evaluation across datasets of the cross-validation, we found that the <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/623/embed/mml-math-49.gif"/></noscript></span></span> of the validation animals and their relatedness to the training animals were significantly associated to the average accuracy of genomic evaluation. Higher accuracy of genomic evaluation of prediction animals with close relatives in the training population (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Habier <em>et al.</em> 2010</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Clark <em>et al.</em> 2012</a>) and within closely related populations, with relatively small effective population size, has been previously reported (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Daetwyler <em>et al.</em> 2013</a>). Accuracy of genomic evaluation in this study was high despite the limited number of animals available for training and the inclusion of animals with relatively low EBV accuracy. Furthermore, we obtained accurate genomic predictions using an equivalent model fitting the genomic relationship matrix instead of a marker based matrix (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hayes <em>et al.</em> 2009b</a>), thereby greatly reducing the computational load. We expect that accuracy of genomic evaluation in this population and other US swine populations with comparable population structure and LD (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Badke <em>et al.</em> 2012</a>), will be feasible for commercial implementation and could be further increased through the inclusion of additional training animals with highly accurate EBV.</p><p id="p-45">Besides assessing the accuracy of genomic evaluation, we also reported accuracies for individual GEBV. The accuracy of GEBV is important because it can influence selection decisions. Moreover, as proposed by <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Goddard <em>et al.</em> (2011)</a>, <em>r<sub>GEBV</sub></em> can also be approximated prior to the implementation of genomic evaluation and used to inform the design of genomic selection in a population. The main difference between <em>r<sub>GEBV</sub></em> and <em>r</em><sub>(</sub><em><sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em><sub>)</sub> is that <em>r</em><sub>(</sub><em><sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em><sub>)</sub> is indicative of the average accuracy of GEBV in a population, whereas <em>r<sub>GEBV</sub></em> gives a measure of accuracy of each individually estimated GEBV. As expected, we observed that accuracy of GEBV increased with increased relatedness between the animal and the training panel. An interesting finding was that under a low accuracy imputation scenario, <em>r<sub>GEBV</sub></em> was overestimated compared with <em>r</em><sub>(</sub><em><sub>GEBV</sub></em><sub>,</sub> <em><sub>EBV</sub></em><sub>)</sub>. We traced this back to the diagonal elements of the genomic relationship matrix and attributed it to an artifact of the imputation using a small reference panel.</p><p id="p-46">Several previous studies in other populations and simulation experiments also showed the importance of relatedness for the prediction of accurate GEBV (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Habier <em>et al.</em> 2010</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Clark <em>et al.</em> 2012</a>), especially when the training population was small (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wientjes <em>et al.</em> 2013</a>) as was the case in our study. In addition, we observed that accuracy of GEBV was higher than accuracy of EBV for only a few animals that had mostly low accuracy of EBV. This finding is further supported by previous reports that implementation of genomic evaluation would be most beneficial for young animals with little information on their own and subsequently low accuracy of traditional EBV (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">VanRaden 2008</a>).</p></div><div id="sec-17" class="subsection"><h3>Effect of genotype imputation on accuracy of genomic evaluation and GEBV</h3><p id="p-47">Genotype imputation is an efficient tool to decrease the cost of obtaining high-density genotypes for selection candidates. One of the goals of this study was to quantify the loss on accuracy of genomic evaluation if GEBV were estimated from imputed rather than observed genotypes in selection candidates. Comparing accuracy of genomic evaluation across three scenarios of genotype imputation we found that for three traits there was no significant loss of accuracy of genomic prediction if genotypes in validation animals were with high accuracy (<em>R</em><sup>2</sup> = 0.95) instead of observed. However, accuracy of genomic evaluation decreased in comparison with the reference scenario when genotypes were imputed with lower overall accuracy (<em>R</em><sup>2</sup> = 0.88). When low-accuracy imputation was applied in training and prediction animals we observed a decrease in accuracy of genomic evaluation. Previously published results support that although it is not feasibly to implement genomic prediction based on low-density genotypes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2009</a>; <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Cleveland <em>et al.</em> 2010</a>) the accuracy of genomic evaluation is still feasible for practical implementation when genotypes in selection candidates are accurately imputed to high density (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Weigel <em>et al.</em> 2010</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cleveland and Hickey 2013</a>). In addition, several studies also support that an increase in imputation accuracy will generate genomic evaluations with nearly identical or even higher accuracy compared with that obtained from observed genotypes (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dassonneville <em>et al.</em> 2011</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Wiggans <em>et al.</em> 2012</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Cleveland and Hickey 2013</a>) because the cost efficiency of low-density genotypes allows a much larger proportion of the population to be included in the genomic evaluation procedure (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Wiggans <em>et al.</em> 2012</a>). In conclusion, an implementation of genomic selection based on observed genotypes for training of the prediction equation and GEBV predictions obtained from genotypes imputed with high accuracy appears to be a promising approach to provide the swine breeding industry with a cost-efficient procedure to obtain GEBV for animals at a young age. A recent study assessing the accuracy of genomic evaluation using high-density genotypes and various imputation schemes in a commercial pig population further supports these findings (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Cleveland and Hickey 2013</a>).</p><p id="p-48">We found that accuracy of individual GEBV was a linear function of the relatedness between a validation animal and the respective training set. As has been previously shown in the literature, animals that are highly related to the training population will have higher <em>r<sub>GEBV</sub></em> (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Habier <em>et al.</em> 2010</a>; <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Clark <em>et al.</em> 2012</a>). As shown in the last scenario, however, when all selection candidates had at least one close relative in the training population, <em>r<sub>GEBV</sub></em> overestimates the accuracy observed for the genomic evaluation (<em>r</em><sub>(</sub><em><sub>EBV</sub></em><sub>,</sub> <em><sub>GEBV</sub></em><sub>)</sub>). Although this measure certainly has value to rank animals according to how trustworthy estimated GEBV are, it is likely overestimated for candidates with close relatives.</p><p id="p-49">The other case in which we observed overestimated individual accuracy of GEBV (<em>r<sub>GEBV</sub></em>) pertains to the last of the imputation scenarios where genotypes were imputed in training and prediction animals. Specifically, when genotypes were imputed in training and prediction animals with lower accuracy, the average <em>r<sub>GEBV</sub></em> was larger than the accuracy of genomic evaluation, which we found was an artifact of lower estimates of the diagonal elements of the <strong>G</strong> matrix. This was caused by a decrease in the variance of the allelic dosage of imputed genotypes due to the relatively small number of reference haplotypes available. When the variance of imputed allelic dosages was decreased, the deviation from the expected value estimated from MAF (2<strong>p</strong>) also decreased, causing overall smaller estimates of <strong>Z</strong> and the resulting diagonal elements of the <strong>G</strong> matrix. This increase in the homogeneity of allelic dosages in the imputed genotypes causes the observed inflation in accuracy of estimated GEBV, such that in any case when GEBV are obtained from imputed genotypes the estimated accuracy of GEBV should be used with caution. The average GEBV accuracy notably exceeded the expected accuracy of genomic evaluation in that scenario.</p><p id="p-50">In conclusion, we found that results for the accuracy of GEBV further support the notion that genomic evaluation using high-density genotypes imputed with high accuracy for selection candidates is a feasible method to implement a cost-efficient design for genomic selection in swine. When genotypes were imputed with lower accuracy in training and prediction animals, the accuracy of genomic evaluation was significantly decreased, and estimates of accuracy of GEBV were inflated. From our results, we can affirm that starting a genomic evaluation using low-density genotypes and a small panel of high-density haplotypes will result in reduced accuracy of evaluation. Contrarily, once an evaluation is established with a large number of animals genotyped using a high-density platform, the addition of more animals genotyped at low density is promising. Further research is needed to study the effect of adding those imputed animals to the training population in further model retraining. As mentioned previously, all code and data used in this paper has been made available through an R package, accessible at: <a href="http://tinyurl.com/MSURGEBV">http://tinyurl.com/MSURGEBV</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">This project was supported by Agriculture and Food Research Initiative Competitive Grant no. 2010-65205-20342 from the US Department of Agriculture National Institute of Food and Agriculture, US Department of Agriculture grant no. 2011-67015-30338, and by funding from the National Swine Registry. Estimated breeding values and accuracies for phenotypic traits, as well as tissue samples for DNA, were also provided by the National Swine Registry. Computer resources and programming advice were provided by the Michigan State University High Performance Computing Center.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010504/-/DC1</a></p></li><li class="fn" id="fn-20"><p id="p-52"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-07-29"><span class="received-label">Received </span>July 29, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-02-06"><span class="accepted-label">Accepted </span>February 6, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Badke <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.1" data-doi="10.1186/1471-2164-13-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badke</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwab</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steibel</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of linkage disequilibrium in four US pig breeds.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBadke%26rft.auinit1%253DY.%2BM.%26rft.volume%253D13%26rft.spage%253D24%26rft.epage%253D24%26rft.atitle%253DEstimation%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bfour%2BUS%2Bpig%2Bbreeds.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-24%26rft_id%253Dinfo%253Apmid%252F22252454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252454&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badke</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwab</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fix</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Methods of tagSNP selection and other variables affecting imputation accuracy in swine.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DBadke%26rft.auinit1%253DY.%2BM.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D8%26rft.epage%253D8%26rft.atitle%253DMethods%2Bof%2BtagSNP%2Bselection%2Band%2Bother%2Bvariables%2Baffecting%2Bimputation%2Baccuracy%2Bin%2Bswine.%26rft_id%253Dinfo%253Apmid%252F23433396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23433396&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.3" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.4" data-doi="10.1186/1297-9686-44-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DClark%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DThe%2Bimportance%2Bof%2Binformation%2Bon%2Brelatives%2Bfor%2Bthe%2Bprediction%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Band%2Bthe%2Bimplications%2Bfor%2Bthe%2Bmakeup%2Bof%2Breference%2Bdata%2Bsets%2Bin%2Blivestock%2Bbreeding%2Bschemes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-4%26rft_id%253Dinfo%253Apmid%252F22321529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22321529&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.5" data-doi="10.2527/jas.2013-6270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Practical implementation of cost-effective genomic selection in commercial pig breeding using imputation.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">3583</span>–<span class="cit-lpage">3592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2013-6270%26rft_id%253Dinfo%253Apmid%252F23736050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2013-6270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23736050&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-4.4.623.6"><div class="cit-metadata unstructured">Cleveland, M. A., S. Forni, D. J. Garrick, and N. Deeb, 2010, in Prediction of genomic breeding values in a commercial pig population, pp. 0266 in <em>9th World Congress on Genetics Applied to Livestock Production</em>, August 1−6, 2010, Leipzig, Germany.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-4.4.623.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A common dataset for genomic analysis of livestock populations.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.8" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.9" data-doi="10.3168/jds.2011-4299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dassonneville</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brondum</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effect of imputing markers from a low-density chip on the reliability of genomic breeding values in Holstein populations.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">3679</span>–<span class="cit-lpage">3686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DDassonneville%26rft.auinit1%253DR.%26rft.volume%253D94%26rft.issue%253D7%26rft.spage%253D3679%26rft.epage%253D3686%26rft.atitle%253DEffect%2Bof%2Bimputing%2Bmarkers%2Bfrom%2Ba%2Blow-density%2Bchip%2Bon%2Bthe%2Breliability%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2BHolstein%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4299%26rft_id%253Dinfo%253Apmid%252F21700057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21700057&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-4.4.623.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathur</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knol</span>  <span class="cit-name-given-names">E. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic Improvement of the Pig</span>, pp. <span class="cit-fpage">390</span>–<span class="cit-lpage">425</span> in <span class="cit-source">The Genetics of the Pig</span>, <span class="cit-edition">Ed. 2,</span> edited by <span class="cit-ed"><span class="cit-name-surname">Rothschild</span>  <span class="cit-name-given-names">M. F.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Ruvinsky</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">CABI</span>, <span class="cit-publ-loc">Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.11" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.12" data-doi="10.1111/j.1439-0388.2011.00964.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using the genomic relationship matrix to predict the accuracy of genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D409%26rft.epage%253D421%26rft.atitle%253DUsing%2Bthe%2Bgenomic%2Brelationship%2Bmatrix%2Bto%2Bpredict%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00964.x%26rft_id%253Dinfo%253Apmid%252F22059574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00964.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059574&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gualdrón Duarte</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantet</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotype imputation accuracy in a F2 pig population using high density and low density SNP panels.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D38%26rft_id%253Dinfo%253Apmid%252F23651538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23651538&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.14" data-doi="10.1534/genetics.108.100289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection using low-density marker panels.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100289%26rft_id%253Dinfo%253Apmid%252F19299339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.15" data-doi="10.1186/1297-9686-42-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichtner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic relationship information on genomic breeding values in German Holstein cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D42%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DThe%2Bimpact%2Bof%2Bgenetic%2Brelationship%2Binformation%2Bon%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2BGerman%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-5%26rft_id%253Dinfo%253Apmid%252F20170500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20170500&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.16" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Invited review: genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.17" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genotype imputation in sheep breeds.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D72%26rft.epage%253D80%26rft.atitle%253DAccuracy%2Bof%2Bgenotype%2Bimputation%2Bin%2Bsheep%2Bbreeds.%26rft_id%253Dinfo%253Apmid%252F22221027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22221027&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.19" data-doi="10.1186/1297-9686-44-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maltecca</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Assessment of alternative genotyping strategies to maximize imputation accuracy at minimal cost.</span> <abbr class="cit-jnl-abbrev">Genet Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHuang%26rft.auinit1%253DY.%26rft.volume%253D44%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DAssessment%2Bof%2Balternative%2Bgenotyping%2Bstrategies%2Bto%2Bmaximize%2Bimputation%2Baccuracy%2Bat%2Bminimal%2Bcost.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-25%26rft_id%253Dinfo%253Apmid%252F22849718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22849718&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.20" data-doi="10.1534/genetics.109.107391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svendsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The accuracy of genomic selection in Norwegian red cattle assessed by cross-validation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.107391%26rft_id%253Dinfo%253Apmid%252F19704013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My8zLzExMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-4.4.623.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Linear Models for the Prediction of Animal Breeding Values</em>,</span> <span class="cit-edition">Ed 2</span>. <span class="cit-publ-name">CABI Publishing</span>, <span class="cit-publ-loc">Oxfordshire, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.23" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian linear regression package in R.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-4.4.623.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Cabal</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genome-enabled prediction in a dairy cattle population using different cross-validation layouts</span>. <span class="cit-source">Front. Genet.</span> <span class="cit-vol">3</span>: <span class="cit-fpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DPerez-Cabal%26rft.auinit1%253DM.%2BA.%26rft.volume%253D3%26rft.spage%253D27%26rft.epage%253D27%26rft.atitle%253DAccuracy%2Bof%2BGenome-Enabled%2BPrediction%2Bin%2Ba%2BDairy%2BCattle%2BPopulation%2Busing%2BDifferent%2BCross-Validation%2BLayouts.%26rft_id%253Dinfo%253Apmid%252F22403583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22403583&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.25" data-doi="10.1371/journal.pone.0006524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooijmans</span>  <span class="cit-name-given-names">R. P. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affara</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amaral</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRamos%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6524%26rft.epage%253De6524%26rft.atitle%253DDesign%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bin%2Bthe%2Bpig%2Busing%2BSNPs%2Bidentified%2Band%2Bcharacterized%2Bby%2Bnext%2Bgeneration%2Bsequencing%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006524%26rft_id%253Dinfo%253Apmid%252F19654876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19654876&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-4.4.623.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">The R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.27" data-doi="10.3168/jds.2008-1929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Technical note: derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2971%26rft.epage%253D2975%26rft.atitle%253DTechnical%2Bnote%253A%2BDerivation%2Bof%2Bequivalent%2Bcomputing%2Balgorithms%2Bfor%2Bgenomic%2Bpredictions%2Band%2Breliabilities%2Bof%2Banimal%2Bmerit%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1929%26rft_id%253Dinfo%253Apmid%252F19448030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448030&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.28" data-doi="10.2527/jas.2012-5107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tribout</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larzul</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phocas</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficiency of genomic selection in a purebred pig male line.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">4164</span>–<span class="cit-lpage">4176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2012-5107%26rft_id%253Dinfo%253Apmid%252F22859761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2012-5107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22859761&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.29" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-other" id="cit-4.4.623.30" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tassell</span>  <span class="cit-name-given-names">C. P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: reliability of genomic predictions for North American Holstein bulls.</span> <span class="cit-source">J. Dairy Sci.</span> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.31" data-doi="10.1186/1297-9686-43-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic evaluations with many more genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DVanraden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D43%26rft.spage%253D10%26rft.epage%253D10%26rft.atitle%253DGenomic%2Bevaluations%2Bwith%2Bmany%2Bmore%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-10%26rft_id%253Dinfo%253Apmid%252F21366914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21366914&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.32" data-doi="10.2527/jas.2009-1952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Technical note: an R package for fitting generalized linear mixed models in animal breeding.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-1952%26rft_id%253Dinfo%253Apmid%252F19820058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-1952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19820058&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274040700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.33" data-doi="10.1017/S0016672312000274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping including phenotypes from relatives without genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D94%26rft.issue%253D2%26rft.spage%253D73%26rft.epage%253D83%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bincluding%2Bphenotypes%2Bfrom%2Brelatives%2Bwithout%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672312000274%26rft_id%253Dinfo%253Apmid%252F22624567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672312000274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22624567&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.34" data-doi="10.3168/jds.2010-3149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accuracy of direct genomic values derived from imputed single nucleotide polymorphism genotypes in Jersey cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5423</span>–<span class="cit-lpage">5435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWeigel%26rft.auinit1%253DK.%2BA.%26rft.volume%253D93%26rft.issue%253D11%26rft.spage%253D5423%26rft.epage%253D5435%26rft.atitle%253DAccuracy%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bderived%2Bfrom%2Bimputed%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bgenotypes%2Bin%2BJersey%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3149%26rft_id%253Dinfo%253Apmid%252F20965358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20965358&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.35" data-doi="10.1534/genetics.112.146290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146290%26rft_id%253Dinfo%253Apmid%252F23267052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNjIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.36" data-doi="10.3168/jds.2010-3866"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanraden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genomic evaluation system in the United States: past, present, future.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">3202</span>–<span class="cit-lpage">3211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWiggans%26rft.auinit1%253DG.%2BR.%26rft.volume%253D94%26rft.issue%253D6%26rft.spage%253D3202%26rft.epage%253D3211%26rft.atitle%253DThe%2Bgenomic%2Bevaluation%2Bsystem%2Bin%2Bthe%2BUnited%2BStates%253A%2Bpast%252C%2Bpresent%252C%2Bfuture.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3866%26rft_id%253Dinfo%253Apmid%252F21605789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3866&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605789&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.623.37" data-doi="10.3168/jds.2011-4985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tooker</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Use of the Illumina Bovine3K BeadChip in dairy genomic evaluation.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">1552</span>–<span class="cit-lpage">1558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWiggans%26rft.auinit1%253DG.%2BR.%26rft.volume%253D95%26rft.issue%253D3%26rft.spage%253D1552%26rft.epage%253D1558%26rft.atitle%253DUse%2Bof%2Bthe%2BIllumina%2BBovine3K%2BBeadChip%2Bin%2Bdairy%2Bgenomic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4985%26rft_id%253Dinfo%253Apmid%252F22365235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22365235&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/4/623.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/613" title="Edc3 Function in Yeast and Mammals Is Modulated by Interaction with NAD-Related Compounds" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/633" title="Centromeric Barrier Disruption Leads to Mitotic Defects in Schizosaccharomyces pombe" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/4"><span>Volume 4 Issue 4, April 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/4.cover-source.jpg?itok=Rf0YhETZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/623" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nfDrKnq9anJ4DLL1nQNgNj1xcxWO8IKBvHv3h8bOb8A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/4/623" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/4/623" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/4/623" class="active" data-icon-position="" data-hide-link-title="0">Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459840" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-dDIa4cjhRMp20BsY1DMpCeVEnjbdxFI5I_VKzDC-XeM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/623.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/4/623" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-cgTktMINWLqENzrffEIpyrzvUKWxxDB-Bsvr3tofqGE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/4/623" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/623.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459840" id="node459840--2" data-pisa="ggg;4/4/623" data-pisa-master="ggg;g3.114.010504" data-apath="/ggg/4/4/623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yvonne M. Badke</span>, <span class="highwire-citation-author" data-delta="1">Ronald O. Bates</span>, <span class="highwire-citation-author" data-delta="2">Catherine W. Ernst</span>, <span class="highwire-citation-author" data-delta="3">Justin Fix</span> and <span class="highwire-citation-author" data-delta="4">Juan P. Steibel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">623-631;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010504 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459840--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yvonne M. Badke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BM.%2BBadke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Badke%20YM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYvonne%2BM.%2BBadke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ronald O. Bates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BO.%2BBates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bates%20RO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BO.%2BBates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Catherine W. Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BW.%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BW.%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Justin Fix </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Smithfield Premium Genetics Group, Rose Hill, North Carolina 28458</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BFix%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fix%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BFix%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan P. Steibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Fisheries & Wildlife, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BP.%2BSteibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steibel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BP.%2BSteibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:steibelj@msu.edu" class="" data-icon-position="" data-hide-link-title="0">steibelj@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/4/623" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/4/623&amp;title=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/4/623&amp;title=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/4/623&amp;title=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/4/623&amp;text=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/4/623&amp;title=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/4/623&amp;t=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/4/623&amp;title=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/4/623&amp;title=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459840/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459840" id="node459840--3" data-pisa="ggg;4/4/623" data-pisa-master="ggg;g3.114.010504" data-apath="/ggg/4/4/623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Estimation of Genomic Breeding Values in Pigs Using Low-Density Genotypes and Imputation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yvonne M. Badke</span>, <span class="highwire-citation-author" data-delta="1">Ronald O. Bates</span>, <span class="highwire-citation-author" data-delta="2">Catherine W. Ernst</span>, <span class="highwire-citation-author" data-delta="3">Justin Fix</span> and <span class="highwire-citation-author" data-delta="4">Juan P. Steibel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">623-631;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010504 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459840--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yvonne M. Badke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yvonne%2BM.%2BBadke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Badke%20YM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYvonne%2BM.%2BBadke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ronald O. Bates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BO.%2BBates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bates%20RO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BO.%2BBates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Catherine W. Ernst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BW.%2BErnst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ernst%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BW.%2BErnst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Justin Fix </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Smithfield Premium Genetics Group, Rose Hill, North Carolina 28458</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BFix%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fix%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustin%2BFix%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan P. Steibel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Fisheries & Wildlife, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BP.%2BSteibel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steibel%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BP.%2BSteibel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:steibelj@msu.edu" class="" data-icon-position="" data-hide-link-title="0">steibelj@msu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=623&amp;rft.epage=631&amp;rft.atitle=Accuracy%20of%20Estimation%20of%20Genomic%20Breeding%20Values%20in%20Pigs%20Using%20Low-Density%20Genotypes%20and%20Imputation&amp;rft.volume=4&amp;rft.issue=4&amp;rft.date=2014-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Badke%2C+Yvonne+M.&amp;rft.au=Bates%2C+Ronald+O.&amp;rft.au=Ernst%2C+Catherine+W.&amp;rft.au=Fix%2C+Justin&amp;rft.au=Steibel%2C+Juan+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459840/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459840/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459840/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459840/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459840/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459840/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459840/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459840/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459840/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459840/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459840/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459840/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/4/623" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/4/623.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/4/623.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/4/623.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/4/623.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
