<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/4/669" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/4/669.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/4/669.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/4/669.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/4/669.atom" />
<meta name="HW.pisa" content="ggg;4/4/669" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations" />
<meta name="DC.Identifier" content="10.1534/g3.114.010264" />
<meta name="DC.Date" content="2014-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Hahn et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Current de novo whole-genome sequencing approaches often are inadequate for organisms lacking substantial preexisting genetic data. Problems with these methods are manifest as: large numbers of scaffolds that are not ordered within chromosomes or assigned to individual chromosomes, misassembly of allelic sequences as separate loci when the individual(s) being sequenced are heterozygous, and the collapse of recently duplicated sequences into a single locus, regardless of levels of heterozygosity. Here we propose a new approach for producing de novo whole-genome sequences—which we call recombinant population genome construction—that solves many of the problems encountered in standard genome assembly and that can be applied in model and nonmodel organisms. Our approach takes advantage of next-generation sequencing technologies to simultaneously barcode and sequence a large number of individuals from a recombinant population. The sequences of all recombinants can be combined to create an initial de novo assembly, followed by the use of individual recombinant genotypes to correct assembly splitting/collapsing and to order and orient scaffolds within linkage groups. Recombinant population genome construction can rapidly accelerate the transformation of nonmodel species into genome-enabled systems by simultaneously producing a high-quality genome assembly and providing genomic tools ( e.g. , high-confidence single-nucleotide polymorphisms) for immediate applications. In populations segregating for important functional traits, this approach also enables simultaneous mapping of quantitative trait loci. We demonstrate our method using simulated Illumina data from a recombinant population of Caenorhabditis elegans and show that the method can produce a high-fidelity, high-quality genome assembly for both parents of the cross." />
<meta name="DC.Contributor" content="Matthew W. Hahn" />
<meta name="DC.Contributor" content="Simo V. Zhang" />
<meta name="DC.Contributor" content="Leonie C. Moyle" />
<meta name="article:published_time" content="2014-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Current &lt;i&gt;de novo&lt;/i&gt; whole-genome sequencing approaches often are inadequate for organisms lacking substantial preexisting genetic data. Problems with these methods are manifest as: large numbers of scaffolds that are not ordered within chromosomes or assigned to individual chromosomes, misassembly of allelic sequences as separate loci when the individual(s) being sequenced are heterozygous, and the collapse of recently duplicated sequences into a single locus, regardless of levels of heterozygosity. Here we propose a new approach for producing &lt;i&gt;de novo&lt;/i&gt; whole-genome sequences—which we call recombinant population genome construction—that solves many of the problems encountered in standard genome assembly and that can be applied in model and nonmodel organisms. Our approach takes advantage of next-generation sequencing technologies to simultaneously barcode and sequence a large number of individuals from a recombinant population. The sequences of all recombinants can be combined to create an initial &lt;i&gt;de novo&lt;/i&gt; assembly, followed by the use of individual recombinant genotypes to correct assembly splitting/collapsing and to order and orient scaffolds within linkage groups. Recombinant population genome construction can rapidly accelerate the transformation of nonmodel species into genome-enabled systems by simultaneously producing a high-quality genome assembly and providing genomic tools (&lt;i&gt;e.g.&lt;/i&gt;, high-confidence single-nucleotide polymorphisms) for immediate applications. In populations segregating for important functional traits, this approach also enables simultaneous mapping of quantitative trait loci. We demonstrate our method using simulated Illumina data from a recombinant population of &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; and show that the method can produce a high-fidelity, high-quality genome assembly for both parents of the cross.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/04/01" />
<meta name="citation_mjid" content="ggg;4/4/669" />
<meta name="citation_id" content="4/4/669" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/4/669" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/4/669.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/4/669.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/4/669.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010264" />
<meta name="citation_pmid" content="24531727" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="669" />
<meta name="citation_lastpage" content="679" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Matthew W. Hahn" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_email" content="mwh@indiana.edu" />
<meta name="citation_author" content="Simo V. Zhang" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Leonie C. Moyle" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Adams;citation_author=S. E. Celniker;citation_author=R. A. Holt;citation_author=C. A. Evans;citation_author=J. D. Gocayne;citation_title=The Genome Sequence of Drosophila melanogaster;citation_pages=2185-2195;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731132;citation_doi=10.1126/science.287.5461.2185" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Alkan;citation_author=S. Sajjadian;citation_author=E. E. Eichler;citation_title=Limitations of next-generation genome sequence assembly.;citation_pages=61-65;citation_volume=8;citation_year=2011;citation_issue=1;citation_pmid=21102452;citation_doi=10.1038/nmeth.1527" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=A. Ferrara;citation_author=Q. Fontenot;citation_author=J. H. Postlethwait;citation_title=Genome Evolution and Meiotic Maps by Massively Parallel DNA Sequencing: Spotted Gar, an Outgroup for the Teleost Genome Duplication;citation_pages=799-808;citation_volume=188;citation_year=2011;citation_issue=4;citation_pmid=21828280;citation_doi=10.1534/genetics.111.127324" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. A. Bailey;citation_author=A. M. Yavor;citation_author=H. F. Massa;citation_author=B. J. Trask;citation_author=E. E. Eichler;citation_title=Segmental duplications: organization and impact within the current Human Genome Project assembly.;citation_volume=11;citation_year=2001;citation_pmid=11381028;citation_doi=10.1101/gr.GR-1871R" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. A. Bailey;citation_author=Z. P. Gu;citation_author=R. A. Clark;citation_author=K. Reinert;citation_author=R. V. Samonte;citation_title=Recent Segmental Duplications in the Human Genome;citation_pages=1003-1007;citation_volume=297;citation_year=2002;citation_issue=5583;citation_pmid=12169732;citation_doi=10.1126/science.1072047" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=V. Bansal;citation_author=A. L. Halpern;citation_author=N. Axelrod;citation_author=V. Bafna;citation_title=An MCMC algorithm for haplotype assembly from whole-genome sequence data;citation_pages=1336-1346;citation_volume=18;citation_year=2008;citation_issue=8;citation_pmid=18676820;citation_doi=10.1101/gr.077065.108" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Barrière;citation_author=S.-P. Yang;citation_author=E. Pekarek;citation_author=C. G. Thomas;citation_author=E. S. Haag;citation_title=Detecting heterozygosity in shotgun genome assemblies: lessons from obligately outcrossing nematodes.;citation_volume=19;citation_year=2009;citation_pmid=19204328;citation_doi=10.1101/gr.081851.108" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Casola;citation_author=C. L. Ganote;citation_author=M. W. Hahn;citation_title=Nonallelic gene conversion in the genus Drosophila.;citation_volume=185;citation_year=2010;citation_pmid=20215470;citation_doi=10.1534/genetics.110.115444" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=D. Chagné;citation_author=G. Brown;citation_author=C. Lalanne;citation_author=D. Madur;citation_author=D. Pot;citation_title=Comparative genome and QTL mapping between maritime and loblolly pines.;citation_pages=185-195;citation_volume=12;citation_year=2003;citation_doi=10.1023/A:1026318327911" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Cheung;citation_author=X. Estivill;citation_author=R. Khaja;citation_author=J. MacDonald;citation_author=K. Lau;citation_title=Genome-wide detection of segmental duplications and potential assembly errors in the human genome sequence.;citation_pages=R25-R25;citation_volume=4;citation_year=2003;citation_issue=4;citation_pmid=12702206;citation_doi=10.1186/gb-2003-4-4-r25" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. K. Colbourne;citation_author=M. E. Pfrender;citation_author=D. Gilbert;citation_author=W. K. Thomas;citation_author=A. Tucker;citation_title=The Ecoresponsive Genome of Daphnia pulex;citation_pages=555-561;citation_volume=331;citation_year=2011;citation_issue=6017;citation_pmid=21292972;citation_doi=10.1126/science.1197761" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=Genome Sequence of the Nematode C. elegans: A Platform for Investigating Biology;citation_pages=2012-2018;citation_volume=282;citation_year=1998;citation_issue=5396;citation_pmid=9851916;citation_doi=10.1126/science.282.5396.2012" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Dehal;citation_author=Y. Satou;citation_author=R. K. Campbell;citation_author=J. Chapman;citation_author=B. Degnan;citation_title=The Draft Genome of Ciona intestinalis: Insights into Chordate and Vertebrate Origins;citation_pages=2157-2167;citation_volume=298;citation_year=2002;citation_issue=5601;citation_pmid=12481130;citation_doi=10.1126/science.1080049" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=F. Denoeud;citation_author=S. Henriet;citation_author=S. Mungpakdee;citation_author=J. M. Aury;citation_author=C. Da Silva;citation_title=Plasticity of animal genome architecture unmasked by rapid evolution of a pelagic tunicate.;citation_volume=330;citation_year=2010;citation_pmid=21097902;citation_doi=10.1126/science.1194167" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=G. Genovese;citation_author=R. E. Handsaker;citation_author=H. Li;citation_author=N. Altemose;citation_author=A. M. Lindgren;citation_title=Using population admixture to help complete maps of the human genome.;citation_pages=406-414;citation_volume=45;citation_year=2013;citation_pmid=23435088;citation_doi=10.1038/ng.2565" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gnerre;citation_author=I. MacCallum;citation_author=D. Przybylski;citation_author=F. J. Ribeiro;citation_author=J. N. Burton;citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data.;citation_volume=108;citation_year=2011;citation_pmid=21187386;citation_doi=10.1073/pnas.1017351108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Grattapaglia;citation_author=R. Sederoff;citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers.;citation_pages=1121-1137;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982566" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. C. Hall;citation_author=J. H. Willis;citation_title=Transmission ratio distortion in intraspecific hybrids of Mimulus guttatus: implications for genomic divergence.;citation_volume=170;citation_year=2005;citation_pmid=15781698;citation_doi=10.1534/genetics.104.038653" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=M. V. Han;citation_author=J. P. Demuth;citation_author=C. L. McGrath;citation_author=C. Casola;citation_author=M. W. Hahn;citation_title=Adaptive evolution of young gene duplicates in mammals;citation_pages=859-867;citation_volume=19;citation_year=2009;citation_issue=5;citation_pmid=19411603;citation_doi=10.1101/gr.085951.108" />
<meta name="citation_reference" content="citation_journal_title=Journal of Microbiological Methods;citation_journal_abbrev=Journal of Microbiological Methods;citation_author=S. Haridas;citation_author=C. Breuill;citation_author=J. Bohlmann;citation_author=T. Hsiang;citation_title=A biologist's guide to de novo genome assembly using next-generation sequence data: A test with fungal genomes.;citation_pages=368-375;citation_volume=86;citation_year=2011;citation_issue=3;citation_pmid=21749903;citation_doi=10.1016/j.mimet.2011.06.019" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. A. Holt;citation_author=G. M. Subramanian;citation_author=A. Halpern;citation_author=G. G. Sutton;citation_author=R. Charlab;citation_title=The Genome Sequence of the Malaria Mosquito Anopheles gambiae;citation_pages=129-149;citation_volume=298;citation_year=2002;citation_issue=5591;citation_pmid=12364791;citation_doi=10.1126/science.1076181" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Hu;citation_author=J. Yuan;citation_author=Y. Shi;citation_author=J. Lu;citation_author=B. Liu;citation_title=pIRS: Profile-based Illumina pair-end reads simulator.;citation_volume=28;citation_year=2012;citation_pmid=22508794;citation_doi=10.1093/bioinformatics/bts187" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High-throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. L. Hyten;citation_author=S. B. Cannon;citation_author=Q. Song;citation_author=N. Weeks;citation_author=E. W. Fickus;citation_title=High-throughput SNP discovery through deep resequencing of a reduced representation library to anchor and orient scaffolds in the soybean whole genome sequence.;citation_pages=38-38;citation_volume=11;citation_year=2010;citation_pmid=20078886;citation_doi=10.1186/1471-2164-11-38" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. R. Kelley;citation_author=S. L. Salzberg;citation_title=Detection and correction of false segmental duplications caused by genome mis-assembly.;citation_pages=R28-R28;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20219098;citation_doi=10.1186/gb-2010-11-3-r28" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. H. Kim;citation_author=M. S. Waterman;citation_author=L. M. Li;citation_title=Diploid genome reconstruction of Ciona intestinalis and comparative analysis with Ciona savignyi.;citation_volume=17;citation_year=2007;citation_pmid=17567986;citation_doi=10.1101/gr.5894107" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. O. Kitzman;citation_author=A. P. MacKenzie;citation_author=A. Adey;citation_author=J. B. Hiatt;citation_author=R. P. Patwardhan;citation_title=Haplotype-resolved genome sequencing of a Gujarati Indian individual.;citation_pages=59-63;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21170042;citation_doi=10.1038/nbt.1740" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. S. Lander;citation_author=L. M. Linton;citation_author=B. Birren;citation_author=C. Nusbaum;citation_author=M. C. Zody;citation_title=Initial sequencing and analysis of the human genome.;citation_pages=860-921;citation_volume=409;citation_year=2001;citation_issue=6822;citation_pmid=11237011;citation_doi=10.1038/35057062" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Langley;citation_author=K. Stevens;citation_author=C. Cardeno;citation_author=Y. C. G. Lee;citation_author=D. R. Schrider;citation_title=Genomic variation in natural populations of Drosophila melanogaster.;citation_volume=192;citation_year=2012;citation_pmid=22673804;citation_doi=10.1534/genetics.112.142018" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. A. Lewin;citation_author=D. M. Larkin;citation_author=J. Pontius;citation_author=S. J. O’Brien;citation_title=Every genome sequence needs a good map.;citation_volume=19;citation_year=2009;citation_pmid=19596977;citation_doi=10.1101/gr.094557.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Lin;citation_author=J. Li;citation_author=H. Shen;citation_author=L. Zhang;citation_author=C. J. Papasian;citation_title=Comparative studies of de novo assembly tools for next-generation sequencing technologies.;citation_volume=27;citation_year=2011;citation_pmid=21636596;citation_doi=10.1093/bioinformatics/btr319" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. C. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=K. Matsubara;citation_author=K. Ebana;citation_author=T. Mizubayashi;citation_author=S. Itoh;citation_author=T. Ando;citation_title=Relationship between transmission ratio distortion and genetic divergence in intraspecific rice crosses.;citation_pages=307-319;citation_volume=286;citation_year=2011;citation_issue=5-6;citation_pmid=21918817;citation_doi=10.1007/s00438-011-0648-6" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Mortazavi;citation_author=E. M. Schwarz;citation_author=B. Williams;citation_author=L. Schaeffer;citation_author=I. Antoshechkin;citation_title=Scaffolding a Caenorhabditis nematode genome with RNA-seq.;citation_volume=20;citation_year=2010;citation_pmid=20980554;citation_doi=10.1101/gr.111021.110" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. C. Moyle;citation_author=E. B. Graham;citation_title=Genome-wide associations between hybrid sterility QTL and marker transmission ratio distortion.;citation_volume=23;citation_year=2006;citation_pmid=16495346;citation_doi=10.1093/molbev/msj112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. A. Myburg;citation_author=C. Vogl;citation_author=A. R. Griffin;citation_author=R. R. Sederoff;citation_author=R. W. Whetten;citation_title=Genetics of Postzygotic Isolation in Eucalyptus: Whole-Genome Analysis of Barriers to Introgression in a Wide Interspecific Cross of Eucalyptus grandis and E. globulus;citation_pages=1405-1418;citation_volume=166;citation_year=2004;citation_issue=3;citation_pmid=15082559;citation_doi=10.1534/genetics.166.3.1405" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. A. Payseur;citation_author=H. E. Hoekstra;citation_title=Signatures of reproductive isolation in patterns of single nucleotide diversity across inbred strains of mice.;citation_volume=171;citation_year=2005;citation_pmid=16143616;citation_doi=10.1534/genetics.105.046193" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. A. Ross;citation_author=D. C. Koboldt;citation_author=J. E. Staisch;citation_author=H. M. Chamberlin;citation_author=B. P. Gupta;citation_title=Caenorhabditis briggsae recombinant inbred line genotypes reveal inter-strain incompatibility and the evolution of recombination.;citation_pages=e1002174-e1002174;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21779179;citation_doi=10.1371/journal.pgen.1002174" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=T. B. Sackton;citation_author=R. J. Kulathinal;citation_author=C. M. Bergman;citation_author=A. R. Quinlan;citation_author=E. B. Dopman;citation_title=Population genomic inferences from sparse high-throughput sequencing of two populations of Drosophila melanogaster.;citation_volume=1;citation_year=2009;citation_pmid=20333214;citation_doi=10.1093/gbe/evp048" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. C. Schatz;citation_author=A. L. Delcher;citation_author=S. L. Salzberg;citation_title=Assembly of large genomes using second-generation sequencing.;citation_volume=20;citation_year=2010;citation_pmid=20508146;citation_doi=10.1101/gr.101360.109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Solignac;citation_author=D. Vautrin;citation_author=E. Baudry;citation_author=F. Mougel;citation_author=A. Loiseau;citation_title=A Microsatellite-Based Linkage Map of the Honeybee, Apis mellifera L.;citation_pages=253-262;citation_volume=167;citation_year=2004;citation_issue=1;citation_pmid=15166152;citation_doi=10.1534/genetics.167.1.253" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. Tautz;citation_author=H. Ellegren;citation_author=D. Weigel;citation_title=Next generation molecular ecology.;citation_pages=1-3;citation_volume=19;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=L. K. Tulsieram;citation_author=J. C. Glaubitz;citation_author=G. Kiss;citation_author=J. E. Carlson;citation_title=Single tree genetic linkage mapping in conifers using haploid DNA from megagametophytes.;citation_pages=686-690;citation_volume=10;citation_year=1992;citation_doi=10.1038/nbt0692-686" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. R. Wicks;citation_author=R. T. Yeh;citation_author=W. R. Gish;citation_author=R. H. Waterston;citation_author=R. H. A. Plasterk;citation_title=Rapid gene mapping in Caenorhabditis elegans using a high density polymorphism map.;citation_pages=160-164;citation_volume=28;citation_year=2001;citation_issue=2;citation_pmid=11381264;citation_doi=10.1038/88878" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Xie;citation_author=Q. Feng;citation_author=H. Yu;citation_author=X. Huang;citation_author=Q. Zhao;citation_title=Parent-independent genotyping for constructing an ultrahigh-density linkage map based on population sequencing.;citation_volume=107;citation_year=2010;citation_pmid=20498060;citation_doi=10.1073/pnas.1005931107" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=L. Ye;citation_author=L. Hillier;citation_author=P. Minx;citation_author=N. Thane;citation_author=D. P. Locke;citation_title=A vertebrate case study of the quality of assemblies derived from next-generation sequences.;citation_pages=R31-R31;citation_volume=12;citation_year=2011;citation_issue=3;citation_pmid=21453517;citation_doi=10.1186/gb-2011-12-3-r31" />
<meta name="citation_reference" content="citation_journal_title=Bot. Gaz.;citation_author=D. Zamir;citation_author=Y. Tadmor;citation_title=Unequal segregation of nuclear genes in plants.;citation_pages=355-358;citation_volume=147;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Zhou;citation_author=F. Wei;citation_author=J. Nguyen;citation_author=M. Bechner;citation_author=K. Potamousis;citation_title=A single molecule scaffold for the maize genome.;citation_pages=e1000711-e1000711;citation_volume=5;citation_year=2009;citation_issue=11;citation_pmid=19936062;citation_doi=10.1371/journal.pgen.1000711" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="twitter:description" content="Current de novo whole-genome sequencing approaches often are inadequate for organisms lacking substantial preexisting genetic data. Problems with these methods are manifest as: large numbers of scaffolds that are not ordered within chromosomes or assigned to individual chromosomes, misassembly of allelic sequences as separate loci when the individual(s) being sequenced are heterozygous, and the collapse of recently duplicated sequences into a single locus, regardless of levels of heterozygosity. Here we propose a new approach for producing de novo whole-genome sequences—which we call recombinant population genome construction—that solves many of the problems encountered in standard genome assembly and that can be applied in model and nonmodel organisms. Our approach takes advantage of next-generation sequencing technologies to simultaneously barcode and sequence a large number of individuals from a recombinant population. The sequences of all recombinants can be combined to create an initial de novo assembly, followed by the use of individual recombinant genotypes to correct assembly splitting/collapsing and to order and orient scaffolds within linkage groups. Recombinant population genome construction can rapidly accelerate the transformation of nonmodel species into genome-enabled systems by simultaneously producing a high-quality genome assembly and providing genomic tools ( e.g. , high-confidence single-nucleotide polymorphisms) for immediate applications. In populations segregating for important functional traits, this approach also enables simultaneous mapping of quantitative trait loci. We demonstrate our method using simulated Illumina data from a recombinant population of Caenorhabditis elegans and show that the method can produce a high-fidelity, high-quality genome assembly for both parents of the cross." />
<meta name="og-title" property="og:title" content="Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/4/669" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Current de novo whole-genome sequencing approaches often are inadequate for organisms lacking substantial preexisting genetic data. Problems with these methods are manifest as: large numbers of scaffolds that are not ordered within chromosomes or assigned to individual chromosomes, misassembly of allelic sequences as separate loci when the individual(s) being sequenced are heterozygous, and the collapse of recently duplicated sequences into a single locus, regardless of levels of heterozygosity. Here we propose a new approach for producing de novo whole-genome sequences—which we call recombinant population genome construction—that solves many of the problems encountered in standard genome assembly and that can be applied in model and nonmodel organisms. Our approach takes advantage of next-generation sequencing technologies to simultaneously barcode and sequence a large number of individuals from a recombinant population. The sequences of all recombinants can be combined to create an initial de novo assembly, followed by the use of individual recombinant genotypes to correct assembly splitting/collapsing and to order and orient scaffolds within linkage groups. Recombinant population genome construction can rapidly accelerate the transformation of nonmodel species into genome-enabled systems by simultaneously producing a high-quality genome assembly and providing genomic tools ( e.g. , high-confidence single-nucleotide polymorphisms) for immediate applications. In populations segregating for important functional traits, this approach also enables simultaneous mapping of quantitative trait loci. We demonstrate our method using simulated Illumina data from a recombinant population of Caenorhabditis elegans and show that the method can produce a high-fidelity, high-quality genome assembly for both parents of the cross." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
    <title>Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Ekze41YzbTkv75Vbd5nsB-XmPTFOQMHyJMwXpI67UBw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459832","apath":"\/ggg\/4\/4\/669.atom","pisa":"ggg;4\/4\/669","ac":{"\/ggg\/4\/4\/669.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/4\/669.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/4\/669"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/4\/669"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/4\/669":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c1ggAXQ3NpheRyYwN7I0TCOthSuBF65n9n5J2G\\\\\\\/7R0k\\\\\\\/rNyWnaGPPv27unVM7PpbxM03myZHHSvAheFpx82RBFJCy9sLpj861oPQRCdo+lm5vcRPai4WYxgkkbWAeD\\\\\\\/A1FrIeohxSbxPb9ikZxxus8wAFZ9Ieb5lulRJ5fbeE8++0OWntlHs+sGxNn5mIKkRZdxwFaCTy\\\\\\\/iJSiME3emxpYRjLvYpYuEZplSLWFxDzuZ\\\\\\\/g6djwekfrKCKLZ7N77jjHru+Y\\\\\\\/tuejuvwZF6LsKP7Ril8iHaBgXa80muAxCVIAXE54N6hRhHEkXowQsLkNEGl5wOrX2O0XQrSatVqYPURdp5i4xZeUKuEhNxLMJ8HtmSHwC+aPOKHoDx71h6+T28bLuDl7mutzxT81MHisBpA4z5V3i0aB5TSAN9ey15r\\\\\\\/pLoc4w3m4qM6echtsyO+nJgehEp3Kc6jBJoUP8akIjmsK+GDwzhG2+xB6Xopf5RwlCmIGyp6i9y4qRw683zJRPvIlqX52Deup20vXML9oxk5D2zMQWRpqDX6lZUGARWZ2Eywm0MkRzaKBzEKDCefRGP6h4Pp\\\\\\\/6weV7INNqzNQ9xvWad\\\\\\\/iEKMMQIm6Oa3CSnUTRDujYo5gw8LOw9oufzgDyDn46YcEgymwL1Fy6Qg0bUIigGolyS\\\\\\\/wpFjO4ao+XIuFU8O0JJete+FSSIvfq3XWM\\\\\\\/I0s5t8j8v8mbpzTNjT4CJkUZzxYT1wiDUUhHYJXltQ9uXqsx\\\\\\\/pI4Gm9TVY\\\\\\\/YbeuFKgAFQ8CXV3gCdEajDUH4JVqB7RwgUKTmjwXm39UiedZLZjIeTKOB5YtDl4SE1Tlw8hBiMdDLV5zX768i9hSRMK5++\\\\\\\/3gYHHark+fPXlVDZsISEVUigEkRghFyMahhRqV2dsuZmAs7hG1MGe0nDpzId\\\\\\\/lzKIf73reYCxjJqCQ87PtzsE1575T7MsZsJXZd2IJ3ucFQe3DAXLnCN8hjerF0a0CjSy3hGyib2QvkT5Zz0jjoiixKh7AHE3KNRmc6RFYlxcxaXgTay0novy\\\\\\\/SdmHiIa2xUFbTYI7NDT6P7jXaJff5P5yB14gzawBO\\\\\\\/hbF\\\\\\\/ISNGCQCKKcNT31SuXLPcz275yNctVuusxCjyX4K+sfQxbT0V1AUK0YufFfUzv6GHPzqwi\\\\\\\/B4k8IccK6zs3x9vUTC18QqXjKOCNg4uxrbTzwpxCQxDUXVixyX\\\\\\\/2SBPKkkZlM7KKvomUwU7S6H+tdbq2dF5RMlUwjtIIwQ9DXB0YdaX84Wv+olaKIO5AOAyWjmK\\\u0022,\\\u0022iv\\\u0022:\\\u0022kU33OPDzHa6kfE6EhFoCOw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b4d8e9edbd5cfb6cf1deaa20076561666fa4495b0a1fb99e5801ec3122061b10\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P7qiCk54Hu3B4aqunPD0L8tnwKqMAAjbQTLsvd9G\\\\\\\/FwZoa9wEWIgYBNgr5V0nU+W4FJbn07we4QDijzaadLE03mk3PPzYmCNpuTLp8sjdiod9q0Z0NESMXv4kPcw8jrkm+OtP1+pIdfo8sNCDt2wzCdYAReZM8oOR01YjcxsBXca7xgn+zKevmx2168IbZ60sEBNt4UDgqWEZTZuhJ3y6BekNwzo4IwmCYNh6wXOUHCjPh8ajja5z6DwIvpT0QJhRw+cMDJvUb7rPlnU1Eh\\\\\\\/aTsvDdnUHOR05QLj1+lZf2nCJjOQgbG0q1+IQobKinSItv+IS6fQ8lDoz10G+2mFF\\\\\\\/lDvcmReF0+sbFtyfOOs2drdepEHWxyp2CA77IHl7zspudCqanQWerZ9kDRR\\\\\\\/Dnblu1YYEMgrfEGBZKL8uMpyqPY7mmbW225s6vBvvqy6wRoITqozyI\\\\\\\/nCEHSEy\\\\\\\/ud9MSGVOKGVe9+9SBIvOSwfzqOKm+3zG8p+dsRNd1Ht5w8m\\\\\\\/GVHw8W8W01Vbah8ZJOnIDmAxsEEqPrO+XlW9KmclR3ReNB3waWsiebP35rSB5r0c8kyYlSGFy3SoBz66OdnH9Z2Nx31TxiFgdm\\\\\\\/KRm+yW1YuLlLgmA5PRiU6LYapGKT7FYiYchmie7Eu51g2KdBH9KNEg4dHDsQMlyUKX6NNEVjCH\\\\\\\/SYF\\\\\\\/1sWh62ZjSAtqXHVOCRzP9oQ3g7yQAmsx1MSKL50KVn\\\\\\\/f7B2taJPtqHA9+VbuB5Mk4\\\\\\\/pAgyTi675RzG+sf71LXwQokjnsKShrE2VtPU10hrj\\\\\\\/RISKNaIgalXL8JoHdo2ToA7XlE1aaYlGX7GKHfEfkiBpMae8mh3ZS7e+rLeb8TmcHb+JNlp2s4bIbE3kQoy+gEOIJw\\\\\\\/QjFPtPpX7tImNADJnxlnxpvSHY2rnxj2eCIYyzZCeuCrjnq+T5jmQ8Gg7h3yi\\\\\\\/23+dY7ZY52f5hv1XkozSNVv2Ykzzf3Obd9S\\\\\\\/ee3k5Czp+dZvtSonYTQIrvOjfv7TJ+9WPFAWJNYkrkTz+lavv62nQZ3HUtq2uWK8sBj0s2xqjLKnRuNvEEUBV688pUMETI\\\\\\\/kLoc2wG0xr5XaXUwzml0xvPyT8knN3FrDAPe5+F3YGc+JGfcHeFoyeXF1IDILR61Z3A3GGckCl4jYUpJAdmn5vAQEGR\\\\\\\/zyrzvCosHKMIYebAe7yqKgM+zJ3MY+OrP\\\\\\\/f4+nnqqKWOxM42o1PYcMgRfzLIVdbUQShEs15KZ2CZ2HepUmsmbHqlVGi3wSSAg1LB\\\\\\\/IjOex5cSpqQe4gVWnC8t0fZS+Ru0tOHBksGGID4CwSev0LXaA1mXw353IPQ5cibM2Z2MF\\\\\\\/1AxAt3JJEL9fVPWN+pK0x9zPVFWu73k5H7KBuSPRfktG2xXvkBn\\\\\\\/P9cUflnK6wOa5Gvf6X0vBGGBO1+oC59ITH6r1uDxJZfOhJF7lsNOwwCdNI2AkaAUefgPL3XkEpLN9eMIrEwINgKpTtZJ4O9RLlRAzBRQcHskhIOb9rqc7vO90yqbnc3e3zXiL7M1TNKLqJLJKn4HGsrZIOxWvrwIP+ic7Q7qij+WiU3Q\\\\\\\/1aIKcZkThJUEp+t+\\\\\\\/xUDoHT4DBZjzmJj1BKDWyMrAvxWs8kF\\\\\\\/wd8B7RZmvXc0Gn5rp3bdAawZoOhavCJtRAHWZstSagoKIohXkC2jVw8ogYuAZoBoKys5Ydiyl\\\\\\\/VwpLzBbxlkeZrybhRXL6FrxRhgX9I\\\\\\\/10klfR+6HBmlUupYemMLH80es8zInm3qazdUmyjh0m4hMx2gXIL6WxTkGOYJUairk5B52EzYTLeg8K0uZZ78Av0DLlH7az0HeMxxJ2SjUWEypTleDommicdJeOVJKb9IWC5JIEL6nBYlO4oc1ECioxuh\\\\\\\/\\\\\\\/ZxfpJPdRYWnlFd56pnPhWR6Z7PolfNEHBNvIhMB9x\\\\\\\/WZU81ic04AAMG3jHAQHwNPRGw88QHurN0xzNQDYbc9LzGXQ3FrvrYDLFYuFFVW\\\\\\\/J73AsF8mp5Ht3oOo8YnY9CWN7yHQghncxDPdbmZj13ATjf1k2hhLkzz8qf1Ymu0Jf5PRm2ahIpaDq0IoF6U90xXrxZEq5hjZV5OH7BP8GBL0bV0vniyLlBUNPrbqEsgBMfkRiN1\\\\\\\/0g2ESqH7v7VUj+BI2DIa4w0aHVJZICHLD\\\\\\\/jumfumBG4V9ho+PNkH8HoNrviScvuxB2+7AV+J9SshHkFV9+1EbeUgT7JOrV\\\\\\\/KTLZS8TYvknFz\\\\\\\/6ueSAN8vWej9go5\\\\\\\/fn4vWHnTMz0d3ai0JHyjEU7FuVxEQOBNI27xGN0E45Pz4vRhcWov58\\\\\\\/mxL1TiMFGp+UbbtQcXKdyoV8NWKwFOQlmnOWcgR7iGUaCiDO18G9bJjcyNt1UQkAVc8FiB+U2jMBtXUJXx9Z43VyeWONZtiVFuREVqol8b8SgLVzoJoU+EU5NQ87cA1MZgKbKjXnkwWWjHRwNu2gpiYem8mo6SUE0n7bWAtl2kPSsyq6QSHJZUwnD3WimRIcfu9wnp212v4FT\\\\\\\/2av724VkLsWrKpnqBTosIjrybrW1eGivv6QoYAZnhF2sWvDJJDVdTti1CnQLJugyHEejU+NfKdQYfGCHqdRMQXsyi9y4wRpDvsAGakz7ENf\\\\\\\/QFmMPThcKtHNhyktqDWrLYGVdVXxb6HKIMm2pOhgb+dQ89jjQOtUNgj1zG4psCjtbbKF8Ndwvv8csxo72btmtmQ4q6UdpisT6dSpS7sToewecKX8nG\\\\\\\/D3vaNKrgSXoF4CyjbtH26Ex+K+cAYsb8R4xXKPMOll3Vf5YIPoZkL5j2H+EqNf2\\\\\\\/c\\\\\\\/261XhvFqwNOGKUTClnMQJ\\\\\\\/yQdM1YEJltKmm6i2AAnHyiQCiIkW2amEaVrrlFG6by0DVqby0QdPqLAPEHiBm+c3i0osYUOWdSuuq7je3hZM5\\\\\\\/FSX16S\\\\\\\/kf5XD\\\\\\\/IdedPr6D3y0umz0amHSzH6AIE8WHwR\\\\\\\/pmCVJzOS\\\\\\\/Vkkn9wkqWKtw6k9Ws4c5LJKP32AKS396jlo28psxZhXZCSDqYJMcZgTPaVWaB64O0ql\\\\\\\/mp3FaCGsuG35PDO+9eOiSyAhh7Xq8OFTosSRrbmYdkRwzY4vv4gUDMxBbn8gCDO5ab1qq5AMWCk44LLdsK+qOy8RCDCkToj7LW9exLGTMNB3b3nezWQ3xRfb72QSfrkeIOEhRxHs3NBrhXFf2Dq7jo5GNHMrdrfGYXxpueWLaJmtoX2z3vhI9i63CwQSIpyv6N7+m86nZA3beKTguK2ZelsoJPsiL2xWLbyW+nQ6VHy\\\\\\\/pn94eMQrHglkfB3T+nhDG+3nnQKT2w3zujDxTUMMRhZyl9uBECLW\\\\\\\/iM2gGKOp3JnGZKYv0Yby8\\\\\\\/ZzjKSw4f+c8bh5WU\\\\\\\/zbNwrj+iR1PCu9iu6ePSM4JrM3QIkMWV+Ww6RU4NI92Vs3AFuJ\\\\\\\/QIXG5+TI\\\\\\\/D\\\\\\\/ZvHSAEUvtUjdaha\\\\\\\/VG0AN70qEmOI1asfJ3R1OIoyOstLFSrQeQL7WgLQjLluQ1Xs4AzBiq7YzeAXE77bYessdjuCTRMuKgihbFgin5Ljf3NzE\\\\\\\/VTTjegBzfYxPhitUhuCG4WzhSDzAnSMRugdX\\\\\\\/+EDNjoyy9NbbwJ4Yc1L6RP9K063GYtnAmG\\\\\\\/w+\\\\\\\/p9lJkDkqz6ggDTEs2GVcPvePxQpNyWOhOryAFr5DhJLqTySCDPhcaKsgxgZ0C3fZ3IBMJCqhZT2n+m9iZ3NwIe6tdN9EqeRU1TU3M8OTClvtM8HuSaehekHmTLguPrnnOtpccSGeUkKbJNcyXe1XGFqKFPTW896gc9WqnQCPFkGUMaMe+z8tjlS0ri0IsPoC2QY+g41xgERmF0DCWbkK3yptiXzdUGlwpKN9K59YSIkEF3YEh8Kh36NY+c4rvx+wX7yS4mUHawjEe0n3280yUXnN6yjIbO5TfuzNBDoP\\\\\\\/SyzdDGFgcDkU5QBjhyoaldcz7GnNKofG0LQuXyECcBInxwfbkDGmUNPEKEdjYtEudhFP4wCS+vxEgQBU6DMrYdKLp87agPSTiFdu\\\\\\\/DAvEtv7k3QLUHbD3zMqb+vnk67PZvMwoE6swEs8kgfi9si9o0dR7sGLUO0A5T6Vd5sXYzIQmALCeUNXwBcrVtzHGAftdDGZ\\\\\\\/PFmvOFmmll2SUPii8NW3ybik\\\\\\\/nDCeYGRvllCwCEs7LaECkNar4ux6dFCiSt\\\\\\\/AJ8wsRPGpMuIH\\\\\\\/w1mMU29YdW70\\\\\\\/zLSlQgLWk7+ka9im3aDIerOWjQreveIk4aEtUnp+PQ+q3kxcWUTkyt0fo7dVRXvfCaUhWUm7NmWjJe7F5RLDg8lf4+4Y7IsnAdEOv37cxxyMyRjhlTgMzLeN9bfYKTakGTdWijJdZVyDeJuqho3JGBfVaGg\\\\\\\/eadZw2sg4\\\\\\\/xf4fEItS8iHxIcRjPkBftcusjSC5n7Xre9qw1RC3jhrixo7M4cp5IJ6HsQGn0cAyyLHFciPlLOMq5tphLtF7CYGw6s0AVReKyl3+Lete6kUOSjH23LnHjVIBIygNoAk9yiVin3I9ironyG4JaVIYKYkCsnEWg+pdUkKJllCN20bEnaM+uptjtt9QlHjKJP5Jhz8g0vhp5nBpuSERA66bVaLEcqK2J2bHDtYPnHhu919GaELYDa4OWtgfYx4T5eadUszKoqSE2uJpkIfonZNKiPT\\\\\\\/INF5Cm3aOq6zI26LEXbifmhvjPO6k4M78nASuoBVXIbHVV5YWPQXXlBafS63mwAif+q+uDhkFnqp8FtAlu+VJICbWPXvASADBdM32o+3PVW\\\\\\\/KjPsRzvmXobG9D4X97xG4p4M3kfvVZCDj+u87aGvFgvB5UOWl5BxMsLh6SVfdFh2c6A1lXXgO9NoJPL+utEe44FJnBF2zLxi1jeRbsbVE0wtVwqrRtTaY4siwJquEikPbNPatHMNGNxENuK5eVDEoBgPuh5ABsfVkb\\\\\\\/17f3tLpUuefcMt1jeveuBDZ41FiscV6qbD1CfPQqf6DnDfyRYYt9wNXQf7eRVGwEuIoULCT7aG2rb2bhc4TsCC4cep738tAtxqfNAR7pFo0dAKlQGqU75a3TGOV+MqHww5T\\\\\\\/+comd8ZV+EQbcfwkf1YPpFJpPft9Dzv3R+kvmBTl8Uv5LNt14fToWU7ljTRjvg90msqo\\\\\\\/TCsknGZl4nWIrQKnSGejzyguWmeJeqglgskpYC2QCwjCVK8JYX57JnRu8\\\\\\\/Q64NN4cZNZdkwcuZWqbVy\\\\\\\/GFVtq6\\\\\\\/VKXgJlDZC+S9d0s55Lu29mMQ1kwIqRHLKpIfwB8napWEzkovC+OQT1Wg82l8WtNQ9SuZBLr3orOeAnKliBaIMTEzrcf8fJbHlEnd6e3GUzSz9Q9+mYmTIaDrc5h3S\\\\\\\/it4FxYJX8NwlRK9jZ3z9OKf9AnOmMsenNpy+N8dR8njpVXhgFv7oHvEdBpdzeo77RzredmcqYTa0019IW32K8aNnyCrqCKqXBhHfnyq3\\\\\\\/qOblJOFCb8fCpRND8nTSZxstR3BVsgxpBKQy6ysVm2Uv+8Fn3Rjd5arRjwcbgL8Ko7mvU\\\\\\\/kXpKXtDkXurt\\\\\\\/Sk0CHqNOEbhRz1YiCwQlJEiW68OjKurCDW+LrV4BL7VcCYJ1wY71bGSt1LfASP1kuawu4CMj7JpsVRk5XlbolpEskviXen7951AMQ92RVXNNmMGW\\\\\\\/x45F48q5NZ+WgoD7ldw\\\\\\\/6mmcW8z7o404UKFbAJj9Fnf0coqYSos8yHc+LIPwjna9p7kVdqtwAdxI7v6Yb9D1T\\\\\\\/2a5PTKMt+0fV7+LcEM1BUABo8HNzkCCEov35oqJVktDb++tdCstH0yrYhUAlsRnB4Tp91xGEIX10wXDImrPnqLYjNdnIYz2LyKGsY5hE3hqxgXhmLSIdVYExlGOnE6\\\\\\\/yQeN0ATht3WUcgIjWBhzo9BbZXbpoCl\\\\\\\/ALDMuKYRak\\\\\\\/DivXANMAbPsyFKcq9x1FYsQa+JqJJAtoZtY+PJ4FIu+Td25R0YLkmvZ8vSy2aqdfGm1tRULNUaIDXpL658n7p0CGchUanKQGdZ\\\\\\\/0kgQeQB3cVTXbtQuc+aWJ1UayVpRyuGKzIu+piJcNQ+SjOC3L9WcHdt+P\\\\\\\/fmS\\\\\\\/j6BrmTTfaYZug8O5dJX1UAPmRkugi67\\\\\\\/8g8nl3V8xoo7Lrh6QPpo055nJHQqj72Cdc12LPs0MlWqTAwosv0EQztE+p9OV1oypclzg7hcW2yjFMB\\\\\\\/DFK17VniKCwCGpeL7M85LN9e\\\\\\\/pa8pUBcY4+w\\\\\\\/NVvrd3ylRO3Sz7dvfm\\\\\\\/m1JPKYx2j13dGlVmjDosaA7ttW6Vdk\\\\\\\/+QHITwGkWB6XE02oVr8wJ5bg8Rrd8KFb+E+BCkBw60Wa1HT6m2+jr78MzWbvPo2BKgftAqQgNQYjJpv1PMi0WhCpckMu2GB+95QYg8HJw4BGc\\\\\\\/Euh04V6pgx65osLtlyK24czg3FrZRfFYx7N7Z6jeY3Y41ZPGg7oCPrqsUwr1T778INQKAZ4AVSR7smgrLDdLlKxlf1nLh7qxf7xu5E+FC\\\\\\\/uJCZigfJeYNFpgunmrgFUcTITPPqDq8jM2Dgqr\\\\\\\/2daZbaid83csVd0rO1ZQeaj4lQdb6odENiz1nZNUGRaYC1lwfXmsidxvDdu9SfrHdrDFXK\\\\\\\/KKt7KbBWho1w3NEyOjWWkIZqckh8rki1oXE2Q2JA4ArN2RWf8wdQdIiZdohRS7ej3sZ3uSKYaUvp+1e64+CvSp39OucFGHRGPaZAC2xQcUeduK1l2RPISD4\\\\\\\/EgNsqQl+4bSuRg9Lf\\\\\\\/WRtzjwbrGtD5FyJm1xUcNF6QJ\\\\\\\/ayCCn6DuHnCJfqPU8MaQ8rbLoo0JEvdxaNPFS1dAPTKtHdmywHtWgVygdRJNByBLvFjroQM4iUwrgL0GNO7jze+rxs676BXW\\\\\\\/nyMllp\\\\\\\/SK8G52d3jeZbyPEUiezsiSx9ZCBmmlzDcTRZwsWdPMPEVL671EMcKrAfVK+TlsfXRh8engRbZkn0+Sv3rkPER41Sv87\\\\\\\/hMjlX9DBxL2W5CRziO4ZOb67YUhrNjD0zF7yxoO6UOG9Zyp2w6mOjsbtIuWY992zKfWfe305flf\\\\\\\/xW8IQ8+P\\\\\\\/L8k1eXu01wTz8sDHGZ9eKG5I4qsaHDWrOSxM4OXuvzU8Zsn24apwMamVu2Nhs2kZoTIbvw+54YPrccms3pT5O79ZUnz0Keg39hzhjZ4IJX3NR9mLQ3xbyT+j5h\\\\\\\/VCLWfiTLXK+qQJrLZTE2+6i7JGVj4jo+RHbaN6NYkkxpO\\\\\\\/O3gCoFx9OZk7B7hlzd9DZ88L4nae9gJ61J8w6OYFc\\\\\\\/ubLLm\\\\\\\/8wxyNOjtU9m70rrLgb7y6QjJvsbfi\\\\\\\/0obMlKGmhohRlWaSGuc12gfWYbtBICW1WLaqpCTWupILDCSbbhV5vh\\\\\\\/SYQZ7qYuxc6\\\\\\\/CXUJ+KgRlQ850eBD961sKm1ymE\\\\\\\/15C2F\\\\\\\/Jx8Uw65tj7CFqXyvnUTcbGMjh3u4BiOWVpswv3h5kJKtonUA\\\\\\\/h55Y\\\\\\\/Nqh3lA7GlLkKJI7TYFU8Y+iPrgUmpczV+SU8O++yZf+NyoFYuag0aINGsb1wkmNGVyq4PH1h+PKhZXt+ff3ZC\\\\\\\/8vtrlgfEK\\\\\\\/yQBZV++OyV9b5gzDT9Jc+vqZxx3LQB+0LJlzzWPgj8WzgLySZ8L2o+0vlIrwk23okScTNTRAgORBRbZJS1vDSo\\\\\\\/uAI1Yvic76rTnPzm0xxz77dWw3\\\\\\\/UOgxnlNDwBB5bOGBeOV1NOKuIQ5hcLig3115DrCeNYlalPdHjv4exWK1gmgMnJ6U9WJTA5t84CJ3YcM6SNq\\\\\\\/YKiwn\\\\\\\/d3D\\\\\\\/NkXV0AcSPXV+8QUNytIfr\\\\\\\/0WyZfDgfnC4POC7jrh2JYSpLq03o6wGuDwWrP9+irNnSFa3iTWLNVAzP3GGAcz\\\\\\\/7UD23P7LoIfmJNzO+0EUFZEU444U\\\\\\\/Pn+wZODTot8JGp74PG1wIh1XTStQ5v2D3VjANXw0cl9fL1Sb1QsDBsaHos3C+FN+B2TMzJmguKwnC4GdwfEmgkAmar0cwiTl\\\\\\\/sKtUf93oY1WFdRt\\\\\\\/EfA9xERHHbSDISFta6CxiNfvKMOg75zEM9x411shFfTgsOoU1wzchslzRh+6fIHBtr3GPnI2cSiqQF92RzF6kWccXHRkTxgkQA20Jg\\\\\\\/Mk0bG+OBMvvH3GKYOZ7iuoMlP1U9BiZG2zqlBnCcGYNDvzw\\\\\\\/K5rc+fxlRPx0xTgHG4RWGd7xfeYiz20yijAen8N9RyRhPdRxqb5C\\\\\\\/G3JXlIbGaaxU4YDhh3KOyCf6RJxO2pWpbnUWyk7HZsoCwbw22GkwnU10cyBzxkR+ShhykmyvbG198rSp\\\\\\\/6XQ0GUARna+L9Jal37\\\\\\\/4Y\\\\\\\/sYybrU04INhj1cCIbH7yL1n2F8z7MFHN7AFXLR\\\\\\\/v0iRUW8c5fFz\\\\\\\/07IVs6HOQYBfcPt7wRsygJLmgij\\\\\\\/oWlfaR97vkEQB2tHclR1kZ0vKOpdWXPRfnKFliZLXt25Axenxs6K7qy68UDWdv9T+XX0DR4vS+9OivhMr1AoUwMG9s53zFjWmIqh1sFCgDcHlvekrcndaljY2d1knyEm\\\\\\\/YQk0S3VocYgNeVyQEivNIGZMJVHI4RdMlK+D3bsYsAFX+EJwdfrttQ3yGSc9CbmgDcaaUB+gnmKFN6ac4czB+L4JNczSqjvlfTa\\\\\\\/9X+ddGX9kdoxPFLf7+A9HCSKfBZ2NPFmaZAseK3n0BkS\\\\\\\/FjC+bTPwWF0pooOJxdxGiEbu4fh\\\\\\\/CanFr7MP6BkokGhNODJ19CO72UwZfJHuq\\\\\\\/Mmonf9h839y1sWal0FkxcEEC7EgrhQD\\\\\\\/ix1q9BRYRKE7kgDOFE\\\\\\\/Z300imhaWvKcLL4+euC3\\\\\\\/6pvkOHgCOZw6Y03Vks+tNNqhiJmvd0bI0Yh6\\\\\\\/pmPgyN7vM9ZZoQEpydEfZngLH1ZbjluvewgVlphxrdfxZa4WRGvhvR62G3I0kY7C8fKfVsiQS0q28JOasRII6btkXfIHnmV+k+hTRC\\\\\\\/eOjDt57SyzHCwh2PQrxkiBSNYYipDeoW\\\\\\\/wq7adG2JUiNm7n\\\\\\\/a4I9QmJ6Rlg2Qd2bt4B7LZ5HiatLigL0+xcl1lrfFeyVc2w6EEIFLfWVu+vyGSECBrsyo8Q03IeFFw2XwBQ17rhTSG4VnsLt0tX3kPHqXYCggQDUJliPHIfNUjtKRVkok9vj2EhmrCnhAzP\\\\\\\/LQkrjZ1uBqQKTLNOjcSiq4jU6KDQ9fAd2IKHa62srQENNzLsoHYEbAvohOrcu+rKOtGRg72Tg9pfWW3ysHcYeWz8eSW\\\\\\\/eS5\\\\\\\/PxVgH6pLW+xk6vMF+95yOUN3SlpRvKiOztsw9F4\\\\\\\/kSHuD5NUd2V2xUffpPLmIYKGUyubz06pATVv2JXcUICiHgs\\\\\\\/2arrTJMWJnQ4ixJL9INIeltjCSkjuLPfoZMjvnFpYEYffu2lQQFV4lJlN2PdjbuyX3ekOOHHv4dth0\\\\\\\/QBIYUSK0uPKXwhCzAB7eQ7IB7YNly+nVh\\\\\\\/UOf19Wz8pQKlxWymIXQzBDRklHb0DDwZcMMtpRbwyB+HQ3UYWgIt3ZUwALieNYljCEPEm9e0hkLUYmpcql4JSfuPMsBhD7mH0KOjfKLSHibMFUuXb2kIuDv\\\\\\\/LdWIBSsJRiNG1c77kVKN0aQDvDOYHdhpYtrrkAqz7JJc+e9udLCdkGtDH+0QE9Bn\\\\\\\/HwhOX8Ws6g+RKsHybg4mCoLFa1TM3HLsTvf4AvrGzagwL+\\\\\\\/eb6KxtRhsrBMQAWIZ3e9ZO0v6t633hhO+BUQFvxq+HKti+xD9LsX9KshyCr1iOvO50bjPdeqXLwuQk+IkIdDuYJnEAORcno+3SCZCR18Uc0r0BtG37zN1bAMzyxij0ZSxQAg8CKVoSjPUTenf3us7+HoogOWedzdSeIM8yR1DM2kfrr63f\\\\\\\/6\\\\\\\/aNn3kEnz67F5N4ijL71PCXTURhxn87ESz+Cbjluwu5LVoKbFXvGKMf9jgBsyOP5Rk9EN3h7KJa6nNg9IGX8e377kYyW3fAWqjDo0QGDN74\\\\\\\/m91auTH9JsIu6jPu1jD6VZUgsodMDCEMBKNtwX+0AfKm3iLL\\\\\\\/EX9KdxyTX1lw9IPdR44p0MTsJGL0VA+lRsXFmgyOraw0H3O3as4KVtClW3JNdhmaYy3p5HUQ+QHJFOrV\\\\\\\/NsUGiOBLk5928uworXmas8Jmm6vehGopjIYi51NJAE37AAU+FSGWIXyz612y44gpinlKIAlf0ylj5qoQFDqj5Wdc716VuDNGEwMO6hhmsOYtxAvFWgNJ5Mj6DuuTm6llSlS2QUx0RnHCXoJOSRKhrwnN+KAWUM8c097QofRZ9GhsrkceVFiXjrenLFAuPU\\\\\\\/ONIXk6C9HthK\\\\\\\/CDJyqXIYkW86QSL9CRyeqTyb13tTOZMuYMPFy9DT2DvUga4L4KeZ+OogJjO\\\\\\\/27tbOR8eK1ZbDXjf4tEnQyQ8x\\\\\\\/Bx2N+z2dPIng9YTNtSGLAhgfFXOvaH+TI1tnoRQLfVBEedTLq\\\\\\\/CMq1gnb\\\\\\\/ZNOt+S6Xora0zVvoOrH\\\\\\\/hwUUg8CcImZZzH3IlQ9gwTOGnGItMkAQOmIITfqqGQ5g7GkU5JobKo4DKaDzlIwzanz048sPlZ0UMYcJsJOF5\\\\\\\/9UHx8UZMaG3MXd6LE29KsQ4pcMV7KU9nQdK2OnXvJ8yHmqR2ScyqIEYJAyb4NzGf\\\\\\\/kz4pkLokugBTH1OfaF45G4zY1Z26IbIqAiMhmxjQ4e+FCXWR+bolOtl314J1BdYQeFO86oTl07c4oS+lVpkwJIL+plYqFO75xzDjrFTsu4RwYtMhvPvhJyU6bTEbpFjZHdSV40JdhQyayRXgwNHHKBx\\\\\\\/XbK54HBXu2UlITCB77bSk3JwgUphNbvcT0zp4FcHXKopN1zbT4kDpfMHe73j6viTJW7kjQ5LKPSOo4C2Xxq2xTqANn63WiGyR6ur0joQwxJDvg\\\\\\\/6V6GBftAAr+TVOFGwVMdqRI2VdH7Dli6DkYLsG4dOfEmNk0QD\\\\\\\/KByM7ain5rFbwJiFMwLHzxO8f13yS57v4uPyt0o948xA8nP6CsZlRi3Coxc8S+Hnd\\\\\\\/Hrl\\\\\\\/GT7UwekNm1JocQTBQ\\\\\\\/iPXf5CRr6jfolZ4Cf3kQwzmSwnFNmGcZrHgdTD16MAxxr3Tp4IpWFOy6NmQw\\\\\\\/c52dL+ud2C2USfmi8lBAsZ6WlskkjzLJm61Q4Sg5eZXr5VXlTvXZ1tVYWCc5DCSfan1XtnHMLYtzJAzXWUWOJTWhzpmvjGojXn4KOVVzR\\\\\\\/qz\\\\\\\/kb\\\\\\\/BO3iNUU9Czjmz9ZdkvT1GPwXwy3oz9FTdI0pb82bsqxb\\\\\\\/speevj1QV5XSmR3AzKtZlLYCw8x7YPjcoeQQzF2A+5f99OuScthuma+bpbrUfOMJ3ubB3dMj68QlvQ4oZW8JXlUtfhcGnU5EyXNqGsvXb0G1MzxNkGgbdNSGtQlTSaY40YGD0jliKyZ2QIU\\\\\\\/8HSCTrcUtJLzPhLJLzCRuORXcQi6q8+hp8hzNXBZnn+6dcpl2W+3hzw1RMBmOIe2sDco1cQNVNH6AGnElkDTI6UqsA4cyn0G8RpwBM9KljdONrSU7Fk7ycc3UTfcahJtNPU7+O\\\\\\\/\\\\\\\/WndimfCQKEYLDZMScwtL9K7UFTuMEspNq8ufii20gmD4zi5YugBlUFMRtYxLphjVvWPw+cHijvlEa1qcYtWelCFT\\\\\\\/TPD1QgkVDp\\\\\\\/RPyNbzMVo0iKVTWv6wXLCLWkHmo504WSwZyvKOGTNkg7S+\\\\\\\/LciiPjZLiu+eOlQQzatxXSs+CejVXirKgmfpfeWAQF412b3GR1X1FQvmsSJC9NJGwp5gJCHmqSlnG5dsbA35U1XKBCmd5IF0nbXZWNIHhIdcvlgXQimd\\\\\\\/Qubw+mcefR8\\\\\\\/tgN5mXGRdMwwh+06XPSoo5hwGyOPtM0TSPt2yOaXsu2KMWQsekZN7JWR\\\\\\\/T+5oqA83fE0eayvGe2CiGmrim7Wr9DbyoQuq27tD3FDDBgNFTZmBh5xy8QbX45M7zKl9vKzBZ9f+rzcU3ByfYtf9o8xuE8gybkoY7UTogdqs\\\\\\\/SSfSrc\\\\\\\/bAQofcTKCNzBysBDC3v6rkt5rYGKnBWK4V4v39VUe4t8K5+PYSM9vGLN9S3HNmdQwsVhObLar\\\\\\\/VtmhNBvfSdiwwqss0juazTjeSs0bKzmYQkDvKGJIHt133sFSvGFeBlEMlPLTncH7rE3NG+0Fpd5YZlL4UZF9\\\\\\\/qBXT5zf4GyGRuB\\\\\\\/8nDTm1FveLCY1L1SHJ4YLY6HYUsWH9fLsczLotstWNRXbPDSBDOGFWfFNAm4yXkDeR5n4W1UbC98Y5kV\\\\\\\/DQGcHKVIznjkLTe32W4nvlaEM4wOfAR3t0zL+NWVrAFLyAaFltErtWomiodSTAlujc8jvLqXfQr6V5STX3I0BBHeqFqUP7CExq+Oec144EASy6aUf1+ed\\\\\\\/rkyN5rtiId1e6nYWh61CG5C19Z\\\\\\\/1TELhSPAHEq538Bx4hwUIsuqRjvCFJPUH6cIRar4+iS8yBbsjoc+D40clzp+9ZOxnjQ\\\\\\\/SZG+OsqnzPYbq09DYjldqq6qXfPH1sZ0OWG+Ezf\\\\\\\/7MZHE+26a9Wp72hR2PCM4GOIpFSNouiuihAPbkUMCKyAMJHMsh0VQAxck4gcR6q3r9TnNDHj9X0DG0ReHbks5xkckBL2B12mGRnGKHy5x4wjevAvObEc9xBA8Mk3B+cD9ih63RJm9danUBE89fXcCH41oa1Y25ZbN7v\\\\\\\/ylkeU8Vz70knSJ4OeP2k7mfOh\\\\\\\/dSvdkqbBFViblEMfdxpS6DB2a4+zMCOcNPbZeiw1fpq8PMhwtUAp\\\\\\\/CFpN7Q4dNRZaXCgGNIDBKK2s7TbHldlhRzD+M2NNlcsl4K8ex6TzGXh2Ym2UEHp25LwP+2KVmDajKzzzHxMLezK3bicrOjYhTEPXfUGio5B+izJ8K6FEzqFaGhVGvtgTxXZiUb+NVYl2VG6KKlrnzRUgf8li4ZjRtEyRydmkzALGrt168xNxuHAV8jj5iaBR\\\\\\\/CoRSJ\\\\\\\/HFVeAy8wqLUlTciY6Gaku51FMz8S+FubbTLhasq33+8greZNspwCTxidf1OQ16oW1EZ2XGK1DXtaaVkmtjwAwh0ClT\\\\\\\/9sp1E55wp4kY56xqv2aYTB8UTYijDoPCIKy7JZcos5cXuIBWPKpypC5iqCZWOJ+VzQEEFB05kx1lRCkcRoQtEJC48KfRwyGt9X3eRiHZwJKiZZWRaR2O\\\\\\\/p63j8TfTEKa5ZYkh48TlbJLL25Q+4LFqMsFfbtDxLrVbGTSalEeF1AxMXRrlGBrgj93Zuc+Be2IBOcVgv6j3\\\\\\\/pQZZ1zHxmKkGynEJynyQPaOaHTDt1qWpD8SWOW4OblLvABglub5rx7qs6jFcXX3YoFjo4Z8EJ5fJKp0gOMme5euy9RAExu4Kr+qAET\\\\\\\/\\\\\\\/6mvy+Lms2vfMRU6Xnw3qzzjTw79cgErtOyvxXt5OMaEmckmL7gquFqqqWGCybmtSouhf+6n0EM1dIUOAf1ypX0En0eeI5CPBZAKJXf2xmg3021+VBMd4+QIqJZliwA\\\\\\\/hR1fj\\\\\\\/GoNMshNPdXGibGywz0A6WHhLZfp4p+1xe3FNGMMHzO6\\\\\\\/\\\\\\\/+qyz+pfMsJ4f8KceC5P6\\\\\\\/lMrp4TEdOJiklIaeYcV98MenkflC5caCq7wwFa2lnNvvQsWX9lKMkLnbpXh3V+QzqtnjjOTLI07nxImTvm1Qm5qVgw4SOr3AMzdtMRlBigfKJ4VdpizdKpAJGDLUIBODEoZEmXZCNDtH\\\\\\\/G5nkEEt4szBqkv0OubBG\\\\\\\/XWOLDYoEPhxZyNHa2NqpS+P4im7Vc+fC0Hu1z+ybQHaSlRGQXEfDoQn32ozIb0ps\\\\\\\/u1toASb56wpXyrxb8qHyvwQ0mizz67ZXGTOj3OXrJUmbKWpjF6uCL7PUIcuxzAzt955ri3Ke23TQKGI77iiaxCkc1e+pr8TSU6J4GnyWjdxtlmgO4nuO\\\\\\\/BHfuULmC0e+f7c+pHXcM7CNQliN+KYgeX4zGx8IiVpCTYHaWpr7KAMoOkaHJQ81pMi1B3TBKCP6IkHr1+Vigk7CfMj27l8zs0LINTOQAFLGWMV30XwCZcdU+L1p3Bw8EIPxXXE6vxD4CVvwIIycHa0aIZmnHn2X+l7ZeiC8UpLLtJpm8bV9l18UxOqoM6KIpPq\\\\\\\/bVfXZmnwM8XiZqaPWorp\\\\\\\/rm5Aq4+KemvHq4m8XttBYj0o76BdLOUxGvZvyPf3YG45JqLtcVhno\\\\\\\/wk+JVlxz7Dq1+UvITvqyJi0lbU\\\\\\\/r\\\\\\\/VBonLCBmtNj9gLmHRJXt9RroCfy21tQ3aXfMV5f5ZlSc5Id6fej2k+jOxAwt5sMSms4IgTix5hkMIpSX3AMCf8YCr9Wnzf1an8\\\\\\\/1mfhi94KWj17KZq+sojtONYjl7NDPmbfNRIZlcCljgMkS2yGwZz2rPwimXU8Qh6nhh004vDNemipNPFswvKs+o+kMvE4Utc9i0+57CmQRwz+t85v2SD0ALIZtQRsV5YuCYh5UHxLDtuVK7U6nvzZTuZsSqZxdAA7eV5kU16XGcQTbfehdyXcxeEgc8a\\\\\\\/ib+yqUNN2+2Suhrld7rsXdU0w6LXKQ6ka4WE7eHRO9R7cCqHgM6AVXPTK9FTFtIsNXwsRaoC6pS3XcVGIajsyRxCPyDjh5IFPpLxHjxC1QseyfrzfJ6IiDpKv4AG0a45nD5w30hCvP0RHURmFX2A8i2Ld9PeFfCBlyHyg2w3i5cxMU55AgfIiyM9QeQUuIRoXLRAcfUmJ+wVnXt9\\\\\\\/QDF1Bc1Wq6TTYHuKeRhh6UTxsb64E4EDgiOIAnPCidNpkV+kUC0q2cTTPT+6VFhadMa5woA26RO2YMZsnv\\\\\\\/BRJSp19c9sp4YZLb8YRD+CQN2AHr0T1NfduiD7tYbsRA7EOcDd08tBt5+l2fbc+rwo7Q5CqVuS75LxuUIs+KMOWmKAbyx8qr3YXbQf1QjRI9gPX0zeEpCmv85yCJ23DiYI7hQ9A+hG49ehBoIqOIAjMDmyFlFt7E1X8UlPbQccWgwJpSFUanNEROZf1uAgz2Teaj8XxOyAuTSgnmbt7VT8ZE0dGX8dKoM00RfzYmpElkEsolpfLmDjgwpStsYb\\\\\\\/2xgLHD+ViGpmyWd2e0MdvVer3SnOVddvn9qiSzsPY+jaCNNFzqlPJabIWcqJtcrSIlNz2zee\\\\\\\/OZWDU+OlMLldEzFn4sAOsV0gtK2QjaA8sqyKki1BYXlDJDVTx+fmAyRV8Ay4D6tT5k11k69TyOwGHausU+HOH36P729lbZdTP4ARZ1NnKJUUtWB+xa\\\\\\\/hnoppizvhStMhw68JFH9o794olY3iL4xkDLbpEkrHJeC1x3e76+Ci+47JPlxCPNSw4Fogycnf2op\\\\\\\/bFfXYwD4UpSt4awTkw4nwjd\\\\\\\/f6L+vhN3MQzyiFx9ellwPAFtaK6nNi6fhvTar30Uy0WS7p6WfThyt5ngwaWeUx+Aa7dx3PueKwfXFvtx++OACVHrfPPnUflwcdZF+Q8qEBt26Jj38SUufF04csHusYCPX8W0Z7N46tzStXcl6YRvCkp29vFoyWamrIzVpazefCIDr9eaUWK3g\\\\\\\/SgbqIfutSeSzu+Bblpjb1bbURudNtr8TUnObEwQnBjIIzxcxowR\\\\\\\/zJa3+IUKtkKb4f7aYhhpSca81Qf5UfWPMzo4vmBcPGC9\\\\\\\/600h2gk3VUkHVQKkTszqOoffcfHw44LPxnn37z54CjOEClLxOun1OPCFgNbetvhHgEvfDTCO0sZ3GhCJpEnyqu5ddbHGd\\\\\\\/bxQKK4L3KUtIh3Bn3RhUMgHV5MjOLsMJqDixwfaMV5k6YHfBsjpbnPT2UagUPqiUs7m\\\\\\\/pQDTNJuQLsUcdHtaWEGZMqadN0bzyVPVfsWtgoSCTWE0qw0H58Keb6HsfgR+XERWOa5J4uKtG0HLOVbxFA3e3ckAUI2mk5B3dspsG2L0X61kYJXafnZO2wXSyDbRy7j+7ogVEFslJnKSJD14ln9tfbCR\\\\\\\/2nHvcKW6gq3EtAfS3bxzCdWE5voaeXQYjEld5s0JglXWM2cBtBn+8fOKJWXJwU67SOLHWas7zDWQDngEbKbQIQg4fB1Yf2CfLq9iAfSXvKPmmLXjqW58oRsewerN19CemxcLOoyam+mUKnA7cor8Yue60umq7Sid8+MUdKngqhnUPaA2ELU\\\\\\\/ZoK9zFShfjAoVY4FTALSF\\\\\\\/8eoOKSwgchvR+q00Sk0SVNSMxCS9s31A2B2WNklYPEt8KoiovSmBIq4jKJ8zXrZHfK9abMc1ksf5lGtJnmhOfkJ9B24JKusytnpOl1xyrLS\\\\\\\/+9O8+Hig7zdXqFSXFitzAJj8JJQfWwnH0bZYRcQyMvOGxWW9GnwUfpJZlvZxnRhR1xvEFhGDsMqgJsZpVVe2kMNaC6+XXVj0ujsZuRNA+N84i\\\\\\\/tQIndnGfiVbd14vw19eP4SNvGdy7FFf\\\\\\\/8ExCTmKF+lL+DJX\\\\\\\/vIElgtC9AuMoON14sGvXhc3JAq1OU5OHbDoi85M0K3qUNUSEit0z8S5AYWSzzliR3QsTw0Iv65qJ6atGJB+ZstAz40nKr6LXiIKSfyUFZTwsRwV0DM8JAWqCnAdNwGDP2\\\\\\\/9sxFg6W9OPXgn7n7qkJTPXsT89bwnqBJmvVZ7I+eQxXAFKw5mgRgGl1Mk7yIKJPAliqRbyjFZuqy760wwpGnFQ7FkKDmR2T9QLGOs+U2BK\\\\\\\/8AXTlGUxfASX1VxtJRie0Q6D6ridTDaG22wqkBOlQYcBZRPkHqeHmJgtld\\\\\\\/JEojzCXzI\\\\\\\/4aH56p6efR7WKGkmecXvAkvPrdWE1\\\\\\\/ipv2NJ5+xIXUHAGqylVCP1XMn\\\\\\\/rC2NB7w7\\\\\\\/cm9qnPzcS+\\\\\\\/mf7AZWKuKleX\\\\\\\/VhFnuf8GTfD0q7ohS5aiuoN\\\\\\\/sFZfoNGsTpTscJOEcqQS\\\\\\\/AmH3IU4J\\\\\\\/mvvV47xg370tQJqlIxIMUXWNzqsGZ76pkDQo4LvTWbeOVwr0WSYO7jRozplII7kO9OKxEhQUFoi8kUTHHdwJHCsynM35Sy8auKpgo3sziDHF1frVLZfvuzfk+ozpPd+apr6Y7d+SnZHLVTxy8C1DSYhq8xb2GsrsLMPZHfgerxJBxxputRNBqXID7r7Yl3nBRCkYP0rljkADbEK+wwudSfkGfX0ZGBjMBaQOGTx2P6qK0HP8Lt+I4nH15WB1c9SzOMIp9D35Vw4kG+cegWrfZvO0zZKM0vMclUgDpyjdYBAkfmQ2U+P+3xMRR50chNgk2AcLHRVH3UQKoUsEb69cR9spZ\\\\\\\/sDT8VtYUZnDqe3wF5+CHxsjdCA3nWKk0S26w8Y0z05WUF+e9xdcLOJSifg4VhdB7JP575ofa3M2usjY2G4O4AN3yosfIY\\\\\\\/zCqFdzmM5p5snUq9Zx\\\\\\\/yTBE835KZAvrrfWEzXzc49MIYYGl8kT1xbCXII92KH7q1rf+7M0FnGVIiDOabTXDEH+IZH961ynUuxomlrQg6CCvhP\\\\\\\/FBnPuqJqwuJ7OGJQda8eQ57qf3oOmN0HVm75sKDH19dCGzNYfkeTJ\\\\\\\/6rnqpLh9qA2RXSFzfEAG8SIBfkYhV7GeI8BcM7q78N9QurFRbvSffuOItH5NNVWpZx6FQW6SLvYA8DPlm6mcG7rU2FHHe6eQBB4VtDJ0NK+NN5vOYGxI\\\\\\\/GkYC2+rYnwKDhBgKFAs27kYkDRLhOHqlOsGdh5rp9xwVI0bJH2MD69BCcX3tNm6iLF3931binQwz\\\\\\\/Bin\\\\\\\/zeFI\\\\\\\/1lYp22qgy+ngjf77qTn9sDCmk886nvzCyQ6BSHA9hmLrzb1tjmfnTW4\\\\\\\/caMp4dsl8bCjEak+U7nUZoyLlY4PmUQlMQJjthkW485tQmJ\\\\\\\/DFntCVuTHmV8PCJCynt13nlFGAeLN\\\\\\\/jcBXv1ihQP83Q\\\\\\\/nDOmPQQiUveE15h3KhpX1G3jSm92oEx85iEpOuOyYtSPc9V1Ub0Is\\\\\\\/65NOaO8H6mtQZWf5GqgE7AM7\\\\\\\/E\\\\\\\/x2b\\\\\\\/YzBJ5bpxGnyyuOQIEa6qzPtXJEd7Aqre189KPZEq9TvDRB4LgH0q3IIT9PD9RXW4Tn8pw5vE4bw\\\\\\\/abUaOyupIHA0pBzuxmpiFWNwODRdSnRLMqpCozMmxF0jPSmjARp6FGnuzXNE\\\\\\\/TTWXY\\\\\\\/gcvmWwbqSbSatQkIaFwitA2+wq4JiY1j3VTIGiMDpaPGMUq\\\\\\\/aDP93wDUL3cnJt2O5T\\\\\\\/08NTT5YD2KzOxc2jLzsFuIptmuopIIW2wINuILXA\\\\\\\/dhMoBtfPNXuDECLIxwJVhAKuB2996jNioSZuRVzFze8xQfYFZJTqIbURRF0T+fuhrBxAjZzIufG+FC9Q20IiMrjQFxXEhn03lghnl+nh0LplSSw\\\\\\\/v5GBU7kTnDh5YIjOd\\\\\\\/RHVIktMuTDkwbPkgU28tH4e3Tmd0PMU3f3jrx9wvPqIAStX6TCDmqW3dXa6mbrfPRflR6fublRbkDb\\\\\\\/Ji\\\\\\\/yxcWHwfe98jrA7orHj3ywMmMpGG+n5vIbHE1gTRrFEziTSCBlqGRIvQmw90lvMXmOx07bHRr1F5kMOiMelOKLpWbm+vDTb+QeLTuVjvS\\\\\\\/80lHsBNUJnm0CV8hj9D77wkfEZ7RV+YRNtguEquKIPpXJ8G+cSsbKbEsRRJ+57KeLRmgmy4bP8i2Y11a2z3ugVmfIVEG0H6h5acdOkIhERrVPVv+WkSTo5TUzm5QHr7jy0MRedE5378NSCo\\\\\\\/mC102T\\\\\\\/GFelK2iyVeg3QBlAhMuGE5AepjAUnfnqXZMlxNknPUa2bF+qMmkY6KlqUXMo0dkqE1lVk5t332Y1YkloWPL8dZcApL1AJHsd\\\\\\\/P4lFFWq+zv95XCytLwOugdNJr1HywEPdmhSDLWoUaU9cHbpLOMgz6rTafZzAAGzS4H8L9JGNS2Iw1FPvC+nR2FUBlifvNOkwqhtd0Y7aGMOaQ+YLW2c6Ed5GT1QIINHAnuZNN\\\\\\\/uZM988LZzzf7PWKLQOvwi6JR4ReNXsDzwn6LJl6rOyAC+8gLU\\\\\\\/cD7VaO13zuWDUUDfc41nVEtOZL9S6l9XeIZxuJWVfhAJWHCP6k8beRKpZpDWAdMkj0JmvRJBMpq+cb\\\\\\\/1aPySu9UlVWKKiIaGmIy8Ufq\\\\\\\/F0ODee34z\\\\\\\/Hp8YT4+Q181bg84m5I+xRMXBUDqjP0sP+7qlhN+r80LpUYAKLI\\\\\\\/RkxcoVNExKUg4tAbMmZqCHIaeJiZjYnwKkdJOlZlznA+5WZk2wtiLu+BxGUxzT379GEKL6gz7l3jaHaFVRMqikmpIC8mE2YK+OrYOly0POnZMn00O2ZE5jP9ADHVDYQaM8UaCVxijiLFH7qgz9F2Qg0Caew6QDGymnufH9OCbp\\\\\\\/htmWmLDE+eTlrPOqFxjGTF1NMWQP2cj11dNW404Ot7HCrzoUJiR25jkfxnvOdYxD+iqh\\\\\\\/dI7XZKb3nqG0txAga0rd3IBbtzkgy8+nLbWQAR0ttgYVYREFfscHiMV4rRmdSzTzhmxI\\\\\\\/YD\\\\\\\/qBGYfoc6NLXiO+USbNqLJ+e8A0Mh6oaHYRChS0ns9VDG1wHLhsw6X3AWReGEhVtWCFlh6GDV11AWSK7NlizI+f8QtBSIMcRDi36VRVBJ2z\\\\\\\/0G+Gi+R4kPfJl60mVsVbQZaPl7lpgMsbSbEubwiwNLLDmSwgQcDngAsyu+BMJ\\\\\\\/AUkT\\\\\\\/ChKWDPXGUfaKIU7uHdXoG2EUcjsZIWuPeP6grMca6hMZQ7DMTjeG3PT+oEIzpTzKgCNUTajlCHo5Xk2stxR2R01Brh\\\\\\\/1vKLs7DTDeiLHGQ9NPU52GH5yhtGz7S9nEF4aQhyzWvVc5Fn0+xjRpjChhFe1bydqTk68QeXHw49EdRF5w4OOFQThKyD3eLEPDJUXRD\\\\\\\/75O65yHJXxIgRzl3AxDo+WFN6JTRAvXeIomqBVMHHfOR84PJx2tEQ6Yy3ZcyFY+telNV\\\\\\\/9DNflb73ayphww1CvW7MAkKVYUlYNs1\\\\\\\/1fcx29+eeKfEtnWbxOpHYlLNl\\\\\\\/uVSzmFIeSTm6cNE6cF7tADG1JRzWT7QdViT2+6AUfV+Mf8gGWlQvCq7Th3l3\\\\\\\/z5XVY0CvHrWI7xAvsIb+XPzaHUKbdC2LbdQEg\\\\\\\/xoXNKsfQmrDl+aqQ5CFVILNaGm5hnjmJloAjYMo3sd9mD9Wblrb9ijH0FKBeVGtYIEQsivVC9sKYe4tBXwYd3k4sUpYDQM81tuR4eZvSd8ym0DLf99Zpyf91fQ2vYw8BN7XvSzpMBdM8kgpv6GcffNe3Dy+Foc\\\\\\\/z9oDKPtN31JASUXdSYM\\\\\\\/+g6wDolhnJr5Y52qS4qM8yqZ4RJoZTHRNcOAcAB4+Tdlvrv7UejgXX9hZHPif3CThc8a+EC1Ltmn61jH+mOESMix8WiQjONnAQnpahqzRXe\\\\\\\/kfEEVhVHgHur7nxtzKd5JS0w1h5n+v15AMUvAPz8fx+Z6bc5Atb1lLCtFPVpoTdkK3q5AcRP\\\\\\\/vCvtl5RFBKpSoGQ47afEI4\\\\\\\/5gDXMkHLn\\\\\\\/wXdCXD+dk\\\\\\\/JmifVDiZMuhcobm2grys61lOcUAID7MO9VlVKMUS7uzVIdnUlb6QKCIv6DA9dLow6N1YRm3jrtyxO90KOfWG39weK2fo8BSm4tVl+SFJrtnJjxG+zZ+lHwBTf0TpszveI5fpjyxLSoVUYOw7JR7Rjixb0EoCYvPyeEyd\\\\\\\/xMqdgf0CxFZoJ1KFshPnyfoL6FQjUTApMrPasR8GCSho1g2H+mpJmKlyZaBXcaXwbQ9Zz2InbiNM10KHOtCauJ4pVBPWR1qv8yyFjiuSiwmvf8xjFFAELEIVB3LmQdfKfwmvQPbcAhwUZPm1\\\\\\\/MCbS2AbVoBz8E4Fsq+st6NngwrtflmpCOsz+qA9f4rMcsHP30mtKXJMYfmKfNa4jA9XOKEt4b9SPWRAbCMrbxocQXlM8wV2wAzCRYgRqCEnJxt9RUtCr38eD8ZF39ynxt+qvqijbXccxCzxGTm\\\\\\\/LSk0wFBOBHUu0L\\\\\\\/7QualXsSRE0iNv7dyVr+AEwh5t1lVJvyAiejXYhoj5HDzkvL+IkrOSO2vGtjpexXqWC2MQmNxvLAwS4nobJYUpjluN9wY0aob9nq5L2KnjB6BiGiiN+aQ480eZ84CcTLD+DrkKXbv4Cr9BfHCoxK35rD7DE3zCmwtLoL17tGPaPlocfGWiMCJQjTAtFXJu8QNX+AWWpHSv17jlGdwV5fKn+JG7rXa60fYqevf4yj4xsfUERtTAsYOP2UtreWCghhXys3RD8IeZv1z4ifYV2hhI6H1ZUW8Wel2ivoToRH8PEB6ifw\\\\\\\/gIv4fuiC1noqHH1Pqr5RmUJhGgsxju6q+OO2uY5wbU5vLjouXsgcI2kN2qWl2kNZQgIVQKqWSuyYuhgvp+dtwA3Rz5fnZv7+\\\\\\\/osGQafNChXpIp3wAYMaF4Hr+sCKbWFA90k9LD6J20\\\\\\\/c+8wM6o+ugyYAonCVUy54tiLxVvaAlnxufpEr+QHRA3XmdDNHp5sObLZAvU0j0QDFpd77Ba8+8kn0mjVZJlyCZz6ncs+Qi\\\\\\\/fop9JFi+wtWJzggzyuYz9uRbfqsHMhg05MFsChuWjElkfqhL\\\\\\\/cXK0CVRmQrJArgkNTgcFM5YvvAGgtA2YwpAD4GZW+m6ntfbObRdXvwqeDPhJJIo63zNLMtFNvcBvZRcvG331w9h9FXWOSeyCqauVaqmd9rHSR+eKf2otSCTu5sp354LoSMWf9nk8uniCAjokLBKLSZQcYqwGoisyxKvtwB+52uMNL8rKS5N2ySJT5FNrAFLiwjex0VlzHnSW2IJEI4535X+bZmvMHT7M3plzAGsGZe9uqJsCQlEN5K2Ozzb5WJ2lxab6fsg11im37Bie4N1cvrucNZqq1eHh2lnmc35cKkR3gtX+b+7mN8AZyw5+i3LVsy4E2Q8yHz+FWoPyyZZ8WNC+ftWytUO8Orxqw+d60zEK15+hV0amWymYXvDyMXIMIzsv350v6JyfqaLQZdsHSVIK1AbzIPNoKD7atQl9APOPrGMesjZzgsbit\\\\\\\/IRA+3OK7SRpwDbkzoA3fJ00O3khXnjRgfjQqyGqAkG2gqM+dKwR5UyXALxa0mqHylmSrTTy2V\\\\\\\/xsci50IJroNA5QtIuBl5b+1UTHJE\\\\\\\/grexIGcMxKPVOhaYFpzQ20AptY8U8XTOHiQiRo\\\\\\\/Jxklgu7RUpCj+G80gCWc7pGnnlkNRQ41fQe2e8AB83Grlt7LFEcXAz8JpEOsBJRWsdSwcZcqJVrp6ITvee5rw69AM0ogX7op9+ti1T3I7w6modlC1keFjiChAKBvHtmQsmdVEUbf+YuIsRNE7d2t95GYSyBh1AFUGAvtOkd3jRVvKhVbMMKjL78dhBN1u5ZPUQOz4TPlVw7wJbQJQ4nBqw0ZyUSdXKz12fXxa2p2GRUuzvtHb6uiSnbg\\\\\\\/9SHi7fL+MlvAtuIwuGwk2cMULP7EYiYcMClOxvoyv9wvR8b\\\\\\\/t0kWeMYMGI92fXVXc5mUPcoA7EKfD3J9lNIWt7+eYbaEcyyZFdw4DFN4puOfBG98cZf\\\\\\\/0mddvmbccf4+fuje0sk5e\\\\\\\/uQnIN8pvKsUlejLAPIcIK5c8O1OiWm3Hf1eqk9dRVENBWdUNi3V4LSMOgCMEycQK4aCxfY02Ecx38FB2Fo6xIqmFGFpNVDcTu\\\\\\\/Ts5YCBkLzMGgvg7Kp4RwAAtzANCVpcjSiFy9g2U\\\\\\\/op5rwsPkLibI1oDU83G22S2tbGblbFvF\\\\\\\/vz+rNQ6\\\\\\\/HSkOhWcGEdWCklLc7mCvLlJ7G2M9XU1DmwDn\\\\\\\/B2DVZmdkbx+kGq2juM99zsss5o1sKESZuWVzHNbdsVuHlaJk+zqGoTsf7+7sVSAmlAx1ThrgJ0P9o9K4Dma5mpHBaaSpF68LxsoZQaaaGH0JDBgWLGjY17fX0IDpdXS55gpR7jG+58sDQV+9h2GWtnhChPX1lCIt71ihGOvAyiqR7cbEH\\\\\\\/Mv2xYZRi64j32qUJJHNbytVrxXn299w9d87hwqb3khb8D1TLSUOmWFWbPZIAGq6E1fVjvsnfUIci+UFkmui50VJ3nFQ3dnzGSzNFWr\\\\\\\/tNyYmQUr3+vmTLLkaS3pjyYz9CY2\\\\\\\/\\\\\\\/uCxIuL2461XgwuwDid78mbzdWijK9dXYc\\\\\\\/jvvGUF82FObLnfH33m3BTWh+7UZlu7GQXUkC18hXUDo3rBv\\\\\\\/3u2L2Eos6PW6I51LTqcOfkcMqZqtbBy4xz7NUawWBsgsVEKrZDjXS0rWWiqOj2z2mYEP\\\\\\\/JdEbBfXyqI\\\\\\\/7DXVhEo4YHmyKdBNCXZNBeZ2lf\\\\\\\/V\\\\\\\/EfxzEvwpZXz1hyTpAWaUpVM1kQ4fioI3ZX+VzoV1P91PSEad82dljNvae9oef+\\\\\\\/NDER3i8xRwR6whxy6hw5t96UGU3IgkXXeX1MMv9usy0ongOr034GwgLM8nn4GCfu7fcpXal9yoSs6Yq31FoqAB9ZCCRyWhUWgcBoEr0vqxT6CaxF8gv4rNu8+2JoLMWK\\\\\\\/If2+FS3coNvFzHR49NNu+AdIPYNJiGrjhfXWTvzGMBoh57LpKnUcW1tvzVFF01RYK8wq3eNY4diuRUqR0T12VDhq\\\\\\\/\\\\\\\/oZbbLSTuHhYpBiuwBkMzd26XujmpS0eYI74xbf6AIuD7Qmrhq\\\\\\\/6SHtsXwfitS3aRO3FwqUqebZ8gJTMWzCUCL+Jqh00K5Nj16pUA8yzAFORszPNe3eXTrBMnCaA4olXRNBvvh1XeoPooMizw4x\\\\\\\/evFm4b\\\\\\\/hCKugqYiO\\\\\\\/o4PVe51HPpkKYc3PqjPD2J9dvLD0Vo9iofIO3GdtwExEihS98bJVHMNxldCA+1+dpNZuSUd6Myd3RT42mZMHuN5iciGCbmjM5fry8hPaBFIOS8gV3B+\\\\\\\/lqx0YvYcjQMICsoj5O9ctArmySP\\\\\\\/nfUGSgaX3l7SmmSrsB3hYQSrm66wZqDDqTG9sJ4mUBE\\\\\\\/LuO+5BMdGH+dSObZiVIcUccLYQGa\\\\\\\/RdoMcLPviNjVnZO3rC+8hvAUYDw1gAw5qQs+PvVuTNdPYFKiXg2jgrHrxe0cSL3KA5WPXREUSuCLXivBvZX64ouC4hN7EK1ZEfyRjE5sy2KwTwMEAi8SZLiRAc1TDlsrlWAYuRU4LJF70oly\\\\\\\/FRJmQpAlB34RzVovOxjl5pxrjMGFlyiIiS6L7F8Sz\\\\\\\/1+JVPySr5oj2x3w1p9Qg94+gC+CmK5nTlPTaaeitXHSMoiBmVTw\\\\\\\/rJtXORqC7N1smLDt82LsjBp6WMJmSGbUKCK7oUxLkToMl8G4TEAM8gYHfGSnHxcFLpFGFTSkb9l8FienLtcPmxFDqyEaom0Ah2jI1pLUv+a3OnJmWIPhS\\\\\\\/lvxueWxAfxvDKAX5Hy+0SgjCPj6wtysUaCsn5LvAYR4SyEVT86UCn0vze8MuxScuinJmCDWiXRVxEgcbm45MhIiJgvwYdbnSK99fKEXrlUR86OGtA0fmSifqi6nBPVUpgLY+oAPCDbviWP4VaIjoejuShjLoct\\\\\\\/ww3SzjN4TjjmNMcQFlOuSQ48+g6PVlT7fKxHBjsWndpifXKw\\\\\\\/XmTKOyHFbglS+p0PAh+YVktdgoT3iQRqO0frv\\\\\\\/RQ1gI9Y6YtnszWzqQL72oNJtnQ1trx\\\\\\\/oT4SuO5\\\\\\\/pW7APBIy9CcKb7Ve9wdDvD2M0mPaC2U96jyQw70aMmbUAJdMlWV4NJQwekFqVVMmEYcr8oydlZRZPq3XKGEapGiTtXiJu6Vc1oJX0791lqUXtNqpu5suLVXWrmnZnBGMvY0LRwrIvlrbBuH2qms6F0QeSQCFMjESRGtiiMBihDgA8snBj72MdC3iT\\\\\\\/29TqvSnRfEpD9xMy7jB+g25Yi8yBQn7J\\\\\\\/96VQdRopfLlU8WgsPev3cg8ecA9FguoJsDjkR52LH6zC3rvAmX4LoO\\\\\\\/vNPwHGkY1pQOaubVKFwmfZ83SEuD6Mjn1veMjaHIxEiVDR0dEy\\\\\\\/eOepEul3P\\\\\\\/Y8Q3FwS\\\\\\\/hhJ0h510IeKj\\\\\\\/dGDOmAdnYf9LbmggxzsRGOdJFs2ptMR3AI6AxmUqfeUNI3WbLx8Cysge6WV44sBrsBhj\\\\\\\/yFQ1MN3p+CZC1FXYcLorDGIYPwrDcfaYcVP3PUoDxh37oqNVME8HEvVmMYNzwqXA9h7wXL4V7xuTU\\\\\\\/\\\\\\\/CQTVt5hcUkYcGw91hdHmZCJ7SQCm1P6WQSJBBHRoMYFtPVxiZqvX4iuRzrbMpqmDdnfLC4nUwyKusMS\\\\\\\/hgNHiCO\\\\\\\/ICHLFa9TpgxuNceWeCWS9MjChy6XIYp9chtsXf574z+kawwsvVR2G8nqeTo0PrCre5A7w5XJl07xT67gOiygrE8ej6ZlNmKsVkWnyarj+rl1U\\\\\\\/0gzBwiAIPl+edRmDBbBOu+Nt2976R6ERWwNgYPiaW+Ofaq6e0frPEj55hjcEY6N7a7Q\\\\\\\/LdCB1mdbDXdkIDK14ynhvoio6LcUJI0KWtGAupQ2n+KjGFNjbKNf8595K7IKJtmnGxSps5FllZir6NY\\\\\\\/CoKJZN2D+t4TvnKADGUY4blBtYsesPwsSPqtoeM9mfQ6RmXpiZ1gxoNjD6m1hVP0o0DQBpVMmz9hHyvZkP88I7iCQ5yh17pTFG+gHiQYs5Z1Z7Bi1MmgNRVKv16Ho8jZnn6FYsj0JFQPh9v6H89Qe\\\\\\\/wCGS0Q8ELIht1Sh2pAswOqh2P08IsHDqWnnKgabSHjb9GA8+HnevRmydlDWs2HicwyhQ656UUR0KJUhwbXgeINTf5fXP3oCUA4yxhtz1dJ4Zj8065L8aP53hcsKqIbtVsPizdgdAygqBEqrYr1e8aJ049tbI7+Js+bi8BUSkDtu6bU2XgIuPj4rAMr2Mj5dgyisyuID9XOYcoZ0pzpggXzXEWgAl7N\\\\\\\/DiQ6hmoSZTbX6HHXRubV8AuQMVquyAn91t+ETgqvOQXJZhA6Vz1lNDNmwkNaCN\\\\\\\/nyub2V8KVasEqWAjxmTXHgt6svNx1vlTgE0DSMCH\\\\\\\/Vm0unlITUi1f4op5WT+DsDfFLbP7cRdMBusxiited1wzUru++9EEVBjdRnbhI9+j\\\\\\\/5N8tFm0GaYssxJP6\\\\\\\/CepXYYOBgBRWdwm32+VwZ6DaJIIEGsR6aNBDmMefqAgiMVpnFi\\\\\\\/NTPctXq+dJajAKX2Wjd2Ct6+Gld\\\\\\\/AJ1wRzto9XCDS3jpXg\\\\\\\/iqvdTI\\\\\\\/g2b91+37a7Qbn1xWOv\\\\\\\/R6of1scy3jNP+pXL\\\\\\\/UTSIYWGgzFM0668hzq2GjlH1UuduY0bhxnHrlBAblctrOm\\\\\\\/rQ9ExIEPPEIv1VHhXJZJUxtuteHNtpgXyVEsIm5PBwK4sSETKgphZZm31CgWmQP3Tm6bdiMuZ8Wm8P9uC4cwDkMoNXmWUqttuDkAxu6Q+4nsRrWXeRyilvQk+5Ia0VR9Mp89v56n1qRH6mHHNA4++O1VcZLquh5DGx3OrUvOo4dA8Vz0EEWAMbKR7UidqfjjpRGfzYwURucgH6pALOQ492AsqvsYzmVIKCxx7jodvjRF5zD\\\\\\\/NpykPvNeuQvL8ZncYjqCoF2bGtv3rwlBjvXSEkCzYmCm3XwfiCcSu1WNCkM1S5CZeYEV8s\\\\\\\/QyJ8EcmCKjRbRgE9Yeg7XIWTx3gGLtqlVNLO\\\\\\\/93SLMnzkOYxMpjdOuUgGneEKCNk1BAFugmiMfsnLjOVYNmQx2zPx3QQ8VFCofbni9xTiDxm0KMOrVOtMrXg8KfBe9I4huZOCUAnUA0gxFk30iiCk6WZQEzDEad0anKZ+l\\\\\\\/E1l+y507kuj0xOFxtMBaB7XutWPuGDxjHgcxFvjllVI0aEYWprobM\\\\\\\/G9wNFa2lSEVKR7AFjA73s62pYKk9+bdSei6K0xIK9GGPWncY3nE0Rtc14m7NHjjurR74hQ0dm2jfETbeCaVRc+kaE86RWIg\\\\\\\/dMKgScZZ2v\\\\\\\/pdVfUdrWegrbxXvwKOYa5nSXK+HnYaFdgX4pfXs\\\\\\\/gpbvM1aGHLAwZVRIU7sNo7YWTudfhorLatW7BjiIPzD5w7TNZf5VnlB9sLEd+r3AemKP539x3PpfNj2AdXJbXMJ5FGO70anPLXTA3cPeH5650K9zjmShVRZT\\\\\\\/cASewo1JHXmAM3prw+Md0\\\\\\\/qosFkZ5rO3gjd6DdgsfuFimRINx+Yqzu2wtrc\\\\\\\/Dn5JJbrKj15TiaAdwyFujHDWhxTTYExEil1u4WIc7Fw5VC+SMwFFroKnLJy77HJfR9XaO8TyPz8+2+v8aj7IcdbMCcrMwxM9+RgfJ0CJhoRiEsYDMVMmdkdtTFPjufW0NqYsvInGXZhjtt8eGoOHc+\\\\\\\/YUiR1Jej6OzYdmcFZJYLOQNAthiCw\\\\\\\/0GcH\\\\\\\/L3CwjwhphiKk\\\\\\\/D4eIWPT1yGT1SkjOt1GI2h3z+QcyDeKDmCscV+4Re8seijbzZS0818TUS6F4CJIyblfrdbufFHvbj9ZNXD2p6rjuk5Ey+tlzcMmh\\\\\\\/OhplDUfXV5LKtZSeXm8NCUmhF3cQfKFE0Hf4qZ6FM7oCT8wkMMdVbCXxvLJGAnYqQPXaEk\\\\\\\/VWEqOmqW0wC3SfDme385ksQM4w85cmGWi5SD\\\\\\\/H7fO2tsUttuNy4U8FHNZrD33T+5c5\\\\\\\/fFUy7L46N5IQCuU1PuPhDIEUt62TOG0mWnbWipzvdJWjk2edXRxPnjzb1z9N2MD1eA6MffZp7xPFxBybfuAqJ+ebEEvFoPpQdn3vCG2IeRIeYZpweaILRalrutL8xc1RlazBaqRH7V+DX4KqwxKv\\\\\\\/FVVzCPiPGNoddthsRfftX46rHsxLCCNAEEJGx1uKRe2ffS5exee+VqkOEogOVFovp7EZquvLLmexUFSn7RGoZlPy2fRIXqcWvL+BESyZryTOT2Ec\\\\\\\/DY+YcavjJ8x9JOAWDrIKSkxjnvaQWHZ8fvrdEn9boCT3ByXvFcBmbbCOabk46KO\\\\\\\/ie4xR9iAO8n2GuDHqtlm8vkCNQKgnSsWcALk+qys8htfl2gaaNDvtExO3v\\\\\\\/cxkNj2sYIEsrJBmOdiQ4rqlsF7bKZdnb\\\\\\\/1dR8Ke7\\\\\\\/JUIeKLu\\\\\\\/CCVdx6R9xAbZcslpi0+pWI47uG9hmDiAI48AxBSGj6o8W0n\\\\\\\/iahhA7HjlhbkJOzTtqflvI6xPEYS+2MwXyVOWwUOgHZS91b1MCzI9zoi2c8KGp324WC2A3y4mdEm0FESRqIs4IcJxsE5rdOLS0o0hm0ZbrGICqJKwhi0nsOSZq5E4Vwvgsp3VruKPcdTNO3uIu4fO3aaYy09SNJSDX5HzW\\\\\\\/KSaTi0jJDRQokDFVZwBTwDtfNbkwZbVxfThx2bAg\\\\\\\/sBD76b4zUw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022O3ukgEOAlt4AAUdMNuL1vw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f18121bcd5d5f9ee8a5e66f7e8593b9df2fbef45cbda64d5694c8fbd395a57fb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kVGwM5Vc\\\\\\\/TYup36q\\\\\\\/BkR4U40wGVQJWrroyPrOQ\\\\\\\/5xnKE3uBVY0ZwTenrtAKCw6szJFWoovpPZTengX3SZFHgAplRar+XZbv8EzUn7uByhUnZsXnO0Ht08ucMyUCUD2rFAB6Q5cpZ9vrc7kgWK\\\\\\\/+JF8sJoUoRbUqhFnLQsnMthWQVPa9xdYhXRyUoJwXEuRQo6Kf5VUZ0Fl1ZRGtUyZBSw57IoL5D80zmIBUnTiB0+bBPclK7+eM3oXGV5hNoHQhKKWYZYncxEuehsCJ4SmPDRQaAG5ta+4WqW0rRKl0WHbTWHwmLc17zCvxp4YVfqxmZdG0GtqV1HdmHOjp9BdFuI+o\\\\\\\/J\\\\\\\/wBnBOZycYrBhFH8jLWjUGhBrH\\\\\\\/5hXG5tozEC+cmpFcix3yQVyLtKwRdAtyGYfgv7VBoU0HBcrXWH7vnlmv6f1kOLxylL3SPAzq741cPFv3x1ByMyT+gCOHQTt0UF5FnyzmxkW\\\\\\\/iNUa3QXN9gerhPZ8EWfRmDKa\\\\\\\/Fj1GPA7L+cdmWuWUe6YJj0+5h4zBqMS1u5mbIGjMqzNO3rpc2RG4Guyc9Xw4ENOINfToh6DKUo+lqSQK3sCEQU+cglEkIAoezgoyqT5qzUbje32Us7FaHzDPcxJRmkCK3m4epeEmWD9jO05SHCO7myoNrbc02CYfBTmyPGZz3xK2gyOc9S8v8DJaKSKwzZo5DWE9ezon6Rom\\\\\\\/NPc3kcjgP5mW+tO8J\\\\\\\/xhbI9ChUgI2tzDgEtfC\\\\\\\/CcGevZsX3temYHohC5Qbq7+hV9oS7Q+qoRb5\\\\\\\/J8+GYCohk8x4MqPlXmqlaP78wqx9dmFMLRJr3S2pRf0WMBvJRVE8JKPdhARb6u5qgSjcchEavRUq\\\\\\\/k5yqs1qRycQp3aZxsp2Mr9uM\\\\\\\/eD9G1KwOYCsvTRuRKXDY+bK6UZNvT+bryhKYe3T9U\\\\\\\/RbOOgvjBJgBXEU3rI077vBFszQPA+f6pT0Z+t0dBbGCjormy4Fq3PNLCupYXF9B2dOsHrrV5RfyewYj2kDqLKm2gUlbaPfJ2n3YZkPHcQNWG8vwI5nJvgs\\\\\\\/O\\\\\\\/LoL\\\\\\\/+9uxVePAD0VWmzg2flOvOPJOo0SNjRPb1mYgBz1pG5ebVwL62iOyyfAEcn7VAPJhPNqZ2IsMnWLtcDy6Zkv4hyswnd0NZ+0VTk1hq5mV2zybz8HYD9lGfwpIL5SAq2F+HvOhlnGYPODTTZ8mX0JLgzLmkknh59oN1JU962dnok2aZUSffvMx6LkWGGFkjX4IIt1qIiQNGsQzI9HMZbk9dvCpoK8axwlpYw5SEelXCWJexhJi\\\\\\\/gfZLmlxhQAGJ+F85wHV4icXI6pyOkBph+BbnpR6q4ONv79nDs7ej\\\\\\\/2TwJr+\\\\\\\/Og+LYoElp8yrnz7dmHISRT\\\\\\\/K2TIfUtnETGvAiKxhbFvUXWz0YxETJIAlkSWYLUVVqwad0KgOxtN\\\\\\\/X\\\\\\\/SRX2HqU6u9JzVmqdQlqW+RPNNJXSYuWQxYAFzrq16Ch9RRUgsVUCxD6tbTnJwPp0KDOCXkem2du7w4EES+Vjv7eQ9wg57gji3f6\\\\\\\/Y2sHdDM1nKT3y26vCkyrxqca\\\\\\\/ivsIpTtGbfioG66yVb2YIGFnHh3w41sex397tPpi\\\\\\\/huiDFrqjRkCu9v6bP2jl3ODQlvFKfC33e9QQB6WyyeVqGoZb2M3C9jun6SPsYeVQcQyStXVTnskWlViyBKh1MVTl\\\\\\\/Qu\\\\\\\/RXE6xDhS9Dz3urZRkkFsJxuizx+5buI9LsZDQolCsYqiHhMSF2o9jgegcQtD+KEcRgzbcIOCI3EGrhvdRtKFdNXyJTfN1UsVd6bdmh6xA5WubTY2Sl3XhDtP8IYGQugOSIg6+c6MH9na9302Sm0GSCW2TsqDtNmbJAYGBr0FAnmfLw5H5z\\\\\\\/efp1Sm1jmTSWO0aE+7UF6lHNsOw4J58kE7bIx7F4bmQKOkgfFi+0Ppr54Cxe7pvXoeM4lTLvr2zYFtlM+L9IKzoTEiuQ9sDXnken9Ho4ztWgDmzjhYIPSRrC75LJYxlWSvTcI5ew==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EGTXLfDQcQ5FE2S9nCOueA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f9c93681bf67bc5b8826b1364766af9428a6a3ba193fe9458daed5c9c11a46d0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459832 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/669" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Rin1JK-pt5dAT6XLywQBzdNR0VTb74B0zV0TfqWDA-M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kSUfgkfhwCRt9j0I-_8YNJ1Rf1Snxpmh7LffIx45YG0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/657" title="EMS Mutagenesis in the Pea Aphid Acyrthosiphon pisum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/681" title="Evidence that Natural Selection on Codon Usage in Drosophila pseudoobscura Varies Across Codons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459832" id="node459832" data-pisa="ggg;4/4/669" data-pisa-master="ggg;g3.114.010264" data-apath="/ggg/4/4/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew W. Hahn</span>, <span class="highwire-citation-author" data-delta="1">Simo V. Zhang</span> and <span class="highwire-citation-author" data-delta="2">Leonie C. Moyle</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-679;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010264 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459832" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew W. Hahn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BHahn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hahn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BW.%2BHahn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mwh@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">mwh@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Simo V. Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simo%2BV.%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimo%2BV.%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Leonie C. Moyle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonie%2BC.%2BMoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moyle%20LC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonie%2BC.%2BMoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/4/669" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459832" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/669.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459832" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/669.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459832" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/669.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459832" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/4/669.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459832" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Current <em>de novo</em> whole-genome sequencing approaches often are inadequate for organisms lacking substantial preexisting genetic data. Problems with these methods are manifest as: large numbers of scaffolds that are not ordered within chromosomes or assigned to individual chromosomes, misassembly of allelic sequences as separate loci when the individual(s) being sequenced are heterozygous, and the collapse of recently duplicated sequences into a single locus, regardless of levels of heterozygosity. Here we propose a new approach for producing <em>de novo</em> whole-genome sequences—which we call recombinant population genome construction—that solves many of the problems encountered in standard genome assembly and that can be applied in model and nonmodel organisms. Our approach takes advantage of next-generation sequencing technologies to simultaneously barcode and sequence a large number of individuals from a recombinant population. The sequences of all recombinants can be combined to create an initial <em>de novo</em> assembly, followed by the use of individual recombinant genotypes to correct assembly splitting/collapsing and to order and orient scaffolds within linkage groups. Recombinant population genome construction can rapidly accelerate the transformation of nonmodel species into genome-enabled systems by simultaneously producing a high-quality genome assembly and providing genomic tools (<em>e.g.</em>, high-confidence single-nucleotide polymorphisms) for immediate applications. In populations segregating for important functional traits, this approach also enables simultaneous mapping of quantitative trait loci. We demonstrate our method using simulated Illumina data from a recombinant population of <em>Caenorhabditis elegans</em> and show that the method can produce a high-fidelity, high-quality genome assembly for both parents of the cross.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome" class="hw-term hw-article-keyword hw-article-keyword-genome" rel="nofollow">genome</a></li><li class="kwd"><a href="/keyword/genetics" class="hw-term hw-article-keyword hw-article-keyword-genetics" rel="nofollow">genetics</a></li><li class="kwd"><a href="/keyword/assembly" class="hw-term hw-article-keyword hw-article-keyword-assembly" rel="nofollow">assembly</a></li><li class="kwd"><a href="/keyword/next-generation-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next-generation sequencing</a></li><li class="kwd"><a href="/keyword/duplication" class="hw-term hw-article-keyword hw-article-keyword-duplication" rel="nofollow">duplication</a></li></ul><p id="p-4">Whole-genome sequence information has transformed research in traditional model systems, and now next-generation sequencing (NGS) technology is at a turning point in its application to nonmodel organisms (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tautz <em>et al.</em> 2011</a>). Researchers working in data-rich, but sequence-poor, organismal systems are eager to apply these technologies to their own species. However, the challenges involved in generating useful, high-quality, whole-genome sequences in these naïve systems are substantial. Nonmodel systems can have biologic features—enforced outbreeding, absent or sparse linkage maps, complex genome structures, large numbers of repetitive elements, etc.—that directly hamper <em>de novo</em> assembly and therefore the general availability of genomic tools.</p><p id="p-5">Of the challenges inherent to generating a <em>de novo</em> genome sequence from nonmodel species, the generation time and the reproductive biology of many of these species impose especially important constraints on making accurate assemblies. The inability to make highly inbred individuals and/or to use a single individual as a reference genome frequently means that there will be a large amount of variation among the assembled alleles (<em>i.e.</em>, haplotypes). This variation (present as heterozygosity in a single individual) is challenging for standard assembly approaches because it is difficult to distinguish allelic sequences from truly paralogous loci that have low divergence between them. One consequence of these errors is large-scale, systematic bias in assembly and annotation. The problem is especially severe for NGS technologies because of the short nature of the reads (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alkan <em>et al.</em> 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schatz <em>et al.</em> 2010</a>). The result of such “allelic splitting” is to increase the total size of initial genome assemblies (<em>e.g.</em>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Barrière <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cheung <em>et al.</em> 2003</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Holt <em>et al.</em> 2002</a>). Even in genomes assembled from highly inbred, but not completely homozygous, individuals, large numbers of split alleles will falsely inflate the apparent number of new gene duplicates (<em>e.g.</em>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Colbourne <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Denoeud <em>et al.</em> 2010</a>). One approach to dealing with the problem of allelic variation is to allow greater numbers of mismatches between the sequences combined together into a single locus during the assembly process (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dehal <em>et al.</em> 2002</a>). Unfortunately, the effect of this approach is to make it more likely that true paralogs will be collapsed into a single locus, causing an underestimation of the number of recent duplication events and erroneously decreasing the apparent size of the genome. The collapse of highly similar paralogous loci can occur during assembly regardless of levels of heterozygosity (<em>e.g.</em>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lander <em>et al.</em> 2001</a>) but is an especially acute problem when assemblies must also account for high levels of allelic variation. Therefore, an overall challenge of any <em>de novo</em> genome sequencing project is to find the right balance between the splitting and collapsing of loci and to apply appropriate <em>post hoc</em> methods that can be used to identify such errors (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bailey <em>et al.</em> 2002</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kelley and Salzberg 2010</a>). This challenge is likely to be particularly acute in emerging nonmodel systems.</p><p id="p-6">The availability of completely inbred (or haploid) individuals does not solve all of the problems of either genome assembly or genome-enabled science. One irony of genomics in model organisms is that although the use of nearly homozygous strains has allowed researchers to generate highly accurate reference genomes, these sequences contained no data on within-species variation (<em>e.g.</em>, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2000</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14"><em>C. elegans</em> Sequencing Consortium 1998</a>). Because variation data—often in the form of single nucleotide polymorphisms (SNPs)—are necessary for genetic mapping, studies of population structure, and many other purposes, this has left these systems at a disadvantage relative to nonmodel organisms in which SNPs were identified during the genome sequencing process. Obtaining SNPs in model organisms has required further sequencing efforts (<em>e.g.</em>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wicks <em>et al.</em> 2001</a>), some of which did not come about for nearly a decade after the initial genome release (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Langley <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sackton <em>et al.</em> 2009</a>). Moreover, even when SNPs are present in the individual sequenced to make the reference genome, the haplotypic phase among alleles is still not usually known, except in exceptional circumstances when multiple SNPs are contained within single sequence reads or paired reads (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bansal <em>et al.</em> 2008</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kim <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kitzman <em>et al.</em> 2011</a>). That is, the arrangement of alleles on an individual’s maternal and paternal chromosomes is not known in most current genome assemblies.</p><p id="p-7">Finally, even when a reasonable assembly of sequence reads can be performed, standard genome sequencing is unable to place scaffolds onto linkage groups without substantial additional investment in approaches such as fluorescent <em>in situ</em> hybridization, genetic mapping (<em>e.g.</em>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hyten <em>et al.</em> 2010</a>), optical mapping (<em>e.g.</em>, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhou <em>et al.</em> 2009</a>), or, when possible, radiation hybrid mapping (<em>e.g.</em>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lander <em>et al.</em> 2001</a>). Without the ability to place sequences on a physical map, genome sequences contain little information about synteny, the colocalization of genes with quantitative trait loci (QTL), the content of any sex chromosomes, or the presence of inversions (see <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lewin <em>et al.</em> 2009</a> for additional shortcomings). In addition, the distance between duplicated sequences cannot be estimated, and therefore no inferences about duplication mechanisms or the effects of physical distance on either the probability of interlocus gene conversion (<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Casola <em>et al.</em> 2010</a>) or adaptive evolution (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Han <em>et al.</em> 2009</a>) can be made.</p><p id="p-8">Here we propose a new method for producing whole-genome sequences that solves many of these problems. Our method involves sequencing a population of recombinant individuals from a known crossing design, and we therefore refer to it as recombinant population genome construction (RPGC). The basic idea of RPGC is to barcode and sequence (using the most up-to-date NGS technologies) a recombinant set of individuals from a known crossing design to generate the reads used for genome assembly. This set of total reads can be used together for the initial <em>de novo</em> genome assembly, but each individual’s genotype also can be determined by examining reads with the appropriate barcode. The availability of individual recombinant genotypes makes it possible to address many problems in genome construction, including the assignment and ordering of scaffolds to linkage groups, the ability to distinguish alleles from paralogs, and the identification of a high-quality set of markers. In addition, our approach allows researchers to infer the phase of whole chromosomes in the parents of the cross and to map the location of loci underlying any phenotypes segregating in the cross. Herein we describe the method in greater detail and present the analysis of an idealized simulated dataset that demonstrates the power of the method. As we discuss, because RPGC is flexible in its application to a wide range of organismal systems, its primary anticipated application is to whole-genome sequencing and assembly in traditionally nonmodel organisms.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> provides an overview of the steps involved in RPGC. By iteratively assembling reads, calling genotypes, and resolving assembly inconsistencies (<em>i.e.</em>, alleles that have been split into multiple loci, or multiple loci that have been collapsed into one), RPGC combines multiple genome construction steps—each of which has been achieved on its own—into one sequencing approach. This integrated approach not only saves money and time but also promises to provide a much higher quality finished product. Here we outline the proposed steps using the case of two inbred parental strains that are crossed together, with the F<sub>1</sub> self-fertilized to generate F<sub>2</sub>s, although this approach can be used in a much wider range of offspring population structures and therefore biological situations (further addressed in the <em>Discussion</em>). Throughout, we identify instances in which crossing-design considerations will influence the difficulty of individual steps. Although the method we describe is not tied to any specific NGS technology, we will generally refer to the constraints of the Illumina HiSeq platform because of its relative ubiquity and low cost. In the <em>Results</em>, we carry out the steps described here on a simulated dataset by using a recombinant inbred line (RIL) mapping population.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/669/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="General outline of the steps involved in recombinant population genome construction. Each individually numbered step is described in detail in the text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1987982476" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;General outline of the steps involved in recombinant population genome construction. Each individually numbered step is described in detail in the text.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/669/F1.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/4/669/F1.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/669/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/669/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464237" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">General outline of the steps involved in recombinant population genome construction. Each individually numbered step is described in detail in the text.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Step 1</h3><p id="p-11">The method starts with the sequencing of the F<sub>2</sub> population (1a) and the parents of the cross (1b). As with all next-generation genome assembly projects, at least two classes of sequences, with different insert lengths, should be used to generate long scaffolds (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gnerre <em>et al.</em> 2011</a>). This combined approach ensures adequate depth of sequence coverage (using cheaper, short-insert technologies) to make contigs, in addition to smaller numbers of long reads (using more expensive long-insert or long-read technologies) to enable assembly of these contigs into scaffolds. For RPGC, given the relatively cheaper library costs for short-insert (paired-end) Illumina sequencing, it is most cost-effective to sequence the recombinant population (step 1a) using this (or equivalent) short-read technology. With the current sequencing power of HiSeq, a large number of recombinants can be barcoded and sequenced within a single flowcell lane, and a few lanes in total might be sufficient for a genome project. The number of F<sub>2</sub>s that must be sequenced will differ depending on the crossing design and a number of other factors (as discussed herein in <em>Methodological trade-offs</em>). In contrast, because of the greater costs of individual long-insert libraries and long-insert (<em>e.g.</em>, Illumina mate-pair) or long-read (<em>e.g.</em>, PacBio) sequencing technologies, these methods should likely only be used to sequence the parents of the crosses (step 1b) to increase scaffold lengths. These recommendations are obviously subject to modification with evolving technologies and costs. In some outbred experimental designs, it also may be advisable to sequence F<sub>1</sub>s rather than (or in addition to) parents in step 1b, because only a subset of parental alleles will be present in the F<sub>2</sub> generation.</p></div><div id="sec-3" class="subsection"><h3>Step 2</h3><p id="p-12">Using data from steps 1a and 1b, we can generate an initial genome assembly and identify SNPs. The basic steps of the initial genome assembly under RPGC mirror those taken for a standard sequencing project. The most up-to-date assembly programs (<em>e.g.</em>, ALLPATHS-LG; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gnerre <em>et al.</em> 2011</a>) can be used to jointly assemble the paired-end and long-insert/long-read sequences. One important aspect of assembly quality is total sequencing coverage. Total coverage in RPGC is equal to (# F<sub>2</sub>s)*(coverage/F<sub>2</sub>), so that 5× coverage for each of 96 F<sub>2</sub>s would give 480× total coverage for the genome assembly. Adding in the long-insert/long-read sequencing of the parents increases overall coverage and ensures long contigs and scaffolds. One consideration with standard next-gen assembly software is that too much coverage can actually harm assembly quality, that is, extremely high coverage can introduce a non-negligible number of errors into assemblies and assembly quality can suffer, or at least not continue to improve (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Haridas <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lin <em>et al.</em> 2011</a>). For this reason, <em>de novo</em> assembly might use all of the long-insert/long-read data plus only a subsample of the paired-end reads from F<sub>2</sub>s. The quality of the initial genome assembly will depend not only on the number and length of reads but also on the number of variable sites between the parents (higher variant density likely means worse initial assembly). Although this finding suggests that the optimal choice for this step is to minimize variation in the sequenced population, downstream assembly, correction, and mapping steps actually benefit from intermediate levels of variation between parents of the recombinant population (as we discuss herein).</p><p id="p-13">The second major analysis to be conducted in step 2 is to identify SNPs (or any other variation—we will refer only to SNPs for simplicity) from the total pool of sequences used for assembly. In an F<sub>2</sub> population constructed from inbred parents, we expect informative variable sites to be at frequency 50%. For other crossing designs, and for sex-specific and sex-linked regions, expected allele frequencies come from equivalent Mendelian predictions. In crosses involving species with sex chromosomes, ensuring that half the recombinants come from each sex will make it easier to identify sex-linked scaffolds. In all crosses, the deep sequencing coverage of the initial assembly can be used to identify sites that have only two alleles present, with each allele represented by multiple reads with high-quality scores. One can also restrict analyses to variable sites with both alleles represented by 50% of the reads (± allowable error), to ensure informative markers.</p></div><div id="sec-4" class="subsection"><h3>Step 3</h3><p id="p-14">Using the SNPs identified in earlier steps, we can begin to assign genotypes and to identify/differentiate allelic from paralogous variation. After finding variable positions, each individual F<sub>2</sub> must be assigned a genotype (homozygote for either allele or heterozygote) at each site (step 3a). The accurate assignment of genotypes for each variable site will largely be determined by the read coverage at that site within each recombinant. This step can also take advantage of software that takes into account the prior knowledge that a site is variable when calling genotypes (<em>e.g.</em>, GATK; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McKenna <em>et al.</em> 2010</a>); this ensures more accurate genotype calls. Either in this step or in later, updated, genotyping steps, multiple variable sites within a single scaffold can be combined to infer more accurate genotypes.</p><p id="p-15">Simultaneously, RPGC also aims to identify all possible allele-splitting events and paralog-collapsing events for later testing (step 3b). To find the possible split alleles, one must first find all highly similar paralogous loci in the initial assembly. The most straightforward way to do this is the whole-genome assembly comparison (WGAC) approach (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bailey <em>et al.</em> 2001</a>). This method identifies all pairs or larger sets of windows with high sequence similarity (&gt;90%); these are set aside for downstream testing to evaluate whether they are truly split alleles. One can also use the read-depth at loci throughout the genome to help identify split alleles and collapsed paralogs, an approach taken by the widely used whole-genome shotgun sequence detection (WSSD) method (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bailey <em>et al.</em> 2002</a>). After mapping all reads against the initial assembly, split alleles should show 50% of the average read-depth if a single locus is split into two loci; percentages can be lower than 50% if more than two alleles are present in the cross and are therefore split into more than two loci. Conversely, two collapsed paralogs should show 200% of the average read-depth, because all of the reads from two physical loci will be mapped to one locus in the assembly. More than two paralogs can also be collapsed, with commensurately higher read-depth.</p></div><div id="sec-5" class="subsection"><h3>Step 4</h3><p id="p-16">Using information about the segregation of variants in the recombinant population, we can then use RPGC to refine the identification of alleles <em>vs.</em> paralogs. Given an initial <em>de novo</em> assembly and initial genotypes for each recombinant individual at all variable sites, we can determine whether variants segregate as a single locus or multiple loci. This step allows us to get “true” loci that can be used to both improve the assembly (step 5) and to construct a linkage map (step 6). To distinguish alleles from paralogs, one can use genotypic proportions among the F<sub>2</sub>s to detect deviations from expected values. For example, for inbred parents there are straightforward genotypic expectations for both split and collapsed loci. For two or more duplicates that have been collapsed (assuming there is a fixed difference between them), all of the F<sub>2</sub>s will appear to be heterozygous at positions that distinguish the paralogous loci (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). This pattern is not expected—and can easily be rejected statistically—with even a modest number of F<sub>2</sub>s. Conversely, for one locus with alleles that have been split into two loci, we expect each of the two loci to segregate as presence/absence in a 1:2:1 ratio (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This pattern will never be seen for two loci that exist in a physical genome, even if there are copy-number variants segregating in the cross. The “presence” and “absence” of these loci will only manifest itself at sites that differ between true alleles (<em>i.e.</em>, sites that are truly heterozygous), so these distinguishing sites must be determined beforehand in order to quantify segregation patterns. For both collapsed and split loci we expect read-depth to be high or low, respectively, as described in the previous step; this information can be used to bolster confidence in the inference of split or collapsed loci.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/669/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The pattern of segregation of collapsed duplicates through an F2 cross. The top panel shows the physical reality of genes arranged on chromosomes, with two duplicates present in each parent, F1, and F2. The bottom panel shows how, when duplicates are collapsed into a single locus, all individuals appear to be heterozygous at sites that differentiate the two copies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1987982476" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The pattern of segregation of collapsed duplicates through an F&lt;sub&gt;2&lt;/sub&gt; cross. The top panel shows the physical reality of genes arranged on chromosomes, with two duplicates present in each parent, F&lt;sub&gt;1&lt;/sub&gt;, and F&lt;sub&gt;2&lt;/sub&gt;. The bottom panel shows how, when duplicates are collapsed into a single locus, all individuals appear to be heterozygous at sites that differentiate the two copies.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/669/F2.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/4/669/F2.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/669/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/669/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464121" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">The pattern of segregation of collapsed duplicates through an F<sub>2</sub> cross. The top panel shows the physical reality of genes arranged on chromosomes, with two duplicates present in each parent, F<sub>1</sub>, and F<sub>2</sub>. The bottom panel shows how, when duplicates are collapsed into a single locus, all individuals appear to be heterozygous at sites that differentiate the two copies.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/669/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The pattern of segregation of split alleles through an F2 cross. The top panel shows the physical reality of a single gene that differs in allelic sequence between the parents. The F1 and half the F2s are heterozygous. The bottom panel shows how, when alleles are split into two loci, each parent appears to be missing a locus, whereas the F1 and half the F2s have both loci present." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1987982476" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The pattern of segregation of split alleles through an F&lt;sub&gt;2&lt;/sub&gt; cross. The top panel shows the physical reality of a single gene that differs in allelic sequence between the parents. The F&lt;sub&gt;1&lt;/sub&gt; and half the F&lt;sub&gt;2&lt;/sub&gt;s are heterozygous. The bottom panel shows how, when alleles are split into two loci, each parent appears to be missing a locus, whereas the F&lt;sub&gt;1&lt;/sub&gt; and half the F&lt;sub&gt;2&lt;/sub&gt;s have both loci present.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/669/F3.medium.gif" width="440" height="322"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/4/669/F3.medium.gif" width="440" height="322"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/669/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/669/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464256" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-18" class="first-child">The pattern of segregation of split alleles through an F<sub>2</sub> cross. The top panel shows the physical reality of a single gene that differs in allelic sequence between the parents. The F<sub>1</sub> and half the F<sub>2</sub>s are heterozygous. The bottom panel shows how, when alleles are split into two loci, each parent appears to be missing a locus, whereas the F<sub>1</sub> and half the F<sub>2</sub>s have both loci present.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Step 5</h3><p id="p-19">Once split alleles and collapsed paralogs have been identified, one can use these data to iteratively improve the assembly and update genotype calls at each locus. Assembly software makes several typical decisions regarding the placement of split alleles—it generally places them in tandem to one another (<em>e.g.</em>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Holt <em>et al.</em> 2002</a>) or it places one on a miniscaffold by itself (<em>e.g.</em>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Colbourne <em>et al.</em> 2011</a>). In both of these cases either the two tandem loci can be collapsed into one in the updated assembly, or the copy on a mini-scaffold can be removed from the assembly altogether. If the two loci have been assembled onto equally supported larger scaffolds, one can use the genotype information contained within both loci together to determine the true location during the linkage-mapping step. That is, given the pattern of segregation shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, one can infer the true genotypes for each individual recombinant at the single-locus that has been split, and these genotypes can be used to place the locus in the larger assembly. For separate loci collapsed in the initial assembly, further local assembly may be needed to find paired-end information that reveals the actual location of the collapsed locus or loci. These correction steps should be carried out on all the problematic loci identified in step 4. Given an updated assembly the genotypes at all variable sites can also be updated, in all recombinants and parents.</p></div><div id="sec-7" class="subsection"><h3>Step 6</h3><p id="p-20">With an updated assembly and updated genotypes, scaffolds can be placed onto a linkage group—and ordered and oriented within linkage groups—with the use of standard map construction methods. Only scaffolds with variable sites can be mapped, but increased scaffold lengths due to high coverage and long-insert/long-read sequencing will ensure that most will contain at least one such site. As an additional potential bonus of the linkage mapping step, it should be possible to reconstruct whole parental haplotypes. For crosses between two inbred strains, this means that one should be able to reconstruct two entirely separate genomes—one for each parent. Finally, with the fully genotyped set of recombinant individuals and parents, one can easily QTL map any phenotypes that have been scored.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-21">To demonstrate the power of RPGC, we simulated Illumina sequence reads from an RIL crossing design using the <em>Caenorhabditis elegans</em> genome as a template. This simulation represents a highly idealized dataset because RILs simplify genotyping expectations (they should always be homozygous at every locus) and because the <em>C. elegans</em> genome is quite small (~100 Mb) and individuals can self-fertilize. Nevertheless, this dataset allows us to demonstrate the methods outlined previously while presenting a best-case scenario for the success of RPGC. Throughout the pipeline described herein, we have had to make decisions about acceptable minimum quality scores for calling variants, as well as many other important assembly and mapping parameters. Although these do not necessarily represent the optimal parameter choices for each particular step, they are typical choices used by researchers in the field. Similarly, we have taken advantage of software that is commonly used in the field, although RPGC is not tied to specific software packages or algorithms. A manual detailing the software and specific commands used at each step is available in the supporting materials (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010264/-/DC1">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010264/-/DC1/FileS1.pdf">File S1</a>); all of the scripts written specifically for RPGC are also available from our website at <a href="http://sites.bio.indiana.edu/~hahnlab/RPGC.html">http://sites.bio.indiana.edu/~hahnlab/RPGC.html</a>.</p><div id="sec-9" class="subsection"><h3>Implementing RPGC on a simulated recombinant population</h3><p id="p-22">We used the reference <em>C. elegans</em> strain N2 as one parental genome and simulated a second parent that had 120,618 single nucleotide differences and 20,273 short indel differences from N2 spread across the genome randomly; we refer to the second parent as N2b. This level of diversity approximates that seen between two divergent strains of <em>C. elegans</em> (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Wicks <em>et al.</em> 2001</a>). We simulated individual RILs equivalent to eight generations of selfing by a set of individual F<sub>2</sub>s that were produced from crossing perfectly isogenic parental strains and self-fertilizing the resulting F<sub>1</sub>. We assumed that this resulted in a total of five crossing-over events per chromosome in the sequenced RILs and that recombination events were uniformly distributed along chromosomes. For 70 female RILs and the two parental strains, we simulated 5× sequence coverage of 2×100-bp paired-end Illumina reads from 180-bp fragment libraries (Step 1a). The 180-bp fragments mean that the two paired reads overlap by approximately 20 bp, as are required for assembly by ALLPATHS-LG (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gnerre <em>et al.</em> 2011</a>). We also simulated 15× coverage for each parent with 3-kb mate-pair libraries (with 2× 100-bp reads), as well as 10× coverage each of 6-kb mate-pair libraries (also with 2× 100-bp reads; Step 1b). All the data used here has been submitted to the National Center for Biotechnology Information Sequence Read Archive under project accession number SRP031655.</p><p id="p-23">To generate sequences that incorporate known error tendencies of the Illumina technology, reads from all individuals were simulated using the program pIRS (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hu <em>et al.</em> 2012</a>). These simulated data include nucleotide errors, indel errors, variance in the mean insert size of paired-end sequences, and the GC-bias in read-depth typical of Illumina data (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hu <em>et al.</em> 2012</a>). Although there may be other errors in next-generation sequences not considered here—such as “chimeric” reads from mate-pairs that have incorrectly been put together or adapter sequence contamination—several software packages are available to detect and remove such errors (<em>e.g.</em>, MIRA; <a href="http://sourceforge.net/projects/mira-assembler/">http://sourceforge.net/projects/mira-assembler/</a>). In comparisons reported herein between the RPGC-produced assembly and the standard assembly, all of these types of errors are of course present (or not) in both datasets.</p><p id="p-24">For our initial genome assembly (Step 2a) we included reads from 10 randomly chosen RILs and both parents, for a total of 60× coverage from the short-insert (180 bp) libraries, 30× coverage from the 3-kb libraries, and 20× coverage from the 6-kb libraries. These sequences were used as input to the ALLPATHS-LG assembler, which was run using default settings. The resulting genome assembly had 1874 contigs contained within 213 scaffolds. Without including gaps, the total length of the assembled genome was 97,561,960 bp; with gaps, it was 99,265,117 bp. By comparison, the most recent assembly of the reference <em>C. elegans</em> N2 genome (WBcel215) has a golden path length of 100,286,070 bp. Our initial assembly was used as the framework on which to perform downstream analyses. In addition, we generated a “standard” assembly using reads based only on the N2 parent, including 50× coverage from short-insert libraries, 30× from 3-kb libraries, and 20× from 6-kb libraries. Because these reads are generated from an inbred line—one that presumably will generate fewer errors due to heterozygosity—it can serve as standard against which we compare the RPGC-generated assembly below.</p><p id="p-25">To identify high-quality SNPs and indels (Step 2b), reads from the parents and all of the RILs were mapped against our genome using BWA (v0.6.1; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li and Durbin 2009</a>) with default settings. The raw mapping results were then cleaned and sorted using Picard (v1.77; <a href="http://picard.sourceforge.net">http://picard.sourceforge.net</a>). We used reads from the same 10 RILs used for genome assembly to make initial variant calls. To minimize the number of false-positive calls caused by misalignment due to nearby indels, the processed mapping results were locally realigned using GATK (v2.34; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">McKenna <em>et al.</em> 2010</a>), and the variants were identified using the GATK UnifiedGenotyper. To ensure call quality, we independently called variants using the GATK HaplotypeCaller and SAMtools (v0.1.17; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2009</a>). The overlapping set of variants called by all programs was further filtered to include only (i) sites with two alleles, (ii) sites where at least one of the 10 individuals is identified as homozygous for the nonreference base with a minimum of 4× coverage, and (iii) sites where at most one RIL is genotyped as heterozygous (no sites should be heterozygous, apart from those that are due to assembly or sequencing errors). This high-confidence set of variants was then used for base-quality recalibration using GATK, with the default set of covariates. The base-quality recalibrated BAM files were used as input for a second round of variant calling on the same 10 RILs, followed again by the same three filters as described previously. The top 10% of the highest-scoring calls in this set were used as a training set for variant-quality recalibration of the entire set of calls. We then repeated base-quality recalibration and realignment on the whole filtered set of variants on the two parents and all 70 RILs. Finally, we called genotypes for all 70 RILs as well as the two parental genomes using the GATK UnifiedGenotyper (Step 3a). In total, our analysis of the simulated Illumina sequences identified 137,662 SNPs and 20,289 indels. Note that this is more than the simulated number of variants, as both split and merged loci artificially inflate the number of apparent SNPs and indels in our assembly.</p><p id="p-26">We identified candidate alleles that had been erroneously split into separate loci (Step 3b) by first finding duplicated sequences in the genome using the program LASTZ (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Harris 2007</a>). Such pairs were further filtered to include only those whose total read-depth was less than 600× (350× was expected for single loci), and those longer than 1000 bp with nucleotide identity &gt;90%. There were 31 such pairs of loci longer than 1000 bp in our initial assembly, with the longest spanning 8946 bp.</p><p id="p-27">For each of the 31 pairs of highly similar loci in the genome, we then asked whether pairs of variants identified at homologous positions within the apparently duplicated pair segregated as indels, with the insertion state in one copy always associated with the deletion state in the other copy within a single individual (Step 4); this is the expected pattern for SNPs at split alleles (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). In addition to this expected pattern, we also found such variants represented as typical SNPs (and not indels), but with missing genotypes present at one locus instead of a deletion state (<a href="http://www.g3journal.org/lookup/suppl/2014/02/11/g3.114.010264.DC1/010264SI.pdf">Figure S1</a>). Of the initial 31 pairs, 16 had markers within them and could therefore be examined for patterns of inheritance. Of these 16, eight showed genotypes among the recombinants expected for alleles that have been split into multiple loci, and eight showed genotypes expected from two truly paralogous loci. To fix the split alleles for the final assembly (Step 5), we examined the position of the two copies in our initial assembly. Four of the pairs were located adjacent to one another on the same scaffold (sometimes separated by a small gap) and were simply merged together into a single locus. The other four pairs were located on different scaffolds, and we attempted to resolve them based on the linkage map (see below in this section).</p><p id="p-28">We identified paralogous loci that had been erroneously collapsed into a single locus during assembly (Step 4) by finding variants that were genotyped as heterozygous in &gt;90% of RILs. We then asked whether the read-depth around these variants from all RILs together was greater than 650× and less than 1100× and used this information to identify the boundaries of collapsed loci. (We limited the read-depth to less than 1100× to avoid transposable elements, but in principle these could also be disambiguated.) There were 69 such loci longer than 1000 bp in our initial assembly, with the longest spanning 6501 bp. To fix merged loci (Step 5) we examined the mapping of the paired-ends from all three libraries. We expected that merged loci would consist of one duplicate mapped to the correct position in the genome—the one identified with high read-depth and heterozygous sites in the initial assembly—and one that is not in the assembly. To find the location of the unassembled paralog, we looked for paired-end reads with one read mapping to the merged locus and one read mapping to another position in the genome. Manual inspection of the mapping results revealed five merged loci that had multiple reads mapping to a single position near the collapsed locus and to a different scaffold. In all five of these cases, we found a gap at the proposed position where the duplicate locus should have been located in the assembly. Realignment of the sequence from the merged locus with the region surrounding these gaps allowed us to fill in all of them.</p><p id="p-29">We generated a linkage map (Step 6) using the program MSTMap (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2008</a>). Markers with identical genotypes across all 70 RILs were collapsed into a single marker in order to reduce redundant genotype information from closely linked sites. To minimize mapping errors due to sequencing errors, we only used variants where at most 5% of the RILs had been genotyped as heterozygotes. We also manually collapsed the genotypes of all split loci identified in Step 4 into a single marker genotype. The final set of markers used to construct the linkage map included 2658 SNPs and indels on 113 of the 213 scaffolds. The minimum number of markers on a single scaffold was 1, and the maximum number was 153. Parameters for MSTMap were adjusted until six major linkage groups were obtained, matching the number of known chromosomes in the <em>C. elegans</em> genome. Markers from the six major linkage groups were then run separately in MSTMap to determine the order of markers within each linkage group. The order and orientation of individual scaffolds was resolved using a majority rule in cases where there were discrepancies among markers (scaffolds with only a single markers could not be oriented). In total, at this step we were initially able to place 110 of the 213 scaffolds onto one of the six major linkage groups.</p><p id="p-30">For the four cases in which we detected alleles that had been split into loci on different scaffolds, we examined the mapping of relevant markers in the linkage map in order to determine the correct location of the locus (Step 5). In three of the cases, we were able to find the true location of the single locus. In one case, the two scaffolds mapped to the same location in the genetic map, and closer examination revealed that the two scaffolds could be merged by collapsing the split loci (each of which was located at an end of its scaffold) into a single sequence. In the other two cases one of the loci mapped to the middle of a linkage group, while its paired scaffold did not map at all and appeared to consist of only the split allele. These two mini-scaffolds were removed from the assembly.</p><p id="p-31">We also discovered an unanticipated benefit of the linkage map in improving the assembly. On the basis of the genetic map, we found eight smaller scaffolds varying from 1100 to 3400 bp in length that mapped within larger scaffolds. For seven of these small scaffolds the mapping location appeared to coincide with a gap of approximately the same size within the corresponding larger scaffold. We therefore hypothesized that the small scaffolds could fill the matching gaps, and aligned the smaller scaffold to the region surrounding the gap. In six of the cases the smaller scaffold overlapped sequence flanking the gap, indicating that the shorter sequence was embedded within the longer one. In the last case there was no detectable similarity between the smaller scaffold and flanking sequence, so we inserted it into the middle of the larger corresponding gap with flanking N’s on either side.</p></div><div id="sec-10" class="subsection"><h3>Evaluating assembly accuracy and reconstructing parental genomes</h3><p id="p-32">Our final genome consisted of 88 ordered and oriented scaffolds (including 97 of the original 213 scaffolds), comprising 98,533,986 bp (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Of the remaining 13 small scaffolds with markers, 10 were ordered but not oriented, and 3 had the same position in the linkage map and could not be locally ordered with respect to one another. To assess the accuracy of this genome we determined the fraction of the N2 reference genome covered by correctly ordered and oriented scaffolds. To do this we mapped each scaffold to the reference genome using LASTZ. We found that 100% of our scaffolds were correctly ordered. Similarly, 100% of our scaffolds were correctly oriented; this number does not include the 10 scaffolds that were correctly ordered but that could not be oriented, or the single embedded scaffold that was inserted into a gap without orientation. In total, 96.35% of the N2 genome was covered by the 88 correctly ordered and oriented scaffolds.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464151/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464151/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of assemblies</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">These data compare very well to the genome produced by the standard assembly approach, although they may not be typical of every application of RPGC. Notably, our “standard” assembly—which was generated from an inbred line containing no variation—contains a number of collapsed and split loci (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The split loci are of course 100% identical, but the assembly software has chosen to place them on different scaffolds (in fact, all of these were placed on a mini-scaffold by themselves). In total there are ~13,000 more bases from split loci included in the RPGC assembly, although there are the same number of split loci (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). RPGC was able to correct 95% of these split loci and was also able to confirm that an equal number of highly similar pairs of loci with reduced read-depth were in fact true paralogs (see next paragraph). Collapsed loci could be identified in a standard assembly of an inbred individual by looking for regions that are highly “heterozygous,” but these loci could not be corrected, and in individuals that cannot be inbred these cases cannot be distinguished from native levels of heterozygosity. Overall, we conclude that although sequencing inbred strains undoubtedly has many advantages, sequencing a heterozygous population has not introduced a large amount of error into the assembly and has additionally enabled us to correct or identify many of the assembly errors.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464099/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464099/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Assembly error and corrections</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">We wished to evaluate the accuracy of the changes we made to the initial assembly on the basis of our analysis of variant segregation patterns among recombinant individuals. These changes are not readily accomplished in standard genome assembly, even with follow-up experiments. For split, collapsed, and embedded loci, we used LASTZ to search our candidate regions against the reference N2 genome (a similar analysis was performed to identify errors in the “standard” assembly). We first confirmed that all seven of the small scaffolds embedded into larger scaffolds were at the correct locations. Also consistent with our predictions, the eight split loci that we identified in our assembly—those with the expected genotypic proportions for split loci—do in fact have only one copy in the N2 reference (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). For all seven of the loci that we were able to merge, we correctly identified the location of the true locus. For the other eight pairs of highly similar loci whose patterns of segregation did not suggest that they were erroneously split, we confirmed that there were two copies in the N2 reference for all of them. Of the 15 highly similar pairs for which we did not have markers, nine had one copy in the reference and six had multiple copies in the reference. Overall, we missed nine pairs of split loci and were not able to correctly identify the true genomic location for another pair that we were able to detect as split.</p><p id="p-37">Of the 69 collapsed loci identified by our analysis, we confirmed that 68 had two or more copies in the reference genome (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>); this finding indicates that we have accurately identified such loci. The remaining single collapsed locus yielded fragmented alignments from which it was difficult to determine the correct number of loci in the reference genome. For the five merged loci that we were able to disambiguate and place back into our assembly, comparison with the reference showed that we accurately reassembled the paralog that had been missing into the correct position.</p><p id="p-38">We have likely been able to detect all of the split and collapsed loci in our assembly that are longer than 1000 bp. Even if we conservatively assume that we have detected 50% of such loci, errors of these kinds would still only have affected a very small fraction of the genome. This is likely due to two major factors: the low number of repeats in the <em>C. elegans</em> genome and the relatively low level of heterozygosity simulated between the parental strains of the cross. The aim of this idealized test-case was to demonstrate the utility of RPGC, and we do not expect such low numbers of errors in larger, more complex genomes subjected to assembly. Although we were not able to fix all split and collapsed loci detected in our initial assembly, these regions could be subject to further local assembly. Even in cases in which the assembly cannot be improved, variants found within these regions can be marked as likely false positives and can be appropriately handled in follow-up studies.</p><p id="p-39">In addition to constructing a highly accurate reference genome, we wished to know whether we could accurately determine the sequences of the two parental genomes separately. To do this requires that we determine the haplotypic phase of all our varying sites, correctly assigning each allele to the appropriate parental genome. We input the updated genotypes of all 70 RILs and an additional individual purposefully created to be heterozygous at all sites (<em>i.e.</em>, an F<sub>1</sub> individual for the cross described here) into the program BEAGLE (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Browning and Browning 2007</a>); markers for each linkage group were run separately for computational efficiency. The haplotypic phase of the completely heterozygous (F<sub>1</sub>) individual is the only one relevant to the analysis, as all of the RILs are homozygous at the vast majority of sites. We were able to determine the phase for 112,310 SNPs in this individual. Of the phased markers, 91.4% were correctly assigned to a haplotype matching the N2 genome. These results also indicate that we are able to infer the genome of the other (N2b) parent using the RPGC approach. No other current genome assembly approach is able to simultaneously reconstruct two phased genomes in a single experiment.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-40">Here we have proposed a new synthetic approach to <em>de novo</em> sequence assembly and correction that addresses many of the current challenges of whole-genome sequencing, especially for non-model systems. We evaluated the implementation and accuracy of this proposed methodology by analyzing a simulated recombinant population generated from known sequence data. Our results indicate that RPGC provides a highly accurate method for genome construction, although we have only simulated a single type of recombinant population under a subset of possible conditions. RPGC also provides unprecedented power to reconstruct two phased parental genomes in a single experiment. Nonetheless, the implementation of this method in any specific system will require some careful strategic considerations. We have thus far pointed to several factors that will influence the complexity of genome assembly and construction using RPGC, including the structure and size of the recombinant population, the average number of variants differentiating the parental lines, and the level of heterozygosity expected at each locus. These factors influence the ability to accurately distinguish alleles and paralogs, to accurately and efficiently call genotypes, and to therefore use these data in various steps of assembly and construction within RPGC. Accordingly, they should influence design choices for implementing RPGC in any given system. In the subsections to follow, we discuss two of the most important, and relatively fixed, design considerations: methodological trade-offs and biological constraints.</p><div id="sec-12" class="subsection"><h3>Methodological trade-offs</h3><p id="p-41">The main trade-offs to consider for implementing RPGC involve balancing experimental size and read-coverage against the accuracy and efficiency of the method. Given a fixed investment in sequencing costs, these trade-offs will influence the selection of the parents to generate recombinant populations and the number of recombinant individuals able to be sequenced.</p><p id="p-42">First, only crosses involving parents that differ at a large enough number of sites will be informative for placing and ordering scaffolds on linkage maps. Because errors at any single SNP may result in incorrect genotypes (see the section <em>Biological constraints</em>), arguably the most important factor in this regard is the number of SNPs per assembled scaffold. This quantity can be maximized either by investing in more sequencing (or in a different kind of sequencing) to get longer scaffolds, or by generating or examining recombinant populations from distantly related parents. In the first case, more sequence can be obtained either by adding more coverage per individual to a fixed number of recombinants, or by adding more recombinant individuals; however, sequence read length and coverage can be limited by financial and/or technical limits. Rather than investing in the generation of longer sequences to maximize SNPs/scaffold, an alternative solution for this problem might seem to be selecting parents that are more divergent. However, parents that are too distantly related can produce recombinant populations that suffer from skewed genotypic ratios due to widespread transmission ratio distortion (<em>e.g.</em>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Moyle and Graham 2006</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Myburg <em>et al.</em> 2004</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ross <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Solignac <em>et al.</em> 2004</a>); this compromises the ability to use Mendelian expectations to infer linkage relationships in the recombinant population. Therefore, one design challenge will be identifying experimental parents that differ at a sufficient number of sites, but are not so divergent that their hybrids exhibit substantial levels of marker distortion. Because it is not evident where this point will be in any particular system (<em>e.g.</em>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hall and Willis 2005</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Matsubara <em>et al.</em> 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Payseur and Hoekstra 2005</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zamir and Tadmor 1986</a>), a reasonable compromise might be to choose parents from a single, diverse population or from recently diverged ecotypes. In the latter case, choosing ecotypically diverged parents also allows simultaneous QTL mapping of functional traits that differ between ecotypes, as well as the reconstruction of separate genomes for the two ecotypes. Regardless, because of expected difficulties with sterility and/or marker transmission distortion, parents from different species should generally not be used for RPGC.</p><p id="p-43">Second, for a fixed amount of sequencing investment—ignoring the not insignificant library costs for a moment—there is a trade-off between capturing more meioses (that is, including more recombinant individuals) and more accurately calling genotypes (that is, increasing sequencing coverage of each individual included). The number of recombinant individuals sequenced determines many things, including the ability to order and orient scaffolds within linkage groups and the power to distinguish between alleles and paralogs. In general, an F<sub>2</sub> population will provide twice as many meioses as a BC<sub>1</sub> population, with more advanced intercross lines providing more recombination events. Within a chromosome, the number of ordered blocks of scaffolds will also be determined by the number of meioses. Within each such block there will be no information about the ordering or orientation of scaffolds, so increasing the number of meioses has a direct effect on genome construction steps involving the placement and ordering of scaffolds and subsequent QTL mapping resolution.</p><p id="p-44">Nonetheless, with fixed sequencing investment, a larger number of recombinant individuals means lower sequence coverage per individual. Sequence coverage is essential to accurately call genotypes at variable loci—this is the only way to distinguish heterozygotes from homozygotes (recalling that SNPs are initially identified in the total pool of sequences). Without accurate genotype calls alleles cannot be distinguished from paralogs, and constructing accurate linkage maps is made much more difficult. For example, if read-coverage is approximately Poisson-distributed, then with 5× coverage per recombinant we should be able to accurately detect 84% of heterozygous sites in an individual; for 10× coverage, this number jumps to 98% of heterozygous sites. These calculations assume that there is only one SNP per scaffold, and therefore that each genotype call must be accurate. Instead, we can use multiple independent SNPs to bolster our confidence in the genotype of a whole scaffold or, if there is a meiosis within a scaffold, the genotype of each half. The individual genotypes at multiple independent SNPs within a scaffold can therefore be used to better call the genotype of the entire scaffold. Low-coverage sequencing of recombinant populations has already proved successful in organisms with sequenced genomes largely because the location of all markers is known <em>a priori</em> in these cases (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huang <em>et al.</em> 2009</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xie <em>et al.</em> 2010</a>). This allows the use of models that can genotype whole stretches of each chromosome, even when there are less than perfect genotype calls at each marker. For RPGC where there is no information about the relative location of markers along chromosomes, more SNPs per scaffold (up to a limit) will help to accurately call genotypes. With &gt;1 SNP/scaffold, we likely need at least 5× coverage per potentially heterozygous recombinant individual (<em>e.g.</em>, F<sub>2</sub> individual) to accomplish accurate genome construction.</p><p id="p-45">These methodological trade-offs will be necessarily influenced by financial constraints on the design of the experiment. In particular, even when increasing the total number of sequenced recombinants might be preferable, this involves increasing the number of sequencing libraries. Libraries are a fixed, per individual, cost. Given the large amount of DNA sequence produced by a single Illumina HiSeq lane (currently at least dozens of gigabases) for a relatively low price, the costs of library construction for a large recombinant population are expected to rapidly outstrip the sequencing costs. Therefore, increasing per-individual coverage might be more financially feasible in comparison with increasing the total number of recombinant individuals.</p><p id="p-46">Finally, it is worth noting that there are cheaper, alternative approaches to RPGC available if the only aim is to order and orient scaffolds on a map. A number of reduced-representation approaches (<em>e.g.</em>, RAD-tags; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baird <em>et al.</em> 2008</a>) offer the ability to detect and genotype large numbers of variants more cheaply than whole-genome sequencing. These methods have been previously used to construct linkage maps connecting scaffolds to chromosomes (<em>e.g.</em>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hyten <em>et al.</em> 2010</a>) or even individual gene models to chromosomes (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amores <em>et al.</em> 2011</a>). Alternatively, if the average scaffold is long enough and high-molecular-weight DNA is available, then optical mapping also offers the ability to place sequences on chromosomes (<em>e.g.</em>, <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Zhou <em>et al.</em> 2009</a>). Both of these approaches avoid the need for a large number of standard barcoded libraries, each of which has a significant cost. However, neither of these approaches has the ability to correct split or collapsed loci either because they do not have dense enough marker information within such loci (RAD-tags) or do not contain genetic information (optical mapping). They therefore are not expected to produce a genome of the same quality as RPGC. By definition, reduced representation approaches also will not encompass as broad a representation of the genome as RPGC.</p></div><div id="sec-13" class="subsection"><h3>Biological constraints</h3><p id="p-47">Biological factors such as the reproductive biology and/or evolutionary history of a species can also impose absolute constraints on experimental design choices. Some of these factors (<em>e.g.</em>, genome size, generation time, etc.) are outside the control of the researcher and should be considered when identifying an appropriate system for RPGC, rather than when deciding on a population design for the sequencing. However, for several biological factors that rest in the hands of the researcher, the efficiency, accuracy, and success of RPGC can depend on thoughtful design considerations.</p><p id="p-48">Two such factors are the specific identity of the parents of the recombinant population sequenced for RPGC and the genetic structure of the recombinant population itself. Both of these factors will to some extent be constrained by reproductive biology (clutch size, inability to self or inbreed, etc.), and by the evolutionary and breeding history (and therefore genetic characteristics) of the target individuals and the populations from which they are drawn. These constraints and their effects on experimental design can be illustrated by contrasting a relatively simple <em>vs.</em> more complex potential scenario when applying RPGC. Under the simplest scenario, the target species is able to self-fertilize (or allows consanguineous brother-sister mating), individual fecundity is high (family sizes can be large), the species segregates few deleterious recessive alleles (inbreeding depression is low), and individuals from different ecotypes are moderately differentiated at the molecular level. This might be the case in species with a history of mixed-mating or biparental inbreeding, and known patterns of local adaptation. For this case the ideal experimental design would use two inbred parents drawn from different locally adapted populations to generate a single F<sub>1</sub> individual that is selfed to generate a large F<sub>2</sub> population. This design assures only two alleles per locus in the F<sub>2</sub>s, thereby simplifying both the identification of alleles <em>vs.</em> paralogs, and the expected genotypic ratios in the F<sub>2</sub>s at each locus. In addition, large family sizes enable a large F<sub>2</sub> population to be generated from a single brother-sister F<sub>1</sub> pair (or selfed F<sub>1</sub>), producing sufficient recombinant individuals to capture enough meioses for genome construction and for QTL mapping. As such, this scenario is potentially the best-case application for RPGC, and closely resembles the <em>C. elegans</em> dataset we have simulated here.</p><p id="p-49">Under a more realistic scenario, parents are outbred and F<sub>1</sub>s cannot be self-fertilized, although F<sub>1</sub> brother-sister mating is possible. In this case, there can be up to four alleles (<em>i.e.</em>, haplotypes) segregating at any locus in an F<sub>2</sub> recombinant population. This will be more challenging for the discrimination of alleles <em>vs.</em> paralogs, although in any single F<sub>2</sub> individual there can be at most only two alleles at each locus, and expected genotypic ratios are still relatively simple (<em>e.g.</em>, even with four alleles per locus, there are only 10 possible genotypes and the expected frequency of each homozygote genotype is 1/16). Even so, these challenges are expected to increase rapidly as greater constraints are imposed by the reproductive biology of the target species. For example, if consangineous F<sub>1</sub> mating is prevented, then four outbred parents will be involved in generating the recombinant population (because F<sub>1</sub> crosses must be between individuals from two different families), meaning up to eight alleles (haplotypes) and 36 possible genotypes per locus. In this extreme case, each unique homozygous genotype will appear at a frequency of only 1/64.</p><p id="p-50">As these cases illustrate, one of the essential design problems in generating (or identifying existing) populations for RPGC is to minimize the number of alleles segregating per locus—the ideal number is two. The greater the number of segregating alleles, the more sequence coverage is needed to accurate identify each allele and each genotype at a site. The greater the number of individuals contributing alleles to the recombinant population, the larger this problem can become. In some cases, strategic choice of crossing design can reduce some of this dimensionality. For example, in comparison to F<sub>2</sub>s, backcross designs on average reduce by 1/4 the total number of alleles expected to be segregating in a recombinant population. Therefore, backcrossed populations might be preferable in cases where allelic variation is expected to be high.</p><p id="p-51">As an alternative approach to controlled genetic crosses, we can also envision the application of RPGC to systems in which classical genetics is not feasible, although the aforementioned analysis and inference steps will be correspondingly more complex. These cases can use alternative approaches to linkage mapping, or may even take advantage of linkage disequilibrium (LD) in natural populations. For instance, if DNA from a multigeneration pedigree is available, this can be used to a generate linkage map (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chagné <em>et al.</em> 2003</a>). If a large enough number of recombinants is available, linkage maps can be constructed even when only heterozygous parents and offspring are available (<em>e.g.</em>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grattapaglia and Sederoff 1994</a>) or when only a single heterozygous parent and its gametes are available (<em>e.g.</em>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tulsieram <em>et al.</em> 1992</a>). Although the distance over which LD extends in natural populations is generally not large enough to aid in constructing a genetic map, certain demographic histories will increase LD tract lengths. A recent study in humans used the long tracts of LD created by population admixture to map the location of dozens of unplaced scaffolds in the genome (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Genovese <em>et al.</em> 2013</a>). If similar populations are available in other species, patterns of LD in nature could be used in an RPGC-like approach.</p><p id="p-52">Finally, other biological factors also will shape experimental design decisions by imposing logistical constraints. For example, organismal body size imposes constraints on the amount of DNA available per individual; given current technologies, small organisms (<em>e.g.</em>, nematodes, flies, etc.) are unlikely to have sufficient high-quality DNA per individual to allow sequencing, especially if the goal is to perform simultaneous trait analysis on the same genotypes. For these organisms it would be necessary to use alternative (non-F<sub>2</sub>) population structures—such as RILs—that enable replicate homozygous recombinant genotypes and therefore allow both destructive sampling to obtain DNA and simultaneous phenotype analysis within each line.</p><p id="p-53">Given these biological realities, some systems will clearly be better targets for RPGC than others. However, it should be clear that many complex scenarios can be accommodated with appropriate design choices during the selection of population parents and the generation or identification of the recombinant population. This includes cases where inbred lines cannot be generated and outcrossing is enforced, or even cases in which individuals are sampled from nature. Although these cases might place specific bounds on the efficiency and accuracy of individual steps in the assembly, genome construction, and mapping process, they are not absolute barriers to using the RPGC approach. This makes RPGC especially relevant to non-model systems that are poorly served by current whole-genome sequencing approaches.</p></div></div><div class="section conclusions" id="sec-14"><h2 class="">Conclusions</h2><p id="p-54">Here we have proposed an approach that resolves three critical problems in applying NGS technology to generating <em>de novo</em> genomes, especially in traditionally nonmodel systems: first, genome assembly problems, including resolving alleles <em>vs.</em> paralogs, and identifying collapsed paralogs; second, genome construction problems, including the identification of linkage groups and the placement and ordering of scaffolds onto chromosomes; and third, limitations on immediate applications, including the absence of SNPs and the inability to conduct quantitative trait mapping. Together, these factors currently conspire to place whole-genome sequencing beyond the reach of many species that are rich in organismal (<em>e.g.</em>, physiological, behavioral, and ecological) data. Although recombinant populations have been used to assign scaffolds to chromosomes in multiple genome projects, RPGC collapses all three phases of the transition from genomic nonmodel to model organism (genome sequencing, linkage map construction, application) into a single experiment, and solves problems of genome assembly that are not addressed by typical linkage mapping approaches.</p><p id="p-55">We have applied RPGC to a simulated recombinant population, one that is idealized in many respects. It will almost surely be the case that when applied to real sequence data (possibly containing errors not included in our simulated reads) and real recombinant populations (containing, for instance, residual heterozygosity not simulated here), the genome produced will not reach 96% accuracy. However, we also have not designed RPGC with a single sequencing platform, assembly algorithm, read-mapping software package, or SNP-calling tool in mind; our approach can be used with any combination of programs, and can therefore take advantage of all future improvements in any of these steps.</p><p id="p-56">It is important to note that RPGC does not solve all of the problems of genome construction, including sequencing errors (<em>e.g.</em>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ye <em>et al.</em> 2011</a>) or highly fragmented scaffolds and contigs (<em>e.g.</em>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mortazavi <em>et al.</em> 2010</a>). It is also not ideal for organisms in which controlled genetic crosses are not possible. However, the flexibility of the method in terms of both choices in sequencing technology and assembly tools means that the general approach will be feasible and economical for the foreseeable future. Moreover, because RPGC is able to accommodate and/or correct for the challenges that frequently emerge for nonmodel systems, implementing this method in a range of empirical systems is expected to be more successful than traditional genome assembly processes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We thank Bruce McClure for his helpful advice, as well as Corbin Jones and multiple reviewers for their comments. This work was supported by funding from National Science Foundation grant DEB-1249633 to M.W.H., and the computational resources made available by the National Center for Genome Analysis Support (National Science Foundation grant DBI-1062432).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010264/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.010264/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-58"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-10-28"><span class="received-label">Received </span>October 28, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-02-10"><span class="accepted-label">Accepted </span>February 10, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Hahn <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.1" data-doi="10.1126/science.287.5461.2185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Celniker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gocayne</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The genome sequence of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2185</span>–<span class="cit-lpage">2195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAdams%26rft.auinit1%253DM.%2BD.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2185%26rft.epage%253D2195%26rft.atitle%253DThe%2BGenome%2BSequence%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2185%26rft_id%253Dinfo%253Apmid%252F10731132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.2" data-doi="10.1038/nmeth.1527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sajjadian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Limitations of next-generation genome sequence assembly.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DAlkan%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D61%26rft.epage%253D65%26rft.atitle%253DLimitations%2Bof%2Bnext-generation%2Bgenome%2Bsequence%2Bassembly.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1527%26rft_id%253Dinfo%253Apmid%252F21102452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102452&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285712000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.3" data-doi="10.1534/genetics.111.127324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontenot</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome evolution and meiotic maps by massively parallel DNA sequencing: spotted gar, an outgroup for the teleost genome duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAmores%26rft.auinit1%253DA.%26rft.volume%253D188%26rft.issue%253D4%26rft.spage%253D799%26rft.epage%253D808%26rft.atitle%253DGenome%2BEvolution%2Band%2BMeiotic%2BMaps%2Bby%2BMassively%2BParallel%2BDNA%2BSequencing%253A%2BSpotted%2BGar%252C%2Ban%2BOutgroup%2Bfor%2Bthe%2BTeleost%2BGenome%2BDuplication%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127324%26rft_id%253Dinfo%253Apmid%252F21828280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzQvNzk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.4" data-doi="10.1101/gr.GR-1871R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yavor</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massa</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trask</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Segmental duplications: organization and impact within the current Human Genome Project assembly.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.GR-1871R%26rft_id%253Dinfo%253Apmid%252F11381028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjExLzYvMTAwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.5" data-doi="10.1126/science.1072047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinert</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samonte</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Recent segmental duplications in the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">297</span>: <span class="cit-fpage">1003</span>–<span class="cit-lpage">1007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBailey%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D297%26rft.issue%253D5583%26rft.spage%253D1003%26rft.epage%253D1007%26rft.atitle%253DRecent%2BSegmental%2BDuplications%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1072047%26rft_id%253Dinfo%253Apmid%252F12169732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTcvNTU4My8xMDAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.6"><div class="cit-metadata unstructured">Baird, N. A., P. D. Etter, T. S. Atwood, M. C. Currey, A. L. Shiver <em>et al.</em>, 2008 Rapid SNP discovery and genetic mapping using sequenced RAD markers. PLoS One 3: e3376.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.7" data-doi="10.1101/gr.077065.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpern</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axelrod</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bafna</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An MCMC algorithm for haplotype assembly from whole-genome sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1336</span>–<span class="cit-lpage">1346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DBansal%26rft.auinit1%253DV.%26rft.volume%253D18%26rft.issue%253D8%26rft.spage%253D1336%26rft.epage%253D1346%26rft.atitle%253DAn%2BMCMC%2Balgorithm%2Bfor%2Bhaplotype%2Bassembly%2Bfrom%2Bwhole-genome%2Bsequence%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.077065.108%26rft_id%253Dinfo%253Apmid%252F18676820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzgvMTMzNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.8" data-doi="10.1101/gr.081851.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrière</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pekarek</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haag</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detecting heterozygosity in shotgun genome assemblies: lessons from obligately outcrossing nematodes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081851.108%26rft_id%253Dinfo%253Apmid%252F19204328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNDcwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.9" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.10" data-doi="10.1534/genetics.110.115444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casola</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganote</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nonallelic gene conversion in the genus <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115444%26rft_id%253Dinfo%253Apmid%252F20215470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg1LzEvOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.11" data-doi="10.1023/A:1026318327911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lalanne</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madur</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pot</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparative genome and QTL mapping between maritime and loblolly pines.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D12%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1026318327911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1026318327911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.12" data-doi="10.1186/gb-2003-4-4-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estivill</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khaja</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacDonald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide detection of segmental duplications and potential assembly errors in the human genome sequence.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCheung%26rft.auinit1%253DJ.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DGenome-wide%2Bdetection%2Bof%2Bsegmental%2Bduplications%2Band%2Bpotential%2Bassembly%2Berrors%2Bin%2Bthe%2Bhuman%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2003-4-4-r25%26rft_id%253Dinfo%253Apmid%252F12702206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2003-4-4-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12702206&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.13" data-doi="10.1126/science.1197761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colbourne</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfrender</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The ecoresponsive genome of <em>Daphnia pulex</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColbourne%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D331%26rft.issue%253D6017%26rft.spage%253D555%26rft.epage%253D561%26rft.atitle%253DThe%2BEcoresponsive%2BGenome%2Bof%2BDaphnia%2Bpulex%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1197761%26rft_id%253Dinfo%253Apmid%252F21292972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzEvNjAxNy81NTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.14" data-doi="10.1126/science.282.5396.2012"><div class="cit-metadata"><cite><span class="cit-comment"><em>C. elegans</em> Sequencing Consortium</span>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genome sequence of the nematode <em>C. elegans</em>: a platform for investigating biology.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">2012</span>–<span class="cit-lpage">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BC.%2Belegans%2BSequencing%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D282%26rft.issue%253D5396%26rft.spage%253D2012%26rft.epage%253D2018%26rft.atitle%253DGenome%2BSequence%2Bof%2Bthe%2BNematode%2BC.%2Belegans%253A%2BA%2BPlatform%2Bfor%2BInvestigating%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5396.2012%26rft_id%253Dinfo%253Apmid%252F9851916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5Ni8yMDEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.15" data-doi="10.1126/science.1080049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dehal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degnan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The draft genome of <em>Ciona intestinalis</em>: insights into chordate and vertebrate origins.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">2157</span>–<span class="cit-lpage">2167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDehal%26rft.auinit1%253DP.%26rft.volume%253D298%26rft.issue%253D5601%26rft.spage%253D2157%26rft.epage%253D2167%26rft.atitle%253DThe%2BDraft%2BGenome%2Bof%2BCiona%2Bintestinalis%253A%2BInsights%2Binto%2BChordate%2Band%2BVertebrate%2BOrigins%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1080049%26rft_id%253Dinfo%253Apmid%252F12481130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTYwMS8yMTU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.16" data-doi="10.1126/science.1194167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denoeud</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henriet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mungpakdee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aury</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plasticity of animal genome architecture unmasked by rapid evolution of a pelagic tunicate.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1381</span>–<span class="cit-lpage">1385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194167%26rft_id%253Dinfo%253Apmid%252F21097902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAwOS8xMzgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.17" data-doi="10.1038/ng.2565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Genovese</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altemose</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindgren</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using population admixture to help complete maps of the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D406%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2565%26rft_id%253Dinfo%253Apmid%252F23435088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23435088&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.18" data-doi="10.1073/pnas.1017351108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacCallum</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017351108%26rft_id%253Dinfo%253Apmid%252F21187386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1121%26rft.epage%253D1137%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bof%2BEucalyptus%2Bgrandis%2Band%2BEucalyptus%2Burophylla%2Busing%2Ba%2Bpseudo-testcross%253A%2Bmapping%2Bstrategy%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7982566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.20" data-doi="10.1534/genetics.104.038653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Transmission ratio distortion in intraspecific hybrids of <em>Mimulus guttatus:</em> implications for genomic divergence.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038653%26rft_id%253Dinfo%253Apmid%252F15781698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcwLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.21" data-doi="10.1101/gr.085951.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demuth</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casola</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Adaptive evolution of young duplicated genes in mammals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHan%26rft.auinit1%253DM.%2BV.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D859%26rft.epage%253D867%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Byoung%2Bgene%2Bduplicates%2Bin%2Bmammals%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.085951.108%26rft_id%253Dinfo%253Apmid%252F19411603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvODU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.22" data-doi="10.1016/j.mimet.2011.06.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haridas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breuill</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohlmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsiang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A biologist’s guide to de novo genome assembly using next-generation sequence data: A test with fungal genomes.</span> <abbr class="cit-jnl-abbrev">J. Microbiol. Methods</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">368</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMicrobiological%2BMethods%26rft.stitle%253DJournal%2Bof%2BMicrobiological%2BMethods%26rft.aulast%253DHaridas%26rft.auinit1%253DS.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D368%26rft.epage%253D375%26rft.atitle%253DA%2Bbiologist%2527s%2Bguide%2Bto%2Bde%2Bnovo%2Bgenome%2Bassembly%2Busing%2Bnext-generation%2Bsequence%2Bdata%253A%2BA%2Btest%2Bwith%2Bfungal%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mimet.2011.06.019%26rft_id%253Dinfo%253Apmid%252F21749903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mimet.2011.06.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21749903&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294507400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-thesis" id="cit-4.4.669.23"><div class="cit-metadata unstructured">Harris, R. S., 2007 Improved pairwise alignment of genomic DNA. Ph.D. Thesis, The Pennsylvania State University.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.24" data-doi="10.1126/science.1076181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpern</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlab</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The genome sequence of the malaria mosquito <em>Anopheles gambiae</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHolt%26rft.auinit1%253DR.%2BA.%26rft.volume%253D298%26rft.issue%253D5591%26rft.spage%253D129%26rft.epage%253D149%26rft.atitle%253DThe%2BGenome%2BSequence%2Bof%2Bthe%2BMalaria%2BMosquito%2BAnopheles%2Bgambiae%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1076181%26rft_id%253Dinfo%253Apmid%252F12364791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTgvNTU5MS8xMjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.25" data-doi="10.1093/bioinformatics/bts187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">pIRS: Profile-based Illumina pair-end reads simulator.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts187%26rft_id%253Dinfo%253Apmid%252F22508794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTEvMTUzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.26" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.27" data-doi="10.1186/1471-2164-11-38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High-throughput SNP discovery through deep resequencing of a reduced representation library to anchor and orient scaffolds in the soybean whole genome sequence.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHyten%26rft.auinit1%253DD.%2BL.%26rft.volume%253D11%26rft.spage%253D38%26rft.epage%253D38%26rft.atitle%253DHigh-throughput%2BSNP%2Bdiscovery%2Bthrough%2Bdeep%2Bresequencing%2Bof%2Ba%2Breduced%2Brepresentation%2Blibrary%2Bto%2Banchor%2Band%2Borient%2Bscaffolds%2Bin%2Bthe%2Bsoybean%2Bwhole%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-38%26rft_id%253Dinfo%253Apmid%252F20078886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20078886&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.28" data-doi="10.1186/gb-2010-11-3-r28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detection and correction of false segmental duplications caused by genome mis-assembly.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKelley%26rft.auinit1%253DD.%2BR.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253DR28%26rft.epage%253DR28%26rft.atitle%253DDetection%2Band%2Bcorrection%2Bof%2Bfalse%2Bsegmental%2Bduplications%2Bcaused%2Bby%2Bgenome%2Bmis-assembly.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-3-r28%26rft_id%253Dinfo%253Apmid%252F20219098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-3-r28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20219098&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.29" data-doi="10.1101/gr.5894107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Diploid genome reconstruction of <em>Ciona intestinalis</em> and comparative analysis with <em>Ciona savignyi</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5894107%26rft_id%253Dinfo%253Apmid%252F17567986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE3LzcvMTEwMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.30" data-doi="10.1038/nbt.1740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiatt</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patwardhan</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Haplotype-resolved genome sequencing of a Gujarati Indian individual.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKitzman%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D63%26rft.atitle%253DHaplotype-resolved%2Bgenome%2Bsequencing%2Bof%2Ba%2BGujarati%2BIndian%2Bindividual.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1740%26rft_id%253Dinfo%253Apmid%252F21170042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1740&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21170042&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.31" data-doi="10.1038/35057062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linton</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Initial sequencing and analysis of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">860</span>–<span class="cit-lpage">921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D409%26rft.issue%253D6822%26rft.spage%253D860%26rft.epage%253D921%26rft.atitle%253DInitial%2Bsequencing%2Band%2Banalysis%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35057062%26rft_id%253Dinfo%253Apmid%252F11237011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35057062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237011&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166938800058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.32" data-doi="10.1534/genetics.112.142018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic variation in natural populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142018%26rft_id%253Dinfo%253Apmid%252F22673804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.33" data-doi="10.1101/gr.094557.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewin</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontius</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Every genome sequence needs a good map.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1925</span>–<span class="cit-lpage">1928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094557.109%26rft_id%253Dinfo%253Apmid%252F19596977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8xOTI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.34" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.35" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.36" data-doi="10.1093/bioinformatics/btr319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papasian</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative studies of <em>de novo</em> assembly tools for next-generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2031</span>–<span class="cit-lpage">2037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr319%26rft_id%253Dinfo%253Apmid%252F21636596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjAzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.37" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> Genetic Reference Panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.38" data-doi="10.1007/s00438-011-0648-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsubara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebana</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizubayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ando</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Relationship between transmission ratio distortion and genetic divergence in intraspecific rice crosses.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DMatsubara%26rft.auinit1%253DK.%26rft.volume%253D286%26rft.issue%253D5-6%26rft.spage%253D307%26rft.epage%253D319%26rft.atitle%253DRelationship%2Bbetween%2Btransmission%2Bratio%2Bdistortion%2Band%2Bgenetic%2Bdivergence%2Bin%2Bintraspecific%2Brice%2Bcrosses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-011-0648-6%26rft_id%253Dinfo%253Apmid%252F21918817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-011-0648-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21918817&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.39" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.40" data-doi="10.1101/gr.111021.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoshechkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Scaffolding a <em>Caenorhabditis</em> nematode genome with RNA-seq.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1740</span>–<span class="cit-lpage">1747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111021.110%26rft_id%253Dinfo%253Apmid%252F20980554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMi8xNzQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.41" data-doi="10.1093/molbev/msj112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moyle</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">E. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide associations between hybrid sterility QTL and marker transmission ratio distortion.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj112%26rft_id%253Dinfo%253Apmid%252F16495346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIzLzUvOTczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.42" data-doi="10.1534/genetics.166.3.1405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myburg</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffin</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whetten</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetics of postzygotic isolation in eucalyptus: whole-genome analysis of barriers to introgression in a wide interspecific cross of <em>Eucalyptus grandis</em> and <em>E. globulus</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMyburg%26rft.auinit1%253DA.%2BA.%26rft.volume%253D166%26rft.issue%253D3%26rft.spage%253D1405%26rft.epage%253D1418%26rft.atitle%253DGenetics%2Bof%2BPostzygotic%2BIsolation%2Bin%2BEucalyptus%253A%2BWhole-Genome%2BAnalysis%2Bof%2BBarriers%2Bto%2BIntrogression%2Bin%2Ba%2BWide%2BInterspecific%2BCross%2Bof%2BEucalyptus%2Bgrandis%2Band%2BE.%2Bglobulus%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.3.1405%26rft_id%253Dinfo%253Apmid%252F15082559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni8zLzE0MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.43" data-doi="10.1534/genetics.105.046193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Signatures of reproductive isolation in patterns of single nucleotide diversity across inbred strains of mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1905</span>–<span class="cit-lpage">1916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.046193%26rft_id%253Dinfo%253Apmid%252F16143616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzE5MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.44" data-doi="10.1371/journal.pgen.1002174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staisch</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlin</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Caenorhabditis briggsae</em> recombinant inbred line genotypes reveal inter-strain incompatibility and the evolution of recombination.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRoss%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002174%26rft.epage%253De1002174%26rft.atitle%253DCaenorhabditis%2Bbriggsae%2Brecombinant%2Binbred%2Bline%2Bgenotypes%2Breveal%2Binter-strain%2Bincompatibility%2Band%2Bthe%2Bevolution%2Bof%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002174%26rft_id%253Dinfo%253Apmid%252F21779179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21779179&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.45" data-doi="10.1093/gbe/evp048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sackton</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulathinal</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dopman</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomic inferences from sparse high-throughput sequencing of two populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevp048%26rft_id%253Dinfo%253Apmid%252F20333214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjEvMC80NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.46" data-doi="10.1101/gr.101360.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Assembly of large genomes using second-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.101360.109%26rft_id%253Dinfo%253Apmid%252F20508146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTE2NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.47" data-doi="10.1534/genetics.167.1.253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solignac</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vautrin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baudry</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mougel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loiseau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A microsatellite-based linkage map of the honeybee, <em>Apis mellifera</em> L.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSolignac%26rft.auinit1%253DM.%26rft.volume%253D167%26rft.issue%253D1%26rft.spage%253D253%26rft.epage%253D262%26rft.atitle%253DA%2BMicrosatellite-Based%2BLinkage%2BMap%2Bof%2Bthe%2BHoneybee%252C%2BApis%2Bmellifera%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.167.1.253%26rft_id%253Dinfo%253Apmid%252F15166152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY3LzEvMjUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Next generation molecular ecology.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D19%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.49" data-doi="10.1038/nbt0692-686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tulsieram</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Single tree genetic linkage mapping in conifers using haploid DNA from megagametophytes.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D10%26rft.spage%253D686%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0692-686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0692-686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.50" data-doi="10.1038/88878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicks</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plasterk</span>  <span class="cit-name-given-names">R. H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Rapid gene mapping in <em>Caenorhabditis elegans</em> using a high density polymorphism map.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWicks%26rft.auinit1%253DS.%2BR.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D164%26rft.atitle%253DRapid%2Bgene%2Bmapping%2Bin%2BCaenorhabditis%2Belegans%2Busing%2Ba%2Bhigh%2Bdensity%2Bpolymorphism%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F88878%26rft_id%253Dinfo%253Apmid%252F11381264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/88878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11381264&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169011800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.51" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.52" data-doi="10.1073/pnas.1005931107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Parent-independent genotyping for constructing an ultrahigh-density linkage map based on population sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">10578</span>–<span class="cit-lpage">10583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1005931107%26rft_id%253Dinfo%253Apmid%252F20498060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzIzLzEwNTc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.53" data-doi="10.1186/gb-2011-12-3-r31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minx</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thane</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Locke</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A vertebrate case study of the quality of assemblies derived from next-generation sequences.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DYe%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253DR31%26rft.epage%253DR31%26rft.atitle%253DA%2Bvertebrate%2Bcase%2Bstudy%2Bof%2Bthe%2Bquality%2Bof%2Bassemblies%2Bderived%2Bfrom%2Bnext-generation%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-3-r31%26rft_id%253Dinfo%253Apmid%252F21453517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-3-r31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21453517&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tadmor</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Unequal segregation of nuclear genes in plants.</span> <abbr class="cit-jnl-abbrev">Bot. Gaz.</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BGaz.%26rft.volume%253D147%26rft.spage%253D355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.669.55" data-doi="10.1371/journal.pgen.1000711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potamousis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A single molecule scaffold for the maize genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De1000711%26rft.epage%253De1000711%26rft.atitle%253DA%2Bsingle%2Bmolecule%2Bscaffold%2Bfor%2Bthe%2Bmaize%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000711%26rft_id%253Dinfo%253Apmid%252F19936062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19936062&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/4/669.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/657" title="EMS Mutagenesis in the Pea Aphid Acyrthosiphon pisum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/681" title="Evidence that Natural Selection on Codon Usage in Drosophila pseudoobscura Varies Across Codons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/4"><span>Volume 4 Issue 4, April 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/4.cover-source.jpg?itok=Rf0YhETZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Dpz0CzNIabt6ZUZMt5oHESu2D5HLupkgMD6Nt9lDw_c" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div><div class="field-item odd"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/4/669" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/4/669" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/4/669" class="active" data-icon-position="" data-hide-link-title="0">Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459832" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3xxYyQcnemFtQvbH8fJF-9giBexqp6etg9Eis8uLH8I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/669.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/4/669" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ay0I_4nCp1jeUVve5_zzsU0KJU-rnLi14yi5gvdo5Qg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/4/669" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/669.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459832" id="node459832--2" data-pisa="ggg;4/4/669" data-pisa-master="ggg;g3.114.010264" data-apath="/ggg/4/4/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew W. Hahn</span>, <span class="highwire-citation-author" data-delta="1">Simo V. Zhang</span> and <span class="highwire-citation-author" data-delta="2">Leonie C. Moyle</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-679;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010264 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459832--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew W. Hahn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BHahn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hahn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BW.%2BHahn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mwh@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">mwh@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Simo V. Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simo%2BV.%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimo%2BV.%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Leonie C. Moyle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonie%2BC.%2BMoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moyle%20LC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonie%2BC.%2BMoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/4/669" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/4/669&amp;title=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/4/669&amp;title=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/4/669&amp;title=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/4/669&amp;text=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/4/669&amp;title=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/4/669&amp;t=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/4/669&amp;title=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/4/669&amp;title=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459832/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459832" id="node459832--3" data-pisa="ggg;4/4/669" data-pisa-master="ggg;g3.114.010264" data-apath="/ggg/4/4/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew W. Hahn</span>, <span class="highwire-citation-author" data-delta="1">Simo V. Zhang</span> and <span class="highwire-citation-author" data-delta="2">Leonie C. Moyle</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-679;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010264 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459832--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew W. Hahn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BHahn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hahn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BW.%2BHahn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mwh@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">mwh@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Simo V. Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simo%2BV.%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimo%2BV.%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Leonie C. Moyle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonie%2BC.%2BMoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moyle%20LC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeonie%2BC.%2BMoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=669&amp;rft.epage=679&amp;rft.atitle=Sequencing%2C%20Assembling%2C%20and%20Correcting%20Draft%20Genomes%20Using%20Recombinant%20Populations&amp;rft.volume=4&amp;rft.issue=4&amp;rft.date=2014-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hahn%2C+Matthew+W.&amp;rft.au=Zhang%2C+Simo+V.&amp;rft.au=Moyle%2C+Leonie+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459832/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459832/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459832/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459832/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459832/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459832/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459832/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459832/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459832/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459832/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459832/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459832/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list"><div class="highwire-articles-in-toc investigations"><h3 class="highwire-toc-heading">Investigations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span>
</a></div></div><div class="highwire-articles-in-toc multiparental-populations"><h3 class="highwire-toc-heading">Multiparental Populations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496049" id="node496049" data-pisa="ggg;g3.119.400658v1" data-pisa-master="ggg;g3.119.400658" data-apath="/ggg/early/2020/03/26/g3.119.400658.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.119.400658" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Dissecting Adaptive Traits with Nested Association Mapping: Genetic Architecture of Inflorescence Morphology in Sorghum</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span>
</a></div></div></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/4/669" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/4/669.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/4/669.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/4/669.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/4/669.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
