<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/4/681" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/4/681.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/4/681.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/4/681.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/4/681/embed/mml-math-2.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/4/681.atom" />
<meta name="HW.pisa" content="ggg;4/4/681" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evidence that Natural Selection on Codon Usage in Drosophila pseudoobscura Varies Across Codons" />
<meta name="DC.Identifier" content="10.1534/g3.114.010488" />
<meta name="DC.Date" content="2014-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2014 Kliman. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.114.012955" />
<meta name="DC.Description" content="Like other species of Drosophila, Drosophila pseudoobscura has a distinct bias toward the usage of C- and G-ending codons. Previous studies have indicated that this bias is due, at least in part, to natural selection. Codon bias clearly differs among amino acids (and other codon classes) in Drosophila, which may reflect differences in the intensity of selection on codon usage. Ongoing natural selection on synonymous codon usage should be reflected in the shapes of the site frequency spectra of derived states at polymorphic positions. Specifically, regardless of other demographic effects on the spectrum, it should be shifted toward higher values for changes from less-preferred to more-preferred codons, and toward lower values for the converse. If the intensity of natural selection is increased, shifts in the site frequency spectra should be more pronounced. A total of 33,729 synonymous polymorphic sites on Chromosome 2 in D. pseudoobscura were analyzed. Shifts in the site frequency spectra are consistent with differential intensity of natural selection on codon usage, with stronger shifts associated with higher codon bias. The shifts, in general, are greater for polymorphic synonymous sites than for polymorphic intron sites, also consistent with natural selection. However, unlike observations in D. melanogaster , codon bias is not reduced in areas of low recombination in D. pseudoobscura ; the site frequency spectrum signal for selection on codon usage remains strong in these regions. However, diversity is reduced, as expected. It is possible that estimates of low recombination reflect a recent change in recombination rate." />
<meta name="DC.Contributor" content="Richard M. Kliman" />
<meta name="article:published_time" content="2014-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Evidence that Natural Selection on Codon Usage in Drosophila pseudoobscura Varies Across Codons" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Like other species of Drosophila, &lt;i&gt;Drosophila pseudoobscura&lt;/i&gt; has a distinct bias toward the usage of C- and G-ending codons. Previous studies have indicated that this bias is due, at least in part, to natural selection. Codon bias clearly differs among amino acids (and other codon classes) in Drosophila, which may reflect differences in the intensity of selection on codon usage. Ongoing natural selection on synonymous codon usage should be reflected in the shapes of the site frequency spectra of derived states at polymorphic positions. Specifically, regardless of other demographic effects on the spectrum, it should be shifted toward higher values for changes from less-preferred to more-preferred codons, and toward lower values for the converse. If the intensity of natural selection is increased, shifts in the site frequency spectra should be more pronounced. A total of 33,729 synonymous polymorphic sites on Chromosome 2 in &lt;i&gt;D. pseudoobscura&lt;/i&gt; were analyzed. Shifts in the site frequency spectra are consistent with differential intensity of natural selection on codon usage, with stronger shifts associated with higher codon bias. The shifts, in general, are greater for polymorphic synonymous sites than for polymorphic intron sites, also consistent with natural selection. However, unlike observations in &lt;i&gt;D. melanogaster&lt;/i&gt;, codon bias is not reduced in areas of low recombination in &lt;i&gt;D. pseudoobscura&lt;/i&gt;; the site frequency spectrum signal for selection on codon usage remains strong in these regions. However, diversity is reduced, as expected. It is possible that estimates of low recombination reflect a recent change in recombination rate.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/04/01" />
<meta name="citation_mjid" content="ggg;4/4/681" />
<meta name="citation_id" content="4/4/681" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/4/681" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/4/681.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/4/681.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/4/681.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.010488" />
<meta name="citation_pmid" content="24531731" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="681" />
<meta name="citation_lastpage" content="692" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Richard M. Kliman" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Cedar Crest College, Allentown, Pennsylvania 18104" />
<meta name="citation_author_email" content="rmkliman@cedarcrest.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.;citation_pages=927-935;citation_volume=136;citation_year=1994;citation_issue=3;citation_pmid=8005445" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_author=S. W. Schaeffer;citation_title=Natural Selection and the Frequency Distributions of ``Silent'' DNA Polymorphism in Drosophila;citation_pages=295-307;citation_volume=146;citation_year=1997;citation_issue=1;citation_pmid=9136019" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Begun;citation_author=C. F. Aquadro;citation_title=Levels of naturally occurring DNA polymorphism correlate with recombination rates in D. melanogaster.;citation_pages=519-520;citation_volume=356;citation_year=1992;citation_issue=6369;citation_pmid=1560824;citation_doi=10.1038/356519a0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Bernardi;citation_author=B. Olofsson;citation_author=J. Filipski;citation_author=M. Zerial;citation_author=J. Salinas;citation_title=The mosaic genome of warm-blooded vertebrates;citation_pages=953-958;citation_volume=228;citation_year=1985;citation_issue=4702;citation_pmid=4001930;citation_doi=10.1126/science.4001930" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. B. Carlini;citation_author=W. Stephan;citation_title=In Vivo Introduction of Unpreferred Synonymous Codons Into the Drosophila Adh Gene Results in Reduced Levels of ADH Protein;citation_pages=239-243;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586711" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Charlesworth;citation_author=M. T. Morgan;citation_author=D. Charlesworth;citation_title=The effect of deleterious mutations on neutral molecular variation.;citation_pages=1289-1303;citation_volume=134;citation_year=1993;citation_issue=4;citation_pmid=8375663" />
<meta name="citation_reference" content="citation_journal_title=Biochimie;citation_author=G. Chavancy;citation_author=A. Chevallier;citation_author=A. Fournier;citation_author=J. P. Garel;citation_title=Adaptation of iso-tRNA concentration to mRNA codon frequency in the eukaryote cell.;citation_pages=71-78;citation_volume=61;citation_year=1979;citation_pmid=435560;citation_doi=10.1016/S0300-9084(79)80314-4" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. T. Cirulli;citation_author=R. M. Kliman;citation_author=M. A. Noor;citation_title=Fine-scale crossover rate heterogeneity in Drosophila pseudoobscura.;citation_pages=129-135;citation_volume=64;citation_year=2007;citation_issue=1;citation_pmid=17160365;citation_doi=10.1007/s00239-006-0142-7" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. M. Comeron;citation_author=R. Ratnappan;citation_author=S. Bailin;citation_title=The Many Landscapes of Recombination in Drosophila melanogaster.;citation_pages=e1002905-e1002905;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23071443;citation_doi=10.1371/journal.pgen.1002905" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=J. M. Comeron;citation_author=A. Williford;citation_author=R. M. Kliman;citation_title=The Hill-Robertson effect: evolutionary consequences of weak selection and linkage in finite populations.;citation_pages=19-31;citation_volume=100;citation_year=2008;citation_pmid=17878920;citation_doi=10.1038/sj.hdy.6801059" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=S. M. de Proce;citation_author=K. Zeng;citation_author=A. J. Betancourt;citation_author=B. Charlesworth;citation_title=Selection on codon usage and base composition in Drosophila americana.;citation_volume=8;citation_year=2012;citation_pmid=21849309;citation_doi=10.1098/rsbl.2011.0601" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Doherty;citation_author=J. O. McInerney;citation_title=Translational selection frequently overcomes genetic drift in shaping synonymous codon usage patterns in vertebrates.;citation_volume=30;citation_year=2013;citation_pmid=23883522;citation_doi=10.1093/molbev/mst128" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=Y. X. Fu;citation_title=Statistical properties of segregating sites.;citation_pages=172-197;citation_volume=48;citation_year=1995;citation_issue=2;citation_pmid=7482370;citation_doi=10.1006/tpbi.1995.1025" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. H. Gillespie;citation_title=Genetic Drift in an Infinite Population: The Pseudohitchhiking Model;citation_pages=909-919;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835409" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Gouy;citation_author=C. Gautier;citation_title=Codon usage in bacteria: correlation with gene expressivity;citation_pages=7055-7074;citation_volume=10;citation_year=1982;citation_issue=22;citation_pmid=6760125;citation_doi=10.1093/nar/10.22.7055" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. R. Haddrill;citation_author=K. Zeng;citation_author=B. Charlesworth;citation_title=Determinants of synonymous and nonsynonymous variability in three species of Drosophila.;citation_volume=28;citation_year=2011;citation_pmid=21191087;citation_doi=10.1093/molbev/msq354" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Halligan;citation_author=P. D. Keightley;citation_title=Ubiquitous selective constraints in the Drosophila genome revealed by a genome-wide interspecies comparison.;citation_volume=16;citation_year=2006;citation_pmid=16751341;citation_doi=10.1101/gr.5022906" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Selection on codon bias.;citation_pages=287-299;citation_volume=42;citation_year=2008;citation_pmid=18983258;citation_doi=10.1146/annurev.genet.42.110807.091442" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Hey;citation_author=R. M. Kliman;citation_title=Interactions Between Natural Selection, Recombination and Gene Density in the Genes of Drosophila;citation_pages=595-608;citation_volume=160;citation_year=2002;citation_issue=2;citation_pmid=11861564" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=T. Ikemura;citation_title=Codon usage and tRNA content in unicellular and multicellular organisms;citation_pages=13-34;citation_volume=2;citation_year=1985;citation_issue=1;citation_pmid=3916708" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=R. M. Kliman;citation_title=Recent selection on synonymous codon usage in Drosophila.;citation_pages=343-351;citation_volume=49;citation_year=1999;citation_issue=3;citation_pmid=10473775;citation_doi=10.1007/PL00006557" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=R. M. Kliman;citation_author=J. Hey;citation_title=Reduced natural selection associated with low recombination in Drosophila melanogaster;citation_pages=1239-1258;citation_volume=10;citation_year=1993;citation_issue=6;citation_pmid=8277853" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Sci.;citation_author=R. M. Kliman;citation_author=A. N. Henry;citation_title=Inference of codon preferences in Arabidopsis thaliana.;citation_pages=3-11;citation_volume=166;citation_year=2005;citation_doi=10.1086/425668" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=R. M. Kliman;citation_author=C. A. Bernal;citation_title=Unusual usage of AGG and TTG codons in humans and their viruses.;citation_pages=92-99;citation_volume=352;citation_year=2005;citation_pmid=15922516;citation_doi=10.1016/j.gene.2005.04.001" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Kulathinal;citation_author=S. M. Bennett;citation_author=C. L. Fitzpatrick;citation_author=M. A. Noor;citation_title=Fine-scale mapping of recombination rate in Drosophila refines its correlation to diversity and divergence.;citation_volume=105;citation_year=2008;citation_pmid=18621713;citation_doi=10.1073/pnas.0801848105" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=Y. Lavner;citation_author=D. Kotlar;citation_title=Codon bias as a factor in regulating expression via translation rate in the human genome.;citation_pages=127-138;citation_volume=345;citation_year=2005;citation_issue=1;citation_pmid=15716084;citation_doi=10.1016/j.gene.2004.11.035" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biomed Biotechnol;citation_author=L. Liu;citation_author=Y. Li;citation_author=S. Li;citation_author=N. Hu;citation_author=Y. He;citation_title=Comparison of next-generation sequencing systems.;citation_pages=251364-251364;citation_volume=2012;citation_year=2012;citation_pmid=22829749;citation_doi=10.1155/2012/251364" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=A. Llopart;citation_author=A. Mabille;citation_author=J. R. Peters-Hall;citation_author=J. M. Comeron;citation_author=R. M. Kliman;citation_title=A new test for selection applied to codon usage in Drosophila simulans and D. mauritiana.;citation_pages=224-231;citation_volume=66;citation_year=2008;citation_issue=3;citation_pmid=18288436;citation_doi=10.1007/s00239-008-9072-x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. A. Machado;citation_author=R. M. Kliman;citation_author=J. A. Markert;citation_author=J. Hey;citation_title=Inferring the History of Speciation from Multilocus DNA Sequence Data: The Case of Drosophila pseudoobscura and Close Relatives;citation_pages=472-488;citation_volume=19;citation_year=2002;citation_issue=4;citation_pmid=11919289;citation_doi=10.1093/oxfordjournals.molbev.a004103" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Marais;citation_author=D. Mouchiroud;citation_author=L. Duret;citation_title=Does recombination improve selection on codon usage? Lessons from nematode and fly complete genomes.;citation_volume=98;citation_year=2001;citation_pmid=11320215;citation_doi=10.1073/pnas.091427698" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. Marais;citation_author=D. Mouchiroud;citation_author=L. Duret;citation_title=Neutral effect of recombination on base composition in Drosophila.;citation_pages=79-87;citation_volume=81;citation_year=2003;citation_issue=2;citation_pmid=12872909;citation_doi=10.1017/S0016672302006079" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Maynard Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=1;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. E. McGaugh;citation_author=C. S. Heil;citation_author=B. Manzano-Winkler;citation_author=L. Loewe;citation_author=S. Goldstein;citation_title=Recombination modulates how selection affects linked sites in Drosophila.;citation_pages=e1001422-e1001422;citation_volume=10;citation_year=2012;citation_issue=11;citation_pmid=23152720;citation_doi=10.1371/journal.pbio.1001422" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. McVean;citation_author=B. Charlesworth;citation_title=The Effects of Hill-Robertson Interference Between Weakly Selected Mutations on Patterns of Molecular Evolution and Variation;citation_pages=929-944;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835411" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=E. N. Moriyama;citation_author=J. R. Powell;citation_title=Codon usage bias and tRNA abundance in Drosophila.;citation_pages=514-523;citation_volume=45;citation_year=1997;citation_issue=5;citation_pmid=9342399;citation_doi=10.1007/PL00006256" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=M. W. Nachman;citation_title=Variation in recombination rate across the genome: evidence and implications.;citation_pages=657-663;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433578;citation_doi=10.1016/S0959-437X(02)00358-1" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Nakamura;citation_author=T. Oshima;citation_author=T. Morimoto;citation_author=S. Ikeda;citation_author=H. Yoshikawa;citation_title=Sequence-specific error profile of Illumina sequencers.;citation_volume=39;citation_year=2011;citation_pmid=21576222;citation_doi=10.1093/nar/gkr344" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. A. Noor;citation_author=M. D. Schug;citation_author=C. F. Aquadro;citation_title=Microsatellite variation in populations of Drosophila pseudoobscura and Drosophila persimilis.;citation_pages=25-35;citation_volume=75;citation_year=2000;citation_issue=1;citation_pmid=10740918;citation_doi=10.1017/S0016672399004024" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. R. Powell;citation_author=E. N. Moriyama;citation_title=Evolution of codon usage bias in Drosophila;citation_pages=7784-7790;citation_volume=94;citation_year=1997;citation_issue=15;citation_pmid=9223264;citation_doi=10.1073/pnas.94.15.7784" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=W. R. Rice;citation_title=Analyzing tables of statistical tests.;citation_pages=223-225;citation_volume=43;citation_year=1989;citation_doi=10.2307/2409177" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. Richards;citation_author=Y. Liu;citation_author=B. R. Bettencourt;citation_author=P. Hradecky;citation_author=S. Letovsky;citation_title=Comparative genome sequencing of Drosophila pseudoobscura: Chromosomal, gene, and cis-element evolution;citation_pages=1-18;citation_volume=15;citation_year=2005;citation_issue=1;citation_pmid=15632085;citation_doi=10.1101/gr.3059305" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=H. H. Rogers;citation_author=C. M. Bergman;citation_author=S. Griffiths-Jones;citation_title=The evolution of tRNA genes in Drosophila.;citation_volume=2;citation_year=2010;citation_pmid=20624748;citation_doi=10.1093/gbe/evq034" />
<meta name="citation_reference" content="citation_title=RStudio: Integregated Development Environment for R (Version 0.97.551);citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. W. Schaeffer;citation_author=E. L. Miller;citation_title=Estimates of gene flow in Drosophila pseudoobscura determined from nucleotide sequence analysis of the alcohol dehydrogenase region.;citation_pages=471-480;citation_volume=132;citation_year=1992;citation_issue=2;citation_pmid=1427038" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=K. M. Devine;citation_title=Codon usage and gene expression level in Dictyosteiium discoidtum: highly expressed genes do [prefer[ optimal codons;citation_pages=5029-5040;citation_volume=17;citation_year=1989;citation_issue=13;citation_pmid=2762118;citation_doi=10.1093/nar/17.13.5029" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=D. C. Shields;citation_author=P. M. Sharp;citation_author=D. G. Higgins;citation_author=F. Wright;citation_title=&quot;Silent&quot; sites in Drosophila genes are not neutral: evidence of selection among synonymous codons;citation_pages=704-716;citation_volume=5;citation_year=1988;citation_issue=6;citation_pmid=3146682" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=N. D. Singh;citation_author=J. C. Davis;citation_author=D. A. Petrov;citation_title=Codon bias and noncoding GC content correlate negatively with recombination rate on the Drosophila X chromosome.;citation_pages=315-324;citation_volume=61;citation_year=2005;citation_issue=3;citation_pmid=16044248;citation_doi=10.1007/s00239-004-0287-1" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=W. Stephan;citation_title=Genetic hitchhiking versus background selection: the controversy and its implications;citation_pages=1245-1253;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308100;citation_doi=10.1098/rstb.2009.0278" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=L. S. Stevison;citation_author=M. A. Noor;citation_title=Genetic and evolutionary correlates of fine-scale recombination rate variation in Drosophila persimilis.;citation_pages=332-345;citation_volume=71;citation_year=2010;citation_issue=5-6;citation_pmid=20890595;citation_doi=10.1007/s00239-010-9388-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=The effect of change in population size on DNA polymorphism.;citation_pages=597-601;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2599369" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=S. Vicario;citation_author=E. N. Moriyama;citation_author=J. R. Powell;citation_title=Codon usage in twelve species of Drosophila.;citation_pages=226-226;citation_volume=7;citation_year=2007;citation_pmid=18005411;citation_doi=10.1186/1471-2148-7-226" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=B. N. White;citation_author=G. M. Tener;citation_author=J. Holden;citation_author=D. T. Suzuki;citation_title=Analysis of tRNAs during the development of Drosophila.;citation_pages=185-195;citation_volume=33;citation_year=1973;citation_issue=1;citation_pmid=4207975;citation_doi=10.1016/0012-1606(73)90173-5" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=F. Wright;citation_title=The 'effective number of codons' used in a gene.;citation_pages=23-29;citation_volume=87;citation_year=1990;citation_issue=1;citation_pmid=2110097;citation_doi=10.1016/0378-1119(90)90491-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_author=R. Nielsen;citation_title=Mutation-selection models of codon substitution and their use to estimate selective strengths on codon usage.;citation_volume=25;citation_year=2008;citation_pmid=18178545;citation_doi=10.1093/molbev/msm284" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Evidence that Natural Selection on Codon Usage in Drosophila pseudoobscura Varies Across Codons" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="twitter:description" content="Like other species of Drosophila, Drosophila pseudoobscura has a distinct bias toward the usage of C- and G-ending codons. Previous studies have indicated that this bias is due, at least in part, to natural selection. Codon bias clearly differs among amino acids (and other codon classes) in Drosophila, which may reflect differences in the intensity of selection on codon usage. Ongoing natural selection on synonymous codon usage should be reflected in the shapes of the site frequency spectra of derived states at polymorphic positions. Specifically, regardless of other demographic effects on the spectrum, it should be shifted toward higher values for changes from less-preferred to more-preferred codons, and toward lower values for the converse. If the intensity of natural selection is increased, shifts in the site frequency spectra should be more pronounced. A total of 33,729 synonymous polymorphic sites on Chromosome 2 in D. pseudoobscura were analyzed. Shifts in the site frequency spectra are consistent with differential intensity of natural selection on codon usage, with stronger shifts associated with higher codon bias. The shifts, in general, are greater for polymorphic synonymous sites than for polymorphic intron sites, also consistent with natural selection. However, unlike observations in D. melanogaster , codon bias is not reduced in areas of low recombination in D. pseudoobscura ; the site frequency spectrum signal for selection on codon usage remains strong in these regions. However, diversity is reduced, as expected. It is possible that estimates of low recombination reflect a recent change in recombination rate." />
<meta name="og-title" property="og:title" content="Evidence that Natural Selection on Codon Usage in Drosophila pseudoobscura Varies Across Codons" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/4/681" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Like other species of Drosophila, Drosophila pseudoobscura has a distinct bias toward the usage of C- and G-ending codons. Previous studies have indicated that this bias is due, at least in part, to natural selection. Codon bias clearly differs among amino acids (and other codon classes) in Drosophila, which may reflect differences in the intensity of selection on codon usage. Ongoing natural selection on synonymous codon usage should be reflected in the shapes of the site frequency spectra of derived states at polymorphic positions. Specifically, regardless of other demographic effects on the spectrum, it should be shifted toward higher values for changes from less-preferred to more-preferred codons, and toward lower values for the converse. If the intensity of natural selection is increased, shifts in the site frequency spectra should be more pronounced. A total of 33,729 synonymous polymorphic sites on Chromosome 2 in D. pseudoobscura were analyzed. Shifts in the site frequency spectra are consistent with differential intensity of natural selection on codon usage, with stronger shifts associated with higher codon bias. The shifts, in general, are greater for polymorphic synonymous sites than for polymorphic intron sites, also consistent with natural selection. However, unlike observations in D. melanogaster , codon bias is not reduced in areas of low recombination in D. pseudoobscura ; the site frequency spectrum signal for selection on codon usage remains strong in these regions. However, diversity is reduced, as expected. It is possible that estimates of low recombination reflect a recent change in recombination rate." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
    <title>Evidence that Natural Selection on Codon Usage in Drosophila pseudoobscura Varies Across Codons | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7fX8viBAygjqDpK7nXstoMV4Dfb3qN56tyhvlYm5wts","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459830","apath":"\/ggg\/4\/4\/681.atom","pisa":"ggg;4\/4\/681","ac":{"\/ggg\/4\/4\/681.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/4\/681.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/4\/681"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/4\/681"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/4\/681":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CckS5LdeG\\\\\\\/RQM0tmggDK+9svSAc7JjwtFxM2XHIUzg5f\\\\\\\/BKBWRbDtdlwKmmjWg0w0j6iZ3Fg54XTV4NQ5qwtMFeuLOuLHIMe4IpYh1AXv+WdjVDyUmOhvgxggDSAwLYESKA7jlIuptQV2EJ9hZQKvQouPqGApPzmYmg3EiG5sv6+7+8RcIcss2iKrI+KPQ99PMrbj8MT3yx1f88fo9S8W8YXeqqh7hsr0xmDOZgms53AsbOTkldsI9cs++85pG227Eznw3e+33S4ax1KdwmYTJVGX5q3J\\\\\\\/X9sHNbLzB9fwo77fTC++KgLvmLUZ1o7f6uYr0TYQBoXBe2Q7Z8m5oVbkcmwYF3kuKbXlNIVGe3sBMxBPjf0gCt1DwJ1NG5CUV9kG7lHyriyRLwsNA7dTOPTUCiMc5jYliQnVtfG7V+aFz5uDrCRBQP\\\\\\\/gMPnBSzdzCpjDS3zNYUku4daHQo\\\\\\\/JP+vtU7LiFRlBvYIibulLUoxqyPi85syfHfaVHf8nnQWk8pLFpZR91o5LKvcCn1YPA1vmtq0rWDPw\\\\\\\/AZTrBlJq3EzTSUcpuSDIc+islBYLtfOvQbuiUr54N49jcDsn00GN4wefqUWHtph765Ao\\\\\\\/zoay5viwZJoyVvz5xKPBeRF9NRMKCOpCmT3x+D85rleKBBwB1yUQjTxU7UDjzqsQAuPaYrR6kqfmompeIo9PAVDpTNZm1Zn+DVTdhv3LyBrkTGPdlenyxcaJA21k2eJZ6tk3JQaEq5EmWrL13c\\\\\\\/6uSOBCSAtJPR0zEOuqVAXaFUPWJEke5kqOAzF7xuQGu3maDQ8DpGx1pa4S3W7u+p2iPdDMZeioAZXiSHG+t3zxr4QDaV9Gz01Id\\\\\\\/g+l1LwqKToSjE+nZSKqYeCfueHCMRnVMXgElHCvVNdja2wtUgAVjJXGygslB2QsCxU2qgY3dkeOTNxyNvlGUtAkXZ93gjv8kRZymkoLfy6r2iaqZe7OUFfmUalfOahc+m4sVMZa4oMGfmNkO5rheBdzmmL89mw3+1UF2R68tIkLfgwbXCToRBYFq99sMwHVpsWr5hh4J5XEmoPZFqEqgvlbujnUyckOVcgNGmYxU2S5x++wRHkIbqdR\\\\\\\/LbqtSCIQZhQwDErKagD9AMANDOXm\\\\\\\/yDCVH4sRASjf2n\\\\\\\/5XeTV8YR9jtflAp5gXjD\\\\\\\/YcCCAeDDh7Le6T2sWo4yDdIorRV7GWzrkn7F\\\\\\\/4TZgge\\\\\\\/efS\\\\\\\/crx3QKGizpW405AJd4aHz6i0J8nYxoI7XM3MfSZDsPGSMbD5igtEBBzr3Aep\\\u0022,\\\u0022iv\\\u0022:\\\u0022gqsftpaww+HJzU03VFOQUQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220407667db11772378b946c023f982c81e5fb4f17cbafea1e13c83d6bf51b2abd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022l1AP5FzVPVcUngFz7VTzh9eS7SrYxUTJY54kTkppt3EQJhpsUkDs6Oz79Uy0b7t9pBVk+kw2\\\\\\\/fsAelg6OMQ\\\\\\\/JMt7od4MVLps8LSjKMx0xTQXzvz+HkjQ1YiKGpyDEL7sqale1fXLIjJcaOReHe6bTRcx0At94q8766q44XCBxsdex1B34j0njYoFL8EwqzMSHbenpmSFR+D2LZ0CXC2wxv43n2rxpy1wYJ\\\\\\\/5LJjZoU4+juoFtxcc3rxaxVO5ny6HadW\\\\\\\/on\\\\\\\/9lbCQG8rnaX\\\\\\\/Ut00OcBP0JXDtDcXB7DlcnsLEGadu1xw2xGIUNyVD0PAubBNvgVfT3orXGRo2e0JIbDx\\\\\\\/o\\\\\\\/BDPPUJLNC2JbqBnBG5jYY6qi4nPB7G4TrPpaO+lKauN19F3LZ1EMng0qMxBq9XmGELwACR7igY68TyPtx4D0zgr2zOaIY2NshO5uPk7uosoneEw9FHlKRrch4zqPP6MkPulubIVOEFSW8pzpzkMbZ7PkLaJEeisN0Y25TA2B4lrm6tr8Dev+FuUSGVJfIqrG0JcEZuTQd3Sf3l+9k7RGgkAxEKrUdElnOGSeEpoVqCWWyUcO7HHE1LwQMcQwhekQpheGC4NzUOgxdEktLM4AjXtNgISB9qG30FZrcLwCqItkl\\\\\\\/2VpOsTkevcdSqcnY53cVVa2XZScHXfqeaG9LIFW0aWIATtX6N3v\\\\\\\/R5PzLtWqMausuPhb+BPgGHB9gRrXzOaUlMvdBREYJEBWyuQi5qay1BbU+BXB2Zt9axctHiRJ9rlQiOrOaPd7kzbK0fc8aodieS2sgdF4MWtVDR1aR3un37nlbua4v9j34REmDR5ynuMiTI6uxwWWq0bLnBE\\\\\\\/44OtQ7z8XlAa++WJ7fpl27pHt2QoUUFRPMu4IeBBf8H+qopFamJ2budmtcjgtwUfZBRobj\\\\\\\/EhhwBAdWdEatm+ldr2D0hRRbG\\\\\\\/6Uy2OSxRDxx514I2lgQVL6mN1hipAYLYSp87AMxVsn0r8Fl95\\\\\\\/gXYSoyzcT9+NVIF5RUVnuyGv+iyjYlNPj40zAqfDgJPygnJV\\\\\\\/LduCPF7puhRSubO2wLcdAwxK78C3QxHINrBQlUKftpiCkjGgkMSuj\\\\\\\/c7HSQIvnzDvtBBEvFHPssUfarqlQCPU6FpWiTnqXm0AkvDHePrErOZ6GONVYyWA9pMgD4KXvIzTdFByl+PsCe\\\\\\\/QJyCn8t8XhVIMxL31QsfsYfx7Cch44ZJZzkYA3gFMy3sSFu2h+Llt75IhMfEPpEgJ+ZYipjA8qi4yzVqNBCaBZn6T9q5mKymuyTE+l+\\\\\\\/yFPixEiHelcdvrf7tb++VUcxczAgfGrkEh9XZbug7TO2+4LeY7WH3jWRiajFDgrfeKyIDvLBIb1N1uGxfh6ioHLNqcYR0kfbcgAAGKT1wHGX6uALVmvp\\\\\\\/5IcsbMa0Ysa6P6t5HoFY9CPXy9uqKDPm08zLflUO6nYAKXdD76ecme+Ld\\\\\\\/UffoLAZAxOSG6fW8JhBnEDfKRvEMTOrOaJGTfBlG44YGVYBcRwX9fz4rLpUv1cR6Ex24z6vtyxtZWHdLTarHq+na0tvGZCUejqdRkJgcG+VswnL2jc+vuhRh66ZvslnjO8ILdxB6jCCpvVZq1c5SwAMByECIEyorx7sD5A7pMLJ\\\\\\\/8ooucAN6voYeQ2se9A6reh74K5hFP0KV9FhCBSHjuWhidbpDeMbg19nn15MHt7518Q1RxHqHMZQhbXccUvmx53jkVuqGBo3N8S10yBwj4wA+naXwjn\\\\\\\/6RU7RqJ6GfNAiUOSTewKDEA89NRPu5GxyxxsIY30ZiBDxcoRn\\\\\\\/huFI2vvEkED6F31nO8arYqR5QZ4wkRZlWlss7gyVI1EG8e8kqgXZn\\\\\\\/zq81dJYNQvdamo8o5QtmnZAPhT7kxRKcWIv1hV5VVcsw4RAl3QeO2SAnKvxOAhRshTd8L4RoeNk+Pu50mTmTZTSY5SJ3sHq4bhVZBGZIbG2t2Q4mFbK6DqLWy8C7SCsXmumOdtoyo41mwoe63OW2PXaPN5\\\\\\\/VFsh3fmaEJbgpYd8M2cpcm4Rh\\\\\\\/3lyTVHLuNjK1gtcNSpBmWAHUzlqzvnpgA9TlRRG+YgW2GMqDhqEybOYtl1L57aABKwD7bmNBADyxytmpxOiBixCOPTy\\\\\\\/BMGfiOfnEztA\\\\\\\/XkdL9Efg32eJHGhP8m2EtIXKJZ1jEMKrmgodZMQFS7JAlVLhY4tRpbLc0KG7ytwFR1hc7eOlxVjwsKOJ1FrbzJnyjVkj9uK5J5B3JedY+9c9x6Sxx7+2J\\\\\\\/SKQMp7qlSgM3f7oT3078dlONqHpufgZEfQiz24wNrx8cRS9vB3OeTCgR5JNDSEf7cmdTPkbZd86J3IJwDaywJ16YrGWp82OvW\\\\\\\/HBMkzGA1W0OYdNeqCKF3lhGN6uRHIuj7EpJwGsauyk8LOLNAG2fetcP2KSwlYabUeYBRz599tEjbsFx1KgyKodwiYn81\\\\\\\/SpRdkTLpfyKTCwmxelrOwTL2lw\\\\\\\/PFXT+9OJm95IF7wRpuTAVk73JM8orY3lF0ovfrG6GIpg9PNWGkvXoNXoKFPZL7pqXOuRLIDRaiJgpsAZJS+ZVvoISSijJsywzjJnJ+60DeKsPELF+Up2dQrCeAZQ0WbCNn6Wnr26yiTEj3lqZqpGkl2U6x0KDrFQn0R67dSEyurSRnx0Xicm7BYCvjDm2n\\\\\\\/UaP86YBYHvTsHQ+Z3I8BU6DO5rtwGTPClEIj9glBk0Y\\\\\\\/xVl5Lm+PMxNSn0xNoFplZ9xYbgYKf1gmhqZE3Bs\\\\\\\/kgd0q03P8OlFaWBSj6ogt2FG\\\\\\\/9mk9VI4gVFyqnqCpomdI4IbMCbW0KeU1oTANAEX9BALxcoQHGEZsE7WlNl+7vWBHyn\\\\\\\/jtP3iSnxbdMqlZE7WnF85n1c7p4sIpCfDiUJRlFg1buA2XUifRjqYrtFbolXn4Hglq6JpEHUeC+1ZbY3k\\\\\\\/rA3QSkvOD5nr3p\\\\\\\/0P\\\\\\\/POw2YsvUKXpE9NMW+sDSVWxg2JGe+rn2kwqv8KP6CuklPzt7+Ep9+MMuACS3GYKAdvNh8FV4Urdy6XRxxmnFR6s+scQ2qIK3BivHIlTrMXOziKXJ8gQIFc2b+KCMM0SfdDTaBNhAmYZJ05hWok\\\\\\\/PCl2Y70n7ne0EWZXqOLnYTN\\\\\\\/biZ7BPzkgiOT6lJvZIKWiqH2qYFzQQRSW2bH0hF16HYP\\\\\\\/9PYnMFyy4eNYfzFo0FLYIzLMNCNlsB3BFenBn8FZwR39gr09HY3ZX+7KzekfjWlMo4ZM21xUOyR\\\\\\\/Yr9jFhy+Of7PxfKmekTG3yVd0dz2lGbaf3TtfiHGyvN3CqLi4sM20ToelNLh8qe3sWhupBTtOz63epvc05m6ya4wF5QO368tKvuVNeLcQodWrmxsQOHCx0hhBFuEQ6latcZ0WyIgwRKSxk1zAXXb0LL3UBPMjd5d6brdGZ2CaJmJ+n6f9FEczD3zoYS4jJu7arx7H1u0R8rEi1OMKXl7Cgf5SibIHTL2nTQw\\\\\\\/dtYzwAxUSj7MXzcJP36s6\\\\\\\/kshx8UYzTCaVyaCZftcrBU+EKOh8XT4EaahI6hlMVZWR5wGbXBD11BT0XMIhuZ9ZJsH1Er8qUYoFha9bw2JttRvK24lfPu24x2vpTlZyPioh9BGwun1Gx+LOg4AKzB5p4hdW1ecIH9vUem66\\\\\\\/ZbBwi3qIQMjxTQ4DX5Xkrja\\\\\\\/s9F7EiSQyu3gNXudfqHm7LBuRr8n4MbZRCUgHtIXo00oCmKp4gxR6PSFI3Fg94jmOk+yMMUAGhKOGwBlGsLD8a5vrw9FenjLoOj+hocvfnMOqxBSthQtmGhZbI8ZuiMx8THurchaYD+aWS+HYLu1DkosEX8qtJNrWM8i+Sum3lRa9s5C09u8Yk2R8V8puIZ3QpMq6guqBlZfFR4xS4ji7k+U0Bspnu8ZtCbbglLjSeGnppj0tV4ddHtDJMUUDkzi9UhHZWB3ut7BMIO6inA7ri3mTDOLBwB15y94QTeI+mdyttJt78VyLiG8Jz469MNo483ltiOcbJLTcbfapFuWAGStcq3HegYgV0IPUdEICVPBgXqVG7gibqGxWhPhvYU4wPSeTCWTnE2BZiUA2mbXDfBFa1LYFi\\\\\\\/\\\\\\\/yX44zmcRXgAsvyex\\\\\\\/Lp3tfCDJFLErmSdkWDBVOolUx1csZngRDKp13GrQlH2VTyA2JGA3sBdUNogvPzzdH8lh6G2yqaF6+VVaYr4Wn3Di2hQLJfxRUIcQve9+R70RfWnX5DwA2Qw1gvC7G\\\\\\\/yUZjy\\\\\\\/2PMaumMtkD0rS2l\\\\\\\/8ce2hZynqFCmU0OeXffyPbRkFyi5HHqkZ3VOTtJex\\\\\\\/akgkd24tt\\\\\\\/nPR4XAd\\\\\\\/ojqf+Sj0TbhRjK7KyyeNcnlJJfuDjNmjDx5dBNXaZh8X4nTkv20lX00mJpsnCrUdgQrOLK89rNl8g4gEcwAz38YjRuRuZ3egzpL7j+jzuSb27Vs3jp7RRIlsCjpZmDKK1Qdlhw1WBu127qOoE9z0jfPwbtgRUrP017QKDQSy3pYnktzh1MzfCbE0507ZRQk+y7yl4QCO+gqNvD\\\\\\\/zpXCzl1836XzYSpwIpBJjx\\\\\\\/j73bqNEuFQAMDdYQ60DRIeUm1FVQbqfANrJouioNvk0nFzChfyvjGkrVxxfoNn\\\\\\\/nJRt3yWC2ZpkyNxbwHwRbKzFckuF+DC3tyDTdlR+SHMWpxW0SJiIcMnQS22iilBaZ5IeJPVyFECwOYUTZNX0K5gEf3oXvj5XC+r1nR\\\\\\\/BqbzdSsvCwbhf83KqjJmMoHGwIn1FhxqvlSxbmIEHVLZnu+h\\\\\\\/\\\\\\\/zczGJGMv2gbIGiqcsFCBrNnUifxTD2kz6ScyNsR3HFnGHdUJR0pthqk3ndIKqM88dBo51PKxvRHe2EFFNtebhesBRwYZRpza4s\\\\\\\/vPvWJsZnNeMaEiA+DU+8O6PIAkXuxoH+HqQ+xKTb6Zal0BM4vCd+xgPasQOKMYEvzr40vlL9XyPr9wpGpdfUi1oXsFpHR+YLuFQEzJC8AcWhMWibbFKF4bxmtGAzjakoxFj6Ffgm3DXGPkHe897tqyeHMd5yEg9NeHPW9a6qxnx2nfc5XOnR8wjEiEQR1wWOOJDdE4Ww9OhZklw80sNle+5sbBx+pAefLoWNIdSNv7YoipcdzSdI6fyMDVp3ETisA8E6stWf8RuS+vKVFNr2TGSvqFDGu5fvZJY0QfHLUmdWLNIgZJKnBRE60uRvN5r762xwc8WOEq6\\\\\\\/qOzTfQrnH\\\\\\\/GHd9BUBOHqZuNMpC4X9b28Dq2Pt\\\\\\\/tR0qGVYFhQ8TimzFcm7qrNaE5T9fBL7wK\\\\\\\/lahEc6PT0bLqdq56Dyz9qFApXjWHlB8Q5PKE6RIrvgbImp0mWHzqhzPSCNU+CpDG296nLLn6r0OZLjoJV6T2M+bDBfW0rWPvgprhRfTnAFJMgzdeYaj1tO9EkqiEHE2sd35z+acjUbBWrUkPMz0nm3iIiaY2tePxT\\\\\\\/qrr1Fa6JiDvHBQ17M7ievwaJhsb\\\\\\\/0MpcuVSJGnEdSRwIlaNbEUVo2cVInhfEiKrwu8sQ6eW7KmoNt0cf+qbIIwfze0USAhhTDHFIGTazN\\\\\\\/dEn\\\\\\\/pwfZ4wztPU7HodlVN3VE8eQ3LHcWfCeuEgFjfYwbQ36SunfIDx+hB3DQvwSBh+HCfx67vKMzFIjNEy1O+G9WiuLkXaLYnAwVcNRhftAaiwL1gwf5qZp6hOGbn2pv1Qv52yTk5yO7CjYPZTOe0kKuRMWmOUqeVcnsEbiNAWu2GoQgtUARrRHH8EPfPCdEgl6mN7Id\\\\\\\/EqUyMeu5MqDZnLae3632lmjQL1bBfUfYg1GVvZWkVc3l2MwAAmNHBFCBGhuEjSTbHjakjBuC31FkCfgF7f35Qkv8UBVpIGwHjT8IjCusOlmAX+smDS9h9+FhcAXvLR4Lpc5gy6GOCGyrBLK\\\\\\\/+LjVSL7XgbCwj1YJMNxbHg0AijzGBTTOOJRHbV\\\\\\\/8kPFbLdxc+jCF3NLyhqFRUXNeBHRkbiF9hJKDTJI6g08R9\\\\\\\/wX\\\\\\\/7IHoCHRuQQQhYvzI+s796xdHkhJQpAtkB62P3UazjtwYgC\\\\\\\/r8cJzNCtezwJNq8SUtZ6YdQCoyUEsFYx8AILNcRXARc8f7KspFWRznSu0gb3Sxk1RG+kbqrq6X4nD1Lsa6WvnlzNNQ10+NZ1IZcbFc4mq2lYZybzD0C9XYWaCxjQZKguVyrnRE7fQsz5iPv7WUgJi8OIwb+w3O5j5JYyX0j8FbrvVW\\\\\\\/GchdrS\\\\\\\/Blhf+s\\\\\\\/XXf+cmLWXvkz7utRI5N4QcVSpogVIzO9BF8vuHR18sod2G23YqaL1yLgxa4ViDIUng\\\\\\\/+tAMfptrGzbXVfNePmRjTg3ZToCMDUdMUumMYoUvz8sCDZ5OiQVCQtC9SnEgCnCy+LBIuc1V9XpFUx1ojYMq7XvFy7bJvthKipfon1jkEurDgZv2+osf7PHhxHy4LSqTWIvjTmdSYnlxmK7DpfNfp5xlMfD+SHi+jYyA0p2CidnPOuJd9euLeCi6v4NxgtI4e5hv0IZRX2gqEyRHoZgKXPcMjSlZ1wKpZ21klzz3vIC3L4UamrIl7IwzpKvhn1mlUZNjV2+UxUFr10e6xNIfd+T5z2sAsgDMWygJkNYSr4oBsM1Zu2gDTsJCAKONDMqEDUBFQjY5+z\\\\\\\/dfQmQzXucqPXt+MEdj5qED\\\\\\\/HXKRMzuBx64qiPF4FI86HPdM9nGRLHHe9snf13RWCRjXoyE\\\\\\\/znj+yU4jq1SuIiz7qk3Ow3dWqpQLt0Fif4UhmOU110dtnIW+l+pfaPqQKgKgVhp+kHf1nvd6THd1hRZtuQGNUSZ\\\\\\\/7laCjJmqlHvqC1LK2Mv9B+2\\\\\\\/dfgpXGoJuHM8ba1TkOEFebzxARqPF4vdj4vAhPW9boJXW+j1zh9ikwkSSb1pdsRUqX3brcMsIChnzj6hmxwxREoQYiGtSPCA1r2vuiO1uUiWlZ2wj4xva6RN8guxEJ9s7ZFmnpIyh7bTQZ\\\\\\\/xrzxATl5437xvm\\\\\\\/3umsuh17iALs8dqLi38dSZgv2SHOpUwFvgvFkYfFfJGzzcx0vBpTFuWqER5a5EmE5Z9B+NyrjVoPpGApyJ04WKbTsBp3QfdoR77ty6J5nemI4CElixTPdcmxGpe2ec87cI6KYySCmzlqFvwkWKHo1q8aDO5ELwWXNhq6TA2n5Y3UtezdBukTkgsD1GY81GLScogGT9AoWfOg7+hHHNrIta+r+8RVUQrcTTeTGKkFRsIZTwzj2QCQi\\\\\\\/jM0fwjJa1YrT9g\\\\\\\/8RThhM+hH87ozTIqwAaqaBMqEkHKdHeO5LNhV3Jt\\\\\\\/mjAyOFDPArqZYlF7Re1gZIYjmwjzVbSJYITesXW4OMtWq9Giojf7VqKf7h+yIDfFEmkEHmUILmgLF7CM3waKNLD5ZzGjW4PevxsHQDc1GDMiheCaSDivYhLKW2b7RuJrk141G67lzL04+Vq4mq+BJEohfSediX33A7qwzVXnZ1XXAxaIdR9i93+PAgmHo+qhnMOFhvAgX73yLnwp\\\\\\\/gDJaLnWBKiVbMTgntfuv3nAtp+ofN3tqhNXAdwGB5xiClm8X1jyo5ZR7NyJ3q9AOadj3aaGH348+oUsa56e0kWubCk7uspddCFrM9OzLdhApIizxtMImht4qcrLvHk6u5ZnpsTRc\\\\\\\/BoUyJh881HS7OHpZ2ZT\\\\\\\/eQYHCxDLcP6ZTs0HwIDAD9pXMCEHT5W3bMFrqhwjYd9lGorXJdPCHueZZuAY19Xs7B2TCrbfo7gb1dNGwJe9YvlCR1Yo3CwXWGOAIGq2TbtSuYqMOvYMrEuSwXPSKv2kyGb6+Xe\\\\\\\/z4w+exfeSbb6QiM7ei6wDJqC4DoeSGOHM5g2NJMHhShbiWq9GmW8ojuKDY6dOgvHn5JQvLFBHEg84FHr8kclyyFLkcJvguJFgULo3WF8gB5eY7TFkSgWoBhDOQDHFaLl1eMFhRVpcH7YItlB9pGin8WqVg59JWWynhsjaDjnUNv0yM2CU3dzdxZ0bnydnHrtKI+cDF+056sbccjMpZoVaYo5M3M4iWaCQkm8yFyxTZVeA1WCjDWsJE5mKu1VmbEjYnABooFjcbL\\\\\\\/EX79K3YAt9h3RCXyQLZlAFnr7loWPPnmYBUmJ0pesNihD\\\\\\\/i+Co5LTiUvyB0MpJFvvp64ftCIUq4IiL92xFxKZrFtEYiMXTQ+iR+Rtxh+kMS32QKVG7U9MsA\\\\\\\/MuZuJ65Vt88bpznh5poFQytwZLOfoaiwVtqzu\\\\\\\/GLfrCfZvFN\\\\\\\/zrsT+I1xGtKeuSt4FlIODHWAbThZS0lX2d3w\\\\\\\/hnx8KeKL0AlBj0z+5xIoC8g5MjKEjfSqQlSKXugXFtXgQyh5MMIlbT595IS2TPE2aSJ\\\\\\\/2flf9KUXqdb4wgD9bd0FnEMWqu4pw\\\\\\\/MIq1wQEmvbSHCPLb4TMaLyR4ERnWtuanUjk4orEKrDv7Sl9PCEGVpCcTXkpeRE9K\\\\\\\/H8S8ly4aS08C+\\\\\\\/nvmwxVLYiP10mcSmAFEOSycn4KPv038D8brqwy7HaVYRAA7AfnedlJ9xAMRzSVMymCMsX57uBh29t5wvL8U+tBZmadxJ5gbvMfnu98ZaIkE5oisTchQMT4tWyG2VbFPNR\\\\\\\/fu0vbnePpC2eAjej3OHZyMFNi2sO+E6RESpzwp4FUSEKt+WQbMHW7Zbx5ykrSrEyC+6sgZpij\\\\\\\/GrOOwXO2+1Ld8QUvMipGTV0uTCMJd6r749VSfRSFE7Wr7hfBDWebwj38ySmdHGN3MqTAcB6xaBqaLPOWVp1Mr\\\\\\\/n3D3bFso0xiEHwZOBaUmq5AZRwwhYVWZ+MwO8Z7zNGmYog2IsKI300fZTr\\\\\\\/jo9C5\\\\\\\/qh43UtZSqFtGsUAbo197QEfUyexsPXIFH4INrMk4G4CHghQ2rwKohBOOZgWrzRN2tL5tDpSY1JCKKwl3eXoLjJH7KCR6ZbBLKULgpCbDgb9eYjP8bT0mkccZVCV8Aaao7qignXuRb2fwSU1JR1GboOMSRkASO8FBFXKDulmJ7KcDTwCJe0kUudbZMYea4Fsyhn8N3gxwJ7gga3hMXFXLgUSURMGik\\\\\\\/pSUptT6YHFNnJ\\\\\\\/kWgjq1TXF6dyMZ6uBwQEotTMlGk80SaM7ntK5We+b4WExA31Nq8zn706wATVT26C2LjIgtAZGvbmwPqAgqCMQXuvyRnE6zSB8sy35Bcid\\\\\\\/LLLu0XNfWlHQ9cG4Vp6hgoms3HjvWexFFPBbZ+mGFH82ANam2ay2d0oS+qRX+OdUWTVpPX3VGsfacVRniuaJOuWgLyPEZWzgNE6PIkK5b3nFBdO5rbKBLDsL+yr7U6S5wBf560WS3KBLeepA+I0WhvdPIG+Oipv+ITyLdb6L13JpFEGdkTra154TMq\\\\\\\/YIMwCCbefIySFYW+f2mlRbyNHbpFze8s67tCAu\\\\\\\/mwocyVavUmnGOuJSBYgDMB3QGn4vlZHa+D8CGlf3L90GLrgDohN1n2sPykw6+iI3Y2b1VubHrPNN6V5R6TIVTNWy9513i6l0zaCEqt5hHLwjDsTcD4k0ffEheyinXr6By9Rv1McNIW2+EH4ABnTLgQwPmLloR93KrJ9EIUwGX\\\\\\\/o4eyG927ECkg4xuzBtEFERVMBbIIBCtRSzPcuCnv\\\\\\\/\\\\\\\/H6EhqrWx\\\\\\\/dKQkPnwc+5aD0GWFuMOD2+0OPo\\\\\\\/SluHI6TsqrChs5HEg5wmG2HL0cyt+SpFEOtfFFWdMK\\\\\\\/XxZo0hjOjIewIAJ62mpkOJmnMq+BxAdMtPjAemPI7JjXb+b8rsQ1XIfuzd\\\\\\\/Cl\\\\\\\/jra43frZnT0YTcRUajWAbkAPSmvhbDbDoHeoDM6f4hCwOpRQ3W6fHx7+ka9NqTTxSbvoWds0NH50KysTVXDLiP7w5B4BYFE5hgd6cwiEXM58sKybDLCSchYf49\\\\\\\/GNW3jMAbpwxW0DNaCX9wM8XjBCT7JyJnYfX8QpgMrtmqQOYDrBm32amzxFMwMBtVqE2EWN6D3jmBBl3L7Llc7SMflnHh9VW6sRaCfTlMuaS+aAMpBjwXnJQxS8kvP699xOwZAbgVtM\\\\\\\/CeDSXY+x9bBNoNJ1YLWNam0BIGzdhzB2CCjZ8E2h6mr6oGLSGrWPJGPY4FI9qwiG93l8aXiWKRfCq7dIgXRZtEIIyW7olxRQvF2VpIrrcULarFXxKtnGy06zxc7Gif71CCmzdG1t7I+ZRxV96DR57aJIfyVp6JIJTSCb6V5d6HumDsp1F0EJjqkYfa85\\\\\\\/V5JCDC0wRPMxRO+y+o6g0I+hKctojxIRGR14I\\\\\\\/6EbfU8duIeVyQCwV9h4\\\\\\\/lJo6jpfQ49tx6NGouLfMCWhsw0\\\\\\\/VKyT7nByAWOMNVBMobkNk4b6B8QGqXtMlYaj3NXWvY3boriAACr8sPRbpIH9AUv24AR\\\\\\\/g2+SFQTYyubAN44A5e6dRcv1ERXd\\\\\\\/x1BNMeY90xcmV7qOgoCruodOS0iWJda4B8mHod7Avs2mETeCDNFHdnwdz62GsU7InDzzk9GzgmswxDLDZ97PM7DoNycnQDNmErkFHo1pphy5eefMInV6DxsOm4RBAV9XwxUyyGauPHyURXjURt0MX5FLePqiLqmTrTaaYHCxKIP8wzVj7sBzU+zqo80kbtHR5QoQEHNKKSEK+T6fFdzoBKPfG3VAFCEkE2fk+SK3iBTTwRlakAieP6QikpDJHvR9rlk0fmFg23\\\\\\\/ekNSg+5YAPgd00jHNSpCI244P3vD2zsxqtc+q6Jenn\\\\\\\/Exzf4pSa48vl44QZQtKaih8a1EksOs+og6CK0BcfsvczaHeCrzUnnyCoLHxs3oNUovkPXVdSXFmlYrxBAMg2QEebK44fQwQfGGUsFNRwSFEH0ZPZD+fTzN1LEq33cmFdaoYcYIYKj3+kMsj1ZrJkLLBnVJaX8WHu0I4DyefBJP2IbvoW2pqmsu\\\\\\\/So3CDyN\\\\\\\/lbL\\\\\\\/I9YT1wbv321unVStD1D65lUbNZ\\\\\\\/SxNiDcUwvkXDe+IBARDsiCphlMUqZW5dGFCJAe+27a5uSIodoTFRLvyxnLxZ5Pbsy65+MZjqB2xzWSxLKXxBweYNKMqVGuTCw4H+l8YXKqooiBCgboMH82EBaYoatGqXgJPoUJcCXm10dMY1EgGs7MMhPgYEuWuJRiuK7LnfM2p8u7Qjh86MehspMbtIZxWqW4FBWzf8TSFIuA5Hw0UI8TrXSyO\\\\\\\/aCS5pVAoqdFRl8itjqkHuMdwA6RT2ePImgAvKDrwfu3ws5Nu7Y5u4PBGEYrQvQkhTtlLP+RC7J1nOv0ZvR9uZO1gEifhC4gNW7LPYoneug7p4CKOmi8pMk+IV4U0mPhITVyU+dNcgP4TReRTU2Rgg89Tr0UrXS2HSm0YFUfGN2\\\\\\\/vlxmgOLYL+FIGG0sjGkVm1i2QOZth96s7+WQFVyWsiXaJaado19GunfVfKwOeHM8wHa0S4PTNkPABvB1fU6WsIqaa7wLM9Uv7hnA\\\\\\\/N1TmBZCYPz4wiS46U86JlZjkazS7fH\\\\\\\/VTPVtbScwfjPH\\\\\\\/4TPyCiW0ub7n9R1T6dxUt1AwrQB+e2I7aekfj\\\\\\\/aGK62kJrR0+G1VSZzGj8F4PDQA45YWksSBX996zWkB+Wnh\\\\\\\/Csv6F76uCGRYyhycrPtbWtiEx8L78pyFC+0+OKf+1Dlv\\\\\\\/0WjRzaXCUPn\\\\\\\/ez8W41flxWciX7I31oBFAFIfMuBNStBGuu+rx+vJTNtBWnh2e5efa9pXLngJGd4nvcT8+5pHXvJLAIXO\\\\\\\/FaAk2NHPR5gbV4bH\\\\\\\/ahVCiziNIRzUP1CokF43ZF19WeA2Rc\\\\\\\/EX2deWlc1579N\\\\\\\/9gs\\\\\\\/lBNRH\\\\\\\/J7FxagBplOnZfZJ\\\\\\\/C6\\\\\\\/JSNVraHaEo54VBn3bR+TPO6Eg2HKJgGr72OA\\\\\\\/742CQa6KxeiOKzCIvE9ZppcMekm6Kj8lef2NAyqb34VvdmfzpYzeZft78VYvL2zq6MMoRNRTpfvEODSWvEYWKTLOeYAT9niZjdlPEs3ZSWC0GhRoYDGhB1mXxsxxKjpRZREpoaNPqPaoRy87NVwislcC41LNXRMioOpz68SCDUCfspu5j9YRT2aHqB2uAayWvaCDRf1KS8InsL0OCZ3IB\\\\\\\/bAzXpbCD2t3xliSnFa4WIcfUpTtonksm7f2RpNo0RCJZun8MiOfmSqhH9EwTinGUkyaWSkRnySdl9pPXWtHpYLHhb8e2VNezqKaH2pY93GPKFy2wgYysXFmDoSoYm4CU1UlCZA2kHvrIL7rPHf04PZpQtwJA7HMeFmHVV0zxw4ve1ca0vFZXPYQ3wSVGaAMIF3Z9pb3Jxwd0grv6TyUdz3M98ov1mO6HCHVtzdEDIcXlpseVL+dO\\\\\\\/5unyQVYzUhB5RzwgazlELE\\\\\\\/rl3Xky5SOqBLYSbj\\\\\\\/WceAd2\\\\\\\/dtAgfHcwLeRlXb5VPU6yQcZ+13IclLbVrC+qLalfesDVd13iYebrqRnDfYiblVfC+nDH1REHHGslm2gX\\\\\\\/mqWsHGSN2zioQNlD6FR3aShfnjcaJHxGxEbwXYPKfebay+0JSE8gmo\\\\\\\/tWU2ABLuRbGkEzjUH\\\\\\\/XDv0dYlp+8x2H+99A0UPqce2OwRHJaIbr0jXi7U7wTju8aPEslN7DXFK6GH90RR9Zck1zQj0AzC6hSx9cnhYRu+GxnFlqQl6csuu9eMqGIBwpZ42eUPCb8aG1zeqUOlSXf61NUoZHlgMevR8s8sdRd+dbrtk187N9D5jPI9KTU40h8irTnNihXnclnTo6+5g3HDfKPfBAHwkgUPvqEHoBMiuzu0eATVb9v2S8ymyDsy7kowxl7L3VyPaoeQwujL1sHkwg7TJZC9S3ijWNvtvL6GlTi6w0kM+\\\\\\\/tmMVS+ll\\\\\\\/Gd6GROiYckOdyuqIjeEcMXgB7miGzoXtXpXJbgzYldd\\\\\\\/bqfR5JDFcwgjfvcgBlxYLTBJaiFzRICYGi2HEYVHLDVdveqUogsXzvKvA4f\\\\\\\/aDiTvi6LZsHyX\\\\\\\/sZPicLgXzYTJpCaXaYi7SsZ\\\\\\\/3N9xk0GIzhyXHynE4No9zsBuIZsFlrzohPIE+R1\\\\\\\/t3SSJeFfgsMOce1VJi6cLew6arARBsMBFCFf3zvWESzboc1YjfWsYOITLgkkyF\\\\\\\/TSILEmzZK5oBSjWexhVXSl1XzyHtiGcyX\\\\\\\/Eo9iuuGan\\\\\\\/53J7MND+tvB4aLe\\\\\\\/+rQnaGqHKs5VWwWI3SzNCRjTDPxdL5rQ+oKnkWHnuPJ2kEmpSPkIYOXPQTN+QlsAJCOHzkYlJ+BVQYHEjPsvuKw8IIJURxvw0q0o6zT032u9GNx393+msC4oFYyQidgFWtlrG96vElU6aRcUJ5EFOW7N72dat1NrwXUh1aC0JEg0iQq+ZGJ6oTqf2PnpFNdeT2Eri4QRc6nKuJb6ZNkrD0fFXjxOqmAfB4hA\\\\\\\/W38D4B8ko0DUde2YDZeDhD\\\\\\\/L4P9k2MRs9lnmziqyU2oqldUprQli6vjaEDWz\\\\\\\/erA6d+JHD0CReOorRE6D1XxmJ2quy07Pcisq5QjngGb9Q44U4pFDDI4+ovChnMo9LGl4oOTTPcmmsj\\\\\\\/CG2rnGl9lGMmS\\\\\\\/8C\\\\\\\/5LYS3HLCi4i4qDvgKSHPdtfligpGnTz5zaY2gHHVnDxhTI5j17jmq54ehu24qhwzOFM7i5j95sH13eQZx9itMe8CEzrXHaP7I4lV94yBZxJrBKtp1BUoZqCJOLKuQ2wiVZVG2nvt0wOW3m+zmfjvBFrwUrUU13k4SLlyJcSNF7L918Z4nxUIRCcZKcGVPjzycC2hX2QOPQCq4bYoAJ+SCMtyqU3qh2zaexS6j1UAsufXbEL8uraDakTAJfkGCWoBMfctGpy16KFrucGrszob8h\\\\\\\/7+KzLpZZ\\\\\\\/iaYC+Ie11s2bX5g+9s0o2RTTC6BsVsQqQ0DfU9N\\\\\\\/dCP3c+bKFcBX01vZubHMcXC08+d2OQO6jpv4o1+wMWUdCDnpUMwILEyHW2j3x1F8GPKwLOzsTU6N+sy50xiRVbZ5AecpFIYRAhP+zlNRg07G+2a1qUo43fCvzJ46bYzM8dt6St2zVZQH5B0PmPrjqYLGARAczwmRaEMHGgI2wus5wHNi7ZLZEBVfMcClZbWZLB2UNVl6zqnhejI37R7MabDAtpYw7+msCB4KLk3Xo+98YJ0JyzjB4v+ZyE3eNzGsBHM4JKJszFApxqLo1NT9L51xbt0hpTuFMmB6CaWL3Y3RUxPj9Dq3z7\\\\\\\/sIQoYN0+AoHV0kf4mpBaaqJpcB9UlmRMSFBV1H9\\\\\\\/MGti0OWgJcLn7HpouwWYVErLv2koExnJHl1kxBuNQyN07W59x1ccEugDU21gwTbzoCmAo1PMXXmOsTq+iWOX80+IhSXW3dLCFmymqVQ0SgtSe6WZS8mfDaS6ogQ5G54F0RL7p9qaNL4eVfplnqNKIZyApb9aNaklqd6Sp\\\\\\\/5qigpNuERMWTxTxCkqXQjLLwExjCZCVE88UywjxxvNToWqjoeOZhQntfTn8Zl5ERAesLkPe1naP32AP4rewXGH5f+r1A6E9HrvkXfl9ENuMg3digrRUlZnG1lfmV2\\\\\\\/4WpXyr\\\\\\\/QkUyrioNo2SqtZ\\\\\\\/nMkOTdb8guVCQKhv24Dp4b109bGyX\\\\\\\/KCrdrShms1QB2pK92Q\\\\\\\/+uT16U\\\\\\\/7+dSu6Sm\\\\\\\/4s5iThZtHwYhMNbOQeCVu4zA8H6XxBEnG\\\\\\\/FfLAnEpMuDN73AfwxCcvgFnefMlimKTJSJ4sGbCHoyyqr6JYmm0FnvWg+6MsnTCuxUSM7skxZGetERJoW87YV\\\\\\\/T9kpFW1B+eGSqwzE4hpaLltIhi9ItcCBaUUNgkwJkH8rwufJO3ngMnwI18LXcXIFXaAlv84Un26anRCcZoLLY\\\\\\\/Mg7ZCoJC9XRKOLcXGdmiD+An\\\\\\\/A8FhkV7\\\\\\\/vcWF0\\\\\\\/NEbkKZgXv\\\\\\\/MWWm1pKK8JXhRXHRTwWHBeZ4cyKzs1Df17PL9jiSAYIx6RHzK+SqGdYB6e1GJQBk0ejjvYSGYd4N4o+BHb6QSIlq3jUB\\\\\\\/hvKqv4oMEl7Krhj0fm1nTMK318gpQ2tUte0wnyA5JKn5rAX5URc3xHqXr0tlC0bAjenzJ\\\\\\\/rkLStW1YutPhrT0LkqAWCNjglwST2AJ53reUXppy8UkKIGFY1ZDGI6Qi\\\\\\\/LlRquzNz\\\\\\\/7fJe8KQHBhN\\\\\\\/YjFjvJL0TD2VEqzNsVSUVNO8O9fJAMQ8kU8j2bigTS3lV10K+ExvT3RJzCcW9j7m35L4aEFRrTT5kxhLWE3PWbZdajptBFMik1TCpxpA1ay7KVhvq9JK30ukFoeh6Lg4nlkeMzmEBFF9M2FdxhMeHiVywE9eRjBklBrISkKR3v6s2dm+g1GlBfXeb4EQow37plJhvjPF9ciKCtI1kBkCB+pIhk1Gg\\\\\\\/8saCgLGejUWUAKHdE80CFxJ58h+FhqrHXeYWWuoLsRpuQwu4gW1wV7Kwep9uyo+Wv4TJBISB1NZGDnv\\\\\\\/Le75hDbr9\\\\\\\/0zsDq9UDRtgqfDYdNSPUq0UbbdYuo4f6Kxw8YLFtPkT3MYovRcJ1Y87vY4CdpEqeE0iSNERiQWFvconrWqt5hVPSFAnwNbe5ICeftgtsekh0pfIu9opSgVQCvBgX3t6lTg5OaQzXntiDBBT1j2eIPNLKhLW8SE0Adxwwwdob41auKJBSvt2Me9503fWPNJwxsVZAQDPEgylPkt6euTrwJXXlxqBIIhPIbjlQTNGzsY6VsaWqziSC9dmYrEn181tKom3cCBmZTAw3qutBSICcwQ+SsUc1wRS9CLD9GFGkY1eu3ff3fLkMmCNj+xxaO345xmfp8sYUlfTrJXkkIUAQ6Tryqi9W33Lbc0uN0LycdXglAGlZDvx9h\\\\\\\/qHsnH+2OAXsAX3F5peOEXxsZS1CDUMJYDwIPzWQ06FsjSjwUtpVzFbsX9iuNVNbucG\\\\\\\/j05\\\\\\\/UAZI6DgfMLdFixdfhyNbf5e2+arYF7pMz9pxXvBbOqKdccZux9\\\\\\\/xfqsGMLCW+UzHd4gVf4b\\\\\\\/240qN+VwQ+wEZKrL94jBF\\\\\\\/XpKJTrvm4bLBA2Nnqn404llrXIqhVKK0Ip\\\\\\\/Uvd0eghrayUz9wYyTi25RYL3WWCMQovKnu+biMTqWg0DCD65kQ3Td4q+ty7uXmgLdKo\\\\\\\/LppPokBYvSYcAfH4iJz2KNo9TzKc+HqXJlVs+25zDU5rApyGfCUiWv0QcW7nhlg271FUVWRVV6XusPDOtGV7uCAbFKyZXqtbXSgUbumJWTmEAwBbRIhMjvGcqcWUVjD7HExn+jqmTPE1QbuvD2r5Pcz03bhPCjyIEwA13khcVcfzjIHNVJsaMWAs3ODXyX9\\\\\\\/XWzu11uDD4r\\\\\\\/ultbA5LQzhbFK+vBqojaO1ZhBbQkyaRANiapqMbB0maQZ\\\\\\\/ABi6J689FJu7\\\\\\\/svTs\\\\\\\/QcOGzbUN9kGufaghtW3ehuajJVGoHvGgXtAJ54ohGOZAP7WmEGlYImDjqPR+T0coww5MW3zT+WD9Q36svZo\\\\\\\/4PhyGoJ4KsqLYK4IJK8yukwmcEZj56k67niHSC5Si+HPDVOLOFVP0jWeqMEIoOjqyGh2geocrd\\\\\\\/qfaS\\\\\\\/9nh5edtNPZc1fC9VQL9FacRWpOhHJrQNRushWDPe0RE\\\\\\\/gL5sHI6n3wxbJ\\\\\\\/so4qskj+\\\\\\\/l1MYg0N4FVmBayhExWO3A\\\\\\\/HmZPEzY7xWMx482eraCsjqQojtOc+SPKRkg+7bulpt+JJA0uHEajQFgw8QzE5XoCOBoMiuetsNcBJDfI803y9IMw3I8Npk+UrSGabwBTYoK3bkEyZk0cIQktl4EPJlQ1d8yVEGieXPBCUX1HAFunrsco2r8NcFcIOm3XuJtHJAwop4E1NEPzzLGyT26VTjx51TkxzpizpdvRFT6uDpYcZCWhmFLVAMfJBYp1At3SlpGzV9KVSGHcNRufBJ1qqGdVvyIuAoSR4YYkCHspQ9noTngjh\\\\\\\/gpbDAYIwIp3XBE2ELhqCB5Oz6Wxxs4x0oP03tHdSePpzMpwk71ZVXlx7pR0bW9P8XTsmwBtf0KtknYauHyV3eNNRJ0Ave1em7\\\\\\\/i8hwQE93jR7jV2KQso3XbQHM4ZndSDph3GoCWxv2MC55vVVTCd8JTJqBcFspCwFc6VmcpShwu5PPNYPiXAaqGaxrCTC5aKK0HPxdVIkWWFIGEQqFiry0Hv5fVkXZlTT2pVgJ2tHLNJwsuYFGyslzoNCIWbfXgAR3W367pbo8i+WXsqriDGLwjqw6F1WTkPCfblLzlWUiS8+FoYBJLZCqmfhjOT3CjBtF6t8OFuiSw\\\\\\\/7ZEW9lUxga71VpBTH3wC0f\\\\\\\/fWhg7qOJdIpSAX5lQxQb1vq9Yiw2h6sth1vEPN9pFmEwqPqkco0sGJeBTSNlXn6tb194A+9rO+qDULe52lCTfx9C4xZrjIwf7wQA\\\\\\\/T5f2fqz2\\\\\\\/moqDtnhqmrnQMV1s0\\\\\\\/rhqao\\\\\\\/VcfX\\\\\\\/Id57YFWwzwr\\\\\\\/LFfWlrEikbv47h8NnJr47SHCy8b0cJ3zIMOXBTCL\\\\\\\/7KviHciS9XozKfVoCZ374oEDdsIYLZEfSDkWmzhhveradM8rOc1x3li5CmcnLz6rIWcd38n1\\\\\\\/DXw+R+v65YSQKxh\\\\\\\/N3igRGO+O91nsiVFQvKBSlu0L1YK+3hCIUz+fcurQ7EDkJxuB3ptVp7z636i6N6R559Kcf1T4\\\\\\\/8hgAHkQCp+Sg6QXY9tc9AeFPauNZk9deIEtJLYVfmvsQXXci6aIBLsr9ccLPaEgV9+j9ZSpzzNKiVLONAPWp1DCo8LoyRki3zimg3A338XuS70xVLTDEzNFjFletq4SeuithrjGlF7V2bBMegAOdpT6JhVNpXBJoHx1f9BO1Fq1UqCVn4ciHfbCigejzE1u2AAaerSQ7DtIkH+yEvmBlP\\\\\\\/LiG5oe93JyzlrnmRzWATHiz0XHxTXexhTmFJ8Wwd4b1kYQ4gdWAVxli+LC8D51umcBegrWKpPEXvyBNu0sEirYEBF3M378VBSLNuuCre+IgL\\\\\\\/l3nlwDC3H6GxbxhuyOBK7xmGVOThUbYR7R+2k6UDIPsQTkhDKQ+oI9Qx\\\\\\\/bkWTHNrI3rCUzuMQmJWRKVavcZFMX0yd3\\\\\\\/OLkH8pznCu5LD8y6P7B3xg97NDgaBefgO+vmvD9ER2EsiNcDG8f2TP7UGvS0y+X4nYqBUL6KN6Ag9L8APWt6386joFcUABCcObcB+4NXyfEVflUW1\\\\\\\/50sKJG2H5k1hLBKVt1\\\\\\\/nYdvl5ewCRTVkl7QzeIqS1l9GnYkc7+WrKQ0+PdWmusOxcZGJbpPne1CvMR4VrV1kbz0t\\\\\\\/uLENFEt6E3ZzGRVoIkZ\\\\\\\/CBVOmGhNd85OMWc2xke6Hk10O3lPXRZRQheYDSYI+E+nfPkaBpB3+lwwejE5hYI9753foeiDul3U7KFUcJSc+Rcc5XkvYh4WJeji4YjvCb4SsYKt1jQf5cHccu6lFAMQJHEOMpDsIHFRwhXh2bvX+frcm6G9r3TNHxrvh9WJZADTslea2qSHrCerX5SXbua3zPH+K66\\\\\\\/tExaFeDl5nI+A\\\\\\\/pgnhLCUr3zRuiaePEjAHju8k5WS9VYBh3TTu\\\\\\\/itTtewkbvyUar2e7fBAjae3tpRpJEdOdS\\\\\\\/PTEWQSRhh1SVhz9voIoiptsSrzOK52AqmjRGJu\\\\\\\/Uiz+LTt9j87b2VfMttH8hG8JadUIL31N\\\\\\\/l\\\\\\\/Ziqi0Um72chhm7vT9g9ipKmrq88SpVihD9x8Z4sh\\\\\\\/\\\\\\\/0woccfSpp9M2uXL6jky03DEnll1OEeifdukf98jT+8vaprate0i34Ps4WaWYpCykk2IQBSIhLX8QqsNORI\\\\\\\/OjAQz5Km4KMrlp7+UeuF\\\\\\\/jo5LXOtvLyNCktYUbN5VO2Ojnrcxy4qlwdFNQypQ9cHEmjieL4zDjtxjIQatg6LkZTCe6KTvopegrLj0ExDyXaisllo4oLQWwYnY7YuDGzbfGyUc3z9ucblJ\\\\\\\/5jA17VB0GN1cw\\\\\\\/2w07bSd1ZQSVxF\\\\\\\/CsHdZbTXnUm5aNUoPl0yXayArBjI8eK8WNCzUs54V6AOWFSJS7a84h9LXlpMF25AAdxjK6GX4eJtb0UQ41USffSqekgee4L6hinEEyJkHAPoYHO2a\\\\\\\/c8QJxHaf6n5ZUKv0nr8z4X5+Ipp3Vj88V6YK0EU5fT3aEUvb6po1P2qBAFeZ3iudpctEE9nXG8fEoZ5T3cON6eewK7c3oH4CopPuBkFXIDppST66ZgIuSE9EfTPTEB5322HrBYAUbDBJAkrDs8tHqXEAH+NjeFY6G+AkQH1wDPQotg6bG2beVqW51wbz8Qu6eBjLCkBcmdxbRAhnCrgjNqHiMyfGrxXdpZsAlvQIl1qufQHbn0O9AMZJ9s4ffK99SlKboy9Ph0rTcDmT1UcbdYMI4+aoGavYIgnJwd\\\\\\\/8AE9HddxpygoXqVgzIhrMBha1LnJwP8NPhsarq+yftYyx34iPSs2xpBZNz4E262J\\\\\\\/ybSfMUBPtIQbK8bjmUPwXycsRdRQccebHFA5FZQ4P01lvzE5IKe3EyUai2M+017FylhveirI2\\\\\\\/RHPSbBBuX9J1iSAOAL6WHAW97u8iM3S5rr5CSCwScVdgPoF6fqcLp5wUmJatFWDjRUel97rRKi95RkVtM5lxbF7790GamFfUfASMV1p13e9N0i+vn4\\\\\\\/Ou0hNovxp7NJJRE7FSHyJcGVuOzc5P7gJ316QnEkUmmhaGGZclq9PjxUaM8rmfrKK40tDk0V8hiOjaWKxsU7N8T8opm4L23yX\\\\\\\/4SUz2qpDLrXbDDTLREccdHUN4ZCDO94Z2b+y90iWR93bgb\\\\\\\/gcQz3+gPMCCyUzz83iaUBQBq2QlBI0xZUOSFBcUUpbiFLggRJJ4hHHLOtpMkCWAUNm9WvuLnjUeCRRkURVcO9onoUUVSRZ9cFE2x1dC5Qx8PpC2\\\\\\\/kv2NJAHdcI297PY29pHaHM2RkGySgpsG3lRrNGYogMxgB+4K4KuVn4Au98oVGDu0oCZsFnZuWuCNEuPsMjdH04IeIGluf9+SO2+gBP\\\\\\\/78XutSVn\\\\\\\/alHdOX41AVvZ6AbkBYp8eg3spi\\\\\\\/XccMdtvb1+VAbFGxsJPVh3HaWgYsiVqeYT5dYHlSJiC\\\\\\\/9R2v3dDZUm3Yf1OBtqd1SF9PJnVinV7g9iMtHbP6+WT4QFXZMSWp1QIc8GzcZj\\\\\\\/qPT97NaGDDzVI77m4+6DUKq1luKNk\\\\\\\/KMMqX3qelSb6gx6ViMKqbPFNVqyMmpbXGbe4P2irg+GQA3VtA3Dpbc7uohsbLgq0rsXTwkD0wJZ68n1CzOl7fUvI+Q121NDSoioGzOQRX7x0QpFvSQtMNJuOrZ8LOst5zgGre3tr9euUc4afGiwxRxmrn8ipB7UC6MeHWxEsMqDCV8P5uw6aO2fqdWKGLYsXDj1aKMPPIOcal6bKi+tKtfIfSzbhs4nxv2+LHt5PIvhGxjy4J7vFmjwz3Y5zAkPkAJ68naHey19FsL8Zkeu2OUfuiw1reD+oq9Sk00ybXgMjCzaM5Ao2N+kVpjsPhuO137l3YXm+w0jQeCd1BpmIKwGJMJjuT1VqmGUXVIzldP5dCppwMFbPzyFh2W+R19vVaLX\\\\\\\/t3zMXfqgELD1rd78oB9TmfGBSrsGG1eNoNPi8cFoNDfbyF8m03\\\\\\\/EJI1TYVlTTPMT69hgwAux2n41RrXtV8Fa3WZsZJoGa0sCvYVn6uvyY2lodQASwTLiH1B\\\\\\\/dzkEwkeq3CsK+IN1HR\\\\\\\/9\\\\\\\/udqLVDgWz+L0pSjM4ftwwKCUzl\\\\\\\/\\\\\\\/C7za\\\\\\\/tiIeIpml9b9XwIRhbIWOg46YZtlW1dq9e97U8HkpLx0fLs02\\\\\\\/R\\\\\\\/zJthbYYQJyJF4MLYNEgKwaDFAfpwthr3BqHot7LBlfq9Dsph9mY6iyLzmDyEUvJ3nlIJDr6zboC+RlYsDR0MvaS\\\\\\\/RMByN8xavuddW8IpNdqeywrVVyRsfsFVr6jz0OyyRPDe5leELyMemo8Ghofh8GUCwFg96W1sLaLA5BjvSCR1fC43oN3oZimDq7X0\\\\\\\/r0px3786+rD6BXZNsk4X+66tuKQPULsm8LAui1wb5pgwzpJjnTQiIi5tuEJNOYK2m98k8AjJdfziF9k0hMC4l4Msr4yrj1hop\\\\\\\/bz0yTNeQuj\\\\\\\/9JeZUdSt+udGtJfPyL5vbCOKDvcKnuxpvrbuULkJNDnmysB\\\\\\\/fZurqeeW5b5ZSDbxLA4mJHbu1V6AB3KoxjiPMM\\\\\\\/ZCsKSnrj+tyirCOFU4+GFeC4nFjrnVtVjtPEZXSMFh3SoxLxMw6PBNEg0GoqvpADG9\\\\\\\/1fldY2z890rdTGmxKGFNR0+bEikCKzjibqoDDwk7\\\\\\\/YV5XwBBbpu4Eef16fZEyoUkMGqKg4pkiVEYdmezmYaoftq+O6+1QWZyqSYmJczEkVUwRVpAN\\\\\\\/7C2fsp6vHDPvHKjmq\\\\\\\/Sc5\\\\\\\/CP\\\\\\\/7W3QE4mqJvy8TsYRYCG0jHrRtvf5VVXlGjlBo1sqwCFi5wEVqJCIR0ApXoz7A\\\\\\\/mzHEgXrfNxpjNxVZnfAN2FtGbboPd7OlUacHxZhfAjGNfE+cKsf2qgFd0\\\\\\\/Vwh2JluYBvV\\\\\\\/I\\\\\\\/t1XxFPdY0aGk\\\\\\\/Bh6WYClGTkQDbgNdTbdbcOTeJA3KSwIDjScb9gd5qO9hlMuBPCu\\\\\\\/SiKAx+rBgGlDCTLCoOB9o9Zo7DExIZCa36epaKxDicwX+kvVmy41boBu5bhJMmiJtyRFg2Vy6n5RsI7Rd1rHRVwfjfv+usTH64pqpyHtHDxinZyH+zQhpmsBByMfuh3gZKgOt2VUXeh04\\\\\\\/+bsy3F\\\\\\\/f\\\\\\\/MuXsDo\\\\\\\/Fm3dd949xWhC\\\\\\\/PXqp2vPBJe6IYOP6jC2Quf3L0Rg8Ez9lX\\\\\\\/+9rJ75NpioVhP1KEDUG\\\\\\\/HjCSUPrNSNg2A3wPG6y3AceNVHZeIguJTqHLKyrEqIR1K5i2lq783gHqFf+GIh9Cn3xjw2Kfw1IR01dHSSZDXfFGaUqamqoWCWZu6Wb17EIepS\\\\\\\/zsGpugkKQemBLz6DCE7W6Rt+bgOrNSPOuaXDtf3+\\\\\\\/XdioJMSPB9fuhndc7OLIGDcMND9hrauv36vtwalqb4ElJgzo\\\\\\\/JMANmT\\\\\\\/KzQBijz3vUMR2W8fxE2emRyBaqa60g+mmpRvFr3eO+XYQvGfnRNbGBbDO\\\\\\\/cFB8JqrdBcHX9hgbKmB2+WthOI6zIhH8PubWfDRyeMJ8viwO20eP4XhfyaucJbOSZwWzgya5rUCS2BspnWO0nTxe3IV7UpvVuP5k3zuoVY55qOlqVtaZSfPf8lG05sxtvUi8WqqBsMsD+xmyd8sEvUVQA0UWBYlv3wb5vxlj52gKJ803U2yrRlIOxnXxJPEufLf+riMDioeFzVJAuDxrG\\\\\\\/N97rK1j9oMDyZH0puQPGmfwc1k9YHg4S96QIw5qpBwg9UEvERviAQO5q7QZY3yKd+yAZpI4uGYiVIu0Rwjl7nBPNC8WUpWY0T4txXp4l0TpXhbT82XGx9OTAS20eDn4mEU27W285e5CNLaYdkrukL5Cqrxnc4Xi0M2i\\\\\\\/q3KP8UkIRSx7LByjvAp0J008ZxzpDcu0zp6L1T67\\\\\\\/UgKe2B\\\\\\\/dR2UcaHTRAX0LHSYsJVzGrM545xZVfb4ELZRH45a2rFg3Xp5DJhSkl4X2sG06sV4hQPDkrZGgwqsEcdXZym2OHNQKPtIRxONkGhqkSKlyY+ATGd0F8KTAsxUCjTDtq8h+P7r1TDf6CS7Br4Op5VFIXb1pkqY\\\\\\\/9edCzJPzVIqlmkOsjVhPgMw9WZ+1HSH0saDC1PhiC9OpD3AISulhD64laoNfwFenGNo3g3bGqReTVMiKzkKq\\\\\\\/ewmejOKWVDZ4nSmutq9uBjqAoHCMwlH58FUQjlMQ9mc1pYANM526acrZCvfso2piybDc8cUCzT2VoKKtJXbTT3sUhCmgHFvFtVZXQ0ZaECajizZf4M\\\\\\\/odhbi0W\\\\\\\/at9Ph1QoFWje2K39uxhL7Y9YOoUX06J9nlh4JSPN7CuyAUcFuuQeiBQauq5Ip8V0OZWH3sRZ4XcvD6XJoQUNfGtkHAlf6ydb4RgKoCGpfK3v2VOr0O2bM2fyGXq88K2kuy\\\\\\\/BSvkjhwEDiFqGGDVDP9NmjW36my6gY33mzCDB1LpJMu\\\\\\\/MPG6uc3fsNVNy0x9sMcl8XwHYwwIxDNApabZBMtkhqaUgN3tKAUBIF2K7lOPcllBQNa36uXAMdDn6g2dFMzvpgEtnECLuqMrhCUPX6lxaRV6Iq\\\\\\\/s\\\\\\\/3K2NiFdPgK6JVKBtTxPi9bhLKtRCNmDBtpvkmZbF7Ul6jbcwbYHQS6Ka6vDSbCHxQ0vsGo7YvZxdejGkPxgn9fzzn6yhNxsSZA6mhcSucj5bAXSC1Tn4afTBoV+uqZ9JC7hjsqCAS6yNNWrWYIwHjUApLJ8cJ7cBhJ2FtHKWPNhr31Oz9QU1hPyWNHq+OwPrKe\\\\\\\/PlD4YFCIKBQ9FP\\\\\\\/M5DHJASwI\\\\\\\/oU\\\\\\\/17YzPoq6skt4Zz7sUfdWpYqd27KkuGFWIg4DSkhILLw\\\\\\\/XXLxL4z5zQstyvc4OUKYpVvkJtwDfxPYyvjUImGjmF6WYn5IcTsA6Sa3mo4\\\\\\\/nrBbzD\\\\\\\/3b9KGJcoTsd0mGdJgJsjwpL06X\\\\\\\/9fTeX+Mnt2AMTTMqxy6uo\\\\\\\/JKl6nb7kG90n1V3Ya5\\\\\\\/DFsAe66cXH\\\\\\\/TSF0ITBCbtuvRJnF2RS6RNAOgRAuZ8Qnll6LaR6jdaP22CY6gSqeM3URJ0c732\\\\\\\/hCzFFyk0kSSq37+5J9z7teJ8HKEDSaqdrNs7E1nJmZReW3enEm2Odes8owSqig9iff9\\\\\\\/qJbpiW\\\\\\\/4tfJKXxd6OMoWTJe0TTBoDd3WQf3Fi+ShXSjLaFfl7wCMuLQacbpZUi82i6745tko+y4dPArfbTMhVKeTG8mt4JuwbJwmxWYwmT6wBBt19oKzDslhZNaHXsNQCYd5V8eeF\\\\\\\/+ARKHur\\\\\\\/\\\\\\\/SqDvZLe\\\\\\\/QzLZ6CvNRwjDPSDRZkZ6ZxqnZgKbtpVUNi+ntwBtJPOi0b3lpunQyeIOKsl\\\\\\\/70AdpLc+L4WA4ytKti0Y31+Jrk+XiPbgNsmq8VKpl\\\\\\\/kAPKYBfSpvoqjUdkYKxdC4mUSX\\\\\\\/5n\\\\\\\/RhYz8B6VyaDj9FwsRAioG5dSpwyzKVuEnUTPBx2nJ45BOeEpdn9NhPMDkAvuRkt2w307PueVrNYDPHQzxYYd0RVJhQQStYWEOPZmvZYFomxPAEFdrgpqMxxk+ePr9ciye5Y\\\\\\\/fHKJp\\\\\\\/gOPMCgBkyBYGDu2yevvsr8wW3hT\\\\\\\/66mbTHmskoBniWRkuldLml7T\\\\\\\/p2aPcXkzxr+dqziJCIDnNL3fjOojmkrNfSq1X2Rb4CmpQx+dW1kFZNvLjo3fdVr+NslyQ0K8yY1F67NbXquiMhADpo9T6+Z5d7cLGvgv0\\\\\\\/s4lRuNxT7NLLAaN4RPRc03M5jUQmf8VK1iY6OJOyfq\\\\\\\/QDqw9BmKlF7JPdm56HmbcFDVJR+FQ72J0rp6lUc3d6OP0\\\\\\\/dgeJv9SpFtPcP9CMEFL4MWYHEqH+AAjTeqlBlpdh5u3GFc7Mo2VBn56EJ70Ob1wjZPsiaQNIcMOdXyuwpJ1Ls5UDn2LlVPe7GaEbnpvTTuvKrWxIrSR521ySGcjPBHNZ5h3kavxv9QPdCPcD952uHSeLLwhTuFo9Z255xOZP1oL7rubSsfqO3U3uWGjDj2Z4HbRA24j11rPYSvyvWluGPAiJ00ln4KJXfNoKwVvDcxTf+SepwSkdFLiDxvzVQ2OcmOUlacEO5DWXJ\\\\\\\/ytE+gbgeuNMzuIfcLVUXt5mk3OtCHW0ioyF9JCp1\\\\\\\/j19oGvw+8mURzE+4YQtPyVcVfJkSedoblQ0nHwyiVxmyIYu3kTPqKFuRP0PxV7z9KLvUcxzqeBkWJtbKFQjoYWPKeIsb71xTVmuxQ+0GT8bNDVj4amgMprK1+PkMWoOKIprFH9PRCYaksfckVgyKDMj1CxMuxHWFVrC\\\\\\\/HHQoselnSJ7AxMjx9jv0Xvp+aSVoc2CbliDN+orR4VQ2oDmkR1sOjz6g2vG+f6UzJyoT0iNbVmRkX32yOAR1WZA1EKyo\\\\\\\/fBpHbJ6grAUgroc9OqjLNVSWoK+0Zh3Q50p\\\\\\\/2+EGSwM+XvPH2W+vLQM4++jL549pJbe1vTGcQisvajw0Cf9vZAibrqpKP+vGqc7JRgg934ablNAofZypbGwB9i5T8ArMWFiCydQXw9FeG3jHt0mv+G7Zt6aUtrp0J7MeaY+ZcHsGrQWxFlTmAx2GcrYGdY6r+pPpFI7Ajz8\\\\\\\/gp\\\\\\\/2rYhriqwxv0MvasOxdidpP9dRFs9X\\\\\\\/\\\\\\\/R3FVKMDKlOlNjX7Xw98YAb5ue1xkl8+Ay\\\\\\\/4CKS35HtuscQBRdYZDfFPPaR5Ra+aCfjb4bfqzF4mLb43ZPWYM1eNlx4IGHrKQjGuJIlGtRSXPHo\\\\\\\/25RP4VEKjpMBYMZ1DOJXThSmkFVOhG116AvlW5cS86IkN0lCjAphL9JWbo9qFCAi33wV\\\\\\\/g\\\\\\\/ogGlq8Vg6IQTPSvWYsIT6T65jIUF4pTsoGYbYnnl+9fFJsV7g7bcMYYpfHqL7hJLi45jsDJXVe\\\\\\\/BQ3EQDJbMJrlDmLvca0LFgH4gDIP2LdFErFAMEeoG3LlCIEvEbyB\\\\\\\/ox+T2fB2PYX9XeseCWOh45E1ICZ0f1r+LqqepjtBkVKENtCfrS0+l\\\\\\\/fT2JjYbswEQEUP4jU8XkHfFApp8US+Pw7BSEViaslHN2L5bygH2KkXzBHGdIGF+CZePaU84iBvONEdJogwWkxpSk9LiWwHSNT3UDc6Oq3tVeqkZqMs8Be4r+U2Py5fwYQi0xqMehp2VsVdJpKS5Ai4VRHio1Cy02qsqDS\\\\\\\/SnPTgX9KCb8THHHKSMatU7e\\\\\\\/ERUaVlxvv6lAtaYRfauUT1kmb71dQwGlpFEovr5ufviwB2oP92\\\\\\\/NSNcCwhw0TyliuUGPSrGPgSkvvst\\\\\\\/khqr64MyMGiKcyL7tZirfNFCgcoeOZtartUE5T\\\\\\\/r8D39Ti8HQ2huhr14sXXEPWfioh1h8fC3unQVAlwu50lOGtFOrIUlmyujzmxKhBJisTpHcaX9LBBGDasaFN7D+9ycE3LvrU6FF06uruxUcWyczCZuJ8qOMnqyLIEZs7\\\\\\\/6URkdYLXD37MXMdlnLXZLwmSsXj3lNqPT6bmc806zqWf9FQPOvBNLm\\\\\\\/eQJBOngyEc6N31XNxEOl1tMw7cZbW3bHLYaUY9Nz96\\\\\\\/NufMTQu7qqiYND75r9GKcxaaFyqdJoQUhHPgX58gKZosJvo3OiUHbWKacU8U++ZdeWrq0Xq4MsvmvroMdMU\\\\\\\/eIWOwKp12VKjOFE5V0UI\\\\\\\/psomNUmf4h7Uh4n\\\\\\\/M+VdmOkgLBURU1UvD4MRrnuGJkMmtK7vxvaeVBKwbI7jBYdSIXPJnGSWHa\\\\\\\/hVfBgyQcl6LVUMuOF3sVG3I04eeTMM\\\\\\\/8wws3pTxO6Z63f7yi+amDSNnftY55sUFT8OgRMVZ4y3YMkVbMlWSTIbyDAOCkIWZzZgqtim6wuIVKnvuVaQJ7OT\\\\\\\/+m7DHfhL82ht3FVjqbJXxopXZsp7S7NodPydTLawhzQNJHy8PCQQlO8Yko4KOZ8W+GokU7voYO7d9NiKtidQdZv7tTPz6i5RVIyvexMc+nC6mKU7dQLAGn0Q9a5jm+T4+gtGdC4BdO0QxfBN+pHBB7isZMy9+IFSofyivoUNy8X2WWe5HYGeeZmGgA2MtJJW3maYKJgE5ZByeWd7KSdeuKPa5A6M+5k4nOcv1fMDQvUpVrswTNb8vyZO+\\\\\\\/rHwo6J4+DykoVUX4JIsz2EYAeIBByKVNrhzP7\\\\\\\/JX2XLG2byqk7xvsA3Bxrkm70OITofu8vmdV6gSiCgLUSWHcMcCgySJYEEK1HkxV6C+WlPoedDIIDLGAQfgbNPNiN6mkRdrBnJksGo\\\\\\\/eHHPqADRmEj\\\\\\\/ioDkzgg6xW9iPp6JM6GZMnSb9nw6ZCbuFEnQF4pi+5tyPi1EMtLJhBtrbj58yIuo\\\\\\\/tOVG9f318hPCydsZM7c13fK9tyruHTN2oVYbCXTUU3gFOHkskP4LO3TD6WP2U\\\\\\\/3F6cMH0PGDutu+eYfR1oeC\\\\\\\/Fn+CENgGH45pZH4pT1sz0aR1OVr19ESzWurUeHAR9+SvwGu\\\\\\\/DgE\\\\\\\/8iztmYW+OrKycDtWNk0zRGRhMrjlX+N5eYoh7lDdA4spGZn83nwFRHYuK4YSHTeTRU4dJgt9+upcM0ULrEjowSkAKVcrubWiKis+L7ZTkq07l4or7sis+2NEjcQtGvDpUZOg8MtoxPqGSQwmjhXkRV11xkoHMbfpQMWayHM9uDszsdUZ4lGkgKzLilHO+PiQhMzDtbfvDyxAoW3vcz\\\\\\\/RKxbU19qNSENTsOVZfZ3xAuxJnaNvkIKuisfSadpcgA\\\\\\\/y1WOslBLnGVp9bRUUPfvmBuRd1kQgJ4E6Yf\\\\\\\/Y3ytRImxsePue4QJy9\\\\\\\/Cb8ZHCJYskINbDjfP6XJo4aIyxi+TMq7bDRKnwTUu+v5eINY83bte\\\\\\\/YAuw2Zar649SZHjEIvMeIf770rIVJrMuhkEwPQlXjslRJDlok5n21eBgQ8dY8vY49AX9+SvTRH3KUdbPgBdE2\\\\\\\/O1262JufR+2\\\\\\\/cRVtnUJ4vxAbCugvWaNWAmpKarYBKSXAQIQeW2h1MlilPNpnom6mzHhVPKjeSChEcZIXktT5OEAJr4t9crVF6TxoPajuV436l70qnA6PWGQot+vpYzXwGvDs2fhuseHdCsNhGwL2hXpDzzaJevnqRO\\\\\\\/IdhKyDzm2inkUIyrhn2GplOQtT9o\\\\\\\/opabD+f9L9darzqebI8ETRUTlGNGkJTPk7JDjBkIn37mHrJ6RbWAc+58ONUZtBsNFCwWJ0hnzNesTSoOFXoi7t3YwI\\\\\\\/U4KA\\\\\\\/Uwzhg5Mihx8zicxjXOKIesRaFdPau403mXmCHKpOeBcZv0VzhGVd0AjkHZCBCeBpLnzBH6emzS5P4ohF\\\\\\\/LWYlE+gNgNLGoWCGcyrQr5s7C+EWQuN1JgAIlgtZtwesmKFsUrrgapBjeBeeSXoFxwBkUviEoEZhVmPs+JAEnWJAqTPXppHYWifFquZBqUTWGIWGydzi3OAQtK8CUKyBPZiNNbBJbBYuDB\\\\\\\/MTl+EdCAzZTp8MalzRx+AWzwmjBsBf25cP18DNgmOyTPnpqTl3t2T0iJvLV4Myw4E6oS6wSN+OP+MEBuxOlfB0EfC+uXAu5kK45Rh2Kv3Y2VAFLOZ3uz7TaeNe8F9z647CPPnmdgLubH7ncN4UfFzIG1E+km68Sou3kDxCPEiDnDaUxra2mcT6Ky4hhhrlg\\\\\\\/5uJ9QSGJsbiinUPpfoNfl9+bMJC3pkxiSM4L9Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cFqbA5Dfy6yqN95CKqDjgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b90ac0e100411e5015de270e0db601c8e502a90711aa66f5a8ebe0b434a152e9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O55sdSkTvYg5G8IYp8ztkwJd9A5UIFnZUY+bpF+AmMvBHZTMiSnFIcl0jRlyvvTvcoO2gh7YScuJp7\\\\\\\/7Bqq6eynRbMtferDr0k5QDeNUnh1GiXA4bw920fbwe0vm0BPG1IYzkHN6eCWDtQ8HbcbyL3fisf2xR8FMil0inwPq7AstnfbJlVm3Ic0aEiQuWGeG6a3nyKnMBF8HP1eFwsM5VZfM1xgI8CM9a1Rm6Co2gIDlxoig3zmBGAsfd9ghX7LQ9JCOWEGoFqAhm1qX1rwXsRE+j4AjExo9E6t4X4xXh+CUlUenDqRfot4uuhktycPfDlgnojgVqxJ51FAJBdJdsmPel5SEt4uqCLtXBnq31h0OELJrzX2nzkB8uao\\\\\\\/wfRHvzVcra94QhlFDSn4OX3EHKxFfTSZM7H3r\\\\\\\/v+QURHZbEfsjQc4lThfdYiw+qpSGh6b7rrNMU8LD4EtXnxg0O6XSJrJgUIgCpNkLJtYLzK5vLpWLFG07ASnDX\\\\\\\/9RYj8cgIIgEeRs\\\\\\\/zwyFXlXm9VurmGnNfsRNIZn9Xv2gcRc14XR2qg\\\\\\\/HvMZwx9rCHdQTsGWk9x7jd6\\\\\\\/WCShnJBXqA4d4Ci8TyN3xGozsWUFJGA2PhV2G40f+FeinxOc5i83Rmg+SVoeWvFDDG7We2tHvn2myFNrYBaQHFttgO0FgbjKm28rnHOQAmBJuTQoqo5g\\\\\\\/7tDImypQNa6PDhLPzRoGFxHgtXDsASZMURR+cnnj4k+aqRAti97MnQT6nNNI85BSwQNVjtQMChmY0j3huK6mAhpApWjPqcAklVFsaUPBr0vU19ckPK4IeP4trqjH5nbC0zGl5oXiGycteflWskxJZs2E2JVP0VRQVuDQKmD+VpZGctPdD9y6+d5bjsSKbhxYTrCMRAEtS+4ZYPsMRu1lRNbmbklPqsAq2rG0zTB1E8QSd\\\\\\\/BTcPgyx6NF12zONYP5plo7pPrpFzQkKLdHReCYsQcIm55m6kfOyqY9GEF\\\\\\\/2GWw6bwU3kdhRMQzgEJ7i9b2LkB1kuTHCaCIrN1rJHWa5XcMLYgP0BV0fObFU2bXB1EdJJl2BXLXIjHQXx6WaZ306+83ZDaCHmyWnwzU6M\\\\\\\/wAYfKRwLjMiWDRvLy4JR6AV7KtYMAtmZMKkyozrb\\\\\\\/R5jBLFsbBMehd4UpRl5xVhJxftDOtgU7cGEo8GzUO+djqHvTPg\\\\\\\/By5lWsKWhACbCpndo5n+B0UsTWl4R1FGV4RUleIgkOYyyAh5syc89a7qsL5RRYdqMBtk0jmPwrsz1XuD\\\\\\\/v7zxCRn9zE8xANyCddJVDXWoeE5m2iP65wkk7ETsVrxr57p5DXyMNz71yRzLGziA+blKFWaLJy5p08UnC0y51svG1qIkwGe6vM1hx0FoTR+VLLmfVj5z+L5fOBVbc4ljemufe5HmomxeTwuAUXZIqNkPYP7RAY6kIo7r6ZaYN83WY+B7kulIMeNdBeJnUNjQeQAK+nBR7bk9jYS4giuLI4lF1DqJu2XmoOKJEgZ\\\\\\\/kAezyLpFIileaUcHPrRhOOAsukSCkYOqNOT+QjcyidAo875RDrgzTpAgYqI93s1+uOgMm6504K3dpLfyvRd2XC7fYvW7UgYMuxI7rwz12gsDydh1En+TlEaV8Dql9kdahTmqDpl3SRrxKDJJg\\\\\\\/L+O4eyXHZ+GBsCIdWEJDHZ4K2luCd7c3eJ3rULThK3LHLd+P0miFjFXtILJht9LBP7iWPe92OIhjvew5hbuoqH2oFMJ61nWUNF1YcgQP6Amqe0kAJp9dvThnwIzlra7jCG79kPndE+XQO6TBIpctpO9s+e11MxuCyLykLVxsJwG20TQl0c\\\\\\\/skYB+lgN7BzcGOn\\\\\\\/8Quaf+d8K0QQ7l0v+yjccQfTrGGjMlrRMaSoF44PVwQW+bi0ulD49RTIFqb+SZFxaMPoUouuC7WnqlD01174C2Z7KtQUY566THl6ftlhgyFrZGWRGwX\\\\\\\/R2GfSCOC0\\\\\\\/tPD625TD+6WOgb6DzMhKOazw2xsB3TSA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EvipwhF7JaB\\\\\\\/p8rRSCnuYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a9c56956d2d325ddc1c43a90ff605bafcda18d400974c103c53341e6fb97d413\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459830 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/681" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SoyWESQ-uKAVGrG-TevmQ7v1rkW17iDDRpVUy42-Wfc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/681" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NheG4E7tGONNGFZLNEsO6xTHb6_ijbRirPkALJFwa4E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/669" title="Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/693" title="unfulfilled Interacting Genes Display Branch-Specific Roles in the Development of Mushroom Body Axons in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459830" id="node459830" data-pisa="ggg;4/4/681" data-pisa-master="ggg;g3.114.010488" data-apath="/ggg/4/4/681.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evidence that Natural Selection on Codon Usage in <em>Drosophila pseudoobscura</em> Varies Across Codons</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Richard M. Kliman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">681-692;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459830" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Richard M. Kliman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Cedar Crest College, Allentown, Pennsylvania 18104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BM.%2BKliman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kliman%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BM.%2BKliman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rmkliman@cedarcrest.edu" class="" data-icon-position="" data-hide-link-title="0">rmkliman@cedarcrest.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/4/681" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459830" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/681.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459830" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/681.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459830" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/4/681.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459830" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67880 first last"><a href="/content/4/8/1565" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - August 01, 2014</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Like other species of Drosophila, <em>Drosophila pseudoobscura</em> has a distinct bias toward the usage of C- and G-ending codons. Previous studies have indicated that this bias is due, at least in part, to natural selection. Codon bias clearly differs among amino acids (and other codon classes) in Drosophila, which may reflect differences in the intensity of selection on codon usage. Ongoing natural selection on synonymous codon usage should be reflected in the shapes of the site frequency spectra of derived states at polymorphic positions. Specifically, regardless of other demographic effects on the spectrum, it should be shifted toward higher values for changes from less-preferred to more-preferred codons, and toward lower values for the converse. If the intensity of natural selection is increased, shifts in the site frequency spectra should be more pronounced. A total of 33,729 synonymous polymorphic sites on Chromosome 2 in <em>D. pseudoobscura</em> were analyzed. Shifts in the site frequency spectra are consistent with differential intensity of natural selection on codon usage, with stronger shifts associated with higher codon bias. The shifts, in general, are greater for polymorphic synonymous sites than for polymorphic intron sites, also consistent with natural selection. However, unlike observations in <em>D. melanogaster</em>, codon bias is not reduced in areas of low recombination in <em>D. pseudoobscura</em>; the site frequency spectrum signal for selection on codon usage remains strong in these regions. However, diversity is reduced, as expected. It is possible that estimates of low recombination reflect a recent change in recombination rate.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/codon-bias" class="hw-term hw-article-keyword hw-article-keyword-codon-bias" rel="nofollow">codon bias</a></li><li class="kwd"><a href="/keyword/natural-selection" class="hw-term hw-article-keyword hw-article-keyword-natural-selection" rel="nofollow">natural selection</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila-pseudoobscura" class="hw-term hw-article-keyword hw-article-keyword-drosophila-pseudoobscura" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila pseudoobscura</em></a></li><li class="kwd"><a href="/keyword/site-frequency-spectrum" class="hw-term hw-article-keyword hw-article-keyword-site-frequency-spectrum" rel="nofollow">site frequency spectrum</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li></ul><p id="p-3">The relative usage of synonymous codons varies among genes within an organism. In some organisms (<em>e.g.</em>, humans), this variation may largely reflect base composition variation across the genome (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardi <em>et al.</em> 1985</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kliman and Bernal 2005</a>). In many organisms, however, natural selection appears to directly influence codon usage, with positive correlations between the levels of codon bias and gene expression that are consistent with selection on transcriptional efficiency and/or fidelity (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chavancy <em>et al.</em> 1979</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gouy and Gautier 1982</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ikemura 1985</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akashi 1994</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Moriyama and Powell 1997</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kliman and Henry 2005</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Plotkin and Kudla 2011</a>). This relationship was first reported for <em>Drosophila</em> by <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Shields <em>et al.</em> (1988)</a>, and numerous studies using diverse approaches have supported the hypothesis that natural selection influences codon usage in several <em>Drosophila</em> species (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kliman and Hey 1993</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akashi and Schaeffer 1997</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carlini and Stephan 2003</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Haddrill <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">De Proce <em>et al.</em> 2012</a>). Effective weak selection on codon usage requires a sufficient effective population size to overcome the effects of genetic drift [although see <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hershberg and Petrov (2008)</a>, who point out that selection on codon usage may not be always be weak]. However, evidence has been emerging that natural selection may even be influencing codon usage in humans (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lavner and Kotlar 2005</a>), mammals more generally (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yang and Nielsen 2008</a>), and other vertebrates (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doherty and McInerney 2013</a>).</p><p id="p-4">Among the studies supporting the hypothesis that selection influences codon usage are (1) those that show shifts in the site frequency spectra (SFS) of derived states at synonymous polymorphic sites, such that the SFS is shifted toward higher values for changes to more preferred codons (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akashi and Schaeffer 1997</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kliman 1999</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Llopart <em>et al.</em> 2008</a>); and (2) those that show significantly reduced codon bias in areas of the genome with very low recombination rates (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kliman and Hey 1993</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hey and Kliman 2002</a>). The latter is consistent with the expectation that natural selection will be less effective in the absence of recombination due to linkage disequilibrium among targets of selection; these are often in repulsion due to independent emergence of selectively favored or disfavored mutations on different copies of a chromosome in a population (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hill and Robertson 1966</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Felsenstein 1974</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">McVean and Charlesworth 2000</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Comeron <em>et al.</em> 2008</a>). Limited recombination among targets of selection is also predicted to lead to reduced diversity, whether by selective sweeps that wipe out standing variation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Maynard Smith and Haigh 1974</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gillespie 2000</a>) or by background selection against continually arising deleterious mutations that prevent diversity from accumulating in the first place (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charlesworth <em>et al.</em> 1993</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">McVean and Charlesworth 2000</a>). This prediction was most notably confirmed by <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Begun and Aquadro (1992)</a> in <em>D. melanogaster</em> and has been corroborated by subsequent studies in <em>Drosophila</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Comeron <em>et al.</em> 2012</a>) and other organisms [reviewed by <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nachman (2002)</a> and <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Stephan (2010)</a>].</p><p id="p-5">These earlier studies often relied on estimates of recombination rate derived by fitting recombination maps to physical maps, using a variety of line- or curve-fitting approaches. The advent of ânext-generationâ DNA-sequencing methods has allowed investigators to identify numerous single-nucleotide polymorphisms that can be used in testcrosses to directly estimate recombination rate at a fine scale. <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cirulli <em>et al.</em> (2007)</a> directly estimated recombination rates across a section of the <em>D. pseudoobscura</em> X chromosome and found considerable heterogeneity in recombination rate. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kulathinal <em>et al.</em> (2008)</a> showed that estimates of recombination rate at finer scales (<em>i.e.</em>, with more densely spaced markers) correlated better with diversity, a finding that suggests that fine-scale recombination rates are more reliable when they can be obtained. <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> (2012)</a> extended this work to three complete chromosomes, not only confirming heterogeneity, but showing that estimates could be replicated using crosses of different strains. Importantly, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> (2012)</a> also sequenced 10 additional <em>D. pseudoobscura</em> genomes (along with those of other close relatives) and observed the predicted correlation between recombination rate and diversity. <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Stevison and Noor (2010)</a> observed a similar relationship between fine-scale recombination rate and diversity in closely related <em>D. persimilis</em>.</p><p id="p-6">Although previous polymorphism-based studies on natural selection codon usage in <em>D. pseudoobscura</em> have relied on hundreds of variable sites, the Chromosome 2 data from <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> (2012)</a> provide tens of thousands of variable sites. These data, therefore, allow us to much more thoroughly investigate the effects of natural selection on codon usage in this species. In addition to providing increased statistical power to detect subtle effects, it becomes possible to subdivide data and retain statistical power to test for differential effects. In particular, we investigate whether variation among amino acids in the degree of codon bias can be explained by variation in selection intensity. Furthermore, because of its generally higher recombination rate, analysis with <em>D. pseudoobscura</em> provides a valuable contrast to <em>D. melanogaster</em>.</p><p id="p-7">As expected, we observe a fairly strong correlation between recombination rate and diversity. Although other composition-biasing influences may be influencing patterns of diversity, most notably G/C-biased gene conversion (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Marais <em>et al.</em> 2001</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2003</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Singh <em>et al.</em> 2005</a>), a comparison of SFSs of synonymous and intron sites shows that the SFS shifts are significantly stronger at synonymous sites. Therefore, although selectively neutral influences may be partially responsible for the observed SFS shifts, the data support an influence of natural selection on codon usage. Furthermore, differences among subsets of codons in the SFS shifts are consistent with the differential influence of natural selection. We do not, however, find that selection on codon usage is consistently weaker in areas with lower estimates of recombination rate in <em>D. pseudoobscura</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data set</h3><p id="p-8">Chromosome 2 was recently sequenced in 10 strains of <em>D. pseudoobscura</em>, along with one strain of the outgroup <em>D. lowei</em>, using Illumina platforms (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> 2012</a>) (National Center for Biotechnology Information sequence read archive accession numbers SRA044960.1, SRA044955.2, and SRA044956.1). The reference strain of <em>D. pseudoobscura</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Richards <em>et al.</em> 2005</a>) v2.9 was also included in analyses. The <em>D. pseudoobscura</em> strains represent nearly isogenic lines generated by full-sibling matings over several generations (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Machado <em>et al.</em> 2002</a>; <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> 2012</a>). Population structure is very limited in <em>D. pseudoobscura</em> (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Schaeffer and Miller 1992</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Noor <em>et al.</em> 2000</a>), making it unlikely that the choice of strains, including the reference strain, would influence patterns of diversity. Of the 55 possible pairwise contrasts, 10 of the 11 that showed the lowest pairwise difference at synonymous sites included the reference strain. The reference strain also contributes the smallest number of derived singletons to the polymorphism data set, suggesting that, if anything, there are fewer sequencing errors in the reference strain than in the others.</p></div><div id="sec-3" class="subsection"><h3>Genes</h3><p id="p-9">Genes were excluded from analysis if any of the following criteria were met in the reference sequence: the annotated start codon was not AUG; the annotated stop codon was not UGA, UAG, or UAA; any amino acid codon was incompletely resolved; there was a premature stop codon; or the intron/exon boundaries were noncanonical. A total of 3548 genes met all inclusion criteria.</p></div><div id="sec-4" class="subsection"><h3>Recombination rates</h3><p id="p-10">Estimates were obtained from a pair of testcrosses (Flagstaff and Pikes Peak) described in <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> (2012)</a>. The Flagstaff testcross involved two strains from Arizona bearing the âArrowheadâ arrangement on chromosome 3; the Pikes Peak testcross involved two lines from New Mexico bearing the âPikes Peakâ arrangement on chromosome 3. As these authors noted, recombination rates estimated from the independent testcrosses were very similar along Chromosome 2 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Estimates were obtained for 140 segments in the Flagstaff cross and for 158 segments in the Pikes Peak cross. Except near the ends of the chromosome, most positions are represented in both recombination maps. Unless otherwise stated, for all analyses, the average of the two recombination rates was used for these positions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/681/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recombination rate along Chromosome 2. Estimates are from independent testcrosses reported in McGaugh et al. (2012)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349730852" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Recombination rate along Chromosome 2. Estimates are from independent testcrosses reported in McGaugh &lt;em&gt;et al.&lt;/em&gt; (2012).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/681/F1.medium.gif" width="440" height="170"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/4/681/F1.medium.gif" width="440" height="170"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/681/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/681/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464100" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Recombination rate along Chromosome 2. Estimates are from independent testcrosses reported in <a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> (2012)</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Site inclusion criteria</h3><p id="p-12">For each sequence, excluding the reference strain, a base was considered unresolved (âNâ) if either the phred consensus quality score was below 30 or the depth of coverage in the alignment was below 15 Ã . To be included in the analysis, an intron site or complete codon (<em>i.e.</em>, all three sites) had to be completely resolved in all 12 strains. A total of 35,376 codons meeting these criteria displayed synonymous polymorphism in <em>D. pseudoobscura</em>. Of these, 33,755 segregated two character states; 26 of these loci were in regions for which no recombination rate has been obtained (<a id="xref-ref-35-8" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> 2012</a>) and were excluded from analyses involving recombination. A total of 68,332 intron sites meeting the inclusion criteria were polymorphic in <em>D. pseudoobscura</em>; of these, 65,266 segregated two character states.</p></div><div id="sec-6" class="subsection"><h3>Statistical analyses</h3><p id="p-13">Statistical analyses were performed using R v3.0.1 for Mac OS X (the R Foundation for Statistical Computing), implemented in RStudio v0.97.551 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">RStudio 2013</a>).</p></div><div id="sec-7" class="subsection"><h3>Inference of preferred codons</h3><p id="p-14">Following <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hey and Kliman (2002)</a>, preferred codons were inferred by factor analysis on the 59 codons that encode the 18 degenerate amino acids. Only genes that used all 18 amino acids were included in the factor analysis. The primary factor was polarized, such that values correlated positively with Chi/L (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Shields <em>et al.</em> 1988</a>) and negatively with effective number of codons (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wright 1990</a>); both are measures of uneven codon usage, with Chi/L increasing and ENC decreasing as codon usage becomes less even. Codons that loaded positively on the primary factor were considered âpreferred.â The degree of preference (or âpreference scoreâ) of each codon was defined as its loading score (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Llopart <em>et al.</em> 2008</a>).</p><p id="p-15">For each synonymous polymorphic site, Îpref was defined as pref<sub>derived</sub> â pref<sub>ancestral</sub>, where pref<sub>derived</sub> and pref<sub>ancestral</sub> are the codon preference scores for the derived and ancestral states, respectively, as inferred by parsimony (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Llopart <em>et al.</em> 2008</a>). For convenience, a site is defined as PâU if Îpref is negative, and as UâP if Îpref is positive. This designation is not clear-cut for amino acids with degeneracy above 2; for example, a mutation may substitute one preferred codon with another preferred codon. However, for our analyses, the polarity of the fitness effect is more important than the assignment of âpreferredâ or âunpreferred.â</p></div><div id="sec-8" class="subsection"><h3>Analysis of diversity</h3><p id="p-16">SFS of derived preferred <em>vs.</em> derived unpreferred codons (or similar contrasts) were compared using parametric and nonparametric tests (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Akashi and Schaeffer 1997</a>), as well as a permutation test (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Llopart <em>et al.</em> 2008</a>; described in context under <em>Site frequency spectra relative to Îpref</em>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Inference of preferred codons</h3><p id="p-17">All C-ending codons, and all but three G-ending codons, are preferred in <em>D. pseudoobscura</em>. All A-and T-ending codons are unpreferred (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464278/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464278/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Codon preference scores</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Estimates of diversity, divergence, and codon bias</h3><p id="p-19">Synonymous sites were counted taking into account the degeneracy of codons. For example, a fourfold degenerate codon would provide one synonymous site, whereas a twofold degenerate codon would provide 1/3 of a synonymous site. A total of 543,985 synonymous sites and 2,746,629 intron sites were completely resolved in all 12 strains. The <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Watterson (1975)</a> estimator of synonymous Î¸ in <em>D. pseudoobscura</em> was 0.0222 across all sites; average synonymous divergence from <em>D. lowei</em> was 0.0760. Diversity and divergence varied among amino acids (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), with twofold degenerate amino acids having higher values of both. Wattersonâs estimator of intron Î¸ in <em>D. pseudoobscura</em> was 0.0085; divergence from <em>D. lowei</em> was 0.0297. The lower values for intron sites probably reflect a larger denominator, as each intron site was counted as one full site.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464219/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464219/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimates of synonymous divergence and diversity</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Association of diversity and codon bias with recombination rate</h3><p id="p-26">Using average recombination rate (Flagstaff and Pikes Peak), loci were placed into 25 recombination categories: 0.00â0.25 cM/Mb, &gt;0.25â0.50 cm/Mb, ..., &gt;5.75â6.00 cM/Mb, and &gt;6.00 cM/Mb. (No sites are in regions with &gt;0.25â0.50 cM/Mb.) As shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, synonymous diversity increases with recombination rate (defined by the upper bound of the category) until about 2 cM/Mb, at which point diversity levels off [<em>r</em> = 0.800, 21 degrees of freedom (d.f.), 1-tailed <em>P</em> &lt; 10<sup>â5</sup>]. A very similar relationship is observed for intron diversity (<em>r</em> = 0.703, 21 d.f., 1-tailed <em>P</em> &lt; 10<sup>â4</sup>) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). However, there is no clear relationship between recombination rate and the frequency of optimal codons [F<sub>op</sub>, a measure of preferred codon usage (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sharp and Devine 1989</a>)] (<em>r</em> = 0.209, 21 d.f., 1-tailed <em>P</em> = 0.158) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/681/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diversity and codon bias relative to recombination rate. Points are plotted at the upper end of the recombination rate range (e.g., at 0.25 for 0&#x2212;0.25 cM/Mb); the red point represents sites in regions with recombination rate above 6 cM/Mb. (A) Synonymous diversity measure is the Watterson (1975) estimator of &#x3B8;. (B) Intron diversity. (C) Codon bias measure is Fop (Sharp and Devine 1989)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349730852" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diversity and codon bias relative to recombination rate. Points are plotted at the upper end of the recombination rate range (&lt;em&gt;e.g.&lt;/em&gt;, at 0.25 for 0&#x2212;0.25 cM/Mb); the red point represents sites in regions with recombination rate above 6 cM/Mb. (A) Synonymous diversity measure is the Watterson (1975) estimator of &#x3B8;. (B) Intron diversity. (C) Codon bias measure is F&lt;sub&gt;op&lt;/sub&gt; (Sharp and Devine 1989).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/681/F2.medium.gif" width="243" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/4/681/F2.medium.gif" width="243" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/681/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/681/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464201" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Diversity and codon bias relative to recombination rate. Points are plotted at the upper end of the recombination rate range (<em>e.g.</em>, at 0.25 for 0â0.25 cM/Mb); the red point represents sites in regions with recombination rate above 6 cM/Mb. (A) Synonymous diversity measure is the <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Watterson (1975)</a> estimator of Î¸. (B) Intron diversity. (C) Codon bias measure is F<sub>op</sub> (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Sharp and Devine 1989</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Synonymous diversity varied spatially along Chromosome 2. Using the 142 segments defined by the Flagstaff recombination map (the 140 regions with recombination rate estimates, along with the two external regions), there is obvious heterogeneity in levels of synonymous diversity (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). However, there is also considerable variation in the number of sites analyzed, with as few as 14 sites to as many as 22,635. With a median of 2897 sites per segment, 18 of the 142 segments had fewer than 1000 sites, and 29 had more than 5000 sites. When only the latter are plotted to minimize sampling error (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), diversity is clearly reduced at the two ends of the chromosome, and it is also somewhat suppressed in the center of the chromosome. As expected based on the analysis of recombination rate classes, there is a positive correlation between recombination rate in each segment and synonymous diversity (<em>r</em> = 0.533, 2-tailed <em>P</em> = 0.0026 for the regions with at least 5000 sites). The correlation, although slightly weaker (<em>r</em> = 0.350), remains highly significant (<em>P</em> &lt; 10<sup>â4</sup>) when all regions are included. For intron sites, the correlation between diversity and recombination rate is somewhat weaker, but still significant, for regions with at least 5,000 sites (<em>r</em> = 0.268, <em>n</em> = 120, <em>P</em> = 0.0029). However, when regions with fewer sites are included, the correlation is lost (<em>r</em> = â0.029). In contrast to diversity, variation along Chromosome 2 in F<sub>op</sub> appears to be negligible (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and D</a>). Results were qualitatively similar, including the significant correlation between recombination rate and either synonymous or intron diversity, using the Pikes Peak recombination map (data not shown).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/681/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diversity and codon bias along Chromosome 2. (A) Diversity in all recombination map segments. Segments upstream (FLint_upout) and downstream (FLint_dnout) of the recombination map are also shown; for these, there is no corresponding recombination rate estimate. (B) Diversity in segments with at least 5000 synonymous sites. (C) Codon bias (Fop) in all segments. (D) Codon bias in segments with at least 5,000 synonymous sites. Recombination rates (cM/Mb from the Flagstaff testcross) are shown for reference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349730852" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diversity and codon bias along Chromosome 2. (A) Diversity in all recombination map segments. Segments upstream (FLint_upout) and downstream (FLint_dnout) of the recombination map are also shown; for these, there is no corresponding recombination rate estimate. (B) Diversity in segments with at least 5000 synonymous sites. (C) Codon bias (F&lt;sub&gt;op&lt;/sub&gt;) in all segments. (D) Codon bias in segments with at least 5,000 synonymous sites. Recombination rates (cM/Mb from the Flagstaff testcross) are shown for reference.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/681/F3.medium.gif" width="185" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/4/681/F3.medium.gif" width="185" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/681/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/681/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Diversity and codon bias along Chromosome 2. (A) Diversity in all recombination map segments. Segments upstream (FLint_upout) and downstream (FLint_dnout) of the recombination map are also shown; for these, there is no corresponding recombination rate estimate. (B) Diversity in segments with at least 5000 synonymous sites. (C) Codon bias (F<sub>op</sub>) in all segments. (D) Codon bias in segments with at least 5,000 synonymous sites. Recombination rates (cM/Mb from the Flagstaff testcross) are shown for reference.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Site frequency spectra relative to Îpref</h3><p id="p-30">Under a constant-<em>N</em><sub>e</sub> Wright-Fisher neutral model, the relative frequency of sites with <em>d</em>-derived states is 1/<em>d</em>, such that the expected mean <em>d</em> is (<em>k</em> â 1)/<em>a</em>, where <em>k</em> is the sample size and <em>a</em> is the sum of 1, 1/2, ..., 1/(<em>k</em> â 1) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fu 1995</a>). For <em>k</em> = 11 <em>D. pseudoobscura</em> sequences, we expect 3.414 derived states/site. Overall, our data for synonymous polymorphic sites indicate a shift of the SFS toward lower values, with a mean of 2.59 derived states/site. For intron sites, the mean was 2.279 derived states/site. However, there are noticeably raised tails in the SFSs, likely due in part to ancestral state misassignment (ASM; discussed in <em>Impact of ASM</em>).</p><p id="p-31">If natural selection is acting on codon usage, then the SFSs for PâU and UâP changes should be shifted relative to each other; <em>i.e.</em>, the SFS for UâP changes should be shifted toward higher values (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Akashi and Schaeffer 1997</a>). Consistent with natural selection, the synonymous SFS was shifted toward higher values for the 9918 UâP sites (mean = 3.395) than for the 23,811 PâU sites (mean = 2.258) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). However, the SFS for the UâP sites is not right-shifted relative to the expectations of a constant-<em>N</em><sub>e</sub> Wright-Fisher neutral model; this may indicate a demographic effect on diversity, such as historically increasing <em>N</em><sub>e</sub> (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tajima 1989a</a>). The difference between UâP and PâU sites is highly significant using either a 1-tailed Studentâs <em>t</em>-test (t.test in R, <em>t</em> = 33.715, <em>P</em> &lt; 10<sup>â15</sup>) or a 1-tailed Mann-Whitney <em>U</em>-test (following <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Akashi and Schaeffer (1997)</a>) (wilcox.test in R, <em>P</em> &lt; 10<sup>â15</sup>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/681/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sites frequency spectrum for synonymous polymorphic sites. Shown are sites that segregate two codons and fall within a region for which recombination rate was estimated. &#x201C;P to U,&#x201D; a change to a more unpreferred codon; &#x201C;U to P,&#x201D; a change to a more preferred codon." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349730852" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sites frequency spectrum for synonymous polymorphic sites. Shown are sites that segregate two codons and fall within a region for which recombination rate was estimated. &#x201C;P to U,&#x201D; a change to a more unpreferred codon; &#x201C;U to P,&#x201D; a change to a more preferred codon.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/681/F4.medium.gif" width="440" height="258"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/4/681/F4.medium.gif" width="440" height="258"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/681/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/681/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464233" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">Sites frequency spectrum for synonymous polymorphic sites. Shown are sites that segregate two codons and fall within a region for which recombination rate was estimated. âP to U,â a change to a more unpreferred codon; âU to P,â a change to a more preferred codon.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Given that singletons (<em>i.e.</em>, sites with 1 or 10 individuals carrying the inferred derived state) are the most likely polymorphic sites to represent sequencing errors, the analyses were repeated excluding 5045 UâP and 14,912 PâU singletons. The SFS shift remained highly significant (UâP mean = 4.456; PâU mean = 3.767; <em>t</em> = 16.807, <em>P</em> &lt; 10<sup>â15</sup>; <em>U</em>-test, <em>P</em> &lt; 10<sup>â15</sup>). Under a constant-<em>N</em><sub>e</sub> Wright-Fisher neutral model, the expected frequency of derived states per nonsingleton site would be 4.374. Thus, the SFS for UâP sites was shifted slightly toward higher values, whereas that for PâU sites was shifted toward lower values.</p><p id="p-34">Analyses were repeated for each amino acid. In every case, the SFS was shifted toward higher values for UâP sites than for PâU sites, although there was variation among amino acids in the extent of the shift (see <em>Variation among amino acids in the SFS shift</em>). The results were qualitatively unchanged when singletons were excluded (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464200/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464200/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Shifts in site frequency spectra for each amino acid</span><div class="sb-div caption-clear"></div></div></div><p id="p-38"><a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Llopart <em>et al.</em> (2008)</a> proposed an alternative test, based on the prediction that natural selection should lead to a positive relationship between <em>d</em> and Îpref. Computationally, the sum of <em>d</em> Ã Îpref can serve as a proxy for a correlation or regression coefficient; therefore, 1-tailed statistical significance can be estimated from the proportion of random permutations of <em>d</em> <em>vs.</em> Îpref that lead to a higher sum of products. The test is significant for all amino acids (including or excluding singletons; see <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>), as well as for all sites pooled.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464218/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464218/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Shifts in site frequency spectra for each amino acid, Monte Carlo analyses<sup><em><sup>a</sup></em></sup></span><div class="sb-div caption-clear"></div></div></div><p id="p-42"><a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Llopart <em>et al.</em> (2008)</a> also proposed a modification to the test to correct for ASM. Essentially, a simple Bayesian approach was suggested to calculate a posterior odds ratio of correct assignment by parsimony to ASM. The likelihood ratio was based on the neutral SFS in a constant-N population: likelihood ratio = (<em>k</em> â <em>d</em>)/<em>d</em>, where <em>d</em> is the frequency of derived states assuming parsimony, <em>k</em> is the number of individuals in the sample (here, 11), and <em>k</em> â <em>d</em> is the number of derived states when parsimony is incorrect (<em>i.e.</em>, when there is ASM). The prior odds ratio is the relative probability of no mutation on the branch connecting the outgroup to the base of the ingroup coalescent to the probability of a single mutation on that branch:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/681/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/681/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/681/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/681/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/681/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/681/embed/mml-math-4.gif"/></noscript></span></span> are estimates of divergence and diversity, respectively. Thus, following <a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Llopart <em>et al.</em> (2008)</a>, the posterior odds ratio is calculated as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/681/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4/681/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>The permutation test can then be performed after randomly assigning an ancestral state for each site using the posterior odds ratio. As shown in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, this usually slightly more conservative test (using distinct estimates of Î¸ and <em>D</em> for each amino acid) remains significant for all analyses.</p><p id="p-43">Following up on the recombination analyses, we compared the slopes of the regression lines (<em>d</em> on Îpref) for sites in four recombination classes [following <a id="xref-ref-35-9" class="xref-bibr" href="#ref-35">McGaugh <em>et al.</em> (2012)</a>]: 0â0.5, &gt;0.5â3.0, &gt;3.0â6.0, and &gt;6.0 cM/Mb. For the individual testcrosses (Flagstaff and Pikes Peak), we compared sites in 0-recombination regions to sites found elsewhere. The slope was not lower for the 0â0.5 cM/Mb class (<em>b</em> = 0.864, <em>n</em> = 1164) than for the three other classes (&gt;0.5â3.0 cM/Mb: <em>b</em> = 0.648, <em>n</em> = 14,088; &gt;3.0â6.0 cM/Mb, <em>b</em> = 0.672, <em>n</em> = 13,817; &gt;6.0 cM/Mb, <em>b</em> = 0.832, <em>n</em> = 4660). For the Flagstaff testcross, the slope was significantly lower for the 0-recombination regions (<em>b</em> = 0.164, <em>n</em> = 217) than for higher-recombination regions (<em>b</em> = 0.694, <em>n</em> = 33,226; <em>P</em> = 0.0002, 1-tailed Tukey-Kramer test). This result was not mirrored, however, for the Pikes Peak testcross (0-recombination regions: <em>b</em> = 0.903, <em>n</em> = 1467; higher recombination regions: <em>b</em> = 0.679, <em>n</em> = 32,262). In fact, mean <em>d</em> for UâP changes exceeded mean <em>d</em> for PâU changes in all five regions with recombination estimates of 0, the difference ranging from 0.804 (interval 152, <em>n</em> = 258) to 2.011 (interval 151, <em>n</em> = 133). One-tailed Mann-Whitney <em>U</em>-tests were significant after sequential Bonferroni correction (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rice 1989</a>) for four of the five contrasts. The permutation test on each of the five regions produced similar results; all five tests were significant after Bonferroni correction assuming parsimony, and three were significant after Bonferroni correction when allowing for ASM. Therefore, if the slope of the regression line corresponds to effectiveness of selection on codon usage, there is only equivocal evidence for an effect of low recombination in <em>D. pseudoobscura</em>.</p></div><div id="sec-14" class="subsection"><h3>Variation among amino acids in the SFS shift</h3><p id="p-44">As noted previously, for all amino acids, the average frequency of derived states at synonymous polymorphic sites was greater for PâU changes than for UâP changes. This result is consistent with natural selection on synonymous codon usage. However, it is also consistent with the G/C-biased gene conversion (although recent work by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Comeron <em>et al.</em> (2012)</a> suggests that that this does not occur in <em>D. melanogaster</em>). In the latter, individuals heterozygous for a preferred and an unpreferred codon will usually be segregating a pair of purines or a pair of pyrimidines at the synonymous site (usually the third position of a codon). If heteroduplex intermediates generated during crossing-over tend to resolve toward G or C, then this process could lead to shifts in the SFS even if crossing-over is not, itself, mutagenic.</p><p id="p-45">In the standard genetic code, there are 16 T/C-ending synonymous codon pairs and 13 A/G-ending synonymous codon pairs. Although the degree of bias varies, C- or G-ending codons are usually used disproportionately (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). The C-ending codon is always used disproportionately, although barely so for Asp (50.5% C-ending). For the A/G pairs, the G-ending codon is used disproportionately in all cases except for Gly, where unpreferred GGG is used less often than unpreferred GGA.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464094/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464094/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Analysis of variance (site type <strong>Ã</strong> direction)</span><div class="sb-div caption-clear"></div></div></div><p id="p-49">For all 29 pairs, the average frequency of derived states per polymorphic site is higher for TâC or AâG sites than for corresponding CâT or GâA sites. If biased gene conversion is responsible for these SFS shifts, the magnitude of the shifts should be similar for all codon pairs (at least within the A/G or C/T class). There is, however, considerable variation among codon pairs in relative codon usage and the difference in derived states/site (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). For C/T pairs, two-way analysis of variance [ANOVA; codon pair by direction (CâT <em>vs.</em> TâC)] indicated highly significant effects of codon pair (<em>F</em><sub>15,12681</sub> = 3.859, <em>P</em> &lt; 10<sup>â6</sup>) and direction (<em>F</em><sub>1,12681</sub> = 730.5, <em>P</em> &lt; 10<sup>â15</sup>), as well as a highly significant interaction effect (<em>F</em><sub>15,12681</sub> = 3.250, <em>P</em> &lt; 10<sup>â4</sup>). Similar results were obtained for A/G pairs (site type: <em>F</em><sub>12,9514</sub> = 2.841, <em>P</em> &lt; 10<sup>â4</sup>; direction: <em>F</em><sub>1,9514</sub> = 576.1, <em>P</em> &lt; 10<sup>â15</sup>; interaction: <em>F</em><sub>12,9514</sub> = 5.607, <em>P</em> &lt; 10<sup>â8</sup>). For the C/T pairs, the difference in mean <em>d</em> correlates moderately with the difference in the preference scores between the C- and T-ending codons, although not well with the degree of bias (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and B</a>). For the A/G pairs, the difference correlates well with degree of bias, and somewhat with the difference in G- and A-ending codon preference scores (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, C and D</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/681/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shifts in site frequency spectra among codon pairs. (A) Difference in average frequency of derived states/polymorphic site for C/T codon pairs relative to codon usage (i.e., proportion of C-ending codons). (B) Difference in average frequency of derived states/site for C/T codon pairs relative to &#x394;pref for T&#x2192;C changes. (C, D) Corresponding figures for G/A codon pairs. Letters in legend correspond to single-letter amino acid codes; blue, fourfold degenerate amino acids; light blue, codon pair from fourfold degenerate subclass of sixfold degenerate amino acids; red, codon pair from isoleucine or twofold degenerate subclass of sixfold degenerate amino acids; gold, twofold degenerate amino acids. Dashed lines correspond to linear regression through all points." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349730852" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Shifts in site frequency spectra among codon pairs. (A) Difference in average frequency of derived states/polymorphic site for C/T codon pairs relative to codon usage (&lt;em&gt;i.e.&lt;/em&gt;, proportion of C-ending codons). (B) Difference in average frequency of derived states/site for C/T codon pairs relative to &#x394;pref for T&#x2192;C changes. (C, D) Corresponding figures for G/A codon pairs. Letters in legend correspond to single-letter amino acid codes; blue, fourfold degenerate amino acids; light blue, codon pair from fourfold degenerate subclass of sixfold degenerate amino acids; red, codon pair from isoleucine or twofold degenerate subclass of sixfold degenerate amino acids; gold, twofold degenerate amino acids. Dashed lines correspond to linear regression through all points.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/681/F5.medium.gif" width="180" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/4/681/F5.medium.gif" width="180" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/681/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/681/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464254" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">Shifts in site frequency spectra among codon pairs. (A) Difference in average frequency of derived states/polymorphic site for C/T codon pairs relative to codon usage (<em>i.e.</em>, proportion of C-ending codons). (B) Difference in average frequency of derived states/site for C/T codon pairs relative to Îpref for TâC changes. (C, D) Corresponding figures for G/A codon pairs. Letters in legend correspond to single-letter amino acid codes; blue, fourfold degenerate amino acids; light blue, codon pair from fourfold degenerate subclass of sixfold degenerate amino acids; red, codon pair from isoleucine or twofold degenerate subclass of sixfold degenerate amino acids; gold, twofold degenerate amino acids. Dashed lines correspond to linear regression through all points.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">It is worth contrasting the SFS shifts for codons to those of introns, as shifts in SFSs may provide insight into composition-biasing process, such as biased gene conversion. Using <em>D. lowei</em> to infer the ancestral states, the SFSs for all 12 possible changes were obtained. Of particular note are those for CâT, TâC, GâA, and AâG, because these mirror the changes discussed previously for codons. The differences in mean derived states/site, although highly significant, are not as pronounced for introns as they are for synonymous sites. For CâT and TâC, mean <em>d</em> was 1.977 and 2.802, respectively (a difference of 0.825). This contrasts with means of 2.231 and 3.582, respectively, for all C/T-segregating codons (a difference of 1.351). Two-way ANOVA on <em>d</em> (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>) indicated highly significant effects of site type (intron <em>vs.</em> codon) and direction, as well as a site type Ã direction interaction. Similarly, for GâA and AâG intron sites, mean <em>d</em> was 1.939 and 2.885, respectively (a difference of 0.946). This contrasts with means of 2.187 and 3.614 for all G/A-segregating codons (a difference of 1.427). Again, two-way ANOVA on <em>d</em> indicated highly significant effects of site type and direction, as well as a strong site type Ã direction interaction. The site type Ã direction interaction effects indicate significantly larger shifts in codons relative to introns, as expected if the SFS shifts in codons are due to selection, and not only a composition-biasing influence shared by all sites, such as G/C-biased gene conversion.</p><p id="p-52">It is worth noting that restricting the analysis to the much smaller subset of sites in small introns does not qualitatively affect the results. Following <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Halligan and Keightley (2006)</a>, who proposed that short introns were less constrained than longer introns, sites were restricted to introns of 80 bp or shorter, excluding the first nine and last eight bases adjacent to splice junctions. There were 2414 CâT sites and 1544 GâA sites. The SFS shift for CâT sites was slightly reduced (0.645), whereas the SFS shift for GâA sites (0.935) was essentially unchanged.</p><p id="p-53">Mean <em>d</em> in introns was nearly identical for GâC and CâG (2.341 and 2.375, respectively) and for AâT and TâA (2.117 and 2.174, respectively); neither difference was statistically significant. Two-way ANOVA indicated only significant effects of site type (intron <em>vs.</em> codon); of note, there was no significant interaction effect (nor was there a significant direction effect). Intermediate results were obtained for GâT <em>vs.</em> TâG and CâA <em>vs.</em> AâC (both favoring changes toward G or C); all three effects in the ANOVA were significant. The general implication is that the forces shifting SFSs for synonymous codon pairs are stronger than those for introns, consistent with previous observations in Drosophila (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">De Proce <em>et al.</em> 2012</a>). Therefore, although there may be some effect of biased gene conversion on codon usage, this is insufficient to explain our observations.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-54">Our analyses corroborate the likely influence of natural selection on codon usage in <em>Drosophila</em>, specifically <em>D. pseudoobscura</em>. We also observe, as expected, a positive correlation between diversity and recombination rate. As shown by <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kulathinal <em>et al.</em> (2008)</a>, this association is stronger when recombination is estimated at a finer scale. However, in contrast to analyses on <em>D. melanogaster</em>, we observe no significant association of codon bias with recombination rate, despite considerable statistical power and reliable estimates of recombination rate at the finer scale advocated by <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kulathinal <em>et al.</em> (2008)</a>. Evidence from polymorphism data for reduced natural selection on codon usage in areas of low recombination in our dataset is limited at best. It is possible that recombination rate is recently reduced in these areas, such that <em>N</em><sub>e</sub>-reducing effects on diversity of linkage (especially by positive selection on recent beneficial mutations) are apparent, but the <em>N</em><sub>e</sub> remains sufficient for effective selection. Changes in codon bias would not become apparent for some time, given that this requires accumulation of synonymous substitutions.</p><p id="p-55">Both findings indicate that recombination rate across chromosome 2 exceeds the threshold necessary for effective natural selection. Recombination rates in <em>D. pseudoobscura</em> are generally higher than those estimated for <em>D. melanogaster</em> (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Comeron <em>et al.</em> 2012</a>). In <em>D. melanogaster</em>, 21% of intervals had recombination rate estimates below 0.5 cM/Mb, in contrast with 6% in <em>D. pseudoobscura</em>. Although 59% of intervals had recombination rates above 1.5 cM/Mb in <em>D. melanogaster</em> (56% for autosomal regions only), this value was exceeded in 82% of intervals in <em>D. pseudoobscura</em>. Furthermore, while the local <em>N</em><sub>e</sub> could effectively vary across the chromosome due to the <em>N</em><sub>e</sub>-reducing effect of selection on linked sites in areas of lower recombination, the generally higher <em>N</em><sub>e</sub> in <em>D. pseudoobscura</em> may mitigate the Hill-Robertson effect to some extent. That is, the product of <em>N</em><sub>e</sub> and <em>s</em> may be sufficient over most of the chromosome for selection to effectively fix the optimal codons.</p><div id="sec-16" class="subsection"><h3>Distortion of the SFS</h3><p id="p-56">The shape of the overall SFS for the 32,729 synonymous polymorphic sites clearly differs from that expected for neutral variants in a constant-N population. First, the proportion of derived singletons (55.1%) is much higher than the 34.1% expected in a population of constant size (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Second, there is a raised âtailâ in the SFS, with more sites having 10 derived states than 9 derived states.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/681/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Site frequency spectra corrected for sequencing error and ancestral state misassignment (ASM). Expected proportions under a constant-Ne Wright-Fisher neutral model are shown in black; our data, assuming parsimony, are shown in blue. (A) Correction for ASM based on observed levels of diversity and divergence (following Llopart et al. (2008)). (B) Correction for ASM with a 0.1% sequencing error rate. (C) Correction for ASM with a 0.54% error rate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1349730852" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Site frequency spectra corrected for sequencing error and ancestral state misassignment (ASM). Expected proportions under a constant-&lt;em&gt;N&lt;/em&gt;&lt;sub&gt;e&lt;/sub&gt; Wright-Fisher neutral model are shown in black; our data, assuming parsimony, are shown in blue. (A) Correction for ASM based on observed levels of diversity and divergence (following Llopart &lt;em&gt;et al.&lt;/em&gt; (2008)). (B) Correction for ASM with a 0.1% sequencing error rate. (C) Correction for ASM with a 0.54% error rate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/681/F6.medium.gif" width="235" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/4/681/F6.medium.gif" width="235" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/681/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/681/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464213" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-57" class="first-child">Site frequency spectra corrected for sequencing error and ancestral state misassignment (ASM). Expected proportions under a constant-<em>N</em><sub>e</sub> Wright-Fisher neutral model are shown in black; our data, assuming parsimony, are shown in blue. (A) Correction for ASM based on observed levels of diversity and divergence (following <a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">Llopart <em>et al.</em> (2008)</a>). (B) Correction for ASM with a 0.1% sequencing error rate. (C) Correction for ASM with a 0.54% error rate.</p><div class="sb-div caption-clear"></div></div></div><p id="p-58">Biological explanations for an excess of singletons include purifying background selection (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Charlesworth <em>et al.</em> 1993</a>) and population expansion (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Tajima 1989a</a>), both of which distort the coalescent of a population with constant <em>N</em><sub>e</sub> to increase the relative lengths of branches upon which mutations would be observed as singletons (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Tajima 1989a</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">b</a>). Background selection or a demographic influence on the SFS should affect UâP and PâU sites similarly, but we observe a significantly stronger shift toward low-frequency derived states in PâU sites. Therefore, while both influences may be at work in <em>D. pseudoobscura</em>, they are not sufficient to explain our observations.</p><p id="p-59">In addition to demographic effects, an excess of singletons can arise, in principle, from sequencing error. With a low value of Î¸, most sites in a small sample (here, <em>k</em> = 11) would be invariant. A single error at an invariant site would produce an apparent singleton. Additional errors would add to the remainder of the SFS, but assuming that errors are independent, the impact would be seen mainly on singletons. Assuming a 0.1% error rate (equivalent to a phred consensus quality score of 30), following the binomial distribution, 98.91% of truly invariant sites would be observed as invariant. However, 1.09% of invariant sites would be apparent singletons, whereas only 5.5 Ã 10<sup>â3</sup>% would appear to have two derived states (assuming that all errors produce the same character state). Likewise, 99.00% of true singletons would be observed as singletons, whereas 0.99% would present as having two derived states (<em>i.e.</em>, be observed as âdoubletonsâ) and 4.4 Ã 10<sup>â3</sup>% would present as having three derived states. This approach can be extended to all possible values for true and observed derived states (<em>i.e.</em>, for a true invariant site appearing to have 1, 2, ... 10 derived states; for a true singleton appearing to have 2, 3... 10 derived states; and so on), ultimately leading to 42.3% of sites having one apparent derived state. This is well below the observed value of 55.1%, which would require an error rate of approximately 0.33%. Given the requirements of consensus quality scores of 30 or better, a minimum of 15 Ã coverage in highly inbred strains, and full resolution of all three codon positions in every strain (including the outgroup), sequencing error alone does not explain the high proportion of singletons observed.</p><p id="p-60">It is further unlikely that the resequencing using the Illumina platform is leading to an accumulation of false As and Ts. Across a range of genomic G+C contents, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nakamura <em>et al.</em> (2011)</a> found that A/TâG/C errors were more likely than G/CâA/T errors. Although Illumina sequencing is more prone to base call errors than either ABI SOLiD or Roche 454 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2012</a>), error bias is unlikely to explain our observed shifts in SFSs. Most A/TâG/C polymorphic sites reflect UâP changes in <em>D. pseudoobscura</em> yet the proportion of singletons is much lower for UâP sites (43.1%) than it is for PâU sites (60.7%; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p></div><div id="sec-17" class="subsection"><h3>SFS differences among codon pairs</h3><p id="p-61">The SFS data are consistent with an influence of natural selection on codon usage in <em>D. pseudoobscura</em>, corroborating previous studies (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Akashi and Schaeffer 1997</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Haddrill <em>et al.</em> 2011</a>). Additionally, the data indicate that intensity of selection varies among synonymous mutations. <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Vicario <em>et al.</em> (2007)</a> previously proposed this possibility in a comparison of codon usage in the genomes of 12 Drosophila species. Although there is a correspondence between codon bias and SFS shifts consistent with differential selection, the nature of that differential selection is uncertain. Selection does not appear to be strongest on the more common amino acids within a degeneracy class, as might be expected for selection on efficiency of translation. For example, phenylalanine and tyrosine are used at an intermediate level among the C/T-ending twofold degenerate amino acids, despite showing the strongest SFS shifts in this class. Alanine is the most commonly used fourfold degenerate amino acid, but its SFS shifts for CâT and AâG changes are intermediate within the degeneracy class. On the other hand, for sixfold degenerate amino acids, leucine and serine are used more often than arginine (and are among the most used amino acids overall) and show much stronger SFS shifts for changes in the fourfold degenerate subsets; in fact, their SFS shifts are among the strongest observed in this analysis. <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Vicario <em>et al.</em> (2007)</a> proposed <em>ad hoc</em> explanations for stronger selection on some amino acids (<em>e.g.</em>, for accurate translation of disulfide bridge-forming cysteine or for accurate and efficient translation of heavily used hydrophobic leucine). Potential influences of isoaccepting tRNA pools on codon bias have been identified for Drosophila (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Moriyama and Powell 1997</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Powell and Moriyama 1997</a>), but these authors note that pools likely change over time within an individual (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">White <em>et al.</em> 1973</a>) and that this plasticity may itself influence codon bias among amino acids. Furthermore, the complement of tRNA genes likely changes over evolutionary time in Drosophila, with evidence of numerous gains, losses, and reassignments (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rogers <em>et al.</em> 2010</a>). The observed slightly stronger shifts for AâG <em>vs.</em> CâT synonymous changes may reflect differences in composition-biasing influences, as reflected in corresponding SFS shifts for introns. It is, therefore, probably premature to speculate too extensively on the bases of differential selection among amino acids.</p><p id="p-62">That this main result is not an artifact of sequencing error is reinforced by differences among codon pairs in the proportion of derived singletons and the proportion of polymorphic sites. Variation in the proportion of derived singletons among all 16 ancestrally C-ending codons is not quite significant (<em>G</em> = 24.91, 15 d.f., <em>P</em> = 0.0511), and there is no significant variation for ancestral T-ending codons (<em>G</em> = 14.85, 15 d.f., <em>P</em> = 0.462). However, variation in the proportion of derived singletons is significant for both ancestral G-ending (<em>G</em> = 21.42, 12 d.f., <em>P</em> = 0.0445) and A-ending codons (<em>G</em> = 22.98, 12 d.f., <em>P</em> = 0.0279).</p><p id="p-63">For C/T codon pairs, 3.25% of ancestral C-ending codons are polymorphic, whereas 2.51% of ancestral T-ending codons are polymorphic; this difference is highly significant (<em>G</em> test of independence, <em>G</em> = 189.4, 1 d.f., <em>P</em> &lt; 10<sup>â15</sup>). Although this finding could indicate a higher probability of CâT errors (which is not likely with Illumina platforms; see <em>Distortion of the SFS</em>), there is considerable variation among C/T codon pairs in the proportion of polymorphic ancestral C-ending codons (<em>G</em> test of independence: <em>G</em> = 199.3, 15 d.f., <em>P</em> &lt; 10<sup>â15</sup>) and polymorphic T-ending codons (<em>G</em> = 197.4, 15 d.f., <em>P</em> &lt; 10<sup>â15</sup>). Similar results were obtained for G/A codon pairs; 3.13% of ancestral G-ending codons were polymorphic and 2.22% of ancestral A-ending codons were polymorphic (<em>G</em> = 224.5, 1 d.f., <em>P</em> &lt; 10<sup>â15</sup>) . However, the proportions varied among codon pairs (ancestral G-ending: <em>G</em> = 102.2, 12 d.f., <em>P</em> &lt; 10<sup>â15</sup>; ancestral A-ending: <em>G</em> = 42.7, 12 d.f., <em>P</em> &lt; 10<sup>â4</sup>) (see <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> for proportions). Examining codon pairs from fourfold degenerate sites only, <em>G</em> tests remained significant for all four ancestral bases (all <em>P</em>-values below 10<sup>â4</sup>). Likewise, for twofold degenerate sites only, <em>G</em> tests were highly significant for ancestral C-ending and T-ending codons (<em>P</em> &lt; 10<sup>â15</sup>) and for ancestral G-ending codons (<em>P</em> = 0.00012); however, there was no significant heterogeneity among codon pairs for ancestral A-ending codons (<em>P</em> = 0.403). Thus, there is considerable variation in the proportion of polymorphic codons, even within similar degeneracy classes, a result that does not support a major role for sequencing error, but is consistent with differences among codon pairs in the influence of weak selection.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464097/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464097/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Summary data for C/T and G/A segregating and fixed different codon third positions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Impact of ASM</h3><p id="p-69">The raised tail of the SFS may be explained by ASM, especially if the SFS is already left-shifted. Under a model with constant <em>N</em><sub>e</sub>, we expect a ratio of 10:9 for sites with 9 <em>vs.</em> 10 derived states. However, if we allow ASM with a probability of 1/(LR+1), where LR is the likelihood ratio in Equation 1, we can solve for the value of divergence required for a given value of Î¸ that would lead to a 1:1 ratio by (a) factoring in sites with 1 or 2 derived states that present as having 10 or 9 derived states and (b) factoring out sites with 10 or 9 derived states that would present as having 1 or 2 derived states. For our Î¸ = 0.0222, a divergence of 0.0424 would be sufficient and would lead to 4.1% of sites presenting 9 and 4.1% of sites presenting 10 derived states under a constant-N model. While both values exceed our observations, the excess of singletons by necessity decreases the proportion of sites in the remainder of the remainder of the SFS. We estimated per-site synonymous divergence at 0.0763, which leads to a slightly raised tail (a ratio of 1.12 for 10:9 derived states); we observed a ratio of 1.54.</p><p id="p-70">If we assume a constant-N model, but with ASM and a sequencing error rate of 0.1%, we can begin to approach the observed SFS (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). However, we still observe an excess of singletons; we still need a much higher error rate to approach the observed SFS. In fact, for PâU sites, the error rate would have to be approximately 0.54% to produce the extreme left shift. It is not possible to reproduce the SFS for UâP sites, with a slight excess of singletons and a markedly raised tail, with sequencing error and ASM alone.</p><p id="p-71">Analysis of SFS of derived synonymous mutations in <em>D. pseudoobscura</em> indicates that the intensity of natural selection varies among classes of synonymous mutation. The shapes of the SFSs are likely shaped by other influences, possibly including ASM and sequencing error, but variation among synonymous codon pairs in the extent of the SFS shifts support differential intensity of selection.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">I thank Mohamed Noor and Laurence Loewe for helpful discussions and comments on the manuscript. I also thank an anonymous reviewer for helpful comments. Josep Comeron provided estimates of recombination rate in <em>Drosophila melanogaster</em>. Suzanne McGaugh provided Pileup alignments used to generate the data set. This work was supported by National Institutes of Health grant R01 GM086445 to R.M.K. and to M. A. F. Noor.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-22"><p id="p-73"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-08-31"><span class="received-label">Received </span>August 31, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-02-10"><span class="accepted-label">Accepted </span>February 10, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2014 Kliman</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Synonymous codon usage in <em>Drosophila melanogaster</em>: natural selection and translational accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">927</span>â<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D136%26rft.issue%253D3%26rft.spage%253D927%26rft.epage%253D935%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2BDrosophila%2Bmelanogaster%253A%2Bnatural%2Bselection%2Band%2Btranslational%2Baccuracy.%26rft_id%253Dinfo%253Apmid%252F8005445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM2LzMvOTI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Natural selection and the frequency distributions of âsilentâ DNA polymorphism in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">295</span>â<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D295%26rft.epage%253D307%26rft.atitle%253DNatural%2BSelection%2Band%2Bthe%2BFrequency%2BDistributions%2Bof%2B%2560%2560Silent%2527%2527%2BDNA%2BPolymorphism%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F9136019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzEvMjk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.3" data-doi="10.1038/356519a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Levels of naturally occurring DNA polymorphism correlate with recombination rates in <em>D. melanogaster</em></span> <span class="cit-comment">(see comments)</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">356</span>: <span class="cit-fpage">519</span>â<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D356%26rft.issue%253D6369%26rft.spage%253D519%26rft.epage%253D520%26rft.atitle%253DLevels%2Bof%2Bnaturally%2Boccurring%2BDNA%2Bpolymorphism%2Bcorrelate%2Bwith%2Brecombination%2Brates%2Bin%2BD.%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F356519a0%26rft_id%253Dinfo%253Apmid%252F1560824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/356519a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1560824&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HN23100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.4" data-doi="10.1126/science.4001930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olofsson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zerial</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">The mosaic genome of warm-blooded vertebrates.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">228</span>: <span class="cit-fpage">953</span>â<span class="cit-lpage">958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBernardi%26rft.auinit1%253DG%26rft.volume%253D228%26rft.issue%253D4702%26rft.spage%253D953%26rft.epage%253D958%26rft.atitle%253DThe%2Bmosaic%2Bgenome%2Bof%2Bwarm-blooded%2Bvertebrates%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.4001930%26rft_id%253Dinfo%253Apmid%252F4001930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyMjgvNDcwMi85NTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlini</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">In vivo introduction of unpreferred synonymous codons into the Drosophila Adh gene results in reduced levels of ADH protein.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">239</span>â<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCarlini%26rft.auinit1%253DD.%2BB.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D239%26rft.epage%253D243%26rft.atitle%253DIn%2BVivo%2BIntroduction%2Bof%2BUnpreferred%2BSynonymous%2BCodons%2BInto%2Bthe%2BDrosophila%2BAdh%2BGene%2BResults%2Bin%2BReduced%2BLevels%2Bof%2BADH%2BProtein%26rft_id%253Dinfo%253Apmid%252F12586711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvMjM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">The effect of deleterious mutations on neutral molecular variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1289</span>â<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D134%26rft.issue%253D4%26rft.spage%253D1289%26rft.epage%253D1303%26rft.atitle%253DThe%2Beffect%2Bof%2Bdeleterious%2Bmutations%2Bon%2Bneutral%2Bmolecular%2Bvariation.%26rft_id%253Dinfo%253Apmid%252F8375663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNC80LzEyODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.7" data-doi="10.1016/S0300-9084(79)80314-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chavancy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chevallier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fournier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garel</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span>â<span class="cit-article-title">Adaptation of iso-tRNA concentration to mRNA codon frequency in the eukaryote cell.</span> <abbr class="cit-jnl-abbrev">Biochimie</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">71</span>â<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D61%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0300-9084%252879%252980314-4%26rft_id%253Dinfo%253Apmid%252F435560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0300-9084(79)80314-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=435560&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.8" data-doi="10.1007/s00239-006-0142-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cirulli</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Fine-scale crossover rate heterogeneity in <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">129</span>â<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCirulli%26rft.auinit1%253DE.%2BT.%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D135%26rft.atitle%253DFine-scale%2Bcrossover%2Brate%2Bheterogeneity%2Bin%2BDrosophila%2Bpseudoobscura.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-006-0142-7%26rft_id%253Dinfo%253Apmid%252F17160365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-006-0142-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17160365&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243047800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.9" data-doi="10.1371/journal.pgen.1002905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnappan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The many landscapes of recombination in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1002905%26rft.epage%253De1002905%26rft.atitle%253DThe%2BMany%2BLandscapes%2Bof%2BRecombination%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002905%26rft_id%253Dinfo%253Apmid%252F23071443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071443&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.10" data-doi="10.1038/sj.hdy.6801059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The Hill-Robertson effect: evolutionary consequences of weak selection and linkage in finite populations.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">19</span>â<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D100%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6801059%26rft_id%253Dinfo%253Apmid%252F17878920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6801059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17878920&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.11" data-doi="10.1098/rsbl.2011.0601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Proce</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Selection on codon usage and base composition in <em>Drosophila americana</em>.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">82</span>â<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2011.0601%26rft_id%253Dinfo%253Apmid%252F21849309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveWJpb2xldHQiO3M6NToicmVzaWQiO3M6NjoiOC8xLzgyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.12" data-doi="10.1093/molbev/mst128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doherty</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McInerney</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Translational selection frequently overcomes genetic drift in shaping synonymous codon usage patterns in vertebrates.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2263</span>â<span class="cit-lpage">2267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst128%26rft_id%253Dinfo%253Apmid%252F23883522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMC8yMjYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span>â<span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>â<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.14" data-doi="10.1006/tpbi.1995.1025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Statistical properties of segregating sites.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">172</span>â<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BX.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D172%26rft.epage%253D197%26rft.atitle%253DStatistical%2Bproperties%2Bof%2Bsegregating%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1995.1025%26rft_id%253Dinfo%253Apmid%252F7482370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1995.1025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7482370&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RZ41300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Genetic drift in an infinite population. The pseudohitchhiking model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">909</span>â<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGillespie%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D909%26rft.epage%253D919%26rft.atitle%253DGenetic%2BDrift%2Bin%2Ban%2BInfinite%2BPopulation%253A%2BThe%2BPseudohitchhiking%2BModel%26rft_id%253Dinfo%253Apmid%252F10835409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.16" data-doi="10.1093/nar/10.22.7055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span>â<span class="cit-article-title">Codon usage in bacteria: correlation with gene expressivity.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">7055</span>â<span class="cit-lpage">7074</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGouy%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D22%26rft.spage%253D7055%26rft.epage%253D7074%26rft.atitle%253DCodon%2Busage%2Bin%2Bbacteria%253A%2Bcorrelation%2Bwith%2Bgene%2Bexpressivity%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F10.22.7055%26rft_id%253Dinfo%253Apmid%252F6760125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8yMi83MDU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.17" data-doi="10.1093/molbev/msq354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Determinants of synonymous and nonsynonymous variability in three species of Drosophila.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1731</span>â<span class="cit-lpage">1743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq354%26rft_id%253Dinfo%253Apmid%252F21191087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzUvMTczMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.18" data-doi="10.1101/gr.5022906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Ubiquitous selective constraints in the Drosophila genome revealed by a genome-wide interspecies comparison.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">875</span>â<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5022906%26rft_id%253Dinfo%253Apmid%252F16751341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzcvODc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.19" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Selection on codon bias.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">287</span>â<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Interactions between natural selection, recombination and gene density in the genes of Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">595</span>â<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHey%26rft.auinit1%253DJ.%26rft.volume%253D160%26rft.issue%253D2%26rft.spage%253D595%26rft.epage%253D608%26rft.atitle%253DInteractions%2BBetween%2BNatural%2BSelection%252C%2BRecombination%2Band%2BGene%2BDensity%2Bin%2Bthe%2BGenes%2Bof%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F11861564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYwLzIvNTk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.21" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span>â<span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>â<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">Codon usage and tRNA content in unicellular and multicellular organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">13</span>â<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjY6IjIvMS8xMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.23" data-doi="10.1007/PL00006557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Recent selection on synonymous codon usage in Drosophila.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">343</span>â<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKliman%26rft.auinit1%253DR.%2BM.%26rft.volume%253D49%26rft.issue%253D3%26rft.spage%253D343%26rft.epage%253D351%26rft.atitle%253DRecent%2Bselection%2Bon%2Bsynonymous%2Bcodon%2Busage%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006557%26rft_id%253Dinfo%253Apmid%252F10473775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10473775&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083473400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Reduced natural selection associated with low recombination in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1239</span>â<span class="cit-lpage">1258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DKliman%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D1239%26rft.epage%253D1258%26rft.atitle%253DReduced%2Bnatural%2Bselection%2Bassociated%2Bwith%2Blow%2Brecombination%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F8277853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjEwLzYvMTIzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.25" data-doi="10.1086/425668"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">A. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Inference of codon preferences in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">3</span>â<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BSci.%26rft.volume%253D166%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1086%252F425668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/425668&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227523200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.26" data-doi="10.1016/j.gene.2005.04.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernal</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Unusual usage of AGG and TTG codons in humans and their viruses.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">352</span>: <span class="cit-fpage">92</span>â<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DKliman%26rft.auinit1%253DR.%2BM.%26rft.volume%253D352%26rft.spage%253D92%26rft.epage%253D99%26rft.atitle%253DUnusual%2Busage%2Bof%2BAGG%2Band%2BTTG%2Bcodons%2Bin%2Bhumans%2Band%2Btheir%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2005.04.001%26rft_id%253Dinfo%253Apmid%252F15922516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2005.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15922516&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.27" data-doi="10.1073/pnas.0801848105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulathinal</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Fine-scale mapping of recombination rate in Drosophila refines its correlation to diversity and divergence.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">10051</span>â<span class="cit-lpage">10056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0801848105%26rft_id%253Dinfo%253Apmid%252F18621713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzI5LzEwMDUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.28" data-doi="10.1016/j.gene.2004.11.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavner</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotlar</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Codon bias as a factor in regulating expression via translation rate in the human genome.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">127</span>â<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DLavner%26rft.auinit1%253DY.%26rft.volume%253D345%26rft.issue%253D1%26rft.spage%253D127%26rft.epage%253D138%26rft.atitle%253DCodon%2Bbias%2Bas%2Ba%2Bfactor%2Bin%2Bregulating%2Bexpression%2Bvia%2Btranslation%2Brate%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2004.11.035%26rft_id%253Dinfo%253Apmid%252F15716084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2004.11.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716084&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227202000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.29" data-doi="10.1155/2012/251364"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Comparison of next-generation sequencing systems.</span> <abbr class="cit-jnl-abbrev">J. Biomed. Biotechnol.</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">251364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiomed%2BBiotechnol%26rft.aulast%253DLiu%26rft.auinit1%253DL.%26rft.volume%253D2012%26rft.spage%253D251364%26rft.epage%253D251364%26rft.atitle%253DComparison%2Bof%2Bnext-generation%2Bsequencing%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2012%252F251364%26rft_id%253Dinfo%253Apmid%252F22829749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2012/251364&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22829749&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.30" data-doi="10.1007/s00239-008-9072-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Llopart</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mabille</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters-Hall</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">A new test for selection applied to codon usage in <em>Drosophila simulans</em> and <em>D. mauritiana</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">224</span>â<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DLlopart%26rft.auinit1%253DA.%26rft.volume%253D66%26rft.issue%253D3%26rft.spage%253D224%26rft.epage%253D231%26rft.atitle%253DA%2Bnew%2Btest%2Bfor%2Bselection%2Bapplied%2Bto%2Bcodon%2Busage%2Bin%2BDrosophila%2Bsimulans%2Band%2BD.%2Bmauritiana.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-008-9072-x%26rft_id%253Dinfo%253Apmid%252F18288436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-008-9072-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18288436&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255629600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.31" data-doi="10.1093/oxfordjournals.molbev.a004103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markert</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Inferring the history of speciation from multilocus DNA sequence data: the case of <em>Drosophila pseudoobscura</em> and close relatives.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">472</span>â<span class="cit-lpage">488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DMachado%26rft.auinit1%253DC.%2BA.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D472%26rft.epage%253D488%26rft.atitle%253DInferring%2Bthe%2BHistory%2Bof%2BSpeciation%2Bfrom%2BMultilocus%2BDNA%2BSequence%2BData%253A%2BThe%2BCase%2Bof%2BDrosophila%2Bpseudoobscura%2Band%2BClose%2BRelatives%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004103%26rft_id%253Dinfo%253Apmid%252F11919289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE5LzQvNDcyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.32" data-doi="10.1073/pnas.091427698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Does recombination improve selection on codon usage? Lessons from nematode and fly complete genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">5688</span>â<span class="cit-lpage">5692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.091427698%26rft_id%253Dinfo%253Apmid%252F11320215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTAvNTY4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.33" data-doi="10.1017/S0016672302006079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Neutral effect of recombination on base composition in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">79</span>â<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DMarais%26rft.auinit1%253DG.%26rft.volume%253D81%26rft.issue%253D2%26rft.spage%253D79%26rft.epage%253D87%26rft.atitle%253DNeutral%2Beffect%2Bof%2Brecombination%2Bon%2Bbase%2Bcomposition%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672302006079%26rft_id%253Dinfo%253Apmid%252F12872909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672302006079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12872909&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183477600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.34" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard Smith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haigh</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span>â<span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">23</span>â<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.35" data-doi="10.1371/journal.pbio.1001422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGaugh</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heil</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manzano-Winkler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loewe</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Recombination modulates how selection affects linked sites in Drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1001422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DMcGaugh%26rft.auinit1%253DS.%2BE.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253De1001422%26rft.epage%253De1001422%26rft.atitle%253DRecombination%2Bmodulates%2Bhow%2Bselection%2Baffects%2Blinked%2Bsites%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001422%26rft_id%253Dinfo%253Apmid%252F23152720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001422&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23152720&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">The effects of Hill-Robertson interference between weakly selected mutations on patterns of molecular evolution and variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">929</span>â<span class="cit-lpage">944</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D929%26rft.epage%253D944%26rft.atitle%253DThe%2BEffects%2Bof%2BHill-Robertson%2BInterference%2BBetween%2BWeakly%2BSelected%2BMutations%2Bon%2BPatterns%2Bof%2BMolecular%2BEvolution%2Band%2BVariation%26rft_id%253Dinfo%253Apmid%252F10835411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.37" data-doi="10.1007/PL00006256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Codon usage bias and tRNA abundance in Drosophila.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">514</span>â<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMoriyama%26rft.auinit1%253DE.%2BN.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D514%26rft.epage%253D523%26rft.atitle%253DCodon%2Busage%2Bbias%2Band%2BtRNA%2Babundance%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006256%26rft_id%253Dinfo%253Apmid%252F9342399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9342399&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YA94500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.38" data-doi="10.1016/S0959-437X(02)00358-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Variation in recombination rate across the genome: evidence and implications.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">657</span>â<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D657%26rft.epage%253D663%26rft.atitle%253DVariation%2Bin%2Brecombination%2Brate%2Bacross%2Bthe%2Bgenome%253A%2Bevidence%2Band%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900358-1%26rft_id%253Dinfo%253Apmid%252F12433578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00358-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433578&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.39" data-doi="10.1093/nar/gkr344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morimoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshikawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Sequence-specific error profile of Illumina sequencers.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr344%26rft_id%253Dinfo%253Apmid%252F21576222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM5LzEzL2U5MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.40" data-doi="10.1017/S0016672399004024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schug</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Microsatellite variation in populations of <em>Drosophila pseudoobscura</em> and <em>Drosophila persimilis</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">25</span>â<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DNoor%26rft.auinit1%253DM.%2BA.%26rft.volume%253D75%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D35%26rft.atitle%253DMicrosatellite%2Bvariation%2Bin%2Bpopulations%2Bof%2BDrosophila%2Bpseudoobscura%2Band%2BDrosophila%2Bpersimilis.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004024%26rft_id%253Dinfo%253Apmid%252F10740918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10740918&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086118600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.41" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>â<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.42" data-doi="10.1073/pnas.94.15.7784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Evolution of codon usage bias in Drosophila.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">7784</span>â<span class="cit-lpage">7790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D94%26rft.issue%253D15%26rft.spage%253D7784%26rft.epage%253D7790%26rft.atitle%253DEvolution%2Bof%2Bcodon%2Busage%2Bbias%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.15.7784%26rft_id%253Dinfo%253Apmid%252F9223264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTUvNzc4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.43" data-doi="10.2307/2409177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">Analyzing tables of statistical tests.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">223</span>â<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D43%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989R828900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.44" data-doi="10.1101/gr.3059305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bettencourt</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hradecky</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Letovsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Comparative genome sequencing of <em>Drosophila pseudoobscura</em>: chromosomal, gene, and cis-element evolution.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DRichards%26rft.auinit1%253DS.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DComparative%2Bgenome%2Bsequencing%2Bof%2BDrosophila%2Bpseudoobscura%253A%2BChromosomal%252C%2Bgene%252C%2Band%2Bcis-element%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3059305%26rft_id%253Dinfo%253Apmid%252F15632085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjY6IjE1LzEvMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC82ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.45" data-doi="10.1093/gbe/evq034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths-Jones</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The evolution of tRNA genes in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>â<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq034%26rft_id%253Dinfo%253Apmid%252F20624748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjIvMC80NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-book" id="cit-4.4.681.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">RStudio</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-source">RStudio: Integregated Development Environment for R (Version 0.97.551)</span>. <span class="cit-publ-name">RStudio</span>, <span class="cit-publ-loc">Boston, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Estimates of gene flow in <em>Drosophila pseudoobscura</em> determined from nucleotide sequence analysis of the alcohol dehydrogenase region.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">471</span>â<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchaeffer%26rft.auinit1%253DS.%2BW.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D471%26rft.epage%253D480%26rft.atitle%253DEstimates%2Bof%2Bgene%2Bflow%2Bin%2BDrosophila%2Bpseudoobscura%2Bdetermined%2Bfrom%2Bnucleotide%2Bsequence%2Banalysis%2Bof%2Bthe%2Balcohol%2Bdehydrogenase%2Bregion.%26rft_id%253Dinfo%253Apmid%252F1427038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzIvNDcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.48" data-doi="10.1093/nar/17.13.5029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devine</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">Codon usage and gene expression level in <em>Dictyostelium discoideum</em>: highly expressed genes do âpreferâ optimal codons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">5029</span>â<span class="cit-lpage">5039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D17%26rft.issue%253D13%26rft.spage%253D5029%26rft.epage%253D5040%26rft.atitle%253DCodon%2Busage%2Band%2Bgene%2Bexpression%2Blevel%2Bin%2BDictyosteiium%2Bdiscoidtum%253A%2Bhighly%2Bexpressed%2Bgenes%2Bdo%2B%255Bprefer%255B%2Boptimal%2Bcodons%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.13.5029%26rft_id%253Dinfo%253Apmid%252F2762118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMy81MDI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span>â<span class="cit-article-title">âSilentâ sites in Drosophila genes are not neutral: evidence of selection among synonymous codons.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">704</span>â<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DShields%26rft.auinit1%253DD.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D704%26rft.epage%253D716%26rft.atitle%253D%2522Silent%2522%2Bsites%2Bin%2BDrosophila%2Bgenes%2Bare%2Bnot%2Bneutral%253A%2Bevidence%2Bof%2Bselection%2Bamong%2Bsynonymous%2Bcodons%26rft_id%253Dinfo%253Apmid%252F3146682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjUvNi83MDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNjgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.50" data-doi="10.1007/s00239-004-0287-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Codon bias and noncoding GC content correlate negatively with recombination rate on the Drosophila X chromosome.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">315</span>â<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSingh%26rft.auinit1%253DN.%2BD.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DCodon%2Bbias%2Band%2Bnoncoding%2BGC%2Bcontent%2Bcorrelate%2Bnegatively%2Bwith%2Brecombination%2Brate%2Bon%2Bthe%2BDrosophila%2BX%2Bchromosome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-004-0287-1%26rft_id%253Dinfo%253Apmid%252F16044248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-004-0287-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16044248&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231732400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.51" data-doi="10.1098/rstb.2009.0278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genetic hitchhiking <em>vs.</em> background selection: the controversy and its implications.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1245</span>â<span class="cit-lpage">1253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DStephan%26rft.auinit1%253DW.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1245%26rft.epage%253D1253%26rft.atitle%253DGenetic%2Bhitchhiking%2Bversus%2Bbackground%2Bselection%253A%2Bthe%2Bcontroversy%2Band%2Bits%2Bimplications%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0278%26rft_id%253Dinfo%253Apmid%252F20308100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU0NC8xMjQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.52" data-doi="10.1007/s00239-010-9388-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevison</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genetic and evolutionary correlates of fine-scale recombination rate variation in <em>Drosophila persimilis</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">332</span>â<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DStevison%26rft.auinit1%253DL.%2BS.%26rft.volume%253D71%26rft.issue%253D5-6%26rft.spage%253D332%26rft.epage%253D345%26rft.atitle%253DGenetic%2Band%2Bevolutionary%2Bcorrelates%2Bof%2Bfine-scale%2Brecombination%2Brate%2Bvariation%2Bin%2BDrosophila%2Bpersimilis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-010-9388-1%26rft_id%253Dinfo%253Apmid%252F20890595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-010-9388-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20890595&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284549800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span><span class="cit-comment">a</span>â<span class="cit-article-title">The effect of change in population size on DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">597</span>â<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D597%26rft.epage%253D601%26rft.atitle%253DThe%2Beffect%2Bof%2Bchange%2Bin%2Bpopulation%2Bsize%2Bon%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2599369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span><span class="cit-comment">b</span>â<span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>â<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.55" data-doi="10.1186/1471-2148-7-226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vicario</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Codon usage in twelve species of Drosophila.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DVicario%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.spage%253D226%26rft.epage%253D226%26rft.atitle%253DCodon%2Busage%2Bin%2Btwelve%2Bspecies%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-226%26rft_id%253Dinfo%253Apmid%252F18005411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18005411&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.56" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span>â<span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>â<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.57" data-doi="10.1016/0012-1606(73)90173-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tener</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span>â<span class="cit-article-title">Analysis of tRNAs during the development of Drosophila.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">185</span>â<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DWhite%26rft.auinit1%253DB.%2BN.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D195%26rft.atitle%253DAnalysis%2Bof%2BtRNAs%2Bduring%2Bthe%2Bdevelopment%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252873%252990173-5%26rft_id%253Dinfo%253Apmid%252F4207975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(73)90173-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4207975&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973Q115300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.58" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">The âeffective number of codonsâ used in a gene.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">23</span>â<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F681.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.4.681.59" data-doi="10.1093/molbev/msm284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Mutation-selection models of codon substitution and their use to estimate selective strengths on codon usage.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">568</span>â<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm284%26rft_id%253Dinfo%253Apmid%252F18178545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNTY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzY4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/4/681.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/4/669" title="Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/693" title="unfulfilled Interacting Genes Display Branch-Specific Roles in the Development of Mushroom Body Axons in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/4"><span>Volume 4 Issue 4, April 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/4.cover-source.jpg?itok=Rf0YhETZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/681" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Qwm8-7C70lKyR4T1dYMhsiu3D9NrrZFcRiW2R2nIV40" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/4/681" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/4/681" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/4/681" class="active" data-icon-position="" data-hide-link-title="0">Evidence that Natural Selection on Codon Usage in Drosophila pseudoobscura Varies Across Codons</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459830" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-vj0gGAUYCxoEidKD3HTBPKdadgQp6RxwFn89W4Z2XtU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/681.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/4/681" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-n163XRcwZEUx3haG9_tIxCcZpCwIIwEVAoR-z5yK3bM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/4/681" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/681.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459830" id="node459830--2" data-pisa="ggg;4/4/681" data-pisa-master="ggg;g3.114.010488" data-apath="/ggg/4/4/681.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evidence that Natural Selection on Codon Usage in <em>Drosophila pseudoobscura</em> Varies Across Codons</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Richard M. Kliman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">681-692;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459830--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Richard M. Kliman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Cedar Crest College, Allentown, Pennsylvania 18104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BM.%2BKliman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kliman%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BM.%2BKliman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rmkliman@cedarcrest.edu" class="" data-icon-position="" data-hide-link-title="0">rmkliman@cedarcrest.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/4/681" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/4/681&amp;title=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/4/681&amp;title=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/4/681&amp;title=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/4/681&amp;text=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/4/681&amp;title=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/4/681&amp;t=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/4/681&amp;title=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/4/681&amp;title=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459830/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459830" id="node459830--3" data-pisa="ggg;4/4/681" data-pisa-master="ggg;g3.114.010488" data-apath="/ggg/4/4/681.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evidence that Natural Selection on Codon Usage in <em>Drosophila pseudoobscura</em> Varies Across Codons</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Richard M. Kliman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">681-692;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.010488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459830--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Richard M. Kliman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, Cedar Crest College, Allentown, Pennsylvania 18104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BM.%2BKliman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kliman%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BM.%2BKliman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rmkliman@cedarcrest.edu" class="" data-icon-position="" data-hide-link-title="0">rmkliman@cedarcrest.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=681&amp;rft.epage=692&amp;rft.atitle=Evidence%20that%20Natural%20Selection%20on%20Codon%20Usage%20in%20Drosophila%20pseudoobscura%20Varies%20Across%20Codons&amp;rft.volume=4&amp;rft.issue=4&amp;rft.date=2014-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kliman%2C+Richard+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459830/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459830/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459830/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459830/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459830/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459830/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459830/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459830/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459830/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459830/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459830/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459830/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/4/681" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/4/681.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/4/681.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/4/4/681.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
