<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/4/553" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/4/553.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/4/553.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/4/553.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/4/553.atom" />
<meta name="HW.pisa" content="ggg;4/4/553" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean" />
<meta name="DC.Identifier" content="10.1534/g3.113.009589" />
<meta name="DC.Date" content="2014-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Li et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Identification of genes underlying agronomic traits is dependent on the segregation of quantitative trait loci (QTL). A popular hypothesis is that elite lines are becoming increasingly similar to each other, resulting in large genomic regions with fixed genes. Here, we resequenced two parental modern elite soybean lines [ ZhongHuang13 (ZH) and ZhongPin03-5373 (ZP)] and discovered 794,876 SNPs with reference to the published Williams82 genome. SNPs were distributed unevenly across the chromosomes, with 87.1% of SNPs clustering in 4.9% of the soybean reference genome. Most of the regions with a high density of SNP polymorphisms were located in the chromosome arms. Moreover, seven large regions that were highly similar between parental lines were identified. A GoldenGate SNP genotyping array was designed using 384 SNPs and the 254 recombinant inbred lines (F8) derived from the cross of ZP × ZH were genotyped. We constructed a genetic linkage map using a total of 485 molecular markers, including 313 SNPs from the array, 167 simple sequence repeats (SSRs), 4 expressed sequence tag–derived SSRs, and 1 insertion/deletion marker. The total length of the genetic map was 2594.34 cM, with an average marker spacing of 5.58 cM. Comparing physical and genetic distances, we found 20 hotspot and 14 coldspot regions of recombination. Our results suggest that the technology of resequencing of parental lines coupled with high-throughput SNP genotyping could efficiently bridge the genotyping gap and provide deep insights into the landscape of recombination and fixed genomic regions in biparental segregating populations of soybean with implications for fine mapping of QTL." />
<meta name="DC.Contributor" content="Ying-hui Li" />
<meta name="DC.Contributor" content="Yu-lin Liu" />
<meta name="DC.Contributor" content="Jochen C. Reif" />
<meta name="DC.Contributor" content="Zhang-xiong Liu" />
<meta name="DC.Contributor" content="Bo Liu" />
<meta name="DC.Contributor" content="Michael F. Mette" />
<meta name="DC.Contributor" content="Ru-zhen Chang" />
<meta name="DC.Contributor" content="Li-juan Qiu" />
<meta name="article:published_time" content="2014-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Identification of genes underlying agronomic traits is dependent on the segregation of quantitative trait loci (QTL). A popular hypothesis is that elite lines are becoming increasingly similar to each other, resulting in large genomic regions with fixed genes. Here, we resequenced two parental modern elite soybean lines [&lt;i&gt;ZhongHuang13&lt;/i&gt; (ZH) and &lt;i&gt;ZhongPin03-5373&lt;/i&gt; (ZP)] and discovered 794,876 SNPs with reference to the published Williams82 genome. SNPs were distributed unevenly across the chromosomes, with 87.1% of SNPs clustering in 4.9% of the soybean reference genome. Most of the regions with a high density of SNP polymorphisms were located in the chromosome arms. Moreover, seven large regions that were highly similar between parental lines were identified. A GoldenGate SNP genotyping array was designed using 384 SNPs and the 254 recombinant inbred lines (F&lt;sub&gt;8&lt;/sub&gt;) derived from the cross of ZP × ZH were genotyped. We constructed a genetic linkage map using a total of 485 molecular markers, including 313 SNPs from the array, 167 simple sequence repeats (SSRs), 4 expressed sequence tag–derived SSRs, and 1 insertion/deletion marker. The total length of the genetic map was 2594.34 cM, with an average marker spacing of 5.58 cM. Comparing physical and genetic distances, we found 20 hotspot and 14 coldspot regions of recombination. Our results suggest that the technology of resequencing of parental lines coupled with high-throughput SNP genotyping could efficiently bridge the genotyping gap and provide deep insights into the landscape of recombination and fixed genomic regions in biparental segregating populations of soybean with implications for fine mapping of QTL.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/04/01" />
<meta name="citation_mjid" content="ggg;4/4/553" />
<meta name="citation_id" content="4/4/553" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/4/553" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/4/553.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/4/553.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/4/553.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.009589" />
<meta name="citation_pmid" content="24476671" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="553" />
<meta name="citation_lastpage" content="560" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ying-hui Li" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China" />
<meta name="citation_author" content="Yu-lin Liu" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China" />
<meta name="citation_author" content="Jochen C. Reif" />
<meta name="citation_author_institution" content="Department of Cytogenetics and Genome Analysis, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Corrensstrasse 3, D-06466 Stadt Seeland, Gatersleben, Germany" />
<meta name="citation_author" content="Zhang-xiong Liu" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China" />
<meta name="citation_author" content="Bo Liu" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China" />
<meta name="citation_author" content="Michael F. Mette" />
<meta name="citation_author_institution" content="Department of Cytogenetics and Genome Analysis, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Corrensstrasse 3, D-06466 Stadt Seeland, Gatersleben, Germany" />
<meta name="citation_author" content="Ru-zhen Chang" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China" />
<meta name="citation_author" content="Li-juan Qiu" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China" />
<meta name="citation_author_email" content="qiulijuan@caas.cn" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. H. Buetow;citation_author=A. Chakravarti;citation_title=Multipoint gene mapping using seriation. I. General methods.;citation_pages=180-188;citation_volume=41;citation_year=1987;citation_issue=2;citation_pmid=3475978" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. Carels;citation_author=A. Barakat;citation_author=G. Bernardi;citation_title=The gene distribution of the maize genome.;citation_pages=11057-11060;citation_volume=92;citation_year=1995;citation_issue=24;citation_pmid=7479936;citation_doi=10.1073/pnas.92.24.11057" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=M. Chen;citation_author=G. Presting;citation_author=W. B. Barbazuk;citation_author=J. L. Goicoechea;citation_author=B. Blackmon;citation_title=An Integrated Physical and Genetic Map of the Rice Genome;citation_pages=537-545;citation_volume=14;citation_year=2002;citation_issue=3;citation_pmid=11910002;citation_doi=10.1105/tpc.010485" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Y. Choi;citation_author=D. L. Hyten;citation_author=L. K. Matukumalli;citation_author=Q. Song;citation_author=J. M. Chaky;citation_title=A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis.;citation_volume=176;citation_year=2007;citation_pmid=17339218;citation_doi=10.1534/genetics.107.070821" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. B. Cregan;citation_author=T. Jarvik;citation_author=A. L. Bush;citation_author=R. C. Shoemaker;citation_author=K. G. Lark;citation_title=An Integrated Genetic Linkage Map of the Soybean Genome;citation_pages=1464-1490;citation_volume=39;citation_year=1999;citation_issue=5;citation_doi=10.2135/cropsci1999.3951464x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. Du;citation_author=Z. Tian;citation_author=Y. Sui;citation_author=M. Zhao;citation_author=Q. Song;citation_title=Pericentromeric effects shape the patterns of divergence, retention, and expression of duplicated genes in the paleopolyploid soybean.;citation_volume=24;citation_year=2012;citation_pmid=22227891;citation_doi=10.1105/tpc.111.092759" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Erayman;citation_author=D. Sandhu;citation_author=D. Sidhu;citation_author=M. Dilbirligi;citation_author=P. S. Baenziger;citation_title=Demarcating the gene-rich regions of the wheat genome;citation_pages=3546-3565;citation_volume=32;citation_year=2004;citation_issue=12;citation_pmid=15240829;citation_doi=10.1093/nar/gkh639" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. S. Gaut;citation_author=S. I. Wright;citation_author=C. Rizzon;citation_author=J. Dvorak;citation_author=L. K. Anderson;citation_title=Recombination: an underappreciated factor in the evolution of plant genomes.;citation_pages=77-84;citation_volume=8;citation_year=2007;citation_issue=1;citation_pmid=17173059;citation_doi=10.1038/nrg1970" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=Z. Gizlice;citation_author=T. E. Carter;citation_author=J. W. Burton;citation_title=Genetic Base for North American Public Soybean Cultivars Released between 1947 and 1988;citation_pages=1143-1151;citation_volume=34;citation_year=1994;citation_issue=5;citation_doi=10.2135/cropsci1994.0011183X003400050001x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High-throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Huang;citation_author=X. Wei;citation_author=T. Sang;citation_author=Q. Zhao;citation_author=Q. Feng;citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces.;citation_pages=961-967;citation_volume=42;citation_year=2010;citation_issue=11;citation_pmid=20972439;citation_doi=10.1038/ng.695" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=T. Y. Hwang;citation_author=T. Sayama;citation_author=M. Takahashi;citation_author=Y. Takada;citation_author=Y. Nakamoto;citation_title=High-density integrated linkage map based on SSR markers in soybean.;citation_volume=16;citation_year=2009;citation_pmid=19531560;citation_doi=10.1093/dnares/dsp010" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. L. Hyten;citation_author=I. Y. Choi;citation_author=Q. Song;citation_author=J. E. Specht;citation_author=T. E. Carter;citation_title=A High Density Integrated Genetic Linkage Map of Soybean and the Development of a 1536 Universal Soy Linkage Panel for Quantitative Trait Locus Mapping;citation_pages=960-968;citation_volume=50;citation_year=2010;citation_issue=3;citation_doi=10.2135/cropsci2009.06.0360" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. L. Hyten;citation_author=Q. Song;citation_author=I. Y. Choi;citation_author=M. S. Yoon;citation_author=J. E. Specht;citation_title=High-throughput genotyping with the GoldenGate assay in the complex genome of soybean.;citation_pages=945-952;citation_volume=116;citation_year=2008;citation_issue=7;citation_pmid=18278477;citation_doi=10.1007/s00122-008-0726-2" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. L. Jannink;citation_author=X.-L. Wu;citation_title=Estimating allelic number and identity in state of QTLs in interconnected families.;citation_pages=133-144;citation_volume=81;citation_year=2003;citation_issue=2;citation_pmid=12872915;citation_doi=10.1017/S0016672303006153" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. Kosambi;citation_title=The estimation of map distance from recombination values.;citation_pages=172-175;citation_volume=12;citation_year=1943;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Lam;citation_author=X. Xu;citation_author=X. Liu;citation_author=W. Chen;citation_author=G. Yang;citation_title=Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection.;citation_pages=1053-1059;citation_volume=42;citation_year=2010;citation_issue=12;citation_pmid=21076406;citation_doi=10.1038/ng.715" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Li;citation_author=J. M. Ribaut;citation_author=Z. Li;citation_author=J. Wang;citation_title=Inclusive composite interval mapping (ICIM) for digenic epistasis of quantitative traits in biparental populations.;citation_pages=243-260;citation_volume=116;citation_year=2008;citation_issue=2;citation_pmid=17985112;citation_doi=10.1007/s00122-007-0663-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Li;citation_author=M. Thomson;citation_author=S. R. McCouch;citation_title=Fine Mapping of a Grain-Weight Quantitative Trait Locus in the Pericentromeric Region of Rice Chromosome 3;citation_pages=2187-2195;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611185;citation_doi=10.1534/genetics.104.034165" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Li;citation_author=C. Yu;citation_author=Y. Li;citation_author=T. W. Lam;citation_author=S. M. Yiu;citation_title=SOAP2: an improved ultrafast tool for short read alignment.;citation_volume=25;citation_year=2009;citation_pmid=19497933;citation_doi=10.1093/bioinformatics/btp336" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=Y. H. Li;citation_author=W. Li;citation_author=C. Zhang;citation_author=L. Yang;citation_author=R. Z. Chang;citation_title=Genetic diversity in domesticated soybean (Glycine max) and its wild progenitor (Glycine soja) for simple sequence repeat and single-nucleotide polymorphism loci.;citation_pages=242-253;citation_volume=188;citation_year=2010;citation_issue=1;citation_pmid=20618914;citation_doi=10.1111/j.1469-8137.2010.03344.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. H. Li;citation_author=S. C. Zhao;citation_author=J. X. Ma;citation_author=D. Li;citation_author=L. Yan;citation_title=Molecular footprints of domestication and improvement in soybean revealed by whole genome re-sequencing.;citation_pages=579;citation_volume=14;citation_year=2013;citation_pmid=23984715;citation_doi=10.1186/1471-2164-14-579" />
<meta name="citation_reference" content="citation_journal_title=Agric. Sci. China;citation_author=Y. L. Liu;citation_author=Y. H. Li;citation_author=G. A. Zhou;citation_author=N. Uzokwe;citation_author=R. Z. Chang;citation_title=Development of soybean EST-SSR markers and their use to assess genetic diversity in the Subgenus Soja.;citation_pages=1423-1429;citation_volume=9;citation_year=2010;citation_doi=10.1016/S1671-2927(09)60233-9" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Nagaki;citation_author=Z. Cheng;citation_author=S. Ouyang;citation_author=P. B. Talbert;citation_author=M. Kim;citation_title=Sequencing of a rice centromere uncovers active genes.;citation_pages=138-145;citation_volume=36;citation_year=2004;citation_issue=2;citation_pmid=14716315;citation_doi=10.1038/ng1289" />
<meta name="citation_reference" content="citation_journal_title=Acta Agron. Sin.;citation_author=H. Y. Nan;citation_author=Y. H. Li;citation_author=R. Z. Chang;citation_author=L. J. Qiu;citation_title=Development and identification of InDel markers based on rhg1 gene for resistance to soybean cyst nematode (Heterodera glycines Ichinohe);citation_pages=1236-1243;citation_volume=35;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=H. P. Piepho;citation_title=Optimal marker density for interval mapping in a backcross population.;citation_pages=437-440;citation_volume=84;citation_year=2000;citation_pmid=10849067;citation_doi=10.1046/j.1365-2540.2000.00678.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schmutz;citation_author=S. B. Cannon;citation_author=J. Schlueter;citation_author=J. Ma;citation_author=T. Mitros;citation_title=Genome sequence of the palaeopolyploid soybean.;citation_pages=178-183;citation_volume=463;citation_year=2010;citation_issue=7278;citation_pmid=20075913;citation_doi=10.1038/nature08670" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Q. Song;citation_author=L. Marek;citation_author=R. Shoemaker;citation_author=K. Lark;citation_author=V. Concibido;citation_title=A new integrated genetic linkage map of the soybean.;citation_pages=122-128;citation_volume=109;citation_year=2004;citation_issue=1;citation_pmid=14991109;citation_doi=10.1007/s00122-004-1602-3" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Q. Song;citation_author=G. Jia;citation_author=Y. Zhu;citation_author=D. Grant;citation_author=R. T. Nelson;citation_title=Abundance of SSR motifs and development of candidate polymorphic SSR markers (BARCSOYSSR_1. 0) in soybean.;citation_pages=1950-1960;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.10.0607" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Spindel;citation_author=M. Wright;citation_author=C. Chen;citation_author=J. Cobb;citation_author=J. Gage;citation_title=Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.;citation_pages=2699-2716;citation_volume=126;citation_year=2013;citation_pmid=23918062;citation_doi=10.1007/s00122-013-2166-x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biotechnol.;citation_author=L. Wei;citation_author=Y. H. Li;citation_author=W. G. Lu;citation_author=R. Z. Chang;citation_author=L. J. Qiu;citation_title=Inducing expression analysis of Gmhs1pro-1 by inoculating soybean cyst nematode 4 race.;citation_pages=77-84;citation_volume=19;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Soybean Sci;citation_author=D. J. Xiong;citation_author=T. J. Zhao;citation_author=J. Y. Gai;citation_title=The core ancestors of soybean cultivars released during 1923–2005 in China.;citation_pages=641-647;citation_volume=26;citation_year=2007" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
<meta name="twitter:description" content="Identification of genes underlying agronomic traits is dependent on the segregation of quantitative trait loci (QTL). A popular hypothesis is that elite lines are becoming increasingly similar to each other, resulting in large genomic regions with fixed genes. Here, we resequenced two parental modern elite soybean lines [ ZhongHuang13 (ZH) and ZhongPin03-5373 (ZP)] and discovered 794,876 SNPs with reference to the published Williams82 genome. SNPs were distributed unevenly across the chromosomes, with 87.1% of SNPs clustering in 4.9% of the soybean reference genome. Most of the regions with a high density of SNP polymorphisms were located in the chromosome arms. Moreover, seven large regions that were highly similar between parental lines were identified. A GoldenGate SNP genotyping array was designed using 384 SNPs and the 254 recombinant inbred lines (F8) derived from the cross of ZP × ZH were genotyped. We constructed a genetic linkage map using a total of 485 molecular markers, including 313 SNPs from the array, 167 simple sequence repeats (SSRs), 4 expressed sequence tag–derived SSRs, and 1 insertion/deletion marker. The total length of the genetic map was 2594.34 cM, with an average marker spacing of 5.58 cM. Comparing physical and genetic distances, we found 20 hotspot and 14 coldspot regions of recombination. Our results suggest that the technology of resequencing of parental lines coupled with high-throughput SNP genotyping could efficiently bridge the genotyping gap and provide deep insights into the landscape of recombination and fixed genomic regions in biparental segregating populations of soybean with implications for fine mapping of QTL." />
<meta name="og-title" property="og:title" content="Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/4/553" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Identification of genes underlying agronomic traits is dependent on the segregation of quantitative trait loci (QTL). A popular hypothesis is that elite lines are becoming increasingly similar to each other, resulting in large genomic regions with fixed genes. Here, we resequenced two parental modern elite soybean lines [ ZhongHuang13 (ZH) and ZhongPin03-5373 (ZP)] and discovered 794,876 SNPs with reference to the published Williams82 genome. SNPs were distributed unevenly across the chromosomes, with 87.1% of SNPs clustering in 4.9% of the soybean reference genome. Most of the regions with a high density of SNP polymorphisms were located in the chromosome arms. Moreover, seven large regions that were highly similar between parental lines were identified. A GoldenGate SNP genotyping array was designed using 384 SNPs and the 254 recombinant inbred lines (F8) derived from the cross of ZP × ZH were genotyped. We constructed a genetic linkage map using a total of 485 molecular markers, including 313 SNPs from the array, 167 simple sequence repeats (SSRs), 4 expressed sequence tag–derived SSRs, and 1 insertion/deletion marker. The total length of the genetic map was 2594.34 cM, with an average marker spacing of 5.58 cM. Comparing physical and genetic distances, we found 20 hotspot and 14 coldspot regions of recombination. Our results suggest that the technology of resequencing of parental lines coupled with high-throughput SNP genotyping could efficiently bridge the genotyping gap and provide deep insights into the landscape of recombination and fixed genomic regions in biparental segregating populations of soybean with implications for fine mapping of QTL." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/4.cover-source.jpg" />
    <title>Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"l7hAx40rdx7dSD9LFE6tAiPjKQxGiDWstQU-M4Azdi0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459851","apath":"\/ggg\/4\/4\/553.atom","pisa":"ggg;4\/4\/553","ac":{"\/ggg\/4\/4\/553.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/4\/553.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/4\/553"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/4\/553"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/4\/553.figures-only":true,"\/content\/4\/4\/553":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022x1u+gmGU5UJ394UeIW4AXscdgp+gwf8GCJOg9Wg7Q4VIrwaGnjBDVtPi6OEgAUOzs2B6xR2baj\\\\\\\/p6n\\\\\\\/zuTW63rKwU8Apxv2glExajo5cmwaA\\\\\\\/ID3GMWoAMlvKiM\\\\\\\/zyqeYaI2gZxGTyXQ7fcwD4Lgy69Waacz2VOP+qtn\\\\\\\/dy3Frd4s3\\\\\\\/0azhvqtuLuXAhYsdOCAtHNUEG1shs9xdGLmNg9kYN7ZYTMGhCKb7szDy9KzODVp305A8yr0zhP+Ye+nyZtQ9GfPwdDqs+ANOrhQdEvxkuQAo+XHQ3airHgUQ9d408lfmbt389lDjGDnuHPAz\\\\\\\/\\\\\\\/nxMPovWyPLYEAAkNVHsQVlY53wyVDJmgTp90lQSldZCkCOeIP1nSNEWjO2o1T3ZZgoFYjHkLmiIDXj6Ho2DDSif9Zz0OHYJ8gTZW8tlszRT+VGlglib4MZ5ZHHO6WhIDxLtK4duBF5X5jmcLm1qdo\\\\\\\/Qmm2En6UporPk2RE+urhlGo50XCskZpYWC0R7ZE4rh6KbKnHEe57zKpckmnUulGxDUhFYm0z9q6C0CCKbSPaXpImvLLDjSbloBhU7eNXDYrBmHdmf2MzuHtyFzpnm0uCpC1nrIXEwv+F4jiVFWSqahAEpDiM3nDxvvmZkIYU2zAH3tEktkRXvJqmrRMpSDv8Pdo\\\\\\\/QJfiN8bvwUskXP\\\\\\\/z+kNPh9JcXPJdWCn1zwfJTYY2qVVkiZdsFw7xDI57mrQBzI6Q39HLrR6hQw9x2xL2G6ruXxE+wyMz\\\\\\\/FCTd2RzGCy2S1UrAkMESfTgzkjjkmoL+r402RINtEzaMhcwuNzl+Im50y2OIYo3eFEtlPpKbLI8Oj4P0j3+q9EBpH4\\\\\\\/4wPKlyUulamzaE2wa+tSSrm66MIOPm\\\\\\\/DXKwWh0YgJspkJaOXsujZ45L+l6ZaXYdOLKjTWElBxU1YyHiEZmFvXJGy3j5FMtXjcgq6lxWFAiP9V9sI7qgsr3bIpDIYTddWTYZJYEbR1gZXs1g0Z2VhdwHlmo5z1F8FXMlFTbCMtzC+xZT9dsr05JcOZHrxnqrBFc1JGM7ztV9zFRYqkGH1t700MUH7weXG75ZWMSxA+3xghEWL6xMVC51EMHq5vuiT9rZKIltlMPTlofJluNlbxlRiJSz6NBKNkwRVxBarCMnRXXkjf\\\\\\\/RTYKWrFVHCnl3HJBrhMQs5FtVz94RzGRK+blHrENCo0kurvE\\\\\\\/hIPLHWa0utpscYt8bDfRCv4Qqdcmu0VoOJ\\\\\\\/HtZgxHRHsU5a5JXom2GIB3XUA8g8cByQUgYXnVo\\\u0022,\\\u0022iv\\\u0022:\\\u0022QXYwPphfqR7E4mCYF4udbg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f0d45f1ccbb69274dfae07b9b2b63920712fbf1ec9a0fe2fc9a099b2ee8156de\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022caMt3sxYbzEKNCC5aLiDhpgStjKBF8v6kRKGq8OEngqordqiEAbHkyjoFJGCIRlpDDnFGmcoNdHBz+aRNuLldTY+r7\\\\\\\/63zroud+UwKVAoJ7Q4Gi2h7NAZmPe1YbuE4dga2g8ZWT3kXivChsMRjsinGBe+6VK7yL2kdLQkGTNF0VXQU0VB8pb7Lo9whPHKQynlqj6tGu5bJEOMW8KXl\\\\\\\/VuSftTsXg0D8NWbm0XjJ35Zo2LNUNfIn7Ae7d8HlRE2x6KSusxW+V9qTz7wB19Kebpolv24Ka1XlpCMQV8ZheRvJ4JOz1GGeCFssaGxAabwLZZ1IYfSh2trpua4Dzp65K+q5hlxYFmKGu3nsn9Tcke2E23N1gwMUJO47Y+Vbd+yjYE+dkE64HVyUWRHSzyQ2ayGHtzdoTaafzk8fvfhcvZR5TVnFtO4h8mIN\\\\\\\/DG3fsGzPolhHND\\\\\\\/ZNiV61Hf0vOnyvPHaHWDP6X\\\\\\\/BbPBwNVr9zHPuNlqkjmN9FucG4xPcTj5Y015fcfNL+Yoxg8N7\\\\\\\/VBabaHttNvN2zB2zIigfCAyhjC4IFY1n9U3vhh+CbsNRQSl8MWajbgUNSYeskBJL16GNo6ohYvOuYM5ZRRo9c4zfSCbQqbzHzBrOinCrIZStJplTI\\\\\\\/viFtmcDiHKxxPFFBJoKmB3KIsYGDEjHBEaraNLMo8VR8lTmX\\\\\\\/ZBTjG0MAXBAgQSvqK3a61Vvj8yV8WS9g3\\\\\\\/3bbomcv6l5TY6\\\\\\\/oEKbDmSbFUTaoVqcxvwg8IpxwmW1F76vjuHu+HvoSzFCIYFlg7Kn4HsrYRKbm2tl2m1yIPupEZtTV\\\\\\\/u6atgDG7KLIrG8OXe7qIq\\\\\\\/+9sl8YqCoPaWhT4KerAcX\\\\\\\/Q0zzec7Ob2AbF7t2hZ1HyX1sJ5JaT9p2ETCZ4Z4UVkIfhj\\\\\\\/YXVg4GBTYxgRQJIDbZezd8ZvhZgEPSX0MglOtb+nlWA6kKeiYEmnGne8g5PvjQxVLERLYVE0VnSo0mzTs62fNPy5XFQShUumYhjCfJT8xlttWmsYBZsvstHCFX03ilarU414AAFkURsKCOeY\\\\\\\/Qnvo3vsKTuJDfSFcPOBZEqsBEp0Sf8Qy4sRptiqpQjTanShwh9JSKLw\\\\\\\/jzcoSyzcdqePf2NFcSNzRAggpYuv2MT+5a0KK3Ou3wQ+SYkkuiQieeT4i\\\\\\\/avT1HG15xMVGobun6nTDMrRMMWKbTlTF8chvAOdZaq0RNt2IYhFTxZfcYLYDLk8WT9\\\\\\\/mBCUhtwTqsB35tFQrSLzLO8UJn5DM7rKSTNFbuYdJn85G\\\\\\\/jh10vQkXHBe\\\\\\\/uvlV4lFxu2A1lbmJBbn0CPaFlz7Rl\\\\\\\/TpM8JC4ajLROsS1UXYipVgicTl7KrjrwQBkmpjgbJ3rjD53qCA7xU0l7zNISbmcSd5TQ87XFYXijRKtSWESscnNpVIs+2XCRSuhZCOt4vyXbN1I\\\\\\\/Ul93KXD+ZsnVdrGToBjeTJP8I\\\\\\\/r\\\\\\\/g3dRK5eQhKhhQTG9G5EpYN4SUq3PN0+2vLxgsSv8s6pb7E4jPro2xlmQ9vT2vEVETLQ7w+nrB7wWw6oHE8syJqAuQ3rVISETi9y3q2n94VeXdwqHHu9EgIMCdZ1opOxGF69NdmJwwxHPL7bKBPIrlNc9HlGoDwAbD2qOV\\\\\\\/7Q0Z6AXT+DhNkdxhh9uC7xdqUp1Goax9gWmFcPZJVxwG76u4SKD78wvZvLnobTFTPp0089eEbi1Dhmra05XOtawTKTQp1cftDKhULGbyYWmueN\\\\\\\/hVYtcl15nIqg9RbT\\\\\\\/CO1H6xOBbxAxL7n+SEvKGA1swXOmbMsrYq5rh0t8O+zi0TNYqqyPx8zmpNWQI\\\\\\\/\\\\\\\/X\\\\\\\/vSozoDzdkLaKV3mDrfoH\\\\\\\/R06rRFj1ydAfYoyFdBPoYvDdPfn6SPyXEt0Z1\\\\\\\/HSQU6CaDAdSAuX5loxHJph32ogOJA1pmxxX8XzIaRfpMHZ7CrmCCmQDJfLN31EoDZn3rUQqSArLZ6OBbiBK65\\\\\\\/5kceTwlvCHbSnTm+vyxAjHXY5V4GZDZjEpF382pZwvoVIO4x5E+FcPnYNvcV7gJIvk+TTE6qJBziWneiq6DX1++8PwCTuPu4dQbDh2k6\\\\\\\/I6uAAxKJpQFIxGykt9sqR\\\\\\\/3CWYgalmSbJxEXFQIZrVxGjAZTb75WzzMTNSJEWZVX1sCNZK3qQboJwSKkvBQ+FA45ffv5X7FYiwbPaBLi0Ttt1+FLlV1HtiTqvcoFsid8uej4lhnv6yIA5FAgwG7f9\\\\\\\/58ihWediZSTxdofJXUbPbZveKLaVP3aK1hdK8jmhgIbXj7yVp9r6y83R4q52BQWCqcaJ56ezdq345TzFXT8SUqGzRR+yJZg6wAxeGEfRgxPJkjIENcgce2QAN5UQP07KKRAR2rMg1NKQAIPYD7RnojLDob5WXE8YFCOeS8cCk\\\\\\\/jEDgw1u1fjKwNPT95G8t7xbF4re6TW7kbo1JAXF1SNJyDV\\\\\\\/uMhL6kVIywUMcILWZy9OLlIQNfuuKUksaxeL9p1IoclaquNF41qzRG4Rt89II8KwKLUxQWusWMaJfq9NkwBRwGVwxP8nO2624E9g9Ov2MckTf1WARNfpOqohS9aLuX+GgpiBXdZWQ7pky3BagordXU\\\\\\\/sU1hIdEs73FjNhFbtEVv\\\\\\\/S\\\\\\\/lvYwsiZcaoacW0QhIBrkKviuvgj+FOQkSXH2sqxciLJeWuS70hlNCpYYY+hu\\\\\\\/IGKlN2lDsGcpfpyDNiGJZQcm0enBiaadO4mZpMp+OemIer4G6vc73+fOCDK4hzsCVYPSimXkYw9ivrnGjZ1eiiRK8SGaHmR+3ykx+NfdbLJQq8UR0InZaAHpUFzdQ9MLTHSCiOZBZNx8oWa4Igljb6mwpcSMuexKKXcrfHRZpvXf2MHrzwQ08wSIR3fzJ7C+Serh2IxrG6m1sf+JdmMuPxi0ZiaeyNutCL2g0OPCVX0cOdF2f2\\\\\\\/fnPtpvx0vyNNsk4zTtHhIkiS2+hDViU7LBu6zDQdWXIXq5ertZ44d55LhZOy2QbCezgvRSeoBSNoklQ\\\\\\\/6r58FQsSy5JTbVR1ki3IXjeUgg7YvdfTCVf7cZs4BlKdQpvN5ZKzegaOgcIOlIdf0LbV0\\\\\\\/qstVwXgbY10hUjuxOXpXErjjOitXbm4XliaJ37bR5ij2tSVOp6KIJY1LFPmfXRX2KiILWe+vp83VB8MdGvrh+UV8q9YwvPcCQRjgirwrm0siygIer\\\\\\\/EV6i8BRhnHq2Da+H7LnC0fOmJkD7p6vGU17okaxw43PRQTS+Q\\\\\\\/oWwlzo49vYtVdeY0O2SEi+5qjQMtQFNL\\\\\\\/KG5\\\\\\\/\\\\\\\/Jr7gwWLZBF69RxjEoueer7dWtu3SDdtSjK7N5R5aHPMgbc7LIcnKD2vSuzI3bk3zKnse0hy1\\\\\\\/yHQKhuYAhzFawPNz64limFY+DeRS3QoSC2Moetpqf\\\\\\\/kHygwnpez2uHGFq0HX1L80gvhpaU7w7sQxOx4kRmAA19BWOqiRwG9+wCw7nI+iTUH6InWLGa\\\\\\\/R4zV6IjMLYSiGo3XqDS+7\\\\\\\/5zw8m6vL9L+YneBF0fMwqjxLEJApPH4JjWVAK+yew\\\\\\\/csHxJ2lkFnX5H3R7i2zX+c3BtqRQ4G2MmrYp8Iaut4cv3BApO8ch5xTJWt8MFLSumr6KW0Ggy6ht0X23kgcN\\\\\\\/2vgKR6s0lz692AR+V\\\\\\\/LV2+fz6ijdyHfc\\\\\\\/ZX9MqhTc4zGtIgstZP27k\\\\\\\/3IwnXUd77Mjc1qJYIC9i2ffOMv9Ns7mfX+tWj9Dt\\\\\\\/e15jOJkI1IZNzGvMcQXCCb8QMbAxWrMePoFWuZSDFQdSWq5azRhPo8N6OiVVaeaaZFJnQxBbvEca3PhLfSBuoY1qzIh4hXGIdTz53i2d+1hBoy7iKP+lAmgbbnaVTASgUgZkZAtHIj5Xd8WpEMd3qk7gBRnJDppooRGcRpAMCnygZ2bWe+sK4czaC4rFCFA16YVDOZBgM5IvWnUKG81yf+i9mk3DHXKZeqF0qVPF6Ckm2UcPGsgBUk4R\\\\\\\/JpEoN5q8ow9dQDAhI3prluDMCXCdBico\\\\\\\/TnPv4aTjz4rhgArDzguix52eRTo46tunDlB3IfhL52zX8ox63DFuEDgowsWRQXNiKwG0itgheeCufuPOsraLWoCNt6Z7\\\\\\\/3GsINhP8ZG9+irTbmx5bcKP05yRPbBGoBPPvX04YaONgy+AHBcq4CUMRuRHQj4D3O+D5jllTgQ47uCHnYvgWPOVlybe6H7rku8riYmH\\\\\\\/pnNsYWr5bn7\\\\\\\/uf8y+wYJkoD\\\\\\\/peBTNHR+BCR7ga0WXJEYcDwx2B84LfDWPtK1p81c9qJuWEZXGMNASkvMkTMnkzWLJ4aeOQOYCCeJnDo899lKhQflj32DqyhxhoB6cPwCbi7mVqfmtAIJsSeSPXzE1izRvS3aL74xXD2SGhTNsbY9+jQQK4IOtZ\\\\\\\/8mCRCAZZMOw2kyRfynQ0XDqtw24cRHWb7H31lYXa9z2XQBeNjRVck3bcYXLAds5B9i1Lci6U42D0DZj6kgh7C3f8US3VHz88eLuJzJTaoja4iEEjlXzIzBx355bzBfO2UdGpe36f199YVt0WGFC4v5Ke5\\\\\\\/izVgrpQLMhZYWVysbOtRAMkHw6MYE1uIWlk6dQ5lhMSvz4CojXFrTxyTQpYHzFNw0avTtJKTg8V4at6I0fRQLBSaL0Szg89aXxwePu4LcIaFoK3bBElKiiJAZbuzuUPmLgDhbG94V082KLhNHwOdY956VmeOOxyCJBEF+FKeDXVta4ESsH\\\\\\\/0ULDlX29BTd\\\\\\\/IaQ80KmmCvFZzMV1MiiOn541fCSAcRgytxl5+7VDby7BeD9E147M3Yy4UwyVmItsdgs6gKjdEo66Bh5J68hWorZ+OI8qmCnyn3pxq86GaatCQcR7BgcBZI8LS\\\\\\\/Ha672455VWssI4el0CUR5QulvWK8MXxX4Qxb6MlUq6dGsxYGDuUXQxXXci8va55AOwHSmRpyxXq4RTtmf96K6ZsdA\\\\\\\/ZLZ+ljwQBhZvnfEghnYy2fwD+ptGgpXaYBDahO7snXXG9myQ5qa1SIbkzmZiKL9d+oaW7NHwo\\\\\\\/K7hX3cwb2DsXN\\\\\\\/izXng8pQjkpbyv85istaeDk257Zvh196jHb5azvV8qJepuHAxMpmK25q9TJvQxcMFthBt6FAYkMYtrh8uNi8pUUWGgQA+o31kv6tZPgBuZkkoUF+zxuvoc\\\\\\\/9QVpt\\\\\\\/yGhxV2nCX0vJoTuSfPv8ugHuqQlUApfb6IqO7t3wH5Ls+n442v1cpyX7853uEP9ybhajYrIJxNR4ZgAsG4lkJxphTILuMCiWOuWSQAFmsJu2ABNhxTUYkBiYuL+3EpH+0+OEy+NEYBYKMWmSk0sWU4tXXJI0E8irVmfcteaQzmxSWOXtQ1d1Fch3V82o3LOL3wTiHIczgCySXWqiKNrQOrON2Tu\\\\\\\/FEha4YhLXx8fpnuojXWS+YDBUA6Lx5dPYYbDwnFjWK8Cp759B3RFCvx7T3VxWEXraxWwm9385gOMYPWyxe58qbGmjTh\\\\\\\/hJvNIZN0lDIo8QXUNuFdkEEH3sJ+fx5DzdiaDfUXTsRU++dD7UsjLiSWKxA6OrAHyz13l4GTgfL\\\\\\\/+SZ81bfVoAeo8kyJKe7dTXAosSTsrB977RoJ1O6Opazx0HbWeytBUMYN\\\\\\\/CSlNrYRiWiCKoaeCqO+r15CKlMXx\\\\\\\/wXgJeal7UfG\\\\\\\/hRrsX1oFPMoLiAztbQUylA1d+oC3qOZMDfBGbMzMIesMP6n809abxpJOBR4KY1H3fr08dYTGGZAO5cdYIBbInyBggwaX0AY1p2Ghr8Kyw2DRvqLvHc7o7FDk9MN7ksNydAHtnTPCjyzEsVxkAeksRfG+8nsR3xravM2qzLvUuNalRjKgHl8XkIAKTa8tNFdm46kMWmfMC3TQKjiztumgCiZfLVw1gtBXcTXEuWY7XNcEgW6WMwuGLEi4tZaYBSXXWCTdxjTMa6GlgLB1XfDLUiGbvTEv4izsgAjoPSGQRf+7cqvkK0zsTFu16o4h2HKKpN1YmQMViqbkCjH7+knley7g8cJTFhvZqt2t10A3Q5MUuC8LTJvHofIn9PolamG0REbhncxXL8DMplxXlkIvxdZX6mciRZgNY+vmgiO5gkuTtkEfQC\\\\\\\/zlvaEsVogzasVf\\\\\\\/0YVkkN5J\\\\\\\/DssCOvPjCBDyU7veEpeGB52f7E7DNHSDezoMd3B4ClLCbQDzcsR57awCjsUcyZ3S01EivoxVY5ubChmW5OdRmmzVEprzkNZetbBLibJ6Fb0cKmtXcu\\\\\\\/oQ1l5y2Z6UxnppViWSkwxtE7i4WBhXGqeu6eqRyONlPTgGwrIOMLQ3R+4TbWKcORguH8Kdb7GSEKMBAekSEJ5rPzC9jG3XBESxziws4wmrbDLTkdcD4X+UW2P6YXN0ImZrEbmZAVeRFz6mFFWF8kBRCmIwtSkml6pSZQcNe4WyQxi1ApzgvOF8iXB2r9v+vuP1Fl5RVNvCcF6JSyCMaZTUx2JN0PqFU6\\\\\\\/z1QB7xBOMeGpBhOFkNx8GPUdXP2UkYFJp12y+L5ba\\\\\\\/ZCD0ptZ2PXEyQA\\\\\\\/3t9vWoTy9lO1E8uLcRrXDRJhg1DAuFkrBceV8B8cgTc\\\\\\\/9dfF10BvyCWHaY\\\\\\\/yubFeYJbE4haxDQGQumfFPUvG2sdbd\\\\\\\/2+NI0Io8rS7S6t8GValWftmTK2HWaH3mTy5E+AcMQI9SGX5ClFIYhUQXxzoXdTsyEAGcpvecbGOHVjXr5gUjqhv+09YoQjiXQCZoowZYBycGPqDHG2KiOGrGbTR\\\\\\\/Xg74Ig+9fpb76+QKA\\\\\\\/zGjMXD4Lw3S0WgtDa9zuRkb3n0wKJfuaNPMSK3xfTCdxveDqI9s\\\\\\\/cYpo0I2gLF+P2uBX1hZOCow7695rqRG5Qr3iMU4+672Axu54HoytUtRTJkwaMSDHwXH1jmuKCF+UjSWv+BmRPczOCaRX4pTdQj1Jctpqc9mbUoQPWszgfZt9Ury4592df1hiOcBWS+k2jGO2UlOwQvukPB+Qpe9ywndnY06cvE5da6ZtTVpx4P3fN\\\\\\\/OwVtfOMtjtcLcjSUzHH2RBFVr6zMTGLQbcxL2o7aVmU6jtJcwwYLKGwsH8FeiRT9dQln89SfRoqIdK4xyrIhe9D832W97NTbbdTPF9RhmtGJ91gSqsYJ9uYkyi6dYxqSPeQyoQcGbOln8hEQui5nLbCDIdJ4s9XTQRvjajAbylMNLxH199otb2ooJJPBvxDnNPIBLEi3rc5jtCD7Kv5QDMSMK4PuoV7IaeHPeuP1vt7vwfg1OEF2b0uhq1umn2CPwd7z\\\\\\\/8xkfAuGAdKYYr\\\\\\\/zLxZxMKOKduSk5AyrTq5KY9o+LNxvJqIAmBRU7a+rrOPVcK95tfc58tXcSh+fHd5BUfzoxrYdOXu\\\\\\\/QRHOe4UXJOy\\\\\\\/wZNFJTgUAh1PWshT0Byu42fjFSItjYLQZhoOTQmZEJkX7jSExEj5PCFBKt7dRlZlMtuRxu4\\\\\\\/iitLuUWTU5G5X2I1CBo3btFYvCswBwhKWY5QIvGNSSqj3lNcpUKVdp31euKQczQwxjANEejiyWaE8YxlwxHtUFYjJapWz9aSPVzFaKddo7nZZcC8PtwbvYcU\\\\\\\/2tmCYVQGO9YNl\\\\\\\/MC1LBAbdS37ZQsCTFLSeNbkEe1Ecv7\\\\\\\/\\\\\\\/CCh+JJszp68AaScr\\\\\\\/yR5Kw0VnEyfy7vOuQffPwFCE8w\\\\\\\/JWMh+7ll21dVM4uUkOJni6BUg7x3N24FALwaWPZZyh2jsoEIPuKrZQfh3b301DcJUYJOoYcZIlfOO05toFBam4MTzzgJ834ki12aCE1ewqirhiZj07L4Fqca5IjHaEDWd2sH53ba7e04BLA1qBAT5TCBjworoQdqYtd48xZ8d999\\\\\\\/KA\\\\\\\/COUYOHk\\\\\\\/ImnxXzQrfwRbOP5ntDtVvvZnuTeHn40r7MAG23OzpWuZNj8d7kqhBEAjTzVsoZHa7q9cGgYDAziuEU0e2C4Awofifjz3qAJogYEurvwjjoJS4waiPFRpxwuNVyJ6vO4vdRzrgaS8B+lJs2CPmCbFrqj2ZSF+Oz56RcG2KhOk0HLlqehG7A+Sq8aMhBBZcgMamUbuvL+BvDPJw36jO1sNAAf13qnkrX78JfTJzUQzjSxIGsCTUxqnPggA1Fp\\\\\\\/TezuB9KGyVu0Ay1yXWPdnvNXeY+jR9Pi8lTxl2ZN6KTt\\\\\\\/WN1ST\\\\\\\/amCc6nonNWaEHuMFHFOZSOvVS5UQfb4F4fKz5KGRGcdw42BG4rnuGP8ugTJelN1VtZON\\\\\\\/Y6Xqpy8IlUxCHtj89R689Zf25JZBJ8xYnnzpxsjgfjqp4A3NmD6cfmUdu\\\\\\\/nkJ4SMJA1oMMNkliKevs23BvZj357oaJo3ZGf6hnq6GdALqPRB35IrSx8FX0I7h3\\\\\\\/yJBAFzzMkQrRZT2frWP77jU9bk\\\\\\\/ZD+vyws1Ur3c4sxFAp4YaUQPAo9AgU5DgodPwCZm78ghTDAgxBS+RTPC58FYkXx2AdckauD4Mmtwpfrc+rsliSarSIVx590xc+WU4MkaBMT0U1+EJ7J8aKjpdm0YPK9MMapZ6QfR6KpTJ6jdTHcx8EusEguJTdRn46RIyXIQaIGVAhkq1NuWCvwwIxzI+pTw8jxJuaDy8g6XJz6\\\\\\\/WUgrpHCxH1pkOtnexcJi4K+qaX\\\\\\\/Z\\\\\\\/A\\\\\\\/zfKIgC5LLU7uNfGKbPop6eoh0az3h81jDI\\\\\\\/1RUO48KRLC2lTU0Ur9\\\\\\\/DAsmjqR76htBsTKU7KwS7+7aKjxu1doexGJSsaeFPC9YGJx1HanR\\\\\\\/j+OryXq\\\\\\\/J1a77HAp9hNluSAAIH+TGfRppQxvBU0O\\\\\\\/1fx06Ats5MgDl6IpJWnuT8CXMvopYvtMESTzSPvX+uWfwEUq+MGp5qUg7jO2DeV2AvJINukHVm72O2\\\\\\\/7dktRgUM9AlH8IpNwo0YJsPL2xfgnCbLltC7aJIb8Z5gAKqPK\\\\\\\/wK3zK3B3SSx9Ro8g75Tq2oIt7WkUozqc5PVTVRQB3IA8njirj5aUbZVNefkc5UG9o8hfrRQ0PPcMkvAT\\\\\\\/PSCalTkyhEXtW3tCkcPRJ1wYBTYz1DS7Agt17PKUHKZ9I77US\\\\\\\/yZPnwVUT7WGbCxApwHV90R8pueOaoe+qsepVYCSsZeSGbxLCO9AjFQpaFgp15HwGM\\\\\\\/ZU9GmieNI+mmHY0ID3j1V1CTIwDeOpwTqs9CX5bSXW26aajpjJTautKGuExsFFxUM9CrH5FcPTIjBhzIeNBU4iVFMi5Mgog6jyQK5C70Xg+LvVIJZ8nLmIy9q2C3+WUsePXug7LI3ix9raVRg53QVjMJqxPAXRLtTn+JB9yLQ12AP+CDswRYJoegeOmiKO2PH7a3XQY7RPQ6M5PfKEqUGdTrZNIoBtC32X+UgES2k2e+\\\\\\\/b1GxFFYyaJ01AsCoJJETacABb41YoDGy6aW5mG8OIIX9w9b1L+k7wJyJsmmbLfYhmi5WCyfmFqHoVqZv\\\\\\\/sUrBTmrdjqdH2K+rkCjdZ1J65qlhxGMv7u2T6uY2ZOFJ6s0cjPBMz3LMbsvc\\\\\\\/CSj4fFWluQaecadyV4ZYI8az9gkVLyoCYW9hQ+MBgNbdrJoZNgx0nkqA9e3m7EUlJc1mKQun5YEIIXC63ZbgytjocoqnG8qYnca3ej+uRsZYqNtKgXx5L0qbamwXwLoN2UimDivAgRSZiDjerLGlIGLFcnFwr\\\\\\\/QqSQVRwz1906sHD2MiVxCyOgKIBuTy4x9eJuyjB6xAZgBvuvpauNrWc9GT7BmEX3oWgAzsYCzOALuKJK19DkT05CkZh\\\\\\\/oQDyO3oBIfNAMZShKkGb751VGjGYpDJBrcJvE5XFWuSr\\\\\\\/frpzZUbVLyYoVMdMZqUxsHppT0wlblVRZSjK5\\\\\\\/5uyncPk8FfFeZCkz8NqQRXPlqqXZ+Uk8gbC+tKGZzuw6NyraMN2Mp3r3AA2vhx6Uvil1m2prfCC1G7o53P00BoVbJ+L7nLQND6H41\\\\\\\/EoEYCR\\\\\\\/PlzuVkQIkjyzUELTZKN0UvAOfhIw+CdoRhKPxm1fkaWLNomil0pNeZJWIVmHERiyphLV\\\\\\\/ap89OvDMRRpPCebVujZuGegmBF6QDr0WHuUvQh8iD5eG1Pg9CEO462kShIl2lS+FuVcCriYN8ai7fp2jBZpQjCLmk7Ze+mQXeSkmCChd0rRaIpsoYpYUezI+KLiB30rvq3t3LuRaF0WKo6j84b79ZAdcrDv7xBAgcQ2i1J+0Rd84tRE6fh7b+aKe6Ho+xuYqIjySkn0NGWjTcswwo3dw0G57PQElgiqlwb3PxWgwJ\\\\\\\/hMM0s3t1lyxtebjUzTTeNLF5xdzTThWu+JIPikRbIdtvlK+0X391MKuXmT1BexpaZ0Tuutgtt3SdZ\\\\\\\/EVx5Dq\\\\\\\/vaSi5M1sH8sOGGhDB654oHdVZk\\\\\\\/Cm4DTw2EGK0NUn1O+NVcfkjlhclZPfo\\\\\\\/jWjlxotv1PxkMQBRt2uadrlT1l0g4L\\\\\\\/rG9UyzLMCsB5yqiEE72tgZW9aV49\\\\\\\/QmKgCyNHOrLVtP8diZl1oLwmZX7LpxzgzL6YppXZfmEL95qR64CibKzIwD78QxcBA\\\\\\\/KXb+e+KGPL8ka\\\\\\\/kebw1fBZWBZQ3iEXlJXYEPnKj\\\\\\\/7myQQ9+r1rBmnUCyNTRQepOtcAkOKGGNwNVtE4\\\\\\\/GCP7cCQFea4A2rVfdjIfHplmiGOonKJuc\\\\\\\/rsf2D47VfNyZPzl+8YOzJ6kAOFld9ss9L8v0AGvGINWt15ljQQRDSWej9RcAaCkHlRgn0wcAfLwjLPmhJrD2e8RNzcRwl5hog\\\\\\\/l038LvYpF2Y\\\\\\\/cXUs9zxT2OkMXwknlAF34J1NgGXkxybGMv4bKO743dKLMCMJYJTTXIQLmngJxueHbHHcPszmLoWBKcbiwFK+K01wvamB4Tk5UFZnsJzYny+vsUG1hYgkldFaBn1b5s5JM3ZX2zxdbVNdcOUGjXEPYOOQidxIXzTt\\\\\\\/oFg6eVfMhOzeEUGWR9iBoGr6xnkIeuQ+n3ZlapM2loxmKZ\\\\\\\/mbGtHxEw480JOinq9hc\\\\\\\/7H73pqycUGQjYCEXKB+xor8zS8Pmi6JXRYot3l7LKfBeaEErLetWRwHKxxBEW5rEIgK67q4PY3AT\\\\\\\/k9Wg2TH16CgEr\\\\\\\/bqfIYqDM20vvZ7UWHd8q5JyDYyQUiZCQy9hhtw0Syt6to4YWmpiMSMWgnIDj\\\\\\\/zDgcLX6WS8MmmGyiMjZgIJUtukTYC65+yR2629MVF82uikWo40WzR3ZxPLlTAFU12uAkvz8NzrGrYhKLAuK76G9OXVPpkCU2ydGgOj83yKZrGvSykzcwV1gKquxOhxIMR+u18lzoEjJcdvLhO8KWdUKnqOxfc52qGt65izGoCfbGhpNXf2T432a7UvoKtE79TJJP1tPP7937w5vnrbXczd1A\\\\\\\/Qukj0eKxSomodzA5wOM5lKfmsc8DY6kZinQVyWmeyvyEgFVZen8MTIbVmfqD0feqLe45CyUvi2ogHcDP26j+oerDgP+Bp\\\\\\\/ALaNmHBKFUvyV\\\\\\\/30tGpfjkecE5qf6QIeV0begFeXNpDVTVceMavLWcE1MJrw8cV\\\\\\\/9JgEc3BFi1O+t7ZCqqpfxo0zGWkR4gEuSI5ztVPEcnQjD2eRfLctQPmdGNIiyPTTbRnag52DW5S4kC\\\\\\\/\\\\\\\/MVhVTYQQ98z7KnJdcOI0v+5obQ9pDffnMcWmjwXQucpxEG6X3IGG6nTyA\\\\\\\/vUDC0yRHtm44vuCGU\\\\\\\/b\\\\\\\/cYc5EC2YONelTaD7xjc0kxNocL9UO1XKBDtiRGP6sM9ezxRFQCX\\\\\\\/5xUbVmRvsRAM5V2FbmmZ7mwZwLdHxxcRey6U+AptvGlMPm\\\\\\\/PaSfzwfVCDBkoYbIQciul7V0IT9DRhayIQUZs3iS8UZF\\\\\\\/MqVyJMS5knu7Ql0z8a0VL\\\\\\\/kpgTbRZXjGHSFf\\\\\\\/zLFlS30u3xqOcncz3MWq27wRiqSLNB0EkF9c3AN5yoy508DvPtPcf1xe8Oes\\\\\\\/NQI9p6hWv9n0rLH\\\\\\\/NycNmp50cjkG9b5CDStYcBY3WA9Izd3fClvyr2nOyP4qny4rAnZToZlOqv3CGDRHs\\\\\\\/a49mHXIagvoyOpJ3xknjFWO30aTy4OvJVw3SPKEjvsKkFpX9t19Zf0KJakU66egvjX\\\\\\\/77tvFbbO4sBJePM416ssU0BdKPpJ+tfDFgNS1YKU2at5BU+rjCI843uvJKpT0IU\\\\\\\/hGHB+YJ7DiAkTQk5mzI4TYjWDl9ZxxOEw25EjC3P1gu\\\\\\\/RveFhOkQFlas\\\\\\\/d9y9yxZyZG058aNwgKp4PCVsnJOsVgSHfUW5m5NdnDn9KTCQZ2HtVxAy010hFHx0mgVzWFO3WsGM+Xv71WgD7momFQ\\\\\\\/pKg5K7u1GjQImOIAFHjUQsgNsc8e3U6muibZpHdkDM\\\\\\\/C2JWADa91+gNsFnfE1e5JOZPY38nJlcLfaVTmPWg9n7+NGaaGlWgdZHFhxRzQndLTIDdo9oSZvwKa+el6DlFCZdWYbgl0c4URyRD6Bga2w5FqsDfPJKMJcNo+yYQ6NRYOaYzi+mAMF1mwDZk+mN0oT9Fb8bdtP1VFwEwdbEi92Pmp7EOrSwR+F8rYBg9QRsU9NDv0GF+SKVCQkB24STkU2mTf6BCRqxmStUVoNtLIJ2BD9bgrP1HgTDyP0GaQnwXlWq902UNGythuTxRLvVUfv1zOE1l+caYCZ5EXhytMtUphMsZMBOulupuENs7mSQ8fmeToQlpZUf81o8EIRtgrM0dTM7W6KElEIhPLoPaJGx\\\\\\\/T2CTsb6IDkRWwFdWetlSDuZ6OVshJreSghE+eWTw1Xvd3Ksu1DmGEl6DQHPqZ2PYScNXHYNZKPn9\\\\\\\/Zs0jiOwg1KkR5qokvh6+FKOK6GcQRj3QeycbRAP3sACHsZW+snoFvahPxMsH2nfD981gzxBqLbADsATH4L+PKO4pc0tJkOxLHZBNnTwPE17stad0OQRo7HxhlHQePCuK5w2v\\\\\\\/fP1IQGKUJ3v1KBOi+UzprKyYHExFWtsTuAwmf1izCpGaf+lzvWULKQDfWNVsOr5skg4jO6sLnzzs3JU2B5Pgtz5YVwlhaDlWd\\\\\\\/rg1AzzlQx9tdQqTAWC3nQiZGgY0ofAUZ9dpqsk2ZDEsUlex7xW22HWACONqWlsN22dpXl\\\\\\\/23CP03glALmh9iI15G5yMIPjMPoIjPaL1AIug7TNOlFvZvsWyS+M+SRmkdpWxnDIrLVIJdTN2vao+KBA1A1LzEkzTP9hEaY4x7qYoxKo\\\\\\\/ieV2K\\\\\\\/AZwsuMWwps7W\\\\\\\/anlnHW5Rtz4\\\\\\\/gCemYbAaSLsR8C0vZqf58B01GS17CS2WPsm83aLw+yAG3dKIpVCKHbLnP+x6aD0xzKhtWzjVEXNUd5c4SYVFFSHeiCpiFhuIGKcJOprtTRKXXYVCtfadZa9zBOQn9Z9DIDQm\\\\\\\/bvSN+MeARIwpCvcWxgJLbsY8g5SP7L7UGYETYVxbBxrjybvppdCeFjNVdafn1mkWt+6QJp4SUy29fOCjGgys+gmTzZ9TgSVMr7pC3kvE8O4126tizfbAgdnpkw7lxyE2Tx+ncBpvp+NfdZnquoBa9x4yFhbIxxd+QavHQqYsdgmYnRFUWAM3Gz7XYIY42KtzptkRsJnv26Rh3lIs1jc6boyrFNbnpXbuZ2O2xZcPDu2atQbexnKGCJhJFOrHrmFfDWxgLszOpPbmiHG5DbfTX2iFTOfaTiQTQaxuEZTGnXtTbj8JrkoIGuOWwcBIQHxfkoNBrbnJZq8Ub8FpoR45CwYxjPlPJxLKiKwiVQZx7D4teGoSoxwaxnHpxWvIGimgxA4CKzKFCmolyvk3xGwzDyyfqrKSLePZ9Ze0RS0LWoo+YQ8tp87+5Ds8dTEMPJC2VRP+VIVtotmnmYFejln\\\\\\\/IMWDz33ph\\\\\\\/z436LnBoBzxVaY3jtvl0JMFEwLKa2Sc3xoLtSXtbSmAZ0g5AQULEQPrOyqGNDn1rfMpKOGht\\\\\\\/5f668WuiCobluyFX9DI+U3XzLuHxQ5cSBWXkXU8l7CMmkCWk2hIEBpuY3APpD\\\\\\\/QsnO4kPQp28trCv6kHfst4lZr\\\\\\\/Y+SDGgbZPVlRGcVk8JAx5C\\\\\\\/Qozz\\\\\\\/juQCkVu3z0iZDLf7cadHevD3PT91Bpe6EBNq\\\\\\\/T2IqqHpkX9enqNtxOcheXpAvsecCLjCc0+BJbLhQt+svBKZxdFin2ai7HDsrsee36k39TTWXR5gBZBgsM62h3x41CfoK7Xq6UG3CUBBVlh3pFxwEsuwGzyQMfQefpIRjC9eLHSYQOU0YTA4gDwJW\\\\\\\/7oPuWur1efOGRTg8YaZpP8MkBg6x915fFXZ5lrrpEZKG347rqOhm9sgQcBj\\\\\\\/ByBzQdhqL0OUg1H62sb9N\\\\\\\/H\\\\\\\/BVUK\\\\\\\/0J9aa7yeLcoi8VJ1NW+k5MC9qQiqJbWVBSqZeJVoYSyDSMwEbquuMYezLFBrX0ylBtNJaxsOmB+i4DzIXeQvwcke5mWwcETBevgnf\\\\\\\/LCVkayn6fAfj1ZplidecuxTL9hkCWlRhClQry56fS9lFNSvDlSpLslrSpVsV4YgBgJqiay0taXbNZCdJBFGH6bhQSkDfpWh9yDmZn880q+nF9q1Ue7ulRR5l7pfaY298Ye0SvwSdf7CXBLkTGDkkFWEKNEuhmOZwRHLZ5xV\\\\\\\/sMnsRbB7ZWkuvNFgR8IoUDCHqOlI9gWqXHZNoiSeH3gaU8+JIa6gEDpcx\\\\\\\/9UXbd+Bz03GVtUQToY1utIqJ9bAHqKcRwszd9aSFaZ75Rl+Cer7QwD9FkQjfbHOYvAfyXWPjf\\\\\\\/x4URAvSkY1h+EW+Yi6IeTx0QrtINUmDEoxiRqgQH1tUCkFTunw1stGIQd3AqgETrnJtNL97g3RvvKscA03Ck72VDqVfrDwjFeV\\\\\\\/UXD7FFbi3oowhy6mNB41OHISjr6eeGkdIaNADqZeVvdyiuQUQnO96Ph0SQLMGj4HjZ93vux16NHIMC3\\\\\\\/c5pNyJq8nHK3kpiilBffk5L41X8JKuF6D5jtiBirh5yOb2YVafdCfPJ56FTmyxm1xckZNUfV1WOAnm4TlIBPZVvWsNQIHiHqiXm4ZZmnnpr4arpYeqg4CdxtD9hoaD\\\\\\\/QbiOJXVTXXF+b3oT20J+By9VJ7Dah19FJ35ELle\\\\\\\/BXGNrJoEmp96+kDjvvXBDP7RX27Ku5yb5GJMJA\\\\\\\/AXXjAO8UG1S7cko\\\\\\\/Ivgu2+Hcj4AX1DWU146l\\\\\\\/\\\\\\\/6t+dLE5tBJHHydTCLBGCSRo0z+BnvcyCKv1YitHy4sclt4AdRdai8RVoGQwNPelUTYxKd2mEBWpR1ZLIgRVp30aWRbRskwTwxZXj\\\\\\\/SeCRn8h483bTP5+JVvXQq++UwgGyF4vPnJwCCXx1MtvmZtY67HNepJEC8S6YEm2izZHnapmfwyPLEJyeK\\\\\\\/M9qfVLSiUHcK0pf1U60nHWgBE2SyOawZ5iO0i0NZxaBufOqW3OQh6qlv20QLq3ilcNSsGIQ\\\\\\\/JZpydYb9a\\\\\\\/c74iwKOzcfzbZH5goxFhE95+HmgFQQM72k6X7Z6FD5K8jfCuEWqUUO+nCnoEOYxIWOxBitq5lRz9PO\\\\\\\/zfinkyn+bwT+2IQ1vlLR4TlIsFthCNDMg5gSvJh7YDirKUXH6TfOolWq5nRvftMQWib0XSRDLQOC1B3CPxex1nkeCE22Xebxm39EDcBEKFlLYFG+uusm2A5nFYgqNvwxFMXsZqy3PJyWQN8SeDJYW3+h1sXKGokO\\\\\\\/uAEdwB3TPasFj6CJpB1nRn1NDh6FkD6wAM7l46mgjgEMaqyCP+rZW3IFpTSuFDaXTCSRjPSjajTBswizx5gCJKaRmTF7HlH8KggmyUmU4V8tqh\\\\\\\/tKZY2cwgTywbKR3UMwLV1CbsOg46nkyrg+fH3aj3m7tH2yMTwOhqiNaVKz3SAwJcQrm+9NfgwJCy0kQvWWTLdw1wtqhnsx6Hu8EzYej9TppGuyr1NYxuGJMZ8BGQBfVeKulV\\\\\\\/nuWZlIgxSeUvqX74N7odovy9d1xDzHF57XhKWSKm03CDIxzCjD3f3WZJYLQCInlNfDR9i5U5TfB6gonif4\\\\\\\/lpGr+9x8G8KoMZPW\\\\\\\/vlC9rLhN9cq5uD5GUG2sLYg\\\\\\\/Y4bDNL8FnvfWTJEdFOoYHtpHsXBtJERxpfn7B1JHBWUKzE1FYtS0tcuaFI67GBcrQUTmVDQhrhvsFbdvfhLDX\\\\\\\/PAgufUZ2Obvq7UWkJAzZZDK5lIeahP51dd434FUEhDTMTBHhKE3CfWQ+3W7pmhVUqleNawVPGR0bPbT+LC\\\\\\\/sPbwBmoRba66OG7Xyf4\\\\\\\/U6HgW+XOVWN90xwj0ZUfWAbkpZcjfREIzCBLa9hagFTbyPnSGeRMw+paGDVTyi8Ea92MoVOEVsR7U1KIAkFE264Js0JEblhYzvBdQ69FJ293UittmK9MThH046GFbK+pXGFbJdc8ipURPNdyBLFwmWHclQLOwvyvUgoDaPTn1rcOS809BMRaGlRYwqaPnDwJ0E3A8hV+yRsiH8AHfIHD7OcPZCB0Od\\\\\\\/ejK+L6DS0QvZbMSYTDv\\\\\\\/1DdL6f+tixmwmxd9g\\\\\\\/s7W56JRBr0guITZXWgld\\\\\\\/jL2xvRzXLfQH+1F77jF9qjVTM5eBNnWmkGG7wQEs58R8mfGf94nOVS3sNgRrIZujy186f\\\\\\\/51wEximd8Faig975bQp4ahVLVn96PyPlbS9bwGniddqRcV+FrX87RpQxwhUzRbtaGWxQlqF5thU+2KC0rh5xA6Wr5zieDWXl7LpaiCuuJNQuuskWIe8T78MULLliLHNy0doBNJhW32MyohwKW58QCiprPCbUCeSDNnPrKZR+eV4L6BXm5I3PMgsrZz6Hbm9TBe60bhbwimYsKQxnxwDiVn73rLrLYRxGu5pQOUiF82pWwQwfTXdUm4HJgi1yqhh1sa6A9gJRNnMcJEYU8+xfrnt0D1MJoP6VYMma8APxQX3iQvtznUDuib1O6g3FP0co12nJvgw71kYV3M\\\\\\\/n7KVRXnvzVDY4Jufneyyk11YDYMAp1cNbjTM35JZteAbILcTGcDKxeeYfleWK2S\\\\\\\/N2qhQ5SF2ZoPabOcUoWjNmRMVnAA\\\\\\\/ztK9pzEQNWCMLCaxLRlUZob9SGEAFxyPS9NOuL7UmWBPp359hSvR5eC5EKb714hjwWcU106JZVFo64HYPOw2sy5ediwRPCFGZhdqHbbt06qedM57vFFOrON3Wgs2bBz7\\\\\\\/j2wG\\\\\\\/LU689TsDn+Q3yd5RUe4YloCturT9OQChWSGTb+IgQMGIW6KS24vbmuaGIXa5L4W6mDlc9+lPMoJqdgUJ0Ib59k6g66g49UJnhSAD1YnLacSoEXAJDPhztwVCVf8D8u2ad49dVYe+ctVemxh7UCZKRbd1QLRNjnPc12Pbv+8RgdH39r5YJJxl1dw2n5WTT3HSvVb4ZQCZDpqJfoF58KQiFqA6PKXuJdihLRQtaCCasJXVRupxoQ2CfrhjcHMwhrnliLMol0UTJgvxkRg5iGzOfoA4P3RENCjqJju314cFHvu9kHwX9D0t5pnqHo+ZxI39bL24dIISn67vnKWUJUSaGyQXy4ZdNvdRvjyeiimYhe0DZGwT+9q0p2yrcdjP\\\\\\\/Tjbj0EYn9iTo1V\\\\\\\/H1s2Pxbva8R28i27Rzc6qSf2mcmXGuhZvX1HxvICq1JrB4p2yP4rUJMkmdznYGVjWuOebS2zg3DQNxG8TV4BnCQV5s6ruhviUtC7x2UMYRwGFLNT2IjPeN5nv2swiKc0Dk6i8iw4P\\\\\\\/ODsZ4i7HPEDr8apLVHMNRjpClajaFuhG2sm3WVcnt7MKLRM6hWn2gYeYjImUOnOZdrs1npVSuk73+rjkesonJnqFyw679JMjrCzs92PiIqKQfEprNqRvH3UI\\\\\\\/QGYsmV0xBVn1IvgM4prv+kJgOZlCp7h2oBYu2\\\\\\\/cF7OhMC6B54kZZfRhi0b\\\\\\\/caVE5FspDn7pEr4kUXKIg9bncpNVJM\\\\\\\/yJTcSMD2PGQ8EO83H9\\\\\\\/h5K1tz7vdEa\\\\\\\/68Zu03vXP+ZezswkKgSkBeJLXbUFD\\\\\\\/K9ZWe3uYnZq9nK9ENl35Dkbym06GAaGW5tw\\\\\\\/Hww+Dpq9ew47ZQVmMXtDcHyJNZVZYeOHab5P7oIgv7fvGocSEeWwlVIpg4BpFRUXKkvT52QMRXY8wV9U5pvFjI\\\\\\\/m+RLQRcGnWjIecuvSRg70MBQpr8iD90dV4e\\\\\\\/XJtHX7tcMzr45d4IDd+nNF7jMMEbTyhFWQGsDCXwga4\\\\\\\/ZuAD7ezK\\\\\\\/NvEWQuIl5asdOK8GILN5AnyyvZ11YFmnEwTegZ++TGFemZjv5XlkAS6STmdrHwYzotGCd4JgSDzac8R3AVIx+XAx9JsVjU2PyOAls8CvBMSVqyfuBXROTKm7s60NkUs\\\\\\\/RhlBwhoqKSVFhJe1kkC6G7iyRc4fb+EdF+eARqiugd68YFtAOjRgM80OR5s4LBE11YUpr+HWEUDvHaiBN+jlsvKNRGu\\\\\\\/uq0IXvDFLeFYPNlpeqw7sfviLIeYOUei0ZhYlV415t5BBvd9H7CY1IE7gZyekiQSxw09LaxexUJSbve7Rr4n7cgNqm0KHBiFoCH+w1v5b5wri6u1XVko3B7WoR+VZUmCQVqxMcFZql7j4AfGnx6FL4Zoqlcy7KWYkEUp+SKNHW\\\\\\\/tnTgI6oVoTVzAv3y1ccHyMvJA\\\\\\\/zAFiGxQrDLSn2tAuci4ZT8hdeR2quDim8gbOKQ26gFgC5RlcAtQ42+QzU+B03ri3cITWWz4cfxdNr8SWfy98UEbUT43FPzybZ\\\\\\\/jHIX9XiIobG3azfMnkYabz75QH7+vd\\\\\\\/GFTD1Tk5ePJJ2M5JoNMc9xA+izKvq8u\\\\\\\/CAuMXgTgI2VToJmNdwDyl8PT61oK\\\\\\\/kwxjy+HAxTOjuw8xykJePNGc6NVKmo5UEHHihRhGGqzIAsdxjJ4G9Yr+8UIrUdoxfhqAYLxC9e8wKta4F8bCokvQR8KSkLYzyB2Id0nfT8JCvy+hAvnLtCtm26RnaSWf7JCPMTT0h6NBhhWPz2hpB+lHwt6K8tBbDTLz7yEP9kuoqHFhiUOTY6xM54aZnxacT4nghW68s1oUZWps8UmZDjTgefsBMh8QGUsy1jdgxCGpGPZt7sJ62j9xj\\\\\\\/lr9jOklEExGCrJ0NCQ+hOLpzuLAomj0C6B2hoxocWE30Hn+384gVrYtR5Mw5DZfzoXc6WGmHt5KmRVP\\\\\\\/YP+bsT9sGPEgeLiGglcFFwLvabE+bNGvVFfcRZchkvne\\\\\\\/H53v3morPWAiaPhMDs1l3OLnNOpBVLJUus79KgAnc+8OO2DDLDJ8Cz45on2eMbiJg93AAS1Vep\\\\\\\/iwnDIU\\\\\\\/frILPlNhI09\\\\\\\/cK238M074ZkxZ3kQFaS6lqJbuCgCfUyGT6g05kPAhcl\\\\\\\/c44B3IAy9lLDnLwSG0jWhcuOgXxEILv+yvs9RbI6HJQL0Ec\\\\\\\/U2mjUtTJKPax+XhBjF33ZQY1BfFJdSMWTwCQnzlh4CqNWzxxDaxa1SbdcmX0aczfNEdx4wzr8OHQYnbHIXFSoTpW4fv\\\\\\\/gmNWkXnvNeAN09T7JPPBvdxwkpHnd47YgDBQ0AV2wyfJED+6lFzTo68fDDzyJVsMICAixVBy\\\\\\\/KB+rEIhpyNDkuSAD7JbaH0is\\\\\\\/N\\\\\\\/JlW27qtdJIYbWldYcQn3lgngIJPl0+yYSQTr7IBNRwdgP1\\\\\\\/ubiIdYGsLluDw+vBA+3dWJIsyE7AkeUlFy\\\\\\\/N1v3+hKolQKdSYB5FyHGUCtOxdl+85cqYXUIszC35QHMF9Yhsuxkknt9cAoBEhZLhP7QkiQlIZez1mJCOeIqGzM2BNgo122nIe0u+rpYsc56N5ipCVnQUCaj5Dk06eQSfI4tYMcVFQCMOx+5DjiN2h\\\\\\\/N0pP916wqcR0JTB1teL81XgqdioydbOKXVchsNTqp7Fdrj9udj3qKHYX2\\\\\\\/Stqpm\\\\\\\/2Uj\\\\\\\/2nKqjxl7CjyBTEM+E30\\\\\\\/+c23MshWdRAJ+5C2BOIqJTKzd3iayYFKFgLjINGgDrHi\\\\\\\/bnfZzF7XlBz2l7BOPO6x+Z8o7Sc6IGSAK7KJYoB2jh3YMYatFbPZUxeu3q8X6bch191BQ63V3eLz3P+nTdJPOwi+k1y9z6Gj7DgVS774sYQ3KTki4Kqxq9GXZq9KIX36A+\\\\\\\/mQspHR9DCTITcQDUTSwMLRHPtg3XWwFQv3hU0hgQCitbW\\\\\\\/1TkjEyIDbK6ZzmPqxJsZ1Y6dhiEKIO9Sud9Yk2KicXyY\\\\\\\/dmuAd4YXLm\\\\\\\/tji87omWki8MZk7HiTo6bkiXW3FerhmBrK26DMot22F8VMx98JlYzyYCAWmT0kYsa\\\\\\\/tTXplZlo2+EGty+kqhfA032KoebJg\\\\\\\/wsghNW3t7yz7vFebvXT7Bs+PrFYJ+PE98WxmTBmnZwiY4MzUX2gKnP+YXQh5l7\\\\\\\/YD\\\\\\\/7DTjnu2WEfb5WIEuNjbiY07Ma8+Bk8Dri6xvL3a33Ypy9yodxN+kz5lrmVHmC4NQvYc6wxUr1t\\\\\\\/\\\\\\\/JrOsoZc7ywTDZWHrB7O0FmhuQHZfafTafPQZClE78UYueVYnJAjTf2YvITXGSuw5sopunXdNmujD1NM94os0VBphGBaVgBc44ptwxjekmkIztBKiMkdod4ICjm3aVKLT51Km\\\\\\\/MhtSjdouqRdJSMSh+7nmjzxcr56qZFtFwuetdhT01ihubnMjrCQLmQBUxWyG+q\\\\\\\/cyrNchaZRVfH7Vb0+RRblfEkkt6mdyr9eSFcN3wnxxY8utGpbK5xTj9uoNTvxf3ZVBh7iRxePloEg6XNi7OFXC+w4tunGJcOa0xa8420K9OhrPXoS6F6qVqGkKZGWWUA\\\\\\\/Bze5zHB9d8+ITmzHqaYfyleNotcG+3BpESpQKuNGuGI0ZGvKlaa5zL4qDO6Cssu\\\\\\\/ThIpy5MnxoxUs1XLgNLpBRHMP1nUqGNEFd1fhPu5BZz0R5w893zkO4HGK7q3mHe1QclzN77zuWM4KZ0zuqBofA5W7MlKmft3tLPyJIZUIjWKOHE0\\\\\\\/0ZbA\\\\\\\/6gDIbRezOFh8JA4W8n6WJTMB76iEoAJA00immQF9tQOGm3ZTHuTWjfWfQ2CPPIEXCB14Ga0Be1vkAKmmisM4kQqdi5TXUzzDd1yDzlvIZnoS34mGT0t4xnjZuz6Kvs4qS6umW6To0gu7iPm87GRB4w\\\\\\\/OVPf1XPFc8zB3HL0yitaqdAI+aJcOUfW2yZjQvrn01dMoHx2TCMDQ8OdbJGvwIwytSEExQusvQkVLHU9tRQ0aLsh2aHMadsqwXRBUsZTmJ+LZVTbuO7l891tF\\\\\\\/qp5HGyhAdfA1wf8virA0rGmQk4hkPLIIU9ApFr2tKhn61MmqmWHRzZdr5fyUzWEo8rP4YxOLPmQ7QagWu+jy3zxk3so4suc1xt12IlsVRD9+oItNOtCCOY\\\\\\\/kMrwIqY3m\\\\\\\/XkBaj17HW3gAVcrjTdtzsZNuzvDtKmsO06WbHQAFAaNgHxItGIHnc+rnaiqDgHZwsPt5fCQVorSbz+a4trvNTg0pJTbxwNO3yncmIbt0tzDpFg68eOtfwcm+KvbkuVNXNHxtGTx+PPW8FF3c3bks0dLfXmLJX2ojIsjZ3aqTy0nBTDnKblydUgXjqltDu2EE7LXf50r5FBjf16gcaodzMKzRwrgp4H9tB7Lb+jMGh0PvoPFZZ1EQNe8MXwqFhxc5LYTd\\\\\\\/lSmtUQdwbMTuFVYkjsDP8dzEewpqVphQllDtGe\\\\\\\/krImRilJXkDKtN\\\\\\\/ggQiz5VNjHhO+6YNpevOMzSAvt7s7wXReTXfZD11NujemGzcn6ga2FSDtgIFAN8rf6RR1CjZXzd2oemU5aVzytxj1MJ6i+\\\\\\\/p1SALDWjJZTIdy81CSu8KCijEcrAu1krqG5ingKm\\\\\\\/AQ3MA9BmCOJmVMvYoLDxJ7WnsP9hpeWe7IvFRDgHXahKXLBi48VWfewIz9+a41Zg1YzYwUx5tRLKDuZgWpMw1XVobRBlQjvZEpeeJC7eCkj7Su5lnzSWdwrNcmGfzd\\\\\\\/+8zTdzHjzx+J0sDD3dyM9+FzNyI0\\\\\\\/0lYHBiuJVLxZOmw1fx59tEUFzpUmhytihSkyuLG9kqey1wcryQ\\\\\\\/0z6ikz\\\\\\\/eQPmX9vzc5qMRjWgq2ZLPZg\\\\\\\/qySR4YO\\\\\\\/zxTRmj1H\\\\\\\/kX44KZyK0NuqgJQ0n6yizHJqgoqZEjT9TjiVjpab7vlkssGisGL4pf3luPLXZq7JorUsBHCPtjx2sAhBl9\\\\\\\/Z3iripd4ygwUFgnWNU5t08Ug2sHWFlG+DBsLeTKZFCz0DEkSqIdWbCa8FfTq4qV4v+GYAmAv+ljVQgm0uTtbISilLN+ht4UmANl570kzgE6dWzaN1EqIF7RUzjL5zsknblheQOBMVFrdRrFpWyF9dQ5\\\\\\\/gtXwgbMStpWoxH\\\\\\\/Nk1tRJcHYluzrtBoe3dXN0yK+MSC\\\\\\\/WHHF4Icqxka8dfVzQMO+uhQioo5LxA6VAy2KjEQxjV2qF18RN6P6dEjPOOXlaDCNuJg6CV3bAjMRXSNo02wzu58sLGJ2tWA9JflEJy5O3Oszb\\\\\\\/Bqpo+lwu02qickeM7R7oEoIhhp78OL3a7o+qjyBvpVkvdkViuzSPrN9Q8ayJrQk+9isVyZNkPvn25Jfj8iwALTz441Y\\\\\\\/\\\\\\\/98HcqBqsNNhzbrY69jWk4f5rJjkZz8Y0dtch3WTFpnEhNFCY\\\\\\\/kWIC8Ck+pN+oZm1sPq47Y0GxGFw\\\\\\\/iNgIKNIHT2ig41AXxGxGkYadNJBiKMLqHLDWTyd3HJtCmdr\\\\\\\/FPdzBHI5A6GExr1rw9TC2Br0okB0Zx1M4jnGKWNuSMIS0Ue8WRBVOlrukeqA1ZTiNIXahpiUh5umYfOxHwwlPaNdyg7UAj7zywRS9aI+uObGJCd2hHk\\\\\\\/lfVmrWlX92BQGfZwVVpw2qarQirzgaDfr5C5McvqN\\\\\\\/sifHt50VFJoElcZ1gXmm4I1kkQb\\\\\\\/k0xrN9FZHX1nNTShpJ5HO+ypyth3ZyN2RTzc7CtEqAgBoOPw7+kSrE4f8GDq+GB0OupGaYo0y00qXIp7msVG5ZuSnQRWM\\\\\\\/dVtbgg0aSE4klwRLuTdy5iU8bAaas9vFPUCASOTkSyhvEFD\\\\\\\/mvX9P+kR5D88B4FR8oeRdnm+3XIQHOTTolPwDw+lAz\\\\\\\/Kibx739Q4papC6H75u97iD1ZeJjd9oXOKXGa5cjIHtjPLNz\\\\\\\/3Gjq5BmM5S6\\\\\\\/GAu5DfDspKiOE2w\\\\\\\/vQqoT7mGQYe2IYgvksOVUDXzy4DSaDvU0BksVW7PowZp78SpDun8h5HsNwCJKusppufcRWWRclF7rdbOeGq+2icl1B+MzqYtf89p5Qvi8KtKVGbpi9aI6j+gxqNdHsE8xEMrBOwHSKfGUjCu+s43n5RA3txsCcB05YtsMqVzLjPCTzT3Ndf8V1Cq8e5PNl9B4U7Uvg5DPsYqbugmZhnn0c10YuZopmB4IA0WRLwkYpuhiiCHhL1OsrXzJgCIHG2L2MJFgQBuC+RKkua4VoEGG10fVhAQx0DzKhinQPWhk2hbVPtVMOuSYuV\\\\\\\/8w9ksxuHT\\\\\\\/+zGQyZhwp3lvvphcC9sFC576Kda8l0gR4rjOAZyHhU18gKa\\\\\\\/uNe0tJJ7kSiaXmnRRDSxJKN1yOFN+qjyZbrscLOzlkJGeG0n8Y85TQzeWaLUFSIZX2nAJgmeIU8awyHTDmzqKTkrKYNaDUGxhbEIxz2z8eybS4uZT5K3BzJlqpAMG4GFXJI2rF9rVm3NiaYTy2iBlz7VuKRFIp5LqD+7StkKMWU0d2GJ1bDre0Wp8lzbvJ1IsLOT0r\\\\\\\/AVJTCHrWQ8VLUO7vhSeKioZgexKaElh8S+f7\\\\\\\/TSK2bXB0qhRegme1iAOOlQQnWw16AWKdcDiszP+uQfeHWdN8HTCFbqlFzGWxnEgUnaioOPDOt1Wu0lsAyGIf5foSAbUPge1JygGvdbX5vZ\\\\\\\/nSRfsMq8LX3g2RNOc7BnSwIOqs3MHV71lvU2xkvqfuz8aMgq5vdsFsua7FCSAiHfYLjUpsItmizHWcMuB4iY54PCSoUtGILSb8BgmtxCOQ+liO5qyw6r6Um+LgJVCMet5D0xET3y6FKHIGbm9WaFORvTpGsjqO6\\\\\\\/GqG28NTcyMrGavEZPslhSYZ2gpsrAVhBl\\\\\\\/TG9SPqDomjKg3VKCb1RMT62c0o\\\\\\\/4hC8OeUCLDP3jpxUrgn+RuTiBqgnRP5yP+dAp6lMylGgUEGmqbEbPPsXcigmn0dO0bMYtRF54QuDb3prCE97QGGH7y9TnBnHKsyKxNJjgJ0roFXMA2zjLHDQcaszVIDdORgdxKFx73KQYq1K2ryDD3BZ3hNtnFr5YJzDs5UmdwnVtTiKO+tr5oHXDePjzTALA+\\\\\\\/dIXmZe+crAhaDwK8bYBCoaNcyWRZXEB7VwOV8Eooy+EZHJfDJckM\\\\\\\/49VqJgYKpsJ1MiZd3B2nNCAaQ0Ezc7lltfduKVEXp9bMDB5EFVuv+r04F1y3KGAeL90T8SxZ5I1uJF5AY1Eer+FTPoW5ESLd+HpuihxUQM2pkkEvtwDzYnFgVtojZEVA86lJDf60EAssPBd0n8Zx\\\\\\\/Aso1syANUprfBff\\\\\\\/EliAOlBEITqs6yoAFzDjEZFEM1hQsBsfU+8PtWq3oCoZMXisyLNoy\\\\\\\/5+5iosq9pIvKOEaFRIzBXZnZnsj1Dl8agCSgSxidApH2xHxl5NE0ZIon64xmUhRXr2mfbcZPYhZXm4Hj+Ey94GbyyqKKT5MYzf3gGO3GJvqbTnwYWLuO4sIe1YJ7g8HMSssANpdEk\\\\\\\/uJriipjUZpoHv+6nFS5wp85o1Kqu90EVaYAdRYQrP8ymzi5fxgPcYR2gTe1PYZvEsBb82VH4mVBve2E\\\\\\\/D8dN7m+2Xalr9S+sP1nR+4sSmXh72EjMqFinHSuOIxN+\\\\\\\/DgpBKmR\\\\\\\/Xs517HTRumMQvCAng1zVt0An0LZBSCLugce6NQZXCYO0XFu1OO1yROouuYo42UHI9ySIRzOKUjpWi2nyfJC7IK7oSNMAKZ60mePp0zXPnq+vKgYdHt\\\\\\\/ZO7KF\\\\\\\/yzh6e8fxsvv0DI3odh4bG+vgh3mZmrZtS3a01aZhtI27ljNVdwnyUTTzhdOqTKN1eM5fWXcXQneIaNzN+VDQbIWiLdfrXtVrZaIZTYNC+zWSsu1JADbKjbwqeU+I8RX\\\\\\\/odlEtoT\\\\\\\/QlvqAGP\\\\\\\/twq7cwHPxr2LQgCvnJbTeCKz4Z\\\\\\\/8I59N2YUtx7cqdgNNi8j+ZY93jaB9qwllrPcmnI2aU8XNvw5N569rsdXH8OoUHqy5GRwJ6xVeaii9u0LVtsKJiVk6Su+kAf33jCFGmx5DWnbuSjj97EFb4T7nANsAJ9tdkzIum30aK75bBPBw92Lz9xVlO098tj0TSQzCNM5k7e\\\\\\\/3BJQBbeBOgcbGIBp73G+xjdavi4p3N4GC09OYxE3FP1sihF22O2zNhRqZGBp7VGnQ8SZJQB4qUNLH9CebpUX7rYerWBZ8q69RRv6AGl+05ybykXiExMNusRI0IKP2Swp1qx0ljG03eF\\\\\\\/IPGALmlLZWB69W+DNxRHuq4B9Z3HjY3swhkIATqEBuEUg3NNsd9UAGj\\\\\\\/+9EVUsU8LZlhzRriIvrmAyA7YQQtt33ApbXwy0iAVcnpo4HaVtK7bQJddvhAHI5zzHW3vXA1apg83q4d7kBBFHIJSg6\\\\\\\/+UTTDkFDJwBN7ksBKYoQETdZ9S\\\\\\\/Z3iODS\\\\\\\/bxlVZMNngHZTNjbryaIPqr4Vq3Y2ce70HRVNtcI6\\\\\\\/SRkmPGyZpULy07mGhIfOo36d5ImCBnYaxt0prRCra2DGiBWMLfd6j46XXVApHz9ADT38SIyt3W2qVwa3VNh\\\\\\\/ivqwD1mNoiIQmLlUMn5NDIaRXyKAs1Nyxh0dH1aYRxe\\\\\\\/MJjQuhxOkwq+utdm3SlKnoRW58JZNUITnaemalQK\\\\\\\/UtOluwjm7opBpR\\\\\\\/5jCcUK2ljtXYgnp2b4pgc3iiXuAq2o0Tjq7Cw\\\\\\\/unTyUZcRAPTz\\\\\\\/oOWcgV\\\\\\\/3llIzzADnonzcdHM+zZ4a0OdJuMJ9UYnmQHrW3xp5gChKbUwbecGjHmwJEcYvsvlXqhehQynOyejva7jX\\\\\\\/TdtVLA37WejuqscCJF8\\\\\\\/hXwuHBEmC1YHrE8TRX30fU0uk1bAUZHms2gyPr69hs8t0lZ1bo5DP4IVSNiYLEmSN7Pl0jHr+u\\\\\\\/b3Ci9zWiDOj2fNwCjI1Pu\\\\\\\/cmu0nEzi3UicooGO4CF9YHrnlVAyJ3eso55Zs2KB9gUXSN\\\\\\\/TPnASK1CBZCgHs8+27Jwto4I0IEvI4ngVoCqupATT0v5n0Rg0xFVD9BrpyDTpB2N5vev3MjXLX2XNP4OnJRSJl6hQzPXYctc9HPWm\\\\\\\/FiK6l+DKtS5TkGx32Qsy\\\\\\\/evqAX6NcoQ6PUpVLFQW17f4eiH1tiX\\\\\\\/Ai+uPd8n3RCVdhylppOxyFu0cH13p1uEJwGCOrjtcQDi5f2Mc8nCtHN7PW2uYCaPZGxA3q+jwsGP2qtRrdvn513IPSVqqgrJj97mk969XvbTEA304P4lbHUTO3EnLbDx\\\\\\\/4jtbUA\\\\\\\/HAuwpgBJiKd5qc1WXNL4VgHETUkEdF\\\\\\\/0PafiZlflFKQgURcSL5sgxEBNmeOjEyuJbxC1jjVGIGCKyrMRmj+BTd4Gb5Obvw0KRF1Wnh9NkL4yJnlzoKe85yXF+2XcU+M3fa+D\\\\\\\/RMG81LNuGgP\\\\\\\/4ntgfv2AmRMFS8mG8c82MxPzNvWI\\\\\\\/wcUBvPMpufzYD9XhtvJQND2JimF89MZEk+di356k0LlV7i8RFpqCgX7hbZDok3ip\\\\\\\/TfgtAb1OJnJgupKqcg5ZJirvtNTu4xHE+WvSQjPanQs+9z58M+WGfxaLEgWNpGQGdh5lH\\\\\\\/xbOMdrR6TflNBq7UjOJLbOGYhlEifW5sO4\\\\\\\/frehN0bERFt1pk3U+SoiOdGG8R1U4PQnHeAxinQI0skXMsHPa6adAJfnouEFT\\\\\\\/IK4mhMcifRvz9yBAGYukCSP6xx8kORe6qeq\\\\\\\/ptU9Zm6BeyFKbdbfHzZ0XDy32gIlQm\\\\\\\/C\\\\\\\/OjPMlJ4WplSYK\\\\\\\/tGs0DB+BNZfxixA\\\\\\\/S9EgVyXBdAo9lKjBURr2nsdvVSZkm6donsyoTVm8c8BZFxXeANeSJTQ2txzftVD8AVXH7mjfE5B19FNIFyjA1IYdkL7pcGyJnWDFgeSF\\\\\\\/eApLwFAOhMvsWO1e4HQyEsCOJcYAuAsagVY9wrEI28+1O+fp0vRgc5WhqfxjmTVLBF7uERxF3Up3ok4+TnaOTu\\\\\\\/ArhZw4W8WFw4y7+T844P3GncUo4\\\\\\\/CJjMVgsJEfb+mPUmW8tfCEOxe0wlTrpVZf6GYxHQaTGxSYb5abqHWSLHpPhk+yWR4y9k1+Yzp1mCngAslsGSFU1880OqaYMetl7iuz9DsvaboTI9Il3PryfdjFU+7F\\\\\\\/VX2cPg5woUyxkis5+dCPEOv37ikYv5NuWGHGsKeoVZeh6Uu47YhPy\\\\\\\/w\\\\\\\/TKaTjYsvnFns\\\\\\\/UpcdqOKNtRAqww9xrELn7THjmC\\\\\\\/ubHHEmTfmZv+1mV8a9Kl9YtZ6QfK0PW5p6LQGzQp2oNrNsU5jGaNITaAT9RFi34ZvdUY2xCKQEpbD0VebhRnL0yTfKTHZgfB5DsmiU8txRdty3obiKi3oLPLx9WnzZhfx7YT1gc5cUtcJoiAbEjpt1hl\\\\\\\/eWr5zwSmyPICtNjl9T9Q4oQxro2qdlM4tet0kP7CNr4RVJkdFmWncNkgdVGuTDWM5CSCiX8fE1mwWYFhwaNTSQ6LO45TWmnEa7uI2oKgWCYlWRKEX1ylrdTTYFTaW86OlWFTDjrJ2GxsFfHx2sue8C0xxUzY0SM9lVPqm+spQKEjeK2+PeWMygi6o2gldEPgwe5z5p4QnmdV52l81CxJZSz73zvlD\\\\\\\/ZoIS3KmnuBOzmEkT5SZrBbGBwIi7yuYeQnECNkSaEyBaL0PKqf1ye1lGoZaGRr9RqFQOB6Ys1OLROqYbwPCZ14ybxPH1UhVAMggAhSJCjhJG1deFA+rh1wOvCr8U2YiUy0v92P\\\\\\\/dnE3aPlrz\\\\\\\/hFNOWgLwYip0v9GNM0rqtZvWpBXQ082+yDiftW4p\\\\\\\/tNSP\\\\\\\/3BWOp+fQJgjGDlC+x5I\\\\\\\/Sj5i2qaNvY+9vGyvkRmUj9hhCm6ovhFy5qZPKiRU0MfJV+iXiCibqbM\\\\\\\/mzjyroCL1o4sHs0Q15Ym8gcReseykoksXmQbZCHc+0uiYoqEEU3rngESqmK3eErJHzBtN6RCIWTtIGjl9XHVwayi+XKjj5L5dF0uSyDLtU63a4BEiD5B9UI8aL5J\\\\\\\/8\\\\\\\/xnnDwqsmvyIUdcYKYDuSXvKotn3W3VEPsQUKFqPV0XPGsezWqHruw916H\\\\\\\/\\\\\\\/bFu5yKUoWly6bLZwU6XHGqVBq\\\u0022,\\\u0022iv\\\u0022:\\\u0022UWayepIfR49RUARyKqsLnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022614a2b8186f46ca558b26fdbc089833c1fd2d6e7c72adcdad26f01db772c87ab\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cA9id8OqynSANBcCi5zfccR2373tIft5cb+pseznEKK45dRM7RTprdS1M1kvIoiMcdQRQPfC73KMfdj7q2vMyb\\\\\\\/zBidgIg9e01hjbVAuozzROJDa+vaK6vBP+WUE37w68aPkmg+oYMs0\\\\\\\/qOQsw\\\\\\\/ejJiRJNQcgFbuh1yhKI72MbLp4vLuB8\\\\\\\/ZLo3viZxv\\\\\\\/2e1LO7MixOKOisr\\\\\\\/7Ttn6cEKYe6cHxH11ZDYLzQrQ3nu4nEWpt4Oo3KHSc5fqebrLXfOJqg1XX7mcQCVdzzhUvS1aSFUnCGnkPSlUCzEhhQ9RkoL8SI6x5Cea43DyoyWG3AOnJIAObPRuophQN\\\\\\\/JQ1pCLDL5\\\\\\\/4OYhiF+eknlvYpNpxX4Xqisj+n+O\\\\\\\/lFJ8m4aElEBAdH4KnQx7U40oEYARLDJc1jc+wIvD3hjVe5Z+rzHQKOy9MqLL9nkFnlQ2L6e2yqSuNQNc6mqFQ5I67KLEPRrIdUSDcBnoGQ1w2kKaPqMCdFbieeImuZk\\\\\\\/XvM02pAG6zqxlPmjxKWQSMZzkmTtUiy3iVGfeYFXgprrMtBaWynIGYsKWdZUBg1c+J1kI8xBN+l8via4KVOv3mUcQ14a8rjKA0Y7hAgmDD7NSnu+qMJbiSKL3nlPVIcpyddKonnxzJQy2Lgeus64dlilG8v1Jx+STUPhWGl1+ocA93fOhbI\\\\\\\/QOR7xxLn6EP3LyyvkQCiSDrMJutLEGLa6b1aynitaN3jsiAZorhX23SgDvOvfKUdJBIhlkMu839WoI58wHoCvivanm1z+v3gfWhKLn\\\\\\\/GyFrvUTX\\\\\\\/8fnOaoSvlCOqXlRSzKEGxmLTd54A3bN5zTa4aFbuc1c6dVdoM4y17ZWCRy8Fj68XotyxHqPG8DNqNtqOPECMxgO3wD0YXJk1pEY0hNNe7407PyJPXyv48iCDSnfg\\\\\\\/EgMlZk\\\\\\\/UZ\\\\\\\/K5ZBMwNndZ7k4ets9pNLK4EKZ0MYOqXUBUlVXsQEBhWpQE8R9h9DN02z+jCb8Uqqo9HaUSxDj0ejqzx\\\\\\\/MExTOiItEGr8YA3N1t+W172cAaABlLQjDNZhNbhiYJiKN1Yl+NXwVsQnoxOobF7Oi9NpJwL9qM7BHgVNN5XrYW1HaE7xbE4E4NkZpQtDq0RuFiP3Vb99AQxEo7onBwmm3YBdFN45b42Nn2TcwJM3zLm+xlGINpdF6BRkygTiKKQgO5sbbnQI0IIm3Pysrn4XRq+dcRZeq4Cn6tAa3+Gf21leaX1zGYUm4r9oIEXAaqoqYRvqbsc1ciOgr8B3D+paFf+NXV3rxIIT+gg2J3XtwSlyeJ9z4pvxs5CSKa97EYA6U\\\\\\\/ZEaJpZ9A0hqHjZ80Bgm1JW9KuqLVt33VAPVv1K\\\\\\\/BHif1zalp197XgqHg0qgg3\\\\\\\/Hyrxy9cPjLwQmQ8voJDihIKlvx0VeRJvV9km7QVnXebPmRK64jOVhqPwpyiJ3df9FTeWk1K2Lese11871upjJROoqaFUcGuRrKW9su\\\\\\\/Lm3YWfCExrk671V1inGzmXB00BB+wnN7PKZrWf4oroEa7ZgsAHxAbZufDIGRIY8G8mEMu8u\\\\\\\/wCcAcfjWjhrXAQ1U2N6yE2VamItNrPXFYry2KFihjuxZaJeQLy0we8U9Rn2KUIxNHRNVHVbRxj9pYc16f\\\\\\\/xcbEOT4hzmu+xCcTvBphrQxAc0JSvAix2ieLY4XHoJBIuhDYqSp\\\\\\\/4BYNaHC62UzfeDGEjLxq27HYFJRyZrIba+T2R6NsToh8zP8pvdf8JHh2UWIMrNZWQrMNGuRmQToG3tcC6FAdJDorGyDV9LtwXEEp3WpAsYjZJ8fhmHpmQ6aohR8FtC3tulPXHl0qGDUfIVqMXd5tJGQWLsRjx0AXT1NPigpMhfSdIyR1fMedbxeXQMr9miniW06dv6Kiq94Ot8oZYa\\\\\\\/AG9vDAUz85nAD24yPM7+XkcrlNfGlDy4DxGYQW9XXygs\\\\\\\/Er6MFe1n6bIC5DiCo3uDCp3\\\\\\\/7ZP6NiYM2I9ja3MLs\\\\\\\/ZXRefK88tOuHhLELqKZGb37829\\\\\\\/e7erD0iEPbDVbIguY184iZjowHfjOSzX1HT9pyOh\\\u0022,\\\u0022iv\\\u0022:\\\u00222lFNXzvSbskoPukkHRcZNw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226d292c8c569ce00dbbc2ff0aa843d85439ae537d6e34c84eaff58407b4b06f3f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459851 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/553" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FxsOOb9aak8OF_4T853rJwvUvUI90ANx7ZuQ4fHOF0o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/553" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lJLEMhCiGkuZTKcBbL5jm36k9eObGGugvxYJqawt6tk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/547" title="gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/561" title="Comparative Genomic and Transcriptomic Analysis of Wangiella dermatitidis, A Major Cause of Phaeohyphomycosis and a Model Black Yeast Human Pathogen" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459851" id="node459851" data-pisa="ggg;4/4/553" data-pisa-master="ggg;g3.113.009589" data-apath="/ggg/4/4/553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying-hui Li</span>, <span class="highwire-citation-author" data-delta="1">Yu-lin Liu</span>, <span class="highwire-citation-author" data-delta="2">Jochen C. Reif</span>, <span class="highwire-citation-author" data-delta="3">Zhang-xiong Liu</span>, <span class="highwire-citation-author" data-delta="4">Bo Liu</span>, <span class="highwire-citation-author" data-delta="5">Michael F. Mette</span>, <span class="highwire-citation-author" data-delta="6">Ru-zhen Chang</span> and <span class="highwire-citation-author" data-delta="7">Li-juan Qiu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">553-560;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009589 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459851" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying-hui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying-hui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing-hui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yu-lin Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yu-lin%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYu-lin%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cytogenetics and Genome Analysis, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Corrensstrasse 3, D-06466 Stadt Seeland, Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhang-xiong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhang-xiong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhang-xiong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bo Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael F. Mette </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cytogenetics and Genome Analysis, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Corrensstrasse 3, D-06466 Stadt Seeland, Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BMette%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mette%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BMette%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ru-zhen Chang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ru-zhen%2BChang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARu-zhen%2BChang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Li-juan Qiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li-juan%2BQiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qiu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi-juan%2BQiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:qiulijuan@caas.cn" class="" data-icon-position="" data-hide-link-title="0">qiulijuan@caas.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/4/553" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459851" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/553.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459851" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/553.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459851" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/4/553.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459851" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/4/553.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459851" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Identification of genes underlying agronomic traits is dependent on the segregation of quantitative trait loci (QTL). A popular hypothesis is that elite lines are becoming increasingly similar to each other, resulting in large genomic regions with fixed genes. Here, we resequenced two parental modern elite soybean lines [<em>ZhongHuang13</em> (ZH) and <em>ZhongPin03-5373</em> (ZP)] and discovered 794,876 SNPs with reference to the published Williams82 genome. SNPs were distributed unevenly across the chromosomes, with 87.1% of SNPs clustering in 4.9% of the soybean reference genome. Most of the regions with a high density of SNP polymorphisms were located in the chromosome arms. Moreover, seven large regions that were highly similar between parental lines were identified. A GoldenGate SNP genotyping array was designed using 384 SNPs and the 254 recombinant inbred lines (F<sub>8</sub>) derived from the cross of ZP × ZH were genotyped. We constructed a genetic linkage map using a total of 485 molecular markers, including 313 SNPs from the array, 167 simple sequence repeats (SSRs), 4 expressed sequence tag–derived SSRs, and 1 insertion/deletion marker. The total length of the genetic map was 2594.34 cM, with an average marker spacing of 5.58 cM. Comparing physical and genetic distances, we found 20 hotspot and 14 coldspot regions of recombination. Our results suggest that the technology of resequencing of parental lines coupled with high-throughput SNP genotyping could efficiently bridge the genotyping gap and provide deep insights into the landscape of recombination and fixed genomic regions in biparental segregating populations of soybean with implications for fine mapping of QTL.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/soybean" class="hw-term hw-article-keyword hw-article-keyword-soybean" rel="nofollow">soybean</a></li><li class="kwd"><a href="/keyword/biparental-segregating-population" class="hw-term hw-article-keyword hw-article-keyword-biparental-segregating-population" rel="nofollow">biparental segregating population</a></li><li class="kwd"><a href="/keyword/resequencing" class="hw-term hw-article-keyword hw-article-keyword-resequencing" rel="nofollow">resequencing</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li></ul><p id="p-5">Theoretical considerations suggest that an important prerequisite for a high power of quantitative trait loci (QTL) detection within biparental crosses is that the genetic distance between adjacent polymorphic markers is less than 10 cM (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Piepho 2000</a>). Moreover, it is often important to saturate the genetic linkage map in genomic regions where QTL are located to develop stable diagnostic markers for marker-assisted selection. Despite the accessibility of established molecular markers in major crops such as soybean, it is often difficult to find a sufficiently dense set of polymorphic markers for a specific biparental cross. The current approaches to screen available established molecular markers for polymorphism are time-intensive and labor-intensive, and there might be genomic regions that are barely polymorphic between parental materials, which are particularly probable among related elite lines.</p><p id="p-6">Many molecular markers are available for soybean [<em>Glycine max</em> (L.) Merr.] and have been used to establish integrated genetic linkage maps. The first and second versions of the soybean integrated genetic linkage map were based on isozyme markers, restriction fragment length polymorphism, random amplified polymorphic DNA, amplified fragment length polymorphisms, and simple sequence repeats (SSR) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cregan <em>et al. et al.</em> 1999</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Song <em>et al.</em> 2004</a>). More recently, several thousand single-nucleotide polymorphisms (SNPs) markers were added to the integrated genetic map (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Choi <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hyten <em>et al.</em> 2010</a>), bringing the total of integrated markers to 3641. Focusing on specific traits, a series of biparental segregating populations have been developed for QTL mapping, mostly derived from crosses between two elite cultivars. The use of crosses among elite cultivars, however, is potentially afflicted with a high proportion of fixed QTL. Studies based on pedigree data suggested that most released cultivars possess high genetic similarity because of use of only a few prominent founder lines in soybean breeding programs (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gizlice <em>et al.</em> 1994</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Xiong <em>et al.</em> 2007</a>). This genetic similarity is particularly pronounced for cultivars originating from the same geographical region.</p><p id="p-7">Integrating next-generation sequencing technology with the development of resequencing analysis provides an opportunity to quickly identify large numbers of population-specific SNPs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Huang <em>et al.</em> 2010</a>). For instance, 150 rice recombinant inbred lines were resequenced and a SNP-based ultrahigh-density linkage map was constructed (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Huang <em>et al.</em> 2009</a>). Using genotyping by sequencing technology, 30,984 SNPs were added to a rice recombinant inbred lines (RIL) population for QTL mapping (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Spindel <em>et al.</em> 2013</a>), suggesting that a sequencing-based approach is fast and economic in terms of data collection. Moreover, genotyping by sequencing facilitates increased precision to determine recombination breakpoint compared to previously constructed genetic linkages maps based on sparse PCR-based markers (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Huang <em>et al.</em> 2009</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Spindel <em>et al.</em> 2013</a>).</p><p id="p-8">Soon after the publication of a soybean reference genome sequence (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Schmutz <em>et al.</em> 2010</a>), resequencing developed into a major technology for genome-wide polymorphism discovery. Until now, 32 cultivated soybean [<em>Glycine max</em> (L.) Merr.] accessions, including two parental lines [<em>ZhongHuang13</em> (ZH) and <em>ZhongPin03-5373</em> (ZP)] used in this study, have been resequenced, and several millions of SNPs have been identified (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lam <em>et al.</em> 2010</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2013</a>). In this study, we developed a customized SNP array based on the resequencing data from ZH and ZP and applied it to genotype 254 F<sub>8</sub> plants from a segregating biparental soybean population using a high-throughput Illumina GoldenGate genotyping platform (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hyten <em>et al.</em> 2008</a>). Combining information from the genetic linkage map and the resequencing data on the genetic divergence among the two parents, we constructed a soybean linkage map and observed the presence of recombination hotspot and coldspot regions and of several large genomic regions that were genetically fixed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping population</h3><p id="p-9">A cross was made between the modern cultivar ZH (provided by Professor Lianzheng Wang, Institute of Crop Science, Chinese Academy of Agricultural Sciences ICS-CAAS, Beijing, China) as the male parent and elite line ZP (developed by ICS-CAAS) as the female parent. ZH is widely planted in China for elite agronomic traits such as high grain yield, large seed weight, low plant height, and wide adaptability. ZP is a line with resistance to soybean cyst nematode race 4 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wei <em>et al.</em> 2011</a>). A population consisting of 254 recombinant inbred lines at F<sub>8</sub> generation was developed from the cross of ZP × ZH. Genomic DNA was extracted from young leaves of 10 F<sub>8</sub> seedlings per inbred line using a DNAquick Plant System (Tiangen Biotech Beijing, Co., Ltd.). For resequencing analysis, total genomic DNA of ZH and ZP was extracted from fresh leaves of dark-grown plants at the first trifoliolate stage using a DNeasy Plant Mini Kit (QIAGEN).</p></div><div id="sec-3" class="subsection"><h3>Whole-genome sequencing, alignment, and SNP calling</h3><p id="p-10">To screen for SNPs between parents, ZH and ZP were resequenced by BGI-Shenzhen (Shenzhen, China) using the Illumina Solexa system. DNA library construction, short reads derivation, alignment onto the soybean reference genome (<em>G. max</em> var. Williams 82; <a href="http://www.phytozome.net">www.phytozome.net</a>) (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Schmutz <em>et al.</em> 2010</a>), and SNP calling had been described previously (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2013</a>). The repeat mapped, unique mapped, and unmapped short reads were identified by aligning these reads onto the reference genome using SOAP2 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2009</a>) with the parameters reported previously (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2013</a>). Only unique mapped sequences were used to call SNPs. The type and location of SNPs with regard to the reference sequence were determined based on the Bayesian theory and the maximum likelihood estimation method. The certain criteria for improving the accuracy of each SNP were also followed during the previous study (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2013</a>). Based on the open reading frame in the gene prediction, we used a self-developed Perl script to identify the synonymous and nonsynonymous site after we replaced the individual genotyping at each SNP site. All sequence reads and the SNPs were included in the Sequence Read Archive under accession number SRP015830 and the Database of Short Genetic Variations (dbSNP) with batch ID 1058942, respectively (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2013</a>).</p></div><div id="sec-4" class="subsection"><h3>Marker selection and polymorphism detection</h3><div id="sec-5" class="subsection"><h4>SSR, EST-SSR, and insertion/deletion markers:</h4><p id="p-11">In total, 928 molecular markers, including 880 simple sequence repeats (SSRs), 37 EST-SSRs, and 1 insertion/deletion (InDel) marker, randomly distributed over the 20 genetic linkage groups of soybean were screened for polymorphisms between the parental elite inbred lines ZH and ZP. Among 880 SSRs, 789 SSRs were from the integrated linkage map of soybean (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Song <em>et al.</em> 2004</a>) and 101 were newly developed markers (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Song <em>et al.</em> 2010</a>). All the SSR primer sequences were obtained from the soybean database Soybase (<a href="http://www.soybase.org/">http://www.soybase.org/</a>). In addition, 37 EST-SSR and 1 InDel marker, rhg1-I4, were included in the analysis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Liu <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nan <em>et al.</em> 2010</a>). From these markers, a total of 179 were robustly polymorphic between ZH and ZP and used to genotype the complete set of 254 recombinant inbred lines.</p><p id="p-12">Polymerase chain reactions (PCRs) were conducted on a Model MG96<sup>+</sup> thermal cycler from LongGene Scientific Instruments, Co. Ltd. (Hangzhou, China). Each reaction volume of 20 μl consisted of 50–100 ng genomic DNA template, 2 μl 10×PCR buffer (containing Mg<sup>2+</sup>), 1.5 μl 2 mM of each dNTP, 1.5 μl 2 mM of forward and reverse primers, and 1 U <em>Taq</em> polymerase. The program for PCR was as follows: 5 min at 95° followed by 35 cycles of 30 sec at 94°; 30 sec at a primer-specific optimized annealing temperature; 30 sec at 72°; and a final extension at 72° for 8 min. The PCR products were separated on 6% denaturing polyacrylamide gels and visualized by silver staining.</p></div><div id="sec-6" class="subsection"><h4>SNPs:</h4><p id="p-13">The primary linkage genetic map constructed using 179 polymorphic SSRs, EST-SSRs, and InDel markers included 45 large gaps with distances more than 20 cM, which partitioned 20 normal soybean linkage groups into 45 linkage groups. To bridge these gaps, SNP markers were selected from the genomic region of these gaps. Of 384 SNPs selected, most (240; 62.5%) were nonsynonymous or nonsense. In addition, all of 384 SNPs were with a design ability rank score &gt;0.6 and a pre-evaluation of 60 bp of upstream and downstream flanking regions. Then, an Illumina GoldenGate SNP array (Illumina, San Diego, CA; <a href="http://www.illumina.com">www.illumina.com</a>) was developed using these SNPs. The SNP array was used to genotype the complete set of 254 recombinant inbred lines and the two parental lines using the Illumina GoldenGate assay system following the manufacturer’s protocol. For each SNP, the lowest acceptable scores were set to 80% for GenCall and to 0.6 for GenTrain to separate homozygote and heterozygote clusters.</p></div></div><div id="sec-7" class="subsection"><h3>Genetic linkage map construction</h3><p id="p-14">A genetic linkage map was constructed using the software package IciMapping V3.1 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2008</a>) (freely available from <a href="http://www.isbreeding.net">www.isbreeding.net</a>). For grouping, the threshold for limit of detection (LOD) scores was set to 3.0 and the threshold for marker distance was defined as 50 cM. The recombination frequency was converted to genetic map distances (in cM) using the Kosambi mapping function (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kosambi 1943</a>). SERiation (SER) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Buetow and Chakravarti 1987</a>) was used as the ordering algorithm. The criterion of the sum of adjacent LOD scores was used for rippling after ordering each marker sequence. Linkage groups were assigned to chromosomes based on the publicly available genetic linkage map (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Song <em>et al.</em> 2004</a>) and the soybean genome reference sequence (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Schmutz <em>et al.</em> 2010</a>). The genetic linkage map was graphically displayed using MapChart2.2 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Voorrips 2002</a>). We used previously identified pericentromeric regions and chromosome arms (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Du <em>et al.</em> 2012</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>SNPs discovery</h3><p id="p-15">A total of 928 established molecular markers, including 890 SSRs, 37 EST-SSRs, and 1 InDel, were first screened for polymorphisms between the parents ZH and ZP. Only 179 (19.3% of total markers tested) showed stable amplification and clear polymorphic bands, including 173 SSRs, 5 EST-SSRs, and the InDel marker. When a soybean linkage map was constructed using these sparse markers, the 20 normal soybean linkage groups were broke into 45 linkage groups (data not shown) because of 25 large gaps with distances more than 50 cM. To increase the number of available polymorphic markers for bridging the gaps in the genetic map for the biparental cross of interest, the resequenced genomes of the two parental modern lines ZP and ZH were analyzed. A total of 2980 Mb and 5240 Mb raw sequence data consisting of sequence reads of 75 bp in length were obtained from ZP and ZH, respectively. After filtering out repetitive as well as unmapped sequences, the remaining 2700 Mb and 4470 Mb of low copy genomic sequences, respectively, were aligned to the soybean reference genome (variety Williams 82) pseudomolecules (<a href="http://www.phytozome.net">www.phytozome.net</a>) (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Schmutz <em>et al.</em> 2010</a>) for SNP discovery. The sequence data from ZP represented 87.5% of the reference genome with 2.82-fold coverage, whereas the sequence data from ZH represented 93.4% of the reference genome with 4.68-fold coverage. The ZP sequences exhibited a lower average rate (0.66%) of mismatches with the reference sequences than the ZH sequences (0.88%). After sequence alignment and SNP determination with regard to the reference sequence, 1,127,624 SNPs were identified discriminating between the genomes of ZP or ZH and the reference. Of them, 70.5% (794,876 SNPs) were detected between ZP and ZH with regard to the reference sequence, with an average of 39,744 SNPs in each chromosome (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>; <a href="http://www.ncbi.nlm.nih.gov/SNP/snp_viewTable.cgi?handle=">http://www.ncbi.nlm.nih.gov/SNP/snp_viewTable.cgi?handle=</a> NFCRI_ MOA_CAAS), with a distribution that differed significantly among the chromosomes. The highest number of SNPs (67,921) was on Gm18 and the lowest number (14,682) was on Gm04. Moreover, SNPs were unevenly distributed within chromosomes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>); 87.1% (692,085) of all SNPs had a distance of less than 1000 bp to its nearest neighboring SNP (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The 4.9% of the soybean reference genome containing multiple adjacent SNPs between the two parents were mostly located on chromosome arms.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464276/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464276/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Distribution pattern of 794,876 SNPs discovered by resequencing of ZP and ZH</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/553/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of SNPs discovered by resequencing between parental inbred lines ZH and ZP across 20 soybean reference chromosomes on the physical map. SNPs are displayed with green lines. The 313 SNPs used to construct a genetic linkage map are indicated by red arrows. Extended identical regions that are mostly void of SNPs are marked by blue bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2132676844" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of SNPs discovered by resequencing between parental inbred lines ZH and ZP across 20 soybean reference chromosomes on the physical map. SNPs are displayed with green lines. The 313 SNPs used to construct a genetic linkage map are indicated by red arrows. Extended identical regions that are mostly void of SNPs are marked by blue bars.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/553/F1.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/4/553/F1.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/553/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/553/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464245" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Distribution of SNPs discovered by resequencing between parental inbred lines ZH and ZP across 20 soybean reference chromosomes on the physical map. SNPs are displayed with green lines. The 313 SNPs used to construct a genetic linkage map are indicated by red arrows. Extended identical regions that are mostly void of SNPs are marked by blue bars.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/553/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the length between adjacent SNPs (A) and the frequency of synonymous and nonsynonymous SNPs on the chromosomes (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2132676844" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of the length between adjacent SNPs (A) and the frequency of synonymous and nonsynonymous SNPs on the chromosomes (B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/553/F2.medium.gif" width="440" height="146"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/4/553/F2.medium.gif" width="440" height="146"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/553/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/553/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464210" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Distribution of the length between adjacent SNPs (A) and the frequency of synonymous and nonsynonymous SNPs on the chromosomes (B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">With regard to functional annotation, most of the identified SNPs (84%) were located in intergenic regions. Among the 126,910 SNPs in genic regions, only 22.9% (28,650) were identified in coding sequences (CDS), including 13,156 synonymous SNPs and 15,494 nonsynonymous SNPs. With the exception of Gm09, the number of nonsynonymous SNPs was slightly higher than that of synonymous SNPs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) on 19 of the 20 chromosomes. In Gm09, the number of synonymous SNPs (470) was slightly higher than that of nonsynonymous SNPs (456).</p></div><div id="sec-10" class="subsection"><h3>Genetic linkage map construction</h3><p id="p-19">We selected 384 SNPs and designed an array using the Illumina GoldenGate assay system to fingerprint the segregating biparental population. From the 384 SNPs, 71 were excluded from analysis because of low data quality. The remaining 313 high-quality SNPs, including 195 nonsynonymous or nonsense, 44 synonymous, and 74 intergenic SNPs, were complemented by 167 SSRs, 4 EST-SSRs, and 1 InDel from the initial set mentioned above to construct a genetic linkage map (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009589/-/DC1">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009589/-/DC1/TableS1.xlsx">Table S1</a>). Of the 485 markers, a total of 464 could be mapped on the soybean reference genome (Glyma1.01) covering 880 Mb (92.6%) of the genome length. The average distance between two adjacent marker loci was 1.86 Mb.</p><p id="p-20">The resulting genetic linkage map with a total length of 2594.34 cM had an average marker spacing of 5.58 cM (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). All marker loci were assigned to one of the 20 soybean linkage groups by aligning the markers to the soybean reference genome sequence (Glyma1.01) (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Schmutz <em>et al.</em> 2010</a>) and the integrated soybean genetic map (Consensus Map 4.0) (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hyten <em>et al.</em> 2010</a>). The number of markers on Gm19 was highest with 33, followed by Gm18 and Gm 20 with 32. The lowest number of markers was detected for Gm08 with only 18 markers. The linear order of the 464 markers with information on their genomic locations was, in most cases, consistent with their order on the soybean reference genome, except for 59 markers showing deviations. Moreover, two SNP markers, Map-0090 on Gm01 and Map-0668 on Gm04, were assigned to Gm20 in the presented genetic map, cosegregating with Map-4067, Map-4081, and Map-4087, respectively. An additional marker, Map-0281 on Gm02, was assigned to Gm13.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/553/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Soybean genetic linkage map constructed using 485 molecular markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2132676844" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Soybean genetic linkage map constructed using 485 molecular markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/553/F3.medium.gif" width="440" height="239"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/4/553/F3.medium.gif" width="440" height="239"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/553/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/553/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464253" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Soybean genetic linkage map constructed using 485 molecular markers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">A total of 18 gaps of &gt;20 cM in length were present in 14 of the linkage groups, with exceptions Gm07(M), Gm08(A2), Gm10(O), Gm16(J), Gm17(D2), and Gm18(G). Most of these gaps (83.3%) were ≤30 cM in length. The largest gap was located on Gm06(C2) at 33.45 cM in length. The markers in all of the 18 gaps were sparse. A total of 48,411 SNPs were detected within the length of 127.51 Mb of 18 gaps, with a frequency of 1 SNP every 2.6-kp genomic region. This frequency is much lower than that of the other SNPs (1 SNP every 1.1-kb genomic region). Segregation distortion of each locus was estimated by the goodness of fit test, and 21 marker loci showed significant (<em>P</em> &lt; 0.01) segregation distortions. These loci were located on Gm07(M), Gm09(K), Gm10(O), Gm18(G), and Gm20(I). The distorted segregation markers were unevenly distributed in the genome. For example, two hotspots of regions with significant segregation distortion were detected on Gm18(G) and Gm10(O) and included eight markers, from Map-3441 to satt570, and four markers, from Map-1806 to satt173, respectively.</p></div><div id="sec-11" class="subsection"><h3>Landscape of fixed genomic regions and of recombinations</h3><p id="p-23">The rate of potential miscallings of SNPs by Sanger sequencing was determined to amount to 2.7% in soybean (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2013</a>). We examined the regions with low degrees of polymorphism identified in our study in more detail using this rate as an estimate of miscalling as threshold. Based on this criterion, 16 extended regions (&gt;5 Mb) were scored as identical between the two parental lines, which were according to the physical map located mainly in pericentromeric regions (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Within these regions, no SNPs were identified in seven large regions (≥1 Mb), including nucleotide positions 20,375,352–21,461,521 (1.09 Mb) on Gm01, 20,570,338–21,883,869 (1.31 Mb) and 26,841,519–27,919,452 (1.08 Mb) on Gm04, 23,354,052–24,387,140 (1.03 Mb), 30,670,807–31,966,253 (1.30 Mb), and 32,290,845–33,542,372 (1.25 Mb) on Gm06, and 21,025,993–22,030,212 (1.00 Mb) on Gm14.</p><p id="p-24">Investigating the association between genetic and physical distances of adjacent SNP markers revealed that the recombination frequency varied along the chromosome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The average genetic-to-physical distance ratio was 7.76 cM/Mb, with a range from 0 to 414.1 cM/Mb. Twenty hotspot regions with high frequencies of recombination (genetic-to-physical distance ratio &gt;20 cM/Mb) were identified on the chromosomal arms of Gm01 (1), Gm02 (1), Gm04 (1), Gm05 (3), Gm06 (1), Gm07 (2), Gm09 (1), Gm10 (2), Gm11 (1), Gm12 (2), Gm13 (1), Gm18 (3), and Gm20 (1).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/4/553/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of genetic recombination frequencies along the chromosomes in soybean genome. In each plot, the horizontal axis (in Mb) represents the physical location along the reference chromosomes according to the order of markers in the present genetic map and the vertical axis (cM/Mb) represents the genetic-to-physical distance ratio. The regions with large suppression of recombination (genetic-to-physical distance ratio 5 Mb) were marked by hollow arrows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2132676844" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of genetic recombination frequencies along the chromosomes in soybean genome. In each plot, the horizontal axis (in Mb) represents the physical location along the reference chromosomes according to the order of markers in the present genetic map and the vertical axis (cM/Mb) represents the genetic-to-physical distance ratio. The regions with large suppression of recombination (genetic-to-physical distance ratio &lt;2 cM/Mb and physical distance range &gt;5 Mb) were marked by hollow arrows.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/4/553/F4.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/4/553/F4.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/4/553/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/4/553/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464202" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Distribution of genetic recombination frequencies along the chromosomes in soybean genome. In each plot, the horizontal axis (in Mb) represents the physical location along the reference chromosomes according to the order of markers in the present genetic map and the vertical axis (cM/Mb) represents the genetic-to-physical distance ratio. The regions with large suppression of recombination (genetic-to-physical distance ratio &lt;2 cM/Mb and physical distance range &gt;5 Mb) were marked by hollow arrows.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We observed low recombination frequencies (genetic-to-physical distance ratio &lt;2 cM/Mb and physical distances &gt;5 Mb) in 14 regions of the genome. All were located in the pericentromeric regions of chromosomes. In addition, 13 pairs of markers that were separated in the soybean reference genome (from 9.3 kb to 12.3 Mb with an average of 1.85 Mb of physical distance) had no observable recombination events between them. All pairs were located in pericentromeric regions, with 10 of them being included in recombination coldspots, with the exception of Map-2639 in Gm14, Map-3033 in Gm16, and satt505 in Gm18.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-27">The traditional approach to select polymorphic PCR-based molecular markers in biparental populations is time-consuming. For instance, we tested 890 SSRs, mostly from the integrated linkage map of soybean (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Song <em>et al.</em> 2004</a>) for polymorphism between the parental inbred lines ZH and ZP. Only 19.2% of the SSRs, however, were polymorphic. To solve the issue of low polymorphism, we resequenced the genomes of ZH and ZP at 4.68-fold and 2.82-fold coverage respectively for SNP detection (<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2013</a>). Using Glyma1.0 (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Schmutz <em>et al.</em> 2010</a>) as a reference sequence, 794,876 SNPs were identified across the genome. Consequently, resequencing closely related parents is a promising avenue to provide polymorphic markers for constructing high-density genetic map and fine-mapping QTL.</p><p id="p-28">In addition, resequencing analysis provides us an opportunity to take a closer look at the landscape of polymorphisms among the two parental lines. Our analysis revealed that there are several regions with a low degree of polymorphisms between the two parents (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), which makes it difficult to screen for polymorphic markers in these areas. The potential miscalling of SNPs was reported previously to amount to 2.7% of misclassifications based on Sanger sequencing results (<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2013</a>). Therefore, it is tempting to speculate that several of these large regions with a degree of polymorphisms below the misclassification rate might be identical by descent. This information is of particular relevance when using multiple-line crosses for QTL mapping, because it allows to estimate the maximum number of alleles segregating among parents (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jannink and Wu 2003</a>).</p><p id="p-29">Using SNPs developed from resequencing of parental lines offers the potential to develop customized SNP arrays optimized for the targeted germplasm. We tested this option and developed 384 stable SNPs from the 794,876 SNPs to genotype the 254 RILs available. High-quality data were generated for 81.5% of the 384 SNP loci, which is comparable to that of previous studies (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hyten <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2010</a>). The genetic linkage map encompassed 2594.34 cM, which is consistent with the third and fourth versions of the soybean integrated genetic linkage map as well as a high-density SSR integrated genetic linkage map (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hwang <em>et al.</em> 2009</a>). The number of intervals void of markers &gt;20 cM in the current map was lower as compared to that in the map based on 412 SSRs constructed for the “Minsoy” × “Noir 1” mapping population (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cregan <em>et al.</em> 1999</a>). Consequently, this finding clearly underlines the potential to use resequencing data to design customized SNP arrays for QTL mapping.</p><p id="p-30">Variation of recombination rates plays a key role in shaping genomic features and influencing the strength of natural selection (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gaut <em>et al.</em> 2007</a>). The distribution of the genetic recombination frequency across the genome was analyzed based on the comparison of physical and genetic distances (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). We observed that the pericentromeric regions are coldspots of recombinations similar to that reported for rice (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> 2002</a>), maize (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Carels <em>et al.</em> 1995</a>), wheat (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Erayman <em>et al.</em> 2004</a>), and soybean (<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Schmutz <em>et al.</em> 2010</a>). The pericentromeric regions in soybean are gene-poor and recombinationally inactive, and most genes are clustered in the chromosome arms (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Du <em>et al.</em> 2012</a>). Nevertheless, the centromeric sequences within the pericentromeric regions are key factors for cell division and stable transmission of genetic information. Several active genes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nagaki <em>et al.</em> 2004</a>) and a series of QTL or genes related to important agronomy traits such as grain weight in rice have been identified in pericentromeric regions (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2004</a>). The severely suppressed recombination resulted in sharply reduced diversity in these regions and might have contributed to the formation of extended regions of sequence identity between the two parents. Despite the need to maintain conserved regions for stable cell division, widening the bottleneck of reduced numbers of recombinations in the centromeric regions might be an important aspect of breaking the current yield barriers in soybean breeding.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">We thank Prof. Dr. Scott Allen Jackson (Center for Applied Genetic Technologies, University of Georgia, Athens, GA) for critical reading and useful suggestions. This research was supported by the National Natural Science Foundation of China (no. 31171575 and 31271753) and the State High-tech Program (no. 2012AA021602). L. Qiu, Y. Li, J. Reif, and R. Chang designed the research. Y. Li, Y. Liu, Z. Liu, and B. Liu performed the research. Y. Li and Y. Liu analyzed the data. Y. Li, Y. Liu, J. Reif, M. Mette, and L. Qiu wrote the manuscript. All authors read and approved the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009589/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009589/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-32"><em>Communicating editor: S. A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-11-24"><span class="received-label">Received </span>November 24, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-01-24"><span class="accepted-label">Accepted </span>January 24, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Li <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buetow</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Multipoint gene mapping using seriation. I. General methods.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">180</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBuetow%26rft.auinit1%253DK.%2BH.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D180%26rft.epage%253D188%26rft.atitle%253DMultipoint%2Bgene%2Bmapping%2Busing%2Bseriation.%2BI.%2BGeneral%2Bmethods.%26rft_id%253Dinfo%253Apmid%252F3475978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3475978&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J757500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.2" data-doi="10.1073/pnas.92.24.11057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carels</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barakat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The gene distribution of the maize genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">11057</span>–<span class="cit-lpage">11060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCarels%26rft.auinit1%253DN.%26rft.volume%253D92%26rft.issue%253D24%26rft.spage%253D11057%26rft.epage%253D11060%26rft.atitle%253DThe%2Bgene%2Bdistribution%2Bof%2Bthe%2Bmaize%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.24.11057%26rft_id%253Dinfo%253Apmid%252F7479936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTIvMjQvMTEwNTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.3" data-doi="10.1105/tpc.010485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Presting</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goicoechea</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackmon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An integrated physical and genetic map of the rice genome.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DChen%26rft.auinit1%253DM.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D537%26rft.epage%253D545%26rft.atitle%253DAn%2BIntegrated%2BPhysical%2Band%2BGenetic%2BMap%2Bof%2Bthe%2BRice%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.010485%26rft_id%253Dinfo%253Apmid%252F11910002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE0LzMvNTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.4" data-doi="10.1534/genetics.107.070821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaky</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070821%26rft_id%253Dinfo%253Apmid%252F17339218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzEvNjg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.5" data-doi="10.2135/cropsci1999.3951464x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvik</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bush</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lark</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">An integrated genetic linkage map of the soybean genome.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1464</span>–<span class="cit-lpage">1490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCregan%26rft.auinit1%253DP.B.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D1464%26rft.epage%253D1490%26rft.atitle%253DAn%2BIntegrated%2BGenetic%2BLinkage%2BMap%2Bof%2Bthe%2BSoybean%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.3951464x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.3951464x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000082813800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.6" data-doi="10.1105/tpc.111.092759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sui</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Pericentromeric effects shape the patterns of divergence, retention, and expression of duplicated genes in the paleopolyploid soybean.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.111.092759%26rft_id%253Dinfo%253Apmid%252F22227891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjI0LzEvMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.7" data-doi="10.1093/nar/gkh639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erayman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dilbirligi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baenziger</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Demarcating the gene-rich regions of the wheat genome.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">3546</span>–<span class="cit-lpage">3565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DErayman%26rft.auinit1%253DM.%26rft.volume%253D32%26rft.issue%253D12%26rft.spage%253D3546%26rft.epage%253D3565%26rft.atitle%253DDemarcating%2Bthe%2Bgene-rich%2Bregions%2Bof%2Bthe%2Bwheat%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh639%26rft_id%253Dinfo%253Apmid%252F15240829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xMi8zNTQ2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.8" data-doi="10.1038/nrg1970"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rizzon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination: an underappreciated factor in the evolution of plant genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGaut%26rft.auinit1%253DB.%2BS.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D84%26rft.atitle%253DRecombination%253A%2Ban%2Bunderappreciated%2Bfactor%2Bin%2Bthe%2Bevolution%2Bof%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1970%26rft_id%253Dinfo%253Apmid%252F17173059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1970&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17173059&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242934300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.9" data-doi="10.2135/cropsci1994.0011183X003400050001x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gizlice</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic base for North American public soybean cultivars released between 1947 and 1988.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DGizlice%26rft.auinit1%253DZ.%26rft.volume%253D34%26rft.issue%253D5%26rft.spage%253D1143%26rft.epage%253D1151%26rft.atitle%253DGenetic%2BBase%2Bfor%2BNorth%2BAmerican%2BPublic%2BSoybean%2BCultivars%2BReleased%2Bbetween%2B1947%2Band%2B1988%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400050001x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400050001x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PF01600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.10" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC81NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.11" data-doi="10.1038/ng.695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies of 14 agronomic traits in rice landraces.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D961%26rft.epage%253D967%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bof%2B14%2Bagronomic%2Btraits%2Bin%2Brice%2Blandraces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.695%26rft_id%253Dinfo%253Apmid%252F20972439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972439&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283540500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.12" data-doi="10.1093/dnares/dsp010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-density integrated linkage map based on SSR markers in soybean.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsp010%26rft_id%253Dinfo%253Apmid%252F19531560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjE2LzQvMjEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC80LzU1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.13" data-doi="10.2135/cropsci2009.06.0360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. E.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A High density integrated genetic linkage map of soybean and the development of a 1536 universal soy linkage panel for quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHyten%26rft.auinit1%253DD.%2BL.%26rft.volume%253D50%26rft.issue%253D3%26rft.spage%253D960%26rft.epage%253D968%26rft.atitle%253DA%2BHigh%2BDensity%2BIntegrated%2BGenetic%2BLinkage%2BMap%2Bof%2BSoybean%2Band%2Bthe%2BDevelopment%2Bof%2Ba%2B1536%2BUniversal%2BSoy%2BLinkage%2BPanel%2Bfor%2BQuantitative%2BTrait%2BLocus%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.06.0360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.06.0360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000276961900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.14" data-doi="10.1007/s00122-008-0726-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-throughput genotyping with the GoldenGate assay in the complex genome of soybean.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHyten%26rft.auinit1%253DD.%2BL.%26rft.volume%253D116%26rft.issue%253D7%26rft.spage%253D945%26rft.epage%253D952%26rft.atitle%253DHigh-throughput%2Bgenotyping%2Bwith%2Bthe%2BGoldenGate%2Bassay%2Bin%2Bthe%2Bcomplex%2Bgenome%2Bof%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0726-2%26rft_id%253Dinfo%253Apmid%252F18278477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0726-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18278477&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254907200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.15" data-doi="10.1017/S0016672303006153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating allelic number and identity in state of QTLs in interconnected families.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D81%26rft.issue%253D2%26rft.spage%253D133%26rft.epage%253D144%26rft.atitle%253DEstimating%2Ballelic%2Bnumber%2Band%2Bidentity%2Bin%2Bstate%2Bof%2BQTLs%2Bin%2Binterconnected%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672303006153%26rft_id%253Dinfo%253Apmid%252F12872915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672303006153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12872915&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183477600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.16" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">The estimation of map distance from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.17" data-doi="10.1038/ng.715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLam%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D12%26rft.spage%253D1053%26rft.epage%253D1059%26rft.atitle%253DResequencing%2Bof%2B31%2Bwild%2Band%2Bcultivated%2Bsoybean%2Bgenomes%2Bidentifies%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Band%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.715%26rft_id%253Dinfo%253Apmid%252F21076406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076406&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.18" data-doi="10.1007/s00122-007-0663-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Inclusive composite interval mapping (ICIM) for digenic epistasis of quantitative traits in biparental populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D116%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D260%26rft.atitle%253DInclusive%2Bcomposite%2Binterval%2Bmapping%2B%2528ICIM%2529%2Bfor%2Bdigenic%2Bepistasis%2Bof%2Bquantitative%2Btraits%2Bin%2Bbiparental%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0663-5%26rft_id%253Dinfo%253Apmid%252F17985112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0663-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17985112&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.19" data-doi="10.1534/genetics.104.034165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Fine mapping of a grain-weight quantitative trait locus in the pericentromeric region of rice chromosome 3.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">2187</span>–<span class="cit-lpage">2195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLi%26rft.auinit1%253DJ.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D2187%26rft.epage%253D2195%26rft.atitle%253DFine%2BMapping%2Bof%2Ba%2BGrain-Weight%2BQuantitative%2BTrait%2BLocus%2Bin%2Bthe%2BPericentromeric%2BRegion%2Bof%2BRice%2BChromosome%2B3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.034165%26rft_id%253Dinfo%253Apmid%252F15611185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzIxODciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.20" data-doi="10.1093/bioinformatics/btp336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yiu</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SOAP2: an improved ultrafast tool for short read alignment.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1966</span>–<span class="cit-lpage">1967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp336%26rft_id%253Dinfo%253Apmid%252F19497933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTUvMTk2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvNC81NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.21" data-doi="10.1111/j.1469-8137.2010.03344.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">R. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic diversity in domesticated soybean (<em>Glycine ma</em>x) and its wild progenitor (<em>Glycine soja</em>) for simple sequence repeat and single-nucleotide polymorphism loci.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DLi%26rft.auinit1%253DY.%2BH.%26rft.volume%253D188%26rft.issue%253D1%26rft.spage%253D242%26rft.epage%253D253%26rft.atitle%253DGenetic%2Bdiversity%2Bin%2Bdomesticated%2Bsoybean%2B%2528Glycine%2Bmax%2529%2Band%2Bits%2Bwild%2Bprogenitor%2B%2528Glycine%2Bsoja%2529%2Bfor%2Bsimple%2Bsequence%2Brepeat%2Band%2Bsingle-nucleotide%2Bpolymorphism%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03344.x%26rft_id%253Dinfo%253Apmid%252F20618914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03344.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20618914&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281551500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.22" data-doi="10.1186/1471-2164-14-579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Molecular footprints of domestication and improvement in soybean revealed by whole genome re-sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-579%26rft_id%253Dinfo%253Apmid%252F23984715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23984715&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.23" data-doi="10.1016/S1671-2927(09)60233-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uzokwe</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">R. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Development of soybean EST-SSR markers and their use to assess genetic diversity in the Subgenus <em>Soja</em>.</span> <abbr class="cit-jnl-abbrev">Agric. Sci. China</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1423</span>–<span class="cit-lpage">1429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgric.%2BSci.%2BChina%26rft.volume%253D9%26rft.spage%253D1423%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1671-2927%252809%252960233-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1671-2927(09)60233-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.24" data-doi="10.1038/ng1289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagaki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouyang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sequencing of a rice centromere uncovers active genes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">138</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNagaki%26rft.auinit1%253DK.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D138%26rft.epage%253D145%26rft.atitle%253DSequencing%2Bof%2Ba%2Brice%2Bcentromere%2Buncovers%2Bactive%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1289%26rft_id%253Dinfo%253Apmid%252F14716315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14716315&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188542700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nan</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">R. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Development and identification of InDel markers based on <em>rhg1</em> gene for resistance to soybean cyst nematode (<em>Heterodera glycines</em> Ichinohe)</span>. <abbr class="cit-jnl-abbrev">Acta Agron. Sin.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgron.%2BSin.%26rft.volume%253D35%26rft.spage%253D1236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.26" data-doi="10.1046/j.1365-2540.2000.00678.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Optimal marker density for interval mapping in a backcross population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D84%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2000.00678.x%26rft_id%253Dinfo%253Apmid%252F10849067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2000.00678.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10849067&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.27" data-doi="10.1038/nature08670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlueter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the palaeopolyploid soybean.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchmutz%26rft.auinit1%253DJ.%26rft.volume%253D463%26rft.issue%253D7278%26rft.spage%253D178%26rft.epage%253D183%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpalaeopolyploid%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08670%26rft_id%253Dinfo%253Apmid%252F20075913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20075913&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273582700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.28" data-doi="10.1007/s00122-004-1602-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marek</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lark</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Concibido</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A new integrated genetic linkage map of the soybean.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSong%26rft.auinit1%253DQ.%2BJ.%26rft.volume%253D109%26rft.issue%253D1%26rft.spage%253D122%26rft.epage%253D128%26rft.atitle%253DA%2Bnew%2Bintegrated%2Bgenetic%2Blinkage%2Bmap%2Bof%2Bthe%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1602-3%26rft_id%253Dinfo%253Apmid%252F14991109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1602-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14991109&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221936300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.29" data-doi="10.2135/cropsci2009.10.0607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Abundance of SSR motifs and development of candidate polymorphic SSR markers (BARCSOYSSR_1. 0) in soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1950</span>–<span class="cit-lpage">1960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1950%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.10.0607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.10.0607&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.30" data-doi="10.1007/s00122-013-2166-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2699</span>–<span class="cit-lpage">2716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2699%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2166-x%26rft_id%253Dinfo%253Apmid%252F23918062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2166-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23918062&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F4%2F553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.31" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzQvNTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">R. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inducing expression analysis of <em>Gmhs1<sup>pro-1</sup></em> by inoculating soybean cyst nematode 4 race.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biotechnol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiotechnol.%26rft.volume%253D19%26rft.spage%253D77%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.4.553.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gai</span>  <span class="cit-name-given-names">J. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The core ancestors of soybean cultivars released during 1923–2005 in China.</span> <abbr class="cit-jnl-abbrev">Soybean Sci</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoybean%2BSci%26rft.volume%253D26%26rft.spage%253D641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/4/553.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/547" title="gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/561" title="Comparative Genomic and Transcriptomic Analysis of Wangiella dermatitidis, A Major Cause of Phaeohyphomycosis and a Model Black Yeast Human Pathogen" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/4"><span>Volume 4 Issue 4, April 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/4.cover-source.jpg?itok=Rf0YhETZ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/4/553.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VPhJOx7_Pl_T1pAiUY1dEAzYH2c-pz2pDCNyB3zOvqw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/4/553" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/4/553" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/4/553" class="active" data-icon-position="" data-hide-link-title="0">Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459851" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3EnN4haF41WHLgKcG_lT5VjOtxV0TjnbZYfiQq4kzeE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/553.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/4/553" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-UhUAO5-V-MvdrVangiA4j0mGQmCEdA9Ws8nns-hfTp0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/4/553" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/4/553.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459851" id="node459851--2" data-pisa="ggg;4/4/553" data-pisa-master="ggg;g3.113.009589" data-apath="/ggg/4/4/553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying-hui Li</span>, <span class="highwire-citation-author" data-delta="1">Yu-lin Liu</span>, <span class="highwire-citation-author" data-delta="2">Jochen C. Reif</span>, <span class="highwire-citation-author" data-delta="3">Zhang-xiong Liu</span>, <span class="highwire-citation-author" data-delta="4">Bo Liu</span>, <span class="highwire-citation-author" data-delta="5">Michael F. Mette</span>, <span class="highwire-citation-author" data-delta="6">Ru-zhen Chang</span> and <span class="highwire-citation-author" data-delta="7">Li-juan Qiu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">553-560;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009589 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459851--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying-hui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying-hui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing-hui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yu-lin Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yu-lin%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYu-lin%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cytogenetics and Genome Analysis, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Corrensstrasse 3, D-06466 Stadt Seeland, Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhang-xiong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhang-xiong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhang-xiong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bo Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael F. Mette </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cytogenetics and Genome Analysis, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Corrensstrasse 3, D-06466 Stadt Seeland, Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BMette%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mette%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BMette%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ru-zhen Chang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ru-zhen%2BChang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARu-zhen%2BChang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Li-juan Qiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li-juan%2BQiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qiu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi-juan%2BQiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:qiulijuan@caas.cn" class="" data-icon-position="" data-hide-link-title="0">qiulijuan@caas.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/4/553" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/4/553&amp;title=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/4/553&amp;title=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/4/553&amp;title=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/4/553&amp;text=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/4/553&amp;title=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/4/553&amp;t=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/4/553&amp;title=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/4/553&amp;title=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459851/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459851" id="node459851--3" data-pisa="ggg;4/4/553" data-pisa-master="ggg;g3.113.009589" data-apath="/ggg/4/4/553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ying-hui Li</span>, <span class="highwire-citation-author" data-delta="1">Yu-lin Liu</span>, <span class="highwire-citation-author" data-delta="2">Jochen C. Reif</span>, <span class="highwire-citation-author" data-delta="3">Zhang-xiong Liu</span>, <span class="highwire-citation-author" data-delta="4">Bo Liu</span>, <span class="highwire-citation-author" data-delta="5">Michael F. Mette</span>, <span class="highwire-citation-author" data-delta="6">Ru-zhen Chang</span> and <span class="highwire-citation-author" data-delta="7">Li-juan Qiu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">553-560;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009589 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459851--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ying-hui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying-hui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing-hui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yu-lin Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yu-lin%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYu-lin%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cytogenetics and Genome Analysis, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Corrensstrasse 3, D-06466 Stadt Seeland, Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhang-xiong Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhang-xiong%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhang-xiong%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bo Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael F. Mette </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Cytogenetics and Genome Analysis, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Corrensstrasse 3, D-06466 Stadt Seeland, Gatersleben, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BMette%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mette%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BMette%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ru-zhen Chang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ru-zhen%2BChang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARu-zhen%2BChang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Li-juan Qiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The National Key Facility for Crop Gene Resources and Genetic Improvement (NFCRI)/Key Lab of Germplasm Utilization (MOA), Institute of Crop Science, Chinese Academy of Agricultural Sciences, 100081 Beijing, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li-juan%2BQiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qiu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi-juan%2BQiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:qiulijuan@caas.cn" class="" data-icon-position="" data-hide-link-title="0">qiulijuan@caas.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=553&amp;rft.epage=560&amp;rft.atitle=Biparental%20Resequencing%20Coupled%20With%20SNP%20Genotyping%20of%20a%20Segregating%20Population%20Offers%20Insights%20Into%20the%20Landscape%20of%20Recombination%20and%20Fixed%20Genomic%20Regions%20in%20Elite%20Soybean&amp;rft.volume=4&amp;rft.issue=4&amp;rft.date=2014-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Li%2C+Ying-hui&amp;rft.au=Liu%2C+Yu-lin&amp;rft.au=Reif%2C+Jochen+C.&amp;rft.au=Liu%2C+Zhang-xiong&amp;rft.au=Liu%2C+Bo&amp;rft.au=Mette%2C+Michael+F.&amp;rft.au=Chang%2C+Ru-zhen&amp;rft.au=Qiu%2C+Li-juan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459851/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459851/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459851/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459851/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459851/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459851/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459851/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459851/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459851/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459851/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459851/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459851/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/4/553" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/4/553.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/4/553.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/4/553.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/4/553.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
