<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/461" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/461.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/461.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/461.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/2/461/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/461.atom" />
<meta name="HW.pisa" content="ggg;8/2/461" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (Medicago sativa)" />
<meta name="DC.Identifier" content="10.1534/g3.117.300099" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Munjal et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Autumn dormancy in alfalfa ( Medicago sativa ) is associated with agronomically important traits including regrowth rate, maturity, and winter survival. Historical recurrent selection experiments have been able to manipulate the dormancy response. We hypothesized that artificial selection for dormancy phenotypes in these experiments had altered allele frequencies of dormancy-related genes. Here, we follow this hypothesis and analyze allele frequency changes using genome-wide polymorphisms in the pre- and postselection populations from one historical selection experiment. We screened the nondormant cultivar CUF 101 and populations developed by three cycles of recurrent phenotypic selection for taller and shorter plants in autumn with markers derived from genotyping-by-sequencing (GBS). We validated the robustness of our GBS-derived allele frequency estimates using an empirical approach. Our results suggest that selection mapping is a powerful means of identifying genomic regions associated with traits, and that it can be exploited to provide regions on which to focus further mapping and cloning projects." />
<meta name="DC.Contributor" content="Gitanshu Munjal" />
<meta name="DC.Contributor" content="Jingjie Hao" />
<meta name="DC.Contributor" content="Larry R. Teuber" />
<meta name="DC.Contributor" content="E. Charles Brummer" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (Medicago sativa)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Autumn dormancy in alfalfa (&lt;i&gt;Medicago sativa&lt;/i&gt;) is associated with agronomically important traits including regrowth rate, maturity, and winter survival. Historical recurrent selection experiments have been able to manipulate the dormancy response. We hypothesized that artificial selection for dormancy phenotypes in these experiments had altered allele frequencies of dormancy-related genes. Here, we follow this hypothesis and analyze allele frequency changes using genome-wide polymorphisms in the pre- and postselection populations from one historical selection experiment. We screened the nondormant cultivar CUF 101 and populations developed by three cycles of recurrent phenotypic selection for taller and shorter plants in autumn with markers derived from genotyping-by-sequencing (GBS). We validated the robustness of our GBS-derived allele frequency estimates using an empirical approach. Our results suggest that selection mapping is a powerful means of identifying genomic regions associated with traits, and that it can be exploited to provide regions on which to focus further mapping and cloning projects.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/461" />
<meta name="citation_id" content="8/2/461" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/461" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/461.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/461.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/461.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300099" />
<meta name="citation_pmid" content="29255116" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="461" />
<meta name="citation_lastpage" content="468" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gitanshu Munjal" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California-Davis, California 95616" />
<meta name="citation_author_institution" content="Plant Breeding Center, University of California-Davis, California 95616" />
<meta name="citation_author" content="Jingjie Hao" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California-Davis, California 95616" />
<meta name="citation_author_institution" content="Plant Breeding Center, University of California-Davis, California 95616" />
<meta name="citation_author" content="Larry R. Teuber" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California-Davis, California 95616" />
<meta name="citation_author" content="E. Charles Brummer" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California-Davis, California 95616" />
<meta name="citation_author_institution" content="Plant Breeding Center, University of California-Davis, California 95616" />
<meta name="citation_author_email" content="ecbrummer@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3621-4516" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Arnold;citation_author=K. Bomblies;citation_author=J. Wakeley;citation_title=Extending coalescent theory to autotetraploids.;citation_volume=192;citation_year=2012;citation_pmid=22714411;citation_doi=10.1534/genetics.112.140582" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=B. Vaillancourt;citation_author=S. Deshpande;citation_author=K. Barry;citation_title=A genome-wide scan for evidence of selection in a maize population under long-term artificial selection for ear number.;citation_volume=196;citation_year=2014;citation_pmid=24381334;citation_doi=10.1534/genetics.113.160655" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. Bingham;citation_author=T. McCoy;citation_title=Cultivated Alfalfa at the Diploid Level: Origin, Reproductive Stability, and Yield of Seed and Forage;citation_pages=97-100;citation_volume=19;citation_year=1979;citation_issue=1;citation_doi=10.2135/cropsci1979.0011183X001900010024x" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=F. Blondon;citation_author=D. Marie;citation_author=S. Brown;citation_author=A. Kondorosi;citation_title=Genome size and base composition in Medicago sativa and M. truncatula species.;citation_pages=264-270;citation_volume=37;citation_year=1994;citation_issue=2;citation_pmid=18470076" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Boudet;citation_author=J. Buitink;citation_author=F. A. Hoekstra;citation_author=H. Rogniaux;citation_author=C. Larré;citation_title=Comparative analysis of the heat stable proteome of radicles of Medicago truncatula seeds during germination identifies late embryogenesis abundant proteins associated with desiccation tolerance.;citation_volume=140;citation_year=2006;citation_pmid=16461389;citation_doi=10.1104/pp.105.074039" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. J. Brouwer;citation_author=S. H. Duke;citation_author=T. C. Osborn;citation_title=Mapping Genetic Factors Associated with Winter Hardiness, Fall Growth, and Freezing Injury in Autotetraploid Alfalfa;citation_pages=1387-1396;citation_volume=40;citation_year=2000;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. C. Brummer;citation_author=M. M. Shah;citation_author=D. Luth;citation_title=Reexamining the Relationship between Fall Dormancy and Winter Hardiness in Alfalfa;citation_pages=971-977;citation_volume=40;citation_year=2000;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=T. Busbice;citation_author=C. Wilsie;citation_title=Fall Growth, Winterhardiness, Recovery after Cutting and Wilt Resistance in F2 Progenies of Vernal x DuPuits Alfalfa Crosses;citation_pages=429-432;citation_volume=5;citation_year=1965;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Byrne;citation_author=A. Czaban;citation_author=B. Studer;citation_author=F. Panitz;citation_author=C. Bendixen;citation_title=Genome wide allele frequency fingerprints (gwaffs) of populations via genotyping by sequencing.;citation_pages=e57438;citation_volume=8;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=Y. Castonguay;citation_author=S. Laberge;citation_author=E. C. Brummer;citation_author=J. J. Volenec;citation_title=Alfalfa winter hardiness: a research retrospective and integrated perspective.;citation_pages=203-265;citation_volume=90;citation_year=2006;citation_doi=10.1016/S0065-2113(06)90006-6" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. Cunningham;citation_author=J. Volenec;citation_author=L. Teuber;citation_title=Plant Survival and Root and Bud Composition of Alfalfa Populations Selected for Contrasting Fall Dormancy;citation_pages=962-969;citation_volume=38;citation_year=1998;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=H. Daday;citation_author=C. G. Greenham;citation_title=GENETIC STUDIES ON COLD HARDINESS IN MEDICAGO SATIVA L.;citation_pages=249-255;citation_volume=51;citation_year=1960;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Duforet-Frebourg;citation_author=K. Luu;citation_author=G. Laval;citation_author=E. Bazin;citation_author=M. G. Blum;citation_title=Detecting genomic signatures of natural selection with principal component analysis: application to the 1000 genomes data.;citation_pages=1082-1093;citation_volume=33;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Gautier;citation_author=J. Foucaud;citation_author=K. Gharbi;citation_author=T. Cézard;citation_author=M. Galan;citation_title=Estimation of population allele frequencies from next-generation sequencing data: pool-vs. individual-based genotyping.;citation_pages=3766-3779;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12360" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. Goddard;citation_author=B. Hayes;citation_author=T. Meuwissen;citation_title=Using the genomic relationship matrix to predict the accuracy of genomic selection.;citation_pages=409-421;citation_volume=128;citation_year=2011;citation_issue=6;citation_pmid=22059574;citation_doi=10.1111/j.1439-0388.2011.00964.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. Gutiérrez-Gil;citation_author=J. J. Arranz;citation_author=R. Pong-Wong;citation_author=E. García-Gámez;citation_author=J. Kijas;citation_title=Application of selection mapping to identify genomic regions associated with dairy production in sheep.;citation_pages=e94623;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Jónás;citation_author=T. Taus;citation_author=C. Kosiol;citation_author=C. Schlötterer;citation_author=A. Futschik;citation_title=Estimating the effective population size from temporal allele frequency changes in experimental evolution.;citation_volume=204;citation_year=2016;citation_pmid=27542959;citation_doi=10.1534/genetics.116.191197" />
<meta name="citation_reference" content="citation_journal_title=Stochastic Process. Appl.;citation_author=J. F. C. Kingman;citation_title=The coalescent.;citation_pages=235-248;citation_volume=13;citation_year=1982;citation_doi=10.1016/0304-4149(82)90011-4" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=W. Lehman;citation_author=M. Nielson;citation_author=V. Marble;citation_author=E. Stanford;citation_title=Registration of Cuf 101 Alfalfa (Reg. No. 119);citation_pages=398-398;citation_volume=23;citation_year=1983;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Soil Sci.;citation_author=J. Levitt;citation_title=The hardiness of plants.;citation_pages=346;citation_volume=82;citation_year=1956" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Lewontin;citation_author=J. Krakauer;citation_title=DISTRIBUTION OF GENE FREQUENCY AS A TEST OF THE THEORY OF THE SELECTIVE NEUTRALITY OF POLYMORPHISMS;citation_pages=175-195;citation_volume=74;citation_year=1973;citation_issue=1;citation_pmid=4711903" />
<meta name="citation_reference" content="citation_title=arXiv;citation_title=Aligning sequence reads, clone sequences and assembly contigs with bwa-mem.;citation_author=H. Li;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=X. Li;citation_author=Y. Wei;citation_author=A. Acharya;citation_author=Q. Jiang;citation_author=J. Kang;citation_title=A saturated genetic linkage map of autotetraploid alfalfa (Medicago sativa L.) developed using genotyping-by-sequencing is highly syntenous with the Medicago truncatula genome.;citation_pages=1971-1979;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=X. Li;citation_author=B. Alarcón-Zúñiga;citation_author=J. Kang;citation_author=N. Tahir;citation_author=M. Hammad;citation_title=Mapping fall dormancy and winter injury in tetraploid alfalfa.;citation_pages=1995-2011;citation_volume=55;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=K. Luu;citation_author=E. Bazin;citation_author=M. G. Blum;citation_title=pcadapt: an r package to perform genome scans for selection based on principal component analysis.;citation_pages=67-77;citation_volume=17;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. Lynch;citation_author=D. Bost;citation_author=S. Wilson;citation_author=T. Maruki;citation_author=S. Harrison;citation_title=Population-genetic inference from pooled-sequencing data.;citation_volume=6;citation_year=2014;citation_pmid=24787620;citation_doi=10.1093/gbe/evu085" />
<meta name="citation_reference" content="citation_title=Alfalfa and Alfalfa Improvement.;citation_title=Cold and heat tolerance;citation_author=J. McKenzie;citation_author=R. Paquin;citation_author=S. H. Duke;citation_pages=259-302;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=A. T. Melo;citation_author=R. Bartaula;citation_author=I. Hale;citation_title=Gbs-snp-crop: a reference-optional pipeline for snp discovery and plant germplasm characterization using variable length, paired-end genotyping-by-sequencing data.;citation_pages=1;citation_volume=17;citation_year=2016;citation_pmid=26817711;citation_doi=10.1186/s12859-015-0844-1" />
<meta name="citation_reference" content="citation_title=An Introduction to Population Genetics: Theory and Applications.;citation_author=R. Nielsen;citation_author=M. Slatkin;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Plant Desiccation Tolerance.;citation_title=Lessons on dehydration tolerance from desiccation tolerant plants;citation_author=M. J. Oliver;citation_pages=11-50;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Patterson;citation_author=A. L. Price;citation_author=D. Reich;citation_title=Population structure and eigenanalysis.;citation_pages=e190-e190;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17194218;citation_doi=10.1371/journal.pgen.0020190" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Rellstab;citation_author=S. Zoller;citation_author=A. Tedder;citation_author=F. Gugerli;citation_author=M. C. Fischer;citation_title=Validation of snp allele frequencies determined by pooled next-generation sequencing in natural populations of a non-model plant species.;citation_pages=e80422;citation_volume=8;citation_year=2013;citation_pmid=24244686;citation_doi=10.1371/journal.pone.0080422" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=T. Rognes;citation_author=T. Flouri;citation_author=B. Nichols;citation_author=C. Quince;citation_author=F. Mahé;citation_title=Vsearch: a versatile open source tool for metagenomics.;citation_pages=e2584;citation_volume=4;citation_year=2016;citation_pmid=27781170;citation_doi=10.7717/peerj.2584" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Sakiroglu;citation_author=S. Sherman-Broyles;citation_author=A. Story;citation_author=K. J. Moore;citation_author=J. J. Doyle;citation_title=Patterns of linkage disequilibrium and association mapping in diploid alfalfa (M. sativa L.).;citation_pages=577-590;citation_volume=125;citation_year=2012;citation_issue=3;citation_pmid=22476875;citation_doi=10.1007/s00122-012-1854-2" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. Schlötterer;citation_author=R. Kofler;citation_author=E. Versace;citation_author=R. Tobler;citation_author=S. Franssen;citation_title=Combining experimental evolution with next-generation sequencing: a powerful tool to study adaptation from standing genetic variation.;citation_pages=431-440;citation_volume=114;citation_year=2015;citation_pmid=25269380;citation_doi=10.1038/hdy.2014.86" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. Schwab;citation_author=D. Barnes;citation_author=C. Sheaffer;citation_title=The Relationship between Field Winter Injury and Fall Growth Score for 251 Alfalfa Cultivars;citation_pages=418-426;citation_volume=36;citation_year=1996;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Sci.;citation_author=D. Smith;citation_title=Association of fall growth habit and winter survival in alfalfa.;citation_pages=244-251;citation_volume=41;citation_year=1961" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. M. Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=1;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. Tang;citation_author=V. Krishnakumar;citation_author=S. Bidwell;citation_author=B. Rosen;citation_author=A. Chan;citation_title=An improved genome release (version mt4. 0) for the model legume Medicago truncatula.;citation_pages=1;citation_volume=15;citation_year=2014;citation_pmid=24382143;citation_doi=10.1186/1471-2164-15-349" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=M. F. Thomashow;citation_title=PLANT COLD ACCLIMATION: Freezing Tolerance Genes and Regulatory Mechanisms;citation_pages=571-599;citation_volume=50;citation_year=1999;citation_issue=1;citation_doi=10.1146/annurev.arplant.50.1.571" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Vaysse;citation_author=A. Ratnakumar;citation_author=T. Derrien;citation_author=E. Axelsson;citation_author=G. R. Pielberg;citation_title=Identification of genomic regions associated with phenotypic variation between dog breeds using selection mapping.;citation_pages=e1002316-e1002316;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22022279;citation_doi=10.1371/journal.pgen.1002316" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Verdier;citation_author=D. Lalanne;citation_author=S. Pelletier;citation_author=I. Torres-Jerez;citation_author=K. Righetti;citation_title=A regulatory network-based approach dissects late maturation processes related to the acquisition of desiccation tolerance and longevity of Medicago truncatula seeds.;citation_volume=163;citation_year=2013;citation_pmid=23929721;citation_doi=10.1104/pp.113.222380" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. A. Weishaar;citation_author=E. C. Brummer;citation_author=J. J. Volenec;citation_author=K. J. Moore;citation_author=S. Cunningham;citation_title=Improving Winter Hardiness in Nondormant Alfalfa Germplasm;citation_pages=60-65;citation_volume=45;citation_year=2005;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. J. Wisser;citation_author=S. C. Murray;citation_author=J. M. Kolkman;citation_author=H. Ceballos;citation_author=R. J. Nelson;citation_title=Selection mapping of loci for quantitative disease resistance in a diverse maize population.;citation_volume=180;citation_year=2008;citation_pmid=18723892;citation_doi=10.1534/genetics.108.090118" />
<meta name="citation_reference" content="citation_journal_title=Plant Genet. Resour.;citation_author=R. J. Wisser;citation_author=P. J. Balint-Kurti;citation_author=J. B. Holland;citation_title=A novel genetic framework for studying response to artificial selection.;citation_pages=281-283;citation_volume=9;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Zhu;citation_author=A. O. Bergland;citation_author=J. González;citation_author=D. A. Petrov;citation_title=Empirical validation of pooled whole genome population re-sequencing in Drosophila melanogaster.;citation_pages=e41901-e41901;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22848651;citation_doi=10.1371/journal.pone.0041901" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (Medicago sativa)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Autumn dormancy in alfalfa ( Medicago sativa ) is associated with agronomically important traits including regrowth rate, maturity, and winter survival. Historical recurrent selection experiments have been able to manipulate the dormancy response. We hypothesized that artificial selection for dormancy phenotypes in these experiments had altered allele frequencies of dormancy-related genes. Here, we follow this hypothesis and analyze allele frequency changes using genome-wide polymorphisms in the pre- and postselection populations from one historical selection experiment. We screened the nondormant cultivar CUF 101 and populations developed by three cycles of recurrent phenotypic selection for taller and shorter plants in autumn with markers derived from genotyping-by-sequencing (GBS). We validated the robustness of our GBS-derived allele frequency estimates using an empirical approach. Our results suggest that selection mapping is a powerful means of identifying genomic regions associated with traits, and that it can be exploited to provide regions on which to focus further mapping and cloning projects." />
<meta name="og-title" property="og:title" content="Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (Medicago sativa)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/461" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Autumn dormancy in alfalfa ( Medicago sativa ) is associated with agronomically important traits including regrowth rate, maturity, and winter survival. Historical recurrent selection experiments have been able to manipulate the dormancy response. We hypothesized that artificial selection for dormancy phenotypes in these experiments had altered allele frequencies of dormancy-related genes. Here, we follow this hypothesis and analyze allele frequency changes using genome-wide polymorphisms in the pre- and postselection populations from one historical selection experiment. We screened the nondormant cultivar CUF 101 and populations developed by three cycles of recurrent phenotypic selection for taller and shorter plants in autumn with markers derived from genotyping-by-sequencing (GBS). We validated the robustness of our GBS-derived allele frequency estimates using an empirical approach. Our results suggest that selection mapping is a powerful means of identifying genomic regions associated with traits, and that it can be exploited to provide regions on which to focus further mapping and cloning projects." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (Medicago sativa) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__MRoKQICmX-Ek5ZIyBlwXzUtbKmCmIVqNnsCLjO1o65o__abNV7fJaqFhsjvAynloQDGCCstgzs17fQnVdDEt9-NA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"hn67Xn1BFjQjTwTWTXEmvE9JV_wq7mcuKfulY6KT4Oc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488309","apath":"\/ggg\/8\/2\/461.atom","pisa":"ggg;8\/2\/461","ac":{"\/ggg\/8\/2\/461.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/461.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/461"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/461"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/461":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228pfnbr3JOft9S4xXsfQPTArqz5dkgFGaMxfrqezodvaMQ4VO2CqHssLWXCnZjLkLHVrQxHbyjalZfed1ZRsupURHI6bOsHC8l27xFyiWqmndqYvN6gvqmxmjGT8hxf9ORVc3wXEbfbK505F3yOl9k67lLyGEM7TGrGLST1MLQA7KxTMtU7xGHSwkG4BQz1oeB3wGHy6W1yjHsOeFWf5OVvgkv5x3s0K4UwmaZzkEr2xX49+mlbXn1Hxa9kpHqfDsSwAmJXYfLdWkYSAy\\\\\\\/40OMO8PVwfrS1zn1dFMlMKlbYfRA7oyEffXHf\\\\\\\/fIIMKQrwpH9HRa\\\\\\\/WlyRy+kC\\\\\\\/1IhlkB1s1+ttT+CW\\\\\\\/GCL879kjeVTZUi7bilPq\\\\\\\/MwxIWZaaY6dtfVlvzYQ92YU6sS+kQsCyLoOMSLverINgfdUrNGEL8P8D+4sJhFLRzEoM\\\\\\\/KKV3CmJJRXTOhh7ZBOqrDP5tF2CYmhK2LObovTioeiSggNcIf0Yq9AwZD56ROKxHNstQVIayN5NRNYnjuWCvgWgBtaVBWty6tLsIpz+CwEtBNpMR28SKoA3TJ9s6ag6mwJKe+GQuIld\\\\\\\/tUULUAuaIk\\\\\\\/mTkZ8Ch99I4rE\\\\\\\/zbgmBRADrcW3CfwSEPfNiM4Pg5CrELemKmCkV\\\\\\\/oy0VPXNKdQzrAWQ4vcGfRsaNNDZ0rg5xSaVAfNGqH6gwM9qxSgnbtQXsiqN7ZwBDhI2QCioQrnrNlZmqnw9QuJEDV67L5nkQmO\\\\\\\/mom+ObW8hcp3Sr6+shrO5FfqfjrDsX90sggekWTZsnjahrdfDkHL6\\\\\\\/y3oy4pPyiGpGa\\\\\\\/RZFdrC4YxHjBvlMQC0eb1xL6iBhyI2MB4fuk2Ny8G3thMq\\\\\\\/vjWwDvwDAmBw23D88dbDB\\\\\\\/2PYlSPvHp5\\\\\\\/quIafTeUrlpIgUNH7DvJOlSXvo16Z7jZ8og0u7W2UqfsXptKDfFhrnbHseh0y\\\\\\\/1wPUSyFKim8chHuKW5atNA0izmPELyIKbIzrvvutOsmhFjPb\\\\\\\/H5rQA6TNq+Fh2yk0rTHaEXXBBOEuJSf\\\\\\\/9kPNX3XTNGyyCzynoRuiwfGtw79YAqru3AUCnPCkZASJQbfEWJVH37VLng1MPGG95Etu4hHjsPcIIEOfSWIVA\\\\\\\/\\\\\\\/+Gcw8wi2T0tDr\\\\\\\/CQvWKXljPlhyMKPOzZYdsWS6Smej9UdZcZpcF0JDDiPLdrNwO5\\\\\\\/DGUm5XYJ\\\\\\\/Pp+z3lzKkRNVwAB7\\\\\\\/\\\\\\\/9s3qiTwtLvUNLn\\\\\\\/Zpzm9gEvAasQ6yqRWKAUI6JEhfLNsr2yn1N+1+5yhSA661zaaf7\\\u0022,\\\u0022iv\\\u0022:\\\u0022w9hcjMemXeRVpYrqo\\\\\\\/96qg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226d9bdfa87b1a6e05a333ce545a947683f97f8304a314b7d432ab06136d61aa30\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yufMm8KXh3AtyHUEXvfiXJV3fEDc3XJ79dU\\\\\\\/g4mRdmlH17dPlqYK6kprycMXlUodOYnOo8Du1pLIkw9Ddnofa9qjwPBQHDk\\\\\\\/6R+3CkdY4mXEt9PVCpxI01UzHdbqnlIQ61E9Ffir9dzBMCHL4WMhws3quzzBj0bAp7yQiHRDLHy6XLWU0DNuYKUzEdmPC3y5VxSp9CYGsDVef\\\\\\\/XV0sJs6iSDYJRMM6v\\\\\\\/ighm\\\\\\\/AKyPWTNVlWXSY\\\\\\\/HpHMMZEElmQFxCbI\\\\\\\/PYh9\\\\\\\/4r56oC0\\\\\\\/Ev50nRuJeWbeqesIBd6jpQgH4+Y2V5FiCfepvNziDcOjKu47DJ9Divs0JYZtLsJ1awzRsopA9gE\\\\\\\/FWDkyOLFOZ+HVqGj85lv32BQABUCco3MjBIVucy\\\\\\\/tMXXurE35ioB+OHyee2+1sVu3o7qsttczLIQFQ8KCEUDnncIFoT\\\\\\\/2tmsEIqw8GTo0zKYETyg2UT55qDJlxyB1qSoEXWD6M9+0weDCcpAWdXpoA9FY1W7ckSkXD17YwBTHg9Kh5iyE90oAEKOuZbJizeNii2KUOO4w986WoB\\\\\\\/+oB5kmTchqhKuR6eOqPqbJ8U8Equ+ZewuXVxFn\\\\\\\/m44\\\\\\\/8NQQIHA2ckStH1ICyZ5R7NcoVnyXfz7Z1VGk0RDsaBMsdQF92M3QlvC4nXpuI9BopyWGDIBLBHFCPA5f\\\\\\\/1WFZPzxkfIWVwr3wxfT0FiCQ764SyqzBmy1N7qOfePjeTMXId7kpzd035u0kuG2EMtwRf+18lKS9znv\\\\\\\/3Z0uiv1xdQND+irsAxtF5J7ZuadmL\\\\\\\/ocqpUu9Hq4jBzu3HiORyQ2dhHiJRcrpCjo8OG2o6GXyUB\\\\\\\/lL60+h6vKti407zQRi\\\\\\\/+\\\\\\\/8lTqAprPdTg13q9b5Hkni56sDibjVAlJL15LOtLSuwuKJfVbiFY+U9uO6uMyS1Hu1SwoBwZShstYeuqAGxhVkewIcyADG7TEvgsGm2fYdufroBJ3m+UwWptpgTcELEM8RsXIAmoJkqvUXwfUFFQ8gdEImIwItZScteGHSMlQDCj7n3DneQHcm+AEwkTk0wyRMEeC9TsGVb8xEzbZq5fE1oAQPjz36dl68Xa07+b7EULV9FFjDZGJ3BU3IrJ+Og\\\\\\\/Th\\\\\\\/8oD54ZxF6jKTxN1E5LgZcGCi0XtDaAwi4e2Vt+Mb3p\\\\\\\/RoRhnVFCWvm4djfsQcFqaTHJDLM4okDAgcGFtlwCJfllVx9r8bBZlmz0\\\\\\\/oTHn+2PtvqtOneJ8T5\\\\\\\/ZBHufNKwXp3VOLp+2NWN5OaVc9XVFPUHnZtw7yG9ToZiO92617k2AvEDgmXa1dtHCHuRUW08OkPhQUGSeOJ\\\\\\\/Yo+8tfgysEjIytIDbvwz5vjbtJfoy0Dlb2Uw\\\\\\\/qYDfWNBzipR\\\\\\\/XHErhDq1o4pdQzdRv3ppzGqvsD8ifPO0wYzYjE+GmP4qjWIMNOY0gfcIT0W4jf20Y2rMJQuYE8zyLOfNZd9cjmKLnHdSjgrBzJvuWanm33eXtCupLDSANLUfZB9sUI6GOKvYruUgJ7f4OSjwzqgWxhdJqZ1ig6Q\\\\\\\/eFUdwzrdAysYgdXyNJpROF8IkPl\\\\\\\/yJquWzeXBi2COzQJ34+\\\\\\\/8XJE2L4WhxsJVLWwVrN545HPQvQifWsgeqlZXxZ\\\\\\\/YqjAWVyZlcsxEpv\\\\\\\/aSQnITRha2WPhMAz4g0PAuM9LTvUVf7E+YQBYLWKlFac9w9cGL++HiXq4nU6j5pMDa\\\\\\\/zaEYHTyLxEWBp61f09elfvoX\\\\\\\/ME7wsbl1G7FuakYMpMQq\\\\\\\/izVj3oAicDkPeZ+kVKlI+kZtuML9dcyzN6X8cHs2rlFLYea2a\\\\\\\/u4qbWEJglH9xPb\\\\\\\/zx18I3pJhwXwr14Z8G4ipPqlqwKt+mDLoQGGFM0bXmJrhelszlkFve\\\\\\\/UVAm\\\\\\\/W7NEs98g9V4iii2IcSdwTzTKZo381vqImVznjVEviAaZSNcGBf7A90rBDMyczbVNsJPOQiGpWlNL2fN5DEGsH6iAyNVvC8z04cu5GJKvpikwr6sL0WKfxbqYp2YSEjLP8Df3J1Xd9\\\\\\\/PZfvFEsepBB8eGe\\\\\\\/Hau6h3f+WADCzvBSd\\\\\\\/E8Zg+Tebx18sWUyefCkEURnIRyb2f9y\\\\\\\/cYQAh1AlOERkH75rDrrgVJumPGEDqLd7vxT+qDZntT8nAgyjq\\\\\\\/62Eu1RXNJ4vW6kGvUKv2FmML8vuhNSHKNY5UgIa9xEIeY37FXtbDMxyfwcFujCyvEl+azeQ2ejtnfjciLfU8HzRDNWTEQu5Don9IhO+Mmnr2+WNhxZkAOhe\\\\\\\/htcUbav1PPM8ngitDKxe86An5knPCaobfTsoxnHoYrttVmJXOoyvyLXLxPc1hhl2KoeONTSjz6Sc34EQelHER88Y4rIKMPR04wq6kBWCFJ5MiPi9iu2q\\\\\\\/Cdosg3eyYl8zjUs16uYcZW3PDafOEBhdJeYSS\\\\\\\/DXYu72R7RzknbSmtYd\\\\\\\/So\\\\\\\/WmCboByMYmaJpic9b5pcCrrpWF\\\\\\\/mLA+pLPzap+ESstvuNLbkdOK\\\\\\\/snefCsRxPvy1KnSMSlnVyOTNGiq+zvF1xo\\\\\\\/j0lCyga03wSYUw+BABZDGMs\\\\\\\/C9E1lKGVt+AZclLfJABtfeA8Ct4CA7lxE0bRTTgmkNHmpfvl6Ar2MDEIJTCl1JLqCThGXdfOXe8hPzWId1Rujcc2gLl3k2f\\\\\\\/MGEI6k1h8Oc4zXS2ryNqfIvykDRkwu8b2qQdiRG723uOVBhOErPHuzwKvAn6MV4\\\\\\\/LjdKo5FK3IfrpHEaR\\\\\\\/dRlEItCTuysV8osmS7GYQ9Z12pwWVrWeEDDlZIrn6jeUBMTOum9jDXLsezslvPG\\\\\\\/p26HiA95lYJ+\\\\\\\/wpQ+gx9ErUsUI8T98ay2PF8UmPkuSFzr0xM2qIT1oa\\\\\\\/CZrO\\\\\\\/a6Y4XXQNKvS0z9aGcmmg8y4lKgv+CPIC\\\\\\\/5l7U+xNQf8EiqN+bp8NXTqWFUKSFbGhXB69By4+sVAGMepmVi7AL0etZYUCJDz\\\\\\\/ArgBNncstPaqCvom5ZgqVK5erTdqPIOL+qgqwT413Ma2bZFsQECuyHc6mCE3nqzS2MOkxNm5Ig0RehoPwjrAB\\\\\\\/pnBHV+UCGpJ7r\\\\\\\/b138NoGp8InnzKcPCkXcsmWHL6if0SajVMIriSDGd2iDW5kVNwuTRyARMSfgD\\\\\\\/60BRHTsDJ2UOn\\\\\\\/X3ftvnegjuIWln4xzKR1Rj\\\\\\\/l3ya4inNecpwDMeux45BIh+ZvbiARfM\\\\\\\/n6RXfMktts9bJdf4Wu4OmN3XHPGK8P7hHIRN71fWfon5i5salNrAoiiJeJ38BRsJpgZsZwnznmvg0Yo7PojObfOeYwifDYX22KHLdFKFR\\\\\\\/BAd8lZBf1H45lLS2jFxlCpz4Ve0BRceGr\\\\\\\/kOsndw\\\\\\\/+M9BbKreliAzx5aG1QNlR4WOPRo9iEwPdFLHpd3DHdjvgySjuV8WcVsyCI8e5x4T5x3NX2Dk+ufVNygps8ontltSUvh9VByhVx1a1tU8IymXNWZ9Y9Ha+yIBff6xHdOCgiHGSxNTXqH4pRzprxwkJ5vRJnEpqueaXCfcXQVDYKR14ipJz\\\\\\\/1ZUi8YOyzi4e6KNfx7NEjTfMZlPzj4lv0BI1XOcD2f3ssWwWOTf34b7kEl2GMZez+vT\\\\\\\/7uuFZEqOhMiyjJi6kMZT8EnAqSHmvLyr1b48BO7rAbDvx2PjaJ9f4\\\\\\\/k9M8713zDnkqTph52kOog\\\\\\\/1QXTVmiEO1hEQKU89UlDxA+V4rj\\\\\\\/IpdOQMZ99zmBJOOhkQ326JNPdVTeoDaGMgYOQ0fSnGqzuXplII4Tqhz9Olo8OBLZkj2yXbvCT8GhBTsGDOC7E80E0s3fsss1J+P7E6IsZ8V0UWgOOfA\\\\\\\/\\\\\\\/bL\\\\\\\/m7NHZ\\\\\\\/7ZtdnytaAWqmrgd3nuwwzGAo4QyO+k+zcYt2mKzrEYGz6RilALrkNYhGE4R6uLK99s6s8kpCguY9LDvZ\\\\\\\/SjSw+jpyp4nEJak8QIwEiNzSxGbrmm3HnhGuqBEwiyX1iqW4YaAsSAJ4vXNfqGy7na4MM7EZzyrIk4LMl+ca4wvLSfwXd5iZrvXdzucPSFN4nfA1wLzinDTJNgh+tBhBSxSh54fjblDf0aK28vNaN87gJsu2IP5Q0kPOtHs9DHjFUbiMdz7VQvUm3FElrhq9Fxv8SO+tg5L\\\\\\\/5n5oq+vst32Cey58Ns3fIc\\\\\\\/UGHFg9Dq7nxkh977\\\\\\\/GZJjtfMewB2RCw8JPvKomsEWm1mT0DgbHnQmQujjPzZp0PBeLXcXPtytcwyeT6CCXCR1wTCtGgZRXK6Pqql7kA5lCJLEdWhId3wxMzbtg\\\\\\\/eb29lBDHv3BeOIhAfiBBlOYrfUVZoAzcM0nFyZ7EAqKdbHHWEkKqXt6bEdvfFAKP+y0S4a32u3sAtuY0xkQU46sjtWf3PGbvPodSLJ7Jkh1+UFHjKfpKvhzTM4Wiw82HyfFeeRShS3xKoGJuyGj8QSJtQ7\\\\\\\/IQkzFTUbIfNZ7DijbibuKKJhWSEItRTJ5iJGALQw6Xym1M0Q+jWMEGlFUckQOxtfl\\\\\\\/hQpDaHFpZi+NF18xBVH+ShTnDqYwP7pvu85ctmzQm4HYgEtPi\\\\\\\/05dCwRNCC4pTPaCs4LigbKTcrlN29\\\\\\\/4xU0\\\\\\\/Lxo8eLuoKPJs6FuFd\\\\\\\/ERY+jIJw8v+8J2JQkXx835ltuPHevATUmQejEcXI+0Lyz7AxELUwdFpmeMyAZHVmNwZAph4neQ5r6MTEhqcdN2N5SJj4CVyv5rnqrDO2HudnMRwz7jvLSKz1X6ADMPsKJjO\\\\\\\/4MCLaMxEN2+x1NsIMD98iHTX+HMRy1beGlpY1x6qsmYxE8CFdgbntrdS+XjnYWgJEz7VTx+hzPa7XfFsa2GTFTRTL+zZYxK4lUndE81K\\\\\\\/dDa7VVeWKW7lEzKHWtRn2Q4Ozsmiq2fWaGygd9f+WXebRI1lu7Oh9CgQklQIzimtvCiBkKYa1A3x8jqHPa4cn7H1Org9MdicGD9oHHAqHUApVtXqeqteSCYB3cGdpxtvLtgVfbvqvEGFcP+q19Ep0M7lWlQc3zzIWQdoSye5Sj\\\\\\\/4pR4QFnWYV3zryNs5xLzH8ykZtBdN7\\\\\\\/tIBFk\\\\\\\/cySTw5GGAvMVbjNqEodpic0hzxB0913j0pRJ9MG87tzyb4NTOXjkAG+a8m2MjHIlwHJrrJFq0mW90XVsXDI4RUkp1ApyquOFvJrrCbpY6kTN10lX\\\\\\\/jfw40ITkiPd9Hxl+5NWvT742N9Z2m3xKb3KdFrCZJ2sA1iht057kHLWV\\\\\\\/I+QYDEl\\\\\\\/jGcJ51Imvaqxpk\\\\\\\/Jcv84eu86nNvYVfHnozn\\\\\\\/ERphMt5c6S0s+4jd9u4ocXRJRbp1cPxdZFX6+5tmt4SlWkapnFEtFJxhkXyscnW5l\\\\\\\/TWNCQjQudogor5Vdg2Mr\\\\\\\/dL1XJ0n4QShJQp64RRBbILkpFpGCSo\\\\\\\/C6ASOKA0B\\\\\\\/tqLVx6gZMpEVFZ8a9grPf5jI5jqAHN4Lsxs8zvCHTYD1047F6OJHDmMVbFWuQohJ4B8ft8gRG+gD+tqKfA4XRaiNlRBvjOJcJitxO+TUO+losUNR3ihfH78v68zKb8DUYg8fkl3SE2I8yk+BPnMInm2q1wgj0CfJkuoVpg6VQkVjuypRZ4zbPRYVIHQxbnIWfI+11ip67CgB7clqTPq9LX1kylnHhnJSHCLUKxJ\\\\\\\/hDmkB5ms3DpeXuYHmEiWbP7HQnf8MwLKpBKNwOS4MQfEEDiSAfvukHALR\\\\\\\/ElNEZz3katK8sVXZWmx5wGh9j\\\\\\\/pQ83fw5EIzbioafTcxacMWe\\\\\\\/mLhKA+4Pi\\\\\\\/QiejG+9BYiG7IhWW04T3pk2bwc\\\\\\\/vt4SxpPdumIBNWE9QfLEr9burrilUN+e2Ihn05yssCofauhOOq6gKPMj\\\\\\\/owc6rcCuJzIDA2g2ikdtFW9KCVpYVEGKFELoA2D6sKI5Q1T\\\\\\\/VkekZwXVjmObmu2r36fCwclas1EdNfCx9s8xeSsRR4yBAAdiLoVhjRw8MA38UdGSCfO6X9ITHuDKzynH2PwxHPZS9h1HnSxdVPI\\\\\\\/ZxzulYo2qVOf7BNUM32yGaRjI0\\\\\\\/+FwzbI7FS8jNHwU\\\\\\\/sWnBswfmkqpM4kF3xD0peDTzlOUgDjxpqNySGkxA6EPws\\\\\\\/\\\\\\\/g+Jj5yhCnD9WiCuh\\\\\\\/u0wmAZL9gUz5pBwGuCd1W37G8G5oqFfRFiqfS4MMe2amvEnBy75h6nN+hkoQY21sMSaGUTmp0hKEWmDkaHQE3kOD7zd9QsdVwnx5Cpus+qc8ESQleHwtaA5xFaegPpggvPg8FfyD7HJIWV7JFO4\\\\\\\/ev\\\\\\\/d9rNFHKteR66TlKcEGOWxJmFZkm60SeA06rEM60BHB6RvSJN6AHgEdmnNg18kKYhmd41kyIsCwjQqwLNGVr+rP4\\\\\\\/gY2nrZIWEvILBaz9UdQmb+jZMHiknLxqRnF+88DuRVb30MvHrVHIX6OxBFjWNsTNGskSgAiN0Tc19pGOtr+6X98TNNIIVXzV4Td\\\\\\\/CqmJ8J2X1omLh5haHYYEPzP0+585040comUcxDpKwGNvoBjjOfTuaEsnYEpl97u5qZQ4he\\\\\\\/X9rlr5c2CeQcbxrycgGWrtPw0rHPEgn70SK2hB7cAwzLPz\\\\\\\/7W8xaVuIR6BFPZGeSN9JLo+PaRv0YxhQyk0rmrKYwUS71cRbTO7dxVYCasDiEMS+WBYPvnWtGgAh0Adwf0yLoIdfAGxObDo9gep2Nh3PpcgGolz8zJTHFL3Z4aud+ahDy6UiZASHN+TCQtWb6QpFE9lpbUsruHtm2qCZrWW46cg\\\\\\\/YOpKY6dM1o73xaiIh0WE5hVMxVtPAG3id+\\\\\\\/rU0icfdi84QPC+T1P19E16SLaAlmHbt43g++T1caJm7WcOmKqtKbfNrkMnafFUVAyqTLW\\\\\\\/yGyPuEAM2dhzBgb3G14zOgB14H6Vkklstzep2o4XQN5QXJwpVcIj\\\\\\\/V4Gsaf9r\\\\\\\/P1xfF4W1QDtJIVxbooudrgZCuosSAD52HA8qS71OhZxQJVz8C7zWcz\\\\\\\/zd75W+L\\\\\\\/NWSJWbo2Jo9YwEWqmidZM3Bh2L7Kho44mI+qqR06np+FoTQe1EzUo8lnFKRUlzRl0xmcbuL8y0dpEzvR+S0B50zbTe9jLLQuayDAw7PMzQ9qrHbNlK81xZNPudNhl+511oJw\\\\\\\/mMzArXGCiS+ZOJpbeoiMbUoeZFDy\\\\\\\/sgw4\\\\\\\/zhIkorsq8ZVfWBg31\\\\\\\/K87JQWAuUYq\\\\\\\/Db1GiM1DJejmbxl7jtIsQif7fQEzD9worK4vJaKX4EBHRXzJ2rLD9cSK\\\\\\\/0flObZtBxpD\\\\\\\/qFz5634V\\\\\\\/s0Q0k6jQVTlzSOtHJfJUnBtVmpOStiBcTi2\\\\\\\/xu3biwYEDV22HqTfpaSqCfc3BKBOoiVqEE0qaEP0F8IYs1b5btnjIinsEsNZtfJO6l0b\\\\\\\/okgBuLpC9NRU8NdXYL9KIMklQdhqvqmlCkSOkI4xiqvCRCfBWaLpm4prbpQ0p3nioKkV8fbogDgxfMwXe5VRKMGzxzgK0xI1oQqGqPxh\\\\\\\/zCp+FuwNdAFT8LExJJ6UjEwqzhlNe\\\\\\\/U1agMOXSZNb+49RuVDVLPJFCKXerirPekCt4s+1cUI8YCmrzIOYVSDd2zWSJMZyRkCUVh9CDOZWnEzijWZhu9k0mE0du5oGQRtY6m8NXIrGWdsbp1jxE\\\\\\\/AOz4eDo2a1c1IYjJdkBdd720Z0QN+GsMoCm4bnfRiqKG8pHr+sgWbQbXK6FsyNnIbl6dWSjwQS27yr06I346zk8Zzk5jmbutLHoykccrFnoBo\\\\\\\/1N6puKWwGRsINz3fJQPYH4wDMaQEsj0aOUFElE\\\\\\\/NEFDDcwP2DUXOleGj904bByqsJjSToLDJR0ZJlvz\\\\\\\/vCxpRqi2qOP26GUVvKE9dHo3XDVlnfcJ6MrLGk6Gw2q73K+0Rh4Oq8MXp8FCSuDjzY2j7cULZQudS0aIR7kUsUAUUGUtUTuYNGd1lkk6QLx2O\\\\\\\/IKuyf1IxIcWYXj4zvdlEHOSLxvgQm\\\\\\\/VlkFX5bCJ\\\\\\\/6pdQ2sKnR4SaemMLxDhe0YMgbVAbZiuEs66dsr1oN8Qr0tNLFCd8ez0Av+xK0nXcifdTLkAJRghOQ1NfaynWNI6Q0kaw11ab7UbEaOfx+XlRWqx79ji7H14J+AoEENedYcrmh6+WTI+nH8oWA4+y8pPgrcJEW6DJVlGGDziCBvsvckQvD\\\\\\\/c0WQPD4a4XKMT5S52VTaEINfHXcaOM3iJATij80wMdCQ1ubzaSbrbwrE1nF0M8LwFiFlbsmmO2vAXvk4PVKQv8mkVjpi\\\\\\\/3P1Xvn418j2g4epD+bl+9Hlc+3+9fpd92zaZJe6SRgF26a6EZJYrxVgrobEbSAdCf4huFuYsR93vonrk6wV1VUiB9uK9dY+FN+3H6qfzvs4G7a8cgxH5TDkgPGKxKNj0QeqyxGZeCtpCUxBzDhtqirMg6LHQYmr+pR9hVOOdYOU3t15SrUAVcirp0gI0nheR3AmmrUhqLWyiMYZOOn7gZRKqU7fyvkVPTn5pyU7YSUBSN4GYEJ8t01UYUERqDZBffHzxjKa00b3+WnspKfD6RbcDpV01t3JW81rBHo9lx9QTZ1q1YSGbw9GNk2RVzRWBSLX8N4gD0s2LOkjFyuEExweUZ2Gf83ULseTwRFc57vTYLMRBInfnn+ptuM1wBAx4tvVTGgTfCbQy2OWqS4ftFr4bo4LT7JPPbPyQUzQV78JyvWdtdjXwBxLW5i10su8\\\\\\\/pMfAIOGrZYngevkveur8SCTAt8k2wl1uwwVSsVNP7tekUYPl8dh5Cyd4BA70cOBS6DdiHyKP+H\\\\\\\/GKYBBnLroixPBjQELqYkMU1TU99QLp94OWwgCvVwAm3LSPQXZTZfada3ktWwafGRyodL5JL21ewzJXChXUEKWHRbVs59jbv7IUYrSG6hB19DvHxd2sPmRXOX7d10VleyPwa7Etgkzv8FvY4vZ+LYdzZ4oWvpc6M9y6KhwCPKMjt9bKDxEQwyjNmv4qGbKuhNXD2s9G+NdmnU9u6mHjcNgAi802I9nLW9wNCZKqq2LY+bbKWMOHYx8MHUntofknjmexNBijc15pUFEv3YIpWaRujGGGMhfEBh74D8Lu7XZF9Z1pOlxnCH2sZmDW+qSktKDBIEKnNIdZTS4tNhbAWDoJjDjWEhUiYuVMvL\\\\\\\/nbC3yrVS2v9\\\\\\\/WsnU28z\\\\\\\/EhzLD9Pk98WxSLjk2HgTAexo2mSGPRq7XWJh18xKLHB25HkBoHm1LdZy5BkbBUVat4XPymevwz3pRaTjNVCLbY0jVTDiehVRGJPcE1\\\\\\\/icri9ADMt7yOyRAlqhXGEyIxGOsp\\\\\\\/l6sayH\\\\\\\/wAxSGrvbmk6+mNf5VE97xyvdfD87\\\\\\\/SSyZFLVGCWu2BOx0jnp4C+qQrzmvKzqNunCEUkNOZxTBjcVw9cJUgsM8Pm9DegPMLdHJBSe9FV4n9rqscOJd2WMjRQYZQzWUp9ypc4n5blUSIyj8TVL9Q+t1fGEkzF8tWdw52uQ2D+mjamm5DGuw+zThZlT5E\\\\\\\/6kJEDs8UONGxLwWzGNS917c9\\\\\\\/oYBmYBNbQIRmK8Na6vauWJmtggOPqwnB4jn5VFG0OKkW+h2RCjcpcAODl5LfAHCUEJluD7EIAu6rZTEkZRn2zQBoRQMtz4i6jFJ1MENX27Ixe6nmO+lwqbhuWnaet10sSrCjrWCQj7ijgcQ78T\\\\\\\/yyfY\\\\\\\/00TeHPHa5iu8GY7eADwuaGFH\\\\\\\/95xiiM1dgFCcpU2GP+w3StZR8c+iE0D9gmtsPW4B5O9mKi\\\\\\\/mSCJB5BycoqANEUOjBSsxdzD8Hp9UCpofFaMNBNUOqdg6dr07HvIf+h192Gma0p7t5MHDsnc+t\\\\\\\/fwu88EMq5AiONHoOJ34W1YMJQtKoxh7pEVH5cdZYZxkcJGe35EU5KlDqG6FMSKSNHME\\\\\\\/Qt1qJ8HH6ftqZdMsI1kIDWXO0LgwDtLInPE7Apwg99Hshs6lnSDf26gGAwwP+JIsxC9TrF+y5t0FzWcq7bLUpZHjQp6CDoiMmGnKPy3O9k8sQQzlHslrdP\\\\\\\/iAIwkuw9GoNm3JkOkIXACabhxajp44hMCvE72VY0xAG0ZNPFgJDTjWPQ8iSM9PuXsooEBMVPIyhuEQ3jD2sUB5iZ1w\\\\\\\/XpOrqDNUhQuyAQXbA0b5nd1Pg2d27X1FDiekjpTFXC+CVdfPZDusFVyiMxtg\\\\\\\/gmxwdbv\\\\\\\/t+wcPrN\\\\\\\/BR3IXRqHUz9BIvdzaW1zNZL3C5eFnfg0nE+bvUG976nWmPn0+n9VhianfZUMaNIJIniMIEN6oT9J+aPVNPJhlUNDzAebkWxTVcFxt6VTPMuwnh\\\\\\\/HN+RCrHtu\\\\\\\/DTlfU0\\\\\\\/YOVCdY2OkdpIWYWD8gHFcF8EHmCRsiy5WQqF84TFV\\\\\\\/pbr+Ut5Kw5MbxhHKg+EExGUTs1QVk4JMfC\\\\\\\/iiYQuPkaChkuIm43Mtn\\\\\\\/tbDLl6f4iauqwvxoTDQvCfNE3ZzBkoRX\\\\\\\/4Rep8lQuYj3eRgzY4CvrKX867N+AHfGQfPbUWXzP7zQHPFd67GDFwC60UGN0zcdcBKP2oUvwKHAUBgPa1FeX7uQXmlMbFb4toH8YfWo0caCdUeTAldNsHUlGQGyxghTcUIVa3HZk24MxhfzlqaJ7S9exG+cY\\\\\\\/kCCLLhDY02dGUNzoL45g5UA+fCzAdOAKVMV\\\\\\\/djSq+ajF47oNkgNEEGwb2NPzCEmcMFDSswrWwySlvDXcfZXMFxYytLkqZHb+nzKiWfiBIaUsVMT6pDjUOGM3cr2xQLaoaz6Bb5NcPlbhDu9WKWKdJdV\\\\\\\/xLfPOuVXjYAzj1Jp7mVqI05d1VoEq9LjeMwJZIg9Jhh3Piq8ueeDMwYaRAj14rl3sNcpkTb1GzJbgAVFTq95oR5MqLH22y0ttjgZ1Zew9SBrKH+lC9Lb\\\\\\\/\\\\\\\/V7qWP+bNCWSkHKcTP+6TV51dhnfmIaO5m4WBJL9yPz5v91W+aEXDL6AI2JHn9tR246iiRoupPEMPDqkA9Ro2SjZjfieIMzklprndZoAhfKKs5CCZMO72N2DhhUwb89uFT66vNJpNkKpuoqeXOsI1V4nBK0a9VVUzs\\\\\\\/eJ3KEThoz1goKjGGHd67q680HWIkS7moRcDql\\\\\\\/CdA7Hf3HIU7ZXpLeIjnb24Ynln7hPPBN0TJlYB0ncRLIvDrvDxYlTcs34eBOJE3ZS0I7b2BIDlDakPXDU15gGX+WBugDs\\\\\\\/y0niNzizoMELlSaoeeARpqFR1nmgRFF39BLWG1g+IgCcfsAZsM3FVrJ0kj3AfMQv4Mv5d0nk5Bh66I6OZCYB3XSUH8HhmGFh+F9369\\\\\\\/7VnGcym0nvWeF4b0fL6vSJdAQjC0Uh9oZ9SmnwNtT1HvdzgrvXbt6WbJb79Fd2NYd8cuPQLF6I992DUoFAF9tKsmWbuwvotDWKJkCWnZmrQMMK2Lgzd8llDsQYckMAOQjF23Tbi35q6dMUQjfEgnNbi8J5ZApbyhsssCz3WkIiGlXQu3kGMpelvkcuFvLc\\\\\\\/ZkD541jD3ulrsg40nx61jadp0lapMBSIvxJi9XpyrRlcxLgOsJWx1QoCBd3xUXpETb9UxmZ60IEyaqBiwEtg7UGdZ7Q\\\\\\\/IupZ\\\\\\\/OWFTZ7Pjq\\\\\\\/pIUukgpZepoGpjV6yLr\\\\\\\/STqjljjhzjHouhwTkG7vIxsp4I4wh3DhEmeXdpMW\\\\\\\/Cw2s63oK1QdmDbOMlTkhcyqKwjAD8ODfoQ80+z0FjABto7xBHUsF9rIf+wG1MQlL0gbO8Ng3r6EBoFe+elKKUDpw0MU7o0gTf3c\\\\\\\/+HB1szBL34emz8raD5yqeim7vlcJaJqgk1tpRmNEraF8ccWJr6hanBai5aIxHw+Nmd3ah\\\\\\\/ygS4Sp0U4iFwIfihv40GNtq8tCa6m9\\\\\\\/ootTSPKlL\\\\\\\/2kUVjYqrlbCDyFWLk4Ec+16GLJyy0N8Ukayy+OaxLrJRg3F558+ECis85jU6v2gbX5aYvBwsB+oZqQwFiuLSwTKmKuafDgxS5tc1E7fkr0+5sdh0Hkhx5azrS6npIBsRlFZTWPIfy6N4jvIZy2dcp4K+\\\\\\\/tJfdGKgOGXMl\\\\\\\/9wngDabPptC1L6PeEwFI8R2sUhkOHuGMHAqXU\\\\\\\/n3XZ2V3h0\\\\\\\/1zizjmq1Y1ooiSptFJm8spZXnwOjl85+AwE7134M8NlEtbechexfjgBkBnuBZL7npuxCTDAezHbgAiKgGbKZOcAvDeHyIm1PDMygBQtJfqQfD++9bnTutcu+4i2CmhiW2oTdZZqrIHz34Fv1uxpDxf99VtMboAMA5B4PooWfXajwxm8ZZIEHp1gXt9P3Birs3wdqvf2wjd0cCiCPfvmvmUJr+MCX55kOuTLpbAOvBnp+\\\\\\\/4144m8N4LYtuYCBmRQMVkTuX4TMAMenbn7PHyXG4bQ66X8mswJO2CtF8UNGHCXc3lTpaDXGveFG9Zm08A2zdLCFmuLFQb3k3CHfSn+gC+bDqnlanljOMCUd\\\\\\\/c\\\\\\\/fpia1vlD3RAN5rtdSNCx+\\\\\\\/\\\\\\\/Ze2VJfbDK5oUGzUPAhEhKRuDgZNzFet\\\\\\\/P2D7dtptMUKvXQeQ2b5Ep\\\\\\\/h1ysep4oCx85pL83miJf6eUeEx+U3sxObeD8urljX2eDfHdTkKrYlRhGfwwmvFJdWQIykuwdoHFSiVSm11AmxOQyaY3o\\\\\\\/z6WhSf1VfBA3Xxcz4eD8vQ9j0ySdvtLqd4n3eMG6DmLJwqrQYdRG4fJ34izpOrEmYVfTXu10jgSLmx\\\\\\\/P2+QpBuvOmCBXUSoTmOZvZD+BUQtOcnKuEbpkOAZeaBNNUB8eaVjiyMe6oDVMbPFj50PQGYJ\\\\\\\/utrcQALJDY93mHLoOA\\\\\\\/DNKbk+7KPrfk+0zbgUxgS02Ti7SoyiYzUUPFM\\\\\\\/bgbOOujj6jn\\\\\\\/tQ0sNj6S4YybZB\\\\\\\/1Yxd7ML4jo5oB7Cah3nl2Vce9NoZZLlZb4uYdMv2m7TcZT2h1rKuY\\\\\\\/vH2EYMjYxWl5J\\\\\\\/10wmHL6M9RuHWL5jk5d17EFmAmEU4zNRoRxw06jyd1W4RzG\\\\\\\/JgBb0AaRKZBfUU30mRZ76DyTedfL+3Iw2JXM0lWLShyUJWFNn1ExeOuopFcFrOip2t7lO2qSQ6PVkwHEUdCSDXU6L5U1KPCxh1QCX248dWR9I\\\\\\\/VtHlJ3E7NdU2H4hfEAKasYNZsTuGicnSReodkNAE+404Jsrfj4F9Q4mDPtc2aNx4PaLvODqobbck9e2KppPmiRNAlAuDCgb7LrpOsqwZkLYBy3ki4ciWKDdmIHlMliF7BdiwPgVNom46xuONEix7SUAoTgX0gOjRBr9Dgcur93sXhXsOXgCc++9gSeOWUXmUkfDz5xHW9lhRWGFCsBqr4a3TB5NZyaZQRShzDJXrla0ETq45Vyvp00WhJgZQkOCc2lE+lOHzSdJx0qoTkRbAxLikX5LTnCMKuCRWIEePj4Kg50filA0H9evCcyHpeepO31WzvQwucnZGMz8kMOM3krpB2c+ryxKU1qLlYhP3eHu\\\\\\\/ZyoDzhRkiQ09IcHl23HMEcanwLPfDSWM3R28mygtSrxAHSSVDRJFwqriv06FrrBkLX\\\\\\\/A+WpsfI6Du9hfqEd3g0WocyF6TdqmFllgjcl2zQq+0T4SDP\\\\\\\/+WWI1TY166HzOF4YJwJ\\\\\\\/2KTOumpZgN9KzcEHDfk129CRrIiPQmduuq\\\\\\\/iwmAuGC1l1hyTlf7g\\\\\\\/dDRLFe\\\\\\\/DMoZnG73xYJ43SSiojhum4xCKJw3Khxj76rZ+evTPkRGa\\\\\\\/9gFxoNG2dZeCZl5TnpFQhdByMOcQR3bAvIiTYcW\\\\\\\/lPNO+ZyVjZvzLhGE+PCRA43\\\\\\\/GBsvQmCAbFmmrAOc4C0m3+8x3YIOG0FNU+CwEks4aCXXDpeS1Qgp8OOi97uqYLeAe9MHzL4eUTeyx\\\\\\\/HdzeOfJpb4SpglEMUQL5RKTeZidzpF4Vst+FX4zw5CS+vidZz2gwKxQ\\\\\\\/qLcMZlsy\\\\\\\/P1XPNUKgWj6zHoCmrFsCL6Ybl\\\\\\\/JIO5o1tC9UWCRrLTIK1wVbdyboN\\\\\\\/tBzMjOkcwebZyyN1n7Tk8AN2XH5EtZgPMEE+a+pSx3InQHrtjojdGwRPzpjIWjiu8kec7rGiLEaNbQbnV8ynj4syUOa5l1XdY2w8J0brFnnsM4bWdomaCEf34mrgjZNUmH\\\\\\\/lhgsX8TMlDJo4LwktYy3fKhc751kAqHekMfQjrrYKBuFGb6xQv6cUZNQf2afwkKJ026h+5Olj6384p1p1QJXNyEQQQFwZrt8Bz69uUwZPUz\\\\\\\/XJkXiKIICXxY9Lnkwk\\\\\\\/IXTsC86X38pG\\\\\\\/EmJZ4U030D+AkN9r4ic6qzJTzH8\\\\\\\/ehCRTNPNx8PgGBaXk\\\\\\\/hQG4E93\\\\\\\/8B8f8T7mm0pvDa9qprbnoEVQzmz4ECV7JvC0dXTaHH3gpBs7x4JdVdGghOFTaHrrHL32yj2+2jPrEbAP7NYf3+p2S2Hwmlmr858aJtZdOonOuLf7STPMUP8aLzBWj5ERrB9Zq+FXcHMGdGTJbVcXzNt+TctQLoJ1wBLiQgxQL6DC4M2GzZ35nsn79Kbd1neXXsBm2AwuqYfK6phedn7ZIGp02dxoy\\\\\\\/CdXZwGV8DPF0RPagimyPpyX0waXTVZms3bANghRadGGIgHXUFLwnrAjtKBWGvJsbEaDiTA6Ix14lQ4u3u726k1\\\\\\\/OnW\\\\\\\/\\\\\\\/Yb7FHFhvOmJwdm+vAZAkDctpS\\\\\\\/mGTQjg6cgW3\\\\\\\/7m9rP9KHue4m9dvwY6XBASKuL+LEgUt03xEXIJoj\\\\\\\/ldMJI1QNn80rPSd\\\\\\\/QiPMvxD3OXIyD+kAUETSQinJt3sgMbYno9l5pQ72ndMC3+adwg8H6vYuBAezIFQpN\\\\\\\/auE3BOId23fSXeVkQW3YjxwDRupfOLmnxhdIabOgdwkTF0WlaELbriClLZHZugROjE1Et1EVbab65JQovBuocaZBXuBM+Igs7hqtBqPPiK7tH5878ChFEi2ZQeF0PSWPo3aNCAh3zUe2TJa0vj8L54V6akkiZXaqabQiU5QryC6DRNx6lhsRzrcnKLjrBmf5RwikxKCcGFEeKZ\\\\\\\/iMtwXaIGkoql2+0L1ufoWLFMBvTHKqncHCQY6lyiB3wpJjufUrK\\\\\\\/q+C3LVLroCIGozS3GSMiWFGnpDcVbmypBRs+0yUGAp88x8h0sDcUwcWVeyzVGe9feqxktc9Nk3Jctn5cQtPG+AlJQdefPEwr3G0METyTm1\\\\\\\/Z95l+nKrhKzQJRQnzCkNmz2JyNH0eSAvnv\\\\\\\/wAfw2EEbp7SrUYPtac++LE\\\\\\\/qZLOwGYAYyPTYxkLkb6YAZVfrUxmF8xcCm8NcIqbuaNjCHoIf1O4Kgd3a1kQCza8nYQutIp80mZSPYB8E4WDwSP+MryglW3EydhZO7qXhuosmcUOiry9slrp4zPfImzshE\\\\\\\/HQJy9idBrb4nlSyZ6U2bAlZKqBLbMz\\\\\\\/bWO8Mmn4HRvjQ4zJQunRAWcN6dLrErBU6TdFDJI2hMbyqSHjcMTz3z0ASfSin59AusDmDT8GIALv8gc7gn0XWpTr+PmtPsIKrjUKNnh2D7VQFCIIQvUmXdUBFvVOuHsuBro9HE+X18B7YmNPYazvukuRQ4\\\\\\\/Uh98Seo9CAd1aLIaBVQVNU9FjeDtpWLR0QMOJ1842xuFfb3fkmcWi0UmTLvfQflmxGMgto7v+DE3n9TGNDOJ3xYppyEXN+oEWlugIhLQdytrCEPC1CcWTYv2T23PTSvi\\\\\\\/M0ikHqq2Jh5ViPe5MS+L1\\\\\\\/fubvfnyBCfccqjE9xNY17mRAcPAAqpPFOvpQXozAXQbFtohiafFw3xQTMq5fyIlUQzkVisDYgKUETYk8VZAJUJUd3hPlhbMdHoE5zButMa1jAHqTIugM+DX1VoX91\\\\\\\/QdCexE\\\\\\\/qUDBK2ShzbM+EK7NDZwSglOVVIWAQ2\\\\\\\/A1FBQVAFa30jrU\\\\\\\/FsXW21RZYimXsIjRhSxtoErs\\\\\\\/xry2U+KCequcdjUBe6+UNHdXMRcsZLeOQwWIaA0F+Derlv5L+UqUITxhcKweXopjFpzCr1jhOA4XUhO6KD7G6pCPZ3BqAFXc\\\\\\\/NPPj7WjiK7LdwNVbew3mVndnK8Z05OAmn5RUCNtN23ESjRPnx2NvfFoBpRZkbEVDdPJZJiv3+8fDqsizg2BhT+bSrZ6ZaMwy9GEeb3mOxMHbtNi8QYpYk\\\\\\\/VnwINz2++ujLfh6avuvTFbyaDc0jUM+hzd0QFs3JS5h1x9rifQYNBQm+PQZKxWvDK\\\\\\\/MHb2qwT80J7r34AnwWaEQadju4eDuELe\\\\\\\/szzz51vU0AnkM1yGy91BKK7EsLEgTzGGUZjW0MCus3xogkRwrJKvCAOPZUXFSAqvVdHslmGKqcDE\\\\\\\/istviyMBt\\\\\\\/oyE1xb43dHhvOrQIX+bLrV+mekW1LmXS0JkB9QrzbFcz2osKILGc5xUcrjUkDyYEQuDNSV9xZ5r8FV6W7NWCLOjFEoWJ09vjFic8yD1Ll2JAalHHTXlbtkMq3BSx1887QrFx3VExqgFYkn2a4OVvhs6hysac3vawVb47A3DKqMyewHAgawmGHIX51cDoRdwfOExJUp0h1+nrRHjmoUb2FpeBFe51A1bv7vZ+1RhrrPgzrKqA+M8pH3o288kemBqNx3RCoz5Rpc7YBBWUF+25gQiOQAO5u0EDyTJtHyMYk5heKdrhHFxh5bcIZQMVjXdtJM9vWDPUNWvRO9IwBrmAbuoPjvJwCpuVZIacWvF6IGQpCO3a++tz0ZFY7+e+lh0l+T62o3Dz+pkf39eSH6kM0+eP9IMpEdwfyLuzX8x2Tufqbvj5PP8U5QbzNN3gbiHikdK04nCip\\\\\\\/fICk7lKCyWDc7MYV+VgB8N+8E6mG1NZQp4RY9L387WszvR\\\\\\\/0\\\\\\\/muUPtBAVMS4CzrPQP1rMIyg7T54EoWa4Jb2kLAYcGqxq9Uwku2\\\\\\\/zDsN5V8Y3aJPWjvZLPuz9honEDa5PXCSOdicqsoYCYK1pjDqBIY8boPx0XRKovAj2Sam8lyDtjq5sIXu61d+J70YKphY57nsRV1W93e6aNnuVUKX614+ceLPbw0EGHty1SofCNPaUEdVitFVmuoFCEG1mEvMRo8d+IwIoFM3\\\\\\\/J+JJk6oOSYmNS1kUZ5zqReVAztoCkD1liXcpW2ls4z\\\\\\\/r5BQVRuz2LoJBveZ7R5mIJF6J82vL8Rsbcnlr\\\\\\\/fFKMIOljeA9aC108y4M2F2SovkyDt9fEaskAGnVxYRkaTtChDAAM+FNu+IBi0bd75RUCJrHWq\\\\\\\/iuu9oYYoMJRtTkItMUafThkXMGYGrmXb6Pi5\\\\\\\/\\\\\\\/uIlI0I7xv6KponWKKbjNjxTHkXfmIMJzeZLLusqFUG69Gr1C58hUEorY\\\\\\\/YAEld5BzMFdljm4ZSfFHcUey4dQ3EmE\\\\\\\/PL+W59QaNyfY18o9XqvMgOevabpB2L2ecCE9TNMOISgh\\\\\\\/QFG5PUrj7BVT9Pkdme8A5WOlaWj+qENf3rXMu3zsFsKGlkt7TInEH\\\\\\\/g5KwGe5+2nSqE6lbWOiiZElC4nhARTOo2qusuG7Z3pyfnfGdQWx7ndgcnD5pXjB\\\\\\\/mfZOMvY8r4VknoIerI7VEYHIN1tcDYGDyIyO\\\\\\\/6gmF3Y4no5KbmDOaVl5LkjV7tYLtvDsSwJDSgJqw0UnAi6ipi6iJxtB3x5ANvWFoGlcO7I4U6iPQOCShw4r5I\\\\\\\/VTwd7zKdMHq2A1tPy+KauTvCWEZ+JILYz\\\\\\\/9CNj+8YxKfFUByP8Hq+RP63R8stYYjBiOSUa4AlwAsENZT7ETS+v75AUx7IhhtH3c53376ml0jF+HfKocuX21ZA2pyRhPSaN1Ad+2A2EkQZtK29cqX\\\\\\\/O187BxQqm9mTdjMt5F1oNznV5RpDKSqv5v+0j23Za8iUazUVdATwWNj5m4UYA4k+nBldrdmsQE9O9gjMhqhx83DFUk0feIL7dLtwKEiglMRZra8udsBctKdf0XKStmEWL8xtEX7HABb4C5vCHP54XVB0IcK4HJ29e3KDY+Ix9XOQf\\\\\\\/6nvoxtQBvmGty6sJtWU21mBF2n7R67Se2j5AoyIBpYh\\\\\\\/Fe0ss3mcjtxixg7JNC\\\\\\\/6951zAeSCDD0fTbmgH+pjs2RkhJZezbKyXJHOM59oCjTcRK4xHqvCCb1+SdDLhQyU1S+K7\\\\\\\/0R1XTZt716LPedTIWE91lS7APHesfzvgPCsXHO2ZriSfnqqMQKqY2CTllmIoUPvoJqLsUl6fJImKSw3jfRg3JIhD3SfEs85k3WpoXoWjdzHOpb+UCDmUMxD1CD82LZXfwSr2IXovuB3sk\\\\\\\/88Yuuh2FfdBRiqfYP84XnxRi+NWdrXEWR3azk9je0gN+CeRPJpfBrOP0LZ26xz6PopIh9cAkghq2KHgtJxMa5uAAGBOySEsmPNYaaushd8eSL6NWkB134Dw97W8Rd3WB6e\\\\\\\/aZiUK2HWwVUms285Pq+SL3QGLraNkzgErgJqmBYFbH\\\\\\\/fEAPlROSnQEn30Ct21mwcCPTbrlGorXj+rUyUC8ktDmoAJ6NvfsqAStVb4iofZtKZy8vcbTEIiJ83txPGwmLQwklJr++vwg3ahX+J\\\\\\\/8IjWEopO3O\\\\\\\/c78fgIjk40WY+b798CAXsjxU3hxVfjBx4AiwMdVum4+MRaBIEYVaW1M7jMcA8OVlZY1MFWQ8lVYKgAvOfVHTg5Gnkum2vDX779L9VDqKoS\\\\\\\/VQIjeCPnzmswtKS5ilfCWtBSH5002sFx4HFwuk+kSFbBEn+XdgEbea5vXqRh1K4zaudieOPz0dW+3JLrT6x43beSfdB18cJyq+Ah6c0BFb27vos84uE5Q0JOcC2T+awcSWZ1DDt2sPGsDFEfNcubUBxfxAiEksFTFLc0IjP2VGNjU6\\\\\\\/kKXrdrLK0LoQUmNV84olzYurtSAUyGpqigtmX8t4HQVFauf81Ofrz3OMF39TibsqREGlwFpD3BN4\\\\\\\/dCzU5R2qyF4ln9vwgr8G6h2LMly9cnAeBClEdTqXhwv5cCR6Bb6iLscIeBabZiY2Dvb\\\\\\\/D3WwREVPGFa+lhRKJlVvXsIBhTErZApQsyrVlzTCvJYnxpkKqnZhV45Ic9VFc8WKE\\\\\\\/7RAb0lxag\\\\\\\/zaR68jsx9KqC5xT+SHvyN8wDIZ7rqdAhh4XNntGiRtWrySRyd38pMTCODHEB4D7FcHggFXfnIRkPrijgzw1CY36OsEl5D1OgbTX7w7NYvlua5bvIy2j3\\\\\\\/iRlf+OM57sbCfLKY0KjzDeabVc0GVePLWlgKL1oIS+JjXn1ORFSksn3IxZx7IIEqJm8Iuw0RIPPNmGchulN3+0m3Jji\\\\\\\/fP5L1+Cz2CQtHetdu9mTCp2VgZfO1kC+VC9CAeVEB37b2tJCC59oNYctVsrqJ1IHKYSJeKB4ciVj3voPaifL0EN2\\\\\\\/fAcKXv4wSHWH5gutKU8fiwUhOlWEBDbweQpvHVEvZNpkghDmQJmRI51T\\\\\\\/SF5r\\\\\\\/2SmN4LNwYnNVn66T45mK62PKp\\\\\\\/qkxxDct0DHrn1JQlz68oXi9m9HYDpnHnumlIjVhZDZ2Ja91mq0wSzxkOaUsYjGJnyos6RQYb2A\\\\\\\/bBl2osX7D\\\\\\\/6ZtAdtyJcMEVPuszXFa6T01uXXsnAs1EDgy2aeEYBoQ0scgdG9+XguzRXiUr4INNhlAGYXLgqLo7vVd91mAG6zsJt3tm2JdfE0RltVfDrYsFLKHEtCo0zRWCWPwAzHeT18rWdPqhLJaK5O+qpgJh2M62s2y9XXpY4oQkR8y\\\\\\\/6LOnimLsPkLIRtA85S9cHs3no0V\\\\\\\/mFstW7O8QP1lDPfn2+ZTSiCt\\\\\\\/DEVC7rHn1emcTep1OjOg4xOaKEaUkFaINY3sSHmTKSp\\\\\\\/WNbZV39SWXKJQ99Q4MYwyLhnNWIypBPCmzUooPGQTjtRcVuL0jWzlIY7o3zqL40lRQr58T8DE9l\\\\\\\/L9yFEZ9cI5cHhO+XfWNjifqIAEfgUdwr5r33XCL5NqEVn+4qcrCgQvZHZ\\\\\\\/q515DVV4ghLrxon6BL0hM0Hooa0Deg73X7F5rrNEISxq0LKf6Pl02+iiFH+AMP8KsGkI1uOljUSTbq1TKgBHnzXZJX3icU1aGv7xkcTmx5rxQxGfP3mG3PFYlJSqbLh4rhBve+XmSuU0slgCxmkLP32WlbgPpprCcYDCcrt3cg4bsAaB+BFtVqNORcpasQ8QoANTUquhMFeumfpz96fMZ8Hgsq2knoawMspsP1B+fbZd8AIxbKqgOGdw23jrMsNW1RPFmD6JwuBXr6hZI8YWTCe7fLT2yaleo\\\\\\\/C16qIew\\\\\\\/P\\\\\\\/4E1rJlCUmHFUUI5UaOcTuOWtufTwJHaBcrHnImtU7GHdIM1LwVwuFgwKMklAbZNMdGMT0NJMfNYCze05tICPdcjZBsHVLKmOA+36Ww0N73mnkU2aLoc6CnjlPEvlBlzoDmnK0R\\\\\\\/5OvBpb6\\\\\\\/fkn1nbRq8R6e+3DrEGuuFAfKdTdM619gWWKTXymXLeP\\\\\\\/YBuFfbl2hjTBQs\\\\\\\/vTyfqviAmdXrs+X3XzEHw+h5VxiaF7u0Ta2x+AT1X6eQg71ToWM7ySjosNDBGKevzf2TpwM2YjN3EbnTBCPUtTf3\\\\\\\/56fQd2Es2YGWU6\\\\\\\/3j9WlCVBJMSeICNVyoOQJGE7D8EdlTV4cl3Oemeu9eDw776mIrYHsccbsleoa+OogaCwTYKngzfJ5zx4NJmcxWqeBN6fz\\\\\\\/XP3uDZg9Jd\\\\\\\/DVF0HJjgH89i1Rq0PfaGqOoejEbsQrsvs4PqHyTy9NmYSZEDDqyz6RsD3DvtbTRZXv6PicwOdEWbuCJc+PxZEcg9FnbZKk1i0wwKmZ2QXM+sL4xVK\\\\\\\/xiGh7I2009F7u942aC3g\\\\\\\/Bn48u1TL3qsOrYF1vkMfXLAepLidxh1C3S8UCMK5GYDf3teHjYOqWXodGczxpJemSzUHzkyIJbq99\\\\\\\/DtfC1rjl2d1Sjq5\\\\\\\/u4voGQ0a\\\\\\\/zF3LCPjSdnXrduja4wXjnduk3Ma96ZcGdh5FQRXBglHzlC0P83bNiC+KIbAJW8Ze2pGVOPATvf1HQL3YrPXvrYFYTYr3ERJ4TKo1I0S5DjiuuI3VMa5iDDTzkSPb5ZspZxaUz5obKx08+NSjaTAPArYhvNYK74t3eABJuU3wQYnzkI2qvxcMw4hO7SeYt4RR8JjFciOGlrFk5+LfCt1mjbmZD4SpR6amLle3\\\\\\\/DkoUC3wqp1AY1VvkLVSv5IGoiVZvLxNdWiI2EswUsIB3vcjESl\\\\\\\/x3EBK+1LpV1KTRIrIERAdbo2lvK4e1hFKkV8HAv1B2rBov6o0dXCLmDgK+akH1AqQbp57Ri5a8c2uilKpJMc0xHOPypsMcLbrw2jYDQVMcvQk+X84e7SYZSwzKF4JcmDH8FInY\\\\\\\/1oj6rpRrQQkcbvYUHvMG0bYJZ6+sSrim\\\\\\\/nHYZ34NJSOSp32dwV42WxvBt0IMG90G4eGqmGLRfvhSA\\\\\\\/rGQ57dOSQO60NztzVXIK5Pdimgkp3hg\\\\\\\/5WoSpNPW8w1DEKjO4EJCLVrPJ7FP5nBzpwvpoIDYSh\\\\\\\/gRRhui3rgA\\\\\\\/lHOHW3DegmKgyIvRcLLFAEMZolm41n62d6LVHmjMNTETLbbpMwpzg8xWm8HvTMqbIy22x7\\\\\\\/kP8vLEOJW8uDIF4T4vwq6ttAfEaPZJtcg\\\\\\\/kcM60zLTqhXOPaPJe4iuX7IUBb7XlCgFKaoAQgQl+5STNV3AJNYSQTi3NIng7ZCSPezy2tNmgkYle\\\\\\\/9czpwszd+aKybuQqYYYdqBf2\\\\\\\/oMqe3m+79f7wZ8nqTeF0AEVDV3z7mM8p3HU8x5H1jXFi2iUXH+HbeURMTM\\\\\\\/UM\\\\\\\/uFy9uYJAXXEZt6IMq6roHw272N0vSi36NubGkR1oJv9bXCu69Xzhp5rOKGBvnMl6L10do7ae2vmzlIYBN171c94tautChHLNy9Q9HVMfZltuSGUieRP4pzGZqpNmeQCCXQsP4irVK9iHhJlF49iAp0RY4Q3YK6BvsOWwqtpURkp\\\\\\\/IguHSBCyNZ1IwfN9XkZK5KJKU4\\\\\\\/SpHfepJUWGml+pa1NwJ6bgliifZzlv\\\\\\\/oLxCHpU0YpFOObk2KHNG9v9GPy9XoCFZ2MbOGL19qjRth\\\\\\\/tHKKpqSkJElcU592lyXfCHJjqgJhBsCEhiRfF6knVozETD2+3CoFv8uwy4qDX7iauG9SWhY55K5I9BfEZhitHccTuLKcYKeHyi3POZ6fLVFnBsmtxo\\\\\\\/V5q+hZXhPXBowAnkLCVeqKa9LE0NFlOcoSqHHljcSWASgcr8DlcG260M+01a1kDgl6cexLoRSnD7SMolC5yPLLzuCo5gP3mv0MEQcBnT\\\\\\\/HCM3AFAJQw45f4SlTBMRF+5L8IxMDOjjzVrhqcryjYcQcQ8g9Ql4JaBa8YhEJafvW0lPSHHRokS+k\\\\\\\/XUTHXz9gB1HNhTBDSBDIXRavi73VfygbF\\\\\\\/M2zOlRnzjWLrVEgnpuDKQg68B9uvmlKjgTX4EIdCeMwejFg9vNbs8QxMarzz0Nsrdu73NcCXLBuKPaCn1R7x\\\\\\\/pxjwniAVGwIBF4Us4nbs3C31nHtigZMxBjBvo2fzj5HDKv7+NFTSkD1NioTXhB+PTFyUj2qRmBxCoEzR5B8xQshZtNU3aSOZCjvpjZGP3ml16NOA5Q+VvUzRgUPLMpMKY+9OU1eAZLLQWxSGbAPF4dKZLNLq4qZC2u6vW95J663qFS\\\\\\\/f4F0bkJB7VjiBxFfkBPLzWycQan0c4sgTNXG4IyepQ\\\\\\\/8WytO8qsCtJw7VIpKOd1fL5VgrL2idBR5p9s+BGQVysgMshlYOMu9s9\\\\\\\/RR34R+XyAXd2c0NlGYZvL7JMTPj\\\\\\\/VDP+EqMKQ5GvchkxDacmVFbCegEQyuRE+zbwaOhx6EiluRPZEBd1VAWgMhvWOj\\\\\\\/ugeTeRg+a78VU9KuHuIsULHNagWwAESe+kW5J7vaRCqkmgpgTTVioRgcF+YCMHxpmitTYauX5NfIH46Jz4eGTIHmGB8qqbfTdFoDEU2ym0StjEoyeaaRowOeb2Nx9J2\\\\\\\/gCG0adKld12c3\\\\\\\/dfwb0TuA+pYsDxdVJs7NyV4olC\\\\\\\/JKet4eUhQCodBXLWdgH7xtkiy\\\\\\\/91zk5MT3o8MCinb21BCrpgMmEVhFio3AvZr7wp+Rk6AhGtTW\\\\\\\/wK246KRynFrB\\\\\\\/pBasshJvC2E4RA1amES27Z6mJMU6ZAgunuIQDOy7F1CKrAYy8dTFIQd8H1oY91UQUcEpoREPQW9Lj1qQQXOjVQQnbQmBovkIih5pZ9rCuVM1jh5DuVH00okWqVIk5bIGYBZsa09aFND5Nsa6fGCO8z9pbjAoQQyfhOWoYk1XFTX6N7xQKWJhkJmIlvuspGmESXRZ20EGSfMjt28J9WAw1pq4C+vXyU++P6RE3+iVVRWwEEr3fxt+4Q99tmHdeDZjThjZIHbbFXe1kQNoXkwVVZNB\\\\\\\/KQDTV6N1PFcnHZgHF3JJ0d241Lzak2ZD50sxilcAViKW+rdLIKy9EnHTH2wf1U4aKtEnjrkRnuaciR9wg4wpctb5JrL7mWhRYqtMnKlfh5RmQtxzc9KnaaOUozBML\\\\\\\/i9P6RnY4jx\\\\\\\/FvcIcG7nux6Qwz\\\\\\\/X0Iidgt0\\\\\\\/Q\\\\\\\/szVSDi0WdAzqpFDZZX4EFNnnZYRFBMteAz1emCefIXO+Co6BGXwooS9IcX09JOGMqgPScM932S\\\\\\\/O6xphQ8WQsnt0XOrKvmZvgVUSwNdMpOuTZUnBZgdcY9xX5jz1jGn9bBocUGw+h5XvhRjtRgcJd\\\\\\\/b8\\\\\\\/QoO6JGYxd0zt9M00M55bzUoVHrjk48K79ltk04iyW+iQFcSdI14F7amu8qczuGZmlh6Mk5Qu6xJYA9fBcYOiIWJYCRi3wk9mOq46ARHsJP1+GKqGFkuR2GhSa\\\\\\\/ePwQWbFs\\\\\\\/98tTteIGhFlGdk2NdtErqs7TCNbVu8td5w95KWcDJAq1DXhW6To9pfLw4iEVzyAqtJ9aCpPDknhEoHvPoVe7UpZqBq8ECLdBRNPir2oM1EbtodSbkMqI7JAC9FH\\\\\\\/PMNsjrTuSp7oxPHjZOxa4406YpbFK5pq6nL+U2jdV4g1kkh6VUZt4o2FMxgR30Yq6jxofFnANLkMoK4YyW9cbWByv1HJZPuMF3UDp6fqNLYnB7Ng56iMOV\\\\\\\/T4mNlXgRPuYSQjYGKl7fyinqSfh8RSwqHTs+0kl0yeF99\\\\\\\/fveRkIWcoQE4XRgZc82A98oBY5RNfau6PQF1eT3JlAjGTgS9D05FcVXK5NWfSjzzM+sxaFRP5HgQb7NIPm8LG5K9c81TCmljQz1GGMk9Gv5TDj4yl\\\\\\\/PNZlrDQkW6oOnL4UDj\\\\\\\/1+t1JAE9gAXrao2lV4Nn8GG+OZFSw5GVvUmxX35OS1uF2cH+yNfJasqUbZWV51c8ugTvwALGT3kAvpFPOHipC2yxbU\\\\\\\/gKMOq0eGAcbH1n4q2IpgvVCmneUbr+KtPQ\\\\\\\/badgh10f2S80Y4RKQ9DtYhaiygzPe7G4MNQK\\\\\\\/zDDHElv\\\\\\\/fbIqbxhE+LMvANtWt138fOFdqUUho8vpQ4cXgQf0+KslIkQBULda62mbsIczvQbhVGd22QdbK7L+eb\\\\\\\/FyHW4j6A7yd9xr5Bc176gHBklSiWXeJoHoEgVuJoH6gaBC\\\\\\\/lMj3Fwt2Y\\\\\\\/iKkj+Au3RSgDNN2bBsi6sqaONvIXwEMAQXPxmF+c\\\\\\\/\\\\\\\/ROzkPh2x2Jb+V9DUvwcAvSmWDWjfz1AX2BeiX\\\\\\\/Ae\\\\\\\/5mt84yhzAQsi6h0\\\\\\\/NldiRecj0215oHSY9YEV0VJ4ET5JE7umKBbDRtbMmu\\\\\\\/Xxothh0YTWH3hpSdz1X58sChFmc0deq1MUTkAPVj79D6ButMP11q2sLoBMOcnXNRAu0L+dQ277kGBDEG042WxnKNYagE7pFHFiZcJ8cgo\\\\\\\/0IBbh+HyJAMg67HOzjQ4s0PPq6xnYsGpntgsDCaCHsQx100Nqq\\\\\\\/MU4coXeOGaV0MDeZmcesOKUo3J7raMQyJlWyDqYQmHQL2Ihlemk16C2HKXJw6Jd3diZ0N5+VMwq+sg2MaRM9Z+9Was4ZgL6OsGsbTfGvOr9jCBes1BpeJQCfM7RMTmMIBR8\\\\\\\/EzvW0m3qRGDxDneXBD\\\\\\\/D+tgwAzsOA0apq\\\\\\\/1VQG17hwgAuGojRX\\\\\\\/x0zMqw03v4PgTMiuROjOjwDZlHa+uUvjP7E6iguAVsfSJl2N3UvEESvcc6kMOXnWvyaHQbSODtfViag7\\\\\\\/F\\\\\\\/m\\\\\\\/SY7MAM89qrHlSLwp6pUvUDPtx8lIxAs\\\\\\\/M4VB\\\\\\\/kjbEIC1VIq93QSCTfKpVBvxcDc9W9XeZLaCXW5fDIw77DyGANBZMg1oG4jkmMM+fI1Ay+7VI2ojG4vzxcNM14Xw+PHqHFMcI1mfVx0ilUDQtGSNjtp0EEX00TE0Oluv91ntnwh9w5B0rKrHcgtJyTbvLIzEopK2cF1Bq4WNy91smUqmMvtyUGYRpU4QilVJ5D5DiaJnSw0jKoV5s7AAe7Riuc+jjXHj7gx7EDv78aVNLmALbqW8gu2YzZ+VlswkIgooyUAtXYLWSye98ytTuPmDwLSzT0euZwG5JtNYmLSBscTBuTF52OefC3G8LNO57TrfYXxTAd31ZfaxgCyzNp9fbXF6t7kzm6RD+QPCax+97Lx+IleTZ+4ryGEx\\\\\\\/5EO9jAHrMuKVkKeELjuqspJz35p9aATZOgnXKOEVjlDkeUkWDsi3dBC3CDd0iHHLYrP5hQBgvifdBanTZ3gs0u1t29RVKFzMEJeVSVOc\\\\\\\/IuUuqOIUPHFc3IZSX8DIEaxykbP8GNfhT8k+tFgtHRI73TuxNs7RBSatUqI6rjNSdv1SIVS2kvmXZ3zUHG1f5fe571oXUQgIsvHewF46KM9sfLf\\\\\\\/NYKFXe5WIV02mYvceaQYNXZ1PJnwdlb1Y7Q7xkKfRtZzbbgTUtnSr9JvvWcUFn\\\\\\\/PTvFQd\\\\\\\/bEb5nWOo67x0GWw5kFP0WhMSAotXRqmsnHwjD3huHNOYgAJHaEEAv6mnYVPomfaTfKt7hp58UcCZuSPqzGTEAUux7dchFQmQoGcSPsKAR9ijXC3b8CKPAzkiBMSA2zVQv+7aTT+yyTNKmESf2QcAnPqvMm75s51uEBo8kY\\\\\\\/luxbRiuqEq5qDwmyTFFIuwQ\\\\\\\/+63l6di8mz0yAJyjOFM1JMdrH80zqxnAemk0xl6QkChVba6W0OWMlzWVfXTnQDN7DlSOEfSXSOrTLESS93mPGkBf0Y5zA2KsQzZPYDEUzRXe4yer8FL9lghcW+c5qvlB6YytO6KeQ19kl7ZWWXYFm\\\\\\\/kiCHG6KQ8nCE3wQq3cgADKROb3uCW1DpSXMPaxWDHYhVvI8hwWQBBZ7ufBzBJ9V4DhowSy1li0W18+1ABTuMxl9b9do7XBIRJRxFSf\\\\\\\/on5rydgj6f4CP4WL7Pu1HND29m\\\\\\\/rDTxTphojAYf6owRVHayviyMA5S1owRf0EnW+bjBrFbH1Ronmjx2veknc3f6re77Tz1VZCVuqg89oGiZOFQZEYbxrgKczMnGNNlOji2FTAnVaQz3gtWq7nk7RNxTlgPPblyHB6e\\\\\\\/gTpjyJmCFEnECIIEmWttLDiHM\\\\\\\/9bBd27zBj1Sgb2SswK+HVvOJrUYiEogX2NaNtSb34Kpx\\\\\\\/Qh4JtLTT3X3ITwaCi+qtlLBJt7v9ni+IDD+teAdowaXLdOriPWtUdm9cdtaWt6jJP5bxT2huBq20QFra6iyP7+xCYtLzbBKHXvF5JkpYKw7fO6OoiHsF6cK0MnJrrXfkRPJZVzo6hsGLpOPxp\\\\\\\/qwsPhjJkOMhzpFtNuG608ynH8LrUo\\\\\\\/\\\\\\\/lLZqZGkCvn+AbKxiAjRqYiQLf5V9pJEvgBnTEoTOaH7SFMqHROGskrHL2vPDX5Qy5a4S0vADeDnU39njbx5tNgDr0MXRgTM55RcPV9Dj4k9twnK\\\\\\\/h2rtEP4ofqWSII6P271F9clpGJVRtFW7nfb5X6SB0bHfboaZbhq+qQBlIBhmMPPA8bRCKJko1MI8A0stRKpiVlGnL22xgjQ1HGXmO\\\\\\\/C+O+ks0hjNu7wFuBg7Nyfa7MCpH5X0TumlJBMwGxMHGexP71ZQ0X1JpukkFiXbm4EIeLcM2JtITMPgvG1ix6tIQI2gn1RZudg4VFGuZo6d9FJ+Gzj+hyGmniOKpdCCJZi0R0NnqpxGP\\\\\\\/5gspDScAp91mYKneQgPT8TZjRWxLSuA4m47\\\\\\\/5WII\\\\\\\/APYrUsU2Nf7\\\\\\\/vQWVvGi6pRaZP2Pefu2LtQKIz6YSg3eaTzNRaQ+qHHhY4U5zZH5jYpKmjFl+J4Bi5RL0yDRhCtldPxht0Ispjx+BvOIM1m0hzdugZz2qwJGKU0TEXQWf7+lwHWr1CcebZRtIE7Ul9aiVeaiEhj672d4\\\\\\\/GI5oVXyoARZzLMh\\\\\\\/Akcw8GIT1lTK6KQHpKC13c3KxYpYqTPYZS\\\\\\\/esv1Cu0EgtMgV8TzjpyO5+dK6dgf5toBQ5WT7XlhZWITzgt5o4BeApPaQz47bg59J9+ws+O3K\\\\\\\/FWvL3Na0Kr6+S5JSV92s1bMW+7eRoOjRuft\\\\\\\/G2q0K57OMDOPktSGl7sdMPRtyaRbHQyL8c5I9GTVFeNpjyvEYfTtT\\\\\\\/0fgdP+pTT5w7pnEV3QAI4ZOseV\\\\\\\/boyJ+ycYt2CZEa5Pjn502klhRMceR2Ee8KkDV7G7LGeEjt1LdsUmyvv191tBe7I0THc7V0wDy4WYjX\\\\\\\/uA+7rtc65P3cJ9oOuGYV2YTP8I77LZmGKDE2H0oNyuUzQpTu5tUbngv\\\\\\\/rDmxjcDIeXIAGb1WN6b4HWQoZmv6E7DzQ9iifiEuVkkCthsZH49l1bmjUrc5xFytUvNYVeTtaHMmJObIAx6k\\\\\\\/ppUdC2YNFPIRnMKJWlUHJ2\\\\\\\/rTQQ\\\\\\\/jPxX7y0OqzMtTV3Go68LpuoRghwXprNQv70G0y8\\\\\\\/EYsfyk+M+cKMCc+XNJyfxUu+kjuFhHP180lX3L4C7P3hVaPt72E7n5Pw3SUYwl8NwBbnnKI3BSAVibNRNB9KdrzGYHFJUvjuvhV7JglFGbE35fYLVyssTYIjkBnsEusi4Lb1GkDSPAndqp+ZKpBBPDTvsMJ9KJbp2de0DS7prdLJFQVj\\\\\\\/5Ui5RxBWPm7Jf4fXuT7jUW8fjPpT2Ky6Z0rowf0V+PRiMrHQi\\\\\\\/vWvOfxhBb0eiTsbULf4IxnpbxgVR1M41Rqic3wYrxuLsWDgjRk6mw6SqzgqPxoF6Gx8bGMccpekVlvbULdV2p9EMJ425O16HV7rEboKkztq1MHl2+inQBc2D3YzwdYzlfl9GtJ\\\\\\\/6ABANLo11N3oqxwZtPAh7YoWhH0Ofl\\\\\\\/BacIUHcj2B9vgYZIs3Of31CIOIMplFlALPo\\\\\\\/IRpswHcadbKzLmD1ePzC+Je1mm7e+2YnMYNFjLaxUXRTOgJnfHA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Z7Y5bF5aK3Ox3u25RCanDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f1c5985d65dccb4ea033bf805753c1fc2419a6c358f7e76ea8aa3e97a6f16a69\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n20Q8covB0CCcyOfuoioAkJ2eJCFHVZOk1r5fHlL57ZvzDgLwtm7tDo1KCSJDZhjUJNygcYFtNLnj35EhKTZRpnCVPnZgSxFxDN3iYNyrcrj\\\\\\\/YDks6ePO+9LHXq4CazLQeQU+kT4QEM986feyHu1ux3\\\\\\\/M9iJolwX9TIGwruUxesBeuOzOP4sdGY1W0jFLHanzktujRAKA0gnyNxgjY71gYSeM4Crztq7z4Z5Couh+YJQcUhIqfIfXpkL+jCvqJ4TPIUllT10zwJjOREsxXtJ+c++m9zE1L0VHj+kXQIjj3ybjiug3okfbpbytzbcqiyhCWC5nCJVHJFVaCWLCDG5kvxCYZ8IZWvGplB3ozLrZWggmQClKQ64maVFrNRzpYA8d7LBpr9RxF7PGdLZAms7NnZeB69WyIpw4f2eh7\\\\\\\/\\\\\\\/1Y2yXv\\\\\\\/CUxIf0kP0nfnEeqQeAsz2W6r0VM6UQKj0Jtf27nJFX+ObUKiFY6Rq15ZpJA6Q\\\\\\\/R7+qLBQweXKHwC6M5Ct6CUTDpIU\\\\\\\/PcYCrNCNf4SLqs9pNa1mPL9gaSs5jJXz\\\\\\\/CYOSbOJsXx3jCWhbdZFrWRTJSSIcb5TGH+HMRkhtdwjLIUyK1q1xnxKqIR\\\\\\\/1C5DWjiL4s6MjVW94pzgZNcQDfTZydZQNODVH\\\\\\\/jTQ0a\\\\\\\/D6OPuUK5gvOyF8O1dWWba7d\\\\\\\/C7qoDTiKuiiw3IXvDoFSYUv\\\\\\\/epRZfpSzBHhjM94zY8TFgXXuMIhHNpd+Q92ioZ4zAM1aIywbRDN6Az1gFcXa9Yc96z9lJNXBT\\\\\\\/d9FQoJDxVDS8AKNEXLTAGj0mpe8sMTrcvfItU844JPfKZa7Q76K3xo3g22mG1Y3NVr7Af3nGJAhf0aMOut+lxB94FhSDGwIadR758GyrRUE1v\\\\\\\/lnPOLX6IJzCpoQ\\\\\\\/oNZ8X4UR1Q8MUx9KyrESjAVaH88cgVOAl9f6Cs8ASx9RqfybIJeXDjNXz1jHt7rMTxywGORs\\\\\\\/BIBR4ILVgs60a8aKhLhaPvJtgBzqJVF6lRojy\\\\\\\/sy96MUJsPOgot+vU3g0752x5hZ3ctewmrZwhdvhbOL1By57C8r4U6ImsR6yfUh+RpCtLDjyVB7UGt0W+abd8Kpx9Zx5HNbUXqgc+7X\\\\\\\/TmauyEAsbwx+1dWUltwqxPHATt+nozKgATOerCVh9KnHWxg5JRAGPtll9N1utdSywhFvUufMywqoVLOHEMFD4juiB2+8Lmpkbm6\\\\\\\/QWPd+fVQk5l0C2eocHndE7ndB8FN\\\\\\\/XfjBcN12Sj\\\\\\\/XYymWAb+5YJI727hlw5aTA2pjAyCiMDp5XoHbb\\\\\\\/G8pgM0aLANwtUIJhWFU7PFKcocRNRyimlq0Jnk6K0LSRusj3bCcQCPG3z8TdHrGCMvY3slooiUbUNnMCBrzVYqXAhG8RTkjwKhCYxjjV0d9AC4KSbcrnAiVend9u7Ft4j9Kvky0mHu1OeAqjYfoDh\\\\\\\/J9DKC2rnQLyWe5lnTFr3K4wwU+0duu040Ra9gnlAORfkIKqmzPl\\\\\\\/nyeqMKEebncakroQiT3jsQ4cndlkxN3E7IXyA+hYhtDluZaiPCutEyqmNF+eKjoNdg6bqPCmGbr3nkyUioR8y7HAq7cLqDGt2i1oG0acMds7iH8VLRILdfBkfJgkUmgOK5\\\\\\\/K8aYFu1ZAp7mZfUgVTQOnom+uRNTVRES4XR3CsdO3ZOuMxaXVjUFc1arnO2PlF3SoHNc4s\\\\\\\/OZUmHb6zf\\\\\\\/HH7LSXasftqA3RFRiEUF\\\\\\\/mGP5JrtmRvEjGOO66Tn8LUkwG4BHiZK6ED7LKKBH3OI18w1JS12h78D+1XLAksMgUPZLAJJx14OAHJ89KxcN0sTNk4EOes3+h5aA0xy3q4PWaf\\\\\\\/UE3HOAqphCLNYhBbFc3QCa6EMYxJo8\\\\\\\/NHjOcbFxAhhFv\\\\\\\/EQU9F2b+p3n\\\\\\\/mP+Lb7a9Urik\\\\\\\/YbatMrfhSgECM5TuHwHhJzCRz4\\\\\\\/\\\\\\\/gEo8IO2sAHJODL63WUuVSSRC8gcbCN7v5qhPENvUxNf0dt3ca1jvC1vnvbMfxSe6Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MVsqLRewltLhYAxbXbGxZA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d9051a6566d6f3c9f52eab5fd403429260995c6f554864eb3df0b7a9d86a0792\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488309 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/461" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pF4qvLQ9wdHNULTtr1G2bzxXxZq-i74C0YknZESf32M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/461" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5S-ul5v2OaEyUcjthoPymoFtdeG9XakEGrGmcTthGuw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/447" title="Direct Testing for Allele-Specific Expression Differences Between Conditions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/469" title="Identifying Genetic Differences Between Dongxiang Blue-Shelled and White Leghorn Chickens Using Sequencing Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488309" id="node488309" data-pisa="ggg;8/2/461" data-pisa-master="ggg;g3.117.300099" data-apath="/ggg/8/2/461.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (<em>Medicago sativa</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gitanshu Munjal</span>, <span class="highwire-citation-author" data-delta="1">Jingjie Hao</span>, <span class="highwire-citation-author" data-delta="2">Larry R. Teuber</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3621-4516" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>E. Charles Brummer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">461-468;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300099 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488309" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gitanshu Munjal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gitanshu%2BMunjal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munjal%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGitanshu%2BMunjal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jingjie Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingjie%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJingjie%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Larry R. Teuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Larry%2BR.%2BTeuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuber%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALarry%2BR.%2BTeuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">E. Charles Brummer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=E.%2BCharles%2BBrummer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brummer%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AE.%2BCharles%2BBrummer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3621-4516" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for E. Charles Brummer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecbrummer@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">ecbrummer@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/461" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488309" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/461.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488309" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/461.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488309" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/461.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488309" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/461.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488309" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Autumn dormancy in alfalfa (<em>Medicago sativa</em>) is associated with agronomically important traits including regrowth rate, maturity, and winter survival. Historical recurrent selection experiments have been able to manipulate the dormancy response. We hypothesized that artificial selection for dormancy phenotypes in these experiments had altered allele frequencies of dormancy-related genes. Here, we follow this hypothesis and analyze allele frequency changes using genome-wide polymorphisms in the pre- and postselection populations from one historical selection experiment. We screened the nondormant cultivar CUF 101 and populations developed by three cycles of recurrent phenotypic selection for taller and shorter plants in autumn with markers derived from genotyping-by-sequencing (GBS). We validated the robustness of our GBS-derived allele frequency estimates using an empirical approach. Our results suggest that selection mapping is a powerful means of identifying genomic regions associated with traits, and that it can be exploited to provide regions on which to focus further mapping and cloning projects.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/selection" class="hw-term hw-article-keyword hw-article-keyword-selection" rel="nofollow">selection</a></li><li class="kwd"><a href="/keyword/mapping" class="hw-term hw-article-keyword hw-article-keyword-mapping" rel="nofollow">mapping</a></li><li class="kwd"><a href="/keyword/alfalfa" class="hw-term hw-article-keyword hw-article-keyword-alfalfa" rel="nofollow">alfalfa</a></li><li class="kwd"><a href="/keyword/dormancy" class="hw-term hw-article-keyword hw-article-keyword-dormancy" rel="nofollow">dormancy</a></li></ul><p id="p-3">Alfalfa (<em>Medicago sativa</em>) is a highly nutritious forage used as hay, silage, and pasture for dairy and beef cattle. The major trait used to classify alfalfa germplasm is the expression of autumn dormancy (or “fall dormancy”), a plant’s response to decreasing photoperiod and temperature (and possibly light quality) in autumn (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Castonguay <em>et al.</em> 2006</a>). The onset of dormancy in autumn occurs during a period of acclimation that eventually concludes with the expression of cold and desiccation tolerance (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Thomashow 1999</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McKenzie <em>et al.</em> 1988</a>). Further, these changes in metabolism are associated with the slowing and eventual complete cessation of growth during the dormant period. Dormant germplasm exhibits reduced growth and an increased decumbent shoot orientation in autumn compared with nondormant germplasm, which continues to grow upright. This process allows the dormant plants to survive stress until favorable conditions for growth return. Such seasonal rhythms of dormancy are observed in both woody and herbaceous perennial species (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Thomashow 1999</a>).</p><p id="p-4">Autumn dormancy in alfalfa is under complex quantitative genetic control. However, results from inheritance studies indicate that additive components of the genetic variance of the trait are generally of more importance than the nonadditive components (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Daday and Greenham 1960</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Castonguay <em>et al.</em> 2006</a>). This suggests the potential for selection programs to effectively make gains for the trait in variable populations, as has indeed been successfully implemented (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cunningham <em>et al.</em> 1998</a>). Across multiple populations, phenotypic selection for increased or decreased autumn plant height resulted in either more or less winter injury, respectively (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cunningham <em>et al.</em> 1998</a>); conversely, selection for decreased winter injury resulted in a decrease in autumn plant height (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Weishaar <em>et al.</em> 2005</a>). The relationship between the traits, when evaluated across germplasm spanning the entire range of dormancy levels, is strong (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schwab <em>et al.</em> 1996</a>); however, the relationship when evaluated on germplasm with a smaller dormancy range is minimal (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brummer <em>et al.</em> 2000</a>, and references therein). These results suggest the feasibility of manipulating fall growth without affecting winter survival at least to a degree. Further, this opportunity may be realized quickly if genomic regions controlling autumn dormancy could be targeted by marker-assisted selection (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brouwer <em>et al.</em> 2000</a>). In addition to winter survival, fall dormancy is also associated with other important agronomic traits, including spring regrowth, regrowth rate after harvest throughout the year, forage nutritive value, plant maturity, and biomass yield (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Levitt 1956</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Smith 1961</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Busbice and Wilsie 1965</a>). Thus, understanding autumn dormancy could further our understanding of the genetic basis of these other traits as well.</p><p id="p-5">Selection on genetically determined traits alters the frequency of trait-associated variant alleles in a population. Characterizing genomic changes that occurred in a population as a result of selection, <em>e.g.</em>, changes in gene frequencies (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lewontin and Krakauer 1973</a>) and/or in the amount of genetic variation (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tajima 1989</a>), is a useful technique to identify loci associated with the trait. Further, mapping in experimental populations developed by recurrent phenotypic selection has the considerable advantage over other mapping approaches—<em>e.g.</em>, quantitative trait locus mapping in biparental populations and genome-wide association studies (GWAS) in diversity panels—of encompassing a large amount of genetic variation while also enabling the detection of initially rare alleles that accumulate over generations (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wisser <em>et al.</em> 2011</a>). Such “selection mapping” has been used to study the genetic control of diverse traits in various organisms such as maize (<em>Zea mays</em> L.) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wisser <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beissinger <em>et al.</em> 2014</a>), dog (<em>Canis lupus familiaris</em>) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Vaysse <em>et al.</em> 2011</a>), and sheep (<em>Ovis aries</em>) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gutiérrez-Gil <em>et al.</em> 2014</a>). In recent years, selection mapping has been conducted by pooling multiple individuals from pre- and postselection populations and using next-generation sequencing to yield highly accurate population-specific allele frequency estimates, thereby reducing experimental costs [for a detailed discussion, see <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schlötterer <em>et al.</em> (2015)</a>].</p><p id="p-6">Here, we hypothesize that mapping in experimental populations selected for more and less dormancy will help identify loci controlling the expression of autumn dormancy in alfalfa. Following this hypothesis, the present study uses the cultivar CUF 101 and populations derived from it by recurrent selection for more and less autumn growth in order to (1) assess phenotypic divergence between populations, (2) calculate allele frequencies of biallelic markers in each population, (3) assess the reliability of the calculated frequencies, and (4) identify genomic regions controlling autumn dormancy by analyzing allele frequency shifts during selection.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-7">The cultivar CUF 101 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lehman <em>et al.</em> 1983</a>) was divergently selected for three cycles for taller and shorter plants in autumn as described previously (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Cunningham <em>et al.</em> 1998</a>). Briefly, 500 plants from the original population were evaluated for autumn dormancy by measuring regrowth height following autumn clipping. The 45 shortest and the 45 tallest plants were selected as the parents for the more- and the less-dormant cycle 1 populations, respectively. The selected parents were intercrossed in the greenhouse without emasculation, and the resulting seed was used for the next cycle of selection. Three cycles of selection, maintaining the same population size, were completed from the initial cycle 0 population. For this experiment, we analyzed the initial CUF 101 population and the cycle 3 shorter and taller populations, hereafter referred to as CUF-O, CUF-L, and CUF-H, respectively.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data</h3><p id="p-8">Seeds from the three populations were planted in a greenhouse in Davis, CA, in June 2014. For each population, 96 healthy plants were randomly chosen for field evaluation and subsequently transplanted to the field embedded within a broader experiment containing other entries. The experimental design in the field was a randomized complete block design with four replications, with plots of 24 plants representing an experimental unit. Autumn regrowth data were collected in November 2014 by measuring the natural height of each plant as the distance from the soil surface to the top of the tallest stem as it stood in the field (<em>i.e.</em>, without artificially extending to full length). The raw data were standardized and fitted to a linear model treating blocks as a fixed effect. Residuals from this model were then used to estimate phenotypic distributions for each of the three populations. The variances and means of the distributions were compared at the 95% confidence level using <em>F</em>- and <em>t</em>-tests, respectively.</p></div><div id="sec-4" class="subsection"><h3>Genotypic data</h3><p id="p-9">Two to three newly formed trifoliolate leaves were sampled individually from each of the 288 plants (96 plants per population × 3 populations). Subsequently, tissue samples were lyophilized and ground. DNA was extracted from ground tissue using DNeasy 96 plant kits (Qiagen, 69181) and quantified using Quant-iT PicoGreen dsDNA Assay Kit (Life Technologies, P7589). GBS libraries were constructed using a protocol modified from <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> (2011)</a> as described previously in <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> (2014)</a>. Briefly, 100 ng of DNA from each sample was digested with ApeKI (NEB, R0643L). Two different types of adapters, a barcode adapter and a common adapter, were ligated with the DNA fragment using T4 DNA ligase (NEB, M0202L). Equal volumes of the ligation product from each sample were pooled together and purified using the QIAquick PCR Purification Kit (QIAGEN, 28104). The ligation product was then amplified with the KAPA Library Amplification Readymix (Kapa Biosystems, KK2611) using 50 ng of DNA as the template and 25 nmol of each PCR primer in a 50 μl reaction system. PCR was performed under the following thermal cycling conditions: 72° for 5 min; denaturation at 98° for 30 sec; followed by 10 cycles with 10 sec of denaturation at 98°, 30 sec of primer annealing at 65°, and 30 sec extension at 72°. The resulting library was again purified using the QIAquick kit (QIAGEN, 28104). The library was sequenced in two lanes on an Illumina Hi-Seq2500 at the University of Texas Southwestern Medical Center in Dallas, Texas.</p></div><div id="sec-5" class="subsection"><h3>Allele frequency estimates</h3><p id="p-10">Cultivated alfalfa (<em>M. sativa</em>) lacks a reference genome, and the closest relative with a well-documented reference genome (<em>Medicago truncatula</em>) has a genome size roughly half that of <em>M. sativa</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blondon <em>et al.</em> 1994</a>). Therefore, in order to target alfalfa specific tags, we used the GBS-SNP-CROP pipeline (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Melo <em>et al.</em> 2016</a>) for <em>de novo</em> single-nucleotide polymorphism (SNP) discovery. This pipeline uses quality-filtered demultiplexed reads to build a GBS-specific reduced-representation reference (hereafter <em>mock reference</em>) by clustering reads based on a similarity threshold. SNP calling is then performed relative to the mock reference. Here, we used one lane of data in clustering. We followed the default suggestions for single-end data with a few exceptions. The similarity threshold was raised to 0.95 and the clustering step was implemented using the program <em>vsearch</em> <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rognes <em>et al.</em> (2016)</a> in order to overcome the memory limitations of the default clustering algorithm. Further, all pipeline steps following the reference development were parallelized using either the perl Parallel::ForkManager module or custom bash scripts (demo available at <a href="https://github.com/grmunjal/tutgit">https://github.com/grmunjal/tutgit</a>).</p><p id="p-11">For every individual, read counts were extracted for every variable position. Read counts across all individuals from a given field plot (hereafter <em>sample</em>) were pooled <em>in silico</em> for every marker to get sample specific read counts. To retain a set of high-confidence SNPs for frequency estimation, we required every marker in every population to have at least 96 reads, equal to the number of individuals in the population. To control for copy number variation, we set the upper limit on read counts within each population equal to the mean coverage plus two standard deviation (SD) within that population. Further, using the list of nonredundant consensus sequences (hereafter <em>clusters</em>) provided by the GBS-SNP-CROP pipeline, we required every marker to originate from a cluster with valid alignments (here defined as primary alignments with mapping quality ≥30) to either the <em>M. truncatula</em> reference genome (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tang <em>et al.</em> 2014</a>) or the recently released cultivated alfalfa at the diploid level (CADL; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bingham and McCoy 1979</a>) assembly (v0.95; Medicago HapMap, Alfalfa Breeder’s Toolbox) using BWA-MEM (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li 2013</a>) with default settings. This final set of 84,978 markers represents our high-confidence markers. Sample-specific allele frequencies were then calculated as their maximum-likelihood estimates given by the number of reads representing an allele at a given marker in a given sample divided by the total number of reads representing that marker within that sample.</p></div><div id="sec-6" class="subsection"><h3>Genetic relationships</h3><p id="p-12">To assess the robustness of our allele frequency estimates across biological replicates, we calculated a genomic relationship matrix (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">VanRaden 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goddard <em>et al.</em> 2011</a>) with the expectation that samples originating from the same population (<em>i.e.</em>, O, H, or L) would be more closely related to one another than to samples from a different population, as determined by pairwise covariances. A tree of relatedness using Euclidean distances was also calculated. We calculated the genomic relationship matrix as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>G</em> is a <em>n</em> × <em>n</em> genomic relationship matrix, <em>n</em> is the number of samples, <em>m</em> is the number of markers, and <em>W</em> is a <em>n</em> × <em>m</em> matrix of marker frequencies centered around their mean and scaled by the SD.</p></div><div id="sec-7" class="subsection"><h3>Divergence scan</h3><p id="p-13">To evaluate divergence among populations for individual loci, we calculated <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-2.gif"/></noscript></span></span> for each genome-wide marker for all pairwise combinations of the pre- and postselection populations using the heterozygosity-based formula from <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nielsen and Slatkin (2013)</a> but estimating heterozygosity in population <em>k</em> with allele frequency <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-3.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-4.gif"/></noscript></span></span></p><p id="p-14">Another commonly used method to assess population structure from marker data is the use of principal component analysis (PCA) to decompose variation from correlated markers into uncorrelated dimensions (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Patterson <em>et al.</em> 2006</a>). Given the controlled nature of the original phenotypic selection experiment, we expected markers most strongly contributing to population subdivision to be under selection and thus trait related. To distinguish these candidates of selection, we followed the methodology of <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Luu <em>et al.</em> (2017)</a> to identify markers excessively related to population structure in our experiment. Using our frequency estimates and the PCAdapt software (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Luu <em>et al.</em> 2017</a>), we generated sequencing read counts for every marker for 250 tetraploid individuals from the 12 pooled samples using the <em>read.pcadapt</em> function under pool-seq mode. These data were then used in conducting a PCA using the <em>pcadapt</em> function.</p><p id="p-15">Using the first principal component, we calculated a z-score for the <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-5.gif"/></noscript></span></span> marker as described in <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Luu <em>et al.</em> (2017)</a> for one principal component:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-7.gif"/></noscript></span></span> is the regression coefficient for the <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-8.gif"/></noscript></span></span> marker regressed on the first principal component, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-9.gif"/></noscript></span></span> is an estimate of the residual variance for the <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-10.gif"/></noscript></span></span> marker, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-11.gif"/></noscript></span></span> is the score on the first principal component for the <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-12.gif"/></noscript></span></span> individual. The z-scores were then used to calculate the test statistic, a robust Mahalanobis distance. Markers in the 0.1% tail of the test statistic were labeled as candidates for selection.</p></div><div id="sec-8" class="subsection"><h3><em>Medicago truncatula</em> genes near candidate markers</h3><p id="p-16">The genomic positions of candidate markers were queried against annotated <em>M. truncatula</em> genes (<a href="http://www.medicagogenome.org/downloads">Mt4.0v2_genes_20140818_1100.gff3</a>) to identify genes located near these markers. Based on findings of fast-decaying linkage disequilibrium in nondormant alfalfa germplasm (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sakiroglu <em>et al.</em> 2012</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2014</a>), we looked for genes located within 20 kb, on <em>M. truncatula</em>, of the candidate markers.</p></div><div id="sec-9" class="subsection"><h3>Effective population size and drift simulations</h3><p id="p-17">We inferred genome-wide average effective population size (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-13.gif"/></noscript></span></span>) from the allele frequency data using a recently proposed estimator (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jónás <em>et al.</em> 2016</a>) implemented in the R package <em>Nest</em> (<a href="https://github.com/ThomasTaus/Nest">https://github.com/ThomasTaus/Nest</a>). This implementation extends previous methods of estimation from temporal data to pooled sequencing by correcting for the variance due to the inherent two-step sampling process. We estimated (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-14.gif"/></noscript></span></span>) from the O-H and the O-L population pairs using the appropriate frequency and coverage data as well as parameter values to represent 96 autotetraploid individuals per population separated by three generations under “Plan II” in <em>Nest</em> where sampled individuals are removed from the respective population.</p><p id="p-18">After determining <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-15.gif"/></noscript></span></span>, we were able to simulate genetic drift to assess whether the observed changes in allele frequencies for our candidate markers could be accounted for by drift alone. For every candidate marker, 10,000 allele frequency trajectories were calculated to model drift in a Wright–Fisher population of size, in chromosomes, equal to the higher effective population size estimate from <em>Nest</em>. For every trajectory, <em>j</em> the number of alleles corresponding to the starting frequency in the base population were allowed to appear in a population of size <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-16.gif"/></noscript></span></span> chromosomes. For every generation, sampling probabilities were calculated as the frequency of the allele in the prior generation. Binomial sampling was then conducted to generate counts in the current generation. Counts from the starting and the final generation were then binomially sampled, using a sample size of 384 chromosomes (96 autotetraploid individuals), and used in calculating the frequencies. Markers showing postselection frequency changes greater than could be described by a 99.9% confidence interval for genetic drift, defined using the simulated trajectories, were labeled as good candidates for selection.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-19">All analyses, unless otherwise noted, were performed in (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">R Core Team 2015</a>). Raw sequencing data were deposited to the Sequence Read Archive under accession PRJNA420993. Read count, frequency, and phenotypic data used for analysis are available for download from <a href="https://figshare.com/articles/CUF101-selmap/5266537">https://figshare.com/articles/CUF101-selmap/5266537</a>. Scripts for analyses are available in a GitHub repository (<a href="https://github.com/grmunjal/singlepop">https://github.com/grmunjal/singlepop</a>).Please cite this paper if reusing any of these resources.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>Selection for reduced autumn height in a nondormant background resulted in remarkable phenotypic and genotypic differentiation</h3><p id="p-20">In each of the three populations, some plants were either lost during transplanting and/or failed to establish in the field; consequently, the number of individuals available for phenotypic analyses was less than the number available for genotypic analyses, as tissue sampling was performed in the greenhouse prior to transplanting. The phenotypic variances of the preselection and postselection populations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) were not different. As evidenced by the best linear unbiased estimators, selection was effective at reducing autumn height from 0.92 (<em>n</em> = 90) in CUF-O to −0.53 (<em>n</em> = 64) in CUF-L, but a contrasting response was not observed in CUF-H (0.93, <em>n</em> = 63). Failure to make genetic gain when selecting for less dormancy is curious, but it possibly suggests a lack of additive genetic variance for the trait in the starting population. Given that the starting population, CUF 101, is primarily nondormant, it may be approaching fixation for alleles underlying decreased dormancy and/or improved autumn growth. However, because a very nondormant germplasm Wadi Quarayat in <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Cunningham <em>et al.</em> (1998)</a> responded to selection for taller plants, it is unlikely that a biological limit is curtailing gain in CUF 101.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/461/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Top panel: distributions of autumn regrowth in the pre- and postselection populations evaluated in Davis, CA. Bottom panel: distribution of minor allele (across the experiment) frequencies of genome-wide markers. Each population is represented by four curves corresponding to each of four pools. The right panel is an expanded view of the leftmost section of the graph in the left panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-325782251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Top panel: distributions of autumn regrowth in the pre- and postselection populations evaluated in Davis, CA. Bottom panel: distribution of minor allele (across the experiment) frequencies of genome-wide markers. Each population is represented by four curves corresponding to each of four pools. The right panel is an expanded view of the leftmost section of the graph in the left panel.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/461/F1.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/461/F1.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/461/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/461/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488424" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Top panel: distributions of autumn regrowth in the pre- and postselection populations evaluated in Davis, CA. Bottom panel: distribution of minor allele (across the experiment) frequencies of genome-wide markers. Each population is represented by four curves corresponding to each of four pools. The right panel is an expanded view of the leftmost section of the graph in the left panel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Our estimates of the distributions of minor allele frequencies of genome-wide markers in each of the populations indicated that the phenotypic differentiation of the CUF-L population was accompanied by a strong genetic response (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). An excess of low-frequency variants postselection, as seen in CUF-L (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), is indicative of strong directional selection, as neutral loci closely linked to the loci under selection are swept to extreme frequencies (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Smith and Haigh 1974</a>). The genome-wide result of these changes was also apparent in branch lengths of a tree estimated from mean genome-wide <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-17.gif"/></noscript></span></span> values (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1/FileS5.png">File S5</a>), where the branch length of the shorter population (0.02; L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1/FileS5.png">File S5</a>), is nearly three times the branch lengths of the remaining populations (0.007 for O and 0.009 for H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1/FileS5.png">File S5</a>). Interestingly, a small enrichment of low-frequency alleles was also observed in the CUF-H population, indicative of a genetically differentiating force operating in this population that is apparently not related to dormancy.</p></div><div id="sec-13" class="subsection"><h3>Allele frequency estimates are robust</h3><p id="p-23">Allele frequency estimates from pooled sequencing could be biased by two sources of variation: (1) sampling of chromosomes during pooling, and (2) sampling of reads in sequencing. The robustness of maximum-likelihood allele frequency estimates calculated from pooled sequencing to these sources of error is a topic of debate in the literature (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhu <em>et al.</em> 2012</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gautier <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rellstab <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lynch <em>et al.</em> 2014</a>). Here, we sought to assess whether allele frequency estimates calculated using our population-level filtering criteria based on 96 plants were robust by assessing frequencies within subsamples of the overall population, an empirical approach essentially similar to that of <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Byrne <em>et al.</em> (2013)</a>, who investigated the reliability of frequency estimates across sampling replicates.</p><p id="p-24">Covariance between samples is directly proportional to the degree of relatedness, while covariance of a sample with itself is inversely proportional to the amount of heterozygosity. A genomic relationship matrix calculated using sampling-level pools indicated that each of the samples was genetically closer to other samples from the same population than to samples from another population, showing that our frequency estimation criteria were able to retain population-specific signals (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The observed genomic relationships between the populations parallel the phenotypic observations, as samples from the CUF-L population are estimated to be much more genotypically removed from the starting population than samples from the less-dormant population, as indicated by the off-diagonal covariance elements of the matrix. Inspection of the variance terms (<em>i.e.</em>, the diagonal elements) highlights that, across populations, there is more homozygosity (or alternatively less heterozygosity) in the CUF-L population. These findings are in agreement with the expectations for directional selection and reflect the observation of more alleles at or close to fixation in CUF-L (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/461/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat map of a genomic relationship matrix constructed using 24 plant sampling-level pools. Labels are named by the convention &#x201C;population&#x201D;-&#x201C;number of plants.&#x201D; Dendrogram represents tree based on Euclidean distance between populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-325782251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heat map of a genomic relationship matrix constructed using 24 plant sampling-level pools. Labels are named by the convention &#x201C;population&#x201D;-&#x201C;number of plants.&#x201D; Dendrogram represents tree based on Euclidean distance between populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/461/F2.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/461/F2.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/461/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/461/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488458" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Heat map of a genomic relationship matrix constructed using 24 plant sampling-level pools. Labels are named by the convention “population”-“number of plants.” Dendrogram represents tree based on Euclidean distance between populations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Frequency shifts suggest directional selection</h3><p id="p-26">Phenotypically, the CUF-O and CUF-H populations were not different. In order to assess genomic divergence, we used two methods, one based on <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-18.gif"/></noscript></span></span> (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Beissinger <em>et al.</em> 2014</a>) and the other on PCA (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Duforet-Frebourg <em>et al.</em> 2015</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Luu <em>et al.</em> 2017</a>). Mean <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-19.gif"/></noscript></span></span> values across all loci showed that CUF-O|CUF-H had a small value (0.0162, 99% bootstrap confidence interval 0.0159–0.0164) compared with CUF-O|CUF-L (0.0294, 99% bootstrap confidence interval 0.0290–0.0298]) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), reflecting the phenotypic separation we observed in the field.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/461/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of  between combinations of the pre- and postselection populations. Right panel is a zoomed-in version of the tails of the distributions. Elevated levels of the statistic are observed in the population pair (O-L) showing selection response in comparison with the pair showing no response (O-H)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-325782251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distributions of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-20&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; between combinations of the pre- and postselection populations. Right panel is a zoomed-in version of the tails of the distributions. Elevated levels of the statistic are observed in the population pair (O-L) showing selection response in comparison with the pair showing no response (O-H).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/461/F3.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/2/461/F3.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/461/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/461/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488357" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Distributions of <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/F3/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/F3/embed/mml-math-20.gif"/></noscript></span></span> between combinations of the pre- and postselection populations. Right panel is a zoomed-in version of the tails of the distributions. Elevated levels of the statistic are observed in the population pair (O-L) showing selection response in comparison with the pair showing no response (O-H).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">For the PCA method, we used a scree plot (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1/FileS1.pdf">File S1</a>) to show that the first two principal components were most related to the population structure observed in the selection experiment. PC1 effectively differentiated CUF-L from CUF-O and CUF-H, with PC2 separating CUF-O and CUF-H. Since the latter two populations were not phenotypically different, we expected markers most strongly separating the CUF-L population from the remaining two to be dormancy related (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1/FileS2.pdf">File S2</a>). Consequently, we only used the first principal component for our subsequent differentiation analysis.</p><p id="p-29">Based on z-scores computed for each marker locus, we identified 85 loci as potentially under selection because they exceeded a 99.9% empirical threshold (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Of these 85 loci, 79 were also in the 1% tail of CUF-O|CUF-L <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/461/embed/mml-math-21.gif"/></noscript></span></span> values, providing additional evidence that these loci could be associated with dormancy. Of the 85 loci, 22 had no valid alignments to <em>M. truncatula</em>, suggesting that they are alfalfa specific, as they did align to the CADL scaffolds. Positions and annotations of <em>M. truncatula</em> genes within 20 kb of the remaining 63 loci can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1/FileS3.txt">File S3</a>. Some of the genes in this small set have been previously implicated in desiccation or cold tolerance, including expansins, late embryogenesis abundant proteins, dehydrins, and heat shock proteins (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Oliver 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boudet <em>et al.</em> 2006</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Verdier <em>et al.</em> 2013</a>). That our results include genes that could be expected to be identified shows that selection mapping is a powerful way to identify loci that have a role in autumn dormancy. Although there is considerable evidence of high genome synteny between <em>M. truncatula</em> and <em>M. sativa</em> (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2014</a>), the relative significance of the <em>M. truncatula</em> genes identified here on the overall dormancy phenotype is not clear. Unlike alfalfa, <em>M. truncatula</em> is an annual species that grows in a Mediterranean habitat, where autumn dormancy would be neither needed nor expressed. Obviously, the loci showing frequency changes that were unique to alfalfa <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> may be expected to be significant, given the biology of perennial alfalfa.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/461/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plot of d (Mahalanobis distance) calculated using PC1 scores. Mapping positions are based on alignment to the M. truncatula reference genome. The synthetic chromosomes &#x201C;chr0&#x201D; and &#x201C;chr9&#x201D; represent markers that do not align to M. truncatula chromosomes but have valid alignments to both CADL and M. truncatula scaffolds (&#x201C;chr9&#x201D;) or CADL scaffolds only (&#x201C;chr0&#x201D;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-325782251" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plot of &lt;em&gt;d&lt;/em&gt; (Mahalanobis distance) calculated using PC1 scores. Mapping positions are based on alignment to the &lt;em&gt;M. truncatula&lt;/em&gt; reference genome. The synthetic chromosomes &#x201C;chr0&#x201D; and &#x201C;chr9&#x201D; represent markers that do not align to &lt;em&gt;M. truncatula&lt;/em&gt; chromosomes but have valid alignments to both CADL and &lt;em&gt;M. truncatula&lt;/em&gt; scaffolds (&#x201C;chr9&#x201D;) or CADL scaffolds only (&#x201C;chr0&#x201D;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/461/F4.medium.gif" width="440" height="143"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/2/461/F4.medium.gif" width="440" height="143"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/461/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/461/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488479" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Manhattan plot of <em>d</em> (Mahalanobis distance) calculated using PC1 scores. Mapping positions are based on alignment to the <em>M. truncatula</em> reference genome. The synthetic chromosomes “chr0” and “chr9” represent markers that do not align to <em>M. truncatula</em> chromosomes but have valid alignments to both CADL and <em>M. truncatula</em> scaffolds (“chr9”) or CADL scaffolds only (“chr0”).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Effective population size estimates agree with expectations from known origins of CUF 101</h3><p id="p-31">The ancestry of CUF 101 traces back to seed bulked from 91 parent plants in a field in El Centro, California (<a href="http://www.naaic.org/varietyaps/CUF101.html">http://www.naaic.org/varietyaps/CUF101.html</a>). These plants were selected for their superior disease resistance. The vast majority (89 plants) of these plants were derived, directly or indirectly, from “UC Cargo” (<a href="https://www.naaic.org/varietyaps/UCcargo.html">https://www.naaic.org/varietyaps/UCcargo.html</a>), most of whose parents trace to “UC Salton” (<a href="https://www.naaic.org/varietyaps/UCsalton.html">https://www.naaic.org/varietyaps/UCsalton.html</a>), which basically descends from a single genetically diverse germplasm. Population genetic theory dictates that the effective population size is usually smaller than the census size. In the presence of a selective force, as is the case for CUF 101, we expect the effective size to be even smaller.</p><p id="p-32">Using a Wright–Fisher model-based approach, we estimated the effective size for CUF 101 to be ∼15 autotetraploid individuals from the CUF-O|CUF-H population pair and ∼7 autotetraploid individuals from the CUF-O|CUF-L population pair. The former is likely a more realistic approximation as, unlike the latter, it is not confounded by a strong selection response. We justify using a Wright–Fisher model-based approach for inferring effective population size from autotetraploid frequency data based on recent extensions of Kingman’s coalescent (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kingman 1982</a>) to autotetraploids that demonstrate its robustness to tetrasomic inheritance (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arnold <em>et al.</em> 2012</a>) under the assumption that double-reduction and selfing are absent or rare.</p></div><div id="sec-16" class="subsection"><h3>Drift simulations</h3><p id="p-33">The loci identified from our PCA may have differentiated as a result of directional selection, and hence could be dormancy associated, or they could differ owing to genetic drift. To test whether frequency changes for these loci could be explained by drift, we conducted simulations of genetic drift for each of the candidate loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1/FileS4.pdf">File S4</a>). In the shorter (L) population, all 85 markers were found to have frequency changes greater than the calculated confidence intervals. In the taller (H) population, 10 out of the 85 markers were found to be have frequency changes greater than the calculated confidence intervals; however, these changes had similar directionality to the shorter population, indicating that although directional selection for autumn dormancy was a pervasive force in selection for more dormancy, a genetic influence other than selection for the focal trait and genetic drift may have been involved as well. We hypothesize that these 10 loci experienced a selective pressure separate from that intentionally applied based on autumn plant height. One explanation could be related to survival; Figure 1 in <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Cunningham <em>et al.</em> (1998)</a> shows increased survival for both CUF-H (7%) and CUF-L (93%) in comparison with CUF-O, a result not commented upon in that paper. Further, it is also conceivable that differential flowering time within the selected progeny during intercrossing in the greenhouse, as described in <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Cunningham <em>et al.</em> (1998)</a>, could have resulted in assortative mating and consequent allele frequency shifts unrelated to selection for dormancy. Additionally, three markers appeared to be fixed in the starting and less-dormant populations but nearing fixation in the opposite direction in the shorter population. Given that allele frequency estimates are simple ratios of read counts, it is likely that these markers have extremely low frequency in the former two populations.</p></div><div id="sec-17" class="subsection"><h3>Future directions</h3><p id="p-34">Selection mapping offers the unique opportunity to identify variants affecting a trait by following their accumulation over generations of selection. Further, population-based pooled sequencing makes the costs of experiments like this easily within reach of most laboratories. The plethora of recurrently selected populations available to the plant science community provides a unique resource on which to apply selection mapping to conduct genetic studies of various traits. Here, we used populations from one experiment to calculate population-specific allele frequency estimates and scan the genome for loci controlling autumn dormancy in alfalfa. We are currently evaluating similar changes in five other alfalfa populations divergently selected for autumn dormancy, giving us further power to detect changes across genetic backgrounds as well as to identify population-specific changes. By combining these results with those found in other experiments using biparental or GWAS populations, transcriptome analyses, or others, we will be able understand this critically important alfalfa trait.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">This research was funded by USDA-NIFA-AFRI Foundational Program grant number 2016-67013-24453 to E.C.B.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-36">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300099/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-37"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-01"><span class="received-label">Received </span>August 1, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-16"><span class="accepted-label">Accepted </span>November 16, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Munjal <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.1" data-doi="10.1534/genetics.112.140582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomblies</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extending coalescent theory to autotetraploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140582%26rft_id%253Dinfo%253Apmid%252F22714411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzEvMTk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.2" data-doi="10.1534/genetics.113.160655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genome-wide scan for evidence of selection in a maize population under long-term artificial selection for ear number.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160655%26rft_id%253Dinfo%253Apmid%252F24381334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.3" data-doi="10.2135/cropsci1979.0011183X001900010024x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bingham</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCoy</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Cultivated alfalfa at the diploid level: origin, reproductive stability, and yield of seed and forage.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBingham%26rft.auinit1%253DE.%2BT.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D97%26rft.epage%253D100%26rft.atitle%253DCultivated%2BAlfalfa%2Bat%2Bthe%2BDiploid%2BLevel%253A%2BOrigin%252C%2BReproductive%2BStability%252C%2Band%2BYield%2Bof%2BSeed%2Band%2BForage%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1979.0011183X001900010024x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1979.0011183X001900010024x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GQ30800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blondon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marie</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondorosi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genome size and base composition in Medicago sativa and M. truncatula species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBlondon%26rft.auinit1%253DF.%26rft.volume%253D37%26rft.issue%253D2%26rft.spage%253D264%26rft.epage%253D270%26rft.atitle%253DGenome%2Bsize%2Band%2Bbase%2Bcomposition%2Bin%2BMedicago%2Bsativa%2Band%2BM.%2Btruncatula%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F18470076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18470076&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.5" data-doi="10.1104/pp.105.074039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boudet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buitink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogniaux</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larré</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comparative analysis of the heat stable proteome of radicles of Medicago truncatula seeds during germination identifies late embryogenesis abundant proteins associated with desiccation tolerance.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1418</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.074039%26rft_id%253Dinfo%253Apmid%252F16461389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQwLzQvMTQxOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brouwer</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duke</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mapping genetic factors associated with winter hardiness, fall growth, and freezing injury in autotetraploid alfalfa.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1387</span>–<span class="cit-lpage">1396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBrouwer%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D1387%26rft.epage%253D1396%26rft.atitle%253DMapping%2BGenetic%2BFactors%2BAssociated%2Bwith%2BWinter%2BHardiness%252C%2BFall%2BGrowth%252C%2Band%2BFreezing%2BInjury%2Bin%2BAutotetraploid%2BAlfalfa%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000089612600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Reexamining the relationship between fall dormancy and winter hardiness in alfalfa.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">971</span>–<span class="cit-lpage">977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBrummer%26rft.auinit1%253DE.C.%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D971%26rft.epage%253D977%26rft.atitle%253DReexamining%2Bthe%2BRelationship%2Bbetween%2BFall%2BDormancy%2Band%2BWinter%2BHardiness%2Bin%2BAlfalfa%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000089410600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Busbice</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilsie</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Growth, winterhardiness, recovery after cutting and wilt resistance in f2 progenies of vernal x dupuits alfalfa crosses.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBusbice%26rft.auinit1%253DThad.%2BH.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253D429%26rft.epage%253D432%26rft.atitle%253DFall%2BGrowth%252C%2BWinterhardiness%252C%2BRecovery%2Bafter%2BCutting%2Band%2BWilt%2BResistance%2Bin%2BF2%2BProgenies%2Bof%2BVernal%2Bx%2BDuPuits%2BAlfalfa%2BCrosses%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czaban</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Studer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panitz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bendixen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome wide allele frequency fingerprints (gwaffs) of populations via genotyping by sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e57438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De57438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.10" data-doi="10.1016/S0065-2113(06)90006-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castonguay</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laberge</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volenec</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Alfalfa winter hardiness: a research retrospective and integrated perspective.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D90%26rft.spage%253D203%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2113%252806%252990006-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2113(06)90006-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volenec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teuber</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Plant survival and root and bud composition of alfalfa populations selected for contrasting fall dormancy.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">962</span>–<span class="cit-lpage">969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCunningham%26rft.auinit1%253DS.%2BM.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D962%26rft.epage%253D969%26rft.atitle%253DPlant%2BSurvival%2Band%2BRoot%2Band%2BBud%2BComposition%2Bof%2BAlfalfa%2BPopulations%2BSelected%2Bfor%2BContrasting%2BFall%2BDormancy%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000075054600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daday</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenham</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">Genetic studies on cold hardiness in Medicago sativa L.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DDADAY%26rft.auinit1%253DH.%26rft.volume%253D51%26rft.issue%253D6%26rft.spage%253D249%26rft.epage%253D255%26rft.atitle%253DGENETIC%2BSTUDIES%2BON%2BCOLD%2BHARDINESS%2BIN%2BMEDICAGO%2BSATIVA%2BL.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duforet-Frebourg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Detecting genomic signatures of natural selection with principal component analysis: application to the 1000 genomes data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D33%26rft.spage%253D1082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.14" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (gbs) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.15" data-doi="10.1111/mec.12360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foucaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cézard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimation of population allele frequencies from next-generation sequencing data: pool-<em>vs.</em> individual-based genotyping.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3766</span>–<span class="cit-lpage">3779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3766%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000321552800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.16" data-doi="10.1111/j.1439-0388.2011.00964.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using the genomic relationship matrix to predict the accuracy of genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D409%26rft.epage%253D421%26rft.atitle%253DUsing%2Bthe%2Bgenomic%2Brelationship%2Bmatrix%2Bto%2Bpredict%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00964.x%26rft_id%253Dinfo%253Apmid%252F22059574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00964.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059574&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez-Gil</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arranz</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Gámez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Application of selection mapping to identify genomic regions associated with dairy production in sheep.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e94623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De94623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.18" data-doi="10.1534/genetics.116.191197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jónás</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taus</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosiol</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futschik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Estimating the effective population size from temporal allele frequency changes in experimental evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.191197%26rft_id%253Dinfo%253Apmid%252F27542959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA0LzIvNzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.19" data-doi="10.1016/0304-4149(82)90011-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingman</span>  <span class="cit-name-given-names">J. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">The coalescent.</span> <abbr class="cit-jnl-abbrev">Stochastic Process. Appl.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStochastic%2BProcess.%2BAppl.%26rft.volume%253D13%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0304-4149%252882%252990011-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0304-4149(82)90011-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehman</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marble</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanford</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Registration of cuf 101 alfalfa1 (reg. no. 119).</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DLehman%26rft.auinit1%253DW.%2BF.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D398%26rft.epage%253D398%26rft.atitle%253DRegistration%2Bof%2BCuf%2B101%2BAlfalfa%2B%2528Reg.%2BNo.%2B119%2529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1983QQ71500054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levitt</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">The hardiness of plants.</span> <abbr class="cit-jnl-abbrev">Soil Sci.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoil%2BSci.%26rft.volume%253D82%26rft.spage%253D346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D195%26rft.atitle%253DDISTRIBUTION%2BOF%2BGENE%2BFREQUENCY%2BAS%2BA%2BTEST%2BOF%2BTHE%2BTHEORY%2BOF%2BTHE%2BSELECTIVE%2BNEUTRALITY%2BOF%2BPOLYMORPHISMS%26rft_id%253Dinfo%253Apmid%252F4711903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvMS8xNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-8.2.461.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aligning sequence reads, clone sequences and assembly contigs with bwa-mem.</span> <span class="cit-source">arXiv</span>: 1303.3997</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A saturated genetic linkage map of autotetraploid alfalfa (Medicago sativa L.) developed using genotyping-by-sequencing is highly syntenous with the Medicago truncatula genome.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1971</span>–<span class="cit-lpage">1979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alarcón-Zúñiga</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tahir</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mapping fall dormancy and winter injury in tetraploid alfalfa.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1995</span>–<span class="cit-lpage">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">pcadapt: an r package to perform genome scans for selection based on principal component analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.27" data-doi="10.1093/gbe/evu085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bost</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population-genetic inference from pooled-sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1210</span>–<span class="cit-lpage">1218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu085%26rft_id%253Dinfo%253Apmid%252F24787620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24787620&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.461.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paquin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duke</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Cold and heat tolerance</span>, pp. <span class="cit-fpage">259</span>–<span class="cit-lpage">302</span> in <span class="cit-source"><em>Alfalfa and Alfalfa Improvement</em>.</span> <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.29" data-doi="10.1186/s12859-015-0844-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melo</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartaula</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hale</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Gbs-snp-crop: a reference-optional pipeline for snp discovery and plant germplasm characterization using variable length, paired-end genotyping-by-sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0844-1%26rft_id%253Dinfo%253Apmid%252F26817711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0844-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26817711&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.461.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source"><em>An Introduction to Population Genetics: Theory and Applications</em>.</span> <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.461.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Lessons on dehydration tolerance from desiccation tolerant plants</span>, pp. <span class="cit-fpage">11</span>–<span class="cit-lpage">50</span> in <span class="cit-source"><em>Plant Desiccation Tolerance</em>.</span> <span class="cit-publ-name">Blackwell Publishing</span>, <span class="cit-publ-loc">Ames, IA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.32" data-doi="10.1371/journal.pgen.0020190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population structure and eigenanalysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De190%26rft.epage%253De190%26rft.atitle%253DPopulation%2Bstructure%2Band%2Beigenanalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020190%26rft_id%253Dinfo%253Apmid%252F17194218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194218&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.461.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em>.</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.34" data-doi="10.1371/journal.pone.0080422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rellstab</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zoller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tedder</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gugerli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Validation of snp allele frequencies determined by pooled next-generation sequencing in natural populations of a non-model plant species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e80422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De80422%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0080422%26rft_id%253Dinfo%253Apmid%252F24244686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0080422&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244686&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.35" data-doi="10.7717/peerj.2584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rognes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flouri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quince</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahé</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Vsearch: a versatile open source tool for metagenomics.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e2584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D4%26rft.spage%253De2584%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.2584%26rft_id%253Dinfo%253Apmid%252F27781170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.2584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27781170&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.36" data-doi="10.1007/s00122-012-1854-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakiroglu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman-Broyles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Story</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Patterns of linkage disequilibrium and association mapping in diploid alfalfa (m. sativa l.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSakiroglu%26rft.auinit1%253DM.%26rft.volume%253D125%26rft.issue%253D3%26rft.spage%253D577%26rft.epage%253D590%26rft.atitle%253DPatterns%2Bof%2Blinkage%2Bdisequilibrium%2Band%2Bassociation%2Bmapping%2Bin%2Bdiploid%2Balfalfa%2B%2528M.%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1854-2%26rft_id%253Dinfo%253Apmid%252F22476875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1854-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22476875&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.37" data-doi="10.1038/hdy.2014.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Versace</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franssen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Combining experimental evolution with next-generation sequencing: a powerful tool to study adaptation from standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.86%26rft_id%253Dinfo%253Apmid%252F25269380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25269380&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwab</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheaffer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The relationship between field winter injury and fall growth score for 251 alfalfa cultivars.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSchwab%26rft.auinit1%253DP.%2BM.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D418%26rft.epage%253D426%26rft.atitle%253DThe%2BRelationship%2Bbetween%2BField%2BWinter%2BInjury%2Band%2BFall%2BGrowth%2BScore%2Bfor%2B251%2BAlfalfa%2BCultivars%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1996UE90300034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Association of fall growth habit and winter survival in alfalfa.</span> <abbr class="cit-jnl-abbrev">Can. J. Plant Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">244</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BSci.%26rft.volume%253D41%26rft.spage%253D244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.40" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haigh</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by dna polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.42" data-doi="10.1186/1471-2164-15-349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnakumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bidwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An improved genome release (version mt4. 0) for the model legume Medicago truncatula.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-349%26rft_id%253Dinfo%253Apmid%252F24382143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24382143&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.43" data-doi="10.1146/annurev.arplant.50.1.571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomashow</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Plant cold acclimation: freezing tolerance genes and regulatory mechanisms.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DThomashow%26rft.auinit1%253DM.%2BF.%26rft.volume%253D50%26rft.issue%253D1%26rft.spage%253D571%26rft.epage%253D599%26rft.atitle%253DPLANT%2BCOLD%2BACCLIMATION%253A%2BFreezing%2BTolerance%2BGenes%2Band%2BRegulatory%2BMechanisms%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.50.1.571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.50.1.571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000081288400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.44" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.45" data-doi="10.1371/journal.pgen.1002316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaysse</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnakumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derrien</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axelsson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pielberg</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of genomic regions associated with phenotypic variation between dog breeds using selection mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVaysse%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002316%26rft.epage%253De1002316%26rft.atitle%253DIdentification%2Bof%2Bgenomic%2Bregions%2Bassociated%2Bwith%2Bphenotypic%2Bvariation%2Bbetween%2Bdog%2Bbreeds%2Busing%2Bselection%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002316%26rft_id%253Dinfo%253Apmid%252F22022279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22022279&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.46" data-doi="10.1104/pp.113.222380"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verdier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lalanne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelletier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres-Jerez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Righetti</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A regulatory network-based approach dissects late maturation processes related to the acquisition of desiccation tolerance and longevity of Medicago truncatula seeds.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.222380%26rft_id%253Dinfo%253Apmid%252F23929721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjMvMi83NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weishaar</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volenec</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Improving winter hardiness in nondormant alfalfa germplasm.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWeishaar%26rft.auinit1%253DM.%2BA.%26rft.volume%253D45%26rft.issue%253D1%26rft.spage%253D60%26rft.epage%253D65%26rft.atitle%253DImproving%2BWinter%2BHardiness%2Bin%2BNondormant%2BAlfalfa%2BGermplasm%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000226435300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.48" data-doi="10.1534/genetics.108.090118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolkman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection mapping of loci for quantitative disease resistance in a diverse maize population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090118%26rft_id%253Dinfo%253Apmid%252F18723892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNTgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A novel genetic framework for studying response to artificial selection.</span> <abbr class="cit-jnl-abbrev">Plant Genet. Resour.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenet.%2BResour.%26rft.volume%253D9%26rft.spage%253D281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.461.50" data-doi="10.1371/journal.pone.0041901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergland</span>  <span class="cit-name-given-names">A. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Empirical validation of pooled whole genome population re-sequencing in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e41901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZhu%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41901%26rft.epage%253De41901%26rft.atitle%253DEmpirical%2Bvalidation%2Bof%2Bpooled%2Bwhole%2Bgenome%2Bpopulation%2Bre-sequencing%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041901%26rft_id%253Dinfo%253Apmid%252F22848651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22848651&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F461.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/461.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/447" title="Direct Testing for Allele-Specific Expression Differences Between Conditions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/469" title="Identifying Genetic Differences Between Dongxiang Blue-Shelled and White Leghorn Chickens Using Sequencing Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/461" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mDQgGRGwDiEeeL5V7v16zGpFSzodU3udJinU6apfffY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/461" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/461" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/461" class="active" data-icon-position="" data-hide-link-title="0">Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (Medicago sativa)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488309" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-pr5U3IkjzaabvHicGa-X_xXBEo4gf2hKwWs1fFybpjM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/461.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/461" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-zG_NnGUJEDkjWiLD_btHtSc20gQUEIkGDiiUO8YnkZs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/461" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/461.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488309" id="node488309--2" data-pisa="ggg;8/2/461" data-pisa-master="ggg;g3.117.300099" data-apath="/ggg/8/2/461.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (<em>Medicago sativa</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gitanshu Munjal</span>, <span class="highwire-citation-author" data-delta="1">Jingjie Hao</span>, <span class="highwire-citation-author" data-delta="2">Larry R. Teuber</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3621-4516" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>E. Charles Brummer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">461-468;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300099 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488309--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gitanshu Munjal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gitanshu%2BMunjal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munjal%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGitanshu%2BMunjal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jingjie Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingjie%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJingjie%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Larry R. Teuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Larry%2BR.%2BTeuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuber%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALarry%2BR.%2BTeuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">E. Charles Brummer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=E.%2BCharles%2BBrummer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brummer%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AE.%2BCharles%2BBrummer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3621-4516" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for E. Charles Brummer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecbrummer@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">ecbrummer@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/461" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/461&amp;title=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/461&amp;title=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/461&amp;title=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/461&amp;text=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/461&amp;title=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/461&amp;t=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/461&amp;title=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/461&amp;title=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488309/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488309" id="node488309--3" data-pisa="ggg;8/2/461" data-pisa-master="ggg;g3.117.300099" data-apath="/ggg/8/2/461.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (<em>Medicago sativa</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gitanshu Munjal</span>, <span class="highwire-citation-author" data-delta="1">Jingjie Hao</span>, <span class="highwire-citation-author" data-delta="2">Larry R. Teuber</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3621-4516" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>E. Charles Brummer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">461-468;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300099 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488309--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gitanshu Munjal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gitanshu%2BMunjal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munjal%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGitanshu%2BMunjal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jingjie Hao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingjie%2BHao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJingjie%2BHao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Larry R. Teuber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Larry%2BR.%2BTeuber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuber%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALarry%2BR.%2BTeuber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">E. Charles Brummer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California-Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Breeding Center, University of California-Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=E.%2BCharles%2BBrummer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brummer%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AE.%2BCharles%2BBrummer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3621-4516" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for E. Charles Brummer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ecbrummer@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">ecbrummer@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=461&amp;rft.epage=468&amp;rft.atitle=Selection%20Mapping%20Identifies%20Loci%20Underpinning%20Autumn%20Dormancy%20in%20Alfalfa%20%28Medicago%20sativa%29&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Munjal%2C+Gitanshu&amp;rft.au=Hao%2C+Jingjie&amp;rft.au=Teuber%2C+Larry+R.&amp;rft.au=Brummer%2C+E.+Charles"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488309/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488309/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488309/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488309/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488309/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488309/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488309/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488309/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488309/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488309/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488309/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488309/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/461" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/461.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/461.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/461.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/461.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
