<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/385" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/385.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/385.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/385.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/385.atom" />
<meta name="HW.pisa" content="ggg;8/2/385" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome Assembly and Annotation of the Medicinal Plant Calotropis gigantea, a Producer of Anticancer and Antimalarial Cardenolides" />
<meta name="DC.Identifier" content="10.1534/g3.117.300331" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Hoopes et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Calotropis gigantea produces specialized secondary metabolites known as cardenolides, which have anticancer and antimalarial properties. Although transcriptomic studies have been conducted in other cardenolide-producing species, no nuclear genome assembly for an Asterid cardenolide-producing species has been reported to date. A high-quality de novo assembly was generated for C. gigantea , representing 157,284,427 bp with an N50 scaffold size of 805,959 bp, for which quality assessments indicated a near complete representation of the genic space. Transcriptome data in the form of RNA-sequencing libraries from a developmental tissue series was generated to aid the annotation and construction of a gene expression atlas. Using an ab initio and evidence-driven gene annotation pipeline, 18,197 high-confidence genes were annotated. Homologous and syntenic relationships between C. gigantea and other species within the Apocynaceae family confirmed previously identified evolutionary relationships, and suggest the emergence or loss of the specialized cardenolide metabolites after the divergence of the Apocynaceae subfamilies. The C. gigantea genome assembly, annotation, and RNA-sequencing data provide a novel resource to study the cardenolide biosynthesis pathway, especially for understanding the evolutionary origin of cardenolides and the engineering of cardenolide production in heterologous organisms for existing and novel pharmaceutical applications." />
<meta name="DC.Contributor" content="Genevieve M. Hoopes" />
<meta name="DC.Contributor" content="John P. Hamilton" />
<meta name="DC.Contributor" content="Jeongwoon Kim" />
<meta name="DC.Contributor" content="Dongyan Zhao" />
<meta name="DC.Contributor" content="Krystle Wiegert-Rininger" />
<meta name="DC.Contributor" content="Emily Crisovan" />
<meta name="DC.Contributor" content="C. Robin Buell" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Genome Assembly and Annotation of the Medicinal Plant Calotropis gigantea, a Producer of Anticancer and Antimalarial Cardenolides" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Calotropis gigantea&lt;/i&gt; produces specialized secondary metabolites known as cardenolides, which have anticancer and antimalarial properties. Although transcriptomic studies have been conducted in other cardenolide-producing species, no nuclear genome assembly for an Asterid cardenolide-producing species has been reported to date. A high-quality &lt;i&gt;de novo&lt;/i&gt; assembly was generated for &lt;i&gt;C. gigantea&lt;/i&gt;, representing 157,284,427 bp with an N50 scaffold size of 805,959 bp, for which quality assessments indicated a near complete representation of the genic space. Transcriptome data in the form of RNA-sequencing libraries from a developmental tissue series was generated to aid the annotation and construction of a gene expression atlas. Using an &lt;i&gt;ab initio&lt;/i&gt; and evidence-driven gene annotation pipeline, 18,197 high-confidence genes were annotated. Homologous and syntenic relationships between &lt;i&gt;C. gigantea&lt;/i&gt; and other species within the Apocynaceae family confirmed previously identified evolutionary relationships, and suggest the emergence or loss of the specialized cardenolide metabolites after the divergence of the Apocynaceae subfamilies. The &lt;i&gt;C. gigantea&lt;/i&gt; genome assembly, annotation, and RNA-sequencing data provide a novel resource to study the cardenolide biosynthesis pathway, especially for understanding the evolutionary origin of cardenolides and the engineering of cardenolide production in heterologous organisms for existing and novel pharmaceutical applications.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/385" />
<meta name="citation_id" content="8/2/385" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/385" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/385.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/385.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/385.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300331" />
<meta name="citation_pmid" content="29237703" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="385" />
<meta name="citation_lastpage" content="391" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Genevieve M. Hoopes" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="John P. Hamilton" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8682-5526" />
<meta name="citation_author" content="Jeongwoon Kim" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Dongyan Zhao" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Krystle Wiegert-Rininger" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="Emily Crisovan" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author" content="C. Robin Buell" />
<meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_email" content="buell@msu.edu" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. A. Agrawal;citation_author=G. Petschenka;citation_author=R. A. Bingham;citation_author=M. G. Weber;citation_author=S. Rasmann;citation_title=Toxic cardenolides: chemical ecology and coevolution of specialized plant-herbivore interactions.;citation_pages=28-45;citation_volume=194;citation_year=2012;citation_issue=1;citation_pmid=22292897;citation_doi=10.1111/j.1469-8137.2011.04049.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=The Amborella Genome and the Evolution of Flowering Plants;citation_pages=1241089-1241089;citation_volume=342;citation_year=2013;citation_issue=6165;citation_pmid=24357323;citation_doi=10.1126/science.1241089" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Bairoch;citation_author=R. Apweiler;citation_title=The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000;citation_pages=45-48;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592178;citation_doi=10.1093/nar/28.1.45" />
<meta name="citation_reference" content="citation_journal_title=Phytochemistry;citation_author=P. Bauer;citation_author=J. Munkert;citation_author=M. Brydziun;citation_author=E. Burda;citation_author=F. Müller-Uri;citation_title=Highly conserved progesterone 5β-reductase genes (P5βR) from 5β-cardenolide-free and 5β-cardenolide-producing angiosperms.;citation_pages=1495-1505;citation_volume=71;citation_year=2010;citation_pmid=20598327;citation_doi=10.1016/j.phytochem.2010.06.004" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=Regularized gene selection in cancer microarray meta-analysis.;citation_pages=1-1;citation_volume=10;citation_year=2009;citation_pmid=19118496;citation_doi=10.1186/1471-2105-10-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. S. Campbell;citation_author=M. Law;citation_author=C. Holt;citation_author=J. C. Stein;citation_author=G. D. Moghe;citation_title=MAKER-P: a tool kit for the rapid creation, management, and quality control of plant genome annotations.;citation_volume=164;citation_year=2014;citation_pmid=24306534;citation_doi=10.1104/pp.113.230144" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Med.;citation_author=E. W. C. Chan;citation_author=S. K. Wong;citation_author=H. T. Chan;citation_title=Apocynaceae species with antiproliferative and/or antiplasmodial properties: a review of ten genera.;citation_pages=269-284;citation_volume=14;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=F. Denoeud;citation_author=L. Carretero-Paulet;citation_author=A. Dereeper;citation_author=G. Droc;citation_author=R. Guyot;citation_title=The coffee genome provides insight into the convergent evolution of caffeine biosynthesis;citation_pages=1181-1184;citation_volume=345;citation_year=2014;citation_issue=6201;citation_pmid=25190796;citation_doi=10.1126/science.1255274" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. M. Emms;citation_author=S. Kelly;citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.;citation_pages=157;citation_volume=16;citation_year=2015;citation_pmid=26243257;citation_doi=10.1186/s13059-015-0721-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_author=S. R. Eddy;citation_author=J. Mistry;citation_title=The Pfam protein families database: towards a more sustainable future.;citation_volume=44;citation_year=2016;citation_pmid=26673716;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gnerre;citation_author=I. Maccallum;citation_author=D. Przybylski;citation_author=F. J. Ribeiro;citation_author=J. N. Burton;citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data.;citation_volume=108;citation_year=2011;citation_pmid=21187386;citation_doi=10.1073/pnas.1017351108" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. J. Haas;citation_author=A. L. Delcher;citation_author=S. M. Mount;citation_author=J. R. Wortman;citation_author=R. K. Smith;citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;citation_pages=5654-5666;citation_volume=31;citation_year=2003;citation_issue=19;citation_pmid=14500829;citation_doi=10.1093/nar/gkg770" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. A. Hardigan;citation_author=E. Crisovan;citation_author=J. P. Hamilton;citation_author=J. Kim;citation_author=P. Laimbeer;citation_title=Genome reduction uncovers a large dispensable genome and adaptive role for copy number variation in asexually propagated Solanum tuberosum.;citation_volume=28;citation_year=2016;citation_pmid=26772996;citation_doi=10.1105/tpc.15.00538" />
<meta name="citation_reference" content="citation_journal_title=Planta medica;citation_journal_abbrev=Planta Med;citation_author=V. Herl;citation_author=J. Frankenstein;citation_author=N. Meitinger;citation_author=F. Müller-Uri;citation_author=W. Kreis;citation_title=Delta 5-3beta-hydroxysteroid dehydrogenase (3 beta HSD) from Digitalis lanata. Heterologous expression and characterisation of the recombinant enzyme.;citation_pages=704-710;citation_volume=73;citation_year=2007;citation_issue=7;citation_pmid=17564944" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Itkin;citation_author=U. Heinig;citation_author=O. Tzfadia;citation_author=A. J. Bhide;citation_author=B. Shinde;citation_title=Biosynthesis of antinutritional alkaloids in Solanaceous crops is mediated by clustered genes.;citation_volume=341;citation_year=2013;citation_pmid=23788733;citation_doi=10.1126/science.1240230" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in structural biology;citation_journal_abbrev=Curr Opin Struct Biol;citation_author=J. Jurka;citation_title=Repeats in genomic DNA: mining and meaning.;citation_pages=333-337;citation_volume=8;citation_year=1998;citation_issue=3;citation_pmid=9666329;citation_doi=10.1016/S0959-440X(98)80067-5" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Kajitani;citation_author=K. Toshimoto;citation_author=H. Noguchi;citation_author=A. Toyoda;citation_author=Y. Ogura;citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.;citation_volume=24;citation_year=2014;citation_pmid=24755901;citation_doi=10.1101/gr.170720.113" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=F. Kellner;citation_author=J. Kim;citation_author=B. J. Clavijo;citation_author=J. P. Hamilton;citation_author=K. L. Childs;citation_title=Genome-guided investigation of plant natural product biosynthesis.;citation_pages=680-692;citation_volume=82;citation_year=2015;citation_pmid=25759247;citation_doi=10.1111/tpj.12827" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=K. Tamura;citation_title=MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets.;citation_volume=33;citation_year=2016;citation_pmid=27004904;citation_doi=10.1093/molbev/msw054" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Lamesch;citation_author=T. Z. Berardini;citation_author=D. Li;citation_author=D. Swarbreck;citation_author=C. Wilks;citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools.;citation_pages=1202-1210;citation_volume=40;citation_year=2012;citation_doi=10.1093/nar/gkr1090" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. M. Leggett;citation_author=B. J. Clavijo;citation_author=L. Clissold;citation_author=M. D. Clark;citation_author=M. Caccamo;citation_title=Next clip: an analysis and read preparation tool for nextera long mate pair libraries.;citation_volume=30;citation_year=2014;citation_pmid=24297520;citation_doi=10.1093/bioinformatics/btt702" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Marçais;citation_author=C. Kingsford;citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.;citation_volume=27;citation_year=2011;citation_pmid=21217122;citation_doi=10.1093/bioinformatics/btr011" />
<meta name="citation_reference" content="citation_journal_title=EMBnet.journal;citation_author=M. Martin;citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads.;citation_pages=10-12;citation_volume=17;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit : a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Planta Med.;citation_author=T. Mijatovic;citation_author=R. Kiss;citation_title=Cardiotonic STEROIDS-MEDIATED Na+/K+-ATPase targeting could circumvent various chemoresistance pathways.;citation_pages=189-198;citation_volume=79;citation_year=2013;citation_pmid=23412992;citation_doi=10.1055/s-0032-1328243" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Mistry;citation_author=R. D. Finn;citation_author=S. R. Eddy;citation_author=A. Bateman;citation_author=M. Punta;citation_title=Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions.;citation_volume=41;citation_year=2013;citation_pmid=23598997;citation_doi=10.1093/nar/gkt263" />
<meta name="citation_reference" content="citation_journal_title=Phytochemistry;citation_author=J. Munkert;citation_author=M. Ernst;citation_author=F. Müller-Uri;citation_author=W. Kreis;citation_title=Identification and stress-induced expression of three 3β- hydroxysteroid dehydrogenases from Erysimum crepidifolium Rchb. and their putative role in cardenolide biosynthesis.;citation_pages=26-33;citation_volume=100;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=Database resources of the National Center for Biotechnology Information.;citation_volume=44;citation_year=2016;citation_pmid=26615191;citation_doi=10.1093/nar/gkv1290" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=H. W. Nützmann;citation_author=A. Huang;citation_author=A. Osbourn;citation_title=Plant metabolic clusters - from genetics to genomics.;citation_pages=771-789;citation_volume=211;citation_year=2016;citation_pmid=27112429;citation_doi=10.1111/nph.13981" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. Pandey;citation_author=V. Swarnkar;citation_author=T. Pandey;citation_author=P. Srivastava;citation_author=S. Kanojiya;citation_title=Transcriptome and metabolite analysis reveal candidate genes of the cardiac glycoside biosynthetic pathway from Calotropis procera.;citation_pages=34464;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Proc. Indiana Acad. Sci.;citation_author=R. S. Raghavan;citation_title=Chromosome numbers in Indian medicinal plants.;citation_pages=294-298;citation_volume=45;citation_year=1957" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. S. M. Sabir;citation_author=R. K. Jansen;citation_author=D. Arasappan;citation_author=V. Calderon;citation_author=E. Noutahi;citation_title=The nuclear genome of Rhazya stricta and the evolution of alkaloid diversity in a medically relevant clade of Apocynaceae.;citation_pages=33782;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. A. Saghai-Maroof;citation_author=K. M. Soliman;citation_author=R. A. Jorgensen;citation_author=R. W. Allard;citation_title=Ribosomal DNA spacer-length polymorphisms in barley: mendelian inheritance, chromosomal location, and population dynamics.;citation_pages=8014-8018;citation_volume=81;citation_year=1984;citation_issue=24;citation_pmid=6096873;citation_doi=10.1073/pnas.81.24.8014" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Stanke;citation_author=O. Schöffmann;citation_author=B. Morgenstern;citation_author=S. Waack;citation_title=Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources.;citation_pages=62-62;citation_volume=7;citation_year=2006;citation_pmid=16469098;citation_doi=10.1186/1471-2105-7-62" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Wang;citation_author=H. Tang;citation_author=J. D. Debarry;citation_author=X. Tan;citation_author=J. Li;citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity.;citation_volume=40;citation_year=2012;citation_pmid=21908400;citation_doi=10.1093/nar/gkr648" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome Assembly and Annotation of the Medicinal Plant Calotropis gigantea, a Producer of Anticancer and Antimalarial Cardenolides" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Calotropis gigantea produces specialized secondary metabolites known as cardenolides, which have anticancer and antimalarial properties. Although transcriptomic studies have been conducted in other cardenolide-producing species, no nuclear genome assembly for an Asterid cardenolide-producing species has been reported to date. A high-quality de novo assembly was generated for C. gigantea , representing 157,284,427 bp with an N50 scaffold size of 805,959 bp, for which quality assessments indicated a near complete representation of the genic space. Transcriptome data in the form of RNA-sequencing libraries from a developmental tissue series was generated to aid the annotation and construction of a gene expression atlas. Using an ab initio and evidence-driven gene annotation pipeline, 18,197 high-confidence genes were annotated. Homologous and syntenic relationships between C. gigantea and other species within the Apocynaceae family confirmed previously identified evolutionary relationships, and suggest the emergence or loss of the specialized cardenolide metabolites after the divergence of the Apocynaceae subfamilies. The C. gigantea genome assembly, annotation, and RNA-sequencing data provide a novel resource to study the cardenolide biosynthesis pathway, especially for understanding the evolutionary origin of cardenolides and the engineering of cardenolide production in heterologous organisms for existing and novel pharmaceutical applications." />
<meta name="og-title" property="og:title" content="Genome Assembly and Annotation of the Medicinal Plant Calotropis gigantea, a Producer of Anticancer and Antimalarial Cardenolides" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/385" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Calotropis gigantea produces specialized secondary metabolites known as cardenolides, which have anticancer and antimalarial properties. Although transcriptomic studies have been conducted in other cardenolide-producing species, no nuclear genome assembly for an Asterid cardenolide-producing species has been reported to date. A high-quality de novo assembly was generated for C. gigantea , representing 157,284,427 bp with an N50 scaffold size of 805,959 bp, for which quality assessments indicated a near complete representation of the genic space. Transcriptome data in the form of RNA-sequencing libraries from a developmental tissue series was generated to aid the annotation and construction of a gene expression atlas. Using an ab initio and evidence-driven gene annotation pipeline, 18,197 high-confidence genes were annotated. Homologous and syntenic relationships between C. gigantea and other species within the Apocynaceae family confirmed previously identified evolutionary relationships, and suggest the emergence or loss of the specialized cardenolide metabolites after the divergence of the Apocynaceae subfamilies. The C. gigantea genome assembly, annotation, and RNA-sequencing data provide a novel resource to study the cardenolide biosynthesis pathway, especially for understanding the evolutionary origin of cardenolides and the engineering of cardenolide production in heterologous organisms for existing and novel pharmaceutical applications." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Genome Assembly and Annotation of the Medicinal Plant Calotropis gigantea, a Producer of Anticancer and Antimalarial Cardenolides | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3B5GIeL-QVTjqdSmHUtZmZOoteIHtr_RLEjPE-DISu8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488299","apath":"\/ggg\/8\/2\/385.atom","pisa":"ggg;8\/2\/385","ac":{"\/ggg\/8\/2\/385.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/385.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/385"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/385"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/385":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yotrsg6eou1gndFm8IlzEIR3x9ok21JCfh8bZ726yknKEgpmOoYwkyJ5FSrl7IsKQ4pLELoSgTVTiv6FXYIr8tIhoAyAC73Jp2sgdiyB3ABMgU6zrhQo0zUCAZJzTlT58SyLu7ZDb9hSWHhEnTn+bPfbpQ07a9wGhLd2Pe834mB3pC0Iv18m19fxXzBIcltrazBEuoFCy3xSOJkOvOBccvu\\\\\\\/\\\\\\\/0sx2NipADMhXjDnvOAqinWd25pBAAnIFa4Yw9uY\\\\\\\/W+RHcTqG\\\\\\\/qsgfWmxRxo4ZLO3dXHtPqkUfmBqcz5FrMTBtdM6NENh6445\\\\\\\/rjwa8vTR8tgBSK3WiRcpJfMm5n0i5NSYOUAMNNKVKjRpip7VZaC84liJGJK0kZpDTWusixNy70GHo+0PoIJ4JVQG97ieyb+cH+trTUVU1G7TipSOyJcsQ9M1yxn1Nf4431fSs1CLneRm10W2NVLgXTlWoLP\\\\\\\/ISCbzw4M1uY51s66SHcqWDDAgiTMfv0BU2uGitBCTdKoOSt3wJ5OSFqnGjEgWvAo0GTPcM6vw1o0f64lOWnTMAS61X7cfmDqdSufjZVzMpeMv6hunnA146yD2Vy7UqCjyd9RucEtmsW7GKwQOFZhWqLgCSsd65987X7rNm8wdY3A81+lwM4dnxxbaD0PN7b3wXEAfaQoyWJIEexcgG2LCxKP+aU8LNwjWYTc8vNm3MhR9v5WySAujqLcJajWa5jMz3bC+3UI54WPCr+2FKF2ixEO2cd3SquMQPsUQVlAdHKT7Q3v80LEQXhYQVIIhkm2J8eShnT03O+QgGe1SRqS68FQcZPIpvvDCodYInlFjP2JC1rCOxTqMs+yVggDCOJJ0qWYfqUhUh5agj52k5EakWS+jVwaQr08s\\\\\\\/yWoT8iDurE5gjLYetoPrQ25UQ7pnaXnq3yil5vOYc\\\\\\\/xxINuXTsyXdBB8PtY1ufLqgfkOj\\\\\\\/eWt3rXEqCQ0YXg5vIVRY7jrvMNlobPQP8EmTJJ4SorLSXtalavQS7pHyztIsp7ZoIK5uOBQ7PioTyrlugNVPLZ7nL955D26OPakZiEUlh6VGW9A7G7AfapEKKhjCI+qSimMlO8fVSFak3uc7MUmqHUkAEi4Y4KSNqaLf9TEdhYmxl66Ivrfghm86NoDxBRABsKjQyTp4YNUcd1Ob0EXe2QhGCpVZokQOaUkkkTF3ygESYumMh8COOEprPotI8bdWMAeOSBok3D7jcyLrpsgnmSdTT2cF8vM+OhIxTY57JtjjU6whRMh2RjtdpSH6sLXx9Y\\\u0022,\\\u0022iv\\\u0022:\\\u0022IE7ijAuElAuLH111bo76MQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002262451874e52d1da8eb43705c6cffa08d6abf82f6c48672abd5a0f49370a588b9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oCyJXbse8RyYjcBel2ISxXt59RamKy7uAKxgAbcfTr1G2105KlM+FLG1pVUZonEc5fBdU1tw8CN4P68GZ9X8DvebTFQFPE7+kADU\\\\\\\/KnxhyNRw+1hmMMjc+WAi+nkxowKmysTTNacrEmPXXAP2YWI8B28M20RGuGaKXAVe4Go75M6yQ6FH4Vfs+8euPTCev3MUAXtbxDrAS1jldUkFVcV4A5THyIP1wYmJ9r3VqmzgGX+g5j0cpPFb8c2qnBnZc4ryneVtok23BksHHbg6fDjRQX5fc8uJSewqrJvgBb7OA\\\\\\\/OSr6U1pBWFRBBGA\\\\\\\/SOB2nvA7gaEDilL8DqoNYaU3pqBUS4O8vR+poeXmv2PQaoB4utO3QXgZ4v\\\\\\\/H8SB2GrDHRF1YnNmy5qh+hAKGJY6QIVWZq0Mzc\\\\\\\/C7Qi+IxcArJdOe4GTzWgFYkGcyJrVUbJ4\\\\\\\/l0\\\\\\\/AXBZqXPrfac9c+da952NtWv\\\\\\\/Wb7cebT3d1t2xaiPWyNYOtmy1hdxC3tOG+pyXJGw+Y07v2YP1D8L7R7L87InGpBca9LzF2bNMr0h52E0DxuaVqhviQZcohZXsPPCobp5Yqn6zU\\\\\\\/Z35wKZgqxhXfqcwLDq4rE\\\\\\\/SN0RTl7DdPsvhT+8yCmCi28DwJSHhXx9rqOuXXsSsuVyXEcxrOEW\\\\\\\/7NCMhuYAo+5xa29ir76I6SsiIJPzzPRSQcrmov91IhLiG1GOgOn5JFOC3DZQz\\\\\\\/HUKe7QIaONF1bLGU8mvjPN818eBJ1\\\\\\\/g3nYj20Hhha7JLaHtpl9eml6oowKxSkcwZgtSZu1w+RfAkhvS9OeWKVIOIASulyGZOCdfMTZhJ3C1XCfnSvMWBd4IEF6nQh3YS3DXhKHaFupo31WIfrXNJceqij43z2UoakZtOIw4euA3G0f1P+VVPNLyaZP+XjVZt+eXQvhliUPQrD3KYIUTpC+Ua+S1UsY1w9Kf3M4UsYLfrA6rDF7qm4xmsyR8YHNt3AjUpb9VFdqgz1W0yxxKcqe66SXesAZH0tt0M\\\\\\\/IG1MEMtWForpP77XODVMcr0OqHWSTUDokOEJYxGR4C1SfMgZVIbTLmVbQClaaC30FJhY4KPOY4lbTF\\\\\\\/T5jrjwHfhyF2CUG\\\\\\\/JA1fbYik3eTCAkHlqFA+JtPea67j02Ukc639LJAHZ4REpyfPe7XI80XgnYhfLN0wQex7E1wHGb8eIIJTQP1Ps1HRiriFzq0\\\\\\\/hVaurnY7aRZX1hVabuPs0cVUTjv3kKBpWvDGXqn4H0+hoSttbPo2+LdXCYQ3VXx3wEnMfKGvLus\\\\\\\/rDvsY2MqgHaj8AeP\\\\\\\/MpGCnybQiXxanZSv02KyvSWqqpRonVVeLU78hDv5ZFziG7M7qJAgI5jKO1yRNJs46Q\\\\\\\/eEQZrweg2gLDZ8vZ1DTBTiRbti2GrR\\\\\\\/vrq9pop47BG3hHtYOAleCrEvPyWFIe+SbUqnJtxCoBsJihZEQOkl0yy3956sZmfRt1jU\\\\\\\/fxiMmhZ1BC0\\\\\\\/2K288pIlsUqIlEVzMt5t3TD5sRwVJHNpym6BtbZPJG9F+0Pmyk\\\\\\\/6qFTlUNgO6B9ppRtPQbctnNZfIcdFdf+GVl4AdT30H\\\\\\\/MK4w+2dWBWfX0gAR9ycE7SyxsFR5be3Gs225tAchUykEABTELpbmv+Z3hh\\\\\\\/ykr+NhM\\\\\\\/10nGEOLUHfmeWIjHv7j+jiGbGzzl\\\\\\\/rQJjiZ9+pty1BZLvpCB0vZol9nygFeVyWroXyMn98KdOvyBvkzDwVnA3FIrtq130nnKy4Tw4Oxfx9zpWaaSu1yo42YxPoOipYYoeffyn0ZWm200gY7coQgntMHYe0vfzA6SPY6sUHItjBjByGuwu+iFEWEiUkfgT2tt1Pg1yeVEp4WAZZoJ9rZ6bCedAtv0e\\\\\\\/2YzepMLiqbnBwTfCwFay+t3PHIFtMVlw4UNN0djFhJeGIF4aPTyrxTGU7jhUeNbXWVrCfOibWCyKabcQbIeqXP1D+pA8tcKsJ3g1EW0gvBwMecOv2Dp833Q4a2cfNq\\\\\\\/xSg+bhHUvOvvqavvP+\\\\\\\/2ZREKTHidITBnicO8OgoCw0AbaG2kkKuRquAUGVS+5xyYZkDrdJ6G3jGU8wGyFB9jQjTx21yR8Mp0fmVxxa8hI\\\\\\\/bBoqAHyaYWIBJfqIQrkPxEVwY\\\\\\\/xXlnSLVISnJPAI6gameGwCyUz1ZUeM7o5KC1FxXOlLky+e5h1\\\\\\\/C3DIhIt4K7Ycprs7NepBjFv+tEjP3u3zm8OR1ZrVlBRMxObNpeBlQcJVsWBY3kSyEK4PBN5qNRaMFRm+wlcqBaxZkMUg16CPxMnvzZ9hvTHfCx0u\\\\\\\/lW7QBkT8LLShzC\\\\\\\/N\\\\\\\/sncrtlurAlzIxx2VkkxPB4C\\\\\\\/JgGinxx4FEwJavJIhK1m8CkLUgb6E3au7r64TOHRpX9zhwFPbQRAZqOewjVboPVC1IYLZV5ZoPJU9zo2qNlM\\\\\\\/M+ronVGr2WKHgP3DWbx7DWkkEf1uud+kI9SjtBsHyEU0w1hAtU+xsuwch7eDyYDKi4us5g6zn\\\\\\\/gMsJdpdMW\\\\\\\/V6EnlbPM1kHjtH58NgpgsMyZ9VuiazCi9SWrPUjtwW9XAgC\\\\\\\/eWk40z4nE27RJtNmOGW\\\\\\\/apuaYTJfBbts1e9QIzMxwhSLKF\\\\\\\/oprULMI4D84mh012y07BgWIaqo5f8nZ6NFGQCKM13LfX+Z9E4a0GYtLPmnmMInh6KAxGIq19MWFq1kkoDB3c\\\\\\\/Kf73OjELHFbfTLKoVqh43to64+Z+mGN72ej\\\\\\\/RBLLKkEIX6mV7YSwWmszEAQ67Isd3ESQuzRlq+GmHTp3Lybol\\\\\\\/EvyRlGKOhx\\\\\\\/koleuSVapogH7sW8fBxGK3TrL5O9rU\\\\\\\/Q717eB60+0kzgTw0EfqEjdVD6q75zFjm\\\\\\\/dZX3Cx0OIOLPMkGuVVNjcWCO2qca6eEpMtv9LU0Wmah2qUUwhW++dgb+78iCdo6tUCL8YmzBrt2TeLBy4E+dhYcWs7xPSAD6\\\\\\\/S5OJMCRJ7sCSHR4UJk\\\\\\\/ojLYFyuIJiIkgxlLp2fHQ3xnOpcRHcsZSAdndzLb0h0GEhMXoKEpQejRJ7dYcEqqklwuLAALIr7uWsL+15U1RmUMslhTx1q8tCzomcGm+Z12P+IVXqqV6QrzAzuf28tSo99++tX+SDTF7fHWh1MlnbswFHHdiUGZV53Xwpo57\\\\\\\/teJO+0GbCePiepguvBW0qq4MTGpRr0zBiuNldYt56eUt6jakstXvmCL77tsSf8v5yEQDKyGYn9GWK698IyQ\\\\\\\/Et6MH8kz7L4t9fakNJ+jEgvmqqEJJ6OSNkqOx0Ut2bhnoAVnhvETpS\\\\\\\/aGbNvRxslAN5ta15UHkrGbAbk9pZ+6W22fXwAz0Sfq1oUKRQzDs5ZPiToWRKSlYQZ\\\\\\\/rCCTw5xsbXW5+h+AK5fq\\\\\\\/MFEFhWO53TNiJkTS6aR62snsTC2fKB7tQrPBeNwHYJFyhZMuKe6yR64YpJ8048PLWebU6l3BHc0VMDyzRJz5QZoanKc2V44WYky+alYyzSxs+B4BWDFQsyhYTMv9Sa5stDMUhpbvebfrBxE56YmS\\\\\\\/CObU3xslvC9ciXUr7YPLOPjOKGWRYAK8pAsfW3wXQCMstA3JicOhTV\\\\\\\/r5AA3H8gAjebvFQDiJgf1GmA3gwCQHs64C04fmNHbHx7QKhHRvXnzqiMz6br4glr313GGSderGFx3Hq5RtS06t2S8DCY1x5\\\\\\\/7IkLRKS\\\\\\\/r9YCtbs+CZNFJ7ivwqi5efdYm1wD91bFuhCdkM2heU7P7kxan1Km6MytFRvcKpF8re5Wnsy5QzgPC\\\\\\\/DN2eNBGdsma\\\\\\\/aneSokjvp12gatjwHZSKhyGfB\\\\\\\/A+XFnZ3CkUPVdvk4MzLcsz7MrCnPz1+j6ZNM+CY6fK9kbnT6FyU2nRIMhNY9V0NDj\\\\\\\/PucsaOZYQa0hYLOO91R\\\\\\\/KWY5zO1m3uhZ3IOeHc9vr97S5T6UhdeepAnrTP\\\\\\\/1ZwuHvx9dAe3vXDfgy94ZPlCHXfDUCpFtFVFJZV1n7926dWqv2BmiHgErvmoSDfQ\\\\\\\/GCh94fjf6zhROkU4\\\\\\\/n9CVSuoEm\\\\\\\/gg8cY\\\\\\\/TZVSE\\\\\\\/UdriQdReU6yhrNtNYJ6RZXeVZ68M0LMH6UAT\\\\\\\/zERdmucWRCtm71dYH6KV9m2S3BkPMH0kpUhcyky7cZ24l+5a5EjeJ3DRA\\\\\\\/1lZeMIgDd7db5Ubv\\\\\\\/cdst6lGIRhXORBHo6avrhNicz6UzcPzWlU\\\\\\\/tcT\\\\\\\/eRBBY0L0md8Sp7ik0JOoBpk91TbGC9NjfE9ocFre3rzUvIoXdrPoFuIC7V1hmchLdBCbIQhhnd+haXDyYmiv3fa5SrP4ZWc4T1fMsK\\\\\\\/Fr80m9\\\\\\\/WmSc6UHbBAPFhKYVedn9Sjat2aMi7Uwz\\\\\\\/o6mKY1LN9\\\\\\\/2HmnrAhgptDfGFeKv7lK9TU3TCfK5c6vpIiTCverYzbaf238uA1oTbSjxna1BlSVuGnDpCv0PVgb\\\\\\\/8B4gFMQaw59Dt\\\\\\\/mUGc55q7JkerzTPFvEdMtLbmJfvUDrPigKKXrYyGpEcebQOGfS\\\\\\\/r8cDS8iqO6JocM1Of4McnZ5t\\\\\\\/eOe7NdmfLEgmY96bIJuSdDmc0Qc5kWc7c9qswisgM70DaHAydspGcImY0n\\\\\\\/A9Os0n8W\\\\\\\/nNtl04US9pTDEHkSv53Nx21QdEzCcuG3w7ELkrfGnvgFCtR4vKIK03H0qU16Dx3FbmPqGNQYeVn6up0u1XTe6khWx9GxrN\\\\\\\/YMMsEn5HasWSF31H+DGh7AIg9lCiEih+FCuy2LrlEs35ZdhUdVSPSHHTKijEnPqOv5o5PptkZrbaG2Kjn1Yyo5VIKYm9v\\\\\\\/bTPlLcxoOo2+EjN6qySGAxQV+ZkDDb6JKy49RsKA07XHCXN9Q\\\\\\\/SNVwYzYW77L6fOAblhnPc6Y\\\\\\\/88rR7IEJyz+yEavQOc9mZs1PHCF2i\\\\\\\/ricOaiCj3Q9c38fGLUaWQ5tkR8a+mpwcUNk+s2dAjBt72AwnHu3mMOgjROsIPeF2Mebv9JEUn\\\\\\\/BHX9Jz1EWFgoRES9OHugYlgyelC0ayHRT2Ol2VH\\\\\\\/zWXbBhiZ87PQohfprQH69DTbCeOdEq2shE6lO23TkVMCFcPaJ1dEgdmjYThcBjBfZ5Sb\\\\\\\/3hsynKfX86ZMu2vjRnh2wjWxN0J3j0e7RF2CR3wBhBc\\\\\\\/Bxhke8dx8q74ZvPDXNvGkVq5wrn+iuSfKfKKHLVzRV5sgeBtdqnF4oZ\\\\\\\/jyv9wpgHZNHMlmTI4KfR5LL2evRDPakqK\\\\\\\/00j52fmMnAJrW3ivERgv2i2oLRmey3NjkVRByybFxSUOBMBk3gl8PnpJUEVBBk05uhka5oU36Iu5u4mzU5A1lNVLhTiZWojdWmi2Jta6iddlyOmQK5G1RDUlmHL+oYwYyUy76DMLAmyH9BxR1+aPsOosIPf2wrYObOBArAF\\\\\\\/Osnr2o0z16srn11AiJMEmvGbNUe5\\\\\\\/CM3bsPGCVIlsWCbHIFWm0pHVW2ITZaJjOxoMN\\\\\\\/W91zfeExFwf3XlVMxcnHCtZ78+2YwYyCBPuySqrKoC42LitxIhEhKB8QAgRkmhzSPuzZ6kAnnaKv+HYGH6JQxe5lVU5XBV1u9hNkiaca2EmINPPoTKctqPDsXkaquqnEjKn8U0Ch\\\\\\\/zKED2Gr5sQqCeb3hwsTIYhzSX4u9efrvabwCQIe6+cZ0xN4JfYMmtbCcjsGIOQ6CX2bmjAGyiODm\\\\\\\/RHb9zR9Nju50\\\\\\\/ZXxMY\\\\\\\/hMUQOymAsxHDdvitXvrdIRvDdFjPXjEzno41K6MGWjBRyPW4jegPkK4OVw\\\\\\\/qqUdkYcntQgd0\\\\\\\/uUFSvKgxGyUWzRWlunIXbNwyWP\\\\\\\/QaT5E2kQAoblFSk+RTlQw1SvkPsRrbIcOQOXS5\\\\\\\/ahEqWKAhxZE3CA2YlKOSiutCcnRZc3XjTvIHB6OFkc6CtaFDKYuVrx9+RZSKU9l10z4x6q8SP93VHOpIH73dp0dW5C\\\\\\\/vX2moiS+8maQZLi9upe6p8DnsnRju5iXg9glQbECM6YTYo94NtqwM4MvXZx\\\\\\\/3bzXOSujhnM2db4NKFjhzFqwCix5mg2czOnoEW6BOZOwuDjh1uX+3bN2eHjhVHZwtW4+6KXDxovaiQ5BHFfalc5RlDfTnEzWA+uNZJ\\\\\\\/9a9U1hjEFCipKtuJTs2yWIzKcGGxyetKdTkvWuQZi3XMMNwSmhapw1jBljt2+IIhHOmdb30hF4\\\\\\\/7EQnYLyVkpPIX+4stAJyDZVy\\\\\\\/jKbJ+RlwnyTIs6Q4b8xRaftG5lgopBSFjyBDbrU2e+6z1KVZHF6l5lrPc5DJ85tTshSfkF1kQkUiiivCDq6NScC3SOixlRb9c1uzOJ3RaRyZMf\\\\\\\/I4sHiLKQuvEVqx5bU5+r6JYXyJ+M9167wvXunJUsYIf2D1bb0OpHEPTRrQfyRHvZ1Tbx65vSo+wwCitV\\\\\\\/9jqe5h+cEJVShY1fnoZ4XgXmCW1kOMspPTMhOeRowrVafT2S\\\\\\\/wKjLFzpOXQckw5j\\\\\\\/mw++Dt4FiXCBBtuKE5xFzj5MP9qVBERkQWXq47bMottAtV1zta3xFStGvyw6myMVPetXjZ04wCLQArvdRVLkWfWsSuqzWo8oF9sYvPfNHgXt9QAUAbTrFYfxeh6j8chtWhUOgIy4asVoGIACGSSRtGAn7tysjq40IwxNdysgRF4NTGsi65li8jwCkPNn+LOxawstdYvBKionaO5yDH+0JpQCUlBD2zZAssM4fU0w7uOwSu5D6ol27x\\\\\\\/IfXFk5EOgWxcbAW7JlXu5loJv9+zFdp3tVJu1jBXpivPbhc9nr1UKeNQzj6WgSOk+R82bsM99n6tKwG6OODW+GXa41obwkYKdia+qMLHc0LmlRzME3Hec2u4HYrph4gB9OCZ8MVss18u9PelSGVdda4UxJ2b71QH1OYHprLaAhtTDyaocYVdeKQBfNFuhkizWKiC46Qv7kOrYMbIjiS\\\\\\\/J9tt9JnkUo2HKVP3MsCahm2fliv9W0AdlhOKLPxNFr9pCa+E23XdBf3\\\\\\\/2R2EURzC037mYdLpDAkYfkeJaHigc08GHQmX\\\\\\\/Z+SeZKnjTQ6eaSl1lshmTfmRp9GjQ0A6jYCody6cUmYqRucJENEz3AfqWE9++dmAe2BGBVLiS4D5\\\\\\\/3Opl5c5OhdKh0N9X8lEkyLIajkSOhVP5e4MeJ1Hm0w9g3yxEEBJ3qzqGpMxJV97oaA8DulTsvczM5diEkmO53slGvqbEZjvy+dLY1ZLZCElvWoATU2gilsD2ByNKyEVT8F4IF++bw0Nm3Uwc48XAx9ysnSn+dwuOzAVZQuAjYT5+lIfXAZT0dPTVDepoOjybuSU+p2TG0\\\\\\\/jH98hcbF31MpzDpFJyesqizctz77h7c0Oyc2xitKazCfMQSlSxvF6XxCuWKVqwXaq3n1uBvnlUzpFBCjoxSrKRRkjHksebK5pF5VB5ZzpLGwRhYXt\\\\\\\/i8afIosKLy1J32yaiJg\\\\\\\/SlBYuiKXV4520hrnulWz1H6NbMfjZnMjBqY2PciVwyhO3l2MD9Op6CbrOaj9mHs7X1RKNcVIgKxXYJBJsSAqDXCvgpdX4Q\\\\\\\/iLslr8tZmCgqKe1A9\\\\\\\/IqOwpnGoAJp7rwCtxGWuvIZ4WdwUdeRxwg8hohTly1qhV4ntIr9KH+\\\\\\\/eygobiIkQ9eOCR9mX+Q6HMPczgvImUFnAXKs3nd0RySA+koUQ5reGzO0nyh\\\\\\\/\\\\\\\/L8Et3tn5Iwy25d+K2NPsIKhZFk6SWPf7bYqdu6n\\\\\\\/\\\\\\\/ieB4EKglu32wT6tRbn8RqVjkOytVTWIe8e+5VKvRrVXMy\\\\\\\/fwJm7nlO07mQSD1MLx8ZD6o53\\\\\\\/JHc+E086ISMHCrwBBpcXJWDGIaZ4F0qblBEdd05HL2bOrkZI39qyq7QFmHERaIV21R0s5swQC7PeqiJzMqAvVWfBs9NSXFA\\\\\\\/SbisMcWgOrmhCygAKZIKy81WnX6LUGO\\\\\\\/wqvBD4GBPFg7H0GbxB13bEEHGW\\\\\\\/3VLgrrLss2Tg9Wh9r3fiSsbP1ZluxM5UgRh5xFfbQd9wfEKkBmH1kVA+llsYs21avGn15G6391qNP9eorOZXFFZFH5Yby4jILyQIcrd84nJAwYV\\\\\\\/VW+1K2SjC40nyX3LOspaP4YrNYzvcZfOSAGdEgddHpiqK4nPSCCOb1oaJXar9gf+AEiTBIqe5x31JqKkld\\\\\\\/iYvm9nSAD4e5tIGn34I5tJYJZv72+QlhBm9GDPxqk4ijgfChfD6k2FXqN8+JBetT\\\\\\\/nwwXRPlNKiykvBbS1\\\\\\\/NPwzG9+fdBKVG54gm1mn\\\\\\\/rD4TrbMNLPH73qNYxcS\\\\\\\/7F5XNcXKhF3F92+S0sA3Boo8fENOa1eKPNET\\\\\\\/h4OK5pa5LRaa76NHX6J502wixjhgfEomcYZqPIV5qKOF2ty7oT0tzoGlkq5eW6SymIKUvxMvP7OVYMKwYDvwran+winPz7vuUXz4ojseGvvKVI1KCV0AZmUIilRSwERJVVBEkbo89\\\\\\\/xKwMsajILIyb1vmevXiK9ilbtTXpx2eUZ8Xe8PHGXnCIUXhfyXBu7JstBuuPuRk6ehokCh3k4vowKzKJeUEyf5RIlpozPgOmrqmDCv7ugQzv+In8OcjcQFTcTBm4jiDaZpENbHqZtt2jUMO4FBTdolaRddHF7F00OkQg343Zya+6O4C36YEzt7zswcDOmAuwUGge6fkYGGrHSVevaET2XcXRUStU\\\\\\\/1GQvongg5rjCrZmQPNa61VJaLn9sZy7LH7EoAYRA\\\\\\\/i+lybbyzd5mpcC9sQyM5tnIw6q2NKn5obxZHnQHXTt8k+tqzjTpF+Zv4XfnEhYEQYlVEuvQwA+PO7Ls3Q6eYIkFW2Wkli\\\\\\\/UF5VzOE9J929ktzy7Bnco+bDTBEM2gFKjOX0YpPzzQvYGx35gskn5\\\\\\\/Yj+5khqHtJ\\\\\\\/PV2oWLYw7FyK3MBBfIuog6Y8J8fmPHu35LluePADwGoul9wzECBSda1MaV8NyvpwA5ypvqK7iUbXBG3qiMtESP62HoAbs0xqykjN7Kcchs4E+NIICm9fIlSH5KULoeUe8v\\\\\\\/R7u2en9FhULXwbj\\\\\\\/5sme+Z\\\\\\\/LIXhsXI0WoRpY+4QtmHvu\\\\\\\/h1BNAemRuomApMvZtxRvMXqPff6Q07y7hmHhLnefX5u2oDK2flWzBQaw1gQZE99DM3jOXejhSB9JqAWsRihw2RrnER51aAe3dr2Kkm1L2ossf4OcOkQpK3NBXIm+JA+CYuWnXrYKLwGEHA2IJVNjTA68h1JwG2C85asuMyreQERWfPZ08Il8uxHUxRdJep+abAvb1+zmkD2hprFU4DiMlrmH0A0oF1LI+WLnBnrEZV3AiW61\\\\\\\/kZVG8FtnQZpXtWhN6Vyl+keUQSSi\\\\\\\/cKXo7Ci2uZGuXkjlWukSyptvchzh2H+aF46gyIFg51lbTPLjjiNMq+go9alZoL3VWLbrOTjxBpxLO23lQcsiUUaF44x0+PSQ7FuXVPDPq0g5Butooomy8P9kkzpfNVe0ow9jgxtwm5bqdkSD+mzXH2LQewrGIe6\\\\\\\/GGbta40WWOhNvN0Q9bBVwJmu4+mHj+hEqWIR+cr69z5lXa\\\\\\\/qSF\\\\\\\/2HRSjucz+cMcC0uePCTNRdgt85ZaBh5nPBMImwuxf6y7vLfRRmUQz7j7ue4b1O9xNoi+WG6YUsSVLxcClfKKBlVq7ZoQTFhz9BBWkLss7QSR8KtuIrja7yiOnKsjJXNrv2tI26VMnQ728RZNiVFLEP0aLazfmrtGP9qwnvfv4yx3+fs42Y5xJs5kpjCduio49MuXykQPDTh3aKlSM5rFUzMZ7LIx3orFQJEfCNOAWSfgd\\\\\\\/KLKfjby3VAGjCyipewEfDf6\\\\\\\/dXtdcVC1lp43XPPVqDIX02VCKtfO2pA0gmPkerdjgRYw\\\\\\\/qXCdQb3vTIm44Xn1Ar6IgN5LhbUUB7er8pMpU+rZW9nnPscKuYtUj8xzUfpBUIL3LZ441Ozm90McsK6ItGCpVLm+U3C+yu0cTBMGawCrmUet65gE\\\\\\\/OxuwwsnYbkdGey97xGbSVNOFKvo+Myb8DWTWhLWoRRncZiooIimBbJO1fwEBEM0WP\\\\\\\/6oJ2l7iTNdPieuGRXjeBa3lSi5IrbZ4rAkJN\\\\\\\/Zd75bHpC+EH7Gc04SGAQa3wBGDNGb3fJBy1RAGQLFv1A3d9pPSdC\\\\\\\/X4qXGkNObVvVHxko4ZMXvtTLYvFV5vbRgJG1He7EVyRScfyNK9q\\\\\\\/C84DZlEf+Eh5CynvnVazHls1\\\\\\\/Jo+jklWIg0ohM\\\\\\\/SlKd\\\\\\\/zKOT6DgYCuLtslcHfFF3ZxSsNFcaB57M8K0YL+zKx3NAVbdSNCH8T\\\\\\\/gHVbop\\\\\\\/HirSEoh1KA2BIx2KkV2c+QifRDtanvcElmt7VwNSQhYZ1cyd5zJb\\\\\\\/C78+KVzq+trlZqOt0VdfXtDfY2hcg0Sm8+\\\\\\\/u+Xi6QhVMjZaLkfUNjILL2NolL60E3CL5I7rGSo532d4yIwYi1AcijHCspLyi0txU2mvAFcKiPIjv1h\\\\\\\/WZ32OKBwGKfEnBVwHs37czo+zii+WR8yMPwqxmKQxDrjpBlUhPwuXih4R9wh0rH15\\\\\\\/1VyI0xFM89IWsnPMxQCUTYwR4VOS7GDSdCsPMj7v5QexxbNut1gp8HrcTDgERY\\\\\\\/mIpT0IL1Zqgttp6w0sNYTlACz7sIwiJS2uk5ZrHosegMBeIBGHpknY8vH6eySPlhDqM1KWXov2gKSAlkYOTXQBD8VV2GOYbecZ5xPyA+qouvdwx9+su6qvQW466jT1dd1Vs5u2qZtQy2tBuYd1BUd\\\\\\\/YNzWY9Y1crfJ0dxVPEE3YXojWGmnNEbytO78S9f3ywqfT1g148qAvIzxGwPFYpupjZyBZEe+psyCn5u4sA7Nl0tn7XANvYBVxgcyTSj+J7T5t8FdTN50SF8b9mMdgUK1Iy6JzUsDzsMradsWl37kKSBU1yvOBEjI3uYUfvTtUn2kgEN0WWyofrAa8SAvwU7MaBzduvxyWPcQEjBoXlg6WQAhw5SUwPG7bMDZTXPnukPTsPe8T0SqyfkgkNM05l+K8xshjMNTTVNmhiYjeYX9JBYwiwH8MkYwNP1vI9g1LhSLVC7n+Ogy4Ft0tBm6MWRalI+dYyIa3S4xzDyoKz3quL4oY3+8c6RzYXmNH4KN\\\\\\\/Xu4rN7SJuiwo8h0giGYc\\\\\\\/pJADliW1MtL0+2MP40Yi0ReSh31mniuuylyi3ZC8ECkzaDfKRvXokbhZCanMEPHCWdTMd3jHVoLC6iwmKCtHjqvj8PG3TMg6vEEDUN0xVYeaCGUA6sC5uFzNkEt94JgdXYIST3E7QAdeH0Gk1dd3E5mD8VhZq2h6XWvSYihh2pxCyIfkr7\\\\\\\/segulD7ZZU\\\\\\\/HgvXkOhRTjGFS8etJDu6cn5n0y3zimtV52V9EY6NESN3jfSHPUkVHJqvG9gBJcnQ+y2pEbO+1zL\\\\\\\/OzNXMlcXgCCcw0hhL4\\\\\\\/YFOydpUrPn\\\\\\\/We1SUbrUiq8WYWbWsqLHdtpns22DBzqLFWhGSqLn5o5dvQAGh1AragFQ6ohYZXBPPp27nS\\\\\\\/5LPo6tprw3hciWk3byfc6PG21Y+JZB2uSeLIKUM3nJYZMhZFoEEt7csuq\\\\\\\/cAnbYBUqzotq9acl41vOuRZ7v+OX8jU0FXMvN6c34Mrem5Hs69zep66Sfib00tLgloosWpbACQlDGBuJByYUpNQfMdjsLPxmkeBdjZYPmzxv1oaw\\\\\\\/mEkRZ\\\\\\\/+2k3mT8Y+BxAevpIlIjjVVvDDd7v9o9M6Odio36DNVztC\\\\\\\/goJWxUTKNNi\\\\\\\/RgNuDCJXLEKsW78ja4elJwl3e3ynRB6zl0x769Htu515\\\\\\\/Rh+8YJENtdO8pJynVStEfGzRH6Xq2m2QbzKzJDUxblmJjf6la22Bv3pJZ3bFmee\\\\\\\/6lV+tMfF9JZYEavCOcgWzagsNe7LcKOKqOPhtVFSyU2P5oO46AAkGOMd3y4i58iD5YoEh0IpLviQtMHJY3Tq6p8E82Ew11ufgQ\\\\\\\/smUMg8+eOPfdeyILIwRNbfZxI7dHnP18XI1jecba6mgoqYFIHEa28BmXupeJgLuqFTgKiYeuAMlbrOqUbaITK23xvY2+qPR1lGgM0USaaNmfAxgmIZgJR+YMUz+\\\\\\\/RjpZClbLkMc4HU0l9PgPWOX9Lc860f1Y1tp2R70clcImsK5NjF7kKR02tOc5Lv\\\\\\\/y76wS03n4u0IxB14Y\\\\\\\/LnHrHu6DmrSIvvzaaLMGxEBC\\\\\\\/tbkfLETfDPMXF3pBXUE7HoZ9N4MS2HNzki+Hdb3IKMtJuJeLGbsfhtX+1P+Gv2QcrZqTb+vt4b4aSN2bzIT4VBLlCXqUDlECKh60lQgaeGV+8htSo3rH58veSVm61eUtBsnvr\\\\\\\/dwgiHlWFyD0i3rXtm2+Se3F0ZhDLqKxA73RQ0rAFXx9RRPepRTaVoFFMvfXXMk9nMyJEIhGD9C7XlUAzIoqlQCpDDRU8idzCkaOeBCEaVhkGetsPLilSDITaAJA0msqHXFnlUNYN95P3c23SD2SXxRlifi5ANerESkqhjRyVarEYYXZCCKPqBUyTmJnM8cf+ynClbLG1X7vQmzw2sJPddnN4U7VjldmHGaRS+aR8W9wVwgU2MhC\\\\\\\/LHzRx+MNTnUMYzyzMn\\\\\\\/pQcFcjUOWumr11I4qrz6fygeV9\\\\\\\/RfDk6\\\\\\\/FMoEFexq557YDHFQuyzUWxGQMeooX7PSHn\\\\\\\/v3MEZ0iPwCDTVAQgIJU0qzQE7QReeJnQpq+L1MPzkMtsTd99fdKTs+S8ZwPi+G84dPuY0fJxyUwHuui6CANfegwWXwaSjmCqxMpJwhK\\\\\\\/edtJhA0XkAOtcSn+jfsIXKziarfvIJ7POCixyezWzPeixU3X7Y81KGDHt+WuyIctqOLeKZCAGL1Iyz\\\\\\\/db9JdVJIiNFA+0jABMHoWVp6NrZ316zQR4jUSOHQn6R8MTSXhmvmRZVXa8+ckQnCtEmJTVuEiTgeYnKT4kvHGfMdrnQbv7535YfKh4AH7B\\\\\\\/X2ub3Ar9k4AMlm88FW3ao\\\\\\\/HTzBsiFLBqew5hoKAEKofpdRW+ecNLowVkKEFHpaEjo5LJLNPlun2q48RRhNwqirPKBCJBPwdzH+rTt78BDWi31rXnHT2zM0G2Bmvrfr6EBrKh3XnjnFcTeBVLEeNw8nUpdwG6LFOZIvK5tfcg2\\\\\\\/rZbotlu4tC6+IZerNF6NHyplEvDb4eyoN1kF\\\\\\\/muQmxX4CPo9yJk9sHfGGTWh4zND\\\\\\\/Wb+AZoIDC5g0RERKJu8l117cKcRbmduRWDmrOUbwQE7u3CRwc7Cx467mNL73P30D57E2jAJMuXWNvA5FJ6N+p5YCCv8+FXcgOH+qKVWrOvkGTjuK\\\\\\\/UH7fxBQ2xwP5zjVSaKD7GCBFBdx8b1UMSzoeJCJ6KFmkGTvkmqbOOFdPUA72G72huzeYGpgwd3sitUit\\\\\\\/X+fGz+lAB3XAB1oZwm+emTJkirAc17SGXWG5s6vL\\\\\\\/uuA0q+RfbuyT06OT4pl0YXlu3KVkhEFz9j36busraMqZrVYf+uw14pE3KS3AkqqsX4nnpnBJkr+yT8WSbWGjTmwQx9KaiFFyh530+BwYj0hPXeNV+TlKTqXfaQbypzCcXO\\\\\\\/TOFfbTLLyoLwRzJKT8y6fyjQW75mC8IlyjdQt0Pt\\\\\\\/S4eFNkbCdN9qR2ejeq7aLUX2JUsX+UTvUYRIOBuq6n7JdHdOV6R3cVwODoWBAg8gDpdhlNVNYCxrOzHkFp5\\\\\\\/jVtgXqi25l7pLrdPfXdDhxnsU74ZjBOtOFryK+QKZhegC2nBzHW7Vt+tjqfrclSAbw6lukKp8IV0UZRQqHMlVRTRMx\\\\\\\/Xwjdwo+EmVVuCJKBfovBxd+ny2GN0UhQH0rhORbo9druonkgKdsEcR2ZeQHSHMVJy3x\\\\\\\/s3kTWxfy45y\\\\\\\/\\\\\\\/4yKhbtbAZ5TW8X2o+CAEI1reeoao6WFBLw8x4YpR0i8gaJmGxGRzHsPR2iOKpqb9zn2Ltf\\\\\\\/Nxrj59+O7tvQILt7BqSyTBTt68V389UgvrpvxxP8\\\\\\\/Pf\\\\\\\/fD\\\\\\\/mTHXEKJjwFwyw1xTlYKX6D64DdAT4pA38jlrSAmFIpHWAgvb1eGS1q3ZSg29WZbXkBVGf+0+EApdU+nbvrKFa5F1nb4LP+7LoSiHhjULb\\\\\\\/YD9qXh+aATCzlkkINPu\\\\\\\/mh1gkyDE2fAWUK+uWub+scyEBZIQQJKK8yZUos2y7n7BPhe7WIule0Hq1FdfMRvLTFKw0ShgoA81doyjcIdGdKiOyTa1C7eAEP5+HS7ZNlxq9dQ78FwIYwBRI4FQQpm0LNQJ7f9V6YCfoRnI3nthMjd8QixwXULTvMHBpXVvPPB0RiEb+WHBG4QiuAaZ+HkbFKdhU+ZYw7db+8tmRriBqRCfxupRtSjqMXaaH2Mjjh3BHva4kXdKZEpKnTZ7saFbcovwxXvW\\\\\\\/JwxL75B8GRdSoHp8UtBRo76kG37e2eJuZWfDI\\\\\\\/qdugz\\\\\\\/dDJOH9cxQf2IU4byc0yTtFhdohT3ov40M8\\\\\\\/KJlOA9PAvulKcHjM7TGNjnVJ8FXiA8pgYoolTqnx8GSipulPEBuuF4VwVtYvm+ZdfFx+lQZEIXQrKjq7nvHNEg3zk+m1\\\\\\\/h0yuuOvoRLFoHwoJ\\\\\\\/j4vagNOJXbfw76VwAQDf2DHUIGdmr0zyzrKX9\\\\\\\/2atbCVQUdHs02SEHldUPdSbb3M6lMpthRnvF\\\\\\\/jABgybcwBMCka4t8mNPikB\\\\\\\/3wt1m4\\\\\\\/o8fOh3QtfRNZGoma7IUGN1DOOieZBc57\\\\\\\/3AsrQVxGT5MnGGoBQ+N13jsnrJm+PPfGJZRV8uCdZeD86k8\\\\\\\/ig\\\\\\\/eKbqmGljA+8JCMcaVLZcJdHiNL3yaZcwD38aH\\\\\\\/c1hksiTENfSomMiAxJYTk5P6IDv282TGGUEq3FLP4QHMAmbM1TceQdJbsnSafoY2k6U+uwODyZlD45KzSWLDbtvJSrjMrGcE4rYdgWv83MZwKLxgCrZxHcREzOSZJGQR7kRheVkhKTadsP7tYqsm8wVLqGLL3y7XX7X8nVB5mBLfiInFls\\\\\\\/ub7lq9Jp2hw6+wZv\\\\\\\/Hn0+P79xsAekF99XMKumnMDrLmghPhx5EV+dcZ1Zf42Lw8XarxbTu41cIERGzN7irKnWjmB8AZ1o3iD9CHdyf78g+nebX2GK9+293GrOHKx6XX4xdGd6BitIoXV9I+hlZW8\\\\\\\/2mEO\\\\\\\/ik2C6NfCKyWJwmq4\\\\\\\/kIkWUnEMRMj3Fh3dk0IWFJ5htFVsTtgi5ZDlzXvlrKZCcCQwr1ZDbsrZmwF7etD833esVeWkkQLAvQnXCwj6VJXBMNyuRkuYDtEynduqb6bBhqzsBekodg5BWGH7w1OU8E5cm3x2hm+oMxT2x5\\\\\\\/S9dl4mMcF6eGVcWU+7QzjHsn44GcTOmMaa3kjSCI\\\\\\\/jFd5ZeBxECCnGKRXPvTITRvqnqGFDTSdWWYMCtSTp6hw3cuhacBLxeVLDqGeaLvWlYwsKeQU1QvByH7M531S0wgNZJAOoqsCzMHvQf3E3Lo4o3hD2lcYLZXGyMegCM7NMFr0lTSvvZklc9wD8raSHk438NNWW9jaNMqtqhlRDguckxRtS6vaxN1tCesRuN8PMEA5CnJ9yKnnZ3BsD5aJu8keLSEBHvCJsbMoHSFUI09Iy5qNrW3y65X7eTVIhsCiTM4N50eXn+piui5vCiLUwPUFCaBXXmqUzKoLszKSZunSNB1YlGW5ksacrGX18Iy33n3FezLY7VHpaN1DKmQNa0HcUEWPtK\\\\\\\/CWmO6TA4I+j8PvS\\\\\\\/QP4ZcCEqozOOgByy9uT7hYF8SYDge5cGzdYqj+XA26WkM+E+lbqbQkvWVEsVT9qhV5HOAx0DSbh3EY4mNt+\\\\\\\/oQS32IPE1rN5vHMafS9N5zVRu6xpIcvl\\\\\\\/l4kSH3uoP3ARtwdq6EnJDa+a6advKMsGYHIdpJtX30N33R6+QY9i0S1JPULzDQ1j7hTFeVlPh7Fl3UuSk02HAuhCfxI8x+1DFqG\\\\\\\/S4Zazy4fTUuVU4AUiyQnYQJwfLeP9RFCeYxL0CmLi8HFpKs7zfSs\\\\\\\/HOxnr4qq\\\\\\\/kvEZ6QGOGhk9P7TEfXzLddpkfdPDS\\\\\\\/vISU1UA9+hEx\\\\\\\/U1PgQgf3CQAFpllefP\\\\\\\/tItRGpwuK8z+WKBSONKx4ilAznsQrH0cX9AqCsDFy9yJj6VvbMVDaOJIALiZoYZJ1gefsKzXwTxfg\\\\\\\/WrKWRFAvKsQne1d7e6XP20hBDNAjZ+pDA9AvSozV7WplEqDw\\\\\\\/vLGRLg+bjpChF7GpyXeuI9sBjJoqneGnRr4SYcjwyWf9huKMU1G29C4wJ1sr8m5WTU9ndDcrVBUnST9hU8Qs7WqKILUscLINMO2LqbqkM+wCYm43omBqFfOrz+3EGeebhozxsfFvP4s1+8t2OnaKzVPb8X8rRPLAUPrIPSjGIdbNmZcKvX+3+RmpkU26+TAt3WLbLaxa3Njzc4pBJSTnNMyfJQ3DsjgK1s\\\\\\\/SyyyAoozX5eIOU5UQB8l7LtLvEoGD0Sb8Xp6WFz6Ujj0hTqtT44JyPpKKEkyrAsq0TbHV9JiA9tcT317QgqXNMNxWD5NDU0EzeqmV31xKH1WLD52oCvuhWhrljH8RrgivjXbLAcI7gXtEDDbd+NE6iEx+SfZGy912iQZBst+gR4rN7LHWR0O6MtVxQ77RGnfCjbFzdcw4ewjjDXNbqrYgfnFEPCImgg5rGKLCTqw7At4fe54M+QXAZh1oeYNbI1LsNcJr0+xxAnoDRsEty+1pBcmyngByRgA6cBtf5NeWyO7R2Fp4bKjHcGUrXKuA6nPSjOnvhDgg62EgHd1yNGZo9KrJGfnVUQfVCsg\\\\\\\/4A2pjAgRbuMyee80frM0Y1NkOm8wYM8+UiPXk2LpOL\\\\\\\/ui+vVGcv0Qx54XTy8aHFZIOugM+9v5tmfy4bQodEnsJm3SWZgTRhVltiIHqMQGbpDCF\\\\\\\/Kd2Hn6rvoP1ST2ptbTvT9vDK4w+9kfRivfhJOIgLvWTEjTQU73y8WcFaJ4dNnO9vuksgIGGujSVTid1cRkhrXH8SfK34Gq+KO5RZZtWjSiCobgoi6s43FFzqTaFv4WiiNsyAExMSkqe60t5JytLHF3biAUufxOMn5+IiMCvyseF1C+5ev2HPcj++a6bPj1XMSRcqeAowQj6i5cmhgjmodOrnWJBdIfHS5gBRd\\\\\\\/b8fntCL2c36Y4sJPDJs\\\\\\\/Hhm9PwELNN8zt2anKhR2fA8+AzRWv2dhqimn1fNLp5HIujet4ie6QnOXZ3yAKGuiqf1N\\\\\\\/i20aFnqduGe24\\\\\\\/b7f0ozjUAcsf3Nmc4MZUFuTeEKbLSlE8CohSb17N39XOMx+n1\\\\\\\/0I5CqBNbl0TOELhKB6E3GAXCczDmIehPAkgKkH4aNs9jZURmZhY\\\\\\\/mE1wKwAvSCGhMWznJeMkRe4jrm+SMPwtvLX5BP3ZoDRIQ0Iyv8uYd8hpksKPP5VUcc+ejuallkpGz2evz8oLz13sB0aDP4zEShHWobr\\\\\\\/DnHBEzKNLD3Sl5Ns6grDNyuujpk920AU20CazERJ61QTxK16HzFtOC2znqrrbejC0vtuQCp3m+0fNL\\\\\\\/67R0b\\\\\\\/zq010y\\\\\\\/fqDh+j2I6Ajp2qetEEFT25wJ30Miz\\\\\\\/qoRaQ03ldRZGC+iPKnKz+RukWp56iigRmuY8NA0ywRzi4wpKWqXz5HdlFoYKj\\\\\\\/xL+du0Jq40UllBbCOIFgFUbC6tme+KNGP3aPEUoBXuqy9E1LwYBdUvx8Fop2EufAsNLkzHwD\\\\\\\/NScpa\\\\\\\/RS6z1PIuz7a9QPzHZw977oWgcoY\\\\\\\/OQeXhTKER3TiIzJ8vT4v+6lkE1dy6N8rPBdLHSYohlTyv9H0fW7CE07HyHMWOZD7qnuTwEqf1E2sAO03A6JLibu5OVNsWYjc0hx5ubTGgPb9QhwR3yXKe7\\\\\\\/lGP3BGi7i7y5Tox+CCnPewV\\\\\\\/jeHEDIwCzTIwrPEPpjsqRnijzyOWntH4mNkWllHK6iun7lskq7eMuwanDmT854GvijMHZ\\\\\\\/qRRMO2xz5ZRvzm7hOC5HiA\\\\\\\/n3hcpHs+nalxhbcpLZ3rpCb4tzHFD1h19NSl7n5umoSrnmvkoLib1anhcRr3MsvE6cbfyQtFwW6oYGu3sSgA7yl4Zmheo+d2IgmI\\\\\\\/K9uG3c02G29eUQ\\\\\\\/raBfdXJUEJsOsm1wNfbw0Aiw2UKhhWy7Ru1uqatznBpmUfD9RtpD1KfVSjXgoHxCRb8CTFZzEKdEZ9iiOaY6vVVRSCY7uTWkPeLZ4wmtV0FqsPadPJnmpMmrtFG22Xe0imbvXS7jI+8r1AyCOs1OyX9qLWgb0IiQ7GA4Ngxd1qFC46lMLQ0Yz+Ib5EkZ9Om2sxsH9WZA6eOHNMwOvvmGKf+EUl0wzOzAcCgCswuweLJMeE1g0OaDA\\\\\\\/T1fhB7AbpR7QwxXlz\\\\\\\/tgvBHZS0UyFFB7UatFX4jDmtvekd2+UpGA3zGoGc4voEejzXqQ4LTQ\\\\\\\/379utBkE9QRNqYe56Kw\\\\\\\/1cp9xufa4u8S7o5+5n2U0BJDCtBO4SK9KooqGc2Y2kLC\\\\\\\/D10yzfDwgFz\\\\\\\/5qsrDYCTiAp2i9q5QJ315QGuilmFKkx3LDKx6UKDXtVpC47IB8Z9cMiia+g1DU6i7YbLnnezz7\\\\\\\/qyKl+oC6zHb73uCHIVyIbE7dZdWtBkpinM3ZRucG94+J21c\\\\\\\/e5l+dUSosPRXQa2ZD2OZ3qIbk7qYOXX5wgozTGcSq0OGAh91JJNkiqAyNMV7yf65s8Dpw6vdNjKyqnq0ijngUp4ptL7v1GTTJcbnykekKYLSmCdCqmq35033ePC8Qu\\\\\\\/LXexcMgWxgH5RKNSw2mXdtv6VBPpnq1gqZD7L+HudhjTJcQI04Qctg45QsQaUTwCeKVFUmnBkfyN568\\\\\\\/y11n6rJJY1QMDRhpV3AGdFGTkQvuye5yHFfG1IQvEeLKp3d2RWu\\\\\\\/2fw1x06SRUxOmXmmYkX15+jVewW0eACP0ce\\\\\\\/I9S8Gq\\\\\\\/zzh8yBt1vi5pppZDkRaViyT45B9AYCvN814dGCFL5I5dNuy4mfd+izkjlId8HceRfBa0+akPVutwC\\\\\\\/Jeay0XZJPazAdc7Nt4IOvJF6u6\\\\\\\/REMYfiyxrRdgann0h1pwKbnzj2lG20xzhbQxCHDHwo11UMblefukedBdpV8cJm\\\\\\\/GLoDeA66y8DltU+9ss4V2rlwkLhDWn6VAGNdxNI\\\\\\\/vUUAiYF7GkiP6PsrbtRkDIweXt+svvSRdE5ovrrRjGODyi+LkNb+8iEC4ou0z0F1Giop1lmEpRkD\\\\\\\/Tx7ckDh35fpPEtHGaylOPRQ2R4UpDh7KDxBQ7Q4KhTUbPTTCm+Zjo9UnDN4tjxnhYPnaYIcMa7IAokN6l3Lz1CGCbi\\\\\\\/\\\\\\\/ZHRVpDN9pP2aIrnXcVZBzH8bW8rK2dPtQY4SpX6lmACzpdj13j1SNhSJJQIK+4Iwt6QR7I3q07DOsrvj7hk4EM6xzuQ1ih2WjFDASkxGh3tqhUQLhdvHa7EZITTW0uKXf5PtAJnE+DXX5rNAOMvJVMQLxNvMHJnXWqBSuXCUz6OUL+YDZYsJzVAKTMECM0hEKK9BSK\\\\\\\/gFNqqbR12PHC5sbo3Y2jlq+3Woc+evIiw7kEeE8A4dmmtFqvjBa9HSEFGW10Q\\\\\\\/m\\\\\\\/4pvktwpu5l4fQVDZ+G\\\\\\\/25DRjNUHo13rhaxFLusgrmogwA3nWboCTwK69271tP6suuTU7UgG0jzazxEi+FgJQxc0rMExYocsexH8uqqKYB6zXZ\\\\\\\/o2r6SoxV86qw4Y+R0UbA04uBr2KcOCSrXqGx6kKd\\\\\\\/l+qmOzjCer2c4exrA4t+u8+d\\\\\\\/GLdP\\\\\\\/GcDfpMjqCnFkHN4JZCQpKnLTxH5tss6hpGV+odH0x3qOXxX0v4tk6yKdE\\\\\\\/ghHPTPU1ntu1ndUVholAO2OyokdRzL+4xoPkaeg2V0+t5WVeyIVTt8G+\\\\\\\/2LpUYjDovrxYpc550pWR3Wp3YE2uKGUdTEJXog5lQjrn7\\\\\\\/wFafIy1Ia8ja8OZSi8tWr+2N9A0B5dcv4YfgYiOiZlr3lKtjD0Po\\\\\\\/u+GxsajrbwH13XTsn7+D2lTECQdEMiU83hok9yX6NuOzPOuDPZ3UFocCyMwrqJCb6Phv7WskEqV\\\\\\\/KcQkQcdIU3GqN0SJQ6BrKDpmElQjCvER8g+CzfReHjIee7qPiN4Bi+Fx8UXTuIGoDIoB3Xu3lSn2kauYz5DvggSMXvtm7nkGFFD8KeVfEaBWtMIz6GY7tGdouIWXctIGbxFJ9lExinWDjKF4JHfKrRsPlyoSepV65Aw08Px+l+7kPFQGL1WmHKF+iLCtW7ke+ncy5BCc+P\\\\\\\/ymMoCwMafMJ6yHrqDoisPERrbqW42yI4xzyWYeBjvOb+jl8rATPLrQzwv88xwx0PLGbIz\\\\\\\/FNxej89WURMv6X1ObR3M26UOTw93Hl6HXoRL6Qk16tz3o\\\\\\\/yvx3StvkXHcU6NQ\\\\\\\/ZiwI6QKYwBq5CccWgQZ2xMaMpdbUwoMJsdGIznDZYjmVcRpvQjURQQWsbVVdQmy67g8Bd\\\\\\\/9Eljpd7yxuC+T\\\\\\\/2q7rAX0+EH7P\\\\\\\/ZNzbCQ6H+RHD4Xwc4N2Bn4BaPN1bGzcEBFlzaRqTe3NQNhpOmOEJ6K3tDwGbV5u2qQba3\\\\\\\/WQrF3Y7uaBkkb65jWqVrW7I8DoMd5wvM6cMUHL1Kf064ZPAYe77MbKlRg\\\\\\\/yyRSu8aiGxrprcjzm+JSsfSZJER98jXQ5kINFjEswqlpBl+jgjEW+fYmrwbuiAc4JyQWtXR\\\\\\\/4eozSRJOT31swQVxcjJH8DaKFctnhRpG98qlTWrEOTwod7oRDGbHtVN7WbpZK4DeLwBEuiM2IHJ0G2I1NU8brwzSJWhAgwLIBp0oXLiqpoVsUtlMW9qU5WlxNn4cMw62a36Idzi6L5o1+H53DYNtYPmF2SW6fOLJSzPkAheCCSnx+DLfi4nahdyRZrkmCd6NX8cWS1gyKxJEBb1s2+M1PoGrv67N9yYy0NuC9fPa+Nt8J\\\\\\\/zy4P7bn+TahY02cOwD4vlAIDFxStzIlTjbrtpXsz9UNcz8fDnzCYpMRviUEj1RjMMCnXVgrwOxP1fiXf9dKQiPUZR1S0fVZOXjT5s9TJKa8aHBGxF28zTvWdVHGnZi9JN3KVTUifXfDng968NhtM5xmYKXtOpmD+A7aJ4iUORjizNQOAZYIYultdKSboN597TeVmxgzZFXhsgm2ycpOBgHGuBFuYKeBtC57aWsN1PDAKdTgHHVUk1P+rCz2OAXr1Q++DXfKGbRkquRNTw4se9toClQYXje+Mo9Xp6rTlrfprMib+6RZJ\\\\\\\/z7jP+ZDMG2em1ELK6Rn\\\\\\\/AoTZaeLRBFrtaqu0Hn\\\\\\\/avZFCrHTG7DWMpsRt33PscqXeXUXuF8lUCwXcSuF28g2QfjNLH8+mdXzh6Sb+qlJ1BZUWELfAeGenUGdMkasinFAvsIdY555d5bkgTczbVeTouB5cFdtOmBjyW6tgpAemPAXeWStpQ1UirghXxxvx713PoV4Io1RvT3GT9jQhL5rbXfMglt0+4glsLuJztaavVz76EGKv3xZje6ZvhuuCnZoD9pWKOkAQ3HMy38nNIsuTCY2pF2SST+ikn2ao8+ZZkzTYWhDpk+76Pin\\\\\\\/PNOdDEm2Gbfx6O7vI4DNmv0zR0MnW7LqJ2zPIdHePFuexN4+gigLZIUJ+4O9kVQY2dOQPrwuhLG+d7ykfJUZuT9e9\\\\\\\/o+GHVwXdMhKFwsFS3QZwcAJg8k9t7DXE5+B8Dt2JyjnqDs\\\\\\\/f+25digYRnGUEIX8St3be8\\\\\\\/rzOttpQuSJutNHjy0YYgxOOJ6eFIe\\\\\\\/l\\\\\\\/mk\\\\\\\/0U51afgwlFD2iwE9eOvDy8S0SSjNGEPRxuJYFiM1UaQzaOx3Cq78Vlwph8h62lPNYuXknW5hwW8Gk75r9eqqNZNOdB153jGfTfQvHte\\\\\\\/FhgMd3QzdMvdefWk4rjjMFBqG1LeK6\\\\\\\/icGh752zCzRhYw980up5F0vkbkcwni5goKoIpU0QU9dVWC1YhGHJskOvGsU0A9tXBd7adddZCtrTSWiPtEuubGkZgRG4qND24SYfU3a9Fgw+7kmL\\\\\\\/UJGDcrMxVaKMLT3t7\\\\\\\/SV5wvqG\\\\\\\/wQNrSoD8ebiISFzRrg6UWhQHnSeSWiCJIj4xGDL\\\\\\\/YEzcFK4x6efW\\\\\\\/NYyoHzQbLVONEbcanDvzsd6MzuCrTfDeZnlTNKHbQB+csoq4kWUzoXHIPRn5NOLiwNiJ1sdFAMKEwg0AQUBjAu2s3ZLLcHuoEnEdl5IBzdFePZaQ6Nbx9Ah0ABJSWytm6e5+0EUT7MfGf1YC2l8+fsAGwLk5kxxDj\\\\\\\/2Zzr+nHlq7BApR5OXedccGbLBAn9JHDvQYJDf34BA8XtDiLTC23lPnloFJSRGQlfZaXf0tP5CHG72\\\\\\\/M\\\\\\\/\\\\\\\/4whlAKHQsNC0DYWz5THSLkH7n4NPEkYzzCQAnFMQNPk++K9qYJXr1ynVonNsJAimDlAS4fDeOvsrB\\\\\\\/TL5PDCrzGJ\\\\\\\/is6sxsdF\\\\\\\/kzQn7pVd8xjhT2sLC\\\\\\\/QRRCP1\\\\\\\/vuqZPCrkGFMS6N6oQXZC8Ir76qz83XBqevHtvvDeVCHM45uOtRrKK348qPuLJJ0X8k\\\\\\\/zzHyaGU0XTPgxLijjBkQy97T0zpD9znaNYSdRVtfzPsIJGqZ5Nf47T6eiM0TKEtQHkysxp6oye+Y1DygABFGmvxFvBNCsnRROCUOXn4GMrNxh1FrM8QsUwPsPHn2x\\\\\\\/smLb\\\\\\\/GNRrQRxCfkZ+u6+HZ\\\\\\\/LTxTm5bQiIbuzcbDXIHDlm3ljsAVJUx9r4HOwDREBVkvJRxWR+tyhUgqebuYDfHV\\\\\\\/F5Kz05d\\\\\\\/1KpDJZwM7VRWuo1NMe6qLTaR9ncSM9xYeAoiyGD86KfehoDYmiU19Nk8kMXQkUsZb3X7rJ79iTAzTRyyU+5OXRX6dG8nQmfZnc+CW0CsOv0E6k4497oyXYdJqF3OCFa7Y5J8qYbkwkqAUaE0yl5X+ng313Y+7JCQXDngowXBWDq9TQaeB3ZFwoUUqITmsJ5DsoMiY4vZD+nfRN6KbMybhfCxxRamenZDivf\\\\\\\/k2yXVsMuM3BPFlhC4RqkXtRnDGIuw+HzR6LJ2l9khxEU4xR4nsP\\\\\\\/czUnVhl\\\\\\\/x6a1iMHJe1IVmQbHpMLK+qak327CGIfmjV7A6N0s0Guy233QWLeINzUv7q5V6jXHmjqh5\\\\\\\/KzzFnzZJpokjn5XMipnijvPOCD6A1XqtW9kuww\\\\\\\/AepNGSJXat3WJhhY5ONZYzkffydC\\\\\\\/tOC5FFOcnWdowzeoTsVH0smUvLkqjsShsFm8xon4idgbQiznpHbVPDla1pV0O2lrud73t1ojG0I4hKZ+SIm1d3\\\\\\\/0K93kuFkBe1Nzq9XtTwKF0EG5rPGTLQmdZMobvCAfWhSyXd3WT1LAZXsu+a2QYMZj5tswFoT6d6EMtXvMmq5eY8kYK3hBFNhrNRraq9b4oi2xk4aHCt58u9cpiRW+wtlGkNLQUIHrRVHsPrG\\\\\\\/P9tE7UD9uygEfEsuSnJrD6XT6gebdz1A1vRXtqJWb8odvP5IhfLDoHzMo4GqZV7Y0n31Kz6BA5HHUziw4YD4yV04wTwNw2R8EP3xXz7VfRPUVqLQAH4WvoZLn6UpURtGth5aIkOEYYVP7ncDas+6Jb8myatHBxSPKSGvrbKZ4RxM8cH54yXexcAZdoiRjQdBQUYL2SdPoKegJxKI4xnyqNnvrMkXvr5T8IfCSv3WtupPY5KYhT4lGvuO19nodVsIUNPAKn4FmqlvmwwIbqZc\\\\\\\/KlOZABGHQiwq3FOqlve7bNLfLQ8+CtDejJnrBI2pkbm93PSRx9\\\\\\\/8+nIYTnwjw5z+m4NxuXjwtpC6wjYjN7fHfom5+yq\\\\\\\/GO1ATqioXxFi9TONt2VZOwPREcOPvXPQqJC1+\\\\\\\/5wCFZljlFtSWSqvVJDjm88i8eHXxFSOZ0bD0vzZQQn\\\\\\\/TII80zy8bHsE1PMKobGMvYH+U19BUD3mMaW5s7IwAv7TgzlBFVAUwP7bhJBavwoFBt1thb6Y9n5HHbBrBjF8ZRnO7zFbC2IG8XilmwB3J5qGINDBH2i3ngUdltHF6lNG\\\\\\\/QMUNk2edSt4kP1Lj2A4VjMeXklB5eG4ct0s2x3DwrYEy0ptCh2xaR9swvnCICTbjcQLKxGwN2dDLSt\\\\\\\/jn+wbOp46pDafppWJzILmxvJJuKHf80ktaNlsXlwtm3xeHnCTi3Zz78Zobw+1gol9dPeW514u14b4Em80LOTrqYu\\\\\\\/JTYwca3OqWXbuLrHfMypF1xT2inmqi19zrsWasstqvv5aTTXcOJm+fo2JYpKvWZpt+lv1wQnjrDvCg9l6y7OPbKR+zm6Og6FVrXdDt6Wb\\\\\\\/kJm190NG4mZ8fgQETjJxDVhWqQbpLJVU32ofQnWIt3fKsFDKdpe+PkIuBKMsqQnjEe6\\\\\\\/Psvxf2l70gRAWFYOufrNy+Rb5eHwWhB4WdE71wr81uDVDadGp2QnUTKuIcAQHxmNNboNBvLhO8zZL6Kti2Y7U+TKFcM93\\\\\\\/tXcRZbhObIw\\\\\\\/M8RijMKyq5TMcOvYS0e0+RcCQN\\\\\\\/qitZp0J7f6TW\\\\\\\/MZEBKc5Jx3D2EntkLe1rek19g9E6UqrPULpgOPpc+AENxboaoWvUH6B5OCHe8L6FL\\\\\\\/SQzscDkbrDteXngtSIdri43y9T\\\\\\\/4jY+xstwrMNviJoxtkmqv248L+jcWaxXicrM0eSGuFO2YXVtEHO6SD364pynHvx2BY3HsOCSp6wiB4pPmAmzDIsAuDfOkrr5UuXx3Dh9ov0gdEqbmFO+kFwKWMNPyPygrxy+zZ1QWGPXoXogvy5Ufsgw7esXV8lzPgN+4xZqpph57M1gniQgdIQmG0smO4VktHq14n+1TgBPui5CTlu9Edwuo1IY9KMLrbpKt\\\\\\\/XPzmDK2bzzMt+QeqBFovffjYp2JcNZ5tZt4B3cCCDqIWf50MipykmkiDu602Fv8LPJicbTabfae6O7mJr5kleMcZWpWXNfritDTRC7\\\\\\\/jfGCi9srHRGCFC2MZTe1DA5oJIg\\\\\\\/lxtN793AIv9LvMdDnk7sJFrg9DFvKlVD8xCl7SRkIbHikwDt0p+RIijakAN+S40StEr3VVGRU2NUTgmVh72fG\\\\\\\/PeQ+CJQQYNkQ\\\\\\\/MGn8RzAkLBSpzLwetx\\\\\\\/BxpwTVjX58LudJZ4lFZD+lkfsXXMGKsG9sAo7rKw14Y\\\\\\\/itIe49ktHCJX4q6fu0BNWB2ZmSry8s1AmxCKSc9loAEFBTnfBFm7edTxAQwCfwVE5PVa\\\\\\\/JtNPPWbgkvc53VScWzT3x5E+ik97HY\\\\\\\/OO73qfe80\\\\\\\/20d0KGkJ8Q1eihg8kmd6V8O4orolJsvXzJjwBOO\\\\\\\/xfzKTtkJUkth90JMEfazQVxLb0R0zG2QoqT7OVrkER2Pe2YqGBTc1uTQ0dcSC4rXATMWX6Ot52\\\\\\\/sYMvBjXINvFdFT6FLTVpGYPcvD3dlrYCDmG6UioN+azzW4LA7gWgmPLogAG2g+YVWN9D2\\\\\\\/EOuv66tA8\\\\\\\/GQbkVrEKnG\\\\\\\/i1wequMzrnVQy8RmhIWHru96cDHJibTA20X\\\\\\\/2vjryhir6W9ch37NuYLYM8tGlDxWR4jfQLAF18RrCO2DVnhFP5gYMLTnaEZKUx73808FaIO392eHzSEut8eL6GGiHMDaVjxXZFiksmk5cNjSUk0epJ4MgTXyModsT2AfqlLqLMvCpL5glDe73zXB8xNN1k+CyZkZXDpElmluVVnKUUGOgieouy2QZkY0bstll4IgujOOqIpLoX9JXpRhWGof39BPUBkIctVnwzrwU6vCJtf5WR35GHBBe9JOxwYTRevjycjHfnhgX4A8iOcYpbXgtSgIM95aLuQO+GjiwoB+gWE936fw12KwjeSwBUjqbvDzqzlNq6tUJ+rJ9iFastYKMg2tCWpq\\\\\\\/KYTVFG7JFwpH8k8xXQN42QeGPPSql6SuF8gwDUmuWRUlxuqQBQ9TJ24hLRhZ7RkKqdi9HLhmu5erySQ3bBHvcn1j5j+NYH+LUOY8BzdVl04qbt2EJU8vZ6bN6ylRjjvvHb4qpXW6rhJ0R9D9vK8JM\\\\\\\/lR8CPRmIPq26qZcz+ZYtzrobOiTWRXIqLLwP3jr+5r2FdL447\\\\\\\/llq9RBU\\\\\\\/uipLAl+E2Q7M7cB7yoSlNr3Hjxe2YT37DXtwhmNMAdu6LQkTJD19mcCh4SW4ItGIOryLeTWTmZsI8wmkvWeMiHngTSQ\\\\\\\/oqpK8z+77Zm7MM4o24Fbx\\\\\\\/SEQFCh4Nbx7jsqmWt4B5dnI5ksVL5QBF3ZVpeAIh1\\\\\\\/7CAm7Yb\\\\\\\/WDv81jr4gl8IzFgbMBS4fV5+egYZyelf0egsvp7sPUeSvuH6qlb+U1ZNq5SbSMD0UkpxIfTHADjZ3uk2gau\\\\\\\/Y+QchsoP4G+TU2paGiWMJFXJwid4QI9wft4OFitfmlxnex8UutwfM\\\\\\\/hHHnAvZ3aM7A5COd7X9EIIJp+HWvqTS9HAyNv7Rm4LIs+is7AaGGe9rSCww5oyvYWW4zln2RFRPLt8qudnEmJJNKhs\\\\\\\/bxbrfR2Q5xPgx3EOhai2YWbkxrmiUSotTjQ+dVvtFfkXs\\\\\\\/roSXKZFeCEnadfBTxrTS+CyXeyOzyqcNWM5t8Di0vqIZ907aW3w1Ky6oP2VjQN\\\\\\\/JAEJnEVl7Mt5zwsx3Vt0\\\\\\\/ThhymDiqHdC5dKCXqUaAtW9d82pGa8Kws6UJgVppstlvqCh2vxIfnQejYb2EciMC+KXVZzj6Nmuh+G6N3kfu3bIy4vNwTBjIBIAVo77L1\\\\\\\/Woaz+ovwj6KqM2jC7ij2LsxPwZRImAtUcopQLkYQopSsd0DfErX0Wmuua0mEp1HjTUPdhoQg54V7vNZQ8F6iXWxt7ds5GdxdU\\\\\\\/IH9WRB3qXQoBITf7ZVDY9WjA2Hb7YndmPgS0tLRGpA5OJdVw4vYI+a8DF0dk3dBlrvdnohUfbUseah6JtMJ0jz\\\\\\\/TcGupB9sN+Pf2Pd5RLWaZy8G3D62P0S6rHaG1046cXr0q5nXITBJlWgKOwYQ7tdO+NPGSGohMJBG79OtA8FXbj2MwOa40PurH5JqDcIso+7CShWEMDhg9QVrWlZfZoMIbMYMBTFj3qmG+Mz9Tb2ADPZKz2B6yIwXiUKibxFiVLIL+ga4QZPWO5ZYEzsf+HMafu8krDqcxp0LhZ5UhkNwbnFygflFydSHSEKHq\\\\\\\/le264Vy3DY9N+cGJshE6ok9YhJotyVNKHAtgQpCCM034pzuQnDbZldtqFOzvZ3PLvftL9rVmeBsyv2njM0OCDDiGSixQq1MHx1tQ\\\\\\\/SS4IzVNd6caQzmeA2VJxpkXz3UnDPPFk5BRb20836fS4yx6PXVrhHWqDoyY89dFTcrK83s\\\\\\\/GkYxoYXBrKCXOHRdug5847x4Dt2efNA+1IwEL1eq6PSR69UpEzZRvEOud2J+9YYwLFmtzHW2QblV++v0F1ohKpTDq4YvFZ1h5Rpq+oo7n1ghi1jeiDG4bqGd1f52NzCyupivT96gquLqMUmCm7dw2n+9qYz81Vnog6cZZx9gkNVHWoprfzj+WloG13aBh1iwDB1ao8jz6QjbIOwHHgMjL6KrjwJeT0Xt9\\\\\\\/+dKa20dTn6xhOcX48va9p52+5ng1AsrVdPgi8UggWptkJ712P8LPlH79T+rAgju7j0l3aHyigX4\\\\\\\/IAQXAscvMMASTEngaQFDiN8MOh\\\\\\\/+LQiEvSr1BC1w\\\\\\\/Ig40GMNCWeNBO\\\\\\\/28DfJC9LNUtGQ5PQFyQMQNnz6vgg\\\\\\\/L\\\\\\\/DA3oC\\\\\\\/15XDd7KHCWAh6BPLXaR1O29eIQo2biwyEevQWXYRnjNs1vTWG1NyPcTRmE+DS3GUaPgoX9INEdcRPow9I1aiu28yzZJmSlXf\\\\\\\/afyWZTkJ8LhxuOGK9IC+bZc6Gj2KpJHMw9gqYw6sVhRXnMy23JyDaIpbmhvzlAzmSbz45qmOp0fGqgs+vMG9B6CFhUuawAIX6Lq6gk2da3SWnxzA7MYC1roq6s0ppl\\\\\\\/NnD1lEByqdoVmsghPIjntvl+iNGRZM4nrGZ5OLCbaz5Pz48Hd6ynHqG7Gy\\\\\\\/lQ0uWxwS\\\\\\\/MsVo\\\\\\\/b59tMh6zZiAym0HO2SM8cAIvf1jcHHYig8XPRybFZRUXgYgbHFEMK7Hj0RlH8KYkYBmSMbaeLdPU5tlZD5fPngjYu5g00YYaEO6djMIXAVp9OEYzKOEdXYmEP5113ksDX\\\\\\\/rLp7G9F9SULYYG40Axf1HliXREZQekV0PAC1zf29QLCCp3NgYcySKGowaJjFGr7yDVpF+TmjckcDgRT2vY9+9HQaT20HQ\\\\\\\/RGoflvCvir01CXnCOhKh2qcsznF65C0yrZc8r4h6K\\\\\\\/2hRsw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022o7ZzmOuji3z8oUZocRK9xA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e52aa602b66b460d60a69e49eb9e4880e8da8c6b18d2785428f5e304be2ffb80\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/88W33WHAAWZDWZkqd80ssaOA5Ny9Ui2j4+12EhxhvN7U2XMzsDHlzdcAPms+6FmBdn12FzrMkR6DASQVQHo5EmRF3VZP3pKaCJSPNbAzRu0l24LI+Xmxrr2xc8tCXkaizOuSKM6CfEA82ywrfkyKECwXQ\\\\\\\/M8qEHYMflCcL4WiUY0Cg12J3VZ3vDBb\\\\\\\/3u0\\\\\\\/SKmQBLkGegSO3Y6MEBx54jt6SVIKmcSVOMYl9rMyVQvOVp7lsMoiwJ9AVtunDIgoHYuoelxSF98Cleg3cE+CZjq5Np1iz4YsyhqnKxNEuhwySVCd\\\\\\\/0jziiojt4XkrqE6fl2FxoKfekYbKuWSckHTo7thXJbWeFjtRSsDT3p0Tke6fn8qbqcA+Ol5AOys6jRWNT73n3S\\\\\\\/CkWrmtb3P5wLvzetIAXypoyuBgjYEKJ8prdVis4IBDrvEwSeeNFvld3+pB8cFux0GJeTH1zGYzBVLG1g4RB13E3+JnMwrW36MRzW4eGK+Cxtk65tiO2cya1+y0Ei14gFndi+tH2Jo5M3jPEd3hbCeEU8Bh20RO3HAggAfwcBAZR2ItXtvWf5v1s3ln0drQmYEWb4JJsHgBr5gambRQ0LZUgPkjgv4Rq6JSRROe5W7WJEbRiG+rKABMFn2WEJ+5MJpPs7679Ddq7lavV3daNVVonJmlOG5h7dk+Gl04820UPQ3nipKz+jlX40mDxQRg8\\\\\\\/5S2tSMxxwqlF8JMToyx6R\\\\\\\/01ZVn1eE4OEL9MiO3XwomwdMIRHsXBREpddXAkS7vFzq96gsEDgpmXhSEWLiE5ouR3AA4ajPNap\\\\\\\/a3zP1os0A0ePm5KUBqixEUezBA+iCTKDn140USX48gGbEi0OuP9k\\\\\\\/nX8YtgK3m79BnXo38Va\\\\\\\/iQEeaJlxoO6ZTxP30f7Irvn4\\\\\\\/cJWGxn+nYPDCGHnRZFZmff\\\\\\\/LlNOycNwVnT3fYQMo\\\\\\\/b9Aih\\\\\\\/nc\\\\\\\/WDK02mPvXGcGmZNG6k5P7sdboPe517nrul6vouvOjYxxnre9rDKABUF7XmctVyY1r4oQdP2JmS6UyT7RPdl9obj2U5ko3fG2QS4p1n3mOhmoSo8Y5koAGwq397vwc4nkDNzoEs\\\\\\\/3RvcM5hFcsIlCiVWogVCtVfpH3HHbRNo2sZrr27eM0sXP8i7JlxzopPi2xnSun4aXAiqP0FYE8+cSF5SYWE55iJX23YxRQlluHPGELXuL2IV78xFP6zkSgzs+t8sgSF6A7cG0eo0tfv\\\\\\\/szjPtJg0fSwK5PqhKvSQbCJUkTU5i2BPUVN6YTjJtZm6J0+mS2vOOB5DVMQWysR7bcN22\\\\\\\/\\\\\\\/O+blPi9dpeodSPmowo9XI9oCWPe\\\\\\\/guJ75UDj+IaizoKaGF3vC4O\\\\\\\/P8OY1dOzhuzKB9mgGgbZRHlmNNjvYTPRf5lgFvXmv6cyT22WRDcb9pFeBH3A8F3FMeZeUryBDlcfhV2m0zZ+XLoDTimoYeugrfKSW\\\\\\\/DUpR0ZWEtOLnTTkt6T9R1fEt\\\\\\\/AdjAK7q12r3o6s7u0kuanBXuXAqJFAoyPeYt83d4+AANXaEN9akNnMDA6mObUYOl3Ze93+S7ia3CaKokfxHyXQPe7Hcr0tm19JEKrR6zNvTjYnOzP9cPPIHoHRPI7V3cXG4KRvQuF4eBYsiJYZJ9xsY5hOrbBpaX\\\\\\\/Tdr9Dn4gosebKdz+OVVNR18QGEEi5qxTAnVYxVLKp2twL2Igs3cSDz+1YmP9A\\\\\\\/qn+B+ojLT7dXYH7R8Hzt0kHtORXQDqUt5LA3IqKOIA3xrrd+k5F9oTYnogW8MHhkODIVNN5fKfvRl64gWP1AHXpoxnxWD3euRethGxd19xQ9DGoCmp3Op9+Cu11V+VSwQb5qYDjS3TInzx+MxZd++KAplm8Crta2ygKZ5JfC60RlDJf35g6eAz5kZFeAR7o2s4VrOnyUnEoG1JvsOsqH2dWaW8jkrRIimzNdTeRSRyzHIggWdZTbwxVwOn4gjH+9cs5aB7CfeGf5BBZyMgQW9mok2oXFnyNph3NmNhWgYIeTA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nn4QCdx\\\\\\\/H46obWH5um+nRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c3430557e9f0100f0e5295452ed36734103431cab243537639bab8a4d30f18d9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488299 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/385" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7z1xocm447gYqg2GSuQeAaTUrFhJrECHI9QJryOc82c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C3pSi68wnAkGjqk3U8IqN7BLYQ6GnS-NSRNRGaJUmdU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/377" title="Whole Genome Sequencing of Hulunbuir Short-Tailed Sheep for Identifying Candidate Genes Related to the Short-Tail Phenotype" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/393" title="Reference Quality Genome Assemblies of Three Parastagonospora nodorum Isolates Differing in Virulence on Wheat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488299" id="node488299" data-pisa="ggg;8/2/385" data-pisa-master="ggg;g3.117.300331" data-apath="/ggg/8/2/385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Assembly and Annotation of the Medicinal Plant <em>Calotropis gigantea</em>, a Producer of Anticancer and Antimalarial Cardenolides</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Genevieve M. Hoopes</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John P. Hamilton</span>, <span class="highwire-citation-author" data-delta="2">Jeongwoon Kim</span>, <span class="highwire-citation-author" data-delta="3">Dongyan Zhao</span>, <span class="highwire-citation-author" data-delta="4">Krystle Wiegert-Rininger</span>, <span class="highwire-citation-author" data-delta="5">Emily Crisovan</span> and <span class="highwire-citation-author" data-delta="6">C. Robin Buell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">385-391;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300331 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488299" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Genevieve M. Hoopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Genevieve%2BM.%2BHoopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoopes%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGenevieve%2BM.%2BHoopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John P. Hamilton</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeongwoon Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeongwoon%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeongwoon%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dongyan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongyan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongyan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Krystle Wiegert-Rininger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krystle%2BWiegert-Rininger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiegert-Rininger%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrystle%2BWiegert-Rininger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Emily Crisovan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BCrisovan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crisovan%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BCrisovan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/385" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488299" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/385.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488299" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/385.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488299" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/385.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488299" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/385.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488299" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488299/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3"><em>Calotropis gigantea</em> produces specialized secondary metabolites known as cardenolides, which have anticancer and antimalarial properties. Although transcriptomic studies have been conducted in other cardenolide-producing species, no nuclear genome assembly for an Asterid cardenolide-producing species has been reported to date. A high-quality <em>de novo</em> assembly was generated for <em>C. gigantea</em>, representing 157,284,427 bp with an N50 scaffold size of 805,959 bp, for which quality assessments indicated a near complete representation of the genic space. Transcriptome data in the form of RNA-sequencing libraries from a developmental tissue series was generated to aid the annotation and construction of a gene expression atlas. Using an <em>ab initio</em> and evidence-driven gene annotation pipeline, 18,197 high-confidence genes were annotated. Homologous and syntenic relationships between <em>C. gigantea</em> and other species within the Apocynaceae family confirmed previously identified evolutionary relationships, and suggest the emergence or loss of the specialized cardenolide metabolites after the divergence of the Apocynaceae subfamilies. The <em>C. gigantea</em> genome assembly, annotation, and RNA-sequencing data provide a novel resource to study the cardenolide biosynthesis pathway, especially for understanding the evolutionary origin of cardenolides and the engineering of cardenolide production in heterologous organisms for existing and novel pharmaceutical applications.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/calotropis-gigantea" class="hw-term hw-article-keyword hw-article-keyword-calotropis-gigantea" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Calotropis gigantea</em></a></li><li class="kwd"><a href="/keyword/cardenolide" class="hw-term hw-article-keyword hw-article-keyword-cardenolide" rel="nofollow">cardenolide</a></li><li class="kwd"><a href="/keyword/apocynaceae-family" class="hw-term hw-article-keyword hw-article-keyword-apocynaceae-family" rel="nofollow">Apocynaceae family</a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/pharmaceutical" class="hw-term hw-article-keyword hw-article-keyword-pharmaceutical" rel="nofollow">pharmaceutical</a></li><li class="kwd"><a href="/keyword/genome-report" class="hw-term hw-article-keyword hw-article-keyword-genome-report" rel="nofollow">Genome Report</a></li></ul><p id="p-4">Species within the Apocynaceae family have traditionally been used to treat fever, pain, and malaria in the Asia-Pacific region, primarily due to their production of secondary metabolites with antiplasmodial properties (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chan <em>et al.</em> 2016</a>). Some of these secondary metabolites have also been found to contain antiproliferative properties and have been used to treat cancer. <em>Calotropis gigantea</em> (2<em>n</em> = 2<em>x</em> = 22) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Raghavan 1957</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), a member of the Apocynaceae subfamily Asclepiadoideae, is known to produce several cardenolides or cardiac glycosides including calactin, calotoxin, calotropin, frugoside, and gofruside (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chan <em>et al.</em> 2016</a>). Cardenolides are C<sub>23</sub> steroids with a butenolide ring at C-17, and are synthesized in plants from mevalonic acid via phytosterol and pregnane intermediates (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bauer <em>et al.</em> 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pandey <em>et al.</em> 2016</a>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1D</a>). While cardenolides have previously been used as clinical drugs for congestive heart failure, they have recently been found to selectively inhibit cancer cells through the induction of apoptosis via complex cell signal transduction pathways associated with the Na<sup>+</sup>/K<sup>+</sup> ATPase (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mijatovic and Kiss 2013</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/385/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="C. gigantea and cardenolide metabolites. (A) C. gigantea plant. (B) Plant tissues used for RNA-sequencing libraries. 1, closed bud 2; sepals and petals; 3, peduncle and pedicel; 4, young leaf; and 5, gynostegium. (C) Mature leaf tissue used for RNA-sequencing libraries. (D) Cardenolide metabolite with the carbons numbered. Rx and Ry indicate ambiguity of the attached groups, which vary depending on the specific cardenolide." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2016609361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;C. gigantea&lt;/em&gt; and cardenolide metabolites. (A) &lt;em&gt;C. gigantea&lt;/em&gt; plant. (B) Plant tissues used for RNA-sequencing libraries. 1, closed bud 2; sepals and petals; 3, peduncle and pedicel; 4, young leaf; and 5, gynostegium. (C) Mature leaf tissue used for RNA-sequencing libraries. (D) Cardenolide metabolite with the carbons numbered. R&lt;sub&gt;x&lt;/sub&gt; and R&lt;sub&gt;y&lt;/sub&gt; indicate ambiguity of the attached groups, which vary depending on the specific cardenolide.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/385/F1.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/385/F1.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/385/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/385/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488466" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child"><em>C. gigantea</em> and cardenolide metabolites. (A) <em>C. gigantea</em> plant. (B) Plant tissues used for RNA-sequencing libraries. 1, closed bud 2; sepals and petals; 3, peduncle and pedicel; 4, young leaf; and 5, gynostegium. (C) Mature leaf tissue used for RNA-sequencing libraries. (D) Cardenolide metabolite with the carbons numbered. R<sub>x</sub> and R<sub>y</sub> indicate ambiguity of the attached groups, which vary depending on the specific cardenolide.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">Cardenolides are currently harvested from plant tissue, a limited resource, and it is crucial to determine the complete biosynthetic pathway to engineer the production of cardenolides in heterologous organisms for pharmaceutical use. While cardenolide-producing species are distributed across 12 angiosperm families, they predominantly occur in the Asterid clade (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agrawal <em>et al.</em> 2012</a>), including the Apocynaceae family, in which published genome sequences are available for two noncardenolide-producing species, <em>Catharanthus roseus</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kellner <em>et al.</em> 2015</a>) and <em>Rhayza stricta</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sabir <em>et al.</em> 2016</a>). Access to an annotated genome for an Asterid cardenolide-producing species would facilitate discovery of the genes involved in cardenolide biosynthesis through comparative genomics, coexpression analyses, and data mining of physically clustered genes, a feature of some plant specialized metabolic pathways (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nützmann <em>et al.</em> 2016</a>). Here, we present an annotated draft genome sequence for <em>C. gigantea</em> and demonstrate the utility of this sequence in elucidating evolutionary relationships in the Apocynaceae family.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genome assembly and analyses</h3><p id="p-7">Genomic DNA was extracted from young leaf tissue via the CTAB method (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Saghai-Maroof <em>et al.</em> 1984</a>), and used to construct three Illumina-compatible (Illumina, San Diego, CA) paired-end libraries with estimated insert sizes of 168, 291, and 519 bp (Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>), as previously described (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hardigan <em>et al.</em> 2016</a>). Four Nextera Mate Pair libraries with estimated inner distances ranging from 2.7 to 8.8 kb were constructed per the manufacturer’s instructions (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). The libraries were sequenced on an Illumina HiSeq 2500 to generate paired-end 150 nt length reads. Read quality was assessed with FastQC (v0.11.5) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andrews 2010</a>), and adapters and bases of low quality (<em>Q</em> &lt; 20) were removed with Cutadapt (v1.11) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Martin 2011</a>), retaining reads &gt;81 nt in length. Mate pair library reads were further processed with NextClip (v1.3.1) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Leggett <em>et al.</em> 2014</a>), retaining paired reads &gt;36 nt in groups A, B, and C (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). K-mers (17, 23, 33, 43, and 63) were counted using JellyFish2 (v2.2.6) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Marçais and Kingsford 2011</a>) with ALLPATHS-LG (v52488) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gnerre <em>et al.</em> 2011</a>) error-corrected reads from all fragment libraries.</p><p id="p-8">Two common assemblers capable of handling heterozygous genomes, ALLPATHS-LG (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gnerre <em>et al.</em> 2011</a>) and Platanus (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kajitani <em>et al.</em> 2014</a>), were used to generate <em>de novo</em> genome assemblies. As ALLPATHS-LG has a fragment library size restriction (less than two times the insert size), only cleaned reads from the 168 bp paired-end library (numbering 204,292,990) and all processed mate pair library reads (34,232,248) were used with the “haploidify” option in ALLPATHS-LG (v52488) (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gnerre <em>et al.</em> 2011</a>), while cleaned reads from all libraries (278,764,758) and all mate pair library reads were used with the default options in Platanus (v1.2.4) (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kajitani <em>et al.</em> 2014</a>). After filtering both assemblies for scaffolds &gt;1000 bp, BUSCO (v2.0) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Simão <em>et al.</em> 2015</a>) was run with the Embryophyta database on both assemblies to determine the representation of conserved plant orthologs.</p><p id="p-9">To reduce the number and size of the gaps, GapCloser (v1.12r6) from SOAPdenovo (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Luo <em>et al.</em> 2012</a>) was run with the 291 and 519 bp fragment library paired-end reads (148,943,536), with maximum read length of 150 bp and the overlap parameter set to 31. To identify any contaminants in the assembly, BLASTN (BLAST+ v2.2.31) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Camacho <em>et al.</em> 2009</a>) was used to search the gap-closed genomic assembly against the National Center for Biotechnology Information (NCBI) nr database (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">NCBI Resource Coordinators 2016</a>). BUSCO (v2.0) (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Simão <em>et al.</em> 2015</a>) was subsequently run with the Embryophyta database on the final genome assembly.</p><p id="p-10">BWA-MEM (v0.7.15) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li and Durbin 2009</a>) was used to align the cleaned reads from all fragment libraries to the assembly, marking secondary alignments (-M). Picard MarkDuplicates (v2.1.1; <a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>) was used to mark PCR and optical duplicates. Picard-processed alignments were realigned around InDels using the GATK RealignerTargetCreator and IndelRealigner tools (v3.7.0; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McKenna <em>et al.</em> 2010</a>) with default options. The HaplotypeCaller tool was subsequently used to call variants with the standard_min_confidence_threshold_for_calling, min_base_quality, heterozygosity, and indel_heterozygosity options set to 30, 20, 0.05, and 0.001, respectively (v3.7.0; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">McKenna <em>et al.</em> 2010</a>). Variants passed filtering with the GATK VariantFitration tool if they had read depth between 50 and 200, a quality depth &gt;7, and a BaseQRankSum value between −5 and 5 (v3.7.0; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">McKenna <em>et al.</em> 2010</a>). Heterozygous variants were further filtered such that only variants with a 30% ≤ allele frequency ≥ 70% were included in the analyses.</p></div><div id="sec-3" class="subsection"><h3>Transcriptome assemblies and analyses</h3><p id="p-11">RNA was extracted from closed bud, sepal and petal, young leaf, peduncle and pedicel, gynostegium, and mature leaf tissues (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, B and C</a>) via the QIAGEN RNeasy Plant Mini Kit, according to the manufacturer’s instructions (QIAGEN, Hilden, Germany), and treated with Turbo DNase to remove any DNA from the samples. Six KAPA Biosystems (Wilmington, MA) stranded libraries with NEBNext indices (Ipswich, MA) were constructed for each of the tissues and sequenced on an Illumina HiSeq 2500 platform, generating 50 nt single-end reads (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). The RNA-seq reads were then processed with FastQC (v0.11.5; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Andrews 2010</a>) and Cutadapt (v1.11; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Martin 2011</a>) as described above, retaining reads &gt;30 nt in length.</p><p id="p-12"><em>De novo</em>-assembled transcriptomes were generated for each RNA-seq library using Trinity (v2.3.2; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grabherr <em>et al.</em> 2011</a>) in the strand-specific mode, and transcripts shorter than 500 bp were removed from the assemblies. The transcripts were aligned to the genome assembly using GMAP (v20161107; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wu and Watanabe 2005</a>) with a 95% coverage and identity cutoff, and unaligned transcripts were searched against the nr database (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">NCBI Resource Coordinators 2016</a>) using BLASTN from BLAST+ (v2.2.31; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Camacho <em>et al.</em> 2009</a>) with an <em>E</em>-value cutoff of 1e<sup>−5</sup>. Cleaned reads for each library were also aligned to the genome assembly using TopHat2 (v2.1.1; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kim <em>et al.</em> 2013</a>) and Bowtie2 (v2.2.9; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Langmead and Salzberg 2012</a>) in stranded mode, with a maximum intron length of 30 kb.</p></div><div id="sec-4" class="subsection"><h3>Genome annotation</h3><p id="p-13">A <em>C. gigantea</em>-specific custom repeat library (CRL) was created with RepeatModeler (v1.0.8; <a href="http://repeatmasker.org">http://repeatmasker.org</a>) using scaffolds &gt;10 kb as input. The CRL was searched against a curated library of plant protein-coding genes and sequences, removing matches with ProtExcluder (v1.1; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Campbell <em>et al.</em> 2014</a>), and then combined with the RepBase (v20150807; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jurka 1998</a>) Viridiplantae repeats to create a final CRL. The genome assembly was then masked with RepeatMasker (v4.0.6; <a href="http://repeatmasker.org">http://repeatmasker.org</a>) using the CRL with the –s option.</p><p id="p-14">Genome-guided transcript assemblies were constructed for each RNA-seq library using Trinity (v2.3.2; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Grabherr <em>et al.</em> 2011</a>) and transcripts shorter than 500 bp were removed from the assemblies. <em>Ab initio</em> gene prediction was performed by training Augustus (v3.2.2; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Stanke <em>et al.</em> 2006</a>), using the hints provided by the alignments of the mature leaf RNA-seq library and the soft-masked genome assembly. Gene predictions were then obtained by running Augustus (v3.2.2) on the hard-masked genome assembly and refined using PASA2 (v2.0.2; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Haas <em>et al.</em> 2003</a>), utilizing the genome-guided transcript assemblies as evidence. To identify high-confidence gene models, the working set of gene models was searched against PFAM (v29; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Finn <em>et al.</em> 2016</a>) using HMMER (v3.1b2; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mistry <em>et al.</em> 2013</a>). Expression abundances were also calculated using Cufflinks2 (v2.21; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Trapnell <em>et al.</em> 2010</a>) with the RNA-seq alignments described above. High-confidence gene models are defined as containing a PFAM hit and/or expression evidence in at least one RNA-seq library. Functional annotation was generated by searching the gene models against the <em>Arabidopsis thaliana</em> proteome (TAIR10; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lamesch <em>et al.</em> 2012</a>), Swiss-Prot (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bairoch and Apweiler 2000</a>), and PFAM (v29; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Finn <em>et al.</em> 2016</a>), and assigning a function in the same order.</p></div><div id="sec-5" class="subsection"><h3>Comparative genomics analyses</h3><p id="p-15">Orthologous and paralogous genes in the three Apocynaceae species, along with the annotated proteomes of <em>Amborella trichopoda</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amborella Genome Project 2013</a>) and <em>A. thaliana</em> (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lamesch <em>et al.</em> 2012</a>; Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>), were determined using OrthoFinder (v1.1.4; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Emms and Kelly 2015</a>) using default settings. To evaluate synteny between the species in the Apocynaceae family, MCScanX (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang <em>et al.</em> 2012</a>) was used with <em>R. stricta</em>, <em>C. roseus</em>, and <em>C. gigantea</em>. The predicted proteomes for the three species were searched against each other using BLASTP from BLAST+ (v2.5.0; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Camacho <em>et al.</em> 2009</a>), with an <em>E</em>-value cutoff of 1e<sup>−5</sup> and a maximum of five hits reported.</p></div><div id="sec-6" class="subsection"><h3>Identification of putative cardenolide biosynthetic enzymes</h3><p id="p-16">The <em>C. gigantea</em> proteome was searched against functionally characterized 3β-hydroxysteroid dehydrogenases (3βHSD; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Herl <em>et al.</em> 2007</a>) and progesterone 5β-reductases (P5βR; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bauer <em>et al.</em> 2010</a>) using BLASTP from BLAST+ (v2.5.0; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Camacho <em>et al.</em> 2009</a>) with an <em>E</em>-value cutoff of 1e<sup>−5</sup>. Matches with percent identity &gt;50% and query coverage &gt;85% were retained. The identified <em>C. gigantea</em> proteins were aligned with the characterized 3βHSD and P5βR proteins using MUSCLE (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Edgar 2004</a>) as implemented in MEGA7 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kumar <em>et al.</em> 2016</a>). Neighbor-joining gene trees were constructed using default parameters with pairwise deletion and 1000 bootstrap replicates in MEGA7 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kumar <em>et al.</em> 2016</a>). Previously obtained fragments per kilobase exon model per million mapped reads (FPKM) values from Cufflinks2 (v2.21; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Trapnell <em>et al.</em> 2010</a>) for the identified genes were log2-transformed and hierarchically clustered in R (v3.4.2) using the “gplots” package (<a href="https://CRAN.R-project.org/package=gplots">https://CRAN.R-project.org/package=gplots</a>) “heatmap.2” function.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-17">Raw sequence reads have been deposited with the NCBI under BioProject PRJNA400797. The genome assembly, annotation, and expression matrix have been deposited in the Dryad Digital Repository (DOI: 10.5061/dryad.fk41r) and the Medicinal Plant Genomics Resource (<a href="http://medicinalplantgenomics.msu.edu/">http://medicinalplantgenomics.msu.edu/</a>). The supplementary tables are included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>.</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>De novo genome assembly generation</h3><p id="p-18">Flow cytometry of <em>C. gigantea</em> leaf tissue revealed a genome size of 225 Mb, and the genome was sequenced on an Illumina HiSeq platform to 193× average genomic coverage with the fragment library reads. To determine the relative level of heterozygosity in the genome, five k-mer lengths (17, 23, 33, 43, and 63) were counted (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In the 17-mer plot, there are two peaks at k-mer depths of 140 and 69 (∼2:1 ratio), corresponding to homozygous and heterozygous regions in the genome, respectively. The ratio between the homozygous peak:heterozygous peak is 4:1 and 1.5:1 in the 23-mer plot and 63-mer plot, respectively. The presence of the small heterozygous peak and the disproportionate increase in the heterozygous peak relative to the homozygous peak as the k-mer size increases is indicative of a slightly heterozygous genome.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/385/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heterozygosity of the C. gigantea genome and assembly. K-mer frequency plots generated in Jellyfish2 (Mar&#xE7;ais and Kingsford 2011) using ALLPATHS-LG (Gnerre et al. 2011) error-corrected genomic fragment reads." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2016609361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heterozygosity of the &lt;em&gt;C. gigantea&lt;/em&gt; genome and assembly. K-mer frequency plots generated in Jellyfish2 (Mar&#xE7;ais and Kingsford 2011) using ALLPATHS-LG (Gnerre &lt;em&gt;et al.&lt;/em&gt; 2011) error-corrected genomic fragment reads.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/385/F2.medium.gif" width="440" height="148"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/385/F2.medium.gif" width="440" height="148"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/385/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/385/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488412" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Heterozygosity of the <em>C. gigantea</em> genome and assembly. K-mer frequency plots generated in Jellyfish2 (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Marçais and Kingsford 2011</a>) using ALLPATHS-LG (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Gnerre <em>et al.</em> 2011</a>) error-corrected genomic fragment reads.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Two common assemblers capable of handling heterozygous genomes, ALLPATHS-LG (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Gnerre <em>et al.</em> 2011</a>) and Platanus (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Kajitani <em>et al.</em> 2014</a>), were used to generate <em>de novo</em> genome assemblies. Both ALLPATHS-LG, via the “haploidify” option, and Platanus contain algorithms that select a single haplotype from heterozygous regions to be incorporated into the assembly, resulting in a mixture of the two haplotypes in the final assembly (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Gnerre <em>et al.</em> 2011</a>; <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Kajitani <em>et al.</em> 2014</a>). ALLPATHS-LG generated an assembly of 157,408,176 bp with an N50 scaffold size of 806,518 bp and Platanus generated an assembly of 146,937,509 bp with an N50 scaffold size of 187,271 bp (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The ALLPATHS-LG and Platanus assemblies contained 91.3% (87.8% complete and 3.5% fragmented) and 85.1% (77.2% complete and 7.9% fragmented) of BUSCO Embryophyta orthologs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), respectively. As ALLPATHS-LG generated a less fragmented, more complete assembly representing more of the genic space, subsequent work was performed with the ALLPATHS-LG assembly.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488468/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488468/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genome assembly metrics</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">The ALLPATHS-LG assembly was further refined by filling gaps and removing scaffolds from contaminant species. Gaps were reduced in the assembly from 13,729, accounting for 18,606,682 bp or 11.8% of the assembly, to 5068, accounting for 8,276,177 bp or 5.3% of the new gap-closed assembly. Two scaffolds in the gap-closed assembly had matches (&gt;95% identity and &gt;50% coverage) to spider mite (<em>Tetranychus urticae</em>) ribosomal DNA and were removed, generating a final genomic assembly of 157,284,427 bp with an N50 scaffold size of 805,959 bp (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-22">To assess the completeness and accuracy of the final assembly, the presence of conserved plant orthologs and the numbers of nucleotide variants were determined. A total of 92.0% (89.8% complete and 2.2% fragmented) of BUSCO orthologs were present in the assembly (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), indicating a near complete representation of the genic space. After marking PCR duplicates, 94.6% of the reads mapped to the final genomic assembly; of the reads that mapped, 95.0% were properly paired. Using these alignments, a total of 1,270,680 initial variants (SNPs and InDels) were identified and reduced to 593,265 after filtering with the GATK VariantFiltration tool (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">McKenna <em>et al.</em> 2010</a>), and separately by allele frequency, with 364,465 variants being heterozygous sites and 228,800 representing assembly errors. The heterozygous and assembly error variants account for 0.23 and 0.15% of the assembly, respectively, indicative of a relatively error-free and slightly heterozygous genome with one heterozygous variant per 432 bp.</p></div><div id="sec-10" class="subsection"><h3>Quality assessment of de novo-assembled transcriptomes and genome annotation</h3><p id="p-23">To determine if any transcripts represented potential contaminants, such as microbial or insect RNA commonly present in greenhouse-grown plants, <em>de novo</em>-assembled transcriptomes from each RNA-seq library (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>) were aligned to the genome and searched against the NCBI nr database. Between 65.7 and 71.6% of unaligned transcripts contained a match to the database, with 80.7–98.7% of these containing a best match to a species in the Viridiplantae kingdom. Libraries from the sepal/petal and gynostegium tissues (SRR6078591 and SRR6078593, respectively) contained a higher percentage of best matches to non-Viridiplantae species at 14.5 and 19.3% (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>), respectively, with thrips (<em>Frankliniella occidentalis</em>), an insect typically associated with floral structures, the most prevalent species. After removing the non-Viridiplantae transcripts, between 93.1 and 94.2% of all transcripts for each library aligned to the genome, with 90.1–91.5% aligning uniquely (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). Excluding SRR6078591 and SRR6078593, which had the most contamination with nonplant sequences, between 93.2 and 94.3% of the RNA-seq reads aligned to the genome assembly (Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>), consistent with the percentage of <em>de novo</em>-generated Trinity transcripts that aligned to the genome after removal of contamination and indicating a robust representation of the genic space.</p><p id="p-24">After confirming the quality of the RNA-seq data, genome annotation was performed. A total of 117,995 repetitive elements, representing 28.3% of the genome assembly, were identified (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). The final gene model working set consisted of 19,536 loci and 22,218 gene models, with 20,832 gene models (18,197 loci) identified as high confidence. A total of 18,436 high-confidence gene models were assigned a putative function, 1264 were annotated as conserved hypothetical, and 1132 were annotated as hypothetical.</p></div><div id="sec-11" class="subsection"><h3>Homologous and syntenic relationships in the Apocynaceae family</h3><p id="p-25">The Apocynaceae family is diverse, containing five subfamilies and 400 genera that produce diversified specialized metabolites (<a href="http://www.mobot.org/MOBOT/Research/APweb/welcome.html">http://www.mobot.org/MOBOT/Research/APweb/welcome.html</a>). Despite the large number of species in the family, only two have an annotated genome assembly, <em>C. roseus</em> and <em>R. stricta</em> (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kellner <em>et al.</em> 2015</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Sabir <em>et al.</em> 2016</a>), both of which are in the Rauvolfioideae subfamily. A total of 13,679 orthologous groups were identified from comparisons between <em>A. trichopoda</em>, <em>A. thaliana</em>, <em>C. roseus</em>, <em>R. stricta</em>, and <em>C. gigantea</em> (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>), of which 8989 groups contained all five species (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). <em>C. gigantea</em> had few singleton genes, with 90.0% of its genes, the most of any species, in 12,118 orthologous groups and 11 paralogous groups; <em>C. gigantea</em>-specific genes included disease-resistance genes, F-box family proteins, and hypothetical genes suggestive of genes rapidly evolving (Table S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). We identified 590 Apocynaceae-specific orthologous groups containing only genes from <em>C. gigantea</em>, <em>C. roseus</em>, and <em>R. stricta</em>, whereas 312 orthologous groups contained only <em>C. roseus</em> and <em>R. stricta</em>, almost double the number shared between <em>C. gigantea</em> and <em>C. rosesus</em> or <em>C. gigantea</em> and <em>R. stricta</em>, highlighting the evolutionary relationships between the species, as <em>C. roseus</em> and <em>R. stricta</em> are in the Rauvolfioideae subfamily while <em>C. gigantea</em> is in the Asclepiadoideae subfamily (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chan <em>et al.</em> 2016</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Sabir <em>et al.</em> 2016</a>). The rooted species tree from OrthoFinder (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Emms and Kelly 2015</a>) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), constructed using the median distances for the genes in orthologous groups, supports this relationship.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/385/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Homologous and syntenic relationships among the Apocynaceae family. (A) Venn diagram showing the number of orthologous and paralogous groups shared among the species. The number of singletons per species is also provided. (B) Rooted cladogram for the species generated in the OrthoFinder (Emms and Kelly 2015) output." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2016609361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Homologous and syntenic relationships among the Apocynaceae family. (A) Venn diagram showing the number of orthologous and paralogous groups shared among the species. The number of singletons per species is also provided. (B) Rooted cladogram for the species generated in the OrthoFinder (Emms and Kelly 2015) output.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/385/F3.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/2/385/F3.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/385/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/385/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488401" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Homologous and syntenic relationships among the Apocynaceae family. (A) Venn diagram showing the number of orthologous and paralogous groups shared among the species. The number of singletons per species is also provided. (B) Rooted cladogram for the species generated in the OrthoFinder (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Emms and Kelly 2015</a>) output.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Syntenic relationships among the species in the Apocynaceae family were determined to further examine the evolutionary relationships. A total of 1836 collinear blocks were present in a comparison of all three species with each species containing a similar number of collinear genes (∼12,500–13,000) (Table S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). In pairwise comparisons, <em>C. roseus</em> and <em>C. gigantea</em> each had 549 and 482 more collinear genes, respectively when paired together compared to when either was paired with <em>R. stricta</em>; however, <em>R. stricta</em> had virtually no difference in the number of collinear genes between its pairing with <em>C. gigantea</em> and <em>C. roseus</em> (Table S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). The relatively small difference in number of collinear genes between the three species, despite there being almost twice as many orthologous groups between <em>C. roseus</em> and <em>R. stricta</em> compared to <em>C. gigantea</em>, could be due to a number of reasons, including: a dispersed distribution of orthologous genes in the genome thereby not meeting our threshold for a collinear block, differences in gene annotation methods among the three genomes, and the collective fragmentation of the assemblies from the three species. Regardless, the expected evolutionary relationships were reflected in the orthologous groupings and suggest the emergence or loss of the cardenolide metabolites subsequent to the divergence of the Apocynaceae subfamilies, as <em>C. roseus</em> and <em>R. stricta</em> do not produce cardenolides.</p></div><div id="sec-12" class="subsection"><h3>Future uses for the C. gigantea genome assembly and annotation</h3><p id="p-28">Cardenolides, specialized compounds produced by <em>C. gigantea</em>, are known to contain anticancer and antimalarial properties (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Chan <em>et al.</em> 2016</a>). Yet, while much is known about the compounds themselves, knowledge of the biosynthetic pathways involved is limited (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bauer <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Munkert <em>et al.</em> 2014</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Pandey <em>et al.</em> 2016</a>). Furthermore, to the best of our knowledge, no nuclear genome assembly has been published for any cardenolide-producing species in the Asterid clade. The <em>C. gigantea de novo</em> genome assembly, annotation, and RNA-seq expression data provide a valuable new resource for the further study of the biosynthesis of cardenolides with applications in the pharmaceutical industry.</p><p id="p-29">With the use of this resource, both evolutionary and coexpression relationships can be utilized to elucidate genes involved in the cardenolide biosynthetic pathway. As demonstrated here, comparative genomic analyses between <em>C. gigantea</em> and other species in the Apocynaceae family can elucidate evolutionary events that led to the emergence of their corresponding specialized secondary metabolites. For example, identification of orthologous genes and syntenic blocks can facilitate the elucidation of genes of interest in the cardenolide biosynthetic pathway as they may be enriched or unique to <em>C. gigantea</em>, or physically clustered in the genome as shown in other species (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Denoeud <em>et al.</em> 2014</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Nützmann <em>et al.</em> 2016</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Sabir <em>et al.</em> 2016</a>). Another method used to identify candidate specialized metabolic pathway genes is coexpression (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Itkin <em>et al.</em> 2013</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kellner <em>et al.</em> 2015</a>), which the developmental <em>C. gigantea</em> RNA-seq data sets can facilitate. Two enzymes known to catalyze key parts of the cardenolide biosynthetic pathway in other cardenolide-producing species (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) were examined in <em>C. gigantea</em>; eight and two <em>C. gigantea</em> genes had sequence similarity to functionally characterized 3βHSD (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Herl <em>et al.</em> 2007</a>) and P5βR (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Bauer <em>et al.</em> 2010</a>) proteins, respectively (Table S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1/FileS1.docx">File S1</a>). Neighbor-joining gene trees with the <em>C</em>. <em>gigantea</em> candidates, and functionally characterized 3βHSD and P5βR genes, indicate that three genes (cal_g004469, cal_g004470, and cal_g016079), classified as paralogous by OrthoFinder, are most closely related to <em>Digitalis</em> 3βHSD proteins (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a>). Of the two <em>C. gigantea</em> P5βR paralogous genes, cal_g019537 was closely related to the <em>Calotropis procera</em> P5βR protein. Hierarchical clustering of normalized expression values for putative <em>C. gigantea</em> 3βHSD and P5βR genes indicates that the putative 3βHSD gene cal_g001209 and the putative P5βR gene cal_g019537 are coordinately expressed (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4D</a>), and are strong candidates for functional validation of the cardenolide biosynthetic pathway in <em>C. gigantea</em>. This case study demonstrates the capability of the <em>C. gigantea de novo</em> genome assembly, annotation, and RNA-seq expression data in determining putative genes involved in the cardenolide biosynthetic pathway.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/385/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of putative C. gigantea genes involved in the cardenolide biosynthetic pathway. (A). Simplified schematic of the cardenolide biosynthetic pathway. (B and C) are neighbor-joining gene trees for 3&#x3B2;-hydroxysteroid dehydrogenase (3&#x3B2;HSD) and progesterone 5&#x3B2;-reductase (P5&#x3B2;R), respectively. Taxa for each tree are the C. gigantea candidates and functionally characterized proteins, for which the taxa labels include the GenBank identifier and species name; taxa are shaded according to their distance from the functionally characterized genes, with darker shades indicating smaller distances. Values on nodes indicate bootstrap support from 1000 bootstrap replicates. (D) Heat map of log2-transformed gene expression values (FPKM: fragments per kb exon model per million mapped reads) of candidate C. gigantea cardenolide biosynthesis genes. Cladograms were generated from conducting hierarchical clustering on the genes and samples. Blue and red colored genes are 3&#x3B2;HSD and P5&#x3B2;R candidates, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2016609361" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of putative &lt;em&gt;C. gigantea&lt;/em&gt; genes involved in the cardenolide biosynthetic pathway. (A). Simplified schematic of the cardenolide biosynthetic pathway. (B and C) are neighbor-joining gene trees for 3&#x3B2;-hydroxysteroid dehydrogenase (3&#x3B2;HSD) and progesterone 5&#x3B2;-reductase (P5&#x3B2;R), respectively. Taxa for each tree are the &lt;em&gt;C. gigantea&lt;/em&gt; candidates and functionally characterized proteins, for which the taxa labels include the GenBank identifier and species name; taxa are shaded according to their distance from the functionally characterized genes, with darker shades indicating smaller distances. Values on nodes indicate bootstrap support from 1000 bootstrap replicates. (D) Heat map of log2-transformed gene expression values (FPKM: fragments per kb exon model per million mapped reads) of candidate &lt;em&gt;C. gigantea&lt;/em&gt; cardenolide biosynthesis genes. Cladograms were generated from conducting hierarchical clustering on the genes and samples. Blue and red colored genes are 3&#x3B2;HSD and P5&#x3B2;R candidates, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/385/F4.medium.gif" width="362" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/2/385/F4.medium.gif" width="362" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/385/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/385/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488375" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Identification of putative <em>C. gigantea</em> genes involved in the cardenolide biosynthetic pathway. (A). Simplified schematic of the cardenolide biosynthetic pathway. (B and C) are neighbor-joining gene trees for 3β-hydroxysteroid dehydrogenase (3βHSD) and progesterone 5β-reductase (P5βR), respectively. Taxa for each tree are the <em>C. gigantea</em> candidates and functionally characterized proteins, for which the taxa labels include the GenBank identifier and species name; taxa are shaded according to their distance from the functionally characterized genes, with darker shades indicating smaller distances. Values on nodes indicate bootstrap support from 1000 bootstrap replicates. (D) Heat map of log2-transformed gene expression values (FPKM: fragments per kb exon model per million mapped reads) of candidate <em>C. gigantea</em> cardenolide biosynthesis genes. Cladograms were generated from conducting hierarchical clustering on the genes and samples. Blue and red colored genes are 3βHSD and P5βR candidates, respectively.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">Funding for this project was provided by the Michigan State University Foundation. G.M.H. was supported by a fellowship from the Michigan State University Plant Sciences Fellowship Program.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-32">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300331/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-33"><em>Communicating editor: J. Udall</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-10-03"><span class="received-label">Received </span>October 3, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-12-10"><span class="accepted-label">Accepted </span>December 10, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Hoopes <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.1" data-doi="10.1111/j.1469-8137.2011.04049.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agrawal</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petschenka</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bingham</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmann</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Toxic cardenolides: chemical ecology and coevolution of specialized plant-herbivore interactions.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DAgrawal%26rft.auinit1%253DA.%2BA.%26rft.volume%253D194%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D45%26rft.atitle%253DToxic%2Bcardenolides%253A%2Bchemical%2Becology%2Band%2Bcoevolution%2Bof%2Bspecialized%2Bplant-herbivore%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.04049.x%26rft_id%253Dinfo%253Apmid%252F22292897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.04049.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22292897&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300719400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.2" data-doi="10.1126/science.1241089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Amborella Genome Project</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Amborella genome and the evolution of flowering plants.</span> <abbr class="cit-jnl-abbrev">Science.</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">1241089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAmborella%2BGenome%2BProject%26rft.auinit1%253D%2B%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1241089%26rft.epage%253D1241089%26rft.atitle%253DThe%2BAmborella%2BGenome%2Band%2Bthe%2BEvolution%2Bof%2BFlowering%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1241089%26rft_id%253Dinfo%253Apmid%252F24357323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDIvNjE2NS8xMjQxMDg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-8.2.385.3"><div class="cit-metadata unstructured">Andrews, S., 2010 FastQC: a quality control tool for high throughput sequence data. Available at: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.4" data-doi="10.1093/nar/28.1.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bairoch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBairoch%26rft.auinit1%253DA.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D48%26rft.atitle%253DThe%2BSWISS-PROT%2Bprotein%2Bsequence%2Bdatabase%2Band%2Bits%2Bsupplement%2BTrEMBL%2Bin%2B2000%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.45%26rft_id%253Dinfo%253Apmid%252F10592178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592178&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.5" data-doi="10.1016/j.phytochem.2010.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munkert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brydziun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burda</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller-Uri</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Highly conserved progesterone 5β-reductase genes (P5βR) from 5β-cardenolide-free and 5β-cardenolide-producing angiosperms.</span> <abbr class="cit-jnl-abbrev">Phytochemistry</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1495</span>–<span class="cit-lpage">1505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochemistry%26rft.volume%253D71%26rft.spage%253D1495%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.phytochem.2010.06.004%26rft_id%253Dinfo%253Apmid%252F20598327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.phytochem.2010.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20598327&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.6" data-doi="10.1186/1471-2105-10-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST plus: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMa%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DRegularized%2Bgene%2Bselection%2Bin%2Bcancer%2Bmicroarray%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-1%26rft_id%253Dinfo%253Apmid%252F19118496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19118496&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.7" data-doi="10.1104/pp.113.230144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moghe</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">MAKER-P: a tool kit for the rapid creation, management, and quality control of plant genome annotations.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.230144%26rft_id%253Dinfo%253Apmid%252F24306534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjQvMi81MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">E. W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">H. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Apocynaceae species with antiproliferative and/or antiplasmodial properties: a review of ten genera.</span> <abbr class="cit-jnl-abbrev">J. Integr. Med.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BMed.%26rft.volume%253D14%26rft.spage%253D269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.9" data-doi="10.1126/science.1255274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denoeud</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carretero-Paulet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dereeper</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Droc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guyot</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The coffee genome provides insight into the convergent evolution of caffeine biosynthesis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDenoeud%26rft.auinit1%253DF.%26rft.volume%253D345%26rft.issue%253D6201%26rft.spage%253D1181%26rft.epage%253D1184%26rft.atitle%253DThe%2Bcoffee%2Bgenome%2Bprovides%2Binsight%2Binto%2Bthe%2Bconvergent%2Bevolution%2Bof%2Bcaffeine%2Bbiosynthesis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1255274%26rft_id%253Dinfo%253Apmid%252F25190796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMS8xMTgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.10" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.11" data-doi="10.1186/s13059-015-0721-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emms</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0721-2%26rft_id%253Dinfo%253Apmid%252F26243257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0721-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243257&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.12" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Pfam protein families database: towards a more sustainable future.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D279</span>–<span class="cit-lpage">D285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F26673716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26673716&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.13" data-doi="10.1073/pnas.1017351108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccallum</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017351108%26rft_id%253Dinfo%253Apmid%252F21187386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.14" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.15" data-doi="10.1093/nar/gkg770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Improving the <em>Arabidopsis</em> genome annotation using maximal transcript alignment assemblies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">5654</span>–<span class="cit-lpage">5666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D31%26rft.issue%253D19%26rft.spage%253D5654%26rft.epage%253D5666%26rft.atitle%253DImproving%2Bthe%2BArabidopsis%2Bgenome%2Bannotation%2Busing%2Bmaximal%2Btranscript%2Balignment%2Bassemblies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg770%26rft_id%253Dinfo%253Apmid%252F14500829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14500829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185600400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.16" data-doi="10.1105/tpc.15.00538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardigan</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisovan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laimbeer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome reduction uncovers a large dispensable genome and adaptive role for copy number variation in asexually propagated <em>Solanum tuberosum</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">388</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00538%26rft_id%253Dinfo%253Apmid%252F26772996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI4LzIvMzg4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herl</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankenstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meitinger</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller-Uri</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreis</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Δ5–3β-hydroxysteroid dehydrogenase (3βHSD) from <em>Digitalis lanata</em>. Heterologous expression and characterisation of the recombinant enzyme.</span> <abbr class="cit-jnl-abbrev">Planta Med.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">710</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%2Bmedica%26rft.stitle%253DPlanta%2BMed%26rft.aulast%253DHerl%26rft.auinit1%253DV.%26rft.volume%253D73%26rft.issue%253D7%26rft.spage%253D704%26rft.epage%253D710%26rft.atitle%253DDelta%2B5-3beta-hydroxysteroid%2Bdehydrogenase%2B%25283%2Bbeta%2BHSD%2529%2Bfrom%2BDigitalis%2Blanata.%2BHeterologous%2Bexpression%2Band%2Bcharacterisation%2Bof%2Bthe%2Brecombinant%2Benzyme.%26rft_id%253Dinfo%253Apmid%252F17564944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17564944&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.18" data-doi="10.1126/science.1240230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinig</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzfadia</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhide</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinde</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Biosynthesis of antinutritional alkaloids in Solanaceous crops is mediated by clustered genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1240230%26rft_id%253Dinfo%253Apmid%252F23788733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0Mi8xNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.19" data-doi="10.1016/S0959-440X(98)80067-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Repeats in genomic DNA: mining and meaning.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DJurka%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D333%26rft.epage%253D337%26rft.atitle%253DRepeats%2Bin%2Bgenomic%2BDNA%253A%2Bmining%2Band%2Bmeaning.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-440X%252898%252980067-5%26rft_id%253Dinfo%253Apmid%252F9666329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-440X(98)80067-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9666329&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074573200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.20" data-doi="10.1101/gr.170720.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kajitani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toshimoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noguchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170720.113%26rft_id%253Dinfo%253Apmid%252F24755901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTM4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.21" data-doi="10.1111/tpj.12827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellner</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clavijo</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-guided investigation of plant natural product biosynthesis.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D82%26rft.spage%253D680%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12827%26rft_id%253Dinfo%253Apmid%252F25759247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12827&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25759247&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.22" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.23" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.24" data-doi="10.1093/nar/gkr1090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamesch</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarbreck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1202</span>–<span class="cit-lpage">1210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253D1202%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.25" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.26" data-doi="10.1093/bioinformatics/btt702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leggett</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clavijo</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clissold</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caccamo</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Next clip: an analysis and read preparation tool for nextera long mate pair libraries.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">566</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt702%26rft_id%253Dinfo%253Apmid%252F24297520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24297520&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.27" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.28" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read <em>de novo</em> assembler.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.29" data-doi="10.1093/bioinformatics/btr011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marçais</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsford</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast, lock-free approach for efficient parallel counting of occurrences of <em>k</em>-mers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">764</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr011%26rft_id%253Dinfo%253Apmid%252F21217122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217122&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288277300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads.</span> <abbr class="cit-jnl-abbrev">EMBnet.journal</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.31" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit : a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.32" data-doi="10.1055/s-0032-1328243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mijatovic</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cardiotonic STEROIDS-MEDIATED Na+/K+-ATPase targeting could circumvent various chemoresistance pathways.</span> <abbr class="cit-jnl-abbrev">Planta Med.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%2BMed.%26rft.volume%253D79%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1055%252Fs-0032-1328243%26rft_id%253Dinfo%253Apmid%252F23412992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1055/s-0032-1328243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23412992&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.33" data-doi="10.1093/nar/gkt263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Punta</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">e121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt263%26rft_id%253Dinfo%253Apmid%252F23598997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23598997&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munkert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller-Uri</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreis</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification and stress-induced expression of three 3β- hydroxysteroid dehydrogenases from <em>Erysimum crepidifolium</em> Rchb. and their putative role in cardenolide biosynthesis.</span> <abbr class="cit-jnl-abbrev">Phytochemistry</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochemistry%26rft.volume%253D100%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.35" data-doi="10.1093/nar/gkv1290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">NCBI Resource Coordinators</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Database resources of the National Center for Biotechnology Information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D7</span>–<span class="cit-lpage">D19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1290%26rft_id%253Dinfo%253Apmid%252F26615191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26615191&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.36" data-doi="10.1111/nph.13981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nützmann</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osbourn</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Plant metabolic clusters - from genetics to genomics.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D211%26rft.spage%253D771%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13981%26rft_id%253Dinfo%253Apmid%252F27112429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27112429&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarnkar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanojiya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Transcriptome and metabolite analysis reveal candidate genes of the cardiac glycoside biosynthetic pathway from <em>Calotropis procera</em>.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">34464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D34464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-article-title">Chromosome numbers in Indian medicinal plants.</span> <abbr class="cit-jnl-abbrev">Proc. Indiana Acad. Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">294</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIndiana%2BAcad.%2BSci.%26rft.volume%253D45%26rft.spage%253D294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabir</span>  <span class="cit-name-given-names">J. S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arasappan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calderon</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noutahi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The nuclear genome of <em>Rhazya stricta</em> and the evolution of alkaloid diversity in a medically relevant clade of Apocynaceae.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">33782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D33782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.40" data-doi="10.1073/pnas.81.24.8014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saghai-Maroof</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soliman</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allard</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Ribosomal DNA spacer-length polymorphisms in barley: mendelian inheritance, chromosomal location, and population dynamics.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">8014</span>–<span class="cit-lpage">8018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSaghai-Maroof%26rft.auinit1%253DM.%2BA.%26rft.volume%253D81%26rft.issue%253D24%26rft.spage%253D8014%26rft.epage%253D8018%26rft.atitle%253DRibosomal%2BDNA%2Bspacer-length%2Bpolymorphisms%2Bin%2Bbarley%253A%2Bmendelian%2Binheritance%252C%2Bchromosomal%2Blocation%252C%2Band%2Bpopulation%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.24.8014%26rft_id%253Dinfo%253Apmid%252F6096873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODEvMjQvODAxNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.41" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.42" data-doi="10.1186/1471-2105-7-62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schöffmann</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgenstern</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waack</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.spage%253D62%26rft.epage%253D62%26rft.atitle%253DGene%2Bprediction%2Bin%2Beukaryotes%2Bwith%2Ba%2Bgeneralized%2Bhidden%2BMarkov%2Bmodel%2Bthat%2Buses%2Bhints%2Bfrom%2Bexternal%2Bsources.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-62%26rft_id%253Dinfo%253Apmid%252F16469098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16469098&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.43" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.44" data-doi="10.1093/nar/gkr648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debarry</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr648%26rft_id%253Dinfo%253Apmid%252F21908400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21908400&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298733500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.385.45" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15728110&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228783000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/385.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/377" title="Whole Genome Sequencing of Hulunbuir Short-Tailed Sheep for Identifying Candidate Genes Related to the Short-Tail Phenotype" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/393" title="Reference Quality Genome Assemblies of Three Parastagonospora nodorum Isolates Differing in Virulence on Wheat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2uCJspCazwYODkoBGVP0eVNKr7xOq3ko7wBdN1LlR8Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/385" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/385" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/385" class="active" data-icon-position="" data-hide-link-title="0">Genome Assembly and Annotation of the Medicinal Plant Calotropis gigantea, a Producer of Anticancer and Antimalarial Cardenolides</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488299" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Wtd_S6tsfRcRSfVH9A69zNhNmPqotN21YtkAVhHoG70" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/385.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/385" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-SjUzWkb_m8CQgNcGmo1RoFkcLlWb3yPaEPJIOeAxPlA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/385" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/385.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488299" id="node488299--2" data-pisa="ggg;8/2/385" data-pisa-master="ggg;g3.117.300331" data-apath="/ggg/8/2/385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Assembly and Annotation of the Medicinal Plant <em>Calotropis gigantea</em>, a Producer of Anticancer and Antimalarial Cardenolides</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Genevieve M. Hoopes</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John P. Hamilton</span>, <span class="highwire-citation-author" data-delta="2">Jeongwoon Kim</span>, <span class="highwire-citation-author" data-delta="3">Dongyan Zhao</span>, <span class="highwire-citation-author" data-delta="4">Krystle Wiegert-Rininger</span>, <span class="highwire-citation-author" data-delta="5">Emily Crisovan</span> and <span class="highwire-citation-author" data-delta="6">C. Robin Buell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">385-391;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300331 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488299--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Genevieve M. Hoopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Genevieve%2BM.%2BHoopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoopes%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGenevieve%2BM.%2BHoopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John P. Hamilton</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeongwoon Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeongwoon%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeongwoon%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dongyan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongyan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongyan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Krystle Wiegert-Rininger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krystle%2BWiegert-Rininger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiegert-Rininger%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrystle%2BWiegert-Rininger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Emily Crisovan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BCrisovan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crisovan%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BCrisovan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/385" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/385&amp;title=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/385&amp;title=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/385&amp;title=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/385&amp;text=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/385&amp;title=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/385&amp;t=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/385&amp;title=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/385&amp;title=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488299/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488299" id="node488299--3" data-pisa="ggg;8/2/385" data-pisa-master="ggg;g3.117.300331" data-apath="/ggg/8/2/385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Assembly and Annotation of the Medicinal Plant <em>Calotropis gigantea</em>, a Producer of Anticancer and Antimalarial Cardenolides</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Genevieve M. Hoopes</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John P. Hamilton</span>, <span class="highwire-citation-author" data-delta="2">Jeongwoon Kim</span>, <span class="highwire-citation-author" data-delta="3">Dongyan Zhao</span>, <span class="highwire-citation-author" data-delta="4">Krystle Wiegert-Rininger</span>, <span class="highwire-citation-author" data-delta="5">Emily Crisovan</span> and <span class="highwire-citation-author" data-delta="6">C. Robin Buell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">385-391;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300331 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488299--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Genevieve M. Hoopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Genevieve%2BM.%2BHoopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoopes%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGenevieve%2BM.%2BHoopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John P. Hamilton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BP.%2BHamilton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamilton%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BP.%2BHamilton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8682-5526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John P. Hamilton</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeongwoon Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeongwoon%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeongwoon%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dongyan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongyan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongyan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Krystle Wiegert-Rininger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krystle%2BWiegert-Rininger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiegert-Rininger%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrystle%2BWiegert-Rininger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Emily Crisovan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BCrisovan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crisovan%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BCrisovan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:buell@msu.edu" class="" data-icon-position="" data-hide-link-title="0">buell@msu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=385&amp;rft.epage=391&amp;rft.atitle=Genome%20Assembly%20and%20Annotation%20of%20the%20Medicinal%20Plant%20Calotropis%20gigantea%2C%20a%20Producer%20of%20Anticancer%20and%20Antimalarial%20Cardenolides&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hoopes%2C+Genevieve+M.&amp;rft.au=Hamilton%2C+John+P.&amp;rft.au=Kim%2C+Jeongwoon&amp;rft.au=Zhao%2C+Dongyan&amp;rft.au=Wiegert-Rininger%2C+Krystle&amp;rft.au=Crisovan%2C+Emily&amp;rft.au=Buell%2C+C.+Robin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488299/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488299/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488299/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488299/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488299/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488299/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488299/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488299/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488299/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488299/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488299/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488299/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/385" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/385.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/385.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/385.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/385.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
