<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/669" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/669.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/669.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/669.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/669.atom" />
<meta name="HW.pisa" content="ggg;8/2/669" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Use of a Sibling Subtraction Method for Identifying Causal Mutations in Caenorhabditis elegans by Whole-Genome Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.117.300135" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Joseph et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Whole-genome sequencing (WGS) is an indispensable tool for identifying causal mutations obtained from genetic screens. To reduce the number of causal mutation candidates typically uncovered by WGS, Caenorhabditis elegans researchers have developed several strategies. One involves crossing N2-background mutants to the polymorphic Hawaiian (HA) strain, which can be used to simultaneously identify mutant strain variants and obtain high-density mapping information. This approach, however, is not well suited for uncovering mutations in complex genetic backgrounds, and HA polymorphisms can alter phenotypes. Other approaches make use of DNA variants present in the initial background or introduced by mutagenesis. This information is used to implicate genomic regions with high densities of DNA lesions that persist after backcrossing, but these methods can provide lower resolution than HA mapping. To identify suppressor mutations using WGS, we developed an approach termed the sibling subtraction method (SSM). This method works by eliminating variants present in both mutants and their nonmutant siblings, thus greatly reducing the number of candidates. We used this method with two members of the C. elegans NimA-related kinase family, [nekl-2][1] and [nekl-3][2] . Combining weak aphenotypic alleles of [nekl-2][1] and [nekl-3][2] leads to penetrant molting defects and larval arrest. We isolated ∼50 suppressors of [nekl-2][1] ; [nekl-3][2] synthetic lethality using F1 clonal screening methods and a peel-1 –based counterselection strategy. When applied to five of the suppressors, SSM led to only one to four suppressor candidates per strain. Thus SSM is a powerful approach for identifying causal mutations in any genetic background and provides an alternative to current methods.

 [1]: http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene" />
<meta name="DC.Contributor" content="Braveen B. Joseph" />
<meta name="DC.Contributor" content="Nicolas A. Blouin" />
<meta name="DC.Contributor" content="David S. Fay" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Use of a Sibling Subtraction Method for Identifying Causal Mutations in Caenorhabditis elegans by Whole-Genome Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Whole-genome sequencing (WGS) is an indispensable tool for identifying causal mutations obtained from genetic screens. To reduce the number of causal mutation candidates typically uncovered by WGS, &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; researchers have developed several strategies. One involves crossing N2-background mutants to the polymorphic Hawaiian (HA) strain, which can be used to simultaneously identify mutant strain variants and obtain high-density mapping information. This approach, however, is not well suited for uncovering mutations in complex genetic backgrounds, and HA polymorphisms can alter phenotypes. Other approaches make use of DNA variants present in the initial background or introduced by mutagenesis. This information is used to implicate genomic regions with high densities of DNA lesions that persist after backcrossing, but these methods can provide lower resolution than HA mapping. To identify suppressor mutations using WGS, we developed an approach termed the sibling subtraction method (SSM). This method works by eliminating variants present in both mutants and their nonmutant siblings, thus greatly reducing the number of candidates. We used this method with two members of the &lt;i&gt;C. elegans&lt;/i&gt; NimA-related kinase family, &lt;i&gt;nekl-2&lt;/i&gt; and &lt;i&gt;nekl-3&lt;/i&gt;. Combining weak aphenotypic alleles of &lt;i&gt;nekl-2&lt;/i&gt; and &lt;i&gt;nekl-3&lt;/i&gt; leads to penetrant molting defects and larval arrest. We isolated ∼50 suppressors of &lt;i&gt;nekl-2&lt;/i&gt;; &lt;i&gt;nekl-3&lt;/i&gt; synthetic lethality using F1 clonal screening methods and a &lt;i&gt;peel-1&lt;/i&gt;–based counterselection strategy. When applied to five of the suppressors, SSM led to only one to four suppressor candidates per strain. Thus SSM is a powerful approach for identifying causal mutations in any genetic background and provides an alternative to current methods.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/669" />
<meta name="citation_id" content="8/2/669" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/669" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/669.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/669.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/669.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300135" />
<meta name="citation_pmid" content="29237702" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="669" />
<meta name="citation_lastpage" content="678" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Braveen B. Joseph" />
<meta name="citation_author_institution" content="Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071" />
<meta name="citation_author" content="Nicolas A. Blouin" />
<meta name="citation_author_institution" content="Idea Networks for Biomedical Research Excellence (INBRE) Bioinformatics Core, University of Wyoming, Laramie, Wyoming 82071" />
<meta name="citation_author_institution" content="Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071" />
<meta name="citation_author" content="David S. Fay" />
<meta name="citation_author_institution" content="Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071" />
<meta name="citation_author_email" content="davidfay@uwyo.edu" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. Afgan;citation_author=D. Baker;citation_author=M. van den Beek;citation_author=D. Blankenberg;citation_author=D. Bouvier;citation_title=The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update.;citation_volume=44;citation_year=2016;citation_pmid=27137889;citation_doi=10.1093/nar/gkw343" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Arribere;citation_author=R. T. Bell;citation_author=B. X. Fu;citation_author=K. L. Artiles;citation_author=P. S. Hartman;citation_title=Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in Caenorhabditis elegans.;citation_volume=198;citation_year=2014;citation_pmid=25161212;citation_doi=10.1534/genetics.114.169730" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Bendesky;citation_author=J. Pitts;citation_author=M. V. Rockman;citation_author=W. C. Chen;citation_author=M. W. Tan;citation_title=Long-range regulatory polymorphisms affecting a GABA receptor constitute a quantitative trait locus (QTL) for social behavior in Caenorhabditis elegans.;citation_pages=e1003157;citation_volume=8;citation_year=2012;citation_pmid=23284308;citation_doi=10.1371/journal.pgen.1003157" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=Genome Sequence of the Nematode C. elegans: A Platform for Investigating Biology;citation_pages=2012-2018;citation_volume=282;citation_year=1998;citation_issue=5396;citation_pmid=9851916;citation_doi=10.1126/science.282.5396.2012" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=H. K. Cheesman;citation_author=R. L. Feinbaum;citation_author=J. Thekkiniath;citation_author=R. H. Dowen;citation_author=A. L. Conery;citation_title=Aberrant activation of p38 MAP kinase-dependent innate immune responses is toxic to Caenorhabditis elegans.;citation_pages=541-549;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=P. Cingolani;citation_author=V. M. Patel;citation_author=M. Coon;citation_author=T. Nguyen;citation_author=S. J. Land;citation_title=Using Drosophila melanogaster as a Model for Genotoxic Chemical Mutational Studies with a New Program, SnpSift.;citation_pages=35-35;citation_volume=3;citation_year=2012;citation_pmid=22435069;citation_doi=10.3389/fgene.2012.00035" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. Wang le;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. J. Dickinson;citation_author=B. Goldstein;citation_title=CRISPR-Based Methods for Caenorhabditis elegans Genome Engineering;citation_pages=885-901;citation_volume=202;citation_year=2016;citation_issue=3;citation_pmid=26953268;citation_doi=10.1534/genetics.115.182162" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Doitsidou;citation_author=R. J. Poole;citation_author=S. Sarin;citation_author=H. Bigelow;citation_author=O. Hobert;citation_title=C. elegans mutant identification with a one-step whole-genome-sequencing and SNP mapping strategy.;citation_pages=e15435-e15435;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21079745;citation_doi=10.1371/journal.pone.0015435" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Doitsidou;citation_author=S. Jarriault;citation_author=R. J. Poole;citation_title=Next-Generation Sequencing-Based Approaches for Mutation Mapping and Identification in Caenorhabditis elegans;citation_pages=451-474;citation_volume=204;citation_year=2016;citation_issue=2;citation_pmid=27729495;citation_doi=10.1534/genetics.115.186197" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Doroszuk;citation_author=L. B. Snoek;citation_author=E. Fradin;citation_author=J. Riksen;citation_author=J. Kammenga;citation_title=A genome-wide library of CB4856/N2 introgression lines of Caenorhabditis elegans.;citation_pages=e110-e110;citation_volume=37;citation_year=2009;citation_issue=16;citation_pmid=19542186" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=J. W. Drake;citation_author=R. H. Baltz;citation_title=The biochemistry of mutagenesis.;citation_pages=11-37;citation_volume=45;citation_year=1976;citation_pmid=786147;citation_doi=10.1146/annurev.bi.45.070176.000303" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Flibotte;citation_author=M. L. Edgley;citation_author=I. Chaudhry;citation_author=J. Taylor;citation_author=S. E. Neil;citation_title=Whole-genome profiling of mutagenesis in Caenorhabditis elegans.;citation_volume=185;citation_year=2010;citation_pmid=20439774;citation_doi=10.1534/genetics.110.116616" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=E. E. Glater;citation_author=M. V. Rockman;citation_author=C. I. Bargmann;citation_title=Multigenic natural variation underlies Caenorhabditis elegans olfactory preference for the bacterial pathogen Serratia marcescens.;citation_pages=265-276;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=J. W. Green;citation_author=L. B. Snoek;citation_author=J. E. Kammenga;citation_author=S. C. Harvey;citation_title=Genetic mapping of variation in dauer larvae development in growing populations of Caenorhabditis elegans.;citation_pages=306-313;citation_volume=111;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=L. W. Hillier;citation_author=G. T. Marth;citation_author=A. R. Quinlan;citation_author=D. Dooling;citation_author=G. Fewell;citation_title=Whole-genome sequencing and variant discovery in C. elegans.;citation_pages=183-188;citation_volume=5;citation_year=2008;citation_issue=2;citation_pmid=18204455;citation_doi=10.1038/nmeth.1179" />
<meta name="citation_reference" content="citation_journal_title=Methods;citation_author=P. J. Hu;citation_title=Whole genome sequencing and the transformation of C. elegans forward genetics.;citation_pages=437-440;citation_volume=68;citation_year=2014;citation_pmid=24874788;citation_doi=10.1016/j.ymeth.2014.05.008" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=G. V. James;citation_author=V. Patel;citation_author=K. J. Nordstrom;citation_author=J. R. Klasen;citation_author=P. A. Salome;citation_title=User guide for mapping-by-sequencing in Arabidopsis.;citation_pages=R61;citation_volume=14;citation_year=2013;citation_pmid=23773572;citation_doi=10.1186/gb-2013-14-6-r61" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=P. Kamkina;citation_author=L. B. Snoek;citation_author=J. Grossmann;citation_author=R. J. Volkers;citation_author=M. G. Sterken;citation_title=Natural genetic variation differentially affects the proteome and transcriptome in Caenorhabditis elegans.;citation_volume=15;citation_year=2016;citation_pmid=26944343;citation_doi=10.1074/mcp.M115.052548" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. Lazetic;citation_author=D. S. Fay;citation_title=Conserved Ankyrin repeat proteins and their NIMA kinase partners regulate extracellular matrix remodeling and intracellular trafficking in Caenorhabditis elegans.;citation_volume=205;citation_year=2017;citation_pmid=27799278;citation_doi=10.1534/genetics.116.194464" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Leshchiner;citation_author=K. Alexa;citation_author=P. Kelsey;citation_author=I. Adzhubei;citation_author=C. A. Austin-Tse;citation_title=Mutation mapping and identification by whole-genome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22555591;citation_doi=10.1101/gr.135541.111" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Minevich;citation_author=D. S. Park;citation_author=D. Blankenberg;citation_author=R. J. Poole;citation_author=O. Hobert;citation_title=CloudMap: a cloud-based pipeline for analysis of mutant genome sequences.;citation_volume=192;citation_year=2012;citation_pmid=23051646;citation_doi=10.1534/genetics.112.144204" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Pathol.;citation_author=E. M. Moresco;citation_author=X. Li;citation_author=B. Beutler;citation_title=Going forward with genetics: recent technological advances and forward genetics in mice.;citation_pages=1462-1473;citation_volume=182;citation_year=2013;citation_pmid=23608223;citation_doi=10.1016/j.ajpath.2013.02.002" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=N. Obholzer;citation_author=I. A. Swinburne;citation_author=E. Schwab;citation_author=A. V. Nechiporuk;citation_author=T. Nicolson;citation_title=Rapid positional cloning of zebrafish mutations by linkage and homozygosity mapping using whole-genome sequencing.;citation_volume=139;citation_year=2012;citation_pmid=23052906;citation_doi=10.1242/dev.083931" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Paix;citation_author=Y. Wang;citation_author=H. E. Smith;citation_author=C. Y. Lee;citation_author=D. Calidas;citation_title=Scalable and versatile genome editing using linear DNAs with microhomology to Cas9 Sites in Caenorhabditis elegans.;citation_volume=198;citation_year=2014;citation_pmid=25249454;citation_doi=10.1534/genetics.114.170423" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Paix;citation_author=A. Folkmann;citation_author=D. Rasoloson;citation_author=G. Seydoux;citation_title=High efficiency, homology-directed genome editing in Caenorhabditis elegans using CRISPR-Cas9 ribonucleoprotein complexes.;citation_volume=201;citation_year=2015;citation_pmid=26187122;citation_doi=10.1534/genetics.115.179382" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=D. A. Pollard;citation_author=M. V. Rockman;citation_title=Resistance to germline RNA interference in a Caenorhabditis elegans wild isolate exhibits complexity and nonadditivity.;citation_pages=941-947;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=K. C. Reddy;citation_author=E. C. Andersen;citation_author=L. Kruglyak;citation_author=D. H. Kim;citation_title=A Polymorphism in npr-1 Is a Behavioral Determinant of Pathogen Susceptibility in C. elegans;citation_pages=382-384;citation_volume=323;citation_year=2009;citation_issue=5912;citation_pmid=19150845;citation_doi=10.1126/science.1166527" />
<meta name="citation_reference" content="citation_journal_title=Experimental gerontology;citation_journal_abbrev=Exp Gerontol;citation_author=M. Rodriguez;citation_author=L. B. Snoek;citation_author=J. A. Riksen;citation_author=R. P. Bevers;citation_author=J. E. Kammenga;citation_title=Genetic variation for stress-response hormesis in C. elegans lifespan.;citation_pages=581-587;citation_volume=47;citation_year=2012;citation_issue=8;citation_pmid=22613270;citation_doi=10.1016/j.exger.2012.05.005" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Sarin;citation_author=S. Prabhu;citation_author=M. M. O’Meara;citation_author=I. Pe’er;citation_author=O. Hobert;citation_title=Caenorhabditis elegans mutant allele identification by whole-genome sequencing.;citation_pages=865-867;citation_volume=5;citation_year=2008;citation_issue=10;citation_pmid=18677319;citation_doi=10.1038/nmeth.1249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Sarin;citation_author=V. Bertrand;citation_author=H. Bigelow;citation_author=A. Boyanov;citation_author=M. Doitsidou;citation_title=Analysis of multiple ethyl methanesulfonate-mutagenized Caenorhabditis elegans strains by whole-genome sequencing.;citation_volume=185;citation_year=2010;citation_pmid=20439776;citation_doi=10.1534/genetics.110.116319" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. Schneeberger;citation_title=Using next-generation sequencing to isolate mutant genes from forward genetic screens.;citation_pages=662-676;citation_volume=15;citation_year=2014;citation_pmid=25139187;citation_doi=10.1038/nrg3745" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=K. Schneeberger;citation_author=D. Weigel;citation_title=Fast-forward genetics enabled by new sequencing technologies.;citation_pages=282-288;citation_volume=16;citation_year=2011;citation_issue=5;citation_pmid=21439889;citation_doi=10.1016/j.tplants.2011.02.006" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=H. S. Seidel;citation_author=M. Ailion;citation_author=J. Li;citation_author=A. van Oudenaarden;citation_author=M. V. Rockman;citation_title=A novel sperm-delivered toxin causes late-stage embryo lethality and transmission ratio distortion in C. elegans.;citation_pages=e1001115-e1001115;citation_volume=9;citation_year=2011;citation_issue=7;citation_pmid=21814493;citation_doi=10.1371/journal.pbio.1001115" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. D. Singh;citation_author=B. Roschitzki;citation_author=L. B. Snoek;citation_author=J. Grossmann;citation_author=X. Zheng;citation_title=Natural genetic variation influences protein abundances in C. elegans developmental signalling pathways.;citation_pages=e0149418;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=H. E. Smith;citation_author=A. S. Fabritius;citation_author=A. Jaramillo-Lambert;citation_author=A. Golden;citation_title=Mapping challenging mutations by whole-genome sequencing.;citation_pages=1297-1304;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=L. B. Snoek;citation_author=H. E. Orbidans;citation_author=J. J. Stastna;citation_author=A. Aartse;citation_author=M. Rodriguez;citation_title=Widespread genomic incompatibilities in Caenorhabditis elegans.;citation_pages=1813-1823;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=O. Thompson;citation_author=M. Edgley;citation_author=P. Strasbourger;citation_author=S. Flibotte;citation_author=B. Ewing;citation_title=The million mutation project: a new approach to genetics in Caenorhabditis elegans.;citation_volume=23;citation_year=2013;citation_pmid=23800452;citation_doi=10.1101/gr.157651.113" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=I. A. Vergara;citation_author=M. Tarailo-Graovac;citation_author=C. Frech;citation_author=J. Wang;citation_author=Z. Qin;citation_title=Genome-wide variations in a natural isolate of the nematode Caenorhabditis elegans.;citation_pages=255;citation_volume=15;citation_year=2014;citation_pmid=24694239;citation_doi=10.1186/1471-2164-15-255" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. R. Wicks;citation_author=R. T. Yeh;citation_author=W. R. Gish;citation_author=R. H. Waterston;citation_author=R. H. Plasterk;citation_title=Rapid gene mapping in Caenorhabditis elegans using a high density polymorphism map.;citation_pages=160-164;citation_volume=28;citation_year=2001;citation_issue=2;citation_pmid=11381264;citation_doi=10.1038/88878" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=J. Yochem;citation_author=V. Lazetic;citation_author=L. Bell;citation_author=L. Chen;citation_author=D. Fay;citation_title=C. elegans NIMA-related kinases NEKL-2 and NEKL-3 are required for the completion of molting.;citation_pages=255-266;citation_volume=398;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Worm;citation_author=S. Zuryn;citation_author=S. Jarriault;citation_title=Deep sequencing strategies for mapping and identifying mutations from genetic screens.;citation_pages=e25081;citation_volume=2;citation_year=2013;citation_pmid=24778934;citation_doi=10.4161/worm.25081" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zuryn;citation_author=S. Le Gras;citation_author=K. Jamet;citation_author=S. Jarriault;citation_title=A strategy for direct mapping and identification of mutations by whole-genome sequencing.;citation_volume=186;citation_year=2010;citation_pmid=20610404;citation_doi=10.1534/genetics.110.119230" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Use of a Sibling Subtraction Method for Identifying Causal Mutations in Caenorhabditis elegans by Whole-Genome Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Whole-genome sequencing (WGS) is an indispensable tool for identifying causal mutations obtained from genetic screens. To reduce the number of causal mutation candidates typically uncovered by WGS, Caenorhabditis elegans researchers have developed several strategies. One involves crossing N2-background mutants to the polymorphic Hawaiian (HA) strain, which can be used to simultaneously identify mutant strain variants and obtain high-density mapping information. This approach, however, is not well suited for uncovering mutations in complex genetic backgrounds, and HA polymorphisms can alter phenotypes. Other approaches make use of DNA variants present in the initial background or introduced by mutagenesis. This information is used to implicate genomic regions with high densities of DNA lesions that persist after backcrossing, but these methods can provide lower resolution than HA mapping. To identify suppressor mutations using WGS, we developed an approach termed the sibling subtraction method (SSM). This method works by eliminating variants present in both mutants and their nonmutant siblings, thus greatly reducing the number of candidates. We used this method with two members of the C. elegans NimA-related kinase family, [nekl-2][1] and [nekl-3][2] . Combining weak aphenotypic alleles of [nekl-2][1] and [nekl-3][2] leads to penetrant molting defects and larval arrest. We isolated ∼50 suppressors of [nekl-2][1] ; [nekl-3][2] synthetic lethality using F1 clonal screening methods and a peel-1 –based counterselection strategy. When applied to five of the suppressors, SSM led to only one to four suppressor candidates per strain. Thus SSM is a powerful approach for identifying causal mutations in any genetic background and provides an alternative to current methods.

 [1]: http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene" />
<meta name="og-title" property="og:title" content="Use of a Sibling Subtraction Method for Identifying Causal Mutations in Caenorhabditis elegans by Whole-Genome Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/669" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Whole-genome sequencing (WGS) is an indispensable tool for identifying causal mutations obtained from genetic screens. To reduce the number of causal mutation candidates typically uncovered by WGS, Caenorhabditis elegans researchers have developed several strategies. One involves crossing N2-background mutants to the polymorphic Hawaiian (HA) strain, which can be used to simultaneously identify mutant strain variants and obtain high-density mapping information. This approach, however, is not well suited for uncovering mutations in complex genetic backgrounds, and HA polymorphisms can alter phenotypes. Other approaches make use of DNA variants present in the initial background or introduced by mutagenesis. This information is used to implicate genomic regions with high densities of DNA lesions that persist after backcrossing, but these methods can provide lower resolution than HA mapping. To identify suppressor mutations using WGS, we developed an approach termed the sibling subtraction method (SSM). This method works by eliminating variants present in both mutants and their nonmutant siblings, thus greatly reducing the number of candidates. We used this method with two members of the C. elegans NimA-related kinase family, [nekl-2][1] and [nekl-3][2] . Combining weak aphenotypic alleles of [nekl-2][1] and [nekl-3][2] leads to penetrant molting defects and larval arrest. We isolated ∼50 suppressors of [nekl-2][1] ; [nekl-3][2] synthetic lethality using F1 clonal screening methods and a peel-1 –based counterselection strategy. When applied to five of the suppressors, SSM led to only one to four suppressor candidates per strain. Thus SSM is a powerful approach for identifying causal mutations in any genetic background and provides an alternative to current methods.

 [1]: http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Use of a Sibling Subtraction Method for Identifying Causal Mutations in Caenorhabditis elegans by Whole-Genome Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nXFqion5ptWJor0PIkzdkRp04PI6P_UmsOHIZdLljz4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488292","apath":"\/ggg\/8\/2\/669.atom","pisa":"ggg;8\/2\/669","ac":{"\/ggg\/8\/2\/669.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/669.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/669"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/669"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/669":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022w6bh0kinIgPbagIdgW9Rk34qlM5Wr34tjgzv4kN23WFBypACdZvmQ3ORcz0fxpyG1XAzWZpqWX1asJs8E9tAZ\\\\\\\/sqjAm1TObbIiWfzSA7xKPnhmbe2F13ZF2yIP7qGlDEulOtC9TzOm4YixXe816\\\\\\\/Srk4j0rrUS3ZXmov4ZeQnwcXZ0SHm9CDTxQ7H9O5\\\\\\\/Kw7fGOoIFN9abJ9bY+U7rkDVriTxNDkZA\\\\\\\/8VLAub8hTMlZUKV+ujxAEYhOfUn7cn6ERSNSer8ckhk6sG2g4Ml1CUNKetZwGHPS6mFH5eSTM3KNQ7K+QKnm8S0nezts0Vr2S2RENniAQir5yf+VH03rXSbNtQYCh7INJe6JiFDz2vKMPLpcEEuS8TnEHALLk8wCi1mFDHltnK+cmK2Gpcq5KIML\\\\\\\/O0Ni9+I4C3AcmQyWX8YeVTRcx21aEAauZ6M8Y84wQjtI9crx+Rg5AsuHVcZ0C01NKcMxkPmDSaqC\\\\\\\/c61hl0hy+6jIkqCO+Kg9kgbKHa5sWMYJJYldHTiEu+1IVXVj+M3e4xGAf2emajq977moGse\\\\\\\/zCK4J7aMp\\\\\\\/GySgucqkKzHfoLB7TxSfIkYKREhf\\\\\\\/fehDUWIqG6QVz27yys35UJ5LOwwl0rPLV9dKO7t6Qtl3RSnljaNn3y0B3bJg2N7dVN8+nE\\\\\\\/twzJAzK4j+AneQQnhqiUvmNGpgh91cWo9RMTqGvq7VFkJ4\\\\\\\/8wFjOtl5Wq1yuljAsUZerzTMkFrieyKaBdzfjC2KEUiJERwpCmfRf3sSMOle2dOWym+nE08vgONOqWJZ4QrVREfhvNOuJXUcJyxtWAQCgbJoh+t2PMD40j5LmJPEcUMPva58PgXVh8ShJrbHMaOTm0841iyKNoGZwAEP7FiApyrZAdW3I4ujlCqWV3jqNu\\\\\\\/ygrx05ttta7MDRiFSFcYTDbCiCfWPc9zLwamlgzbds\\\\\\\/nbVjrB8qMBuAkKMxykHBEfmUU8SeFtYOkUUrwyxQI\\\\\\\/2SBG+6NZz6a55\\\\\\\/p9mwq71a6sKHwPR4q2v5Lz0UHdV2TNdrH+J0l4AWDPH4fPE5JroKCu74NsZq2Mye4F6ikMa5ZNuQsq2KUalyV2qvGq8XTJXg9GLEHoW7\\\\\\\/ZdqlWEoOMQEAMRaW4r2iFHp+w4pk0J7+MEglZJXqCF92Hmi8gyNhooyfeYzUTvkg3wHvpJQ\\\\\\\/1Rl4rAUOjH4I71+\\\\\\\/0xheMw1EmgnelfljXj2bl1IoGwvHFmNqP+xfBnwBCR5u5etOjp7QBOod0J09iqJyDlhbHG6UrNblUcIJqMF\\\u0022,\\\u0022iv\\\u0022:\\\u0022uvBuSTxmGrD2TyTCdDldIQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002280d4f1ea582a2db7e91b5c10a680227da7fafe371b8dbb074aa26c15b27f1382\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z6EcWVZa4xgl6bhPp47xb2mBBGGx7nFI72IodPgtuq0stNlrMGqNs7ASf4z\\\\\\\/9DYWC112Yb0gDEelQOX+ws26faXQiPSYI706XorrxJgmi0o3fd1MMN8NMQmD1+LTK02C0N4cTy1fl4BrrG+R8gYWeXVMCvnrkHFNGEKuskFCS9JAX6WSz5K\\\\\\\/ylEwWsR\\\\\\\/yniEHK+dR1\\\\\\\/U7XTB+RnB\\\\\\\/veQEoI7HmAOb17IQbG0Gxs7acq2YAeUCfUO0+9tfeQeWBjzhtMd0yJk68ensDqjXrgIfr3TOxICQutOnM0CECRfd2W+xxZ0Xck9ngddMeVwDpeRS4Q\\\\\\\/in9LTeuQxuElVkEQTn+l5N67Bv+ItsBqTjs6iRNOn2ICYVUbMGChlPolMAeYS9uwkbdIRIbvIFe0f\\\\\\\/qKEEmj9ZwJM5WwHooL2V77l25PsiDGOTG3XYGFnCwCIFA\\\\\\\/D89RdaVoin\\\\\\\/mtqgWkL7ecVk1zZqBvqqX2Di0bthXXmTbIBG5i3HDS9vJ9uLpAsKAxhrLRwmeptRoqpfaTdfVg2StKu9TUNExyUTZtv9YAxXNC3gWktBEvLUbTvCKf+6P3bV+KwfMdYayXsemXytyIqmHXAErp713OeSH0nBWiSYoV5fvYOeoyU2Y0kwTbH5rXHJgUqfFzB+yYjYtPydK6UMmO\\\\\\\/rAzhO4y5OKfEOAP2LE1iLTqDeKzzwGcGXtbuzUmu8rCLgcpfy6KID5l1XqDvlBIdan3b45hglUmSaI0Gvo+jDRMOc6tojXV6mmmnCYebsx5cUITueYQYpGR77rQm6epvxnsb0TKqr127i3ykmjjrDH48dkl9eONuGlOcIp2mtDIm4Mf0UhFqL2hnFnQbYI0BDbTBrwoeSKu3vFJgNLTQ5OXmxF0Y45bZw1CYZDzxPn5nQzUVCgRrZgrTUnvVtq8liPcZ+\\\\\\\/pGq44SpM5MZgitx6YHPArgdrIvY1n6CIP07uToRuma2S6yiaDtR18uNn2GQ5+0Smx3ijVQLmAIceQ43r\\\\\\\/cocHjG8RWa20kutJwDghdSD9U0rdUm131TKandRszt3a4TUfl+\\\\\\\/szwD0cp6+CUhQcRoavQpLMuFQie9K\\\\\\\/eWv1Qx1ET7dvIqWN6IjkN2p8zaAb\\\\\\\/hbnbYcPm\\\\\\\/GVKf+sneWFguZ7LmylqKDv5VhgF+GfGYAG3\\\\\\\/jxfnyeVvB+y3doQdGUZPGKPeZbzUhqPupy4pKfoL\\\\\\\/zmB3xGanYSWoHREqnkBYf\\\\\\\/JVupxZu64ygC78F9Mn7Jrw4azIwahFcGxdlOZK52wt6pYmxvHVmRoFp2DOTbwgsGUXIwlv144ZGhq8i8ZAgGZ1jHTLxUAPpN3GvViGelPsWY\\\\\\\/KQ\\\\\\\/C6KCZJoDj0z5dsdoT2jLk0rIb8xU7r4Sgr2oNteOI3ZFRw1P46+4gw1Sc4PXeceeRhoR\\\\\\\/tLAbB0rhFjnHluGOrqNuJC0cEmugy\\\\\\\/zyaWdEP8SB3rwlxoCCruU9iAxa6H90hGLEmX6WQjFhkqpLnr\\\\\\\/Nk\\\\\\\/JXf6\\\\\\\/uyMSsz+RMl\\\\\\\/7gQE17yJmRTv4Hq7CgKBLmOaUh8Ex+gP5KwMSYJUkU7Yxj8EnNvioAlu4b+CNAG1dQXTAwEgwoGxAsFhXORhNJD+FfXT88SNEj+UOHwHxCXuMu36vjXJ03ZnhvPQO6Pv7hecVTLUTVE9J8cuJAlexFIlcDaWCfBObh8c7squm3wPUlVn9dCtuyFi5Ki2tc+sadtOERq37SHmIsOhDHz0MBiag8xCgfYHbAIcdnGWJ7H5EkCcD9i35waZfngp+5kZ4Af++IKZsKtSiuQYobFYwx0FtzYmAMrAj8JFvgsOSc11qxtGJzOGEEIqiDkb2DRO4YJoJx9owijybz9LD9cPjKqDPkJRouo2wmzq2DNVJSF7JIkXmGvzCpwm0A1W4d72SUHBU26+SbUPQRiWVnOJffWlZpSSk0xWi4PF6uFKMYYvjHQaE\\\\\\\/Nh0Gx+eyi\\\\\\\/sqVTvSA6zVOi2yGdEqhcXSn0Um4VAk5bkoGA6rPmzTMH9K3WugDYfutqtjD0gaEaIW0z\\\\\\\/PCLnHCx0Ghc19UEtyS8y6G5kP15hl77EUDeb0\\\\\\\/\\\\\\\/CyPIdDSaVlJY5MW7W3sORkXolwh+jOr7qfInjmy7oeCCjY1QFcArykKJJPguoLrLJncU9rWzWrur4SRjcrZOgHYO\\\\\\\/u6L8g5Cpowq\\\\\\\/H5aoDW2aZ8a8Q5qWCwiqkUUmyKAw\\\\\\\/y0NPWxg6XcQ3cyHj161zqKgwCgJIm7aU8giiP9WIKZ\\\\\\\/\\\\\\\/4Yy\\\\\\\/FCrGeoVMRJm6AVTVbRggB+Vvx\\\\\\\/bSEa+LmRagW9pGXiGBSr3FgWGBKvzcOcBwvr3yhts6rhLsUvrcTwREotjk8hBvaF9MSZDQZZJTmdMG6FNT9qF+X0Y5tzu\\\\\\\/DAwN+Psf49+\\\\\\\/AEGnu\\\\\\\/N20+az9XcbEhaPRielNrrEROC85yaLU4unDo4\\\\\\\/uUmfRMvfq+Qot9E49rMYwJnEvLqfEq1Cpwx6DSObKV\\\\\\\/0a9C+CjOZU4pOD\\\\\\\/yMpxSdZS7MyoAcEsXI\\\\\\\/\\\\\\\/66DRVhXtRKpSsnH7r61BJ6aFHaQZYfBZvJHQjhtas0S3SoDtfq0RMBQwqhiEwRxfFdsW9GAUujohvZTfIC1yMKEijsYwfj\\\\\\\/YuPg2Wh0o7xUJQdtezDALnZqTB+fKAbU1XObApRsxpb+CUnKnSsm9\\\\\\\/PTWx7dSKLljBWQiIpU1eW6bSfIahbtVahqfxwJaeei6lI89Vu47HFb9UtC\\\\\\\/taRZFCzgjk4W8\\\\\\\/K9Vas7JpHDkGzvxrnEa9YHtEukPDY7c1gA5N\\\\\\\/BKmB6RYaMmvULOow+isX1+ey\\\\\\\/Ks4u6dxdLiKRVaEVl\\\\\\\/nksMfmLx9MEYDXfqT8tmqQPWR3b6akBCktYlkkDwPbIMoEMlYJqbcwAzXxhoQ1HWkBGYf9HODIJGvs25ojiEQeZNN7\\\\\\\/4ckAQm80yYhZ\\\\\\\/TUF6jMwFJZLZ7VLctWPnK+a8aoNaWbBCohBgomHEyjVUpDa9PmPKdBN9VRWRCCOCxCfSfaPRK39G1YmitR++pzYCA6c44vL2E9tGqqVoCff\\\\\\\/XOCHV3JLR0DCYhn\\\\\\\/MnaGsuWtHHoeKLcdRcWBmMYRXNXLaWpu3EXYg8uU\\\\\\\/mJseeYzougvIhtkgwjaHDjNergBC5WkoeFkbr+TjteDUP5qPEWBEk4c6M3Rkr\\\\\\\/iG4Qs+t1h8iOkEx1ekMsieh3Q+6vlkQCs\\\\\\\/48DG3mNtTi6L9MpmqyUCVP+KXQC1oZvcspyI5ZyUczNsKYzIqxTzhRbW46xyGjVJQUYIEVMEUBmCmDiL8Rn22Y1Y9rNSZKH1BUyo4BqBEs8JuJzLzL6gQFQiAoRCXEjCZZ9fb186XBvQ2imm1nF1XJ56MAGx\\\\\\\/gxeg8\\\\\\\/WLJy\\\\\\\/STmZJVgVA9GcSSTshsrebhWfG0kWAbDY98L2qUcYD2YGcCCEiNKNsITXbwunOqclHb8y3MrO7NpLApSj9vKwpY7ILjT750nKmWe5epqj3hUCjTTdQK1ZbDFV7iIrYjSJOkd2cfPJuyIgBZkiWbD9rbAaIc3RhDhziuQqCjlcWlZf3EW8r3rOfJJiiu0au7eh8BJsyLVJvIqx3HzHuM7PHJ8F121oDMRELPAeLmDvuQajj5mKLhzPdu4aJD03rx26nlW1uZIEEd9Ikt0YUbr+ndGbpb4ZiWEdh3M+sHbax+ejXlqfMoQ3vYYJCquPPIQT64yru4Gt7Su9lmpSrNHC5C5Bx0m5EBMlCqEm2wBE+uy\\\\\\\/iMKV7go5baLWvYGzESm3XYHiIgT1zMZpWCmj2tFep65Etm\\\\\\\/IRkcsawxsdvvLwJIgWrbAfi6563VCGZbm2wbCEhon9HFlmZGBRky\\\\\\\/aC8m4\\\\\\\/+JhUZn95N7liYNWi6eSW5tk8nyaCpKvk6fgzhaStlz8xEbranGDZHtn79jkNyvnH1f3WbtDcs56hcoLAnNPXbbqcFvg8ZEPNZR52SaG908rDanFJg8ENZlTe6fe1fPU\\\\\\\/BmpBBnq4Tnbf9CHAjiTUrQtn3lx3omASglObxQYuBxgmgxfM4fxvVi2MnhWWAX\\\\\\\/E84ALWeXD84Q4OuzJ7WvSDIUnD\\\\\\\/TGVlyqW48+KTSsOh3o0oTOjqrTXyLzE7res6zkwK+sO3bb04uA\\\\\\\/bfI2kVElYo7G1y1AFJmBD6fF6LA7PYgpb\\\\\\\/tGUjq2mFU9IonmMlVkNmViSrDIgQ4lfn6mZpuzNNjL1fdHfMDiC6Piv5qTJl+PtFfwkpL4nv68acUWh6p+Alqb9AFMsuxlx1pPs6\\\\\\\/EZDL4onoK6E7xPxTMjuajPgELlmNSWkhJ0RTxigxK1FGYVo1Zvs3vBSiFJpsPcq+O71aYAtxpZXra61fq\\\\\\\/6tw3Xs2aHWUWsK04SMuMe3Mf3wxtJigpqJrgWQzVMMsN+ANAYN9eELCj\\\\\\\/3e0p\\\\\\\/y0xb6y\\\\\\\/38BI7mj0MFRVLduUfWxc0UPP1ihXHwBHWMUL94x4hkNCiQeL13fEtik1wwnwPHOZOh7TVL1UfkK1my4Yu1vVGZCZQklA\\\\\\\/qro1tRX67OMb1Cl+DAnfFgxbVStpJ83Yz8cxrrn8EhxdrQxjtnv5FwfrRpKnBRwGs80Eox2YxWNfrqoLUPSBE4esk\\\\\\\/25SCaIGs4fVRBj9qn+06nmxNpYNvyBUpNDihi+r6ZN9Ar0NH+37zKGtotsVFyCYO1ADsEBOd54tF5mvI1AueeIRoH+3m7hqorMAc\\\\\\\/DlW6Mz5E5r5w+hb+I49sepjgD0T5vvVXxG1od4pR37NEIo9sONaFTN2OADrGKaWlMkbjv7JWKu701mfu5N+FiNRx\\\\\\\/5K5tc\\\\\\\/r6oiRMPorJ01iLiWjFFVrt4sVzgFy\\\\\\\/ezoBMae5oZhobEV6NlHcyNqt7T\\\\\\\/IGbaNKPSGjHaYdx3MV4+GjXXgx5jONrrExwmmO2eZgl5MMsL26fnzwJYdWmJSOQ6MQ2t81BccDP5WR7FYqZHidaitBEjyvjK6dvy77V5XYBjjHybxQD+OFLITo2eX9mmFPE9I6HDLVCs53Nd+98Wmu0EcEE6eYkb3zlZ6x3xjGmyuDcZIipPXcAZhO4OBYNQilT6H2f8NlfsVTcXJNHmXenlKclCR\\\\\\\/Q+G6kP6g7\\\\\\\/O851JTJc8ypb0MthxDQPqAFXGXngfGeirU+uqmFXLvQsjmqn6ElEuHLmx3Dka6Bcl9QuvJwrwGb9\\\\\\\/8YpjRrgy4WwSBH7DuFLV1nufC7N8tk5C0raXooWx\\\\\\\/TpkmErj6gmtzzi7yJvBAIfnq\\\\\\\/PfjbR2gwgBl6DG0S+UsUSE+axE+qAPD1v07riJ+gE0RuAFn1k84q5XlcQyO8SLb62QrEx+3j5X2M+BFtc3QX+NR7OhcWFU2KOXhTtoK16Mrh+u0CanFPI7W24pfleRyoQJupBAAbFyvJmaqk9LIwqNYqaA7ce4kSxPpWSiFTxBw9Fxl2nj5Sih7VOxfJgMeLvMI4P8q9bjeI1exsrd8Uu5LVrNozRVXlKeLmfaF3TVL5qhRA3mRb4b\\\\\\\/K5HpJG\\\\\\\/4q9WC7OgNI2nsK5i9rwpKyC30lAnjuWguYR8rtWgropSrDQLQynbyZs7GZCXrOwyHyhDymcoIXdYmoTyqeTVz4XphxdoG\\\\\\\/8b0e02ZMNKmwz8iIt9cjBpRF59\\\\\\\/KcB++tcnz1PcgaI1lpcyUFCtNrBWHUxmKXztxLBpugKG9GSCa0oG6ootuB\\\\\\\/egWApcE0iC4OnMjOVbxlfGtbhz5r0cilfrCLSLmtBHtYIvPIRGt95LQASTuQud8t\\\\\\\/N747GiO3HLiFceY8r+ItoD7VSyOJBbIn42QTDLWjvFCV7OJ1Sa2lEbO3wskE72ieHQnV9hU5NVEwf4W\\\\\\\/vcMW8ubnSdgdvibNACop5gvnnjQ4+MRRTyJ5qRfjTSWRN4efNVuL7enw3mkIJUPAOUSTxHCka2kI6FHV4sE0iPDbg+\\\\\\\/gPgriMxpeJz7HbPtLRUVW+KXQubtAHseySXpx8AlmEw33fV5njDNissjYOsD7\\\\\\\/mLrZCphKJ8e9jUUd3NO5WuCAFZc9ir5YRQuVfdDBQgzKwy5ppeQ32Ixqq6QV4XJfvqYC4FF6ARuJhXbhpncB+GTtfiD6auMJiMu3cdvKoz6coDUgIOxzkoSjJ\\\\\\\/ipNMKKS+X5QqEnQoWfSAH0BFZe7fg\\\\\\\/vUpXbkfs+yjkLAoDMZNttsF+qEbq3IGXoX5J5WkeQsmFHYFIqR3NEq9vZjdifVo1vGslJ3r3hfqHMhPDRfCo8GqDWKUhdQqjcg+pGXpBotsarp+YHbDUG4A3xEMaxRDEkYtYE\\\\\\\/m+9mqPIJ7gMQle9gHtJ5gHaZqtOmzGwmN6SmrJzs3C7sk\\\\\\\/B1GDiVirrQLxC5dscYF+amdHRNJgIHB7nXK5PHPG6Li7URNF+AoGPj1J3JaGA1M7glWCfJoth8gk11i96\\\\\\\/Y+7EEShdwPsL\\\\\\\/8\\\\\\\/JTShqD\\\\\\\/T2OgK5VjrBQFDRMXA\\\\\\\/cUVUUi21zlXGc2ESpdYiiFrXMdYe74qmmHuvwu8P1XVzPSKbff+N5gCaL6AGEKNYOaIeytqyFl5gnq6pCOtwq1M2pr7KgMyQudZS8azFUw6F9fmdi1gpJj9LJR5KN05Tf+PxDywWrBYS1i9oqGqXfa6\\\\\\\/TWP5Dn331IXapKOA1w86RXyul5L3HccPsqvoEHWXLoyJweXQXKUk3Ok+CwfknrET5SV+DqYc\\\\\\\/KtuqhasUHKUP8E5\\\\\\\/GtRk8nbXx+VLQCisyt+Usw3\\\\\\\/W9C5lv0IHnpQMVly+jfZc13ori6cpbXUbWWXEL7AXXPqC0eGENlVV6lLg2TZIeDAEsxHAyXHlgc7Egq1tx6WytpIJs6hi+DDcDoUaFG0krsdmqPmVRmBBAGZJtFO9aRrO+EZHokxdrIgKrZlBsJ\\\\\\\/ykDIePtGkfWdHMjIhD5kXlPF4AisJmTKsxkgbN1sihJxUbdHV3S112SCUXmHivuUTWP0ric5FVnrFqwSOuOulOhedT8nnLz3ltIC12RcSErlwtOdARICPBMXMIXmBey0X5MnqQJVtzQ9tWsJgLswsir9XXlSV50ta3VupE8Py0Z7ov7HPTEGotWxCaDTaxRziGS2NWX0fcwCef\\\\\\\/yMOfLR33PAexOqRGXWkcrMUCDURQK0wSSuRZ7SufOoYCG3pdV5qEqJcnSr95qHQVO3XcYWzL5KO\\\\\\\/1dpg\\\\\\\/VJvNgv5JxPk7pDLFKoT4R9c0KHRd3burYc4TzfcvBzUnZSmaClZxLsBS\\\\\\\/+K7F3E2YyrXSr53jl9k61owyeNvzWA5za3VEwYPCH2PhRpjV3sGKlIZXvOGbEhussH2RuaijMLEYHuoi6cmABxldpViI\\\\\\\/YqEPUG8addwS7nAubkpAVvZ+DUFdSgki9ksUWxEs9tEG8VUiSms5JjNHQhCWY6OX2wGhqzco1\\\\\\\/MXyPa9VHBU\\\\\\\/juiQYZBF1Oh38Bz6qtwiLkbrdPYxydk4bidmWANrvWJvK5wtMk8YQ0uIMUTu3mNbp3j8HZli2l03lP73CQMbGs5AnC5CoUVxkAOC5a9nnqmB6JdcV2dReGkP9iTt0mClkxiat00UralBFKipYHJwsNExoEMFT2d0q2N5ZRxG4gD1jY6X8B6su\\\\\\\/G+JGWQdDo8KcrD\\\\\\\/wyq\\\\\\\/ZilDfZKVs7\\\\\\\/YNxhko20OLJ35OwipMo4OMaRKFuUpoexfKs+ZnPpQX2VHlXilc9YcJghEYEazzvfryq\\\\\\\/cQrzX10ayqyTdXYG8Owa\\\\\\\/5YBew3vlmCltUL9Ic\\\\\\\/s8C81lH0SeHrrqI0\\\\\\\/GV9yd7k+7Nz5z1cuaYwaaGrULTb5+z4605FQrKNpBRYDAdNAk1fgRBvUlYLdyMoy4SDGJDJQNQrSRgIB8ONRK4y7F9RN24alanRwXpKna9dHYlhnUzaNsfvJYQfplnR42VisGOQunWkzb4uwPaKbOXS5McxUYa7nDv+k2\\\\\\\/7EWtla6q5G\\\\\\\/oSLpHKC63EmkDvSopvPyGAuWn6uh\\\\\\\/+u25w+c5y\\\\\\\/piBD6FMQi1xibMDdJ26tPnQt0rZqnzr36Tlz3v8eEGzUatFrW7rkZ5KQsCzrVlQPR1g4OIDHReqD6L8FsnABu393i0Y86SALhSchTLtmwpPwUTnm3UoUuQl2NnV7aOmyR0jC8UARMd89C401Rz0Ht6hcRQ7633BtYWHHHHFw3EarUxFno+jPf9wPMnEugBLOBjt8kFhJ4sR0ukh2Aiq7ErIxapykqjxwdFvPvg7nyd2i5KUb9+YQ0K4MmGl29BwdxTTysLOvtyVm4r117UIFZY\\\\\\\/M8BhFnUgfcWAnfAjWq\\\\\\\/p+Ko1QGCRhelhc7RUsFYujG\\\\\\\/jqhzpuNdKF23ZEFvN9+QVs0gxPlKYhnc\\\\\\\/TusI6XGBfQxC5kWF7y0lHEsjmL35xEzyizciFN1lGv+S7giZAyQuLVquNmS57ZrlnPfMpSnvNJUzqtc5Q5lDiWqs1NmGQ4k8y31tYjRzWcQQEk19LbXwM\\\\\\\/AxzQvzXSag0RL8\\\\\\\/ENYaBlYZsF+4MMqmcrBsKJ4iaH0Dr2aaO+4XXChw79lNnxOZQynOWuQ87tlJbayHQFQsNWNx2cep4cZ5QK3c\\\\\\\/EWGCYRnif2sDqar8l5eF1UPDL12umiS7pUbA9P+1yI2oy5d+FYdFWUPrYUNhtuVHp3q4vkAbjdMEv1pa2nwo4A3JWr8f\\\\\\\/kgT7P8DwD40Zb34slsxS5FuUfvTNXAhRqBktgbUD84hUdss5bD2lsB4dRclRHJfOrvL+RHknVk+IO1YYMbEjx4gJRvQ+Uq9gumSRAom1zz1mIltFLUSDE4wYMJTyYVgxpptijeS03Yy0huuZhaEbQTg\\\\\\\/e7AwcvntLo59A9uUp9LQLYuOBqyPOzhbZ91Q\\\\\\\/Uo8CzKEPcqIFekogKu5tQVU8AJchrcRIT\\\\\\\/BiMJuVA9yh9Qgb6OFbPEy3itoRh+ie0ORea5EnQGep+S94\\\\\\\/IL6TAKIBjCpkjAg2l7SXwlG0PH5T3MdASnqmqY37e3ZGHjgGK0aqO2ojoQ6XcT\\\\\\\/UAIeXp3TCxk1NUF1ZNKiKonQucb\\\\\\\/UiFBSY80D6CtqpMO0BdFxtB18nv7Wp7koWDNo\\\\\\\/KTNs\\\\\\\/vO04CZdZ38CKdRNs2YnCsT5Z5fdtdNvQHX5Fsh\\\\\\\/gvM6SI7DkQIdSMyy5zkyDlI29Fc7bBh46+QF19T+\\\\\\\/4GCDaz7NVqihndud5azVALTOjKSoOsNc3MZ3gNPYSMuMIliHM7ZwRogQgj51hIzLZdG3iJJ2SiiaxN3gfJLrFck72M4ulIoTZeDvfuZoV5K7RNvf7NJZrk2QvcQBeHBctoMO\\\\\\\/breLSRZQ9BPfHqL5Z9L5M1lWbZkpn2BB47vzVKnc5gwKoHaC2lW6BgE\\\\\\\/C98kftZSl+f5i1LDFCE9vcJ7c4cEHDIPB4cBnAFjxF32mWxsZSQQPjP74OhabDWTTh6YeFFtj2i6jrklE\\\\\\\/yUfp+p14pcdvbMUcZT1Ns4QcOj5S87dCoeFLSRweaKjePs373mLiFsJAL5XOQTzKa8xthjtq7K+7xWgDTQDi\\\\\\\/LdJ\\\\\\\/3zajUaEx\\\\\\\/P1Bszy33iTxkEvniJWl5TiFo4e0MCudpmqdM\\\\\\\/iPq+ZTawE5UlQ5xP6O7MXf55co3MUSmldeKsjiiLh7Z93hkhIwNGwSu0HgyIeirR5imarL1gnvGe0eVF3db7amxeQRg6vbEuIoc7rR5YmZcA36uDmFbsh8zlHZzr8fRboeTiSECmepmAyUtOllg\\\\\\\/+uecZ8sXvLHAKYdgzv+TMC4p7fko1RREmg1fs+FSj6+b3SMEkwfc4rHa+9riSJ8FHYUYbslcNSVSg5Qefc0KiHuC817ze41TFbNVZlyPI3koZjNOpuVEaiKm37L0Gq6DvnlZufGfPeDVuiMM3IEKH70HN01P8wdouG9KCPNfzZyf7BqJMW9kkFE+8+Dt6OWCTzYwWAywlA4jv4GTNqcVCndvnwwlez78Eim1xYSjRge5rG+JBl3LfwXly991vpccanYcZgEYC39V0RZc2JFUcKylG9e8hDQ0m+1YHRg29TC0U2Ew5YTYa\\\\\\\/s9WeDgZlejtozlEzjgY7+thhF5z3rn2ELpcTR9WQn8QTfAabZ9YwLQDlydcP\\\\\\\/s2xq1L1\\\\\\\/bjFM5lq+XCbGkEEtw9gnfl+1Ae1eRvar+4cbAWlZAROIgY5pk009ng3sdv\\\\\\\/3Yzk1PvyH67McyyUOZKAzcyr39rUdm\\\\\\\/392BzpVDDo5dEHVQl3sdxuJ0J9Ko3iyTKsbTF4A694VJ9mwmhA03WmJXl+ZtWvv3UcFYHEuSh0533UBkGRR23nAdOhfMiyXT6hW4qn8My135mJykboabBh38P+YYfjAI1T78JcrF49+s2XQqQb0BIuw+M80kbUUVKtgw6iI4753OebDxgwZXQ1oEKd76RpNJluqhAlKM5fevakFehyVIMQoFIxUROSqpXaIGUg51JQvG+YNnqf6ICBhJ6U8VB02c+Web+MngTvMNbpVEj0fVKbGcJY0Yjm47\\\\\\\/u1r6xYTMX8sqnSYfgJvy\\\\\\\/Etjx5uQI0whCQuRvHBFB\\\\\\\/Pv9R\\\\\\\/y003+hnzt62zlobkDWSbzz969zXAwwtNO9MS6Yh\\\\\\\/oAiD\\\\\\\/INlH8Tl1XBZL8OMXnldOmmSJqazD8PIMAvJFNFihvnr21epwRsCdcaVyVwHYixXD6gJSQaL7DnCP1cfSlByG7AO2iGMi0pn2T2fGyBOfuaVklCJuph9D5S3CZgYbWqToV08NVk9GXOqrDiY7iPfL49ie5dNAJG1q2HRgjeSR6aXPcY5eKAL\\\\\\\/Lbj4kOjLjn+KwA\\\\\\\/pQ650makT4kN+BOAl7Pr0aV7\\\\\\\/nEhpzMeaB1SWvl+tc1h0cVjKndXX4p5NV+wJBY+WgGkVg2h7TEuaIq7TUcPDfmnqM4sriGNDPHl\\\\\\\/hJ5EHDB9QfdA0R7au5yYomAuEOeXayh3ouo3uy+zgpZUX84q\\\\\\\/a6Zk0Mdacmz6m8IWuN+laC3YUQQ4YPn9aGmq5NJpG\\\\\\\/wyEa5zAlV\\\\\\\/1FrF3jy6LtDeUKwX2r3up2oSJkXSIQ1UPX1bdSM7193Pxq89FGD7RXrXxJSZSic5nXZ74sqnYWvdMAnFqMZOY8Q1NRZp4zIvQMtLr4W3ClQr18ze8xx84HPWszLcfqThq6hC37yH3r2sLUgTDHm7w6\\\\\\\/BPasDcf5Vdpbc3vcHM0zzQnPg2dzE\\\\\\\/6ORLW64nM2Gh3O6vnb3LoKthPeZstDx4OKpGUj9idcPnW0+S\\\\\\\/daF6McFpDLoeRkQRXX\\\\\\\/IgAB\\\\\\\/wM\\\\\\\/yZwmPzzBtzuA5zX30SN2ZtwUvtZBevH05\\\\\\\/xMqfPlkXs6ATVlcUFN8AoVtM076+MOKEE4F9Ot3Xj0cfMh0NgLZ9WALQV077CMORJwX5P9ggn93dd\\\\\\\/kmKFYxRde00L\\\\\\\/Vr5fxWDm+GDRfX9SL9qmVOokamdRIEhcuAGsF8mUMUzlFDqO3iM1IF7XN4TbrhzXPuMtBxQEcrFGMylKYWo3rQ+dLnKl7Ws8L5zcRw88m4L9miTq0y1NCn9Rj8ZG+fMXWUtjOcSOu8px9doOnsty2\\\\\\\/Sw5g2+UAeDhm5XMAKlvXqUbWrSF1+b85R5nes92mktpstWU0r4OnJ5tq8uWW0sNLWdtu4A8MJm92p2CSbU43lweAyiWC2A21XH\\\\\\\/WvwxQkkOvYG4lpUeYZxNqK2KuErtU2h0IW+z+Yt5u1LX50CzjfhdN7XfJ5qzkUsooJWN4gb+Zr9eEdI9MyWWJgGGd4eRcn2JZqoVJdLiGMOfNEm9uPdTeGiNmlfHdTuMVKQ7XWlsIswXZHnDLNvTn75Tzx8u+qcHgpx4\\\\\\\/LYGe9hf3GH+hkROOJpUdjQu2e3zJ7Y4xlRl82uFpr9F2tl+S0kLV4XeEhugNGC4eT9tCiCEVe+CjiZxEYbjzjsNSeK3uXe55T9nvgjDJk8gfkCC\\\\\\\/+kVDQyDWCUM\\\\\\\/y6oCWa9IGNVoYySQ+EjzuORlgxRGtRzf0yBn+TRLulu+wEyzE\\\\\\\/DfuQ1sYouVwN\\\\\\\/sCZLYPrwyWij9\\\\\\\/S+hRcgmWJLAXW21Bo7aHHT2IqMwvLmTzgcv4AEnP1ux3S82FRa91bnCAVK\\\\\\\/TEqDbugKqcGLdb7jG0a3XnyL4mN8bEObJ5PPmjoFxwBdeSgJprqmS7TLdO8\\\\\\\/qV7DhZBg5qXzbY3z5VmYOGpch4sDiMPXMmdWLNXRbezA\\\\\\\/x7RxzUCmhU4a6mIPBfi3MnsJoqdnnDy20D1HbkELnOd7q2rkAD\\\\\\\/5B5I9oZ2RRimzeZyrh2c9DRp3BL\\\\\\\/Zv6T0f3ZdZQGDpA0t3VRmPJ25EeYpQWdYMEwj1lAsS5glXid95g9opZnyePfdQCaZ2QglLxz\\\\\\\/cTf6N6bt3O9eb4ijfjwaCM2LrxxzNHVCcfg8+Vrh1eoJJRi0wXEDX2edL4yOM4OUoW7nmgcYGHPeC6x2ZPHp71BAL6caAKnwk88nJa\\\\\\\/5MRujnTBOTenjXqBJnqCucbFHoCREdq5VWNzj3juxXEV\\\\\\\/HF3pChYtyUGhneFaPaQzKbZzWeFWfJ1nnzSHPu3G5RmgHDjxUt9akrcYFpLaYd4VNpWFMV8uU0lrzFwQhdXrbmn+\\\\\\\/nmWo77epI51OyQOQt\\\\\\\/EFxLlNlkq4ICI2xYJnfv3JGcVbjQKXONyyr+zAuMTUGHnoaulyksDJICB2ToYM+S+zV46Xz1LlfuJEpmRrtcHIcnFht6Vy3B+kEGlJCQq0bcjO88Z3KdkeN0BYPj0wZNLQ5+EyTG6jmVI1aF3jSTpm9CFS+QBdCBlkCoKLplnmmmUZyflith6+jzdhnFmWFcl1nR4RBoqY8zIlJYDFgd+ZMzo9fpyUWDVTUR6h5ZRkzRuzbEfX5JTnrDXLBeF1ARECdy\\\\\\\/fOYdTPEPIht\\\\\\\/\\\\\\\/cM6WaJgcdJFHc5efaWEN8xZ856qcDnAm4c3ibtCp1ZrNRO99r7uTy230dU9AGyiWwVkxZ4d9GUhDXjot1vGCVJH3Hkq5qZ0p5oAAwLsPd8GRtIIZkZJTbGWexEzIZsVBNrsUbpMf2HguhmycCousicIRLkZVGxfm7nE2+0ji31cDVdzNqLJViyp3NTM5o4hDHRk3+9\\\\\\\/HiJxE0nzkvl0L2IFhz8LBQRJt5y5TDkdV1BS2GVRD\\\\\\\/QnYkzPC6oyykzJAZoj2bw65PuUjcNz3b6\\\\\\\/lvMT6WsZx03244b26XDaO66zs6\\\\\\\/\\\\\\\/3HA01qWPLqFuuKra3qxEWBd5Onw60yR\\\\\\\/VNEJauF+dmuJGZ57yj+katQCcJIyyR5mxjnqs4zL8qDFp+FH1RL\\\\\\\/OIMNfZunMOedf2NWmjY4TXTIq6pwguwysN\\\\\\\/ZNzGXcdcJA6gKv3jxQjLUqbZiG0f0bEA\\\\\\\/GzNaNf1aTa+KVtcHQYisf2hLPiTJQPzORBSHOqN7k3baUbqN\\\\\\\/mB48yxP0bEc2PZyLjbC6dgn0Fuxftrh6TwQIoAsS6tLJyqmQlifwyaymiIvLWDC8jNrz8h4lr5XUpHNbJnjfPF8JCOHKNc2PeX8w7TPttZkVP1ZrkdifOrWD6sEZxtpKervt9DTNDKml9Wyxa\\\\\\\/z0wCejDHxWCoASHUJ21XhyH5qEy5wky\\\\\\\/7hPuN\\\\\\\/UQOzK1LKS\\\\\\\/lKtTmOU4A7dvbysqti2fO808MH492bYi2TiLkWS5TdSVOtC7g\\\\\\\/FRy8AOZpWzieLpHYsjiyliz+ZxNmXdPUDuqJm\\\\\\\/2zeMuGgJ8HwN1oieoazQ+5uYUXC5DCx4lcu+AoZDNN\\\\\\\/O2v0hf58LC5lPgxOWjS8GE1BTun+MMVSsonU3TLsggKFDqLDvmqM4PjYfm6rWPwPVGK5bwlG066N60rEbBrwYqVsuNGsTeiSh3nLHnjW1\\\\\\\/iadD44vyU4Cz+w4zWuaswkYMIJdeZCCkS9Yl2me1ROKQiVj5lAzheLJByauCEYLPw4POwVZcrkCQqLMxfqG0iMsaz2209ENP7QMXmhhRm57g7jQTT8TrtHJx\\\\\\\/if4r7k5N7TBnucZCDEEhnpQGxzsxCmk49Rf8q1vx649t8rsj8mbEBBsNNTCTW08Vi6T\\\\\\\/a+o6xFWYN08GundOpr1WbnZ2PrpVXCGzkDcTu7bU9FM3DvReADnaOPY\\\\\\\/tRlv\\\\\\\/T0gswRNnFy1xCD9INBjPpVPF9axzss1NsUlLHLRtickey9b+IHKWENW7ZeagPSzWF2F25OlqDvExwX7g0hcJjT9\\\\\\\/KYFS8FH2YDe5FO7quWkyguUEIVS9gdxbTk+fCvxA7KP0kQ8g8Z8HO9e\\\\\\\/mA\\\\\\\/yJXSkJnX6HR+twgNUOhykBziywJP24ZJ3se4NhXaygGlAXbIDqV47ObfGLwOqp09+GV1F13SVKP6vtQprebR9o8h0\\\\\\\/BZXZs5hpACKQ\\\\\\\/OR3bIBMJ5vfUBtbABNWT0GbE6GKmor\\\\\\\/Hax9NDM8UgVNuB9SUUY8l0sW5OJgNHigKjYFQuOsw+auX9gEXOUEZvk68lSP6fuJsaY3AMrWbhcnbx1aR0aIiaMvNc3AnQpjHv7dNfZBYglOMJ\\\\\\\/3PS29QLWXNyljT96U5yLaTNSFy83BAPoh5Wya5df+GksRZ\\\\\\\/EqY\\\\\\\/6W4+iD1fMZk\\\\\\\/2YAKxJM+oGfVXTWOjpY6gYHNBfvphoW53ernrnrKZnOtnmdlrkTXPPAhVB+OVCj3SXD70fiQgCSjp2P1tpU3S1ov6wct\\\\\\\/AyRr9DejNlNinafHb+vZO\\\\\\\/hn9W79CC6OTp\\\\\\\/aM34KZ5nWZAcW08HDfbfIrjNwXaVhVIxumZADS0iS7\\\\\\\/MhuErzVvTHuo2sk4kY6CpGdAi6i54Rx\\\\\\\/dAoRdpexvpLw421eMpRoNTxDCCcxsjSBUM1L5yizqkhaWNE5iUTAtybi7s\\\\\\\/p76u\\\\\\\/CbV5rOybEU6uw1LtkbnAZoVgH6GmQgVvKaRGOEOymWcQ09rocaz+ywbUZhPPDKIH9+YJU7T9Q1prjHSJZG\\\\\\\/G8w\\\\\\\/kzfHD8ljPjCBibrBMAHfEtOlWAjjGlqLZRaxOLhKTYv7me7Vd0S9ORtFe8oygerhw\\\\\\\/eP8BjRM2PWc5cOc+T7y2dUrlPWSophMb32dt6tRlAVQT1XvIGPgcMyi6nTQLPx98uHbD+LCtSZQ9qaSW6T2BrVYm10z4vr33w55UvtUFhsan03pLCyphkGlAyFA23jRW3M6Lb+Y3p6ycO5FMIJWOVlRhBMbpqRqaU24oa7OFh1DVlHZf1rbxO7PeLHzD4Sx0WxafHMHRrWT9pTr\\\\\\\/Om8b\\\\\\\/Sp9r2O\\\\\\\/Z3suwRSL2zpDhMVuW48lI+uiu6nm8a4pUE+NK1i\\\\\\\/+vDqhDI30YkolexJ+\\\\\\\/bOnt4THC\\\\\\\/WRztPuVaklGJBQbEx80rH9wzEpNDL7L3X6ha2gdaSogDOAL40MpPrqys8x397XbcLKnikqWYhAKw5s38bQ44hQHqs7BCjmRtNwbAuYJU9P02l3\\\\\\\/QtGzq8TxW9PzMOYhCtdFvmYF21op1jLGGrN\\\\\\\/5bsb3iDME9KRni9jxPQ8SQr3lKYmpAVvA2J\\\\\\\/7LL\\\\\\\/Y5\\\\\\\/1ZyvsGDe79W\\\\\\\/mwT6RdjVK5XskvlbrcCbyNQq+f7SImEXN8N6nhYroT3M8beM0Y0PoMafW8IQDafCywFbZk2V5rqZnswaSd8Qcv1jXTjwpyqxo6CKyeEIytZdtbHnXe6beo2TPTldCa0qN7fTc30U629g1zjpjf+sjrgdq1o0DqwjX4fbwBk+t60xJnyr9ptZoDI0KJK6Vo4xBsBpH4dfAGHqzBSGnSDHUsV2ImnaKtZY90pq3lEcrA+qwOXcAj2QdieGTVI9YcII9H+NAJ7z5XsI2WrgyIJ6PoIWAl4SEdPOQtr3w1ZB743W6jOZdE7jWqUe0rLpU7nsRyTWQKFbMEzxft6p28CJqigI1z\\\\\\\/Fwn+Wvt3j9kaUZQjaFnMLwB3aLJh8JifMLJdDA+b4uBkwH6LOEsgdm6uD4e1LmWi1PLP2dxGDxWJlcLjBobhbj4s86ppMkhzZcH6HxcIDTXx8J4LOu3qPo7mnQtIus1a8XKbz\\\\\\\/gLgE36dQ9a89R+GDFO300N5DDJ4nSX\\\\\\\/n8TCNqOrxpazDniqztVlMSt\\\\\\\/lOkoRxiOqxV11SU1+\\\\\\\/SJ6z4BOo8CIs1s0\\\\\\\/OlFndbIsfP2u2g+VIUbHYcIkBHuurWhK5gBrOP3dEuiA9I85BTqr2H+QC+nrW9uY6jps\\\\\\\/FuR3tK41PdaeEJNh6gJMoh6ABJL70oQgzGMS64+na7R27dL9pb58Q+BFC0nH4s6ft6UNH08CMqUOazxU+IyZImrSe1XnhE6zW8+rWaI7p6icfvKU8Aq7vW+iq8so+SSQUQYjiBjWDZ8J+kugLtQT5OdFJVvNV8ZngK4MS+GXSz9ixDQmv17U0GGpwVixORJR0qattfNEeXSQ9ukPu0tKJA9OLFWLoWPf2aZAq1n5lp3mtbu59\\\\\\\/I0uJgHAEyX8n7A61UMltail0+XchBfgj5nDIW9MVQ9WvZwXjAIdZENEkook4+gCmwpAKgUxa4Sp13QTYaReRXFATb3kXqEhVdHDoCSs+fRpn8AtETCA3VqrfD2EItA8mok2uxEpwHUfmxusGm1HbeR7PrfuEiKRbp+K9OoB9h72zHMHZpGpU4CkWyONeGwUvlkKjC9h7xAC9KoMB5K8VZl4CUNpcPb2db6zLV0w20fUS5ujUQfDvZpaJ0KCNcKiwTpvIuVx8HTVBVUbCOPr4i1LVo7k83QYfBUMMEJUOz\\\\\\\/LMth0p8ie+CQO5QtKxirsgu4b6HGRENv3iYJvSmLIMvXnvtDbY5dnZ8LmV8X6uDai8\\\\\\\/GRuyuz+s85KOvVLBcFqpgajzE2rbk4fmr98d8CHwAh58k2VLvStY69Rbo6mQlQbpfEqprASH9Zododr65i8TUChHPBfohzgDSzwgmkMsA1UjiTb3rlhGZPC14FbocSuLYxpX9xze303BqGtyShsflixB7C23dIzv84hZDtO9mvD3BK73TLwQZtuySJ5\\\\\\\/b4VeHeDM3WNwtNGCpZUppDfRAKUrxEYlQdbWWdyZ9HGiJpvfohZnsZCieaFWliYhyX1tUmhSbpKb6WWOfEeFmbSQ1Wbre5fvqIs7DDMJtIYokyhQAO+PnI8nuVRkTwQd0w6r8e0NBdaTsrvwsi2mtKAxl8GJ8MjV6rjHeucQ3HBP+iVWU+7lv744rbARJjQyM34kCbb9vE+ptypPIQDCHfsxbVzve+aiJXKB9pmL6ABpwFyhWchwas6hnAotfPcdFZ9XNmCnSah3J3Ob7GHr8uWmfA2c1+rW04BjNwBTcqWb9NMtcskisu4TvOHpP0mTwFAdnqgBWuXxCxKpIqo4kcKpxIEwJq23M6Wsld1FtuzV8pZLtAjBdSdecyvKGh4BaaRxHw\\\\\\\/gU+zDL5z8ZjJ27pKDPGm+pLzDZgcDREPKVN5sO4jO4OG8N+On3uwd8CR1wDwTvYfuCylEIE74OONlI44GfReG78hmIzkkppCy8LcpSzNfPZ80An8QJzNFEnqdBB7UBk+grAlgvGY2KOWBOPma2++poO2ncjbDiV08tZj4b74QljXcZ52qtp8U5kuotAK+RDuy3gF3aew7CSIxKDJGkcybtjN7nRt6RMKjGjdx9CgepIJsPdPUroecFq19IL6vU3A9jSypYqfgOrQl2JbdqCf0f9rK12yZsz0zwGjByi4HuXVpICIL4aHT4\\\\\\\/miFg0FzSOdBFnp7RIKjEOuNFjcazgsSiviwEE1gl0gUm0VrdGZciU55yg2nQ7KFzWypnAqCdqvxzhZE4Mwd1lUawVuK0HeUxjFydwcKIDTXB8r5xQFEY2z\\\\\\\/piGRdFyIQW2yNsY4YYSLaPTFLVAqBaBeUuvhPQcNIToLD7UdUN1vPiFNYcr4IeIWHwkt8BhSH0UGs5yADXtRgZhPbbR75JTb1IltoLXPyJXEgNXaOCgE9hX7tsOmiSUVNTJujBbVcM6PiCyzt8Ha2\\\\\\\/dMmXgmGzo8K6w\\\\\\\/LIShu3fbbMd2VtcsPPDCqXQ3SKQ+qdPFJDKPBHeeL0U5GI1HxvBk3JjgMFNyvNvcxnMhc4S1X4YS+x02EiVVHuGuzNWg7tgvdiU51zAqbbrhDxZbRs0XgBpt38P+5629w7ECN\\\\\\\/KMcF\\\\\\\/EHYN\\\\\\\/6pCSR1ZutpEAeBLbN2O6E7x0pCkW91NP+sI82k6FX3jut9J1tAsSzBZrMOCpVwglAXUrbAQFredMW12IQ7RsiUvJRZR8S3n9atFsjYHbPMiSdn3ft2bBeMLkHUEKKYOiJBST\\\\\\\/eTwQJhxh8W4Jn1Bqj2veWY801oYSyL3136KfOe8p3eRepGgQutVXn\\\\\\\/bmjyIGRvgQ8UHY844Z6\\\\\\\/9Ys7TVJNmxWpeESYdA\\\\\\\/fQpeNluu2kPO+BubeyY26hntS1Hffiedf\\\\\\\/lwfJDvDilzAU+1w8yrRHqYXiY4OyOItmsSVa69ajBXcwJ\\\\\\\/o3EiFdzWI4tQlqamV+EUM\\\\\\\/ebA4bRFQSXxnx5bDZ34Jg\\\\\\\/Dx4IlHn\\\\\\\/6bmgQP6XwSVhCVwtam4vXrDln13TcnpwTI4dB0+YTINL9ieJa16kewgMwm4Yj9ELNaWoa5iG7C+6qpgSgU9Ea68U\\\\\\\/C+6Lk\\\\\\\/Y5iw+p\\\\\\\/Utzxh36YwCuAV+w8Rcxfh4Kz+QEw66Y7jxjZGLPKrx8CrBkItErf9+eS\\\\\\\/XRVz3WSJXxTadA01KFhKeVuGHKu4KHULOo53p8ee5Ny0Z7cYMQz1Cpi0PVxFl1cOL6I2zWbjQHy9WgjYIpFICbcKYx2A7a1gMw70rmu91+dRGvdl15syossRz6eQQrd8NP1+vE7j3tyqXoKhQL5c0umOjQqKBfCr5O9TpKjXfvk\\\\\\\/yBeazYYCVmmgzMRjJ\\\\\\\/PAAvnqjZdeBhcrOCsUyeFkx0stcU1tqlYufeyCfZTHaIfbJYck6ZgIfp\\\\\\\/gdGQrCObVNp92XWAxzWFhmgX0qi9uV\\\\\\\/CYeyyBoy7oL4cuob1KGj5ARVRNXqSvjaDoS0NZQDVVWHGcTJ2w+TRg5GUkvcnUMNDCw1pnu7ZcM9i529YhknvSjCLqHZMQXP\\\\\\\/pKHUYMknZBebQMUud\\\\\\\/jgeLb2B69OZfhCEHwcU3S7+\\\\\\\/NvtEV3ZV477aX3QxU2yHzWgEJeVTT5Rc9oP0+Z3TnKRp3zOYe3n9J4NSKmWffAWX6BmbZCQK2dzAjt2q+ISj5riYoR0ksT2wog8dvVfptuPXgcCHWbllhj3SF\\\\\\\/PtCwr99zefxOOzldMFffmIoZnnKK+eNVi7QkkEgMtJPkzWfTKsZty6FY1S5NfDRVMl3gDSusMTUF1vo3jj+nEMNQ7tIWFKK3x04dUA+HAcm3r+lc5oQeoeoUNb\\\\\\\/fz7K7ycuK2cOuE28RgNQouUrScM9O9Uxivq3XiMtCKN+btYWg725iTMNrdqAn3rKlpHhc14f+Fb+gIvNY0CT0j7t1JcHa2FvYHq+Bgc3bkaLnACs7P7hFYoMzZKVnNHcPvNfQx0r7l4IbwpzsV1pOb3oXwAx4uphXYJriLkMI6woXpij+2u4ZzOz82QVRg7gynMFpeP6k8tOgHZbeKNpka1PRC5QP1NnSN4A+mnz2TaQamywyhgEFRI7UK\\\\\\\/67TQ\\\\\\\/n74PxgHD4iJyNc6rkqfBU\\\\\\\/6ZSBtYP8SYDVMxasKJXP\\\\\\\/K+HriB0TVz1xLQK6iwYmtBrKPMuji+iuWn7qbgVa+XKS\\\\\\\/Ssrdg6AZOamFhRZI2RMc4YiLhb0Vi1F\\\\\\\/2CyCCJ7xbHdqiuRg+Q2uCxGRLY\\\\\\\/C1RPMlXh4JIztPXmgGvgQNavg03UOZCxTlQTOf1XX2JEWme9Di+7aTgSCEfByp3DhzVyamc84mcJ9VtdSkx1GhOrxQJlfSLeTjOWkC6sM4X\\\\\\\/a7Appfser0AIbkn+EfetTl8A\\\\\\\/Umz\\\\\\\/N2WzVSvQsrO+skNDMfYR\\\\\\\/StoRQSoZik6wEZbVnsW55oZpBqAv60Yun3CLb\\\\\\\/pHpOP99txgY7oEc3zzXSMTsU+rxqkQt3ZnBxKERwPFbg4wuKJVb0egswTODinnuvp5jcW5rpY5qSwwd5xht1RNYJOazZWtaCj8Of9J0tsQvwtYylkEfrIaqyyz\\\\\\\/KNbd7\\\\\\\/5LlbdeL0Ad2kGVQHD1hAkWTY9MXecjQduh2cyrd4MZ6cXAy6SC\\\\\\\/XGjSPKOwhkY6Y2MqpiqhUil+BZNNETuQTfdaYzZNg496+nuAGDrXPxsJfv8D0So7ZPJEFkh3T97+Btbn20gGNG01VB0ITHlBAf4DAzPoapAh7a3VhdCXNmQfT7lczbMI4JYuT70f82YGxgj+ItH4InurWwD+QOymE7F6x5CVo3k\\\\\\\/PmSscaVyFVPsHghDRfzNBOzjfgdrAPzvC3f0gufOAsLf1pWAvnc0\\\\\\\/shssXdoAUSfzIxTdqn8XQo1fbSkrLWA+rUFSzKwXAokUZ4Mlv6KyljrQ+YZK8RNQDgki6ZGfKuQJ3GGSP5XJwpL0WWDX4ZK6D1zBCSiqo3W1iHQ34WpziycjK1vylKi5B5C3mIfW7n0x+x8+rkbVbOUbYGgdfSMc+Sy37HR0SKdbM9eHDWF4gm5FQ8dAPf+ch40UCmLsx6xOCpJ+Ww26N0WEYpq7Lerq7+rNP4+zyBXMyvSdBZygZSG+b7wnh82S1gONmHSJPsWZT00luCYVdFBv\\\\\\\/6Aa0BsMCooIap7+9TLvGSvYgLktdhqCjCJR9eag+4ojzmmwGvvt0yLtJbTsIyz5BiuFbaflplbxQOmpjwviixf\\\\\\\/wuEXbiR1wQ15t0Wn01af2Xgls3tsHcF9pO6Oz+xEty2oEh7gUwvG86p\\\\\\\/xy+tDSPYEIAOTSx5K2294ppMN3uKywtbWqA\\\\\\\/yJZu3dTXti2xem7M+sX1hS7\\\\\\\/Nq1HGE8CsLJiC9EBUQs1IKy4Orwpc+2eLTwyI9dhf1H1UAMb7Eb+n9hKbPsCpVSvS6xWM7HFqvDTFDfm4iNvGNqPquHuJM5Fl6Fd6lTYD+PU7tV2MHp2xzzQpHFlSsQG6zvTVxBuXeL9hKLJbiF7shg+1Dn+FHJHNgutjKMe1LIxffogxJE5V\\\\\\\/38qBHMv8zanBo\\\\\\\/1Fi0\\\\\\\/7\\\\\\\/mzaMQAdo7xyx5sDuiot72st1b4f\\\\\\\/e\\\\\\\/rNNA0LmiGJqKzcNR+YQiVApnvWVxsO+\\\\\\\/4lQBPXBm1JGDHCeMhHlfjZCdDQ\\\\\\\/DXTSXr2hBmMWROuXeC90kUr3hspZRMSSS5ANkX2Lwe2Fv3eO4vniOZm11OY1Rph+nigVSfSVl7B9Juyr5FCd7RnQcCuiLUmY\\\\\\\/6TJlPoSNvBaJ0G\\\\\\\/dYMOVj8w4amt2FKx9nZqKBGoLCvMcCVPVLvUGGXqKm0lG5VVyYFg0UicrEDBUZVNfM8W4Bm57wAjlMp\\\\\\\/R\\\\\\\/JpVJC1rw9gn2\\\\\\\/2kgu6oJJBHBeZGlimPky64lnYfmwqB8JQ\\\\\\\/7jZeJpy5yAUfZzuTD1Z9qtKaDlSxgHhTk6P8v1V9oMxu8hx\\\\\\\/IMvgqpxtBKgd8qSRBAXmWmlgUoifAAd1dSgad+Dn0oi4p2KIk\\\\\\\/+gjFFigrRJBSYIvvOFWDMvDIv30pJ1cesHor0MwH8G0M5f3spOkUpRb7Mqoo9hORAGruXvExdcPJytBsG+O\\\\\\\/dGF6df8B9OvATNvQzp6wlO3am4Letg0Sle75rByh2g\\\\\\\/S0jv\\\\\\\/0TMxzWKtQMjAObpXUjzS4ju8PydOJD4bb5yhHcmpE3ZqjEgFy4CZ9bAj6+Svp+iWKrC2mjj0ZjuRFRgGvi+t3CQRRzHaaFwuoAjTJ8hZ8Xv6xrTwYPtlm94Fl8kbIkVrYABdrycBMIGczao9idr2XfVTyFJelNV9S\\\\\\\/2PONh4\\\\\\\/L3lg1ktJPnBAS2SUmIlSC+VGykysQdVCGiVqW6llGb4Tw2fKgtIzM7Ey+dYbfTtYSS91Q+cJzMcR5yKYPcZJhNSpLEZHlHj5V6jYvFlRFRBII4SAeWIlkE7pKxuY5F1CMNG9ZTAgbdWkvDihGa6pf7OiL1DY6Ak3ZYb8RrLZvGa79vW3fV32Q6Pf+ai0Va7aKV3uGj+TLZ\\\\\\\/ClJtVNzANwOjmd10\\\\\\\/i3oLODDD6r\\\\\\\/gtPX4E1PzqIfvdBI0q0uVaPQPsznk1L4JKJEEdMLc3DNPbBHUgNI27HymjXtA3DYZVDeMGo2Bx\\\\\\\/3zAvnEsftTiXrQ5bYCKWF7vBDkOKZhw19yHMKv2\\\\\\\/1jRx8c7uY4dSUg+Vye14a0nLh3rSQJTLgYO4lKoiBY4TERgonJvWyzQ3DWOAETGbWvdmphlZy5lTAnVNObqYHZ\\\\\\\/M\\\\\\\/MIW4NCfQYuvwiaUbVR5o4+gsZuUj43iv77JkzGxu36yHNnnK\\\\\\\/bdgXM94lcbPxMFKOerrw6+z33tdMuy4C78oCbJs4LW+OdmYNQcLa+Tg1uvrXxVBHK6VpKAOZ9\\\\\\\/9zbRgznp8ZZd3DwVQxk0ceCy1tHebkYQ\\\\\\\/v5+XRkoC0jHTa+KcMQMibQMatoSBu02hU1mVhFxAmjtGw4vxpX8Z5LogTwKE4uZbrwjD5BWOhAQQeE6ZqEBHBymJniKALhV4lgsi00W2\\\\\\\/GFmz2FGdMbPFMIWFp0lq5idKQWq0aztQWPKrOWkp1wu911vmdfbTlY1wmlgAJB7CWKoWyxUCr9+zrVMpzGEECoogzYFtwQCUgCGitf1+H28X93zGHcoPk2cVHE7uZBHTbLWfPnVl3NIVT8k8XtFD5\\\\\\\/3IDBNT+oPS6cMtiXS211v5bc39hcdlqO0CL3LMRce78y98pJz8jUOUGRGodltdKKgo+ZQjNbPX6PLqraHON+RZhDLyeUSBgUHEO6RcuDh9fAG0btT4dzQL1e1i4AwDZXqDQMBqtbEAP\\\\\\\/Y8EKeh1EkZzWRFcz28R2RdDRcolr5PX8w4lsz1ylWs2XFNy3rBhvIoOzOZnyhcTRv7Z9WoH8udL+Qjb14xHutReNZDEj8mcc94kOyXwe2BXz2MCM26NrY5tOpjfQB0FlowDuN\\\\\\\/JxAZ92gPWNUYMwXLj4Ie\\\\\\\/exuk354jmiNc3gFe2bYAfw382aaodASkx3EZcQCXMYYo3EGCOweJuS2VnTYYLhv4UKvHt\\\\\\\/J3HWHUgmHTPRFDnnpTUeDoyWXzg4a3avru9k7FZ8cqeyfcd65+dx3DAjIElqNU0B12GARdRQnDKRTrguvHL5ehyLhnRm+Om3WdYHFEU4HMRZRtTyzaOQ76XA\\\\\\\/BRrVTfEBHMZMkv6qcoKcCy9p8ZbtilE0xntBiMDK\\\\\\\/oQgGoHzROzxjoAvkyhte\\\\\\\/IsOpyyA6VagJ3aD5CqxIpFahJagB6b6ARsQkhXuADDvPKFxn2BfnZSjOMPQhpjyHlEAgVObph7ELGuBKTxL64OqR4xFqJCvp4PyJwANi4AFFgCf\\\\\\\/gDHoEAKn5ZkTnVuTXLi4L6ApfP5R5sd1T1fzUhuylksI83EIB6XtS7qf4AtYpULdgpxzI90kPzFeiQJdJ6FnI4ri4ju0ExqP5LxpojC8uiSvbg\\\\\\\/2lT89TAu25YY0hM9U1rBWgw4InFwbJabAADRYb1lF0nFsnSO4p\\\\\\\/qno+7syHVFhCXi2OOzmbxBFJbbZCz3QSn3fagaiSu4bGrgABWLCzzT3+2H3VAfCHX8vGlpchDl1wa7TcFtdLnExxEjmUqCaG4KhGCM\\\\\\\/317Xm12je1k6NzJQPXE9o\\\\\\\/+DcllKDWWq3nVprEG3XISctnMf+CZEyQiM2O1A7IqwXF2+lDTi9Rdnpf5F\\\\\\\/0BXU93VmUG8qGQRoIbjkJICCzh5pu89Gbnr3D1G93miTaYA59QcIB216lXO+H97Az17Yw\\\\\\\/OUeP0xX30J85319F+5bwCOhSoSbKjn8Hm6zg\\\\\\\/aUSJ5jtTsn2XXDWKMX8czEWwUm5Ozowmt7zIj7BFIJUB6tRGDupvPrAFTjcAoO2vUiqowKuocuS9d\\\\\\\/wlZ977i0QWtpp6K8bZD6BYFPw39FHhugSnlxkPA\\\\\\\/uv\\\\\\\/uEDrtsC612Q5WxZX44mF5dKv0ZkfmdDDGJC8UcLYvfJfvbqEq\\\\\\\/veg\\\\\\\/KmcwCXTLdOCJUyWWzN1WnviBrM5zw8ESYK4Ac5tEenpNvuH+lShUGLdVnqU4Ed+foO7Dx\\\\\\\/JmRN4D9iIM8gtXYhnkToNoCS7wISXZNiuiwQgyaHt1EqAqjxDhZXoLFsDwdaURf1mIPW364TxIr\\\\\\\/e728qNT480JhMyiyOmZLbBovPLqZ53rtCzv7kUBvysqVtxDSbBtQ6KSPJ8\\\\\\\/EfKJRhvl\\\\\\\/d9CQzvUaxIEB0G85mZhyG2ruAU7QHXszStN2\\\\\\\/QUee\\\\\\\/fKYFeppvF70G5giwKuboOkqI\\\\\\\/txKVuh1UI76+TpA2tC52F5vrxPMA5wp9AXK\\\\\\\/xDemSmpuukPHfNrRUJjPLtlTaVciutLsSbTykfUwn5jCXGummYWWgT2Dvwt7iUKz2D7jk5XbSEMnokv8FvWsconFPCwVHWR3MWFYMLB3lXe3HM\\\\\\\/yhZEuJBO9uzKRBSdKoSVvEDM8gEFJmfmgdPioJB6A4Gs1+5FNCvzUX49IC6rb89GUCWUVcyUk7GyWeIfik9n6Ftf83UCcAWw0vfW1H6pTkT3aZeSOPRcHPZaTmho1Nic1J7r31hsu0wPJQPomExJn8PPr2uRBKRpUoSsPfcJ8d8VNXZgfL3rVXRkO7zmjSOeb3b2fLYIoix9NdJZ+pLWbe5Z05RjpE5v+0u7uxacdIKEd7CgwRt9eh97UDw8CL1nc6VBfNM2HV65Lo8SW3eQwFp4hG8g\\\\\\\/jauGLs5pGCxQiHn9ePBl3pQz6lvqm5aL0YHM+ZpwsjmgRkgiiZ6HAdj6tR4u3ZLWjihiVhlbaxJ+zCL9GHHCCyWlemxUq+8x59XOIGsCOM5STiN2kHWYLoqh9cZ1u\\\\\\\/HDIJ4mJDnxl+mpnpzuaDOFihtfvxaKaH0DUhEHNUHkl5EBlw9EbT++42TuXVkrvd8ce2r3uVAbd5M1lqIYwqEeqNtRq4gA5L8HO9u6EMK5qkVHKLvud9nnBYfkgYhVkL26TW6xSEKvxRBM552ff8NIOd0u+jHvNiEpaXtqaxW3PsJ1WWz\\\\\\\/SOdt4JDel9+\\\\\\\/H0gan7JJaCck\\\\\\\/qarzXMNBS2\\\\\\\/xTK28Zb60G4p0aQa3p1i4utqXaSz+gIjMTLoazvuwgekglKwu0Rzgu2I0qcdeuf1rS2lV6mHpjqlDWK9ti5AEKrk+L65TOP5bqCtM1yVBcBVLPl0cPBJOWWyU8hc7AIa+52WIA4xB1OjDJGdYPAwquMU08zZBTJjLJ2YEMfbzPsWejEM9ydFHn6kFYkHkgAhA1lYFEeEv6DBmaiDY8Zs8h2q2jAT6A\\\\\\\/7TCltUue9tiL65mI2jUvjCKHPNEBocwvH223FThlh61w8q0QBYst2pEgklP1a9esjk+S797zSo\\\\\\\/wn+94OkXRJvHfV4et3a7Bd54w56xZKFOAfMhO3BSq1nbq24DXPHqMoEK4mdI8mzxwatWbnHUhZowvXWP1a533Wd7mdznWmczXkLJ0YTCxtE8+gdpVkmYZ8M8rpsTwthlUKdcNgYqxqWvqige59HaTsNZvACbFy5eQ7EwiufihdG4nI4YCNk6xr13GGRJaaDI2WAPnd5b1loINhROnF3fedI5bkTNgAEtqL3VeKvWNoEtYvAZQ6fo\\\\\\\/iisCL+VQhll2AEnlNnwy2n1kWrhaAfAgP\\\\\\\/NMILpDqnPm73mhX3xNbzkXSYpW6hUuox4s8UZ95HDNn\\\\\\\/gbPE16DMkoHZlxBs912zd\\\\\\\/P1UHrru6YN50mnxitDWGifoERYbLG9XYx8fYTWPAAnPb6FsGE9XZq8\\\\\\\/3ogGz4E4rCMkWPVDELjG3GxP79gw5zrlM1hHSYXQLer5yBzXv4w1zuAmiB4njgm1ocJm2m1R+yyA7q1EnlJjx6ZGTsr0U0m3CkHwkWNLxwKecriwsq8qIcYpZOYYuo6z3d2fT\\\\\\\/sbmbZjDcYhe1G7ltbF4Bo81SHde2AC6UT+x4ai\\\\\\\/wZ3CrcPkyx5XBLa1GxLS157ZUG4zyh61MWsutVtQCSkj73eUtB5oVndxXtpA\\\\\\\/6P6kX1ZFIiGXgEs21fJviqK45WYDOi37vU+xcl6mG6lHIbIFD2nHZQjeCD5NRRMP3aRmMoKDZkz9XUeafA7kjeakfOO\\\\\\\/H1onJIbqU++ZQBqjsiEZCcTaCYmKiNY4IpAhhzmOBM4w\\\\\\\/Syh9p0\\\\\\\/kOBFWxoFkUqCYDz78VAbNWiIjW8dKBFyO72dxqIWUVav6ygew5uCxu1XD9EoOQSGSgB969v4LW5an4dnVOvwuXGzIBdOIh55k1ihpUiJAjTqyKvJuWmHhbGcZnuuVjBZU5G5EKRIB3XRXNq1mhF6NQtmjIvEcEARv5zh4bme7EwWz+V1z03zQYwW46l9lCgoQcwOjpYf6dQB44L5ye4pAsQhCcNA5Kq\\\\\\\/SM3cI38P\\\\\\\/w\\\\\\\/v9oHmy5d+g8T\\\\\\\/piac5gFCGO9InQJ+RbYJw+bmufp9IhPSzBjtqlHWNXSzw6SS3WbWMImjgR4GX9MXKPAQLV3zBTk3maGHP9gRTCXMoFmV8E++k2Q33jNUEXQZWXAlUEhTF3Z+73fhJMEHTgxHirNMZ4bUGEeBwEsK6z0Uc2PzMIQUoA\\\\\\\/oftLhUjQodUBMCnf1\\\\\\\/Jces7QwqesOmWINS0wpot6zkmCcSuOcLhiVzMaV0xUyzTzGlf33OJwR+IAXrLR8kLM4rizAGJLJwcUWXS7oeoCSDfCPRnDB2EqHoRiC8+\\\\\\\/IYCs41rtOOPXWeBQobVDK\\\\\\\/nIs5BvDkgHWZxBh7PlJSOwxb18H7xKtGcAnsbYSGOV7NZGLQOElmBHUK\\\\\\\/oZUNavS2ibjTvJIDbpRdBxGD\\\\\\\/d0M65rA0fGdQJVnoAEZDkWQVdDZPEpJkSRvcwBLWMa6zQqV292vzBm\\\\\\\/EROwQSFl7baRDrZ+BavlP+aKr0CDIK51Iu9Z1AhR84C1ACR2Arvgq+BlfHm0IFZMLBxXj4gjh8X5H4sudhvL7D6pk7l\\\\\\\/9CL+afrnhCC7yBp0YfAANwqiehCpV7S7qTcqhLP3pAbbOuD+plktQgSH8uYsySJiYKTIVKtjs\\\\\\\/m027r1mWHJpcqqDhM9ZBuNjhIWjS\\\\\\\/+8g2e0KsOS+IIVfzePXIz+nQUwmde7\\\\\\\/MUn5nmzneIJfSUsJWq0smJrT6fwQ3m0WIAlrMlxSUpMci3MfQwFdzXwiJPPgP33av44AWrlPT+a48435yglig9M9OSgvFARJplwxb5Yb49t01hGvx7EOG1TAcf+k4323WSRbyBmeVs+8ItiFH1S65o3HC\\\\\\\/Lga2klW0Rip0S2fBtBXfh0atS3jjlU4OknXVSJanEu\\\\\\\/vr1wgNNcREwavXzPjVLtr0N58NFonfrNzj\\\\\\\/s4aR9Owk2ViBf7CQrKkoj5c\\\\\\\/OhpjmdfNXSGZCOHXpAlsqOwbexKobBGBFPcvT7cIV7hMISxbjE\\\\\\\/W9FLwZ99pjTSkpFkIjzeiBu0Xk\\\\\\\/vEL+c0xSdy2M7Uo7oV7VftBpQfr85koQuAraoc6MYaKNAQ4luOKAZBXVSMIXNqvppE8qZ9xZ85I6lJfr\\\\\\\/DB6hYinY91bofBMgf+Ijdk3U5mGninDucAr96pDoe8+GYAofsTFlSlMHzJObZNELOMUKiuYk0sOSUXDiNTCLfmJksqnuomwXI+Jj2IRmO4MWdHrLFY6cxDi9zu\\\\\\\/NRHpX0Cd9POhxlvCn5o+HowjF3+IMMm+l2sXvjOTuFIAfBW544NZPBh3aWE35773O7Sxmienw0EvJ6fnOcHekjp4XYRv1raCm7UxgOaKZOh0BKds\\\\\\\/U6J3TYP3otpRtwtG40UabbJSNb7Gy0QAX\\\\\\\/EtFKwNQP7nT4oU\\\\\\\/Fv2d+LRrKLdvDuhMyAmdhDFBDFuZ43ufqdoKZbkamwZqw8uoqCJwR2KH7\\\\\\\/ySV8Dx5OrJ2e+Vb+TyTaXeV6m24yuMr+hJUZjTTi0rxHAKFBvnfzb2QpDRqwyj3rjzirjUjo2yJ733Ezl1HxUE\\\\\\\/3\\\\\\\/f0jP2PN2JwCFBgP+mmGO8e1b6bzb+E\\\\\\\/kjaS7Y5Lokf9qyoT4kz69LLnLFkCLe0eOWDBbSZvIAoW71d4GiEhYUVixrQ6REJJZgKJVDy+C97o469LAh7j9rTaRpJm476GpaywRK4cn1F\\\\\\\/6Qqslw1\\\\\\\/199IaSoret0J592vB5GKRYnyArZy2TywqDz+DLj77JeRwPuF30w8bnuMLcnhQSP+Vg9nVAKc6vjz8ntnpmXNukFs7qboOPQfZwI1xQi\\\\\\\/PAJ9+9HT\\\\\\\/nYmpjr38B2pbybjmPagEBxEjzFhFE7\\\\\\\/ll\\\\\\\/4atceI0gqW6Eh6j5akQQj3W6ZtEaUJKV2tMHEBO98ksgki7K88NbDvcA+hbBdmKpWpgwcdz\\\\\\\/rJNA4TXgZHPqXpnkPPng86bY+B3aOYbTsFXp\\\\\\\/\\\\\\\/+8jr2y\\\\\\\/0bffKPdp4tCwR3JsITfnS3LjbNTDBs\\\\\\\/3nfrgd2pOi0WDXftJ2ALFYljPP7vAnHMaLaLhCvyDb801gg5B\\\\\\\/3wjQ1+QtWLroB25EubRNjw+IGxBoJfmPwKGBGd9gkbs\\\u0022,\\\u0022iv\\\u0022:\\\u0022sNVNYtbffZ9VhQjDa+vpcA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dc000cd7ee78b33343e2ea500ee60315cca63e741b711a5a8b6badf9640dc173\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mP5dMEdCUjLvKSe1sMOt2+LRHuV0CIs0KrY+xPtN3SC\\\\\\\/DrvMQLWzepXyEOWcXkQLC4nIfryFi+GdwJHFViuB9R\\\\\\\/tt2Eku638ZnYtiLgUKFE5Uu4fQ8tmclTi\\\\\\\/nAgeXcossVI5FUnZZeTPJVkuY1b0cizwAS8uKcFwP9+CGtjkfQ37Du\\\\\\\/n2cTj8XcKgUxoTUlticbX2KJQo2xw7aBGbaCS7oHbo+\\\\\\\/ZL8GKDrpu8i+Tq9jIzILRIOGmjdoXzrIBx5VISHBaqJAPEU1KAX1qbvPTInwNWmMRUCUAiWRFvXrHvhRfxPtIsO2nauL6Sop6lGSs7TqoTJ\\\\\\\/N7RHM0yi7Acc\\\\\\\/dxLFts6+e\\\\\\\/tVN5fa6gt7NSgpVvwM2Uk0Kj9SU9hSSP2pyW4wYQX9i7VQ\\\\\\\/qhPHkZ3ZXIQh4xBDXLw+R1749JdfXIOMxuBT\\\\\\\/L\\\\\\\/NJiJSD+vczJkRfJCfTZqZnmzo0SJFuIE3JrUjDmlqBQ0FRdET6v1YGhy4aKxhkZiP8525hiUgVIvDtufU0hvnEXya6NV9gKdwoPRwPWJrQH10K54oxAtlATPo8g7z960QDmp0zexEBKTv9V7+z2tptuKcxkg4vFNhS60wj+3qX23WI14lq3wEioISmdPcmYHzvbe7+n1QCXjfPn01i+y\\\\\\\/MZRbyk5EzceUQ+zliAwEJS1IFT30B39GpAFSouXBuzIIkoA0OU7Gruax2VeRw13uNJLdSamSylOg\\\\\\\/0c7pDPnu\\\\\\\/VixnPH7We7Iml9CawDNBsOcDl09jZ\\\\\\\/cfBvnhs7a6Mz5xOG3yw\\\\\\\/4wh\\\\\\\/w7RzX+1ZbsUx3W4aFIN88SM7JervaQEzSj0oyIlT6mgTHnr1cc4SvLh1BsTuQGehU\\\\\\\/mhVhTs0EJjq8pYEI1vcX3dOEdXZGrhgimUwY4xnpeUOTNKemNizD0jSbIr64ep9noSpolHs00hsxqzHq56fVHBOn+QiGeE18gSU9FwRK8QYN\\\\\\\/039PfCjwqDioL4MciHZNKfjdm4lcbSnLWkgkwhy32+utRHgAB88mQ9ELKbvDnhoy2RJlD7jBq3KI53Swt1vwKY9ZzXH8SeMiAqusHakKh3qMSeIBadgyaxEUeOdi\\\\\\\/w8a9JF+DWqwlXNVGUwuCvWaFAeWnhe9ehteWjPUe9mXPYKgCgjimwIVrCvLcNh7YhMvkz1VGL2uCWM3DPTovLatwN+\\\\\\\/D8lmnM4FBxGgHIjkZeW2FpP0rkUYoAlydz8c3+fRrnJYvkm0iPPPErnIgWBqKltfP2vJcNl8hmNwPUV+xp8G9z0WRMPCFyL2XlUCSJ+lfUHO1MASGcgZPkvY\\\\\\\/aIBXWiIFi4FsFUDy5ji9EfRPEeLktggmDvmjl7IkvLirKBoQTk5lfhnVldsFGfgeWGHo8PJwCMrfHnjG4keVgDNWEgDLCvdN1koGyctCneX5w53jhv2sw5DdemAPIEnCVgG23rP\\\\\\\/9UpvkGH7pgmX\\\\\\\/\\\\\\\/v1t4xvzq0hQmSrDhdWD\\\\\\\/CdDkvZPcwgrMtFdMTEUm\\\\\\\/Q2AeMNEt8LOTl5IHtyw5ca2fav5EkE\\\\\\\/iXCDRX3YuyxE\\\\\\\/tU7I0wx8BPsW9YD9TU46untA\\\\\\\/kF5lUBo5CW1uuiSVdayX\\\\\\\/4T3+sBpA+nJRcgzj1V66Se94DFsRoa4lZgrr48nNg8iOtLo2HdhStk7YdUpDY\\\\\\\/wTpnuWPsoPwopk8vXA2\\\\\\\/NLcE2cjoh+PitiR7XD8XoKi4u0zdTBikE3KYdF7IzMe\\\\\\\/VPSAD8B\\\\\\\/7lq8p2PB6xfZU3Ye+o4jTm\\\\\\\/jgt3kxq4E9Gu4NYxwJj19ouBSu7xPQFQJ7kfB6e\\\\\\\/V0+PVCi1A0ObKoyJCek3jPWVIF2GfeZuCNU27ksXswzGPXo38\\\\\\\/NHZRqvjX480qn9gvH7qC8atlCGU4v4ghMem+r2cU9ALsw0T627k2sbLOxsGdRI+mlmnmYZhyqerw5yxpYLp1dHREInoHP1ZknWhlHlxKmQ1G2FJPLFOiTc4SSD24UCVygMH8C10TiP8fPJcaBuSdWuzjPF5sLOR8UM1mcl8cMCWfWIHqUi3PL6T6EchV4WriDY5lw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022JUL3Uz6RN8J1dE68bIk3bQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022afb4eb6f0a2f28b8e1ed8e0841f32c241810d4fe3fda0b4f76b68d24a1010ce7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488292 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/669" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Es4jb22DNWFhgVdfxRsAklidj6SA-Fh3fQpBSB--E6o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vvI8MBtbuVFbh3x2flvEdDuKxbU28NHgB_mOeKmJl90" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/659" title="Sequencing of Linkage Region on Chromosome 12p11 Identifies PKP2 as a Candidate Gene for Left Ventricular Mass in Dominican Families" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/679" title="Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488292" id="node488292" data-pisa="ggg;8/2/669" data-pisa-master="ggg;g3.117.300135" data-apath="/ggg/8/2/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of a Sibling Subtraction Method for Identifying Causal Mutations in <em>Caenorhabditis elegans</em> by Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Braveen B. Joseph</span>, <span class="highwire-citation-author" data-delta="1">Nicolas A. Blouin</span> and <span class="highwire-citation-author" data-delta="2">David S. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300135 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488292" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Braveen B. Joseph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Braveen%2BB.%2BJoseph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joseph%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABraveen%2BB.%2BJoseph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nicolas A. Blouin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Idea Networks for Biomedical Research Excellence (INBRE) Bioinformatics Core, University of Wyoming, Laramie, Wyoming 82071</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BA.%2BBlouin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blouin%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BA.%2BBlouin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David S. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BS.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:davidfay@uwyo.edu" class="" data-icon-position="" data-hide-link-title="0">davidfay@uwyo.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/669" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488292" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/669.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488292" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/669.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488292" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/669.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488292" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/669.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488292" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Whole-genome sequencing (WGS) is an indispensable tool for identifying causal mutations obtained from genetic screens. To reduce the number of causal mutation candidates typically uncovered by WGS, <em>Caenorhabditis elegans</em> researchers have developed several strategies. One involves crossing N2-background mutants to the polymorphic Hawaiian (HA) strain, which can be used to simultaneously identify mutant strain variants and obtain high-density mapping information. This approach, however, is not well suited for uncovering mutations in complex genetic backgrounds, and HA polymorphisms can alter phenotypes. Other approaches make use of DNA variants present in the initial background or introduced by mutagenesis. This information is used to implicate genomic regions with high densities of DNA lesions that persist after backcrossing, but these methods can provide lower resolution than HA mapping. To identify suppressor mutations using WGS, we developed an approach termed the sibling subtraction method (SSM). This method works by eliminating variants present in both mutants and their nonmutant siblings, thus greatly reducing the number of candidates. We used this method with two members of the <em>C. elegans</em> NimA-related kinase family, <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>. Combining weak aphenotypic alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> leads to penetrant molting defects and larval arrest. We isolated ∼50 suppressors of <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> synthetic lethality using F1 clonal screening methods and a <em>peel-1</em>–based counterselection strategy. When applied to five of the suppressors, SSM led to only one to four suppressor candidates per strain. Thus SSM is a powerful approach for identifying causal mutations in any genetic background and provides an alternative to current methods.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/c-elegans" class="hw-term hw-article-keyword hw-article-keyword-c-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">C</em><em xmlns:default="http://www.w3.org/1999/xhtml">. elegans</em></a></li><li class="kwd"><a href="/keyword/counterselection" class="hw-term hw-article-keyword hw-article-keyword-counterselection" rel="nofollow">counterselection</a></li><li class="kwd"><a href="/keyword/nima-kinases" class="hw-term hw-article-keyword hw-article-keyword-nima-kinases" rel="nofollow">NimA kinases</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/peel-1" class="hw-term hw-article-keyword hw-article-keyword-peel-1" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">peel-1</em></a></li><li class="kwd"><a href="/keyword/suppressor-screens-whole-genome-sequencing" class="hw-term hw-article-keyword hw-article-keyword-suppressor-screens-whole-genome-sequencing" rel="nofollow">suppressor screens whole-genome sequencing</a></li></ul><p id="p-3">Whole-genome sequencing (WGS) was first used to identify causal mutations in <em>Caenorhabditis</em> <em>elegans</em> nearly 10 yr ago (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sarin <em>et al.</em> 2008</a>), and this approach has been progressively refined and applied to a growing number of organisms (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schneeberger and Weigel 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Leshchiner <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Obholzer <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">James <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Moresco <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hu 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schneeberger 2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doitsidou <em>et al.</em> 2016</a>). One complication of WGS is that many hundreds or thousands of variants are typically detected in individual mutant strains, thus making it difficult to pinpoint the causal mutation that is responsible for the observed phenotype. Because the large majority of mutations that alter phenotypes lead to changes in the primary sequence of proteins (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Sarin <em>et al.</em> 2008</a>), filtering steps can be applied so that only DNA variants that alter coding sequences are considered. Nevertheless, in the absence of other information, this can still result in large numbers of exonic variants that must be experimentally tested.</p><p id="p-4">Several WGS strategies have been developed to reduce the number of candidate variants by simultaneously providing mapping data on the causal mutation, an approach termed “mapping by sequencing” (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Schneeberger and Weigel 2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zuryn and Jarriault 2013</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Schneeberger 2014</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Doitsidou <em>et al.</em> 2016</a>). In <em>C. elegans</em>, a widely used approach makes use of the Hawaiian (HA) variant, <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, which differs from the field-standard <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> Bristol strain by &gt;100,000 polymorphisms (C. elegans Sequencing Consortium 1998; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wicks <em>et al.</em> 2001</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hillier <em>et al.</em> 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Doitsidou <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vergara <em>et al.</em> 2014</a>). Typically, mutants generated in the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> background are crossed to the HA strain to generate N2/HA hybrids, which are then allowed to self-fertilize. Homozygous mutant progeny of N2/HA hermaphrodites are then isolated and subjected to WGS and variant identification. N2/HA polymorphisms can then be exploited to map mutations to genomic regions that are homozygous for N2-specific variants, thereby greatly reducing the number of causal variants to be considered (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Doitsidou <em>et al.</em> 2010</a>). Variations on HA mapping have also been developed to facilitate the mapping and identification of diverse allele types (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Smith <em>et al.</em> 2016</a>). This approach, however, has two major caveats. One is that it is not easily applied to complex genotypes, such as mutations that alter phenotypes only when one or more additional mutant loci are present in the background. The other is that genetic differences between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HA can lead to changes in the expression of phenotypes in ways that are not predictable (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Doroszuk <em>et al.</em> 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Reddy <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bendesky <em>et al.</em> 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rodriguez <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Green <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pollard and Rockman 2013</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Glater <em>et al.</em> 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Snoek <em>et al.</em> 2014</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Doitsidou <em>et al.</em> 2016</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kamkina <em>et al.</em> 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Singh <em>et al.</em> 2016</a>).</p><p id="p-5">As an alternative, the method known as ethyl methanesulfonate (EMS) density mapping does not require use of the HA strain and relies on the ability to detect signature changes to DNA that are induced by the commonly used mutagen EMS (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Drake and Baltz 1976</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Flibotte <em>et al.</em> 2010</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sarin <em>et al.</em> 2010</a>). After serial backcrossing, strains are subjected to WGS, and genomic regions that contain higher densities of EMS lesions (such as those flanking causal mutations) are identified (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zuryn <em>et al.</em> 2010</a>). Because the density of EMS-induced lesions is relatively low compared with the number of polymorphisms between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and HA, this method may yield substantially lower mapping resolution than the HA method (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Doitsidou <em>et al.</em> 2016</a>).</p><p id="p-6">Another approach that circumvents use of the HA strain, variant discovery mapping (VDM), also relies on variant frequencies to identify chromosomal regions of interest, but uses both EMS and non-EMS variants (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Minevich <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheesman <em>et al.</em> 2016</a>). Thus, VDM may improve mapping resolution relative to that of EMS density mapping, but may provide less information than HA mapping methods (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Doitsidou <em>et al.</em> 2016</a>).</p><p id="p-7">Here, we describe the sibling subtraction method (SSM), which is an alternative approach for WGS analysis that does not require use of the HA strain and does not conceptually rely on variant mapping methods. We applied this strategy to identify suppressors of synthetically lethal <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> double mutants, which arrest as larvae because of defects in molting (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Yochem <em>et al.</em> 2015</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lazetic and Fay 2017</a>). Our results indicate that this method can be used to reduce the number of candidate causal variants to as few as one or two coding change candidates in most cases, thus providing a powerful alternative to current approaches. This study also highlights the utility of using synthetically lethal combinations of weak aphenotypic alleles as a genetic background for suppressor screening, and includes a description of a counterselection approach to increase the efficiency of genetic suppressor screens.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and maintenance</h3><p id="p-8"><em>C. elegans</em> strains were maintained according to standard protocols (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stiernagle 2006</a>) and were propagated at 22° unless otherwise stated. Strains used in this study include <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> Bristol (wild type), WY1145 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em>; <em>fdEx286</em> (pDF153, <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(+)</em>; pTG96, SUR-5::GFP)] (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lazetic and Fay 2017</a>), WY1208 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148740;class=Variation">fd130</a></em>], WY1209 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148738;class=Variation">fd131</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em>], WY1210 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148737;class=Variation">fd132</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em>], WY1211 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148739;class=Variation">fd133</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em>], WY1217 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148751;class=Variation">fd139</a></em>], WY1232 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em>; <em>fdEx286</em>; <em>fdEx297</em> (pTG96.2, SUR-5::RFP)], and WY1255 [<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>; <em>fdEx303</em> (pDF153, <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(+)</em>; pTG96, SUR-5::GFP, PMA122, P<em><sub>hsp16.41</sub>:peel-1</em>)] (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Seidel <em>et al.</em> 2011</a>). For additional strains, see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Suppressor screens</h3><p id="p-9">Suppressors were obtained through an F1 clonal screen following standard EMS mutagenesis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kutscher and Shaham 2014</a>) of strain WY1145. Suppressed worms were detected by their ability to propagate in the absence of the GFP-marked <em>nekl-3</em>–rescuing array, <em>fdEx286</em>, and further confirmed by backcrossing to WY1145. For the <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> counterselection screen, strain WY1255 was mutagenized with EMS and individual P0s were placed on large NGM plates, grown for two generations, and then heat shocked to eliminate array-containing F2s. After 4 d, plates were heat shocked to eliminated rare escapers and plates were then screened 3–4 d later for the presence of propagating (GFP<sup>−</sup>) suppressed worms.</p></div><div id="sec-4" class="subsection"><h3>Genetic analysis of suppressors</h3><p id="p-10">For each backcross, suppressed hermaphrodites were crossed to WY1145 males, GFP<sup>+</sup> F1 cross-progeny hermaphrodites were individually cloned, and GFP<sup>−</sup> F2 suppressed animals were isolated. From these crosses, the frequencies of GFP<sup>−</sup> males were scored in the F1 generation to determine if the suppressors were either dominant or on LGX. To distinguish between dominant mutations and recessive mutations on LGX, we crossed suppressors to WY1232 (GFP<sup>+</sup> RFP<sup>+</sup>) males and scored for the presence of RFP<sup>+</sup> hermaphrodites in the F1 generation. To determine if the dominant mutation <em><a href="http://www.wormbase.org/db/get?name=WBVar02148737;class=Variation">fd132</a></em> (WY1210) was on LGX, WY1210 hermaphrodites were crossed to WY1232 males, and viable RFP<sup>+</sup> F1 males were then crossed to WY1145 hermaphrodites. We then scored for the presence of suppressed F1 males (GFP<sup>−</sup> RFP<sup>−</sup> or GFP<sup>−</sup> RFP<sup>+</sup>). Although we observed &gt;100 GFP<sup>+</sup> RFP<sup>−</sup> and/or GFP<sup>+</sup> RFP<sup>+</sup> males resulting from this cross, we failed to observe any GFP<sup>−</sup> RFP<sup>+</sup> or GFP<sup>−</sup>RFP<sup>−</sup> males, indicating linkage to LGX. Additionally, we picked F1 GFP<sup>+</sup> RFP<sup>+</sup> hermaphrodites singly to plates and scored their F2 progeny. We observed 100% (<em>n</em> = 40) of GFP<sup>+</sup> RFP<sup>+</sup> hermaphrodites to segregate suppressed progeny, further confirming the location of <em><a href="http://www.wormbase.org/db/get?name=WBVar02148737;class=Variation">fd132</a></em> on LGX.</p></div><div id="sec-5" class="subsection"><h3>Preparation of DNA</h3><p id="p-11">After either two (WY1208) or four (WY1209, WY1210, WY1211, and WY1217) backcrosses, suppressed strains were mated to WY1145 males and several F1 heterozygous (GFP<sup>+</sup>) cross-progeny were allowed to self-fertilize. Then, 50–100 F2 self-progeny were subsequently cloned to individual plates to obtain the <em>sup/sup</em> (<em>Sup</em>), <em>sup/+</em>, and <em>+/+</em> (<em>Non-Sup</em>) genotypes. Thus, sequencing analysis was carried on strains that were backcrossed a total of 3× (WY1208) or 5× (WY1209, WY1210, WY1211, and WY1217). DNA was prepared as previously described using the Gentra Puregene Tissue Kit (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Doitsidou <em>et al.</em> 2010</a>). The following numbers of independent isolates were used to generate the DNA pools: WY1208 (<em>Sup</em>, 10; <em>Non-Sup</em>, 15), WY1209 (<em>Sup</em>, 10; <em>Non-Sup</em>, 10), WY1210 (<em>Sup</em>, 10; <em>Non-Sup</em>, 13), WY1211 (<em>Sup</em>, 10; <em>Non-Sup</em>, 5), and WY1217 (<em>Sup</em>, 5; <em>Non-Sup</em>, 8).</p></div><div id="sec-6" class="subsection"><h3>Variant detection</h3><p id="p-12">Paired-end libraries were prepared for each strain and sequenced on an Illumina HiSeq 2000. Resulting reads were analyzed via SSM as described in the <em>Results</em>. Briefly, our workflow was adapted from CloudMap and incorporated tools including Trimmomatic, BWA, SAMtools, Genome Analysis Toolkit (GATK), Generate pileup, VarScan, SnpEff, SnpSift, and Integrative Genomics Viewer implemented on the UseGalaxy.org platform (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li and Durbin 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cingolani <em>et al.</em> 2012a</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">b</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Koboldt <em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Afgan <em>et al.</em> 2016</a>). A detailed description of variant analysis is provided in the Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS1.xlsx">File S1</a>.</p></div><div id="sec-7" class="subsection"><h3>Data Availability</h3><p id="p-13">Reads for all strains used in this study are deposited in NCBI's Sequence Read Archive under BioProject ID PRJNA415825. Code (workflows) used to generate the results in our SSM method are available as <em>SSM Variant Detection</em> and <em>SSM Variant Subtraction</em> and can be accessed through the shared workflows at usegalaxy.org/workflows (owner: bjoseph). Specific software versions and parameters for workflows are provided in Supplementary Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS1.xlsx">File S1</a>.</p></div><div id="sec-8" class="subsection"><h3>RNAi</h3><p id="p-14">dsRNAs corresponding to exons in candidate suppressor genes were prepared using standard PCR methods followed by T7 RNA synthesis (for primer sequences see Supplemental Methods, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS1.xlsx">File S1</a>) and were injected at 0.8–1.0 µg/µl into strain WY1145 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ahringer 2005</a>). GFP<sup>−</sup> F1 progeny were scored, together with noninjected controls, for each experiment. For WY1217, injection of dsRNA corresponding to exon 11 of <a href="http://www.wormbase.org/db/get?name=WBGene00002207;class=Gene">B0302.1</a> (1033 bp fragment) led to 18.6% (<em>n</em> = 1112) adult viability <em>vs.</em> 0.7% (<em>n</em> = 279) in noninjected controls (<em>P</em> &lt; 0.0001). For WY1209, injection of dsRNA corresponding to exons 9–11 of <a href="http://www.wormbase.org/db/get?name=WBGene00018974;class=Gene">F56D12.6a</a> (970 bp fragment) led to 16.5% (<em>n</em> = 1054) adult viability <em>vs.</em> 1.9% (<em>n</em> = 210) in noninjected controls (<em>P</em> &lt; 0.0001).</p></div><div id="sec-9" class="subsection"><h3>CRISPR/Cas9</h3><p id="p-15">CRISPR/Cas9 RNPs were injected into strain WY1145 using <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> co-CRISPR methods (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arribere <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Paix <em>et al.</em> 2014</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2015</a>), and <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000645;class=Phenotype">Rol</a> and <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000583;class=Phenotype">Dpy</a> progeny were monitored for suppression over two generations (for crRNA and other oligonucleotide sequences see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS1.xlsx">File S1</a>). In cases where mutations in the candidate suppressors led to the premature termination of transcripts (<a href="http://www.wormbase.org/db/get?name=WBGene00000074;class=Gene">C04A11.4</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00018974;class=Gene">F56D12.6</a>, and <a href="http://www.wormbase.org/db/get?name=WBGene00002207;class=Gene">B0302.1</a>), we generated breaks within 45 bp of the candidate lesion and allowed nonhomologous repair mechanisms to generate new stop codons or frameshifts. This led to the isolation of three new alleles of <a href="http://www.wormbase.org/db/get?name=WBGene00000074;class=Gene">C04A11.4</a> (<em><a href="http://www.wormbase.org/db/get?name=WBVar02148741;class=Variation">fd208</a></em>, 1 bp deletion; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148742;class=Variation">fd209</a></em>, 2 bp deletion; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148743;class=Variation">fd210</a></em>, 8 bp deletion), two new alleles of <a href="http://www.wormbase.org/db/get?name=WBGene00018974;class=Gene">F56D12.6</a> (<em><a href="http://www.wormbase.org/db/get?name=WBVar02148744;class=Variation">fd211</a></em>, 13 bp deletion; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148745;class=Variation">fd212</a></em>, 10 bp deletion), and three new alleles of <a href="http://www.wormbase.org/db/get?name=WBGene00002207;class=Gene">B0302.1</a> (<em><a href="http://www.wormbase.org/db/get?name=WBVar02148746;class=Variation">fd213</a></em>, 16 bp insertion; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148747;class=Variation">fd214</a></em>, 7 bp deletion; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148748;class=Variation">fd215</a></em>, multiple substitutions and insertions). For specific sequences, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/TableS2.pdf">Table S2</a>. For the essential gene <a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">F48E8.5</a>, we used a repair template to introduce the specific lesion identified by WGS. This led to the isolation of two independent alleles (<em><a href="http://www.wormbase.org/db/get?name=WBVar02148749;class=Variation">fd216</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBVar02148750;class=Variation">fd217</a></em>), both of which contained the identical (G→A) nucleotide substitution and also displayed suppression.</p></div><div id="sec-10" class="subsection"><h3>Transgenic rescue</h3><p id="p-16">Fosmids containing a wild-type copy of the candidate gene were injected with SUR-5::GFP into <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>; <em>sup</em> adults, and GFP<sup>+</sup> F1 progeny were scored for molting defects. In some cases, incomplete rescue (desuppression) led to our ability to score progeny from the F2 generation, although we were unable to obtain stably transmitting lines. For strain WY1208, we injected fosmids corresponding to <a href="http://www.wormbase.org/db/get?name=WBGene00000074;class=Gene">C04A11.4</a> (<a href="http://www.wormbase.org/db/get?name=WRM0620dD12;class=Clone">WRM0620dD12</a>, <a href="http://www.wormbase.org/db/get?name=WRM0632aG02;class=Clone">WRM0632aG02</a>, and <a href="http://www.wormbase.org/db/get?name=WRM0610cA04;class=Clone">WRM0610cA04</a>, 2–6 ng/µl each + SUR-5::GFP [pTG96], 100 ng/µl). Suppression by <em><a href="http://www.wormbase.org/db/get?name=WBVar02148740;class=Variation">fd130</a></em> in WY1208 led to 83% viability (17% arrest; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Following injection of WY1208, 80% of GFP<sup>+</sup> F1 larvae (<em>n</em> = 25; <em>P</em> &lt; 0.0001) arrested with molting defects, and a similar trend was observed in the F2 progeny from several viable transmitting F1s (83% arrest, <em>n</em> = 37; <em>P</em> &lt; 0.0001). In contrast, injection of this fosmid mix into wild type showed no deleterious effects on larval development or adult viability. WY1217 was injected with fosmids corresponding to candidate <a href="http://www.wormbase.org/db/get?name=WBGene00002207;class=Gene">B0302.1</a> (<a href="http://www.wormbase.org/db/get?name=WRM0610cD03;class=Clone">WRM061cD03</a> and <a href="http://www.wormbase.org/db/get?name=WRM0612dE01;class=Clone">WRM0612dE01</a>, 6 ng/µl each + SUR-5::GFP [pTG96], 100 ng/µl). Whereas the <em><a href="http://www.wormbase.org/db/get?name=WBVar02148751;class=Variation">fd139</a></em> mutation led to 76% viability in this strain (24% arrest), we observed arrest in 93% of GFP<sup>+</sup> F1s (<em>n</em> = 30; <em>P</em> &lt; 0.0001). For strain WY1211 (<em><a href="http://www.wormbase.org/db/get?name=WBVar02148739;class=Variation">fd133</a></em>) we injected a mix of fosmids corresponding to <a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">F48E8.5</a> and an RFP marker (<a href="http://www.wormbase.org/db/get?name=WRM0618bG08;class=Clone">WRM0618bG08</a>, <a href="http://www.wormbase.org/db/get?name=WRM064bA06;class=Clone">WRM064bA06</a>, and <a href="http://www.wormbase.org/db/get?name=WRM0618aG08;class=Clone">WRM0618aG08</a>, ∼6 ng/µl each + SUR-5::RFP [pTG96.2], 100 ng/µl) into WY1211 worms that also carried the <em>fdEx286 <a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em><sup>+</sup> GFP<sup>+</sup> rescuing array. Three stably transmitting GFP<sup>+</sup> RFP<sup>+</sup> lines were then for scored viability in the F3 generation. Arrest among the GFP<sup>−</sup> RFP<sup>+</sup> animals occurred at frequencies of 92% (<em>n</em> = 210), 95% (<em>n</em> = 128), and 97% (<em>n</em> = 181) for the three lines tested, respectively, whereas arrest among GFP<sup>−</sup> RFP<sup>−</sup> worms was 63% (<em>n</em> = 223; <em>P</em> &lt; 0.0001). Furthermore, &lt;10% of GFP<sup>+</sup> RFP<sup>+</sup> progeny underwent arrest (<em>n</em> &gt; 200 for each strain), indicating that the arrays were not highly toxic. <em>P</em> values were determined using the N-1 chi-squared test.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488543/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488543/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Strains used for sequencing analysis</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>An F1 clonal screen to identify suppressors of nekl-2; nekl-3 synthetic lethality</h3><p id="p-18">To better understand the functions of Nek family kinases during development, we sought to identify extragenic suppressors of molting defects in <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> mutants. Previous studies had identified an allelic series of <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> mutations, including null alleles that arrest at the L1/L2 molt, moderate loss-of-function alleles that arrest as L2/L3s, and weak alleles that are aphenotypic (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Yochem <em>et al.</em> 2015</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Lazetic and Fay 2017</a>). Furthermore, certain combinations of weak alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> lead to double mutants that display penetrant molting defects and concomitant larval arrest. In the case of strain WY1145, synthetically lethal <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em> double mutants are maintained by the presence of a rescuing extrachromosomal array (<em>fdEx286</em>) that is transmitted to ∼65% of self-progeny, and expresses wild-type <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> along with a fluorescent reporter (SUR-5::GFP) (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Lazetic and Fay 2017</a>). Thus, in this study, WY1145 hermaphrodites gave rise to viable array-containing GFP<sup>+</sup> progeny as well as array-negative GFP<sup>−</sup> progeny, 98.8% (<em>n</em> = 444) of which arrested as larvae with molting defects (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Rare GFP<sup>−</sup> escapers that reached adulthood produced progeny that arrested nearly uniformly with molting defects (98.5%, <em>n</em> = 1721).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/669/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypes of molting-defective and suppressed strains. (A, C, and E) DIC and (B, D, and F) fluorescence images of strains WY1145 [nekl-2(fd81); nekl-3(gk894345)] and WY1211 [nekl-2(fd81); paa-1(fd134); nekl-3(gk894345)]. GFP+ worms carry an extrachromosomal array (fdEx286) that expresses wild-type nekl-3 and SUR-5::GFP. (A and B) Whereas GFP&#x2212; worms (white arrows) in the starting strain, WY1145, arrest uniformly with molting defects. (C and D) GFP&#x2212; worms in the suppressed strain, WY1211, can reach adulthood. (E and F) Suppression is reverted in strain WY1211 by the expression of wild-type paa-1 from a fosmid, which is carried by an independent extrachromosomal array marked with SUR-5::RFP. Compare GFP&#x2212; RFP+ arrested larva (E and F) with GFP&#x2212; gravid adults (C and D). Also note that the paa-1+ RFP-marked array is not generically deleterious for growth as evidenced by the viability of GFP+ RFP+ adults (E and F). White arrows indicate GFP&#x2212; worms; white arrowheads indicate GFP+ worms; white boxes in (E) and (F) indicate that the image was acquired from a region outside the main panel; yellow boxes indicate regions of increased magnification. Black scale bar in (A) (for A&#x2013;F), 100 &#xB5;m; white scale bars in insets (A and E), 20 &#xB5;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-54046974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotypes of molting-defective and suppressed strains. (A, C, and E) DIC and (B, D, and F) fluorescence images of strains WY1145 [&lt;em&gt;nekl-2(fd81)&lt;/em&gt;; &lt;em&gt;nekl-3(gk894345)&lt;/em&gt;] and WY1211 [&lt;em&gt;nekl-2(fd81)&lt;/em&gt;; &lt;em&gt;paa-1(fd134)&lt;/em&gt;; &lt;em&gt;nekl-3(gk894345)&lt;/em&gt;]. GFP&lt;sup&gt;+&lt;/sup&gt; worms carry an extrachromosomal array (&lt;em&gt;fdEx286&lt;/em&gt;) that expresses wild-type &lt;em&gt;nekl-3&lt;/em&gt; and SUR-5::GFP. (A and B) Whereas GFP&lt;sup&gt;&#x2212;&lt;/sup&gt; worms (white arrows) in the starting strain, WY1145, arrest uniformly with molting defects. (C and D) GFP&lt;sup&gt;&#x2212;&lt;/sup&gt; worms in the suppressed strain, WY1211, can reach adulthood. (E and F) Suppression is reverted in strain WY1211 by the expression of wild-type &lt;em&gt;paa-1&lt;/em&gt; from a fosmid, which is carried by an independent extrachromosomal array marked with SUR-5::RFP. Compare GFP&lt;sup&gt;&#x2212;&lt;/sup&gt; RFP&lt;sup&gt;+&lt;/sup&gt; arrested larva (E and F) with GFP&lt;sup&gt;&#x2212;&lt;/sup&gt; gravid adults (C and D). Also note that the &lt;em&gt;paa-1&lt;/em&gt;&lt;sup&gt;+&lt;/sup&gt; RFP-marked array is not generically deleterious for growth as evidenced by the viability of GFP&lt;sup&gt;+&lt;/sup&gt; RFP&lt;sup&gt;+&lt;/sup&gt; adults (E and F). White arrows indicate GFP&lt;sup&gt;&#x2212;&lt;/sup&gt; worms; white arrowheads indicate GFP&lt;sup&gt;+&lt;/sup&gt; worms; white boxes in (E) and (F) indicate that the image was acquired from a region outside the main panel; yellow boxes indicate regions of increased magnification. Black scale bar in (A) (for A&#x2013;F), 100 &#xB5;m; white scale bars in insets (A and E), 20 &#xB5;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/669/F1.medium.gif" width="370" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/669/F1.medium.gif" width="370" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/669/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/669/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488414" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Phenotypes of molting-defective and suppressed strains. (A, C, and E) DIC and (B, D, and F) fluorescence images of strains WY1145 [<em>nekl-2(fd81)</em>; <em>nekl-3(gk894345)</em>] and WY1211 [<em>nekl-2(fd81)</em>; <em>paa-1(fd134)</em>; <em>nekl-3(gk894345)</em>]. GFP<sup>+</sup> worms carry an extrachromosomal array (<em>fdEx286</em>) that expresses wild-type <em>nekl-3</em> and SUR-5::GFP. (A and B) Whereas GFP<sup>−</sup> worms (white arrows) in the starting strain, WY1145, arrest uniformly with molting defects. (C and D) GFP<sup>−</sup> worms in the suppressed strain, WY1211, can reach adulthood. (E and F) Suppression is reverted in strain WY1211 by the expression of wild-type <em>paa-1</em> from a fosmid, which is carried by an independent extrachromosomal array marked with SUR-5::RFP. Compare GFP<sup>−</sup> RFP<sup>+</sup> arrested larva (E and F) with GFP<sup>−</sup> gravid adults (C and D). Also note that the <em>paa-1</em><sup>+</sup> RFP-marked array is not generically deleterious for growth as evidenced by the viability of GFP<sup>+</sup> RFP<sup>+</sup> adults (E and F). White arrows indicate GFP<sup>−</sup> worms; white arrowheads indicate GFP<sup>+</sup> worms; white boxes in (E) and (F) indicate that the image was acquired from a region outside the main panel; yellow boxes indicate regions of increased magnification. Black scale bar in (A) (for A–F), 100 µm; white scale bars in insets (A and E), 20 µm.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">We reasoned that WY1145 could be effective for identifying extragenic suppressors because both <em><a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a></em> are very weak loss-of-function alleles and thus are potentially amenable to genetic suppression, and by carrying out the screen with double mutants, we could obtain suppressors that are specific to either <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em> or <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>, as well as suppressors of both. As a first approach, we mutagenized WY1145 with EMS and carried out a semiclonal F1 screen (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS1.jpg">Figure S1</a>). Plates containing candidate suppressor mutations were initially identified based on the increased frequency of F2 GFP<sup>−</sup> animals reaching adulthood and were subsequently verified by propagating strains derived from isolated F2 GFP<sup>−</sup> animals for multiple generations. From this screen of ∼8000 haploid genomes, 27 independent isolates were obtained with suppression frequencies ranging from ∼20% to nearly 90% (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/TableS1.pdf">Table S1</a>). Notably, a pilot screen using a moderate loss-of-function allele, <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar02147302;class=Variation">sv3</a></em>), alone failed to uncover strong suppressors (data not shown), suggesting that approaches using synthetic-lethal combinations of weak alleles may be particularly productive.</p></div><div id="sec-13" class="subsection"><h3>Use of the peel-1 counterselectable marker to aid suppressor screening</h3><p id="p-21">To reduce the labor associated with F1 clonal/semiclonal screening methods, we considered alternative strategies. Because the frequency of naturally occurring GFP<sup>−</sup> escapers reaching adulthood in WY1145 (∼1–2%) was likely to be much higher than the frequency of authentic suppressors, standard nonclonal approaches were deemed impractical because of the high incidence of false positives. We therefore considered a strategy involving a counterselectable marker to eliminate animals carrying the <em>nekl-3<sup>+</sup></em>–rescuing array. <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> is a sperm-derived toxin that, when overexpressed in <em>C. elegans</em> larvae and adults, leads to death at all postembryonic stages (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Seidel <em>et al.</em> 2008</a>, <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">2011</a>). We engineered a <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em> double-mutant strain (WY1255) with a rescuing array containing wild-type <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>, SUR-5::GFP, and a heat shock–inducible <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> transgene. After incubation of WY1255 at 34° for 2 hr, nearly 100% of array-containing GFP<sup>+</sup> worms perished within 24 hr. As outlined in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS2.jpg">Figure S2</a>, WY1255 worms were mutagenized, and 100 individual P0 animals were picked to large plates and incubated at 22° until F2 progeny began to emerge (∼5–6 d). Plates were then heat shocked to kill array-containing animals and were monitored for an additional 7–8 d to identify plates with actively propagating populations of GFP<sup>−</sup> animals. To prevent the re-emergence of any surviving GFP<sup>+</sup> worms, plates were heat shocked again after 4 d. From this screen we identified an additional 23 suppressors, demonstrating the utility of this counterselection approach (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/TableS1.pdf">Table S1</a>). In particular, this method should be useful for identifying suppressors of phenotypes that are &lt;100% penetrant, but it can also be applied to phenotypes that are fully penetrant.</p></div><div id="sec-14" class="subsection"><h3>Development of an SSM to identify suppressors using WGS</h3><p id="p-22">Our combined genetic screens identified ∼50 mutants that displayed a wide range of suppressor strengths and included both dominant and recessive alleles (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, C and D</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/TableS1.pdf">Table S1</a>, and data not shown). Suppressed strains (<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>; <em>sup</em>) were backcrossed to WY1145 and, after preliminary genetic characterization, five strains were selected for further analysis by WGS (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). In addition, genetic studies determined that three of these five suppressors were on LGX, whereas two were autosomal (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and <em>Materials and Methods</em>). The molecular identification of the suppressors, however, presented several technical challenges given that the suppressor mutations did not exhibit obvious phenotypes on their own. In addition, the <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a></em>) and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>(<em><a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a></em>) mutations are aphenotypic and show defects only when combined as double mutants (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Lazetic and Fay 2017</a>). Although it was possible to introduce both the <em><a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a></em> mutations into the HA mapping strain (CB4846) using CRISPR/Cas9 methods, as discussed at the beginning of this article, this divergent background has the potential to alter the expression or penetrance of phenotypes. Furthermore, it was unclear if methods that did not make use of the HA strain would provide sufficient mapping resolution of the affected loci to consistently enable facile identification of causal mutations (also see below) (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Doitsidou <em>et al.</em> 2016</a>).</p><p id="p-23">We therefore devised a WGS strategy to identify <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> suppressors that did not depend on either variant density mapping or the use of <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>. We refer to our approach as the SSM, and a generic version of this strategy is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Importantly, this strategy can be applied to a variety of phenotypes as well as both simple and complex genetic backgrounds, including mutations that lead to enhancement or suppression. To carry out this approach, <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02147307;class=Variation">fd81</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar01194373;class=Variation">gk894345</a>)</em> suppressed strains (<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>; <em>sup</em>) were crossed to WY1145 to generate heterozygous (<em>sup/+</em>) GFP<sup>+</sup> F1 cross-progeny, which were picked to individual plates and allowed to self-fertilize (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS3.jpg">Figure S3</a>). Next, sibling GFP<sup>+</sup> F2 progeny from a single F1 parent were picked to individual plates and allowed to produce the F3 generation. Based on standard Mendelian inheritance patterns, a 1:2:1 ratio corresponding to <em>sup/sup</em>, <em>sup/+</em>, and <em>+/+</em> would be expected among the F2 generation. Plates containing F2 parents of the <em>sup/sup</em> genotype were recognized based on the high frequency of viable GFP<sup>−</sup> adult F3s. Moreover, viable GFP<sup>−</sup> F3 worms from candidate <em>sup/sup</em> plates were picked to new plates and allowed to propagate to ensure suppressor homozygosity. Conversely, plates containing parental F2s of the <em>+/+</em> genotype were identified by the absence of suppressed (adult GFP<sup>−</sup>) F3 animals (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS3.jpg">Figure S3</a>). For each suppressor strain, individual <em>sup/sup</em> isolates were combined to make genomic DNA, which we refer to as the “suppressed” DNA pool (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS3.jpg">Figure S3</a>). Corresponding <em>+/+</em> isolates were also combined to generate a “nonsuppressed” DNA pool (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS3.jpg">Figure S3</a>). We note that the number of independent <em>sup/sup</em> or <em>+/+</em> isolates used to generate the DNA pools ranged from 5 to 15, indicating that as few as five isolates may be sufficient for our approach to be successful.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/669/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic design of the SSM. (A) The genetic crosses required for generating multiple independent m/m and sibling +/+ isolates, which are combined to produce the mutant DNA pool and the nonmutant sibling comparator DNA pool, respectively (also see Figure S3). (B) Simplified representations of sequenced chromosomes from the mutant pool (pink) and nonmutant sibling comparator pool (blue) are shown. Yellow lines indicate variants detected by WGS, and asterisks indicate variants that are homozygous in the mutant pool. Note that the depiction underestimates the true number of variants per chromosome. Dashed lines and purple asterisks indicate variants that are present in both the mutant pool and the nonmutant sibling comparator pool, which can be eliminated (subtracted) as candidate causal mutations. Note that subtraction will remove most or all variants on unlinked chromosomes, whereas homozygous variants very close to the causal mutation (red) may not be subtracted (black asterisks). (C) Venn diagram of subtracted variants (purple) along with the relatively small proportion of remaining candidate variants (pink) after application of the SSM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-54046974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic design of the SSM. (A) The genetic crosses required for generating multiple independent &lt;em&gt;m/m&lt;/em&gt; and sibling +/+ isolates, which are combined to produce the mutant DNA pool and the nonmutant sibling comparator DNA pool, respectively (also see Figure S3). (B) Simplified representations of sequenced chromosomes from the mutant pool (pink) and nonmutant sibling comparator pool (blue) are shown. Yellow lines indicate variants detected by WGS, and asterisks indicate variants that are homozygous in the mutant pool. Note that the depiction underestimates the true number of variants per chromosome. Dashed lines and purple asterisks indicate variants that are present in both the mutant pool and the nonmutant sibling comparator pool, which can be eliminated (subtracted) as candidate causal mutations. Note that subtraction will remove most or all variants on unlinked chromosomes, whereas homozygous variants very close to the causal mutation (red) may not be subtracted (black asterisks). (C) Venn diagram of subtracted variants (purple) along with the relatively small proportion of remaining candidate variants (pink) after application of the SSM.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/669/F2.medium.gif" width="427" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/669/F2.medium.gif" width="427" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/669/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/669/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488410" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Schematic design of the SSM. (A) The genetic crosses required for generating multiple independent <em>m/m</em> and sibling +/+ isolates, which are combined to produce the mutant DNA pool and the nonmutant sibling comparator DNA pool, respectively (also see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS3.jpg">Figure S3</a>). (B) Simplified representations of sequenced chromosomes from the mutant pool (pink) and nonmutant sibling comparator pool (blue) are shown. Yellow lines indicate variants detected by WGS, and asterisks indicate variants that are homozygous in the mutant pool. Note that the depiction underestimates the true number of variants per chromosome. Dashed lines and purple asterisks indicate variants that are present in both the mutant pool and the nonmutant sibling comparator pool, which can be eliminated (subtracted) as candidate causal mutations. Note that subtraction will remove most or all variants on unlinked chromosomes, whereas homozygous variants very close to the causal mutation (red) may not be subtracted (black asterisks). (C) Venn diagram of subtracted variants (purple) along with the relatively small proportion of remaining candidate variants (pink) after application of the SSM.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Bioinformatic workflow and parameters</h3><p id="p-25">The workflow for the experimental and bioinformatic analysis is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and is described in detail in the Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS1.xlsx">File S1</a>. For our SSM analysis, we tested four different filtering criteria for each of the suppressor strains. This allowed us to determine how different parameters would impact the identification of candidate variants. In our most stringent version (filtering analysis 1; FA1), we required that variants from the suppressed DNA pools be present in 100% of reads, whereas variants from the nonsuppressed sibling DNA pools need only be present in &gt;0% of reads. These parameters therefore minimized the number of candidate variants from the suppressed pools while maximizing the number of variants from the nonsuppressed pools used for subtraction. We predicted that the large majority of strain-specific noncausal mutations acquired before or after mutagenesis would therefore be subtracted, leaving a manageable number of candidates requiring experimental validation (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, B and C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/669/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SSM workflow. Overview of the experimental and bioinformatical steps involved in the SSM/WGS method. For additional details, see text and Supplemental Methods, File S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-54046974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SSM workflow. Overview of the experimental and bioinformatical steps involved in the SSM/WGS method. For additional details, see text and Supplemental Methods, File S1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/669/F3.medium.gif" width="166" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/2/669/F3.medium.gif" width="166" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/669/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/669/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488384" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">SSM workflow. Overview of the experimental and bioinformatical steps involved in the SSM/WGS method. For additional details, see text and Supplemental Methods, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS1.xlsx">File S1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">In practice, we found that FA1 eliminated 98.4% (range, 98.0–99.0%) of all variants detected in the five suppressed DNA pools (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Likewise, when considering variants leading to changes in protein structure (nonsense, missense, frameshift, and splice site mutations), an average of 98.8% (range, 98.2–100%) of variants were subtracted. Manual examination of candidate variants following computational subtraction led to an additional reduction in the total number or candidate variants for all five strains from 13 to 10 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS2.xlsx">File S2</a>). In two of these cases, the subtraction of a common variant failed because the required number of nonsuppressed sibling reads fell below our set computational threshold of three. Nevertheless, an examination of the two high-quality reads available from each strain indicated that the variants were present in both suppressed and nonsuppressed strains and thus could be eliminated as causal.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488517/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488517/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of SSM/WGS data</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Although effective in limiting the number of candidate causal variants, it was possible that our most stringent parameters (FA1) could, however, lead to the occurrence of false negatives. Consistent with this possibility, FA1 failed to identify coding change candidates for strain WY1217, suggesting that at least one true positive may have been eliminated in our analysis (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). One source of false negatives could be due to sequencing errors or misalignments, which could lead to a failure to call an authentic variant in the suppressed DNA pool. False negatives could also result if one of the worm isolates used for DNA sequencing was incorrectly assigned at the level of phenotype. For example, it is possible that DNA pools for dominant alleles could contain animals that were heterozygous, thus reducing the percentage of causal variant reads in the suppressed DNA pool to &lt;100%. Conversely, in the case of weakly penetrant recessive alleles, heterozygous strains could be accidentally assigned to the nonsuppressed pool, thereby increasing the percentage of causal variant reads to &gt;0% and leading to their subtraction. Finally, regardless of the above filtering criteria, a false negative would occur if the causal mutation were to be located in a noncoding region (also see below).</p><p id="p-32">We therefore analyzed sequencing data from all our strains using three additional filtering criteria (FA2–4). In the case of FA2, we reduced the requirement for variant calls from the suppressed DNA pools from 100 to ≥90%, while maintaining the requirement for nonsuppressed variant reads at &gt;0% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS2.xlsx">File S2</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Compared to FA1, this increased the number of called variants in the suppressed DNA pools by an average of 31.0% (range, 23.7–37.1%), increased the average number of variants after subtraction by 18.7% (range, 7.1–52.6%), and raised the total number of coding-change variants (for all five strains) after subtraction from 13 to 20. Notably, FA2 parameters led to the identification of a single coding-change candidate for strain WY1217 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS2.xlsx">File S2</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Further examination of this candidate revealed that a single read from the suppressed DNA pool had been improperly aligned with BWA, leading to its omission in FA1.</p><p id="p-33">For FA3, we required variants from the suppressed DNA pools to be present in 100% of reads but mandated that variants from the nonsuppressed DNA pools be present at ≥10%, thereby reducing the number of variants used for subtraction by an average of 87.8% (range, 75.0–92.6%). This led to an average increase in the number of candidate variants following subtraction by 8.1% (range, 2.8–22.6%) relative to FA1, and increased the total number of coding-change variants from 13 to 15 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS2.xlsx">File S2</a> and <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Finally, by requiring variants to be present in ≥90% of reads in the suppressed DNA pools and ≥10% in the nonsuppressed DNA pools (FA4), we observed an average increase over FA1 in the number of variants following subtraction by 27.6% (range, 21.6–64.5%) along with an increase in total coding-change variants from 13 to 22. We note that even under these more liberal filtering criteria, an average of 98.2% of variants were subtracted using SSM. In summary, by using less stringent filtering criteria (FA2–FA4), we observed a modest increase in the number of candidate causal variants and, in the case of FA2 and FA4, identified a single coding-change candidate for strain WY1217.</p></div><div id="sec-16" class="subsection"><h3>Experimental validation of variants identified by the SSM</h3><p id="p-34">Having identified candidate casual variants, we next sought to validate the identities of all four recessive causal mutations using a combination of RNAi, transgenic rescue experiments, and CRISPR/Cas9. In the case of strains WY1208, WY1209, and WY1211, we focused on candidates obtained using FA1, whereas for strain WY1217, FA2 criteria were used (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FileS2.xlsx">File S2</a> and <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). A summary of the findings and methods used for validation is shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. One example is strain WY1211 (<em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBVar02148752;class=Variation">fd134</a></em>), which exhibits 34% suppression (<em>n</em> = 199), making it the weakest of the suppressors subjected to SSM/WGS (compare <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, A and B</a> to <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, C and D</a>). After subtraction using the most stringent parameters (FA1), only two candidate variants remained that affected coding regions, corresponding to missense mutations in <a href="http://www.wormbase.org/db/get?name=WBGene00043156;class=Gene">C27F2.9</a> and <a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">F48E8.5</a>/<em><a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">paa-1</a></em> (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). Notably, both variants reside within a 0.6-map unit-long region on the left arm of LGIII. <a href="http://www.wormbase.org/db/get?name=WBGene00043156;class=Gene">C27F2.9</a> encodes a protein with homology to JMJD8, and the detected variant leads to a R220C substitution in a residue that is not highly conserved among closely related species. <em><a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">paa-1</a></em> encodes an essential gene that is orthologous to the PR65 structural subunit of the mammalian PP2A protein phosphatase. Moreover, the identified variant in <em><a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">paa-1</a></em> causes a G550E substitution in a glycine residue that is highly conserved throughout metazoans (data not shown). Two lines of evidence demonstrated that the <em><a href="http://www.wormbase.org/db/get?name=WBVar02148752;class=Variation">fd134</a></em> causal mutation corresponds to the lesion in <em><a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">paa-1</a></em>. First, we recreated the identical (G→A) mutation using CRISPR/Cas9 methods in the WY1145 background and obtained two independent suppressed lines. Second, we injected fosmids encoding the wild-type <em><a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">paa-1</a></em> locus into WY1211 and observed clear desuppression in three of three lines (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, E and F</a>). Our identification of <em><a href="http://www.wormbase.org/db/get?name=WBGene00003901;class=Gene">paa-1</a></em> as a suppressor of <em><a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">nekl-2</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00008956;class=Gene">nekl-3</a></em> defects suggests that the <em>C. elegans</em> PP2A complex may oppose <a href="http://www.wormbase.org/db/get?name=WBGene00022631;class=Gene">NEKL-2</a>/3 kinase activity, possibly by dephosphorylating NEKL2/3 targets. Likewise the causal mutations for strains WY1208, WY1209, and WY1217 were successfully identified using a minimum of two independent methods (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488539/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488539/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Confirmation of recessive suppressor mutations</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">In the case of the dominant suppressor, <em><a href="http://www.wormbase.org/db/get?name=WBVar02148737;class=Variation">fd132</a></em> (WY1210), FA1 criteria identified coding variants in two genes, <a href="http://www.wormbase.org/db/get?name=WBGene00011645;class=Gene">T09B9.4</a> (amino acid substitution M399I) and <a href="http://www.wormbase.org/db/get?name=WBGene00012326;class=Gene">W07E11.1</a> (amino acid substitution P1181L). Notably, both genes are located on LGX, which genetic mapping had shown to harbor the causal mutation, and are positioned within ∼300,000 bp of each other (∼0.25 map units). Moreover, both genes reside within a 1.7 Mb region that contains 13 consecutive variants that are present in 100% of suppressed DNA reads and 0% of the nonsuppressed. Genomic engineering of the <a href="http://www.wormbase.org/db/get?name=WBGene00011645;class=Gene">T09B9.4</a> and <a href="http://www.wormbase.org/db/get?name=WBGene00012326;class=Gene">W07E11.1</a> mutations using CRISPR/Cas9, however, did not lead to genetic suppression, indicating that neither mutation likely corresponds to the causative change in this strain. Consistent with this, RNAi of both genes failed to revert suppression of WY1210, which could be expected if <em><a href="http://www.wormbase.org/db/get?name=WBVar02148737;class=Variation">fd132</a></em> was a gain-of-function allele, or to induce suppression in strain WY1145, which might be anticipated if <em><a href="http://www.wormbase.org/db/get?name=WBVar02148737;class=Variation">fd132</a></em> was a dominant negative allele. We next examined coding variants in three additional genes (<a href="http://www.wormbase.org/db/get?name=WBGene00018730;class=Gene">F53A9.7</a>, <a href="http://www.wormbase.org/db/get?name=WBGene00004862;class=Gene">T05A10.1</a>, and <a href="http://www.wormbase.org/db/get?name=WBGene00002057;class=Gene">R04E5.10</a>) that were detected using FA2/4 criteria (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>). All three genes are on LGX and closely flank the 1.7 Mb region containing <a href="http://www.wormbase.org/db/get?name=WBGene00011645;class=Gene">T09B9.4</a> and <a href="http://www.wormbase.org/db/get?name=WBGene00012326;class=Gene">W07E11.1</a>. However, precise editing of these genes using CRISPR/Cas9 failed to produce genetic suppression. Thus, the <em><a href="http://www.wormbase.org/db/get?name=WBVar02148737;class=Variation">fd132</a></em> causal mutation may reside in a noncoding region, such as an element that promotes transcriptional or posttranscriptional repression, or it is possible WGS/SSM may have failed to identify the causal coding variant. However, we note that the 1.7 Mb region, which also corresponds to the region implicated by EMS density mapping (see below), does not contain any sequencing gaps within coding regions, further implicating noncoding elements.</p></div><div id="sec-17" class="subsection"><h3>Comparison of SSM and EMS density mapping methods</h3><p id="p-40">We next compared our subtraction approach to the EMS density mapping method, which identifies regions of the genome harboring high frequencies of EMS signature variants that are maintained following serial backcrossing (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Zuryn <em>et al.</em> 2010</a>). Importantly, use of SSM does not preclude EMS density mapping methods and these methods can be viewed as complementary. We first obtained EMS density maps for strains WY1208, WY1209, WY1210, WY1211, and WY1217 using the published CloudMap workflow, which is available on the Galaxy web platform (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Minevich <em>et al.</em> 2012</a>). In each case, we subtracted called variants that were present in any of the other suppressed strains. Although a subset of the major peaks generated by EMS density mapping were consistent with the locations of our identified causal mutations; in most cases, this analysis failed to provide unambiguous localization of our causal mutations to either chromosomes or to chromosomal subregions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS4.pdf">Figure S4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/669/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="EMS density mapping comparison in WY1208. EMS density mapping for the five suppressor strains was carried out using the CloudMap workflow on Galaxy (A) or using our variant identification workflow based on allele frequencies (B). The x axis indicates the location on each chromosome in megabases (Mb). The number of EMS signature SNPs are indicated on the y axis; red bars indicate a 0.5-Mb region, and gray bars indicate a 1.0-Mb region. Black arrows indicate the location of the identified causal mutations, and gray arrows indicate noncausal mutations identified by SSM. The blue bar in (B) indicates the implicated region on LGX that contained five coding-change candidates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-54046974" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;EMS density mapping comparison in WY1208. EMS density mapping for the five suppressor strains was carried out using the CloudMap workflow on Galaxy (A) or using our variant identification workflow based on allele frequencies (B). The &lt;em&gt;x&lt;/em&gt; axis indicates the location on each chromosome in megabases (Mb). The number of EMS signature SNPs are indicated on the &lt;em&gt;y&lt;/em&gt; axis; red bars indicate a 0.5-Mb region, and gray bars indicate a 1.0-Mb region. Black arrows indicate the location of the identified causal mutations, and gray arrows indicate noncausal mutations identified by SSM. The blue bar in (B) indicates the implicated region on LGX that contained five coding-change candidates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/669/F4.medium.gif" width="388" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/2/669/F4.medium.gif" width="388" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/669/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/669/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488338" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">EMS density mapping comparison in WY1208. EMS density mapping for the five suppressor strains was carried out using the CloudMap workflow on Galaxy (A) or using our variant identification workflow based on allele frequencies (B). The <em>x</em> axis indicates the location on each chromosome in megabases (Mb). The number of EMS signature SNPs are indicated on the <em>y</em> axis; red bars indicate a 0.5-Mb region, and gray bars indicate a 1.0-Mb region. Black arrows indicate the location of the identified causal mutations, and gray arrows indicate noncausal mutations identified by SSM. The blue bar in (B) indicates the implicated region on LGX that contained five coding-change candidates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Because the EMS density mapping workflow on CloudMap uses Bayesian methods to call variants, which we found to lead to some false positives in our analysis, we were next interested to perform EMS density mapping using our workflow, which bases variant calls on absolute allele frequencies (<em>e.g.</em>, &gt;0–100%). Once again, we subtracted variants that were present in any of the other suppressed strains before generating density maps. Notably, this approach greatly reduced the total number of EMS variants and dramatically improved the clarity of the mapping method (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS5.pdf">Figure S5</a>). For example, in the case of WY1208 this resolved the chromosomal location of <em><a href="http://www.wormbase.org/db/get?name=WBVar02148740;class=Variation">fd130</a></em> to a region on LGX (5–14 Mb) that contained five homozygous coding-change candidates, including <a href="http://www.wormbase.org/db/get?name=WBGene00000074;class=Gene">C04A11.4</a>. Likewise, candidates for WY1209 (4), WY1210 (3), and WY1211 (2) were identified, although in the case of WY1217 no coding-change candidates were identified under the major peak on LGX (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS5.pdf">Figure S5</a>).</p><p id="p-43">When carrying out EMS density mapping, it is theoretically advantageous to sequence multiple mutant strains derived from the same parent as this increases the number of background variants that can be identified and subtracted before generating density maps. To test this directly, we generated EMS density maps for WY1208 using WY1211 only as our subtraction strain. Notably, this led to considerably poorer resolution of the causal candidate to either a chromosome or chromosomal subregion; eight coding-change candidates were implicated under the major peak on LGX (2.5–14 Mb) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS6.pdf">Figure S6</a>). In summary, when applying our variant identification workflow, SSM generally outperformed EMS density mapping and may be particularly advantageous in situations where a limited number of strains are to be sequenced. However, additional studies will be necessary to determine the relative advantages or disadvantages of SSM and other available methods.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-44">We have demonstrated SSM/WGS to be a simple and effective approach for identifying causal mutations in <em>C. elegans</em>. Importantly, this approach does not require the use of polymorphic strains (<em>e.g.</em>, <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>) and does not rely on variant mapping methods. In addition, SSM should be generally applicable to other animal systems. SSM can be made flexible in its use of different filtering parameters (FA1–4; <a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>), which can allow for a progressive interrogation in the search for causal candidate variants.</p><p id="p-45">Based on our study, SSM appears to provide somewhat better resolution of candidates than EMS density mapping methods alone, particularly in cases where fewer total strains are subjected to sequencing (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS5.pdf">Figure S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1/FigureS6.pdf">Figure S6</a>). We note that although the majority of published studies using EMS density mapping do not directly state the number of causal candidates obtained using this approach, Zuryn and colleagues reported two, four, and nine candidates for the three test strains examined (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Zuryn <em>et al.</em> 2010</a>). Importantly, EMS density mapping can be used in a complementary fashion with SSM, and altering the published workflow (<em>e.g.</em>, calling variants based on absolute counts rather than probabilities) may provide increased resolution over the current version available through CloudMap, as implemented on Galaxy.</p><p id="p-46">One drawback to SSM is the additional costs of having to sequence both mutant and nonmutant sibling comparator samples. This, however, must be balanced by the potential for reducing the number of causal candidates, which can be time consuming and expensive to validate. Another drawback of SSM relative to high-density mapping methods, such as the use of <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> or VDM, is that noncoding causal variants may be more difficult to identify using SSM. This is because the reduced mapping resolution provides less guidance regarding which variants should be prioritized for initial validation tests. Finally, we note that our average coverage for the five genomes was 20–30×, and it is possible that a somewhat higher coverage level would provide better information. Nevertheless, our current coverage did allow for the identification of four out of four recessive alleles from our screen. Our approach did not allow for a direct comparison between SSM and “bulked segregant” VDM methods; future parallel assessments of these methods may prove useful.</p><p id="p-47">In addition, we showed that a counterselection method using an extrachromosomal array encoding the inducible toxin <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em> can provide a highly efficient means for identifying genetic suppressors, especially in cases of incomplete penetrance of the starting strain. Our studies also underscore the utility of using weak alleles in synthetic-lethal combinations as starting strains for suppressor screens. In particular, recent advances in genomic engineering (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Arribere <em>et al.</em> 2014</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Paix <em>et al.</em> 2014</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">2015</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dickinson and Goldstein 2016</a>), as well as the Million Mutation resource (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Thompson <em>et al.</em> 2013</a>), make the identification of weak alleles and combinatorial-dependent phenotypes highly feasible. Finally, because SSM/WGS is not affected by genetic background, it should be possible to identify genetic modifiers of complex genotypes in a straightforward manner.</p></div><div class="section" id="sec-19"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank Amy Fluet for editing. Some strains were provided by the Caenorhabditis Genetics Center, which is funded by National Institutes of Health (NIH) Office of Research Infrastructure Programs (P40 OD010440). This work was supported by NIH grants GM066868 (to D.S.F.), P20 GM103432 (Wyoming INBRE), and P20 GM103451 (New Mexico INBRE, Bioinformatics Core). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-49">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300135/-/DC1</a>.</p></li><li class="fn" id="fn-10"><p id="p-50"><em>Communicating editor: K. Gunsalus</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-10"><span class="received-label">Received </span>August 10, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-12-12"><span class="accepted-label">Accepted </span>December 12, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Joseph <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.1" data-doi="10.1093/nar/gkw343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Afgan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Beek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouvier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">W3</span>–<span class="cit-lpage">W10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw343%26rft_id%253Dinfo%253Apmid%252F27137889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27137889&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-8.2.669.2"><div class="cit-metadata unstructured">Ahringer, J., 2005 Reverse genetics (April 6, 20016), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.47.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.3" data-doi="10.1534/genetics.114.169730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arribere</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artiles</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169730%26rft_id%253Dinfo%253Apmid%252F25161212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvODM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.4" data-doi="10.1371/journal.pgen.1003157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bendesky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pitts</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Long-range regulatory polymorphisms affecting a GABA receptor constitute a quantitative trait locus (QTL) for social behavior in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003157%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003157%26rft_id%253Dinfo%253Apmid%252F23284308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284308&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.5" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.2.669.6" data-doi="10.1126/science.282.5396.2012"><div class="cit-metadata"><cite><em>C. elegans</em> Sequencing Consortium, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genome sequence of the nematode C. elegans: a platform for investigating biology</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">2012</span>–<span class="cit-lpage">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BC.%2Belegans%2BSequencing%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D282%26rft.issue%253D5396%26rft.spage%253D2012%26rft.epage%253D2018%26rft.atitle%253DGenome%2BSequence%2Bof%2Bthe%2BNematode%2BC.%2Belegans%253A%2BA%2BPlatform%2Bfor%2BInvestigating%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5396.2012%26rft_id%253Dinfo%253Apmid%252F9851916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5Ni8yMDEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheesman</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feinbaum</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thekkiniath</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dowen</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Aberrant activation of p38 MAP kinase-dependent innate immune responses is toxic to <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.8" data-doi="10.3389/fgene.2012.00035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Using <em>Drosophila melanogaster</em> as a model for genotoxic chemical mutational studies with a new program, SnpSift.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DUsing%2BDrosophila%2Bmelanogaster%2Bas%2Ba%2BModel%2Bfor%2BGenotoxic%2BChemical%2BMutational%2BStudies%2Bwith%2Ba%2BNew%2BProgram%252C%2BSnpSift.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2012.00035%26rft_id%253Dinfo%253Apmid%252F22435069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2012.00035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22435069&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.9" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang le</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain <em>w<sup>1118</sup></em>; <em>iso-2</em>; <em>iso-3</em>.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.10" data-doi="10.1534/genetics.115.182162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CRISPR-based methods for <em>Caenorhabditis elegans</em> genome engineering.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDickinson%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D202%26rft.issue%253D3%26rft.spage%253D885%26rft.epage%253D901%26rft.atitle%253DCRISPR-Based%2BMethods%2Bfor%2BCaenorhabditis%2Belegans%2BGenome%2BEngineering%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182162%26rft_id%253Dinfo%253Apmid%252F26953268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzMvODg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.11" data-doi="10.1371/journal.pone.0015435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doitsidou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigelow</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>C. elegans</em> mutant identification with a one-step whole-genome-sequencing and SNP mapping strategy.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e15435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDoitsidou%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De15435%26rft.epage%253De15435%26rft.atitle%253DC.%2Belegans%2Bmutant%2Bidentification%2Bwith%2Ba%2Bone-step%2Bwhole-genome-sequencing%2Band%2BSNP%2Bmapping%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015435%26rft_id%253Dinfo%253Apmid%252F21079745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079745&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.12" data-doi="10.1534/genetics.115.186197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doitsidou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarriault</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Next-generation sequencing-based approaches for mutation mapping and identification in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDoitsidou%26rft.auinit1%253DM.%26rft.volume%253D204%26rft.issue%253D2%26rft.spage%253D451%26rft.epage%253D474%26rft.atitle%253DNext-Generation%2BSequencing-Based%2BApproaches%2Bfor%2BMutation%2BMapping%2Band%2BIdentification%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.186197%26rft_id%253Dinfo%253Apmid%252F27729495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA0LzIvNDUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doroszuk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fradin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riksen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genome-wide library of CB4856/N2 introgression lines of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">e110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DDoroszuk%26rft.auinit1%253DA.%26rft.volume%253D37%26rft.issue%253D16%26rft.spage%253De110%26rft.epage%253De110%26rft.atitle%253DA%2Bgenome-wide%2Blibrary%2Bof%2BCB4856%252FN2%2Bintrogression%2Blines%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F19542186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19542186&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.14" data-doi="10.1146/annurev.bi.45.070176.000303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baltz</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">The biochemistry of mutagenesis.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DDrake%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D45%26rft.spage%253D11%26rft.epage%253D37%26rft.atitle%253DThe%2Bbiochemistry%2Bof%2Bmutagenesis.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.bi.45.070176.000303%26rft_id%253Dinfo%253Apmid%252F786147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.bi.45.070176.000303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=786147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976CA02000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.15" data-doi="10.1534/genetics.110.116616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neil</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome profiling of mutagenesis in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116616%26rft_id%253Dinfo%253Apmid%252F20439774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNDMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glater</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multigenic natural variation underlies <em>Caenorhabditis elegans</em> olfactory preference for the bacterial pathogen <em>Serratia marcescens</em>.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D4%26rft.spage%253D265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic mapping of variation in dauer larvae development in growing populations of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D111%26rft.spage%253D306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.18" data-doi="10.1038/nmeth.1179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dooling</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fewell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Whole-genome sequencing and variant discovery in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DHillier%26rft.auinit1%253DL.%2BW.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D183%26rft.epage%253D188%26rft.atitle%253DWhole-genome%2Bsequencing%2Band%2Bvariant%2Bdiscovery%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1179%26rft_id%253Dinfo%253Apmid%252F18204455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18204455&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252750700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.19" data-doi="10.1016/j.ymeth.2014.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole genome sequencing and the transformation of <em>C. elegans</em> forward genetics.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D68%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2014.05.008%26rft_id%253Dinfo%253Apmid%252F24874788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2014.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24874788&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.20" data-doi="10.1186/gb-2013-14-6-r61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordstrom</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klasen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salome</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">User guide for mapping-by-sequencing in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR61%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r61%26rft_id%253Dinfo%253Apmid%252F23773572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23773572&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.21" data-doi="10.1074/mcp.M115.052548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamkina</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volkers</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterken</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Natural genetic variation differentially affects the proteome and transcriptome in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1670</span>–<span class="cit-lpage">1680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M115.052548%26rft_id%253Dinfo%253Apmid%252F26944343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjk6IjE1LzUvMTY3MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.22" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-8.2.669.23"><div class="cit-metadata unstructured">Kutscher, L. M., and S. Shaham, 2014 Forward and reverse mutagenesis in <em>C. elegans</em> (January 17, 2014), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.167.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.24" data-doi="10.1534/genetics.116.194464"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lazetic</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Conserved Ankyrin repeat proteins and their NIMA kinase partners regulate extracellular matrix remodeling and intracellular trafficking in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.194464%26rft_id%253Dinfo%253Apmid%252F27799278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA1LzEvMjczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.25" data-doi="10.1101/gr.135541.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leshchiner</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelsey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adzhubei</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austin-Tse</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mutation mapping and identification by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.135541.111%26rft_id%253Dinfo%253Apmid%252F22555591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.26" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.27" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.28" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.29" data-doi="10.1534/genetics.112.144204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minevich</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CloudMap: a cloud-based pipeline for analysis of mutant genome sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1249</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144204%26rft_id%253Dinfo%253Apmid%252F23051646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzEyNDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.30" data-doi="10.1016/j.ajpath.2013.02.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moresco</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beutler</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Going forward with genetics: recent technological advances and forward genetics in mice.</span> <abbr class="cit-jnl-abbrev">Am. J. Pathol.</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1462</span>–<span class="cit-lpage">1473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPathol.%26rft.volume%253D182%26rft.spage%253D1462%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajpath.2013.02.002%26rft_id%253Dinfo%253Apmid%252F23608223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajpath.2013.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23608223&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.31" data-doi="10.1242/dev.083931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Obholzer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swinburne</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwab</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nechiporuk</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rapid positional cloning of zebrafish mutations by linkage and homozygosity mapping using whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">4280</span>–<span class="cit-lpage">4290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.083931%26rft_id%253Dinfo%253Apmid%252F23052906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM5LzIyLzQyODAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.32" data-doi="10.1534/genetics.114.170423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calidas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Scalable and versatile genome editing using linear DNAs with microhomology to Cas9 Sites in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1347</span>–<span class="cit-lpage">1356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.170423%26rft_id%253Dinfo%253Apmid%252F25249454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzEzNDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.33" data-doi="10.1534/genetics.115.179382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folkmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasoloson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seydoux</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High efficiency, homology-directed genome editing in <em>Caenorhabditis elegans</em> using CRISPR-Cas9 ribonucleoprotein complexes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179382%26rft_id%253Dinfo%253Apmid%252F26187122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAxLzEvNDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Resistance to germline RNA interference in a <em>Caenorhabditis elegans</em> wild isolate exhibits complexity and nonadditivity.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">941</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D3%26rft.spage%253D941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.35" data-doi="10.1126/science.1166527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A polymorphism in <em>npr-1</em> is a behavioral determinant of pathogen susceptibility in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReddy%26rft.auinit1%253DK.%2BC.%26rft.volume%253D323%26rft.issue%253D5912%26rft.spage%253D382%26rft.epage%253D384%26rft.atitle%253DA%2BPolymorphism%2Bin%2Bnpr-1%2BIs%2Ba%2BBehavioral%2BDeterminant%2Bof%2BPathogen%2BSusceptibility%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166527%26rft_id%253Dinfo%253Apmid%252F19150845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMi8zODIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.36" data-doi="10.1016/j.exger.2012.05.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riksen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bevers</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic variation for stress-response hormesis in <em>C. elegans</em> lifespan.</span> <abbr class="cit-jnl-abbrev">Exp. Gerontol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Bgerontology%26rft.stitle%253DExp%2BGerontol%26rft.aulast%253DRodriguez%26rft.auinit1%253DM.%26rft.volume%253D47%26rft.issue%253D8%26rft.spage%253D581%26rft.epage%253D587%26rft.atitle%253DGenetic%2Bvariation%2Bfor%2Bstress-response%2Bhormesis%2Bin%2BC.%2Belegans%2Blifespan.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.exger.2012.05.005%26rft_id%253Dinfo%253Apmid%252F22613270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.exger.2012.05.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22613270&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.37" data-doi="10.1038/nmeth.1249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Meara</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pe’er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> mutant allele identification by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSarin%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D865%26rft.epage%253D867%26rft.atitle%253DCaenorhabditis%2Belegans%2Bmutant%2Ballele%2Bidentification%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1249%26rft_id%253Dinfo%253Apmid%252F18677319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677319&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259650100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.38" data-doi="10.1534/genetics.110.116319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigelow</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyanov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doitsidou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of multiple ethyl methanesulfonate-mutagenized <em>Caenorhabditis elegans</em> strains by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116319%26rft_id%253Dinfo%253Apmid%252F20439776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNDE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.39" data-doi="10.1038/nrg3745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using next-generation sequencing to isolate mutant genes from forward genetic screens.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3745%26rft_id%253Dinfo%253Apmid%252F25139187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25139187&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.40" data-doi="10.1016/j.tplants.2011.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast-forward genetics enabled by new sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DSchneeberger%26rft.auinit1%253DK.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D282%26rft.epage%253D288%26rft.atitle%253DFast-forward%2Bgenetics%2Benabled%2Bby%2Bnew%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2011.02.006%26rft_id%253Dinfo%253Apmid%252F21439889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2011.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21439889&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291191000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.41" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Widespread genetic incompatibility in <em>C. elegans</em> maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.42" data-doi="10.1371/journal.pbio.1001115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ailion</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oudenaarden</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A novel sperm-delivered toxin causes late-stage embryo lethality and transmission ratio distortion in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1001115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSeidel%26rft.auinit1%253DH.%2BS.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253De1001115%26rft.epage%253De1001115%26rft.atitle%253DA%2Bnovel%2Bsperm-delivered%2Btoxin%2Bcauses%2Blate-stage%2Bembryo%2Blethality%2Band%2Btransmission%2Bratio%2Bdistortion%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001115%26rft_id%253Dinfo%253Apmid%252F21814493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21814493&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roschitzki</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Natural genetic variation influences protein abundances in <em>C. elegans</em> developmental signalling pathways.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0149418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0149418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabritius</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo-Lambert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golden</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mapping challenging mutations by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D1297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orbidans</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stastna</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aartse</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Widespread genomic incompatibilities in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D4%26rft.spage%253D1813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-web" id="cit-8.2.669.46"><div class="cit-metadata unstructured">Stiernagle, T., 2006 Maintenance of <em>C. elegans</em> (February 11, 2006), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.101.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.47" data-doi="10.1101/gr.157651.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strasbourger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The million mutation project: a new approach to genetics in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.157651.113%26rft_id%253Dinfo%253Apmid%252F23800452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMC8xNzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.48" data-doi="10.1186/1471-2164-15-255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vergara</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarailo-Graovac</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frech</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide variations in a natural isolate of the nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D255%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-255%26rft_id%253Dinfo%253Apmid%252F24694239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24694239&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.49" data-doi="10.1038/88878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicks</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plasterk</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Rapid gene mapping in <em>Caenorhabditis elegans</em> using a high density polymorphism map.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWicks%26rft.auinit1%253DS.%2BR.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D164%26rft.atitle%253DRapid%2Bgene%2Bmapping%2Bin%2BCaenorhabditis%2Belegans%2Busing%2Ba%2Bhigh%2Bdensity%2Bpolymorphism%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F88878%26rft_id%253Dinfo%253Apmid%252F11381264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/88878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11381264&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169011800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yochem</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazetic</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>C. elegans</em> NIMA-related kinases NEKL-2 and NEKL-3 are required for the completion of molting.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">398</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D398%26rft.spage%253D255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.51" data-doi="10.4161/worm.25081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuryn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarriault</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Deep sequencing strategies for mapping and identifying mutations from genetic screens.</span> <abbr class="cit-jnl-abbrev">Worm</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e25081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWorm%26rft.volume%253D2%26rft.spage%253De25081%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fworm.25081%26rft_id%253Dinfo%253Apmid%252F24778934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/worm.25081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24778934&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.669.52" data-doi="10.1534/genetics.110.119230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuryn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gras</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamet</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarriault</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A strategy for direct mapping and identification of mutations by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119230%26rft_id%253Dinfo%253Apmid%252F20610404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvNDI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/669.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/659" title="Sequencing of Linkage Region on Chromosome 12p11 Identifies PKP2 as a Candidate Gene for Left Ventricular Mass in Dominican Families" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/679" title="Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sunze7TqbOvXEAQv5kUfDd9N-4XAYBCzX8dcLOj3yVM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/669" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/669" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/669" class="active" data-icon-position="" data-hide-link-title="0">Use of a Sibling Subtraction Method for Identifying Causal Mutations in Caenorhabditis elegans by Whole-Genome Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488292" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-uICzLwBXpmlxPGxYM6bTgHY2oDsiX5tUEGw9A8EI7EM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/669.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/669" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Yq5tE8NbZooKbAxwK2ITuLdgmcp8yXFHam4G8QdBCKs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/669" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/669.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488292" id="node488292--2" data-pisa="ggg;8/2/669" data-pisa-master="ggg;g3.117.300135" data-apath="/ggg/8/2/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of a Sibling Subtraction Method for Identifying Causal Mutations in <em>Caenorhabditis elegans</em> by Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Braveen B. Joseph</span>, <span class="highwire-citation-author" data-delta="1">Nicolas A. Blouin</span> and <span class="highwire-citation-author" data-delta="2">David S. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300135 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488292--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Braveen B. Joseph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Braveen%2BB.%2BJoseph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joseph%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABraveen%2BB.%2BJoseph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nicolas A. Blouin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Idea Networks for Biomedical Research Excellence (INBRE) Bioinformatics Core, University of Wyoming, Laramie, Wyoming 82071</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BA.%2BBlouin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blouin%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BA.%2BBlouin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David S. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BS.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:davidfay@uwyo.edu" class="" data-icon-position="" data-hide-link-title="0">davidfay@uwyo.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/669" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/669&amp;title=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/669&amp;title=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/669&amp;title=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/669&amp;text=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/669&amp;title=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/669&amp;t=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/669&amp;title=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/669&amp;title=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488292/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488292" id="node488292--3" data-pisa="ggg;8/2/669" data-pisa-master="ggg;g3.117.300135" data-apath="/ggg/8/2/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of a Sibling Subtraction Method for Identifying Causal Mutations in <em>Caenorhabditis elegans</em> by Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Braveen B. Joseph</span>, <span class="highwire-citation-author" data-delta="1">Nicolas A. Blouin</span> and <span class="highwire-citation-author" data-delta="2">David S. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300135 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488292--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Braveen B. Joseph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Braveen%2BB.%2BJoseph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joseph%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABraveen%2BB.%2BJoseph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nicolas A. Blouin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Idea Networks for Biomedical Research Excellence (INBRE) Bioinformatics Core, University of Wyoming, Laramie, Wyoming 82071</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BA.%2BBlouin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blouin%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BA.%2BBlouin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David S. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, College of Agriculture and Natural Resources, University of Wyoming, Laramie, Wyoming 82071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BS.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:davidfay@uwyo.edu" class="" data-icon-position="" data-hide-link-title="0">davidfay@uwyo.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=669&amp;rft.epage=678&amp;rft.atitle=Use%20of%20a%20Sibling%20Subtraction%20Method%20for%20Identifying%20Causal%20Mutations%20in%20Caenorhabditis%20elegans%20by%20Whole-Genome%20Sequencing&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Joseph%2C+Braveen+B.&amp;rft.au=Blouin%2C+Nicolas+A.&amp;rft.au=Fay%2C+David+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488292/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488292/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488292/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488292/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488292/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488292/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488292/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488292/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488292/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488292/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488292/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488292/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/669" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/669.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/669.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/669.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/669.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
