<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/427" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/427.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/427.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/427.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/2/427/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/427.atom" />
<meta name="HW.pisa" content="ggg;8/2/427" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Bayesian Diallel Analysis Reveals Mx1-Dependent and Mx1-Independent Effects on Response to Influenza A Virus in Mice" />
<meta name="DC.Identifier" content="10.1534/g3.117.300438" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Maurizio et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Influenza A virus (IAV) is a respiratory pathogen that causes substantial morbidity and mortality during both seasonal and pandemic outbreaks. Infection outcomes in unexposed populations are affected by host genetics, but the host genetic architecture is not well understood. Here, we obtain a broad view of how heritable factors affect a mouse model of response to IAV infection using an 8 × 8 diallel of the eight inbred founder strains of the Collaborative Cross (CC). Expanding on a prior statistical framework for modeling treatment response in diallels, we explore how a range of heritable effects modify acute host response to IAV through 4 d postinfection. Heritable effects in aggregate explained ∼57% of the variance in IAV-induced weight loss. Much of this was attributable to a pattern of additive effects that became more prominent through day 4 postinfection and was consistent with previous reports of antiinfluenza myxovirus resistance 1 ( Mx1 ) polymorphisms segregating between these strains; these additive effects largely recapitulated haplotype effects observed at the Mx1 locus in a previous study of the incipient CC, and are also replicated here in a CC recombinant intercross population. Genetic dominance of protective Mx1 haplotypes was observed to differ by subspecies of origin: relative to the domesticus null Mx1 allele, musculus acts dominantly whereas castaneus acts additively. After controlling for Mx1 , heritable effects, though less distinct, accounted for ∼34% of the phenotypic variance. Implications for future mapping studies are discussed." />
<meta name="DC.Contributor" content="Paul L. Maurizio" />
<meta name="DC.Contributor" content="Martin T. Ferris" />
<meta name="DC.Contributor" content="Gregory R. Keele" />
<meta name="DC.Contributor" content="Darla R. Miller" />
<meta name="DC.Contributor" content="Ginger D. Shaw" />
<meta name="DC.Contributor" content="Alan C. Whitmore" />
<meta name="DC.Contributor" content="Ande West" />
<meta name="DC.Contributor" content="Clayton R. Morrison" />
<meta name="DC.Contributor" content="Kelsey E. Noll" />
<meta name="DC.Contributor" content="Kenneth S. Plante" />
<meta name="DC.Contributor" content="Adam S. Cockrell" />
<meta name="DC.Contributor" content="David W. Threadgill" />
<meta name="DC.Contributor" content="Fernando Pardo-Manuel de Villena" />
<meta name="DC.Contributor" content="Ralph S. Baric" />
<meta name="DC.Contributor" content="Mark T. Heise" />
<meta name="DC.Contributor" content="William Valdar" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Bayesian Diallel Analysis Reveals Mx1-Dependent and Mx1-Independent Effects on Response to Influenza A Virus in Mice" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Influenza A virus (IAV) is a respiratory pathogen that causes substantial morbidity and mortality during both seasonal and pandemic outbreaks. Infection outcomes in unexposed populations are affected by host genetics, but the host genetic architecture is not well understood. Here, we obtain a broad view of how heritable factors affect a mouse model of response to IAV infection using an 8 × 8 diallel of the eight inbred founder strains of the Collaborative Cross (CC). Expanding on a prior statistical framework for modeling treatment response in diallels, we explore how a range of heritable effects modify acute host response to IAV through 4 d postinfection. Heritable effects in aggregate explained ∼57% of the variance in IAV-induced weight loss. Much of this was attributable to a pattern of additive effects that became more prominent through day 4 postinfection and was consistent with previous reports of antiinfluenza myxovirus resistance 1 (&lt;i&gt;Mx1&lt;/i&gt;) polymorphisms segregating between these strains; these additive effects largely recapitulated haplotype effects observed at the &lt;i&gt;Mx1&lt;/i&gt; locus in a previous study of the incipient CC, and are also replicated here in a CC recombinant intercross population. Genetic dominance of protective &lt;i&gt;Mx1&lt;/i&gt; haplotypes was observed to differ by subspecies of origin: relative to the &lt;i&gt;domesticus&lt;/i&gt; null &lt;i&gt;Mx1&lt;/i&gt; allele, &lt;i&gt;musculus&lt;/i&gt; acts dominantly whereas &lt;i&gt;castaneus&lt;/i&gt; acts additively. After controlling for &lt;i&gt;Mx1&lt;/i&gt;, heritable effects, though less distinct, accounted for ∼34% of the phenotypic variance. Implications for future mapping studies are discussed.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/427" />
<meta name="citation_id" content="8/2/427" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/427" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/427.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/427.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/427.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300438" />
<meta name="citation_pmid" content="29187420" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="19" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="427" />
<meta name="citation_lastpage" content="445" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Paul L. Maurizio" />
<meta name="citation_author_institution" content="Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5859-6260" />
<meta name="citation_author" content="Martin T. Ferris" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1241-6268" />
<meta name="citation_author" content="Gregory R. Keele" />
<meta name="citation_author_institution" content="Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1843-7900" />
<meta name="citation_author" content="Darla R. Miller" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0781-7254" />
<meta name="citation_author" content="Ginger D. Shaw" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2590-4973" />
<meta name="citation_author" content="Alan C. Whitmore" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Ande West" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Clayton R. Morrison" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Kelsey E. Noll" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3977-0083" />
<meta name="citation_author" content="Kenneth S. Plante" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="Adam S. Cockrell" />
<meta name="citation_author_institution" content="Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="David W. Threadgill" />
<meta name="citation_author_institution" content="Institute of Genome Sciences and Society, Texas A&amp;M University, College Station, Texas 77433-2470" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3538-1635" />
<meta name="citation_author" content="Fernando Pardo-Manuel de Villena" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5738-5795" />
<meta name="citation_author" content="Ralph S. Baric" />
<meta name="citation_author_institution" content="Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6827-8701" />
<meta name="citation_author" content="Mark T. Heise" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author" content="William Valdar" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_email" content="william.valdar@unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2419-0430" />
<meta name="citation_reference" content="citation_journal_title=Microbes and infection / Institut Pasteur;citation_journal_abbrev=Microbes Infect;citation_author=R. Alberts;citation_author=B. Srivastava;citation_author=H. Wu;citation_author=N. Viegas;citation_author=R. Geffers;citation_title=Gene expression changes in the host response between resistant and susceptible inbred mouse strains after influenza A infection.;citation_pages=309-318;citation_volume=12;citation_year=2010;citation_issue=4;citation_pmid=20114087;citation_doi=10.1016/j.micinf.2010.01.008" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Mächler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting linear mixed-effects models using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. W. Bigham;citation_author=K. J. Buckingham;citation_author=S. Husain;citation_author=M. J. Emond;citation_author=K. M. Bofferding;citation_title=Host genetic risk factors for West Nile virus infection and disease progression.;citation_pages=e24745-e24745;citation_volume=6;citation_year=2011;citation_issue=9;citation_pmid=21935451;citation_doi=10.1371/journal.pone.0024745" />
<meta name="citation_reference" content="citation_journal_title=J. Immunol.;citation_author=G. A. Boivin;citation_author=J. Pothlichet;citation_author=E. Skamene;citation_author=E. G. Brown;citation_author=J. C. Loredo-Osti;citation_title=Mapping of clinical and expression quantitative trait loci in a sex-dependent effect of host susceptibility to mouse-adapted influenza H3N2/HK/1/68.;citation_volume=188;citation_year=2012;citation_pmid=22427645;citation_doi=10.4049/jimmunol.1103320" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=A. C. M. Boon;citation_author=J. DeBeauchamp;citation_author=A. Hollmann;citation_author=J. Luke;citation_author=M. Kotb;citation_title=Host genetic variation affects resistance to infection with a highly pathogenic H5N1 influenza A virus in mice.;citation_volume=83;citation_year=2009;citation_pmid=19706712;citation_doi=10.1128/JVI.00514-09" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Bottomly;citation_author=M. T. Ferris;citation_author=L. D. Aicher;citation_author=E. Rosenzweig;citation_author=A. Whitmore;citation_title=Expression quantitative trait Loci for extreme host response to influenza A in pre-Collaborative Cross mice.;citation_pages=213-221;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=E. J. Chesler;citation_author=D. R. Miller;citation_author=L. R. Branstetter;citation_author=L. D. Galloway;citation_author=B. L. Jackson;citation_title=The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.;citation_pages=382-389;citation_volume=19;citation_year=2008;citation_issue=6;citation_pmid=18716833;citation_doi=10.1007/s00335-008-9135-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=B. R. Christie;citation_author=V. I. Shattuck;citation_title=The diallel cross: design, analysis, and use for plant breeders.;citation_pages=9-36;citation_volume=9;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=G. A. Churchill;citation_author=D. M. Gatti;citation_author=S. C. Munger;citation_author=K. L. Svenson;citation_title=The diversity outbred mouse population.;citation_pages=713-718;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892839;citation_doi=10.1007/s00335-012-9414-2" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. J. Ciancanelli;citation_author=S. X. L. Huang;citation_author=P. Luthra;citation_author=H. Garner;citation_author=Y. Itan;citation_title=Life-threatening influenza and impaired interferon amplification in human IRF7 deficiency.;citation_volume=348;citation_year=2015;citation_pmid=25814066;citation_doi=10.1126/science.aaa1578" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the Collaborative Cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=R. Comstock;citation_author=H. Robinson;citation_title=The components of genetic variance in populations of biparental progenies and their use in estimating the average degree of dominance.;citation_pages=254-266;citation_volume=4;citation_year=1948;citation_issue=4;citation_pmid=18108899;citation_doi=10.2307/3001412" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. J. Crowley;citation_author=Y. Kim;citation_author=A. B. Lenarcic;citation_author=C. R. Quackenbush;citation_author=C. J. Barrick;citation_title=Genetics of adverse reactions to haloperidol in a mouse diallel: a drug-placebo experiment and Bayesian causal analysis.;citation_volume=196;citation_year=2014;citation_pmid=24240528;citation_doi=10.1534/genetics.113.156901" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=J. Dittmann;citation_author=S. Stertz;citation_author=D. Grimm;citation_author=J. Steel;citation_author=A. García-Sastre;citation_title=Influenza A virus strains differ in sensitivity to the antiviral action of Mx-GTPase.;citation_volume=82;citation_year=2008;citation_pmid=18199636;citation_doi=10.1128/JVI.01753-07" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=P. L. Earl;citation_author=J. L. Americo;citation_author=B. Moss;citation_title=Lethal monkeypox virus infection of CAST/EiJ mice is associated with a deficient gamma interferon response.;citation_volume=86;citation_year=2012;citation_pmid=22696658;citation_doi=10.1128/JVI.00162-12" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=K. M. Edenborough;citation_author=B. P. Gilbertson;citation_author=L. E. Brown;citation_title=A mouse model for the study of contact-dependent transmission of influenza A virus and the factors that govern transmissibility.;citation_volume=86;citation_year=2012;citation_pmid=22951824;citation_doi=10.1128/JVI.00859-12" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. R. Everitt;citation_author=S. Clare;citation_author=T. Pertel;citation_author=S. P. John;citation_author=R. S. Wash;citation_title=IFITM3 restricts the morbidity and mortality associated with influenza.;citation_pages=519-523;citation_volume=484;citation_year=2012;citation_issue=7395;citation_pmid=22446628;citation_doi=10.1038/nature10921" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=M. T. Ferris;citation_author=D. L. Aylor;citation_author=D. Bottomly;citation_author=A. C. Whitmore;citation_author=L. D. Aicher;citation_title=Modeling host genetic regulation of influenza pathogenesis in the Collaborative Cross.;citation_pages=e1003196;citation_volume=9;citation_year=2013;citation_pmid=23468633;citation_doi=10.1371/journal.ppat.1003196" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=R. S. S. Fraser;citation_title=The Genetics Of Resistance To Plant Viruses;citation_pages=179-200;citation_volume=28;citation_year=1990;citation_issue=1;citation_doi=10.1146/annurev.py.28.090190.001143" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=R. S. S. Fraser;citation_title=The genetics of plant-virus interactions: implications for plant breeding.;citation_pages=175-185;citation_volume=63;citation_year=1992;citation_doi=10.1007/BF00023922" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=R. S. S. Fraser;citation_author=L. C. V. Loon;citation_title=Genes for resistance to plant viruses.;citation_pages=257-294;citation_volume=3;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=C. Gardner;citation_author=J. Lonnquist;citation_title=Linkage and the Degree of Dominance of Genes Controlling Quantitative Characters in Maize;citation_pages=524-528;citation_volume=51;citation_year=1959;citation_issue=9;citation_doi=10.2134/agronj1959.00021962005100090005x" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L.-Y. Wu;citation_author=W. Valdar;citation_title=Quantitative trait locus mapping methods for Diversity Outbred mice.;citation_pages=1623-1633;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.114.013748" />
<meta name="citation_reference" content="citation_title=Data Analysis Using Regression and Multilevel/Hierarchical Models;citation_title=Missing data imputation;citation_author=A. Gelman;citation_author=J. Hill;citation_pages=529-543;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. Gonzalo;citation_author=T. J. Vyn;citation_author=J. B. Holland;citation_author=L. M. McIntyre;citation_title=Mapping reciprocal effects and interactions with plant density stress in Zea mays L.;citation_pages=14-30;citation_volume=99;citation_year=2007;citation_issue=1;citation_pmid=17473872;citation_doi=10.1038/sj.hdy.6800955" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Greenberg;citation_author=S. R. Hackett;citation_author=L. G. Harshman;citation_author=A. G. Clark;citation_title=A hierarchical Bayesian model for a novel sparse partial diallel crossing design.;citation_volume=185;citation_year=2010;citation_pmid=20157001;citation_doi=10.1534/genetics.110.115055" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. C. Grubb;citation_author=C. J. Bult;citation_author=M. A. Bogue;citation_title=Mouse phenome database.;citation_volume=42;citation_year=2014;citation_pmid=24243846;citation_doi=10.1093/nar/gkt1159" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Hallin;citation_author=K. Märtens;citation_author=A. I. Young;citation_author=M. Zackrisson;citation_author=F. Salinas;citation_title=Powerful decomposition of complex traits in a diploid model.;citation_pages=13311;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=M. Hashimoto;citation_author=Y. Neriya;citation_author=Y. Yamaji;citation_author=S. Namba;citation_title=Recessive resistance to plant viruses: potential resistance genes beyond translation initiation factors.;citation_pages=1695;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Clinical immunology (Orlando, Fla.);citation_journal_abbrev=Clin Immunol;citation_author=F. Hidaka;citation_author=S. Matsuo;citation_author=T. Muta;citation_author=K. Takeshige;citation_author=T. Mizukami;citation_title=A missense mutation of the Toll-like receptor 3 gene in a patient with influenza-associated encephalopathy.;citation_pages=188-194;citation_volume=119;citation_year=2006;citation_issue=2;citation_pmid=16517210;citation_doi=10.1016/j.clim.2006.01.005" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=P. W. Holland;citation_title=Statistics and causal inference.;citation_pages=945-960;citation_volume=81;citation_year=1986;citation_doi=10.2307/2289064" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Hua;citation_author=Y. Xing;citation_author=W. Wu;citation_author=C. Xu;citation_author=X. Sun;citation_title=Single-locus heterotic effects and dominance by dominance interactions can adequately explain the genetic basis of heterosis in an elite rice hybrid.;citation_volume=100;citation_year=2003;citation_pmid=12604771;citation_doi=10.1073/pnas.0437907100" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. P. Hua;citation_author=Y. Z. Xing;citation_author=C. G. Xu;citation_author=X. L. Sun;citation_author=S. B. Yu;citation_title=Genetic Dissection of an Elite Rice Hybrid Revealed That Heterozygotes Are Not Always Advantageous for Performance;citation_pages=1885-1895;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524357" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=W. Huang;citation_author=T. F. C. Mackay;citation_title=The genetic architecture of quantitative traits cannot be inferred from variance component analysis.;citation_pages=e1006421;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=G. Hütter;citation_author=D. Nowak;citation_author=M. Mossner;citation_author=S. Ganepola;citation_author=A. Müßig;citation_title=Long-Term Control of HIV by CCR5 Delta32/Delta32 Stem-Cell Transplantation;citation_pages=692-698;citation_volume=360;citation_year=2009;citation_issue=7;citation_pmid=19213682;citation_doi=10.1056/NEJMoa0802905" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Itoh;citation_author=K. Shinya;citation_author=M. Kiso;citation_author=T. Watanabe;citation_author=Y. Sakoda;citation_title=In vitro and in vivo characterization of new swine-origin H1N1 influenza viruses.;citation_pages=1021-1025;citation_volume=460;citation_year=2009;citation_issue=7258;citation_pmid=19672242;citation_doi=10.1038/nature08260" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Kacser;citation_author=J. A. Burns;citation_title=THE MOLECULAR BASIS OF DOMINANCE;citation_pages=639-666;citation_volume=97;citation_year=1981;citation_issue=3-4;citation_pmid=7297851" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=B.-C. Kang;citation_author=I. Yeam;citation_author=M. M. Jahn;citation_title=Genetics of plant virus resistance.;citation_pages=581-621;citation_volume=43;citation_year=2005;citation_pmid=16078896;citation_doi=10.1146/annurev.phyto.43.011205.141140" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=R. E. Kass;citation_author=A. E. Raftery;citation_title=Bayes factors.;citation_pages=773-795;citation_volume=90;citation_year=1995;citation_pmid=10783804;citation_doi=10.2307/2291091" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. D. Keightley;citation_title=A Metabolic Basis for Dominance and Recessivity;citation_pages=621-625;citation_volume=143;citation_year=1996;citation_issue=2;citation_pmid=8725213" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=K. P. Kollipara;citation_author=I. N. Saab;citation_author=R. D. Wych;citation_author=M. J. Lauer;citation_author=G. W. Singletary;citation_title=Expression profiling of reciprocal maize hybrids divergent for cold germination and desiccation tolerance.;citation_volume=129;citation_year=2002;citation_pmid=12114554;citation_doi=10.1104/pp.000729" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. R. Leist;citation_author=C. Pilzner;citation_author=J. M. A. van den Brand;citation_author=L. Dengler;citation_author=R. Geffers;citation_title=Influenza H3N2 infection of the Collaborative Cross founder strains reveals highly divergent host responses and identifies a unique phenotype in CAST/EiJ mice.;citation_pages=143;citation_volume=17;citation_year=2016;citation_pmid=26921172;citation_doi=10.1186/s12864-016-2483-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. B. Lenarcic;citation_author=K. L. Svenson;citation_author=G. A. Churchill;citation_author=W. Valdar;citation_title=A general Bayesian approach to analyzing diallel crosses of inbred strains.;citation_volume=190;citation_year=2012;citation_pmid=22345610;citation_doi=10.1534/genetics.111.132563" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. J. Lenschow;citation_author=C. Lai;citation_author=N. Frias-Staheli;citation_author=N. V. Giannakopoulos;citation_author=A. Lutz;citation_title=From the cover: IFN-stimulated gene 15 functions as a critical antiviral molecule against influenza, herpes, and Sindbis viruses.;citation_volume=104;citation_year=2007;citation_pmid=17227866;citation_doi=10.1073/pnas.0607038104" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Liu;citation_author=W. A. Paxton;citation_author=S. Choe;citation_author=D. Ceradini;citation_author=S. R. Martin;citation_title=Homozygous defect in HIV-1 coreceptor accounts for resistance of some multiply-exposed individuals to HIV-1 infection.;citation_pages=367-377;citation_volume=86;citation_year=1996;citation_issue=3;citation_pmid=8756719;citation_doi=10.1016/S0092-8674(00)80110-5" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=R. Liu;citation_author=B. Wang;citation_author=W. Guo;citation_author=Y. Qin;citation_author=L. Wang;citation_title=Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in Gossypium hirsutum L.;citation_pages=297-311;citation_volume=29;citation_year=2012;citation_doi=10.1007/s11032-011-9547-0" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=Y. Liu;citation_author=Z. B. Zeng;citation_title=A general mixture model approach for mapping quantitative trait loci from diverse cross designs involving multiple inbred lines.;citation_pages=345-355;citation_volume=75;citation_year=2000;citation_issue=3;citation_pmid=10893870;citation_doi=10.1017/S0016672300004493" />
<meta name="citation_reference" content="citation_journal_title=Vaccine;citation_author=M. E. Lorenzo;citation_author=A. Hodgson;citation_author=D. P. Robinson;citation_author=J. B. Kaplan;citation_author=A. Pekosz;citation_title=Antibody responses and cross protection against lethal influenza A viruses differ between the sexes in C57BL/6 mice.;citation_pages=9246-9255;citation_volume=29;citation_year=2011;citation_pmid=21983155;citation_doi=10.1016/j.vaccine.2011.09.110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. C. Lowen;citation_author=S. Mubareka;citation_author=T. M. Tumpey;citation_author=A. Garcia-Sastre;citation_author=P. Palese;citation_title=The guinea pig as a transmission model for human influenza viruses.;citation_volume=103;citation_year=2006;citation_pmid=16785447;citation_doi=10.1073/pnas.0604157103" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=B. Mänz;citation_author=D. Dornfeld;citation_author=V. Götz;citation_author=R. Zell;citation_author=P. Zimmermann;citation_title=Pandemic influenza A viruses escape from restriction by human MxA through adaptive mutations in the nucleoprotein.;citation_pages=e1003279;citation_volume=9;citation_year=2013;citation_pmid=23555271;citation_doi=10.1371/journal.ppat.1003279" />
<meta name="citation_reference" content="citation_journal_title=Journal of acquired immune deficiency syndromes;citation_journal_abbrev=J Acquir Immune Defic Syndr;citation_author=M. Marmor;citation_author=H. W. Sheppard;citation_author=D. Donnell;citation_author=S. Bozeman;citation_author=C. Celum;citation_title=Homozygous and heterozygous CCR5-Delta32 genotypes are associated with resistance to HIV infection.;citation_pages=472-481;citation_volume=27;citation_year=2001;citation_issue=5;citation_pmid=11511825;citation_doi=10.1097/00126334-200108150-00009" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat. Data Anal.;citation_author=T. G. Martins;citation_author=D. Simpson;citation_author=F. Lindgren;citation_author=H. Rue;citation_title=Bayesian computing with INLA: new features.;citation_pages=68-83;citation_volume=67;citation_year=2013;citation_doi=10.1016/j.csda.2013.04.014" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. W. Mei;citation_author=Z. K. Li;citation_author=Q. Y. Shu;citation_author=L. B. Guo;citation_author=Y. P. Wang;citation_title=Gene actions of QTLs affecting several agronomic traits resolved in a recombinant inbred rice population and two backcross populations.;citation_pages=649-659;citation_volume=110;citation_year=2005;citation_issue=4;citation_pmid=15647921;citation_doi=10.1007/s00122-004-1890-7" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=D. M. Morens;citation_author=J. K. Taubenberger;citation_author=A. S. Fauci;citation_title=The 2009 H1N1 pandemic influenza virus: what next?;citation_volume=1;citation_year=2010;citation_pmid=20877580;citation_doi=10.1128/mBio.00211-10" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=A. P. Morgan;citation_author=C. E. Welsh;citation_title=Informatics resources for the Collaborative Cross and related mouse populations.;citation_pages=521-539;citation_volume=26;citation_year=2015;citation_pmid=26135136;citation_doi=10.1007/s00335-015-9581-z" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. P. Morgan;citation_author=C.-P. Fu;citation_author=C.-Y. Kao;citation_author=C. E. Welsh;citation_author=J. P. Didion;citation_title=The mouse universal genotyping array: from substrains to subspecies.;citation_pages=263-279;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. Nedelko;citation_author=H. Kollmus;citation_author=F. Klawonn;citation_author=S. Spijker;citation_author=L. Lu;citation_title=Distinct gene loci control the host response to influenza H1N1 virus infection in a time-dependent manner.;citation_pages=411-411;citation_volume=13;citation_year=2012;citation_pmid=22905720;citation_doi=10.1186/1471-2164-13-411" />
<meta name="citation_reference" content="citation_journal_title=Roczniki Nauk Rolniczch;citation_author=J. Neyman;citation_title=On the application of probability theory to agricultural experiments: principles.;citation_pages=1-51;citation_volume=10;citation_year=1923" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=C. Nürnberger;citation_author=V. Zimmermann;citation_author=M. Gerhardt;citation_author=P. Staeheli;citation_title=Influenza virus susceptibility of wild-derived CAST/EiJ mice results from two amino acid changes in the MX1 restriction factor.;citation_volume=90;citation_year=2016;citation_pmid=27654285;citation_doi=10.1128/JVI.01213-16" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=F. Ogut;citation_author=Y. Bian;citation_author=P. J. Bradbury;citation_author=J. B. Holland;citation_title=Joint-multiple family linkage analysis predicts within-family variation better than single-family analysis of the maize nested association mapping population.;citation_pages=552-563;citation_volume=114;citation_year=2015;citation_pmid=25585918;citation_doi=10.1038/hdy.2014.123" />
<meta name="citation_reference" content="citation_journal_title=Indian J. Anim. Res.;citation_author=V. Okoro;citation_author=C. Mbajiorgu;citation_title=Diallel cross in swine production: a review.;citation_pages=212-218;citation_volume=51;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. G. Oreper;citation_author=Y. Cai;citation_author=L. M. Tarantino;citation_author=F. Pardo-Manuel de Villena;citation_author=W. Valdar;citation_title=Inbred strain variant database (ISVdb): a repository for probabilistically informed sequence differences among the Collaborative Cross strains and their founders.;citation_volume=7;citation_year=2017;citation_pmid=28592645;citation_doi=10.1534/g3.117.041491" />
<meta name="citation_reference" content="citation_journal_title=Genes Immun.;citation_author=J. Phillippi;citation_author=Y. Xie;citation_author=D. Miller;citation_author=T. Bell;citation_author=Z. Zhang;citation_title=Using the emerging Collaborative Cross to probe the immune system.;citation_pages=38-46;citation_volume=15;citation_year=2014;citation_pmid=24195963;citation_doi=10.1038/gene.2013.59" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=M. J. Rantala;citation_author=D. A. Roff;citation_title=Analysis of the importance of genotypic variation, metabolic rate, morphology, sex and development time on immune function in the cricket, Gryllus firmus.;citation_pages=834-843;citation_volume=19;citation_year=2006;citation_issue=3;citation_pmid=16674580;citation_doi=10.1111/j.1420-9101.2005.01048.x" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Rebai;citation_author=B. Goffinet;citation_title=Power of tests for QTL detection using replicated progenies derived from a diallel cross.;citation_pages=1014-1022;citation_volume=86;citation_year=1993;citation_pmid=24194011" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=A. Rebaï;citation_author=B. Goffinet;citation_title=More about quantitative trait locus mapping with diallel designs.;citation_pages=243-247;citation_volume=75;citation_year=2000;citation_issue=2;citation_pmid=10816981;citation_doi=10.1017/S0016672399004358" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=D. Riegger;citation_author=R. Hai;citation_author=D. Dornfeld;citation_author=B. Mänz;citation_author=V. Leyva-Grado;citation_title=The nucleoprotein of newly emerged H7N9 influenza A virus harbors a unique motif conferring resistance to antiviral human MxA.;citation_volume=89;citation_year=2015;citation_pmid=25505067;citation_doi=10.1128/JVI.02406-14" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=D. P. Robinson;citation_author=M. E. Lorenzo;citation_author=W. Jian;citation_author=S. L. Klein;citation_title=Elevated 17Î²-estradiol protects females from influenza A virus pathogenesis by suppressing inflammatory responses.;citation_pages=e1002149-e1002149;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21829352;citation_doi=10.1371/journal.ppat.1002149" />
<meta name="citation_reference" content="citation_journal_title=J. Educ. Psychol.;citation_author=D. B. Rubin;citation_title=Estimating causal effects of treatments in randomized and nonrandomized studies.;citation_pages=688-701;citation_volume=66;citation_year=1974;citation_doi=10.1037/h0037350" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=J. Sabourin;citation_author=A. B. Nobel;citation_author=W. Valdar;citation_title=Fine-mapping additive and dominant SNP effects using group-LASSO and fractional resample model averaging.;citation_pages=77-88;citation_volume=39;citation_year=2015;citation_pmid=25417853;citation_doi=10.1002/gepi.21869" />
<meta name="citation_reference" content="citation_journal_title=Yale J. Biol. Med.;citation_author=S. J. Samet;citation_author=S. M. Tompkins;citation_title=Focus: comparative medicine: influenza pathogenesis in genetically defined resistant and susceptible murine strains.;citation_pages=471;citation_volume=90;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Samson;citation_author=F. Libert;citation_author=B. J. Doranz;citation_author=J. Rucker;citation_author=C. Liesnard;citation_title=Resistance to HIV-1 infection in caucasian individuals bearing mutant alleles of the CCR-5 chemokine receptor gene.;citation_pages=722-725;citation_volume=382;citation_year=1996;citation_issue=6593;citation_pmid=8751444;citation_doi=10.1038/382722a0" />
<meta name="citation_reference" content="citation_journal_title=C. R. Trav. Lab. Carlsberg;citation_author=J. Schmidt;citation_title=La valeur de l’individu à titre de génratéur appréciée suivant la méthode du croisement dialléle.;citation_pages=1-33;citation_volume=14;citation_year=1919" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=L. Shang;citation_author=Y. Wang;citation_author=S. Cai;citation_author=X. Wang;citation_author=Y. Li;citation_title=Partial dominance, overdominance, epistasis and QTL by environment interactions contribute to heterosis in two upland cotton hybrids.;citation_pages=499-507;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=D.-L. Shin;citation_author=B. Hatesuer;citation_author=S. Bergmann;citation_author=T. Nedelko;citation_author=K. Schughart;citation_title=Protection from severe influenza virus infections in mice carrying the Mx1 influenza virus resistance gene strongly depends on genetic background.;citation_volume=89;citation_year=2015;citation_pmid=26202236;citation_doi=10.1128/JVI.01305-15" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Srivastava;citation_author=A. P. Morgan;citation_author=M. L. Najarian;citation_author=V. K. Sarsani;citation_author=J. S. Sigmon;citation_title=Genomes of the mouse Collaborative Cross.;citation_volume=206;citation_year=2017;citation_pmid=28592495;citation_doi=10.1534/genetics.116.198838" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Srivastava;citation_author=P. Błazejewska;citation_author=M. Hessmann;citation_author=D. Bruder;citation_author=R. Geffers;citation_title=Host genetic background strongly influences the response to influenza a virus infections.;citation_pages=e4857-e4857;citation_volume=4;citation_year=2009;citation_issue=3;citation_pmid=19293935;citation_doi=10.1371/journal.pone.0004857" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. Staeheli;citation_author=R. Grob;citation_author=E. Meier;citation_author=J. G. Sutcliffe;citation_author=O. Haller;citation_title=Influenza virus-susceptible mice carry Mx genes with a large deletion or a nonsense mutation.;citation_pages=4518-4523;citation_volume=8;citation_year=1988;citation_issue=10;citation_pmid=2903437;citation_doi=10.1128/MCB.8.10.4518" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Infect. Dis.;citation_author=T. Strunk;citation_author=S. E. Jamieson;citation_author=D. Burgner;citation_title=Genetic and epigenetic susceptibility to early life infection.;citation_pages=241-247;citation_volume=26;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. A. Swanson-Wagner;citation_author=R. DeCook;citation_author=Y. Jia;citation_author=T. Bancroft;citation_author=T. Ji;citation_title=Paternal Dominance of Trans-eQTL Influences Gene Expression Patterns in Maize Hybrids;citation_pages=1118-1120;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965432;citation_doi=10.1126/science.1178294" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-resolution genetic mapping using the mouse Diversity Outbred population;citation_volume=190;citation_year=2012;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=D. W. Threadgill;citation_author=K. W. Hunter;citation_author=R. W. Williams;citation_title=Genetic dissection of complex and quantitative traits: from fantasy to reality via a community effort.;citation_pages=175-178;citation_volume=13;citation_year=2002;citation_issue=4;citation_pmid=11956758;citation_doi=10.1007/s00335-001-4001-y" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Virol.;citation_author=K. K.-W. To;citation_author=J. Zhou;citation_author=J. F.-W. Chan;citation_author=K.-Y. Yuen;citation_title=Host genes and influenza pathogenesis in humans: an emerging paradigm.;citation_pages=7-15;citation_volume=14;citation_year=2015;citation_pmid=26079652;citation_doi=10.1016/j.coviro.2015.04.010" />
<meta name="citation_reference" content="citation_journal_abbrev=AIDS Res Ther;citation_author=E. M. Trecarichi;citation_author=M. Tumbarello;citation_author=K. de Gaetano Donati;citation_author=E. Tamburrini;citation_author=R. Cauda;citation_title=Partial protective effect of CCR5-Delta 32 heterozygosity in a cohort of heterosexual Italian HIV-1 exposed uninfected individuals.;citation_pages=22-22;citation_volume=3;citation_year=2006;citation_pmid=16999868;citation_doi=10.1186/1742-6405-3-22" />
<meta name="citation_reference" content="citation_title=Advances in virus research;citation_journal_abbrev=Adv Virus Res;citation_author=V. Truniger;citation_author=M. Aranda;citation_title=Recessive resistance to plant viruses.;citation_pages=119-159;citation_volume=Vol. 75;citation_year=2009;citation_pmid=20109665;citation_doi=10.1016/S0065-3527(09)07504-6" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=S.-W. Tsaih;citation_author=L. Lu;citation_author=D. C. Airey;citation_author=R. W. Williams;citation_author=G. A. Churchill;citation_title=Quantitative trait mapping in a diallel cross of recombinant inbred lines.;citation_pages=344-355;citation_volume=16;citation_year=2005;citation_issue=5;citation_pmid=16104382;citation_doi=10.1007/s00335-004-2466-1" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. D. Turner;citation_author=P. L. Maurizio;citation_author=W. Valdar;citation_author=B. S. Yandell;citation_author=P. W. Simon;citation_title=Dissecting the genetic architecture of shoot growth in carrot (Daucus carota L.) using a diallel mating design.;citation_pages=411-426;citation_volume=8;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=J. Verhelst;citation_author=E. Parthoens;citation_author=B. Schepens;citation_author=W. Fiers;citation_author=X. Saelens;citation_title=Interferon-inducible protein Mx1 inhibits influenza virus by interfering with functional viral ribonucleoprotein complex assembly.;citation_volume=86;citation_year=2012;citation_pmid=23015724;citation_doi=10.1128/JVI.01682-12" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=K. J. F. Verhoeven;citation_author=J.-L. Jannink;citation_author=L. M. McIntyre;citation_title=Using mating designs to uncover QTL and the genetic architecture of complex traits.;citation_pages=139-149;citation_volume=96;citation_year=2006;citation_issue=2;citation_pmid=16304603;citation_doi=10.1038/sj.hdy.6800763" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. L. Wayne;citation_author=J. Pienaar;citation_author=M. Telonis-Scott;citation_author=L. S. Sylvestre;citation_author=S. V. Nuzhdin;citation_title=Expression of defense genes in Drosophila evolves under a different selective regime from expression of other genes.;citation_pages=1068-1078;citation_volume=65;citation_year=2011;citation_issue=4;citation_pmid=21108635;citation_doi=10.1111/j.1558-5646.2010.01197.x" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=R. W. Williams;citation_author=E. G. Williams;citation_title=Resources for systems genetics.;citation_pages=3-29;citation_volume=1488;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=S. Wright;citation_title=Physiological and evolutionary theories of dominance.;citation_pages=24-53;citation_volume=68;citation_year=1934;citation_doi=10.1086/280521" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Xiao;citation_author=J. Li;citation_author=L. Yuan;citation_author=S. D. Tanksley;citation_title=Dominance is the major genetic basis of heterosis in rice as revealed by QTL analysis using molecular markers.;citation_pages=745-754;citation_volume=140;citation_year=1995;citation_issue=2;citation_pmid=7498751" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=H. Xiong;citation_author=J. Morrison;citation_author=M. T. Ferris;citation_author=L. E. Gralinski;citation_author=A. C. Whitmore;citation_title=Genomic profiling of Collaborative Cross founder mice infected with respiratory viruses reveals novel transcripts and infection-related strain-specific gene and isoform expression.;citation_pages=1429-1444;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Mapping Quantitative Trait Loci Using Multiple Families of Line Crosses;citation_pages=517-524;citation_volume=148;citation_year=1998;citation_issue=1;citation_pmid=9475760" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Yang;citation_author=Y. Ding;citation_author=L. Hutchins;citation_author=J. Szatkiewicz;citation_author=T. Bell;citation_title=A customized and versatile high-density genotyping array for the mouse.;citation_pages=663-666;citation_volume=6;citation_year=2009;citation_issue=9;citation_pmid=19668205;citation_doi=10.1038/nmeth.1359" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Zhang;citation_author=W. Wang;citation_author=W. Valdar;citation_title=Bayesian modeling of haplotype effects in multiparent populations.;citation_volume=198;citation_year=2014;citation_pmid=25236455;citation_doi=10.1534/genetics.114.166249" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Zhou;citation_author=Y. Chen;citation_author=W. Yao;citation_author=C. Zhang;citation_author=W. Xie;citation_title=Genetic composition of yield heterosis in an elite rice hybrid.;citation_volume=109;citation_year=2012;citation_pmid=23019369;citation_doi=10.1073/pnas.1214141109" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Zhu;citation_author=B. Weir;citation_title=Mixed model approaches for diallel analysis based on a bio-model.;citation_pages=233-240;citation_volume=68;citation_year=1996;citation_issue=3;citation_pmid=9062080" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=P. Zimmermann;citation_author=B. Manz;citation_author=O. Haller;citation_author=M. Schwemmle;citation_author=G. Kochs;citation_title=The viral nucleoprotein determines Mx sensitivity of influenza A viruses.;citation_volume=85;citation_year=2011;citation_pmid=21680506;citation_doi=10.1128/JVI.00712-11" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Bayesian Diallel Analysis Reveals Mx1-Dependent and Mx1-Independent Effects on Response to Influenza A Virus in Mice" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Influenza A virus (IAV) is a respiratory pathogen that causes substantial morbidity and mortality during both seasonal and pandemic outbreaks. Infection outcomes in unexposed populations are affected by host genetics, but the host genetic architecture is not well understood. Here, we obtain a broad view of how heritable factors affect a mouse model of response to IAV infection using an 8 × 8 diallel of the eight inbred founder strains of the Collaborative Cross (CC). Expanding on a prior statistical framework for modeling treatment response in diallels, we explore how a range of heritable effects modify acute host response to IAV through 4 d postinfection. Heritable effects in aggregate explained ∼57% of the variance in IAV-induced weight loss. Much of this was attributable to a pattern of additive effects that became more prominent through day 4 postinfection and was consistent with previous reports of antiinfluenza myxovirus resistance 1 ( Mx1 ) polymorphisms segregating between these strains; these additive effects largely recapitulated haplotype effects observed at the Mx1 locus in a previous study of the incipient CC, and are also replicated here in a CC recombinant intercross population. Genetic dominance of protective Mx1 haplotypes was observed to differ by subspecies of origin: relative to the domesticus null Mx1 allele, musculus acts dominantly whereas castaneus acts additively. After controlling for Mx1 , heritable effects, though less distinct, accounted for ∼34% of the phenotypic variance. Implications for future mapping studies are discussed." />
<meta name="og-title" property="og:title" content="Bayesian Diallel Analysis Reveals Mx1-Dependent and Mx1-Independent Effects on Response to Influenza A Virus in Mice" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/427" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Influenza A virus (IAV) is a respiratory pathogen that causes substantial morbidity and mortality during both seasonal and pandemic outbreaks. Infection outcomes in unexposed populations are affected by host genetics, but the host genetic architecture is not well understood. Here, we obtain a broad view of how heritable factors affect a mouse model of response to IAV infection using an 8 × 8 diallel of the eight inbred founder strains of the Collaborative Cross (CC). Expanding on a prior statistical framework for modeling treatment response in diallels, we explore how a range of heritable effects modify acute host response to IAV through 4 d postinfection. Heritable effects in aggregate explained ∼57% of the variance in IAV-induced weight loss. Much of this was attributable to a pattern of additive effects that became more prominent through day 4 postinfection and was consistent with previous reports of antiinfluenza myxovirus resistance 1 ( Mx1 ) polymorphisms segregating between these strains; these additive effects largely recapitulated haplotype effects observed at the Mx1 locus in a previous study of the incipient CC, and are also replicated here in a CC recombinant intercross population. Genetic dominance of protective Mx1 haplotypes was observed to differ by subspecies of origin: relative to the domesticus null Mx1 allele, musculus acts dominantly whereas castaneus acts additively. After controlling for Mx1 , heritable effects, though less distinct, accounted for ∼34% of the phenotypic variance. Implications for future mapping studies are discussed." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Bayesian Diallel Analysis Reveals Mx1-Dependent and Mx1-Independent Effects on Response to Influenza A Virus in Mice | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VPAfsD5X7Ki0_RKNf746DVHhs5RjlhsJ-2qbQv70Qr8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488286","apath":"\/ggg\/8\/2\/427.atom","pisa":"ggg;8\/2\/427","ac":{"\/ggg\/8\/2\/427.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/427.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/427"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/427"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/427":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ypq6Mq7Iit\\\\\\\/zxjBMC2HZiJANEC5\\\\\\\/WL3H\\\\\\\/EwT30q44HAxEwAt87vNObgrogYc+jYNtbpeWmbb9Ndlka+j+rtG9yc0inT+NxTqqzL45uvlOS0ExoKLKDJwQNk6g3I1XjgYqz3YO+AEK+M\\\\\\\/el\\\\\\\/W+VEtTRZcpPiu2a8FyOGKML4g0HhA4Gj0Qj1rXPjxHIiJgKlp8TQln9O9kP63DgjCykN0EsBqGZGGn7scg1H5opTdLvW1Ks7VYs0SNw\\\\\\\/bR+vx1JPRs\\\\\\\/WX+Ur8XdaFPr\\\\\\\/rsAVu8sD7BZCU+0R112aMMw7ywW8R5tcMIiU8c7OSpI5UfFkQ6\\\\\\\/qDH0HyAIqRau0mPkiC+NTwh\\\\\\\/EqhmAqJy0XPOTkC0PKZXV2DF9UHbfaoW5L4skF\\\\\\\/ZeoJ89SSHmhzP6IlgS9eAH4vDOJOiTjCz\\\\\\\/uJDlDuSq1bGIUnzfBxgORz4sDFM1yxqBP7FZ5VLJlMGNgqKD0Tgd0HRaFtwxy679X1BbD3oBsGqfR2xMvDqCCGfEG0faK3qsHQgy\\\\\\\/9FIoSMUMeAjjNWarPckXB+oVRHNYzxdB6dC\\\\\\\/wvvMJYKc2ViRAzt9\\\\\\\/GgXI\\\\\\\/RGwLCemhgt43zO6SMJLpKBj5JGht7EFcQjQNP+aZkuFKrDIBQBixLazvmew5W2sbMMHGtwox5+iDIaqNp5V8wcGaGhXCaYeSBJlk4b7WCiNdyt\\\\\\\/i\\\\\\\/V4\\\\\\\/vIIPSw3Q6mKAC1wN8xbr7XiXGhfJ3hFbrip2WMZn\\\\\\\/E4jugWv9ezQ9jc1mB5b1SbMGMA+zG47gkFo6iThHSKW5yQEJOKBoUlP\\\\\\\/jVpqIT8oan1jZv4TZ2bWduhd4CcBKiD604TbBu4swluqkYiE6tGD\\\\\\\/1OQZzjsVBEqXx1R5vE1Unx\\\\\\\/fP4QOLXcAkfY\\\\\\\/5JqxPVpIzLkdGE+Pgl\\\\\\\/2SmqUjQZcxZrnJEXWjRLveRTyKVTu73LzCNad1yiwgxwWfZ9y16JByn2+TNUTdNox+qSkaXgotqDk3T4HnZ+XSbHmqb4cQfa+Z79Bxnz48nCqKH1+OMzUfv7lUI1SEEX1XW5ij3fqCg0hMAm8S5W0055AhwVmsu4PO+n6s0F5qHXsK2zqAiQkfVgNrBKEIHDyyziSLGIB01Njt+FZsV3xKfaAU7iIKk7yUn3FXCZ36FlT3rmSm75gk2hUL9DDEtVQr8SVNGq50+Jy\\\\\\\/REjfWFjBiEPQw4m3YN95fZ+vCL6jZvwpVZW10usrcnVkfc6VrhzRZD1\\\\\\\/D3jxlMFsCq9JQm47ahdtSjp\\\\\\\/oUdXIUxYn1G7tXiNUOqomFxU9RKJUQv\\\u0022,\\\u0022iv\\\u0022:\\\u0022vvdpMDDHnSg4sW4LaW00Kw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022da3796ab32b50b3310aab29611de90c6fb94a4c9b3f08cce3b45582a2602ad6e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qpM10kkoPZ2rOZItPqLcPC4dEmVJjZ\\\\\\\/XQqvHkqK41\\\\\\\/c8gkgu0Op\\\\\\\/c9viHE3TffvSj8aDnCNZ2SI\\\\\\\/XfGsKWKRNNbfK0MitlcbK2xtucxBqOuOxUY4aHVVWogJBAVoxJBDv4Bgt\\\\\\\/uW8hoAaf8YPE7rUPfOz9V9k0H93g4vrbN1JMy12IQytrv5GAlQQKRUSxf8bvJ91qpc4YX104W4Kb2PMc7DAl+ezzqJsrFxh8UBpsCA9\\\\\\\/czePxcMuHy0R\\\\\\\/vvJ0Xm74O5r3SA1MKQbminoYUuabPPWzwFRy70z5HADn4fsf3yXI087wevGAx0oYDXYs4oGZOsJG3PK1daPzbDR8OafEhTUpCh\\\\\\\/VVB6Eo9xrnyEs5zwKdVFXHu7ZArL5\\\\\\\/Kr2LXyGY1HGmk2BsarZopCWuNT6Rb3IwjG0JkzO2btnGdo176RQh\\\\\\\/b5LRWA5AB0mR4UaTJ5oz6sGozOlN88qqGjlFZnu0jM23WFw+h+zeHcXPKlPpx9PlcOVkasLJa\\\\\\\/bRsHxnOSqYvfpdo9Z6e1BbudkbFxA41ZjvC3J+m2HvBPd\\\\\\\/N\\\\\\\/2MHxJzvsUPZQCDR+8wZhrAxH1Qpp6LUpneH1Ja\\\\\\\/LMaK+BHmdES\\\\\\\/0laS9DEdyzMDCuJhIO1OSU4VZSzTMeZiLWwrOe+FDBptLy2G5m3tft2qc1yvw1rClQEJg2Vq2I9O76qLHxwgNZ67FsV5MqDH\\\\\\\/BxenlTwQRJu06iNiLzYyaarsRfHo8+1syHsxxqmX4gborSEEspGs\\\\\\\/PUnsR39P9gIOi++ZvvZQw6uzMo8ZVP8u1LIVBBTtGPKb8cAs39Gzp65+wwPXC2pNvsP9tQUAwAHprcEJmYbg4jtFSIS\\\\\\\/3LUvAVpKYcdYhjG27Rk5YGj8nlJ52Mi6kuur8OLQfZHevriqKivhyW3+mMgpTnaUs0hUjVRnzrsFQzylwFTNUx4HXS1YqlWiNSltzCuiCwjT2gOui03036qSYy1QNgGnnPALZdUn8wNAzCokD2HRDLIlx6x2u11w3lJgxiUkQ3YKPxjGzqjykhedWjZHJXic6v5kloGLmp+sV\\\\\\\/9ZrNEfoO6GfyptFaIw8io2i+zLJL8Fb6EWeA469WK63A3J6QdUlVD2z62AzEV4Gv+sTXQH\\\\\\\/VzntO+cLv\\\\\\\/Rf\\\\\\\/utfn1he0Aat7RlRgQx9ruL2OQgdUVDXxo55RMDJMNEjR7tTjnv9Q6HuLrtH5CDR6j1sLXLYwjhs3CWtsvY1oJUc0uS9ZxAfaDhZ1Pf7lRYbEHYUEUcoM2pJzfA45c8lOwwJ4dAW\\\\\\\/64Zh+WpdYowqdeLCLDZmYBU2DkSQ4wPVSw7NiOQAJe9XUnEYrrtRGDrpegl1HnwJqQXcagwRc1kXWgttIS+8nZU9kbjGZrtBvxUe8d85ovmU06D2l8JEzG76vfo\\\\\\\/RXV8RovO4TWdpr5pZ+uLuRwzsxCoS1gUXmRTFHfazmDUMV5ueAZHYxyV0oLqD9mmprY9SbmPPj22Whsr56nVDllyEl+FH1ODDuzSWDu7u8ioWw\\\\\\\/Ku4mFBZNQuaKaRZ1tWPLlATkhGR8YCb0yAWZ\\\\\\\/6pijuh1DTUIjHc52n56GJ+WurbbQb8YgktqwK0vsPzUje5xdQOLPCJVkjHHlA7RtLD\\\\\\\/Qp3geb8oujyTOobPhiRQElEgdCH5GTU\\\\\\\/5SbmqAgprebwgti7lABBeGrYVWTG8GfKg8oCvG+Xik1v6ShmmHorweytmpSQhalZ9TgvEDdFFSa5epTASEBVUom9ampVNMHkU+RxfXKZ7PZjgPzQJdQUFqBPyahrr2T3zT+0fPj21BrKyWbBJQuwJEocie56+DpIg902ZYvD\\\\\\\/CftuKrf\\\\\\\/ZYyKnqP6e2ayhph7GMGJtzEO2ryJCP7pNGCEwcFzbDRbGdga4klp2l5Y2bBJ1WmCI3KKdUmMTfMe5FcgSOLka+OQLli9ZIJ5aVmM5akYbP8vhqnLxzAZ86m5rIl5IP1h9mAutmbZEBORhSB3D3bAHJUwwITJfRNPJWm0mqgkhr94k2Y4EZPGIIlEbkiriTRqE8usoMBQSR3Zv4UP0wcXTQpeUQfKNHFMKiDug86C+o2lgilDoqUb5DxR9GLWNuw8+fimhOjGlVhjEJ19ORFgTcKtiQoQM5Ndi22vjWZLdWyKpkLttuGYB8wrum1UKEXe\\\\\\\/Z4jyQ2\\\\\\\/2c+MwmIozI7BPcSnez0InV4Jz1KhV9NwWivZO6BIbrlUOF4bqBZpeVWFSmXnrOUim1MO5aCgdThGgkFNVHPl7q3gEMrZDUQehf51lDAsYUeu3uz9BeyWqhmVi+gj6CB7vOs9hPnwgilzu7LLjQurQoQoxlK3MtbrRiw5ZEBuotDyw7Dl+1H3AqMBJkiW0riMgV01\\\\\\\/48xcvOsRytqaVaJrhAx2pie\\\\\\\/4Z9vhrqOBdNn8N96VO9I5KxBGX0vftw6+mlPbnGq7itI0RSlR6KGyWSewRJWj\\\\\\\/bntu3AEaygFYzx4xLxZFy5grW\\\\\\\/7DRR5VOF+szrK6sI83MTvwdUE6+XarmbxLVRb7np23pjLihX+2oG7ZdI+hN3WIJf\\\\\\\/h09dgjKCImYzGgUvprv3yrIxJ\\\\\\\/E8c9Eq4bYExa38BUEk3qovCdp8yAA\\\\\\\/OlUGLY\\\\\\\/B4f1MC01HZOWy9NpGWdv2Rajb8imyGhm\\\\\\\/WOvgebtsC0r\\\\\\\/rQbc2Ke0k+q6LsTBhpDuXIdFL8LTqUvwnMuth6HnGU8wFWZpgHKwJVZ6CyqUa1NvkcoM5FKO5oGgVHvkRX+7Q1tjZwKCgn480IMYbWxvbexNEsc9Sb8v+\\\\\\\/LPwFXLxrKjm7riH+X2Ane9AdxyhjjCvvOjnaY0Gnoot+DDa4RL+BCjJnvBLrrrXSg7+cew2yyeH7jEo\\\\\\\/KPEjU4JsJ5yKbKNLAren\\\\\\\/FKcTaPJC9YMJF3tmH87UbA54hb3lzVejJwBxgesTPQirXrFtj21fho1M0DEcV1WCHb3U1GC\\\\\\\/W4vMxpCNu07Yn37zI\\\\\\\/A8C9rzDxnzoanXPVIQu88tsI7lz\\\\\\\/VspqLeNMzN\\\\\\\/OV+GVnPUiKZLY4\\\\\\\/1qRe2u8nP5UOKPYnrv5+CB\\\\\\\/tbI25jRSk0s68U\\\\\\\/kttinkv7e6W10LvcXPfs0AvLCiGdO2NcC\\\\\\\/lKezszGv0FJezWnKFUMfa\\\\\\\/B5ybVLtyMoo1fzdYDkUHVHzVnzIdTBrCjZp+vIhPUzPdBpOLzRAlmQ7Wlhp7LbYSOSFKWu+P+MlDNnhwUsOGi+xdDM4wzsTsPs3Jy+3Cs6YWfZ7hMBwm2ix0pd8w0ohx7GtHGTv9QgNU5mWqdRq2lYEG\\\\\\\/DOQ4EclDF16OABffG+kD0wH0dJKdxvfjjCD6agwe5EV1gQCqUOx7Fng8WZH1g4W5SCceHr+1nB7ci7MIf0u1Avvu4lvf+95wjRPeEk+YdANkOvIli3FCQImlUhCVp8jMArWlEV8cHvtZ8RoC8VQzeXbQGKgKQOzf75B9Mkkay6XzHcPlD0nLPnPaUZGDutMEwot6vWFpDkzzOk4QTB3CoZNDaDeaG6fnqTp6QQs4UtCvEnStDm\\\\\\\/qD3owQKTfbC7lMp6s\\\\\\\/ZWeAuu6nfZa187g+qvmxcb4FrfOcLTV8sfPmpXBJGn0H7GpzLHzWz8F1dL+ZliHsMAyDWVFLuTEp3cNdQ1i7j8nL\\\\\\\/9zaDvFCfrroSsmhKws7PDdflpaINx3QcvX3Oth\\\\\\\/QGQ5EZuwQKdHEVGxcBX2EpYbyRL3rLlRNB53TWUfx4YkcK7Nawi0b0Qr45PoBixHPprVQQH4Bdf+jZjXepmBhOC6\\\\\\\/aFaqllyzWjFCC530638kG0xL8J54Z24\\\\\\\/IHcyQGgpu9C6TdH++3ytVpeSVhLkKTUfYRMjqeXuChEvIEyQ9nIBv39HXsU\\\\\\\/sIYGlM3Skh26fONME52Ll1UYFXdnhRO+f6eQXRFR39Ip9VshzBr3fd6zLS0NMtpVWSfusXbHCV\\\\\\\/ovyt5y1f2P+OVEheTy6VHVKxwhVRzrkRMbJJi2s\\\\\\\/t3iqsiLyAXv7\\\\\\\/bihbFFu6TSD30DQkIKzfDTxpx90Tpvqs5GSbLa4+pZ1Zt6Z6NYzDXbU5qNqiVYj3wdTfT7BVJcvoaw4X5FUkV65TR4PYkvWPmNbFG0ipFL124ggzEb6fZ+wMDOJFSsqD5\\\\\\\/SvOpOEC0nJsuwZn1RVbjjLI6vLfkzVTw69ipk06BABhHejqw8LqMSTHIKROay0D3hxm1qeBdZmyPjxP8X2G\\\\\\\/OnWIDZ5QrzBVt0j47W1dlyBdgMLMinQPb65a5mzntrNF52oZIFcQ8f7dfJDINfYWcBsRvyqP71j66fpo7tCUMU7vEX8OqEbsAw+L79NrDi5iSqm3nQQLGCeVqK4AEGUJ5zlOTEM6K1j6D0Xic9Vwinn4Z3lZP\\\\\\\/+Ch1CbDLD71RBYgYvHQoP5CA+UaHvSw5Wo4ge5vJ0rr7HG8wz7jg8H1ln66NKzh2fJsPBDEW7BnLBFHj+hjkdzS25iKgZYTgl5rAv6EPde8TQpVKqqQEikiESG+ouYT0IMyJ+KasYXRmOr4Lf3IsXCiFgvuadtuPQ5aMZ0yd3wtGfQpJEiwSJUh3q2MnYMM88RFnQkjGapl5Xxe+s3L\\\\\\\/ZW9VV9jUlI1\\\\\\\/XC8pFjreoT5X6HGL1Etai7vNCvjDptBkpjxdaYrAkBmaKy2KmIPw3C37BU\\\\\\\/e92++Zj\\\\\\\/Uv4zP4qMfzXB\\\\\\\/gVD1vIN8HG+jSQJgssyZXX2HhFS5VuF2yrncelgIMIXPr7WaLB4NRk1S+4m4AeisEK+xCCr5y7QlC8YN6PmpcLX4RShasn5t7QDMbPNQN2XE1v5TJrH3DAwYqMmM6OVZYoep\\\\\\\/G4PbKzAPSoNpUUBZDkgIySo6aRY0zd2dpK20I8Ri6RK\\\\\\\/WGZ\\\\\\\/cQqJTiH7LDiFUs9dlCOdQvfeE7Zkqu383JAphz4Fj0g1LlWN0MXKYSG\\\\\\\/uPnDBMHVGvcLpPaq8E35D91ZCQVeW60XN6WsTZLclCTvcAmJ5f+LLg7\\\\\\\/1Pmkq5LvBSVs4R3LftjB+RfyE2LLOQQ1n5glJzrJbSBZyhNPoQp1D6T6raZWLbuqlQO0AP\\\\\\\/MordJjigD+xlCdnZfoB4uMjAM8SyuZGp3STed+ra3iQSUD0hdgdReIocrFJPHB\\\\\\\/6JU9eSuuajVtazlkfXbYAp9lo8o4PPgcBKmz6IfA5ZObOxVJjDRYwPYBFWVgO1MRkF1NuWB3H4+FgpDcuLlHdLgsqXIFlBWOrdEjZrdU619cdxLlQLoaelpgh8e4t2Q2z44rj5KAt1+SLpZVu2uvbyE8m6CjruSak2E789AgaRzy\\\\\\\/gaEMog1emhSNn+mIH1pu0nc6uWPLOPs\\\\\\\/AGcOcoKjZM9fWPq44TBgbDslPD5fs+gvbITuaOjxmLpW\\\\\\\/3vMOKL62UfdvDd74zCA+uizMoyuMdROS6vo0t188MT4tTDpB9eSEHuWSQMLCAfaugzkBhde2Ymm2Efjosoflw2IA92E2X3MHGXjeZ0HXXHL3dNZr2DIORb6k\\\\\\\/RRvii+nnXfCdoJIWAFC2O68MV8CBNcYzNQ3\\\\\\\/36HIuuyyvu96+rAd4OZ1qekLcl\\\\\\\/Pl3Up2Q8HhOMlCikhNsKeJj+zalS4dX11EUK8HTU\\\\\\\/jrRIR9IOMzhR4f+F\\\\\\\/mynhB5+aG9Xrb4GR0jKM4VRuxVdNbs\\\\\\\/5t5P1gpCJ6vzsczFdr4rg8qKuJpVBcnCu3xnfy9U+5epf2mb3fLR8sCCiK7i7ZDyHI\\\\\\\/Zk5YNTxEksg+HeanCdWxw3U7DrTS0z6OGLa+Rz6cKly5W+GaraZunsjnaCJR+LU9LT2cGTuzGgXgIJxMJQLmI2Mv0gWyP9+aMs3OWosH5mXOlsCEwb01XMgKDoka8iBSKVP4rMjgY5A0cj8zQu4JQR8I0dAQe1Pqo9O5FUPP4pdpz3QH5XbeI+Xy959Ckb3W+T42MSmwiph0F3sPOwkchIu7ZAR5UzEC1WGg7LxVV28MUFtpaYwpTZ0I8aXnWBC3f+GT4IhUZMmukiruhHBL66Ju3LQMaeQhifC6xq\\\\\\\/C1aFcVtCmTRHdPJ\\\\\\\/F7Ct1iDVuyJul5N0ZcIx\\\\\\\/mwEp1na+LXNbqApfFCgO7WbkYc7t7+UOK1jF57wSH5DyFMWb7L1s5i+AdUkmfFD3JtxD7MRKqBbFJOvE7uaLxaVYJHXBZy0wj3uz48Moblue2bSREP1rsJmt9RTMO+t5qfSAKHwzkLvH1VgfISYdNXM8YQ6yAkDDjNUuFosYPgDGfkvx1EspmB6ucvHDd8ZhgcjuSP4ZtcyA7Yk6dXJRnSirvhGM9XcpQ\\\\\\\/P2ZAFFI6bKv3FMqnvBpHpgltIEU\\\\\\\/L3fJdFqKdajWcUqus0UZfTQPcT14ZmLIA2PET5TFje9168XRIjIBvwQROwlyBJ1hSheQroUk4o\\\\\\\/wNo\\\\\\\/26ZWIZKUYtBJS972yXMFU887FVHgiAF1JYXT\\\\\\\/K7Cobcvk0r8d4Z42LKn2HmWKOL1zXHxva02p1BnVieqazcdgsgOvk07l9Fx\\\\\\\/DcEVDxS\\\\\\\/Scxkb6Rg33HfkcabbczBy0h1i4GM2th2CbA8ZM2q9phswk+BP+nBW7aYLHFKfyxIZEIRcX\\\\\\\/WgSFwtWsmJv49hbhf+FooebUk3qqqqW8TAJrdZkGgnhExA1uOhUS+85bqJ1xyOUJxzo4Cf\\\\\\\/muNuRX4TEF7AqMViTusiNz3hgDNucDBhV4rTqbbeWyLmZFYW2H1HdoLXc0CSrMEeuhkwwIPKaqhyL\\\\\\\/xmMqV\\\\\\\/\\\\\\\/MXSDXIEtIw3bBfWC9Tn1Q+6hSbU3hy\\\\\\\/nO2wLCE+WbOWs80LVXg1HUDBzzOSMgAvwvWUfVowiVwno+2OGmUm48tGzDN+c\\\\\\\/fuV213XbrTSgH3o4iIqOLW\\\\\\\/FLdHA2xsy+mSpoA9RcbT9Umo+4ZFSz2KMddvWGPYMCOkCk6cvFOQwiQlLl38ek785PG\\\\\\\/zPE\\\\\\\/mTrNel4TRPLe8R6v2K4rDWUEdU2fkPxnw9Rh9cMkiqdwS1N+U\\\\\\\/6niK1XJv15ETc7U5HL9yqDigQoVYeocdET\\\\\\\/SN+QAQEBjVb\\\\\\\/LOdD2UM9xAqnLzgOlNQNevmB4mmyjNwdAJ\\\\\\\/mlKV\\\\\\\/GigQb1casaoh+dDT2KMvgj8rl1zURWci5uUYOXyvlsi23PLMF17OYpDM\\\\\\\/eg6RDWxWp3S0wQ8d34om0hauGNgWJVZupvkiXw5y8KQLJGwQx21mDSk\\\\\\\/PrskdyTpuf2ua4kDvFgybfgYig2jlpo838jGwPH1gdhksprmy4oxdMJELGdLjfua5Gg2SArMKAv10itvtKniNmnNx4DW9vgNxYtEY8M6qL9rGaGgOOn+p5wqYETZOeDOWDhx0FxZOmzb3NiPrejTOSLd6GymIumzdJTuH8iANk+rIkxzwNgYFXTUMJlCrai0eA3OJv3IWAKmB6kprSvsAqi47v1zRekhB7FdE1Gwf8E97JyRaTSN4xX+y3F05KQGfrmaP0u2BJFuICs3Z61Lvmf8+rXvwAArZ2fFulO+gr1ZUgsZhBGc0KbCVdSisoeONr2TnrcgGhE+f\\\\\\\/UoDsuWqa2rxcuCWdhUBAS6LVBlhZW+wWs0RIJ75eqKCl44tovXv8n6OVNSCPJbp25\\\\\\\/RqgyhuS4jRlOpBk46ZNG3IyX3wE68fb+qdZtuEVlvtSt\\\\\\\/96EjiiiFalbTqlII8bsKDkkWNsiFsiPB68AfdGj06aPtACfWoYoGS6OkQuiXRLnJamSCyrFzup5pzTAkSQlCOU5QqbWALrQmdieb5TcVh1RhK4HSJWmA7jNf+4Lqu2iq6Ze0GHYZAuVn3mffq95ydGiwWHCDmV+MVa965h+x9e4oUe14DBjcmGmHEQ9O0M8aQZrzTp2W6FAH1am\\\\\\\/WOnkB96+GV+hK5bKyPHZyH3X8GzMkict5BJK94Yq6FOowGRrHvLhnkQaXxpBaOu0S6z6PUYSeA+g5jadFoMVCr8lJBNA7l9AZPlsWRQ3P16jM7wp6hjF5thbRn4Gy1WkkBTuvcCRsB+hSdf+w88uFlcPhZUXbuLxQnyqYHLTG5MkevvJHbpySewlYEIFY9dj9eQGriuNHii+4P+qTzUBZNddqHTimPJzzG8YZxRivfvx0IUXl\\\\\\\/4DoUG6nEYqjuA0b0H1zC8OCjI5VI3N7mUQzrYNneDpf\\\\\\\/v6Wn2R5wwMmvkRkNvdEUCD6YR6A5pQxkz3IbQpu78Qgm3gMRtZ7pv5+24AtDVaVU1DoKLDpmnrQtenY8i8TQdxjjGD+UL+M+ah6ltIiA60VhXZfyTbUSYepsBA5U2vs99tEkLdi4nSud7bOlMFsrmXyrHmoEnu2F9IQAWh44VkIKw6PIbBR4VIikC17r+oEyCuT1egdzt8q6uW4LBMCwToQfTDu0a7iqg9lVx5ypeZyIjIbwNLZAQhC4bqtEMbFoxo3bjE49vZNWMPtB46Oy\\\\\\\/oeBl4tYJh92vc1M6u76IFp8KnZbRJFGRAntmcfTlXfH1SLyrf+oswrEeV7qL8FVDoDffMLdeh+fho9DUkkGxasKxYJY+aDIbagHDQidI7a99Qyx0raMh58HbWJ6sFbcAh4wKid0FyuaMKgAULVq6fKeKrZPU9k0KzI0sXgn05GKHh\\\\\\\/imd3VKRx75lWvHjesTciuZlDfHhD6iHjvuovQ9eok4U5Vaqksshc\\\\\\\/Vuhnj995xvRQl4qd46b74t6HPK5HO1mUQF\\\\\\\/JjIV+JC1dtlslhhb8CvFDSoFnGn9MC6yhY6ZHEMiQQtyLfzRRBuO8ekHnjoEUeFTpPwy+Kpjfty1Xe3rFQ5NhJoIv+sgitGRmD44JMeYNkDLbwkCLZ8qiSewpiEEBlJIUGW4B6wt3uEK3Bw4h6CErFmpSDbz88KD4dCMlycRfn7pdLNMjkFQUPkAdep3vUyN1ONaNqwYVWrbX2VYbAhDcYXJB5+v\\\\\\\/PyKKsNMyrPY7wP6xRoreL0MOXQvQT+cYetY8bOC+wG+LfHdpXcCctGxpSZhW4+WcsnDWEKUu6EUTtnSCg\\\\\\\/WRwDB9fzHUoen2t4hehIgjM96kZ6qQ+gwS3lTcfqS5K\\\\\\\/493pMQpUOchOt\\\\\\\/7b3H1SKjND2ospB1X9\\\\\\\/35\\\\\\\/S2cDBpnMJIJf5RYnsO9jwSeiy23rip0EsynGv7HXMMOFV3n7Dqa862WNbo4ca\\\\\\\/Zs3LoQxtn1dyzs69gsoZbZKVo9XTQ14VKxks+LkXTtbjKD\\\\\\\/y7m3I5qwEQeQySB11f9qjfx81eJ4x\\\\\\\/BAuBHkScLzGdDcfn1gbr\\\\\\\/a32o\\\\\\\/XGkF+vMASZienUcI4xzHkzEtyMMZ2jiiHbvVMPOZJQhTLDOVC2MljxmN6NgO9DtaLeNcMWgEKldacEMKb8\\\\\\\/1qw53KKcESelyqKrrkDM57oaiZRPHV7R3TSH61cC0LHjehBZfM88I8HMFboH2WucxmJgcQe5+bQh+RCnwTjoUoT\\\\\\\/ykhrMSwhWodBxxLaVCwts6W6+31Gdniu3Z8ksQQZcaVMYJ0MyZiPZxtcq2EFGV8ITszixTzSa3sdjll4zuH+4enE8pIbXeJv80qMKR+uEWs86hqntaFqiMPeM9k1kZrygROG7pwxfVP\\\\\\\/xTsftX2x2J02qJx5oP6hr0NG2c04vgKa7f5GDpTNdMsJrmJZhQptx75FYmnxcmYWCvXBsvE\\\\\\\/VOoxactqS4gtE6EnNsrOZ3YP3TLstW9XQ1AKJYLw3HBYYxCHnPG1iLHS2AMCJJOQGRLplVxzKSUm2WK1jcCeSr4j4DjxImcTNyHgIwPpuT05Pu8GSRm1ANoYeIL4l1oZB9KlHCZ26XkZFyjlQ2qJmCRikmGYnajoaSYKUdy9FhE9W7v9YzL7OQFB62oTk3VO\\\\\\\/t0+6DlLkeDPtbV1K3UXsBhr2qglyJuuWQFSsgEkNvzq\\\\\\\/RigwxTTI\\\\\\\/BAtj7o5jK15tDfnrc9m8g9r6KWVDwHoR3UK8q3Fp7Vq1RYKKi4tbDRjuvzDX+x7\\\\\\\/2OyPWB1SmvI\\\\\\\/TIt2j1lB5vVGdv1vcisp3f3vIbNy9ZvXezyJWDUtFdCistBg9WC6c0t+k6cRsb3sv1tBaeNrOvAu\\\\\\\/JbGcr0b1f4WcbVadsmmqI5yvLgLmVRw5MnnD4\\\\\\\/iKGtQqdEHkyAJEIwUq5wU7ihPTjIkeW0X0TZmAQL1+XO+0zmbm2RIDtRFdbbY74fEp++ud3R+vTTkIMP7RjFrNuTJzi9TVRQrPm45ZLoBZvju6kE+i81FJZh\\\\\\\/P9zhR4yBX9SG7+\\\\\\\/S\\\\\\\/Vp3ENr3VEQQ7BAV\\\\\\\/x5lYMAZI3ZJEV7RpRTEnkjDgTCn9eF5jxm+PP308kMSfKZYrig9S9LtzllCzYYHTt25e4QahfGYGTMHPnbReDlRCGvtgUxjfxBEZ14JGo54A0+L9RW4OAeqi2NRK1XI42a2xzuf9WAOLwCY2zNVw284RAUZ75ybTD2VGeJGEfEzjyPYRX9pGATu88ovrctufwBnqNGIVxJJRblIld88NdOoSltlJbj09kUUhzNnXs5UfyuaP1WpqDvdYVS0dswLTOctip+dWIWsj2llerTdq2srVDwO5sp6JM70foqKzurgdn6tplkY2cgJQiCFx9erJ\\\\\\\/IBQQ5ogZgahadvSt\\\\\\\/kHA\\\\\\\/w1mUG17J5+ZxR4HnX2WPu52pLbzGl0UTGH6sl9DqSqJMc28nMlS5Mdh4Urot9EghB9pS\\\\\\\/s3GcGLBqbwKhPAqwwdkivhHUDPJ5Hplh7X6t+5ExUddQEvPf3fzwbsSBlK+JgrCk\\\\\\\/MV3NjA\\\\\\\/VWwpXURDgz9gzNarTM8uEtoo8eoM9B3qfki\\\\\\\/4QeW7z16WF5yWi+a8Er6fqfdAOI5\\\\\\\/OgBf6MydtscHEBS0QZgGDawos+KQmn2o1oV6qMn6Be5ZApb+HViQ01QQGXhukfVnkenJSJrNynlBbWyveTGkntyeV\\\\\\\/18BaniztdP7M8wFj\\\\\\\/AFcwhvNpCcU7c0dw71o5RTW9CyExFLqEGAbkD0h1d9Jfh18SUG9lnmvsoS8AiCyVbd0mmA0m1BbRS1ZxQFi9G4dMEKOWrBrkt+LXfsmzxlvLhtLyRUYwfB2F\\\\\\\/8T\\\\\\\/pm3MPapMsa\\\\\\\/wjSyYQqOmBnxbTimZDRFnUzIhho+rOxC9WSTA3Hmz9vJgpn9GeFRDscp78QkhZycy94qmITtfv8+BypJP+8yRRhn\\\\\\\/ZD3EpCj006WyEQILxaiT1a8n16G+r4U0n4nm14DYDpXso3io\\\\\\\/z+4VB3uLgkSejIz32m6ljylwf0XppAy2RJd0BetdVGCA4UKSfW1dm+3bIyCvgwq1x5j71b78f8rC2ugmPdQSj+YXOcyG+HUxu7nM3PK1VzhaF3\\\\\\\/zT5zDwbM\\\\\\\/xqII7CKZNRd7nvrrTvUEaekVuTtNLPNpcJjczPuL8cVOm9uf0HrK9rbgqZwlG8Oc\\\\\\\/K13RY9zLLCpRPD50sWS5Ew89XF9BsjQMjrHc55uxK6aczvLyRmaMXIqsdWtlx1c56AshpDpxphQjtt+VK2polejKpMfRiEqzpyRYFP5Yf4+phBLX+Y+4+maRuWgJCCFnKEU4\\\\\\\/8MYs7Ks5DaKEkpvKvWAiZ4zg6LCyATc3+vWLyV\\\\\\\/4oIv3zHKigwuk+hbbfaxuAN5YGxNR3RoJoXY0uLDWHp1fEt47tmltsfATVyPHpvIrI4TJV5vaP\\\\\\\/nw2zI22EQh9+1Zttmzyb\\\\\\\/CHdQNR8xG+gNQxETklscLVHwCic5IUzGhL2vu0eyWP41vvDNmQLilsqrfPSG312i9dW962wQzdVP0CSQHLO7HRjiT\\\\\\\/CIrQyomV7d9V3DNEcV6h1eNYLKVCOgmx4coLil62JmNr0lxtrg8xyzyeddVd3AaqQiSzrTCK+kktvqpcQme67ejhMKLgK3RZgYuJUfmErePBGxs8iK2Zy0fBiGQ\\\\\\\/qh\\\\\\\/V0WQmOuyJ+lddZT9sz\\\\\\\/\\\\\\\/R4tesTZyZuMfjRSYlH0KUckggwRqHYPE2MXlM4Y8hAaaoB8h5vh+MkMs5pq13WqhsBfaSTwQWWbar3oeuj31gKnwPHHSNs44BM5vf6\\\\\\\/mb5nwr0FfbDttCy6gSljpaBMQfyatv7s9fdlwm+HbGhJUMfBHkRwpC3lCwQ\\\\\\\/yoQsRMk5M9ELPCIj4W12DQ+QgnXxJrsOHs101lvP99Ru39y4mQG8uoHBXeHtKh4N6hOUxziM\\\\\\\/CVazZvgD8Gaka0G\\\\\\\/0RFmrH0nk+tGFL1oS6ulCaaa2jEAHsVcHAI8RJrJuyQkBfNW6hxWt0skcy76sldn42KUH9jimGuYLOxwyM0AlYdqMrAlkdCssf1qz0P+yKtAOQcd7xCC9jrD6YlM+iJdac1KRuyOAbK0RvkoTy2mBJenNUR\\\\\\\/7mrAnFn55uVIPgiauKH35E5TfPe+y5v4iLN\\\\\\\/fWY9KJVRKGI3d53RSzMDhtM1P38EiNE2LZeYwhNBhUlbNDLbZvUdx4dC3\\\\\\\/sab\\\\\\\/4cwgJwu0byf+peJ1qNdfjLn4zCi7JKfs3AfyPYhJ3lbNOUB+3UHFJSW+pmHiRgyNhCBjhpBBNktcelgzfHHAN8rQs7iZz6zLbPBwk5OoW39e+htn4+HqchFigINk9DY68DRPyiXUNNN5iakf2BYQJRYhlavZn2RHQxdXLl42kkJt30kfTqUrffhrctzApK6CQmuMuUXUwlFFYCbbdOVmA\\\\\\\/Ddj8BTOaWpHyyAgdBYlmsRKOK4PDNsmHEJwyfFFczlFQfS3Se1zLMdeSnaPeakXuzPf0Z\\\\\\\/KvIVtdKcdsvWpUBsCD4\\\\\\\/RXRDNrrnH52SNUzzqoigvui2beoxEIibrRoxSBhD104ALwAqJRb5e0WjFkKB9WTDI42t1bJj0M+DL8Htsqate2Zcj\\\\\\\/1G3I4rOZ8Q0EBzQjHJQECQm9VHnAcIuPwUiNmkadeyHR0yqwpotdIbla+abZZ71sQUYIHF5avTkr7ZC7zWAW7cRVw1i3PgngRVsAv7BrIOrNY4SA0VzxjMJxKaIc6cTS7d005XEOMmhy21oiyPDq2IXaWGaE32s0hUPJoAmdW481zJV0ZwN+Ymtw5mKqhf7cmPuXFUkkMBpa4\\\\\\\/GtKexrPx3MaYEvQyx\\\\\\\/Ifh9PUmXwTHgt+3kUbpzP6lSZovMGK58NTez29Y4SwzRKUkLY6FC0Bb4FZjaAd+Pp+T2hWhPkllAUZut+3Re\\\\\\\/ZzQcv7xaJZ8f3gj5AoRsUv67LN2cn9tte7l\\\\\\\/touAc5Q3Z\\\\\\\/4dndNEfVauUqznCYepG0kr0kVtgA7wO9XStvf9Bq+EKLO1IdqM2V+lq3Rm2\\\\\\\/LAB\\\\\\\/ug8vthCiqnD2\\\\\\\/wsZDJAMnSQhrmdhLt+miFYQQ3YJi7FYLE08GNCIjoGAWtA0cH3tQfWHH+ou1ZE7LrnfSzX1AM\\\\\\\/TSizAP8cDS08AekH6ldWSIjOUNoWhCQMQORmK8MRNC24JP5xbNx2DP8Vw2qcsDeD+bV3PepHVP5GtY+qz4ed3rn5eyQpffFesShPWCPM1wskZZDhsoG7zI80fpbrMCFAnHV6brwAxafCQ+f6AcCwUsg8+MUx9f+44LvwiRkoKvObrcbCNXoD72mZCX53kPgXt25XdRcIvR+bJ6VabRqsWSreIGDrKIRfFfVoJTEN+tks6MXM7+6YndmFt\\\\\\\/Xnp9nyXLq9cIFi3doiYe5qyuX568u+kZWqMn3GcVV8HeoRU9mPu\\\\\\\/RZNDpPatQI9SsBqZvV6y0JwFpTlegl0LVxYZDKwhsABFuI2GWGdyX6xBuCfiD6sW18d9hkVYQWfkrhdP3goLRlQ14AIOrAQW6FjwD49CNXT9G2Qj5JRSuZfjgHh9dBoNkxHiR723WsBCbf6broNWiV4TZk0\\\\\\\/4RtDMdSowZZhLTX6IlY9wrI8fMaJJt74LJhONy+L+jv+qK60WJF8Tn\\\\\\\/lmTnN60jz1gXlNsTuYtn5is1z1jqQtIU79im36x9Gzo0tkcv0KXa4PAbDIbi+1eULhnffQTNKFUAwhHi2PzKE6vMp87sij3q6n02x9pjpH0jh\\\\\\\/eoy\\\\\\\/L+qEHBars9jL4wScQwQOhV8PW4q2l1lriNxp2ga8meo8Lecmpo40OBqHYTCu\\\\\\\/xrzKfzzlxSm9rkmfF1YsQLMVQnDaW9Va7NIISpc+9Z+5yuAotK29muJr\\\\\\\/sJAiIn8Jww3R3pD0UIyAJmhASW02fK+PxM6\\\\\\\/FJXgzCn5gwJCFICcZDMjO9JnKrlQbw15bXjg2EQSpVVEkW85gwumn\\\\\\\/sUBQ6iNKe804BL7PyyJqQMxbnmjTLJErm\\\\\\\/MMovcPHO8knAGwhAnM9036osEBh6TrcihfZQqvLJyCH6bgvi29FZmq41MS7FduAU\\\\\\\/q5NOSeV0j7nGxEICfeSECA6A1kCCFmvT3ocNWd1UG9jAz4BBoxFzORX1BTGaI6AQ\\\\\\\/eauzt\\\\\\\/ATtZYQ7ReAKHwaFD1BpttBxa\\\\\\\/BKSkm\\\\\\\/I4+\\\\\\\/Lci3VfHvYGoyjtToZ9Qqxp9zYAdWLyLh2YpbfooVpUcmTIwhnurrtIjdoouN\\\\\\\/3+FdWh45bB586taCkuHRIjLz3W7gXkekYX7MnAwxF4t1qDAPAWyCLq3Vi07Ez8u5k8jLgcUB8o0mMTCrHEQ2hnLjIm7WS\\\\\\\/WYf0bmNgLJBb0wmzn4Cb3uqLm3\\\\\\\/Sf5oSk6sBOIXnChnygbAWzsAVx08aNnAzZ47TI6IWarrE6Z9wtRrpHolCUu726Y3c0wn5MmmHUEGKiLq8r+F0P0Y2J5Nh2IQL5nwgOU7bQhyVW\\\\\\\/nAUGWRi64XrWdVjJfYEZRzSmdT1G2\\\\\\\/bz1wmKePjpSwEXW6TBqyekXNZUTBZTKCGLil+8tsiQ8NJeF70dIZ27zk5RkmjcIXnjKX8LhE0Su8Ukzkno9AxhtmXxWh8MiN26DmgUrE3lb3DcRt0uB5ePEzGLQwNKDL0CTfY+XIYq6O0L3SYlG3Dn2ZFPC9QbvIk+WoGqowhtGPFwxIHTN0I83iECPaNrdRczqHcAuOF5OsxXzlNOAgzWBPJPhXuXnWmoQe85lOrE0Dr4gvy3BNi1C1\\\\\\\/NnXq24+nPY4bQ0fsNc6ZqT6PYklurlusLWHSkJxLlBXsf5U8AAc4bgDI54tUqdBYRnzr7\\\\\\\/peFu0q87xvq4jyyqL4SwsbTWQyZDrMBNYSbKzm87oP+jlVeEIaxyrLtx0f4YD+WKlorQC84CXndmOXQ1lO83F\\\\\\\/+bm6toj5rX6CapZIaO3rU5hU25kSh6DI4xR9WV9Rh1buJKmyi6q4rYSfPjcHqrUX7CaZuJcVTx\\\\\\\/HufD7I6F8auVdMpEECn4fObex4mw0nGnCto9QM7w3qsUizAlseAYvwM7pi\\\\\\\/cW9qc7XbPT2qjqtTke292LjCfY7D2xlaFenyYfAjQLDbj6EAWexokV6yFz4CXP+RAgw1aVK7OeUDiVlbYASxzcXEFP+KM9aqNUq0Ka0B6by0TWRH2SRw7LRqWVxXFli4keDAV097FW4UWX7ylkJ9yPFjuCF86dmE7Z9xnAEOeu0meeX42FAwf4Oa96zG\\\\\\\/EMjmytBqjyT7emSzeV4aQXikMyN3U4TDPHOn38JcJzxWsYTHRKhH1p2nkYSUH3Z0IVsJB4NUHOhoPiC8eFRWiHrmyfxu0Yt1mKhkyv+psJN\\\\\\\/\\\\\\\/SsltjrzT7Yv7QlpHQOi4Fr7vK0C9cPOZR9paECzdCqcJOf5KMAQWdG8EDnBYtKLQl0TLjmWYXwogpkdQXdd9+4mbbAulKInMfKardd3KZ187BB6jyuuK0rofT\\\\\\\/NMaEI\\\\\\\/vRO25JRzse3w2PYQdB4jGUztbyYQZvh6gJ3W3CEyfc+PSFgGRNOMlRG2Tjwmm+t0xtE1JyGyzGgxZo2Y0xz4zaqs7zB2lW5PFc2WqTvkAS4EQR0Kf9O3RtESMLEsHwvkTncLEPJiHBcd+CA2d6zWAP69YgLj2lgxP3JtjeAjyJt3XssHgMzZgnTOXkCF\\\\\\\/wLlyn7uxHb2M0D0YgsQch5+Mzsh8rAyx+ew8gzvPNETouRb0xlRUq1A5QEyO\\\\\\\/5sZa5PV0uR8HP5bfPcbcLpn8FOLoNUGTaB27hEySlARCt+fj5sRCTiC0qO\\\\\\\/Ht2P2ne0zv062dCAtMkrFrchaSp7h7QMArIzewkDxixdmneZtXtIBbTB9DKpueCbTPvf4xQotws0eedW3\\\\\\\/eSjLnXsNbnFfi+0TTtmcvYNNDml1eeqt5HBSW9tdjfRj2MmNB2Smbnc3AVp08hjfvLlhkkDGtZD29\\\\\\\/xXfYWIXojdRsxVJ61+T7Wh6ge5NxvUJC7hy78ck3AttPdQEgyDhI8BfgrG+oXxHQnABGDGhGMycnW0HGQoUEX6ywUYYSNXhqL69wCtCYkUEw6mHdT8rhXq9nmUp+yVeGBaD7SJWk+iKcJA0ycNuzo++mbZ6\\\\\\\/X82fzYqQMWgaZYJolXDHeBSaXUwyQy+mdzL+Efu\\\\\\\/nlGKRT9oINejr+iJBE2ndI3RypdmWC900bxxQ93ahY5wFmC\\\\\\\/6n4XHofFXpyIwonyANZMeciLQ4TRFh+wdM+WoqRDAq3ZVeKUqNmJIbCY\\\\\\\/Pkb5v2JOpeGGMG3dpeBYQjQvPxSHEz62J9QhYSpWlRoqId+aRpjBZesNh3d6yzIY5h6wG3FLGooW4SZ5GrDdPFuIJYrP\\\\\\\/y9Hu8yixu3Vwcj6zHFyYeGEVKEGGwqSHH20oa5JPnW6wbQc\\\\\\\/H42ag\\\\\\\/HnSBXvmhDnxmwDlScQdFmVZY8KOMbc8cfqZ2kebQoxqDS4T8U73bP2rKCtNN9p9OvAf9iewhASrFndDWlZkin78TFlkf+wKv2hf9Fkii0EZt+UkgTiMLxvXmEXZ8crXXywgJi0Exja2yAgOsyphTOdUHfnily+N9+HvhXqxiwGJxOEgllZ6zAaFnQHjxQMPrlbTUobPLIOK34518CQLx7iSfV0jeouuZVW3w66UtP5FdB6Mv6cyTASWMinUjpzpzpCN5AZRl\\\\\\\/wMmiS1DgzreqVUuSgRO64JPYAQnptqYlW7Y\\\\\\\/aONN\\\\\\\/TziPaW3RPgQ5SHeEzxO5R6L9w2aVk7Jn8CnqRJiV3IEdciZVMxyAOlC5lXNoKNbfcXL0yqjJeB7qbShI147X0rylbVChDLcMufQElwuOgN1SDeXVwYcU7t\\\\\\\/f7aumCfW25ZDXX0aZ5eZJIsMkNbwY\\\\\\\/KuqYkXwayIvcekXrf9qnlGQ7hhVvwAfIqkyE8CM7TI2v3dCe2LTYD8FLOf8CaeUITYWA4CLk1NPSgwy\\\\\\\/7s0tdjoP1HQsI6ZaEwKsHWHPFpAovryB1wdYqoPu\\\\\\\/OKh57RkrsQaBY\\\\\\\/eErJznr1sw1FRXOLqbdrbzKx1CTqxjiLfX1yyGqCi\\\\\\\/4deXwtVb5OKJa5gEEWV7L2aXr0GhZ\\\\\\\/ys\\\\\\\/CTl7hjuGvyCpWh8sR\\\\\\\/sxRrBw9sfNNyLfMatGU9nwKDxzCwsnHiOiuY5hNldXT4qczQTHyQzzZjiTsH3JE4KeVAkH2KyUJ0JTvFBCYUvkYt5+aHlYFXjm4Eq88qOV2SZ6yP2qzNPASznpEPCFCU\\\\\\\/xl6M24b25Y21BSFUtFcNC0Mow4fkD9v2iFiniTI0e43olGqDqY2jGmueMG3FcBUVqs9nUg45NZGhpFAQ62JjYc32YHmQ03tWXhOFlvVNCRaRYKKUSls0Sy6wzljOGZGHcYD5eAeM7sGP\\\\\\\/YLvoPmyXqj7HDgpCtjI8fGwrlqeu7vYgGrjtSyiRrM83\\\\\\\/AxhfaUyPximxm4D+puiWuoymsvvYymqMjF4\\\\\\\/CKuJUWlJjboWzQB09h0wDSFwzu+PfeQFe+7IcAq5eGvxzrU38\\\\\\\/i2kOffo0lZ7lAmqT0IH+smsq5FJlReCzGFjlV6fbfXD0IPhxJn4DfI01oF4zZhHrYy5iHc8YImlZg4tXwYgf6vkDOQf45jKCb46990IoCp5ZvT5puFDS7JySuLDEDaco5i7SrnH5+pn3YvRJfPBMRq2OxOkRs6\\\\\\\/0XayOBS6LXQe93TAL9TsDy192Fn6nWGDvObmGrhm83Aa\\\\\\\/HbEbSbMQe3DkmNdtLUvrcige8Shlr53MUNgD5alCqrQyAYEJ2ZirpPgmbwYK5fZF4fVmBgSxqtAd0CaAQkW3cGmqqcDv34SrsQqjV\\\\\\\/381rJY2vszg9tDDsHObv+N2dso6\\\\\\\/jKd0jhE8zzgmAE76XAfAplsCdXds8govlWFAQTrFkClA8Vx8iDWdQH0iLpLOPAYywVqNtKJirpuW614l\\\\\\\/XF5tBQbzcYl4k0s4JuVJWvSFzwGN2yhI3EnRJmxzIVmd\\\\\\\/J+0OOreRoA1Ij7essjf3hwycQhltb2Ji0PnXpqzySUYc\\\\\\\/V9twAvXyozLh8H5pKt2cgIU6e7dI0P5GQmneMYmYyrnHcoMqDww9WLlU5TJ92u7lgSxS9ubVe2m34XFbf9jwnG9SWH+9xoRmupgEp446tOPeDxgRxVGZDe5V11STUkWL3E3k2vjUC6yW8PEGY2x7oTWCf0jaO5uYgDFpYYFdPYBJdzrP63d+AqzFHQkB2g8WNiE9TO6nXL1lxp7fNkU\\\\\\\/ss9FgvqRcRM39vbW0pxPoOdfCBfZ5Eerx7WIoUyX2YvgZ9haBlQ7XEQQPBcGOaV7aG1zrygf+ZvUi1WAgFsdwR0OP+nO8NtvrzGeDtyZz9MOtc3oD1PIJ3Cotd6jHGN\\\\\\\/jUvq9a2D51yw7mbwe0zPXClrjTPUsp2kKmyb30j\\\\\\\/+22IS7VE4zAI8n\\\\\\\/53O2Hdi0dfoPVhTqDS0IOhwJ5avdIASWc81mjOTSgj+Q0UZqH7\\\\\\\/tAzaPkJUOxrk+xX1UZIji6yn2k11ievoKbzgAyM\\\\\\\/rI9uKDoyufshVJ9FaMgti9NjtLQJyV0Z36YWD9B\\\\\\\/\\\\\\\/hhRViGoPlMoRGAgW0SZ1c+SWvxaJ8pKzuBHeN0TMrmkCKbPYHN2JrtjuRTW0+qNDp6jtNsmGXQgTuQ2OEc9hYpkvRxoDJP9yQzJU\\\\\\\/LloDctqpYHRKTT2UGZzFexoUK1WIMyLUv6gZeXhEu3r1GqvX4S+s3yM9EP1CFVM9hZPRQWgUHP6KIp4Ao1gSa2ruW7ucYq9rQIO5ZeNg4eWSnjsjZHgqACXnTwcje7j1493cS6uvRZE27cDYWoNjQVvJ5mwI53DhTLCn+d4MYRMHy\\\\\\\/ZuaKrS2u4rarcPpSkXc9uUtB\\\\\\\/GjOprpx7+GKHaZqt3XJyz8oUiv6fMcbPozuRnjp+lQM96VY6qn180NvZfylMzpi51LB8otrPId8UdBvSlkn+oSe+kyPhd8wSpVj+2G0on4\\\\\\\/Sj1zXGLe8KMRv\\\\\\\/Iczc2UT+dVysq7SygI3Eot+rA22lw+iakHC\\\\\\\/R8cUn503u6z9ioS0wyAfrisXPiZvDPLvdk3ij53AnSupmJFZ7WYwt1JhRX3vBgm55eMNDNxE\\\\\\\/XrmE1mut+Vm+Em5bJFmLp\\\\\\\/TzXC0co4uyA2+89lIpfQYPka5oPMVHWcctxutar8zaD6gsM7kPtBfHxKZOtAKFQn41+SNEBa+\\\\\\\/mX5lTYoIykjgprvOawXRz1sZVoJl+r2d5EHbwbaeDJ0GqoN6gzqmVn9XFlU9HOI4V85sD+vkecExU0BRHSR\\\\\\\/SaWjc5tLiCxgL7v9D1794+HeXjQmXcVy745+adgtIBT8xNDeOXf2sAiSLX3Hw+Yv8+wtjoEYmipA2GWDsgVIiEC2ZJH5ppc7Gpn409goSlboUpIeT\\\\\\\/L3sREfabkx5yEeOh8E9CARNQo97TuvNCvgO7myBbvy\\\\\\\/CFprMXSDmBHxoqEDsNQn4uIon+AoXsxTy1NLHlLhIY\\\\\\\/iExW6AJO7yjsm4cmTrevrKT073iQE9+Nk5qUSt\\\\\\\/vJlNMvGWWO\\\\\\\/Mzkqla34U2hcxZlvu782DXIrnPhsfC4LKKp2lGHPqgtTfG6kdJQQoi0Vw82FqosxSMrJpN2i807AkMTRn2Hx\\\\\\\/7FwyjThoZV7pMpfQHd9tkr6WOTnIxEYiWULTsJVDiGWZhe7XCvymhPcTH\\\\\\\/XH3HmCh67HDC6+6jYC6bgk10NrleyiGJ4XRxGacf4PvT6JnZNE8ZYjKdDfLenykuXU0IkdO14YN9vXqKdg\\\\\\\/4bI4uGIs8LPXFHmvORJcty3q5bkOlldkybmB3gVdHBMfq9j5BbSmARp8xFii8I+BtcIWmMO4vrcqQkugeMWrY\\\\\\\/dh7texKlBe\\\\\\\/QiCPIqvj3Y3j8rZ98Q5z7VToL2nQtg6\\\\\\\/MIv1IikpiOExetRyGy5TqOoqX4dQyWxuUcgFNzGfyW3Zr4GtWCnYrwT\\\\\\\/FEDgivrwxg57oEfbI6bxjKOzLjSrQQLXdZkWwhNpa\\\\\\\/T980H+WegQ0NZ\\\\\\\/8C5I9WNKWPzq4+FEk6tw9nB9KMP51jSvrjYSBslg9esqN\\\\\\\/4gLiDKBufxffTL9odRKMl75nhvl2yrnZ7Q2XXoEyltuIdj\\\\\\\/b5SmGwrFywu8yNzeEBpKxAN+cB4QTVg\\\\\\\/dmaqxVXRK4KMw62JsgRYbYITZkqzehHGKXkMQPrCFyhIwg9m1UcZoTdCoyDli5HzZMQ3rKrIvoLSTCnjyTCIa8ljIP\\\\\\\/AaMj4VP4wsmPkxkGlo8kFbviGxicF4lLPT5XU0SFmaSR9atNa0a3qZeCUYVFGHR1ScusQCTOrPk8zdft5SVpk9EdjJPPyC4fMcSwsVxlMFNKYL5bY5vxvZhPIgHwQQal75QGd+TBrEBcji9+4n\\\\\\\/7oomN+S0cyH4BVRTMN2guDU2tJ4uA4Ttopn+uyLiIfPsxbPRD9AZFILhXRN8zSb+f72wpT8RcYoGrcyRo7Jm5yGBkM4\\\\\\\/TkYDc\\\\\\\/0t4oW4YkVyQqQ6oKc5B2BSk+Xg7wAC5TDRYVUiD++rIlhnfPIG9QA9Jq7BZuZyIdIWIANyHajAdOjhGxE6P5hicL1CIVWNtb0xQYletsxeZDY6jKXaTQpQUJbX681X1XYGhYv47GiydwbCQvqnZJVER8Z84Ag4ecmuU3f\\\\\\\/zDIPOP+Y5Ni\\\\\\\/7lVnPidmIcVWdbCn3RRSrHdbFes9O\\\\\\\/lQo\\\\\\\/gpOZddfh9DXs0kvzHRcxSIG7BHfKfrKTRh3IHtPlAmTAnJ1AFUMg87O3BQcQW9IBf4zICtQDYVTcHH2Q9NGTW\\\\\\\/lABN59C8\\\\\\\/GtqppHJgPj4nXU2ksM7Ulj20dVj8scq9cz3Qc2v5nPZx56ZyZcJBlzZivD373nkuNgRH60llbk+u4Dew00mpesGYPt+ufPhmeqQWl2wn4jcNluQSnW7fl9HLd\\\\\\\/t3NDQQRqMzBaapotnJ0QXfE3eq3\\\\\\\/Fe2+ksDObXGTi1sWC9Wi2cEDzhnMiXKFNqHGONLb4di9bGgz9rbUa6FRlYOffASJL++WNktZs6HuxyRINokWLnABV0bI86jgF0i0BHIgjcP6Om4upOHn05\\\\\\\/uIglflXpUOAk4eWwg9kgJMUJ3ABYsLV94N\\\\\\\/kHh4B936P5y1eGM4RDG\\\\\\\/eZd3FDl+qqllIX3K7B7uJqgN5ySbW8yx4Pbz8im0VVAZSfU3OhpkZOpxtzjmd4wBswdyAHDW20VhyOlxCw2n6pEyOhncmbAXR+qOleUtD6SmjQ+EpcXfG90+oNUVvvC8LhzfI0Okx6ZAECXTqNoXk8IJ3pMwX2SKkkh9fmHw39z+klA3M8eTBoK9IcGe3yFq5hMo9y8o\\\\\\\/QBSE108oyLykvNNwtoEouD42E0y6w8FYYnbjAGSfKn6NwijDsxIGWJlylIyubyI9XvMF3Eo1uB5aSYhmRweC0DaCesMOcApkB7NVINUrdoSEe3IQrNvB36rvPC9sKWsO7QQZSGHyONjrAOBD+VzcIhXECpgl3yj0IrmHqAtU7anNsmGZ7iVWWC\\\\\\\/1CDz2fUiAygmcbsYcfVOg5zJIKnkEMX4eOueS8qQCtok7k4rwP9P1h8adRU4P2Q\\\\\\\/GXN3iQ7nmChK8gkmGcaKUrB3yw0aLeFXDey9C6RzK\\\\\\\/29kV6YyCMvhuhQcDPWGMfD7vxub8K0thKcUM9Btr58ZPoE2\\\\\\\/k3wJOeAGuZEZM46JRKGMLHuU3ERV9WCj5S4LBbc\\\\\\\/L4x3NJVFPnwJUgJrtPoCzDfgHZ3AY1VHbU22NdSN\\\\\\\/grWgeuCfdKn445jF7FrUdyRtf7NjXufUVheJjr5dB+5JequEV1O0eHBI+SfavR19hJwOP\\\\\\\/YTqC59lH8Bx\\\\\\\/cEV1CPmGWFJJyeTMs0yNGAD\\\\\\\/Mk\\\\\\\/3ZEICGBYlpYFHAGbQnxJ\\\\\\\/+5KxFhmh+M4zcziKcycg4ICp+q4ZFTPfbMwLD\\\\\\\/crRN5a+T9z1nMbuyPMC+1C0T2n1BiM\\\\\\\/JVIfpXqJjyZJU\\\\\\\/HYz6ONoDq\\\\\\\/UN1x5oTqrMVdZ\\\\\\\/gB11hWL7oVhuuAhHZXfTw2+Po68Sz0mlz+P8\\\\\\\/qpJEIiSdcu30V2bYmlOELztFmNeBR1Ilfg4A6BGYyggx5X4iM\\\\\\\/Qe9N0PHSO\\\\\\\/1BaVdoQk4mgW9aqkKijWLmBsEUVJyKwj+YdmfBpTgYKJfym8v0mF83cGeSSfzT5adrBWdSYfUTHbJqCakYd7FWHwkFgCtlR+HYD8eU3wbBZZg2qd0YNOIKhG9o3B1\\\\\\\/8v9BS0A9XHVn+ENEbRvrlwAE0AbI0ZFPNW\\\\\\\/c2qI9Ec3x9N31ERptycMeSHB7Hiv\\\\\\\/ON+HHX7nURCmrkcaKIXG971yfiBNtSJKNVQ6P3ihZNNCr19AFGcHP0dGrgSSsarDFI9WL0eIbncK8gZCaxs\\\\\\\/KhBmpulllVZcFJcD+RWUE6M+Pz4G07VTtEMT1Bg5lfqk7Oc2VGxugseL\\\\\\\/VTRpBD1vWOGONP582mkL6DmyWqrNvAZZN4r4a1\\\\\\\/gJCIkyqjrLjukHMd9Rcc0iFaRg0bNJfU8hJp0DSTkLE4kYfHxqN+SzfUnsTYkvSvIR2\\\\\\\/gOupkwTomajRor4agtSLyULMLAtYOIXAgJHC1tT9hEofV5v4pC8DLphNcW8JCzk2IGrH+T+uLPfCT7b44CBak2f5AjOh9e+z0k6Ech0sPrdKzsM4ocpBu0uL\\\\\\\/5i\\\\\\\/3Wit3R4muKWzi+ipRNiI4hHFcBY1FKdWsyWoop2FbETbPZFvxn6Yo+MLlqs+vb1KyVoBAU1nWVeBCalSEtreJv1aOiETxr1YLLD1kIWIOeVDaVucxLo+yqQ\\\\\\\/yk0tNX\\\\\\\/s83JaZY2uoup9sau8m8NzTl8d65MozL197Ile6OoCQ5teuwGA1P0lSmgbbS\\\\\\\/2Ls8s6hL47U57DMrUwShiXcJdImj3aaZVmLG\\\\\\\/FgzPBS4EjEOoJUMzZkDFjzrdCKRN\\\\\\\/6IKP1z0ZMN2HvotEbHbxb3b3jgkuojcKSA5F+x13mqLKIKi8HaAJTZukUppC\\\\\\\/oKmGEqrCj4GOo5X9U82T7zttcu4LvTqtDegel06qwxon+9QBnLaj2aimIecVtpfvnxPtekOjFEPGHinTZTglSGGzq8aKvndagZhWd98X1V0tWtijlqtfZv9EjG5pSSfaKlboMoARl\\\\\\\/jlQT6bguIrZBMLbLFbIahLKqCZXCr60tqAld266JZEeUff2FOfBYodmN++kgbAe+ZXj8FBCaUZ3YECp+dVmKUzKdvGfMYReshBHsVpqy72nyijK60aKoTAgjCYudncs\\\\\\\/XWZrUDzgqLSCGdbJKlAmb+njCEsdOZsDSx0WJDdbjOyhop2lmFe0AgZ6zeXieyo5shTnTdlpIC4qR20vKfsgr9AMXhk\\\\\\\/+TDgGwknP76mhIBUcqbhrdaXGEVbwUn\\\\\\\/7GwosCAXB7E56e8JYPWjKEauS1yuEoz6JGQdwBdGZaejtFFWBe5ZSkpLHqlU+8EsEGwChTm3OeW5fNY0SU0qqyvpjyOla8p7mdj\\\\\\\/dgvoMxFoHSiGKkWVXA2Y2bpjQB+tWW\\\\\\\/mg7X1aWiTusbRLYgAScP+ndyqbmgft6uMAza33GDt3DXZyryQ5RakrBGu59pPuyWCaKm+\\\\\\\/x6MOhB32NLYHSbvOhLRv2SFLS5Wu+rsm\\\\\\\/Mes7BawdVn+NG+iJsJLpGSdasQnmbW54LF0KQVLZZJKrruP9qhs+mNgpXknWsMY7VrrB+tW556hGqmMZ76kacTSjD252sLRJRjD0c5EmRBUUXn20DFLq2gn8xX7q5OJQcgmb4HYiUnZbcmubLKSRRE5mfX+Sgwlkky51n07\\\\\\\/SfTCMcdCD7c3JGyLtZyP7j1muPcbGugXoxIX9j62ARj0QEzwwn+qmiUIXtNir7c1ZxSO7Y8jbt78u1g6Ix69vqNGEDbOAip+gYcf2Qm6ZzVfaV+amk8WReMinqOvaZzv2IW7eTDajhZeu8nMnU10B2NvfO0MifWJ3DX1xnycGP8PskHd1HypNpNqBaUTbdM4ShnyT60iJoM3LFPucZ5YHSUYJCxcEGYdAiX4aTV9v8eDD+FQYF7vuO9ST8E0eSjyJWVttohecOwz+LGitVfF9JkdvozwxRDYvHLziLtH5ja5\\\\\\\/I0fyzEdeXo00mBX9qtd7E\\\\\\\/oSXUnbSdy8SpV64AnMmkXlAU06lGBYHrV+XP4qGqGLCYgrMpQIrErvbdp1hNcsI0hR1sp363rKxBDGwGT+EPWOLhia+0WzlmmZKXyD8hB92CiHhsiIpy8GaDQmnslAThwXTOIVUjRz\\\\\\\/LhjcUeD8lLUO8x+ck\\\\\\\/S1c\\\\\\\/qfhorj4ndVR6rZQ0VKcD8Phuz0qrHXmAQxcPM6JW2mab0YAeE0zvZCASMXMY6SKmG1rmW4zlwPrLAPzml6JXVhxyFQ6AWEkjBsYNT5IEPIijU1gM3qi3sIRX\\\\\\\/CP4bWTUwSlRO3MwyHRiyIxzBUI7K4GgL1yRIH6YFnacaBnnMGbj0Aj5O53Cp6J9rnhjcKNRVHP3KzIbEPGWLvesnLSjFN1T5VvIgmuazIyxxgB3Hsc68C+PM+2HXBsvlPak9B9lqaJf\\\\\\\/ucBW0F\\\\\\\/ix5YQwOPsgIBTkmW9FiFAwLEp16EL1JiiPqw\\\\\\\/eOaV0MpsZDYzKnncxa1z\\\\\\\/IWZ4Bx34kDNncF\\\\\\\/isRYTbPJs+Zrf7L0RmOjK\\\\\\\/qIXW4gf0HIYXt0Qu4oT1DumnSzLKROGewLWSnrp+mRmv55OJuGzfnTDdVErt1\\\\\\\/4wglmnVXAhG75rJZmYIclh0IJKY5iNv9AEtE7IXIDSdL2MzOMHVzZ6sz1JKic5UCtPOlOaCUPWjJTudRC38hrnJA1ABv6+16JGCP27h1oeAVDwPzBRrcpiGilp6Qo2ycSOG26Recarj+UU8oCrPMSsnOupBAfQ1MCFrAnDJ52KKKajaYYjEYTf+AiPQToj2OId2N44hmWNeLCtG6vQgWToAbYtH61juemxnJf3+MODdiqAl3DCKxzpWswuQzjxp1S7cB+xaPk2NiZPgR31fH73NL\\\\\\\/gP6rj74zGUoj6NSSAeZ\\\\\\\/vjp4i9bHPZ743de8zyFj57l\\\\\\\/tOz6BCwf1MgP7JEOXs7wWriWa3Kv9RSAua0dgf8yKkc+w6cg8gxCaQ7ayPgXqQyLL0eO370Wp8LFi5x5qRSWY3IyEgyHD4fQ4X8ywp6RdFHDlByY4BbdyxBIc2Rcv0lOczwX5dmoSgFmPF2FTo0X6uq3cLry19xnJ\\\\\\\/Ud4q75MQgbYi6f\\\\\\\/xuin9W9OWX4nitEu\\\\\\\/6vs+1ZG\\\\\\\/1dTtEv7eFL4ilFaAerQ\\\\\\\/ava2ZB7xuzIQkcTzS\\\\\\\/XRdsG\\\\\\\/9kUqTCLA3vl0Gk\\\\\\\/tS1SzhykZs\\\\\\\/miqLUShQ9r0yu4Q6DJ3yg23BBBiKUBgog6BGEC436zYbmP8s6+0BdMD3nxVwGL0s6hlQ4sdWqpiOqpUGgjykdvLeOmVKtjI+ZXzgjMaiC7DFC7SjM5OkieHDGpgFetgHwTjlhjoB4NOUoz02BsL9\\\\\\\/YTWtass\\\\\\\/4jjCHNWHTjFhiJBIZ9SsBp5PJNp7+fp0NqtO4qCxnz6pZvbeK81lGE5PpYqcjXGAynTIl5y2TpNgz48s+fi8+1jOjKzsFSswzZRc2j2yhzNTnScLWv417Pr+uRVvJCbT1seBO5l+tsBnrdUFnscna2RtzW+35lNwr13\\\\\\\/JN0YweaeSmJhOh6VKirumpA+ND6GiWEl8dZpLuaMsyAP2cjg+Ut8gdAd2Xi8Y+rW4TOzpMS4dSJuOMoitFtcv01Sa3Y6aM86yI8py+B2d+soS8OTnmBNJcxg\\\\\\\/ORYneQHy\\\\\\\/wQNqU0H7Jh0H2xK2z2b+f6aaB\\\\\\\/SLMjc4PovrvZBBY8+24gGHHQOpAO1RROY2QG9sNegcogBvCdOco7zvncsBvpcEmZSN\\\\\\\/bEr86femRX5lT2I19O5icD5EFx\\\\\\\/Ms6lKe1y4+lw50WX\\\\\\\/VEAAfXs6ThwEYUVnZ\\\\\\\/xD+XZ8Xj0s5tELXCTruClGUQlK8WAYWsgbZ6fvXNVNfWGQAvYXtGeYqbeQwswEmYK88YrVyBQS2FNM7QUqos3lGEC4QSwBm5XkvaGAgAK2sI\\\\\\\/6pyukK9MARxN28jpJGm\\\\\\\/LVHmfPRuohukYkx2zYgDDAH3CIFNK11M\\\\\\\/HLJWIfu285UrI3f8Bb4pkhiW7A+LxK5Mz6N33Zot916SLM7LCiEgDFFdshlNmKE21x1RiZvuTrAwvCucUOScSquTYlndJ2ndXJAJXSclVqyIkcDJrZF1WhJ5enFuitDZmnTAWg49e\\\\\\\/kHNRsA4T5rPljC490cezeZCDLjZuuf9G2lMqx7\\\\\\\/wFOOc\\\\\\\/xTkF2214ALys9cWfNJFdVoKw3Yu3v9Qt\\\\\\\/bGaO6wgv56yJ6mvUbCQD6ZZ3W64dNwNfqlKGcVTuLumBhFbm8rAix8vv44xVRZXKqFrWtnJvJ3uYZqyR8ncR3QtNszZxAdHO8BmPT6nDHaaLsjs+b9d+k+I2UyiBxrKDUd0EBrz2aS\\\\\\\/eiB6VsOGSuG1PAIrYMFgv0H0iZWZFXEQlfI6SSS1GcNevMSL82mOIw0Iyd7srKUlJe\\\\\\\/DJyK3hEJIoQbDPppNRruv1Z4FdBcKE+p4SpLdjwCIanB1F\\\\\\\/De0xgQXH0Ibfy+IIfUbB3CVHc+Ms9RuDV1iRlTrFdbn56LmMP\\\\\\\/oWi6ll3+1YHspe4a9M7v8Qk0UqKWJoYyEdRJXjaktqaSqI2s+m7PnW9XyyC\\\\\\\/i0VJpnQla7E7SjH97lBLSm+CcT2\\\\\\\/zl8ac38rvtVk4Oz5ISccNCLp7yqDATDcBkkt7MkqDSVN+lNiIMVhJ8+gxIoa3PH3GOW78EU3rmEUfJT6M3mduQYtlaESKbktE9\\\\\\\/I+Fh2v9n6NwEuBPeOaKGtI1HXQh++MXJKDNnnTnU2C\\\\\\\/MoXlKwf7tuVWZuvbYbN\\\\\\\/WRHBVKAaroQwMve9QYWx6MMmTuqTCErXCYdfcRChfXwWsxkqraMV2ynL+dnDEw46WqzVMsb2ta1qlqhcKA6Yp3\\\\\\\/pWmw7Kn3PEqQt1FlRKdPmS7Srkz3LMkjet90dLtkGFjq1JN8jCrbN4ewkN1G8U2VE15awG4LEvA\\\\\\\/EODOuJLZtMMd1LdSGleb6+PAo22M\\\\\\\/4V0lwS+4RyWU7bp2Gjj3SKOPBkiX0JCAHPhU6kbh2oehOWz9oXeakmPZaU8iRuse\\\\\\\/LpZ3XCJapWema7CaDpdvmRiAukbU+9S5qsYqVlOu+qxUa+ZpH9Y5uUNExDheRiZWLs6+dfYF5m2VWt1CNsI2n2IAGXgah2psnzJTTytZxspf9FNBP4ccurSe+nvxwcECDJu3iwf4\\\\\\\/TBYinvM\\\\\\\/WSuV58qM6e7o37kbwEVfKawKqS4p4gLAFTfZsDqtFJuVR8BGxSWeDfX2vzXgwEPcM4c5WBcPdxhcYjWg\\\\\\\/OSl3TCftmzthg1YLHohCdPk0YaKoKrLJmvrCObhtzMkxBLXEyaGzXtLQv+o82Ob3Q7vlpUFOYuPiyD\\\\\\\/vfJgjrkyXXUH4jJYasoA86oEVt9fredup\\\\\\\/Y5KLkANBi8GXPRjHeTD6ZSnP3jWSmsfR3yZVRTry3MxXe0MzWO2E5HZVH4rb5PyhVQIiyDQput5rJT+nl18Pln2RmPqN1S3uYdWR2kJxHDee0hRYbaUTENvTEN4mEYF1ywT7EoN9O5DLLoxElcNLuoFs3Lm6a1Vh0AwieuE9\\\\\\\/UdygoO7CLuZICk9CnSFkTW3pGSVDLuSWGqKcg6sPZAyMaqnJ3N0vDj8GjRhCJ7l7BqXOiSot7Z0+ix22yqGIcZbKFZN3OkMaDcCO2TZWlttCpfdv8pg5Mcg7626dr7NVatW+Ea3MjRyRDudwePDALyY5bTz\\\\\\\/wR6TfVXztfJ8W4BlaeaSR3oepmgLkwrsRrzGWVw1u\\\\\\\/QY9jtFdgLWBO9iyNQX6GzUsw0o2n\\\\\\\/mnh8uNyb5gAWSHubBx7vPdagYpMaNqsbIgeiadYsy0\\\\\\\/QO8CCf\\\\\\\/UdROtQyj9fceY33wWIphUeuG0716zBTNZ+0xajBfrAjxQw5vXNcmoC93aT5Y7UOz4+TnrnA1bu46PNVVBqJy1DOatFm39cM6P2vTSGkWwAxLSADHE998kJaXiy0Nlinn9AW+NYm\\\\\\\/nPqVWTpSCAOO\\\\\\\/oyhOahlV+iGwdJDcRaa\\\\\\\/V0vDFkviC28II4OWt51xc3yKdXQNriWqKnYgvOgyveqS2HF+KZOJoU02u\\\u0022,\\\u0022iv\\\u0022:\\\u0022LYKtugK2tYWDw7ez1GWfdQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e93dceda77a1cb4c2995410b4cf26ec18829a6483c201603a2f94da169e4fb0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KsZV8RgfoWUpblfBHIlQSBpj+ZKm2M20EhHJRNEZ0b0nWkFn4SkoXaMANj\\\\\\\/PiJTiM5pVO5+COMs5cs1AKGpCpn91r0SzNM0Ofr8U3Tg7YpvlAkPKvdtjzI7JYJBjmpRFb3eYbIB8SJC7\\\\\\\/emZCO0ymafz84X1emiOtWAS\\\\\\\/eYgIEr2WpL5EGvHgZmrNom+ukI2IyshIkMv4H6qgdlkwKgwYHOpP5GuGaOBItg2UFNziV9WFBI7NI4gEwk+DpYV8crjutne20ucl2NDmvy7HjldZ9cZwPOH1C\\\\\\\/7WMlEvrEOoVjRv30yg8pgbozDDrjm7icLCIzirxQYXJeuWSL8s6oXezRDh1imZ9p1W0QLIcg7SSbUKCZ6FDNKGoykj3MgvJp\\\\\\\/qC0SU+Dh66VB9uDhVvzkSg28DL0Dt4hlTAIVzkqaFgwH0FtIwIRjzhRBJZNwSktVX76MI2TRE8\\\\\\\/BmAZ9cGCw6h+JSnhYw1IZc5NskejujPZ4w7t811naLww6sjzr8eE5o1AgWAQmxUBG5Hwi79KHFeHDmXBxj8nIaYx39VoINWRcjHeEtpyJ6XyTKVVT\\\\\\\/tCOQrHkAdm20emeS077XgaWa4hUqNpwXW2gAPG9\\\\\\\/RlNqJuOA3qR7906GeUYglBGzYIEXyTOQWcJklR6XQ0yA8CshzHAEHDZcuLPl75kEvcZImZjf+u6MfFO3P5axW1FXQ3h\\\\\\\/1nUEm3CQ+Ey3WCkCgUmDvlyZ1cR91H3OFpmNwSVe1ifoscF79AWzpdY\\\\\\\/W1QN3nFNvZCLXFIIW7Rjf7LfaY2tYwlHYEV7X+5wzSvYl+wB8qRWOfX\\\\\\\/U5ehSZzfbpXHiWAfOWzZYO+qfDb6eris\\\\\\\/e7Bq0XWXSWlxkt1lKkikuu6bzqyvIBymL021Bj1QQYS5DsRWlkv66EPUhU397sS0jp2N7efNC07IzG7lTVina+qeexxPXnbj0x82sh9t+8fbBR9DkS0yCL15T0iKGrhkZI94zZqLl2Q\\\\\\\/LLqTGFAOkrV4HvGkMksaEUBLGdUhmNLLvXDe9OFs1G\\\\\\\/bWltNqFHxTwZPfP5ALXKE1TJjUSpsWYCza2ytzHB0iLsmUgKzfQCo+Hw4Ql44ayZAEcRFbilgnnvvp7W6uMF26p5aKazyzfbeo0qW7bLH6J5Ebbe8ekfHJzvxcxIXmqwh+iXeQjLSbB\\\\\\\/oCdRauvNRKzogzlTXACV871L21aKPEhOtQA1mxrsbyqv83B19gJ4GRErM0ICiy3VLasdeJfYlK1ijUY5+GupfRitdql7xuV77WULi4ptiqfZqJlQieuNeyafHGralLE9akHe8dPia5NotB8zXAIfAGjmmKtFBZxUSDYgmEdEuBbDZPmzrjHBrwOBpSKaqUYTg5urAZTZCeGmg9\\\\\\\/9H2K7qJkrbwhaAotnLRgT6czrnztTOoWcLVnIS+K+2IjVot\\\\\\\/2bvVk5tHtXC0XZUrEEQ3WXrEueH\\\\\\\/YprFMu\\\\\\\/+kpoaL6mnwm74PxzzYq680qEFVsuB9lGZGs7nbS62GJrHTimV9Nuk8sTvfuLuwGkPVWzy\\\\\\\/5alM4VE+2FDnZcc8g3CUoIpLe9EPhz7JuPiDcCPzRWeMmgs7n8vd1fBdKzhPjob3CPMwPJUrv6d\\\\\\\/0nx270vbzSh1fFVO2zMgyf+TJxc8Vlk\\\\\\\/\\\\\\\/nboX9NBWKndU4lT5n8dac8v057+QNeF8tBJbIaY5W7k3gRfbrJA4GHzI39Yzl0omgPZiUKmNYe\\\\\\\/U84Ll32xjrBQmkmqXzcXc3v\\\\\\\/gd7R37AAGssK8Bve41zx4aHkgH\\\\\\\/RZXDTo8gq0TUQYAR5n\\\\\\\/k\\\\\\\/NC0fcPp8O6kfjZNzXEn\\\\\\\/oRQjKThLlQrVEydqqdqo0Tf5YCmsCRggeICmtlfD4ETxDprQevXVadfe8ePErntT6lexQ3nCTpZStxu5KLzDPJ9lUZCKpWUtydTkb2DEl7OoAOmppTlx9QshSRCzi6nkJl5J2jqgEUW6naG6HkpvE3\\\\\\\/o9s87+h6tnLQLzcFS1sMR+G19uyMF2UNLHzlGMJqncEzT8atvx84nG2QdA5fKzm9MuK305d\\\\\\\/ks2I4j9WpuyyYbOS\\\u0022,\\\u0022iv\\\u0022:\\\u0022NvDGVtwdUlSbGXv8+WBQoA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002242a4f544513e6acd5c7a5e99677dfb02365b305cca30f881e778cf678df4ae1e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488286 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/427" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jsNl9rcTQUtDoxwy0a5WL8GJdbvKgoIntu4_G24zCMY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tKUuHDecChij7T0FtktML7cLTgKeQm7iX07t_Hrhm6g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/411" title="Dissecting the Genetic Architecture of Shoot Growth in Carrot (Daucus carota L.) Using a Diallel Mating Design" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/447" title="Direct Testing for Allele-Specific Expression Differences Between Conditions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488286" id="node488286" data-pisa="ggg;8/2/427" data-pisa-master="ggg;g3.117.300438" data-apath="/ggg/8/2/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Diallel Analysis Reveals <em>Mx1</em>-Dependent and <em>Mx1</em>-Independent Effects on Response to Influenza A Virus in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5859-6260" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul L. Maurizio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin T. Ferris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory R. Keele</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0781-7254" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Darla R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2590-4973" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ginger D. Shaw</span>, <span class="highwire-citation-author" data-delta="5">Alan C. Whitmore</span>, <span class="highwire-citation-author" data-delta="6">Ande West</span>, <span class="highwire-citation-author" data-delta="7">Clayton R. Morrison</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3977-0083" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kelsey E. Noll</span>, <span class="highwire-citation-author" data-delta="9">Kenneth S. Plante</span>, <span class="highwire-citation-author" data-delta="10">Adam S. Cockrell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-3538-1635" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David W. Threadgill</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0001-6827-8701" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ralph S. Baric</span>, <span class="highwire-citation-author" data-delta="14">Mark T. Heise</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-445;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488286" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul L. Maurizio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BL.%2BMaurizio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maurizio%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BL.%2BMaurizio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5859-6260" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul L. Maurizio</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin T. Ferris</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gregory R. Keele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BR.%2BKeele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keele%20GR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BR.%2BKeele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory R. Keele</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0781-7254" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Darla R. Miller</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ginger D. Shaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ginger%2BD.%2BShaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shaw%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGinger%2BD.%2BShaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2590-4973" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ginger D. Shaw</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alan C. Whitmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BC.%2BWhitmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmore%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BC.%2BWhitmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ande West </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ande%2BWest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=West%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnde%2BWest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Clayton R. Morrison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clayton%2BR.%2BMorrison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrison%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClayton%2BR.%2BMorrison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kelsey E. Noll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology and Immunology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsey%2BE.%2BNoll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noll%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelsey%2BE.%2BNoll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3977-0083" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelsey E. Noll</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kenneth S. Plante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BS.%2BPlante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plante%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BS.%2BPlante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Adam S. Cockrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BS.%2BCockrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cockrell%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BS.%2BCockrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">David W. Threadgill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Genome Sciences and Society, Texas A&M University, College Station, Texas 77433-2470</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BThreadgill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Threadgill%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BW.%2BThreadgill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3538-1635" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David W. Threadgill</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Ralph S. Baric </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ralph%2BS.%2BBaric%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baric%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARalph%2BS.%2BBaric%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6827-8701" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ralph S. Baric</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Mark T. Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BT.%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BT.%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/427" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488286" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/427.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488286" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/427.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488286" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/427.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488286" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/427.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488286" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488286/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Influenza A virus (IAV) is a respiratory pathogen that causes substantial morbidity and mortality during both seasonal and pandemic outbreaks. Infection outcomes in unexposed populations are affected by host genetics, but the host genetic architecture is not well understood. Here, we obtain a broad view of how heritable factors affect a mouse model of response to IAV infection using an 8 × 8 diallel of the eight inbred founder strains of the Collaborative Cross (CC). Expanding on a prior statistical framework for modeling treatment response in diallels, we explore how a range of heritable effects modify acute host response to IAV through 4 d postinfection. Heritable effects in aggregate explained ∼57% of the variance in IAV-induced weight loss. Much of this was attributable to a pattern of additive effects that became more prominent through day 4 postinfection and was consistent with previous reports of antiinfluenza myxovirus resistance 1 (<em>Mx1</em>) polymorphisms segregating between these strains; these additive effects largely recapitulated haplotype effects observed at the <em>Mx1</em> locus in a previous study of the incipient CC, and are also replicated here in a CC recombinant intercross population. Genetic dominance of protective <em>Mx1</em> haplotypes was observed to differ by subspecies of origin: relative to the <em>domesticus</em> null <em>Mx1</em> allele, <em>musculus</em> acts dominantly whereas <em>castaneus</em> acts additively. After controlling for <em>Mx1</em>, heritable effects, though less distinct, accounted for ∼34% of the phenotypic variance. Implications for future mapping studies are discussed.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/treatment-response" class="hw-term hw-article-keyword hw-article-keyword-treatment-response" rel="nofollow">treatment response</a></li><li class="kwd"><a href="/keyword/causal-effect" class="hw-term hw-article-keyword hw-article-keyword-causal-effect" rel="nofollow">causal effect</a></li><li class="kwd"><a href="/keyword/bayesian-mixed-model" class="hw-term hw-article-keyword hw-article-keyword-bayesian-mixed-model" rel="nofollow">Bayesian mixed model</a></li><li class="kwd"><a href="/keyword/multiple-imputation" class="hw-term hw-article-keyword hw-article-keyword-multiple-imputation" rel="nofollow">multiple imputation</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Pathogenic response to viral infection varies dramatically between individuals infected with the same viral strain and dose, and much of this variation is heritable. The impact of host genetics is evident both on the primary exposure to a virus during early life (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Strunk <em>et al.</em> 2013</a>) and upon infection with newly emerging viral strains; the latter, where prior immune exposure to a variant viral strain is not cross-protective, being especially common for quickly evolving RNA viruses such as the influenza A virus (IAV) (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Morens <em>et al.</em> 2010</a>). Pathogenesis induced by IAV, whether contracted during early childhood or later in life, is thus likely to have a significant heritable component. A greater understanding of this heritability should improve our ability to not only identify populations at risk of enhanced morbidity and mortality during an emerging pandemic, but also to identify successful options for treatment.</p><p id="p-4">The past several years have seen significant progress in identifying and characterizing host genes that modulate susceptibility to IAV infection via knockout mouse studies, <em>in vitro</em> screens, and studies of primary immunodeficiencies and allelic variants in humans (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">To <em>et al.</em> 2015</a>). In humans, screening for inborn errors identified a major role for interferon regulatory factor 7 (<em>Irf7</em>) in modulating the severity of primary IAV infection (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ciancanelli <em>et al.</em> 2015</a>), and allelic variation in <em>Ifitm3</em>, which was identified in a high-throughput siRNA screen, was associated with differential severity of IAV-infection outcomes during the 2009 H1N1 pandemic (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Everitt <em>et al.</em> 2012</a>).</p><p id="p-5">Most of our insights into genes modulating host IAV resistance, however, have come from studies on mice. These include studies using knockout mice—which have identified host genetic factors critical to antiviral responses, including <em>Tlr3</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hidaka <em>et al.</em> 2006</a>) and <em>Isg15</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lenschow <em>et al.</em> 2007</a>)—and studies that examine differences between laboratory inbred strains. Inbred strain studies were the first to identify the <em>Myxovirus resistance</em> (<em>Mx</em>) family of proteins as important for host antiviral response (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Staeheli <em>et al.</em> 1988</a>), and inbred studies have continued to demonstrate the relevance of genetic background to multiple aspects of IAV pathogenesis (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Srivastava <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alberts <em>et al.</em> 2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Leist <em>et al.</em> 2016</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Samet and Tompkins 2017</a>).</p><p id="p-6">Yet, despite the identification of clear phenotypic differences between inbred strains, there have been relatively few attempts to dissect the genetic basis of those differences using traditional quantitative trait locus (QTL) mapping approaches such as the use of F2s or backcrosses (although see <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boivin <em>et al.</em> 2012</a>). This may be, in part, because traditional QTL mapping approaches tend to rely on outbred animals—and when it comes to studying viral pathogenesis, outbreds are in many respects problematic. One important limitation is phenotyping: studying the response to an infection is equivalent to studying the causal effect of an applied treatment in that its strict definition relies on a comparison between otherwise identical individuals subject to infection <em>vs.</em> control. But such like-for-like comparisons are biologically and technically challenging to make in an outbred population, where every individual is genetically distinct, and this has undesirable consequences for downstream interpretation: namely, that when genetic determinants of severe IAV pathogenesis are confounded with those influencing baseline phenotypes, the roles of any detected QTL are ambiguous. A related disadvantage of outbreds from the perspective of genetics is the inability to obtain biological replicates, which makes it harder to distinguish which aspects of pathology are stable consequences of genetics <em>vs.</em> products of stochastic variability. This is particularly important, since it also makes it almost impossible to follow-up on genuinely extreme responders for additional mechanistic and genetic analysis. Translating strain differences in IAV pathogenesis to meaningful QTL studies ideally requires an experimental paradigm that combines population-level genetic diversity with individual-level replicability.</p><p id="p-7">An exciting opportunity is therefore presented by replicable genetic reference populations, in particular, those based on panels of recombinant inbred (RI) strains. Across a panel of RIs, genetic background varies, providing a basis for QTL mapping; within a RI strain, individuals are genetically identical, providing a basis for replication. The combination allows infection response to be rigorously defined and genomic regions affecting that response to be mapped. It also permits the creation of sophisticated experiments that target a wider range of heritable mechanisms: crossing RIs with each other to form RI intercrosses (RIXs), or crossing them with outside strains, produces replicable systems capable of distinguishing, for example, additive, dominance, and parent-of-origin effects, among others (<em>e.g.</em>, <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Xiao <em>et al.</em> 1995</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hua <em>et al.</em> 2002</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2003</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kollipara <em>et al.</em> 2002</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Threadgill <em>et al.</em> 2002</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Mei <em>et al.</em> 2005</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gonzalo <em>et al.</em> 2007</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Swanson-Wagner <em>et al.</em> 2009</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Shang <em>et al.</em> 2015</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Liu <em>et al.</em> 2012</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Zhou <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hallin <em>et al.</em> 2016</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Williams and Williams 2017</a>).</p><p id="p-8">RI genetic reference panels range from inbred lines derived from crosses between two mouse strains to more complex multiparental crosses. The BXD RI panel, derived from two founder strains, C57BL/6J and DBA/2J, has been used to study the impact of genetic variation on susceptibility to IAV infection and map QTL associated with these effects. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boon <em>et al.</em> (2009)</a> studied H5N1 infection in females from 66 BXD strains, and <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Nedelko <em>et al.</em> (2012)</a> studied H1N1 infection in 53 BXD strains, with both studies identifying QTL associated with susceptibility to infection. The Collaborative Cross (CC) RI panel is a multiparental population (MPP) descended from eight inbred founder strains (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Threadgill <em>et al.</em> 2002</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Churchill <em>et al.</em> 2004</a>), with these founders including representatives from the three major domesticated house mouse subspecies (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Yang <em>et al.</em> 2011</a>). As such, the CC captures considerably more genetic diversity and, thanks to its breeding structure, this diversity is also more uniformly distributed across the genome, with as many as eight distinct haplotypes segregating at any given locus within the population (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Srivastava <em>et al.</em> 2017</a>). The eight CC founder strains have distinct pathogenesis profiles in response to influenza virus (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Leist <em>et al.</em> 2016</a>), suggesting that the CC RI panel is capable of a broader phenotypic range than would be observed in less complex populations. Indeed, studies using an incompletely inbred, ancestor population of the CC (pre-CC), demonstrated high levels of phenotypic variation across the population and successfully mapped several QTL associated with variation in susceptibility to IAV infection (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bottomly <em>et al.</em> 2012</a>). The CC therefore represents a promising resource for studying how genetically diverse populations respond to IAV infection.</p><p id="p-9">Determining an optimal strategy for how the CC should be used to study the genetic architecture of IAV pathogenesis is nonetheless complicated because (1) the space of possible experimental designs is vast, and (2) information about what types of heritable effects are likely to be present is extremely limited. Regarding (1), with ∼75 CC strains currently available, including all reciprocal F1 hybrids (so called CC-RIXs), there are &gt;5600 potential replicable configurations. Since only a subset of these configurations can be explored within any realistic experiment, any chosen experimental design necessarily targets some types of heritable effects to the exclusion of others. Regarding (2), to date, most <em>in vivo</em> studies of IAV pathogenesis have been confined to candidate genes or additive interactions at single loci; studies investigating the broader question of what types of heritability are at play during IAV infection are largely absent.</p><p id="p-10">To rationally design studies of heritable effects in complex populations such as the CC it is therefore helpful to have advance knowledge of which types of heritable effects might be present. One source of such information is phenotype data collected on the multiparental founders and their F1 hybrid offspring, a combination that can be more formally described as an (inbred) diallel. Diallels have a long history in quantitative genetics, having been used originally in plant breeding studies to judge the relative merits of different strain combinations and subsequently for gaining insight into the heritable architecture of a broad range of phenotypes [<em>e.g.</em>, references in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Christie and Shattuck (1992)</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> (2012)</a>, and <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Okoro and Mbajiorgu (2017)</a>], including host–pathogen interactions in, <em>e.g.</em>, crickets (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Rantala and Roff 2006</a>) and flies (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Wayne <em>et al.</em> 2011</a>) (see <em>Statistical Models and Methods</em> and <em>Discussion</em> for connections to other diallel literature).</p><p id="p-11">Here we use a diallel of the CC founders and their reciprocal F1 hybrids (hereafter, a CC founder diallel) to give an overall predictive picture of the range and relative influence of the different types of heritable effects on IAV pathogenesis that are likely to be present in CC founder-derived MPPs, a group that includes not only replicable MPPs such as the CC and the CC-RIX but also irreplicable ones such as the Diversity Outbred (DO) population (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Svenson <em>et al</em>., 2012</a>). We take advantage of the diallel design’s replicability to measure IAV-induced pathogenesis in a precise way, as the response to an applied treatment defined in terms of postinfection (p.i.) weight-loss differences (deltas) between matched sets of mock and infected individuals (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Adapting a recently developed statistical framework for analyzing treatment-response diallels (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> 2014</a>), we use those deltas to model how pathogenic response to IAV is modulated by parentage, sex, and their interaction, framed in terms of additive genetics, dominance, epistasis, parent-of-origin, and sex-specific versions thereof.</p><p id="p-12">After observing that, following IAV infection, diallel individuals show a broad, continuous distribution of day 4 (D4) p.i. weight loss, we find, through statistical modeling, that the IAV-induced weight loss includes substantial contributions of host additive, epistatic, and sex-specific effects, with much of the heritable variation closely tracking the genotype state implied by the three distinct functional alleles of the previously identified resistance locus <em>Mx1</em>. Confirming previous findings, the functional CAST/EiJ (CAST) <em>Mx1</em> allele, in contrast with functional NZO/HlLtJ (NZO) and PWK/PhJ (PWK) <em>Mx1</em> alleles, provides intermediate levels of protection against H1N1 influenza strains. Unexpectedly, and confirmed through additional modeling, we found that different classes of functional <em>Mx1</em> alleles exhibit distinct functional patterns, additive or dominant, when combined with null <em>Mx1</em> alleles. Further, illustrating our general rationale, we show that the major strain-specific, <em>Mx1</em>-effect patterns are consistent across two CC founder-derived MPPs: the pre-CC, as determined from reanalysis of a previously published data set (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>), and a previously unpublished 105-line CC-RIX, in which we conduct a limited analysis focused on the <em>Mx1</em> locus.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/427/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotype and treatment-response classes for analysis of IAV infection in the diallel. Each filled square represents a weight or weight-change phenotype that is modeled independently. The gray square represents the starting body weight in all animals, prior to treatment, at D0 (analyzed with model 1 in Table 2). Light blue squares represent animals that were mock treated and red squares represent animals infected with IAV, with daily weights for each taken from D1 to D4 p.i. (and these were analyzed with model 2 in Table 2). Purple squares represent infection response, the primary quantity of interest, estimated using match quartets of one mock to three infected mice (analyzed with models 3 and 4 in Table 2). Labels within each square indicate phenotypes analyzed, where weight = preinfection body weight, pct = p.i. percent change in starting D0 weight (post), and &#x201C;delta&#x201D; = infection response, as described in the Statistical Models and Methods section. The coloring increases in saturation from D1 to D4 for the influenza and matched quartet groups to indicate an overall increasing amount of p.i. weight loss over time." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1976058624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotype and treatment-response classes for analysis of IAV infection in the diallel. Each filled square represents a weight or weight-change phenotype that is modeled independently. The gray square represents the starting body weight in all animals, prior to treatment, at D0 (analyzed with model 1 in Table 2). Light blue squares represent animals that were mock treated and red squares represent animals infected with IAV, with daily weights for each taken from D1 to D4 p.i. (and these were analyzed with model 2 in Table 2). Purple squares represent infection response, the primary quantity of interest, estimated using match quartets of one mock to three infected mice (analyzed with models 3 and 4 in Table 2). Labels within each square indicate phenotypes analyzed, where weight = preinfection body weight, pct = p.i. percent change in starting D0 weight (post), and &#x201C;delta&#x201D; = infection response, as described in the &lt;em&gt;Statistical Models and Methods&lt;/em&gt; section. The coloring increases in saturation from D1 to D4 for the influenza and matched quartet groups to indicate an overall increasing amount of p.i. weight loss over time.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/427/F1.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/427/F1.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/427/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/427/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488358" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Phenotype and treatment-response classes for analysis of IAV infection in the diallel. Each filled square represents a weight or weight-change phenotype that is modeled independently. The gray square represents the starting body weight in all animals, prior to treatment, at D0 (analyzed with model 1 in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Light blue squares represent animals that were mock treated and red squares represent animals infected with IAV, with daily weights for each taken from D1 to D4 p.i. (and these were analyzed with model 2 in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Purple squares represent infection response, the primary quantity of interest, estimated using match quartets of one mock to three infected mice (analyzed with models 3 and 4 in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Labels within each square indicate phenotypes analyzed, where weight = preinfection body weight, pct = p.i. percent change in starting D0 weight (post), and “delta” = infection response, as described in the <em>Statistical Models and Methods</em> section. The coloring increases in saturation from D1 to D4 for the influenza and matched quartet groups to indicate an overall increasing amount of p.i. weight loss over time.</p><div class="sb-div caption-clear"></div></div></div><div class="section methods-materials" id="sec-1"><h2 class="">Experimental Materials and Methods</h2><div id="sec-2" class="subsection"><h3>CC founder diallel mice</h3><p id="p-14">The inbred and F1 mice used within this study were bred in-house at the University of North Carolina at Chapel Hill (UNC-CH). This colony was directly descended from the subset of animals used to generate the initial CC funnels (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Collaborative Cross Consortium 2012</a>) and included mice from the following eight strains at The Jackson Laboratory: A/J (AJ; #000646), C57BL/6J (B6; #000664), 129S1/SvImJ (129; #002448), NOD/ShiLtJ (NOD; #001976), NZO (#002105), CAST (#000928), PWK (#003715), and WSB/EiJ (WSB; #001145). Mice from the UNC-CH colony were then used to generate all 62 possible inbred and (reciprocal) F1 combinations between these eight strains, excluding NZO × CAST and NZO × PWK matings which are nonproductive (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chesler <em>et al.</em> 2008</a>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). This yielded a total of 124 distinct combinations of sex and parentage (hereafter, described as “diallel categories”). Lung tissues were collected from a subset of each of the founder inbred strains in this study, at day 2 (D2) and D4 p.i., and were used for a separate comparative RNA-seq analysis by <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Xiong <em>et al.</em> (2014)</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/427/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram of breeding strategy for diallel, pre-CC, and CC-RIX. (A) The diallel cross produces inbred (n = 8) and F1 (n = 54 lines) genotypes from an 8 &#xD7; 8 cross of inbred strains. (B) The pre-CC is comprised of incompletely inbred (n = 155 lines) genotypes from 155 inbreeding funnels. (C) The CC-RIX produces F1 hybrid lines (n = 105 lines) from a sparse, round robin-like cross of 65 inbred CC strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1976058624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diagram of breeding strategy for diallel, pre-CC, and CC-RIX. (A) The diallel cross produces inbred (&lt;em&gt;n&lt;/em&gt; = 8) and F1 (&lt;em&gt;n&lt;/em&gt; = 54 lines) genotypes from an 8 &#xD7; 8 cross of inbred strains. (B) The pre-CC is comprised of incompletely inbred (&lt;em&gt;n&lt;/em&gt; = 155 lines) genotypes from 155 inbreeding funnels. (C) The CC-RIX produces F1 hybrid lines (&lt;em&gt;n&lt;/em&gt; = 105 lines) from a sparse, round robin-like cross of 65 inbred CC strains.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/427/F2.medium.gif" width="440" height="158"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/427/F2.medium.gif" width="440" height="158"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/427/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/427/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-15" class="first-child">Diagram of breeding strategy for diallel, pre-CC, and CC-RIX. (A) The diallel cross produces inbred (<em>n</em> = 8) and F1 (<em>n</em> = 54 lines) genotypes from an 8 × 8 cross of inbred strains. (B) The pre-CC is comprised of incompletely inbred (<em>n</em> = 155 lines) genotypes from 155 inbreeding funnels. (C) The CC-RIX produces F1 hybrid lines (<em>n</em> = 105 lines) from a sparse, round robin-like cross of 65 inbred CC strains.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Mouse infections in the diallel</h3><p id="p-16">Mice were weaned at ∼21-d old and housed four per cage, within each diallel category, under standard conditions (12 hr light/dark; food and water <em>ad libitum</em>). Of the four mice in a cage, one was randomly assigned to mock and three to influenza infection, as there is no evidence that mice can transmit influenza virus. Each cage was then assigned to a harvest time point: D2 p.i. (<em>n</em> = 533 mice), or D4 p.i. (<em>n</em> = 510 mice).</p><p id="p-17">At 8–12 wk of age, based on their assignments, mice were anesthetized with isoflurane and inoculated intranasally with 500 plaque-forming units (PFU) of mouse-adapted IAV (H1N1 A/Puerto Rico/8/1934; short name PR8) or with the diluent, phosphobuffered saline (PBS), alone as a mock control. For each inbred strain and F1 cross, about six mice (range: 5–9) of each sex were infected with IAV PR8, and about two mice (range: 2–3) of each sex were mock infected. This gave a total of 1043 mice across 54 experimental batches. Treatment assignment was random: same-sex siblings from the same cage (and therefore batch) were randomly assigned at weaning to mock or infected groups prior to being moved to new cages. The 1043 mice were housed in ∼260 cages (about four mice per cage), with 775 infected mice and 268 mock-infected mice. Body mass was recorded daily. All animal experiments were carried out in compliance with the Guide for the Care and Use of Laboratory Animals (Institute of Laboratory Animal Resources, National Research Council, 1996, <a href="https://www.ncbi.nlm.nih.gov/books/NBK232589/">https://www.ncbi.nlm.nih.gov/books/NBK232589/</a>). Animal protocols were approved by the Institutional Animal Care and Use Committee of UNC-CH.</p></div><div id="sec-4" class="subsection"><h3>Mouse infections in the pre-CC and CC-RIX</h3><p id="p-18">To verify that strain-specific haplotype effects measured in the diallel were consistent with those at the host resistance locus <em>Mx1</em>, we sought out CC-related IAV infection data sets for which we could isolate <em>Mx1</em> locus-specific effects.</p><div id="sec-5" class="subsection"><h4>Existing data from pre-CC study:</h4><p id="p-19">In the QTL mapping study of host response to IAV infection of <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> (2013)</a>, 155 female pre-CC mice from as many pre-CC lines were infected with IAV (PR8) at 8–12 wk of age and assayed for p.i. weight loss via daily weights, with phenotypes collected including starting weight (D0) and weight at D4 p.i. (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). This study did not include mock-infected mice.</p></div><div id="sec-6" class="subsection"><h4>CC-RIX study:</h4><p id="p-20">In total, 1402 female mice were bred from 105 F1 crosses of CC strains (<em>i.e.</em>, 105 CC-RIX lines) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a> and Supplemental Material Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>), as part of an ongoing QTL mapping study. These mice were infected at 8–12 wk of age with 5000 PFU IAV (A/California/04/2009; short name CA04), a human 2009 pandemic H1N1 isolate (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Itoh <em>et al.</em> 2009</a>), and phenotypes were collected, including starting weight (D0) and weight at D7 p.i. CC-RIX were bred under similar conditions to diallel mice. This experiment, whose broader analysis is still ongoing, included both flu-infected and mock-treated mice. However, since the design did not match these to the same exacting degree as the diallel, with mock controls missing entirely for some batch/line combinations, in the current study we consider data from the infected mice only. CC animals used to generate CC-RIX lines were purchased from the Systems Genetics Core at UNC-CH; information about CC strains available for distribution is found at <a href="http://csbio.unc.edu/CCstatus/index.py?run=AvailableLines">http://csbio.unc.edu/CCstatus/index.py?run=AvailableLines</a> (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Morgan and Welsh 2015</a>).</p></div></div></div><div class="section methods" id="sec-7"><h2 class="">Statistical Models and Methods</h2><p id="p-21">Our statistical analysis of heritable effects in the diallel (hereafter, diallel effects) relies heavily on the BayesDiallel model and approach described by <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> (2012)</a> and <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> (2014)</a>. BayesDiallel was originally proposed in <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> (2012)</a> for diallel analysis of routine, single outcome phenotypes, describing how the mean value of those phenotypes was shifted by changes in parentage and sex. Although in some ways the method was built upon a canon of existing diallel literature (<em>e.g.</em>, references in <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Christie and Shattuck 1992</a>), including more recent work that used random effects (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Zhu and Weir 1996</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Tsaih <em>et al.</em> 2005</a>) and Bayesian hierarchical modeling (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Greenberg <em>et al.</em> 2010</a>), in other ways it represents a new parameterization and a generalization of many earlier methods [see <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> (2012)</a> for explicit connections to those methods]. In <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> (2014)</a>, we extended BayesDiallel to treatment-response phenotypes, in particular, to when the modeled outcome is the phenotypic difference between placebo and treated matched pairs; the model in this case describes a causal effect modification, or, in a slight abuse of terminology, a gene-by-treatment (G × T) effect. Herein, that treatment-response approach is extended further to our more complex matching regime of quartets rather than pairs, and with a different imputation procedure to deal with quartets that are incomplete.</p><p id="p-22">This section begins by reviewing the BayesDiallel model for single outcome phenotypes. This is used not only to analyze our primary baseline phenotype, body weight at day 0 (D0 weight), but is also foundational for our subsequent analyses. Then we introduce our definition of infection response based on matched quartets, which gives rise to treatment responses defined for each of four time points (D1, D2, D3, and D4 p.i.), and describe how they are modeled using BayesDiallel. The analysis is then modified further to estimate the impact of haplotype state at the resistance locus <em>Mx1</em>, and we describe how the interaction of haplotype pairs at this locus is examined by estimating relative degrees of haplotype additivity and dominance. Finally, we describe an illustrative comparative analysis of the effect of the <em>Mx1</em> locus on IAV response in pre-CC and CC-RIX mice.</p><div id="sec-8" class="subsection"><h3>Diallel model for single outcome phenotypes</h3><p id="p-23">Diallel effects for single outcome phenotypes, that is, phenotypes measured as a single value per mouse, were modeled using the “fulls” model of BayesDiallel (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> 2012</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> 2014</a>). BayesDiallel is a Bayesian linear mixed model that decomposes phenotypic variation into separate heritable components corresponding to additive genetics, dominance/inbred effects, parent-of-origin (“maternal”), epistasis, and all sex-specific versions thereof. It models the phenotype value <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-1.gif"/></noscript></span></span> of mouse <em>i</em> as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>μ</em> is the intercept, and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-3.gif"/></noscript></span></span> is the residual error, normally distributed as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-4.gif"/></noscript></span></span> with variance <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-5.gif"/></noscript></span></span> The <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-6.gif"/></noscript></span></span> term represents the contribution of an arbitrary set of user-specified fixed effect covariates, with predictors encoded in vector <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-7.gif"/></noscript></span></span> and fixed effects <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-8.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-9.gif"/></noscript></span></span> term represents the contribution of an arbitrary set of <em>R</em> user-defined random effect covariates, which for single outcome phenotypes in this study always includes an effect of experimental batch; and the <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-10.gif"/></noscript></span></span> term represents the contribution of heritable components of the diallel, written as a linear combination of the diallel effects vector <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-11.gif"/></noscript></span></span> and diallel category vector <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-12.gif"/></noscript></span></span> Here <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-13.gif"/></noscript></span></span> is shorthand for <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-14.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-15.gif"/></noscript></span></span> denotes <em>i</em>’s diallel category, that is, its unique combination of mother strain <em>j</em>, father strain <em>k</em>, and sex <em>s</em>. The diallel category vector <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-16.gif"/></noscript></span></span> is defined with the diallel effects <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-17.gif"/></noscript></span></span> so as to give the linear combination shown in Equation 2, where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-18.gif"/></noscript></span></span> is the additive effect of strain <em>j</em> (<em>e.g.</em>, the additive effect parameter <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-19.gif"/></noscript></span></span> is the expected increase in phenotype on adding one haploid genome of strain AJ); <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-20.gif"/></noscript></span></span> is an additional increase in phenotype induced by strain <em>j</em> being the mother (parent-of-origin effect); indicator <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-21.gif"/></noscript></span></span> is 1 if <em>X</em> is true and 0 otherwise; <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-22.gif"/></noscript></span></span> is the overall effect of being inbred; <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-23.gif"/></noscript></span></span> is the additional effect of being inbred for strain <em>j</em>; <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-24.gif"/></noscript></span></span> is the additional effect of combining strains <em>j</em> with <em>k</em> regardless of which is the mother (symmetric epistasis); indicator <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-25.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-26.gif"/></noscript></span></span> if <em>X</em> is true and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-27.gif"/></noscript></span></span> otherwise; <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-28.gif"/></noscript></span></span> is a deviation from <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-29.gif"/></noscript></span></span> induced by parent-of-origin (asymmetric epistasis); <em>φ</em> is the effect of being female rather than male; and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-30.gif"/></noscript></span></span> is the sex-specific deviation from additive effect <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-31.gif"/></noscript></span></span> with other superscripted <em>φ</em> terms (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-32.gif"/></noscript></span></span>) defined analogously. Each set of related variables, <em>e.g.</em>, the additive effects <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-33.gif"/></noscript></span></span> for <em>J</em> parents, is modeled as a group via a constrained normal distribution, that is, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-34.gif"/></noscript></span></span> but subject to <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-35.gif"/></noscript></span></span> after <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> (2014)</a>. The variance of each group, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-36.gif"/></noscript></span></span> was modeled with a weak inverse gamma prior, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-37.gif"/></noscript></span></span> with this prior also used for the residual variance <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-38.gif"/></noscript></span></span> The prior for fixed effects, <em>e.g.</em>, <em>μ</em>, is set to a vague normal distribution, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-39.gif"/></noscript></span></span> A summary of the diallel effects parameters is given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Model fitting proceeded using Markov chain Monte Carlo (MCMC) via Gibbs sampling (algorithm in <a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> 2012</a>), with results based on samples from <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-40.gif"/></noscript></span></span> MCMC iterations (five chains of length 2500, after 500 iterations burn-in). See also the later section <em>Reporting BayesDiallel results: highest posterior density</em>, <em>model inclusion probability</em>, <em>variance projection</em>, <em>and treatment response variance projections</em>.</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-41.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488456/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488456/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Model parameters, random and fixed (overall), from Equations 1, 2, and 5</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Modeling infection response as mock-corrected percent change in body weight p.i.</h3><p id="p-25">A standard measure used to assess pathogenesis in IAV-infected mice is weight loss. Weight loss correlates with several host and viral factors, including viral load, immune response phenotypes, and lung histopathology (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Leist <em>et al.</em> 2016</a>); as such, it provides a simple, noninvasive measure of infection pathology that can be assessed for a large number of mice. We measured the percentage change in body weight relative to D0,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-57.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>for mouse <em>i</em> on <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-58.gif"/></noscript></span></span> in <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-59.gif"/></noscript></span></span> where, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-60.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-61.gif"/></noscript></span></span> are the body weights for IAV-infected mouse <em>i</em> at D4 and at D0, respectively. These measures, which we describe as single outcome phenotypes, were analyzed using BayesDiallel as above (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), but they were not the main focus of our study. Our main focus was a derived measure, IAV-infection response, defined next.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488524/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488524/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Models used for each analysis in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">In defining IAV-infection response, we note that from a causal inference perspective (described more fully in Appendix A) weight loss in an IAV-infected mouse (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-83.gif"/></noscript></span></span>) reflects two confounded processes: weight loss due to IAV-induced pathogenesis, and weight loss due to other aspects of the experimental procedure. To obtain an unconfounded estimate of weight loss due to IAV-induced pathogenesis alone, we defined IAV-infection response as the difference between weight loss in mice subject to infection by IAV and those subject to mock. Specifically, since in our experimental design we match one mock mouse to three infected—this reflecting our expectation that phenotypes from infected mice will be more variable and will thus need more replicates for comparable precision—infection response was defined in terms of “matched quartets,” <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-84.gif"/></noscript></span></span> where each matched quartet <em>q</em> comprised four mice of the same diallel category from the same experimental batch, with the first three mice, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-85.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-86.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-87.gif"/></noscript></span></span> being IAV infected and the last mouse, <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-88.gif"/></noscript></span></span> receiving mock treatment. Infection response at a given day for quartet <em>q</em> was thus defined as a delta,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-89.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>following the more general definition in Equation A2 in Appendix A.</p><p id="p-33">Diallel effects on infection response were then modeled using BayesDiallel in manner analogous to the single outcome case in Equation 1, as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-90.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where now the unit of observation is the matched quartet <em>q</em> rather than the individual <em>i</em> and where, for example, <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-91.gif"/></noscript></span></span> is shorthand for <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-92.gif"/></noscript></span></span> the diallel category appropriate for <em>q</em>. The shift to modeling treatment response does, however, change how the parameters are interpreted. The intercept in the above formula, relabeled as <em>θ</em>, now acquires a special meaning, representing an overall causal effect due to infection, and the diallel effects in <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-93.gif"/></noscript></span></span> now describe how that causal effect is modified by parentage, sex, and their interaction. For example, the additive effect parameter <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-94.gif"/></noscript></span></span> is the expected increase in infection response on adding one haploid genome of strain AJ. Regarding covariates, as for the single outcome phenotypes, this model included a random effect of batch and, to reduce potential dependence between the delta’s and baseline body weight, we also included a fixed effect covariate for the quartet mean D0 weight (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-95.gif"/></noscript></span></span>) in <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-96.gif"/></noscript></span></span> (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-34">Although our experimental design stipulated even multiples of four mice per diallel category, practical constraints on animal breeding and availability meant that in some cases this number was three or five, such that some quartets had either missing infecteds or surplus mocks. To ensure the definition of delta in Equation 4 remained consistent, and in particular that delta’s from different quartets had comparable precision, the diallel analysis was performed on <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-97.gif"/></noscript></span></span> imputed versions of the data, with each imputed data set being comprised of exact quartets in which missing phenotypes had been filled using stochastic regression imputation and surplus mocks had been (randomly) deleted (details in Appendix B). On each imputed data set we collected 125 MCMC samples from 12,500 total time steps (<em>i.e.</em>, by recording values at every 100th time step); results were based on the aggregate of these samples from the <em>M</em> imputed data sets (<em>i.e.</em>, on 125,000 MCMC samples in total).</p></div><div id="sec-10" class="subsection"><h3>Reporting BayesDiallel results: highest posterior density, model inclusion probability, variance projection, and treatment response variance projections</h3><p id="p-35">Point and interval estimates of individual diallel effects, <em>e.g.</em>, additive effect <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-98.gif"/></noscript></span></span> are reported as posterior means and 95% highest posterior density (HPD) intervals. The overall contribution of a particular inheritance group is reported in two ways: as a variance projection (VarP), <em>e.g.</em>, VarP[a] for the contribution of additive effects to a phenotype or treatment response VarPs (TreVarPs), <em>e.g.</em>, TreVarP[a] for the contribution of additive effects to an infection response; and as a model inclusion probability (MIP), <em>e.g.</em>, MIP[a] for the probability of additive effects being included in the model.</p><p id="p-36">The VarP is a heritability-like measure that predicts how much of the total phenotypic sum of squares would be explained by each component in a new, completely balanced diallel. Unlike traditional heritability, it is calculated based on the effects, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-99.gif"/></noscript></span></span> rather than the variance components, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-100.gif"/></noscript></span></span> and as such benefits not only from greater interpretability but also from the stability and accuracy provided by hierarchical shrinkage (as detailed in <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> 2014</a>). Since the VarP is a function of the posterior predictive distribution and calculated at each iteration of the MCMC, it is reported via Bayesian posterior summaries, specifically, the posterior median and the 95% HPD interval. The VarPs for infection response phenotypes are, following <a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> (2014)</a>, given the special name of TreVarPs to acknowledge their more delicate interpretation.</p><p id="p-37">The MIP reflects a different type of inference: rather than being a function of the parameters estimated in the full, sexed BayesDiallel model of Equations 1 and 2, it describes the results of model selection, that is, an assessment of which diallel categories could be excluded without a substantial loss in fit. As in <a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> (2014)</a>, we use the exclusionary Gibbs group sampler of <a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> (2012)</a>. Each diallel category is set to have a prior inclusion probability of 0.5, reflecting a prior opinion that inclusion and exclusion are equally likely. This prior is then updated by the phenotype data and the model selection procedure to give a (posterior) MIP. MIPs are interpreted following the conventions in <a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> (2014)</a>: MIPs in the range (0.25, 0.75) indicate that the data does not provide sufficient evidence to make an informed decision about exclusion or inclusion; MIPs within (0.05, 0.25] or [0.75, 0.95) represent positive evidence for exclusion or inclusion respectively; (0.01, 0.05] or [0.95, 0.99) represent strong evidence; and [0, 0.01] or [0.99, 1] represent strong to decisive evidence. These conventions are based on those proposed by <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kass and Raftery (1995)</a> for Bayes factors, which are connected to MIPs by the relation<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-101.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-102.gif"/></noscript></span></span> is the prior inclusion probability, and where the above simplifies to <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-103.gif"/></noscript></span></span> in our case of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-104.gif"/></noscript></span></span></p></div><div id="sec-11" class="subsection"><h3>Estimating Mx1 effects in the diallel</h3><p id="p-38">The critical host resistance factor (<em>Mx1</em>) has been shown to drive IAV resistance in the CC founder strains and has been mapped in the pre-CC (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>). <em>Mx1</em> was previously described as having three major, naturally occurring functional classes of resistance to influenza H1N1 arising from the subspecies <em>Mus musculus domesticus</em> (hereafter, <em>dom</em>; members include AJ, B6, 129, NOD, and WSB), <em>M. musculus castaneus</em> (<em>cast</em>; CAST), and <em>M. musculus musculus</em> (<em>mus</em>; PWK and NZO), of which <em>domesticus</em> is considered to be null whereas <em>musculus</em> and <em>castaneus</em> are protective. (Note that <em>domesticus Mx1</em> in the CC founder strains is comprised of two unique null alleles, and that the subspecific <em>Mx1</em> alleles observed in the CC may not be representative of the those segregating in the wild.) To estimate the contribution of <em>Mx1</em> haplotypes as discernible in the diallel, and thereby also estimate the extent of heritable effects that remain after <em>Mx1</em> is controlled for, we define the following haplotype combinations (diplotypes) as six levels of the random effect, <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-105.gif"/></noscript></span></span> {<em>dom</em> × <em>dom</em>}, {<em>dom</em> × <em>cast</em>}, {<em>cast</em> × <em>cast</em>}, {<em>cast</em> × <em>mus</em>}, {<em>mus</em> × <em>dom</em>}, and {<em>mus</em> × <em>mus</em>}; we then repeat our diallel analysis with this effect included (model 4 in <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>).</p><div id="sec-12" class="subsection"><h4>Estimating a dominance index for Mx1 alleles:</h4><p id="p-39">Dominance is typically defined in the context of bialleles, but since in this population <em>Mx1</em> has a multiallelic series, we define dominance of <em>Mx1</em> between allele pairs. Following <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kacser and Burns (1981)</a>, which is built on the work of <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Wright (1934)</a>, we define the “dominance index” for a wild-type (wt) against a mutant (mut) allele as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-106.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where values for <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-107.gif"/></noscript></span></span> are close to −0.5 when the effect of the wt is overdominant to the mut (the effect of the mut is underrecessive), 0 when the effect of the wt is completely dominant to the mut (the effect of the mut is recessive), close to 0.5 when the effect of the wt is additive (not dominant, or incompletely dominant) to the mut, close to 1 when the effect of the wt is recessive (the effect of the mut is dominant), and close to 1.5 when the effect is underrecessive (the effect of the mut is overdominant). Overdominance is given by values of <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-108.gif"/></noscript></span></span> that are much less than zero and underdominance by values that are much greater than one. This definition is used to define dominance indices <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-109.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-110.gif"/></noscript></span></span> describing the degree of dominance of the protective alleles <em>castaneus</em> and <em>musculus</em>, respectively, against the null allelle <em>domesticus</em>. To assess the degree to which <em>castaneus</em> and <em>musculus</em> differ in their relation to <em>domesticus</em>, we further define a “dominance difference index,”<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-111.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where negative values indicate that <em>musculus</em> has more of a dominance-based relationship to <em>domesticus</em> than does <em>castaneus</em>, positive values indicate the converse, and zero indicates that the relationships of <em>castaneus</em> and <em>musculus</em> to <em>domesticus</em> show dominance equally.</p><p id="p-40">When the BayesDiallel model includes <em>Mx1</em> effects, the aforementioned dominance index and dominance difference index are both functionals of the posterior; posterior samples of these indices were therefore obtained by simply applying Equations 6 and 7 to the sampled <em>Mx1</em> effects at each time step of the MCMC.</p><p id="p-41">The <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kacser and Burns (1981)</a> dominance index is a simple reparameterization of the degree of dominance parameter, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-112.gif"/></noscript></span></span> defined by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Comstock and Robinson (1948)</a> and used by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gardner and Lonnquist (1959)</a>. In the Comstock–Robinson model, the mean-centered phenotypes are coded as (translating from our model above): <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-113.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-114.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-115.gif"/></noscript></span></span> This gives the relation <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-116.gif"/></noscript></span></span> or equivalently, <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-117.gif"/></noscript></span></span> This alternate dominance parameterization is explored further using BayesDiallel in <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Turner <em>et al.</em> (2017)</a>.</p></div></div><div id="sec-13" class="subsection"><h3>Estimating haplotype effects at the Mx1 locus in the pre-CC and CC-RIX</h3><p id="p-42">The additive effect parameters estimated in the diallel do not precisely distinguish the effects at the <em>Mx1</em> locus because they are confounded with any potential genome-wide effects that follow the same pattern of strain classification. An unconfounded estimate of haplotype effects at <em>Mx1</em> requires a population in which the remainder of the genome is randomized, <em>e.g.</em>, by recombination. To this end, we make use of two related data sets on IAV-induced weight loss in two CC-derived MPPs: IAV (PR8) infection in the pre-CC and IAV (CA04) infection in a set of CC-RIX lines. These two studies, described in more detail below, were in other respects less rigorous than our diallel: the experimental measurement of the infection response was based on infected mice only with no mocks in the pre-CC, and although mocks were collected in the CC-RIX, their relative sparsity (200–300 mocks to &gt;1400 infecteds) complicates analysis based on matching alternate treatment groups; the experimental batching was subject to a less exacting degree of randomization across genetically distinct categories; the available combinations of <em>Mx1</em> diplotypes are limited mostly to homozygotes in the pre-CC, and incompletely and unevenly sampled in the CC-RIX; and the <em>Mx1</em> diplotype state for each line is known only probabilistically, having been inferred by hidden Markov models (HMMs) applied to genotyping data. Nonetheless, if effects at the <em>Mx1</em> locus were largely independent of those elsewhere in the genome, we might expect that <em>Mx1</em> effects in the pre-CC and CC-RIX would be broadly consistent with those in the diallel.</p><p id="p-43">Estimation of haplotype effects at the <em>Mx1</em> locus was performed using the Diploffect model (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Zhang <em>et al.</em> 2014</a>), a Bayesian hierarchical model that estimates effects of diplotype substitutions at a specified QTL when the diplotype states themselves are known only probabilistically. The effects estimated by Diploffect are analogous to those estimated by BayesDiallel: phenotype <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-118.gif"/></noscript></span></span> of mouse <em>i</em> is modeled as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-119.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-120.gif"/></noscript></span></span> is a vector representing the diplotype state of mouse <em>i</em> at the QTL and is shorthand for <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-121.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-122.gif"/></noscript></span></span> denotes <em>i</em>’s diplotype state comprised of haplotypes from CC founder strains <em>j</em> and <em>k</em>, <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-123.gif"/></noscript></span></span> are the corresponding effects, and all other variables are as in Equation 1. The diplotype vector <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-124.gif"/></noscript></span></span> is defined with <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-125.gif"/></noscript></span></span> so as to give the linear predictor<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-126.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-127.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-128.gif"/></noscript></span></span> are additive (haplotype) effects modeled as <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-129.gif"/></noscript></span></span> broadly equivalent to the additive effects in BayesDiallel’s Equation 2; and <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-130.gif"/></noscript></span></span> are dominance deviations, which are the converse to BayesDiallel’s inbred parameters. Dominance deviations are expected to be poorly informed when heterozygotes are sparsely represented, as in the CC-RIX and in particular the largely inbred pre-CC, but are nonetheless included to stabilize inference of additive effects. For numerical stability, phenotypes were first centered and scaled to unit variance, and variance parameters (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-131.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-132.gif"/></noscript></span></span> where effect is add, dom, or <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-133.gif"/></noscript></span></span>) were given mildly informative priors of the form <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-134.gif"/></noscript></span></span> Estimation proceeded by importance sampling (the DF.IS and DF.IS.kinship methods in <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Zhang <em>et al.</em> 2014</a>) using integrated nested Laplace approximations (INLA; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Martins <em>et al.</em> 2013</a>), with 100 importance samples taken, and parameter estimates for additive effects are reported as posterior means, posterior medians, and HPD intervals.</p><div id="sec-14" class="subsection"><h4>Pre-CC study:</h4><p id="p-44">In the study of <a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> (2013)</a>, IAV-infection response was measured on 155 mice from as many pre-CC lines as weight loss following infection with IAV (PR8 variant, as for the diallel). QTL mapping of D4 p.i. weight loss, equivalent to <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-135.gif"/></noscript></span></span> in the diallel study, identified a QTL, <em>HrI1</em>, containing the <em>Mx1</em> gene, with peak marker JAX00072951 (chr16:98,148,641; Mouse Diversity Array of <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Yang <em>et al.</em> 2009</a>). We estimated haplotype effects at this peak marker using Diploffect (<a id="xref-ref-100-3" class="xref-bibr" href="#ref-100">Zhang <em>et al.</em> 2014</a>), applied to the phenotype and the original HMM probabilities of <a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> (2013)</a>, with the model including a fixed effect covariate for D0 weight.</p></div><div id="sec-15" class="subsection"><h4>CC-RIX study:</h4><p id="p-45">For the CC-RIX study of infection response to IAV (CA04 strain), we calculated weight loss values for all 1402 infected mice at D7 p.i. (analogous to a <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-136.gif"/></noscript></span></span> measure), and for all 105 CC-RIX lines obtained diplotype probabilities at marker UNC27478095 (16:97,591,482; MegaMUGA array, described in <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Morgan <em>et al.</em> 2016</a>) from the Inbred Strain Variant database (ISVdb; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Oreper <em>et al.</em> 2017</a>). Haplotype effects were then estimated by Diploffect applied to debatched CC-RIX line means as follows. First, we fit a linear mixed model (by REML using the R package lme4 of <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bates <em>et al.</em> 2015</a>) to the individual-level phenotypes (<em>n</em> = 1402) with fixed effects of D0 weight and laboratory (two levels), and random effects of mating (107 levels: 105 RIXs + 2 additional levels distinguishing minor breeding differences, when CC010 and CC042 strains were rederived from breeder females into a new facility) and infection date (59 levels). The residuals of this model were then averaged over the <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-137.gif"/></noscript></span></span> mice of each CC-RIX line <em>i</em> and used as the response <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-138.gif"/></noscript></span></span> in Equation 8 with precision weighting <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-139.gif"/></noscript></span></span> and a between-line polygenic random effect <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-140.gif"/></noscript></span></span> where the <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-141.gif"/></noscript></span></span> genetic relationship matrix <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-142.gif"/></noscript></span></span> was calculated between all CC-RIX pairs based on the founder haplotype probabilities (dosages) at each locus, according to the method described in <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gatti <em>et al.</em> (2014)</a>.</p></div></div><div id="sec-16" class="subsection"><h3>Data availability</h3><p id="p-46">Analyses were conducted in the statistical programming language R (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">R Core Team 2017</a>). In addition to R packages cited above, we used the packages BayesDiallel (<a id="xref-ref-44-8" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> 2012</a>) and Diploffect.INLA (<a id="xref-ref-100-4" class="xref-bibr" href="#ref-100">Zhang <em>et al.</em> 2014</a>). The data, analysis software, and scripts are available on the flu-diallel repository on GitHub at <a href="https://github.com/mauriziopaul/flu-diallel">https://github.com/mauriziopaul/flu-diallel</a>. A static version is posted as a public, open-access Zenodo repository at <a href="http://dx.doi.org/10.5281/zenodo.293015">http://dx.doi.org/10.5281/zenodo.293015</a>. Phenotype data from the diallel and CC-RIX animals used in this study will be available on the Mouse Phenome Database (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Grubb <em>et al.</em> 2014</a>) at <a href="https://phenome.jax.org">https://phenome.jax.org</a> with persistent identifier RRID:SCR_003212.</p><p id="p-47"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS1.pdf">File S1</a> contains an account of the supplemental files which can be used to reproduce our analysis. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS2.gz">File S2</a> contains the software packages used for this analysis. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS3.csv">File S3</a> contains the diallel data file, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS4.gz">File S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS5.zip">File S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS6.gz">File S6</a> contain the data analysis files required for analyzing the diallel, pre-CC, and CC-RIX, respectively. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a> contains supplemental figures, tables, and an algorithm. After unzipping, the files FluDiData.csv, Flu-pre-CC-data.csv, and Flu-CC-RIX-data.csv contain raw phenotypes, cross (or line, strain), and mouse ID information from the three mouse populations used in this study. The script files MIMQ*.sh are used in bash to call R scripts to run the BayesDiallel analysis on diallel phenotypes. The script files main_analysis*.R are used with Diploffect to run Diploffect analysis on the pre-CC and CC-RIX phenotypes. Additional *.RData, *.pl, *.alleles, and *.csv files are uploaded which contain settings, genotypes, and founder haplotype probabilities used by the scripts.</p></div></div><div class="section results" id="sec-17"><h2 class="">Results</h2><p id="p-48">Mice from the eight inbred founder strains of the CC were used to generate a near-complete <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-143.gif"/></noscript></span></span> diallel. This study used offspring (<em>n</em> = 1043) of both sexes (519 females and 524 males) representing 62 of the 64 crosses (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>), including all inbred combinations (<em>n</em> = 129) and all F1 hybrids (<em>n</em> = 914) except NZO × CAST and NZO × PWK. Within each diallel category—defined as the combination of sex and (reciprocal) parentage—and in each experimental batch, mice were randomly assigned at weaning to infection or mock groups in a ratio of 3:1; complete sets of three infected with one mock were described as matched quartets. Mice in the infected group were inoculated with IAV PR8, and in the mock group with PBS. For each mouse, body weight was measured prior to infection (D0 or baseline weight), and at days 1–4 p.i. (D1, D2, D3, D4). D0 weight is reported in grams whereas p.i. weight is hereafter reported as a percentage of D0 weight, <em>e.g.</em>, D4pct. Not all mice survived the protocol: one infected mouse died after D3 weights were taken and one mouse died from anesthesia on D0.</p><div id="sec-18" class="subsection"><h3>F1 hybrids of the CC founders show a wide range of phenotypic outcomes</h3><p id="p-49">The CC founders include five strains we have previously characterized as susceptible to IAV-induced pathology (AJ, B6, 129, NOD, and WSB), two strains as resistant (NZO and PWK), and one (CAST) that exhibits a distinct intermediate weight loss phenotype (<a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>). Results for the inbred founders measured in our diallel replicate those earlier findings, and the p.i. weight loss among the infected F1 hybrids spanned the range of phenotypes observed in the founders (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>), consistent with the notion of IAV-induced weight loss being a complex trait with contributions from multiple loci.</p></div><div id="sec-19" class="subsection"><h3>Diallel effects on baseline mouse weight strongly replicate previous CC founder diallel studies</h3><p id="p-50">The effects of parentage and sex on D0 weight were estimated using BayesDiallel. Described further in <em>Statistical Models and Methods</em>, BayesDiallel decomposes the heritable effects observable in the diallel into 160 parameters (diallel effects) grouped into 13 distinct heritability classes. In sketch form, it models the average phenotype of mice of sex <em>s</em> bred from mother of strain <em>j</em> and father of strain <em>k</em> as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-144.gif"/></noscript></span></span>where covariates always includes experimental batch, <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-145.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-146.gif"/></noscript></span></span> are the additive effects of the two parents, <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-147.gif"/></noscript></span></span> is an additional effect included only when <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-148.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-149.gif"/></noscript></span></span> models the effects of further nuances of sex and parentage as deviations from this base model (listed in <em>Statistical Models and Methods</em> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-51">Diallel effects estimated for D0 weight are reported in Figure S6A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a> as 95% HPD intervals for each parameter, and two summary measures, VarPs and MIPs, for each of the 13 heritability classes are given in Figure S6, B and C, in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>. Briefly, VarPs (Figure S6C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>) report the contribution of the effect group as the proportion of the total phenotypic variance, whereas MIPs (Figure S6B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>) assess the strength of support for whether an effect group should be included at all, with probabilities near 1 providing stronger support for inclusion, probabilities near 0 supporting exclusion, and probabilities near 0.5 reflecting a lack of information either way.</p><p id="p-52">The pattern of effects for D0 weight was strikingly similar to that seen for baseline body weight in two previous diallels of the CC founders (<a id="xref-ref-44-9" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> 2012</a>; <a id="xref-ref-14-10" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> 2014</a>), despite those earlier studies being independent experiments with no particular attempt made to align experimental protocols, and included substantial additive effects, strain-specific parent-of-origin effects, signals of epistasis, and sex-specific versions thereof. For example, we largely replicated the pattern of inbred, additive, and maternal effects observed in both <a id="xref-ref-44-10" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> (2012)</a> and <a id="xref-ref-14-11" class="xref-bibr" href="#ref-14">Crowley <em>et al.</em> (2014)</a>, and also found a higher-order, sex-specific PWK × CAST symmetric epistatic effect in <a id="xref-ref-44-11" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> (2012)</a>. We also observed some new epistatic and sex-specific epistatic effects largely due to increased power from a larger sample size.</p></div><div id="sec-20" class="subsection"><h3>Diallel effects on IAV-infection response</h3><p id="p-53">Infection response was defined as the percentage change in body weight induced by IAV infection, with more negative values indicating more severe pathology. This was calculated at each time point—D1, D2, D3, and D4 p.i.—as the difference between matched infected and mock mice, yielding a single infection response number (a delta, <em>e.g.</em>, D4delta) for each matched quartet (three infected mice and one mock). The effects of parentage and sex on infection response were then analyzed for each time point separately using BayesDiallel as above, with an additional covariate of D0 weight (see <em>Statistical Models and Methods</em> for details). Although results are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a> for all time points, we will focus on those for D4 p.i. since this showed the greatest difference between infected and mock.</p></div><div id="sec-21" class="subsection"><h3>IAV infection causes weight loss through D4 p.i., with greater susceptibility in females</h3><p id="p-54">IAV infection in the diallel induced an overall mean change in body weight (<em>i.e.</em>, overall infection response <em>θ</em> in Equation 5 and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) of −0.13% (95% HPD interval: −0.48, 0.22; MIP = 1) on D1 p.i., −0.83% (−1.33, −0.32%; MIP = 1) on D2 p.i., −5.60% (−6.47, −4.73%; MIP = 1) on D3, and −8.85% (−9.92, −7.78%; MIP = 1) on D4 (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>; see also progression in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Consistent with previous mouse studies of sex effects on infection (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lorenzo <em>et al.</em> 2011</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Robinson <em>et al.</em> 2011</a>), females given the same dose of virus as male mice had increased weight loss: a negative effect of female sex was estimated at all four time points p.i., gradually increasing in magnitude from −0.89% (−1.45, −0.36%) at D1 p.i. to −2.11% (−3.87, −0.30%) at D4 p.i. (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), suggesting that enhanced susceptibility in females may occur at least as early as D1 p.i. Although all mice received the same dose of virus regardless of starting body weight, heavier mice experienced a transient increase in percent weight loss at D2 p.i. compared with lighter mice: the D0 weight effect (<em>α</em> in Equation 5) on the infection response at D2 p.i. was −0.31% (−0.52, −0.09%), such that for every 10 g of starting weight beyond 0 g, an additional ∼3.1% weight was lost on D2; however, this effect disappeared by D3 p.i. No other significant effects of starting weight on IAV-induced weight loss were detected at other time points, indicating that heavier mice were infected at least as effectively as lighter mice, and that starting body weight does not in general confound our exploration of strain- and cross-specific effects.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/427/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Influenza-induced weight loss in an 8 &#xD7; 8 diallel cross of mice, through 4 d p.i. Mean weight change, as % D0 weight, is shown at (A) D1, (B) D2, (C) D3, and (D) D4 p.i. with 500 pfu IAV (PR8) in male and female inbreds and F1 hybrids of CC founder strains (n = 774 for D1 and D2, n = 382 for D3, and n = 381 for D4). Results from mock-infected mice not shown. Squares with a gray &#x201C;X&#x201D; indicate matings that do not produce offspring." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1976058624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Influenza-induced weight loss in an 8 &#xD7; 8 diallel cross of mice, through 4 d p.i. Mean weight change, as % D0 weight, is shown at (A) D1, (B) D2, (C) D3, and (D) D4 p.i. with 500 pfu IAV (PR8) in male and female inbreds and F1 hybrids of CC founder strains (&lt;em&gt;n&lt;/em&gt; = 774 for D1 and D2, &lt;em&gt;n&lt;/em&gt; = 382 for D3, and &lt;em&gt;n&lt;/em&gt; = 381 for D4). Results from mock-infected mice not shown. Squares with a gray &#x201C;X&#x201D; indicate matings that do not produce offspring.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/427/F3.medium.gif" width="440" height="112"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/2/427/F3.medium.gif" width="440" height="112"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/427/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/427/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488340" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-55" class="first-child">Influenza-induced weight loss in an 8 × 8 diallel cross of mice, through 4 d p.i. Mean weight change, as % D0 weight, is shown at (A) D1, (B) D2, (C) D3, and (D) D4 p.i. with 500 pfu IAV (PR8) in male and female inbreds and F1 hybrids of CC founder strains (<em>n</em> = 774 for D1 and D2, <em>n</em> = 382 for D3, and <em>n</em> = 381 for D4). Results from mock-infected mice not shown. Squares with a gray “X” indicate matings that do not produce offspring.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/427/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diallel effects on host weight IAV-infection response, before and after accounting for Mx1 haplotypes. (A) Effect estimates for additive, maternal, inbred, and epistatic effects, including sex-specific effects, are presented as HPD intervals across 163 individual effects categories for IAV-induced weight change at D4 p.i. (phenotype D4delta). HPDs are given for each parameter, including 95% (thin line) and 50% (thick line) intervals, and median (white break) and mean (black vertical line). Parameters are labeled according to the methods. Symmetric epistatic, asymmetric epistatic, and sex-specific parameters are indicated by &#x201C;v:&#x201D;, &#x201C;w:&#x201D;, and &#x201C;f:&#x201D;, respectively. The overall treatment effect (data not shown), &#x3B8;, is &#x2212;8.85% (&#x2212;9.92, &#x2212;7.78%). (B and C) TReVarPs, a generalization of heritability for diallel effects classes, at D4 are shown for three fixed (overall) effects, five random effects classes, and five corresponding sex-specific random effects classes (posterior median and 95% HPD intervals) before (B) and (C) after accounting for diplotypes at the host influenza resistance locus, Mx1. (D and E) TReVarPs before and after Mx1 for all four p.i. time points." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1976058624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diallel effects on host weight IAV-infection response, before and after accounting for &lt;em&gt;Mx1&lt;/em&gt; haplotypes. (A) Effect estimates for additive, maternal, inbred, and epistatic effects, including sex-specific effects, are presented as HPD intervals across 163 individual effects categories for IAV-induced weight change at D4 p.i. (phenotype D4delta). HPDs are given for each parameter, including 95% (thin line) and 50% (thick line) intervals, and median (white break) and mean (black vertical line). Parameters are labeled according to the methods. Symmetric epistatic, asymmetric epistatic, and sex-specific parameters are indicated by &#x201C;v:&#x201D;, &#x201C;w:&#x201D;, and &#x201C;f:&#x201D;, respectively. The overall treatment effect (data not shown), &lt;em&gt;&#x3B8;&lt;/em&gt;, is &#x2212;8.85% (&#x2212;9.92, &#x2212;7.78%). (B and C) TReVarPs, a generalization of heritability for diallel effects classes, at D4 are shown for three fixed (overall) effects, five random effects classes, and five corresponding sex-specific random effects classes (posterior median and 95% HPD intervals) before (B) and (C) after accounting for diplotypes at the host influenza resistance locus, &lt;em&gt;Mx1&lt;/em&gt;. (D and E) TReVarPs before and after &lt;em&gt;Mx1&lt;/em&gt; for all four p.i. time points.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/427/F4.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/2/427/F4.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/427/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/427/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488492" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-56" class="first-child">Diallel effects on host weight IAV-infection response, before and after accounting for <em>Mx1</em> haplotypes. (A) Effect estimates for additive, maternal, inbred, and epistatic effects, including sex-specific effects, are presented as HPD intervals across 163 individual effects categories for IAV-induced weight change at D4 p.i. (phenotype D4delta). HPDs are given for each parameter, including 95% (thin line) and 50% (thick line) intervals, and median (white break) and mean (black vertical line). Parameters are labeled according to the methods. Symmetric epistatic, asymmetric epistatic, and sex-specific parameters are indicated by “v:”, “w:”, and “f:”, respectively. The overall treatment effect (data not shown), <em>θ</em>, is −8.85% (−9.92, −7.78%). (B and C) TReVarPs, a generalization of heritability for diallel effects classes, at D4 are shown for three fixed (overall) effects, five random effects classes, and five corresponding sex-specific random effects classes (posterior median and 95% HPD intervals) before (B) and (C) after accounting for diplotypes at the host influenza resistance locus, <em>Mx1</em>. (D and E) TReVarPs before and after <em>Mx1</em> for all four p.i. time points.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-22" class="subsection"><h4>Diallel effects on infection response reflect mostly additive genetics, consistent with differences in Mx1 haplotype:</h4><p id="p-57">Infection response in our diallel was strongly driven by additive effects. On D3 p.i., enhanced susceptibility to weight loss in infected animals was affected the most by contributions from strain AJ, −2.17% (−3.72, −0.61%), and enhanced resistance from contributions of NZO, 2.54% (0.72, 4.27%), and PWK, 1.70% (0.12, 3.23%), strains. On D4 p.i., enhanced susceptibility was greatest from AJ, −2.77% (−4.66, −0.86%), and WSB, −3.09% (−5.01, −1.18%), with enhanced resistance greatest from NZO, 4.07% (1.95, 6.12%), and PWK, 4.06% (1.97, 6.08%) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). In terms of its additive effect, CAST was more resistant than the <em>Mx1</em>-null strains (AJ, B6, 129, NOD, and WSB) but about half as resistant as the <em>Mx1</em>-functional strains (NZO and PWK), consistent with it conferring intermediate protection in the heterozygote state.</p><p id="p-58">To summarize these effects: for each dose of AJ or WSB genomes inherited from a parent, ∼2–3% of additional starting body weight is lost p.i., indicating enhanced susceptibility compared with the overall mean weight loss; for each NZO and PWK genome inherited, ∼4% more of starting body weight is retained p.i., compared with the mean treatment effect, indicating enhanced resistance.</p><p id="p-59">Diallel effects explained over half of the total variance of infection response at D4, with a treatment-response VarP for all effect groups collectively of 57% (TReVarP[all] = 0.571; 0.418, 0.721). The variance explained by additive effects only, which is related to the narrow-sense heritability, was estimated as 34.8% (TReVarP[a] = 0.348; 0.190, 0.491), and also detected were potential additional contributions of epistasis (TReVarP[v] = 0.069; −0.001, 0.212) and maternal effects (TReVarP[m] = 0.020; 0.000, 0.059) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, B and D</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>).</p></div><div id="sec-23" class="subsection"><h4>Evidence for additive, inbred, epistatic, and parent-of-origin effects mounts as disease progresses:</h4><p id="p-60">The relevance of diallel effects to infection response became more marked with time (Figures S7–S10 and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>). At D1 and D2 p.i., model inclusion probabilities gave strong support only to an overall infection response, with no evidence of this effect being modified by sex or parentage (Figures S7 and S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>). At D3 p.i., however, we found positive to strong evidence of additive (MIP[a] = 0.978), inbred (MIP[b] = 0.958), and asymmetric epistatic (MIP[w] = 0.820; <em>i.e.</em>, parent-of-origin epistatic) effects (Figure S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>). By D4 p.i., support for additive (MIP[a] = 0.998) and inbred (MIP[b] = 0.999) effects had become decisive (see <em>Statistical Models and Methods</em> for MIP interpretation) and there was strong support for both symmetric epistatic (MIP[v] = 0.960) and asymmetric epistatic (MIP[w] = 0.966) effects (Figure S10 and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>).</p></div></div><div id="sec-24" class="subsection"><h3>Modeling effects consistent with Mx1 haplotype</h3><p id="p-61">To help distinguish diallel effects that are consistent with the subspecies haplotype of the resistance factor <em>Mx1</em> (hereafter, <em>Mx1</em> effects), we incorporated the <em>Mx1</em> subtype explicitly into the model as a genotype covariate with three alleles, one for each subspecies branch: <em>domesticus</em> (AJ, B6, 129, NOD, and WSB), <em>castaneus</em> (CAST), and <em>musculus</em> (NZO and PWK).</p><div id="sec-25" class="subsection"><h4>Mx1 effects are increasingly evident with disease progression and explain ∼40% of the diallel effects at D4 p.i.:</h4><p id="p-62">In keeping with the increased support seen for diallel effects over time, evidence for a nonzero <em>Mx1</em> effect increases from positive evidence of exclusion on D1 (MIP = 0.035), to no evidence for inclusion or exclusion on D2 (MIP = 0.552), to decisive evidence for inclusion on D3 (MIP = 1.000) and D4 (MIP = 1.000) (Figures S11–S14 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>); a comparable level of support for inclusion in the model was seen only for effects of overall treatment and batch. After controlling for <em>Mx1</em>, the variance explained by diallel effects at D4 was substantially reduced, from 57 to 33.8% (TReVarP[all<span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-150.gif"/></noscript></span></span>] = 0.338; 0.174, 0.537) (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, C and E</a> and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>). This was consistent with <em>Mx1</em> accounting for ∼40% of the variance explained by the diallel, including most of the additive effects (mathematically the <em>Mx1</em> term models effects that compete with a subset of the additive and dominance diallel effects).</p></div><div id="sec-26" class="subsection"><h4>Evidence for distinct additive and nonadditive effects of Mx1 functional groups:</h4><p id="p-63">After controlling for other diallel effects, the predicted weight loss over the course of 4 d varies in a manner consistent with <em>Mx1</em> allele combination (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). We observed that, as expected, <em>domesticus</em> × <em>domesticus</em> crosses were predicted to have much more overall post-IAV-infection weight loss at D3 and D4 compared with all other crosses. Notably, the most protected group appeared to be the <em>domesticus</em> × <em>musculus</em> haplotype, at both D3 and D4 p.i., although the HPD intervals overlap with other <em>Mx1</em>-functional groups. The rank order of effects changes from D3 to D4 due to the dramatic slowing of weight loss in the <em>musculus</em> × <em>musculus</em> crosses from D3 to D4 compared with D2 to D3.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/427/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time course of subspecies-specific Mx1 haplotype effects on IAV-induced weight change in the diallel. (A) Predictive means of Mx1 diplotype effects across 4 d p.i., modeled simultaneously with other diallel effects and covariates. (B) HPD intervals of Mx1 diplotype effects on weight change on D4 p.i. Increased resistance is indicated by values further to the right. Dashed lines highlight the mode of interaction between Mx1 haplotypes: the green line shows the additive effect of crossing castaneus with domesticus, the blue line shows the dominant effect of crossing musculus with domesticus, and the orange line shows the negligible effect of castaneus crossed with musculus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1976058624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Time course of subspecies-specific &lt;em&gt;Mx1&lt;/em&gt; haplotype effects on IAV-induced weight change in the diallel. (A) Predictive means of &lt;em&gt;Mx1&lt;/em&gt; diplotype effects across 4 d p.i., modeled simultaneously with other diallel effects and covariates. (B) HPD intervals of &lt;em&gt;Mx1&lt;/em&gt; diplotype effects on weight change on D4 p.i. Increased resistance is indicated by values further to the right. Dashed lines highlight the mode of interaction between &lt;em&gt;Mx1&lt;/em&gt; haplotypes: the green line shows the additive effect of crossing &lt;em&gt;castaneus&lt;/em&gt; with &lt;em&gt;domesticus&lt;/em&gt;, the blue line shows the dominant effect of crossing &lt;em&gt;musculus&lt;/em&gt; with &lt;em&gt;domesticus&lt;/em&gt;, and the orange line shows the negligible effect of &lt;em&gt;castaneus&lt;/em&gt; crossed with &lt;em&gt;musculus&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/427/F5.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/2/427/F5.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/427/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/427/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488527" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-64" class="first-child">Time course of subspecies-specific <em>Mx1</em> haplotype effects on IAV-induced weight change in the diallel. (A) Predictive means of <em>Mx1</em> diplotype effects across 4 d p.i., modeled simultaneously with other diallel effects and covariates. (B) HPD intervals of <em>Mx1</em> diplotype effects on weight change on D4 p.i. Increased resistance is indicated by values further to the right. Dashed lines highlight the mode of interaction between <em>Mx1</em> haplotypes: the green line shows the additive effect of crossing <em>castaneus</em> with <em>domesticus</em>, the blue line shows the dominant effect of crossing <em>musculus</em> with <em>domesticus</em>, and the orange line shows the negligible effect of <em>castaneus</em> crossed with <em>musculus</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">Although we did not observe any strain- or pairwise-specific nonadditive effects in the diallel prior to inclusion of the <em>Mx1</em> random effect, we did observe a pattern of dominance in crosses between <em>musculus</em> and <em>domesticus</em>, even as there was a pattern of additivity in the crosses between <em>castaneus</em> and <em>domesticus</em> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Whereas it might be expected that host alleles from <em>Mx1</em>-null strains should act in a recessive manner, this appears not to be the case for this phenotype and time point in crosses of <em>castaneus</em> with <em>domesticus</em>, such that the functional <em>Mx1</em> allele from CAST appears to operate in an additive manner. This further supports the previous observation that the CAST <em>Mx1</em> alleles differ from the <em>musculus Mx1</em> alleles in their protective host response to IAV (<a id="xref-ref-19-9" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>).</p></div><div id="sec-27" class="subsection"><h4>Dominance and additivity of Mx1 alleles against the functional null: musculus is dominant, castaneus acts additively:</h4><p id="p-66">To better characterize how the <em>Mx1</em> effects on infection response exhibit aspects of genetic dominance <em>vs.</em> genetic additivity, we estimated for each functional <em>Mx1</em> allele a dominance index, after <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Kacser and Burns (1981)</a>. This measures the distance between the expected phenotype of a homozygous functional allele, in our case <em>musculus</em> or <em>castaneus</em>, and the heterozygote formed with a null allele, in our case <em>domesticus</em>. On this scale, 0 denotes the functional allele being dominant to the null, 1 denotes it being recessive, and 0.5 indicates pure additivity (see <em>x</em>-axis scale in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6, A and B</a>, and more details in <em>Statistical Models and Methods</em>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/427/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Posterior density of the dominance index on (A) D3 and (B) D4. (C) Posterior density of the dominance difference index, i.e., the difference between the dominance indices of castaneus and musculus, across all 4 d." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1976058624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Posterior density of the dominance index on (A) D3 and (B) D4. (C) Posterior density of the dominance difference index, &lt;em&gt;i.e.&lt;/em&gt;, the difference between the dominance indices of &lt;em&gt;castaneus&lt;/em&gt; and &lt;em&gt;musculus&lt;/em&gt;, across all 4 d.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/427/F6.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/2/427/F6.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/427/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/427/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488470" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-67" class="first-child">Posterior density of the dominance index on (A) D3 and (B) D4. (C) Posterior density of the dominance difference index, <em>i.e.</em>, the difference between the dominance indices of <em>castaneus</em> and <em>musculus</em>, across all 4 d.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">The dominance indices of the two functional <em>Mx1</em> alleles, <em>musculus</em> and <em>castaneus</em>, were sharply different (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6, A and B</a>, and Table S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>). We found that <em>musculus</em> against <em>domesticus</em> was −0.278 [= posterior mode of <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-151.gif"/></noscript></span></span> 80% HPD interval −2.547, 0.329] at D3 and 0.068 at D4 (−0.568, 0.380), a clear signal of <em>musculus</em> exerting classical dominance over the functional null. In contrast, the dominance index of <em>castaneus</em> against <em>domesticus</em> was 0.421 (−0.534, 0.907) and 0.491 (−0.028, 0.836) for D3 and D4, consistent with <em>castaneus</em> and the functional null being codominant (<em>i.e.</em>, having an additive relationship). The difference of the two dominance indices, whose posterior distribution is shown in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> for each time point, quantifies the distinction between <em>musculus</em> and <em>castaneus</em> more directly, putting the probability that <em>musculus</em> is more dominant than <em>castaneus</em> (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-152.gif"/></noscript></span></span>) at 83.6% for D3 and 86.6% for D4.</p></div></div><div id="sec-28" class="subsection"><h3>Mx1 effects show consistent pattern in related MPPs for pre-CC and CC-RIX</h3><p id="p-69">We examined effects associated with the <em>Mx1</em> locus in two related recombinant CC populations, the pre-CC of <a id="xref-ref-19-10" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> (2013)</a> and a set of CC-RIX lines first described here, and observed that the pattern of locus-specific strain haplotype effects was strikingly similar to that observed in our diallel (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). This suggests that the pattern of genome-wide additive effects in the diallel is largely driven by the effect of <em>Mx1</em> haplotypes in the founder strains. This similarity in pattern is consistent, even though the virus isolate and the peak weight loss time point differed in the CC-RIX population (CA04 human pandemic strain, D7 p.i.) compared with the diallel and pre-CC (PR8 mouse-adapted strain, D4 p.i.) (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>). In all three populations, NZO and PWK alleles provide the most resistance to IAV-induced weight loss, and CAST alleles are slightly less protective. In the pre-CC, effects of AJ, B6, 129, NOD, and WSB haplotypes are all approximately the same, and clearly separated from the additive effects of strains with functional <em>Mx1</em>. In the diallel and in the CC-RIX (at <em>Mx1</em>), however, AJ and WSB haplotypes are on average more susceptible than the B6 haplotype, and there is less separation between additive effects of CAST and those from <em>Mx1</em>-null strains. The proportion of variance in weight loss explained by <em>Mx1</em> was estimated as 0.5 (95% HPD interval: 0.43, 0.54) and 0.54 (0.42, 0.63) for pre-CC and CC-RIX mice, respectively (Figures S16 and S17 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>). Note that an in-depth analysis of dominance indices for the <em>Mx1</em> locus was not possible in these populations because of the relatively sparse coverage of heterozygote diplotype states in the pre-CC and homozygous functional diplotype states in the CC.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/427/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additive CC-strain haplotype effects on IAV-induced weight loss across three CC-related populations. (A) Additive effects from the CC founder diallel of mice infected with IAV (PR8) or mock (nflu = 393, nmock = 131) at D4 p.i. (from Figure 1). (B) Additive strain haplotype effects at the Mx1 locus for female pre-CC mice (n = 155) infected with IAV (PR8) at D4 p.i. (C) Additive strain haplotype effects at Mx1 for female CC-RIX mice (n = 1402) infected with IAV (CA04) at D7 p.i. Estimates are shown as HPD intervals as described in Figure 1, with blue lines connecting posterior means. Parameter scales are given as additional IAV-induced weight loss per dose of strain in % of (A) D0, and (B and C) normalized effect size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1976058624" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Additive CC-strain haplotype effects on IAV-induced weight loss across three CC-related populations. (A) Additive effects from the CC founder diallel of mice infected with IAV (PR8) or mock (&lt;em&gt;n&lt;/em&gt;&lt;sub&gt;flu&lt;/sub&gt; = 393, &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;mock&lt;/sub&gt; = 131) at D4 p.i. (from Figure 1). (B) Additive strain haplotype effects at the &lt;em&gt;Mx1&lt;/em&gt; locus for female pre-CC mice (&lt;em&gt;n&lt;/em&gt; = 155) infected with IAV (PR8) at D4 p.i. (C) Additive strain haplotype effects at &lt;em&gt;Mx1&lt;/em&gt; for female CC-RIX mice (&lt;em&gt;n&lt;/em&gt; = 1402) infected with IAV (CA04) at D7 p.i. Estimates are shown as HPD intervals as described in Figure 1, with blue lines connecting posterior means. Parameter scales are given as additional IAV-induced weight loss per dose of strain in % of (A) D0, and (B and C) normalized effect size.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/427/F7.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/2/427/F7.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/427/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/427/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488351" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-70" class="first-child">Additive CC-strain haplotype effects on IAV-induced weight loss across three CC-related populations. (A) Additive effects from the CC founder diallel of mice infected with IAV (PR8) or mock (<em>n</em><sub>flu</sub> = 393, <em>n</em><sub>mock</sub> = 131) at D4 p.i. (from <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). (B) Additive strain haplotype effects at the <em>Mx1</em> locus for female pre-CC mice (<em>n</em> = 155) infected with IAV (PR8) at D4 p.i. (C) Additive strain haplotype effects at <em>Mx1</em> for female CC-RIX mice (<em>n</em> = 1402) infected with IAV (CA04) at D7 p.i. Estimates are shown as HPD intervals as described in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, with blue lines connecting posterior means. Parameter scales are given as additional IAV-induced weight loss per dose of strain in % of (A) D0, and (B and C) normalized effect size.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-29"><h2 class="">Discussion</h2><p id="p-71">We describe a general approach for investigating heritable effects on host susceptibility to virus-induced disease, in our case pathogenesis induced by IAV, using a diallel cross of the eight CC founder strains. The results from this diallel are informative not only in more clearly defining the genetic architecture of the host influenza response, but also prospectively: they anticipate sources of heritable variation likely to be present in the CC, the DO, and other derived experimental populations, and therefore provide a ready basis for the rational design of future studies. As an illustration of this, we demonstrate concordant effects of viral resistance locus <em>Mx1</em> across the CC founder diallel, pre-CC, and a set of CC-RIX lines.</p><p id="p-72">With regard specifically to IAV pathogenesis, our study sought to better understand host genetic effects on this outcome in terms of their (1) time-dependence, (2) consistency across related populations, and (3) conditionality—for example, dependence on interactions between alleles at the same locus (dominance, at <em>Mx1</em>) or at different loci (epistasis). Regarding time-dependence (1), we found that whereas the effect of being female rather than male is evident from D1, the effects of genetics appear later, becoming evident only on D3 and then increasing through D4 p.i. Regarding consistency (2), we found that the effects of the <em>Mx1</em> alleles seen previously in the CC founders remain stable across inbred, F1, and recombinant populations. Regarding conditionality (3), we found something unexpected: evidence that the two <em>Mx1</em> functional classes, <em>castaneus</em> (CAST) and <em>musculus</em> (NZO and PWK), which were previously characterized as being functional alleles, in fact behave differently when present in the heterozygous state with susceptible <em>Mx1</em> alleles from <em>domesticus</em> (AJ, B6, 129, NOD, and WSB). Specifically, the protection conferred by the presence of a <em>musculus Mx1</em> allele is the same regardless of whether it is in the homozygote state or paired as a heterozygote with the null <em>domesticus</em> allele; the <em>musculus</em> allele is therefore dominant to <em>domesticus</em>. But for the CAST allele, when paired in the same way with <em>domesticus</em>, its protection is weakened to an extent consistent with CAST and <em>domesticus</em> being codominant, that is, having an additive relationship.</p><div id="sec-30" class="subsection"><h3>Level of resistance to IAV among different inbred mice is conditional on IAV subtype and strain</h3><p id="p-73">Differences in <em>Mx1</em> function have been identified between a variety of inbred mouse strains, including the CC founders (<a id="xref-ref-19-11" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Xiong <em>et al.</em> 2014</a>; <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Leist <em>et al.</em> 2016</a>). Our results were largely consistent with those studies.</p><p id="p-74">Notably, in their examination of the CC founders with H3N2 infection, <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Leist <em>et al.</em> (2016)</a> identified AJ and WSB strains as being most susceptible, and NZO and PWK as being most resistant, which agrees with our diallel additive effects. However, in contrast with our results showing partial protection against H1N1 IAV with CAST <em>Mx1</em>, which is consistent with our prior findings in the pre-CC (<a id="xref-ref-19-12" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>); they found CAST mice, grouping with AJ and WSB, to be highly susceptible. This difference could arise for at least two reasons. First, across the influenza field, even in identical RI panels (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Boon <em>et al.</em> 2009</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Nedelko <em>et al.</em> 2012</a>), host genetic effects appear to be IAV subtype specific. Second, the effectiveness of <em>Mx1</em>’s antiviral activities can vary depending on IAV subtypes (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Riegger <em>et al.</em> 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dittmann <em>et al.</em> 2008</a>; <a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Zimmermann <em>et al.</em> 2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mänz <em>et al.</em> 2013</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Verhelst <em>et al.</em> 2012</a>). Differentiating these two possibilities, however, is beyond the scope of this work.</p><p id="p-75">Although the molecular differences in CAST <em>Mx1</em> that produce a deficient response in comparison with <em>musculus Mx1</em> have not been defined, some work has been done in inbred mice to better understand CAST (strain)-specific antiviral responses. To interpret what they saw as a unique antiviral deficiency of CAST mice, transcriptomic experiments by <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Leist <em>et al.</em> (2016)</a> suggested enhanced susceptibility is due to leukocyte recruitment deficiency (relative to NZO and PWK) in the lung. In the CC founder study of <a id="xref-ref-96-3" class="xref-bibr" href="#ref-96">Xiong <em>et al.</em> (2014)</a>, several transcriptomic differences separated the CAST response to PR8 from the that of the other strains, including differential splicing of <em>Irak1</em> and lack of <em>Ifng</em> expression at D4 p.i., which was consistent with <em>Ifng</em> deficiency observed by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Earl <em>et al.</em> (2012)</a> leading to lethal monkeypox infection of CAST mice. Because these studies were completed in inbred CAST mice, the role of CAST <em>Mx1</em> is confounded with the genome-wide differences between CAST and the other CC founders.</p><p id="p-76">Thus, there are several challenges to understanding the unique IAV-resistance profile of CAST <em>Mx1</em> based on existing studies: (1) studies in inbred lines are unable to probe the overall or <em>Mx1</em>-specific dominance architecture due to a lack of heterozygosity, and (2) studies in nonrecombinant lines that identify a unique phenotype in CAST compared with other founders are unable to separate the effect of CAST <em>Mx1</em> from effects arising from the rest of the CAST genome. Our study in part circumvents these shortcomings by (1) additionally examining F1 hybrids; and (2) exploring the emerging phenotypes from an ongoing IAV-infection screen using CC-RIX, themselves F1s of RI strains.</p></div><div id="sec-31" class="subsection"><h3>Complex additive effects patterns mask strong signals of dominance</h3><p id="p-77">In our initial analysis, we found that most of the phenotypic variation explained in infection response is driven by additive genetics with no particular signal of dominance. However, when we explicitly modeled <em>Mx1</em> status, using a term that competes with a subset of the additive and dominance diallel effects, we found that the <em>Mx1</em> functional classes act in a manner consistent with a strong dominance pattern for <em>musculus Mx1</em> (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). It seems striking that such a pattern of dominance could be underlying an apparently heavily additive effect signal.</p><p id="p-78">Identifying dominance requires a good basis for comparing inbreds with hybrids. However, since the diallel is mainly composed of F1 hybrids with relatively few (8 <em>vs.</em> 54) inbreds, this basis for comparison is often weak. The BayesDiallel model handles this by considering the hybrid state as the baseline and treats the inbred state as the exception (a deviation) relevant to a minority of categories, as discussed further in <a id="xref-ref-44-12" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> (2012)</a>. Inferred dominance effects are therefore vague because the data that informs them is sparse, and low estimates of dominance variance comes from absence of information rather than from information about the absence of an effect. Nonetheless, greater precision was available when considering dominance of substrain-specific <em>Mx1</em> because dominance information was pooled across multiple strains and strain pairs.</p><p id="p-79">The fact that the proportion of estimated additive <em>vs.</em> nonadditive variance is influenced by model parameterization motivates careful consideration of both study design and analysis. As <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Huang and Mackay (2016)</a> have recently described, model parameterizations can have critical effects on the detection of nonadditivity, with the same data strongly supporting evidence for mostly additive or mostly nonadditive effects, depending on the model. Related issues have been described at the locus level by <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sabourin <em>et al.</em> (2015)</a>, who showed that when applying penalized regression to multi-SNP fine-mapping in GWAS, genotype parameterization interacts with how priors/penalties are assigned and can make biallelic dominance hard to identify in some cases. Yet even when dominance is not of interest <em>per se</em>, failure to accommodate it can disrupt estimation of additivity: in the pre-CC QTL mapping study of <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Phillippi <em>et al.</em> (2014)</a>, dominance signals arising from residual heterozygosity disrupted detection of an additive QTL for basal levels of CD23 (encoded by <em>Fcer</em>2); this was resolved by treating heterozygote diplotypes, whose occurrence was too sparse to be modeled, as inherently noisier via downweighting.</p></div><div id="sec-32" class="subsection"><h3>Antiviral genes are expected to be dominant, but CAST Mx1 exhibits additivity</h3><p id="p-80">The degree of genetic dominance of host resistance factors to viral infection in humans and mice has not been thoroughly explored. In general, in the context of biochemical and immunological studies one might expect, just as with <em>musculus Mx1</em> combined with <em>domesticus Mx1</em>, that genes encoding strong-acting antivirals when combined with a null mut would be mostly dominant. In quantitative genetics, however, it is more often expected that genetic contributions will be mostly additive. In this study, at the <em>Mx1</em> locus, we observe both.</p><p id="p-81">In genetic crosses of functional and null mice, major host determinants of pathogenesis are normally expected to be classified as either recessive or dominant: recessive when null results in loss of function for a host factor required for disease susceptibility, dominant when null results in loss of function for a host gene required for virus resistance. The recessive case is especially true of passive immunity gained by knockout of host genes critical to viral entry and life cycle, and has been demonstrated in a variety of studies on crop resistance (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fraser 1990</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kang <em>et al.</em> 2005</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Truniger and Aranda 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hashimoto <em>et al.</em> 2016</a>) and explored in studies of the effects of CCR5 deficiency (CCR5-Δ32 deletion) in resistance to HIV infection and pathogenesis in humans (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Liu <em>et al.</em> 1996</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Samson <em>et al.</em> 1996</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hütter <em>et al.</em> 2009</a>), however the degree of protection in the CCR5-Δ32 heterozygous individuals is not fully understood (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Marmor <em>et al.</em> 2001</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Trecarichi <em>et al.</em> 2006</a>). The dominant case could be considered for a viral sensor, where a single inherited functional copy still provides sufficient sensitivity for viral detection and control, resembling that of an individual inheriting two copies, one from each parent. This type of dominance is best explained by the model proposed by <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Kacser and Burns (1981)</a>, a metabolic–enzymatic model for the architecture of dominance at specific loci, and has been explored further in studies of viral resistance in plants, such as in <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fraser and Loon (1986)</a> and <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fraser (1992)</a>. The Kacser–Burns model also provides a mechanism that could in some cases give rise to additivity.</p><p id="p-82"><a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Kacser and Burns (1981)</a> predicted that, biochemically, for most enzymes, if there is a 50% reduction in enzyme activity in the heterozygote of a null × functional cross, then in most cases the resulting phenotype will resemble that in the homozygous functional individual and the null allele would likely be characterized as operating in a “recessive” manner. According to their model, the phenotype (or “flux”) resulting from a given enzymatic pathway with multiple enzymes joined by “kinetic linking” is a summation of the change in flux due to each specific enzyme activity (“selectivity coefficient”). This means that even a dramatic change in activity for any one enzyme in a physiological system results in barely discernible changes in the system overall, as long as some functional enzyme from the locus of interest is produced.</p><p id="p-83">However, the authors also describe two cases where systemic flux can be partially reduced in the heterozygote: (1) in pathways where there are exceptionally few enzymes involved in the system (this case is unlikely for an IFN-responsive antiviral pathway such as <em>Mx1</em>); and (2) in pathways where the selectivity coefficient (functional activity) of the enzyme is very low, a case termed heterozygote “indeterminacy,” which we henceforth equate to additivity. As further explored by <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Keightley (1996)</a>, dominance may be incomplete when less active allelic members of a series are involved in a cross with null muts, resulting in a more additive relationship; this seems most likely to explain our observation of CAST <em>Mx1</em> effects, and the lower antiviral activity of CAST <em>Mx1</em> observed in <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Nürnberger <em>et al.</em> (2016)</a>, discussed below, appears to support this.</p></div><div id="sec-33" class="subsection"><h3>Recent work exploring CAST Mx1 antiviral deficiency</h3><p id="p-84">Important insights into why CAST <em>Mx1</em> might be additive come from recent functional studies. <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Nürnberger <em>et al.</em> (2016)</a> engineered B6 mice expressing either the CAST-derived or A2G-derived MX1 proteins. A2G encodes an MX1 protein sequence similar to the NZO and PWK <em>musculus</em> class described in this study. CAST MX1 differs from A2G and <em>musculus</em>, with corresponding amino acid changes G83R and A222V in the G domain, which is important for enzymatic and antiviral function. <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Nürnberger <em>et al.</em> (2016)</a> clearly show that CAST provides intermediate protection from IAV in their case using H7N7 (SC35M) and H5N1 (R65) viruses, and suggest that sequence changes in the CAST <em>Mx1</em> allele result in reduced enzyme stability, metabolic instability, and possibly in altered dimerization of MX1 monomers and/or changes in MX1 GTPase antiviral activity. It is unknown whether the differences they observed would lead to changes in the dominance of CAST and A2G <em>Mx1</em>, although we might expect this to be the case given our mouse infection results. We have verified that the same variants, G83R and A222V, differentiate CAST coding sequence from NZO and PWK, as in <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Srivastava <em>et al.</em> (2009)</a> and using <a href="http://isvdb.unc.edu">http://isvdb.unc.edu</a> (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Oreper <em>et al.</em> 2017</a>), and that these are the only nonsynonymous variants on coding transcripts of <em>Mx1</em> that differentiate CAST from NZO and PWK. Although we see substantial protection from weight loss in CAST mice, we see a deficiency in the antiviral effects (as measured by RNA-seq viral reads in infected lungs) of CAST <em>Mx1</em> on D2 and D4 p.i. [data not shown, via RNA-seq reads from <a id="xref-ref-96-4" class="xref-bibr" href="#ref-96">Xiong <em>et al.</em> (2014)</a> and transcript analysis in <a id="xref-ref-19-13" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> (2013)</a>]. Our work motivates further functional studies of the MX1 protein using <em>Mx1</em> transgenic mice.</p></div><div id="sec-34" class="subsection"><h3>Mx1-independent effects and their follow-up: new studies should leverage CAST Mx1 additivity</h3><p id="p-85">A substantial proportion of heritable variance in the diallel was <em>Mx1</em> independent (VarP[all<span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-153.gif"/></noscript></span></span>] = 33.81, Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>). This was broadly driven by additive genetics and both symmetric and asymmetric epistasis (<em>i.e.</em>, differing by parent-of-origin) (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4, C and E</a>). Relatedly, in our analysis of the <em>Mx1</em> locus in the CC-RIX, we estimated <em>Mx1</em>-independent effects attributable to overall genome similarity to account for 21% of phenotypic variance. Both observations suggest the presence of additional QTL that could be drawn out given a suitable follow-up design.</p><p id="p-86">Consider the design of a second CC-RIX. Here our knowledge of differences in <em>Mx1</em> dominance becomes a valuable guide: prioritizing CC F1s with one copy of <em>musculus Mx1</em> would reduce power because it would cause <em>Mx1</em>-independent drivers to be masked; however, prioritizing CC F1s with one or fewer copies of <em>castaneus Mx1</em> would leave the <em>Mx1</em>-independent effects exposed and QTL underlying them more easily detected.</p><p id="p-87">The inclusion of mice with a single functional <em>Mx1</em> in a mapping population provides a basis for mapping loci that modify the effect of <em>Mx1</em>, as well as mapping <em>Mx1</em>-independent loci controlling disease. <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Shin <em>et al.</em> (2015)</a> showed that even the protectiveness of <em>Mx1</em> from the A2G inbred strain is conditional and depends on host genetic background. Thus, CC-RIX designs that incorporate heterozygous classes of <em>domesticus Mx1</em> crossed with either CAST <em>Mx1</em> or <em>musculus Mx1</em> can be of substantial benefit for mapping novel loci affecting infection outcomes, and at least 40% of the F1 crosses in our CC-RIX study incorporate lines which have one single copy (CAST or <em>musculus</em>) of <em>Mx1</em>.</p></div><div id="sec-35" class="subsection"><h3>Practical use of the diallel in quantitative genetics</h3><p id="p-88">Diallels have a long history in quantitative genetics (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Schmidt 1919</a>; and references in, <em>e.g.</em>, <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Christie and Shattuck 1992</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Verhoeven <em>et al.</em> 2006</a>; <a id="xref-ref-44-13" class="xref-bibr" href="#ref-44">Lenarcic <em>et al.</em> 2012</a>). They have most commonly been used as a way to assess the relative potency of different genomes with respect to a studied trait, yielding, for example, estimates of generalized combining ability for each strain and estimates of specific combining ability for each F1. More ambitiously, they have been used to obtain an overall picture of a trait’s genetic architecture. In many respects, this picture is clearly incomplete: even within the limited genetic space spanned by the founders, the diallel shows only the effects of swapping intact haploid genomes, with no ability to see the effects of recombination. But in other respects it is comprehensive: in considering every F1 combination, one can observe evidence for types of effects—dominance, epistasis, parent-of-origin, epistasis by parent-of-origin, and all sex-specific versions thereof—that would be hard or impossible to identify in other settings, <em>e.g.</em>, outbreeding populations derived from the same set of founders.</p><p id="p-89">A number of studies have sought to combine the features of a diallel with those of such derived outbred crosses to obtain a picture of genetic architecture that is in some way informed by both. These include studies that map QTL across multiple biparental (<em>e.g.</em>, F2) crosses derived from a diallel or diallel-like population (<em>e.g.</em>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Rebai and Goffinet 1993</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Xu 1998</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Liu and Zeng 2000</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Rebaï and Goffinet 2000</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Ogut <em>et al.</em> 2015</a>) and at least one theoretical study, that of <a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Verhoeven <em>et al.</em> (2006)</a>, examining the extent to which such information can be analyzed jointly and reconciled with data from the original diallel itself.</p><p id="p-90">The goals of our study were more prospective. We use the diallel to prioritize follow-up designs in target populations that segregate genetic material from the same set of founders: the diallel provides evidence of heritable features that would be expected to exist in the CC, and that could be examined in more detail in a suitably designed CC-based experiment. Of course, a comprehensive view of IAV-resistance architecture, even within the genetic space of the CC founder genomes, would be achievable only asymptotically through countless, diverse studies; but in this, the diallel can be seen as a compass, identifying promising initial directions.</p></div><div id="sec-36" class="subsection"><h3>Summary</h3><p id="p-91">Our study demonstrates the use of diallel crosses for identifying different types of heritable effects that can affect host responses to IAV infection. As such, we find reproducible effects of <em>Mx1</em> alleles across first order crosses and recombined populations (despite coordination between protocols being inexact), confirming our previous findings that the CAST <em>Mx1</em> allele exhibits an intermediate resistance phenotype against H1N1 strains of influenza virus (<a id="xref-ref-19-14" class="xref-bibr" href="#ref-19">Ferris <em>et al.</em> 2013</a>), and also identifying novel attributes of the CAST and <em>musculus Mx1</em> alleles with respect to additivity and dominance. Despite a body of literature on the effects of null mutations in <em>Mx1</em>, the importance of allelic variation at this antiviral gene is just beginning to be understood. A GWAS study published in 2011 found that <em>Mx1</em> allelic variation likely plays a role in viral disease manifestation in humans, specifically with regards to West Nile virus infection (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bigham <em>et al.</em> 2011</a>), highlighting a need for further study of the role of natural allelic variation in <em>Mx1</em> on virus infections in future research.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-92">We thank Peter Palese, at the Icahn School of Medicine at Mount Sinai, New York City, NY, for provision of mouse-adapted IAV H1N1 PR8 viral stocks (A/Puerto Rico/8/1934) for the diallel and pre-CC infections. We thank Yoshihiro Kawaoka, at the University of Wisconsin Department of Pathobiological Sciences, for providing the IAV H1N1 CA04 (A/California/04/2009) infectious clone plasmids that were used to generate viral stocks for the CC-RIX infections. We thank Timothy A. Bell for breeding the diallel and CC-RIX mice. We thank Alan B. Lenarcic for assistance with statistical analysis software and Sarah D. Turner for assistance with data visualization. We are grateful to the referees and editors who contributed to this article by providing the authors with detailed questions and comments. We acknowledge support from the National Institutes of Health T32 Virology Training Grant (5T32AI007419-23) to P.L.M.; National Institutes of Health U19 AI100625 to M.T.H., R.S.B., M.T.F., and F.P.-M.d.V.; National Institutes of Health U54 AI081680 to M.T.H., R.S.B., and F.P.-M.d.V.; and National Institutes of Health R01 GM104125 to W.V. and G.R.K. The organizations that funded this study did not have any role in the design, data collection/interpretation, nor the decision to submit the article for publication.</p><p id="p-93">Author contributions: P.L.M., M.T.F., G.R.K., M.T.H., and W.V. wrote the manuscript. P.L.M., M.T.F., D.W.T., F.P.-M.d.V., M.T.H., R.S.B., and W.V. designed experiments. P.L.M., M.T.F., D.R.M., A.C.W., A.W., C.R.M., K.E.N., K.S.P., A.S.C., and F.P.-M.d.V. performed experiments. G.D.S. bred the mice. P.L.M., G.R.K., and W.V. performed the statistical analysis. The authors declare no conflicts of interest.</p><p id="p-94"><em>Note added in proof:</em> See Turner <em>et al.</em> 2018 (pp. <a href="http://www.g3journal.org/content/8/2/411.full">411–426</a>) for a related work.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A: Defining a Potential Outcomes Model of Treatment Response with Quartets</h2><p id="p-97">In the potential outcomes framework of <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Neyman (1923)</a> and <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Rubin (1974)</a>, the causal effect of an applied treatment on a measured outcome in an individual <em>i</em> is defined as the difference between the outcome under treatment and the outcome that would have been observed if <em>i</em> were instead to have received the control. In our case, for some outcome measure <em>y</em>, we defined the causal effect as the infection response<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-154.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-155.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-156.gif"/></noscript></span></span> are “potential outcomes,” one of which is observed (the factual) and other of which is unobserved (the counterfactual). Since it is impossible to observe both simultaneously, the causal effect <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-157.gif"/></noscript></span></span> can never be measured directly (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Holland 1986</a>). It can however be estimated as<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-158.gif"/></noscript></span></span>with the accuracy of this estimate depending on how closely <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-159.gif"/></noscript></span></span> matches <em>i</em>. Our desire for lack of bias in this measure motivates our treatment assignment being randomly assigned within a group of matched individuals.</p><p id="p-98">In the treatment-response diallel, we are primarily interested not in infection response for a particular mouse but rather the expectation of this quantity for mice within a given diallel category, or more generally within a group of matched individuals <em>q</em>,</p><div class="section" id="sec-37"><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-160.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span><p id="p-99">where in our case <em>q</em> is defined as mice specific to a given diallel category and experimental batch. In practice it is natural to estimate this quantity as<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-161.gif"/></noscript></span><span class="disp-formula-label">(A2)</span></span>where <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-162.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-163.gif"/></noscript></span></span> are, respectively, the set of mice in group <em>q</em> assigned to flu and mock treatment groups. The variance of this estimate is<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-164.gif"/></noscript></span><span class="disp-formula-label">(A3)</span></span>and if it is considered likely, as in this study, that the infected phenotypes will be more variable than the mock, <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-165.gif"/></noscript></span></span> then it is most efficient experimentally to devote more individuals to the infected arm than the mock arm, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-166.gif"/></noscript></span></span></p><p id="p-100">In our experimental design, we have flu:mock in the ratio 3:1 for each group <em>q</em>. It is therefore natural to define the unit of observation <em>q</em> as a quartet, that is, Equation A2 with <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-167.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-168.gif"/></noscript></span></span> This means that each diallel category can be represented by multiple quartets, corresponding to multiple observations of <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-169.gif"/></noscript></span></span> denoted in the <em>Statistical Models and Methods</em> as <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-170.gif"/></noscript></span></span></p><p id="p-101">We now note several assumptions and connections. Equation A1 equates unit-level with marginal causal effects and thereby assumes no interference between units, specifically, that mice in the same quartet do not affect each other’s outcomes; this is approximately true based on the well-established evidence that mice do not transmit H1N1 influenza virus (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lowen <em>et al.</em> 2006</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Edenborough <em>et al.</em> 2012</a>), a finding we have also verified by weight-loss profiles and RNA-seq of CC founder strains cohoused with H1N1(PR8)-infected mice (<a id="xref-ref-96-5" class="xref-bibr" href="#ref-96">Xiong <em>et al.</em> 2014</a>). Last, we note that the definition of <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-171.gif"/></noscript></span></span> in Equation A2 is analogous to an inverse probability-weighted causal effect estimate.</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix B: Stochastic Regression Imputation of Missing Quartet Partners</h2><p id="p-102">The equation for the variance of <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-172.gif"/></noscript></span></span> namely Equation A3, implies that modeling the residual in Equation 5 as homoskedastic would require <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-173.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-174.gif"/></noscript></span></span> to be constant throughout; in other words, to ensure comparable precision of infection responses, all quartets should be complete. However, in the diallel experiment, some combinations of batch and diallel category had one or more flu mice missing. In these cases, quartets were defined to have missing values that would be filled in by imputation. The imputation scheme used here corresponds to stochastic regression imputation (<em>e.g.</em>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gelman and Hill 2006</a>), whereby the incomplete data set is repeatedly augmented to a completed data set using sampled variates from a prediction model, each completed data set is subject to the BayesDiallel analysis described in <em>Statistical Models and Methods</em>, and then results across the completed data sets are aggregated.</p><p id="p-103">Each imputation required two steps: since the target phenotype of a missing mouse, namely its p.i. weight loss, was considered potentially dependent on its D0 weight, we first imputed missing values for D0 and then imputed missing p.i. weight loss conditional on D0. In addition, at each day p.i., there was one batch/diallel category combination with one mock and four infecteds; for this case only, in each round of imputation, we created a completed quartet by randomly deleting one of the four infecteds.</p><p id="p-104">The two-step stochastic regression imputation was performed as follows. Define the observed diallel data for D0 and <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-175.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-176.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-177.gif"/></noscript></span></span> respectively, and let <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-178.gif"/></noscript></span></span> represent regression with BayesDiallel followed by stochastic regression imputation, that is, sampling from the posterior predictive. For each <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-179.gif"/></noscript></span></span> round of imputation, we first impute missing D0 values as</p><div class="section" id="sec-38"><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-180.gif"/></noscript></span><span class="disp-formula-label">(B1)</span></span><p id="p-105">where BayesDiallel is fitted as model 1 in <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>. These imputed values are then combined with observed D0 values to give the completed set,<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-181.gif"/></noscript></span><span class="disp-formula-label">(B2)</span></span>which are then used to impute p.i. weight loss at all time points (D1–D4) for the missing mice as<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-182.gif"/></noscript></span><span class="disp-formula-label">(B3)</span></span>leading in each case to the completed flu p.i. weight loss data,<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-183.gif"/></noscript></span><span class="disp-formula-label">(B4)</span></span>and subsequent calculation of quartet-based infection response values as<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-184.gif"/></noscript></span><span class="disp-formula-label">(B5)</span></span>Each of the 1000 infection response data sets is analyzed separately using BayesDiallel (model 3 in <a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). The MCMCs from each replicate are aggregated and thinned (sampled across even intervals) and the aggregate results are reported according to the procedure outlined in Algorithm 1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1/FileS7.zip">File S7</a>.</p><p id="p-106">The number of animals imputed were 33 at D1, 33 at D2, 15 at D3, and 16 at D4, in each case corresponding to a small proportion (2.8–3.1%) of the total data set. Phenotypes were not imputed for nonproductive diallel genotypes (which contain no mock or infected mice at all), <em>i.e.</em>, <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/427/embed/mml-math-185.gif"/></noscript></span></span>{NZO × CAST, NZO × PWK} (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chesler <em>et al.</em> 2008</a>).</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-95">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300438/-/DC1</a>.</p></li><li class="fn" id="fn-9"><p id="p-96"><em>Communicating editor: L. McIntyre</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-06"><span class="received-label">Received </span>January 6, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-15"><span class="accepted-label">Accepted </span>November 15, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Maurizio <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.1" data-doi="10.1016/j.micinf.2010.01.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alberts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viegas</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geffers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene expression changes in the host response between resistant and susceptible inbred mouse strains after influenza A infection.</span> <abbr class="cit-jnl-abbrev">Microbes Infect.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobes%2Band%2Binfection%2B%252F%2BInstitut%2BPasteur%26rft.stitle%253DMicrobes%2BInfect%26rft.aulast%253DAlberts%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D309%26rft.epage%253D318%26rft.atitle%253DGene%2Bexpression%2Bchanges%2Bin%2Bthe%2Bhost%2Bresponse%2Bbetween%2Bresistant%2Band%2Bsusceptible%2Binbred%2Bmouse%2Bstrains%2Bafter%2Binfluenza%2BA%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.micinf.2010.01.008%26rft_id%253Dinfo%253Apmid%252F20114087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.micinf.2010.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20114087&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mächler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting linear mixed-effects models using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.3" data-doi="10.1371/journal.pone.0024745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckingham</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husain</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emond</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bofferding</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Host genetic risk factors for West Nile virus infection and disease progression.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e24745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBigham%26rft.auinit1%253DA.%2BW.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De24745%26rft.epage%253De24745%26rft.atitle%253DHost%2Bgenetic%2Brisk%2Bfactors%2Bfor%2BWest%2BNile%2Bvirus%2Binfection%2Band%2Bdisease%2Bprogression.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0024745%26rft_id%253Dinfo%253Apmid%252F21935451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0024745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21935451&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.4" data-doi="10.4049/jimmunol.1103320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boivin</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pothlichet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skamene</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loredo-Osti</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mapping of clinical and expression quantitative trait loci in a sex-dependent effect of host susceptibility to mouse-adapted influenza H3N2/HK/1/68.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">3949</span>–<span class="cit-lpage">3960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1103320%26rft_id%253Dinfo%253Apmid%252F22427645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4OC84LzM5NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.5" data-doi="10.1128/JVI.00514-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boon</span>  <span class="cit-name-given-names">A. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeBeauchamp</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotb</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Host genetic variation affects resistance to infection with a highly pathogenic H5N1 influenza A virus in mice.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">10417</span>–<span class="cit-lpage">10426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00514-09%26rft_id%253Dinfo%253Apmid%252F19706712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8yMC8xMDQxNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bottomly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aicher</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Expression quantitative trait Loci for extreme host response to influenza A in pre-Collaborative Cross mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.7" data-doi="10.1007/s00335-008-9135-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branstetter</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galloway</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChesler%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D382%26rft.epage%253D389%26rft.atitle%253DThe%2BCollaborative%2BCross%2Bat%2BOak%2BRidge%2BNational%2BLaboratory%253A%2Bdeveloping%2Ba%2Bpowerful%2Bresource%2Bfor%2Bsystems%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9135-8%26rft_id%253Dinfo%253Apmid%252F18716833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9135-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18716833&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christie</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shattuck</span>  <span class="cit-name-given-names">V. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The diallel cross: design, analysis, and use for plant breeders.</span> <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D9%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.9" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Collaborative Cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.2.427.10" data-doi="10.1007/s00335-012-9414-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Diversity Outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D713%26rft.epage%253D718%26rft.atitle%253DThe%2Bdiversity%2Boutbred%2Bmouse%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9414-2%26rft_id%253Dinfo%253Apmid%252F22892839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9414-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892839&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.11" data-doi="10.1126/science.aaa1578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ciancanelli</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S. X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luthra</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Life-threatening influenza and impaired interferon amplification in human IRF7 deficiency.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">348</span>: <span class="cit-fpage">448</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaa1578%26rft_id%253Dinfo%253Apmid%252F25814066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDgvNjIzMy80NDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.12" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.13" data-doi="10.2307/3001412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comstock</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">The components of genetic variance in populations of biparental progenies and their use in estimating the average degree of dominance.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DComstock%26rft.auinit1%253DR.%2BE.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D254%26rft.epage%253D266%26rft.atitle%253DThe%2Bcomponents%2Bof%2Bgenetic%2Bvariance%2Bin%2Bpopulations%2Bof%2Bbiparental%2Bprogenies%2Band%2Btheir%2Buse%2Bin%2Bestimating%2Bthe%2Baverage%2Bdegree%2Bof%2Bdominance.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3001412%26rft_id%253Dinfo%253Apmid%252F18108899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3001412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18108899&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1948UG24700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.14" data-doi="10.1534/genetics.113.156901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crowley</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenarcic</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of adverse reactions to haloperidol in a mouse diallel: a drug-placebo experiment and Bayesian causal analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156901%26rft_id%253Dinfo%253Apmid%252F24240528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMzIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.15" data-doi="10.1128/JVI.01753-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dittmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stertz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimm</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Sastre</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Influenza A virus strains differ in sensitivity to the antiviral action of Mx-GTPase.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">3624</span>–<span class="cit-lpage">3631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01753-07%26rft_id%253Dinfo%253Apmid%252F18199636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzcvMzYyNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.16" data-doi="10.1128/JVI.00162-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Americo</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moss</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Lethal monkeypox virus infection of CAST/EiJ mice is associated with a deficient gamma interferon response.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">9105</span>–<span class="cit-lpage">9112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00162-12%26rft_id%253Dinfo%253Apmid%252F22696658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xNy85MTA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.17" data-doi="10.1128/JVI.00859-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edenborough</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbertson</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A mouse model for the study of contact-dependent transmission of influenza A virus and the factors that govern transmissibility.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">12544</span>–<span class="cit-lpage">12551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00859-12%26rft_id%253Dinfo%253Apmid%252F22951824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yMy8xMjU0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.18" data-doi="10.1038/nature10921"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everitt</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clare</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wash</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">IFITM3 restricts the morbidity and mortality associated with influenza.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEveritt%26rft.auinit1%253DA.%2BR.%26rft.volume%253D484%26rft.issue%253D7395%26rft.spage%253D519%26rft.epage%253D523%26rft.atitle%253DIFITM3%2Brestricts%2Bthe%2Bmorbidity%2Band%2Bmortality%2Bassociated%2Bwith%2Binfluenza.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10921%26rft_id%253Dinfo%253Apmid%252F22446628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10921&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22446628&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303200400052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.19" data-doi="10.1371/journal.ppat.1003196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottomly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aicher</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Modeling host genetic regulation of influenza pathogenesis in the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253De1003196%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003196%26rft_id%253Dinfo%253Apmid%252F23468633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23468633&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.20" data-doi="10.1146/annurev.py.28.090190.001143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">R. S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The genetics of resistance to plant viruses.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DRSS%26rft.auinit1%253DF.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D179%26rft.epage%253D200%26rft.atitle%253DThe%2BGenetics%2BOf%2BResistance%2BTo%2BPlant%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.py.28.090190.001143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.py.28.090190.001143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990DX23900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.21" data-doi="10.1007/BF00023922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">R. S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The genetics of plant-virus interactions: implications for plant breeding.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D63%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00023922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00023922&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">R. S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loon</span>  <span class="cit-name-given-names">L. C. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Genes for resistance to plant viruses.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D3%26rft.spage%253D257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.23" data-doi="10.2134/agronj1959.00021962005100090005x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonnquist</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Linkage and the degree of dominance of genes controlling quantitative characters in maize.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DGardner%26rft.auinit1%253DC.%2BO.%26rft.volume%253D51%26rft.issue%253D9%26rft.spage%253D524%26rft.epage%253D528%26rft.atitle%253DLinkage%2Band%2Bthe%2BDegree%2Bof%2BDominance%2Bof%2BGenes%2BControlling%2BQuantitative%2BCharacters%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1959.00021962005100090005x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1959.00021962005100090005x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.24" data-doi="10.1534/g3.114.013748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus mapping methods for Diversity Outbred mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1623%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.013748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.427.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Missing data imputation</span>, pp. <span class="cit-fpage">529</span>–<span class="cit-lpage">543</span> in <span class="cit-source">Data Analysis Using Regression and Multilevel/Hierarchical Models</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.26" data-doi="10.1038/sj.hdy.6800955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyn</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping reciprocal effects and interactions with plant density stress in <em>Zea mays</em> L.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DGonzalo%26rft.auinit1%253DM.%26rft.volume%253D99%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D30%26rft.atitle%253DMapping%2Breciprocal%2Beffects%2Band%2Binteractions%2Bwith%2Bplant%2Bdensity%2Bstress%2Bin%2BZea%2Bmays%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800955%26rft_id%253Dinfo%253Apmid%252F17473872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17473872&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247597500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.27" data-doi="10.1534/genetics.110.115055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harshman</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A hierarchical Bayesian model for a novel sparse partial diallel crossing design.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115055%26rft_id%253Dinfo%253Apmid%252F20157001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzEvMzYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.28" data-doi="10.1093/nar/gkt1159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grubb</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bult</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogue</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mouse phenome database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D825</span>–<span class="cit-lpage">D834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1159%26rft_id%253Dinfo%253Apmid%252F24243846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24243846&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800121&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Märtens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zackrisson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Powerful decomposition of complex traits in a diploid model.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">13311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D13311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neriya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaji</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Namba</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Recessive resistance to plant viruses: potential resistance genes beyond translation initiation factors.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D7%26rft.spage%253D1695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.31" data-doi="10.1016/j.clim.2006.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hidaka</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeshige</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizukami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A missense mutation of the Toll-like receptor 3 gene in a patient with influenza-associated encephalopathy.</span> <abbr class="cit-jnl-abbrev">Clin. Immunol.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bimmunology%2B%2528Orlando%252C%2BFla.%2529%26rft.stitle%253DClin%2BImmunol%26rft.aulast%253DHidaka%26rft.auinit1%253DF.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D188%26rft.epage%253D194%26rft.atitle%253DA%2Bmissense%2Bmutation%2Bof%2Bthe%2BToll-like%2Breceptor%2B3%2Bgene%2Bin%2Ba%2Bpatient%2Bwith%2Binfluenza-associated%2Bencephalopathy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.clim.2006.01.005%26rft_id%253Dinfo%253Apmid%252F16517210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.clim.2006.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16517210&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237176300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.32" data-doi="10.2307/2289064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Statistics and causal inference.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D81%26rft.spage%253D945%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2289064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2289064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986F049000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.33" data-doi="10.1073/pnas.0437907100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Single-locus heterotic effects and dominance by dominance interactions can adequately explain the genetic basis of heterosis in an elite rice hybrid.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">2574</span>–<span class="cit-lpage">2579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0437907100%26rft_id%253Dinfo%253Apmid%252F12604771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzUvMjU3NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic dissection of an elite rice hybrid revealed that heterozygotes are not always advantageous for performance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">1885</span>–<span class="cit-lpage">1895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHua%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D1885%26rft.epage%253D1895%26rft.atitle%253DGenetic%2BDissection%2Bof%2Ban%2BElite%2BRice%2BHybrid%2BRevealed%2BThat%2BHeterozygotes%2BAre%2BNot%2BAlways%2BAdvantageous%2Bfor%2BPerformance%26rft_id%253Dinfo%253Apmid%252F12524357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzE4ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetic architecture of quantitative traits cannot be inferred from variance component analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.36" data-doi="10.1056/NEJMoa0802905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hütter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mossner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganepola</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müßig</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Long-term control of HIV by CCR5 Delta32/Delta32 stem-cell transplantation.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">692</span>–<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DHutter%26rft.auinit1%253DG.%26rft.volume%253D360%26rft.issue%253D7%26rft.spage%253D692%26rft.epage%253D698%26rft.atitle%253DLong-Term%2BControl%2Bof%2BHIV%2Bby%2BCCR5%2BDelta32%252FDelta32%2BStem-Cell%2BTransplantation%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa0802905%26rft_id%253Dinfo%253Apmid%252F19213682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa0802905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19213682&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263260500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.37" data-doi="10.1038/nature08260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakoda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>In vitro</em> and <em>in vivo</em> characterization of new swine-origin H1N1 influenza viruses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">460</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DItoh%26rft.auinit1%253DY.%26rft.volume%253D460%26rft.issue%253D7258%26rft.spage%253D1021%26rft.epage%253D1025%26rft.atitle%253DIn%2Bvitro%2Band%2Bin%2Bvivo%2Bcharacterization%2Bof%2Bnew%2Bswine-origin%2BH1N1%2Binfluenza%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08260%26rft_id%253Dinfo%253Apmid%252F19672242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19672242&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269085500036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kacser</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">The molecular basis of dominance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKacser%26rft.auinit1%253DH.%26rft.volume%253D97%26rft.issue%253D3-4%26rft.spage%253D639%26rft.epage%253D666%26rft.atitle%253DTHE%2BMOLECULAR%2BBASIS%2BOF%2BDOMINANCE%26rft_id%253Dinfo%253Apmid%252F7297851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6Ijk3LzMtNC82MzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.39" data-doi="10.1146/annurev.phyto.43.011205.141140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">B.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeam</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahn</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetics of plant virus resistance.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DKang%26rft.auinit1%253DB.%2BC.%26rft.volume%253D43%26rft.spage%253D581%26rft.epage%253D621%26rft.atitle%253DGenetics%2Bof%2Bplant%2Bvirus%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.43.011205.141140%26rft_id%253Dinfo%253Apmid%252F16078896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.43.011205.141140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16078896&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232286700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.40" data-doi="10.2307/2291091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayes factors.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D90%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291091%26rft_id%253Dinfo%253Apmid%252F10783804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2291091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10783804&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RA10400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A metabolic basis for dominance and recessivity.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKeightley%26rft.auinit1%253DP.%2BD.%26rft.volume%253D143%26rft.issue%253D2%26rft.spage%253D621%26rft.epage%253D625%26rft.atitle%253DA%2BMetabolic%2BBasis%2Bfor%2BDominance%2Band%2BRecessivity%26rft_id%253Dinfo%253Apmid%252F8725213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiIxNDMvMi82MjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.42" data-doi="10.1104/pp.000729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kollipara</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saab</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wych</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lauer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singletary</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Expression profiling of reciprocal maize hybrids divergent for cold germination and desiccation tolerance.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">974</span>–<span class="cit-lpage">992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.000729%26rft_id%253Dinfo%253Apmid%252F12114554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMjkvMy85NzQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.43" data-doi="10.1186/s12864-016-2483-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leist</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilzner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Brand</span>  <span class="cit-name-given-names">J. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dengler</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geffers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Influenza H3N2 infection of the Collaborative Cross founder strains reveals highly divergent host responses and identifies a unique phenotype in CAST/EiJ mice.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2483-y%26rft_id%253Dinfo%253Apmid%252F26921172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2483-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26921172&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.44" data-doi="10.1534/genetics.111.132563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenarcic</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A general Bayesian approach to analyzing diallel crosses of inbred strains.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132563%26rft_id%253Dinfo%253Apmid%252F22345610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.45" data-doi="10.1073/pnas.0607038104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenschow</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frias-Staheli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giannakopoulos</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">From the cover: IFN-stimulated gene 15 functions as a critical antiviral molecule against influenza, herpes, and Sindbis viruses.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">1371</span>–<span class="cit-lpage">1376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0607038104%26rft_id%253Dinfo%253Apmid%252F17227866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzQvMTM3MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.46" data-doi="10.1016/S0092-8674(00)80110-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paxton</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceradini</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Homozygous defect in HIV-1 coreceptor accounts for resistance of some multiply-exposed individuals to HIV-1 infection.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLiu%26rft.auinit1%253DR.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D367%26rft.epage%253D377%26rft.atitle%253DHomozygous%2Bdefect%2Bin%2BHIV-1%2Bcoreceptor%2Baccounts%2Bfor%2Bresistance%2Bof%2Bsome%2Bmultiply-exposed%2Bindividuals%2Bto%2BHIV-1%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980110-5%26rft_id%253Dinfo%253Apmid%252F8756719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80110-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8756719&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VC30900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.47" data-doi="10.1007/s11032-011-9547-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in <em>Gossypium hirsutum</em> L.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D29%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-011-9547-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-011-9547-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.48" data-doi="10.1017/S0016672300004493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A general mixture model approach for mapping quantitative trait loci from diverse cross designs involving multiple inbred lines.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DLiu%26rft.auinit1%253DY.%26rft.volume%253D75%26rft.issue%253D3%26rft.spage%253D345%26rft.epage%253D355%26rft.atitle%253DA%2Bgeneral%2Bmixture%2Bmodel%2Bapproach%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bfrom%2Bdiverse%2Bcross%2Bdesigns%2Binvolving%2Bmultiple%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300004493%26rft_id%253Dinfo%253Apmid%252F10893870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300004493&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10893870&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087800600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.49" data-doi="10.1016/j.vaccine.2011.09.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzo</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodgson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pekosz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Antibody responses and cross protection against lethal influenza A viruses differ between the sexes in C57BL/6 mice.</span> <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">9246</span>–<span class="cit-lpage">9255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D29%26rft.spage%253D9246%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2011.09.110%26rft_id%253Dinfo%253Apmid%252F21983155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2011.09.110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21983155&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297525500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.50" data-doi="10.1073/pnas.0604157103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowen</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mubareka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tumpey</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Sastre</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palese</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The guinea pig as a transmission model for human influenza viruses.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">9988</span>–<span class="cit-lpage">9992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0604157103%26rft_id%253Dinfo%253Apmid%252F16785447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzI2Lzk5ODgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.51" data-doi="10.1371/journal.ppat.1003279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mänz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dornfeld</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Götz</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zell</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pandemic influenza A viruses escape from restriction by human MxA through adaptive mutations in the nucleoprotein.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253De1003279%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003279%26rft_id%253Dinfo%253Apmid%252F23555271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555271&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.52" data-doi="10.1097/00126334-200108150-00009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marmor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheppard</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bozeman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Celum</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Homozygous and heterozygous CCR5-[Delta] 32 genotypes are associated with resistance to HIV infection.</span> <abbr class="cit-jnl-abbrev">J. Acquir. Immune Defic. Syndr.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">472</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bacquired%2Bimmune%2Bdeficiency%2Bsyndromes%26rft.stitle%253DJ%2BAcquir%2BImmune%2BDefic%2BSyndr%26rft.aulast%253DMarmor%26rft.auinit1%253DM.%26rft.volume%253D27%26rft.issue%253D5%26rft.spage%253D472%26rft.epage%253D481%26rft.atitle%253DHomozygous%2Band%2Bheterozygous%2BCCR5-Delta32%2Bgenotypes%2Bare%2Bassociated%2Bwith%2Bresistance%2Bto%2BHIV%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00126334-200108150-00009%26rft_id%253Dinfo%253Apmid%252F11511825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00126334-200108150-00009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11511825&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170656400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.53" data-doi="10.1016/j.csda.2013.04.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martins</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindgren</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rue</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bayesian computing with INLA: new features.</span> <abbr class="cit-jnl-abbrev">Comput. Stat. Data Anal.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%2BData%2BAnal.%26rft.volume%253D67%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.csda.2013.04.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.csda.2013.04.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000323297000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.54" data-doi="10.1007/s00122-004-1890-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">Q. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene actions of QTLs affecting several agronomic traits resolved in a recombinant inbred rice population and two backcross populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMei%26rft.auinit1%253DH.%2BW.%26rft.volume%253D110%26rft.issue%253D4%26rft.spage%253D649%26rft.epage%253D659%26rft.atitle%253DGene%2Bactions%2Bof%2BQTLs%2Baffecting%2Bseveral%2Bagronomic%2Btraits%2Bresolved%2Bin%2Ba%2Brecombinant%2Binbred%2Brice%2Bpopulation%2Band%2Btwo%2Bbackcross%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1890-7%26rft_id%253Dinfo%253Apmid%252F15647921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1890-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15647921&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227714400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.55" data-doi="10.1128/mBio.00211-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morens</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taubenberger</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fauci</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The 2009 H1N1 pandemic influenza virus: what next?</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00211-10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00211-10%26rft_id%253Dinfo%253Apmid%252F20877580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiMS80L2UwMDIxMS0xMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.56" data-doi="10.1007/s00335-015-9581-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Informatics resources for the Collaborative Cross and related mouse populations.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D26%26rft.spage%253D521%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-015-9581-z%26rft_id%253Dinfo%253Apmid%252F26135136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-015-9581-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26135136&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The mouse universal genotyping array: from substrains to subspecies.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.58" data-doi="10.1186/1471-2164-13-411"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nedelko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kollmus</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klawonn</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spijker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Distinct gene loci control the host response to influenza H1N1 virus infection in a time-dependent manner.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNedelko%26rft.auinit1%253DT.%26rft.volume%253D13%26rft.spage%253D411%26rft.epage%253D411%26rft.atitle%253DDistinct%2Bgene%2Bloci%2Bcontrol%2Bthe%2Bhost%2Bresponse%2Bto%2Binfluenza%2BH1N1%2Bvirus%2Binfection%2Bin%2Ba%2Btime-dependent%2Bmanner.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-411%26rft_id%253Dinfo%253Apmid%252F22905720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-411&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22905720&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neyman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1923</span> <span class="cit-article-title">On the application of probability theory to agricultural experiments: principles.</span> <abbr class="cit-jnl-abbrev">Roczniki Nauk Rolniczch</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">51</span> (in Polish with German summary).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRoczniki%2BNauk%2BRolniczch%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.60" data-doi="10.1128/JVI.01213-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nürnberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerhardt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staeheli</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Influenza virus susceptibility of wild-derived CAST/EiJ mice results from two amino acid changes in the MX1 restriction factor.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">10682</span>–<span class="cit-lpage">10692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01213-16%26rft_id%253Dinfo%253Apmid%252F27654285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI5MC8yMy8xMDY4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.61" data-doi="10.1038/hdy.2014.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogut</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint-multiple family linkage analysis predicts within-family variation better than single-family analysis of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.123%26rft_id%253Dinfo%253Apmid%252F25585918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25585918&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okoro</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mbajiorgu</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Diallel cross in swine production: a review.</span> <abbr class="cit-jnl-abbrev">Indian J. Anim. Res.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIndian%2BJ.%2BAnim.%2BRes.%26rft.volume%253D51%26rft.spage%253D212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.63" data-doi="10.1534/g3.117.041491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oreper</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarantino</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Inbred strain variant database (ISVdb): a repository for probabilistically informed sequence differences among the Collaborative Cross strains and their founders.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041491%26rft_id%253Dinfo%253Apmid%252F28592645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xNjIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.64" data-doi="10.1038/gene.2013.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillippi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using the emerging Collaborative Cross to probe the immune system.</span> <abbr class="cit-jnl-abbrev">Genes Immun.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BImmun.%26rft.volume%253D15%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fgene.2013.59%26rft_id%253Dinfo%253Apmid%252F24195963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/gene.2013.59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24195963&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.65" data-doi="10.1111/j.1420-9101.2005.01048.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rantala</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roff</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Analysis of the importance of genotypic variation, metabolic rate, morphology, sex and development time on immune function in the cricket, <em>Gryllus firmus</em>.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">834</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DRantala%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D834%26rft.epage%253D843%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bimportance%2Bof%2Bgenotypic%2Bvariation%252C%2Bmetabolic%2Brate%252C%2Bmorphology%252C%2Bsex%2Band%2Bdevelopment%2Btime%2Bon%2Bimmune%2Bfunction%2Bin%2Bthe%2Bcricket%252C%2BGryllus%2Bfirmus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2005.01048.x%26rft_id%253Dinfo%253Apmid%252F16674580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2005.01048.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16674580&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236943000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.427.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Power of tests for QTL detection using replicated progenies derived from a diallel cross.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">1014</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D86%26rft.spage%253D1014%26rft_id%253Dinfo%253Apmid%252F24194011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24194011&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.68" data-doi="10.1017/S0016672399004358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebaï</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">More about quantitative trait locus mapping with diallel designs.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DReba%25C3%25AF%26rft.auinit1%253DA.%26rft.volume%253D75%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D247%26rft.atitle%253DMore%2Babout%2Bquantitative%2Btrait%2Blocus%2Bmapping%2Bwith%2Bdiallel%2Bdesigns.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004358%26rft_id%253Dinfo%253Apmid%252F10816981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10816981&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086569100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.69" data-doi="10.1128/JVI.02406-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riegger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dornfeld</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mänz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leyva-Grado</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The nucleoprotein of newly emerged H7N9 influenza A virus harbors a unique motif conferring resistance to antiviral human MxA.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">2241</span>–<span class="cit-lpage">2252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02406-14%26rft_id%253Dinfo%253Apmid%252F25505067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzQvMjI0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.70" data-doi="10.1371/journal.ppat.1002149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenzo</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Elevated 17β-estradiol protects females from influenza A virus pathogenesis by suppressing inflammatory responses.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DRobinson%26rft.auinit1%253DD.%2BP.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002149%26rft.epage%253De1002149%26rft.atitle%253DElevated%2B17%25CE%25B2-estradiol%2Bprotects%2Bfemales%2Bfrom%2Binfluenza%2BA%2Bvirus%2Bpathogenesis%2Bby%2Bsuppressing%2Binflammatory%2Bresponses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002149%26rft_id%253Dinfo%253Apmid%252F21829352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829352&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.71" data-doi="10.1037/h0037350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Estimating causal effects of treatments in randomized and nonrandomized studies.</span> <abbr class="cit-jnl-abbrev">J. Educ. Psychol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">688</span>–<span class="cit-lpage">701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEduc.%2BPsychol.%26rft.volume%253D66%26rft.spage%253D688%26rft_id%253Dinfo%253Adoi%252F10.1037%252Fh0037350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1037/h0037350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1974U496900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.72" data-doi="10.1002/gepi.21869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabourin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nobel</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fine-mapping additive and dominant SNP effects using group-LASSO and fractional resample model averaging.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D39%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21869%26rft_id%253Dinfo%253Apmid%252F25417853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25417853&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samet</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tompkins</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Focus: comparative medicine: influenza pathogenesis in genetically defined resistant and susceptible murine strains.</span> <abbr class="cit-jnl-abbrev">Yale J. Biol. Med.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYale%2BJ.%2BBiol.%2BMed.%26rft.volume%253D90%26rft.spage%253D471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.74" data-doi="10.1038/382722a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doranz</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rucker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liesnard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Resistance to HIV-1 infection in caucasian individuals bearing mutant alleles of the CCR-5 chemokine receptor gene.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">382</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSamson%26rft.auinit1%253DM.%26rft.volume%253D382%26rft.issue%253D6593%26rft.spage%253D722%26rft.epage%253D725%26rft.atitle%253DResistance%2Bto%2BHIV-1%2Binfection%2Bin%2Bcaucasian%2Bindividuals%2Bbearing%2Bmutant%2Balleles%2Bof%2Bthe%2BCCR-5%2Bchemokine%2Breceptor%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F382722a0%26rft_id%253Dinfo%253Apmid%252F8751444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/382722a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8751444&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VD33300052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">La valeur de l’individu à titre de génratéur appréciée suivant la méthode du croisement dialléle.</span> <abbr class="cit-jnl-abbrev">C. R. Trav. Lab. Carlsberg</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">33</span> (in French).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DC.%2BR.%2BTrav.%2BLab.%2BCarlsberg%26rft.volume%253D14%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Partial dominance, overdominance, epistasis and QTL by environment interactions contribute to heterosis in two upland cotton hybrids.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.77" data-doi="10.1128/JVI.01305-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">D.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatesuer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nedelko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schughart</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Protection from severe influenza virus infections in mice carrying the <em>Mx1</em> influenza virus resistance gene strongly depends on genetic background.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">9998</span>–<span class="cit-lpage">10009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01305-15%26rft_id%253Dinfo%253Apmid%252F26202236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xOS85OTk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.78" data-doi="10.1534/genetics.116.198838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarsani</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigmon</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomes of the mouse Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198838%26rft_id%253Dinfo%253Apmid%252F28592495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.79" data-doi="10.1371/journal.pone.0004857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Błazejewska</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hessmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geffers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Host genetic background strongly influences the response to influenza A virus infections.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e4857</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSrivastava%26rft.auinit1%253DB.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De4857%26rft.epage%253De4857%26rft.atitle%253DHost%2Bgenetic%2Bbackground%2Bstrongly%2Binfluences%2Bthe%2Bresponse%2Bto%2Binfluenza%2Ba%2Bvirus%2Binfections.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0004857%26rft_id%253Dinfo%253Apmid%252F19293935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0004857&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19293935&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.80" data-doi="10.1128/MCB.8.10.4518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staeheli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grob</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutcliffe</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Influenza virus-susceptible mice carry <em>Mx</em> genes with a large deletion or a nonsense mutation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">4518</span>–<span class="cit-lpage">4523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DStaeheli%26rft.auinit1%253DP%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D4518%26rft.epage%253D4523%26rft.atitle%253DInfluenza%2Bvirus-susceptible%2Bmice%2Bcarry%2BMx%2Bgenes%2Bwith%2Ba%2Blarge%2Bdeletion%2Bor%2Ba%2Bnonsense%2Bmutation.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.8.10.4518%26rft_id%253Dinfo%253Apmid%252F2903437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjgvMTAvNDUxOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strunk</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamieson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic and epigenetic susceptibility to early life infection.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Infect. Dis.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BInfect.%2BDis.%26rft.volume%253D26%26rft.spage%253D241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.82" data-doi="10.1126/science.1178294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swanson-Wagner</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeCook</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bancroft</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Paternal dominance of trans-eQTL influences gene expression patterns in maize hybrids.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1118</span>–<span class="cit-lpage">1120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSwanson-Wagner%26rft.auinit1%253DR.%2BA.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1118%26rft.epage%253D1120%26rft.atitle%253DPaternal%2BDominance%2Bof%2BTrans-eQTL%2BInfluences%2BGene%2BExpression%2BPatterns%2Bin%2BMaize%2BHybrids%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178294%26rft_id%253Dinfo%253Apmid%252F19965432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTE4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.83" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the mouse Diversity Outbred population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.84" data-doi="10.1007/s00335-001-4001-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic dissection of complex and quantitative traits: from fantasy to reality via a community effort.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DThreadgill%26rft.auinit1%253DD.%2BW.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D175%26rft.epage%253D178%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bcomplex%2Band%2Bquantitative%2Btraits%253A%2Bfrom%2Bfantasy%2Bto%2Breality%2Bvia%2Ba%2Bcommunity%2Beffort.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-001-4001-y%26rft_id%253Dinfo%253Apmid%252F11956758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-001-4001-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11956758&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175046300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.85" data-doi="10.1016/j.coviro.2015.04.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">To</span>  <span class="cit-name-given-names">K. K.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">J. F.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">K.-Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Host genes and influenza pathogenesis in humans: an emerging paradigm.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Virol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BVirol.%26rft.volume%253D14%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2015.04.010%26rft_id%253Dinfo%253Apmid%252F26079652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2015.04.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26079652&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.86" data-doi="10.1186/1742-6405-3-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trecarichi</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tumbarello</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Gaetano Donati</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamburrini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cauda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Partial protective effect of CCR5-Delta 32 heterozygosity in a cohort of heterosexual Italian HIV-1 exposed uninfected individuals.</span> <abbr class="cit-jnl-abbrev">AIDS Res. Ther.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAIDS%2BRes%2BTher%26rft.aulast%253DTrecarichi%26rft.auinit1%253DE.%2BM.%26rft.volume%253D3%26rft.spage%253D22%26rft.epage%253D22%26rft.atitle%253DPartial%2Bprotective%2Beffect%2Bof%2BCCR5-Delta%2B32%2Bheterozygosity%2Bin%2Ba%2Bcohort%2Bof%2Bheterosexual%2BItalian%2BHIV-1%2Bexposed%2Buninfected%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-6405-3-22%26rft_id%253Dinfo%253Apmid%252F16999868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-6405-3-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16999868&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.427.87" data-doi="10.1016/S0065-3527(09)07504-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truniger</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranda</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recessive resistance to plant viruses</span>, pp. <span class="cit-fpage">119</span>–<span class="cit-lpage">231</span> in <span class="cit-source">Advances in Virus Research</span>, <span class="cit-vol">Vol. 75</span>, <span class="cit-edition">Ed. 1</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Loebenstein</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">J. P.</span></span>. <span class="cit-publ-name">Academic Press/Elsevier</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253DTruniger%26rft.auinit1%253DV.%26rft.volume%253D75%26rft.spage%253D119%26rft.epage%253D159%26rft.atitle%253DRecessive%2Bresistance%2Bto%2Bplant%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252809%252907504-6%26rft_id%253Dinfo%253Apmid%252F20109665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-3527(09)07504-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20109665&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272540800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.88" data-doi="10.1007/s00335-004-2466-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsaih</span>  <span class="cit-name-given-names">S.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait mapping in a diallel cross of recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DTsaih%26rft.auinit1%253DS.%2BW.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D344%26rft.epage%253D355%26rft.atitle%253DQuantitative%2Btrait%2Bmapping%2Bin%2Ba%2Bdiallel%2Bcross%2Bof%2Brecombinant%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-004-2466-1%26rft_id%253Dinfo%253Apmid%252F16104382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-004-2466-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16104382&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229640800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurizio</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dissecting the genetic architecture of shoot growth in carrot (<em>Daucus carota</em> L.) using a diallel mating design.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D8%26rft.spage%253D411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.90" data-doi="10.1128/JVI.01682-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verhelst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parthoens</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schepens</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiers</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saelens</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Interferon-inducible protein Mx1 inhibits influenza virus by interfering with functional viral ribonucleoprotein complex assembly.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">13445</span>–<span class="cit-lpage">13455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01682-12%26rft_id%253Dinfo%253Apmid%252F23015724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yNC8xMzQ0NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.91" data-doi="10.1038/sj.hdy.6800763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verhoeven</span>  <span class="cit-name-given-names">K. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Using mating designs to uncover QTL and the genetic architecture of complex traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DVerhoeven%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D96%26rft.issue%253D2%26rft.spage%253D139%26rft.epage%253D149%26rft.atitle%253DUsing%2Bmating%2Bdesigns%2Bto%2Buncover%2BQTL%2Band%2Bthe%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800763%26rft_id%253Dinfo%253Apmid%252F16304603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304603&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234843500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.92" data-doi="10.1111/j.1558-5646.2010.01197.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pienaar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telonis-Scott</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sylvestre</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Expression of defense genes in <em>Drosophila</em> evolves under a different selective regime from expression of other genes.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DWayne%26rft.auinit1%253DM.%2BL.%26rft.volume%253D65%26rft.issue%253D4%26rft.spage%253D1068%26rft.epage%253D1078%26rft.atitle%253DExpression%2Bof%2Bdefense%2Bgenes%2Bin%2BDrosophila%2Bevolves%2Bunder%2Ba%2Bdifferent%2Bselective%2Bregime%2Bfrom%2Bexpression%2Bof%2Bother%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.01197.x%26rft_id%253Dinfo%253Apmid%252F21108635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.01197.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21108635&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289158300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Resources for systems genetics.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1488</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1488%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.94" data-doi="10.1086/280521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1934</span> <span class="cit-article-title">Physiological and evolutionary theories of dominance.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D68%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1086%252F280521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/280521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200907000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Dominance is the major genetic basis of heterosis in rice as revealed by QTL analysis using molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXiao%26rft.auinit1%253DJ.%26rft.volume%253D140%26rft.issue%253D2%26rft.spage%253D745%26rft.epage%253D754%26rft.atitle%253DDominance%2Bis%2Bthe%2Bmajor%2Bgenetic%2Bbasis%2Bof%2Bheterosis%2Bin%2Brice%2Bas%2Brevealed%2Bby%2BQTL%2Banalysis%2Busing%2Bmolecular%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7498751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQwLzIvNzQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic profiling of Collaborative Cross founder mice infected with respiratory viruses reveals novel transcripts and infection-related strain-specific gene and isoform expression.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1429</span>–<span class="cit-lpage">1444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Mapping quantitative trait loci using multiple families of line crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D148%26rft.issue%253D1%26rft.spage%253D517%26rft.epage%253D524%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2BUsing%2BMultiple%2BFamilies%2Bof%2BLine%2BCrosses%26rft_id%253Dinfo%253Apmid%252F9475760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzEvNTE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.98" data-doi="10.1038/nmeth.1359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchins</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A customized and versatile high-density genotyping array for the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D663%26rft.epage%253D666%26rft.atitle%253DA%2Bcustomized%2Band%2Bversatile%2Bhigh-density%2Bgenotyping%2Barray%2Bfor%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1359%26rft_id%253Dinfo%253Apmid%252F19668205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668205&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.99" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.100" data-doi="10.1534/genetics.114.166249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bayesian modeling of haplotype effects in multiparent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166249%26rft_id%253Dinfo%253Apmid%252F25236455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.101" data-doi="10.1073/pnas.1214141109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic composition of yield heterosis in an elite rice hybrid.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">15847</span>–<span class="cit-lpage">15852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1214141109%26rft_id%253Dinfo%253Apmid%252F23019369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM5LzE1ODQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mixed model approaches for diallel analysis based on a bio-model.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DZhu%26rft.auinit1%253DJ.%26rft.volume%253D68%26rft.issue%253D3%26rft.spage%253D233%26rft.epage%253D240%26rft.atitle%253DMixed%2Bmodel%2Bapproaches%2Bfor%2Bdiallel%2Banalysis%2Bbased%2Bon%2Ba%2Bbio-model.%26rft_id%253Dinfo%253Apmid%252F9062080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9062080&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.427.103" data-doi="10.1128/JVI.00712-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwemmle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kochs</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The viral nucleoprotein determines Mx sensitivity of influenza A viruses.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">8133</span>–<span class="cit-lpage">8140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00712-11%26rft_id%253Dinfo%253Apmid%252F21680506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NS8xNi84MTMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/427.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/411" title="Dissecting the Genetic Architecture of Shoot Growth in Carrot (Daucus carota L.) Using a Diallel Mating Design" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/447" title="Direct Testing for Allele-Specific Expression Differences Between Conditions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DYeEMO1Scf48MQoopeMOyQYJLGRr7WjgT90OLIsYYdQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/427" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/427" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/427" class="active" data-icon-position="" data-hide-link-title="0">Bayesian Diallel Analysis Reveals Mx1-Dependent and Mx1-Independent Effects on Response to Influenza A Virus in Mice</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488286" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-uznXWCXGdoMYArQaLAaRjMqEHXJjEmxidqtkfc9t0jM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/427.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/427" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Avq2MGq0DFj6nMvXNONAhUuPsfOdgchxzL4aaLj_VGA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/427" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/427.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488286" id="node488286--2" data-pisa="ggg;8/2/427" data-pisa-master="ggg;g3.117.300438" data-apath="/ggg/8/2/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Diallel Analysis Reveals <em>Mx1</em>-Dependent and <em>Mx1</em>-Independent Effects on Response to Influenza A Virus in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5859-6260" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul L. Maurizio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin T. Ferris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory R. Keele</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0781-7254" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Darla R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2590-4973" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ginger D. Shaw</span>, <span class="highwire-citation-author" data-delta="5">Alan C. Whitmore</span>, <span class="highwire-citation-author" data-delta="6">Ande West</span>, <span class="highwire-citation-author" data-delta="7">Clayton R. Morrison</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3977-0083" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kelsey E. Noll</span>, <span class="highwire-citation-author" data-delta="9">Kenneth S. Plante</span>, <span class="highwire-citation-author" data-delta="10">Adam S. Cockrell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-3538-1635" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David W. Threadgill</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0001-6827-8701" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ralph S. Baric</span>, <span class="highwire-citation-author" data-delta="14">Mark T. Heise</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-445;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488286--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul L. Maurizio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BL.%2BMaurizio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maurizio%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BL.%2BMaurizio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5859-6260" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul L. Maurizio</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin T. Ferris</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gregory R. Keele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BR.%2BKeele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keele%20GR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BR.%2BKeele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory R. Keele</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0781-7254" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Darla R. Miller</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ginger D. Shaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ginger%2BD.%2BShaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shaw%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGinger%2BD.%2BShaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2590-4973" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ginger D. Shaw</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alan C. Whitmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BC.%2BWhitmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmore%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BC.%2BWhitmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ande West </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ande%2BWest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=West%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnde%2BWest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Clayton R. Morrison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clayton%2BR.%2BMorrison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrison%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClayton%2BR.%2BMorrison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kelsey E. Noll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology and Immunology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsey%2BE.%2BNoll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noll%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelsey%2BE.%2BNoll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3977-0083" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelsey E. Noll</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kenneth S. Plante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BS.%2BPlante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plante%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BS.%2BPlante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Adam S. Cockrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BS.%2BCockrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cockrell%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BS.%2BCockrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">David W. Threadgill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Genome Sciences and Society, Texas A&M University, College Station, Texas 77433-2470</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BThreadgill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Threadgill%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BW.%2BThreadgill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3538-1635" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David W. Threadgill</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Ralph S. Baric </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ralph%2BS.%2BBaric%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baric%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARalph%2BS.%2BBaric%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6827-8701" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ralph S. Baric</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Mark T. Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BT.%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BT.%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/427" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/427&amp;title=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/427&amp;title=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/427&amp;title=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/427&amp;text=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/427&amp;title=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/427&amp;t=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/427&amp;title=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/427&amp;title=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488286/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488286" id="node488286--3" data-pisa="ggg;8/2/427" data-pisa-master="ggg;g3.117.300438" data-apath="/ggg/8/2/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bayesian Diallel Analysis Reveals <em>Mx1</em>-Dependent and <em>Mx1</em>-Independent Effects on Response to Influenza A Virus in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5859-6260" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul L. Maurizio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin T. Ferris</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory R. Keele</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0781-7254" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Darla R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-2590-4973" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ginger D. Shaw</span>, <span class="highwire-citation-author" data-delta="5">Alan C. Whitmore</span>, <span class="highwire-citation-author" data-delta="6">Ande West</span>, <span class="highwire-citation-author" data-delta="7">Clayton R. Morrison</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3977-0083" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kelsey E. Noll</span>, <span class="highwire-citation-author" data-delta="9">Kenneth S. Plante</span>, <span class="highwire-citation-author" data-delta="10">Adam S. Cockrell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-3538-1635" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David W. Threadgill</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0001-6827-8701" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ralph S. Baric</span>, <span class="highwire-citation-author" data-delta="14">Mark T. Heise</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-445;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488286--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul L. Maurizio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BL.%2BMaurizio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maurizio%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BL.%2BMaurizio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5859-6260" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul L. Maurizio</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1241-6268" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin T. Ferris</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gregory R. Keele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BR.%2BKeele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keele%20GR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BR.%2BKeele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1843-7900" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory R. Keele</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Darla R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darla%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADarla%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0781-7254" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Darla R. Miller</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ginger D. Shaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ginger%2BD.%2BShaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shaw%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGinger%2BD.%2BShaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2590-4973" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ginger D. Shaw</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alan C. Whitmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BC.%2BWhitmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmore%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BC.%2BWhitmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ande West </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ande%2BWest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=West%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnde%2BWest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Clayton R. Morrison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clayton%2BR.%2BMorrison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrison%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClayton%2BR.%2BMorrison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kelsey E. Noll </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology and Immunology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsey%2BE.%2BNoll%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noll%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelsey%2BE.%2BNoll%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3977-0083" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelsey E. Noll</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kenneth S. Plante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BS.%2BPlante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plante%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BS.%2BPlante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Adam S. Cockrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BS.%2BCockrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cockrell%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BS.%2BCockrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">David W. Threadgill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Genome Sciences and Society, Texas A&M University, College Station, Texas 77433-2470</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BThreadgill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Threadgill%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BW.%2BThreadgill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3538-1635" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David W. Threadgill</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Ralph S. Baric </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ralph%2BS.%2BBaric%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baric%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARalph%2BS.%2BBaric%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6827-8701" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ralph S. Baric</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Mark T. Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BT.%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BT.%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina at Chapel Hill, North Carolina 27599</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=427&amp;rft.epage=445&amp;rft.atitle=Bayesian%20Diallel%20Analysis%20Reveals%20Mx1-Dependent%20and%20Mx1-Independent%20Effects%20on%20Response%20to%20Influenza%20A%20Virus%20in%20Mice&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Maurizio%2C+Paul+L.&amp;rft.au=Ferris%2C+Martin+T.&amp;rft.au=Keele%2C+Gregory+R.&amp;rft.au=Miller%2C+Darla+R.&amp;rft.au=Shaw%2C+Ginger+D.&amp;rft.au=Whitmore%2C+Alan+C.&amp;rft.au=West%2C+Ande&amp;rft.au=Morrison%2C+Clayton+R.&amp;rft.au=Noll%2C+Kelsey+E.&amp;rft.au=Plante%2C+Kenneth+S.&amp;rft.au=Cockrell%2C+Adam+S.&amp;rft.au=Threadgill%2C+David+W.&amp;rft.au=Pardo-Manuel+de+Villena%2C+Fernando&amp;rft.au=Baric%2C+Ralph+S.&amp;rft.au=Heise%2C+Mark+T.&amp;rft.au=Valdar%2C+William"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488286/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488286/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488286/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488286/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488286/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488286/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488286/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488286/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488286/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488286/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488286/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488286/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/427" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Experimental Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Statistical Models and Methods</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-29" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A: Defining a Potential Outcomes Model of Treatment Response with Quartets</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B: Stochastic Regression Imputation of Missing Quartet Partners</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/427.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/427.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/427.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/427.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
