<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/707" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/707.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/707.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/707.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/2/707/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/707.atom" />
<meta name="HW.pisa" content="ggg;8/2/707" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower" />
<meta name="DC.Identifier" content="10.1534/g3.117.300199" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Thorwarth et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic resources are an important source of genetic variation for plant breeding. Genome-wide association studies (GWAS) and genomic prediction greatly facilitate the analysis and utilization of useful genetic diversity for improving complex phenotypic traits in crop plants. We explored the potential of GWAS and genomic prediction for improving curd-related traits in cauliflower ( Brassica oleracea var. botrytis ) by combining 174 randomly selected cauliflower gene bank accessions from two different gene banks. The collection was genotyped with genotyping-by-sequencing (GBS) and phenotyped for six curd-related traits at two locations and three growing seasons. A GWAS analysis based on 120,693 single-nucleotide polymorphisms identified a total of 24 significant associations for curd-related traits. The potential for genomic prediction was assessed with a genomic best linear unbiased prediction model and BayesB. Prediction abilities ranged from 0.10 to 0.66 for different traits and did not differ between prediction methods. Imputation of missing genotypes only slightly improved prediction ability. Our results demonstrate that GWAS and genomic prediction in combination with GBS and phenotyping of highly heritable traits can be used to identify useful quantitative trait loci and genotypes among genetically diverse gene bank material for subsequent utilization as genetic resources in cauliflower breeding." />
<meta name="DC.Contributor" content="Patrick Thorwarth" />
<meta name="DC.Contributor" content="Eltohamy A. A. Yousef" />
<meta name="DC.Contributor" content="Karl J. Schmid" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic resources are an important source of genetic variation for plant breeding. Genome-wide association studies (GWAS) and genomic prediction greatly facilitate the analysis and utilization of useful genetic diversity for improving complex phenotypic traits in crop plants. We explored the potential of GWAS and genomic prediction for improving curd-related traits in cauliflower (&lt;i&gt;Brassica oleracea&lt;/i&gt; var. &lt;i&gt;botrytis&lt;/i&gt;) by combining 174 randomly selected cauliflower gene bank accessions from two different gene banks. The collection was genotyped with genotyping-by-sequencing (GBS) and phenotyped for six curd-related traits at two locations and three growing seasons. A GWAS analysis based on 120,693 single-nucleotide polymorphisms identified a total of 24 significant associations for curd-related traits. The potential for genomic prediction was assessed with a genomic best linear unbiased prediction model and BayesB. Prediction abilities ranged from 0.10 to 0.66 for different traits and did not differ between prediction methods. Imputation of missing genotypes only slightly improved prediction ability. Our results demonstrate that GWAS and genomic prediction in combination with GBS and phenotyping of highly heritable traits can be used to identify useful quantitative trait loci and genotypes among genetically diverse gene bank material for subsequent utilization as genetic resources in cauliflower breeding.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/707" />
<meta name="citation_id" content="8/2/707" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/707" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/707.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/707.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/707.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300199" />
<meta name="citation_pmid" content="29255118" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="707" />
<meta name="citation_lastpage" content="718" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Patrick Thorwarth" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4456-2358" />
<meta name="citation_author" content="Eltohamy A. A. Yousef" />
<meta name="citation_author_institution" content="Department of Horticulture, Faculty of Agriculture, University of Suez Canal, Ismailia 41522, Egypt" />
<meta name="citation_author" content="Karl J. Schmid" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author_email" content="Karl.Schmid@uni-hohenheim.de" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Atwell;citation_author=Y. S. Huang;citation_author=B. J. Vilhjálmsson;citation_author=G. Willems;citation_author=M. Horton;citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.;citation_pages=627-631;citation_volume=465;citation_year=2010;citation_issue=7298;citation_pmid=20336072;citation_doi=10.1038/nature08800" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Potential of association mapping and genomic selection to explore PI 88788 derived soybean cyst nematode resistance.;citation_author=Y. Bao;citation_author=T. Vuong;citation_author=C. Meinhardt;citation_author=P. Tiffin;citation_author=R. Denny;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_title=R package version;citation_author=D. Bates;citation_author=M. Maechler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=lme4: linear mixed-effects models using Eigen and S4.;citation_pages=1-23;citation_volume=1;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. W. Bevan;citation_author=C. Uauy;citation_author=B. B. Wulff;citation_author=J. Zhou;citation_author=K. Krasileva;citation_title=Genomic innovation for crop improvement.;citation_pages=346-354;citation_volume=543;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=Y. Bian;citation_author=J. Holland;citation_title=Enhancing genomic prediction with genome-wide association studies in multiparental maize populations.;citation_pages=585-593;citation_volume=118;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Breseghello;citation_author=M. E. Sorrells;citation_title=Association mapping of kernel size and milling quality in wheat (Triticum aestivum L.) cultivars.;citation_volume=172;citation_year=2006;citation_pmid=16079235;citation_doi=10.1534/genetics.105.044586" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Haplotype phasing: existing methods and new developments.;citation_pages=703-714;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21921926;citation_doi=10.1038/nrg3054" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Cai;citation_author=Y. Xiao;citation_author=W. Yang;citation_author=W. Ye;citation_author=B. Wang;citation_title=Association mapping of six yield-related traits in rapeseed (Brassica napus L.).;citation_pages=85-96;citation_volume=127;citation_year=2014;citation_pmid=24121524;citation_doi=10.1007/s00122-013-2203-9" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=M. P. Conomos;citation_author=A. P. Reiner;citation_author=B. S. Weir;citation_author=T. A. Thornton;citation_title=Model-free estimation of recent genetic relatedness.;citation_pages=127-148;citation_volume=98;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. l. Campos;citation_author=P. Perez;citation_author=D. Gianola;citation_author=J. Burgueno;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=G3(Bethesda);citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Perez;citation_author=J. M. Hickey;citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing.;citation_pages=1903-1926;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. De Roos;citation_author=B. J. Hayes;citation_author=R. J. Spelman;citation_author=M. E. Goddard;citation_title=Linkage disequilibrium and persistence of phase in Holstein-Friesian, Jersey and Angus cattle.;citation_volume=179;citation_year=2008;citation_pmid=18622038;citation_doi=10.1534/genetics.107.084301" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=B. Devlin;citation_author=K. Roeder;citation_title=Genomic control for association studies.;citation_pages=997-1004;citation_volume=55;citation_year=1999;citation_issue=4;citation_pmid=11315092;citation_doi=10.1111/j.0006-341X.1999.00997.x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. A. Edae;citation_author=P. F. Byrne;citation_author=S. D. Haley;citation_author=M. S. Lopes;citation_author=M. P. Reynolds;citation_title=Genome-wide association mapping of yield and yield components of spring wheat under contrasting moisture regimes.;citation_pages=791-807;citation_volume=127;citation_year=2014;citation_pmid=24408378;citation_doi=10.1007/s00122-013-2257-8" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=Y.-B. Fu;citation_author=B. Cheng;citation_author=G. W. Peterson;citation_title=Genetic diversity analysis of yellow mustard (Sinapis alba L.) germplasm based on genotyping by sequencing.;citation_pages=579-594;citation_volume=61;citation_year=2014;citation_doi=10.1007/s10722-013-0058-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=I. Gawenda;citation_author=P. Thorwarth;citation_author=T. Günther;citation_author=F. Ordon;citation_author=K. J. Schmid;citation_title=Genome-wide association studies in elite varieties of German winter barley using single-marker and haplotype-based methods.;citation_pages=28-39;citation_volume=134;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. A. Golicz;citation_author=P. E. Bayer;citation_author=G. C. Barker;citation_author=P. P. Edger;citation_author=H. Kim;citation_title=The pangenome of an agronomically important crop plant Brassica oleracea.;citation_pages=13390;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Z. Guo;citation_author=D. M. Tucker;citation_author=C. J. Basten;citation_author=H. Gandhi;citation_author=E. Ersoz;citation_title=The impact of population structure on genomic prediction in stratified populations.;citation_pages=749-762;citation_volume=127;citation_year=2014;citation_pmid=24452438;citation_doi=10.1007/s00122-013-2255-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Hasan;citation_author=W. Friedt;citation_author=J. Pons-Kühnemann;citation_author=N. M. Freitag;citation_author=K. Link;citation_title=Association of gene-linked SSR markers to seed glucosinolate content in oilseed rape (Brassica napus ssp. napus).;citation_pages=1035-1049;citation_volume=116;citation_year=2008;citation_issue=8;citation_pmid=18322671;citation_doi=10.1007/s00122-008-0733-3" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Y. Hasan;citation_author=W. Briggs;citation_author=C. Matschegewski;citation_author=F. Ordon;citation_author=H. Stützel;citation_title=Quantitative trait loci controlling leaf appearance and curd initiation of cauliflower in relation to temperature.;citation_pages=1273-1288;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Biological Journal of the Linncan Society;citation_author=J. Hawkes;citation_title=The importance of genetic resources in plant breeding.;citation_pages=3-10;citation_volume=43;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. Hayes;citation_author=P. Bowman;citation_author=A. Chamberlain;citation_author=M. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost.;citation_pages=1681;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=BioMed Res. Int.;citation_author=I. Hwang;citation_author=R. K. Manoharan;citation_author=J.-G. Kang;citation_author=M.-Y. Chung;citation_author=Y.-W. Kim;citation_title=Genome-wide identification and characterization of bzip transcription factors in Brassica oleracea under cold stress.;citation_pages=4376598;citation_volume=2016;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. Jarquín;citation_author=K. Kocak;citation_author=L. Posadas;citation_author=K. Hyma;citation_author=J. Jedlicka;citation_title=Genotyping by sequencing for genomic prediction in a soybean breeding population.;citation_pages=740;citation_volume=15;citation_year=2014;citation_pmid=25174348;citation_doi=10.1186/1471-2164-15-740" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=C. Jestin;citation_author=M. Lodé;citation_author=P. Vallée;citation_author=C. Domin;citation_author=C. Falentin;citation_title=Association mapping of quantitative resistance for Leptosphaeria maculans in oilseed rape (Brassica napus L.).;citation_pages=271-287;citation_volume=27;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Jombart;citation_author=I. Ahmed;citation_title=Adegenet 1.3–1: new tools for the analysis of genome-wide SNP data.;citation_volume=27;citation_year=2011;citation_pmid=21926124;citation_doi=10.1093/bioinformatics/btr521" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S.-y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=A. Korte;citation_author=A. Farlow;citation_title=The advantages and limitations of trait analysis with gwas: a review.;citation_pages=29;citation_volume=9;citation_year=2013;citation_pmid=23876160;citation_doi=10.1186/1746-4811-9-29" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=S. König;citation_author=H. Simianer;citation_author=A. Willam;citation_title=Economic evaluation of genomic breeding programs;citation_pages=382-391;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109296;citation_doi=10.3168/jds.2008-1310" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T.-H. Lan;citation_author=A. H. Paterson;citation_title=Comparative Mapping of Quantitative Trait Loci Sculpting the Curd of Brassica oleracea;citation_pages=1927-1954;citation_volume=155;citation_year=2000;citation_issue=4;citation_pmid=10924486" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=F. Li;citation_author=B. Chen;citation_author=K. Xu;citation_author=J. Wu;citation_author=W. Song;citation_title=Genome-wide association study dissects the genetic architecture of seed weight and seed quality in rapeseed (Brassica napus L.).;citation_volume=21;citation_year=2014;citation_pmid=24510440;citation_doi=10.1093/dnares/dsu002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. Li;citation_author=Z. Peng;citation_author=X. Yang;citation_author=W. Wang;citation_author=J. Fu;citation_title=Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels.;citation_pages=43-50;citation_volume=45;citation_year=2012;citation_pmid=23242369;citation_doi=10.1038/ng.2484" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=H. Li;citation_author=Q. Liu;citation_author=Q. Zhang;citation_author=E. Qin;citation_author=C. Jin;citation_title=Curd development associated gene (cdag1) in cauliflower (Brassica oleracea l. var. botrytis) could result in enlarged organ size and increased biomass.;citation_pages=82-94;citation_volume=254;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Liu;citation_author=Y. Liu;citation_author=X. Yang;citation_author=C. Tong;citation_author=D. Edwards;citation_title=The Brassica oleracea genome reveals the asymmetrical evolution of polyploid genomes.;citation_pages=3930;citation_volume=5;citation_year=2014;citation_pmid=24852848;citation_doi=10.1038/ncomms4930" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=C. F. H. Longin;citation_author=J. C. Reif;citation_title=Redesigning the exploitation of wheat genetic resources.;citation_pages=631-636;citation_volume=19;citation_year=2014;citation_pmid=25052155;citation_doi=10.1016/j.tplants.2014.06.012" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=C. Matschegewski;citation_author=H. Zetzsche;citation_author=Y. Hasan;citation_author=L. Leibeguth;citation_author=W. Briggs;citation_title=Genetic variation of temperature-regulated curd induction in cauliflower: elucidation of floral transition by genome-wide association mapping and gene expression analysis.;citation_pages=720;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=G3(Bethesda);citation_author=G. P. Morris;citation_author=D. H. Rhodes;citation_author=Z. Brenton;citation_author=P. Ramu;citation_author=V. M. Thayil;citation_title=Dissecting genome-wide association signals for loss-of-function phenotypes in sorghum flavonoid pigmentation traits.;citation_pages=2085-2094;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. J. Norton;citation_author=A. Douglas;citation_author=B. Lahner;citation_author=E. Yakubova;citation_author=M. L. Guerinot;citation_title=Genome wide association mapping of grain arsenic, copper, molybdenum and zinc in rice (Oryza sativa L.) grown at four international field sites.;citation_pages=e89685;citation_volume=9;citation_year=2014;citation_pmid=24586963;citation_doi=10.1371/journal.pone.0089685" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=W. E. Nyquist;citation_author=R. Baker;citation_title=Estimation of heritability and prediction of selection response in plant populations.;citation_pages=235-322;citation_volume=10;citation_year=1991;citation_doi=10.1080/07352689109382313" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. Okazaki;citation_author=K. Sakamoto;citation_author=R. Kikuchi;citation_author=A. Saito;citation_author=E. Togashi;citation_title=Mapping and characterization of FLC homologs and QTL analysis of flowering time in Brassica oleracea.;citation_pages=595-608;citation_volume=114;citation_year=2007;citation_issue=4;citation_pmid=17136371;citation_doi=10.1007/s00122-006-0460-6" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y.-F. Pei;citation_author=J. Li;citation_author=L. Zhang;citation_author=C. J. Papasian;citation_author=H.-W. Deng;citation_title=Analyses and comparison of accuracy of different genotype imputation methods.;citation_pages=e3551-e3551;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18958166;citation_doi=10.1371/journal.pone.0003551" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=BMJ;citation_journal_abbrev=BMJ;citation_author=T. V. Perneger;citation_title=What's wrong with Bonferroni adjustments;citation_pages=1236-1238;citation_volume=316;citation_year=1998;citation_issue=7139;citation_pmid=9553006;citation_doi=10.1136/bmj.316.7139.1236" />
<meta name="citation_reference" content="citation_journal_title=Food Chem.;citation_author=V. Picchi;citation_author=C. Migliori;citation_author=R. L. Scalzo;citation_author=G. Campanelli;citation_author=V. Ferrari;citation_title=Phytochemical content in organic and conventionally grown Italian cauliflower.;citation_pages=501-509;citation_volume=130;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome Journal;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103;citation_volume=5;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome Journal;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92;citation_volume=5;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Rezaeizad;citation_author=B. Wittkop;citation_author=R. Snowdon;citation_author=M. Hasan;citation_author=V. Mohammadi;citation_title=Identification of QTLs for phenolic compounds in oilseed rape (Brassica napus L.) by association mapping using SSR markers.;citation_pages=335-342;citation_volume=177;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. C. Romay;citation_author=M. J. Millard;citation_author=J. C. Glaubitz;citation_author=J. A. Peiffer;citation_author=K. L. Swarts;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_pages=1;citation_volume=14;citation_year=2013;citation_doi=10.1186/gb-2013-14-1-r1" />
<meta name="citation_reference" content="citation_journal_title=G3(Bethesda);citation_author=J. E. Rutkoski;citation_author=J. Poland;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Imputation of unordered markers and the impact on genomic selection accuracy.;citation_pages=427-439;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=L. R. Schaeffer;citation_title=Strategy for applying genome-wide selection in dairy cattle.;citation_pages=218-223;citation_volume=123;citation_year=2006;citation_issue=4;citation_pmid=16882088;citation_doi=10.1111/j.1439-0388.2006.00595.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Springer Berlin Heidelberg;citation_author=K. Schmid;citation_author=P. Thorwarth;citation_title=Genomic selection in barley breeding.;citation_pages=367-378;citation_volume=69;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=V. Segura;citation_author=B. J. Vilhjálmsson;citation_author=A. Platt;citation_author=A. Korte;citation_author=U. Seren;citation_title=An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.;citation_pages=825-830;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706313;citation_doi=10.1038/ng.2314" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=P. C. Sham;citation_author=S. M. Purcell;citation_title=Statistical power and significance testing in large-scale genetic studies.;citation_pages=335-346;citation_volume=15;citation_year=2014;citation_pmid=24739678;citation_doi=10.1038/nrg3706" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Sonah;citation_author=M. Bastien;citation_author=E. Iquira;citation_author=A. Tardivel;citation_author=G. Légaré;citation_title=An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.;citation_pages=e54603;citation_volume=8;citation_year=2013;citation_pmid=23372741;citation_doi=10.1371/journal.pone.0054603" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=B. Collard;citation_author=E. Redona;citation_title=Genome-wide prediction models that incorporate de novo gwas are a powerful new tool for tropical rice improvement.;citation_pages=395;citation_volume=116;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Rapid identification of alleles at the soybean maturity gene E3 using genotyping by sequencing and a haplotype-based approach.;citation_author=A. Tardivel;citation_author=H. Sonah;citation_author=F. Belzile;citation_author=L. S. O’Donoughue;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Tester;citation_author=P. Langridge;citation_title=Breeding Technologies to Increase Crop Production in a Changing World;citation_pages=818-822;citation_volume=327;citation_year=2010;citation_issue=5967;citation_pmid=20150489;citation_doi=10.1126/science.1183700" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Thorwarth;citation_author=J. Ahlemeyer;citation_author=A.-M. Bochard;citation_author=K. Krumnacker;citation_author=H. Blümel;citation_title=Genomic prediction ability for yield-related traits in German winter barley elite material.;citation_pages=1669-1683;citation_volume=130;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=R. Uptmoor;citation_author=T. Schrag;citation_author=H. Stützel;citation_author=E. Esch;citation_title=Crop model based QTL analysis across environments and QTL based estimation of time to floral induction and flowering in Brassica oleracea.;citation_pages=205-216;citation_volume=21;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. F. Utz;citation_author=A. E. Melchinger;citation_author=C. C. Schön;citation_title=Bias and Sampling Error of the Estimated Proportion of Genotypic Variance Explained by Quantitative Trait Loci Determined From Experimental Data in Maize Using Cross Validation and Validation With Independent Samples;citation_pages=1839-1849;citation_volume=154;citation_year=2000;citation_issue=4;citation_pmid=10866652" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. VanLiere;citation_author=N. Rosenberg;citation_title=Mathematical properties of the r2 measure of linkage disequilibrium.;citation_pages=130-137;citation_volume=74;citation_year=2008;citation_issue=1;citation_pmid=18572214;citation_doi=10.1016/j.tpb.2008.05.006" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Wimmer;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=C.-C. Schon;citation_title=Synbreed: a framework for the analysis of genomic prediction data using R.;citation_volume=28;citation_year=2012;citation_pmid=22689388;citation_doi=10.1093/bioinformatics/bts335" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.;citation_pages=1427-1436;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=T. Würschum;citation_author=S. Abel;citation_author=Y. Zhao;citation_title=Potential of genomic selection in rapeseed (Brassica napus L.) breeding.;citation_pages=45-51;citation_volume=133;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=A. Xavier;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=Impact of imputation methods on the amount of genetic variation captured by a single-nucleotide polymorphism panel in soybeans.;citation_pages=55;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Yang;citation_author=Q. Wang;citation_author=Q. Chen;citation_author=R. Liao;citation_author=X. Zhang;citation_title=A new genotype imputation method with tolerance to high missing rate and rare variants.;citation_pages=e101025;citation_volume=9;citation_year=2014;citation_pmid=24972110;citation_doi=10.1371/journal.pone.0101025" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=E. A. A. Yousef;citation_author=C. Lampei;citation_author=K. J. Schmid;citation_title=Evaluation of cauliflower genebank accessions under organic and conventional cultivation in Southern Germany.;citation_pages=389-400;citation_volume=201;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title= bioRxiv;citation_title=Comparative analysis of genetic diversity and differentiation of cauliflower (Brassica oleracea var. botrytis) accessions from two ex situ genebanks.;citation_author=T. Yousef;citation_author=T. Müller;citation_author=K. J. Schmid;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=X. Yu;citation_author=X. Li;citation_author=T. Guo;citation_author=C. Zhu;citation_author=Y. Wu;citation_title=Genomic prediction contributing to a promising global strategy to turbocharge gene banks.;citation_pages=16150;citation_volume=2;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=Z. Zhao;citation_author=H. Gu;citation_author=X. Sheng;citation_author=H. Yu;citation_author=J. Wang;citation_title=Genome-wide single-nucleotide polymorphisms discovery and high-density genetic map construction in cauliflower using specific-locus amplified fragment sequencing.;citation_pages=334;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. Zou;citation_author=C. Jiang;citation_author=Z. Cao;citation_author=R. Li;citation_author=Y. Long;citation_title=Association mapping of seed oil content in Brassica napus and comparison with quantitative trait loci identified from linkage mapping.;citation_pages=908-916;citation_volume=53;citation_year=2010;citation_issue=11;citation_pmid=21076506" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Genetic resources are an important source of genetic variation for plant breeding. Genome-wide association studies (GWAS) and genomic prediction greatly facilitate the analysis and utilization of useful genetic diversity for improving complex phenotypic traits in crop plants. We explored the potential of GWAS and genomic prediction for improving curd-related traits in cauliflower ( Brassica oleracea var. botrytis ) by combining 174 randomly selected cauliflower gene bank accessions from two different gene banks. The collection was genotyped with genotyping-by-sequencing (GBS) and phenotyped for six curd-related traits at two locations and three growing seasons. A GWAS analysis based on 120,693 single-nucleotide polymorphisms identified a total of 24 significant associations for curd-related traits. The potential for genomic prediction was assessed with a genomic best linear unbiased prediction model and BayesB. Prediction abilities ranged from 0.10 to 0.66 for different traits and did not differ between prediction methods. Imputation of missing genotypes only slightly improved prediction ability. Our results demonstrate that GWAS and genomic prediction in combination with GBS and phenotyping of highly heritable traits can be used to identify useful quantitative trait loci and genotypes among genetically diverse gene bank material for subsequent utilization as genetic resources in cauliflower breeding." />
<meta name="og-title" property="og:title" content="Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/707" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic resources are an important source of genetic variation for plant breeding. Genome-wide association studies (GWAS) and genomic prediction greatly facilitate the analysis and utilization of useful genetic diversity for improving complex phenotypic traits in crop plants. We explored the potential of GWAS and genomic prediction for improving curd-related traits in cauliflower ( Brassica oleracea var. botrytis ) by combining 174 randomly selected cauliflower gene bank accessions from two different gene banks. The collection was genotyped with genotyping-by-sequencing (GBS) and phenotyped for six curd-related traits at two locations and three growing seasons. A GWAS analysis based on 120,693 single-nucleotide polymorphisms identified a total of 24 significant associations for curd-related traits. The potential for genomic prediction was assessed with a genomic best linear unbiased prediction model and BayesB. Prediction abilities ranged from 0.10 to 0.66 for different traits and did not differ between prediction methods. Imputation of missing genotypes only slightly improved prediction ability. Our results demonstrate that GWAS and genomic prediction in combination with GBS and phenotyping of highly heritable traits can be used to identify useful quantitative trait loci and genotypes among genetically diverse gene bank material for subsequent utilization as genetic resources in cauliflower breeding." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"xyTR1Hp_Pt2FqFZyOKOe5bYZ_gdr1g47w8XxLu2a0UY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488298","apath":"\/ggg\/8\/2\/707.atom","pisa":"ggg;8\/2\/707","ac":{"\/ggg\/8\/2\/707.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/707.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/707"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/707"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/707":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Q3NHUqSXds3x5wlzZfy6Al1b7OhY5cVoufxwv0mIDp8kwPuYdM69i7NIrfI3lMIaeY7XJ1OCJH5lumRXMCvXszcJ4E96G90uESNfRnoxLszhyvJjjz1bZhTCQlqnJlOhPOKRWmcCJPBhQYzvT\\\\\\\/xBadO93h29neDYCr1qBY8vMOqzAtnELtoYhmVTjEPqn3mHLPZUh9yt2gK5JLbvWaJHKTnrubSs7WxihnORM1GiOc+goCbepYJvdjx+ZWbKM6x3CON+awpQqE\\\\\\\/B5uSzJKlsA5HrzIowUf2nOhADyFwmHUO1pRKcUZHOr8jTSVLpgfvtR+\\\\\\\/MUwUu3O9Far8OTi3VNlj8uN9elX8tuovvikN2rt\\\\\\\/d3xOQJ0\\\\\\\/Fyguz\\\\\\\/4\\\\\\\/U53eZwEzGK+5k\\\\\\\/Gd4Ag9nmO+jFwiCThLdC6wFUnwa5RO0aYY54\\\\\\\/NwUMhqGjrqkvwJ0vgJODSR6CXDa+4ExMRBQlUfbNmxnivjEMQjGUMbv8+0Gd47X+kRPe3h72AtynS\\\\\\\/GYIvZAQmAKLXJwchiUFgT93YStxqUg04biPmUfXoEUrSWoAYXw+rW21ZWrPpZM6X1G6NIDWKz2fc9+uRhbPaUfbA9gqMhjzrQmQRsSG630B2TT12IMUGy97ehpmthXEUL2W1RVBMDqPXF5oaIGGjlH1VNMEgAvr4pMgCOcXk37UARPDHdSx\\\\\\\/Pf+VldqvfiKkK7jl+rKfEsnKxLgVJ4TTwakQMJ+FXUnU3gr0ErwVrW5H5NMEogKSTShZsFtduIbENj7Kl0o1SZ0PYVKaLcnf25TW8NjzqhgB\\\\\\\/CKP4fV0yhhX5+dKSfRrS2CSZG98niUMs5yfN9Ot5O1WzEQi8HboMwXldKp5aSw8iD+W4\\\\\\\/kDN6rj193PPLyp7wuA8JVHazD+HAmsjQzGcsEm5c968+dO6GzmcFP1dLcY5ymLcsBV8OuUzfHtNL82r1hHJxdB08\\\\\\\/eNYe3nFpdTrXXPtUhs3yt\\\\\\\/uNKtUhZQYQnoARoA6wD36S5JHGzgYzb49Y7v0X9gFuQkE5A9pg2dLYf51lXQH4Gwh14ERDPfYvK4xMPh5vpqt6RUsTkfb2o3Lf5PKjoVyFvqO\\\\\\\/ulbqffOF8Vm5vsTcp3Vt6mP0m5N3E6IGttvweIB6YT6L+vs6+8paquEQBLao5w2pQK1FEjuJY0fhFxA0atCvTsKj2xRu2HgSJ+5ZfxoWhhKV\\\\\\\/zwNGOLJZp4kPg0cerabpjgvhvuvrhaEIXC0NiZzA65DDusbu2CWEs6WULYlPdXsn8jMhWHNqI3Q9AEIPLmaz\\\u0022,\\\u0022iv\\\u0022:\\\u0022LGEXkRXMG69hBkshpp+mag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e13fe52f44c486e9af39e6cb8a0d4790a37fdf22652d3f9f225ff35f6e90691f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YlsLTyTe2fu2o+12CHFusH4Rz9kAXaxxotdTinHTyMX7L2hd7\\\\\\\/PhCrzWtY+Ou6lohH6zY\\\\\\\/b+DeeFZNePVsNxeNFZt0rRK7yYR712GGjgyQUaHb8rVrjIsXMHovJ8HNYYVArPnVgi6m9IDiwqHyJUArAxHpnF7Joh2IZVt+MkH92LNIthJUhCWQF4AM09hcTgnrKkPhRXiyNyOqoLTOiRbY0NM2p9RRfDLCWo\\\\\\\/lPpawyS6z9EmRXDLdIYKS7WHp5RSVYE5TERJaY89PGM8yzdx9zHxZzTYBsMMRcZ7zL5xsZEt60xAj5ohrMW3Ujwd4ZOML4LoAFFKJHIX2w0xvvsLgZv8nsZfEaBiHGF8cPVm3irvcTrhhoBRXw9puaz0lLJ6kvlnv6nMGrcJAe57xA82vcvHxpbLtnTiZonWk\\\\\\\/TR0nRlixxzVCBY0KvSJqkGFmURf8D9LMHWBt7\\\\\\\/7\\\\\\\/5vWONIzWgT0x6cXgX4puaWOm7YNuQq9UlFooX3s8\\\\\\\/y0pe\\\\\\\/zQ4rDwID6LbZKZlVbG0zgNRNC2yX4yQWUHhYwmeSQkSO6T+RNeZhpT0KKltpUaPL10ct9oj2sYne\\\\\\\/jCoymT\\\\\\\/Ihz31FNdVstYgrGtDYsXVcuF1nFZ0QQTUKSRXUQ1kbbeM16wgSVi0BQafDDUP\\\\\\\/r5B3feJvh4vn4m5AZsf21p\\\\\\\/uK18DceocL9Oq8ZU7+Wh48veq9gEUZiQXR1AgFBp\\\\\\\/+eTwW1ZJlDTTib6YYb\\\\\\\/qVFQ0V1on0EgdUJ8GaTvyJsyyXBVwCilVBH8VL4OkMlWCyXkrSMGP\\\\\\\/RWPfzNb3NzoSXLH9aasd3+0p5pD8veDZHt4wQGTcCs3Y8bUMPcNsFLIIVBUqf0lfyFPkgsCKTvjHU5kBBGiqzsPSDPZn\\\\\\\/0rfY\\\\\\\/iu9hwLforT8L\\\\\\\/BcAZ\\\\\\\/uDu5WvprBGfg726ZRzHIz8Hb+M\\\\\\\/IYAHoDFyQQGAeAKiH6K6cLN4c4qJ2ab6URBrSjrf6Drfd1CyDF+sGjPAVb\\\\\\\/wjjvmBADOEUbHfYmGyRsYWjElphEgdJ7hyNSwKbQMR28GHtq0OpXn\\\\\\\/26vNqV9rl0zFXVa9bsTwfWsbxv8WBL0YgI0sfgYXTtijhhCFcKpyWtFrw6XvtbFOVMfMbKqphDond+0exlCnEwcxf16m6EPzPVeJ0nqkG9lmxc4ni+J7q3tjW+R5O6lkHUyO35r01Ci+p5Vo+W9kPTn6F3KDISYOjvsvH3zx1MCLmWHyZP\\\\\\\/hAN\\\\\\\/gZ1S+E9lgfmbICHVaCOuEpSDHiYHNuHnPn4rqDZ4aPjAIpuDbg4qzkhVxAmFod6wbQua\\\\\\\/tbICEeJ3wHGlK152nMg799V4NYNTDTjUUbsB\\\\\\\/9xOE5kAolCGS\\\\\\\/px6mAlgPd5nZNdTGR9q4hDCN\\\\\\\/1EJNcZbQXT+i1XQnbtHkw\\\\\\\/\\\\\\\/yYJsr2lQ0V+lGsgNA2PQmxuwvL\\\\\\\/mKT+WIt0+A4iXYpbDnCYYERO4MzmP7hDabomQ2CgS9SFihZIQUDvnpfiKWjj0uHcLGBb9Qx7BaR8o7cpXuZNhhOQoQCHvPwRiWZasyBQglSGK\\\\\\\/KI134VYsMIhPMYi70BS9BPiF6syEz5SzOP5FCGMo8qFkVS8RTAdbNM5tqcqu8k6CM3cyH\\\\\\\/sPURwjHcrKb01xRXXcNrP\\\\\\\/Pjhb5bURBWY+8BZmsz9q0Mdvbv1+SDztqc2gg6lsf7VRtj3bwRyn1taPcpPZfW9za\\\\\\\/adN5Udk9wuoiqkXWdQpVxCyQxvP7OFyelUv56w4YZdizukml0RnjbULgRTDz5xV\\\\\\\/j4d2+wM6SV1Syc0\\\\\\\/r2P6cKm4+rZUfrDisLuX4Fn9DPAXZ9B4a4qPf+K9zEHssdQSEeYDJ+NSstHMkBLr7ce600ch\\\\\\\/cz14QF3aKcnzgG7lXKpWmRmLUrX1nUakQM7Sx2gH05AdvEigQ8ynSi5veblMPU7\\\\\\\/rxt6UO639Zu+60b5giUh2tX80\\\\\\\/dcul8QjRw+luG6egLBZUFPmKujgwdCdya8Hf8WRiniQW95G6zkLytnaDueYG849as9vHdmH0zRn+bwo2ZEVIcdY\\\\\\\/Z3lHD8J9rPLVJG4mUw1NkqU5QjxPpU7uKbOTBrwik7xlF3h8C2fB7g5RPuMiEPXPo0aNw3AtbglkZF15+YhffmxKrlABA6sadEb8Bt2LLpNLLNFCXq4r94ukXJf3yk+SbkdwW5WtixnhFN4lzpAAv27\\\\\\\/BFKzPNsoq3tiElOhxGzP2\\\\\\\/sbv1NflM1VO6Jxd3D+NWnsv2Hz3Df73\\\\\\\/ob2qc20TOptTG4KPBufM29QdYNn3X60mN4LALx6t4y4sI8KyN4kpdSchotVhSLkf3YfPIEO9F7vGKabSYdKrHT5a5sVUzxSXwSobHRuEOZcoO0zRYvwYvYtce6J3S5+rMp0Sqo3584EjCqELY+m2P2ki14yMJmMI0AZz3c+etgT45coabIOOcBgNRQ4IZHl37aqR+c9pUOIfJhauPtN\\\\\\\/IxoXQmn7FfBkVBTr5ChJjG1p3UYCNIbeo\\\\\\\/B608mz\\\\\\\/dnIaxIE+XhR\\\\\\\/nsQXh6ZCBsfnBR3PyDSMKvVubtFxgs9daIRQGVlEKb2JKiK6GiHfBd2OtAzlLijxpFDktowxab\\\\\\\/f4WjpKzX7N9NuWRSxFhZ22ldPLHfl3HnIUokNSldn8FBzXqDaSbSmZNc7rB0OWMZTm8N+MefrfOeNCTS5SZxtO5GXUTHmLSd\\\\\\\/uJ\\\\\\\/IZhJZg3qqYMpOULx9mfcLIyfaGDl0NyqSHR4hb+sAYAVX6G6tMnDAGo8KqdDuUa9EmlqvS6qJtR77g1A5vySCeNppqrc6Ux48LQgzHfLWcHCuF3NjbCN+5Hgi9Okp6VFa9BOMPbQosZGVu+R7xswWFrbgtNAd9Ey9btExvLbVkGvH9gRsCkFkakoyK8\\\\\\\/KOxHrRIEiLD187xlAvgik4jP8HJEtYFerpXABkfIpGDl8tVGBv5IqzX1UMVmkfNS9tppV9owyppfdm2VT1winjuQ+Z16\\\\\\\/0bgg+H4DYBsetAmOb5kqc7nCYvdzopjd4w0lhYBsG1wZy4p2+PB7hxg9vAabB4T4qFnsXfAP1nm5gwJj9dMNZYN2jnFB96ZwqZn\\\\\\\/\\\\\\\/CxS3DdCb35OmFxMuQGWPcJOZebxBHDN\\\\\\\/\\\\\\\/viNPaXl7IbFqBE5oq2cqEJiWs3yNQR7LE90H879U7MdvQlDMlePCjRcpf8y0JvNYa41hklpBcsapBmvL5q\\\\\\\/tz8qr7tv57s2JbwnLkDYpb96\\\\\\\/Np68xtXh+uLHJyE8rwKMLtQu\\\\\\\/V36lRnYywfKgqZeWhU3w0IzvoWwWK9HOsmAUogetSjMOB4lWHAjGUsStrSg+MVGVHXjO5bfsdjxe53oOkBQa4J8FF56d8asw5NxCSvY+gz6G2BrBUxf2KZbTYK6hoptz4W97F8To0ppoJx0OKAOYUB4zypP\\\\\\\/u5O6apN+qlwOjDGS+tzHOK1t+qG7PX1Pe0nP+lePEsPiSMFjnsVWR1N9L7AE09k42z2N5SIAjMNXMNsEVVYE1TOoDBUGDQlekx\\\\\\\/TWDjRK\\\\\\\/SNzkQFQ62WUEMM4FJ1u7H7sxk0Uhib8Hs5VT\\\\\\\/RNYB8jGvwQXTy27q6NFmahMKOIWBT8nBReKaEk6E1ngemICaynqL59XhEhxEdQfm3GFLSzsfTHhfXu+9qMWKAQmLuW3NMtch7I2bzFhmk4fLlUor+5qcgx6MhLO3CbzHb2GJTRnZtWhsE+h51rx8WTsIlADdpWqJHc4C3N2bC+vnDyTPNGECkF\\\\\\\/WC9Bj7dK5iLWHwdEvY76hwfy3jkZyL2XlvDyiVHifpzSWuynstw22QiLOde9\\\\\\\/Y+gJzJjNXcuKUQldqVcKqkNFYVGj9blAxQrmAXu7F1L6fpqRKFRk6KZr2RKh1FvSI+fbx67zjG2GDgLMVqajf9ZALqJOl2Ma\\\\\\\/e3cZf7k3XA3rjfVDWRpLXLdhlidLKF9z085iP7JwmN0qYX90zQIOCYX1eM5RlOeY8chiQlt59DzZ54ckY8aFArzc9Cmb0zgcQqLlxrVmCvmI7t5slXE87wS\\\\\\\/qgcbpvM8tMX51rNhdz2a3mEsLuf+zhh3Rn6mjbBcgngb5SpSjBcRt1eMlaRTs0EN1CPeusVLF1LAjzA1ThhcKaAlp573hn55MFATBdXgZTL9IEUeYDKufGfVwu+NeQWFkS9YSvTT4QZu9j2qIGxZY9lOt7dfKQH04UD13XSmnDc4BGPmLUIx24OBylX2Had5OdivP3B6VhyijIMlzSHu8EEhRKExrPqZXvgySDrcfw6Aa2X5QbgFz+DbPSWp8sW83ezi0sdFCNudL7bmDEFxbanbop0Z7Z2hdQy92RmRY0\\\\\\\/NRk3uFaYU2d+YLwqli5rVEkQZWNcqa\\\\\\\/EWTnaHqPpA3eXhSccSIEP9z5ergVqEwnlllg8ISD0I\\\\\\\/94SaXxAHFeGFDRzqx1abpQahs0AzVyzre6S9BXpH7hlVrG0UAStjF7wH9pzH1GGBFZuxNZPtgFxtD3kQU1kBWYuwFv3Epp81fZAji1c7Cem7Z0LjAtTXAu8KXJnvWKZD9LJ5w6ZLX2JgBnledpvuZHBIM+8kJq+yRnpVPx4DK4dQ1P9eNUrDEkfUaXcaAd1yYoe47GXqkX07KyKOTyTS3k+pgbnrbSw06lssZmvCm\\\\\\\/LgXDLoJM9+j2WZ9yKD+JpWnfR5O8OxAfeqLHs+Ua3HKmemRinNol1dQa\\\\\\\/FTX\\\\\\\/OWIemNtLpDmNTU2p0KLF\\\\\\\/dasM6cR9Uxg49PwOaj2TNl6mtWfATumKlmhUE07n8TWnmvQpAdUb3hNAbw1TtxVScFF1720cndZE2YBd4qnhsc5p0Sb31rY6nMvZby+jSwq4othkCg1d2u4NsC4cH1C7s\\\\\\\/U0DgkM5OAQDllJ5iF3Vskt\\\\\\\/OxXQT4BnyO0n7cTk6whxjrvo47AXr3j5TZ0gJwbOTreqg\\\\\\\/NRwK0ftu6VwGt47h\\\\\\\/7QR2CiDWrOFMRnNvZn1\\\\\\\/e3OMJi4ce9vk1XDlKA701+NQwZNvH8j8d7Dp433aIpCo+mM5O8K9ro8+xXUDgYUbnze2u61kGIcmFsBjPtOiIqVBtPz27ZWKMnBCadjslwflglAt2hGW8YzM6iKpRrATxAziQJvv0rN1i44RGopmbakwY5zasNCByFcKquhlpQ2EcAUibgQp5syB1dMqfWxY8DLLLo0dAij4tLqUKSuD07vizU9DjuRyNjv6DcdEhlOAAESunwbnSih6RffDKJ26oPPehLIbaclBddPLIcaml7IfgqAVY+t8CAxLyZ4q4nrxX\\\\\\\/5mTrv4by3VV+1Df8KCBxbSMFkLr7s63eCtuGJeCHZnw7OACI21yh6Azl\\\\\\\/7R0zkk24fsNNYd+pLddqSn+PH3e0d6YDj9tXyC8L0Sg7O5gVWeDcbtXj\\\\\\\/SaVKcPvcyg5alRJ7f7peOCu8Lc2LNC9KDhX1oDbsnNLnA2yS3VZaRFWVbgb7+v0+3XecW1d5Je4g70+2Ijszy1NNViP6Rxr8EC10ruWFXYjB+jcYoR+UNVE9j+29i5tgdg\\\\\\\/HmsJ7HZ1BYLmqKlXMLroPdkpc3ury51w04KKeQrJPyRSmcp\\\\\\\/Xtt6nfDHVE8rXtYO1VdnvmX6Em6P0UPwKbtYJqG48KF8NcY3M+QG2zcdPZVT95w7ZwbCvwYtXmc7ciQIaD+4DtgjAqEcFB9x20KLNfVrkhfUttxBC5Vwjo9Sjxga+GEjrZOkBg7aL65kSICgh8VRDeg0fVmv2nOLWnlMi1itXKk3l5w+HgBs+QhwPM0zQRJprw1NVB4Dbtb2sVV43ukbLo9HR0mpUCy6sajVJ4Rj6W10X4IVW7KIRLJH6g42R+N5B\\\\\\\/ODSRBn\\\\\\\/MRsfmVEwydn0WVB5hEWTMU1qzy1jTw99B1q3oOwZO\\\\\\\/A3tkJPzECtDJc\\\\\\\/s\\\\\\\/H7\\\\\\\/ga+aXPGfAkbJ26+grgNBptn0+6H9vtwQQ6DzfIrgwyhx35JJb7YQsUupyQUfxXch72et2mfvvYyenloj06H\\\\\\\/wRnvQJmp9C9WbBCfTpnpCjr5lSF5cRmmFcGNxmjpd9DcOsatRRqj\\\\\\\/qwx3D5CTZPVMH84JDPbjJoGeHMcitkhfG5aPXh3y5SL4EUYQLhc\\\\\\\/lq79HpX8\\\\\\\/UI\\\\\\\/P8Bir8HCm5laWOQDHYb5mvB0CdJKq+Ztetku5KptU\\\\\\\/Ysy1rOxIWAvTuTymvague6USGpVxKV\\\\\\\/8e5oybR2QMss+xCfts1S9WUCT7\\\\\\\/NY64W88UziN\\\\\\\/HF0ik1fcFpa9X+SRRY1aDmfXMdtZ19yqzfI5kMJwciggcVeY+ScdGkMz0NoJ4L6iQOQbgXK\\\\\\\/Z\\\\\\\/H59ZrMomGVLjRKKgQFsB7hGiNY0Hqx8f2BFsms3uAxoqUmF2qhZSDbSACWdrv9Dq920S+jJ1iuyaT8qC159Fxfa95hgygdYt6jdglGoArLsvYc2ls3nbjsVxghgnL0y1uiAvUKUo\\\\\\\/9\\\\\\\/DOEkRjkyEtia96X8BM79V6dOsuWL+SQyq3dQNbIN+BNk68BXwiolwPxg\\\\\\\/v61BEYln9lwbdnKKfe3wyoJRLoEfs7zp2QziYhGvGO33XnsOLs6Lc0mpevDT8PJLpIJr5eYeYfYMzJJx31LaXR1jWMFlY6AcJrlj08z5w3JuHDmdp0PIyfwFhMBMCvkjqdUbLC7fNb4n8JpNfy8pDT4hlMUcp7wzM9qIWcSKu3EG8j5hZ3PiqaxaUdNDmB5WOKzMT+8e2N6d\\\\\\\/2ksiU3dF48A+14CZxUbgI5gQT8Zke2P6QY2bR4ppTT4cE+jUsnNiNnwK7GNTO2+uG3KZgoO7\\\\\\\/7tN0bkj\\\\\\\/T9\\\\\\\/QTeqmCp4kpC1N1pc\\\\\\\/ciDDnWCZwtPM4tQdv\\\\\\\/5QfBxLFtowTJWSPCr41eMZu3d0snpDPuPeVI9rQ5eJNJ4uU5qLb2EXFAbVeGygGLsEJny59DyZjAD4HCbiGKwKU1uA15ToAk\\\\\\\/eH6uJZ+T2hT053pRLX35\\\\\\\/KXT2L+xxKkM43WfwnTfLlDcIzlR\\\\\\\/L+T7uc9YmD5uvVDfjitcUHAMbtrcTzUnkmx7wlua29u3kB2AGPSNXAioDxo96ec+wgXa3SFzYp51AXjtanqimMhMFvPu9IGmeFje9nRN9rDNNqZITj\\\\\\\/DMPp4UtkU1VlxioqHfJDdu5Xg73ERKAfenKGYZpbDiY0+I2fhoBUFWpWaQBmoSJ220\\\\\\\/IgIFEckeQzx3c+NOji5VP\\\\\\\/+EYeYoDc2iBaCN4QhnU21OWQPSGs6xhOVnaLxvCjRafJKvsXsqtM\\\\\\\/rUgZwjVgxN65VcMHkICMp9\\\\\\\/fAh6isnVbNP1wsJp2Qre2iRUmB5zD0YGTKXppyGB5pxQDfXvs\\\\\\\/YHVVjoYbWw5+pcKzbNETd\\\\\\\/LHQ0dolGYvE+2PHG1fCie3DmXhxY7W6CCSwhj57o2XjQ1GxXQBghhiYlat4wotoq6hHNuqH\\\\\\\/rHui8dwZ8Xv0Us5AAfh1ACnbQFcDsfLtoMP2pRWqj4seduJOGrLHeXuV4uKdtg5eXZ2QmiRN3grZ3IhgGXWEWhI3DxAGZh4Dcg+AlBVZqT8CjysBOcalDPIDx\\\\\\\/qOa3nYy89eDT0M9UWhfPJ0tL0Mxl45bZJKL6+HnDDw3hJuY1uac+pnnP1NzUqnna2uo66leWiGH8j4OKJfa4NOWrbJ90QRSaTre9uLz0Y3MrUKoi6jwUn5WCGKhFPV4eD1Qlw1k8sK8vyMVVMhUwic2CM1E0326VClfgUAZTD4URT0rvYmALdOIsFqXYsaf6L3QtTYcF7bj0xEdpyZ\\\\\\\/ujzyhkovgQrcTfyMCBRNr73izcvKxGBbu7CKqpZPpgwonFk\\\\\\\/4FaxbpCphN2Twx9vTpxGYvQ6ZfRr3wNrcaQOaeEKc3DWVHnQUxdDHxEylJ1gzqQNFtkcNXw0bFmgO3Zb39Iyw5ACjdH9q8D0UenCRQtS2Xk\\\\\\\/rL9zYqoZCy3i8n5eNaH7I7XjvdGxSF3llOOCo2zs4aZSUcZnsuH1XO5Lsye2s7rLsfet8E+aLp2BZPmC6Xcv8gLC5uUjk1pvCx11hIqbksOsNyd1OpSMNxCvrqQlV0y9lzKQyST8tMf3uhuhmUPgQkXUcYSt36PP9+jrCzeX5KXMrKspPP9SzPkrB9VF80dZPUzZlcWEQ95N+vjw+XRcXzcB2VR503sj9CjTykoj2QVRlaJjiTyErxcaXADqcjpF90iFs5Gj6l4lb76cFqyXMBBi+uANIqNaD2uWfTBfhwfhm43ns81H5msyM7qYOvGyxTOw2tJ0CFaR5HnlCQiXj9X+IdHc+xKymNpOu2gTnRkE7So3FPhHr66owKqHmdZDbtDfTvRS9JGj07rM\\\\\\\/8sE6ZqUK7edVA5es5F0PiQHCHzSLWzIFaydXhH0L68tcfPeO+J1jwrP5tzT7JLxb7dL+yb5dSranwc9vCjGGTiSoWr8cp9AQjwMl+KMecXw+TkTreZDZsTnS5riVeMymHMKj2A9XtQIkz9JOP5zNmhJ0b5l3JyXVs\\\\\\\/5TFGNJffB+YIs9GQotxSYG01bfIn3ZFI8ar22CleLdHd7v1XaYmvaTTQPqAQ7Zfe0UDKi6hTX693U\\\\\\\/6Eqg233mJN9uVqMlOoKY3ENU8HLsnR6PZlXLooOXcqivpWmiRXg7FijO\\\\\\\/I1tMsENaLuaZ\\\\\\\/yiNmA+mpdyvQ4kNCagC8z\\\\\\\/Np1BxHoe0v9BorV4+uNuzt2q205KXw\\\\\\\/I5VpIUaxJYfuIPgvd4W8Tw5X7l\\\\\\\/9m1JaKyDr2IGbJFcCXMDblPth7Hvq95B6vtW6kYerrtMY9tvRdZYzrRDoYHFMIuUfx24Sgn0WWiffKYZ+szBWFYxaA55hqSkAVY3Sfl5f\\\\\\\/ptMG5yeBTTReiBsXG4LqbLcWDNN9fQ666IxzgDwo9QBook7UzbzklAsUEWcWuHcFdVKPaBQATcKFB5fSGBUK9wEl7TTHRyU1ZBLm5PuACk9dZJbG0F+uGIGieGokCPyPIBfhaXR8EkYX5ZBcgAlbySWp9ZzuvwyIomEX\\\\\\\/UD6lIy5I9v8xOe6x0TN9ZHnP1FBcXhJIfZ+lDRQqwlLSBfbB+L1t6UH4JUAXizWLu9fjO2NAnY2swa8xvP7Rzsgk4TCjsPL9fDuz\\\\\\\/H9llGMjDCu2Pt5gH+5X+b1+vx7\\\\\\\/11xTj1UZ7OOREGWI2+IUJmRVM4FoHOnREQLaJ6\\\\\\\/tmCgmjmUF9XUmBERhUetMsGfxvD+f0ri3BuC4m9axTOfBnElP+GhIs72KMn949wMx\\\\\\\/TkXzxtxYeMJ\\\\\\\/I3SYrWRqRcfLDCcZ4\\\\\\\/4S2nrehKXTSfR5Tjs4drOhWlUFnJ9vnyBExbB2T0qjCcOy8jidWVICqRwqjRz3HMHlT9MKrR1wdGIZ+az937xovdEAeQ8viWNMkD8TZK3mfKPdek9GjyZCkHfIFe1SDLmlrfroKwHyyLAGKXgaD2sM1pdsG2rX3\\\\\\\/SZzpyIPDemwDJpQlpCdyrP2EaxnkIPQqm2Bcw4sselYMeZ1qfKEjbo\\\\\\\/bBXoAF\\\\\\\/tR8PqZy0uQBUTFhEIjEJ7kH435uCShDbfbg40QGKviKjke2b\\\\\\\/1wFIykFL7ucu+0U+xbTjYnAQq5rby0gSCImRU+RGc+HXBI8bFpDRhkeh6HHMW2x2vklCQAPGPRZQBdvilVB9lE16151UFYavRrN9bNfdsE+LMiQSsvivbNtyfGEdLO2bH5IIUjeL0aQo13W3+iT+7+uA3Y9CpHBIU+Da9AhGc2LjF1SXWLtuBYfDK7ePTEc4L+x3sltoyhTC7uCaENjM8Odlt8VkuZRBVxfx8nAQfFaNF7bAavMju+IOxl06eyGnetnizrZg67tB9pi7OoodgfMgmoLYxgNnoETHKfK\\\\\\\/\\\\\\\/ahf9UpH+rHeQ1pzwEFbveeW3dddRXOCe25IJu7lef7b2dXaDreYbS4bKXv8vHg4Yzweai0dUC5KTmqaG\\\\\\\/m3wtoYkFCXQRIR27FsSvM6y2G2FMJrlL06vMstZqu2Sqn97iyEDcEBtBSlJdaRWCvQ5GITDqpPzoVEGTnSet9ouU3IGNjx7sW5d+8mokydQKoUwk7NHlEBoOa66bdGXm8MFsxG5sz+Gy8ctKXU1yE0yXOlJIixhbDVbthadDSrTsf0ho2qowCS2twSIA4Hm3hvPBqHLcOjvtLSWye0+slTdJx\\\\\\\/d\\\\\\\/0UQZSwFmIFITLsa0zYWz3ReYdjKQ3CZplxoRKFbN7qvjsaUGPGluSm0QPNXIDsFG4hr4bb64QJii1qOdZF+3czQu39f2i8G69sjvzyIhmxB3Vu9kiWJ+d35u0g48WHEL9INIc0xNOnxU6XinDIL19eh261BCjFOfcLHwCvzhdZ2\\\\\\\/18RMQA1PIsJw9yRM9K7+U55y21wcFFHrHxfstagp4zt+ZkYRI\\\\\\\/qG6KXcqhz4rmxD8UvjDO6TxqD29w5QImc2E3lu8oLCe2y9QNBdPBNN2Rb2CMvzScPQI13vuMe\\\\\\\/lhM5vTWMS+Fjk5QUfW\\\\\\\/KH+wPANvUJ8iLUNt\\\\\\\/k45cdDNOQsY8a0ZKbM5OAuMe\\\\\\\/JXfdZpLny7oqcM2n5V7koWYpAHpaYF1RMfGcrG\\\\\\\/8iLLILfPFCLNMeV8Ye\\\\\\\/9x6sNBKc6R5MS4iuAU0w0mtzfIhLiyq6NR6PHAMNFpX3iCtZkqFY1oiBdxhQjxzL9hfZQ1K93+zZol9AU3BFtr6+I9RDLpERamUoat0PeAZg6MgS0Be5qR+eMQkTgu6W7UNj3ccrsP84J+FOScd1PylOyG6vBGpbi\\\\\\\/OY\\\\\\\/Ids6zZ+McsYp9ACTyEJ3oQXfasYp3XiG1tJsWFREbFXWj+Smqx8WJTFUPUzIQH+eQ2StxlVDp+Zq\\\\\\\/SCyzWlqQq97mmAZfTlaw2s+Cmhee20Mwc+BcbKCQz98Dd7y++uLgZoysq8jBiYQ\\\\\\\/VLgXfKApmfdoAtxra7LE1DtN58RR56q3DcLJPiID7OcWVBmT9cv365aIURqc\\\\\\\/MAP01TrNDv8FDB+4GEZefmlOOV97NCNXbTqHhZh947a5hhQtB1tfLFUo2BG3ja\\\\\\\/eUOeFy0xGCEdogD5Xv\\\\\\\/B3bG6mGtQnawHP\\\\\\\/OLIsuZ0C4S9030zDYD3d49Lus7SEyh0oVmD+6CAvS9PF+PC\\\\\\\/VO5ymRgZZxYsAUR8LBeKqzw7WLiU37GCvjAPsQw48BZ9ucQ3RjQj5yEICHlVSBGoNquIdzT+1ExDIhhHdViUYEMT\\\\\\\/PQDFda70dlIE6216mZIhW4spF05m4za\\\\\\\/l3F3OqcvfeeNrWOyu0OidUdf27Iz6LVre0NgCqdHXkv9OHtsn5fe0UDdwb\\\\\\\/TWpYAaRbngFWBqosSK86h9skQPdVqYnhnRAEkLhFFLo4GkYQcp53rba97drJRWp7l3LOvZdvi6OyuYpQaMANg7fG4o8K1izNYe13rqNX3LoyR6Xe\\\\\\\/qR8bqGw4doFgyNxMmt+5Wof0XIFnwaktYa\\\\\\\/bL9sCHOvuxLzAPveDz0YTmxWhwtn6lcyOYmWoYXmRKEavfwJMXlyoA4FYQHdJ7sS4mxuWK+5LUlOZy+JW7dDMV21Z9m45AQhjSEvGUHyzsiX+KSN3\\\\\\\/uzokX30xF5xMDr2K7sTf2lQokJoNfSWd0KAoE3NZcOBZcBWBS7+tL54uA5SBqUhIKwLBabK+pRjKMjAqppqlOsGI\\\\\\\/KLBUBdhw1JYq84n5qKrYMyG+muZ4QAER48BlVmYbx\\\\\\\/vvavwSrlXH6aRrPxvINzVuPrQg7Nn1rIOt+rQRRB8IPlADqGaYVr7yGtWbgWgBNINtfZC5X+4w\\\\\\\/lIoDMMVpUhU15iEIR7e+XE6nAjJGJMKMgkM8xNl1TAeC+StIhapEeU6NjcypdT\\\\\\\/pIwBmcgKZz6m26ITfQqwejevBCXRvaqTsm9UWOn8\\\\\\\/RyW4efoZ3+fUP9HlC\\\\\\\/rCXMlAZrwfL9CyXiNZh5TDlsbADr60CfNHqm1EjmCj9rUad9mmAFtkhDj5avalF\\\\\\\/5CPFGQNlXROYpEGfaA7\\\\\\\/m77KqFH8iwmaC9PNr+Me72FtmP+XnnDEg26jjZFYULldDYdkLalpZIMh9i648ytCOdEEqq9LgOb9LpWKIzKVB68Xn70ghLZlKtmT3L58Ex+UnXBFS35k1x+dEZmqGvPaK4QfyvKzsR8cKgJJ+Z67OYGo9JEiuIoZBwVfwo4BXRe7ZrLZCqOTJpertW3AIqStprQLSOuSkGaCSgaj+Qdufd7aPJ7DlAXp3+J0qdabCHbghmfhEGb5UD1oUKBXDJUrM6pGBUBeeM39Z0ymatSr3Ti8LyW22Tt9pMPQYieiXpi3IV6FVT0EyBS\\\\\\\/zbx+iviCO03uzDknc5gm0D068Ig7SmlnRncFRVsGJmtfFxIXaXOSmto1PCGNKddUFhkUooA6rhszpxvjNwxQHkWzAjGogBUBUCxn8tDIjvkWpa0LgfMLQiMhCBy2zFmFOaaoxejp\\\\\\\/P31TujkDMOQVCYOr5ibNcR37DL3xBN1NvSV2WTmdeaTCOIwJ3AtC2bXNJuqQy6Gbd4onsdGeSK7h37bJ5XMHiLJbwbAbp8tDbZkderyHlROy\\\\\\\/mYaztgHQd5ZZCYv2T5EXhslC2vDbPJ80G6GWHwUTA6gqZ02KpNIAgHR9jYS+INcL2lnkGmQX9yGewnhn2wll62dIIokmnF3\\\\\\\/Eb56nBjbEuGwA3ond4cjTItp1pRKJFouBHOAVGKA0wQNCIS2Q1opI6FtHkccugG8dEszoUHvEH8s\\\\\\\/Nw7qE8wpk1QNZ2R\\\\\\\/EZDJcmR0JKP+q29i8djMLDRF9yWnN8lEDbMbGYjvjs57F+paKy8xrqtxBFE6W765TVmPmwd4wmqUt6Kc7T3bwAPKnC2ywnMY+UW+03eCBk7g9ep7XUZY+\\\\\\\/NLzxDKMAYYi+tICd51NkoIDeVekYJNxk7OxUxqtbjfbqzSUFTqR4BD7gkEQL+QwoVeQo6RF0dvml40j7SQMLz3gRPE5mqJKRBV6vBv\\\\\\\/X5+5JYxk7dNE9gYVOQktMqWsVFNB7uhvHMBpWckEuxfYUcfa1khV9bP0iBXOtEybxZ\\\\\\\/jD2ITzB2M38OdEvkLfPUJne2AnVwtYdSQqOs1qa4\\\\\\\/VJwPU1dM71fBeAcCv6GhGLP+AnIk3MYuo9eYHxM6dhIHZzC9HnI\\\\\\\/mg16fVsaH3R6M0pao8HCXb6bI9NflltuWdGRPIMw+I9\\\\\\\/kZyKpqA3WQlzPvUcl4gGeqw5ic6jDv+qn+wRX8UMpLXc07Z3VFiK992GXm7gUpfaOzJPyU2Oj1c5Lx22SJjXhkXItBOPHZju93VT4F1\\\\\\\/2VlV2Y5GiLxwVIgMzcLCYSRhF1aEO4Yuu7LnNKRcyhxK+VsN7aRN723pGoc31FkAZQahm4GxJxucvy9jFQQx\\\\\\\/MBwN3v8HeV92YLoQL6s\\\\\\\/xWAHv3JePW1zuHjV\\\\\\\/Pew8lM+xYpbp7KqVizD5TJ0\\\\\\\/6emv6ARNm2UzXFVsp3\\\\\\\/DLPMAJiTUVtnNc7wBJflRzerWvW1jVhfS+arnYrOx562gF0wC4GG\\\\\\\/heEzmwIJBBLzeKlGzjCdFRyU4\\\\\\\/IR+RIlVyTZAYbYdiRbWk\\\\\\\/6deH0j2IFqBOgXiNMgSkdMtKUt000wRtNyXfHJiaKKCp0xGUwjbFdZJs9czGDGwsmBZZ7qA8f18aZN9H96s\\\\\\\/WJhLjlgN8SVeWlECgdpbLk+B8Diyb48nbg9ilLLQ+B4wH7ZmJgEl53wr\\\\\\\/JmxiB4OQIpA8AC04PR8guPW1zZG7P3njL6FTJZqubM8YWSI5BDqcK9CP1p1lSoRHVrPYdQuSeufndVjBTlTRSEFX2ZTIhQX2h9nP9uY6iE7LlQ71e7oV3UFUspt4iKIHJoZXt9LC49i4N6gANLqN1TpD0nPCY227DhyD8Uj4UdPqxIIkhQTZQ6uLLqSW0dCsqC3GVvgHqYW\\\\\\\/zaonH4IvDTrP78BjAcyfj97394OT4EvaCp6ObG1hbUZk4bdiY\\\\\\\/ZtCIE22kS\\\\\\\/+JTRweAJ6v\\\\\\\/UYRV6rKghQ\\\\\\\/A3Yfr4lMpACGcqF9Biuc0Cd0KMFzxdBSGMRFmdz3zh3njt9wbzYqAvz5q3oW2hGlvnOH8\\\\\\\/O\\\\\\\/+Ob\\\\\\\/E\\\\\\\/QiR2E0PKxKu6SxhQQXwbto3vcda6ROxx0rznSTlsU\\\\\\\/Am4JztuLJKJOlbtnAax0ETS15eBp2AhlmK0EMgPLaLC+4dKxUQZXMVzvl9q45V6tLZ3v2+BzjBQmH7FWwFDH71RV+DisUXgIFBObZD\\\\\\\/ufDZypBW4XwYwegdcdfO+Es4et49tWyXCwsZYfR41osuh3rTjavm7kNq\\\\\\\/\\\\\\\/R06tdE252jSVg9n4S10OYUYZhdRnyC4ZwXJsytPe8ttUSblEHn5FhtQsBDAm0ctEo6zxtBmV1VWmwPkH9KvoQ3+Ntp4ptot4nprQ32W0moFIzKcjnpD+oLCb6psktgOh7yF1+EwR7FY1vJGG6U8fgiu2h9v0BJH2Ss2mya+DpFbNeaQYKrGuaA41XKNlEWUFwKWBe3G5kqoIDkqdZj5tCHuBJm0dsXuVJ3w8+d9dYMfhR2tD\\\\\\\/i3rKSK+O5Tr7gNA0iqeP3gmGTSG0tZKT4BKRplnqD5MiIltQlIhLuoUizRjBXuic2kClEiQqdARQDQVdmI44D8JxgdxI9jFduWYUINL1QWldGH+DVAGHSRgwEV6LCX9WEMl5GEXAE+500KLxtq4w\\\\\\\/9Uk\\\\\\\/IuaGYe0ysOVEdjTcKhMz7RTvR6wdczH0o+W8v2SFCFaqZENMabKD5B4ZyhdiQqVdSUkzXK+Hw+Kyu0N1EcADxhCkjiaITB2y06kyp88j+\\\\\\\/WdbQreVhf9d99KYGhyoIrYyoEoInigMjnbWYmq\\\\\\\/7z\\\\\\\/79pAd88F3nDEMluB7+HB+uXiZxRdPMAlm\\\\\\\/7VZV9b21\\\\\\\/KDkP\\\\\\\/OfT5M0iiE7rZoHVTXr+APkW+QvpDivW6ntYfQU53B9f82rIje+n8zX2RN9AL70AmkJ1m9yZM8u4kOgGhyOFQdX+MEIj2EfzYo5ElHQ0OJuZAW2KMVJ8VVc91q+Iw9K8dtWJU6\\\\\\\/wSngz7kt950SYQZSl2jOEwwSneuinBHH24wFRRKRlgVVpjGLvE+E07JFJlLRQaxA3m2ALLWwZW4EbiojRFsyP2bZJ4oPek4Wf\\\\\\\/f8FDbcmMvMIEcpE67ySD7TEiLpGnnXtx6JKezr9RVjgIqJmoUggNhrf7m2rp6ZkRL3Y3kMu3IuFf+Vhc0yrnUTHyf8b3v+0CwS3URaCgiRGNQQkTS4gAy255TuJJrYa\\\\\\\/dS69akBLgtqA7w7O1d2P3xUEQuLeA1rWwrXahXOP0uDsF7ASAsWJuAnL\\\\\\\/0mWZCJwuLi6hRDOTsPraP9oj\\\\\\\/J3hFduj6vsSMQC6Udam8im2rmqnIp63j\\\\\\\/JJw6sl2Mtb4hA4jcl9isW5xTUWEm0Ile5OYLPdDBA5j91lwLmNEC5E2LLNZXHD87Mck5uxXJ4slD5Idxb\\\\\\\/OCCo5CBCPJ6Q0H9pcD922am92TLF9BF3DQq4iqWwULfWNIv6yofdj9YKtCBFazgpuLa5frtkhqfP8ow8DFa6QYOwe8KjRkd4VYKwkfhS3DCYCpqvMdhh1E2A0iC1OlnkhPyyZ5rGxKniDY0Zv9Ud5v6uw8p1xwcZbBvVVA\\\\\\\/GNzKI0G35A7MDksOT8Gttx39Jrgajj7tLbr7NfVmNfOMbUMtPEzSP4IqUb98xfRS0bSer3XU5GVfVtx7CQS9gPm5YYia8i+wzP\\\\\\\/NKQa11rcRguPu5c+PBLvQhVZ\\\\\\\/wc\\\\\\\/r\\\\\\\/i7S3jo70elADDee9wM5tb6mdsn+wIbinZyR4qqjfRLO+xDouPXjYjofZNi0PtF3hSO1mESL6KmG9hH2m\\\\\\\/mkAWjqXTo0bB543SCaSJuMCtiOm+HGT+DaTlcMWy21HqWrGe8WV95qnzmdlJ+BiOquaKEp2A79hjjo3hznl3iCfcECkokQ96i37geDqAXlyUwG2yWrAyWF9DtPk1000z9mG+h93kH23AP5Po+4Io\\\\\\\/dsj3ImmbXu8YqUHK0SlLTreHldkIHloyMXMqyYliqULAQ4\\\\\\\/8iMir6lgl+juhdrmoby+2GfOPXcKXYJCBrdOiN2EHC5B0wn28VvM2lsy9ylgGeFrDpSD6TVqyXWYyL686ZwJ94JiGKgiaqGsUqEw3YZ\\\\\\\/3SUaiH1Nl96cNn6EHWa0Av0z9GwZtbUZXt1JawwUMf8TZrx9wCwAVvSw+o0KxpGtRPNFegDq7PEUfsFi6E9PEavpVkNPY5zAeGA9INrRpDtfhjs6GMzygQWYHE3tvrFJsn7Neg7Ih27qU\\\\\\\/D1UUOpnNRugNPsm5r1piPbN7XudFgfZMgZ4Nf4jF+ieWmiOYbjxnALSNJfCfxWZkR\\\\\\\/qJx2iqyxR\\\\\\\/auvTkoSn915i5UCo64RFLmrE890BgRacxZ94hPNJVfOu4DdC9vGoXkanuVSS2xn5lONLZzCQ3vK0wzLM9CK458nZrs\\\\\\\/VgIKtOSEo0yt9aCSZUKkwY0qFNH1GvWIYybX59Q5zqDAdCtNKdugNU4ZHp\\\\\\\/P1MhOuru\\\\\\\/wMI6q6HGeNgwYfEj98FtrXehoavOzvQpQ24Tl486jaYWVOSd9iXY5PnpdwguuW+v6k68Gd8\\\\\\\/aIU3mccM5WLg1U4d61HCsDiXR1GvpN6N6eqQngMLR9CnqXTpR6yyke2YOFaJuzhlo+uXbdI2I86NfNON\\\\\\\/ksv8JfmutnkWc1tQobYefc1MuYUmTyj+6CgtZIuM2V6bXRnyPWBvJvv43TwH+quRZDnxWs\\\\\\\/KfMSqGQ5eu6ev6vedb1WPCMOl5fC7ilG35GSOAGjHo0gJere8\\\\\\\/3Fms8rakIHhJ7yeyqym5nzUd3PmizInjSQqgtRMJEfC\\\\\\\/nb9rUkDnY3NZs\\\\\\\/viBfqs2muTBjSjUXwIhLINI5dSg8t\\\\\\\/w7cUYXxeXf\\\\\\\/uL+8G+lHlM3oZ88TwohuC7uTqQT9Cm45slpsYkJvRwMqWgOvDMfbfrx+B1JINImCE7kefw1jdNs0oI8U55ZO78eKORChYl17n25qr9Dpu0AO7gVSTVsTY3J1Faf74l8Runuo8\\\\\\\/X7iY0AEbIyo4qWGLNsKfua8o+qABYdMktZvbqhckabT8VRFr4gzIfFDDYEX3eKa9M1YAKtkUmsmLd0C36\\\\\\\/Yd7FI71npGxelbCgTxc+I+vR929Mpy+TDCHYJbkBbrJi6cWJgM+brhLVwPUWD4eLW4NFkNS0P2ED5pVEs3fLD77bN12UfJdZMd4XP1wcD+I0tjBDpFrTpLtitST0zGVxChpALvsZtIchsn4sChJjiaGtGMQDCNhx40\\\\\\\/U6XpUXTemrMzy63N0QPi6\\\\\\\/s8inpeiSpMBP7LQrjVB\\\\\\\/2hi3PyMjsIQU8D1cyoIWENfYwDj\\\\\\\/A2ceNjvKf51XirtAc3x2Ud4dwA4okjjyfXIiRGlEbBHiWVpysDbTKWe2cJ13nVZV+wz11eFNs0Nl2ZjTgP7ISPV6HwBWns9Br\\\\\\\/AEs9Jzc8v9FAAFXJ2vSdmvQKE6C3H2gwHvOu\\\\\\\/VgXJEl3m795bwthHmBhIm2ltnjnB3P5MjceAvaNuvToNaY8vHLwcxKuMef\\\\\\\/CVCEsz2makv\\\\\\\/e2F54l5hj+sVPt7D2PTSE0kM9SnzpWYFdDsW28EyEKSgV2SZJ6lHDI28\\\\\\\/QHhlin7ef953YVd7sMNInHg\\\\\\\/pScGy00qxpTrSmvlOrS0rEWbFt10cs1qbHfG3vB2fk++3XFamkj8lyr+\\\\\\\/m18Ef9GSCezCjqqjwORdNEqR3VDaJnDHteXhi1cGoQ2nbfAg66iHBxCcY\\\\\\\/wDb37hBxGzr5EGth5DhbLNwdUXEHjD3KmXbZHut212M5\\\\\\\/EEBL\\\\\\\/RITU9KNa8UQ0glWjjF4gOl3hkb8LcbcfbUhAxiGBlYaWi0JZSdiQgr8G8ETIzhtecf9SexdnIPDqlBo8julf6W81u2J0KykEMxnTt2RPLpzmP1b6rNhU0\\\\\\\/Zaf6QGNAIoGnEMvr+W7dCqIezG5Wqu1\\\\\\\/53J70fGF6tSgnKVyIeePZ6MaEHcbIiEW10ozJAkjrqgSoV2I8pF0K+2\\\\\\\/wpBHypfUXg2AKb6dK6iVBmwYJkmbnWGygFbbrIXXw1nzSFdoU5UdPZEPYeUuo0w1F\\\\\\\/Tx0MktJ7a4F5\\\\\\\/emVnTyLBySF\\\\\\\/UGbE6UE+fmbCQ6vS11\\\\\\\/MSN4jCpy2bS0mNanxSpxKVXktpwxkHpnescC3rVr+oz4O0oxbxhYY\\\\\\\/yHjF0hrSUXoJHw7LGIRDT4V30tynu8RaM+yDpjlgvtOSK84nQujxubRM70oKmOgiH6P2X\\\\\\\/gP3fy\\\\\\\/JYB50l87+ulPww8PMEoc4HwDotNQ+m7+YSmZ58WXnFEdKGnvi2nYR1wG6S2kgX+7HDNL6DNfpsxtCHKoXgPwxuoqCcA9C2ahJKq50biXB+o+Ub8MgPgZRAN1xVZc2qY278CmGcNIM97RdizRL3NWHs1ASQz95zFnwapNz1B\\\\\\\/+jo4niM50R7Twj\\\\\\\/YIQA+8k53XYFXYdvfpSdyhKYgelbTfFJsjSJQ7o3c6WlemPAfNHyhYLUYu6jqM7p2MUNv3MOHQiGx2r\\\\\\\/yVOpkVYq7\\\\\\\/ifOWqoHwBcHaG4mi2Ur+ArCfjVno+Zm+J+eHawPO0KSA3kg\\\\\\\/PpDcFtycZ\\\\\\\/ANYljb99Xlq0vfs2UA9gLJas237TVplTWDZUIYPKojBRTd94gr6TvFYrPuJEw3dlpiNNj5nt02Rifkf6usGImZa3MC4A\\\\\\\/2+fFDcwz25k9S7rBxTk2Dw9D72Q5k\\\\\\\/r1pb3nP0Xf\\\\\\\/7cKiys2GR+yOg7vpaIv3ciau3LwLv56EXPzd2Jj8VBVYUIlGbgHaVh\\\\\\\/1cEgF\\\\\\\/TEe8UyGm2u6EHrlhGglL8K4NWlO1+\\\\\\\/m5c5r4DIzew3sByAj3KNUJrRo49CBV5sH1qM4b\\\\\\\/dVeJaYwM3GHzdO8tAPFc0BSjF7eouINqNISZxOf5OeQraVBF284vbbe53Xrjgt3oX5PDbELQfOlP9wc0Ipo22QwJPPsjEziRzWP52uD3F\\\\\\\/Gj2Raz+HfLbe6\\\\\\\/dQZ1V5POy+1rAbvb2pxaH2ayrbMdIpz1KuZAGIZgNMDyJRfbiuXhzCXjZ3TQBWASBpcFJuzuiJHYIaS6o8cfPKHlljKqJyBBUohF1xGt1CCphDb3Q3Sn2nYBbYON3ISDrCLcD+cJ4\\\\\\\/i7ITcwMpVAXwcx62kMUZeyxLif5BQDxHrJnL7oOlKtT9oM9c7eJ+e+DWp\\\\\\\/21t4ZfsauYXK4bekbyam3Qhl91Z4J2wSjt9jyOS98Ib5Pae4+DHNenOz414DndZBrvzC7uozBdI7eFyXt8MJWGmfQBSz000hW2iQZYQYklGb4ENncu2vIpHvsjb6QWf1UNl29GP1UYz0mhd6ubBL3ox9i\\\\\\\/6zVVKX9tVaKp74XU+hMM1dwZUwkJfv6i2uN3x846CS4gfYCVa9z1QGYgvcVB16LTD9qJ4DEyQBE5IedR24QsinycPGPfzo5XDcAerCxnMVgm8gMO2LPXrfnfGFqY03Z3CVRv9YMhL32c0gfHaElUktCiR\\\\\\\/5ioRtosbsV35fTlv14ZuceQKoiIitP76eqCRosb5MA\\\\\\\/5qzFmd+aW8yaddjyJEB0X6yHkVu\\\\\\\/AppmlW1TghLH7H4Sf38wD4Y6a7BIuxxXO2XLbqwKBeju2VOmAj5YlHNsq2B9Gb4vIEqTD4pSlt1Hg4v0yDjub7h955iLJjB7DLSVFXph\\\\\\\/jtcXNwGGqVnTPg2YhYBLQyl6oxY\\\\\\\/5ARUM5Y0Z7i6flWXv52L4bvlmLLVfvdrrJBuawaCGvdpYlM6nCXCUtOLOPJ2dUemXr3Gv7O9c0Cx16LucM8ZF0DNF4YOWCdVGaa7nkjCZIbH6azmJSvKDn\\\\\\\/8yMeYYh2Gt+vwr4sjyZelDHaKCHdmm7DPKWw+XwZjpjifE5lbX4bRTbsDXKm5hOQ5ztiw8RrWD6bMgBpiYG0nFC8lMrT\\\\\\\/rawCi9g7ge5ERk3g\\\\\\\/DBAJJ2UDkclTfjbv8iCt8l709NMZSFNBok\\\\\\\/tu0rphav80Z3Gj8pbSA\\\\\\\/O6lqffwDtnh6ICmA9PkzUceQ814Vvtdy8qCmSmktik\\\\\\\/868Fj43n33q0FRkKhcXK3XiHUnylJzxBOOS+NpSNGhtIyhNxYWQNcIgAgBwMFesW3yNT7jJMkgOdeNjkCEhHEsLbdaW+tF0LzAo\\\\\\\/920FPNeVMg\\\\\\\/XI6v\\\\\\\/I+SGrsTH5mXqXJR2LdrU3mxkviEVDJJWuL9OduSLBYLrJQfoNUWsf6Ehseq+OL7gn2alPXelcZBK1FFcJoPOtMGSDrc7aCGjjqr52Ds9LEgynWMLIGrb9T0RAwZmFA8dEKOyRjEnI8Gyw3fEtR3l86rfrtG2TOr2rCjpKZbWbnc9haHZ3bF4MmDRIv\\\\\\\/hGE\\\\\\\/loOMZjrCL4TDihjhgepDVMGhxfp9Lkb05boGWrsIMF0NAxbi19kbSSWV9oq8mGPdD4wNGQB00V9Sj22hXRXVc48UtFsH40Y\\\\\\\/Zk5G1RjRPwQmlVPyCmfwWb\\\\\\\/YS+NF1Eg2OSx\\\\\\\/iQuvs5cJibnD4jiLAZEgntsBahH1uX1V3Agw+QytS4xGvVfaJelHYpiQ8GJFkJr\\\\\\\/9OeJjpljTSgEdsALwvOeJa8xs89dUgakfJS+TApsJhNE43i8mGnYx4IJdwX+rdeKX+e12PftfGC7BgAjg4ljDim4GPUAh\\\\\\\/0UsqshxBJGBwnGiCEsYJtzbcdTfJIU6Gt8oKZw4JQMt1dlSuFYuGKcvV47NwIwXYYN6+u47Sm8Rm9UFfkRRQ+Q\\\\\\\/pEKTUm1S7Q\\\\\\\/Iq5aIHkXlvOuh\\\\\\\/l0yG+0OrCjm+BcFO56Rvz6lVMLlFqYjbSR296276hQDQXPDj3ctvkZ4tiMcwM9OFkqjaA5QWFA2+IT+9HsqboCjQ+fYHCp9f7BjEu1S\\\\\\\/Ib2FvRNc\\\\\\\/i8iDSfRsu0yqPYVGaClMu1eyTXdMjeil1bx4w3glmuMKCZ7BuWNFAUZiT6iaP8AWcLXthzkpfszji+PVXau5X24OCv1vVlI0zk0Q5xvouOdvcemXGQIl16afhA70b2kIetWM8cmv5y9xtMS8uoXDoC+376WJtUuo5ruqevJ6A7byLCKbWzhRpR6BN8TTWD6PZJSibBMK5VCBDokZp4LIPK3gO3gnCA8swy5xKUM6yNm06dxKRyZow1XUbHrNd1ZBvGxBxBqZrS+2FkvayJS8i8\\\\\\\/XMf5K8Y2bCQzEg+GN\\\\\\\/1qGF+rAMtIiS+YFYW6zuOJ\\\\\\\/YqmKbmGT3R+8vKRNecllBkC9XjEfVrC3TSD0YrcaFz7aLBtRgu2TQiDbZVohHOm6f0P+CfWv4XZUGSIpLAfFk225B3O5O6s2ZdQHuLpxdbMyAf18AWn8x2qddKevL4c4miD4P\\\\\\\/md9MbIPrFaUqczfxufqIAA9oWKHhmBFZgjCsdTxy9xMLO4R+XnaSfgKLfWNB2Snnow7PbaBKuyg0hFenIQb0gYb\\\\\\\/\\\\\\\/7gihNAakayafCPBcqbbMHWwVSyjT6YjpatvTmuSuuZm5Wzm\\\\\\\/iTx3IKpB29eZrZ7\\\\\\\/w8SSdHoT1SnaL0hzoceJoCKcsyynYLGxfZu44lSZHEbo+KYOo1qYJaPFogCKyt+zOCxAJWngLvYsN+WSZeIojYpTaVPDfdTt3yp2ycbALvvmLB2zya3N4dpwyOgo2lh8n3wj21RTicIvLmtptQ6eqpTEvhD6RxHoutO1UIbJ9EEZPt+GoTjzbUddQVtSJdzvCtGm+MtNnI99BTG8dZ7IW4i8n1Fad\\\\\\\/XV+\\\\\\\/SMkmhk8bZ0ieOV8ULe71sy2IYfesU0qaKkjV4oT\\\\\\\/iKX0VfKako8TMqoDpUzVTmqcuD\\\\\\\/Nnzvj8y8n4FGieWaQiFwZk8W9OcYvLBe61+acMOUQ0uex46FuQRcFPl\\\\\\\/kueMDdYkLZAdXF3ndBEmRbuNwqLtJzhr1B8164hzgUXl0cX21cNiu+DUsadJZHc+lQwvakHczwrsWy1T8c9A0fROlOD8tbzA0hLRp9\\\\\\\/G\\\\\\\/rRjsRjp9i9de8TmjrhtP2QIqapByOUEXfmQESk3qAYXZd7pDSssQs5r\\\\\\\/8uK1sGWOxih7f9PrY+6AKQeIPWeFopAf34emDxkVp9Hbea2srmzop72sbRU+MuWPjNhinPH4cX+uj5JhelKZtuB+LCFiSqyRjH\\\\\\\/VxbP33idssbdAppKKvnXWpL7Gv5A\\\\\\\/h592fu+IZWm5eNx+8ImXZ2o9bsTnR22k7vlvzCfYCvE9m6dSuj2kxqTnzhQk4eueqLjU2pgxnBIhTlGNI9VzSdZzcBeJdVAH0er9ydEbPDy1fH0espPeNU1xC2FsNnoujTjDRer9BNfyDXR4s1wFuhtHbzRi\\\\\\\/yaJexASdVfeb8iLA+3A9E4qKeKCfJSAX6pQWptkOWAnblFaoUWXmZnjIiF0xBQLFJiYzVsjzARVdkAvaHmSFyY3OwbKjZamzgwY4SaZbeP9JpUqnx6Pazg4TeSK6Emzs8eCkXlGi8c0DjbQ52A+D0bwu87FQ0WKCIWCNADIwkYFirYLQAk+pKBbXwVpXla2fixK1Hgd9ngGLl2NGgQRkXY\\\\\\\/TizmpclI4lT6Li42c9PcLVZFQRGkbHYPaR9y2NkaNodex3MOOlW5TrVPi2xktnB6YAoG8WDgy\\\\\\\/Fe4P+z4S1akCSp1Eq73OP9hQMmJjPir3zrbk2ySIyCL7swr2JKEKDE7vqG8kSI1CJeO5bPkUAndG\\\\\\\/m4jWIoc+pXkU9fAC+10fevJqjMqtm7Y4qasc1rLXwfYb3iyas1w2v6R0RrtPGntkeIT4U+akb3OpCeaebncg7m6tEju+24STasCuH\\\\\\\/HAPUO5vOwpMo1eZXeMUtAVEIBL4xyoI521wcDNQEi8ATIITkpMco94tyOQqxbOxDwzmnlNRPmHr68+A3HAbqsFsFmwaw7NsimnbXkdGMWQ85h9iWqAAxxzoydatGepykrrQcQeCJ\\\\\\\/SfmYzIwFEdJLBPQCt1vLv1hnlfwX\\\\\\\/37nw8GoQWyJ2ilp9YV5lyl1KbwaI0yczgdg46BqiBulXXdTiL9YUb5KkLGCrtTXnjiUoq2KcFoZwpYwWSjigSx6yEiI72OOrWKH0EcB\\\\\\\/ws+Hr2Gb08d\\\\\\\/g8JMbqTI78zQ1Xsmmjfd667O+k4vj9\\\\\\\/j6uhO57TD14tzdmUCyHuJnbcpe0S8UtIfVksJ5uXT2KasMNoHt6Lci6vUL\\\\\\\/hiDnjEAGWTXjhnbZ3merLct1Ia6sRT8WKIHEPXz2U6j0UmIwKcvsjWgByg8xhW04v0JNko8zbZVNbyWDXmsK1zIz7pl3niM5jHtvvBEXvHOASbQP5EkQRNqHuCTfVfl7RlhrMdTPibfNPWmH10lPelmDdbr3vP4XdTHXjbm87AXzGOebcpRgPwld7cszKjE3rxiTxG8TckvI\\\\\\\/dfkw875vzCVcLSzH0MvpfkO9mQSnSpIslvWVltCjN00we4NxCKzxFPbXU8tTbNnn5AcqnlDmz3\\\\\\\/nDbHbAFDLt6wUH66tP1BJMLoyVV4a\\\\\\\/89Hdkr22OWD2Xt3sM45gcP2BwVdCS64LQyI+dzcGrzhnCL2frQEmtFP20D23INsMTylzq6BLs+q\\\\\\\/ZQOFNucgMJOy2ROANYc5G6hI7o81TL9wjo41Vx\\\\\\\/EN2CQJWpu2gnLpcGoqoOribUeSMFQMgSyPbGa3NJrVc2MQL37\\\\\\\/HIwhgU3OFF+zobmNJw4fKksxrwcwRVzNNYBdGHIV6F0BvQzEuDc6EjCI5jzn280UI2Nhz6Nv2I\\\\\\\/tbpiqInH4OtLh1H\\\\\\\/0CCCc+X8TgXTwyE7qiUFYWzEaQOD\\\\\\\/19vD+jd0Y677qN092hnE6NJHFH3FW\\\\\\\/dVhfwObV4sQPtuCuuPANqPfZ9mw+fu8nHvaG2ENYTvz\\\\\\\/G1fAKJHKRUxOaoU8\\\\\\\/iCPflB\\\\\\\/QK9b5v4inOuQ6WJeD+SkrChbLZYYOebImHmuMPjVHy1FZmwvSh6xCfbr5gd9qBOIDc4mhQJpiZBp\\\\\\\/9SBIiUFRsJUD7L9Q9pcMF8bamzupWelWSkBUpMDDwBrnBuDQx53uarxRpIEQlSGTOFfAuRrZedZ72nGJFZ1+iBgg8jK9S396rcZzOGWY9gxuw62PN2qURHRURw9dF7bbnDohYZY5531jP+\\\\\\\/YI6r0m0gGCA6pmmSYTvZgIlFdsJGNAvk8YVEei8rZnvy4ogk2RbodA\\\\\\\/H8ON\\\\\\\/ABefH3SuOfjJkEjWqr5uUFcpYt64mBZCiqCmh\\\\\\\/\\\\\\\/sF8yKyEKhZZ7DpcDetPKeN+z8XcrNTala\\\\\\\/xaOKs28oIAYKI4yzEq1e6F07SoN6\\\\\\\/e0IC2HsyzH64M3rFqjXUmLXnBUusAyA4vYTIdho1pc4E9gv0NiRy6Phm264XD5jRsEbh9EiKkl6ciC65ok9Es6OV6YKokiB4IenAwWKLN3BAIjvnpbH1ePggrcyC0S+\\\\\\\/vKzsCqzSxqd0eo\\\\\\\/TOp48lxH1ftTEfpf45kfoiGiFFYI6GGLO1bRa9yo2dfGlJlSeWwLg0Djg5ufiRm\\\\\\\/sVafgZEAgWYlytzJgaiZ+NUizuwXwv6s52MMygW9iAmiA9QVl+pkHuDDlaa1i\\\\\\\/vDKTCn8vVuHb6h6Ep0A6iOeibWEJ4yrus3n1xA7Oj8HNNUuiTIdnutQE+XeridQKeQar8Eo5N1bNznzytS7dgqk3XB0AP7OUln+MvCKZkHhL+notWLqhwkplSZdicxJ3Sjb8n8JCtJpXPA4ZzK1dVPUSuyWvk4cjvDqSJZa9iC+pOE1bXjhVeM9vH7H5PGLvA1o33f0FKqml3vfWu5mOopQGNB0syeEFjdekMamOiYnmX+VHsyzaMGd\\\\\\\/GUCwrhiH8pRwmnO7Jz+REHYpsihXKBqhytb+2P1zrDQOKpqY7KfK6mObYoB3XvopE5cbcS3s2FqkXMylI8OOXaQkPKCyVBCHOCIUJBuwVIbKvNgMYW41Zr0lujEhIkcCA2bVGYl5OaZtKTK3pWL23ZBlWkWNSDgxwfb+nfLYrTt\\\\\\\/nHsBuOoVVeoSJcnCkoVojPlbFLITK3xAR\\\\\\\/h9POHIBr1ppJizvA6G5f3HhxeYrZlGPWttf0MW3ndYCBnqUwIIbwi9SHyHa1HOdeuiGVGb28Vp\\\\\\\/im5rMFOkZiraQACZdgFnIV9IQuu7N0PZbxtZ+V+ZtvmKekT+qqFpLrsz6WWvzkgp5shbzQVIxZhE1otTPhGD+GpbBVqF\\\\\\\/iewb9XgIE3DHhb90Oowd966K5gmkGyHZt+c8PUA9a7W6VMRgp96j+bqI9+xUE31YeRiKAbX6d7g4VbIPa1KyMqkD6Ys6vRG6BoTcMaElXdEY2x1Nz7H+GLu0QDz7Hgc7qHDN1xEO\\\\\\\/DiUZps+2YVKwxk8qLnKvrPC09LNom2dCo\\\\\\\/R1OKAjCS9wCLHYTa+So\\\\\\\/56NQjYbpH3gTLZyNJuByTQ1IHzpCwK8Gk7mRJUflSsIjPfolR6WPeg6EMhk9RVA3928EVctAa0KgVxemyT2kRFU17JL3EBqvlsiUBm6wy4y7Qtr+G21ggY1Zk4iGDODzx916xUE2hdISkASqVPobknwxK74aDQIfUnALlQMzhHOyo9hmkdmB3UzelrqCO7t5iG\\\\\\\/NR+7YEsW8CK5ThHFzAhyNU4o6VGkjy6D\\\\\\\/Gca1pP4Z3V4alkAGs77rRUNMI+P846JCLFxicPJWWXcwht7IqP5V8crQmxPcjFOzkgxtRggTmCWSpeGbEegOgoJegURZ2jUXLSeA3vwZ5l42F51C0LzQ3qUQaO50nEKncz5Pjrjf7MsFq0gFo3gQw99h5EpNNJQ\\\\\\\/btLQX11Qb4TzDWiAqZjNmVVCWy7\\\\\\\/1L+L7fY6XmMO19rW9OxmcP8sN\\\\\\\/sStrEg1OZ\\\\\\\/FDmPi4bF1azNjsXBn1dd9DEZBSUYGG8VqXhYJHQ3RqLiFxVw1\\\\\\\/EIL2k6TlpjT8XkL6fg6X6XpvhT9\\\\\\\/edL3uT1f9c6IAgFJ7p8YypqnOWl2RIRiU1HUBxCMVgZ7dAgpf2AdBt0S\\\\\\\/HoRL+DWLlVdkLn9LxHcTYhvnV1CT2XDMOOWQ0zB6v89HaV8QltWGlaBeTwHz94brpZJSNIj03yysCy4+vQ2QqqG7GEZV1dvdDB50OnQfv5JXkAmJZJDLSUKDcsgteFbBPBI5g6em4H2HeKWsa8kQ0zj3eKjBeaqBqXtKXYFKJfNsy1jUJPikmy8KQE80jikkMchGdpX9bmOf0yCZsU5mgXBSw59gnCbsmOmrcaUWSBSTnTRT6cWWnFu8dxeC0BCb\\\\\\\/6jnzwkpwuoTWWrq4crtDXEzXJ9wpA7N6g5HgyC+YRvVfAaPrL6F\\\\\\\/PcUlDG4bKIdbU79pfrfPX5Zj3oLcec3jhPShB9cNO\\\\\\\/BveYldC7SAxDHG3XXunwAEjniw8a9hoqVYK3UgquixJjljMXDLZChRt4FG73L8Ry+pNYYUcbDOpHLvQiUaMrvVMuV+8ABnK1wJoH1azLHCvjlCh8ZNIsntjvTypeNYMsox6wyeVU4JsUAVoA7N\\\\\\\/GAW3bQA14TUUi1lWFJW9Uv3IRHj29SW2swjgg4F8AUtyKEnkLtV4cHE2fneKfZ8\\\\\\\/DdazYEFDif8xqruzgk1L+VMZ\\\\\\\/y0bKzpabB25LWQ\\\\\\\/e+GLbTZZvy2D0skdxEN8GnLg17\\\\\\\/1lNEXMntTlgjhr3dbt\\\\\\\/2gvMTyqaUxxBK2NNhtEdqDwHzXNsBD84F5CveAtsp4qcqvH+AL2VMwwy\\\\\\\/zI1tTUav\\\\\\\/yOcQmKpH56i6FcaH8L9W3Wny\\\\\\\/sEUc0vpQ1pj9cAUSxBNoKWVOn3us3L6t2RDl\\\\\\\/AjX5tegVF6PaHbUl9hAaiZ7jpQGzPKHp93\\\\\\\/syNxR\\\\\\\/ynUvEmeExrmcDKKE4lF3hrKxmfQn9eaKIZ0PfssQHVDl2GkLOXXNQ04j9rT6uW95\\\\\\\/VCF\\\\\\\/8oOiiPX8pyT0ufE37dCQU5ubYLmMEPo8ORbAYu2SBrboxr1dv68R\\\\\\\/mmls2gDrAgZWHKvhEAVZIZrNqJSb8EyGrbCU7nshukzY\\\\\\\/MvSynOhCtjKvU7qu4MDjwYke7wfFKD14wiKkvXHQHZ7akaG3K20n59Nz39Guz0dC8WCWeZKIv05fmQaoICEdtzP9SguWU6\\\\\\\/SB+06VB3cQWQ1utxCBsGf3gvHd0edVfVjo0pMioyQGYz+1s+6cSabt2aScAnWxD\\\\\\\/cELEc1muycuYU1WgsGlQCDrO+nUkw+LqUal+jysS6qw8VClCLhtBU6KsKFqPA\\\\\\\/lUtOsUqwcEKJV4hEj32YjZ98AvOKU7nTqXhwHv2f72ISdjG5j7kobF1bBwFkEznyfiZKH26rdoxnRLma8m6uCyPyx4W\\\\\\\/Xlym2paZojhoOy20L9nVJQHPF+yAc+LVxlkeCp1AHGiljiZwLd0\\\\\\\/y6ozeEoeIvy\\\\\\\/k290zG3NUl\\\\\\\/neV77sYGLXyOCUn7ZqZZmtOvGiCKsiWa4hjoNLEIStA4oB\\\\\\\/V9ETqhg9h+sF945V5JytLJSS4vqX7Irwl4\\\\\\\/gDJ\\\\\\\/FYexHw1+Rxmpzv77Jh2ITXRI4vmIosC24G6AhcCvLQm+WzF0B0K9UjrrjJwz89RoVvKNj6z1Wn+cH4qChSzDa16wS9gCJ3M4NRYcSYliCTVWuTBPLxnzwFL5J8D5\\\\\\\/PaSxh1d7A3un0c9+5bnZQ20o06LJynOWbDKfttme1b8gcnrhjkP54ENxunkrg0qMFOWcX+kUoNo9S+ReME7wKbpMLF8U0ediBkXCKjP1uh2WmPA+RrVjBi5s7wHTXraDhu2fnSNDeVLR\\\\\\\/dhF9KesCe+qwPGjV9xZEWkZF45QKexEDmj11tqJf6Oa9Cd4f4kPkMNvGWvt5LHioxTh\\\\\\\/ij6TU7UpNmFn0bcG4cGMu5z\\\\\\\/fxKAH4ZpcUU0uHxtemdwHP6MTVgrVRXO0n\\\\\\\/iwuifRiCpk1v1mcDNsUBZHbYxNrQEM3QvkRkqpdgUv4szgF2zMW68kEe0VsVi6K1xa5NMsd\\\\\\\/+SKmvkiPhmi8YGwhnnx+vEt9PlmR+cn3y59N2nQSo\\\\\\\/hbnkk4lu2q\\\\\\\/lP3dIOg5NBjy7OarDCgkQomb3+L2FOtPtW9qkg8QkITgY84MBx0EpqsF7CUfnm9p8MN+m9dx\\\\\\\/pfyoweuXOQdDgehA+D6WCr7OukL+hwbTrxW7qsjT0IwfRenWnhbNUjl5IUwid7RxO7PdBfHR0BRTlk4YX\\\\\\\/9LRMCiWbqUDyYpMb1HtHWPG8Wm77Fu4Ul5iFgHUX6wR9DlG8\\\\\\\/aneZ2M3hyC9uQhj8sDhTc7C39aVuQg9FPoviggD0D5B5dVGHQkcPICGuwCnSKzeGG4\\\\\\\/1L5J\\\\\\\/1tpYb7f007+DMIFClnm5aWay9z5dCZGhuDD2PfTSnXdsTtzn346+lWNcbB0xtPNf7JySKaDAs9qjaGDFib9rm7T6S9j22DhZA7J0Mmk+X+gKxlq2O+e6g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Shb0LlOboPg3RWHvBjg63g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002267fcf0432c4e5529c7dc072dd2b125e37881f652b154ea8f709fea1ab172210a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cchJFNJ6F\\\\\\\/Zwfu5cW15o\\\\\\\/gzHZn8Gt7FJmYcS82pf\\\\\\\/ormjkC1Uy1cUl2HXHtdsG6V7jkcpYMOkQA+7OGKU1fNkXgCGGMyb6ewAYxLIJK+Vju1aFtGLGa4CNYaCEfHDTs+9UINQPJQn1w4iDgk7dWkT9intb92ymvLV6EDtd2TuLZwh7WGBWl6JIlKOk2MbjWK95sa3OdIbbL37u1gnjaPNkj9cVw1V6J5RxKj0W9XQaQVglZ073TSju+sV8ghMopHbiKV024ZzliCV3m2Aloi\\\\\\\/udWT32ZTguXSUcYHAJazZpXFnWOrujSDFttwuS1aXSLbAHMwI2c1NwhLcu9daRPX3rPhWDQQbozuqA9VLBDjWLtE4O2wAIDyQ5A0oG9BuXbwb6AiOi9GdigP06A4UWdqehJnkpqXwrs4\\\\\\\/4HyTRDX5JFgxAlZ4+aqmAalUNasuIJKGL\\\\\\\/v3qkQPBO+w+AuHhr1LZoJ1ugHG8\\\\\\\/KWByoxVT3EMqrNXVjJI4YxbXXalnkU8WQjK\\\\\\\/Ba2qmayfxlRzPg2Y658DfupI2XWZV+9b+2IiPc+0wEe9GFggOi6SZoQSKUGNyWx6\\\\\\\/QkOz2Uka5WDYNP1VfCE24bzKCzovpLVHXfcYTIcO3R56nWpTSWayuug6NXF49D0wafTHhJ0XEiDcATzW2b4H3BAJUvBTJ6G4zYbIANUGMKw9XGXMXqDpwspWRHeXtQOi5WFbY37IOP5+7FUzdPMwx9pDH4M0A50C2xVzz4CYk9MrJPIHkK3vdXvc4wvg9oKfIvk8NdbE3ZFjUnBDzkUymMBA3Uvq5OYt8SPDaJhd0SUWa7mCNkjRqTp+pfOXjPRKHRz\\\\\\\/ajyKQLHJMKLXbjTdPNTzk05mUyjMYHJZZ4YC31Xrh+nUSJmnkiPwu6MAcCjxHKhZSH6IDLFe7azp\\\\\\\/pl\\\\\\\/ZeiQ79pD8qff6UtF3wtuz3l2wSoQADrCyZgNCugOH2okG99uhl5OoStAb6kDxTs0fMPaLuFr8Sldmlq\\\\\\\/mRaNfEFBxgO8XRTSsdRY91mZI5GFPzqqz1wzkPAoSMcT54ZgHok1hx927CILBa4zVxIOCMf3qHHp9RFZz0FSPvFghzQrxWn8tVuogWoOtUjirI5OK4gR3vcgzpg\\\\\\\/\\\\\\\/oE1ufSo5WhVaOKb8EXvrq1meOy6bvzTeQ5uTFT\\\\\\\/MN5W91G\\\\\\\/pqM3xUBllPuGPYjghK4ik3zVTM5b5Bqnr0O5kFxSX40oh1XnoMo62WSxzE0hWqaLeImZzTi4F3r4DXnu6Ud1vijXkZAtvAqi5hUum6FgjwIbPAW9jWROKECy2Evg2b7WU8X8MQRRnP5yoxxoUucPBbGk2A6vtophdCqgx8NK0qj+luqg8Bj0EGBETAdirpOJpcxdcHFERS4go2LUIx4NFXEtALDuzXa2e+rGlYIU8QGv093cg3Q8OVWdKf1CHNHrtA2fdfLH1mdghPcE3ow2+aUrI2cAKier64vxLyNFLq0uWHdIuEIQeKcbLJHpRwojwuWx6jJ2H8uN\\\\\\\/etp8Fq+hczOMorabIrtxgb4u9nZ1GeDYgYJsRfG1CJ\\\\\\\/YELIXpCshMWHFWfgiqtTh\\\\\\\/CNj3GU\\\\\\\/en3y27IGMw0kiGSX8LK2xs+1HWg6DhTlN0zTm5FRBooJWixcunm5i+fYZm536lj2wvVjephWWWDphRB\\\\\\\/a+wHFrt7q28s+BLk0yxw1nJqXiBX3xfhgwTh+uEtMuSrTKKUCFvPRO6pLXETXyIWF\\\\\\\/kL0\\\\\\\/2nsJ3BdULcWFRNl3OJ0HzU92MNHyU4Bakz5ORnH2c6XiqhK4AByNo9GrtROWVibIECSUvoADtUuMejLe9mQUPSMPOdJxuYRGvpfMymobIhUXzME4r9w+ya6r9+BwPJqrUK41zHs92s\\\\\\\/C58k+MNZ9HoAAJJQhNyI2crUSC+HsbwXXBi7Isw1SSD8LSXM\\\\\\\/g1ReNE3yoOe6IbgGjmsuhfmYtlYXgTynwoBdT1AtFA6su4YxqDDTMXjwMBXAtoUkMvT2+C6sCfvoIzW5cU150\\\\\\\/LCtA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022le+fAW4zRjQY+QAx3qB\\\\\\\/7Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225022ede5231c848b55ec7b2ffadb684fd919a3806efab1ed05c50a1a3df29949\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488298 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/707" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-J9SUib7dpbAqTGTVNyDXmlEO2AjbhLnsf43ct5hlxNs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/707" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ldYsIhLPs7VNcgsLNax5mSHJ9a8aG46UGS3SzxUGobY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/695" title="Genetic Identification of Separase Regulators in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/719" title="Genomic Prediction Accuracy for Resistance Against Piscirickettsia salmonis in Farmed Rainbow Trout" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488298" id="node488298" data-pisa="ggg;8/2/707" data-pisa-master="ggg;g3.117.300199" data-apath="/ggg/8/2/707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4456-2358" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Thorwarth</span>, <span class="highwire-citation-author" data-delta="1">Eltohamy A. A. Yousef</span> and <span class="highwire-citation-author" data-delta="2">Karl J. Schmid</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">707-718;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300199 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488298" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patrick Thorwarth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BThorwarth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thorwarth%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BThorwarth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4456-2358" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Thorwarth</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eltohamy A. A. Yousef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, Faculty of Agriculture, University of Suez Canal, Ismailia 41522, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eltohamy%2BA.%2BA.%2BYousef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yousef%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEltohamy%2BA.%2BA.%2BYousef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karl J. Schmid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BJ.%2BSchmid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmid%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BJ.%2BSchmid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Karl.Schmid@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">Karl.Schmid@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/707" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488298" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/707.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488298" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/707.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488298" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/707.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488298" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/707.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488298" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488298/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genetic resources are an important source of genetic variation for plant breeding. Genome-wide association studies (GWAS) and genomic prediction greatly facilitate the analysis and utilization of useful genetic diversity for improving complex phenotypic traits in crop plants. We explored the potential of GWAS and genomic prediction for improving curd-related traits in cauliflower (<em>Brassica oleracea</em> var. <em>botrytis</em>) by combining 174 randomly selected cauliflower gene bank accessions from two different gene banks. The collection was genotyped with genotyping-by-sequencing (GBS) and phenotyped for six curd-related traits at two locations and three growing seasons. A GWAS analysis based on 120,693 single-nucleotide polymorphisms identified a total of 24 significant associations for curd-related traits. The potential for genomic prediction was assessed with a genomic best linear unbiased prediction model and BayesB. Prediction abilities ranged from 0.10 to 0.66 for different traits and did not differ between prediction methods. Imputation of missing genotypes only slightly improved prediction ability. Our results demonstrate that GWAS and genomic prediction in combination with GBS and phenotyping of highly heritable traits can be used to identify useful quantitative trait loci and genotypes among genetically diverse gene bank material for subsequent utilization as genetic resources in cauliflower breeding.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cauliflower" class="hw-term hw-article-keyword hw-article-keyword-cauliflower" rel="nofollow">cauliflower</a></li><li class="kwd"><a href="/keyword/gene-bank" class="hw-term hw-article-keyword hw-article-keyword-gene-bank" rel="nofollow">gene bank</a></li><li class="kwd"><a href="/keyword/genome-wide-association-study" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study" rel="nofollow">genome-wide association study</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">Wild ancestors, landraces, old varieties, and breeding stocks of crop plants that are preserved in <em>ex situ</em> gene banks are an important resource of genetic diversity for introducing favorable alleles into modern cultivars. Adaptation to unfavorable environmental conditions, disease resistance, and nutritional value are traits for which new alleles can be found in genetic resources (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hawkes 1991</a>). The improvement of quantitative traits such as yield with genetic resources is more challenging because of deleterious alleles and missing adaptation to modern agricultural practices. However, new improvement strategies may be used for such traits (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Longin and Reif 2014</a>). Molecular breeding methods such as association mapping and genomic selection combined with current sequencing technology offer new possibilities for an efficient utilization of genetic resources (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tester and Langridge 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bevan <em>et al.</em> 2017</a>).</p><p id="p-4">Cauliflower (<em>Brassica oleracea</em> var. <em>botrytis</em>, 2<em>n</em> = 2× = 18) is an important vegetable crop, owing to its high nutritional value (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Picchi <em>et al.</em> 2012</a>). With an annual production area of ∼1.4 million ha and 24 million tons harvested in 2014 (<a href="http://faostat.fao.org/">http://faostat.fao.org/</a>), cauliflower is of great economic importance, particularly in Asia. The most important trait and yield determinant is the cauliflower curd (edible, often white inflorescence meristem; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Li <em>et al.</em> 2017</a>), but response to vernalization is also an important agronomic trait, and genotypes with obligate and facultative vernalization requirements have been identified (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Matschegewski <em>et al.</em> 2015</a>).</p><p id="p-5">Genome-wide association studies (GWAS) have been carried out in major cereal crops, including maize (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2012</a>), rice (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Norton <em>et al.</em> 2014</a>), barley (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gawenda <em>et al.</em> 2015</a>), and wheat (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Edae <em>et al.</em> 2014</a>). In <em>Brassicaceae</em>, GWAS have mainly been conducted in rapeseed (<em>Brassica napus</em>) to dissect the genetic basis of disease resistance (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jestin <em>et al.</em> 2011</a>), seed oil content and quality (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Zou <em>et al.</em> 2010</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Rezaeizad <em>et al.</em> 2011</a>), seed weight and quality (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2014</a>), seed glucosinolate content (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hasan <em>et al.</em> 2008</a>), and several morphological and phenological traits (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cai <em>et al.</em> 2014</a>). Recently, GWAS was used in cauliflower to identify flowering-related quantitative trait loci (QTL) (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Matschegewski <em>et al.</em> 2015</a>), which are useful for breeding varieties adapted to different temperature regimes. For example, expression analysis identified a homolog of the <em>Arabidopsis thaliana CDAG1</em> (<em>Curd Development Associated Gene 1</em>) gene in cauliflower, whose overexpression increased curd yield (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Li <em>et al.</em> 2017</a>).</p><p id="p-6">In addition to genetic mapping, genomic selection (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Meuwissen <em>et al.</em> 2001</a>) has become an important method in plant breeding (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schmid and Thorwarth 2014</a>). Genomic selection is based on the prediction of breeding values based on marker information alone. A training population with genotypic and phenotypic information is used to estimate marker effects with a statistical model, which is then applied to calculate the breeding values of the potential selection candidates in the breeding population (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Heffner <em>et al.</em> 2009</a>). Cross-validation allows selection of the best model for a certain population, trait, and genetic architecture (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2010</a>). Genomic selection provides several benefits in both animal and plant breeding (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jannink <em>et al.</em> 2010</a>), including more rapid breeding cycles and fewer field trials, leading to increased genetic gain per time unit at a lower cost (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Schaeffer 2006</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">König <em>et al.</em> 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Heffner <em>et al.</em> 2010</a>). Genomic prediction has been successfully applied in wheat (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Poland <em>et al.</em> 2012b</a>), maize (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Crossa <em>et al.</em> 2013</a>), and soybean breeding (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jarquín <em>et al.</em> 2014</a>). In <em>Brassicaceae</em>, one study investigated the potential of genomic selection in rapeseed (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Würschum <em>et al.</em> 2014</a>) and concluded that it is a promising method for rapeseed breeding.</p><p id="p-7">Advances in sequencing technology allow the detection of single-nucleotide polymorphisms (SNPs) from large and diverse germplasm collections (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Crossa <em>et al.</em> 2013</a>). Genotyping-by-sequencing (GBS) generates tens of thousands of molecular markers at low cost (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Poland <em>et al.</em> 2012b</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sonah <em>et al.</em> 2013</a>) and is an efficient tool for plant genetics and breeding (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Poland and Rife 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fu <em>et al.</em> 2014</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Tardivel <em>et al.</em> 2014</a>). It has been successfully used for genomic selection in wheat, maize, and soybean (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Poland <em>et al.</em> 2012a</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Crossa <em>et al.</em> 2013</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Jarquín <em>et al.</em> 2014</a>), and in GWAS of morphological traits and flavonoid pigmentation in maize and sorghum, respectively (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Romay <em>et al.</em> 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Morris <em>et al.</em> 2013</a>). Recently, <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zhao <em>et al.</em> (2016)</a> used specific-locus amplified fragment sequencing to create a genetic map of cauliflower. One disadvantage of GBS is a high proportion of missing data, which may be alleviated by genotype imputation (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Poland and Rife 2012</a>), although the value of imputation for association mapping and genomic prediction is debated (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Marchini and Howie 2010</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rutkoski <em>et al.</em> 2013</a>).</p><p id="p-8">In this study, we combined GBS with a phenotypic analysis of cauliflower genetic resources to investigate the potential of GWAS for the identification and genomic prediction for the selection of useful genetic variation in cauliflower breeding. More specifically, the main objectives of this study were (1) to identify SNP markers that are associated with phenotypic variation in curd-related traits using GWAS, (2) to quantify the predictive ability of genomic prediction models for curd-related traits in diverse cauliflower genotypes obtained from <em>ex situ</em> gene banks, and (3) to evaluate the effect of data imputation on association mapping and genomic prediction. We show that GWAS identifies genomic regions harboring potentially useful variation and that genetic resources are suitable for genomic prediction of phenotypic variation in highly heritable traits.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and phenotyping</h3><p id="p-9">A total of 192 cauliflower accessions representing a wide range of morphological diversity and geographical origins were obtained from the gene banks of the United States Department of Agriculture (USDA) and from the Leibniz-Institut für Pflanzengenetik und Kulturpflanzenforschung (IPK) Gatersleben, Germany. Detailed information about accessions is given in Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>. All accessions were phenotyped for curd-related traits in replicated field trials at two locations (experimental stations: Heidfeldhof and Kleinhohenheim in Stuttgart, Germany), for three successive growing seasons (June 2011, April 2012, and August 2012). The field experiment was conducted in randomized complete block design with two replications (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Yousef <em>et al.</em> 2015</a>). Five ripened curds were harvested from each plot and used to measure six traits that reflect various aspects of curd development and morphology according to <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lan and Paterson (2000)</a>.</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-10">Curd width (cm): width of the curd.</p></li><li id="list-item-2"><p id="p-11">Cluster width (cm): width of the largest floral cluster.</p></li><li id="list-item-3"><p id="p-12">Number of branches: number of branches within the curd that originated from the main stem.</p></li><li id="list-item-4"><p id="p-13">Apical shoot length (cm): stem length from the apical meristem to where the closest first-rank branch originated from the main stem.</p></li><li id="list-item-5"><p id="p-14">Nearest branch length (cm): length of the branch that is nearest to the apical meristem.</p></li><li id="list-item-6"><p id="p-15">Days to budding: number of days from planting to appearance of the first floral bud.</p></li></ol></div><div id="sec-3" class="subsection"><h3>Genotyping and marker imputation</h3><p id="p-16">We genotyped the material with the original GBS protocol using the <em>ApekI</em> restriction enzyme (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Elshire <em>et al.</em> 2011</a>). The sequencing and bioinformatic analyses of our material are described in <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Yousef <em>et al.</em> (2017)</a>. Briefly, 192 genotypes were sequenced as two sequencing libraries of 96 individuals on an Illumina HiSeq 1000, using a set of in-house scripts and public sequence analysis tools including bwa (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li and Durbin 2009</a>) and FastQC (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). Eighteen genotypes with &lt;300,000 reads were excluded from further analyses, which resulted in a total sample of 174 accessions for further analyses. The preprocessed reads were aligned to the genome of <em>B. oleracea</em> sp. <em>capitata</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Liu <em>et al.</em> 2014</a>) using bwa. SNP calling was performed with SAMtools (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Li and Durbin 2009</a>), bcfutils, vcfutils, and custom Python scripts. The vcf file was parsed to retain only SNP positions with a coverage of ≥30, and ≥10 reads confirming the variant nucleotide. In the end, 120,693 SNPs were detected with 19.02–76.73% of missing values per genotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS2.csv">File S2</a>).</p><p id="p-17">We imputed missing genotypes with fastPHASE (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Scheet and Stephens 2006</a>) and BEAGLE (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Browning and Browning 2007</a>); both use a hidden Markov model to cluster haplotypes but they differ in the underlying model. The fastPHASE method uses an expectation–maximization algorithm for parameter estimation and fixes the number of haplotype clusters in the model. By contrast, BEAGLE uses empirical frequencies and allows the cluster number to be changed at each locus for a better fit to the localized linkage disequilibrium (LD) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pei <em>et al.</em> 2008</a>). We used fastPHASE as the main imputation method because we expected it to perform better than BEAGLE with our data set, which is characterized by a low LD, small sample size, and high marker density. The advantages of imputation methods under different scenarios were described by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Browning and Browning (2011)</a>. We included BEAGLE for comparison for some analyses, but used only fastPHASE-imputed data for the GWAS to keep the number of analyses manageable. SNP markers with minor allele frequency (MAF) <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-1.gif"/></noscript></span></span> and any missing values were excluded from further analyses, which resulted in a total of 675 markers (unimputed data set). Next, SNP alleles were imputed with fastPHASE (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Scheet and Stephens 2006</a>) and markers with a MAF <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-2.gif"/></noscript></span></span> were excluded, which resulted in a total of 64,372 SNPs (imputed data set with fastPHASE). Genomic prediction was conducted with a third data set, in which SNP alleles were imputed with BEAGLE 4 (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Browning and Browning 2007</a>) and markers with a MAF <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-3.gif"/></noscript></span></span> were excluded, which resulted in a total of 62,566 SNPs (imputed data set with BEAGLE).</p></div><div id="sec-4" class="subsection"><h3>Analysis of phenotypic variation</h3><p id="p-18">The effects of the genotype and environment (environment was treated as the combination of location and season) and their interactions with phenotypic variation were evaluated using analysis of variance with the aov function of R (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">R Core Team 2015</a>). Details of the phenotypic data analysis are described in <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Yousef <em>et al.</em> (2015)</a>. A mixed-effects model was fitted using restricted maximum likelihood (REML) with the lmer function from the R package lme4 (version 1.0-5) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bates <em>et al.</em> 2014</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-5.gif"/></noscript></span></span> are the adjusted means of the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-6.gif"/></noscript></span></span> genotype in the <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-7.gif"/></noscript></span></span> environment, <em>m</em> is the overall mean, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-8.gif"/></noscript></span></span> is the effect of the <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-9.gif"/></noscript></span></span> genotype, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-10.gif"/></noscript></span></span> is the effect of the <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-11.gif"/></noscript></span></span> environment, and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-12.gif"/></noscript></span></span> the genotype × environment interaction. All effects were considered as random, except the intercept, which was treated as a fixed effect. Variance components of this model were used to calculate broad sense heritability for each trait according to <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nyquist and Baker (1991)</a> as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-14.gif"/></noscript></span></span> is the broad sense heritability, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-15.gif"/></noscript></span></span> is the genetic variance, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-16.gif"/></noscript></span></span> is the genotype-by-environment variance, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-17.gif"/></noscript></span></span> is the error variance, <em>r</em> is the number of replications, and <em>e</em> is the number of environments. Best linear unbiased predictors (BLUPs) for the genotypic effects were extracted from model (1) and used to calculate the genetic correlation (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-18.gif"/></noscript></span></span>) among all traits. The genetic and phenotypic correlation coefficients are based on the Pearson correlation coefficient.</p></div><div id="sec-5" class="subsection"><h3>Population structure and LD</h3><p id="p-19">We used a discriminant analysis of principle components (DAPC) to infer clusters of genetically related individuals by using a k-means algorithm as implemented by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jombart and Ahmed (2011)</a>. LD between adjacent markers was calculated and the LD decay over distance for each chromosome was assessed. To identify differences in LD levels between the complete sample and the clusters identified by k-means, we used PLINK (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Purcell <em>et al.</em> 2007</a>) to calculate LD as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-20.gif"/></noscript></span></span> is the frequency of haplotypes with allele <em>a</em> at one locus and allele <em>b</em> at the other locus (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">VanLiere and Rosenberg 2008</a>). The extent of background LD was estimated as the correlation of the 95% percentiles of all pairwise markers between chromosomes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Breseghello and Sorrells 2006</a>). Additionally, we analyzed the persistence of linkage phase between DAPC-inferred clusters and the whole sample to validate whether a marker effect estimated in one cluster will contribute to the prediction ability in other clusters. Persistence of linkage phase is calculated as correlation coefficient <em>r</em> by:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-22.gif"/></noscript></span></span> As a measure of LD, <em>r</em> ranges from −1 to 1 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">De Roos <em>et al.</em> 2008</a>). Persistence of linkage phase is expressed as correlation of <em>r</em> between the same chromosomes of each cluster. The number of markers differed between clusters; therefore, we averaged the correlation of <em>r</em> values between clusters over groups of 50 markers.</p></div><div id="sec-6" class="subsection"><h3>Association mapping</h3><p id="p-20">GWAS was performed with the R (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">R Core Team 2015</a>) implementations of the Efficient Mixed Model Association eXpedited (EMMAX; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kang <em>et al.</em> 2010</a>) and the Multi Locus Mixed Model (MLMM <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Segura <em>et al.</em> 2012</a>) methods. The MLMM analysis was conducted with R scripts available at <a href="https://github.com/Gregor-Mendel-Institute/mlmm">https://github.com/Gregor-Mendel-Institute/mlmm</a>. EMMAX is a method that uses a linear model in combination with a marker-derived relationship matrix to correct for population structure. The linear model has the form:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-24.gif"/></noscript></span></span> is the vector of phenotypes; <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-25.gif"/></noscript></span></span> is a matrix containing the markers; <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-26.gif"/></noscript></span></span> is the vector of fixed-effects coefficients; <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-27.gif"/></noscript></span></span> is an incidence matrix; <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-28.gif"/></noscript></span></span> is the random effect, where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-29.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-30.gif"/></noscript></span></span> representing the relationship between genotypes inferred from genetic markers; and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-31.gif"/></noscript></span></span> is the residual effect with <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-32.gif"/></noscript></span></span>. Additive genetic variance (<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-33.gif"/></noscript></span></span>) and environmental variance (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-34.gif"/></noscript></span></span>) are derived from the REML estimates. Variance components are only calculated once and are taken as fixed in EMMAX, which speeds up computation (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Kang <em>et al.</em> 2010</a>). MLMM uses the same linear model as EMMAX, but additionally includes significant SNPs as covariates in the model by using a forward–backward stepwise algorithm with reestimation of variance parameters (<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-35.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-36.gif"/></noscript></span></span>) at each step.</p><p id="p-21">Following <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Utz <em>et al.</em> (2000)</a>, the proportion of phenotypic variance explained by a QTL was calculated as:</p><p id="p-22"><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-37.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-38.gif"/></noscript></span></span> is the coefficient of determination, <em>z</em> is the number of predictors (number of significant SNPs in GWAS), and <em>N</em> is the number of observations. The proportion of genotypic variance explained was calculated as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-39.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-40.gif"/></noscript></span></span> is the heritability of a given trait as defined in equation (2).</p><p id="p-23">Confounding effects due to population structure were evaluated with the inflation factor λ, which is the ratio of the observed median to the expected median of a test statistic distribution (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Devlin and Roeder 1999</a>). Values close to 1 indicate no inflation. The significance threshold was set for the unimputed and imputed data for each trait separately using the false discovery rate (FDR; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>), where FDR values are computed from the <em>P</em>-values. The FDR was set to 0.2 for all data sets.</p></div><div id="sec-7" class="subsection"><h3>Genomic prediction</h3><p id="p-24">We evaluated three different genomic prediction methods, namely genomic BLUP (GBLUP), ridge regression BLUP (RRBLUP), and BayesB. The GBLUP model uses a realized relationship calculated from genetic markers (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Habier <em>et al.</em> 2013</a>) and is defined as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-41.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-42.gif"/></noscript></span></span> is an <em>n</em> × 1 vector of random effects and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-43.gif"/></noscript></span></span> is the design matrix. Genetic values are modeled as random effects with <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-44.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-45.gif"/></noscript></span></span> as the genetic variance and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-46.gif"/></noscript></span></span> the realized relationship matrix.</p><p id="p-25">An RRBLUP model (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Meuwissen <em>et al.</em> 2001</a>) was used to estimate the marker effects and to calculate the prediction ability. The model is of the form:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-47.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-48.gif"/></noscript></span></span> is the vector of <em>n</em> phenotypic records, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-49.gif"/></noscript></span></span> is a vector of fixed effects that represents the overall mean, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-50.gif"/></noscript></span></span> is an <em>n</em> × 1 vector of random effects, and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-51.gif"/></noscript></span></span> is the marker matrix. The residuals follow a normal distribution <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-52.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-53.gif"/></noscript></span></span> is the identity matrix. The GBLUP and RRBLUP implementations of the <em>synbreed</em> R package (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wimmer <em>et al.</em> 2012</a>) were used.</p><p id="p-26">Additionally, a BayesB (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Meuwissen <em>et al.</em> 2001</a>) model as implemented in the BGLR R package (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pérez and de los Campos 2014</a>) was used to estimate marker effects and to calculate prediction ability. BayesB uses the same linear model as RRBLUP, but a prior for the marker effects is modeled as mixture of a point of mass at zero and a slab that has a scaled-t density. Following <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Pérez and de los Campos (2014)</a>, the notation of the prior distribution is:</p><p id="p-27"><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-54.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span> where β represents the vector of regression coefficients and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-55.gif"/></noscript></span></span>is the respective variance. Parameter π is the proportion of nonzero effects and follows a β prior distribution, which implies the possibility of variable selection. <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-56.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-57.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-58.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-59.gif"/></noscript></span></span>represent the normal, chi-squared, β, and γ density distributions, respectively (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Pérez and de los Campos 2014</a>). BayesB was chosen over other Bayesian models such as BayesA and BayesC because it assumes an <em>a priori</em> distribution of marker effects following a mixture distribution with point mass at zero and a scaled-t slab similar to BayesA, and utilizes both shrinkage and variable selection, similar to BayesC (<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Pérez and de los Campos 2014</a>). The hyper-parameters were chosen according to the default values in BGLR.</p><p id="p-28">Since population structure can inflate prediction ability (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Guo <em>et al.</em> 2014</a>), we estimated the effect of population structure on prediction ability by directly correcting the kinship matrix in the GBLUP model for confounding effects of population structure (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Thorwarth <em>et al.</em> 2017</a>). The corrected kinship matrix was calculated with PC-Relate (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Conomos <em>et al.</em> 2016</a>). Prediction ability was defined as correlation between observed phenotypic and predicted genotypic values [<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-60.gif"/></noscript></span></span>] and was calculated by a five-fold cross-validation with 10 replications for each trait. To test whether genotypic effects in the genomic prediction and GWAS were caused by the same QTL, the position of SNPs with the greatest effects in genomic prediction models were compared with the most significant SNPs in the GWAS.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-29">Raw sequence data have been submitted to the Sequence Read Archive under accession number PRJEB8701. SNP calls are available on figshare at <a href="https://doi.org/10.6084/m9.figshare.5709784">https://doi.org/10.6084/m9.figshare.5709784</a>. Best Linear Unbiased Estimators (BLUEs) of phenotypic data are provided as supplementary material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS2.csv">File S2</a>).</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Phenotypic analysis of the six yield-related traits</h3><p id="p-30">The 174 gene bank accessions were evaluated for six curd-related traits and exhibited a large phenotypic variation in all traits (<a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">Yousef <em>et al.</em> 2015</a>). For example, number of days to budding ranged from 45 to 118 d with an average of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-61.gif"/></noscript></span></span> d, and curd width ranged from 11.11 to 15.29 cm with an average of <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-62.gif"/></noscript></span></span> cm (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>). Analysis of variance showed that all traits were strongly affected by genotype (G), environment (E) and genotype by environment interaction (G × E; <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-63.gif"/></noscript></span></span>). Broad-sense heritability (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-64.gif"/></noscript></span></span>) differed strongly between traits. The traits of cluster width and number of days to budding showed moderate (56%) and high (94%) heritabilities, respectively. Furthermore, the traits of curd width and cluster width showed high phenotypic (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-65.gif"/></noscript></span></span>) and genotypic (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-66.gif"/></noscript></span></span>) correlations (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>), as did apical length and nearest branch length (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-68.gif"/></noscript></span></span>). Number of days to budding was negatively correlated with number of branches (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-69.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-70.gif"/></noscript></span></span>).</p></div><div id="sec-11" class="subsection"><h3>Analyses of population structure and LD decay</h3><p id="p-31">We inferred five genetic clusters (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) by k-means clustering (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>). The clusters are differentiated by geographic origin and flowering time. Cluster 1 (<em>n</em> = 25) consists of geographically diverse accessions (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>) without a distinct geographic origin, but differs from the other clusters by its high average time to flowering (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Clusters 2 (<em>n</em> = 56) and 4 (<em>n</em> = 40) consist of predominately European accessions that differ by their mean time to budding. Clusters 3 (<em>n</em> = 24) and 5 (<em>n</em> = 22) consist mainly of Asian accessions that also differ by mean time to budding.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/707/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Discriminant analysis of principal components plot for the five inferred clusters using the k-means algorithm (Jombart and Ahmed 2011). (B) Boxplots for number of days to budding for each DAPC-inferred cluster. Letters above boxplots display Tukey-test results. Clusters with the same letter are not significantly differentiated from each other. Values within boxplots display the mean time to budding for each cluster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685854749" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Discriminant analysis of principal components plot for the five inferred clusters using the k-means algorithm (Jombart and Ahmed 2011). (B) Boxplots for number of days to budding for each DAPC-inferred cluster. Letters above boxplots display &lt;em&gt;Tukey&lt;/em&gt;-test results. Clusters with the same letter are not significantly differentiated from each other. Values within boxplots display the mean time to budding for each cluster.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/707/F1.medium.gif" width="440" height="143"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/707/F1.medium.gif" width="440" height="143"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/707/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/707/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488512" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-32" class="first-child">(A) Discriminant analysis of principal components plot for the five inferred clusters using the k-means algorithm (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Jombart and Ahmed 2011</a>). (B) Boxplots for number of days to budding for each DAPC-inferred cluster. Letters above boxplots display <em>Tukey</em>-test results. Clusters with the same letter are not significantly differentiated from each other. Values within boxplots display the mean time to budding for each cluster.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">We next tested whether the five clusters also differ by the extent of genome-wide LD, which may reflect differences in the breeding history. Based on the method described by <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Breseghello and Sorrells (2006)</a>, we estimated background LD as <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-71.gif"/></noscript></span></span> intersecting with the nonlinear regression curve at ∼151 kbp (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The extent of LD is influenced by population structure and history, and we therefore calculated LD parameters for each cluster. Clusters 1, 3, and 5 show a rapid decay in comparison with the whole population, with an average background LD of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-72.gif"/></noscript></span></span>, and 0.24, extending to ∼98, ∼83, and ∼41 kbp (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Clusters 2 and 4 have lower background LD values of 0.17 and 0.16, but higher long-range LD with averages of ∼280 and ∼231 kbp.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/707/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LD decay in the whole population (A) and clusters 1&#x2013;5 (B&#x2013;F). The dashed horizontal line indicates the average background LD of all chromosomes of a respective population. The dashed vertical line indicates the maximum distance between linked markers and is used as reference point for the LD decay." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685854749" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;LD decay in the whole population (A) and clusters 1&#x2013;5 (B&#x2013;F). The dashed horizontal line indicates the average background LD of all chromosomes of a respective population. The dashed vertical line indicates the maximum distance between linked markers and is used as reference point for the LD decay.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/707/F2.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/707/F2.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/707/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/707/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488321" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">LD decay in the whole population (A) and clusters 1–5 (B–F). The dashed horizontal line indicates the average background LD of all chromosomes of a respective population. The dashed vertical line indicates the maximum distance between linked markers and is used as reference point for the LD decay.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">The average persistence of linkage phase is moderate between all clusters except for cluster 5, which reaches the expectation value of independent segregation (50%) for unlinked markers between clusters already at close distances in comparison with cluster 2 and cluster 4 (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>), indicating a stronger differentiation between those clusters.</p></div><div id="sec-12" class="subsection"><h3>GWAS of six yield-related traits</h3><p id="p-36">Since the accessions show a strong population structure, we conducted a GWAS with models that correct for population structure and used the λ parameter to assess how well a correction was achieved. Overall, both GWAS methods showed λ values close to 1 (reflecting a good correction) for all traits and data (imputed <em>vs.</em> unimputed; Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>), which shows that the two methods account sufficiently well for population structure.</p><p id="p-37">In total, 24 SNPs are associated with at least one of the six curd-related traits (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>; FDR<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-73.gif"/></noscript></span></span>). With EMMAX, six of 675 unimputed SNPs were associated with the traits of curd width, cluster width, number of branches, and number of days to budding. Only three SNPs of the imputed data set were associated with number of days to budding using EMMAX. Significant SNPs between the imputed and unimputed data sets differ from each other; using the imputed SNPs we identified an additional putative major QTL on chromosome O6 (Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). MLMM identified nine SNPs associated with apical length, number of branches, nearest branch length, or number of days to budding using the unimputed data set, whereas in total six SNPs from the imputed data set were associated with either apical length or number of days to budding (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488522/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488522/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of significant associations detected with EMMAX and MLMM</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">Of the 24 SNPs significantly associated with the six traits, six QTL were identified by EMMAX and by MLMM. Moreover, one SNP (C02:5063181) was associated with both curd width and cluster width. Another SNP (C07:41524584) was associated with number of days to budding and number of branches (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). We identified minor-effect QTL related to apical length, number of branches, and nearest branch length. Taken together, the results indicate that the imputation slightly increased the number of significantly associated SNPs.</p><p id="p-40">Further, we tested whether SNPs identified as significantly associated with phenotypic variation also contribute to population structure (expressed as high DAPC loadings using only the imputed data; Figure S5 and S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>). For the traits of curd width and cluster width, no significant SNP detected by the GWAS was linked to a SNP with a high DAPC loading (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>), but for number of days to budding the significant association detected on chromosome O6 is located close to SNPs with high DAPC loadings (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>). This suggests that SNPs linked with flowering time variation also contribute to population differentiation.</p></div><div id="sec-13" class="subsection"><h3>Genomic prediction with GBLUP and BayesB</h3><p id="p-41">Finally, we tested whether genomic prediction of phenotypic traits can be carried out with genetically diverse gene bank accessions to select new genetic resources for breeding purposes. The average prediction ability for each trait was calculated using five cross-validations with 10 replications; it ranged from 0.13 to 0.65 with GBLUP (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) and from 0.09 to 0.66 with BayesB (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) for the different traits and data sets. The prediction ability for RRBLUP was the same as for GBLUP, and we used RRBLUP estimates of marker effects for comparison with GWAS results. For all traits and both methods (GBLUP and BayesB), average prediction abilities were higher with imputed than with unimputed data, but the differences were minor (0.42 <em>vs.</em> 0.39).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488501/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488501/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Prediction ability for six curd-related traits with different data sets using GBLUP</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488345/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488345/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Prediction ability for six curd-related traits with different data sets using BayesB</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">With BayesB, prediction ability ranged from 0.09 to 0.66 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Imputation resulted in slightly higher prediction abilities for all traits, except for number of days to budding. The prediction ability of fastPHASE was slightly higher than with BEAGLE for all traits except number of days to budding and apical length (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). To compare the GWAS with the genomic prediction results, we ranked the marker effects estimated by RRBLUP and BayesB in descending order and compared them with the 24 significant associations detected by EMMAX and MLMM. Of these 24 SNPs, 18 also produced the largest marker effects and were among the top eight SNPs with the highest <em>P</em>-values in the GWAS (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-45">We assessed the influence of population structure on prediction ability using cross-validation. A correction for population structure resulted in a minor decrease in prediction ability for most traits, except curd width. The decrease in prediction ability was substantial for number of days to budding (0.64 to 0.39; <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). To test the influence of population structure on prediction ability, we randomly sampled subsets of markers and observed fairly high prediction abilities for small marker numbers (&lt;100; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Another approach to characterize the effect of population structure on prediction ability is to estimate the phenotypic variance explained by the first three principal components of a principal component analysis (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>). According to this analysis, population structure had a strong effect on the genomic prediction ability of cluster width (40.01% variation explained by principal component 1), curd width (16.37% by principal component 1), and number of days to budding (27.66% by principal component 2). The variance explained by principal components was marginal for the other traits.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/707/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of increasing the number of markers, included in a five-fold cross-validation with 10 replications using a standard GBLUP model, on prediction ability. Values represent averages of 100 runs. 10, 25, 50, 100, 250 and 500 markers, respectively, were sampled randomly for each run." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685854749" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of increasing the number of markers, included in a five-fold cross-validation with 10 replications using a standard GBLUP model, on prediction ability. Values represent averages of 100 runs. 10, 25, 50, 100, 250 and 500 markers, respectively, were sampled randomly for each run.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/707/F3.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/2/707/F3.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/707/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/707/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488346" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-46" class="first-child">Effect of increasing the number of markers, included in a five-fold cross-validation with 10 replications using a standard GBLUP model, on prediction ability. Values represent averages of 100 runs. 10, 25, 50, 100, 250 and 500 markers, respectively, were sampled randomly for each run.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><div id="sec-15" class="subsection"><h3>Phenotypic variation</h3><p id="p-47">Our sample of cauliflower accessions could be grouped into distinct genetic clusters that also differed in phenotypic traits. Heritability estimates of the traits of cluster width, curd width, and number of days to budding were similar to previous studies (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lan and Paterson 2000</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Matschegewski <em>et al.</em> 2015</a>) and indicate a sufficient quality of the field trial data for GWAS and genomic prediction. Heritabilities for number of branches, apical length, and nearest branch were very low, which reflects a low data quality or a more complex genetic architecture.</p></div><div id="sec-16" class="subsection"><h3>Strong population structure and differences in LD decay</h3><p id="p-48">A comparison of the population structure inferred by genetic markers and the passport data indicates limitations of available passport data. Among the 93 USDA accessions in the sample, 49 have a European origin according to their passport data. Of these, 24 (49%) cluster with the European accessions from IPK and the remaining 25 (51%) are distributed between cluster 1 (late flowering, geographically diverse accessions) and cluster 5 (predominately Asian accessions). This is unexpected under the assumption of a strong correlation between geographic origin and genetic relationship, but may be explained by incomplete passport data. Data about the seed donor but not the collection site were available for 79 of 93 USDA accessions (85%), and in these cases the true geographic origin could not be verified. For this reason, the analysis of genetic relationship allows a putative geographic origin of accessions to be assigned and can be used to complement missing passport data.</p><p id="p-49">Genome-wide LD levels in the complete sample, which we measured as LD decay, were fairly high for an outcrossing species. The maximum physical distance for genetic linkage is reached at 151 kbp using the whole population. This value is comparable with that of self-fertilizing species, but lower than a previous estimate for cauliflower of up to 400 kbp (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Matschegewski <em>et al.</em> 2015</a>). A comparison of LD decay and background LD for the complete sample and each inferred cluster revealed differences between these groups. For example, the high LD in clusters 2 and 4 that consist mainly of European cultivars suggests that breeding and small population size likely contributed to the observed differences.</p><p id="p-50">The highest level of long-range LD is located on chromosome O4 in accessions of clusters 2 and 4 (data not shown). This chromosome harbors multiple paralogs of the <em>BolbZIP</em> gene family, which is involved into cold stress tolerance (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hwang <em>et al.</em> 2016</a>). Although this trait was not evaluated in our field trials, we hypothesize that selection for cold tolerance in European varieties may explain the higher level of LD.</p><p id="p-51">Finally, we compared the persistence of linkage phase between clusters (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>). The proportion of markers in the same linkage phase was moderate between most clusters, but slightly lower between clusters 2 and 5 and clusters 4 and 5, which indicates that genomic prediction ability is mainly influenced by the genetic relationship of individuals and not by a persistent linkage of marker alleles with causal QTL in different clusters.</p></div><div id="sec-17" class="subsection"><h3>Significant marker–trait associations in a GWAS of traits with a high heritability</h3><p id="p-52">The phenotypic differences in flowering time and curd width between genetic clusters suggest a genetic basis for these differences. We used two GWAS methods for the imputed and unimputed SNP data to map QTL controlling these traits and identified 24 SNPs that were associated with at least one of the six phenotypic traits. The genomic locations of SNPs associated with flowering time (<em>e.g.</em>, time to budding) coincide well with QTL regions identified in previous studies. For example, a SNP on chromosome O6 explained 26.8% of variation and was close to a QTL detected by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hasan <em>et al.</em> (2016)</a>. Another SNP found on chromosome O2 explains 16.4% of the phenotypic variation for the same trait and maps to a known QTL (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Okazaki <em>et al.</em> 2007</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Uptmoor <em>et al.</em> 2008</a>; <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Matschegewski <em>et al.</em> 2015</a>). The last of these studies also demonstrated the influence of QTL on chromosome O2 on time to flowering. The QTL region harbors a homolog of the <em>A. thaliana Flowering Locus C</em> (<em>FLC</em>) gene, which controls vernalization and has a major influence on flowering time.</p><p id="p-53">A third SNP on chromosome O1 explained 13.5% of genetic variation. It is located in a genomic region that harbors an ortholog to the <em>A. thaliana</em> the flowering time gene <em>VRN1</em> (<a id="xref-ref-46-6" class="xref-bibr" href="#ref-46">Matschegewski <em>et al.</em> 2015</a>). Additional SNPs that explain a minor proportion of the genetic variation for flowering time are located on chromosome O7 but have not been described in literature.</p><p id="p-54">The GWAS also identified three polymorphisms on chromosome O2 that are associated with curd width and cluster width. Strongly associated polymorphisms overlapped for these traits, which can be explained by the fact that the two traits are highly correlated with each other (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>). The SNPs explain between 27.4 and 34.1% of the genetic variation and therefore seem to be linked to a major QTL. These SNPs are located in genes whose <em>A. thaliana</em> orthologs regulate response to heat stress, which is consistent with the observation that temperature has a strong effect on cauliflower curd development (<a id="xref-ref-46-7" class="xref-bibr" href="#ref-46">Matschegewski <em>et al.</em> 2015</a>). <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hasan <em>et al.</em> (2016)</a> identified a QTL on the same chromosome (O2), which was only expressed under high-temperature conditions (27°).</p><p id="p-55">The differences between the two GWAS methods (EMMAX and MLMM) may result from the small sample size. Sample size has a great influence on detection power for complex traits in GWAS (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Korte and Farlow 2013</a>). However, small samples are sufficient to detect major QTL, because simulations show that QTL explaining 10% of genetic variance can be identified in a sample of 100 genotypes (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gawenda <em>et al.</em> 2015</a>). As an example, a flowering time QTL caused by variation in the vernalization response gene <em>FRIGIDA</em> was found in a sample of only 107 natural accessions of <em>A. thaliana</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Atwell <em>et al.</em> 2010</a>). We therefore consider our sample of 174 individuals sufficiently large to reliably detect major QTL for highly heritable traits such as flowering time regulation, curd width, and cluster width.</p></div><div id="sec-18" class="subsection"><h3>Evaluation of genomic prediction in cauliflower genetic resources</h3><p id="p-56">Genomic prediction is a useful method for characterizing gene bank accessions, because it may allow phenotyping to be restricted to a subset of accessions in order to predict trait values for the complete collection. We used GBLUP and BayesB for prediction, because these models have a good performance, stable prediction ability, and are suited for different genetic architectures (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Heslot <em>et al.</em> 2012</a>). Genomic prediction worked best for traits with a high heritability such as number of days to budding and cluster width, and less well for the traits of apical length and length of nearest branch (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). In addition to a low heritability, the precise phenotyping of the latter two traits is challenging, and measurement errors result in biased estimates of variance components and adjusted means that affect prediction ability.</p><p id="p-57">Since this is the first study that uses genomic prediction in cauliflower, we can only compare our results with those for other <em>Brassica</em> species. Prediction abilities for number of days to budding and curd width are similar to those for flowering time (0.70) and grain yield (0.50) in rapeseed (<em>B. napus</em>) (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Würschum <em>et al.</em> 2014</a>), a close relative of <em>B. oleracea</em>, which suggests that genomic selection is a robust and promising breeding method for <em>Brassica</em> species. Prediction ability is influenced by the presence of population structure (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Thorwarth <em>et al.</em> 2017</a>). If both training and validation sets for genomic selection show a population structure, a correction for structure reduces prediction ability (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Guo <em>et al.</em> 2014</a>). The effect of population structure on prediction ability depends on the trait. For the trait days to budding, prediction ability decreased from 0.64 to 0.39 after correction, whereas a correction had no effect on prediction ability of the other five traits.</p><p id="p-58">The strong effect of structure correction on prediction ability for flowering is consistent with our observation that this trait differs significantly between genetic clusters. Mainly loci close to the putative QTL on chromosome O6 seem to contribute to population differentiation (measured as high loading values in the DAPC analysis) and are close to SNPs linked to flowering time QTL in the GWAS. This indicates that selection for different flowering time or adaption to different areas contributed to the genetic population structure (<a id="xref-ref-46-8" class="xref-bibr" href="#ref-46">Matschegewski <em>et al.</em> 2015</a>). For the other traits such as curd with or cluster width there is no such overlap, which may be explained by a more complex genetic architecture of these traits.</p><p id="p-59">Genomic prediction models simultaneously utilize all SNPs for calculating the breeding value. RRBLUB and BayesB estimate marker effects, which can be used for comparison with GWAS models that consider each SNP separately. We found strong overlaps of significant SNPs in the GWAS, with the highest marker effects obtained from the genomic prediction models (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). This overlap reflects the similarity of the statistical models used for GWAS and genomic prediction, and suggests that these SNPs are linked to robust QTL. In summary, a comparison of putative causal SNPs identified by GWAS, the estimation of marker effects in genomic prediction, and the analysis of allele contribution to population structure (DAPC loading) may be considered as validation of significant marker–trait associations and provides useful information to improve genetic analysis (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Spindel <em>et al.</em> 2016</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bian and Holland 2017</a>).</p><p id="p-60">The success of genomic prediction depends on the method used and the size of the training set. We observed only minor differences between GBLUP and BayesB regarding their mean prediction ability over all traits and data sets (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). This observation confirms earlier work that Bayesian models do not outperform GBLUP (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bao <em>et al.</em> 2014</a>). Since GBLUP is computationally less expensive and much simpler to implement than BayesB, our results suggest that GBLUP can be recommended for genomic selection in cauliflower breeding. The size of the training set is another point to consider, because larger training sets improve prediction ability and allow a robust estimation of marker effects (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Windhausen <em>et al.</em> 2012</a>). We provide a first assessment of genomic prediction in a small sample of genetically diverse cauliflower gene bank material, but larger training sets are required, especially for traits with a complex genetic architecture (&gt;1000; <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Thorwarth <em>et al.</em> 2017</a>).</p></div><div id="sec-19" class="subsection"><h3>Imputation effect on GWAS and genomic prediction results</h3><p id="p-61">Although GBS is an efficient method for obtaining large numbers of polymorphisms, the resulting data have a high proportion of missing values. Imputation of missing data may be used to overcome this limitation. In our GWAS analysis, we obtained fewer significant associations (10) with the imputed data than with unimputed data (14; Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1/FileS1.pdf">File S1</a>). However, imputation may improve the power of GWAS because in the imputed data only we identified one additional QTL for flowering time on chromosome O6, which was also found by <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hasan <em>et al.</em> (2016)</a>. However, several QTL observed in the unimputed data were not identified in the imputed data. One explanation for this discrepancy is significance levels that are based on marker number. For quantitative traits that are influenced by QTL with small effects, a Bonferroni correction of <em>P</em>-values in GWAS may be too conservative and result in a high proportion of false negatives (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Perneger 1998</a>). Therefore, we used the FDR, which is the expected proportion of significant associations that are false positives. It is defined as <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/707/embed/mml-math-74.gif"/></noscript></span></span>, where <em>i</em> is the rank of the ascending <em>P</em>-values, <em>n</em> is the number of markers (tests), and <em>Q</em> is the FDR (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>). We used a FDR of 0.05, which implies that 20% of the observed significant associations are false positives. A smaller or higher <em>Q</em> value will lead to a more stringent or a more relaxed threshold, respectively. For example, with an FDR value 5% we observed 10 and with an FDR of 30% we obtained a total of 35 significant associations with EMMAX and MLMM. It should be noted that FDR thresholds are specific for each combination of traits and data sets, and for this reason the identification and implementation of optimal significance thresholds is still debated (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sham and Purcell 2014</a>). Thus, a careful assessment of different thresholds together with the ranking of marker effects and a comparison of the DAPC loadings can help to improve the conclusions drawn from GWAS studies and validate the robustness of putative QTL.</p><p id="p-62">The difference in the number of significant associations between the unimputed and imputed data sets is also influenced by the imputation method. The fastPHASE method is not well suited for GBS data, because a parameter vector of allele frequencies has to be estimated for each SNP. As fastPHASE performs best with SNPs genotyped in many individuals (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Scheet and Stephens 2006</a>), the high proportion of missing data reduces the average number of individuals available per SNP. As the second imputation method, we used BEAGLE, which performs well with medium to large sample sizes (&gt;1000 individuals), but not as well as fastPHASE when compared for small samples of 100 individuals, as demonstrated by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Browning and Browning (2011)</a>. The clustering approach of BEAGLE flexibly changes cluster numbers to better accommodate local LD patterns, but in general neither method can cope very well with a large amount of missing data as observed in this study (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Yang <em>et al.</em> 2014</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Xavier <em>et al.</em> 2016</a>).</p><p id="p-63">The imputation of missing markers slightly improved prediction ability for most traits, consistent with previous studies that revealed only minor advantages of imputation, particularly in self-fertilizing crops with a slow LD decay (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Rutkoski <em>et al.</em> 2013</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Poland <em>et al.</em> 2012a</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Jarquín <em>et al.</em> 2014</a>). Prediction ability is mainly determined by relatedness and less by linkage between marker and QTL, as indicated by the linkage phase analysis. For this reason, imputation has little influence on prediction ability, because small numbers of high-quality SNPs are sufficient to capture the relationship structure among individuals.</p><p id="p-64">In summary, there was only a minor advantage of imputation for GWAS and genomic prediction with our sample. However, the rapid development of genome sequencing technologies and rapidly decreasing costs will alleviate the problem of missing data for genomic prediction, as genome resequencing will uncover most genetic variation, in particular for crops with small genome sizes such as <em>B. oleracea</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Golicz <em>et al.</em> 2016</a>).</p></div><div id="sec-20" class="subsection"><h3>Implications for the utilization of genomic resources</h3><p id="p-65">Breeding populations of cauliflower are characterized by a low genetic diversity (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Golicz <em>et al.</em> 2016</a>), which limits the potential for improving varieties that meet the expectations of growers and consumers. We showed that both GWAS and genomic prediction contribute to the genetic analysis of complex traits and to the identification of novel and potentially useful genetic variation in gene bank material (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Yu <em>et al.</em> 2016</a>). Our study also provides a perspective with respect to the utilization of <em>ex situ</em> conserved gene bank accessions. Our sample was randomly selected and represents a broad genetic and geographic diversity. We achieved reasonable genomic prediction abilities, although genetic clustering inflates prediction ability if the cluster structure is correlated with trait distribution. For this reason, genotyping whole collections of gene bank accessions and the phenotyping of a sufficiently large subset allows the prediction of relevant phenotypic traits in the whole collection and the subsequent selection of accessions for further use as genetic resources. This will contribute to an efficient description and utilization of <em>ex situ</em> conserved germplasm resources.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-66">We thank Thomas Müller for his support in genotypic data analysis. We are grateful to the USDA and IPK gene banks for sending us the seeds. This work was funded by a DAAD GERLS Fellowship to E.A.A.Y. and an endowment of the Stifterverband der Deutschen Wissenschaft to K.J.S. The authors declare that they have no conflict of interest.</p><p id="p-67">Author contributions: P.T., E.A.A.Y., and K.J.S. designed the study. E.A.A.Y. analyzed the phenotypic data. P.T. performed the remaining analyses (population structure, LD, GWAS, genomic selection, and imputation). P.T. and E.A.A.Y. wrote the first version of the manuscript. P.T. and K.J.S. wrote the final version of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-68">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300199/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-69"><em>Communicating editor: J.-L. Jannink</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-25"><span class="received-label">Received </span>August 25, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-12-13"><span class="accepted-label">Accepted </span>December 13, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Thorwarth <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.1" data-doi="10.1038/nature08800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association study of 107 phenotypes in a common set of Arabidopsis thaliana inbred lines.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">465</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAtwell%26rft.auinit1%253DS.%26rft.volume%253D465%26rft.issue%253D7298%26rft.spage%253D627%26rft.epage%253D631%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B107%2Bphenotypes%2Bin%2BArabidopsis%2Bthaliana%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08800%26rft_id%253Dinfo%253Apmid%252F20336072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336072&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278249000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuong</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denny</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Potential of association mapping and genomic selection to explore PI 88788 derived soybean cyst nematode resistance.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span> DOI:10.3835/plantgenome2013.11.0039</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-other" id="cit-8.2.707.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maechler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">lme4: linear mixed-effects models using Eigen and S4.</span> <span class="cit-source">R package version</span> <span class="cit-vol">1</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2Bpackage%2Bversion%26rft.volume%253D1%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bevan</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uauy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wulff</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasileva</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic innovation for crop improvement.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">543</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D543%26rft.spage%253D346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Enhancing genomic prediction with genome-wide association studies in multiparental maize populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D118%26rft.spage%253D585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.7" data-doi="10.1534/genetics.105.044586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breseghello</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Association mapping of kernel size and milling quality in wheat (Triticum aestivum L.) cultivars.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044586%26rft_id%253Dinfo%253Apmid%252F16079235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8yLzExNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.8" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.9" data-doi="10.1038/nrg3054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Haplotype phasing: existing methods and new developments.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D703%26rft.epage%253D714%26rft.atitle%253DHaplotype%2Bphasing%253A%2Bexisting%2Bmethods%2Band%2Bnew%2Bdevelopments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3054%26rft_id%253Dinfo%253Apmid%252F21921926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921926&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.10" data-doi="10.1007/s00122-013-2203-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Association mapping of six yield-related traits in rapeseed (Brassica napus L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2203-9%26rft_id%253Dinfo%253Apmid%252F24121524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2203-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24121524&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conomos</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiner</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Model-free estimation of recent genetic relatedness.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.12" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">G. l.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">G3(Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D1903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.14" data-doi="10.1534/genetics.107.084301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Linkage disequilibrium and persistence of phase in Holstein-Friesian, Jersey and Angus cattle.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084301%26rft_id%253Dinfo%253Apmid%252F18622038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1MDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.15" data-doi="10.1111/j.0006-341X.1999.00997.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genomic control for association studies.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DDevlin%26rft.auinit1%253DB.%26rft.volume%253D55%26rft.issue%253D4%26rft.spage%253D997%26rft.epage%253D1004%26rft.atitle%253DGenomic%2Bcontrol%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0006-341X.1999.00997.x%26rft_id%253Dinfo%253Apmid%252F11315092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0006-341X.1999.00997.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11315092&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084218000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.16" data-doi="10.1007/s00122-013-2257-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edae</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association mapping of yield and yield components of spring wheat under contrasting moisture regimes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">791</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D791%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2257-8%26rft_id%253Dinfo%253Apmid%252F24408378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2257-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24408378&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.17" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.18" data-doi="10.1007/s10722-013-0058-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity analysis of yellow mustard (Sinapis alba L.) germplasm based on genotyping by sequencing.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D61%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-013-0058-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-013-0058-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gawenda</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorwarth</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ordon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association studies in elite varieties of German winter barley using single-marker and haplotype-based methods.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D134%26rft.spage%253D28%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golicz</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The pangenome of an agronomically important crop plant Brassica oleracea.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">13390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D13390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.21" data-doi="10.1007/s00122-013-2255-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gandhi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The impact of population structure on genomic prediction in stratified populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D749%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2255-x%26rft_id%253Dinfo%253Apmid%252F24452438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2255-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24452438&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.22" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.23" data-doi="10.1007/s00122-008-0733-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hasan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedt</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pons-Kühnemann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Link</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Association of gene-linked SSR markers to seed glucosinolate content in oilseed rape (Brassica napus ssp. napus).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHasan%26rft.auinit1%253DM.%26rft.volume%253D116%26rft.issue%253D8%26rft.spage%253D1035%26rft.epage%253D1049%26rft.atitle%253DAssociation%2Bof%2Bgene-linked%2BSSR%2Bmarkers%2Bto%2Bseed%2Bglucosinolate%2Bcontent%2Bin%2Boilseed%2Brape%2B%2528Brassica%2Bnapus%2Bssp.%2Bnapus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0733-3%26rft_id%253Dinfo%253Apmid%252F18322671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0733-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18322671&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256354300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hasan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matschegewski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ordon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stützel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Quantitative trait loci controlling leaf appearance and curd initiation of cauliflower in relation to temperature.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1273</span>–<span class="cit-lpage">1288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawkes</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The importance of genetic resources in plant breeding.</span> <abbr class="cit-jnl-abbrev">Biological Journal of the Linncan Society</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BJournal%2Bof%2Bthe%2BLinncan%2BSociety%26rft.volume%253D43%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.26" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.27" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.28" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.29" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manoharan</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">J.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">M.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.-W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide identification and characterization of bzip transcription factors in Brassica oleracea under cold stress.</span> <abbr class="cit-jnl-abbrev">BioMed Res. Int.</abbr> <span class="cit-vol">2016</span>: <span class="cit-fpage">4376598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioMed%2BRes.%2BInt.%26rft.volume%253D2016%26rft.spage%253D4376598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.31" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.32" data-doi="10.1186/1471-2164-15-740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kocak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posadas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jedlicka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping by sequencing for genomic prediction in a soybean breeding population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D740%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-740%26rft_id%253Dinfo%253Apmid%252F25174348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-740&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25174348&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jestin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lodé</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallée</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falentin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association mapping of quantitative resistance for Leptosphaeria maculans in oilseed rape (Brassica napus L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D27%26rft.spage%253D271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.34" data-doi="10.1093/bioinformatics/btr521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Adegenet 1.3–1: new tools for the analysis of genome-wide SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3070</span>–<span class="cit-lpage">3071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr521%26rft_id%253Dinfo%253Apmid%252F21926124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21926124&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.35" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.36" data-doi="10.1186/1746-4811-9-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farlow</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The advantages and limitations of trait analysis with gwas: a review.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D9%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-9-29%26rft_id%253Dinfo%253Apmid%252F23876160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-9-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23876160&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.37" data-doi="10.3168/jds.2008-1310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">König</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willam</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Economic evaluation of genomic breeding programs.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DKonig%26rft.auinit1%253DS.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D382%26rft.epage%253D391%26rft.atitle%253DEconomic%2Bevaluation%2Bof%2Bgenomic%2Bbreeding%2Bprograms%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1310%26rft_id%253Dinfo%253Apmid%252F19109296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109296&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lan</span>  <span class="cit-name-given-names">T.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Comparative mapping of quantitative trait loci sculpting the curd of Brassica oleracea.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1927</span>–<span class="cit-lpage">1954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLan%26rft.auinit1%253DT.-H.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D1927%26rft.epage%253D1954%26rft.atitle%253DComparative%2BMapping%2Bof%2BQuantitative%2BTrait%2BLoci%2BSculpting%2Bthe%2BCurd%2Bof%2BBrassica%2Boleracea%26rft_id%253Dinfo%253Apmid%252F10924486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS80LzE5MjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.39" data-doi="10.1093/dnares/dsu002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association study dissects the genetic architecture of seed weight and seed quality in rapeseed (Brassica napus L.).</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu002%26rft_id%253Dinfo%253Apmid%252F24510440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsu002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24510440&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000344243900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.40" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.41" data-doi="10.1038/ng.2484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2484%26rft_id%253Dinfo%253Apmid%252F23242369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23242369&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Curd development associated gene (cdag1) in cauliflower (Brassica oleracea l. var. botrytis) could result in enlarged organ size and increased biomass.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">254</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D254%26rft.spage%253D82%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.43" data-doi="10.1038/ncomms4930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Brassica oleracea genome reveals the asymmetrical evolution of polyploid genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3930%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4930%26rft_id%253Dinfo%253Apmid%252F24852848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24852848&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.44" data-doi="10.1016/j.tplants.2014.06.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Redesigning the exploitation of wheat genetic resources.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D631%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.06.012%26rft_id%253Dinfo%253Apmid%252F25052155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.06.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25052155&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343359900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.45" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matschegewski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zetzsche</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibeguth</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic variation of temperature-regulated curd induction in cauliflower: elucidation of floral transition by genome-wide association mapping and gene expression analysis.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhodes</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenton</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thayil</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dissecting genome-wide association signals for loss-of-function phenotypes in sorghum flavonoid pigmentation traits.</span> <abbr class="cit-jnl-abbrev">G3(Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2085</span>–<span class="cit-lpage">2094</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D2085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.49" data-doi="10.1371/journal.pone.0089685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norton</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douglas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakubova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerinot</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome wide association mapping of grain arsenic, copper, molybdenum and zinc in rice (Oryza sativa L.) grown at four international field sites.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e89685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De89685%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0089685%26rft_id%253Dinfo%253Apmid%252F24586963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0089685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586963&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.50" data-doi="10.1080/07352689109382313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nyquist</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Estimation of heritability and prediction of selection response in plant populations.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D10%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07352689109382313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07352689109382313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991GF83500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.51" data-doi="10.1007/s00122-006-0460-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okazaki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Togashi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping and characterization of FLC homologs and QTL analysis of flowering time in Brassica oleracea.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DOkazaki%26rft.auinit1%253DK.%26rft.volume%253D114%26rft.issue%253D4%26rft.spage%253D595%26rft.epage%253D608%26rft.atitle%253DMapping%2Band%2Bcharacterization%2Bof%2BFLC%2Bhomologs%2Band%2BQTL%2Banalysis%2Bof%2Bflowering%2Btime%2Bin%2BBrassica%2Boleracea.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0460-6%26rft_id%253Dinfo%253Apmid%252F17136371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0460-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17136371&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243973300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.52" data-doi="10.1371/journal.pone.0003551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">Y.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papasian</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H.-W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analyses and comparison of accuracy of different genotype imputation methods.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPei%26rft.auinit1%253DY.%2BF.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3551%26rft.epage%253De3551%26rft.atitle%253DAnalyses%2Band%2Bcomparison%2Bof%2Baccuracy%2Bof%2Bdifferent%2Bgenotype%2Bimputation%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003551%26rft_id%253Dinfo%253Apmid%252F18958166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18958166&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.707.53" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.54" data-doi="10.1136/bmj.316.7139.1236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perneger</span>  <span class="cit-name-given-names">T. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">What’s wrong with Bonferroni adjustments.</span> <abbr class="cit-jnl-abbrev">BMJ</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMJ%26rft.stitle%253DBMJ%26rft.aulast%253DPerneger%26rft.auinit1%253DT.%2BV%26rft.volume%253D316%26rft.issue%253D7139%26rft.spage%253D1236%26rft.epage%253D1238%26rft.atitle%253DWhat%2527s%2Bwrong%2Bwith%2BBonferroni%2Badjustments%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fbmj.316.7139.1236%26rft_id%253Dinfo%253Apmid%252F9553006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYm1qIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNzEzOS8xMjM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzcwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picchi</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Migliori</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scalzo</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campanelli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrari</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Phytochemical content in organic and conventionally grown Italian cauliflower.</span> <abbr class="cit-jnl-abbrev">Food Chem.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BChem.%26rft.volume%253D130%26rft.spage%253D501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">The Plant Genome Journal</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%2BJournal%26rft.volume%253D5%26rft.spage%253D103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">The Plant Genome Journal</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%2BJournal%26rft.volume%253D5%26rft.spage%253D92%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.58" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.59" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.707.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing.</em></span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rezaeizad</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittkop</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of QTLs for phenolic compounds in oilseed rape (Brassica napus L.) by association mapping using SSR markers.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D177%26rft.spage%253D335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.62" data-doi="10.1186/gb-2013-14-1-r1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-1-r1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-1-r1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Imputation of unordered markers and the impact on genomic selection accuracy.</span> <abbr class="cit-jnl-abbrev">G3(Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.64" data-doi="10.1111/j.1439-0388.2006.00595.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Strategy for applying genome-wide selection in dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DSchaeffer%26rft.auinit1%253DL.%2BR.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D218%26rft.epage%253D223%26rft.atitle%253DStrategy%2Bfor%2Bapplying%2Bgenome-wide%2Bselection%2Bin%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2006.00595.x%26rft_id%253Dinfo%253Apmid%252F16882088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2006.00595.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16882088&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239006300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.65" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorwarth</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection in barley breeding.</span> <abbr class="cit-jnl-abbrev">Springer Berlin Heidelberg</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSpringer%2BBerlin%2BHeidelberg%26rft.volume%253D69%26rft.spage%253D367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.67" data-doi="10.1038/ng.2314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seren</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegura%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D825%26rft.epage%253D830%26rft.atitle%253DAn%2Befficient%2Bmulti-locus%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2314%26rft_id%253Dinfo%253Apmid%252F22706313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706313&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.68" data-doi="10.1038/nrg3706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sham</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Statistical power and significance testing in large-scale genetic studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3706%26rft_id%253Dinfo%253Apmid%252F24739678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24739678&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.69" data-doi="10.1371/journal.pone.0054603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iquira</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardivel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Légaré</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54603%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054603%26rft_id%253Dinfo%253Apmid%252F23372741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372741&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redona</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide prediction models that incorporate de novo gwas are a powerful new tool for tropical rice improvement.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D116%26rft.spage%253D395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tardivel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belzile</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donoughue</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rapid identification of alleles at the soybean maturity gene E3 using genotyping by sequencing and a haplotype-based approach.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span> DOI:10.3835/plantgenome2013.10.0034</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.72" data-doi="10.1126/science.1183700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Breeding technologies to increase crop production in a changing world.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">818</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTester%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5967%26rft.spage%253D818%26rft.epage%253D822%26rft.atitle%253DBreeding%2BTechnologies%2Bto%2BIncrease%2BCrop%2BProduction%2Bin%2Ba%2BChanging%2BWorld%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183700%26rft_id%253Dinfo%253Apmid%252F20150489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorwarth</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahlemeyer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bochard</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krumnacker</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blümel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction ability for yield-related traits in German winter barley elite material.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">1669</span>–<span class="cit-lpage">1683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D1669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uptmoor</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stützel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esch</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Crop model based QTL analysis across environments and QTL based estimation of time to floral induction and flowering in Brassica oleracea.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D21%26rft.spage%253D205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Bias and sampling error of the estimated proportion of genotypic variance explained by quantitative trait loci determined from experimental data in maize using cross validation and validation with independent samples.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1839</span>–<span class="cit-lpage">1849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DUtz%26rft.auinit1%253DH.%2BF.%26rft.volume%253D154%26rft.issue%253D4%26rft.spage%253D1839%26rft.epage%253D1849%26rft.atitle%253DBias%2Band%2BSampling%2BError%2Bof%2Bthe%2BEstimated%2BProportion%2Bof%2BGenotypic%2BVariance%2BExplained%2Bby%2BQuantitative%2BTrait%2BLoci%2BDetermined%2BFrom%2BExperimental%2BData%2Bin%2BMaize%2BUsing%2BCross%2BValidation%2Band%2BValidation%2BWith%2BIndependent%2BSamples%26rft_id%253Dinfo%253Apmid%252F10866652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC80LzE4MzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNzA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.76" data-doi="10.1016/j.tpb.2008.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanLiere</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mathematical properties of the <em>r</em><sup>2</sup> measure of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DVanliere%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D130%26rft.epage%253D137%26rft.atitle%253DMathematical%2Bproperties%2Bof%2Bthe%2Br2%2Bmeasure%2Bof%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2008.05.006%26rft_id%253Dinfo%253Apmid%252F18572214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2008.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18572214&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257912400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.77" data-doi="10.1093/bioinformatics/bts335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Synbreed: a framework for the analysis of genomic prediction data using R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2086</span>–<span class="cit-lpage">2087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts335%26rft_id%253Dinfo%253Apmid%252F22689388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22689388&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D1427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Potential of genomic selection in rapeseed (<em>Brassica napus</em> L.) breeding.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D133%26rft.spage%253D45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rainey</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Impact of imputation methods on the amount of genetic variation captured by a single-nucleotide polymorphism panel in soybeans.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.81" data-doi="10.1371/journal.pone.0101025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A new genotype imputation method with tolerance to high missing rate and rare variants.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e101025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101025%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0101025%26rft_id%253Dinfo%253Apmid%252F24972110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0101025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24972110&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yousef</span>  <span class="cit-name-given-names">E. A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lampei</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evaluation of cauliflower genebank accessions under organic and conventional cultivation in Southern Germany.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D201%26rft.spage%253D389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yousef</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative analysis of genetic diversity and differentiation of cauliflower (Brassica oleracea var. botrytis) accessions from two ex situ genebanks.</span><abbr class="cit-jnl-abbrev"> bioRxiv</abbr>. doi: https://doi.org/10.1101/238840</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction contributing to a promising global strategy to turbocharge gene banks.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">16150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D16150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide single-nucleotide polymorphisms discovery and high-density genetic map construction in cauliflower using specific-locus amplified fragment sequencing.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.707.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Association mapping of seed oil content in Brassica napus and comparison with quantitative trait loci identified from linkage mapping.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">908</span>–<span class="cit-lpage">916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DZou%26rft.auinit1%253DJ.%26rft.volume%253D53%26rft.issue%253D11%26rft.spage%253D908%26rft.epage%253D916%26rft.atitle%253DAssociation%2Bmapping%2Bof%2Bseed%2Boil%2Bcontent%2Bin%2BBrassica%2Bnapus%2Band%2Bcomparison%2Bwith%2Bquantitative%2Btrait%2Bloci%2Bidentified%2Bfrom%2Blinkage%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F21076506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21076506&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F707.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/707.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/695" title="Genetic Identification of Separase Regulators in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/719" title="Genomic Prediction Accuracy for Resistance Against Piscirickettsia salmonis in Farmed Rainbow Trout" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/707" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--jE8D3UK4A3BuIegqVTkUOXtYeSSrkaG65xLKZkAKEE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/707" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/707" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/707" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488298" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Pv3at7sNIYGrccdbpxtKFqcyGztlXHdD3_2r2Jas9Jo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/707.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/707" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-B-cho5oFvuWKgiea_qUWbKy8c8PjvcetjCK1Ku15GlY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/707" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/707.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488298" id="node488298--2" data-pisa="ggg;8/2/707" data-pisa-master="ggg;g3.117.300199" data-apath="/ggg/8/2/707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4456-2358" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Thorwarth</span>, <span class="highwire-citation-author" data-delta="1">Eltohamy A. A. Yousef</span> and <span class="highwire-citation-author" data-delta="2">Karl J. Schmid</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">707-718;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300199 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488298--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patrick Thorwarth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BThorwarth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thorwarth%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BThorwarth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4456-2358" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Thorwarth</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eltohamy A. A. Yousef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, Faculty of Agriculture, University of Suez Canal, Ismailia 41522, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eltohamy%2BA.%2BA.%2BYousef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yousef%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEltohamy%2BA.%2BA.%2BYousef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karl J. Schmid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BJ.%2BSchmid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmid%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BJ.%2BSchmid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Karl.Schmid@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">Karl.Schmid@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/707" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/707&amp;title=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/707&amp;title=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/707&amp;title=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/707&amp;text=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/707&amp;title=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/707&amp;t=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/707&amp;title=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/707&amp;title=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488298/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488298" id="node488298--3" data-pisa="ggg;8/2/707" data-pisa-master="ggg;g3.117.300199" data-apath="/ggg/8/2/707.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction and Association Mapping of Curd-Related Traits in Gene Bank Accessions of Cauliflower</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4456-2358" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick Thorwarth</span>, <span class="highwire-citation-author" data-delta="1">Eltohamy A. A. Yousef</span> and <span class="highwire-citation-author" data-delta="2">Karl J. Schmid</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">707-718;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300199 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488298--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patrick Thorwarth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BThorwarth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thorwarth%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BThorwarth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4456-2358" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick Thorwarth</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eltohamy A. A. Yousef </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Horticulture, Faculty of Agriculture, University of Suez Canal, Ismailia 41522, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eltohamy%2BA.%2BA.%2BYousef%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yousef%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEltohamy%2BA.%2BA.%2BYousef%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karl J. Schmid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BJ.%2BSchmid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmid%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BJ.%2BSchmid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Karl.Schmid@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">Karl.Schmid@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=707&amp;rft.epage=718&amp;rft.atitle=Genomic%20Prediction%20and%20Association%20Mapping%20of%20Curd-Related%20Traits%20in%20Gene%20Bank%20Accessions%20of%20Cauliflower&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Thorwarth%2C+Patrick&amp;rft.au=Yousef%2C+Eltohamy+A.+A.&amp;rft.au=Schmid%2C+Karl+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488298/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488298/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488298/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488298/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488298/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488298/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488298/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488298/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488298/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488298/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488298/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488298/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/707" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/707.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/707.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/707.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/707.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
