<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/447" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/447.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/447.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/447.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/2/447/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/447.atom" />
<meta name="HW.pisa" content="ggg;8/2/447" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Direct Testing for Allele-Specific Expression Differences Between Conditions" />
<meta name="DC.Identifier" content="10.1534/g3.117.300139" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Novelo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Allelic imbalance (AI) indicates the presence of functional variation in cis regulatory regions. Detecting cis regulatory differences using AI is widespread, yet there is no formal statistical methodology that tests whether AI differs between conditions. Here, we present a novel model and formally test differences in AI across conditions using Bayesian credible intervals. The approach tests AI by environment (G×E) interactions, and can be used to test AI between environments, genotypes, sex, and any other condition. We incorporate bias into the modeling process. Bias is allowed to vary between conditions, making the formulation of the model general. As gene expression affects power for detection of AI, and, as expression may vary between conditions, the model explicitly takes coverage into account. The proposed model has low type I and II error under several scenarios, and is robust to large differences in coverage between conditions. We reanalyze RNA-seq data from a Drosophila melanogaster population panel, with F1 genotypes, to compare levels of AI between mated and virgin female flies, and we show that AI × genotype interactions can also be tested. To demonstrate the use of the model to test genetic differences and interactions, a formal test between two F1s was performed, showing the expected 20% difference in AI. The proposed model allows a formal test of G×E and G×G, and reaffirms a previous finding that cis regulation is robust between environments." />
<meta name="DC.Contributor" content="Luis León-Novelo" />
<meta name="DC.Contributor" content="Alison R. Gerken" />
<meta name="DC.Contributor" content="Rita M. Graze" />
<meta name="DC.Contributor" content="Lauren M. McIntyre" />
<meta name="DC.Contributor" content="Fabio Marroni" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Direct Testing for Allele-Specific Expression Differences Between Conditions" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Allelic imbalance (AI) indicates the presence of functional variation in &lt;i&gt;cis&lt;/i&gt; regulatory regions. Detecting &lt;i&gt;cis&lt;/i&gt; regulatory differences using AI is widespread, yet there is no formal statistical methodology that tests whether AI differs between conditions. Here, we present a novel model and formally test differences in AI across conditions using Bayesian credible intervals. The approach tests AI by environment (G×E) interactions, and can be used to test AI between environments, genotypes, sex, and any other condition. We incorporate bias into the modeling process. Bias is allowed to vary between conditions, making the formulation of the model general. As gene expression affects power for detection of AI, and, as expression may vary between conditions, the model explicitly takes coverage into account. The proposed model has low type&amp;nbsp;I and II error under several scenarios, and is robust to large differences in coverage between conditions. We reanalyze RNA-seq data from a &lt;i&gt;Drosophila melanogaster&lt;/i&gt; population panel, with F1 genotypes, to compare levels of AI between mated and virgin female flies, and we show that AI × genotype interactions can also be tested. To demonstrate the use of the model to test genetic differences and interactions, a formal test between two F1s was performed, showing the expected 20% difference in AI. The proposed model allows a formal test of G×E and G×G, and reaffirms a previous finding that &lt;i&gt;cis&lt;/i&gt; regulation is robust between environments.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/447" />
<meta name="citation_id" content="8/2/447" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/447" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/447.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/447.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/447.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300139" />
<meta name="citation_pmid" content="29167272" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="447" />
<meta name="citation_lastpage" content="460" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Luis León-Novelo" />
<meta name="citation_author_institution" content="Department of Biostatistics and Data Science, The University of Texas Health Science Center at Houston-School of Public Health, Texas 77030" />
<meta name="citation_author" content="Alison R. Gerken" />
<meta name="citation_author_institution" content="United States Department of Agriculture, Agricultural Research Service, Center for Grain and Animal Health Research, Manhattan, Kansas 66502" />
<meta name="citation_author" content="Rita M. Graze" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Auburn University, Alabama 36849" />
<meta name="citation_author" content="Lauren M. McIntyre" />
<meta name="citation_author_institution" content="Genetics Institute and Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, Florida 32603" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0077-3359" />
<meta name="citation_author" content="Fabio Marroni" />
<meta name="citation_author_institution" content="Dipartimento di Scienze Agroalimentari, Ambientali e Animali, Università di Udine, 33100, Italy" />
<meta name="citation_author_institution" content="Istituto di Genomica Applicata, 33100 Udine, Italy" />
<meta name="citation_author_email" content="marroni@appliedgenomics.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1556-5907" />
<meta name="citation_reference" content="citation_journal_title=Experimental gerontology;citation_journal_abbrev=Exp Gerontol;citation_author=T. Aigaki;citation_author=S. Ohba;citation_title=Effect of mating status on Drosophila virilis lifespan.;citation_pages=267-278;citation_volume=19;citation_year=1984;citation_issue=4;citation_pmid=6499967;citation_doi=10.1016/0531-5565(84)90022-6" />
<meta name="citation_reference" content="citation_journal_title=Stat.;citation_author=D. G. Altman;citation_author=J. M. Bland;citation_title=Measurement in medicine: the analysis of method comparison studies.;citation_pages=307;citation_volume=32;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Anders;citation_author=W. Huber;citation_title=Differential expression analysis for sequence count data.;citation_pages=R106-R106;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20979621;citation_doi=10.1186/gb-2010-11-10-r106" />
<meta name="citation_reference" content="citation_title=Science;citation_author=R. B. Brem;citation_author=G. Yvert;citation_author=R. Clinton;citation_author=L. Kruglyak;citation_title=Genetic dissection of transcriptional regulation in budding yeast.;citation_volume=296;citation_year=2002;citation_pmid=11923494;citation_doi=10.1126/science.1069516" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Buil;citation_author=A. A. Brown;citation_author=T. Lappalainen;citation_author=A. Viñuela;citation_author=M. N. Davies;citation_title=Gene-gene and gene-environment interactions detected by transcriptome sequence analysis in twins.;citation_pages=88-91;citation_volume=47;citation_year=2015;citation_pmid=25436857;citation_doi=10.1038/ng.3162" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. E. Castel;citation_author=A. Levy-Moonshine;citation_author=P. Mohammadi;citation_author=E. Banks;citation_author=T. Lappalainen;citation_title=Tools and best practices for data processing in allelic expression analysis.;citation_pages=195;citation_volume=16;citation_year=2015;citation_pmid=26381377;citation_doi=10.1186/s13059-015-0762-6" />
<meta name="citation_reference" content="citation_title=PLOS Genet.;citation_author=J. Chen;citation_author=V. Nolte;citation_author=C. Schlötterer;citation_title=Temperature stress mediates decanalization and dominance of gene expression in Drosophila melanogaster.;citation_pages=e1004883;citation_volume=11;citation_year=2015;citation_pmid=25719753;citation_doi=10.1371/journal.pgen.1004883" />
<meta name="citation_reference" content="citation_journal_title=Educational and Psychological Measurement;citation_journal_abbrev=Educational and Psychological Measurement;citation_author=J. Cohen;citation_title=A Coefficient of Agreement for Nominal Scales;citation_pages=37-46;citation_volume=20;citation_year=1960;citation_issue=1;citation_doi=10.1177/001316446002000104" />
<meta name="citation_reference" content="citation_journal_title=BMC Genetics;citation_title=Intersection tests for single marker QTL analysis can be more powerful than two marker QTL analysis.;citation_pages=10;citation_volume=4;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. J. Crowley;citation_author=V. Zhabotynsky;citation_author=W. Sun;citation_author=S. Huang;citation_author=I. K. Pakatci;citation_title=Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance.;citation_pages=353-360;citation_volume=47;citation_year=2015;citation_pmid=25730764;citation_doi=10.1038/ng.3222" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. A. Cubillos;citation_author=O. Stegle;citation_author=C. Grondin;citation_author=M. Canut;citation_author=S. Tisné;citation_title=Extensive cis-regulatory variation robust to environmental perturbation in Arabidopsis.;citation_volume=26;citation_year=2014;citation_pmid=25428981;citation_doi=10.1105/tpc.114.130310" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. F. Degner;citation_author=J. C. Marioni;citation_author=A. A. Pai;citation_author=J. K. Pickrell;citation_author=E. Nkadori;citation_title=Effect of read-mapping biases on detecting allele-specific expression from RNA-sequencing data.;citation_volume=25;citation_year=2009;citation_pmid=19808877;citation_doi=10.1093/bioinformatics/btp579" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=Y. Di;citation_author=D. W. Schafer;citation_author=J. S. Cumbie;citation_author=J. H. Chang;citation_title=The NBP negative binomial model for assessing differential gene expression from RNA-Seq.;citation_pages=1-28;citation_volume=10;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=D. Edsgärd;citation_author=M. J. Iglesias;citation_author=S.-J. Reilly;citation_author=A. Hamsten;citation_author=P. Tornvall;citation_title=GeneiASE: detection of condition-dependent and static allele-specific expression from RNA-seq data without haplotype information.;citation_pages=21134;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. Everaerts;citation_author=J. P. Farine;citation_author=M. Cobb;citation_author=J. F. Ferveur;citation_title=Drosophila cuticular hydrocarbons revisited: mating status alters cuticular profiles.;citation_pages=e9607-e9607;citation_volume=5;citation_year=2010;citation_issue=3;citation_pmid=20231905;citation_doi=10.1371/journal.pone.0009607" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. M. Fear;citation_author=L. G. Leon-Novelo;citation_author=A. M. Morse;citation_author=A. R. Gerken;citation_author=K. Van Lehmann;citation_title=Buffering of genetic regulatory networks in Drosophila melanogaster.;citation_volume=203;citation_year=2016;citation_pmid=27194752;citation_doi=10.1534/genetics.116.188797" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. M. Graze;citation_author=L. L. Novelo;citation_author=V. Amin;citation_author=J. M. Fear;citation_author=G. Casella;citation_title=Allelic imbalance in Drosophila hybrid heads: exons, isoforms, and evolution.;citation_volume=29;citation_year=2012;citation_pmid=22319150;citation_doi=10.1093/molbev/msr318" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=R. M. Graze;citation_author=L. M. McIntyre;citation_author=A. M. Morse;citation_author=B. M. Boyd;citation_author=S. V. Nuzhdin;citation_title=What the X has to do with it: differences in regulatory variability between the sexes in Drosophila simulans.;citation_volume=6;citation_year=2014;citation_pmid=24696400;citation_doi=10.1093/gbe/evu060" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=C. Gregg;citation_author=J. Zhang;citation_author=B. Weissbourd;citation_author=S. Luo;citation_author=G. P. Schroth;citation_title=High-resolution analysis of parent-of-origin allelic expression in the mouse brain.;citation_volume=329;citation_year=2010;citation_pmid=20616232;citation_doi=10.1126/science.1190830" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. A. Knowles;citation_author=J. R. Davis;citation_author=H. Edgington;citation_author=A. Raj;citation_author=M. J. Favé;citation_title=Allele-specific expression reveals interactions between genetic variation and environment.;citation_pages=699-702;citation_volume=14;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Z. Kurmangaliyev;citation_author=A. V. Favorov;citation_author=N. M. Osman;citation_author=K.-V. Lehmann;citation_author=D. Campo;citation_title=Natural variation of gene models in Drosophila melanogaster.;citation_pages=198;citation_volume=16;citation_year=2015;citation_pmid=25888292;citation_doi=10.1186/s12864-015-1415-6" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=M. K. Lawniczak;citation_author=D. J. Begun;citation_title=A genome-wide analysis of courting and mating responses in Drosophila melanogaster females.;citation_pages=900-910;citation_volume=47;citation_year=2004;citation_issue=5;citation_pmid=15499404;citation_doi=10.1139/g04-050" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. G. León-Novelo;citation_author=L. M. McIntyre;citation_author=J. M. Fear;citation_author=R. M. Graze;citation_title=A flexible Bayesian method for detecting allelic imbalance in RNA-seq data.;citation_pages=920;citation_volume=15;citation_year=2014;citation_pmid=25339465;citation_doi=10.1186/1471-2164-15-920" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=L. León-Novelo;citation_author=C. Fuentes;citation_author=S. Emerson;citation_title=Marginal likelihood estimation of negative binomial parameters with applications to RNA-seq data.;citation_pages=637-650;citation_volume=18;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Lin;citation_author=A. Hrabovsky;citation_author=E. Pedrosa;citation_author=T. Wang;citation_author=D. Zheng;citation_title=Allele-biased expression in differentiating human neurons: implications for neuropsychiatric disorders.;citation_pages=e44017-e44017;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22952857;citation_doi=10.1371/journal.pone.0044017" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=H. S. Lo;citation_author=Z. Wang;citation_author=Y. Hu;citation_author=H. H. Yang;citation_author=S. Gere;citation_title=Allelic Variation in Gene Expression Is Common in the Human Genome;citation_pages=1855-1862;citation_volume=13;citation_year=2003;citation_issue=8;citation_pmid=12902379" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. T. Maurano;citation_author=R. Humbert;citation_author=E. Rynes;citation_author=R. E. Thurman;citation_author=E. Haugen;citation_title=Systematic localization of common disease-associated variation in regulatory DNA.;citation_volume=337;citation_year=2012;citation_pmid=22955828;citation_doi=10.1126/science.1222794" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. McCarroll;citation_author=A. Huett;citation_author=P. Kuballa;citation_author=S. D. Chilewski;citation_author=A. Landry;citation_title=Deletion polymorphism upstream of IRGM associated with altered IRGM expression and Crohn's disease.;citation_pages=1107-1112;citation_volume=40;citation_year=2008;citation_issue=9;citation_pmid=19165925;citation_doi=10.1038/ng.215" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. A. McGraw;citation_author=A. G. Clark;citation_author=M. F. Wolfner;citation_title=Post-mating gene expression profiles of female Drosophila melanogaster in response to time and to four male accessory gland proteins.;citation_volume=179;citation_year=2008;citation_pmid=18562649;citation_doi=10.1534/genetics.108.086934" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. J. McManus;citation_author=J. D. Coolon;citation_author=M. O. Duff;citation_author=J. Eipper-Mains;citation_author=B. R. Graveley;citation_title=Regulatory divergence in Drosophila revealed by mRNA-seq.;citation_volume=20;citation_year=2010;citation_pmid=20354124;citation_doi=10.1101/gr.102491.109" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. A. Moyerbrailean;citation_author=A. L. Richards;citation_author=D. Kurtz;citation_author=C. A. Kalita;citation_author=G. O. Davis;citation_title=High-throughput allele-specific expression across 250 environmental conditions.;citation_volume=26;citation_year=2016;citation_pmid=27934696;citation_doi=10.1101/gr.209759.116" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. C. Munger;citation_author=N. Raghupathy;citation_author=K. Choi;citation_author=A. K. Simons;citation_author=D. M. Gatti;citation_title=RNA-Seq Alignment to Individualized Genomes Improves Transcript Abundance Estimates in Multiparent Populations;citation_pages=59-73;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236449;citation_doi=10.1534/genetics.114.165886" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=M. D. Robinson;citation_author=G. K. Smyth;citation_title=Small-sample estimation of negative binomial dispersion, with applications to SAGE data.;citation_pages=321-332;citation_volume=9;citation_year=2007;citation_pmid=17728317;citation_doi=10.1093/biostatistics/kxm030" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=M. V. Rockman;citation_author=G. A. Wray;citation_title=Abundant Raw Material for Cis-Regulatory Evolution in Humans;citation_pages=1991-2004;citation_volume=19;citation_year=2002;citation_issue=11;citation_pmid=12411608;citation_doi=10.1093/oxfordjournals.molbev.a004023" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. Ronald;citation_author=J. M. Akey;citation_author=J. Whittle;citation_author=E. N. Smith;citation_author=G. Yvert;citation_title=Simultaneous genotyping, gene-expression measurement, and detection of allele-specific expression with oligonucleotide arrays;citation_pages=284-291;citation_volume=15;citation_year=2005;citation_issue=2;citation_pmid=15687292;citation_doi=10.1101/gr.2850605" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=J. Rozowsky;citation_author=A. Abyzov;citation_author=J. Wang;citation_author=P. Alves;citation_author=D. Raha;citation_title=AlleleSeq: analysis of allele-specific expression and binding in a network framework;citation_pages=522-522;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21811232;citation_doi=10.1038/msb.2011.54" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. V. Satya;citation_author=N. Zavaljevski;citation_author=J. Reifman;citation_title=A new strategy to reduce allelic bias in RNA-Seq readmapping.;citation_volume=40;citation_year=2012;citation_pmid=22584625;citation_doi=10.1093/nar/gks425" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Skelly;citation_author=M. Johansson;citation_author=J. Madeoy;citation_author=J. Wakefield;citation_author=J. M. Akey;citation_title=A powerful and flexible statistical framework for testing hypotheses of allele-specific gene expression from RNA-seq data.;citation_volume=21;citation_year=2011;citation_pmid=21873452;citation_doi=10.1101/gr.119784.110" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. R. Stevenson;citation_author=J. D. Coolon;citation_author=P. J. Wittkopp;citation_title=Sources of bias in measures of allele-specific expression derived from RNA-sequence data aligned to a single reference genome.;citation_pages=536;citation_volume=14;citation_year=2013;citation_pmid=23919664;citation_doi=10.1186/1471-2164-14-536" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. E. Stranger;citation_author=S. B. Montgomery;citation_author=A. S. Dimas;citation_author=L. Parts;citation_author=O. Stegle;citation_title=Patterns of cis regulatory variation in diverse human populations.;citation_pages=e1002639-e1002639;citation_volume=8;citation_year=2012;citation_issue=4;citation_pmid=22532805;citation_doi=10.1371/journal.pgen.1002639" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. Tung;citation_author=M. Y. Akinyi;citation_author=S. Mutura;citation_author=J. Altmann;citation_author=G. A. Wray;citation_title=Allele-specific gene expression in a wild nonhuman primate population.;citation_pages=725-739;citation_volume=20;citation_year=2011;citation_issue=4;citation_pmid=21226779;citation_doi=10.1111/j.1365-294X.2010.04970.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=E. Turro;citation_author=S.-Y. Su;citation_author=Â. Gonçalves;citation_author=L. J. M. Coin;citation_author=S. Richardson;citation_title=Haplotype and isoform specific expression estimation using multi-mapping RNA-seq reads.;citation_pages=R13-R13;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21310039;citation_doi=10.1186/gb-2011-12-2-r13" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=M. von Korff;citation_author=S. Radovic;citation_author=W. Choumane;citation_author=K. Stamati;citation_author=S. M. Udupa;citation_title=Asymmetric allele-specific expression in relation to developmental variation and drought stress in barley hybrids.;citation_pages=14-26;citation_volume=59;citation_year=2009;citation_issue=1;citation_pmid=19309461;citation_doi=10.1111/j.1365-313X.2009.03848.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. Wang;citation_author=S. Uebbing;citation_author=H. Ellegren;citation_title=Bayesian inference of allele-specific gene expression indicates abundant cis-regulatory variation in natural flycatcher populations.;citation_pages=1266-1279;citation_volume=9;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. J. Wittkopp;citation_author=B. K. Haerum;citation_author=A. G. Clark;citation_title=Evolutionary changes in cis and trans gene regulation.;citation_pages=85-88;citation_volume=430;citation_year=2004;citation_issue=6995;citation_pmid=15229602;citation_doi=10.1038/nature02698" />
<meta name="citation_reference" content="citation_title=Science;citation_journal_abbrev=Science;citation_author=H. Yan;citation_author=W. Yuan;citation_author=V. E. Velculescu;citation_author=B. Vogelstein;citation_author=K. W. Kinzler;citation_title=Allelic Variation in Human Gene Expression;citation_pages=1143-1143;citation_volume=297;citation_year=2002;citation_issue=5584;citation_pmid=12183620;citation_doi=10.1126/science.1072545" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Zhang;citation_author=J. O. Borevitz;citation_title=Global analysis of allele-specific expression in Arabidopsis thaliana.;citation_volume=182;citation_year=2009;citation_pmid=19474198;citation_doi=10.1534/genetics.109.103499" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Zhou;citation_author=T. F. C. Mackay;citation_author=R. R. H. Anholt;citation_title=Transcriptional and epigenetic responses to mating and aging in Drosophila melanogaster.;citation_pages=927;citation_volume=15;citation_year=2014;citation_pmid=25344338;citation_doi=10.1186/1471-2164-15-927" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Direct Testing for Allele-Specific Expression Differences Between Conditions" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Allelic imbalance (AI) indicates the presence of functional variation in cis regulatory regions. Detecting cis regulatory differences using AI is widespread, yet there is no formal statistical methodology that tests whether AI differs between conditions. Here, we present a novel model and formally test differences in AI across conditions using Bayesian credible intervals. The approach tests AI by environment (G×E) interactions, and can be used to test AI between environments, genotypes, sex, and any other condition. We incorporate bias into the modeling process. Bias is allowed to vary between conditions, making the formulation of the model general. As gene expression affects power for detection of AI, and, as expression may vary between conditions, the model explicitly takes coverage into account. The proposed model has low type I and II error under several scenarios, and is robust to large differences in coverage between conditions. We reanalyze RNA-seq data from a Drosophila melanogaster population panel, with F1 genotypes, to compare levels of AI between mated and virgin female flies, and we show that AI × genotype interactions can also be tested. To demonstrate the use of the model to test genetic differences and interactions, a formal test between two F1s was performed, showing the expected 20% difference in AI. The proposed model allows a formal test of G×E and G×G, and reaffirms a previous finding that cis regulation is robust between environments." />
<meta name="og-title" property="og:title" content="Direct Testing for Allele-Specific Expression Differences Between Conditions" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/447" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Allelic imbalance (AI) indicates the presence of functional variation in cis regulatory regions. Detecting cis regulatory differences using AI is widespread, yet there is no formal statistical methodology that tests whether AI differs between conditions. Here, we present a novel model and formally test differences in AI across conditions using Bayesian credible intervals. The approach tests AI by environment (G×E) interactions, and can be used to test AI between environments, genotypes, sex, and any other condition. We incorporate bias into the modeling process. Bias is allowed to vary between conditions, making the formulation of the model general. As gene expression affects power for detection of AI, and, as expression may vary between conditions, the model explicitly takes coverage into account. The proposed model has low type I and II error under several scenarios, and is robust to large differences in coverage between conditions. We reanalyze RNA-seq data from a Drosophila melanogaster population panel, with F1 genotypes, to compare levels of AI between mated and virgin female flies, and we show that AI × genotype interactions can also be tested. To demonstrate the use of the model to test genetic differences and interactions, a formal test between two F1s was performed, showing the expected 20% difference in AI. The proposed model allows a formal test of G×E and G×G, and reaffirms a previous finding that cis regulation is robust between environments." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Direct Testing for Allele-Specific Expression Differences Between Conditions | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RR4oBAYjKNRvL8G_LFEUWKXjLWUZMCq9e4SoXZpvBzY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488291","apath":"\/ggg\/8\/2\/447.atom","pisa":"ggg;8\/2\/447","ac":{"\/ggg\/8\/2\/447.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/447.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/447"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/447"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/447":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YV0XMBoW5mzs4oQ9vM2R8XhjOL7jwRtmTclb5h209lZGVF8m\\\\\\\/7fP6DLToc\\\\\\\/Egay4bd+s7hz6ONa5DI9cM+JE12\\\\\\\/0PLjkpZDAbPH2tZQKTbe2sSezR63P86DDc4DW9Mqs\\\\\\\/95KOvhYOv7tLaxZAiwjboZPSkoR4iEHJIKJdgRKB\\\\\\\/E\\\\\\\/wvGkDtwLl3RxMVbrZR2VM5Iwc\\\\\\\/gF80jMMyagFEAWMYPNyziaxf81Rif9A\\\\\\\/3AoRzSEYe559p7ImNrpAgK8fE3hFa4QDytpTT9lqFD\\\\\\\/ZGUaorrIysCDf5Nq1dr8Z8ckjY0Q6wiOZVfMuIm2mm1japIIyp5xnyk2+LOoBMOcqDLijW+EF\\\\\\\/BfnbC0rdUjMSio4m1rRRhobUEnPuuz4BIphARkDG7nWzpa70sbnZ0ySUV37HWbiqt+aO62qowzXCjUlen9dbLGF9sveA2gPCtooYqO6r2rH1\\\\\\\/g9OPKHUUEqrMsNIpJRwecg5+rpGo8MatEXl3\\\\\\\/GJ4ts+kq4oBEXT0DM4MBYSivtv8Xf16VD6S6ejVjz1vlmNHay5CFZ\\\\\\\/ZFhlk9qZcKTBdMSqEDrXFdOt3WV9PwMRqwUa7670V9MXdXBU\\\\\\\/uSCZWy9a9KVbLMuRPztaFIr1WQUsR3f+001AzDWt61MQS+TBNs2y2DT4W\\\\\\\/sDHT1PYXLRmrGs47guIxFgZxCMCFTJM\\\\\\\/+uKur8Xf\\\\\\\/H2oP8qGkGq3T\\\\\\\/9zpbkprCjIRRx2F\\\\\\\/JQCZYDApxRqyXWMJNGwWXXpT3TGuPePnT5NtoJ1+oO9FvuTFatsENUw0iljienWWF\\\\\\\/tYi2v13AGHyzpaTjHw+8+XU91dL4livCdBry5C+PoEJbftnXU80UIoWZCLWhFCqC0AAQNvsUt++ItctgmsB7qVW3K6t3Y4o9uj9bw4wvzxtUcauJ+qPMZ1qL3BuzWOlseILg67z54M87zmKdwuyRuNWBtzOjin5yZuTZuxueaR+Nr2IcXztyCQTliGAbL3gquMFl82u2EUB4EnBGItbm6fTVOi5nvjaXaCmT3CKqRJnM30RS2N4lrOz1SIwVZfOWiM+yGjgYL8BNpcPVORwzU4tW1c2kPlySuceq4Wf2teJFB6w7uQnTbJhG0zynjj821XMXPuC63Uws91ZWu8HLLwZCoMTYMyvYB4MTJi43y0OCPJfO4YZYpBFZNla1jWk0OoOk+ai+nZnIjs4v6gPj93cYM43nANtYRIXwWOEopyXVEzWQBQpO7\\\\\\\/IiR1zjT54O6kUbwyz2PokSUdHVorkkzXbZGyH1lfTFNX5D7nD3ra\\\u0022,\\\u0022iv\\\u0022:\\\u0022O4Q2IaNHNq+5ic\\\\\\\/tdzGcbA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e70b85ff340b56cb9169ba2f22407d2d2426f49f3e9ae4df11ccd0dda36d1056\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+gIiHcRQ\\\\\\\/x8DKYaYXxCEcTs+LcIitPOSJwiFWG\\\\\\\/tv7GO96S7dSEzAIBcbYGaHVBIU+8ZawtA8HQOrUSGaSXqjpKTkiHcvJp9DCZ1tx9lIPPYvKWyesYxGUjUF0DKgtt63\\\\\\\/In7CH90LnIgupy0UdlzCuFczo0PzyejvRNc0kO18\\\\\\\/d55+iVjNIN+I\\\\\\\/vxH5aK5fD+8LZTEaRmOAiwtislqqQjzMgcgkrvgwtEMi4WkABii8EjRphzz1nRosf68E8f\\\\\\\/AGy6\\\\\\\/NDsgeuphjMxNtTAF5LKew10u8cPsmRXiO\\\\\\\/oC7YwqmgS9DUXYxVOwsZjOmn4W3fpyYGT\\\\\\\/iHjvlofQsO9HMrMU7P5zOsB80INU\\\\\\\/5vXxQ32bjvogVb76xDRujG4c5\\\\\\\/O\\\\\\\/9NLp5YSjJtZbvwPyqD0He43lhWkkk6MebDFJUq7bL0Q0mJ6QwPbbzMMdLJ9ruNG\\\\\\\/CvbhCES6oqVNE0Iv+YQGYfZPhFpDEu80r6bLebzY4VCkaTU0hFWnIHOP8idH0suDcsdqiAOP7x\\\\\\\/GkQfGS6IDWoa\\\\\\\/PszBghc4HNQ5RL+wJCykGKWCEq0WpdmUPeyABeCJi4mb7K8hjky5rREnX5zbUWpAxS3GexNYn8RBFhvsad0afIvPHgk4RtFrvj3T+4vB4ZAm7B4q9LAtYmxirrpiJFWMCJKBbAB3kNbYg9WZldbJDELtxfFw7j7xvDFv4k\\\\\\\/nK3H\\\\\\\/bP73Ew7z5yp5bNHeEk7xpCXpXJRpYMMd7wPLn+hhRKMOAGhmCM2AOll9tfUvme0KdOyD8J0G3cXPdaPENNUIQasZYtG34hMFSdKUowWLJxX7R4HmtP\\\\\\\/jyJgTxo7iMZ3ZAQMDUqZ\\\\\\\/7TnzpX0B9hJvRCgUYG1hoPs5I5bkeBHbCOm289J8opySQl6gtBa2Oto53\\\\\\\/avrjcYt1jzVi5rJPIPiL+WOWANd9J+kE6jU1eqavkrB+\\\\\\\/P2aXFpo3IDYJf2vpJLkn0BrrtANmLcYGdJhh8xJFMJg6fPRuudXHKavT7rMj5fI5m39pLxVUS1RLZaiBA08FgZjiS76GdVHJqc9nZ4WyxS8Nz5fnGW+XuDah+UxkgeBYRuiUJM\\\\\\\/0vRrc\\\\\\\/gbWeJmg5jti9ZJUSsT1SXgnYqn99svAGmQf\\\\\\\/1U6r3fkHlfOehlyYcBoMkR+Ry0KrpG+0+yL1eQwv5ScvGjowNATN+iSUMJEMIlhg5uPMihTP5t\\\\\\\/bVLRhMBaBG8o\\\\\\\/Jst0mLTlZAio4LvV27fEUSJvs\\\\\\\/1gOtQ3JQYGN+H8FxEDuMtCyEr6ebOrbfF9E17udSOo20RG2sBygoQyTxm9ZHQcXRGP0duIFQYlycBR+Htr4HDELlKFNtEgQrZpBZTfDw7rVT1q+mAHlrp6zfAjMWpBOQOD1D2NZEKq59hXpJGqo1ZKOxClFDAgG0bkQPYBq19iJ\\\\\\\/Rozbj5W3RzLAI324V+fGp64e\\\\\\\/sDB2P0V7FP0wd+e+Vv92o2dxhJf1nrjE2tBFwOwCwxnb9sQOJe+D0KYe3tl\\\\\\\/FhUzY2ssZ2txmLjBiD5iYhgRVsY57\\\\\\\/x0dpuWc4Dyu1z2fRqeIGc4fvwKHAQEiQpkHDmEkCRbzIL5xPkjcMbtHt0uWMSzUqCMWoJADCCaFxGSBwdU101z4HyueutJPoQvMrCEhvPrIKWMDM16hPQXHhlyi7ccE\\\\\\\/uTxZZW9gCPaeAuZnYd9qGfTffhIpB\\\\\\\/VANUXc3P9JMxlN0EzaGWR1H7rEn6z+G19C4E2gWMW0ohoCfkfJLYHVKeydYyHJX+2\\\\\\\/xbiLVpI9sUQGEEojkqQS30USe1CfzOwAvmDIBo5cVbLfDjz\\\\\\\/7RsE\\\\\\\/raWKt8asjh9G2CVMVa1d7oBJzr2AzcvJHkxVLB97oIlXKyei5HvZGtDVOjCke1NGlt\\\\\\\/qli5wutrV6J9+FrzNDSyzWPeQjeFUDWPgPiBKeVyiZRMNryvhUYtlBlO3yJsfmh7Er785OFBZ9s6lb6oLcEgsV2NV2\\\\\\\/\\\\\\\/qTDdJK0HKcf\\\\\\\/oZT7aoE\\\\\\\/VdZ4ByguFzFs4uBEA+rBZsmnrR0UNxmmiBBPstAE+3I\\\\\\\/G1Aa9yya1wL63fH\\\\\\\/i2doCcMpQk8jEqeBQoQlp71\\\\\\\/EX8h5epMNAnD2m7P9X+AufMfgi7it\\\\\\\/S+ZAZO7cbUlvHd9WMckCEuKG+UYnwx4z9\\\\\\\/GwQmp3cPnHd\\\\\\\/hRGdz72FyRzXgawgJq++pFmZcjLGwOLYEtKAarDg3D9gZ14w1pMNmo0Caz2TcFcw4WRefUDi\\\\\\\/gR2iTN\\\\\\\/BaHCUHvMqy65cAkcssQeUqh+FSPp\\\\\\\/G2TMz3nXyOKTnFfsnIXaK2CPfQYXxPzkvY\\\\\\\/48l+HIwlUTZRZxsC5G7jrQPY\\\\\\\/0hLmor5l3DKY9\\\\\\\/ewvX4Bl7FpgIn5CB6lW7+RcbcHz5Zrzq4KpUnfoufBcOVp0+wWxXS1ZJPgmFiXvF4BlT6nErfCLVPetBeey\\\\\\\/8ZKO\\\\\\\/nFiV3PfuJyfXlvtOv\\\\\\\/uqEx+1D2tKnSqid\\\\\\\/lBDzU4TnoVebCUrDdlcICXnP2rtwYjI7uqiB6ZETc0zfS4VSox2p0F1v9FrTc79qQsXjDXBMCvwoHEs3KN5D1+52q33X3xX+JVcpRiLQfwvBgWTZUKbADrxG2yxsPte7PETi2zm45B6xk9NxDgnRAcPj16s\\\\\\\/gaUnAx0STk+ap37MByiAGX3WrK29Am5x5CyLrmygniYV4\\\\\\\/mFvoX+8TYScG49a7Fv3mO8Oy8l7lEpsw6TCNA+thHL85cfxD7HzcmvAOZZMLxCJVoDh7UfmhUXFDkt8cKgFtTlrjcrYPCAmlis4FNHt9reuC67PPJe7MxAnsIYhYISRIzeUJ1b8o+DYsJXK6YkzG12hJn9tPQQvU7MYOK\\\\\\\/6IK3XlW95XHeZrRvqKKNgzNQskhnFmEhK+co914iUH5QDIQqei\\\\\\\/4sPCRwiS6k0UjJWYRoWzqI3s+35IlfzXUXqUPE0vP8jtTTHTcfB9t2ydgyUStJOqby1c9rFildw29Q2f3S6h0BzclVX6HS+TQQu3uQhouXFrNZxHL+uO\\\\\\\/Galvyv82IRt+wpOPJqsTI\\\\\\\/pahK0u8AXzhzj8fhc10yTd22JhJCrq+f3qKRulhe5g6oxF\\\\\\\/hPwkIziE5Re5xUm3tvodrTCuEUXqThzTVET\\\\\\\/pyFWrTR5qzjGr\\\\\\\/Vf\\\\\\\/iGunBKE5SABpAaPwipogirawEhYg+CgGSvH+dH3bdjqwEtUwhWsaUHC4Ck9gzwq8Iwo+s9tZuYRDFNo4ZiHrTkqeQ+M+Dmz6Q\\\\\\\/BAG4YDD5vBIWYDMeRdpQbyYwb+sjCCLF6C7CSErm912idYNhF4IJ\\\\\\\/Necg3rkrt7FhxvoiiDWg3Ai5dr60g2bBMKLx5kZKTbwrz\\\\\\\/Na7P6tCkGpg9ceh6pndBYjM68DTpzgL3\\\\\\\/zYylgKr8JSzUustr5XjN3uKgK9B0bnZKW7URrIzxl6Hz0hRXku50xaw29EsCBMnGj1malIkwK3PNrQkLl0dMuhAlrx9oQE44\\\\\\\/Plg5UCtJzSGrQ\\\\\\\/MzFBvr2bDaeNxyuG8CzWw0BSdTlp0IRqZdiweEA0Zl6ln8toLiyvsmLD8OmeHPQolN954T42HR1F8mwYtRmj6fiTQ+RlZ\\\\\\\/5beHS4pGnDJDd7VSqHDKoT+Otg2jrZMg7V9pwe0wlSu7Qr8kHYA\\\\\\\/o9sm+hCrhE7oamtmqDRPVg15yjGBQ4LdcOQ3fSWzC6V5SqzoUVcRYybzNC\\\\\\\/ayBEcfi\\\\\\\/Ivgn6MJFOT7ecA+ulrq2KmZv9OT0lIyjPC5KzDGkzfDq5WbWLB4W5oPJGmp44m4Uvu1kMn4POsI8csGxoKOp2uLw7yCcom4ROzL1+Ql9jJA5WUU0HdUKZFz57dAvzJ\\\\\\\/E+aw0EI6h+POvHD6DZP389JpXOrr4MyNQsXaJz5YGmllCqN92FLzOk5nv\\\\\\\/YvsVFH7g1iFIo3LTCDjSKZYvD3FmP9x4SJUs5ilM82XgaZn\\\\\\\/2yRC749Ufqomu50IV5aQR\\\\\\\/NIPkRNhX7OTfLvqzdKQhCgfoOJQ5WKzpYQTaTDWRWr4DJx9XSRPWhL5k7ZqWyFx6ieF9he6PhoyViJ4FAq4dyhlK2brdQuikxOVJIYjO+KijOTboIgj1dqYe6Gyrxp8\\\\\\\/f5cyl7Qdr0gIVvrjNYEhhN5hIAcpin3afpmxV5qOZYgEk4scwctVuGJmhyt+bMOxAGL87KdSow80HOPB5JNaeHXVBT8FXt6bx35MftwqQBvz7yYUOpZEWI3spK+FJHl7ULG4eo0Rxe1bxjs0OfHZdaCT0+hC1yOuaIVlw\\\\\\\/fRP1cEgcESxY2trUo4pWsYew6BHh6aVqk66hlUuOckbQiIqWZ5JgnMm29nQrLZsBOOnEFjfzk2AqBCRR922T5C8yvM6qHsQsTh3sVv8lHjwvSVOFEpa0y\\\\\\\/v7p+FjN8yKNFEg+j\\\\\\\/D7Tg7QADVD6js6d82svVKuYWym7wAFt\\\\\\\/iKKSpYpU9+Dkoxdvw3IG0u8\\\\\\\/FCAwMtxK7mBJR6D9fKEGoZ8yVXLLHkOO99hniar5YeGkJ4NQs9NNKBJqY5SYyjUY4o\\\\\\\/WYSPzLJmZbF07La5ckaXvO4gp9S5ljP2SC7mHFHd2VoRckT1iv8bIqjdrZO2PBu2Lb\\\\\\\/OxfoAXF9If+XeLTm5OUPATzVGAQv7nkd8frG4yM\\\\\\\/ewVgfry6t+1qHsr81NTnN6wPU3GTBsVM3Ziq3woNI5a\\\\\\\/cn6x1nzdwSAPtWaTLPrT+WDoEWAivSRH+okYnxmIhZbMo2UjvRpto9ejLGi1CXTOqBz2Aa2VvGGSn3V6VVqRtDQ949hTk26hw5GUW0lEoYwuQf9m4vwuEa\\\\\\\/abXDfCNjNnxIQBW5AiNRSwrO6xoiNyNRbpS\\\\\\\/j8Odmk8WI4t+cFl5+RrreGr1naKrmZMquCST\\\\\\\/7TJlCNJnGTb9Cl\\\\\\\/p6rZWo9xsArE5uV95XyDRnFw04zySsaiupom9zP7j5WI0o60vE4s4AwFcCvJ0DzvSHXlVUlsAN2o5tjg+c+yWX8wqGmrMigHAW1vjoW\\\\\\\/CSvvkz5JQ1gMs43WA+K0F4TQtTclMNX8OLuCmGEW+wvHb02vBcpH69+CRL8ORUmy4GLHFAG5CWqTJWGEQSU\\\\\\\/PxIFB6K7gf\\\\\\\/+ZF3KbXtbsF\\\\\\\/KHqXBP9vgEIArgF0rTE9tplPhdLDepAxeWImwdgP9Q6XJKxRx9lYceWVddjUUzuFLFRVmxRjXDImY3UJWBTn8wfbj8klp8WU6SQBD7vds\\\\\\\/089\\\\\\\/81SZ6Zk5dluKdEt72DTu0ioXdWMLaRXTnohEB+fJYs35r0APZm2ZpguULOUY8Nav42owKo23SJ0aUiYprh4VcPxmGz+Oj7RsOG4AZWbqYgXP6HaygfFSb+4Qy\\\\\\\/92QKiPO+n3HC3wFth186XUzRfwBKzCNdTecQrqP+oIbvjpamADBQ66L2PiNKNkGEmeM25mn08k5lKZky\\\\\\\/ssT2fRJVbtEfJitgm7gEqn8\\\\\\\/\\\\\\\/Z6i4rYOaU9ldQp5BGeJqT7IWWsLMq9T\\\\\\\/dXnQe6idMcmQQN5N\\\\\\\/uEHZOj6ChOghL+Nm\\\\\\\/PRHrgombFxSP1gaCtsO4v7SqALjlWNcCDULNskAzw2x+gpqrtC0HN2rNjwIf\\\\\\\/p+q6zIS0n3AORoEib39PCYt20FPg0ptnCiTMRGg9WgUKngJI\\\\\\\/KGhf1lZnuDyqQNO7j5W4cSv2jzRNhd4OYZueXDIFd98XPV0dAzL2Es2wDWlchFijSbGeFECIPbkpzbUEbI+2Jh5TBBj\\\\\\\/KvvVze0YbHN0o70VVmMrsWHMG3JAFO7s4YRwo1M7X++GqdljuIwCTOlr+NwCmiJus8JYqi6nOBJfXLLIGnOL9SWCIksWsUGoaTpj8QBBjnjNY8h4cyRlRPX3iiYLmIGnH+Xh3bWcI4nTTrVEur8I5YN0TZVXMbTmBOHSJ1X9mQ\\\\\\\/7IhpfIc8UckflAVEeybu+aV5IbAYBeFdEuY0m9zK806wmFtquZ4jQYL63uobD0zkLv1Y6O6i+yKWU7ZyLj1ClnL4cjmmr7eG07KaHYl8ecL6pCbJOuMWMcUxCnIwcZVwPybxCVEV61Lh\\\\\\\/NgLj1sjA+bVHAhbFqN+8lKvgm5vtPYC\\\\\\\/yrDy\\\\\\\/f3xIdrQhosjWEPRlHO2hgUQtREXCOaJJW1ukxcQwVuQD7W0dUbmtwudwUfHpQGCwgbY0geHxweRl\\\\\\\/3S6gB5pYpKNADJz2J\\\\\\\/21cRxbjoGjoQaJ+Hc3MyOFGqcXkhP2WhvjeYoPHz9vRPaUZ3VaC6ppkDa7k\\\\\\\/9IrN27JX5yhoOBg6ot2HSGQcMrelhfh0xYX+8RySTtm2ShhcSK5i5FfsSLpTqWzpV++wfaQe1nw+fGXmhBHEEg2wkHAErztR6pTePHq4G2rCiih94FZZPecVLQ91CWvF3mk7dyjv5EOktFKh5yHLmC6a7+P6p\\\\\\\/PWgX8732lpY8sCAaiPaZl2PiuHW4Ogt5v8D4tk8pL13CZ+tuTguB1mfxgbq0FfZiTJYmZJRcdTdVq6tujnHIOni0FWq2W3xKVW\\\\\\\/dHV66kz7oEX4mPvabYs2ZStXT\\\\\\\/gSf1jrgAlfN84\\\\\\\/eUIMn6FsCPmZs3JG7yihadp0UqRJ4\\\\\\\/Go1K6cWFv8xca0iGVGvV\\\\\\\/JOUdZZF8Ct0ni3NA5M7pQohWFfOmItoUkqVaB\\\\\\\/4bxYuMPMic90WGNLstpTIBsq\\\\\\\/Vmdwb3SbbFgTxZzSP6RcxcnZ13NjXY5VsC02bRMv0qkIUwlrr\\\\\\\/eY13geayXFdLNoYRPrjfto3DljqV3p9cbHVWcaJMJy9Oufha0ijaXYe9Wkf5wc3Skbr4MMmmTQr+jD\\\\\\\/wU\\\\\\\/mPweYw7qDsKx1BHsJ3yD5D8NntlU8GWczFhfwguKmKNynVr6AvddFnFvS\\\\\\\/C42OB6vLTNY+27uX+yP41aPVq5YI5DD8qBLbTwwjdKO8uCPSxCAn6UIg6fGWTAlR\\\\\\\/6a65OOiMcuBSMvQxZpgE6LPYdZjsYbsLaRPeFg8FtUt+QvwPCXDS12g6\\\\\\\/77kaEiYbTK6CbY\\\\\\\/kAC7Yd6331NLAq5LTv6EPPd2NBRqR6SYeUNo3SufHN93+99\\\\\\\/\\\\\\\/ODgtZY2uHd7RI1qTMFZuHcLqU2hdNBADJwm\\\\\\\/O+kq0ud9+LTn0Azy60uoOFXFhLaXPeMEMuXVCj3zr2vheOqvSEolxllYmjO61JDza7vSopfSeVQZ6AkNqkdIMjxzf6xPoNoU3exZdzHRWyJX6pJ+NvFd25A4rIUlC+UdiLMQECWbSFS4gayuxjkAVn5sBTKsb7d710lji6mSGCgZUdcrpGnLCWsGl\\\\\\\/CLNe1BitQVvjH2mtI4oiW87z5buPfc+FKc7xWipfUDablOZkEvihGT4lyNI7nB+xhc3eIP657j0qLB6IdO9YXF2nHvihds6rfqAI86tZWcSn3+RNqEGEb7+4opkWMNh7fDZ5dsxzPp2QM9Pk2+q5WdvOJvVZPVydsnXFihzkh+tDiEl\\\\\\\/aDcfr6ISJFr4Xg68Pt5ZNqkY\\\\\\\/DeuATItvaLzoa95jFNydQDrMo1iNs45HQflud+I0ZPSOV6wTguV3BRwngB\\\\\\\/4km9mYqCWbZwAH5OJwnLdYhHcpqEGgfdLL0TgamY7cZslNAGGkxt\\\\\\\/NcNjYK1K0RpYukmQBZ9SQT6IFXrCgTXHrPhALxS+Y4YtPk+YeP\\\\\\\/adO8dykUsJbhh1u+llckys4HN0Hsi7sEUgQFH4Ie8+QBNc8a5qbfhK+5rERKGCPhlpbS3i3RGAqy9n30ypYIhjzqjiP7eSTZoiuc0Mdfsc4uBv7Drm8swmrKl8pA2nPH1Fbw\\\\\\\/TcYP28Cp2J7BK8T0R\\\\\\\/hgpdFAkDqfnVXdRxiDyJt7W1tDO0CIYjDFkqvsNXcjp1m6zmuwof7\\\\\\\/xgWG828jqYZVJ83\\\\\\\/nMiWDs7Sicb0RzmGrBzpVli4o1W6NMdg+2OemAfFaTDvHcJuITbuDO9FkzNmx0Ni2C25ygpBEaEqcy8QU8QUl\\\\\\\/UGVWGnaWIbOE0fIWg2EmTTlJgsEFR1jdVN1mz6NmP4GAoSaTKLOnqyi75hLPD+bsaaNxAJY7KIdxGjXdz0PwuySWBwUy0Rr7aJlHGmkyu7RrCC+uaJfs82KJGvod\\\\\\\/CAVpdZhvhFaCQNWdlCF09VkZy4zfzUOZAEvSZM3WCH4pKsefaNAR\\\\\\\/RHz0MrGffiYfB6aPQUrrDpLqbLvoWe1ly0dFhmTXffR9vH00NHzaWWrfTahC1y9RjODPnCsmL+IIgkfuagSi9aDXP1kx8KzJ6cOowYKKd6zFNhgYHHalqCEbyGpZDHKDf5hbofso2aNw7TGQchPhd1nY+rikvqm2x1pqXtL4MXmZTynXRMmYk\\\\\\\/d8CkdXkAJOIF8ADUZbeMsLyIXKEFmAPpI7z0I1Olu2Fc39D829CcEjXFBjHWfXcPQBa6mEjHjOiJZDo6\\\\\\\/pWu2aSffxPWjZgcow8DLFrQlmCqAB+u8BY6ls9uxKsQaNmZ\\\\\\\/8HkzTDaJaDogVDE+jY7hardx1PBE4NGPv4DXziOePkEtKBQYGWO0ZH4kuW7FW+pC3cUxzbEmu1eaxYwJw9TkteKlKzl3XxWmo48zCI9\\\\\\\/05EZSPmzQZa8apc120YNn7kPOp3J3JCpLselHRpSz3b+aggDZAV4vqvKUOAWNeQU2SdFeLbF6uFEERGmVakoiEMuPlN5Ij1LDhctoCBgyuP+3jWJe3FKM774wh6NqZMdYVrDnv0uO5dTNv0P4Ln6W8\\\\\\\/dy1XCd1e9qQlpIXYmmmBCKmmoVDn6jTYr6bDVsALskpRafmnMzwWpVyz5C7nu95S6q1BvQvS2sAB2xVAmts40A0ThQEqI8HGFVxmaUkGdqHFVRnjIISxVMiqMqVIj8yTE\\\\\\\/kt9gjrWURVyw\\\\\\\/w3MAcy9lIm\\\\\\\/mVijHZFAsxyfUCxOwMNagBm3DUYMfXGGqCIM43woebMf6hYp8Su2Xbi2ItT2ULPmKqd5e6Hopt3mBgZhFHmc7NP4g5UZk4PajJ2wynBbkPeon1dZIE3WDN+ySMa2YJ9yYbw9hhSnXUveSVK2u0rwZMfIr4E3mDLrLDDv\\\\\\\/gHwhTHwETKEBmxpDZIaHHVOpq3jH7qFi9ZR0hgSqVs0rKHRRQIGnrwNRm5x0fa6vMdKAn4c\\\\\\\/U1WQE74Z2+Mv+XUCHaaw\\\\\\\/NiRUFJYlBwulgkGEyFJZGwHBfPGtruarxriDr4E2uMTzV0\\\\\\\/V9e3N3CCtGsmen3zyJYv8dcPA8Ni+we98yBfm5XnzBeMTxGYlZ5OnhyoCjmeK8sAYKSbVc\\\\\\\/jFp5x2YPdyxQrxXxpzORc0izDWyH1TgCjRc1cvVNiFOBKVvEzwWQJGRkvDiy9A81JgntWQvfKUihqVwTTKobp1cIUzp6slORtU+KulJtOjfWRWRZ2ogqkt1XdWx4sp2Ko\\\\\\\/IcXT6ByWWcdFoyLEm7eCfBHdXL4obNfqJpANse5ZZSpxNzq1rPYv8nyOWESDQQocBfgC0zKHzCnyrDnMnNSYCsTI7d3xtrtw+mkgWwTANdf+l+b2NYkcK3KxErhuo4iaV4IfYwx00Y3MvHV2S+WReduQA3QiZZ3IJ8HCJ1bwbVtHdjBwR4R2rr1BJaKHqeiCD1t\\\\\\\/NfrdtWfWcbpMSCQZI0UDUnczSYphwRw9fuNocczDqQ6D1ZoJL\\\\\\\/Q5Q8vUIUELgKdcFWBMFC2X99vjyLgRDjwyfgrEvMeYa7d5rnE61ax1VfJnveRzwRM7CgjhAyWgj8hivpLl\\\\\\\/T5Zctk\\\\\\\/lbGmQ17Y971cdJCw7T7QnbYLnCqY6jAuqPGIyYU+pAgmwpOvqx0+AUcAIpy5J6PG6H7mUVj4BxPSOoihZPO282vPPIEOd6dH7gMDYn5nIMqNiZopvvxPEpsmzzQPgdaVbf3GucDvWShkrR4hqXjri2dlmxp0An0vfjJIpBfXDxfiDlAaN6cF+D3\\\\\\\/DWvd2zfKOfJj\\\\\\\/KCevTJn9j9pAh9PyGd1QuTWNrWpJXfGv8maAXKcf+6Y4OXTJSTR2SVt2uLtzP+pdRrznnN2vTrcOasHcP7pIkk40cLqlIOM4NB5HBYFFV7O64CPXm63Hoa5dyZKkziW+cG1YIBqeTOBsqylUMuYDNgwCL9pAkIkyk4Od2SUVHDAt+YuIfcUf5naa8dJHwaEIjCMpmkJRu57ifci9VpDHUySdYcI5bmxStzL1bPgM7vW1EYSVs4N5kjiKgHSLN8Np8MZHzS+UvcACENKPcbtluJkpMQdBDfVQECvYEvorVdnqv9IjqxjRjX1jjgyZQN5Z75aclBJHSWxAlZ4611W7Wae5MrfY8njDiM8MUJWZxLq0SlysO8ejSIo\\\\\\\/GxqGNiEBVEEFAQRZJq0atwoX9PRXr4UUB9soNXrG4TtCkZEGQpPEUuG47ED0f9DJvA+B9Yvf9Npk2kbcpDooXBAAvbY8nEwVlheuUecZ+hSLxVhhgmbpFaUAbX7OULRdIW39\\\\\\\/k4RaJMB8\\\\\\\/lmCgVb2hr9rP7EFZGYwcPfXdIEP\\\\\\\/7sORVrUFRsOPI9oKNvLedCzY9Tyqa1S0QexzJMJ5ZHpv7ZRgzT7tGwu2+3X6\\\\\\\/ahkovOkjFdA8dLgt3gmsAz+PKY1HdoM5dpoazsF4RriafkSlXcH7ok4cmvsIRMp7A\\\\\\\/Pl1kwmhW1GG\\\\\\\/mvbL7HaxManYFu3XQJCMfXPJHxoYqrN5huNcI9ajuGpEVgWnUg\\\\\\\/FfP40qQ88ksvu1MLysm5X0g0KcThRIESjO2m+2OQ4gwoAQGWhCSt8FWr0UKs59Jb9VWZl8plp01oo1\\\\\\\/x7cIVqSHXmDvMKYes5GuPdfwoxydp7meMls4+6tRflGd9\\\\\\\/XKuPXHBLQAbKdYF5j1hFHc6YofPFJ2IaPK4V\\\\\\\/7pIuIfQ93\\\\\\\/cpwGDO8TeiWVlkDXcsysZX7vu6JhQ14NjOdp86dMsHOvOEPksjJDLkdu7nDVraPNaH+X2a5q96AooU9BpIMCKCGJBExGRexg9WzedJS5QB5wyIZzpLNykyEujGw7CFar1buZJSHO5V53duCY1\\\\\\\/BveZA0ZzvXN8kgzw6w2X7ZPimNFTkAVjcHd+NNzumU4f8tom4ZmwMLoV1P+f4M16diaiYIza\\\\\\\/IksyUg2xN6rlDComXGPU1bS8rAz5Nss+nilOLcp1bVpx8s3CMJIldSkUly8hSA4kK0A0V\\\\\\\/tW+VBTC6COkLRHKdj51OrV7fQq0XtaTl43W0Ek2DiYdUUEqWPTsCzvP7GvoBUPl3fZRjOW4Jy8VSQksunTsdNGR0tXjTP5VCQkMja2AOFG2dwSlbAREYc\\\\\\\/h6ohaylAzSBQZqCPJsxwsYJ1HEzzL3FLBkplOdU+CdYmM0Wb90dGNNs7MZ2Tqmqjb4D0Z6Oh1q1MBgAH4eKNJx0la1Pyy196HYDobTRxpxswSYZ8zh72eZw3\\\\\\\/YrA0mTU1\\\\\\\/IEXM19oEDvPUzbq0qMR1XjaRThQeKyI42Y1LAvPgYQ5GrO5xm4\\\\\\\/s6+4t\\\\\\\/RICN2up4faqu58CzVFM62RB45yVydn5aSjjpwW9whu+JLixXs1kBmR0bzb5DX2HgLpUdWYEwZrRsCSAEfnPLZ6Pfb6ToK8d1Y4nnA7KqBO\\\\\\\/yhEHss7Bj0bzgFmTfWHhKbPY\\\\\\\/vkwJQVEjyex7meT2o\\\\\\\/0jP9jOGgnm+bLdu1lB4m7tjPJ8mobDBIQzdx7OCfK7uQZGCiXm55mJsGl0KlJt0ReuGVOE9D2P1tIocHy22nl00eQM\\\\\\\/MeiTYqZQNoRDGGw1UbWkJSx5vZXDk1TpYj+yJCtoWbhX1ah+KWgjqOwZcFzWmTgjxEk61g3PBYzxi81MEVj2+ocZq9jfzPOZqAvw4ej+9naj8AWP84dqCcgEmwkic7x76SX3RgVVfhtvRGhrrQ2wS8U8TMlJzDDc6csmExMyOI6LtekvVZPJ9sdtjEatbY3XgaFucm8pX27iXnCPNnWArNJxbUZhQEX+RM8Q3QD+DtFe\\\\\\\/wO65TDIFLpOhOzqTFr3dO9+jWzRK1KVRkjBg5u4QTd+\\\\\\\/QQmeCdepErQtH7pSZBje0\\\\\\\/5WTBV3SO4c+cZNGubQvpzYsJeEY3oAspY8oMT2IvpxYgmzjf8InVzXLffC9yBlATJ4F5IlHLXWLrtwKJydYNBRnq8\\\\\\\/KSbPH+S7xeUYZcALLsJ0gkKU6wzh+0hgvmCj47V\\\\\\\/Sgv5\\\\\\\/uwN5W0SuKwaSvLZ7k6k6Llf9tw5jzyMtlVbrlhFcxwOMTY3MVefjJXIjtWgyUQP1eLN+9VX\\\\\\\/fy4Oh7X227BiUFcXS2w9J59Rogx0O2qqRuTlBERX6K5nAK4WoMPokJBFfOka4xUlkn4YASI6AIJWPTCU7eTmjaOt5F1dt2JpzhSih137qZOF1EMpxpcpvKq9n2zGW6hc+KzhqUFmIFOYrjyt4vBs1hQPIyITSYUJO1XjB+wXVw6dH5M2tslgvXgoyrvhcuhbeRFxTLFPDpjgE6sympGciWC+nqvQmw2YXmjn9honvOdIO4J25VRKC6eGRhTFDlbWXbbZjAJ0skyrCejmgeqi3WoVtn66QiDrI4fDElSwVXjkwdOi1P8jGdhfIhunMd6k3\\\\\\\/+te+\\\\\\\/hsfAdtprS+VTvqI6+9CgxtgE072z5wT9uAJEeJQHwBoAdeOvHbv\\\\\\\/\\\\\\\/ihVm2IPujWQVBe7I7VvWlFwAYl84ENxxfGka9wZu\\\\\\\/c635WdlfceneVlxHOhUyt+GxlT2qCP8nCHiYjId3cJoDLxXSESg5JAi8OKo7h\\\\\\\/TWTPNuA2N21HrJPna8Oer6FTBFaMu\\\\\\\/tC+gyr6NSeITbcsou\\\\\\\/\\\\\\\/acm2LUJO9U69oUc+KFLyYEkdBwkacOVFr45R6mErdWZk8GXFGGKoJgQrfInbK9Dx1kdlNKVDXRGCBTO1C+IVKWztjpK2BZ+39a4PlCc6ZWIJTJX76MVr2kAZsAmcoTc4kYVoJjqL19RN\\\\\\\/a3ms2PI+uJ243uKw+8q+sJHyA178tjCgkb85Ti8UKHwJty9w4EmoqX9i1XQLeV7e2gKn5JrQGEWrH\\\\\\\/qHvBYMovrKPnHrFTY0bAKnBXCXPqUzFHyKHNhgicf0GxqBucmd9eXZ4l5RrqAbd+f85klmhBboBaUNpYBj7YmW9OaYWT+G+QOWz2wEvabqnuJ9qbIScLzxt+r0ROIQ1mJDmQURuxpfl8UiNbesAVgNMGU1wNSrI0Bf4Q4k5LjuOXA\\\\\\\/pqPPwQ4aus\\\\\\\/ZG0JNcQPsfRYEEFZK53Q3sly2HB9WoJxiTW91TbgWBBA2NfVEjky5sw0R0tKwvaFYkPS6mt0vJtK+P8QccoxK4VLlUw3WqTyxRXwDjDc6Ueb1g8iSaktcYfe44wOGqPShGtmiGQcWAg1GmeafX0wKtPyCT2yFnv131LUFrIZBQuiy5z\\\\\\\/9xH9wMP\\\\\\\/nQYjJoZalqD0x\\\\\\\/s3e0\\\\\\\/HxS7FUDMQD8n\\\\\\\/4JgdaJUByxNW7Z5pANBgayCBV0KcyJwbUptQ3HMPgnRGOVwXIMtBvkCk+OPlU93hEAtYixts4TFdFMTOs\\\\\\\/oSrRjbTPbJ8ekGje04kRsjBsud221yvymAl6onCyNgW1DBX1c8Wf65cMrv3C4YQQyUwV9R2sApWKAEvPekC6yIAGWx4SfjV2zF7mlFtjtIUYQh\\\\\\\/0O4EUnH\\\\\\\/QDvhVcorIQxeIjDkLgpmpWMv7yHJJyYyk3M61lnyD1AbEh8LbOk+s\\\\\\\/O9BUhj4YbYj3ADAJtg2uBNEdtrAkQu9b3ufqCFGvxMry0OxjoUK7EBIZ6YJgGsT85VZ05xGYUou3QRZ0uTpa5ce8\\\\\\\/Wa8RqTlnhY6qRGl3obOrmyenZfGU7Exhjqww7J8WNSiKzeFDDbOcNJ67EWdzVLIZQcI1NihSaGw6uhWxsaoyC1tBdGq\\\\\\\/DWVimYBgn1rW9IDsyh5PcRRgwhJe8JHnEClcqkvbif+zOLk1IQv1KjylL\\\\\\\/2sdMJazW7Q\\\\\\\/Potq+j8A40ZFbrNaruoUH9BF4nmKbb+Jt96zcL\\\\\\\/uc98pLvnW0fZM604wZ0PzM2Cak36YBHA3\\\\\\\/RPD21MLzAfdUCvPPLmHQO33OMTQ7nBIdgOGNtkHdN+l9plt8DKyX4OUwpIZjJO33Ar3XVW1WMFVlEjVx4oXbWiOrfI\\\\\\\/KmuVb19LNyX1RspODrZtSOWLZXNCxGv18AxFfFaV4Z75MEaFSEdTcRDduk8fF\\\\\\\/0tpUmnNJZaeE5jLvy6A85ligcBN3jawzZR1lca4k6fs4Hw0UidJW4gdjLtn7COhsK7dn7t2zB3mnjxaRda4izBcFX9v2J+YWwqLBYZeDfnxzTseCZVLMhw6RLAvRypC5+zxxDKnuvqnnkDOx1bgnJvtm\\\\\\\/dLMiLNnhOsBMO81biNwZEzTjum4\\\\\\\/Dzytc0M3agLZUPOqwRbSBFe+nHfeTXpinADLQxxN8NazD+WgImFTnYdx1rP0WG3fR2gk7rdhvdPJViuZqIOG+hgmJfdzBrF1UzDaBN+3hK\\\\\\\/1qoPAeor\\\\\\\/G7j3JflHjlEtBvs+c59FyOf5I7cTHG8AuW5u9BOLetUcqSVCuG2YfM1Yy3dQ3MGD9p\\\\\\\/JanNE6DmdLzv\\\\\\\/E1kf2KotPJ+3hKz\\\\\\\/Ja0uEdJcqR9FEGYpBmJE5JOH7GT28DQY\\\\\\\/rSF7d0+OORotTRd0yalPLhL\\\\\\\/Bp7EasTEDFEsFjJZYYyO2ywAWF5n6I7AUajBmQL6zI\\\\\\\/VoKJqljlYTaSZ5d+3oew9SxTKDODeF+0Qv\\\\\\\/TtrHkK7+q\\\\\\\/nl05behzDZzUSreEWuirj44rZ6\\\\\\\/v\\\\\\\/j6kAkuZBBC0PBzhiZRrdp7fpXyknBc\\\\\\\/gthhA3qbC3YPW31jrwF4Wa47DkNn5ojbpRyVp4ZGwUhJS40Yrk5eZfLEcyqGbtLosNdYTTuk7phfJdzOlGeBEBVzRKqT9BeRV6xsYb566UAhLLuzj9SRIF5QJUmmEWm7rpwbDedE+OV7+VBYSyrQ3ql8jUvQV1t+GapeAMhdR3JgeZ5\\\\\\\/Vd\\\\\\\/rxdZD\\\\\\\/ffk7gNYgxIUjr5z52OvSI25KBxx\\\\\\\/DV53s11vHJHVAyt+fXh2WNxXWOSn3n7b85EJ+fxwVSixJTT3LFTfFCm+b6GVzxZpZ+wdc0lMDbTXib9OM+3nIzYVE0Z3WsSArnak6DdRm25e7fJrUAWWsfk8BW18Mr8IlJ9XQQPsKOBAUHfPJxKPEM\\\\\\\/1ReMFTujtmtUaeCzlfbAauk3ugJvDdcDkDAPXZ2\\\\\\\/1JsE26ZBr9rs48kLf4WBPWO4H1izWvfiS1SXQz+Tguo4oQ8KCiN7CJpR2JBtaU0LNrEcUpJnvAh7ddki2KzGW7aWZz6p+8axAYDz5VOp64GnrtjTRu8o+e7UJuQpm6wXYQOAp6+Y4uOWC7CH0JRDZIUv+W4qz9bGD3u01ZQMlp6aWqe5G1ZSfKcaxKc5fsswPXeDiUW+rVPA7YafJe72WuZv3ecaF0prlfWt\\\\\\\/PEnjeJhl62E\\\\\\\/S3Z2+Xnkx+CWjqyHpQMTIMJ2J3b\\\\\\\/IjotOaXCBzOVbuJbCf\\\\\\\/ydZspTZcA6vQ854ewx4XvuvQ1wzcwUuPX6NNukwyriTB439XMhOdw+3qbzoPEiiiGH5jmhJZvuH9RwrM9k+K2num7Jll7Bxoh53o3uBO7OcbCq98EkiI466dmYiB+PJbOseonS2r2xH+r\\\\\\\/2bPOd\\\\\\\/lCslo7wjDZsgr+DqR5WOWKWej8+9CpZWaM1DYDNdEFA1DCiTd8vSomgJ72HCAVl+YLWKv7KVWlXzt1DYPm8V9DqK2ULRVTa0HldEA4x\\\\\\\/hNnBBGAbgJTBqy2pCEL+C\\\\\\\/Tr0QwflpxFhMiMSOg9kU9mlDpxieM+gFbIOpmgGAkOU6OwwLq+wxLmNOU6m05I6JoxMaqByBEIrqW5YlZTpGtRYfgXlNMqSDoUb7fdAzKFzeZOchNnImBSgXGw\\\\\\\/P5jMxgbPb1tD6qAMlKbKlviTp2L5NVKvCeuYS3mBTqtp9Akv66Cfgydm3Xi5V\\\\\\\/f3fYnnpP07UlndvEYcdY0eTH8G0C9j18shW1Lwfm4+iIxkoulcVPT4MTKgeyJ5wROuqUENuKbffNzkXZPLtsiUqHbKP73rbLM5qAmbnsP6fsgvNv4bw+8fBYWDh094men+TE2vZkToWrjdAF+mgu0I\\\\\\\/7wEUjfWJIAHUhdo0AyW\\\\\\\/+l0XkDEGQrRwwjSeKoIrfHqL6i\\\\\\\/+KWSIGYzvpjxzKO46bvNaNaptOhu8Vw989idN8YY7U7RP8GGC+YRSgxJQmP+wFLDkyRacYPB474IhT+gnD2\\\\\\\/QaMPkiB7ihCAC1Mu86tpOLEz2dPWCv5lQTnz\\\\\\\/wxL7tzUkt\\\\\\\/+lpqKrsyntMhTnOTyPcCDF6+Ff\\\\\\\/Lz+1RaYx51VZWDkmgSUfIShFXvtAp454soveJwh87CaMIhVdwDSPXNM82X4r6uu876WF6oGw7gE1rbnPgcTdjJKotXpx0FzJs5vEPD7zr1OwwIOl3yKTrJ\\\\\\\/AbuCK8UbjoqOmSJi\\\\\\\/KunE5Dn7gAA\\\\\\\/YQJZtP+\\\\\\\/Z1t+FIHD4Nak5GT4CngGrQ+AG5Kdq8uoH1VCKOEeb5nTTiP6B7TY8t15t3x8QXlfH5JYAg3ntfnjmaS3LM45AKLxo6jN6NmHHAd6q9u\\\\\\\/ZuhmwDUXPcm0TCKbtB+LpSQIeJAVLJWNB\\\\\\\/B\\\\\\\/rGVaKQDiJysIYAy3dMYSULqnePSE1xfQ6PPWWkujReGxAwfvv8woluaNrhP2up0qT0TOsoucYLNPwMMMWGayUh977cUPdm2Zd8d+11LAr7\\\\\\\/H1D3i08vwwMqOH74Zysk\\\\\\\/YumxY2Y4B7tJnhrssbzzc\\\\\\\/Bv2ephiAjEGrm5En7Q\\\\\\\/PptjLKgQWJ3oJG8W8yrGwWAnVrAJBKAOYnEfhitSCYv+A2BtCmi1FRtZCTJ4gs1gt2SBiq2H\\\\\\\/13qiJTNUUUh8Mn\\\\\\\/SUWgJOQtwSSO384Fw9CKWefd\\\\\\\/eVw7Udv5JG5HZR\\\\\\\/LYlSaWsXZPAwZHZIf7Ks+ILJtlvamTONfCw2kSrPSEvm1u7p0jgCNlO1drsIGl+DIwL9JQQJsVNSMd34NhN8x88ARuZNjzTAOp2PL34hlz6f8gXK9VYBX3yfbgKXjDmo2Nu8ACeF7P6BwUGlz7PiC4CIpMw7CcXFneBE\\\\\\\/8O2maW7RG0KJy2kiBqoJFOfbT9f3CRAiGqztcyfR+Tyakc\\\\\\\/lib8Erl9k\\\\\\\/YwBNoOoi345e9OUUgETdCg07sRz0rGfCRkGzL\\\\\\\/6XjB2xqTJORE71q9JKYMcK4tRDciGCGoIESljM25+OVusOji5S5QTAFU9o\\\\\\\/QzVaoQMkMT9Mb4g3qlq2sQ0SUFgN+faXaELW7x09HOKY9ha0w+CCo8PIdOQs5oHgJpq9lbQOHo0mg3lQ4ja4OExGR6f2AKNtQGKBdSzZoLzmDfnaRDN9Ob8I9Hs7TMza44mie+PSXDll8sBbKCqLJ\\\\\\\/qlVEsrpSoiU22NXaYV1MLZ2d8Rt9yY5julop7uQU+dMlYCTO32dIvfVghXfbt7o9MTSK0FLAIyJ9sTIWc0hoAJGR0mifsuB8NRGW4kTgWeRIYDyuwUwaL4BWWvWhrrLa5VNfUpOfs9mPUox715WzMw94dl4q2mGurEYK\\\\\\\/0oimSp\\\\\\\/GuXqvp0sGNoTiNzDucCNAWl\\\\\\\/5W\\\\\\\/httvDWYsGcyA1jcE+vvqedZvPjEXbH0OD1XLEpW5ui+VwrseH5PfwdFojoWU93kmcnuiTya7FuX8jN5vYkB+4RkT87K+nLR88RXntAJDxB9+hw8WtI\\\\\\\/2zLLtyO7joKVf4TZRFFR8sTTTPXuXXQRZ+MVu6yUX4+LffiNMoSDLKpWziChmpSZUIuVCPP0J7JauPx2Af3FtWMPCU41fE7CX0+SaUoLqllKg\\\\\\\/fF9mH4BD\\\\\\\/wWF8EOS\\\\\\\/GOs4I255et6LUu38BMYoUuEHWIAeEQBZqk\\\\\\\/C+Ws7m5Lc5UNGkofAE7HVR\\\\\\\/xYQ5uMHOEy+QgVUXtBvr6EWzOAEj6V26n3AmkHxUcPsKkapN5EWcQwC\\\\\\\/bnyhIpJEP5wXo8OBKZrG65+vmLFRd1owqsNq\\\\\\\/\\\\\\\/012hZNr0YNjk\\\\\\\/DmcHcLWOfbsIbb0mF2+Ive6uEAmQGNzSc2c7WWBSoWCdqZhq4W6VVQBcOqqt+LBziLgpmjlnsUCnQdqFbJ74iGS9R+NcAXtZ3jLuJiTLdSgm89MCjjcmNip7JuN\\\\\\\/sV8BaCy9zkAzleP0NSMORecjcWu+GwqVjFPi4UTlKbTohLhaGE3992ncv29Wp2Yk+B3X1w4NcujfcWlKHi3psz+\\\\\\\/j9dkz5QG5ruu7piBrl+Pv+ZA2NtGm7I+QmyimBR5S+9iNbO8yy+jbySUvI4bvzbvI0cKDATC5fOQIbniUNWfB3hHzEzPtH2HmuUBQex7EHMqIYM5zXh0+SnL4pMqbeHHCY9KVPFub8kD1Rew6ygO8NetebGP+1A+17dXust6frbhNUPH+XJAMfJWLZDzrobgGvey1w3FdL1bj3+SLB9b8qej1f3wg2eAgHMaI3k32M5bTlCvFmxrbC5ebrLnuT1a2ogRto7BVLwkKo5CcWLAaMvK3pZ\\\\\\\/N0iioVHqmi7GogAfAcogU\\\\\\\/OaZfupyZPaPcIww1Kiqt8jYh2IvUghnYY6clpUkt4Yywr59QHVJGmq5Wp3R8wwPXhoSBL9pVjDZqVWwpkVATNdjxnnbuCS7JI1h0ZQ9agY8GMfMdV0Ku1xx3uGp6szALw1S9DEDXWLVWOeqhIztlPxo7NZ\\\\\\\/3Sx1zIfLPgbzJgr4+yTebNV2CdpiKS0D8MXMbpo4ohuFEALjvG\\\\\\\/dKcDsrKHEIaJ8aG6iLHyql5ouNXXrXIsuTYrSlz8GV4MtGeb4WMjc6I3IwfHi7ORMKrzNZJo4OOqKh4RCTTrhvX3kOoWP0NOy7GLtxiSprVeCB6qwCNytNFVr82mzpu3a3WsTFVNx+FdjJqxCUxi4\\\\\\\/afcqQWc1XfZlrcZUU8xStEfhQumnreaW0Er6I0+dIaBQQXWTaoxRYUr8bmHx3sEB+hpVvHXjlCzSSmjuSs6I+1J8KTnMuy57aqMUsi8aiIdBnuXc9ozbFfZD+RZUEf7Z\\\\\\\/P7AT8qxFpCQm1XdQowqNRmWeQIb2XqKZeRNfCXI7TLZQSg0FgTkbUCcF2sZQ8nrtZZRsaZz00JrjUVUgQWmcPMCOXIRYvmvTdJ3twIc8A83j\\\\\\\/5e2esnjouJBPNvJRNpwlVQB\\\\\\\/FHRPQF44YLtVO7uyz43xI79vygaQh3289ZxCq9wa5x9KQxYGNqeG3KLRwPRo0UBxyYRU0knW3PoBVeafjvkhljWXbkVES\\\\\\\/tQv5rH8iGIA1PI6V5SLqOArZHSRNWW3Lv3xYG7zLElFbNXjcbW4JSN2ac1T6++c\\\\\\\/A9TXCQKtzus+F0hP98wD6VsaAchZSAksVivtzcQz3jBBba\\\\\\\/tC63cl3HpnKXgetrFMOWELkuPnmcjf\\\\\\\/\\\\\\\/rZY8PdbUrK+bdXLJEzzTVGm6Mc\\\\\\\/wnG+Dr6utZn8\\\\\\\/8IL0MX0XZMGn0K\\\\\\\/v30CURSYt3JV3xl51wxnZaEsc8Ydw59Gv8WQF9NJufTh4itTiHWvwWtjemK3L7F1i8Bqb3Oy+ztzpnofvU5PWW8YJohhvx295lPLIFGkE41JD0+5js5w8j0r1AvZ8wE4wOR2aDzL5E1iHzu0n3ur31yYHp4+J9CFIx4Z4waDuuXtlxTqrom7ttZG3Y0mAJe65E2\\\\\\\/WeNTkFoF0AvRehhM8pIuyYbHA6A8eJdEqd+Uvhkk7jk9BWDUubijImIVdIJLuLaDEk6rfHzXii2eJ4bJcC+QBa3WOajNwZ1S\\\\\\\/zZaD37EdyAZzF3agugylj6gzA53L5GvjUR677qs6UdbAQm0NN93KsZ98Gid+i5LBzdHDUOVzmlnToRJkoxOf3SFWWslPLdo\\\\\\\/cVAE9UCin1PBeNcjK0PHfy3r8iGJObHtOYjqNcvnLDrM+3tUKednH7uaol0EGQ6Er+rn9J\\\\\\\/XshLe44na\\\\\\\/\\\\\\\/iRqsUcwq3KFPFPU38QgiUW4r6kqrHQEmQJEJBSV6zFLnRVVgfedj\\\\\\\/iCU5CKn5lDm6MuqAp+GgA5q5toAmPMGhu4dTTUOWaygMEfMJ9tnFvPsh2XKJo6ikE321o06NdgTP3NpqTQB6WvuRxybsMiOErK49zLxnv+OdFC\\\\\\\/6AcAIeydcVi56GrJcKkS5hc0x3ThcqYzGpwehHcbGb7UgUSzq24Fx2NclDVXMrgyA2FrUc64Y0qfcJ6WYF0fzo1DrGx9d3x\\\\\\\/hbfDR4gJPBuvSWTcLjHHSfpXuUjvYPgHgLzXlHLBK0R\\\\\\\/uYIRM4ZrgKmQuPMPYCUN1nAV85KFexGJiR6bj5y5KPihStyi6sCTvwE2YOnHiddPBHySeusTFAvhxYWLwvpN5RKc7RqF9aeImq0ezKtEtMmvmaj9fbAe4I+i96fy\\\\\\\/K5TDusIgd0SFHbZzhq1pvu5NhUt8XZ6e+OxdFa0QwK5QcUFrCPOIzO7Tm4mMPhDJAq+tSx47zhhYRnhRGYV\\\\\\\/5EB1yLBahngduIVPrJVOZKxBI+2hWn5t3SS9Gh8OdKp4V0oJrJAOuymv\\\\\\\/38C6I9brZVIenVj9BT+0v0vllxpwxPzHNTI9DIrH3jGivcd31O2PDhfKW6DYg3I0cCrxFi9z1mxPDtef4fPujR7vLCB5hTK2l8L9H43UZsruO178KnhtvY\\\\\\\/id9Xu3zQSzA\\\\\\\/P+4KwwCbL+sGFSicyefTR2WDlA2W2zhe\\\\\\\/VQ42W3ydLqjkNWDbaJtMqTGtNAYKhUDiAPcil6KHJoadFvhrzfgKdKY3ZEILNEqns7J1YHnJ0DiiQjWUSLNxLVKn5e5uUOX2qjqVi8yxOEhxTqJYfU3UwDounKCHCODlrWxNeSxnDGP6zEip1NTyYNti9SmGdnEWDoAGEPMlZkisGzPIpfaY+WuTtHhOQkC8nZLoxfeSA0T3eiNmUJquOe1BcYyoRtqdT0I\\\\\\\/is7JwssjCxhgDHgeKd12Cwp0xFIs4VpJBOONZJajC2YYuwQojO0v6ra+F89B+qlQp6JjiwKA3j8VbrX0IoAOEhJpc+lIH7PJflyRSVZhB7Dlv5JYPifBZyqn+O6JmIClPCEX6eaaRrnLO0Pl9hUD+y2533dW5Z785qliqD+BVh3xAUaby+hBXOaskbIF4txGLO9RyxU2hS+PY3LptcfHN4DrL1yP18807ZTY5IwvJ8ThL1CuVmo+w8r1VCp3o5puX00PLNaPNbE\\\\\\\/L8r6iBXIdeCjVcV5gVSErzKkYCb81+D0253Y1U5ARALX3Z5eVpiPeismbkEiUyNcESRpxToLzxIuBLb7mt6xmSvEA2bZppFHo79fbsrZ3Qu18tqjdO5TCAmh0J\\\\\\\/G4iZqqfwO96510zzoRXnu1VQHQMkbJVE\\\\\\\/D\\\\\\\/Arq7HVGX2o61DmEGcOtjR0dPNUz4gUWHLU4\\\\\\\/CYQa6djUGb3VKSENMPaEDA38SQftE7AwOLSX22eE07KIAQ\\\\\\\/BMvKsbArDtCAhDZ8mFio57UnjxIX86NiGZ63RPaaNqwMt5XgjvdUKB4WOUkNIqrHJhJVjpa1QwFyFXgPEFm8xf1zJi2FCwhRRXbag++goHxheXjXJkEFokVBefL3UYnufzaiwIN6YOAjhUmKWwN1Ndy9AGHhds6Q8kFDw0LxFLZVD2uYf253HbTDLfCAfacVIRVd6gH0NFKEwvF4ci7QGCkjaashZHKAB38xE40PHkmhi+4wUN2P7YqsPXQWZGh3RH5008UcOB74lronarV2zAxHDfIkXwv90NZmAckBjuDUA08y6+EkzyXX0+ORSGIKaYymoxC8zAtKZfVxWjNedhMchKUU6fCIRuLQ7VpxaqjvObqYTnTBZGxxWAYysPDEu6So7P\\\\\\\/TvOxwptTbi7mzYEX9zK4ISjeJDEcA92fwcek0ZY4\\\\\\\/n0ig+sZfr8ZV+8a2KzH684jjfKzs0PM4AlGW5QZQgPzE5eJBh3FbjfSLuT0087GkjH2ex1yEqOvXxMoeZEJ9R0M0gpfCFBSMRzAAtYJH4\\\\\\\/JKBICr36OBf7LvOPDjwkDNmQGUWYZlQY+VH9RNt0P\\\\\\\/HgwYJ6NWpeSxH9wcU8I+AP2elpxIc0E7Rq7mVPz+46\\\\\\\/xh8Z7zP262\\\\\\\/05vJztoBdgFUmBqE8TPaFtdjF+X7Muj+c21IMLQqWVZzeYnbL1EsOIUy9IE4N6lxgQLhRPM1UVJLiG\\\\\\\/UPlrajSV8AGFCuCvz6TsIH+3mqvmlpmJCMGFow2LRzO7JnTz8qteHY8W\\\\\\\/9I5OjqB4L9+gd4KIL\\\\\\\/L+1cMAHqea4inAY9OC4dGuAYFfd5Dx5MZpW9zVCu7YLRxLplaZ3RtEVdQQ4yrjC\\\\\\\/MREWqewvqycih30Fl3pAxWc\\\\\\\/zWp00vrGv34HfcRwL0ViA5oGXjKlbu6RbaSYDbnEsJ+K6OqfnzV78sdreIVoyLwBKFcs277hl\\\\\\\/UvgpreFN3gJwKu1l39eIhrJesiAeTFYvtin2fEWOak7MX1C8RZAhwUztNZBf+dMN2HMYjszrxaXR55EVnl+\\\\\\\/ZpGL0akX9sqoUrR+a\\\\\\\/6DT81Td\\\\\\\/LfDacSWCdQvD0ms0h0ARWGcwrZb12yzcVlVKyvh6qWJaNu54CK7EKCH39TWuAbM8GKy72L5svSpXq1miXML6S\\\\\\\/WhuhMzg7sNs4WPTei5sN4bfSy8F9vWS6LIkp+GmE8QliekjOmTdAB9UWWmWQwKIDML6dKwk83qLbLWKqf30pjYM7iZ0oGjA5zE\\\\\\\/Hm0wCfNiOWOI8Ro0Qt0L5U5sK6aKqOmk3JnzZLUr7TG3He8Mm4ROh+XfqonUfxwN779CBG5h4nC6\\\\\\\/P86hv+63kHA1DYVp4J1gQIu49DUmAo\\\\\\\/fVKidE3zps93votvDANsTKsha\\\\\\\/SHdy\\\\\\\/i7FPV5c86srXhPXNTqXyDWdlIXmE5vgA\\\\\\\/4sue0WptJMedpiU7eNgouD7RJVvqN0MbG2zcqFsM+fOAoNwPuyPRT+yEJIVDTrzmtZA5pleTICcDmYZiNYBusbFkDQe0UYqYv8oeTM+Q\\\\\\\/CQKQJORJdKSOPrAGbcS3XK5Nik\\\\\\\/srn8\\\\\\\/S60DjSu2i4OEism21hE60HtgxRHwKai+Dv+rhqu90qbr\\\\\\\/P2o3R3aAVKp9QF4JWDw+bh7K97DXBNFJKYjCHOmXxFFnuD5v0\\\\\\\/9Zrga4GuPaMK619zoVnQdzaQapxoK\\\\\\\/PzvOBlhGUe2Q7Q69xqFJYEjMp4Ho7dDRljpO5Zxs4NJqBMtB3tDBksQvLZTUCDGVvDBTTdnSY4mIu38UTrEeML02Kbr8MLY1xnq6yKbV6gJ82XlAzh9TzI0bOwnclkAdggfrpGQfD11vVriS4nwGU2m4PiivhQ1fRiYKp6eNSX5ryL0ezXkAV7y7bTcVavNOVTGQdktO8VY9\\\\\\\/jFg5KKBy7nzYuAlKCnKsSNjsykCouyJBXY65wNVbuEm4SzUcc9Dmj7M2J1syDesNBQQEFATrbGf3D85LHnoA\\\\\\\/cd72sCIiq12lB0tLiaFu+8S7ZKkE1mx1UKd1iw3PoCxHZnY6v2eeHgIVKwS7aumWjkRDFgyysGQX+GG57YUoFehoh3YFUGdfxt+RouniifvHvxKXeIIRp5O0wkQl0EL3ujpogBsXHZCwZndzZ9M+zCeGFGKrshqQzHCW3K9o31KFTDIl\\\\\\\/6CcvQkmKWwEWX1unoPHWaje6XvKxwf68X+FCBZHG00PZ9uI9eBTWJ0PfNsvKS\\\\\\\/+unsxEAfgk74vKIpGqV2Uf729eRsgtadZSmqg++aaO2tQD7iF\\\\\\\/hQw0lrZVA4pk8bvcU0qZ+X3pkQYMztrRrv2D56DTCfMVt1fftNHxle5HoXVZOy3rNhRuyS92vaqmuklURb52SRInzQFrefvg2GM2eVXHvd7a0jpwYCq5+vD0C1XKxPqiUSpeJ+jszwu6MRU2x7K0ENIBNDLSVMDL7EK9tJJRMIwWhcZq8md44VdZK0G16PuD4qd7YuNddYu9ob+uS6VA7\\\\\\\/cZh\\\\\\\/2\\\\\\\/Z0SRcLYbvIDDnVrAnkj1fBwvgyPbpyj+ce7qPtN8HlArDDGaGLYE3JI8nH8D+bP47h\\\\\\\/HEnyrMD\\\\\\\/iPktKCXYMDteX5Vae7Ca99SUVOscl9UJdxJtVkpIaFmSFMwm5hNxWsqfEMIZrEAt7w1a1eosNdTJ5KqalG7kCgPXhAsUNMVzQ+A6eUqboSGCdRZq+qX\\\\\\\/wIEcXRgQzg5f+Nq34RGyPhcxWczekD5m1plhnIIjpU7jA5rft\\\\\\\/LnJ3pJuAb9wv2bAWnKsUuoJiy7cyLoaZwoRbBRmOwr8lAEXNYnznQ8ybkAo5LWixehSTnf3MUqf27QRnHfofh84nXDzaQzSOpvXqJO218LAhaeWLEhEnXuqFVf7IkK\\\\\\\/fKe+BaAB68m0fZmOrO4meNzGHdpQERaiWet0sCoKPIw3XWhyFC66q+af6NMxPNWxzSWMeWrVdkgMHTIO4UlgAdERucAfZkmG0zYhrB3fmSvtsaDLafjKq4XySYZnxntiXhtrkqi664EJslAMdsS7fA3PbDoQTqWIAbGCMh7ltDxymZQKoweH1+DUcJ+JG1q4TL\\\\\\\/3o2VOrWzT3rkQUKK5HaVBym1Jy20+1hb5VbvmCGwIDV6aPRdXuCJQMuL2K+a2Tok5gg7LK+R+Uvwk0qfP4xpMpn6msUmSxMMzyHfWPE2waji\\\\\\\/rFy2LDCuT90lJT5chG3G1oSy3DwHca3yoF8ux7+UGnb9BdzVVMArsBc3qaFzv3MSnWrcd\\\\\\\/gx4Hannw\\\\\\\/M6LAGg47ZfB5oX64jkdKNRdg5hpqBXHh0E\\\\\\\/ywrg1CG2bJn6oKDS9N3qyLwgx2oD+ep997PGvjviev4YNDOE\\\\\\\/exSScXkde0p5XxA2HjM8V+z0ZGG7a8aPJif3GFqWKP5u\\\\\\\/CzojViImhmn+mL8w5FT+iP4yfuP8toqAMa6no3oL44IoZdv21FchgNgsXM4HR4t0PK\\\\\\\/+do7tTA8aidEsUUICwmSVSdi\\\\\\\/52jPsz8\\\\\\\/8xrbIlamu2j3DjyRHDy8u7K2OelzZyjmEmy9jl28qasbmkrOQenHbkzf0NIMVI2GgHqcsTTE8f5i5zxDw7qbZQZwiVfR4CUPluhcE+Q5Xh6R5e77Qgs+zWcYjQq8VqxgqSkmXuPmIvSRbkTv\\\\\\\/VnQbLUyrS0xaU6B0GD65NI9NF1qkbBKCquDRjfGxFZvubQvXHbVS75Wn6\\\\\\\/W0+PcXZgf5Jyn0XBexJg1juF+FcaJsRZL02blelrRYod+RcAG0YYwoySiXMAPBmWpusC8dNOUtw8R\\\\\\\/i7PzOkxb8qBpO\\\\\\\/+zwNqiIJSxYLY6yVX+Mv1T9dLOH6uX1wP+EisB4bczTgDu0PEffIHoSvp57xhRjR3MrBrEO9jdLPhe0wLEh73LhBSGuUtrFOhAgVWJCYxmrNfqmLNpG0DmheW4MkNz2dknrnpf5AN59S\\\\\\\/nTm64pOh0V5MtrzwYoZDaO89miS7gc6tObs135M4k6PS4RihuVs5YJVVSPH6n47fXNU6N20XM5cz0blwaOrA9sHXfstwfQFsIkJW3apADL0V306VuNBDBHatFHIjo7yq6UBIHUfNdLCvLQR4up12QkTEgwGrQ1P89pemLmD81+RmmvJhh+OPpJ5U3g5dwC3G0a+yHzZ70BiY1\\\\\\\/LyTWWtY1ocl4iw0IYWJGH\\\\\\\/VUEulajCz3JIXh6EKkuoMHdKCYnbxRkKwy31RAc+tItKXewPwCyuF9daTQiRvenHl9Om7+b57USPsnotPTeCUt+U5JU0D7TltCc5D80RXySsyqmECof9dcwvwxL3sTPzJZzz69cijVizdDV1xN\\\\\\\/9CSLhJiGuki\\\\\\\/traRfA2bC8GmY8B6HaEugho0DmufvSNx25Al4TJE+IZfFmR6+63JUZmoefkDYoQmVGbAualDGi9nIGKL1MJaJP+8SdW8n2hpreANSIgVKiOZtlDmx9tH1mrCz4uNa2imx4HjBiKf9QjQxLCq9T4Vo3+kBwYGA2J4CtMXw1vo\\\\\\\/s3t3mUWi54Hcc0abDpvnGJDGFH6DYdF9+Mpdt9Xcf0tWm9EfjiM1Jdgm85xL1gylpjkKfh8zdcUW5JvAF+0xPTNszCi3j5CaZOA7upZkIZStj2PftUvu8nIHnWajehcxrRSLr2m3EdPARpf8WGHW7683PK3MgtLWVKjb24PpeIXQdANZ2bPkZo3Zo7OoOoNbR1y2PRyPiXxQ1gK6yNdId2PSyUDTWps3u8B\\\\\\\/raQQwsMylasMtNqWoVFN0P9ThCQbCClQHt81T0IN56euaV5bzRufa0X4eNKm6TPT2MNR5qEOlhCOjiKQiQv5QYn90GZqczTzn90leZFvXW7ffQJr0vgq624ZCLntQWI+wUy+eYuwOvdk68ygPDmvLD2mTyDMxyo\\\\\\\/boZTeBeUinV5D7+ZaH98i36U37jSjG\\\\\\\/NzwTARNDgp\\\\\\\/uTrNHEGtujTWkzytlEmFyAoyLypQ+Xr73w2A26LzzabtgQ6Y2Zc2v7yhNcxHJihxbfacgodpYKo6mSngSp9ItEk3bG6P0lS4VSLjvDSUQGpUXaaPOx4nJilZ+OfEbrBSpFfGUYPuD7bFMm\\\\\\\/sRlzuU2U47bcmgHBeLqeOcRcw1R1dwGB\\\\\\\/AcwFoWBK2vPqHvbCtWYe+0EwtSTXb27xxaYAfTBLwUKOlOpCvf6VFSjzcCtzlMx38BE5Qmmssub0sJ1pmuMyQx3fRuGDFdEbrptvRt8v17WwcgexVVxEFmS2kmSHOrLydcmWddXjgUG2irU\\\\\\\/LohrEV88hpQH6Xs5kjzTqj\\\\\\\/p2BFvsJgAC1o5S4VlBnbHhIxu9Ns2YD7pUeaS\\\\\\\/NKCFdl1mWwgLm0rGorgLClDtKEYyrHzAoxdroO8pbPGvzI69fbWBh\\\\\\\/MJcAg7cfFNH4r6+ca7lRK+uxjOzaC3we0UORQeG\\\\\\\/2oOTMvzPkf1e06sa75QQFNdVXGmly5MmGgrTR3\\\\\\\/yEX\\\\\\\/uBDZz\\\\\\\/YtUBuTxjnRDxDXzMHaYMy844aUu5WvWOlzpEPyqEJyJRe3Ma+q+4F\\\\\\\/gUpEcCvP+EGSWmCFDM5L6HyWeNAV2WqtaoFonBp3GQhC2xnYTqc0LrMNo45NwXIdzIGKLmKF7IxouWK3W+TO8DcNbjiFp8AbktAW5GHxIdsTu78s84wFGwohn3JjpQC1PH0yrJb0UZ\\\\\\\/i\\\\\\\/wmp9hDek1gx789KRsFm73PEGMYkKb+z0DHxkKuXRDLL23lP9ec7SeDoh1YuE9FbnPdrQH4WTmCb3inH5frHN2mNwEOc2\\\\\\\/QUz2l1qy+UNtjRY6I3lgtwm8uQrb5fxYkQ+dduXICUrLhSbdo6pvVk3ToFUa3qjp\\\\\\\/8MdXOnwBCaGgY4w76R6zAVSmjvV0deWPjNNJshR7YkQzLuy+BGZfQitRkmXDAZkQXZ1V\\\\\\\/yz10Pnd\\\\\\\/maZvGsGnV7YVtvaZRzecxyxBect8df4Q+RnLH\\\\\\\/KinFBuRFjQ3wp1C7NnbIhCiVPaXUTybdzjzPONARMuvilxhCyEJM5cr8zmEMheROd5LNR7G4vTYWTFg490HDPaFrCWhkpa9vlab1xyKM7sIlooIaZEMQRggy3mTp2HFKnR2K1MN\\\\\\\/wB9yGOr90SooKae4AXLoIXjvb\\\\\\\/pY6W2+\\\\\\\/LN25jBUG2vJ9qP0gDSBAhLmpT1rH8ksCaJn5dQbTDH5HU\\\\\\\/4NLnIZEdUA29xDdvsIMtpHr+zaBmLoytd+FolRYrBkNsJx4hmFEf50DxTaseO9CZLGSm0I99eih36KoDl6X\\\\\\\/0LI1Glwb\\\\\\\/sqFkiiol5R5yTsD4WTwU\\\\\\\/aDSzPGfKkmaqn21Ju\\\\\\\/ANpUtSJXyK2Um7aQrG+8N252lHEo84KVsAMonv540DTmGjSLkGmu3UgIK\\\\\\\/70LOr8lsuI8Jc6WJ4sobP+hw1Gy2xfVFhVk5ORwKtz1LcTdHl1ruoQ3vlrj5lI5pN7HwgyRXTuwbTs1Fo4WYHmWBtBWSmd5KssjC6kyGBieQUaB9NV5UP0gHSfSUz+BYW1XCeZ2iXGv4HhUi7ktcu564xHoE\\\\\\\/lPVJF6fEAhY3hsWqFvAt\\\\\\\/GsZ0k8tr0sigOl91fiEB47+fAktCU5FHsLXlcHUiVWESdKDOMoWLTkJim5Z1Ei5mUkePEbdIUtit+bqTDlpChXXNijSAkDHQ1FmXiP+P0zbEN8R8BrG4+jn5\\\\\\\/0Dr1lL6nO6SJp7f\\\\\\\/Lm+PbPJ+IUMJPVWxvbM5FNrSwugcP5Ih0j5rX8cF4so1Drv1zCyeYtchbSq5Hv\\\\\\\/fMIO5xWoW0wgfr3dgSmfULJ4DDfm2Ds1Liq5V\\\\\\\/JWrJGsDHLV58GEvoyb8\\\\\\\/IW3L1A4Rx60RzSq7p6bBP5qK\\\\\\\/tq68p+voNj2GeMB8XohtnmZY1WZ8MjehF0BQHx5a7l4rOULO5Q4gKx0lmCECOVVWHuB5iAyHjLATK4xamuRrpub4F18c9f+FDrAkShgWrJ\\\\\\\/0XCwN+pzT0Xn9non2\\\u0022,\\\u0022iv\\\u0022:\\\u00224IAAqxCqZmwMM4aCKehj2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b6c06cd17dd792e9b822ba7d8bba21ae5d492d086c7e9ac3618a3b577b5cf097\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221XPa59QSKc7fr\\\\\\\/YRFiLggQ3vTRppcbHls1tHzuKyJ5c0TTua9gfHUUMeV2LdCoEhgaG7hlgufOIkKFXdtRIwn8dJ7qM0utASzf5kLbdzwca++A5bb9obm8EdiEVbZmDu+nR0sX3ZyBlQjjvTGRpJDFt+8r1CKaoWdryRYPekUs4PVkm\\\\\\\/0pbo99fo+coHPNpW4RFBIIcOru6aIuS+03EuIQcecQ6ch73OXfi06uvlphfPYkWsvTUjSjnV\\\\\\\/3BSiAYV0YY\\\\\\\/9iVfsXpTnHuFg961whEOewGjFU3yNcMn7KkmvAuPMz0izIWYPNxMnzdxtAqJqJgn3HKk1puYTQqGggUxLod2A7\\\\\\\/MeiQyc4CbgRhDkOsQbJvkUJW+ko+sswCyImPhvPOcmL4N9Hz4LIyxuSFjpR\\\\\\\/9PlW2084s0\\\\\\\/TfDAPzOrK81wr36qpkymQoEQthep6DLu76pJmmnV5wh8gbfcBH3cbxw3bim7cYMfXmnuQ9WDLRUOZ+DQS856Lk19bEc0o3H+X2bzZA4c1AaVZIBYnjExJlGhmck\\\\\\\/rMX\\\\\\\/PLsyaR1SIcYweiODaUlF0gebcsJuUM+oJfyUxixwchJBzagdnowhDZ2HBzKlvw3\\\\\\\/L4Do5cIr1KMTiZ1iXN6+VY86yCtjRtAQHQDO628lOYS55s9WVyFwzv4nWq1HzCByriblK8q1ykJl64D4QhOqHG9dMUgMifphbnMBjQVWTrcogPrMgisn72hiZuuiRGrblsOZxY2Yll2nzyeR+ILb0mRDw5w8m5BeE4lhH0GyOrojPJZ3fk8FDOJ41i6SX5PpWYFPuQCxPtQJSr2ce3Hy\\\\\\\/TDNWBSgnKtkxQB+7fGD+bxsAqxgI4ZVMQPoJc47RQb6XrJY7uvdPrT\\\\\\\/BGYKssrdZHnIYEj59p\\\\\\\/I7zTm3uUcHggvurwS8ByRV\\\\\\\/2HbhsErbfvp5RFvX5h9qIOvC+Bxl7eSQ551v3JfpkE7APxQeDoXVod6S027UL6vBLQMy7QkRITtOGxTynz0lcLtIRyG08g2EkooaGVHC4xEIL3Jd1cunaHgapLMAwjO7sJ1jWuz\\\\\\\/55Gn3tHwQ3Y4V8ikToEhHAhYXM5EH1TzOucsXWRy+\\\\\\\/hfIGJgh6uhJJYou4IiVARnp+n8Fde9z2bfyGGNEuMBb3Ry21iiGajKZD6eIr8+NGWh2yxASK+ecu7n7sdGZUqi2ytDDyrVLCldKmmJ9oSf1l5muEUUMKSEUhjmJc4gXJoCGv7eNQW34PeSrdZ7vTZcQZtrW851KXqAIK2XewhjCGWXYnDdwjepTZMZueI8HtlT+5uidRXfpikAV8X31g1+1KezGpGqTYnIeZev7VvzIpoRK9d8BBOdvVW7TyU1zaHXln6Sg286ND5O74ZjFIar2Rzl\\\\\\\/Wrp8oofeobecOGY0tCLo83SfXwE\\\\\\\/CthndjtxJCXjmHxwzUZVmDQ9rtwDJFxACnbedODqvcZ068OlWRuD7xsgHCKjZ8DR63b3MYF2LNNyeyf4X6jcz94dhC0Kl9e4Rk60w43ZkIVpbGN6pQrjuCRVxyD0v8WBcJzPnhUKQB4RB\\\\\\\/e70sVpe9vj0T0w4rUs0dclyK0LsDQFm8IIlhaxBVoIyr7Es+AMq+KgEzXom04CCY7L+C4+Bo1rQV7giBYpfTO09IzvHz1oYxIsBruJlTgCSfQquXMkfH1ms3CuasRTCscYY\\\\\\\/Ys9jk1qaIAdpx6kh0eQ3v0H7x82CevHKB76Gz5Wo+S4Nx39msGtc49TWrY2cGGJpaUSeOYa85gH\\\\\\\/HzyQ4+BydytRFPPkvndwN2pxc7IMJDJbxT+Ljvd3us\\\\\\\/RKFDkXzqBdUUJu8uXmaGLeeH8C\\\\\\\/fWYUmcWRPQ2zBIyv80JyrSokhfBzqoujpRy04NZ4fYheH\\\\\\\/bb80wJL+XdrAuKlJJ+n\\\\\\\/N5NF2SSFSDjVQVA8jteOkv1zEnAIvGyC6kwH7NYQlhUP8+H+QN1pWfENE0Ci3Qw2pCiuA0fiC3FX+1W2I9NEF6GJQZdipd9jDJO3VgUmxHD68d3RCxErkVDXg6cvbXeCbGTc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ni+qCRlC0Yoa4hK0yWxabw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022874ad5d2b7dc8d2159052e67eb993e14ab8ce9eed23616f5d1400aed65af5898\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488291 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/447" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-siKq-ZETdy3of9RD4gTo6IbiRgyBGd4O_XQfIRAlviM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/447" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FGAOAB9XAp5mGhvAskFwKbzFBhO5GwcVoAzoiz8t6Hw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/427" title="Bayesian Diallel Analysis Reveals Mx1-Dependent and Mx1-Independent Effects on Response to Influenza A Virus in Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/461" title="Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (Medicago sativa)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488291" id="node488291" data-pisa="ggg;8/2/447" data-pisa-master="ggg;g3.117.300139" data-apath="/ggg/8/2/447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Direct Testing for Allele-Specific Expression Differences Between Conditions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Luis León-Novelo</span>, <span class="highwire-citation-author" data-delta="1">Alison R. Gerken</span>, <span class="highwire-citation-author" data-delta="2">Rita M. Graze</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lauren M. McIntyre</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1556-5907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabio Marroni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">447-460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300139 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488291" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis León-Novelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Data Science, The University of Texas Health Science Center at Houston-School of Public Health, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BLe%C3%B3n-Novelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%C3%B3n-Novelo%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuis%2BLe%25C3%25B3n-Novelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alison R. Gerken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture, Agricultural Research Service, Center for Grain and Animal Health Research, Manhattan, Kansas 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BR.%2BGerken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerken%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlison%2BR.%2BGerken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rita M. Graze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, Auburn University, Alabama 36849</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BM.%2BGraze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graze%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BM.%2BGraze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Institute and Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, Florida 32603</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lauren M. McIntyre</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fabio Marroni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Dipartimento di Scienze Agroalimentari, Ambientali e Animali, Università di Udine, 33100, Italy</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Istituto di Genomica Applicata, 33100 Udine, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabio%2BMarroni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marroni%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabio%2BMarroni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1556-5907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabio Marroni</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marroni@appliedgenomics.org" class="" data-icon-position="" data-hide-link-title="0">marroni@appliedgenomics.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/447" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488291" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/447.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488291" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/447.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488291" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/447.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488291" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/447.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488291" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488291/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Allelic imbalance (AI) indicates the presence of functional variation in <em>cis</em> regulatory regions. Detecting <em>cis</em> regulatory differences using AI is widespread, yet there is no formal statistical methodology that tests whether AI differs between conditions. Here, we present a novel model and formally test differences in AI across conditions using Bayesian credible intervals. The approach tests AI by environment (G×E) interactions, and can be used to test AI between environments, genotypes, sex, and any other condition. We incorporate bias into the modeling process. Bias is allowed to vary between conditions, making the formulation of the model general. As gene expression affects power for detection of AI, and, as expression may vary between conditions, the model explicitly takes coverage into account. The proposed model has low type I and II error under several scenarios, and is robust to large differences in coverage between conditions. We reanalyze RNA-seq data from a <em>Drosophila melanogaster</em> population panel, with F1 genotypes, to compare levels of AI between mated and virgin female flies, and we show that AI × genotype interactions can also be tested. To demonstrate the use of the model to test genetic differences and interactions, a formal test between two F1s was performed, showing the expected 20% difference in AI. The proposed model allows a formal test of G×E and G×G, and reaffirms a previous finding that <em>cis</em> regulation is robust between environments.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/robust-regulation" class="hw-term hw-article-keyword hw-article-keyword-robust-regulation" rel="nofollow">robust regulation</a></li><li class="kwd"><a href="/keyword/allelic-imbalance" class="hw-term hw-article-keyword hw-article-keyword-allelic-imbalance" rel="nofollow">allelic imbalance</a></li><li class="kwd"><a href="/keyword/allele-specific-expression" class="hw-term hw-article-keyword hw-article-keyword-allele-specific-expression" rel="nofollow">allele-specific expression</a></li><li class="kwd"><a href="/keyword/gene-regulatory-networks" class="hw-term hw-article-keyword hw-article-keyword-gene-regulatory-networks" rel="nofollow">gene regulatory networks</a></li></ul><p id="p-3">Regulatory variation can occur within <em>cis</em> acting regions (<em>e.g.</em>, promoters, enhancers, and other noncoding sequences, directly altering expression of a gene. Other <em>trans</em> acting factors such as transcription factors contribute to regulatory variation, but are expected to affect transcription of many target genes with which they interact. By revealing <em>cis</em> differences in regulation and controlling for <em>trans</em> effects, allelic imbalance (AI) provides a direct window into the relationship between <em>cis</em> regulatory sequence and regulation of transcript levels (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brem <em>et al.</em> 2002</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yan <em>et al.</em> 2002</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lo <em>et al.</em> 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wittkopp <em>et al.</em> 2004</a>). Testing for regulatory variation that affects expression in <em>cis</em> is conceptually straightforward: two alleles within a heterozygote are compared to one another. If AI is observed, there is direct evidence of <em>cis</em> differences between alleles; <em>trans</em> acting regulation alters expression of both alleles equally, producing an average effect across alleles at a locus, and does not contribute to AI within the heterozygote. Genetic interactions between <em>trans</em> acting factors and <em>cis</em> regions are expected in some cases, and these interactions are not easily separated from <em>cis</em> effects.</p><p id="p-4"><em>Cis</em> regulatory differences play a critical role in expression variation within populations (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rockman and Wray 2002</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Stranger <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Graze <em>et al.</em> 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wang <em>et al.</em> 2017</a>), divergence of gene expression between species (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Wittkopp <em>et al.</em> 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Graze <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McManus <em>et al.</em> 2010</a>), parental imprinting (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crowley <em>et al.</em> 2015</a>), and in human health and disease (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McCarroll <em>et al.</em> 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Maurano <em>et al.</em> 2012</a>). Importantly, many experimental designs have also incorporated comparisons across different physiological or environmental conditions (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">von Korff <em>et al.</em> 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tung <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cubillos <em>et al.</em> 2014</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buil <em>et al.</em> 2015</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2015</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Moyerbrailean <em>et al.</em> 2016</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Knowles <em>et al.</em> 2017</a>). <em>Cis</em> regulatory differences that are robust to environmental conditions, as well as apparent <em>cis</em> variant by environmental interactions, have been reported in these and other studies. However, it is unclear if <em>cis</em> variation is similar in most environments, or if there is a significant environmental component of variation in <em>cis</em> regulation. Current approaches rely on informal comparisons of AI estimates made separately for each condition, and there is no current formal test for AI by environment interaction.</p><p id="p-5">One inherent, and often neglected, difficulty in measurement of AI is to discriminate real allelic imbalance from bias in estimation of AI, often due to differential mapping of the two alleles on the reference (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Degner <em>et al.</em> 2009</a>). Early studies of AI used simple designs consisting of two parental strains and a single F1 genotype, with expression measured under standard conditions. These studies often used empirical controls (typically F1 DNA samples) to identify bias in estimation of AI (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Wittkopp <em>et al.</em> 2004</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Graze <em>et al.</em> 2009</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">2012</a>). However, sequencing DNA for many genotypes, in addition to RNA samples for multiple genotypes and conditions, can be prohibitively expensive. Alternatively, parental DNA reads can be simulated for identification of some forms of bias (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Degner <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Stevenson <em>et al.</em> 2013</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>), and/or a range of bias can be examined for the impact on inferences (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>). The use of a strain-specific reference or direct identification of genetic variants from the data also improves the estimation of AI (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Skelly <em>et al.</em> 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Turro <em>et al.</em> 2011</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Graze <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Satya <em>et al.</em> 2012</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Munger <em>et al.</em> 2014</a>; <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>). These analytical approaches account for bias by filtering regions of likely bias, incorporating information from controls or simulations, or use both filtering and modeling (reviewed in <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Castel <em>et al.</em> 2015</a>).</p><p id="p-6">A variety of approaches have been taken to test for significant allelic imbalance in simple designs (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ronald <em>et al.</em> 2005</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Degner <em>et al.</em> 2009</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang and Borevitz 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gregg <em>et al.</em> 2010</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">McManus <em>et al.</em> 2010</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rozowsky <em>et al.</em> 2011</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Skelly <em>et al.</em> 2011</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Turro <em>et al.</em> 2011</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Graze <em>et al.</em> 2012</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>). Approaches have included linear models for array based studies (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Ronald <em>et al.</em> 2005</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zhang and Borevitz 2009</a>), binomial or chi-square tests (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Degner <em>et al.</em> 2009</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gregg <em>et al.</em> 2010</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">McManus <em>et al.</em> 2010</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Rozowsky <em>et al.</em> 2011</a>), and Bayesian models for RNA-seq based studies (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Skelly <em>et al.</em> 2011</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Turro <em>et al.</em> 2011</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Graze <em>et al.</em> 2012</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>). The variance due to random sampling reads from a library can be modeled by treating the total number of reads in a biological replication as a random variable (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Graze <em>et al.</em> 2012</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>).</p><p id="p-7">For more complicated designs that include multiple genotypes, sexes, and/or environmental conditions, existing studies have primarily used two-step methods, applying an initial model to detect AI in each condition, and then a comparison to determine if AI differs between sample groups. This two-step method has been implemented for a variety of different approaches, including binomial tests with log odds ratios, likelihood ratio tests with Bayesian meta-analysis, and Bayesian models with pairwise Wilcoxon tests (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Edsgärd <em>et al.</em> 2016</a>; <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Moyerbrailean <em>et al.</em> 2016</a>). For example, as part of their analyses, <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> (2016)</a> apply a Bayesian model to determine AI, and identify the genes in AI across 49 test crosses of female <em>Drosophila melanogaster</em> flies in only one environment, either virgin or mated. The approach proposed here is more general in the sense that not only does it, as in <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> (2016)</a>, determines AI in mated and/or in virgin flies, but, in contrast to <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> (2016)</a>, it also formally tests whether the levels of AI are significantly different in the two environments.</p><p id="p-8">We introduce here a novel Bayesian model (referred to as the environmental model) that allows formal testing of AI between environments, while accounting for potential bias, and model the total number of reads as a random sample from the library. The model can be used with or without empirical control samples to account for bias. In addition, the environmental model explicitly takes into account the expression level of the genic region being examined in each condition, as well as the proportion of reads that are informative for estimating allele-specific expression. The ability to assign reads to the paternal or maternal allele depends upon the amount and location of sequence differences observed between the alleles. The higher the number of differences, and the more even the distribution, the greater the discrimination ability. Lower numbers of differences or large regions with no differences result in greater numbers of unassigned reads that are not informative for estimation of AI. To our knowledge, in all analyses of AI to date, reads uninformative for AI analyses are discarded. The unassigned reads provide us information about the variability of the read counts, and more precise estimate of this variance increases the power to detect AI.</p><p id="p-9">To test the performance of the model with real data, we also conduct a reanalysis of existing data, examining differences in AI for an environmental perturbation in <em>Drosophila</em> that has dramatic effects on overall expression, the change from virgin to mated status. During mating, male <em>D. melanogaster</em> transfer, together with sperm, a mixture of peptides and proteins into the female reproductive tract; these peptides cause profound changes in female physiology (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhou <em>et al.</em> 2014</a>). As a consequence, mated female flies experience changes in body composition (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Everaerts <em>et al.</em> 2010</a>), life span (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aigaki and Ohba 1984</a>), and gene expression (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lawniczak and Begun 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McGraw <em>et al.</em> 2008</a>). Here, we use the environmental model to understand how this large physiological change affects variation in <em>cis</em> regulation.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>The baseline model</h3><p id="p-10">The baseline model was developed by <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> (2014)</a>. Briefly, the observed read counts in biological replicate (i) for any single exon/environment are assigned to maternal or paternal alleles, and unassigned reads are discarded (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Let <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-1.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-2.gif"/></noscript></span></span> be the RNA allele-specific read counts from a heterozygote in biological replicate <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-3.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-17.gif"/></noscript></span></span>Here, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-18.gif"/></noscript></span></span> is the overall mean, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-19.gif"/></noscript></span></span> is the variation of biological replicates (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-20.gif"/></noscript></span></span>) due to the random sampling of reads from libraries, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-21.gif"/></noscript></span></span> is the effect of a read having AI, and q incorporates bias information, where values &gt;0.5 indicates bias toward the <em>y</em> allele, and values &lt;0.5 is bias toward the <em>x</em> allele. If <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-22.gif"/></noscript></span></span> is the real proportion of reads from the <em>y</em> allele then:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-23.gif"/></noscript></span></span>when there is no AI (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-24.gif"/></noscript></span></span>), therefore<span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-25.gif"/></noscript></span></span> The bias correction parameter is <em>q</em>; it can be a random variable estimated from DNA controls, from simulation, or a fixed constant varied to reflect uncertainty (<a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read assignment. Reads aligning to the maternal allele (m) are purple, those aligning to the paternal allele (p) are green, and those aligning equally well to both alleles (unassigned) are gray. Expected values according to the environmental model are given in Table 1, and are in black type above. The baseline model uses information of the reads aligning to paternal or maternal alleles only (red boxes), while the environmental model additionally incorporates information about unassigned reads (blue boxes). In the baseline model,  is the overall expected value,  models the biological replicate variation,  is equivalent to AI, and the known quantity  is interpreted as the expected proportion of maternal read counts when there is no AI. In other terms,  The priors for all model parameters are gamma(1/2,1/2). The quantities  and  in the notation of the baseline model play the role of the parameters  and  respectively in the environmental model. When  in the environmental model,  This leads us to an interpretation of  under the null in the baseline model as  in the environmental model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Read assignment. Reads aligning to the maternal allele (&lt;em&gt;m&lt;/em&gt;) are purple, those aligning to the paternal allele (&lt;em&gt;p&lt;/em&gt;) are green, and those aligning equally well to both alleles (unassigned) are gray. Expected values according to the environmental model are given in Table 1, and are in black type above. The baseline model uses information of the reads aligning to paternal or maternal alleles only (red boxes), while the environmental model additionally incorporates information about unassigned reads (blue boxes). In the baseline model, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-4&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the overall expected value, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-5&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; models the biological replicate variation, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-6&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is equivalent to AI, and the known quantity &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-7&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is interpreted as the expected proportion of maternal read counts when there is no AI. In other terms, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-8&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; The priors for all model parameters are gamma(1/2,1/2). The quantities &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in the notation of the baseline model play the role of the parameters &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-11&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-12&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; respectively in the environmental model. When &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-13&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in the environmental model, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-14&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; This leads us to an interpretation of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-15&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; under the null in the baseline model as &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-16&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in the environmental model.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F1.medium.gif" width="406" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/447/F1.medium.gif" width="406" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488523" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Read assignment. Reads aligning to the maternal allele (<em>m</em>) are purple, those aligning to the paternal allele (<em>p</em>) are green, and those aligning equally well to both alleles (unassigned) are gray. Expected values according to the environmental model are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, and are in black type above. The baseline model uses information of the reads aligning to paternal or maternal alleles only (red boxes), while the environmental model additionally incorporates information about unassigned reads (blue boxes). In the baseline model, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-4.gif"/></noscript></span></span> is the overall expected value, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-5.gif"/></noscript></span></span> models the biological replicate variation, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-6.gif"/></noscript></span></span> is equivalent to AI, and the known quantity <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-7.gif"/></noscript></span></span> is interpreted as the expected proportion of maternal read counts when there is no AI. In other terms, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-8.gif"/></noscript></span></span> The priors for all model parameters are gamma(1/2,1/2). The quantities <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-10.gif"/></noscript></span></span> in the notation of the baseline model play the role of the parameters <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-12.gif"/></noscript></span></span> respectively in the environmental model. When <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-13.gif"/></noscript></span></span> in the environmental model, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-14.gif"/></noscript></span></span> This leads us to an interpretation of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-15.gif"/></noscript></span></span> under the null in the baseline model as <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/F1/embed/mml-math-16.gif"/></noscript></span></span> in the environmental model.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>The environmental model</h3><p id="p-12">Let <em>i</em> = 1,2 be the index of condition, which can represent environmental differences or differences in treatments (<em>e.g.</em>, mated and virgin status) or differences between genotypes. We define them as environment 1 and environment 2.</p><p id="p-13">Assume that we have <em>K</em><sub>1</sub> biological replicates for environment 1 and <em>K</em><sub>2</sub> replicates for environment 2.</p><p id="p-14">For each gene/gene region to be examined, we define:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-15"><em>x<sub>i</sub></em>,<em><sub>k</sub></em> = number of RNA reads aligned to the “maternal” allele in environment <em>i</em> and replicate <em>k</em>,</p></li><li id="list-item-2"><p id="p-16"><em>y<sub>i</sub></em>,<em><sub>k</sub></em> = number of RNA reads aligned to the paternal allele.</p></li><li id="list-item-3"><p id="p-17"><em>z<sub>i</sub></em>,<em><sub>k</sub></em> = number of RNA reads aligned equally well to both alleles, and that cannot be assigned to either allele; they are also referred to as “unassigned”; and</p></li><li id="list-item-4"><p id="p-18"><em>r<sub>i</sub></em>,<em><sub>p</sub></em> (<em>r<sub>i</sub></em>,<em><sub>m</sub></em>) = probability that a read generated from paternal (maternal) aligns to paternal.</p></li><li id="list-item-5"><p id="p-19">(maternal) for environment <em>i</em>. In specific notation:</p></li><li id="list-item-6"><p id="p-20">r<em><sub>i</sub></em>,<em><sub>p</sub></em> = P<em>r</em>[read aligns to paternal|read generated by paternal] for <em>i</em> = 1,2.</p></li><li id="list-item-7"><p id="p-21">1−<em>r<sub>i</sub></em>,<em><sub>p</sub></em> = Pr[read is unassigned|read generated by paternal] for the genotype in environment <em>i</em>.</p></li></ul><p id="p-22"><em>r<sub>i</sub></em>,<em><sub>p</sub></em> (<em>r<sub>i</sub></em>,<em><sub>m</sub></em>) can be estimated from control DNA or from simulation. To estimate <em>r<sub>i</sub></em>,<em><sub>p</sub></em>, (1) use the paternal genome to simulate all possible reads that can be generated by the paternal allele, (2) align these reads to both the paternal and maternal genomes, and (3) estimate <em>r<sub>i</sub></em>,<em><sub>p</sub></em> as the proportion of the simulated reads aligning best to the paternal genome. Note that the true proportion of simulated reads aligning best to the parental allele in the two environments (<em>r<sub>1</sub></em>,<em><sub>p</sub></em> and <em>r<sub>2</sub></em>,<em><sub>p</sub></em>) for the same genotype is expected to be similar. However, to allow for comparisons across genotypes where this assumption may not hold, the model does not force these two parameters to be identical.</p><p id="p-23">We assume that the distribution of the counts <em>x<sub>i</sub></em>,<em><sub>k</sub></em>; <em>y<sub>i</sub></em>,<em><sub>k</sub></em>; <em>z<sub>i</sub></em>,<em><sub>k</sub>|α<sub>i</sub></em>, <em>β<sub>i</sub></em>,<em><sub>k</sub></em> have the expected values given in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. A graphical representation of the expected values is provided in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. The biological replication specific random effect <em>β<sub>i</sub></em>,<em><sub>k</sub></em> corrects for the random sampling of reads from the library for each biological replication. As with the baseline model, we assume that the total number of reads from a biological replication is a random effect with expected value <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-26.gif"/></noscript></span></span> This explicitly recognizes the sampling of material from a library in order to generate the observed reads and incorporates the resulting sampling variance directly into the model. Approaches that consider the total number of reads in a biological replication to be fixed often fail to model the overdispersion of the data. Allowing it to be random allows modeling of the overdispersion.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488402/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488402/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Expected values for read counts under the proposed model for environment <em>i</em> and replicate (<em>i</em>, <em>k</em>), where <em>i</em> = 1, 2 and <em>k</em> = 1,2<em>….k<sub>i</sub></em></span><div class="sb-div caption-clear"></div></div></div><p id="p-25">For example, the maternal read count of the third virgin replicate, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-27.gif"/></noscript></span></span> has an expectation of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-28.gif"/></noscript></span></span> Note that when <em>r<sub>i</sub></em>,<em><sub>p</sub></em> and <em>r<sub>i</sub></em>,<em><sub>m</sub></em> are both equal to 1, we expect all the reads to be assigned to an allele, and the proportion unassigned to be zero (<em>i.e.</em>, <em>z<sub>i</sub></em>,<em><sub>k</sub></em> = 0). The proportion of reads assigned to the maternal (paternal) allele compared to those that are not assigned will depend on several factors, such as read length, the level of divergence between the two haplotypes, the alignment algorithm, and the stringency of alignment. These parameters are not equal to reflect the possibility that the quality of the genotype specific references may not be equivalent between the maternal and paternal reference, and/or that for a particular gene/gene region there may be unobserved structural variation in one but not both alleles. DNA controls can account for factors such as hidden structural variation, but can be prohibitively expensive. In the absence of DNA controls, the use of simulations can estimate some of the potential bias, and inclusion of these estimates in the model is preferable to ignoring sequence and mapping bias altogether (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>).</p><p id="p-26"><em>β<sub>i</sub></em>,<em><sub>k</sub>/α<sub>i</sub></em> and <em>α<sub>i</sub>β<sub>i</sub></em>,<em><sub>k</sub></em> are the total expected number of reads from the paternal and maternal alleles, respectively, in environment <em>i</em> and biological replication <em>k</em>. Of the <em>β<sub>i</sub></em>,<em><sub>k</sub>/α<sub>i</sub></em> reads coming from the paternal allele, we expect (1/<em>α<sub>i</sub></em>)<em>β<sub>i</sub></em>,<em><sub>k</sub>r<sub>i</sub></em>,<em><sub>p</sub></em> to be assigned to the paternal allele, and (1−<em>r<sub>i</sub></em>,<em><sub>p</sub></em>)<em>β<sub>i</sub></em>,<em><sub>k</sub>/α<sub>i</sub></em> to be unassigned. Similarly, of the <em>α<sub>i</sub>β<sub>i</sub></em>,<em><sub>k</sub></em> reads from the maternal allele, we expect <em>α<sub>i</sub>β<sub>i</sub></em>,<em><sub>k</sub>r<sub>i</sub></em>,<em><sub>m</sub></em> to be assigned to the maternal allele and (1−<em>r<sub>i</sub></em>,<em><sub>m</sub></em>)<em>α<sub>i</sub>β<sub>i</sub></em>,<em><sub>k</sub></em> to be unassigned. The parameter <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-29.gif"/></noscript></span></span> is the ratio of the maternal and paternal expected value counts for genotype <em>i</em>, after adjusting for assignment bias. Explicitly allowing for reads to be unassigned allows the level of expression to be included in the modeling.</p><p id="p-27">The notation is the following:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-30.gif"/></noscript></span></span>for <em>i</em> = 1,2 and <em>k</em> = 1,2,…,<em>K<sub>i</sub></em>.</p><p id="p-28">We are interested in testing the null hypotheses:</p><ol class="list-ord " id="list-2"><li id="list-item-8"><p id="p-29">Allelic balance in environment 1 (<em>e.g.</em>, mated) or, equivalently, H<sub>01</sub>: <em>α</em><sub>1</sub> = 1.</p></li><li id="list-item-9"><p id="p-30">Allelic balance in environment 2 (<em>e.g.</em>, virgin), H<sub>02</sub>: <em>α</em><sub>2</sub> = 1.</p></li><li id="list-item-10"><p id="p-31">The level of AI is the same in both (or independent of) environments. Equivalently, H<sub>03</sub>: <em>α</em><sub>1</sub> = <em>α</em><sub>2.</sub></p></li></ol><p id="p-32">Notice that H<sub>03</sub> is testing if the true proportion of reads coming from the paternal allele in environment 1 (<em>e.g.</em>, mated) is the same as in environment 2 (<em>e.g.</em>, virgin), or, in other words, the level of AI is the same in both conditions (<em>e.g.</em>, mated and virgin). This is a formal test for the interaction between AI and environment.</p><p id="p-33">The proportion of counts coming from the maternal allele (line) in environment <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-31.gif"/></noscript></span></span> is then:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-32.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-33.gif"/></noscript></span></span> is expected to be 0 when no counts are coming from the maternal allele, 1 when all the counts are coming from the maternal allele, and 0.5 in the case of perfect allelic balance. The model is implemented in R (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">R Core Team 2017</a>), and is available as Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS2.zip">File S2</a>.</p></div><div id="sec-4" class="subsection"><h3>Negative binomial sampling model</h3><p id="p-34">Above, we describe the model for the expected value counts, and now we model the counts. We assume the counts follow a negative binomial distribution, with expected values given in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. For example, the distribution for the read counts in the third virgin biological replicate aligning to the maternal allele is:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-34.gif"/></noscript></span></span>The dispersion parameter <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-35.gif"/></noscript></span></span> is common to the distribution of all read counts. Here <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-36.gif"/></noscript></span></span> denotes the negative binomial distribution with mean <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-37.gif"/></noscript></span></span> and variance <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-38.gif"/></noscript></span></span> RNA-Seq data typically exhibit overdispersion (with respect to the Poisson model; <em>i.e.</em>, the variance is greater than the mean). Since the negative binomial sampling distribution can model overdispersed data, it has been proposed as a sampling distribution for RNA counts (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Robinson and Smyth 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anders and Huber 2010</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Di <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">León-Novelo <em>et al.</em> 2017</a>).</p><p id="p-35">To complete the Bayesian model, we specify the prior distributions for the model parameters:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-39.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-40.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-41.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-42.gif"/></noscript></span></span>Here, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-43.gif"/></noscript></span></span>denotes the gamma distribution with mean shape and rate parameters <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-44.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-45.gif"/></noscript></span></span></p><p id="p-36">The computation of credible intervals for the parameters of interest <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-47.gif"/></noscript></span></span> is carried out using Markov Chain Monte Carlo. We reject H<sub>01</sub> when the credible interval for <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-48.gif"/></noscript></span></span>does not contain 1. Similarly, we reject H<sub>02</sub> (or H<sub>03</sub>) when the credible interval for <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-49.gif"/></noscript></span></span> (or <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-50.gif"/></noscript></span></span> does not contain 1.</p><p id="p-37">The lognormal prior distributions for the parameters <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-51.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-52.gif"/></noscript></span></span> were used to make the model symmetric with respect to the labels paternal and maternal (in our example, tester and line, respectively). This is <em>a priori</em> <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-53.gif"/></noscript></span></span> so the estimates of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-54.gif"/></noscript></span></span> in the baseline model are the same as the estimates of <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-55.gif"/></noscript></span></span> in the model swapping the labels of tester and line. The hyper-parameter <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-56.gif"/></noscript></span></span> is chosen so that the prior expected value of <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-57.gif"/></noscript></span></span> equals <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-58.gif"/></noscript></span></span> and prior variance <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-59.gif"/></noscript></span></span> so, in the case of noninformative data (<em>e.g.</em>, very small counts), the posterior distribution of <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-60.gif"/></noscript></span></span> is similar to the prior, and we do not conclude AI. The gamma prior for <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-61.gif"/></noscript></span></span> “tightens” the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-62.gif"/></noscript></span></span>s to be similar.</p><p id="p-38">Setting a prior in the hyper-parameters of the prior distribution of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-63.gif"/></noscript></span></span>s allows a “borrow of strength” across different biological replicates. To set the hyper-parameters of the prior distribution for <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-64.gif"/></noscript></span></span> we use an “empirical Bayes approach.” First, we obtain (rough) estimates for <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-65.gif"/></noscript></span></span>s. Second, we assume that they follow a gamma distribution, and compute the MLE estimates <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-67.gif"/></noscript></span></span> of the shape and rate parameters, respectively. Third, we set the parameter in the model <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-68.gif"/></noscript></span></span> and we set <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-69.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-70.gif"/></noscript></span></span> so that <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-71.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-72.gif"/></noscript></span></span> Since we have few data points to estimate <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-73.gif"/></noscript></span></span> we use an informative prior with mean <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-74.gif"/></noscript></span></span> prior with variance <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-75.gif"/></noscript></span></span></p></div><div id="sec-5" class="subsection"><h3>Measuring type I and II error rate</h3><p id="p-39">Read counts were simulated according to a negative binomial distribution in several different scenarios. In each scenario, read counts were simulated for 1000 gene regions (for convenience, we refer to these as exons) independently in two different environments, with three replicates per environment. The number of reads mapping in each exon was simulated according to a negative binomial distribution with size = 50 and expected value equal to <em>β<sub>i</sub></em>,<em><sub>k</sub></em> × <em>η</em>, where <em>β<sub>i</sub></em>,<em><sub>k</sub></em> is the effect of biological replicate in environment <em>i</em> and replicate <em>k</em>, and <em>η</em> is a factor used to simulate different levels of gene expression (and/or sequencing coverage), which in turn affect the power of detecting AI. Simulations were run with <em>η</em> = 10 (low expression/coverage) and <em>η</em> = 100 (high expression/coverage). <em>β<sub>i</sub></em>,<em><sub>k</sub></em> was varied as shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488324/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488324/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Values of β used in the simulations</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Different levels of AI in the environment 2 were simulated for sets of 1000 exons by varying <em>α</em><sub>2</sub> from 0.5 to 2. No AI was simulated in environment 1 (<em>i.e.</em>, <em>α</em><sub>1</sub> = 1); 11 different levels of log2 of the ratio α<sub>1</sub>/α<sub>2</sub> were simulated, varying from −1 and 1 with step 0.2.</p><p id="p-41">Read counts were simulated without bias, <em>i.e.</em>, with <em>r<sub>p</sub></em> = <em>r<sub>m</sub></em>. To assess robustness of the model to misspecification of bias, unbiased simulated counts were fed to the model, and analysis of AI was conducted after providing biased prior estimates of <em>r<sub>p</sub></em> and <em>r<sub>m</sub></em>.</p><p id="p-42">The bias in estimation of AI is not always correctly captured by simulation or by DNA controls. This means that, even when controls are used, bias may be mis-specified. It is important to understand how the model performs with different levels of mis-specification. Bias mis-specification was measured as <em>x</em> = 2(100)(<em>r</em>−1), where (<em>r</em>−1) is the difference between the mis-specified value of <em>r<sub>p</sub></em> (or <em>r<sub>m</sub></em>) used to fit the model, and the simulated value of <em>r<sub>p</sub></em> (or <em>r<sub>m</sub></em>), <em>i.e.</em>, 1. Nine different levels of mis-specification <em>x</em> were used: −40, −30, −20, −10, 0, 10, 20, 30, and 40%.</p><p id="p-43">When maternal and paternal alleles have similar sequences in the coding region, the proportion of reads aligning equally well to both alleles is high. A series of simulations was conducted using the expected number of unassigned reads <em>z<sub>i</sub></em>,<em><sub>k</sub></em>. To assess the effect of variation in the proportion of unassigned reads, an additional series of simulations was conducted by multiplying the expected values of <em>z<sub>i</sub></em>,<em><sub>k</sub></em> by 10,000. We will refer to these two scenarios as “unassigned inflation factor = 1” and “unassigned inflation factor = 10,000,” respectively.</p><p id="p-44">The first scenario can be observed when the paternal and maternal haplotypes are different, and most reads can be assigned. The second scenario is observed when most of the reads cannot be assigned to either haplotype, and is used as a worst case scenario of very similar paternal and maternal haplotypes. In any given experiment, we expect some genes to follow each of these two scenarios.</p><p id="p-45">To measure the robustness of the model to unequal gene expression across conditions, we repeated the simulations comparing one condition with high coverage, and one with low coverage, <em>i.e.</em>, we run a simulation setting <em>η</em><sub>1</sub> = 10 and <em>η</em><sub>2</sub> = 100, and a simulation setting <em>η</em><sub>1</sub> = 100 and <em>η</em><sub>2</sub> = 10. This is an important consideration as differential expression should not be confused with AI.</p><p id="p-46">Type I error was assessed using simulations in which <em>α</em><sub>1</sub> = 1 and <em>α</em><sub>2</sub> = 1 (<em>i.e.</em>, no AI in either environment). Type II error of AI detection, and of the detection of difference in AI across environments, was assessed using simulations in which <em>α</em><sub>1</sub> = 1 and <em>α</em><sub>2</sub> ≠ 1.</p></div><div id="sec-6" class="subsection"><h3>Reanalysis of D. melanogaster data (mated <em>vs.</em> virgin)</h3><div id="sec-7" class="subsection"><h4>Data retrieval and cleaning:</h4><p id="p-47">RNA-seq data from a panel of 68 <em>D. melanogaster</em> F1 hybrids obtained from crossing 68 strains with the w1118 laboratory strain (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kurmangaliyev <em>et al.</em> 2015</a>) were used to measure allele specific expression. Mapping of reads on genotype-specific references and quantification of reads aligning to each exon was performed as previously described (<a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>). Briefly, reads originating from each cross were aligned on the strain-specific references (obtained from FlyBase v5.51) for the two parental genomes, and reads were classified as aligning in paternal (tester), maternal (ine), or unassigned (when reads mapped equally well on both references).</p><p id="p-48">Based on simulations, low numbers of reads assigned to maternal or paternal allele, and/or high numbers of reads that cannot be assigned to either allele lead to inflated type I error. For this reason, instances in which the proportion of total reads assigned to either parental allele was &lt;1% (<em>i.e.</em>, having either very low number of assigned reads or very high number of unassigned reads) were excluded from analysis.</p><p id="p-49">Analysis was performed on a total of 169,842 data points (termed “exons × lines,” <em>i.e.</em>, the number of exons across all lines), belonging to 13,898 different exons in 68 crosses. In the manuscript, the terms “data point” and “exons × lines” will be used to indicate the sum of informative lines across all the informative exons (or equivalently, the sum of informative exons across all the informative lines), as follows:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/447/embed/mml-math-76.gif"/></noscript></span></span>where <em>e<sub>i</sub></em> and <em>l<sub>j</sub></em> are 1 if exon <em>i</em> in line <em>j</em> is informative and zero otherwise, and <em>n</em> and <em>m</em> are the total number of exons and lines considered, respectively.</p><p id="p-50">For the purpose of analysis, data were aggregated by (a) exon, (b) line, or (c) exon in line. We detail below the different meanings. One gene composed of two exons, one with data in 10 crosses, and one with data in 15 crosses, respectively, has information in two exons, in 12.5 lines (average) and 25 data points. A gene consisting of 25 exons, all with data in just one cross, is said to have information in 25 exons, one line, and 25 data points.</p></div><div id="sec-8" class="subsection"><h4>Comparing the environmental model with the baseline model:</h4><p id="p-51">A Bayesian model that accounts for bias (<a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>) has been already applied to the <em>D. melanogaster</em> data presented here (<a id="xref-ref-16-11" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>); we will refer to it as the “baseline” model. The model presented here extends the baseline model significantly, and directly compares AI between environments; it will be referred to as the “environmental” (as opposed to the “baseline”) model. Results were compared for a total of 66,504 data points that passed quality control in both the previous (<a id="xref-ref-16-12" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>) and present study. Pearson’s correlation coefficients between the posterior estimates of the proportion of reads aligning on the paternal allele (ϑ) were estimated for mated and virgin flies according to results of the baseline and environmental model, respectively.</p><p id="p-52">To determine the effect of coverage (number of reads mapped on each exon) on AI in the baseline and environmental model, exons were stratified into four groups: (1) no AI according to either model, (2) AI according to the environmental model only, (3) AI according to the baseline model only, and (4) AI in both models. Mean coverage was assessed by measuring the mean number of reads mapping an allele specifically in each exon in mated and virgin lines. The distribution of coverage among exons without AI was compared to the distribution of coverage in the remaining three classes using the Wilcoxon-Mann-Whitney test, separately for mated and virgin flies. The distribution of the deviation of ϑ from the expected value of 0.5 was plotted in exons showing, or not showing, AI in either model in virgin and mated flies, separately. The distribution of the differences in the estimate of ϑ between the two models were plotted, stratifying AI classification according to the two models: no AI, AI according to either one of the models, AI according to both models. Statistical analysis were performed in R (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">R Core Team 2017</a>).</p></div><div id="sec-9" class="subsection"><h4>Analysis of AI in mated and virgin flies using the environmental model:</h4><p id="p-53">The RNA-seq data included in this study has been described before (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kurmangaliyev <em>et al.</em> 2015</a>). Briefly, RNA was obtained from F1 mated and virgin flies originated by crossing females of each of 68 lines with mates of a common tester (w1118). Data points were included in the study if at least 50 reads could be mapped to either paternal or maternal allele in both virgin and mated flies, and if at least 200 total reads were mapped to the exon (irrespectively of them being assigned to the paternal or maternal allele, or being unassigned). The environmental model was used to simultaneously estimate AI in mated female flies, in virgin female flies, and to test for AI by environment interaction (G×E). In total, 169,842 data points were analyzed.</p><p id="p-54">As each of the F1 genotypes has been evaluated for AI, it is of interest to assess the population frequency of AI in any given exon, Fisher’s exact test was performed on a 2 × 2 contingency table, reporting for each exon and the number of F1 genotypes showing different levels of AI between conditions and those not showing differences in AI between conditions. Exons showing higher (or lower) than expected frequency of AI between conditions were identified by testing the hypothesis that the odds ratio in the contingency table is &gt;1 or &lt;1.</p><p id="p-55">A further demonstrative analysis of how genotypes may be compared across the population was performed by explicitly comparing AI in two different F1 genotypes, treating each F1 genotype as an environment. The environmental model was used to simultaneously estimate AI in r365×w1118 F1’s, in r907×w1118 F1’s, and to test for AI by F1 interaction (G×G).</p></div></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-56">Sequences used for the present work were retrieved from SRA under the accession number PRJNA281652. Detailed description of the procedures for obtaining F1 genotypes is available at <a href="https://github.com/McIntyre-Lab/papers/tree/master/lehmann_2015/original_data">https://github.com/McIntyre-Lab/papers/tree/master/lehmann_2015/original_data</a>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a> includes supplemental methods, Figures S1–S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a> with legends, and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a> with legend. The supplemental material includes an implementation of the model in R and a toy data set. Instructions are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS2.zip">File S2</a>.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>Measuring type I and II error rate of the environmental model</h3><p id="p-57">We assessed type I and II error simulating various scenarios that could be experienced in real experiments. We varied <em>η</em>, a proxy of the number of reads mapped to each feature, and thus representative of the combination of gene expression and average sequencing depth (coverage). In addition, we varied the proportion of reads that cannot be assigned to either allele (unassignment). High unassignment rates can be observed when the paternal and maternal allele are similar, and a large proportion of reads cannot be unambiguously attributed to either allele.</p><p id="p-58">Results shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> report Type 1 error rate as a function of bias misspecification at two different levels of gene expression (or coverage), and two different levels of unassigned reads. Unlike the baseline model, the type I error rate when the number of unassigned reads follows the model expectation (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), and the bias is not higher than previously reported in interspecific crosses, where it has been measured by DNA (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Graze <em>et al.</em> 2012</a>; <a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>), is close to the expected nominal levels.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Type&#xA0;I error rate. The two upper panels show Type&#xA0;I error rate in the case of low gene expression, and the two bottom panels show Type&#xA0;I error rate in the case of high gene expression. Panels on the left show results obtained by specifying the number of reads aligning equally well to both alleles as expected under the model in Table 1. Panels on the right shows results obtained by multiplying that quantity by 10,000. We plot in red the type&#xA0;I error rate in environment&#xA0;1, in gray the type&#xA0;I error rate in environment&#xA0;2, and in blue and green the type&#xA0;I error rate in detecting different levels of AI between environments using the Bayesian approach implemented in the environmental model, and the descriptive method as implemented in the baseline model, respectively. Low expression: &#x3B7;&#xA0;=&#xA0;10; High Expression: &#x3B7;&#xA0;=&#xA0;100; Low unassignment: unassigned reads&#xA0;=&#xA0;zi,k; High unassignment: unassigned reads =&#xA0;10,000 &#xD7; zi,k" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Type&#xA0;I error rate. The two upper panels show Type&#xA0;I error rate in the case of low gene expression, and the two bottom panels show Type&#xA0;I error rate in the case of high gene expression. Panels on the left show results obtained by specifying the number of reads aligning equally well to both alleles as expected under the model in Table 1. Panels on the right shows results obtained by multiplying that quantity by 10,000. We plot in red the type&#xA0;I error rate in environment&#xA0;1, in gray the type&#xA0;I error rate in environment&#xA0;2, and in blue and green the type&#xA0;I error rate in detecting different levels of AI between environments using the Bayesian approach implemented in the environmental model, and the descriptive method as implemented in the baseline model, respectively. Low expression: &lt;em&gt;&#x3B7;&lt;/em&gt;&#xA0;=&#xA0;10; High Expression: &lt;em&gt;&#x3B7;&lt;/em&gt;&#xA0;=&#xA0;100; Low unassignment: unassigned reads&#xA0;=&#xA0;&lt;em&gt;z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;,&lt;em&gt;&lt;sub&gt;k&lt;/sub&gt;&lt;/em&gt;; High unassignment: unassigned reads =&#xA0;10,000 &#xD7; &lt;em&gt;z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;,&lt;em&gt;&lt;sub&gt;k&lt;/sub&gt;&lt;/em&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F2.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/447/F2.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488418" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-59" class="first-child">Type I error rate. The two upper panels show Type I error rate in the case of low gene expression, and the two bottom panels show Type I error rate in the case of high gene expression. Panels on the left show results obtained by specifying the number of reads aligning equally well to both alleles as expected under the model in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>. Panels on the right shows results obtained by multiplying that quantity by 10,000. We plot in red the type I error rate in environment 1, in gray the type I error rate in environment 2, and in blue and green the type I error rate in detecting different levels of AI between environments using the Bayesian approach implemented in the environmental model, and the descriptive method as implemented in the baseline model, respectively. Low expression: <em>η</em> = 10; High Expression: <em>η</em> = 100; Low unassignment: unassigned reads = <em>z<sub>i</sub></em>,<em><sub>k</sub></em>; High unassignment: unassigned reads = 10,000 × <em>z<sub>i</sub></em>,<em><sub>k</sub></em></p><div class="sb-div caption-clear"></div></div></div><p id="p-60">Only when the number of unassigned reads is 10,000 times the expected value (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), do type I error rates climb. As expected, when bias is high, and unaccounted for in the model, the type I error rate can be very high. Under this “worst case” scenario, the environmental model still has lower type I error relative to the baseline model.</p><p id="p-61">When the number of unassigned reads matches the model expectation (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>), the type I error rate for detecting differences in AI across environments (blue lines and dots in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) is &lt;5% for all bias misspecification levels.</p><p id="p-62">As expected, the type II error rate increases when log2 <em>α</em><sub>1</sub>/<em>α</em><sub>2</sub> approaches zero. At each value of log2 <em>α</em><sub>1</sub>/<em>α</em><sub>2</sub>, lower expression leads to a higher type II error rate. The effect of the misspecification of the proportion of unassigned reads on type II error is small (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Type&#xA0;II error rate. Type&#xA0;II error rate when comparing an environment for which no AI was present (&#x3B1;1&#xA0;=&#xA0;1) with an environment with varying levels of AI. Differences in AI between environments are reported as log2 &#x3B1;1/&#x3B1;2. The two upper panels show Type&#xA0;II error rate in case of low gene expression, and the two bottom panels show Type&#xA0;II error rate in case of high gene expression. Panels on the left show results obtained by specifying the number of reads aligning equally well to both alleles as expected under the model in Table 1. Panels on the right shows results obtained by multiplying that quantity by 10,000. We plot in red the type&#xA0;II error rate in environment&#xA0;1, in blue the type&#xA0;II error rate for difference in AI between environments, and in green the type&#xA0;II error rate between the environments when using the descriptive approach as implemented in the baseline model. Each point represents the average over the seven simulated levels of bias misspecification, and the error bars represent SE. Low expression: &#x3B7;&#xA0;=&#xA0;10; High Expression: &#x3B7;&#xA0;=&#xA0;100; Low unassignment: unassigned reads&#xA0;=&#xA0;zi,k; High unassignment: unassigned reads&#xA0;=&#xA0;10,000 &#xD7; zi,k" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Type&#xA0;II error rate. Type&#xA0;II error rate when comparing an environment for which no AI was present (&lt;em&gt;&#x3B1;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&#xA0;=&#xA0;1) with an environment with varying levels of AI. Differences in AI between environments are reported as log2 &lt;em&gt;&#x3B1;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;/&lt;em&gt;&#x3B1;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;. The two upper panels show Type&#xA0;II error rate in case of low gene expression, and the two bottom panels show Type&#xA0;II error rate in case of high gene expression. Panels on the left show results obtained by specifying the number of reads aligning equally well to both alleles as expected under the model in Table 1. Panels on the right shows results obtained by multiplying that quantity by 10,000. We plot in red the type&#xA0;II error rate in environment&#xA0;1, in blue the type&#xA0;II error rate for difference in AI between environments, and in green the type&#xA0;II error rate between the environments when using the descriptive approach as implemented in the baseline model. Each point represents the average over the seven simulated levels of bias misspecification, and the error bars represent SE. Low expression: &lt;em&gt;&#x3B7;&lt;/em&gt;&#xA0;=&#xA0;10; High Expression: &lt;em&gt;&#x3B7;&lt;/em&gt;&#xA0;=&#xA0;100; Low unassignment: unassigned reads&#xA0;=&#xA0;&lt;em&gt;z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;,&lt;em&gt;&lt;sub&gt;k&lt;/sub&gt;&lt;/em&gt;; High unassignment: unassigned reads&#xA0;=&#xA0;10,000 &#xD7; &lt;em&gt;z&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt;,&lt;em&gt;&lt;sub&gt;k&lt;/sub&gt;&lt;/em&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F3.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/2/447/F3.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488413" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-63" class="first-child">Type II error rate. Type II error rate when comparing an environment for which no AI was present (<em>α</em><sub>1</sub> = 1) with an environment with varying levels of AI. Differences in AI between environments are reported as log2 <em>α</em><sub>1</sub>/<em>α</em><sub>2</sub>. The two upper panels show Type II error rate in case of low gene expression, and the two bottom panels show Type II error rate in case of high gene expression. Panels on the left show results obtained by specifying the number of reads aligning equally well to both alleles as expected under the model in <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>. Panels on the right shows results obtained by multiplying that quantity by 10,000. We plot in red the type II error rate in environment 1, in blue the type II error rate for difference in AI between environments, and in green the type II error rate between the environments when using the descriptive approach as implemented in the baseline model. Each point represents the average over the seven simulated levels of bias misspecification, and the error bars represent SE. Low expression: <em>η</em> = 10; High Expression: <em>η</em> = 100; Low unassignment: unassigned reads = <em>z<sub>i</sub></em>,<em><sub>k</sub></em>; High unassignment: unassigned reads = 10,000 × <em>z<sub>i</sub></em>,<em><sub>k</sub></em></p><div class="sb-div caption-clear"></div></div></div><p id="p-64">Further analyses were performed simulating 10-fold difference in coverage between conditions. When higher coverage occurs in environment 2 (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a> left panel), type I error in environment 2 is higher, and vice versa (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a> right panel). The detection of differences in AI between environments might be negatively affected by different levels of coverage across the experiments. Our simulations show that in this case, as previously reported, the baseline model has a high type I error while the environmental model has very low type I error (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a>). This suggests that the environmental model does, as expected, protect from type I error under different coverage levels. Correspondingly, the Type II error is higher for the environmental model compared to the baseline model. This is especially true when the difference in α is small (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a>).</p></div><div id="sec-13" class="subsection"><h3>Reanalysis of D. melanogaster data</h3><div id="sec-14" class="subsection"><h4>Comparing the environmental model with the baseline model:</h4><p id="p-65">We provide here a comparison of the results from the environmental model to those published using the baseline model (<a id="xref-ref-16-13" class="xref-bibr" href="#ref-16">Fear <em>et al.</em> 2016</a>), for analysis of differences in AI across mated and virgin physiological states and F1 genotypes. A total of 62,448 data points (4880 different exons and 49 different lines) were included in the final analysis. On average, each exon had detectable expression in 14 lines (ranging from 1 to 49 detected), and each line contains 1375 exons (ranging from 446 to 2832 detected). Figures S3 and S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a> show, for each of the 49 lines, the proportion of exons flagged with AI in mated and virgin flies, respectively. The figures suggest that the two models have similar behavior in the analyzed lines, with the environmental model being generally more conservative and showing less extreme differences across lines. According to the environmental model, line r907 has the highest proportion of exons with AI both in mated (44%) and virgin flies (44%). According to the baseline model, line r907 has the highest proportion of exons with AI in mated flies (42%), and line r502 has the highest proportion of exons with AI in virgin flies (46%).</p><p id="p-66"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> reports the number of data points showing AI in mated and virgin flies, according to the baseline and environmental models, respectively. In general, the baseline model is less conservative than the environmental model, detecting 12,330 and 12,581 events of AI in mated and virgin flies, respectively, <em>vs.</em> 10,051 and 10,133 detected by the environmental model. Cohen’s kappa (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cohen 1960</a>) between the two models is 0.49 in both mated and virgin flies. Posterior estimates of ϑ were strongly correlated between the two models, with <em>R</em><sup>2</sup> of 0.92 in both mated and virgin flies (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488422/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488422/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number of data points (exons × lines) showing AI</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Posterior estimate of &#x3D1;. Posterior estimate of &#x3D1; according to the baseline and environmental model in mated (left) and virgin (right) flies, respectively. Each dot represents a data point. Black points: no AI in either set. Red Points: AI detected by baseline model alone. Blue points: AI detected by environmental model alone. Purple points: AI detected by both models. Blue line: bisector of the first quadrant angle. Red line: regression line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Posterior estimate of &#x3D1;. Posterior estimate of &#x3D1; according to the baseline and environmental model in mated (left) and virgin (right) flies, respectively. Each dot represents a data point. Black points: no AI in either set. Red Points: AI detected by baseline model alone. Blue points: AI detected by environmental model alone. Purple points: AI detected by both models. Blue line: bisector of the first quadrant angle. Red line: regression line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F4.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/2/447/F4.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488471" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-67" class="first-child">Posterior estimate of ϑ. Posterior estimate of ϑ according to the baseline and environmental model in mated (left) and virgin (right) flies, respectively. Each dot represents a data point. Black points: no AI in either set. Red Points: AI detected by baseline model alone. Blue points: AI detected by environmental model alone. Purple points: AI detected by both models. Blue line: bisector of the first quadrant angle. Red line: regression line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">We compared coverage of exons stratified by significance according to either environmental or baseline model. According to Wilcoxon’s test, coverage in exons not showing AI in either model was significantly lower than all other classes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The result is expected, since genes with low coverage are expected to provide lower power for the detection of AI (<a id="xref-ref-24-10" class="xref-bibr" href="#ref-24">León-Novelo <em>et al.</em> 2014</a>). Exons showing AI only according to the baseline model had higher coverage than all the remaining categories. This might be due to false positives in the baseline model. Simulation results showed that the power of the environmental model increases as coverage increases, with a relatively constant type I error rate, while the baseline model has an increase in type I error at higher coverage. This is likely due to the inclusion of the unassigned reads as an estimate of coverage in the environmental model. The coverage of exons for which only the environmental model showed AI and of exons for which both models detected AI, did not differ significantly.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Exon coverage and AI. Base&#xA0;10 logarithm of coverage of exons for which no model detected AI (gray boxes), only the environmental model detected AI (yellow boxes), only the baseline model detected AI (dark red boxes), or both models detected AI (orange boxes), respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Exon coverage and AI. Base&#xA0;10 logarithm of coverage of exons for which no model detected AI (gray boxes), only the environmental model detected AI (yellow boxes), only the baseline model detected AI (dark red boxes), or both models detected AI (orange boxes), respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F5.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/2/447/F5.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488533" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-69" class="first-child">Exon coverage and AI. Base 10 logarithm of coverage of exons for which no model detected AI (gray boxes), only the environmental model detected AI (yellow boxes), only the baseline model detected AI (dark red boxes), or both models detected AI (orange boxes), respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-70">As expected, the estimated deviation in AI from the null expectation is greater in exons for which AI was detected in both models (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). In addition, exons in which AI was detected by the environmental model showed higher median deviation than exons for which AI was detected by the baseline model. No significant difference between baseline and environmental model was observed for the estimates of ϑ in exons without AI.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deviation from expected &#x3D1; and AI. Distribution of the absolute value for the deviation of &#x3D1; from the expected value of 0.5 in exons for which AI was detected (&#x3B1;&#xA0;=&#xA0;1) or not (&#x3B1;&#xA0;&#x2260;&#xA0;1), according to the baseline (light pink) or environmental model (dark red). Differences between groups according to Wilcoxon pairwise test are shown. Groups sharing a letter are not different. Groups not sharing letters are different." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Deviation from expected &#x3D1; and AI. Distribution of the absolute value for the deviation of &#x3D1; from the expected value of 0.5 in exons for which AI was detected (&lt;em&gt;&#x3B1;&lt;/em&gt;&#xA0;=&#xA0;1) or not (&lt;em&gt;&#x3B1;&lt;/em&gt;&#xA0;&#x2260;&#xA0;1), according to the baseline (light pink) or environmental model (dark red). Differences between groups according to Wilcoxon pairwise test are shown. Groups sharing a letter are not different. Groups not sharing letters are different.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F6.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/2/447/F6.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488444" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-71" class="first-child">Deviation from expected ϑ and AI. Distribution of the absolute value for the deviation of ϑ from the expected value of 0.5 in exons for which AI was detected (<em>α</em> = 1) or not (<em>α</em> ≠ 1), according to the baseline (light pink) or environmental model (dark red). Differences between groups according to Wilcoxon pairwise test are shown. Groups sharing a letter are not different. Groups not sharing letters are different.</p><div class="sb-div caption-clear"></div></div></div><p id="p-72"><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows the BA plot comparing the two methods (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altman and Bland 1983</a>). The plot shows that a minority of data points elicit different estimates of ϑ between baseline and the environmental model, especially for intermediate estimates of ϑ. We compared the number of data points for which differential AI between conditions was detected by the environmental and baseline models stratifying by discrepancy in ϑ estimates (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a>). Data points with a difference in ϑ estimate (<em>i.e.</em>, those lying outside of the 95% confidence interval) showed stronger disagreement between the environmental and the baseline model in detecting differential AI across conditions (Fisher’s exact test p-value = 7.01E−07 for data stratified by difference in ϑ estimates in mated flies, and p-value = 2.27E−06 for data stratified by differences in virgin flies). The difference is mainly due to instances in which the baseline model detects differences in AI and the environmental model does not (p-value = 2.62E−07 and p-value = 1.79E−05 for estimates of ϑ in mated and virgin flies, respectively). Instances in which the environmental model alone detects differential AI between conditions do not vary between points lying inside or outside of the confidence intervals (p-value = 0.87 and p-value = 0.07 for estimates of ϑ in mated and virgin flies, respectively). This means that when the estimates of ϑ differ, the baseline model shows a smaller difference in AI across conditions.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BA plot. BA plot comparing estimates of &#x3D1; according to the baseline (Base) and environmental (Env) model in mated and virgin flies, respectively. The red line represents the mean difference. The dotted lines are the 95% confidence intervals of the difference. The regression of the difference over the mean is represented by the blue line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;BA plot. BA plot comparing estimates of &#x3D1; according to the baseline (Base) and environmental (Env) model in mated and virgin flies, respectively. The red line represents the mean difference. The dotted lines are the 95% confidence intervals of the difference. The regression of the difference over the mean is represented by the blue line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F7.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/2/447/F7.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488368" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-73" class="first-child">BA plot. BA plot comparing estimates of ϑ according to the baseline (Base) and environmental (Env) model in mated and virgin flies, respectively. The red line represents the mean difference. The dotted lines are the 95% confidence intervals of the difference. The regression of the difference over the mean is represented by the blue line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-74">The baseline model uses a descriptive approach to consider AI differences in mating status, by which an exon is classified as showing different AI values in the two statuses, if and only if, it shows AI only in one status. This descriptive approach can be applied to the environmental model (in addition to the explicit test of the null hypothesis), and we implemented this approach to further compare the two models.</p><p id="p-75">Using the baseline model, different AI between environments is declared when AI is present in one of the two environments alone; using the environmental model, it is possible to both identify significantly different levels of AI between environments with a direct test, and indirectly as a comparison between the individual tests within each environment. Using the three possible approaches, we plot in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> the distribution of ϑ in exons for which no AI was detected (Both <em>α</em> = 1), AI was detected in one environment (One <em>α</em> = 1), and AI was detected in both environments (Both <em>α</em> ≠ 1).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model comparison. Distribution of absolute difference in &#x3D1; estimates between mated and virgin flies in several contrasts according to different models. Both &#x3B1;&#xA0;=&#xA0;1: AI not detected in mated nor virgin flies. One &#x3B1;&#xA0;=&#xA0;1: AI detected in one mating status alone. Both &#x3B1;&#xA0;&#x2260;&#xA0;1: AI detected in both mated and virgin flies. Classifications are based on results obtained by the baseline model, for which the difference in AI between environments is reported using a descriptive approach (red), the environmental model, using the descriptive approach for describing difference in AI (dark blue), and the environmental model, using a formal Bayesian test of the null hypothesis for detecting difference in AI (light blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Model comparison. Distribution of absolute difference in &#x3D1; estimates between mated and virgin flies in several contrasts according to different models. Both &lt;em&gt;&#x3B1;&lt;/em&gt;&#xA0;=&#xA0;1: AI not detected in mated nor virgin flies. One &lt;em&gt;&#x3B1;&lt;/em&gt;&#xA0;=&#xA0;1: AI detected in one mating status alone. Both &lt;em&gt;&#x3B1;&lt;/em&gt;&#xA0;&#x2260;&#xA0;1: AI detected in both mated and virgin flies. Classifications are based on results obtained by the baseline model, for which the difference in AI between environments is reported using a descriptive approach (red), the environmental model, using the descriptive approach for describing difference in AI (dark blue), and the environmental model, using a formal Bayesian test of the null hypothesis for detecting difference in AI (light blue).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F8.medium.gif" width="440" height="407"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/2/447/F8.medium.gif" width="440" height="407"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488529" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-76" class="first-child">Model comparison. Distribution of absolute difference in ϑ estimates between mated and virgin flies in several contrasts according to different models. Both <em>α</em> = 1: AI not detected in mated nor virgin flies. One <em>α</em> = 1: AI detected in one mating status alone. Both <em>α</em> ≠ 1: AI detected in both mated and virgin flies. Classifications are based on results obtained by the baseline model, for which the difference in AI between environments is reported using a descriptive approach (red), the environmental model, using the descriptive approach for describing difference in AI (dark blue), and the environmental model, using a formal Bayesian test of the null hypothesis for detecting difference in AI (light blue).</p><div class="sb-div caption-clear"></div></div></div><p id="p-77">The expectation is that, when the models detected AI in one environment and not in the other (coded here as “One <em>α</em> = 1”), the difference in the ϑ distributions is greater than for other situations. This was the case for all the approaches, but the larger difference was observed with the direct test of the null hypothesis <em>a</em>1 = <em>a</em>2 in the environmental model (Coffman <em>et al.</em> 2003).</p><p id="p-78">We counted the number of genes for which at least one exon showed AI in at least one line in mated and virgin flies, respectively; results are shown in <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of AI detected in mated and virgin flies by the two models. Numbers represents the number of genes. A gene was classified as having &#x3B1;&#xA0;&#x2260;&#xA0;1 when at least one exon in at least one line had &#x3B1;&#xA0;&#x2260;&#xA0;1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of AI detected in mated and virgin flies by the two models. Numbers represents the number of genes. A gene was classified as having &lt;em&gt;&#x3B1;&lt;/em&gt;&#xA0;&#x2260;&#xA0;1 when at least one exon in at least one line had &lt;em&gt;&#x3B1;&lt;/em&gt;&#xA0;&#x2260;&#xA0;1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F9.medium.gif" width="440" height="398"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/2/447/F9.medium.gif" width="440" height="398"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488515" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-79" class="first-child">Comparison of AI detected in mated and virgin flies by the two models. Numbers represents the number of genes. A gene was classified as having <em>α</em> ≠ 1 when at least one exon in at least one line had <em>α</em> ≠ 1.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488397/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488397/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Comparison of AI detected in mated and virgin flies by the two models</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>Analysis of AI in mated and virgin flies using the environmental model:</h4><p id="p-81">Analysis of AI using the environmental model was performed on a total of 169,842 exons in lines, belonging to 13,898 different exons in 68 crosses. On average, each exon was analyzed in 12 crosses (range 1–68). The distribution of ϑ in the whole data set is shown in Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1/FileS1.docx">File S1</a>. As expected, the distribution of ϑ is similar in the two environments—mated and virgin flies—and is centered on 0.5.</p><p id="p-82"><a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a> shows the distribution of the proportion of exons showing AI in mated and virgin flies, separately, together with the proportion of exons showing a difference in AI between mated and virgin flies. The proportion of exons with a significant difference in AI across environments is significantly lower than the proportion of exons with AI in either one of the environments. This suggests that genetic regulation is generally robust to environmental changes, with the exception of line r149, which is clearly visible as the outlier in the third bar of the boxplot.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/447/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of exons showing AI. Distribution of proportion of exons showing AI in mated flies, virgin flies, and with different levels of AI between the two mating statuses. Samples that have significantly different distributions are represented by different letters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-593170181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of exons showing AI. Distribution of proportion of exons showing AI in mated flies, virgin flies, and with different levels of AI between the two mating statuses. Samples that have significantly different distributions are represented by different letters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/447/F10.medium.gif" width="440" height="375"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/8/2/447/F10.medium.gif" width="440" height="375"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/447/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/447/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488325" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-83" class="first-child">Proportion of exons showing AI. Distribution of proportion of exons showing AI in mated flies, virgin flies, and with different levels of AI between the two mating statuses. Samples that have significantly different distributions are represented by different letters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-84">To identify genes for which <em>cis</em> regulatory variation is either responsive or robust to environmental changes, we list genes that show an excess of AI variation in response to environmental changes, and those that show no variation in AI in response to environmental changes.</p><p id="p-85">We performed Fisher’s exact test to determine if the proportion of lines showing AI in a given exon was significantly higher (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>) or lower (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>) than the proportion of lines showing AI for all other exons. While the environmental approach does not depend on population frequency of variants, this further analysis does. By searching for excess (or depletion) of differences in AI across environments in the study population, our power depends on the number of exons that can be tested in the population, and this number is smaller for rare variants. This is especially true for <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>, where a significant depletion of AI across environments can be detected only when a large number of exons (50 or more) have been tested, and none show differences of AI across environments.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488387/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488387/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">List of the 20 exons showing the highest proportion of differences in AI between mating statuses</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488480/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488480/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">List of the 20 exons showing the least proportion of differences in AI between mating statuses</span><div class="sb-div caption-clear"></div></div></div><p id="p-100"><a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a> lists the 20 genes with the highest excess of lines showing differences in AI between mated and virgin flies, according to Fisher’s exact test. These are the genes that show the highest levels of <em>cis</em> by environment variation, which suggests regulatory genetic variation in the response to mating itself.</p><p id="p-101"><a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a> lists the 20 genes showing the least difference in AI between mated and virgin flies, according to Fisher’s exact test. These are genes that show unusually low levels of <em>cis</em> by environment variation, compared to other genes in the genome, suggesting robust <em>cis</em> activity. Another possible interpretation is that the allele frequency for different regulatory alleles in this population is low.</p><p id="p-102">The environmental model can test variation of AI across environments, genotypes, sex, and other conditions. We compared AI levels between the F1’s r365×w1118 and r907×w1118—the two genotypes having, respectively, the lower and higher proportion of exons with AI. A total of 406 exons was analyzed in both lines and used for this analysis. In mated flies, AI was detected in 16 exons in line r365 and 124 in line r907, while in virgin flies, AI was detected in 14 exons in line r365 and 123 in line r907. Difference in AI between r365 and r907 was detected in 88 and 83 exons, respectively, or ∼20%, indicating a large number of differences between alleles in these two F1s.</p></div></div><div id="sec-16" class="subsection"><h3>Conclusions</h3><p id="p-103">We present a direct approach for simultaneously testing of AI and differences in AI across environments. We measured performance using simulated data, showing that the method has relatively low Type I and Type II error rates. Our model also accounts for differences in magnitude of gene expression counts and unassigned reads, while providing a more conservative method to estimate AI differences between different environments. Our model protects from increases in type I error even when the number of sequenced reads in the two environments differ by 10-fold. We reanalyzed published data to further investigate robustness of AI to environmental conditions. Our results indicate that gene regulation is substantially robust to environmental changes, with a small number of notable exceptions among genes whose expression is affected by mating status. An analysis directly comparing two different F1s shows ∼20% of the exons have differences in <em>cis</em> regulation among lines; this estimate is on par with the detection of <em>cis</em> differences in regulation in the population, and demonstrates that the environmental model can be used to test G×G across F1s as well as G×E effects.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-104">This work was partly financed by the Marie Curie International Research Staff Exchange Scheme (IRSES) Project “Developing an European American Next Generation Sequencing (NGS) Network (DEANN)” (grant agreement number: PIRSES-GA-2013-612583), by National Institutes of Health (NIH) grant National Institute of General Medical Sciences (NIGMS) 102227 to L.M.M., and by the European Research Council under the European Union’s Seventh Framework Program (FP7/2007–2013, grant agreement number 294780, “Novabreed”). Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information, and does not imply recommendation or endorsement by the United States Department of Agriculture (USDA). USDA is an equal opportunity employer. The authors declare no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-17"><p id="p-105">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300139/-/DC1</a>.</p></li><li class="fn" id="fn-18"><p id="p-106"><em>Communicating editor: R. W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-11"><span class="received-label">Received </span>August 11, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-18"><span class="accepted-label">Accepted </span>November 18, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Novelo <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.1" data-doi="10.1016/0531-5565(84)90022-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aigaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohba</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Effect of mating status on <em>Drosophila virilis</em> lifespan.</span> <abbr class="cit-jnl-abbrev">Exp. Gerontol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Bgerontology%26rft.stitle%253DExp%2BGerontol%26rft.aulast%253DAigaki%26rft.auinit1%253DT.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D267%26rft.epage%253D278%26rft.atitle%253DEffect%2Bof%2Bmating%2Bstatus%2Bon%2BDrosophila%2Bvirilis%2Blifespan.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0531-5565%252884%252990022-6%26rft_id%253Dinfo%253Apmid%252F6499967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0531-5565(84)90022-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6499967&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TB45100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bland</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Measurement in medicine: the analysis of method comparison studies.</span> <abbr class="cit-jnl-abbrev">Stat.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%26rft.volume%253D32%26rft.spage%253D307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.3" data-doi="10.1186/gb-2010-11-10-r106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression analysis for sequence count data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAnders%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253DR106%26rft.epage%253DR106%26rft.atitle%253DDifferential%2Bexpression%2Banalysis%2Bfor%2Bsequence%2Bcount%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-10-r106%26rft_id%253Dinfo%253Apmid%252F20979621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-10-r106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20979621&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-8.2.447.4" data-doi="10.1126/science.1069516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yvert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clinton</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic dissection of transcriptional regulation in budding yeast.</span> <span class="cit-source">Science</span> <span class="cit-vol">296</span>: <span class="cit-fpage">752</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069516%26rft_id%253Dinfo%253Apmid%252F11923494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTYvNTU2OC83NTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.5" data-doi="10.1038/ng.3162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buil</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lappalainen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viñuela</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene-gene and gene-environment interactions detected by transcriptome sequence analysis in twins.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D88%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3162%26rft_id%253Dinfo%253Apmid%252F25436857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25436857&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.6" data-doi="10.1186/s13059-015-0762-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castel</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy-Moonshine</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lappalainen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tools and best practices for data processing in allelic expression analysis.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0762-6%26rft_id%253Dinfo%253Apmid%252F26381377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0762-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26381377&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.447.7" data-doi="10.1371/journal.pgen.1004883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Temperature stress mediates decanalization and dominance of gene expression in <em>Drosophila melanogaster</em>.</span> <span class="cit-source">PLOS Genet.</span> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004883%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004883%26rft_id%253Dinfo%253Apmid%252F25719753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25719753&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.8" data-doi="10.1177/001316446002000104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">A coefficient of agreement for nominal scales.</span> <abbr class="cit-jnl-abbrev">Educ. Psychol. Meas.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEducational%2Band%2BPsychological%2BMeasurement%26rft.stitle%253DEducational%2Band%2BPsychological%2BMeasurement%26rft.aulast%253DCohen%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D46%26rft.atitle%253DA%2BCoefficient%2Bof%2BAgreement%2Bfor%2BNominal%2BScales%26rft_id%253Dinfo%253Adoi%252F10.1177%252F001316446002000104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/001316446002000104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1960CCC3600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.2.447.9"><div class="cit-metadata"><cite>Coffman, C., R. W. Doerge, M. L. Wayne, and L. M. McIntyre, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Intersection tests for single marker QTL analysis can be more powerful than two marker QTL analysis.</span> <abbr class="cit-jnl-abbrev">BMC Genetics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenetics%26rft.volume%253D4%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.10" data-doi="10.1038/ng.3222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crowley</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhabotynsky</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pakatci</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analyses of allele-specific gene expression in highly divergent mouse crosses identifies pervasive allelic imbalance.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D353%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3222%26rft_id%253Dinfo%253Apmid%252F25730764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25730764&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.11" data-doi="10.1105/tpc.114.130310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grondin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canut</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tisné</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Extensive cis-regulatory variation robust to environmental perturbation in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4298</span>–<span class="cit-lpage">4310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.130310%26rft_id%253Dinfo%253Apmid%252F25428981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMS80Mjk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.12" data-doi="10.1093/bioinformatics/btp579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Degner</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marioni</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nkadori</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effect of read-mapping biases on detecting allele-specific expression from RNA-sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3207</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp579%26rft_id%253Dinfo%253Apmid%252F19808877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19808877&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272464000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cumbie</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The NBP negative binomial model for assessing differential gene expression from RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edsgärd</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iglesias</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reilly</span>  <span class="cit-name-given-names">S.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamsten</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornvall</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">GeneiASE: detection of condition-dependent and static allele-specific expression from RNA-seq data without haplotype information.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">21134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D21134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.447.15" data-doi="10.1371/journal.pone.0009607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everaerts</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farine</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferveur</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Drosophila</em> cuticular hydrocarbons revisited: mating status alters cuticular profiles.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">5</span>: <span class="cit-fpage">e9607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEveraerts%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9607%26rft.epage%253De9607%26rft.atitle%253DDrosophila%2Bcuticular%2Bhydrocarbons%2Brevisited%253A%2Bmating%2Bstatus%2Balters%2Bcuticular%2Bprofiles.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009607%26rft_id%253Dinfo%253Apmid%252F20231905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009607&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20231905&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.16" data-doi="10.1534/genetics.116.188797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fear</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon-Novelo</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerken</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Lehmann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Buffering of genetic regulatory networks in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.188797%26rft_id%253Dinfo%253Apmid%252F27194752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMy8zLzExNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-other" id="cit-8.2.447.17"><div class="cit-metadata unstructured">Graze, R. M., L. M. McIntyre, B. J. Main, M. L. Wayne, and S. V Nuzhdin, 2009 Regulatory divergence in <em>Drosophila melanogaster</em> and <em>D. simulans</em>, a genome-wide analysis of allele-specific expression. Genetics 183: 547–561, 1SI–21SI.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.18" data-doi="10.1093/molbev/msr318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novelo</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fear</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Allelic imbalance in <em>Drosophila</em> hybrid heads: exons, isoforms, and evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1521</span>–<span class="cit-lpage">1532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr318%26rft_id%253Dinfo%253Apmid%252F22319150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22319150&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304200600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.19" data-doi="10.1093/gbe/evu060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">What the X has to do with it: differences in regulatory variability between the sexes in <em>Drosophila simulans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">818</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu060%26rft_id%253Dinfo%253Apmid%252F24696400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24696400&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.20" data-doi="10.1126/science.1190830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissbourd</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroth</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High-resolution analysis of parent-of-origin allelic expression in the mouse brain.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190830%26rft_id%253Dinfo%253Apmid%252F20616232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5Mi82NDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgington</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Favé</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Allele-specific expression reveals interactions between genetic variation and environment.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.22" data-doi="10.1186/s12864-015-1415-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurmangaliyev</span>  <span class="cit-name-given-names">Y. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Favorov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osman</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">K.-V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Natural variation of gene models in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D198%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1415-6%26rft_id%253Dinfo%253Apmid%252F25888292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1415-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25888292&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.23" data-doi="10.1139/g04-050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawniczak</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genome-wide analysis of courting and mating responses in <em>Drosophila melanogaster</em> females.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">900</span>–<span class="cit-lpage">910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLawniczak%26rft.auinit1%253DM.%2BK.%26rft.volume%253D47%26rft.issue%253D5%26rft.spage%253D900%26rft.epage%253D910%26rft.atitle%253DA%2Bgenome-wide%2Banalysis%2Bof%2Bcourting%2Band%2Bmating%2Bresponses%2Bin%2BDrosophila%2Bmelanogaster%2Bfemales.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg04-050%26rft_id%253Dinfo%253Apmid%252F15499404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g04-050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15499404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.24" data-doi="10.1186/1471-2164-15-920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">León-Novelo</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fear</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A flexible Bayesian method for detecting allelic imbalance in RNA-seq data.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D920%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-920%26rft_id%253Dinfo%253Apmid%252F25339465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25339465&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">León-Novelo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuentes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Marginal likelihood estimation of negative binomial parameters with applications to RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft.volume%253D18%26rft.spage%253D637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.447.26" data-doi="10.1371/journal.pone.0044017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hrabovsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedrosa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Allele-biased expression in differentiating human neurons: implications for neuropsychiatric disorders.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">7</span>: <span class="cit-fpage">e44017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLin%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De44017%26rft.epage%253De44017%26rft.atitle%253DAllele-biased%2Bexpression%2Bin%2Bdifferentiating%2Bhuman%2Bneurons%253A%2Bimplications%2Bfor%2Bneuropsychiatric%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0044017%26rft_id%253Dinfo%253Apmid%252F22952857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0044017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22952857&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Allelic variation in gene expression is common in the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1855</span>–<span class="cit-lpage">1862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLo%26rft.auinit1%253DH.%2BS.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1855%26rft.epage%253D1862%26rft.atitle%253DAllelic%2BVariation%2Bin%2BGene%2BExpression%2BIs%2BCommon%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Apmid%252F12902379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTg1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.28" data-doi="10.1126/science.1222794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maurano</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humbert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rynes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haugen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Systematic localization of common disease-associated variation in regulatory DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">1190</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1222794%26rft_id%253Dinfo%253Apmid%252F22955828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzcvNjA5OS8xMTkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.29" data-doi="10.1038/ng.215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huett</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuballa</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chilewski</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Deletion polymorphism upstream of IRGM associated with altered IRGM expression and Crohn’s disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1107</span>–<span class="cit-lpage">1112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMcCarroll%26rft.auinit1%253DS.%2BA.%26rft.volume%253D40%26rft.issue%253D9%26rft.spage%253D1107%26rft.epage%253D1112%26rft.atitle%253DDeletion%2Bpolymorphism%2Bupstream%2Bof%2BIRGM%2Bassociated%2Bwith%2Baltered%2BIRGM%2Bexpression%2Band%2BCrohn%2527s%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.215%26rft_id%253Dinfo%253Apmid%252F19165925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165925&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258761200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.30" data-doi="10.1534/genetics.108.086934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGraw</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfner</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Post-mating gene expression profiles of female <em>Drosophila melanogaster</em> in response to time and to four male accessory gland proteins.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1395</span>–<span class="cit-lpage">1408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.086934%26rft_id%253Dinfo%253Apmid%252F18562649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzEzOTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.31" data-doi="10.1101/gr.102491.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McManus</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coolon</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eipper-Mains</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regulatory divergence in <em>Drosophila</em> revealed by mRNA-seq.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102491.109%26rft_id%253Dinfo%253Apmid%252F20354124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvODE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.32" data-doi="10.1101/gr.209759.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moyerbrailean</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalita</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">G. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-throughput allele-specific expression across 250 environmental conditions.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1627</span>–<span class="cit-lpage">1638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.209759.116%26rft_id%253Dinfo%253Apmid%252F27934696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi8xNjI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.33" data-doi="10.1534/genetics.114.165886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munger</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghupathy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simons</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNA-Seq alignment to individualized genomes improves transcript abundance estimates in multiparent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMunger%26rft.auinit1%253DS.%2BC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D73%26rft.atitle%253DRNA-Seq%2BAlignment%2Bto%2BIndividualized%2BGenomes%2BImproves%2BTranscript%2BAbundance%2BEstimates%2Bin%2BMultiparent%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165886%26rft_id%253Dinfo%253Apmid%252F25236449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk4LzEvNTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.447.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.35" data-doi="10.1093/biostatistics/kxm030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Small-sample estimation of negative binomial dispersion, with applications to SAGE data.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft.volume%253D9%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxm030%26rft_id%253Dinfo%253Apmid%252F17728317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxm030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17728317&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.36" data-doi="10.1093/oxfordjournals.molbev.a004023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Abundant raw material for cis-regulatory evolution in humans.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1991</span>–<span class="cit-lpage">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D1991%26rft.epage%253D2004%26rft.atitle%253DAbundant%2BRaw%2BMaterial%2Bfor%2BCis-Regulatory%2BEvolution%2Bin%2BHumans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004023%26rft_id%253Dinfo%253Apmid%252F12411608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a004023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12411608&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179367200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.37" data-doi="10.1101/gr.2850605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yvert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Simultaneous genotyping, gene-expression measurement, and detection of allele-specific expression with oligonucleotide arrays.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DRonald%26rft.auinit1%253DJ.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D284%26rft.epage%253D291%26rft.atitle%253DSimultaneous%2Bgenotyping%252C%2Bgene-expression%2Bmeasurement%252C%2Band%2Bdetection%2Bof%2Ballele-specific%2Bexpression%2Bwith%2Boligonucleotide%2Barrays%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2850605%26rft_id%253Dinfo%253Apmid%252F15687292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzIvMjg0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.38" data-doi="10.1038/msb.2011.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozowsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abyzov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raha</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">AlleleSeq: analysis of allele-specific expression and binding in a network framework.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DRozowsky%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D522%26rft.epage%253D522%26rft.atitle%253DAlleleSeq%253A%2Banalysis%2Bof%2Ballele-specific%2Bexpression%2Band%2Bbinding%2Bin%2Ba%2Bnetwork%2Bframework%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.54%26rft_id%253Dinfo%253Apmid%252F21811232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81MjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.39" data-doi="10.1093/nar/gks425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Satya</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zavaljevski</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reifman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A new strategy to reduce allelic bias in RNA-Seq readmapping.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks425%26rft_id%253Dinfo%253Apmid%252F22584625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22584625&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.40" data-doi="10.1101/gr.119784.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madeoy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakefield</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A powerful and flexible statistical framework for testing hypotheses of allele-specific gene expression from RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1728</span>–<span class="cit-lpage">1737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.119784.110%26rft_id%253Dinfo%253Apmid%252F21873452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMC8xNzI4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.41" data-doi="10.1186/1471-2164-14-536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevenson</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coolon</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittkopp</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sources of bias in measures of allele-specific expression derived from RNA-sequence data aligned to a single reference genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D536%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-536%26rft_id%253Dinfo%253Apmid%252F23919664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23919664&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.447.42" data-doi="10.1371/journal.pgen.1002639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranger</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimas</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Patterns of cis regulatory variation in diverse human populations.</span> <span class="cit-source">PLoS Genet.</span> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DStranger%26rft.auinit1%253DB.%2BE.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1002639%26rft.epage%253De1002639%26rft.atitle%253DPatterns%2Bof%2Bcis%2Bregulatory%2Bvariation%2Bin%2Bdiverse%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002639%26rft_id%253Dinfo%253Apmid%252F22532805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002639&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22532805&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.43" data-doi="10.1111/j.1365-294X.2010.04970.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akinyi</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mutura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Allele-specific gene expression in a wild nonhuman primate population.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DTung%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D725%26rft.epage%253D739%26rft.atitle%253DAllele-specific%2Bgene%2Bexpression%2Bin%2Ba%2Bwild%2Bnonhuman%2Bprimate%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04970.x%26rft_id%253Dinfo%253Apmid%252F21226779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04970.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21226779&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286837300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.44" data-doi="10.1186/gb-2011-12-2-r13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turro</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">Â.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coin</span>  <span class="cit-name-given-names">L. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Haplotype and isoform specific expression estimation using multi-mapping RNA-seq reads.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DTurro%26rft.auinit1%253DE.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253DR13%26rft.epage%253DR13%26rft.atitle%253DHaplotype%2Band%2Bisoform%2Bspecific%2Bexpression%2Bestimation%2Busing%2Bmulti-mapping%2BRNA-seq%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-2-r13%26rft_id%253Dinfo%253Apmid%252F21310039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-2-r13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21310039&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.45" data-doi="10.1111/j.1365-313X.2009.03848.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Korff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radovic</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choumane</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamati</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udupa</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Asymmetric allele-specific expression in relation to developmental variation and drought stress in barley hybrids.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253Dvon%2BKorff%26rft.auinit1%253DM.%26rft.volume%253D59%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D26%26rft.atitle%253DAsymmetric%2Ballele-specific%2Bexpression%2Bin%2Brelation%2Bto%2Bdevelopmental%2Bvariation%2Band%2Bdrought%2Bstress%2Bin%2Bbarley%2Bhybrids.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03848.x%26rft_id%253Dinfo%253Apmid%252F19309461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03848.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19309461&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267428400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uebbing</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bayesian inference of allele-specific gene expression indicates abundant <em>cis</em>-regulatory variation in natural flycatcher populations.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1266</span>–<span class="cit-lpage">1279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D9%26rft.spage%253D1266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.47" data-doi="10.1038/nature02698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittkopp</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haerum</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evolutionary changes in <em>cis</em> and <em>trans</em> gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWittkopp%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D430%26rft.issue%253D6995%26rft.spage%253D85%26rft.epage%253D88%26rft.atitle%253DEvolutionary%2Bchanges%2Bin%2Bcis%2Band%2Btrans%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02698%26rft_id%253Dinfo%253Apmid%252F15229602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15229602&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222356800048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-other" id="cit-8.2.447.48" data-doi="10.1126/science.1072545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velculescu</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogelstein</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinzler</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Allelic variation in human gene expression.</span> <span class="cit-source">Science</span> <span class="cit-vol">297</span>: <span class="cit-fpage">1143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYan%26rft.auinit1%253DH.%26rft.volume%253D297%26rft.issue%253D5584%26rft.spage%253D1143%26rft.epage%253D1143%26rft.atitle%253DAllelic%2BVariation%2Bin%2BHuman%2BGene%2BExpression%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1072545%26rft_id%253Dinfo%253Apmid%252F12183620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTcvNTU4NC8xMTQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.49" data-doi="10.1534/genetics.109.103499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Global analysis of allele-specific expression in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103499%26rft_id%253Dinfo%253Apmid%252F19474198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzQvOTQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.447.50" data-doi="10.1186/1471-2164-15-927"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anholt</span>  <span class="cit-name-given-names">R. R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptional and epigenetic responses to mating and aging in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D927%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-927%26rft_id%253Dinfo%253Apmid%252F25344338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-927&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25344338&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/447.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/427" title="Bayesian Diallel Analysis Reveals Mx1-Dependent and Mx1-Independent Effects on Response to Influenza A Virus in Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/461" title="Selection Mapping Identifies Loci Underpinning Autumn Dormancy in Alfalfa (Medicago sativa)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/447" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wTapHkofPOfcifZErtCwGg8mBSoQ_wQM-t2heFmbcW4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/447" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/447" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/447" class="active" data-icon-position="" data-hide-link-title="0">Direct Testing for Allele-Specific Expression Differences Between Conditions</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488291" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Lx_fXhzwQAGHoWbLiWyMazgI51edRBvMCBUreYYBDYQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/447.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/447" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-XvNcCWoQTJUTDCWy3aXIpae79VzN39JIU5cuYBuD7-M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/447" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/447.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488291" id="node488291--2" data-pisa="ggg;8/2/447" data-pisa-master="ggg;g3.117.300139" data-apath="/ggg/8/2/447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Direct Testing for Allele-Specific Expression Differences Between Conditions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Luis León-Novelo</span>, <span class="highwire-citation-author" data-delta="1">Alison R. Gerken</span>, <span class="highwire-citation-author" data-delta="2">Rita M. Graze</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lauren M. McIntyre</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1556-5907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabio Marroni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">447-460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300139 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488291--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis León-Novelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Data Science, The University of Texas Health Science Center at Houston-School of Public Health, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BLe%C3%B3n-Novelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%C3%B3n-Novelo%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuis%2BLe%25C3%25B3n-Novelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alison R. Gerken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture, Agricultural Research Service, Center for Grain and Animal Health Research, Manhattan, Kansas 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BR.%2BGerken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerken%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlison%2BR.%2BGerken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rita M. Graze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, Auburn University, Alabama 36849</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BM.%2BGraze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graze%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BM.%2BGraze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Institute and Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, Florida 32603</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lauren M. McIntyre</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fabio Marroni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Dipartimento di Scienze Agroalimentari, Ambientali e Animali, Università di Udine, 33100, Italy</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Istituto di Genomica Applicata, 33100 Udine, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabio%2BMarroni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marroni%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabio%2BMarroni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1556-5907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabio Marroni</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marroni@appliedgenomics.org" class="" data-icon-position="" data-hide-link-title="0">marroni@appliedgenomics.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/447" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/447&amp;title=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/447&amp;title=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/447&amp;title=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/447&amp;text=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/447&amp;title=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/447&amp;t=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/447&amp;title=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/447&amp;title=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488291/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488291" id="node488291--3" data-pisa="ggg;8/2/447" data-pisa-master="ggg;g3.117.300139" data-apath="/ggg/8/2/447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Direct Testing for Allele-Specific Expression Differences Between Conditions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Luis León-Novelo</span>, <span class="highwire-citation-author" data-delta="1">Alison R. Gerken</span>, <span class="highwire-citation-author" data-delta="2">Rita M. Graze</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lauren M. McIntyre</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1556-5907" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabio Marroni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">447-460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300139 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488291--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luis León-Novelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Data Science, The University of Texas Health Science Center at Houston-School of Public Health, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BLe%C3%B3n-Novelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%C3%B3n-Novelo%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuis%2BLe%25C3%25B3n-Novelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alison R. Gerken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture, Agricultural Research Service, Center for Grain and Animal Health Research, Manhattan, Kansas 66502</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BR.%2BGerken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gerken%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlison%2BR.%2BGerken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rita M. Graze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, Auburn University, Alabama 36849</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BM.%2BGraze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graze%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BM.%2BGraze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genetics Institute and Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, Florida 32603</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0077-3359" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lauren M. McIntyre</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fabio Marroni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Dipartimento di Scienze Agroalimentari, Ambientali e Animali, Università di Udine, 33100, Italy</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Istituto di Genomica Applicata, 33100 Udine, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabio%2BMarroni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marroni%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabio%2BMarroni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1556-5907" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabio Marroni</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marroni@appliedgenomics.org" class="" data-icon-position="" data-hide-link-title="0">marroni@appliedgenomics.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=447&amp;rft.epage=460&amp;rft.atitle=Direct%20Testing%20for%20Allele-Specific%20Expression%20Differences%20Between%20Conditions&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Le%C3%B3n-Novelo%2C+Luis&amp;rft.au=Gerken%2C+Alison+R.&amp;rft.au=Graze%2C+Rita+M.&amp;rft.au=McIntyre%2C+Lauren+M.&amp;rft.au=Marroni%2C+Fabio"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488291/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488291/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488291/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488291/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488291/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488291/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488291/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488291/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488291/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488291/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488291/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488291/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/447" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/447.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/447.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/447.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/447.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
