<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/401" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/401.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/401.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/401.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/401.atom" />
<meta name="HW.pisa" content="ggg;8/2/401" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice" />
<meta name="DC.Identifier" content="10.1534/g3.117.300292" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Geister et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Positional cloning of ENU-induced mutations has traditionally relied on analysis of polymorphic variation between two strains. In contrast, the application of whole-genome sequencing (WGS) has enabled gene discovery in mutant lines maintained on an inbred genetic background. This approach utilizes genetic variation derived from ENU-induced variants for mapping and reduces the likelihood of phenotypic variation, making it an ideal method for genetic modifier screening. Here, we describe the results of such a screen, wherein we determined the minimal number of mutant genomic DNA samples to include in our analyses and improved the sensitivity of our screen by individually barcoding each genomic DNA library. We present several unique cases to illustrate this approach’s efficacy, including the discovery of two distinct mutations that generate essentially identical mutant phenotypes, the ascertainment of a non-ENU-induced candidate variant through homozygosity mapping, and an approach for the identification of putative dominant genetic modifiers." />
<meta name="DC.Contributor" content="Krista A. Geister" />
<meta name="DC.Contributor" content="Andrew E. Timms" />
<meta name="DC.Contributor" content="David R. Beier" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Mutant Screen Report" />
<meta name="citation_title" content="Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Positional cloning of ENU-induced mutations has traditionally relied on analysis of polymorphic variation between two strains. In contrast, the application of whole-genome sequencing (WGS) has enabled gene discovery in mutant lines maintained on an inbred genetic background. This approach utilizes genetic variation derived from ENU-induced variants for mapping and reduces the likelihood of phenotypic variation, making it an ideal method for genetic modifier screening. Here, we describe the results of such a screen, wherein we determined the minimal number of mutant genomic DNA samples to include in our analyses and improved the sensitivity of our screen by individually barcoding each genomic DNA library. We present several unique cases to illustrate this approach’s efficacy, including the discovery of two distinct mutations that generate essentially identical mutant phenotypes, the ascertainment of a non-ENU-induced candidate variant through homozygosity mapping, and an approach for the identification of putative dominant genetic modifiers.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/401" />
<meta name="citation_id" content="8/2/401" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/401" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/401.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/401.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/401.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300292" />
<meta name="citation_pmid" content="29208648" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mutant Screen Report" />
<meta name="citation_firstpage" content="401" />
<meta name="citation_lastpage" content="409" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Krista A. Geister" />
<meta name="citation_author_institution" content="Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Washington School of Medicine, Seattle, Washington 98195" />
<meta name="citation_author" content="Andrew E. Timms" />
<meta name="citation_author_institution" content="Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101" />
<meta name="citation_author" content="David R. Beier" />
<meta name="citation_author_institution" content="Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101" />
<meta name="citation_author_institution" content="Department of Pediatrics, University of Washington School of Medicine, Seattle, Washington 98195" />
<meta name="citation_author_email" content="david.beier@seattlechildrens.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0325-8520" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=E. Arteaga-Solis;citation_author=B. Gayraud;citation_author=S. Y. Lee;citation_author=L. Shum;citation_author=L. Sakai;citation_title=Regulation of limb patterning by extracellular microfibrils;citation_pages=275-282;citation_volume=154;citation_year=2001;citation_issue=2;citation_pmid=11470817;citation_doi=10.1083/jcb.200105046" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=V. L. Browning;citation_author=S. S. Chaudhry;citation_author=A. Planchart;citation_author=M. J. Dixon;citation_author=J. C. Schimenti;citation_title=Mutations of the mouse Twist and sy (fibrillin 2) genes induced by chemical mutagenesis of ES cells.;citation_pages=291-298;citation_volume=73;citation_year=2001;citation_issue=3;citation_pmid=11350121;citation_doi=10.1006/geno.2001.6523" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. R. Bull;citation_author=A. J. Rimmer;citation_author=O. M. Siggs;citation_author=L. A. Miosge;citation_author=C. M. Roots;citation_title=Unlocking the bottleneck in forward genetics using whole-genome sequencing and identity by descent to isolate causative mutations.;citation_pages=e1003219;citation_volume=9;citation_year=2013;citation_pmid=23382690;citation_doi=10.1371/journal.pgen.1003219" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Caruana;citation_author=P. G. Farlie;citation_author=A. H. Hart;citation_author=S. Bagheri-Fam;citation_author=M. J. Wallace;citation_title=Genome-wide ENU mutagenesis in combination with high density SNP analysis and exome sequencing provides rapid identification of novel mouse models of developmental disease.;citation_pages=e55429;citation_volume=8;citation_year=2013;citation_pmid=23469164;citation_doi=10.1371/journal.pone.0055429" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=S. S. Chaudhry;citation_author=J. Gazzard;citation_author=C. Baldock;citation_author=J. Dixon;citation_author=M. J. Rock;citation_title=Mutation of the gene encoding fibrillin-2 results in syndactyly in mice;citation_pages=835-843;citation_volume=10;citation_year=2001;citation_issue=8;citation_pmid=11285249;citation_doi=10.1093/hmg/10.8.835" />
<meta name="citation_reference" content="citation_journal_title=Neural Dev.;citation_author=N. D. Dwyer;citation_author=D. K. Manning;citation_author=J. L. Moran;citation_author=R. Mudbhary;citation_author=M. S. Fleming;citation_title=A forward genetic screen with a thalamocortical axon reporter mouse yields novel neurodevelopment mutants and a distinct emx2 mutant phenotype.;citation_pages=3;citation_volume=6;citation_year=2011;citation_pmid=21214893;citation_doi=10.1186/1749-8104-6-3" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=N. Ehmke;citation_author=A. Caliebe;citation_author=R. Koenig;citation_author=S. G. Kant;citation_author=Z. Stark;citation_title=Homozygous and compound-heterozygous mutations in TGDS cause Catel-Manzke syndrome.;citation_pages=763-770;citation_volume=95;citation_year=2014;citation_pmid=25480037;citation_doi=10.1016/j.ajhg.2014.11.004" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Gallego-Llamas;citation_author=A. E. Timms;citation_author=K. A. Geister;citation_author=A. Lindsay;citation_author=D. R. Beier;citation_title=Variant mapping and mutation discovery in inbred mice using next-generation sequencing.;citation_pages=913;citation_volume=16;citation_year=2015;citation_pmid=26552429;citation_doi=10.1186/s12864-015-2173-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Gallego-Llamas;citation_author=A. E. Timms;citation_author=R. Pitstick;citation_author=J. Peters;citation_author=G. A. Carlson;citation_title=Improvement of ENU mutagenesis efficiency using serial injection and mismatch repair deficiency mice.;citation_pages=e0159377;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Investigative Ophthalmology &amp; Visual Science;citation_journal_abbrev=IOVS;citation_author=A. W. Hart;citation_author=L. McKie;citation_author=J. E. Morgan;citation_author=P. Gautier;citation_author=K. West;citation_title=Genotype-Phenotype Correlation of Mouse Pde6b Mutations;citation_pages=3443-3450;citation_volume=46;citation_year=2005;citation_issue=9;citation_pmid=16123450;citation_doi=10.1167/iovs.05-0254" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=J. Heikkinen;citation_author=M. Risteli;citation_author=C. Wang;citation_author=J. Latvala;citation_author=M. Rossi;citation_title=Lysyl hydroxylase 3 is a multifunctional protein possessing collagen glucosyltransferase activity.;citation_volume=275;citation_year=2000;citation_pmid=10934207;citation_doi=10.1074/jbc.M006203200" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=K. R. Johnson;citation_author=S. A. Cook;citation_author=Q. Y. Zheng;citation_title=The original shaker-with-syndactylism mutation (sy) is a contiguous gene deletion syndrome.;citation_pages=889-892;citation_volume=9;citation_year=1998;citation_issue=11;citation_pmid=9799839;citation_doi=10.1007/s003359900889" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. Lewis;citation_author=E. Quint;citation_author=A. M. Glazier;citation_author=H. Fuchs;citation_author=M. H. De Angelis;citation_title=An ENU-induced mutation of miR-96 associated with progressive hearing loss in mice.;citation_pages=614-618;citation_volume=41;citation_year=2009;citation_issue=5;citation_pmid=19363478;citation_doi=10.1038/ng.369" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Miller;citation_author=M. Neilan;citation_author=R. Chia;citation_author=N. Gheryani;citation_author=N. Holt;citation_title=ENU mutagenesis reveals a novel phenotype of reduced limb strength in mice lacking fibrillin 2.;citation_pages=e9137-e9137;citation_volume=5;citation_year=2010;citation_issue=2;citation_pmid=20161761;citation_doi=10.1371/journal.pone.0009137" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. L. Moran;citation_author=A. D. Bolton;citation_author=P. V. Tran;citation_author=A. Brown;citation_author=N. D. Dwyer;citation_title=Utilization of a whole genome SNP panel for efficient genetic mapping in the mouse.;citation_volume=16;citation_year=2006;citation_pmid=16461637;citation_doi=10.1101/gr.4563306" />
<meta name="citation_reference" content="citation_title=Cold Spring Harb. Protoc.;citation_author=D. Ovchinnikov;citation_title=Alcian blue/alizarin red staining of cartilage and bone in mouse.;citation_pages=pdb.prot5170;citation_volume=3;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Immunity;citation_journal_abbrev=Immunity;citation_author=S. Y. Park;citation_author=K. Saijo;citation_author=T. Takahashi;citation_author=M. Osawa;citation_author=H. Arase;citation_title=Developmental defects of lymphoid cells in Jak3 kinase-deficient mice.;citation_pages=771-782;citation_volume=3;citation_year=1995;citation_issue=6;citation_pmid=8777722;citation_doi=10.1016/1074-7613(95)90066-7" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=M. M. Quwailid;citation_author=A. Hugill;citation_author=N. Dear;citation_author=L. Vizor;citation_author=S. Wells;citation_title=A gene-driven ENU-based approach to generating an allelic series in any gene.;citation_pages=585-591;citation_volume=15;citation_year=2004;citation_issue=8;citation_pmid=15457338;citation_doi=10.1007/s00335-004-2379-z" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Rautavuoma;citation_author=K. Takaluoma;citation_author=R. Sormunen;citation_author=J. Myllyharju;citation_author=K. I. Kivirikko;citation_title=Premature aggregation of type IV collagen and early lethality in lysyl hydroxylase 3 null mice.;citation_volume=101;citation_year=2004;citation_pmid=15377789;citation_doi=10.1073/pnas.0404966101" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=H. Ruotsalainen;citation_author=L. Sipila;citation_author=M. Vapola;citation_author=R. Sormunen;citation_author=A. M. Salo;citation_title=Glycosylation catalyzed by lysyl hydroxylase 3 is essential for basement membranes;citation_pages=625-635;citation_volume=119;citation_year=2006;citation_issue=4;citation_pmid=16467571;citation_doi=10.1242/jcs.02780" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=T. Sagai;citation_author=H. Masuya;citation_author=M. Tamura;citation_author=K. Shimizu;citation_author=Y. Yada;citation_title=Phylogenetic conservation of a limb-specific, cis-acting regulator of Sonic hedgehog ( Shh).;citation_pages=23-34;citation_volume=15;citation_year=2004;citation_issue=1;citation_pmid=14727139;citation_doi=10.1007/s00335-033-2317-5" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=A. M. Salo;citation_author=H. Cox;citation_author=P. Farndon;citation_author=C. Moss;citation_author=H. Grindulis;citation_title=A connective tissue disorder caused by mutations of the lysyl hydroxylase 3 gene.;citation_pages=495-503;citation_volume=83;citation_year=2008;citation_issue=4;citation_pmid=18834968;citation_doi=10.1016/j.ajhg.2008.09.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=B. Schegg;citation_author=A. J. Hulsmeier;citation_author=C. Rutschmann;citation_author=C. Maag;citation_author=T. Hennet;citation_title=Core glycosylation of collagen is initiated by two beta(1-O)galactosyltransferases.;citation_volume=29;citation_year=2009;citation_pmid=19075007;citation_doi=10.1128/MCB.02085-07" />
<meta name="citation_reference" content="citation_journal_title=Invest. Ophthalmol. Vis. Sci.;citation_author=Y. Shi;citation_author=Y. Tu;citation_author=R. P. Mecham;citation_author=S. Bassnett;citation_title=Ocular phenotype of Fbn2-null mice.;citation_volume=54;citation_year=2013;citation_pmid=24130178;citation_doi=10.1167/iovs.13-12687" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=M. M. Simon;citation_author=E. M. Moresco;citation_author=K. R. Bull;citation_author=S. Kumar;citation_author=A. M. Mallon;citation_title=Current strategies for mutation detection in phenotype-driven screens utilising next generation sequencing.;citation_pages=486-500;citation_volume=26;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Mouse Biol.;citation_author=R. Stottmann;citation_author=D. Beier;citation_title=ENU mutagenesis in the mouse.;citation_pages=25-35;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Matrix biology :  journal of the International Society for Matrix Biology;citation_journal_abbrev=Matrix Biol;citation_author=C. Wang;citation_author=H. Luosujarvi;citation_author=J. Heikkinen;citation_author=M. Risteli;citation_author=L. Uitto;citation_title=The third activity for lysyl hydroxylase 3: galactosylation of hydroxylysyl residues in collagens in vitro.;citation_pages=559-566;citation_volume=21;citation_year=2002;citation_issue=7;citation_pmid=12475640;citation_doi=10.1016/S0945-053X(02)00071-9" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=C. Wang;citation_author=M. Risteli;citation_author=J. Heikkinen;citation_author=A. K. Hussa;citation_author=L. Uitto;citation_title=Identification of amino acids important for the catalytic activity of the collagen glucosyltransferase associated with the multifunctional lysyl hydroxylase 3 (LH3).;citation_volume=277;citation_year=2002;citation_pmid=11896059;citation_doi=10.1074/jbc.M201389200" />
<meta name="citation_reference" content="citation_journal_title=Essays In Biochemistry;citation_journal_abbrev=Essays Biochem.;citation_author=M. Yamauchi;citation_author=M. Sricholpech;citation_title=Lysine post-translational modifications of collagen;citation_pages=113-133;citation_volume=52;citation_year=2012;citation_issue=0;citation_pmid=22708567;citation_doi=10.1042/bse0520113" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Positional cloning of ENU-induced mutations has traditionally relied on analysis of polymorphic variation between two strains. In contrast, the application of whole-genome sequencing (WGS) has enabled gene discovery in mutant lines maintained on an inbred genetic background. This approach utilizes genetic variation derived from ENU-induced variants for mapping and reduces the likelihood of phenotypic variation, making it an ideal method for genetic modifier screening. Here, we describe the results of such a screen, wherein we determined the minimal number of mutant genomic DNA samples to include in our analyses and improved the sensitivity of our screen by individually barcoding each genomic DNA library. We present several unique cases to illustrate this approach’s efficacy, including the discovery of two distinct mutations that generate essentially identical mutant phenotypes, the ascertainment of a non-ENU-induced candidate variant through homozygosity mapping, and an approach for the identification of putative dominant genetic modifiers." />
<meta name="og-title" property="og:title" content="Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/401" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Positional cloning of ENU-induced mutations has traditionally relied on analysis of polymorphic variation between two strains. In contrast, the application of whole-genome sequencing (WGS) has enabled gene discovery in mutant lines maintained on an inbred genetic background. This approach utilizes genetic variation derived from ENU-induced variants for mapping and reduces the likelihood of phenotypic variation, making it an ideal method for genetic modifier screening. Here, we describe the results of such a screen, wherein we determined the minimal number of mutant genomic DNA samples to include in our analyses and improved the sensitivity of our screen by individually barcoding each genomic DNA library. We present several unique cases to illustrate this approach’s efficacy, including the discovery of two distinct mutations that generate essentially identical mutant phenotypes, the ascertainment of a non-ENU-induced candidate variant through homozygosity mapping, and an approach for the identification of putative dominant genetic modifiers." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Tat_DTkRoRipPOJuxQeB64ykVODK5bsAPTRtD5NzW0o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488296","apath":"\/ggg\/8\/2\/401.atom","pisa":"ggg;8\/2\/401","ac":{"\/ggg\/8\/2\/401.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/401.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/401"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/401"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/401":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Jx72NDMkyH2\\\\\\\/KRreLMYyPWPDEcY7vJOkKAivzRySoxhKsajOrNBgP7q4GUeRh9PeLkIFQNkxFMHVw14i1ZvlUI7U3foy\\\\\\\/qhNqDod74JDZwOKGRUIVft807PYkq4DK8DnqYTmonsJs5C9rtcmYiJ1W0dIRwXhO6OkFwnO1ZatSwD+cT2+sO9L4E\\\\\\\/+j9UsHQ25C9KN0XiR7B0sjz0YbvTTo7O8mXXkfB87X7nn\\\\\\\/gAJtRMFK0o8E3pkGNABkNC7aD5i7YIA7hM4nU7bWavm\\\\\\\/94vckTX5VUmG69cavFtPqgmshtx9ulMsb8cGWdwrE113ZyHW\\\\\\\/VI3IJNq6Ey3dcWp87Eokwyph5LCXT4L\\\\\\\/LevPWCOMIBvAgvfG0CUdMPt6hAOUTTbEsmFpsr1PXzJd\\\\\\\/4GzlDuKnQObt33WM9p7LmwKqGUCWgtH0s7c9\\\\\\\/7MT6Oruag11zputTMNE\\\\\\\/pJc6kfh6C3z507noE5zqm2HpkN+M+N4RbXAzaaKxvao5DG+XO08D92i4Tk5cUZn\\\\\\\/Hxjh1AJrW6HZVzFG9ZMqY6rGZHwLU4O8zkkaYqHQ93C0c1BE48U+x7gKY88iieOT2UjL5zH0PWnZwO1aTxyi\\\\\\\/LnZLnntxxOJVI42zA690s2G+XywjScUVS3jXNPSvKOu\\\\\\\/zHrHWi4waIUIOrOIBeK7GNWU2xm4eoeVRHcI9tfbpf4x2c0ZB7otvY1TloF0dFsfNsthHTvlPaMuQovHyqWGAqdTTcFefao7tLq1f1cnKIQb2R5Ll6uxPUJr6Nu6rKw8ZgqI0Fhv9F2SfZlfks1J+51iHbto6rQqlv9EwtNkBCmt20hqIh9dqnbqivuu6WAs\\\\\\\/BM7Nc+Ob31e2AGZe8S3lZoIu8+3UTtBn7rpk0k2Y4hujG8+jdZjDaPspBChiWQWviCOgAMzWXWk5Q6bEsa8SKRpIwmKVUu0EkDzOuKvxTY42TDqDUIANQqNV23PgeDk955utB5I6G4vLxMXsq2QUr3Ezr0Myk6UvbezjnqW7BYc1HyXY4AyB0nu8dUgUt7Wv77PentqH3BgWtMxQ151W2ZFnOkYHncJ\\\\\\\/JBWwJ5DCc7B39+b3HcS1NLUoY9v\\\\\\\/ETYNSnkaUowRJlUmn4QbENuSGs7NtfF6ukLlECqVbm7N0NAzifBNExqJjnIQ\\\\\\\/TgHGd3pyugqiVOTMtgBnkynu9dwwXQvgrc6hNOODKuug8GRDCwgwcGejcftHDw3OXURakopyovgWmmzWcclb9\\\\\\\/QSabCI9UcJ+stlPFeHYTYpklygLVtv2YmnlMxjl\\\u0022,\\\u0022iv\\\u0022:\\\u0022XOBnLeZ7dVu86gf4tq0y4Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223d3b40accfdd2eebba97fc7d857e85bea32ff67a01a3c0538e2a52bbcbc5ea18\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zlV1v4wuNiL2WhAMSIT2n35kUDPaNyRYxv8PWAXDMY0QgtBE2hKnVXmo9o8V6dd\\\\\\\/i27iaqWZl1sh417lu5Ehlu+3vov460ttBy0gjpM7GCopmYJr\\\\\\\/alp2L3Gbl0LJLasMTHFBxDrxesqfuOzkWRjvTxqXTvV1yU4g90\\\\\\\/UnRoboOhEDml5LiC+m\\\\\\\/eNWbEZXYPppXw7dlTaaLVdfqUCz1351dhXZYBFm2bd0EiDFA41afRtFEZUiMQ2\\\\\\\/rzPRLgOqIhO2OTtXuI6riJWkAeQ1QdFl\\\\\\\/cKtDZKrIth4F8y5IoBBhNGyQME0nkYJhBeOYjooPNSoKHyQd4LvSRDNjNRq2YvLp32Nym3V5W7ZMmyXA8dnajAO2GuW73Ia34Gs8BzbUweAB59CWBdpr3SCWcQGuYDSbwOTLJLDPRIHpr91Lc3Ngx2vmENMCdfCLKpgBIGpryhP6IXZtJ\\\\\\\/oM8A7cfpaO1UqF+Lxp5r+KkOXiS44vqQULaAMMyOAwdRRMGlPFE2pd6tzAJ62OdxTRD7Q+sj\\\\\\\/xV4kC9nV7UDIWxz4QQDm\\\\\\\/efrNeTo19ZZC5n80q70DYIBDNrNN3tG\\\\\\\/pcW3rNchZo46ACLrcW+uIY8uasvo38BnldZeV4p3idtYmCYbV04rrRPiPcUxWostZf8eYmKDPcGMCXOOxZTL1GvL2P6EaKNlh3wiFqBOVGmcv7tpxA9V4gkiqX7mQRSV3Psl2Hiaj7YR56i0XuMRI4\\\\\\\/8bYTmIGFEdsFJux4ZBVc02S2IosdXe7XCqR5y++uuwWoS8\\\\\\\/PmlpZcJvFVpgJdwGzZB+ZKeX7zesTbCx6twvS8PRaMwi\\\\\\\/LnuHYjwCq2mMoyeFSlc0GXAroqTHy40j5U7pZAoSguLzV+3iJPyBzfOsvlQ2hCLfkOIq1ejQ\\\\\\\/G8\\\\\\\/yLCvsIEG795f9TUwI\\\\\\\/M0NmAUZpn5spr9Sb17rHSEYFVLI4+BL+SwizymZ\\\\\\\/jJ+T7JUz9od1d9CrPlmwTvRyJDEfoM1wFwUTVBCOXTkO9nKi0GaQ1qzPt\\\\\\\/nBAqs4D5LiHkx5iepfsOadE\\\\\\\/h8Ys5\\\\\\\/2n5BAdsOAmNPbP9sTKIAx4zNj++NImjBWa1epmGf+QO4e3UGbO1mk1oKvLHEFxpmq+ppZT7aivTQgdOuOkeEo7R7dxRrNmzC1PpgG+4hH5p\\\\\\\/UoMgG8oh1HwyVLz6gBpSqnudM0JKwhjQ3JFXEuUtRhZp5EhT9VAx0lnPNeFvvHaGSjnWyP\\\\\\\/C6mUKJu+C3q0qF1ppuL1w\\\\\\\/huBusGF9ivJUJYyw5IXpQAfDGQL\\\\\\\/3G+SOew4Ikqx0AD9ff+CWBcHkldi+qcehEzwHvRDPUERpbR2zgLw08Q+xmBv5rg4\\\\\\\/R6\\\\\\\/cSzklvAxQTWBCIAFCFTQFV1VHdF9ERnby0uG4y7x5eZYrweNybAQ\\\\\\\/h8Y\\\\\\\/4FYRMoFVjDrjEdT5UPk4A5ec+R2EwEwfKyvujsFF0+wltdSah0XX5N7xRgef\\\\\\\/e\\\\\\\/PQQC4bgFlVq+Cp8SZvFUwYDwuIIhnbZpWmppbTI7AStrwEf1Nzh2Tj61W83zq5FlaOEhFZQbh1C0nd6Puo5F9eIdmY1Lh7QEahC5Hu8wIXASE5iVUJ8VrnoAeR49kJECnVC8584TPCLnQJ8K57+zJBs8VW6nWEk22s2n7wdLbsXO6pQYmr50GjSM1VwRCBV\\\\\\\/tRWLd8K78zgD4hx1jznDGnDgpFAf3VBZdM925rjmY0QKxb1+QpqxPM5KkjyneESyI07+sUq4V8kZC4K\\\\\\\/ijQSUIkr9dyoc+\\\\\\\/3vVe+H5rRTmNMzEF9zQROqk+r6peFoM6kt4fPGRckPPj3BJOPUAh6jq1Q5TZkIyPznULR2nrmBSIn9sYdkzQvBlZZhVQMy+jSIvlKr6ZiUmNkkD\\\\\\\/PGOgZHeTqpYSVnvvbTrbbYKUrFoTSc1F3e6lNJWtL\\\\\\\/55LNob7YurjfkBT\\\\\\\/izlIYVsQyPG7cDpGiOBzIKCceI6BAwwuG4I\\\\\\\/iUAZeMltdJiTiOxgj578GPVe4jQhmPR71kq1eyDfXHZQiVNn7mVNYseBmB9OiIFlAj5xlwv3WqTUIvegYkU0a17CNsvF+V6A0vn6IIT4bAVNvTmI8uZZcDVyeNAzVQqdUl77DD\\\\\\\/F0MA9bPdTS+iFLO8rwbSULlhfu15yhV9UtP0XqGeH52J+GKr+G+4+nmqFBHqSntuYFjWGlja2A\\\\\\\/JyFGphbqyH1PVNraC7yIWvIsaAy27HPPpkcW9kuP+Hom\\\\\\\/bUd1f1eo3koeny7wjlbMeDRcUoqxCMpsfZXQOougx47Wsx+NiGYWnbxwbq7IGw48mw7EK\\\\\\\/I\\\\\\\/W7ZVT244fPf1h5sC4iC\\\\\\\/v0Rf\\\\\\\/VHY\\\\\\\/\\\\\\\/470pwYDhWP3CzkNVj72za2B50yaxKWzzabXAXT5tuxxJgJMsupL7MENXbHrCBWU8FEGxQvMWeYWPJg2V5hlIs006rpvYTtpH9kKAwEoJwQBQnWRUZ1SZJ7kKjA\\\\\\\/rukG92zZDDi59dFGOA5xqDWyp1kkDb5u9TjQSg7Ge1uGMKHi7n2pYSnjhoxy+Adm09WKnvcSD3abGqDbJKtJAlHBFOsa\\\\\\\/ijEf2bpL1LIQdHUxv06GLBsUXx1Gt3lPlRlF4Dm2fAX7nwWklUGipbLSWKGnulS\\\\\\\/sT1ZdKh7QBWpGl799sl3JHsVZVw+dPvKz74TveRb8KhMGhN01LU0x7UhbDIi2qZNEC7h5sFuosDSQAYBXxPZ8oZ6FrFk2fJ4gSOozmOgiBdUR7eGlyVF99saLxI2gdQdNWK+MYPMc5N7Bp2+sPMyN3Q9p7QU3qA6D0DWVLibq7Fvdo9wHevmPp3GPoKXz3sdE83VpELbJdjzLxgJ1etgtv2xmIoRXuw6RK\\\\\\\/Tb7PSxlh\\\\\\\/5827MczGge57srZnDTH9d5auaJs+Isfp0JapwvbVzHQ0Kx3z9YHE\\\\\\\/IThPpkt3NkRUmA\\\\\\\/GI+R5kRDJL0hN\\\\\\\/dXk49cG\\\\\\\/AHfsB\\\\\\\/SYr\\\\\\\/NTltu2agljxhm8WT5Hf8xV5uEGQAofjXMRDq+1GAan\\\\\\\/oJ7s8kv7gG2O9fbhpl6CBpSOcr7mkpgGvihqNvJxsVi9HVvZKiA5ubVdqKq1i7msuKUWnzxdcXl9pE7uELYUep3oxN+EYxgvvDg3LNyw8HPm56npGmCQPbo4CFOWXx23Gm0jzTNzy+rmOYVusByE4troLz2dADXXM6cH3HubeO8h4fexTpExtO\\\\\\\/e4MkPwN3ur7AncH7o18vzZTMeWONGVRIjRwG7wGR6RKCYKjjBVIQJ3b13sfI5sySmzJslSPUDbbmeeh5aE+9qUC6wd\\\\\\\/EFCleLOfHEqEjQ9q7q4bYpdpHyJiWB8Fum1ioVB7aVLUDTv2KShiuW5HD9\\\\\\\/14k6ZGiqlfcOPSgjU4GF\\\\\\\/D3l1DI1RD5Ev5WVn3TY+jsvb8wdIQl1fubwhYHCG4kIOqucod8iSomXtkPmJWY9z9xzSP9KTUmxHD5NbLUszX4u7\\\\\\\/EKGCSl4QsnyG1BRhxtICvUXtd0fW0lxh2nzZSCUahw+hbFQ\\\\\\\/3tCvMcQ2AjW\\\\\\\/IiWqUtcpjztuqZNvhKNnL54oKpJVdD7UyD9TonwEURXC8ZXVE35toVUpp5nJCGGSWHqdfBSTvz9YYYOpGYClKprE\\\\\\\/n7V43aDioKuwgyhnKKlzEcqCyFWbaggJvCketzxRStBzQXZYY+z+Bsjwv\\\\\\\/6z65n3VLJDkisni76lAIAo\\\\\\\/smWWpGVTo\\\\\\\/kltA1zl8PzePApaURYrEjRmX8nGhGNBMs0UM9Ok8yEvYAMBo\\\\\\\/YvcKIjdenZ1c6EQuPDvP8sdy2aAdL0b7+eE8CfF6+3kJyiDjGCqu2EYcMhRhtMz1aMe3mGEcJ+q+Vq5XOg1OnJbyk++mrZD9DyjL+NAhwNIhrpfCCGG7R886NYRuksxqZ8a2rRoIzGRjLKuI+9\\\\\\\/EEQ82vEjeCrxLtBzPOufIitJnlXZcONqJ+38DBinuXsfido4RUzSnPp75J2lWsEzDlw4o9MJh7NqRZPHe\\\\\\\/ASMhryWVlL6CxCdEAbP\\\\\\\/zie37yJBowvkb4XYdOlYTXp51CgJ446hP8MJIwqLL9hr2\\\\\\\/Y7gOt0JJq8M795i7ryU5BtBL6drw8rKX4dJK8B9HhSO9zFjyW4+jkDKi7FrSzCsiUeNGiwoN5wJp7+mWvWtwTPfj6Tcz\\\\\\\/6dpaltYV1TB32gadetvY6vkEhBWVP1fEThpqYHSK\\\\\\\/ClA17z\\\\\\\/Ap6idiB7XFOBK6mQ3zDJeIVeORqpsmgJXtsVfqnndsAOKsG7IkAcmYx8cLPjP4V4ybuWxdrBoD2jte8Teg4gdtCyouNWTq3UD3oRkvtBVnxzNMIsZOzoL09qyw\\\\\\\/oJXFYLdPuFNd\\\\\\\/Am8Two0Ye66CF1HPr0yDvzBwtOKaMA4Wkz41ow6O0nZX4n3vy9oC6BDC+D0ZVjc2oz6LfPBwNRG6N4Yco8lI+O54Uv4Vtk0+pv09E2kwwQ+brxzoV6xJS\\\\\\\/LpQuSNX6+HPKoTzu+E78toXy98Eg5ZbjQU5TJ1HtX26eSHuW+Tnl6oYsIohe\\\\\\\/lvZPxnT9dnAw4Ght9HP0gKc4AX9uwMHmGjBVSoKzrPkj0cGwWNaO8NxbznuyzRHpkBPgzAunq+Jjz6iY1cE+W1ZAzLR4gCFssUFsCCT6qb04OMGmAXnZxfnxQGKYwkVyCHUdvsCwIryOoDPWNZy\\\\\\\/BvOCDC+BLp6E+lX7U40pFDCLdZwW1SlGvvXOn4LX7VMiVUbnu7PPb3cCpGl7VovZ+xjhZkFzFU\\\\\\\/6MUfCs4qUECer3pKTKyh0uSChz8qDyMLwskYFZ+WfuIwzPELkA7djkaZ0S158KqNZ27XWshIvIwC5i3IsPHDf19ZQl1E6Ef7wdp1EN+Rw9Q684rYf+tAtjHgGdiNhLKHYCNuFoIQvKkJ\\\\\\\/0LMznFvVGshhew91JpwaI7DlUt\\\\\\\/peG0fnnOD1yC4q8Ys29KD9T5Ysntj+Hrdj6BG5\\\\\\\/LHxrEEfn1VzpM2eB97BoDMZQHAspqQbEzyeL9yoP9YsFxHmFSQLnHnWvH66nUnQEZaawymFukMe\\\\\\\/lBl1qXRN9jIdeH99dDXS+\\\\\\\/4x+BsM8NTIbgbYt5FQRpmQ8M8KynC9KfWATC2UBK81H5OXZExW3iMVuyr3IkbtQqztQbz6k+919C4qaBiFhAcNKfRN+Xq5MNA8xu3YFHxUCmvv8Yh0A\\\\\\\/oKrvJ2hhKw0GMqazKm7H5BAEZDY0Dhs36y8WV1kk3BHLTchspza3QUKcjMC0l\\\\\\\/Cz0NvBNhrUkBKZbDqcxOXMH8F3feX+kwhxYEOBNe0kH4ncrQygdEniKVeNdHidBsVYjNWZTM5GLm1M4ehPiV8Lar04PHpjf0hG7uBafzsT6zQgdlwNXmYtD6jyApvQv\\\\\\\/PIYa+9ALwiFOLBsszZGGnMNlg6CXol1Sw7X3CmWGJ4FZ2dL6u5qgXRFJMJAwd7GGfilSM68EJfzAkl+HP01WwLYS2cKgmGJ38b0AfL7NDxd64RcM12jXpkFGAwbaPB2PsZ6b2IEXjygGte5TdT7OFsN5PzBAUzQ9TOe6leg0ulp4LvEMZCz+0c1mv0Iy+Oce0wMAyRkS+nRD4hel1cAO9QWqTnDN\\\\\\\/UHdemrKHlVLpDb9luj0BE4TJ52f7LeXAEa0HqgWrJv4A3G5bIQUx51Xlt3ana7A\\\\\\\/aL1AUCR3Lvd5n374UpV6ay1oPhgAG7H6Xhd9vbNRcANDL3GVsAwMiGAfp9x18CMc+1jYI5hAZTG2qFkO9zhL5iUhIdCf3pytDPic7fc\\\\\\\/8mAyemcRz0UINOqFJnPGMzaDkKZSklg\\\\\\\/tJN\\\\\\\/7KtArRjlKJ4B3EtWfD8Wf\\\\\\\/whfdqaEQpWIs4uMFNefyYBq74K0wg2YefEcaSvnDRo1jDDMX5n\\\\\\\/e\\\\\\\/WgCJ2zGZc2w9B\\\\\\\/5ARuG0HkEMz865mFa5bSeEnYB4GxjyDnCOEsjuY12NVQNDux+xraNB6+SaymzXr\\\\\\\/qxzHnSVNll4jJ1FlzJKS124qE5v5MK9n\\\\\\\/0Y110kjOcWDKDFhHN6RIjLT0iIIaVbNGa67rI0mVaIsnaJBaT5esRrlqzOeoBOOd6+3UPx4Cv7OcoTOJ280pDtWlcjE9xsEZF\\\\\\\/QPAD2QJ+NUFSbUvtSNio9tCaJn3NrPu7b64E1CJkzSHrjQeEnNVcZQW32anvF6vYzpZdqKEqO47MEXE9x934bdUSaUoN1Bk2YS0v3lIfUVKT9O4IG8ZKnla3zIeYnW64gLCyQG3WUNk+NuxtDvZGSt2H3G6Zj4i1nzrpxJjmGPtPZCjQzNG\\\\\\\/DJlgTNbvIgPPyFIIS6qxQGqcBcvyQzIChcpdfWjo8r2jnqw\\\\\\\/ryf7qb8brU5FxNMbMz0JSvcbW30mS\\\\\\\/8CY9LqLGAz\\\\\\\/M9TzT2NQrB+xCBY6EmBMKtXApCfIhicUZu9k8YxZf8iXg2bZdofV9TzWT0OhtPqcHh3rPspJ1IlOBCKZj1wqxWP2juuN1QYkwYwppTKUj0IV\\\\\\\/66qA1T6Jrberi2DBcVp5NfGQ2aaRZl+gsS1AGZyxL3GbVxe9VOXD+5+x8uXXsUwhAAkpr28ALhaVaZSBCh0zksvNn5+fh+1ydhEQsI+gW5LdoBIcwC33K8DXN5u2esdjMG2f3VrPRIpAjZWw30hUpMhQmLdTcDwZ2UPXjSktSXmWRun85Oo1dKNbok1Ha5qRNRh+dxApgr3CpDlt05cUmzcfRtwDQ8K61Hkhr78J0IOzJcMGuGklkGNfD5o5cHRpwKavOjKluDRqQWG9eVs2L3KE4uLvHIULZwEeofXgUWg5ElHvwFUnR+NWG8Nr7XMpwxEopsylrI+Jxx1wcqkZjGxiXz4S0avKMMeY0WaDz2E+1bNW7KQbjUEeaelhE+uvr3DGa0nKFrEJi94TVwlidIJ3MEXUHdGoIQOr1sW1DWgBAOZbQwg\\\\\\\/0NI9Ynr+dvtm9vY3hez\\\\\\\/pMYB\\\\\\\/lQqM1nGJrBPvuayu2x\\\\\\\/PIy2azO7riqSOnYRRjGNccildCSD1PJmeVeo\\\\\\\/Auh2FjwlJbrRbb3EcKLc0G\\\\\\\/TuWezImdyvdK\\\\\\\/ZPUa\\\\\\\/\\\\\\\/ZGcKVV0m3sQAgW9gMwxaofaFOqmATxd1whdDlPplCt\\\\\\\/0klwwa4wdK+eK+dv+66iQHTpjzXrIdsP7A2wfOCI6gFVo1gBTdeg8iTCn5AEWn1teL3Vmh6GyAqHy+RPXy+ilG3WvWVFoV46+2QrqPN+9Vs6PEOj78kACKynIhNxQG9IKE4YXv5+rWyBUIqlrGhDCPM1UmeF\\\\\\\/fRgibrM6BUNUD7TV83ti4QrfEJxjMhHL12cyIvNwt0Ci6K3olzJayZvD3WzmrsAwleNSfk+Au\\\\\\\/dV1SeQ3XGTntxwp9ovToqBGu2lk9hAmB+VwTU53rhYNPZOTfqNLjnT+2Z4tC9S6poN\\\\\\\/U8G82C0rrIBOg5LSHAg+m1U3EPxA38EpVnlvFRV9wnLCjyIk6hTEOKGmO+RyIzZCVFZgo4FHjBGNIn+fa\\\\\\\/cJuTDFYF\\\\\\\/p6rlt7wWcVSkY7fQj0zyJaKcs1Y1fXG8UY5C98QA8L9v28a\\\\\\\/mZ07xLHF9vK9zZpPUd5pieZ3hyMbrknqf8LRnfHJBiujvIKyZcrUcnFaDVuvYsoj6JUp0gjLL5lY3nY2P6j8NOYAnickFIF0df5S9M3dyQAAsJIADpaZ6NixS\\\\\\\/tkutBsg9iV+NwSRzvgP4ZheYP5lA2osoE\\\\\\\/K1MtkrVkr5cBOKpFZ5577QOsXT0a4jxBzZc3JkDexpZpKCZNQ7ScJNBbPi9FTQOKNzRE5le24Lzr\\\\\\\/fKhl6i7imfhKYxaNQquEvrD+NnG++EZ0K0eAzUwv8d7S3iwmO2vN5SjkmW+fpRy3hFUEpRBsVaVqEsj9k9FOR6yMBkKAGyqbbsSjnFRlzm7NU4wliF6eMrL09gkahtNBk4TYT7vbWB0yoaZRGWJu5CPk+7YEik7eGpbPBzpKn5nT9TFXIBYCSkLibcMSFk9XH\\\\\\\/NCVeE8ZL5WYIEhucSrzG9EYa8Ag8NepvAMH98Rrx3ENrhWKjLlbeUkcBGGc+IUo73Fg2SceNQNXbc8HR1RsyeSqhH6KOey2v6Yu+wgrtDPeozKqNpx7Im+OqCJ4eAwpHGgjnWFuVbgeQFBou3+vL285w6Rhxu+wHAXqVRMzFBS2Yq93rw2RABDeEtIQaEfHbWwHfA4GUrkm+OCFSw0AtU87vOif19FINxAm45TexHiT\\\\\\\/QQrxxx7eYPQ+eq8\\\\\\\/SlILeFUExDUsTawQWCmnbq99yWSybqo5+lepPVmuiuCHPkMT4fqAQ4ETU8p5Elv0gyTfNBmQFEQww3Dm9vsqrwRhgm2a8w+eOGBSZGdaYNbKEFXJFf9Op18RwJje1sDrOr0hG1AlyKcRfEpFhxPjyWUvg+u0dXBt4Jhokm7WAv+Y\\\\\\\/guyx+LPTLZQ4\\\\\\\/3ZcN3hqDtv4W3vhbQRe0RjstlKRYzmppsaDETj46zsYxbPPULBIZHvRdTaNQZdC5gRDcHLTAOqJaasTRWQKEBhuBgqCXP6DNM7mENfCgjI++bl0DQ+WPDleke+KvMlH5VpGl\\\\\\\/nodX2xLu1sNsbS49DZurH+5OHFAPCPa+DVO8MCKRllfPLnOCVodwAL4SsY2PQnYoV3zGK\\\\\\\/Fp6msTH\\\\\\\/Gsj9cTLuyu7UjxshqMEDcuyTNqYEwfpE1jNGIYuXH4qPZdDjb3rl5g+4eEWhEjAsDSWMIvMorcPhrN\\\\\\\/yWz\\\\\\\/Y9k4m+PtzTX9YyVZ4savU1\\\\\\\/r\\\\\\\/qRyz0sm4X+52dM495cxWv5YFZ1FpdSFNTn6v+s6b0VVlVDWfy8K\\\\\\\/OKmyHR3CAmVK5Zz0m92lMeI84gWHqh9gYC\\\\\\\/o90tXqfqVrG0gA9rs7LbYkQpGZz\\\\\\\/\\\\\\\/O6bFyItxShDLJEu\\\\\\\/5p53WYJgZ2iOWMkWuZEl006PPz2S3N7+FxxCIl2GUJlgrT6Dr\\\\\\\/lKSGanhDfb4FB9lgTvw453pEurXNH1\\\\\\\/y4\\\\\\\/T6Lle67LWNukfnjXb0\\\\\\\/Ng8c0TNqEEAZZSs8RRu+Ti+CxJ\\\\\\\/q9vAS\\\\\\\/miiWrYmw2bcyOHYzLSZ7+oWP27fFpOn2hYB0ET\\\\\\\/jTVVkNb9Z+UG29EEqVWC3SQ7TjDkfCMLLAjKn0TJQlfid7jcwtGtI0gcDncIV8EPLeZ3FuGVAY3QSfGzpcY3DmExxC7eVHcTzIefmeTnqg7zGMr+PsW+DZldc8DLAnsp6I0ktT8hrRdRWPFwLvnvZK1b9RsZmPO8ZhVu75ZJWtZAPCz8m3O91FWItYaRre8M0FUHq1dF\\\\\\\/nQiXPdb6t8ulN8e7mODhYdRnclzyz8C\\\\\\\/fpwLzp752GaRbcPuZFezEezeM5R2+zSBheTCBMUuhlYmV454f\\\\\\\/9teJhqapzHeX8XHlPeoLB\\\\\\\/YsF\\\\\\\/\\\\\\\/2mePTJR40gS4Xtxxc1RXmZhXKpMbnu9psV1dxdp1TbYWHz9DVjVYduLFrCruYPrDRYTW06QVmGJpySUX\\\\\\\/Cbw4apwEmm79USzwcFhAqswcy1F2VcoVrhwn\\\\\\\/zZcNHf5YuBTIBvhGH6UOGHWpKaIflk\\\\\\\/dStcufUXbscNVFrSpVIzaSnVMOSF2ZMRuhm0ZxybINC5Of6gAv7snoZ+GiqELR7gsJ272Ua8FFaDtgiMfAyknPBjuUYWMHoqcJD7CT+HB24pSTeEZo\\\\\\\/OllxujfMBbr4iaaDKS\\\\\\\/1mWyAf3WiHcEHuk66z6JRFpQ6yHxclnNveucSJ7BYkgEZbgI9XDcoZ9TjgPN2\\\\\\\/InFbabssT+vPHBnxlizvsaX4WfFGC0Ztot6zdKpW2QEX\\\\\\\/aJ9vdKo4Z\\\\\\\/dm3ue7rj1Rx5fjaRwnY7QdXZ0l5\\\\\\\/TBirzrkkzFWKschSVDHIU2ESpy6xrV++DYqbLiYLihQcw7MYdwwVh81i5ixz+UCGE9T9zwaCTI7DRunsoEwVLgQSYVwpo+6yO6XiNI54TtgSTk\\\\\\\/qpOhJBQnRfl7gpZ52FvAQbBNYC4kYG+6gftN3isoW1IjuU2wKRdq250cIfOdjb4\\\\\\\/eaqKnIUMIE2062gkXqmxr3jkjL\\\\\\\/T5p+orSa\\\\\\\/WkorRVghF4N1kyfk1AdDSpmsp15rBFZpGQeAH8KtX\\\\\\\/jspb2MEbHciU9kG4OEJRKaUaidCTPOFDjCMpzKsxobATppY0bpr40IzaLrzKS4E2CjE51\\\\\\\/9xq150uPiDLagKLQLC1\\\\\\\/aIpMr8SXx5zZEwUjb3kB5ULvJ33Fq0drCx4r3hcJIp\\\\\\\/NWITtmIwhWLHdaV35RwQiaiMpaJ3RtASiWKxOh6Dimu4CNHPZfUwxfVdXyDI+A3biIxoQqoJ\\\\\\\/YTwNvNZsZ2s29unEVYajWbjtU7S2EQou0HVnG6crm+ad1SIMHvujemB11EHqeme7OMYMUFyCksX10j3gsYOaIeBOcu5na3HF9lupECnM8wgev83UN\\\\\\\/yaJmlSb9BaObBCdydxwvIs8\\\\\\\/kmTxpRdkjuo5sk+Ki7t4lbpfDcKLykBlax08Eb\\\\\\\/UleE5SIRtvI3hmEj2OFlscUok9U3+8+X7ymR\\\\\\\/WPYFrqMblmiyBVu6zn25eHwDic6c+ImyVsNeKfQCg8MqcTKtan1vp9Jm5UFCNV+R3TgjzR0J3UKsJEZggXD5RQAHjNQz4kISYSBklxikynYfXfmxOYnICQx9+9IywtdMCg2t5NQClWDqPeCrogGVfp+VhYNQSK33WObDbVO6A9Q48SZt7eOv0gQR6K+rZRds+voo9sqk5tUjB2cZPLgZvj8ao3up0BPhKFdPLyVl75MpainpYz4\\\\\\\/\\\\\\\/h9bDolDH7mlHsMMRuOqRoLVXH7qvL7PBegJTz78xrwjHtZZQ+SRIZAJoLfXB1J5GN66grDKx5ibalbcUNp6h9tWreQSui\\\\\\\/seU8B9Ez5r2aCEIxbsdDyXIz0RxQglcqXNg\\\\\\\/f4ZGp3B8AMaWFZJ+dZ+9HIFp30I8bnlHvwQd2\\\\\\\/yw8ZqT4kVgsmSFqZZet8dkyntdZaL8GO6FOTmt6DNVXYyjKVt0jtC34g1mSkwMBUZfaSbGkLfYHznK+vPyAZVPSLa9pdyO0cv2ubyApkIjxR2dgp98MBezBGaSWIqItMxS5hNnUQ9Io9Gbk\\\\\\\/aq5atH+KrgLYPXN0bxAHj1g+Bsa7YwbAGMbLAwS34xJh8\\\\\\\/WuC78I6nNsgd4mE8\\\\\\\/AIhC\\\\\\\/RsvPqItv9ApqPVlrG6GTUdmMwMiBo\\\\\\\/hEsxsNv5\\\\\\\/nvgocPZSn8kzk9nagWJzkiaYOBojtqVHcfl3FjVAm4AlTnOm7BiGoQWpYqfNo8bPX+63Gh3X7QtABG3JgZvYpgSG6Va6CtJQqxDVrMd+NRn+UdYOiyXbgb8BDwSlZnoznqeZ+5oKBPAaPeEanakeWCOxjuXYI69tLHNqrB4BCpBs8723kOAsajjntqZ00l0zSBJH1xVvbPpVYpDpCKUvqDpipnOkwXwt0fNzDxVKTOagYCDPWf7QN7HjTG3XPZzTZoU7Tx4v23MorRyq+jRA34ZAnMRHd7h30NbIHceBn6nCoUhmTRIj4dBk0dj1z4ppn0UtUgovSE+\\\\\\\/Ga9bxIskpxbjGIiCxulhd1WS+bgvsEdUDQD\\\\\\\/DpRoWUvTtFG3Wm\\\\\\\/1ds989DS1Rf\\\\\\\/EA+e9Om9wMQ78DoB2hUCqFFz9pXdpSJKbD8rv+dgsDYgTdZzB0N9nPIvs1YimhfOUTNlqP0jGRne5dyzNcU4ACXFt+oQhGDv+8KQQtWYtJlPfWl6WMn0N0rXhfsQKus5uV5fkHfOZEg5ox6ZGss05Wdh2aF4NcXHXRSmvSd59l\\\\\\\/Bjh60NFe+0MjfcxseLQlNBCJGNs5+8lpCqQQavvA4ijDep105t8rbMpSmtNmSU\\\\\\\/iYd5ZZWXbcJDL68Gw13PtmPFSBLf4D4tQkgD6NxRSlwDP94byY+AGelwpbqpREfwcCY9IDoXCPb\\\\\\\/eJUib97Ui5C+r6kj2xEzlDB2RlNn+obFIFQSG5tMmYuJAIL89QiMPPSF8QVuSTs6Je2ZG8LWO8N\\\\\\\/HC8t82deL6hCU86Y1sEmK+nRyqtMkd2TbU7uuxiyX\\\\\\\/H81b4GlzfQDWVXVXbq2wU3rfT9mAVg+dtHCey117sh4WRtjt9P63sSWsMXR5Yp2S74jSgDhv8OE9JAWgloAH3gTyaGD67p0vm1J+8YvZRKnb11blAdCjO4es6HW3bCQobPPwOS20Pfimw6jXxtfCXbGFP5dIxH+yf4hSktXDJmLdWSjKOH+qWWiIWAlsiWPUdr+fCSU2uVgyUoLtKEd6j6\\\\\\\/lz2g0Rf\\\\\\\/uN36weFBPU95SXPR2eIyKB6oshLltVb54C702QerF8e3H0S1tHYe3+Jsme9pECzAzGz\\\\\\\/c0KCCiIXpZKS0G8rq\\\\\\\/m4ehD1YgBAFEnO1eUKTm89orU\\\\\\\/i0Fi2vsnaO5FS55TX7r4xBQf6VppoT9VZQMtRp\\\\\\\/gjoaw96tKh0HO0SQSO0ibfgmPJGIW98rfw1jEGim39ldB2usV+MBM1bCMCXvZumOOH4WiQjouemsFXWGQfWFGMhnzju5vlun+eLlSVz0yWtWfMqlbIntbJP2kyh2DziIDHJQos1F2GvOyiN\\\\\\\/zrZYR8diEjVRFl\\\\\\\/\\\\\\\/pN5gVrzaiuJ2QAgh1cgWZynWiYzHwx7aDiOarYSHLIaLMFjkTkGnxF9b5YjeL2guwoqjcOtxAzfc4w2f6sLJJR1xBsmmrbmRoy0EkIQhjaLe+WK8wajjRcHFgBgG7+9ePzNJKRSs31wVzkYTHVgLtOopLTdqBDBm+\\\\\\\/e\\\\\\\/N84VslgMSq+8luOvcKkjzcD02PzbrZHRlVQZN+sGfuHwNCW1MHUztg4atYOKwD\\\\\\\/y0lwviRXWhatmcz+887hLpQxlfxq6pTri5WNPNOCfQbs6j6NN7PwNLVgHEVRBX2BuAPFElpispdlNCwfLPQQgS5yJ7\\\\\\\/qeVQfHImbf5FZedum+9GQG3BJAqHKWx3IPnpViI1SrtcjRCq6HMTaC6EXYuiRcDvTTc4GVgnlZbNvAAldvjqcOQdLg4qB\\\\\\\/LrkYMXA6etH2sdyb1nrBLhoxwPUtHt2xhHdVTivUfJE35bldvL8N40GNBVPFfFMgU2CaWhkiXopo11gzn\\\\\\\/tegN9Lsa3V5Rhb2\\\\\\\/iUK6hohKOwxSzxTRY0hzUgHXrIrib6vaIM28ZsPth\\\\\\\/CyepCsRCd4I0uB7Tfb4WUzlj6zo814pKnkfiabmda7uQncQ6wkYB6ZtzW8lRp3mTlKzU7ASbebEVqONNQGmUUSRBpKryAc5B87\\\\\\\/HWww\\\\\\\/04UfPJtsCMJxTjkBGHTTU57\\\\\\\/W1l76sId+JMFqpguef1ZBLWqPyh2djSHmmX9aX\\\\\\\/wM8EQPUjlIehKGzR7gmZoGBZ3VGuYdK9L\\\\\\\/vLd\\\\\\\/nG61y2leyNMV9ifdmeXx5InVOQbULiudwzeOE1tlEiVaVcVrk+aXSBu5+nRx46tOjCbd+2p+5AwzcF2Sk6p\\\\\\\/mKTKCy7QmI5iebhrtB8llfIdhWeEUbVIpjrt+BIb0ZTElPpHFgyF\\\\\\\/tFvBLYTcPqUth0vyanMO+UIgLgvD4NCN++3o6BkJ2u0AliGPd4ZftipeVEfvEnEoo3t8R\\\\\\\/lFZkZV+IQ9Gfg\\\\\\\/nqXhZ8VelDXKUA9eRAXP\\\\\\\/Uxb1CZVMZvqGGT7bVQvIIb8fd3LN1YSVTqyzNkJz+Ci+OMKZCpJqA535vSHUo6CVhPAMqXeMaTruFtVfNydFRqZuX1yCehB5fL0nkmJ\\\\\\\/lIqZZ8\\\\\\\/Itk04j41xE5snm3WbEjGYe1mA6Ub1+b41UV\\\\\\\/dCp+w+\\\\\\\/6n\\\\\\\/MhqXoQmAnoAf6aeEfOpCOAuTltZydpnUpEDn\\\\\\\/rXszcftLzDDps1N+g0Oba1d3faXXkzhzTYdmPgXKyORFKz3lzNwrDZP9BDzngFxrIsQtYEx5jKY0zEvkKTbs03YqrNdWlScKQXgMWBpjnw8sGAA2YxVOQQMbz0EldxKoeIwJvDJGhueFUhXoV0SeRQFv1jkjbPLTYkly60dBks7JCiqvVw0tMOn9s0T7YRZfEa5WQOaeGkkASmE7qDEXs88DBCG70vum++Jv5B6Gm0MXMW8mul3YSqqZVKNA5Ylm6VERM3LjC2Bs4UeQy20YTxYfuS0N1Q6eFDWUShthmtUiW3npOTUJrRdH9PNYYHbGlkaAFOid99kx07cej4xlDd\\\\\\\/ivIfXOJ4zXufONJZH+2BOOSeCrHV8bP9p6rNLyk0r+5y0QjIrJWODYlUKTQSX+7nHg2xnHH5ErZOMxt8uxNifrK3D\\\\\\\/wOnEf5oGp3e7wBEE5NITVKOkBtuazf22MkmxRcX7BBvQXuK3P3BQl\\\\\\\/68eHaSqigkOJEfMlIl\\\\\\\/rT8PabP3jpDh5vuAZNwbvYYKb71hKN73ZQ8r60FYrXjktSGHiY2btRIoCmy\\\\\\\/yFaAujFn7Q2pW9Y8m7l2ldbY2rU\\\\\\\/5cvd3z2PiFJsOcQSJo5\\\\\\\/a7F\\\\\\\/pIvT39QjMvsAAv7z48eRhObYsrA55x2P2VRyaWfhvwzzCsor45YpJzOC\\\\\\\/EQbBExEz5aJ\\\\\\\/21hu8GWAZK1zSsvMaHj+ZmQsS1c5X54lDRDfW9WDwTLm3Uvu3p0vqa5YBQuAm326ii+5WVTiF2XQ2AzCC9BQT5hz2BRGT0q\\\\\\\/zC0qYX2QzJPYzjl6niM1wnYQtO92fyMoyv2EsRAsbkzkuuIfOI1Aue\\\\\\\/lRnY1QADn3nDifSk7xGgmk+r0NTBmoGcgZLJi1FJJBDDhjarEeamIJWJTXepQBHCnAFniAjlBjjEUNDS840qC3c7VciUhPa89BolLvzenh4J66bYsds\\\\\\\/sMSs1cHV75RRCd87KRPSgf11GwZeHA4jbBds7uIaGBjH3r9yrAJ+BVZ\\\\\\\/7YHBynCxA+g6whsbRCSZ\\\\\\\/S1R2j+nxhlKdDXG2PvoteOwe9PZHDDixTzpYuwXhcXzeYfuvDXM67H2vDC6R8XReIQRhW5Z6JlHN3\\\\\\\/ZlVz9YXfN93cbAr90pyBE7D0xnkwZCFXizf2OotxwFAzrtS6BXb5UdZWVe\\\\\\\/AIV8onyWhyU0kZWd1YCAJMTsHNeNBQ575QrD0WUQIvS9lVAfgbzSkbNOfZun8XbBmhxRvDKHUtTbyYzivNwDRMpv\\\\\\\/zu\\\\\\\/HCeRxrYdxLCLsAN7Ne3LyPvlyHxkXf4XvqBF\\\\\\\/8uc8KKqR2+ePKQG2IAhcVzE+AL5w1eckappn0enjOI7MOia07D+8vS5YopM8jbDEhELcmHU+ew5rWqY6AnQNvbaQ0Q48M0jl+4nQ3evIuiakVTY134EWgmKHpqzyz\\\\\\\/Fx7EufdM5XD67BL+yI94gPcM+3e4ZUB9zO6MZ+nldtgjnlIpFVi7xZVIRowrqA\\\\\\\/v\\\\\\\/lnPdOPqMbBWG9Tb+Y1SPCM7xXYNZY\\\\\\\/wjaiBB0i9X7AhljaOQ2ukEWcGrPdodpOjZzP8nzREsxMNSX0QuxBgXakxHZ5kicxmSHHELQq+\\\\\\\/3xwTJQbvD5S98BzpiCLlngX8YVwdMxdbfrPpF7E5744SLY1cibEhip4oaWUrx58tDc9\\\\\\\/9\\\\\\\/KlQjFmdoSzmUSL7MeEgTaJyEzJDsk3trhQesOAJG4Elzgez0D5nhQ5BpIMLzIY+7RwyRAMOwtoVK3G6XORlOKbZCvipyTtUpl0qYsqXJzIC6YEcQCgI8EEd+UbrDZP3OhKhDDqrJZsvbR3sRKdqXhAY0REgRGuzvb8G\\\\\\\/o3TMJKY7lLpCtOzWwKCFPMW1TT2Wp2Vru47lEKKzi65NfxvqHrw1wLC3oxRLm26Q8dGI6bODf8TJlQmctfAPE8OBgNT0MKjp9RTFUxA7AIsJJdZumGDrfA3D\\\\\\\/ub8n5vK\\\\\\\/OzZ6A\\\\\\\/4DvjYQ5+4jzHsfVuLyWGAtD23PytDN09KjT2p5P6L+e6ZY8IVi+eZ4xLH\\\\\\\/9eja6j3BWAjoMQPTw3hZWBpOAEFnaQ2\\\\\\\/p8VR00swizQY3XhcCbg6ltIyuu62GPUSLZrBn5YAwEZDwd4Yg8P111B9KS5DlqxUYj3Pqw6lxP228qEH4bP8zBuB2vpGwNShUfQoPWpkTA76U7ObcMPindf5fEkb3HbAiKjtwvU+YzE8WPAyRiHBtC0ZHssIpPRBzdsYtwDMXqS9QhFNcYsFkUpGJ2hKlNNZ+wIv1+WpB80bju4sdo7Fl\\\\\\\/NxCiplpFrlmi++SyGyeYW4982Kexu45JbtcyT2FHwk19SekQ2dJXJDtFCMzfWB86g2WyOdYAQ5IWdQskdm7nGhSL8dGIXI7cwf6HGzhv0fpDCF2Nl4c3b4+OeXD2+ZFAW868Yx9vQD16Fd4CAYPSNxfgFUUzrj5a8cy6t\\\\\\\/2soMPJh9D2aJKMjTw4C\\\\\\\/rHPbRP9a3k45HWIxiamE0qKDYql3llWe73Z30\\\\\\\/nQuQABKexyf+i4q8frQ6zEOg9oJcL4dUSaT+cxcAfSCduHDaSidA9YERdXu0hNp6xMKGyeIZwXCOuZiZsfYdFzfi5QZ9jEQPefEJJSIfe\\\\\\\/Bkhp9fWhdabzGhEFbQbUpRNoMtAco0UAKJfLAxi1GXiQUeb1z0VjSm\\\\\\\/4EbnxW\\\\\\\/eQa\\\\\\\/HWjM+Jy\\\\\\\/mqWI3EgmKkbjjhRUn8ytemi0miHOQg5BUvD0ROrH9AT2iiRX\\\\\\\/YgQWNO53I0gjqjVEVxi1PncgHNaaAp1xRFrB2xsTcEIv1PcdbRR1esOUnaAMLGnfZpdrgvjk0zWm0nBkT\\\\\\\/oxy7StGuFUZ2\\\\\\\/mH4aRZvwbefGRQwuYy90dKDUrzYotwvJMO53tFw372MCK7L2j1Bf1OK0CgdTDPefJYUnBcusL21JIi7TwrwJgcPZ3UIQkhuDYGkbbA35wdVMrMflU7tB5WOsJjaJZ9opCNorxgIy9pxFHH3sxjJLgcGirOfNnDin9CXLxMdSlIjZ1HKInddpbGbhJQyXCisUk1k+kPOyi4MnlX5xwTs\\\\\\\/nLhwuy\\\\\\\/xMKidFJC78US2UYUnmRO3CXWXq3M4NND4UCnYZztOe184JUPc4tvUCBt83Mn\\\\\\\/rr7yHbgQryVbm9x0dD4pGmd\\\\\\\/xVN5HpchJKp55KMqQYo2l06Tx6O7juJWc05ZdcIjNtmwnORHRNEetRMxbbIYsmvRdtnIQ3H6zlZntQZLtqzTWOjaatAt+fe4A7BW52Zl9Kde5BgQjXYs1q0EWI19QM1AaqrOaqf0Zz5loW6DvDmvp0fV6gd2c2w2BwT\\\\\\\/dMPNNDFFgrDqAjvKlQNN1b0SRZJfEF6KxC2397IRv5uZD5HpCODnMH\\\\\\\/RVz\\\\\\\/6PnOfqTNviEji9vwimpXf682dzg9pS6Eu1B\\\\\\\/KsXExiVFW7Q1kKZP1mqyBuYDy\\\\\\\/u0rlMDahvbwFeZDnszYOjXWz2wiDcJmhwxba1OGln+FeoDKYNExqtQTFsCWaFIS6cnTqO4qbci0PFztFfDNFGsqIJVAn+F4Hl9GdZawdC\\\\\\\/MJWrCyBu9wbnUr6A2sogsmR3Okxuh0zC9Vmjx0CCejAIswnSVxO26uDbe4tJr7ZFT2EYNFc15ZpQF1tpieRxby89mIRux\\\\\\\/TP0srU2oG8CbyOJ+t16OeJ5tfVtlHfDudp\\\\\\\/6oOM2YOli\\\\\\\/cZfhOFlCqYVnqqExD9WtediVl3pXeo2sNVbrkj\\\\\\\/u5TAAop7X8OHv0c94eeVAifXFkysRGTUjgeAamvywjR+gT\\\\\\\/\\\\\\\/FlFoN4q6MpdbU4G2EvihFd9ku7PQMqdCmQrpy3OVaj6f1kzfroja0QRYhyfQ+oOk5flGXqMHidsuyBjyPyY3bIj7ckA+jxht3ttVS0AN810i7Vyi2cD\\\\\\\/YlZ38mAlzbt\\\\\\\/RaU9\\\\\\\/buraZhzetjI4QuJQ3yWC+i9SYsWZVAATbZoCqmY\\\\\\\/tFczES0FKh+UfDfkaoA\\\\\\\/equzhpGLIrH3Ktgk\\\\\\\/BQRCjb60b\\\\\\\/y6h2uOdsS2a3QGBBhwmNLfvm20t\\\\\\\/\\\\\\\/WbaoRtBlwuUEImQHlfwxxe2oCwwN2\\\\\\\/2UsaSJ9BJMS4tuW6YkOtRVO6VfQhxrJeNJ8gqgS4GlzTYbvQzHJgI38C8fnreo\\\\\\\/Bc657SLa+jouOcySWpV6sDKC7QLYba1G91WoCwVDOQcUWB6qsGPxgBbjuSnifye1Nm2Jq3PRhxudo5zyZSV1w9oti5V52HcCcjzTym\\\\\\\/aQXZE0vIfwVz1CrGitsj8C4JCfV114YaERPVf4hz7Ja6iz750n58p8nuFmofGqYxqtERQDD16ODaIjJdnuu9\\\\\\\/cWX\\\\\\\/K3\\\\\\\/wPzjU3R9r7MBBzSPX30xiGtKungiwOmrQNB09yPdk8LwCAJ7w75VkFySQF8CyReTqDl0TaJBVrhEEcY0iyWEkTKCa9xnVFOkPVyK7AYkaNOMFyIuR6rR87+b4rdQ29CWT2o35BDM\\\\\\\/O+cTrRGZpvwdXLhwQFGLJkfZnRLs5IcUZY\\\\\\\/obDKB7jAtFlBrzkzvQS6hA6ggU+eOxfqtYT\\\\\\\/z85c8gyxY2ZcvR2HN8VKW5MPMOoyrUmH9gRxSBqgfksxnwk8I6cEI+Tnsu2gaKjTyaZo9yIMW1etgQuepVf6+iaCxTOiBL\\\\\\\/qjlv\\\\\\\/9SUNdLNVCdcyJ+i\\\\\\\/C6CTvr1p0D8o6MUroKrO7VsdvzOJR480G4qqLefJg7CsMvPtOA7\\\\\\\/5BtNGOBB7kr+hH3YZVtmpdDxPM2HpnIjV4h77metqcoGQU4MNxKWUc3NFR7u\\\\\\\/H6F6lefvrwS3WSXRejbUWornu5qgPZuZm4idiHs6A2YPbNNaeb7H7uhpRYxfU2qbXzOjrgLGCnPzDC8vVQoh7+V9nJVSHcn60sOJzOuMwncmx5m\\\\\\\/rhNXQ5V9lYECkzWr8ITEqaIuAq\\\\\\\/azbtpNMRqPvuHRsjd0MSDQC84nqHQFLA8MZY6qQLLvq9GUVfTEV3e93yDkBr7r72JW1F3SdnqWBdigD+hZoh+qmpiZLOt49IpC92AWIB2fpp7XttwhSgV5IONHd\\\\\\\/JcecFnh\\\\\\\/YRwi5q0tt14OJhhxiXue1fvKGiBdrMPuK0\\\\\\\/TEP7zwmec0xVLsxYCLhKc7yZdBzMdi\\\\\\\/KySCrDhkVTCKHhciPKzjxRtU6AJG1bGUq\\\\\\\/V04kPfNGkFLTMi7FnPsg4IA2A9XVjOPvJzr+aogwspRnzuNLnUegjNkxyfFhkqrPqYMjepg9zquupHP4l0lD0t51jKQuPhkUkm+qx+869pIYCX5Y5YSKeLAHrmo3Ib2l5eo09v0fst5XwEj8A0GnDB2l655vVs6UprutorFKe4L0JVUV00yLabmcz6sM6\\\\\\\/n\\\\\\\/wvvMXeV4DlAE1cfPzqII5pMsFaGLBCeFoo47Izw0ixo6qh53OeIAVFgtxXMirlmLK7uTLQpBy\\\\\\\/9dqI45AHyX2AKTKOpWF8OjJy6pEw0w9SMZz8brzNRqlMv3wwvCcxlTRUpIut9Y\\\\\\\/7z\\\\\\\/f5C1zIndLuv7DAbuCNrRUDRWCnB\\\\\\\/RpgwHTyrXiEQHsc2rlCHhZ2pREjK8M9PRpT345uM8FiP9qrtYZp9nchzwaYHBkCgmrlnVhOw19xLmPloSozncH1CTntvBmoXBuaP9a9bHQjMPuFK8pHZo71HXc\\\\\\\/6P1jsEvOljhLYA3zMtP0euZ6GpoIUJ4uwxABUFgS9ax+Zn5g4U6dY79\\\\\\\/UMmbAfqtXDelY4Ut6A9eVpEVkngkk3ITTTCGzOY3I7cAvGXXdtAWW5m5pAIcE8FG\\\\\\\/duvaqL8H2nXlq3AFEyqHumx2XvkX7++xjdmk4HqiRFJ51VAcpd+\\\\\\\/TggMxPIUGyWvvygu1WCemkj69w4RJCKcJHd\\\\\\\/RgOhqOqdlEj2BTpZuWJiBDimu4rKqVL+c9M7v1fguJzfyqVZWaMGWXHgOCmtNkA8jrB6dNkgOZSNcRER+uXV7ZyjZ+0VPL7k2qsuPlBgIwTkAHGrS1KCA5unPQtDAJBoysxC1nREDmSjXEyWA1YmAyCEEP1zTbthhWfsSvlJDLLsn+lC6CUcuL4MxNYqx2TGLNE98ojR9+\\\\\\\/lqKDvNIe82XPapF2D12Yarzu2cx4GaRd0v+BrsZ9A6UQfRq10\\\\\\\/2AAjFVaoQ12aKsEfIbo30n61OhNbDgiymdrORBMciSVZM7WcTJnkoREgw+B05dZhMUDwyxvbBgaY21+w9neAthd7hNKi\\\\\\\/8EpLxgODs80OfAm7DANn7xUX3FqtGpOT50fHrDrVPRv+SuviBbtVfo+KPxNtYFHJusfBSRcwW73DK8NLvR0belO\\\\\\\/ThJHxKnmK16L74dyMi7uKbg6KUvH2RPtw8+bbd9jPejLqUofT8hyoySU0VZZn7GokzH5y98AJsoTjfsi9yJ+qYtqpqY5IjeQa+tBmpfWa6to4AKNcyr3XzIq7hVnlXwT33MYt+u2iYfAfn0nanMKkY3wEZH+5\\\\\\\/CIrx2xmAVHrJql1mAAyrRmkbKTRV9u+eaVcPPrTB4UTQuv9wBiNNnX0XXQyzWmW7o2CuQgrhoLAtaB\\\\\\\/GTbchx9tWOqGFB2Xocz4qqCDnqbPC7ro3X4Nbx1SIPF2bRLtnqaqypyEgjBcBnfkivJ5kBSvZ99Mrd3Hv8g0eNJDrbptodBspA+lxwqbcAs88PeJIV\\\\\\\/duLRN74kya0f\\\\\\\/fGb5NpkcHLDNIeEl2wuKlxNIq6EUDlXprkqEBtH\\\\\\\/gfVJKRLT0I2VD+ttJopFTC1OIH2ZfspsgObdtPZkNSyeivnZIUxEbRS+9YtrjE4sgWCeqtkEqXV2z2DeZny0ke9FruYAA6\\\\\\\/K\\\\\\\/zAjaSrMSffNYtbXJZWVWp1Muz583VMqAGjpnNrWCnGZJZgZeosK0GP99kZuceWgbqN2A15P05QKUgxymChywrgBfJLSUsU8gIVyROK5AXRCMaEiK0bcVpEu3p6hojmzYJn9KnpviVYiy+104AAtnmvzct6PvhJwGe\\\\\\\/CTQp6uth+1OgBw+As\\\\\\\/lmwJoLPTOFJEDSzGESo0uuPHL02Raz0MoH0PBGze\\\\\\\/mvY1VUWu+XAQGW0tdv9A9pN1euIVkHteETEyXKmKlkilwGZJXX04RKBxj0FKInql9jrK2c69CgttXCmDcoeUmOMdHMg+ZqKyIKrcrvKX4qO0oxEm4Xc+loyiBifoRRXLrS5sfZjD2MBalVA77KYITrBi2UCIaw0s\\\\\\\/HPo1X09dgAwlJ5XFqT0OkL7RyDfASakaDKAbRLaWijD7Sz268ElBJmT8bT\\\\\\\/6gi5z+jqZ0nQFF93Y1osvVJdha7gGexd\\\\\\\/ldWoT5RnLSzqJUeOCR4zDb+bVa\\\\\\\/ezQAoY75fgczYTN8kqGU7QwcFPj1zYgGus6LDo0ZF0CAq45UWwXxVQO+lZ3CBM6Li4eY\\\\\\\/EgM0g8QKDu3FnASqWfPltSZEVSp0eAjAZOtGMPYh69dxdsD4SuO3f0x4YOM3Yfl6PICdveqAlG+WybOYdZyyjVbpNT2ZNDVpKxgoirw0Q9iW5G8s+Ik4OSnfG3xlxn7toaxQWjjArqrL9hVn3UHyMKKwoS3Ha3qdBlw3Qp0dmGJYuT9hwOAPFrT6rBqzNzllGW7pIav06cbrCfR\\\\\\\/XRi+1rsAqvinFSCvgfzBXa5cqGm6euGuJaBFO31PYA83hUXYz6h8dq5Q62vXb3\\\\\\\/kQWj4QU5VBmsc4mLeJzG84J6adQDvNvzE1mioN1ZY9kF9DdtxyoKg9O4uXKxdWFDJpN2+B4kEniR4UzKNcb1B4pjvUemAfV6rYZl7AmXbLBebnZe8UpG0tBfJjM0btJGOdVLPCnGhRv1\\\\\\\/Xe+dkv64U7pxbOJd2TkI+KN0zW9CvNOLjq1Lcbu33p0bohmJpBEtdt4AwG96c7z0G9ErYR6wr46rTUw8bblZX5Uv\\\\\\\/jg+bZU8s\\\\\\\/p1agMgiAr0nuTdsTsS7AwXS5mC3zJIxW7Acm6hWgANN8yDKGSct3fMd4a0AXnLbjfcgqEjU9MGH8vvwWlY8azIZwiANXoVNisUIBF22nC6aDnL9IYIXpTM6BXtbRLVcrmr8L1RP1FiEl4LcXL6R1\\\\\\\/xR59LOxPF1mZOK3xqSNogTfiWeB2zj3YjcaF0WIJE1dxWPDwux1aAuch5Qg83GheYnuyLgi6U66SSAXPhNEWJk9fQFiUA04ruWtiDqCFkxUT0cZLN0WGic+odAw5MlUSH0uKLKbdv2KujVEWAyChXDZ88P2SXOHXeK19lZtUjvcX5+i2FNKOgm6eWDoGVPUOTOfgDzXdqXkyMWkmCU4bqbenpSNOlLkph3X7pUUTOwrYRBeKrrzvZm3PrUDusNsvzrIFc+pfMLMLXGj8Bk0BP9nFXdax6PcFl8ytZYch9au161ldWGMDJ6C8ErIBtvm74YQcEDB\\\\\\\/iQIYWp5E2LUojDtq7fY+xML02JjOofgTcuLhIcouV2d4k+vDMvdYBxEiJNnuLv7CNCjHaeYKqn1+W4qobWVsX4DesFaXKe1uxRomkAztafb+d7DXvszhk5OHdkYanPtsIr\\\\\\\/FOca1TrT3Fdnlq0cK2LCqM7MuiqFdGE5KVXhq2t2vpzcMDXt4z6Xlf8df0YCcxxeEjbHCJ8pTPos4mGWufYpKeJJiW13QL21K9CgwjPVNzUa4ptk0Rne2s6yPHn\\\\\\\/B3DcYOxuWGp5gC1xGS3rIUILb0LFgAkjBFcFF4A4Uka5VZJJYuUijL8UcyERSRs4PgvEGy\\\\\\\/sUd1cduw6OZX\\\\\\\/F0Ly6b\\\\\\\/25c0mj2bvHiOOe5miX2yWp0NemtVtmdEa4IWZKUzFEAbUXPMMpL9y8TIWtgdHtLBNic4fkWPiq+a+A+Ha3VBJMrUlZh7lfylBtnnSpHhc7y6EaknrZHFUTSvN+VC+iWLcWwqj3LzCrwgUmeyRphDNWYMw8QOpl3YzgX3Od9DSLSy4xdpRQlwIZwpMeiBACbwFG3Up72SEI3Svy7tfxRzG9wQxvekAWnasP7gGC\\\\\\\/YeKfcQJP2zNI8LK7ezyguxTDwYcJDyEhQb9fc5CLUFOWXJzrq0HcJ1kq09c7wLC88jbEq6xBTGyFHFyzRDcNIVdIHlhXIeK9jeeoZNEmcAZD6VMZVVDKvIPaITmuqRMrtVogNWXL3hEzSBqiSlWLTvHomoUQe4VjVKo3RfwsZqMOClxThO8WsqI1Qj57Aj3VpFsbhZ+IjnV1mjB\\\\\\\/AQrP+qDgkuKoqaTbDN0uBmOL48fqz4e5M9TLB1nCnMUicc\\\\\\\/GrfUsP6RchfEH1IhoiYWNXWK1CHTHHCdr+CZQVnH8leAOEr0OeeBsZ6SnAluMcZYexfjDwMDdYr5xLDamppRwNJwENpid\\\\\\\/zsj1iYT0RHl4vDhQTuTkl92a5VoPfNJBpq79NVQKcFDXxZLiLWQuujQ9r+l1fxwnSzrlt4qp2ipJdxEphnXJQnSrkWq7+tTyuly9IZtmb8E3PXHXMqmb2vArxrcCfQwRQt1AtqD9anD5MU+uNOY69k2lwVt4vCp9NFXKwNVFXblscEd1rifVx4Jtym2Ne6UYCAvQ4phFVCJ59JiylCEKZMxpKoUHqoRhLK9K\\\\\\\/biTevVPlUjRKONhyfwMI9\\\\\\\/ELD6gI42DxzZbzKRPAd6cttz52EJeTVt0L5V2e2EAeEW4fxcU3shUV7hgwX5t82TMCeCBIrqFgvYcd8ubutIyZdsJvThTL57+sMCXJm\\\\\\\/iTd3lIgrTbaHJpgaCEWDx\\\\\\\/ktaKPfsdPXgAK87R5SLkxVNQCUA6vBEG4ygjmBJx6SARItzHWkhEalPg3SPE4IMtwrGMIx4SuKJjOIw1xYpkw9psn3sg1az964wGb\\\\\\\/+KNgpMuRfW952jflIHTFmNFfu0Ez25PeUDCuMh4jU\\\\\\\/Xku0h275pSFuYnTsST2EwsGBSniaeCWnzrO+HKxKPil+PTxhM8KKD\\\\\\\/0NOhmM+sIUsJ0+fu83eH9Ab2A\\\\\\\/D9oDWfDb0Fp94Tr4e9eLIUvK\\\\\\\/CeIxZU4FffhVQOPOmHPh1iEI4qRSgtVpbD7z468x+K\\\\\\\/H18CufkLMwtNkCAErjbmuXO2V\\\\\\\/a30PpmoEQOfwiXERmIIS9\\\\\\\/zgfuGEXBCPaaTpGuw3L7poMYh\\\\\\\/6pAjVqM8Kzph0xGm1cwLgzsHtIwaZPrjWHUyU7VBvEf6UY7r7x5hcqWOCzy9tU71l6LmE0TkktwXLNx7DoTi9DuLUOuqKaIzbqOvgEQ0+\\\\\\\/0q2uUCEVms4IezKd6RF5Ooks45uWysly9lBnyXvGbRECzJEfI1NhFpJi9I6fJuNP98Cy0r+sDmjq\\\\\\\/8FscIU26xu9cBXtSbZTL9Ca6ognDhBKRDoMTIf6PZxEv4VQzugEQZqAGvzZZRmOdfcTpQ8m+avyjixpwgH8IClcecQjEwtC3s6pIhSbNbRcogeKSpQDx7PBuLM7u9JCiFhAJFG4KYP\\\\\\\/ycM6zEijHuqlCH3E0cOX0PiIanQGL6MIEz0U8TFIq8z0iWAI81TWMsoWrXU6KRQNutgtFUalyYyJIF2HxXJIXdXkzJa3AjkT3iD6ggikbBjp\\\\\\\/XwV+ESpZhRVQvHESZ2ZoowsqUbKQ+hGrPUtgLTQhcaZUqWVJjwCJeVbG10F0y8Ngoa+Tw6dnQhDHYbw7OTk8gjstzejDU\\\\\\\/uXTRH7rP+pJ7QfZ7QOoqcdSs5b9Z4656CFvinWi9Pjnzz0eMkotHjbJ8sccIz73n1q\\\\\\\/rpvrdJHXO83antNzvWlV17HXjSgwZ6PqAzW5l0IKxV4GV0KIUVnKvW19UQtySfoocdvH92bcPJ0iC4q1CIv6ESZbF85IFH2\\\\\\\/OWBtzUmHZ3h9QA4G1UKll+CiqYckLH9PIM+oftkpsFcySdGQQpnxQjadUKdya0+OZ3luGKKb+y76KXCdAT5zeEO3ICfojP7DsFmqlBTTGBiNDQ0WBZ2WIb7dnVq5cdUCkjSjA5PCICXLv+gzfSIhvr909vP\\\\\\\/DuaIOYNydL+HBOcu\\\\\\\/z7Uwu+nwAPceVGyoYkuD4OlOlVXWvrOT\\\\\\\/JMJMzXBL0zN5xpGDjbkNauUO3\\\\\\\/Q6mK7uUViUhp4Lc4gCR\\\\\\\/nyZEJ5A8opyqMiSHTRSKue\\\\\\\/CxC9FiATF3hqqm8wuJdl4nEATpSj56P7NmhQ7BACWcB055pFR3EBz09pOtDZ+oUr6TXrP87aJJUu\\\\\\\/5ds56nKE4BoC5MJrLKmxZQSqFAcnGBUvfAyFoEl7yKZCCZkaER0ZuI4QfMnZx3wQJnplHWj80+ZjSEKGXcAQ1ucrSpVOftowN6ptcOLR\\\\\\\/CZSQQEOSNzMud2e5Xk6bBAxT91hd8DiWkYVCjIkfpbrTCiezchJ9yt3mBigd8BWu+7zHOwDyiZbX\\\\\\\/HiAkNPgGMuqDpqzqgOp0AgqeNoZz8ZWZ\\\\\\\/iui9d7crikKDAXg4s9pds6qlgF5IIgochOjsW2ps7Ok4D+HNzz1OgGPv8uo1s6riHxDLenEkWfC++9KoLTlC1SVLZDFcnULrsrAciewKoZN29i+\\\\\\\/XczdkcRiYZAvpr\\\\\\\/pcc7CAV9i8Rv7wvo98OOQGVi4XByIZqyNg8yJYhQ9nWDnciA6nLBexCtrHeyGfZw25PONO11XnItCdEV8OUFkgLLwBdjWX58GM0TVeekb1bprRcSHsA9v5XZVpvEbjZT\\\\\\\/VvlMvsxF61jfxTcXygh02Rk56yaDdjxo26+FDcJyRFJmnxCsKPIWf7TASNiFlFarp9ItmoEd1\\\\\\\/kBxgpG+yUhN3+KKeNPR4phXTVas5Ijo1LUqtvdcsYxHfHKZZPnrGmJqjgLC5z7BhebEjJ5sLR+Mtg7gIasSzIUy00VkaIo6yvEAFkEUmObkrtVeH9fvvkfCCo0iAqmJ6nDSFvPcDwfbGzGlFPns5nU41B1RFoWAel9zSDMNqjNF\\\\\\\/tJXZ3zOjM9MmKXGzud8dT\\\\\\\/H+ZuX6sCZqLf9rfyG30wW1yIhBY3hX73e9KSiXMNg0CfLQUPolp0gBSavmqbw0h584GLDFrdJPC1Ni52kYupy5AQIDt0k9dCbZGpFZOSm3aN6KJxv6Bo6kmeGcQMS1\\\\\\\/Ia0SeZC6d7RaFwQh\\\\\\\/FpCLMjdaTFygJHZYoj7fkKTLylubEFsNzw0FqCNE+VuF2\\\\\\\/kgOBlgPlVO1KHE3cJBWlADHxfCXzQB3FAm3imVQYLtzvl8XUMIqRUa5ex7afQL9HoBiEk8ckdvFzaxuocr\\\\\\\/qxBcEnHvZhCpVoR9DwcmTrm0195VjTKUss5+oRWXedIwAVuRQdTBpolkyqJv2q3t67FvTOBlWXpGUecbrBL78oRQ6SBpJmW5uik+AvmJP+2KxZI22PhWoY\\\\\\\/gzT2wyitRHPupFJhc0E35qH3JLKFkdeAtI1QNZshYa0OaTLhZ9D124cY7D7UyTKEVFr2txtsvkzpafUyvGCD9HeGb+1IIOiDK4aPHxX3xTD7cvbkAb7LlGvsRW7Y8tBAW8pdplbzM0twRsgfzSevTmH3fwRHsOV8AGmVzJNz4ZjErjxFcYlVWti9nI4AXK2mcvJKeBIYs\\\\\\\/5dUPzqAhaXTe5EeIAO66ylTu864ykMhs7txzGeFF2QjKTUzHl2F7Kd+JFGxz1n9if3+5tl6dj9G05My7\\\\\\\/WrN1U44+m1d6y2oMm3MQz\\\\\\\/EKmt2qRwRNaStM06OqtkFOarcjdnZMMprZr6egcpgTvxrkHB90uIR9CTgEPO1w\\\\\\\/PzgaeLL0Eq5mzT5KsLXcBf3tCTHqnjDACGFJB2qiBvx4Mad25bZq9auwjEg2HcwMmruvEr7jy\\\\\\\/6q2M39f07YhFMwhCQoaHMLe4WpbNADK3qcZLjbyd8fOl0iDgikAcKZHT\\\\\\\/o7RvCOzBKY40KJBAVoX0ldbbRWFV5BO1POjZcVSuTlTHPTiRoKJeM+9PKzcxW0pQklETvA1StXllcu34sf2A4x2b116sHt7RMf8Y2YdBro1Avf1zwarF\\\\\\\/U1C1Fe+Q\\\\\\\/lVgeZ1K8FrsCHpinZvNVTo\\\\\\\/NzfAg3+4BIGq7FcLfchAyTv\\\\\\\/BxFrgJsFPFF0j3SJrpGNT6nY24S30aufcjYYqLv56TQHkrPYfVjAxnHW9UdrvWPfI3UtyDkNYCU+wX2UuFFZ75TRJW+B1qx49lSWBi8mMlZQtO2mR7ixN1N6SJqs\\\\\\\/tMbeXumzqaeRhlRJ0CVD956fuGUeFIRdOnNbf3Hj0rFbfp0+MffgDzhMgomgcLkIHok1EgvvIym265T8lVmTlmHPEEg7Vz6CdwTtjLI2qUEl\\\\\\\/HwSHvi+C0iM+01dS7npswWbF0\\\\\\\/ja3wqQt\\\\\\\/8TKEoEVshdJV94GdABcQDrnUyYKTrpDVeHy2Iwd\\\\\\\/qF3NavSTWGFIf9fRNoSTQ1IEnEF8BDe+Li3OmBr7Qqth\\\\\\\/cTvtFYRDYClhFbXOfZ40fcJOM\\\\\\\/EbjuDA2ZRUWNz+tb30ERAqKO0KaKIZoLy5wg7N47hJ3A1Kr0at2ARzNLnIW2BDquP0OGHPU3ByRQMAkgVhR+5EamNIJvp5P4tuJciJuueakRv+ClgWsEuBTiF+fGgw6b\\\\\\\/Yw1upjFKd8Q4jxGhwsm1tnN0130p46IgiV4WugADkXHdJrnV3IdfcLj59rFmW3ntex5Z\\\\\\\/fLYQHwshIf9T14lLTnqFEKaZqMRniYV08gjuePf0k8Jn7ey4JlrYBm4yJhYqmXxmQJcQVPgwMkeV9oWvLr45xt2xQGbk0UOQLmpy1TQRONcQqag7zIRuDuaXjaXOqyG5o9dDPiTHTGvTpQIY5R6kL\\\\\\\/cvDMvY4jk5\\\\\\\/sgebtMj588jX7q0Fu7h6JT4ovllpgbTC1u45VxjffHbIHGUrBwSCQB2a59y5CLcXa5A\\\\\\\/Fg5ihqPj79xwUaHHjP\\\\\\\/KWaAbQA0qIHA+AgB+MjKUPZ\\\\\\\/gmajcBXr4f\\\\\\\/bTW8G8zdtWg3cjYxtZB23HtrRD3mBPG4bFVATRIMEAl7zsEVx4Op8rhsJ4OShCKBIPDUSpo3dBg19prfo4hEvYPa7yTlfi8XeZ28ZtXdryzQwjVuTXk9rHh0plQgs9CRpRGhCwESzPQwfXMw6FOzxj+hqqv7H1kG6mzMTsDpoby9xkaJJRrvcRkajO6w6tWMuft3IiD6VvWJDprnbHFbvVQDI7ReTU61dYT88g0W73U065qETCfQQk\\\\\\\/I6gCDaroiErJPWtuPwb3skfrIrttj883MYjHCmYzLtkxbS2TZq6vNZsAeZ5jZkgSgqXYrXvk+w2h+1+j5KrO8ZmPoho3rDB6lOXeXPT5FlXzwouoXSNXSfOxTdRcTuQ69GLD7S8dOl8MuoE6ubKUjm8dnqPREtLIVVM\\\\\\\/r5nJWFzUnzObdwffDDMg3cPDbY6U6hctnbafGwFmXnwoJBDaj7syzK18ZPxJfPK5vPfvk7NvNrQALokSJ82qDUC6yR3CxBnJx31QwulqyxJ0tIG521P49i8\\\\\\\/FRra9ZFHC4ORx1VxqVCjMFhnjSP\\\\\\\/DszXSHw85qPRXjMOqPugQgJzATZxf7YrcmBgJZ4cT17dk6MwjR+V9Ew3WnHVW11dImQm4l5tfc\\\\\\\/ZFTvM3oRlJFmsRRdVzquRHzE8LycqayBjhocw8sX1QwViZwCAasyNCfcgPA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wwiNb0+QMLWHtyvAHoTCmA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002269d6f69acf5cfbd010636c67169ebc8a7ba21ed8ff80a219b4aaa7ef6d25c386\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vhi3DBqs5yrsYE6WXE3MyKzmBVD3NJEDQSpPAP4nbPOhb4amuvM5JLNbq0+XPOUt8FcNQZWjr95VsH5pkpiYgCZ\\\\\\\/TLY0iou3n6pjzc3jcakdC2j75TxOt\\\\\\\/QcUKbcdw6iakbzRQfIGofeDTQQJAsAi0bDgWTT2\\\\\\\/WFxub95dkkTLIlnh0+BySEYVMTBMONUNYBuok3qf372SQHWuDt+pyXmNQKQZW0smzre3PbqrTSptyoCgLB3xkKdYMiq3DJX+pqEQpl60nQYc9oXY6m6hF6RqLjd5L8SK+XpXLYxNIrzccOUJj0yNs7tsM7wi0T674dNfMVjbRxm7bMRI5R1PjDtsU2gX2hFxxvj4j1e3McUffoC6NQuSs5Fz9IiaIke43ReAvlisb1z7Joxm4nhJamtE8UUREE6LgNOkwxA\\\\\\\/82CjU+qNZH2KJwwTeB\\\\\\\/A0Z8e1Ov6WmSinIosM+bQv4K0Htjv\\\\\\\/\\\\\\\/VL+xDb+i8Lvo1IqdCMimGrnZ\\\\\\\/ZGm8ELvW+QoRdaAkrnSk+R7\\\\\\\/5qUtSusYX3OTnM5PZkjKXLGrvOUUDMimi\\\\\\\/MrmYIRXptrlJX35T6wg48ClVseLvqNCWyHvswDnmK\\\\\\\/dfDGyflKdpbFQtXNFrHQcZZR9Yyf6z6wg+5jkK5mBw7PcyOLTv7EhPtCkptx+4TF0BMeenBWa9Dnn6P1Gpwhxj998ioAAmUg3eA8I\\\\\\\/kaL6rCyM1PaX7uik6AKF5V6tq5LvJb6ch2o9im\\\\\\\/jHU4WYkkJtqwalx7aEaxfHGWVbk58v4U51cSh7SCRHpJ9SZREoBAB3tghXWqAH8E3l7mslm\\\\\\\/Bt0FN4wsyrKbA\\\\\\\/CeJoiy2iClIgg+SLhmA1AZ8oXmrxy3w328L8RaIxWLWcR5yKl6VxVTmiXkF1jqsoLLH20CWiCfrdPJVsmKwQn2X74XTDUSLx\\\\\\\/QALdN96f2siXbPrA4KRrCdbjl5zyRWpLXfl1xX8qj+fSuumkmtCBNc\\\\\\\/i8k\\\\\\\/3h2P57613UFvhzJW3HPR2KXgKFx1uMOw7weMkpj9pJUHCCJBFgr381k\\\\\\\/bT7FwXEak58KNhyAm8WjJ\\\\\\\/LebCosOT3\\\\\\\/eBJx12\\\\\\\/hrO0oczCZz83YcPEoo7Bq5KPQ317R+muLgNjxXJBK5T4VBJWIpPhxZSk9CWv6ZxihBF24EDTc8Ayik5ZJXY9wdtqTDfJjXHjOZROLbk9oBzu2DqpqxQSaM2VoPQle115RoidbheUq+ngLbFUxGn+oISLP2trajy2w9t6+l8boCn2mre8ir4NwJiI4Dao98pb7j7jMaX5OZk2OALZWuCqsSTjKtJhJ8y\\\\\\\/U4c4dN9Uo3aWwVqWUgfDRA37KpgsJR7wv6AMQxHdMLXgcriTQlesPYG3elUgJ87KEQ16CVu3MBkatizpT7a+L8kNuQD\\\\\\\/Mr3ptlmjwHpW81Wg1KLZ3BkGlctDOYBealhXf9rh7zvO7kxAtge5lVOH2mUnszC4MjV6w+COU4nqKgwoPN+QUrajeFpjCx1yjsB48MTbK5cfgGpVSZYSdJaa2+s3YoqRtjaRZ3\\\\\\\/muK5zjYxsWjBLsnL\\\\\\\/n+2rirDHVWagADGO+lJLY9WJvTw3jSRcvtGyEwGHSWnHn5I96\\\\\\\/m1N1Dh6Bqc78BRO2bHofg74zVsotZI9G1bwmwHma9zzDarj9r4KMuqEbLu8IAwM8Ahac6E4dwrSW4B6CmGyLI+ugu9kB9MKw2G\\\\\\\/KoDG0qm2sXjw6FVtAzVV\\\\\\\/wQUuonQGdIxDHS78NKKXbArQADsckg9QBM5QKUDkAMx1kMRGBKQY2V68jHX5Vu97VHaoSZlfAnighMUDUkS9o9oD3ygqS4wjhVlVzpgwcVvnDKio5QfXZx1TeqsPhG5CM1sEBj64r23qh9bSKdh0mGL86ew2XSb4XfnyVd4YSIoWsOxuK4RDzfEIr51lFuU\\\\\\\/Bvu9V8zwdTq9IkhlTdlctmHet3Wfhm+xvjEZ8lMhTk6u06oDcVZJmc2i4I+IbZDrEDvfvhveA+kJdmR12j6HzhG7X8F7HCM6LJrOQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qMHx3MLjvhySAAMas7qZ2g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227ae8c5dafea989807401e200ce5c1bc56b8830085c0d2cd7e0fbe726c497e093\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488296 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mutant-screen-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/401" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AB0mep31pdQ105ifevmYnglPBUZn1yRCpgT33wnoL20" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/401" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d_d-PL1pRe5E0snu8qagQ18uf9dIL5AqhtoDOr3hJOo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/393" title="Reference Quality Genome Assemblies of Three Parastagonospora nodorum Isolates Differing in Virulence on Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/411" title="Dissecting the Genetic Architecture of Shoot Growth in Carrot (Daucus carota L.) Using a Diallel Mating Design" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488296" id="node488296" data-pisa="ggg;8/2/401" data-pisa-master="ggg;g3.117.300292" data-apath="/ggg/8/2/401.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Krista A. Geister</span>, <span class="highwire-citation-author" data-delta="1">Andrew E. Timms</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0325-8520" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David R. Beier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">401-409;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300292 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488296" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Krista A. Geister </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics, University of Washington School of Medicine, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krista%2BA.%2BGeister%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geister%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrista%2BA.%2BGeister%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew E. Timms </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BE.%2BTimms%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Timms%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BE.%2BTimms%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David R. Beier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics, University of Washington School of Medicine, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BR.%2BBeier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beier%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BR.%2BBeier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0325-8520" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David R. Beier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.beier@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">david.beier@seattlechildrens.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/401" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488296" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/401.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488296" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/401.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488296" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/401.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488296" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/401.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488296" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Positional cloning of ENU-induced mutations has traditionally relied on analysis of polymorphic variation between two strains. In contrast, the application of whole-genome sequencing (WGS) has enabled gene discovery in mutant lines maintained on an inbred genetic background. This approach utilizes genetic variation derived from ENU-induced variants for mapping and reduces the likelihood of phenotypic variation, making it an ideal method for genetic modifier screening. Here, we describe the results of such a screen, wherein we determined the minimal number of mutant genomic DNA samples to include in our analyses and improved the sensitivity of our screen by individually barcoding each genomic DNA library. We present several unique cases to illustrate this approach’s efficacy, including the discovery of two distinct mutations that generate essentially identical mutant phenotypes, the ascertainment of a non-ENU-induced candidate variant through homozygosity mapping, and an approach for the identification of putative dominant genetic modifiers.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mus-musculus" class="hw-term hw-article-keyword hw-article-keyword-mus-musculus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mus musculus</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/colgalt1" class="hw-term hw-article-keyword hw-article-keyword-colgalt1" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Colgalt1</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/plod3" class="hw-term hw-article-keyword hw-article-keyword-plod3" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Plod3</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/kif20b" class="hw-term hw-article-keyword hw-article-keyword-kif20b" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Kif20b</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/tgds" class="hw-term hw-article-keyword hw-article-keyword-tgds" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Tgds</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/fbn2" class="hw-term hw-article-keyword hw-article-keyword-fbn2" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Fbn2</em></a></li><li class="kwd"><a href="/keyword/mutant-screen-report" class="hw-term hw-article-keyword hw-article-keyword-mutant-screen-report" rel="nofollow">Mutant screen report</a></li></ul><p id="p-3">Historically, mutagenesis screens in the mouse have relied on polymorphic variation between strains to facilitate homozygosity mapping. This strategy requires an outcross, which may involve a great deal of time, breeding, and effort. After mapping the locus, one may need to rely on beneficial recombination in subsequent affected individuals to narrow the interval, while searching for ENU-induced mutations by individually sequencing exons within the locus. This method also does not lend itself particularly well to genetic modifier screening, as the polymorphic variation from the outcross strain may add additional complexity to both the phenotype and the genetic analysis. Our laboratory (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gallego-Llamas <em>et al.</em> 2015</a>) and others (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bull <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Simon <em>et al.</em> 2015</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Caruana <em>et al.</em> 2013</a>) have developed strategies that use next-generation sequencing technologies to accelerate homozygosity mapping, variant calling, and confirmation on an inbred background. Doing so minimizes phenotypic variation and limits genetic variation to ENU-induced variants. We sought to optimize this approach for the discovery of genetic modifiers of developmental phenotypes.</p><p id="p-4">The traditional outcross has been similarly optimized to provide fine mapping of the locus with a minimal number of markers (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Moran <em>et al.</em> 2006</a>). ENU induces roughly 3000 heritable variants per genome (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Quwailid <em>et al.</em> 2004</a>); ascertainment of even half that number of variants is more than enough to map a causal locus (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Moran <em>et al.</em> 2006</a>). Our bioinformatics analysis of mutagenized inbred lines involves dividing the genome into windows of a specified size and calculating the following for each window: the number of homozygous SNPs, percentage of homozygous SNPs, and the percentage of nonreference alleles, which we call the novel allele frequency (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Gallego-Llamas <em>et al.</em> 2015</a>). Loci with a high number and percentage of homozygous SNPs and a high novel allele frequency represent regions that contain a concentration of homozygous ENU-induced variants. It is worth noting that the variants used to map the locus will likely include the causal mutation. We focused our analysis on variants that occur in coding regions or splice sites and conducted Sanger sequencing on additional affected embryonic DNA samples to identify the causal variant.</p><p id="p-5">Our initial attempt to map and ascertain ENU-induced variants using this method was successful and yielded valuable insights (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Gallego-Llamas <em>et al.</em> 2015</a>). First, we found that WGS provided better mapping resolution than whole-exome sequencing. Second, pooling samples without individually barcoding them can complicate the analysis. We conducted WGS on individually barcoded sample libraries in all the work described in this report. This led to some additional insights regarding the sensitivity, cost-effectiveness, and power of our approach. We discovered five mutant alleles: two of which have not been previously described, and three that are remutations of previously characterized mutant loci (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Rautavuoma <em>et al.</em> 2004</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ruotsalainen <em>et al.</em> 2006</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dwyer <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Johnson <em>et al.</em> 1998</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arteaga-Solis <em>et al.</em> 2001</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Browning <em>et al.</em> 2001</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chaudhry <em>et al.</em> 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Miller <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Shi <em>et al.</em> 2013</a>). We have successfully mapped loci using as few as two mutant samples, we have learned that this technique is sensitive enough to identify two independent mutations that cause what was thought to be one mutant phenotype, and we have used this method to clone a non-ENU-induced mutant locus. Additionally, this strategy can be used to query putative modifying loci regardless of inheritance pattern.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Generation of mutant mice</h3><p id="p-6">All animal work was reviewed and approved by the Seattle Children’s Research Institute’s (SCRI) Institutional Care and Use Committee. ENU mutagenesis was carried out by George Carlson’s laboratory at the McLaughlin Research Institute (Great Falls, MT) as previously described (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Gallego-Llamas <em>et al.</em> 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Stottmann and Beier 2014</a>). G1 male mice were shipped to Seattle Children’s Research Institute and bred to C57BL/6J females. G2 daughters were backcrossed to the G1 male and were checked daily for the presence of a copulation plug. Embryos were screened at either E17.5 or E18.5 for mutant phenotypes.</p></div><div id="sec-3" class="subsection"><h3>Genomic DNA isolation</h3><p id="p-7">Genomic DNA for WGS was extracted from mutant embryonic liver samples using the QIAGEN DNeasy Blood and Tissue Kit (#69504) according to the manufacturer’s instructions, but with some modifications. Approximately 10 mg of embryonic liver was used, and was snipped into very small pieces in individual plastic weigh boats with fine dissecting scissors. Lysis buffer was used to collect all of the tissue from the weigh boat. Samples were digested with the Proteinase K provided in the kit (20 mg/ml) for 1 hr, with vortexing every 10 min. RNA was digested with 8 μl of RNase A (QIAGEN #19101, 100 mg/ml at 7000 units/ml) for 2 min. Two elutions were conducted into two separate microfuge tubes, with elution buffer incubated on the column for 5 min before centrifugation. DNA was tested at SCRI for purity and quality using a Nanodrop spectrophotometer and the Qubit dsDNA Broad Range Assay (#Q32850; Thermo Fisher Scientific). DNA samples were shipped to HudsonAlpha Genomic Services Laboratory (Huntsville, AL) or NovoGene (Chula Vista, CA), for WGS.</p></div><div id="sec-4" class="subsection"><h3>Homozyosity mapping</h3><p id="p-8">Mapping, variant calling, and homozygosity mapping were performed as in our previous publication (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Gallego-Llamas <em>et al.</em> 2015</a>). In summary, we divided the genome in windows and identified high-quality ENU SNPs. We subsequently counted the number of homozygous variants, percentage of homozygous variants, and average nonreference allele frequencies for each window. To identify potential casual variants, we identified all coding and splice site (within 10 bp of an exon) homozygous mutations of good quality.</p></div><div id="sec-5" class="subsection"><h3>Heterozygous modifier identification</h3><p id="p-9">We used two approaches to identify ENU-induced dominant modifiers, either using individual mice or a combination of all reads for a phenotype. For individual mice, we identified ENU SNPs by eliminating from further consideration those in dbSNP, Sanger mouse strains, in-house pedigrees, or repeat regions. We then identified SNPs that had total coverage of five reads or greater per individual sample, and had either a heterozygous or homozygous mutant genotype in all mice analyzed (five individuals for cleft palate and three individuals for exencephaly). For the combined sets, we identified high-quality ENU SNPs by; removing SNPS if they were in dbSNP, a Sanger mouse strain, an in-house unrelated pedigree, or a repeat region. We also required SNPs to have a quality score of ≥ 30 and coverage of five or more reads. We then removed SNPs if in other “<em>fosse</em>” types, <em>i.e.</em>, exencephaly or cleft palate, and then counted the number of SNPs per sliding window (10 Mb with a 1 Mb overlap).</p></div><div id="sec-6" class="subsection"><h3>Validation of candidate variants</h3><p id="p-10">Primers were designed to amplify regions containing our variants of interest using Primer-BLAST (NCBI, Bethesda, MD). We conducted Sanger sequencing on these PCR products amplified from additional affected embryonic DNA samples (Eurofins Genomics, Louisville, KY). Sanger sequencing data were analyzed using FinchTV (Geospiza, Seattle, WA), 4Peaks (Nucleobytes, Aalsmer, The Netherlands), Sequencher 5.0 (Gene Codes Corp., Ann Arbor, MI), and/or the Lasergene 12 Suite (DNAStar, Madison, WI).</p></div><div id="sec-7" class="subsection"><h3>Skeletal preparations</h3><p id="p-11">Skeletal preparations were performed as previously described (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ovchinnikov 2009</a>).</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-12">Strains are available upon request. Sequence data are available in the BioProject database via BioProject ID: PRJNA412014. Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a> contains code used for homozygosity mapping.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Mutant phenotypes obtained and sequencing analysis</h3><p id="p-13">We selected the C57BL/6J background for our inbred screen and carried out a standard G2 backcross breeding scheme as previously described (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Gallego-Llamas <em>et al.</em> 2015</a>). In addition, the founders of these mutant lines were offspring of our previously described sequential dosage pipeline (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gallego-Llamas <em>et al.</em> 2016</a>), which resulted in many of our founder males belonging to one of several pedigrees. Mutant lines where multiple affected embryos were observed in more than one litter were selected for further study. Five mutant phenotypes were discovered (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Together, they represent a collection with craniofacial, eye, and digital anomalies. Embryonic liver served as the source of genomic DNA for WGS, while liver and tail were used for other downstream applications, including PCR amplification followed by Sanger sequencing. Once more than eight embryos with these phenotypes had been collected, their genomic DNA was extracted and the samples with the highest yield and best A<sub>260</sub>:A<sub>280</sub> ratio were selected for WGS, followed by homozygosity mapping to map ENU-induced homozygous loci and variant calling to determine likely candidate variants.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/401/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutant phenotypes discovered by ENU mutagenesis. (A) The fosse phenotype exhibits bent wrists, eye defects, skeletal defects, and variable incidence of cleft palate and exencephaly. (B) The ugli phenotype closely resembles the fosse phenotype, even with regard to cleft palate and exencephaly. (C) The daredevil phenotype is characterized by bilateral anophthalmia, growth insufficiency, and cleft palate. (D) bub mutant embryos display severe micrognathia and cleft palate. (E) Bent wrists, micrognathia, eye and ear defects, and hindlimb syndactyly are hallmarks of the timon mutant phenotype. ENU, N-ethyl-N-nitrosourea." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-664544181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mutant phenotypes discovered by ENU mutagenesis. (A) The &lt;em&gt;fosse&lt;/em&gt; phenotype exhibits bent wrists, eye defects, skeletal defects, and variable incidence of cleft palate and exencephaly. (B) The &lt;em&gt;ugli&lt;/em&gt; phenotype closely resembles the &lt;em&gt;fosse&lt;/em&gt; phenotype, even with regard to cleft palate and exencephaly. (C) The &lt;em&gt;daredevil&lt;/em&gt; phenotype is characterized by bilateral anophthalmia, growth insufficiency, and cleft palate. (D) &lt;em&gt;bub&lt;/em&gt; mutant embryos display severe micrognathia and cleft palate. (E) Bent wrists, micrognathia, eye and ear defects, and hindlimb syndactyly are hallmarks of the &lt;em&gt;timon&lt;/em&gt; mutant phenotype. ENU, N-ethyl-N-nitrosourea.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/401/F1.medium.gif" width="414" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/401/F1.medium.gif" width="414" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/401/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/401/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488441" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Mutant phenotypes discovered by ENU mutagenesis. (A) The <em>fosse</em> phenotype exhibits bent wrists, eye defects, skeletal defects, and variable incidence of cleft palate and exencephaly. (B) The <em>ugli</em> phenotype closely resembles the <em>fosse</em> phenotype, even with regard to cleft palate and exencephaly. (C) The <em>daredevil</em> phenotype is characterized by bilateral anophthalmia, growth insufficiency, and cleft palate. (D) <em>bub</em> mutant embryos display severe micrognathia and cleft palate. (E) Bent wrists, micrognathia, eye and ear defects, and hindlimb syndactyly are hallmarks of the <em>timon</em> mutant phenotype. ENU, N-ethyl-N-nitrosourea.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Homozygosity mapping can distinguish two mutations that cause nearly identical phenotypes</h3><p id="p-15">Two of the mutant phenotypes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, A and B</a>, <em>fosse</em> and <em>ugli</em>), were impossible to distinguish from one another prior to sequencing analysis. Three founder males from the same ENU pedigree produced embryonic progeny with these phenotypes. Due to their shared lineage, we initially presumed that this phenotype was caused by a single mutation. Embryos exhibited bent wrists, open eyelids, eye defects, edema, short rib cages, and growth insufficiency. We named this presumed single mutant phenotype <em>fosse</em>. A total of eight genomic DNA samples from <em>fosse</em> embryos from two of the three founder males were used for library preparation and WGS analysis. All samples were individually barcoded to allow us to separate each individual library’s results from the others, if necessary.</p><p id="p-16">We were unable to map the <em>fosse</em> locus when all eight samples were included in our sequencing analysis. Multiple iterations of our analysis with different combinations of samples were conducted, which ultimately revealed that the presumptive <em>fosse</em> phenotype was, in fact, two nearly identical mutant phenotypes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, A and B</a>) caused by two independent mutations (Tables S1 and S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>). Six of our <em>fosse</em> samples had a common homozygous region on mouse chromosome 8 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), and the two remaining samples shared several regions of homozygosity on mouse chromosomes 4, 5, and 18 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). We chose to name the phenotype of these two outliers <em>ugli</em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/401/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Homozygosity mapping of fosse and ugli reveals distinct causal genetic loci associated with these two highly similar phenotypes. (A) Our homozygosity mapping of six affected individuals indicates that the fosse causal variant is located on chromosome 8. (B) Our remaining two samples&#x2019; combined homozygosity mapping results highlighted homozygous regions of interest on chromosomes 4, 5, and 18. Both maps were generated using windows 10 Mb in size with a 1 Mb overlap to calculate all three variables shown. Average naf indicates the average frequency of the nonreference allele in all putative ENU SNPs within a window. Hom count refers to the number of homozygous SNPs within each window, whereas hom percentage is the percentage of SNPs that were called homozygous. Regions of interest should have high average naf (likely ENU-induced), and high homozygous counts and percentages. ENU, N-ethyl-N-nitrosourea; hom, homozygous; naf, novel allele frequency; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-664544181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Homozygosity mapping of &lt;em&gt;fosse&lt;/em&gt; and &lt;em&gt;ugli&lt;/em&gt; reveals distinct causal genetic loci associated with these two highly similar phenotypes. (A) Our homozygosity mapping of six affected individuals indicates that the &lt;em&gt;fosse&lt;/em&gt; causal variant is located on chromosome 8. (B) Our remaining two samples&#x2019; combined homozygosity mapping results highlighted homozygous regions of interest on chromosomes 4, 5, and 18. Both maps were generated using windows 10 Mb in size with a 1 Mb overlap to calculate all three variables shown. Average naf indicates the average frequency of the nonreference allele in all putative ENU SNPs within a window. Hom count refers to the number of homozygous SNPs within each window, whereas hom percentage is the percentage of SNPs that were called homozygous. Regions of interest should have high average naf (likely ENU-induced), and high homozygous counts and percentages. ENU, N-ethyl-N-nitrosourea; hom, homozygous; naf, novel allele frequency; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/401/F2.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/401/F2.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/401/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/401/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488497" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Homozygosity mapping of <em>fosse</em> and <em>ugli</em> reveals distinct causal genetic loci associated with these two highly similar phenotypes. (A) Our homozygosity mapping of six affected individuals indicates that the <em>fosse</em> causal variant is located on chromosome 8. (B) Our remaining two samples’ combined homozygosity mapping results highlighted homozygous regions of interest on chromosomes 4, 5, and 18. Both maps were generated using windows 10 Mb in size with a 1 Mb overlap to calculate all three variables shown. Average naf indicates the average frequency of the nonreference allele in all putative ENU SNPs within a window. Hom count refers to the number of homozygous SNPs within each window, whereas hom percentage is the percentage of SNPs that were called homozygous. Regions of interest should have high average naf (likely ENU-induced), and high homozygous counts and percentages. ENU, N-ethyl-N-nitrosourea; hom, homozygous; naf, novel allele frequency; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">After our mapping analysis, we restricted our focus to variants in all homozygous regions identified that were predicted to occur in coding regions and near splice sites. This generated the list of variants in Tables S1 and S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>. While it is possible that a causal mutation will occur in nontranscribed sequences, there is presently only one report documenting an ENU-induced mutation occurring in a regulatory region (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Sagai <em>et al.</em> 2004</a>) and a single report describing an ENU-induced mutation in a microRNA seed region (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Lewis <em>et al.</em> 2009</a>). There may be bias in the conclusion that exons and their flanking regions are likely to contain the mutated nucleotide, as nontranscribed regions are often not examined in detail. However, in the characterization of ENU mutants mapped to loci where known candidates reside, causal sequence variants can usually be found in coding regions or splice sites [<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hart <em>et al.</em> (2005)</a>].</p><p id="p-19">Analysis of the homozygous regions in the two <em>ugli</em> mutants identified a missense mutation in <em>Plod3</em> as the only coding/splice site variant. Additional genotyping confirmed that the variant in <em>Plod3</em> was homozygous in all affected embryos (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>). Several variants on chromosome 8 were analyzed to determine the <em>fosse</em> candidate mutation (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>). All mutant embryos were homozygous for variants in <em>Colgalt1</em> and <em>Jak3</em>, which are 65 kb apart. However, the <em>Jak3</em> null phenotype is compatible with postnatal life (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Park <em>et al.</em> 1995</a>) and the <em>fosse</em> mutant phenotype is embryonic lethal.</p><p id="p-20">This leads us to conclude that the <em>fosse</em> phenotype is caused by the missense variant in <em>Colgalt1</em>, and that the <em>ugli</em> phenotype is caused by the missense variant in <em>Plod3</em>; this is supported by the functional relatedness of these two genes. <em>Colgalt1</em> and <em>Plod3</em> both encode enzymes involved in the post-translational modification of collagens [reviewed in <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Yamauchi and Sricholpech (2012)</a>]. <em>Colgalt1</em> is a galactosyltransferase and has been shown to act <em>in vitro</em> on hydroxylysines in collagen types I–V (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Schegg <em>et al.</em> 2009</a>). A <em>Colgalt1</em> null mouse phenotype has not been described, but there are <em>Plod3</em> mutant mouse lines with an array of phenotypes (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Rautavuoma <em>et al.</em> 2004</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Ruotsalainen <em>et al.</em> 2006</a>). <em>Plod3</em> can act as a lysyl hydroxylase and a galactosylhydroxylysine glucosyltransferase (GGT) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Heikkinen <em>et al.</em> 2000</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wang <em>et al.</em> 2002a</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">b</a>). The GGT activity of <em>Plod3</em> is critical for collagen type IV deposition and function <em>in vivo</em> (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Rautavuoma <em>et al.</em> 2004</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Ruotsalainen <em>et al.</em> 2006</a>). Most importantly, <em>Plod3</em> mutations that result in either complete loss of function or loss of GGT activity result in embryonic lethality (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Rautavuoma <em>et al.</em> 2004</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Ruotsalainen <em>et al.</em> 2006</a>). Our <em>Plod3</em><em><sup>ugli/ugli</sup></em> embryos die in either late gestation or neonatally. Mutations in <em>PLOD3</em> have been documented in humans with lysyl hydroxylase 3 deficiency (bone fragility with contractures, arterial rupture, and deafness) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Salo <em>et al.</em> 2008</a>). The similarities in phenotype between our <em>Plod3</em><em><sup>ugli/ugli</sup></em> mutant and previously described <em>Plod3</em> mutant phenotypes (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Rautavuoma <em>et al.</em> 2004</a>; <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Ruotsalainen <em>et al.</em> 2006</a>), as well as the nearly identical phenotypes observed in <em>Colgalt1</em><em><sup>fosse/fosse</sup></em> and <em>Plod3</em><em><sup>ugli/ugli</sup></em> embryos, provide support that we have identified the candidate mutations in both cases.</p></div><div id="sec-12" class="subsection"><h3>Identification of two additional candidate ENU-induced variants from WGS of three individually barcoded samples</h3><p id="p-21">Given the time and numbers of mice required to discover mutant strains, as well as the cost of WGS, any strategies that can accelerate and ensure the identification of the causal variant are invaluable. We included a total of six <em>Colgalt1</em><em><sup>fosse/fosse</sup></em> samples and two <em>Plod3</em><em><sup>ugli/ugli</sup></em> samples in our WGS analysis, and successfully identified the candidate variant in each case. This led us to test <em>in silico</em> and in actuality how many samples are required to ensure successful identification of a causative variant. Using our data from the <em>Colgalt1</em><em><sup>fosse/fosse</sup></em> WGS analysis, we randomly combined different samples and different numbers of samples, and were able to generate homozygosity maps that matched our <em>Colgalt1</em><em><sup>fosse/fosse</sup></em> mapping data with only three samples (data not shown). Importantly, this number also ensured that we had enough coverage from each individual’s library to call a true variant.</p><p id="p-22">To test this in practice, we included three samples from two mutant lines that we ascertained, which we named <em>daredevil</em> and <em>bub</em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, C and D</a>). Both mutants exhibit cleft palate and a number of other phenotypic abnormalities. Our homozygosity mapping from both of these mutants determined that <em>daredevil</em> mapped to chromosome 19 and <em>bub</em> mapped to chromosome 14 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>). There were only small numbers of coding region/splice sites variants in these regions (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>), and we confirmed the genetic causes for both mutants by genotyping additional affected embryos (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>). The <em>bub</em> phenotype is due to a variant in <em>Tgds</em>, and <em>daredevil</em> is caused by a mutation in <em>Kif20b</em>. The <em>Kif20b</em><em><sup>daredevil/daredevil</sup></em> mutant phenocopies the <em>Kif20b</em><em><sup>magoo/magoo</sup></em> mutant (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Dwyer <em>et al.</em> 2011</a>). <em>TGDS</em> mutations have been identified in human patients with Catel–Mantzke syndrome, which causes cleft palate and micrognathia (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ehmke <em>et al.</em> 2014</a>), phenotypic hallmarks of the bub <em>Tgds</em><em><sup>bub/bub</sup></em> mutant phenotype. These observations support our conclusion that both of these variants are likely candidates for each of these mutant phenotypes, and illustrate that we are able to reliably identify these variants using only three genomic DNA samples.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/401/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Homozygosity mapping of daredevil and bub uncovers causal loci on chromosomes 19 and 14. (A) A single homozygous nonreference peak exists for daredevil on chromosome 19. (B) Likewise, the bub homozygosity map indicates that the causal locus is on chromosome 14. Both maps were generated using 10 Mb windows with a 1 Mb overlap." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-664544181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Homozygosity mapping of &lt;em&gt;daredevil&lt;/em&gt; and &lt;em&gt;bub&lt;/em&gt; uncovers causal loci on chromosomes 19 and 14. (A) A single homozygous nonreference peak exists for &lt;em&gt;daredevil&lt;/em&gt; on chromosome 19. (B) Likewise, the &lt;em&gt;bub&lt;/em&gt; homozygosity map indicates that the causal locus is on chromosome 14. Both maps were generated using 10 Mb windows with a 1 Mb overlap.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/401/F3.medium.gif" width="440" height="195"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/2/401/F3.medium.gif" width="440" height="195"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/401/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/401/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488541" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Homozygosity mapping of <em>daredevil</em> and <em>bub</em> uncovers causal loci on chromosomes 19 and 14. (A) A single homozygous nonreference peak exists for <em>daredevil</em> on chromosome 19. (B) Likewise, the <em>bub</em> homozygosity map indicates that the causal locus is on chromosome 14. Both maps were generated using 10 Mb windows with a 1 Mb overlap.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Identification of a non-ENU-induced candidate mutation through homozygosity mapping and individual sequencing read analysis</h3><p id="p-24"><em>Tdgs</em><sup>bub</sup> and <em>Kif20b</em><em><sup>daredevil</sup></em> illustrate our ability to map and ascertain causal mutations from only three samples. Our analysis of one of our mutants (<em>timon</em>, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1E</a>), led to the identification of a non-ENU-induced candidate variant. The pedigree of this line was rather complex, given that the G1 male and his only G2 male offspring had reduced fertility (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). To secure the line, progeny testing from brother–sister matings was successfully utilized to identify obligate carriers, and the carrier males obtained were fertile. Three affected individuals’ genomic DNA samples were sent for WGS (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>, arrows). However, the homozygosity map that we generated from all three individuals’ data did not yield a clear region of interest. Analysis of a map generated from two of the more closely related individuals’ data (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>, asterisks) identified two homozygous regions, one on chromosome 7 and one on chromosome 18 (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>). Genotyping of additional affected embryos across these two intervals allowed us to exclude the locus on chromosome 7, and indicated that the causal mutation was in the distal portion of the interval on chromosome 18 (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>). Importantly, examination of the third sequenced mutant confirmed that it also carried homozygous variants on distal chromosome 18. However, a recombination event occurred in the region that resulted in an interval in all three affected mice that was smaller than the window used for homozygosity mapping. While this initially confounded our mapping analysis, it ultimately facilitated the localization of the likely candidate mutation. In fact, reduction of window size uncovered the homozygous region; however, this was at the expense of marker numbers (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/401/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pedigree and homozygosity mapping of timon indicates autosomal recessive inheritance and a causal locus on mouse chromosome 18. (A) The timon pedigree indicates two distinct branches: one that is more closely related to the G1 and one that is farther removed. Two individuals (gray diamonds) exhibited more severe craniofacial and limb phenotypes, and were excluded as affected by genotyping. Therefore, these two embryos represent a mutant phenotype distinct from timon. Arrows, genomic DNA samples selected for WGS. Asterisks: two individuals used to map the critical interval in (B). (B) The timon causal locus was mapped to chromosome 18 using WGS data from two of the three samples submitted. A peak on chromosome 7 was eliminated by genotyping affected individuals across the critical interval for candidate ENU-induced variants (Table S4 in File S1). (C) Reducing the window and overlap size to 50 kb in all three individuals&#x2019; mapping analysis revealed that the third sample also possessed the homozygous peak on chromosome 18. ENU, N-ethyl-N-nitrosourea; WGS, whole-genome sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-664544181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pedigree and homozygosity mapping of &lt;em&gt;timon&lt;/em&gt; indicates autosomal recessive inheritance and a causal locus on mouse chromosome 18. (A) The &lt;em&gt;timon&lt;/em&gt; pedigree indicates two distinct branches: one that is more closely related to the G1 and one that is farther removed. Two individuals (gray diamonds) exhibited more severe craniofacial and limb phenotypes, and were excluded as affected by genotyping. Therefore, these two embryos represent a mutant phenotype distinct from &lt;em&gt;timon&lt;/em&gt;. Arrows, genomic DNA samples selected for WGS. Asterisks: two individuals used to map the critical interval in (B). (B) The &lt;em&gt;timon&lt;/em&gt; causal locus was mapped to chromosome 18 using WGS data from two of the three samples submitted. A peak on chromosome 7 was eliminated by genotyping affected individuals across the critical interval for candidate ENU-induced variants (Table S4 in File S1). (C) Reducing the window and overlap size to 50 kb in all three individuals&#x2019; mapping analysis revealed that the third sample also possessed the homozygous peak on chromosome 18. ENU, N-ethyl-N-nitrosourea; WGS, whole-genome sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/401/F4.medium.gif" width="397" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/2/401/F4.medium.gif" width="397" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/401/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/401/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488473" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Pedigree and homozygosity mapping of <em>timon</em> indicates autosomal recessive inheritance and a causal locus on mouse chromosome 18. (A) The <em>timon</em> pedigree indicates two distinct branches: one that is more closely related to the G1 and one that is farther removed. Two individuals (gray diamonds) exhibited more severe craniofacial and limb phenotypes, and were excluded as affected by genotyping. Therefore, these two embryos represent a mutant phenotype distinct from <em>timon</em>. Arrows, genomic DNA samples selected for WGS. Asterisks: two individuals used to map the critical interval in (B). (B) The <em>timon</em> causal locus was mapped to chromosome 18 using WGS data from two of the three samples submitted. A peak on chromosome 7 was eliminated by genotyping affected individuals across the critical interval for candidate ENU-induced variants (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>). (C) Reducing the window and overlap size to 50 kb in all three individuals’ mapping analysis revealed that the third sample also possessed the homozygous peak on chromosome 18. ENU, N-ethyl-N-nitrosourea; WGS, whole-genome sequencing.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Of the ENU variants in the recombinant interval, both coding and noncoding, none appeared to explain the mutant phenotype. We tested the effect of an intronic variant in <em>Pdgfrb</em> on the mRNA transcript via RT-PCR followed by Sanger sequencing, but the transcript sequence was normal (data not shown). This suggested the possibility that the causal variant was not induced by ENU, but was a spontaneous mutation of some kind.</p><p id="p-27">To identify a larger insertion or deletion, we identified all reads within the recombination region that had been trimmed, <em>i.e.</em>, the full read did not map at the same location. We examined all positions where we saw more than five reads with the same trimmed location. After comparing with C57BL/6J in the Integrative Genomics Viewer (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Robinson <em>et al.</em> 2011</a>), we identified a region in all three <em>timon</em> samples with split reads (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> and Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1/FileS1">File S1</a>), indicating that the region is deleted in the mutant genome. The <em>timon</em> deletion (chromosome 18: 58,012,626–58,014,322) is ∼1.7 kb in length, and includes exon 63 of the 65 total exons in the gene <em>Fbn2</em>, which encodes fibrillin 2. RT-PCR followed by gel electrophoresis (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>) and Sanger sequencing (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>) of mutant cDNA from exons 62–64 of <em>Fbn2</em> revealed improper splicing of exon 62 into exon 64. The loss of exon 63 in the mutant transcript results in a premature stop codon at the end of exon 64. Other mutant phenotypes due to mutations in <em>Fbn2</em> have been described, and many have the same anomalies as our <em>timon</em> mutant: hindlimb syndactyly, flexure of the wrists, and eye malformations (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Johnson <em>et al.</em> 1998</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Arteaga-Solis <em>et al.</em> 2001</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Browning <em>et al.</em> 2001</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chaudhry <em>et al.</em> 2001</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Miller <em>et al.</em> 2010</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Shi <em>et al.</em> 2013</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/401/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The timon mutant phenotype is due to deletion of exon 63 in Fbn2. (A) After variant calling and genotyping for individual ENU-induced variants within the interval, analysis of split reads uncovered a 1696 bp deletion that encompasses exon 63 of Fbn2. (B) RT-PCR of Fbn2 transcripts using primers from exon 62&#x2013;64 reveals a Mut product of 196 bp instead of 428 bp, as seen in WT and Het samples. The band in the RT reaction coincides with a band of similar size in the genomic DNA reaction (G), so this band likely reflects the presence of trace amounts of genomic DNA in our RT reactions. (C) Sequencing of RT-PCR products from WT and homozygous mutant samples demonstrate the complete loss of exon 63 sequence from the transcript. This results in frameshift and stop codons in exon 64. ENU, N-ethyl-N-nitrosourea; Het, heterozygous; Mut, mutant; NT, no template control; RT-PCR, reverse transcriptase polymerase chain reaction; WT, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-664544181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;timon&lt;/em&gt; mutant phenotype is due to deletion of exon 63 in &lt;em&gt;Fbn2&lt;/em&gt;. (A) After variant calling and genotyping for individual ENU-induced variants within the interval, analysis of split reads uncovered a 1696 bp deletion that encompasses exon 63 of &lt;em&gt;Fbn2&lt;/em&gt;. (B) RT-PCR of &lt;em&gt;Fbn2&lt;/em&gt; transcripts using primers from exon 62&#x2013;64 reveals a Mut product of 196 bp instead of 428 bp, as seen in WT and Het samples. The band in the RT reaction coincides with a band of similar size in the genomic DNA reaction (G), so this band likely reflects the presence of trace amounts of genomic DNA in our RT reactions. (C) Sequencing of RT-PCR products from WT and homozygous mutant samples demonstrate the complete loss of exon 63 sequence from the transcript. This results in frameshift and stop codons in exon 64. ENU, N-ethyl-N-nitrosourea; Het, heterozygous; Mut, mutant; NT, no template control; RT-PCR, reverse transcriptase polymerase chain reaction; WT, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/401/F5.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/2/401/F5.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/401/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/401/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488405" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">The <em>timon</em> mutant phenotype is due to deletion of exon 63 in <em>Fbn2</em>. (A) After variant calling and genotyping for individual ENU-induced variants within the interval, analysis of split reads uncovered a 1696 bp deletion that encompasses exon 63 of <em>Fbn2</em>. (B) RT-PCR of <em>Fbn2</em> transcripts using primers from exon 62–64 reveals a Mut product of 196 bp instead of 428 bp, as seen in WT and Het samples. The band in the RT reaction coincides with a band of similar size in the genomic DNA reaction (G), so this band likely reflects the presence of trace amounts of genomic DNA in our RT reactions. (C) Sequencing of RT-PCR products from WT and homozygous mutant samples demonstrate the complete loss of exon 63 sequence from the transcript. This results in frameshift and stop codons in exon 64. ENU, N-ethyl-N-nitrosourea; Het, heterozygous; Mut, mutant; NT, no template control; RT-PCR, reverse transcriptase polymerase chain reaction; WT, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">This is an interesting case where ENU-induced variants around the deletion served as genetic markers that allowed us to map the candidate locus and mutation. For future analyses, it is worth keeping in mind that mutations ascertained in an ENU mutagenesis screen may not be ENU-induced. By conducting WGS, one is able to readily probe the entire genome for these types of mutations, should the need arise, as it did with <em>Fbn2</em><em><sup>timon</sup></em>.</p></div><div id="sec-14" class="subsection"><h3>Inbred screening strategy can be used to query for dominant-acting modifying loci</h3><p id="p-30">In addition to the phenotypic features listed above, 18% of <em>fosse</em> embryos presented with cleft palate and 26% presented with exencephaly. These phenotypes were never found in unaffected siblings. A χ<sup>2</sup> analysis indicated that this result was highly significant, demonstrating that these features were contingent on homozygosity for the <em>fosse</em> allele and were not due to independently acting mutations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). As previously stated, our inbred screening strategy was designed with the intent that it could be used to identify genetic modifiers, so we attempted to assess whether the variable expressivity of cleft palate and exencephaly in <em>Colgalt1</em><em><sup>fosse/fosse</sup></em> mutants was due to genetic interactions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488366/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488366/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">χ<sup>2</sup> analysis of cleft palate and exencephaly indicates that both phenotypes are contingent with the <em>fosse</em> mutant phenotype</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Our initial WGS analysis of <em>fosse</em> mutants included five embryos with cleft palate and one with exencephaly. For the modifier analysis, we added two additional <em>fosse</em> embryos with exencephaly. For both cases, we identified only a single region of homozygosity (corresponding to the <em>Colgalt1</em> locus), excluding the possibility that these phenotypes were due to unlinked recessive-acting modifying mutations.</p><p id="p-33">Therefore, we decided to test whether cleft palate may be due to a dominant-acting modifier that, when combined with homozygosity for the <em>Colgalt1</em><sup>fosse</sup> variant, would generate an embryo with cleft palate or exencephaly. We first focused our analysis on heterozygous ENU-induced variants that were shared between the five cleft palate samples, but were not present in the heterozygous state in our three samples with exencephaly. This resulted in the identification of 22 variants to consider for validation. After genotyping an additional four samples with cleft palate for all of these variants, as well as multiple fosse embryos without cleft palate, we concluded that cleft palate does not appear to be due to a unique heterozygous variant in our <em>Colgalt1<sup>fosse/fosse</sup></em> embryos (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a> and data not shown).</p><p id="p-34">For the exencephaly phenotype, we identified 10 dominant modifier candidates, and genotyped an additional four individuals with and without exencephaly to determine if any of them were causal. None of these variants were associated with exencephaly (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a> and data not shown). These data demonstrate that the exencephaly phenotype in <em>Colgalt1</em><em><sup>fosse/fosse</sup></em> embryos, like cleft palate, is probably due to stochastic processes during embryonic development and not to heritable genetic interactions.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/401/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Filtering of variants by phenotype generates a map of possible modifying loci contributing to cleft palate or exencephaly. (A) Number of SNPs in each 10 Mb window (1 Mb overlap) specific to samples with cleft palate (n = 5). Arrows indicate regions where one SNP was tested for association, bars indicate a locus where more than one SNP has been tested. (B) The same parameters used in (A) were applied to the sequencing data from our samples with exencephaly (n = 3). SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-664544181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Filtering of variants by phenotype generates a map of possible modifying loci contributing to cleft palate or exencephaly. (A) Number of SNPs in each 10 Mb window (1 Mb overlap) specific to samples with cleft palate (&lt;em&gt;n&lt;/em&gt; = 5). Arrows indicate regions where one SNP was tested for association, bars indicate a locus where more than one SNP has been tested. (B) The same parameters used in (A) were applied to the sequencing data from our samples with exencephaly (&lt;em&gt;n&lt;/em&gt; = 3). SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/401/F6.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/2/401/F6.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/401/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/401/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488478" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-35" class="first-child">Filtering of variants by phenotype generates a map of possible modifying loci contributing to cleft palate or exencephaly. (A) Number of SNPs in each 10 Mb window (1 Mb overlap) specific to samples with cleft palate (<em>n</em> = 5). Arrows indicate regions where one SNP was tested for association, bars indicate a locus where more than one SNP has been tested. (B) The same parameters used in (A) were applied to the sequencing data from our samples with exencephaly (<em>n</em> = 3). SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">In conclusion, we have successfully identified the causal variants for five mutant lines, three of which are remutations of previously characterized loci [<em>Plod3</em><em><sup>ugli/ugli</sup></em> (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Rautavuoma <em>et al.</em> 2004</a>; <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Ruotsalainen <em>et al.</em> 2006</a>), <em>Kif20b</em><em><sup>daredevil/daredevil</sup></em> (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Dwyer <em>et al.</em> 2011</a>), and <em>Fbn2</em><em><sup>timon/timon</sup></em> (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Johnson <em>et al.</em> 1998</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Arteaga-Solis <em>et al.</em> 2001</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Browning <em>et al.</em> 2001</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Chaudhry <em>et al.</em> 2001</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Miller <em>et al.</em> 2010</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Shi <em>et al.</em> 2013</a>)] and two of which are novel (<em>Colgalt1</em><em><sup>fosse/fosse</sup></em> and <em>Tgds</em><em><sup>bub/bub</sup></em>). After many years of productive forward genetic screens in mice, this work illustrates that there are still new discoveries to be made using this strategy. In addition to generating new models of human disease, the generation of nonlethal hypomorphic alleles can allow the study of gene function <em>in vivo</em> in cases where a null allele would prove these analyses to be impossible. We have further refined our inbred screening genetic analysis, demonstrating that analysis of three samples from a mutant phenotype will identify candidate regions with high confidence. Importantly, individually barcoding the samples increases the likelihood that one will find the causal loci in genetically complicated scenarios, such as we experienced with <em>Colgalt1</em><sup>fosse</sup>, <em>Plod3</em><sup>ugli</sup>, and <em>Fbn2</em><sup>timon</sup>. We suggest that our strategy is useful for identifying both recessive and dominant genetic modifiers, and deeper sequencing of affected individuals could reduce the number of candidates to an even more manageable number. This report documents continued improvement of methods that we have developed to accelerate genetic analysis of a screen (<a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Gallego-Llamas <em>et al.</em> 2015</a>). These efforts will ultimately act to reduce cost and enable an expeditious transition from genomic to molecular characterization.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">We thank George A. Carlson and staff at the McLaughlin Research Institute, Great Falls, MT, for providing ENU-treated mice. This work was supported by the following source of funding: NICHD/R01HD36404.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-38">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300292/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-39"><em>Communicating editor: F. Pardo-Manuel de Villena</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-19"><span class="received-label">Received </span>September 19, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-23"><span class="accepted-label">Accepted </span>November 23, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Geister <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.1" data-doi="10.1083/jcb.200105046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arteaga-Solis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gayraud</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shum</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Regulation of limb patterning by extracellular microfibrils.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DArteaga-Solis%26rft.auinit1%253DE.%26rft.volume%253D154%26rft.issue%253D2%26rft.spage%253D275%26rft.epage%253D282%26rft.atitle%253DRegulation%2Bof%2Blimb%2Bpatterning%2Bby%2Bextracellular%2Bmicrofibrils%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200105046%26rft_id%253Dinfo%253Apmid%252F11470817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE1NC8yLzI3NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.2" data-doi="10.1006/geno.2001.6523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Planchart</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schimenti</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mutations of the mouse Twist and sy (fibrillin 2) genes induced by chemical mutagenesis of ES cells.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DBrowning%26rft.auinit1%253DV.%2BL.%26rft.volume%253D73%26rft.issue%253D3%26rft.spage%253D291%26rft.epage%253D298%26rft.atitle%253DMutations%2Bof%2Bthe%2Bmouse%2BTwist%2Band%2Bsy%2B%2528fibrillin%2B2%2529%2Bgenes%2Binduced%2Bby%2Bchemical%2Bmutagenesis%2Bof%2BES%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fgeno.2001.6523%26rft_id%253Dinfo%253Apmid%252F11350121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/geno.2001.6523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11350121&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168774400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.3" data-doi="10.1371/journal.pgen.1003219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rimmer</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siggs</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miosge</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roots</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Unlocking the bottleneck in forward genetics using whole-genome sequencing and identity by descent to isolate causative mutations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003219%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003219%26rft_id%253Dinfo%253Apmid%252F23382690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23382690&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.4" data-doi="10.1371/journal.pone.0055429"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caruana</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farlie</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bagheri-Fam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide ENU mutagenesis in combination with high density SNP analysis and exome sequencing provides rapid identification of novel mouse models of developmental disease.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e55429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De55429%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0055429%26rft_id%253Dinfo%253Apmid%252F23469164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0055429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23469164&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.5" data-doi="10.1093/hmg/10.8.835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaudhry</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gazzard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldock</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rock</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mutation of the gene encoding fibrillin-2 results in syndactyly in mice.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DChaudhry%26rft.auinit1%253DS.%2BS.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D835%26rft.epage%253D843%26rft.atitle%253DMutation%2Bof%2Bthe%2Bgene%2Bencoding%2Bfibrillin-2%2Bresults%2Bin%2Bsyndactyly%2Bin%2Bmice%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252F10.8.835%26rft_id%253Dinfo%253Apmid%252F11285249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/10.8.835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11285249&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168223900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.6" data-doi="10.1186/1749-8104-6-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dwyer</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manning</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mudbhary</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleming</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A forward genetic screen with a thalamocortical axon reporter mouse yields novel neurodevelopment mutants and a distinct emx2 mutant phenotype.</span> <abbr class="cit-jnl-abbrev">Neural Dev.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2BDev.%26rft.volume%253D6%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1749-8104-6-3%26rft_id%253Dinfo%253Apmid%252F21214893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1749-8104-6-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21214893&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.7" data-doi="10.1016/j.ajhg.2014.11.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehmke</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caliebe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koenig</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kant</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Homozygous and compound-heterozygous mutations in TGDS cause Catel-Manzke syndrome.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D763%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.11.004%26rft_id%253Dinfo%253Apmid%252F25480037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.11.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25480037&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.8" data-doi="10.1186/s12864-015-2173-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallego-Llamas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timms</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geister</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beier</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Variant mapping and mutation discovery in inbred mice using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D913%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2173-1%26rft_id%253Dinfo%253Apmid%252F26552429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2173-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26552429&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallego-Llamas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timms</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pitstick</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improvement of ENU mutagenesis efficiency using serial injection and mismatch repair deficiency mice.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0159377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0159377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.10" data-doi="10.1167/iovs.05-0254"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKie</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genotype-phenotype correlation of mouse pde6b mutations.</span> <abbr class="cit-jnl-abbrev">Invest. Ophthalmol. Vis. Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">3443</span>–<span class="cit-lpage">3450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInvestigative%2BOphthalmology%2B%2526%2BVisual%2BScience%26rft.stitle%253DIOVS%26rft.aulast%253DHart%26rft.auinit1%253DA.%2BW.%26rft.volume%253D46%26rft.issue%253D9%26rft.spage%253D3443%26rft.epage%253D3450%26rft.atitle%253DGenotype-Phenotype%2BCorrelation%2Bof%2BMouse%2BPde6b%2BMutations%26rft_id%253Dinfo%253Adoi%252F10.1167%252Fiovs.05-0254%26rft_id%253Dinfo%253Apmid%252F16123450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiaW92cyI7czo1OiJyZXNpZCI7czo5OiI0Ni85LzM0NDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.11" data-doi="10.1074/jbc.M006203200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heikkinen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risteli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latvala</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Lysyl hydroxylase 3 is a multifunctional protein possessing collagen glucosyltransferase activity.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">36158</span>–<span class="cit-lpage">36163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M006203200%26rft_id%253Dinfo%253Apmid%252F10934207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvNDYvMzYxNTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.12" data-doi="10.1007/s003359900889"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The original shaker-with-syndactylism mutation (sy) is a contiguous gene deletion syndrome.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DJohnson%26rft.auinit1%253DK.%2BR.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253D889%26rft.epage%253D892%26rft.atitle%253DThe%2Boriginal%2Bshaker-with-syndactylism%2Bmutation%2B%2528sy%2529%2Bis%2Ba%2Bcontiguous%2Bgene%2Bdeletion%2Bsyndrome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs003359900889%26rft_id%253Dinfo%253Apmid%252F9799839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s003359900889&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9799839&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076642200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.13" data-doi="10.1038/ng.369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quint</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glazier</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Angelis</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An ENU-induced mutation of miR-96 associated with progressive hearing loss in mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">614</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLewis%26rft.auinit1%253DM.%2BA.%26rft.volume%253D41%26rft.issue%253D5%26rft.spage%253D614%26rft.epage%253D618%26rft.atitle%253DAn%2BENU-induced%2Bmutation%2Bof%2BmiR-96%2Bassociated%2Bwith%2Bprogressive%2Bhearing%2Bloss%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.369%26rft_id%253Dinfo%253Apmid%252F19363478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19363478&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265575600025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.14" data-doi="10.1371/journal.pone.0009137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neilan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gheryani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ENU mutagenesis reveals a novel phenotype of reduced limb strength in mice lacking fibrillin 2.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMiller%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253De9137%26rft.epage%253De9137%26rft.atitle%253DENU%2Bmutagenesis%2Breveals%2Ba%2Bnovel%2Bphenotype%2Bof%2Breduced%2Blimb%2Bstrength%2Bin%2Bmice%2Blacking%2Bfibrillin%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009137%26rft_id%253Dinfo%253Apmid%252F20161761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20161761&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.15" data-doi="10.1101/gr.4563306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolton</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dwyer</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Utilization of a whole genome SNP panel for efficient genetic mapping in the mouse.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4563306%26rft_id%253Dinfo%253Apmid%252F16461637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvNDM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-8.2.401.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ovchinnikov</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Alcian blue/alizarin red staining of cartilage and bone in mouse.</span> <span class="cit-source">Cold Spring Harb. Protoc.</span> <span class="cit-vol">3</span>: <span class="cit-fpage">pdb.prot5170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BProtoc.%26rft.volume%253D3%26rft.spage%253Dpdb.prot5170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.17" data-doi="10.1016/1074-7613(95)90066-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saijo</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arase</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Developmental defects of lymphoid cells in Jak3 kinase-deficient mice.</span> <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.stitle%253DImmunity%26rft.aulast%253DPark%26rft.auinit1%253DS.%2BY.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253D771%26rft.epage%253D782%26rft.atitle%253DDevelopmental%2Bdefects%2Bof%2Blymphoid%2Bcells%2Bin%2BJak3%2Bkinase-deficient%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F1074-7613%252895%252990066-7%26rft_id%253Dinfo%253Apmid%252F8777722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/1074-7613(95)90066-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8777722&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TM02300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.18" data-doi="10.1007/s00335-004-2379-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quwailid</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hugill</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dear</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vizor</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A gene-driven ENU-based approach to generating an allelic series in any gene.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DQuwailid%26rft.auinit1%253DM.%2BM.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D585%26rft.epage%253D591%26rft.atitle%253DA%2Bgene-driven%2BENU-based%2Bapproach%2Bto%2Bgenerating%2Ban%2Ballelic%2Bseries%2Bin%2Bany%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-004-2379-z%26rft_id%253Dinfo%253Apmid%252F15457338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-004-2379-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15457338&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223063600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.19" data-doi="10.1073/pnas.0404966101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rautavuoma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takaluoma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sormunen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myllyharju</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kivirikko</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Premature aggregation of type IV collagen and early lethality in lysyl hydroxylase 3 null mice.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">14120</span>–<span class="cit-lpage">14125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404966101%26rft_id%253Dinfo%253Apmid%252F15377789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM5LzE0MTIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.20" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.21" data-doi="10.1242/jcs.02780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruotsalainen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sipila</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vapola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sormunen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salo</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Glycosylation catalyzed by lysyl hydroxylase 3 is essential for basement membranes.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DRuotsalainen%26rft.auinit1%253DH.%26rft.volume%253D119%26rft.issue%253D4%26rft.spage%253D625%26rft.epage%253D635%26rft.atitle%253DGlycosylation%2Bcatalyzed%2Bby%2Blysyl%2Bhydroxylase%2B3%2Bis%2Bessential%2Bfor%2Bbasement%2Bmembranes%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.02780%26rft_id%253Dinfo%253Apmid%252F16467571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6OToiMTE5LzQvNjI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.22" data-doi="10.1007/s00335-033-2317-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sagai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masuya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Phylogenetic conservation of a limb-specific, cis-acting regulator of Sonic hedgehog (Shh).</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DSagai%26rft.auinit1%253DT.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D34%26rft.atitle%253DPhylogenetic%2Bconservation%2Bof%2Ba%2Blimb-specific%252C%2Bcis-acting%2Bregulator%2Bof%2BSonic%2Bhedgehog%2B%2528%2BShh%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-033-2317-5%26rft_id%253Dinfo%253Apmid%252F14727139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-033-2317-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14727139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187926100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.23" data-doi="10.1016/j.ajhg.2008.09.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salo</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farndon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moss</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grindulis</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A connective tissue disorder caused by mutations of the lysyl hydroxylase 3 gene.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSalo%26rft.auinit1%253DA.%2BM.%26rft.volume%253D83%26rft.issue%253D4%26rft.spage%253D495%26rft.epage%253D503%26rft.atitle%253DA%2Bconnective%2Btissue%2Bdisorder%2Bcaused%2Bby%2Bmutations%2Bof%2Bthe%2Blysyl%2Bhydroxylase%2B3%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.09.004%26rft_id%253Dinfo%253Apmid%252F18834968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18834968&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.24" data-doi="10.1128/MCB.02085-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schegg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulsmeier</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutschmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maag</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hennet</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Core glycosylation of collagen is initiated by two beta(1-O)galactosyltransferases.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.02085-07%26rft_id%253Dinfo%253Apmid%252F19075007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI5LzQvOTQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.25" data-doi="10.1167/iovs.13-12687"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mecham</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassnett</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ocular phenotype of Fbn2-null mice.</span> <abbr class="cit-jnl-abbrev">Invest. Ophthalmol. Vis. Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">7163</span>–<span class="cit-lpage">7173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInvest.%2BOphthalmol.%2BVis.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1167%252Fiovs.13-12687%26rft_id%253Dinfo%253Apmid%252F24130178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiaW92cyI7czo1OiJyZXNpZCI7czoxMDoiNTQvMTIvNzE2MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi80MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moresco</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallon</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Current strategies for mutation detection in phenotype-driven screens utilising next generation sequencing.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D26%26rft.spage%253D486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stottmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beier</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ENU mutagenesis in the mouse.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Mouse Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BMouse%2BBiol.%26rft.volume%253D4%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.28" data-doi="10.1016/S0945-053X(02)00071-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luosujarvi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heikkinen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risteli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">a</span> <span class="cit-article-title">The third activity for lysyl hydroxylase 3: galactosylation of hydroxylysyl residues in collagens in vitro.</span> <abbr class="cit-jnl-abbrev">Matrix Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMatrix%2Bbiology%2B%253A%2B%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BMatrix%2BBiology%26rft.stitle%253DMatrix%2BBiol%26rft.aulast%253DWang%26rft.auinit1%253DC.%26rft.volume%253D21%26rft.issue%253D7%26rft.spage%253D559%26rft.epage%253D566%26rft.atitle%253DThe%2Bthird%2Bactivity%2Bfor%2Blysyl%2Bhydroxylase%2B3%253A%2Bgalactosylation%2Bof%2Bhydroxylysyl%2Bresidues%2Bin%2Bcollagens%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0945-053X%252802%252900071-9%26rft_id%253Dinfo%253Apmid%252F12475640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0945-053X(02)00071-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12475640&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F401.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179928600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.29" data-doi="10.1074/jbc.M201389200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risteli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heikkinen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussa</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">b</span> <span class="cit-article-title">Identification of amino acids important for the catalytic activity of the collagen glucosyltransferase associated with the multifunctional lysyl hydroxylase 3 (LH3).</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">18568</span>–<span class="cit-lpage">18573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M201389200%26rft_id%253Dinfo%253Apmid%252F11896059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvMjEvMTg1NjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.401.30" data-doi="10.1042/bse0520113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamauchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sricholpech</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Lysine post-translational modifications of collagen.</span> <abbr class="cit-jnl-abbrev">Essays Biochem.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEssays%2BIn%2BBiochemistry%26rft.stitle%253DEssays%2BBiochem.%26rft.aulast%253DYamauchi%26rft.auinit1%253DM.%26rft.volume%253D52%26rft.issue%253D0%26rft.spage%253D113%26rft.epage%253D133%26rft.atitle%253DLysine%2Bpost-translational%2Bmodifications%2Bof%2Bcollagen%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbse0520113%26rft_id%253Dinfo%253Apmid%252F22708567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicHBlYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUyLzAvMTEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzQwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/401.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/393" title="Reference Quality Genome Assemblies of Three Parastagonospora nodorum Isolates Differing in Virulence on Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/411" title="Dissecting the Genetic Architecture of Shoot Growth in Carrot (Daucus carota L.) Using a Diallel Mating Design" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/401" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j7wGoF58AxUhGFr8yQQFvxf32xKcbGpfc70oenpcvNc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mutant-screen-report" class="" data-icon-position="" data-hide-link-title="0">Mutant Screen Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/401" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/401" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/401" class="active" data-icon-position="" data-hide-link-title="0">Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488296" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9cZF5881lI9hXlfkST-AVgswOUfJ34k45f_EUd_ifNU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/401.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/401" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-D6oKWKWi1bvoeqeAdoCr-8HE0x9Jo4ty6UKUs6VbWYY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/401" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/401.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488296" id="node488296--2" data-pisa="ggg;8/2/401" data-pisa-master="ggg;g3.117.300292" data-apath="/ggg/8/2/401.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Krista A. Geister</span>, <span class="highwire-citation-author" data-delta="1">Andrew E. Timms</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0325-8520" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David R. Beier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">401-409;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300292 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488296--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Krista A. Geister </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics, University of Washington School of Medicine, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krista%2BA.%2BGeister%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geister%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrista%2BA.%2BGeister%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew E. Timms </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BE.%2BTimms%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Timms%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BE.%2BTimms%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David R. Beier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics, University of Washington School of Medicine, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BR.%2BBeier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beier%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BR.%2BBeier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0325-8520" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David R. Beier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.beier@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">david.beier@seattlechildrens.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/401" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/401&amp;title=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/401&amp;title=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/401&amp;title=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/401&amp;text=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/401&amp;title=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/401&amp;t=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/401&amp;title=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/401&amp;title=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488296/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488296" id="node488296--3" data-pisa="ggg;8/2/401" data-pisa-master="ggg;g3.117.300292" data-apath="/ggg/8/2/401.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Optimizing Genomic Methods for Mapping and Identification of Candidate Variants in ENU Mutagenesis Screens Using Inbred Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Krista A. Geister</span>, <span class="highwire-citation-author" data-delta="1">Andrew E. Timms</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0325-8520" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David R. Beier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">401-409;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300292 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488296--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Krista A. Geister </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics, University of Washington School of Medicine, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krista%2BA.%2BGeister%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geister%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrista%2BA.%2BGeister%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew E. Timms </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BE.%2BTimms%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Timms%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BE.%2BTimms%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David R. Beier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics, University of Washington School of Medicine, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BR.%2BBeier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beier%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BR.%2BBeier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0325-8520" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David R. Beier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.beier@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">david.beier@seattlechildrens.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=401&amp;rft.epage=409&amp;rft.atitle=Optimizing%20Genomic%20Methods%20for%20Mapping%20and%20Identification%20of%20Candidate%20Variants%20in%20ENU%20Mutagenesis%20Screens%20Using%20Inbred%20Mice&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Geister%2C+Krista+A.&amp;rft.au=Timms%2C+Andrew+E.&amp;rft.au=Beier%2C+David+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488296/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488296/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488296/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488296/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488296/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488296/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488296/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488296/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488296/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488296/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488296/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488296/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495856" id="node495856" data-pisa="ggg;10/3/917" data-pisa-master="ggg;g3.119.400936" data-apath="/ggg/10/3/917.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/917" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Mutational Analysis of N-Ethyl-N-Nitrosourea (ENU) in the Fission Yeast <em>Schizosaccharomyces pombe</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495835" id="node495835" data-pisa="ggg;g3.119.400927v1" data-pisa-master="ggg;g3.119.400927" data-apath="/ggg/early/2020/02/21/g3.119.400927.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/02/21/g3.119.400927" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of Suppressors of <em>top-2 </em>Embryonic Lethality in <em>Caenorhabditis elegans</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495548" id="node495548" data-pisa="ggg;10/2/479" data-pisa-master="ggg;g3.119.400933" data-apath="/ggg/10/2/479.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/479" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of the Genetic Requirements for Zinc Tolerance and Toxicity in <em>Saccharomyces cerevisiae</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mutant%20Screen%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mutant Screen Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/401" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/401.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/401.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/401.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/401.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
