<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/679" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/679.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/679.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/679.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/679.atom" />
<meta name="HW.pisa" content="ggg;8/2/679" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies" />
<meta name="DC.Identifier" content="10.1534/g3.117.300495" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Brekke et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Nonmodel rodents are widely used as subjects for both basic and applied biological research, but the genetic diversity of the study individuals is rarely quantified. University-housed colonies tend to be small and subject to founder effects and genetic drift; so they may be highly inbred or show substantial genetic divergence from other colonies, even those derived from the same source. Disregard for the levels of genetic diversity in an animal colony may result in a failure to replicate results if a different colony is used to repeat an experiment, as different colonies may have fixed alternative variants. Here we use high throughput sequencing to demonstrate genetic divergence in three isolated colonies of Mongolian gerbil ( Meriones unguiculatus ) even though they were all established recently from the same source. We also show that genetic diversity in allegedly “outbred” colonies of nonmodel rodents (gerbils, hamsters, house mice, deer mice, and rats) varies considerably from nearly no segregating diversity to very high levels of polymorphism. We conclude that genetic divergence in isolated colonies may play an important role in the “replication crisis.” In a more positive light, divergent rodent colonies represent an opportunity to leverage genetically distinct individuals in genetic crossing experiments. In sum, awareness of the genetic diversity of an animal colony is paramount as it allows researchers to properly replicate experiments and also to capitalize on other genetically distinct individuals to explore the genetic basis of a trait." />
<meta name="DC.Contributor" content="Thomas D. Brekke" />
<meta name="DC.Contributor" content="Katherine A. Steele" />
<meta name="DC.Contributor" content="John F. Mulley" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Nonmodel rodents are widely used as subjects for both basic and applied biological research, but the genetic diversity of the study individuals is rarely quantified. University-housed colonies tend to be small and subject to founder effects and genetic drift; so they may be highly inbred or show substantial genetic divergence from other colonies, even those derived from the same source. Disregard for the levels of genetic diversity in an animal colony may result in a failure to replicate results if a different colony is used to repeat an experiment, as different colonies may have fixed alternative variants. Here we use high throughput sequencing to demonstrate genetic divergence in three isolated colonies of Mongolian gerbil (&lt;i&gt;Meriones unguiculatus&lt;/i&gt;) even though they were all established recently from the same source. We also show that genetic diversity in allegedly “outbred” colonies of nonmodel rodents (gerbils, hamsters, house mice, deer mice, and rats) varies considerably from nearly no segregating diversity to very high levels of polymorphism. We conclude that genetic divergence in isolated colonies may play an important role in the “replication crisis.” In a more positive light, divergent rodent colonies represent an opportunity to leverage genetically distinct individuals in genetic crossing experiments. In sum, awareness of the genetic diversity of an animal colony is paramount as it allows researchers to properly replicate experiments and also to capitalize on other genetically distinct individuals to explore the genetic basis of a trait.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/679" />
<meta name="citation_id" content="8/2/679" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/679" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/679.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/679.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/679.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300495" />
<meta name="citation_pmid" content="29242387" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="679" />
<meta name="citation_lastpage" content="686" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Thomas D. Brekke" />
<meta name="citation_author_institution" content="School of Biological Sciences, Bangor University, LL57 2DG, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4479-1847" />
<meta name="citation_author" content="Katherine A. Steele" />
<meta name="citation_author_institution" content="School of Environment, Natural Resources and Geography, Bangor University, LL57 2DG, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4896-8857" />
<meta name="citation_author" content="John F. Mulley" />
<meta name="citation_author_institution" content="School of Biological Sciences, Bangor University, LL57 2DG, United Kingdom" />
<meta name="citation_author_email" content="j.mulley@bangor.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1537-7316" />
<meta name="citation_reference" content="citation_journal_title=Hear. Res.;citation_author=L. Abbas;citation_author=M. N. Rivolta;citation_title=Aminoglycoside ototoxicity and hair cell ablation in the adult gerbil: a simple model to study hair cell loss and regeneration.;citation_pages=12-26;citation_volume=325;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. Balloux;citation_author=W. Amos;citation_author=T. Coulson;citation_title=Does heterozygosity estimate inbreeding in real populations?;citation_pages=3021-3031;citation_volume=13;citation_year=2004;citation_issue=10;citation_pmid=15367117;citation_doi=10.1111/j.1365-294X.2004.02318.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. D. Brekke;citation_author=J. M. Good;citation_title=Parent-of-origin growth effects and the evolution of hybrid inviability in dwarf hamsters.;citation_pages=3134-3148;citation_volume=68;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. D. Brekke;citation_author=L. A. Henry;citation_author=J. M. Good;citation_title=Genomic imprinting, disrupted placental expression, and speciation.;citation_pages=2690-2703;citation_volume=70;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Epilepsia;citation_author=J. R. Buchhalter;citation_title=Animal models of inherited epilepsy.;citation_pages=S31-S41;citation_volume=34;citation_year=1993" />
<meta name="citation_reference" content="citation_title=Models of Seizures and Epilepsy;citation_title=Inherited epilepsy in Mongolian gerbils;citation_author=P. S. Buckmaster;citation_pages=273-294;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_abbrev=Animal;citation_author=J. Casellas;citation_title=Inbred mouse strains and genetic stability: a review.;citation_pages=1-7;citation_volume=5;citation_year=2011;citation_issue=1;citation_pmid=22440695;citation_doi=10.1017/S1751731110001667" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: building and genotyping loci de novo from short-read sequences.;citation_pages=171-182;citation_volume=1;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. Chen;citation_author=N. Jongkamonwiwat;citation_author=L. Abbas;citation_author=S. J. Eshtan;citation_author=S. L. Johnson;citation_title=Restoration of auditory evoked responses by human ES-cell-derived otic progenitors.;citation_pages=278-282;citation_volume=490;citation_year=2012;citation_issue=7419;citation_pmid=22972191;citation_doi=10.1038/nature11415" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. Chia;citation_author=F. Achilli;citation_author=M. F. W. Festing;citation_author=E. M. C. Fisher;citation_title=The origins and uses of mouse outbred stocks.;citation_pages=1181-1186;citation_volume=37;citation_year=2005;citation_issue=11;citation_pmid=16254564;citation_doi=10.1038/ng1665" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Reproduction;citation_journal_abbrev=Reproduction;citation_author=M. M. Clark;citation_author=M. Ham;citation_author=B. G. Galef;citation_title=Differences in the sex ratios of offspring originating in the right and left ovaries of Mongolian gerbils (Meriones unguiculatus);citation_pages=393-396;citation_volume=101;citation_year=1994;citation_issue=2;citation_pmid=7932374;citation_doi=10.1530/jrf.0.1010393" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the collaborative cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D.-J. de Koning;citation_author=L. M. McIntyre;citation_title=Back to the future: multiparent populations provide the key to unlocking the genetic basis of complex traits.;citation_volume=206;citation_year=2017;citation_pmid=28592493;citation_doi=10.1534/genetics.117.203265" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=X. Du;citation_author=Z. Chen;citation_author=W. Li;citation_author=Y. Tan;citation_author=J. Lu;citation_title=Development of novel microsatellite DNA markers by cross-amplification and analysis of genetic variation in gerbils.;citation_volume=101;citation_year=2010;citation_pmid=20525768;citation_doi=10.1093/jhered/esq066" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Res.;citation_author=X. Y. Du;citation_author=W. Li;citation_author=X. Y. Sa;citation_author=C. L. Li;citation_author=J. Lu;citation_title=Selection of an effective microsatellite marker system for genetic control and analysis of gerbil populations in China.;citation_pages=11030-11042;citation_volume=14;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_title=The Mouse in Biomedical Research;citation_title=Mouse strain and genetic nomenclature: an abbreviated guide;citation_author=J. T. Eppig;citation_pages=79-98;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Notes;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of population structure using multilocus genotype data: dominant markers and null alleles.;citation_pages=574-578;citation_volume=7;citation_year=2007;citation_issue=4;citation_pmid=18784791;citation_doi=10.1111/j.1471-8286.2007.01758.x" />
<meta name="citation_reference" content="citation_journal_title=Neurobiology of aging;citation_journal_abbrev=Neurobiol Aging;citation_author=M. F. Festing;citation_title=Warning: the use of heterogeneous mice may seriously damage your research.;citation_pages=237-244;citation_volume=20;citation_year=1999;citation_issue=2;citation_pmid=10537033;citation_doi=10.1016/S0197-4580(99)00040-8" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Pathol.;citation_author=M. F. W. Festing;citation_title=Inbred strains should replace outbred stocks in toxicology, safety testing, and drug development.;citation_volume=38;citation_year=2010;citation_pmid=20562325;citation_doi=10.1177/0192623310373776" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=C. Fuchsberger;citation_author=J. Flannick;citation_author=T. M. Teslovich;citation_author=A. Mahajan;citation_author=V. Agarwala;citation_title=The genetic architecture of type 2 diabetes.;citation_pages=1-29;citation_volume=536;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. R. Gawriluk;citation_author=J. Simkin;citation_author=K. L. Thompson;citation_author=S. K. Biswas;citation_author=Z. Clare-Salzler;citation_title=Comparative analysis of ear-hole closure identifies epimorphic regeneration as a discrete trait in mammals.;citation_pages=1-16;citation_volume=7;citation_year=2016;citation_pmid=27009409;citation_doi=10.1038/ncomms11388" />
<meta name="citation_reference" content="citation_journal_title=Laboratory Animals;citation_journal_abbrev=Lab Anim;citation_author=A. Groen;citation_author=A. J. Lagerwerf;citation_title=Genic heterogeneity and genetic monitoring of mouse outbred stocks;citation_pages=81-85;citation_volume=13;citation_year=1979;citation_issue=2;citation_pmid=480905;citation_doi=10.1258/002367779780943387" />
<meta name="citation_reference" content="citation_journal_title=Age (Dordr.);citation_author=J. M. Harper;citation_title=Wild-derived mouse stocks: an underappreciated tool for aging research.;citation_pages=135-145;citation_volume=30;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Disease Models &amp; Mechanisms;citation_journal_abbrev=DMM;citation_author=M. J. Justice;citation_author=P. Dhillon;citation_title=Using the mouse to model human disease: increasing validity and reproducibility;citation_pages=101-103;citation_volume=9;citation_year=2016;citation_issue=2;citation_pmid=26839397;citation_doi=10.1242/dmm.024547" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. Li;citation_author=J. Lu;citation_author=Y. Wang;citation_author=X. Huo;citation_author=Z. Li;citation_title=Establishment and characterization of a newly established diabetic gerbil line.;citation_pages=e0159420;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=T. J. Little;citation_author=N. Colegrave;citation_title=Caging and uncaging genetics.;citation_pages=e1002525;citation_volume=14;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. M. Love;citation_author=A. M. Knight;citation_author=M. A. McAleer;citation_author=J. A. Todd;citation_title=Towards construction of a high resolution map of the mouse genome using PCR-analysed microsatellites;citation_pages=4123-4130;citation_volume=18;citation_year=1990;citation_issue=14;citation_pmid=2377456;citation_doi=10.1093/nar/18.14.4123" />
<meta name="citation_reference" content="citation_journal_title=Comp. Biochem. Physiol. Part A. Physiol.;citation_author=J. J. McManus;citation_title=Water relations and food consumption of the mongolian gerbil, Meriones unguiculatus.;citation_pages=959-967;citation_volume=43;citation_year=1972" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=M. H. Meisler;citation_author=J. Kearney;citation_author=R. Ottman;citation_author=A. Escayg;citation_title=IDENTIFICATION OF EPILEPSY GENES IN HUMAN AND MOUSE;citation_pages=567-588;citation_volume=35;citation_year=2001;citation_issue=1;citation_pmid=11700294;citation_doi=10.1146/annurev.genet.35.102401.091142" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R. W. Ness;citation_author=Y.-H. Zhang;citation_author=L. Cong;citation_author=Y. Wang;citation_author=J.-X. Zhang;citation_title=Nuclear gene variation in wild brown rats.;citation_pages=1661-1664;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=K. Neumann;citation_author=S. Maak;citation_author=I. W. Stuermer;citation_author=G. von Lengerken;citation_author=R. Gattermann;citation_title=Low Microsatellite Variation in Laboratory Gerbils;citation_pages=71-74;citation_volume=92;citation_year=2001;citation_issue=1;citation_pmid=11336232;citation_doi=10.1093/jhered/92.1.71" />
<meta name="citation_reference" content="citation_journal_title=Laboratory Animals;citation_journal_abbrev=Lab Anim;citation_author=V. E. Papaioannou;citation_author=M. F. Festing;citation_title=Genetic drift in a stock of laboratory mice;citation_pages=11-13;citation_volume=14;citation_year=1980;citation_issue=1;citation_pmid=6928500;citation_doi=10.1258/002367780780943015" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=F. Petrij;citation_author=K. van Veen;citation_author=M. Mettler;citation_author=V. Brückmann;citation_title=A Second Acromelanistic Allelomorph at the Albino Locus of the Mongolian Gerbil (Meriones unguiculatus);citation_pages=74-78;citation_volume=92;citation_year=2001;citation_issue=1;citation_pmid=11336233;citation_doi=10.1093/jhered/92.1.74" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. Razzoli;citation_title=AFLP to Assess Genetic Variation in Laboratory Gerbils (Meriones unguiculatus);citation_pages=507-511;citation_volume=94;citation_year=2003;citation_issue=6;citation_pmid=14691317;citation_doi=10.1093/jhered/esg097" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. S. Rich;citation_title=Diabetes: still a geneticist’s nightmare.;citation_pages=37-38;citation_volume=536;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. H. Richter;citation_author=J. P. Garner;citation_author=B. Zipser;citation_author=L. Lewejohann;citation_author=N. Sachser;citation_title=Effect of population heterogenization on the reproducibility of mouse behavior: a multi-laboratory study.;citation_pages=e16461-e16461;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21305027;citation_doi=10.1371/journal.pone.0016461" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. Roberts;citation_author=F. Pardo-Manuel de Villena;citation_author=W. Wang;citation_author=L. McMillan;citation_author=D. W. Threadgill;citation_title=The polymorphism architecture of mouse genetic resources elucidated using genome-wide resequencing data: implications for QTL discovery and systems genetics.;citation_pages=473-481;citation_volume=18;citation_year=2007;citation_issue=6-7;citation_pmid=17674098;citation_doi=10.1007/s00335-007-9045-1" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=N. A. Rosenberg;citation_title=distruct: a program for the graphical display of population structure.;citation_pages=137-138;citation_volume=4;citation_year=2003;citation_doi=10.1046/j.1471-8286.2003.00566.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. A. Rosenberg;citation_author=J. K. Pritchard;citation_author=J. L. Weber;citation_author=H. M. Cann;citation_author=K. K. Kidd;citation_title=Genetic Structure of Human Populations;citation_pages=2381-2385;citation_volume=298;citation_year=2002;citation_issue=5602;citation_pmid=12493913;citation_doi=10.1126/science.1078311" />
<meta name="citation_reference" content="citation_title=The Principles of Humane Experimental Technique;citation_author=W. M. S. Russell;citation_author=R. L. Burch;citation_year=1959" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Salcedo;citation_author=A. Geraldes;citation_author=M. W. Nachman;citation_title=Nucleotide Variation in Wild and Inbred Mice;citation_pages=2277-2291;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073432;citation_doi=10.1534/genetics.107.079988" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. W. Santure;citation_author=J. Stapley;citation_author=A. D. Ball;citation_author=T. R. Birkhead;citation_author=T. Burke;citation_title=On the use of large marker panels to estimate inbreeding and relatedness: empirical and simulation studies of a pedigreed zebra finch population typed at 771 SNPs.;citation_pages=1439-1451;citation_volume=19;citation_year=2010;citation_issue=7;citation_pmid=20149098;citation_doi=10.1111/j.1365-294X.2010.04554.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Serikawa;citation_author=T. Kuramoto;citation_author=P. Hilbert;citation_author=M. Mori;citation_author=J. Yamada;citation_title=Rat gene mapping using PCR-analyzed microsatellites.;citation_pages=701-721;citation_volume=131;citation_year=1992;citation_issue=3;citation_pmid=1628813" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=L. J. Sittig;citation_author=P. Carbonetto;citation_author=K. A. Engel;citation_author=K. S. Krauss;citation_author=C. M. Barrios-Camacho;citation_title=Genetic background limits generalizability of genotype-phenotype relationships.;citation_pages=1253-1259;citation_volume=91;citation_year=2016;citation_pmid=27618673;citation_doi=10.1016/j.neuron.2016.08.013" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=B. M. G. Smits;citation_author=B. F. M. van Zutphen;citation_author=R. H. A. Plasterk;citation_author=E. Cuppen;citation_title=Genetic Variation in Coding Regions Between and Within Commonly Used Inbred Rat Strains;citation_pages=1285-1290;citation_volume=14;citation_year=2004;citation_issue=7;citation_pmid=15231744;citation_doi=10.1101/gr.2155004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Srivastava;citation_author=A. P. Morgan;citation_author=M. L. Najarian;citation_author=V. K. Sarsani;citation_author=J. S. Sigmon;citation_title=Genomes of the mouse collaborative cross.;citation_volume=206;citation_year=2017;citation_pmid=28592495;citation_doi=10.1534/genetics.116.198838" />
<meta name="citation_reference" content="citation_journal_title=Behavioural brain research;citation_journal_abbrev=Behav Brain Res;citation_author=I. W. Stuermer;citation_author=W. Wetzel;citation_title=Early experience and domestication affect auditory discrimination learning, open field behaviour and brain size in wild Mongolian gerbils and domesticated laboratory gerbils (Meriones unguiculatus forma domestica).;citation_pages=11-21;citation_volume=173;citation_year=2006;citation_issue=1;citation_pmid=16846650;citation_doi=10.1016/j.bbr.2006.05.025" />
<meta name="citation_reference" content="citation_journal_title=Zool. Anz.;citation_author=I. W. Stuermer;citation_author=K. Plotz;citation_author=A. Leybold;citation_author=O. Zinke;citation_author=O. Kalberlah;citation_title=Intraspecific allometric comparison of laboratory gerbils with Mongolian gerbils trapped in the wild indicates domestication in Meriones unguiculatus (Milne-Edwards, 1867) (Rodentia: Gerbillinae).;citation_pages=249-266;citation_volume=242;citation_year=2003;citation_doi=10.1078/0044-5231-00102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-resolution genetic mapping using the Mouse Diversity outbred population.;citation_volume=190;citation_year=2012;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Physiology &amp; Behavior;citation_journal_abbrev=Physiology &amp; Behavior;citation_author=D. D. Thiessen;citation_author=E. M. Kittrell;citation_title=The Harderian gland and thermoregulation in the gerbil (Meriones unguiculatus).;citation_pages=417-424;citation_volume=24;citation_year=1980;citation_issue=3;citation_pmid=7378139;citation_doi=10.1016/0031-9384(80)90229-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. W. Threadgill;citation_author=G. A. Churchill;citation_title=Ten years of the collaborative cross.;citation_volume=190;citation_year=2012;citation_pmid=22345604;citation_doi=10.1534/genetics.111.138032" />
<meta name="citation_reference" content="citation_journal_title=Laboratory animal science;citation_journal_abbrev=Lab Anim Sci;citation_author=A. L. Vincent;citation_author=G. E. Rodrick;citation_title=The pathology of the Mongolian Gerbil (Meriones unguiculatus): a review.;citation_pages=645-651;citation_volume=29;citation_year=1979;citation_issue=5;citation_pmid=92595" />
<meta name="citation_reference" content="citation_journal_title=Acta Theriol. (Warsz.);citation_author=D. Wang;citation_author=Y. Wang;citation_author=Z. Wang;citation_title=Metabolism and thermoregulation in the Mongolian gerbil Meriones unguiculatus.;citation_pages=183-192;citation_volume=45;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. N. Weber;citation_author=B. K. Peterson;citation_author=H. E. Hoekstra;citation_title=Discrete genetic modules are responsible for complex burrow evolution in Peromyscus mice.;citation_pages=402-405;citation_volume=493;citation_year=2013;citation_pmid=23325221;citation_doi=10.1038/nature11816" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. Yalcin;citation_author=J. Nicod;citation_author=A. Bhomra;citation_author=S. Davidson;citation_author=J. Cleak;citation_title=Commercially available outbred mice for genome-wide association studies.;citation_pages=e1001085;citation_volume=6;citation_year=2010;citation_pmid=20838427;citation_doi=10.1371/journal.pgen.1001085" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=X. Zheng;citation_author=D. Levine;citation_author=J. Shen;citation_author=S. M. Gogarten;citation_title=A high-performance computing toolset for relatedness and principal component analysis of SNP data.;citation_pages=3326-3328;citation_volume=28;citation_year=2012" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Nonmodel rodents are widely used as subjects for both basic and applied biological research, but the genetic diversity of the study individuals is rarely quantified. University-housed colonies tend to be small and subject to founder effects and genetic drift; so they may be highly inbred or show substantial genetic divergence from other colonies, even those derived from the same source. Disregard for the levels of genetic diversity in an animal colony may result in a failure to replicate results if a different colony is used to repeat an experiment, as different colonies may have fixed alternative variants. Here we use high throughput sequencing to demonstrate genetic divergence in three isolated colonies of Mongolian gerbil ( Meriones unguiculatus ) even though they were all established recently from the same source. We also show that genetic diversity in allegedly “outbred” colonies of nonmodel rodents (gerbils, hamsters, house mice, deer mice, and rats) varies considerably from nearly no segregating diversity to very high levels of polymorphism. We conclude that genetic divergence in isolated colonies may play an important role in the “replication crisis.” In a more positive light, divergent rodent colonies represent an opportunity to leverage genetically distinct individuals in genetic crossing experiments. In sum, awareness of the genetic diversity of an animal colony is paramount as it allows researchers to properly replicate experiments and also to capitalize on other genetically distinct individuals to explore the genetic basis of a trait." />
<meta name="og-title" property="og:title" content="Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/679" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Nonmodel rodents are widely used as subjects for both basic and applied biological research, but the genetic diversity of the study individuals is rarely quantified. University-housed colonies tend to be small and subject to founder effects and genetic drift; so they may be highly inbred or show substantial genetic divergence from other colonies, even those derived from the same source. Disregard for the levels of genetic diversity in an animal colony may result in a failure to replicate results if a different colony is used to repeat an experiment, as different colonies may have fixed alternative variants. Here we use high throughput sequencing to demonstrate genetic divergence in three isolated colonies of Mongolian gerbil ( Meriones unguiculatus ) even though they were all established recently from the same source. We also show that genetic diversity in allegedly “outbred” colonies of nonmodel rodents (gerbils, hamsters, house mice, deer mice, and rats) varies considerably from nearly no segregating diversity to very high levels of polymorphism. We conclude that genetic divergence in isolated colonies may play an important role in the “replication crisis.” In a more positive light, divergent rodent colonies represent an opportunity to leverage genetically distinct individuals in genetic crossing experiments. In sum, awareness of the genetic diversity of an animal colony is paramount as it allows researchers to properly replicate experiments and also to capitalize on other genetically distinct individuals to explore the genetic basis of a trait." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rFZkUKLowpsALWNficZ_qRB0i0gZWXIEy-K7JnA2NUU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488304","apath":"\/ggg\/8\/2\/679.atom","pisa":"ggg;8\/2\/679","ac":{"\/ggg\/8\/2\/679.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/679.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/679"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/679"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/679":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zDwyj+s6vaZP8PH2ZWLFAZdsRVPbplgLjbcWL4cUe1TWnBOfvcXPv3yGoe5unww2oPi0rEm9d4b8cK0BYzHEe51UEIWQKHQTjBuNEEmEdbcQtlWhiS25YTyCIpVLebTegmtGkh9OANnrQr\\\\\\\/DcHPJh16hqnPccJEzYYhnsGda\\\\\\\/sjWd2YRTolnpaHTxS+J33jUG9XlAjicZi\\\\\\\/oHId7uWexkOvsQ8UM53NT\\\\\\\/tr7e8j46ROuqtPz55cYBIZOCCN4kRi5wasuaYDjrs9RYk42UrUCHHJAjLScPvSL7djKYwAGRT4rLee1RRAgB3hBft6TPWarpI46DDfSH4SGuFdQRJRCFzZJe27ef+oMRSim8hAqtZz8jfvaySo1HWs0zy7tjC2s767s6pYcUp9qKNGWkhDdXUFQDaSZWmd\\\\\\\/RhAOPrA3Pb9JqS6jsvUTzQpXoMJ7kO3EcL7yKEbbYfG449Dn49p+Qq2eOv7dtn2F9tPakEeFkdeX3on1WRd0cSNW\\\\\\\/V6O+LUKBhJ\\\\\\\/+r\\\\\\\/sIm0E9fl30YcfCkR5y20AJkKlAUZm3joWakvMTvQycoDuIyBxbzxsanYbtiS\\\\\\\/3NAGuet+JJQHwyQWurcSbWcT5YW8RdwK\\\\\\\/SzHwL\\\\\\\/lbga0lEM1Mx6jZxG3xnUsL7Aycq43xu868m+1SQwgLXiIdGYuQBd4ela4HP9076w6tC4zt4JoeUcqBFJt1kZWQaE48j2Al726YlAM6hdaQ9V8ZhTXRuGaYCQ3HUBYoEew40lcK1LcQSbRP+nCGu6QufTFh4RqC41wkZhGOozhbsNXTfUmL8qJF+\\\\\\\/5iscGow9JVu1djqSBY7SGehmWDKubRyS2MMnUhNpkjNuLEJLkeGDlHNxYmxPHwlWscHi7CbGX7DP3c1fiWBy1yh3PRDEItRE4ZGRaAxnqcuNhc2oG7FuV\\\\\\\/aOK0v+jLKqp+0TSc9ijfrfNzmqPGN+nqwpUagOIPRrCUlJjjpYzSBt37CSiQOovSsc+7GEbXCew4Jiuk5xl5B8gVspLwEAUfrHBxGwlD3c++KVZhhIp2t9W6n1W00EAC51Bc32HOQsdfEtzrooZ30Nr0noWJxER2j\\\\\\\/PzC2vNqf0KumhHDNLg0O9vEMAq\\\\\\\/i2I8tfN+\\\\\\\/8L7wx8kgvKvbwU9AKLzYGNAVJ6KAw6uit0awJFt1xqYsnG36qZn3PSVabCvel5IvycZ9xntWgYcqKRWE7o9WPQ5EwS\\\\\\\/q8MBXrKEbwsWItQ6VbOveIaTHzUr\\\\\\\/ZstMz90DZf9NhqxKrP0uIw6Bf92WKV6Yf62yQ5VVO\\\u0022,\\\u0022iv\\\u0022:\\\u0022QWc4rjEEdQKOFRUfK9EYcQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221513c30f89b564a290ab7c6dc2f701c66e3fe27eb4e025e34784b491b331cc42\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xbcDRmyMGpJXHzys+DCUJd\\\\\\\/rN2dEI\\\\\\\/PtfniMIXwOWrpDrTYPSPmNW1NCd4T+NtB1+S5pLnc\\\\\\\/GHbGUtr6fpxuEOfPYrKYqtluqUF\\\\\\\/Ddio3pL5na\\\\\\\/altK4vHay8fL7Uze\\\\\\\/pRD8ljiDrO1846QWwPNnJzxKEqwK8Ir0dh8KdgYji3HNy7tNqxZqxTgFPtKly0E6zTTe\\\\\\\/OUlOdEOIoYuX0FqnYy3ce73HIWCstsQRRbENnVAqdHCTjmL1qHh+qi9O2NMK3z+Rbp+VSc6SVIma0fmXb2B9X5xONvqNKxwnAExCNNplh5vvRqjYy4kuuJBMkhBzBQDWHF+slBPNzC6nE2N4I6HUbPCy\\\\\\\/OGT5jZhfRMfCcziwt3tnWGVYMxgUN+PQ8MWTbFpYGazi5YzP37BFepj27BBjlRgigvlau06rLxMsSniuqExkjqRLmiMcTmwLi1em0QMCm6fe24c5GEQCrXFuYRkaNKA9M56TD64DoTNxclFFI0Y0ZuLNsx6dD8VuRNQFHa4BpckdemViP0ENJP+\\\\\\\/bpSidCvK2txz5G0UWnhrXb5wnlxAgYo8PgyHciS\\\\\\\/Bc5UsRNc1c49BAOH49qhQGZeR7ZW\\\\\\\/PuIixIt5CjX1wHznY7zFilmaTBRUGiyf5+u0A8yYjOT5\\\\\\\/wvZ2tIWZxWJ9d\\\\\\\/ZT61rjpfyXy6\\\\\\\/GNJsuc7f7hKTTMzvpeUcBMv2CzqgsMIrnZLHwnn5hX56ofZ0K8Cmhc8TPeEy2TJi2dPjmUujYVyK+uQSQChKCc7A3UTYQuIiq1APR12KHRNgq9y\\\\\\\/\\\\\\\/a640jYEItV3nBEBdWTZUSxnrCU6ODKDVevPG6LmWMANcbMXIup3X3ucUsNXaMRBDhMiUmyhSIoIhtzhOAQLC+Ugx\\\\\\\/xi0Ka3djzDQRrGNt7l7g+zFBaLryZNr2l02Zr8g7KW7Vzwwi2bDdKBhuT9QF1nYOhlBxebZw9pU+t0NRrQpXaWQp6a3gxSFAiDnwhl0nO374mTDLBvtvgD37\\\\\\\/m4QQsr8NBYwCCEDB7Jbx79\\\\\\\/nD2nmYr3mnKKq9yg0QDgtgLBzYRijy9FTFnd43xeS6KiZ5eLf3otJQtzB\\\\\\\/OcqQel0aa7kQzsPOkQh1nH50X4A94eJ\\\\\\\/MPFmZqdPmyBbgvq3uNiz3LvT5yJOzCC0WXlSY310hOWT2RWdkoj83ErnpwjVpN2MmlyWV0YMsu0AeZMJB7GLRI9xKsYI14WPrYK8AWgQPXORIjetdL+n1tJ2qGp7Q7x4OeXjJHFvb5ds3aq5WgQ3VEZQCiqSBt7o\\\\\\\/MUpdqHW8pDau67GD+S0siBH6MMNsXFSJyhmdjUY\\\\\\\/\\\\\\\/I1CARZGMUCozgycpGOjPv+yiWi78ypa6JTwRiZY6\\\\\\\/SBrSf6URwvr9fQ8aOFkQKvv1VdR7eokquOiRRIHJkWDlmTCAwFaDNdXUrLA\\\\\\\/bgERitWtJliV1H4C9OQcGBN3ZwqscAHUc0KjYAoIrkkpWdMMzauxIXIj+\\\\\\\/fGNqLsV\\\\\\\/GGPq01eDEIpjfD7xNhn30nCMBFF4F22yVp\\\\\\\/fA0TUJpjVJ3G6dom8adl6T0T2AlJIi84j+bWVClCLLoqYSH0FXOKLHbJjwPdwFdG1bHkpb9NII1XIrWGfoY66pZ++9c8raBsfsz1CEIO7GixUm0Ff\\\\\\\/mBiu2AFi5XQ2zZJKwsXYlNi4YMbYhTSU\\\\\\\/vH\\\\\\\/sTsWYMHzFVprleD2seKuRPpy0cHrZzkIklIteRRivkrKZsuY3HV6\\\\\\\/gTpreRPmuvH5Yzq+kakoQ0rlFWQ5xpxBuY2TWYN6UEHK4FTUeljCnmT1ri70zbxMJ\\\\\\\/5OrnapkPNpqfClpK23E2cUY\\\\\\\/1xlOtRntPghIuNoOg47j4uJCreqGfRTzIUXqAXK3kyWDoq0Vs4HmdqV+es+GgrExXDVxLmhf\\\\\\\/1UjDT+vpWcduU7lqLpin3si30wZA02uNH4trYiUXJraWqafKv97Cy4r0UZ\\\\\\\/l5A8yYRdnMC5R99ubL0owEOL7EXaWDL5G05hYiLNL0PsDpSReQNO0PDp5h2Ze+4mzbd+0lv9CdrGnDKI1Er\\\\\\\/AIAUAKFqMq7JijqELZauoj+2GtkR4oi95jW3a0olLLrCB\\\\\\\/KZEWOL5Q6nkiJo9XyVI9Y01kv2P5jznh2NvtkJW8ZRLd8an1GlTwfu\\\\\\\/6NN8r+GFbPn9LZR6h6J5JdbizRlRu73HuzryuykR6AyKfnyAivNAPoumwlJTcd51vuzHaemNrn2HCqSdezYa314LirRiZ\\\\\\\/ZtjUg8BsVC4ye78rpT1nDBTYefd4cXWNLhxPMNXSFzJcmNF6Xh2xgH03ZLptg+LH2RbxenhZqXdsPXFoPt1+Z077VIXNVW8FRqHjFaEH9A7XsEqpscmAxYdM0UVFqcVkzwsLIjAC0cOEaGouRzTwA15v2Vpzdbq0RNdMW8TVOi+aqrD1d0cGvlqCmrdOHEv0u59jAkmH7aqsWOR8baBIc9CLRoLDgsQzSW3Xmk8Q2wi\\\\\\\/0zkr8KmyoIpTqqMQ8e+hoOPn+E8\\\\\\\/viVa6zjU4WQGwe2XZeParyOVnCgONIv8ZKlncgZ7T22oBUd7bxZXz3vG+aMrFZFLNHM\\\\\\\/YhIeeQT0oMb48CKIBGhysLCKXKCRsD5nG4kDH4\\\\\\\/ZzaOSB1Y1X6uQzS1nKVkMQ9\\\\\\\/35KZt2Z0hIUqhHnHlYbn8jIcPij8lK\\\\\\\/2j9qeBHwoAKB\\\\\\\/jwuS+6XYe5rjyTKP\\\\\\\/Bbvsx09fqV7FJrAWtc1JY+qxh7ef0FGVG4UOhyxqZ2ULOD\\\\\\\/hOQQ4v+FbTtulxr3DnRHp3aC1XWaOnKhpTM3KIJOGnMg5082LnUloEwovg65T7n3tUiLHT+noRsyn+QRgtLCPgoY1r9oNRjW6MD2BPXpVqCF6rF2yx0R9FenLeJrcAPlBfQ+j9njNaEz9OVp9pmnjYDhQrfCKK\\\\\\\/EJtCCtIXNtaZtJIiXWrNq1gbv0dAzXv+ZygbNC+m8es8ZlRQZmAmdVQRW0APsg7vDGqfvfNGO\\\\\\\/Pq03euZxKrpkK7ZaKCOmqw6q9RHyYrPM\\\\\\\/2lTlLQoKRXY8Y78WYzrp9GiFrxYG1kMPbNGkdd2BLIm3eIjpTEoM3ZgRbHL+DE7maDH6LnDgm4HJ\\\\\\\/4q5G0zEtCe1oKGH7mL1pQJkrGwnriYoiQRPVRdKtjnuodHSNLQJGtmo8MjShxAtxl9GQpYNefhOLReQ6GLk1PdG0G3C53F2RIQfZHww5FaXjZ\\\\\\\/NHJnNXto1ALDPMQYbWirnwpkbiP8FsenC7G4NspEr4W+wyJVI8kKAHL\\\\\\\/Ey9ZU4YeQxH2NVo1gC9flyva2udOaiITTRxyzp7bforC9bZ102VYYvHiYvseHTSgVSDF2A1RPDOi8tJmDiW5gIaQ2BUCvcVlywJWwzVBCAB8y1hfmOKtl8L5n4KgOp2dVzQcM3++uI\\\\\\\/KVQZueMmp8gy\\\\\\\/iYRnr3u3ggstMJAipatiwWE6PcAI9SjeOwvgKhW2td7X1gTy1x6QzUR08OsMn12FxKvgJiDoUxJLj30J08jF31xge3ZK3PBc20LYcNoEv49IfOoUUyf6wqm5rCW0bPoxSxvN5PRZLYn7\\\\\\\/GG9PhypJKEYMBHMtc7uhC\\\\\\\/FK3VSK7ToezUoFbg+5wC+lESNnWhFcQtbaQbY5Hi+DC3VqG+sUIVXd5BjKywPKChSGiNJjMJl2dEN7DXca1pDDr1TCAbbr29Q\\\\\\\/sWwwD3Bo8NICfl\\\\\\\/amtNvZLRzkSCd09efzUbyr8eiqNgq6jtrZOS38QV22AfviLjj04AQmBi42jWItSjk6wrqbAtsBIxSGxl5LoZ0cnLMuGigEx+dtowOQPNJN5dIzbcjRlD3\\\\\\\/W1Y1hRVShkZ56DXjRxwAW4x+\\\\\\\/KSxFbzqv9Jf4vyO7VvLPUlDtkGcze8nmdsVslok\\\\\\\/Kl8zcyOPbOBYUIF7A6cVYAnhEXalkztbZ\\\\\\\/lf7Z7nLtOFHmtQ8\\\\\\\/UfZEnNTCsuW0yJBWhU944HUP0vqmJi+y5YXXRKw01nwjWowAKY8qlunjaC4vb2acRBNhbJKEzzOBCP2n2XhM0V5jiqyGmsZowxYZpNdbC1Nm5gVW\\\\\\\/rkBtSCscovXXnff4iYZa8YrH9jiOFDqhdhKR+Da+J6JvyMf7It+O+bwY591eRMqicJTSHfJV45G6i8qdPIaOtrqIYYpMzAOo6oKFRXjxy4gc+3D1XBiD5M0S5YUcBmLIGWtvds8m0PQr5FaY2RpCxRX84wLxmkajyFoUtHI\\\\\\\/v5wQM+cIubIrPSTbZILpXFZUJqYtNePGL0aIE9Ia6z1akUNEnxdgR7BDQAjEeTyrbkjQ4vczpMyFLMPIBinBLhNGDUlO8NCG23JtFY1NrRFP0icQuttZiDf5k2NaHaA2Dl6sZvwb8hSQYMuEOtEEcb8GUT\\\\\\\/XmLtAcKf3A15swojsfXtCS7O559Czf6IyqnOiCW6jre\\\\\\\/XyQTTcqnKOQIt5jslhxwAhWJuKyfEA15kZBfCK1bu\\\\\\\/Lp+h2IsPTSeTYzy2R4ZW7uiQBTwVtAJ6BLiyntmcnNilyDCzvsg6+tSfyoePhsRTRbpagtfEKNj8XYV940R69OU5RCi42oAvmBgVPAQUyqgCGzK9AN3S2z0OCv7zbtgGJDOI7koNTaqbvoculmmlsbxeQ3Ne84EudTabaZ7B2cnik3bLg15TI+9+5LTdTq2lDo94\\\\\\\/ktyE+hRfgrheMMD4dS5ydwhMUFsd9oZP42sE51SPdQZgeAUuvEZAS8IX9JOHmU6NmxLMBYQ1qoI0IxXLgYMmj+TuadWOrgcou5VABu1YtMyHCz04RIfVMqc+MggzN+uhe4tVYqWxEOPd+RDqe2W3WcJfFNxsIkQuO7VP5UIuqHy9YI3zACyqK0cJSQnXRX8ywboYjAFFnM3e89SPy7aPREwQ528u80BRnIR0Fyq\\\\\\\/7xG8sMwDw6eF3MIFF+bafIZFpNxL+v+WxMHE8mMk7mpjfHNSyu\\\\\\\/NZTJIWsfzrW\\\\\\\/iQZJleIdn9cVIIy0FSgYGxtLFdIAMypmLZIgP85SdyIGq5lpx8d35eiSw\\\\\\\/EDRBm9dBzmgALWvExNefCoAYG8ASYg1mP9sf\\\\\\\/7+kJIpKBQyOLCnnGPcBWUpyxVkzt2Zfi4OEzESMjiJlKGUeklPBnW7I6xA35VO8WwF7kdJD3qZlIwXNesvNIFf0VOK7WYI5qqs1v\\\\\\\/67YCTXEzDuWIn8K0ae25WRU6vJhDU9Fn7BAwvJGGAKHqoXKllK+rjF11rYyWCVW2t9mrGOctART5nX8\\\\\\\/YXveRVM2yGZttJv9CEiNsURiZyPD9F0mthje28gIxhibTD\\\\\\\/BnohbtwpC\\\\\\\/Z29HMpdjq\\\\\\\/IGdUmlph8GSUstvwtL\\\\\\\/5ZivMh3CGi7nlL4yf9FTkx1IMS8zGkQCAPyEY2jyQO8jOrwIrybr0pPg7McWXlH+Fp6jFBklZI5fsfolBS1HaR3x08xlOadOlbykbB60Z+9W5PTiw9miO+ptvZUKhczdU1wDTQMoTECM0andj1oxFvrInO2jfo5ptJkOGgRH06XtELVTNmj\\\\\\\/oojl09VNY5gIvc1el6NoO467WnWceGprXRyFiyOea2Yhj0pcIdW0FXaOaeQXSX9L6pEihyRfBJGHKelLD212d36r2XDDb1vRJysjCIESA4TBkMgH\\\\\\\/N+kUcu3gWYsJf+20qcpb\\\\\\\/y3kMsvtZ5fWUYfGkes51aJdcC\\\\\\\/l36rJu\\\\\\\/BUc205pty6veGorzGL7gMH2jnckyUd\\\\\\\/JGRUwdGpAg9Vo9LJuLvYpFOQmFpSkBYrPB9KEjGObGbeWkYWHY5aqIqy1rM6K\\\\\\\/XSQ4eUlWdVHm2d7zdERGEBx4\\\\\\\/ShOmUCIVpoYipOi7IE8GrfyJVf96eGUjAnp2UhaDUFYr2L4dQTx+qZ7k1+aW17Qozj1n6+ocRMGi7zYM4\\\\\\\/c3s4ZF2EO1l9fIctIap3wx1mOGsaapJWmDF7zpBIGBlDR2FUvE7LRRhbmWj6CflGEkRYmQV\\\\\\\/MwT7778MvnrN8Xo+ov9XVtpsvOyop8lY0VRSd\\\\\\\/tC9OdZzeoAFp9kj\\\\\\\/JY5OeqLT8vSWr85uqgkOU3Xcsdt+Ev2Y0cWcdvVuMsj1IfktD2wMbeFHoQKC01T9HcYuYsSq46CXlzEiAbkJ4vepz7So+z8i8ljpt2DDsMP+vafxDQbegOw9pGo+Uc4pyjpC7EdUI3N3hyviFSnaFW7HlXg6vTg4a9UGH5y05U8G1sYCQzEGwwBjrQucG3t6dXeSygrCghfJx8qysB2D0lSug8pXuVOSmnhKVOyrNojM4q07RlMxZwGbANDoyhb5UlTRU\\\\\\\/\\\\\\\/ztzkMR5Kca25g\\\\\\\/tPXiEo8yvmi3aJbJK2d+53\\\\\\\/Txk2e+rJLRXFyminuuPmDFJD5nbcY2xgYqGxhuGUTr8ZBOnnwi59OldsC1J0OEtUeau0c88K0hGBvqfTkw3R1ZNP7KkIG3VoYmt7z48g2k0BNY51ZQ6aYg3pwsrOid0xLVuHYfnPLwyCtoz45j26FErgwCjtSPIWAoPdeJkNQcMqEPeJOr0gGpWVgqmmau4hYfzWLJUtkAfKQEmJ94q99L8oR1skN8tA46FjmcVXknB5EC9V9YjEfYupkYIWV6blQwm+5h4LhlDIqpn0u96KKJ3JJ58zf1\\\\\\\/SpZ2XqHYV6ByCzR3wvC4ph21mDIxSVNpcCF1nFvVF6ySsfl54Gqy45QcanpWk8a8u1poay\\\\\\\/y94vyYcfCFZUhaFK+Y9Q43OKcIxnZ1VKEJG21RgIq2MseoWgpdVFuYZCX2wjxJfsl5wsrvY+u\\\\\\\/9fawNDQj5pX9JdtgQlzg6748GVdRvw4vZm02PLuV6XEwsgBX2ToZz3qzWdWFpX2k7aT+Od7M8htGQHubU2FssHeEfh3ZZT4KdSpn2J106nI2tmnLGS3eMk3hjMExzPJL8COvNUAcYWDiWxoZyluUWPouQmq3t8sS8A3p3cHlQtUKqphwawynetDxlKj+8frp4pwGW4HmSUnTlRvMhvLsmHwDvcBbnak61LAiGZFY7Hm1g0159GgE\\\\\\\/AlYhdEcOG4apd60zNVRb72GELSPqzfpYPo\\\\\\\/GB8dic3q1Yy8v1TmqJAhpF2ycwQXtBvcj4jg\\\\\\\/x8xbBwN0ZaNMZ7Ydk+weqRmavX0zJNlIlmXXJMCxhlu452Ln5cIvzgXdCExc\\\\\\\/svcGmsDT3CHBFG0eeZLQBTQy\\\\\\\/qfeGo9Vge\\\\\\\/SFhkDsktcyfl09IpyZQMCMwuHF3uujMvPbDuCj\\\\\\\/Bi\\\\\\\/4oNy\\\\\\\/sPbJuCMozduWoMeVVJB39S1r4G9c5uIBSke+ql\\\\\\\/vVlNmdZLNRzn3OzP\\\\\\\/OOBZqnewm8UaSMQ\\\\\\\/ZfflXADvBAnVCkEqA5+EC+iIobwJ86Cmlr5gSBvWBbiRExAb8hinAd+TFaOLmnAmf0aQFCxZaZ20EOdZbCac337s\\\\\\\/Wgb9rOOneO8ce+5rJjh4k5cB6roJSjcfO32XnLvQmuc\\\\\\\/0mZhIUcmgsa8uSTTtjOuS+6RKBd1pPWTZMnPRvMIrCoehaNIkMgbzYod89X4JJc2pZCg6CT86GwFImW9AANw5Yv85LZvCaV65TlqqxKFfIa\\\\\\\/oL1fz+6S9AcK0ccBDuD9tmjve96mr2Vqp77o6GSCsF360Q634Q0r2CIeXX\\\\\\\/ieq2BXOUv0vii5UI03CA43QwDKMU7Er6LI9wbvhvDBY8prs2w3e0Exkbj\\\\\\\/xfGKw+YdlA0NB6QowMSfmdLKjV8A7N4zItmM022ztpoSooS373RI78gLCD+ZtVMDzGXmReSQ94CBp\\\\\\\/oYQJwLE\\\\\\\/1\\\\\\\/PC3PuZG\\\\\\\/lvxM7nxtrg4m5kKIacv6gwmNPIpUSMoo2boGyBvdLF9YyhsJ\\\\\\\/wU+E16BcYVWNrXaF4S0ZNEMNMqP\\\\\\\/Am9kQEkXCGm0NcvKUv7U9qsxBvQFtdht6GLJVx+0edRWo5hzMEkqyqNzJAQ6Qs8VeT4BCycQDSciLMuYRVoJGWwZYfXt6zbgB4M\\\\\\\/nMcfJ2tjJMWs5zihYc1CjhZkoybf+1bkKhXkHI7xQYryCs2o5l+j21AEMu+KwyUKXpqCZd7NjjFghNncQkVR3G1haTEgJmz1jI9hh8IFvRF\\\\\\\/WX62CjQ7jjdjB1h6lWNj+PffZWq2BEf2FDgjSlaPTjg1P\\\\\\\/f\\\\\\\/NyHzGW\\\\\\\/9Ji+YpMzmEBDCQOtws5cQGoQQ4I5x6U8CRT1Og6b9NKQvPWvl3xtsxUEbzJzOzpYOoS1GUJ\\\\\\\/gOe3nEpMutl08hGPY05hLHRXwkLGJszxz4ZtK1ZIetGwbShXKGB7FyIypcIKtMPlk3NvmRX\\\\\\\/ZYZxtXPwGOYcYWA4\\\\\\\/Gfaa5PTolCMPFqFz0+SduuVPSNrJmA+3WlclqBNZTTCK+En\\\\\\\/rvXY0HJDynL0TyPVPyhBE\\\\\\\/naCC\\\\\\\/2Fv\\\\\\\/ro61M6BcTzyy9p8SvIUXsm4V\\\\\\\/w9ECqV+xWbXqAuJtQR52a9OV5eiqlErXAa+zgT\\\\\\\/mZvpFwI+toYjwhKwGjTrgdUj95ltjxI+jDA2rsiA56WOI+5d+TwmTaw7yVsb4k79Ge6Yn0gfRvi1+3xkgmX4r0EkwwlcNLgJ\\\\\\\/0df8v4+WEjVpeV6RK6nU9u2pGX+OyYEGlRrtWPQWvrtbQruALGVTqLx\\\\\\\/l7UEfzkZTPucpNJBlkeNLgnh6CZ2DDaadmE8Wyt3Lw4MMdrHH9NfTmvjsVgFIlhES7xt+d+iWFB78QeeOv1+ONtcrxF8HQfaZKnfIA0\\\\\\\/7iLhmN\\\\\\\/yj\\\\\\\/QDSq0zuEu+1IgHFJe5ZH01teme\\\\\\\/JQP2PW9RCylCYGM3PwRCzCJK82JgIzuKEJLjJtjF4O2HFp03oBbPrilJu81J3ETUJikSCOPca+O6kKxJbo6YJk\\\\\\\/8waCdUUkx\\\\\\\/4OX0d0+cq0pAEBAstzJN1pFOdjuIyVqCFO7iz7bux3YFIOprSRTQrxTQE766bPkXrDbGyn7lfC2ajHyXx1QYaGK5nhLfobLViZbtmp381aTrOiO4L2gFZ9qVoSdgDzLW9c1ayionRbDmXBHnmjRAv2j75htDiFntoQ85JVvDkSOQJMr8fYi7iq4ry3hAGPbXWkJYJJOz\\\\\\\/yv1\\\\\\\/GdPQR5wZ5cHYZKug3L6xtS4jti4x\\\\\\\/VLfW998rN8DetH8OmFH625baRoMAJU5kvaqLLVdD6M2ouaped\\\\\\\/G\\\\\\\/v4K2UM4odXo\\\\\\\/7Z9xSyQLqQoSU5tF9sn0wkW84gonVil0A\\\\\\\/ESXCdC53h9pxYDbdAyHMe9cdO+AGetTkLmvSExZk5n5OyL1jniE\\\\\\\/eJD7ozNNa5QOY+CMvSAyKfxxY2U\\\\\\\/JZ+lX5PA9dBPS70u30BTw2oBHbHaigzCwZ+GzLyrAsTWiLNed5EhzgA00zRacco5qL7V2Vntz2sB2GQ8WI\\\\\\\/ZPBYrU5VK1adZAA+9C\\\\\\\/toVxEnKYFE9Ie7u8QOZaJtevqp0QSrKEoSovz+n5hQqV2XwGfm1Ph1u1GQGrHDhNWMeiVGGQ4ACdQ31KNnEIgnmAqa\\\\\\\/v6sb\\\\\\\/VQsfU7Wvke1M\\\\\\\/6xnnjZvzgRxdZdZNOrCNdJY776rfegi45uif30Tnbj9gn8lyjh+52lt\\\\\\\/1OhDvErKFUrMAZAbhJMrLeIglOsexPLZLV2ntPXEPbB+nvexKCpx20AAJzrXiFOP12HwfoUCCGuu51Kgc\\\\\\\/LY4fU6AhjX5mfPx3LCHPKsjTUM\\\\\\\/Wqmp6XVE\\\\\\\/hM3k21B\\\\\\\/Q8FZoe6NIZQ7tfkU3Rmw9cgxvb7SkBL55r7Jk3NrhTcoAz3\\\\\\\/mvTkLSx78b4e2e4XNBxusfDrxLQ2O6FSeDJiANF3bJOrLZHiLKHnAGuSVc\\\\\\\/UIDcI89v5jGtRaLQiTweRhcgPW2P36uiuxRIHCDVdh8lFZHDtJW5O0w\\\\\\\/iMGC5k0KHOH5EEkIZ8O8wemCZrXi25BeNeeAGYey2fcyCI8KGldOzhzqW\\\\\\\/C1h9cY4AOr6LVjHqhTgpc+KhRb7jsUkh81LXwylVgxO7ZecN5StdE2YlHrhUxwAbBwnLt8fTxr3eItO2dSKHc00KK0Hu47ibnuM5iwCcA098IVpQrxw7CCfZjZBaeZe6wty1IaQzgJf2VF2ytJyUITYPHW1MevvAKmoaMONzZldJinUctqeUgkRVDbnXYo5ORj9yrczEhedymYa52M+ycAH2kaFedVrSIcX60j+AwJrmgsQLWAuCZkxZTSarjDPjTL6Ze+bCX4AD2HKLgmrx5fGOZVLO4vutaSagxnImAJs9e5pepdMLjygYxBqu3j922bukd3V3RxYGFksyp8O6WZwq34N8vt3or+ZSJocFO6Xn4eN3WUyQS1dxrk+C0VLEuArnI74aQ2sdnPLmYhupQHKvzb0KTi1d8H6uIaKAoGFZBpuls7zv5QVb2vnjxHz0XPQqchEFGW6pmMAearGVnNNMJcC2zC4a1lzHXUPf3otDbykeP7P1JOqQcUhz7wzUO9o6YMg1JriUk\\\\\\\/cRvHQfzciDqNGz3i2wuv5Ah9+x\\\\\\\/r7I+YoBL9jMYOry6pE10LdqpaZorNf7nXeOr2EiHtcRAdpjZi0NiT2vIRtTmfldDuvM\\\\\\\/l4M4+K4vRUsxcdgCezelJZyxkmDH\\\\\\\/bPalZeDMNrCHnjWtWo+FEoBRrMeT9rqsEWhIsME2jqFjaYUqmfIhS1uNc5cwrqj4fXc\\\\\\\/IixolHTSPxPHoq9ncZoTskKp\\\\\\\/AJu0wl60mV4N3xFjd1GrRSXvSDkNUulAELfw3rOjASvmjD0DPccy\\\\\\\/YZ5dsdqBq5Bobwt7ORLBoSXYLHArtY6IeTXRorAspcexbic1XRMB3aZwxeCi6rQVcS\\\\\\\/UrFrjL1heWOOUSHELicvf0wk5CHwzaL1HK23enNE7b6mKGXxEOgXLyRkULqFMJMEWNE3nyY7nkXlkZ7WSNo6Ihjw+3B9jCMQ1i5FDMEkChob5h7jW6MiqPRTNkUxqw1hnOC5OiEkUQX03QOSQiSWSBFllNTXK74vT1n2XiYyAERzU6s4C1PzxRmnbT7OeR3lWJ1secWhlfPftHTpSTZrTDexcMYgYHNjfBhO6GEY\\\\\\\/rU5d8\\\\\\\/zTnRnIq3BPaLPsHEibq01IIphvkeh73FAIW9EunLDMw0CZHn2DNTcx36js0GuytEUOShMBWTBJUJjoWQh5k97GRQSkEKZs63K8aWv2FKRb0tv6Rz\\\\\\\/olz0DUJWAaooJO2ydLONvnWgKOnMb86Nc9rm0+uIKtyx5GgFAeAm6eQcPFa8dEa25HSYeFFSHq0fi8rB3t7iORfHynQaXmH8t9olUpwpipIMpSvQsV\\\\\\\/ONBx7tsTpPSbuCqYx3sGUEiYx1Gbpw1GgmVkl\\\\\\\/FPl2xTGIoo\\\\\\\/wguB9J2HbMQFBMRs+DHU5gIFtx+ecuzfJogfBJ8wkt7RPkSlSkQe0BMfijy8wIFa3MhYsyLhWsa7ehvLX+P0a\\\\\\\/3nGJRgejSC1xJyXWTw3TOi2YmSA7phOPvfCUi8Hm7kWTVJqz9zdm1jF\\\\\\\/87Ejlc5ed7pzQp\\\\\\\/MLOrlVdi3\\\\\\\/6bGNQyTfbpZuLrrDLLHukKC+AY21XkpGtB2k98H5WqNky4bKFPdh0MQ\\\\\\\/ACr4eO0ZAcI9hdRWnOyQLLV63WUuPBG\\\\\\\/q9YMBjpbOEtzFRvBNtuSxWyKAq\\\\\\\/ahAMxjvfUTKciVqGCkh+Q5cxKgb53dtFhLWHb6oDp0uTOohxccr0YS\\\\\\\/XPKPp377iCvEIBlNm5\\\\\\\/p+lYvnmAPj0TjWY0MDuHBxAMusyXAeAtP9JpN62ZQUOidBwz5tgAyIKE96WA5M2kukPKlDqfdbycxa5AKcKm5a6s+N3NZBqKL2FVaOQyyj8oqrzeXQBG5qeXviunNbQUG3a8e9\\\\\\\/a2fhoNkFz5XsPaHwBC5U3jRb0GaHoEiPV9oaeuZOP0jfz85OfOInE5VcoqJBWdQOGpyzqycvgX7d9D3b\\\\\\\/fGy5mjQi3H5i4EbWAF+99vAJPq25jeJbDtxJ6S97+s+hi6XUEXmJg9j5VS7KLE1GLA8SMnG78jb6JifpOxxDgyoM583fNzbOMwLjCJAtYqbqr\\\\\\\/cxzr\\\\\\\/NcTPou6ON0+DU8BvuN018gZQfpfYqy+8aLYMl+thvuip7X9tJfwuPaxEMKjE0N\\\\\\\/PoThYhHuOvVlwJcil5Yagw0fDa3BF7isrzwRCVuw4KjmYspSRj5y4xOgftxCrsGUA00c+UMwZxG3RJyNz2IEnti8lfa\\\\\\\/majG7dlH1EEx5mxUcyouXHdaYhSl3u3aSNEHP6hNttawCW7nfnj5HsBpClrfG78XPYvBSGDx0d8iGLuMOoCDvTkTsCV1YQMde4GkRkL\\\\\\\/nf\\\\\\\/vMr3FdcGxbxbhaxkc5Qrpj6BEe28UauDaDXVlbLK+eMQM2x+ggpC5O0mdVbrEst9\\\\\\\/y6wsKdZ2crISWh+G4pwGkjrC1SBqkU7tPTWrWL7zCwUIYe0rHo2bEIp9NPjceIZ9lGZsD1VCZnj\\\\\\\/sGj4Igmwj+PZATdi61KFT8sO9hzuACfDe9fJVgEhYakmVJWB8rQDtZiS\\\\\\\/C8SIOyfN\\\\\\\/jrz27pS7rFx0hDKizX1rtX46c3AsjPgDFWwm6Oa7OieMRyT1xbSFrbcy9jJT5wfY+p4ckhLHq05oHuF3hcQCw8tNaH+zFXP1kyQbfN\\\\\\\/IcQOpd4BctOt9LDRI7YPtR7WByl\\\\\\\/YkyjKNwvYzNXKi0ueKBwHRn1jvdDJyrv9QCVdqvTBC+SWDdoF3m3hN8osly0AqeIwIBjT+cjURHCzlS306mXaQa0BfrjsrIHeDpZPv8xmWthYx44DEI\\\\\\\/zKy8cPm2QNu1Ks3YyM6UP4J2bLYFpsXaFJr85pXbL2f\\\\\\\/dybMi8+hgo8XShrFuBd2QpyZ\\\\\\\/TAFgZOHiwWzcq5wW20snLrYIr1CF54KUzfJSk05X7p\\\\\\\/5JU90PXGIjxo5jz1Z+UochGTLJvbHDnPRlQKmAaabIwkWX\\\\\\\/MaMVjKkfNCCO5q5I9EjEZwrzT4VD68Pf2jb+WhIVEPXmKxIZZ5ZmZd8Zr93wz8SgC5PRVmEolX9hBMOK17b\\\\\\\/7zFuAokluwNtdjAPEhcCWNtlBZ76IC7D0nUuhAIXrgSH2ePwZhTzZ+aA9bB5V+YZIm9P2Bc3HkiYzQK06anq\\\\\\\/m7l5SBeMzqQGgJt8laJD1HglIiUnODeWKGFwPnyfM5G4haKTKXOAqyq7h8\\\\\\\/FbQJyS7UKWCbBFl95vd5SsbV1LIxMK+2zq3kHf+pbSdOy0mLMGthTg4WWRLXhKK6jOIB0sG\\\\\\\/DxzCLA6VJwBQR8wRDQUYVMtT7yvOpUB+2IfjICfzmVpMWX\\\\\\\/2qRHQ6CjjqKWom6ZVksBkxFECskgEN+4W2XXiJzFqVsUaFjziXAaWJO1lXBoCepWBSLtfXMCWfq3HURdt\\\\\\\/HCUl1VdXtmWPFDTVDs\\\\\\\/TJBVfVIHYSxEWGmkk+cjdUtCpowrgR263bW4MgC0PVsbvrm9Kj2p5gcq3nauYJl58CqQ93L4V5yr8049h0pkHzpiGmm2+AqOttwwvP5IJwihkMQ+8dId5Uf53AA2wPowS3SVph8n6Q5GBBe9EJCOO+JehqnxtYilDJv6RPeCJVswlkHifxsBR9nriO\\\\\\\/tmaxzvzdsVWj\\\\\\\/bCCwA4oBMWM5lYZATOPmV32RCqeKIXN7XyorWa8GSf+7znWvsiInlbMkVt5U6ONcY4lvuUT2dq1sAlZu6nm8GiW6amWeO0JRBFlHERvepVKqNUUlDtcgIIU0CHzCGDvvT6ZIj8KESeiNc92yLth8V6qt5e\\\\\\\/DNxJ2QxTq2r14+SH007r4tFbmbjCMZxPoZCW5ECcBWe5AI1fBEQ00Ee9xOmF17mTwVm0QiDAwGhkcCp8CJDoua+yDII\\\\\\\/E1fyMwr3Ooyy07N6GnHoLDQSaO\\\\\\\/Ww0QJhr9CLz\\\\\\\/Cl8qhrmFNCrzDXry\\\\\\\/wFzavC6ljc1Nlcv\\\\\\\/fqKozo12mLTaJSyGnTESfMoCqNQdkG13hBXNyIUweOwbbx0UBAmdGbi1rdcxxP0azLGK7R2925pdHA9wL33cEi0SxOtsEVSFL6S64hbo2omHv7lzAKdfW3AYFpkDVUzR8OxxymDQAGMDDRcFMChpFjl3poi51Koj6e91ssbBRVtIuF+UsFYNZUr5M5KsI0H8osOJLPHYHIOBquSBN3FwjSdFjW8lwKU5F8v\\\\\\\/jXwYhvyQSvDwJhoarUeC6DCRlrTOjzjXSdQSpxUQAw22IF644m6t6a7bxK4Z1qAhEdq9tuNnh7KZmVaApneiljW6aKOYDi9nB8AX2ACMN\\\\\\\/asVcx9UknlEmQ+VLl8JeYQCnZzmjgTs8egIfCjcln2FGbMFSN4aFJCGluwE6I314D\\\\\\\/lImSzql2CjgHskjMlTONl4suKbAQsYvg55UcvF5EzXdQwmYZ08rsdxvmP0wfDivDXgT23IZsWXQwJTMN5oMb+mHqp8bpQpRyZLOcuaonvJf48qvnxgdIpppF66wgzzjL7M1zdG+Wjq60uFeJTyUhDA7DyGc3K5Xs9K9jPn1iprqcV5LrjiewDsRT3TQnrQUziLg4zGQWkSGjDlnvpbu11Ort3+4g3J3LjwyHigJ+wVuuKQiuT2PTOodzgXeljxxMI2MQd10q3s2exQ4mtwZ3PSdkJY\\\\\\\/rL9675uJBom2NKLEFr+2VSzNQrclk+1G860i8Uvh91H17w2JzNys0NNchZ2digEX+zugCMFlnJYPjieTpTKC6IEktMCvebd0PT4JicLzSJsIbxF5n1Yu\\\\\\\/ZChGfHyYwzxl+yR+QZQn8RlL3D3DqlJksvJvJuWbQjWPSYjATZb\\\\\\\/TFXtaxxgJOR+KPPtyH1Pr07nLgdmDbIPjD7EP4DrrAf30+f\\\\\\\/Vr32xlMHg6P8us+L8aNmyExn7jjBRQXeT7tI5ZzCYFjTF7sL1kcFWYe54HEwtAjDuq9YhnYlLCc4JBfMf26BrwLccHaSfDcVoM994X0Pv126i2uhSathWDa1dE0bIUvk1D7tCCrph3mU+ChulQcR6dUD+lv0peaqs6aufVGyybE17nhAY9U3aKOVOABK1h3yPxof\\\\\\\/TF\\\\\\\/GmHZi88m3kOp3uKktXMEIAgI386FFqkh7hpkgMmcKLtxL56gecV8CZnoza\\\\\\\/bLP0tL2jsThsAEXSdxb4c2ifTpa0b6gQn\\\\\\\/1Sr63XwUGgq86TDs7tLiJ4urfSsPW1GgWt88tPHK2znoND+WE3uqMu\\\\\\\/RIJgXBFR2cTd6JmPmptaEL83Pq5BH2cFuP2wSeH55vyPrydCl1fqUHrvYO0LJAX8jxgZXnRAFSS9tDRZRzd+JEUAEeNhvI8bxVFZFs4U6bhp8ToNLIcgf8ozjc60knjjYij6rv9A1zl4nvEANmdo7mDBo6tFTasi5JHQTvs73XRHrcVvACp9fFaG7DzBf7ynywt6qZq9lr3KN\\\\\\\/GuR6V0KgUdvB33mHjyRw2LGfBPS+DxYtRW\\\\\\\/LL0CTFkElMzc5jDb2fG4wcgcVnPr5Q+6EOza4HZgz53zrAtzFFH30pCEffbyc4xxgE5Bq+8auskyC6hnnNM5IUsOroQvL1YTyCNWjDns1CTx6UjrTmUZaKr2oDUYipDMyNsDvQnoz29VibV\\\\\\\/o+JdzwgQMw4HPN3o8IiLUHRXpdHP66EGHmGqdhGll0CiSMbsr8t+PzHotg6xdKVuV6fSfOQTNZI\\\\\\\/xKTahw67LWFwhJJm3GJOJCjbL8qBV4Jlelll8pYL8TUVm82D0qFu5pv4LKP1QeS7LpB\\\\\\\/HGj3w1kAsmJKKUvrmK2s8jfYaNTCJuvKPU1NwKNQhMXRCD32vXnI+ILqijSNGUECJkLkzWMYvktXo4W6Bq8GAgnBJfsfqj30GfXid6xLFwh8ph7yKJqX1sRksDnL+C9FQ6lrVtczmExYaRfjOxvfH+yNhQkuEo3W+XGoSkU219OWxJC7uWJ0WnyXV2wL2e+avxcQiSd7CtyqcueCTzGkf4nTieZ7Q26zgtUkcZRP73OCFtcvb+JUN2WGIalShIXapZv1rrS1It07kzyO\\\\\\\/\\\\\\\/y5MklSMPmv62zenJKuWESotYNJ71ATFtIjJdYyonG9pgWJ8Za3S46yZqdqxnepDuT18iRRoGTKffBaB4uyoThXdJIYnJoOWcVKujkKgh03HZi963vSZrVKwSjxTbwHoVQnCOByIyU5rnFgt9fq6klrHcVadCc4Ox5TFloPr1V5vzi+jqRNrE7P\\\\\\\/dgAlFdiH8Odm1XxOvE75rhd8rvyfIsxCcNXtzH4QWKg5\\\\\\\/9pB50FZs18TfSVYQEGduvCD7LOoKBSvWfr\\\\\\\/Wbact7ja31N9dLvQq9pSl80JwnYY4kAIgLGlj+yUIKdaXiQcvr2M2QvM3kUoOt+B9uN+7s5R5X0+S8kcgCt0x2xkkTq5Z2uvKF+We1fJc3+yHMDewTgwBMly6tTKltBOymfhfthvRNRsNDmjsb9q6wyz3jGB9\\\\\\\/VthAaIkWAGZZy2pYr2UE9i12g82N164SlD7tGpsZx\\\\\\\/uQWBmnjIyBPcHuFK+GpqpupFOSa6yWBM3TEx1pMsssrDQ2zIILQmE44y+L3bIbMnvhbY7lq3RMWFBcxStTP5iXI1HcbjHPTzkdUq9MsErfgBtHY2r9enPGz9iBCMCCYMsTBdpZI0xW3WYs3nxaCoyAyGu2NCKxZiqureL9V4WhSjmiV12oyzDIsxxIQe6aE9TfZYZONJtcKKh0t4zj3nEmL0PMvPvpQOLWGirxzQOHlXtkVk21hvhoRDbOzlclgA+9mZyIw+FWBUEN31mPck00C3ZyvzdjmE1vZinvEb4pdDCbZ95BYgk3yeT8pG6ArfFxewS+JUtMrFR0q1xAG1eVEP9IlbieVYaXmbTMreDWDh\\\\\\\/uvMJZadXPxFdXJjmZS75TbQ8F7ylilqvcGP5swBsge6kP7ZyRk7OzLECFOrHPgQQnJdBvt1fjfZ6W6v\\\\\\\/7pehzXff2RShv+85a\\\\\\\/d6i8lMDKK1adhiMRSfxeV8A7o+FhyYMvycVXoQiv7q\\\\\\\/nJuOe3yKRrqlPFFutwYZgMYQ+07dnncXx+y3e+PToisgaItCAGVjvbVd1gZd\\\\\\\/R\\\\\\\/QtLVmGt1oQh\\\\\\\/W85ZszLXsQDSdERgi8i3O2GH7srJ7ovXY07yzfc4J\\\\\\\/oxTBbM2\\\\\\\/y2\\\\\\\/g\\\\\\\/Yh+NqvDdVf9N1jJVyciHX4kgbD71fyw71h4rGq5r1bqZJsaqhe6uVJRmlAwTHpUbE4xxZyKCO1P0Br65dN6hZiFAbW2F91V1dh6vojHK9NAWa5zS6HbKDnzF1laEqpPOPWIUvPCQo\\\\\\\/f9FonzacdkqUZwTIDl+j5ZwQPydPZrh8BqdZbqtoSmEauerJ9dqH8vzQ6R\\\\\\\/u1zz66tQ2\\\\\\\/VpJ8Mowu\\\\\\\/57kb2JhVAXRVskpR+MaLDFqpUoXThxK2MgIPW8jEJQ90\\\\\\\/+GifNHWKOybiYpZqpTipv4udViSyZw3JwuM008U\\\\\\\/ERiT4QL\\\\\\\/cE0WX4QhpiZ+0SwJzZCTJbZ2QA0QA2a0qukzaiYZZZLqg0n\\\\\\\/jV0JtPXgAK7GQiw+XEpe3EJIr+OJ6fopusW5cExX1rLSasRsWMZI99n4y8V\\\\\\\/l2q4F98lmu9fZS4BJ25bEIPqbuFERqXBU810x2Yokm2hDFNdmoH7E5IY9zcfwvci09V95ZraIL4XJG7MxPVwCnEDmr7AT+1jU+u0x1PxZKD6DaxYM0bdxGn082nS7xvbW2Eunqjn04pxHr9ue7IFCkKEdfaebYmwYhHYCPdLncjl27STj5mulPCbrkZT7i6OjUcVbZnguAjFsnhwwHTDXErdDEY9kAZdcaIWCB3VspFqrgMJAESWOC+fx4NxArep36YseOFmRBzbnCRf8VrFySngwLXD+Xg\\\\\\\/bJlgHPS2nXwSMcZqjhuMmCqvkMrZDj5Qzbi0mx3Q+qlLuRNhKNIvzpbnOu1eqJZrcHPSfZEpNnz\\\\\\\/eZ5uplpwnVjsBLryzdRD2GvdsZ2HrzchGBgtSsWxuDxu8g\\\\\\\/cpjeIgeULkRrSlWIoU9erS6rMJ3JMOiIZ2UaErUYmKy+A9mwNvflNWQ2js84FJc\\\\\\\/jaONpgvRqdBoeg7FX3dgep+oMBji8LiHLaawb2sLie3QYxW0W4up+NZt6rwWCHijxw6DQZ1G\\\\\\\/xvGzMCG7IqQF3eCiYwp6T1FuJSctDbUmLjst6YQ7s11mUCyI8MNGWwHRzBtxPETaVTCsP5uYW6UcrUWYd\\\\\\\/pkgs9ZnHNIoRczqBA7fr4wdlaVPB4YYcaaM3U4wJ9YeVF5wD8I0fVwPu3A2sOtk6IGgk7BWL\\\\\\\/cmZJteoDB1ZZ3Gi7wkGgd62NWPu\\\\\\\/hlxfdYg0FBXubtDQhW6\\\\\\\/rHGpZfktEVVFEYUCiQnQxX6JZ0QvEbCw4l4il8JEnijWlzkur4wwUq1s1zpBhslPSXIc2IkE1LiaMyH0T\\\\\\\/xywc9Y3l2HPyDRevEn5neNKBgBvbv5vmJrf40gYxorWKAmXTdZlxt+2kGfLvTuP6hvtNYfYjhTXUIYo3tIAHDbhYE1ZdNGqw\\\\\\\/uoQO+NyGQ0kezL8RZPhCtg26akWdn3bvGx+xfGS826DZnI4I1ZQuF8X8vv1WAXK22lgxl2NIt7ixx9gmTTR9cQT4DtMtnWVesY\\\\\\\/RXrcj9R192mh2vcAruQZqQWBdm8xagMv1SO9c7QmbaJTPKx\\\\\\\/8z9yORa7ClCuUyCPfE+pnNV6C8oXIsD3RSi7U2zjdTMuX54mItrOA9wCmA6VRMSpfmt6GBkSyo+VMyhdfTIqucMS\\\\\\\/3crEOVyhY3SegDdnQQPsxopODk2KpibbdfCfueWR56ReGV921hIIUPp8T5cwBDt2wMUM6p7aSX370b0ecoK1saVzbm1CmRich\\\\\\\/Xn97NXraFL2DLvOSIR4d2FJp8I8xLpp5csqLmGtc01tsuTyO5\\\\\\\/BNBPLmaKPjOJE3v5mM34gPX5hcXnxd+\\\\\\\/86H\\\\\\\/6ytJ\\\\\\\/StDLQqXEHGNQwb\\\\\\\/obEbmPBBcfyRqaiykRhzc+HjYrRQw3g4IOfjzvlXKfx8CdpiNcqUnwN74tcg7W+Pp2MGJIXSGBrd5lX86WypxtkCHQJXreAVrkKPj78WjWw01hPWgWxc7FI9jp+UoxRsC1CEBzkcs+rALf37v+50GHn7UlaEyKtRP3dSaJyb\\\\\\\/dKjaTDKvgKxwcrLCovePckYP6p13\\\\\\\/oGulnX5MY5b\\\\\\\/P6pKRT8cwR5rC76i8Qoo4dYcPoYv5JnxLrzAQFxOfbiOeUzewUjUstkbfA39NpuCGM6MLxY+rat0sBQoQ9Bx0Jq3See+g99NSp0U2pmP8zItOMoIklVLJcnfiniBlKX1Yb8cPT2d7R7nrJ\\\\\\\/cv8KXVgbEiC2tC\\\\\\\/oTh9+PRNFGcVCDSdfDe2qZnniSKQbZ8VT7Z4bMBFWX\\\\\\\/5oNcl61fkagOHDN7oRT1qvQAn4dOTD2spfGCy6loDKdtOUwV5DYZJ0cpcTuTIenZUrVfsz\\\\\\\/9LlE1WDKdMUHYrT0TTSESLIJlp1ChwtK1ln1F6LGME12JESV2hCD5lBSwhIGfKcunhqM4sQJHFvVBt\\\\\\\/oFvIkgOyQeEx3mOOsnzlXakjhQev9eHDoKbnTLqYcgYVKNDdOMoUqeTqrUX\\\\\\\/DCCVogTHJzoFjmOChik8lYgZ3EYHx\\\\\\\/rBrJxo9Y0\\\\\\\/I7viw912NAd6e8pOuDYKBkoGzs+vBgfnRPB+ovcF09H5vGHFB3MS3M2MtizK6mVLNkBcOAgKr9AXreS\\\\\\\/CXy9kAAaL0Bqeej5gG1gDosRtoMRSUZMI77s+FO0A+WHJDeW42rBU5DMa0+ldugtJMI6sDjcsnFjbF5zCInms628eZwPqnzwgXgUVClFkix5ZPlFNNZUKioFyGhjBE2a9eYclD+wg2\\\\\\\/RXV\\\\\\\/HN3NitQe9\\\\\\\/qsu3eQJhl0+H++z72JiOlyg6McentMtferPsPGmgOrNUFdTZTTfH4aK6OX2p2QmLMhCP2Va49lHtF5JxwoTITkECN3OZtnrW0OmXIDTA5eiqrKcVgfMARuRxYAVI0igCmcWhgVdlFQ9pRQuvgT9ghenPybVPiyNAQmblnsm8rAWgPN1YzGd8M+yrpLRRV3g05OzUBj4GK1dh8avh8HWbYlThMScDv+6tb5G6TZERp4GE4rP2cbm5jLRe90ZZC4e3kMN5OX1Bc5HetepNkVt6zVQfFutSz+RyPqVTMWhYZ4RAoZhLqo2y5mRik9esNnWOI8N5D5NWDyM\\\\\\\/5AVBTAvhxmSPyi4p1Ho1kQEwIxPcZ53R3cFSyuHOc+enz2Dpv+5j1hQfZBXHZC6u7slLZsek4eJGCTiWG0Y6D01aciYDYC3t49K70qWVtJE\\\\\\\/BX6hZg14qOwIyQJ0Pfp17fOo2Bxss\\\\\\\/hIgmsh3equc\\\\\\\/H8iek9VO+VNOeLykctrmpfBT920sQRAdaM9hYhHHBKzlTAsFb1jrMrmWxbsidH9lOUjttfFZPGo\\\\\\\/CF\\\\\\\/MnaJNe3n4rUS6xJ7D1xD4zaavTH\\\\\\\/C0v1+pqm33DIOA2TvPhjvZSrJLxx0jDMOK38Lqr9nrBqevk2y64sqmAmVeDDnB3C5a6YDoyQV+RsEqEfUcjFXZsyGSBvOL5E5FwzQ97q86WiuqrLOVOyAJ9xWJaF2\\\\\\\/ibj8vz0qSmxqgV18TLDDZLbbgmHQqZlQj6zBR23IyEMAj+OqZv25h0HLG1jLHmPi06V\\\\\\\/q9RY9iAu2lfOdb6lJQB0C\\\\\\\/\\\\\\\/aJ7dqqOWpYD88flCWXgVj96ePsUKVju7K7XCKjbK+IhQpkxxwQkI9Se5Oko7hvaoK8JDwXqWnGN\\\\\\\/zo+sewFzUaF4s8IpBvjptQX2A9gJPnxu0c\\\\\\\/KBLEMWhh0kD4LaXCMQF9VYFTSN7wijhlascBVgb+N3F1kJptGq5TgKlQ7Ul6z4PfK+\\\\\\\/Wf5FQW3AYB5iWHNicDAU\\\\\\\/f3tkZnG00pN8BTrH5OubcnI0V\\\\\\\/u8MwwCLdjGCfyaQd2fjwWVU9cHrbo47\\\\\\\/8dLOBlH1bMhk3nXGDZ+mn5UpY4V\\\\\\\/pPI1lb5LKtX7FPd0t482WD3g0FrwMSB7H\\\\\\\/AOPY9j5YsELDufyTtHDrbtWkDnnLuqw2zfqd9OLYq+4p9WVlGvNk2DFzj2hwusyzy80Pd86uS8tC93kajaEesQMsnDaahc9nzGevpk4z9bqmndxIc\\\\\\\/XJz6X2owWkPR3qD4ztXS2a7fol0XTu\\\\\\\/wkSWVQEhAU2oaCFyev9\\\\\\\/wi\\\\\\\/wSBpp4XtIkkSy9maA5yKBp7TqsBi1nirbu2CxuM6m3RpRa5ONQ6TX8VSxJfFzYJIrPermR1cO0wmvxxc0MX\\\\\\\/\\\\\\\/J+9fYhwwHymRi8KGKlrMYU1Y\\\\\\\/DoQEzIUsWDwEeCqJIIv2WtRoWgVY\\\\\\\/dR5nVqTEkw8IL8Xo8637D7F1HnJSLxQc+jxDfwSzUzt3UE9NOl5+n7scWVxbodppTaSxFaURx6o0kD5l0OyMM4XxPqb\\\\\\\/+iM+S2IIETRBiaNg062ddUcJtwgyzuL0eHlMGAsLAZEdcPnNzn4yxtZSMluKEop4bTADZ79LTWRMXpsLzUmFt+GajrgOnhDeNfwuseC3Az3fjnqIUrTjXAiGEEnTrttCNNex3PKj6r5fw6WBSkKX33BBru+F+nH0ntI5iEyGilSXoTyq5bHs2zTOJgHUwuoVZS33Ic+K3eWZvhPx+5yN\\\\\\\/ixWXgf\\\\\\\/yn27xRC7s5I3ZsuZBH0KaNY\\\\\\\/1bByB2u8l0g2LTdeUthtL3JAyykpCuRD6oEDs081l+3OXk7oKF4kET1vnQIKgG5k4EoCMa6+OKZiy9m0QioCRpQi0yyjC7yMmKktHoFoSeo6xGtqFm0utlR3jYmZaSf3ksRFgbDVGxrUb+H7+8dznETTfurRux\\\\\\\/cQOgNgU6+1QWBoGYioMc8xmZG+1mp+U8ZYAf1pFWJLbfgBjwXYaaDJU8+MbkkwK6NhzuqVJO\\\\\\\/LMHWnj+wrqNbpnlButhpmUp+b9bhhvtmMntPipPjxdSbB2nI2hTDfU79qaIJShifFV9Cwc9u1rew9i6SqqM4UbUFROTP498LngyRE\\\\\\\/ykqfimIDOFfxLIJlBDOT4mDr4Nq66VXJYe8PdFxg9p3UZ9ZTnRbldO+fUrVU8+0cVimjoY1cTulJ+MfzcgLt+WFB3nvE2r+ganVxcZ0lAoAOubn5NzDbzzbbppB3cqDi1rAB7OIjlyge2RfFCcXkIFwHxnTv6vs9anlU1FdqwS94YlALyPWVdELi+FH9505LLGbgjM5ROTdpoOHbXtQRl9PMoZ\\\\\\\/Xnhh+fuJceX6B7opgZr5\\\\\\\/nz5sO15m\\\\\\\/iMzSXA+7cf\\\\\\\/c7XuDjxWELkVMGwDkXzj4nty+Of4dJJHqnotZy+5RRZPvxQQniHs222E5Z3xarIPeBTPmFfQx6XyjsS7eOzycmlCASdxRT6aOquDSOr8eQiF2SNv5eOczBDRWENuqRqkNnAAGcPWgtaA4PiDevKQqQ0C0sdq2h23e\\\\\\\/\\\\\\\/4O8yvQvN9BlAEkhyPfEVBeqon81d9MLe661VN5Xvz741oyv8+vsSOksFGk2nxS3Lzg58t1D3EVZ+Uzu73j5MAaL9JUV0qKc6wjtOusH5c6ETbtvGgbOxB057gWAvhMwIHSWEla4i09tDnRjdVyF1zKEgJN2vp+HkQyxsPTHULrmwUekmx33TBcbUuJ7icdncgZbbCuD7uPZLSHRfljXb+8w3ppTeP5rUTwXkrLMIKcarLxZbEKTSqFHzc\\\\\\\/2qGNcXx6Z\\\\\\\/CxrLIQ98dQRQleH6\\\\\\\/jgNQAn24698F7yTZIRQnki3WMRuSW4cFomKeUo2JWb0FZt8cVewHDFT5GGEOc6Kz8a\\\\\\\/DZnvQ27MMRm4xvYga3A7H88V2VoRItJQwUDQdj2ygQ97aEdYnTleAxU66+NT9+M133otoq10eFiX8SOvs+VWxty51CbPK8Q\\\\\\\/yBLJneTWejnfCx3eE2bCUHSNTzUixhQOc+IEjihhW+\\\\\\\/eM7kldJQGg+7n6BYcIXl+LUg2VamScGiHW+lFy7sF5iEfIPuldTPlJaROoUUeM\\\\\\\/an\\\\\\\/TAw65sP\\\\\\\/w6nc\\\\\\\/v5DjVWVxC3dpbJLeqGYYzx+TzdyeL5tOd3QI+zy\\\\\\\/Pa3NwAue9VOAJnv7ABoPj2xWev4WchqwJmEdRY2QrT3U4egxkhiMqMHBuyTuPHgep2TLTaOa5BnrZPIr+8C4O5nC\\\\\\\/zBVoL6E8nOSWSnPMwU29+K8dLIZsdxFOiPnv0Bp3Vlo\\\\\\\/wfQT25UTm1Vb4HqfOfHkUdCObN126ubYZdI1GyaZ4hDa7xe3uIGWHpDHFDIdfzFslrNImnFcYSAtmJf\\\\\\\/XqnJKOWMDlqYHdhuLNhK8Xg6R0zt1jNcH\\\\\\\/j1jvSG1UJ+OAoIRZIKlWhhOe\\\\\\\/bfH+aVbIW\\\\\\\/obHdiPxurXSIfBUbvPxaKvupOIpJen8rELEoMW1uX40wt4ZoAl51fu8Pbh3REEVIjXVy6qmGgZTqEQIHMvXAo3c\\\\\\\/Jl2lU9i9yf+kZEbmZTbjsvfkWVSJHTji5WGLu5x65ofSRrB\\\\\\\/JobbKdIfg3SwLbAXk99NUNm91l4jlx4Bey1Hp5ILRJTAGk5eN1UzAFymNZSUwpvcgRkupK4PwDwuV2RQqFLKWEfUpG5yk5V3\\\\\\\/Q6Pqd0P82EbnLIAVfB2t+FrHHXQy2wRnu7bxkFAuqyPjEsJ8NraGbAfdhQRc4hKJlqgq320ZdXZf41ZIzsljyXzDNagUmDDWfWlClpIgqNx8AHdTIESgf18X\\\\\\\/sqU5e44uPD1O75Y9zHoduCnwVP\\\\\\\/z86maye5DoN8waAItsVfqA9USxl49evrlMzb74qd1Lrw7Tuq7LQM54J\\\\\\\/c49nGTvHmgmg7+Gzo3aj9Pqznekop16qUtbmo6sWBmeHjS8Ogbx6DtExIK+V3n5Qa3UbWtEr28s4brm5GB1kJ7T8lEA0hwFTaaRWRWVWSZPNYvy+K2LwgNc056ZQt7h7ySoYTaY58VU+NHUTgRQ0qbV1SwBWNRUTTgLS1+w4sjyJz8LHl94XZkCnhL9wPJdt9dxO58K6+JW5bikDIt8N1PSRUylfF\\\\\\\/RaU12i0gIKB8KosH31mRXceG5Lt9OWOISWIppCrepATFQ2SvPATSA5O\\\\\\\/Js0w4puGiYXZ8QZRaBoCmgzaneoinI6ix\\\\\\\/EI\\\\\\\/lM1otyFeQulkSdQkMrW4wx1r7daRNyD+hvXXSNuIlJOuIhWHWHolqV7B9jdISt8JzbF\\\\\\\/IS+Cy7oa2sKvQ7EH1Vm2+4YbtkLbMxEq3thylz4jWqVQHaBDT9z\\\\\\\/jntu87ojUbkeiw7F4ezy2LhztBZSLqlyyNd5UAp6gMP03lPrHX0p+8cv9TKyA4aeNkwHfGcr3OGpXIGprohBsbEHaatbIeVG1ekbYDTpxLR495x2T+CNOlH8uX+mrB4QHfxQRa7k71xmvhJEnMRvZIpnxNgwKTLkpONHJyFjghQJhkzyb+c6woJwgMSWlzKLFoJTeknVtWcKGNxKdAspx9yLfiuTma4YIJJqk\\\\\\\/NcveXRxaLIDqot5yBq7sfyS0jal8QheTCuP7NYD94+MSyGYs4fAlx6vUOlhXi0wReHq1B28wGN0obfqtFh7\\\\\\\/572z4TzZNuzmizqBIq9f2i+Pxm2qR9QD2EuUqrA6hGofYEHrWmZav4CjefmCcNJwpzQkpnARA+S3sLNI5eZFnL6+\\\\\\\/NkdwURZ2smcO7u6g4Xlo5jPB9yRMV+w06CSkr3sZpOHi2tM+IDd+PMqt9ogd3vwr7c3rESTDJxVX+z38Oh0BH7YI2xjlQXDC7I+VB1y4pW1De\\\\\\\/0WhIc+DBO40yl\\\\\\\/2BS09puv2hFPa2eYzJNIoYM7OGw0xrhGG04+uohmzIepBAOnRjl6RioSBnestYV56BaycR4UdMDryquozxJrVjS96i+WGbfEFSRhbUNX99jip5LDvobbtr5AVp7pzDY2mAJXECQ+wtzjfN0BWCTZyEydhj5FjrFYqreHxkJw1VqRemH2qYMorq3cGeoH1UJPnDooUcn016x3EVnJuV6W\\\\\\\/0bm7h1+HmINz+EABC1SfuHyJzw02+Sf\\\\\\\/s0vEXSCXhmySjPQ2VY8zNfLyzhXTvUqJbRfGD6Epr6koy5W0WJcWDxpmTHGapJDYzdgSt3b9c5xDEmFWotCJOKMdWC+S5J2N5mOphmcu1znQveg6WvNc3ssMwc6Lf1ILdqSI37QC1t4UgG3dFsuruV1tgXE\\\\\\\/\\\\\\\/u4qCkwYnuRIhN+SokboRTRuzPSZI\\\\\\\/kXxrj3DM\\\\\\\/bFyh0RSEXy\\\\\\\/4MBQOfq\\\\\\\/mUc2Z45l4M7+DpAv2Mhq6P+JG9JNXUpU0Ke1owDW2XrzDURC4W3E4HVr1FQcxfKCXBENy8ZhXoEElYQVJUNpn3yfeLGMjGuCHXoxPdDRLQ7h8ON0b1tqlrA32zCR8uEp5KvZdu3e4ua8H8Dsgi6JArr4BCjhw2uTz5dZr+\\\\\\\/6z2M1IwGFc4GNkM0mKuv1cn3tPvaqIHdNxwuJ7sGVOstgcDFxEttd1gi48n70CO4jmxuQpXcSiL3hG6WbqPELmmHJ8\\\\\\\/9lG+WEoZD27ReHNCSzHLOlo2\\\\\\\/zQ+WfhIGfMjfta5OuYvNzaob3gy8i698LSGOgzl0x0cEEK9PYjjTfXFJecjMSpcy7tZC5gfGhXNl+0gSPX5BEsyZ27vjM7xUltzXLhDSSEciHaH9f1YKpCbbWJ+MLTMyfpBa3p\\\\\\\/1Wx1CUfyVXAIPu9DAgcXL7SHUsBmsf4y4Jav2jxvYb9GfnrIqUcsXKp6fuXfNKcVjkQ6hURd8YHaWwp0dY6+R6NGuK6i8AdGwX329pm9mCysTZBWADWjGJT2g7HS4xfk7LYq1UkPcAN8NViLrTij9WEchkRboZuGDyK4V\\\\\\\/9tVUmQ\\\\\\\/LwXyY1I1y4Xsyb9PTCXQPwk+eyAUtIdqH6o7q+uuGW8N8MzRR7kQedtS6VeqDOhom7Rdv8dwoGyiTTIGznSaNsvrJpEVVtJzzbZVCw6EJQ8gctVklz2X4T8ULZxp+ilYQ5LkEYQou5A0jpiestSBqluVq3EHcY+f10OwQjsrAvNhqIJBUQ1J8vxe3CvjH91yFCYsV2XQDhZmgEfAzKMEbTxrE3hGuD79kLJA7L9kBIFwoUWfFyComjQNFlEvB5\\\\\\\/z1sBZfYiYXdSCIdZ2SMuMt\\\\\\\/fh1aZNmzDbUVjWS\\\\\\\/t1D\\\\\\\/us7XwtYF7i\\\\\\\/Txowt4ea0DP4TxyjNudIcx9XH0ZLj5MKVSYGB7N9NCEIXhHYPSf6oDNbh2m4qCEC+sOGg3W5zOnToOJ\\\\\\\/9dy1uuGt\\\\\\\/HcTxbV\\\\\\\/fTnYuvhbdAQ4OABmSxdokYdtFS0BUacZe+RXmyV5vOhvYpJk\\\\\\\/bzSAheL1xWFGb0Ua3W9+4OjuziCKLG9RJkZNowgxfZ6tRtFf7kCLDnCxldvrZuK3xutr2VVGj5fpc6EePtfTIziXaoZlU2zjfUvpk6LaDA92zEOk4xMKFnFXxOFanuZZQEUC+nTd15KjW\\\\\\\/bRQbjrp2cFyLeinoSyoFkGPwmc2m+hMC27fijYaSuqXYQ5yHVVsUp4rNSeKZyNGIu\\\\\\\/JkaWfdegOqkMkNj010gB3ivRe+oKo7zEeOEYazjxQE6JyeqPP+TRVw251onkBKHaeUqXSNUamnFgPZ9Ff9CNm456FQkuUxrsjOj0kk8eg\\\\\\\/aFrc17rOFlhcclYYBCocwzKcMHC2SsSah2zgMOiWOYUItSUDeN3BVvLsax53a\\\\\\\/bbjLoJ0MP3FtWkQ4e89D7ndKSmM943kphqvWTjn+mBgVFpAIvV2yOdM8cBpEWsOWeBCWovH\\\\\\\/dCdZupxkYgMJQfE0qLL8HVrEohZmHQdzsHw6h88IyEsEilAF3hg+vhE+gSjpo8qmpC3bCc25mQkVfIM2fCBgKZioYKwpH0Foq+38\\\\\\\/xstmmDYkOLstJKkdr9y6TjrOxkJJueOBeTMhEamw9MqKJfiAthWzgpJRyncVNOvZOc9AIX5qXuFZiEBOyx+oeFUT4IdWgXIK0HTfZQ\\\\\\\/Epniwk6K1Z5Ec3IswQwl4lVWdpDAd5iANAVFLq7BxAjSosKu4FFJHOmEKadDPfGnbKbzCeinniesLyi9aXVK1bxRhU+THK7lXtF8pS59tzuE9JaAVhTdW9S0qwiyKGv1lgUD1J1SRC\\\\\\\/dS988RgW0JbW8thYeiNKcZfxgD+MWrp\\\\\\\/v\\\\\\\/ITJadxRx4ke4PgUZ6w3PfXFKfW\\\\\\\/5s32ETA3mPqTm0omypLo7WWIpscJYvcXqvnii0SM8f3G\\\\\\\/CqUXFCgVuFReFUfQGlLwMVjB8E6MJ6ryyE4n9HEclHyB3o9IcCb+VAqCeQcvf2IVQl0t\\\\\\\/Rn3cWtUSSaCESUoSZznc304lXwImRdjm5PY08Rwyi8cu6a+EAEPV5V9aEZf96bucUPC33AtyFUthzc0H5adyGBuHAEBg\\\\\\\/HOWYUMA5Kj8pxhMhcW+ugnlxqak1cs2XQQAwaQK4bEMM8oeRRHHBGzwH7rAo3Lt2qKCOkIoKRz7bEmyGN+cfTXiBXK1jUqE\\\\\\\/UqYbku8t+Qn2yz7hPgnal1V8JpiiTB9ebDlo2K6IGOAlb4tVuzuVzcxJJFakiMcOhKth\\\\\\\/992HSbxvuLDJqpRHh8ZvJe3AGHNDAkhKQ64LiOhPI6JAJWmYUJGXmV3DqyLctLM+JmPnQtzlTnwC\\\\\\\/n6xIGjjSPsW9KwJy\\\\\\\/zjaKO61cWnRBWTzP+2MP9ne942irS+WCRW\\\\\\\/ckwyb2QZ7w6O0SHPGIHnkTya5+GNKA913PRMk3UojFKWbvzfSBkPXUQea9beALXsReoY\\\\\\\/DsMm6NudpuzhB\\\\\\\/2QpAlIG4uxuWPI43RUxupGMKEJ9c97hvDNWwU+keDa462jGzkR4g7CYvEqEbXhojM0VPs12FSlS5Ec1gTBA6s2yn5Vjl6S3l8dNAgj9uolzbVLsEsNgfBuqwEE9TzJ8e5tih3FaB\\\\\\\/pkI0ny4+0D5\\\\\\\/CKcSqeFbwNJQRMtU2z3AFDjEdWh6+MLNSLrGj9PezmHy0inxQDVo+oW7Sci9bB6\\\\\\\/c5eZlO5y3NAa1y2Bm8qIcZpIr4Arqx4u+3ZGnScnI5SyxRy66hyARWWqFh4wb\\\\\\\/y0XV9lYnAQy1wrHPnHPrIlk2CFJwXNjVIafrTV2N07lZX46NfNPnXOPEj1JqOg9w6ajAIS\\\\\\\/ojSk6H2gy27mhvoCRBq0FPMmT1KUZRDMB5H+av4lBVG7XSx7Hy92EcH9gYfAziPOEcSRBg9jtaGquonPwYUgW2JYI\\\\\\\/W1wfzl3FZ3ieVzSE3\\\\\\\/MBGRIpuN\\\\\\\/xQlM9f2q6IK73euwcsr+jfQowaf2W5BKZnC0ICBZLFj95tAsbd+L9p+RmLJfIf\\\\\\\/p5CU2ROG2lh8\\\\\\\/GkwR8IazXDKGwreAMf\\\\\\\/5f3HOGo+hoSSfo5tbqt3PAQb6alGWgInGLYC0WzAFMN1IFHu9saDDC5qQ1Dy91k8bBkeF4SHImkx93FeDQj0GdiEsfbPO1BRQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00224FICPVY9uYZBlco7+BybNA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eb4841d7ba0bafbc5dd109f17192e3a84620b6b69e76ffdaf115de4aa5766430\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229Wlc+xSr0DIJ8frBWiRLjv08btxQfnaSTz\\\\\\\/JVXNFvAnAXHSEZ7S06oDlU5BDvyqhlqMEa\\\\\\\/3x9a1Kgp7fvfN\\\\\\\/CMGDgVrw6KjQiawa+pMUL9phFbw0nuoNEQpQgP8YgqH0OtoSf9BICJSkRESK82X5TH+D9JN4MWJ4v+xSLniG0lZXUyuK8o6xaRPT2lm0TlgEMGADOOGHWB5A4x8VMZCkml6rywyEBC\\\\\\\/1vNC3fr5zXYpHdMRCTZECG7\\\\\\\/7MzVRODyAd7O84A18m7q5DgGp6reITF26jLBUJCn7SuLfzmhhqBLPF4kVqUFaFuLeIhPDzyzXPBtO1Of563YAzI2E47l42vLCWHkZniUZkC3f1fRGP9PkE0Yl8JRFi1L0qSRw2AunUDcdFVnB3UhAyV\\\\\\\/S598ZrjJaY9JX1nCqtQilQkrdgocjkidcYiFnrgGJTIwFKZHKv5VmQlsdonD7kRSKFsaszn6uG1uR4\\\\\\\/P7mTAGOpS6O8DmudfPcAg8NkOr3fu5BPdkb2mRtEYc8hLe7W90+XRRkQlaKVHnSFRPthRzztyVjcm3OE0a868fmahrXfcs+Z5Qz7JAM69nONzG\\\\\\\/LAvc0ijVVU4tMPn8aWB00HQZcijK3rDcutk0IZXwcqjWt9xx1F352vqom1ZTi6dCFJ4JWuDANzu\\\\\\\/caNSlAfc32ccQSVaKrFMpJVGgMTZlf1Cetd+zHfs0tVKXgBQMOmWobe3\\\\\\\/7VTL41ZMVsFo1a+oDUZI7igcW0KLZthxAMYouNfDXPrOA07y9DlND9g\\\\\\\/j\\\\\\\/WGwhgQ5UUroD986VkeCtkYkc8q2oCuDSlnm9j06FEUDT+jGuy+kCy0Vm1r\\\\\\\/UlIHJlQN2L1zb+XYDwiCgIdhmvx+0O3yh5enlGlnFlKHa6W8wx\\\\\\\/loRCNgibiKebIFAYTQvwTVkOxS5c1clPRm5O\\\\\\\/5INo4tlgEg2WhVbcmpT8I5IObKAsFtUJh\\\\\\\/k0jyGYDhl9pmVd+ujth5Anc9p+iM\\\\\\\/f4hc\\\\\\\/GkqgbJswVj3muJIscZGN54cgdj6UF1aqE6GJPG6IOQSU55JH\\\\\\\/6sYbumpxPzOHI1MagYEIB2f\\\\\\\/3VhXC7Xos0YZBzh3DEP112Tsv9QG8Q8mo5r\\\\\\\/HoavVw1lY2\\\\\\\/V2JuYlvnSgi075z2zG8J2ElD5U+VciFS\\\\\\\/Pa5eWb8IIhbk\\\\\\\/G+8zUZuoPbhQv\\\\\\\/SNGwwC6A86Cwlm04+yaJ2BgmV+\\\\\\\/XzSyGxlEQVGocUuQ0baOWUHH3dO1FGSua2lEnliNEBBDBJ7rTTeRYEpWTnkMpznvqLYM9e9kVz\\\\\\\/Zqk6bym2Hme2t2w0UPdsUu67pZPpr5cGOuR+y+v9PcI\\\\\\\/NH7WyNjUu4+FqWoO47dbnYWxolQbcHLiHypVodAtECGUPnWpNc\\\\\\\/bMBmmQsH1SeFAVYAj+q+opEF7R6LtKOJuMGMgqBf06sxkoEmuY95nVbJY2EjJfN2oKiyyE7\\\\\\\/AvhAyoHL0738JsWZZ6rttZQI2bMR1UrhBZTSyZHsSyBXtMuW5cKR0TKWBlE4888QMmU3ptmtnBeQEURP38ctGEGNe400aY9UvbQDVHKpYq2k7Kx4ukQkMpY4gUCngRmC8LLc4VhV\\\\\\\/q5QbFckXO9J28FbU9lL3MJelMvUekJIeeVzlrq7bLohV0OZzvoP2q\\\\\\\/4uuyQMMeA\\\\\\\/P+MvHTNcwip0sP6kI4APgAjzzn+nbF+FkdcDpFlPiY27eBrru2zbeSgLNgM\\\\\\\/en6xej6P8slHsZ\\\\\\\/4n0NNG5JhLBzjIcGHnFBZ7MKru25feEpKWxCAWeLeTtSUt\\\\\\\/+XJQMFcCPgJvzNehr7p42tOGT+mMhbtx5UN8yWkvCIaeQTT4mokxY+1KYRihOWOLvESX22DUGJBZZ6xn052kDoNVzsTeNMYvoMm2cDU\\\\\\\/75vGi3SSHW1RWNO2YSt9U04ITR779gSPThA76YgZrEgxHbCyF4VWf4140MQOsvuHYlXWwl91jbOjgXDI0lFuiLx5uoX9JJGyhvKRN8zCCN5ftDJzSDQMNRalOvqWZHFZv110XKg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022fj2JZOGssajqhOhdkd\\\\\\\/o3A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f2d52ae494091f5ad27261ed67fecc363fd1ed6fb35c76f92cbdd4b68d65242b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488304 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/679" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-O75A0eZIkxJUMP-FB1FLl8xk3Wsy0mW6s7P3B-szQpM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/679" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mSziermtNW2rIzug3CaHS2dIt_8sir_Hw2CZp87-5qE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/669" title="Use of a Sibling Subtraction Method for Identifying Causal Mutations in Caenorhabditis elegans by Whole-Genome Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/687" title="Differential Regulation of Zfp30 Expression in Murine Airway Epithelia Through Altered Binding of ZFP148 to rs51434084" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488304" id="node488304" data-pisa="ggg;8/2/679" data-pisa-master="ggg;g3.117.300495" data-apath="/ggg/8/2/679.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4479-1847" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas D. Brekke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4896-8857" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katherine A. Steele</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1537-7316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John F. Mulley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">679-686;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300495 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488304" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas D. Brekke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BD.%2BBrekke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brekke%20TD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BD.%2BBrekke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4479-1847" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas D. Brekke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katherine A. Steele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environment, Natural Resources and Geography, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BA.%2BSteele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steele%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatherine%2BA.%2BSteele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4896-8857" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katherine A. Steele</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John F. Mulley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BF.%2BMulley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mulley%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BF.%2BMulley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1537-7316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John F. Mulley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.mulley@bangor.ac.uk" class="" data-icon-position="" data-hide-link-title="0">j.mulley@bangor.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/679" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488304" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/679.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488304" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/679.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488304" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/679.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488304" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/679.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488304" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488304/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Nonmodel rodents are widely used as subjects for both basic and applied biological research, but the genetic diversity of the study individuals is rarely quantified. University-housed colonies tend to be small and subject to founder effects and genetic drift; so they may be highly inbred or show substantial genetic divergence from other colonies, even those derived from the same source. Disregard for the levels of genetic diversity in an animal colony may result in a failure to replicate results if a different colony is used to repeat an experiment, as different colonies may have fixed alternative variants. Here we use high throughput sequencing to demonstrate genetic divergence in three isolated colonies of Mongolian gerbil (<em>Meriones unguiculatus</em>) even though they were all established recently from the same source. We also show that genetic diversity in allegedly “outbred” colonies of nonmodel rodents (gerbils, hamsters, house mice, deer mice, and rats) varies considerably from nearly no segregating diversity to very high levels of polymorphism. We conclude that genetic divergence in isolated colonies may play an important role in the “replication crisis.” In a more positive light, divergent rodent colonies represent an opportunity to leverage genetically distinct individuals in genetic crossing experiments. In sum, awareness of the genetic diversity of an animal colony is paramount as it allows researchers to properly replicate experiments and also to capitalize on other genetically distinct individuals to explore the genetic basis of a trait.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/meriones" class="hw-term hw-article-keyword hw-article-keyword-meriones" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Meriones</em></a></li><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li><li class="kwd"><a href="/keyword/inbreeding" class="hw-term hw-article-keyword hw-article-keyword-inbreeding" rel="nofollow">inbreeding</a></li><li class="kwd"><a href="/keyword/laboratory-rodents" class="hw-term hw-article-keyword hw-article-keyword-laboratory-rodents" rel="nofollow">laboratory rodents</a></li><li class="kwd"><a href="/keyword/replication" class="hw-term hw-article-keyword hw-article-keyword-replication" rel="nofollow">replication</a></li></ul><p id="p-3">The genetic variation present in laboratory rodent colonies has important implications for the design, outcome, and reproducibility of biological experiments (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Justice and Dhillon 2016</a>). High levels of genetic variation reduce power and increase variation in the response to a treatment, but the experimental results may be more applicable to natural or human populations. Alternatively, inbred colonies provide more power and require fewer animals per experiment by limiting the noise caused by segregating genetic variation [here we define an inbred strain as the result of ≥20 generations of brother–sister mating or equivalent (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eppig 2007</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Casellas 2011</a>)]. Indeed, minimizing the number of animals [in accordance with the principle of <em>reduction</em> in the 3Rs (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Russell and Burch 1959</a>)] is one of the main reasons cited for the use of inbred lines rather than outbred colonies (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Groen and Lagerwerf 1979</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Festing 1999</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chia <em>et al.</em> 2005</a>). Inbred lines with single genes knocked out have proven tremendously powerful for identifying the phenotypic effect of those genes (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Festing 2010</a>), but phenotypic traits and diseases often have complex genetic bases [<em>e.g.</em>, diabetes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fuchsberger <em>et al.</em> 2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rich 2016</a>) and epilepsy (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Meisler <em>et al.</em> 2001</a>)], so inbred models with no genetic variation may preclude a complete understanding of the underlying genetic architecture. Different epistatic interactions that occur in different genetic backgrounds may even reverse the phenotypic effect of a null allele (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sittig <em>et al.</em> 2016</a>), making experimental design and the choice of inbred strain paramount (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Little and Colegrave 2016</a>). Genetic variation is essential for the identification of candidate genes underlying complex phenotypes, and projects such as the Collaborative Cross have gone to great effort (and expense) to capture variation from across multiple inbred mouse strains in a controlled manner (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Churchill <em>et al.</em> 2004</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Roberts <em>et al.</em> 2007</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Threadgill and Churchill 2012</a>). Such projects rely on the fact that although there is no segregating variation within a single inbred line, multiple inbred lines have fixed alternative variants and immense power can be gained by leveraging these fixed alleles in a genetic mapping experiment (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Svenson <em>et al.</em> 2012</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de Koning and McIntyre 2017</a>).</p><p id="p-4">Genetic crosses involving multiple inbred lines are hugely powerful for genetic experiments, but true inbred strains of mammals are rare outside of “model” rodents such as mice and rats. The use of “nonmodel” rodents, such as gerbils (<em>Meriones unguiculatus</em>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Stuermer and Wetzel 2006</a>), hamsters (<em>Phodopus</em> sp.; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brekke <em>et al.</em> 2016</a>), spiny mice (<em>Acomys</em> sp.; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gawriluk <em>et al.</em> 2016</a>), and deer mice (<em>Peromyscus</em> sp.; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Weber <em>et al.</em> 2013</a>), is mainly restricted to outbred colonies with standing genetic variation. While significant steps are being taken to survey and characterize genetic diversity in outbred strains of house mice (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yalcin <em>et al.</em> 2010</a>), surprisingly little work has been done to quantify diversity in colonies of nonmodel rodents. Indeed, the labeling of a strain of animals as “outbred” (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chia <em>et al.</em> 2005</a>) or “wild derived” (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Harper 2008</a>), or even using a breeding scheme designed to minimize inbreeding (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Yalcin <em>et al.</em> 2010</a>), may have little to no bearing on the genetic diversity present. Instead, such labels only demonstrate that the animals have not <em>purposely</em> undergone the ≥20 generations of brother–sister mating necessary to purge segregating variation and establish a true inbred line (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Eppig 2007</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Casellas 2011</a>). Whereas it is recognized that large colonies will slow the loss of genetic variation through drift (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Papaioannou and Festing 1980</a>), and commercial providers of outbred animals may maintain 50–100 or more breeding pairs per colony (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Yalcin <em>et al.</em> 2010</a>), the size of colonies especially in academic institutions is constrained by housing space, finances, and human resources. Furthermore, bottleneck or founder effects are likely to occur as animals are moved between colonies, used to establish a new colony, or rederive an old one. Thus, we should expect the amount of standing genetic variation to differ even between colonies of the same species and strain.</p><p id="p-5">Mongolian gerbils (<em>M. unguiculatus</em>) are a common nonmodel rodent that have been used in biological research for many years and have informed our understanding of diseases such as epilepsy (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buchhalter 1993</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Buckmaster 2006</a>), stroke (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Vincent and Rodrick 1979</a>), and diabetes (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2016</a>); as well as basic biology questions about thermal regulation (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Thiessen and Kittrell 1980</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wang <em>et al.</em> 2000</a>), desert adaptation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">McManus 1972</a>), domestication (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stuermer <em>et al.</em> 2003</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Stuermer and Wetzel 2006</a>), reproductive biology (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Clark <em>et al.</em> 1994</a>), hearing (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abbas and Rivolta 2015</a>), and more.</p><p id="p-6">Despite the widespread use of gerbils in scientific research, few widely accessible transcriptomic and genomic resources have been developed, and the small numbers of genetic markers available are limited in their ability to accurately reveal genome-wide levels of diversity. Early reports using microsatellites (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Neumann <em>et al.</em> 2001</a>) suggested that genetic diversity in laboratory gerbil colonies is a small fraction of that in the wild and lower than the diversity captured across multiple inbred mouse (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Love <em>et al.</em> 1990</a>) or rat strains (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Serikawa <em>et al.</em> 1992</a>). Similarly, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Razzoli (2003)</a> found fairly low levels of heterozygosity using AFLPs. But more recent reports using microsatellites suggest that variation may be high, even similar to the levels found in the wild (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Du <em>et al.</em> 2010</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2015</a>). A simple explanation for this contradiction is that different strains of animals were used in each study. <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Du <em>et al.</em> (2010</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">2015)</a> surveyed four laboratory colonies from China, all of which were recently established from wild-caught individuals; whereas <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Neumann <em>et al.</em> (2001)</a> and <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Razzoli (2003)</a> mainly used animals originating from the Tumblebrook Farm strain. This strain has its origins in 20 pairs of wild-caught animals that were used to establish a colony in the Kitasato Institute in Japan in 1935. Since then, it has been rederived, sold, bottlenecked, and transferred repeatedly, beginning with a move to the Central Laboratories for Experimental Animals in 1949 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Petrij <em>et al.</em> 2001</a>). From there, 11 animals were moved to the United States at Tumblebrook Farm in 1954 (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Stuermer <em>et al.</em> 2003</a>) and then again to Charles River Ltd. in Italy in 1996 (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Neumann <em>et al.</em> 2001</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Razzoli 2003</a>), where the colony was rederived and has been maintained since with ≥100 breeding pairs and an outbred crossing scheme (C. Parady, personal communication). The population history of laboratory gerbils is thus punctuated by a series of bottleneck events each time the colony was moved and rederived. Thus, there seems to be a discrepancy in how these gerbils are maintained and sold by commercial providers (as a highly diverse outbred stock) and the results of previous genetic analyses [which suggest very low levels of diversity (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Neumann <em>et al.</em> 2001</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Razzoli 2003</a>) and heterozygosity (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Razzoli 2003</a>)].</p><p id="p-7">If laboratory gerbils are inbred, fewer are needed to achieve statistically significant results and maintain a breeding colony, but any results obtained may not be widely generalizable to other strains or species (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Little and Colegrave 2016</a>). Given the limitations of small-scale microsatellite and AFLP experiments, we therefore decided to use a genome-wide approach to quantify the genetic diversity present in Tumblebrook Farm-strain gerbils. Here, we evaluate patterns of standing genetic variation in animals from three different gerbil colonies to identify differences that may stem from a history of bottlenecks and isolation. All three colonies originated from the European colony managed by Charles River Ltd. We also compared these with the recently released whole genome sequence of an individual from an American stock of the Tumblebrook Farm strain (GenBank accession GCA_002204375.1). We interpret the levels of genetic variation in gerbils in comparison with colonies of other species such as house mice (<em>Mus musculus</em> subsp.), hamsters (<em>Phodopus</em> sp.), deer mice (<em>Peromyscus</em> sp.), and rats (<em>Rattus norvegicus</em>). We also discuss the possibility of leveraging the inescapable genetic drift in small mammal colonies to identify differentiated genetic markers for use in genetic mapping and association studies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Animals</h3><p id="p-8">Mongolian gerbils are listed in Annex 1 of European Union (EU) Directive 2010/63/EU and must therefore be purposely bred for scientific research. The majority (if not all) gerbils used in the EU are derived from the Tumblebrook Farm stock and many academic institutions in the UK, and elsewhere, maintain their own colonies derived from these animals. We analyzed animals from three of these colonies, and to avoid confusion we refer to each colony by the name of the city in which it was first established: Edinburgh, Sheffield, and Bangor. The Edinburgh colony was established by Dr. Judith Allen at the University of Edinburgh <em>circa</em> 2005. In 2014, Dr. Leila Abbas established a new colony at the University of Sheffield from the Charles River Ltd. Tumblebrook stock and at the same time took over care and housing of three to four pairs of animals from Edinburgh, with both stocks maintained separately. In 2016, we took delivery of 12 new Tumblebrook animals from Charles River (seven female and five male) to establish the Bangor colony. We also received five animals from each of the Edinburgh (three female and two male) and Sheffield (two female and three male) stocks. All three groups were maintained in isolation in Bangor, except for a single testcross between an Edinburgh female and a Sheffield male. All animals were housed in accordance with EU and Home Office animal care regulations, and experiments were reviewed and approved by the Bangor University Animal Welfare and Ethical Review Board.</p></div><div id="sec-3" class="subsection"><h3>Tissue collection, DNA extraction, library preparation, and sequencing</h3><p id="p-9">Liver tissue was collected from the 22 founder animals and two F<sub>1</sub> Edinburgh × Sheffield offspring and was immediately snap frozen in liquid nitrogen after each animal was killed as part of routine colony management. DNA was extracted with the Qiagen DNeasy Blood and Tissue Kit and treated with RNase according to the manufacturer’s instructions. Extracted DNA was shipped to the Beijing Genome Institute (BGI) (Hong Kong) for library preparation and sequencing. Uniquely barcoded 100-bp, paired-end, genotyping-by-sequencing (GBS) libraries were prepared with the 5-bp cutter ApeKI, pooled, and sequenced on a single lane of Illumina 4000 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Elshire <em>et al.</em> 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Bioinformatics</h3><p id="p-10">BGI filtered the raw data through their SOAPnuke filter, which includes demultiplexing the reads, removing proprietary barcode sequences, and dropping reads that were &gt;26% adapter sequence and/or where &gt;40% of the bases were below a PHRED quality score of 15. We used the Stacks pipeline (v1.46; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Catchen <em>et al.</em> 2011</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2013</a>) to identify tags and call SNPs from the first reads, resulting in an average sequencing effort of 7.8 million reads per individual. This analysis included the standard Stacks pipeline components: process_radtags, ustacks, cstacks, sstacks, and populations. All scripts were run with default flags with the following exceptions: With process_radtags we cleaned reads (-c), discarded reads with low quality scores (-q), rescued radtags (-r), and truncated read length to 92 bases (-t 92) to avoid variation in read length that would otherwise disrupt the remaining pipeline. Thus, our final markers were all 92 bp long. We ran the deleveraging algorithm in ustacks (-d) and used six individuals (a female and male from each Bangor, Edinburgh, and Sheffield strains) for cstacks. We generated a reference fasta (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300495/-/DC1/FileS1.zip">File S1</a>) from the output of cstacks and to it we aligned the raw reads with bwa mem (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li and Durbin 2009</a>). From these alignments, we extracted depth of coverage with samtools (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2009</a>) to annotate autosomal, X-, and Y-linked markers. Coverage was standardized by the sequencing effort of each individual and multiplied by 1,000,000 before being summed across males and females. Sex linkage is apparent by comparing standardized coverage of each marker in males <em>vs.</em> females. We first annotated 422,664 markers with &lt;10× total standardized coverage as “unknown” and removed these from the data set as their coverage is too low to reliably differentiate X- and Y-linked tags from autosomal tags or call variants. From the remaining 725,888 markers, we identified Y-linked ones as those with &lt;1× standardized coverage in females. X-linked markers fulfilled the inequality: coverage<sup>male</sup> &lt; 3/4 coverage<sup>female</sup> − 5. The slope of this line was chosen to discriminate points in the X-linked cluster (slope = 1/2) from those in the autosomal cluster (slope = 1). The intercept was chosen to remain fairly conservative near the origin; that is erring toward labeling a true X-linked tag as an autosome rather than labeling a true autosomal tag as X-linked. All remaining tags were annotated as autosomal. The populations script was used to generate diversity metrics and F statistics across the genome for autosomal, X-, and Y-linked markers. Genotypes were called only for individuals with &gt;10× coverage (-m 10) and only for SNPs in the first 90 bases. We blacklisted SNPs in the final two bases because of an unusually high number of SNPs on those bases (for further discussion see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300495/-/DC1/FigureS1.pdf">Figure S1</a>). Finally, we evaluated genetic similarity and population structure with the SNPRelate package in R (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zheng <em>et al.</em> 2012</a>) and the program Structure (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rosenberg <em>et al.</em> 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Falush <em>et al.</em> 2003</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2007</a>). We visualized structure data with the program distruct (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rosenberg 2003</a>). We calculated pairwise diversity between our reference and the recently released gerbil whole genome sequence (GenBank accession GCA_002204375.1) by aligning the reference sequences to the genome with bwa mem, discarding partial-length alignments, and counting mismatches across the first 90 bases of the reference. Finally, we determined how many SNP-containing GBS tags were found in coding regions by comparing the alignment locations of the tags with the gene annotations in the gerbil genome.</p><p id="p-11">To evaluate the levels of nucleotide diversity in gerbils in a more general sense, we downloaded restriction-site associated DNA (RAD) sequencing data from deer mice (SRA accession PRJNA186607; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Weber <em>et al.</em> 2013</a>) and were provided with RAD sequence data for hamsters (J. Good, personal communication). These RAD data sets were analyzed with the Stacks pipeline described above (omitting the chromosomal annotation steps) to be directly comparable with diversity estimates in gerbils. To provide further context for our estimates of nucleotide diversity (π), we also retrieved recently published diversity metrics from house mice in the Collaborative Cross (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Srivastava <em>et al.</em> 2017</a>), various inbred and wild-caught mouse colonies (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Salcedo <em>et al.</em> 2007</a>), and inbred and wild-caught rats (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Smits <em>et al.</em> 2004</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ness <em>et al.</em> 2012</a>).</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-12">Raw sequencing data are archived in the SRA under the BioProject accession number PRJNA397533 and the sample accession numbers SAMN07460176–SAMN07460199. The reference fasta file (including autosomal, X-, or Y-linkage of each tag) as well as the VCF file containing the locations of all SNPs are available as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300495/-/DC1/FileS1.zip">File S1</a>.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-13">We compared sequencing coverage in females and males to annotate 718,385 autosomal markers, 5148 X-linked markers, and 2355 Y-linked markers (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We identified 30,365 SNPs spread across 24,326 markers (1.25 SNPs per marker). Average π is 0.0059 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Most SNP-containing markers (∼95%) are intergenic and so this value tends to describe variation in unconstrained, noncoding regions. Around 5% of SNP-containing markers (1186) are found in 1109 different genes and these are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300495/-/DC1/TableS1.txt">Table S1</a>. Average heterozygosity at autosomal variant sites is 0.447 and is slightly higher on the sex chromosomes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/679/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative coverage in females and males can be used to identify sex chromosomes. Plotted here is standardized coverage in females against standardized coverage in males; histograms show the density of markers along each axis. The markers shown are those with low overall coverage; a long tail exists in both females and males and is not shown here. Markers with " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1473403226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative coverage in females and males can be used to identify sex chromosomes. Plotted here is standardized coverage in females against standardized coverage in males; histograms show the density of markers along each axis. The markers shown are those with low overall coverage; a long tail exists in both females and males and is not shown here. Markers with &lt;10&#xD7; total standardized coverage were annotated as unknown (gray) because those have too little coverage to reliably distinguish X- and Y-linkage from autosomes. Of the remaining tags, those with &lt;1&#xD7; standardized coverage in females are annotated as Y-linked (blue). Those that satisfy the inequality: coverage&lt;sup&gt;male&lt;/sup&gt; &lt; 3/4 coverage&lt;sup&gt;female&lt;/sup&gt; &#x2212; 5 were identified as X-linked (red). This line has a slope designed to discriminate points in the X-linked cluster (slope = 1/2) from those in the autosomal cluster (slope = 1) while remaining fairly conservative near the origin. All remaining tags were annotated as autosomal (black).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/679/F1.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/679/F1.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/679/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/679/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488547" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Relative coverage in females and males can be used to identify sex chromosomes. Plotted here is standardized coverage in females against standardized coverage in males; histograms show the density of markers along each axis. The markers shown are those with low overall coverage; a long tail exists in both females and males and is not shown here. Markers with &lt;10× total standardized coverage were annotated as unknown (gray) because those have too little coverage to reliably distinguish X- and Y-linkage from autosomes. Of the remaining tags, those with &lt;1× standardized coverage in females are annotated as Y-linked (blue). Those that satisfy the inequality: coverage<sup>male</sup> &lt; 3/4 coverage<sup>female</sup> − 5 were identified as X-linked (red). This line has a slope designed to discriminate points in the X-linked cluster (slope = 1/2) from those in the autosomal cluster (slope = 1) while remaining fairly conservative near the origin. All remaining tags were annotated as autosomal (black).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488365/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488365/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Diversity metrics in gerbils across the genome</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">To evaluate how different the gerbil genome (GCA_002204375.1) is from our colonies, we counted the number of differences between our GBS reference and the genome. Full-length alignments were found for 674,342 (93%) of our reference sequences when aligned to the genome. We found 47,223 single-base differences in these aligned regions, far more than the SNPs that segregate within the colonies we assayed. This pattern is consistent with the known population history of laboratory gerbils: the Charles River colony, from which our animals originate, was rederived from a United States colony from which the DNA for the genome was supplied.</p><p id="p-17">We used 28,885 autosomal SNPs to evaluate the diversity between the three colonies and found that whereas each colony does possess a small set of private alleles, most alleles are shared across all colonies (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). A substantial portion of the variation (24%) is explained by differences between Edinburgh and the other colonies (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Eigenvector 2 shows that much of the remaining variation (7%) segregates within the Bangor colony. No higher-order eigenvectors discriminate the colonies, instead they partition variation that is common to all. F<sub>st</sub> metrics between the colonies suggest a high overall similarity between Bangor and Sheffield (F<sub>st</sub> = 0.069) and identify Edinburgh as an outlier with an F<sub>st</sub> of 0.235 when compared with Bangor and 0.352 when compared with Sheffield. The structure analysis also suggests little overall differentiation between Bangor and Sheffield, and finds that Edinburgh is slightly more divergent, though still very similar (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Overall, these data suggest that although Edinburgh animals have marked differences from other gerbils, they still share many genetic variants.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/679/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PCA of 28,885 autosomal SNPs. Eigenvector 1 explains the majority of the diversity in these samples and strongly differentiates the Edinburgh colony from the others. Sheffield contains a subset of the genetic diversity found within Bangor colony. F1 offspring between Edinburgh female and Sheffield male are intermediate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1473403226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PCA of 28,885 autosomal SNPs. Eigenvector 1 explains the majority of the diversity in these samples and strongly differentiates the Edinburgh colony from the others. Sheffield contains a subset of the genetic diversity found within Bangor colony. F&lt;sub&gt;1&lt;/sub&gt; offspring between Edinburgh female and Sheffield male are intermediate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/679/F2.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/679/F2.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/679/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/679/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488342" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">PCA of 28,885 autosomal SNPs. Eigenvector 1 explains the majority of the diversity in these samples and strongly differentiates the Edinburgh colony from the others. Sheffield contains a subset of the genetic diversity found within Bangor colony. F<sub>1</sub> offspring between Edinburgh female and Sheffield male are intermediate.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/679/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure plots of gerbil colonies. Structure was run for 20,000 iterations with 10,000 iterations of burn-in for K = 2, 3, and 4. In general, the different colonies have similar assignment especially at low K values. Edinburgh animals fall out uniquely at higher K values (i.e., blue). The F1 hybrids have mixed ancestry as expected for the offspring of a Sheffield &#xD7; Edinburgh cross." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1473403226" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structure plots of gerbil colonies. Structure was run for 20,000 iterations with 10,000 iterations of burn-in for &lt;em&gt;K&lt;/em&gt; = 2, 3, and 4. In general, the different colonies have similar assignment especially at low &lt;em&gt;K&lt;/em&gt; values. Edinburgh animals fall out uniquely at higher &lt;em&gt;K&lt;/em&gt; values (&lt;em&gt;i.e.&lt;/em&gt;, blue). The F&lt;sub&gt;1&lt;/sub&gt; hybrids have mixed ancestry as expected for the offspring of a Sheffield &#xD7; Edinburgh cross.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/679/F3.medium.gif" width="357" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/2/679/F3.medium.gif" width="357" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/679/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/679/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488532" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">Structure plots of gerbil colonies. Structure was run for 20,000 iterations with 10,000 iterations of burn-in for <em>K</em> = 2, 3, and 4. In general, the different colonies have similar assignment especially at low <em>K</em> values. Edinburgh animals fall out uniquely at higher <em>K</em> values (<em>i.e.</em>, blue). The F<sub>1</sub> hybrids have mixed ancestry as expected for the offspring of a Sheffield × Edinburgh cross.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">In general, the highest diversity is found in the Bangor animals. This is apparent in both the principal component analysis (PCA) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) and the number of polymorphic sites segregating within the Bangor strain (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) and may be due to the higher number of individuals screened or to the fact that they have not been maintained at low population sizes like the animals from Sheffield and Edinburgh. Edinburgh animals have far fewer segregating sites, lower nucleotide diversity, and lower heterozygosity overall, which may be a result of serial bottlenecks. The PCA and the low number of private alleles suggest that the Sheffield animals contain a subset of the diversity found within the Bangor strain (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). As expected, the F<sub>1</sub> offspring between Edinburgh and Sheffield are found to be intermediate to the parents.</p><p id="p-21">Although genetic diversity in laboratory gerbils (π = 0.0059) is lower than that captured in the mice of the Collaborative Cross (π = 0.0289; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Srivastava <em>et al.</em> 2017</a>), gerbil diversity is quite high compared with many other laboratory rodents (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). In fact, gerbils rival the diversity found in wild-derived mouse colonies [<em>M. musculus</em>: π = 0.0054; <em>M. domesticus</em>: π = 0.0102 (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Salcedo <em>et al.</em> 2007</a>)] and, contrary to previous claims (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Neumann <em>et al.</em> 2001</a>; <a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Razzoli 2003</a>), exceed the diversity in rats, whether wild caught (π = 0.0022; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Ness <em>et al.</em> 2012</a>) or between inbred strains (0.0029 &gt; π &gt; 0.0015; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Smits <em>et al.</em> 2004</a>). It is clear that genetic diversity in Tumblebrook gerbils is much higher than previous reports suggest (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Neumann <em>et al.</em> 2001</a>; <a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">Razzoli 2003</a>). It is also clear that breeding scheme alone does not robustly predict the amount of standing genetic diversity of an animal colony (<a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Yalcin <em>et al.</em> 2010</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488476/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488476/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Nucleotide diversity in various rodent colonies</span><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-7"><h2 class="">Discussion</h2><p id="p-25">We have demonstrated that appreciable genetic diversity segregates within Tumblebrook Farm-strain Mongolian gerbils. Our findings are contrary to earlier reports suggesting that diversity may be as low as, or lower than, laboratory mouse colonies (<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Neumann <em>et al.</em> 2001</a>; <a id="xref-ref-41-8" class="xref-bibr" href="#ref-41">Razzoli 2003</a>). These reports evaluated a small portion of the genome and, perhaps unsurprisingly, found little variation. For instance, <a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Neumann <em>et al.</em> (2001)</a> evaluated diversity at nine microsatellites and found that laboratory strains had severely reduced allelic diversity compared to wild animals, and <a id="xref-ref-41-9" class="xref-bibr" href="#ref-41">Razzoli (2003)</a> found low heterozygosity using 228 AFLP fragments from six primer combinations. Our genome-wide assay evaluated millions of bases and so has much higher power to find rare variants. Using these data, we find that genetic diversity in Mongolian gerbils is relatively high among outbred nonmodel rodent colonies (π = 0.0059 in gerbils and π ≤ 0.0010 in other nonmodel rodents, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-26">Laboratory-maintained rodent colonies are often small due to the costs and space needed for maintenance of many animals. With such small populations, genetic drift plays an important role in determining the standing level of variation. Drift can be expected to increase genetic differentiation between colonies through time, especially given the population bottleneck that often occurs when a colony is established or moved to a new location. Knowledge of levels of genetic diversity in an institutional colony is therefore vital for correct colony management; for example, <em>Phodopus</em> hamsters have been referred to as outbred and maintained in large colonies (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Brekke and Good 2014</a>). However, analysis of double-digest RAD data from two hamster species (J. Good, personal communication) shows that, in fact, genetic diversity is extremely low in both (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), and so hamster colonies could be maintained with few individuals with no resulting loss of diversity. Despite the length of time in captivity, the Tumblebrook gerbils are (correctly) maintained as a large outbred colony (≥100 breeding pairs) by Charles River Ltd. Our data suggest that the diversity present in that original stock has been subsampled and exposed to drift in each of the three independent colonies we assayed. At one extreme is the Edinburgh colony which was not only the first to be isolated from Tumblebrook, but has been transferred through three universities and experienced the associated bottlenecks. Given this history, it is not surprising that the Edinburgh animals have the fewest SNPs segregating within them, nor that they are somewhat differentiated from Bangor and Sheffield. The Sheffield animals, which were established from Tumblebrook strain founders in 2014 and have been moved through only two universities, also show a reduced diversity, though still higher than Edinburgh. The Bangor colony was established most recently in 2016 and has the highest amount of diversity. As these animals were sent directly from Charles River Ltd., they likely represent a large portion of the variation contained in the Tumblebrook stock. Our data suggest that genetic drift in these three colonies is actively eroding the standing genetic variation and, as they have been maintained in isolation from each other, it has resulted in noticeable differentiation between the colonies.</p><p id="p-27">There are two major ramifications of the loss and partitioning of genetic variation in laboratory colonies. First, animals from the same original outbred stock may respond very differently to an experiment if they come from different isolated colonies. Many articles state that diversity in gerbils is quite low, one even suggesting that smaller error bars in laboratory individuals than wild-caught individuals are due to the lower genetic diversity (<em>i.e.</em>, <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Stuermer and Wetzel 2006</a>). Although it is almost certainly correct that the diversity in their colony is low, our data suggest that this is likely a reflection of high drift in an isolated colony rather than low diversity in the original Tumblebrook stocks or even across all laboratory gerbils in general. Low diversity is an important factor in interpreting many experimental results, but generally missing from this acknowledgment is that although diversity is likely low in any specific colony, that does not mean that all colonies are genetically similar. This may partly explain why some experimental outcomes are not able to be replicated despite using animals from the same original outbred strain (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Richter <em>et al.</em> 2011</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Justice and Dhillon 2016</a>). This general argument is applicable not only to rodent colonies, but any laboratory animals of any taxa where the population size is limited.</p><p id="p-28">The second important ramification of high genetic drift in laboratory colonies is that although diversity will be lost in any single colony through time, across multiple isolated colonies much of the original diversity may be preserved. This is not a new idea and there are major ongoing efforts using multiple inbred strains to capture the range of natural diversity (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Churchill <em>et al.</em> 2004</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Threadgill and Churchill 2012</a>). By intercrossing between multiple differentiated colonies, researchers can do controlled experiments designed to uncover the genetic architecture of complex traits (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Festing 2010</a>). In this regard, gerbils seem to be good candidates. Indeed, the high overall variation, number of private alleles within colonies, and intermediate location of the Sheffield × Edinburgh F<sub>1</sub> offspring in the PCA (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) all suggest that sufficient diversity exists between the colonies for successful genetic experiments.</p><div id="sec-8" class="subsection"><h3>Conclusions</h3><p id="p-29">Despite being derived from a relatively small number of founders and experiencing repeated bottlenecks over the past 80 years in captivity, the Tumblebrook Farm strain of Mongolian gerbils does not have low levels of genetic diversity. However, we have shown that genetic drift in small institutional colonies has increased differentiation and such drift may affect the reproducibility of experiments. For those doing animal research, the choice of whether to use inbred or outbred animals depends on the specific question being addressed (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Little and Colegrave 2016</a>). For instance, a few strains of highly inbred animals may be best for identifying the function of a specific gene, while a highly outbred population that captures much of the natural variation may be better for understanding things like the population level response to climate change. Regardless of the question, it is important to know the genetic diversity of the animals. Thus, we advise that experimenters consider the history of their colony when planning and performing research projects using gerbils and other animal models. Specifically, as the label of “outbred” is an unreliable metric of genetic variation, we suggest that researchers using such colonies verify the diversity of their animals directly through sequencing. Suggestions have previously been made for the ideal number of markers to calculate relatedness (&gt;125 but &lt;500; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Santure <em>et al.</em> 2010</a>) and heterozygosity (&gt;200; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Balloux <em>et al.</em> 2004</a>). However, with current sequencing prices, it is generally less expensive to use a GBS or RAD approach and assay tens of thousands of markers than to develop a microsatellite or Sanger panel for hundreds of markers. Either way, with a bit of sequencing, researchers will be able to quickly and reliably assay genetic diversity in their colonies and then make appropriate choices about which animals to use for which experiments. We further suggest that published claims on levels of genetic diversity in laboratory rodents based on small numbers of genetic markers should be taken with a pinch of salt.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-30">We thank Leila Abbas for providing us with the Edinburgh and Sheffield strains of gerbils and Charlie Parady (Charles River Ltd.) and Judith Allen for information on the maintenance of the Charles River and Edinburg gerbil colonies, respectively. We thank Matt Hegarty (Aberystwyth University) for advice on GBS sequencing and Rhys Morgan and Emlyn Roberts for technical assistance with animal husbandry. We thank our editors, David Threadgill and Dirk-Jan de Konig, along with two anonymous reviewers for helpful comments. Finally, we would like to thank Jeff Good, Jesse Weber, and Kyle Turner for helping us with diversity estimates from various rodent colonies. This work was funded by a Leverhulme Trust research project grant to J.F.M. and K.A.S. (RPG-2015-450).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-31">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300495/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300495/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-32"><em>Communicating editor: D. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-09"><span class="received-label">Received </span>August 9, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-12-12"><span class="accepted-label">Accepted </span>December 12, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Brekke <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abbas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivolta</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Aminoglycoside ototoxicity and hair cell ablation in the adult gerbil: a simple model to study hair cell loss and regeneration.</span> <abbr class="cit-jnl-abbrev">Hear. Res.</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHear.%2BRes.%26rft.volume%253D325%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.2" data-doi="10.1111/j.1365-294X.2004.02318.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Does heterozygosity estimate inbreeding in real populations?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3021</span>–<span class="cit-lpage">3031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBalloux%26rft.auinit1%253DF.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D3021%26rft.epage%253D3031%26rft.atitle%253DDoes%2Bheterozygosity%2Bestimate%2Binbreeding%2Bin%2Breal%2Bpopulations%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02318.x%26rft_id%253Dinfo%253Apmid%252F15367117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02318.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15367117&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223863000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-2" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brekke</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Parent-of-origin growth effects and the evolution of hybrid inviability in dwarf hamsters.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">3134</span>–<span class="cit-lpage">3148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D3134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brekke</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic imprinting, disrupted placental expression, and speciation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">2690</span>–<span class="cit-lpage">2703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D70%26rft.spage%253D2690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchhalter</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Animal models of inherited epilepsy.</span> <abbr class="cit-jnl-abbrev">Epilepsia</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">S31</span>–<span class="cit-lpage">S41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpilepsia%26rft.volume%253D34%26rft.spage%253DS31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.679.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckmaster</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inherited epilepsy in Mongolian gerbils</span>, pp. <span class="cit-fpage">273</span>–<span class="cit-lpage">294</span> in <span class="cit-source">Models of Seizures and Epilepsy</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Pitkänen</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Schwartzkroin</span>  <span class="cit-name-given-names">P. A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Moshé</span>  <span class="cit-name-given-names">S. L.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">Burlington, ON</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.7" data-doi="10.1017/S1751731110001667"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casellas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inbred mouse strains and genetic stability: a review.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DCasellas%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DInbred%2Bmouse%2Bstrains%2Band%2Bgenetic%2Bstability%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731110001667%26rft_id%253Dinfo%253Apmid%252F22440695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731110001667&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22440695&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.8" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping loci de novo from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D1%26rft.spage%253D171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.10" data-doi="10.1038/nature11415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jongkamonwiwat</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshtan</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Restoration of auditory evoked responses by human ES-cell-derived otic progenitors.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">490</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChen%26rft.auinit1%253DW.%26rft.volume%253D490%26rft.issue%253D7419%26rft.spage%253D278%26rft.epage%253D282%26rft.atitle%253DRestoration%2Bof%2Bauditory%2Bevoked%2Bresponses%2Bby%2Bhuman%2BES-cell-derived%2Botic%2Bprogenitors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11415%26rft_id%253Dinfo%253Apmid%252F22972191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22972191&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309733300053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.11" data-doi="10.1038/ng1665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achilli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Festing</span>  <span class="cit-name-given-names">M. F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">E. M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The origins and uses of mouse outbred stocks.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChia%26rft.auinit1%253DR.%26rft.volume%253D37%26rft.issue%253D11%26rft.spage%253D1181%26rft.epage%253D1186%26rft.atitle%253DThe%2Borigins%2Band%2Buses%2Bof%2Bmouse%2Boutbred%2Bstocks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1665%26rft_id%253Dinfo%253Apmid%252F16254564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1665&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16254564&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233045200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.12" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The collaborative cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.13" data-doi="10.1530/jrf.0.1010393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ham</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galef</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Differences in the sex ratios of offspring originating in the right and left ovaries of Mongolian gerbils (<em>Meriones unguiculatus</em>).</span> <abbr class="cit-jnl-abbrev">J. Reprod. Fertil.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReproduction%26rft.stitle%253DReproduction%26rft.aulast%253DClark%26rft.auinit1%253DM.%2BM.%26rft.volume%253D101%26rft.issue%253D2%26rft.spage%253D393%26rft.epage%253D396%26rft.atitle%253DDifferences%2Bin%2Bthe%2Bsex%2Bratios%2Bof%2Boffspring%2Boriginating%2Bin%2Bthe%2Bright%2Band%2Bleft%2Bovaries%2Bof%2BMongolian%2Bgerbils%2B%2528Meriones%2Bunguiculatus%2529%26rft_id%253Dinfo%253Adoi%252F10.1530%252Fjrf.0.1010393%26rft_id%253Dinfo%253Apmid%252F7932374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicmVwcm9kIjtzOjU6InJlc2lkIjtzOjk6IjEwMS8yLzM5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi82NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.14" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the collaborative cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.15" data-doi="10.1534/genetics.117.203265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Back to the future: multiparent populations provide the key to unlocking the genetic basis of complex traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">527</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.203265%26rft_id%253Dinfo%253Apmid%252F28592493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.16" data-doi="10.1093/jhered/esq066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Development of novel microsatellite DNA markers by cross-amplification and analysis of genetic variation in gerbils.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">710</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesq066%26rft_id%253Dinfo%253Apmid%252F20525768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esq066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525768&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283691400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sa</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection of an effective microsatellite marker system for genetic control and analysis of gerbil populations in China.</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">11030</span>–<span class="cit-lpage">11042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BRes.%26rft.volume%253D14%26rft.spage%253D11030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.18" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.679.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eppig</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mouse strain and genetic nomenclature: an abbreviated guide</span>, pp. <span class="cit-fpage">79</span>–<span class="cit-lpage">98</span> in <span class="cit-source">The Mouse in Biomedical Research</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">J. G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Davisson</span>  <span class="cit-name-given-names">M. T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Quimby</span>  <span class="cit-name-given-names">F. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Barthold</span>  <span class="cit-name-given-names">S. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Newcomer</span>  <span class="cit-name-given-names">C. E.</span></span> <em>et al</em>. <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.21" data-doi="10.1111/j.1471-8286.2007.01758.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: dominant markers and null alleles.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">574</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BNotes%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D574%26rft.epage%253D578%26rft.atitle%253DInference%2Bof%2Bpopulation%2Bstructure%2Busing%2Bmultilocus%2Bgenotype%2Bdata%253A%2Bdominant%2Bmarkers%2Band%2Bnull%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01758.x%26rft_id%253Dinfo%253Apmid%252F18784791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01758.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18784791&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247758100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.22" data-doi="10.1016/S0197-4580(99)00040-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Festing</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Warning: the use of heterogeneous mice may seriously damage your research.</span> <abbr class="cit-jnl-abbrev">Neurobiol. Aging</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurobiology%2Bof%2Baging%26rft.stitle%253DNeurobiol%2BAging%26rft.aulast%253DFesting%26rft.auinit1%253DM.%2BF.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D244%26rft.atitle%253DWarning%253A%2Bthe%2Buse%2Bof%2Bheterogeneous%2Bmice%2Bmay%2Bseriously%2Bdamage%2Byour%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0197-4580%252899%252900040-8%26rft_id%253Dinfo%253Apmid%252F10537033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0197-4580(99)00040-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10537033&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083271000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.23" data-doi="10.1177/0192623310373776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Festing</span>  <span class="cit-name-given-names">M. F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Inbred strains should replace outbred stocks in toxicology, safety testing, and drug development.</span> <abbr class="cit-jnl-abbrev">Toxicol. Pathol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BPathol.%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0192623310373776%26rft_id%253Dinfo%253Apmid%252F20562325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0192623310373776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562325&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flannick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teslovich</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahajan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetic architecture of type 2 diabetes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">536</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.25" data-doi="10.1038/ncomms11388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gawriluk</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simkin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biswas</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clare-Salzler</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparative analysis of ear-hole closure identifies epimorphic regeneration as a discrete trait in mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11388%26rft_id%253Dinfo%253Apmid%252F27009409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27009409&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.26" data-doi="10.1258/002367779780943387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagerwerf</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Genic heterogeneity and genetic monitoring of mouse outbred stocks.</span> <abbr class="cit-jnl-abbrev">Lab. Anim.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLaboratory%2BAnimals%26rft.stitle%253DLab%2BAnim%26rft.aulast%253DGroen%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D81%26rft.epage%253D85%26rft.atitle%253DGenic%2Bheterogeneity%2Band%2Bgenetic%2Bmonitoring%2Bof%2Bmouse%2Boutbred%2Bstocks%26rft_id%253Dinfo%253Adoi%252F10.1258%252F002367779780943387%26rft_id%253Dinfo%253Apmid%252F480905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1258/002367779780943387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=480905&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Wild-derived mouse stocks: an underappreciated tool for aging research.</span> <abbr class="cit-jnl-abbrev">Age (Dordr.)</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAge%2B%2528Dordr.%2529%26rft.volume%253D30%26rft.spage%253D135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.28" data-doi="10.1242/dmm.024547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Justice</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Using the mouse to model human disease: increasing validity and reproducibility.</span> <abbr class="cit-jnl-abbrev">Dis. Model. Mech.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDisease%2BModels%2B%2526%2BMechanisms%26rft.stitle%253DDMM%26rft.aulast%253DJustice%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D101%26rft.epage%253D103%26rft.atitle%253DUsing%2Bthe%2Bmouse%2Bto%2Bmodel%2Bhuman%2Bdisease%253A%2Bincreasing%2Bvalidity%2Band%2Breproducibility%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdmm.024547%26rft_id%253Dinfo%253Apmid%252F26839397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZG1tIjtzOjU6InJlc2lkIjtzOjc6IjkvMi8xMDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.29" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.30" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Establishment and characterization of a newly established diabetic gerbil line.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0159420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0159420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colegrave</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Caging and uncaging genetics.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1002525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D14%26rft.spage%253De1002525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.33" data-doi="10.1093/nar/18.14.4123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAleer</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Towards construction of a high resolution map of the mouse genome using PCR-analysed microsatellites.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">4123</span>–<span class="cit-lpage">4130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLove%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D18%26rft.issue%253D14%26rft.spage%253D4123%26rft.epage%253D4130%26rft.atitle%253DTowards%2Bconstruction%2Bof%2Ba%2Bhigh%2Bresolution%2Bmap%2Bof%2Bthe%2Bmouse%2Bgenome%2Busing%2BPCR-analysed%2Bmicrosatellites%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F18.14.4123%26rft_id%253Dinfo%253Apmid%252F2377456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/18.14.4123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2377456&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DR45400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McManus</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Water relations and food consumption of the mongolian gerbil, <em>Meriones unguiculatus</em>.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. Part A. Physiol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComp.%2BBiochem.%2BPhysiol.%2BPart%2BA.%2BPhysiol.%26rft.volume%253D43%26rft.spage%253D959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.35" data-doi="10.1146/annurev.genet.35.102401.091142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meisler</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearney</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ottman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Escayg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification of epilepsy genes in human and mouse.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DMeisler%26rft.auinit1%253DM.%2BH.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D567%26rft.epage%253D588%26rft.atitle%253DIDENTIFICATION%2BOF%2BEPILEPSY%2BGENES%2BIN%2BHUMAN%2BAND%2BMOUSE%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.35.102401.091142%26rft_id%253Dinfo%253Apmid%252F11700294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.35.102401.091142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700294&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172991500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ness</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.-X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Nuclear gene variation in wild brown rats.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1661</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.37" data-doi="10.1093/jhered/92.1.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuermer</span>  <span class="cit-name-given-names">I. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Lengerken</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gattermann</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Low microsatellite variation in laboratory gerbrils.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DNeumann%26rft.auinit1%253DK.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D74%26rft.atitle%253DLow%2BMicrosatellite%2BVariation%2Bin%2BLaboratory%2BGerbils%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F92.1.71%26rft_id%253Dinfo%253Apmid%252F11336232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/92.1.71&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11336232&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168342100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.38" data-doi="10.1258/002367780780943015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Papaioannou</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Festing</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Genetic drift in a stock of laboratory mice.</span> <abbr class="cit-jnl-abbrev">Lab. Anim.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLaboratory%2BAnimals%26rft.stitle%253DLab%2BAnim%26rft.aulast%253DPapaioannou%26rft.auinit1%253DV.%2BE.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D13%26rft.atitle%253DGenetic%2Bdrift%2Bin%2Ba%2Bstock%2Bof%2Blaboratory%2Bmice%26rft_id%253Dinfo%253Adoi%252F10.1258%252F002367780780943015%26rft_id%253Dinfo%253Apmid%252F6928500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1258/002367780780943015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6928500&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.39" data-doi="10.1093/jhered/92.1.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petrij</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Veen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mettler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brückmann</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A second acromelanistic allelomorph at the albino locus of the Mongolian gerbil (<em>Meriones unguiculatus</em>).</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DPetrij%26rft.auinit1%253DF.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D74%26rft.epage%253D78%26rft.atitle%253DA%2BSecond%2BAcromelanistic%2BAllelomorph%2Bat%2Bthe%2BAlbino%2BLocus%2Bof%2Bthe%2BMongolian%2BGerbil%2B%2528Meriones%2Bunguiculatus%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F92.1.74%26rft_id%253Dinfo%253Apmid%252F11336233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/92.1.74&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11336233&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168342100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.40" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.41" data-doi="10.1093/jhered/esg097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Razzoli</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">AFLP to assess genetic variation in laboratory gerbils (<em>Meriones unguiculatus</em>).</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DRazzoli%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D6%26rft.spage%253D507%26rft.epage%253D511%26rft.atitle%253DAFLP%2Bto%2BAssess%2BGenetic%2BVariation%2Bin%2BLaboratory%2BGerbils%2B%2528Meriones%2Bunguiculatus%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesg097%26rft_id%253Dinfo%253Apmid%252F14691317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esg097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14691317&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187987900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rich</span>  <span class="cit-name-given-names">S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Diabetes: still a geneticist’s nightmare.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">536</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.43" data-doi="10.1371/journal.pone.0016461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zipser</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewejohann</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachser</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effect of population heterogenization on the reproducibility of mouse behavior: a multi-laboratory study.</span> <abbr class="cit-jnl-abbrev">PLoS One.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e16461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRichter%26rft.auinit1%253DS.%2BH.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De16461%26rft.epage%253De16461%26rft.atitle%253DEffect%2Bof%2Bpopulation%2Bheterogenization%2Bon%2Bthe%2Breproducibility%2Bof%2Bmouse%2Bbehavior%253A%2Ba%2Bmulti-laboratory%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016461%26rft_id%253Dinfo%253Apmid%252F21305027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21305027&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.44" data-doi="10.1007/s00335-007-9045-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The polymorphism architecture of mouse genetic resources elucidated using genome-wide resequencing data: implications for QTL discovery and systems genetics.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DRoberts%26rft.auinit1%253DA.%26rft.volume%253D18%26rft.issue%253D6-7%26rft.spage%253D473%26rft.epage%253D481%26rft.atitle%253DThe%2Bpolymorphism%2Barchitecture%2Bof%2Bmouse%2Bgenetic%2Bresources%2Belucidated%2Busing%2Bgenome-wide%2Bresequencing%2Bdata%253A%2Bimplications%2Bfor%2BQTL%2Bdiscovery%2Band%2Bsystems%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-007-9045-1%26rft_id%253Dinfo%253Apmid%252F17674098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-007-9045-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17674098&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249125700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.45" data-doi="10.1046/j.1471-8286.2003.00566.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">distruct: a program for the graphical display of population structure.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D4%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8286.2003.00566.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8286.2003.00566.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.46" data-doi="10.1126/science.1078311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic structure of human populations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">2381</span>–<span class="cit-lpage">2385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D298%26rft.issue%253D5602%26rft.spage%253D2381%26rft.epage%253D2385%26rft.atitle%253DGenetic%2BStructure%2Bof%2BHuman%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1078311%26rft_id%253Dinfo%253Apmid%252F12493913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTYwMi8yMzgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.679.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">W. M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burch</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-source">The Principles of Humane Experimental Technique</span>. <span class="cit-publ-name">Methuen &amp; Co. Ltd.</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.48" data-doi="10.1534/genetics.107.079988"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salcedo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geraldes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Nucleotide variation in wild and inbred mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2277</span>–<span class="cit-lpage">2291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSalcedo%26rft.auinit1%253DT.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2277%26rft.epage%253D2291%26rft.atitle%253DNucleotide%2BVariation%2Bin%2BWild%2Band%2BInbred%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.079988%26rft_id%253Dinfo%253Apmid%252F18073432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIyNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.49" data-doi="10.1111/j.1365-294X.2010.04554.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santure</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkhead</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the use of large marker panels to estimate inbreeding and relatedness: empirical and simulation studies of a pedigreed zebra finch population typed at 771 SNPs.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1439</span>–<span class="cit-lpage">1451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSanture%26rft.auinit1%253DA.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D1439%26rft.epage%253D1451%26rft.atitle%253DOn%2Bthe%2Buse%2Bof%2Blarge%2Bmarker%2Bpanels%2Bto%2Bestimate%2Binbreeding%2Band%2Brelatedness%253A%2Bempirical%2Band%2Bsimulation%2Bstudies%2Bof%2Ba%2Bpedigreed%2Bzebra%2Bfinch%2Bpopulation%2Btyped%2Bat%2B771%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04554.x%26rft_id%253Dinfo%253Apmid%252F20149098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04554.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20149098&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275761300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serikawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuramoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hilbert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Rat gene mapping using PCR-analyzed microsatellites.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">701</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSerikawa%26rft.auinit1%253DT.%26rft.volume%253D131%26rft.issue%253D3%26rft.spage%253D701%26rft.epage%253D721%26rft.atitle%253DRat%2Bgene%2Bmapping%2Busing%2BPCR-analyzed%2Bmicrosatellites.%26rft_id%253Dinfo%253Apmid%252F1628813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMxLzMvNzAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.51" data-doi="10.1016/j.neuron.2016.08.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sittig</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbonetto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krauss</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrios-Camacho</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic background limits generalizability of genotype-phenotype relationships.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">1253</span>–<span class="cit-lpage">1259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D91%26rft.spage%253D1253%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2016.08.013%26rft_id%253Dinfo%253Apmid%252F27618673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2016.08.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27618673&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.52" data-doi="10.1101/gr.2155004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smits</span>  <span class="cit-name-given-names">B. M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Zutphen</span>  <span class="cit-name-given-names">B. F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plasterk</span>  <span class="cit-name-given-names">R. H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuppen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic variation in coding regions between and within commonly used inbred rat strains.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1285</span>–<span class="cit-lpage">1290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSmits%26rft.auinit1%253DB.%2BM.G.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1285%26rft.epage%253D1290%26rft.atitle%253DGenetic%2BVariation%2Bin%2BCoding%2BRegions%2BBetween%2Band%2BWithin%2BCommonly%2BUsed%2BInbred%2BRat%2BStrains%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2155004%26rft_id%253Dinfo%253Apmid%252F15231744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvMTI4NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi82NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.53" data-doi="10.1534/genetics.116.198838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarsani</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigmon</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomes of the mouse collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198838%26rft_id%253Dinfo%253Apmid%252F28592495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.54" data-doi="10.1016/j.bbr.2006.05.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuermer</span>  <span class="cit-name-given-names">I. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wetzel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Early experience and domestication affect auditory discrimination learning, open field behaviour and brain size in wild Mongolian gerbils and domesticated laboratory gerbils (<em>Meriones unguiculatus forma domestica</em>).</span> <abbr class="cit-jnl-abbrev">Behav. Brain Res.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavioural%2Bbrain%2Bresearch%26rft.stitle%253DBehav%2BBrain%2BRes%26rft.aulast%253DStuermer%26rft.auinit1%253DI.%2BW.%26rft.volume%253D173%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D21%26rft.atitle%253DEarly%2Bexperience%2Band%2Bdomestication%2Baffect%2Bauditory%2Bdiscrimination%2Blearning%252C%2Bopen%2Bfield%2Bbehaviour%2Band%2Bbrain%2Bsize%2Bin%2Bwild%2BMongolian%2Bgerbils%2Band%2Bdomesticated%2Blaboratory%2Bgerbils%2B%2528Meriones%2Bunguiculatus%2Bforma%2Bdomestica%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbr.2006.05.025%26rft_id%253Dinfo%253Apmid%252F16846650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbr.2006.05.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16846650&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240652500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.55" data-doi="10.1078/0044-5231-00102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuermer</span>  <span class="cit-name-given-names">I. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leybold</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinke</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalberlah</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Intraspecific allometric comparison of laboratory gerbils with Mongolian gerbils trapped in the wild indicates domestication in <em>Meriones unguiculatus</em> (Milne-Edwards, 1867) (Rodentia: Gerbillinae).</span> <abbr class="cit-jnl-abbrev">Zool. Anz.</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool.%2BAnz.%26rft.volume%253D242%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1078%252F0044-5231-00102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1078/0044-5231-00102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.56" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the Mouse Diversity outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.57" data-doi="10.1016/0031-9384(80)90229-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thiessen</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kittrell</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The Harderian gland and thermoregulation in the gerbil (<em>Meriones unguiculatus</em>).</span> <abbr class="cit-jnl-abbrev">Physiol. Behav.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiology%2B%2526%2BBehavior%26rft.stitle%253DPhysiology%2B%2526%2BBehavior%26rft.aulast%253DThiessen%26rft.auinit1%253DD.%2BD.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D417%26rft.epage%253D424%26rft.atitle%253DThe%2BHarderian%2Bgland%2Band%2Bthermoregulation%2Bin%2Bthe%2Bgerbil%2B%2528Meriones%2Bunguiculatus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0031-9384%252880%252990229-2%26rft_id%253Dinfo%253Apmid%252F7378139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0031-9384(80)90229-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7378139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.58" data-doi="10.1534/genetics.111.138032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ten years of the collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.138032%26rft_id%253Dinfo%253Apmid%252F22345604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzY3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodrick</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">The pathology of the Mongolian gerbil (<em>Meriones unguiculatus</em>): a review.</span> <abbr class="cit-jnl-abbrev">Lab. Anim. Sci.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLaboratory%2Banimal%2Bscience%26rft.stitle%253DLab%2BAnim%2BSci%26rft.aulast%253DVincent%26rft.auinit1%253DA.%2BL.%26rft.volume%253D29%26rft.issue%253D5%26rft.spage%253D645%26rft.epage%253D651%26rft.atitle%253DThe%2Bpathology%2Bof%2Bthe%2BMongolian%2BGerbil%2B%2528Meriones%2Bunguiculatus%2529%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Apmid%252F92595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=92595&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Metabolism and thermoregulation in the Mongolian gerbil <em>Meriones unguiculatus</em>.</span> <abbr class="cit-jnl-abbrev">Acta Theriol. (Warsz.)</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BTheriol.%2B%2528Warsz.%2529%26rft.volume%253D45%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.61" data-doi="10.1038/nature11816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Discrete genetic modules are responsible for complex burrow evolution in <em>Peromyscus</em> mice.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">493</span>: <span class="cit-fpage">402</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D493%26rft.spage%253D402%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11816%26rft_id%253Dinfo%253Apmid%252F23325221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23325221&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313615900051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.62" data-doi="10.1371/journal.pgen.1001085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicod</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhomra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Commercially available outbred mice for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001085</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001085%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001085%26rft_id%253Dinfo%253Apmid%252F20838427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838427&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.679.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D28%26rft.spage%253D3326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/679.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/669" title="Use of a Sibling Subtraction Method for Identifying Causal Mutations in Caenorhabditis elegans by Whole-Genome Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/687" title="Differential Regulation of Zfp30 Expression in Murine Airway Epithelia Through Altered Binding of ZFP148 to rs51434084" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/679" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FAdbY4FgXMr-J5c-hdXKDVwnddasEyGv6hJr6zZyYdc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/679" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/679" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/679" class="active" data-icon-position="" data-hide-link-title="0">Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488304" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-cngyA7nb9fMt-KGaX_pVsOGMqlZc1621rft0uWU4EXs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/679.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/679" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ddY_7WArNiKeFFujl9slchdnehpoez27122rLsCKjBE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/679" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/679.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488304" id="node488304--2" data-pisa="ggg;8/2/679" data-pisa-master="ggg;g3.117.300495" data-apath="/ggg/8/2/679.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4479-1847" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas D. Brekke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4896-8857" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katherine A. Steele</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1537-7316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John F. Mulley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">679-686;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300495 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488304--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas D. Brekke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BD.%2BBrekke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brekke%20TD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BD.%2BBrekke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4479-1847" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas D. Brekke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katherine A. Steele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environment, Natural Resources and Geography, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BA.%2BSteele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steele%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatherine%2BA.%2BSteele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4896-8857" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katherine A. Steele</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John F. Mulley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BF.%2BMulley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mulley%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BF.%2BMulley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1537-7316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John F. Mulley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.mulley@bangor.ac.uk" class="" data-icon-position="" data-hide-link-title="0">j.mulley@bangor.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/679" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/679&amp;title=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/679&amp;title=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/679&amp;title=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/679&amp;text=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/679&amp;title=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/679&amp;t=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/679&amp;title=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/679&amp;title=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488304/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488304" id="node488304--3" data-pisa="ggg;8/2/679" data-pisa-master="ggg;g3.117.300495" data-apath="/ggg/8/2/679.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inbred or Outbred? Genetic Diversity in Laboratory Rodent Colonies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4479-1847" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas D. Brekke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4896-8857" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katherine A. Steele</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1537-7316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John F. Mulley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">679-686;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300495 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488304--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas D. Brekke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BD.%2BBrekke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brekke%20TD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BD.%2BBrekke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4479-1847" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas D. Brekke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katherine A. Steele </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Environment, Natural Resources and Geography, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BA.%2BSteele%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steele%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatherine%2BA.%2BSteele%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4896-8857" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katherine A. Steele</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John F. Mulley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Biological Sciences, Bangor University, LL57 2DG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BF.%2BMulley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mulley%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BF.%2BMulley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1537-7316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John F. Mulley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.mulley@bangor.ac.uk" class="" data-icon-position="" data-hide-link-title="0">j.mulley@bangor.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=679&amp;rft.epage=686&amp;rft.atitle=Inbred%20or%20Outbred%3F%20Genetic%20Diversity%20in%20Laboratory%20Rodent%20Colonies&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Brekke%2C+Thomas+D.&amp;rft.au=Steele%2C+Katherine+A.&amp;rft.au=Mulley%2C+John+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488304/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488304/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488304/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488304/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488304/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488304/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488304/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488304/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488304/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488304/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488304/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488304/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/679" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/679.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/679.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/679.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/679.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
