<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/2/599" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/2/599.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/2/599.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/2/599.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/2/599/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/2/599.atom" />
<meta name="HW.pisa" content="ggg;8/2/599" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example" />
<meta name="DC.Identifier" content="10.1534/g3.117.300483" />
<meta name="DC.Date" content="2018-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Liu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multiparent populations (MPP) have become popular resources for complex trait mapping because of their wider allelic diversity and larger population size compared with traditional two-way recombinant inbred (RI) strains. In mice, the collaborative cross (CC) is one of the most popular MPP and is derived from eight genetically diverse inbred founder strains. The strategy of generating RI intercrosses (RIX) from MPP in general and from the CC in particular can produce a large number of completely reproducible heterozygote genomes that better represent the (outbred) human population. Since both maternal and paternal haplotypes of each RIX are readily available, RIX is a powerful resource for studying both standing genetic and epigenetic variations of complex traits, in particular, the parent-of-origin (PoO) effects, which are important contributors to many complex traits. Furthermore, most complex traits are affected by &gt;1 genes, where multiple quantitative trait locus mapping could be more advantageous. In this paper, for MPP-RIX data but taking CC-RIX as a working example, we propose a general Bayesian variable selection procedure to simultaneously search for multiple genes with founder allelic effects and PoO effects. The proposed model respects the complex relationship among RIX samples, and the performance of the proposed method is examined by extensive simulations." />
<meta name="DC.Contributor" content="Yanyan Liu" />
<meta name="DC.Contributor" content="Sican Xiong" />
<meta name="DC.Contributor" content="Wei Sun" />
<meta name="DC.Contributor" content="Fei Zou" />
<meta name="article:published_time" content="2018-02-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multiparent populations (MPP) have become popular resources for complex trait mapping because of their wider allelic diversity and larger population size compared with traditional two-way recombinant inbred (RI) strains. In mice, the collaborative cross (CC) is one of the most popular MPP and is derived from eight genetically diverse inbred founder strains. The strategy of generating RI intercrosses (RIX) from MPP in general and from the CC in particular can produce a large number of completely reproducible heterozygote genomes that better represent the (outbred) human population. Since both maternal and paternal haplotypes of each RIX are readily available, RIX is a powerful resource for studying both standing genetic and epigenetic variations of complex traits, in particular, the parent-of-origin (PoO) effects, which are important contributors to many complex traits. Furthermore, most complex traits are affected by &amp;gt;1 genes, where multiple quantitative trait locus mapping could be more advantageous. In this paper, for MPP-RIX data but taking CC-RIX as a working example, we propose a general Bayesian variable selection procedure to simultaneously search for multiple genes with founder allelic effects and PoO effects. The proposed model respects the complex relationship among RIX samples, and the performance of the proposed method is examined by extensive simulations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/02/01" />
<meta name="citation_mjid" content="ggg;8/2/599" />
<meta name="citation_id" content="8/2/599" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/2/599" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/2/599.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/2/599.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/2/599.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300483" />
<meta name="citation_pmid" content="29255115" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="599" />
<meta name="citation_lastpage" content="605" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yanyan Liu" />
<meta name="citation_author_institution" content="School of Mathematics and Statistics, Wuhan University, Hubei 430072, China" />
<meta name="citation_author" content="Sican Xiong" />
<meta name="citation_author_institution" content="School of Mathematics and Statistics, Wuhan University, Hubei 430072, China" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8304-5663" />
<meta name="citation_author" content="Wei Sun" />
<meta name="citation_author_institution" content="Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington 98109" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6350-1107" />
<meta name="citation_author" content="Fei Zou" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of North Carolina at Chapel Hill, North Carolina 27599" />
<meta name="citation_author_email" content="feizou@email.unc.edu" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=D. P. Barlow;citation_title=Genomic imprinting: a mammalian epigenetic discovery model.;citation_pages=379-403;citation_volume=45;citation_year=2011;citation_pmid=21942369;citation_doi=10.1146/annurev-genet-110410-132459" />
<meta name="citation_reference" content="citation_title=QTL Cartographer: A Reference Manual and Tutorial for QTL Mapping.;citation_author=C. J. Basten;citation_author=B. S. Weir;citation_author=Z.-B. Zeng;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Lab animal;citation_journal_abbrev=Lab Anim (NY);citation_author=K. W. Broman;citation_title=Review of statistical methods for QTL mapping in experimental crosses.;citation_pages=44-52;citation_volume=30;citation_year=2001;citation_issue=7;citation_pmid=11469113" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=C. Cavanagh;citation_author=M. Morell;citation_author=I. Mackay;citation_author=W. Powell;citation_title=From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.;citation_pages=215-221;citation_volume=11;citation_year=2008;citation_issue=2;citation_pmid=18295532;citation_doi=10.1016/j.pbi.2008.01.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the collaborative cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. J. Crowley;citation_author=Y. Kim;citation_author=A. B. Lenarcic;citation_author=C. R. Quackenbush;citation_author=C. J. Barrick;citation_title=Genetics of adverse reactions to haloperidol in a mouse diallel: a drug–placebo experiment and Bayesian causal analysis.;citation_volume=196;citation_year=2014;citation_pmid=24240528;citation_doi=10.1534/genetics.113.156901" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D.-J. de Koning;citation_author=L. M. McIntyre;citation_title=Back to the future: multiparent populations provide the key to unlocking the genetic basis of complex traits.;citation_volume=206;citation_year=2017;citation_pmid=28592493;citation_doi=10.1534/genetics.117.203265" />
<meta name="citation_reference" content="citation_journal_title=Pattern Recognit. Lett.;citation_author=T. Fawcett;citation_title=An introduction to ROC analysis.;citation_pages=861-874;citation_volume=27;citation_year=2006;citation_doi=10.1016/j.patrec.2005.10.010" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=E. I. George;citation_author=R. E. McCulloch;citation_title=Variable selection via Gibbs sampling.;citation_pages=881-889;citation_volume=88;citation_year=1993;citation_doi=10.2307/2290777" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Gong;citation_author=F. Zou;citation_title=Varying coefficient models for mapping quantitative trait loci using recombinant inbred intercrosses.;citation_volume=190;citation_year=2012;citation_pmid=22345613;citation_doi=10.1534/genetics.111.132522" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Hallin;citation_author=K. Märtens;citation_author=A. I. Young;citation_author=M. Zackrisson;citation_author=F. Salinas;citation_title=Powerful decomposition of complex traits in a diploid model.;citation_pages=13311;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. E. Huang;citation_author=K. L. Verbyla;citation_author=A. P. Verbyla;citation_author=C. Raghavan;citation_author=V. K. Singh;citation_title=MAGIC populations in crops: current status and future prospects.;citation_pages=999-1017;citation_volume=128;citation_year=2015;citation_pmid=25855139;citation_doi=10.1007/s00122-015-2506-0" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=I. Ionita-Laza;citation_author=S. Lee;citation_author=V. Makarov;citation_author=J. D. Buxbaum;citation_author=X. Lin;citation_title=Sequence kernel association tests for the combined effect of rare and common variants.;citation_pages=841-853;citation_volume=92;citation_year=2013;citation_pmid=23684009;citation_doi=10.1016/j.ajhg.2013.04.015" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=F. A. Iraqi;citation_author=G. Churchill;citation_author=R. Mott;citation_title=The Collaborative Cross, developing a resource for mammalian systems genetics: a status report of the Wellcome Trust cohort.;citation_pages=379-381;citation_volume=19;citation_year=2008;citation_issue=6;citation_pmid=18521666;citation_doi=10.1007/s00335-008-9113-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Jansen;citation_author=P. Stam;citation_title=High resolution of quantitative traits into multiple loci via interval mapping.;citation_pages=1447-1455;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013917" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C.-H. Kao;citation_author=Z.-B. Zeng;citation_title=General formulas for obtaining the MLEs and the asymptotic variance-covariance matrix in mapping quantitative trait loci when using the EM algorithm.;citation_pages=653-665;citation_volume=53;citation_year=1997;citation_issue=2;citation_pmid=9192455;citation_doi=10.2307/2533965" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C.-H. Kao;citation_author=Z.-B. Zeng;citation_author=R. D. Teasdale;citation_title=Multiple Interval Mapping for Quantitative Trait Loci;citation_pages=1203-1216;citation_volume=152;citation_year=1999;citation_issue=3;citation_pmid=10388834" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. G. King;citation_author=C. M. Merkes;citation_author=C. L. McNeil;citation_author=S. R. Hoofer;citation_author=S. Sen;citation_title=Genetic dissection of a model complex trait using the Drosophila synthetic population resource.;citation_volume=22;citation_year=2012;citation_pmid=22496517;citation_doi=10.1101/gr.134031.111" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_title=Sankhya Ser. B;citation_author=L. Kuo;citation_author=B. Mallick;citation_title=Variable selection for regression models.;citation_pages=65-81;citation_volume=60;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Appl. Transl. Genomics;citation_author=O. Ladejobi;citation_author=J. Elderfield;citation_author=K. A. Gardner;citation_author=R. C. Gaynor;citation_author=J. Hickey;citation_title=Maximizing the potential of multi-parental crop populations.;citation_pages=9-17;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=H. A. Lawson;citation_author=J. M. Cheverud;citation_author=J. B. Wolf;citation_title=Genomic imprinting and parent-of-origin effects on complex traits.;citation_pages=609-617;citation_volume=14;citation_year=2013;citation_pmid=23917626;citation_doi=10.1038/nrg3543" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. B. Lenarcic;citation_author=K. L. Svenson;citation_author=G. A. Churchill;citation_author=W. Valdar;citation_title=A general Bayesian approach to analyzing diallel crosses of inbred strains.;citation_volume=190;citation_year=2012;citation_pmid=22345610;citation_doi=10.1534/genetics.111.132563" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=Z. Lu;citation_author=H. Zhu;citation_author=R. C. Knickmeyer;citation_author=P. F. Sullivan;citation_author=W. N. Stephanie;citation_title=Multiple SNP-sets analysis for genome-wide association studies through Bayesian latent variable selection.;citation_pages=664-677;citation_volume=39;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=K. F. Manly;citation_author=J. M. Olson;citation_title=Overview of QTL mapping software and introduction to map manager QT.;citation_pages=327-334;citation_volume=10;citation_year=1999;citation_issue=4;citation_pmid=10087288;citation_doi=10.1007/s003359900997" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. J. Mitchell;citation_author=J. J. Beauchamp;citation_title=Bayesian variable selection in linear regression.;citation_pages=1023-1032;citation_volume=83;citation_year=1988;citation_doi=10.2307/2290129" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=R. Mott;citation_author=W. Yuan;citation_author=P. Kaisaki;citation_author=X. Gan;citation_author=J. Cleak;citation_title=The architecture of parent-of-origin effects in mice.;citation_pages=332-342;citation_volume=156;citation_year=2014;citation_pmid=24439386;citation_doi=10.1016/j.cell.2013.11.043" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. L. Rasmussen;citation_author=A. Okumura;citation_author=M. T. Ferris;citation_author=R. Green;citation_author=F. Feldmann;citation_title=Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance.;citation_volume=346;citation_year=2014;citation_pmid=25359852;citation_doi=10.1126/science.1259595" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. J. F. ter Braak;citation_author=M. P. Boer;citation_author=M. C. A. M. Bink;citation_title=Extending Xu’s Bayesian model for estimating polygenic effects using markers of the entire genome.;citation_volume=170;citation_year=2005;citation_pmid=15911578;citation_doi=10.1534/genetics.105.040469" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. W. Threadgill;citation_author=G. A. Churchill;citation_title=Ten years of the collaborative cross.;citation_volume=190;citation_year=2012;citation_pmid=22345604;citation_doi=10.1534/genetics.111.138032" />
<meta name="citation_reference" content="citation_journal_title=ILAR Journal;citation_journal_abbrev=ILAR J;citation_author=D. W. Threadgill;citation_author=D. R. Miller;citation_author=G. A. Churchill;citation_author=F. P.-M. de Villena;citation_title=The collaborative cross: a recombinant inbred mouse population for the systems genetic era.;citation_pages=24-31;citation_volume=52;citation_year=2011;citation_issue=1;citation_pmid=21411855" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=S.-W. Tsaih;citation_author=L. Lu;citation_author=D. C. Airey;citation_author=R. W. Williams;citation_author=G. A. Churchill;citation_title=Quantitative trait mapping in a diallel cross of recombinant inbred lines.;citation_pages=344-355;citation_volume=16;citation_year=2005;citation_issue=5;citation_pmid=16104382;citation_doi=10.1007/s00335-004-2466-1" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. Vered;citation_author=C. Durrant;citation_author=R. Mott;citation_author=F. A. Iraqi;citation_title=Susceptibility to Klebsiella pneumonaie infection in collaborative cross mice is a complex trait controlled by at least three loci acting at different time points.;citation_pages=865;citation_volume=15;citation_year=2014;citation_pmid=25283706;citation_doi=10.1186/1471-2164-15-865" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Wei;citation_author=S. Xu;citation_title=A random-model approach to QTL mapping in multiparent advanced generation intercross (MAGIC) populations.;citation_volume=202;citation_year=2016;citation_pmid=26715662;citation_doi=10.1534/genetics.115.179945" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=L. C. S. Woods;citation_author=K. L. Holl;citation_author=D. Oreper;citation_author=Y. Xie;citation_author=S.-W. Tsaih;citation_title=Fine-mapping diabetes-related traits, including insulin resistance, in heterogeneous stock rats.;citation_volume=44;citation_year=2012;citation_pmid=22947656;citation_doi=10.1152/physiolgenomics.00040.2012" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. C. Wu;citation_author=P. Kraft;citation_author=M. P. Epstein;citation_author=D. M. Taylor;citation_author=S. J. Chanock;citation_title=Powerful SNP-set analysis for case-control genome-wide association studies.;citation_pages=929-942;citation_volume=86;citation_year=2010;citation_issue=6;citation_pmid=20560208;citation_doi=10.1016/j.ajhg.2010.05.002" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Yang;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=GCTA: a tool for genome-wide complex trait analysis.;citation_pages=76-82;citation_volume=88;citation_year=2011;citation_issue=1;citation_pmid=21167468;citation_doi=10.1016/j.ajhg.2010.11.011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic design and statistical power of nested association mapping in maize.;citation_volume=178;citation_year=2008;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Yuan;citation_author=F. Zou;citation_author=Y. Liu;citation_title=Bayesian multiple quantitative trait loci mapping for recombinant inbred intercrosses.;citation_volume=188;citation_year=2011;citation_pmid=21385723;citation_doi=10.1534/genetics.110.125542" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z.-B. Zeng;citation_title=Precision mapping of quantitative trait loci.;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013918" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Zhang;citation_author=W. Wang;citation_author=W. Valdar;citation_title=Bayesian modeling of haplotype effects in multiparent populations.;citation_volume=198;citation_year=2014;citation_pmid=25236455;citation_doi=10.1534/genetics.114.166249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Zou;citation_author=J. A. L. Gelfond;citation_author=D. C. Airey;citation_author=L. Lu;citation_author=K. F. Manly;citation_title=Quantitative trait locus analysis using recombinant inbred intercrosses: theoretical and empirical considerations.;citation_volume=170;citation_year=2005;citation_pmid=15879512;citation_doi=10.1534/genetics.104.035709" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
<meta name="twitter:description" content="Multiparent populations (MPP) have become popular resources for complex trait mapping because of their wider allelic diversity and larger population size compared with traditional two-way recombinant inbred (RI) strains. In mice, the collaborative cross (CC) is one of the most popular MPP and is derived from eight genetically diverse inbred founder strains. The strategy of generating RI intercrosses (RIX) from MPP in general and from the CC in particular can produce a large number of completely reproducible heterozygote genomes that better represent the (outbred) human population. Since both maternal and paternal haplotypes of each RIX are readily available, RIX is a powerful resource for studying both standing genetic and epigenetic variations of complex traits, in particular, the parent-of-origin (PoO) effects, which are important contributors to many complex traits. Furthermore, most complex traits are affected by &gt;1 genes, where multiple quantitative trait locus mapping could be more advantageous. In this paper, for MPP-RIX data but taking CC-RIX as a working example, we propose a general Bayesian variable selection procedure to simultaneously search for multiple genes with founder allelic effects and PoO effects. The proposed model respects the complex relationship among RIX samples, and the performance of the proposed method is examined by extensive simulations." />
<meta name="og-title" property="og:title" content="Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/2/599" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multiparent populations (MPP) have become popular resources for complex trait mapping because of their wider allelic diversity and larger population size compared with traditional two-way recombinant inbred (RI) strains. In mice, the collaborative cross (CC) is one of the most popular MPP and is derived from eight genetically diverse inbred founder strains. The strategy of generating RI intercrosses (RIX) from MPP in general and from the CC in particular can produce a large number of completely reproducible heterozygote genomes that better represent the (outbred) human population. Since both maternal and paternal haplotypes of each RIX are readily available, RIX is a powerful resource for studying both standing genetic and epigenetic variations of complex traits, in particular, the parent-of-origin (PoO) effects, which are important contributors to many complex traits. Furthermore, most complex traits are affected by &gt;1 genes, where multiple quantitative trait locus mapping could be more advantageous. In this paper, for MPP-RIX data but taking CC-RIX as a working example, we propose a general Bayesian variable selection procedure to simultaneously search for multiple genes with founder allelic effects and PoO effects. The proposed model respects the complex relationship among RIX samples, and the performance of the proposed method is examined by extensive simulations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2.cover-source.jpg" />
    <title>Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Qcku3i1_mrvlJ223te4VmaufDU39ynvTiX0paEIh0-w","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"488303","apath":"\/ggg\/8\/2\/599.atom","pisa":"ggg;8\/2\/599","ac":{"\/ggg\/8\/2\/599.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/2\/599.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/2\/599"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/2\/599"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/2\/599":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022L4ePeuyczn1Pa2TiWz\\\\\\\/QYvY1Anu9PgjftqjcTEYQqm8MBVFUTSGLaTDNd3+wxasr46Miru4sOTj6WsdigfVouz8gwVW3RhWWqeo4dqC4pojQ8mQXxlBqXA6h7XIztKkiUmYDJ8Bo\\\\\\\/aTUj\\\\\\\/GtOJ0xfvZLe2hhfVoreA24uRC1Kum\\\\\\\/nKuZnWk0WGLbzuTc8F98G22tqZje1RLF6Wt1Pk3\\\\\\\/mIfDjNRtNspxtXP\\\\\\\/tkUy9uT952W02WvHDdrgRs3f7QA9xsfUPKwOq2DiNRy4J4YEe01XCj3WJvTFlug1mxmzfbnkoVUziIGhRJr0W7d+ps4rWjl5Z2mnFUNVMk5E11LU+2P2Oc39nf1V0RIXupbNIhr3OnEMKvLqDoDV+EZrt4BJDUKMhlB3AaefIkkhC587qihedUBN5Adoa\\\\\\\/4z9jG5awekDEqiYl1RVl\\\\\\\/Ry2D42S\\\\\\\/Yxp0PPdJJSXTE8ab+z2Tcf99VokXFcZBV4NiMw+BQrhZ7YFm4oEta6hjbYyDgF4gRrqBzZzKvo+o4lXKVykYZ0tIm+LkUNAdcDmo7r0+XlVamabPpmXXxfB2kyP899xm78ffsgjx8UFJCSeduTG+emDgeqt2PcpmpYggqy479Mz6IqvzDfVjr8pk9+eNhHdPXzHCVMy0VCnr89sKiT1hbKbOBDQiceSz43H\\\\\\\/yKtS\\\\\\\/PG0NUgC1EcGeuS7pJLomN4m+vmmpDGLCVTHjp5t7nr2obKIe1yOmEvpd33Du8lfiPB9AtWY\\\\\\\/cn1bT3+zlD96LTyWqtKrOTXaBuRMLuse8j7O9GN613glaBBRMS4kFspDknrOLqnVRM3fchUlFOMLdzSHI9LWVCiSV0uUrgCTYgTu0Vq56cemyeClrWTfjPwlsY48JAChsRw38FhRl5svCpFhx0zVYmrz2n\\\\\\\/a1yHZ3s0vg1dIusGTgP6cV0EyCSh5rUTacuM4qHMZQxdrv15f7yFuZcgx2eXYcO9CrK3VnzI8AoOnlqybqCwvo93g+wXjhB7OVD6+wSPnRvRzkoRKPIk9GKkvvEHj9agi8H530zRVQAMWUbRuI2Vg8Nd07khgRlOQLn1tIKnJGUhNlf2TvKc\\\\\\\/PSCuuXNGyJVY2tsElghY8vGSpANHFSy5npkkykm06jZ66zf8vywLn3xZJV8OiMxxJUhtSGpvGZXuGp\\\\\\\/94TUxz68nOYCklUx10x6DnWWpArC657K+BmMmUn6MLe2Bswd6UVbJuZZQAi0exEEu8m7PmI1kusY\\\\\\\/Whp5dUnxBVSmUCg+zPAhHJ74rj8AwmlDiEdz\\\u0022,\\\u0022iv\\\u0022:\\\u0022N5WDfEGfsoYuWA2FMvJ0aA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022db464d25d1413741a634e533ff07c8ae0f144e2029b300df171ce3b1f8a00227\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EO6gstDJIKfP9uq7U61WuRR6OovhPE1EjtbcS3CYTD0zmH\\\\\\\/KZ+bjBoQFvugxSfWw3qfZbQatA2Indm\\\\\\\/okPV2jHaSEJ8KXElJnVFRA7FarvA8JCi9fzn1Ejqgp\\\\\\\/qKYkA0fwrEms76xfKzvMSMfuimx5LWDocpFzIJSzbEO+8rzg2DebrXpZZvdhj4TE8aeeuq2Kl9st7GSxsg5dIrxmTgVB4W4OXNMrk0258+XpnO2HkTlVByhaA16bxAJR\\\\\\\/cepVQgS5wAVEJVyV\\\\\\\/QzO\\\\\\\/4RBNvQwBn\\\\\\\/+Zv\\\\\\\/2IBfan5kKMtoLrcY6mrQlRsV0Vc\\\\\\\/3BETVl7MC+hK8+zoOnqKQYzEcEdrkz2JVXF4HBwHfHts3V3Rvnyzel0i3EwolCAhKO\\\\\\\/h3Zb4JjR3zsHvp\\\\\\\/hgXxrit5RMoa6c4uZWT3JuBvCa1q+JZocvNZSTx1hx9vwPZaMmBXF2W0cOxyzb3nHnoGiPdfE3ewRYuBM+7R2jxm97oVXnv7JP9O9hjMm8BA+ZON+Sm0YOr1bouY\\\\\\\/AXetHJopnoQD3h2vyfy7rggmEUdPpakHaM1\\\\\\\/QUXLF\\\\\\\/gZiFEPfZhKbdEeZBjoWmSl7TSA0FHuFOCgm7iitw4knOqO6rhyXoNIt0C+EhtILmLqQxBj4qtBDIH982nE1ND03T\\\\\\\/KUsrbgFjEJye8Io5uUNRRkq3CK0Cn29l5+GfXAWAz+i3HwTzlLe\\\\\\\/5J9A\\\\\\\/BsPaWbCrckIcPf\\\\\\\/Y9bZtsGx8YIZ4JcQMe2puNBjsVcWPCjdUGrrbz\\\\\\\/cXY\\\\\\\/YpWTR9DW3NeJ\\\\\\\/x8MZTS7+B2n5bMYjknS5d9csZwxUndk8k+9D1FL\\\\\\\/Cok0VQPPLdToJC6B0x2fk+65VFCABNCNtF97msQRYUAUfQQvYfYidwFrxSS23idTk2MYngadAPRu\\\\\\\/1rPbZz+KeRo\\\\\\\/7wLC5ILvRsScBqHfYPO50p6YFrqZr4FY8glNtxDjImRcFKjpZlwvW9chPuLqLgBApnSH14qT0hdRqcn5ro5lRiixzGZKT5jx2f+HvbKJi1d076kDswvlRJGNAvwwaffpD9QOcx+rwEPaDZG\\\\\\\/aUj0mcNpSDfYWLWjQVJpICPCAYLeBrAGx8Iu5RilNsW91hRDr2mUEcPtfifGbPbdWf07tAi8U7czg50yc4QompCJjZef2JgcPhopT7EVZky4rSXJafFCcYf8ZUJHioDN8Rx\\\\\\\/zWHyV0Z8mIgJtymW0RI3+xrmL5d4fdHI3V1vueaK49Om+JBv4qsj4aFr5wqdx4fOm3ijDv7H4L6tnlO7rPadpM1MMyknLC3KlagxIowgpfu\\\\\\\/IMvLNYI5vdoR9zRiH9g\\\\\\\/yUsscnbQPARhRDeoDs\\\\\\\/DinqoTpwCTENv66wvMHhI9tW4DibFW5VqHFf+wlNZSTzad2RCwHLwfWe\\\\\\\/\\\\\\\/E9kzDaf4BHlhfaMPjgbkymz72erBYWPfm5QnObrEEIgzrC+AiP7yMlil43bqKByZXwNPL+J6X8XJKvX4k1t379uj4bRJnvgyXT+iEdQxBuWVG8+8vY9xEAn43RhXOz0l4IdCqvUdVDRzczzXA1bkdGN5X1QnhbZEZ6W17e+Dh73+859+GF59ufrM0PBoytfG4TVFNn7KZvgNararvGBQGINVulAnf7DiUjnQj3Okbh96PaRmgwOJI\\\\\\\/37GmhtlzbxVrU9+EtaaVbmv\\\\\\\/i+rvgBDWtJK40dxQeKDxcJICkTrPPozpA6x2sCRmeU0UWC+LVD0ePu9xvBpzww9D9d5mN7pTk29Wi2gPAY5VSfQnIyW2\\\\\\\/NNxvHy\\\\\\\/927NZEebay1JKMVKpNeu6QiXeXrEVaQQDqXSRXbCDWwMvZX09e0jOh\\\\\\\/7sDp85UXi80FWMIL9hHnd9aoScsbkLtf240X1ucRfwwgorzFyUQR+rwnzsnimjefQAUKeX2InYijaERQSYJpyQUJTh5T6XORfoEVPN90wZCtKUMnzFbDpZuz0yTs1i1J331k2CAvWXlQrjBg4mLksFk0MQHH+MgYts5DogyEXgn5UgXS+SvPTHfpWW2HA2h9ZmVQCJPh67cxWxeqk\\\\\\\/M3EFrXqBe1qzqo73dqZj59G1G4SYJjIaOiMZYZMn3jatZkMC1TCtcUwfT46PJ3NFzbNRfXt3TVH6aftkGQ2au8QoPuJd+HGqHVW755SaIQska2rfDJRJ2YfK8vW3gjSx3\\\\\\\/spODaqWfUpLL0oxIWR9LibaaEfFO88m0cYkJCwFKP3ixU2hx8E9fGb5p6ztc7WyHzG6oi6K\\\\\\\/Qr\\\\\\\/B9i9CxKyA1TUhuRbKWuoJ5iYmdeD\\\\\\\/MK7XtxNdHtXV3YGu7\\\\\\\/h1G8+dCBPER9ZwMaIIIzHwX6rYslfjIm6ea4cFg8lpdmDGOXKS8MtjYYyZDAo4SbQ4uEUUWnIQElaIgXcb5\\\\\\\/JF2nTY5cQf3E+o+\\\\\\\/ODZg9ELugJnPFPW+LSaF1b+xqYaYN1UQB8P\\\\\\\/4L7hYtiM3km+oPuYaA4OJjPkACx0AIklYbp8SMBK1wPQTjzKGh7pYtix2JMEl\\\\\\\/yaUsPkXr4naZqxJeqNvBiKzxi7zdhp4aGIDiABud5MXsioz96WJqFlh6yoCbVgblOOuK25INHNIc9pgLI8L6rK8abwL01Jg8aRDvtBxWd4GawZIodfxrmrQmdc47coVZZV\\\\\\\/Isy40wiv79hS9wpUUQFkxGJBKlNpkp5L\\\\\\\/XgjrEFsSMgl29uIECsu67VTwgvkWeMVT4n2ETGy\\\\\\\/doqa2rSG9DU5l932nDeD41uN9KBfOR8vTCdlgqpEuyij+oF8x3FDscMkA\\\\\\\/peVBNX2Gbfpvzsp88mvNvOfAEIm8u6yCOew2Gc99zKBvEdQrWUcEX9ryzvoW7eusoVF5lPjA41tT+07YCcrfqqI65MFUJthOdsBXF09tooNSuGGHkuzN6t2RfQXiKzzk+dOAa0lymEsnBu+oAJMiJE6YYr07sEQQ6dKzKm7XEQtXg6y9qh7zgj4YuEnruUF\\\\\\\/GQAR0+dQOxWRKh7PSrm4qEObEVNq+htzwOghcdUkLUj8+1eHG5eDOo+ErQVjSZC9X2Ca56SjSUsylhYF7KwMQBrl7LpbzVXHC9c3aeq6AiOloRY\\\\\\\/aSl08+vkxgg5mKNWgxQmY1vUI5KqbczFDj\\\\\\\/84Sarcey7YImsP1lnWL+hhi6+DFwQB2WrDrkPcsL25o9HL28fAEWps\\\\\\\/wKggjUDTvTYCLW7jIodhRSH8f+v3Dou23biWzenceL9G8jVhvZRqf3x+I9ihCfEcB1NH1T91UsMDsIY7HWbCt9s5jGnEjyd0RG1l6sXb8knEbNxUS32FlNN+mpemY0mChA9obecOIcAsncr8Ld6jJEvl9u4lokNkdzyO5HqCqIlNXcKq5jLkIrMKaw59x5NY58KSrLqA\\\\\\\/xoQcm1QNgymzKAvPAmMhI\\\\\\\/zeELAT6gwpIZgSSMrjh06dSE91XFZ0tWdIG19WKMl8ovaNpJBlEt+lJ+IoleyXVkQl77WH72UNj4xX8t0ba7lm6uiMdWurE12mrJ+0gCticzT+lgKybQWdypRDDx9qeRSvUBp6EOYxt5gjbn+QE9+Yxi9xfb8agae5q+4rGb3gCxeyKDaiGcOyW5\\\\\\\/Cdob4CefSVHJp3Ug3+vALbvJWiLEuADjH6rwaG4r70JoEnZuXNx6Hcr0ETgcXsUGrGBw3tgmuxLa9lUec3\\\\\\\/8aE6Rczr4fLYiKFMs2dpzPOf3FhgF\\\\\\\/83r7U2ysFityamZPjyfZKk6XE9jdanPiJA6yyoZl1TPRslTCQZ4WtskKbsJATRta2b\\\\\\\/z2PfXM3oiqIXi2xlBdbwOvPJudcyKkJnjQqIjSrGYJiDe\\\\\\\/L4RApDFbUVZyBYq4ztIQw1bkd804iEXuLJ6nz6A5GYvjBPt+nO9wOY5IAMYdmSBmxkjUv1hoYWUSyEkqnwGnz6c5l0Xe99UdU9uf3iKRK9gkUW\\\\\\\/+b0FNeA11V1yMySpe0FWzan0KAVzB5WloAHpaDLzpShe8o\\\\\\\/BQYYQyCEZ\\\\\\\/R03VGY\\\\\\\/ybV1Fx2+Oj1PVJs3d4xknfDktxNNCEtm8ufNjZLvDMOofYwC0rhCwxS5r2sHqkPBNzbhmtiFToELoJS3TFztOsuH74g+jbsaiibILWvUNnIcSvYp4bYJAGX+SFOV4dd49LbB\\\\\\\/HpHUZYZcde8KcqHL8vsOGRAPOF+7t4G5nRTymbTYbPgRwAHwhHo7+7piBZ2SMPIALniZTrpf4nG5UtJd9RBkbi2dkBDipE46zJMILCdq94a0tXEcyXB\\\\\\\/HPbuFjkGPc3NLW1aEyRU\\\\\\\/Bn6iMctLKTXqOR0t6ZO1LJ3NeYmwF9ViW1bYxAUPWjF7lL9xp\\\\\\\/cpfawBHod8DQhp8clWMU6vkHcvoZBc\\\\\\\/WY9McsYlLMLNk19DEcY7DTXaHFv5UYOhBgm4MuI7gqeC4eqMXrBgD9wiszFGENBNhFxiIfcZ72ZoO1qiY276y5aPVqof0G8sxwGU3AOnK0S8JG9DaCQ9cIAhZfgxk5y4cbknQXFHX8rOumdMI56LSfdAbfN6X2DxnYg2yzrqx4vwgbZIZQo6y7kQHOr94ShTXIZTsr8EecZFXckNxB5uSQ+c4hZeXQHKPJX8SzsWifGvhhk7xEPtAWNA3vPva4W+0Ru8KJoYhxqn4TDFjZFDH2B8SdDN9L2sZONBSRMAJPmi7Dm2lPgj+vf\\\\\\\/ngciYajICs5aGkLa42BAYv9v14U2k\\\\\\\/V5u4jr6vixRLEnuX3Nm0Axq\\\\\\\/QabBiMspfsoXF3dWesylJu9dPRr+QzRkb2Jai0Mniw1ydUPUPC\\\\\\\/6wm7gpWaM6L1y\\\\\\\/fpkhFwmAjtl4wxa70AfHnnazPa5syiQwBX8V2C7tUFhGKqxdA19+lQuf4gv+3mpTBAsCKmadmpSTXSqQCbKu6TqTODytE8T0DlV1qHlJ4h9ktk3L+Md\\\\\\\/\\\\\\\/bi8M+E7Us2wSxhj2dCK9s\\\\\\\/gAEE0E\\\\\\\/Z+rKfv83GC7rIjoH5RxpSRD0MEhaSrTWII6lMffSah0g3VBtKxrj5ohGi59tnwXZ\\\\\\\/M5i1N900idHIXMptHN5eVJMVVZV9E3FzXbkMvqZH2B5vpm+ui6+wF0D0GBJZ23yD9pPff3SWlnJir8qKZjGJLOgdpG28v7MdWWakhiys4FPfCGv39OQUFGt8WDv2yo8j3Vr6n6IflIkP3JUkspEVyYEt32HptzsjJDUFqI\\\\\\\/95ek822W8uH4toh0p33c5nrahvaC9dbjmmNNfaCXX5QwdHlWuVwkWioZW9ghieyv5mIe9J1saQiDbbVx93l6B1PwrkWMqNxfrcGOLObwqEg3Gk0Fcu9Gm5QuvTZk0GLLLjd0\\\\\\\/oN2igAt+354T6t+yyZBN3UtqIBRSnthxXik8vAk9Kh1bZTnOY+pjQ516ZP3BRU69rU6HUKR+TDEVp4hKzz\\\\\\\/4CbK+1FZ0CQHf8k1qP+Jg2omePf7fFTAsBvizUoiv1DNXPiefGmq8vxiUATGMBz\\\\\\\/KimIgt3W9++IFGAUm3iHZ9jVUaLWJ1ESksH6x8jr3gL+i\\\\\\\/vTmzM73yc+9r1MuNsuI9fZxK6CHv\\\\\\\/sTeJD\\\\\\\/QtJoOtDbsDLFA9MSIbzRpoP2EeHzrHromw8v1JdeAQHYy8y1F11rDtd4irXzOf8WgGXhVmV\\\\\\\/V6BvQc3of6t8JCEEYzQW7rE6Guo2hLlrewDcf7T5d9aRyrVCEWsk5Vx72W+VPfS9CS8cXb\\\\\\\/R1vIxj1gAprKIoC1wqCBxDQhbJi7A77P0RwbluuxpCfPhyIG+dOFcmFNu2irDKf\\\\\\\/t3haoR03iT\\\\\\\/0uXMvEi3rRQQ5Z7uBtS+0YOPtRbrW7Z\\\\\\\/MieUHFCBLlCcNeT89ko8pMvmql6ruWeeBzewsrovB5\\\\\\\/ETh+MvdoXIoH6aDBVE1\\\\\\\/yO4ORX2cSK8ytYblbHJzpjnMBV0BKM4\\\\\\\/dhDXSYoJwQdLgYplVIIh5YwDCezynEvb1OwwA5EZdYbD4aaln725GAeFDa9LJZs5XeQIGzoHTrv45u5fmVRSFRqXlFKDE9h3+QWyfn0MfJAeT601+ZGWmUNiUvbVpVNtOv9lrYYQrypxZ98bLoe\\\\\\\/Tbmh6+zWuXum0CzHlXL0cnQlQCm+lo\\\\\\\/pj8hGzR1zqpyYa6eK\\\\\\\/WXnKan6oemsr+LAcbUZMgPZuzklN3iytozZhrRapFoVBAyGyd3ZsYH5NgdzKAUs+b7azjbGpMdBVJFuVYP6lQ8RfylCNJA2QE4WD\\\\\\\/J5sAuZPoc5LxjUchqOBwAyCjxttEHJsdX84eeAeTzNUXt5Y13mvg0qbFXHnz69DvaJ5uA2ekKeau7cOKtgie4UWgrA6aZSUAImab91wUnmmzXqTfQM26lDuCEhWeKVL\\\\\\\/n5Cdt0rNovYM9wQ5Iq6Fmc8MaT+JiGqIhiqbNDHINgrQ1sva0Gicle4wQKfWSuPPdNB+9SVG7R56GcHlVUaGRJ9W+S9xwJbVBXe2DYxVSWUBvTVQphsx1nUBcjwgE9pbavk849nm5gBDE0hKjzHyz9F3KJejX2ibZ4A6jZpgd9XCGQfuyIWVnu+jmhBv40qaBWR8QUEF19aQyGwU0BNw+HzTIWdMshwE31XtQZChU9wvMX7XqwmW3+wZKy970DVe+5\\\\\\\/ebztznalNSVa0uQrNKhMfFE4sCv8Fd2l\\\\\\\/1qTNsc1TUsOFymTXNFjZIggmZs40iYNmA1mnr2qWCkTelYR\\\\\\\/WDJnNPg53nvBfZtFAc7WFK6PbitpaxDuidqhbZUgzOZE9sfdhzs4WHcmdDkKHsUBHzWQkS+vqp4sz2ZtYGwdIXv3G9I3cuLnNzdXaL3FsKu+xaENUeece90ZA5Xdp7EbKN5CLV1LddYVO7Noe6BegWli9vvm9v9irkopz0N+rBD+HKHxyUIPFVYVj5INTothPjpzn8KR6fTMD4zj7yuWzqK6OtV7HA41j1UxyItgewN90O0pjPV9NSBdYtz3Sj1qUNZlJkggng2II2QrZs5fK\\\\\\\/zdR27XLA9kreF1\\\\\\\/dKsA+qhOqDyyU7maC5AOqoylnXUrDE1ZgivKdisT6qfoFRIo43GI4+VhdntNkYM21XeglJsuk4ll+tSrlPiEmjZJ\\\\\\\/T+3xdJlMGZUuavr7+lhXLWz3g0506nn\\\\\\\/Ug78z7pYqVJapq9UyG00wtb0xGKhb9BESeO7ecLuTb5FzKpG+gRCHU0t+4NhBnRv2dPg8igSWwSIrc+Bn7IAW4zwFB5fquZgK0QsV+U+aNuhSuL50PKT7BCcnQ42cB3fAnOzWh3vXaIlPAtJa6c4eUbpZn8EuZ94vwlgxsnYMow0YSLLV+7yOzyi+muRBE2q90hLgg8JBbrPVKFontFfbmMamF1J92C6ahT99ZFZnqWaaxOUehsFOJbFzQCYjSc1TKeoH9lWVMeIIwWuZJZqCCORqMpdmvLujIo3YTlyE0UvbMG5YeDPnMFO9l2KFotr\\\\\\\/dLuN1oIdCAG9cm4Zd2sQzgqFr51uvIyvoamwh3d0\\\\\\\/iGSthNwkqj\\\\\\\/7hUIn9Vity9eDb\\\\\\\/BkDuzRtxkvu8sGlaGdee3usO8s9QcTGPkOwAPh4zMQW98UXoAjxbZtrZDxCS\\\\\\\/Ty8GRHQ0\\\\\\\/ighe6aGbggNo6W1i8enq1BuT05Bj9dWgqBc\\\\\\\/MSKrRSXxjm7e5UxpveoXMmLanuZ+o7aSUCQLBx7WgfiJRSwHyFHfPgHSfu1WNCOUJGV1C3Aqp0nqbclxUd\\\\\\\/2rnOipNDUqgPbvXn2oB3gsbEN5\\\\\\\/V3ax4OAf\\\\\\\/Int27oO9WXUmFW7EM5IjuYeHlc1UacaGB8w+srMbVzV+Nt9x4MRJSZr2nxWBmI4HFw8Kdw2rKY6Mx3mVomY6pc3FUQhkeZqv6ndF1hlXCbBCqZIXPw54FfvZoWYdYFcUbpZznA0Ruobysx3toGATwHhc5j1EDvjMO6ULEdt2fK51Uh27JkjNWdeH5YdtEJQUBb5JacRFzIsgK+fyzvGsPISIt0alJS++Hs0t8mVXC9P3Q+QSJt14E\\\\\\\/\\\\\\\/Qw+prGlQ2evVNG88Mg7179BTU4WFcHEc4RPEFca0h5SIQE36T0N4rZiWd6NTXTC80FwCX47AmgTt58SYkJtxxFfQNFmVh4cZ6m6wW5YvZv62eV9N+YCNLqxDGAsUebbMmFLeDZKThzas\\\\\\\/kadTMmTAGOyHOXXD0MyrUG0TuJhkb1z+dvBsnW0PJ5iNH6BacIYQXTVnD2ZePEbWbrSF09c\\\\\\\/7XHHScGtoEH+X87GKqkF17dpRQj8sW50LGGIVIkhlh3tjVsypxbFlS2GuHCMrtmkIrKDiZ5s4KdNGaE\\\\\\\/AF832VOWupLLMmnem8TgAuLlpslVaQrH7XWY5Nv4oZYRw62qMSp3GkOhkCt1KhoaL9q\\\\\\\/XMrKNDaX4dSauArqcxA1CXNoR3XawcxMgKckrf3uRnEd28aw1oRbn5pBwuiODgxuv\\\\\\\/5XovfnHLZsvwTRD3h108ixKQ\\\\\\\/3hx5wNM1pq\\\\\\\/y9zlFNo1Jj41qA2rr48sI0iAa1I3Y7c2m8ZwSESnjJhOaxz2wNd6b6aMUuDm2FdIidnDj5WRGVpl7DjkFbcFaRHoSiYhx2uaq5ByG80HyCZ0JEEHZs4SQcBQSC2d9QK3MGjIUQza2bjqtzKNsW1RbmaTTGyIsV+5dHAXQa36+PKWHmhKi29I2ymy41IGBY2pgru1jskrRc2Jud6wvPMLk6ckhDL+xDjFXusXDfYkTqlCFwsvB9WSpE20SK6rYz2WSko8RqZnLOeJG0oMiRzmdrcHPoUptdbl\\\\\\\/x+5XLTthjeiXRSI6hRzQukoJYpmjZ8+TIIHGueOsadSmnM6WkFyBfbb5d8rpZ\\\\\\\/8QHKsjWOsDIXAVKSw9ekCngwCO4SneWF5vnwZkhxfdoF\\\\\\\/btrI+tCZVqdAWV1EwOnnuzvXhDvEfI8ooMNLSYmvk8Jnm4KU39yr3Ci5qnIkeglYhWcRZ10gqiZ3j9\\\\\\\/34HcxI7Wbg6lBSmqebKGkSO+ejrquo0gTYD1hOfgW+FWOwQoxIrzWB4c8VoWpMGiwBzyj8NibqxBI3mfuGjhkMBzcNIXeRMO6b4jveXVSAAAR\\\\\\\/v98Obx9q0VH\\\\\\\/k6udncZOuXgchOfQh1PhCGSloa7Ae3\\\\\\\/0Nge8EpeODUmQ+qUsilIomGcXTceY\\\\\\\/Wan82dEgYC6wHiwxynwlfy9CZ75o3GwMxlCgTzrhTbuoPi58YtpeLMc7u3gIke+tur3ypAX4d1sjy5hhrGK7+KQn97jyIlLWWzf3ZezkCffvfdB6QNh+xGSE\\\\\\\/I8phpFBcJYIUDf1PKsJymZkUvZCLgkX+KEfOQHzaU2DcafKJyvEbs2JoonGmvnwF9jKTlwyu6TLiAxmdK6n3DFzDSoj8uzosun8iTtZ6pUsWI0taKzcnEWUFflg7tCln+gBUknj\\\\\\\/UzpzFymLmXOr+K\\\\\\\/qSLbOUkLEKN8ZX2Q12wmkEle4aeKIxjuR+aGWZAfyUQ63ZEkRXsXgnhyKxycXfXhqEKi1trykRd53E2agS7Vec70RFZlGCJQvfWHPSzTOzDxwEKm3lywsmGmSYCbpnN+zRXU4H7zlHXJL1LUi29KPOekLWCMhw3nBT06geBAefwxAjLxLR9CqMDWj0coqgkg2PzrcwjLLqdwXxMY7OFDTYZgbZuw9DdmLZKVJHWZCxYTi83P4vC2ZFGp7C3CsBeTeHMVmt9fJBot5+iNG4WPEE787pqwHzyppP8XnyBG\\\\\\\/fPFmAKGxUy7nMUzeBM69lvLou7DLQ4k7tmdi9D\\\\\\\/vLU3bVrNyUpEptX1gBmHLShXrKFZGcrwQLY5Nn84c5fPlSRC\\\\\\\/6YtHbRbXCZfELSup23GgAjHNTiJWp5yDzKILAWC6otq\\\\\\\/PG6DzdcCKAou4ZaMU8wNzWJyERxn8JMzwwCWrnUB9LTqq3Erb4Ki7DPStIno05ELeXsiW9G15cfzLfnC+7NZLBb1tFRBkaz5g6WJKqrdW7\\\\\\\/PzkVzYJ0qhHOmdk0wkWvNQssp7tHENFrW8ijOSYR6WOTtt9Xj+IIMV1OucRkLsJARvzf0\\\\\\\/Gh0MKla7BcZJqAFhgx4BpMnAyyaXiNlrKCMLFNbra0dgEnO8CbbqKkek76QLa\\\\\\\/mjnvwHDNQfyk897XgSG6N6LaaUvn8K4fV6l7mjw+PSXyCcvVi+znOUQm1xiPd7TJjgnPvADfLgfJ0gs4Jac2S2ZW5DIKrTtK7jVBkYADw0C36k7IpxGfum+WvqWSZ9gpmboD9Y\\\\\\\/M4NCRVd+XXpurgpZzY2ACENFSgaCFBennoxGQFwGgV9QB\\\\\\\/EUHdUrgc1R\\\\\\\/GsapmvuKR7Klu2OKt6HtUNXq+R+Ag+tgSXdSdui4vHVS\\\\\\\/CgDTnDtS3KgY2lxjFo8H9kc2Jwc+7zfZiEVc5EFCJ7DtvyC4SSMQ0Qb7c3CS8hfgjjHTp98qGRbbfyyac9OpCri50+fs77QZiWNTWrdKr89\\\\\\\/NfxU5iOe2c0yusrZZbvzCIyKBVHNqncMaYdIoFG6WEaAbVFzlXWk7hDh6qSWbqVnoUffe8K152euIQYcHL6AtMLFIxAZDs3zWeWBrHUZlMt57vU6AivACSlMMlbbHGazzxaBq\\\\\\\/jBANXVgt066lSVm9HCwDn2LcXRtUqMXn\\\\\\\/QQIPvCoE5teTGRLWAHQ3aJQ8sEeqcs9TdsSPinRbD+jM7IFQoXV0PIkQZW2I5zIQkKomNLu8DLXPcoqeATQ\\\\\\\/Dl4772p\\\\\\\/ioVOBhoFWT26lflbR7osOFpdRYNBdyVa0m2AIIGTSH8udUYLTiQcjtwot+ew51eh0rTlH49MVK1DW6N5H6mhtsVbovmCxPmUTKGLEXei3Gsa\\\\\\\/zfnnlBvoaNCMWCjmZ4SwNOtTBRxIYppysA9ZZ4ksVCtCZvSvecz8S4Byo2UuJpWNPt+cXq17qza2glceJ1V8YC\\\\\\\/Fo1qPWDev\\\\\\\/\\\\\\\/X3DuhnXowFpQe2tuiobhcz4tstW3BXnwtTv+vRZrDauZXkV+fijQN+n56tIz6CtMKLncWWG\\\\\\\/TJmXHoyyBdArBTj+zGFjE1Cj0\\\\\\\/ECL6EYw1uuOS\\\\\\\/\\\\\\\/WDT3wRlQIhlibdi0jf6CoaPM3VwBSUtOWAe4m3xldSQ7Rw8FqJwYFb6+2WNHdHDQ\\\\\\\/VA9YDA+5Jj7hGfi0gRQqhVhC1BSNot3cDMx0Pf38s6RBXVl8jj9tUeUgWniRA1c1qzAKgJyXcw93HhlN8YvcRBqzSAgt\\\\\\\/3TjVmvsLKIYTC3pk2+K\\\\\\\/aJSzr1z4lQHT1LxElOkolPiIYq60tHcIaLNYxTSdWhNG\\\\\\\/avbClgkbgxtEusCyXl65JtdUocXVFePQftrhrBlCK6YXqitsCKhQWLLO4DTSJCSD6iOc0r8kRSD9wF1KTxROkd6opeSjTWivWt\\\\\\\/oIj7yJXCiLTj4GC3DHpY8rpjcPqzE\\\\\\\/xjnPu9cB5ZIOmunjqag5L2EvmBfYvMXbh07pHsipbTVdx2pj5ShPCjXMxp3qRXU0OcnesLEcis8OndawMVTerdrDjWxZix+bMfbtjPXIlSzQSbSqyxaolXmiCNx5xVPje8zxK72avRTnSazhqlECKrUEYPozQh30Gy862W9XTm6YYxaX1K74+50Fz\\\\\\\/ZffwbEoZ9EEEicdJWcw5bR3H1CPl+Uj36caj5GZBCmIzg8Wamf3Iko7nZldg7Q5Zjz6eowWqxTTgm\\\\\\\/FFp1Bmg5ImcLUtqZ41Hrob+qKyalbRbv4u4CQkoBTNcEc4L0vvTIhRmHNWPPjo2l2RKJ+wuNFWstA2WK2IaB9GvXRnnDbbthJE1vB46y5otnvgEA49kiLy9wso6BMoy6wWi6G7oW179H3GKKxgQZ07V4pBlIUYK5hKQROBcnP1ySfKJ5\\\\\\\/mdNjkjJWHDIzABXpD3oFNaSMt1OuPdKYXoC2DjJ6ikAmR8XInPZvLMNIyzWiwc+n13CypxKwTF3SGukx594ZoxF1VD9W49OeqdDFNGTMRxxUa1Ws9cOy0HzAoBG6qzYp9BpmYaI2GDkegNeKJzWdvuN\\\\\\\/AbBj0flDr2IbF2Yxur6W8lyV9HWxQPjvsjSfH5+RajHD0tCMOxwiXFM6SQHgVXzy1nHyQ72hZdS5c96LPqkPz6gjkt3E960S\\\\\\\/Jzvo7ULrKmDdJG9zFBvj\\\\\\\/AsyO5IlHFoNFBRLa3FrLdFihKB5tC1WeiIfvvj0xC2PUQELqePl\\\\\\\/Qm4VnnTnhf2ZeIARG2j7NYI\\\\\\\/wZChZcyFTMPx6zz8mKpSDnXgsdOu7Wlai+8SSCOAK2dwhT9Vgz8JErSzHTMXDXCNmT\\\\\\\/Kz45A80tzz6r3pCNkUYaPqC5GMGf9Cbe8xVc1JgVbgSrvKarI1hKW\\\\\\\/bIqbtDVsLRda3QDazDMu5XMuzbQbFYUecpscD2fDrjVDipuIVf8T7XUv9FNfvYK0Q5ibV6LdDTQfU+cw5jOp\\\\\\\/YqQX\\\\\\\/iqrWKIujs+FZUcZr26OXn\\\\\\\/cnWc7JUclxR529Aero1N5ne4wOHaSEF2DjPg4UoyaQZNLSFdVr5clLqqEf9WeKMjdJCzyfcca7k+ktEhEXaI\\\\\\\/mfvD9zckgTIIc2R6AXVfMcwpLC2BQXXUSjzRGP9Ntn5ymJjEqOBfL6k21\\\\\\\/WKUOaDlp+YFPgjf5upvlQhIB5UFJxeeIe+IuQolzdnOMGTcGyTc+KARXR6syxcTuwHbae8P4pUIMg2ByKt8ZSVStLq9qUv5hJ7e22+erTdN\\\\\\\/+lXn8ysv1cjneB4rmeeZkapUO6DZTyc60bsIAVbbT8mQvSZm+VGq5BRQhSb3nC1LzI2k+AfkoA9QQ0C+WM1jNotOSM1sFGQGKFkLBEhpftKVW0mKZFuQasN3SAnRrnIB3yIYdzbi1SUbxu8jCR4FQAQH7yONp58pS+VQMc8tja37qLvbZGTaMxi99Iv9QMpf8bI4dHTXl7UINQttcCLJOViSeu0Px4zOjaxINyFFNiLtagrhLYMvFDtUjp3ObI8mZ5uD90DZR2NTn8E49SAxIb0vY23r3O0+C7e\\\\\\\/AbBHlkqY2rKP08Id2poUd8BJFSvXU8NGvbcDjbwjJniTo41qQXYB0omJI\\\\\\\/LWUzGwjN9l5CG1NzP6oApV56uDLSTv2j91+kj8rBVwFZSE8YBgT7lT0+SSgmZfZ0SAzRVn8aqlZu2n9DSv+i7rCDpFgx0DqX4nKdVenTTsoEx8ROdiGTlhDVF\\\\\\\/j51lHfNj27FWQFoo9a65GXm6mh7t9r8dQz9JFwcICGob6PUS3Wi3j08JrV2K9\\\\\\\/hhoaDYAG69+GkunnweHPv6roHKo4WNEJWjLctqLSoaehP3UPllRGwbdGiwmPQSrHjd9t0lw3cOOj2VRXqBwEhak0RXJBdMJ9AclyCQHRxgtYu\\\\\\\/CLwNicAeKwdeoyZaZu1NmIftbOEFeLe2M+AI46opWxygsqtCeA7sd4naUrMwY0aoq5Gsw7XfCW890AGG8KtdsJroKbNBDh4I5H4wcqftKAu14vdhtDigORXGpd0Aa5FMhJvssAmS+9m+Z11iWLYowxsMutjt8ZtrkOmEV9W8qYWKDCykHkRb\\\\\\\/16AC+3DcQeX5A\\\\\\\/qzFKWgfXHfItWd1jy\\\\\\\/fhcsntwUmOMv\\\\\\\/Rsetvmv8XVc4TpgB5zjWyRN9JK8NAbGRvg62dfRi2KXa355IpBsdhJiAXlCWfjrDk\\\\\\\/YiZLOGIuiyJBI3ZoixTg0TmPVPgg8Sh4YXII8D4OoDGEXWrZKKPd6EYn8pgFtJoCcQQTbvAaHj7RVqWnYMdL6bUI3o9yhZ8UaAiC+iExYN3KRPfy9qKZOlqQF9UAI+ivERfawbpnjK3pvC3iqKDzTtIpiaxAlf4JKskEIs9mKaoI4H4wQFbWZleAlxvDkLSFo+VotmQL+nYnlm\\\\\\\/zq3eyOSwEIS7u5fQMrpJQZ1Ewmmw09gybTv7SFL0iMyh7LY2jTS3cg1Yxim0Qq5gLWtKbAZpqROxiJLcx28MCX705eg9sAJqUYgjCYtqoDhhlZfyCw09lEoRTN+mH8qOtH13DkNs6+HUUW2tQYL7qcJxqL8cHx827cnzTMP7pnul0L\\\\\\\/WJVIqz4GjbPGVUkEFwK9xMed0nBacdO\\\\\\\/rlASU3CyK4okeyxS3qOSoD24eFh1GzbeGPP0vhhmCfqa6PIDDz6RtGrYjIRNNy+BKy8RDVteGm0iPmj58NrVaHY3PfFSsWsTYOj8iQZgWPSpCJMkMHJ9HVFRt7ezUks\\\\\\\/+kEis523KfLIcbPeR+KkBw\\\\\\\/O\\\\\\\/g89mchJyFXr1N5yR+GujM3Pc3xkXIKWXQEbfQMwkhLzbjHL8xoG6y9AiCKss6ZUTxu2rm6WSn1k56HzhGnWoibCW3Rhc6ZTJJ85tddsNIHjQTkQyC0JSOFjH5uRlOXwbjHWudcmIdBajUhwtMCPLOsXH3ivK1ifyuSQz+JW5xF6DyS0ttdbhyPj+geh7Q4Joaio6w\\\\\\\/f0Py7Km3s9mKi5uG5xTKicU1Dx2Q5PRXpULVIOtg\\\\\\\/7EmmNC6pPANRi+Zno3nZaAe9snKTJq1wMz+F8CfCJZ7KApvDGc4sBsAC7gvTv1yThzZVspwyD6yOhtkjiaxhHFRTxHihxWr+sq+7VgRoZuJTULE8oAT6zvpIYlrV3sGsGq67CiI7\\\\\\\/eSD+C1shwtWEsdsyd4XxoSrZOfN+Ov4kDxq9Njx4Zbg2\\\\\\\/QF6McwuuOYkK\\\\\\\/+KSRq0aRfii4FYOeS\\\\\\\/jSKRGliosOwADMJUua1BdStbak0fsrjwr2\\\\\\\/O2jEwBkX0skWZw2iUnglAWcL85ZxG1RiiOLp+++t+QDcgENk6ALlN4sMlUkhKv02Nh3senb8Dyg56KSrKocZxrFhMdUeeFUg3heM8TBf3y7WqEoP6J+yqvXv3aKs88cJ5f1TKdM3F6OxJEQMg9nXNNihWZ5CGBkrRU+8d8RZDlQLwywLJrZzps+49+5sd2XmTGRsPCGfpeBdXG3L6xl\\\\\\\/Nhz\\\\\\\/9eLIklSIZIdhRTG1f4XTxmluRbNBMh8P25M5oHI4Fz1TEcy45GxaeXujOGi3mO4UY7NLsBPq+FCfbMxAq4TGht3cA4X82nUBzuPRPg95aU+tx9EfC6qU9mHG0vSxr+ULBjeu2pcL\\\\\\\/xpMaGpfeWHRhzKsQkvtUvVUNZ8Y3U2VpVEJHlWFQ5IIG1SVxpRI1RlMcWEI65COgqXKi5skFmsXWsrLAtlYKaRw0+y1RIfB6e5bJDI\\\\\\\/uaiC5dCGkgIeUV6tvBOfhJ1uZeBdOFuzMVw9Kn8BtCPzMdI3nkHfb0TDB\\\\\\\/vLuwzKkdA0v4I6d1Yna7Tz4Ba58Vtm+fsjqe\\\\\\\/KD+ul0Kr1F2QHTc+Zd6UB7jQD0YqUSiDm73q29gvlIPDIoNiLNSp3+jRNEsvbRzDtQtC82ZJ+vLCpP+YooUe+yvb\\\\\\\/SIUdNVklObpdXgNafShH1RZRu9rjrFO8OSujzkNVQO4SSIxTzDufmOPjXCn3wE6beCTdCT5PwA91SaC8D8ew1oEu1k3UaecpyUi3Ti4ps+gswN+1wwkf7T1Dh5mazrGfkTx0gLMhZml24C4cc\\\\\\\/KD3DkqvUr9+wNGym9Ri9bSM390mKrx4OOpLUjK+eWGgcz19ah0z5wXQhL2rtJjCo12pA\\\\\\\/LoXLhBLo3Beb4MefOSAvI3cLAm8eGFJikmqd7BXoePV9YaO9ENCcWSzsG1IsCj4prLDFPPUIZ\\\\\\\/qXpoeCwGXUFLXTngWTBMS6wWqEeczdaFv24+oPu4KCSsk84x2h42KDkq9OzurcIjFQkDZSBGFu89ezhxiOCFDCh1v4d4j1W0p9mxXx7LCECtsw1rl0CbwWGCGkmhYb8bgVbUV8k\\\\\\\/A+RB3PDCS6czoMCFqh1uMrvqaaqjfxsMrQHkf6Hsb4E\\\\\\\/mVS3JFZsNRgHlGVIPflNbLY9tUpoU9kAhuvVpbqiiOmORdc\\\\\\\/wKzU9b7g+JJmVOiwgJ5C5HeYLKtgKdlCMeA8x+wkth62QicSdd0igrP3Yj2vuxHseo7lRwruOcVKidj14DUFSU3kwSGh2SajVsyljM\\\\\\\/mUGbdFgUlIgnoEoDfl8\\\\\\\/KJ5wjHJlh2DFx5Y7+zPT\\\\\\\/m9Blyz8f984oYB3XHzjoJMLCUx0YgFuBDsJMbProAoJG2cgNgwE\\\\\\\/\\\\\\\/V\\\\\\\/QY5FfF3cYBiMDuj7rGiOJXQWgNuGKiExZF11+7L9dON+r84EQI7mzLUyCZ1EwTWGuRsIs+m5K4SdGq3nJzUVX0dua\\\\\\\/S18NGQ5jZdAclrgP3cqvz1OecNZYKcly6ohl8RRgZamepiaDqUhuRG5tugLKNkdOx8uYdIDFEeUQreICHLEXDg75UqipgkG6vRaxn\\\\\\\/U3jGbSAQOL7enlq0AiS6YyoT7ZQM8BEiV8tMHJoGwfW5R7p+97mqHFIz\\\\\\\/b5DGFdj2KnuratOIKZNU2sAffNohhR8uq9yjqQvV0izZ68IwzXdlo9UEOYVk0hHpcYr2l\\\\\\\/MZLcbwrrnJ2a5CEsd9Y0Jyllxm6ZN4rJASDVkMYkGNeWVI7xw67fRfMUl7DjQeSjTooIjdY8sg1IfzQY\\\\\\\/ZfmdLsVEqEKqrShAoDrCe3UdI7HBJoYNFttzb2uBL4obrlKCoDNdJhMgeUq8mHEm52h2WI+6VWgE43hyUl6u5dPnDpx712fwu3pP1xN11cpKxpwRxEH0HQmV\\\\\\\/+LZlhDs6TOc08F1T+QPpVrHvhOGMyw5UU\\\\\\\/PojsmPeIwq4VVA9OLsrCoskDHO1+77czq4PnKnvrTBKWwxHXMo27cxAOIreQL32dWjkAoPBGFIfN+iEikncEWLyrNKyimjvkLPz1gu026fTyVa+6wLUAXhMpTWxoVGJAvdEthDNBals8nf\\\\\\\/lIxaAzBaQDyw7cQBv8wA\\\\\\\/0u1EVebmtbcoBTTVVExLcVE3GoH6sKnb6N13lnetgEGF5Zf6bxTaV8xLxLPbYnC9HPxBHV7W1V\\\\\\\/VvdmhrOKcK\\\\\\\/5C0swI3sRVwYp4vkCAWeBII3ZWRn04sWoOna3xgMRIeQyajHTP3SW0dGU1zZOHQaaJCbAtDcyYXz4mI2lcoNA9GRfhz4JaUxIwvVXsU2Qc+0ADUvIbxNczP8BlhJGHSOrEvJdHz9kEaR\\\\\\\/BZCzuMHzPl4Uetjd\\\\\\\/dDozRHdEALPAknmX6aNXtxzlc8SOso2UJPnLMctefnlFtP8dTL3pYGQtzLApJFlryX6UjVv3\\\\\\\/rlAFNLE735uLJfynRN7Y0KAWaervciwisPKenIGqNZcrUHTiFDMbdq1roPTDdnsUxRDEakza8QkMRrG1xWIvkxAGt\\\\\\\/ek0KBbpxCZTDSDRHhZeBck5bBofRsNPkoSmN0sGZcfRGulgP3loSpWmFkroSKDXrjONtRQpeuWD4BE+B6RvpBQFTXzrUAAfnGmLpsmP4xeeNb8iS21QwwpAF156fyoq\\\\\\\/7u8b8peTF5VvQVWcnQEq08P1qVvmEUeOBNEUlf8delSA2SiYc9Y0WdTHn+\\\\\\\/itJv5SuhySfX8Rou4vpcHxbj0zpojLbrP6Afae5GkuNvoaJFASN+PpbNgdUu42oi9hma35rnoRUUVO1rq5xo9zAnoyUeqUxru93\\\\\\\/A2jTzV32Qx+qaakYVTusSLLfKLK+DQuU\\\\\\\/Fd4ZOkePwndIrGJjBdO9bVPsgk2mKrKwX3ER8YCnksmu5+IWOtG+ik3NrUkjhDYhZcvr7fVBFdkq0eB9vlfcx7JShC\\\\\\\/1uCUaCRAs+AN5Nun7OZEzII2C6tDwR01aGNx7kQN9Stjr7FE8gqRzghVm\\\\\\\/PxP5jguQEVWcVJ7jsXSEsneLy35JRtcHCoH2iN6jJ7YO7dSmpjykTTlIXpTB1gHm+yi2tXfYVv06PGFl5uRk48qD+F2ErAkz8isQFVf153N9GTMoGv6gA1Eqn9F1688vP6rM\\\\\\\/bka+ycoNxnUBPTFTNnDpwVRDxQ9NqiG+OsI7Uf4HZyjTq+HZv5IIfmKA6+5In4\\\\\\\/\\\\\\\/+2wbHnzjh3mTx98sY+NdVEm3I4DnOM1r7DSQAyl8GlPG7THMSwBQsKCoSuWahIERx9bPNMW0wqcvXGfvJZ08wa3GKf4nohpjf+gRnNxdjbyehUFwFKPYKVMpGs7FVn19qXqSorVPE\\\\\\\/1lNoy031HWhx+j+XkW2OioMQB27WbFR+nPg8kmQZRxwl0M2yVEu1nwPHfbJEbxuGCAOMtOqLTOHQfE\\\\\\\/88yhhu0lc\\\\\\\/vI\\\\\\\/TI3zOqxD9rija057Ok9toeebU80+h9U5Wr0SKCMIyKz1iwnuWKGF36NaMrU3U9H5pmYQ+wqMsak\\\\\\\/lk7FiRIaV\\\\\\\/hfK4+0yZQ0i5s8WKr7LJfEuoRjZuwdky3T959cGAUCoO0pKtEBZ8txHmGJp1GR0jb96w3SH0zUjntlAPhjULUSqHZ8G+Eoesrt9lxYzp29rmMVqU\\\\\\\/oOf0YOQ5XG2tmgG9Ie6U9MruiFPXYQdNQ5cdv8bJVaF8eNPR8FOmiraVNg\\\\\\\/Nm1HsLgjpk+vWZlkoUZFoNdRLYnzTfutAybDN+7QU3pUFm7Z5ZJuyIA2wesOjrPTICItHtvgI371UFPU6CCYSQ5aeSLQZsb+jDPCWGOZF+EaXOEbhuq0c0tG0IEz8QIvA\\\\\\\/y3zf82N3u97e4b6C1E8j3lttDl72jKBnRha1omXMbebSip4zKVY5LOw03ZGI\\\\\\\/nu0cPDdxlex\\\\\\\/ga\\\\\\\/6qkiXbLj2Ct1Z7nZG1P\\\\\\\/HHSJNqjq+upwKLbeNSCP+ucgdZSMgi5Hnr5x1oFz6kUOGU0Itm72X41MVZvgI43Tveb4A\\\\\\\/k+IhIZdjh6Uo0MNyUjNj2t08cmdscQrERXim\\\\\\\/lm6DseqB2E9SoyS9Cn5jytZmi5f6wMe7d\\\\\\\/L6vFaL0gwvxD9O6SgVW8Yz\\\\\\\/7VtfUy+AlzR6xqUbPvdm+T5cP560o69xMfo8rl4wG+5xYVOFDg9N2SwE2PHAsYCytQftyu3qkzyXAwU8uZYo\\\\\\\/Euzx+\\\\\\\/2S1QB7Rpvz9PKYfZZ6\\\\\\\/0Ec4rXGL1rpXTLiiQiwOLYi9UlIH3cLcMKUSMwzRNvFHXLWFOCSq5V8w4UjB1d309ZSK2DZvo9nY+AXsWb+fCE4lRuupl6mN9vlkbEXEf9peg5PsVUnH3r8Lfh2qmWSmb1wZkDhUsnL7FQwFQvewYzKftA48F\\\\\\\/xVllu+61r6wN9F4zwk66cX9u5+V4SoeTHst71apMswJmR5n\\\\\\\/coFcpPAFT0K48GkV8m+OdM31hAFbeI840yMdUspo3g+2+EJ7r69uqz99ocHZ6SoNQkWzhBNU544P8Hq1NAHr\\\\\\\/YyURJL0vFCoeBHLBr2MqqFjRJt9J50to+aef39a57N1L776LT4Th+3\\\\\\\/RRmzuW5qc8NmOGSDBZh719wrBTd3Xkvh5RfWhnklrBsqHQhEcRzTQzUWCRJC+bQ8ZsmMKFvFju3MIUqKcsf41rXiQIO26CY0cZWbNjxlhHHv2Vqi\\\\\\\/oVWcqi6Fht9EHoWbgu4oclnhUpHH+xw3y2bYz1utTJfSwqOcRghTDgBoIbuh3rXqwgJ3pvx1W+btL5uu7tqImcJnzBs2C9fVS2XrIEFvRGJ8xmpt3ha4e0Dkel088Wgn3WsPe3OG+CTkssRqLG6heX3u6OozoXxqkUvzH50A9\\\\\\\/aiosCJVxKxYyUj8CrenSFZcXLrEU5i0qkxaCJxC1A5TR0KvbZB18TA4ZLfa\\\\\\\/aoXkorklt\\\\\\\/v4neA+Yvn3uF5\\\\\\\/1WUPdK30QLxzwVCnLnbrOd6H\\\\\\\/5MqLHT76exUaXC3\\\\\\\/H2TBmbXIVudlp7sLmKKI7VCpXZAiWOkD6iYLzymGG0IeDFTbmH6hfmzym1tl9OW08nrjMsaaWz0hqpSEWlKTR7q+SyKS1+5V28QZ\\\\\\\/I\\\\\\\/AUQ698R4cDIwZWMBpGmf8wjKdj6QM\\\\\\\/rViSVi55Oh1EFbcC75JLmt3Aat7q8bZQTQtkzQyUq7RYgMD1M2sWlY5XGjfZ1IRpzCWxpgKO2zy6wJbyQDJ++bydddsi3cUgbILOUSro0VLIZtbvVoW7\\\\\\\/CAdf2CdWjh9QnuYyuB0aorQNMaCWXGsxEr+MBLY1JjAgFri1U5Jd9iDw2Rfm5kui+sP+KBSZOjfv4wCEoMEwUUebp\\\\\\\/\\\\\\\/NW1P\\\\\\\/7QK266As26fl8fhvxo117mMktMfibWbfvnmP4BNZRfiYLUlGlx1MkD7BZsEs\\\\\\\/3B9gJfoSRkLOaNir4wq4Sd7ktRuteDle5Xm\\\\\\\/dXlbtNlU09BDdY5AI99A5Y0kDZpUJYwsijQrKsv9TpbHR8lwp4\\\\\\\/cyavh8LbMRr\\\\\\\/pa8dnkFt1sZWL2+2JjthL3wxq+XC\\\\\\\/Sq\\\\\\\/Y4WieerKO+TIyFXHkc0Fg1YH3D4+GPF4iml2vjLGf11eKjTpvi67T78mElk\\\\\\\/C+o2v\\\\\\\/3ALq+Y3YFvMiV7HQ+qSAn3jivmfUdF0LvUIVh1YZOH5ZO3fEzxZgtYDooNFPmkgFmhJJaovCGcKsI7jPKTJLILltZo7xpG6QF7GSZSa2DhVZLHLNQceAHtf3AXiRqFdlAQwMUtAc+WD9uWP3qlmW\\\\\\\/Qh\\\\\\\/xhKUiUcE6VUXJqFFqvSF8IqzjhRvLNR2JgJZayl+OKFkKSfdKEQzzomItzEurtLMgYE9yQE9jSnWvuEVPs7iQsPs5d3xURtj8Wb2roLVIgcNyaaITIaAFakT3iZiWNs1E99NTAiexGK8jlZbJi0cjNJ89wclyIi1JXixBXp2pi35voAVlSw9+2uZ+JB0GBCmTONMH5UiHpgTU5R+zpgLJ8z30VVGjhm+By8JTkyug8iyPU7El6RyB\\\\\\\/ZNI1MBOeY2tX\\\\\\\/BfrwsAlYHelrQ2wvn2qEg\\\\\\\/73mR+t+pPS3IgPWu8u7P3mpVgI7m9lRJv4RPRxyCj37xIkNMbKJD\\\\\\\/6gmI9F8AeeeRh2m2yrla0JPcjWY+61d+g6m4psybomhLqUeB94j4WuYoeCbxPcHt591WNGjnCMBTWkj+n8IrptgccNMPqubwzF8xXo8vAlBO5Y3Mx5aomtnKu7X9E\\\\\\\/tHkt3QXcSYUCOrcM\\\\\\\/7XHRJFi\\\\\\\/jKK9kcOj9QlccQaT1QnZeTT1CuzjcYz5oV\\\\\\\/cfeH4oNBxoyQjhrb5rUEutoAvM9czWrcIFuh7uFSn7xZrnadveQFJKowYP3PbSqZfCYR8iApgNY\\\\\\\/L\\\\\\\/GhixBsF0ZmCcTEdvoktTBA+z5YbGQO7VDr2m8ZuZghklFTcam3qauMWVHTtXIUq1iF2UQWCzMq4YGkbqnj+N1eWyjlmRnoLTKB5Enil32kZpQum0uLmfyk43gJAvjGXsrQztRkkl+Wnpiyh6ShBTOvWANUTqfx5cOXikhgG12lm1YXN1yM9CNvCrizXCa54DtVcAPJ2Xn0HXZS\\\\\\\/btZ\\\\\\\/mvTVVddDUa1o+ma0JfQZQn26AX3VEBghRhiE9oEOJ6p3bj8RhoJ0jqIJzWUZiHDTniOQD8TLut\\\\\\\/amkgev593jECCt95fuaa\\\\\\\/ScZlq0m\\\\\\\/wppGkscdkBkBK5DmYvSRMQM6pob+f8KwK1iXeU7e+pFreRWP0pXYTCwE6PEds0EyY0OrJYGg\\\\\\\/ZMwVdm7elFDVWp5g2FZnk6+J9du\\\\\\\/\\\\\\\/FQxRL8AU5VN\\\\\\\/KtW1e5x23pEhbj0JVCrAbGF\\\\\\\/UxRhWh2XaL+i+e2NMKkniZ96lVBcyQdYL8GylYIiW1v4Wvwu101s01aBkjyXglAS0XYr7nxpqX5tF4lUmUoDjeBl8EjRp9AAGpZcvD1Qj+IYZuxhualYFNaXh4jiZ++T\\\\\\\/rrCrguA6KZUEICZ3+ar7RTSke4L6KvQOnjZ48D40Uz49Iy5C8H6zOQo67+7D9XEAhE9aDmEVlG5DkydFvDxgtnnv4QVAewyzWsyBiV9E2RFgnO\\\\\\\/HrxX+83XAtM9cICbVNeCNKYJdkVxnrvg\\\\\\\/IgWj30NuUzKaOZHr3+AZQEFbDVSQfLGd143Dr9kBa\\\\\\\/Sd9xGYvCjj4wO9TzdHGxwci66FUCRkCxENk3pNvRQZEKGWDE4XikxqbnQqLzGJl0aMoY+KUBi8iLhiazKt2HvMAwuTmrA0XyakR\\\\\\\/yWD0LW3MU3+ntzxukvrCPYgli\\\\\\\/pvAfEzIkJfrJOBtDRigG+RZZMJARZVt+HzkAs\\\\\\\/Z03vJAegdgVOURY\\\\\\\/hfROFuRhbLLaqQYbq\\\\\\\/8pxjAP\\\\\\\/5LO712WKyWRf91gFouasK\\\\\\\/BYjMB5BSyH0rc1cR8oyL3C3+c0X3iRL0Mxq6faQGqjiAJ\\\\\\\/bm6rDqAZraSVVTM7GKxl9du3lsMjnRtZ77LApk2F9y9kyLC6X0MTyOZ2SQuONInHs6LtV0HqgjFWZah4588mSZRwCsWk\\\\\\\/zBNU6te+oSXppaTVVqJfo3aZ8VWbluHs9mB3X7tW5SvaRmStFXSvkv395H4IQqGUFXcOPlYU1HYm7KRWViAZOo+QG3x+Ui+Tb7gCufsGngKlvmKNoF5LzcuAt47Ypkln3QrO1eHBTE9iD465lbrNtu1hZyf0faBCNwkWAN1XkjtxEEyjB50bwobrmLDbWtXhDN\\\\\\\/WdL2jEcwLWx0QiJhnzWPtAXyUvqSyJeb0nZq+AQoBRuOtSEgidA2otpsbYIqLeruXYusgqZiAFhzrc+Rj5Wh\\\\\\\/yfOSVkICszVLo\\\\\\\/9MeN\\\\\\\/dz+KM9N7mdII4w9QH4DPSsVuHMO\\\\\\\/RjG5wdr3DOG9HNUbpvmw32mlnUyGSTH1f5chXyAXc0YVBlcXbz5C\\\\\\\/4+vSgLq8ppr4qji3YXNO3Txg67Ij7txjSnAOXBJBbhWvbFWMM3RRDWGUgmzKh4EPE84ZRyhnjiGtUyQk7Z8HSg9acwc4a+ti0EHPeQGJ2xBI1w2NXVlRIeWgDEB0kweYLJQTqiXMm+MeDdyRT4CoBnktrH3uS5aD2vkVEXZppgu5UJh5LKGYefyagx4bJx9JTRhZoP35XVDS\\\\\\\/059yy0eu\\\\\\\/SNxBqpdbq\\\\\\\/AejHzytl31h+cFqposzGeZnr5RO2auFxh3Vm6MU30mi9L88YkJGbTbpK01pXvfjl52+9D2VMoZjNmzBfLuNO72gkk0pIxzgFDe\\\\\\\/AMbsrS+hppX8NVmvDNqqd+Isnule7AVnl\\\\\\\/JBiinIcVcssub9\\\\\\\/V6eQMYC8cWwm2\\\\\\\/65ztRvv\\\\\\\/uU4dfTatogbEDna5ZFNPToyl\\\\\\\/guA4tTnatD7DCzwHiId0muc0VOOTSFmjJqMYnpbKljVehw1\\\\\\\/54ckG6KG6GuyCNjov4SDl1JnL2zi7\\\\\\\/Biqt4kPEyicOquanWb3+C\\\\\\\/lAuBC79wmF26d3PCCts8T5D8yU4AU7iExWKzQ3oOoA9wbTuiUrQDmwzUr9Fai3HVaeK4K+sbc8VsmONLmLdRo77QbODJOYJva2+sb6cyb0poT4app9v7tHADr6jE9zDesXkYFJArW0h+kT56oA\\\\\\\/xIl\\\\\\\/rMp\\\\\\\/d4wHa5Fxb+HkRg\\\\\\\/T4idNDnLYs97AtK3LRDE8Uy6Ll3FH1pFNzS4m+fosHYskmfyJEwt++1DvPZK3gWvXezVkSFN3Toy31359All+N5TGpApS82G1PE3ESxbihPzwh\\\\\\\/4f2hbr6OLWD8eD5812jDUOqpV8CY674bF5ufANbCSaKnNEPw5nxPn5rQUQZmNsm5oZWnBBL+kZxUQZUneQIMoHmhoHQEcDUOq7E19\\\\\\\/Lz6k8V0Hiok82VsuNfMrKMrNCT8aRhCnanrN3UHPUcU\\\\\\\/NXvPQbrz+mgA18M6e8h\\\\\\\/tTxSYTd6GYKPHYo7C2qW5twS4u11bM1Up\\\\\\\/laov5672TywE5YuG+jDH28hasm1t7rFvGkUJ1DRDayhe8u40bUdVQmo9jTtE8H44WZLlceo0NahhJWBoDjyQQhOnt+w1VZ7eWjHG5bbVM6iCEw2sLJbNrRgWQoRDAl5yyUdwaHN+po0QnEcI57pCeocqkzeFibEQGM5fZcTEamRbDeBV9VWlNtpnycdee9sy2vfLaE9bTs8l9pU5q5mHF2vQzNa1mwHkTZIlCi5FpcMLYxe6aDuUEIGilOhQJ\\\\\\\/DKd7RlTglmZfYTEwn03W\\\\\\\/jTSb+re99Dw8raE0tdVW1XAyB6x1HQj05PjyU+sqB4EBaiHQSHrPweON8QDn6jJMMWo9W+XBak2uelUllDoM4aG3OXCRsOpmiLvBI4x4+78joinH5ozE8DB66+8q8kcO\\\\\\\/Uo6NbsbKP0Wv7F7VWKzdIdi0cH78U\\\\\\\/5w7\\\\\\\/tIFR6nuR0dVPF+vR6O1oEacKRciBTEiG1sBuNvdoibyob2P1P5lGRbIAmGcKLRRs8pLIuQT7NE1YGmCbCaspykd\\\\\\\/TGnObNQGwzmTtQ\\\\\\\/FLiN1hvqr\\\\\\\/Gny+8ie\\\\\\\/qZR9sc9Yatlcej+MLbJxF7GYgQueguABGH3FXWw2fLZMKvNJUH\\\\\\\/XdsWbbMUiT8ZN+E3cy3DIuNxWpik1GjMmpYvBLCWmyLxQF8CWegbUWrdAtWrHVyzcPgsnP735ClVUl9RH+fRBk6M7qBfP2zoizF5gQYahc9HkydW\\\\\\\/pt4oUdTSe5Q\\\\\\\/KMUBtIzZSOy0V4+JdiHx9ROkHBM0KFNO8805GtcmqV3VavFCfS5tNGSSsdgpIyQZchljjNs5uu\\\\\\\/1L4GWsT2xQ8HcM2JIF2QjQDo430v+V8eW2Qr5LB9DL0lgHvOUWZJcLD6JQFDnexBcxgQeOhBTsomALJyEtEstgqlPzB8fgJFg687Q4t8lFWuD3lJ8PbwMRFs9xaveRw0VoDW4Fc1067LAVx6gqZUyLbGjVdhsC3IHqTmbmkPfg+J0We8qkRUs\\\\\\\/QrlH1HQ01ERmgh5oICF0D1qaZBEnb8d6tLzI6Gt0S8vtI2rQ9cjIklc\\\\\\\/fxxArfGPa7+jDRARTdQOYwbUOUoV84vMXIiOmNI1Hdzy+FSZWsfbQnOIW\\\\\\\/v67rRtamKvYoaX5XvGwIMHQ\\\\\\\/nKMqDrAmBP4AzuQLir3WliCwEKetE+lvi6P6aAEDQ23lwPDzARTgTlsRN0UfuSl9mGXUUFSgTcAZctmabja5ththQQTBZ\\\\\\\/OCwsynS8sj0rg7QCFSl3c4yJZBKVb3WSesQNMZAdgi\\\\\\\/JKrNYkvoSeHbGnrIPn8SsI9FC0Z6AO\\\\\\\/0scxHShuTZQjdMFuK+PSONeMq5x9a9T8U51CRjm6BJPG3WaCxUj7NEH8\\\\\\\/2VQ69k8JXgGj+gUG3WKLfhFwZiIQAa6vVER64Ccni5\\\\\\\/eaJmf7QXRv17fr7quFNO8NaQEyKRL3oW2dwlUaiZB6expeg4F1JXVl52jJvrn7i9l5ADkPhfybpNYKAwgMKivLkATePmVLB2XCFFuE+\\\\\\\/LS9DL6CeW6I4IpND1JDf3CsDbnFaEWs5ZWa6fKPvaEMIRqMieQtjeUjdbDbj\\\\\\\/QYud09VomkCpulcT18Cxv0ucCH3iJEr60ho1kK8WcJcwurvUzVv8\\\\\\\/xHoN9brLKgc0Txd8F1smhM\\\\\\\/dMKNfNxs8bOAHlJf+iJ3ea9zQ0jrG28djBORdzDAHr8OSFhc0I900uk2+xjn0637Xx\\\\\\\/ayrq93+wMn8FMMsJbYx2x2wiPST3909rBL5cVkWlH6LTcK8+2yAB+ryOBuBu+GiUrOUAMJbASEr4MYgbhtqcIZhUE+k6T+SgOj5HFSpvspsP\\\\\\\/V03whTTDBFMv2qi\\\\\\\/Vis6KyCEpn5Mc3zd3gCMExRZ+ulCcLKugHuhf4kQa+pAIpSu7mNeUEkB\\\\\\\/Vu2PzYlA\\\\\\\/f0MkRIZEN8GrzLsHNQ3XIxKSd3uSSOLfnky8Hw05jbOunkb3fOKCsUkqFIo6cya\\\\\\\/\\\\\\\/RRnAFsX9sW+gdLp+ilZHC8zsv4SVWBBluSqW\\\\\\\/MEPH1YJ7+R2XoFC42AeKhRS2Y92HntbbB\\\\\\\/R+VdBEacAiwK0EOi6h8tFj8Y\\\\\\\/HSELy\\\\\\\/Doovmf9qjg1+VK1f1+Z8hbzwEztC4QJHZ270wsL5+1+dlIZ9si0GiBGxT0wcy\\\\\\\/BbvJQ24VAe0H5ZeKX4hZrTlF8WIXYdq0bf2wx6pKNiEIZ5JXUuG2N5NN51iO7hpuiScEAuSQRDhnf+QH2oyS8GrPDh8n5PXqrc6g9V8qYF9SvbKOeONhVQeuB\\\\\\\/uhkbgTE7RltWY7z1Rr0QZi5bkI9Re9FmzUbrwzikzWrhIGEyna2H78m3HEk7FdDR9H24muiKIhcGtseqbYAFLV1SpPMFHUnBSS+g4l4hdKdUU96jU+8CiJq0ONRw0jxxFHi\\\\\\\/fyLnxaABd41IR3l1bbJ1Oy2++sKSj2cunlOz31lE2TpairPQ7cSJdcDp\\\\\\\/IQrN9oyYZgpdEi3m4eiX8oEjwF9nzdAuOkBTf+PdO1lqf0oZX+jHeOAkaqsZlMhr4KA3MEdeOHZGCalWuWX0bcvC0Y\\\\\\\/8efNkFHrGiYd4JupzMhUyFVp0QzT8XRrOFTJDtg6eHsLOG8Jb76cnHXOeynkDz9WZpoMGEqQM61+2UByMIO5zWl81rHoPnvyT3LBvZZlfx+tNdZZbKj1TP4raj94BxocnWQx09SmIJ6fvnLgtLOqVprKwqwaWaGuxE30w1uEYQ\\\\\\\/uouFesJZKNOFPDEM5vX1soxFycHMg2KSDslet8MuO+ezNntwmI52m2Ic10c00p4Ls7qcsHMsr6G372dpEvSNzi\\\\\\\/szjEHLoK5qSscxduNAtH2StLq4Bj57GFAWW5ft3DW20zlEYVEvjD6j0D+KGavoXNtJHRao0rAb2LYDVG5bb3PV8bqkUD2itH772NZfYrEbl77UazICBNqL3h48l+FA+YRxqnB6R0iRbGjj6Uu2\\\\\\\/Ay+kR4d564VH1YmPiy4lkjOUTkSs1C6bxKXK3ng2REUhIR3hfknuS\\\\\\\/V5Mqm6d14fYmab9hnHYBv6Wn1ZgO04PHBBiCEAr6XuVpP8udYnSZXsQQaU+VTjq4IruOkzx3EhQDpl5QixOjyeho7ZL+2An2\\\\\\\/qWCh04A5D+8thJGtn84UmwR\\\\\\\/nMJfwGIn\\\\\\\/JVe4fJEMijIiP2ePCkkFjaZNPsR+\\\\\\\/wS86PRbsy6bc4SGNs19p8JtdYFV2oI9X\\\\\\\/2nuvCEbBVQALxhs9N61ClgQQKzXnnAdoStdFLyAoPvffhX+ZhgmmPDcqDL2y0UANiPIbRnzITBkeQvRsnrf6TXeLRB9CQeINfBtpV9zP4QzCzOdpzL0Po3JXHtN1Tc+ar9ZXjfOTbI89OR\\\\\\\/kPUbHpEqDbssteUR5OPon0FY+EoFdkh0N5Lelyl8RhJEw1XQaZt9AnLC7XBE\\\\\\\/jpDUEa553zRHdq8xpnUcGCluGbTV0UQxgeppYH6DVRphlv7gz2s3EQ\\\\\\\/F9oqG1qhQFsw5cmwAjlNNJXoGY4aOVhlVECMg\\\\\\\/W\\\\\\\/ovOTNOmoauVca3m23BmCKDOnOSmaPH+iK6EhjbfNe\\\\\\\/ob83SOgkx8qALZunXpooiJi0Fw\\\\\\\/NMRTyRwSR3xwEI3Jg7cDbX5wv\\\\\\\/smItSLqnRCjwGfO0oYH+fKBWNPrVRNLdG4IhYtwMdamoneHjtDdMJx2gz8ItKc+UjGF25h70TloRRgAR3qUaCS846G2yv2I1f+J+rkjFbCjF1x9CVA6HX9SZVLKN0vPvRPSDJCDWQOTobYAv8ue4qgbPxvTsYxMjgUwbU4A+D\\\\\\\/TypGaRBbFNCohgssGykODLYjRf2PjB27vL46XulFn9UBZGdeotppTeMOUaAWcJ\\\\\\\/67U2sbycK6CZ9WynVST7I2E9XmVBfc2Ca4Aa8QajXVFQ718XYuc7A1YehPDcHQ4+ny6UW5qK3QUHOTK\\\\\\\/B85ljiS6XZk\\\\\\\/J6EyjOHb+IxNVDR86yMk6+hna9C9YRRwCGPn5yF9iUn\\\\\\\/wUBkM+Acn76wnnT0ywwZUVEn0KzlNSagnSYj+ZONX8fOT52393Ohq1eW3DC6w7541i2ZkFuVTigx\\\\\\\/yCdKkYdooAhaWnjmTcisDS1qHQ38ppc85pxw86fQOfyCYg7w+Fv+pfjcryqx\\\\\\\/h\\\\\\\/6Ta8T2EWfaIaXSj0eDZD36WxUx3KFzxm+XyM5Hqro+KgleNum17rEF3ejARWRYLS9clQxSLVsQRq+NmOXyNcmXFhBWNP+bIqioIVkwSGlvGSZSXk7qslugxJqfNu3vMvnDNfDm4oNMyo9bWSYbxdv70wGPAnQ7QeieoXG0eQ0+EiaclQjeoj2oNzbPipHmePwVn7dLnTrv3XE212pKhS8h1fomvw9ycbmSfY7i8xBOhtJpVldN7wj\\\\\\\/WkaRNBRoUlmH29+DuI3KUg0f7Z8Hu5qSZHV+KnsmizIKZB5tFE+WqpTtcqNRUf1H2hT0zys3gkX8QXfzX51PpN\\\\\\\/BvzPTnSz2u3GYXkgfiQo4gXkqexnDlYoc6svLuE8LPxd8i\\\\\\\/3WMcfvYwFjLikCygrYk+KvaBCGSwIol2PIIN8EOt9Jhp+dy2wjEg6LlBZBagJiZrF+j43qbFkTZFjSBZgN6YytGARgAaUjSjZEwmjgRYAAoKZEMMh9yb6s+E0RDwrMpsC5ZwX+4j0LkDcwCT5gxm1v1qgBIJKKRylxPaTlILT0QaZZBWX3lx3A8Rgf7x6zv+SvR7GIA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ps0VwtYxXtiOLi5yBGlbsg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ebbe123fc1e0d584439b812ca1b01e2abaaac8a5f4eada2864727e6b798f5ee1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226mUS9tv1YqjkQbBqK8Rh8eLjZcFw7Kibat6OM8e5hjk49W6eg6kYOAubSf1+BhrrJ0BynXDZuEa4uD89CuztU9rJUPTG\\\\\\\/Bd42Jf0AkWp6SlfJk2vVwur0XYcQrVm6EBowEL\\\\\\\/NWwq9Q3YgYceOqpaU6p3IrVr6ZrLHYI2lfVVGZ60Ko+xUB7iWVjpdfzISVaYFOCbTgExvP9lvIHwVMia\\\\\\\/Mt+shULi3ZI2St1sNKLB\\\\\\\/ZFN4vYdZ69RXDoCY\\\\\\\/kYDFrds9M9qB7IG3nc1HsyAy7TnrHeeUORbB8zojtwQb05tGA6TQkElzK3WLvAhQ1PhdAgPf05WRGFnAvc4axi+72LTC03RSp48U6Fs04ejZbnwOWTjis3FMak\\\\\\\/MqmCZjz7OD5qXBg0caHk4L3ah3Yy\\\\\\\/dtPWzeSww4CLEyhTGebfi4nowZXwrINRwNMdEa3xoqFk6QfQykC3mX6dt9VGBkpb7THdK\\\\\\\/qcZLXhAg1HKxaDVj5ANRtjcrpWDG6MvMNCqWRBYb5whjLT3iGjr+cZ6xv0U4EIrPvL6CfYvd3qsRdtRAzpOLHU8cmaKWMxQhPbJCnyB5iTmTkjD4RcR30MOXlpRGPwNw75fMhf+L\\\\\\\/IR+t6kZPsGXtz0O3T4T7Yp5pBRj0GobQPYarHOVs76iIgeqAGur4TfCGcrpQVGVf7itYT\\\\\\\/Bcp+w\\\\\\\/97hkeIFv7EhH3yN\\\\\\\/PF9HkAr9IIqbW9je2NMCjnQ+Ju3tRDELsw3ZumDslu92atb19xPoI508p1lMovWviHv7HDqSh+DVg0\\\\\\\/reeERtJVYBwHY5RsX2micx6aKLYKniCCvlsP1E1ZForqqsU2m8SifYwJixs2\\\\\\\/xS09RmCkWr\\\\\\\/L\\\\\\\/Zq5czPWCl8Tww42FcwXGLYoFRtC13qLyE35IEb\\\\\\\/vMgfFmN4lvJ+OUzanwJ0l7uRTezg2xWZtQadBTbj\\\\\\\/ekQf5dNajIxr8X2NiS2gYYzMuQCZ8yigVK24FLM5D7PhxUEqMIQLLsArIFrSEqXYwSsG5BK4CsK2UDLHxn2Dm+pg5orezvDf+Wd6gkhsh7GB0sCaTmzvAIsO7aixArz7Yq5ye9lMuDni4\\\\\\\/5Ni\\\\\\\/nwa2m+TX\\\\\\\/m8DObhQaIBklUlch01sTuqtnfoHBo\\\\\\\/wDReKZyr5UruRs+16eFhCzHUfqc7rVarJXapKEctNz4P1CzwCgtUDuFonb8GiBib7V\\\\\\\/7ZFhv+dpX9BOdfoB43r0umdubQ86zD4wf9EPdxT9i7sWzzVzoqJUBCq6t\\\\\\\/fdGFj7h37gG0H5284hxnKBht7mHJvkluzXjkHyDRQRm3Z0aPD5WoXSw3CCW5opsb0My+q9zpw1YsghofO9iUYLhaFRb\\\\\\\/ERtpGUKdqRythbUbgZP0ptsIE\\\\\\\/qbR2t1XKWnYSXNI+CBU8DOHCfbIRNi7SFPOPclWieEkxDFqpxATD0O\\\\\\\/o+20mwmgeJuytcFUj5fbhXvGcco8dp91QYdlMttAeutjzl42B1tMIuawuQfmK6Oo1\\\\\\\/i6bXO7i2gbnPbxe25jYBh974ghomiEzZ8T3zPD3cpzbmJHxPQA5huaCivP7xDwj0IM6bXZa7u3NDCF\\\\\\\/nu98FKT56g15WAwGaU\\\\\\\/Et29ZofsSK9IjXX805CrC5mFZZsTZo331Ar4dWAAmSvsnKv6ZGZsbHYJSqMED18xsLem2Nmnfy5PcTe853ZAXijj4k9yjO+B0LlY5GWMwN+AnXlNkhyVuZuXc1w\\\\\\\/6\\\\\\\/4iFu9sfhtGG4ToUVicQq\\\\\\\/OYMa5ag6W4bDKx3j7hYapzpH6GvLN7ZZzVJvsAfQW4wYCzWsa0JlA7xvneBFu3yDY4lrXa5ja7ioBr3qsaYEZxuHuquNrzZLi5eBfew1j9nBTkccxaCBPcQmFbN5Rb8mg\\\\\\\/RsmM1FMGpHGl9mA5Q5S\\\\\\\/vgGzKO3AEfcKvZcnktZ6GfIjqD3fn6m3uKz\\\\\\\/iylTNqChsdoAoiHFjy5UIb6U6Y76VxdY44s3r31bNdCn1CNse5VlF\\\\\\\/k1b0BfknVPTCCl\\\\\\\/xNavUPWssYTOU4FrH+fH7A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UqwIp3DG5\\\\\\\/8ytBhNEFbdsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c7ffe8e29619e14c2ee3cba0bb86108e7564c028576241f1e11c89eafbde91cc\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-488303 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/599" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tajbox-wsg-fEAB5QzQfGSuc4dq-kCfhV7B9f9CUz-0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/599" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MaeJ0lM-sc6xSGipqmS6DvDzrBcWDmKBXleuLkqnk5E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/587" title="Dosage-Dependent Expression Variation Suppressed on the Drosophila Male X Chromosome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/607" title="Transcriptional Profiling of Saccharomyces cerevisiae Reveals the Impact of Variation of a Single Transcription Factor on Differential Gene Expression in 4NQO, Fermentable, and Nonfermentable Carbon Sources" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488303" id="node488303" data-pisa="ggg;8/2/599" data-pisa-master="ggg;g3.117.300483" data-apath="/ggg/8/2/599.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yanyan Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8304-5663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sican Xiong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6350-1107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Sun</span> and <span class="highwire-citation-author" data-delta="3">Fei Zou</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">599-605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300483 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488303" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yanyan Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Mathematics and Statistics, Wuhan University, Hubei 430072, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanyan%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYanyan%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sican Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Mathematics and Statistics, Wuhan University, Hubei 430072, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sican%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASican%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8304-5663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sican Xiong</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6350-1107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Sun</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fei Zou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BZou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zou%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFei%2BZou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:feizou@email.unc.edu" class="" data-icon-position="" data-hide-link-title="0">feizou@email.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/2/599" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:488303" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:488303/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/599.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:488303" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:488303/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/599.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:488303" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:488303/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/2/599.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:488303" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:488303/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/2/599.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:488303" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:488303/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multiparent populations (MPP) have become popular resources for complex trait mapping because of their wider allelic diversity and larger population size compared with traditional two-way recombinant inbred (RI) strains. In mice, the collaborative cross (CC) is one of the most popular MPP and is derived from eight genetically diverse inbred founder strains. The strategy of generating RI intercrosses (RIX) from MPP in general and from the CC in particular can produce a large number of completely reproducible heterozygote genomes that better represent the (outbred) human population. Since both maternal and paternal haplotypes of each RIX are readily available, RIX is a powerful resource for studying both standing genetic and epigenetic variations of complex traits, in particular, the parent-of-origin (PoO) effects, which are important contributors to many complex traits. Furthermore, most complex traits are affected by &gt;1 genes, where multiple quantitative trait locus mapping could be more advantageous. In this paper, for MPP-RIX data but taking CC-RIX as a working example, we propose a general Bayesian variable selection procedure to simultaneously search for multiple genes with founder allelic effects and PoO effects. The proposed model respects the complex relationship among RIX samples, and the performance of the proposed method is examined by extensive simulations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/imprinting" class="hw-term hw-article-keyword hw-article-keyword-imprinting" rel="nofollow">imprinting</a></li><li class="kwd"><a href="/keyword/bayesian-variable-selection" class="hw-term hw-article-keyword hw-article-keyword-bayesian-variable-selection" rel="nofollow">Bayesian variable selection</a></li><li class="kwd"><a href="/keyword/parameter-expanded-gaussian-priors-0" class="hw-term hw-article-keyword hw-article-keyword-parameter-expanded-gaussian-priors" rel="nofollow">parameter-expanded Gaussian priors</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">Multiparental Populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Most human genes have functional mouse counterparts, and genomes of both organisms are organized similarly. Thus, mouse serves as a good model organism for complex human diseases. Recombinant inbred (RI) mice are among the major mouse resources in biomedical and genetic research. However, traditional mouse RI lines are derived from only two inbred parental strains, with limited a number of lines available, resulting in a low percentage (15%) of genetic variation across all mouse inbred strains (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yuan <em>et al.</em> 2011</a>) and extensive blind spots where a sizable proportion of the genome is identical by descent. These limitations make traditional RI insufficient for studying complex traits. With the need for more powerful resources, multiparent populations (MPP) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de Koning and McIntyre 2017</a>), a set of inbred lines using multiple lines as founders, can overcome the limitations of traditional RI lines and have become an innovative tool for fine quantitative trait locus (QTL) mapping. In the past 15 yr, different kinds of MPP have been established in plants and animals, such as nested association mapping design (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Yu <em>et al.</em> 2008</a>) and multiparent advanced generation intercross (MAGIC) populations (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cavanagh <em>et al.</em> 2008</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Huang <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ladejobi <em>et al.</em> 2016</a>) in plants, and the <em>Drosophila</em> Synthetic Population Resource (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">King <em>et al.</em> 2012</a>) in animals. The mouse MPP include the collaborative cross (CC) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Complex Trait Consortium 2004</a>), in which a genetically diverse set of eight inbred strains were selected as breeding founders (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Iraqi <em>et al.</em> 2008</a>) to generate a large number of RI lines. The eight founder strains were predicted to represent ∼90% of the genetic variation presented in laboratory mice (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Threadgill and Churchill 2012</a>). The CC thus greatly overcomes the limitations of the traditional mouse RI lines, and is the only mammalian resource with high genome-wide genetic variation that is uniformly distributed across a large, heterogeneous, and infinitely reproducible population (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Threadgill <em>et al.</em> 2011</a>).</p><p id="p-4">Through the generation of RI intercrosses (RIX) of RI lines, a large number of potential “outbred” RIX samples can be generated. That is, given <em>L</em> RI lines, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-1.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-2.gif"/></noscript></span></span> genetically distinct reciprocal or nonreciprocal <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-3.gif"/></noscript></span></span> individuals, or RIX, can be produced. An example of using CC-RIX to illustrate a distinct response to Ebola virus infection, by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rasmussen <em>et al.</em> (2014)</a>, was summarized by the editor: “the CC-RIX mice could prove valuable for preliminary screens of candidate therapeutics and vaccines.” Since all parental RI lines are isogenic at each locus, genotypes of RIX can be imputed in advance from those of their parental RI lines. Additional advantages of RIX can be found in <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Threadgill and Churchill (2012)</a>, in particular, its power in support of analysis of parent-of-origin (PoO) effects, where effects of certain alleles are different depending on whether those alleles are inherited maternally or paternally. PoO makes a significant contribution to the heritability of most complex traits (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mott <em>et al.</em> 2014</a>). In addition, genomic PoO effects provide a great model to study epigenetic regulation of gene expression (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Barlow 2011</a>).</p><p id="p-5">During the past 20 yr, QTL mapping methods, including analysis of variance (ANOVA), interval mapping (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lander and Botstein 1989</a>), composite interval mapping (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jansen and Stam 1994</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zeng 1994</a>), and multiple interval mapping (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kao and Zeng 1997</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kao <em>et al.</em> 1999</a>), have been well developed using experimental crosses, such as backcross, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-4.gif"/></noscript></span></span>, and RI (see <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broman (2001)</a> for reviews), and many excellent open software packages, such as QTLCart (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Basten <em>et al.</em> 1999</a>), MapManager (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Manly and Olson 1999</a>), and R/qtl (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2003</a>), are freely available online. For diallel data obtained from the CC founder strains, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lenarcic <em>et al.</em> (2012)</a> employed a general Bayesian model for decomposing phenotypic variance into biologically intuitive components. <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crowley <em>et al.</em> (2014)</a> also applied a Bayesian method to a diallel cross of the eight founder strains to estimate genome-wide genetic and PoO effects (not QTL mapping) on responses to haloperidol, an antipsychotic drug. <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhang <em>et al.</em> (2014)</a> proposed a general single QTL Bayesian framework for MAGIC data to coherently estimate haplotype and diplotype effects of founder alleles. Similarly, for MAGIC data, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wei and Xu (2016)</a> developed a single QTL mapping method with a random-effects model by treating the founder allelic effects of each locus as random, and scanning the entire genome one locus at a time using a likelihood ratio test. For RIX mice, a mixed-effects model was developed for modeling the unbalanced relatedness among them (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tsaih <em>et al.</em> 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zou <em>et al.</em> 2005</a>). <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gong and Zou (2012)</a> proposed a more flexible, nonparametric single QTL mapping method for detecting QTL with time-varying coefficients. <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hallin <em>et al.</em> (2016)</a> applied a random-effects model to map single QTL by partitioning the variance of growth traits across different environments of yeast strains into additive, dominance, and pairwise epistatic components. However, many complex traits are affected by &gt;1 gene, and multiple QTL mapping may be more powerful than locus-by-locus analysis. For CC-RIX data, <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Yuan <em>et al.</em> (2011)</a> constructed a mixed-effects model to simultaneously map multiple QTL, but treated the eight CC founder alleles as standard biallelic single-nucleotide polymorphisms (SNPs). For many complex traits, it is arguable that modeling the effects of the eight founder alleles could lead to improved mapping power (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Vered <em>et al.</em> 2014</a>). In addition, for complex traits where PoO effects are suspected, modeling such effects may further improve QTL mapping power (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lawson <em>et al.</em> 2013</a>) and understanding of etiologies of complex traits (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Threadgill and Churchill 2012</a>).</p><p id="p-6">In this paper, for MPP-RIX data, we develop a Bayesian variable selection procedure to simultaneously map multiple genes with founder allelic effects and PoO effects. We demonstrate our method with CC-RIX data, but the method is general enough for other MPP-RIX populations. We place parameter-expanded Gaussian (PeG) priors on both the random founder allelic effects and PoO effects for variable selection.</p><p id="p-7">The paper is organized as follows. In the <em>Statistical Method</em> section, we first introduce the CC-RIX experiment, then propose a random-effects model and describe a Bayesian variable selection procedure. In the <em>Simulation Study</em> section, we perform extensive simulation to examine the proposed method. Summary comments are given in the <em>Discussion</em> section.</p><div class="section methods" id="sec-1"><h2 class="">Statistical Method</h2><p id="p-8">The CC-RIX panel is the RI intercross of CC lines. For <em>L</em> CC RI lines, a total of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-5.gif"/></noscript></span></span> reciprocal CC-RIX and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-6.gif"/></noscript></span></span> nonreciprocal CC-RIX can be generated [see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> of <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Zou <em>et al.</em> (2005)</a>, <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Yuan <em>et al.</em> (2011)</a>]. Let <em>n</em> be the total number of CC-RIX samples and <em>p</em> be the total number of genetic markers. Further, let the phenotype of individual <em>i</em> (the dependent variable) be <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-7.gif"/></noscript></span></span></p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/599/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ROC curves. (A&#x2013;C) for cases 1, 2, and 3, respectively, and (D&#x2013;F) for cases  , and  respectively. The legends &#x201C;POE&#x201D;, &#x201C;Mixed&#x201D;, &#x201C;Yuan&#x201D; and &#x201C;LMM&#x201D; represent our proposed model (2), the mixed model (3), Yuan&#x2019;s model (5), and the LMM model (4), respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-183981797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ROC curves. (A&#x2013;C) for cases 1, 2, and 3, respectively, and (D&#x2013;F) for cases &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-8&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; respectively. The legends &#x201C;POE&#x201D;, &#x201C;Mixed&#x201D;, &#x201C;Yuan&#x201D; and &#x201C;LMM&#x201D; represent our proposed model (2), the mixed model (3), Yuan&#x2019;s model (5), and the LMM model (4), respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/599/F1.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/2/599/F1.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/599/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/599/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488385" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">ROC curves. (A–C) for cases 1, 2, and 3, respectively, and (D–F) for cases <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F1/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F1/embed/mml-math-8.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F1/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F1/embed/mml-math-9.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F1/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F1/embed/mml-math-10.gif"/></noscript></span></span> respectively. The legends “POE”, “Mixed”, “Yuan” and “LMM” represent our proposed model (2), the mixed model (3), Yuan’s model (5), and the LMM model (4), respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Model</h3><p id="p-10">In order to account for the unbalanced relatedness between the CC-RIX mice and model the eight founder alleles and PoO effects, we extend the mixed additive random-effects model of <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gong and Zou (2012)</a> and <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Yuan <em>et al.</em> (2011)</a> as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-13.gif"/></noscript></span></span> are binary {0, 1} variables used to decide whether the <em>j</em>th QTL and PoO should be included in or excluded from the model and µ is the overall mean; <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-14.gif"/></noscript></span></span> and the <em>k</em>th element of <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-15.gif"/></noscript></span></span> equals 2, 1, or 0, depending on whether <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-16.gif"/></noscript></span></span> inherits 2, 1, or 0 copies of the <em>k</em>th founder allele (<em>k</em> = 1, ⋯, 8) at the <em>j</em>th candidate locus; <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-17.gif"/></noscript></span></span> of parents of <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-18.gif"/></noscript></span></span> that are equal to <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-19.gif"/></noscript></span></span> Let <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-20.gif"/></noscript></span></span> be an <em>n</em> × L matrix whose <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-21.gif"/></noscript></span></span>th element equals <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-22.gif"/></noscript></span></span> Clearly, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-23.gif"/></noscript></span></span> for all <em>i</em> = 1, 2, ⋯, <em>n</em>, since each CC-RIX has two and only two parents. Therefore, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-24.gif"/></noscript></span></span> represents the <em>k</em>th founder allelic effect of locus <em>j</em>, and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-25.gif"/></noscript></span></span> represents the random polygenic effect of founder strain <em>l</em>. We let <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-26.gif"/></noscript></span></span> follow <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-27.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-28.gif"/></noscript></span></span> and the random error <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-29.gif"/></noscript></span></span> follow <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-30.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-31.gif"/></noscript></span></span></p><p id="p-11">At a given locus, PoO effects can only be estimated from individuals with heterogeneous genotypes, since for those with homozygote genotypes, the parental contributions cannot be distinguished. Label the eight-founder alleles A, B, C, D, E, F, G, and H by numbers 1–8, and let the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-32.gif"/></noscript></span></span> element of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-33.gif"/></noscript></span></span> be equal to 1 or −1 if the maternal or paternal allele of the <em>i</em>th subject at the <em>j</em>th locus equals the <em>k</em>th founder allele, and 0 otherwise. For those CC-RIX samples with a homozygous genotype at a given locus, such as AA or BB, we let <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-34.gif"/></noscript></span></span> Therefore, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-35.gif"/></noscript></span></span> represents the PoO effects of the <em>j</em>th locus.</p><p id="p-12">Model (1) can be expressed by the following matrix form:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-37.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-38.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-39.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-40.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-41.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-43.gif"/></noscript></span></span></p><p id="p-13">To enable selection of QTL and PoO effects, we reexpress the eight-dimensional vectors <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-44.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-45.gif"/></noscript></span></span> as the expanded parameters <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-47.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-48.gif"/></noscript></span></span> respectively, such that <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-49.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-50.gif"/></noscript></span></span> This parameter-expansion idea was first proposed by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kuo and Mallick (1998)</a> and has an advantage in selecting important variables and shrinking coefficients.</p><p id="p-14">In the above model (2), the observed data are <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-51.gif"/></noscript></span></span> marker genotypes <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-52.gif"/></noscript></span></span> PoO information matrix <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-53.gif"/></noscript></span></span>, and parental CC-RI information matrix <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-54.gif"/></noscript></span></span> The unobserved variables include <em>μ</em>, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-55.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-56.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-57.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-58.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-59.gif"/></noscript></span></span>, and the indicators <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-60.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-61.gif"/></noscript></span></span>. We assign the following conjugate Gaussian priors to the <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-62.gif"/></noscript></span></span> QTL and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-63.gif"/></noscript></span></span> PoO effects as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-64.gif"/></noscript></span></span>For convenience, we name the above hierarchical priors for QTL and PoO coefficients as PeG priors in the sequel. We assign conjugate noninformation hyper-priors to the variance parameters<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-65.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-66.gif"/></noscript></span></span> is used to ensure that the posterior distribution is proper (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">ter Braak <em>et al.</em> 2005</a>). The above PeG priors are equivalent to “spike and slab” point mass mixture Gaussian priors (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mitchell and Beauchamp 1988</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">George and McCulloch 1993</a>) for the expanded parameters<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-67.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-68.gif"/></noscript></span></span>It is well known that these priors can achieve variable selection. However, compared with mixture priors, the PeG priors can employ a block Gibbs sampler to update the two blocks of parameters, one for <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-69.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-70.gif"/></noscript></span></span> (corresponding to the selected predictors) and one for <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-71.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-72.gif"/></noscript></span></span> (corresponding to the unselected predictors) in turn, and therefore can dramatically reduce computational time, especially for high-dimensional data with large <em>p</em>.</p><p id="p-15">For indicator variables <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-73.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-74.gif"/></noscript></span></span> we specify Bernoulli priors with inclusion probability <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-76.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-77.gif"/></noscript></span></span> respectively. To be more flexible, we further apply hierarchical uniform priors to <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-78.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-79.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-80.gif"/></noscript></span></span>Lastly, we specify a flat prior on <em>μ</em> as <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-81.gif"/></noscript></span></span></p></div><div id="sec-3" class="subsection"><h3>Block Gibbs sampling algorithm for posterior computation</h3><p id="p-16">The specific priors above result in known marginal conditional distributions for all variables. The blockwise Gibbs sampling algorithm that we employ can be summarized as follows:</p><p id="p-17">First, we initiate <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-82.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-83.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-84.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-85.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-86.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-87.gif"/></noscript></span></span> from uniform distribution <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-88.gif"/></noscript></span></span> then sample other parameters <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-89.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-90.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-91.gif"/></noscript></span></span>, and indicators <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-92.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-93.gif"/></noscript></span></span> from their priors. We then perform the following block Gibbs sampling procedures. Superscripts <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-94.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-95.gif"/></noscript></span></span> signify the Markov chain Monte Carlo (MCMC) iterations, and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-96.gif"/></noscript></span></span> refers to the initial iteration.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-18"><em>Step 1</em>. Updating <em>μ</em>: <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-97.gif"/></noscript></span></span> is drawn from the normal distribution.<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-98.gif"/></noscript></span></span></p></li><li id="list-item-2"><p id="p-19"><em>Step 2</em>. Updating <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-99.gif"/></noscript></span></span> we divide the long vector <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-100.gif"/></noscript></span></span> into two blocks <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-102.gif"/></noscript></span></span> corresponding to the selected (<span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-103.gif"/></noscript></span></span>) and unselected (<span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-104.gif"/></noscript></span></span>) predictors, respectively. We then sample <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-105.gif"/></noscript></span></span> from their priors, <em>i.e.</em>, the eight-dimensional multivariate normal distribution with zero mean and covariance matrix <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-106.gif"/></noscript></span></span> and sample <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-107.gif"/></noscript></span></span> from the eight-dimensional multivariate normal distribution with mean <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-108.gif"/></noscript></span></span> and covariance matrix <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-109.gif"/></noscript></span></span></p></li><li id="list-item-3"><p id="p-20"><em>Step 3</em>. Updating <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-110.gif"/></noscript></span></span> similar to Step 2, we divide <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-111.gif"/></noscript></span></span> into two parts, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-112.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-113.gif"/></noscript></span></span> corresponding to the selected (<span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-114.gif"/></noscript></span></span>) and unselected (<span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-115.gif"/></noscript></span></span>) predictors, respectively. Then <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-116.gif"/></noscript></span></span> is sampled from their priors, <em>i.e.</em>, the eight-dimensional multivariate normal distribution with zero mean and covariance matrix <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-117.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-118.gif"/></noscript></span></span> is sampled from the eight-dimensional multivariate normal distribution with mean<span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-119.gif"/></noscript></span></span> and covariance matrix <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-120.gif"/></noscript></span></span></p></li><li id="list-item-4"><p id="p-21"><em>Step 4</em>. Updating <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-121.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-122.gif"/></noscript></span></span> is drawn from the multivariate normal distribution with mean <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-123.gif"/></noscript></span></span> and covariance matrix <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-124.gif"/></noscript></span></span></p></li><li id="list-item-5"><p id="p-22"><em>Step 5</em>. Updating <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-125.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-126.gif"/></noscript></span></span> is sampled from the scale-inverted <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-127.gif"/></noscript></span></span> distribution, <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-128.gif"/></noscript></span></span></p></li><li id="list-item-6"><p id="p-23"><em>Step 6</em>. Updating <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-129.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-130.gif"/></noscript></span></span> is sampled from the scale-inverted <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-131.gif"/></noscript></span></span> distribution, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-132.gif"/></noscript></span></span></p></li><li id="list-item-7"><p id="p-24"><em>Step 7</em>. Updating <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-133.gif"/></noscript></span></span> the random-effects variance <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-134.gif"/></noscript></span></span> is sampled from the scale-inverted <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-135.gif"/></noscript></span></span> distribution, <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-136.gif"/></noscript></span></span></p></li><li id="list-item-8"><p id="p-25"><em>Step 8</em>. Updating <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-137.gif"/></noscript></span></span> the residual variance <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-138.gif"/></noscript></span></span> is sampled from the scale-inverted <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-139.gif"/></noscript></span></span> distribution, <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-140.gif"/></noscript></span></span></p></li><li id="list-item-9"><p id="p-26"><em>Step 9</em>. Updating <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-141.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-142.gif"/></noscript></span></span> is sampled from the Bernoulli distribution with success probability <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-143.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-144.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-145.gif"/></noscript></span></span></p></li><li id="list-item-10"><p id="p-27"><em>Step 10</em>. Updating <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-146.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-147.gif"/></noscript></span></span> is sampled from the Bernoulli distribution with success probability <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-148.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-149.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-150.gif"/></noscript></span></span></p></li><li id="list-item-11"><p id="p-28"><em>Step 11</em>. Updating <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-151.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-152.gif"/></noscript></span></span> is sampled from the beta distribution <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-153.gif"/></noscript></span></span></p></li><li id="list-item-12"><p id="p-29"><em>Step 12</em>. Updating <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-154.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-155.gif"/></noscript></span></span> is sampled from the beta distribution <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-156.gif"/></noscript></span></span></p></li></ul></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-30">Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300483/-/DC1/FileS1.docx">File S1</a> contains five supplemental figures. The MATLAB code used to analyze the simulated data is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300483/-/DC1/FileS2.zip">File S2</a>.</p></div></div><div class="section other1" id="sec-5"><h2 class="">Simulation Study</h2><p id="p-31">In this section, we run extensive simulations to evaluate the performance of the proposed Bayesian method. We apply the loop design in <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Zou <em>et al.</em> (2005)</a> and <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Yuan <em>et al.</em> (2011)</a> by ordering the <em>L</em> CC-RI lines randomly and forming them into a circle, and then mating each CC-RI line (clockwise) with the next three CC-RI lines after it (<em>i.e.</em>, CC-RI<sub>1</sub> mated with CC-RI<sub>2</sub>, CC-RI<sub>3</sub>, and CC-RI<sub>4</sub>; CC-RI<sub>2</sub>mated with CC-RI<sub>3</sub>, CC-RI<span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-157.gif"/></noscript></span></span>, and CC-RI; and so on). For <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-158.gif"/></noscript></span></span> in this way, we can generate a total of <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-159.gif"/></noscript></span></span> CC-RIX samples. Parental CC-RI genotypes are simulated in R/qtl (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Broman <em>et al.</em> 2003</a>). All the simulation results are based on total 100 replications for each simulation setup.</p><p id="p-32">In model (2), we set the overall mean <em>μ</em> = 1, the variance of random errors <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-160.gif"/></noscript></span></span> the polygenic effect variance <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-161.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-162.gif"/></noscript></span></span> Nineteen chromosomes each with a length of 70 cM are simulated, on which total <em>p</em> = 133, 266, and 1330 evenly spaced markers (resulting in 10-, 5-, and 1-cM intervals between nearby markers on each chromosome, named cases 1–3) are generated, corresponding to three marker density cases. Among the total <em>p</em> simulated markers, we randomly select five markers, and let the first three markers have QTL effects the last three markers have PoO effects. That is, the first two genes only have QTL effects, and the last two only have PoO effects. However, the middle-selected marker has both QTL and PoO effects. The corresponding variances of QTL or PoO effects of the selected markers are all set to 1.</p><p id="p-33">To better assess the performance of our method in situations where multiple nearby SNPs jointly affect the outcome, for cases 1–3, we now let the number of causal SNPs in each of the first two QTL be 2 instead of 1. Specifically, we randomly select two nearby SNPs for each of the first two simulated QTL and denote the alleles of the two SNPs <em>A</em> and <em>a</em>. Based on the haplotype frequencies <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-163.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-164.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-165.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-166.gif"/></noscript></span></span> (without loss of generality, we assume that <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-167.gif"/></noscript></span></span>), we create three haplotype allelic groups, where haplotype <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-168.gif"/></noscript></span></span> is group 1, <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-169.gif"/></noscript></span></span> is group 2, and the other two are group 3, and set the genetic effects of the three groups to 1, 2, and 3, respectively. The three new simulation setups are labeled as cases 1<sup>∗</sup> to 3<sup>∗</sup>.</p><p id="p-34">For each simulated data point, we generate a single long chain with 20,000 cycles, of which the first 10,000 cycles are discarded as burn-in, resulting in a total of 10,000 samples for post-MCMC analysis. All the analysis is done in MATLAB, and the MATLAB source code is submitted as supplemental material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300483/-/DC1/FileS2.zip">File S2</a>).</p><p id="p-35">For comparison, we fit each simulated data point with the following three models.</p><div id="sec-6" class="subsection"><h3>The same model as (2), but without the PoO terms</h3><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-170.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><p id="p-36">All the priors are set to be the same as their counterparts in model (2). For convenience, we subsequently call model (3) the “mixed model”.</p></div><div id="sec-7" class="subsection"><h3>Linear mixed-effects model (LMM) for single gene mapping</h3><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-171.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span><p id="p-37">Here, random effects <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-172.gif"/></noscript></span></span> and random errors <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-173.gif"/></noscript></span></span> and the random effects <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-174.gif"/></noscript></span></span> are also assumed to be independent of the random errors <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-175.gif"/></noscript></span></span> as before. Moreover, <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-176.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-177.gif"/></noscript></span></span> are the fixed QTL and PoO effects, respectively, of the <em>j</em>th tested locus. Given the constraints that <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-178.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-179.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-180.gif"/></noscript></span></span> we force the intercept <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-181.gif"/></noscript></span></span> in model (4) to be <strong>0</strong> to overcome the identifiability problem, and jointly test the effects of the <em>j</em>th locus as:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-182.gif"/></noscript></span></span>In contrast to <em>H</em><sub>1</sub><em><sub>j</sub></em>, some of the equations in <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-183.gif"/></noscript></span></span> are not satisfied. After obtaining the maximum likelihood estimates of the parameters in model (4), we perform a test with the following log-odds ratio (LOD):<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-184.gif"/></noscript></span></span>which is equivalent to the log-likelihood ratio test, where <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-185.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-186.gif"/></noscript></span></span> are the likelihood of model (4) under null hypothesis <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-187.gif"/></noscript></span></span> and alternative hypothesis <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-188.gif"/></noscript></span></span> respectively. Since the above hypothesis test is performed <em>p</em> times, it is necessary to find an appropriate significance threshold to control the multiple testing, which can be obtained, for example, by modified permutation procedures (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Zou <em>et al.</em> 2005</a>). However, in our comparison, we evaluate the receiver operating characteristic (ROC) curve, which only requires the use of LOD scores.</p></div><div id="sec-8" class="subsection"><h3>Yuan’s Bayesian method</h3><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-189.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span><p id="p-38">Here, <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-190.gif"/></noscript></span></span> is the effect of the <em>j</em>th putative QTL with <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-191.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-192.gif"/></noscript></span></span> equals −1, 0, or 1 if the putative QTL genotype is <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-193.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-194.gif"/></noscript></span></span>, or <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-195.gif"/></noscript></span></span> respectively. The other parameters are set the same as those in the mixed model. The prior of <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-196.gif"/></noscript></span></span> is set to <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-197.gif"/></noscript></span></span> 1 ≤ <em>j</em> ≤ <em>p</em> in <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Yuan <em>et al.</em> (2011)</a>.</p><p id="p-39">To compare the methods, ROC curves (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fawcett 2006</a>), where true positive rates (also known as sensitivity) are plotted against false positive rates (also known as 1-specificity) evaluated at various threshold cut-offs. To estimate the sensitivity (the proportion of positives that are correctly identified as such) and specificity (the proportion of negatives that are correctly identified as such), we define false and true positive findings as follows. If a detected locus falls no more than 10 cM apart from any simulated genes, we call it a true positive finding, otherwise a false positive finding. Then, combining the outputs of each model for the 100 data sets, we can calculate the corresponding sensitivity and specificity. Specifically, for the <em>j</em>th marker, we record the LOD scores, <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-198.gif"/></noscript></span></span>, for the LMM model; the maximum posterior frequency between <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-199.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-200.gif"/></noscript></span></span> for our proposed method; the posterior frequency <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-201.gif"/></noscript></span></span> for the mixed model; and the posterior mean of <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-202.gif"/></noscript></span></span> for Yuan’s model. The corresponding area under the ROC curve (AUC) is also calculated for each of the four models, and the results are presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Generally speaking, a model with a higher AUC value indicates on average a better performance compared with those with lower AUC scores (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fawcett 2006</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488450/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488450/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Simulation settings and AUC values</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">From <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, it is clear that our proposed method outperforms the other three methods for all cases, regardless of whether there is a single causal SNP or multiple causal SNPs in each QTL. Yuan’s method fails in all the simulated cases, which is expected as the method only models biallelic SNP effects instead of the founder allelic effects that we simulate. The LMM model outperforms the mixed model, in particular when the marker density is sparse. This phenomenon is further confirmed by <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, where the ROC curves of the proposed method are always higher than the ROC curves of the other three methods; the ROC curves of Yuan’s model fall along the 45-degree line, indicating its low power for mapping genes with founder allelic effects. The ROC curves of the LMM model fall between those of our proposed model and the mixed model in cases 1 and 2, and cases 1* and 2*, but cross with the ROC curves of the mixed model in cases 3 and 3*. <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> show that the mapping precisions of all the methods increase as the number of markers increases, as expected.</p><p id="p-46"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> presents the Manhattan plots of the four models based on the average estimate of a total of 100 replications across the whole genome for case 1. Clearly, our proposed method is the winner in mapping genes with founder allelic effects (marked by an asterisk) and PoO effects (marked by a circle). The Manhattan plots for cases 2 and 3, and cases 1*, 2*, and 3*, are included in Figures S1–S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300483/-/DC1/FileS1.docx">File S1</a>, respectively, and show similar patterns.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/2/599/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimate plot for case 1. X-axis: * indicates QTL location, and o indicates PoO location. Y-axis: (A)  (1 &#x2264; j &#x2264; p) in our proposed model (2); (B)  (1 &#x2264; j &#x2264; p) in the mixed model (3); (C)  (1 &#x2264; j &#x2264; p) in Yuan&#x2019;s model (5); and (D) LOD scores for linear mixed-effects model (LMM) (4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-183981797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Estimate plot for case 1. &lt;em&gt;X&lt;/em&gt;-axis: * indicates QTL location, and o indicates PoO location. &lt;em&gt;Y&lt;/em&gt;-axis: (A) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-207&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (1 &#x2264; &lt;em&gt;j&lt;/em&gt; &#x2264; &lt;em&gt;p&lt;/em&gt;) in our proposed model (2); (B) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-208&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (1 &#x2264; &lt;em&gt;j&lt;/em&gt; &#x2264; &lt;em&gt;p&lt;/em&gt;) in the mixed model (3); (C) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-209&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (1 &#x2264; &lt;em&gt;j&lt;/em&gt; &#x2264; &lt;em&gt;p&lt;/em&gt;) in Yuan&#x2019;s model (5); and (D) LOD scores for linear mixed-effects model (LMM) (4).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/2/599/F2.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/2/599/F2.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/2/599/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/2/599/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488398" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-47" class="first-child">Estimate plot for case 1. <em>X</em>-axis: * indicates QTL location, and o indicates PoO location. <em>Y</em>-axis: (A) <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F2/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F2/embed/mml-math-207.gif"/></noscript></span></span> (1 ≤ <em>j</em> ≤ <em>p</em>) in our proposed model (2); (B) <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F2/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F2/embed/mml-math-208.gif"/></noscript></span></span> (1 ≤ <em>j</em> ≤ <em>p</em>) in the mixed model (3); (C) <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F2/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/F2/embed/mml-math-209.gif"/></noscript></span></span> (1 ≤ <em>j</em> ≤ <em>p</em>) in Yuan’s model (5); and (D) LOD scores for linear mixed-effects model (LMM) (4).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-48">The CC (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Threadgill and Churchill 2012</a>), a panel of newly emerged multiparent RI mouse strains, was developed, similar to other MPPs, to provide greater genetic diversity than the traditional RI populations and thereby to improve our power of understanding of complex traits. CC-RIX, F1 crosses that are generated from parental CC RI lines (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Zou <em>et al.</em> 2005</a>) can serve as excellent mouse models for mapping genes with both traditional genetic effects and epigenetic effects, such as imprinting. This study extends the model of <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Gong and Zou (2012)</a> and <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Yuan <em>et al.</em> (2011)</a> and, to the best of our knowledge, it is the first use of Bayesian variable selection methods to jointly map multiple QTL with both founder allelic effects and PoO effects for MPP-RIX data, in particular, CC-RIX data.</p><p id="p-49">In this article, it is assumed that the RI lines are equally distanced from each other in terms of genetic distance which is a sensible assumption given the funnel design used for generating CC RI lines. However, we do observe that some RI lines share more or fewer founder alleles than expected. Our limited simulation shows that such genetic unbalance has a negligible effect on mapping genes, but this issue deserves further investigation. Alternatively, as genotypes of the parental RI lines are available, we could modify the design matrices <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-210.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-211.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/2/599/embed/mml-math-212.gif"/></noscript></span></span> in model (1) and replace them with local and genome-wide similarity matrices as done in sequence kernel association tests (SKAT) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ionita-Laza <em>et al.</em> 2013</a>) and genome-wide complex trait analysis (GCTA) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Yang <em>et al.</em> 2011</a>). In addition, in our model we assume additive founder allelic effects. This assumption can be easily relaxed to allow genes with both additive and dominance effects (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Woods <em>et al.</em> 2012</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Zhang <em>et al.</em> 2014</a>). Because dominant genetic effects are orthogonal to PoO effects, missing the dominant genetic effects does not cause any bias in the PoO effects estimate.</p><p id="p-50">One of the drawbacks of Bayesian methods is their computational cost, especially for data with large numbers of samples and markers. Our model employs a block Gibbs sampler, which dramatically reduces computational time. However, further improvements may be possible. For example, instead of modeling each marker separately, we could jointly model multiple nearby markers and reduce the magnitude of <em>p</em>. Similar ideas have been proposed for genome-wide association study data (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wu <em>et al.</em> 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lu <em>et al.</em> 2015</a>). This approach is also biologically meaningful, since for some complex traits multiple causal SNPs may be located in a single region, and our simulations (cases 1*–3*) have shown that the proposed method is powerful in mapping regions with multiple causal SNPs. However, grouping nearby markers or SNPs may offer further help in improving mapping power, which deserves more thorough investigation.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We thank two anonymous referees for their valuable comments on our manuscript. This research was partially supported by NIH grant R01 GM105785, and by the scholarship from China Scholarship Council (CSC) under the Grant CSC No.201506270042.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-52">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300483/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300483/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-53"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-11-07"><span class="received-label">Received </span>November 7, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-12-03"><span class="accepted-label">Accepted </span>December 3, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Liu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.1" data-doi="10.1146/annurev-genet-110410-132459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic imprinting: a mammalian epigenetic discovery model.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DBarlow%26rft.auinit1%253DD.%2BP.%26rft.volume%253D45%26rft.spage%253D379%26rft.epage%253D403%26rft.atitle%253DGenomic%2Bimprinting%253A%2Ba%2Bmammalian%2Bepigenetic%2Bdiscovery%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-110410-132459%26rft_id%253Dinfo%253Apmid%252F21942369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-110410-132459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21942369&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299299600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-8.2.599.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-source"><em>QTL Cartographer: A Reference Manual and Tutorial for QTL Mapping</em>.</span> <span class="cit-publ-name">Department of Statistics, North Carolina State University</span>, <span class="cit-publ-loc">Raleigh, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Review of statistical methods for QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Lab Anim. (NY)</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLab%2Banimal%26rft.stitle%253DLab%2BAnim%2B%2528NY%2529%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D30%26rft.issue%253D7%26rft.spage%253D44%26rft.epage%253D52%26rft.atitle%253DReview%2Bof%2Bstatistical%2Bmethods%2Bfor%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses.%26rft_id%253Dinfo%253Apmid%252F11469113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11469113&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.4" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.5" data-doi="10.1016/j.pbi.2008.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBeyer%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D215%26rft.epage%253D221%26rft.atitle%253DFrom%2Bmutations%2Bto%2BMAGIC%253A%2Bresources%2Bfor%2Bgene%2Bdiscovery%252C%2Bvalidation%2Band%2Bdelivery%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2008.01.002%26rft_id%253Dinfo%253Apmid%252F18295532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2008.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18295532&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255623300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.6" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the collaborative cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.7" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Complex Trait Consortium</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The collaborative cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.8" data-doi="10.1534/genetics.113.156901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crowley</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenarcic</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of adverse reactions to haloperidol in a mouse diallel: a drug–placebo experiment and Bayesian causal analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156901%26rft_id%253Dinfo%253Apmid%252F24240528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMzIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.9" data-doi="10.1534/genetics.117.203265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Back to the future: multiparent populations provide the key to unlocking the genetic basis of complex traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">527</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.203265%26rft_id%253Dinfo%253Apmid%252F28592493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.10" data-doi="10.1016/j.patrec.2005.10.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">An introduction to ROC analysis.</span> <abbr class="cit-jnl-abbrev">Pattern Recognit. Lett.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPattern%2BRecognit.%2BLett.%26rft.volume%253D27%26rft.spage%253D861%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.patrec.2005.10.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.patrec.2005.10.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237462800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.11" data-doi="10.2307/2290777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Variable selection via Gibbs sampling.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D88%26rft.spage%253D881%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2290777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2290777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LT93200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.12" data-doi="10.1534/genetics.111.132522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Varying coefficient models for mapping quantitative trait loci using recombinant inbred intercrosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132522%26rft_id%253Dinfo%253Apmid%252F22345613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Märtens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zackrisson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Powerful decomposition of complex traits in a diploid model.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">13311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D13311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.14" data-doi="10.1007/s00122-015-2506-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">MAGIC populations in crops: current status and future prospects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">999</span>–<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D999%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2506-0%26rft_id%253Dinfo%253Apmid%252F25855139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2506-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25855139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.15" data-doi="10.1016/j.ajhg.2013.04.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ionita-Laza</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarov</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buxbaum</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequence kernel association tests for the combined effect of rare and common variants.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D92%26rft.spage%253D841%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.04.015%26rft_id%253Dinfo%253Apmid%252F23684009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.04.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23684009&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.16" data-doi="10.1007/s00335-008-9113-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The collaborative cross, developing a resource for mammalian systems genetics: a status report of the Wellcome trust cohort.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DIraqi%26rft.auinit1%253DF.%2BA.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D379%26rft.epage%253D381%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Bdeveloping%2Ba%2Bresource%2Bfor%2Bmammalian%2Bsystems%2Bgenetics%253A%2Ba%2Bstatus%2Breport%2Bof%2Bthe%2BWellcome%2BTrust%2Bcohort.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9113-1%26rft_id%253Dinfo%253Apmid%252F18521666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9113-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18521666&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259514800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">High resolution of quantitative traits into multiple loci via interval mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1447</span>–<span class="cit-lpage">1455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJansen%26rft.auinit1%253DR.%2BC.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1447%26rft.epage%253D1455%26rft.atitle%253DHigh%2Bresolution%2Bof%2Bquantitative%2Btraits%2Binto%2Bmultiple%2Bloci%2Bvia%2Binterval%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F8013917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0NDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.18" data-doi="10.2307/2533965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">General formulas for obtaining the MLEs and the asymptotic variance-covariance matrix in mapping quantitative trait loci when using the EM algorithm.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DKao%26rft.auinit1%253DC.%2BH.%26rft.volume%253D53%26rft.issue%253D2%26rft.spage%253D653%26rft.epage%253D665%26rft.atitle%253DGeneral%2Bformulas%2Bfor%2Bobtaining%2Bthe%2BMLEs%2Band%2Bthe%2Basymptotic%2Bvariance-covariance%2Bmatrix%2Bin%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bwhen%2Busing%2Bthe%2BEM%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533965%26rft_id%253Dinfo%253Apmid%252F9192455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533965&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9192455&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teasdale</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multiple interval mapping for quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1203</span>–<span class="cit-lpage">1216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKao%26rft.auinit1%253DC.-H.%26rft.volume%253D152%26rft.issue%253D3%26rft.spage%253D1203%26rft.epage%253D1216%26rft.atitle%253DMultiple%2BInterval%2BMapping%2Bfor%2BQuantitative%2BTrait%2BLoci%26rft_id%253Dinfo%253Apmid%252F10388834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi8zLzEyMDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.20" data-doi="10.1101/gr.134031.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkes</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoofer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic dissection of a model complex trait using the <em>Drosophila</em> synthetic population resource.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134031.111%26rft_id%253Dinfo%253Apmid%252F22496517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU1OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMi81OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.21" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-other" id="cit-8.2.599.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallick</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Variable selection for regression models.</span> <span class="cit-source">Sankhya Ser. B</span> <span class="cit-vol">60</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSankhya%2BSer.%2BB%26rft.volume%253D60%26rft.spage%253D65%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ladejobi</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elderfield</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaynor</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Maximizing the potential of multi-parental crop populations.</span> <abbr class="cit-jnl-abbrev">Appl. Transl. Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BTransl.%2BGenomics%26rft.volume%253D11%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.25" data-doi="10.1038/nrg3543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic imprinting and parent-of-origin effects on complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D609%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3543%26rft_id%253Dinfo%253Apmid%252F23917626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23917626&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.26" data-doi="10.1534/genetics.111.132563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenarcic</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A general Bayesian approach to analyzing diallel crosses of inbred strains.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132563%26rft_id%253Dinfo%253Apmid%252F22345610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knickmeyer</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephanie</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multiple SNP-sets analysis for genome-wide association studies through Bayesian latent variable selection.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">664</span>–<span class="cit-lpage">677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D39%26rft.spage%253D664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.28" data-doi="10.1007/s003359900997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manly</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Overview of QTL mapping software and introduction to map manager QT.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DManly%26rft.auinit1%253DK.%2BF.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D327%26rft.epage%253D334%26rft.atitle%253DOverview%2Bof%2BQTL%2Bmapping%2Bsoftware%2Band%2Bintroduction%2Bto%2Bmap%2Bmanager%2BQT.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs003359900997%26rft_id%253Dinfo%253Apmid%252F10087288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s003359900997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10087288&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079321300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.29" data-doi="10.2307/2290129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beauchamp</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Bayesian variable selection in linear regression.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">1023</span>–<span class="cit-lpage">1032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D83%26rft.spage%253D1023%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2290129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2290129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.30" data-doi="10.1016/j.cell.2013.11.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaisaki</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The architecture of parent-of-origin effects in mice.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">332</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D156%26rft.spage%253D332%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.11.043%26rft_id%253Dinfo%253Apmid%252F24439386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.11.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24439386&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329912200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.31" data-doi="10.1126/science.1259595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259595%26rft_id%253Dinfo%253Apmid%252F25359852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDYvNjIxMi85ODciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.32" data-doi="10.1534/genetics.105.040469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">ter Braak</span>  <span class="cit-name-given-names">C. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Extending Xu’s Bayesian model for estimating polygenic effects using markers of the entire genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1435</span>–<span class="cit-lpage">1438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.040469%26rft_id%253Dinfo%253Apmid%252F15911578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzE0MzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.33" data-doi="10.1534/genetics.111.138032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ten years of the collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.138032%26rft_id%253Dinfo%253Apmid%252F22345604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Villena</span>  <span class="cit-name-given-names">F. P.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The collaborative cross: a recombinant inbred mouse population for the systems genetic era.</span> <abbr class="cit-jnl-abbrev">ILAR J.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DILAR%2BJournal%26rft.stitle%253DILAR%2BJ%26rft.aulast%253DThreadgill%26rft.auinit1%253DD.%2BW.%26rft.volume%253D52%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D31%26rft.atitle%253DThe%2Bcollaborative%2Bcross%253A%2Ba%2Brecombinant%2Binbred%2Bmouse%2Bpopulation%2Bfor%2Bthe%2Bsystems%2Bgenetic%2Bera.%26rft_id%253Dinfo%253Apmid%252F21411855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21411855&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291388900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.35" data-doi="10.1007/s00335-004-2466-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsaih</span>  <span class="cit-name-given-names">S.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait mapping in a diallel cross of recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DTsaih%26rft.auinit1%253DS.%2BW.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D344%26rft.epage%253D355%26rft.atitle%253DQuantitative%2Btrait%2Bmapping%2Bin%2Ba%2Bdiallel%2Bcross%2Bof%2Brecombinant%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-004-2466-1%26rft_id%253Dinfo%253Apmid%252F16104382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-004-2466-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16104382&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229640800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.36" data-doi="10.1186/1471-2164-15-865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vered</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iraqi</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Susceptibility to <em>Klebsiella pneumonaie</em> infection in collaborative cross mice is a complex trait controlled by at least three loci acting at different time points.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D865%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-865%26rft_id%253Dinfo%253Apmid%252F25283706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25283706&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.37" data-doi="10.1534/genetics.115.179945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A random-model approach to QTL mapping in multiparent advanced generation intercross (MAGIC) populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179945%26rft_id%253Dinfo%253Apmid%252F26715662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNDcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.38" data-doi="10.1152/physiolgenomics.00040.2012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">L. C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holl</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oreper</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsaih</span>  <span class="cit-name-given-names">S.-W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fine-mapping diabetes-related traits, including insulin resistance, in heterogeneous stock rats.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00040.2012%26rft_id%253Dinfo%253Apmid%252F22947656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00040.2012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22947656&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310699800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.39" data-doi="10.1016/j.ajhg.2010.05.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraft</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chanock</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Powerful SNP-set analysis for case-control genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DM.%2BC.%26rft.volume%253D86%26rft.issue%253D6%26rft.spage%253D929%26rft.epage%253D942%26rft.atitle%253DPowerful%2BSNP-set%2Banalysis%2Bfor%2Bcase-control%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.05.002%26rft_id%253Dinfo%253Apmid%252F20560208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20560208&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278948900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.40" data-doi="10.1016/j.ajhg.2010.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GCTA: a tool for genome-wide complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D82%26rft.atitle%253DGCTA%253A%2Ba%2Btool%2Bfor%2Bgenome-wide%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.11.011%26rft_id%253Dinfo%253Apmid%252F21167468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21167468&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F2%2F599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.41" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.42" data-doi="10.1534/genetics.110.125542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bayesian multiple quantitative trait loci mapping for recombinant inbred intercrosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125542%26rft_id%253Dinfo%253Apmid%252F21385723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzEvMTg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Precision mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1457%26rft.epage%253D1468%26rft.atitle%253DPrecision%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.44" data-doi="10.1534/genetics.114.166249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bayesian modeling of haplotype effects in multiparent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166249%26rft_id%253Dinfo%253Apmid%252F25236455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8yLzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.2.599.45" data-doi="10.1534/genetics.104.035709"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelfond</span>  <span class="cit-name-given-names">J. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manly</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait locus analysis using recombinant inbred intercrosses: theoretical and empirical considerations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1299</span>–<span class="cit-lpage">1311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035709%26rft_id%253Dinfo%253Apmid%252F15879512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzEyOTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzIvNTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/2/599.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/2/587" title="Dosage-Dependent Expression Variation Suppressed on the Drosophila Male X Chromosome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/2/607" title="Transcriptional Profiling of Saccharomyces cerevisiae Reveals the Impact of Variation of a Single Transcription Factor on Differential Gene Expression in 4NQO, Fermentable, and Nonfermentable Carbon Sources" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/2"><span>Volume 8 Issue 2, February 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/2.cover-source.jpg?itok=sUOF_BxC" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/2/599" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uvdtjrGvjYMARtkm4737-gaOpk45Pp6PLeY_kmY2Cko" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/2/599" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/2/599" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/2/599" class="active" data-icon-position="" data-hide-link-title="0">Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/488303" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-k4YKp3eh8VrhvN7C1GR0EHg08tl06nJfDeAf33Rq5Is" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/599.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/2/599" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FWi3k2Xn-xfAA3mRiWT4CCXMI9IpAMSPUHTkdvZK1Vw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/2/599" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/2/599.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488303" id="node488303--2" data-pisa="ggg;8/2/599" data-pisa-master="ggg;g3.117.300483" data-apath="/ggg/8/2/599.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yanyan Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8304-5663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sican Xiong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6350-1107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Sun</span> and <span class="highwire-citation-author" data-delta="3">Fei Zou</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">599-605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300483 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488303--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yanyan Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Mathematics and Statistics, Wuhan University, Hubei 430072, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanyan%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYanyan%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sican Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Mathematics and Statistics, Wuhan University, Hubei 430072, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sican%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASican%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8304-5663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sican Xiong</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6350-1107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Sun</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fei Zou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BZou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zou%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFei%2BZou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:feizou@email.unc.edu" class="" data-icon-position="" data-hide-link-title="0">feizou@email.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/2/599" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/2/599&amp;title=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/2/599&amp;title=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/2/599&amp;title=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/2/599&amp;text=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/2/599&amp;title=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/2/599&amp;t=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/2/599&amp;title=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/2/599&amp;title=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/488303/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="488303" id="node488303--3" data-pisa="ggg;8/2/599" data-pisa-master="ggg;g3.117.300483" data-apath="/ggg/8/2/599.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Joint Analysis of Strain and Parent-of-Origin Effects for Recombinant Inbred Intercrosses Generated from Multiparent Populations with the Collaborative Cross as an Example</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yanyan Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8304-5663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sican Xiong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6350-1107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Sun</span> and <span class="highwire-citation-author" data-delta="3">Fei Zou</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">599-605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300483 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node488303--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yanyan Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Mathematics and Statistics, Wuhan University, Hubei 430072, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanyan%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYanyan%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sican Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Mathematics and Statistics, Wuhan University, Hubei 430072, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sican%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASican%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8304-5663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sican Xiong</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6350-1107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Sun</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fei Zou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, University of North Carolina at Chapel Hill, North Carolina 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BZou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zou%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFei%2BZou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:feizou@email.unc.edu" class="" data-icon-position="" data-hide-link-title="0">feizou@email.unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=599&amp;rft.epage=605&amp;rft.atitle=Joint%20Analysis%20of%20Strain%20and%20Parent-of-Origin%20Effects%20for%20Recombinant%20Inbred%20Intercrosses%20Generated%20from%20Multiparent%20Populations%20with%20the%20Collaborative%20Cross%20as%20an%20Example&amp;rft.volume=8&amp;rft.issue=2&amp;rft.date=2018-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Liu%2C+Yanyan&amp;rft.au=Xiong%2C+Sican&amp;rft.au=Sun%2C+Wei&amp;rft.au=Zou%2C+Fei"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/488303/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/488303/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/488303/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/488303/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/488303/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/488303/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/488303/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/488303/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/488303/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/488303/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/488303/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/488303/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496049" id="node496049" data-pisa="ggg;g3.119.400658v1" data-pisa-master="ggg;g3.119.400658" data-apath="/ggg/early/2020/03/26/g3.119.400658.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.119.400658" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Dissecting Adaptive Traits with Nested Association Mapping: Genetic Architecture of Inflorescence Morphology in Sorghum</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/2/599" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Statistical Method</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Simulation Study</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/2/599.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/2/599.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/2/599.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/2/599.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
