<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/6/417" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/6/417.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/6/417.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/6/417.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/6/417.atom" />
<meta name="HW.pisa" content="ggg;1/6/417" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase Escherichia coli Cells" />
<meta name="DC.Identifier" content="10.1534/g3.111.001057" />
<meta name="DC.Date" content="2011-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Williams et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="As nutrients are depleted and cell division ceases in batch cultures of bacteria, active processes are required to ensure that each cell has a complete copy of its genome. How chromosome number is manipulated and maintained in nondividing bacterial cells is not fully understood. Using flow cytometric analysis of cells from different growth phases, we show that the Holliday junction–processing enzymes RuvABC and RecG, as well as RecBCD, the enzyme complex that initiates DNA double-strand break repair, are required to establish the normal distribution of fluorescent peaks, which is commonly accepted to reflect the distribution of chromosome numbers. Our results reveal that these proteins are required for the proper processing of chromosomes in stationary phase." />
<meta name="DC.Contributor" content="Ashley B. Williams" />
<meta name="DC.Contributor" content="Kyle M. Hetrick" />
<meta name="DC.Contributor" content="Patricia L. Foster" />
<meta name="article:published_time" content="2011-11-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase Escherichia coli Cells" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;As nutrients are depleted and cell division ceases in batch cultures of bacteria, active processes are required to ensure that each cell has a complete copy of its genome. How chromosome number is manipulated and maintained in nondividing bacterial cells is not fully understood. Using flow cytometric analysis of cells from different growth phases, we show that the Holliday junction–processing enzymes RuvABC and RecG, as well as RecBCD, the enzyme complex that initiates DNA double-strand break repair, are required to establish the normal distribution of fluorescent peaks, which is commonly accepted to reflect the distribution of chromosome numbers. Our results reveal that these proteins are required for the proper processing of chromosomes in stationary phase.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/11/01" />
<meta name="citation_mjid" content="ggg;1/6/417" />
<meta name="citation_id" content="1/6/417" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/6/417" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/6/417.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/6/417.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/6/417.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001057" />
<meta name="citation_pmid" content="22384352" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="417" />
<meta name="citation_lastpage" content="426" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ashley B. Williams" />
<meta name="citation_author_institution" content="Molecular and Computational Biology, University of Southern California, Los Angeles, California 90089" />
<meta name="citation_author" content="Kyle M. Hetrick" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Patricia L. Foster" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_email" content="plfoster@indiana.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=T. Åkerlund;citation_author=K. Nordstrom;citation_author=R. Bernander;citation_title=Analysis of cell size and DNA content in exponentially growing and stationary-phase batch cultures of Escherichia coli.;citation_pages=6791-6797;citation_volume=177;citation_year=1995;citation_issue=23;citation_pmid=7592469;citation_doi=10.1128/jb.177.23.6791-6797.1995" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=L. Boe;citation_title=Mechanism for induction of adaptive mutations in Escherichia coli.;citation_pages=597-601;citation_volume=4;citation_year=1990;citation_issue=4;citation_pmid=2191182;citation_doi=10.1111/j.1365-2958.1990.tb00628.x" />
<meta name="citation_reference" content="citation_journal_title=Research in Microbiology (Paris);citation_journal_abbrev=Research in Microbiology (Paris);citation_author=E. Boye;citation_author=A. Løbner-Olesen;citation_title=Bacterial growth control studied by flow cytometry.;citation_pages=131-135;citation_volume=142;citation_year=1991;citation_issue=2-3;citation_pmid=1925010;citation_doi=10.1016/0923-2508(91)90020-B" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Cairns;citation_author=P. L. Foster;citation_title=Adaptive reversion of a frameshift mutation in Escherichia coli.;citation_pages=695-701;citation_volume=128;citation_year=1991;citation_pmid=1916241" />
<meta name="citation_reference" content="citation_journal_title=Biochimie;citation_author=K. L. Chua;citation_author=Y. K. Mak;citation_author=P. Oliver;citation_title=Expression of the recA gene in recombination-deficient (rec-) strains of Escherichia coli.;citation_pages=775-783;citation_volume=75;citation_year=1993;citation_pmid=8274529;citation_doi=10.1016/0300-9084(93)90127-E" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=Y. Cui;citation_author=Z. M. Petrushenko;citation_author=V. V. Rybenkov;citation_title=MukB acts as a macromolecular clamp in DNA condensation.;citation_pages=411-418;citation_volume=15;citation_year=2008;citation_issue=4;citation_pmid=18376412;citation_doi=10.1038/nsmb.1410" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=M. S. Dillingham;citation_author=S. C. Kowalczykowski;citation_title=RecBCD Enzyme and the Repair of Double-Stranded DNA Breaks;citation_pages=642-671;citation_volume=72;citation_year=2008;citation_issue=4;citation_pmid=19052323;citation_doi=10.1128/MMBR.00020-08" />
<meta name="citation_reference" content="citation_journal_title=Advances in applied microbiology;citation_journal_abbrev=Adv Appl Microbiol;citation_author=C. J. Dorman;citation_title=Nucleoid-associated proteins and bacterial physiology.;citation_pages=47-64;citation_volume=67;citation_year=2009;citation_pmid=19245936;citation_doi=10.1016/S0065-2164(08)01002-2" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=D. J. Ferullo;citation_author=D. L. Cooper;citation_author=H. R. Moore;citation_author=S. T. Lovett;citation_title=Cell cycle synchronization of Escherichia coli using the stringent response, with fluorescence labeling assays for DNA content and replication.;citation_pages=8-13;citation_volume=48;citation_year=2009;citation_issue=1;citation_pmid=19245839;citation_doi=10.1016/j.ymeth.2009.02.010" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=P. L. Foster;citation_title=Adaptive Mutation in Escherichia coli;citation_pages=21-30;citation_volume=65;citation_year=2000;citation_issue=0;citation_pmid=12760017;citation_doi=10.1101/sqb.2000.65.21" />
<meta name="citation_reference" content="citation_journal_title=Critical Reviews in Biochemistry and Molecular Biology;citation_journal_abbrev=CRIT REV BIOCHEM MOL BIOL;citation_author=P. L. Foster;citation_title=Stress-induced mutagenesis in bacteria.;citation_pages=373-397;citation_volume=42;citation_year=2007;citation_issue=5;citation_pmid=17917873;citation_doi=10.1080/10409230701648494" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. L. Foster;citation_author=J. M. Trimarchi;citation_author=R. A. Maurer;citation_title=Two enzymes, both of which process recombination intermediates, have opposite effects on adaptive mutation in Escherichia coli.;citation_pages=25-37;citation_volume=142;citation_year=1996;citation_pmid=8770582" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=D. Frenkiel-Krispin;citation_author=I. Ben-Avraham;citation_author=J. Englander;citation_author=E. Shimoni;citation_author=S. G. Wolf;citation_title=Nucleoid restructuring in stationary-state bacteria.;citation_pages=395-405;citation_volume=51;citation_year=2004;citation_issue=2;citation_pmid=14756781;citation_doi=10.1046/j.1365-2958.2003.03855.x" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=I. Grainge;citation_author=M. Bregu;citation_author=M. Vazquez;citation_author=V. Sivanathan;citation_author=S. C. Ip;citation_title=Unlinking chromosome catenanes in vivo by site-specific recombination.;citation_pages=4228-4238;citation_volume=26;citation_year=2007;citation_issue=19;citation_pmid=17805344;citation_doi=10.1038/sj.emboj.7601849" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. R. Grivell;citation_author=M. B. Grivell;citation_author=P. C. Hanawalt;citation_title=Turnover in bacterial DNA containing thymine or 5-bromouracil.;citation_pages=219-233;citation_volume=98;citation_year=1975;citation_issue=1;citation_pmid=1104866;citation_doi=10.1016/S0022-2836(75)80110-0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. S. Harris;citation_author=S. Longerich;citation_author=S. M. Rosenberg;citation_title=Recombination in adaptive mutation;citation_pages=258-260;citation_volume=264;citation_year=1994;citation_issue=5156;citation_pmid=8146657;citation_doi=10.1126/science.8146657" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. S. Harris;citation_author=K. J. Ross;citation_author=S. M. Rosenberg;citation_title=Opposing roles of the Holliday junction processing systems of Escherichia coli in recombination-dependent adaptive mutation.;citation_pages=681-691;citation_volume=142;citation_year=1996;citation_pmid=8849879" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=K. Ishioka;citation_author=A. Fukuoh;citation_author=H. Iwasaki;citation_author=A. Nakata;citation_author=H. Shinagawa;citation_title=Abortive recombination in Escherichia coli ruv mutants blocks chromosome partitioning;citation_pages=209-220;citation_volume=3;citation_year=1998;citation_issue=4;citation_pmid=9663656;citation_doi=10.1046/j.1365-2443.1998.00185.x" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; genetic systems;citation_journal_abbrev=Genes Genet Syst;citation_author=K. Ishioka;citation_author=H. Iwasaki;citation_author=H. Shinagawa;citation_title=Roles of the recG gene product of Escherichia coli in recombination repair: effects of the delta recG mutation on cell division and chromosome partition.;citation_pages=91-99;citation_volume=72;citation_year=1997;citation_issue=2;citation_pmid=9265736;citation_doi=10.1266/ggs.72.91" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=I. Ivančić-Baće;citation_author=I. Vlašić;citation_author=E. Salaj-Šmic;citation_author=K. Brčić-Kostić;citation_title=Genetic Evidence for the Requirement of RecA Loading Activity in SOS Induction after UV Irradiation in Escherichia coli;citation_pages=5024-5032;citation_volume=188;citation_year=2006;citation_issue=14;citation_pmid=16816175;citation_doi=10.1128/JB.00130-06" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=J. C. Layton;citation_author=P. L. Foster;citation_title=Error-prone DNA polymerase IV is controlled by the stress-response sigma factor, RpoS, in Escherichia coli.;citation_pages=549-561;citation_volume=50;citation_year=2003;citation_issue=2;citation_pmid=14617178;citation_doi=10.1046/j.1365-2958.2003.03704.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=R. G. Lloyd;citation_author=C. Buckman;citation_title=Genetic analysis of the recG locus of Escherichia coli K-12 and of its role in recombination and DNA repair.;citation_pages=1004-1011;citation_volume=173;citation_year=1991;citation_issue=3;citation_pmid=1846849;citation_doi=10.1128/jb.173.3.1004-1011.1991" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=J. D. McCool;citation_author=E. Long;citation_author=J. F. Petrosino;citation_author=H. A. Sandler;citation_author=S. M. Rosenberg;citation_title=Measurement of SOS expression in individual Escherichia coli K-12 cells using fluorescence microscopy.;citation_pages=1343-1357;citation_volume=53;citation_year=2004;citation_issue=5;citation_pmid=15387814;citation_doi=10.1111/j.1365-2958.2004.04225.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. McGlynn;citation_author=R. G. Lloyd;citation_title=RecG helicase activity at three- and four-strand DNA structures.;citation_volume=27;citation_year=1999;citation_pmid=10454599;citation_doi=10.1093/nar/27.15.3049" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=G. J. McKenzie;citation_author=P. L. Lee;citation_author=M. J. Lombardo;citation_author=P. J. Hastings;citation_author=S. M. Rosenberg;citation_title=SOS mutator DNA polymerase IV functions in adaptive mutation and not adaptive amplification.;citation_pages=571-579;citation_volume=7;citation_year=2001;citation_issue=3;citation_pmid=11463382;citation_doi=10.1016/S1097-2765(01)00204-0" />
<meta name="citation_reference" content="citation_journal_title=DNA repair;citation_journal_abbrev=DNA Repair (Amst);citation_author=B. Michel;citation_author=H. Boubakri;citation_author=Z. Baharoglu;citation_author=M. LeMasson;citation_author=R. Lestini;citation_title=Recombination proteins and rescue of arrested replication forks.;citation_pages=967-980;citation_volume=6;citation_year=2007;citation_issue=7;citation_pmid=17395553;citation_doi=10.1016/j.dnarep.2007.02.016" />
<meta name="citation_reference" content="citation_title=A Short Course in Bacterial Genetics: A Laboratory Manual and Handbook for Escherichia coli and Related Bacteria.;citation_author=J. H. Miller;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=S. Nair;citation_author=S. E. Finkel;citation_title=Dps Protects Cells against Multiple Stresses during Stationary Phase;citation_pages=4192-4198;citation_volume=186;citation_year=2004;citation_issue=13;citation_pmid=15205421;citation_doi=10.1128/JB.186.13.4192-4198.2004" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=C. L. Patten;citation_author=M. G. Kirchhof;citation_author=M. R. Schertzberg;citation_author=R. A. Morton;citation_author=H. E. Schellhorn;citation_title=Microarray analysis of RpoS-mediated gene expression in Escherichia coli K-12.;citation_pages=580-591;citation_volume=272;citation_year=2004;citation_issue=5;citation_pmid=15558318;citation_doi=10.1007/s00438-004-1089-2" />
<meta name="citation_reference" content="citation_journal_title=Biotechnology and Bioengineering;citation_journal_abbrev=Biotechnology and Bioengineering;citation_author=M. Rahman;citation_author=M. R. Hasan;citation_author=T. Oba;citation_author=K. Shimizu;citation_title=Effect of rpoS gene knockout on the metabolism of Escherichia coli during exponential growth phase and early stationary phase based on gene expressions, enzyme activities and intracellular metabolite concentrations.;citation_pages=585-595;citation_volume=94;citation_year=2006;citation_issue=3;citation_pmid=16511888;citation_doi=10.1002/bit.20858" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=C. J. Rudolph;citation_author=A. L. Upton;citation_author=L. Harris;citation_author=R. G. Lloyd;citation_title=Pathological replication in cells lacking RecG DNA translocase.;citation_pages=352-366;citation_volume=73;citation_year=2009a;citation_issue=3;citation_pmid=19538444;citation_doi=10.1111/j.1365-2958.2009.06773.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=C. J. Rudolph;citation_author=A. L. Upton;citation_author=R. G. Lloyd;citation_title=Replication fork collisions cause pathological chromosomal amplification in cells lacking RecG DNA translocase.;citation_pages=940-955;citation_volume=74;citation_year=2009b;citation_issue=4;citation_pmid=19818016;citation_doi=10.1111/j.1365-2958.2009.06909.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Salic;citation_author=T. J. Mitchison;citation_title=A chemical method for fast and sensitive detection of DNA synthesis in vivo.;citation_volume=105;citation_year=2008;citation_pmid=18272492;citation_doi=10.1073/pnas.0712168105" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Sarkar;citation_author=I. Vitoc;citation_author=I. Mukerji;citation_author=N. V. Hud;citation_title=Bacterial protein HU dictates the morphology of DNA condensates produced by crowding agents and polyamines.;citation_volume=35;citation_year=2007;citation_pmid=17259223;citation_doi=10.1093/nar/gkl1093" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=K. A. Storvik;citation_author=P. L. Foster;citation_title=RpoS, the stress response sigma factor, plays a dual role in the regulation of Escherichia coli's error-prone DNA polymerase IV.;citation_volume=192;citation_year=2010;citation_pmid=20472798;citation_doi=10.1128/JB.00358-10" />
<meta name="citation_reference" content="citation_journal_title=Photochem. Photobiol.;citation_author=M. S. Tang;citation_author=T. C. V. Wang;citation_author=M. H. Patrick;citation_title=DNA turnover in buffer-held Escherichia coli and its effect on repair of UV damage.;citation_pages=511-520;citation_volume=79;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. D. Tompkins;citation_author=J. E. Nelson;citation_author=J. C. Hazel;citation_author=S. L. Leugers;citation_author=J. D. Stumpf;citation_title=Error-Prone Polymerase, DNA Polymerase IV, Is Responsible for Transient Hypermutation during Adaptive Mutation in Escherichia coli;citation_pages=3469-3472;citation_volume=185;citation_year=2003;citation_issue=11;citation_pmid=12754247;citation_doi=10.1128/JB.185.11.3469-3472.2003" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=M. Villarroya;citation_author=I. Pérez-Roger;citation_author=F. Macián;citation_author=M. E. Armengod;citation_title=Stationary phase induction of dnaN and recF, two genes of Escherichia coli involved in DNA replication and repair;citation_pages=1829-1837;citation_volume=17;citation_year=1998;citation_issue=6;citation_pmid=9501104;citation_doi=10.1093/emboj/17.6.1829" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. C. West;citation_title=PROCESSING OF RECOMBINATION INTERMEDIATES BY THE RUVABC PROTEINS;citation_pages=213-244;citation_volume=31;citation_year=1997;citation_issue=1;citation_pmid=9442895;citation_doi=10.1146/annurev.genet.31.1.213" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. B. Williams;citation_author=P. L. Foster;citation_title=The Escherichia coli histone-like protein HU has a role in stationary phase adaptive mutation.;citation_pages=723-735;citation_volume=177;citation_year=2007;citation_pmid=17720921;citation_doi=10.1534/genetics.107.075861" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=E. M. Witkin;citation_title=Ultraviolet mutagenesis and inducible DNA repair in Escherichia coli.;citation_pages=869-907;citation_volume=40;citation_year=1976;citation_issue=4;citation_pmid=795416" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Yeiser;citation_author=E. D. Pepper;citation_author=M. F. Goodman;citation_author=S. E. Finkel;citation_title=SOS-induced DNA polymerases enhance long-term survival and evolutionary fitness.;citation_volume=99;citation_year=2002;citation_pmid=12060704;citation_doi=10.1073/pnas.092269199" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase Escherichia coli Cells" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="twitter:description" content="As nutrients are depleted and cell division ceases in batch cultures of bacteria, active processes are required to ensure that each cell has a complete copy of its genome. How chromosome number is manipulated and maintained in nondividing bacterial cells is not fully understood. Using flow cytometric analysis of cells from different growth phases, we show that the Holliday junction–processing enzymes RuvABC and RecG, as well as RecBCD, the enzyme complex that initiates DNA double-strand break repair, are required to establish the normal distribution of fluorescent peaks, which is commonly accepted to reflect the distribution of chromosome numbers. Our results reveal that these proteins are required for the proper processing of chromosomes in stationary phase." />
<meta name="og-title" property="og:title" content="Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase Escherichia coli Cells" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/6/417" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="As nutrients are depleted and cell division ceases in batch cultures of bacteria, active processes are required to ensure that each cell has a complete copy of its genome. How chromosome number is manipulated and maintained in nondividing bacterial cells is not fully understood. Using flow cytometric analysis of cells from different growth phases, we show that the Holliday junction–processing enzymes RuvABC and RecG, as well as RecBCD, the enzyme complex that initiates DNA double-strand break repair, are required to establish the normal distribution of fluorescent peaks, which is commonly accepted to reflect the distribution of chromosome numbers. Our results reveal that these proteins are required for the proper processing of chromosomes in stationary phase." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
    <title>Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase Escherichia coli Cells | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"36Tkm0RCb_J_TfmFfq-FQdOfKjuAMbXk1fm0QjzBkUk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459192","apath":"\/ggg\/1\/6\/417.atom","pisa":"ggg;1\/6\/417","ac":{"\/ggg\/1\/6\/417.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/6\/417.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/6\/417"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/6\/417"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/6\/417":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bD6mLV0rBowczQvXxdWwgDmiv242ExdWIsW0uUhsjk\\\\\\\/1fuQ8KKLxPGeC2U9hfqYboUpp+dn3B4Y+PZLaOFDuKRt5sbySWs4CGpA7BRPEAHEBYxpJj9pONpwou\\\\\\\/FalMkz16Syfx7V7a5OT2b+M7JEpwE5NRg3sKZKnKGwML259k4cJjTWosvzXV1i84IKCsG8eDxwRqL8FnSEvQAJ3BB5SUYcZi3BCxfx8An8kBSORs\\\\\\\/Hc6rcnCrn\\\\\\\/dszQxXBGOUUqN3A\\\\\\\/\\\\\\\/g6GSXIu\\\\\\\/2308mz2yZnji3rjTdNiH8xqjY3NPsx8sZsMk5MTcHrn3\\\\\\\/Z6DoH9xY9YsIU8bsTxR4suKNophbKltGYB8w+oESuHBd\\\\\\\/tVd9XFEF5BQK3ViTX3j0MqQq0Wwbzf0eD+NQL\\\\\\\/6n07GIbNSWTIyoMALaW8TZUrKgRMfp\\\\\\\/9y3VHRUfjuLAB41qFFiaUPGEh4ztXJTUl5XolvfPKCOhdD1Ftb3Hfqs+ySVnj9ThW8V+ILvuu9AQvbCG1pSCjBU+z5Wl2O7v59ys2MLjviP3Swxoo18hmHi3Bqhdk0MAahAeE3qc+s1Hs\\\\\\\/Bf0kQIvXdyWStF5OwVvVtuMtRGQasCv8bAU2T9w9HcuHR4cj5ygV2Wb0X\\\\\\\/t26uOhy\\\\\\\/hiJEG8QKnBDKMJCe9R1hMMMYHNHsqK93JnzDpp4JosiRABqCL\\\\\\\/1KcHZh\\\\\\\/+wMdwT2sBZ3WGR1vdDqgeGPTLE39RiexCiFwkp83pM+Wiq+ijymyKV9kGVG\\\\\\\/bx+a7chv3lc\\\\\\\/LPF9arrps29dk+lRLEBplApA2u0YMmrnn\\\\\\\/3pY+sShgq6CttMGtqAXXNUz2vHdhQqq7ra\\\\\\\/Sqdy1efDEiVLhs1WhweFG3f4NZtTDaeamEdXFks+ZzrCPYNzIuKFjGRj2hS4L0FgnkSeKVIETe7wbLx85McmYptVG01qN8sdcqkXqhaxQpIpVTQIfpJWo2sDJl4Kj\\\\\\\/qnkgbbbKYDZzm+ej0OybfWe23kVU815AfG7k2b\\\\\\\/XDztGig05I8i7Ex\\\\\\\/wA5p+D5BqB2zC6xCgKQW+9GfiXzS4IgQaUy6Q8133n5\\\\\\\/oFyOZHQcKiuWjJfjdm\\\\\\\/QYB9az1w1+hMWaZC4kN\\\\\\\/Fant+O7Hg9cHtnmQ6UX1DkqLJjG3NgwaGbE1yVbYvTN9E1z7\\\\\\\/i+572Buu\\\\\\\/ITFTouKX1MTUFUVb\\\\\\\/2hedzYAHBU0w\\\\\\\/g1MZnOvhox+cYKKzX+M4\\\\\\\/1PtddczCta2weca9DBdLD3zKj0VxYwVeNRy4EoUnadI7FEdyEp0Yd89aYv6r3diBWTpNY2PO\\\u0022,\\\u0022iv\\\u0022:\\\u0022tuksH4heAORkN9wczeTEkg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221133d939e6a567e8183fdde2c627de8e7b84ea0b48269d01acd65527bfcfa087\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hTN0UtuzcHYlRvS67LcJq6j8KV67hNqZUC7mBVv7Buz\\\\\\\/ocMFSD+GCaO2W05Aw\\\\\\\/lcVx6LscSsRE5DkfRr6I9x5mPPhy5e0Iw0+QsFvv2Ztb15bsQt+JEBQ52J6WOOJIGmj0rZ\\\\\\\/fBtrI0PK+rvrhWgc\\\\\\\/O5FsdDU2g1KAzWHjeqVtCLD4Ly+4qh3GTccOpzXH0GA4ifGx73zDloM9HyGNt4QYPRKhkG7+RQ5IqiCHga0sh68y9yQRpxYEaBUXnpvZv5rv1juon\\\\\\\/Y7h6WUzQZRNZoQUwHLznNxTFz1+d\\\\\\\/vzeelBlOoxp8TCdvqXpRTPKfAblp8Z2ADrn61tSlUSmK0jxa5ritTXsByv9PuktUP4Ddy+cerfx7XHJ9oqILqsyaLu7LxEtSxMKVOZ\\\\\\\/dsc6ys5D+mRPCB\\\\\\\/olxpqtFx4kW\\\\\\\/6aGZI53mCK+g0Ar601fU7HeQKG1U49GRH6jzqyV3xQeCfNGdNdApowG+kz2F7t0MgHSp7oMcJHWiUEG9KrqysdgdKqY9L9mSN+lyMkvkReMfKFlZdqGrrrdPTtQDGdPt3BwdmKV2ATvtEcomhOIAH1gFyM6xTzgxw9QsuOhRfbkRVv0PX\\\\\\\/q2zF1iv4UU8CRjanDwq8276cLlzx4KPJ5bmDe+dlV4BSc\\\\\\\/L3lyWnDM98JM\\\\\\\/esCX+xTlo7KeWdlToHugPmM+XPn6LTcFvUsL9yQeJYEKwIMgyRZLdy7kGeKggPjb1OZ6cDMkhIZt0JbClUQNRqbxUSJKgd9JzOFzOTJQjvA0799Dh1KY0sZYYzVNfXf6XuzDGyiz6OgAQY+5iQ5XPq4ToBGJWsRGH+9MN5ZHLvdefVAWfD8I19fyi9tts\\\\\\\/JvwcANxBcgcz9+ql1UTKgNA\\\\\\\/iMOzDPvJzjPlhU0JX\\\\\\\/BO7GHY31tlvDSPprdqjwebvPIOgLtS9RXef8MV6EHjzyq8WX+QndaxnZWqAQuFXyyQTIGnENrarFTtXFMy3kk09UHMPjDaViXgnhlCc9FFGO6ptYIplyqSg2jQwVBhtLmteyfG4H1aXmPOIFNkcPLiJqR9aApfnEeyToihhyfkT13jiiyxFbLdVrsqshrsPd1rdBB5TxRJ9mIx6OdxiU0ptJaDTxNiCakGfbgnPe+6PTWXI1jXlOXhPJgJ4WtFAzY3BZgHAalqiX3qlJG1WahlmyHSk3dcjlV0nSwO5o+Mu8ZBXWdjzBKrz9grLXJDrfBUppgOT5POVxefCPiTHJ++eo3RSHWjgn2Xrb80xmd9RHzNJh0Y46gG9j3n2stpoQyRm1VW03qnN9m\\\\\\\/t0ODKzYQJKkYx\\\\\\\/diLlLVRpgOFiyBrEPEtWCCHDQPb6GvDtlYMhnrzMBEbA\\\\\\\/Ah5Nmvpz9K8pBGzo+8qjSaHkVmfMP2avwYHPGEv9KMX8Btksu2X\\\\\\\/whyGLOHORHqgnK9VU3aRoOB0y86SOsYl\\\\\\\/sGkCQ8yVmXAF8mwJVGwYpyRauobJCJxR5o4YLwjj1Tb4YMrSJqLS+SAFmb0MjD4hPyCzNU8glBC7F6hxZr8HxLjzp1EBIQDiUVWx3jcgsZSsVf+D42AsrdhXkvI+Tmhqnf4obMxcQFqaKvlaGDTN+X0h8KZJUeYOc+9kWH7PfMhdWanQ8WWBe\\\\\\\/fu9hkl0LYtr+NCZN6AH0oMexoq\\\\\\\/wcMze7eg99rxAKboi+HC+Prjx1elM+Fzu5N1gkcyp\\\\\\\/OT0s5PVq51sc3kbCYurPlhWYSKCzQSWxxYoF\\\\\\\/OZ44aWeK\\\\\\\/MbFNun3wjKI5qDIJAdor7XPhMyUMqYLLLiATqzEcgGqJaJUmSGcLI7IHVVtc4KKyjDhv\\\\\\\/VYklCdFw71brXcV0fd5TR7MRzbA072E1h1hq71fwZTyTvnVvv5QtxuDk4Fo646lth4\\\\\\\/QnDpSsToV9Rfsf2SDhxggIc+qxPGyHNH5VfTpr6zvQf368xTsFipYbEpEYrCvIoOqTp\\\\\\\/eeQri1McjP3Z6xf\\\\\\\/NW4adj5vzK8C1QjevbH45tJPE5CZVDwQHOBW+y6JPmilEKk57nGiTu5AuuBz5ksChzV8OOxKUaCGWDHDQv5p2JuholN3bx04H2QC02YwPXutwVFCQgoXNwWvS5CdRB4lmv+pxFRdqDGV9FlmTDmpw798h3vY\\\\\\\/9HBwHHh3+vwlcEFCKBR8\\\\\\\/Hj0F7LK+\\\\\\\/868d33sCSwTuugaxlu4BFmDSrD\\\\\\\/CwkG22HnpOY7yXkiXJrjFU0Rx6BfrZ4hfT\\\\\\\/5+EdRkQe8Nrv0x9x1qhhA8bhNmDcE\\\\\\\/3BPenmDYhkpSQfIuRhkXAHxUh23DRVhgrexLEH5AM+zJh3k7y+A7z8UweefGuLoQIYnpzzGUUWUefmyseUJQ\\\\\\\/EjcosBGjAHSUTAO5pVyiZdK6KE\\\\\\\/RD+QjVD1+ZultiS9WotyK2m75e8t9Lhs+DwOKOcZB5kiapEIG7Yu6gP1D51j8bwmxYtAtEmpU5oN9wnRv+z5TxFAcCu1oHnzEyIIotQWnZ9SJr30nBZGfT1q153aT7m3qcMXXTeYhmIPin5tTSw5oaylFSx8RQI9DB5RWpLfD8U2FQDDoSlPq6JhpT9ZmwbXW57TCBsPnr6hZRdvh7iJAagU6lol00wd2BsyRp6wA3czZtu2NyfcC70HGdIMNC+qvqJNcQWz2ctzk79JE49j4s9hcxm\\\\\\\/AWCW+MW4ikpnMjNe7XSWrs52x7ZgfazV3ucbUU7dUpTd+z0ByoJRT3QgfU5qgaqPQEHBz8Nd2NivQCgVNe5fm9FdN510nHG7U9sGLv4JmfOUswQwl2lN19uzUWimSi8yBhpmQUnKR06phbrqf2UByVy54Tw\\\\\\\/kEWaQIecYvx+U\\\\\\\/s84R9PDbITDPe00u2UEvLibyU9RYu0mA5MB6sLQwNPrE8OWiupHXQmANQm6e5yHzafCqGwo5U0TjKVb+drAtMkYbkmu0T8T+xcTA11xXpqFYL1kKlQKkgrggxRWIG1gdvp66C6QSWoct+Wcif6jjibRAnWRb9FPMzlvuCSGkm9IQlX1Sm0TWk1Tf+IC\\\\\\\/LDop5aB6cqscHTUHi0p5et\\\\\\\/yrJHc1DmKZRHhMihRAkEwrmJ5BTfFO+o3+8iYVJb7NS39FGtyeV8iq4nZf4kLXLv1eDpuw7CB8UqBLTMGizNc2dt\\\\\\\/swkAZRGOGhp3g7\\\\\\\/B05l4XZ510Ousmsp4FQ7Tf+StPN7VRoDEANdTV1viSgoskx3FIwF0G6XcjzN4yrPqvcFBgaNlCOmVTJ4kHESc2kIr87KCNIbXNr9P7eV2GSaiKP+kbwJ43T2HI9a6q7nOLkV5i5skrDGQ8wkDmRH3ZRdoNh\\\\\\\/KEXzDj4QBrsLFqv\\\\\\\/Nx6goUosSdAPWZ9SoCOXsyFq1Mjm3vpq9SFjTgW6nzLgpEx4obBVFi4qtY0PWWJ7bCBH4ihPYUk1LZxFUNRuEsc+lhm\\\\\\\/Ji4V99aHrxi+Wpg655BJkzeQDt8GWamNt6YtgaxlGFuy3sDeM+teuxlfBvVrOsStvFhjBm5LqlPLfEgNoS+ws08svpoGHChqAM0DC3Hy6PjY\\\\\\\/UIvKyupZSFupM\\\\\\\/hPNAnLUFku\\\\\\\/tnyqWcJ\\\\\\\/H3DEDhggp8CoAsK92ZtuBKpVL6nlnuxePPs46D9IKLs8REOI97sYZ84Te1uDt\\\\\\\/F3gLiUW\\\\\\\/GmvWSaC60GF5ooaKUH2Bdnf\\\\\\\/ukzvngc5xEWDc7JDut+eEx5wX6PP4e3pI2Q5\\\\\\\/3++ZRD8HN9CmJFl9w\\\\\\\/6BJIwSr6J1Jhv9hOZ41SWZFVZveITlw3woWJVqxZD1iUC9sM8EMQjKIHnokhJEGZdhTvEn\\\\\\\/ttC++4GiX+juCHPLRHovylIRJ5lxOswIYaMxbc06qsglOyn7Rb6fRiFd2MIQ5LEvN0\\\\\\\/VM5H9FzuF\\\\\\\/3cUA+1NiX3bzV+74LCl9zyQixUBMWHVCNONNpFMBIUqzUbz1ShIytyk2TSqInTJSvVoOZbRsYrl1a3Z13DHrM\\\\\\\/TaaoOpMcJ\\\\\\\/gq+BX16vw6tHnezSQG5uyJyiU7OCpPJKo4O+od3SYWqlUBbq77e9awJLT2b8I0+57FoZ6CrbQJVnL\\\\\\\/13hkBWs+51E\\\\\\\/XNPSPBBoVtVHBAOW3Eh4WKYI4\\\\\\\/P5SltxqG+3SYwyzAS+xVzTSCZcjUlSp8kW\\\\\\\/lgaU5aon8MLgf6NYdik\\\\\\\/Lshc\\\\\\\/WU6D4vSEi8HfIyGzxVlaqYc7BEnh1kHEjjeInF3oppwHbDnbItPqObSyphxJhWMgPaGtwbd7taBMgwgkUsnU0kRr82G9qK4qjkZ\\\\\\\/fGvZx9QpGktRTZXcTSW+OUFV9kr3juOA+yvybnbYaGgmhqwxRdKaNjQRlRHu4YPMLQOX5oNRNfueSDqs2Ad4Yi77NhnVU1QAG0c8uzoOUPmAlNT7+v0s9BTSwWztGTfcYQxl\\\\\\\/SWpDDqjrp1a+vf\\\\\\\/dc3h9SJoBEFPkIebwM41gr7k9xK\\\\\\\/9RlIOiKiHS\\\\\\\/RGfybJPbj09N5hbHMZxfHMzOg0ufiEbwMEM6GA0\\\\\\\/f2h1RBQz+FM60Ej0WA03FYzu19Xhtf8YGhAFkeFCZAUmB1c\\\\\\\/vX8lWAPbw1TdsVe0RQn2+iX4\\\\\\\/ByTR+bvbmcsH2geFKHO4Z49+Rt3eCLrLF7EbmqYPG9nQaIYgXgBTGgaEl8tSTRMRUtJCRBjhnMMgMwji\\\\\\\/wcvq01XPHe1JNQzD5u3xkKVTVKavf45kuQKsmK6fn5VjeMmnNe4KEy9f4SseITpIopCoPUL8PfiLKVaUf0nwt6KOI\\\\\\\/kdpaM8KiXAlVGiklWmnNG8bSlafZeXEa9Uqqsx\\\\\\\/L+7ncVIi4CoHicl23kRcQSS04NEJxGsBsJ01l6\\\\\\\/9SrATqBx1ovVPRIqrJujRUZw5FPv9z+YOA5Q7ILu3k7TE22QvCDlbBk67DQfYOyPdyzjpamKniPbXYUlWcterIzbgaWzNqHSUpdw3KVji7bAh+O+4e+XUrRujIbPRTGZClFmAKMvJOtGMd8ukYPKf8D88RwezOCJRmDzG3Tj1+1KgsTxmuYhImu8KclPoT8FEuLEr6W\\\\\\\/icb13xqgoOMo5ZUCr6NCJ0dAKDxQu9Nl8NzakQCKrl3zso8Zs5lW6SVoKfVRQK\\\\\\\/jakHpM237qHWL3KPRiV7dVRWOwN6fpc4IBGXdAM9uk9KBqv6WfD9pQROIoKJ\\\\\\\/9X9e09Agizzdfcu854H+IXw9M2IR7YV2U15DkTjIp9JGGmifniZsWGAmtk\\\\\\\/Wl7xAyqFPLwOR01nVzeZh8A8oFiLvTyXYNu3fsiVNxwLHZ9lkZhBRY\\\\\\\/61NWpo4rMqx2zKm2LRg9InEQKNO5KSN2TRQ3mgLsAlrUumZ\\\\\\\/jKsylAGFUmRfUeTn9ee6vXUGFqZwtia08xHece4cf2FgnV4cjE6Shw9g36bKc9l3tEv1mTdlNoNpn52KBrVJS4rj8B\\\\\\\/eUSIeEInzyZAr5ElZstUBl\\\\\\\/1M32KfuOVe1OAsi90t66Q5Xr00Kfi\\\\\\\/iDu+eSsGZMIcgffh8vLcyzNLesFMdTpHFPLqbSv9fL1PBsjNHPWZLepRf1sUNY20RyDYxoEJFktlYS2Ha6YZgpTAOIfD8LQaeWeLBfGjwTAH+j9Gn+i48Ah2H5uKz05\\\\\\\/eL+oa\\\\\\\/Y0qZ\\\\\\\/bEdQsrwstjNcmB3UGnz7s0Cyyy9+l1YBow2HQ+iiw+FAhcLsno+oPxO5Me\\\\\\\/PGN4Mqx0EKIeNrLMlN5b3CwhHEVQIirSKvbPRC4p7J\\\\\\\/h8vA\\\\\\\/\\\\\\\/\\\\\\\/zqDT5\\\\\\\/ES+wWlh2aWxRCXdLdV2fsWITUBWpXg8I\\\\\\\/DYUWPS5SndOy9YT+w\\\\\\\/\\\\\\\/Pw1JOiyHSd6\\\\\\\/LiqoGrBbZ8rF3WureioyOIC83TmJMhDI30wBnlj2UGB3QThpvCcF48dLXeOqmGfBU88Ic3sYtUsJzdIIujI7BoeB\\\\\\\/4knNHLy8V+mbL4f1GZETiDSswLV0BbJPUTCOuZnAvFvNZCxwD8DqFc+evjmnf\\\\\\\/qkJpzOIlJgAZyNW+kJF4KHi1DJUW1xLCPzO24h7KnuLCmhrzbIJA1uc6w9YhQXNp28RiWqoQi4TDiWDmD1W1khoh3TF\\\\\\\/eRqy6Q4F9Q444xeWx6Xlhwc45J\\\\\\\/uSC6VKop8xlAo1P7fdER76MsXHQda1axavm0jtX6vXs1ufWCqd0oPttramJRK8Jk\\\\\\\/8Tnk5jhkfP2YwE6dmkWxcbuJI0ZjxYIStHQkKC7MHTWsRQNet4VfFFS4eB6ez3o3NaRRgHe0F5JwfTFJyo5zy0S4eB6Sl5xNFtwfAbnzEaCRJutr7GI4UQX8yIHrR4\\\\\\\/3ds6UDSrQsuJt8p7nmM7nrNh\\\\\\\/Mq0pIrRcdlee9wbt9CoJYPMc\\\\\\\/Z4e2TGym9w2+rdI30Tzp4gnxA+ssatCocGysYhI0L\\\\\\\/gvmw9p5HUTiFxXlNYodk0EJa9FV0k3CTtPuTLSggeALM9KcnEaJSrkU6ZyO97jfsufGDiF1ye5o95k5myGtFaOLnmluwZkrnn0haibd\\\\\\\/+t53GkBfQRXAzCijfayYxSLb61qraQjfNMO285ShbdSjYZrb0mvdLtPaiSkBO98F5qTCIARHTIzmkpSUjySUj2dTaLBZBWcjhtK8elYOhB4xD3CTjAci8W9WQjA7IkVOfNHCqrmd4ASKnWrMB+uoa5eZYYa9t88i19ejjY+Dq\\\\\\\/1sPdqz\\\\\\\/UJpShbnAEcRwvUli2CubdqPvt1KAevEkVKFTHu88\\\\\\\/KDEVk2AiBfQepI61GfXeMEBwgi7bnmTgclaFcoB25rWbusT2\\\\\\\/QrpFUoGDHQ8IBYIPv5CYW4bsQZoiLYfcfJU1LaZt6BoSTS+v07EDdM5ic1Wg1fhoy1aUCIBbkybytKLKVXY7mvqLxozBS2Bo8\\\\\\\/YtN4k7YOnn0atwGrRXvApnj0XYQv2LXDvOg02qbNwX2Oi9LBWbtfO5wywm8+\\\\\\\/YnircMNchfOY0dJToBq1IxGW2pWL1HBcQoHj7XH1TFnZws1Q7v\\\\\\\/SZA7Ci5ReOu+EfXcJytdkxboYYLoNQYiCviUbjJo1hNI97alS1lq7Wjtpjxej8LyBQ+xETjInIdqEhNSAMN7Sikdu7eFsf837KYudHGDGJ0JAqyf2zfWknJZyoSuH82aLN9ufFxgzG6C\\\\\\\/tKvESR\\\\\\\/wL3a65mlNiwEOfYtzjnS868bcGxHAJoXFdL3oSGHMRsYu2p6\\\\\\\/3ssDcwzujgfJT2oxSvGP1uq7UUgegrCiFNavEZeN0usbozdLwPXD9\\\\\\\/AtMAUZ1mblVfH7tdfNs5A9T6F12yxZCS9TQpx3Dci0CzmbsurjWnh1w7jmPOh8EnZuucrtHHWay1nkWRreBA\\\\\\\/KAvUIiqRnV7Ko6yGGZ0iW4GVNvSBWIDkDlxcIhHG\\\\\\\/WO274T3TxO8Qwkl4+nGjqhMevVCB3zhvJpWC7SeWD6WZtejhu+fUnbVw9szhUrEQxjRq9dWU60y+13fLb0W8zsTLrGLR0ABlyZmybmDDnLhiQTltOO0Kv2rRHO6JvVysEM5FOXP9+bY12buRP7lzWWLiE+atIg1paUEKSnpyy6XA0LEJo7faTVwvjNgzps9U6ueTi96J5GGG+qOjdJD\\\\\\\/LIGlf0mj\\\\\\\/iF6PpCFGKciQpHr0VYxiVrfpFnJmS1gtIPEwi35Yt7GsydD73+Ix6S9k9YAIq37aJUNNEdMojiEXAgiSpcL7jOG7lS81Z1pc+7ATPcCzq6f0BwcTvARw4vqcXdN2fx7S8eXfQingdXm45uXoK+WBvqRr3j9n0oGL8DPCk9dYvRg35U\\\\\\\/HXw9tEok5WNdj8fvbjQvJ2ZVANwQ5dzVC5cEOecM6gDanl+uqqnlmCjCCO5pVi3tl9ox8CrnHcd12p5j3c1zKgKucvZyevPk8PLbJ38vQgbqNeeDREXpVbloze9ZqLERzzrCPO2VEZUtHGLprOLK69SXka74m34z7sDgrp8Bau0dEYIYY63gRWiWV4HTQI+1W0qdwSI2wvEel2gwv+eabkgzpGwvVUxT4uWcX51RSDN7qmX4Mav66p9Zof6JjmCpCDWWvjJiUtTytXcW08RaZ6gm9Sz+C5643aDphvat6pFj4m0MUoq3cHjtcjQ75GNWuOk\\\\\\\/Na5\\\\\\\/rD\\\\\\\/Au\\\\\\\/ebAKkADtFc17ECbMJobjyql1txuBgCATJPHrVsPwZ5ivT4joAh7R1o0VtxmBbHiyph9Jb4k6EBoyFUP\\\\\\\/8A1t8ItEr1eaKRjYvdv7MdCHPGf527CEjJl4kdKFySR9ZhUJdYsBpsT5YT3J3Cewza1WsCvOLyL6f9GcUGqpuTXYZ7EnmAqhKlgl+pTvRe5dpfXY0D8TO13QjAQwFCQLRJJqSoCBrchexic+BLm+7Vg9BrLcSqo0Pe9rbl6AkFuLwgpA+oStG0yJNWHz1rWalJGq7DTL4Wkc4Riy+50iPk\\\\\\\/QfpnVlNzQe9yzd7OTVY3BRbjrR3HCKRHzk3+wLDAq+RrjlbupDlwx8Vt817ZpSE+o1MXYiDyVca4yUXwtrRA4ocVyECXbs\\\\\\\/RLe5H+zuXGAqWQAP80nO57fdhSxwBlxURbBuDWY5FWgO27Q2kQS4J8JvhsI9F4boUTvb206SH8ERnsEQ7fi6dvQtF5jVnc\\\\\\\/l6vZ7BoP8GPqNxqIIGHN0QcVRxMMq50TkJOjpKvUYnCiVBZ6abX7vNawJ8q73Eo13F1grG6NUi9QyWcClb84n3xMhq\\\\\\\/CbAVnJW2jGZ8Iz3KPgXZPl+rUVYMBxzlV1qMGrps+hcGOUoS5alulXUJt41TTw\\\\\\\/3DZz7V4stfiNG\\\\\\\/Dc7YUoREx7RtOaO4YnKGQ1wLmmVv+ncvUcqGlBXxwdsGTY+K\\\\\\\/Ts1Wmazy8yLpXpviRt4HA3qg5wckb+4SSxWXzs6WmMRQ8KeUbxwYJEUtKjlzMlVTt7UJokmtMvWnzN49IRhSV041Ao8FKIqqZ+lHwo46LUOd1lJZZU91mQQ0DU8NSTGA7gvpoHHFHDEr29LsowjWjeVbQ8dd0kaJgkLEB0p6WoP1jPen3fnCiBwbSGv9RqKGrBKjgjlKF2rl2NVbHyfl833IR6O03ThVeE09wzHjfO5fw6Z\\\\\\\/gnZkmhxfUlkmm\\\\\\\/NHFV\\\\\\\/uyhGq6syTtb9tlof5S0GnbN7W5wD7\\\\\\\/0dpk7DqrxkN2k2VQXtcy98iPZLMBYgxdVraOStS7L4\\\\\\\/Fvi2wgMNv3zHnNmTKY9p53V7bVtAIGbPR5+ijN+\\\\\\\/E85ntZfe15iTlU6zGxYjWEa5\\\\\\\/IBMpTsorQ+\\\\\\\/hNgz8jrLe1jccwQDKzDMMZYQ0zbbo2uILPb6AzjwGHN7ERTnwmyD5dveTqpPRsswe80pRXGWPcZ7CWdrfw6E\\\\\\\/791xT3T3aTprzraeYPTxhtz1FlHbrpD4cJKQIyluegirUKWo5kFmOG0IUbHW1jF25uLWcJ3tS8pDIuhAOGE0WnHkLqK3x4Xd7tE6wwnwIlx7A7nqr075jw0\\\\\\\/AMMu4DYHyBjJ9aiiuaxxjAOdUVRjlApQSnIK2rfkJ97BYkGUCw4oJNsiLIHfrA9semfii4gKwmFmvrivNnxkCkvX5MACg9NNPxc9A13L2G923Ujl3gsHqcPiEXq3kmB0OhSrquLZRMegA4vomVq8t0zEALy1YbXipeu8RDuNYx99JBz0rHaDKkDoLjQrwlcKJpklsuidsplA7XEbMp4TZ4gzbgMw9016Mu443KzAx8h\\\\\\\/sZ5qEOrZZo4dgsn2tDtyuBWoNEKSGWpmst+YKETJuBjNgnSWplNwlMAJTqIEr86JOyZeBQUH9EA8waYU2tot6T\\\\\\\/nAQV\\\\\\\/S8XLxSVcEr9YMdQ\\\\\\\/+oGu6u1Q37qMjYvBo0QnOwTXQRDyF79xZg1us+OWidYb+kJen\\\\\\\/5PL7Eg4puX0WikMtesFKj51ONDFEBAuqaAScaj3Eo\\\\\\\/OHU6WHQXi6z4AiTwGoJQpA0BgA3JJNuDkWXfHrlJCRjVC24MzgFL7xA2Zo1EUO\\\\\\\/SnTQpdq7lbzjzh3JliKslusPKZZLq\\\\\\\/QCSUotPFbNyXzfMqPysaY9t1e+zEWENl0YPAo2I7UwZo7llkTU8\\\\\\\/VcEIGBdPyCDlqxwyWNHPs\\\\\\\/xjopbAoDkmJ0RmMFDKmABYHLlagllaJN7t42r0Dju8quOA1sG2WQLJ7TO0eFRFyTXV842d+Eo5biNJdtVGFFTD+shniWDk2EvcGoiqRIepOXwMK5LzVRxKMwD3j1d00+bw0\\\\\\\/I+2Cfsaed+pwdNZRexdxyFZJNxdWaaL2Ec2nguXDTYbvE4d8nbouuVMFbX9Yp5fAmGB8+QK8BFevbfASjVF\\\\\\\/pFG\\\\\\\/JQRDm1pGOKZJf+tlHPITipGQdS6j4BR8YCP0OBMjtVUkoWDBLLsRl1C1Ue\\\\\\\/LS1pz4wx\\\\\\\/1sBFUOjy\\\\\\\/cIJkRxNgLBxFWTtpaUmFQlgcMd7qWazhFCSvBHyRPI2Db3IEfUkBd0o6hk4vTgIembPFxbi0Ld2Db+ioBbJdUEZY\\\\\\\/v\\\\\\\/6cPXYbh6mi4yb1qf7\\\\\\\/OUaQQo6wjIAmk2LkLQT5cyIqALMB8bMEGM+qa8rwQ9qEjaMUsE3i2gpmcbRgGxF3zMInnAooEYyZ+a0odhfjb4aD+9ubRovJ7uqRXiFJnjEIX52ufLsB6k4DNuVxrUi8veoTl3ZmZduU4EOy8vTJNj4yiVJ2zirHzJnm4c36v5zBGfA7NS4bAVxggAiblngYNPU8jOMRHilrlb7apQ5vSOJwj7e9W7\\\\\\\/L0Ga+9IciR5xoGaeAKe2rUJRoDtglL2oaUF0\\\\\\\/6d+u2geZEK\\\\\\\/6joBwevz1iOUdhDxfmsnJuTXEr4oydi74YJvV3vBGQXCmwt0VVuV3+Zf3\\\\\\\/tKEgISe6JSv0Xo+QPmvBTjpoLXryF8n1erBMGqBDNkzI27EKWbutkAAr1db+AGtHTMvCwlHVZmP4AkyupFa1Tk4tXeMgjK1f2mNPJLXNofutKNkVU\\\\\\\/A5OdvUGy6Rw0wFL62Q8vAHIQW0OhrpQfjwIMzgnVBoYPweLtqTgHg6AAXP2+k5qxOyZtCXjaEY0oQ0tDLx3sBeN6rDy8xqgnT\\\\\\\/dDydgM+tc\\\\\\\/+fmsGx+8dDQm8bCxcWS5D5td\\\\\\\/Q88l\\\\\\\/zb2KbIjyvjeaIty3Oue\\\\\\\/JwOx\\\\\\\/jBHcsTellZFoUv6cUKl3ZWdmuYRlL3m4Eghrae9yLYXIE+SBvRXOXZqKzwjZU8bBMROvuMLStUTHeRNyZBUcEgvXsxb8XBev+\\\\\\\/r9GPzeJNcS2I1tlJENdV9qFfY1NuzflEXOktKRuVJKU0K7yVutWTDhxx\\\\\\\/O\\\\\\\/Rkx19iJdozJ1G54I44a95KvsO9mylemBWZcoMkFovGjEQ3m0NCaa2KRXtP8sEzprExtJD9v90AptvnKf4Ar\\\\\\\/BC9HNlVGUNrnpwoBeIpK5uFeT07or7MaLTODXCwghSI3KylNtNemlY7gvvbTyixbjoD9L52fCqhQ14fIOLz6Qtg6RYUqcpFldlVqGYYkvE4nQ0RGGi3RJSEodvvRg8+zx9H7qXkbbKiyjg24F2JSdUX7N+RHISpjWcx\\\\\\\/j2IjjpjQElBsCTTwCJaENkhWKoiZ+\\\\\\\/Z8Vj\\\\\\\/5wPcvTQEdPGX00XXZKlv\\\\\\\/w0xclKJbjs0Fa7REawiusf7XDtIwUUIf\\\\\\\/RphX47gCCNl7aSkE+2JTpwGIu2WKNqtHSSLcUDDBFVSGtYLF\\\\\\\/vk+q2Ed2JVVaapAnTuNrHPSnCWB0LUqDnWb77fEidf9ey1IN4mMMiUSM\\\\\\\/Gw2OmNJZCmPk3HvHKT6zifXKH1BBCtMGCwunuyV7SYimuybv5f96xxzVtG7cu2hLddK7jVsj1mcbfBikS2ZlHRrh3ft2G60KGrcmE\\\\\\\/5Dg1ojzyxWjXueiWonax5NGtAdZ6Xfmf\\\\\\\/pq7B5sOaNTqf21App45jwpM7cJQZ5Cd49+vkbEuS\\\\\\\/t+g44fHcgycGI3jHukv+wAKJlQAO68lZQX6cR09WBiZN5J5LHlYfNnmkgk7UDGtVMCXGiT4KtpuWKxjmnwPzaAZtxeQzug4jCJr1VKnwE\\\\\\\/\\\\\\\/GhjBs1AYigDhSEEL0ujZPGlysFXQGWeNwyvLArB\\\\\\\/Mn8NulWNffq+mFQ2h\\\\\\\/S94JdUw7Pnj8cRWYnmWCJV9D7PgxaIChd7KFEc1kLoJ\\\\\\\/q7F85Snyhz8pm5vLgaZ7DU85zSPOGYXpSD9JPThLLPagBZrDaIjh2gfFdMxTRaglduxiQQTGyOYqLQZbqklrS5UAzCFeyMa7PdvQ\\\\\\\/GLzYjKltTrrNbEHNnNE1n+a8FikoQMhDYMz9ik2SBHT0Yk9XYIUMZpF6YbbmbgQSGy7lwvo561mvcSlmlI0wX2bZ+7wucz\\\\\\\/oP7o322wV19siQjOPGNeIlBns9fOuQkNsWsko+QclH9bNpp8n1oATuAhNNa7Kz4DwnwlohrR5KuyFRhwS2Es7csBj2C+w9bGCMB+9SO\\\\\\\/+RwL1l4T2zCkCJ49G8VwuHiEziuxRFE4k5ZLddDZ\\\\\\\/66s2OekU1Ric6qW8W3Dxd1XSaHAvnCCLNmPfTE5UQL6Epl\\\\\\\/8r1hGVRiwSQGqWSr9YQ0S5fWjWAVxqdkNkWPqe+PlAnTndL9Ayf0UN1USLaEikynrwbpk1ydSzouWDYaGMhO1wf+kGkKZNYPsOSbb5xVdzNpiWu47fR16u9dtgw8wJVBxOWTp38\\\\\\\/N1TJQ1OAtOejosRwYAVMo5\\\\\\\/pRHmvLB6CVJ5EzOZGCYH0g3OB8VxW7E1RzXO3kPAmRXZk29rG9tVJuAM4\\\\\\\/hwRi+siImK2p+4VPVugObtIvsBJJiZpUvMyNDxffof5CjHLXoafx33LlMUG3vMZJHlMEbfNlyA3g\\\\\\\/ISVmrXH6WGFKs8vFuP8BHuzlyPqb9lrZQoZ6tMn2qwNZA+yH1SrQkz3PtGq6gzoJYW7plzWH8GFWS5BInRUpomVWYaaZVG11AqS++KKJvRXmtiEUd+qaiBinKYwqBQmWV1HFDrxVUwiuWhSb40AzHLrOlmu8RQ4ZAn\\\\\\\/ODW3oUvqQXp1WC4ZzeCh7lgVxFtbo9IoxGOMtSA+TEwsfYWvz3\\\\\\\/VkmYKHk4GRO9j8Xr8i02EXIU7wjjuepvVs1A5n8om59VdTOKozGtONNfcBsP5S2wkUUTe2ssBPCAGONTnoH7JlxlhG+jl9JEsSFOfsiW01wxPOb\\\\\\\/Q64h8ZhQECJfEsBi0EDqWudv+nV27bqC45tfk1pWprIZ\\\\\\\/ufdpIfSYExp0+oEgD9+t83il3zsXAs2VT48629QXX1HEuM+\\\\\\\/MYW1yUcAcgBK3wB8acvvqtQIbq4L1d5+pF2nQoBBdidUXT8ntJNiwzhhTTaFEQfaox7kFEtPx4AmO4GV+9Ho4+31SriMVBnRELciyR6AM1wmwHq8or6Kqqb2Y6NP4EfVlp1jKUwFrVUCWDYLMxcAsjVz\\\\\\\/rUBIdh6FR3NkycblzXPn7Ke2aPuiqhJd+KCYMsngTeP73CeiiJzVnuw55G+dvP6wxtvMSkcTONr4JWkyoIMQxWOVPqquWe04IuVijuX0JtHBaJwT5QbHjP18Xr2C0pBjsK8yyytULh8B9\\\\\\\/ts5qCC7ON1xKo+8h99KOxGjRTEGW9ORbszPG8+JXxxVCcAD6SIOdZDyho1COCx\\\\\\\/WHLU5AVMZK5a4bd83Fymesr3j7RV27SYm7lpOir2YxZOCLkzFl8xrW4\\\\\\\/y2Vi5e301iA46mkwLFbz97JOgUPmo\\\\\\\/vQVSDeeartPyz+rInwBp35oIvEjjYEussy9oBzJcpR9p6GQdyaud2qdZLCxit0fDYmSAACdAANLSXtpuqS2pw3S0UHE9gEshGg4Stj3DO2pEBBJUmUWlyLNYkmEFjWwsitsha4490OG+52fLNoe75x0QM6aHxwfj5VDwEYb5nWLtcXBmilUP\\\\\\\/Q3HeTJr\\\\\\\/qal4zG2Jy8qFvwfWP4faF2\\\\\\\/eXG4Fpuyn\\\\\\\/hyWoUCYhF\\\\\\\/jIKVYYrdYVhP4f6j1XqBJBvpzKpCb8f2XIIZ+3ZEaEBnSM4jxd2JgwD8ESVyz8mHbnt2NMYm2G4FVUtbMOKsD\\\\\\\/\\\\\\\/u5qAvZNAexxe0g84wlv0ON5nvpJGICXZzkxurnOn5PZcyjgms2uhPWiwdkyzrauSKc1rB3iiga6zD2behPEYcxTkJ1bYQgSFhq9BkX3rwX0FbA62SSpRlbCS\\\\\\\/D+OPpvn7VHMaEnqm+P0BrrOfocXUvoNRoYQAEpMivMhCj76mklmjta+RaerFlzl72hy\\\\\\\/wj0q417oL\\\\\\\/VTDzDyR\\\\\\\/1Px0FK900E16sPH3azyV0rYIEmf5XgmxTKzYrNL9AxSswm14f6zUqyHz2sstzUniuKc28PHmO09z90PzeHyPqRebwhlBff59ZZqne5kEqFPVlOfJlzneVyY1uLXMRqCjMU8lxdNVko+6L+m\\\\\\\/QzAm6cXyDP52cmf8U\\\\\\\/KhV\\\\\\\/GZ4Jq00QynpbhJvNWZb7RzVRuHH72LuSeY+dhmbu+PmEAeeOj1budlByB0CJ24mzvqHjw\\\\\\\/SpBB3pDYeA22xuncGmIpnvgZ\\\\\\\/a+bx3MdTeKabJZkCAklUFhMgfVpkxGfhP18BAx+iS6GrdgpodnjGgApc35wTARVENySH8K6pm6doQchSKFmWw0QFgZMlTdPV69jy2mJE6u0zFnPpnerxplJEc8l5VXhdDC+jGiBxS\\\\\\\/1LQw80tegpcXh+2q6ra6fsmfE1+v58ixiJ3RHPQryA7JIr1FxYACM7Am3gB8GFyrKUE0dbZVO+fudVRrB6P9ehhbYP+jyrbo8e179eAj2ydjekNWHBfVKtisV6t6C24+vi9JX+PTi7ciLSo0MJ6D3Fgn6VhLUQY09t5BEvW4bm9HCvdp2BTvJ6Jg3ytCYDA\\\\\\\/gfOzZyJudBvSSn934uRk8SBh36sqaq0l4Cf0NcIQKwamSk2z5rk5x\\\\\\\/Gv2avv7CZ8\\\\\\\/47bRUm+3\\\\\\\/4iHaUa6VmieDT9ab3Rvs+irS+G2DK7CUOzkbcIGe6jWnIrbkTKN5N6lFHlZC48zLBUbWFXmja2TQZn3f+q66rOZzBH9rnoRTLGcbXaNvIg\\\\\\\/ESEVpD1ycoh6nc+OVrJEqbKoMSCLeDz27GIlS3fI+zWQCBMIOUu5MRfl0ganYyuOJXCTU0CCQJLWYttrem\\\\\\\/UzaE1CidFMT4FleURdVn4jIcTz0aqkt2Rawc31PkQR96xJvEnlWKPFknqCAA9f2Fn2hsnuB2lExt\\\\\\\/GvjJcGBs6\\\\\\\/0jKYEzKJ0ngnz+BcNoNB9YdukhpwxVdfGvrFGoX3f4n9pPvFLCSJRQUbGVKAWUIs1AwN0D\\\\\\\/G5V6Rmix4+DeRM+noZ8D4LJ25erY\\\\\\\/v+ClHsAgNoruMXCUhWKL+CkBVbkXj2QUbNVAsDCxP+x7uV+xrQHW1rKNtw8kD7xr\\\\\\\/+89sL\\\\\\\/OF47hbygD9UojNsgydz15yFFEQBszVfjjV0UkTexZz7qR44GpcQqqJ4X6uY0Nw0CittuOVdpUuJZIo\\\\\\\/3MLM0E9vFtVB+qA5Wv\\\\\\\/u8a9\\\\\\\/fuwJXJGI1LTYY5nwyC\\\\\\\/KOAJyZeSBdVBICa+QT0m1ey4cz16oZ1SQezPjGNvLut6IgfvSLjOoCU3Y8Fm1RnM8kWeTznEFBc9a7FmJQ\\\\\\\/jK74ev79jgIxjNT+9pKql3DztJN701PqpPxFbFgcJkgOQsN3fvBDP3UbOecU2Nb+HrYEc5xuTJDySkmUNVdoqQEjOMLFvcPZa4kK+fqYlRhIsgzrRRxhSXWg6TevUKGWtATYoN80Qr3Et2YEYhUB3NMQOaS+Ley+y+E5Ypj6SGj6v0xyyJPWWMmdS8QMiORKl696oxzwNzzkRDHGBYpdzCrbiXDVe1coVPsIIJb45u7bn5pi7sVkVDO3m+8p6XbXOzbhEepfuapVwu7STJOCcNM5MtaXPtmpNQyUICukqrHrrnpfpV\\\\\\\/+Q1vn3Vkj3v2qMkQxy6Gtg5LkaxniFgzJB9eGfn1Q2C1ZhnkmUXORyyCEWtFGGqdplYR5Ic7T+xYJpFI4yw\\\\\\\/KEDjdAMBUzDzGrfYYEeX4F5AbrY36Uh1s\\\\\\\/zAjATHq3r6FmcH6EcRBMUUCRYuXHuJF637YuJfbiqT6NlLMU\\\\\\\/JPmI+S9Cahdz9+c9qgOcslw61aqLAezWiDtiVhCogQNICQhJVZujGS3hMf6oxe9kc70N2dupNyWrWIn8MxoPM5DJzj7D1IQzcj2CGY+WUM4aHoeBsjDveg7nXz9\\\\\\\/9nHFW8ySMZENQmjOcYeMK0cyprdAI4tYsrHvRmdwJ7sUQRd7Y2j+ekSFMUlS4eJIJYE6gR\\\\\\\/QelZnkJzPuMQ\\\\\\\/vwj4KafkDECsm4ujkFpv1ZaszDCts2+kSrp79AI6H6JNnxC4nXE9AeOYW62ZAFj7lkUdIgp6sti2bKvhXsFURR4BYInX92ITaeiDwWhkb\\\\\\\/WGwEZyx4XNzk9ROPVyQgqprjODnZugukR8YRRGWN7F25vP\\\\\\\/hjxQv9CLDsknxc8I0oJMX01AGNv9gCNlE1J2S8P+8Erqkj\\\\\\\/35pSHxuA3doFSiWlPEJFvL6Y0lQEd4sSFQDV9BL\\\\\\\/hdKBWNvGbNXkIrmy\\\\\\\/0wLRH8A+LWNJTMCznfAA5XjkkBK25Xh8T6n2UKUDBPZABrphuYTpgW4U5OmI7BzjKabn7V1K2dxFJUUksk5xmGcy98h4Vop1Oe5lVJLmZlLxZE33br6hmdsQYgqev51OubsP69pHA1WFXFWaKaUNdTFY+unOD451wgdGuShDEedhNGSsjYlZhM\\\\\\\/GVCPW5oW43R58YditZZwqHqwmGeGd3XBueaq6\\\\\\\/RIRXYUOqmHFZQB9f\\\\\\\/g98p+5dpC5W4DgwJ1UdASFBQkpvEvnR81LeTAih2MAq5MXM2ZODmYpJ2F523RvTFgjODTbhSeIqUHBmM5hmoJUqcWA3abY59mYnSAvyVj\\\\\\\/sWZtS\\\\\\\/m7Ai7cRnzestdi2yEkbUBw\\\\\\\/8U\\\\\\\/tgXQ9p0\\\\\\\/4Je5XdVMYp\\\\\\\/SRmv0wLH+HOQ75rfwwVug3yxojlG2pypfUUqHgFKZJjja9fXwd73Lc2cgn72tPuZxbl33F52udp4mPYicMc9gjbZ1GC3EQH7X+WpeO6vo9a7q1pjCiRAwdnm0n74Z80Xv5YCEfBk8t5CC2aDHRoenPQdDW0vjWs1QpjU9Ie\\\\\\\/Xkyl5glQSWjzpfkleFTpGf\\\\\\\/7C4I6DwUWERiwehXShTQNyT\\\\\\\/uDDfpTY7QEHG0jwQdkvXaznCAPbAMbkdNjzN7EmV8CaoxmxZY6cNv621gv9cNdHUvIfyKTrzYKQNWKH6F\\\\\\\/1NHIJvl689nIVrrGik0uk\\\\\\\/+vcv1zZJVplXfdMMirkK+qQc80aEr9r8GZKCui9oLRxqtionIRoZnsoBPZUGB8VYB3pgmMUPRljIRWoAUBD6MC6FycBZwlhZ5Oe2p29oaVoUOhw+uDMCGnobr3tV+CJrYx5j4c\\\\\\\/rh1I0HlC5Hk\\\\\\\/Rm7JDc2tkcr9vNJlC26r0uUnEihZYX3uXLewrFi1zouQWl5AJwkhqNuA8MddI5vCVdqpcE3D+Ik6OmZGxZfFi\\\\\\\/2tZxFj4J0aW6gWMdJgO9IWlGpzh2USprlTrw4Nl8bUU6iseOHgh0yZaklnx152HXVO8BdpU2BZNeW6vwrK9GfFUdSIWiJTP+PXBksX8Dv6GC9EL5\\\\\\\/U4ReAIYJcNOlEfM4ZXeVE0PFwrZ9R8ZQ07WUlDCsDLceBr9lEmKL3ST8KuqF6+Tk46M\\\\\\\/2JEZiN1sSRDBwZ1fzNPOu7l2NRQ82zXlThTjgNxJuPsk0G3JUj69B5LgplkdOXcclU+ikwx9y5mrE\\\\\\\/hSIAnEhZoIG6syspcuXVIk9gx5mXYrEeaGdoFETzIsYptARqe9HzxlVqGtGUMw8IYhyQtt5ZScNuo0Fmo6TbRJV2Fv0jf3igKAOiORNMHPWONFKb2ClKOlXIJ80C71mXQYi5uNLgsXaDBvnZyAaYDrYefYgXUbOQo7DCJImrlEKsEaTg5v83N+xifk8ZdcRZQB5dWjPez\\\\\\\/shWF9SEYkPTsDd5Zr\\\\\\\/HAeIBYllmB5G0+vu1nHCqnREdcvUqQ2m87a2AVamVUYDluHeBANBc+TbZYJJ23gOKvxTnhWLaDmteQMmfnK01jU3s4SCltj9mV+Z5foux50tI9Mrh5UMrKt4xxj9lNh0poBvByTEEZ1Pwk8N3+6TtkcKigqSWpserNC3tBCivVtpFtQcFNWuCBCy68xqr+ytv7H3UT46HRVqGSdTqHlNa71Ba5O9gr2\\\\\\\/ja\\\\\\\/iaF2NJ+7MNI0rL+TPUalWzR2Kn3Iey7qt2YEdVmY40BY+HeZ1Pe3STv+jdJ9Ufhj5wFOgKxfcqnrS2tEw7Zv5h7IrfWMJOoEC+\\\\\\\/2G2gRYRJkBl5oMV2rrdG4tTexhCAcXExV36d3ijesVOxv3XBc9WhC3\\\\\\\/mH6YbU6\\\\\\\/K6Mkz0oTfg5M6cm+quFTc4RU0bG8rEjf9YWls0qfYIQd9PUW7e4QAv\\\\\\\/ogJLDTvqxlDOy\\\\\\\/X\\\\\\\/KR+MTtmj2JjpXbSIwED2bOFK5TOHyQcs+Tn\\\\\\\/a0qnnUJ6UgdmHbaQCB6ifpfa5tIl7K1GBW\\\\\\\/VETm4\\\\\\\/yZJN3F0Qf3nrQpLMyi2tNlDo+wrv26R4jrwnNsmcCfPyTuIE14R9a+dGTAbGNjByVd9hdcLX64s0bOYuGDQQsyqKo1t9F1XecHpU6ZlkNh4WQ8E8\\\\\\\/cp3MoYlgY4g9H1PlrBj6oiRVCBq8enZnW7Ar8s+ZJAuE60B7XUmWc\\\\\\\/xAe8GehUiwxplFaS74N13xOpKI2QvdEcQxfcR8F1Y\\\\\\\/PNTYT2AhO7pY\\\\\\\/8YAdpKe6wLWWtdsZ7IwMGNY1Fg3r38j3sUcoUjrKWmEkfkImsPcZRgXXcCcr9zAEebh4LtkgtQhYo+flwep7+14wN9yh7mC2IPm6GzppGlSHf57H1D0EqvRzYraOC9qHpOXEB\\\\\\\/vS9BIQ9PCxNqbzZSl1O7NbghsGIW4Xil1ChTmh5e\\\\\\\/iPG9dFy4en605hNIySqSGmDTeZISpP9R0NrgLivvsNE7h6fQ7sQc358UdKGIOsg2S9wpMECI1f1FYyv1VIzbYAq+j+awD86SVmh6TKmAFv+wtutpaD5PInF26QZRzRkkRfKqVoJQCwVuTQVtixqbVw7gsJYhVr5YZLV4yVdPulexDWBYpnEtdQ7HYGBVrTPsluqHbJTg6kG\\\\\\\/SB9DhaTsaye6Ix62fOhtIcJVO16GJWYjXJwHhrjyBucQC1aUMTYeLkUvsJcA585bqwrw2mfM3FVfjCvqi322rLQT9g6BOnc6pI9md4Pb2+KhFr8uo3p5T1jdNACSwTj+U3eEdeAGSTtssmihbm3A8s3tlE1zfW0iWOAFoYjVBo97TkDnehJqRgldWvLgFpRRytKdhHS9OcF\\\\\\\/PD1YcN4kAMkteywt3EXFjSlFkuCsj1PHTM6Sw98YkJO3YlFP+ezE6ybtBusFaBLzSYvZcOsjaLw\\\\\\\/YZzN5ZoEZGMccje1kwxfqf2Rb2UwFtR1jg1IxUsvWV+TTYmECI+kdXZFnTpKv44RrGWaDhb1MyhOg51TXuB8tsPX5gF3Y4bm0oQPMqfXaKnjXBAXvXojUCbm\\\\\\\/iqkWnaEwAmhmqBUyEcoCS7agIuPkrbau2UPYbCCnvF4nLkjQ0H+h1Z8NryoicykEFOhvVxakqjvjJym34SJim00LxLPsayO5eX0r93PfZ0EIT6700GUU3Wk1+JTR0CfR1u6LPo\\\\\\\/1og2BSGtayewp9exqIqzroL8qJusfvrF5g7zajPPMzUy7xScZiHw5Zy7FecYuBQ9fYudNw9\\\\\\\/wLlaFbowmWpVNLShQacns9nPDZ4C7s6GO+g8eKBW2fJE\\\\\\\/9fCs2TI0cBd\\\\\\\/n0WDRuRbpEXiVU5S0um+LHidtrJJ4pQM4ERKKcg4EMZsi2uN9COGZZvy2r5wrdt00Pu2CKodVCh536yipx7NUIjloYKthey9bZFMho0sQbe2N9cXthzq+FlN7up+xcEEnasM66Ar323+s0ORqOf+9shs\\\\\\\/pouvGBNB6wBr+Nk2ahWRlJ9qAPN\\\\\\\/SjRxqRXkwCGkI0hUf2Py24kGVvf2C7XqpZQu9FxWvsEXIj9WlO2XkFA+uWVTlR+m3XRxVTBczjZun3Ww751ONWZzzpGGK8cswheDqBcTMVAI3gHfM637oXT+yFOg0C4kI1YSU13SbueF5+9\\\\\\\/OwjEwWdLtBNYlT7th7TIrSLSeK+r8eNBENAHOCIZqEZzto74APelbUYu2gwaJuZd9AT+xZJoGozJHw8556WxBmj0vr5178nXSWQS4N5KPzLBoMZtvZTz0lMzTIFN96afqdq6cMBs28VLcF+tG3ed5VWRNtfhUadU5UjorCxVedAURUmHUFnha6rXludAoSUvV3hcCKGDWk8nCk1U\\\\\\\/G0p25ddY0C13EcvW0MNhiEg2nrsWjrqW0AbvjjpMneQWeenpMVAdGwVjRDOl59ebeS\\\\\\\/IpCbng3+qDeQUkQVPyldP1+vJtPTgKt+l8EWXN8dhBPOHTK9DJCEK6nYK2Y733uRxjSLav93BwEwPNO80nu0m7nf2VFsBC9jA+Jt9kyW4KeVPar8fOX7Cds5alEachT4x3uBb2zgLN\\\\\\\/orTfd3TQEn2jfZdEGitct5lsLl1vbgddK1B9FSL1f76Kcr4\\\\\\\/05g7JNMOrvi4vIDpw+wMttqSt7VhEhe54sCEwHdJD4rk2UCfaSIRCw\\\\\\\/EgMPGOtHvfen3khbKIU9WOgwdfOcZheynQ+UIXCJiMF589svcMYu5IfH3VT2GT51pf9qN7OuGx8QkhCuWXT9SxQ24hYdJJ8yqiscBACbW1nAey50F+i2\\\\\\\/vaZjzV72wB5zFhatSC6GVxi6nS9IHXNItPAJ+kCsvpxmoGQ8rYP1l2n2yvMMQgCRsZhDDzdL+ki3sEtc0CxXbWxhE0gOQ8Lr\\\\\\\/\\\\\\\/D4enrm6JHz3xPTUR3sFRwoGpez2Sjzr96FBZgEGtJP+ty8rBBjzMR3EoGGQZXbqLE4qYh6Sg8NhWgFH5y9JyeK+v3GCiqoSr02HzqVDIYptTbrglLnldWwWhTiPvEMTgC74Xfw+HJ85OGnD7AhV6I8gbSydo0003z+fEwiOvdUr1alzSOr0p7juDU8kxMW8Fh96K+y9csq1Zfx5f+d1Xa3aqz\\\\\\\/sdt5AMKf5IOhlyipUyQdV+BZ7NsFfRHhWYRA6f0rgeaHImYGxZNOJiVJNiwn57dgWfAk4CvTWdwr1b2NWsBqoifBk5deULxiniLxV4EELZ+nIdVVCRWXpaVKAQMPlZse3VIQuTAakMc0ygUv+Vb38wKn5Jn+VDQ5B5+7vBWCBO\\\\\\\/oSetNdGIuIFubWh52Nbx8NGCV5K6rxJ4YaqA\\\\\\\/bgNuj1BLiEKak8xlpt5Q2gmwhao4f4PBY71EwBtSr5CCmRZV3MRqUjaLBL2u0fszr\\\\\\\/zxzED9P6mnxHWUmoxxhNVDPE2tUF04QG94k8Z1bxLAVzfaxH5e3L3qB8mlL8TqtGKt8L8X\\\\\\\/FQO5qd\\\\\\\/Gc5pI6TNXf05oSa0EpQwLxZ\\\\\\\/HGMBTb2\\\\\\\/2hZZxlOV6XGnzArh55W\\\\\\\/QJ3tYlLC+lqZq524h325isg1Z8u\\\\\\\/+3m5C8D50DuClgs5DVpYqyPaDizPgK4OmTOwCkQ4yUKodJA1Xw1\\\\\\\/Wqi32BC8LgHYU\\\\\\\/C6yFjCSDSon2x10l41hVOJO+k4N4vkCIGPZ3DMYxgczwOFOVP+QhwZJguID5PHGoCZ9N+krvJNrC16Q87IsNkCp7oRjPH0Qs0MsY2QauJC1obIFUkX9fVxUm83iXeUI98X6ReBrb7oSyksJ5tDTrGs6MHBT72ELXdfi04Z\\\\\\\/u3xv+rzrG3ByDDViQFoVjbPSy2ohZnUAlK4V2q7h0Gn4r2JyvSnrAPTZhnsjjfsAqU6OLEhLveMc6GDtHogSeN6RtyCrAdTfTyiMV0AEZoDCkzb8ew+ocjY8YksbaNcmjU+8dj\\\\\\\/JlGqy0XBnu1M9HZKJDU\\\\\\\/q8w1I1jHrwYEKGecf2Yjd9ZhHBAC8f0vxsgBvFRXt67Vuzoyutdf7UaiRQBm+pFEuZ8NLUI5GngpciNQJ8salZ\\\\\\\/z1jFDptucusv+OIMsBXvqHMEJdLb1pBcxmrsXHwtvyl2emA0\\\\\\\/HwYyozdHyNhGXcWVSDrjdhyWukOuSTumbehXTi42RqR\\\\\\\/b2bu0SxGEBIOJgTkvW20vjVmZHN+ru0GLJXuHkZYQGR8kt\\\\\\\/LOHALO5J2gfdIcqAvCCc0eOrV8bHY0PtXVk1H4E3n0HX4tut4zZvZdkukAQoN3WOO5Dhl1Vht7qxmuqEMeeQ3sXl3uHqzPyL\\\\\\\/iBMjvFn73g3At3JNytCDs6HlPuLyeoTrVCV\\\\\\\/hJvqijbcMz4dpb2Xti4dEy7b4GI\\\\\\\/t2JQoi8IsFzdgAaVJQc7p8RQ7CVsWVeErKlOEWINWdI0sd8FJgrzKvXttRNt\\\\\\\/QvKLgsCAsriGiVtyJasRgUS5A4Bi7tAwHiZrFMAFz41Id2ORx41Z0rOsjxdCjdJoHfnoGLvguRMc154sle5LaeB8+Z8Pd1CMzz\\\\\\\/Ad+4z+KjzY08ZEQBlXTVxnzsZxS4pV\\\\\\\/7PYXCZ3l8ZaFVoHa8l+Uyd\\\\\\\/7xaYlTPrSVUIUx67HrncRHp6Oi8Pf2FZlpayTgkm\\\\\\\/uie\\\\\\\/Bb5vRYRQLUggk1rk8cmdcr+iqhuHYMjtHrTabkEI0y3ieTjXR8fsWMyC2v8hQJS3V+5orxXHwIIsjOoCpj+63ZRwmasObEhyAgnQevGf6BBEpXhBauWgQnTpTWlOCdK1zs30KIeROwdwNaRs60lwtYLY6t1LwRe3ejfTXquT\\\\\\\/ICqNT0e\\\\\\\/NaDRA9uFlkJMmg7MZn9iIgOhC8PzFUsGv6eDCCWW217xzb6kUPuhvLS0tT8+omLSP0tiXoK+8aTB9ko+Ot20tEDwMaoaSXRjCBTA8x1ZJCjWK8W\\\\\\\/uQ2DzpjumjEskBqqb3vkPjgVi\\\\\\\/XJuJ94W2\\\\\\\/lRjEzQ9l6AjHveqHEDyoZblYvky04uduQZJXFVKrXsEVMOlJaJnCjuaulVldLsUgIO8GG7jYo+jcbkTmN3ZijhkpbwhHfZ1yitqZCtgmUiSQGGzt5uQLPX+t7T6ZWLtlqsKy+txQjf3Jj9bVIfX2mTkgKPXg\\\\\\\/bxGHLHo8FV3UDZauEzqTScfR51GMEM2zA7fYtKkinV8rbnfCEwL5VnoUJiZ58zazhvw86JItLjYGAWZWyOkHlIUQGTekBocbUBfktd35muiF43ZLjQr4GD\\\\\\\/kE\\\\\\\/z7wsZX2c+eTcSoaIdTHmFX1v2FFu1y2Vo02a9oMKK2Le5fdTcZRlSzd78ECpxpEWvQTMfqVsARDAiKTSv38n8an+4HTgsUCsqx53+KfO0Nhqqzq2VE56QelC+Ck+xUEL0McnM0AflyzecpeRxRyml\\\\\\\/cagU2PqXKtOnoUk8aC+t+8Q9T0FV6y1YB+gYbawv8f9NIYdnRNFwE4U7KP+2KJDeFp2yt1qvn2vtC0A15SD\\\\\\\/EL0hW\\\\\\\/li95R9JitGalEth1WStKqU\\\\\\\/HE1hh\\\\\\\/XrDQSgi0dy7WB7KNqAkcRrV9y5yaQ6\\\\\\\/+GBYzyCOEBHd5w+J88WYKQ0jfEUnXqJL2AkxQ4Mnupb3Q4HEMObF5JmOf5jRBlMq0sEunjK4WgtVCfC23bU1\\\\\\\/qrLhDhU3BE1eqbLhq53t8MMejpH7zbElhWxFJZ567cdZbX9Q\\\\\\\/kH12IFitgvhDX0jTre4tzEkWQTpmae7jzULOe77\\\\\\\/NX3PttVWJAitQB1lMbqTBnTE8FtY\\\\\\\/baQFX2lh7m69Lkg3verkhrZmQt6\\\\\\\/CKHxMgcBlai3Nz1lDfYnZWi5pzvlACY921v+mZ6\\\\\\\/9AvyUOoMwp4nvetBpFi7mJM1lUN3GFdpg2egxD4lef+DqPoAaDIKyPOCWluZbjLi+cgx4AVcWrQjq6jIbbSv0HKJu4pFBuIStULA96GG+GlJ7oEtpob6SiK5iX7UQwSG6IeKLSg6HWKGQxIWv\\\\\\\/Fv1Lx3ktxkHIIbodYiuXLDJ4Hs9Fk8fq6Bg92WRgIhs1hzkYQXuq8muRHm72Ne8TxEn74ZGjApiZK0DNW3yq6tyshi60WdeBZlntajegPPRwqd0Ik+rjJKccYe1\\\\\\\/aZ0w\\\\\\\/+ZcfZeJ+AD9dc9\\\\\\\/O+v+4ceozSHp0bb8H4am9k98APrnfb7vLMc3qGun7VABdZFS7ihdIrVL0ngiuOhz8FKJF99GYiJGHVUfgpDTFJo1PQjW\\\\\\\/vSNEhReg5w1lcwGs6Y2hyh9iBBF1HTvlTyXTkuZjMli06iBBbIdncLfU0svvJvX9hk9D332cov6XYv\\\\\\\/59F9zEN1+ZFgOX+qUhp1IwPUZqI9fv\\\\\\\/v++i7DKAdq2cCUTZA\\\\\\\/5oH00P2p24PPabYJ88WZqGQIwa5Ps4IrF+jHz+vOVy5b7Lx5s9PQ8r+cv8MJJNDHIpWywGHjCJ8WIxAycQanNyGFoXHcH2nK9mfIVRK6KUJk+gtvTnBgrn1wj9pySQ6eF4WJessugVbbu4DznQ7rDC7kMqMFZuXObAKedfnPUTiFCV4jSYYp3\\\\\\\/buQHSpOlSWTUMJEbnzSQzunJE2dn0kgRTgOiPlfqevGkf\\\\\\\/pzIxijTstL03IcYjkEzzcVo1W0EtExGVUlNF2r+DxFpYfy2zZEt97zBTGQDY7bMTKe8D5wYSlvLz5kuXYsGioxn2CF5sd+pc08KINgLXKkvJCuh0dk+aaRdnZx3D6nPEzS71Q+eGFA5EYsQKPof7X0AvJF0Oyi5USSbM71lN0wnE9YQ3FLdh6zQ0T9Hdbb+8HCKpX4ZdvKCPwUPJWOlBHwCeHdO3LD4K\\\\\\\/NLm9bsPve\\\\\\\/RycB06CbdHsxZgnFjnz45+FK2GGy4H9ba4u\\\\\\\/rCSD8ZaLC0oBKrewtGhqMyK3YXpdGAQJH\\\\\\\/rlvZg3NSStpceWpsjYg0uAUeyFReij2nL3ncm3gFLZt91Pnb7cifw6Ey4BUGcm8XwgwiPLlzGAyA8S3qmzdXc8NXVUfAdM\\\\\\\/n484mgvyewIVPSIymocWt64\\\\\\\/j4b0yaFH9Kvj5Otycia9CWDjlAw8EqOa867v6fH4KMcHgLeZp7FKunIpYDjlr5\\\\\\\/suRLMaJu2+9PAHV36edWiWoAB34Ad9rTTZt8wucfp89DpiAGV+GmBtu6E6QvzRinQ4D8\\\\\\\/\\\\\\\/H705VOLyR+HzzZtGUwEqz72JvwJ28OezjT554pmU7+49dZp0JKSI48QdLnu6Kdp\\\\\\\/xpmhAO2EeAy0iP71pF8Bxf7F9cIiQ\\\\\\\/R\\\\\\\/7RqLMqp5fGm8E30WCkR1ZZUgjulkIz66yoyUb6vUNrJTgoYMJcq7I5m7QQ6Ui8HRMcJv2iYBtyj7SAdG89QuaSvDJoUDjgYMLQK3taD+74memwQSTVpKQeTLtOnlUSxCdr1eClutZdf1p3ePh9vPUqsFNTHIMsmDT+++\\\\\\\/XfW41uHsm+JERERezpazM3J5CO+QOd1vJbu8nw4nUXKw5cHQuwScKxOMuO\\\\\\\/D6Aduxmb\\\\\\\/KE9C3Rs5PhyE4uB9G0HxUi8MHbtLyiUgP\\\\\\\/Gd5Oiw3mY3heYM+KUluSP3YJ0Ld8C2PKN2v\\\\\\\/G++OynELTMraXxsv17oyPwiklGsKULMiHaJ0zXBK2U2noC0wdGuUBPgA\\\\\\\/CFHJyEatifsQzCpUOjOhDNKLW+CgTiuwkDOTVN0yOrj5FIu30iws+\\\\\\\/h8D848mZMSV0QsA17QcJ0hZVD7IyAgU18Q44AVydnd4MoRylrtAUcM0EoTBNWflUN2oQkSbbOyw\\\\\\\/IS7BYQGQa9caEgTscf45S1dbN8ISQHY+lHicJqry+4osJ5g5dPF\\\\\\\/ZyN1v6aVqJAeufPbFfPr4RY47\\\\\\\/5pCnSrxu8Pk4WNXi9smFtJhT6qmdq8NWXUNs4oe+G\\\\\\\/w9jYIdAhlHmwEqJKUqkkoSXtGlj4HYPKgkLnVW\\\\\\\/DU4i0xiV0LHt4fPfvLSqQUXUxnyLicx7cU4D6wcQvcwrJSUGzGgLQl9IKbWoloA5E0SqSkqbn27Fks+vfYaaD61IZGli2boXzibSG\\\\\\\/jjhBeHz7P+Ah6GGm2WGOg+19oAPaZRe93wMUBA11wBZr4n13lU584tV\\\\\\\/x5gIaukUNuZIK2IRUfaBDIjtrQMi2RFqFzhGoz5xTUrbS1ohuzBKXkgwjfDfeyMKobXWjMkDLhhS+wBGQYWO7MvtB4ErwSN3hQkUSTe\\\\\\\/GiEdbyNZEIGy88GsdTNV4hmP3AX3ghNzNRQEmmQUFiVwcxbrguURVlk68IPhaupr3ohU\\\\\\\/NsTdnZdt4Z0Sz8wgujCEK+S0c9tp3KaaNagb7w+CjGuIgxqXh+eftFxVJd0m4fMbS1lc0rBhQnxgqPdQgrQwmzLfTL4i\\\\\\\/A3OUED0kLUzbWIOEzLT4zNmNerqUNiDnRHMqhyuV3n98fbSRsAkTj5hZuPz3bi86sNI+U1\\\\\\\/ZgXeES\\\\\\\/KYiBxIVmieogvmhpIqYZSs4wdub4l80Pr\\\\\\\/psox3AbUrdSl7mAecvisgqyBymeSTtWEyghNE2ZM5V7F6HJwf1LxuFl0u9wSWIUOUrvp0\\\\\\\/AlcWg6LhEQ4sIpOYchUS6eVJ9UTKMx31erD6ZSLRrVA\\\\\\\/2pfw3fWmTZutYZx\\\\\\\/MT5NhNns5pDTvEW6wwpp8a1VuI74ESNQSOg4ykdQLf7AxTY9OSjkhDWQ\\\\\\\/qhZzP08oX7RbCBJ+5LmCzz+\\\\\\\/IdTMCpVtTBoMB3g2Iu9r5dR09G7vSzxIgfKtEV9vmPSPg+4DhitNPJ7d5skaT3lj82byu87MfQSvmc8HOAfRbGzgxF0XiNtvkb6Lg3\\\\\\\/3uVhktT\\\\\\\/oqbRjQGosU1GOlEqwQPZx9\\\\\\\/Z7jdnpu6eNDfmAfDVzeUG9Yjn6VqjNvjT1g+XmzhxdUEYoVPA+3YPjILEdmSt+QPaL6yy5SJcd7xlvmwV64a6hEL83cZlGtD+FwRQdIIlYX6ogRUatlDxH81Q+8FAFewm2LsMI7x+4NVpKiMGQoEthQbbYdTknHfaifo0th4dHL14pfjgdjv5cX+wBDMobEEaLjPW2HPbBjX3E2PZQkJqUeobEaF8OnyOuEbmaeppvvUGiMwzSYLt9VWjAfpWCsA8MtvzHfG5poX4bv5C5\\\\\\\/HI2L8QxoHAAKXsiMX3m7rXaiws1mBx40fzRGsmHdzACnyi3HHoLJfrrfxQjK+SVmkfyVIsd50elJwYlNNMkIQOKWgU8135HVatbq26VZH2xiAjFalR5Kcw6vNl2O9cXNO8RczivNB3kgKGH224m6OzsikbRmiNNBwjZhXtZRQIj6ZU+l1+ukv3rsbR4xp8mefVI4NJZ8u8IBmISAwmNNNBsC1UbZsTl58fGyRPAsHyZQTnGcMP6dP3rVS3aPjwq3BDwmV52HtH\\\\\\\/fceIsaRQYYDnIVNaYmpezalquyB0xCnLc34WPwiWFXBhMElq2wwqgV\\\\\\\/ILSDauB4mWUA0lRdCRGGlm0QD+U0xljcKAs1e466Ncv\\\\\\\/bimvHs2u8BF0uXXVt62yhmF8usdHMGRtjiyPVWk0LfjzWkQM6bz8zvmw9bUV1ifHAheeJU0MEWstgsdpEm1vhXHvwabbCTYkMKkFSRlKgic6h7qDc+9kbXu1QzUQxxsMTveeCkUR0Y4GKudzqjwEMTZO8pV+VndqrvBIiTWguMMrq0yo90kvsQEgw2OVYBhVHy1Ls4DXGl58OJbEU8yrEZJ1DXFEAYUnzQQncmYk3YPV\\\\\\\/kR7g9PzqqJIvggieQqoeaiRfEfAKlUSwTC8F3ippI6T9CfdYooE7xXQ5zZq47DKJid95FOPKkusKTkEzFD0mfTD1oY1yYRuZc7Tvpb8bEKmJyNrpT1A6YgebXV5RVgbZG0WALdyfpdtwKX6VF6uzsOv6AwHDeK81HfB7Tgyks1gCqD4T0ToIolEVrViGjIr7bcubFTjSMEpfDslxPr+Aswx41dypfO1XP8yLOtdjDANO03Rmk6lsyQTpD1Qb+r\\\\\\\/OWESbhQK4ecZq0i2oBLLfszblyh4lbkzjL0zqTad3ysFHpURXBgm7c0bbk9pnQdFRF96bnthIaBHH+khHhnPjW0OmVPCuTOHACW2w9bJFoDMXWC\\\\\\\/gmHjsepe6\\\\\\\/ZxfEEcPS\\\\\\\/7wMCb4foXmWuEsJ46w6bxnRyQY3KJZe81CuX3dja2adcH4Rnr3CINtZLTQEVbMPUvbiWX6RIF1ptA2IyFraeVKb7VoTnt8hEBKLk9SUGksH09b1IyP6rLfYSQ4sC9Gl8PC+7fs5oyqOZEHkSfeQC0sLJAdFB8DLF+QIjyW\\\\\\\/UGFrupxuZf9TC9neftmADGXtbqzzVpgv97G1BzqD5oaVAaxHYW20ei6a2jtSfEOkAkdGVU5Le3SF4gRh+Bw3EhUIWzRHROioVRiK5HcUc+4x1YIeU8vEG98zb5lP5wGJyD7vnX4aqIghlJKQtLyxQch4aRGZy7+8W\\\\\\\/KsHa1o+2J2mQjo+YfmjVU8asiout+pDSA==\\\u0022,\\\u0022iv\\\u0022:\\\u00224lcBpl0r5RrG3DIeVo3U+w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228360075dd7eee485d7b41f4b4a0cd172f4e26c56151ad97b9d2e84d814b65fcc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222RqhCuYtfTNh4x5NPCVfHC5sxyBujH1MyxamHKCjCKa3+Kc7enirOGqKzb1ff4h\\\\\\\/Llg9VLbSmak489OBx9LxTcRcYtt1M+IFUuzf8UFf2oZy+unXr4sK58TT+kaft+n6V3JLP0XG+1XXiUSdC+R5FwY2OepNQNyjHDCT4y8jIniFL3IlcjfU0WVc86rPMa3OhohR5uOVMA6RZWEEX1w+Dm9QnT6bHfWohtPu33b6yl32QPM34sim2t4tu14dVs1aYs7d9ex9nvHbOA\\\\\\\/00wnlnfJspJmGq2bjAl2K4iuhuzn5I1iltd4JSChhoqePagHU0i2eajwqfMbYVl4zUdx99SkPDM2A8oRPhvVzsX5207NPJF64CDtt1ikW9ikIYJYeIrr+RCvQ593iN3K1+KGEyR+8PvsmmIYDiSfiIgZkUKyz2fXfriMH75mcjjMdS8D9Ah3WLjh6DKgMAseJjq4vTI5YE5fFslDCCl+svSfHj\\\\\\\/R0C+jywYZXVzq70mY1lyh3tZYQ7m6RlLgDrAcS3vyEPuHqX4P5+ZGFgmgNSpJ564E1MTnGlOmvdcStqkYLGkGZHXJuokVq8D0KtWDqpYzzISj\\\\\\\/ElIrz4d0aQDNqoh21Dol8UpC0eU3P+vV7OQUbFyJc5YIvtW9EbUyjNIbM++phcc+k5+8SyaD4AGAXS40K2wC\\\\\\\/ZRSLLfMwQLyRHayKIcXjpF5WoX3nw6rBw0Z6CbrfTdwQUL4gSLK\\\\\\\/st\\\\\\\/ksFB\\\\\\\/1LZplDl2zXVlFgaQoocmjKWmZrWly+KX+OKCA9XSntcVD+wJXc7eMevkNHkuDAaa2tobbSvDnvRaRZoyq3m1ccgJIuUcYSylOq2pPj2eFftv4Md8WzCtn1X2\\\\\\\/uXcl2nMWGL3c4+wPV\\\\\\\/n4sJKaCpGpjgi7wGhhvP9VZDkI8d2U7zTrVymS3X1bvpjozWvqRz2hiiJO\\\\\\\/E2Alri9+sqeU5gEBvJ8a0FrWDli6ViJkRh5Nn+7f7+cG1lPypA81hoCltuj4F9CUyYqu1XyECBavajjo7a7Rs86+DE6Z4\\\\\\\/zsPOmX2vQKiYMLn3WZN60PgkqtJFw55m0hnzatZM+wpNuDQKcgrCth2GXHh0ikhyHYfWplLj83QtCwehMMzozz8aGHHFi7O4qvjNztM0f\\\\\\\/Fra+gRtsYq2g\\\\\\\/5Me\\\\\\\/XXz1nn\\\\\\\/8TEZBU3oDQkwCbyGgHRiK6wkiKcAzV8V3i9bNTSa5x+zF84tAQDGX9vRs\\\\\\\/a7DPKJScse8v\\\\\\\/KMXQVNUGxjY5l6mUYKa35xTU4tCFHHCEHdHLjTF\\\\\\\/94hYYWQGV9+8co6xtCvMXV\\\\\\\/BLDzYEdyuah\\\\\\\/7ObxVylBhmZuSLvx2c5iSvhog\\\\\\\/BjFyAPzBFVFTljQbiE9ihVMGWLQWFNY1ko28UJwgd80M33o09Bk3kwkOA7VUY7OsjspEql7Ld+Z3N4EHsr06X5SlgofEVGiT9DUQ24a8+NpYhtxCaG7\\\\\\\/6op4wClq2\\\\\\\/lxs88PZmI4FPBqHcPUm4qrGfaiwHlzJUXB57EN8Y49CUI8\\\\\\\/UT0acsky0CPLDNl4oh6eZmCPAJAsxaVCQqbWqxrLpDAp1Tk+7rTK1vmuKwFNBbPat\\\\\\\/NlUfWbojYxkKujRMpLknbs0wfxLNOhNPvmjKkMcrul+rHUbPJAz\\\\\\\/gBTXvshpHEYbjOi7Gx2GUDhxozhSf7tLOBu\\\\\\\/AsWUp6y37jE8KfcB8OSCGiPQ0c+fRjReQtIdJG6y5+mpYSZEp8BpFj2jyoaHL74hDLg6XPagiYmaVbxNIEAdYO91G4RUdkporGqtA1IUzxzV\\\\\\\/43XshatbVZPI8Ow9WEXhelDfR7WpoazuUie72hZYWVustEhXrSH7aw\\\\\\\/npx3nyGwmUL+hKBn3U\\\\\\\/ZL0CiumHOtZJQ22CsP7to4zjjXb8g\\\\\\\/OYKVAgvNP6u2BP3J4xxBYnIkylMYTyNOJCyzAF\\\\\\\/FN4oMkTvr9zumA66xHxHOiq83XCPZ2F2\\\\\\\/LtofpeCU2ZChlEH3XJFGShO2l4JmS9G7y5ydTOwFzKA0xqGszZB2O7Q==\\\u0022,\\\u0022iv\\\u0022:\\\u00221h\\\\\\\/EvxfRLoW0+G7ZcTqcwQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224a21644c8c1bfe9944925c839b1763c6fcdc40f311c0fd5e0b9a56a31d225b00\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459192 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/417" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vrQ6NZrMVV3Gs5yFlTRGEIxdPSNTCMQxNFmT-RjOo2I" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/417" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GjUFHX5MfRHXXbG6oHQ_vLN6frT_1SzuwTlHOCdyMHA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/411" title="Amino Acid Substitutions in the Caenorhabditis elegans RNA Polymerase II Large Subunit AMA-1/RPB-1 that Result in α-Amanitin Resistance and/or Reduced Function" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/6/427" title="Partitioning Transcript Variation in Drosophila: Abundance, Isoforms, and Alleles" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459192" id="node459192" data-pisa="ggg;1/6/417" data-pisa-master="ggg;g3.111.001057" data-apath="/ggg/1/6/417.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase <em>Escherichia coli</em> Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ashley B. Williams</span>, <span class="highwire-citation-author" data-delta="1">Kyle M. Hetrick</span> and <span class="highwire-citation-author" data-delta="2">Patricia L. Foster</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">417-426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001057 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459192" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ashley B. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Computational Biology, University of Southern California, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashley%2BB.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAshley%2BB.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kyle M. Hetrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BM.%2BHetrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hetrick%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BM.%2BHetrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricia L. Foster </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BL.%2BFoster%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foster%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BL.%2BFoster%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plfoster@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">plfoster@indiana.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/6/417" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459192" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/417.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459192" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/417.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459192" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/417.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459192" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/6/417.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459192" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">As nutrients are depleted and cell division ceases in batch cultures of bacteria, active processes are required to ensure that each cell has a complete copy of its genome. How chromosome number is manipulated and maintained in nondividing bacterial cells is not fully understood. Using flow cytometric analysis of cells from different growth phases, we show that the Holliday junction–processing enzymes RuvABC and RecG, as well as RecBCD, the enzyme complex that initiates DNA double-strand break repair, are required to establish the normal distribution of fluorescent peaks, which is commonly accepted to reflect the distribution of chromosome numbers. Our results reveal that these proteins are required for the proper processing of chromosomes in stationary phase.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dna-repair" class="hw-term hw-article-keyword hw-article-keyword-dna-repair" rel="nofollow">DNA repair</a></li><li class="kwd"><a href="/keyword/stationary-phase" class="hw-term hw-article-keyword hw-article-keyword-stationary-phase" rel="nofollow">stationary phase</a></li><li class="kwd"><a href="/keyword/homologous-recombination" class="hw-term hw-article-keyword hw-article-keyword-homologous-recombination" rel="nofollow">homologous recombination</a></li><li class="kwd"><a href="/keyword/genome-stability" class="hw-term hw-article-keyword hw-article-keyword-genome-stability" rel="nofollow">genome stability</a></li><li class="kwd"><a href="/keyword/double-strand-break-repair" class="hw-term hw-article-keyword hw-article-keyword-double-strand-break-repair" rel="nofollow">double-strand break repair</a></li></ul><p id="p-4">The molecular processes by which bacterial cells replicate, repair, recombine, and partition their chromosomes during exponential growth have been extensively studied, but little is known about how DNA is manipulated during stationary phase when cells are not dividing. It is usually assumed that DNA metabolism ceases when cell division stops; however, at least three reports have documented that tritium-labeled thymidine can be incorporated into the genomic DNA of nondividing <em>Escherichia coli</em> cells (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Boe 1990</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Grivell <em>et al.</em> 1975</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Tang <em>et al.</em> 1979</a>). Evidence that DNA synthesis and recombination continue in stationary-phase cells is also provided by a phenomenon known as “adaptive mutation,” during which mutations occur in the absence of cell division. In the most-studied case, reversion to lactose utilization of <em>Escherichia coli</em> strain FC40, adaptive mutation requires recombination functions, proteins for double-strand break repair, and the DNA damage-induced DNA polymerase IV (Pol IV) (reviewed in <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Foster 2007</a>). Taken together, these results suggest that DNA-related molecular processes continue into stationary phase, even in the absence of bulk chromosomal replication.</p><p id="p-5">As cells enter stationary phase, a complex cascade of events prepares them to survive famine conditions. Genes required for exponential growth are downregulated, and genes required for survival during starvation conditions are upregulated (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Patten <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rahman <em>et al.</em> 2006</a>). The chromosome is also reorganized and compacted, which can contribute to gene regulation but also protects the DNA from environmental and endogenous DNA-damaging agents [for examples, see <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Frenkiel-Krispin <em>et al.</em> (2004)</a> and <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nair and Finkel (2004)</a>]. These changes in chromosome organization are largely due to several DNA binding proteins, some of which are upregulated in stationary-phase cells (reviewed in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dorman 2009</a>). Interestingly, during the same period that this chromosomal reorganization occurs, transcription of the <em>dnaN</em> and <em>recF</em> genes, which are adjacent in a complex operon, is activated from promoters that are not used during exponential growth (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Villarroya <em>et al.</em> 1998</a>). <em>dnaN</em> encodes the replication processivity factor β, and <em>recF</em> encodes a protein involved in recombinational DNA repair. The increased expression of these proteins as cells enter stationary phase suggests that recombination processes continue to be important after genomic replication has ceased.</p><p id="p-6">Holliday junctions (HJ) are mobile four-stranded DNA structures that are formed during homologous recombination. To restore two intact double-stranded DNA molecules, HJs must be processed and resolved. <em>E. coli</em> has two pathways for processing HJs: the RuvABC resolvasome and the RecG translocase. RuvA and RuvB bind to HJs, hydrolyze ATP, and migrate the junction along the DNA. An interaction with RuvA stimulates RuvC to cleave the DNA, resolving the junction (reviewed in <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">West 1997</a>). RecG can also bind and migrate HJs; however, unlike the RuvABC resolvasome, RecG does not have an intrinsic ability to resolve the structure (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mcglynn and Lloyd 1999</a>). While a need for HJ processing in stationary-phase cells has not been explicitly shown, RuvABC promotes and RecG inhibits stationary-phase adaptive mutation (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Foster <em>et al.</em> 1996</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Harris <em>et al.</em> 1996</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Williams and Foster 2007</a>).</p><p id="p-7">During normal cell growth, double-strand breaks can be caused by reactive oxygen species formed by normal cellular metabolism or failure of replication forks (for example, see <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Michel <em>et al.</em> 2007</a>). Repair of DNA double-strand breaks via the RecBCD pathway leads to the formation of Holliday junctions. When a double-strand break occurs, the RecBCD complex binds to and processes the double-strand end, producing a 3′ single-strand overhang. RecA then catalyzes invasion of this strand into a homologous duplex, ultimately leading to the formation of one or two HJs (reviewed in <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dillingham and Kowalczykowski 2008</a>). RecBCD is also required for stationary-phase adaptive mutation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Harris <em>et al.</em> 1994</a>), but to date, no other requirement for RecBCD in stationary-phase cells has been reported.</p><p id="p-8">Flow cytometric analysis of cell populations in batch cultures revealed that stationary-phase cells might contain one, two, four, or even eight chromosomes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Åkerlund <em>et al.</em> 1995</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boye and Løbner-Olesen 1991</a>). Because DNA recombination activities are required for stationary-phase adaptive mutation, we sought to determine whether these activities also function in the establishment and/or maintenance of higher-order genomic structures. Specifically, we examined whether the RuvABC resolvasome, the RecG HJ translocase, and the RecBCD double-strand break repair machinery are required to generate the distribution of chromosome numbers that normally arises during early- to mid-stationary phase. The results presented here reveal that RuvABC, RecG, and RecBCD ensure proper chromosome processing during stationary phase.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Bacterial strains</h3><p id="p-9">All bacterial strains used in this study are <em>Escherichia coli</em> K-12 derivatives and are described in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460921/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460921/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Strains used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Media and growth conditions</h3><p id="p-10">Cells of each strain were cultured in Luria-Bertani (LB) broth (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Miller 1992</a>) at 37° with aeration. After growth overnight, cultures were diluted 1:1000 into 25 mL fresh LB broth with no antibiotics in 125 mL flasks and grown at 37° with shaking. The optical density (OD<sub>600</sub>) was measured at each time point by removing appropriate samples of the cultures. In a separate experiment, samples were taken at comparable time points, diluted appropriately, and plated on LB 15% agar medium to determine the total number of colony-forming units (CFU).</p></div><div id="sec-4" class="subsection"><h3>Analysis of cellular DNA content by flow cytometry</h3><p id="p-11">At each time point, 0.5 mL samples were removed from the 25 mL LB broth cultures and added to 4.5 mL of 78% ice-cold ethanol to produce a final concentration of 70% ethanol. The fixed cells were stored at −20° until analysis. For analysis, the cells were first rinsed twice with 10 mL of 1X phosphate buffered saline (PBS) at 4°, then stained with propidium iodide in a solution of 0.2 mg/mL RNase (Sigma, R4875), 0.01% Triton X-100 (Thermo Fisher Scientific, Inc., Waltham, MA), and 3 μ<span class="sc">m</span> propidium iodide (Sigma, P4864) in 1X PBS.</p><p id="p-12">DNA content was analyzed using a FACSCalibur flow cytometer (BD Biosciences, San Jose, CA). Approximately 50,000 cells were analyzed at a speed of 300 to 900 cells per second. The FL2-width (FL2-W) indicates relative cell size, and FL2-area (FL2-A) indicates total propidium iodide fluorescence. To examine how cell size might affect the fluorescence profiles, data analyses including different cell populations (gatings) were done. The overall fluorescence profiles were unaffected by these different gatings, so the histograms shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a>–<a id="xref-fig-7-1" class="xref-fig" href="#F7">7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/001057SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS5.pdf">Figure S5</a> represent the entire data set for each sample. Data analysis was performed using the FlowJo Version 8 for Mac (Tree Star, Inc., Ashland, OR).</p></div><div id="sec-5" class="subsection"><h3>Measurement of DNA synthesis</h3><p id="p-13">Overnight cultures were diluted by a factor of 10<sup>5</sup> into 50 mL fresh LB broth in 125 mL flasks and grown at 37° with shaking. The OD<sub>600</sub> was monitored throughout the experiment. Samples were collected for CFU determination and 5-ethynyl-2′-deoxyuridine (EdU) labeling at 6.5–8.5 hr (exponential phase; OD<sub>600</sub> = 0.6 ± 0.1), 8.5–11 hr (stationary phase; OD<sub>600</sub> = 1.5 ± 0.1), and 21 hr (late-stationary phase; OD<sub>600</sub> = 1.8 ± 0.04) after inoculation. The times of sampling differed from strain to strain depending on growth rate. Cells were labeled with EdU using the Click-iT EdU Alexa Fluor 488 Azide Imaging Kit (Molecular Probes, Eugene, OR; #C10337) as in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ferullo <em>et al.</em> 2009</a>. Briefly, 2 mL aliquots of culture were added to tubes with 24 μL of 10 m<span class="sc">m</span> EdU and incubated for 15 min at 37° with shaking. Cells were fixed with 90% methanol and stored at 4° for 3–8 days. Repeated experiments confirmed that storage time did not affect results. For analysis, cells were washed with PBS, permeabilized in a 0.5% Triton X-100/PBS solution for 30 min at room temperature, washed again with PBS, suspended in Click-iT reaction mixture, and incubated in the dark for 30 min at room temperature. Cells were then washed with reaction rinse buffer and suspended in PBS. Fluorescence microscopy was performed using a Nikon 80i microscope with a FITC HYQ filter (Nikon Instruments, Inc., Melville, NY). Phase-contrast and fluorescence images were viewed and captured as described below. The number of CFUs was determined as described above.</p></div><div id="sec-6" class="subsection"><h3>Phase-contrast microscopy</h3><p id="p-14">Phase-contrast microscopy of unfixed cells was performed using a Nikon 80i microscope (Nikon Instruments). The phase images were viewed using a Nikon Plan Apo 100X objective (Nikon Instruments), and the images were captured using a Photometrics Coolsnap HQ2 camera (Photometrics, Tuscon, AZ) and Metamorph image software (Molecular Devices, Sunnyvale, CA). Representative fields of view are shown in the figures.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Growth rates of recombination-defective mutant strains</h3><p id="p-15">As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, the growth rates of the <em>recG258</em>, <em>ruvA60</em>, <em>recB21</em>, and <em>recG258 ruvA60</em> mutant strains differed little from that of the wild-type strain. Based on a least-squared fit of the linear parts of the curves in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, the growth rates of the mutant strains differed from that of the wild-type strain by no more than 15%. All the mutant strains had somewhat longer lag phases than the wild-type strain; the longest was the lag phase of the <em>recG258 ruvA60</em> double-mutant strain, which was about twice that of the wild-type strain. All of the strains reached comparable final optical densities. In the discussions below, we consider the 7–9 hr time period to be the entry into stationary phase and all times after 11 hr to be stationary phase. As measured by CFUs, after about 40 hr, the culturable cell number began to slowly decline, so that by 80 hr after inoculation only 10–30% of the cells could produce colonies on LB agar (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/417/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Growth parameters and viability of wild-type and recombination-defective strains over time. (A) Optical densities measured at a wavelength of 600 nm (OD600) are plotted against the time (in hours) after inoculation. Samples were removed from the cultures at each time point marked. (B) At each time point, samples were collected from cultures started by diluting overnight cultures by a factor of 105 into fresh LB medium; the number of colony-forming units was determined by appropriate dilution and plating on LB agar. The number of cells per milliliter is plotted on a logarithmic scale against the time (in hours) after inoculation. Diamonds = FC36 (wild-type); triangles = FC457 (recG258); circles = FC484 (ruvA60); squares = FC400 (recB21); dashes = FC513 (recG258 ruvA60)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949640019" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Growth parameters and viability of wild-type and recombination-defective strains over time. (A) Optical densities measured at a wavelength of 600 nm (OD&lt;sub&gt;600&lt;/sub&gt;) are plotted against the time (in hours) after inoculation. Samples were removed from the cultures at each time point marked. (B) At each time point, samples were collected from cultures started by diluting overnight cultures by a factor of 10&lt;sup&gt;5&lt;/sup&gt; into fresh LB medium; the number of colony-forming units was determined by appropriate dilution and plating on LB agar. The number of cells per milliliter is plotted on a logarithmic scale against the time (in hours) after inoculation. Diamonds = FC36 (wild-type); triangles = FC457 (&lt;em&gt;recG258&lt;/em&gt;); circles = FC484 (&lt;em&gt;ruvA60&lt;/em&gt;); squares = FC400 (&lt;em&gt;recB21&lt;/em&gt;); dashes = FC513 (&lt;em&gt;recG258 ruvA60&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/417/F1.medium.gif" width="316" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/417/F1.medium.gif" width="316" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/417/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/417/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461008" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-16" class="first-child">Growth parameters and viability of wild-type and recombination-defective strains over time. (A) Optical densities measured at a wavelength of 600 nm (OD<sub>600</sub>) are plotted against the time (in hours) after inoculation. Samples were removed from the cultures at each time point marked. (B) At each time point, samples were collected from cultures started by diluting overnight cultures by a factor of 10<sup>5</sup> into fresh LB medium; the number of colony-forming units was determined by appropriate dilution and plating on LB agar. The number of cells per milliliter is plotted on a logarithmic scale against the time (in hours) after inoculation. Diamonds = FC36 (wild-type); triangles = FC457 (<em>recG258</em>); circles = FC484 (<em>ruvA60</em>); squares = FC400 (<em>recB21</em>); dashes = FC513 (<em>recG258 ruvA60</em>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Changes in cell size of recombination-defective strains</h3><p id="p-17">Loss of recombination functions can induce the SOS response, leading to cell filamentation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chua <em>et al.</em> 1993</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lloyd and Buckman 1991</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mccool <em>et al.</em> 2004</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Witkin 1976</a>). We used flow cytometric analysis of fixed and propidium iodide–stained cells to examine the morphological heterogeneity of the wild-type (FC36) and isogenic mutant strains during the course of our experiments. Approximately 50,000 cells of each strain were analyzed, and each cell is represented by a dot in the plots shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. The magnitude of the FL2-W parameter is determined by the amount of time each cell remained in the path of the laser beam and is thus a measure of cell size. In each panel of <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, the FL2-W value of each cell (vertical axis) is plotted against its relative fluorescence intensity (horizontal axis). Plots from several time points during the course of incubation are presented. As the plots show, the great majority of the cells in all the cultures were approximately the same size as the wild-type cells, but the cultures of the mutant strains also contained larger, presumably filamenting cells. With the exception of the 4 hr time points, the distributions of cell sizes of the <em>recG258</em> and <em>recB21</em> mutant strains were most similar to that of the wild-type strain, whereas the <em>ruvA60</em> and <em>recG258 ruvA60</em> mutant strains showed the greatest increase in the proportion of larger cells. These results were confirmed by the phase-contrast images in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figures 3</a>–<a id="xref-fig-7-2" class="xref-fig" href="#F7">7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS5.pdf">Figure S5</a>, which show that the morphology of the wild-type strain was constant throughout the course of the experiment but that the cultures of the mutant strains contained both normal and filamenting cells. The reason for the large variation in cell size in the <em>recB21</em> mutant strain 4 hr after inoculation is not clear, but it could be due to double-strand breaks that formed in the stationary-phase cells that were used as the inoculum; as the cells began to divide, these breaks would then induce the SOS response and filamentation. Nonetheless, even in the strains mutant for double-strand break repair, HJ processing, or both, the majority of cells were of normal size and morphology during the course of these experiments. In <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3</a>–<a id="xref-fig-7-3" class="xref-fig" href="#F7">7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS5.pdf">Figure S5</a> presenting flow cytometry data, the histograms include the entire cell populations, as omission of the larger cells had no noticeable impact on the final plots.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/417/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of recombination defects on cell size. The relative FL2-W values (cell size) are plotted against the relative FL2-A values (total propidium iodide fluorescence) in arbitrary units. Increasing values on the Y-axis indicate larger cells, and increasing values on the X-axis indicate greater total propidium iodide fluorescence intensity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949640019" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effects of recombination defects on cell size. The relative FL2-W values (cell size) are plotted against the relative FL2-A values (total propidium iodide fluorescence) in arbitrary units. Increasing values on the Y-axis indicate larger cells, and increasing values on the X-axis indicate greater total propidium iodide fluorescence intensity.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/417/F2.medium.gif" width="337" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/417/F2.medium.gif" width="337" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/417/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/417/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461018" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-18" class="first-child">Effects of recombination defects on cell size. The relative FL2-W values (cell size) are plotted against the relative FL2-A values (total propidium iodide fluorescence) in arbitrary units. Increasing values on the Y-axis indicate larger cells, and increasing values on the X-axis indicate greater total propidium iodide fluorescence intensity.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/417/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Populations of cells with distinct fluorescence intensities develop during growth and stationary phase of the wild-type strain. Propidium iodide&#x2013;stained cells of the wild-type strain (strain FC36) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949640019" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Populations of cells with distinct fluorescence intensities develop during growth and stationary phase of the wild-type strain. Propidium iodide&#x2013;stained cells of the wild-type strain (strain FC36) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/417/F3.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/417/F3.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/417/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/417/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460915" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-19" class="first-child">Populations of cells with distinct fluorescence intensities develop during growth and stationary phase of the wild-type strain. Propidium iodide–stained cells of the wild-type strain (strain FC36) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000× magnification).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Development of the wild-type pattern of fluorescence intensities during exponential and early- to mid-stationary phase requires HJ processing</h3><p id="p-20">As shown in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>, as wild-type cells progressed through stationary phase, populations of cells with different fluorescence intensities developed. Previous studies have identified the two separate fluorescent peaks that develop in LB-grown cultures early in stationary phase to consist of cells containing two (lower fluorescence intensity) or four (higher fluorescence intensity) chromosomes (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Åkerlund <em>et al.</em> 1995</a>). For the remainder of this article, the low fluorescence intensity peak is referred to as peak 1, and the higher fluorescence intensity peak is referred to as peak 2. In our experiments, the wild-type strain contained predominantly cells in peak 2 until 28 hr postinoculation, and then a population of cells in peak 1 developed between 28 and 44 hr postinoculation. The fluorescence profile of our strain at 44 hr postinoculation was similar to that obtained at 28 hr postinoculation by Åkerlund and Colleagues (1995), except our cultures had a higher proportion of cells in peak 1. In addition, we did not see the small peak at even higher intensity that <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Åkerlund <em>et al.</em> (1995)</a> identified as cells with eight chromosomes. These differences are probably due to media composition: we used LB whereas <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Åkerlund <em>et al.</em> (1995)</a> used LB plus glucose. The phase-contrast images in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a> show that up to 44 hr postinoculation, the morphology of wild-type cells did not vary noticeably.</p><p id="p-21">As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, the fluorescence profiles of a strain defective for the RecG translocase were similar to those of the wild-type strain during growth and early-stationary phase. However, the <em>recG</em> mutant strain was slow to develop the large peak 1 cell population that was present in the wild-type population by 36 hr postinoculation. <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a> shows that some filamentous cells were present in the culture of the <em>recG258</em> mutant strain; however, cells of normal size and shape dominated the culture at all time points sampled.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/417/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Holliday junction&#x2013;migrating protein RecG is required for the development of normal fluorescent population dynamics during stationary phase. Propidium iodide&#x2013;stained cells (strain FC457) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949640019" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The Holliday junction&#x2013;migrating protein RecG is required for the development of normal fluorescent population dynamics during stationary phase. Propidium iodide&#x2013;stained cells (strain FC457) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/417/F4.medium.gif" width="440" height="274"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/417/F4.medium.gif" width="440" height="274"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/417/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/417/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-22" class="first-child">The Holliday junction–migrating protein RecG is required for the development of normal fluorescent population dynamics during stationary phase. Propidium iodide–stained cells (strain FC457) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000× magnification).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">To further examine the role of HJ processing, we analyzed a strain carrying the <em>ruvA60</em> mutant allele that inactivates the RuvABC HJ resolvasome. As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>, the fluorescence profiles of the <em>ruvA60</em> mutant strain were similar to those of the wild-type strain up to 28 hr postinoculation; however, the <em>ruvA60</em> mutant strain failed to develop the peak 1 population by 44 hr postinoculation. The phase-contrast images in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a> show that, like the <em>recG258</em> mutant strain, the population of the <em>ruvA60</em> mutant strain contained filamenting cells, although again, most cells were of normal size and shape.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/417/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Holliday junction&#x2013;processing RuvABC complex is required for normal fluorescent population dynamics during growth and in stationary phase. Propidium iodide&#x2013;stained cells (strain FC484) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949640019" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The Holliday junction&#x2013;processing RuvABC complex is required for normal fluorescent population dynamics during growth and in stationary phase. Propidium iodide&#x2013;stained cells (strain FC484) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/417/F5.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/417/F5.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/417/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/417/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460953" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-24" class="first-child">The Holliday junction–processing RuvABC complex is required for normal fluorescent population dynamics during growth and in stationary phase. Propidium iodide–stained cells (strain FC484) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000× magnification).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">In a <em>recG258 ruvA60</em> double-mutant strain, HJs cannot be processed. As shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>, the distributions of fluorescence intensities at all time points for this strain were very different from the wild-type strain. A peak 1 population appeared during exponential phase (as well as what appears to be a peak of even lower intensity that could correspond to cells with one chromosome), briefly disappeared as the cells entered stationary phase, and then reappeared at 13 hr postinoculation, a time earlier than when a similar population appeared in the wild-type or the single mutant strain. However, the peak 1 population did not become dominant, even in late-stationary phase, suggesting that HJ processing is required for cells to enter this population. The morphologies of cells are shown at sample time points in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/417/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of both Holliday junction&#x2013;processing pathways severely alters fluorescent population dynamics during growth and in stationary phase. Propidium iodide&#x2013;stained cells (strain FC513) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949640019" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Loss of both Holliday junction&#x2013;processing pathways severely alters fluorescent population dynamics during growth and in stationary phase. Propidium iodide&#x2013;stained cells (strain FC513) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/417/F6.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/417/F6.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/417/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/417/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460896" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-26" class="first-child">Loss of both Holliday junction–processing pathways severely alters fluorescent population dynamics during growth and in stationary phase. Propidium iodide–stained cells (strain FC513) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000× magnification).</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/417/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of RecB alters fluorescent population dynamics during growth and in stationary phase. Propidium iodide&#x2013;stained cells (strain FC400) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-949640019" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Loss of RecB alters fluorescent population dynamics during growth and in stationary phase. Propidium iodide&#x2013;stained cells (strain FC400) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000&#xD7; magnification).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/417/F7.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/417/F7.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/417/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/417/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460884" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-27" class="first-child">Loss of RecB alters fluorescent population dynamics during growth and in stationary phase. Propidium iodide–stained cells (strain FC400) were analyzed by flow cytometry at the indicated time points. (A) The numbers of cells (Y-axis) are plotted against their relative fluorescence intensities (X-axis). The histograms from the time points indicated are aligned along the Z-axis. (B) Phase-contrast micrographs of unfixed cells from the time points indicated (1000× magnification).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Development of the wild-type pattern of fluorescence intensities during exponential and early- to mid-stationary phase requires the enzyme that initiates double-strand break repair</h3><p id="p-28">HJs are formed during repair of double-strand breaks by the RecBCD complex. To test whether RecBCD was required for changes in fluorescence, we analyzed the distributions of fluorescence intensities in the population of the <em>recB21</em> mutant strain at various points during incubation. As shown in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7A</a>, the fluorescence profiles of the <em>recB21</em> mutant strain were very similar to those of the <em>recG258 ruvA60</em> double-mutant strain: peak 1 appeared during exponential phase, briefly disappeared as the cells entered stationary phase, and then reappeared but never became dominant. This result suggests that double-strand break repair is required for the development of the peak 1 population observed in wild-type cultures. As shown in <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a>, few filamentous cells were observed in the <em>recB21</em> culture in stationary phase. The lack of filamentation indicates that the SOS response was not induced in this strain to the extent that it was in the other mutant strains tested, a result previously reported by <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chua <em>et al.</em> (1993)</a>. This could be because the RecA loading activity of RecBCD is required to produce the SOS-inducing signal, and this function cannot be performed by the alternative double-strand break repair enzyme RecFOR (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ivančić-Baće <em>et al.</em> 2006</a>).</p></div><div id="sec-12" class="subsection"><h3>Changes in fluorescence populations continue late into stationary phase</h3><p id="p-29">The fluorescence profiles of all of the strains tested were dynamic even to 44 hr postinoculation or approximately 30 hr after the onset of stationary phase. To determine whether further changes in the population distributions occurred, we continued the analysis to 72 hr postinoculation, although these results should be considered with caution as a large proportion of the cells in each culture were not culturable at these late time periods (see <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). The fluorescence profile of the wild-type strain was stable from 44 hr to 72 hr postinoculation, and that of the <em>recG258</em> mutant strain was similar to wild-type but slightly slower to stabilize (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS2.pdf">Figure S2</a>). In each of the other mutant strains, a large population of cells in peak 1 eventually developed, but these peaks developed slowly, particularly in the <em>ruvA60 recG258</em> double-mutant strain (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS5.pdf">Figure S5</a>).</p></div><div id="sec-13" class="subsection"><h3>Ongoing DNA replication does not explain variations in fluorescence intensity</h3><p id="p-30">To determine whether ongoing DNA synthesis in the recombination mutant strains could explain our results, we tested whether these strains incorporate EdU, a thymidine analog, into their DNA during stationary phase. Incorporated EdU can be detected by a cycloaddition reaction with a fluorochrome-tagged azide, yielding a fluorescent product (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Ferullo <em>et al.</em> 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Salic and Mitchison 2008</a>). At the appropriate growth stage, cells were incubated briefly with EdU to allow incorporation; they were then fixed, reacted with the Click-iT reagent, and imaged using fluorescence microscopy.</p><p id="p-31">Images of cells in exponential phase revealed that greater than 90% of the cells of the wild-type, <em>recG</em>, <em>recB</em>, and <em>ruvA</em> mutant strains contained foci, whereas only 74% of the cells of the <em>ruvA recG</em> double-mutant strain contained foci (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1/FigureS6.pdf">Figure S6</a>). These results indicated that, as expected, most exponential-phase cells were synthesizing DNA, although fewer were in strains defective in double-strand break repair and HJ processing.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460956/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460956/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Proportions of cells with foci in exponential, stationary, and late-stationary phases</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">Images of cells in early-stationary phase revealed substantial decreases in the numbers of cells that could incorporate EdU (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Less than 20% of the cells of the wild-type, <em>recB</em>, and <em>recG</em> mutant strains contained foci, and only 1–2% of the cells of the <em>ruvA</em> mutant and <em>ruvA recG</em> double-mutant strains contained foci. These results indicate that DNA synthesis was mostly completed by early-stationary phase.</p><p id="p-34">No fluorescent foci were detected in any cells exposed to EdU after about 12 hr into stationary phase, even though nearly all of the cells were culturable (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). These results indicate that DNA synthesis, at least at the level required for detection by EdU incorporation, had ceased in all cells by this time. An alternative is that cells in late-stationary phase cannot incorporate EdU; however, this explanation is unlikely as previous work has shown that such cells can incorporate tritium-labeled thymidine (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Tang <em>et al.</em> 1979</a>).</p><p id="p-35">While the most severe abnormalities in DNA content in stationary-phase cells, as detected by fluorescence profiles, were seen in the <em>recB21</em> and <em>recG ruvA</em> mutant strains, these strains had different amounts of EdU incorporation in stationary phase (17% and 1%, respectively). Thus, it is unlikely that ongoing DNA synthesis in stationary phase explains the differences in fluorescence profiles that we observed among the mutant populations.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-36">The data described in this study reveal new details about how <em>E. coli</em> cells maintain their genomic organization during various phases of growth. Using flow cytometric analysis, we showed that in cultures of wild-type cells, two primary cell populations with different amounts of DNA develop as the cells enter stationary phase (peak 1 and peak 2 populations); these populations correspond to those previously reported to contain two and four chromosomes (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Åkerlund <em>et al.</em> 1995</a>). In our experiments, these distinct populations did not develop in cultures of the wild-type strain until 36–44 hr postinoculation, which was 18–26 hr after the onset of stationary phase. These times are several hours later than reported by <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Åkerlund <em>et al.</em> (1995)</a>, most likely because our cultures were grown in the commonly used LB medium, whereas in <a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Åkerlund <em>et al.</em> (1995)</a>, the cultures were grown in the richer LB plus glucose medium.</p><p id="p-37">Our data show that the development of the typical biphasic population distribution was inhibited or delayed in cultures of strains mutant for HJ processing or double-strand break repair. A striking aspect of these results is that these distributions continued to develop throughout stationary phase, even up to 72 hr postinoculation. The <em>recG258 ruvA60</em> double-mutant strain, which is unable to process HJs, was the most defective in developing distinct peak 1 and peak 2 populations (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). HJs are an intermediate in the repair of double-strand breaks by the RecBCD pathway. That loss of the RecBCD enzyme, which initiates this pathway, had a phenotype similar to loss of HJ processing (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7</a>) strongly suggests that the repair of double-strand breaks in stationary phase is required for the development of the populations detected by cytometric analysis.</p><p id="p-38">In our experiments, the effects of inactivating these repair pathways were manifested as cells entered stationary phase; however, the precise temporal requirement for their activities is not clear, and at least two scenarios are possible. The first scenario is that the activities of these proteins are required to repair the endogenous DNA damage that occurs during exponential growth to ensure that genome-processing events in stationary phase occur smoothly. The second scenario is that their activities are required for genome-processing events during stationary phase. Depending on when these repair proteins act in this genome-processing pathway, different models can be proposed to explain their roles.</p><p id="p-39">One explanation for the requirement of these proteins for stationary-phase chromosome processing is that they are involved in ongoing repair of endogenous DNA damage that occurs during exponential growth to ensure that the chromosomes are free of damage <em>before</em> the cells enter stationary phase. This ongoing repair ensures that the chromosomes are intact and ready for efficient processing as cells enter stationary phase. Thus, in wild-type cultures, most cells enter the peak 1 population rapidly during stationary phase. In contrast, cells of the repair mutant strains could arrive at the onset of stationary phase with damaged chromosomes due to failed repair of DNA damage that occurred during exponential growth. This unrepaired damage could block or slow late-stage chromosome processing pathways (such as chromosome partitioning), causing the persistence of cell populations with higher DNA content during stationary phase. In support of this model, earlier studies by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ishioka <em>et al.</em> (1998</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">1997)</a> and more recent studies by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rudolph <em>et al.</em> (2009a</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2009b)</a> have demonstrated that the RecG and the Ruv pathways are required to ensure proper chromosome segregation, at least in UV-irradiated and multiply repair-defective cells.</p><p id="p-40">If the activities of these repair proteins are required <em>after</em> exponential growth as cells enter stationary phase, then at least three possible explanations can be proposed. One explanation is that cells continue to divide late into stationary phase, partitioning their DNA to daughter cells by a mechanism similar to, but slower than, that of exponentially growing cells. The energy required for this active process could come from the metabolism of nutrients released from dead cells in the culture. If this hypothesis is correct, then HJ processing and double-strand break repair would be implicated in chromosome partitioning, or perhaps decatenation, in late-stationary phase. Normal chromosomal decatenation after DNA replication is a function of the XerCD and FtsK proteins (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grainge <em>et al.</em> 2007</a>), and the RuvABC, RecG, and RecBCD proteins have not previously been implicated in this process. However, cells in late-stationary phase might need HJ processing and double-strand break repair because of increased DNA damage from genotoxic agents that accumulate in the medium.</p><p id="p-41">A second explanation is that the peaks detected by flow cytometry do not consist of populations of cells with different numbers of chromosomes or different amounts of DNA, but instead, consist of populations of cells with different degrees of DNA compaction. During stationary phase, chromosomes are compacted via their association with DNA binding proteins, which include HU (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sarkar <em>et al.</em> 2007</a>), MukB (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cui <em>et al.</em> 2008</a>), and Dps (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Frenkiel-Krispin <em>et al.</em> 2004</a>). This compaction may decrease the ability of dyes like propidium iodide to bind the DNA duplex, resulting in the decreased fluorescence detected during flow cytometric analysis. Previous studies using flow cytometry of <em>E. coli</em> cells have not considered this possibility (<a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">Åkerlund <em>et al.</em> 1995</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Boye and Løbner-Olesen 1991</a>). This second hypothesis is particularly interesting, as it proposes that RuvABC, RecG, and RecBCD are involved in chromosome remodeling during stationary phase.</p><p id="p-42">Finally, a third and perhaps less likely explanation is that controlled DNA degradation occurs during late-stationary phase until the cells have uniform chromosome numbers. Such DNA degradation has not been reported previously, but it would explain the apparent reduction in DNA content (increase in numbers of cells with lower fluorescence intensity) that occurs in wild-type cells in the absence of cell division. Why HJ processing and double-strand break repair would be required for this process is not clear, but such a requirement would represent novel functions for these proteins. A converse possibility is that DNA replication, but not cell division, may continue longer into stationary phase in the mutant strains than in the wild-type strain, resulting in a higher proportion of cells with more DNA (increase in numbers of cells with higher fluorescence intensity); however, as shown in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, we were unable to detect the DNA synthesis that would be required in the mutant strains if this hypothesis were true.</p><p id="p-43">That DNA manipulation continues in late–stationary-phase cells is supported by several other observations. First, expression of <em>E. coli</em>’s three inducible, specialized DNA polymerases, DNA Pol II, Pol IV, and Pol V, continues for days after cells reach stationary phase (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yeiser <em>et al.</em> 2002</a>). In addition, the error-prone Y-family Pol IV is upregulated in late–stationary-phase cells under control of the stationary-phase sigma factor, RpoS (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Layton and Foster 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Storvik and Foster 2010</a>). The reason for this late induction is not fully understood; however, Pol IV has been hypothesized to synthesize DNA during recombination-dependent DNA repair leading to adaptive Lac<sup>+</sup> mutations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Foster 2000</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mckenzie <em>et al.</em> 2001</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tompkins <em>et al.</em> 2003</a>). As mentioned above, the DNA polymerase processivity factor DnaN and the RecA-loading factor RecF are also induced in stationary phase (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Villarroya <em>et al.</em> 1998</a>), further supporting the idea that DNA repair and recombination occur even in the absence of bulk DNA replication and cell division.</p><p id="p-44">This work has demonstrated the requirements of RuvABC, RecG, and RecBCD for establishment and/or maintenance of DNA content in stationary-phase cells. Determining whether double-strand break repair and HJ processing are required during exponential growth to “set the stage” during stationary phase for proper genome processing, for chromosome partitioning and/or reorganization, or for controlled DNA degradation will require further experimentation. For example, artificially inducing double-strand breaks at various points in the cell cycle might elucidate when double-strand break repair functions are required. Further research is needed to understand fully the molecular processes that allow nondividing cells to manage their DNA content and chromosome organization, and our data suggest that flow cytometry is a useful technique for such investigations.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We are grateful for the assistance provided by Christiane Hassel at the Indiana University Flow Cytometry Core Facility (Bloomington, IN). This work was supported by National Institutes of Health Grants GM-065175 (to P.L.F.) and T32-GM-007757 (to A.B.W. and K.M.H.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001057/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-46"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-07-18"><span class="received-label">Received </span>July 18, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-09-10"><span class="accepted-label">Accepted </span>September 10, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Williams <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.1" data-doi="10.1128/jb.177.23.6791-6797.1995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Åkerlund</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordstrom</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernander</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Analysis of cell size and DNA content in exponentially growing and stationary-phase batch cultures of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">6791</span>–<span class="cit-lpage">6797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DAkerlund%26rft.auinit1%253DT.%26rft.volume%253D177%26rft.issue%253D23%26rft.spage%253D6791%26rft.epage%253D6797%26rft.atitle%253DAnalysis%2Bof%2Bcell%2Bsize%2Band%2BDNA%2Bcontent%2Bin%2Bexponentially%2Bgrowing%2Band%2Bstationary-phase%2Bbatch%2Bcultures%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.177.23.6791-6797.1995%26rft_id%253Dinfo%253Apmid%252F7592469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3Ny8yMy82NzkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.2" data-doi="10.1111/j.1365-2958.1990.tb00628.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boe</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Mechanism for induction of adaptive mutations in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DBoe%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D597%26rft.epage%253D601%26rft.atitle%253DMechanism%2Bfor%2Binduction%2Bof%2Badaptive%2Bmutations%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.1990.tb00628.x%26rft_id%253Dinfo%253Apmid%252F2191182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.1990.tb00628.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2191182&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA85300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.3" data-doi="10.1016/0923-2508(91)90020-B"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boye</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Løbner-Olesen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Bacterial growth control studied by flow cytometry.</span> <abbr class="cit-jnl-abbrev">Res. Microbiol.</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.stitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.aulast%253DBoye%26rft.auinit1%253DE.%26rft.volume%253D142%26rft.issue%253D2-3%26rft.spage%253D131%26rft.epage%253D135%26rft.atitle%253DBacterial%2Bgrowth%2Bcontrol%2Bstudied%2Bby%2Bflow%2Bcytometry.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0923-2508%252891%252990020-B%26rft_id%253Dinfo%253Apmid%252F1925010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0923-2508(91)90020-B&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1925010&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FM03500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.417.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Adaptive reversion of a frameshift mutation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D128%26rft.spage%253D695%26rft_id%253Dinfo%253Apmid%252F1916241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1916241&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FZ27000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.5" data-doi="10.1016/0300-9084(93)90127-E"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">Y. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Expression of the <em>recA</em> gene in recombination-deficient (rec-) strains of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Biochimie</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">775</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D75%26rft.spage%253D775%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0300-9084%252893%252990127-E%26rft_id%253Dinfo%253Apmid%252F8274529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0300-9084(93)90127-E&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8274529&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.6" data-doi="10.1038/nsmb.1410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrushenko</span>  <span class="cit-name-given-names">Z. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybenkov</span>  <span class="cit-name-given-names">V. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MukB acts as a macromolecular clamp in DNA condensation.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DCui%26rft.auinit1%253DY.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D411%26rft.epage%253D418%26rft.atitle%253DMukB%2Bacts%2Bas%2Ba%2Bmacromolecular%2Bclamp%2Bin%2BDNA%2Bcondensation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1410%26rft_id%253Dinfo%253Apmid%252F18376412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18376412&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254621500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.7" data-doi="10.1128/MMBR.00020-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dillingham</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RecBCD enzyme and the repair of double-stranded DNA breaks.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DDillingham%26rft.auinit1%253DM.%2BS.%26rft.volume%253D72%26rft.issue%253D4%26rft.spage%253D642%26rft.epage%253D671%26rft.atitle%253DRecBCD%2BEnzyme%2Band%2Bthe%2BRepair%2Bof%2BDouble-Stranded%2BDNA%2BBreaks%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00020-08%26rft_id%253Dinfo%253Apmid%252F19052323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Mi80LzY0MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi80MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.8" data-doi="10.1016/S0065-2164(08)01002-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Nucleoid-associated proteins and bacterial physiology.</span> <abbr class="cit-jnl-abbrev">Adv. Appl. Microbiol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bapplied%2Bmicrobiology%26rft.stitle%253DAdv%2BAppl%2BMicrobiol%26rft.aulast%253DDorman%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D67%26rft.spage%253D47%26rft.epage%253D64%26rft.atitle%253DNucleoid-associated%2Bproteins%2Band%2Bbacterial%2Bphysiology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2164%252808%252901002-2%26rft_id%253Dinfo%253Apmid%252F19245936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2164(08)01002-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19245936&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263948200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.9" data-doi="10.1016/j.ymeth.2009.02.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferullo</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lovett</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Cell cycle synchronization of <em>Escherichia coli</em> using the stringent response, with fluorescence labeling assays for DNA content and replication.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DFerullo%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D48%26rft.issue%253D1%26rft.spage%253D8%26rft.epage%253D13%26rft.atitle%253DCell%2Bcycle%2Bsynchronization%2Bof%2BEscherichia%2Bcoli%2Busing%2Bthe%2Bstringent%2Bresponse%252C%2Bwith%2Bfluorescence%2Blabeling%2Bassays%2Bfor%2BDNA%2Bcontent%2Band%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2009.02.010%26rft_id%253Dinfo%253Apmid%252F19245839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2009.02.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19245839&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267143800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.10" data-doi="10.1101/sqb.2000.65.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Adaptive mutation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DFOSTER%26rft.auinit1%253DP.L.%26rft.volume%253D65%26rft.issue%253D0%26rft.spage%253D21%26rft.epage%253D30%26rft.atitle%253DAdaptive%2BMutation%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fsqb.2000.65.21%26rft_id%253Dinfo%253Apmid%252F12760017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjc6IjY1LzAvMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.11" data-doi="10.1080/10409230701648494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Stress-induced mutagenesis in bacteria.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Biochem. Mol. Biol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2BReviews%2Bin%2BBiochemistry%2Band%2BMolecular%2BBiology%26rft.stitle%253DCRIT%2BREV%2BBIOCHEM%2BMOL%2BBIOL%26rft.aulast%253DFoster%26rft.auinit1%253DP.%2BL.%26rft.volume%253D42%26rft.issue%253D5%26rft.spage%253D373%26rft.epage%253D397%26rft.atitle%253DStress-induced%2Bmutagenesis%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10409230701648494%26rft_id%253Dinfo%253Apmid%252F17917873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10409230701648494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17917873&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250418700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trimarchi</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Two enzymes, both of which process recombination intermediates, have opposite effects on adaptive mutation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D142%26rft.spage%253D25%26rft_id%253Dinfo%253Apmid%252F8770582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8770582&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.13" data-doi="10.1046/j.1365-2958.2003.03855.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frenkiel-Krispin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben-Avraham</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Englander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimoni</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Nucleoid restructuring in stationary-state bacteria.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DFrenkiel-Krispin%26rft.auinit1%253DD.%26rft.volume%253D51%26rft.issue%253D2%26rft.spage%253D395%26rft.epage%253D405%26rft.atitle%253DNucleoid%2Brestructuring%2Bin%2Bstationary-state%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03855.x%26rft_id%253Dinfo%253Apmid%252F14756781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03855.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14756781&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187890800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.14" data-doi="10.1038/sj.emboj.7601849"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grainge</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bregu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivanathan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ip</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Unlinking chromosome catenanes in vivo by site-specific recombination.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4228</span>–<span class="cit-lpage">4238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DGrainge%26rft.auinit1%253DI.%26rft.volume%253D26%26rft.issue%253D19%26rft.spage%253D4228%26rft.epage%253D4238%26rft.atitle%253DUnlinking%2Bchromosome%2Bcatenanes%2Bin%2Bvivo%2Bby%2Bsite-specific%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601849%26rft_id%253Dinfo%253Apmid%252F17805344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.emboj.7601849&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17805344&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250466800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.15" data-doi="10.1016/S0022-2836(75)80110-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grivell</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grivell</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanawalt</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Turnover in bacterial DNA containing thymine or 5-bromouracil.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGrivell%26rft.auinit1%253DA.%2BR.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D219%26rft.epage%253D233%26rft.atitle%253DTurnover%2Bin%2Bbacterial%2BDNA%2Bcontaining%2Bthymine%2Bor%2B5-bromouracil.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252875%252980110-0%26rft_id%253Dinfo%253Apmid%252F1104866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(75)80110-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1104866&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.16" data-doi="10.1126/science.8146657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longerich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Recombination in adaptive mutation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHarris%26rft.auinit1%253DR.%26rft.volume%253D264%26rft.issue%253D5156%26rft.spage%253D258%26rft.epage%253D260%26rft.atitle%253DRecombination%2Bin%2Badaptive%2Bmutation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8146657%26rft_id%253Dinfo%253Apmid%252F8146657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjQvNTE1Ni8yNTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Opposing roles of the Holliday junction processing systems of <em>Escherichia coli</em> in recombination-dependent adaptive mutation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D142%26rft.spage%253D681%26rft_id%253Dinfo%253Apmid%252F8849879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8849879&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TX45500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.18" data-doi="10.1046/j.1365-2443.1998.00185.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishioka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuoh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwasaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakata</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinagawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Abortive recombination in <em>Escherichia coli ruv</em> mutants blocks chromosome partitioning.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DIshioka%26rft.auinit1%253DK%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D209%26rft.epage%253D220%26rft.atitle%253DAbortive%2Brecombination%2Bin%2BEscherichia%2Bcoli%2Bruv%2Bmutants%2Bblocks%2Bchromosome%2Bpartitioning%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2443.1998.00185.x%26rft_id%253Dinfo%253Apmid%252F9663656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2443.1998.00185.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9663656&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.19" data-doi="10.1266/ggs.72.91"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishioka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwasaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinagawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Roles of the <em>recG</em> gene product of <em>Escherichia coli</em> in recombination repair: effects of the delta <em>recG</em> mutation on cell division and chromosome partition.</span> <abbr class="cit-jnl-abbrev">Genes Genet. Syst.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2Bgenetic%2Bsystems%26rft.stitle%253DGenes%2BGenet%2BSyst%26rft.aulast%253DIshioka%26rft.auinit1%253DK.%26rft.volume%253D72%26rft.issue%253D2%26rft.spage%253D91%26rft.epage%253D99%26rft.atitle%253DRoles%2Bof%2Bthe%2BrecG%2Bgene%2Bproduct%2Bof%2BEscherichia%2Bcoli%2Bin%2Brecombination%2Brepair%253A%2Beffects%2Bof%2Bthe%2Bdelta%2BrecG%2Bmutation%2Bon%2Bcell%2Bdivision%2Band%2Bchromosome%2Bpartition.%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fggs.72.91%26rft_id%253Dinfo%253Apmid%252F9265736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/ggs.72.91&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9265736&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XN87900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.20" data-doi="10.1128/JB.00130-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivančić-Baće</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vlašić</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salaj-Šmic</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brčić-Kostić</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic evidence for the requirement of RecA loading activity in SOS induction after UV irradiation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">5024</span>–<span class="cit-lpage">5032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DIvancic-Bace%26rft.auinit1%253DI.%26rft.volume%253D188%26rft.issue%253D14%26rft.spage%253D5024%26rft.epage%253D5032%26rft.atitle%253DGenetic%2BEvidence%2Bfor%2Bthe%2BRequirement%2Bof%2BRecA%2BLoading%2BActivity%2Bin%2BSOS%2BInduction%2Bafter%2BUV%2BIrradiation%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00130-06%26rft_id%253Dinfo%253Apmid%252F16816175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4OC8xNC81MDI0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.21" data-doi="10.1046/j.1365-2958.2003.03704.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Layton</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Error-prone DNA polymerase IV is controlled by the stress-response sigma factor, RpoS, in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DLayton%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D549%26rft.epage%253D561%26rft.atitle%253DError-prone%2BDNA%2Bpolymerase%2BIV%2Bis%2Bcontrolled%2Bby%2Bthe%2Bstress-response%2Bsigma%2Bfactor%252C%2BRpoS%252C%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03704.x%26rft_id%253Dinfo%253Apmid%252F14617178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03704.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14617178&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185961100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.22" data-doi="10.1128/jb.173.3.1004-1011.1991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckman</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic analysis of the <em>recG</em> locus of <em>Escherichia coli</em> K-12 and of its role in recombination and DNA repair.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLloyd%26rft.auinit1%253DR.%2BG.%26rft.volume%253D173%26rft.issue%253D3%26rft.spage%253D1004%26rft.epage%253D1011%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bthe%2BrecG%2Blocus%2Bof%2BEscherichia%2Bcoli%2BK-12%2Band%2Bof%2Bits%2Brole%2Bin%2Brecombination%2Band%2BDNA%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.173.3.1004-1011.1991%26rft_id%253Dinfo%253Apmid%252F1846849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzEwMDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.23" data-doi="10.1111/j.1365-2958.2004.04225.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCool</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrosino</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Measurement of SOS expression in individual <em>Escherichia coli</em> K-12 cells using fluorescence microscopy.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1343</span>–<span class="cit-lpage">1357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DMcCool%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D53%26rft.issue%253D5%26rft.spage%253D1343%26rft.epage%253D1357%26rft.atitle%253DMeasurement%2Bof%2BSOS%2Bexpression%2Bin%2Bindividual%2BEscherichia%2Bcoli%2BK-12%2Bcells%2Busing%2Bfluorescence%2Bmicroscopy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2004.04225.x%26rft_id%253Dinfo%253Apmid%252F15387814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2004.04225.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15387814&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223495100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.24" data-doi="10.1093/nar/27.15.3049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGlynn</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">RecG helicase activity at three- and four-strand DNA structures.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3049</span>–<span class="cit-lpage">3056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.15.3049%26rft_id%253Dinfo%253Apmid%252F10454599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.15.3049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10454599&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081832900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.25" data-doi="10.1016/S1097-2765(01)00204-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lombardo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastings</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">SOS mutator DNA polymerase IV functions in adaptive mutation and not adaptive amplification.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMcKenzie%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D571%26rft.epage%253D579%26rft.atitle%253DSOS%2Bmutator%2BDNA%2Bpolymerase%2BIV%2Bfunctions%2Bin%2Badaptive%2Bmutation%2Band%2Bnot%2Badaptive%2Bamplification.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900204-0%26rft_id%253Dinfo%253Apmid%252F11463382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00204-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11463382&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167966600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.26" data-doi="10.1016/j.dnarep.2007.02.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boubakri</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baharoglu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeMasson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lestini</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination proteins and rescue of arrested replication forks.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2Brepair%26rft.stitle%253DDNA%2BRepair%2B%2528Amst%2529%26rft.aulast%253DMichel%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253D967%26rft.epage%253D980%26rft.atitle%253DRecombination%2Bproteins%2Band%2Brescue%2Bof%2Barrested%2Breplication%2Bforks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2007.02.016%26rft_id%253Dinfo%253Apmid%252F17395553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2007.02.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17395553&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-1.6.417.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title"><em>A Short Course in Bacterial Genetics: A Laboratory Manual and Handbook for</em> Escherichia coli <em>and Related Bacteria</em>.</span> <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.28" data-doi="10.1128/JB.186.13.4192-4198.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nair</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finkel</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Dps protects cells against multiple stresses during stationary phase.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">4192</span>–<span class="cit-lpage">4198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DNair%26rft.auinit1%253DS.%26rft.volume%253D186%26rft.issue%253D13%26rft.spage%253D4192%26rft.epage%253D4198%26rft.atitle%253DDps%2BProtects%2BCells%2Bagainst%2BMultiple%2BStresses%2Bduring%2BStationary%2BPhase%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.186.13.4192-4198.2004%26rft_id%253Dinfo%253Apmid%252F15205421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4Ni8xMy80MTkyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.29" data-doi="10.1007/s00438-004-1089-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patten</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirchhof</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertzberg</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schellhorn</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microarray analysis of RpoS-mediated gene expression in <em>Escherichia coli</em> K-12.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DPatten%26rft.auinit1%253DC.%2BL.%26rft.volume%253D272%26rft.issue%253D5%26rft.spage%253D580%26rft.epage%253D591%26rft.atitle%253DMicroarray%2Banalysis%2Bof%2BRpoS-mediated%2Bgene%2Bexpression%2Bin%2BEscherichia%2Bcoli%2BK-12.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-004-1089-2%26rft_id%253Dinfo%253Apmid%252F15558318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-004-1089-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15558318&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226235900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.30" data-doi="10.1002/bit.20858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasan</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Effect of <em>rpoS</em> gene knockout on the metabolism of <em>Escherichia coli</em> during exponential growth phase and early stationary phase based on gene expressions, enzyme activities and intracellular metabolite concentrations.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Bioeng.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Band%2BBioengineering%26rft.stitle%253DBiotechnology%2Band%2BBioengineering%26rft.aulast%253DRahman%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DEffect%2Bof%2BrpoS%2Bgene%2Bknockout%2Bon%2Bthe%2Bmetabolism%2Bof%2BEscherichia%2Bcoli%2Bduring%2Bexponential%2Bgrowth%2Bphase%2Band%2Bearly%2Bstationary%2Bphase%2Bbased%2Bon%2Bgene%2Bexpressions%252C%2Benzyme%2Bactivities%2Band%2Bintracellular%2Bmetabolite%2Bconcentrations.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbit.20858%26rft_id%253Dinfo%253Apmid%252F16511888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bit.20858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16511888&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237879300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.31" data-doi="10.1111/j.1365-2958.2009.06773.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upton</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009a</span> <span class="cit-article-title">Pathological replication in cells lacking RecG DNA translocase.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">352</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DRudolph%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D73%26rft.issue%253D3%26rft.spage%253D352%26rft.epage%253D366%26rft.atitle%253DPathological%2Breplication%2Bin%2Bcells%2Blacking%2BRecG%2BDNA%2Btranslocase.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2009.06773.x%26rft_id%253Dinfo%253Apmid%252F19538444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2009.06773.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19538444&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.32" data-doi="10.1111/j.1365-2958.2009.06909.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upton</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009b</span> <span class="cit-article-title">Replication fork collisions cause pathological chromosomal amplification in cells lacking RecG DNA translocase.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">940</span>–<span class="cit-lpage">955</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DRudolph%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D74%26rft.issue%253D4%26rft.spage%253D940%26rft.epage%253D955%26rft.atitle%253DReplication%2Bfork%2Bcollisions%2Bcause%2Bpathological%2Bchromosomal%2Bamplification%2Bin%2Bcells%2Blacking%2BRecG%2BDNA%2Btranslocase.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2009.06909.x%26rft_id%253Dinfo%253Apmid%252F19818016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2009.06909.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19818016&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.33" data-doi="10.1073/pnas.0712168105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salic</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchison</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A chemical method for fast and sensitive detection of DNA synthesis <em>in vivo</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">2415</span>–<span class="cit-lpage">2420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0712168105%26rft_id%253Dinfo%253Apmid%252F18272492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzcvMjQxNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi80MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.34" data-doi="10.1093/nar/gkl1093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitoc</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukerji</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hud</span>  <span class="cit-name-given-names">N. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bacterial protein HU dictates the morphology of DNA condensates produced by crowding agents and polyamines.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">951</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl1093%26rft_id%253Dinfo%253Apmid%252F17259223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl1093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17259223&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.35" data-doi="10.1128/JB.00358-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storvik</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RpoS, the stress response sigma factor, plays a dual role in the regulation of <em>Escherichia coli's</em> error-prone DNA polymerase IV.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">3639</span>–<span class="cit-lpage">3644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00358-10%26rft_id%253Dinfo%253Apmid%252F20472798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5Mi8xNC8zNjM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T. C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patrick</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">DNA turnover in buffer-held <em>Escherichia coli</em> and its effect on repair of UV damage.</span> <abbr class="cit-jnl-abbrev">Photochem. Photobiol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhotochem.%2BPhotobiol.%26rft.volume%253D79%26rft.spage%253D511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.37" data-doi="10.1128/JB.185.11.3469-3472.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tompkins</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazel</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leugers</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stumpf</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Error-prone polymerase, DNA polymerase IV, is responsible for transient hypermutation during adaptive mutation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">3469</span>–<span class="cit-lpage">3472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DTompkins%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D185%26rft.issue%253D11%26rft.spage%253D3469%26rft.epage%253D3472%26rft.atitle%253DError-Prone%2BPolymerase%252C%2BDNA%2BPolymerase%2BIV%252C%2BIs%2BResponsible%2Bfor%2BTransient%2BHypermutation%2Bduring%2BAdaptive%2BMutation%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.185.11.3469-3472.2003%26rft_id%253Dinfo%253Apmid%252F12754247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4NS8xMS8zNDY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.38" data-doi="10.1093/emboj/17.6.1829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villarroya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Roger</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macián</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armengod</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Stationary phase induction of <em>dnaN</em> and <em>recF</em>, two genes of <em>Escherichia coli</em> involved in DNA replication and repair.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1829</span>–<span class="cit-lpage">1837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DVillarroya%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D1829%26rft.epage%253D1837%26rft.atitle%253DStationary%2Bphase%2Binduction%2Bof%2BdnaN%2Band%2BrecF%252C%2Btwo%2Bgenes%2Bof%2BEscherichia%2Bcoli%2Binvolved%2Bin%2BDNA%2Breplication%2Band%2Brepair%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.6.1829%26rft_id%253Dinfo%253Apmid%252F9501104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/emboj/17.6.1829&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9501104&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072693700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.39" data-doi="10.1146/annurev.genet.31.1.213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Processing of recombination intermediates by the RuvABC proteins.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DWest%26rft.auinit1%253DS.%2BC.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D213%26rft.epage%253D244%26rft.atitle%253DPROCESSING%2BOF%2BRECOMBINATION%2BINTERMEDIATES%2BBY%2BTHE%2BRUVABC%2BPROTEINS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.31.1.213%26rft_id%253Dinfo%253Apmid%252F9442895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.31.1.213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9442895&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071154300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.40" data-doi="10.1534/genetics.107.075861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The <em>Escherichia coli</em> histone-like protein HU has a role in stationary phase adaptive mutation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D177%26rft.spage%253D723%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075861%26rft_id%253Dinfo%253Apmid%252F17720921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.075861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17720921&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Witkin</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Ultraviolet mutagenesis and inducible DNA repair in <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">Bacteriol. Rev.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DWitkin%26rft.auinit1%253DE%2BM%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D869%26rft.epage%253D907%26rft.atitle%253DUltraviolet%2Bmutagenesis%2Band%2Binducible%2BDNA%2Brepair%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Apmid%252F795416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo0OiJtbWJyIjtzOjU6InJlc2lkIjtzOjg6IjQwLzQvODY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.417.42" data-doi="10.1073/pnas.092269199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeiser</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pepper</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finkel</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">SOS-induced DNA polymerases enhance long-term survival and evolutionary fitness.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">8737</span>–<span class="cit-lpage">8741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.092269199%26rft_id%253Dinfo%253Apmid%252F12060704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTkvMTMvODczNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi80MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/6/417.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/411" title="Amino Acid Substitutions in the Caenorhabditis elegans RNA Polymerase II Large Subunit AMA-1/RPB-1 that Result in α-Amanitin Resistance and/or Reduced Function" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/6/427" title="Partitioning Transcript Variation in Drosophila: Abundance, Isoforms, and Alleles" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/6"><span>Volume 1 Issue 6, November 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/6.cover.gif?itok=aUjFCEJ8" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/417" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qHhRWl-AqBbHwM-S3_BeGK0Pk8Ig0SyQ6D3ia4NgQRY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/6/417" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/6/417" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/6/417" class="active" data-icon-position="" data-hide-link-title="0">Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase Escherichia coli Cells</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459192" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7j-uV62RGFZpWLTxCJAmals2-JeCC9oQHZa9oAOxZwQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/417.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/6/417" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Imvu7OOxblKesgSzjHVCR2GbQViXP9xD7QLI4J_eDTI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/6/417" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/417.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459192" id="node459192--2" data-pisa="ggg;1/6/417" data-pisa-master="ggg;g3.111.001057" data-apath="/ggg/1/6/417.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase <em>Escherichia coli</em> Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ashley B. Williams</span>, <span class="highwire-citation-author" data-delta="1">Kyle M. Hetrick</span> and <span class="highwire-citation-author" data-delta="2">Patricia L. Foster</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">417-426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001057 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459192--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ashley B. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Computational Biology, University of Southern California, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashley%2BB.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAshley%2BB.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kyle M. Hetrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BM.%2BHetrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hetrick%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BM.%2BHetrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricia L. Foster </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BL.%2BFoster%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foster%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BL.%2BFoster%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plfoster@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">plfoster@indiana.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/6/417" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/6/417&amp;title=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/6/417&amp;title=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/6/417&amp;title=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/6/417&amp;text=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/6/417&amp;title=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/6/417&amp;t=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/6/417&amp;title=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/6/417&amp;title=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459192/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459192" id="node459192--3" data-pisa="ggg;1/6/417" data-pisa-master="ggg;g3.111.001057" data-apath="/ggg/1/6/417.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase <em>Escherichia coli</em> Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ashley B. Williams</span>, <span class="highwire-citation-author" data-delta="1">Kyle M. Hetrick</span> and <span class="highwire-citation-author" data-delta="2">Patricia L. Foster</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">417-426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001057 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459192--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ashley B. Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Computational Biology, University of Southern California, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashley%2BB.%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAshley%2BB.%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kyle M. Hetrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BM.%2BHetrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hetrick%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BM.%2BHetrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patricia L. Foster </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BL.%2BFoster%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Foster%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricia%2BL.%2BFoster%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:plfoster@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">plfoster@indiana.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=417&amp;rft.epage=426&amp;rft.atitle=Double-Strand%20Break%20Repair%20and%20Holliday%20Junction%20Processing%20Are%20Required%20for%20Chromosome%20Processing%20in%20Stationary-Phase%20Escherichia%20coli%20Cells&amp;rft.volume=1&amp;rft.issue=6&amp;rft.date=2011-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Williams%2C+Ashley+B.&amp;rft.au=Hetrick%2C+Kyle+M.&amp;rft.au=Foster%2C+Patricia+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459192/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459192/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459192/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459192/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459192/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459192/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459192/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459192/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459192/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459192/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459192/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459192/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/6/417" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/6/417.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/6/417.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/6/417.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/6/417.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
