<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/6/427" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/6/427.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/6/427.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/6/427.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/1/6/427/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/6/427.atom" />
<meta name="HW.pisa" content="ggg;1/6/427" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Partitioning Transcript Variation in Drosophila: Abundance, Isoforms, and Alleles" />
<meta name="DC.Identifier" content="10.1534/g3.111.000596" />
<meta name="DC.Date" content="2011-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Yang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multilevel analysis of transcription is facilitated by a new array design that includes modules for assessment of differential expression, isoform usage, and allelic imbalance in Drosophila . The ∼2.5 million feature chip incorporates a large number of controls, and it contains 18,769 3′ expression probe sets and 61,919 exon probe sets with probe sequences from Drosophila melanogaster and 60,118 SNP probe sets focused on Drosophila simulans . An experiment in D. simulans identified genes differentially expressed between males and females (34% in the 3′ expression module; 32% in the exon module). These proportions are consistent with previous reports, and there was good agreement (κ = 0.63) between the modules. Alternative isoform usage between the sexes was identified for 164 genes. The SNP module was verified with resequencing data. Concordance between resequencing and the chip design was greater than 99%. The design also proved apt in separating alleles based upon hybridization intensity. Concordance between the highest hybridization signals and the expected alleles in the genotype was greater than 96%. Intriguingly, allelic imbalance was detected for 37% of 6579 probe sets examined that contained heterozygous SNP loci. The large number of probes and multiple probe sets per gene in the 3′ expression and exon modules allows the array to be used in D. melanogaster and in closely related species. The SNP module can be used for allele specific expression and genotyping of D. simulans ." />
<meta name="DC.Contributor" content="Yajie Yang" />
<meta name="DC.Contributor" content="Rita M. Graze" />
<meta name="DC.Contributor" content="Brandon M. Walts" />
<meta name="DC.Contributor" content="Cecilia M. Lopez" />
<meta name="DC.Contributor" content="Henry V. Baker" />
<meta name="DC.Contributor" content="Marta L. Wayne" />
<meta name="DC.Contributor" content="Sergey V. Nuzhdin" />
<meta name="DC.Contributor" content="Lauren M. McIntyre" />
<meta name="article:published_time" content="2011-11-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Partitioning Transcript Variation in Drosophila: Abundance, Isoforms, and Alleles" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multilevel analysis of transcription is facilitated by a new array design that includes modules for assessment of differential expression, isoform usage, and allelic imbalance in &lt;i&gt;Drosophila&lt;/i&gt;. The ∼2.5 million feature chip incorporates a large number of controls, and it contains 18,769 3′ expression probe sets and 61,919 exon probe sets with probe sequences from &lt;i&gt;Drosophila melanogaster&lt;/i&gt; and 60,118 SNP probe sets focused on &lt;i&gt;Drosophila simulans&lt;/i&gt;. An experiment in &lt;i&gt;D. simulans&lt;/i&gt; identified genes differentially expressed between males and females (34% in the 3′ expression module; 32% in the exon module). These proportions are consistent with previous reports, and there was good agreement (κ = 0.63) between the modules. Alternative isoform usage between the sexes was identified for 164 genes. The SNP module was verified with resequencing data. Concordance between resequencing and the chip design was greater than 99%. The design also proved apt in separating alleles based upon hybridization intensity. Concordance between the highest hybridization signals and the expected alleles in the genotype was greater than 96%. Intriguingly, allelic imbalance was detected for 37% of 6579 probe sets examined that contained heterozygous SNP loci. The large number of probes and multiple probe sets per gene in the 3′ expression and exon modules allows the array to be used in &lt;i&gt;D. melanogaster&lt;/i&gt; and in closely related species. The SNP module can be used for allele specific expression and genotyping of &lt;i&gt;D. simulans&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/11/01" />
<meta name="citation_mjid" content="ggg;1/6/427" />
<meta name="citation_id" content="1/6/427" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/6/427" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/6/427.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/6/427.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/6/427.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000596" />
<meta name="citation_pmid" content="22384353" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="427" />
<meta name="citation_lastpage" content="436" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yajie Yang" />
<meta name="citation_author_institution" content="Genetics Institute, University of Florida, Gainesville, FL 32610-3610" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266" />
<meta name="citation_author" content="Rita M. Graze" />
<meta name="citation_author_institution" content="Genetics Institute, University of Florida, Gainesville, FL 32610-3610" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266" />
<meta name="citation_author" content="Brandon M. Walts" />
<meta name="citation_author_institution" content="Genetics Institute, University of Florida, Gainesville, FL 32610-3610" />
<meta name="citation_author" content="Cecilia M. Lopez" />
<meta name="citation_author_institution" content="Genetics Institute, University of Florida, Gainesville, FL 32610-3610" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266" />
<meta name="citation_author" content="Henry V. Baker" />
<meta name="citation_author_institution" content="Genetics Institute, University of Florida, Gainesville, FL 32610-3610" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266" />
<meta name="citation_author" content="Marta L. Wayne" />
<meta name="citation_author_institution" content="Genetics Institute, University of Florida, Gainesville, FL 32610-3610" />
<meta name="citation_author_institution" content="Department of Zoology, University of Florida, Gainesville, FL, 32611-8525" />
<meta name="citation_author" content="Sergey V. Nuzhdin" />
<meta name="citation_author_institution" content="Molecular and Computational Biology, University of Southern California, Los Angeles, CA 90089-2910" />
<meta name="citation_author" content="Lauren M. McIntyre" />
<meta name="citation_author_institution" content="Genetics Institute, University of Florida, Gainesville, FL 32610-3610" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266" />
<meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, FL 32611-8545" />
<meta name="citation_author_email" content="mcintyre@ufl.edu" />
<meta name="citation_reference" content="citation_title=Exon array background correction.;citation_author=Affymetrix, Inc;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Analytical biochemistry;citation_journal_abbrev=Anal Biochem;citation_author=A. Ahmadian;citation_author=B. Gharizadeh;citation_author=A. C. Gustafsson;citation_author=F. Sterky;citation_author=P. Nyrén;citation_title=Single-nucleotide polymorphism analysis by pyrosequencing.;citation_pages=103-110;citation_volume=280;citation_year=2000;citation_issue=1;citation_pmid=10805527;citation_doi=10.1006/abio.2000.4493" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Ammar;citation_author=A. M. Smith;citation_author=L. E. Heisler;citation_author=G. Giaever;citation_author=C. Nislow;citation_title=A comparative analysis of DNA barcode microarray feature size.;citation_pages=471-471;citation_volume=10;citation_year=2009;citation_pmid=19825181;citation_doi=10.1186/1471-2164-10-471" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=D. Begun;citation_author=A. Holloway;citation_author=K. Stevens;citation_author=L. Hillier;citation_author=Y.-P. Poh;citation_title=Population genomics: whole-genome analysis of polymorphism and divergence in Drosophila simulans.;citation_pages=e310-e310;citation_volume=5;citation_year=2007;citation_issue=11;citation_pmid=17988176;citation_doi=10.1371/journal.pbio.0050310" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. A. Benson;citation_author=I. Karsch-Mizrachi;citation_author=D. J. Lipman;citation_author=J. Ostell;citation_author=D. L. Wheeler;citation_title=GenBank;citation_pages=D34-D38;citation_volume=33;citation_year=2005;citation_issue=suppl_1;citation_pmid=15608212;citation_doi=10.1093/nar/gni032" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_journal_abbrev=Lancet;citation_author=J. M. Bland;citation_author=D. G. Altman;citation_title=Statistical methods for assessing agreement between two methods of clinical measurement.;citation_pages=307-310;citation_volume=1;citation_year=1986;citation_issue=8476;citation_pmid=2868172;citation_doi=10.1016/S0140-6736(86)90837-8" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Epidemiol.;citation_author=J. M. Bland;citation_author=D. G. Altman;citation_title=MEN WHO DO NOT DRINK: A REPORT FROM THE BRITISH REGIONAL HEART STUDY;citation_pages=201-203;citation_volume=17;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. O. Borevitz;citation_author=S. P. Hazen;citation_author=T. P. Michael;citation_author=G. P. Morris;citation_author=I. R. Baxter;citation_title=Genome-wide patterns of single-feature polymorphism in Arabidopsis thaliana.;citation_volume=104;citation_year=2007;citation_pmid=17626786;citation_doi=10.1073/pnas.0705323104" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=M. Bownes;citation_title=The regulation of the yolk protein genes, a family of sex differentiation genes in Drosophila melanogaster.;citation_pages=745-752;citation_volume=16;citation_year=1994;citation_issue=10;citation_pmid=7980478;citation_doi=10.1002/bies.950161009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. E. Celniker;citation_author=L. A. L. Dillon;citation_author=M. B. Gerstein;citation_author=K. C. Gunsalus;citation_author=S. Henikoff;citation_title=Unlocking the secrets of the genome.;citation_pages=927-930;citation_volume=459;citation_year=2009;citation_issue=7249;citation_pmid=19536255;citation_doi=10.1038/459927a" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. S. Dandy;citation_author=P. Wu;citation_author=D. W. Grainger;citation_title=Array feature size influences nucleic acid surface capture in DNA microarrays.;citation_volume=104;citation_year=2007;citation_pmid=17485675;citation_doi=10.1073/pnas.0606054104" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. F. Degner;citation_author=J. C. Marioni;citation_author=A. A. Pai;citation_author=J. K. Pickrell;citation_author=E. Nkadori;citation_title=Effect of read-mapping biases on detecting allele-specific expression from RNA-sequencing data.;citation_volume=25;citation_year=2009;citation_pmid=19808877;citation_doi=10.1093/bioinformatics/btp579" />
<meta name="citation_reference" content="citation_journal_title=Oncogene;citation_journal_abbrev=Oncogene;citation_author=A. de la Chapelle;citation_title=Genetic predisposition to human disease: allele-specific expression and low-penetrance regulatory loci.;citation_pages=3345-3348;citation_volume=28;citation_year=2009;citation_issue=38;citation_pmid=19597467;citation_doi=10.1038/onc.2009.194" />
<meta name="citation_reference" content="citation_journal_title=Statist. Sinica;citation_author=S. Dudoit;citation_author=Y. H. Yang;citation_author=M. J. Callow;citation_author=T. P. Speed;citation_title=Statistical methods for identifying differentially expressed genes in replicated cDNA microarray experiments.;citation_pages=111-139;citation_volume=12;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Dworkin;citation_author=C. D. Jones;citation_title=Genetic changes accompanying the evolution of host specialization in Drosophila sechellia.;citation_pages=721-736;citation_volume=181;citation_year=2009;citation_pmid=19033155;citation_doi=10.1534/genetics.108.093419" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=H. Edenberg;citation_author=L. Bierut;citation_author=P. Boyce;citation_author=M. Cao;citation_author=S. Cawley;citation_title=Description of the data from the Collaborative Study on the Genetics of Alcoholism (COGA) and single-nucleotide polymorphism genotyping for Genetic Analysis Workshop 14.;citation_pages=S2;citation_volume=6;citation_year=2005;citation_issue=Suppl. 1;citation_pmid=16451628;citation_doi=10.1186/1471-2156-6-S1-S2" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. J. Emerson;citation_author=L.-C. Hsieh;citation_author=H.-M. Sung;citation_author=T.-Y. Wang;citation_author=C.-J. Huang;citation_title=Natural selection on cis and trans regulation in yeasts.;citation_volume=20;citation_year=2010;citation_pmid=20445163;citation_doi=10.1101/gr.101576.109" />
<meta name="citation_reference" content="citation_title=Statistical Methods for Rates and Proportions.;citation_author=J. Fleiss;citation_year=1981" />
<meta name="citation_reference" content="citation_title=A database of Drosophila genes and genomes.;citation_author=FlyBase;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. Fontanillas;citation_author=C. R. Landry;citation_author=P. J. Wittkopp;citation_author=C. Russ;citation_author=J. D. Gruber;citation_title=Key considerations for measuring allelic expression on a genomic scale using high-throughput sequencing.;citation_pages=212-227;citation_volume=19;citation_year=2010;citation_pmid=20331781;citation_doi=10.1111/j.1365-294X.2010.04472.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. C. Frith;citation_author=R. Wan;citation_author=P. Horton;citation_title=Incorporating sequence quality data into alignment improves DNA read mapping.;citation_volume=38;citation_year=2010;citation_pmid=20110255;citation_doi=10.1093/nar/gkq010" />
<meta name="citation_reference" content="citation_journal_title=Genetics.;citation_author=R. M. Graze;citation_author=L. M. McIntyre;citation_author=B. J. Main;citation_author=M. L. Wayne;citation_author=S. V. Nuzhdin;citation_title=Regulatory divergence in Drosophila melanogaster and D. simulans, a genome-wide analysis of allele-specific expression.;citation_pages=547-561;citation_volume=183;citation_year=2009;citation_pmid=19667135;citation_doi=10.1534/genetics.109.105957" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Guo;citation_author=S. Yang;citation_author=M. Rupe;citation_author=B. Hu;citation_author=D. Bickel;citation_title=Genome-wide allele-specific expression analysis using Massively Parallel Signature Sequencing (MPSS) reveals cis- and trans-effects on gene expression in maize hybrid meristem tissue.;citation_pages=551-563;citation_volume=66;citation_year=2008;citation_issue=5;citation_pmid=18224447;citation_doi=10.1007/s11103-008-9290-z" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Jin;citation_author=R. M. Riley;citation_author=R. D. Wolfinger;citation_author=K. P. White;citation_author=G. Passador-Gurgel;citation_title=The contributions of sex, genotype and age to transcriptional variance in Drosophila melanogaster.;citation_pages=389-395;citation_volume=29;citation_year=2001;citation_issue=4;citation_pmid=11726925;citation_doi=10.1038/ng766" />
<meta name="citation_reference" content="citation_journal_title=Pharmacology &amp; therapeutics;citation_journal_abbrev=Pharmacol Ther;citation_author=A. D. Johnson;citation_author=D. Wang;citation_author=W. Sadee;citation_title=Polymorphisms affecting gene regulation and mRNA processing: broad implications for pharmacogenetics.;citation_pages=19-38;citation_volume=106;citation_year=2005;citation_issue=1;citation_pmid=15781120;citation_doi=10.1016/j.pharmthera.2004.11.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. M. Johnson;citation_author=J. Castle;citation_author=P. Garrett-Engele;citation_author=Z. Kan;citation_author=P. M. Loerch;citation_title=Genome-Wide Survey of Human Alternative Pre-mRNA Splicing with Exon Junction Microarrays;citation_pages=2141-2144;citation_volume=302;citation_year=2003;citation_issue=5653;citation_pmid=14684825;citation_doi=10.1126/science.1090100" />
<meta name="citation_reference" content="citation_title=Applied Multivariate Statistical Analysis.;citation_author=R. A. Johnson;citation_author=D. W. Wichern;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Kopp;citation_author=O. Barmina;citation_author=A. M. Hamilton;citation_author=L. Higgins;citation_author=L. M. McIntyre;citation_title=Evolution of gene expression in the drosophila olfactory system.;citation_volume=25;citation_year=2008;citation_pmid=18296696;citation_doi=10.1093/molbev/msn055" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. Kwan;citation_author=D. Benovoy;citation_author=C. Dias;citation_author=S. Gurd;citation_author=C. Provencher;citation_title=Genome-wide analysis of transcript isoform variation in humans.;citation_pages=225-231;citation_volume=40;citation_year=2008;citation_issue=2;citation_pmid=18193047;citation_doi=10.1038/ng.2007.57" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map (SAM) Format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=H. S. Lo;citation_author=Z. Wang;citation_author=Y. Hu;citation_author=H. H. Yang;citation_author=S. Gere;citation_title=Allelic Variation in Gene Expression Is Common in the Human Genome;citation_pages=1855-1862;citation_volume=13;citation_year=2003;citation_issue=8;citation_pmid=12902379" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Lu;citation_author=J. A. Shapiro;citation_author=C.-T. Ting;citation_author=Y. Li;citation_author=C. Li;citation_title=Genome-wide mis-expression of X-linked versus autosomal genes associated with hybrid male sterility.;citation_volume=20;citation_year=2010;citation_pmid=20511493;citation_doi=10.1101/gr.076620.108" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=J. Malone;citation_author=B. Oliver;citation_title=Microarrays, deep sequencing and the true measure of the transcriptome.;citation_pages=34-34;citation_volume=9;citation_year=2011;citation_pmid=21627854;citation_doi=10.1186/1741-7007-9-34" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=L. McIntyre;citation_author=L. Bono;citation_author=A. Genissel;citation_author=R. Westerman;citation_author=D. Junk;citation_title=Sex-specific expression of alternative transcripts in Drosophila.;citation_pages=R79-R79;citation_volume=7;citation_year=2006;citation_issue=8;citation_pmid=16934145;citation_doi=10.1186/gb-2006-7-8-r79" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. M. McIntyre;citation_author=K. K. Lopiano;citation_author=A. M. Morse;citation_author=V. Amin;citation_author=A. L. Oberg;citation_title=RNA-seq: technical variability and sampling.;citation_pages=293-293;citation_volume=12;citation_year=2011;citation_pmid=21645359;citation_doi=10.1186/1471-2164-12-293" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. J. McManus;citation_author=J. D. Coolon;citation_author=M. O. Duff;citation_author=J. Eipper-Mains;citation_author=B. R. Graveley;citation_title=Regulatory divergence in Drosophila revealed by mRNA-seq.;citation_volume=20;citation_year=2010;citation_pmid=20354124;citation_doi=10.1101/gr.102491.109" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=K. B. Meyer;citation_author=A.-T. Maia;citation_author=M. O'Reilly;citation_author=A. E. Teschendorff;citation_author=S.-F. Chin;citation_title=Allele-specific up-regulation of FGFR2 increases susceptibility to breast cancer.;citation_pages=e108-e108;citation_volume=6;citation_year=2008;citation_issue=5;citation_pmid=18462018;citation_doi=10.1371/journal.pbio.0060108" />
<meta name="citation_reference" content="citation_title=Applied Linear Statistical Models: Regression, Analysis of Variance, and Experimental Designs;citation_author=J. Neter;citation_author=W. Wasserman;citation_author=M. H. Kutner;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=A. C. Nica;citation_author=E. T. Dermitzakis;citation_title=Using gene expression to investigate the genetic basis of complex disorders.;citation_volume=17;citation_year=2008;citation_pmid=18852201;citation_doi=10.1093/hmg/ddn285" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. C. Nica;citation_author=S. B. Montgomery;citation_author=A. S. Dimas;citation_author=B. E. Stranger;citation_author=C. Beazley;citation_title=Candidate causal regulatory effects by integration of expression QTLs with complex trait genetic associations.;citation_pages=e1000895-e1000895;citation_volume=6;citation_year=2010;citation_issue=4;citation_pmid=20369022;citation_doi=10.1371/journal.pgen.1000895" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Parisi;citation_author=R. Nuttall;citation_author=D. Naiman;citation_author=G. Bouffard;citation_author=J. Malley;citation_title=Paucity of genes on the Drosophila X chromosome showing male-biased expression.;citation_volume=299;citation_year=2003;citation_pmid=12511656;citation_doi=10.1126/science.1079190" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=T. Pastinen;citation_author=R. Sladek;citation_author=S. Gurd;citation_author=A. A. Sammak;citation_author=B. Ge;citation_title=A survey of genetic and epigenetic variation affecting human gene expression.;citation_volume=16;citation_year=2004;citation_pmid=14583597;citation_doi=10.1152/physiolgenomics.00163.2003" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. K. Pickrell;citation_author=J. C. Marioni;citation_author=A. A. Pai;citation_author=J. F. Degner;citation_author=B. E. Engelhardt;citation_title=Understanding mechanisms underlying human gene expression variation with RNA sequencing.;citation_pages=768-772;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=20220758;citation_doi=10.1038/nature08872" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Rabbee;citation_author=T. P. Speed;citation_title=A genotype calling algorithm for Affymetrix SNP arrays.;citation_volume=22;citation_year=2006;citation_pmid=16267090;citation_doi=10.1093/bioinformatics/bti741" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. M. Ranz;citation_author=C. I. Castillo-Davis;citation_author=C. D. Meiklejohn;citation_author=D. L. Hartl;citation_title=Sex-Dependent Gene Expression and Evolution of the Drosophila Transcriptome;citation_pages=1742-1745;citation_volume=300;citation_year=2003;citation_issue=5626;citation_pmid=12805547;citation_doi=10.1126/science.1085881" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. Rifkin;citation_author=J. Kim;citation_author=K. P. White;citation_title=Evolution of gene expression in the Drosophila melanogaster subgroup.;citation_pages=138-144;citation_volume=33;citation_year=2003;citation_issue=2;citation_pmid=12548287;citation_doi=10.1038/ng1086" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Genetics of global gene expression.;citation_pages=862-872;citation_volume=7;citation_year=2006;citation_issue=11;citation_pmid=17047685;citation_doi=10.1038/nrg1964" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. T. Ross;citation_author=U. Scherf;citation_author=M. B. Eisen;citation_author=C. M. Perou;citation_author=C. Rees;citation_title=Systematic variation in gene expression patterns in human cancer cell lines.;citation_pages=227-235;citation_volume=24;citation_year=2000;citation_issue=3;citation_pmid=10700174;citation_doi=10.1038/73432" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Schena;citation_author=D. Shalon;citation_author=R. W. Davis;citation_author=P. O. Brown;citation_title=Quantitative Monitoring of Gene Expression Patterns with a Complementary DNA Microarray;citation_pages=467-470;citation_volume=270;citation_year=1995;citation_issue=5235;citation_pmid=7569999;citation_doi=10.1126/science.270.5235.467" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Serre;citation_author=S. Gurd;citation_author=B. Ge;citation_author=R. Sladek;citation_author=D. Sinnett;citation_title=Differential allelic expression in the human genome: a robust approach to identify genetic and epigenetic cis-acting mechanisms regulating gene expression.;citation_pages=e1000006-e1000006;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454203;citation_doi=10.1371/journal.pgen.1000006" />
<meta name="citation_reference" content="citation_title=Density Estimation for Statistics and Data Analysis.;citation_author=B. W. Silverman;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=J. A. Stamatoyannopoulos;citation_title=The genomics of gene expression.;citation_pages=449-457;citation_volume=84;citation_year=2004;citation_issue=3;citation_pmid=15498452;citation_doi=10.1016/j.ygeno.2004.05.002" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. E. Stranger;citation_author=A. C. Nica;citation_author=M. S. Forrest;citation_author=A. Dimas;citation_author=C. P. Bird;citation_title=Population genomics of human gene expression.;citation_pages=1217-1224;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17873874;citation_doi=10.1038/ng2142" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. E. Szabó;citation_author=J. R. Mann;citation_title=Biallelic expression of imprinted genes in the mouse germ line: implications for erasure, establishment, and mechanisms of genomic imprinting.;citation_pages=1857-1868;citation_volume=9;citation_year=1995;citation_issue=15;citation_pmid=7649473;citation_doi=10.1101/gad.9.15.1857" />
<meta name="citation_reference" content="citation_author=M. Telonis-Scott;citation_author=A. Kopp;citation_author=M. L. Wayne;citation_author=S. V. Nuzhdin;citation_author=L. M. McIntyre;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. D. Thompson;citation_author=D. G. Higgins;citation_author=T. J. Gibson;citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.;citation_volume=22;citation_year=1994;citation_pmid=7984417;citation_doi=10.1093/nar/22.22.4673" />
<meta name="citation_reference" content="citation_journal_title=Oikos;citation_author=K. J. F. Verhoeven;citation_author=K. L. Simonsen;citation_author=L. M. McIntyre;citation_title=Implementing false discovery rate control: increasing your power.;citation_pages=643-647;citation_volume=108;citation_year=2005;citation_doi=10.1111/j.0030-1299.2005.13727.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. J. Verlaan;citation_author=S. Berlivet;citation_author=G. M. Hunninghake;citation_author=A.-M. Madore;citation_author=M. Larivière;citation_title=Allele-specific chromatin remodeling in the ZPBP2/GSDMB/ORMDL3 locus associated with the risk of asthma and autoimmune disease.;citation_pages=377-393;citation_volume=85;citation_year=2009;citation_issue=3;citation_pmid=19732864;citation_doi=10.1016/j.ajhg.2009.08.007" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=D. Wang;citation_author=H. Chen;citation_author=K. M. Momary;citation_author=L. H. Cavallari;citation_author=J. A. Johnson;citation_title=Regulatory polymorphism in vitamin K epoxide reductase complex subunit 1 (VKORC1) affects gene expression and warfarin dose requirement.;citation_volume=112;citation_year=2008;citation_pmid=18523153;citation_doi=10.1182/blood-2008-03-144899" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. L. Wayne;citation_author=M. Telonis-Scott;citation_author=L. M. Bono;citation_author=L. Harshman;citation_author=A. Kopp;citation_title=Simpler mode of inheritance of transcriptional variation in male Drosophila melanogaster.;citation_volume=104;citation_year=2007;citation_pmid=18003923;citation_doi=10.1073/pnas.0705441104" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Wittkopp;citation_author=B. Haerum;citation_author=A. Clark;citation_title=Regulatory changes underlying expression differences within and between Drosophila species.;citation_pages=346-350;citation_volume=40;citation_year=2008;citation_issue=3;citation_pmid=18278046;citation_doi=10.1038/ng.77" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. J. Wittkopp;citation_author=B. K. Haerum;citation_author=A. G. Clark;citation_title=Evolutionary changes in cis and trans gene regulation.;citation_pages=85-88;citation_volume=430;citation_year=2004;citation_issue=6995;citation_pmid=15229602;citation_doi=10.1038/nature02698" />
<meta name="citation_reference" content="citation_journal_title=Insect biochemistry and molecular biology;citation_journal_abbrev=Insect Biochem Mol Biol;citation_author=M. F. Wolfner;citation_title=Tokens of love: functions and regulation of Drosophila male accessory gland products.;citation_pages=179-192;citation_volume=27;citation_year=1997;citation_issue=3;citation_pmid=9090115;citation_doi=10.1016/S0965-1748(96)00084-7" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=K. Zhang;citation_author=J. B. Li;citation_author=Y. Gao;citation_author=D. Egli;citation_author=B. Xie;citation_title=Digital RNA allelotyping reveals tissue-specific and allele-specific gene expression in human.;citation_pages=613-618;citation_volume=6;citation_year=2009;citation_issue=8;citation_pmid=19620972;citation_doi=10.1038/nmeth.1357" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Zhang;citation_author=J. O. Borevitz;citation_title=Global analysis of allele-specific expression in Arabidopsis thaliana.;citation_pages=943-954;citation_volume=182;citation_year=2009;citation_pmid=19474198;citation_doi=10.1534/genetics.109.103499" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Partitioning Transcript Variation in Drosophila: Abundance, Isoforms, and Alleles" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
<meta name="twitter:description" content="Multilevel analysis of transcription is facilitated by a new array design that includes modules for assessment of differential expression, isoform usage, and allelic imbalance in Drosophila . The ∼2.5 million feature chip incorporates a large number of controls, and it contains 18,769 3′ expression probe sets and 61,919 exon probe sets with probe sequences from Drosophila melanogaster and 60,118 SNP probe sets focused on Drosophila simulans . An experiment in D. simulans identified genes differentially expressed between males and females (34% in the 3′ expression module; 32% in the exon module). These proportions are consistent with previous reports, and there was good agreement (κ = 0.63) between the modules. Alternative isoform usage between the sexes was identified for 164 genes. The SNP module was verified with resequencing data. Concordance between resequencing and the chip design was greater than 99%. The design also proved apt in separating alleles based upon hybridization intensity. Concordance between the highest hybridization signals and the expected alleles in the genotype was greater than 96%. Intriguingly, allelic imbalance was detected for 37% of 6579 probe sets examined that contained heterozygous SNP loci. The large number of probes and multiple probe sets per gene in the 3′ expression and exon modules allows the array to be used in D. melanogaster and in closely related species. The SNP module can be used for allele specific expression and genotyping of D. simulans ." />
<meta name="og-title" property="og:title" content="Partitioning Transcript Variation in Drosophila: Abundance, Isoforms, and Alleles" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/6/427" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multilevel analysis of transcription is facilitated by a new array design that includes modules for assessment of differential expression, isoform usage, and allelic imbalance in Drosophila . The ∼2.5 million feature chip incorporates a large number of controls, and it contains 18,769 3′ expression probe sets and 61,919 exon probe sets with probe sequences from Drosophila melanogaster and 60,118 SNP probe sets focused on Drosophila simulans . An experiment in D. simulans identified genes differentially expressed between males and females (34% in the 3′ expression module; 32% in the exon module). These proportions are consistent with previous reports, and there was good agreement (κ = 0.63) between the modules. Alternative isoform usage between the sexes was identified for 164 genes. The SNP module was verified with resequencing data. Concordance between resequencing and the chip design was greater than 99%. The design also proved apt in separating alleles based upon hybridization intensity. Concordance between the highest hybridization signals and the expected alleles in the genotype was greater than 96%. Intriguingly, allelic imbalance was detected for 37% of 6579 probe sets examined that contained heterozygous SNP loci. The large number of probes and multiple probe sets per gene in the 3′ expression and exon modules allows the array to be used in D. melanogaster and in closely related species. The SNP module can be used for allele specific expression and genotyping of D. simulans ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6.cover.gif" />
    <title>Partitioning Transcript Variation in Drosophila: Abundance, Isoforms, and Alleles | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yRWFDQWScn9kiDXSLfsfpTnsTxLXBI4TNJeEBuZeu4c","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459201","apath":"\/ggg\/1\/6\/427.atom","pisa":"ggg;1\/6\/427","ac":{"\/ggg\/1\/6\/427.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/6\/427.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/6\/427"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/6\/427"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/6\/427":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ifAy7pJzuKpGtmSFlT6WiM4thgbbon1sM6kLle1BkxJxxaMbXYpwBU23hEC9wXewd+bnpxESYgnIXiPvQA3GU03RITlEbHIJPWurObETy2MbiP6F3VWk3NCgV\\\\\\\/+lMuuuRey8Vhu6NKMSyPtjw+OY5\\\\\\\/hGR2omivPAfu82UdABip+p1wyQ7u3OCcv3rNWe897BfZ0xwY46QGHZTEoK8ZIRehWaebE6wxCOMnVdtvgehZZZyNVLDg9hOhD58C+q+YEpCRWVhN8Bz8H5mNpN6FQ\\\\\\\/fVVO+devn955DzSqPlpbx8JcZrScWXW+lVVKwYUnGDEKRF1Ne05jx5AP6S0sofxWbd\\\\\\\/pDphFAHNgIOdtNmZZB67zqHDmWiXN2VsDja96dnh4sut2WjyBh+0ZpuHJlPl13cB9Ai2n7V+DuLtpZ149tQsKh7Wy2OCBzz7eD5WWLs+NOlp8dDVCk0eo5is4TIgVt4Ty4JutlZhG3Uay\\\\\\\/t5+plZAe3kAzKjdHKOooJFQOQAlypMHq4p0Z1yE\\\\\\\/qCcW3tj8r4ppPKEN\\\\\\\/I4rJYsUVYEnYDUw+7hAdJ5pOryi45PN4r4kNcMMMIElvlaH4FVXAX+wyXJuQ3agflLXYP+O51syXOGRHmvpx8oSyEyPGdPAJsuBHUhdSK5Lg3E2S1QLYSYIjaSkDtc1K2nuej5eL19\\\\\\\/8xOked2W9+3wxKOhlwU29B2V9iBwlmoDSldUKIj3lR4aYPtq+xRlZ\\\\\\\/17ranOCYgBJSfNfgBOMzPqrt2Y+VpuKv\\\\\\\/i06OdqO252lwq\\\\\\\/fSwwMcnzalz030FJ3x6JlQp0Ro5fEoCFrH6CyGOXqPrXeYn\\\\\\\/SZAm+Cew5X3YnKwjzFIC\\\\\\\/o8MS82CpOvX89IC32EYxuhiNM4R0c7Kl97wyucJk9JfPRXLonJfAysK3VboqUjl8l\\\\\\\/0SFbE936HfzoaOJWBUerZeA9obAVFbLVKvjSPunA3QOfmOsca08y\\\\\\\/Vt9DDL0dPPoRzOPwxCvKAZz+glFAQAVFSjbYkgT4ONHwgYbEcqzU7QVv7FKlLwBYjP7VmayheV+Aax0nuu2KW2bg6In+w\\\\\\\/mDOVdg3kB6lvPMtN0sLy7fCVtJvamY28CYKtMZFUeA8TNgyL\\\\\\\/7jtX5rE9nEwNoefzilIO50jvrqv8IivWIZq5i4MDI4uFZX36tIkY2LAHrUnDx5+3V1gQuJ8i5btXMXjuaKFRYTdOblrtsw96B7kD1KM0p\\\\\\\/1dG22Ksh\\\\\\\/rY7cFfpmBreL4OAQn6fU9vBqW43UuaPRbocTmLk+tYkGbaSmBZXh\\\u0022,\\\u0022iv\\\u0022:\\\u0022flMG3ZuqWeWNQ\\\\\\\/z4jGM67Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022653ea49c6650a9a431b8c7bb53c2b857a0011b7e4a99aa445732991503f81c45\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cOwShMwd1am0cqV6gAnVpe6x1FUTs5Kx9EE3odtmGIYqJUyYN33pD1wiyf5COME4o3PbXLQ9rlGDEB+0K4VKEes+NCJ\\\\\\\/85AaJD5y6hCmSYRsx90clT3G4NTE9HApUu99e9RCVmPA7XfGzAusK2uFlkA0uI2woZFYEEr5\\\\\\\/FF7wSg\\\\\\\/VRhznHqy0im+rbtWhoYCK59YE6ohmy2G0GViVyyjiwN5AJtR4Y7JgvqSt9ers5Pe6L86yN4pBhTD6D9Bs8+9a1TNFJ0Dtij0Xp6gj+mnC0EA38AsaeuYEoId7K15LvRX1g4Xl4VQ14TtOyQ2kA3nY\\\\\\\/6+TuiOtaOhztwB3isJjvkh9VAzczr7DDeBPgg3gqGEUPUkT2CulZUDk8BNsba2zJoXwgG7UhA048keCJDOa1ta+zbozH2fazOzdWsfwNCiTGRRWmix\\\\\\\/80pjZz9VLCEYpMTugfp7qW+0jmh6JLSfmKpJlbJMP8\\\\\\\/tdevnKGcBw5Ajzm1ImR8Dkc5+Uy7ho\\\\\\\/lGN4dnqSUrlNoAyir1rurGKFbn7hNXSLT3nO5kincRGZLYjJyNszHHW1+6f3hKHV2Ew\\\\\\\/SEFyOJGvsiU+RqgHvSJOBLmvZIXNO\\\\\\\/44sxlWGXW1VpQQdvDK3d71eAkhA1NAEvcyyxKsSnDAYNTVD2dl4i7a5iDGzNmVhaV1x5vob5SpSkDefclR5hcT8qnWhB4BBrFHRznhzKJ9xqhXpskuW1TA7qHunhHP9GNr7S4u\\\\\\\/lMm5mzP10wl9gzxGsCTzLBIWrx8N4egSmqzIDCJmx7yDBWeCJkBn9lTvhAcIOtz0z6rLEBtSVnDSQAkCC1vK9TbdbXxFQy6rD6Dcz+tTMB+6UUZ0T6aWUDiBUhgrNip6CRnUvZFgn\\\\\\\/HqHbef5OnP75yo4sNiCje0NmOZ+6y1wltQMclFxjopeReJEhLWFANhu0x1M4Ns+DMBiegmsGuYptuoNsCIFt31co7daoGCWl1O8F4TqTys51l8myEBpFeCc3iD2ryGhxJ7Mr\\\\\\\/LBoAxeZWVv\\\\\\\/xzcA0qZXFtixn0JZ4q9w85\\\\\\\/ydAKzblV6D47YBkDLC8Pos7rH6y5+UDkL+0I1vjuuRwJe8OZ4Ej2B\\\\\\\/mvl5PMcu6Jcf0FoTd2lMmMLJjNWqE0ecKlm8VM7Lgg5CWh+GsmqHBheg\\\\\\\/PTQuErlJJw9Ocb49miHgnc\\\\\\\/tLykSAqKT4THSY52ze2cRi+8LRJ3FEbOqsjexUnErfEk1UMeBFfUoZKJ7Sb3mQxr1irtPhbT\\\\\\\/guqKORUSwE7XJ+iTUEThJliwgh4ih+JyZlIe5DE+gC9jRHl5Cz2kdqsAyMrM1Tz4qtv3wTJvFtxoSjt6MWgEDqg4TIJjrOCHhA2PggKH7L5Kp71yxW4Oc3v8bNfba4BXCil3LywSSStplRuyF4X4mOPLLJCayiV2i9LdiJGJv4t1HQP8RLXIPS5B1NLCb5IPXJA4Yo1eM\\\\\\\/EkgfwxoWsPH9NdPcQh3dclt2U\\\\\\\/w4VG4L9RURUvN867Y0vWVxKRywMFep9apdEMEYvXYrgZpMNrujiQRxgnmGQvpk3JPrwlfTZKFeC+ckM6vFL9Gk1QmpDm453S1GPhrxp6Dj5ipVpK0gF1dmN91OqFtfgoAjdvcvpOTO9yXLk0tpC5T94k3OAxCNTOkbjv6ML33smj6YRmDPR8kMFW1rGLjXCyvuUC3XOry7qylC4NxoLnsMVJ4WUy7rX3s0rmiplBa6cwbZODRZaiemCWCIfKclhYPEmPgDeBe21Evc73LiRCnfV5l\\\\\\\/hZ6JQ37hidbT1ZOMaflXQFSXRdzaGH+e31DPDklXNXuIFlRbi4DvZj9Cw0achh1GycZsRZ0UX\\\\\\\/XDYdmeDeLqJHywGN3\\\\\\\/qNPZBgFIGU5q57OTPUI\\\\\\\/B2xEvimPCBPupnkr9gLEOCstTQ2kjV353SZyzN8ungahj2BlqhpcXKH+JvUG6wL2U+ABc8K+gLY49ZSGZeknVkyM5EtBafPLpLM79LNLf\\\\\\\/iv9TBaQc\\\\\\\/3PewpRDAf0IJpCgWms35nSOURUiHcl1fLweMmL4pdto37C4I+trHqfMBnpEEQ7g0x2SAMSOZa9aNJ6DBgEADsG7DtOYEdgfHU9073aUx6hUpmvFgwH63EX\\\\\\\/BnDmW7cYzqnz7WuQilyAuayrwuOnnNVr43JiX3\\\\\\\/I5fJlOV0H+U4cK3sPaR5qQ6e6tTHtmoIlnRZuVwCXLCe8AoZmzFzNGZUR9LfblQQnq1L3IP+SZetcdRVdeq8LmnpAxBAgWeF1KjALWOFgwfitmyOFmXzvQDLWTAGpBdqxZwW2H1qunycp8SXgXDc0KJjmf49o7SMkW5pYhM\\\\\\\/ALdoQzQyAPYBfONgE07FAuLvSNzmkQX32fnQCujO8j+lqoIGrnDInMj70m\\\\\\\/rN9MNsokmFmUdvqZY\\\\\\\/e\\\\\\\/YzI+eUt+hJQQ0pTfw6aMZYxBKlBn0soHh8jOSlh7mTe29MluutbpWINtXa9ZAJ8Vgdj\\\\\\\/z80YzouHZi7G55XKr1P5\\\\\\\/md+c0avtD\\\\\\\/jiaeN0L3QLLYlq00eleQ2jUIBDBxNicOZtYvQsaeIBZ2i7URcNKa5RkahdMZiXuwjDMMr+Y+s72sQ5XxYtElMSm+mqih+9nRYtTid1WRbm7vnOM68GC\\\\\\\/CgD\\\\\\\/S44yT2dk4xpvVjEMyuzG7RV+J+wCLpHtec1sze8r1SeDGIr5Vc23SogSgFsbVo+8SS4JYKq8X\\\\\\\/tGZHq8qysUb+raBBks47kWs3oGXxCf8IX2cRd50zTAFQGvncwWigRSSKiU2KY1DM4wko1hFPHgOK4aEsAgLPoLBnNTYDtY8s3Wu2cT9HhTMemrD\\\\\\\/dOC9F50uYuPmmISmC20KfdVu1Z8GEEetWSRbf9AfXBhq+0nltFgSIsU1gXycptBisTuRduqJndLg3tfv4NntVAmaB9P9tUKtO6B6\\\\\\\/wPXgiODonnzif9RB9TaEvnMUwnRofiPC5LE\\\\\\\/JvRHcF37qwv6uxYPgRJBm4AekMDtKm+mzextTRNB1IIq6OkJ8ClLhimu6piZLbxQrMpTjTShOi5umwPzYQ041D95wEWAzSyNOAfjho3wmOQ2bQmP7Q5coU7y4bj6YpoCRcbxMrrrCoQIzmMnp61b4lUzR64PBLZhzfyI3moLphYjT+K3j6lqyd9Onh5tdTUzqrTgEsVQ6ShF3OzIZZ8U\\\\\\\/zBvapHy1F7OLP6YISyV7FD7WoCSoGu6wYIaoy1OMwHpdm7\\\\\\\/CqxG\\\\\\\/c23yeKWrG8Xb7JqKq1IG\\\\\\\/GPOqCU3rGIgqEsNIn54GxZmeGE+eus5TOo4SrKgzT4QhkzTiLkDxSBcAeXzbqpsRhNzV4PCeChZJKgwxBWwOMc4mqQ4jjL7mtH26pA7EStJJylr0NGIaGsLK55HBvFbIgMf4xVMIX72E5y\\\\\\\/ayQgrK82zk2tvvEdx3aSv9OrKJuT2vw5Fzxk8u+OQdplr6GFmdNhuttV4VHp3QLXzqTJVCZxPInfnJaV6r8O2SIe3HQpuYymH4X7TQh2zlYC1cqQ7rocWvD1LDMH8kco\\\\\\\/\\\\\\\/Ry\\\\\\\/xz68nwAQMpwMHkDWzeU3Vd9+y5aqROUn3+A6XUXgEXP0rr+Y88D46H3n2iOBXy2qHM83JMzOlzqPC9D01GokR+A4CE8\\\\\\\/jZ4srKj8rx3rcB\\\\\\\/tQvida0bN76PHHPeEXFl\\\\\\\/62qBjZ+qcVG4ToM1kIlNxUHCcuWe3cxsRmlIHFgXq2eEfDjYcbPEtXwFUNeI74jSmaG1gXNJmuCOwQU8dZ8br6uPvrr\\\\\\\/6l3DSv691Cgo\\\\\\\/HzfadQJMN56iO6LI+rMu9KOCO1RJrQDnZxSv+jWlUPU4PNQEMEOwQ87ERgLobZz0dsU\\\\\\\/eel6F6HRkiRxIErAZKL5LzoM0Dad282GZgfn1IdYEb3ys4FmCdozG3BwPa9xITjdFB3c6i8H0AC0AdM6LRxKAMeTOjmgw5EXMLhOejKakmbL\\\\\\\/yhcGWoffMFdlBuFw86P\\\\\\\/huv1+XN4Wch8AyOKKBj7kNmSrEFjEK6t9A4XZY7G4cjXHDHsdvc21YuBrLXbyEqY5DzOPB3GlBA7DVlzh8XazUixKOW9h49NWxo\\\\\\\/tCoVLbBJBh+yw0ngfJw2RHu4DbmBhORa30ed3tj40yLyL06QplSFzf1OCWSgxcnvL6hJ6QJYcydrqlNpI9yNQ5UZlYnDE0zgT\\\\\\\/ialpIC9Aqt2wQh2Z8TRHCyUpMQYcLE7AEm9jXwfGynavbd8bVmOtvG7KzWmP4SnbMW0UjhAR\\\\\\\/zdemiv6UhF1oDgyb9z461u4gE+4BC\\\\\\\/QVgKzZYGsLLz47xpGmtAWl6dzVPBweynBzKlFUW0M4issZ7VVzqjjgle6HIEVEO6xNJbxTIkYBIJ96lNQSNALDFtaWRSA79zOjQi0jTG5C7tgjmX\\\\\\\/3hv4TbS7zKtwmsWqI1F65arb87XuuagDdMQ6gipecsnha\\\\\\\/FUamKp2C8YqvtFWHyckE6fkwbfLOdmfuMBcV0G\\\\\\\/YGc+BfyAhzSoTQwDmlgmnIE5dIBQmGjZIIr3pe9erQV3noSZbBKoLTg19so2pS+jC704L7Ea0zYHJUKJlZb1z3Xpi2aedIpK2UeO6AWL+75pWBJraTj72\\\\\\\/x1l6v96I\\\\\\\/hJHKavbgxHAniH1e\\\\\\\/zbabNnieml2Cs\\\\\\\/48imPj4k7a3lP0l6\\\\\\\/uo9FYfbZURHqKojqzI0CiCCNXXgO9DLcHUU+RHybUZGM1tuyH4xKc4e9ZLKpu\\\\\\\/Pcvn0IlpBqCWQUQMLLWJjjjZiBwrFDUHobIXseqR3Akb8tEB\\\\\\\/HBt0VihwI6KQqMBklPZUiOhpymzRlQBadEyzgKzxV40qBzEB7E3Yihde7joZHEA4INrpz5YN0hYyjnIUwXxdYHZNH1II9Rdp3vRRMlD03yVidIIO0HJbMa6OYf15jHkI3HGaFxfGqB\\\\\\\/VIhmWQHTNaX90CLg0SRtLCarSzV3ri+K2edjyXzZK4CZSOSIao8soyBvubuGYCFns9fcWFFlvfI\\\\\\\/xeg7+7ziDPnpPtuZ8iLVlqyG05wyWxsOzAFhy6+z9bzWhTXYXonB018raLxdC\\\\\\\/\\\\\\\/nhr5mLAWcONCXtFcOGsocPgHorh0g9Fq4jdVJ3WvzWalZvSAhD+J6J5gPQacovnMfZ6Xe70OcO89Cy28eoYaLseFBhH1rBO+xfSuEN71HFDbW1aurV0oNDGJ5HLBwFuCO4sn04cc74JfSrcFJmphbck+kz5IXrTjjdjSBUW0hZYDDzcgZawMvZIpGa\\\\\\\/3a6MIgTI0Twm4Fux\\\\\\\/ZpOZpNZ7fmHxgNICo16bBKp1qSJRt5WojBVXClFQUB0URrKAd0mnl559B\\\\\\\/ssnWxIfLyuZlpWy3NBjTxdC9\\\\\\\/YObI8iGy+Lybu5Kiuq5F6auNAc8RIeWtw+BkzMtiHnciYeH20MyZnVJ9cqO1W6iW1A3bxPXE7wb6\\\\\\\/PKzGQNkGchAoivabZvjYQVRADtpb6GFpe0OD1E5eqHRvyxSgvcRjDQuEP78eYkptpOQSrjBIkt6QOKAdBlGUsw80goo9RfXJnuIwUyYU2vWN3x\\\\\\\/cgdfVzysc2GlSi2WOzrvX\\\\\\\/uHWJOKDnSYAHshvT9RoaFpqgCsethlOcN3OLdWtfxTDsHvEbWFVnOuPL6bpwR7X+8IYWwisD9MwB5\\\\\\\/jyu4ho+nog2WcLo\\\\\\\/TNmICCw8vLKv33TjfMrDEibwo4WDQ19POHyYDfn9CjOHsjHud59Ba4vx1lf6J817Sf43o5m2DtlRDGyclrnzcyEw1KBdgVGjdFtsa6Joh7S77ZR+gJqtJb41Jr2mbYyxK4wcI92wX3hsVPhHlWg0BVeSBTXIGfbgg+NMwSp1GAntsjTq1bIS7Br0MH8oI8BHc6OjCzyZ07mWtHchjvxjf4K4eBwJKOnXf++4hqwNzyZV9CNS+UjE648jCDTpnX4X5Bc8FEp9B\\\\\\\/LDVFkC3BX6z2IpNzCNyOHhR7pz7tKiVwmA9uYD4gviGSISWSAJvpSF5NnOnT12xy9tqly9xEvcJHbObqHKU40XOyRxfovSlt37PCrm4BooOgKX2ibmdrHZ0Enz2+KDd1qkMtlwzxjhVcNfmiQLFOWxn7et6yoHIZ8LkwdDvfY0VvDCVDkLbatm2Nqq2acwfQPQYndUZWA1ipYZlY7THv5OmNXmB7G0QtCxDdZzWAF9xiMCBhBL3AVPrDmJ1maRMUPN\\\\\\\/V975ih0X+m3pT1GIecTr4QfAgMdN2h\\\\\\\/lvbjPm4zCtO\\\\\\\/w4gzrjY0fziN9\\\\\\\/nUNMIB+0RfZf92JeeonBY\\\\\\\/\\\\\\\/giPExoY5Cw4xovHKBgARSn6ElMcUMVdKSnNVLA2lwb9l\\\\\\\/2GJXpHShRcoZot7FMZ5vA\\\\\\\/GWKk\\\\\\\/gVgun6tFcv4LHnicXKZjtWY7PwW+ymTWO8b7QlAh6hO8\\\\\\\/GKtA73xaHQOjUOHRoPn\\\\\\\/vdWG4ouFoLGrWFPufMryyP7FG1YbPS+EoNoe7pdOe0OuqP+bqhnXQJxPP8kfpKlNFPJTweGIFd1fD97J4tSej1RUspjS47BDI8h20kjlhL0FNnHmfOgB4iDY2vygXprgh3BmDWe3Al7uwHJJ25JdWtla3o+kP+TCQ8ByW6MddVD9HFc5V2aHrlYxkjdDjSkL6qhh8YxDo9edkQNvhTguwCCnkkadtAns4R+ZyVwP+UcXrXeiepDdpdPxLA74KB2gigXqm6zq8ktlfOKJyrFQJbNaGdTluGWV3pJ17HXPAduz8LA4DpQT+jLsF3JkT82OHVC1E2KHR5DVbTQgOs\\\\\\\/07qBDhr+1Hw\\\\\\\/OAwkAz4tdcvik4v1mP6+OopJjg2EmaJbIghSW1CKX38RSwm5APi\\\\\\\/JjZlCic8f6aVmgEAnrZESZqywrlJF4alW41md3slWwOT\\\\\\\/dW9td4Mi4qvOHVCefpTMQ8n+9wEOPKtNv\\\\\\\/8j3psPmpmm7Z0LeL4iDcZHhSR5MdEWRnrV2rZFn5FypF49cHxeJsZvKlgTpJ3FsrFlPVBz+S\\\\\\\/f2SdW22+seSW29xzi7zMDF556vpvK5hWHdGm8+Yektfc+KyK8nSL8yJbBhfO8kM9TYZ05bv0S4\\\\\\\/t0\\\\\\\/zn8ROvVhMpjHQQ1O5O2e0zERaJoxhg9iEGvw6EWctpwEevT7CyBU35yDdTbTEZ2YCutQ3pbUUUetGgxpmJ03lp6buAaPEIknhFhrJ\\\\\\\/qPl4kn5xeoJ4fDSGIuAX\\\\\\\/TR\\\\\\\/Qox1w+K2ZHpJVw95j0ZwISkmTG\\\\\\\/jhzQVQErikiFhU2P7QRz6MmA0tfxzYeH419n8OJwNNoXB9pG8rb3dy7kzKh1mJBjQ4amjeiCiaHtQ+Aj0WVbv5AaRvBv6KGu\\\\\\\/4T8QShTf2NJsDctx9cE8Bn1C4nDdG7ZNcxnKJV12TuCelxL87S1BbGwUzkp3sUd\\\\\\\/mAA6eFYSgJXdWgNNVZtuphXXIcB81y\\\\\\\/kO8ISatmNsVMYiFaBPhutx5vOkQpZtJR6RUrs6\\\\\\\/hHvhAKlVZ+5Il7zSaZwkld3Xq6Je5DrD3Ryvj3K3NKzmsCmKDIK7x0vAvoL7\\\\\\\/9ZrHriIOHl6jkyqlRM4xAD7WKOa3fNq5O\\\\\\\/FVtb4VR0FzLbAryXuNSEupOnP0kL9LRyjQethTh450vk2G7ZnFQQEMVqlgCqNnSc1Yt+c\\\\\\\/8OAukbruw7X\\\\\\\/N2jOi9N4xByss1hgk+aPkoMqbhyBcZ2DtgMTMbpGPf96KEOVcSHp5ArCtzwYYXr+\\\\\\\/KhP7+YOnqeC6Z+5cagA2KWKqlEF6x5OWln6FP+T7nJ\\\\\\\/F+YW5bHjgmB5759feXX5LCrVXQYlXq6rcnxBxpV1JML041ANFMfV5y8IcpS1hTB0SeKOHU1AvYUT17gXJesHaF0ZA7k2oOkGLOi70waDf5A+xIEEudV1MrO770uCW6CrbWSBsyc1Bh4Ux4GByx+\\\\\\\/Tx3UPZ5r7fzgKmZhH1WZuoqstr+8vJaStrsepEoa8JISBIH1nEkfrsbM2PndeojHhkS7Q6vyFC2aWkECHCb9u1uf+7O8kQh2TzE+QmD7NEn94MAWQwuP5dWnGG+3ANupyb\\\\\\\/xvuDNKHvh\\\\\\\/Ry9s0YFnzIR+tgp9fQCIjCTsQSzGbBCYbJjt4munq25GNJ\\\\\\\/VNZNvzOZFh1GfpbwtNV6jIK7s7kfiACLS\\\\\\\/7Yzdds0OaG1uqO1de0pwu+UyKn\\\\\\\/4dtD65qV4XsZqiPFRiCkyOlOkLBehYtUeL19EQn0ORoJJRr\\\\\\\/6QFSayrc5xqKAAyWkPi\\\\\\\/eRtPvNEJx7xxNnHYHb3HApOFkMQos8TBTMNVeMCyStiSxa+V51agmZP1VbwnOldqCqAQpUZDIl5Lwng9vE\\\\\\\/xRGTS4tlCPmujCMuvuySzYam3S4oFf50JctA50MFzS2t8k9uhOldQ4MZxl\\\\\\\/DqqzH\\\\\\\/cqARiPdEdg5krCAOwgG9J0hCT5OBjL\\\\\\\/4H9h4nAjX5OA8Evs05BfWSunFhxxxampUkUyvsI37aBe4T0vksVbuzjAklLHKdUkF2FiDBDAfFBYS4mLVmvuPi\\\\\\\/95h0G5L5Qp6kMhjfZ9h95eLONrBiPbLUhcF97OF\\\\\\\/0wi\\\\\\\/heAqb1fOJVvnLGXidwxtVafSVMdCW4bhIio7ERuEA8ReaETCkdBoBxE1yJDSL4yvYw6JFDYsg6Qu5Mydyv4PaxeL1Lz5UJA5ZBVdRTESdOT8LU7oYASjgbuLG\\\\\\\/DaP\\\\\\\/7a2oluCkFKj0b0nzLrJ8IbqH28LlQt4gwenhj30E2rmYCs3NvQqwUbIEbMUns9uir9RdgHh5a07spsYtxJ4OfOxhPTaKkrbeDUvElyJ1ALRh87H11v6QANZLUlZCcgLvAwuePN\\\\\\\/\\\\\\\/WzU0mEo1gpb3wJ8l2rKtnNFkGHe+YKRjoSOV9pmvLb8\\\\\\\/84lN8lm4U4rRmNf0sGn\\\\\\\/ygjIb0l9UedgtF5cwLwZ\\\\\\\/7ahvIjfr\\\\\\\/mRy9jkDIKa\\\\\\\/nqFlDJpHHxnyhHRYCbfBqbEwmkyrbBwkbB\\\\\\\/TgD8S5c1uvPB47yvdycWK2i5EvOY6PFvZOxZN0DAclBlnva6HphRe4ig5cP5KxHSQNpBzmrwrYpFX2Bs\\\\\\\/AaB21n9ow6Mk2mj1jIVfusrTbbfER056J1Nd2hyzB\\\\\\\/7JBgp78oe+3qTv7s2vAreTCCe4ZzcgIUMUyaLl\\\\\\\/OGPJFY1sVLOZfpmzmrHu8O1Pb5s4B5t34Qdw3NOWA4NpenPmybjUyuKtMUg35q+2Au2rOx0Xz7apAgX1U1Iedj41njE4WlvhVCKpN6GB4yYCXU8GdLZ6S+PmPOYP\\\\\\\/1+G\\\\\\\/SUa1Vp1EkSNojZJIhKxcrfhifxshUih7SCR6fL4xfNjTCaCqr\\\\\\\/qF2qS6XijVKpIQIIQXW4d1jp67nS9+uqBVx1xO4G6NWebd3IM8l5ExQ56Crh6IGioaFAVq6ahHYsD1uehmuk6icQeEbwBuFptVehYwra3hDe8sUUcNoNeHLPkYXhVqL5YjwqOnVtSPWTFgA3RFwRWMAPYaWZmmSL9k6lUb\\\\\\\/F\\\\\\\/rh5O1mO0wY3iqfbndLQkUv7s053Jq+jDT12IAXgtoSI3bFeV342xHSkJm7sxQ2x3gFyho2CB7fN2RwVYb45yR5EcmviJdndt9Mbj5wOBDXRJcmjptJHjRdFuiZcQPtefwLAQ0d2BNNmIs5lHnBj6JPCHKeeZepaD47bt\\\\\\\/hUFcue4wqece\\\\\\\/mSbDrCYDbMlVIlKzW1+IFo1MwG3GVCfJ+0vIA1dBAh906Bl2E2QsG4ewPX8u45OW4HRs0uqq1168yU+TT8w+BAYrvNy+5iHe+Q6P+m4TBRF5ppkkKM9Bq3ajVpjk\\\\\\\/wqxPJ3dRHQu8xHI1YQdUMMaye+umPnw2GPnE2mY3ixaUns\\\\\\\/sqLnHs+IPM4wQUcHs+b2GHfybFxPm9JtDxFuhtriGsdhQE\\\\\\\/tGp9yQTSl4jg3xrEUWjGtaF57v4h9gjnOYkB3eV5GZNU+dqIy1WWl3CsUt5PZCLoT3Eu6kNB0leZDwfRnnuOyd9AyFcC3VotVCVuyZuOIbQs+\\\\\\\/a9wja8yUndOk8iMsS5n4\\\\\\\/K3B3dDG181lMY6p6kbxIkUxvC+Q4iC9cTJgBCiX2mqRoLN84fBSaOpidgcqieAhI3oqvluUbNaGw9xqUlpvBwfIlvU0f7W62cBkQCZYcM2mzczMiK736RP4re8gk1s7hsQ6ahHRi0iGNpGrQYRoUcEaK1fZcaW4VAve\\\\\\\/jLam\\\\\\\/Q8Rbsd1t1mYvyDyVraxZOg4tPm2gWNTFm5WqW4mBeRtdVXQFVW\\\\\\\/ocQGZe9dznqJw98RTTAXsy86eQRiIDt3RdU3DE5HDLyQhcTSlLWOYX9RjTbwkL6i087pblIn0bw6qkJkYxgbVaXoYBIfU2tGWVh85zgw0bOSmGyjXrGuyXnmq5G7YmvrOedd+wY5v8ohvWm6XhUmMq5DzzbabsmHPay+6OdJBNslKFfZ+lAztPxqDQGHTlYfaUm6RGviI0O2wDPss0mqiDCkFetoz08zmrX65DU9xPz7d\\\\\\\/4mbAoFE26lisHWGtI9hAOnGBo\\\\\\\/FDmqOL0tXUaGo0lhBZUYtRCqcPXAAp8kcTYJQq1\\\\\\\/Om\\\\\\\/kKhl7Y4\\\\\\\/\\\\\\\/KgLP\\\\\\\/6rkhWIqvFa3emdQ5QJ6XXMn\\\\\\\/VyN5q5OZqZwvSzGto7H\\\\\\\/LTXWLDWgnUUWZmG4JZLgCJBrbgr1at2hjdOuwYxFfnltLOGa7eWf\\\\\\\/WI6ZXg5Mk+ZO\\\\\\\/Avcio6m9Gf6El5nyfrlJAw7elv4HpI2cEjQhSltylQiQQKKEiANGovhBI\\\\\\\/kT5IuUGUavkLWH8RkVRlog6w4mOZrEbqe3W+4lhpsPRaWzp\\\\\\\/Z+Aramuj44yo9KhJDihw8frzlj5lDC4kpntaZ1Xn3K6pJQ4DYLzCV\\\\\\\/9vWKcJ5LJG8NexWC\\\\\\\/VrpN1DoLQVkTxOjPHo2fXcrzsuijKG\\\\\\\/GFAzIIDq3ZXFG+nPFVC1LER9BZw12z0qkjyqNLZT\\\\\\\/btpA2b2r1s+\\\\\\\/7Y4YcgRw84W1\\\\\\\/HXIBbVigkzS9wp0QDbNdZ4KeDS7geK41kHyfzEow72U8kVtaAwRTRqCWRlQ6UoExLU1FUCqIk4x4a6Plo9WIw+djFlDTbtOD4+eGb0tTRTunV1APy\\\\\\\/D8nhGo6n7XduUC09XymZi2C+azNxToQ65uYVY2EfCOy4IHS\\\\\\\/DsU1OuUJPU5Rc++5eqwdLTnQbjg\\\\\\\/U1apB2QHTJaoRFmYos4pn6fwkwhvezC0WpE8h\\\\\\\/lna1wHZf80WIa8Zuc5WMLMqnDabicnG8qFFxW8uFYtr5Gxeq68pjcueYpkOD+9A1hoL6dKzpmGpMyBYEUbzQ09cBZW61UB56Bx3VahJPnf+XCKRBbaCTN6UOR12DatvSJioSnITnlhI96luUq3eLwPbt3mtLTkN+day4QjDW1UFRBVJGHuYvsXg1txVRDi1UzyvqkikhkoqyOWdpTmI4M46snQXCy0VYuBvzIELFkAKHnl7q+ZdhAo\\\\\\\/zhNdUX8cBK3+jMkhaxK+Es0HlC1usKU7LQu4\\\\\\\/WSn7E1gW+bAIn0iJZSfIaerJNG1UGGtL\\\\\\\/9s4NqtyCc3B\\\\\\\/Je68Pq4UbXboaddvRZwTIV8ct3T+4kydsUnWcfAuupvaBiF9mvIotHYmsvg31G2qGmW36ptk7pDFilcz\\\\\\\/Yqmj7H26dyRjroswDjnI26H4FZEpGYrNNDsLhCBBeceBpkL1pq6FPCyYRF+5SK\\\\\\\/UNp7e1s7cJm37EXixpVaOTMm2vWNVs0ewMnGzKN2\\\\\\\/V765y6ajcZUaHwG1BiF8LUrOBJ2p89b2perUvmbxCZpmih8s1COW6PZ9BYUdtcLLjxdBhNTRCa1fcRrS0mgAe09RQRKfwyF1IRWcQZ0CnfS15dCFJDjjj6yYeGWN8ODYEAG0UuXV\\\\\\\/J0x771h9Xupk1YZbTBkCbHmCMhlB7TYz94NPyj8mVvalkeyejZx+DU+PZeLyW3ZQVdjWy69VngT1k8NGSveEb\\\\\\\/prYWb1cF8I8DJilxki3Q9eeaIYTJkk4S1SybysmGV6jBjtYW5OkWgMcUWjm9BVoHArQCGkyZmRG3vIIpSKGTNxkiYKc4vOj42VDr7aOP2Mlxi8Iuyz\\\\\\\/fKmHyNevIzTEtoua5hBkZr5UuzVZvqAIvnt+iMRTg8QZi\\\\\\\/2FScp4myZn1kctHOIABMdRYKMrdTOhkbi+1BCwH1kB05ZrSR9\\\\\\\/tYYmnUHvotbzamOrlAGlOFJlcEuyXeu8gqA6uPeCAndjAtcmvDG65ZoOAf5wsBv0zkD3qlhJsuf2Q\\\\\\\/r0SjMaEdFToeqywEPLkAMBxwK1GikdNEl3zkTU5rkp1NnIxMxPs8JvkdOHp1Ix7hvMvPspO2iPWmzwJNbKsran5zKBS16iFe2hnNp7sdM0vOfL48usLJcABcf\\\\\\\/y7dXdvALvgYDhnHa4BEZvCU20SBHTBbj+BDTB7DswlGC26E\\\\\\\/ZvyA68Gm\\\\\\\/u3Vdbuv8yh7rnwYfNDRmMSvcVQ1YhO\\\\\\\/tdbY173rz9pJSn7gPtb7krNWrQFz3wj\\\\\\\/RqQ0i3hyIGFPXCJ8wOI+KHX1hyBmzMEuWfwVc2dfS2COTVcuJX2wKUXQIjsuW6OQVhh6HeHblIdjJ9l1Pzv1LRDP\\\\\\\/\\\\\\\/gKU5bOzecQPYSr5PfhmapzQ\\\\\\\/630BmI3vk0QysSpUwRGG5veLDS04JdBJvDqVgg8W06RnSdpU\\\\\\\/04jURSmjreB7eY3KXisRgr3f6pBlJH1WUIt2oAHzkMUpg7OAjUvYWJtosHcDOeQoSECt10usl1ZZV6vGFK+FYzxR9in8DgnPAVRh33ox6XuPhAfsXrOZgTHblPeVktz67FPOKDPc09mugjIVGUo6MTWexFXMbLH\\\\\\\/cXkeC8uT93CbSnWPLQspbpW8BoDT+B7+YNTxpnkS57dZuuPTcHh4I28JtSFyqzTuUDOh2Y4rSZ4MuRkMZ7EYepb4txqwiGZg3ru3SFrTZBjvnyi7q6+ZKlHL70+3lvY9lLLO3EN2KGICkyA44NcHlM7xguKmkM8hsk8tqvnEyBX9KS3daYADtAvnInj\\\\\\\/4FwiyEnKlDxvaEAVwidUSoCmQGd8dh+YbD4KFVLLjZJ24eRrr5bGJDzDg2LhIDSJ0CCGc9LuBT0WorMTO6CMjYfsL41bKiLOJrKcYWwfOEbL\\\\\\\/xAiQGWP+TcLXyLII5WkVImnj0G+xgnKy6Azcop4WycPc\\\\\\\/P2KCHUo\\\\\\\/HxNTfEMqyvOY2fKYRtLPX\\\\\\\/n4lvq7dddj5LvBCtzureqLNMzxf5VXLs9l2Q1BmvMhmrxDTmA30oxq7AGaS3PXfaBQX12Rqotj2P251VP\\\\\\\/fFGDlQ2kKkvnSxw1dvsnOMFaknbFEFgXfeTWLTo8WKJYAJLBJnE1e\\\\\\\/NGfrzKPjCVRcS+UsXNpVPfRjrxiytIgwf5Zr+KOyUSONtDHkj2QHXXlYNiMwyzZY\\\\\\\/Gi0xbMWlz36M9JGn6OPHgiiTXcPEEOt2OovU0GzJZz\\\\\\\/GOdZoDkY9C3jGG6ZapeUxpQjKpxxNU4rervOd8kGMdtwnIfffW3CujR1AQqVjMNeg11yyNIaZNRYJd8BPC2r9NegaoJ+wiOhB3T+veTmh3K3uXTvGT79yq8g6IrF8r32M3ab+r2nv3R8l5+4MBAhnN3Ur4Z22mI4xOa9KQ7Z1yQue+Ea9Dk88RgLPnn2Sp1Ods5\\\\\\\/4HqkJSDTnvD1x7jKh+LR2jIPFq7P64BEbJo7tWk+wR0H5PWgr0UfkkpYHZsJoBxyqcSMoKFpUq4TseS9nb62+Mm\\\\\\\/RZmCYSDwm5SA7k6qDHtdSVDqcG1Or2DLpqEL7vgnHfDYir8AUb3tqUeyVwIYmLhHhHM0HuTayLfrdbujbTjplQx5hW\\\\\\\/NWrfn3i4BTz3VOZo6beiYCgfZ8vzDKM8iX0eZfVXiZa6FuS\\\\\\\/ZrPYw8KA0V38gp2b3soJZCshkA4xct9FPanxbgGsDy\\\\\\\/U4zlDxlyvMQrr1iZGESRNOBIz6qdbsAnz\\\\\\\/ZqOgxwr2g2eCWBpgKD9jhOA82Cb\\\\\\\/P+JQvUP\\\\\\\/bPzO401MeGMsXiokDTTi4gxqeKj6SXzKb4zQObpdQ09JZA12EYUNCQOdlwIisHI4QXZ2DVGDdQqbtK7MwvwLmDHuJQOMLY0xjlJug2tqmaTBQ4Rau6RDj2kHqJN6oqJaaQ8uL380n2mTlL++YfKZELmTKhOZhAktuxJp5ZIrOgJJQwC8zeWCnKw\\\\\\\/P4Ro6C7rEvLToXd0skGRA50dmA1Kh8+D0wQ97dZZFxCr6Sr8ENmmhF1EjHqciGIBBxrExzTCDV\\\\\\\/5PTXiRsS6iUxR7ZcmfArE9Q7l5h8Ss3fbxzddJ18GuOmEqJidf6uJLwVx3wE+wx5xK23acSMZA9uK71kIxdHyoyFAaSraNqKmMomNk140LCApu4jr7wRneZwpqrLaXiT60BOWZxS++k0EQCc2CdbYQM8XtCmz2gYRR1eEIbTb9TyBq8ii4OE7GWQS5ZwrFLDrJxxXki00NZqFSyJI5ILBFGTuyLWSdytL\\\\\\\/mc8XG\\\\\\\/9fWoFhFylzGiH8amM7n51f\\\\\\\/5LtyfoavkzBiH4rGAYYulTHdyzl8wu2HqU5wKrNgAPEORoOEL4owG\\\\\\\/3FU4a8S37ar3qW5+oQFj72vAuvXV0fxZKw4rH8W0uud8LM3sRCNXTbWjEANsU09COqDtL\\\\\\\/geYuj6HR9d83y7iNhroUchHGgEVVQ8BeCX9sUU1nR3sIi6YhPXB+CvMeoRihR4PsH7j8BoIm4BkSruqxgFmF2SrFpXpv+KQp0vX8PMeOxHEncTDPpezHLc5uxy9Nel23Dz977BPRwNTvO1u3phqnTAh8Lq1o+03qdsQgXkPeQTQaEBl7ubfbnhTuLZ+sY5HRjFML8HwxiOn5LDF+2Znk2NmIxZfC8yY5uQkudeVVeLyuqf6NvbYKtJ\\\\\\\/gwpifudYhkGnoHt3FUnMJ9fhmeRQ+PMqqKTV4xyCNLP9ZCmibHdVIjtUhZ4PKeRLNcHork2R90LPiyd\\\\\\\/Re3+kDsyFX7TqJvwne+MY22dFyVlxaUALUCv+r4eJxuO8rdPqoK+lmh2KYjudOdOBaga5DOwAXwBzJVoxsaDgsU7INn46OD1BseFElowCX3kP7501xYMgK1CrA7raXmtMlhJlnOpaHx65y+wBSYjBLRix6fhnInd6\\\\\\\/WJxU3uLDiMhVheMFqqcPV4QCpYMiXYvz91aC5xgW\\\\\\\/skDy3AKEgRX+Ox\\\\\\\/vbxLLzypd+yx3Sdtb2YNDrSXw0+V9YgNPQBpBnhb0CZ\\\\\\\/KAqDGWH2bIz\\\\\\\/sLjd5S4FJJvT9D4FyfoQky666yaZhkVZDmwskkAYxbnZZITE5hB03KmsYGfSDpd+L3ZDge48bUKdXsxL5D9PsyX4nBjHQWkQvI1yBBrxiyD4wwkvZWG3Ll7jOrJkCQY61GshNSOx236yJZgpimPL2EgU4r64SBG7TyF8wBYedhO1RtUw3oL0J\\\\\\\/SfdZjxh2Vr2mnjl1fziNV\\\\\\\/4izvbxSsT3cIvrNeGaxXM8wIq2s0l84CKe+5cX0qK0z1cD2gqCI31JKKEKMYFOuTEj7zoN7zDmyBPHYhhNSd6nR0eHQmQIP+cH3QxfWvhjLdZo8da02TTcWQVOUpheiEABEpH8KLa\\\\\\\/2pwbwhZwK3\\\\\\\/aRAz+VfaPcEGgVrwmgvEHE5MO\\\\\\\/2NOCHQLJz8Tv5dgcbm130t2GWFthNZBAMU2yx08EDUthmjhL2\\\\\\\/S7O9DzsvRCImp+eCPatyOWCvZw4+JVQVa63pf9RLt7KtjxHaI3zu4SyO7gSnBdDh92gUO538n7EoDaHJHPLIT8R0aBm5XYLIzkGBM4v4fi7oEYgriD383eC0ksVEDv7UrA7iRB+ykddIAlVVYY7nnHy5cksVVmG4eyAgzVM3QnPSZufwxWMNYZl29GF3DpzcLWUKWT+SatDLUFrCDJOE8UOVdOf9OIP7jWVQ6Cbdr49GWTn7rd461Fwfa9kB1lCtQwkcp9f8qquBncPoMLiYSuh4AIUeOlwlvPUTfIsV7A\\\\\\\/dRuScdh\\\\\\\/GAHyQUYNGjd7jxSW\\\\\\\/DqHrCeLRea3mnsFf8br9X\\\\\\\/ub0g8wyC14uQKPmj10qskx2VwNlaPIFzISPn1VyxpFnhGdkpthcE2C9kQ+x4APkot1VKiCwQLUF3gboG94n45a3VL77RJJx2kvIrFdfbTWJEC66wJ4a\\\\\\\/c1htZcU\\\\\\\/2LSElstPZoUUxgCIBIHQ8tfrlXBOFwm57AGI6xaG9WRgaEPMAQLEnk8d4fAnCVHOyivPRCqrnx4VrLHcs3JaJ88gL3Sjc19TQkgzWCGm0PLjcFPftJ9b6AVXZDiqo2xYJnLNO02eK4u7FoqGTc5jcwhSBRlbhUihKFtzwoumNKcK6PUCre2NfsacCYGw3jNDbtF\\\\\\\/SrszXNY2Me7mbg2dFbCw7Q10iP\\\\\\\/zHdyaYbCPdMfstARQYW8qbbh0i8SkJPVHlBL\\\\\\\/5\\\\\\\/d3\\\\\\\/V7Vx9Zic9YNJeYjgWPkHrFuvxfqRjSZY\\\\\\\/fTUU6yphvA91t5akDy3SqzWB+gdu4vw9EHwERhNcLJWMMbolH6mYhqv3CbYqJK14zpsbZUObxpF1hRhVZ8l57EjnTfT5QD0LoaC\\\\\\\/n1n7jyb9FujQ9d0w1ZXiG7GW6XR9NXPwzL6anC+EcSp6KjJBmZvUe2tj7eWfSntHf90VQHtfROvXiAiCIVaNFm9hos13PGLCmRq3KgeauGRygcFpXpwqCkFbM1y3vlhcEP1AjWGM4KYo9\\\\\\\/iCCrdKK4BE7i72\\\\\\\/iUnblI9S\\\\\\\/IOYnwJnFv2dLINpJnPQOxk7DLj7zJHw\\\\\\\/h6HVTzXyAzvuUyemSoI3ivhDEPbCrrWRAnH7IUrBG94gWAS+Od3BmPQb8AdpGRBH+INNyPWi5ODf0Pvtqyj7dO2KkR\\\\\\\/UdJrzLLDGSgy2EW0KosggbFh3M6vqGggpBrXucoOldch+SnJ31669nE3aViYzAszSkjO3NmOc6WV1FHQRTD6aEJUGfSAtq\\\\\\\/C9kH5lZzeIlFe\\\\\\\/+EOJxEiR89zFvg4Q\\\\\\\/my\\\\\\\/FxjVal3JRw67LLE5lAXSMY3KjqdI3mVZloPtoKgnOvDvcpQzWLchKJ2C46PMLDDN8nquaKVnOr6c88XJOmd9uBTH7Ve9FToEDYNnqp4uFTUEna4VATb\\\\\\\/5+Sf3MhJEwahseKoT\\\\\\\/vUwqReQVXIqZVxCyPY6U4PxRQwOjulwspw53ziFskESQ\\\\\\\/yc8of5+uIzu\\\\\\\/qiBJHRdHtnMpY7ab9InE\\\\\\\/V2lg8ge\\\\\\\/F2Tw0fKNHkMFW9PLh\\\\\\\/qK\\\\\\\/JOTqNwHeJinseEvvvv1kz5nNoP9JtVKsv\\\\\\\/u2XIlBNayGhhachcHnz6kZkXHOujlhi3\\\\\\\/FdvQMRat\\\\\\\/xySBtn+GssF\\\\\\\/ckPeBVc8qNy84463o045CXi4uO0UDCVcCCQaioIzwlwhU\\\\\\\/o3R24Z3SUQHXS95ccyUakSnXSLq9I0GSiMdSSa5ZBfp6Y6getivo0wP1CNfqoWM4uAO7yGPZdqbW6fzWOgVJ\\\\\\\/M8YIYffkIVE6FpRg2EafkTiHq2sbxFnE3G\\\\\\\/GlBXpubquGauMZdAxhmekFOVmyoCOIUFMc3Sk9tsenTVDOOAUi\\\\\\\/ebyLvLJ28OED4nqHHO5qRofpEWLIigt71VXflDYbGD5rnV94M6sfs7CKP55QDoH9dastjBQ+0XBkoILs82Jwz11fjQTjQFqCKbl7UnzTJVhRwE9JEt7MjfyxIuchSk9Px5tBhvKzJndGYaynrm5Ay4ZyOyG6Tkh6IWPjdm\\\\\\\/mXXuqB2JtEMSvBKKtA1Rx26tt\\\\\\\/ZsD3mlwvcGWeaBA4XD4e4UpyxAOqA3RwRsIm+UQ1wu3kIztPrMAl\\\\\\\/gv53jnXRay8uVLgEvEmB4LOte+DBF1BJxb3m2rDbBgnD1VJlo003gJ4AdljlIsm2mtKI\\\\\\\/QqWzCi0sZ3e+ONA6zCynoEjP41\\\\\\\/Vd8wsvwqEYMR27upWYRQdlrhAMax9siY1LIPB0ndRNO+shurqPkSFkSXSnoVg+w0AsF0LtJa+Nwls4f5Td6sBA1qIEIeSaYNNQvSeoUzxxaz1qHfoWLM6QL+KDCXYEhqKo5j96XuK2PtLPEUSFiFCDEuHAiwcHeNjChqLLx64DeYUiYgelYY9xTaKuSNpxvrxQiaCar3Vn2Vz3PFL+GjjyvdPhJVTqZe13yxg\\\\\\\/DDnyq8CUZOnZzvbgVAdfdavFSWdFmm0MaOXtHVzdSK5DYXUj+M5d6AhPFOsAzrxA62fiTS9eeyrgXNybY19kmcnvm1ZXOY9H1kfYjTrEsA839SctX\\\\\\\/MXF5swquPz95K3InAtRzhbdo7yOYxvgN3akyOuS2YPD0FZJzO6cb6SlTLuyQtZz9KHVn22qgckenPyQFbteBSJTeJHvFNo0zkmRS4kc1bHeaRQ+Ksy0L3xOzLUpppJFVeJCrXV1HaNVjiQD5yDr0wkNftOWSh\\\\\\\/q033WZwlMPQ7ucP3LyIaQJQlcjAFZiAWPnzYnC6tj4scJMLIIQNhbgFsNF0hn0EBuA06np6AFnNw\\\\\\\/RjvotfGE3G09xPNts9qq4RpqjSjLe9ciOnGNArDJnXhagu0KTWU6Wv6FU2DVMwK8L\\\\\\\/Wf\\\\\\\/a8ZzL1+irMPvV31xQkkmSp8UgEo5U+CALdPRkQq3SEaT7gEektziQFqy1Yvd5LbBxQ0N1adM3U9NM3S\\\\\\\/kqiA5jU48ELy2jQS8jc3t67MGxU+ZBysW7ZLTmxZ\\\\\\\/1ELdfUQBW1c73RncqEv9PS8k0cUPm2Ua6b4DQ3mIBsQ0v3r\\\\\\\/d\\\\\\\/iuFtCvYo622+\\\\\\\/NbYWsdahlG7oY2SbH+j5I862pNv1eScl+pBjkzODbaYZ8Nibg+6m9TDrZgXD9V968WWcIfSM6oWgopJkgHyC7pKCl4m9cYbXhC0y0Nw9CVywHCTv3TCA3vMy\\\\\\\/FJkzVx0F7iLDcYvHLQl5MGKQAYf72a+GtyJpifP61igE\\\\\\\/Txc\\\\\\\/useNdO8MvrsXi3RPOpXP3r+TuSDHCkzRSK0MhVrLKvuZ5APrN5JtgtSTta6lIvo3vBhrVSsW8plHo9lPwPNv5O2fanqwcaAp3NA\\\\\\\/6xujm\\\\\\\/xqM1UPo191oP+lpR442NBn4VmlxIRYnucudfv7BgDI2EDq1IYEPdE87K4v\\\\\\\/7fTabjNnHTFPZiLhh19vIY6U\\\\\\\/fWdoOCcKfSx8fbKwaOKZ0cE09qUX3NuGF\\\\\\\/PnPzPPwayJlI2GaUH2\\\\\\\/2ASfwle9szCqLwRPqrjJitUnIQ7aytPJJ1gGsbkqwMFigeLAvO0+juahRySa24F92+DSQGiQR5Cv7wh3AarhtVYneXd\\\\\\\/pUwZSPgPZ94HJHlQXnooJHzMEeV3J3KO3WEVBEu+Exvm2XmFFtqIqrMCPCS+k6LHpMWAUbsoVI3ec9o\\\\\\\/BqKaxzm1jY5YJcTm7J4VzpioMD+FQZcIlilg1YF7WKtrCRIVX\\\\\\\/U0KyAXtMjK5\\\\\\\/0EPPuH3+fRJMCtTg\\\\\\\/bjuysiLrkBUBB3sAAeN2tofNrZ7Aad1Dd\\\\\\\/IKqyj7eu7DBI1F2e7EBMWE40UVjSvnx+aFstSN6NefBPM4wnyErKnDNfO0Qfwvp02RXlhCbG+ThMF5EY+f+gj1RJA1W1H3IXm+aNEbfzcIIdYAFbDzNwMhO76aPI+nFS4KZxGUedpQXiEio6JpOAMCNhFddA23g+PB0EA3kGsqDssiYg665nRbI4GRDmbpo0jSk\\\\\\\/PjGIvFd1LncR11q8WwCvvbFnsMTUIINIvScvRAdw2KalBkpKmicPvlDQIxsB1EdUo4WqDg1xujWVOdJSTEScF8vtxZlaNZxV\\\\\\\/W4gV9MnV8lcYfC+pNWVfScCcZzrRGPs3NgeM9SVk30FlumvzZ97r6SvmoSTt5e1gxftNCRv5ZNaUvWFvBX0ai6+fFKnUocFfbNXCmvBjOyfNwYxpa4QPt9eRLCxaGmLHDyFcwfYnn4bEtSNh4joyfy+k7JLT3UE+4rLPSzmHH8aCR+T5XjwUTBPSrd5s6N+SwhgFvF5UGFGeWicmINFbBONSETimmr+eLCpSnxM9gvFCNsL\\\\\\\/LjgQeTtpI8vdME307JXVDpg6sWUM3tecFlE\\\\\\\/W3zPcYFU2VwXlbspcXFe3AixZOn2yG1WJgahuW9RNiGXJVleZnaVRJp8ZLdZCtBRqLrQo7pV\\\\\\\/E9EWFxDlpWRuy+zI9MkSpwq5Dgy8OND6B3le6USb4HxMeUUhK9xI44auelouvcBG7q93zUsiFCqE2bllCy7JdsKvqYx9mvUp6KJzIcJ4hnVoKf1d96jxrRB45bXMarcsOOOfI16Tx9667lstkD+Wc6UFacD1EHcYOeqBf+b1YSdR73qB8t0382Y0Cis1++7lUZ4O5U7ou8rgn5zmkBIklkyOhkv8kKAtNoRpwNsBI8khJeMEhwGgOU0U6\\\\\\\/+j2ROkV6Av01IEIQ4JhhXFLcsWnlauZPXxm1KLWVnEEOLQawUBTivTD0ZGJHClaXuwrRadieqiwVWeKhkhZLxpyDV5rKNZFwbw2gA4ZlbX3oe4X76G4awlKlUuLFYlWs2FrN3Oz8nqsmTd0ZalZ7p4r+oPBJS+Il+SkKG0qKkF3tEi37ueCfSDdUhwcSFeBqpKGWAGL5H3eQREUi8zu\\\\\\\/tF+gVim\\\\\\\/d3OXwwDJ+GsgSbGSLrMz5ARPDxpk9AtWOwvOaRyTW1fcjQLuoSNzuXONEpoDvAemE9S8pxPdPSfBLkncSDQcaO33kuVnWnQ6E58SzesWdYyktJ024mPWFqvGDLQgoNg8F2JMDtraiNGy00hE0N74UaLYU6OOIEh5ppJ38ugmRd3CcVm+9A9vSF\\\\\\\/3zDbWwWj95PIo0PP6yR6YAUqp+L6qe5AuuIINY4YsVRmcI+F2twUQetTOYB3x9repZ3zYNHCSkB1mryfQGueev4ChhDi22pO0GLZ\\\\\\\/ecHWbhMN2hL+Q4h2oPnuB3bWhhioyRTlJLu6lPmV\\\\\\\/E+TKB8QXnKTLt97wkdnPLkBM8iprD1uoU0ZXbjpP3TpW4CJa8SKM4cGfh8keh73M1UEIkb38iwzmxlSA1Xhgd8MqY7wnZYbyxF4HCkxG8zBOzV2hKU+aDKmEsvRyzR6HlxjvHSwpO8AV0vJu4ykDp5hZdEpSNAz4BeHP0GVavNpUnxEswy97JXOgu1yyq3NRwU5\\\\\\\/cOpSylauB9xoXVcjHGNzq3FN3sVNLFTVqqTKzn4egG6o23XqhkGB277YdIgGOGY\\\\\\\/Bhw8YEUfRVX8Y4cSmc7ZJLAftJiDhAPyriTXF4jCRW9uxGh+h5Ee9xVSIAHsM+atIxbW4CUkGXhLpaw6pQH3K7kIV3W2nKW0nYopTq\\\\\\\/FNSkm5psH5wPelekAUrumt6UUhjXycCcxCpZ5GHP1t3Tco2Jq1vR3GdIJsP62+qiL82rXiWClA80Qr7DARsm5cQri+xobHksqgPZCt0RRtVPmrorys01lkryUU\\\\\\\/drn3HcnPNXBlWjFfFte0197+QllQyAloym3Lrx8fJx3P7aWo3OJjZ9sJZuK+Ce3Ax\\\\\\\/SBRxPCzd8MnEnQ2sJQRrlrO6F8Dm+xi+2aR7xstL6pkFsQjQi+3A9VUo808V7Q4NY8RFOC2JLliCp+QXCZNUYomEtF6a1XdeyBlndAAqeRnfLLsGr73Boh\\\\\\\/1zmWlL+Fa6TIaCB7b58YJjUxk6+5wEdOZpDsgpTSV6nSJbqhdXM5QOV9rbV5jPnTjTEHWBjGnrYdKMUys7piAlAic0\\\\\\\/xDBUUWUhLkDgSC7yIetP8XJ\\\\\\\/LLg2foisNV38GWwINn6OraFATuKY5Ti6+RjAX+aMv7HqkK0WtWw7wxli0mkcxye9jpA2HVF6PVHVAQuRwLBnx2w0OH\\\\\\\/R9XcVOibF3OBfCEKBdaUL5p6tDhFfCJhWSzleLMzPJuVRAzHFUnP9soU6JS0XHyXkAQ3ICXtcFaQ2qe5S0M+rt9s98Vc7uHkX3IdewwJWEo7x4mzg6mQq6e+6rQ+iwEmUsQGFEjAz6NePkB0BDqyruRiL6qcOgE4pVKwZUO6CH6ptpnC8i\\\\\\\/x0Z2W6K\\\\\\\/NSOe9WByNj9q6GytR3Ic+81XZxShJPiQN7A2KJ\\\\\\\/ibZE9AJwto+Q5jXW2BGYKd2tuXo3YTfOlAMOSfidOL+ISl1aSmCH4ziVXNiNFJbv\\\\\\\/PnAJvkMmTzuPSkjbZQMCtbeeaTmcoDATgDt+KelDy8pzyuqWJCS7iql8irUpvKtlsCa8\\\\\\\/AElO5dO\\\\\\\/kBNqcpDjO4uGDpabcSrJjDVksKWdcd++gJY1sBuSNa+C4WdSopSpzl56Qhz\\\\\\\/094QYAZocJ835NaFb+hD5jeiAYdq0UiLrWR07ljmgrbHuNjglkG33S41zPvGzn3WlA\\\\\\\/cJPyy019g0MaDg0iQcz+HYLBtjRl94owFdLPN7xovUGQB400snGm5mDDHBNLERNxUo7a+GZcF4QxPqZvANrM3WWSvPAejs00P9SSIS2QoV1Xh04Qo2mH4nqw7li5ulvEtUm6vOk1fXXNDm+lClfJTRQwaQx+vOz3xnC+j6weQyQK185RTrxeoCVyCqdRw8iYpCKYhobfMUdzDghuRa9aj845jsoRSfo0r2MZHcIaOMKi1\\\\\\\/tR5u3mbI4IcMMtVAmvj\\\\\\\/FKm+UPDbacTflzDIoAZbyyltOEcd71N4YWCkp1oxSjv\\\\\\\/cPqxeaX9rEsN1xi\\\\\\\/0O0oVB2bp+009xkiUJSb5qlnhC1ftjosLBumW7FwI6Qmr92QhHv6JlScnEYSmHeBkmbL64gAByZis8NSFRYDhPbwE6JzhnWDWL5f10grY9mAidBy405\\\\\\\/8D0Od8JH4G96Xbdw3fBnnFVv0N\\\\\\\/IA2MBwjlb7x7cKBJnz3zvW2r0ls\\\\\\\/Ev82ONRa3Eylk73p41h0uhsGNpF9bxtWtoXn8us++gxo0EL4B9BmMC\\\\\\\/kUoqGrbh2Of8gFz4esHI5MWDEAFzXyGFYxfEwzsZr48EETkeurQVn9pmfLuBQeWIuPRSiXNYds62l5WDTe4gGqHpmrLRPN0H36WIEyVbi1CCI9F6diH5Cju8fXBAu\\\\\\\/c2FS\\\\\\\/np6WyEaN3VjztBzHXWokYhnfi35FpfSi7d6tJ2Bx4Pp2o8yULinfQs97sPa9pix08XZk1SJoo14ATqPZRcCv9PlP5D8vQkbBw6fpUTtaqsY+\\\\\\\/Hvk1satnUJBf1O0mGK7ZOjR2KCTaFPpAZD1MmAdR0DorhjRe3Xom1HrCNEqBgXIWc\\\\\\\/uzczZhzFxRsf0+O5TVUGtvGnXkr47eHgVWGgi\\\\\\\/1OdXtXpDJV1PiSkSadW351XIfjYhgK\\\\\\\/sh59HEG8Mt8fMysvR\\\\\\\/p4kGK6et10wEzL2qD+BUdbPBvmYTN829lF2SEASeljIh9owLOHnXmHiDTYF1MEfGZ3x3AW\\\\\\\/mxXqzv1Mc+P0dNOYyEed0\\\\\\\/VXK7OChuKx1MjiEOqaLObVMqSqB1Xjg2gQ5Z+WDSdVl4iKXCmG79THBGxnDt2z2YCZSUJR33ZyeojR\\\\\\\/IQRW\\\\\\\/rwCuGZgUQIYfvpk5IFe7AIuOVoA\\\\\\\/d1LEfst0pP7x6OVd8uEdlh8B0orQDfzxI+k2IfngYILWV5kgNFabRTiqAYfCW7R2rJxmW\\\\\\\/5KE5tYB5b4rVFM8\\\\\\\/aoar36zPP0op8F+vNKb0HNXAIDYIXMpR2ND9dCMivnU1Tv3ovylvbolTsi\\\\\\\/f+aNDtZ9B88NLqwqhNdsvggsKLUoZysaoa4yX0zwwGrjAKUMDMsgfNGv1Rh\\\\\\\/QZzb54S\\\\\\\/cLwsyQFbq7lIQSNNASioDIlXgITnnc3FdK0+9JeKzMZhEJ++ruxqreA4+6egVwZ3A1lbvkIDcrY3zTgc2LWemXKnKUdAsIFbxnafu33KgSzXts9CfHFB9hlSNP4HAxn9KdpjnvoGit12ds8PCEAxPoYRZ3vtmftonE8IpzeoZDEIud62HLRdvDE98J+bsj\\\\\\\/2sExjIZ2wzZnlehoFL5odjtlO1V2\\\\\\\/UsPIXEA1zdlWtN\\\\\\\/yXrrNrY9W2X3SBxLtOQegHXbO8pan1rMOnNjRrxOPpUMRDzOGLctJWb5lWpqvGhfmW9vzHFbzEQcAd7mo3VZ5dosn58raDXxRLFTv6G50Bigj5UpHIrzTE7kr1\\\\\\\/eDjNm0tpr8pWV7zopquuKSr\\\\\\\/3pcg6ftKJlehj62BSNCVsDq0ZclSvzeOrrZssgmzFzPv1kVfvtBoeJKB0gArgClnj+isQl4twVM0N4bCa3lLjLOetpGrWXJr70GsYTMdFISwT9xC2AokvH7S1nOyxje2\\\\\\\/x8WdvRiz+VhYK6h\\\\\\\/nEALgX2\\\\\\\/MM2DcvBbe8gNnp7fTFI0WQLe4vwLwsH8tfhU04d\\\\\\\/sWNEWJ9lOpbaszY62JJRQU1f1J1qcJCCMrheKhiwnBeRoc+JPniEnwdF5t6vcVAhHkRr\\\\\\\/bVHWjsNZtVet0YcaJfzAHPGJDHqQXEOPWfw1h5VKK\\\\\\\/fMnsAwkmwIE27DYPfMtlrWygIZzJTzsd+QqD4NKlVaq8EkWc23wD9CSrSnVr+WJPAA2po6DL0tMZl1PgGCOl\\\\\\\/jyRy6pgEK09X5GuqAA5rptAuBkQjtEo+GiEHubkPRtCHm64+haasvWKEUX45H6aDfLhc9KW+hQTjAEM9onvr2iKpZXHpTzKNreI3\\\\\\\/oxUvUHMEtPt9aEqyq2faKMZ542tGb70WUbk2\\\\\\\/iNAiiP3IFDwLBSOYYkT+SFKajuLy87rK2hCvGa9+qJJ01C6YXrQNpqCz\\\\\\\/VgeHo\\\\\\\/jMn7hEV3LGIN1kgAT77UrTOubtJ1qbJiCxWq63CU6CTiuTI7WttFq7lAdZeoCjHpYyz7bQyZ6UMEccMUgsov3+d8MAOvDnkj0Miq2A\\\\\\\/JXGRGr0US8VbOW5KCc79tmhp5HBIGj8uPxhbfUzny9RehaXbsZ3klbxb9AcA1X9u1mYVHNqUDdQu\\\\\\\/B8iwmAi0VmmwAedd1CSCb3PmoibEEYuRFMVHrVHjbJ9mWrqPyvXONGznJnk0ZxLgKbf1i\\\\\\\/cQIMAN1zGc8s3ucCQtAQ06Z+G\\\\\\\/xWfVu+cevvBjGRgiOLJ1PmNhcKPX242GDu3L6HEGXYoujt07E6wxuezGl\\\\\\\/8whG8dD3yqIZdwXpK\\\\\\\/iVA2w1hWy5yrdPQxNaluoopQ9AT0KoXzMnsgWtNcqQqheOutsydOtuzN9IVJ7drOqbL8YA4c47E5Toj0JK72oxYVbN7DCLDmUz+Ybi4uasnhRfohlXK3aWF3WVTvFlK1eHa\\\\\\\/YbNFZdJLSM8AtIZoATG3h+F9PwRa4ovikt4Lw16NJJ2Gqx1Aqljr+haQCmkdKgXkZYO3jk4OUf\\\\\\\/IOZPAAc7io8+6LrCGWizQqY076NmuCbdnqWJJbetk\\\\\\\/L50bbOk5Ufc75viXeykZ6tgWKR7OGhsMf69mnnutkTNb+zGwoGsTSMwf0dxj5zOLdYAzxsL1ubO0yFyYflbicK1GhKzlh0DnUFnQAdrWxsIiIqZd4zyg2F7XWbrFyEMvB1vXu71skmtGblRHooots9qH\\\\\\\/JhstGTZMq3bHPCL0MtgTcEUKehas62FMBm0ffK7zdrU1lct08qrb1XKSmySuc7CbodDnOUqBMs1ahpvHFxwuW\\\\\\\/\\\\\\\/yC3ic65sD33k68Or\\\\\\\/y+oP8zzdW+VmbNobcORbxhR\\\\\\\/7O1h7wy7yZ9XEGfAELNIV9ybP6qM\\\\\\\/aAfGAvuW8UdCZW12PuW\\\\\\\/oWVDDdtyVprEev0KBOvf0QjdWDo9E9cNToJXgqYyufItAJQtGzg4kuvdDIVvr\\\\\\\/286foJEdOkbJks2pmMrN\\\\\\\/qodwdIP1Bk8qGMGAucfPQhz+PIIhZCJDxl\\\\\\\/rB+uavtK\\\\\\\/6CY+G0tpaXg34kfYJZKJl1erS7DN7emEfp\\\\\\\/g+5sBUxiPY9wipJnopkIv\\\\\\\/IhCdgyk7VXwPrCD5yjcniRV2xeFdp7\\\\\\\/8IQ\\\\\\\/XK6w79JNra36nZW0JdR44nlifIXp5nl9RoblXJD5lf1TtrbIQZ7Euuxa8WdPvM3CTAUyNHaI6lpwBRDaeW52uaSix5rU4L2y8H9EDOFiyaq0K5g\\\\\\\/jMzWCMjwcwB5v\\\\\\\/xFFofF4AY2c5IT2AQF+wYK6fWwkYW0+ZP11Q\\\\\\\/WBtVShTdxUvZCFzOz4t\\\\\\\/spT7iaCldATkUehkk6rvsevgJxTMot+CC6m0GBl+W\\\\\\\/lzH+9+NLuUWeH6CF6gROQW\\\\\\\/o5f5hT5yx5oEL5\\\\\\\/QJdccTxqGmkZ1ZSXNkJpVY42luXisiCeYWoINFREiwIdlc0bcyQfiXZV7V8rVdbL23Te1\\\\\\\/641zgKI\\\\\\\/F09TLVHXX0\\\\\\\/WK3ecfFE\\\\\\\/c35x1sJUMXyqelLf4jmOzvkeFIH0iFAZk4yIUCgNAgXlf7+uZWSH4hY7Zuty0PuSuzdeoJsGRJA8s01eM1KERK4Gn8zKuHujXUwkspP9j1zdCH59KxfgtR9eqZ7BZ8mW0wVOzA72SUXbIswrBcgT1lHO81cYPeh2Nc3befGeC2wKixm2\\\\\\\/56VmvaeACXhvx+TAOWdxqUPIsdQ7ywGnScpq20\\\\\\\/LkcKbDuNwleK+Z93hNzSgAbVn4xSg+6zZjAIG303Y7qC2VE5z\\\\\\\/C78wXno2RgwfodtP97wZFAQ9COx9hJjL7YRwJmPsU4BefNqzGhAiOizwCtIday1olBNDtmDfaX9To6HS56RzfFye3LQXgsxzRvfpFrcYB\\\\\\\/K08pesX0eWLXcgMqFFFRZnCc2YRFcFLJoFJpE9EZ1f3+2U2of2XIeqXDbKFg40pKY65lIQ9tSth9T58eZzIKohTtKMkqOiqG9JomTLvyRysGAsnH\\\\\\\/ULoA734GDHOC2IwpkbkSrlS2sITF+EgYE+RioHkHvYoRezwAd+Ap7Q1MBwVO1d7O3jV2bHvT5lewm8ZoXve0teytE78JejaWf9UE4dP6HvJN0RxGcMrOaYZt5NMjA4vgWnFgtoqQtrGa92e8PDJKPuCRVGpAlwdFX7jRooSnBBumTz3g++tvUtR9MifBThxGL7OjBT+VQ3H7ZNDrEfuJQdKBrbQ2elYv+3cyhTOrHxh6Se9FV4GcEMWtKQYHnBsbdRfJhLwadcMUXO4sJdKm1nlnKJ6DG2hoNTr3NgA\\\\\\\/ZX9kHFHVeX5lgbH9D7mhLrkPsyY+GgEurk0F1plr4un8uMq31ocutWUQX\\\\\\\/m1gVx88WPyDJM6JHyu2aZLh6kvIPmLP9qzpyvZtOrwP6dkYr2dR0aLQpAeZsi8NHpDJ2bzTz4zXdCiwfar9+O7pc2WbpG5C+DPyldhTgpNmo0BHVBNnKe1gQa0BgHFVxlovYFEJJhiAEnL+9LqTWDswbLhyYSoue+Kggte0laCgwC4Pxx3U06Gctd7qk8KG+qQGn5YnAvK0q0DsDDoTCDQE8x9x2bjM1eNHJjcQyYdTo01zAX3uvzY1LvxnqKIyRh8okySNCtGlLROVAABEM9Shkh8eZ3ij07syhEOnSO9oYIpZ7cjDHVwrnZuCs6YgMLbZ7DugaEjun6btZ5iOGxkXkhxPPFHrewZ588rZa9Gn1j5ub9zkng42ExC8dm2fc7xm26In4a\\\\\\\/HOTejWyLZFlBMPmdZsi3BbpCNVKelfHuSRqmkXIJKpPO0AFqg9r+tRJZyity\\\\\\\/uSxpl2eXdQewaFMEFIhcjK8ViZeLF1pgHkPOR0RObotCcUGJOBkpZRSew7Uo2LQfd11\\\\\\\/pcWlzuCFfRp7DFsBkuHXS175rY4i\\\\\\\/w+xX50rbzoLzbGjIuUB8bo9\\\\\\\/Xl5FoybjMUARLYWNWWv48sk09x00X1mdLHedA5NPK3wMyMuhB6EYGx+Qz+3vMd9yM0T+X2fSLC3WVJOVk62ueIfmeENBLvPp4N8s9CjErVcimDB16mtS1osHmNJlIEFqPpt2dF6\\\\\\\/WTnFmOrAvIhv9bT\\\\\\\/m1KuHCsXgep8ZKIPu2xzHsFrzasN+8EAQm+lOzAwFwO8vo3WH18DKgHaqwWFI+rYzM+PolQQYm4SqBZ2fNZUnueBMboSGOKZAcSFrrptkBEmmQOWG2sG8Pkcq3msMS95+gyIaGMTXBPORYBPgVPCca6gZKV1Uk7AnU\\\\\\\/GZWr\\\\\\\/BrCz3WI3amc2K9sqfm9vK4f+JXTpTAJ6nkFf8uoU1V4x43ef905xD4gcOviCVil2iZEt59JZoPQyA2Ie2Jb7zCtivcgDrT8bRa25TGQZXRiNSuOgN7x7VA5qqoZF8CbvjE8fF8SAAVYu+Ugv19SDBsNOvWHL+ZZje5DoyEVHH8WUa8pCR843jKY0Hb3sJBfWB8wC0Z4QlT8toQmem7mmFp5gKwdDOdGeP1SMRBLVIJC1\\\\\\\/4L+JCKairPIFKSch8Z+48\\\\\\\/Pjyu0WUXPzyuR+h30Qayf4bWccDqOkC71RWEcV0QYRW9sQNWAcranfvBO5s1aYF6galCtTmfGlYPfgz1SEMI5PwimgSfA3mc9kILzQ4pobl5OjQ9Amw3rhcWfVcs42egGbc7JpIff43XXzBN5T32uE\\\u0022,\\\u0022iv\\\u0022:\\\u00224KAnYJI8RK0qHvRKw9XmnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bee7b01dc6772cb3fd0ac684d5d5f3d0abd162aa7c86e66cbef29af4a127e4de\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022brVIshl+49TC5YiAOTEppktPR5A6G0v0CzN2m37PMHcASwfog8IBkO74KwImSQFNh8ej6qdhTrgo3NpO2EAeLZKhQHoXdC2llgHYqERI\\\\\\\/XSbuxKl6ytu+jsSpCAYZcoZ88LFd20dAWKG8gPCn3RPlQkYGDYGWT0jC31Xsy8+HQR8fRx2W8xBtxqd8oETlnvlONIosBuL5LIM6iBSEpvuqZAerKeBunnJVyMwoGkeQuKljukGtWy2b3L+bMRC\\\\\\\/9Iw73gcQzNjazN7ARtoiVNNo7UhIu3Sc0bNNV3aJEcfOlh9kS7z+BoT3t1EVeJi0MzH\\\\\\\/cPQ7SO5h49zD8oTo6oVYZyP\\\\\\\/dUSeEc2y0RhcHNLhF0\\\\\\\/Xor\\\\\\\/VIav9++3KkUOboCZI87ss5WMc1cUld1V7Rezbqz55FyULmOUMyPFdkXvehKSIiMoCPxUF8khV3EoJBmVqrCIZjgLoqVNaJ6tAJoD+Vc5hzFHODXn0us9IvuYYNnhf50UtYhvrlkv4Lt5r7Nh1PXxd95GlcZGZj4MvSUHqJ74N8ZmcDpm2q53fPN\\\\\\\/Nct11lwynUpEsNXjrPtPvBGKDGVaCy0Mh09IRp02cfnu6MZ0opHTykpsvCGtZaD5tMXxwwTu790LX126S58LSUKg3P1oNL3esIcx1v2pSIlXt35p2Nz3ubSehxX22XtvovW0kgYEfIFQj43Iy9OLAf4p7C\\\\\\\/3LxaCQYStVPBmXe3MVnaeqgZgH0MKMNAa8j3yx7FH0BWmXbi12TUTTXs4jgkG0Bm\\\\\\\/9Oolw9YIJz77xHTlSLltSZOvcONXytmsd+jNcRqBU2VRNnoGcH9\\\\\\\/7f4gjxsDWDeYmnYf96X9HRYjwebCpVhHhYReWTzxNlICBrIm4mDRlgb4mnG+zBcaJaa5\\\\\\\/dmdsxwJYR+Z8u4Ea9iKkII7CqNz9avqcr0YyRLK7yNN+VRqD6I7vl\\\\\\\/mce3xPOVM72Xo\\\\\\\/qnmIVg3fudTLvSHiCOWwo8+DoyIISCwExXrThg2TNsuc5I+zF1X7Gda0XzyEFMsjzjhAetWaniUHmIJsLm5luQ3YWXRXi9McFW9itsFqHhOe1Znho5rZsIUlzL+ixoXYEKtzpzOl3g7Znm0G6SYEcmN6aj3EMODSYXbkexdFy8JshGteBZKaDduVEK8I0SGYyI2bd2zOrwI8AlKAzUia6fZNp6HFs0r4IggeqnUyV\\\\\\\/frLx3TugffgVhCZZD\\\\\\\/Lfr2sxez9RYQsEhkyFvGGcJwtKOnnBoW5enzTrwjsMXeRkl9tgeQV2NBmZ528hLpnONWwgpRNEck9q8LWh2jadIiqIvqdWrCI9grXoUOVSyKw2qK8kBwIJlnlfk4kEmQtyaqzwfsJVKFRx3oFPR58PZtBSd7cFQwtrXRiqrkSm8MBgwTlrC9tR6E7Kb\\\\\\\/INLxEqb3TnXxfHTpDdsFuXZ+WvfmqKk\\\\\\\/AjqOOEoFarLKL9N1hFJGylQqFYh+kn2b0gJP89JIxv6jQIc8zGwQYach5UNfoaJfuRHS\\\\\\\/EshF4aoI3T9ZRQJHctTqpq70L437BK0RPMS752XPGwftmFamVxwZpn3BhhKaY3oBoyvPxdwIELSxwnu69gdZ58uSn0rcKOAqA\\\\\\\/+V6BVoUg154HpdGtqdnkNRfqTdxnhxcmENkbcZ76Fm5pGXub2hbEeIziqRBqQS1xArz2UnHK\\\\\\\/5cA8hGafAn\\\\\\\/FGqUrcidBQ0XFr1UaX53OTyQ9kDa4IE4l3RJowYB48OckCSrygVyN8GFSuzwPlnxONWvrijBBLYaq8jgDXMr\\\\\\\/PEZSbLSTEJRAQ1ckWwz15X9Yg5jISbXR38wrdqbyaALGTaa\\\\\\\/ULAMVuSvjFN2hWagasDnVsnJvecdJQhRlfB31gyEf3Ol4QOrRrZoWrv72iH1mQvdztYEPl3HkCmhT8hfkBJQ9X2qg2D2BQbgkzRupDroXi9cA6b\\\\\\\/LYkd3kuQJTWd1wRXQ1gDyfx4YW8xENZu0WkutKlXgmEYDJwhJ7vJ5c3nnYaGYiw0ctaRHMHjcKPP3qnc7bxAP7TW\\\\\\\/QQXL2wBEY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZEbG+ZZGV8RUCehIEtwC2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002227f28d7bd2a7adff44fde89ddfd638753d1c7895657727ee7a0694ab6ffdc34a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459201 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/427" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SBo9xjLINq9NxbHt7qwkFS82wSEOajN8HDJy1Wzl6VI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-io0iHfN_US5TLo2lway2g6LiE2thIRxTRlJd8jfo7SA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/417" title="Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase Escherichia coli Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/6/437" title="QTL Mapping and Candidate Gene Analysis of Telomere Length Control Factors in Maize (Zea mays L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459201" id="node459201" data-pisa="ggg;1/6/427" data-pisa-master="ggg;g3.111.000596" data-apath="/ggg/1/6/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Partitioning Transcript Variation in <em>Drosophila</em>: Abundance, Isoforms, and Alleles</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yajie Yang</span>, <span class="highwire-citation-author" data-delta="1">Rita M. Graze</span>, <span class="highwire-citation-author" data-delta="2">Brandon M. Walts</span>, <span class="highwire-citation-author" data-delta="3">Cecilia M. Lopez</span>, <span class="highwire-citation-author" data-delta="4">Henry V. Baker</span>, <span class="highwire-citation-author" data-delta="5">Marta L. Wayne</span>, <span class="highwire-citation-author" data-delta="6">Sergey V. Nuzhdin</span> and <span class="highwire-citation-author" data-delta="7">Lauren M. McIntyre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459201" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yajie Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yajie%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYajie%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rita M. Graze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BM.%2BGraze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graze%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BM.%2BGraze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Brandon M. Walts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandon%2BM.%2BWalts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walts%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandon%2BM.%2BWalts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cecilia M. Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecilia%2BM.%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecilia%2BM.%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Henry V. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henry%2BV.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20HV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenry%2BV.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Marta L. Wayne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology, University of Florida, Gainesville, FL, 32611-8525</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marta%2BL.%2BWayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wayne%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarta%2BL.%2BWayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sergey V. Nuzhdin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Molecular and Computational Biology, University of Southern California, Los Angeles, CA 90089-2910</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergey%2BV.%2BNuzhdin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nuzhdin%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASergey%2BV.%2BNuzhdin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Statistics, University of Florida, Gainesville, FL 32611-8545</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/6/427" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459201" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459201/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/427.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459201" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459201/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/427.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459201" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459201/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/6/427.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459201" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459201/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/6/427.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459201" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459201/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Multilevel analysis of transcription is facilitated by a new array design that includes modules for assessment of differential expression, isoform usage, and allelic imbalance in <em>Drosophila</em>. The ∼2.5 million feature chip incorporates a large number of controls, and it contains 18,769 3′ expression probe sets and 61,919 exon probe sets with probe sequences from <em>Drosophila melanogaster</em> and 60,118 SNP probe sets focused on <em>Drosophila simulans</em>. An experiment in <em>D. simulans</em> identified genes differentially expressed between males and females (34% in the 3′ expression module; 32% in the exon module). These proportions are consistent with previous reports, and there was good agreement (κ = 0.63) between the modules. Alternative isoform usage between the sexes was identified for 164 genes. The SNP module was verified with resequencing data. Concordance between resequencing and the chip design was greater than 99%. The design also proved apt in separating alleles based upon hybridization intensity. Concordance between the highest hybridization signals and the expected alleles in the genotype was greater than 96%. Intriguingly, allelic imbalance was detected for 37% of 6579 probe sets examined that contained heterozygous SNP loci. The large number of probes and multiple probe sets per gene in the 3′ expression and exon modules allows the array to be used in <em>D. melanogaster</em> and in closely related species. The SNP module can be used for allele specific expression and genotyping of <em>D. simulans</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/snp-chip" class="hw-term hw-article-keyword hw-article-keyword-snp-chip" rel="nofollow">SNP chip</a></li></ul><p id="p-5">Gene expression analysis has proceeded from a primary focus on overall transcript level (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schena <em>et al.</em> 1995</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ross <em>et al.</em> 2000</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rifkin <em>et al.</em> 2003</a>) to more sophisticated analyses, including those that examine expression of different isoforms (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Johnson <em>et al.</em> 2003</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kwan <em>et al.</em> 2008</a>) or individual alleles (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lo <em>et al.</em> 2003</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zhang <em>et al.</em> 2009</a>). Commercial platforms exist for measuring 3′ expression or exon expression; however, there is not a single cost-effective platform for measuring expression at multiple levels. This article presents an array with three modules: 3′ expression, exon, and SNP probes for <em>Drosophila</em>.</p><p id="p-6">In diploid organisms, expression from two, potentially different, copies of each gene contribute to transcript level and subsequent protein production. Unequal expression of these alleles is termed allelic imbalance (AI). AI is observed in model organisms and humans (<em>e.g.</em>, <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lo <em>et al.</em> 2003</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Guo <em>et al.</em> 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Graze <em>et al.</em> 2009</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhang and Borevitz 2009</a>). AI is a factor in predisposition to complex diseases (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Meyer <em>et al.</em> 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de la Chapelle 2009</a>) and contributes to phenotypic variation in human populations (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Johnson <em>et al.</em> 2005</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pickrell <em>et al.</em> 2010</a>). For example, AI is associated with the risk of developing breast cancer (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Meyer <em>et al.</em> 2008</a>) and colorectal cancer (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">de la Chapelle 2009</a>).</p><p id="p-7">AI has a genetic (as well as epigenetic) basis (<em>e.g.</em>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pastinen <em>et al.</em> 2004</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Serre <em>et al.</em> 2008</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wang <em>et al.</em> 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Verlaan <em>et al.</em> 2009</a>). Exciting new developments in the study of complex diseases revealed regulatory polymorphisms contributing to the evolution of gene regulation (<em>e.g.</em>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Emerson <em>et al.</em> 2010</a>). Whole-genome associations of gene expression and phenotype identify the genetic basis of disease and other important phenotypic variation (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stranger <em>et al.</em> 2007</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nica and Dermitzakis 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nica <em>et al.</em> 2010</a>). AI identifies causal <em>cis</em> regulatory variants (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wittkopp <em>et al.</em> 2004</a>). Allele-specific association studies advance these analyses and increase scientific knowledge of the regulatory process (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rockman and Kruglyak 2006</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Serre <em>et al.</em> 2008</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stamatoyannopoulos 2004</a>). Analysis of AI is an important next step in identifying the genetic basis of expression differences.</p><p id="p-8">AI has been assayed with pyrosequencing (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ahmadian <em>et al.</em> 2000</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Wittkopp <em>et al.</em> 2004</a>), targeted SNP typing arrays (<em>e.g.</em>, <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Serre <em>et al.</em> 2008</a>), high-density array designs (<em>e.g.</em>, <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Zhang and Borevitz 2009</a>), RNA-Seq–based methods (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Zhang <em>et al.</em> 2009</a> ; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McManus <em>et al.</em> 2010</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Pickrell <em>et al.</em> 2010</a>), and smaller-scale methods, such as allele-specific qPCR (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Szabó and Mann 1995</a>).</p><p id="p-9">This article presents a custom array for measuring 3′ expression, exon expression (and thus alternative splicing), and AI. The array has been designed for <em>Drosophila</em> on an Affymetrix platform (UFL Custom Dros_snpa520726F Array Format: 49-7875; available for purchase from Affymetrix). The use of a single platform is cost effective, and statistical analysis is simplified by the single hybridization. We designed 60,118 <em>D. simulans</em> SNP probe sets from previously reported SNP variants (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Benson <em>et al.</em> 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Begun <em>et al.</em> 2007</a>). In total, these probe sets allow AI to be assessed for 11,929 genes [79% of 15,107 genes in FlyBase R5.11 (August 2008)], with the majority of genes represented by multiple SNP probe sets. The SNP module is complemented by two additional modules: one that measures 3′ expression and another that analyzes exon-level expression concurrently with allele specific expression (ASE). Experiments show an amount of sex bias (34% of 18,769 probe sets), alternative exon usage (164 genes), and AI (37% of 6579 probe sets within a species) consistent with previous reports on other platforms (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">McIntyre <em>et al.</em> 2006</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wayne <em>et al.</em> 2007</a>; Telonis-Scott <em>et al.</em> 2008; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fontanillas <em>et al.</em> 2010</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Chip design</h3><p id="p-10">The chip has 2,424,414 informative features, covering four types of probes: SNP probes (<em>n =</em> 1,442,832; 60,118 probe sets); 3′ expression probes (<em>n =</em> 262,766; 18,769 probe sets); exon probes (<em>n =</em> 699,865; 61,919 probe sets); and control probes (16,943 GC band controls; 2008 hybridization and labeling controls; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The 3′ expression probes consist of all perfect-match (PM) probes from the Affymetrix GeneChip <em>Drosophila</em> Genome 2.0 array (900531, 900532, and 900533). The exon probe sets provide measurements of expression from each individual exon, allowing controls for signal fluctuation caused by 5′ bias in expression assays, as well as measurement of alternative exon usage. The exon probes consist of all Affymetrix <em>Drosophila</em> Tiling 2.0 Array (901021) probes that map uniquely to exonic regions (FlyBase R5.11 August 2008) at the time of chip design. Overlapping exons with alternative start/end sites in the same genomic region were combined into a single exonic region. The majority of exonic regions contain a single exon. (For simplicity, exonic regions are referred to simply as exons throughout this article.) Each exon corresponds to a unique probe set. The 3′ expression probes and exon probes on this custom chip were designed by Affymetrix from <em>D. melanogaster</em> sequences. The probe sets have been used for other <em>Drosophila</em> species (<em>i.e.</em>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kopp <em>et al.</em> 2008</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Graze <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dworkin and Jones 2009</a>; Lu <em>et al.</em> 2010). Using these probe sets allows direct comparisons to existing literature and straightforward quality control. As each probe set has multiple probes, the impact of divergence is likely to be minimal on summary measures of expression. However, investigators comparing among species should consider filtering individual probes. The SNP module was designed for estimating AI. There were three main steps in this design: SNP identification, SNP quality assessment, and probe selection.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/427/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Probe design. A total of 2,424,414 probes were printed on the chip. They are of four types: SNP probes (n = 1,442,832; 60,118 probe sets), 3&#x2032; expression probes (n = 262,766; 18,769 probe sets), exon probes (n = 699,865; 61,919 probe sets), and control probes (not shown). The 3&#x2032; expression probes consist of all perfect-match (PM) probes from the Affymetrix GeneChip Drosophila Genome 2.0 array. An example of a 3&#x2032; expression probe set is shown in navy. The exon probes consist of all Affymetrix Drosophila Tiling 2.0 Array probes that map uniquely to exonic regions annotated in FlyBase R5.11 (August 2008). Exon probe sets within the example gene are shown in light blue. SNP probes are custom made. The SNP probes corresponding to a single SNP site base are shown in dark blue (matching the forward strand) and blue (matching the reverse strand)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901685878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Probe design. A total of 2,424,414 probes were printed on the chip. They are of four types: SNP probes (&lt;em&gt;n =&lt;/em&gt; 1,442,832; 60,118 probe sets), 3&#x2032; expression probes (&lt;em&gt;n =&lt;/em&gt; 262,766; 18,769 probe sets), exon probes (&lt;em&gt;n =&lt;/em&gt; 699,865; 61,919 probe sets), and control probes (not shown). The 3&#x2032; expression probes consist of all perfect-match (PM) probes from the Affymetrix GeneChip &lt;em&gt;Drosophila&lt;/em&gt; Genome 2.0 array. An example of a 3&#x2032; expression probe set is shown in navy. The exon probes consist of all Affymetrix &lt;em&gt;Drosophila&lt;/em&gt; Tiling 2.0 Array probes that map uniquely to exonic regions annotated in FlyBase R5.11 (August 2008). Exon probe sets within the example gene are shown in light blue. SNP probes are custom made. The SNP probes corresponding to a single SNP site base are shown in dark blue (matching the forward strand) and blue (matching the reverse strand).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/427/F1.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/427/F1.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/427/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/427/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461011" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-11" class="first-child">Probe design. A total of 2,424,414 probes were printed on the chip. They are of four types: SNP probes (<em>n =</em> 1,442,832; 60,118 probe sets), 3′ expression probes (<em>n =</em> 262,766; 18,769 probe sets), exon probes (<em>n =</em> 699,865; 61,919 probe sets), and control probes (not shown). The 3′ expression probes consist of all perfect-match (PM) probes from the Affymetrix GeneChip <em>Drosophila</em> Genome 2.0 array. An example of a 3′ expression probe set is shown in navy. The exon probes consist of all Affymetrix <em>Drosophila</em> Tiling 2.0 Array probes that map uniquely to exonic regions annotated in FlyBase R5.11 (August 2008). Exon probe sets within the example gene are shown in light blue. SNP probes are custom made. The SNP probes corresponding to a single SNP site base are shown in dark blue (matching the forward strand) and blue (matching the reverse strand).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-3" class="subsection"><h4>SNP identification:</h4><p id="p-12">Alignment sets were created from multiple sequence sources, including FlyBase R5.4 exons (<em>n</em> = 68,536), six <em>D. simulans</em> strain genomes for <em>Drosophila</em> Population Genomics Project (DPGP, <a href="http://www.dpgp.org">http://www.dpgp.org</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Begun <em>et al.</em> 2007</a>), and all <em>D. simulans</em> sequences (343,420) from GenBank (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Benson <em>et al.</em> 2005</a>) that were not annotated as “whole genome.” In DPGP, <em>D. simulans</em> genomes, except for the heterochromatic regions, were assembled against the FlyBase R4.2 <em>D. melanogaster</em> genome. Exons from FlyBase R5.4 were BLAST (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>) aligned to the DPGP genomes and GenBank sequences. There were 325 exons with only GenBank sequence, 62,161 with only DPGP sequence, 3163 with both GenBank and DPGP sequence, and 2887 for which no sequence was available. For each FlyBase R5.4 exon, its genome location in <em>D. melanogaster</em> R4.2 genome was determined by BLAST exons that matched more than one location; those located on chromosomes four or U were excluded (<em>n</em> = 1912). All matching sequences for each exon were aligned using ClustalW (Thompson <em>et al.</em> 1994) to create a multiple sequence alignment at the exon’s genome position. All SNPs, regardless of location in the exon, were identified from the multiple sequence alignment.</p></div><div id="sec-4" class="subsection"><h4>SNP quality assessment:</h4><p id="p-13">A design window, which consisted of −17 bases upstream and 17 bases downstream from each SNP (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) was the basis of SNP quality assessment. A SNP locus supported by fewer than five sequences was discarded. SNPs were also discarded when the design window mapped to multiple places in the genome or when more than one SNP occurred in the design window. These criteria identified 589,915 SNPs, of which 196,345 were biallelic. Only biallelic SNPs were considered further. There were 558 exons for which SNP data were identified from GenBank alone, 51,418 exons for which SNP data were identified from DPGP alone, and 2992 identified from both, resulting in a total of 54,968 exons with SNPs present; in other words, 81% coverage of the entire FlyBase R5.4 transcriptome (68,536 exons).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/427/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP probe design windows. For each SNP site, there are four sets of probes, one for each SNP site base. The SNP base is designed at three different positions of the probes: middle, shifted four bases upstream, or shifted four bases downstream. Each SNP probe set contains 24 probes, which can be classified based on alleles as PM1 (n = 6), PM2 (n = 6), or MM (n = 12), for a total of 24 probes per probe set. A SNP probe set has a 35-base design window, with sequences of &#x2212;17 bases upstream and 17 bases downstream from the SNP. If there were fewer than five sequences supporting a SNP, the SNP was discarded. If more than one SNP occurred in the design window, then the alignment was considered suspect, and the SNP was not included among those printed on the array. Only biallelic SNPs that were unique in their design window and supported by five or greater sequences in the multiple alignment were considered." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901685878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNP probe design windows. For each SNP site, there are four sets of probes, one for each SNP site base. The SNP base is designed at three different positions of the probes: middle, shifted four bases upstream, or shifted four bases downstream. Each SNP probe set contains 24 probes, which can be classified based on alleles as PM1 (&lt;em&gt;n&lt;/em&gt; = 6), PM2 (&lt;em&gt;n&lt;/em&gt; = 6), or MM (&lt;em&gt;n&lt;/em&gt; = 12), for a total of 24 probes per probe set. A SNP probe set has a 35-base design window, with sequences of &#x2212;17 bases upstream and 17 bases downstream from the SNP. If there were fewer than five sequences supporting a SNP, the SNP was discarded. If more than one SNP occurred in the design window, then the alignment was considered suspect, and the SNP was not included among those printed on the array. Only biallelic SNPs that were unique in their design window and supported by five or greater sequences in the multiple alignment were considered.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/427/F2.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/427/F2.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/427/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/427/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460963" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-14" class="first-child">SNP probe design windows. For each SNP site, there are four sets of probes, one for each SNP site base. The SNP base is designed at three different positions of the probes: middle, shifted four bases upstream, or shifted four bases downstream. Each SNP probe set contains 24 probes, which can be classified based on alleles as PM1 (<em>n</em> = 6), PM2 (<em>n</em> = 6), or MM (<em>n</em> = 12), for a total of 24 probes per probe set. A SNP probe set has a 35-base design window, with sequences of −17 bases upstream and 17 bases downstream from the SNP. If there were fewer than five sequences supporting a SNP, the SNP was discarded. If more than one SNP occurred in the design window, then the alignment was considered suspect, and the SNP was not included among those printed on the array. Only biallelic SNPs that were unique in their design window and supported by five or greater sequences in the multiple alignment were considered.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h4>Probe selection:</h4><p id="p-15">For each SNP, 24 probes were designed, with the SNP at the 0, +4, and −4 positions from the probe center, for the forward and reverse strands, and with each possible base (A, C, G, and T) at the SNP site. Probe hybridization quality was predicted by an Affymetrix internal scoring algorithm that takes into account Tm, secondary structure, and previous empirical observations. If a probe contained a homopolymer run or could not be synthesized or if one third or more probes had poor predicted hybridization, the probe set was eliminated. For genes with seven or fewer SNPs, all SNPs were selected. If a gene had more than seven SNPs, additional probe sets were selected at random (<em>n</em> = 610) to fill the chip.</p><p id="p-16">In sum, 60,118 custom SNP probe sets representing 11,929 genes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) were included on the chip. The mean number of probe sets per gene was 4.4. The majority (8013 genes) had more than 3 probe sets. The chip library files are available at <a href="http://bioinformatics.ufl.edu/McIntyre_Lab/ASE">http://bioinformatics.ufl.edu/McIntyre_Lab/ASE</a>. Probe sequences and chip annotation can be found at Gene Expression Omnibus (GEO) using accession ID GPL11273.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/427/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of SNP probe sets per gene. A total of 60,118 probe sets, representing 11,929 genes, were selected for the SNP module for the custom chip. The number of genes (Y axis) with a given number of corresponding SNP probe sets (X axis) is shown. Most genes are represented on the array by one to five SNP probe sets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901685878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of SNP probe sets per gene. A total of 60,118 probe sets, representing 11,929 genes, were selected for the SNP module for the custom chip. The number of genes (Y axis) with a given number of corresponding SNP probe sets (X axis) is shown. Most genes are represented on the array by one to five SNP probe sets.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/427/F3.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/427/F3.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/427/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/427/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461006" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-17" class="first-child">Distribution of SNP probe sets per gene. A total of 60,118 probe sets, representing 11,929 genes, were selected for the SNP module for the custom chip. The number of genes (Y axis) with a given number of corresponding SNP probe sets (X axis) is shown. Most genes are represented on the array by one to five SNP probe sets.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-6" class="subsection"><h3>Verifying the experiment fly materials</h3><div id="sec-7" class="subsection"><h4>Experimental design:</h4><p id="p-18">Two different isogenic strains of <em>D. simulans</em>, <em>st e</em> and C167.4, and their male and female progeny were used as the basis for the verification study. Three replicates of RNA from female and male progeny of the cross <em>st e</em> × C167.4 were assayed for six RNA samples. In addition, DNA was used as a control for estimating AI (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Wittkopp <em>et al.</em> 2004</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wittkopp <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Degner <em>et al.</em> 2009</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">McManus <em>et al.</em> 2010</a>). Three replicate gDNA samples were prepared for female <em>st e</em>, female C167.4, and the female F<sub>1</sub> progeny of the cross <em>st e</em> × C167.4, for nine gDNA samples.</p></div><div id="sec-8" class="subsection"><h4>Sample collection:</h4><p id="p-19">Flies were reared in incubators (25<strong>°</strong>, 12:12 hr light/dark cycle) on a standard dextrose medium. Isogenic strains of <em>D. simulans</em> (C167.4, BDSC 4736; <em>st e</em> isogenic, DSSC 14021-0251.041 inbred &gt;20 generations) were used. For each of three cross genotypes (C167.4, <em>st e</em>, and C167.4 × <em>st e</em>), 20 virgin females were crossed to 5 males. Female and male progeny were collected on consecutive days (under CO<sub>2</sub>) and aged from 5 to 7.5 days in single sex vials. Flies were then flash frozen in liquid nitrogen (without anesthesia) in a 2.5 hr window (4–6:30 <span class="sc">pm</span>). For RNA samples, two sets of 20 flies (subsamples) were collected for each replicate from multiple cross vials. No vials were used for more than one replicate.</p></div><div id="sec-9" class="subsection"><h4>Sample processing:</h4><p id="p-20">Flies were freeze dried at −20° overnight prior to homogenization. Dried flies were ground to a fine powder using a GenoGrinder (maximum, 3 min, repeated twice). Trizol (1 ml) was added to each homogenized sample and mixed thoroughly in the GenoGrinder (maximum, 3 min). Samples were transferred to a new tube, 1 μl linear acrylamide was added to each, and then samples were incubated at room temperature for 5 min. RNA was extracted using a standard Trizol extraction protocol: phase separation using 0.1 vol BCP, RNA precipitation with isopropanol, 70% ethanol wash, and resuspension in 80 μl DEPC H<sub>2</sub>O. Concentration was measured using a NanoDrop, and up to 30 μg RNA per sample was treated with DNase I in 100 μl reaction volumes for 30 min at 37° (reaction mix: 4 U Cloned DNase I TaKaRa 2220A, 80 U Promega Recombinant RNasin N2515, in 1× TaKaRa Cloned DNase I Buffer II). Samples were cleaned prior to concentration using the Qiagen RNeasy Mini Kit (Cat. #74104) following the manufacturer’s standard protocol with 30 μl DEPC H<sub>2</sub>O elutions (run through the column twice). RNA quality was examined using BioAnalyzer RNA 6000 Nano chips, and all samples were found to be of good quality. Genomic DNA was isolated from 35 to 40 flash frozen females using the AllPrep Mini Kit (Qiagen) following standard manufacturer’s protocols. Samples were concentrated by standard ethanol precipitation and resuspended in 31 μl DEPC H<sub>2</sub>O.</p></div><div id="sec-10" class="subsection"><h4>Fragmentation, labeling, and array hybridization:</h4><p id="p-21">Target materials were prepared for array hybridization using the recommended Affymetrix kits following the no amplification protocol of GeneChip© WT Double-Stranded Target Assay Manual (DNA samples started from Procedure D forward) for single Tiling Arrays. Briefly, 10 μg of total RNA was concentrated to 8 μl in DEPC H<sub>2</sub>O followed by first- and second-strand cDNA synthesis using the WT Double-Stranded DNA Synthesis Kit (P/N 900813). Per the GeneChip Sample Cleanup Manual (P/N 900371), 7.5 μg of dsDNA was fragmented. For each DNA sample, 7.5 μg of gDNA was fragmented to between 25 and 200 bp with 0.02 U/μg DNase I (Takara Cloned DNase I, 2 U/μl) in a 40 μl reaction with 4 μl reaction buffer (10× reaction buffer: 100 mM Tris-acetate, 100 mM magnesium acetate, 100 mM potassium acetate) and 0.8 μl BSA (10 mg/ml). Reactions were incubated 16 minutes at 37° and heat killed at 99° for 15 minutes. Fragment size was checked by agarose gel electrophoresis. Fragmented cDNA and gDNA targets were labeled using WT Double-Stranded DNA Terminal Labeling Kit (P/N 900812). The prepared target samples were hybridized using the Hybridization, Wash, and Stain Kit (P/N 900720) following the manufacturer’s protocol (FS450_0001) for the Fluidics Station 450 with protocol. Arrays were scanned using an Affymetrix 7G scanner. The GEO accession for the array data is GSE31750.</p></div></div><div id="sec-11" class="subsection"><h3>Signal quantification</h3><p id="p-22">Signals were extracted from the scans using the apt-cel-extract program of the Affymetrix Power Tools (version 1.10.2) suite. GC bin control probes provide an estimate of nonspecific hybridization (Affymetrix 2005) and help to assess the overall quality of the hybridization. A GC bin control is a standard Affymetrix control based upon the number of G/C bases (from 3 to 24) in the 25 mer probe. None of the GC bin control probes align to the <em>D. melanogaster</em> or <em>D. simulans</em> reference genomes. Individual probes were classified according to their GC content and matched to the corresponding GC bin controls. A probe was considered detected when signal strength was higher than the median intensity of the corresponding GC band controls. Detection above background (DABG) was calculated at the individual probe level. The overall intensity of the array was evaluated at the individual probe level. To correct for the background noise and to normalize the probe signals, each probe was classified into a GC bin and the 5 percentile signal for that GC bin was subtracted from the probe signal. Y<sub>i</sub>, the signal for probe set <em>i</em>, is estimated as: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/427/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/427/embed/mml-math-1.gif"/></noscript></span></span>. X<sub>ij</sub> is the intensity for probe <em>j</em> in probe set <em>i</em> and GC<sub>j</sub> is the average intensity for control probes in the corresponding GC bin. N<sub>i</sub> is the number of probes in probe set <em>i</em>. Chip verification was analyzed first for the overall hybridization quality, then for each module on the chip (3′ expression module, exon module, and SNP module).</p></div><div id="sec-12" class="subsection"><h3>General quality control</h3><p id="p-23">The distribution of the overall signal across all modules was compared using kernel density estimates for each slide separately (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Silverman 1986</a>), with the goal of identifying any slide with an unusual distribution. Similar marginal distributions of kernel density would be expected for one sample type. Principle component analysis (PCA) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Johnson and Wichern 1992</a>) was carried out to determine whether there was any pattern or grouping to the data.</p><p id="p-24">To verify the veracity of probe set estimates of 3′ expression, we compared the signal from probe sets for the well-known sex-biased genes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bownes 1994</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wolfner 1997</a>): Yp (Yp1, Yp2, Yp3) and Acp (Acp29AB, Acp32CD, Acp36DE, Acp53C14a, Acp53C14b, Acp53C14c, Acp62F, Acp76A). Consistency of estimation of gene expression across modules was also examined using Bland-Altman plots (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bland and Altman 1986</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bland and Altman 1988</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dudoit <em>et al.</em> 2002</a>; McIntyre <em>et al.</em> 2011), in which the exon module and SNP module were plotted against the 3′ expression module.</p><p id="p-25">The feature size of this array is smaller (5 micron) than is the Affymetrix GeneChip <em>Drosophila</em> Genome 2.0 array (11 micron). Although the PM probes are identical for these two chips, feature size may have an impact on differential expression (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dandy <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ammar <em>et al.</em> 2009</a>). This raises the concern of potential loss of sensitivity. To evaluate the performance of the 3′ expression and exon modules, we compared expression for RNA samples between the two sexes of the F<sub>1</sub> progeny. Sex-biased expression is well described for <em>Drosophila</em> (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bownes 1994</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jin <em>et al.</em> 2001</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Parisi <em>et al.</em> 2003</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ranz <em>et al.</em> 2003</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">McIntyre <em>et al.</em> 2006</a>; Telonis-Scott <em>et al.</em> 2008).</p><p id="p-26">The fixed effects model,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/427/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/427/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>was fit for each probe set in the 3′expression and exon modules, where Y<sub>ij</sub> is the signal for probe set <em>i</em>, sample <em>j</em> is as described above, μ is the overall mean, s<sub>i</sub> is the fixed effect of sex, and ε<sub>ij</sub> is the random error. The null hypothesis that male and female sexes had equal expression levels was tested using an F-test (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Neter <em>et al.</em> 1990</a>). All probes in a given probe set were used. As only one genotype is considered, any polymorphisms between the genotype used and the probe will be the same between the two sexes of the same genotype. Results were corrected for multiple testing using False Discovery Rate (FDR) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benjamini and Hochberg 1995</a>; Verhoeven <em>et al.</em> 2005). Where multiple probe sets matched the same genes, the agreement between the exon probe sets and the 3′ expression probe sets were examined for agreement in detecting sex bias using Kappa statistics (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fleiss 1981</a>) and McNemar’s test (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Johnson and Wichern 1992</a>).</p><p id="p-27">There is a sex bias in isoform usage in <em>Drosophila</em> (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kwan <em>et al.</em> 2008</a>; Telonis-Scott <em>et al.</em> 2008; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">McIntyre <em>et al.</em> 2006</a>). The use of alternative transcript isoforms between the two sexes can be detected from the measurements taken by the exon module. A probe set represents a constitutive exon for a gene (included in all annotated isoforms) or an alternative exon (included in a subset of known isoforms). Inferences can become ambiguous when probe set annotations correspond to exon regions located in overlapping regions of multiple gene models. Probe sets mapping to more than one gene or to the ambiguous regions of overlapping exons were excluded from analysis (<em>n</em> = 2611). The model<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/427/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/427/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>x</em> is the fixed effect of exon type and <em>s</em> is the fixed effect of sex was fit. Probe sets from multiple constitutive exons were grouped as one exon type, whereas probe sets representing alternative exons were each considered a different exon type. The variance was estimated separately for each sex. The significance of the interaction (a test for alternative isoform usage <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">McIntyre <em>et al.</em> 2006</a>) was tested using an F test, followed by FDR correction.</p><div id="sec-13" class="subsection"><h4>SNP calling and genotyping accuracy:</h4><p id="p-28">By design, there are DPGP/GenBank sequences for all 60,118 probe sets in the SNP set, from which biallelic SNPs were defined and used for the chip design. SNP alleles were verified using Illumina genome resequencing data for the C167.4 and <em>st e</em> strains (GEO accession SRP005952) of <em>D. simulans</em>. <em>D. simulans</em> C167.4 sequence data were obtained from male head RNA libraries sequenced on multiple lanes with Illumina paired end procedures and chemistry (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Celniker <em>et al.</em> 2009</a>; McIntyre <em>et al.</em> 2011). <em>D. simulans st e</em> sequences were from genomic DNA extracted from adult <em>st e D. simulans</em> females. Average coverage was 30×. Reads were aligned to updated reference genomes (R. Graze <em>et al.</em>, unpublished data) using Bowtie (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Langmead <em>et al.</em> 2009</a>) and LAST (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Frith <em>et al.</em> 2010</a>). Alignments were converted to pileup format using SAMtools (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2009</a>). SNP bases were identified from the pileup alignments and compared with alleles identified from DPGP/GenBank. The bases identified from C167.4 resequencing were also compared with the DPGP genome sequences for the C167.4 strain.</p></div></div><div id="sec-14" class="subsection"><h3>Analysis of AI</h3><p id="p-29">To verify the chip’s capacity to identify differences in AI, a subset of SNP probe sets unambiguous for the two alleles from the design and confirmed by the C167.4 resequencing and <em>st e</em> resequencing (where the F<sub>1</sub> is heterozygous) were selected. The model,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/427/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/6/427/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>was fit for probe sets in this subset using the nine F<sub>1</sub> arrays (six RNA, three DNA). Y<sub>ij</sub> is the normalized signal value for the <em>i<sup>th</sup></em> sex, <em>j<sup>th</sup></em> treatment, and the <em>k<sup>th</sup></em> replicate. The treatment groups were defined by combinations of nucleic acid (DNA/RNA) and allele (PM1, PM2, and MM) for a total of <em>j</em> = 1…6 levels and <em>k</em> = 1…3 replicates. AI was examined by testing the difference in hybridization intensity between PM1 and PM2 in the RNA, compared with the difference in the DNA. An F test for this contrast was performed, and the result was corrected for multiple testing using FDR.</p><p id="p-30">The power of detection of AI effects may differ between the sexes due to sex bias in gene expression. There may also be sex-specific differences in AI. Both phenomena would result in a difference in detection of AI between the sexes. Unfortunately, the power for the test of an interaction was low. The AI was also analyzed considering the female and male data separately so that any differences between the sexes in the results might be apparent.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Quality control</h3><p id="p-31">Quality control evaluations showed that the three C167.4 parental DNA hybridizations had overall weaker signals and that the kernel density distribution was markedly different from all of the other chips. The DABG was only 70% for these hybridizations (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) compared with ∼90% for other DNA samples. The remaining chips showed no obvious problems with hybridization. All modules (3′ expression, exon, and SNP) had similar proportions of probes detected above the median of the GC band control signals (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The proportions were ∼72% for RNA samples and ∼90% for DNA samples. The distribution of signal values across all modules was similar for all RNA hybridizations and differed from DNA hybridizations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). PCA identified no other hybridization anomalies. Both sexes had similar hybridization patterns (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1/000596SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1/FigureS1.pdf">Figure S1</a>). The normalized signal intensities of the exon module probe sets and expression module probe sets for Acp and Yp genes gave the expected results (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The average intensity for each gene was consistent between modules (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/427/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quality control analyses of the normalized data. (A) The proportion of probes detected above background (DABG) is reported for all probes sets of each sample: C167.4 parental DNA, st&#xA0;e parental DNA, and DNA and RNA of the F1 genotype. DNA samples are shown in dark gray. RNA samples are shown in light gray. The Y axis is the overall percentage of DABG. Probes were classified according to their GC content and matched to the GC band controls of the corresponding %GC bin. A probe was considered detected when signal strength was higher than the median intensity of the corresponding GC band controls. The three C167.4 parental DNA hybridizations had lower DABG compared with the other two genotypes of DNA samples. (B) Box plot for probe intensity classified by genotype and nucleic acid. DNA samples are shown in dark gray. RNA samples are shown in light gray. The Y axis is the normalized signal. Probes were classified according to their GC content and matched to the GC band controls of the corresponding %GC bin. The five percentile signal for that GC bin was subtracted from the probe for background correction. The corrected signals were then log-transformed. The three C167.4 parental DNA hybridizations had overall weaker signals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901685878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quality control analyses of the normalized data. (A) The proportion of probes detected above background (DABG) is reported for all probes sets of each sample: C167.4 parental DNA, &lt;em&gt;st&#xA0;e&lt;/em&gt; parental DNA, and DNA and RNA of the F&lt;sub&gt;1&lt;/sub&gt; genotype. DNA samples are shown in dark gray. RNA samples are shown in light gray. The Y axis is the overall percentage of DABG. Probes were classified according to their GC content and matched to the GC band controls of the corresponding %GC bin. A probe was considered detected when signal strength was higher than the median intensity of the corresponding GC band controls. The three C167.4 parental DNA hybridizations had lower DABG compared with the other two genotypes of DNA samples. (B) Box plot for probe intensity classified by genotype and nucleic acid. DNA samples are shown in dark gray. RNA samples are shown in light gray. The Y axis is the normalized signal. Probes were classified according to their GC content and matched to the GC band controls of the corresponding %GC bin. The five percentile signal for that GC bin was subtracted from the probe for background correction. The corrected signals were then log-transformed. The three C167.4 parental DNA hybridizations had overall weaker signals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/427/F4.medium.gif" width="265" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/427/F4.medium.gif" width="265" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/427/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/427/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-32" class="first-child">Quality control analyses of the normalized data. (A) The proportion of probes detected above background (DABG) is reported for all probes sets of each sample: C167.4 parental DNA, <em>st e</em> parental DNA, and DNA and RNA of the F<sub>1</sub> genotype. DNA samples are shown in dark gray. RNA samples are shown in light gray. The Y axis is the overall percentage of DABG. Probes were classified according to their GC content and matched to the GC band controls of the corresponding %GC bin. A probe was considered detected when signal strength was higher than the median intensity of the corresponding GC band controls. The three C167.4 parental DNA hybridizations had lower DABG compared with the other two genotypes of DNA samples. (B) Box plot for probe intensity classified by genotype and nucleic acid. DNA samples are shown in dark gray. RNA samples are shown in light gray. The Y axis is the normalized signal. Probes were classified according to their GC content and matched to the GC band controls of the corresponding %GC bin. The five percentile signal for that GC bin was subtracted from the probe for background correction. The corrected signals were then log-transformed. The three C167.4 parental DNA hybridizations had overall weaker signals.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460947/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460947/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Proportions of probes detected above the median GC band control</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/427/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression for known sex-specific genes in female and male RNA samples. The Y axis is the normalized signal. A value around or lower than 3 is close to the background intensity and, therefore, should be considered as not detected. Female samples are shown in red. Male samples are shown in blue. (A) The mean signals of all probe sets for each Acp gene. (B) The expression of individual probe sets designed for Acp genes. (C) The mean signals of all probe sets for each Yp gene. (D) The expression of an individual probe set that was designed for Yp genes. The directions of sex bias are as expected (Acps are male-specific genes, and Yps are female-specific genes). Individual probe sets for the same gene behave consistently." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901685878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Expression for known sex-specific genes in female and male RNA samples. The Y axis is the normalized signal. A value around or lower than 3 is close to the background intensity and, therefore, should be considered as not detected. Female samples are shown in red. Male samples are shown in blue. (A) The mean signals of all probe sets for each Acp gene. (B) The expression of individual probe sets designed for Acp genes. (C) The mean signals of all probe sets for each Yp gene. (D) The expression of an individual probe set that was designed for Yp genes. The directions of sex bias are as expected (Acps are male-specific genes, and Yps are female-specific genes). Individual probe sets for the same gene behave consistently.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/427/F5.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/427/F5.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/427/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/427/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461016" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-34" class="first-child">Expression for known sex-specific genes in female and male RNA samples. The Y axis is the normalized signal. A value around or lower than 3 is close to the background intensity and, therefore, should be considered as not detected. Female samples are shown in red. Male samples are shown in blue. (A) The mean signals of all probe sets for each Acp gene. (B) The expression of individual probe sets designed for Acp genes. (C) The mean signals of all probe sets for each Yp gene. (D) The expression of an individual probe set that was designed for Yp genes. The directions of sex bias are as expected (Acps are male-specific genes, and Yps are female-specific genes). Individual probe sets for the same gene behave consistently.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">To test the smaller format features of the 3′ expression probe sets, we analyzed the effect of sex on expression level. Previous studies in <em>Drosophila</em> (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Bownes 1994</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jin <em>et al.</em> 2001</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Parisi <em>et al.</em> 2003</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Ranz <em>et al.</em> 2003</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">McIntyre <em>et al.</em> 2006</a>; Telonis-Scott <em>et al.</em> 2008) have all found sex bias in overall expression. Analysis of differential expression on the 3′ expression module revealed a strong effect of sex on gene expression. Three different significance levels (FDR &lt; 0.05, FDR &lt; 0.1, and FDR &lt; 0.2) are reported (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Raw <em>P</em> values and adjusted FDR <em>P</em> values are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1/TableS1.csv">Table S1</a>. Among all sex-biased probe sets at FDR &lt; 0.1, 2216 had higher expression in males and 4140 had higher expression in females. Sex bias of expression was similarly analyzed for exon probe sets corresponding to single exons that exist in all transcripts of a gene (constitutive) (<em>n</em> = 47,122; <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1/TableS3.csv">Table S3</a>). The results for exon probe sets were compared with the 3′ expression probe sets corresponding to the same genes. There were 2091 genes with probe sets corresponding to exons contained in all transcripts, which could be easily matched to a single probe set in the 3′ expression module. The majority of genes showed similar sex bias between the 3′ expression and exon module, and simple agreement was high (1720, 82%). The Kappa statistic was 0.63 between the two modules, indicating good agreement. There was no apparent asymmetry in detection, with 106 genes detected by the exon module alone and 265 genes detected by the 3′ expression module alone. The McNemar’s test statistics was 68.1429 with <em>P</em> value smaller than 0.0001. There were 164 genes that showed a significant interaction between exon type and sex; these were considered as showing putative isoform-specific sex bias (FDR &lt; 0.1; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1/TableS2.csv">Table S2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461017/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461017/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Tests for the effect of sex from the 3′ expression and exon probe modules at multiple FDR levels</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Verifying the SNP module</h3><p id="p-37">The <em>D. simulans</em> sequences used to design the SNP probes were sequenced as part of the DPGP. The sequencing strategy of DPGP was to sequence one line at ∼4× coverage and six additional lines (including C167.4) at 1×. The confidence for SNP calls, therefore, varied depending on quality and depth. There were 60,118 probe sets, and of these, 42,978 had SNP base information for C167.4 from DPGP, 35,379 had additional data available from Illumina sequencing of C167.4 RNA, and 49,758 had additional data from Illumina sequencing of <em>st e</em> DNA.</p><p id="p-38">The concordance of C167.4 SNP base calls from the ∼1× DPGP C167.4 strain genome sequence used for the chip design and the resequencing was 66.72%, significantly larger than expected by chance. This rate did not affect the quality of the SNP probe sets, as C167.4 was only one of the seven lines in DPGP that were used for our chip design. The concordance between the resequencing bases and the alleles used in the design was impressive. The C167.4 RNA-Seq base agreed with one of the two alleles identified in the design 99.58% of the time. Agreement between the <em>st e</em> DNA-Seq bases and the two alleles in the design was 99.75%.</p><p id="p-39">Next, the concordance between the resequencing and hybridization was examined by comparing resequencing SNP base calls to the probe bases ranked by the strength of their hybridization signals. The comparison was carried out separately for cases where the resequencing bases were the same for C167.4 and <em>st e</em> (homozygous F<sub>1</sub>) or where they were different (heterozygous F<sub>1</sub>). For <em>n</em> = 13,573 SNP probe sets homozygous at the SNP site in the C167.4/<em>st e</em> genotype, the ranked hybridization intensities of probes corresponding to each base (within a given SNP probe set) was compared with the predicted genotype at the SNP base (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). A SNP probe set was included in this comparison when the common SNP allele call for C167.4 and <em>st e</em> strains corresponded to one of the alleles in the chip design, and the base was also the same in the DPGP sequence for the C167.4 strain. For example, if the SNP allele in both strains is A (with respect to the forward strand), probes corresponding to targets with A at the SNP position are expected to show increased hybridization intensity relative to the signal for the T, G, and C probes for all genotypes tested. The percentage of probe sets where the probes corresponding to the target SNP allele show the highest intensity is reported overall and separately for each base (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The concordance between the resequencing SNP and the C167.4 DNA arrays was significantly lower than other arrays, likely caused by the weaker signal intensity of the C167.4 DNA arrays. Similarly, the ranked hybridization intensities of probes corresponding to each base (within a given SNP probe set) was compared with the predicted genotype at the SNP base for <em>n</em> = 2769 SNP probe sets heterozygous at the SNP site in the C167.4/<em>st e</em> genotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1/TableS4.pdf">Table S4</a>). Our observed concordance is striking, given the small number of genotypes used in this experiment. A previous study (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Borevitz <em>et al.</em> 2007</a>) using arrays for detecting sequence polymorphisms reported a similar error rate. Larger experiments with more samples can reduce the error rate (Edenberg <em>et al.</em> 2005; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rabbee and Speed 2006</a>). In comparison, a short-read sequencing experiment requires more than 200 reads unambiguously mapped to the gene for each gene to achieve a similar result (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fontanillas <em>et al.</em> 2010</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460987/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460987/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Rank of hybridization signal corresponds to the expectation based on sequence information (homozygous genotypes)</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">The hybridization signals were compared with the two alleles (PM1 and PM2) used in the design. The concordances between the two highest ranked probe bases and the PM1 and PM2 bases were high: 96.22% for C167.4 DNA chips and 98.77% for <em>st e</em> DNA chips. For hybrid F<sub>1</sub>’s, the concordance percentage was 98.62% for DNA chips and 97.53% for RNA chips. To determine whether the pattern of hybridization could be used to predict genotype, we performed a linear discriminate analysis (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Johnson and Wichern 1992</a>) for several arbitrarily selected probe sets heterozygous for the SNP base in the F<sub>1</sub> hybrid. All that we examined showed visual separation of the expression patterns (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). These comparisons confirmed that the hybridizations performed as expected.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/6/427/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linear discriminant plots of three genotypes: AA, AC, and CC. Different genotypes had hybridization patterns that are visually separable by linear discriminant (LD) analysis. Each genotype is a different color." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901685878" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Linear discriminant plots of three genotypes: AA, AC, and CC. Different genotypes had hybridization patterns that are visually separable by linear discriminant (LD) analysis. Each genotype is a different color.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/6/427/F6.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/6/427/F6.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/6/427/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/6/427/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460938" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-42" class="first-child">Linear discriminant plots of three genotypes: AA, AC, and CC. Different genotypes had hybridization patterns that are visually separable by linear discriminant (LD) analysis. Each genotype is a different color.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>AI analysis</h3><p id="p-43">There were 33,914 unambiguous probe sets with sequence information for both the <em>st e</em> and C167.4 resequencing experiments at the SNP site. The SNP base was the same for the parental two lines for 74.78% of these probe sets, which were classified as homozygous for the F<sub>1</sub> genotypes (<em>n</em> = 25,362). The rest of the probe sets (8552) had heterozygous F<sub>1</sub> genotypes. The 6579 autosomal probe sets were analyzed for AI on the combined data from both sexes and for each sex separately (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1/TableS5.csv">Table S5</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460940/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460940/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4 </span> <span class="caption-title">Allele imbalance overall and separated by sex (<em>n</em> = 6,579)</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section conclusions" id="sec-19"><h2 class="">Conclusion</h2><p id="p-45">Our custom platform performs similarly to previous array platforms with a larger feature size (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Jin <em>et al.</em> 2001</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Ranz <em>et al.</em> 2003</a>; <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">McIntyre <em>et al.</em> 2006</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Wayne <em>et al.</em> 2007</a>). <a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">McIntyre <em>et al.</em> (2006)</a> analyzed 10,014 transcripts in eight lines of <em>D. melanogaster</em> and identified 5221 sex-biased transcripts at FDR 0.05 (56% male bias and 44% female bias). The overall sex effect for eight genotypes was 53%. Similarly, on a study using nine <em>D. melanogaster</em> lines, <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Wayne <em>et al.</em> (2007)</a> reported 7617 out of 9312 genes with sexually dimorphic expression (4070 male bias and 3547 female bias). These previous studies used multiple genotypes of <em>D. melanogaster</em>. When the data for <a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Wayne <em>et al.</em> (2007)</a> were reanalyzed for each genotype separately, the percentage of sex-biased genes ranged from 1.29 to 40.09%. For the genotype considered in this study, 31% of genes showed a significant sex effect, close to the upper end of the range. A slight excess of genes with increased expression in females was also observed in this analysis, as is seen in previous analyses (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Ranz <em>et al.</em> 2003</a>). These findings are consistent with findings from arrays with a larger feature size.</p><p id="p-46">Two previous studies using array designs (<a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">McIntyre <em>et al.</em> 2006</a>; Telonis-Scott <em>et al.</em> 2008) found significant sex differences in alternative exon usage for many genes. For the single genotype examined here, approximately 5.6% of the genes examined showed evidence of sex-specific isoform expression. For four genes that are components of the sex determination pathway with previously reported sex-specific splicing in adults (<em><a href="http://flybase.org/reports/FBgn0003742.html">tra2</a></em>, <em><a href="http://flybase.org/reports/FBgn0003659.html">Sxl</a></em>, <em><a href="http://flybase.org/reports/FBgn0000504.html">dsx</a></em>, and <em><a href="http://flybase.org/reports/FBgn0004652.html">fru</a></em>), at least one exon shows evidence of sex bias.</p><p id="p-47">This is the first genome-wide study of allele-specific expression variation within <em>D. simulans</em>. Although only one genotype was used, significant differences in AI were detected for 37% of probe sets examined that contained heterozygous SNP loci. Other work using <em>a priori</em> selected genes found almost 67% of the genes tested showed evidence for AI within species (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Wittkopp <em>et al.</em> 2008</a>). This chip can be used to detect allele-specific variation in expression within species. Large differences between males and females were detected in the number of probe sets that showed significant differences in AI. It is currently unclear whether this result is explained by differences in AI between males and females or by sex bias in overall expression making power for detection uneven.</p><p id="p-48">As in microarray studies, to adequately assess ASE for a particular transcript using RNA-Seq, there must be adequate coverage for both alleles of that particular gene. For samples from the same organism and tissue, detection of transcription for a particular exon in four gigabases of RNA-Seq data are 57% (<span class="sc">r</span>. Graze <em>et al.</em>, unpublished data), whereas detection is 72% for tiling arrays (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Graze <em>et al.</em> 2009</a>). Initial studies of AI examined how many informative reads were needed per gene for estimation of allelic frequencies (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Fontanillas <em>et al.</em> 2010</a>). This study suggests that average depth of coverage needed is quite large if most genes are to be evaluated. The actual coverage needed depends on specific assumptions and the number varies, but it is often in excess of 100×. Other examinations of RNA-Seq find that a minimum average depth of five reads per nucleotide are needed for estimation of expression (McIntyre <em>et al.</em> 2011). One lane of a GAIIX provides sufficient reads at a coverage of 5× to assess ∼30% of the transcriptome (McIntyre <em>et al.</em> 2011). Arrays still provide a cost effective way of assessing transcription for the whole genome (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Malone and Oliver 2011</a>).</p><p id="p-49">Detailed studies within species that examine AI variation genome-wide and identify the impact of sex on this variation are needed to understand the true extent of <em>cis</em> regulatory variation within species in <em>Drosophila</em>. This array is a good tool for such studies because it will allow the overall and allele-specific components of expression variation to be examined in a single experiment on a single platform for many more genes than has previously been possible.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">This work is supported by National Institutes of Health grants 5R01GM077618-A5 and 5R01GM077618-S1.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000596/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Arrays have been submitted to the GEO database at NCBI as the custom platform GPL11273 and series GSE31750. Sequencing data have been submitted to the SRA database at NCBI as SRP005952.</p></li><li class="fn" id="fn-7"><p id="p-51"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-06-15"><span class="received-label">Received </span>June 15, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-09-11"><span class="accepted-label">Accepted </span>September 11, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Yang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-1.6.427.1"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Affymetrix, Inc</span>. <span class="cit-pub-date">2005</span> <span class="cit-article-title">Exon array background correction.</span> <span class="cit-comment">Available at: <a href="http://media.affymetrix.com/support/technical/whitepapers/exon_background_correction_whitepaper.pdf">http://media.affymetrix.com/support/technical/whitepapers/exon_background_correction_whitepaper.pdf</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.2" data-doi="10.1006/abio.2000.4493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahmadian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharizadeh</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gustafsson</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterky</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyrén</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Single-nucleotide polymorphism analysis by pyrosequencing.</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DAhmadian%26rft.auinit1%253DA.%26rft.volume%253D280%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D110%26rft.atitle%253DSingle-nucleotide%2Bpolymorphism%2Banalysis%2Bby%2Bpyrosequencing.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fabio.2000.4493%26rft_id%253Dinfo%253Apmid%252F10805527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/abio.2000.4493&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10805527&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086488400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.3" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.4" data-doi="10.1186/1471-2164-10-471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ammar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heisler</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nislow</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comparative analysis of DNA barcode microarray feature size.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAmmar%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.spage%253D471%26rft.epage%253D471%26rft.atitle%253DA%2Bcomparative%2Banalysis%2Bof%2BDNA%2Bbarcode%2Bmicroarray%2Bfeature%2Bsize.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-471%26rft_id%253Dinfo%253Apmid%252F19825181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19825181&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.5" data-doi="10.1371/journal.pbio.0050310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.-P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Population genomics: whole-genome analysis of polymorphism and divergence in <em>Drosophila simulans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De310%26rft.epage%253De310%26rft.atitle%253DPopulation%2Bgenomics%253A%2Bwhole-genome%2Banalysis%2Bof%2Bpolymorphism%2Band%2Bdivergence%2Bin%2BDrosophila%2Bsimulans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050310%26rft_id%253Dinfo%253Apmid%252F17988176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17988176&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.7" data-doi="10.1093/nar/gni032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karsch-Mizrachi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GenBank.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">D34</span>–<span class="cit-lpage">D38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBenson%26rft.auinit1%253DD.%2BA.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD34%26rft.epage%253DD38%26rft.atitle%253DGenBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgni032%26rft_id%253Dinfo%253Apmid%252F15608212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gni032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15608212&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226524300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.8" data-doi="10.1016/S0140-6736(86)90837-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bland</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Statistical methods for assessing agreement between two methods of clinical measurement.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DBland%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D1%26rft.issue%253D8476%26rft.spage%253D307%26rft.epage%253D310%26rft.atitle%253DStatistical%2Bmethods%2Bfor%2Bassessing%2Bagreement%2Bbetween%2Btwo%2Bmethods%2Bof%2Bclinical%2Bmeasurement.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252886%252990837-8%26rft_id%253Dinfo%253Apmid%252F2868172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(86)90837-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2868172&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986AYW4000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bland</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Misleading statistics: errors in textbooks, software and manuals.</span> <abbr class="cit-jnl-abbrev">Int. J. Epidemiol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BEpidemiol.%26rft.volume%253D17%26rft.spage%253D201%26rft.atitle%253DMEN%2BWHO%2BDO%2BNOT%2BDRINK%253A%2BA%2BREPORT%2BFROM%2BTHE%2BBRITISH%2BREGIONAL%2BHEART%2BSTUDY%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.10" data-doi="10.1073/pnas.0705323104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazen</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide patterns of single-feature polymorphism in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">12057</span>–<span class="cit-lpage">12062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0705323104%26rft_id%253Dinfo%253Apmid%252F17626786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI5LzEyMDU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.11" data-doi="10.1002/bies.950161009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bownes</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The regulation of the yolk protein genes, a family of sex differentiation genes in <em>Drosophila</em> <em>melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DBownes%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D745%26rft.epage%253D752%26rft.atitle%253DThe%2Bregulation%2Bof%2Bthe%2Byolk%2Bprotein%2Bgenes%252C%2Ba%2Bfamily%2Bof%2Bsex%2Bdifferentiation%2Bgenes%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.950161009%26rft_id%253Dinfo%253Apmid%252F7980478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.950161009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7980478&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.12" data-doi="10.1038/459927a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Celniker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">L. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunsalus</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Unlocking the secrets of the genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCelniker%26rft.auinit1%253DS.%2BE.%26rft.volume%253D459%26rft.issue%253D7249%26rft.spage%253D927%26rft.epage%253D930%26rft.atitle%253DUnlocking%2Bthe%2Bsecrets%2Bof%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F459927a%26rft_id%253Dinfo%253Apmid%252F19536255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/459927a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19536255&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267063500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.13" data-doi="10.1073/pnas.0606054104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dandy</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grainger</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Array feature size influences nucleic acid surface capture in DNA microarrays.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8223</span>–<span class="cit-lpage">8228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0606054104%26rft_id%253Dinfo%253Apmid%252F17485675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzIwLzgyMjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.14" data-doi="10.1093/bioinformatics/btp579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Degner</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marioni</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nkadori</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effect of read-mapping biases on detecting allele-specific expression from RNA-sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3207</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp579%26rft_id%253Dinfo%253Apmid%252F19808877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19808877&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272464000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.15" data-doi="10.1038/onc.2009.194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de la Chapelle</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic predisposition to human disease: allele-specific expression and low-penetrance regulatory loci.</span> <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3345</span>–<span class="cit-lpage">3348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253Dde%2Bla%2BChapelle%26rft.auinit1%253DA.%26rft.volume%253D28%26rft.issue%253D38%26rft.spage%253D3345%26rft.epage%253D3348%26rft.atitle%253DGenetic%2Bpredisposition%2Bto%2Bhuman%2Bdisease%253A%2Ballele-specific%2Bexpression%2Band%2Blow-penetrance%2Bregulatory%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fonc.2009.194%26rft_id%253Dinfo%253Apmid%252F19597467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/onc.2009.194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19597467&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270103300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudoit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callow</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Statistical methods for identifying differentially expressed genes in replicated cDNA microarray experiments.</span> <abbr class="cit-jnl-abbrev">Statist. Sinica</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatist.%2BSinica%26rft.volume%253D12%26rft.spage%253D111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.17" data-doi="10.1534/genetics.108.093419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dworkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic changes accompanying the evolution of host specialization in <em>Drosophila sechellia</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D181%26rft.spage%253D721%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.093419%26rft_id%253Dinfo%253Apmid%252F19033155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.093419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19033155&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270213400030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.427.18" data-doi="10.1186/1471-2156-6-S1-S2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edenberg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bierut</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyce</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cawley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Description of the data from the Collaborative Study on the Genetics of Alcoholism (COGA) and single-nucleotide polymorphism genotyping for Genetic Analysis Workshop 14.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">Suppl. 1</span>):<span class="cit-fpage">S2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D6%26rft.spage%253DS2%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-6-S1-S2%26rft_id%253Dinfo%253Apmid%252F16451628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-6-S1-S2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16451628&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.19" data-doi="10.1101/gr.101576.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">L.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">H.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Natural selection on cis and trans regulation in yeasts.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.101576.109%26rft_id%253Dinfo%253Apmid%252F20445163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvODI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-1.6.427.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fleiss</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-source">Statistical Methods for Rates and Proportions.</span> <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-1.6.427.21"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">FlyBase</span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A database of <em>Drosophila</em> genes and genomes.</span> <span class="cit-comment"><a href="http://www.flybase.org">http://www.flybase.org</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.22" data-doi="10.1111/j.1365-294X.2010.04472.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fontanillas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittkopp</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russ</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Key considerations for measuring allelic expression on a genomic scale using high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D19%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04472.x%26rft_id%253Dinfo%253Apmid%252F20331781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04472.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20331781&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275645700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.23" data-doi="10.1093/nar/gkq010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frith</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Incorporating sequence quality data into alignment improves DNA read mapping.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq010%26rft_id%253Dinfo%253Apmid%252F20110255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110255&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.24" data-doi="10.1534/genetics.109.105957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graze</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Main</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Regulatory divergence in Drosophila melanogaster and D. simulans, a genome-wide analysis of allele-specific expression.</span> <abbr class="cit-jnl-abbrev">Genetics.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics.%26rft.volume%253D183%26rft.spage%253D547%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105957%26rft_id%253Dinfo%253Apmid%252F19667135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.105957&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19667135&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271558300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.25" data-doi="10.1007/s11103-008-9290-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rupe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide allele-specific expression analysis using Massively Parallel Signature Sequencing (MPSS) reveals cis- and trans-effects on gene expression in maize hybrid meristem tissue.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DGuo%26rft.auinit1%253DM.%26rft.volume%253D66%26rft.issue%253D5%26rft.spage%253D551%26rft.epage%253D563%26rft.atitle%253DGenome-wide%2Ballele-specific%2Bexpression%2Banalysis%2Busing%2BMassively%2BParallel%2BSignature%2BSequencing%2B%2528MPSS%2529%2Breveals%2Bcis-%2Band%2Btrans-effects%2Bon%2Bgene%2Bexpression%2Bin%2Bmaize%2Bhybrid%2Bmeristem%2Btissue.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-008-9290-z%26rft_id%253Dinfo%253Apmid%252F18224447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-008-9290-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18224447&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253202200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.26" data-doi="10.1038/ng766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfinger</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Passador-Gurgel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The contributions of sex, genotype and age to transcriptional variance in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJin%26rft.auinit1%253DW.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D389%26rft.epage%253D395%26rft.atitle%253DThe%2Bcontributions%2Bof%2Bsex%252C%2Bgenotype%2Band%2Bage%2Bto%2Btranscriptional%2Bvariance%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng766%26rft_id%253Dinfo%253Apmid%252F11726925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11726925&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172507500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.27" data-doi="10.1016/j.pharmthera.2004.11.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadee</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Polymorphisms affecting gene regulation and mRNA processing: broad implications for pharmacogenetics.</span> <abbr class="cit-jnl-abbrev">Pharmacol. Ther.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacology%2B%2526%2Btherapeutics%26rft.stitle%253DPharmacol%2BTher%26rft.aulast%253DJohnson%26rft.auinit1%253DA.%2BD.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D38%26rft.atitle%253DPolymorphisms%2Baffecting%2Bgene%2Bregulation%2Band%2BmRNA%2Bprocessing%253A%2Bbroad%2Bimplications%2Bfor%2Bpharmacogenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pharmthera.2004.11.001%26rft_id%253Dinfo%253Apmid%252F15781120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pharmthera.2004.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15781120&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228195600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.28" data-doi="10.1126/science.1090100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrett-Engele</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loerch</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">2141</span>–<span class="cit-lpage">2144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJohnson%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D302%26rft.issue%253D5653%26rft.spage%253D2141%26rft.epage%253D2144%26rft.atitle%253DGenome-Wide%2BSurvey%2Bof%2BHuman%2BAlternative%2BPre-mRNA%2BSplicing%2Bwith%2BExon%2BJunction%2BMicroarrays%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090100%26rft_id%253Dinfo%253Apmid%252F14684825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDIvNTY1My8yMTQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-1.6.427.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wichern</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Applied Multivariate Statistical Analysis.</span> <span class="cit-publ-name">Prentice Hall</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.30" data-doi="10.1093/molbev/msn055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kopp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barmina</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution of gene expression in the drosophila olfactory system.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn055%26rft_id%253Dinfo%253Apmid%252F18296696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msn055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18296696&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255758200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.31" data-doi="10.1038/ng.2007.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benovoy</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dias</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Provencher</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide analysis of transcript isoform variation in humans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKwan%26rft.auinit1%253DT.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D225%26rft.epage%253D231%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Btranscript%2Bisoform%2Bvariation%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2007.57%26rft_id%253Dinfo%253Apmid%252F18193047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2007.57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18193047&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252732900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.32" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.33" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map (SAM) Format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Allelic variation in gene expression is common in the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1855</span>–<span class="cit-lpage">1862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLo%26rft.auinit1%253DH.%2BS.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1855%26rft.epage%253D1862%26rft.atitle%253DAllelic%2BVariation%2Bin%2BGene%2BExpression%2BIs%2BCommon%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Apmid%252F12902379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTg1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.427.35" data-doi="10.1101/gr.076620.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ting</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide mis-expression of X-linked versus autosomal genes associated with hybrid male sterility.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1097</span>–<span class="cit-lpage">1102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.076620.108%26rft_id%253Dinfo%253Apmid%252F20511493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzgvMTA5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.36" data-doi="10.1186/1741-7007-9-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Microarrays, deep sequencing and the true measure of the transcriptome.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DMalone%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D9%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DMicroarrays%252C%2Bdeep%2Bsequencing%2Band%2Bthe%2Btrue%2Bmeasure%2Bof%2Bthe%2Btranscriptome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-9-34%26rft_id%253Dinfo%253Apmid%252F21627854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-9-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21627854&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.37" data-doi="10.1186/gb-2006-7-8-r79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genissel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westerman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Junk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Sex-specific expression of alternative transcripts in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMcIntyre%26rft.auinit1%253DL.%2BM.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253DR79%26rft.epage%253DR79%26rft.atitle%253DSex-specific%2Bexpression%2Bof%2Balternative%2Btranscripts%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-8-r79%26rft_id%253Dinfo%253Apmid%252F16934145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-8-r79&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16934145&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.427.38" data-doi="10.1186/1471-2164-12-293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopiano</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oberg</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNA-seq: technical variability and sampling.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMcIntyre%26rft.auinit1%253DL.%2BM.%26rft.volume%253D12%26rft.spage%253D293%26rft.epage%253D293%26rft.atitle%253DRNA-seq%253A%2Btechnical%2Bvariability%2Band%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-293%26rft_id%253Dinfo%253Apmid%252F21645359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21645359&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.39" data-doi="10.1101/gr.102491.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McManus</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coolon</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eipper-Mains</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regulatory divergence in Drosophila revealed by mRNA-seq.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102491.109%26rft_id%253Dinfo%253Apmid%252F20354124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvODE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.40" data-doi="10.1371/journal.pbio.0060108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maia</span>  <span class="cit-name-given-names">A.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O'Reilly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teschendorff</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">S.-F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Allele-specific up-regulation of FGFR2 increases susceptibility to breast cancer.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DMeyer%26rft.auinit1%253DK.%2BB.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De108%26rft.epage%253De108%26rft.atitle%253DAllele-specific%2Bup-regulation%2Bof%2BFGFR2%2Bincreases%2Bsusceptibility%2Bto%2Bbreast%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060108%26rft_id%253Dinfo%253Apmid%252F18462018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18462018&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-1.6.427.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kutner</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source"><em>Applied Linear Statistical Models: Regression</em>, <em>Analysis of Variance</em>, <em>and Experimental Designs</em></span>. <span class="cit-publ-name">Irwin</span>, <span class="cit-publ-loc">Homewood, Ill</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.42" data-doi="10.1093/hmg/ddn285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nica</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dermitzakis</span>  <span class="cit-name-given-names">E. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Using gene expression to investigate the genetic basis of complex disorders.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">R129</span>–<span class="cit-lpage">R134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddn285%26rft_id%253Dinfo%253Apmid%252F18852201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddn285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852201&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262764200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.43" data-doi="10.1371/journal.pgen.1000895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nica</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimas</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stranger</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beazley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Candidate causal regulatory effects by integration of expression QTLs with complex trait genetic associations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNica%26rft.auinit1%253DA.%2BC.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De1000895%26rft.epage%253De1000895%26rft.atitle%253DCandidate%2Bcausal%2Bregulatory%2Beffects%2Bby%2Bintegration%2Bof%2Bexpression%2BQTLs%2Bwith%2Bcomplex%2Btrait%2Bgenetic%2Bassociations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000895%26rft_id%253Dinfo%253Apmid%252F20369022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20369022&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.44" data-doi="10.1126/science.1079190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parisi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuttall</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naiman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouffard</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Paucity of genes on the Drosophila X chromosome showing male-biased expression.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1079190%26rft_id%253Dinfo%253Apmid%252F12511656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTkvNTYwNy82OTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.45" data-doi="10.1152/physiolgenomics.00163.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pastinen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sladek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammak</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A survey of genetic and epigenetic variation affecting human gene expression.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00163.2003%26rft_id%253Dinfo%253Apmid%252F14583597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTU6InBoeXNpb2xnZW5vbWljcyI7czo1OiJyZXNpZCI7czo4OiIxNi8yLzE4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.46" data-doi="10.1038/nature08872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marioni</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degner</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engelhardt</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Understanding mechanisms underlying human gene expression variation with RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPickrell%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D768%26rft.epage%253D772%26rft.atitle%253DUnderstanding%2Bmechanisms%2Bunderlying%2Bhuman%2Bgene%2Bexpression%2Bvariation%2Bwith%2BRNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08872%26rft_id%253Dinfo%253Apmid%252F20220758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220758&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.47" data-doi="10.1093/bioinformatics/bti741"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabbee</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A genotype calling algorithm for Affymetrix SNP arrays.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti741%26rft_id%253Dinfo%253Apmid%252F16267090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti741&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16267090&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234433500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.48" data-doi="10.1126/science.1085881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ranz</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castillo-Davis</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meiklejohn</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sex-dependent gene expression and evolution of the Drosophila transcriptome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">1742</span>–<span class="cit-lpage">1745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRanz%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D300%26rft.issue%253D5626%26rft.spage%253D1742%26rft.epage%253D1745%26rft.atitle%253DSex-Dependent%2BGene%2BExpression%2Band%2BEvolution%2Bof%2Bthe%2BDrosophila%2BTranscriptome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085881%26rft_id%253Dinfo%253Apmid%252F12805547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.49" data-doi="10.1038/ng1086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rifkin</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Evolution of gene expression in the Drosophila melanogaster subgroup.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">138</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRifkin%26rft.auinit1%253DS.%2BA.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D138%26rft.epage%253D144%26rft.atitle%253DEvolution%2Bof%2Bgene%2Bexpression%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2Bsubgroup.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1086%26rft_id%253Dinfo%253Apmid%252F12548287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12548287&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180773700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.50" data-doi="10.1038/nrg1964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetics of global gene expression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">862</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253D862%26rft.epage%253D872%26rft.atitle%253DGenetics%2Bof%2Bglobal%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1964%26rft_id%253Dinfo%253Apmid%252F17047685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1964&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17047685&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241384100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.51" data-doi="10.1038/73432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherf</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perou</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rees</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Systematic variation in gene expression patterns in human cancer cell lines.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRoss%26rft.auinit1%253DD.%2BT.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D227%26rft.epage%253D235%26rft.atitle%253DSystematic%2Bvariation%2Bin%2Bgene%2Bexpression%2Bpatterns%2Bin%2Bhuman%2Bcancer%2Bcell%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F73432%26rft_id%253Dinfo%253Apmid%252F10700174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/73432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10700174&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085590600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.52" data-doi="10.1126/science.270.5235.467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schena</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shalon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Quantitative monitoring of gene expression patterns with a complementary DNA microarray.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">270</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchena%26rft.auinit1%253DM.%26rft.volume%253D270%26rft.issue%253D5235%26rft.spage%253D467%26rft.epage%253D470%26rft.atitle%253DQuantitative%2BMonitoring%2Bof%2BGene%2BExpression%2BPatterns%2Bwith%2Ba%2BComplementary%2BDNA%2BMicroarray%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.270.5235.467%26rft_id%253Dinfo%253Apmid%252F7569999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzAvNTIzNS80NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzYvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.53" data-doi="10.1371/journal.pgen.1000006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serre</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sladek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Differential allelic expression in the human genome: a robust approach to identify genetic and epigenetic <em>cis</em>-acting mechanisms regulating gene expression.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSerre%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000006%26rft.epage%253De1000006%26rft.atitle%253DDifferential%2Ballelic%2Bexpression%2Bin%2Bthe%2Bhuman%2Bgenome%253A%2Ba%2Brobust%2Bapproach%2Bto%2Bidentify%2Bgenetic%2Band%2Bepigenetic%2Bcis-acting%2Bmechanisms%2Bregulating%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000006%26rft_id%253Dinfo%253Apmid%252F18454203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454203&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-1.6.427.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Density Estimation for Statistics and Data Analysis.</span> <span class="cit-publ-name">Champman and Hall</span>, <span class="cit-publ-loc">London, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.55" data-doi="10.1016/j.ygeno.2004.05.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatoyannopoulos</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The genomics of gene expression.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DStamatoyannopoulos%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D84%26rft.issue%253D3%26rft.spage%253D449%26rft.epage%253D457%26rft.atitle%253DThe%2Bgenomics%2Bof%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2004.05.002%26rft_id%253Dinfo%253Apmid%252F15498452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2004.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15498452&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223409100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.56" data-doi="10.1038/ng2142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranger</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nica</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Population genomics of human gene expression.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DStranger%26rft.auinit1%253DB.%2BE.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1217%26rft.epage%253D1224%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bhuman%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2142%26rft_id%253Dinfo%253Apmid%252F17873874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17873874&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.57" data-doi="10.1101/gad.9.15.1857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szabó</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Biallelic expression of imprinted genes in the mouse germ line: implications for erasure, establishment, and mechanisms of genomic imprinting.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1857</span>–<span class="cit-lpage">1868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSzabo%26rft.auinit1%253DP%2BE%26rft.volume%253D9%26rft.issue%253D15%26rft.spage%253D1857%26rft.epage%253D1868%26rft.atitle%253DBiallelic%2Bexpression%2Bof%2Bimprinted%2Bgenes%2Bin%2Bthe%2Bmouse%2Bgerm%2Bline%253A%2Bimplications%2Bfor%2Berasure%252C%2Bestablishment%252C%2Band%2Bmechanisms%2Bof%2Bgenomic%2Bimprinting.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.15.1857%26rft_id%253Dinfo%253Apmid%252F7649473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8xNS8xODU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.427.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Telonis-Scott</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kopp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> Sex-specific splicing in <em>Drosophila</em>: widespread occurrence, tissue-specificity, and evolutionary conservation. Genetics 181: 421–434.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.427.59" data-doi="10.1093/nar/22.22.4673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">4673</span>–<span class="cit-lpage">4680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.22.4673%26rft_id%253Dinfo%253Apmid%252F7984417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/22.22.4673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7984417&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PU19900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.60" data-doi="10.1111/j.0030-1299.2005.13727.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verhoeven</span>  <span class="cit-name-given-names">K. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Implementing false discovery rate control: increasing your power.</span> <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D108%26rft.spage%253D643%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0030-1299.2005.13727.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0030-1299.2005.13727.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000226898200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.6.427.61" data-doi="10.1016/j.ajhg.2009.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verlaan</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlivet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunninghake</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madore</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larivière</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Allele-specific chromatin remodeling in the ZPBP2/GSDMB/ORMDL3 locus associated with the risk of asthma and autoimmune disease.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DVerlaan%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D85%26rft.issue%253D3%26rft.spage%253D377%26rft.epage%253D393%26rft.atitle%253DAllele-specific%2Bchromatin%2Bremodeling%2Bin%2Bthe%2BZPBP2%252FGSDMB%252FORMDL3%2Blocus%2Bassociated%2Bwith%2Bthe%2Brisk%2Bof%2Basthma%2Band%2Bautoimmune%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.08.007%26rft_id%253Dinfo%253Apmid%252F19732864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19732864&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270104500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.62" data-doi="10.1182/blood-2008-03-144899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Momary</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavallari</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Regulatory polymorphism in vitamin K epoxide reductase complex subunit 1 (VKORC1) affects gene expression and warfarin dose requirement.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2008-03-144899%26rft_id%253Dinfo%253Apmid%252F18523153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTEyLzQvMTAxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.63" data-doi="10.1073/pnas.0705441104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telonis-Scott</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harshman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kopp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Simpler mode of inheritance of transcriptional variation in male Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">18577</span>–<span class="cit-lpage">18582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0705441104%26rft_id%253Dinfo%253Apmid%252F18003923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ3LzE4NTc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS82LzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.64" data-doi="10.1038/ng.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittkopp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haerum</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Regulatory changes underlying expression differences within and between <em>Drosophila</em> species.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWittkopp%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D346%26rft.epage%253D350%26rft.atitle%253DRegulatory%2Bchanges%2Bunderlying%2Bexpression%2Bdifferences%2Bwithin%2Band%2Bbetween%2BDrosophila%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.77%26rft_id%253Dinfo%253Apmid%252F18278046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18278046&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253548400027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.65" data-doi="10.1038/nature02698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittkopp</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haerum</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evolutionary changes in cis and trans gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWittkopp%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D430%26rft.issue%253D6995%26rft.spage%253D85%26rft.epage%253D88%26rft.atitle%253DEvolutionary%2Bchanges%2Bin%2Bcis%2Band%2Btrans%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02698%26rft_id%253Dinfo%253Apmid%252F15229602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15229602&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222356800048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.66" data-doi="10.1016/S0965-1748(96)00084-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfner</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Tokens of love: functions and regulation of drosophila male accessory gland products.</span> <abbr class="cit-jnl-abbrev">Insect Biochem. Mol. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2Bbiochemistry%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DInsect%2BBiochem%2BMol%2BBiol%26rft.aulast%253DWolfner%26rft.auinit1%253DM.%2BF.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D179%26rft.epage%253D192%26rft.atitle%253DTokens%2Bof%2Blove%253A%2Bfunctions%2Band%2Bregulation%2Bof%2BDrosophila%2Bmale%2Baccessory%2Bgland%2Bproducts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0965-1748%252896%252900084-7%26rft_id%253Dinfo%253Apmid%252F9090115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0965-1748(96)00084-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9090115&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WP14700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.67" data-doi="10.1038/nmeth.1357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egli</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Digital RNA allelotyping reveals tissue-specific and allele-specific gene expression in human.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DZhang%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D613%26rft.epage%253D618%26rft.atitle%253DDigital%2BRNA%2Ballelotyping%2Breveals%2Btissue-specific%2Band%2Ballele-specific%2Bgene%2Bexpression%2Bin%2Bhuman.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1357%26rft_id%253Dinfo%253Apmid%252F19620972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19620972&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268493700024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-1.6.427.68" data-doi="10.1534/genetics.109.103499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Global analysis of allele-specific expression in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D943%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103499%26rft_id%253Dinfo%253Apmid%252F19474198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.103499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19474198&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F6%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270214000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/6/427.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/417" title="Double-Strand Break Repair and Holliday Junction Processing Are Required for Chromosome Processing in Stationary-Phase Escherichia coli Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/6/437" title="QTL Mapping and Candidate Gene Analysis of Telomere Length Control Factors in Maize (Zea mays L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/6"><span>Volume 1 Issue 6, November 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/6.cover.gif?itok=aUjFCEJ8" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/6/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JA98ZyRA3FxB03AovLLN2xCEmuCs7n3FaBhST2H32UE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/6/427" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/6/427" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/6/427" class="active" data-icon-position="" data-hide-link-title="0">Partitioning Transcript Variation in Drosophila: Abundance, Isoforms, and Alleles</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459201" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NK3tHvJ-1BnkTLvW2p9LQ2kRHJlKd356bmV9b25CmmQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/427.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/6/427" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-jZs1EIk0SX6tuJcSpDVotOWuM2nz2-It_Jo1c68bOLA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/6/427" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/6/427.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459201" id="node459201--2" data-pisa="ggg;1/6/427" data-pisa-master="ggg;g3.111.000596" data-apath="/ggg/1/6/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Partitioning Transcript Variation in <em>Drosophila</em>: Abundance, Isoforms, and Alleles</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yajie Yang</span>, <span class="highwire-citation-author" data-delta="1">Rita M. Graze</span>, <span class="highwire-citation-author" data-delta="2">Brandon M. Walts</span>, <span class="highwire-citation-author" data-delta="3">Cecilia M. Lopez</span>, <span class="highwire-citation-author" data-delta="4">Henry V. Baker</span>, <span class="highwire-citation-author" data-delta="5">Marta L. Wayne</span>, <span class="highwire-citation-author" data-delta="6">Sergey V. Nuzhdin</span> and <span class="highwire-citation-author" data-delta="7">Lauren M. McIntyre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459201--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yajie Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yajie%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYajie%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rita M. Graze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BM.%2BGraze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graze%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BM.%2BGraze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Brandon M. Walts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandon%2BM.%2BWalts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walts%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandon%2BM.%2BWalts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cecilia M. Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecilia%2BM.%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecilia%2BM.%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Henry V. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henry%2BV.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20HV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenry%2BV.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Marta L. Wayne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology, University of Florida, Gainesville, FL, 32611-8525</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marta%2BL.%2BWayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wayne%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarta%2BL.%2BWayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sergey V. Nuzhdin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Molecular and Computational Biology, University of Southern California, Los Angeles, CA 90089-2910</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergey%2BV.%2BNuzhdin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nuzhdin%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASergey%2BV.%2BNuzhdin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Statistics, University of Florida, Gainesville, FL 32611-8545</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/6/427" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/6/427&amp;title=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/6/427&amp;title=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/6/427&amp;title=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/6/427&amp;text=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/6/427&amp;title=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/6/427&amp;t=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/6/427&amp;title=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/6/427&amp;title=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459201/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459201" id="node459201--3" data-pisa="ggg;1/6/427" data-pisa-master="ggg;g3.111.000596" data-apath="/ggg/1/6/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Partitioning Transcript Variation in <em>Drosophila</em>: Abundance, Isoforms, and Alleles</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yajie Yang</span>, <span class="highwire-citation-author" data-delta="1">Rita M. Graze</span>, <span class="highwire-citation-author" data-delta="2">Brandon M. Walts</span>, <span class="highwire-citation-author" data-delta="3">Cecilia M. Lopez</span>, <span class="highwire-citation-author" data-delta="4">Henry V. Baker</span>, <span class="highwire-citation-author" data-delta="5">Marta L. Wayne</span>, <span class="highwire-citation-author" data-delta="6">Sergey V. Nuzhdin</span> and <span class="highwire-citation-author" data-delta="7">Lauren M. McIntyre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459201--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yajie Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yajie%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYajie%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rita M. Graze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BM.%2BGraze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graze%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BM.%2BGraze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Brandon M. Walts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandon%2BM.%2BWalts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walts%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandon%2BM.%2BWalts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cecilia M. Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecilia%2BM.%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecilia%2BM.%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Henry V. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henry%2BV.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20HV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHenry%2BV.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Marta L. Wayne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology, University of Florida, Gainesville, FL, 32611-8525</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marta%2BL.%2BWayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wayne%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarta%2BL.%2BWayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sergey V. Nuzhdin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Molecular and Computational Biology, University of Southern California, Los Angeles, CA 90089-2910</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergey%2BV.%2BNuzhdin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nuzhdin%20SV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASergey%2BV.%2BNuzhdin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lauren M. McIntyre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetics Institute, University of Florida, Gainesville, FL 32610-3610</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics and Microbiology, University of Florida, Gainesville, FL 32610-0266</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Statistics, University of Florida, Gainesville, FL 32611-8545</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BM.%2BMcIntyre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McIntyre%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALauren%2BM.%2BMcIntyre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mcintyre@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mcintyre@ufl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=427&amp;rft.epage=436&amp;rft.atitle=Partitioning%20Transcript%20Variation%20in%20Drosophila%3A%20Abundance%2C%20Isoforms%2C%20and%20Alleles&amp;rft.volume=1&amp;rft.issue=6&amp;rft.date=2011-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yang%2C+Yajie&amp;rft.au=Graze%2C+Rita+M.&amp;rft.au=Walts%2C+Brandon+M.&amp;rft.au=Lopez%2C+Cecilia+M.&amp;rft.au=Baker%2C+Henry+V.&amp;rft.au=Wayne%2C+Marta+L.&amp;rft.au=Nuzhdin%2C+Sergey+V.&amp;rft.au=McIntyre%2C+Lauren+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459201/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459201/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459201/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459201/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459201/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459201/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459201/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459201/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459201/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459201/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459201/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459201/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/6/427" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/6/427.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/6/427.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/6/427.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/6/427.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
